GO:0000015
name: phosphopyruvate hydratase complex
def: "A multimeric enzyme complex, usually a dimer or an octamer, that catalyzes the conversion of 2-phospho-D-glycerate to phosphoenolpyruvate and water." [GOC:jl, ISBN:0198506732]
GO:0000108
name: repairosome
def: "OBSOLETE. A stable complex of proteins that carry out the DNA damage recognition and incision reactions characteristic of nucleotide excision repair (NER), such as DNA damage recognition, DNA helix unwinding, and endonucleolytic cleavage at sites flanking damaged DNA; includes TFIIH subunits and additional polypeptides; may form in the absence of DNA damage." [PMID:10681587, PMID:9852079]
GO:0000109
name: nucleotide-excision repair complex
def: "Any complex formed of proteins that act in nucleotide-excision repair." [PMID:10915862]
GO:0000110
name: nucleotide-excision repair factor 1 complex
def: "One of several protein complexes involved in nucleotide-excision repair; possesses DNA damage recognition and endodeoxynuclease activities. In S. cerevisiae, it is composed of Rad1p, Rad10p, and Rad14p; in human the subunits are ERCC4/XPF, ERCC1 and XPA, respectively." [PMID:10915862]
GO:0000111
name: nucleotide-excision repair factor 2 complex
def: "One of several protein complexes involved in nucleotide-excision repair; possesses damaged DNA binding activity. In S. cerevisiae, it is composed of Rad4p and Rad23p." [PMID:10915862]
GO:0000112
name: nucleotide-excision repair factor 3 complex
def: "One of several protein complexes involved in nucleotide-excision repair; possesses endodeoxynuclease and DNA helicase activities. In S. cerevisiae, it is composed of Rad2p and the core TFIIH-Ssl2p complex (core TFIIH is composed of Rad3p, Tfb1p, Tfb2p, Ssl1p, Tfb4p and Tfb5p. Note that Ssl2p is also called Rad25p)." [GOC:ew, PMID:10915862, PMID:14500720, PMID:7813015]
GO:0000113
name: nucleotide-excision repair factor 4 complex
def: "One of several protein complexes involved in nucleotide-excision repair; possesses DNA damage recognition and DNA-dependent ATPase activities. In S. cerevisiae, it is composed of Rad7p and Rad16p." [PMID:10915862]
GO:0000118
name: histone deacetylase complex
def: "A protein complex that possesses histone deacetylase activity." [GOC:mah]
GO:0000120
name: RNA polymerase I transcription factor complex
def: "A transcription factor complex that acts at promoters of genes transcribed by RNA polymerase I." [GOC:mah]
GO:0000123
name: histone acetyltransferase complex
def: "A protein complex that possesses histone acetyltransferase activity." [GOC:mah]
GO:0000124
name: SAGA complex
def: "A SAGA-type histone acetyltransferase complex that contains Spt8 (in budding yeast) or a homolog thereof; additional polypeptides include Spt group, consisting of Spt7, Spt3, and Spt20/Ada5, which interact with the TATA-binding protein (TBP); the Ada group, consisting of Ada1, Ada2, Ada3, Ada4/Gcn5, and Ada5/Spt20, which is functionally linked to the nucleosomal HAT activity; Tra1, an ATM/PI-3 kinase-related protein that targets DNA-bound activators for recruitment to promoters; the TBP-associated factor (TAF) proteins, consisting of Taf5, Taf6, Taf9, Taf10, and Taf12, which mediate nucleosomal HAT activity and are thought to help recruit the basal transcription machinery." [PMID:10637607, PMID:17337012]
GO:0000125
name: PCAF complex
def: "A large multiprotein complex that possesses histone acetyltransferase activity and is involved in regulation of transcription. The composition is similar to that of the SAGA complex, but includes fewer Spt and Ada proteins, and more TAFs." [PMID:10637607]
GO:0000126
name: transcription factor TFIIIB complex
def: "A transcription factor complex that is involved in regulating transcription from RNA polymerase III (Pol III) promoters. TFIIIB contains the TATA-binding protein (TBP) and two Pol III-specific proteins, B'' and BRF." [GOC:mah, PMID:11433012]
GO:0000127
name: transcription factor TFIIIC complex
def: "A heterotrimeric transcription factor complex that is involved in regulating transcription from RNA polymerase III (Pol III) promoters. TFIIIC contains three conserved subunits that associate with the proximal Pol III promoter element, and additional subunits that associate with sequence elements downstream of the promoter and are more diverged among species. It also functions as a boundary element to partition genome content into distinct domains outside Pol III promoter regions." [GOC:mah, GOC:vw, PMID:11433012, PMID:16751097]
GO:0000131
name: incipient cellular bud site
def: "The portion of the budding yeast plasma membrane where a daughter cell will emerge. The yeast marks this spot with bud-site selection proteins before bud emergence occurs. Actin is polarized to this spot just prior to and during bud emergence." [GOC:clt]
GO:0000133
name: polarisome
def: "Protein complex that plays a role in determining cell polarity by directing the localized assembly of actin filaments at polarization sites; in Saccharomyces the polarisome includes Bni1p, Spa2p, Pea2p, and Bud6p." [PMID:14734532, PMID:14998522, PMID:9632790]
GO:0000136
name: alpha-1,6-mannosyltransferase complex
def: "A large, multiprotein complex with alpha-(1->6)-mannosyltransferase activity, located in the cis Golgi membrane; adds mannan to N-linked glycans on proteins." [GOC:mcc, PMID:10037752, PMID:11095735, PMID:18083825]
GO:0000137
name: Golgi cis cisterna
def: "The Golgi cisterna closest to the endoplasmic reticulum; the first processing compartment through which proteins pass after export from the ER." [ISBN:0815316194]
GO:0000138
name: Golgi trans cisterna
def: "The Golgi cisterna farthest from the endoplasmic reticulum; the final processing compartment through which proteins pass before exiting the Golgi apparatus; the compartment in which N-linked protein glycosylation is completed." [ISBN:0815316194]
GO:0000139
name: Golgi membrane
def: "The lipid bilayer surrounding any of the compartments of the Golgi apparatus." [GOC:mah]
GO:0000142
name: cellular bud neck contractile ring
def: "A contractile ring, i.e. a cytoskeletal structure composed of actin filaments and myosin, that forms beneath the plasma membrane at the mother-bud neck in mitotic cells that divide by budding in preparation for completing cytokinesis. An example of this structure is found in Saccharomyces cerevisiae." [GOC:krc, PMID:16009555]
GO:0000144
name: cellular bud neck septin ring
def: "A ring-shaped structure that forms at the site of cytokinesis; composed of members of the conserved family of filament forming proteins called septins as well as septin-associated proteins. In S. cerevisiae, this structure forms at the time of bud emergence and the septins show a high rate of exchange." [GOC:krc, PMID:16009555]
GO:0000145
name: exocyst
def: "A protein complex peripherally associated with the plasma membrane that determines where vesicles dock and fuse. At least eight complex components are conserved between yeast and mammals." [PMID:9700152]
GO:0000148
name: 1,3-beta-D-glucan synthase complex
def: "A protein complex that catalyzes the transfer of a glucose group from UDP-glucose to a (1->3)-beta-D-glucan chain." [EC:2.4.1.34]
GO:0000151
name: ubiquitin ligase complex
def: "A protein complex that includes a ubiquitin-protein ligase and other proteins that may confer substrate specificity on the complex." [GOC:jh2, PMID:9529603]
GO:0000152
name: nuclear ubiquitin ligase complex
def: "A ubiquitin ligase complex found in the nucleus." [GOC:mah]
GO:0000153
name: cytoplasmic ubiquitin ligase complex
def: "A ubiquitin ligase complex found in the cytoplasm." [GOC:mah]
GO:0000159
name: protein phosphatase type 2A complex
def: "A protein complex that has protein serine/threonine phosphatase activity that is polycation-stimulated (PCS), being directly stimulated by protamine, polylysine, or histone H1; it constitutes a subclass of several enzymes activated by different histones and polylysine, and consists of catalytic, scaffolding, and regulatory subunits. The catalytic and scaffolding subunits form the core enzyme, and the holoenzyme also includes the regulatory subunit." [GOC:mah, ISBN:0198547684, PMID:17245430]
GO:0000164
name: protein phosphatase type 1 complex
def: "A protein complex that possesses magnesium-dependent protein serine/threonine phosphatase (AMD phosphatase) activity, and consists of a catalytic subunit and one or more regulatory subunits that dictates the phosphatase's substrate specificity, function, and activity." [GOC:mah, GOC:ssd]
GO:0000172
name: ribonuclease MRP complex
def: "A ribonucleoprotein complex that contains an RNA molecule of the snoRNA family, and cleaves the rRNA precursor as part of rRNA transcript processing. It also has other roles: In S. cerevisiae it is involved in cell cycle-regulated degradation of daughter cell-specific mRNAs, while in mammalian cells it also enters the mitochondria and processes RNAs to create RNA primers for DNA replication." [GOC:sgd_curators, PMID:10690410, PMID:14729943, PMID:7510714]
GO:0000176
name: nuclear exosome (RNase complex)
def: "Complex of 3'-5' exoribonucleases found in the nucleus." [PMID:10465791]
GO:0000177
name: cytoplasmic exosome (RNase complex)
def: "Complex of 3'-5' exoribonucleases found in the cytoplasm." [PMID:10465791]
GO:0000178
name: exosome (RNase complex)
def: "Complex of 3'-5' exoribonucleases." [PMID:10465791]
GO:0000180
name: cytosolic large ribosomal subunit
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0000181
name: cytosolic small ribosomal subunit
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0000214
name: tRNA-intron endonuclease complex
def: "A protein complex that catalyzes the endonucleolytic cleavage of pre-tRNA, producing 5'-hydroxyl and 2',3'-cyclic phosphate termini, and specifically removing the intron." [EC:3.1.27.9]
GO:0000219
name: vacuolar hydrogen-transporting ATPase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0000220
name: vacuolar proton-transporting V-type ATPase, V0 domain
def: "The V0 domain of a proton-transporting V-type ATPase found in the vacuolar membrane." [GOC:mah, PMID:16449553]
GO:0000221
name: vacuolar proton-transporting V-type ATPase, V1 domain
def: "The V1 domain of a proton-transporting V-type ATPase found in the vacuolar membrane." [GOC:mah, PMID:16449553]
GO:0000222
name: plasma membrane proton-transporting V-type ATPase, V0 domain
def: "The V0 domain of a proton-transporting V-type ATPase found in the plasma membrane." [GOC:mah]
GO:0000223
name: plasma membrane proton-transporting V-type ATPase, V1 domain
def: "The V1 domain of a proton-transporting V-type ATPase found in the plasma membrane." [GOC:mah]
GO:0000228
name: nuclear chromosome
def: "A chromosome found in the nucleus of a eukaryotic cell." [GOC:mah]
GO:0000229
name: cytoplasmic chromosome
def: "A chromosome found in the cytoplasm." [GOC:mah]
GO:0000230
name: nuclear mitotic chromosome
def: "OBSOLETE. A chromosome found in the nucleus during mitosis." [GOC:mah]
GO:0000231
name: cytoplasmic mitotic chromosome
def: "OBSOLETE. A chromosome found in the cytoplasm during mitosis." [GOC:mah]
GO:0000232
name: nuclear interphase chromosome
def: "OBSOLETE. A chromosome found in the nucleus during interphase." [GOC:mah]
GO:0000233
name: cytoplasmic interphase chromosome
def: "OBSOLETE. A chromosome found in the cytoplasm during interphase." [GOC:mah]
GO:0000235
name: astral microtubule
def: "Any of the spindle microtubules that radiate in all directions from the spindle poles and are thought to contribute to the forces that separate the poles and position them in relation to the rest of the cell." [ISBN:0815316194]
GO:0000242
name: pericentriolar material
def: "A network of small fibers that surrounds the centrioles in cells; contains the microtubule nucleating activity of the centrosome." [GOC:clt, ISBN:0815316194]
GO:0000243
name: commitment complex
def: "A spliceosomal complex that is formed by association of the U1 snRNP with the 5' splice site of an unspliced intron in an RNA transcript." [GOC:krc, ISBN:0879695897, PMID:9150140]
GO:0000262
name: mitochondrial chromosome
def: "A chromosome found in the mitochondrion of a eukaryotic cell." [GOC:mah]
GO:0000267
name: cell fraction
def: "OBSOLETE: A generic term for parts of cells prepared by disruptive biochemical techniques." [GOC:ma]
GO:0000274
name: mitochondrial proton-transporting ATP synthase, stator stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the mitochondrial membrane-associated F0 proteins; is thought to prevent futile rotation of the catalytic core." [GOC:mtg_sensu, PMID:10838056]
GO:0000275
name: mitochondrial proton-transporting ATP synthase complex, catalytic core F(1)
def: "The catalytic sector of the mitochondrial hydrogen-transporting ATP synthase; it comprises the catalytic core and central stalk, and is peripherally associated with the mitochondrial inner membrane when the entire ATP synthase is assembled." [GOC:mtg_sensu, PMID:10838056]
GO:0000276
name: mitochondrial proton-transporting ATP synthase complex, coupling factor F(o)
def: "All non-F1 subunits of the mitochondrial hydrogen-transporting ATP synthase, including integral and peripheral mitochondrial inner membrane proteins." [GOC:mtg_sensu, PMID:10838056]
GO:0000299
name: integral to membrane of membrane fraction
def: "OBSOLETE: Integral to that fraction of cells, prepared by disruptive biochemical methods, that includes the plasma and other membranes; require detergents, such as Triton X-100, to be released from membranes." [PMID:10512869]
GO:0000300
name: peripheral to membrane of membrane fraction
def: "OBSOLETE: Peripheral to that fraction of cells, prepared by disruptive biochemical methods, that includes the plasma and other membranes; can be extracted from membrane fraction with high concentrations of salt or high pH." [PMID:10512869]
GO:0000306
name: extrinsic component of vacuolar membrane
def: "The component of a vacuolar membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:jl, GOC:mah]
GO:0000307
name: cyclin-dependent protein kinase holoenzyme complex
def: "Cyclin-dependent protein kinases (CDKs) are enzyme complexes that contain a kinase catalytic subunit associated with a regulatory cyclin partner." [GOC:krc, PMID:11602261]
GO:0000308
name: cytoplasmic cyclin-dependent protein kinase holoenzyme complex
def: "Cyclin-dependent protein kinase (CDK) complex found in the cytoplasm." [GOC:krc]
GO:0000311
name: plastid large ribosomal subunit
def: "The larger of the two subunits of a plastid ribosome. Two sites on the ribosomal large subunit are involved in translation: the aminoacyl site (A site) and peptidyl site (P site)." [GOC:mcc]
GO:0000312
name: plastid small ribosomal subunit
def: "The smaller of the two subunits of a plastid ribosome." [GOC:mcc]
GO:0000313
name: organellar ribosome
def: "A ribosome contained within a subcellular membrane-bounded organelle." [GOC:mah, GOC:mcc]
GO:0000314
name: organellar small ribosomal subunit
def: "The smaller of the two subunits of an organellar ribosome." [GOC:mcc]
GO:0000315
name: organellar large ribosomal subunit
def: "The larger of the two subunits of an organellar ribosome. Two sites on the ribosomal large subunit are involved in translation: the aminoacyl site (A site) and peptidyl site (P site)." [GOC:mcc]
GO:0000322
name: storage vacuole
def: "A vacuole that functions primarily in the storage of materials, including nutrients, pigments, waste products, and small molecules." [GOC:krc]
GO:0000323
name: lytic vacuole
def: "A vacuole that is maintained at an acidic pH and which contains degradative enzymes, including a wide variety of acid hydrolases." [GOC:krc]
GO:0000324
name: fungal-type vacuole
def: "A vacuole that has both lytic and storage functions. The fungal vacuole is a large, membrane-bounded organelle that functions as a reservoir for the storage of small molecules (including polyphosphate, amino acids, several divalent cations (e.g. calcium), other ions, and other small molecules) as well as being the primary compartment for degradation. It is an acidic compartment, containing an ensemble of acid hydrolases. At least in S. cerevisiae, there are indications that the morphology of the vacuole is variable and correlated with the cell cycle, with logarithmically growing cells having a multilobed, reticulated vacuole, while stationary phase cells contain a single large structure." [GOC:mah, GOC:mtg_sensu, ISBN:0879693649]
GO:0000325
name: plant-type vacuole
def: "A closed structure that is completely surrounded by a unit membrane, contains liquid, and retains the same shape regardless of cell cycle phase. An example of this structure is found in Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:0815316208]
GO:0000326
name: protein storage vacuole
def: "A storage vacuole that contains a lytic vacuole; identified in plants." [PMID:11739409]
GO:0000327
name: lytic vacuole within protein storage vacuole
def: "A membrane-bounded compartment containing crystals of phytic acid and proteins characteristic of a lytic vacuole, found within a storage vacuole." [PMID:11739490]
GO:0000328
name: fungal-type vacuole lumen
def: "The volume enclosed within the vacuolar membrane of a vacuole, the shape of which correlates with cell cycle phase. An example of this structure is found in Saccharomyces cerevisiae." [GOC:krc, GOC:mtg_sensu]
GO:0000329
name: fungal-type vacuole membrane
def: "The lipid bilayer surrounding a vacuole, the shape of which correlates with cell cycle phase. The membrane separates its contents from the cytoplasm of the cell. An example of this structure is found in Saccharomyces cerevisiae." [GOC:krc, GOC:mtg_sensu]
GO:0000330
name: plant-type vacuole lumen
def: "The volume enclosed within the vacuolar membrane of a vacuole that retains the same shape regardless of cell cycle phase. An example of this is found in Arabidopsis thaliana." [GOC:krc, GOC:mtg_sensu]
GO:0000331
name: contractile vacuole
def: "A specialized vacuole of eukaryotic cells, especially Protozoa, that fills with water from the cytoplasm and then discharges this externally by the opening of contractile vacuole pores. Its function is probably osmoregulatory." [GOC:jl, PMID:10503189]
GO:0000333
name: telomerase catalytic core complex
def: "The minimal catalytic core of telomerase is a ribonucleoprotein complex composed of a catalytic reverse transcriptase subunit and an RNA subunit that provides the template for telomeric DNA addition." [PMID:11884619]
GO:0000343
name: plastid-encoded plastid RNA polymerase complex A
def: "A plastid-encoded DNA-directed RNA polymerase complex that resembles eubacterial multisubunit RNA polymerases, with a core composed of alpha, beta, and beta-prime subunits. An additional subunit, a sigma factor, is required for promoter recognition. PEP-A is generated from the PEP-B form during chloroplast maturation to generate a complex composed of at least thirteen polypeptides that is not sensitive to the antibiotic rifampicin, like its precursor form the PEP-B complex." [PMID:10946105]
GO:0000344
name: plastid-encoded plastid RNA polymerase complex B
def: "A plastid-encoded DNA-directed RNA polymerase complex that resembles eubacterial multisubunit RNA polymerases with a core composed of alpha, beta, and beta-prime subunits. An additional subunit, a sigma factor, is required for promoter recognition. PEP-B is distinguished from PEP-A by its sensitivity to the antibiotic rifampicin. PEP-B is found in both etioplasts and chloroplasts, but is the predominate form in etioplasts. It forms the core of the PEP-A form; the conversion from PEP-B to PEP-A occurs during chloroplast maturation." [PMID:10946105]
GO:0000345
name: cytosolic DNA-directed RNA polymerase complex
def: "The eubacterial DNA-directed RNA polymerase is a multisubunit complex with a core composed of the essential subunits beta-prime, beta, and two copies of alpha and a fifth nonessential subunit called omega. An additional subunit, a sigma factor, is required for promoter recognition and specificity." [PMID:11158566]
GO:0000346
name: transcription export complex
def: "The transcription export (TREX) complex couples transcription elongation by RNA polymerase II to mRNA export. The complex associates with the polymerase and travels with it along the length of the transcribed gene. TREX is composed of the THO transcription elongation complex as well as other proteins that couple THO to mRNA export proteins. The TREX complex is known to be found in a wide range of eukaryotes, including S. cerevisiae and metazoans." [GOC:krc, PMID:11979277]
GO:0000347
name: THO complex
def: "The THO complex is a nuclear complex that is required for transcription elongation through genes containing tandemly repeated DNA sequences. The THO complex is also part of the TREX (TRanscription EXport) complex that is involved in coupling transcription to export of mRNAs to the cytoplasm. In S. cerevisiae, it is composed of four subunits: Hpr1p, Tho2p, Thp1p, and Mft1p, while the human complex is composed of 7 subunits." [GOC:krc, PMID:11060033, PMID:11979277, PMID:16983072]
GO:0000399
name: cellular bud neck septin structure
def: "Any of a series of septin structures that are localized in the bud neck of a budding fungal cell during the cell cycle." [GOC:krc]
GO:0000407
name: pre-autophagosomal structure
def: "A punctate structure localized in the vicinity of the vacuole that is required for the formation of autophagosomes." [GOC:elh, PMID:11689437, PMID:12048214, PMID:12554655]
GO:0000408
name: EKC/KEOPS complex
def: "A protein complex proposed to be involved in transcription as well as promoting telomere uncapping and telomere elongation. For example, in Saccharomyces cerevisiae the complex contains Bud32p, Kae1p, Gon7p, Cgi121p, and Pcc1p." [GOC:elh, PMID:16564010, PMID:16874308]
GO:0000417
name: HIR complex
def: "A protein complex proposed to be involved in replication-independent nucleosome assembly, by promoting histone deposition onto DNA. For example, in Saccharomyces, the complex contains Hir1p, Hir2p, Hir3p, and Hpc2p." [GOC:elh, GOC:mah, PMID:16303565, PMID:17180700]
GO:0000418
name: DNA-directed RNA polymerase IV complex
def: "RNA polymerase IV is a multisubunit RNA polymerase complex found in the nucleus of plants and involved in accumulation of siRNAs and in DNA methylation-dependent silencing of endogenous repeated sequences. Pol IV is composed of subunits that are paralogous or identical to the 12 subunits of Pol II. The largest and second-largest subunits of Pol IV are the catalytic subunits and share similarity with the corresponding subunits of other eukaryotic and bacterial multisubunit RNA polymerases. The second largest subunit is also found in RNA polymerase V, while the largest subunit is found only in RNAP IV complex." [GOC:krc, GOC:mtg_sensu, PMID:15692015, PMID:15766525, PMID:16140984, PMID:19110459]
GO:0000419
name: DNA-directed RNA polymerase V complex
def: "RNA polymerase V is a multisubunit RNA polymerase complex found in the nucleus of plants and involved in accumulation of siRNAs and in DNA methylation-dependent silencing of endogenous repeated sequences. Pol V is composed of subunits that are paralogous or identical to the 12 subunits of Pol II. Two large subunits comprise the most conserved portion including the catalytic site and share similarity with other eukaryotic and bacterial multisubunit RNA polymerases. The second largest subunit is also found in RNA polymerase IVa, while the largest subunit is found only in the IVa complex and contains an extended C-terminal domain (CTD) that includes multiple repeats of a 16 amino-acid consensus sequence as well as other sequences. The remainder of the complex is composed of smaller subunits." [GOC:krc, GOC:mtg_sensu, PMID:16140984, PMID:19110459]
GO:0000421
name: autophagic vacuole membrane
def: "The lipid bilayer surrounding an autophagic vacuole, a double-membrane-bounded vesicle in which endogenous cellular material is sequestered." [GOC:isa_complete]
GO:0000427
name: plastid-encoded plastid RNA polymerase complex
def: "An RNA polymerase complex containing polypeptides encoded by the plastid genome. Plastid-encoded DNA-directed RNA polymerases resemble eubacterial multisubunit RNA polymerases, with a core composed of alpha, beta, and beta-prime subunits. Some forms contain multiple additional subunits. An additional sigma factor subunit is required for promoter recognition." [GOC:krc, GOC:mah, GOC:pj]
GO:0000428
name: DNA-directed RNA polymerase complex
def: "A protein complex that possesses DNA-directed RNA polymerase activity." [GOC:krc]
GO:0000438
name: core TFIIH complex portion of holo TFIIH complex
def: "The core TFIIH complex when it is part of the general transcription factor TFIIH." [GOC:ew, GOC:krc, PMID:14500720, PMID:22308316, PMID:22572993, PMID:7813015]
GO:0000439
name: core TFIIH complex
def: "The 7 subunit core of TFIIH that is a part of either the general transcription factor holo-TFIIH or the nucleotide-excision repair factor 3 complex. In S. cerevisiae/humans the complex is composed of: Ssl2/XPB, Tfb1/p62, Tfb2/p52, Ssl1/p44, Tfb4/p34, Tfb5/p8 and Rad3/XPD." [GOC:ew, GOC:krc, PMID:14500720, PMID:17215295, PMID:22308316, PMID:22572993, PMID:23028141, PMID:7813015]
GO:0000440
name: core TFIIH complex portion of NEF3 complex
def: "The core TFIIH complex when it is part of the nucleotide-excision repair factor 3 (NEF3)." [GOC:ew, GOC:krc, PMID:14500720, PMID:22308316, PMID:22572993, PMID:7813015]
GO:0000444
name: MIS12/MIND type complex
def: "A multiprotein kinetochore subcomplex that binds to centromeric chromatin and forms part of the inner kinetochore. It helps to recruit outer kinetochore subunits that will bind to microtubules. In humans, it consists of MIS12, DSN1, NSL1 and PMF1." [GOC:krc, PMID:14633972, PMID:16585270]
GO:0000445
name: THO complex part of transcription export complex
def: "The THO complex when it is part of the TREX (TRanscription EXport) complex that is involved in coupling transcription to export of mRNAs to the cytoplasm. In S. cerevisiae, it is composed of four subunits: Hpr1, Tho2, Thp1, and Mft1, while the human complex is composed of 7 subunits." [GOC:krc, PMID:11060033, PMID:11979277, PMID:16983072]
GO:0000446
name: nucleoplasmic THO complex
def: "The THO complex when it is acting as a nuclear complex that is required for transcription elongation through genes containing tandemly repeated DNA sequences. In S. cerevisiae, it is composed of four subunits: Hpr1, Tho2, Thp2, and Mft1, while the human complex is composed of 7 subunits." [GOC:krc, GOC:se, PMID:11060033, PMID:11979277, PMID:16983072]
GO:0000500
name: RNA polymerase I upstream activating factor complex
def: "A complex required for the transcription of rDNA by RNA polymerase I. In yeast the complex consists of Rrrn5p, Rrn9p, Rrn10p, histones H3 and H4, and Uaf30p." [PMID:11500378]
GO:0000502
name: proteasome complex
def: "A large multisubunit complex which catalyzes protein degradation, found in eukaryotes, archaea and some bacteria. In eukaryotes, this complex consists of the barrel shaped proteasome core complex and one or two associated proteins or complexes that act in regulating entry into or exit from the core." [GOC:rb, http://en.wikipedia.org/wiki/Proteasome]
GO:0000504
name: proteasome regulatory particle (sensu Bacteria)
def: "OBSOLETE. A multisubunit complex that recognizes and unfolds ubiquitinated proteins, and translocates them to the core complex in an ATP dependent manner. As in, but not restricted to, the taxon Bacteria (Bacteria, ncbi_taxonomy_id:2)." [GOC:rb]
GO:0000506
name: glycosylphosphatidylinositol-N-acetylglucosaminyltransferase (GPI-GnT) complex
def: "An enzyme complex that catalyzes the transfer of GlcNAc from UDP-GlcNAc to an acceptor phosphatidylinositol, the first step in the production of GPI anchors for cell surface proteins. The complex contains PIG-A, PIG-C, PIG-H, PIG-Q, PIG-P, and DPM2 in human, and Eri1p, Gpi1p, Gpi2p, Gpi15p, Gpi19p, and Spt14p in budding yeast." [GOC:kp, GOC:rb, PMID:10944123, PMID:15163411]
GO:0000775
name: chromosome, centromeric region
def: "The region of a chromosome that includes the centromeric DNA and associated proteins. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome." [GOC:cjm, GOC:elh, GOC:kmv, GOC:pr]
GO:0000776
name: kinetochore
def: "A multisubunit complex that is located at the centromeric region of DNA and provides an attachment point for the spindle microtubules." [GOC:elh]
GO:0000777
name: condensed chromosome kinetochore
def: "A multisubunit complex that is located at the centromeric region of a condensed chromosome and provides an attachment point for the spindle microtubules." [GOC:elh]
GO:0000778
name: condensed nuclear chromosome kinetochore
def: "A multisubunit complex that is located at the centromeric region of a condensed chromosome in the nucleus and provides an attachment point for the spindle microtubules." [GOC:elh]
GO:0000779
name: condensed chromosome, centromeric region
def: "The region of a condensed chromosome that includes the centromere and associated proteins, including the kinetochore. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome." [GOC:elh, GOC:kmv]
GO:0000780
name: condensed nuclear chromosome, centromeric region
def: "The region of a condensed chromosome in the nucleus that includes the centromere and associated proteins, including the kinetochore. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome." [GOC:elh, GOC:kmv]
GO:0000781
name: chromosome, telomeric region
def: "The terminal region of a linear chromosome that includes the telomeric DNA repeats and associated proteins." [GOC:elh]
GO:0000782
name: telomere cap complex
def: "A complex of DNA and protein located at the end of a linear chromosome that protects and stabilizes a linear chromosome." [GOC:elh]
GO:0000783
name: nuclear telomere cap complex
def: "A complex of DNA and protein located at the end of a linear chromosome in the nucleus that protects and stabilizes a linear chromosome." [GOC:elh]
GO:0000784
name: nuclear chromosome, telomeric region
def: "The terminal region of a linear chromosome in the nucleus that includes the telomeric DNA repeats and associated proteins." [GOC:elh]
GO:0000785
name: chromatin
def: "The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome." [GOC:elh, PMID:20404130]
GO:0000786
name: nucleosome
def: "A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA into higher order structures." [GOC:elh]
GO:0000787
name: cytoplasmic nucleosome
def: "A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA in the cytoplasm into higher order structures." [GOC:elh]
GO:0000788
name: nuclear nucleosome
def: "A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA in the nucleus into higher order structures." [GOC:elh]
GO:0000789
name: cytoplasmic chromatin
def: "The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome in the cytoplasm." [GOC:elh, PMID:20404130]
GO:0000790
name: nuclear chromatin
def: "The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome in the nucleus." [GOC:elh, PMID:20404130]
GO:0000791
name: euchromatin
def: "A dispersed and relatively uncompacted form of chromatin." [GOC:elh]
GO:0000792
name: heterochromatin
def: "A compact and highly condensed form of chromatin." [GOC:elh]
GO:0000793
name: condensed chromosome
def: "A highly compacted molecule of DNA and associated proteins resulting in a cytologically distinct structure." [GOC:elh]
GO:0000794
name: condensed nuclear chromosome
def: "A highly compacted molecule of DNA and associated proteins resulting in a cytologically distinct structure that remains in the nucleus." [GOC:elh]
GO:0000795
name: synaptonemal complex
def: "A proteinaceous scaffold found between homologous chromosomes during meiosis." [GOC:elh]
GO:0000796
name: condensin complex
def: "A multisubunit protein complex that plays a central role in chromosome condensation." [GOC:elh]
GO:0000797
name: condensin core heterodimer
def: "The core heterodimer of a condensin complex, a multisubunit protein complex that plays a central role in chromosome condensation." [GOC:elh]
GO:0000798
name: nuclear cohesin complex
def: "A cohesin complex required for cohesion between sister chromatids that remain in the nucleus." [GOC:elh]
GO:0000799
name: nuclear condensin complex
def: "A multisubunit protein complex that plays a central role in the condensation of chromosomes that remain in the nucleus." [GOC:elh]
GO:0000800
name: lateral element
def: "A proteinaceous core found between sister chromatids during meiotic prophase." [GOC:elh]
GO:0000801
name: central element
def: "A structural unit of the synaptonemal complex found between the lateral elements." [GOC:elh]
GO:0000802
name: transverse filament
def: "A structural unit of the synaptonemal complex that spans the regions between the lateral elements and connects them." [GOC:elh]
GO:0000803
name: sex chromosome
def: "A chromosome involved in sex determination." [GOC:elh]
GO:0000804
name: W chromosome
def: "The sex chromosome present in females of species in which the female is the heterogametic sex; generally, the sex chromosome that pairs with the Z chromosome in the heterogametic sex. The W chromosome is absent from the cells of males and present in one copy in the somatic cells of females." [GOC:mah, ISBN:0321000382]
GO:0000805
name: X chromosome
def: "The sex chromosome present in both sexes of species in which the male is the heterogametic sex. Two copies of the X chromosome are present in each somatic cell of females and one copy is present in males." [GOC:mah, ISBN:0582227089]
GO:0000806
name: Y chromosome
def: "The sex chromosome present in males of species in which the male is the heterogametic sex; generally, the sex chromosome that pairs with the X chromosome in the heterogametic sex. The Y chromosome is absent from the cells of females and present in one copy in the somatic cells of males." [GOC:mah, ISBN:0582227089]
GO:0000807
name: Z chromosome
def: "The sex chromosome present in both sexes of species in which the female is the heterogametic sex. Two copies of the Z chromosome are present in each somatic cell of males and one copy is present in females." [GOC:mah, ISBN:0321000382]
GO:0000808
name: origin recognition complex
def: "A multisubunit complex that is located at the replication origins of a chromosome." [GOC:elh]
GO:0000809
name: cytoplasmic origin of replication recognition complex
def: "A multisubunit complex that is located at the replication origins of a chromosome in the cytoplasm." [GOC:elh]
GO:0000811
name: GINS complex
def: "A heterotetrameric protein complex that associates with replication origins, where it is required for the initiation of DNA replication, and with replication forks." [GOC:rb, GOC:rn, PMID:12730134, PMID:16990792, PMID:17467990]
GO:0000812
name: Swr1 complex
def: "A multisubunit protein complex that is involved in chromatin remodeling. It is required for the incorporation of the histone variant H2AZ into chromatin. In S. cerevisiae, the complex contains Swr1p, a Swi2/Snf2-related ATPase, and 12 additional subunits." [GOC:rb, PMID:14645854, PMID:14690608, PMID:19355820]
GO:0000813
name: ESCRT I complex
def: "An endosomal sorting complex required for transport. It consists of the class E vacuolar protein sorting (Vps) proteins and interacts with ubiquitinated cargoes." [GOC:rb, PMID:12892785, PMID:12900393]
GO:0000814
name: ESCRT II complex
def: "An endosomal sorting complex required for transport and functions downstream of ESCRT I complex. It consists of the class E vacuolar protein sorting (Vps) proteins and is required for the membrane recruitment of ESCRT III complex and binds to ubiquitinated cargoes." [GOC:rb, PMID:12892785, PMID:12900393]
GO:0000815
name: ESCRT III complex
def: "An endosomal sorting complex required for transport. Consists of two soluble subcomplexes of highly charged coiled-coil proteins and is required for sorting and/or concentration of multivesicular body (MVB) cargoes." [GOC:rb, PMID:12892785, PMID:12900393]
GO:0000817
name: COMA complex
def: "A multiprotein complex in yeast consisting of Ctf19p, Okp1p, Mcm21p, and Ame1p. This complex bridges the subunits that are in contact with centromeric DNA and the subunits bound to microtubules during kinetochore assembly." [GOC:se, PMID:14633972]
GO:0000818
name: nuclear MIS12/MIND complex
def: "A multiprotein kinetochore subcomplex that binds to centromeric chromatin and forms part of the inner kinetochore of a chromosome in the nucleus. It helps to recruit outer kinetochore subunits that will bind to microtubules. Nuclear localization arises in some organisms because the nuclear envelope is not broken down during mitosis. In S. cerevisiae, it consists of at least four proteins: Mtw1p, Nnf1p, Nsl1p, and Dsn1." [GOC:krc, GOC:se, PMID:14633972]
GO:0000835
name: ER ubiquitin ligase complex
def: "A ubiquitin ligase complex found in the ER." [GOC:elh]
GO:0000836
name: Hrd1p ubiquitin ligase complex
def: "A multiprotein complex that recognizes and ubiquitinates proteins with misfolded luminal and membrane domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p." [GOC:elh, PMID:16873066]
GO:0000837
name: Doa10p ubiquitin ligase complex
def: "A multiprotein complex that recognizes and ubiquitinates membrane proteins with misfolded cytosolic domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Ssm4p/Doa10p." [GOC:elh, PMID:16873066]
GO:0000838
name: Hrd1p ubiquitin ligase ERAD-M complex
def: "A multiprotein complex that recognizes and ubiquitinates proteins with misfolded membrane domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p." [GOC:elh, PMID:16873066]
GO:0000839
name: Hrd1p ubiquitin ligase ERAD-L complex
def: "A multiprotein complex that recognizes and ubiquitinates proteins with misfolded luminal domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p." [GOC:elh, PMID:16873065, PMID:16873066]
GO:0000922
name: spindle pole
def: "Either of the ends of a spindle, where spindle microtubules are organized; usually contains a microtubule organizing center and accessory molecules, spindle microtubules and astral microtubules." [GOC:clt]
GO:0000923
name: equatorial microtubule organizing center
def: "A microtubule organizing center formed by a band of gamma-tubulin that is recruited to a circumferential band of F-actin at the midpoint of a cell and which nucleates microtubules from the cell division site at the end of mitosis." [PMID:11792817]
GO:0000924
name: gamma-tubulin ring complex, centrosomal
def: "A centrosome-localized multiprotein complex composed of gamma-tubulin and other non-tubulin proteins assembled into a ring structure that is thought to be the unit of nucleation at the minus end of a microtubule. Gamma-tubulin small complexes are thought to be the core repeating units of the ring." [GOC:clt, GOC:mtg_sensu, PMID:12134075, PMID:17021256]
GO:0000927
name: gamma-tubulin small complex, centrosomal
def: "A centrosomal complex usually comprising two gamma-tubulin molecules, at least two conserved non-tubulin proteins that multimerize along with additional non-tubulin proteins in animal cells into larger functional complexes. Gamma-tubulin small complexes are thought to be the repeating unit making up the core of the gamma-tubulin ring complex. An example of this structure is found in Mus musculus." [GOC:mtg_sensu, PMID:12134075]
GO:0000928
name: gamma-tubulin small complex, spindle pole body
def: "A complex composed of two gamma-tubulin molecules and conserved non-tubulin proteins located in the spindle pole body and isolated by fractionation from cells. The complex, approximately 6S-9S, is analogous to the small complex in animal cells but contains fewer subunits, and is not thought to multimerize into larger functional units, like complexes in those organisms. An example of this structure is found in Saccharomyces cerevisiae." [GOC:mtg_sensu, PMID:12134075]
GO:0000930
name: gamma-tubulin complex
def: "A multiprotein complex composed of gamma-tubulin and other non-tubulin proteins. Gamma-tubulin complexes are localized to microtubule organizing centers, and play an important role in the nucleation of microtubules. The number and complexity of non-tubulin proteins associated with these complexes varies between species." [GOC:clt, PMID:12134075]
GO:0000931
name: gamma-tubulin large complex
def: "A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes. An example of this structure is found in Schizosaccharomyces pombe." [GOC:mtg_sensu, PMID:12134075]
GO:0000932
name: cytoplasmic mRNA processing body
def: "A focus in the cytoplasm where mRNAs may become inactivated by decapping or some other mechanism. mRNA processing and binding proteins are localized to these foci." [GOC:clt, PMID:12730603]
GO:0000933
name: adventitious septum
def: "A cell septum whose formation is independent of nuclear division." [GOC:clt, ISBN:0471940526]
GO:0000934
name: porous cell septum
def: "A septum or cross wall which does not entirely span the space between two portions of cell wall and may contain a specialized central pore structure. A porous septum allows the movement of organelles and/or cytoplasm between compartments." [GOC:clt]
GO:0000935
name: barrier septum
def: "A septum which spans a cell and does not allow exchange of organelles or cytoplasm between compartments." [GOC:clt]
GO:0000936
name: primary cell septum
def: "A cell septum that forms following nuclear division." [GOC:clt, ISBN:0471940526]
GO:0000937
name: dolipore septum
def: "A septum, or cross-wall, between two portions of a cell or hypha; contains a central pore around which the septum is swollen to form a barrel-shaped structure; pore is covered on each side of the septum by a septal pore cap (parenthosome)." [GOC:clt]
GO:0000938
name: GARP complex
def: "A quatrefoil tethering complex required for retrograde traffic from the early endosome back to the late Golgi and biogenesis of cytoplasmic vesicles." [GOC:clt, GOC:rn, PMID:10637310, PMID:12077354, PMID:12446664]
GO:0000939
name: condensed chromosome inner kinetochore
def: "The region of a condensed chromosome kinetochore closest to centromeric DNA; in mammals the CREST antigens (CENP proteins) are found in this layer; this layer may help define underlying centromeric chromatin structure and position of the kinetochore on the chromosome." [GOC:clt, PMID:10619130, PMID:11483983]
GO:0000940
name: condensed chromosome outer kinetochore
def: "The region of a condensed chromosome kinetochore most external to centromeric DNA; this outer region mediates kinetochore-microtubule interactions." [GOC:clt, PMID:11483983]
GO:0000941
name: condensed nuclear chromosome inner kinetochore
def: "The region of a condensed nuclear chromosome kinetochore closest to centromeric DNA; this layer may help define underlying centromeric chromatin structure and position of the kinetochore on the chromosome." [GOC:clt, PMID:10619130, PMID:1148398]
GO:0000942
name: condensed nuclear chromosome outer kinetochore
def: "The region of a condensed nuclear chromosome kinetochore most external to centromeric DNA; this outer region mediates kinetochore-microtubule interactions." [GOC:clt, PMID:1148398]
GO:0000943
name: retrotransposon nucleocapsid
def: "A complex of the retrotransposon RNA genome, reverse transcriptase, integrase, and associated molecules required for reproduction and integration of the retrotransposon into the host genome; the main structural molecule of the nucleocapsid is often a gag protein homolog." [GOC:clt, PMID:10861903]
GO:0000974
name: Prp19 complex
def: "A protein complex consisting of Prp19 and associated proteins that is involved in the transition from the precatalytic spliceosome to the activated form that catalyzes step 1 of splicing, and which remains associated with the spliceosome through the second catalytic step. It is widely conserved, found in both yeast and mammals, though the exact composition varies. In S. cerevisiae, it contains Prp19p, Ntc20p, Snt309p, Isy1p, Syf2p, Cwc2p, Prp46p, Clf1p, Cef1p, and Syf1p." [GOC:krc, PMID:16540691, PMID:19239890]
GO:0001114
name: protein-DNA-RNA complex
def: "A macromolecular complex containing protein, DNA, and RNA molecules." [GOC:txnOH]
GO:0001400
name: mating projection base
def: "The region where the mating projection meets the bulk of the cell, in unicellular fungi exposed to mating pheromone." [GOC:mcc]
GO:0001401
name: mitochondrial sorting and assembly machinery complex
def: "A large complex of the mitochondrial outer membrane that mediates sorting of some imported proteins to the outer membrane and their assembly in the membrane; functions after import of incoming proteins by the mitochondrial outer membrane translocase complex." [PMID:12891361]
GO:0001405
name: presequence translocase-associated import motor
def: "Protein complex located on the matrix side of the mitochondrial inner membrane and associated with the presequence translocase complex; hydrolyzes ATP to provide the force to drive import of proteins into the mitochondrial matrix." [GOC:mcc, PMID:14517234, PMID:14638855]
GO:0001411
name: hyphal tip
def: "The end, or tip, of a fungal hypha, where polarized growth occurs during hyphal elongation." [GOC:mcc]
GO:0001518
name: voltage-gated sodium channel complex
def: "A sodium channel in a cell membrane whose opening is governed by the membrane potential." [ISBN:0198506732]
GO:0001520
name: outer dense fiber
def: "Structure or material found in the flagella of mammalian sperm that surrounds each of the nine microtubule doublets, giving a 9 + 9 + 2 arrangement rather than the 9 + 2 pattern usually seen. These dense fibers are stiff and noncontractile." [ISBN:0824072820]
GO:0001527
name: microfibril
def: "Extracellular matrix components occurring independently or along with elastin. Thought to have force-bearing functions in tendon. In addition to fibrillins, microfibrils may contain other associated proteins." [http://www.xrefer.com/entry/490235]
GO:0001529
name: elastin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001533
name: cornified envelope
def: "A type of plasma membrane that has been modified through addition of distinct intracellular and extracellular components, including ceramide, found in cornifying epithelial cells (corneocytes)." [GOC:add, PMID:11112355, PMID:11590230, PMID:15803139]
GO:0001534
name: radial spoke
def: "Protein complex that links the outer microtubule doublet of the ciliary or flagellum axoneme with the sheath that surrounds the central pair of microtubules. Composed of a stalk that attaches to each doublet microtubule and a globular structure (spoke head) that projects toward the central pair of microtubules." [ISBN:0124325653, PMID:9450971]
GO:0001535
name: radial spokehead
def: "Protein complex forming part of eukaryotic flagellar apparatus." [GOC:hjd]
GO:0001536
name: radial spoke stalk
def: "Globular portion of the radial spoke that projects towards the central pair of microtubules." [GOC:hjd]
GO:0001650
name: fibrillar center
def: "A structure found most metazoan nucleoli, but not usually found in lower eukaryotes; surrounded by the dense fibrillar component; the zone of transcription from multiple copies of the pre-rRNA genes is in the border region between these two structures." [PMID:10754561]
GO:0001651
name: dense fibrillar component
def: "A structure found in the nucleolus, which contains newly synthesized preribosomal RNA (pre-rRNA) and a collection of proteins." [PMID:10754561]
GO:0001652
name: granular component
def: "A structure found in the nucleolus, which contains nearly completed preribosomal particles destined for the cytoplasm." [PMID:10754561]
GO:0001669
name: acrosomal vesicle
def: "A structure in the head of a spermatozoon that contains acid hydrolases, and is concerned with the breakdown of the outer membrane of the ovum during fertilization. It lies just beneath the plasma membrane and is derived from the lysosome." [ISBN:0124325653, ISBN:0198506732]
GO:0001673
name: male germ cell nucleus
def: "The nucleus of a male germ cell, a reproductive cell in males." [CL:0000015, GOC:hjd, GOC:mtg_sensu]
GO:0001674
name: female germ cell nucleus
def: "The nucleus of the female germ cell, a reproductive cell in females." [CL:0000021, GOC:hjd]
GO:0001683
name: axonemal dynein heavy chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001684
name: axonemal dynein intermediate chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001685
name: axonemal dynein intermediate light chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001686
name: axonemal dynein light chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001687
name: cytoplasmic dynein heavy chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001688
name: cytoplasmic dynein intermediate chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001689
name: cytoplasmic dynein intermediate light chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001690
name: cytoplasmic dynein light chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001721
name: intermediate filament associated protein
def: "OBSOLETE. Proteins that associate with intermediate filaments and function in the supramolecular organization of cellular intermediate filament networks." [GOC:ajp, PMID:9484600]
GO:0001722
name: type I intermediate filament associated protein
def: "OBSOLETE. Low molecular weight (10-45 kDa) proteins that associate with intermediate filaments by lateral binding of the filaments and have the effect of creating tight macrofilament aggregates." [GOC:ajp, PMID:9484600]
GO:0001723
name: type II intermediate filament associated protein
def: "OBSOLETE. High molecular weight (100-300 kDa) proteins that associate with intermediate filaments to cross-link them into loose networks." [GOC:ajp, PMID:9484600]
GO:0001724
name: type III intermediate filament associated protein
def: "OBSOLETE. Proteins that associate with the ends of intermediate filaments and couple the intermediate filaments to the plasma membrane." [GOC:ajp, PMID:9484600]
GO:0001725
name: stress fiber
def: "A contractile actin filament bundle that consists of short actin filaments with alternating polarity, cross-linked by alpha-actinin and possibly other actin bundling proteins, and with myosin present in a periodic distribution along the fiber." [PMID:16651381]
GO:0001726
name: ruffle
def: "Projection at the leading edge of a crawling cell; the protrusions are supported by a microfilament meshwork." [ISBN:0124325653]
GO:0001739
name: sex chromatin
def: "Chromatin that is part of a sex chromosome." [GOC:dos, ISBN:0198506732]
GO:0001740
name: Barr body
def: "A structure found in a female mammalian cell containing an unpaired X chromosome that has become densely heterochromatic, silenced and localized at the nuclear periphery." [GOC:hjd, GOC:mr, NIF_Subcellular:sao1571698684]
GO:0001741
name: XY body
def: "A structure found in a male mammalian spermatocyte containing an unpaired X chromosome that has become densely heterochromatic, silenced and localized at the nuclear periphery." [GOC:hjd]
GO:0001750
name: photoreceptor outer segment
def: "The outer segment of a vertebrate photoreceptor that contains discs of photoreceptive membranes." [ISBN:0824072820]
GO:0001772
name: immunological synapse
def: "An area of close contact between a lymphocyte (T-, B-, or natural killer cell) and a target cell formed through the clustering of particular signaling and adhesion molecules and their associated membrane rafts on both the lymphocyte and the target cell and facilitating activation of the lymphocyte, transfer of membrane from the target cell to the lymphocyte, and in some situations killing of the target cell through release of secretory granules and/or death-pathway ligand-receptor interaction." [GOC:mgi_curators, PMID:11244041, PMID:11376300]
GO:0001891
name: phagocytic cup
def: "An invagination of the cell membrane formed by an actin dependent process during phagocytosis. Following internalization it is converted into a phagosome." [PMID:10358769]
GO:0001917
name: photoreceptor inner segment
def: "The inner segment of a vertebrate photoreceptor containing mitochondria, ribosomes and membranes where opsin molecules are assembled and passed to be part of the outer segment discs." [GOC:add, PMID:12019563]
GO:0001931
name: uropod
def: "A membrane projection with related cytoskeletal components at the trailing edge of a cell in the process of migrating or being activated, found on the opposite side of the cell from the leading edge or immunological synapse, respectively." [GOC:add, ISBN:0781735149, PMID:12714569, PMID:12787750]
GO:0001939
name: female pronucleus
def: "The pronucleus originating from the ovum that is being fertilized." [GOC:hjd, ISBN:0198506732]
GO:0001940
name: male pronucleus
def: "The pronucleus originating from the spermatozoa that was involved in fertilization." [GOC:hjd, ISBN:0198506732]
GO:0001950
name: plasma membrane enriched fraction
def: "OBSOLETE: The fraction of cells, prepared by disruptive biochemical methods, that is enriched for plasma membranes." [GOC:mgi_curators, PMID:11562363, PMID:15601832]
GO:0002079
name: inner acrosomal membrane
def: "The acrosomal membrane region that underlies the acrosomal vesicle and is located toward the sperm nucleus. This region is responsible for molecular interactions allowing the sperm to penetrate the zona pellucida and fuses with the egg plasma membrane." [GOC:dph, PMID:3899643, PMID:8936405]
GO:0002080
name: acrosomal membrane
def: "The membrane that surrounds the acrosomal lumen. The acrosome is a special type of lysosome in the head of a spermatozoon that contains acid hydrolases and is concerned with the breakdown of the outer membrane of the ovum during fertilization." [GOC:dph]
GO:0002081
name: outer acrosomal membrane
def: "The acrosomal membrane region that underlies the plasma membrane of the sperm. This membrane fuses with the sperm plasma membrane as part of the acrosome reaction." [GOC:dph, PMID:8936405]
GO:0002095
name: caveolar macromolecular signaling complex
def: "A complex composed of proteins required for beta adrenergic receptor activation of protein kinase A. It includes the Cav 12. subunit of L-type calcium channel, protein kinase A regulatory subunit 2(PKAR2), adenyl cyclase, beta-adrenergic receptor, G-alpha-S, protein phosphatase 2A (PP2A) and caveolin 3 (CAV3)." [PMID:16648270]
GO:0002096
name: polkadots
def: "A punctate, filamentous structure composed of Bcl10 that appears in the cytoplasm of T-cells shortly after T-cell receptor stimulation. Polkadots stands for Punctate Oligomeric Killing and Activating DOmains Transducing Signals." [PMID:14724296, PMID:16495340]
GO:0002102
name: podosome
def: "An actin-rich adhesion structure characterized by formation upon cell substrate contact and localization at the substrate-attached part of the cell, contain an F-actin-rich core surrounded by a ring structure containing proteins such as vinculin and talin, and have a diameter of 0.5 mm." [PMID:12837608, PMID:15890982]
GO:0002111
name: BRCA2-BRAF35 complex
def: "A heterodimeric complex of BRCA2 and BRAF35 (BRCA2-associated factor 35). The BRCA2-BRAF35 complex is often associated with condensed chromatin during mitosis." [GOC:hjd, PMID:11207365]
GO:0002116
name: semaphorin receptor complex
def: "A stable binary complex of a neurophilin and a plexin, together forming a functional semaphorin receptor." [GOC:hjd, PMID:10934324, PMID:12367632, PMID:12613544]
GO:0002133
name: polycystin complex
def: "A stable heterodimeric complex composed of polycystin-1 and polycystin-2." [GOC:hjd, PMID:11901144]
GO:0002139
name: stereocilia coupling link
def: "A structure involved in coupling stereocilia to one another in sensory hair cells There are four morphologically distinct types: tip links, horizontal top connectors, shaft connectors and ankle links. Tip links and horizontal top connectors are the only inter-stereocilia links associated with mature cochlea, whereas ankle links appear during development of the auditory hair bundle." [PMID:16775142]
GO:0002140
name: stereocilia tip link
def: "A stereocilia link that is formed by a fine filament running more or less vertically upward from the tip of each shorter stereocilium to attach at a higher point on its adjacent taller neighbor. Tilting the bundle puts tension on the filaments, which pull on mechanically gated ion channels in the membrane of the stereocilia." [PMID:1108787]
GO:0002141
name: stereocilia ankle link
def: "A stereocilia coupling link that is composed of a fine filament present in developing stereocilia that couples the bases of individual stereocilia to one another. They are not present in mature stereocilia." [PMID:17567809]
GO:0002142
name: stereocilia ankle link complex
def: "A complex of proteins that connect growing stereocilia in developing cochlear hair cells, composed of Vlgr1, usherin, vezatin, and whirlin." [PMID:16775142]
GO:0002144
name: cytosolic tRNA wobble base thiouridylase complex
def: "A complex of two proteins involved in the thiolation of U34 in glutamate, lysine, and glutamine tRNAs of eukaryotes." [PMID:18391219]
GO:0002167
name: VRK3/VHR/ERK complex
def: "A ternary complex consisting of VRK3, VHR (Dusp3), and ERK1 (Mapk3) existing in neuronal cells, and is involved in regulation of the ERK signaling pathway." [GOC:hjd, PMID:16845380]
GO:0002169
name: 3-methylcrotonyl-CoA carboxylase complex, mitochondrial
def: "A heterodimeric complex having 3-methylcrotonyl-CoA carboxylase activity. The alpha subunit has a covalently bound biotin essential for the ATP-dependent carboxylation. The beta subunit possess carboxyltransferase activity which presumably is essential for binding to 3-methylcrotonyl-CoA." [GOC:hjd, PMID:15868465]
GO:0002177
name: manchette
def: "A conical shaped array of microtubules that completely covers the nucleus of a spermatid, thought to be involved in sperm head elongation." [PMID:15018141]
GO:0002178
name: palmitoyltransferase complex
def: "A protein complex with palmitoyltransferase activity." [GOC:hjd]
GO:0002179
name: homodimeric serine palmitoyltransferase complex
def: "A homodimeric complex which transfers a palmitoyl group onto serine, forming 3-dehydro-D-sphinganine." [GOC:hjd]
GO:0002180
name: 5-lipoxygenase complex
def: "An nuclear membrane protein complex having arachidonate 5-lipoxygenase activity." [PMID:19075240]
GO:0002185
name: creatine kinase complex
def: "A protein complex having creatine kinase activity." [GOC:hjd]
GO:0002186
name: cytosolic creatine kinase complex
def: "A dimeric protein complex having creatine kinase activity." [PMID:173175]
GO:0002187
name: mitochondrial creatine kinase complex
def: "An octomeric protein complex having creatine kinase activity." [PMID:16236486]
GO:0002189
name: ribose phosphate diphosphokinase complex
def: "A protein complex having ribose phosphate diphosphokinase activity." [GO:hjd, PMID:9348095]
GO:0002193
name: MAML1-RBP-Jkappa- ICN1 complex
def: "A protein complex that consists of the intracellular domain of Notch1 (ICN1), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-1 (MAML1); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [CORUM:2949, PMID:16510869]
GO:0002197
name: xanthine dehydrogenase complex
def: "A homodimeric protein complex having xanthine dehydrogenase activity." [GOC:hjd, PMID:8224915]
GO:0002199
name: zona pellucida receptor complex
def: "A multisubunit complex comprising the chaperonin-containing T-complex and several other components involved in mediating sperm-oocyte Interaction." [GOC:hjd, PMID:21880732]
GO:0002929
name: MECO complex
def: "A highly stable complex composed of the ATAC complex and the mediator complex (also called TRAP or MED). MECO binds and regulates the transcription of a subset of non-coding RNAs transcribed by RNA polymerase II." [GOC:hjd, PMID:20508642]
GO:0002944
name: cyclin K-CDK12 complex
def: "A protein complex consisting of cyclin Kand cyclin-dependent kinase 12 (CDK12). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [PMID:22012619]
GO:0002945
name: cyclin K-CDK13 complex
def: "A protein complex consisting of cyclin Kand cyclin-dependent kinase 13 (CDK13). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [PMID:22012619]
GO:0002947
name: tumor necrosis factor receptor superfamily complex
def: "A receptor complex that contains one or more members of the tumor necrosis factor (TNF) receptor superfamily." [GOC:krc]
GO:0003675
name: protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005559
name: ribozyme
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005561
name: nucleic acid
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005562
name: RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005564
name: cytosolic tRNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005565
name: mitochondrial tRNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005567
name: cytosolic ribosomal RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005568
name: mitochondrial rRNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005571
name: untranslated RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005572
name: RNA polymerase II transcribed untranslated RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005573
name: telomerase RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:krc]
GO:0005574
name: DNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005575
name: cellular_component
def: "The part of a cell or its extracellular environment in which a gene product is located. A gene product may be located in one or more parts of a cell and its location may be as specific as a particular macromolecular complex, that is, a stable, persistent association of macromolecules that function together." [GOC:go_curators, NIF_Subcellular:sao-1337158144]
GO:0005576
name: extracellular region
def: "The space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers the host cell environment outside an intracellular parasite." [GOC:go_curators]
GO:0005577
name: fibrinogen complex
def: "A highly soluble, elongated protein complex found in blood plasma and involved in clot formation. It is converted into fibrin monomer by the action of thrombin. In the mouse, fibrinogen is a hexamer, 46 nm long and 9 nm maximal diameter, containing two sets of nonidentical chains (alpha, beta, and gamma) linked together by disulfide bonds." [ISBN:0198547684]
GO:0005578
name: proteinaceous extracellular matrix
def: "A layer consisting mainly of proteins (especially collagen) and glycosaminoglycans (mostly as proteoglycans) that forms a sheet underlying or overlying cells such as endothelial and epithelial cells. The proteins are secreted by cells in the vicinity. An example of this component is found in Mus musculus." [GOC:mtg_sensu, ISBN:0198547684]
GO:0005579
name: membrane attack complex
def: "A protein complex produced by sequentially activated components of the complement cascade inserted into a target cell membrane and forming a pore leading to cell lysis via ion and water flow." [GOC:add, ISBN:0198547684, ISBN:068340007X, ISBN:0781735149]
GO:0005580
name: membrane attack complex protein alphaM chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005581
name: collagen trimer
def: "A protein complex consisting of three collagen chains assembled into a left-handed triple helix. These trimers typically assemble into higher order structures." [GOC:dos, GOC:mah, ISBN:0721639976, PMID:19693541, PMID:21421911]
GO:0005582
name: collagen type XV trimer
def: "A collagen homotrimer of alpha1(XV) chains; a chondroitin sulfate proteoglycan often found in specialized basement membranes where it bridges between fibrils." [PMID:11158616, PMID:11937714, PMID:21421911]
GO:0005583
name: fibrillar collagen trimer
def: "Any triple helical collagen trimer that forms fibrils." [GOC:mah, ISBN:0721639976, PMID:21421911]
GO:0005584
name: collagen type I trimer
def: "A collagen trimer containing alpha(I) chains. The most common form of type I collagen is a heterotrimer containing two alpha1(I) chains and one alpha2(I) chain; homotrimers containing three alpha1(I) chains are also found. Type I collagen triple helices associate to form banded fibrils." [GOC:mah, GOC:sl, ISBN:0721639976]
GO:0005585
name: collagen type II trimer
def: "A collagen homotrimer of alpha1(II) chains; type II collagen triple helices associate to form fibrils." [ISBN:0721639976]
GO:0005586
name: collagen type III trimer
def: "A collagen homotrimer of alpha1(III) chains; type III collagen triple helices associate to form fibrils." [ISBN:0721639976]
GO:0005587
name: collagen type IV trimer
def: "A collagen heterotrimer containing type IV alpha chains; [alpha1(IV)]2alpha2(IV) trimers are commonly observed, although more type IV alpha chains exist and may be present in type IV trimers; type IV collagen triple helices associate to form 3 dimensional nets within basement membranes." [ISBN:0721639976, PMID:19693541, PMID:21421911]
GO:0005588
name: collagen type V trimer
def: "A collagen heterotrimer containing type V alpha chains; [alpha1(V)]2alpha2(V) and alpha1(V)alpha2(V)alpha3(V) trimers have been observed; type V collagen triple helices associate to form fibrils." [ISBN:0721639976]
GO:0005589
name: collagen type VI trimer
def: "A collagen heterotrimer containing type VI alpha chains in alpha1(VI)alpha2(VI)alpha3(VI) trimers; type VI collagen triple helices associate to form beaded fibrils." [ISBN:0721639976, PMID:19693541, PMID:21421911]
GO:0005590
name: collagen type VII trimer
def: "A collagen homotrimer of alpha1(VII) chains; type VII collagen triple helices form antiparallel dimer, which in turn associate laterally to form anchoring fibrils that connect type IV collagen in the basal lamina to plaques in the underlying connective tissue. It binds laminin." [ISBN:0721639976] {comment="PMID:19693541"}
GO:0005591
name: collagen type VIII trimer
def: "A collagen heterotrimer containing type VIII alpha chains; [alpha1(VIII)2]alpha2(VIII) and alpha1(VIII)[alpha2(VIII)]2 trimers have been observed; type VIII collagen triple helices associate to form regular hexagonal nets." [ISBN:0721639976, PMID:21421911]
GO:0005592
name: collagen type XI trimer
def: "A collagen heterotrimer containing type XI alpha chains in alpha1(XI)alpha2(XI)alpha3(XI) trimers; type XI collagen triple helices associate to form fibrils." [ISBN:0721639976]
GO:0005593
name: FACIT collagen trimer
def: "A collagen trimer that associates with collagen fibrils and consists of collagen monomers that contain two or more relatively short triple-helical domains connected by non-triple-helical sequences." [ISBN:0198599587, PMID:21421911]
GO:0005594
name: collagen type IX trimer
def: "A collagen heterotrimer containing type IX alpha chains in alpha1(IX)alpha2(IX)alpha3(IX) trimers; type IX collagen triple helices associate to form a structure that links glycosaminoglycans to type II collagen fibrils." [ISBN:0721639976]
GO:0005595
name: collagen type XII trimer
def: "A collagen homotrimer of alpha1(XII) chains; type XII collagen triple helices may link sheet-forming or fibrillar collagens to other structures." [ISBN:0721639976]
GO:0005596
name: collagen type XIV trimer
def: "A collagen homotrimer of alpha1(XIV) chains; type XIV collagen triple helices may link sheet-forming or fibrillar collagens to other structures." [ISBN:0721639976]
GO:0005597
name: collagen type XVI trimer
def: "A collagen trimer containing alpha(XVI) chains; type XVI trimers can associate with microfibrils." [GOC:mah, PMID:12782140]
GO:0005598
name: short-chain collagen trimer
def: "Any collagen trimer that does not form fibrils and that is relatively short compared to the collagen trimers that do form fibrils." [ISBN:0198599587]
GO:0005599
name: collagen type X trimer
def: "A collagen homotrimer of alpha1(X) chains; type X collagen triple helices form hexagonal networks (sheets)." [ISBN:0721639976, PMID:21421911]
GO:0005600
name: collagen type XIII trimer
def: "A collagen homotrimer of alpha1(XIII) chains; type XIII collagen triple helices span the plasma membrane." [GOC:bm, GOC:dos, ISBN:0721639976]
GO:0005601
name: classical-complement-pathway C3/C5 convertase complex
def: "A heterodimeric protein complex that catalyzes the cleavage of complement components C3 and C5, and acts in the classical pathway of complement activation; consists of one monomer of C2a and one monomer of C4b; C2a is the catalytic subunit, but cannot catalyze cleavage alone." [BRENDA:3.4.21.43, GOC:mah, http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/C/Complement.html]
GO:0005602
name: complement component C1 complex
def: "A protein complex composed of six subunits of C1q, each formed of the three homologous polypeptide chains C1QA, C1QB, and C1QB, and tetramer of two C1QR and two C1QS polypeptide chains." [GOC:add, ISBN:0781735149]
GO:0005603
name: complement component C2 complex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005604
name: basement membrane
def: "A thin layer of dense material found in various animal tissues interposed between the cells and the adjacent connective tissue. It consists of the basal lamina plus an associated layer of reticulin fibers." [ISBN:0198547684]
GO:0005605
name: basal lamina
def: "A thin sheet of proteoglycans and glycoproteins, especially laminin, secreted by cells as an extracellular matrix." [ISBN:0198547684]
GO:0005606
name: laminin-1 complex
def: "A laminin complex composed of alpha1, beta1 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005607
name: laminin-2 complex
def: "A laminin complex composed of alpha2, beta1 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005608
name: laminin-3 complex
def: "A laminin complex composed of alpha1, beta2 and gamma1 polypeptide chains." [MEDLINE:95005761]
GO:0005609
name: laminin-4 complex
def: "A laminin complex composed of alpha2, beta2 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005610
name: laminin-5 complex
def: "A laminin complex composed of alpha3, beta3 and gamma2 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005611
name: laminin-6 complex
def: "A laminin complex composed of alpha3, beta1 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005612
name: laminin-7 complex
def: "A laminin complex composed of alpha3, beta2 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0005613
name: laminin receptor protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005614
name: interstitial matrix
def: "A type of extracellular matrix found in interstitial connective tissue, characterized by the presence of fibronectins, proteoglycans, and types I, III, V, VI, VII and XII collagens." [PMID:8450001]
GO:0005615
name: extracellular space
def: "That part of a multicellular organism outside the cells proper, usually taken to be outside the plasma membranes, and occupied by fluid." [ISBN:0198547684]
GO:0005616
name: larval serum protein complex
def: "A multisubunit protein complex which, in Drosophila, is a heterohexamer of three subunits, alpha, beta and gamma. The complex is thought to store amino acids for synthesis of adult proteins." [GOC:jl, PMID:6781759]
GO:0005617
name: larval serum protein-1
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005618
name: cell wall
def: "The rigid or semi-rigid envelope lying outside the cell membrane of plant, fungal, most prokaryotic cells and some protozoan parasites, maintaining their shape and protecting them from osmotic lysis. In plants it is made of cellulose and, often, lignin; in fungi it is composed largely of polysaccharides; in bacteria it is composed of peptidoglycan; in protozoan parasites such as Giardia species, it's made of carbohydrates and proteins." [GOC:giardia, http://en.wikipedia.org/wiki/Microbial_cyst, ISBN:0198547684, PMID:15134259]
GO:0005619
name: ascospore wall
def: "The specialized cell wall of the ascospore (spore), which is the product of meiotic division. Examples of this component are found in Fungi." [GOC:vw, ISBN:0879693568]
GO:0005621
name: cellular bud scar
def: "Crater-like ring of chitinous scar tissue located on the surface of the mother cell. It is formed after the newly emerged daughter cell separates thereby marking the site of cytokinesis and septation. The number of bud scars that accumulate on the surface of a cell is a useful determinant of replicative age." [GOC:rn, PMID:14600225, PMID:2005820]
GO:0005622
name: intracellular
def: "The living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm." [ISBN:0198506732]
GO:0005623
name: cell
def: "The basic structural and functional unit of all organisms. Includes the plasma membrane and any external encapsulating structures such as the cell wall and cell envelope." [GOC:go_curators]
GO:0005624
name: membrane fraction
def: "OBSOLETE: That fraction of cells, prepared by disruptive biochemical methods, that includes the plasma and other membranes." [GOC:ma]
GO:0005625
name: soluble fraction
def: "OBSOLETE: That fraction of cells, prepared by disruptive biochemical methods, that is soluble in water." [GOC:ma]
GO:0005626
name: insoluble fraction
def: "OBSOLETE: That fraction of cells, prepared by disruptive biochemical methods, that is not soluble in water." [GOC:ma]
GO:0005627
name: ascus
def: "OBSOLETE. A sac-like fruiting body (ascomycete Fungi); contains ascospores (typically eight in number)." [ISBN:0198547684]
GO:0005628
name: prospore membrane
def: "The prospore membrane is a double-membraned structure that extends from the cytoplasmic face of the spindle pole bodies to encompass the spindle pole bodies and the four nuclear lobes that are formed during meiosis. It helps isolate the meiotic nuclei from the cytoplasm during spore formation and serves as a foundation for the formation of the spore walls. An example of this component is found in Schizosaccharomyces pombe." [ISBN:0879693649]
GO:0005630
name: dityrosine layer of spore wall
def: "The outermost layer of the spore wall, as described in Saccharomyces." [ISBN:0879693568]
GO:0005631
name: chitosan layer of spore wall
def: "The second outermost layer of the spore wall, as described in Saccharomyces." [ISBN:0879693568]
GO:0005632
name: inner layer of spore wall
def: "Either of the two innermost layers of the spore wall, as described in Saccharomyces." [ISBN:0879693568]
GO:0005633
name: ascus lipid particle
def: "Any particle of coalesced lipids in an ascus or ascospore. May include associated proteins." [GOC:mah, PMID:12702293]
GO:0005634
name: nucleus
def: "A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent." [GOC:go_curators]
GO:0005635
name: nuclear envelope
def: "The double lipid bilayer enclosing the nucleus and separating its contents from the rest of the cytoplasm; includes the intermembrane space, a gap of width 20-40 nm (also called the perinuclear space)." [ISBN:0198547684]
GO:0005637
name: nuclear inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the nuclear envelope." [GOC:ma]
GO:0005638
name: lamin filament
def: "Any of a group of intermediate-filament proteins that form the fibrous matrix on the inner surface of the nuclear envelope. They are classified as lamins A, B and C." [ISBN:0198547684]
GO:0005639
name: integral component of nuclear inner membrane
def: "The component of the nuclear inner membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:go_curators]
GO:0005640
name: nuclear outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the nuclear envelope; continuous with the endoplasmic reticulum of the cell and sometimes studded with ribosomes." [ISBN:0198547684]
GO:0005641
name: nuclear envelope lumen
def: "The region between the two lipid bilayers of the nuclear envelope; 20-40 nm wide." [GOC:ai]
GO:0005642
name: annulate lamellae
def: "Stacks of endoplasmic reticulum (ER) membranes containing a high density of nuclear pores, thought to form from excess nuclear membrane components, that have been described in a number of different cells. Annulate lamellar membranes are continuous with and embedded within the ER." [PMID:12631728]
GO:0005643
name: nuclear pore
def: "Any of the numerous similar discrete openings in the nuclear envelope of a eukaryotic cell, where the inner and outer nuclear membranes are joined." [ISBN:0198547684]
GO:0005645
name: RAN-binding protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005646
name: importin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005647
name: importin, alpha-subunit
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005648
name: importin, beta-subunit
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005649
name: transportin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005650
name: importin, alpha-subunit transport factor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005651
name: exportin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005652
name: nuclear lamina
def: "The fibrous, electron-dense layer lying on the nucleoplasmic side of the inner membrane of a cell nucleus, composed of lamin filaments. The polypeptides of the lamina are thought to be concerned in the dissolution of the nuclear envelope and its re-formation during mitosis. The lamina is composed of lamin A and lamin C filaments cross-linked into an orthogonal lattice, which is attached via lamin B to the inner nuclear membrane through interactions with a lamin B receptor, an IFAP, in the membrane." [ISBN:0198506732, ISBN:0716731363]
GO:0005654
name: nucleoplasm
def: "That part of the nuclear content other than the chromosomes or the nucleolus." [GOC:ma, ISBN:0124325653]
GO:0005655
name: nucleolar ribonuclease P complex
def: "A ribonuclease P complex located in the nucleolus of a eukaryotic cell, where it catalyzes the 5' endonucleolytic cleavage of precursor tRNAs to yield mature tRNAs. Eukaryotic nucleolar ribonuclease P complexes generally contain a single RNA molecule that is necessary but not sufficient for catalysis, and several protein molecules." [GOC:mah, PMID:12045094]
GO:0005656
name: nuclear pre-replicative complex
def: "A protein-DNA complex assembled at eukaryotic DNA replication origins during late mitosis and G1, allowing the origin to become competent, or 'licensed', for replication. The complex normally includes the origin recognition complex (ORC), Cdc6, Cdt1 and the MiniChromosome Maintenance (Mcm2-7) proteins." [PMID:15222894]
GO:0005657
name: replication fork
def: "The Y-shaped region of a replicating DNA molecule, resulting from the separation of the DNA strands and in which the synthesis of new strands takes place. Also includes associated protein complexes." [GOC:mah, ISBN:0198547684]
GO:0005658
name: alpha DNA polymerase:primase complex
def: "A complex of four polypeptides, comprising large and small DNA polymerase alpha subunits and two primase subunits, which catalyzes the synthesis of an RNA primer on the lagging strand of replicating DNA; the smaller of the two primase subunits alone can catalyze oligoribonucleotide synthesis." [GOC:mah, PMID:11395402]
GO:0005660
name: delta-DNA polymerase cofactor complex
def: "OBSOLETE. A complex of proteins that interacts with delta-DNA polymerase, promoting elongation. In humans it is a heteropentamer of subunits of 140/145, 40, 38, 37 and 36.5 kDa, which form a complex with the proliferating cell nuclear antigen (PCNA) in the presence of ATP." [GOC:jl]
GO:0005662
name: DNA replication factor A complex
def: "A conserved heterotrimeric complex that binds nonspecifically to single-stranded DNA and is required for multiple processes in eukaryotic DNA metabolism, including DNA replication, DNA repair, and recombination. In all eukaryotic organisms examined the complex is composed of subunits of approximately 70, 30, and 14 kDa." [PMID:9242902]
GO:0005663
name: DNA replication factor C complex
def: "A complex of five polypeptides in eukaryotes, and two in prokaryotes, that loads the DNA polymerase processivity factor proliferating cell nuclear antigen (PCNA) onto DNA, thereby permitting processive DNA synthesis catalyzed by DNA polymerase." [PMID:14614842, PMID:14646196, PMID:16172520]
GO:0005664
name: nuclear origin of replication recognition complex
def: "A multisubunit complex that is located at the replication origins of a chromosome in the nucleus." [GOC:elh]
GO:0005665
name: DNA-directed RNA polymerase II, core complex
def: "RNA polymerase II, one of three nuclear DNA-directed RNA polymerases found in all eukaryotes, is a multisubunit complex; typically it produces mRNAs, snoRNAs, and some of the snRNAs. Two large subunits comprise the most conserved portion including the catalytic site and share similarity with other eukaryotic and bacterial multisubunit RNA polymerases. The largest subunit of RNA polymerase II contains an essential carboxyl-terminal domain (CTD) composed of a variable number of heptapeptide repeats (YSPTSPS). The remainder of the complex is composed of smaller subunits (generally ten or more), some of which are also found in RNA polymerases I and III. Although the core is competent to mediate ribonucleic acid synthesis, it requires additional factors to select the appropriate template." [GOC:krc, GOC:mtg_sensu]
GO:0005666
name: DNA-directed RNA polymerase III complex
def: "RNA polymerase III, one of three nuclear DNA-directed RNA polymerases found in all eukaryotes, is a multisubunit complex; typically it produces 5S rRNA, tRNAs and some of the small nuclear RNAs. Two large subunits comprise the most conserved portion including the catalytic site and share similarity with other eukaryotic and bacterial multisubunit RNA polymerases. The remainder of the complex is composed of smaller subunits (generally ten or more), some of which are also found in RNA polymerase I and others of which are also found in RNA polymerases I and II. Although the core is competent to mediate ribonucleic acid synthesis, it requires additional factors to select the appropriate template." [GOC:krc, GOC:mtg_sensu]
GO:0005667
name: transcription factor complex
def: "A protein complex that is capable of associating with DNA by direct binding, or via other DNA-binding proteins or complexes, and regulating transcription." [GOC:jl]
GO:0005668
name: RNA polymerase transcription factor SL1 complex
def: "A RNA polymerase I-specific transcription factor complex that contains the TATA-box-binding protein (TBP) and at least three TBP-associated factors including proteins known in mammals as TAFI110, TAFI63 and TAFI48." [PMID:15691654]
GO:0005669
name: transcription factor TFIID complex
def: "A complex composed of TATA binding protein (TBP) and TBP associated factors (TAFs); the total mass is typically about 800 kDa. Most of the TAFs are conserved across species. In TATA-containing promoters for RNA polymerase II (Pol II), TFIID is believed to recognize at least two distinct elements, the TATA element and a downstream promoter element. TFIID is also involved in recognition of TATA-less Pol II promoters. Binding of TFIID to DNA is necessary but not sufficient for transcription initiation from most RNA polymerase II promoters." [GOC:krc, GOC:mah, ISBN:0471953393, ISBN:0879695501]
GO:0005670
name: transcription-activating factor, 30kD
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005671
name: Ada2/Gcn5/Ada3 transcription activator complex
def: "A multiprotein complex that possesses histone acetyltransferase and is involved in regulation of transcription. Contains either GCN5 or PCAF in a mutually exclusive manner. The budding yeast complex includes Gcn5p, two proteins of the Ada family, and two TBP-associate proteins (TAFs); analogous complexes in other species have analogous compositions, and usually contain homologs of the yeast proteins. Both ATAC- or SAGA (see GO:0000124, SAGA complex) are involved in the acetylation of histone H3K9 and K14 residues." [PMID:10637607]
GO:0005672
name: transcription factor TFIIA complex
def: "A component of the transcription machinery of RNA Polymerase II. In humans, TFIIA is a heterotrimer composed of an alpha (P35), beta (P19) and gamma subunits (P12)." [GOC:jl, PMID:17560669]
GO:0005673
name: transcription factor TFIIE complex
def: "A transcription factor which in humans consists of a complex of two alpha and two beta chains. Recruits TFIIH to the initiation complex and helps activate both RNA polymerase II and TFIIH." [GOC:jl, PMID:16547462]
GO:0005674
name: transcription factor TFIIF complex
def: "A general transcription initiation factor which in humans consists of a heterodimer of an alpha and a beta subunit. Helps recruit RNA polymerase II to the initiation complex and promotes translation elongation." [GOC:jl, PMID:7597077]
GO:0005675
name: holo TFIIH complex
def: "A complex that is capable of kinase activity directed towards the C-terminal Domain (CTD) of the largest subunit of RNA polymerase II and is essential for initiation at RNA polymerase II promoters in vitro. It is composed of  the core TFIIH complex and the TFIIK complex." [GOC:ew, GOC:krc, PMID:14500720, PMID:22308316, PMID:22572993, PMID:7813015]
GO:0005677
name: chromatin silencing complex
def: "Any protein complex that mediates changes in chromatin structure that result in transcriptional silencing." [GOC:mah]
GO:0005678
name: chromatin assembly complex
def: "OBSOLETE: Any protein complex that acts in the formation of nucleosomes or higher order chromatin structure." [GOC:mah]
GO:0005680
name: anaphase-promoting complex
def: "A ubiquitin ligase complex that degrades mitotic cyclins and anaphase inhibitory protein, thereby triggering sister chromatid separation and exit from mitosis. Substrate recognition by APC occurs through degradation signals, the most common of which is termed the Dbox degradation motif, originally discovered in cyclin B." [GOC:jh, GOC:vw, PMID:10465783, PMID:10611969]
GO:0005681
name: spliceosomal complex
def: "Any of a series of ribonucleoprotein complexes that contain RNA and small nuclear ribonucleoproteins (snRNPs), and are formed sequentially during the splicing of a messenger RNA primary transcript to excise an intron." [GOC:mah, ISBN:0198547684, PMID:19239890]
GO:0005682
name: U5 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U5." [GOC:krc, GOC:mah]
GO:0005683
name: U7 snRNP
def: "A ribonucleoprotein complex that contains the U7 snRNA and is required for the 3'-end processing of replication-dependent histone pre-mRNAs." [PMID:12872004]
GO:0005684
name: U2-type spliceosomal complex
def: "Any spliceosomal complex that forms during the splicing of a messenger RNA primary transcript to excise an intron that has canonical consensus sequences near the 5' and 3' ends." [GOC:krc, GOC:mah, PMID:11343900]
GO:0005685
name: U1 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U1." [GOC:krc, GOC:mah]
GO:0005686
name: U2 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U2." [GOC:krc, GOC:mah]
GO:0005687
name: U4 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U4." [GOC:krc, GOC:mah]
GO:0005688
name: U6 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U6." [GOC:krc, GOC:mah]
GO:0005689
name: U12-type spliceosomal complex
def: "Any spliceosomal complex that forms during the splicing of a messenger RNA primary transcript to excise an intron; the series of U12-type spliceosomal complexes is involved in the splicing of the majority of introns that contain atypical AT-AC terminal dinucleotides, as well as other non-canonical introns. The entire splice site signal, not just the terminal dinucleotides, is involved in determining which spliceosome utilizes the site." [GOC:krc, GOC:mah, PMID:11574683, PMID:11971955]
GO:0005690
name: U4atac snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U4atac." [GOC:krc, GOC:mah]
GO:0005691
name: U6atac snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U6atac." [GOC:krc, GOC:mah]
GO:0005692
name: U11 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U11." [GOC:krc, GOC:mah]
GO:0005693
name: U12 snRNP
def: "A ribonucleoprotein complex that contains small nuclear RNA U12." [GOC:krc, GOC:mah]
GO:0005694
name: chromosome
def: "A structure composed of a very long molecule of DNA and associated proteins (e.g. histones) that carries hereditary information." [ISBN:0198547684]
GO:0005695
name: chromatid
def: "OBSOLETE. One of the two daughter strands of a duplicated chromosome that become apparent between early prophase and metaphase in mitosis and between diplotene and second metaphase in meiosis." [ISBN:0198506732]
GO:0005696
name: telomere
def: "OBSOLETE. A complex of DNA and protein that seals the end of a chromosome. The telomeric DNA consists of simple tandemly repeated sequences specific for each species. Typically one strand is G-rich and the other C-rich. The G-rich strand forms a 3'-terminal overhang, the length of which varies with species. The single strand overhang is bound by a variety of proteins, including telomere capping proteins that bind to the single-stranded DNA." [ISBN:0198506732, PMID:11352055]
GO:0005697
name: telomerase holoenzyme complex
def: "Telomerase is a ribonucleoprotein enzyme complex, with a minimal catalytic core composed of a catalytic reverse transcriptase subunit and an RNA subunit that provides the template for telomeric DNA addition. In vivo, the holoenzyme complex often contains additional subunits." [PMID:11884619]
GO:0005698
name: centromere
def: "OBSOLETE. The region of a eukaryotic chromosome that is attached to the spindle during nuclear division. It is defined genetically as the region of the chromosome that always segregates at the first division of meiosis; the region of the chromosome in which no crossing over occurs. At the start of M phase, each chromosome consists of two sister chromatids with a constriction at a point which forms the centromere. During late prophase two kinetochores assemble on each centromere, one kinetochore on each sister chromatid." [ISBN:0198506732]
GO:0005700
name: polytene chromosome
def: "A type of chromosome in a polyploid cell, formed when multiple copies of homologous chromosomes are aligned side by side to give a giant chromosome in which distinct chromosome bands are readily visible." [ISBN:0198506732]
GO:0005701
name: polytene chromosome chromocenter
def: "A region at which the centric regions of polytene chromosomes are joined together." [GOC:bf, ISBN:0120649012]
GO:0005702
name: polytene chromosome weak point
def: "A region of the polytene chromosome where the diameter is considerably decreased, probably resulting from local differences in chromosome organization." [GOC:bf, ISBN:0120649012]
GO:0005703
name: polytene chromosome puff
def: "A swelling at a site along the length of a polytene chromosome, thought to be the site of active transcription." [GOC:bf, ISBN:0120649012]
GO:0005704
name: polytene chromosome band
def: "A stretch of densely packed chromatin along the polytene chromosome, visible as a morphologically distinct band." [GOC:bf, PMID:11361342]
GO:0005705
name: polytene chromosome interband
def: "A stretch of less tightly packed chromatin along the polytene chromosome, found between bands." [GOC:bf, PMID:11361342]
GO:0005706
name: polytene chromosome ectopic fiber
def: "A thread-like connection joining two regions of ectopically paired polytene chromosomes." [GOC:bf, ISBN:0120649012]
GO:0005707
name: interphase chromosome
def: "OBSOLETE. A chromosome found in the cell during interphase of the cell cycle. Chromosomes are usually decondensed during interphase and each long DNA molecule in a chromosome is divided into a large number of discrete domains that are folded differently." [GOC:ai, ISBN:0815316194]
GO:0005708
name: mitotic chromosome
def: "OBSOLETE. A chromosome involved in the process of mitosis." [GOC:ai]
GO:0005709
name: prophase chromosome
def: "OBSOLETE. A chromosome found in the cell during prophase." [GOC:ai]
GO:0005710
name: metaphase chromosome
def: "OBSOLETE. A chromosome found in the cell during metaphase. Typically, sister chromatids are held together at their centromeres and chromosomes are covered with a large number of molecules, including ribonucleoproteins." [GOC:ai]
GO:0005711
name: meiotic chromosome
def: "OBSOLETE. A chromosome involved in the process of meiosis." [GOC:ai]
GO:0005712
name: chiasma
def: "A connection formed between chromatids, visible during meiosis, thought to be the point of the interchange involved in crossing-over." [ISBN:0198506732]
GO:0005713
name: recombination nodule
def: "An electron dense structure that is associated with meiotic chromosomes." [GOC:elh]
GO:0005714
name: early recombination nodule
def: "An electron dense structure that is associated with meiotic chromosomes in leptotene or zygotene during meiosis I." [GOC:elh]
GO:0005715
name: late recombination nodule
def: "An electron dense structure that is associated with meiotic chromosomes in pachytene during meiosis I." [GOC:elh]
GO:0005719
name: nuclear euchromatin
def: "The dispersed less dense form of chromatin in the interphase nucleus. It exists in at least two forms, a some being in the form of transcriptionally active chromatin which is the least condensed, while the rest is inactive euchromatin which is more condensed than active chromatin but less condensed than heterochromatin." [ISBN:0198506732]
GO:0005720
name: nuclear heterochromatin
def: "A condensed form of chromatin, occurring in the nucleus during interphase, that stains strongly with basophilic dyes. The DNA of heterochromatin is typically replicated at a later stage in the cell-division cycle than euchromatin." [ISBN:0198506732]
GO:0005721
name: pericentric heterochromatin
def: "Heterochromatin that is located adjacent to the CENP-A rich centromere 'central core' and characterized by the modified histone H3K9me3." [PMID:12019236, PMID:20206496, PMID:22729156]
GO:0005722
name: beta-heterochromatin
def: "A diffusely banded region of heterochromatin located between euchromatin and alpha-heterochromatin in the polytene chromosome chromocenter; normally replicated during polytenization." [PMID:11404334, PMID:8878678]
GO:0005723
name: alpha-heterochromatin
def: "A small, compact region of heterochromatin located in the middle of the polytene chromosome chromocenter, which undergoes little or no replication during polytenization." [PMID:8878678]
GO:0005724
name: nuclear telomeric heterochromatin
def: "Heterochromatic regions of the chromosome found at the telomeres of a chromosome in the nucleus." [GOC:ai]
GO:0005725
name: intercalary heterochromatin
def: "Any of the regions of heterochromatin that form a reproducible set of dense bands scattered along the euchromatic arms in polytene chromosomes." [PMID:14579245]
GO:0005726
name: perichromatin fibrils
def: "Structures of variable diameter visible in the nucleoplasm by electron microscopy, mainly observed near the border of condensed chromatin. The fibrils are enriched in RNA, and are believed to be sites of pre-mRNA splicing and polyadenylylation representing the in situ form of nascent transcripts." [PMID:14731598]
GO:0005727
name: extrachromosomal circular DNA
def: "Circular DNA structures that are not part of a chromosome." [GOC:ai]
GO:0005728
name: extrachromosomal rDNA circle
def: "Circular DNA molecules encoding ribosomal RNA that are replicated independently of chromosomal replication. These molecules originate in the chromosome but are excised and circularized, often by intramolecular homologous recombination between direct tandem repeats." [GOC:mah, PMID:12044934]
GO:0005729
name: 2-micrometer circle DNA
def: "A plasmid commonly found in Saccharomyces, inherited in a non-Mendelian manner and often present in 100-400 copies." [PMID:12073320]
GO:0005730
name: nucleolus
def: "A small, dense body one or more of which are present in the nucleus of eukaryotic cells. It is rich in RNA and protein, is not bounded by a limiting membrane, and is not seen during mitosis. Its prime function is the transcription of the nucleolar DNA into 45S ribosomal-precursor RNA, the processing of this RNA into 5.8S, 18S, and 28S components of ribosomal RNA, and the association of these components with 5S RNA and proteins synthesized outside the nucleolus. This association results in the formation of ribonucleoprotein precursors; these pass into the cytoplasm and mature into the 40S and 60S subunits of the ribosome." [ISBN:0198506732]
GO:0005731
name: nucleolus organizer region
def: "A region of a chromosome where nucleoli form during interphase, and where genes encoding the largest rRNA precursor transcript are tandemly arrayed." [PMID:14504406]
GO:0005732
name: small nucleolar ribonucleoprotein complex
def: "A ribonucleoprotein complex that contains an RNA molecule of the small nucleolar RNA (snoRNA) family and associated proteins. Most are involved in a step of processing of rRNA: cleavage, 2'-O-methylation, or pseudouridylation. The majority, though not all, fall into one of two classes, box C/D type or box H/ACA type." [GOC:krc, GOC:mah, ISBN:0879695897]
GO:0005733
name: small nucleolar RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005734
name: box C + D snoRNP protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005735
name: box H + ACA snoRNP protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005736
name: DNA-directed RNA polymerase I complex
def: "RNA polymerase I, one of three nuclear DNA-directed RNA polymerases found in all eukaryotes, is a multisubunit complex; typically it produces rRNAs. Two large subunits comprise the most conserved portion including the catalytic site and share similarity with other eukaryotic and bacterial multisubunit RNA polymerases. The remainder of the complex is composed of smaller subunits (generally ten or more), some of which are also found in RNA polymerase III and others of which are also found in RNA polymerases II and III. Although the core is competent to mediate ribonucleic acid synthesis, it requires additional factors to select the appropriate template." [GOC:krc, GOC:mtg_sensu]
GO:0005737
name: cytoplasm
def: "All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures." [ISBN:0198547684]
GO:0005739
name: mitochondrion
def: "A semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration." [GOC:giardia, ISBN:0198506732]
GO:0005740
name: mitochondrial envelope
def: "The double lipid bilayer enclosing the mitochondrion and separating its contents from the cell cytoplasm; includes the intermembrane space." [GOC:ai, GOC:pz]
GO:0005741
name: mitochondrial outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the mitochondrial envelope." [GOC:ai]
GO:0005742
name: mitochondrial outer membrane translocase complex
def: "A large complex of the mitochondrial outer membrane that mediates transport of proteins into all mitochondrial compartments." [PMID:12581629]
GO:0005743
name: mitochondrial inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the mitochondrial envelope. It is highly folded to form cristae." [GOC:ai]
GO:0005744
name: mitochondrial inner membrane presequence translocase complex
def: "The protein transport machinery of the mitochondrial inner membrane that contains three essential Tim proteins: Tim17 and Tim23 are thought to build a preprotein translocation channel while Tim44 interacts transiently with the matrix heat-shock protein Hsp70 to form an ATP-driven import motor." [EC:3.6.3.51, PMID:8851659]
GO:0005745
name: m-AAA complex
def: "Protease complex of the mitochondrial inner membrane that is involved in mitochondrial protein turnover and in processing of proteins imported into mitochondria." [GOC:mcc, PMID:12417197, PMID:21147776]
GO:0005746
name: mitochondrial respiratory chain
def: "The protein complexes that form the mitochondrial electron transport system (the respiratory chain), associated with the inner mitochondrial membrane. The respiratory chain complexes transfer electrons from an electron donor to an electron acceptor and are associated with a proton pump to create a transmembrane electrochemical gradient." [GOC:curators, GOC:ecd, ISBN:0198547684]
GO:0005747
name: mitochondrial respiratory chain complex I
def: "A protein complex located in the mitochondrial inner membrane that forms part of the mitochondrial respiratory chain. It contains about 25 different polypeptide subunits, including NADH dehydrogenase (ubiquinone), flavin mononucleotide and several different iron-sulfur clusters containing non-heme iron. The iron undergoes oxidation-reduction between Fe(II) and Fe(III), and catalyzes proton translocation linked to the oxidation of NADH by ubiquinone." [GOC:mtg_sensu, ISBN:0198547684]
GO:0005749
name: mitochondrial respiratory chain complex II
def: "A protein complex located in the mitochondrial inner membrane that forms part of the mitochondrial respiratory chain. Contains the four polypeptide subunits of succinate dehydrogenase, flavin-adenine dinucleotide and iron-sulfur. Catalyzes the oxidation of succinate by ubiquinone. Connects the TCA cycle with the respiratory chain." [GOC:mtg_sensu, ISBN:0198547684]
GO:0005750
name: mitochondrial respiratory chain complex III
def: "A protein complex located in the mitochondrial inner membrane that forms part of the mitochondrial respiratory chain. Contains about 10 polypeptide subunits including four redox centers: cytochrome b/b6, cytochrome c1 and an 2Fe-2S cluster. Catalyzes the oxidation of ubiquinol by oxidized cytochrome c1." [GOC:mtg_sensu, ISBN:0198547684]
GO:0005751
name: mitochondrial respiratory chain complex IV
def: "A protein complex located in the mitochondrial inner membrane that forms part of the mitochondrial respiratory chain. Contains the 13 polypeptide subunits of cytochrome c oxidase, including cytochrome a and cytochrome a3. Catalyzes the oxidation of reduced cytochrome c by dioxygen (O2)." [GOC:mtg_sensu, ISBN:0198547684]
GO:0005753
name: mitochondrial proton-transporting ATP synthase complex
def: "A proton-transporting ATP synthase complex found in the mitochondrial membrane." [GOC:mah, GOC:mtg_sensu]
GO:0005754
name: mitochondrial proton-transporting ATP synthase, catalytic core
def: "The hexamer, comprising three alpha and three beta subunits, that possesses the catalytic activity of the mitochondrial hydrogen-transporting ATP synthase." [GOC:mtg_sensu, PMID:10838056]
GO:0005755
name: hydrogen-transporting ATP synthase, coupling factor CF(0)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005756
name: mitochondrial proton-transporting ATP synthase, central stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the mitochondrial membrane-associated F0 proteins; rotates within the catalytic core during catalysis." [GOC:mtg_sensu, PMID:10838056]
GO:0005757
name: mitochondrial permeability transition pore complex
def: "A protein complex that connects the inner and outer membranes of animal mitochondria and acts as a pore that can open transiently to allow free diffusion of solutes between the mitochondrial matrix and the cytosol. The pore complex is formed of the voltage-dependent anion channel (VDAC), the adenine nucleotide translocase (ANT) and cyclophilin-D (CyP-D)." [PMID:10393078]
GO:0005758
name: mitochondrial intermembrane space
def: "The region between the inner and outer lipid bilayers of the mitochondrial envelope." [GOC:mah]
GO:0005759
name: mitochondrial matrix
def: "The gel-like material, with considerable fine structure, that lies in the matrix space, or lumen, of a mitochondrion. It contains the enzymes of the tricarboxylic acid cycle and, in some organisms, the enzymes concerned with fatty acid oxidation." [GOC:as, ISBN:0198506732]
GO:0005760
name: gamma DNA polymerase complex
def: "A DNA polymerase complex consisting of a large subunit, responsible for the catalytic activities, and a small accessory subunit. Functions in the replication and repair of mitochondrial DNA." [GOC:jl, PMID:12045093]
GO:0005761
name: mitochondrial ribosome
def: "A ribosome found in the mitochondrion of a eukaryotic cell; contains a characteristic set of proteins distinct from those of cytosolic ribosomes." [GOC:mah, ISBN:0198506732]
GO:0005762
name: mitochondrial large ribosomal subunit
def: "The larger of the two subunits of a mitochondrial ribosome. Two sites on the ribosomal large subunit are involved in translation: the aminoacyl site (A site) and peptidyl site (P site)." [GOC:mcc]
GO:0005763
name: mitochondrial small ribosomal subunit
def: "The smaller of the two subunits of a mitochondrial ribosome." [GOC:mcc]
GO:0005764
name: lysosome
def: "A small lytic vacuole that has cell cycle-independent morphology and is found in most animal cells and that contains a variety of hydrolases, most of which have their maximal activities in the pH range 5-6. The contained enzymes display latency if properly isolated. About 40 different lysosomal hydrolases are known and lysosomes have a great variety of morphologies and functions." [GOC:mah, ISBN:0198506732]
GO:0005765
name: lysosomal membrane
def: "The lipid bilayer surrounding the lysosome and separating its contents from the cell cytoplasm." [GOC:ai]
GO:0005766
name: primary lysosome
def: "A lysosome before it has fused with a vesicle or vacuole." [GOC:jl, ISBN:0815316194]
GO:0005767
name: secondary lysosome
def: "Vacuole formed by the fusion of a lysosome with an organelle (autosome) or with a primary phagosome." [GOC:jl, ISBN:0815316194]
GO:0005768
name: endosome
def: "A membrane-bounded organelle to which materials ingested by endocytosis are delivered." [ISBN:0198506732, PMID:19696797]
GO:0005769
name: early endosome
def: "A membrane-bounded organelle that receives incoming material from primary endocytic vesicles that have been generated by clathrin-dependent and clathrin-independent endocytosis; vesicles fuse with the early endosome to deliver cargo for sorting into recycling or degradation pathways." [GOC:mah, NIF_Subcellular:nlx_subcell_20090701, PMID:19696797]
GO:0005770
name: late endosome
def: "A prelysosomal endocytic organelle differentiated from early endosomes by lower lumenal pH and different protein composition. Late endosomes are more spherical than early endosomes and are mostly juxtanuclear, being concentrated near the microtubule organizing center." [NIF_Subcellular:nlx_subcell_20090702, PMID:11964142, PMID:2557062]
GO:0005771
name: multivesicular body
def: "A type of late endosome in which regions of the limiting endosomal membrane invaginate to form internal vesicles; membrane proteins that enter the internal vesicles are sequestered from the cytoplasm." [PMID:11566881]
GO:0005773
name: vacuole
def: "A closed structure, found only in eukaryotic cells, that is completely surrounded by unit membrane and contains liquid material. Cells contain one or several vacuoles, that may have different functions from each other. Vacuoles have a diverse array of functions. They can act as a storage organelle for nutrients or waste products, as a degradative compartment, as a cost-effective way of increasing cell size, and as a homeostatic regulator controlling both turgor pressure and pH of the cytosol." [GOC:mtg_sensu, ISBN:0198506732]
GO:0005774
name: vacuolar membrane
def: "The lipid bilayer surrounding the vacuole and separating its contents from the cytoplasm of the cell." [GOC:ai]
GO:0005775
name: vacuolar lumen
def: "The volume enclosed within the vacuolar membrane." [ISBN:0198506732]
GO:0005776
name: autophagic vacuole
def: "A double-membrane-bounded compartment in which endogenous cellular material is sequestered; known as autophagosome in yeast." [ISBN:0198547684, PMID:11099404]
GO:0005777
name: peroxisome
def: "A small organelle enclosed by a single membrane, and found in most eukaryotic cells. Contains peroxidases and other enzymes involved in a variety of metabolic processes including free radical detoxification, lipid catabolism and biosynthesis, and hydrogen peroxide metabolism." [GOC:pm, PMID:9302272, UniProtKB-KW:KW-0576]
GO:0005778
name: peroxisomal membrane
def: "The lipid bilayer surrounding a peroxisome." [GOC:mah]
GO:0005779
name: integral component of peroxisomal membrane
def: "The component of the peroxisomal membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:mah]
GO:0005780
name: extrinsic component of intraperoxisomal membrane
def: "The component of the intraperoxisomal membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:jl, GOC:mah]
GO:0005781
name: peroxisome targeting signal receptor complex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005782
name: peroxisomal matrix
def: "The volume contained within the membranes of a peroxisome; in many cells the matrix contains a crystalloid core largely composed of urate oxidase." [GOC:curators, ISBN:0815316194]
GO:0005783
name: endoplasmic reticulum
def: "The irregular network of unit membranes, visible only by electron microscopy, that occurs in the cytoplasm of many eukaryotic cells. The membranes form a complex meshwork of tubular channels, which are often expanded into slitlike cavities called cisternae. The ER takes two forms, rough (or granular), with ribosomes adhering to the outer surface, and smooth (with no ribosomes attached)." [ISBN:0198506732]
GO:0005784
name: Sec61 translocon complex
def: "A translocon complex that contains a core heterotrimer of conserved alpha, beta and gamma subunits, and may contain additional proteins (translocon-associated proteins or TRAPs); in budding yeast the core proteins are Sec61p, Sbh1p, and Sss1p. The Sec61 translocon complex functions in cotranslational and posttranslational translocation events." [GOC:mah, PMID:18166647]
GO:0005785
name: signal recognition particle receptor complex
def: "A transmembrane heterodimeric protein located in the membrane of the rough endoplasmic reticulum. Both subunits contain GTPase domains with which signal recognition particle interacts. In the presence of GTP and SRP receptor, SRP is released from the ribosome-nascent chain complex." [ISBN:0198506732]
GO:0005786
name: signal recognition particle, endoplasmic reticulum targeting
def: "A ribonucleoprotein particle of 325 kDa composed of a 7S (300 nucleotide) RNA molecule and a complex of six different polypeptides. This binds both to the N-terminal signal peptide for proteins destined for the endoplasmic reticulum as they emerge from the large ribosomal subunit and also to the ribosome. This binding arrests further translation thereby preventing the proteins from being released into the cytosol. The SRP-ribosome complex then diffuses to the endoplasmic reticulum where it is bound to the signal recognition particle receptor, which allows resumption of protein synthesis and facilitates the passage of the growing polypeptide chain through the translocon. Through a process involving GTP hydrolysis, the SRP-SRP receptor complex dissociates and SRP returns to the cytosol. Of the six polypeptides of SRP the 54 kDa subunit (SRP54) is the central player. It contains an N-terminal GTPase domain and a C-terminal domain that binds directly to the signal peptide and the SRP RNA. Examples of this component are found in Mus musculus, Saccharomyces cerevisiae and Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:0198506732]
GO:0005787
name: signal peptidase complex
def: "A protein complex that is located in the endoplasmic reticulum membrane and cleaves the signal sequence from precursor proteins following their transport out of the cytoplasmic space." [GOC:sgd_curators, PMID:1846444, PMID:7615509]
GO:0005788
name: endoplasmic reticulum lumen
def: "The volume enclosed by the membranes of the endoplasmic reticulum." [ISBN:0198547684]
GO:0005789
name: endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the endoplasmic reticulum." [GOC:mah]
GO:0005790
name: smooth endoplasmic reticulum
def: "The smooth endoplasmic reticulum (smooth ER or SER) has no ribosomes attached to it. The smooth ER is the recipient of the proteins synthesized in the rough ER. Those proteins to be exported are passed to the Golgi complex, the resident proteins are returned to the rough ER and the lysosomal proteins after phosphorylation of their mannose residues are passed to the lysosomes. Glycosylation of the glycoproteins also continues. The smooth ER is the site of synthesis of lipids, including the phospholipids. The membranes of the smooth ER also contain enzymes that catalyze a series of reactions to detoxify both lipid-soluble drugs and harmful products of metabolism. Large quantities of certain compounds such as phenobarbital cause an increase in the amount of the smooth ER." [ISBN:0198506732]
GO:0005791
name: rough endoplasmic reticulum
def: "The rough (or granular) endoplasmic reticulum (ER) has ribosomes adhering to the outer surface; the ribosomes are the site of translation of the mRNA for those proteins which are either to be retained within the cisternae (ER-resident proteins), the proteins of the lysosomes, or the proteins destined for export from the cell. Glycoproteins undergo their initial glycosylation within the cisternae." [ISBN:0198506732]
GO:0005792
name: microsome
def: "OBSOLETE: Any of the small, heterogeneous, artifactual, vesicular particles, 50-150 nm in diameter, that are formed when some eukaryotic cells are homogenized and that sediment on centrifugation at 100000 g." [ISBN:0198506732]
GO:0005793
name: endoplasmic reticulum-Golgi intermediate compartment
def: "A complex system of membrane-bounded compartments located between endoplasmic reticulum (ER) and the Golgi complex, with a distinctive membrane protein composition; involved in ER-to-Golgi  and Golgi-to-ER transport." [GOC:pr, PMID:16723730]
GO:0005794
name: Golgi apparatus
def: "A compound membranous cytoplasmic organelle of eukaryotic cells, consisting of flattened, ribosome-free vesicles arranged in a more or less regular stack. The Golgi apparatus differs from the endoplasmic reticulum in often having slightly thicker membranes, appearing in sections as a characteristic shallow semicircle so that the convex side (cis or entry face) abuts the endoplasmic reticulum, secretory vesicles emerging from the concave side (trans or exit face). In vertebrate cells there is usually one such organelle, while in invertebrates and plants, where they are known usually as dictyosomes, there may be several scattered in the cytoplasm. The Golgi apparatus processes proteins produced on the ribosomes of the rough endoplasmic reticulum; such processing includes modification of the core oligosaccharides of glycoproteins, and the sorting and packaging of proteins for transport to a variety of cellular locations. Three different regions of the Golgi are now recognized both in terms of structure and function: cis, in the vicinity of the cis face, trans, in the vicinity of the trans face, and medial, lying between the cis and trans regions." [ISBN:0198506732]
GO:0005795
name: Golgi stack
def: "The set of thin, flattened membrane-bounded compartments, called cisternae, that form the central portion of the Golgi complex. The stack usually comprises cis, medial, and trans cisternae; the cis- and trans-Golgi networks are not considered part of the stack." [GOC:mah, ISBN:0815316194]
GO:0005796
name: Golgi lumen
def: "The volume enclosed by the membranes of any cisterna or subcompartment of the Golgi apparatus, including the cis- and trans-Golgi networks." [GOC:mah]
GO:0005797
name: Golgi medial cisterna
def: "The middle Golgi cisterna (or cisternae)." [ISBN:0815316194]
GO:0005798
name: Golgi-associated vesicle
def: "Any vesicle associated with the Golgi complex and involved in mediating transport within the Golgi or between the Golgi and other parts of the cell." [GOC:mah]
GO:0005799
name: coatomer
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005800
name: COPII vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005801
name: cis-Golgi network
def: "The network of interconnected tubular and cisternal structures located at the convex side of the Golgi apparatus, which abuts the endoplasmic reticulum." [ISBN:0198506732, ISBN:0815316194]
GO:0005802
name: trans-Golgi network
def: "The network of interconnected tubular and cisternal structures located within the Golgi apparatus on the side distal to the endoplasmic reticulum, from which secretory vesicles emerge. The trans-Golgi network is important in the later stages of protein secretion where it is thought to play a key role in the sorting and targeting of secreted proteins to the correct destination." [GOC:vw, ISBN:0815316194]
GO:0005803
name: secretory vesicle
def: "OBSOLETE. A small subcellular vesicle, or granule, surrounded by a single-layered membrane; formed from the Golgi apparatus and contains a highly concentrated protein destined for secretion." [ISBN:0198547684]
GO:0005804
name: secretory vesicle membrane
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005805
name: ER-Golgi transport vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005806
name: Golgi-ER transport vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005808
name: Golgi-plasma membrane transport vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005809
name: Golgi-vacuole transport vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005810
name: endocytotic transport vesicle
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005811
name: lipid particle
def: "An intracellular non-membrane-bounded organelle comprising a matrix of coalesced lipids surrounded by a phospholipid monolayer. May include associated proteins." [GOC:mah, GOC:tb]
GO:0005813
name: centrosome
def: "A structure comprised of a core structure (in most organisms, a pair of centrioles) and peripheral material from which a microtubule-based structure, such as a spindle apparatus, is organized. Centrosomes occur close to the nucleus during interphase in many eukaryotic cells, though in animal cells it changes continually during the cell-division cycle." [GOC:mah, ISBN:0198547684]
GO:0005814
name: centriole
def: "A cellular organelle, found close to the nucleus in many eukaryotic cells, consisting of a small cylinder with microtubular walls, 300-500 nm long and 150-250 nm in diameter. It contains nine short, parallel, peripheral microtubular fibrils, each fibril consisting of one complete microtubule fused to two incomplete microtubules. Cells usually have two centrioles, lying at right angles to each other. At division, each pair of centrioles generates another pair and the twin pairs form the pole of the mitotic spindle." [ISBN:0198547684]
GO:0005815
name: microtubule organizing center
def: "An intracellular structure that can catalyze gamma-tubulin-dependent microtubule nucleation and that can anchor microtubules by interacting with their minus ends, plus ends or sides." [GOC:vw, http://en.wikipedia.org/wiki/Microtubule_organizing_center, ISBN:0815316194, PMID:17072892, PMID:17245416]
GO:0005816
name: spindle pole body
def: "The microtubule organizing center in fungi; functionally homologous to the animal cell centrosome." [ISBN:0879693568]
GO:0005817
name: centrosomal mitotic factor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:clt]
GO:0005818
name: aster
def: "An array of microtubules emanating from a spindle pole MTOC that do not connect to kinetochores." [GOC:clt]
GO:0005819
name: spindle
def: "The array of microtubules and associated molecules that forms between opposite poles of a eukaryotic cell during mitosis or meiosis and serves to move the duplicated chromosomes apart." [ISBN:0198547684]
GO:0005821
name: intermediate layer of spindle pole body
def: "Structure between the central and outer plaques of the spindle pole body." [PMID:9215630]
GO:0005822
name: inner plaque of spindle pole body
def: "One of three laminate structures that form the spindle pole body; the inner plaque is in the nucleus." [ISBN:0879693568]
GO:0005823
name: central plaque of spindle pole body
def: "One of three laminate structures that form the spindle pole body; the central plaque is embedded in the nuclear envelope." [ISBN:0879693568]
GO:0005824
name: outer plaque of spindle pole body
def: "One of three laminate structures that form the spindle pole body; the outer plaque is in the cytoplasm." [ISBN:0879693568]
GO:0005825
name: half bridge of spindle pole body
def: "Structure adjacent to the plaques of the spindle pole body." [ISBN:0879693568]
GO:0005826
name: actomyosin contractile ring
def: "A cytoskeletal structure composed of actin filaments and myosin that forms beneath the plasma membrane of many cells, including animal cells and yeast cells, in a plane perpendicular to the axis of the spindle, i.e. the cell division plane. Ring contraction is associated with centripetal growth of the membrane that divides the cytoplasm of the two daughter cells. In animal cells, the contractile ring is located inside the plasma membrane at the location of the cleavage furrow. In budding fungal cells, e.g. mitotic S. cerevisiae cells, the contractile ring forms beneath the plasma membrane at the mother-bud neck before mitosis." [GOC:expert_jrp, GOC:sgd_curators, ISBN:0805319409, ISBN:0815316194]
GO:0005827
name: polar microtubule
def: "Any of the spindle microtubules that come from each pole and overlap at the spindle midzone. This interdigitating structure consisting of antiparallel microtubules is responsible for pushing the poles of the spindle apart." [ISBN:0815316194]
GO:0005828
name: kinetochore microtubule
def: "Any of the spindle microtubules that attach to the kinetochores of chromosomes by their plus ends, and maneuver the chromosomes during mitotic or meiotic chromosome segregation." [ISBN:0815316194]
GO:0005829
name: cytosol
def: "The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes." [GOC:hgd, GOC:jl]
GO:0005831
name: steroid hormone aporeceptor complex
def: "A protein complex consisting of a steroid receptor associated with nonreceptor proteins, minimally a dimer of Hsp90 and a monomer of hsp56/FKBP59; forms in the absence of bound ligand." [PMID:7493981]
GO:0005832
name: chaperonin-containing T-complex
def: "A multisubunit ring-shaped complex that mediates protein folding in the cytosol without a cofactor." [GOC:sgd_curators, PMID:11580267]
GO:0005833
name: hemoglobin complex
def: "An iron-containing, oxygen carrying complex. In vertebrates it is made up of two pairs of associated globin polypeptide chains, each chain carrying a noncovalently bound heme prosthetic group." [GOC:jl, ISBN:0198506732]
GO:0005834
name: heterotrimeric G-protein complex
def: "Any of a family of heterotrimeric GTP-binding and hydrolyzing proteins; they belong to a superfamily of GTPases that includes monomeric proteins such as EF-Tu and RAS. Heterotrimeric G-proteins consist of three subunits; the alpha subunit contains the guanine nucleotide binding site and possesses GTPase activity; the beta and gamma subunits are tightly associated and function as a beta-gamma heterodimer; extrinsic plasma membrane proteins (cytoplasmic face) that function as a complex to transduce signals from G-protein coupled receptors to an effector protein." [ISBN:0198547684]
GO:0005835
name: fatty acid synthase complex
def: "A multienzyme complex that catalyses the synthesis of fatty acids from acetyl CoA." [GOC:pde, GOC:sgd_curators, ISBN:0716746840]
GO:0005836
name: fatty-acyl-CoA synthase complex
def: "A protein complex that possesses fatty-acyl-CoA synthase activity." [BRENDA:2.3.1.86, GOC:mah]
GO:0005837
name: 26S proteasome
def: "OBSOLETE. A large multisubunit protease found in the cytosol that recognizes, unfolds, and digests protein substrates that have been marked for degradation by the attachment of a ubiquitin group. Individual subcomplexes of the complete 26S proteasome are involved in these different tasks: the ATP-dependent 19S caps are believed to unfold substrates and feed them to the actual protease, the 20S proteasome." [PMID:10410804]
GO:0005838
name: proteasome regulatory particle
def: "A multisubunit complex, which caps one or both ends of the proteasome core complex. This complex recognizes and unfolds ubiquitinated proteins, and translocates them to the proteasome core complex." [GOC:mtg_sensu, GOC:rb]
GO:0005839
name: proteasome core complex
def: "A multisubunit barrel shaped endoprotease complex, which is the core of the proteasome complex." [GOC:rb, PMID:10806206]
GO:0005840
name: ribosome
def: "An intracellular organelle, about 200 A in diameter, consisting of RNA and protein. It is the site of protein biosynthesis resulting from translation of messenger RNA (mRNA). It consists of two subunits, one large and one small, each containing only protein and RNA. Both the ribosome and its subunits are characterized by their sedimentation coefficients, expressed in Svedberg units (symbol: S). Hence, the prokaryotic ribosome (70S) comprises a large (50S) subunit and a small (30S) subunit, while the eukaryotic ribosome (80S) comprises a large (60S) subunit and a small (40S) subunit. Two sites on the ribosomal large subunit are involved in translation, namely the aminoacyl site (A site) and peptidyl site (P site). Ribosomes from prokaryotes, eukaryotes, mitochondria, and chloroplasts have characteristically distinct ribosomal proteins." [ISBN:0198506732]
GO:0005844
name: polysome
def: "A multiribosomal structure representing a linear array of ribosomes held together by messenger RNA. They represent the active complexes in cellular protein synthesis and are able to incorporate amino acids into polypeptides both in vivo and in vitro." [ISBN:0198506732, NIF_Subcellular:sao1038025871]
GO:0005845
name: mRNA cap binding complex
def: "Any protein complex that binds to an mRNA cap at any time in the lifetime of the mRNA." [GOC:jid]
GO:0005846
name: nuclear cap binding complex
def: "A conserved heterodimeric protein complex that binds to the 5' terminal cap structure m7G(5')ppp(5')N of nascent eukaryotic RNA polymerase II transcripts such as pre-mRNA and U snRNA. The consists of proteins known as CBP20 and CBP80, binds to cap structures in the nucleus, and is involved in pre-mRNA splicing, 3'-end formation, and RNA nuclear export." [PMID:16043498]
GO:0005847
name: mRNA cleavage and polyadenylation specificity factor complex
def: "A multisubunit complex that binds to the canonical AAUAAA hexamer and to U-rich upstream sequence elements on the pre-mRNA, thereby stimulating the otherwise weakly active and nonspecific polymerase to elongate efficiently RNAs containing a poly(A) signal." [PMID:14749727]
GO:0005848
name: mRNA cleavage stimulating factor complex
def: "A protein complex required for mRNA cleavage but not for poly(A) addition." [GOC:mah, PMID:10357856]
GO:0005849
name: mRNA cleavage factor complex
def: "Any macromolecular complex involved in cleavage or polyadenylation of mRNA molecules." [GOC:mah, PMID:10357856]
GO:0005850
name: eukaryotic translation initiation factor 2 complex
def: "Complex of three heterogeneous polypeptide chains, that form a ternary complex with initiator methionyl-tRNA and GTP. This ternary complex binds to free 40S subunit, which subsequently binds the 5' end of mRNA." [PMID:10216940]
GO:0005851
name: eukaryotic translation initiation factor 2B complex
def: "A multisubunit guanine nucleotide exchange factor which catalyzes the exchange of GDP bound to initiation factor eIF2 for GTP, generating active eIF2-GTP. In humans, it is composed of five subunits, alpha, beta, delta, gamma and epsilon." [PMID:9438375]
GO:0005852
name: eukaryotic translation initiation factor 3 complex
def: "A complex of several polypeptides that plays at least two important roles in protein synthesis: First, eIF3 binds to the 40S ribosome and facilitates loading of the Met-tRNA/eIF2.GTP ternary complex to form the 43S preinitiation complex. Subsequently, eIF3 apparently assists eIF4 in recruiting mRNAs to the 43S complex. The eIF3 complex contains five conserved core subunits, and may contain several additional proteins; the non-core subunits are thought to mediate association of the complex with specific sets of mRNAs." [PMID:15904532]
GO:0005853
name: eukaryotic translation elongation factor 1 complex
def: "A multisubunit nucleotide exchange complex that binds GTP and aminoacyl-tRNAs, and catalyzes their codon-dependent placement at the A-site of the ribosome. In humans, the complex is composed of four subunits, alpha, beta, delta and gamma." [GOC:jl, PMID:10216950]
GO:0005854
name: nascent polypeptide-associated complex
def: "A heterodimeric protein complex that can reversibly bind to ribosomes, and is located in direct proximity to newly synthesized polypeptide chains as they emerge from the ribosome." [PMID:12475173, PMID:7568149]
GO:0005856
name: cytoskeleton
def: "Any of the various filamentous elements that form the internal framework of cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the cell. The various elements of the cytoskeleton not only serve in the maintenance of cellular shape but also have roles in other cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles." [GOC:mah, ISBN:0198547684, PMID:16959967]
GO:0005858
name: axonemal dynein complex
def: "A dynein complex found in eukaryotic cilia and flagella; the motor domain heads interact with adjacent microtubules to generate a sliding force which is converted to a bending motion. May contain two or three dynein heavy chains as well as several light chains." [ISBN:0815316194]
GO:0005859
name: muscle myosin complex
def: "A filament of myosin found in a muscle cell of any type." [GOC:mah]
GO:0005861
name: troponin complex
def: "A complex of accessory proteins (typically troponin T, troponin I and troponin C) found associated with actin in muscle thin filaments; involved in calcium regulation of muscle contraction." [ISBN:0815316194]
GO:0005862
name: muscle thin filament tropomyosin
def: "A form of the tropomyosin dimer found associated with actin and the troponin complex in muscle thin filaments." [ISBN:0815316194]
GO:0005863
name: striated muscle myosin thick filament
def: "Bipolar filaments formed of polymers of a muscle-specific myosin II isoform, found in the middle of sarcomeres in myofibrils." [GOC:mtg_muscle, ISBN:0815316194]
GO:0005865
name: striated muscle thin filament
def: "Filaments formed of actin and associated proteins; attached to Z discs at either end of sarcomeres in myofibrils." [ISBN:0815316194]
GO:0005868
name: cytoplasmic dynein complex
def: "Any dynein complex that catalyzes movement along a cytoplasmic microtubule; cytoplasmic dynein complexes participates in many transport activities in eukaryotes, such as mRNA localization, intermediate filament transport, nuclear envelope breakdown, apoptosis, transport of centrosomal proteins, mitotic spindle assembly, virus transport, kinetochore functions, and movement of signaling and spindle checkpoint proteins. Subunits associated with the dynein heavy chain mediate association between dynein heavy chain and cargoes,and may include light chains and light intermediate chains." [GOC:mah, PMID:12600311]
GO:0005869
name: dynactin complex
def: "A 20S multiprotein assembly of total mass about 1.2 MDa that activates dynein-based activity in vivo. A large structural component of the complex is an actin-like 40 nm filament composed of actin-related protein, to which other components attach." [ISBN:0198506732]
GO:0005870
name: actin capping protein of dynactin complex
def: "A heterodimer consisting of alpha and beta subunits that binds to and caps the barbed ends of actin filaments, nucleates the polymerization of actin monomers but does not sever actin filaments, and which is a part of the dynactin complex." [GOC:jl, PMID:18221362, PMID:18544499]
GO:0005871
name: kinesin complex
def: "Any complex that includes a dimer of molecules from the kinesin superfamily, a group of related proteins that contain an extended region of predicted alpha-helical coiled coil in the main chain that likely produces dimerization. The native complexes of several kinesin family members have also been shown to contain additional peptides, often designated light chains as all of the noncatalytic subunits that are currently known are smaller than the chain that contains the motor unit. Kinesin complexes generally possess a force-generating enzymatic activity, or motor, which converts the free energy of the gamma phosphate bond of ATP into mechanical work." [GOC:mah, http://www.proweb.org/kinesin//KinesinMotility.html, http://www.proweb.org/kinesin//KinesinStructure.html]
GO:0005872
name: minus-end kinesin complex
def: "Any complex that includes a dimer of molecules from the kinesin superfamily and any associated proteins, and moves towards the minus end of a microtubule." [GOC:mah]
GO:0005873
name: plus-end kinesin complex
def: "Any complex that includes a dimer of molecules from the kinesin superfamily and any associated proteins, and moves towards the plus end of a microtubule." [GOC:mah]
GO:0005874
name: microtubule
def: "Any of the long, generally straight, hollow tubes of internal diameter 12-15 nm and external diameter 24 nm found in a wide variety of eukaryotic cells; each consists (usually) of 13 protofilaments of polymeric tubulin, staggered in such a manner that the tubulin monomers are arranged in a helical pattern on the microtubular surface, and with the alpha/beta axes of the tubulin subunits parallel to the long axis of the tubule; exist in equilibrium with pool of tubulin monomers and can be rapidly assembled or disassembled in response to physiological stimuli; concerned with force generation, e.g. in the spindle." [ISBN:0879693568]
GO:0005875
name: microtubule associated complex
def: "Any multimeric complex connected to a microtubule." [GOC:jl]
GO:0005876
name: spindle microtubule
def: "Any microtubule that is part of a mitotic or meiotic spindle; anchored at one spindle pole." [ISBN:0815316194]
GO:0005879
name: axonemal microtubule
def: "A microtubule in the axoneme of a eukaryotic cilium or flagellum; an axoneme contains nine modified doublet microtubules, which may or may not surround a pair of single microtubules." [GOC:cilia, ISBN:0815316194]
GO:0005880
name: nuclear microtubule
def: "Any microtubule in the nucleus of a cell." [GOC:mah]
GO:0005881
name: cytoplasmic microtubule
def: "Any microtubule in the cytoplasm of a cell." [GOC:mah]
GO:0005882
name: intermediate filament
def: "A cytoskeletal structure that forms a distinct elongated structure, characteristically 10 nm in diameter, that occurs in the cytoplasm of eukaryotic cells. Intermediate filaments form a fibrous system, composed of chemically heterogeneous subunits and involved in mechanically integrating the various components of the cytoplasmic space. Intermediate filaments may be divided into five chemically distinct classes: Type I, acidic keratins; Type II, basic keratins; Type III, including desmin, vimentin and others; Type IV, neurofilaments and related filaments; and Type V, lamins." [http://www.cytochemistry.net/Cell-biology/intermediate_filaments.htm, ISBN:0198506732]
GO:0005883
name: neurofilament
def: "A type of intermediate filament found in the core of neuronal axons. Neurofilaments are heteropolymers composed of three type IV polypeptides: NF-L, NF-M, and NF-H (for low, middle, and high molecular weight). Neurofilaments are responsible for the radial growth of an axon and determine axonal diameter." [ISBN:0198506732, ISBN:0716731363, ISBN:0815316194]
GO:0005884
name: actin filament
def: "A filamentous structure formed of a two-stranded helical polymer of the protein actin and associated proteins. Actin filaments are a major component of the contractile apparatus of skeletal muscle and the microfilaments of the cytoskeleton of eukaryotic cells. The filaments, comprising polymerized globular actin molecules, appear as flexible structures with a diameter of 5-9 nm. They are organized into a variety of linear bundles, two-dimensional networks, and three dimensional gels. In the cytoskeleton they are most highly concentrated in the cortex of the cell just beneath the plasma membrane." [GOC:mah, ISBN:0198506732, PMID:10666339]
GO:0005885
name: Arp2/3 protein complex
def: "A stable protein complex that contains two actin-related proteins, Arp2 and Arp3, and five novel proteins (ARPC1-5), and functions in the nucleation of branched actin filaments." [GOC:jl, GOC:vw, PMID:12479800]
GO:0005886
name: plasma membrane
def: "The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins." [ISBN:0716731363]
GO:0005887
name: integral component of plasma membrane
def: "The component of the plasma membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:go_curators]
GO:0005888
name: proteoglycan integral to plasma membrane
def: "OBSOLETE. Penetrating at least one phospholipid bilayer of a plasma membrane and consisting of proteoglycan. Also refers to the state of being buried in the bilayer with no exposure outside the bilayer." [GOC:go_curators]
GO:0005889
name: hydrogen:potassium-exchanging ATPase complex
def: "A protein complex that possesses hydrogen:potassium-exchanging ATPase activity; characterized in animal cells, where it maintains ionic gradients of K+ at the expense of ATP hydrolysis; The complex contains two obligatory subunits, the catalytic alpha subunit and a glycosylated beta subunit; two additional subunits, gamma and channel-inducing factor (CHIF), may also be present." [PMID:11756431]
GO:0005890
name: sodium:potassium-exchanging ATPase complex
def: "Sodium:potassium-exchanging ATPases are tetrameric proteins, consisting of two large alpha subunits and two smaller beta subunits. The alpha subunits bear the active site and penetrate the membrane, while the beta subunits carry oligosaccharide groups and face the cell exterior." [ISBN:0198506732]
GO:0005891
name: voltage-gated calcium channel complex
def: "A protein complex that forms a transmembrane channel through which calcium ions may pass in response to changes in membrane potential." [GOC:mah]
GO:0005892
name: acetylcholine-gated channel complex
def: "A homo- or hetero-pentameric protein complex that forms a transmembrane channel through which ions may pass in response to acetylcholine binding." [GOC:bf, GOC:mah, PMID:12381728, PMID:15579462]
GO:0005893
name: interleukin-2 receptor complex
def: "A protein complex that binds interleukin-2; comprises alpha, beta, and gamma subunits." [GOC:mah, PMID:3116143, PMID:8266078]
GO:0005894
name: interleukin-3 receptor complex
def: "A protein complex that binds interleukin-3; comprises an alpha and a beta subunit. The alpha chain is specific to the interleukin-3 receptor, whereas the beta chain is shared with the receptors for granulocyte-macrophage colony-stimulating factor and interleukin-5." [PMID:11839579]
GO:0005895
name: interleukin-5 receptor complex
def: "A protein complex that binds interleukin-3; comprises an alpha and a beta subunit. The alpha chain is specific to the interleukin-5 receptor, whereas the beta chain is shared with the receptors for granulocyte-macrophage colony-stimulating factor and interleukin-3." [GOC:mah, PMID:11312115, PMID:11839579]
GO:0005896
name: interleukin-6 receptor complex
def: "A hexameric protein complex consisting of two molecules each of interleukin-6, interleukin-6 receptor alpha chain, and gp-130." [PMID:8083235]
GO:0005897
name: interleukin-9 receptor complex
def: "A protein complex that binds interleukin-9; comprises an alpha and a beta subunit. The alpha chain is specific to the interleukin-9 receptor, whereas the beta chain is shared with the receptors for several other interleukins." [GOC:mah, PMID:10642536]
GO:0005898
name: interleukin-13 receptor complex
def: "A protein complex that binds interleukin-13; consists of two chains, interleukin-13 receptor alpha1 chain and interleukin-4 receptor alpha chain." [PMID:8552669, PMID:9013879]
GO:0005899
name: insulin receptor complex
def: "A disulfide-bonded, heterotetrameric receptor complex. The alpha chains are entirely extracellular, while each beta chain has one transmembrane domain. The ligand binds to the alpha subunit extracellular domain and the kinase is associated with the beta subunit intracellular domain." [ISBN:0198506732]
GO:0005900
name: oncostatin-M receptor complex
def: "A heterodimeric receptor for the cytokine oncostatin-M (OSM). In humans the receptor complex is made up of the gene products gp130 and OSMR-beta." [GOC:jl, PMID:8999038]
GO:0005901
name: caveola
def: "A membrane raft that forms small pit, depression, or invagination that communicates with the outside of a cell and extends inward, indenting the cytoplasm and the cell membrane. Examples include any of the minute pits or incuppings of the cell membrane formed during pinocytosis. Such caveolae may be pinched off to form free vesicles within the cytoplasm." [GOC:mah, ISBN:0721662544, PMID:16645198]
GO:0005902
name: microvillus
def: "Thin cylindrical membrane-covered projections on the surface of an animal cell containing a core bundle of actin filaments. Present in especially large numbers on the absorptive surface of intestinal cells." [ISBN:0813516194]
GO:0005903
name: brush border
def: "Dense covering of microvilli on the apical surface of epithelial cells in tissues such as the intestine, kidney, and choroid plexus; the microvilli aid absorption by increasing the surface area of the cell." [GOC:sl, ISBN:0815316194]
GO:0005905
name: coated pit
def: "A part of the endomembrane system in the form of an invagination of a membrane upon which a clathrin coat forms, and that can be converted by vesicle budding into a clathrin-coated vesicle. Coated pits form on the plasma membrane, where they are involved in receptor-mediated selective transport of many proteins and other macromolecules across the cell membrane, in the trans-Golgi network, and on some endosomes." [GOC:mah, ISBN:0198506732, NIF_Subcellular:sao1969557946, PMID:10559856, PMID:17284835]
GO:0005906
name: clathrin adaptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005907
name: HA1 clathrin adaptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005908
name: HA2 clathrin adaptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005911
name: cell-cell junction
def: "A cell junction that forms a connection between two cells; excludes direct cytoplasmic junctions such as ring canals." [GOC:dgh, GOC:hb, GOC:mah]
GO:0005912
name: adherens junction
def: "A cell junction at which anchoring proteins (cadherins or integrins) extend through the plasma membrane and are attached to actin filaments." [GOC:mah, http://www.vivo.colostate.edu/hbooks/cmb/cells/pmemb/junctions_a.html, ISBN:0198506732]
GO:0005913
name: cell-cell adherens junction
def: "An adherens junction which connects a cell to another cell." [GOC:hb]
GO:0005914
name: spot adherens junction
def: "A small cell-cell adherens junction assembled during the cellularization stage of insect embyrogenesis; spot adherens junctions later fuse to form the zonula adherens." [PMID:11700298]
GO:0005915
name: zonula adherens
def: "A cell-cell adherens junction which forms a continuous belt near the apex of epithelial cells." [ISBN:0815316208]
GO:0005916
name: fascia adherens
def: "A cell-cell adherens junction that contains the transmembrane protein N-cadherin, which interacts with identical molecules from neighboring cells to form a tight mechanical intercellular link; forms a large portion of the intercalated disc, the structure at which myofibrils terminate in cardiomyocytes." [GOC:mtg_muscle, PMID:11732910]
GO:0005917
name: nephrocyte diaphragm
def: "A specialized cell-cell junction found between nephrocytes of the insect kidney, which is adapted for filtration of hemolymph. The insect nephrocyte is anatomically and functionally similarity to the glomerular podocyte of vertebrates." [GOC:mtg_kidney_jan10, GOC:sart, PMID:18971929]
GO:0005918
name: septate junction
def: "A cell-cell junction that forms a continuous band around each cell in an epithelium; within the septate junction the membranes of adjacent cells maintain a constant distance of approximately 15 nm; found in invertebrates." [ISBN:0815332181, PMID:11700298]
GO:0005919
name: pleated septate junction
def: "A septate junction in which regular arrays of electron-dense septae span the intermembrane space." [PMID:11700298]
GO:0005920
name: smooth septate junction
def: "A septate junction that lacks the regular arrays of electron-dense septae found in pleated septate junctions." [PMID:11700298]
GO:0005921
name: gap junction
def: "A cell-cell junction that is composed of an array of small channels that permit small molecules to pass from one cell to another. At gap junctions, the membranes of two adjacent cells are separated by a uniform narrow gap of about 2-4 nm that is spanned by channel-forming proteins called connexins, which form hexagonal tubes called connexons." [GOC:mah, GOC:mtg_muscle, http://www.vivo.colostate.edu/hbooks/cmb/cells/pmemb/junctions_g.html, ISBN:0815332181]
GO:0005922
name: connexon complex
def: "An assembly of six molecules of connexin, made in the Golgi apparatus and subsequently transported to the plasma membrane, where docking of two connexons on apposed plasma membranes across the extracellular space forms a gap junction." [PMID:11146276]
GO:0005923
name: tight junction
def: "An occluding cell-cell junction that is composed of a branching network of sealing strands that completely encircles the apical end of each cell in an epithelial sheet; the outer leaflets of the two interacting plasma membranes are seen to be tightly apposed where sealing strands are present. Each sealing strand is composed of a long row of transmembrane adhesion proteins embedded in each of the two interacting plasma membranes." [GOC:mah, ISBN:0815332181]
GO:0005924
name: cell-substrate adherens junction
def: "An adherens junction which connects a cell to the extracellular matrix." [GOC:hb]
GO:0005925
name: focal adhesion
def: "Small region on the surface of a cell that anchors the cell to the extracellular matrix and that forms a point of termination of actin filaments." [ISBN:0124325653, ISBN:0815316208]
GO:0005926
name: connecting hemi-adherens junction
def: "A cell-substrate adherens junction, also known as a hemiadherens junction (HAJ) that forms one of a pair of junctions in opposing cells that are separated by only 30-40nm, with a thin line of extracellular electron-dense material in between; found where muscles attach to epidermal cells directly (in insects)." [http://flybase.bio.indiana.edu/allied-data/lk/interactive-fly/aignfam/junction.htm]
GO:0005927
name: muscle tendon junction
def: "A cell-substrate junction found at the terminal anchorage site of skeletal muscle cells to tendons." [GOC:mtg_muscle, PMID:12842007]
GO:0005928
name: apical hemi-adherens junction
def: "A cell-substrate adherens junction found in the apical region of a cell, such as those found in cuticle-secreting epithelia, which connect the apical membrane to the cuticle." [GOC:mah, PMID:11700298]
GO:0005929
name: cilium
def: "A specialized eukaryotic organelle that consists of a filiform extrusion of the cell surface and of some cytoplasmic parts. Each cilium is largely bounded by an extrusion of the cytoplasmic (plasma) membrane, and contains a regular longitudinal array of microtubules, anchored to a basal body." [GOC:cilia, GOC:kmv, GOC:vw, ISBN:0198547684]
GO:0005930
name: axoneme
def: "The bundle of microtubules and associated proteins that forms the core of cilia (also called flagella) in eukaryotic cells and is responsible for their movements." [GOC:bf, GOC:cilia, ISBN:0198547684]
GO:0005931
name: nexin complex
def: "A protein complex found in the axoneme of eukaryotic cilia and flagella. It forms interconnections between the microtubule outer doublets that surround the inner central pair of microtubules." [ISBN:0198506732]
GO:0005933
name: cellular bud
def: "A protuberance from a cell of an organism that reproduces by budding, which will grow larger and become a separate daughter cell after nuclear division, cytokinesis, and cell wall formation (when appropriate). The daughter cell may completely separate from the mother cell, or the mother and daughter cells may remain associated." [GOC:sgd_curators]
GO:0005934
name: cellular bud tip
def: "The end of a cellular bud distal to the site of attachment to the mother cell." [GOC:mah]
GO:0005935
name: cellular bud neck
def: "The constriction between the mother cell and daughter cell (bud) in an organism that reproduces by budding." [GOC:mah]
GO:0005936
name: shmoo
def: "OBSOLETE. The characteristic projection formed in response to mating pheromone by cells of Saccharomyces and other fungi with similar life cycles. Named after the Al Capp cartoon character, whose shape it resembles." [GOC:mah, GOC:mcc]
GO:0005937
name: mating projection
def: "The projection formed by unicellular fungi in response to mating pheromone." [GOC:mcc]
GO:0005938
name: cell cortex
def: "The region of a cell that lies just beneath the plasma membrane and often, but not always, contains a network of actin filaments and associated proteins." [GOC:mah, ISBN:0815316194]
GO:0005940
name: septin ring
def: "A tight ring-shaped structure that forms in the division plane at the site of cytokinesis; composed of members of the conserved family of filament-forming proteins called septins as well as septin-associated proteins. This type of septin structure is observed at the bud neck of budding fungal cells, at the site of cell division in animal cells, at the junction between the mother cell and a pseudohyphal projection, and also within hyphae of filamentous fungi at sites where a septum will form." [GOC:krc, GOC:mah, PMID:16009555, PMID:16151244]
GO:0005941
name: unlocalized protein complex
def: "OBSOLETE. Used as a holding place for cellular components whose precise localization is, as yet, unknown, or has not been determined by GO (the latter is the major reason for nodes to have this parent); this term should not be used for annotation of gene products." [GOC:ma]
GO:0005942
name: phosphatidylinositol 3-kinase complex
def: "A protein complex capable of phosphatidylinositol 3-kinase activity and containing subunits of any phosphatidylinositol 3-kinase (PI3K) enzyme. These complexes are divided in three classes (called I, II and III) that differ for their presence across taxonomic groups and for the type of their constituents. Catalytic subunits of phosphatidylinositol 3-kinase enzymes are present in all 3 classes; regulatory subunits of phosphatidylinositol 3-kinase enzymes are present in classes I and III; adaptor proteins have been observed in class II complexes and may be present in other classes too." [GOC:bf, PMID:24587488]
GO:0005943
name: phosphatidylinositol 3-kinase complex, class IA
def: "A class I phosphatidylinositol 3-kinase complex that possesses 1-phosphatidylinositol-4-phosphate 3-kinase activity; comprises a catalytic class IA phosphoinositide 3-kinase (PI3K) subunit and an associated SH2 domain-containing regulatory subunit that is a member of a family of related proteins often called p85 proteins. Through the interaction with the SH2-containing adaptor subunits, Class IA PI3K catalytic subunits are linked to tyrosine kinase signaling pathways." [PMID:9255069, PMID:9759495]
GO:0005944
name: phosphatidylinositol 3-kinase complex, class IB
def: "A class I phosphatidylinositol 3-kinase complex that possesses 1-phosphatidylinositol-4-phosphate 3-kinase activity; comprises a catalytic class IB phosphoinositide 3-kinase (PI3K) subunit and an associated regulatory subunit that is larger than, and unrelated to, the p85 proteins present in class IA complexes. Class IB PI3Ks are stimulated by G-proteins and do not interact with the SH2-domain containing adaptors that bind to Class IA PI3Ks." [PMID:9255069, PMID:9759495]
GO:0005945
name: 6-phosphofructokinase complex
def: "A protein complex that possesses 6-phosphofructokinase activity; homodimeric, homooctameric, and allosteric homotetrameric forms are known." [GOC:mah, GOC:vw, ISBN:0198506732]
GO:0005946
name: alpha,alpha-trehalose-phosphate synthase complex (UDP-forming)
def: "A protein complex that possesses alpha,alpha-trehalose-phosphate synthase (UDP-forming) and trehalose-phosphatase activities, and thus catalyzes two reactions in trehalose biosynthesis. In the complex identified in Saccharomyces, Tps1p has alpha,alpha-trehalose-phosphate synthase (UDP-forming) activity, Tps2p has trehalose 6-phosphate phosphatase activity; Tps3p is a regulatory subunit, and an additional subunit, Tsl1p, may be present." [PMID:9837904]
GO:0005947
name: mitochondrial alpha-ketoglutarate dehydrogenase complex
def: "Mitochondrial complex that possesses alpha-ketoglutarate dehydrogenase activity." [GOC:mah, GOC:mtg_sensu]
GO:0005948
name: acetolactate synthase complex
def: "A dimeric (a large and a small chain) or tetrameric (two large and two small chains) enzyme complex. Catalyzes the formation of acetolactate from pyruvate." [BRENDA:2.2.1.6, GOC:jl, PMID:16458324, PMID:8756689]
GO:0005949
name: aminoadipate-semialdehyde dehydrogenase complex
def: "OBSOLETE. A heterodimeric enzyme composed of an alpha and beta subunit. Catalyzes the formation of L-2-aminoadipate from L-2-aminoadipate 6-semialdehyde." [EC:1.2.1.31]
GO:0005950
name: anthranilate synthase complex
def: "A heterotetrameric enzyme complex made up of two components I and two components II. Catalyzes the formation of anthranilate, pyruvate and L-glutamate from chorismate and L-glutamine." [EC:4.1.3.27, MetaCyc:ANTHRANSYN-CPLX, PMID:4886290]
GO:0005951
name: carbamoyl-phosphate synthase complex
def: "A protein complex that catalyzes the formation of carbamoyl phosphate; comprises a small subunit that binds and cleaves glutamine, and a large subunit that accepts the ammonia group cleaved from glutamine, binds all of the remaining substrates and effectors, and carries out all of the other catalytic events." [PMID:8626695]
GO:0005952
name: cAMP-dependent protein kinase complex
def: "An enzyme complex, composed of regulatory and catalytic subunits, that catalyzes protein phosphorylation. Inactive forms of the enzyme have two regulatory chains and two catalytic chains; activation by cAMP produces two active catalytic monomers and a regulatory dimer." [EC:2.7.11.11, ISBN:0198506732]
GO:0005953
name: CAAX-protein geranylgeranyltransferase complex
def: "A heterodimeric enzyme, composed of an alpha and a beta subunit. Participates in the post-translational C-terminal modification of several small GTPases, allowing their targeting to the membrane." [PMID:9781874]
GO:0005954
name: calcium- and calmodulin-dependent protein kinase complex
def: "An enzyme complex which in eukaryotes is composed of four different chains: alpha, beta, gamma, and delta. The different isoforms assemble into homo- or heteromultimeric holoenzymes composed of 8 to 12 subunits. Catalyzes the phosphorylation of proteins to O-phosphoproteins." [EC:2.7.11.17]
GO:0005955
name: calcineurin complex
def: "A heterodimeric calcium ion and calmodulin dependent protein phosphatase composed of catalytic and regulatory subunits; the regulatory subunit is very similar in sequence to calmodulin." [ISBN:019859951]
GO:0005956
name: protein kinase CK2 complex
def: "A protein complex that possesses protein serine/threonine kinase activity, and contains two catalytic alpha subunits and two regulatory beta subunits. Protein kinase CK2 complexes are found in nearly every subcellular compartment, and can phosphorylate many protein substrates in addition to casein." [GOC:mah, PMID:10994779]
GO:0005957
name: debranching enzyme
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005958
name: DNA-dependent protein kinase-DNA ligase 4 complex
def: "A large protein complex which is involved in the repair of DNA double-strand breaks and, in mammals, V(D)J recombination events. It consists of the DNA-dependent protein kinase catalytic subunit (DNA-PKcs), the DNA end-binding heterodimer Ku, the nuclear phosphoprotein XRCC4 or a homolog thereof, and DNA ligase IV." [GOC:jl, GOC:mah, PMID:10854421, PMID:12235392, PMID:17072889]
GO:0005960
name: glycine cleavage complex
def: "A protein complex that catalyzes the reversible oxidation of glycine. In E. coli, it has four components: dihydrolipoamide dehydrogenase, glycine dehydrogenase (decarboxylating), lipoyl-GcvH-protein and aminomethyltransferase, also known as L, P, H, and T." [GOC:mah, MetaCyc:GCVMULTI-CPLX]
GO:0005962
name: mitochondrial isocitrate dehydrogenase complex (NAD+)
def: "Mitochondrial complex that possesses isocitrate dehydrogenase (NAD+) activity." [GOC:mah, GOC:mtg_sensu]
GO:0005963
name: magnesium-dependent protein serine/threonine phosphatase complex
def: "An enzyme complex that catalyzes the removal of serine- or threonine-bound phosphate groups from a wide range of phosphoproteins, including a number of enzymes that have been phosphorylated under the action of a kinase." [EC:3.1.3.16]
GO:0005964
name: phosphorylase kinase complex
def: "An enzyme complex that catalyzes the phosphorylation of phosphorylase b to form phosphorylase a." [EC:2.7.11.19]
GO:0005965
name: protein farnesyltransferase complex
def: "A protein complex that possesses protein farnesyltransferase activity." [GOC:mah]
GO:0005966
name: cyclic-nucleotide phosphodiesterase complex
def: "An enzyme complex that catalyzes the hydrolysis of bonds in a cyclic nucleotide." [EC:3.1.4.-]
GO:0005967
name: mitochondrial pyruvate dehydrogenase complex
def: "Complex that carries out the oxidative decarboxylation of pyruvate to form acetyl-CoA in eukaryotes; includes subunits possessing three catalytic activities: pyruvate dehydrogenase (E1), dihydrolipoamide S-acetyltransferase (E2), and dihydrolipoamide dehydrogenase (E3). The This Eukaryotic form usually contains more subunits than its bacterial counterpart; for example, one known complex contains 30 E1 dimers, 60 E2 monomers, and 6 E3 dimers as well as a few copies of pyruvate dehydrogenase kinase and pyruvate dehydrogenase phosphatase." [GOC:mtg_sensu, ISBN:0471331309, ISBN:0716720094]
GO:0005968
name: Rab-protein geranylgeranyltransferase complex
def: "A heterodimeric enzyme complex, which in mammals is composed of an alpha and a beta subunit, and which associates with an accessory protein Rep (Rab escort protein). Catalyzes of the transfer of a geranyl-geranyl group from geranylgeranyl pyrophosphate to a Rab protein." [GOC:jl, PMID:11886217]
GO:0005969
name: serine-pyruvate aminotransferase complex
def: "An enzyme complex that catalyzes the formation of hydroxypyruvate and alanine from serine and pyruvate." [EC:2.6.1.51]
GO:0005971
name: ribonucleoside-diphosphate reductase complex
def: "An enzyme complex composed of 2-4 or more subunits, which usually contains nonheme iron and requires ATP for catalysis. Catalyzes the formation of 2'-deoxyribonucleoside diphosphate from ribonucleoside diphosphate, using either thioredoxin disulfide or glutaredoxin disulfide as an acceptor." [BRENDA:1.17.4.1]
GO:0005972
name: fibrinogen alpha chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005973
name: fibrinogen beta chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005974
name: fibrinogen gamma chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0008002
name: lamina lucida
def: "The electron-lucent layer of the basal lamina adjacent to the basal plasma membrane of the cells that rest on the lamina." [ISBN:0815316194]
GO:0008003
name: lamina densa
def: "The electron-dense layer of the basal lamina; lies just below the lamina lucida." [ISBN:0815316194]
GO:0008004
name: lamina reticularis
def: "A layer of the basal lamina that contains collagen fibrils and connects the basal lamina to the underlying connective tissue." [ISBN:0815316194]
GO:0008008
name: membrane attack complex protein beta2 chain
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0008021
name: synaptic vesicle
def: "A secretory organelle, some 50 nm in diameter, of presynaptic nerve terminals; accumulates in high concentrations of neurotransmitters and secretes these into the synaptic cleft by fusion with the 'active zone' of the presynaptic plasma membrane." [PMID:10099709]
GO:0008023
name: transcription elongation factor complex
def: "Any protein complex that interacts with RNA polymerase II to increase (positive transcription elongation factor) or reduce (negative transcription elongation factor) the rate of transcription elongation." [GOC:jl]
GO:0008024
name: positive transcription elongation factor complex b
def: "A transcription elongation factor complex that facilitates the transition from abortive to productive elongation by phosphorylating the CTD domain of the large subunit of DNA-directed RNA polymerase II, holoenzyme. Contains cyclin T and a cyclin-dependent protein kinase catalytic subunit." [PMID:10766736, PMID:16721054, PMID:17079683, PMID:19328067, PMID:7759473]
GO:0008043
name: intracellular ferritin complex
def: "A ferritin complex located in the cell. Intracellular ferritin complexes contain 24 subunits, in a mixture of L (light) chains and H (heavy) chains." [GOC:jl, GOC:mah, PMID:19154717]
GO:0008051
name: farnesyl-diphosphate farnesyl transferase complex
def: "OBSOLETE. A complex that possesses farnesyl-diphosphate farnesyl transferase activity." [GOC:mah]
GO:0008074
name: guanylate cyclase complex, soluble
def: "Complex that possesses guanylate cyclase activity and is not bound to a membrane." [GOC:mah]
GO:0008076
name: voltage-gated potassium channel complex
def: "A protein complex that forms a transmembrane channel through which potassium ions may cross a cell membrane in response to changes in membrane potential." [GOC:mah]
GO:0008087
name: light-activated voltage-gated calcium channel complex
def: "A protein complex that forms a transmembrane channel through which calcium ions may cross a cell membrane in response to changes in membrane potential generated in response to a light stimulus." [GOC:mah, PMID:9223679]
GO:0008091
name: spectrin
def: "Membrane associated dimeric protein (240 and 220 kDa) of erythrocytes. Forms a complex with ankyrin, actin and probably other components of the membrane cytoskeleton, so that there is a mesh of proteins underlying the plasma membrane, potentially restricting the lateral mobility of integral proteins." [GOC:curators, ISBN:0815316194]
GO:0008180
name: COP9 signalosome
def: "A protein complex that catalyzes the deneddylation of proteins, including the cullin component of SCF ubiquitin E3 ligase; deneddylation increases the activity of cullin family ubiquitin ligases. The signalosome is involved in many regulatory process, including some which control development, in many species; also regulates photomorphogenesis in plants; in many species its subunits are highly similar to those of the proteasome." [PMID:11019806, PMID:12186635, PMID:14570571]
GO:0008230
name: ecdysone receptor holocomplex
def: "A heterodimeric complex containing the products of the insect genes Ecdysone receptor (EcR) and ultraspiracle (usp). Binding of ecdysone promotes association between the two subunits, and the receptor complex then initiates molting and metamorphosis by binding DNA and regulating the transcription of target genes." [GOC:bf, PMID:14592980]
GO:0008231
name: repressor ecdysone receptor complex
def: "A protein complex consisting of a heterodimer of Ecdysone receptor (EcR) and ultraspiracle (usp) plus an associated corepressor such as SMRTER, which represses transcription of target genes." [PMID:10488333]
GO:0008232
name: activator ecdysone receptor complex
def: "A protein complex consisting of a heterodimer of Ecdysone receptor (EcR) and ultraspiracle (usp) bound to the ligand ecdysone, which activates transcription of target genes." [PMID:10488333]
GO:0008245
name: lysosomal membrane hydrogen-transporting ATPase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008247
name: 1-alkyl-2-acetylglycerophosphocholine esterase complex
def: "An enzyme complex composed of two catalytic alpha subunits, which form a catalytic dimer, and a non-catalytic, regulatory beta subunit; the catalytic dimer may be an alpha1/alpha1 or alpha2/alpha2 homodimer, or an alpha1/alpha2 heterodimer. Modulates the action of platelet-activating factor (PAF)." [GOC:jl, PMID:10542206]
GO:0008250
name: oligosaccharyltransferase complex
def: "A protein complex that is found in the endoplasmic reticulum membrane of eukaryotes and transfers lipid-linked oligosaccharide precursor to asparagine residues on nascent proteins. In yeast, the complex includes at least nine different subunits, whereas in mammalian cells at least three different forms of the complex have been detected." [ISBN:0879695595, PMID:15835887]
GO:0008274
name: gamma-tubulin ring complex
def: "A multiprotein complex composed of gamma-tubulin and other non-tubulin proteins that forms a flexible open ring structure thought to be the unit of nucleation at the minus end of a microtubule." [GOC:clt, PMID:12134075]
GO:0008275
name: gamma-tubulin small complex
def: "A complex usually comprising two gamma-tubulin molecules and two conserved non-tubulin proteins. Some gamma-tubulin small complexes are thought to be the repeating unit making up the core of the gamma-tubulin ring complex." [PMID:11297925, PMID:12134075]
GO:0008278
name: cohesin complex
def: "A protein complex that is required for sister chromatid cohesion in eukaryotes. The cohesin complex forms a molecular ring complex, and is composed of structural maintenance of chromosomes (SMC) and kleisin proteins. For example, in yeast, the complex is composed of the SMC proteins Smc1p and Smc3p, and the kleisin protein Scc1p. In vertebrates, the complex is composed of the SMC1 (SMC1A or SMC1B) and SMC3 heterodimer attached via their hinge domains to a kleisin (RAD21, REC8 or RAD21L) which links them, and one STAG protein (STAG1, STAG2 or STAG3)." [GOC:sp, PMID:9887095]
GO:0008280
name: cohesin core heterodimer
def: "The core heterodimer of a cohesin complex; a structure required for sister chromatid cohesion in eukaryotes." [GOC:jl]
GO:0008282
name: ATP-sensitive potassium channel complex
def: "A protein complex that comprises four pore-forming (Kir6.x) and four regulatory sulphonylurea receptor (SURx) subunits and forms a transmembrane channel through which ions may pass. The opening and closing of the channel is regulated by ATP: binding of ATP to the Kir6.2 subunit inhibits channel activity, whereas binding of Mg2+-complexed ATP or ADP to the SUR1 subunit stimulates channel activity." [PMID:16308567, PMID:16956886]
GO:0008287
name: protein serine/threonine phosphatase complex
def: "A complex, normally consisting of a catalytic and a regulatory subunit, which catalyzes the removal of a phosphate group from a serine or threonine residue of a protein." [GOC:bf]
GO:0008290
name: F-actin capping protein complex
def: "A heterodimer consisting of alpha and beta subunits that binds to and caps the barbed ends of actin filaments, thereby regulating the polymerization of actin monomers but not severing actin filaments." [GOC:go_curators, ISBN:0198599560]
GO:0008303
name: caspase complex
def: "A protein complex that is located in the cytosol and contains one or more cysteine-type endopeptidases (also called caspases), which give the complex a peptidase activity with specificity for the hydrolysis of aspartyl bonds. These complexes may be involved e.g. in apoptotic or inflammation processes." [GOC:cna, GOC:mtg_apoptosis]
GO:0008304
name: eukaryotic translation initiation factor 4 complex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008305
name: integrin complex
def: "A protein complex that is composed of one alpha subunit and one beta subunit, both of which are members of the integrin superfamily of cell adhesion receptors; the complex spans the plasma membrane and binds to extracellular matrix ligands, cell-surface ligands, and soluble ligands." [PMID:17543136]
GO:0008328
name: ionotropic glutamate receptor complex
def: "A multimeric assembly of four or five subunits which form a structure with an extracellular N-terminus and a large loop that together form the ligand binding domain. The C-terminus is intracellular. The ionotropic glutamate receptor complex itself acts as a ligand-gated ion channel; on binding glutamate, charged ions pass through a channel in the center of the receptor complex." [http://www.bris.ac.uk/Depts/Synaptic/info/glutamate.html]
GO:0008352
name: katanin complex
def: "A complex possessing an activity that couples ATP hydrolysis to the severing of microtubules; usually a heterodimer comprising a catalytic subunit (often 60kDa) and a regulatory subunit (often 80 kDa)." [PMID:10910766]
GO:0008370
name: obsolete cellular component
def: "OBSOLETE. These are terms that have been removed from the active component ontology." [GOC:go_curators]
GO:0008385
name: IkappaB kinase complex
def: "A trimeric protein complex that phosphorylates inhibitory-kappaB (I-kappaB) proteins. The complex is composed of two kinase subunits (alpha and beta) and a regulatory gamma subunit (also called NEMO). In a resting state, NF-kappaB dimers are bound to inhibitory IKB proteins, sequestering NF-kappaB in the cytoplasm. Phosphorylation of I-kappaB targets I-kappaB for ubiquitination and proteasomal degradation, thus releasing the NF-kappaB dimers, which can translocate to the nucleus to bind DNA and regulate transcription." [GOC:bf, GOC:ma, PMID:12055104, PMID:20300203]
GO:0008537
name: proteasome activator complex
def: "A multisubunit complex that activates the hydrolysis of small nonubiquitinated peptides by binding to the proteasome core complex." [GOC:rb]
GO:0008540
name: proteasome regulatory particle, base subcomplex
def: "The subcomplex of the proteasome regulatory particle that directly associates with the proteasome core complex." [GOC:mtg_sensu, GOC:rb]
GO:0008541
name: proteasome regulatory particle, lid subcomplex
def: "The subcomplex of the proteasome regulatory particle that forms the peripheral lid, which is added on top of the base subcomplex." [GOC:rb]
GO:0008622
name: epsilon DNA polymerase complex
def: "A heterotetrameric DNA polymerase complex that catalyzes processive DNA synthesis in the absence of PCNA, but is further stimulated in the presence of PCNA. The complex contains a large catalytic subunit and three small subunits, and is best characterized in Saccharomyces, in which the subunits are named Pol2p, Dpb2p, Dpb3p, and Dpb4p. Some evidence suggests that DNA polymerase epsilon is the leading strand polymerase; it is also involved in nucleotide-excision repair and mismatch repair." [PMID:15814431, PMID:9745046]
GO:0008623
name: CHRAC
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (SNF2H in mammals, Isw2 in S. cerevisiae), an ACF1 homolog, and additional small histone fold subunits (generally two of these, but Xenopus has only one and some additional non-conserved subunits). CHRAC plays roles in the regulation of RNA polymerase II transcription and in DNA replication and repair." [GOC:bf, GOC:krc, PMID:15284901, PMID:16568949, PMID:21810179, PMID:9252192]
GO:0009274
name: peptidoglycan-based cell wall
def: "A protective structure outside the cytoplasmic membrane composed of peptidoglycan (also known as murein), a molecule made up of a glycan (sugar) backbone of repetitively alternating N-acetylglucosamine and N-acetylmuramic acid with short, attached, cross-linked peptide chains containing unusual amino acids. An example of this component is found in Escherichia coli." [GOC:mlg, ISBN:0815108893]
GO:0009275
name: Gram-positive-bacterium-type cell wall
def: "A layer of peptidoglycan found outside of the cytoplasmic membrane. The peptidoglycan is relatively thick (20-80nm) and retains the primary stain of the Gram procedure, thus cells appear blue after Gram stain. The cell walls often contain teichoic acids (acidic anionic polysaccharides) bound to the peptidoglycan. Examples of this component are found in Gram-positive bacteria." [GOC:mlg, ISBN:0051406910, ISBN:0815108893]
GO:0009276
name: Gram-negative-bacterium-type cell wall
def: "The peptidoglycan layer of the Gram-negative cell envelope. In Gram-negative cells the peptidoglycan is relatively thin (1-2nm) and is linked to the outer membrane by lipoproteins. In Gram-negative cells the peptidoglycan is too thin to retain the primary stain in the Gram staining procedure and therefore cells appear red after Gram stain." [GOC:mlg, ISBN:0815108893]
GO:0009277
name: fungal-type cell wall
def: "A rigid yet dynamic structure surrounding the plasma membrane that affords protection from stresses and contributes to cell morphogenesis, consisting of extensively cross-linked glycoproteins and carbohydrates. The glycoproteins may be modified with N- or O-linked carbohydrates, or glycosylphosphatidylinositol (GPI) anchors; the polysaccharides are primarily branched glucans, including beta-linked and alpha-linked glucans, and may also include chitin and other carbohydrate polymers, but not cellulose or pectin. Enzymes involved in cell wall biosynthesis are also found in the cell wall. Note that some forms of fungi develop a capsule outside of the cell wall under certain circumstances; this is considered a separate structure." [GOC:mcc, GOC:mtg_sensu, ISBN:3540601864, PMID:11283274, PMID:16927300, PMID:3319422]
GO:0009278
name: murein sacculus
def: "OBSOLETE. A peptidoglycan polymer that forms the shape-determining structure of the cell all of Gram-negative bacteria." [GOC:ma]
GO:0009279
name: cell outer membrane
def: "A lipid bilayer that forms the outermost membrane of the cell envelope; enriched in polysaccharide and protein; the outer leaflet of the membrane contains specific lipopolysaccharide structures." [GOC:md, GOC:mtg_sensu, ISBN:0135712254]
GO:0009280
name: cell wall inner membrane
def: "OBSOLETE. In Gram-negative bacteria the membrane that separates the cytoplasm from the murein sacculus." [GOC:ma]
GO:0009288
name: bacterial-type flagellum
def: "A motor complex composed of an extracellular helical protein filament coupled to a rotary motor embedded in the cell envelope." [GOC:cilia, GOC:jh2, GOC:krc, GOC:mtg_sensu, http:en.wikipedia.org/wiki/Flagellum#Bacterial, PMID:7787060]
GO:0009289
name: pilus
def: "A proteinaceous hair-like appendage on the surface of bacteria ranging from 2-8 nm in diameter." [GOC:pamgo_curators]
GO:0009295
name: nucleoid
def: "The region of a virus, bacterial cell, mitochondrion or chloroplast to which the nucleic acid is confined." [GOC:bm, GOC:ma, ISBN:3540076689]
GO:0009316
name: 3-isopropylmalate dehydratase complex
def: "A heterodimeric enzyme complex composed of subunits leuC and leuD. Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate." [BRENDA:4.2.1.33, GOC:jl, MetaCyc:3-ISOPROPYLMALISOM-CPLX, PMID:7026530]
GO:0009317
name: acetyl-CoA carboxylase complex
def: "A protein complex that catalyzes the first step in long-chain fatty acid biosynthesis. For example, in E. coli the complex is heterohexameric and composed of biotin carbonyl carrier protein, biotin carboxylase and the acetate CoA-transferase complex." [GOC:jl, GOC:mah, PMID:12121720]
GO:0009318
name: exodeoxyribonuclease VII complex
def: "An enzyme complex that catalyzes exonucleolytic cleavage in either 5' to 3' or 3' to 5' direction to yield nucleoside 5'-phosphates; it prefers single-stranded DNA." [EC:3.1.11.6]
GO:0009319
name: cytochrome o ubiquinol oxidase complex
def: "A protein complex that possesses cytochrome o ubiquinol oxidase activity; consists of four polypeptide subunits and associated prosthetic groups." [GOC:mah, MetaCyc:CYT-O-UBIOX-CPLX, PMID:11017202, PMID:3052268]
GO:0009320
name: phosphoribosylaminoimidazole carboxylase complex
def: "A protein complex that possesses phosphoribosylaminoimidazole carboxylase activity." [GOC:mah]
GO:0009321
name: alkyl hydroperoxide reductase complex
def: "An enzyme complex, usually a homodimer, which directly reduces cellular levels of organic hydroperoxides." [BRENDA:1.11.1.15, GOC:jl, PMID:2649484]
GO:0009322
name: trimethylamine-N-oxide reductase complex
def: "An enzyme complex that catalyzes the reduction of trimethylamine N-oxide to trimethylamine." [EC:1.6.6.9]
GO:0009323
name: ribosomal-protein-alanine N-acetyltransferase complex
def: "An enzyme complex that catalyzes the transfer of an acetyl group to ribosomal-protein alanine, forming ribosomal-protein acetylalanine." [EC:2.3.1.128]
GO:0009324
name: D-amino-acid dehydrogenase complex
def: "A protein complex that possesses D-amino-acid dehydrogenase activity." [GOC:mah]
GO:0009325
name: nitrate reductase complex
def: "An enzyme complex that catalyzes the formation of nitrate from nitrite with the concomitant reduction of an acceptor." [EC:1.7.99.4]
GO:0009326
name: formate dehydrogenase complex
def: "An enzyme complex that catalyzes the dehydrogenation of formate to produce carbon dioxide (CO2)." [EC:1.2.1.2]
GO:0009327
name: NAD(P)+ transhydrogenase complex (AB-specific)
def: "A protein complex that possesses NAD(P)+ transhydrogenase (AB-specific) activity. Homodimeric, trimeric, and heterotetrameric complexes have been identified." [BRENDA:1.6.1.2, GOC:mah]
GO:0009328
name: phenylalanine-tRNA ligase complex
def: "An enzyme complex that catalyzes the ligation of phenylalanine to tRNA(Phe), forming L-phenylalanyl-tRNA(Phe)." [EC:6.1.1.20]
GO:0009329
name: acetate CoA-transferase complex
def: "A heterotetrameric enzyme complex made up of two alpha subunits and two beta subunits. Part of the acetyl-CoA carboxylase complex. Catalyzes the transfer of a carboxyl group to form malonyl-CoA." [GOC:jl, PMID:2719476, PMID:8423010]
GO:0009330
name: DNA topoisomerase complex (ATP-hydrolyzing)
def: "Complex that possesses DNA topoisomerase (ATP-hydrolyzing) activity." [GOC:mah]
GO:0009331
name: glycerol-3-phosphate dehydrogenase complex
def: "An enzyme complex that catalyzes the dehydrogenation of sn-glycerol 3-phosphate to form glycerone phosphate." [EC:1.1.5.3]
GO:0009332
name: glutamate-tRNA ligase complex
def: "An enzyme complex that catalyzes the ligation of glutamate and tRNA(Glu) to form glutamyl-tRNA(Glu)." [EC:6.1.1.17]
GO:0009333
name: cysteine synthase complex
def: "Cysteine synthase is a multienzyme complex made up, in E. coli, of the heteromeric hexamer serine acetyltransferase and the homodimer O-acetylserine (thiol)-lyase A." [EC:4.2.99.8, MetaCyc:CYSSYNMULTI-CPLX]
GO:0009334
name: 3-phenylpropionate dioxygenase complex
def: "Enzyme complex consisting of four proteins: the two subunits of the hydroxylase component (hcaE and hcaF), a ferredoxin (hcaC) and a ferredoxin reductase (hcaD). Converts 3-phenylpropionic acid (PP) into cis-3-(3-carboxyethyl)-3,5-cyclohexadiene-1,2-diol (PP-dihydrodiol)." [GOC:jl, MetaCyc:HCAMULTI-CPLX, PMID:9603882]
GO:0009335
name: holo-[acyl-carrier protein] synthase complex
def: "OBSOLETE. An enzyme complex that catalyzes the formation of holo-[acyl-carrier protein] from CoA and apo-[acyl-carrier protein]." [EC:2.7.8.7]
GO:0009336
name: sulfate adenylyltransferase complex (ATP)
def: "An enzyme complex that catalyzes the formation adenylylsulfate from sulfate and ATP." [EC:2.7.7.4]
GO:0009337
name: sulfite reductase complex (NADPH)
def: "A multisubunit iron flavoprotein, which in yeast is composed of 2 alpha and 2 beta subunits. Catalyzes the reduction of sulfite to sulfide." [BRENDA:1.8.1.2, GOC:jl]
GO:0009338
name: exodeoxyribonuclease V complex
def: "An enzyme complex that catalyzes exonucleolytic cleavage (in the presence of ATP) in either 5' to 3' or 3' to 5' direction to yield 5'-phosphooligonucleotides. Exodeoxyribonuclease V shows a preference for double-stranded DNA and possesses DNA-dependent ATPase activity. It acts endonucleolytically on single-stranded circular DNA." [EC:3.1.11.5]
GO:0009339
name: glycolate oxidase complex
def: "An enzyme complex that catalyzes the oxidation of 2-hydroxy acid to form 2-oxo acid and hydrogen peroxide (H2O2). The enzyme is a flavoprotein (FMN)." [EC:1.1.3.15]
GO:0009340
name: DNA topoisomerase IV complex
def: "A heterodimeric enzyme, which in most bacterial species is composed of two subunits, ParC and ParE. Functions in chromosome segregation and can relax supercoiled DNA." [GOC:jl, PMID:7783632]
GO:0009341
name: beta-galactosidase complex
def: "A protein complex that possesses beta-galactosidase activity, i.e. catalyzes the hydrolysis of terminal non-reducing beta-D-galactose residues in beta-D-galactosides. In E. coli, the complex is a homotetramer; dimeric and hexameric beta-galactosidase complexes have been observed in other species." [BRENDA:3.2.1.2, EC:3.2.1.23, PMID:15950161]
GO:0009342
name: glutamate synthase complex (NADPH)
def: "A complex that possesses glutamate synthase (NADPH) activity." [EC:1.4.1.13, GOC:mah]
GO:0009343
name: biotin carboxylase complex
def: "An enzyme complex that catalyzes the formation of carboxybiotin-carboxyl-carrier protein from biotin-carboxyl-carrier protein and carbon dioxide (CO2)." [EC:6.3.4.14]
GO:0009344
name: nitrite reductase complex [NAD(P)H]
def: "Complex that possesses nitrite reductase [NAD(P)H] activity." [GOC:mah]
GO:0009345
name: glycine-tRNA ligase complex
def: "A multimeric enzyme complex which, in bacteria, is usually a tetramer of two alpha and two beta chains and in eukaryotes, is usually a homodimer. Functions in the ligation of glycine and tRNA(Gly) to form glycyl-tRNA(Gly)." [EC:6.1.1.14, GOC:jl, PMID:15733854]
GO:0009346
name: citrate lyase complex
def: "Citrate lyase is a multienzyme complex with three constituents: the alpha subunit, citrate-ACP transferase; the beta subunit, citryl-ACP lyase; and the gamma subunit, an acyl-carrier protein which also carries the prosthetic group components. All three subunits are required for citrate lyase enzyme activity." [EC:4.1.3.6, MetaCyc:ACECITLY-CPLX]
GO:0009347
name: aspartate carbamoyltransferase complex
def: "A multienzyme complex that catalyzes the formation N-carbamoyl-L-aspartate from carbamoyl phosphate and L-aspartate. It exhibits a variety of architectural organizations, but in all microorganisms the core catalytic component is a homotrimer of approximately 34 kDa polypeptides." [PMID:10447693]
GO:0009348
name: ornithine carbamoyltransferase complex
def: "A homotrimeric protein complex that catalyzes the transfer of a carbamoyl group to ornithine, forming citrulline." [EC:2.1.3.3, GOC:mah]
GO:0009349
name: riboflavin synthase complex
def: "An flavoprotein that catalyzes the reaction the breakdown of dimethyl(ribityl)lumazine to form riboflavin and ribitylamino-amino-dihydroxypyrimidine." [EC:2.5.1.9]
GO:0009350
name: ethanolamine ammonia-lyase complex
def: "An enzyme complex that catalyzes the breakdown of ethanolamine to form acetaldehyde and ammonia." [EC:4.3.1.7]
GO:0009351
name: dihydrolipoamide S-acyltransferase complex
def: "OBSOLETE. An enzyme complex that catalyzes the transfer of an acyl group from coenzyme A to dihydrolipoamide." [EC:2.3.1.12]
GO:0009352
name: dihydrolipoyl dehydrogenase complex
def: "OBSOLETE. Complex that possesses dihydrolipoyl dehydrogenase activity." [GOC:mah]
GO:0009353
name: mitochondrial oxoglutarate dehydrogenase complex
def: "A complex of multiple copies of three enzymatic components: oxoglutarate dehydrogenase (lipoamide) ; EC:1.2.4.2 (E1), dihydrolipoamide S-succinyltransferase ; EC:2.3.1.61 (E2) and dihydrolipoamide dehydrogenase ; EC:1.8.1.4 (E3); catalyzes the overall conversion of 2-oxoglutarate to succinyl-CoA and carbon dioxide (CO2) within the mitochondrial matrix. An example of this complex is found in Mus musculus." [GOC:mtg_sensu, MetaCyc:CPLX66-42, PMID:10848975]
GO:0009354
name: dihydrolipoamide S-succinyltransferase complex
def: "OBSOLETE. An enzyme complex that catalyzes the transfer of succinyl-CoA to dihydrolipoamide to form S-succinyldihydrolipoamide. The enzyme is a component of the multienzyme 2-oxoglutarate dehydrogenase complex." [EC:2.3.1.61]
GO:0009355
name: DNA polymerase V complex
def: "A DNA polymerase complex that contains two UmuD' and one UmuC subunits, and acts in translesion DNA synthesis." [PMID:10430871, PMID:10542196]
GO:0009356
name: aminodeoxychorismate synthase complex
def: "A heterodimeric protein complex that possesses 4-amino-4-deoxychorismate synthase activity." [PMID:2251281, PMID:7592344]
GO:0009357
name: protein-N(PI)-phosphohistidine-sugar phosphotransferase complex
def: "An enzyme complex that catalyzes the transfer of a phosphate from protein N(PI)-phosphohistidine to a sugar molecule. It is enzyme II of the phosphotransferase system." [EC:2.7.1.69]
GO:0009358
name: polyphosphate kinase complex
def: "A protein complex that possesses polyphosphate kinase activity." [GOC:mah]
GO:0009359
name: Type II site-specific deoxyribonuclease complex
def: "A protein complex that functions as an endonuclease to cleave DNA at or near a specific recognition site, when that site is unmethylated. These complexes may be dimers or tetramers; it is also possible for the endonuclease to be in a complex with the corresponding methyltransferase that methylates the recognition site. DNA restriction systems such as this are used by bacteria to defend against phage and other foreign DNA that may enter a cell." [PMID:12654995]
GO:0009360
name: DNA polymerase III complex
def: "The DNA polymerase III holoenzyme is a complex that contains 10 different types of subunits. These subunits are organized into 3 functionally essential sub-assemblies: the pol III core, the beta sliding clamp processivity factor and the clamp-loading complex. The pol III core carries out the polymerase and the 3'-5' exonuclease proofreading activities. The polymerase is tethered to the template via the sliding clamp processivity factor. The clamp-loading complex assembles the beta processivity factor onto the primer template and plays a central role in the organization and communication at the replication fork." [PMID:11525729, PMID:12940977, UniProt:P06710]
GO:0009361
name: succinate-CoA ligase complex (ADP-forming)
def: "A heterodimeric enzyme complex, composed of an alpha and beta chain, most usually found in (but not limited to) bacteria. Functions in the TCA cycle, hydrolyzing succinyl-CoA into succinate and CoA, thereby forming ATP." [EC:6.2.1.5, GOC:jl]
GO:0009365
name: protein histidine kinase complex
def: "A complex that possesses protein histidine kinase activity." [GOC:mah]
GO:0009366
name: enterobactin synthetase complex
def: "A multienzyme complex usually composed of four proteins, EntB, EntD, EntE and EntF. Plays a role in the enterobactin biosynthesis pathway." [MetaCyc:ENTMULTI-CPLX, PMID:9485415]
GO:0009367
name: prepilin peptidase complex
def: "OBSOLETE. An enzyme complex that catalyzes the cleavage of a Gly-Phe bond to release an N-terminal, basic peptide of 5-8 residues from type IV prepilin, and then N-methylates the new N-terminal amino group." [EC:3.4.23.43]
GO:0009368
name: endopeptidase Clp complex
def: "A protein complex comprised of members of the ClpX, ClpC, ClpD, ClpP or ClpR protein families. ClpPs are the proteolytic subunit of active complexes, and ClpA and ClpX form the regulatory subunits. Enzymatically active and inactive complexes can form." [GOC:mah, PMID:11352464]
GO:0009375
name: ferredoxin hydrogenase complex
def: "An enzyme complex that catalyzes the oxidation of reduced ferredoxin. Hydrogenase contains iron-sulfur clusters, and some contain nickel; it can use molecular hydrogen for the reduction of a variety of substances." [EC:1.12.7.2]
GO:0009376
name: HslUV protease complex
def: "A protein complex that possesses ATP-dependent protease activity; consists of an ATPase large subunit with homology to other ClpX family ATPases and a peptidase small subunit related to the proteasomal beta-subunits of eukaryotes. In the E. coli complex, a double ring-shaped homohexamer of HslV is capped on each side by a ring-shaped HslU homohexamer." [GOC:bhm, PMID:12670962, UniProt:P0A6H5]
GO:0009379
name: Holliday junction helicase complex
def: "A DNA helicase complex that forms part of a Holliday junction resolvase complex, where the helicase activity is involved in the migration of the junction branch point. The best-characterized example is the E. coli RuvAB complex, in which a hexamer of RuvB subunits possesses helicase activity that is modulated by association with RuvA." [PMID:16935884, PMID:9442895]
GO:0009380
name: excinuclease repair complex
def: "Any of the protein complexes formed by the UvrABC excinuclease system, which carries out nucleotide excision repair. Three different complexes are formed by the 3 proteins as they proceed through the excision repair process. First a complex consisting of two A subunits and two B subunits bind DNA and unwind it around the damaged site. Then, the A subunits disassociate leaving behind a stable complex between B subunits and DNA. Now, subunit C binds to this B+DNA complex and causes subunit B to nick the DNA on one side of the complex while subunit C nicks the DNA on the other side of the complex. DNA polymerase I and DNA ligase can then repair the resulting gap." [GOC:mah, GOC:mlg, PMID:12145219, PMID:15192705]
GO:0009382
name: imidazoleglycerol-phosphate synthase complex
def: "Complex that possesses imidazoleglycerol-phosphate synthase activity." [GOC:mah]
GO:0009390
name: dimethyl sulfoxide reductase complex
def: "An enzyme complex that catalyzes the formation of dimethyl sulfide from dimethyl sulfoxide." [UM-BBD_enzymeID:e0188]
GO:0009417
name: fimbrin
def: "OBSOLETE. A class of proteins that are the subunit components of fimbria." [ISBN:0914826859]
GO:0009418
name: pilus shaft
def: "The long, slender, mid section of a pilus." [GOC:jl]
GO:0009419
name: pilus tip
def: "The pointed extremity furthest from the cell of a pilus." [GOC:jl]
GO:0009420
name: bacterial-type flagellum filament
def: "The long (approximately 20 nm), thin external structure of the flagellum, which acts as a propeller. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009421
name: bacterial-type flagellum filament cap
def: "The proteinaceous structure at the distal tip of the flagellar filament. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009422
name: bacterial-type flagellum hook-filament junction
def: "The region of the flagellum where the hook and filament meet. Examples of this component are found in bacteria." [GOC:mah, GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009424
name: bacterial-type flagellum hook
def: "The portion of the flagellum that connects the filament to the basal body. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009425
name: bacterial-type flagellum basal body
def: "One of the three major substructures of the flagellin-based flagellum; a structure consisting of a rod, a series of rings, the Mot proteins, the switch complex and the flagellum-specific export apparatus. The rings anchor the flagellum to the cytoplasmic membrane (MS ring), the peptidoglycan (P ring) and the outer membrane (L ring). Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009426
name: bacterial-type flagellum basal body, distal rod
def: "The portion of the central rod of the flagellar basal body that is distal to the cell membrane; spans most of the distance between the inner and outer membranes. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:11133968, PMID:12624192]
GO:0009427
name: bacterial-type flagellum basal body, distal rod, L ring
def: "One of the rings of the flagellar basal body; anchors the basal body to the outer membrane. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009428
name: bacterial-type flagellum basal body, distal rod, P ring
def: "One of the rings of the flagellar basal body; anchors the basal body to the peptidoglycan layer. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009429
name: bacterial-type flagellum basal body, proximal rod
def: "The portion of the central rod of the flagellar basal body that is proximal to the cell membrane; proximal rod connects the distal rod to the flagellar motor. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:11133968, PMID:12624192]
GO:0009431
name: bacterial-type flagellum basal body, MS ring
def: "One of the rings of the flagellar basal body; a double-flanged ring that anchors the basal body to the cytoplasmic membrane. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009433
name: bacterial-type flagellum basal body, C ring
def: "Cytoplasmic ring located at the base of the flagellar basal body; acts as a rotor; includes three switch proteins, which generate torque and can change their conformational state in a bimodal fashion, so that the motor direction can switch between clockwise and counterclockwise. Examples of this component are found in bacteria." [GOC:mtg_sensu, PMID:10572114, PMID:12624192]
GO:0009501
name: amyloplast
def: "A plastid whose main function is to synthesize and store starch." [ISBN:0140514031]
GO:0009502
name: photosynthetic electron transport chain
def: "OBSOLETE. A series of membrane-linked oxidation-reduction reactions in which electrons are transferred from an initial electron donor through a series of intermediates to a final electron acceptor (usually oxygen)." [GOC:mtg_electron_transport, ISBN:014051403]
GO:0009503
name: thylakoid light-harvesting complex
def: "A thylakoid membrane complex of chlorophylls a and b together with chlorophyll a-b binding proteins. In addition, LHCs contain a number of other proteins, the function of which is speculative, together with accessory pigments. The LHCs capture and transfer energy to photosystems I and II. An example of this is found in Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:0198547684]
GO:0009504
name: cell plate
def: "The nascent cell membrane and cell wall structure that forms between two daughter nuclei near the center of a dividing plant cell. It develops at the equitorial region of the phragmoplast. It grows outwards to join with the lateral walls and form two daughter cells." [ISBN:0198547684]
GO:0009505
name: plant-type cell wall
def: "A more or less rigid stucture lying outside the cell membrane of a cell and composed of cellulose and pectin and other organic and inorganic substances." [ISBN:0471245208]
GO:0009506
name: plasmodesma
def: "A fine cytoplasmic channel, found in all higher plants, that connects the cytoplasm of one cell to that of an adjacent cell." [ISBN:0198506732]
GO:0009507
name: chloroplast
def: "A chlorophyll-containing plastid with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma." [ISBN:0471245208]
GO:0009508
name: plastid chromosome
def: "A circular DNA molecule containing plastid encoded genes." [ISBN:0943883999]
GO:0009509
name: chromoplast
def: "A plastid containing pigments other than chlorophyll, usually yellow and orange carotenoid pigments." [ISBN:0471245208]
GO:0009510
name: plasmodesmatal desmotubule
def: "A tightly wound cylinder of membrane that is located within the plasmodesmal pore and runs the length of the plasmodesma. The desmotubule likely provides a rigid stability to plasmodesmata and confers a fixed diameter and pore size to the plasmodesmal canal, and is linked to the endoplasmic reticulum in each of the adjacent cell." [http://biology.kenyon.edu/edwards/project/greg/pd.htm]
GO:0009511
name: plasmodesmatal endoplasmic reticulum
def: "Endoplasmic reticulum found in plasmodesmata, junctions connecting the cytoplasm of adjacent plant cells." [GOC:ai]
GO:0009512
name: cytochrome b6f complex
def: "Complex that transfers electrons from reduced plastoquinone to oxidized plastocyanin and translocates protons from the stroma to the lumen. The complex contains a core structure of three catalytic subunits: cytochrome b, the Rieske iron sulfur protein (ISP), and cytochrome f, which are arranged in an integral membrane-bound dimeric complex; additional subunits are present, and vary among different species." [ISBN:0943883999, PMID:16228398, PMID:16352458]
GO:0009513
name: etioplast
def: "A plastid arrested in the development of chloroplasts from proplastids due to absence of light or low light conditions." [ISBN:0943883999]
GO:0009514
name: glyoxysome
def: "A specialized form of peroxisome that contains the enzymes of the glyoxylate pathway. The glyoxysome is found in some plant cells, notably the cells of germinating seeds." [GOC:dhl, ISBN:0140514031]
GO:0009515
name: granal stacked thylakoid
def: "Appressed thylakoid membranes that are part of a granum (stacked regions). A characteristic of these appressed regions is the preferential localization of photosystem II." [GOC:lr]
GO:0009516
name: leucoplast
def: "A colorless plastid involved in the synthesis of monoterpenes." [ISBN:0943883999]
GO:0009517
name: PSII associated light-harvesting complex II
def: "Protein-pigment complex associated with photosystem II." [GOC:lr, ISBN:0582227089]
GO:0009518
name: PSI associated light-harvesting complex I
def: "Protein-pigment complex associated with photosystem I." [GOC:lr]
GO:0009519
name: middle lamella
def: "Layer of intercellular material, chiefly pectic substances, cementing together the primary walls of contiguous cells." [ISBN:0471245208]
GO:0009521
name: photosystem
def: "A complex located in a photosynthetic membrane that consists of a photoreaction center associated with accessory pigments and electron carriers. Examples of this component are found in Arabidopsis thaliana and in photosynthetic bacterial and archaeal species." [GOC:ds, GOC:mah, ISBN:0140514031, PMID:9821949]
GO:0009522
name: photosystem I
def: "A photosystem that contains an iron-sulfur reaction center associated with accessory pigments and electron carriers. In cyanobacteria and chloroplasts, photosystem I functions as a light-dependent plastocyanin-ferredoxin oxidoreductase, transferring electrons from plastocyanin to ferredoxin; in photosynthetic bacteria that have only a single type I photosystem, such as the green sulfur bacteria, electrons can go either to ferredoxin (Fd) -> NAD+ or to menaquinone (MK) -> Cytb/FeS -> Cytc555 -> photosystem I (cyclic photophosphorylation)." [GOC:ds, GOC:mah, ISBN:0140514031, PMID:9821949]
GO:0009523
name: photosystem II
def: "A photosystem that contains a pheophytin-quinone reaction center with associated accessory pigments and electron carriers. In cyanobacteria and chloroplasts, in the presence of light, PSII functions as a water-plastoquinone oxidoreductase, transferring electrons from water to plastoquinone, whereas other photosynthetic bacteria carry out anoxygenic photosynthesis and oxidize other compounds to re-reduce the photoreaction center." [GOC:ds, GOC:mah, ISBN:0943088399, PMID:9821949]
GO:0009524
name: phragmoplast
def: "Fibrous structure (light microscope view) that arises between the daughter nuclei at telophase and within which the initial partition (cell plate), dividing the mother cell in two (cytokinesis), is formed. Appears at first as a spindle connected to the two nuclei, but later spreads laterally in the form of a ring. Consists of microtubules." [ISBN:0471245208]
GO:0009525
name: phragmosome
def: "A flattened membranous vesicle containing cell wall components." [ISBN:0943883999]
GO:0009526
name: plastid envelope
def: "The double lipid bilayer enclosing a plastid and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:jy]
GO:0009527
name: plastid outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the plastid envelope." [GOC:lr]
GO:0009528
name: plastid inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the plastid envelope; also faces the plastid stroma." [GOC:lr]
GO:0009529
name: plastid intermembrane space
def: "The region between the inner and outer lipid bilayers of the plastid envelope." [GOC:lr]
GO:0009530
name: primary cell wall
def: "A plant cell wall that is still able to expand, permitting cell growth. Primary cell walls contain more pectin than secondary walls and no lignin is present." [GOC:jid, PMID:9442872]
GO:0009531
name: secondary cell wall
def: "A plant cell wall that is no longer able to expand and so does not permit growth. Secondary cell walls contain less pectin that primary cell walls. The secondary cell is mostly composed of cellulose and is strengthened with lignin." [GOC:jid, ISBN:0943088399]
GO:0009532
name: plastid stroma
def: "The proteinaceous ground substance of plastids." [ISBN:047142520]
GO:0009533
name: chloroplast stromal thylakoid
def: "Unstacked thylakoids that connect the grana stacks through the stroma." [ISBN:0943883999]
GO:0009534
name: chloroplast thylakoid
def: "Sac-like membranous structures (cisternae) in a chloroplast combined into stacks (grana) and present singly in the stroma (stroma thylakoids or frets) as interconnections between grana. An example of this component is found in Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:0943883999]
GO:0009535
name: chloroplast thylakoid membrane
def: "The pigmented membrane of a chloroplast thylakoid. An example of this component is found in Arabidopsis thaliana." [GOC:lr, GOC:mtg_sensu]
GO:0009536
name: plastid
def: "Any member of a family of organelles found in the cytoplasm of plants and some protists, which are membrane-bounded and contain DNA. Plant plastids develop from a common type, the proplastid." [GOC:jl, ISBN:0198547684]
GO:0009537
name: proplastid
def: "The precursor of other plastids." [ISBN:0943883999]
GO:0009538
name: photosystem I reaction center
def: "A photochemical system containing P700, the chlorophyll a dimer that functions as a primary electron donor. Functioning as a light-dependent plastocyanin-ferredoxin oxidoreductase, it transfers electrons from plastocyanin to ferredoxin." [GOC:kd, ISBN:0943088399]
GO:0009539
name: photosystem II reaction center
def: "An integral membrane complex containing P680, the chlorophyll a molecule that functions as a primary electron donor. In the light, functioning as a water-plastoquinone oxidoreductase, it transfers electrons from water to plastoquinone." [GOC:kd, ISBN:0943088399]
GO:0009541
name: etioplast prolamellar body
def: "A three dimensional regular lattice found in etioplasts. It is composed of a continuous system of tubules but when exposed to light the symmetrical arrangement is rapidly lost as tubules become pinched off into two dimensional sections of lattice. These for perforated sheets of membrane that move apart, extend and increase, finally establishing the typical granal and intergranal lamellae of the mature chloroplast." [ISBN:0140514031]
GO:0009542
name: granum
def: "Distinct stack of lamellae seen within chloroplasts. Grana contain the pigments, electron transfer compounds, and enzymes essential to the light-dependent reactions of photosynthesis." [ISBN:014051403]
GO:0009543
name: chloroplast thylakoid lumen
def: "The cavity enclosed within the chloroplast thylakoid membrane. An example of this component is found in Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:006501037]
GO:0009544
name: chloroplast ATP synthase complex
def: "The protein complex that catalyzes the phosphorylation of ADP to ATP in chloroplasts." [ISBN:019854768]
GO:0009545
name: elaioplast
def: "A leucoplast in which oil is stored." [ISBN:0140514031]
GO:0009546
name: plasmodesmatal cytoplasmic sleeve
def: "The space between the plasma membrane and the desmotubule of a plasmodesma." [http://biology.kenyon.edu/edwards/project/greg/pd.htm]
GO:0009547
name: plastid ribosome
def: "A ribosome contained within a plastid." [GOC:tair_curators]
GO:0009548
name: plasmodesmatal plasma membrane
def: "The portion of the plasma membrane surrounding a plasmodesma." [GOC:mah]
GO:0009549
name: cellulose microfibril
def: "A microfibril composed of cellulose arranged in orthogonal layers. Cellulose is a straight chain polysaccharide composed of B(14) linked glucose subunits. It is a major component of plant cell walls. Higher plant microfibrils are about 10nm in diameter and extremely long in relation to their width. The cellulose molecules are oriented parallel to the long axis of the microfibril in a paracrystalline array, which provides great tensile strength. The microfibrils are held in place by the wall matrix and their orientation is closely controlled by the cell." [GOC:jid, ISBN:0943088399]
GO:0009550
name: primary plasmodesma
def: "A plasmodesma that consists of a simple, single channel; found predominantly in young tissue and formed as a function of cell plate formation during cytokinesis." [PMID:15012255]
GO:0009551
name: secondary plasmodesma
def: "A plasmodesma with a branched structure, often with many channels leading into a larger central cavity; found in older tissues and usually derived from preexisting primary plasmodesmata." [PMID:15012255]
GO:0009568
name: amyloplast starch grain
def: "Plant storage body for amylose and amylopectin, 1-100um in diameter, and located in amyloplasts. Also contains small amounts of enzymes, amino acids, lipids and nucleic acids. The shape of the grain varies widely amongst species, but is often spherical or disk-shaped." [GOC:jl, PMID:11217978]
GO:0009569
name: chloroplast starch grain
def: "Plant storage body for amylose and amylopectin, 1-100um in diameter, and located in chloroplasts. Also contains small amounts of enzymes, amino acids, lipids and nucleic acids. The shape of the grain varies widely amongst species, but is often spherical or disk-shaped." [GOC:jl, ISBN:0198506732]
GO:0009570
name: chloroplast stroma
def: "The space enclosed by the double membrane of a chloroplast but excluding the thylakoid space. It contains DNA, ribosomes and some temporary products of photosynthesis." [ISBN:0198547684]
GO:0009571
name: proplastid stroma
def: "The space enclosed by the double membrane of a proplastid." [GOC:jl]
GO:0009573
name: chloroplast ribulose bisphosphate carboxylase complex
def: "A complex, located in the chloroplast, containing either both large and small subunits or just small subunits which carries out the activity of producing 3-phosphoglycerate from carbon dioxide and ribulose-1,5-bisphosphate. An example of this component is found in Arabidopsis thaliana." [GOC:mlg, GOC:mtg_sensu]
GO:0009574
name: preprophase band
def: "A dense band of microtubules, 1-3 pm wide, that appears just beneath the cell membrane before the start of cell division in the cells of higher plants. It precedes the onset of prophase and then disappears as mitosis begins, yet it somehow determines the plane of orientation of the new cell plate forming in late telophase and marks the zone of the parental cell wall where fusion with the growing cell plate ultimately occurs." [ISBN:0198506732]
GO:0009575
name: chromoplast stroma
def: "The space enclosed by the double membrane of a chromoplast but excluding the photosynthetic material." [GOC:jl]
GO:0009576
name: leucoplast stroma
def: "The space enclosed by the double membrane of a leucoplast." [GOC:mah]
GO:0009577
name: elaioplast stroma
def: "The space enclosed by the double membrane of an elaioplast." [GOC:mah]
GO:0009578
name: etioplast stroma
def: "The space enclosed by the double membrane of an etioplast but excluding the prothylakoid space. It contains the etioplast DNA." [GOC:jl]
GO:0009579
name: thylakoid
def: "A membranous cellular structure that bears the photosynthetic pigments in plants, algae, and cyanobacteria. In cyanobacteria thylakoids are of various shapes and are attached to, or continuous with, the plasma membrane. In eukaryotes they are flattened, membrane-bounded disk-like structures located in the chloroplasts; in the chloroplasts of higher plants the thylakoids form dense stacks called grana. Isolated thylakoid preparations can carry out photosynthetic electron transport and the associated phosphorylation." [GOC:ds, GOC:mtg_sensu, ISBN:0198506732]
GO:0009580
name: thylakoid (sensu Bacteria)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0009654
name: photosystem II oxygen evolving complex
def: "A complex, composed of a cluster of manganese, calcium and chloride ions bound to extrinsic proteins, that catalyzes the splitting of water to O2 and 4 H+. In cyanobacteria there are five extrinsic proteins in OEC (PsbO, PsbP-like, PsbQ-like, PsbU and PsbV), while in plants there are only three (PsbO, PsbP and PsbQ)." [GOC:cjm, InterPro:IPR002683]
GO:0009655
name: PSII associated light-harvesting complex II, core complex
def: "The pigment-protein complex primarily associated to PSII in higher plants, green algae and cyanobacteria that directly transfers electrons to the reaction center." [GOC:lr]
GO:0009656
name: PSII associated light-harvesting complex II, peripheral complex
def: "Pigment-protein complex primarily associated to PSII in plants, green algae and cyanobacteria. Involved in state transitions that cause migration to PSI under certain environmental conditions such as high light." [GOC:lr]
GO:0009705
name: plant-type vacuole membrane
def: "The lipid bilayer surrounding a vacuole that retains the same shape regardless of cell cycle phase. The membrane separates its contents from the cytoplasm of the cell. An example of this component is found in Arabidopsis thaliana." [GOC:mtg_sensu, ISBN:0471245208]
GO:0009706
name: chloroplast inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the chloroplast envelope; also faces the chloroplast stroma." [GOC:tb]
GO:0009707
name: chloroplast outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the chloroplast envelope." [GOC:tb]
GO:0009782
name: photosystem I antenna complex
def: "The antenna complex of photosystem I. A photosystem has two closely linked components, an antenna containing light-absorbing pigments and a reaction center. Each antenna contains one or more light-harvesting complexes (LHCs)." [GOC:jid, ISBN:0716731363]
GO:0009783
name: photosystem II antenna complex
def: "The antenna complex of photosystem II. A photosystem has two closely linked components, an antenna containing light-absorbing pigments and a reaction center. Each antenna contains one or more light-harvesting complexes (LHCs)." [GOC:jid, ISBN:0716731363]
GO:0009839
name: SCF complex substrate recognition subunit
def: "OBSOLETE. The portion of the SCF ubiquitin ligase complex that contains sites required for recognition (and recruitment) of the substrate to the complex." [PMID:11790542, PMID:9857172]
GO:0009840
name: chloroplastic endopeptidase Clp complex
def: "A Clp endopeptidase complex located in the chloroplast." [GOC:mah]
GO:0009841
name: mitochondrial endopeptidase Clp complex
def: "A Clp endopeptidase complex located in the mitochondrion." [GOC:mah]
GO:0009842
name: cyanelle
def: "A plastid that contains unstacked, phycobilisome-bearing thylakoid membranes and is surrounded by a double membrane with a peptidoglycan layer in the intermembrane space between the two envelope membranes. Cyanelles are characteristic of algae in the class Glaucophyta, and may represent an ancestral form of plastid." [ISBN:052131687, ISBN:1402001894]
GO:0009843
name: cyanelle thylakoid
def: "A thylakoid found in a cyanelle, which is a type of plastid found in certain algae. The cyanelle contains a photosynthetic membrane resembling that of cyanobacteria." [GOC:lr, GOC:mah, GOC:mtg_sensu]
GO:0009897
name: external side of plasma membrane
def: "The leaflet of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
GO:0009898
name: cytoplasmic side of plasma membrane
def: "The leaflet the plasma membrane that faces the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:dos, GOC:tb]
GO:0009921
name: auxin efflux carrier complex
def: "The protein complex associated with the plasma membrane of certain plant cells (e.g. root cortex, epidermal cells) that functions to transport auxin out of the cell." [PMID:9843496]
GO:0009923
name: fatty acid elongase complex
def: "A tetrameric complex of four different subunits which catalyzes the elongation of fatty acids chains 2 carbon units at a time in the synthesis of very long chain fatty acids." [GOC:tb]
GO:0009925
name: basal plasma membrane
def: "The region of the plasma membrane located at the basal end of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:go_curators]
GO:0009930
name: longitudinal side of cell surface
def: "The side of the cell parallel to the zygotic axis." [GOC:mtg_sensu, GOC:sm]
GO:0009941
name: chloroplast envelope
def: "The double lipid bilayer enclosing the chloroplast and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:tb]
GO:0009986
name: cell surface
def: "The external part of the cell wall and/or plasma membrane." [GOC:jl, GOC:mtg_sensu, GOC:sm]
GO:0010005
name: cortical microtubule, transverse to long axis
def: "Arrays of microtubules underlying and connected to the plasma membrane, in the cortical cytosol, oriented mainly with their axes transverse to the long axis of the cell (and root in plants). In plants it influences the direction of cellulose microfibril deposition." [ISBN:0943088399]
GO:0010006
name: Toc complex
def: "Protein translocon complex at the chloroplast outer membrane." [PMID:10646606]
GO:0010007
name: magnesium chelatase complex
def: "A heterotrimeric enzyme complex composed of three subunits, all of which are required for enzyme activity, which catalyzes the chelation of Mg by proto IX in an ATP-dependent manner." [PMID:11842180]
GO:0010008
name: endosome membrane
def: "The lipid bilayer surrounding an endosome." [GOC:mah]
GO:0010009
name: cytoplasmic side of endosome membrane
def: "The side (leaflet) of the endosome membrane that faces the cytoplasm." [GOC:lr]
GO:0010168
name: ER body
def: "A novel compartment found in plant cells that is derived from the ER. The structures have a characteristic shape and size (10 mm long and 0.5 mm wide) and are surrounded with ribosomes. They have been found in Arabidopsis thaliana and related Brassicaceae species." [PMID:11577182]
GO:0010169
name: thioglucosidase complex
def: "A large (200-800 kDa) multiprotein complex formed by 70-kDa and 5-kDa myrosinases, myrosinase- binding proteins (MBPs), MBP-related proteins and myrosinase-associated proteins. The complex has been identified in Brassica napus seeds." [PMID:10682349]
GO:0010170
name: glucose-1-phosphate adenylyltransferase complex
def: "Complex that catalyzes the synthesis of ADP-glucose and pyrophosphate from glucose-1-phosphate and ATP. In plants, the complex is a heterotetramer composed of two types of subunits (small and large). In bacteria, the enzyme complex is composed of four identical subunits." [GOC:tb, PMID:12748181]
GO:0010240
name: plastid pyruvate dehydrogenase complex
def: "Complex that carries out the oxidative decarboxylation of pyruvate to form acetyl-CoA; comprises subunits possessing three catalytic activities: pyruvate dehydrogenase (E1), dihydrolipoamide S-acetyltransferase (E2), and dihydrolipoamide dehydrogenase (E3). This complex is found in plant plastids and is distinct from the one found in mitochondria." [GOC:mtg_sensu, PMID:9393637]
GO:0010278
name: chloroplast outer membrane translocon
def: "The protein transport machinery of the chloroplast outer membrane that contains at least three components Toc159, Toc75 and Toc34, interacts with precursor proteins which are imported into the chloroplast in a GTP dependant manner." [PMID:11299338]
GO:0010282
name: senescence-associated vacuole
def: "A lytic vacuole that is maintained at acidic pH and has different tonoplast composition compared to the central vacuole. Found during leaf senescence and develops in the peripheral cytoplasm of cells that contain chloroplast." [PMID:15743448]
GO:0010287
name: plastoglobule
def: "A lipoprotein particle present in chloroplasts. They are rich in non-polar lipids (triglycerides, esters) as well as in prenylquinones, plastoquinone and tocopherols. Plastoglobules are often associated with thylakoid membranes, suggesting an exchange of lipids with thylakoids." [GOC:tair_curators, PMID:16461379]
GO:0010316
name: pyrophosphate-dependent phosphofructokinase complex
def: "Heterodimeric complex that catalyzes the pyrophosphate-dependent phosphorylation of D-fructose 6-phosphate into D-fructose 1,6-bisphosphate." [PMID:2170409]
GO:0010317
name: pyrophosphate-dependent phosphofructokinase complex, alpha-subunit complex
def: "Refers to the alpha subunit of the heterodimeric complex that possesses pyrophosphate-dependent phosphofructokinase activity." [PMID:2170409]
GO:0010318
name: pyrophosphate-dependent phosphofructokinase complex, beta-subunit complex
def: "Refers to the beta subunit of the heterodimeric complex that possesses pyrophosphate-dependent phosphofructokinase activity." [PMID:2170409]
GO:0010319
name: stromule
def: "Thin filamentous structure extending from the surface of all plastid types examined so far, including chloroplast, proplastid, etioplast, leucoplast, amyloplast, and chromoplast. In general, stromules are more abundant in tissues containing non-green plastids, and in cells containing smaller plastids. The primary function of stromules is still unresolved, although the presence of stromules markedly increases the plastid surface area, potentially increasing transport to and from the cytosol. Other functions of stromules, such as transfer of macromolecules between plastids and starch granule formation in cereal endosperm, may be restricted to particular tissues and cell types." [PMID:15272881, PMID:15699062, PMID:16582010]
GO:0010330
name: cellulose synthase complex
def: "A large, multimeric protein complex, organized in a rosette, which catalyzes the biosynthesis of cellulose for the plant cell wall." [PMID:12514238, PMID:18485800, PMID:21307367]
GO:0010339
name: external side of cell wall
def: "The side of the cell wall that is opposite to the side that faces the cell and its contents." [GOC:mtg_sensu, GOC:tb]
GO:0010367
name: extracellular isoamylase complex
def: "A protein complex whose composition varies amongst species; in rice it probably exists in a homo-tetramer to homo-hexamer form and in Gram negative bacteria as a dimer. Functions in the hydrolysis of alpha-(1,6)-D-glucosidic branch linkages. Isoamylases in animals are localized in the extracellular space." [GOC:tair_curators]
GO:0010368
name: chloroplast isoamylase complex
def: "A protein complex whose composition varies amongst species; in rice it probably exists in a homo-tetramer to homo-hexamer form and in Gram negative bacteria as a dimer. Functions in the hydrolysis of alpha-(1,6)-D-glucosidic branch linkages. Isoamylases in plants are intracellular and probably chloroplast localized." [GOC:tair_curators]
GO:0010369
name: chromocenter
def: "A region in which centric, heterochromatic portions of one or more chromosomes form a compact structure." [PMID:12384572, PMID:15053486, PMID:16831888]
GO:0010370
name: perinucleolar chromocenter
def: "A chromocenter adjacent to the nucleolus." [PMID:15805479]
GO:0010445
name: nuclear dicing body
def: "A small round nuclear body, measuring 0.2-0.8 microns in diameter that is diffusely distributed throughout the nucleoplasm. Several proteins known to be involved in miRNA processing have been localized to these structures. D-bodies are thought to be involved in primary-miRNA processing and/or storage/assembly of miRNA processing complexes." [PMID:17442570]
GO:0010494
name: cytoplasmic stress granule
def: "A dense aggregation in the cytosol composed of proteins and RNAs that appear when the cell is under stress." [GOC:ans, PMID:17284590, PMID:17601829, PMID:17967451, PMID:20368989]
GO:0010598
name: NAD(P)H dehydrogenase complex (plastoquinone)
def: "Complex that possesses NAD(P)H dehydrogenase (plastoquinone) activity. The complex is one of the components of the electron transport chain. It is involved in electron transport from an unidentified electron donor, possibly NADH, NADPH or ferredoxin(Fd) to the plastoquinone pool." [PMID:15608332]
GO:0012505
name: endomembrane system
def: "A collection of membranous structures involved in transport within the cell. The main components of the endomembrane system are endoplasmic reticulum, Golgi bodies, vesicles, cell membrane and nuclear envelope. Members of the endomembrane system pass materials through each other or though the use of vesicles." [GOC:lh]
GO:0012506
name: vesicle membrane
def: "The lipid bilayer surrounding any membrane-bounded vesicle in the cell." [GOC:mah]
GO:0012507
name: ER to Golgi transport vesicle membrane
def: "The lipid bilayer surrounding a vesicle transporting substances from the endoplasmic reticulum to the Golgi." [GOC:ai, GOC:ascb_2009, GOC:dph, GOC:tb]
GO:0012508
name: Golgi to ER transport vesicle membrane
def: "The lipid bilayer surrounding a vesicle transporting substances from the Golgi to the ER." [GOC:ai]
GO:0012509
name: inter-Golgi transport vesicle membrane
def: "The lipid bilayer surrounding a vesicle transporting substances within the Golgi." [GOC:ai]
GO:0012510
name: trans-Golgi network transport vesicle membrane
def: "The lipid bilayer surrounding a vesicle transporting substances between the trans-Golgi network and other parts of the cell." [GOC:ai]
GO:0012511
name: monolayer-surrounded lipid storage body
def: "A subcellular organelle of plant cells surrounded by 'half-unit' or a monolayer membrane instead of the more usual bilayer. The storage body has a droplet of triglyceride surrounded by a monolayer of phospholipids, interacting with the triglycerides and the hydrophilic head groups facing the cytosol, and containing major protein components called oleosins." [GOC:mtg_sensu, ISBN:0943088372]
GO:0014069
name: postsynaptic density
def: "The postsynaptic density is a region that lies adjacent to the cytoplasmic face of the postsynaptic membrane at excitatory synapse. It forms a disc that consists of a range of proteins with different functions, some of which contact the cytoplasmic domains of ion channels in the postsynaptic membrane. The proteins making up the disc include receptors, and structural proteins linked to the actin cytoskeleton. They also include signalling machinery, such as protein kinases and phosphatases. The postsynaptic density may be part of a neuron or a muscle cell or a glial cell." [GOC:BHF, GOC:ef, GOC:jid, GOC:pr, GOC:sjp, http://molneuro.kaist.ac.kr/psd, PMID:14532281, Wikipedia:Postsynaptic_density]
GO:0014701
name: junctional sarcoplasmic reticulum membrane
def: "The part of the sarcoplasmic reticulum membrane that contains calcium release channels, is devoted to calcium release and is juxtaposed to transverse tubule membrane. The junctional sarcoplasmic reticulum membrane consists of the junctional region of the terminal cisterna membrane." [GOC:mtg_muscle]
GO:0014702
name: free sarcoplasmic reticulum membrane
def: "The part of the sarcoplasmic reticulum membrane that contains calcium pumps and is devoted to calcium uptake. The free sarcoplasmic reticulum membrane consists of the longitudinal sarcoplasmic reticulum membrane and the non-junctional region of the terminal cisterna membrane." [GOC:mtg_muscle]
GO:0014704
name: intercalated disc
def: "A complex cell-cell junction at which myofibrils terminate in cardiomyocytes; mediates mechanical and electrochemical integration between individual cardiomyocytes. The intercalated disc contains regions of tight mechanical attachment (fasciae adherentes and desmosomes) and electrical coupling (gap junctions) between adjacent cells." [GOC:mtg_muscle, PMID:11732910]
GO:0014705
name: C zone
def: "A region of the A band in which myosin-binding protein C is located and that can be seen by electron microscopy. This is a functional zone that also includes myosin." [GOC:mtg_muscle]
GO:0014731
name: spectrin-associated cytoskeleton
def: "The part of the cytoskeleton composed of spectrin, protein 4.1 and ankyrin. Spectrin-associated cytoskeleton is associated with the plasma membrane." [GOC:mtg_muscle, PMID:15970557]
GO:0014801
name: longitudinal sarcoplasmic reticulum
def: "The portion of the free sarcoplasmic reticulum consisting of longitudinal tubules that connect terminal cisternae." [GOC:mtg_muscle]
GO:0014802
name: terminal cisterna
def: "The portion of sarcoplasmic reticulum devoted to calcium ion storage and calcium ion release." [GOC:mtg_muscle]
GO:0014803
name: longitudinal sarcoplasmic reticulum lumen
def: "The region between the inner and outer lipid bilayers of the longitudinal sarcoplasmic reticulum envelope. The longitudinal sarcoplasmic reticulum lumen is continuous with the lumen contained within the terminal cisternae." [GOC:mtg_muscle]
GO:0014804
name: terminal cisterna lumen
def: "The region between the inner and outer lipid bilayers of the terminal cisterna envelope. This space is enriched in calsequestrin." [GOC:mtg_muscle]
GO:0015030
name: Cajal body
def: "A class of nuclear body, first seen after silver staining by Ramon y Cajal in 1903, enriched in small nuclear ribonucleoproteins, and certain general RNA polymerase II transcription factors; ultrastructurally, they appear as a tangle of coiled, electron-dense threads roughly 0.5 micrometers in diameter; involved in aspects of snRNP biogenesis; the protein coilin serves as a marker for Cajal bodies. Some argue that Cajal bodies are the sites for preassembly of transcriptosomes, unitary particles involved in transcription and processing of RNA." [NIF_Subcellular:nlx_subcell_090901, PMID:10944589, PMID:11031238, PMID:7559785]
GO:0015050
name: methane monooxygenase complex
def: "A protein complex that possesses methane monooxygenase activity; dimeric and trimeric complexes have been characterized." [BRENDA:1.14.13.25, GOC:mah]
GO:0015627
name: type II protein secretion system complex
def: "A large protein complex, containing 12-15 subunits, that spans the cell envelope of Gram-negative bacteria and mediates the movement of proteins into the extracellular environment. The complex includes a component in the cytoplasm, an inner membrane subcomplex that reaches into the periplasmic compartment and a secretion pore in the outer membrane. Proteins using the Type II pathway are transported across the cytoplasmic membrane by the Sec or Tat complex." [PMID:16448494]
GO:0015629
name: actin cytoskeleton
def: "The part of the cytoskeleton (the internal framework of a cell) composed of actin and associated proteins. Includes actin cytoskeleton-associated complexes." [GOC:jl, ISBN:0395825172, ISBN:0815316194]
GO:0015630
name: microtubule cytoskeleton
def: "The part of the cytoskeleton (the internal framework of a cell) composed of microtubules and associated proteins." [GOC:jl, ISBN:0395825172]
GO:0015934
name: large ribosomal subunit
def: "The larger of the two subunits of a ribosome. Two sites on the ribosomal large subunit are involved in translation, namely the aminoacyl site (A site) and peptidyl site (P site)." [ISBN:0198506732]
GO:0015935
name: small ribosomal subunit
def: "The smaller of the two subunits of a ribosome." [GOC:mah]
GO:0016006
name: Nebenkern
def: "A product of the fusion of the mitochondria during spermatogenesis. After the completion of meiosis the mitochondria of the spermatid collect along side the nucleus and fuse into two masses; these wrap around each other to produce the spherical Nebenkern. During flagellum elongation the Nebenkern unfolds and the two derivatives (major and minor mitochondrial derivatives) elongate down the axoneme." [GOC:ma]
GO:0016007
name: mitochondrial derivative
def: "The major and minor mitochondrial derivatives are the mitochondria of the sperm tail and derive by the unfolding of the Nebenkern during flagellum elongation." [GOC:ma]
GO:0016008
name: major mitochondrial derivative
def: "The larger of the two mitochondrial derivatives that arise by the unfolding of the Nebenkern during flagellum elongation; the major mitochondrial derivative is ovoid and darker than the minor derivative." [GOC:mah, PMID:17123504]
GO:0016009
name: minor mitochondrial derivative
def: "The smaller of the two mitochondrial derivatives that arise by the unfolding of the Nebenkern during flagellum elongation." [GOC:mah, PMID:17123504]
GO:0016010
name: dystrophin-associated glycoprotein complex
def: "A multiprotein complex that forms a strong mechanical link between the cytoskeleton and extracellular matrix; typical of, but not confined to, muscle cells. The complex is composed of transmembrane, cytoplasmic, and extracellular proteins, including dystrophin, sarcoglycans, dystroglycan, dystrobrevins, syntrophins, sarcospan, caveolin-3, and NO synthase." [PMID:15117830, PMID:16710609]
GO:0016011
name: dystroglycan complex
def: "A protein complex that includes alpha- and beta-dystroglycan, which are alternative products of the same gene; the laminin-binding component of the dystrophin-associated glycoprotein complex, providing a link between the subsarcolemmal cytoskeleton (in muscle cells) and the extracellular matrix. Alpha-dystroglycan is an extracellular protein binding to alpha-laminin and to beta-dystroglycan; beta-dystroglycan is a transmembrane protein which binds alpha-dystroglycan and dystrophin." [PMID:15117830, PMID:16710609]
GO:0016012
name: sarcoglycan complex
def: "A protein complex formed of four sarcoglycans plus sarcospan; there are six known sarcoglycans: alpha-, beta-, gamma-, delta-, epsilon- and zeta-sarcoglycan; all are N-glycosylated single-pass transmembrane proteins. The sarcoglycan-sarcospan complex is a subcomplex of the dystrophin glycoprotein complex, and is fixed to the dystrophin axis by a lateral association with the dystroglycan complex." [PMID:15117830, PMID:16710609]
GO:0016013
name: syntrophin complex
def: "A protein complex that includes alpha-, beta1-, beta2-syntrophins and syntrophin-like proteins; the syntrophin complex binds to the second half of the carboxy-terminal domain of dystrophin; also associates with neuronal nitric oxide synthase." [http://www.dmd.nl/DGC.html#syn]
GO:0016014
name: dystrobrevin complex
def: "A protein complex comprising alpha- and beta-dystrobrevin; forms part of the dystrophin glycoprotein complex." [PMID:15117830, PMID:16710609]
GO:0016020
name: membrane
def: "Double layer of lipid molecules that encloses all cells, and, in eukaryotes, many organelles; may be a single or double lipid bilayer; also includes associated proteins." [GOC:mah, ISBN:0815316194]
GO:0016021
name: integral component of membrane
def: "The component of a membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:go_curators]
GO:0016023
name: cytoplasmic membrane-bounded vesicle
def: "A membrane-bounded vesicle found in the cytoplasm of the cell." [GOC:ai, GOC:mah]
GO:0016027
name: inaD signaling complex
def: "A complex of proteins that are involved in phototransduction and attached to the transient receptor potential (TRP) channel. The protein connections are mediated through inaD." [GOC:hb, PMID:9010208, PMID:9796815]
GO:0016028
name: rhabdomere
def: "The specialized microvilli-containing organelle on the apical surfaces of a photoreceptor cell containing the visual pigment rhodopsin and most of the proteins involved in phototransduction." [GOC:hb, GOC:sart, PMID:8646774]
GO:0016029
name: subrhabdomeral cisterna
def: "A membrane-bounded compartment that is found at the base of the rhabdomere and contains stored calcium, InsP3 receptors and smooth endoplasmic reticulum Ca2+-ATPase." [PMID:11707492, PMID:8646774]
GO:0016035
name: zeta DNA polymerase complex
def: "A heterodimeric DNA polymerase complex that catalyzes error-prone DNA synthesis in contexts such as translesion synthesis and double-stranded break repair. First characterized in Saccharomyces, in which the subunits are Rev3p and Rev7p; a third protein, Rev1p, is often associated with the polymerase dimer." [PMID:16631579, PMID:16971464]
GO:0016222
name: procollagen-proline 4-dioxygenase complex
def: "A protein complex that catalyzes the formation of procollagen trans-4-hydroxy-L-proline and succinate from procollagen L-proline and 2-oxoglutarate, requiring Fe2+ and ascorbate. Contains two alpha subunits that contribute to most parts of the catalytic sites, and two beta subunits that are identical to protein-disulfide isomerase." [PMID:14500733, PMID:7753822]
GO:0016234
name: inclusion body
def: "A discrete intracellular part formed of aggregated molecules such as proteins or other biopolymers." [GOC:mah, PMID:11121744]
GO:0016235
name: aggresome
def: "An inclusion body formed by dynein-dependent retrograde transport of an aggregated protein on microtubules." [PMID:11121744]
GO:0016272
name: prefoldin complex
def: "A multisubunit chaperone that is capable of delivering unfolded proteins to cytosolic chaperonin, which it acts as a cofactor for. In humans, the complex is a heterohexamer of two PFD-alpha and four PFD-beta type subunits. In Saccharomyces cerevisiae, it also acts in the nucleus to regulate the rate of elongation by RNA polymerase II via a direct effect on histone dynamics." [GOC:jl, PMID:17384227, PMID:24068951, PMID:9630229]
GO:0016281
name: eukaryotic translation initiation factor 4F complex
def: "The eukaryotic translation initiation factor 4F complex is composed of eIF4E, eIF4A and eIF4G; it is involved in the recognition of the mRNA cap, ATP-dependent unwinding of the 5'-terminal secondary structure and recruitment of the mRNA to the ribosome." [GOC:hb, PMID:8449919]
GO:0016282
name: eukaryotic 43S preinitiation complex
def: "A protein complex composed of the 40S ribosomal subunit plus eIF1A, eIF3, and eIF2-GTP-bound methionyl-initiator methionine tRNA." [GOC:hjd, PMID:15145049]
GO:0016323
name: basolateral plasma membrane
def: "The region of the plasma membrane that includes the basal end and sides of the cell. Often used in reference to animal polarized epithelial membranes, where the basal membrane is the part attached to the extracellular matrix, or in plant cells, where the basal membrane is defined with respect to the zygotic axis." [GOC:go_curators]
GO:0016324
name: apical plasma membrane
def: "The region of the plasma membrane located at the apical end of the cell." [GOC:curators]
GO:0016327
name: apicolateral plasma membrane
def: "The apical end of the lateral plasma membrane of epithelial cells." [GOC:hb]
GO:0016328
name: lateral plasma membrane
def: "The portion of the plasma membrane at the lateral side of the cell. In epithelial cells, lateral plasma membranes are on the sides of cells which lie at the interface of adjacent cells." [GOC:hb, GOC:mah, GOC:pr]
GO:0016341
name: other collagen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0016342
name: catenin complex
def: "Complex of peripheral cytoplasmic proteins (alpha-, beta- and gamma-catenin) that interact with the cytoplasmic region of uvomorulin/E-cadherin to connect it to the actin cytoskeleton." [ISBN:0198599323]
GO:0016363
name: nuclear matrix
def: "The dense fibrillar network lying on the inner side of the nuclear membrane." [ISBN:0582227089]
GO:0016442
name: RISC complex
def: "A ribonucleoprotein complex that contains members of the Argonaute family of proteins, small interfering RNAs (siRNAs) or microRNAs (miRNAs), and miRNA or siRNA-complementary mRNAs, in addition to a number of accessory factors. The RISC complex is involved in posttranscriptional repression of gene expression through downregulation of translation or induction of mRNA degradation." [PMID:10749213, PMID:15145345]
GO:0016456
name: X chromosome located dosage compensation complex, transcription activating
def: "An RNA-protein complex localized to the X chromosome of males where it is required for the hyper-transcriptional activation of the X chromosome. An example of this is found in Drosophila melanogaster." [GOC:ma, GOC:mtg_sensu]
GO:0016459
name: myosin complex
def: "A protein complex, formed of one or more myosin heavy chains plus associated light chains and other proteins, that functions as a molecular motor; uses the energy of ATP hydrolysis to move actin filaments or to move vesicles or other cargo on fixed actin filaments; has magnesium-ATPase activity and binds actin. Myosin classes are distinguished based on sequence features of the motor, or head, domain, but also have distinct tail regions that are believed to bind specific cargoes." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html, ISBN:96235764]
GO:0016460
name: myosin II complex
def: "A myosin complex containing two class II myosin heavy chains, two myosin essential light chains and two myosin regulatory light chains. Also known as classical myosin or conventional myosin, the myosin II class includes the major muscle myosin of vertebrate and invertebrate muscle, and is characterized by alpha-helical coiled coil tails that self assemble to form a variety of filament structures." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html, ISBN:96235764]
GO:0016461
name: unconventional myosin complex
def: "A portmanteau term for myosins other than myosin II." [GOC:ma]
GO:0016465
name: chaperonin ATPase complex
def: "Multisubunit protein complex with 2x7 (Type I, in most cells) or 2x8 (Type II, in Archaea) ATP-binding sites involved in maintaining an unfolded polypeptide structure before folding or to entry into mitochondria and chloroplasts." [EC:3.6.4.9]
GO:0016469
name: proton-transporting two-sector ATPase complex
def: "A large protein complex that catalyzes the synthesis or hydrolysis of ATP by a rotational mechanism, coupled to the transport of protons across a membrane. The complex comprises a membrane sector (F0, V0, or A0) that carries out proton transport and a cytoplasmic compartment sector (F1, V1, or A1) that catalyzes ATP synthesis or hydrolysis. Two major types have been characterized: V-type ATPases couple ATP hydrolysis to the transport of protons across a concentration gradient, whereas F-type ATPases, also known as ATP synthases, normally run in the reverse direction to utilize energy from a proton concentration or electrochemical gradient to synthesize ATP. A third type, A-type ATPases have been found in archaea, and are closely related to eukaryotic V-type ATPases but are reversible." [GOC:mah, ISBN:0716743663, PMID:16691483]
GO:0016471
name: vacuolar proton-transporting V-type ATPase complex
def: "A proton-transporting two-sector ATPase complex found in the vacuolar membrane, where it acts as a proton pump to mediate acidification of the vacuolar lumen." [GOC:mah, ISBN:0716743663, PMID:16449553]
GO:0016472
name: sodium ion-transporting two-sector ATPase complex
def: "A large protein complex that catalyzes the synthesis or hydrolysis of ATP by a rotational mechanism, coupled to the transport of sodium ions across a membrane. The complex comprises a membrane sector (F0 or V0) that carries out ion transport and a cytoplasmic compartment sector (F1 or V1) that catalyzes ATP synthesis or hydrolysis." [GOC:mah, PMID:14656431]
GO:0016473
name: sodium ion-transporting F-type ATPase complex
def: "A sodium ion-transporting two-sector ATPase complex that catalyzes the phosphorylation of ADP to ATP. The complex comprises a membrane sector (F0) that carries out proton transport and a cytoplasmic compartment sector (F1) that catalyzes ATP synthesis by a rotational mechanism." [GOC:mah, PMID:14656431]
GO:0016474
name: sodium ion-transporting V-type ATPase complex
def: "A sodium ion-transporting two-sector ATPase complex that couples ATP hydrolysis to the transport of sodium ions across a concentration gradient. The complex comprises a membrane sector (V0) that carries out proton transport and a cytoplasmic compartment sector (V1) that catalyzes ATP hydrolysis." [GOC:mah, PMID:15802565]
GO:0016507
name: mitochondrial fatty acid beta-oxidation multienzyme complex
def: "A complex that includes the long-chain 3-hydroxyacyl-CoA dehydrogenase and long-chain enoyl-CoA hydratase activities in two subunits (alpha and beta), catalyzing two steps of the fatty acid beta-oxidation cycle within the mitochondrial matrix." [GOC:ma]
GO:0016513
name: core-binding factor complex
def: "A heterodimeric transcription factor complex that contains an alpha subunit (Runx1, Runx2 or Runx3 in human) that binds DNA and a non-DNA-binding beta subunit (CBFbeta), and binds to a consensus sequence 5'-YGYGGTY-3' found in several enhancers and promoters; the beta subunit enhances the DNA binding of the alpha subunit." [PMID:15156179, PMID:8497254]
GO:0016514
name: SWI/SNF complex
def: "A SWI/SNF-type complex that contains nine or more proteins, including both conserved (core) and nonconserved components; the Swi2/Snf2 ATPase is one of the core components." [GOC:mah, PMID:12672490]
GO:0016516
name: interleukin-4 receptor complex
def: "A protein complex that binds interleukin-4 (IL-4) and consists of an alpha chain that binds IL-4 with high affinity and a gamma common chain that also forms part of the interleukin-2 receptor." [PMID:10358772]
GO:0016528
name: sarcoplasm
def: "The cytoplasm of a muscle cell; includes the sarcoplasmic reticulum." [ISBN:0198547684]
GO:0016529
name: sarcoplasmic reticulum
def: "A fine reticular network of membrane-limited elements that pervades the sarcoplasm of a muscle cell; continuous over large portions of the cell and with the nuclear envelope; that part of the endoplasmic reticulum specialized for calcium release, uptake and storage." [GOC:mtg_muscle, ISBN:0124325653, ISBN:0198547684]
GO:0016533
name: cyclin-dependent protein kinase 5 holoenzyme complex
def: "A protein complex that activates cyclin-dependent kinase 5; composed of regulatory and catalytic subunits." [PMID:15689152]
GO:0016580
name: Sin3 complex
def: "A multiprotein complex that functions broadly in eukaryotic organisms as a transcriptional repressor of protein-coding genes, through the gene-specific deacetylation of histones. Amongst its subunits, the Sin3 complex contains Sin3-like proteins, and a number of core proteins that are shared with the NuRD complex (including histone deacetylases and histone binding proteins). The Sin3 complex does not directly bind DNA itself, but is targeted to specific genes through protein-protein interactions with DNA-binding proteins." [PMID:10589671, PMID:11743021, PMID:12865422]
GO:0016581
name: NuRD complex
def: "An approximately 2 MDa multi-subunit complex that exhibits ATP-dependent chromatin remodeling activity in addition to histone deacetylase (HDAC) activity, and has been shown to establish transcriptional repression of a number of target genes in vertebrates, invertebrates and fungi. Amongst its subunits, the NuRD complex contains histone deacetylases, histone binding proteins and Mi-2-like proteins." [PMID:10589671, PMID:11743021, PMID:17289569]
GO:0016585
name: chromatin remodeling complex
def: "OBSOLETE: Any complex that mediates dynamic changes in eukaryotic chromatin." [GOC:mah]
GO:0016586
name: RSC complex
def: "A protein complex similar to, but more abundant than, the Swi/Snf complex. The RSC complex is generally recruited to RNA polymerase III promoters and is specifically recruited to RNA polymerase II promoters by transcriptional activators and repressors; it is also involved in non-homologous end joining." [GOC:rb, PMID:11937489, PMID:12672490, PMID:15870268, PMID:19355820, PMID:8980231]
GO:0016587
name: Isw1 complex
def: "A protein complex that contains an Isw1 subunit from the ISWI-family of ATPases and acts to modify chromatin structure." [GOC:krc, GOC:mah, PMID:15020051, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0016589
name: NURF complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (SNF2L in mammals), a NURF301 homolog (BPTF in humans), and additional subunits, though the composition of these additional subunits varies slightly with species. NURF is involved in regulation of transcription from TRNA polymerase II promoters." [GOC:bf, GOC:krc, PMID:10779516, PMID:11279013, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0016590
name: ACF complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (SNF2H in mammals, Isw2 in S. cerevisiae), an ACF1 homolog, and generally no other subunits, though Xenopus is an exception with a third non-conserved subunit. ACF plays roles in regulation of RNA polymerase II transcription and in DNA replication and repair." [GOC:bf, GOC:krc, PMID:12192034, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0016591
name: DNA-directed RNA polymerase II, holoenzyme
def: "Large protein complex composed of the RNA polymerase core complex and a variety of other proteins including transcription factor complexes TFIIA, D, E, F, and H which are required for promoter recognition, and the Mediator subcomplex. Catalyzes the synthesis of eukaryotic pre-mRNA." [GOC:jl, PMID:15196470, Wikipedia:Rna_polymerase_ii]
GO:0016592
name: mediator complex
def: "A protein complex that interacts with the carboxy-terminal domain of the largest subunit of RNA polymerase II and plays an active role in transducing the signal from a transcription factor to the transcriptional machinery. The mediator complex is required for activation of transcription of most protein-coding genes, but can also act as a transcriptional corepressor. The Saccharomyces complex contains several identifiable subcomplexes: a head domain comprising Srb2, -4, and -5, Med6, -8, and -11, and Rox3 proteins; a middle domain comprising Med1, -4, and -7, Nut1 and -2, Cse2, Rgr1, Soh1, and Srb7 proteins; a tail consisting of Gal11p, Med2p, Pgd1p, and Sin4p; and a regulatory subcomplex comprising Ssn2, -3, and -8, and Srb8 proteins. Metazoan mediator complexes have similar modular structures and include homologs of yeast Srb and Med proteins." [PMID:11454195, PMID:16168358, PMID:17870225]
GO:0016593
name: Cdc73/Paf1 complex
def: "A multiprotein complex that associates with RNA polymerase II and general RNA polymerase II transcription factor complexes and may be involved in both transcriptional initiation and elongation. In Saccharomyces the complex contains Paf1p, Cdc73p, Ctr9p, Rtf1p, and Leo1p." [PMID:11884586]
GO:0016600
name: flotillin complex
def: "A protein complex that contains flotillin-1 and flotillin-2, and may contain associated proteins. Flotillins associate into membrane microdomains resembling caveolae." [PMID:17206938, PMID:17600709]
GO:0016602
name: CCAAT-binding factor complex
def: "A heteromeric transcription factor complex that binds to the CCAAT-box upstream of promoters; in Saccharomyces it activates the transcription of genes in response to growth in a nonfermentable carbon source; consists of four known subunits: HAP2, HAP3, HAP4 and HAP5." [PMID:7828851]
GO:0016604
name: nuclear body
def: "Extra-nucleolar nuclear domains usually visualized by confocal microscopy and fluorescent antibodies to specific proteins." [GOC:ma, PMID:10330182]
GO:0016605
name: PML body
def: "A class of nuclear body; they react against SP100 auto-antibodies (PML, promyelocytic leukemia); cells typically contain 10-30 PML bodies per nucleus; alterations in the localization of PML bodies occurs after viral infection." [GOC:ma, PMID:10944585]
GO:0016606
name: LYSP100-associated nuclear domain
def: "A nuclear body that is enriched in the lymphoid cell-specific protein LYSp100B; LANDs are globular, electron-dense structures and are morphologically distinct from the annular structures characteristic of PML bodies." [PMID:10921892, PMID:8695863]
GO:0016607
name: nuclear speck
def: "A discrete extra-nucleolar subnuclear domain, 20-50 in number, in which splicing factors are seen to be localized by immunofluorescence microscopy." [http://www.cellnucleus.com/]
GO:0016610
name: nitrogenase complex
def: "An enzyme complex composed of two proteins, dinitrogenase and nitrogenase reductase; dinitrogenase is tetrameric with an alpha2-beta2 structure and nitrogenase reductase is a homodimer, and both are associated with metal ions, which differ between species. Both proteins are required for the enzyme activity of the complex, the formation of oxidized ferredoxin and ammonia from reduced ferredoxin and nitrogen." [EC:1.18.6.1, MetaCyc:CPLX-186, MetaCyc:CPLX-525]
GO:0016611
name: iron-iron nitrogenase complex
def: "An enzyme complex containing an iron-iron cluster found in species such as the photosynthetic bacterium Rhodobacter capsulatus. It is composed of two main subunits, dinitrogenase and nitrogenase reductase. Dinitrogenase, the iron-iron containing subunit, has an alpha1-beta2 or alpha2-beta2 structure, and the nitrogenase reductase subunit is a homodimer. Functions in the catalysis of the formation of oxidized ferredoxin and ammonia from reduced ferredoxin and nitrogen." [EC:1.18.6.1, GOC:jl, PMID:11848850]
GO:0016612
name: molybdenum-iron nitrogenase complex
def: "An enzyme complex containing a molybdenum-iron cluster found in many species. It is composed of two proteins, dinitrogenase and nitrogenase reductase; dinitrogenase, the molybdenum-iron protein, is tetrameric with an alpha2-beta2 structure, and nitrogenase reductase is a homodimer." [EC:1.18.6.1]
GO:0016613
name: vanadium-iron nitrogenase complex
def: "An enzyme complex containing a vanadium-iron cluster found in some species, such as Azotobacter vinelandii. It is composed of two proteins, dinitrogenase and nitrogenase reductase; dinitrogenase, the vanadium-iron protein, is tetrameric with an alpha2-beta2 structure, and nitrogenase reductase is a homodimer." [EC:1.18.6.1, PMID:3474027]
GO:0016914
name: follicle-stimulating hormone complex
def: "A gonadotrophic glycoprotein hormone secreted, in mammals, by the anterior pituitary gland; consists of alpha and beta subunits, the latter of which confers hormonal specificity." [ISBN:0198547684]
GO:0016935
name: glycine-gated chloride channel complex
def: "A protein complex that forms a transmembrane channel through which chloride ions may pass in response to glycine binding to the channel complex or one of its constituent parts." [GOC:mah]
GO:0016938
name: kinesin I complex
def: "A complex of two kinesin heavy chains and two kinesin light chains." [http://www.csfic.mi.cnr.it/centro/lines/8/ric.html]
GO:0016939
name: kinesin II complex
def: "A complex consisting of two distinct motor subunits that form a heterodimer complexed with a third non-motor accessory subunit, the kinesin associated protein or KAP; the KIF3 heterodimer interacts via its C-terminal portion with KAP, which is thought to regulate the binding of the motor to cargo membranes." [http://www.csfic.mi.cnr.it/centro/lines/8/ric.html]
GO:0016942
name: insulin-like growth factor binding protein complex
def: "A complex of proteins which includes the insulin-like growth factor (IGF) and a number of IGF-binding proteins. The complex plays a role in growth and development." [GOC:jl]
GO:0017023
name: myosin phosphatase complex
def: "An enzyme complex that catalyzes the removal of the phosphate group from phosphomyosin." [EC:3.1.3.16]
GO:0017052
name: insulin-like growth factor binding protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0017053
name: transcriptional repressor complex
def: "A protein complex that possesses activity that prevents or downregulates transcription." [GOC:mah]
GO:0017054
name: negative cofactor 2 complex
def: "A heterodimeric protein complex that can stably associate with TATA-binding protein on promoters, thereby preventing the assembly of transcription factors TFIIA and TFIIB and leading to repression of RNA polymerase II transcription. The two subunits, NC2alpha (Drap1) and NC2beta (Dr1), dimerize through histone fold domains of the H2A/H2B type present in the amino termini." [PMID:15574413]
GO:0017059
name: serine C-palmitoyltransferase complex
def: "An enzyme complex that catalyzes the transfer of a palmitoyl on to serine, forming 3-dehydro-D-sphinganine." [EC:2.3.1.50]
GO:0017071
name: intracellular cyclic nucleotide activated cation channel complex
def: "A protein complex that forms a transmembrane channel through which cations ions may pass in response to an intracellular cyclic nucleotide binding to the channel complex or one of its constituent parts." [GOC:mah]
GO:0017086
name: 3-methyl-2-oxobutanoate dehydrogenase (lipoamide) complex
def: "A protein complex that catalyzes the reaction 3-methyl-2-oxobutanoate + lipoamide = S-(2-methylpropanoyl)-dihydrolipoamide + carbon dioxide (CO2). This requires thiamine diphosphate; the enzyme also acts on (S)-3-methyl-2-oxopentanoate and 4-methyl-2-oxo-pentanoate." [EC:1.2.4.4]
GO:0017087
name: mitochondrial processing peptidase complex
def: "A protein complex that consists of a catalytic alpha subunit (alpha-MPP) and a regulatory beta subunit (beta-MPP), and catalyzes the release of N-terminal targeting peptides from precursor proteins imported into the mitochondrion." [BRENDA:3.4.24.64, EC:3.4.24.64, GOC:mah]
GO:0017090
name: meprin A complex
def: "A protein complex that is located in the cell membrane, and is involved in the metabolism of peptides, including neuropeptides. The complex has metalloendopeptidase activity that catalyzes the hydrolysis of protein and peptide substrates, preferentially on carboxyl side of hydrophobic residues." [EC:3.4.24.18, GOC:mah, MEROPS_fam:M12]
GO:0017101
name: aminoacyl-tRNA synthetase multienzyme complex
def: "A multienzyme complex found in all multicellular eukaryotes composed of eight proteins with aminoacyl-tRNA synthetase activities (abbreviated as: ArgRS, AspRS, GluProRS, GlnRS, IleRS, LeuRS, LysRS, MetRS where RS is the enzyme, preceded by the amino acid it uses as a substrate) as well as three non-synthetase proteins (p43, p38, and p18) with diverse functions. Several of these subunits are known dimers, so the total polypeptide count in the multisynthetase complex is at least fifteen. All of the enzymes in this assembly catalyze the same reaction, the covalent attachment of an amino acid to either the 2'- or 3'-hydroxyl of the 3'-terminal adenosine of tRNA, but using different substrates." [GOC:jl, PMID:16169847]
GO:0017102
name: methionyl glutamyl tRNA synthetase complex
def: "A complex consisting of methionyl- and glutamyl-tRNA synthetases. The tRNA synthetases present in the complex bind to their cognate tRNAs more efficiently than they do as monomers." [GOC:mcc, PMID:11069915]
GO:0017109
name: glutamate-cysteine ligase complex
def: "An enzyme complex that catalyzes the ligation of glutamate to cysteine, forming glutamylcysteine." [EC:6.3.2.2]
GO:0017117
name: single-stranded DNA-dependent ATP-dependent DNA helicase complex
def: "A protein complex that possesses single-stranded DNA-dependent DNA helicase activity." [GOC:mah]
GO:0017119
name: Golgi transport complex
def: "A complex of proteins that, in vitro, stimulates intra-Golgi transport; a 13S complex, about 800 kDa in size and consists of at least five polypeptides. In yeast, this complex is called the Sec34/35 complex and is composed of eight subunits (Sec34p, Sec35p, Dor1p, Cod1p, Cod2p, Cod3p, Cod4p, and Cod5p)." [GOC:krc, PMID:9792665]
GO:0017122
name: protein N-acetylglucosaminyltransferase complex
def: "Heterotrimeric enzyme complex, which in humans is composed of two large subunits of the same size, and one smaller subunit. Functions in the addition of nucleotide-activated sugars onto the polypeptide." [PMID:15247246]
GO:0017133
name: mitochondrial electron transfer flavoprotein complex
def: "A protein complex located in the mitochondrion. It contains flavin adenine dinucleotide (FAD) that, together with an acyl-CoA dehydrogenase, forms a system that oxidizes an acyl-CoA molecule and reduces ubiquinone and other acceptors in the mitochondrial electron transport system." [GOC:mtg_sensu, ISBN:0198506732]
GO:0017146
name: N-methyl-D-aspartate selective glutamate receptor complex
def: "An assembly of four or five subunits which form a structure with an extracellular N-terminus and a large loop that together form the ligand binding domain. The C-terminus is intracellular. The ionotropic glutamate receptor complex itself acts as a ligand gated ion channel; on binding glutamate, charged ions pass through a channel in the center of the receptor complex. NMDA receptors are composed of assemblies of NR1 subunits (Figure 3) and NR2 subunits, which can be one of four separate gene products (NR2A-D). Expression of both subunits are required to form functional channels. The glutamate binding domain is formed at the junction of NR1 and NR2 subunits. NMDA receptors are permeable to calcium ions as well as being permeable to other ions. Thus NMDA receptor activation leads to a calcium influx into the post-synaptic cells, a signal thought to be crucial for the induction of NMDA-receptor dependent LTP and LTD." [http://www.bris.ac.uk/Depts/Synaptic/info/glutamate.html]
GO:0017177
name: glucosidase II complex
def: "A heterodimeric complex that catalyzes the trimming of glucose residues from N-linked core glycans on newly synthesized glycoproteins." [PMID:10464333, PMID:8910335]
GO:0018444
name: translation release factor complex
def: "A heterodimeric complex involved in the release of a nascent polypeptide chain from a ribosome." [ISBN:0198547684]
GO:0018995
name: host
def: "Any organism in which another organism, especially a parasite or symbiont, spends part or all of its life cycle and from which it obtains nourishment and/or protection." [ISBN:0198506732]
GO:0018998
name: metaxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0019005
name: SCF ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul1 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 adaptor and an F-box protein. SCF complexes are involved in targeting proteins for degradation by the proteasome. The best characterized complexes are those from yeast and mammals (with core subunits named Cdc53/Cul1, Rbx1/Hrt1/Roc1)." [PMID:15571813, PMID:15688063]
GO:0019008
name: molybdopterin synthase complex
def: "A protein complex that possesses molybdopterin synthase activity. In E. coli, the complex is a heterotetramer consisting of two MoaD and two MoaE subunits." [GOC:mah, PMID:12571227, PMID:15709772]
GO:0019012
name: virion
def: "The complete fully infectious extracellular virus particle." [ISBN:0781718325]
GO:0019013
name: viral nucleocapsid
def: "The complete protein-nucleic acid complex that is the packaged form of the genome in a virus particle." [ISBN:0781702534]
GO:0019015
name: viral genome
def: "The whole of the genetic information of a virus, contained as either DNA or RNA." [ISBN:0198506732]
GO:0019016
name: non-segmented viral genome
def: "A viral genome that consists of one continuous nucleic acid molecule." [GOC:pk]
GO:0019017
name: segmented viral genome
def: "A viral genome that is divided into two or more physically separate molecules of nucleic acid and packaged into a single virion." [ISBN:0121585336]
GO:0019018
name: bipartite viral genome
def: "A segmented viral genome consisting of two sub-genomic nucleic acids but each nucleic acid is packaged into a different virion." [ISBN:0121585336]
GO:0019019
name: tripartite viral genome
def: "A segmented viral genome consisting of three sub-genomic nucleic acids but each nucleic acid is packaged into a different virion." [ISBN:0121585336]
GO:0019020
name: multipartite viral genome
def: "A segmented viral genome consisting of more than three sub-genomic nucleic acids but each nucleic acid is packaged into a different virion." [ISBN:0121585336]
GO:0019021
name: DNA viral genome
def: "A viral genome composed of deoxyribonucleic acid." [ISBN:0121585336]
GO:0019022
name: RNA viral genome
def: "A viral genome composed of ribonucleic acid. This results in genome replication and expression of genetic information being inextricably linked." [ISBN:0121585336]
GO:0019023
name: dsRNA viral genome
def: "A viral genome composed of double stranded RNA." [ISBN:0121585336]
GO:0019024
name: ssRNA viral genome
def: "A viral genome composed of single stranded RNA of either positive or negative sense." [ISBN:0121585336]
GO:0019025
name: positive sense viral genome
def: "A single stranded RNA genome with the same nucleotide polarity as mRNA." [ISBN:0121585336]
GO:0019026
name: negative sense viral genome
def: "A single stranded RNA genome with the opposite nucleotide polarity as mRNA." [ISBN:0121585336]
GO:0019027
name: ambisense viral genome
def: "A RNA genome that contains coding regions that are either positive sense or negative sense on the same RNA molecule." [ISBN:0121585336]
GO:0019028
name: viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles. It comprises numerous regularly arranged subunits, or capsomeres." [GOC:mtg_sensu, ISBN:0198506732]
GO:0019029
name: helical viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles; the subunits are arranged to form a protein helix with the genetic material contained within. Tobacco mosaic virus has such a capsid structure." [ISBN:071673706X, UniProtKB-KW:KW-1139, VZ:885]
GO:0019030
name: icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles; the subunits are arranged to form an icosahedron, a solid with 20 faces and 12 vertices. Icosahedral capsids have 12 pentamers plus 10(T-1) hexamers, where T is the triangulation number. Tobacco satellite necrosis virus has such a capsid structure." [GOC:bm, ISBN:0198506732, ISBN:071673706X, VZ:885, Wikipedia:Capsid]
GO:0019031
name: viral envelope
def: "The lipid bilayer of a virion that surrounds the protein capsid. May also contain glycoproteins." [GOC:bf, GOC:bm, GOC:jl, ISBN:0781718325, Wikipedia:Viral_envelope]
GO:0019032
name: viral glycoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0019033
name: viral tegument
def: "A structure lying between the capsid and envelope of a virus, varying in thickness and often distributed asymmetrically." [ISBN:0721662544]
GO:0019034
name: viral replication complex
def: "Specific locations and structures in the virus infected cell involved in replicating the viral genome." [ISBN:0781718325]
GO:0019035
name: viral integration complex
def: "A nucleoprotein complex containing viral genetic material and associated viral and host proteins, which is capable of inserting a viral genome into a host genome." [ISBN:0781718325, PMID:16712776, Wikipedia:Pre-integration_complex]
GO:0019036
name: viral transcriptional complex
def: "Specific locations and structures in the virus infected cell involved in transcribing the viral genome." [ISBN:0781718325]
GO:0019037
name: viral assembly intermediate
def: "Specific locations and structures in the virus infected cell involved in assembling new virions." [ISBN:0781718325]
GO:0019038
name: provirus
def: "The name given to a viral genome after it has been integrated into the host genome; particularly applies to retroviruses and is a required part of the retroviral replication cycle." [ISBN:0121585336]
GO:0019183
name: histamine-gated chloride channel complex
def: "A protein complex that forms a transmembrane channel through which chloride ions may pass in response to histamine binding to the channel complex or one of its constituent parts." [GOC:mah]
GO:0019185
name: snRNA-activating protein complex
def: "A protein complex that recognizes the proximal sequence element of RNA polymerase II and III snRNA promoters." [PMID:7715707, PMID:9003788]
GO:0019197
name: phosphoenolpyruvate-dependent sugar phosphotransferase complex
def: "Includes phosphoenolpyruvate-protein phosphatase (enzyme I of the phosphotransferase system) and protein-N(PI)-phosphohistidine-sugar phosphotransferase (enzyme II of the phosphotransferase system)." [GOC:ma]
GO:0019717
name: synaptosome
def: "OBSOLETE: Any of the discrete particles (nerve-ending particles) formed from the clublike presynaptic nerve endings that resist disruption and are snapped or torn off their attachments when brain tissue is homogenized in media isosmotic to plasma." [ISBN:0198506732]
GO:0019718
name: rough microsome
def: "OBSOLETE: Vesicular particles formed from disrupted endoplasmic reticulum membranes and studded with ribosomes on the outside." [ISBN:0198506732]
GO:0019719
name: smooth microsome
def: "OBSOLETE: Vesicular particles formed from disrupted endoplasmic reticulum and plasma membranes without any adhering ribosomes." [ISBN:0198506732]
GO:0019773
name: proteasome core complex, alpha-subunit complex
def: "The proteasome core subcomplex that constitutes the two outer rings of the proteasome core complex. An example of this component is found in Mus musculus." [GOC:jl, GOC:mtg_sensu, GOC:rb, PMID:10854779]
GO:0019774
name: proteasome core complex, beta-subunit complex
def: "The proteasome core subcomplex that constitutes the two inner rings of the proteasome core complex. An example of this component is found in Mus musculus." [GOC:jl, GOC:mtg_sensu, GOC:rb, PMID:10854779]
GO:0019804
name: quinolinate synthetase complex
def: "OBSOLETE. A heterodimer which acts as a quinolinate synthetase; quinolinate synthetase B (L-aspartate oxidase) catalyzes the oxidation of L-aspartate to L-iminoaspartate; quinolinate synthetase A condenses L-imidoaspartate and dihydroxyacetone to quinolinate." [UniProtKB:P10902]
GO:0019812
name: Type I site-specific deoxyribonuclease complex
def: "A multisubunit complex composed of two copies of a restriction (R) subunit, two copies of a methylation (M) subunit, and one copy of a specificity (S) subunit. This complex recognizes specific short DNA sequences (through the S subunit), and binds to them. If the recognition site is hemimethylated, the complex acts as a methyltransferase which modifies the recognition site, using S-adenosylmethionine as the methyl donor. Only the M and S subunits are required for this reaction. If the complex binds to an unmethylated recognition site, then the complex translocates the DNA bidirectionally in an ATP-dependent manner. When the translocation is stalled by impact with another complex or unusual DNA structure, the enzyme functions as an endonuclease and cleavage of the DNA will occur, hundreds or thousands of base pairs away from the recognition site. These DNA restriction systems are used by bacteria to defend against phage and other foreign DNA that may enter a cell." [PMID:12654995, PMID:15788748]
GO:0019813
name: Type III site-specific deoxyribonuclease complex
def: "A heterodimeric enzyme complex composed of two subunits, Res and Mod, that functions as an endonuclease and cleaves DNA. Cleavage will only occur when there are two un-methylated copies of a specific recognition site in an inverse orientation on the DNA. Cleavage occurs at a specific distance away from one of the recognition sites. The Mod subunit can act alone as a methyltansferase. DNA restriction systems such as this are used by bacteria to defend against phage and other foreign DNA that may enter a cell." [PMID:12654995]
GO:0019814
name: immunoglobulin complex
def: "A protein complex that in its canonical form is composed of two identical immunoglobulin heavy chains and two identical immunoglobulin light chains, held together by disulfide bonds and sometimes complexed with additional proteins. An immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, GOC:jl, ISBN:0781765196]
GO:0019815
name: B cell receptor complex
def: "An immunoglobulin complex that is present in the plasma membrane of B cells and that in its canonical form is composed of two identical immunoglobulin heavy chains and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781735149]
GO:0019816
name: B cell receptor accessory molecule complex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0019819
name: P1 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P1 peroxisomes are distinguished from the other subforms on the bases of buoyant density and protein content; they contain fewer peroxisomal proteins than the other subforms." [PMID:10629216]
GO:0019820
name: P2 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P2 peroxisomes are distinguished from the other subforms on the bases of buoyant density and protein content; they are the least dense of the subforms observed." [PMID:10629216]
GO:0019821
name: P3 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P3 peroxisomes are formed by fusion of P1 and P2 peroxisomes, and are distinguished from the other subforms on the bases of buoyant density and protein content." [PMID:10629216]
GO:0019822
name: P4 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P4 peroxisomes are distinguished from the other subforms on the bases of buoyant density and protein content." [PMID:10629216]
GO:0019823
name: P5 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P5 peroxisomes are distinguished from the other subforms on the bases of buoyant density and protein content." [PMID:10629216]
GO:0019824
name: P6 peroxisome
def: "A subform of peroxisome that corresponds to an intermediate in a peroxisome assembly pathway, which operates by conversion of peroxisomal subforms in the direction P1, P2 -> P3 -> P4 -> P5 -> P6. P6 peroxisomes are distinguished from the other subforms on the bases of buoyant density and protein content, and are equivalent to mature peroxisomes." [PMID:10629216]
GO:0019861
name: flagellum
def: "OBSOLETE. Long whiplike or feathery structures borne either singly or in groups by the motile cells of many bacteria and unicellular eukaryotes and by the motile male gametes of many eukaryotic organisms, which propel the cell through a liquid medium." [UniProtKB-KW:KW-0282]
GO:0019866
name: organelle inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of an organelle envelope; usually highly selective to most ions and metabolites." [GOC:mah]
GO:0019867
name: outer membrane
def: "The external membrane of Gram-negative bacteria or certain organelles such as mitochondria and chloroplasts; freely permeable to most ions and metabolites." [GOC:go_curators]
GO:0019897
name: extrinsic component of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:curators, GOC:dos]
GO:0019898
name: extrinsic component of membrane
def: "The component of a membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:jl, GOC:mah]
GO:0019907
name: cyclin-dependent protein kinase activating kinase holoenzyme complex
def: "A cyclin-dependent kinase activating kinase complex capable of activating cyclin-dependent kinases by threonine phosphorylation, thus regulating cell cycle progression. consists of a kinase, cyclin and optional assembly factors, in human CDK7, CCNH and MNAT1. CAK activity is itself regulated throughout the cell cycle by T-loop phosphorylation of its kinase component (CDK7 in human). Phosphorylation of serine residues during mitosis inactivates the enzyme. Also capable of CAK phosphorylating the carboxyl-terminal domain (CTD) of RNA polymerase II and other transcription activating proteins, as part of the general transcription factor TFIIH." [GOC:bhm, PMID:8752210]
GO:0019908
name: nuclear cyclin-dependent protein kinase holoenzyme complex
def: "Cyclin-dependent protein kinase (CDK) complex found in the nucleus." [GOC:krc]
GO:0019910
name: mitochondrial pyruvate dehydrogenase (lipoamide) phosphatase complex
def: "A mitochondrial complex of a regulatory and catalytic subunit that catalyzes the dephosphorylation and concomitant reactivation of the alpha subunit of the E1 component of the pyruvate dehydrogenase complex. An example of this component is found in Mus musculus." [GOC:mtg_sensu, PMID:9395502]
GO:0020002
name: host cell plasma membrane
def: "The plasma membrane surrounding a host cell." [GOC:mb]
GO:0020003
name: symbiont-containing vacuole
def: "Membrane-bounded vacuole within a host cell in which a symbiont organism resides. The vacuole membrane is derived from both the host and symbiont." [GOC:jl, GOC:mb]
GO:0020004
name: symbiont-containing vacuolar space
def: "The space between a symbiont plasma membrane and the symbiont-containing vacuole membrane." [GOC:jl, GOC:mb]
GO:0020005
name: symbiont-containing vacuole membrane
def: "The lipid bilayer surrounding a symbiont-containing vacuole, derived from both the host and symbiont." [GOC:jl, GOC:mb]
GO:0020006
name: symbiont-containing vacuolar membrane network
def: "Tubular network of extensions from the symbiont-containing vacuole membrane that protrude into the host cytoplasm." [GOC:jl, PMID:3528173]
GO:0020007
name: apical complex
def: "A group of cytoskeletal structures and associated membrane-bounded organelles found at the anterior end of adult obligate intracellular protozoan parasites in the phylum Apicomplexa. The apical complex is involved in attachment to and penetration of the host cell, and in parasite proliferation." [GOC:giardia, GOC:mb, PMID:16518471]
GO:0020008
name: rhoptry
def: "A large, club-shaped secretory organelle that forms part of the apical complex of an apicomplexan parasite, and consists of a bulbous body and a narrow electron-dense neck that extends through the conoid at the apical tip of the parasite. The rhoptry necks serve as ducts through which the contents of the rhoptries are secreted after attachment to the host has been completed and at the commencement of invasion. Rhoptry proteins function in the biogenesis and host organellar association of the parasitophorous vacuole." [ISBN:0521664470, PMID:11801218, PMID:16002398]
GO:0020009
name: microneme
def: "A small, elongated secretory organelle that forms part of the apical complex, located along the main axis of an apicomplexan parasite cell within the extreme apical region and at the periphery under the inner membrane complex. Of the specialized secretory compartments identified in apicomplexans, micronemes discharge their contents first, during initial contact of the parasite's apical pole with the host cell surface. Micronemal proteins function during parasite attachment and penetration into the target cell." [ISBN:0521664470, PMID:11801218]
GO:0020010
name: conoid
def: "A spiral cytoskeletal structure located at the apical end of the apical complex in some apicomplexan parasites. Fibers form a left-handed spiral, and are comprised of tubulin protofilaments organized in a ribbon-like structure that differs from the conventional tubular structure characteristic of microtubules." [GOC:expert_dr, PMID:11901169, PMID:16518471]
GO:0020011
name: apicoplast
def: "The plastid organelle found in apicomplexans." [ISBN:0521664470]
GO:0020015
name: glycosome
def: "A membrane-bounded organelle found in organisms from the order Kinetoplastida that houses the enzymes of glycolysis." [GOC:mb]
GO:0020016
name: ciliary pocket
def: "Invagination of the plasma membrane from which a cilium (also called flagellum) protrudes." [GOC:cilia, GOC:mb]
GO:0020018
name: ciliary pocket membrane
def: "That part of the plasma membrane found in the ciliary pocket (also called flagellar pocket)." [GOC:cilia, GOC:mb]
GO:0020020
name: food vacuole
def: "Vacuole within a parasite used for digestion of the host cell cytoplasm. An example of this component is found in the Apicomplexa." [GOC:mb]
GO:0020022
name: acidocalcisome
def: "An electron-dense acidic membrane-bounded organelle which contains a matrix of pyrophosphate and polyphosphates with bound calcium and other cations." [GOC:mb]
GO:0020023
name: kinetoplast
def: "A sub-structure within the large single mitochondrion of kinetoplastid parasites and which is closely associated with the flagellar pocket and basal body of the flagellum." [GOC:mb]
GO:0020025
name: subpellicular microtubule
def: "Singlet microtubules that lie underneath the inner membrane complex and emanate from the basal ring of the conoid." [GOC:mb]
GO:0020026
name: merozoite dense granule
def: "Electron-dense organelle with a granular internal matrix found throughout the merozoite life cycle stage of apicomplexan parasites; contains proteins destined to be secreted into the parasitophorous vacuole following parasite invasion of a host cell." [GOC:mb, GOC:mtg_sensu]
GO:0020030
name: infected host cell surface knob
def: "Protrusion that develops in the plasma membrane of a parasitized erythrocyte. An example of this component is found in Plasmodium species." [GOC:mb]
GO:0020031
name: polar ring of apical complex
def: "An electron dense ring at the most anterior position of the apical complex, from which the conoid fibers originate; formed during an invasive life cycle stage of an apicomplexan parasite." [GOC:mb, PMID:16518471]
GO:0020032
name: basal ring of apical complex
def: "An electron dense ring at the most posterior position of the apical complex, from which the subpellicular microtubules originate; formed during an invasive life cycle stage of an apicomplexan parasite." [GOC:mah, PMID:16518471]
GO:0020036
name: Maurer's cleft
def: "A disk-like structure that appears at the periphery of a red blood cell infected by an apicomplexan parasite, characterized by a translucent lumen and an electron-dense coat of variable thickness; often appears to be tethered to the host cell membrane by fibrous connections with the erythrocyte cytoskeleton." [PMID:16705161]
GO:0020038
name: subpellicular network
def: "A mechanically stable cytoskeletal structure associated with the cytoplasmic face of the pellicle and surrounding the microtubule-based cytoskeleton." [PMID:11420112]
GO:0020039
name: pellicle
def: "The structure enclosing an apicomplexan parasite cell; consists of the cell membrane with its associated infrastructure of microtubules, microfilaments and other organelles." [GOC:mah, GOC:mb]
GO:0022623
name: proteasome-activating nucleotidase complex
def: "A homohexameric complex that recognizes and unfolds core proteasome substrate proteins, and translocates them to the core complex in an ATP dependent manner." [GOC:bf, GOC:mtg_sensu, PMID:19363223, PMID:19481528]
GO:0022624
name: proteasome accessory complex
def: "A protein complex, that caps one or both ends of the proteasome core complex and regulates entry into, or exit from, the proteasome core complex." [GOC:mtg_sensu]
GO:0022625
name: cytosolic large ribosomal subunit
def: "The large subunit of a ribosome located in the cytosol." [GOC:mtg_sensu]
GO:0022626
name: cytosolic ribosome
def: "A ribosome located in the cytosol." [GOC:mtg_sensu]
GO:0022627
name: cytosolic small ribosomal subunit
def: "The small subunit of a ribosome located in the cytosol." [GOC:mtg_sensu]
GO:0022628
name: chloroplast large ribosomal subunit
def: "The large subunit of a ribosome contained within a chloroplast." [GOC:mtg_sensu]
GO:0022629
name: chloroplast small ribosomal subunit
def: "The small subunit of a ribosome contained within a chloroplast." [GOC:mtg_sensu]
GO:0030008
name: TRAPP complex
def: "A large complex that acts as a tethering factor involved in transporting vesicles from the ER through the Golgi to the plasma membrane. A TRAPP (transport protein particle) complex has a core set of proteins which are joined by specific subunits depending on the cellular component where a given TRAPP complex is active." [GOC:bhm, GOC:vw, PMID:22669257]
GO:0030014
name: CCR4-NOT complex
def: "The evolutionarily conserved CCR4-NOT complex is involved in several aspects of mRNA metabolism, including repression and activation of mRNA initiation, control of mRNA elongation, and the deadenylation and subsequent degradation of mRNA. In Saccharomyces the CCR4-NOT complex comprises a core complex of 9 proteins (Ccr4p, Caf1p, Caf40p, Caf130p, Not1p, Not2p, Not3p, Not4p, and Not5p), Caf4p, Caf16p, and several less well characterized proteins." [GOC:sart, PMID:11113136]
GO:0030015
name: CCR4-NOT core complex
def: "The core of the CCR4-NOT complex. In Saccharomyces the CCR4-NOT core complex comprises Ccr4p, Caf1p, Caf40p, Caf130p, Not1p, Not2p, Not3p, Not4p, and Not5p." [GOC:sart, PMID:11113136]
GO:0030016
name: myofibril
def: "The contractile element of skeletal and cardiac muscle; a long, highly organized bundle of actin, myosin, and other proteins that contracts by a sliding filament mechanism." [ISBN:0815316194]
GO:0030017
name: sarcomere
def: "The repeating unit of a myofibril in a muscle cell, composed of an array of overlapping thick and thin filaments between two adjacent Z discs." [ISBN:0815316194]
GO:0030018
name: Z disc
def: "Platelike region of a muscle sarcomere to which the plus ends of actin filaments are attached." [GOC:mtg_muscle, ISBN:0815316194]
GO:0030027
name: lamellipodium
def: "A thin sheetlike process extended by the leading edge of a crawling fibroblast; contains a dense meshwork of actin filaments." [ISBN:0815316194]
GO:0030054
name: cell junction
def: "A cellular component that forms a specialized region of connection between two cells or between a cell and the extracellular matrix. At a cell junction, anchoring proteins extend through the plasma membrane to link cytoskeletal proteins in one cell to cytoskeletal proteins in neighboring cells or to proteins in the extracellular matrix." [GOC:mah, http://www.vivo.colostate.edu/hbooks/cmb/cells/pmemb/junctions_a.html, ISBN:0198506732]
GO:0030055
name: cell-substrate junction
def: "A cell junction that forms a connection between a cell and the extracellular matrix." [GOC:hb, GOC:mah]
GO:0030056
name: hemidesmosome
def: "A cell-substrate junction that forms a point of contact between the basal surface of epithelial cells and the basal lamina. Morphologically resembles desmosomes; attached to intermediate filaments." [ISBN:0815316208]
GO:0030057
name: desmosome
def: "A cell-cell junction in which: on the cytoplasmic surface of each interacting plasma membrane is a dense plaque composed of a mixture of intracellular anchor proteins; a bundle of keratin intermediate filaments is attached to the surface of each plaque; transmembrane adhesion proteins of the cadherin family bind to the plaques and interact through their extracellular domains to hold the adjacent membranes together by a Ca2+-dependent mechanism." [GOC:mah, GOC:mtg_muscle, ISBN:0815332181]
GO:0030061
name: mitochondrial crista
def: "Any of the inward folds of the mitochondrial inner membrane. Their number, extent, and shape differ in mitochondria from different tissues and organisms. They appear to be devices for increasing the surface area of the mitochondrial inner membrane, where the enzymes of electron transport and oxidative phosphorylation are found. Their shape can vary with the respiratory state of the mitochondria." [ISBN:0198506732]
GO:0030062
name: mitochondrial tricarboxylic acid cycle enzyme complex
def: "Any of the heteromeric enzymes, located in the mitochondrion, that act in the tricarboxylic acid (TCA) cycle." [GOC:mah, GOC:mtg_sensu]
GO:0030063
name: murein sacculus
def: "OBSOLETE. A structure formed of the cross-linked polymer peptidoglycan (also called murein) that forms a covalently closed net around a bacterial cell, and imparts structural stability to the bacterial cell wall." [GOC:mah, PMID:9529891]
GO:0030064
name: cell wall inner membrane
def: "OBSOLETE. The peptidoglycan layer of the cell wall of Gram-negative bacteria." [ISBN:0135712254]
GO:0030074
name: thylakoid (sensu Proteobacteria)
def: "OBSOLETE. A thylakoid; as in, but not restricted to, the purple bacteria and relatives (Proteobacteria, ncbi_taxonomy_id:1224)." [GOC:mah]
GO:0030075
name: bacterial thylakoid
def: "A thylakoid that is derived from and attached to, but not necessarily continuous with, the plasma membrane, and is not enclosed in a plastid. It bears the photosynthetic pigments in photosynthetic cyanobacteria." [GOC:mah, GOC:mtg_sensu]
GO:0030076
name: light-harvesting complex
def: "A protein-pigment complex that may be closely or peripherally associated to photosynthetic reaction centers that participate in harvesting and transferring radiant energy to the reaction center." [GOC:lr]
GO:0030077
name: plasma membrane light-harvesting complex
def: "A plasma membrane protein-pigment complex that may be closely or peripherally associated to photosynthetic reaction centers that participate in harvesting and transferring radiant energy to the reaction center. Examples of this complex are found in bacterial species." [GOC:mah, GOC:mtg_sensu]
GO:0030078
name: light-harvesting complex, core complex
def: "Light harvesting complex associated with the reaction complex of photosynthetic purple bacteria." [GOC:lr]
GO:0030079
name: light-harvesting complex, peripheral complex
def: "Bacteriochlorophyll a binding complex that is peripherally associated to the bacterial reaction center." [GOC:lr]
GO:0030080
name: B875 antenna complex
def: "Protein complex that surrounds and transfers excitation energy directly to the bacterial reaction center; binds bacteriochlorophyll a and has a single absorption band between 870 and 890 nm." [GOC:kd]
GO:0030081
name: B800-820 antenna complex
def: "Protein-pigment complex that absorbs light at 800 and 820 nm; is peripherally associated to the bacterial reaction center; transfers excitation energy to the B875 antenna complex." [GOC:kd, GOC:lr]
GO:0030082
name: B800-850 antenna complex
def: "Protein-pigment complex that absorbs light at 800 and 850 nm; is peripherally associated to the bacterial reaction center; transfers excitation energy to the B875 antenna complex." [GOC:kd, GOC:lr]
GO:0030083
name: PSI associated light-harvesting complex I, LHCIa subcomplex
def: "A pigment protein complex that forms part of the photosystem I associated light-harvesting complex I; contains two proteins (usually about 24 and 21.5 kDa); has a fluorescence maximum between 680 and 690 nm." [PMID:8825475]
GO:0030084
name: PSI associated light-harvesting complex I, LHCIb subcomplex
def: "A pigment protein complex that forms part of the photosystem I associated light-harvesting complex I; contains two proteins (usually about 20 kDa); has a fluorescence maximum of 730 nm." [PMID:8825475]
GO:0030085
name: PSII associated light-harvesting complex II, peripheral complex, LHCIIb subcomplex
def: "A pigment protein complex that forms part of the photosystem II associated light-harvesting complex II; contains two proteins (usually about 28 and 27 kDa), and may contain a third; peripherally located relative to other LHC polypeptides." [PMID:8825475]
GO:0030086
name: PSII associated light-harvesting complex II, core complex, LHCIIa subcomplex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0030087
name: PSII associated light-harvesting complex II, core complex, LHCIIc subcomplex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0030088
name: PSII associated light-harvesting complex II, core complex, LHCIId subcomplex
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0030089
name: phycobilisome
def: "Any of the granules, approximately 32 nm x 48 nm and consisting of highly aggregated phycobiliproteins, that are attached in arrays to the external face of a thylakoid membrane in algae of the phyla Cyanophyta and Rhodophyta, where they function as light-harvesting devices in photosynthesis. Excitation energy in the phycobilisome flows in the sequence: phycoerythrin, phycocyanin, allophycocyanin before passing to the antenna chlorophyll of photosystem II." [GOC:jl, PMID:11734882, Wikipedia:Phycobilisome]
GO:0030093
name: chloroplast photosystem I
def: "Photosystem located in the chloroplast that functions as a light-dependent plastocyanin-ferredoxin oxidoreductase, transferring electrons from plastocyanin to ferredoxin. An example of this is found in Arabidopsis thaliana." [GOC:jid, GOC:mtg_sensu]
GO:0030094
name: plasma membrane-derived photosystem I
def: "A protein complex located in the plasma membrane-derived thylakoid. The photosystem functions as a light-dependent plastocyanin-ferredoxin oxidoreductase, transferring electrons from plastocyanin to ferredoxin. Examples of this complex are found in bacterial species." [GOC:jid, GOC:mtg_sensu]
GO:0030095
name: chloroplast photosystem II
def: "An integral chloroplast membrane complex containing the P680 reaction center. In the light, PSII functions as a water-plastoquinone oxidoreductase, transferring electrons from water to plastoquinone." [GOC:jid, GOC:mtg_sensu]
GO:0030096
name: plasma membrane-derived thylakoid photosystem II
def: "A protein complex, located in the membrane-derived thylakoid, containing the P680 reaction center. In the light, PSII functions as a water-plastoquinone oxidoreductase, transferring electrons from water to plastoquinone." [GOC:jid, GOC:mtg_sensu]
GO:0030112
name: glycocalyx
def: "A viscous, carbohydrate rich layer at the outermost periphery of a cell." [GOC:mlg, ISBN:0815316208]
GO:0030114
name: slime layer
def: "A slime layer is an easily removed, diffuse, unorganized layer of extracellular material that surrounds a cell. Specifically this consists mostly of exopolysaccharides, glycoproteins, and glycolipids." [GOC:mlg, Wikipedia:Slime_layer]
GO:0030115
name: S-layer
def: "A crystalline protein layer surrounding some bacteria." [GOC:mlg, ISBN:0815108893]
GO:0030117
name: membrane coat
def: "Any of several different proteinaceous coats that can associate with membranes. Membrane coats include those formed by clathrin plus an adaptor complex, the COPI and COPII complexes, and possibly others. They are found associated with membranes on many vesicles as well as other membrane features such as pits and perhaps tubules." [GOC:mah]
GO:0030118
name: clathrin coat
def: "A membrane coat found on coated pits and some coated vesicles; consists of polymerized clathrin triskelions, each comprising three clathrin heavy chains and three clathrin light chains, linked to the membrane via one of the AP adaptor complexes." [GOC:mah, PMID:11252894, PMID:9531549]
GO:0030119
name: AP-type membrane coat adaptor complex
def: "Any of several heterotetrameric complexes that link clathrin (or another coat-forming molecule, as hypothesized for AP-3 and AP-4) to a membrane surface; they are found on coated pits and coated vesicles, and mediate sorting of cargo proteins into vesicles. Each AP complex contains two large (a beta and one of either an alpha, gamma, delta, or epsilon) subunits (110-130 kDa), a medium (mu) subunit (approximately 50 kDa), and a small (sigma) subunit (15-20 kDa)." [GOC:mah, PMID:10611976, PMID:15473838]
GO:0030120
name: vesicle coat
def: "A membrane coat found on a coated vesicle." [GOC:mah]
GO:0030121
name: AP-1 adaptor complex
def: "A heterotetrameric AP-type membrane coat adaptor complex that consists of beta1, gamma, mu1 and sigma1 subunits and links clathrin to the membrane surface of a vesicle; vesicles with AP-1-containing coats are normally found primarily in the trans-Golgi network. In at least humans, the AP-1 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different genes (gamma1 and gamma2, mu1A and mu1B, and sigma1A, sigma1B and sigma1C)." [GOC:mah, PMID:10611976, PMID:21097499]
GO:0030122
name: AP-2 adaptor complex
def: "A heterotetrameric AP-type membrane coat adaptor complex that consists of alpha, beta2, mu2 and sigma2 subunits, and links clathrin to the membrane surface of a vesicle; vesicles with AP-2-containing coats are normally found primarily near the plasma membrane, on endocytic vesicles. In at least humans, the AP-2 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different alpha genes (alphaA and alphaC)." [GOC:mah, PMID:10611976, PMID:21097499]
GO:0030123
name: AP-3 adaptor complex
def: "A heterotetrameric AP-type membrane coat adaptor complex that consists of beta3, delta, mu3 and sigma3 subunits and is found associated with endosomal membranes. AP-3 does not appear to associate with clathrin in all organisms. In at least humans, the AP-3 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different genes (beta3A and beta3B, mu3A and mu3B, and sigma3A and sigma3B)." [GOC:mah, PMID:10611976, PMID:21097499]
GO:0030124
name: AP-4 adaptor complex
def: "An AP-type membrane coat adaptor complex that consists of beta4, epsilon, mu4 and sigma4 subunits and is found associated with membranes in the trans-Golgi network; it is not clear whether AP-4 forms clathrin coats in vivo." [GOC:mah, PMID:10611976]
GO:0030125
name: clathrin vesicle coat
def: "A clathrin coat found on a vesicle." [GOC:mah]
GO:0030126
name: COPI vesicle coat
def: "One of two multimeric complexes that forms a membrane vesicle coat. The mammalian COPI subunits are called alpha-, beta-, beta'-, gamma-, delta-, epsilon- and zeta-COP. Vesicles with COPI coats are found associated with Golgi membranes at steady state." [GOC:mah, PMID:11252894]
GO:0030127
name: COPII vesicle coat
def: "One of two multimeric complexes that forms a membrane vesicle coat. COPII is best characterized in S. cerevisiae, where the subunits are called Sar1p, Sec13p, Sec31p, Sec23p, and Sec24p. Vesicles with COPII coats are found associated with endoplasmic reticulum (ER) membranes at steady state." [GOC:ascb_2009, GOC:dph, GOC:mah, GOC:tb, PMID:11252894]
GO:0030128
name: clathrin coat of endocytic vesicle
def: "A clathrin coat found on an endocytic vesicle." [GOC:mah]
GO:0030129
name: clathrin coat of synaptic vesicle
def: "A clathrin coat found on a synaptic vesicle." [GOC:mah]
GO:0030130
name: clathrin coat of trans-Golgi network vesicle
def: "A clathrin coat found on a vesicle of the trans-Golgi network." [GOC:mah]
GO:0030131
name: clathrin adaptor complex
def: "A membrane coat adaptor complex that links clathrin to a membrane." [GOC:mah]
GO:0030132
name: clathrin coat of coated pit
def: "The coat found on coated pits and the coated vesicles derived from coated pits; comprises clathrin and the AP-2 adaptor complex." [GOC:mah]
GO:0030133
name: transport vesicle
def: "Any of the vesicles of the constitutive secretory pathway, which carry cargo from the endoplasmic reticulum to the Golgi, between Golgi cisternae, or to destinations within or outside the cell." [GOC:mah]
GO:0030134
name: ER to Golgi transport vesicle
def: "A vesicle that mediates transport from the endoplasmic reticulum to the Golgi complex; bears a coat formed of the COPII coat complex proteins; such vesicles found associated with endoplasmic reticulum (ER) membranes at steady state, and are involved in ER to Golgi (anterograde) vesicle transport." [GOC:ascb_2009, GOC:dph, GOC:mah, GOC:tb, PMID:11252894]
GO:0030135
name: coated vesicle
def: "Small membrane-bounded organelle formed by pinching off of a coated region of membrane. Some coats are made of clathrin, whereas others are made from other proteins." [ISBN:0815316194]
GO:0030136
name: clathrin-coated vesicle
def: "A vesicle with a coat formed of clathrin connected to the membrane via one of the clathrin adaptor complexes." [GOC:mah, PMID:11252894]
GO:0030137
name: COPI-coated vesicle
def: "A vesicle with a coat formed of the COPI coat complex proteins. COPI-coated vesicles are found associated with Golgi membranes at steady state, are involved in Golgi to endoplasmic reticulum (retrograde) vesicle transport, and possibly also in intra-Golgi transport." [GOC:mah, PMID:11252894]
GO:0030139
name: endocytic vesicle
def: "A membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance. Endocytic vesicles fuse with early endosomes to deliver the cargo for further sorting." [GOC:go_curators, PMID:19696797]
GO:0030140
name: trans-Golgi network transport vesicle
def: "A vesicle that mediates transport between the trans-Golgi network and other parts of the cell." [GOC:mah]
GO:0030141
name: secretory granule
def: "A small subcellular vesicle, surrounded by a membrane, that is formed from the Golgi apparatus and contains a highly concentrated protein destined for secretion. Secretory granules move towards the periphery of the cell and upon stimulation, their membranes fuse with the cell membrane, and their protein load is exteriorized. Processing of the contained protein may take place in secretory granules." [GOC:mah, ISBN:0198596732]
GO:0030142
name: Golgi to ER transport vesicle
def: "A vesicle that mediates transport from the Golgi to the endoplasmic reticulum." [GOC:mah]
GO:0030143
name: inter-Golgi transport vesicle
def: "A vesicle that mediates transport of cargo within the Golgi complex (for example, between cisternae of the Golgi stack)." [GOC:mah]
GO:0030173
name: integral component of Golgi membrane
def: "The component of the Golgi membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:go_curators]
GO:0030175
name: filopodium
def: "Thin, stiff protrusion extended by the leading edge of a motile cell such as a crawling fibroblast or amoeba, or an axonal or dendritic growth cone, or a dendritic shaft." [GOC:mah, GOC:pr, ISBN:0815316194]
GO:0030176
name: integral component of endoplasmic reticulum membrane
def: "The component of the endoplasmic reticulum membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0030232
name: insulin control element activator complex
def: "Transcription factor complex that binds to the insulin control element (ICE), a DNA sequence element found within the 5'-flanking region of the insulin gene, and activates ICE-mediated transcription." [PMID:7935390]
GO:0030256
name: type I protein secretion system complex
def: "A complex of three secretory proteins that carry out secretion in the type I secretion system: an inner membrane transport ATPase (termed ABC protein for ATP-binding cassette), which provides the energy for protein secretion; an outer membrane protein, which is exported via the sec pathway; and a membrane fusion protein, which is anchored in the inner membrane and spans the periplasmic space." [PMID:9618447]
GO:0030257
name: type III protein secretion system complex
def: "A complex of approximately 20 proteins, most of which are located in the cytoplasmic membrane that carries out protein secretion in the bacterial type III secretion system; type III secretion also requires a cytoplasmic, probably membrane-associated ATPase." [PMID:9618447]
GO:0030285
name: integral component of synaptic vesicle membrane
def: "The component of the synaptic vesicle membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:go_curators]
GO:0030286
name: dynein complex
def: "Any of several large complexes that contain two or three dynein heavy chains and several light chains, and have microtubule motor activity." [ISBN:0815316194]
GO:0030287
name: cell wall-bounded periplasmic space
def: "The region between the plasma membrane and the cell wall, as found in organisms such as yeast and Gram positive bacteria. The region is thinner than the equivalent in Gram negative bacteria." [GOC:mlg, GOC:mtg_sensu]
GO:0030288
name: outer membrane-bounded periplasmic space
def: "The region between the inner (cytoplasmic or plasma) membrane and outer membrane of organisms with two membranes such as Gram negative bacteria. These periplasmic spaces are relatively thick and contain a thin cell wall." [GOC:mlg, GOC:mtg_sensu]
GO:0030289
name: protein phosphatase 4 complex
def: "The complex formed by the catalytic subunit of protein phosphatase 4 plus a regulatory subunit." [PMID:10026142]
GO:0030312
name: external encapsulating structure
def: "A structure that lies outside the plasma membrane and surrounds the entire cell." [GOC:go_curators]
GO:0030313
name: cell envelope
def: "An envelope that surrounds a bacterial cell and includes the cytoplasmic membrane and everything external, encompassing the periplasmic space, cell wall, and outer membrane if present." [GOC:ds, GOC:mlg, http://pathmicro.med.sc.edu/fox/cell_envelope.htm]
GO:0030314
name: junctional membrane complex
def: "Complex formed in muscle cells between the membrane of the sarcoplasmic reticulum and invaginations of the plasma membrane (T-tubules)." [PMID:11535622]
GO:0030315
name: T-tubule
def: "Invagination of the plasma membrane of a muscle cell that extends inward from the cell surface around each myofibril. The ends of T-tubules make contact with the sarcoplasmic reticulum membrane." [GOC:mtg_muscle, ISBN:0815316194]
GO:0030386
name: ferredoxin:thioredoxin reductase complex
def: "A protein complex that possesses ferredoxin:thioredoxin reductase activity." [GOC:mah]
GO:0030424
name: axon
def: "The long process of a neuron that conducts nerve impulses, usually away from the cell body to the terminals and varicosities, which are sites of storage and release of neurotransmitter." [GOC:nln, ISBN:0198506732]
GO:0030425
name: dendrite
def: "A neuron projection that has a short, tapering, often branched, morphology, receives and integrates signals from other neurons or from sensory stimuli, and conducts a nerve impulse towards the axon or the cell body. In most neurons, the impulse is conveyed from dendrites to axon via the cell body, but in some types of unipolar neuron, the impulse does not travel via the cell body." [GOC:dos, GOC:mah, GOC:nln, ISBN:0198506732]
GO:0030426
name: growth cone
def: "The migrating motile tip of a growing nerve cell axon or dendrite." [ISBN:0815316194]
GO:0030427
name: site of polarized growth
def: "Any part of a cell where non-isotropic growth takes place." [GOC:mah]
GO:0030428
name: cell septum
def: "A structure composed of peptidoglycan and often chitin in addition to other materials. It usually forms perpendicular to the long axis of a cell or hypha and grows centripetally from the cell wall to the center of the cell and often functions in the compartmentalization of a cell into two daughter cells." [GOC:clt, ISBN:0471940526]
GO:0030430
name: host cell cytoplasm
def: "The cytoplasm of a host cell." [GOC:mah]
GO:0030445
name: yeast-form cell wall
def: "The wall surrounding a cell of a dimorphic fungus growing in the single-cell budding yeast form, in contrast to the filamentous or hyphal form." [GOC:mah, GOC:mcc]
GO:0030446
name: hyphal cell wall
def: "The cell wall surrounding a fungal hypha." [GOC:mah]
GO:0030478
name: actin cap
def: "Polarized accumulation of cytoskeletal proteins (including F-actin) and regulatory proteins in a cell. An example of this is the actin cap found in Saccharomyces cerevisiae." [GOC:mah, ISBN:10652251]
GO:0030479
name: actin cortical patch
def: "An endocytic patch  that consists of an actin-containing structure found at the plasma membrane in cells; formed of networks of branched actin filaments that lie just beneath the plasma membrane and assemble, move, and disassemble rapidly. An example of this is the actin cortical patch found in Saccharomyces cerevisiae." [GOC:mah, GOC:vw, ISBN:0879693568, ISBN:0879693649, PMID:16959963]
GO:0030484
name: muscle fiber
def: "OBSOLETE. The contractile fibers, composed of actin, myosin, and associated proteins, found in cells of smooth or striated muscle." [GOC:mah, ISBN:0815316194]
GO:0030485
name: smooth muscle contractile fiber
def: "The contractile fiber of smooth muscle cells." [GOC:mah]
GO:0030486
name: smooth muscle dense body
def: "Electron-dense region associated with a smooth muscle contractile fiber." [GOC:mah, ISBN:0815316194]
GO:0030496
name: midbody
def: "A thin cytoplasmic bridge formed between daughter cells at the end of cytokinesis. The midbody forms where the contractile ring constricts, and may persist for some time before finally breaking to complete cytokinesis." [ISBN:0815316194]
GO:0030526
name: granulocyte macrophage colony-stimulating factor receptor complex
def: "The heterodimeric receptor for granulocyte macrophage colony-stimulating factor." [GOC:mah]
GO:0030529
name: ribonucleoprotein complex
def: "A macromolecular complex containing both protein and RNA molecules." [GOC:krc]
GO:0030530
name: heterogeneous nuclear ribonucleoprotein complex
def: "OBSOLETE. Particulate complex of heterogeneous nuclear RNA (hnRNA; a heterogeneous mixture of RNA molecules of high Mr with a rapid turnover rate that occurs in cell nuclei during protein synthesis; it is the form of RNA synthesized in eukaryotes by RNA polymerase II, that which is translated into protein) with protein, which is cell-specific and heterogeneous. The protein component may play a role in the processing of the hnRNA to mRNA." [ISBN:0198506732]
GO:0030531
name: small cytoplasmic ribonucleoprotein complex
def: "OBSOLETE. A complex composed of RNA of the small cytoplasmic RNA (scRNA) class and protein, found in the cytoplasm." [GOC:krc, GOC:mah]
GO:0030532
name: small nuclear ribonucleoprotein complex
def: "A complex composed of RNA of the small nuclear RNA (snRNA) class and protein, found in the nucleus of a eukaryotic cell. These are typically named after the snRNA(s) they contain, e.g. U1 snRNP or U4/U6 snRNP. Many, but not all, of these complexes are involved in splicing of nuclear mRNAs." [GOC:krc, GOC:mah, ISBN:0879695897]
GO:0030658
name: transport vesicle membrane
def: "The lipid bilayer surrounding a transport vesicle." [GOC:mah]
GO:0030659
name: cytoplasmic vesicle membrane
def: "The lipid bilayer surrounding a cytoplasmic vesicle." [GOC:mah]
GO:0030660
name: Golgi-associated vesicle membrane
def: "The lipid bilayer surrounding a vesicle associated with the Golgi apparatus." [GOC:mah]
GO:0030661
name: chitosome membrane
def: "The lipid bilayer surrounding a chitosome." [GOC:mah]
GO:0030662
name: coated vesicle membrane
def: "The lipid bilayer surrounding a coated vesicle." [GOC:mah]
GO:0030663
name: COPI-coated vesicle membrane
def: "The lipid bilayer surrounding a COPI-coated vesicle." [GOC:mah]
GO:0030665
name: clathrin-coated vesicle membrane
def: "The lipid bilayer surrounding a clathrin-coated vesicle." [GOC:mah]
GO:0030666
name: endocytic vesicle membrane
def: "The lipid bilayer surrounding an endocytic vesicle." [GOC:mah]
GO:0030667
name: secretory granule membrane
def: "The lipid bilayer surrounding a secretory granule." [GOC:mah]
GO:0030668
name: merozoite dense granule membrane
def: "The lipid bilayer surrounding a dense granule of the type found in apicomplexan parasites." [GOC:mah, GOC:mtg_sensu]
GO:0030669
name: clathrin-coated endocytic vesicle membrane
def: "The lipid bilayer surrounding a clathrin-coated endocytic vesicle." [GOC:mah]
GO:0030670
name: phagocytic vesicle membrane
def: "The lipid bilayer surrounding a phagocytic vesicle." [GOC:mah]
GO:0030671
name: clathrin-coated phagocytic vesicle membrane
def: "The lipid bilayer surrounding a clathrin-coated phagocytic vesicle." [GOC:mah]
GO:0030672
name: synaptic vesicle membrane
def: "The lipid bilayer surrounding a synaptic vesicle." [GOC:mah]
GO:0030673
name: axolemma
def: "The portion of the plasma membrane surrounding an axon; it is a specialized trilaminar random mosaic of protein molecules floating within a fluid matrix of highly mobile phospholipid molecules, 7-8 nm in thickness." [http://www.medik.sk/clanky/bio_jun.htm, ISBN:0124325653]
GO:0030677
name: ribonuclease P complex
def: "A ribonucleoprotein complex that catalyzes cleavage of the leader sequence of precursor tRNAs (pre-tRNAs), generating the mature 5' end of tRNAs." [GOC:mah, PMID:12045094]
GO:0030678
name: mitochondrial ribonuclease P complex
def: "A ribonuclease P complex located in the mitochondrion of a eukaryotic cell, where it catalyzes the 5' endonucleolytic cleavage of precursor tRNAs to yield mature tRNAs. The subunit composition of mitochondrial ribonuclease P complexes varies between species, but the complex often contains a single RNA molecule and a single protein molecule." [GOC:mah, PMID:12045094]
GO:0030679
name: cyanelle ribonuclease P complex
def: "A ribonuclease P complex located in the cyanelle, where it catalyzes the 5' endonucleolytic cleavage of precursor tRNAs to yield mature tRNAs. The best characterized cyanelle ribonuclease P complex, from the alga Cyanophora paradoxa, contains a single RNA molecule that is necessary but not sufficient for catalysis, and several protein molecules." [GOC:mah, PMID:12045094]
GO:0030680
name: dimeric ribonuclease P complex
def: "A ribonuclease P complex that contains a single RNA molecule that is necessary and usually sufficient for catalysis, and a single protein molecule. Examples of this complex are found in Bacterial species." [GOC:mah, PMID:12045094]
GO:0030681
name: multimeric ribonuclease P complex
def: "A ribonuclease P complex that generally contains a single RNA molecule and several protein molecules. Examples of this complex are found in Archaeal species." [GOC:mah, PMID:11142368, PMID:12045094]
GO:0030684
name: preribosome
def: "Any complex of pre-rRNAs, ribosomal proteins, and associated proteins formed during ribosome biogenesis." [PMID:10567516]
GO:0030685
name: nucleolar preribosome
def: "Any complex of pre-rRNAs, ribosomal proteins, and associated proteins formed in the nucleolus during ribosome biogenesis." [PMID:10567516]
GO:0030686
name: 90S preribosome
def: "A large ribonucleoprotein complex considered to be the earliest preribosomal complex. In S. cerevisiae, it has a size of 90S and consists of the 35S pre-rRNA, early-associating ribosomal proteins most of which are part of the small ribosomal subunit, the U3 snoRNA and associated proteins." [GOC:krc, GOC:vw, PMID:12150911, PMID:12957375, PMID:15120992]
GO:0030687
name: preribosome, large subunit precursor
def: "A preribosomal complex consisting of 27SA, 27SB, and/or 7S pre-rRNA, 5S rRNA, ribosomal proteins including late-associating large subunit proteins, and associated proteins; a precursor of the eukaryotic cytoplasmic large ribosomal subunit." [PMID:10567516]
GO:0030688
name: preribosome, small subunit precursor
def: "A preribosomal complex consisting of 20S pre-rRNA, ribosomal proteins including late-associating small subunit proteins, and associated proteins; a precursor of the eukaryotic cytoplasmic small ribosomal subunit." [PMID:10567516]
GO:0030689
name: Noc complex
def: "Any of several heterodimers containing one or two Noc proteins, associated with preribosomal complexes; involved in ribosome biogenesis." [PMID:12446671]
GO:0030690
name: Noc1p-Noc2p complex
def: "A heterodimer associated with 90S and 66S preribosomes. Predominantly, but not exclusively, nucleolar; involved in ribosomal large subunit biogenesis." [PMID:12446671]
GO:0030691
name: Noc2p-Noc3p complex
def: "A heterodimer associated with 66S preribosomes; predominantly nucleoplasmic, but also locates to the nucleolus; involved in ribosomal large subunit biogenesis." [PMID:12446671]
GO:0030692
name: Noc4p-Nop14p complex
def: "A heterodimer associated with precursors of the eukaryotic small ribosomal subunit, including the 90S preribosome; involved in small subunit biogenesis." [PMID:12446671]
GO:0030694
name: bacterial-type flagellum basal body, rod
def: "The central portion of the flagellar basal body, which spans the periplasm and threads through the rings. Examples of this component are found in Bacterial species." [GOC:mtg_sensu, PMID:10572114, PMID:11133968, PMID:12624192]
GO:0030700
name: glycine reductase complex
def: "Complex that possesses glycine reductase activity; usually comprises three subunits, of which two are selenoproteins; the subunits are typically designated selenoprotein A, selenoprotein B and protein C." [GOC:mah, PMID:2018775]
GO:0030849
name: autosome
def: "Any chromosome other than a sex chromosome." [GOC:mah]
GO:0030863
name: cortical cytoskeleton
def: "The portion of the cytoskeleton that lies just beneath the plasma membrane." [GOC:mah]
GO:0030864
name: cortical actin cytoskeleton
def: "The portion of the actin cytoskeleton, comprising filamentous actin and associated proteins, that lies just beneath the plasma membrane." [GOC:mah]
GO:0030867
name: rough endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the rough endoplasmic reticulum." [GOC:mah]
GO:0030868
name: smooth endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the smooth endoplasmic reticulum." [GOC:mah]
GO:0030869
name: RENT complex
def: "A protein complex that mediates transcriptional silencing at the rDNA locus (the name derives from regulator of nucleolar silencing and telophase). In Saccharomyces the complex contains Net1p, Sir2p, Cdc14p, and at least one more subunit." [PMID:12196389]
GO:0030870
name: Mre11 complex
def: "Trimeric protein complex that possesses endonuclease activity; involved in meiotic recombination, DNA repair and checkpoint signaling. In Saccharomyces cerevisiae, the complex comprises Mre11p, Rad50p, and Xrs2p; complexes identified in other species generally contain proteins orthologous to the Saccharomyces cerevisiae proteins." [GOC:mah, GOC:vw, PMID:11988766, PMID:17674145]
GO:0030874
name: nucleolar chromatin
def: "The portion of nuclear chromatin associated with the nucleolus; includes the DNA encoding the ribosomal RNA." [GOC:mah]
GO:0030875
name: rDNA protrusion
def: "Any of the tandem arrays of rDNA localized at the periphery of the nucleus and protruding into the nucleolus, and associated proteins. May be visible as a single or double spot by DAPI staining." [PMID:1629244]
GO:0030876
name: interleukin-20 receptor complex
def: "A protein complex that binds interleukin-20; comprises an alpha and a beta subunit." [GOC:mah]
GO:0030877
name: beta-catenin destruction complex
def: "A cytoplasmic protein complex containing glycogen synthase kinase-3-beta (GSK-3-beta), the adenomatous polyposis coli protein (APC), and the scaffolding protein axin, among others; phosphorylates beta-catenin, targets it for degradation by the proteasome." [PMID:14600025]
GO:0030880
name: RNA polymerase complex
def: "Any complex that possesses RNA polymerase activity; generally comprises a catalytic subunit and one or more additional subunits." [GOC:mah]
GO:0030891
name: VCB complex
def: "A protein complex that possesses ubiquitin ligase activity; the complex is usually pentameric; for example, in mammals the subunits are pVHL, elongin B, elongin C, cullin-2 (Cul2), and Rbx1." [GOC:mah, PMID:11865071]
GO:0030892
name: mitotic cohesin complex
def: "A cohesin complex that mediates sister chromatid cohesion during mitosis; has a subunit composition distinct from that of the meiotic cohesin complex." [GOC:mah, PMID:12750522]
GO:0030893
name: meiotic cohesin complex
def: "A cohesin complex that mediates sister chromatid cohesion during meiosis; has a subunit composition distinct from that of the mitotic cohesin complex." [GOC:mah, PMID:12750522]
GO:0030894
name: replisome
def: "A multi-component enzymatic machine at the replication fork which mediates DNA replication. Includes DNA primase, one or more DNA polymerases, DNA helicases, and other proteins." [GOC:mah, GOC:vw]
GO:0030895
name: apolipoprotein B mRNA editing enzyme complex
def: "Protein complex that mediates editing of the mRNA encoding apolipoprotein B; catalyzes the deamination of C to U (residue 6666 in the human mRNA). Contains a catalytic subunit, APOBEC-1, and other proteins (e.g. human ASP; rat ASP and KSRP)." [PMID:10781591]
GO:0030896
name: checkpoint clamp complex
def: "Conserved heterotrimeric complex of PCNA-like proteins that is loaded onto DNA at sites of DNA damage." [PMID:12531008]
GO:0030897
name: HOPS complex
def: "A multimeric protein complex that associates with the vacuolar membrane, late endosomal (multivesicular body) and lysosomal membranes. HOPS is a tethering complex involved in vesicle fusion." [PMID:10944212, PMID:23645161]
GO:0030904
name: retromer complex
def: "A conserved multimeric membrane-associated complex involved in retrograde transport from endosomes to the Golgi apparatus. For example, the budding yeast retromer comprises Vps35p, Vps29p, Vps26p, Vps5p, and Vps17p." [PMID:9700157]
GO:0030905
name: retromer complex, outer shell
def: "One of two subcomplexes that combine to form the retromer, believed to be peripherally associated with the membrane. The budding yeast complex comprises Vps5p and Vps17p, and may contain multiple copies of a Vps5p/Vps17p dimer." [PMID:9700157]
GO:0030906
name: retromer complex, inner shell
def: "One of two subcomplexes that combine to form the retromer, believed to be closely associated with the membrane. In budding yeast the complex comprises Vps35p, Vps29p, and Vps26p." [PMID:9700157]
GO:0030907
name: MBF transcription complex
def: "A protein complex that binds to the Mlu1 cell cycle box (MCB) promoter element, consensus sequence ACGCGN, and is involved in regulation of transcription during the G1/S transition of the cell cycle. In Saccharomyces, the complex contains a heterodimer of the DNA binding protein Mbp1p and the activator Swi4p, and is associated with additional proteins known as Nrm1p, Msa1p, and Msa2p; in Schizosaccharomyces the complex contains Res1p, Res2p, and Cdc10p." [GOC:mah, PMID:11206552, PMID:15838511, PMID:18160399, PMID:9343385]
GO:0030914
name: STAGA complex
def: "A large multiprotein complex that possesses histone acetyltransferase and is involved in regulation of transcription. The composition is similar to that of the SAGA complex; for example, the human complex contains the transcription-transformation cofactor TRRAP, hGCN5L acetylase, novel human ADA-like and SPT-like cofactors, and a subset of TAFs." [PMID:11564863]
GO:0030915
name: Smc5-Smc6 complex
def: "A conserved complex that contains a heterodimer of SMC proteins (Smc5p and Smc6p, or homologs thereof) and several other proteins, and is involved in DNA repair and maintaining cell cycle arrest following DNA damage. In S. cerevisiae, this is an octameric complex called Mms21-Smc5-Smc6 complex, with at least five of its subunits conserved in fission yeast and humans." [GOC:rb, PMID:14701739]
GO:0030929
name: ADPG pyrophosphorylase complex
def: "Complex that possesses ADPG pyrophosphorylase activity. In all organisms where it has been found, the complex is a tetramer. In bacteria, it is a homotetramer. In plants, the complex is a heterotetramer composed small and large subunits." [GOC:tb, PMID:9680965]
GO:0030930
name: homotetrameric ADPG pyrophosphorylase complex
def: "A protein complex composed of four identical subunits that possesses ADPG pyrophosphorylase activity. Examples of this component are found in Bacterial species." [GOC:mah, GOC:mtg_sensu]
GO:0030931
name: heterotetrameric ADPG pyrophosphorylase complex
def: "A protein complex composed of four different subunits that possesses ADPG pyrophosphorylase activity. An example of this process is found in Mus musculus." [GOC:mah, GOC:mtg_sensu]
GO:0030932
name: amyloplast ADPG pyrophosphorylase complex
def: "An ADPG pyrophosphorylase complex found in the amyloplast." [GOC:mah]
GO:0030933
name: chloroplast ADPG pyrophosphorylase complex
def: "An ADPG pyrophosphorylase complex found in the chloroplast." [GOC:mah]
GO:0030934
name: anchoring collagen complex
def: "Any collagen complex which links one collagen assembly, such as a collagen fibril or sheet, to other structures." [ISBN:0721639976]
GO:0030935
name: sheet-forming collagen trimer
def: "sheet-forming collagen trimer" [ISBN:0721639976]
GO:0030936
name: transmembrane collagen trimer
def: "Any collagen trimer that passes through a lipid bilayer membrane." [ISBN:0721639976]
GO:0030937
name: collagen type XVII trimer
def: "A collagen homotrimer of alpha1(XVII) chains; type XVII collagen triple helices span the plasma membrane and associate with hemidesmosomes and the basal lamina where they bind laminin." [ISBN:0721639976, PMID:19693541, PMID:21421911]
GO:0030938
name: collagen type XVIII trimer
def: "A collagen homotrimer of alpha1(XVIII) chains." [ISBN:0721639976, PMID:21421911]
GO:0030956
name: glutamyl-tRNA(Gln) amidotransferase complex
def: "A protein complex that possesses glutamyl-tRNA(Gln) amidotransferase activity, and therefore creates Gln-tRNA by amidating Glu-tRNA; usually composed of 3 subunits: A, B, and C. Note that the C subunit may not be required in all organisms." [GOC:mlg]
GO:0030958
name: RITS complex
def: "A protein complex required for heterochromatin assembly; contains an Argonaute homolog, a chromodomain protein, and at least one additional protein; named for RNA-induced initiation of transcriptional gene silencing." [PMID:14704433]
GO:0030964
name: NADH dehydrogenase complex
def: "An integral membrane complex that possesses NADH oxidoreductase activity. The complex is one of the components of the electron transport chain. It catalyzes the transfer of a pair of electrons from NADH to a quinone." [GOC:mah]
GO:0030981
name: cortical microtubule cytoskeleton
def: "The portion of the microtubule cytoskeleton that lies just beneath the plasma membrane." [GOC:mah]
GO:0030988
name: high molecular weight kininogen receptor complex
def: "A protein complex that acts as a receptor for high molecular weight kininogens. In humans, this receptor includes the CK1 and uPAR proteins." [GOC:mah, PMID:11290596]
GO:0030990
name: intraciliary transport particle
def: "A nonmembrane-bound oligomeric protein complex that participates in bidirectional transport of molecules (cargo) along axonemal microtubules." [GOC:cilia, GOC:kmv, PMID:14570576, PMID:22118932, PMID:23945166]
GO:0030991
name: intraciliary transport particle A
def: "The smaller subcomplex of the intraciliary transport particle; characterized complexes have molecular weights of 710-760 kDa." [GOC:cilia, GOC:kmv, PMID:14570576]
GO:0030992
name: intraciliary transport particle B
def: "The larger subcomplex of the intraciliary transport particle; characterized complexes have molecular weights around 550 kDa." [GOC:cilia, GOC:kmv, PMID:14570576, PMID:19253336]
GO:0030993
name: axonemal heterotrimeric kinesin-II complex
def: "A kinesin complex found in eukaryotic axonemes that contains two distinct plus end-directed kinesin motor proteins and at least one accessory subunit, and that functions in the anterograde transport of molecules (cargo) from the basal body to the distal tip of the axoneme." [GOC:kmv, PMID:14570576]
GO:0030998
name: linear element
def: "A proteinaceous scaffold associated with S. pombe chromosomes during meiotic prophase. Linear elements have a structure related to but not equivalent to the synaptonemal complex." [GOC:jb]
GO:0031002
name: actin rod
def: "A cellular structure consisting of parallel, hexagonally arranged actin tubules, comprising filamentous actin and associated proteins. Found in the germinating spores of Dictyostelium discoideum." [GOC:kp]
GO:0031003
name: actin tubule
def: "A cellular structure, approximately 13 nm in diameter, consisting of three actin filaments bundled together." [GOC:kp]
GO:0031004
name: potassium ion-transporting ATPase complex
def: "Protein complex that carries out the reaction: ATP + H2O + K+(out) = ADP + phosphate + K+(in). It is a high affinity potassium uptake system. The E. coli complex consists of 4 proteins: KdpA is the potassium ion translocase, KdpB is the ATPase, and KdpC and KdpF seem to be involved in assembly and stabilization of the complex." [PMID:10608856, PMID:9858692, TIGR_TIGRFAMS:TIGR01497]
GO:0031009
name: plastid ADPG pyrophosphorylase complex
def: "An ADPG pyrophosphorylase complex found in a plastid." [GOC:mah]
GO:0031010
name: ISWI-type complex
def: "Any nuclear protein complex that contains an ATPase subunit of the imitation switch (ISWI) family. ISWI ATPases are involved in assembling chromatin and in sliding and spacing nucleosomes to regulate transcription of nuclear RNA polymerases I, II, and III and also DNA replication, recombination and repair." [GOC:krc, GOC:mah, PMID:15020051, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0031011
name: Ino80 complex
def: "A multisubunit protein complex that contains the Ino80p ATPase; exhibits chromatin remodeling activity and 3' to 5' DNA helicase activity." [GOC:jh, GOC:rb, PMID:19355820]
GO:0031012
name: extracellular matrix
def: "A structure lying external to one or more cells, which provides structural support for cells or tissues; may be completely external to the cell (as in animals and bacteria) or be part of the cell (as in plants)." [GOC:mah, NIF_Subcellular:nlx_subcell_20090513]
GO:0031015
name: karyopherin docking complex
def: "OBSOLETE. A subcomplex of the nuclear pore complex that interacts with karyopherin-cargo complexes; a well-characterized example in Saccharomyces contains Asm4p, Nup53p, and Nup170p." [PMID:11867631, PMID:9864357]
GO:0031019
name: mitochondrial mRNA editing complex
def: "An mRNA editing complex found in the mitochondrion. The best characterized example is that of Trypanosoma brucei, which catalyzes the insertion and deletion of uridylates." [GOC:mah, PMID:12139607]
GO:0031020
name: plastid mRNA editing complex
def: "An mRNA editing complex found in a plastid." [GOC:mah]
GO:0031021
name: interphase microtubule organizing center
def: "A microtubule organizing center found in interphase cells, which organize a longitudinal array of three to five MT bundles from the nuclear envelope during interphase. Each MT bundle is composed of two to seven MTs arranged in an antiparallel configuration, with the dynamic MT plus ends extending toward the cell tips and stable minus ends near the nucleus." [PMID:15068790]
GO:0031026
name: glutamate synthase complex
def: "A complex that possesses glutamate synthase activity." [GOC:mah]
GO:0031027
name: glutamate synthase complex (NADH)
def: "A protein complex that in yeast consists of a large and a small subunit. Possesses glutamate synthase (NADH) activity." [GOC:jl, PMID:7047525]
GO:0031039
name: macronucleus
def: "A membrane-bounded organelle of ciliated protozoan cells that contains polyploid copies of a portion of the cell's complete genome. Transcription of genes occurs in macronuclei. Some ciliate species may contain multiple macronuclei per cell." [GOC:ns]
GO:0031040
name: micronucleus
def: "A membrane-bounded organelle of ciliated protozoan cells that contains a diploid copy of the cell's complete genome. Sections of contiguous sequence in the macronucleus are often interrupted by internal eliminated sequences (IES), and may be permuted, in micronuclei. Genic transcription is not found in micronuclei. Some ciliate species may contain multiple micronuclei per cell." [GOC:ns]
GO:0031045
name: dense core granule
def: "Electron-dense organelle with a granular internal matrix; contains proteins destined to be secreted." [NIF_Subcellular:sao772007592, PMID:14690495]
GO:0031074
name: nucleocytoplasmic shuttling complex
def: "Any complex that acts to move proteins or RNAs into or out of the nucleus through nuclear pores." [GOC:mah]
GO:0031080
name: nuclear pore outer ring
def: "A subcomplex of the nuclear pore complex (NPC) that forms the outer rings of the core scaffold, a lattice-like structure that gives the NPC its shape and strength. In S. cerevisiae, the two outer rings each contain multiple copies of the following proteins: Nup133p, Nup120p, Nup145Cp, Nup85p, Nup84p, Seh1p, and Sec13p. In vertebrates, the two outer rings each contain multiple copies of the following proteins: Nup133, Nup160, Nup96, Nup75, Nup107, Seh1, Sec13, Nup43, Nup37, and ALADIN. Components are arranged in 8-fold symmetrical 'spokes' around the central transport channel. A single 'spoke', can be isolated and is sometimes referred to as the Nup84 complex (S. cerevisiae) or the Nup107-160 complex (vertebrates)." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0031082
name: BLOC complex
def: "Any of several protein complexes required for the biogenesis of specialized organelles of the endosomal-lysosomal system, such as melanosomes, platelet dense granules, and other related organelles; acronym for biogenesis of lysosomal-related organelles complex." [PMID:15102850, PMID:15261680]
GO:0031083
name: BLOC-1 complex
def: "A protein complex required for the biogenesis of specialized organelles of the endosomal-lysosomal system, such as melanosomes and platelet dense granules. Many of the protein subunits are conserved between mouse and human; the mouse complex contains the Pallidin, Muted, Cappuccino, Dysbindin, Snapin, BLOS1, BLOS2, AND BLOS3 proteins." [PMID:15102850]
GO:0031084
name: BLOC-2 complex
def: "A protein complex required for the biogenesis of specialized organelles of the endosomal-lysosomal system, such as melanosomes and platelet dense granules. The human complex contains the Hps3, Hps5, and Hps6 proteins; the mouse complex contains ru2 and ru." [PMID:12548288, PMID:14718540, PMID:15031569]
GO:0031085
name: BLOC-3 complex
def: "A protein complex required for the biogenesis of specialized organelles of the endosomal-lysosomal system, such as melanosomes and platelet dense granules. The human complex contains the Hps1 and Hps4 proteins." [PMID:12756248]
GO:0031088
name: platelet dense granule membrane
def: "The lipid bilayer surrounding the platelet dense granule." [GOC:mah]
GO:0031089
name: platelet dense granule lumen
def: "The volume enclosed by the membrane of the platelet dense granule." [GOC:mah]
GO:0031090
name: organelle membrane
def: "A membrane that is one of the two lipid bilayers of an organelle envelope or the outermost membrane of single membrane bound organelle." [GOC:dos, GOC:mah]
GO:0031091
name: platelet alpha granule
def: "A secretory organelle found in blood platelets, which is unique in that it exhibits further compartmentalization and acquires its protein content via two distinct mechanisms: (1) biosynthesis predominantly at the megakaryocyte (MK) level (with some vestigial platelet synthesis) (e.g. platelet factor 4) and (2) endocytosis and pinocytosis at both the MK and circulating platelet levels (e.g. fibrinogen (Fg) and IgG)." [PMID:8467233]
GO:0031092
name: platelet alpha granule membrane
def: "The lipid bilayer surrounding the platelet alpha granule." [GOC:mah, PMID:8467233]
GO:0031093
name: platelet alpha granule lumen
def: "The volume enclosed by the membrane of the platelet alpha granule." [GOC:mah, PMID:8467233]
GO:0031094
name: platelet dense tubular network
def: "A network of membrane-bounded compartments found in blood platelets, where they regulate platelet activation by sequestering or releasing calcium. The dense tubular network exists as thin elongated membranes in resting platelets, and undergoes a major ultrastructural change, to a rounded vesicular form, upon addition of thrombin." [PMID:1322202]
GO:0031095
name: platelet dense tubular network membrane
def: "The lipid bilayer surrounding the platelet dense tubular network." [GOC:mah, PMID:1322202]
GO:0031096
name: platelet dense tubular network lumen
def: "The volume enclosed by the membranes of the platelet dense tubular network." [GOC:mah, PMID:1322202]
GO:0031097
name: medial cortex
def: "A medial cortical band overlaying the nucleus which acts as a landmark for contractile ring positioning and plays a role in cell cycle regulation." [GOC:vw, PMID:15572668, PMID:19474789]
GO:0031105
name: septin complex
def: "A protein complex containing septins.  Typically, these complexes contain multiple septins and are oligomeric." [GOC:mah, PMID:15385632]
GO:0031143
name: pseudopodium
def: "A temporary protrusion or retractile process of a cell, associated with flowing movements of the protoplasm, and serving for locomotion and feeding." [ISBN:0198506732]
GO:0031160
name: spore wall
def: "The specialized envelope lying outside the cell membrane of a spore." [GOC:mah, GOC:pg]
GO:0031164
name: contractile vacuolar membrane
def: "The lipid bilayer surrounding the contractile vacuole." [GOC:pg]
GO:0031165
name: integral component of contractile vacuolar membrane
def: "The component of the contractile vacuolar membrane consisting of gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:pg]
GO:0031166
name: integral component of vacuolar membrane
def: "The component of the vacuolar membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. May also refer to the state of being buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031201
name: SNARE complex
def: "A protein complex involved in membrane fusion; a stable ternary complex consisting of a four-helix bundle, usually formed from one R-SNARE and three Q-SNAREs with an ionic layer sandwiched between hydrophobic layers. One well-characterized example is the neuronal SNARE complex formed of synaptobrevin 2, syntaxin 1a, and SNAP-25." [GOC:bhm, GOC:pr, PMID:10872468, PMID:19450911]
GO:0031205
name: endoplasmic reticulum Sec complex
def: "An endoplasmic reticulum membrane-associated complex involved in the translocation of proteins that are targeted to the ER. In yeast, this complex consists of two subcomplexes, namely, the Sec61 complex and the Sec62/Sec63 complex." [GOC:mtg_sensu, PMID:12158317, PMID:14617809]
GO:0031207
name: Sec62/Sec63 complex
def: "A protein complex involved in the posttranslational targeting of proteins to the ER. In yeast, it is a tetrameric complex consisting of Sec62p, Sec63p, Sec71p and Sec72p." [PMID:12518317, PMID:14617809]
GO:0031209
name: SCAR complex
def: "A heterotetrameric complex that includes orthologues of human PIR121, Nap125 and HSPC300 and regulates actin polymerization and/or depolymerization through small GTPase mediated signal transduction." [GOC:pg, PMID:12181570]
GO:0031211
name: endoplasmic reticulum palmitoyltransferase complex
def: "A dimeric complex of the endoplasmic reticulum that catalyzes S-palmitoylation, the addition of palmitate (C16:0) or other long-chain fatty acids to proteins at a cysteine residue." [GOC:jh]
GO:0031213
name: RSF complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (SNF2H in mammals) and an RSF1 homolog. It mediates nucleosome deposition and generates regularly spaced nucleosome arrays. In mammals, RSF is involved in regulation of transcription from RNA polymerase II promoters)." [GOC:krc, PMID:12972596, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0031224
name: intrinsic component of membrane
def: "The component of a membrane consisting of the gene products having some covalently attached portion, for example part of a peptide sequence or some other covalently attached group such as a GPI anchor, which spans or is embedded in one or both leaflets of the membrane." [GOC:mah]
GO:0031225
name: anchored component of membrane
def: "The component of a membrane consisting of the gene products that are tethered to the membrane only by a covalently attached anchor, such as a lipid group that is embedded in the membrane. Gene products with peptide sequences that are embedded in the membrane are excluded from this grouping." [GOC:dos, GOC:mah]
GO:0031226
name: intrinsic component of plasma membrane
def: "The component of the plasma membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031227
name: intrinsic component of endoplasmic reticulum membrane
def: "The component of the endoplasmic reticulum membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031228
name: intrinsic component of Golgi membrane
def: "The component of the Golgi membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031229
name: intrinsic component of nuclear inner membrane
def: "The component of the nuclear inner membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031230
name: intrinsic component of cell outer membrane
def: "The component of the cell outer membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031231
name: intrinsic component of peroxisomal membrane
def: "The component of the peroxisomal membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031232
name: extrinsic component of external side of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that are loosely bound to its external surface, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031233
name: intrinsic component of external side of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that penetrate the external side of the plasma membrane only, either directly or via some covalently attached hydrophobic anchor." [GOC:dos, GOC:mah]
GO:0031234
name: extrinsic component of cytoplasmic side of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that are loosely bound to its cytoplasmic surface, but not integrated into the hydrophobic region." [GOC:mah]
GO:0031235
name: intrinsic component of the cytoplasmic side of the plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor) which is embedded in the cytoplasmic side of the plasma membrane only." [GOC:dos, GOC:mah]
GO:0031236
name: extrinsic component of periplasmic side of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that are loosely bound to its periplasmic surface, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031237
name: intrinsic component of periplasmic side of plasma membrane
def: "The component of a plasma membrane consisting of gene products and protein complexes that penetrate the periplasmic side of the plasma membrane only, either directly or via some covalently attached hydrophobic anchor." [GOC:mah, GOC:mtg_sensu]
GO:0031240
name: external side of cell outer membrane
def: "The side of the outer membrane that is opposite to the side that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
GO:0031241
name: periplasmic side of cell outer membrane
def: "The side (leaflet) of the outer membrane that faces the periplasm of the cell." [GOC:mlg, GOC:mtg_sensu]
GO:0031242
name: extrinsic component of external side of cell outer membrane
def: "The component of a cell outer membrane consisting of gene products and protein complexes that are loosely bound to its external surface, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031243
name: intrinsic component of external side of cell outer membrane
def: "The component of the cell outer membrane consisting of the gene products and protein complexes that penetrate the external side of the cell outer membrane only, either directly or via some covalently attached hydrophobic anchor." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031244
name: extrinsic component of cell outer membrane
def: "The component of the cell outer membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031245
name: extrinsic component of periplasmic side of cell outer membrane
def: "The component of the cell outer membrane consisting of gene products and protein complexes that are loosely bound to periplasmic surface, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031246
name: intrinsic component of periplasmic side of cell outer membrane
def: "The component of the cell outer membrane consisting of the gene products that that penetrate the preiplasmic side of the cell outer membrane only, either directly or via some covalently attached hydrophobic anchor." [GOC:dos, GOC:mah, GOC:mtg_sensu]
GO:0031248
name: protein acetyltransferase complex
def: "A complex that catalyzes the transfer of an acetyl group to a protein acceptor molecule." [GOC:bf]
GO:0031250
name: anaerobic ribonucleoside-triphosphate reductase complex
def: "An enzyme complex composed of 4 subunits, 2 copies of the large protein (nrdD in E. coli) and 2 copies of the small protein (nrdG in E. coli). It catalyzes the generation of 2'deoxyribonucleotides under anaerobic growth conditions. The larger subunit is the catalytic unit that is activated by the smaller iron-binding subunit." [GOC:mlg]
GO:0031251
name: PAN complex
def: "A complex that possesses poly(A)-specific ribonuclease activity; catalyzes the message-specific shortening of mRNA poly(A) tails. Contains at least two subunits, known as Pan2p and Pan3p in Saccharomyces." [PMID:9774670]
GO:0031252
name: cell leading edge
def: "The area of a motile cell closest to the direction of movement." [GOC:pg]
GO:0031253
name: cell projection membrane
def: "The portion of the plasma membrane surrounding a cell surface projection." [GOC:mah]
GO:0031254
name: cell trailing edge
def: "The area of a motile cell opposite to the direction of movement." [GOC:pg]
GO:0031255
name: lateral part of motile cell
def: "The area of a motile cell perpendicular to the direction of movement." [GOC:pg, GOC:pr]
GO:0031256
name: leading edge membrane
def: "The portion of the plasma membrane surrounding the leading edge of a motile cell." [GOC:mah]
GO:0031257
name: cell trailing edge membrane
def: "The portion of the plasma membrane surrounding the trailing edge of a motile cell." [GOC:mah]
GO:0031258
name: lamellipodium membrane
def: "The portion of the plasma membrane surrounding a lamellipodium." [GOC:mah]
GO:0031259
name: uropod membrane
def: "The portion of the plasma membrane surrounding a uropod." [GOC:mah]
GO:0031260
name: pseudopodium membrane
def: "The portion of the plasma membrane surrounding a pseudopodium." [GOC:mah]
GO:0031261
name: DNA replication preinitiation complex
def: "A protein-DNA complex assembled at eukaryotic DNA replication origins immediately prior to the initiation of DNA replication. The preinitiation complex is formed by the assembly of additional proteins onto an existing prereplicative complex. In budding yeast, the additional proteins include Cdc45p, Sld2p, Sld3p, Dpb11p, DNA polymerases, and others; in fission yeast the GINS complex is present." [GOC:bf, GOC:hjd, GOC:jl, GOC:pr, GOC:rb, GOC:vw, PMID:12694535, PMID:15194812]
GO:0031262
name: Ndc80 complex
def: "A protein complex conserved among eukaryotes that forms part of the kinetochore and plays an essential role in forming stable kinetochore-microtubule attachments. The complex contains proteins known in several species, including budding and fission yeasts, as Ndc80p, Nuf2p, Spc24p, and Spc25p. In vertebrates it is part of the outer plate of the kinetochore." [PMID:15509863, PMID:15661517]
GO:0031264
name: death-inducing signaling complex
def: "A protein complex formed by the association of signaling proteins with a death receptor upon ligand binding. The complex includes procaspases and death domain-containing proteins in addition to the ligand-bound receptor, and may control the activation of caspases 8 and 10." [GOC:mtg_apoptosis, PMID:12628743, PMID:12655293, PMID:8521815]
GO:0031265
name: CD95 death-inducing signaling complex
def: "A protein complex formed upon binding of Fas/CD95/APO-1 to its ligand. The complex includes FADD/Mort1, procaspase-8/10 and c-FLIP in addition to the ligand-bound receptor." [PMID:12628743, PMID:12655293]
GO:0031266
name: TRAIL death-inducing signaling complex
def: "A protein complex formed upon binding of TRAIL to its ligand. The complex includes FADD/Mort1 and procaspase-8 addition to the ligand-bound receptor." [PMID:12628743, PMID:12655293]
GO:0031298
name: replication fork protection complex
def: "A protein complex conserved in eukaryotes and associated with the replication fork; the complex stabilizes stalled replication forks and is thought to be involved in coordinating leading- and lagging-strand synthesis and in replication checkpoint signaling." [PMID:15367656]
GO:0031300
name: intrinsic component of organelle membrane
def: "The component of the organelle membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031301
name: integral component of organelle membrane
def: "The component of the organelle membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031302
name: intrinsic component of endosome membrane
def: "The component of the endosome membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031303
name: integral component of endosome membrane
def: "The component of the endosome membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031304
name: intrinsic component of mitochondrial inner membrane
def: "The component of the mitochondrial inner membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031305
name: integral component of mitochondrial inner membrane
def: "The component of the mitochondrial inner membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031306
name: intrinsic component of mitochondrial outer membrane
def: "The component of the mitochondrial outer membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031307
name: integral component of mitochondrial outer membrane
def: "The component of the mitochondrial outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031308
name: intrinsic component of nuclear outer membrane
def: "The component of the nuclear outer membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031309
name: integral component of nuclear outer membrane
def: "The component of the nuclear outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031310
name: intrinsic component of vacuolar membrane
def: "The component of the vacuolar membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031311
name: intrinsic component of contractile vacuolar membrane
def: "The component of the contractile vacuolar membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031312
name: extrinsic component of organelle membrane
def: "The component of an organelle membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031313
name: extrinsic component of endosome membrane
def: "The component of an endosome membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031314
name: extrinsic component of mitochondrial inner membrane
def: "The component of mitochondrial inner membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031315
name: extrinsic component of mitochondrial outer membrane
def: "The component of a mitochondrial outer membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031316
name: extrinsic component of nuclear outer membrane
def: "The component of a nuclear outer membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0031317
name: tripartite ATP-independent periplasmic transporter complex
def: "A complex consisting of two membrane proteins and one extracytoplasmic solute receptor. Such transporters transport a variety of substrates without direct ATP power, instead using energy from ion gradients." [GOC:mlg]
GO:0031332
name: RNAi effector complex
def: "Any protein complex that mediates the effects of small interfering RNAs on gene expression. Most known examples contain one or more members of the Argonaute family of proteins." [GOC:mah, PMID:14704433]
GO:0031350
name: intrinsic component of plastid membrane
def: "The component of the plastid membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031351
name: integral component of plastid membrane
def: "The component of the plastid membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031352
name: intrinsic component of plastid inner membrane
def: "The component of the plastid inner membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031353
name: integral component of plastid inner membrane
def: "The component of the plastid inner membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031354
name: intrinsic component of plastid outer membrane
def: "The component of the plastid outer membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031355
name: integral component of plastid outer membrane
def: "The component of the plastid outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031356
name: intrinsic component of chloroplast inner membrane
def: "The component of the chloroplast inner membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031357
name: integral component of chloroplast inner membrane
def: "The component of the chloroplast inner membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031358
name: intrinsic component of chloroplast outer membrane
def: "The component of the chloroplast outer membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031359
name: integral component of chloroplast outer membrane
def: "The component of the chloroplast outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031360
name: intrinsic component of thylakoid membrane
def: "The component of the thylakoid membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0031361
name: integral component of thylakoid membrane
def: "The component of the thylakoid membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0031362
name: anchored component of external side of plasma membrane
def: "The component of the plasma membrane consisting of the gene products that are tethered to the external side of the membrane only by a covalently attached anchor, such as a lipid group embedded in the membrane. Gene products with peptide sequences that are embedded in the membrane are excluded from this grouping." [GOC:dos, GOC:mah]
GO:0031371
name: ubiquitin conjugating enzyme complex
def: "Any complex that possesses ubiquitin conjugating enzyme activity." [GOC:mah]
GO:0031372
name: UBC13-MMS2 complex
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains, which act as a signal to promote error-free DNA postreplication repair; in Saccharomyces the complex comprises Ubc13p and Mms2p." [GOC:mah, PMID:15772086]
GO:0031375
name: type II fatty acid synthase complex
def: "OBSOLETE A fatty acid synthase complex in which each polypeptide chain catalyzes a single activity." [GOC:mah, ISBN:0471331309, ISBN:0716720094, PMID:12957385]
GO:0031376
name: cytosolic type II fatty acid synthase complex
def: "OBSOLETE A fatty acid synthase complex in which each polypeptide chain catalyzes a single activity, located in the cytosol." [GOC:mah, ISBN:0471331309, ISBN:0716720094, PMID:12957385]
GO:0031377
name: mitochondrial type II fatty acid synthase complex
def: "OBSOLETE A fatty acid synthase complex in which each polypeptide chain catalyzes a single activity, located in the mitochondrion." [GOC:mah, ISBN:0471331309, ISBN:0716720094, PMID:12957385]
GO:0031378
name: plastid type II fatty acid synthase complex
def: "OBSOLETE A fatty acid synthase complex in which each polypeptide chain catalyzes a single activity, located in a plastid." [GOC:mah, ISBN:0471331309, ISBN:0716720094, PMID:12957385]
GO:0031379
name: RNA-directed RNA polymerase complex
def: "A protein complex that possesses RNA-directed RNA polymerase activity." [GOC:mah]
GO:0031380
name: nuclear RNA-directed RNA polymerase complex
def: "A complex required for RNAi mediated heterochromatin assembly. In S. pombe this contains RNA-directed RNA polymerase, a putative helicase and a protein containing a pap25 associated domain." [GOC:vw, PMID:15607976]
GO:0031381
name: viral RNA-directed RNA polymerase complex
def: "A virus-specific protein complex that possesses RNA-dependent RNA polymerase activity and replicates the genome of an RNA virus." [GOC:mah, PMID:15574411, PMID:15613301]
GO:0031387
name: MPF complex
def: "A complex consisting of a Cdc2-class (also known as Cdc28) cyclin-dependent kinase and an M-phase cyclin such as S. pombe Cdc13. The MPF complex phosphorylates and activates the anaphase promoting complex (APC)." [PMID:12045216]
GO:0031389
name: Rad17 RFC-like complex
def: "A pentameric protein complex related to replication factor C, which loads a trimeric complex of checkpoint proteins (known as the checkpoint clamp or 9-1-1 complex) onto DNA at damage sites; functions in DNA damage cell cycle checkpoints. In Schizosaccharomyces pombe the subunits are known as Rad17, Rfc2, Rfc3, Rfc4, and Rfc5, while in Saccharomyces cerevisiae the subunits are known as Rad24p, Rfc2p, Rfc3p, Rfc4p, and Rfc5p." [PMID:14614842]
GO:0031390
name: Ctf18 RFC-like complex
def: "A heptameric complex related to replication factor C, which loads the DNA polymerase processivity factor proliferating cell nuclear antigen (PCNA) onto DNA and plays a vital role in chromosome cohesion. In Saccharomyces the subunits are known as Ctf18p, Rfc2p, Rfc3p, Rfc4p, Rfc5p, Dcc1p, and Ctf8p." [PMID:14614842]
GO:0031391
name: Elg1 RFC-like complex
def: "A pentameric complex related to replication factor C, which loads the DNA polymerase processivity factor proliferating cell nuclear antigen (PCNA) onto DNA and has roles in telomere length regulation and other aspects of genome stability. In Saccharomyces the subunits are known as Elg1p, Rfc2p, Rfc3p, Rfc4p, and Rfc5p." [PMID:14614842]
GO:0031395
name: bursicon neuropeptide hormone complex
def: "A neuropeptide hormone secreted by the central nervous system of insects that stimulates the tanning and sclerotization of the adult cuticle following eclosion. The active hormone consists of an obligate heterodimer of the alpha and beta subunits." [GOC:rc]
GO:0031410
name: cytoplasmic vesicle
def: "A vesicle formed of membrane or protein, found in the cytoplasm of a cell." [GOC:mah]
GO:0031411
name: gas vesicle
def: "A hollow structure made of protein, which usually has the form of a cylindrical tube closed by conical end caps. By regulating their relative gas vesicle content aquatic microbes are able to perform vertical migrations." [PMID:8177173]
GO:0031414
name: N-terminal protein acetyltransferase complex
def: "A complex that catalyzes the transfer of an acetyl group to the N-terminal residue of a protein acceptor molecule." [GOC:mah]
GO:0031415
name: NatA complex
def: "A conserved complex that catalyzes the transfer of an acetyl group to an N-terminal Ser, Ala, Gly, or Thr residue of a protein acceptor molecule. In Saccharomyces the complex includes Nat1p and Ard1p, and may contain additional proteins." [PMID:12890471]
GO:0031416
name: NatB complex
def: "A conserved complex that catalyzes the transfer of an acetyl group to the N-terminal residue of a protein acceptor molecule that has a Met-Glu, Met-Asp, Met-Asn, or Met-Met N-terminus. In Saccharomyces the complex includes Nat3p and Mdm20p." [PMID:12890471]
GO:0031417
name: NatC complex
def: "A conserved complex that catalyzes the transfer of an acetyl group to the N-terminal residue of a protein acceptor molecule that has a Met-Ile, Met-Leu, Met-Trp, or Met-Phe N-terminus. In Saccharomyces the complex includes Mak3p, Mak10p, and Mak31p." [PMID:12890471]
GO:0031421
name: invertasome
def: "A complex formed by a recombinase, a regulatory protein, and the DNA sequences bound by each protein; catalyzes a reversible site-specific recombination reaction that results in the alternate expression of one or more genes in various contexts." [PMID:11114897, PMID:9732277]
GO:0031422
name: RecQ helicase-Topo III complex
def: "A complex containing a RecQ family helicase and a topoisomerase III homologue; may also include one or more additional proteins; conserved from E. coli to human." [PMID:15889139]
GO:0031428
name: box C/D snoRNP complex
def: "A ribonucleoprotein complex containing small nucleolar RNA of the box C/D type that can carry out ribose-2'-O-methylation of target RNAs." [ISBN:0879695897, PMID:17284456]
GO:0031429
name: box H/ACA snoRNP complex
def: "A box H/ACA RNP complex that is located in the nucleolus." [GOC:vw, ISBN:0879695897, PMID:17284456, PMID:20227365]
GO:0031430
name: M band
def: "The midline of aligned thick filaments in a sarcomere; location of specific proteins that link thick filaments. Depending on muscle type the M band consists of different numbers of M lines." [GOC:mtg_muscle, ISBN:0198506732, ISBN:0815316194]
GO:0031431
name: Dbf4-dependent protein kinase complex
def: "A heterodimeric protein complex required for the activation of DNA replication origins; comprises a catalytic subunit and a regulatory subunit (in Saccharomyces, Cdc7p and Dbf4p, respectively); complexes identified in other species generally contain proteins related to the Saccharomyces proteins." [PMID:12045100]
GO:0031436
name: BRCA1-BARD1 complex
def: "A heterodimeric complex comprising BRCA1 and BARD1, which possesses ubiquitin ligase activity and is involved in genome maintenance, possibly by functioning in surveillance for DNA damage." [PMID:12787778]
GO:0031461
name: cullin-RING ubiquitin ligase complex
def: "Any ubiquitin ligase complex in which the catalytic core consists of a member of the cullin family and a RING domain protein; the core is associated with one or more additional proteins that confer substrate specificity." [PMID:15571813, PMID:15688063]
GO:0031462
name: Cul2-RING ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul2 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by an elongin-BC adaptor and a SOCS/BC box protein." [PMID:15571813, PMID:15688063]
GO:0031463
name: Cul3-RING ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul3 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a BTB-domain-containing protein." [PMID:15571813, PMID:15688063]
GO:0031464
name: Cul4A-RING E3 ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul4A subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by an adaptor protein." [PMID:15571813, PMID:15688063]
GO:0031465
name: Cul4B-RING E3 ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul4B subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by unknown subunits." [PMID:15571813, PMID:15688063]
GO:0031466
name: Cul5-RING ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul5 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by an elongin-BC adaptor and a SOCS/BC box protein." [PMID:15571813, PMID:15688063]
GO:0031467
name: Cul7-RING ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul7 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 linker and an F-box protein." [PMID:15571813, PMID:15688063]
GO:0031469
name: polyhedral organelle
def: "An organelle found in bacteria consisting of a proteinaceous coat containing metabolic enzymes whose purpose is the sequestration or concentration of metabolites and which has the appearance of a polygonal granule by electron microscopy." [GOC:js, PMID:10498708, PMID:11844753, PMID:12923081]
GO:0031470
name: carboxysome
def: "An organelle found in the Cyanobacteria consisting of a proteinaceous coat and enzymes for the fixation of carbon dioxide including mechanisms for the concentration of carbonate to increase the efficiency of fixation under low-carbon dioxide conditions." [GOC:js, PMID:8157606, PMID:8491708]
GO:0031471
name: ethanolamine degradation polyhedral organelle
def: "An organelle found in bacteria consisting of a proteinaceous coat containing enzymes for the degradation of ethanolamine whose purpose is the protection of the rest of the cell from the toxic acetaldehyde product of the enzyme ethanolamine ammonia lyase." [GOC:js, PMID:11844753]
GO:0031472
name: propanediol degradation polyhedral organelle
def: "An organelle found in bacteria consisting of a proteinaceous coat containing enzymes for the degradation of 1,2-propanediol whose purpose is the protection of the rest of the cell from the toxic propionaldehyde product of the enzyme diol dehydratase." [GOC:js, PMID:10498708, PMID:11844753, PMID:12923081]
GO:0031474
name: myosin IV complex
def: "A myosin complex containing one or more class IV myosin heavy chains and associated light chains; myosin IV is relatively uncharacterized, but is predicted to have a single motor domain, one IQ motif and a tail with a Myosin Tail Homology (myTH4) domain homologous to that in the tails of myosins VII and XV." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031475
name: myosin V complex
def: "A myosin complex containing a dimer of class V myosin heavy chains and associated light chains; involved in intracellular transport. Myosin V is a dimeric molecule consisting of conserved motor domains followed by 6 IQ motifs which bind specific light chains and calmodulin. The tail domain is important for cellular localization and cargo binding and can be divided into an alpha-helical coiled coil region and a C-terminal globular region." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031476
name: myosin VI complex
def: "A myosin complex containing one or more class VI myosin heavy chains and associated light chains. Myosin VI has a single IQ motif in the neck and a tail region with a coiled coil domain followed by a unique globular domain; a unique insertion that enables myosin VI to move towards the pointed or minus end of actin filaments." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031477
name: myosin VII complex
def: "A myosin complex containing a dimer of class VII myosin heavy chains and associated light chains. Myosin VII (240 kDa) is predicted to be a dimeric molecule with 5 IQ motifs and a tail region with a short stretch of coiled coil followed by two myosin-tail homology (MyTH4) domains, two talin-binding (FERM) domains and an SH3-domain." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031478
name: myosin VIII complex
def: "A myosin complex containing a dimer of class VIII myosin heavy chains and associated light chains. Myosin VIII is predicted to be dimeric, and contain an unusual 100-190 residue N-terminal extension prior to their motor domains, 3-4 IQ motifs, a short region (~70 residues) of predicted alpha-helical coiled coil and a C-terminal domain." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031479
name: myosin IX complex
def: "A myosin complex containing a class IX myosin heavy chain and associated light chains. Myosin IX is monomeric with a motor domain containing an N-terminal extension and an insert in the actin binding interface, followed by four to six IQ motifs and a tail region that contains a zinc binding motif and a domain with homology to GTPase activating proteins (GAPs) of the Rho family of G-proteins." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031480
name: myosin X complex
def: "A myosin complex containing one or more class X myosin heavy chains and associated light chains." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031481
name: myosin XI complex
def: "A myosin complex containing a dimer of class XI myosin heavy chains and associated light chains. Myosin XI heavy chain sizes are similar in molecular structure to the class V myosins with 5 to 6 IQ motifs and tail regions with predicted coiled coil domains (forming dimeric molecules) and large C-terminal regions." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031482
name: myosin XII complex
def: "A myosin complex containing one or more class XII myosin heavy chains and associated light chains; myosin XII contains a large tail region with two MyTH4 domains and a short region of coiled coil." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031483
name: myosin XIII complex
def: "A myosin complex containing one or more class XIII myosin heavy chains and associated light chains." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031484
name: myosin XIV complex
def: "A myosin complex containing a class XIV myosin heavy chain and associated light chains; myosin XIV heavy chains are the simplest known, containing a motor domain, no classic IQ motif and variable length tails." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031485
name: myosin XV complex
def: "A myosin complex containing a class XV myosin heavy chain and associated light chains. Myosin XV is single headed, and has a large extension (1200aa) at the N-terminus of the motor domain, two IQ motifs and a tail with a similar domain structure to that of the tail of myosin VII." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031486
name: myosin XVI complex
def: "A myosin complex containing a class XVI myosin heavy chains and associated light chains; myosin XVI heavy chains contain ankyrin repeat." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html, PMID:11294886]
GO:0031487
name: myosin XVII complex
def: "A myosin complex containing one or more class XVII myosin heavy chains and associated light chains." [http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031488
name: myosin XVIII complex
def: "A myosin complex containing a class XVIII myosin heavy chain and associated light chains; myosin XVIII heavy chains contain an N-terminal PDZ domain." [PMID:11294886]
GO:0031499
name: TRAMP complex
def: "A multiprotein complex having distributive polyadenylation activity of a variety of RNA substrates including hypomodified and incorrectly folded tRNAs, pre-snRNAs, pre-snoRNAs, incorrectly spliced or processed pre-mRNAs, cryptic unstable transcripts (CUTs), pre-rRNAs and rRNA fragments released as part of rRNA processing. In S. cerevisiae, the complex consists of either Pap2 (also known as Trf4) or Trf5, Air1 or Air2, and Mtr4, and is involved in RNA 3'-end processing and in RNA surveillance and quality control." [PMID:15173578, PMID:15828860, PMID:15935758, PMID:15935759, PMID:16373491, PMID:16374505, PMID:16431988, PMID:16973437, PMID:17410208, PMID:17652137]
GO:0031500
name: Tea1 cell-end complex
def: "A high molecular weight complex characterized in S. pombe containing the cell-end anchoring protein Tea1. This complex is transported to the cell ends by microtubules and is involved in bipolar growth and the maintennce of normal cell polarity." [PMID:15936270]
GO:0031501
name: mannosyltransferase complex
def: "A complex that posseses mannosyltransferase activity." [GOC:mah]
GO:0031502
name: dolichyl-phosphate-mannose-protein mannosyltransferase complex
def: "A complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity; usually includes members of the PMT1 and PMT2 protein subfamilies." [GOC:mah, GOC:pr, PMID:15948957]
GO:0031510
name: SUMO activating enzyme complex
def: "A conserved heterodimeric complex with SUMO activating enzyme activity." [PMID:15601841]
GO:0031511
name: Mis6-Sim4 complex
def: "A protein complex that forms part of the inner centromere, which is involved in the loading of the centromeric histone h3 variant CENP-A onto centromeres and in centromere specific heterochromatin formation. The complex contains about 12 proteins, of which two are known as Mis6 and Sim4 in S. pombe and CENP-I and CENP-H in human." [GOC:vw, PMID:12719471, PMID:15897182]
GO:0031512
name: motile primary cilium
def: "A primary cilium which may contain a variable array of axonemal microtubules and also contains molecular motors. Motile primary cilia display a distinct twirling motion that directs fluid flow asymmetrically across the cellular surface to affect asymmetric body plan organization." [GOC:dgh, GOC:kmv, PMID:17009929, PMID:20144998]
GO:0031513
name: nonmotile primary cilium
def: "A primary cilium which contains a variable array of axonemal microtubules but does not contain molecular motors. Nonmotile primary cilia are found on many different cell types and function as sensory organelles that concentrate and organize sensory signaling molecules." [GOC:dgh, GOC:kmv, PMID:17009929, PMID:20144998]
GO:0031514
name: motile cilium
def: "A cilium which has a variable arrangement of axonemal microtubules, contains molecular motors, and beats with a characteristic whip-like pattern that promotes cell motility or transport of fluids and other cells across a cell surface. Motile cilia are typically found in multiple copies on epithelial cells that line the lumenal ducts of various tissues. Motile cilia may also function as sensory organelles." [GOC:dgh, GOC:kmv, PMID:17009929, PMID:20144998]
GO:0031515
name: tRNA (m1A) methyltransferase complex
def: "A protein complex involved in the catalysis of the formation of the modified nucleotide 1-methyladenosine (m1A) in tRNA. In yeast, it is a heterotetramer of two subunits, Gcd10p and Gcd14p, while in bacteria and archaea it is a homotetramer." [PMID:10779558, PMID:14739239]
GO:0031518
name: CBF3 complex
def: "A multisubunit protein complex that binds to centromeric DNA and initiates kinetochore assembly. In yeast, this complex consists of four subunits, namely Ctf13p, Skp1p, Cep3p and Cbf2p." [PMID:13679521, PMID:9407032]
GO:0031519
name: PcG protein complex
def: "A chromatin-associated multiprotein complex containing Polycomb Group proteins. In Drosophila, Polycomb group proteins are involved in the long-term maintenance of gene repression, and PcG protein complexes associate with Polycomb group response elements (PREs) in target genes to regulate higher-order chromatin structure." [PMID:9372908]
GO:0031520
name: plasma membrane of cell tip
def: "The portion of the plasma membrane surrounding the cell tip." [GOC:mah]
GO:0031521
name: spitzenkorper
def: "Structure within the hyphal tip of filamentous fungi that acts as an organizing center for hyphal tip growth; may function to supply vesicles to the elongating tip and/or to organize cytoskeletal microfilaments." [PMID:15701784, PMID:15976451]
GO:0031522
name: cell envelope Sec protein transport complex
def: "A transmembrane protein complex involved in the translocation of proteins across the cytoplasmic membrane. In Gram-negative bacteria, Sec-translocated proteins are subsequently secreted via the type II, IV, or V secretion systems. Sec complex components include SecA, D, E, F, G, Y and YajC." [GOC:mtg_sensu, PMID:15223057]
GO:0031523
name: Myb complex
def: "A multisubunit complex consisting of Myb and other proteins that regulates site specific DNA replication, gene amplification and transcriptional repression." [PMID:12490953, PMID:15545624]
GO:0031526
name: brush border membrane
def: "The portion of the plasma membrane surrounding the brush border." [GOC:mah]
GO:0031527
name: filopodium membrane
def: "The portion of the plasma membrane surrounding a filopodium." [GOC:mah]
GO:0031528
name: microvillus membrane
def: "The portion of the plasma membrane surrounding a microvillus." [GOC:mah]
GO:0031533
name: mRNA cap methyltransferase complex
def: "A protein complex that consists of an RNA 5' triphosphatase and a guanyl transferase (Cet1p and Ceg1p in S. cerevisiae; Pct1 and Ceg1 in S. pombe) and is involved in mRNA capping." [GOC:vw, PMID:10347220]
GO:0031560
name: cellular bud neck polarisome
def: "Protein complex that has a role in determining cell polarity, found at the neck of a fungal bud before and during cytokinesis." [PMID:9632790]
GO:0031561
name: cellular bud tip polarisome
def: "Protein complex that has a role in determining cell polarity, found at the tip of a growing fungal bud." [PMID:9632790]
GO:0031562
name: hyphal tip polarisome
def: "Protein complex that has a role in determining cell polarity, found at the tip of a growing fungal hypha." [PMID:15976451]
GO:0031563
name: mating projection tip polarisome
def: "Protein complex that has a role in determining cell polarity, found at the tip of the mating projection in unicellular fungi exposed to mating pheromone." [PMID:14734532]
GO:0031588
name: AMP-activated protein kinase complex
def: "A protein complex that possesses AMP-dependent protein kinase activity." [GOC:mah]
GO:0031592
name: centrosomal corona
def: "An amorphous structure surrounding the core of the centrosome, from which microtubules are nucleated; contains gamma-tubulin." [GOC:kp, GOC:mah]
GO:0031594
name: neuromuscular junction
def: "The junction between the axon of a motor neuron and a muscle fiber. In response to the arrival of action potentials, the presynaptic button releases molecules of neurotransmitters into the synaptic cleft. These diffuse across the cleft and transmit the signal to the postsynaptic membrane of the muscle fiber, leading to a change in post-synaptic potential." [GOC:nln]
GO:0031595
name: nuclear proteasome complex
def: "A proteasome found in the nucleus of a cell." [GOC:mah]
GO:0031596
name: ER proteasome complex
def: "A proteasome found in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031597
name: cytosolic proteasome complex
def: "A proteasome complex found in the cytosol of a cell." [GOC:mah, GOC:mtg_sensu]
GO:0031598
name: nuclear proteasome regulatory particle
def: "The regulatory subcomplex of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031599
name: ER proteasome regulatory particle
def: "The regulatory subcomplex of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031600
name: cytosolic proteasome regulatory particle
def: "A multisubunit complex located in the cytosol of a cell, which caps one or both ends of the proteasome core complex. This complex recognizes, unfolds ubiquitinated proteins and translocates them to the proteasome core complex." [GOC:mah, GOC:mtg_sensu]
GO:0031601
name: nuclear proteasome core complex
def: "The core complex of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031602
name: ER proteasome core complex
def: "The core complex of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031603
name: cytosolic proteasome core complex
def: "The core complex of a proteasome located in the cytosol of a cell." [GOC:mah, GOC:mtg_sensu]
GO:0031604
name: nuclear proteasome core complex, alpha-subunit complex
def: "The subunits forming the outer ring of the core complex of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031605
name: ER proteasome core complex, alpha-subunit complex
def: "The subunits forming the outer ring of the core complex of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031606
name: cytosolic proteasome core complex, alpha-subunit complex
def: "The proteasome core subcomplex that constitutes the two outer rings of the cytosolic proteasome core complex." [GOC:mah, GOC:mtg_sensu]
GO:0031607
name: nuclear proteasome core complex, beta-subunit complex
def: "The subunits forming the inner ring of the core complex of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031608
name: ER proteasome core complex, beta-subunit complex
def: "The subunits forming the inner ring of the core complex of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031609
name: cytosolic proteasome core complex, beta-subunit complex
def: "The proteasome core subcomplex that constitutes the two inner rings of the cytosolic proteasome core complex." [GOC:mah, GOC:mtg_sensu]
GO:0031610
name: nuclear proteasome regulatory particle, base subcomplex
def: "The subunits of the regulatory particle that directly associate with the core complex of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031611
name: ER proteasome regulatory particle, base subcomplex
def: "The subunits of the regulatory particle that directly associate with the core complex of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031612
name: cytosolic proteasome regulatory particle, base subcomplex
def: "The subcomplex of the proteasome regulatory particle that directly associates with the proteasome core complex located in the cytosol of the cell." [GOC:mah, GOC:mtg_sensu]
GO:0031613
name: nuclear proteasome regulatory particle, lid subcomplex
def: "The subunits that form the peripheral lid of the regulatory particle of a proteasome located in the nucleus of a cell." [GOC:mah]
GO:0031614
name: ER proteasome regulatory particle, lid subcomplex
def: "The subunits that form the peripheral lid of the regulatory particle of a proteasome located in the endoplasmic reticulum of a cell." [GOC:mah]
GO:0031615
name: cytosolic proteasome regulatory particle, lid subcomplex
def: "The subcomplex of the cytosolic proteasome regulatory particle that forms the peripheral lid, which is added on top of the base subcomplex." [GOC:mah, GOC:mtg_sensu]
GO:0031616
name: spindle pole centrosome
def: "A centrosome from which one pole of a mitotic or meiotic spindle is organized." [GOC:mah]
GO:0031617
name: NMS complex
def: "A supercomplex formed by the association of two subcomplexes (known as MIND and Ndc80 in Schizosaccharomyces) with additional proteins at the kinetochores of condensed nuclear chromosomes." [PMID:16079914]
GO:0031618
name: nuclear pericentric heterochromatin
def: "Nuclear heterochromatin that is located adjacent to the CENP-A rich centromere 'central core' and characterized by the modified histone H3K9me3." [GOC:mah, PMID:20206496, PMID:22729156, PMID:9413993]
GO:0031633
name: xanthophore
def: "A chromatophore containing yellow pigment." [ISBN:0395825172]
GO:0031672
name: A band
def: "The dark-staining region of a sarcomere, in which myosin thick filaments are present; the center is traversed by the paler H zone, which in turn contains the M line." [ISBN:0321204131]
GO:0031673
name: H zone
def: "A relatively pale zone traversing the center of the A band of a sarcomere, visible in relaxed muscle fibers; consists of the central portion of thick (myosin) filaments that are not overlapped by thin (actin) filaments." [GOC:mtg_muscle, ISBN:0321204131]
GO:0031674
name: I band
def: "A region of a sarcomere that appears as a light band on each side of the Z disc, comprising a region of the sarcomere where thin (actin) filaments are not overlapped by thick (myosin) filaments; contains actin, troponin, and tropomyosin; each sarcomere includes half of an I band at each end." [ISBN:0321204131]
GO:0031676
name: plasma membrane-derived thylakoid membrane
def: "The pigmented membrane of a plasma membrane-derived thylakoid." [GOC:mah, GOC:mtg_sensu]
GO:0031680
name: G-protein beta/gamma-subunit complex
def: "The heterodimer formed by the beta and gamma subunits of a heterotrimeric G protein, which dissociates from the alpha subunit upon guanine nuclotide exchange." [GOC:mah]
GO:0031838
name: haptoglobin-hemoglobin complex
def: "A protein complex formed by the stable binding of a haptoglobin to hemoglobin." [GOC:mah]
GO:0031897
name: Tic complex
def: "The translocon of the inner envelope of chloroplasts, which facilitates the import of proteins across the chloroplast inner membrane." [PMID:12180471, PMID:12393016]
GO:0031898
name: chromoplast envelope
def: "The double lipid bilayer enclosing the chromoplast and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:pz]
GO:0031899
name: chromoplast inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the chromoplast envelope; also faces the chromoplast stroma." [GOC:pz]
GO:0031900
name: chromoplast outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the chromoplast envelope." [GOC:pz]
GO:0031901
name: early endosome membrane
def: "The lipid bilayer surrounding an early endosome." [GOC:pz]
GO:0031902
name: late endosome membrane
def: "The lipid bilayer surrounding a late endosome." [GOC:pz]
GO:0031903
name: microbody membrane
def: "The lipid bilayer surrounding a microbody." [GOC:mah]
GO:0031904
name: endosome lumen
def: "The volume enclosed by the membrane of an endosome." [GOC:mah]
GO:0031905
name: early endosome lumen
def: "The volume enclosed by the membrane of an early endosome." [GOC:mah]
GO:0031906
name: late endosome lumen
def: "The volume enclosed by the membrane of a late endosome." [GOC:mah]
GO:0031907
name: microbody lumen
def: "The volume enclosed by the membranes of a microbody." [GOC:mah]
GO:0031908
name: glyoxysomal lumen
def: "The volume enclosed by the membranes of a glyoxysome." [GOC:mah]
GO:0031910
name: cytostome
def: "Stable, specialized structure for the ingestion of food by the cell into phagosomes." [PMID:10503189]
GO:0031911
name: cytoproct
def: "Stable, specialized structure for extrusion of waste by the cell into the surrounding medium." [PMID:10503189]
GO:0031912
name: oral apparatus
def: "Complex basket- or funnel-like structure used by the cell to collect food and channel it to the cytostome; includes specialized sub-structures made up of closely-spaced cilia and underlying basal bodies and fibrillar systems." [PMID:10503189]
GO:0031913
name: contractile vacuole pore
def: "Stable structure that regulates the flow of liquid between the contractile vacuole and the surrounding medium." [PMID:10503189]
GO:0031931
name: TORC1 complex
def: "A protein complex that contains at least TOR (target of rapamycin) and Raptor (regulatory-associated protein of TOR), or orthologs of, in complex with other signaling components. Mediates the phosphorylation and activation of S6K. In Saccharomyces, the complex contains Kog1p, Lst8p, Tco89p, and either Tor1p or Tor2p." [GOC:jh, PMID:15780592, PMID:16469695, PMID:21548787]
GO:0031932
name: TORC2 complex
def: "A protein complex that contains at least TOR (target of rapamycin) and Rictor (rapamycin-insensitive companion of TOR), or orthologs of, in complex with other signaling components. Mediates the phosphorylation and activation of PKB (also called AKT). In Saccharomyces, the complex contains Avo1p, Avo2p, Tsc11p, Lst8p, Bit61p, Slm1p, Slm2p, and Tor2p." [GOC:bf, GOC:jh, PMID:14736892, PMID:15780592, PMID:16469695, PMID:21548787]
GO:0031933
name: telomeric heterochromatin
def: "Heterochromatic regions of the chromosome found at the telomeres." [GOC:mah]
GO:0031934
name: mating-type region heterochromatin
def: "Heterochromatic regions of the chromosome found at silenced mating-type loci." [GOC:mah]
GO:0031941
name: filamentous actin
def: "A two-stranded helical polymer of the protein actin." [GOC:mah]
GO:0031942
name: i-AAA complex
def: "Protease complex of the mitochondrial inner membrane whose catalytic residues lie on the intermembrane space side of the inner membrane; involved in mitochondrial protein turnover. Contains a subunit belonging to the AAA family of ATP-dependent metalloproteases." [PMID:16247555, PMID:16267274]
GO:0031965
name: nuclear membrane
def: "Either of the lipid bilayers that surround the nucleus and form the nuclear envelope; excludes the intermembrane space." [GOC:mah, GOC:pz]
GO:0031966
name: mitochondrial membrane
def: "Either of the lipid bilayers that surround the mitochondrion and form the mitochondrial envelope." [GOC:mah, NIF_Subcellular:sao1045389829]
GO:0031967
name: organelle envelope
def: "A double membrane structure enclosing an organelle, including two lipid bilayers and the region between them. In some cases, an organelle envelope may have more than two membranes." [GOC:mah, GOC:pz]
GO:0031968
name: organelle outer membrane
def: "The outer, i.e. cytoplasm-facing in a cellular organelle, lipid bilayer of an organelle envelope." [GOC:mah]
GO:0031969
name: chloroplast membrane
def: "Either of the lipid bilayers that surround a chloroplast and form the chloroplast envelope." [GOC:mah, GOC:pz]
GO:0031970
name: organelle envelope lumen
def: "The region between the inner and outer lipid bilayers of an organelle envelope." [GOC:mah]
GO:0031972
name: chloroplast intermembrane space
def: "The region between the inner and outer lipid bilayers of a chloroplast envelope." [GOC:mah]
GO:0031973
name: chromoplast intermembrane space
def: "The region between the inner and outer lipid bilayers of a chromoplast envelope." [GOC:mah]
GO:0031974
name: membrane-enclosed lumen
def: "The enclosed volume within a sealed membrane or between two sealed membranes. Encompasses the volume enclosed by the membranes of a particular organelle, e.g. endoplasmic reticulum lumen, or the space between the two lipid bilayers of a double membrane surrounding an organelle, e.g. nuclear envelope lumen." [GOC:add, GOC:mah]
GO:0031975
name: envelope
def: "A multilayered structure surrounding all or part of a cell; encompasses one or more lipid bilayers, and may include a cell wall layer; also includes the space between layers." [GOC:mah, GOC:pz]
GO:0031976
name: plastid thylakoid
def: "Any thylakoid within a plastid." [GOC:pz]
GO:0031977
name: thylakoid lumen
def: "The volume enclosed by a thylakoid membrane." [GOC:mah, GOC:pz]
GO:0031978
name: plastid thylakoid lumen
def: "The volume enclosed by a plastid thylakoid membrane." [GOC:mah]
GO:0031979
name: plasma membrane-derived thylakoid lumen
def: "The volume enclosed by a plasma membrane-derived thylakoid." [GOC:mah, GOC:mtg_sensu]
GO:0031981
name: nuclear lumen
def: "The volume enclosed by the nuclear inner membrane." [GOC:mah, GOC:pz]
GO:0031982
name: vesicle
def: "Any small, fluid-filled, spherical organelle enclosed by membrane or protein." [GOC:mah, GOC:pz]
GO:0031983
name: vesicle lumen
def: "The volume enclosed by the membrane or protein that forms a vesicle." [GOC:mah]
GO:0031984
name: organelle subcompartment
def: "A compartment that consists of a lumen and an enclosing membrane, and is part of an organelle." [GOC:mah, GOC:pz]
GO:0031985
name: Golgi cisterna
def: "Any of the thin, flattened membrane-bounded compartments that form the central portion of the Golgi complex." [GOC:mah]
GO:0031986
name: proteinoplast
def: "A leucoplast in which protein is stored." [GOC:pz]
GO:0031988
name: membrane-bounded vesicle
def: "Any small, fluid-filled, spherical organelle enclosed by a lipid bilayer." [GOC:mah]
GO:0032002
name: interleukin-28 receptor complex
def: "A protein complex that binds interleukin-28 and interleukin-29. Composed of two subunits, IL-28R alpha and IL-10R beta." [GOC:rph]
GO:0032009
name: early phagosome
def: "A membrane-bounded intracellular vesicle as initially formed upon the ingestion of particulate material by phagocytosis." [GOC:mah, PMID:12388753]
GO:0032010
name: phagolysosome
def: "A membrane-bounded intracellular vesicle formed by maturation of an early phagosome following the ingestion of particulate material by phagocytosis; during maturation, phagosomes acquire markers of late endosomes and lysosomes." [GOC:mah, PMID:12388753]
GO:0032019
name: mitochondrial cloud
def: "A prominent mass in the cytoplasm of previtellogenic oocytes. The cloud contains both mitochondria and electron-dense granulofibrillar material (GFM) and is the source of germinal granule material." [PMID:6541166]
GO:0032021
name: NELF complex
def: "A complex of five proteins, designated NELF-A, -B, -C, -D, and -E in human, that can physically associate with RNP polymerase II to induce transcriptional pausing." [PMID:12612062]
GO:0032039
name: integrator complex
def: "A protein complex that stably associates with the C-terminus of RNA polymerase II and mediates 3'-end processing of small nuclear RNAs generated by RNA polymerase II." [PMID:16239144]
GO:0032040
name: small-subunit processome
def: "A large ribonucleoprotein complex that is an early preribosomal complex. In S. cerevisiae, it has a size of 80S and consists of the 35S pre-rRNA, early-associating ribosomal proteins most of which are part of the small ribosomal subunit, the U3 snoRNA and associated proteins." [GOC:krc, GOC:vw, PMID:12068309, PMID:12957375, PMID:15120992, PMID:15590835]
GO:0032044
name: DSIF complex
def: "A heterodimeric protein complex formed of Spt4 and Spt5 proteins which is expressed in eukaryotes from yeast to man. DSIF is an inhibitory elongation factor that promotes RNA polymerase II transcriptional pausing, but can also stimulate transcriptional elongation under certain conditions, and may play a role in RNA processing via its physical association with mRNA capping enzymes." [PMID:12242279, PMID:12653964, PMID:12676794, PMID:16581788, PMID:19460865]
GO:0032045
name: guanyl-nucleotide exchange factor complex
def: "A protein complex that stimulates the exchange of guanyl nucleotides associated with a GTPase." [GOC:mah]
GO:0032046
name: micropexophagy-specific membrane apparatus
def: "A membrane-bounded flattened sac that is formed during micropexophagy between the membrane tips of an engulfing vacuole, completing the engulfment and sequestration of peroxisomes from the cytosol, and forming a micropexophagic body within the lumen of the vacuole." [PMID:15563611]
GO:0032047
name: mitosome
def: "A double-membrane-bounded organelle that functions in iron-sulfur protein maturation; evolutionarily derived from mitochondria. The mitosome has been detected only in anaerobic or microaerophilic organisms that do not have mitochondria, such as Entamoeba histolytica, Giardia intestinalis and several species of Microsporidia. These organisms are not capable of gaining energy from oxidative phosphorylation, which is normally performed by mitochondria." [GOC:giardia, PMID:10361303, PMID:14614504, PMID:24316280]
GO:0032059
name: bleb
def: "A cell extension characterized by rapid formation, rounded shape, and scarcity of organelles within the protrusions." [PMID:12083798, PMID:16624291]
GO:0032068
name: Type IV site-specific deoxyribonuclease complex
def: "A complex consisting of two proteins which acts as an endonuclease in DNA sequences containing a specific modified recognition site. Modifications may include methylation, hydroxymethylation, and glucosyl-hydroxymethylation." [PMID:12654995]
GO:0032116
name: SMC loading complex
def: "A protein complex required for the loading of a structural maintenance of chromosome (SMC) complex, such as cohesin, condensin or SMC5/SMC6, onto DNA. Appears to be eukaryotically conserved." [GOC:curators, GOC:vw, PMID:10882066]
GO:0032117
name: horsetail-astral microtubule array
def: "An array of astral microtubules that emanates from the spindle pole body during meiosis and facilitates horsetail nuclear movement." [GOC:mah, PMID:16111942]
GO:0032123
name: deep fiber
def: "Inward projections of the cytoskeletal structures of the oral apparatus, which form a fiber that extends past the cytostome into the cytoplasm." [PMID:10503189]
GO:0032126
name: eisosome
def: "A cell part that is composed of the eisosome membrane or MCC domain, a furrow-like plasma membrane sub-domain and associated integral transmembrane proteins, and the proteins (eisosome filaments) that form a scaffolding lattice on the cytoplasmic face. Eisosomes broadly affect overall plasma membrane organization." [GOC:al, GOC:vw, PMID:16496001, PMID:22368779]
GO:0032127
name: dense core granule membrane
def: "The lipid bilayer surrounding a dense core granule." [GOC:mah]
GO:0032133
name: chromosome passenger complex
def: "A eukaryotically conserved protein complex that localizes to kinetochores in early mitosis, the spindle mid-zone in anaphase B and to the telophase midbody. It has been proposed that the passenger complex coordinates various events based on its location to different structures during the course of mitosis. Complex members include the BIR-domain-containing protein Survivin, Aurora kinase, INCENP and Borealin." [GOC:vw, PMID:16824200, PMID:19570910]
GO:0032144
name: 4-aminobutyrate transaminase complex
def: "A homodimeric protein complex that possesses 4-aminobutyrate transaminase activity." [GOC:mah, PMID:15528998]
GO:0032151
name: mitotic septin complex
def: "A heterooligomeric septin complex that acts during mitotic cell division." [GOC:krc, PMID:16009555]
GO:0032152
name: meiotic septin complex
def: "A heterooligomeric septin complex that acts during meiotic cell division." [GOC:krc, PMID:16009555]
GO:0032153
name: cell division site
def: "The eventual plane of cell division (also known as cell cleavage or cytokinesis) in a dividing cell. In Eukaryotes, the cleavage apparatus, composed of septin structures and the actomyosin contractile ring, forms along this plane, and the mitotic, or meiotic, spindle is aligned perpendicular to the division plane. In bacteria, the cell division site is generally located at mid-cell and is the site at which the cytoskeletal structure, the Z-ring, assembles." [GOC:bf, GOC:imk, GOC:krc, GOC:ns, PMID:12101122, PMID:15380095, PMID:16983191, PMID:18165305]
GO:0032154
name: cleavage furrow
def: "In animal cells, the first sign of cleavage, or cytokinesis, is the appearance of a shallow groove in the cell surface near the old metaphase plate. A contractile ring containing actin and myosin is located just inside the plasma membrane at the location of the furrow. Ring contraction is associated with centripetal growth of the membrane that deepens the cleavage furrow and divides the cytoplasm of the two daughter cells. While the term 'cleavage furrow' was initially associated with animal cells, such a structure occurs in many other types of cells, including unicellular protists." [ISBN:0805319409]
GO:0032155
name: cell division site part
def: "Any constituent part of the cell division plane, the eventual plane of cell division in a dividing cell." [GOC:mah]
GO:0032156
name: septin cytoskeleton
def: "The part of the cytoskeleton (the internal framework of a cell) composed of septins and associated proteins. Includes septin cytoskeleton-associated complexes." [GOC:mah]
GO:0032157
name: prospore contractile ring
def: "A contractile ring, i.e. a cytoskeletal structure composed of actin filaments and myosin, that forms beneath the plasma membrane of the prospore envelope in meiotic cells in preparation for completing cytokinesis." [GOC:krc, PMID:16009555]
GO:0032158
name: septin band
def: "A diffuse ring composed of a series of septin bars that run parallel to the long axis of the cell. This type of septin structure has been observed in a number of locations associated with polarized grown and/or deposition of new membrane, but not with cytokinesis, such as at the shmoo (mating projection) neck, at the junction between the mother cell and the germ tube (hypha) of a fungal cell growing filamentously." [GOC:krc, PMID:16151244]
GO:0032159
name: septin cap
def: "A faint structure formed of septins found at the leading edge of growth in germ tubes and hyphae in fungal cells growing filamentously. This cap of septins colocalizes with a region of the plasma membrane that is rich in ergosterol." [GOC:krc, PMID:16151244]
GO:0032160
name: septin filament array
def: "Arrays of septin filaments, or bars, found in a series of filamentous structures. Such structures have been observed in the prospore membrane during spore formation in S. cerevisiae and in the chlamydospore membrane during chlamydospore formation in C. albicans." [GOC:krc, PMID:16151244]
GO:0032161
name: cleavage apparatus septin structure
def: "Any of a series of structures composed of septins and septin-associated proteins localized to the cleavage plane which are involved in cytokinesis." [GOC:krc, PMID:12101122, PMID:15774761, PMID:16009555]
GO:0032162
name: mating projection septin band
def: "A septin band, i.e. a diffuse ring composed of a series of septin bars running parallel to the long axis of the cell, located at the neck of a shmoo (mating projection)." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032163
name: hyphal septin band
def: "A septin band, i.e. a diffuse ring composed of a series of septin bars running parallel to the long axis of the cell, located at the junction between the mother cell and the germ tube (hypha) of a fungal cell growing filamentously." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032164
name: hyphal septin cap
def: "A faint structure formed of septins found at the leading edge of growth in hyphae of fungal cells growing filamentously. This cap of septins colocalizes with a region of the plasma membrane that is rich in ergosterol." [GOC:krc, PMID:16151244]
GO:0032165
name: prospore septin filament array
def: "Arrays of septin filaments, or bars, found in a series of filamentous structures; observed in the prospore membrane during spore formation." [GOC:krc, PMID:16151244]
GO:0032166
name: chlamydospore septin filament array
def: "Arrays of septin filaments, or bars, found in a series of filamentous structures. Observed in the chlamydospore membrane during chlamydospore formation." [GOC:krc, PMID:16151244]
GO:0032167
name: septin patch
def: "OBSOLETE. An amorphous-appearing accumulation of septin proteins at the future site of cytokinesis." [PMID:16009555]
GO:0032168
name: hyphal septin ring
def: "A tight ring-shaped structure that forms in the division plane within hyphae of filamentous fungi at sites where a septum will form; composed of septins as well as septin-associated proteins." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032169
name: prospore septin ring
def: "A tight ring-shaped structure that forms in the division plane at the site of cytokinesis in a prospore; composed of septins as well as septin-associated proteins." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032170
name: pseudohyphal septin ring
def: "A tight ring-shaped structure that forms in the division plane at the junction between the mother cell and a pseudohyphal projection; composed of septins as well as septin-associated proteins." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032171
name: germ tube septin cap
def: "A faint structure formed of septins found at the leading edge of growth in germ tubes of fungal cells growing filamentously. This cap of septins colocalizes with a region of the plasma membrane that is rich in ergosterol." [GOC:krc, PMID:16151244]
GO:0032172
name: germ tube septin ring
def: "A tight ring-shaped structure that forms in the division plane within the germ tube of filamentous fungi at sites where a septum will form; composed of septins as well as septin-associated proteins." [GOC:krc, PMID:16151244]
GO:0032173
name: septin collar
def: "A tubular, hourglass-shaped structure composed of highly ordered arrays of septin filaments; in budding yeast cells, the septin collar forms from the initial septin ring by expanding into the daughter cell." [GOC:krc, PMID:16009555, PMID:16151244]
GO:0032174
name: cellular bud neck septin collar
def: "A tubular structure with flared ends, shaped like an hourglass and composed of highly ordered arrays of septin filaments, that forms at the bud neck of a dividing cell. In S. cerevisiae, this structure is located at the bud neck throughout most of the cell cycle and the septins are fixed within the structure, not exchanging with soluble septins. This septin structure acts as a scaffold for other proteins that function at the bud neck." [GOC:krc, PMID:16009555]
GO:0032175
name: mating projection septin ring
def: "A septin ring, i.e. a ring-shaped structure composed of septins and septin-associated proteins, located at the neck of a shmoo (mating projection). The septin ring in the neck of a shmoo may act as a barrier to localize mating factors in the shmoo tip." [GOC:krc, GOC:mah, PMID:16151244]
GO:0032176
name: split septin rings
def: "A pair of rings that flank the site of cell division, formed by splitting of the septin ring (or collar) prior to cytokinesis; this double ring structure is thought to trap proteins needed for cytokinesis or the formation of the new membrane or cell wall between the two septin rings. Split septin rings are known to occur in budding yeast cells and probably occur in other cell types as well." [GOC:krc, PMID:16009555, PMID:16151244]
GO:0032177
name: cellular bud neck split septin rings
def: "Two separate septin rings that are formed from the septin collar at the time of cytokinesis in cells that divide by budding. These two rings are thought to delineate a special compartment in which factors involved in cytokinesis are concentrated." [GOC:krc, PMID:16009555]
GO:0032178
name: medial membrane band
def: "A sterol-rich region of the plasma membrane which forms at the cell surface overlying the contractile ring and spreads into the invaginating plasma membrane surrounding the septum." [PMID:15517003]
GO:0032179
name: germ tube
def: "The slender tubular outgrowth first produced by most spores in germination." [ISBN:0877799148]
GO:0032195
name: post-lysosomal vacuole
def: "A membrane-bounded intracellular vesicle formed late in the endocytic pathway when the pH in the vacuole becomes neutral prior to exocytosis." [GOC:pf, PMID:9276759, PMID:9394012]
GO:0032221
name: Rpd3S complex
def: "A eukaryotically conserved histone deacetylase complex which deacetylates histones preferentially in promoter regions. Composed of a catalytic histone deacetylase subunit, a chromodomain protein, a SIN3 family co-repressor, and a WD repeat protein (Clr6p, Alp13p, Pst2p, and Prw1p respectively in Schizosaccharomyces; Rpd3p, Sin3p, Ume1p, Rco1p and Eaf3 in Saccharomyces; homologs thereof in other species)." [GOC:vw, PMID:12773392, PMID:17450151]
GO:0032279
name: asymmetric synapse
def: "A type of synapse occurring between an axon and a dendritic spine or dendritic shaft. Asymmetric synapses, the most abundant synapse type in the central nervous system, involve axons that contain predominantly spherical vesicles and contain a thickened postsynaptic density." [GOC:dgh, GOC:ef]
GO:0032280
name: symmetric synapse
def: "A type of synapse occurring primarily on dendrite shafts and neuronal cell bodies. Symmetric synapses involve axons containing clusters of predominantly flattened or elongated vesicles and do not contain a prominent postsynaptic density." [GOC:dgh, GOC:ef]
GO:0032281
name: alpha-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid selective glutamate receptor complex
def: "An assembly of four or five subunits which form a structure with an extracellular N-terminus and a large loop that together form the ligand binding domain. The C-terminus is intracellular. The ionotropic glutamate receptor complex itself acts as a ligand gated ion channel; on binding glutamate, charged ions pass through a channel in the center of the receptor complex. The AMPA receptors mediate fast synaptic transmission in the CNS and are composed of subunits GluR1-4, products from separate genes. These subunits have an extracellular N-terminus and an intracellular C-terminus." [GOC:ef]
GO:0032282
name: plastid acetyl-CoA carboxylase complex
def: "An acetyl-CoA carboxylase complex located in the stroma of a plastid." [GOC:mah]
GO:0032283
name: plastid acetate CoA-transferase complex
def: "An acetate CoA-transferase complex located in the stroma of a plastid." [GOC:mah]
GO:0032284
name: plastid biotin carboxylase complex
def: "A biotin carboxylase complex located in the stroma of a plastid." [GOC:mah]
GO:0032299
name: ribonuclease H2 complex
def: "A protein complex that possesses ribonuclease H activity, in which the catalytic subunit is a member of the RNase H2 (or HII) class. For example, in Saccharomyces the complex contains Rnh201p, Rnh202p and Rnh203p." [GOC:mah, PMID:14734815]
GO:0032300
name: mismatch repair complex
def: "Any complex formed of proteins that act in mismatch repair." [GOC:mah]
GO:0032301
name: MutSalpha complex
def: "A heterodimer involved in the recognition and repair of base-base and small insertion/deletion mismatches. In human the complex consists of two subunits, MSH2 and MSH6." [PMID:11005803]
GO:0032302
name: MutSbeta complex
def: "A heterodimer involved in binding to and correcting insertion/deletion mutations. In human the complex consists of two subunits, MSH2 and MSH3." [PMID:11005803]
GO:0032311
name: angiogenin-PRI complex
def: "A stable heterodimer of angiogenin and placental ribonuclease inhibitor; interaction between angiogenin and PRI prevents angiogenin binding to its receptor to stimulate angiogenesis." [PMID:2706246, PMID:3470787]
GO:0032389
name: MutLalpha complex
def: "A heterodimer involved in the recognition of base-base and small insertion/deletion mismatches. In human the complex consists of two subunits, MLH1 and PMS2." [GOC:vk]
GO:0032390
name: MutLbeta complex
def: "A heterodimer involved in the recognition of base-base and small insertion/deletion mismatches. In human the complex consists of two subunits, MLH1 and PMS1." [GOC:vk]
GO:0032391
name: photoreceptor connecting cilium
def: "A nonmotile primary cilium that has a 9+0 microtubule array and forms the portion of the axoneme traversing the boundary between the photoreceptor inner and outer segments." [PMID:15917207, PMID:8718680]
GO:0032398
name: MHC class Ib protein complex
def: "A transmembrane protein complex composed of a MHC class Ib alpha chain and, in most cases, an invariant beta2-microglobin chain, and with or without a bound peptide or lipid antigen. Class Ib here refers to non-classical class I molecules, such as those of the CD1 or HLA-E gene families." [GOC:add, ISBN:0781735149, PMID:15928678]
GO:0032419
name: extrinsic component of lysosome membrane
def: "The component of an lysosome membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:mah]
GO:0032420
name: stereocilium
def: "An actin-based protrusion from the apical surface of auditory and vestibular hair cells. Bundles of stereocilia act as mechanosensory organelles." [GOC:ecd, PMID:15661519, PMID:7840137]
GO:0032421
name: stereocilium bundle
def: "An actin-based, cross-linked cellular protrusion on the apical surface of auditory and vestibular hair cells. Stereocilium bundles act as mechanosensory organelles by responding to fluid motion or fluid pressure changes." [GOC:ecd, PMID:15661519, PMID:7840137]
GO:0032426
name: stereocilium bundle tip
def: "The end of a stereocilium bundle, distal to the site of the bundle's attachment to the apical cell surface." [GOC:ecd, PMID:17021180]
GO:0032432
name: actin filament bundle
def: "An assembly of actin filaments that are on the same axis but may be oriented with the same or opposite polarities and may be packed with different levels of tightness." [GOC:mah]
GO:0032433
name: filopodium tip
def: "The end of a filopodium distal to the body of the cell." [GOC:mah]
GO:0032437
name: cuticular plate
def: "A dense network of actin filaments found beneath the apical cell surface of hair cells, and into which stereocilia are inserted." [PMID:12485990, PMID:2592408, PMID:8071151]
GO:0032444
name: activin responsive factor complex
def: "A transcriptionally active complex that binds to an activin response element (ARE) in the promoter of target genes, and is composed of two SMAD2 proteins, one SMAD4 protein and a Forkhead activin signal transducer (FAST) transcription factor." [PMID:12374795, PMID:9288972]
GO:0032449
name: CBM complex
def: "A protein complex comprising Carma1, Bcl10 and MALT1; plays a role in signal transduction during NF-kappaB activation." [PMID:12909454]
GO:0032473
name: cytoplasmic side of mitochondrial outer membrane
def: "The external (cytoplasmic) face of the mitochondrial outer membrane." [GOC:mah]
GO:0032476
name: decaprenyl diphosphate synthase complex
def: "A complex that possesses di-trans,poly-cis-decaprenylcistransferase activity; involved in ubiquinone biosynthesis." [GOC:mah, PMID:14519123]
GO:0032477
name: homodimeric decaprenyl diphosphate synthase complex
def: "A homodimeric complex that possesses di-trans,poly-cis-decaprenylcistransferase activity; involved in ubiquinone biosynthesis." [PMID:14519123]
GO:0032478
name: heterotetrameric decaprenyl diphosphate synthase complex
def: "A heterotetrameric complex located in the mitochondrial inner membrane that possesses di-trans,poly-cis-decaprenylcistransferase activity; involved in ubiquinone biosynthesis. In S. pombe it is a heterotetramer of Dlp1 and Dps1." [PMID:14519123]
GO:0032517
name: SOD1-calcineurin complex
def: "A protein complex formed by the association of superoxide dismutase 1 (SOD1) with calcineurin; complex formation is implicated in activation of calcineurin by SOD1." [GOC:mah, PMID:17324120]
GO:0032541
name: cortical endoplasmic reticulum
def: "A cortical network of highly dynamic tubules that are juxtaposed to the plasma membrane and undergo ring closure and tubule-branching movements." [GOC:se, PMID:10931860, PMID:17686782]
GO:0032545
name: CURI complex
def: "A protein complex that is involved in the transcription of ribosomal genes. In Saccharomyces this complex consists of Ckb2p, Utp22p, Rrp7p and Ifh1p." [PMID:17452446]
GO:0032578
name: aleurone grain membrane
def: "The lipid bilayer surrounding an aleurone grain." [GOC:ecd]
GO:0032579
name: apical lamina of hyaline layer
def: "A fibrous network that is part of the hyalin layer extracellular matrix. The apical lamina is thought to be principally composed of the glycoproteins fibropellins. This matrix has been found in echinoderms." [GOC:ecd, PMID:2060714, PMID:7608987, PMID:9638331]
GO:0032580
name: Golgi cisterna membrane
def: "The lipid bilayer surrounding any of the thin, flattened compartments that form the central portion of the Golgi complex." [GOC:ecd, GOC:mah]
GO:0032584
name: growth cone membrane
def: "The portion of the plasma membrane surrounding a growth cone." [GOC:mah]
GO:0032585
name: multivesicular body membrane
def: "The lipid bilayer surrounding a multivesicular body." [GOC:mah]
GO:0032586
name: protein storage vacuole membrane
def: "The lipid bilayer surrounding a protein storage vacuole." [GOC:mah]
GO:0032587
name: ruffle membrane
def: "The portion of the plasma membrane surrounding a ruffle." [GOC:mah]
GO:0032588
name: trans-Golgi network membrane
def: "The lipid bilayer surrounding any of the compartments that make up the trans-Golgi network." [GOC:mah]
GO:0032589
name: neuron projection membrane
def: "The portion of the plasma membrane surrounding a neuron projection." [GOC:mah]
GO:0032590
name: dendrite membrane
def: "The portion of the plasma membrane surrounding a dendrite." [GOC:mah]
GO:0032591
name: dendritic spine membrane
def: "The portion of the plasma membrane surrounding a dendritic spine." [GOC:mah]
GO:0032592
name: integral component of mitochondrial membrane
def: "The component of the mitochondrial membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0032593
name: insulin-responsive compartment
def: "A small membrane-bounded vesicle that releases its contents by exocytosis in response to insulin stimulation; the contents are enriched in GLUT4, IRAP and VAMP2." [PMID:17644329]
GO:0032766
name: NHE3/E3KARP/ACTN4 complex
def: "A heterotrimeric protein complex formed by the association of NHE3, E3KARP and alpha-actinin upon an increase in calcium ion concentration; found in clusters localized on plasma membrane and in intracellular compartments." [PMID:11948184]
GO:0032777
name: Piccolo NuA4 histone acetyltransferase complex
def: "A heterotrimeric H4/H2A histone acetyltransferase complex with a substrate preference of chromatin over free histones. It contains a subset of the proteins found in the larger NuA4 histone acetyltransferase complex; for example, the S. cerevisiae complex contains Esa1p, Yng2p, and Epl1p." [GOC:rb, PMID:12782659, PMID:15964809]
GO:0032783
name: ELL-EAF complex
def: "A heterodimeric protein complex that acts as an RNA polymerase II elongation factor; the complex is conserved from yeast to humans, and is present in S. pombe, but absent from S. cerevisiae." [PMID:17150956]
GO:0032797
name: SMN complex
def: "A protein complex that contains the survival motor neuron (SMN) protein and at least eight additional integral components, including the Gemin2-8 and Unrip proteins; the complex is found in the cytoplasm and in nuclear Gems, and is involved in spliceosomal snRNP assembly in the cytoplasm and in pre-mRNA splicing in the nucleus." [PMID:16434402, PMID:17023415]
GO:0032798
name: Swi5-Sfr1 complex
def: "A conserved DNA recombinase mediator complex that contains two Swi5 monomers and one Sfr1 monomer in Schizosaccharomyces, or orthologs thereof (e.g. Sae3p and Mei5p in Saccharomyces)." [PMID:15620352, PMID:16921379]
GO:0032806
name: carboxy-terminal domain protein kinase complex
def: "A protein complex that phosphorylates amino acid residues of RNA polymerase II C-terminal domain repeats; phosphorylation occurs mainly on Ser2 and Ser5." [PMID:15047695, PMID:16721054, PMID:17079683]
GO:0032807
name: DNA ligase IV complex
def: "A eukaryotically conserved protein complex that contains DNA ligase IV and is involved in DNA repair by non-homologous end joining; in addition to the ligase, the complex also contains XRCC4 or a homolog, e.g. Saccharomyces Lif1p." [PMID:16314503]
GO:0032809
name: neuronal cell body membrane
def: "The plasma membrane of a neuron cell body - excludes the plasma membrane of cell projections such as axons and dendrites." [GOC:jl]
GO:0032838
name: cell projection cytoplasm
def: "All of the contents of a cell projection, excluding the plasma membrane surrounding the projection." [GOC:mah]
GO:0032839
name: dendrite cytoplasm
def: "All of the contents of a dendrite, excluding the surrounding plasma membrane." [GOC:mah]
GO:0032865
name: ERMES complex
def: "A protein complex that links the endoplasmic reticulum with mitochondria and may have a role in promoting exchange of calcium and phospholipids between the two organelles. The complex is also associated with actively replicating mitochondrial DNA nucleoids, and may further coordinate mitochondrial genome replication and membrane growth." [GOC:mcc, PMID:19556461]
GO:0032921
name: sarcosine oxidase complex
def: "A complex consisting of 4 protein subunits as a heterotetramer, that possesses sarcosine oxidase activity." [GOC:mah, GOC:mlg]
GO:0032936
name: SREBP-SCAP complex
def: "A protein complex formed by the association of sterol regulatory element binding protein (SREBP) and SREBP-cleavage-activating protein (SCAP) in the ER membrane; in the absence of sterols, the SREBP-SCAP complex is packaged into COPII vesicles and travels to the Golgi apparatus to be processed." [PMID:12923525]
GO:0032937
name: SREBP-SCAP-Insig complex
def: "A protein complex formed by the association of sterol regulatory element binding protein (SREBP), SREBP-cleavage-activating protein (SCAP), and an Insig protein (Insig-1 or Insig-2) in the ER membrane." [PMID:12923525]
GO:0032969
name: endosomal scaffold complex
def: "A protein complex that contains MAPKSP1 (MP1, Map2k1ip1) and ROBLD3 (p14, Mapbpip), is anchored to late endosomes, and is involved in selective activation of the ERK1 in ERK/MAPK signaling." [PMID:15263099, PMID:16227978, PMID:17496910]
GO:0032982
name: myosin filament
def: "A protein complex containing myosin heavy chains, plus associated light chains and other proteins, in which the myosin heavy chains are arranged into a filament." [GOC:mah]
GO:0032983
name: kainate selective glutamate receptor complex
def: "An assembly of four or five subunits which form a structure with an extracellular N-terminus and a large loop that together form the ligand binding domain. The C-terminus is intracellular. The ionotropic glutamate receptor complex itself acts as a ligand gated ion channel; on binding glutamate, charged ions pass through a channel in the center of the receptor complex. Kainate receptors are multimeric assemblies of GluR5-7 and KA-1/2 subunits." [http://www.bris.ac.uk/Depts/Synaptic/info/glutamate.html]
GO:0032991
name: macromolecular complex
def: "A stable assembly of two or more macromolecules, i.e. proteins, nucleic acids, carbohydrates or lipids, in which the constituent parts function together." [GOC:mah]
GO:0032992
name: protein-carbohydrate complex
def: "A macromolecular complex containing separate protein and carbohydrate molecules.  Separate in this context means not covalently bound to each other." [GOC:mah]
GO:0032993
name: protein-DNA complex
def: "A macromolecular complex containing both protein and DNA molecules." [GOC:mah]
GO:0032994
name: protein-lipid complex
def: "A macromolecular complex containing separate protein and lipid molecules.  Separate in this context means not covalently bound to each other." [GOC:mah]
GO:0032996
name: Bcl3-Bcl10 complex
def: "A protein complex containing Bcl3 and Bcl10, which forms when Akt1 is activated by TNF-alpha to phosphorylate Bcl10; the Bcl3-Bcl10 complex is translocated to the nucleus." [PMID:16280327]
GO:0032997
name: Fc receptor complex
def: "A protein complex composed of a subunit or subunits capable of binding the Fc portion of an immunoglobulin with additional signaling components. The complex functions as a receptor for immunoglobulin." [GOC:add, ISBN:0781735149]
GO:0032998
name: Fc-epsilon receptor I complex
def: "A protein complex composed of an Fc-epsilon RI alpha chain and an Fc-epsilon RI gamma chain dimer with or without an Fc-episilon RI beta chain and additional signaling components. The complex functions primarily as an activating receptor for IgE." [GOC:add, ISBN:0781735149]
GO:0032999
name: Fc-alpha receptor I complex
def: "A protein complex composed of an Fc-alpha R alpha chain and an Fc-epsilon RI gamma chain dimer with or without additional signaling components. The complex functions primarily as an activating receptor for IgA." [GOC:add, ISBN:0781735149, PMID:12524384]
GO:0033000
name: Fc-gamma receptor I complex
def: "A protein complex composed of an Fc-gamma RI alpha chain and an Fc-epsilon RI gamma chain dimer with or without additional signaling components. The complex functions primarily as an activating receptor for IgG." [GOC:add, ISBN:0781735149, PMID:11244038, PMID:12413532]
GO:0033001
name: Fc-gamma receptor III complex
def: "A protein complex composed of an Fc-gamma RIII alpha chain and an Fc-epsilon RI gamma chain dimer with or without an Fc-epsilon RI beta chain and additional signaling components. The complex functions primarily as an activating receptor for IgG." [GOC:add, ISBN:0781735149, PMID:11244038, PMID:12413532]
GO:0033009
name: nucleomorph
def: "A small, vestigial nucleus found in some plastids that derive from a eukaryotic endosymbiont. Observed in chlorarachniophytes and cryptomonads, which acquired their plastids from a green and red alga respectively." [PMID:16760254]
GO:0033010
name: paranodal junction
def: "A highly specialized cell-cell junction found in vertebrates, which forms between a neuron and a glial cell, and has structural similarity to Drosophila septate junctions. It flanks the node of Ranvier in myelinated nerve and electrically isolates the myelinated from unmyelinated nerve segments and physically separates the voltage-gated sodium channels at the node from the cluster of potassium channels underneath the myelin sheath." [PMID:11395001, PMID:14630217]
GO:0033011
name: perinuclear theca
def: "A condensed cytoplasmic structure that covers the nucleus of mammalian spermatozoa except for a narrow zone around the insertion of the tail. It shows two distinct regions, a subacrosomal layer and, continuing caudally beyond the acrosomic system, the postacrosomal sheath. The perinuclear theca has been considered a cytoskeletal scaffold responsible for maintaining the overall architecture of the mature sperm head; however, recent studies indicate that the bulk of its constituent proteins are not traditional cytoskeletal proteins but rather a variety of cytosolic proteins." [PMID:17289678, PMID:8025156]
GO:0033012
name: porosome
def: "A permanent cup-shaped structure at the cell plasma membrane in secretory cells. Following a secretory stimulus, secretory vesicles transiently dock and fuse at the base of porosomes and release intravesicular contents dictated by the turgor pressure generated from the swelling of secretory vesicles." [PMID:15090256, PMID:16563225]
GO:0033016
name: rhoptry membrane
def: "The lipid bilayer surrounding a rhoptry." [GOC:mah]
GO:0033017
name: sarcoplasmic reticulum membrane
def: "The lipid bilayer surrounding the sarcoplasmic reticulum." [GOC:rph]
GO:0033018
name: sarcoplasmic reticulum lumen
def: "The volume enclosed by the membranes of the sarcoplasmic reticulum." [GOC:rph]
GO:0033061
name: DNA recombinase mediator complex
def: "A protein complex containing accessory proteins which bind a recombinase (e.g. Rad51) and bind single-stranded DNA (ssDNA), and promote nucleation of the recombinase onto ssDNA." [GOC:elh, GOC:mah, GOC:vw, InterPro:IPR003488, PMID:12912992]
GO:0033062
name: Rhp55-Rhp57 complex
def: "A conserved heterodimeric DNA recombinase mediator complex that contains the RecA family proteins Rhp55p and Rph57 in Schizosaccharomyces, or orthologs thereof (e.g. Rad55p and Rad57p in Saccharomyces)." [GOC:mah, GOC:vw]
GO:0033063
name: Rad51B-Rad51C-Rad51D-XRCC2 complex
def: "A DNA recombinase mediator complex that contains the Rad51 paralogs RAD51B, RAD51C, RAD51D, and XRCC2, or orthologs thereof." [GOC:mah, PMID:16093548, PMID:17114795]
GO:0033064
name: XRCC2-RAD51D complex
def: "A heterodimeric DNA recombinase mediator complex that contains the Rad51 paralogs RAD51D and XRCC2, or orthologs thereof; conserved from fission yeast to human but absent from budding yeast." [GOC:mah, GOC:vw, PMID:16093548]
GO:0033065
name: Rad51C-XRCC3 complex
def: "A DNA recombinase mediator complex that contains the Rad51 paralogs RAD51C and XRCC3, or orthologs thereof." [GOC:mah, PMID:16093548, PMID:17114795]
GO:0033066
name: Rad51B-Rad51C complex
def: "A DNA recombinase mediator complex that contains the Rad51 paralogs RAD51B and RAD51C, or orthologs thereof." [GOC:mah, PMID:16093548, PMID:17114795]
GO:0033093
name: Weibel-Palade body
def: "A large, elongated, rod-shaped secretory granule characteristic of vascular endothelial cells that contain a number of structurally and functionally distinct proteins, of which the best characterized are von Willebrand factor (VWF) and P-selectin. Weibel-Palade bodies are formed from the trans-Golgi network in a process that depends on VWF, which is densely packed in a highly organized manner, and on coat proteins that remain associated with the granules. Upon cell stimulation, regulated exocytosis releases the contained proteins to the cell surface, where they act in the recruitment of platelets and leukocytes and in inflammatory and vasoactive responses." [PMID:11935287, PMID:16087708]
GO:0033095
name: aleurone grain
def: "A membrane-bounded storage granule found in cells of the aleurone layer in plants; contains either a protein matrix, protein-carbohydrate bodies and/or globoids. Aleurone grains are formed by the vacuole, rough endoplasmic reticulum and dictyosomes." [http://ejournal.sinica.edu.tw/bbas/content/2004/1/bot451-08.pdf, http://www.springerlink.com/content/q7183h5vr7358278/]
GO:0033096
name: amyloplast envelope
def: "The double lipid bilayer enclosing the amyloplast and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:mah]
GO:0033097
name: amyloplast membrane
def: "Either of the lipid bilayers that surround an amyloplast and form the amyloplast envelope." [GOC:ecd]
GO:0033098
name: amyloplast inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the amyloplast envelope; also faces the amyloplast stroma." [GOC:ecd]
GO:0033099
name: attachment organelle
def: "A membrane-bounded extension of the cell, originally characterized in Mycoplasma species, that contains an electron-dense core that is part of the cytoskeleton and is oriented lengthwise and ends distally in a bulbous knob (terminal button). Required for adherence to host cells and involved in gliding motility and cell division." [http://authors.library.caltech.edu/3529/, PMID:11325545, PMID:12003948]
GO:0033100
name: NuA3 histone acetyltransferase complex
def: "A Gcn5-independent multisubunit complex that catalyzes the acetylation of histone H3. The budding yeast complex includes Sas3p, Taf30p, and Yng1p." [PMID:10817755, PMID:17157260]
GO:0033101
name: cellular bud membrane
def: "The portion of the plasma membrane surrounding a cellular bud." [GOC:mah]
GO:0033102
name: acidocalcisome membrane
def: "The lipid bilayer surrounding an acidocalcisome." [GOC:ecd, PMID:11378195]
GO:0033104
name: type VI protein secretion system complex
def: "A complex of proteins that permits the transfer of proteins into the extracellular milieu or directly into host cells via the type VI secretion system. Proteins secreted by this complex do not require an N-terminal signal sequence." [GOC:mlg, PMID:16432199, PMID:16763151]
GO:0033105
name: chlorosome envelope
def: "The structure, composed of a monolayer of glycolipids with embedded proteins, that encloses the pigments and other contents of the chlorosome." [PMID:14507718, PMID:14729689, PMID:17303128]
GO:0033106
name: cis-Golgi network membrane
def: "The lipid bilayer surrounding any of the compartments that make up the cis-Golgi network." [GOC:mah]
GO:0033107
name: CVT vesicle
def: "A cytosolic vesicle that is enclosed by a double membrane and is implicated in the cytoplasm to vacuole targeting pathway. These vesicles are found in the yeast S. cerevisiae, and contain vacuolar hydrolases, aminopeptidase I (Ape1p) and alpha-mannosidase (Ams1p)." [GOC:rb, PMID:15138258]
GO:0033110
name: Cvt vesicle membrane
def: "The double lipid bilayer surrounding a Cvt vesicle, a vesicle that functions in the cytoplasm-to-vacuole targeting (Cvt) pathway." [GOC:ecd, PMID:20065092]
GO:0033111
name: attachment organelle membrane
def: "The lipid bilayer surrounding an attachment organelle." [GOC:ecd]
GO:0033112
name: cyanelle envelope
def: "The double lipid bilayer enclosing the cyanelle and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:mah]
GO:0033113
name: cyanelle membrane
def: "Either of the lipid bilayers that surround a cyanelle and form the cyanelle envelope." [GOC:ecd]
GO:0033114
name: cyanelle thylakoid lumen
def: "The volume enclosed by a cyanelle thylakoid membrane." [GOC:mah]
GO:0033115
name: cyanelle thylakoid membrane
def: "The lipid bilayer membrane of any thylakoid within a cyanelle." [GOC:mah]
GO:0033116
name: endoplasmic reticulum-Golgi intermediate compartment membrane
def: "The lipid bilayer surrounding any of the compartments of the endoplasmic reticulum (ER)-Golgi intermediate compartment system." [GOC:mah, GOC:pr, PMID:16723730]
GO:0033117
name: esterosome
def: "A vesicle filled with crystalline protein that shows sequence similarities with various esterases." [GOC:ecd, PMID:2307702]
GO:0033118
name: esterosome membrane
def: "The lipid bilayer surrounding an esterosome. This membrane has characteristics of rough endoplasmic reticulum (RER) membranes." [GOC:ecd, PMID:2307702]
GO:0033150
name: cytoskeletal calyx
def: "A large cytoskeletal structure located at the posterior end of the perinuclear theca of a mammalian sperm head. The nucleus is tightly associated with the calyx, which contains calicin and basic cylicin proteins." [PMID:12243744, PMID:9184090]
GO:0033162
name: melanosome membrane
def: "The lipid bilayer surrounding a melanosome." [GOC:mah]
GO:0033163
name: microneme membrane
def: "The lipid bilayer surrounding a microneme." [GOC:mah]
GO:0033165
name: interphotoreceptor matrix
def: "A specialized extracellularc matrix that surrounds the photoreceptors of the retina and lies between them and the apical surface of the retinal pigment epithelium. The IPM has been implicated in several important activities required for photoreceptor function and maintenance." [http://www.glycoforum.gr.jp/science/hyaluronan/HA17/HA17E.html, PMID:1862095, PMID:2194288]
GO:0033166
name: hyaline layer
def: "A multilayered extraembryonic matrix that functions as a substrate for cell adhesion through early development. It is thought to protect and lubricate the embryo, stabilize the blastomeres during morphogenesis, and regulate nutrient intake. The major constituent of the hyaline layer is the protein hyalin. This matrix has been found in echinoderms." [http://worms.zoology.wisc.edu/urchins/SUgast_ECM3.html, PMID:1721506, PMID:9473317]
GO:0033167
name: ARC complex
def: "A ribonucleoprotein complex that contains members of the Argonaute family of proteins, additional protein subunits, and duplex siRNA; required for heterochromatin assembly and siRNA generation. Possibly involved in the conversion of ds siRNA to ss siRNA." [GOC:vw, PMID:17310250]
GO:0033172
name: gas vesicle shell
def: "The proteinaceous structure surrounding a gas vesicle." [GOC:ecd]
GO:0033174
name: chloroplast proton-transporting ATP synthase complex, catalytic core CF(1)
def: "The catalytic sector of the mitochondrial hydrogen-transporting ATP synthase; it comprises the catalytic core and central stalk, and is peripherally associated with the chloroplast thylakoid membrane when the entire ATP synthase is assembled. The chloroplast F0 domain contains three alpha, three beta, one gamma, one delta, and one epsilon subunits." [GOC:mah, GOC:pj, PMID:11032839]
GO:0033175
name: chloroplast proton-transporting ATP synthase complex, coupling factor CF(o)
def: "All non-F1 subunits of the chloroplast hydrogen-transporting ATP synthase, including integral and peripheral chloroplast thylakoid membrane proteins." [GOC:mah]
GO:0033176
name: proton-transporting V-type ATPase complex
def: "A proton-transporting two-sector ATPase complex that couples ATP hydrolysis to the transport of protons across a concentration gradient. The resulting transmembrane electrochemical potential of H+ is used to drive a variety of (i) secondary active transport systems via H+-dependent symporters and antiporters and (ii) channel-mediated transport systems. The complex comprises a membrane sector (V0) that carries out proton transport and a cytoplasmic compartment sector (V1) that catalyzes ATP hydrolysis. V-type ATPases are found in the membranes of organelles such as vacuoles, endosomes, and lysosomes, and in the plasma membrane." [GOC:mah, ISBN:0716743663, PMID:16449553]
GO:0033177
name: proton-transporting two-sector ATPase complex, proton-transporting domain
def: "A protein complex that forms part of a proton-transporting two-sector ATPase complex and carries out proton transport across a membrane. The proton-transporting domain (F0, V0, or A0) includes integral and peripheral membrane proteins." [GOC:mah, PMID:10838056]
GO:0033178
name: proton-transporting two-sector ATPase complex, catalytic domain
def: "A protein complex that forms part of a proton-transporting two-sector ATPase complex and catalyzes ATP hydrolysis or synthesis. The catalytic domain (F1, V1, or A1) comprises a hexameric catalytic core and a central stalk, and is peripherally associated with the membrane when the two-sector ATPase is assembled." [GOC:mah, PMID:10838056]
GO:0033179
name: proton-transporting V-type ATPase, V0 domain
def: "A protein complex that forms part of a proton-transporting V-type ATPase and mediates proton transport across a membrane. The V0 complex consists of at least four different subunits (a,c,d and e); six or more c subunits form a proton-binding rotor ring." [GOC:mah, ISBN:0716743663, PMID:16449553]
GO:0033180
name: proton-transporting V-type ATPase, V1 domain
def: "A protein complex that forms part of a proton-transporting V-type ATPase and catalyzes ATP hydrolysis. The V1 complex consists of: (1) a globular headpiece with three alternating copies of subunits A and B that form a ring, (2) a central rotational stalk composed of single copies of subunits D and F, and (3) a peripheral stalk made of subunits C, E, G and H. Subunits A and B mediate the hydrolysis of ATP at three reaction sites associated with subunit A." [GOC:mah, ISBN:0716743663, PMID:16449553]
GO:0033181
name: plasma membrane proton-transporting V-type ATPase complex
def: "A proton-transporting two-sector ATPase complex found in the plasma membrane." [GOC:mah]
GO:0033185
name: dolichol-phosphate-mannose synthase complex
def: "A protein complex that possesses dolichyl-phosphate beta-D-mannosyltransferase activity; contains a catalytic subunit, a regulatory subunit, and a third subunit that stabilizes the complex. In human and several other metazoa, the subunits are named DPM1, DPM2 and DPM3, respectively." [PMID:10835346]
GO:0033186
name: CAF-1 complex
def: "A conserved heterotrimeric protein complex that promotes histone H3 and H4 deposition onto newly synthesized DNA during replication or DNA repair; specifically facilitates replication-dependent nucleosome assembly with the major histone H3 (H3.1). In many species the CAF-1 subunits are designated p150, p60, and p48." [PMID:17065558, PMID:17083276]
GO:0033193
name: Lsd1/2 complex
def: "A nucleosome-binding protein complex that comprises two SWIRM domain histone demethylases and two PHD finger proteins. The complex is involved in transcriptional regulation via heterochromatic silencing and the regulation of chromatin boundary formation, and was first identified in fission yeast." [GOC:vw, PMID:17371846, PMID:17434129, PMID:17440621]
GO:0033202
name: DNA helicase complex
def: "A protein complex that possesses DNA helicase activity." [GOC:mah]
GO:0033203
name: DNA helicase A complex
def: "A homohexameric protein complex that possesses DNA helicase activity; associates with DNA polymerase alpha-primase and translocates in the 5' to 3' direction." [PMID:9341218]
GO:0033254
name: vacuolar transporter chaperone complex
def: "A protein complex that contains four related proteins that have been implicated in several membrane-related processes, such as sorting of H+-translocating ATPases, endocytosis, ER-Golgi trafficking, vacuole fusion, vacuolar polyphosphate homeostasis and the microautophagic scission of vesicles into the vacuolar lumen. The complex is enriched at the vacuolar membrane, but also found in other cellular compartments, including the ER and the cell periphery. In Saccharomyces, the subunits are Vtc1p, Vtc2p, Vtc3p and Vtc4p." [PMID:11823419, PMID:17079729]
GO:0033255
name: SAS acetyltransferase complex
def: "A protein complex that possesses histone acetyltransferase activity and links histone acetylation to the assembly of transcriptionally silent chromatin. In vitro, the complex acetylates lysine 16 of histone H4 and lysine 14 of histone H3, although the latter may not be relevant in vivo. The complex contains a catalytic subunit and at least two other subunits; in Saccharomyces, the catalytic subunit is Sas2p and additional subunits are Sas4p and Sas5p." [PMID:11731480, PMID:12626510, PMID:15788653]
GO:0033256
name: I-kappaB/NF-kappaB complex
def: "A protein complex containing an inhibitory-kappaB (I-kappaB/IKB) protein and one or more copies of an NF-kappaB protein. In the resting state, NF-kappaB dimers are bound to I-kappaB proteins, sequestering NF-kappaB in the cytoplasm." [GOC:bf, GOC:mah, PMID:9407099]
GO:0033257
name: Bcl3/NF-kappaB2 complex
def: "A protein complex containing one Bcl protein and one or more copies of NF-kappaB2; formation of complexes of different stoichiometry depends on the Bcl3:NF-kappaB2 ratio, and allow Bcl3 to exert different regulatory effects on NF-kappaB2-dependent transcription." [GOC:mah, PMID:9407099]
GO:0033263
name: CORVET complex
def: "A multimeric protein complex that acts as an endosomal tethering complex (CORVET = class C core vacuole/endosome tethering) by cooperating with Rab GTPases to capture endosomal vesicles and trap them prior to the action of SNAREs; the complex is involved in endo-lysosomal biogenesis and required for transport between endosome and vacuole. The Saccharomyces cerevisiae complex contains Vps8p, Vps3p, Pep5p, Vps16p, Pep3p, and Vps33p." [PMID:17488625]
GO:0033267
name: axon part
def: "A part of an axon, a cell projection of a neuron." [GOC:jl]
GO:0033268
name: node of Ranvier
def: "An axon part that is a gap in the myelin where voltage-gated sodium channels cluster and saltatory conduction is executed." [GOC:mh]
GO:0033269
name: internode region of axon
def: "An axon part that is located between the nodes of Ranvier and surrounded by compact myelin sheath." [GOC:mah, GOC:mh]
GO:0033270
name: paranode region of axon
def: "An axon part that is located adjacent to the nodes of Ranvier and surrounded by lateral loop portions of myelin sheath." [GOC:mah, GOC:mh, NIF_Subcellular:sao936144858]
GO:0033276
name: transcription factor TFTC complex
def: "A protein complex that does not contain either a TATA-binding protein (TBP) or a TBP-like factor, but is composed of several TAFIIs and other proteins, including a histone acetyltransferase. This complex is able to nucleate transcription initiation by RNA polymerase II, can mediate transcriptional activation, and has histone acetyltransferase activity." [PMID:10373431, PMID:9603525]
GO:0033281
name: TAT protein transport complex
def: "A complex of three proteins integral to the cytoplasmic membrane of bacteria and membranes of organelles derived from bacteria (chloroplasts and mitochondria) involved in membrane transport of folded proteins." [GOC:pamgo_curators]
GO:0033282
name: protein C inhibitor-acrosin complex
def: "A heterodimeric protein complex of protein C inhibitor (SERPINA5) and acrosin; formation of the complex inhibits the protease activity of acrosin." [GOC:pr, PMID:11120760, PMID:7521127]
GO:0033289
name: intraconoid microtubule
def: "A microtubule located such that it threads through the conoid and projects through the polar ring." [GOC:mah, PMID:11901169, PMID:16518471]
GO:0033290
name: eukaryotic 48S preinitiation complex
def: "A protein complex composed of the small ribosomal subunit, eIF3, eIF1A, methionyl-initiatior methionine and a capped mRNA. The complex is initially positioned at the 5'-end of the capped mRNA." [GOC:hjd, PMID:15145049]
GO:0033291
name: eukaryotic 80S initiation complex
def: "A protein complex composed of the large and small ribosomal subunits, methionyl-initiatior tRNA, and the capped mRNA. The initiator tRNA is positioned at the ribosomal P site at the AUG codon corresponding to the beginning of the coding region." [GOC:hjd, PMID:15145049]
GO:0033309
name: SBF transcription complex
def: "A protein complex that binds to the Swi4/6 cell cycle box (SCB) promoter element, consensus sequence CRCGAAA, and activates transcription during the G1/S transition of the cell cycle. In Saccharomyces, the complex contains a heterodimer of the DNA binding protein Swi6p and the activator Swi4p, and is associated with additional proteins known as Whi5p and Msa1p." [GOC:mah, PMID:11206552, PMID:15838511, PMID:18160399, PMID:7917338]
GO:0033391
name: chromatoid body
def: "A ribonucleoprotein complex found in the cytoplasm of male germ cells, composed of exceedingly thin filaments that are consolidated into a compact mass or into dense strands of varying thickness that branch to form an irregular network. Contains mRNAs, miRNAs, and protein components involved in miRNA processing (such as Argonaute proteins and the endonuclease Dicer) and in RNA decay (such as the decapping enzyme DCP1a and GW182)." [PMID:17183363]
GO:0033503
name: HULC complex
def: "A ubiquitin-conjugating enzyme complex that contains two RING finger proteins, which have ubiquitin ligase activity, in addition to a protein with ubiquitin-conjugating enzyme activity; catalyzes the ubiquitination of histone H2B at lysine 119 (or the equivalent residue). In Schizosaccharomyces the subunits are Rhp1, Brl2/Rfp1 and Brl1/Rfp2." [GOC:mah, PMID:17363370, PMID:17374714]
GO:0033551
name: monopolin complex
def: "A protein complex required for clamping microtubule binding sites, ensuring orientation of sister kinetochores to the same pole (mono-orientation) during meiosis I. In the yeast S. cerevisiae this complex consists of Csm1p, Lrs4p, Hrr25p and Mam1p; in S. pombe Psc1 and Mde4 have been identified as subunits." [GOC:mah, GOC:rb, PMID:17627824]
GO:0033553
name: rDNA heterochromatin
def: "A region of heterochromatin located at the rDNA repeats in a chromosome." [GOC:mah]
GO:0033557
name: Slx1-Slx4 complex
def: "A heterodimeric protein complex that possesses an endonuclease activity that specifically cleaves certain types of branched DNA structures; because such structures often form during the replication ribosomal DNA (rDNA) repeats, the complex plays a role in the maintenance of rDNA. The subunits are known as Slx1 and Slx 4 in budding and fission yeasts, and are conserved in eukaryotes." [PMID:14528010, PMID:16467377]
GO:0033565
name: ESCRT-0 complex
def: "A protein complex required for the recycling of Golgi proteins, formation of lumenal membranes and sorting of ubiquitinated proteins into those membranes. This complex includes Vps1p and Hse1p in yeast and the Hrs and STAM proteins in mammals." [GOC:rb, PMID:12055639, PMID:17543868]
GO:0033573
name: high-affinity iron permease complex
def: "A protein complex composed of a multicopper ferroxidase that oxidizes Fe(II) to Fe(III), and a ferric iron permease that transports the produced Fe(III) into the cell. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:jp, PMID:16522632, PMID:8599111]
GO:0033583
name: rhabdomere membrane
def: "The portion of the plasma membrane surrounding the rhabdomere." [GOC:mah]
GO:0033588
name: Elongator holoenzyme complex
def: "A heterohexameric protein complex that is involved in modification of wobble nucleosides in tRNA, and exerts direct effects on transcriptional elongation and exocytosis. The complex can associate physically with hyperphosphorylated RNA polymerase II; it contains two discrete heterotrimeric subcomplexes." [GOC:jh, GOC:mah, GOC:vw, PMID:11689709, PMID:15769872, PMID:17018299, PMID:18755837]
GO:0033593
name: BRCA2-MAGE-D1 complex
def: "A heterodimeric protein complex formed of BRCA2 and MAGE-D1; may mediate the synergistic activities of the two proteins in regulating cell growth." [PMID:15930293]
GO:0033596
name: TSC1-TSC2 complex
def: "A protein complex consisting of at least tumerin and hamartin; its formation may regulate hamartin homomultimer formation. The complex acts as a GTPase activating protein (GAP) for the small GTPase (Rheb), and inhibits the TOR signaling pathway." [PMID:10585443, PMID:17121544, PMID:9580671]
GO:0033597
name: mitotic checkpoint complex
def: "A multiprotein complex that functions as a mitotic checkpoint inhibitor of the anaphase-promoting complex/cyclosome (APC/C). In budding yeast this complex consists of Mad2p, Mad3p, Bub3p and Cdc20p, and in mammalian cells it consists of MAD2, BUBR1, BUB3, and CDC20." [PMID:10704439, PMID:11535616, PMID:11726501, PMID:17650307]
GO:0033607
name: SOD1-Bcl-2 complex
def: "A heterodimeric protein complex formed of superoxide dismutase 1 and Bcl-2. Complex formation is thought to link superoxide dismutase to an apoptotic pathway." [PMID:15233914, PMID:16790527]
GO:0033620
name: Mei2 nuclear dot complex
def: "A ribonucleoprotein complex that forms during meiotic prophase in a fixed position in the horsetail nucleus; contains Mei2 and meiRNA. May play a role in the progression of meiosis I." [GOC:vw, PMID:12808043]
GO:0033643
name: host cell part
def: "Any constituent part of a host cell. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033644
name: host cell membrane
def: "Double layer of lipid molecules as it encloses host cells, and, in eukaryotes, many organelles; may be a single or double lipid bilayer; also includes associated proteins. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033645
name: host cell endomembrane system
def: "A collection of membranous structures involved in transport within the host cell. The main components of the endomembrane system are endoplasmic reticulum, Golgi bodies, vesicles, cell membrane and nuclear envelope. Members of the endomembrane system pass materials through each other or though the use of vesicles. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033646
name: host intracellular part
def: "Any constituent part of the living contents of a host cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033647
name: host intracellular organelle
def: "Organized structure of distinctive morphology and function, occurring within the host cell. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton. Excludes the plasma membrane. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033648
name: host intracellular membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, as found in host cells, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033650
name: host cell mitochondrion
def: "A semiautonomous, self replicating organelle as found in host cells that occurs in varying numbers, shapes, and sizes in the cell cytoplasm. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033651
name: host cell plastid
def: "Any member of a family of organelles as found in the cytoplasm of host cells, which are membrane-bounded and contain DNA. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033652
name: host cell chloroplast
def: "A chlorophyll-containing plastid as found within host cells with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033653
name: host cell chloroplast part
def: "Any constituent part of a chloroplast as it is found in host cells and which are a chlorophyll-containing plastid with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033654
name: host cell chloroplast thylakoid membrane
def: "Any sac-like membranous structures (cisternae) in a chloroplast found in host cells, combined into stacks (grana) and present singly in the stroma (stroma thylakoids or frets) as interconnections between grana. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033655
name: host cell cytoplasm part
def: "Any constituent part of the host cell cytoplasm, all of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0033675
name: pericanalicular vesicle
def: "A membrane-bounded vesicle found near the apical, or pericanalicular, membrane of a hepatocyte; contains proteins involved in bile salt transport and other fluid and solute transport processes." [PMID:15763347, PMID:9790571]
GO:0033698
name: Rpd3L complex
def: "A histone deacetylase complex which deacetylates histones across gene coding regions. Composed of a catalytic histone deacetylase subunit, an Sds-3 family protein, a SIN3 family co-repressor, a WD repeat protein, and a zf- PHD finger (Clr6, Sds3, Pst1, Prw1, Png2 in Schizosaccharomyces pombe; Rpd3p, Sin3p, Ume1p, Pho23p, Sap30p, Sds3p, Cti6p, Rxt2p, Rxt3p, Dep1p, Ume6p and Ash1p in Saccharomyces cerevisiae)." [GOC:vw, PMID:17450151]
GO:0033768
name: SUMO-targeted ubiquitin ligase complex
def: "A nuclear ubiquitin ligase complex that specifically targets SUMOylated proteins; the complex is formed of homodimers or heterodimers of RNF4 family ubiquitin ligases and is conserved in eukaryotes." [GOC:vw, PMID:17762864, PMID:17762865]
GO:0033774
name: basal labyrinth
def: "A region in the lower half of some cells formed from extensive infoldings of the basal plasma membrane; includes cytoplasm adjacent to the infolded membrane." [GOC:mah, GOC:sart, PMID:11640882]
GO:0033985
name: acidocalcisome lumen
def: "The volume enclosed by the membranes of an acidocalcisome." [GOC:mah]
GO:0034044
name: exomer complex
def: "A protein complex that forms a coat structure on vesicles involved in exocytosis of proteins from the trans-Golgi network to the cell surface; in Saccharomyces, the complex contains Chs5p, Chs6p, and Chs6p paralogues." [PMID:16498409, PMID:17000877]
GO:0034045
name: pre-autophagosomal structure membrane
def: "A cellular membrane associated with the pre-autophagosomal structure." [GOC:mah, GOC:rph, PMID:16874040, PMID:17382324]
GO:0034060
name: cyanelle stroma
def: "The space enclosed by the double membrane of a cyanelle." [GOC:rph]
GO:0034064
name: Tor2-Mei2-Ste11 complex
def: "A protein complex that contains the transcription factor Ste11 and the RNA binding protein Mei2; involved in regulation of conjugation in fission yeast." [GOC:vw, PMID:17046992]
GO:0034066
name: RIC1-RGP1 guanyl-nucleotide exchange factor complex
def: "A protein complex that acts as a nucleotide exchange factor for the GTPase Ypt6p, and is required for fusion of endosome-derived vesicles with the Golgi." [GOC:jh, GOC:mah, PMID:10990452]
GO:0034081
name: polyketide synthase complex
def: "A protein complex that carries out enzymatic reactions involved in the biosynthesis of polyketides, any of a diverse group of natural products synthesized via linear poly-beta-ketones." [GOC:mah, PMID:12636085]
GO:0034082
name: type II polyketide synthase complex
def: "A polyketide synthase complex that consists of several different polypeptide chains, each of which catalyzes a single reaction." [GOC:cb, GOC:mah, PMID:12636085]
GO:0034083
name: type III polyketide synthase complex
def: "A polyketide synthase complex that consists of two identical ketosynthase polypeptides." [GOC:cb, PMID:12636085]
GO:0034098
name: Cdc48p-Npl4p-Ufd1p AAA ATPase complex
def: "A multiprotein ATPase complex required for the efficient dislocation of ER-lumenal degradation substrates, and their subsequent proteolysis by the proteasome. In budding yeast, this complex includes Cdc48p, Npl4p and Ufd1p proteins." [PMID:11813000, PMID:16179952]
GO:0034099
name: luminal surveillance complex
def: "A multiprotein complex that recognizes ERAD-luminal misfolded substrates and brings them to the ubiquitination/extraction machinery. In yeast, this complex consists of Yos9p, Kar2p and Hrd3p proteins." [PMID:16873065]
GO:0034206
name: enhanceosome
def: "A protein-DNA complex formed by the association of a distinct set of general and specific transcription factors with a region of enhancer DNA. The cooperative assembly of an enhanceosome confers specificity of transcriptional regulation." [PMID:11250145, PMID:17574024]
GO:0034245
name: mitochondrial DNA-directed RNA polymerase complex
def: "A DNA-directed RNA polymerase complex located in the mitochondrion. Mitochondrial RNA polymerase is composed of two subunits, a catalytic core, which resembles the enzymes from bacteriophage T7 and T3, and a specificity factor required for promoter recognition, which is similar to members of the eubacterial sigma factor family. In S. cerevisiae, these are encoded by the nuclear genes RPO41 and MTF1 and the specificity factor, required for promoter recognition and initiation, is not present in the elongating form." [GOC:krc, GOC:mah, PMID:7929382]
GO:0034270
name: CVT complex
def: "A protein complex that is involved in the CVT pathway. In budding yeast, the CVT complex consists of multimers of preApe1p." [GOC:rb, PMID:15659643]
GO:0034271
name: phosphatidylinositol 3-kinase complex, class III, type I
def: "A class III phosphatidylinositol 3-kinase complex that is involved in autophagy. In budding yeast, this complex consists of Vps30p, Vps34p, Apg14p and Vps15p." [GOC:ha, GOC:rb, PMID:11157979, PMID:16421251]
GO:0034272
name: phosphatidylinositol 3-kinase complex, class III, type II
def: "A class III phosphatidylinositol 3-kinase complex that is involved in vacuolar protein sorting (VPS) via endosomes. In budding yeast, this complex consists of Vps30p, Vps34p, Vps38 and Vps15p." [GOC:ha, GOC:rb, PMID:11157979, PMID:16421251]
GO:0034273
name: ATG1/UKL1 signaling complex
def: "A protein complex consisting of ATG1/UKL1 and ATG13 along with other proteins that regulate its function (e.g. ATG17 in yeast). This complex has serine/threonine protein kinase activity and is involved in autophagosome formation." [GOC:rb, PMID:15743910]
GO:0034274
name: Atg12-Atg5-Atg16 complex
def: "A protein complex required for the expansion of the autophagosomal membrane. In budding yeast, this complex consists of Atg12p, Atg5p and Atg16p." [GOC:rb, PMID:17986448]
GO:0034357
name: photosynthetic membrane
def: "A membrane enriched in complexes formed of reaction centers, accessory pigments and electron carriers, in which photosynthetic reactions take place." [GOC:ds, GOC:mah]
GO:0034358
name: plasma lipoprotein particle
def: "A spherical particle with a hydrophobic core of triglycerides and/or cholesterol esters, surrounded by an amphipathic monolayer of phospholipids, cholesterol and apolipoproteins. Plasma lipoprotein particles transport lipids, which are non-covalently associated with the particles, in the blood or lymph." [GOC:BHF, GOC:expert_pt, GOC:rl]
GO:0034359
name: mature chylomicron
def: "A chylomicron that contains apolipoprotein C2 (APOC2), a cofactor for lipoprotein lipase (LPL) activity, and has a mean diameter of 500 nm and density of 0.95g/ml. Mature chylomicron particles transport exogenous (dietary) lipids from the intestines to other body tissues, via the blood and lymph." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034360
name: chylomicron remnant
def: "A lipoprotein particle that is derived from a mature chylomicron particle by the removal of triglycerides from the chylomicron core by lipoprotein lipase and the subsequent loss of surface components. It characteristically contains apolipoprotein E (APOE) and is cleared from the blood by the liver." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034361
name: very-low-density lipoprotein particle
def: "A triglyceride-rich lipoprotein particle that is typically composed of APOB100, APOE and APOCs and has a density of about 1.006 g/ml and a diameter of between 20-80 nm. It is found in blood and transports endogenous products (newly synthesized cholesterol and triglycerides) from the liver." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034362
name: low-density lipoprotein particle
def: "A lipoprotein particle, rich in cholesterol esters and low in triglycerides that is typically composed of APOB100 and APOE and has a density of 1.02-1.06 g/ml and a diameter of between 20-25 nm. LDL particles are formed from VLDL particles (via IDL) by the loss of triglyceride and gain of cholesterol ester. They transport endogenous cholesterol (and to some extent triglycerides) from peripheral tissues back to the liver." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034363
name: intermediate-density lipoprotein particle
def: "A triglyceride-rich lipoprotein particle that typically contains APOB100, APOE and APOCs and has a density of 1.006-1.019 g/ml and a diameter of between 25-30 nm. IDL particles are found in blood and are formed by the delipidation of very-low-density lipoprotein particles (VLDL). IDL particles are removed from blood by the liver, following binding to the APOE receptor, or are converted to low-density lipoprotein (LDL)." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034364
name: high-density lipoprotein particle
def: "A lipoprotein particle with a high density (typically 1.063-1.21 g/ml) and a diameter of 5-10 nm that contains APOAs and may contain APOCs and APOE; found in blood and carries lipids from body tissues to the liver as part of the reverse cholesterol transport process." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:pde, GOC:rl]
GO:0034365
name: discoidal high-density lipoprotein particle
def: "A newly formed high-density lipoprotein particle; consists of a phospholipid bilayer surrounded by two or more APOA1 molecules. The discoidal HDL particle is formed when lipid-free or lipid-poor APOA1 acquires phospholipids and unesterified cholesterol from either cell membranes or triglyceride-rich lipoproteins (undergoing lipolysis by lipoprotein lipase)." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034366
name: spherical high-density lipoprotein particle
def: "A mature high-density lipoprotein (HDL) particle, converted from discoidal HDL particles following the esterification of cholesterol in the particle by phosphatidylcholine-sterol O-acyltransferase (lecithin cholesterol acyltransferase; LCAT)." [GOC:BHF, GOC:expert_pt, GOC:mah, GOC:rl]
GO:0034385
name: triglyceride-rich lipoprotein particle
def: "A plasma lipoprotein particle that has a hydrophobic core enriched in triglycerides surrounded by an amphipathic monolayer of phospholipids, cholesterol and apolipoproteins. Triglyceride-rich lipoproteinparticles transport lipids, which are non-covalently associated with the particles, in the blood." [GOC:BHF, GOC:mah, GOC:rl]
GO:0034388
name: Pwp2p-containing subcomplex of 90S preribosome
def: "A protein complex that forms a subcomplex of the 90S preribosome and can interact directly with the 5' External Transcribed Spacer (ETS) of the full length pre-rRNA transcript. In S. cerevisiae, it sediments at 25-30 S and is composed of Pwp2p, Dip2p, Utp21p, Utp13p, Utp18p, and Utp6p." [GOC:krc, PMID:15231838]
GO:0034399
name: nuclear periphery
def: "The portion of the nuclear lumen proximal to the inner nuclear membrane." [GOC:krc, GOC:mah]
GO:0034400
name: gerontoplast
def: "A plastid found in senescing, formerly green tissues that is derived from a chloroplast that undergoes an organized developmental program of senescence." [PMID:12654863, PMID:16151876]
GO:0034422
name: aleurone grain lumen
def: "The volume enclosed by the membrane of an aleurone grain." [GOC:rph]
GO:0034423
name: autophagic vacuole lumen
def: "The volume enclosed within the autophagic vacuole membrane." [GOC:rph]
GO:0034424
name: Vps55/Vps68 complex
def: "A membrane-associated protein complex that is required for a late stage of endosomal transport. In budding yeast, this complex consists of Vps55p and Vps68p proteins." [PMID:18216282]
GO:0034425
name: etioplast envelope
def: "The double lipid bilayer enclosing the etioplast and separating its contents from the rest of the cytoplasm; includes the intermembrane space." [GOC:mah]
GO:0034426
name: etioplast membrane
def: "Either of the lipid bilayers that surround a etioplast and form the etioplast envelope." [GOC:rph]
GO:0034430
name: monolayer-surrounded lipid storage body outer lipid monolayer
def: "The single layer of phopholipids surrounding a lipid storage body." [GOC:rph]
GO:0034448
name: EGO complex
def: "A vacuolar membrane-associated protein complex that is required for activation of microautophagy during exit from rapamycin-induced growth arrest. In budding yeast, S. cerevisiae, this complex includes Gtr1p, Gtr2p, Meh1p, and Slm4p." [GOC:dgf, GOC:rb, PMID:15989961, PMID:16732272, PMID:19748353]
GO:0034451
name: centriolar satellite
def: "A small (70-100 nm) cytoplasmic granule that contains a number of centrosomal proteins; centriolar satellites traffic toward microtubule minus ends and are enriched near the centrosome." [GOC:BHF, PMID:10579718, PMID:12403812]
GO:0034455
name: t-UTP complex
def: "A protein complex that forms a subcomplex of the 90S preribosome and is required for the subsequent assembly of the rest of the preribosome. In S. cerevisiae, it is composed of Utp5p, Utp4p, Nan1p, Utp8p, Utp9p, Utp10 and Utp15p." [GOC:krc, GOC:mah, GOC:vw, PMID:17515605]
GO:0034456
name: UTP-C complex
def: "A protein complex that forms a subcomplex of the 90S preribosome. In S. cerevisiae, it is composed of Rrp7p, Utp22p, Ckb1p, Cka1p, Ckb2p and Cka2p." [GOC:mah, PMID:17515605]
GO:0034457
name: Mpp10 complex
def: "A protein complex that forms a subcomplex of the 90S preribosome. In S. cerevisiae, it is composed of Mpp10p, Imp3p and Imp4p." [GOC:mah, PMID:17515605]
GO:0034464
name: BBSome
def: "A protein complex that associates with the primary cilium and is involved in cilium biogenesis; consists of seven conserved proteins: BBS1, BBS2, BBS4, BBS5, BBS7, BBS8 and BBS9." [GOC:BHF, PMID:17574030]
GO:0034466
name: chromaffin granule lumen
def: "The volume enclosed by the membrane of a chromaffin granule." [GOC:rph]
GO:0034467
name: esterosome lumen
def: "The volume enclosed by the membrane of an esterosome." [GOC:rph]
GO:0034468
name: glycosome lumen
def: "The volume enclosed by the membrane of a glycosome." [GOC:rph]
GO:0034469
name: Golgi stack lumen
def: "The volume enclosed by any of the membranes of the thin, flattened cisternae that form the central portion of the Golgi complex." [GOC:mah]
GO:0034492
name: hydrogenosome lumen
def: "The volume enclosed by the hydrogenosome membrane." [GOC:rph]
GO:0034493
name: melanosome lumen
def: "The volume enclosed by the melanosome membrane." [GOC:rph]
GO:0034494
name: microneme lumen
def: "The volume enclosed by the microneme membrane." [GOC:rph]
GO:0034495
name: protein storage vacuole lumen
def: "The volume enclosed by the protein storage vacuole membrane." [GOC:rph]
GO:0034506
name: chromosome, centromeric core domain
def: "The innermost portion of the centromeric region of a chromosome, encompassing the core region of a chromosome centromere and the proteins that bind to it." [GOC:mah, GOC:vw]
GO:0034507
name: chromosome, centromeric outer repeat region
def: "The portion of the centromeric region of a chromosome that flanks the core region, encompassing repeated regions of a chromosome centromere and the proteins that bind to it." [GOC:mah, GOC:vw]
GO:0034515
name: proteasome storage granule
def: "A multisubunit proteasome complex that localizes in the cytoplasm as dot-like structures when cells are in a quiescent state." [GOC:rb, PMID:18504300]
GO:0034518
name: RNA cap binding complex
def: "Any protein complex that binds to a specialized RNA cap structure at any time in the lifetime of the RNA." [GOC:mah]
GO:0034519
name: cytoplasmic RNA cap binding complex
def: "A protein complex found in the cytoplasm that binds the 5' cap structure of an mRNA, and typically consists of the cap-binding protein eIF4E, the adaptor protein eIF4G, and a multi-factor complex comprising eIF1, eIF2, eIF3 and eIF5. This complex mediates recruitment of the 40S subunit to mRNA." [PMID:16405910]
GO:0034591
name: rhoptry lumen
def: "The volume enclosed by the rhoptry membrane." [GOC:rph, PMID:17997128]
GO:0034592
name: synaptic vesicle lumen
def: "The volume enclosed by the synaptic vesicle membrane." [GOC:rph]
GO:0034615
name: GCH1 complex
def: "A protein complex that possesses GTP cyclohydrolase I activity. In E. coli and human, the complex is a homodecamer, and monomers are catalytically inactive." [PMID:16696853]
GO:0034646
name: organelle-enclosing lipid monolayer
def: "A lipid monolayer that surrounds and encloses an organelle." [GOC:mah]
GO:0034657
name: GID complex
def: "A protein complex with ubiquitin ligase activity that is involved in proteasomal degradation of fructose-1,6-bisphosphatase (FBPase) and phosphoenolpyruvate carboxykinase during the transition from gluconeogenic to glycolytic growth conditions. In S. cerevisiae, the GID (Glucose Induced degradation Deficient) complex consists of Vid30p, Rmd5p, Vid24p, Vid28p, Gid7p, Gid8p, and Fyv10p." [PMID:12686616, PMID:18508925]
GO:0034662
name: CFTR-NHERF-ezrin complex
def: "A protein complex that contains ezrin, Na+/H+ exchanger regulatory factor (NHERF, also called EBP50), and two copies of the cystic fibrosis transmembrane conductance regulator (CFTR). The CFTR molecules interact with NHERF via their cytoplasmic tail domains; the complex is thought to link the CFTR channel to the actin cytoskeleton and contribute to the regulation of channel activity." [PMID:16129695, PMID:16798722, PMID:16926444]
GO:0034663
name: endoplasmic reticulum chaperone complex
def: "A protein complex that is located in the endoplasmic reticulum and is composed of chaperone proteins, including BiP, GRP94; CaBP1, protein disulfide isomerase (PDI), ERdj3, cyclophilin B, ERp72, GRP170, UDP-glucosyltransferase, and SDF2-L1." [PMID:12475965]
GO:0034664
name: Ig heavy chain-bound endoplasmic reticulum chaperone complex
def: "A protein complex that is located in the endoplasmic reticulum (ER) and is formed by the association of an immunoglobulin heavy chain with the proteins of the ER chaperone complex; the latter include BiP, GRP94; CaBP1, protein disulfide isomerase (PDI), ERdj3, cyclophilin B, ERp72, GRP170, UDP-glucosyltransferase, and SDF2-L1." [PMID:12475965]
GO:0034665
name: integrin alpha1-beta1 complex
def: "An integrin complex that comprises one alpha1 subunit and one beta1 subunit." [PMID:12297042]
GO:0034666
name: integrin alpha2-beta1 complex
def: "An integrin complex that comprises one alpha2 subunit and one beta1 subunit." [PMID:12297042]
GO:0034667
name: integrin alpha3-beta1 complex
def: "An integrin complex that comprises one alpha3 subunit and one beta1 subunit." [PMID:12297042]
GO:0034668
name: integrin alpha4-beta1 complex
def: "An integrin complex that comprises one alpha4 subunit and one beta1 subunit." [PMID:12297042]
GO:0034669
name: integrin alpha4-beta7 complex
def: "An integrin complex that comprises one alpha4 subunit and one beta7 subunit." [PMID:12297042]
GO:0034673
name: inhibin-betaglycan-ActRII complex
def: "A protein complex that consists of inhibin, type III transforming growth factor beta receptor (also known as betaglycan), and the type II activin receptor ActRII. The complex is thought to negatively regulate the activity of activin B." [GOC:BHF, PMID:10746731]
GO:0034674
name: integrin alpha5-beta1 complex
def: "An integrin complex that comprises one alpha5 subunit and one beta1 subunit." [PMID:12297042]
GO:0034675
name: integrin alpha6-beta1 complex
def: "An integrin complex that comprises one alpha6 subunit and one beta1 subunit." [PMID:12297042]
GO:0034676
name: integrin alpha6-beta4 complex
def: "An integrin complex that comprises one alpha6 subunit and one beta1 subunit." [PMID:12297042]
GO:0034677
name: integrin alpha7-beta1 complex
def: "An integrin complex that comprises one alpha7 subunit and one beta1 subunit." [PMID:12297042]
GO:0034678
name: integrin alpha8-beta1 complex
def: "An integrin complex that comprises one alpha8 subunit and one beta1 subunit." [PMID:12297042]
GO:0034679
name: integrin alpha9-beta1 complex
def: "An integrin complex that comprises one alpha9 subunit and one beta1 subunit." [PMID:12297042]
GO:0034680
name: integrin alpha10-beta1 complex
def: "An integrin complex that comprises one alpha10 subunit and one beta1 subunit." [PMID:12297042]
GO:0034681
name: integrin alpha11-beta1 complex
def: "An integrin complex that comprises one alpha11 subunit and one beta1 subunit." [PMID:12297042]
GO:0034682
name: integrin alphav-beta1 complex
def: "An integrin complex that comprises one alphav subunit and one beta1 subunit." [PMID:12297042]
GO:0034683
name: integrin alphav-beta3 complex
def: "An integrin complex that comprises one alphav subunit and one beta3 subunit." [PMID:12297042]
GO:0034684
name: integrin alphav-beta5 complex
def: "An integrin complex that comprises one alphav subunit and one beta5 subunit." [PMID:12297042]
GO:0034685
name: integrin alphav-beta6 complex
def: "An integrin complex that comprises one alphav subunit and one beta6 subunit." [PMID:12297042]
GO:0034686
name: integrin alphav-beta8 complex
def: "An integrin complex that comprises one alphav subunit and one beta8 subunit." [PMID:12297042]
GO:0034687
name: integrin alphaL-beta2 complex
def: "An integrin complex that comprises one alphaL subunit and one beta2 subunit." [PMID:12297042]
GO:0034688
name: integrin alphaM-beta2 complex
def: "An integrin complex that comprises one alphaM subunit and one beta2 subunit." [PMID:12297042]
GO:0034689
name: integrin alphaX-beta2 complex
def: "An integrin complex that comprises one alphaX subunit and one beta2 subunit." [PMID:12297042]
GO:0034690
name: integrin alphaD-beta2 complex
def: "An integrin complex that comprises one alphaD subunit and one beta2 subunit." [PMID:12297042]
GO:0034691
name: integrin alphaE-beta7 complex
def: "An integrin complex that comprises one alphaE subunit and one beta7 subunit." [PMID:12297042]
GO:0034692
name: E.F.G complex
def: "A protein complex that comprises three core spliceosomal proteins, designated E, F, and G. Formation of the E.F.G complex is essential but not sufficient for the formation of a stable U1 snRNP complex." [PMID:8641291]
GO:0034693
name: U11/U12 snRNP
def: "A ribonucleoprotein complex formed by the association of the U11 and U12 small nuclear ribonucleoproteins." [GOC:mah, PMID:15146077]
GO:0034702
name: ion channel complex
def: "A protein complex that spans a membrane and forms a water-filled channel across the phospholipid bilayer allowing selective ion transport down its electrochemical gradient." [GOC:mah, ISBN:071673706X]
GO:0034703
name: cation channel complex
def: "An ion channel complex through which cations pass." [GOC:mah]
GO:0034704
name: calcium channel complex
def: "An ion channel complex through which calcium ions pass." [GOC:mah]
GO:0034705
name: potassium channel complex
def: "An ion channel complex through which potassium ions pass." [GOC:mah]
GO:0034706
name: sodium channel complex
def: "An ion channel complex through which sodium ions pass." [GOC:mah]
GO:0034707
name: chloride channel complex
def: "An ion channel complex through which chloride ions pass." [GOC:mah]
GO:0034708
name: methyltransferase complex
def: "A protein complex that possesses methyltransferase activity." [GOC:mah]
GO:0034709
name: methylosome
def: "A large (20 S) protein complex that possesses protein arginine methyltransferase activity and modifies specific arginines to dimethylarginines in the arginine- and glycine-rich domains of several spliceosomal Sm proteins, thereby targeting these proteins to the survival of motor neurons (SMN) complex for assembly into small nuclear ribonucleoprotein (snRNP) core particles. Proteins found in the methylosome include the methyltransferase JBP1 (PRMT5), pICln (CLNS1A), MEP50 (WDR77), and unmethylated forms of SM proteins that have RG domains." [PMID:11713266, PMID:11756452]
GO:0034715
name: pICln-Sm protein complex
def: "A protein complex that contains pICln (CLNS1A) and several Sm proteins, including SmD1, SmD2, SmE, SmF, and SmG." [GOC:mah, PMID:11713266]
GO:0034716
name: Gemin3-Gemin4-Gemin5 complex
def: "A protein complex that contains Gemin3 (DDX20), Gemin4, and Gemin5, and can bind to snRNAs; may be an intermediate in SMN complex assembly." [GOC:mah, PMID:17640873]
GO:0034717
name: Gemin6-Gemin7-unrip complex
def: "A protein complex that contains Gemin6, Gemin7, and unrip (STRAP), and can bind to snRNAs; may play a role in snRNP assembly." [GOC:mah, PMID:17640873]
GO:0034718
name: SMN-Gemin2 complex
def: "A protein complex that contains the survival motor neuron (SMN) protein and Gemin2; may form the stable core of the larger SMN complex." [GOC:mah, PMID:17640873]
GO:0034719
name: SMN-Sm protein complex
def: "A protein complex formed by the association of several methylated Sm proteins with the SMN complex; the latter contains the survival motor neuron (SMN) protein and at least eight additional integral components, including the Gemin2-8 and unrip proteins; additional proteins, including galectin-1 and galectin-3, are also found in the SMN-SM complex. The SMN-Sm complex is involved in spliceosomal snRNP assembly in the cytoplasm." [PMID:11522829, PMID:17401408]
GO:0034730
name: SmD-containing SMN-Sm protein complex
def: "An SMN-Sm protein complex formed by the association of the methylated Sm proteins B/B', D1, D2, D3, E, F, and G with the SMN complex." [PMID:12975319, PMID:17401408]
GO:0034731
name: Lsm-containing SMN-Sm protein complex
def: "An SMN-Sm protein complex formed by the association of the methylated Sm proteins B/B', D3, E, F, and G, and Lsm10 and Lsm11, with the SMN complex. This complex forms Sm cores on U7 snRNA." [PMID:12975319, PMID:17401408]
GO:0034732
name: transcription factor TFIIIB-alpha complex
def: "A transcription factor TFIIIB-beta complex that contains the TATA-binding protein (TBP), B'' and a specialized homolog of the conserved subunit BRF referred to as BRFU or TFIIIB50, which found in human but not conserved in yeast; the complex is involved in the regulation of transcription from type 3 (upstream) RNA polymerase III promoters." [PMID:11433012]
GO:0034733
name: transcription factor TFIIIB-beta complex
def: "A transcription factor TFIIIB-beta complex that contains the TATA-binding protein (TBP), B'' and BRF, and is involved in the regulation of transcription from type 2 RNA polymerase III promoters." [PMID:11433012]
GO:0034734
name: transcription factor TFIIIC1 complex
def: "A transcription factor complex that forms part of the TFIIIC complex, observed in human. The complex is poorly characterized, but contains the 250-kDa form of HsBdp1, and is thought to include nuclear factor 1 (NF1). It stimulates binding by human TFIIIC2 and is required for transcription activity." [GOC:mah, PMID:11433012, PMID:15096501]
GO:0034735
name: transcription factor TFIIIC2 complex
def: "A transcription factor complex that forms part of the TFIIIC complex, observed in human; composed of five subunits (GTF3C1/hTFIIIC220/TFIIICalpha, GTF3C2/hTFIIIC110/TFIIICbeta, GTF3C3/hTFIIIC102/TFIIICgamma, GTF3C4/hTFIIIC90/TFIIICdelta and GTF3C5/hTFIIIC63/TFIIICepsilon in human) that together recognize the type 2 RNA polymerase III promoter." [GOC:mah, PMID:11433012]
GO:0034740
name: TFIIIC-TOP1-SUB1 complex
def: "A protein complex that contains TFIIIC, topoisomerase 1, and Sub1/PC4. Characterized in human, the complex is involved in regulating transcription from RNA polymerase III (Pol III) promoters. Topoisomerase 1 and Sub1 enhance the accuracy of transcription termination, and promote reinitiation by Pol III." [PMID:9660958]
GO:0034741
name: APC-tubulin-IQGAP1 complex
def: "A protein complex that contains the tumor suppressor protein adenomatous polyposis coli (APC), alpha-tubulin, gamma-tubulin, and the Rac1 and Cdc42 effector IQGAP1; may play a role in cytoskeleton organization." [PMID:17126424]
GO:0034743
name: APC-IQGAP complex
def: "A protein complex that contains the tumor suppressor protein adenomatous polyposis coli (APC) and the Rac1 and Cdc42 effector IQGAP1; may play a role in cytoskeleton organization and cell migration." [PMID:15572129]
GO:0034744
name: APC-IQGAP1-Cdc42 complex
def: "A protein complex that contains the tumor suppressor protein adenomatous polyposis coli (APC), the small GTPase Cdc42, and the Rac1 and Cdc42 effector IQGAP1; may play a role in cytoskeleton organization and cell migration." [PMID:15572129]
GO:0034745
name: APC-IQGAP1-Rac1 complex
def: "A protein complex that contains the tumor suppressor protein adenomatous polyposis coli (APC), the small GTPase Rac1, and the Rac1 and Cdc42 effector IQGAP1; may play a role in cytoskeleton organization and cell migration." [PMID:15572129]
GO:0034746
name: APC-IQGAP1-CLIP-170 complex
def: "A protein complex that contains the tumor suppressor protein adenomatous polyposis coli (APC), the small GTPase Cdc42, and CLIP-170; may play a role in cytoskeleton organization and cell migration." [PMID:15572129]
GO:0034748
name: Par3-APC-KIF3A complex
def: "A protein complex that contains Par3, the tumor suppressor protein adenomatous polyposis coli (APC), and the kinesin-related protein KIF3A; involved in establishing neuronal cell polarity." [PMID:15556865]
GO:0034749
name: Scrib-APC complex
def: "A protein complex that contains the Scribble protein (a cell polarity determinant) and the tumor suppressor protein adenomatous polyposis coli (APC); may be involved in the control of cell proliferation." [PMID:16611247]
GO:0034750
name: Scrib-APC-beta-catenin complex
def: "A protein complex that contains the Scribble protein (a cell polarity determinant), the tumor suppressor protein adenomatous polyposis coli (APC), and beta-catenin; may be involved in the control of cell proliferation." [PMID:16611247]
GO:0034751
name: aryl hydrocarbon receptor complex
def: "A protein complex that acts as an aryl hydrocarbon (Ah) receptor. Cytosolic and nuclear Ah receptor complexes have different subunit composition, but both contain the ligand-binding subunit AhR." [GOC:mah, PMID:7598497]
GO:0034752
name: cytosolic aryl hydrocarbon receptor complex
def: "An aryl hydrocarbon receptor complex found in the cytosol, in which the ligand-binding subunit AhR is not bound to ligand; consists of AhR, two molecules of HSP90, the protein kinase c-Src, and the immunophilin XAP2/AIP." [PMID:7598497, PMID:8937476, PMID:9447995]
GO:0034753
name: nuclear aryl hydrocarbon receptor complex
def: "An aryl hydrocarbon receptor (AhR) complex found in the nucleus; ; consists of ligand-bound AhR and the aryl hydrocarbon receptor nuclear translocator (ARNT)." [PMID:7598497]
GO:0034774
name: secretory granule lumen
def: "The volume enclosed by the membrane of a secretory granule." [GOC:rph]
GO:0034777
name: recycling endosome lumen
def: "The volume enclosed by the membranes of a recycling endosome." [GOC:rph]
GO:0034967
name: Set3 complex
def: "A histone deacetylase complex that is involved in transcriptional regulation. In S. cerevisiae, this complex consists of Set3p, Snt1p, Hos4p, Sif2p, Cpr1p, Hos2p, and Hst1p." [GOC:ds, PMID:11711434]
GO:0034973
name: Sid2-Mob1 complex
def: "A protein complex that contains a protein kinase (Sid2 in S. pombe) and its regulatory subunit (Mob1). The Sid2p-Mob1p kinase complex is a component of the septation initiation network in fission yeast (called the mitotic exit network in S. cerevisiae) and is required for cytokinesis. The analogous complex in S. cerevisiae is called Dbf2p-Mob1p complex." [GOC:vw, PMID:10837231, PMID:15060149]
GO:0034974
name: Swi5-Swi2 complex
def: "A protein complex involved that contains proteins known in Schizosaccharomyces as Swi5 monomers and Swi2, and is involved in mating type switching." [PMID:14663140]
GO:0034977
name: ABIN2-NFKB1-MAP3K8 complex
def: "A protein complex that contains the precursor form of NF-kappaB (p105), the NF-kappaB inhibitor ABIN-2, and the kinase TPL-2 (MAP3K8); the complex stabilizes TPL-2 and is involved in signaling in lipopolysaccharide (LPS)-stimulated macrophages." [PMID:15169888]
GO:0034978
name: PDX1-PBX1b-MRG1 complex
def: "A protein complex that contains the homeodomain proteins PDX1, PBX1b and MRG1 (MEIS2) and is involved in the transcriptional regulation of pancreatic acinar cell-specific genes." [PMID:11279116, PMID:9710595]
GO:0034980
name: FHL2-CREB complex
def: "A protein complex that contains CREB and FHL2, and is involved in transcriptional regulation." [PMID:11046156]
GO:0034981
name: FHL3-CREB complex
def: "A protein complex that contains CREB and FHL3, and is involved in transcriptional regulation." [PMID:11046156]
GO:0034985
name: Ecsit-NDUFAF1 complex
def: "Any large protein complex that contains Ecsit and NDUFAF1, is located in the mitochondrion, and is involved in the assembly of complex I of the oxidative phosphorylation system. In mammalian cells, three complexes of approximately 500, 600, and 850 kDa containing the 45 kDa isoform of Ecsit and NDUFAF1 have been observed." [PMID:17344420]
GO:0034990
name: nuclear mitotic cohesin complex
def: "A cohesin complex that mediates sister chromatid cohesion in the nucleus during mitosis; has a subunit composition distinct from that of the meiotic cohesin complex." [GOC:mah]
GO:0034991
name: nuclear meiotic cohesin complex
def: "A cohesin complex that mediates sister chromatid cohesion in the nucleus during meiosis; has a subunit composition distinct from that of the meiotic cohesin complex." [GOC:mah]
GO:0034992
name: microtubule organizing center attachment site
def: "A region of the nuclear envelope to which a microtubule organizing center (MTOC) attaches; protein complexes embedded in the nuclear envelope mediate direct or indirect linkages between the microtubule cytoskeleton and the nuclear envelope." [GOC:mah, PMID:18692466]
GO:0034993
name: LINC complex
def: "A protein complex that spans the nuclear outer and inner membranes, thereby linking the major cytoplasmic cytoskeleton elements to the nuclear lumen; the complex is conserved in eukaryotes and contains proteins with SUN and KASH domains." [GOC:mah, PMID:18692466]
GO:0034995
name: SC5b-7 complex
def: "A protein complex that consist of complement components C5b6 and C7 stably inserted in a cell membrane. Formation of the SC5b-7 complex is the first phase of membrane attack complex assembly." [PMID:10090939]
GO:0034996
name: RasGAP-Fyn-Lyn-Yes complex
def: "A protein complex that consists of a GTPase activator protein (GAP) for Ras and three Src family protein tyrosine kinases, Fyn, Lyn and Yes. The complex is involved in signaling upon platelet activation." [PMID:1544885]
GO:0034997
name: alphav-beta5 integrin-vitronectin complex
def: "A protein complex that comprises one integrin alphav subunit, one integrin beta5 subunit, and vitronectin." [PMID:1694173, Reactome:REACT_14045.1]
GO:0034998
name: oligosaccharyltransferase I complex
def: "An oligosaccharyltransferase (OST) complex that contains at least seven polypeptides and is the major OST complex in mammalian cells. Of the three forms of mammalian OST complex identified, the OSTI complex has the weakest affinity for ribosomes." [PMID:15835887]
GO:0034999
name: oligosaccharyltransferase II complex
def: "An oligosaccharyltransferase (OST) complex that contains the seven polypeptides found in OST complex I, plus heterotrimeric Sec61alpha-beta-gamma. Of the three forms of mammalian OST complexes identified, the OSTII complex has intermediate affinity for ribosomes." [GOC:BHF, PMID:15835887]
GO:0035000
name: oligosaccharyltransferase III complex
def: "An oligosaccharyltransferase (OST) complex that contains the seven polypeptides found in OST complex I, plus heterotrimeric Sec61alpha-beta-gamma and the tetrameric TRAP complex. Of the three forms of mammalian OST complexes identified, the OSTIII complex has the strongest affinity for ribosomes." [PMID:15835887]
GO:0035003
name: subapical complex
def: "The most apical region of the lateral plasma membrane of an invertebrate epithelial cell. The subapical complex lies above the zonula adherens and the septate junction, and is comparable to the position of the tight junction of vertebrate cells." [PMID:11752566, PMID:12500938]
GO:0035012
name: polytene chromosome, telomeric region
def: "The terminal region of a polytene chromosome." [GOC:bf]
GO:0035032
name: phosphatidylinositol 3-kinase complex, class III
def: "A phosphatidylinositol 3-kinase complex that contains a catalytic class III phosphoinositide 3-kinase (PI3K) subunit bound to a regulatory (adaptor) subunit. Additional adaptor proteins may be present. Class III PI3Ks have a substrate specificity restricted to phosphatidylinositol (PI)." [GOC:bf, PMID:9255069]
GO:0035059
name: RCAF complex
def: "A protein complex that facilitates the assembly of nucleosomes on to newly synthesized DNA. In Drosophila, the complex comprises ASF1 and histones H3 and H4." [GOC:bf, PMID:10591219]
GO:0035060
name: brahma complex
def: "A SWI/SNF-type complex that contains the ATPase product of the Drosophila brahma gene, or an ortholog thereof." [GOC:bf, PMID:10809665, PMID:12482982]
GO:0035061
name: interchromatin granule
def: "A class of nuclear body measuring 20-25 nm in diameter and distributed throughout the interchromatin space, linked together by thin fibrils. They are believed to be storage centers for various snRNAs, snRNPs, serine/arginine-rich proteins and RNA polymerase II. A typical mammalian cell contains 25-50 clusters of interchromatin granules. Interchromatin granule clusters do not contain the heterogeneous nuclear RNA-binding proteins (hnRNPs)." [GOC:bf, PMID:10984439]
GO:0035062
name: omega speckle
def: "A nucleoplasmic speckle distributed in the interchromatin space of cells in close proximity to chromatin. Omega speckles are distinct from interchromatin granules and contain heterogeneous nuclear RNA-binding proteins (hnRNPs)." [GOC:bf, PMID:10984439]
GO:0035068
name: micro-ribonucleoprotein complex
def: "A complex containing both protein and micro-RNA (miRNA) molecules. miRNAs are approximately 22 nucleotide noncoding RNAs derived from endogenous genes; they are processed from the stem of a longer hairpin like structure termed a pre-miRNA." [GOC:bf, PMID:14559182]
GO:0035097
name: histone methyltransferase complex
def: "A multimeric complex that is able to catalyze the addition of methyl groups to histone proteins." [GOC:bf]
GO:0035098
name: ESC/E(Z) complex
def: "A multimeric protein complex that can methylate lysine-27 and lysine-9 residues of histone H3. In Drosophila the core subunits of the complex include ESC, E(Z), CAF1 (NURF-55) and SU(Z)12. In mammals the core subunits of the complex include EED, EZH2, SUZ12 and RBBP4." [GOC:bf, GOC:sp, PMID:12408863, PMID:12408864, PMID:20064375]
GO:0035101
name: FACT complex
def: "An abundant nuclear complex, which was originally identified in mammalian systems as a factor required for transcription elongation on chromatin templates. The FACT complex has been shown to destablilize the interaction between the H2A/H2B dimer and the H3/H4 tetramer of the nucleosome, thus reorganizing the structure of the nucleosome. In this way, the FACT complex may play a role in DNA replication and other processes that traverse the chromatin, as well as in transcription elongation. FACT is composed of two proteins that are evolutionarily conserved in all eukaryotes and homologous to mammalian Spt16 and SSRP1. In metazoans, the SSRP1 homolog contains an HMG domain; however in fungi and protists, it does not. For example, in S. cerevisiae the Pob3 protein is homologous to SSRP1, but lacks the HMG chromatin binding domain. Instead, the yFACT complex of Spt16p and Pob3p, binds to nucleosomes where multiple copies of the HMG-domain containing protein Nhp6p have already bound, but Nhp6p does not form a stable complex with the Spt16p/Pob3p heterodimer." [GOC:bf, GOC:expert_ks, GOC:expert_ras, GOC:expert_tf, GOC:krc, PMID:12934006, PMID:12934007, PMID:16678108]
GO:0035102
name: PRC1 complex
def: "A multiprotein complex that mediates monoubiquitination of lysine residues of histone H2A (lysine-118 in Drosophila or lysine-119 in mammals). The complex is required for stable long-term maintenance of transcriptionally repressed states and is involved in chromatin remodeling." [GOC:bf, PMID:10412979]
GO:0035145
name: exon-exon junction complex
def: "A multi-subunit complex deposited by the spliceosome upstream of messenger RNA exon-exon junctions. The exon-exon junction complex provides a binding platform for factors involved in mRNA export and nonsense-mediated mRNA decay." [PMID:11532962, PMID:11743026]
GO:0035182
name: female germline ring canal outer rim
def: "An electron opaque backbone of the insect ovarian ring canal that is a part of or adjacent to the plasma membrane. The outer rim is established as the cleavage furrow is arrested, and contains F-actin, anillin, glycoproteins and at least one a protein with a high content of phosphorylated tyrosine residues." [PMID:12435357, PMID:7925006]
GO:0035183
name: female germline ring canal inner rim
def: "A proteinaceous actin-rich layer of the insect ovarian ring canal that forms subcortically to the outer rim. The electron dense inner rim accumulates after the final mitotic division of each germline syncytia, and contains actin, a phosphotyrosine protein, and a number of cytoskeletal proteins." [PMID:10556087, PMID:7925006, PMID:9093858]
GO:0035189
name: Rb-E2F complex
def: "A multiprotein complex containing a heterodimeric E2F transcription factor and a Retinoblastoma (Rb) family member. This complex is capable of repressing transcription of E2F-regulated genes in order to regulate cell cycle progression." [PMID:14616073]
GO:0035230
name: cytoneme
def: "A long, thin, polarized cell projection that contains actin and can extend for distances many times the diameter of the cell. Cytonemes represent extensions of cell cytoplasm and typically have a diameter of approximately 0.2um." [PMID:10367889, PMID:10675901]
GO:0035253
name: ciliary rootlet
def: "A cytoskeleton-like structure, originating from the basal body at the proximal end of a cilium, and extending proximally toward the cell nucleus. Rootlets are typically 80-100 nm in diameter and contain cross striae distributed at regular intervals of approximately 55-70 nm." [GOC:cilia, PMID:12427867]
GO:0035267
name: NuA4 histone acetyltransferase complex
def: "A complex having histone acetylase activity on chromatin, as well as ATPase, DNA helicase and structural DNA binding activities. The complex is thought to be involved in double-strand DNA break repair. Subunits of the human complex include HTATIP/TIP60, TRRAP, RUVBL1, BUVBL2, beta-actin and BAF53/ACTL6A. In yeast, the complex has 13 subunits, including the catalytic subunit Esa1 (homologous to human Tip60)." [GOC:ecd, PMID:10966108, PMID:14966270]
GO:0035301
name: Hedgehog signaling complex
def: "A multiprotein complex that binds microtubules in a Hedgehog-dependent manner, and is required for signal transduction by members of the Hedgehog family of proteins. The core components of the complex are the serine/threonine protein kinase Fused, the kinesin motor protein Costal2 (Cos2), and a zinc finger transcription factor (Gli family members in humans, and Cubitus interruptus (Ci) in Drosophila)." [PMID:10825151, PMID:15057936]
GO:0035323
name: male germline ring canal
def: "An intercellular bridge that connects the germline cells of a male cyst." [PMID:9635420]
GO:0035324
name: female germline ring canal
def: "An intercellular bridge that connects the germline cells of a female cyst." [PMID:9635420]
GO:0035327
name: transcriptionally active chromatin
def: "The ordered and organized complex of DNA and protein that forms regions of the chromosome that are being actively transcribed." [GOC:sart, PMID:17965872]
GO:0035328
name: transcriptionally silent chromatin
def: "The ordered and organized complex of DNA and protein that forms regions of the chromosome that are not being actively transcribed." [GOC:sart, PMID:17965872]
GO:0035339
name: SPOTS complex
def: "A multiprotein complex at least composed of serine palmitoyltransferases and ORM proteins (known as ORMDL proteins in mammals and other higher vertebrates) that plays a key role in sphingolipid homeostasis." [PMID:20182505]
GO:0035354
name: Toll-like receptor 1-Toll-like receptor 2 protein complex
def: "A heterodimeric protein complex containing Toll-like receptor 1 (TLR1) and Toll-like receptor 2 (TLR2)." [GOC:add, GOC:signaling, PMID:17889651, PMID:21481769]
GO:0035355
name: Toll-like receptor 2-Toll-like receptor 6 protein complex
def: "A heterodimeric protein complex containing Toll-like receptor 2 (TLR2) and Toll-like receptor 6 (TLR6)." [GOC:add, GOC:signaling, PMID:19931471, PMID:21481769]
GO:0035361
name: Cul8-RING ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul8 subfamily and a RING domain protein form the catalytic core. In S. cerevisiae, Mms1p acts as the adaptor protein and substrate specificity is conferred by any of a number of different proteins." [GOC:krc, PMID:20139071]
GO:0035363
name: histone locus body
def: "A nuclear body associated with the histone gene locus that is thought to contain all of the factors necessary for histone mRNA transcription and pre-mRNA processing. In Drosophila, U7 snRNP is located in the histone locus body rather than the distinct Cajal body." [GOC:sart, PMID:16533947, PMID:18927579, PMID:19620235]
GO:0035369
name: pre-B cell receptor complex
def: "An immunoglobulin-like complex that is present in at least the plasma membrane of pre-B cells, and that is composed of two identical immunoglobulin heavy chains and two surrogate light chains, each composed of the lambda-5 and VpreB proteins, and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:16464608, PMID:17306522]
GO:0035370
name: UBC13-UEV1A complex
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains and is involved in NF-kappaB activation. In humans at least, the complex comprises the ubiquitin-conjugating enzyme UBC13 and ubiquitin-conjugating enzyme variant 1A (UEV1A)." [GOC:amm, PMID:16129784]
GO:0035371
name: microtubule plus-end
def: "The growing (plus) end of a microtubule. In vitro, microtubules polymerize more quickly at the plus end than at the minus end. In vivo, microtubule growth occurs only at the plus end, and the plus end switches between periods of growth and shortening, a behavior known as dynamic instability." [GOC:bf, GOC:lb, PMID:12700769, PMID:16643273]
GO:0035448
name: extrinsic component of thylakoid membrane
def: "The component of a thylakoid membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035449
name: extrinsic component of plastid thylakoid membrane
def: "The component of a plastid thylakoid membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035450
name: extrinsic component of lumenal side of plastid thylakoid membrane
def: "The component of a plastid thylakoid membrane consisting of gene products and protein complexes that are loosely bound to its lumenal surface, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035451
name: extrinsic component of stromal side of plastid thylakoid membrane
def: "The component of a plastid thylakoid membrane consisting of gene products and protein complexes that are loosely bound to its stromal surface, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035452
name: extrinsic component of plastid membrane
def: "The component of a plastid membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035453
name: extrinsic component of plastid inner membrane
def: "The component of a plastid inner membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035454
name: extrinsic component of stromal side of plastid inner membrane
def: "The component of a plastid inner membrane consisting of gene products and protein complexes that are loosely bound to its stromal surface, but not integrated into the hydrophobic region." [GOC:bf, GOC:dos]
GO:0035517
name: PR-DUB complex
def: "A multimeric protein complex that removes monoubiquitin from histone H2A. In Drosophila and mammals, the core of the complex is composed of Calypso/BAP1 and Asx/ASXL1, respectively." [PMID:20436459]
GO:0035525
name: NF-kappaB p50/p65 complex
def: "A heterodimer of NF-kappa B p50 and p65 subunits." [GO:add, PMID:20393192, PMID:9299584]
GO:0035550
name: urease complex
def: "A multiprotein nickel-containing complex that possesses urease activity (catalysis of the hydrolysis of urea to ammonia and carbon dioxide)." [InterPro:IPR008221, PMID:2651866]
GO:0035577
name: azurophil granule membrane
def: "The lipid bilayer surrounding an azurophil granule, a primary lysosomal granule found in neutrophil granulocytes that contains a wide range of hydrolytic enzymes and is released into the extracellular fluid." [GOC:bf, PMID:17152095]
GO:0035578
name: azurophil granule lumen
def: "The volume enclosed by the membrane of an azurophil granule, a primary lysosomal granule found in neutrophil granulocytes that contains a wide range of hydrolytic enzymes and is released into the extracellular fluid." [GOC:bf, PMID:17152095]
GO:0035579
name: specific granule membrane
def: "The lipid bilayer surrounding a specific granule, a granule with a membranous, tubular internal structure, found primarily in mature neutrophil cells. Most are released into the extracellular fluid. Specific granules contain lactoferrin, lysozyme, vitamin B12 binding protein and elastase." [GOC:bf, PMID:7334549]
GO:0035580
name: specific granule lumen
def: "The volume enclosed by the membrane of a specific granule, a granule with a membranous, tubular internal structure, found primarily in mature neutrophil cells. Most are released into the extracellular fluid. Specific granules contain lactoferrin, lysozyme, vitamin B12 binding protein and elastase." [GOC:bf, PMID:7334549]
GO:0035618
name: root hair
def: "A long, thin projection from a root epidermal cell that contains F-actin and tubulin, and a cell wall." [http://www.jstor.org/stable/4354264, PO:0000256]
GO:0035619
name: root hair tip
def: "The tip portion of an outgrowth of a root epidermal cell." [PO:0000029]
GO:0035631
name: CD40 receptor complex
def: "A protein complex that contains at least CD40 (a cell surface receptor of the tumour necrosis factor receptor (TNFR) superfamily), and other signaling molecules." [GOC:BHF, PMID:20614026, PMID:9221764]
GO:0035632
name: mitochondrial prohibitin complex
def: "A complex composed of two proteins, prohibitin 1 and prohibitin 2 (PHB1/PHB-1 and PHB2/PHB-2) that is highly conserved amongst eukaryotes and associated with the inner mitochondrial membrane. The mitochondrial prohibitin complex is a macromolecular supercomplex composed of repeating heterodimeric subunits of PHB1 and PHB2. The mitochondrial prohibitin complex plays a role in a number of biological processes, including mitochondrial biogenesis and function, development, replicative senescence, and cell death." [GOC:kmv, PMID:12237468, PMID:21164222]
GO:0035649
name: Nrd1 complex
def: "A complex that functions in transcription termination of RNA polymerase II transcribed non-coding RNAs. This complex interacts with the carboxy-terminal domain (CTD) of PolII and the terminator sequences in the nascent RNA transcript. In yeast this complex consists of Nrd1p, Nab3p, and Sen1p." [GOC:jh, PMID:10655211, PMID:16427013, PMID:21084293]
GO:0035657
name: eRF1 methyltransferase complex
def: "A protein complex required for the methylation of a glutamine (Gln) residue in the protein release factor eRF1. In S. cerevisiae, this complex consists of at least Trm112p and Mtq2p." [GOC:rb, PMID:17008308, PMID:20400505]
GO:0035658
name: Mon1-Ccz1 complex
def: "A protein complex that functions as a guanine nucleotide exchange factor (GEF) and converts Rab-GDP to Rab-GTP. In S. cerevisiae, this complex consists of at least Mon1 and Ccz1, and serves as a GEF for the Rab Ypt7p." [GOC:rb, PMID:20797862]
GO:0035686
name: sperm fibrous sheath
def: "A cytoskeletal structure surrounding the axoneme and outer dense fibers of the sperm flagellum. Consists of two longitudinal columns connected by closely arrayed semicircular ribs that assemble from distal to proximal throughout spermiogenesis. The fibrous sheath probably influences the degree of flexibility, plane of flagellar motion, and the shape of the flagellar beat." [GOC:BHF, PMID:20731842, PMID:3282552]
GO:0035692
name: macrophage migration inhibitory factor receptor complex
def: "A protein complex that binds macrophage migration inhibitory factor. Comprises CD74 and CD44 cell surface proteins." [GOC:BHF, PMID:12782713, PMID:17045821]
GO:0035693
name: NOS2-CD74 complex
def: "A protein complex comprising nitric oxide synthase 2 and CD74. This stable complex formation is thought to prevent CD74 degradation by caspases." [GOC:BHF, PMID:18003616]
GO:0035748
name: myelin sheath abaxonal region
def: "The region of the myelin sheath furthest from the axon." [GOC:BHF, PMID:20237282]
GO:0035749
name: myelin sheath adaxonal region
def: "The region of the myelin sheath nearest to the axon." [GOC:BHF, PMID:20237282]
GO:0035770
name: ribonucleoprotein granule
def: "A non-membranous macromolecular complex containing proteins and translationally silenced mRNAs. RNA granules contain proteins that control the localization, stability, and translation of their RNA cargo. Different types of RNA granules (RGs) exist, depending on the cell type and cellular conditions." [GOC:go_curators, GOC:sp, PMID:16520386, PMID:20368989, PMID:21436445]
GO:0035792
name: other organism postsynaptic membrane
def: "A postsynaptic membrane that is part of another organism, i.e. a secondary organism with which the first organism is interacting. A postsynaptic membrane is a specialized area of membrane facing the presynaptic membrane on the tip of the nerve ending and separated from it by a minute cleft (the synaptic cleft). Neurotransmitters transmit the signal across the synaptic cleft to the postsynaptic membrane." [GOC:ecd]
GO:0035796
name: ATP-binding cassette (ABC) transporter complex, transmembrane substrate-binding subunit-containing
def: "A complex for the transport of metabolites into the cell, consisting of 4 subunits: a transmembrane substrate-binding protein (known as the S component), and an energy-coupling module that comprises two ATP-binding proteins (known as the A and A' components) and a transmembrane protein (known as the T component). Transport of the substrate across the membrane is driven by the hydrolysis of ATP." [PMID:18931129, PMID:20972419, PMID:21135102]
GO:0035805
name: egg coat
def: "A specialized extracellular matrix that surrounds the ovum of animals. The egg coat provides structural support and can play an essential role in oogenesis, fertilization and early development." [PMID:16944418, PMID:17163408]
GO:0035808
name: meiotic recombination initiation complex
def: "A protein complex that initiates the formation of double-strand breaks (DSBs) required for meiotic recombination. Consists of a protein that catalyses formation of the double-strand breaks (Spo11 in S. cerevisiae and Rec12 in S. pombe), and a number of accessory proteins." [GOC:vw, PMID:12897161, PMID:20364342, PMID:21429938]
GO:0035838
name: growing cell tip
def: "The region at either end of the longest axis of a cylindrical or elongated cell, where polarized growth occurs." [GOC:mah]
GO:0035839
name: non-growing cell tip
def: "A cell tip at which no growth takes place. For example, in fission yeast the cell end newly formed by cell division does not grow immediately upon its formation, and lacks actin cytoskeletal structures." [GOC:expert_jd, GOC:mah]
GO:0035840
name: old growing cell tip
def: "A cell tip which has existed for at least one complete cell cycle, and at which polarized growth occurs. For example, in fission yeast the cell end that existed prior to cell division grows immediately after division, and contains a distinctive complement of proteins including actin cytoskeletal structures." [GOC:expert_jd, GOC:mah]
GO:0035841
name: new growing cell tip
def: "A cell tip that was newly formed at the last cell division, and that has started to grow after the cell has activated bipolar cell growth (i.e. in which new end take-off, NETO, has taken place). New end take-off is when monopolar cells initiate bipolar growth." [GOC:expert_jd, GOC:mah, PMID:19431238]
GO:0035842
name: old cell tip after activation of bipolar cell growth
def: "A cell tip which has existed for at least one complete cell cycle, and at which polarized growth occurs, which is part of a cell that has activated bipolar cell growth (i.e. in which new end take-off, NETO, has taken place). For example, in fission yeast the cell end that existed prior to cell division grows immediately after division, and contains a distinctive complement of proteins including actin cytoskeletal structures." [GOC:expert_jd, GOC:mah]
GO:0035843
name: endonuclear canal
def: "A membrane-bound structure present in the nucleus of a spermatozoon. There is variation in the number of endonuclear canals between sperm of different organisms, and some species lack these structures altogether. The endonuclear canal may provide a supporting role for the sperm nucleus, and originates during spermiogenesis from an invagination of the nuclear envelope." [GOC:bf, PMID:18359585]
GO:0035859
name: Seh1-associated complex
def: "A protein complex that associates dynamically with the vacuolar membrane, and is proposed to have a role in membrane-associated trafficking or regulatory processes. In S. cerevisiae the complex contains Seh1p, Sec13p, Npr2p, Npr3p, Iml1p, Mtc5p, Rtc1p, and Sea4p." [GOC:jh, PMID:21454883, PMID:23974112]
GO:0035861
name: site of double-strand break
def: "A region of a chromosome at which a DNA double-strand break has occurred. DNA damage signaling and repair proteins accumulate at the lesion to respond to the damage and repair the DNA to form a continuous DNA helix." [GOC:bf, GOC:mah, GOC:vw, PMID:20096808, PMID:21035408]
GO:0035866
name: alphav-beta3 integrin-PKCalpha complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to protein kinase C alpha." [GOC:BHF, GOC:ebc, PMID:16014375]
GO:0035867
name: alphav-beta3 integrin-IGF-1-IGF1R complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to insulin-like growth factor-1 (IGF-1) and type I insulin-like growth factor receptor (IGF1R). IGF1R is a heterotetramer that consists of two alpha-subunits and two beta-subunits." [GOC:BHF, GOC:ebc, PMID:19578119]
GO:0035868
name: alphav-beta3 integrin-HMGB1 complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to high mobility group box 1 protein." [GOC:BHF, GOC:ebc, PMID:20826760]
GO:0035869
name: ciliary transition zone
def: "A region of the cilium between the basal body and proximal segment that is characterized by Y-shaped assemblages that connect axonemal microtubules to the ciliary membrane. The ciliary transition zone appears to function as a gate that controls ciliary membrane composition." [GOC:cilia, GOC:kmv, PMID:21422230]
GO:0035974
name: meiotic spindle pole body
def: "The microtubule organizing center on a spindle that forms as part of meiosis; functionally homologous to the animal cell centrosome." [GOC:vw, PMID:21775631]
GO:0035976
name: AP1 complex
def: "A heterodimeric transcription factor complex composed of proteins from the c-Fos, c-Jun, activating transcription factor (ATF) or JDP families. The subunits contain a basic leucine zipper (bZIP) domain that is essential for dimerization and DNA binding. Jun-Fos heterodimers bind preferentially to a heptamer consensus sequence (TPA responsive element (TRE)), whereas Jun-ATF dimers bind the cyclic AMP responsive element (CRE) to regulate transcription of target genes." [GOC:bf, GOC:BHF, GOC:rl, PMID:20060892, PMID:9069263, Wikipedia:AP-1_transcription_factor]
GO:0035985
name: senescence-associated heterochromatin focus
def: "A transcriptionally-silent heterochromatin structure present in senescent cells. Contains the condensed chromatin of one chromosome and is enriched for histone modifications. Thought to repress expression of proliferation-promoting genes." [GOC:yaf, PMID:15621527, PMID:21248468]
GO:0035996
name: rhabdomere microvillus
def: "Thin cylindrical membrane-covered projection on the surface of a rhabdomere." [GOC:bf, GOC:sart, PMID:14744998]
GO:0035997
name: rhabdomere microvillus membrane
def: "The portion of the plasma membrane surrounding a microvillus of a rhabdomere." [GOC:bf, GOC:sart, PMID:14744998]
GO:0036000
name: mucocyst
def: "A small subcellular vesicle, surrounded by a membrane, in the pellicle of ciliate protozoans that discharges a mucus-like secretion." [GOC:mag, PMID:10723937, PMID:4629881]
GO:0036007
name: scintillon
def: "A body present in the cytoplasm of some dinoflagellates, which is the source of bioluminescence; emits light on acidification in the presence of oxygen." [GOC:mag, GOC:pr, PMID:4501583, PMID:5642469]
GO:0036012
name: cyanelle inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the cyanelle envelope; also faces the cyanelle stroma." [GOC:aa, PMID:18976493]
GO:0036013
name: cyanelle outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the cyanelle envelope." [GOC:aa]
GO:0036014
name: cyanelle intermembrane space
def: "The region between the inner and outer lipid bilayers of the cyanelle envelope; includes the peptidoglycan layer." [GOC:aa]
GO:0036019
name: endolysosome
def: "An transient hybrid organelle formed by fusion of a late endosome with a lysosome, and in which active degradation takes place." [GOC:pde, PMID:21878991]
GO:0036020
name: endolysosome membrane
def: "The lipid bilayer surrounding an endolysosome. An endolysosome is a transient hybrid organelle formed by fusion of a late endosome with a lysosome." [GOC:pde]
GO:0036021
name: endolysosome lumen
def: "The volume enclosed by the membrane of an endolysosome. An endolysosome is a transient hybrid organelle formed by fusion of a late endosome with a lysosome." [GOC:pde]
GO:0036024
name: protein C inhibitor-TMPRSS7 complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and transmembrane protease serine 7 (TMPRSS7); formation of the complex inhibits the serine protease activity of transmembrane protease serine 7." [GOC:ans, PMID:15853774]
GO:0036025
name: protein C inhibitor-TMPRSS11E complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and transmembrane protease serine 11E (TMPRSS11E); formation of the complex inhibits the serine protease activity of transmembrane protease serine 11E." [GOC:ans, PMID:15328353]
GO:0036026
name: protein C inhibitor-PLAT complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and tissue-type plasminogen activator (PLAT); formation of the complex inhibits the serine protease activity of tissue-type plasminogen activator." [GOC:ans, PMID:10340997]
GO:0036027
name: protein C inhibitor-PLAU complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and urokinase-type plasminogen activator (PLAU); formation of the complex inhibits the serine protease activity of urokinase-type plasminogen activator." [GOC:ans, PMID:10340997, PMID:3501295, PMID:8536714]
GO:0036028
name: protein C inhibitor-thrombin complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and thrombin (F2); formation of the complex inhibits the serine protease activity of thrombin." [GOC:ans, PMID:6323392]
GO:0036029
name: protein C inhibitor-KLK3 complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and prostate-specific antigen (KLK3); formation of the complex inhibits the serine protease activity of prostate-specific antigen." [GOC:ans, PMID:1725227]
GO:0036030
name: protein C inhibitor-plasma kallikrein complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and plasma kallikrein (KLK1B); formation of the complex inhibits the serine protease activity of plasma kallikrein." [GOC:ans, PMID:2844223, PMID:8536714]
GO:0036038
name: TCTN-B9D complex
def: "A protein complex that is located at the ciliary transition zone and consists of tectonic proteins, B9 domain-containing proteins and other proteins. Acts as a barrier that prevents diffusion of transmembrane proteins between the cilia and plasma membranes. In mouse, members of the complex include TCTN1, TCTN2, B9D1, MKS1, CC2D2A and other proteins." [GOC:sp, PMID:21725307, PMID:22179047]
GO:0036043
name: microspike
def: "OBSOLETE. A dynamic, actin-rich projection extending from the surface of a migrating animal cell." [PMID:11429692, PMID:12153987, PMID:19095735]
GO:0036053
name: glomerular endothelium fenestra
def: "A large plasma membrane-lined circular pore that perforates the flattened glomerular endothelium and, unlike those of other fenestrated capillaries, is not spanned by diaphragms; the density and size of glomerular fenestrae account, at least in part, for the high permeability of the glomerular capillary wall to water and small solutes." [GOC:cjm, MP:0011454, PMID:19129259]
GO:0036056
name: filtration diaphragm
def: "A specialized cell-cell junction found between the cells of the excretory system, which provides a barrier for filtration of blood or hemolymph." [GOC:mtg_kidney_jan10, GOC:sart, PMID:18971929]
GO:0036057
name: slit diaphragm
def: "A specialized cell-cell junction found between the interdigitating foot processes of the glomerular epithelium (the podocytes) in the vertebrate kidney, which is adapted for facilitating glomerular filtration." [GOC:mtg_kidney_jan10, GOC:rph, PMID:12386277, PMID:15994232, PMID:18971929, PMID:19478094]
GO:0036062
name: presynaptic periactive zone
def: "A region that surrounds the active zone of the presynaptic plasma membrane, and is specialized for the control of synaptic development." [GOC:sart, PMID:10976048, PMID:18439406]
GO:0036063
name: acroblast
def: "A cone-shaped structure in the head of a spermatozoon, which is formed by the coalescence of Golgi fragments following the completion of meiosis. The acroblast is situated adjacent to the acrosomal vesicle." [GOC:sart, PMID:19934220]
GO:0036064
name: ciliary basal body
def: "A membrane-tethered, short cylindrical array of microtubules and associated proteins found at the base of a eukaryotic cilium (also called flagellum) that is similar in structure to a centriole and derives from it. The cilium basal body is the site of assembly and remodelling of the cilium and serves as a nucleation site for axoneme growth. As well as anchoring the cilium, it is thought to provide a selective gateway regulating the entry of ciliary proteins and vesicles by intraflagellar transport." [GOC:cilia, GOC:clt, PMID:21750193]
GO:0036087
name: glutathione synthase complex
def: "A protein complex composed of two or more polypeptide subunits, and which possesses glutathione synthase activity (catalysis of the reaction: L-gamma-glutamyl-L-cysteine + ATP + glycine = ADP + glutathione + 2 H(+) + phosphate). In eukaryotes, the complex is homodimeric, in E. coli glutathione synthase exists as a tetramer, and in S. pombe the complex exists as a homodimer or a heterotetramer." [GOC:al, PMID:12734194, PMID:14990577, PMID:1958212]
GO:0036117
name: hyaluranon cable
def: "A cable structure, surrounding some cell types (e.g. proximal or bronchial tubular epithelial cells), and composed of hyaluranon (HA), a ubiquitous connective tissue glycosaminoglycan." [GOC:yaf, PMID:16900089]
GO:0036125
name: fatty acid beta-oxidation multienzyme complex
def: "A multienzyme complex possessing three kinds of enzymes that catalyze the chain reactions in the fatty acid beta-oxidation cycle, enoyl-CoA hydratase (ECH), 3-hydroxyacyl-CoA dehydrogenase (HACD), and acetyl-CoA C-acyltransferase (KACT)." [GOC:imk, PMID:12115060, PMID:16472743]
GO:0036126
name: sperm flagellum
def: "A microtubule-based flagellum (or cilium) that is part of a sperm, a mature male germ cell that develops from a spermatid." [GOC:cilia, GOC:sart, PMID:8441407]
GO:0036128
name: CatSper complex
def: "A sperm-specific voltage-gated calcium channel that controls the intracellular calcium ion concentration and, thereby, the swimming behavior of sperm. Consists of a heteromeric tetramer surrounding a calcium ion- selective pore. May also contain additional auxiliary subunits." [GOC:sp, PMID:17478420, PMID:21224844, PMID:22354039]
GO:0036156
name: inner dynein arm
def: "Inner arm structure present on the outer doublet microtubules of ciliary and flagellar axonemes. The structure of inner dynein arms is complex and may vary within the axoneme. Inner and outer dynein arms have different functions in the generation of microtubule-based motility." [GOC:BHF, GOC:vk, PMID:19347929, PMID:2557057, PMID:7962092]
GO:0036157
name: outer dynein arm
def: "Outer arm structure present on the outer doublet microtubules of ciliary and flagellar axonemes. Outer dynein arms contain 2-3 heavy chains, two or more intermediate chains and a cluster of 4-8 light chains. Inner and outer dynein arms have different functions in the generation of microtubule-based motility." [GOC:BHF, GOC:vk, PMID:2557057, PMID:6218174]
GO:0036186
name: early phagosome membrane
def: "The lipid bilayer surrounding an early phagosome." [GOC:phg]
GO:0036194
name: muscle cell projection
def: "A prolongation or process extending from a muscle cell. A muscle cell is a mature contractile cell, commonly known as a myocyte. This cell has as part of its cytoplasm myofibrils organized in various patterns." [CL:0000187, GOC:kmv, PMID:15930100, PMID:22464329]
GO:0036195
name: muscle cell projection membrane
def: "The portion of the plasma membrane surrounding a muscle cell projection." [CL:0000187, GOC:kmv, PMID:15930100, PMID:22464329]
GO:0036224
name: pairing center
def: "A special chromosome region located towards one end of a chromosome that contains dispersed copies of short, repetitive DNA sequences and functions as a cis-acting element essential for presynaptic homologous chromosome pairing and chromosome-nuclear envelope attachment." [GOC:kmv, PMID:18597662]
GO:0036240
name: septal periplasm
def: "The region between the plasma membrane and the cell wall, as found in organisms such as filamentous fungi." [GOC:di, PMID:21564341]
GO:0036266
name: Cdc48p-Npl4p-Vms1p AAA ATPase complex
def: "A multiprotein ATPase complex involved in the release of polyubiquitinated proteins, including those damaged by oxidative stress, from the outer mitochondria membrane into the cytoplasm where they are presented to the proteasome for proteolysis, a process also referred to as mitochondria-associated degradation (MAD). In budding yeast, this complex includes Cdc48p, Npl4p and Vms1p." [GOC:rn, PMID:21070972, PMID:21936843]
GO:0036284
name: tubulobulbar complex
def: "Actin-based structures involved in establishing close contact between Sertoli-Sertoli cells or Sertoli-spermatids in the seminiferous tubules of the testes." [GOC:sl, PMID:22510523]
GO:0036286
name: eisosome filament
def: "A filamentous cortical structure formed, in S. pombe, by the eisosome component Pil1." [GOC:vw, PMID:21900489, PMID:23722945]
GO:0036338
name: viral membrane
def: "The lipid bilayer of a virion, a complete fully infectious extracellular virus particle." [GOC:bm]
GO:0036362
name: ascus membrane
def: "A double layer of lipid molecules that surrounds an ascus, a capsule containing the sexual spores in some fungi." [GOC:mcc, GOC:vw, PMID:21900489]
GO:0036375
name: Kibra-Ex-Mer complex
def: "An apical protein complex that contains the proteins Kibra, Expanded and Merlin (Mer), or orthologs thereof. In humans, the complex contains KIBRA, FDM6 and NF2." [PMID:20159598]
GO:0036379
name: myofilament
def: "Any of the smallest contractile units of a myofibril (striated muscle fiber)." [Wikipedia:Myofilament]
GO:0036387
name: pre-replicative complex
def: "A protein-DNA complex that forms at the origin of replication during the initial step of DNA replication and allows the origin to become competent, or 'licensed', for replication." [GOC:bf, GOC:bhm, GOC:jh2, Wikipedia:Pre-replication_complex]
GO:0036389
name: bacterial pre-replicative complex
def: "A protein-DNA complex that forms at the bacterial oriC during the initial step of DNA replication and allows the origin to become competent, or 'licensed', for replication." [GOC:bf, GOC:bhm, GOC:jh2, PMID:19833870, PMID:21035377, Wikipedia:Pre-replication_complex]
GO:0036391
name: medial cortex septin ring
def: "A double ring-shaped structure that forms at the medial cortex at the onset of cytokinesis; composed of members of the conserved family of filament forming proteins called septins as well as septin-associated proteins." [GOC:vw, PMID:16009555]
GO:0036396
name: MIS complex
def: "An mRNA methyltransferase complex that catalyzes the post-transcriptional methylation of adenosine to form N6-methyladenosine (m6A). In budding yeast, the MIS complex consists of Mum2p, Ime4p and Slz1p. In vertebrates, the complex consists of METTL3, METTL14 and WTAP." [GOC:dgf, GOC:sp, PMID:22685417, PMID:24316715]
GO:0036398
name: TCR signalosome
def: "A multi-protein complex containing at least the T-cell receptor complex and the LAT (linker for activation of T cells) scaffold protein. Also contains a variety of signaling proteins including co-receptors, kinases, phosphatases and adaptors such as CD8. Connects events on the plasma membrane to distal signaling cascades to ultimately modulate T cell biology." [GOC:krc, PMID:17534068, PMID:20107804, PMID:22426112]
GO:0036405
name: anchored component of cell outer membrane
def: "The component of the cell outer membrane consisting of the gene products that are tethered to the membrane only by a covalently attached anchor, such as a lipid group embedded in the membrane. Gene products with peptide sequences that are embedded in the membrane are excluded from this grouping." [GOC:dos, GOC:md]
GO:0036406
name: anchored component of periplasmic side of cell outer membrane
def: "The component of the cell outer membrane consisting of gene products and protein complexes that are tethered to the periplasmic side of membrane by only a covalently attached anchor, embedded in the periplasmic side of the membrane only." [GOC:dos, GOC:md]
GO:0036407
name: mycolate outer membrane
def: "A mycolic acid-rich cell outer membrane containing a lipid bilayer and long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network. Found in mycobacteria and related genera (e.g. corynebacteria)." [GOC:bf, GOC:das, GOC:md, PMID:18316738, PMID:18567661]
GO:0036409
name: histone H3-K14 acetyltransferase complex
def: "A protein complex that can catalyze the acetylation of lysine at position 14 in histone H3." [GOC:vw, PMID:21289066]
GO:0036410
name: Mst2 histone acetyltransferase complex
def: "A protein complex that can catalyze the acetylation of lysine at position 14 in histone H3, and contains Mst2 as the catalytic subunit. In fission yeast, contains at least Mst2, Nto1, Ptf2, Ptf1 and Eaf6." [GOC:vw, PMID:21289066]
GO:0036411
name: H-NS-Cnu complex
def: "A trimeric protein complex containing a H-NS homodimer and a Cnu monomer. In bacteria, this complex negatively regulates transcription of a range of genes." [GOC:bhm, PMID:18189420, PMID:22358512]
GO:0036418
name: intrinsic component of mycolate outer membrane
def: "The component of the mycolate outer membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:md]
GO:0036419
name: integral component of mycolate outer membrane
def: "The component of the mycolate outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:md]
GO:0036420
name: extrinsic component of mycolate outer membrane
def: "The component of mycolate outer membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, GOC:md]
GO:0036421
name: extrinsic component of external side of mycolate outer membrane
def: "The component of mycolate membrane consisting of gene products and protein complexes that are loosely bound to its external surface, but not integrated into the hydrophobic region." [GOC:md]
GO:0036436
name: Isw1a complex
def: "An Isw1 complex that binds DNA and has nucleosome-stimulated ATPase activity. In S. cerevisiae, contains an Isw1p ATPase subunit in complex with Ioc3p." [GOC:jd, PMID:12482963]
GO:0036437
name: Isw1b complex
def: "An Isw1 complex that binds DNA and has nucleosome-stimulated ATPase activity. In S. cerevisiae, contains an Isw1p ATPase subunit in complex with Ioc2p and Ioc4p." [GOC:jd, PMID:12482963]
GO:0036449
name: microtubule minus-end
def: "The end of a microtubule that does not preferentially grow (polymerize)." [GOC:lb, PMID:23169647]
GO:0036452
name: ESCRT complex
def: "An endosomal sorting complex required for transport." [PMID:16689637, VZ:1536]
GO:0036454
name: growth factor complex
def: "A protein complex that has growth factor activity." [GOC:bm]
GO:0036457
name: keratohyalin granule
def: "A cytoplasmic, non-membrane bound granule of, at least, keratinocyte. Associated to keratin intermediate filaments and partially crosslinked to the cell envelope." [GOC:krc, PMID:15854042]
GO:0036464
name: cytoplasmic ribonucleoprotein granule
def: "A ribonucleoprotein granule located in the cytoplasm." [GOC:bf, GOC:PARL, PMID:15121898]
GO:0036477
name: somatodendritic compartment
def: "The region of a neuron that includes the cell body (cell soma) and the dendrite, but excludes the axon." [GOC:pad, GOC:PARL]
GO:0036488
name: CHOP-C/EBP complex
def: "A heterodimeric protein complex that is composed of the transcription factor CHOP (GADD153) and a member of the C/EBP family of transcription factors." [GOC:bf, GOC:PARL, PMID:1547942]
GO:0038037
name: G-protein coupled receptor dimeric complex
def: "A protein complex that contains two G-protein coupled receptors." [GOC:al, GOC:bf, PMID:10713101]
GO:0038038
name: G-protein coupled receptor homodimeric complex
def: "A protein complex that contains two G-protein coupled receptors (GPCRs) of the same subtype. Formation of a GPCR homodimer may be important for the transport of newly formed receptors to the cell surface, and the function of the receptor." [GOC:al, GOC:bf, PMID:10713101, PMID:16670762]
GO:0038039
name: G-protein coupled receptor heterodimeric complex
def: "A protein complex that contains two G-protein coupled receptors (GPCRs) of different subtypes. Formation of a GPCR heterodimer may alter the functional property of the GPCR." [GOC:al, GOC:bf, PMID:16109836, PMID:20150590]
GO:0038045
name: large latent transforming growth factor-beta complex
def: "A protein complex containing latency-associated proteins (LAPs), mature disulphide-linked dimeric TGF-beta, and latent TGF-beta binding proteins (LTBPs). TGF-beta is mostly secreted as part of the large latent complex, and must be subsequently released from the LLC in order to bind to cell surface receptors." [GOC:bf, PMID:2350783, PMID:8680476, PMID:9805445, Reactome:REACT_6888.1]
GO:0038059
name: IKKalpha-IKKalpha complex
def: "A homodimeric protein complex containing two IkappaB kinase (IKK) alpha subunits." [GOC:bf, PMID:18626576, PMID:21173796]
GO:0038104
name: nodal receptor complex
def: "A protein complex containing at least a type II activin receptor, a type I activin receptor, and a coreceptor (EGF-CFC protein) such as Cripto or Cryptic. Nodal receptor complexes are capable of binding a nodal protein and transducing the signal into the cell." [GOC:bf, GOC:signaling, PMID:11024047, PMID:15062104]
GO:0038139
name: ERBB4-EGFR complex
def: "A heterodimeric complex between the tyrosine kinase receptors ERBB4 (also called HER4) and epidermal growth factor receptor (EGFR/ERBB1)." [GOC:signaling, PMID:16460914, Reactome:REACT_116654.1]
GO:0038140
name: ERBB4-ERBB3 complex
def: "A heterodimeric complex between the tyrosine kinase receptors ERBB4 (also called HER4) and ERBB3 (also called HER3). ERBB3 has impaired kinase activity so relies on the kinase activity of its heterodimer partner for activation and signal transmission." [GOC:signaling, PMID:16460914, Reactome:REACT_117805.1]
GO:0038141
name: ERBB4-ERBB4 complex
def: "A homodimeric complex containing two monomers of the tyrosine kinase receptor ERBB4 (also called HER4)." [GOC:signaling, PMID:16460914, Reactome:REACT_117065.1]
GO:0038142
name: EGFR:ERBB2 complex
def: "A heterodimeric complex between the tyrosine kinase receptor ERBB2 and a ligand-activated epidermal growth factor receptor (EGFR/ERBB1). ERBB2, which does not bind any known ligand, is activated through formation of a heterodimer with another ligand-activated ERBB family member such as EGFR." [GOC:signaling, PMID:16460914, PMID:1973074, Reactome:REACT_115761.1, Reactome:REACT_116379.1, Reactome:REACT_117594.1]
GO:0038143
name: ERBB3:ERBB2 complex
def: "A heterodimeric complex between the tyrosine kinase receptor ERBB2 and a ligand-activated receptor ERBB3. ERBB2, which does not bind any known ligand, is activated through formation of a heterodimer with another ligand-activated ERBB family member such as ERBB3." [GOC:signaling, PMID:16460914, PMID:8665853, Reactome:REACT_115761.1, Reactome:REACT_116379.1, Reactome:REACT_117748.1]
GO:0038144
name: ERBB4:ERBB2 complex
def: "A heterodimeric complex between the tyrosine kinase receptor ERBB2 and a ligand-activated receptor ERBB4. ERBB2, which does not bind any known ligand, is activated through formation of a heterodimer with another ligand-activated ERBB family member such as ERBB4." [GOC:signaling, PMID:16460914, PMID:16978839, Reactome:REACT_115761.1, Reactome:REACT_116375.1, Reactome:REACT_116379.1]
GO:0038197
name: type I interferon receptor complex
def: "A heterodimeric protein complex that binds a type I interferon and transmits the signal across the membrane into the cell. Consists of an alpha subunit (IFNAR1) and a beta subunit (IFNAR2)." [GOC:cjm, GOC:signaling, PMID:17502368]
GO:0038201
name: TOR complex
def: "A protein complex that contains at least TOR (target of rapamycin) in complex with other signaling components. Mediates the phosphorylation and activation of downstream signaling components including PKB (AKT) or S6K." [Wikipedia:MTORC1, Wikipedia:MTORC2]
GO:0039615
name: T=1 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=1 symmetry. The T=1 capsid is composed of 12 pentameric capsomeres." [UniProtKB-KW:KW-1140, VZ:1057]
GO:0039616
name: T=2 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=2 symmetry. The T=2 capsid is composed of 12 pentameric dimers." [UniProtKB-KW:KW-1141, VZ:838]
GO:0039617
name: T=3 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=3 symmetry. The T=3 capsid is composed of 12 pentameric and 20 hexameric capsomeres." [UniProtKB-KW:KW-1142, VZ:806]
GO:0039618
name: T=pseudo3 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with pseudo T=3 symmetry. The T=pseudo3 capsid is composed of 12 pentameric and 20 hexameric capsomeres." [UniProtKB-KW:KW-1143, VZ:809]
GO:0039619
name: T=4 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=4 symmetry. The T=4 capsid is composed of 12 pentameric and 30 hexameric capsomeres." [UniProtKB-KW:KW-1144, VZ:808]
GO:0039620
name: T=7 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=7 symmetry. The T=7 capsid is composed of 12 pentameric and 60 hexameric capsomeres." [UniProtKB-KW:KW-1145, VZ:804]
GO:0039621
name: T=13 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=13 symmetry. The T=13 capsid is composed of 12 pentameric and 120 hexameric capsomeres." [UniProtKB-KW:KW-1146, VZ:260]
GO:0039622
name: T=16 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=16 symmetry. The T=16 capsid is composed of 12 pentameric and 150 hexameric capsomeres." [UniProtKB-KW:KW-1147, VZ:807]
GO:0039623
name: T=25 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=25 symmetry. The T=25 capsid is composed of 12 pentameric and 240 hexameric capsomeres." [UniProtKB-KW:KW-1148, VZ:810]
GO:0039624
name: viral outer capsid
def: "The outer layer of a double or triple concentric icosahedral capsid. Outer capsids are part of reoviridae and cystoviridae virions." [UniProtKB-KW:KW-1152]
GO:0039625
name: viral inner capsid
def: "The inner layer of a double or triple concentric icosahedral capsid. Inner capsids are part of reoviridae and cystoviridae virions." [UniProtKB-KW:KW-1153]
GO:0039626
name: viral intermediate capsid
def: "The intermediate layer of a triple concentric icosahedral capsid. Intermediate capsids are part of reoviridae virions." [UniProtKB-KW:KW-1154]
GO:0039627
name: T=147 icosahedral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=147 symmetry. T=147 icosahedral capsid is composed of 12 pentameric and 1460 hexameric capsomeres for a total of 8820 capsid proteins." [GOC:plm, UniProtKB-KW:KW-0167]
GO:0039628
name: T=169 icosahedral viral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=169 symmetry. T=169 icosahedral capsid is composed of 12 pentameric and 1680 hexameric capsomeres for a total of 10140 capsid proteins." [GOC:plm, UniProtKB-KW:KW-1150]
GO:0039629
name: T=219 icosahedral capsid
def: "The protein coat that surrounds the infective nucleic acid in some virus particles where the subunits (capsomeres) are arranged to form an icosahedron with T=219 symmetry. T=219 icosahedral capsid is composed of 12 pentameric and 2180 hexameric capsomeres for a total of 13140 capsid proteins." [GOC:plm, UniProtKB-KW:KW-1151]
GO:0039641
name: viral inner membrane
def: "The lipid bilayer of a virion contained inside the protein capsid." [GOC:bm, PMID:15331712]
GO:0039642
name: virion nucleoid
def: "The region of a virion in which the nucleic acid is confined." [GOC:bm, PMID:14291596]
GO:0039643
name: host cell viral nucleoid
def: "The region of a host cell that contains the viral genome." [GOC:bf, GOC:bm, GOC:jl]
GO:0039658
name: TBK1-IKKE-DDX3 complex
def: "A protein complex containing TBK1 (TANK-binding kinase 1), IKBKE (I-Kappa-B kinase epsilon/IKKE/IKK-epsilon) and the DEAD box family RNA helicase DDX3." [PMID:18636090, VZ:719]
GO:0039661
name: host organelle outer membrane
def: "The outer, i.e. cytoplasm-facing in a cellular organelle, lipid bilayer of an organelle envelope, occurring in a host cell." [GOC:bf, GOC:ch]
GO:0039662
name: host cell outer membrane
def: "A lipid bilayer that forms the outermost layer of the cell envelope, occurring in a host cell." [GOC:bf, GOC:ch]
GO:0039670
name: viral capsid, turret
def: "A turret-like appendage formed at the vertices of an icosahedral capsid." [GOC:jh2, PMID:20592081]
GO:0039679
name: viral occlusion body
def: "A crystalline protein matrix surrounding the nucleocapsids of some insect viruses after their release in the environment. Produced in the host cell, the occlusion body protects the infectious virion after death of the host." [UniProtKB-KW:KW-0842, VZ:1949]
GO:0039713
name: viral factory
def: "An intracellular compartment in a host cell which increases the efficiency of viral replication and/or assembly, and shields the virus from host defenses. Viral factories can be either cytoplasmic or nuclear and often arise from extensive rearrangement of host cell cytoskeletal and/or cell membrane compartments." [PMID:22440839, VZ:1951]
GO:0039714
name: cytoplasmic viral factory
def: "A viral factory located in the cytoplasm of a host cell." [VZ:1951]
GO:0039715
name: nuclear viral factory
def: "A viral factory located in the nucleus of a host cell." [VZ:1951]
GO:0039716
name: viroplasm viral factory
def: "A cytoplasmic viral factory that is electron dense due to high levels of viral RNA. Produced by nucleo-cytoplasmic large DNA viruses (NCLDV) like Poxviridae, Asfarviridae and Iridoviridae, and dsRNA viruses like Reoviridae." [VZ:1951, Wikipedia:Viroplasm]
GO:0039717
name: spherule viral factory
def: "A cytoplasmic viral factory which is a 50-400nm diameter membrane invagination. Spherules can appear on several enveloped cellular components depending on the virus." [VZ:1951]
GO:0039718
name: double membrane vesicle viral factory
def: "A cytoplasmic viral factory that consists of a double-membrane bound vesicle. Has a diameter of 200-300nm and is derived from the endoplasmic reticulum or Golgi apparatus. Produced by Picornaviridae, Nidovirales, Arteriviridae and Coronaviridae." [PMID:22440839, VZ:1951]
GO:0039719
name: tube viral factory
def: "A cytoplasmic viral factory derived from the Golgi in which Bunyaviridae replication takes place. Tubes are membranous structures close to the assembly and budding sites, and their function may be to connect viral replication and morphogenesis inside viral factories." [VZ:1951]
GO:0039720
name: virogenic stroma
def: "A nuclear viral factory formed by Baculoviruses. A vesicular structure in which virions are assembled." [PMID:13358757, PMID:1433508, VZ:1951]
GO:0039721
name: peristromal region viral factory
def: "A nuclear viral factory formed at the periphery of the host cell nucleus by Baculoviruses." [PMID:18434402, VZ:1951]
GO:0042022
name: interleukin-12 receptor complex
def: "A protein complex that binds interleukin-12; comprises a beta1 and a beta2 subunit." [GOC:ebc, GOC:mah, PMID:10971505]
GO:0042025
name: host cell nucleus
def: "A membrane-bounded organelle as it is found in the host cell in which chromosomes are housed and replicated. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:pamgo_curators]
GO:0042101
name: T cell receptor complex
def: "A protein complex that contains a disulfide-linked heterodimer of T cell receptor (TCR) chains, which are members of the immunoglobulin superfamily, and mediates antigen recognition, ultimately resulting in T cell activation. The TCR heterodimer is associated with the CD3 complex, which consists of the nonpolymorphic polypeptides gamma, delta, epsilon, zeta, and, in some cases, eta (an RNA splice variant of zeta) or Fc epsilon chains." [GOC:mah, ISBN:0781735149]
GO:0042105
name: alpha-beta T cell receptor complex
def: "A T cell receptor complex in which the TCR heterodimer comprises alpha and beta chains, associated with the CD3 complex; recognizes a complex consisting of an antigen-derived peptide bound to a class I or class II MHC protein." [GOC:mah, ISBN:0781735149]
GO:0042106
name: gamma-delta T cell receptor complex
def: "A T cell receptor complex in which the TCR heterodimer comprises gamma and delta chains, associated with the CD3 complex; recognizes antigen directly, without a requirement for processing and presentation by an MHC protein." [GOC:mah, ISBN:0781735149]
GO:0042151
name: nematocyst
def: "An organelle found in cnidoblast (nematoblast) cells. When matured, these stinging organelles store toxins and can deliver them when the cnidocil (a short extension of the cnidocyst) is stimulated by a prey or another stimulus." [DOI:10.1139/z02-135, GOC:jl]
GO:0042170
name: plastid membrane
def: "Either of the lipid bilayers that surround a plastid and form the plastid envelope." [GOC:mah]
GO:0042175
name: nuclear outer membrane-endoplasmic reticulum membrane network
def: "The continuous network of membranes encompassing the nuclear outer membrane and the endoplasmic reticulum membrane." [GOC:bf, GOC:jl, GOC:mah, GOC:mcc, GOC:pr, GOC:vw]
GO:0042272
name: nuclear RNA export factor complex
def: "A protein complex that contains two proteins (know in several organisms, including Drosophila, as NXF1 and NXF2) and is required for the export of the majority of mRNAs from the nucleus to the cytoplasm; localized in the nucleoplasm and at both the nucleoplasmic and cytoplasmic faces of the nuclear pore complex; shuttles between the nucleus and the cytoplasm." [PMID:11780633]
GO:0042382
name: paraspeckles
def: "Discrete subnuclear bodies in the interchromatin nucleoplasmic space, often located adjacent to nuclear specks. 10-20 paraspeckles are typically found in human cell nuclei." [GOC:jl, PMID:11790299]
GO:0042383
name: sarcolemma
def: "The outer membrane of a muscle cell, consisting of the plasma membrane, a covering basement membrane (about 100 nm thick and sometimes common to more than one fiber), and the associated loose network of collagen fibers." [ISBN:0198506732]
GO:0042385
name: myosin III complex
def: "A myosin complex containing a class III myosin heavy chain and associated light chains; myosin III is monomeric myosin that serves as a link between the cytoskeleton and the signaling complex involved in phototransduction, and differs from all other myosins in having an N-terminal kinase domain." [GOC:jl, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0042405
name: nuclear inclusion body
def: "An intranuclear focus at which aggregated proteins have been sequestered." [GOC:jl]
GO:0042406
name: extrinsic component of endoplasmic reticulum membrane
def: "The component of the endoplasmic reticulum membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:curators, GOC:dos]
GO:0042470
name: melanosome
def: "A tissue-specific, membrane-bounded cytoplasmic organelle within which melanin pigments are synthesized and stored. Melanosomes are synthesized in melanocyte cells." [GOC:jl, PMID:11584301]
GO:0042555
name: MCM complex
def: "A hexameric protein complex required for the initiation and regulation of DNA replication." [GOC:jl, PMID:11282021]
GO:0042563
name: importin alpha-subunit nuclear export complex
def: "A protein complex which usually consists of three components, e.g. in Xenopus, the importin alpha-subunit/CAS/Ran, and which functions to shuttle the importin alpha-subunit out of the nucleus through the nuclear pore." [GOC:jl, PMID:9323123, PMID:9323134]
GO:0042564
name: NLS-dependent protein nuclear import complex
def: "A protein complex which usually consists of three components, e.g. in Xenopus, the importin alpha and beta-subunits and any protein which has a nuclear localization sequence (NLS). The complex acts to import proteins with an NLS into the nucleus through a nuclear pore." [GOC:jl, PMID:9323123, PMID:9323134]
GO:0042565
name: RNA nuclear export complex
def: "A complex which usually consists of three components, e.g. in Xenopus and yeast, the export receptor CRM1 (also known as exportin 1), the Ran protein and any RNA with a nuclear export sequence (NES). The complex acts to export RNA molecules with a NES from the nucleus through a nuclear pore." [GOC:jl, PMID:9323123]
GO:0042566
name: hydrogenosome
def: "A spherical, membrane-bounded organelle found in some anaerobic protozoa, which participates in ATP and molecular hydrogen formation." [GOC:jl, PMID:11197234, PMID:11293569]
GO:0042567
name: insulin-like growth factor ternary complex
def: "A complex of three proteins, which in animals is approximately 150kDa and consists of the insulin-like growth factor (IGF), the insulin-like growth factor binding protein-3 (IGFBP-3), or -5 (IGFBP-5) and an acid-labile subunit (ALS). The complex plays a role in growth and development." [GOC:jl, PMID:12239079]
GO:0042568
name: insulin-like growth factor binary complex
def: "A complex of two proteins, which in animals is 50kDa and consists of the insulin-like growth factor (IGF) and one of the insulin-like growth factor binding protein-1 (IGFBP-1), -2 (IGFBP-2), -4 (IGFBP-4) and -6 (IGFBP-6). The complex plays a role in growth and development." [GOC:jl, PMID:12239079]
GO:0042571
name: immunoglobulin complex, circulating
def: "An immunoglobulin complex that is secreted into extracellular space and found in mucosal areas or other tissues or circulating in the blood or lymph. In its canonical form, a circulating immunoglobulin complex is composed of two identical heavy chains and two identical light chains, held together by disulfide bonds. Some forms of are polymers of the basic structure and contain additional components such as J-chain and the secretory component." [GOC:add, ISBN:0781735149]
GO:0042575
name: DNA polymerase complex
def: "A protein complex that possesses DNA polymerase activity and is involved in template directed synthesis of DNA." [GOC:jl, PMID:12045093]
GO:0042579
name: microbody
def: "Cytoplasmic organelles, spherical or oval in shape, that are bounded by a single membrane and contain oxidative enzymes, especially those utilizing hydrogen peroxide (H2O2)." [ISBN:0198506732]
GO:0042580
name: mannosome
def: "A specialised tubular organelle, assembled in hexagonal bundles within an external membrane. Mannosomes are specific to molluscs and are thought to be involved in a general stress reaction." [GOC:jl, PMID:11912051, PMID:9799531]
GO:0042581
name: specific granule
def: "Granule with a membranous, tubular internal structure, found primarily in mature neutrophil cells. Most are released into the extracellular fluid. Specific granules contain lactoferrin, lysozyme, vitamin B12 binding protein and elastase." [GOC:jl, ISBN:0721662544, PMID:7334549]
GO:0042582
name: azurophil granule
def: "Primary lysosomal granule found in neutrophil granulocytes. Contains a wide range of hydrolytic enzymes and is released into the extracellular fluid." [GOC:jl, PMID:17152095]
GO:0042583
name: chromaffin granule
def: "Specialized secretory vesicle found in the cells of adrenal glands and various other organs, which is concerned with the synthesis, storage, metabolism, and secretion of epinephrine and norepinephrine." [GOC:jl, PMID:19158310, PMID:1961743]
GO:0042584
name: chromaffin granule membrane
def: "The lipid bilayer surrounding a chromaffin granule, a specialized secretory vesicle found in the cells of adrenal glands and various other organs, which is concerned with the synthesis, storage, metabolism, and secretion of epinephrine and norepinephrine." [GOC:jl]
GO:0042585
name: germinal vesicle
def: "The enlarged, fluid filled nucleus of a primary oocyte, the development of which is suspended in prophase I of the first meiotic division between embryohood and sexual maturity." [GOC:jl, GOC:mtg_sensu, PMID:19019837]
GO:0042587
name: glycogen granule
def: "Cytoplasmic bead-like structures of animal cells, visible by electron microscope. Each granule is a functional unit with the biosynthesis and catabolism of glycogen being catalyzed by enzymes bound to the granule surface." [GOC:jl, http://148.216.10.83/cellbio/the_mi19.gif, PMID:12179957]
GO:0042588
name: zymogen granule
def: "A membrane-bounded, cytoplasmic secretory granule found in enzyme-secreting cells and visible by light microscopy. Contain zymogen, an inactive enzyme precursor, often of a digestive enzyme." [GOC:jl, ISBN:0198506732]
GO:0042589
name: zymogen granule membrane
def: "The lipid bilayer surrounding a zymogen granule." [GOC:jl]
GO:0042597
name: periplasmic space
def: "The region between the inner (cytoplasmic) and outer membrane (Gram-negative Bacteria) or cytoplasmic membrane and cell wall (Fungi and Gram-positive Bacteria)." [GOC:go_curators, GOC:md]
GO:0042598
name: vesicular fraction
def: "OBSOLETE: Any of the small, heterogeneous, artifactual, vesicular particles that are formed when some cells are homogenized." [GOC:jl]
GO:0042599
name: lamellar body
def: "A membrane-bounded organelle, specialized for the storage and secretion of various substances (surfactant phospholipids, glycoproteins and acid phosphates) which are arranged in the form of tightly packed, concentric, membrane sheets or lamellae. Has some similar properties to, but is distinct from, a lysosome." [GOC:cjm, GOC:jl, http://en.wikipedia.org/wiki/Lamellar_granule, PMID:12243725]
GO:0042600
name: chorion
def: "A protective, noncellular membrane that surrounds the eggs of various animals including insects and fish." [GOC:jl, ISBN:0721662544]
GO:0042601
name: endospore-forming forespore
def: "Portion of the cell formed during the process of bacterial sporulation that will ultimately become the core of the endospore. An endospore is a type of dormant cell that is resistant to adverse conditions." [GOC:jl, GOC:mtg_sensu, ISBN:0697286029]
GO:0042603
name: capsule
def: "A protective structure surrounding some fungi and bacteria, attached externally to the cell wall and composed primarily of polysaccharides. Capsules are highly organized structures that adhere strongly to cells and cannot be easily removed. Capsules play important roles in pathogenicity, preventing phagocytosis by other cells, adherance, and resistance to dessication." [GOC:mlg]
GO:0042611
name: MHC protein complex
def: "A transmembrane protein complex composed of an MHC alpha chain and, in most cases, either an MHC class II beta chain or an invariant beta2-microglobin chain, and with or without a bound peptide, lipid, or polysaccharide antigen." [GOC:add, GOC:jl, ISBN:0781735149, PMID:15928678, PMID:16153240]
GO:0042612
name: MHC class I protein complex
def: "A transmembrane protein complex composed of a MHC class I alpha chain and an invariant beta2-microglobin chain, and with or without a bound peptide antigen. Class I here refers to classical class I molecules." [GOC:add, GOC:jl, ISBN:0120781859, ISBN:0781735149]
GO:0042613
name: MHC class II protein complex
def: "A transmembrane protein complex composed of an MHC class II alpha and MHC class II beta chain, and with or without a bound peptide or polysaccharide antigen." [GOC:add, GOC:jl, ISBN:0120781859, PMID:15928678]
GO:0042622
name: photoreceptor outer segment membrane
def: "The membrane surrounding the outer segment of a vertebrate photoreceptor." [GOC:jl]
GO:0042627
name: chylomicron
def: "A large lipoprotein particle (diameter 75-1200 nm) composed of a central core of triglycerides and cholesterol surrounded by a protein-phospholipid coating. The proteins include one molecule of apolipoprotein B-48 and may include a variety of apolipoproteins, including APOAs, APOCs and APOE. Chylomicrons are found in blood or lymph and carry lipids from the intestines into other body tissues." [GOC:jl, GOC:rl, http://biotech.icmb.utexas.edu/search/dict-search.html, PMID:10580165]
GO:0042629
name: mast cell granule
def: "Coarse, bluish-black staining cytoplasmic granules, bounded by a plasma membrane and found in mast cells and basophils. Contents include histamine, heparin, chondroitin sulfates, chymase and tryptase." [GOC:jl, http://www.ijp-online.com/archives/1969/001/02/r0000-0000tc.htm, PMID:12360215]
GO:0042641
name: actomyosin
def: "Any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
GO:0042642
name: actomyosin, myosin complex part
def: "The myosin part of any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
GO:0042643
name: actomyosin, actin portion
def: "The actin part of any complex of actin, myosin, and accessory proteins." [GOC:go_curators]
GO:0042644
name: chloroplast nucleoid
def: "The region of a chloroplast to which the DNA is confined." [GOC:jl]
GO:0042645
name: mitochondrial nucleoid
def: "The region of a mitochondrion to which the DNA is confined." [GOC:jl]
GO:0042646
name: plastid nucleoid
def: "The region of a plastid to which the DNA is confined." [GOC:jl]
GO:0042647
name: proplastid nucleoid
def: "The region of a proplastid to which the DNA is confined." [GOC:jl]
GO:0042648
name: chloroplast chromosome
def: "A circular DNA molecule containing chloroplast encoded genes." [GOC:jl]
GO:0042649
name: prothylakoid
def: "Underdeveloped thylakoids found in etioplasts, lacking competent photosynthetic membranes. Rapidly develop into mature thylakoids in the presence of light." [GOC:jl, PMID:11532175]
GO:0042650
name: prothylakoid membrane
def: "The membrane of prothylakoids, underdeveloped thylakoids found in etioplasts, lacking competent photosynthetic membranes." [GOC:jl, PMID:11532175]
GO:0042651
name: thylakoid membrane
def: "The pigmented membrane of any thylakoid." [GOC:jl, GOC:pr]
GO:0042652
name: mitochondrial respiratory chain complex I, peripheral segment
def: "The peripheral segment of respiratory chain complex I located in the mitochondrion. Respiratory chain complex I is an enzyme of the respiratory chain, consisting of at least 34 polypeptide chains. The electrons of NADH enter the chain at this complex. The complete complex is L-shaped, with a horizontal arm lying in the membrane and a vertical arm that projects into the matrix." [GOC:jid, GOC:mtg_sensu, ISBN:0716749556]
GO:0042653
name: mitochondrial respiratory chain complex I, membrane segment
def: "The mitochondrial membrane segment of respiratory chain complex I. Respiratory chain complex I is an enzyme of the respiratory chain, consisting of at least 34 polypeptide chains. The electrons of NADH enter the chain at this complex. The complete complex is L-shaped, with a horizontal arm lying in the membrane and a vertical arm that projects into the matrix." [GOC:jid, GOC:mtg_sensu, ISBN:0716749556]
GO:0042709
name: succinate-CoA ligase complex
def: "A heterodimeric enzyme complex, usually composed of an alpha and beta chain. Functions in the TCA cycle, hydrolyzing succinyl-CoA into succinate and CoA, thereby forming ATP or GTP." [EC:6.2.1.4, EC:6.2.1.5, GOC:jl]
GO:0042716
name: plasma membrane-derived chromatophore
def: "A pigment-bearing structure that is derived from the cytoplasmic membrane, sometimes consisting of simple invaginations and sometimes a complete vesicle. This component is found in certain photosynthetic bacteria and cyanobacteria." [GOC:jl, ISBN:0395825172, PMID:11867431]
GO:0042717
name: plasma membrane-derived chromatophore membrane
def: "The lipid bilayer associated with a plasma membrane-derived chromatophore; surrounds chromatophores that form complete vesicles." [GOC:jl, GOC:mah, ISBN:0395825172, PMID:11867431]
GO:0042718
name: yolk granule
def: "Discrete structures that partition the water-insoluble portion of the yolk of oocytes and ova, which may or may not be membrane enclosed." [GOC:jl, http://148.216.10.83/cellbio/eggs.htm, PMID:18046696]
GO:0042719
name: mitochondrial intermembrane space protein transporter complex
def: "Soluble complex of the mitochondrial intermembrane space composed of various combinations of small Tim proteins; acts as a protein transporter to guide proteins to the Tim22 complex for insertion into the mitochondrial inner membrane." [PMID:12581629]
GO:0042720
name: mitochondrial inner membrane peptidase complex
def: "Protease complex of the mitochondrial inner membrane, consisting of at least two subunits, involved in processing of both nuclear- and mitochondrially-encoded proteins targeted to the intermembrane space." [PMID:10821182, PMID:12191769]
GO:0042721
name: mitochondrial inner membrane protein insertion complex
def: "A multi-subunit complex embedded in the mitochondrial inner membrane that mediates insertion of carrier proteins into the inner membrane." [PMID:12191765]
GO:0042729
name: DASH complex
def: "A large protein complex, containing around 8-10 subunits in yeast, including Duo1p, Dam1p, Dad1p and Ask1p. The complex forms part of the kinetochore, associates with microtubules when the kinetochore attaches to the spindle, and plays a role in spindle attachment, chromosome segregation and spindle stability." [GOC:jl, GOC:vw, http://www.wikigenes.org/e/gene/e/853090.html, PMID:11782438, PMID:11799062, PMID:15632076, PMID:15640796]
GO:0042734
name: presynaptic membrane
def: "A specialized area of membrane of the axon terminal that faces the plasma membrane of the neuron or muscle fiber with which the axon terminal establishes a synaptic junction; many synaptic junctions exhibit structural presynaptic characteristics, such as conical, electron-dense internal protrusions, that distinguish it from the remainder of the axon plasma membrane." [GOC:jl, ISBN:0815316194]
GO:0042735
name: protein body
def: "A membrane-bounded plant organelle found in the developing endosperm, contains storage proteins." [GOC:jl, PMID:7704047]
GO:0042757
name: giant axon
def: "Extremely large, unmyelinated axon found in invertebrates. Has high conduction speeds and is usually involved in panic or escape responses." [GOC:jl, PMID:9705477]
GO:0042763
name: intracellular immature spore
def: "A cell or part of the cell that constitutes an early developmental stage of a spore, a small reproductive body that is highly resistant to desiccation and heat and is capable of growing into a new organism, produced especially by certain bacteria, fungi, algae, and nonflowering plants." [GOC:jl, ISBN:0395825172]
GO:0042764
name: ascospore-type prospore
def: "An immature spore undergoing development. The spore usually consists of nucleic acid, prospore membrane(s) that encase the nucleic acid, and ultimately a cell wall that covers the membrane(s). This type of spore is observed in ascospore-forming fungi." [GOC:go_curators]
GO:0042765
name: GPI-anchor transamidase complex
def: "An enzyme complex which in humans and yeast consists of at least five proteins; for example, the complex contains GAA1, GPI8, PIG-S, PIG-U, and PIG-T in human, and Gaa1p, Gab1p, Gpi8p, Gpi16p, and Gpi17p in yeast. Catalyzes the posttranslational attachment of the carboxyl-terminus of a precursor protein to a GPI-anchor." [GOC:jl, GOC:rb, PMID:12802054]
GO:0042788
name: polysomal ribosome
def: "A ribosome bound to mRNA that forms part of a polysome." [GOC:jl]
GO:0042807
name: central vacuole
def: "A membrane-enclosed sac that takes up most of the volume of a mature plant cell. Functions include storage, separation of toxic byproducts, and cell growth determination." [ISBN:9780815341116, Wikipedia:Vacuole]
GO:0042824
name: MHC class I peptide loading complex
def: "A large, multisubunit complex which consists of the MHC class I-beta 2 microglobulin dimer, the transporter associated with antigen presentation (TAP), tapasin (an MHC-encoded membrane protein), the chaperone calreticulin and the thiol oxidoreductase ERp57. Functions in the assembly of peptides with newly synthesized MHC class I molecules." [GOC:jl, PMID:10631934]
GO:0042825
name: TAP complex
def: "A heterodimer composed of the subunits TAP1 and TAP2 (transporter associated with antigen presentation). Functions in the transport of antigenic peptides from the cytosol to the lumen of the endoplasmic reticulum." [GOC:jl, PMID:10618487, PMID:10631934]
GO:0042827
name: platelet dense granule
def: "Electron-dense granule occurring in blood platelets that stores and secretes adenosine nucleotides and serotonin. They contain a highly condensed core consisting of serotonin, histamine, calcium, magnesium, ATP, ADP, pyrophosphate and membrane lysosomal proteins." [GOC:jl, http://www.mercksource.com/, PMID:10403682, PMID:11487378]
GO:0042995
name: cell projection
def: "A prolongation or process extending from a cell, e.g. a flagellum or axon." [GOC:jl, http://www.cogsci.princeton.edu/~wn/]
GO:0043005
name: neuron projection
def: "A prolongation or process extending from a nerve cell, e.g. an axon or dendrite." [GOC:jl, http://www.cogsci.princeton.edu/~wn/]
GO:0043020
name: NADPH oxidase complex
def: "A enzyme complex of which the core is a heterodimer composed of a light (alpha) and heavy (beta) chain, and requires several other water-soluble proteins of cytosolic origin for activity. Functions in superoxide generation by the NADPH-dependent reduction of O2." [GOC:jl, PMID:11483596, PMID:12440767]
GO:0043025
name: neuronal cell body
def: "The portion of a neuron that includes the nucleus, but excludes cell projections such as axons and dendrites." [GOC:go_curators]
GO:0043033
name: isoamylase complex
def: "A protein complex whose composition varies amongst species; in rice it probably exists in a homo-tetramer to homo-hexamer form and in Gram-negative bacteria as a dimer. Functions in the hydrolysis of alpha-(1,6)-D-glucosidic branch linkages." [GOC:jl, PMID:10333591]
GO:0043034
name: costamere
def: "Regular periodic sub membranous arrays of vinculin in skeletal and cardiac muscle cells, these arrays link Z-discs to the sarcolemma and are associated with links to extracellular matrix." [GOC:jl, GOC:mtg_muscle, ISBN:0198506732, PMID:6405378]
GO:0043036
name: starch grain
def: "Plant storage body for amylose and amylopectin, 1-100um in diameter. Also contains small amounts of enzymes, amino acids, lipids and nucleic acids. The shape of the grain varies widely amongst species, but is often spherical or disk-shaped." [GOC:jl, PMID:11217978]
GO:0043073
name: germ cell nucleus
def: "The nucleus of a germ cell, a reproductive cell in multicellular organisms." [CL:0000586, GOC:go_curators]
GO:0043075
name: sperm cell nucleus (sensu Magnoliophyta)
def: "OBSOLETE. The nucleus of a plant pollen cell, the male gamete, and its descendents." [GOC:jl]
GO:0043076
name: megasporocyte nucleus
def: "The nucleus of a megasporocyte, a diploid cell that undergoes meiosis to produce four megaspores, and its descendents." [GOC:jl, ISBN:0618254153]
GO:0043078
name: polar nucleus
def: "Either of two nuclei located centrally in a flowering plant embryo sac that eventually fuse to form the endosperm nucleus." [ISBN:0618254153]
GO:0043079
name: antipodal cell nucleus
def: "The nucleus of an antipodal cell, one of three cells of the embryo sac in angiosperms, found at the chalazal end of the embryo away from the point of entry of the pollen tube, and its descendents." [CL:0000537, GOC:jl]
GO:0043082
name: megagametophyte egg cell nucleus
def: "The nucleus of a plant egg cell. This nucleus is found at the micropylar end of the embryo." [GOC:jl, GOC:mtg_sensu]
GO:0043083
name: synaptic cleft
def: "The narrow gap that separates the presynaptic and postsynaptic membranes, into which neurotransmitter is released." [GOC:jl, http://synapses.mcg.edu/anatomy/chemical/synapse.stm]
GO:0043159
name: acrosomal matrix
def: "A structural framework, or 'dense core' at the interior of an acrosome. May regulate the distribution of hydrolases within the acrosome and their release during the acrosome reaction." [GOC:jl, PMID:8949900, PMID:9139729]
GO:0043160
name: acrosomal lumen
def: "The volume enclosed within the acrosome membrane." [GOC:go_curators]
GO:0043186
name: P granule
def: "A small cytoplasmic, non-membranous RNA/protein complex aggregates in the primordial germ cells of many higher eukaryotes." [GOC:dph, GOC:kmv, PMID:11262230]
GO:0043187
name: cell septum surface
def: "The extracellular (rather than the intracellular) exterior of a dividing septum; this surface is usually composed of cell wall material, for example, lineal (1,3)-beta-D-glucan in S. pombe." [GOC:go_curators]
GO:0043188
name: cell septum edging
def: "The cell wall material that surrounds the septum in fungal cells." [GOC:vw]
GO:0043189
name: H4/H2A histone acetyltransferase complex
def: "A multisubunit complex that catalyzes the acetylation of histones H4 and H2A." [GOC:mah, GOC:rb]
GO:0043190
name: ATP-binding cassette (ABC) transporter complex
def: "A complex for the transport of metabolites into and out of the cell, typically comprised of four domains; two membrane-associated domains and two ATP-binding domains at the intracellular face of the membrane, that form a central pore through the plasma membrane. Each of the four core domains may be encoded as a separate polypeptide or the domains can be fused in any one of a number of ways into multidomain polypeptides. In Bacteria and Archaebacteria, ABC transporters also include substrate binding proteins to bind substrate external to the cytoplasm and deliver it to the transporter." [GOC:jl, GOC:mtg_sensu, PMID:11421269, PMID:15111107]
GO:0043194
name: axon initial segment
def: "Portion of the axon proximal to the neuronal cell body, at the level of the axon hillock. The action potentials that propagate along the axon are generated at the level of this initial segment." [GOC:nln, GOC:sl, PMID:1754851, PMID:21551097]
GO:0043195
name: terminal bouton
def: "Terminal inflated portion of the axon, containing the specialized apparatus necessary to release neurotransmitters. The axon terminus is considered to be the whole region of thickening and the terminal bouton is a specialized region of it." [GOC:dph, GOC:mc, GOC:nln, PMID:10218156, PMID:8409967]
GO:0043196
name: varicosity
def: "Non-terminal inflated portion of the axon, containing the specialized apparatus necessary to release neurotransmitters." [GOC:nln]
GO:0043197
name: dendritic spine
def: "Protrusion from a dendrite. Spines are specialised subcellular compartments involved in the synaptic transmission. They are linked to the dendritic shaft by a restriction. Because of their bulb shape, they function as a biochemical and an electrical compartment. Spine remodeling is though to be involved in synaptic plasticity." [GOC:nln]
GO:0043198
name: dendritic shaft
def: "Cylindric portion of the dendrite, directly stemming from the perikaryon, and carrying the dendritic spines." [GOC:nln]
GO:0043202
name: lysosomal lumen
def: "The volume enclosed within the lysosomal membrane." [GOC:jl, PMID:15213228]
GO:0043203
name: axon hillock
def: "Portion of the neuronal cell soma from which the axon originates." [GOC:nln]
GO:0043204
name: perikaryon
def: "The portion of the cell soma (cell body) that excludes the nucleus." [GOC:jl]
GO:0043205
name: fibril
def: "Extracellular matrix material consisting of polysaccharides and protein." [PMID:12704238]
GO:0043209
name: myelin sheath
def: "An electrically insulating fatty layer that surrounds the axons of many neurons. It is an outgrowth of glial cells: Schwann cells supply the myelin for peripheral neurons while oligodendrocytes supply it to those of the central nervous system." [GOC:cjm, GOC:jl, NIF_Subcellular:sao-593830697, Wikipedia:Myelin]
GO:0043218
name: compact myelin
def: "The portion of the myelin sheath in which layers of cell membrane are tightly juxtaposed, completely excluding cytoplasm. The juxtaposed cytoplasmic surfaces form the major dense line, while the juxtaposed extracellular surfaces form the interperiod line visible in electron micrographs." [GOC:dgh, NIF_Subcellular:sao-1123256993]
GO:0043219
name: lateral loop
def: "Non-compact myelin located adjacent to the nodes of Ranvier in a myelin segment. These non-compact regions include cytoplasm from the cell responsible for synthesizing the myelin. Lateral loops are found in the paranodal region adjacent to the nodes of Ranvier, while Schmidt-Lantermann clefts are analogous structures found within the compact myelin internode." [GOC:dgh]
GO:0043220
name: Schmidt-Lanterman incisure
def: "Regions within compact myelin in which the cytoplasmic faces of the enveloping myelin sheath are not tightly juxtaposed, and include cytoplasm from the cell responsible for making the myelin. Schmidt-Lanterman incisures occur in the compact myelin internode, while lateral loops are analogous structures found in the paranodal region adjacent to the nodes of Ranvier." [GOC:dgh]
GO:0043223
name: cytoplasmic SCF ubiquitin ligase complex
def: "A ubiquitin ligase complex, located in the cytoplasm, in which a cullin from the Cul1 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 adaptor and an F-box protein. SCF complexes are involved in targeting proteins for degradation by the proteasome. The best characterized complexes are those from yeast and mammals (with core subunits named Cdc53/Cul1, Rbx1/Hrt1/Roc1)." [PMID:15571813, PMID:15688063]
GO:0043224
name: nuclear SCF ubiquitin ligase complex
def: "A ubiquitin ligase complex, located in the nucleus, in which a cullin from the Cul1 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 adaptor and an F-box protein. SCF complexes are involved in targeting proteins for degradation by the proteasome. The best characterized complexes are those from yeast and mammals (with core subunits named Cdc53/Cul1, Rbx1/Hrt1/Roc1)." [PMID:15571813, PMID:15688063]
GO:0043226
name: organelle
def: "Organized structure of distinctive morphology and function. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton, and prokaryotic structures such as anammoxosomes and pirellulosomes. Excludes the plasma membrane." [GOC:go_curators]
GO:0043227
name: membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:go_curators]
GO:0043228
name: non-membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:go_curators]
GO:0043229
name: intracellular organelle
def: "Organized structure of distinctive morphology and function, occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton. Excludes the plasma membrane." [GOC:go_curators]
GO:0043230
name: extracellular organelle
def: "Organized structure of distinctive morphology and function, occurring outside the cell. Includes, for example, extracellular membrane vesicles (EMVs) and the cellulosomes of anaerobic bacteria and fungi." [GOC:jl, PMID:9914479]
GO:0043231
name: intracellular membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane." [GOC:go_curators]
GO:0043232
name: intracellular non-membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane and occurring within the cell. Includes ribosomes, the cytoskeleton and chromosomes." [GOC:go_curators]
GO:0043233
name: organelle lumen
def: "The internal volume enclosed by the membranes of a particular organelle; includes the volume enclosed by a single organelle membrane, e.g. endoplasmic reticulum lumen, or the volume enclosed by the innermost of the two lipid bilayers of an organelle envelope, e.g. nuclear lumen." [GOC:jl, GOC:mah]
GO:0043234
name: protein complex
def: "Any macromolecular complex composed (only) of two or more polypeptide subunits along with any covalently attached molecules (such as lipid anchors or oligosaccharide) or non-protein prosthetic groups (such as nucleotides or metal ions). Prosthetic group in this context refers to a tightly bound cofactor. The component polypeptide subunits may be identical." [GOC:go_curators]
GO:0043235
name: receptor complex
def: "Any protein complex that undergoes combination with a hormone, neurotransmitter, drug or intracellular messenger to initiate a change in cell function." [GOC:go_curators]
GO:0043240
name: Fanconi anaemia nuclear complex
def: "A protein complex composed of the Fanconi anaemia (FA) proteins including A, C, E, G and F (FANCA-F). Functions in the activation of the downstream protein FANCD2 by monoubiquitylation, and is essential for protection against chromosome breakage." [GOC:jl, PMID:12093742]
GO:0043245
name: extraorganismal space
def: "The environmental space outside of an organism; this may be a host organism in the case of parasitic and symbiotic organisms." [GOC:jl]
GO:0043246
name: megasome
def: "Large, cysteine proteinase rich lysosomes, often found in the amastigote (an intracytoplasmic, nonflagellated form of the parasite) stage of Leishmania species belonging to the mexicana complex." [PMID:11206117, PMID:1999020]
GO:0043253
name: chloroplast ribosome
def: "A ribosome contained within a chloroplast." [GOC:ecd]
GO:0043256
name: laminin complex
def: "A large, extracellular glycoprotein complex composed of three different polypeptide chains, alpha, beta and gamma. Provides an integral part of the structural scaffolding of basement membranes." [GOC:jl, http://www.sdbonline.org/fly/newgene/laminna1.htm, PMID:10842354]
GO:0043257
name: laminin-8 complex
def: "A laminin complex composed of alpha4, beta1 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0043258
name: laminin-9 complex
def: "A laminin complex composed of alpha4, beta2 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0043259
name: laminin-10 complex
def: "A laminin complex composed of alpha5, beta1 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0043260
name: laminin-11 complex
def: "A laminin complex composed of alpha5, beta2 and gamma1 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0043261
name: laminin-12 complex
def: "A laminin complex composed of alpha2, beta1 and gamma3 polypeptide chains." [GOC:jl, PMID:10842354]
GO:0043263
name: cellulosome
def: "An extracellular multi-enzyme complex containing up to 11 different enzymes aligned on a non-catalytic scaffolding glycoprotein. Functions to hydrolyze cellulose." [GOC:jl, PMID:11601609]
GO:0043264
name: extracellular non-membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, not bounded by a lipid bilayer membrane and occurring outside the cell." [GOC:jl]
GO:0043265
name: ectoplasm
def: "Granule free cytoplasm, lying immediately below the plasma membrane." [GOC:curators, PMID:12211103]
GO:0043291
name: RAVE complex
def: "A multisubunit complex that in Saccharomyces is composed of three subunits, Rav1p, Rav2p and Skp1p. Acts transiently to catalyze assembly of cytoplasmic V1, with membrane embedded V0 to form the V-ATPase holoenzyme." [PMID:11283612, PMID:11844802]
GO:0043292
name: contractile fiber
def: "Fibers, composed of actin, myosin, and associated proteins, found in cells of smooth or striated muscle." [GOC:go_curators, ISBN:0815316194]
GO:0043293
name: apoptosome
def: "A multisubunit protein complex involved in the signaling phase of the apoptotic process. In mammals it is typically composed of seven Apaf-1 subunits bound to cytochrome c and caspase-9. A similar complex to promote apoptosis is formed from homologous gene products in other eukaryotic organisms." [GOC:mtg_apoptosis, PMID:10428850, PMID:11406413, PMID:12176339, PMID:15189137]
GO:0043294
name: mitochondrial glutamate synthase complex (NADH)
def: "A protein complex, found in the mitochondria, that in yeast consists of a large and a small subunit. Possesses glutamate synthase (NADH) activity." [GOC:jl, PMID:7047525]
GO:0043296
name: apical junction complex
def: "A functional unit located near the cell apex at the points of contact between epithelial cells, which in vertebrates is composed of the tight junction, the zonula adherens, and desmosomes and in some invertebrates, such as Drosophila, is composed of the subapical complex (SAC), the zonula adherens and the septate junction. Functions in the regulation of cell polarity, tissue integrity and intercellular adhesion and permeability." [GOC:go_curators, GOC:kmv, PMID:12525486, PMID:15196556]
GO:0043332
name: mating projection tip
def: "The apex of the mating projection in unicellular fungi exposed to mating pheromone; site of polarized growth." [GOC:mcc]
GO:0043384
name: pre-T cell receptor complex
def: "A receptor complex found on immature T cells consisting of a T cell receptor beta chain and the pre-TCR-alpha chain, along with additional signaling components including CD3 family members and additional signaling proteins." [ISBN:0781735149, PMID:12220932]
GO:0043493
name: viral terminase complex
def: "A complex of a large and small subunit which catalyze the packaging of DNA into viral heads. Note that not all viral terminases have this structure, some exist as single polypeptides." [GOC:bf, GOC:bm, GOC:jl, GOC:mlg]
GO:0043494
name: CLRC ubiquitin ligase complex
def: "An cullin-dependent E3 ubiquitin ligase/histone H3-K9 methyltransferase complex essential for heterochromatin assembly by RNAi." [GOC:vw, PMID:16127433, PMID:20211136]
GO:0043505
name: CENP-A containing nucleosome
def: "A form of nucleosome located only at the centromere, in which the histone H3 is replaced by the variant form CENP-A (sometimes known as CenH3)." [GOC:go_curators, PMID:15175412, PMID:16183641]
GO:0043509
name: activin A complex
def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-A subunits (sometimes known as activin beta-A or activin/inhibin beta-A)." [GOC:go_curators]
GO:0043510
name: activin B complex
def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta-B subunits (sometimes known as activin beta-B or activin/inhibin beta-B)." [GOC:go_curators]
GO:0043511
name: inhibin complex
def: "Heterodimeric hormone composed of an inhibin alpha subunit complexed with either an inhibin beta-A subunit, to form inhibin A, or an inhibin beta-B subunit, to form inhibin B." [GOC:jl]
GO:0043512
name: inhibin A complex
def: "Heterodimeric hormone composed of an inhibin alpha subunit complexed with an inhibin beta-A subunit." [GOC:jl]
GO:0043513
name: inhibin B complex
def: "Heterodimeric hormone composed of an inhibin alpha subunit complexed with an inhibin beta-B subunit." [GOC:jl]
GO:0043514
name: interleukin-12 complex
def: "A protein complex that is composed of an interleukin-12 alpha (p35, product of the IL12A gene) and an interleukin-12 beta subunit (p40, product of the IL12B gene) and is secreted into the extracellular space." [GOC:add, GOC:ebc, GOC:mah, PMID:12948519, PMID:1381512]
GO:0043527
name: tRNA methyltransferase complex
def: "A multimeric protein complex involved in the methylation of specific nucleotides in tRNA." [GOC:jl]
GO:0043528
name: tRNA (m2G10) methyltransferase complex
def: "A protein complex required for the methylation of the guanosine nucleotide at position 10 (m2G10) in tRNA. In S. cerevisiae, this complex consists of at least two subunits, Trm11p and Trm112p." [PMID:15899842]
GO:0043529
name: GET complex
def: "A multisubunit complex involved in ER/Golgi trafficking (Golgi to ER Traffic). In yeast, includes Get1p, Get2p and Get3p proteins." [PMID:16269340]
GO:0043540
name: 6-phosphofructo-2-kinase/fructose-2,6-biphosphatase complex
def: "A homodimeric, bifunctional enzyme complex which catalyzes the synthesis and degradation of fructose 2,6-bisphosphate, and is required for both glycolysis and gluconeogenesis." [GOC:jl, GOC:so]
GO:0043541
name: UDP-N-acetylglucosamine transferase complex
def: "A multienzyme, heterooligomeric complex involved in dolichyl-linked oligosaccharide synthesis. In yeast the complex is composed of Alg7p, which catalyzes the first step (GlcNAc1-PP-Dol from dolichol-phosphate and UDP-GlcNAc), and Alg13p plus Alg14p, the catalytic and anchoring subunits respectively, which together catalyze the second step (GlcNAc2-PP-dolichol from GlcNAc1-PP-Dol and UDP-GlcNAc) of dolichyl-linked oligosaccharide synthesis." [GOC:rn, PMID:19129246]
GO:0043564
name: Ku70:Ku80 complex
def: "Heterodimeric protein complex composed of a 70 kDa and a 80 kDa subunit, binds DNA through a channel formed by the heterodimer. Functions in DNA double stranded break repair, chromosome maintenance, transcription regulation, V(D)J recombination, and activation of DNA-PK." [PMID:12518983]
GO:0043590
name: bacterial nucleoid
def: "The region of a bacterial cell to which the DNA is confined." [GOC:jl]
GO:0043591
name: endospore external encapsulating structure
def: "The structures that lie outside the inner membrane and surround the entire endospore; consists of a peptidoglycan-containing inner layer (the endospore cortex) surrounded by a multilayered proteinaceous coat. An exosporium may be present as an extreme outer layer." [GOC:go_curators, PMID:15035041]
GO:0043592
name: exosporium
def: "The outermost layer of a bacterial endospore, which is loosely attached and located outside of the endospore coat. It is generally composed of protein, carbohydrate, and perhaps lipid." [GOC:mlg]
GO:0043593
name: endospore coat
def: "The layer in a bacterial endospore that lies under the exosporium, and is impermeable to many toxic molecules. The coat may also contain enzymes that are involved in endospore germination." [GOC:mlg]
GO:0043594
name: outer endospore membrane
def: "The outer membrane around a bacterial endospore, located between the endospore cortex and endospore coat." [GOC:mlg]
GO:0043595
name: endospore cortex
def: "A layer surrounding a bacterial endospore found inside the outer endospore membrane, but outside the membrane surrounding the endospore core. It consists of peptidoglycan of a different chemical nature than that found in vegetative cell walls which results in less cross-linking of peptidoglycan." [GOC:mlg]
GO:0043596
name: nuclear replication fork
def: "The Y-shaped region of a nuclear replicating DNA molecule, resulting from the separation of the DNA strands and in which the synthesis of new strands takes place. Also includes associated protein complexes." [GOC:jl, GOC:mtg_sensu]
GO:0043597
name: cytoplasmic replication fork
def: "The Y-shaped region of a cytoplasmic replicating DNA molecule, resulting from the separation of the DNA strands and in which the synthesis of new strands takes place. Also includes associated protein complexes." [GOC:jl, GOC:mtg_sensu]
GO:0043598
name: cytoplasmic DNA replication factor C complex
def: "A cytoplasmic complex of two polypeptides that loads the DNA polymerase processivity factor proliferating cell nuclear antigen (PCNA) onto DNA, thereby permitting processive DNA synthesis catalyzed by DNA polymerase. Examples of this component are found in prokaryotic species." [GOC:mtg_sensu, PMID:14646196, PMID:16172520]
GO:0043599
name: nuclear DNA replication factor C complex
def: "A nuclear complex of five polypeptides that loads the DNA polymerase processivity factor proliferating cell nuclear antigen (PCNA) onto DNA, thereby permitting processive DNA synthesis catalyzed by DNA polymerase delta or epsilon. In Saccharomyces and several other species, the subunits are known as Rfc1p-Rfc5p, although subunit names do not necessarily correspond between different species." [GOC:mtg_sensu, PMID:14614842]
GO:0043600
name: cytoplasmic replisome
def: "A multi-component enzymatic machine at the cytoplasmic replication fork, which mediates DNA replication. Includes DNA primase, DNA polymerase, DNA helicase, and other proteins." [GOC:jl, GOC:mtg_sensu]
GO:0043601
name: nuclear replisome
def: "A multi-component enzymatic machine at the nuclear replication fork, which mediates DNA replication. Includes DNA primase, one or more DNA polymerases, DNA helicases, and other proteins." [GOC:jl, GOC:mtg_sensu]
GO:0043614
name: multi-eIF complex
def: "A multifactor complex composed of multiple translation initiation factors and the initiatior tRNAiMet, which is ready to bind to the small (40S) ribosome to form the 43S preinitiation complex. In S. cerevisiae, this complex is composed of eIF1, eIF2, eIF3, and eIF5." [GOC:krc]
GO:0043625
name: delta DNA polymerase complex
def: "A multimeric DNA polymerase enzyme complex which differs in composition amongst species; in humans it is a heterotetramer of four subunits of approximately 125, 50, 68 and 12kDa, while in S. cerevisiae, it has three different subunits which form a heterotrimer, and the active enzyme is a dimer of this heterotrimer. Functions in DNA replication, mismatch repair and excision repair." [GOC:jl, ISBN:0198547684, PMID:11205330, PMID:12403614]
GO:0043626
name: PCNA complex
def: "A protein complex composed of three identical PCNA monomers, each comprising two similar domains, which are joined in a head-to-tail arrangement to form a homotrimer. Forms a ring-like structure in solution, with a central hole sufficiently large to accommodate the double helix of DNA. Originally characterized as a DNA sliding clamp for replicative DNA polymerases and as an essential component of the replisome, and has also been shown to be involved in other processes including Okazaki fragment processing, DNA repair, translesion DNA synthesis, DNA methylation, chromatin remodeling and cell cycle regulation." [GOC:jl, PMID:12829735]
GO:0043655
name: extracellular space of host
def: "The space within a host but external to the plasma membrane of host cells, e.g. within host bloodstream." [GOC:cc]
GO:0043656
name: intracellular region of host
def: "That space within the plasma membrane of a host cell." [GOC:cc]
GO:0043657
name: host cell
def: "A cell within a host organism. Includes the host plasma membrane and any external encapsulating structures such as the host cell wall and cell envelope." [GOC:jl]
GO:0043658
name: host symbiosome
def: "A double-enveloped cell compartment, composed of the endosymbiont with its plasmalemma (as inner envelope) and an outer envelope (the perisymbiontic membrane) derived from the host cell." [GOC:cc]
GO:0043659
name: symbiosome
def: "A double-enveloped cell compartment, composed of an endosymbiont with its plasmalemma (as inner envelope) and a non-endosymbiotic outer envelope (the perisymbiontic membrane)." [GOC:cc]
GO:0043660
name: bacteroid-containing symbiosome
def: "A symbiosome containing any of various structurally modified bacteria, such as those occurring on the root nodules of leguminous plants." [GOC:cc]
GO:0043661
name: peribacteroid membrane
def: "A membrane that surrounds one or more bacteroids (such as nitrogen-fixing bacteroids within legume root nodule cells)." [GOC:cc]
GO:0043662
name: peribacteroid fluid
def: "The soluble material inside the peribacteroid membrane, but outside of the bacteroid, within a bacteroid-containing symbiosome." [GOC:cc]
GO:0043663
name: host bacteroid-containing symbiosome
def: "A symbiosome containing any of various structurally modified bacteria, such as those occurring on the root nodules of leguminous plants, of a host cell." [GOC:cc]
GO:0043664
name: host peribacteroid membrane
def: "A host-derived membrane that surrounds one or more bacteroids (such as nitrogen-fixing bacteroids within legume root nodule cells)." [GOC:cc]
GO:0043665
name: host peribacteroid fluid
def: "The soluble material inside the peribacteroid membrane, but outside of the bacteroid, within a bacteroid-containing symbiosome of a host cell." [GOC:cc]
GO:0043667
name: pollen wall
def: "The complex wall surrounding a pollen grain." [GOC:fz]
GO:0043668
name: exine
def: "The outer layer of the pollen grain wall which is composed primarily of sporopollenin." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043669
name: ectexine
def: "The outer part of the exine, which stains positively with basic fuchsin in optical microscopy and has higher electron density in conventionally prepared TEM sections." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043670
name: foot layer
def: "The inner layer of the ectexine." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043671
name: endexine
def: "The inner part of the exine, which stains." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043672
name: nexine
def: "The inner, non-sculptured part of the exine which lies below the sexine." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043673
name: sexine
def: "The outer, sculptured layer of the exine, which lies above the nexine." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043674
name: columella
def: "A rod-like element of the sexine and ectexine, supporting either the tectum (the layer of sexine which forms a roof over the columella), or supporting a caput (an architectural element on top of a columella)." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043675
name: sculpture element
def: "The third layer of the sexine." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043676
name: tectum
def: "The layer of sexine which forms a roof over the columella, granules or other infratectal elements." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043677
name: germination pore
def: "A small pore in the outer wall of a mycelial spore through which the germ tube exits upon germination. It can be apical or eccentric in its location." [Wikipedia:Germ_pore]
GO:0043678
name: intine
def: "The innermost of the major layers of the pollen grain wall which underlies the exine and borders the cytoplasm." [http://www.mobot.org/MOBOT/research/APweb/]
GO:0043679
name: axon terminus
def: "Terminal inflated portion of the axon, containing the specialized apparatus necessary to release neurotransmitters. The axon terminus is considered to be the whole region of thickening and the terminal button is a specialized region of it." [GOC:dph, GOC:jl]
GO:0043680
name: filiform apparatus
def: "A complex of cell wall invaginations in a synergid cell, similar to those in transfer cells." [ISBN:0471245208]
GO:0043684
name: type IV secretion system complex
def: "A complex of proteins related to those involved in bacterial DNA conjugative transfer, that permits the transfer of DNA or proteins into the extracellular milieu or directly into host cells. In general the type IV complex forms a multisubunit cell-envelope-spanning structure composed of a secretion channel and often a pilus or other surface filament or protein(s)." [GOC:ml]
GO:0043698
name: iridosome
def: "A tissue-specific, membrane-bounded cytoplasmic organelle within which purines crystalize in reflective stacks. Iridosomes are synthesized in iridophore cells and are silver, gold or iridescent in appearance." [GOC:mh]
GO:0043699
name: leucosome
def: "A tissue-specific, membrane-bounded cytoplasmic organelle within which uric acid and/or purines crystalize in reflective stacks. Leucosomes are synthesized in leucophore cells and have a whitish cast." [GOC:mh]
GO:0043700
name: pterinosome
def: "A tissue-specific, membrane-bounded cytoplasmic organelle within which pteridine pigments are synthesized and stored. Pterinosomes are synthesized in xanthophores and erythrophore cells and are yellow, orange or red in appearance." [GOC:mh]
GO:0043701
name: cyanosome
def: "A tissue-specific, membrane-bounded cytoplasmic organelle within which an unknown blue pigment is localized. Cyanosomes are synthesized in cyanophores and are blue in appearance." [GOC:mh]
GO:0043702
name: carotenoid vesicle
def: "A tissue-specific cytoplasmic vesicle surrounded by a membrane half-leaflet within which carotenoid pigments are stored. Carotenoid vescicles are synthesized in xanthophores and erythrophore cells and are yellow, orange or red in appearance." [GOC:mh]
GO:0043769
name: Tpg-containing telomere binding complex
def: "A complex composed of four polypeptides, a telomere-protecting terminal protein (Tpg), a telomere-associated protein (Tap), DNA polymerase (PolA) and topoisomerase I (TopA), that functions in the replication of the telomeric regions of linear chromosomes, plasmids and circular replicons of some bacterial species." [PMID:15353591]
GO:0043845
name: DNA polymerase III, proofreading complex
def: "A subcomplex of DNA polymerase III composed of the epsilon subunit which has proofreading activity, and the theta subunit which enhances the epsilon subunit's proofreading activity." [IMG:01721, PMID:16973612, Wikipedia:Pol_III]
GO:0043846
name: DNA polymerase III, clamp loader complex
def: "A heptamer that includes the tau and gamma products of the dnaX gene and the chi/psi subcomplex. Confers structural asymmetry that allows the polymerase to replicate both leading and lagging strands." [PMID:12940977]
GO:0043847
name: DNA polymerase III, clamp loader chi/psi subcomplex
def: "A dimer composed of the chi and psi subunits which is a subassembly of the DNA polymerase III clamp loader complex and serves as a bridge between the DnaX complex and the single-stranded DNA-binding protein (SSB)." [IMG:01719, PMID:12940977]
GO:0043850
name: RecFOR complex
def: "A heterotrimeric complex composed of the subunits RecF, RecO and RecR. Mediates the loading of RecA protein specifically onto SSB-coated gapped DNA during DNA repair." [IMG:01764, PMID:12769856]
GO:0043853
name: methanol-CoM methyltransferase complex
def: "A heterotrimeric protein complex composed of a methanol methyltransferase subunit, a corrinoid protein and a methanol-specific corrinoid:coenzyme M methyltransferase subunit. Catalyzes the transfer of a methyl group from methanol to coenzyme M as part of the pathway of methanogenesis from methanol." [IMG:01437, MetaCyc:CPLX-421, PMID:9363780]
GO:0043913
name: chromosome segregation-directing complex
def: "A trimeric protein complex which in E. coli is composed of the subunits MreB, MreC and MreD. The complex directs longitudinal cell wall synthesis, maintaining cell morphology." [GOC:jl, PMID:15612918]
GO:0044084
name: host cell membrane pore complex
def: "Any small opening in a host cell membrane that allows the passage of gases and/or liquids, composed of host proteins." [MITRE:tk]
GO:0044094
name: host cell nuclear part
def: "Any constituent part of a host cell's nucleus, a membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. The host is the larger of the organisms involved in a symbiotic interaction." [GOC:ecd]
GO:0044095
name: host cell nucleoplasm
def: "That part of a host cell's nuclear content other than the chromosomes or the nucleolus. The host is the larger of the organisms involved in a symbiotic interaction." [GOC:ecd]
GO:0044096
name: type IV pilus
def: "A short filamentous structure on the surface of a bacterial cell distinguished from other pili by post-translational N-methylation of the pilin monomers." [GOC:pamgo_curators]
GO:0044099
name: polar tube
def: "A highly specialized structure unique to microsporidia that is required for host cell invasion. In the spore, the polar tube is connected at the anterior end, and then coils around the sporoplasm. Upon appropriate environmental stimulation, the polar tube rapidly discharges out of the spore, pierces a cell membrane and serves as a conduit for sporoplasm passage into the new host cell." [GOC:mf, PMID:12076771, PMID:9723921]
GO:0044100
name: sporoplasm
def: "The complex infective apparatus corresponding to the central mass of cytoplasm within a spore that is injected into a host cell by various parasitic microorganisms." [GOC:mf, PMID:12076771, PMID:16004371, PMID:9723921]
GO:0044155
name: host caveola
def: "A small pit, depression, or invagination, such as any of the minute pits or incuppings of the host cell membrane formed during pinocytosis, that communicates with the outside of a host cell and extends inward, indenting the host cytoplasm and the host cell membrane. Such caveolae may be pinched off to form free vesicles within the host cytoplasm. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:rph]
GO:0044156
name: host cell junction
def: "A plasma membrane part that forms a specialized region of connection between two host cells or between a host cell and the host extracellular matrix. At a host cell junction, anchoring proteins extend through the host plasma membrane to link cytoskeletal proteins in one cell to cytoskeletal proteins in neighboring cells or to proteins in the extracellular matrix." [GOC:rph]
GO:0044157
name: host cell projection
def: "A prolongation or process extending from a host cell, e.g. a flagellum or axon." [GOC:rph]
GO:0044158
name: host cell wall
def: "The rigid or semi-rigid envelope lying outside the host cell membrane of plant, fungal, and most prokaryotic cells, maintaining their shape and protecting them from osmotic lysis. In plants it is made of cellulose and, often, lignin; in fungi it is composed largely of polysaccharides; in bacteria it is composed of peptidoglycan." [GOC:rph]
GO:0044159
name: host thylakoid
def: "A membranous cellular structure within the host cell that bears the photosynthetic pigments in plants, algae, and cyanobacteria. In cyanobacteria thylakoids are of various shapes and are attached to, or continuous with, the host plasma membrane. In eukaryotic host cells they are flattened, membrane-bounded disk-like structures located in the chloroplasts; in the chloroplasts of higher plants the thylakoids form dense stacks called grana. Isolated thylakoid preparations can carry out photosynthetic electron transport and the associated phosphorylation." [GOC:rph]
GO:0044160
name: host thylakoid membrane
def: "The pigmented membrane of any host thylakoid." [GOC:rph]
GO:0044161
name: host cell cytoplasmic vesicle
def: "A vesicle formed of membrane or protein, found in the cytoplasm of a host cell." [GOC:rph]
GO:0044162
name: host cell cytoplasmic vesicle membrane
def: "The lipid bilayer surrounding a host cell cytoplasmic vesicle." [GOC:rph]
GO:0044163
name: host cytoskeleton
def: "Any of the various filamentous elements that form the internal framework of host cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the host cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the host cell. The various elements of the host cytoskeleton not only serve in the maintenance of host cellular shape but also have roles in other host cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles." [GOC:rph]
GO:0044164
name: host cell cytosol
def: "The part of the host cell cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes." [GOC:jl]
GO:0044165
name: host cell endoplasmic reticulum
def: "The irregular network of unit membranes, visible only by electron microscopy, that occurs in the host cell cytoplasm of many eukaryotic cells. The membranes form a complex meshwork of tubular channels, which are often expanded into slitlike cavities called cisternae. The host ER takes two forms, rough (or granular), with ribosomes adhering to the outer surface, and smooth (with no ribosomes attached)." [GOC:jl]
GO:0044166
name: host cell endoplasmic reticulum lumen
def: "The volume enclosed by the membranes of the host cell endoplasmic reticulum." [GOC:jl]
GO:0044167
name: host cell endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the host cell endoplasmic reticulum." [GOC:jl]
GO:0044168
name: host cell rough endoplasmic reticulum
def: "The irregular network of unit membranes, visible only by electron microscopy, that occurs in the host cell cytoplasm of many eukaryotic cells. The membranes form a complex meshwork of tubular channels, which are often expanded into slitlike cavities called cisternae. The host rough ER has ribosomes adhering to the outer surface." [GOC:jl]
GO:0044169
name: host cell rough endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the host cell rough endoplasmic reticulum." [GOC:jl]
GO:0044170
name: host cell smooth endoplasmic reticulum
def: "The irregular network of unit membranes, visible only by electron microscopy, that occurs in the host cell cytoplasm of many eukaryotic cells. The membranes form a complex meshwork of tubular channels, which are often expanded into slitlike cavities called cisternae. The host smooth ER has no ribosomes adhering to the outer surface." [GOC:jl]
GO:0044171
name: host cell smooth endoplasmic reticulum membrane
def: "The lipid bilayer surrounding the host cell smooth endoplasmic reticulum." [GOC:jl]
GO:0044172
name: host cell endoplasmic reticulum-Golgi intermediate compartment
def: "A complex system of membrane-bounded compartments located between host cell endoplasmic reticulum (ER) and the host Golgi complex, with a distinctive membrane protein composition; involved in ER-to-Golgi transport." [GOC:jl, GOC:pr]
GO:0044173
name: host cell endoplasmic reticulum-Golgi intermediate compartment membrane
def: "The lipid bilayer surrounding any of the compartments of the host cell ER-Golgi intermediate compartment system." [GOC:jl]
GO:0044174
name: host cell endosome
def: "A membrane-bounded organelle that carries materials newly ingested by endocytosis. It passes many of the materials to host cell lysosomes for degradation." [GOC:jl]
GO:0044175
name: host cell endosome membrane
def: "The lipid bilayer surrounding a host cell endosome." [GOC:jl]
GO:0044176
name: host cell filopodium
def: "Thin, stiff protrusion extended by the leading edge of a motile host cell such as a crawling fibroblast or amoeba, or an axonal growth cone; usually approximately 0.1 um wide, 5-10 um long, can be up to 50 um long in axon growth cones; contains a loose bundle of about 20 actin filaments oriented with their plus ends pointing outward." [GOC:jl]
GO:0044177
name: host cell Golgi apparatus
def: "A compound membranous cytoplasmic organelle of eukaryotic host cells, consisting of flattened, ribosome-free vesicles arranged in a more or less regular stack." [GOC:jl]
GO:0044178
name: host cell Golgi membrane
def: "The lipid bilayer surrounding any of the compartments of the host cell Golgi apparatus." [GOC:jl]
GO:0044184
name: host cell late endosome
def: "A prelysosomal endocytic organelle differentiated from host early endosomes by lower lumenal pH and different protein composition. Host late endosomes are more spherical than early endosomes and are mostly juxtanuclear, being concentrated near the microtubule organizing center." [GOC:jl]
GO:0044185
name: host cell late endosome membrane
def: "The lipid bilayer surrounding a host cell late endosome." [GOC:jl]
GO:0044186
name: host cell lipid particle
def: "Any particle of coalesced lipids in the cytoplasm of a host cell. May include associated proteins." [GOC:jl]
GO:0044187
name: host cell lysosome
def: "A small lytic vacuole that has cell cycle-independent morphology and is found in most host animal cells and that contains a variety of hydrolases, most of which have their maximal activities in the pH range 5-6. The contained enzymes display latency if properly isolated. About 40 different lysosomal hydrolases are known and host cell lysosomes have a great variety of morphologies and functions." [GOC:jl]
GO:0044188
name: host cell lysosomal membrane
def: "The lipid bilayer surrounding the host cell lysosome and separating its contents from the host cell cytoplasm." [GOC:jl]
GO:0044189
name: host cell microsome
def: "OBSOLETE: Any of the small, heterogeneous, artifactual, vesicular particles, 50-150 nm in diameter, that are formed when some eukaryotic host cells are homogenized and that sediment on centrifugation at 100000 g." [GOC:jl]
GO:0044190
name: host cell mitochondrial envelope
def: "The double lipid bilayer enclosing the host cell mitochondrion and separating its contents from the host cell cytoplasm; includes the intermembrane space." [GOC:jl]
GO:0044191
name: host cell mitochondrial membrane
def: "Either of the lipid bilayers that surround the host cell mitochondrion and form the host cell mitochondrial envelope." [GOC:jl]
GO:0044192
name: host cell mitochondrial inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the host cell mitochondrial envelope. It is highly folded to form cristae." [GOC:jl]
GO:0044193
name: host cell mitochondrial outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the host cell mitochondrial envelope." [GOC:jl]
GO:0044194
name: cytolytic granule
def: "A specialized secretory lysosome that is present in cells with cytolytic capability such as cytotoxic T lymphocytes and natural killer cells. Cytolytic granules mediate the storage and regulated excretion of lytic molecules for killing of target cells." [GOC:jl, PMID:11052265, PMID:12766758]
GO:0044195
name: nucleoplasmic reticulum
def: "Long, dynamic tubular channels, formed by invagination of the nuclear envelope, that extend deep into the nucleoplasm. The channels have an underlying lamina and are implicated in functioning in signaling and transport." [GOC:jl, PMID:17959832, PMID:9024685]
GO:0044196
name: host cell nucleolus
def: "A small, dense body one or more of which are present in the nucleus of eukaryotic host cells." [GOC:jl]
GO:0044199
name: host cell nuclear envelope
def: "The double lipid bilayer enclosing the host nucleus and separating its contents from the rest of the host cytoplasm; includes the intermembrane space, a gap of width 20-40 nm (also called the perinuclear space)." [GOC:jl]
GO:0044200
name: host cell nuclear membrane
def: "Either of the lipid bilayers that surround the host nucleus and form the nuclear envelope; excludes the intermembrane space." [GOC:jl]
GO:0044201
name: host cell nuclear inner membrane
def: "The inner, i.e. lumen-facing, lipid bilayer of the host nuclear envelope." [GOC:jl]
GO:0044202
name: host cell nuclear outer membrane
def: "The outer, i.e. cytoplasm-facing, lipid bilayer of the host nuclear envelope; continuous with the endoplasmic reticulum of the host cell and sometimes studded with ribosomes." [GOC:jl]
GO:0044203
name: host cell nuclear lamina
def: "The fibrous, electron-dense layer lying on the nucleoplasmic side of the inner membrane of a host cell nucleus, composed of lamin filaments." [GOC:jl]
GO:0044204
name: host cell nuclear matrix
def: "The dense fibrillar network lying on the inner side of the host nuclear membrane." [GOC:jl]
GO:0044207
name: translation initiation ternary complex
def: "A ribonucleoprotein complex that contains aminoacylated initiator methionine tRNA, GTP, and initiation factor 2 (either eIF2 in eukaryotes, or IF2 in prokaryotes). In prokaryotes, fMet-tRNA (initiator) is used rather than Met-tRNA (initiator)." [GOC:jl]
GO:0044214
name: spanning component of plasma membrane
def: "The component of the plasma membrane consisting of gene products and protein complexes that have some part that spans both leaflets of the membrane.\"" [GOC:ecd]
GO:0044215
name: other organism
def: "A secondary organism with which the first organism is interacting." [GOC:jl]
GO:0044216
name: other organism cell
def: "A cell of a secondary organism with which the first organism is interacting." [GOC:jl]
GO:0044217
name: other organism part
def: "Any constituent part of a secondary organism with which the first organism is interacting." [GOC:jl]
GO:0044218
name: other organism cell membrane
def: "The cell membrane of a secondary organism with which the first organism is interacting." [GOC:jl]
GO:0044219
name: host cell plasmodesma
def: "A fine cytoplasmic channel, found in all higher plants, that connects the cytoplasm of one host cell to that of an adjacent host cell." [GOC:rph]
GO:0044220
name: host cell perinuclear region of cytoplasm
def: "The host cell cytoplasm situated near, or occurring around, the host nucleus." [GOC:rph]
GO:0044221
name: host cell synapse
def: "The junction between a nerve fiber of one host neuron and another host neuron or muscle fiber or glial cell; the site of interneuronal communication." [GOC:rph]
GO:0044222
name: anammoxosome
def: "An intracytoplasmic membrane-bounded compartment in anaerobic ammonium oxidation (anammox) bacteria, is the site of anammox catabolism." [GOC:dh, PMID:17993524, PMID:19682260]
GO:0044223
name: pirellulosome
def: "A cytoplasmic structure found in bacterial phyla Planctomycetes and Verrucomicrobia containing a condensed nucleoid and ribosomes and surrounded by an intracytoplasmic membrane. It is surrounded by ribosome-free cytoplasm, in a compartment called the paryphoplasm." [GOC:dh, PMID:19133117]
GO:0044224
name: juxtaparanode region of axon
def: "A region of an axon near a node of Ranvier that is between the paranode and internode regions." [GOC:BHF, GOC:jl, PMID:10624965, PMID:14682359]
GO:0044225
name: apical pole of neuron
def: "Portion of a neuron cell soma closest to the point where the apical dendrite emerges." [NIF_Subcellular:sao1186862860]
GO:0044226
name: basal pole of neuron
def: "Portion of a neuron cell soma closest to the point where the basilar dendrite emerges." [NIF_Subcellular:sao1186862860]
GO:0044227
name: methane-oxidizing organelle
def: "A cytoplasmic, membrane-bounded compartment found within Methanotrophic bacteria that contains enzymes and electron transfer proteins for methane catabolism. This structure is analogous to the thylakoid of Cyanobacteria and the anammoxosome of anaerobic ammonium oxidation organisms." [GOC:dh]
GO:0044228
name: host cell surface
def: "The external part of the host cell wall and/or host plasma membrane." [GOC:rph]
GO:0044229
name: host cell periplasmic space
def: "The region between the inner (cytoplasmic) and outer host membrane (Gram-negative Bacteria) or inner host membrane and host cell wall (Fungi)." [GOC:rph]
GO:0044230
name: host cell envelope
def: "An envelope that surrounds a bacterial host cell and includes the host cytoplasmic membrane and everything external, encompassing the host periplasmic space, host cell wall, and host outer membrane if present." [GOC:rph]
GO:0044231
name: host cell presynaptic membrane
def: "A specialized area of membrane of the host axon terminal that faces the plasma membrane of the host neuron or muscle fiber with which the axon terminal establishes a synaptic junction; many host synaptic junctions exhibit structural presynaptic characteristics, such as conical, electron-dense internal protrusions, that distinguish it from the remainder of the axon plasma membrane." [GOC:rph]
GO:0044232
name: organelle membrane contact site
def: "A zone of apposition between the membranes of two organelles, structured by bridging complexes. Membrane contact sites (MCSs) are specialized for communication, including the efficient traffic of small molecules such as Ca2+ ions and lipids, as well as enzyme-substrate interactions." [GOC:jl, PMID:16806880]
GO:0044233
name: ER-mitochondrion membrane contact site
def: "A zone of apposition between endoplasmic-reticulum and mitochondrial membranes, structured by bridging complexes. These contact sites are thought to facilitate inter-organelle calcium and phospholipid exchange." [GOC:jl, PMID:19556461]
GO:0044279
name: other organism membrane
def: "A membrane of a secondary organism with which the first organism is interacting." [GOC:jl]
GO:0044280
name: subplasmalemmal coating
def: "Electron dense material observed coating the cytoplasmic face of the plasma membrane in certain regions of a neuron, e.g., the axon initial segment; the nodal membrane at the Node of Ranvier." [NIF_Subcellular:sao1938587839]
GO:0044284
name: mitochondrial crista junction
def: "A tubular structure of relatively uniform size that connects a mitochondrial crista to the mitochondrial inner boundary membrane." [GOC:mcc, NIF_Subcellular:sao1825845900, PMID:21944719, PMID:21987634, PMID:22009199]
GO:0044285
name: bridge contact site
def: "Site of contact between the inner and outer mitochondrial membrane found in neuronal mitochondria; may play a role in maintaining the structural integrity of the inner and outer boundary membranes." [NIF_Subcellular:sao1858501007]
GO:0044286
name: peg and socket contact
def: "A cell-cell contact zone that consists of membrane invaginations extending from either cell, which contain tight-, gap-, and adherens junctions. Peg and socket contacts form between endothelial cells and pericytes, and between lens fiber cells." [GOC:tfm, NIF_Subcellular:sao1943947957, PMID:12883993, PMID:16166562, PMID:17591898]
GO:0044288
name: puncta adhaerentia
def: "A small version of the zonula adherens type junction, characterized by a symmetrical adherent point between two cells." [NIF_Subcellular:sao257629430]
GO:0044289
name: contact site
def: "Sites of close apposition of the inner and outer mitochondrial membrane." [NIF_Subcellular:sao447856407]
GO:0044290
name: mitochondrial intracristal space
def: "The space bounded by the mitochondrial cristae membranes, continuous with the intermembrane space." [NIF_Subcellular:sao508958414]
GO:0044291
name: cell-cell contact zone
def: "Extended zone of intimate apposition between two cells containing one or more types of intercellular junctions, e.g., the intercalated disk of muscle." [NIF_Subcellular:sao1299635018]
GO:0044292
name: dendrite terminus
def: "A structure at the distal end of a dendrite adapted to carry out a specific function, e.g. dendriole." [GOC:jl, NIF_Subcellular:sao28175134]
GO:0044293
name: dendriole
def: "Small dendrites that makes up a brush structure found as the terminal specialization of a dendrite of a unipolar brush cell (UBC)." [GOC:jl, NIF_Subcellular:sao28175134, NIF_Subcellular:sao295057932, PMID:8300904]
GO:0044294
name: dendritic growth cone
def: "The migrating motile tip of a growing nerve cell dendrite." [GOC:jl]
GO:0044295
name: axonal growth cone
def: "The migrating motile tip of a growing nerve cell axon." [GOC:jl, NIF_Subcellular:sao203987954]
GO:0044296
name: dendritic tuft
def: "The terminal specialization found in some types of dendrites which consists of numerous small terminal branches, giving the dendrite a tufted appearance." [NIF_Subcellular:sao1340260079]
GO:0044297
name: cell body
def: "The portion of a cell bearing surface projections such as axons, dendrites, cilia, or flagella that includes the nucleus, but excludes all cell projections." [GOC:go_curators]
GO:0044298
name: cell body membrane
def: "The plasma membrane of a cell that bears surface projections such as axons, dendrites, cilia, or flagella, excluding the plasma membrane on cell projections." [GOC:ecd]
GO:0044299
name: C-fiber
def: "The axon of a dorsal root ganglion cell that are responsive to pain and temperature. C-fibers are small in diameter (0.2-1.5 um) and unmyelinated." [NIF_Subcellular:nlx_subcell_20090210]
GO:0044300
name: cerebellar mossy fiber
def: "An axon arising from cerebellar projecting cells in the cochlea, vestibular nuclei, spinal cord, reticular formation, cerebellar nuclei and basilar pontine nuclei. Mossy fibers enter through all three cerebellar peduncles and send collaterals to the deep cerebellar nuclei, then branch in the white matter and terminate in the granule cell layer. Through this branching, a given mossy fiber can innervate several folia. Mossy fibers synapse on granule cells. The synaptic contacts are made at enlargements along the length of the mossy fiber called mossy fiber rosettes. The enlargements of the rosettes give the axons as \"mossy\" appearance in Golgi stained preparations." [NIF_Subcellular:nlx_subcell_20090209]
GO:0044301
name: climbing fiber
def: "The axon of inferior olive neuron that projects to the cerebellar cortex, largely via the inferior cerebellar peduncle. They range in diameter from 1-3 um and are myelinated until they enter the granule cell layer. They give off collaterals to the deep cerebellar nuclei. They synapse extensively with the dendrites of Purkinje cells in the molecular layer, where each fiber branches repeatedly to \"climb\" along the Purkinje cell dendritic tree. Each Purkinje cell is innervated by only a single climbing fiber." [NIF_Subcellular:nlx_subcell_20090203]
GO:0044302
name: dentate gyrus mossy fiber
def: "Distinctive, unmyelinated axons produced by granule cells." [NIF_Subcellular:nlx_subcell_20090601, PMID:17765709]
GO:0044303
name: axon collateral
def: "Any of the smaller branches of an axon that emanate from the main axon cylinder." [NIF_Subcellular:sao1470140754]
GO:0044304
name: main axon
def: "The main axonal trunk, as opposed to the collaterals." [NIF_Subcellular:sao1596975044]
GO:0044305
name: calyx of Held
def: "The terminal specialization of a calyciferous axon which forms large synapses in the mammalian auditory central nervous system." [NIF_Subcellular:sao1684283879, PMID:11823805]
GO:0044306
name: neuron projection terminus
def: "The specialized, terminal region of a neuron projection such as an axon or a dendrite." [GOC:jl]
GO:0044307
name: dendritic branch
def: "A dendrite arising from another dendrite." [NIF_Subcellular:sao884265541]
GO:0044308
name: axonal spine
def: "A spine that originates from the axon, usually from the initial segment." [NIF_Subcellular:sao18239917]
GO:0044309
name: neuron spine
def: "A small membranous protrusion, often ending in a bulbous head and attached to the neuron by a narrow stalk or neck." [ISBN:0198504888, NIF_Subcellular:sao1145756102]
GO:0044310
name: osmiophilic body
def: "A membrane-bounded vesicle found predominantly in Plasmodium female gametocytes, that becomes progressively more abundant as the gametocyte reaches full maturity. These vesicles lie beneath the subpellicular membrane of the gametocyte, and the release of their contents into the parasitophorous vacuole has been postulated to aid in the escape of gametocytes from the erythrocyte after ingestion by the mosquito." [GOC:jl, PMID:18086189]
GO:0044311
name: exoneme
def: "A dense granule-like organelle of the apical complex of merozoites, released into the parasitophorous vacuole, mediating protease-dependent rupture and parasite exit from the infected erythrocyte." [GOC:jl, PMID:18083092, PMID:18083098]
GO:0044312
name: crystalloid
def: "A transient, cytoplasmic organelle found in Plasmodium species that resembles a cytoplasmic inclusion body and whose function is poorly understood. Crystalloids form in ookinetes and disappear after ookinete-to-oocyst transformation." [GOC:jl, PMID:19932717]
GO:0044316
name: cone cell pedicle
def: "A specialized axon terminus which is produced by retinal cone cells. Pedicles are large, conical, flat end-feet (8-10 micrometers diameter) of the retinal cone axon that lie more or less side by side on the same plane at the outer edge of the outer plexiform layer (OPL)." [http://webvision.med.utah.edu/photo2.html, PMID:10939333]
GO:0044317
name: rod spherule
def: "A specialized neuron projection which is the site of synaptic transmission produced by retinal rod cells. Rod spherules are small round enlargements of the axon (3-5 micrometers diameter) or even extensions of the cell body." [http://webvision.med.utah.edu/photo2.html]
GO:0044322
name: endoplasmic reticulum quality control compartment
def: "A subcompartment of the endoplasmic reticulum in which proteins with improper or incorrect folding accumulate. Enzymes in this compartment direct proteins with major folding problems to translocation to the cytosol and degradation, and proteins with minor folding problems to the ER, to interact with chaperon proteins." [PMID:11408579]
GO:0044326
name: dendritic spine neck
def: "Part of the dendritic spine that connects the dendritic shaft to the head of the dendritic spine." [GOC:nln]
GO:0044327
name: dendritic spine head
def: "Distal part of the dendritic spine, that carries the post-synaptic density." [GOC:BHF, GOC:nln, GOC:rl]
GO:0044352
name: pinosome
def: "A membrane-bounded, uncoated intracellular vesicle formed by the process of pinocytosis." [PMID:14731589, PMID:14732047]
GO:0044353
name: micropinosome
def: "A membrane-bounded, uncoated intracellular vesicle formed by the process of micropinocytosis." [PMID:14731589, PMID:14732047]
GO:0044354
name: macropinosome
def: "A membrane-bounded, uncoated intracellular vesicle formed by the process of macropinocytosis." [PMID:14732047]
GO:0044383
name: host chromosome
def: "A structure composed of a very long molecule of DNA and associated proteins (e.g. histones) that carries hereditary information, occurring within a host cell." [GOC:jl]
GO:0044384
name: host outer membrane
def: "The external membrane of Gram-negative bacteria or certain organelles such as mitochondria and chloroplasts; freely permeable to most ions and metabolites, occurring in a host cell." [GOC:jl]
GO:0044385
name: integral to membrane of host cell
def: "Penetrating at least one phospholipid bilayer of a membrane. May also refer to the state of being buried in the bilayer with no exposure outside the bilayer. When used to describe a protein, indicates that all or part of the peptide sequence is embedded in the membrane. Occurring in a host cell." [GOC:jl]
GO:0044386
name: integral to host endoplasmic reticulum membrane
def: "Penetrating at least one phospholipid bilayer of an endoplasmic reticulum membrane. May also refer to the state of being buried in the bilayer with no exposure outside the bilayer. Occurring in a host cell." [GOC:jl]
GO:0044391
name: ribosomal subunit
def: "Either of the two subunits of a ribosome: the ribosomal large subunit or the ribosomal small subunit." [GOC:jl]
GO:0044393
name: microspike
def: "A dynamic, actin-rich projection extending from the surface of a migrating animal cell." [PMID:11429692, PMID:12153987, PMID:19095735]
GO:0044420
name: extracellular matrix part
def: "Any constituent part of the extracellular matrix, the structure lying external to one or more cells, which provides structural support for cells or tissues; may be completely external to the cell (as in animals) or be part of the cell (as often seen in plants)." [GOC:jl]
GO:0044421
name: extracellular region part
def: "Any constituent part of the extracellular region, the space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers constituent parts of the host cell environment outside an intracellular parasite." [GOC:jl]
GO:0044422
name: organelle part
def: "Any constituent part of an organelle, an organized structure of distinctive morphology and function. Includes constituent parts of the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton, but excludes the plasma membrane." [GOC:jl]
GO:0044423
name: virion part
def: "Any constituent part of a virion, a complete fully infectious extracellular virus particle." [GOC:jl]
GO:0044424
name: intracellular part
def: "Any constituent part of the living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm." [GOC:jl]
GO:0044425
name: membrane part
def: "Any constituent part of a membrane, a double layer of lipid molecules that encloses all cells, and, in eukaryotes, many organelles; may be a single or double lipid bilayer; also includes associated proteins." [GOC:jl]
GO:0044426
name: cell wall part
def: "Any constituent part of the cell wall, the rigid or semi-rigid envelope lying outside the cell membrane of plant, fungal, and most prokaryotic cells, maintaining their shape and protecting them from osmotic lysis." [GOC:jl]
GO:0044427
name: chromosomal part
def: "Any constituent part of a chromosome, a structure composed of a very long molecule of DNA and associated proteins (e.g. histones) that carries hereditary information." [GOC:jl]
GO:0044428
name: nuclear part
def: "Any constituent part of the nucleus, a membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated." [GOC:jl]
GO:0044429
name: mitochondrial part
def: "Any constituent part of a mitochondrion, a semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration." [GOC:jl]
GO:0044430
name: cytoskeletal part
def: "Any constituent part of the cytoskeleton, a cellular scaffolding or skeleton that maintains cell shape, enables some cell motion (using structures such as flagella and cilia), and plays important roles in both intra-cellular transport (e.g. the movement of vesicles and organelles) and cellular division. Includes constituent parts of intermediate filaments, microfilaments, microtubules, and the microtrabecular lattice." [GOC:jl]
GO:0044431
name: Golgi apparatus part
def: "Any constituent part of the Golgi apparatus, a compound membranous cytoplasmic organelle of eukaryotic cells, consisting of flattened, ribosome-free vesicles arranged in a more or less regular stack." [GOC:jl]
GO:0044432
name: endoplasmic reticulum part
def: "Any constituent part of the endoplasmic reticulum, the irregular network of unit membranes, visible only by electron microscopy, that occurs in the cytoplasm of many eukaryotic cells. The membranes form a complex meshwork of tubular channels, which are often expanded into slitlike cavities called cisternae." [GOC:jl]
GO:0044433
name: cytoplasmic vesicle part
def: "Any constituent part of cytoplasmic vesicle, a vesicle formed of membrane or protein, found in the cytoplasm of a cell." [GOC:jl]
GO:0044434
name: chloroplast part
def: "Any constituent part of a chloroplast, a chlorophyll-containing plastid with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma." [GOC:jl]
GO:0044435
name: plastid part
def: "Any constituent part of a plastid, a member of a family of organelles found in the cytoplasm of plants and some protists, which are membrane-bounded and contain DNA. Plant plastids develop from a common type, the proplastid." [GOC:jl]
GO:0044436
name: thylakoid part
def: "Any constituent part of a thylakoid, a sac-like vesicle that bears the photosynthetic pigments in photosynthetic organisms." [GOC:jl]
GO:0044437
name: vacuolar part
def: "Any constituent part of a vacuole, a closed structure, found only in eukaryotic cells, that is completely surrounded by unit membrane and contains liquid material." [GOC:jl]
GO:0044438
name: microbody part
def: "Any constituent part of a microbody, a cytoplasmic organelle, spherical or oval in shape, that is bounded by a single membrane and contains oxidative enzymes, especially those utilizing hydrogen peroxide (H2O2)." [GOC:jl]
GO:0044439
name: peroxisomal part
def: "Any constituent part of a peroxisome, a small, membrane-bounded organelle that uses dioxygen (O2) to oxidize organic molecules; contains some enzymes that produce and others that degrade hydrogen peroxide (H2O2)." [GOC:jl]
GO:0044440
name: endosomal part
def: "Any constituent part of an endosome, a membrane-bounded organelle to which materials ingested by endocytosis are delivered." [GOC:mah, PMID:19696797]
GO:0044441
name: ciliary part
def: "Any constituent part of a cilium, a specialized eukaryotic organelle that consists of a filiform extrusion of the cell surface. Each cilium is bounded by an extrusion of the cytoplasmic (plasma) membrane, and contains a regular longitudinal array of microtubules, anchored basally in a centriole." [GOC:cilia, GOC:jl]
GO:0044443
name: pilus part
def: "Any constituent part of a pilus, a proteinaceous hair-like appendage on the surface of bacteria ranging from 2-8 nm in diameter." [GOC:pamgo_curators]
GO:0044444
name: cytoplasmic part
def: "Any constituent part of the cytoplasm, all of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures." [GOC:jl]
GO:0044445
name: cytosolic part
def: "Any constituent part of cytosol, that part of the cytoplasm that does not contain membranous or particulate subcellular components." [GOC:jl]
GO:0044446
name: intracellular organelle part
def: "A constituent part of an intracellular organelle, an organized structure of distinctive morphology and function, occurring within the cell. Includes constituent parts of the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton but excludes the plasma membrane." [GOC:jl]
GO:0044447
name: axoneme part
def: "Any constituent part of an axoneme, the bundle of microtubules and associated proteins that forms the core of cilia (also called flagella) in eukaryotic cells and is responsible for their movements." [GOC:cilia, GOC:jl]
GO:0044448
name: cell cortex part
def: "Any constituent part of the cell cortex, the region of a cell that lies just beneath the plasma membrane and often, but not always, contains a network of actin filaments and associated proteins." [GOC:jl]
GO:0044449
name: contractile fiber part
def: "Any constituent part of a contractile fiber, a fiber composed of actin, myosin, and associated proteins, found in cells of smooth or striated muscle." [GOC:jl]
GO:0044450
name: microtubule organizing center part
def: "Any constituent part of a microtubule organizing center, a region in a eukaryotic cell, such as a centrosome or basal body, from which microtubules grow." [GOC:jl]
GO:0044451
name: nucleoplasm part
def: "Any constituent part of the nucleoplasm, that part of the nuclear content other than the chromosomes or the nucleolus." [GOC:jl]
GO:0044452
name: nucleolar part
def: "Any constituent part of a nucleolus, a small, dense body one or more of which are present in the nucleus of eukaryotic cells. It is rich in RNA and protein, is not bounded by a limiting membrane, and is not seen during mitosis." [GOC:jl]
GO:0044453
name: nuclear membrane part
def: "Any constituent part of the nuclear membrane, the envelope that surrounds the nucleus of eukaryotic cells." [GOC:jl]
GO:0044454
name: nuclear chromosome part
def: "Any constituent part of a nuclear chromosome, a chromosome found in the nucleus of a eukaryotic cell." [GOC:jl]
GO:0044455
name: mitochondrial membrane part
def: "Any constituent part of a mitochondrial membrane, either of the lipid bilayers that surround the mitochondrion and form the mitochondrial envelope." [GOC:jl]
GO:0044456
name: synapse part
def: "Any constituent part of a synapse, the junction between a nerve fiber of one neuron and another neuron or muscle fiber or glial cell." [GOC:jl]
GO:0044457
name: cell septum part
def: "Any constituent part of a cell septum, a structure composed of peptidoglycan and often chitin in addition to other materials. It usually forms perpendicular to the long axis of a cell or hypha and grows centripetally from the cell wall to the center of the cell and often functions in the compartmentalization of a cell into two daughter cells." [GOC:jl]
GO:0044459
name: plasma membrane part
def: "Any constituent part of the plasma membrane, the membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins." [GOC:jl]
GO:0044460
name: flagellum part
def: "OBSOLETE. Any constituent part of a flagellum, a long whiplike or feathery structure borne either singly or in groups by the motile cells of many bacteria and unicellular eukaryotes and by the motile male gametes of many eukaryotic organisms, which propel the cell through a liquid medium." [GOC:jl]
GO:0044461
name: bacterial-type flagellum part
def: "Any constituent part of flagellum, a 20 nm diameter filament composed of subunits of flagellin driven passively at its base by a motor powered by the transmembrane proton potential. Examples of this component are found in bacterial species." [GOC:jl, GOC:mtg_sensu]
GO:0044462
name: external encapsulating structure part
def: "Any constituent part of an external encapsulating structure, a structure that lies outside the plasma membrane and surrounds the entire cell." [GOC:jl]
GO:0044463
name: cell projection part
def: "Any constituent part of a cell projection, a prolongation or process extending from a cell, e.g. a flagellum or axon." [GOC:jl]
GO:0044464
name: cell part
def: "Any constituent part of a cell, the basic structural and functional unit of all organisms." [GOC:jl]
GO:0044530
name: supraspliceosomal complex
def: "Multicomponent complex of RNA and proteins that is composed of four active spliceosomes, termed native spliceosomes, connected to each other by the pre-mRNA. The supraspliceosome is the nuclear machine where the pre-mRNA processing takes place, like the 5'-end capping, 3'-end cleavage, splicing and editing." [GOC:ans, GOC:jl, PMID:19282290]
GO:0044538
name: host cell periphery
def: "The part of a cell encompassing the cell cortex, the plasma membrane, and any external encapsulating structures of a host cell." [GOC:jl, PMID:20463076]
GO:0044545
name: NSL complex
def: "A histone acetyltransferase complex that catalyzes the acetylation of a histone H4 lysine residues at several positions. In human, it contains the catalytic subunit MOF, NSL1/KIAA1267, NSL2/KANSL2, NSL3/KANSL3, MCRS1, PHF20, OGT1, WDR5 and HCF1." [GOC:lb, PMID:20018852]
GO:0044567
name: primary cell wall cellulose synthase complex
def: "A large, multimeric protein complex which catalyzes the biosynthesis of cellulose for the plant primary cell wall. In Arabidopsis, contains the essential component proteins CESA1 and -3, and a CESA6-related protein." [GOC:mengo_curators, GOC:tt, PMID:17878302, PMID:21307367]
GO:0044568
name: secondary cell wall cellulose synthase complex
def: "A large, multimeric protein complex which catalyzes the biosynthesis of cellulose for the plant secondary cell wall. In Arabidopsis, contains the essential component proteins CESA8, CESA7, and CESA4." [GOC:mengo_curators, GOC:tt, PMID:21307367]
GO:0044569
name: [Ni-Fe] hydrogenase complex
def: "A microbial enzyme complex which contains nickel and iron in its active site. In Acetomicrobium flavidum it is an alpha 2 beta 2 tetramer." [GOC:mengo_curators, GOC:tt, PMID:8936309]
GO:0044570
name: starch utilization system complex
def: "A bacterial cell envelope-associated multiprotein system, which binds and degrades starch." [GOC:mengo_curators, GOC:tt, PMID:19553672]
GO:0044599
name: AP-5 adaptor complex
def: "An AP-type membrane coat adaptor complex that in humans consists of beta5, zeta, mu5 and sigma5 subunits and is found associated with membranes in the endosomes; it is not clear whether AP-5 forms clathrin coats in vivo." [PMID:22022230]
GO:0044609
name: DBIRD complex
def: "A protein complex that associates with mRNP particles and RNA polymerase II and is proposed to integrate transcript elongation with the regulation of alternative splicing. In humans it is composed of the proteins KIAA1967/DBC1 and ZNF326/ZIRD." [GOC:sp, PMID:22446626]
GO:0044611
name: nuclear pore inner ring
def: "A subcomplex of the nuclear pore complex (NPC) that forms the inner rings of the core scaffold, a lattice-like structure that gives the NPC its shape and strength. In S. cerevisiae, the two inner rings are each composed of Nup192p, Nup188p, Nup170p and Nup157p. In vertebrates, the two inner rings are each composed of Nup205, Nup188 and Nup155. Components are arranged in 8-fold symmetrical 'spokes' around the central transport channel. A single 'spoke', can be isolated and is sometimes referred to as the Nup170 complex." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0044612
name: nuclear pore linkers
def: "A substructure of the nuclear pore complex (NPC) that serves to connect members of the central transport channel (composed of FG-nucleoporins) to the core scaffold (composed of the inner and outer NPC rings). In S. cerevisiae, the linkers are Nic96p and Nup82p. In vertebrates, they are Nup93 and Nup88. Components are arranged in 8-fold symmetrical 'spokes' around the central transport channel. Both linkers can be isolated in association with specific FG-nucleoporins, complexes that are sometimes referred to as the Nic96 complex (Nic96p-Nsp1p-Nup49p-Nup57p) and the Nup82 complex (Nup82p-Nup116p-Nup159p-Gle2p)." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0044613
name: nuclear pore central transport channel
def: "The central substructure of the nuclear pore complex (NPC), through which nucleocytoplasmic transport of RNAs, proteins and small molecules occurs. The central transport channel is filled with FG-nucleoporins, which form a selective barrier and provide a series of binding sites for transporter proteins. Characterized S. cerevisiae FG-nucleoporins include Nup159p, Nup145Np, Nup116p, Nup100p, Nsp1p, Nup57p, Nup49p, Nup42p, Nup53p, Nup59p/Asm4p, Nup60p and Nup1. Characterized vertebrate FG-nucleoporins include Nup214, Nup98, Nup62, Nup54, Nup58/45, NLP1, and Nup153." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0044614
name: nuclear pore cytoplasmic filaments
def: "Filamentous extensions on cytoplasmic face of the nuclear pore complex (NPC). In S. cerevisiae, Nup159p, Nup82p, and Nup42p contribute to the cytoplasmic filaments. In vertebrates, Nup358 is a major component." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0044615
name: nuclear pore nuclear basket
def: "A filamentous, cage-like assembly on the nuclear face of the nuclear pore complex (NPC). In S. cerevisiae, Mlp1p and Mlp2p are two major components of the NPC nuclear basket. In vertebrates, Tpr is a major component." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0044647
name: host-symbiont tight junction
def: "An occluding cell-cell junction formed between the membranes of the apical end of an invading cell (e.g. a merozoite in Plasmodium) and a host target cell (e.g. erythrocyte for Plasmodium infection). The junction is a stable yet dynamic structure that moves around the symbiont cell during invasion, enclosing it in a vacuole surrounded by a membrane." [GOC:jl, PMID:21803641]
GO:0044665
name: MLL1/2 complex
def: "A protein complex that can methylate lysine-4 of histone H3, and which contains either of the protein subunits MLL1 or MLL2 in human, or equivalent in other species." [GOC:sart, PMID:21875999]
GO:0044666
name: MLL3/4 complex
def: "A protein complex that can methylate lysine-4 of histone H3, and which contains either of the protein subunits MLL3 or MLL4 in mammals, or equivalent in other species." [GOC:sart, PMID:21875999]
GO:0044672
name: acetyl-CoA decarbonylase/synthase-carbon monoxide dehydrogenase complex
def: "A multifunctional enzyme complex composed of five different polypeptides that catalyzes the decarbonylation of acetyl-CoA, cleaves the C-C and C-S bonds in the acetyl moiety of acetyl-CoA, oxidizes the carbonyl group to CO2 and transfers the methyl group to tetrahydrosarcinapterin. These reactions are important for methanogenesis." [GOC:mengo_curators, PMID:11607176, PMID:7693685, PMID:8955306]
GO:0044673
name: 7,8-didemethyl-8-hydroxy-5-deazariboflavin synthase complex
def: "A heterodimer which catalyses the reaction of 5-amino-6-(D-ribitylamino)uracil and 4-hydroxyphenylpyruvate to form 7,8-didemethyl-8-hydroxy-5-deazariboflavin (FO), an intermediate of coenzyme F420." [GOC:mengo_curators, PMID:14593448]
GO:0044674
name: methyl coenzyme M reductase complex
def: "A hexameric complex consisting of three polypeptides in an alpha2beta2gamma2 arrangement. Involved in the reduction of the coenzyme M-bound methyl group to methane, which is the final step in methanogenesis." [GOC:mengo_curators, PMID:9367957]
GO:0044675
name: formyl-methanofuran dehydrogenase (tungsten enzyme) complex
def: "A protein complex consisting of four polypeptides which also contains tungsten, a molybdopterin guanine dinucleotide, and iron-sulfur clusters. This protein complex catalyzes the reversible conversion of CO2 and methanofuran to formylmethanofuran during methanogenesis." [GOC:mengo_curators, PMID:8125106, PMID:8575452]
GO:0044676
name: formyl-methanofuran dehydrogenase (molybdenum enzyme) complex
def: "A protein complex consisting of three polypeptides which also contains molybdenum, a molybdopterin guanine dinucleotide and iron-sulfur clusters. This protein complex catalyzes the reversible conversion of CO2 and methanofuran to formylmethanofuran during methanogenesis." [GOC:mengo_curators, PMID:1915887, PMID:8954165]
GO:0044677
name: methyl-tetrahydromethanopterin:coenzyme M methyltransferase complex
def: "A protein complex consisted of eight polypeptides. This complex catalyzes the formation of methyl-coenzyme M and H4MPT from N5-methyl-H4MPT and CoM during methanogenesis." [GOC:mengo_curators, PMID:8477726]
GO:0044678
name: CoB-CoM heterodisulfide reductase complex
def: "A protein complex that in Methanobacterium thermoautotrophicum is composed of six subunits, and in Methanosarcina barkeri contains is composed of either two subunits or nine subunits. Catalyzes the conversion of coenzyme B, coenzyme M, and methanophenazine to form N-{7-[(2-sulfoethyl)dithio]heptanoyl}-3-O-phospho-L-threonine and dihydromethanophenazine." [GOC:mengo_curators, PMID:8119281, PMID:8174566, PMID:9063468]
GO:0044679
name: methanophenazine reducing hydrogenase complex
def: "A protein complex which catalyzes the conversion of methanophenazine and hydrogen to form dihydromethanophenazine. This typically consists of three polypeptides" [GOC:mengo_curators, PMID:9555882]
GO:0044680
name: methylthiol:coenzyme M methyltransferase complex
def: "A protein complex of two polypeptides which catalyzes the transfer of methyl group from methylthiol to coenzyme M during methanogenesis." [GOC:mengo_curators, PMID:11073950, PMID:9371433]
GO:0044681
name: sulfopyruvate decarboxylase complex
def: "A complex of two polypeptides which form a dodecamer (A6B6). Catalyzes the decarboxylation of sulfopyruvic acid to sulfoacetaldehyde. This reaction is involved in coenzyme M biosynthesis." [GOC:mengo_curators, PMID:10940029]
GO:0044695
name: Dsc E3 ubiquitin ligase complex
def: "An E3 ubiquitin ligase complex localized to the ER and Golgi membrane. In fission yeast comprises Dsc1, 2, 3 and 4. Involved in the processes of fission yeast sre1 (human SREBP) transcriptional activator proteolytic cleavage, the multivesicular body (MVB) pathway, and a post-endoplasmic reticulum pathway for protein catabolism." [GOC:vw, PMID:21504829]
GO:0044697
name: HICS complex
def: "A multisubunit complex involved in cytokinesis. In the yeast Saccharomyces cerevisiae this complex consists of Sho1p, Hof1p, Inn1p and Cyk3p proteins." [PMID:22623719]
GO:0044731
name: Ost-alpha/Ost-beta complex
def: "A heterodimeric protein complex composed of Ost-alpha/SLC51A and Ost-beta/SLC51B subunits and involved in bile acid transport activity." [PMID:17650074, PMID:22535958]
GO:0044732
name: mitotic spindle pole body
def: "The microtubule organizing center on a spindle that forms as part of mitosis; functionally homologous to the animal cell centrosome." [GOC:vw]
GO:0044753
name: amphisome
def: "An autophagic vacuole formed upon fusion between autophagosomes and endosomes." [GOC:sart, PMID:19008921, PMID:9705327]
GO:0044754
name: autolysosome
def: "A type of secondary lysosome in which a primary lysosome has fused with the outer membrane of an autophagosome. It is involved in the second step of autophagy in which it degrades contents with acidic lysosomal hydrolases." [GOC:sart, NIF_Subcellular:sao8444068431, PMID:19008921]
GO:0044775
name: DNA polymerase III, beta sliding clamp processivity factor complex
def: "A subcomplex of the DNA polymerase III holoenzyme which is responsible for tethering the catalytic subunit of DNA polymerase to DNA during high-speed replication. The complex is homodimeric in prokaryotes, and homotrimeric in other species." [GOC:jl, UniProt:O73947]
GO:0044776
name: DNA polymerase III, core complex
def: "The DNA polymerase III core complex consists of the alpha,epsilon and theta subunits and is carries out the polymerase and the 3'-5' exonuclease proofreading activities." [GOC:jl, UniProt:P06710]
GO:0044777
name: single-stranded DNA-binding protein complex
def: "A homotetrameric protein complex that is essential for DNA replication. It supercoils the single-stranded DNA preventing DNA duplexing before the polymerase holoenzyme passes and synthesizes the complementary strand. It is also involved in DNA recombination and repair." [GOC:jl, UniProt:P0AGE0]
GO:0044796
name: DNA polymerase processivity factor complex
def: "A protein complex which is capable of increasing the processivity of nucleotide polymerization by DNA polymerase as a part of DNA replication." [GOC:bhm, GOC:jl]
GO:0044797
name: cytoplasmic transcription factor complex
def: "A protein complex, located in the cytoplasm, that is capable of associating with DNA by direct binding, or via other DNA-binding proteins or complexes, and regulating transcription." [GOC:jl]
GO:0044798
name: nuclear transcription factor complex
def: "A protein complex, located in the nucleus, that is capable of associating with DNA by direct binding, or via other DNA-binding proteins or complexes, and regulating transcription." [GOC:jl]
GO:0044799
name: NarGHI complex
def: "A heterotrimeric protein complex with iron-sulfur and molybdenum cofactors that functions as a terminal reductase in electron transport pathways that operate during anaerobic nitrate respiration. In E. coli electrons are passed from the FdnGHI complex to the NarGHI complex via menoquinone and menaquinol. Within NarGHI, electrons are passed from the two heme molecules in the NarI subunit down a Fe-S cluster chain in the NarH and NarG subunits to the Molybdenum cofactor, Mo-bisMGD, in the NarG subunit." [GOC:bhm, PMID:11289299, PMID:12910261, PMID:17964535]
GO:0044815
name: DNA packaging complex
def: "A protein complex that plays a role in the process of DNA packaging." [GOC:jl]
GO:0044816
name: Nsk1-Dlc1 complex
def: "A dimer of Nsk1 (nucleolus spindle kinetochore 1) and the dynein light chain, Dlc1. The dimers form an oligomeric chain structure. Functions in the regulation of kinetochore-microtubule interactions and chromosome segregation." [GOC:vw, PMID:22065639]
GO:0044834
name: retroviral intasome
def: "A tetramer of retroviral integrase subunits tightly associated with a pair of viral DNA ends. Functions to insert viral DNA into a host cell chromosome." [PMID:20118915]
GO:0044840
name: gut granule
def: "A lysosome-related organelle contained within the intestinal cells of the nematode C. elegans. Gut granules are acidified, birefringent, autofluorescent, and contain the vacuolar H+-ATPase. They also serve as sites of cellular zinc storage." [GOC:kmv, PMID:22916203, PMID:24204312]
GO:0044841
name: gut granule membrane
def: "The lumen of a gut granule, a lysosome-related organelle contained within the intestinal cells of the nematode C. elegans." [GOC:kmv, PMID:22916203, PMID:24204312]
GO:0044842
name: gut granule lumen
def: "The lumen of a gut granule, a lysosome-related organelle contained within the intestinal cells of the nematode C. elegans." [GOC:kmv, PMID:22916203, PMID:24204312]
GO:0044853
name: plasma membrane raft
def: "A membrane raft that is part of the plasma membrane." [GOC:jl]
GO:0045009
name: chitosome
def: "An intracellular membrane-bounded particle found in fungi and containing chitin synthase; it synthesizes chitin microfibrils. Chitin synthase activity exists in chitosomes and they are proposed to act as a reservoir for regulated transport of chitin synthase enzymes to the division septum." [ISBN:0198506732, PMID:8970154]
GO:0045025
name: mitochondrial degradosome
def: "A mitochondrial protein complex with 3' to 5' exoribonuclease activity that participates in intron-independent turnover and processing of mitochondrial transcripts. In humans, the mitochondrial degradosome is a pentameric complex, and in yeast it exists as a heterodimer." [PMID:10397341, PMID:9829834]
GO:0045092
name: interleukin-18 receptor complex
def: "A protein complex that binds interleukin-18; comprises an alpha and a beta subunit." [GOC:mah, PMID:12759435]
GO:0045095
name: keratin filament
def: "A filament composed of acidic and basic keratins (types I and II), typically expressed in epithelial cells. The keratins are the most diverse classes of IF proteins, with a large number of keratin isoforms being expressed. Each type of epithelium always expresses a characteristic combination of type I and type II keratins." [ISBN:0716731363]
GO:0045096
name: acidic keratin
def: "OBSOLETE. A type of intermediate filament." [ISBN:0716731363, ISBN:0815316194]
GO:0045097
name: basic/neutral keratin
def: "OBSOLETE. A type of intermediate filament." [ISBN:0716731363, ISBN:0815316194]
GO:0045098
name: type III intermediate filament
def: "A type of intermediate filament, typically made up of one or more of the proteins vimentin, desmin, glial fibrillary acidic protein (GFAP), and peripherin. Unlike the keratins, the type III proteins can form both homo- and heteropolymeric IF filaments." [ISBN:0716731363]
GO:0045099
name: vimentin
def: "OBSOLETE. A type of intermediate filament." [ISBN:0716731363]
GO:0045100
name: desmin
def: "OBSOLETE. A type of intermediate filament." [ISBN:0815316194]
GO:0045101
name: glial fibrillary acidic protein
def: "OBSOLETE. Glial fibrillary acidic protein forms filaments in the glial cells that surround neurons and in astrocytes." [ISBN:0716731363]
GO:0045102
name: peripherin
def: "OBSOLETE. Peripherin is a type III intermediate filament protein found in neurons of the peripheral nervous system." [ISBN:0716731363]
GO:0045111
name: intermediate filament cytoskeleton
def: "Cytoskeletal structure made from intermediate filaments, typically organized in the cytosol as an extended system that stretches from the nuclear envelope to the plasma membrane. Some intermediate filaments run parallel to the cell surface, while others traverse the cytosol; together they form an internal framework that helps support the shape and resilience of the cell." [ISBN:0716731363]
GO:0045120
name: pronucleus
def: "The nucleus of either the ovum or the spermatozoon following fertilization. Thus, in the fertilized ovum, there are two pronuclei, one originating from the ovum, the other from the spermatozoon that brought about fertilization; they approach each other, but do not fuse until just before the first cleavage, when each pronucleus loses its membrane to release its contents." [ISBN:0198506732]
GO:0045121
name: membrane raft
def: "Any of the small (10-200 nm), heterogeneous, highly dynamic, sterol- and sphingolipid-enriched membrane domains that compartmentalize cellular processes. Small rafts can sometimes be stabilized to form larger platforms through protein-protein and protein-lipid interactions." [PMID:16645198]
GO:0045160
name: myosin I complex
def: "A myosin complex containing a class I myosin heavy chain and associated light chains; myosin I heavy chains are single-headed, possess tails of various lengths, and do not self-associate into bipolar filaments; myosin I complexes are involved in diverse processes related to membrane traffic and cell movement." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html, PMID:9438839]
GO:0045169
name: fusome
def: "A large intracellular spectrin-rich structure that has been found in insect germline cells and mammalian hematopoietic cells. The fusome is an elongated, branched structure, formed from the spherical spectrosome organelle." [GOC:bf, PMID:12655376]
GO:0045170
name: spectrosome
def: "A germline specific spherical organelle, rich in membrane skeletal proteins. Precursor to the fusome." [GOC:bf]
GO:0045171
name: intercellular bridge
def: "A direct connection between the cytoplasm of two cells that is formed following the completion of cleavage furrow ingression during cell division. They are usually present only briefly prior to completion of cytokinesis. However, in some cases, such as the bridges between germ cells during their development, they become stabilised." [PMID:9635420]
GO:0045172
name: germline ring canal
def: "Germline specific intercellular bridge. During cyst formation in insects, ring canals interconnect the cells of the cyst, facilitating the passage of cytoplasmic components between cells." [GOC:mtg_sensu, PMID:9635420, PMID:9655801]
GO:0045177
name: apical part of cell
def: "The region of a polarized cell that forms a tip or is distal to a base. For example, in a polarized epithelial cell, the apical region has an exposed surface and lies opposite to the basal lamina that separates the epithelium from other tissue." [GOC:mah, ISBN:0815316194]
GO:0045178
name: basal part of cell
def: "The region of a cell situated near the base. For example, in a polarized epithelial cell, the basal surface rests on the basal lamina that separates the epithelium from other tissue." [GOC:mah, ISBN:0185316194]
GO:0045179
name: apical cortex
def: "The region that lies just beneath the plasma membrane on the apical edge of a cell." [GOC:bf]
GO:0045180
name: basal cortex
def: "The region that lies just beneath the plasma membrane on the basal edge of a cell." [GOC:bf]
GO:0045202
name: synapse
def: "The junction between a nerve fiber of one neuron and another neuron or muscle fiber or glial cell; the site of interneuronal communication. As the nerve fiber approaches the synapse it enlarges into a specialized structure, the presynaptic nerve ending, which contains mitochondria and synaptic vesicles. At the tip of the nerve ending is the presynaptic membrane; facing it, and separated from it by a minute cleft (the synaptic cleft) is a specialized area of membrane on the receiving cell, known as the postsynaptic membrane. In response to the arrival of nerve impulses, the presynaptic nerve ending secretes molecules of neurotransmitters into the synaptic cleft. These diffuse across the cleft and transmit the signal to the postsynaptic membrane." [ISBN:0198506732]
GO:0045203
name: integral component of cell outer membrane
def: "The component of the cell outer membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:go_curators, GOC:mtg_sensu]
GO:0045211
name: postsynaptic membrane
def: "A specialized area of membrane facing the presynaptic membrane on the tip of the nerve ending and separated from it by a minute cleft (the synaptic cleft). Neurotransmitters across the synaptic cleft and transmit the signal to the postsynaptic membrane." [ISBN:0198506732]
GO:0045239
name: tricarboxylic acid cycle enzyme complex
def: "Any of the heteromeric enzymes that act in the TCA cycle." [GOC:mah]
GO:0045240
name: dihydrolipoyl dehydrogenase complex
def: "A protein complex that possesses alpha-ketoglutarate dehydrogenase activity." [GOC:mah]
GO:0045241
name: cytosolic alpha-ketoglutarate dehydrogenase complex
def: "Cytosolic complex that possesses alpha-ketoglutarate dehydrogenase activity." [GOC:mah, GOC:mtg_sensu]
GO:0045242
name: isocitrate dehydrogenase complex (NAD+)
def: "Complex that possesses isocitrate dehydrogenase (NAD+) activity." [GOC:mah]
GO:0045243
name: cytosolic isocitrate dehydrogenase complex (NAD+)
def: "Cytosolic complex that possesses isocitrate dehydrogenase (NAD+) activity." [GOC:mah, GOC:mtg_sensu]
GO:0045244
name: succinate-CoA ligase complex (GDP-forming)
def: "A heterodimeric enzyme complex, usually composed of an alpha and beta chain. Functions in the TCA cycle, hydrolyzing succinyl-CoA into succinate and CoA, thereby forming GTP." [EC:6.2.1.4, GOC:jl]
GO:0045246
name: cytosolic tricarboxylic acid cycle enzyme complex
def: "Any of the heteromeric enzymes, located in the cytosol, that act in the tricarboxylic acid (TCA) cycle." [GOC:mah, GOC:mtg_sensu]
GO:0045247
name: cytosolic electron transfer flavoprotein complex
def: "A protein complex located in the cytosol containing flavin adenine dinucleotide (FAD) that, together with an acyl-CoA dehydrogenase, forms a system that oxidizes an acyl-CoA molecule and reduces ubiquinone and other acceptors." [GOC:mtg_sensu, ISBN:0198506732]
GO:0045248
name: cytosolic oxoglutarate dehydrogenase complex
def: "A cytosolic complex of multiple copies of three enzymatic components: oxoglutarate dehydrogenase (lipoamide) ; EC:1.2.4.2 (E1), dihydrolipoamide S-succinyltransferase ; EC:2.3.1.61 (E2) and dihydrolipoamide dehydrogenase ; EC:1.8.1.4 (E3); catalyzes the overall conversion of 2-oxoglutarate to succinyl-CoA and carbon dioxide (CO2)." [GOC:mtg_sensu, PMID:10848975]
GO:0045249
name: cytosol pyruvate dehydrogenase (lipoamide) phosphatase complex
def: "A cytosolic complex of a regulatory and catalytic subunit that catalyzes the dephosphorylation and concomitant reactivation of the alpha subunit of the E1 component of the pyruvate dehydrogenase complex." [GOC:mtg_sensu, PMID:9395502]
GO:0045250
name: cytosolic pyruvate dehydrogenase complex
def: "Complex that carries out the oxidative decarboxylation of pyruvate to form acetyl-CoA; comprises subunits possessing three catalytic activities: pyruvate dehydrogenase (E1), dihydrolipoamide S-acetyltransferase (E2), and dihydrolipoamide dehydrogenase (E3). Usually contains fewer subunits than its eukaryotic counterpart; for example, the E. coli complex contains 12 E1 dimers, 8 E2 trimers, and 6 E3 dimers arranged in highly symmetric cubic order." [GOC:mtg_sensu, ISBN:0471331309, ISBN:0716720094]
GO:0045251
name: electron transfer flavoprotein complex
def: "A protein complex containing flavin adenine dinucleotide (FAD) and acyl-CoA dehydrogenase, which form a system that oxidizes an acyl-CoA molecule and reduces ubiquinone and other acceptors in the mitochondrial electron transport system." [ISBN:0198506732]
GO:0045252
name: oxoglutarate dehydrogenase complex
def: "A complex of multiple copies of three enzymatic components: oxoglutarate dehydrogenase (lipoamide) ; EC:1.2.4.2 (E1), dihydrolipoamide S-succinyltransferase ; EC:2.3.1.61 (E2) and dihydrolipoamide dehydrogenase ; EC:1.8.1.4 (E3); catalyzes the overall conversion of 2-oxoglutarate to succinyl-CoA and carbon dioxide (CO2)." [MetaCyc:CPLX66-42, PMID:10848975]
GO:0045253
name: pyruvate dehydrogenase (lipoamide) phosphatase complex
def: "A complex of a regulatory and catalytic subunit that catalyzes the dephosphorylation and concomitant reactivation of the alpha subunit of the E1 component of the pyruvate dehydrogenase complex." [PMID:9395502]
GO:0045254
name: pyruvate dehydrogenase complex
def: "Complex that carries out the oxidative decarboxylation of pyruvate to form acetyl-CoA; comprises subunits possessing three catalytic activities: pyruvate dehydrogenase (E1), dihydrolipoamide S-acetyltransferase (E2), and dihydrolipoamide dehydrogenase (E3)." [ISBN:0716720094]
GO:0045257
name: succinate dehydrogenase complex (ubiquinone)
def: "The enzyme that catalyzes the oxidation of succinate and ubiquinone to fumarate and ubiquinol; involved in aerobic respiration, repressed in anaerobic respiration." [GOC:kd, ISBN:0198547684]
GO:0045258
name: plasma membrane succinate dehydrogenase complex (ubiquinone)
def: "The enzyme, located in the plasma membrane, that catalyzes the oxidation of succinate and ubiquinone to fumarate and ubiquinol; involved in aerobic respiration, repressed in anaerobic respiration." [GOC:kd, GOC:mtg_sensu, ISBN:0198547684]
GO:0045259
name: proton-transporting ATP synthase complex
def: "A proton-transporting two-sector ATPase complex that catalyzes the phosphorylation of ADP to ATP during oxidative phosphorylation. The complex comprises a membrane sector (F0) that carries out proton transport and a cytoplasmic compartment sector (F1) that catalyzes ATP synthesis by a rotational mechanism; the extramembrane sector (containing 3 a and 3 b subunits) is connected via the d-subunit to the membrane sector by several smaller subunits. Within this complex, the g and e subunits and the 9-12 c subunits rotate by consecutive 120 degree angles and perform parts of ATP synthesis. This movement is driven by the hydrogen ion electrochemical potential gradient." [EC:3.6.3.14, ISBN:0198547684, ISBN:0716743663]
GO:0045260
name: plasma membrane proton-transporting ATP synthase complex
def: "A proton-transporting ATP synthase complex found in the plasma membrane. Examples of this component are found in Bacterial species." [GOC:mah, GOC:mtg_sensu, ISBN:0198547684]
GO:0045261
name: proton-transporting ATP synthase complex, catalytic core F(1)
def: "The sector of a hydrogen-transporting ATP synthase complex in which the catalytic activity resides; it comprises the catalytic core and central stalk, and is peripherally associated with a membrane, such as the plasma membrane or the mitochondrial inner membrane, when the entire ATP synthase is assembled." [GOC:mah, PMID:10838056]
GO:0045262
name: plasma membrane proton-transporting ATP synthase complex, catalytic core F(1)
def: "The catalytic sector of the plasma membrane hydrogen-transporting ATP synthase; it comprises the catalytic core and central stalk, and is peripherally associated with the plasma membrane when the entire ATP synthase is assembled. Examples of this component are found in Bacterial species." [GOC:mah, GOC:mtg_sensu, PMID:10838056]
GO:0045263
name: proton-transporting ATP synthase complex, coupling factor F(o)
def: "All non-F1 subunits of a hydrogen-transporting ATP synthase, including integral and peripheral membrane proteins." [PMID:10838056]
GO:0045264
name: plasma membrane proton-transporting ATP synthase complex, coupling factor F(o)
def: "All non-F1 subunits of the plasma membrane hydrogen-transporting ATP synthase, including integral and peripheral plasma membrane proteins." [GOC:mah, GOC:mtg_sensu, PMID:10838056]
GO:0045265
name: proton-transporting ATP synthase, stator stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the membrane-associated Fo proteins; is thought to prevent futile rotation of the catalytic core." [PMID:10838056]
GO:0045266
name: plasma membrane proton-transporting ATP synthase, stator stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the plasma membrane-associated F0 proteins; is thought to prevent futile rotation of the catalytic core. Examples of this component are found in Bacterial species." [GOC:mtg_sensu, PMID:10838056]
GO:0045267
name: proton-transporting ATP synthase, catalytic core
def: "The hexamer that possesses the catalytic activity of the mitochondrial hydrogen-transporting ATP synthase." [PMID:10838056]
GO:0045268
name: plasma membrane proton-transporting ATP synthase, catalytic core
def: "The hexamer that possesses the catalytic activity of the plasma membrane hydrogen-transporting ATP synthase. Examples of this component are found in Bacterial species." [GOC:mtg_sensu, PMID:10838056]
GO:0045269
name: proton-transporting ATP synthase, central stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the membrane-associated Fo proteins; rotates within the catalytic core during catalysis." [PMID:10838056]
GO:0045270
name: plasma membrane proton-transporting ATP synthase, central stalk
def: "One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the plasma membrane-associated Fo proteins; rotates within the catalytic core during catalysis. Examples of this component are found in Bacterial species." [GOC:mtg_sensu, PMID:10838056]
GO:0045271
name: respiratory chain complex I
def: "Respiratory chain complex I is an enzyme of the respiratory chain. It consists of several polypeptide chains and is L-shaped, with a horizontal arm lying in the membrane and a vertical arm that projects into the matrix. The electrons of NADH enter the chain at this complex." [GOC:imk, GOC:jid, ISBN:0716749556]
GO:0045272
name: plasma membrane respiratory chain complex I
def: "A subcomplex of the respiratory chain located in the plasma membrane. It contains about 25 different polypeptide subunits, including NADH dehydrogenase (ubiquinone), flavin mononucleotide and several different iron-sulfur clusters containing non-heme iron. The iron undergoes oxidation-reduction between Fe(II) and Fe(III), and catalyzes proton translocation linked to the oxidation of NADH by ubiquinone. Examples of this component are found in bacterial species." [GOC:mtg_sensu, ISBN:0198547684]
GO:0045273
name: respiratory chain complex II
def: "A part of the respiratory chain, containing the four polypeptide subunits of succinate dehydrogenase, flavin-adenine dinucleotide and iron-sulfur. Catalyzes the oxidation of succinate by ubiquinone. Connects the TCA cycle with the respiratory chain." [ISBN:0198547684]
GO:0045274
name: plasma membrane respiratory chain complex II
def: "A part of the respiratory chain located in the plasma membrane, containing the four polypeptide subunits of succinate dehydrogenase, flavin-adenine dinucleotide and iron-sulfur. Catalyzes the oxidation of succinate by ubiquinone. Connects the TCA cycle with the respiratory chain. Examples of this component are found in bacterial species." [GOC:mtg_sensu, ISBN:0198547684]
GO:0045275
name: respiratory chain complex III
def: "A protein complex that transfers electrons from ubiquinol to cytochrome c and translocates two protons across a membrane. The complex contains a core structure of three catalytic subunits: cytochrome b, the Rieske iron sulfur protein (ISP), and cytochrome c1, which are arranged in an integral membrane-bound dimeric complex; additional subunits are present, and vary among different species." [PMID:16228398, PMID:16352458, PMID:17200733]
GO:0045276
name: plasma membrane respiratory chain complex III
def: "A part of the respiratory chain located in the plasma membrane, containing about 10 polypeptide subunits including four redox centers: cytochrome b/b6, cytochrome c1 and an 2Fe-2S cluster. Catalyzes the oxidation of ubiquinol by oxidized cytochrome c1. Examples of this component are found in bacterial species." [GOC:mtg_sensu, ISBN:0198547684]
GO:0045277
name: respiratory chain complex IV
def: "A part of the respiratory chain, containing the 13 polypeptide subunits of cytochrome c oxidase, including cytochrome a and cytochrome a3. Catalyzes the oxidation of reduced cytochrome c by dioxygen (O2)." [ISBN:0198547684]
GO:0045278
name: plasma membrane respiratory chain complex IV
def: "A part of the respiratory chain located in the plasma membrane, containing the 13 polypeptide subunits of cytochrome c oxidase, including cytochrome a and cytochrome a3. Catalyzes the oxidation of reduced cytochrome c by dioxygen (O2). Examples of this component are found in bacterial species." [GOC:mtg_sensu, ISBN:0198547684]
GO:0045281
name: succinate dehydrogenase complex
def: "A multimeric complex which consists of flavoprotein (subunit A ; InterPro:IPR003952), iron-sulfur protein (subunit B) and membrane-bound cytochrome b560 (subunit C; InterPro:IPR000701). In some Archaea, the membrane-bound subunits (C or C and D) do not necessarily contain heme. Membrane-bound subunits can bind or react with quinones." [GOC:kd, InterPro:IPR000701]
GO:0045282
name: plasma membrane succinate dehydrogenase complex
def: "A multimeric complex which consists of flavoprotein (subunit A ; InterPro:IPR003952), iron-sulfur protein (subunit B) and membrane-bound cytochrome b560 (subunit C; InterPro:IPR000701). In some Archaea, the membrane-bound subunits (C or C and D) do not necessarily contain heme. Membrane-bound subunits can bind/react with quinones. Examples of this component are found in Bacterial species." [GOC:kd, GOC:mtg_sensu, InterPro:IPR000701]
GO:0045283
name: fumarate reductase complex
def: "A membrane-bound flavoenzyme complex consisting of four subunits, A, B, C, and D. A and B comprise the membrane-extrinsic catalytic domain and C (InterPro:IPR003510; InterPro:IPR004224) and D (InterPro:IPR003418) link the catalytic centers to the electron-transport chain. This family consists of the 13 kDa hydrophobic subunit D. This component may be required to anchor the catalytic components of the fumarate reductase complex to the cytoplasmic membrane. Fumarate reductase couples the reduction of fumarate to succinate to the oxidation of quinol to quinone, in a reaction opposite to that catalyzed by the related complex II of the respiratory chain (succinate dehydrogenase-(ubiquinone))." [InterPro:IPR003418, InterPro:IPR004224]
GO:0045284
name: plasma membrane fumarate reductase complex
def: "A membrane-bound flavoenzyme complex consisting of four subunits, A, B, C, and D. A and B comprise the membrane-extrinsic catalytic domain and C (InterPro:IPR003510; InterPro:IPR00224) and D (InterPro:IPR003418) link the catalytic centers to the electron-transport chain. In some species, the complex has only three subunits, and in these cases, there is only one membrane anchor instead of two. This family consists of the 13 kDa hydrophobic subunit D. This component may be required to anchor the catalytic components of the fumarate reductase complex to the cytoplasmic membrane. Fumarate reductase couples the reduction of fumarate to succinate to the oxidation of quinol to quinone, in a reaction opposite to that catalyzed by the related complex II of the respiratory chain (succinate dehydrogenase-(ubiquinone)). Examples of this component are found in bacterial species." [GOC:mtg_sensu, InterPro:IPR003418, InterPro:IPR004224]
GO:0045293
name: mRNA editing complex
def: "A protein complex that posttranscriptionally catalyzes insertion, deletion or substitution of nucleotides at multiple sites within nascent mRNA transcripts to produce mature mRNAs in eukaryotes." [http://www.ejbiotechnology.info/content/vol1/issue1/full/4/, PMID:11564867, PMID:12139607]
GO:0045298
name: tubulin complex
def: "A heterodimer of tubulins alpha and beta that constitutes the protomer for microtubule assembly." [ISBN:0716731363]
GO:0045320
name: chloroplast proton-transporting ATP synthase complex
def: "A proton-transporting ATP synthase complex found in the chloroplast thylakoid membrane; it catalyzes the phosphorylation of ADP to ATP during photo-phosphorylation." [GOC:mtg_sensu, GOC:pj, ISBN:0716743663]
GO:0045323
name: interleukin-1 receptor complex
def: "A protein complex that binds interleukin-1; comprises an alpha and a beta subunit." [GOC:mah, InterPro:IPR004075]
GO:0045334
name: clathrin-coated endocytic vesicle
def: "A clathrin-coated, membrane-bounded intracellular vesicle formed by invagination of the plasma membrane around an extracellular substance." [GOC:go_curators]
GO:0045335
name: phagocytic vesicle
def: "A membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:go_curators, ISBN:0198506732]
GO:0045336
name: clathrin-coated phagocytic vesicle
def: "A clathrin-coated, membrane-bounded intracellular vesicle that arises from the ingestion of particulate material by phagocytosis." [GOC:go_curators, ISBN:0198506732]
GO:0045495
name: pole plasm
def: "Differentiated cytoplasm associated with a pole (animal, vegetal, anterior, or posterior) of an oocyte, egg or early embryo." [GOC:kmv, PMID:17113380]
GO:0046536
name: dosage compensation complex
def: "A protein or protein-RNA complex that localizes to one or more of the sex chromosome(s), where it acts to normalize transcription between different sexes." [GOC:kmv, GOC:mah]
GO:0046540
name: U4/U6 x U5 tri-snRNP complex
def: "A ribonucleoprotein complex formed by the association of the U4/U6 and U5 small nuclear ribonucleoproteins." [GOC:krc, GOC:pr, PMID:11867543]
GO:0046581
name: intercellular canaliculus
def: "An extremely narrow tubular channel located between adjacent cells. An instance of this is the secretory canaliculi occurring between adjacent parietal cells in the gastric mucosa of vertebrates." [ISBN:0721662544]
GO:0046610
name: lysosomal proton-transporting V-type ATPase, V0 domain
def: "The V0 domain of a proton-transporting V-type ATPase found in the lysosomal membrane." [GOC:mah]
GO:0046611
name: lysosomal proton-transporting V-type ATPase complex
def: "A proton-transporting two-sector ATPase complex found in the lysosomal membrane, where it acts as a proton pump to mediate acidification of the lysosomal lumen." [GOC:mah, ISBN:0716743663, PMID:16449553]
GO:0046612
name: lysosomal proton-transporting V-type ATPase, V1 domain
def: "The V1 domain of a proton-transporting V-type ATPase found in the lysosomal membrane." [GOC:mah]
GO:0046658
name: anchored component of plasma membrane
def: "The component of the plasma membrane consisting of the gene products that are tethered to the membrane only by a covalently attached anchor, such as a lipid group, that is embedded in the membrane. Gene products with peptide sequences that are embedded in the membrane are excluded from this grouping." [GOC:dos, GOC:mah]
GO:0046691
name: intracellular canaliculus
def: "An apical plasma membrane part that forms a narrow enfolded luminal membrane channel, lined with numerous microvilli, that appears to extend into the cytoplasm of the cell. A specialized network of intracellular canaliculi is a characteristic feature of parietal cells of the gastric mucosa in vertebrates." [GOC:mah, ISBN:0721662544, PMID:10700045]
GO:0046695
name: SLIK (SAGA-like) complex
def: "A SAGA-type histone acetyltransferase complex that contains Rtg2 and a smaller form of Spt7 than the fungal SAGA complex, and lacks Spt8. The complex is involved in the yeast retrograde response pathway, which is important for gene expression changes during mitochondrial dysfunction." [PMID:12101232, PMID:12186975, PMID:17337012]
GO:0046696
name: lipopolysaccharide receptor complex
def: "A multiprotein complex that consists of at least three proteins, CD14, TLR4, and MD-2, each of which is glycosylated and which functions as a lipopolysaccharide (LPS) receptor that primes the innate immune response against bacterial pathogens." [PMID:11706042, PMID:9665271]
GO:0046727
name: capsomere
def: "Any of the protein subunits that comprise the closed shell or coat (capsid) of certain viruses." [ISBN:0198506732]
GO:0046729
name: viral procapsid
def: "A stable empty viral capsid produced during the assembly of viruses." [ISBN:0072370319, ISBN:1555811272]
GO:0046798
name: viral portal complex
def: "A multimeric ring of proteins through which the DNA enters and exits the viral capsid." [PMID:11602732]
GO:0046806
name: viral scaffold
def: "A complex of proteins that form a scaffold around which the viral capsid is constructed." [ISBN:0072370319]
GO:0046808
name: assemblon
def: "Antigenically dense structures located at the periphery of nuclei, close to but not abutting nuclear membranes. Assemblons contain the proteins for immature-capsid assembly; they are located at the periphery of a diffuse structure composed of proteins involved in DNA synthesis, which overlaps only minimally with the assemblons. More than one site can be present simultaneously." [PMID:8676489]
GO:0046809
name: replication compartment
def: "Globular nuclear domains where the transcription and replication of the viral genome occurs. More than one site can be present simultaneously." [PMID:9499108, VZ:1951]
GO:0046816
name: virion transport vesicle
def: "A vesicle used to transport the partial or complete virion between cellular compartments." [PMID:7933124]
GO:0046818
name: dense nuclear body
def: "A location in the host cell nucleus where viral proteins colocalize late in infection prior to the onset of viral DNA synthesis. More than one site can be present simultaneously." [PMID:10233976]
GO:0046821
name: extrachromosomal DNA
def: "DNA structures that are not part of a chromosome." [GOC:ai]
GO:0046858
name: chlorosome
def: "A large enclosure of aggregated pigment, typically bacteriochlorophyll c (BChl c), that acts as a light-harvesting antenna structure and is characteristic of green photosynthetic bacteria (e.g. Chlorobiaceae). The BChl aggregates are organized into lamellar elements by pigment-pigment rather than pigment-protein interactions. Chlorosomes also contain BChl a, carotenoids, quinones, lipids, and proteins, and are attached to the cytoplasmic membrane via a BChl a-containing protein baseplate." [ISBN:0198506732, PMID:14729689, PMID:15298919]
GO:0046859
name: hydrogenosomal membrane
def: "The lipid bilayer surrounding a hydrogenosome." [GOC:ai]
GO:0046860
name: glycosome membrane
def: "The lipid bilayer surrounding a glycosome." [GOC:ai]
GO:0046861
name: glyoxysomal membrane
def: "The lipid bilayer surrounding a glyoxysome." [GOC:ai]
GO:0046862
name: chromoplast membrane
def: "Either of the lipid bilayers that surround a chromoplast and form the chromoplast envelope." [GOC:ai, GOC:mah]
GO:0046868
name: mesosome
def: "An intracellular, often complex, membranous structure, sometimes with additional membranous lamellae inside, found in bacteria. They are associated with synthesis of DNA and secretion of proteins." [ISBN:0198506732, ISBN:0716731363]
GO:0046930
name: pore complex
def: "Any small opening in a membrane that allows the passage of gases and/or liquids." [ISBN:0198506732]
GO:0048046
name: apoplast
def: "The cell membranes and intracellular regions in a plant are connected through plasmodesmata, and plants may be described as having two major compartments: the living symplast and the non-living apoplast. The apoplast is external to the plasma membrane and includes cell walls, intercellular spaces and the lumen of dead structures such as xylem vessels. Water and solutes pass freely through it." [GOC:jid]
GO:0048179
name: activin receptor complex
def: "A protein complex that acts as an activin receptor. Heterodimeric activin receptors, comprising one Type I activin receptor and one Type II receptor polypeptide, and heterotrimeric receptors have been observed." [PMID:8307945, PMID:8622651]
GO:0048180
name: activin complex
def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits, inhibin beta-A and inhibin beta-B (sometimes known as activin beta or activin/inhibin beta). There are three forms of activin complex, activin A, which is composed of 2 inhibin beta-A subunits, activin B, which is composed of 2 inhibin beta-B subunits, and activin AB, which is composed of an inhibin beta-A and an inhibin beta-B subunit." [GOC:go_curators, http://www.mercksource.com, http://www.stedmans.com/]
GO:0048183
name: activin AB complex
def: "A nonsteroidal regulator, composed of two covalently linked inhibin beta subunits (sometimes known as activin beta or activin/inhibin beta), inhibin beta-A and inhibin beta-B." [GOC:go_curators, http://www.mercksource.com, http://www.stedmans.com/]
GO:0048188
name: Set1C/COMPASS complex
def: "A conserved protein complex that catalyzes methylation of histone H3. In Saccharomyces the complex contains Shg1p, Sdc1p, Swd1p, Swd2p, Swd3p, Spp1p, Bre2p, and the trithorax-related Set1p; in mammals it contains the catalytic subunit (SETD1A or SETD1B), WDR5, WDR82, RBBP5, ASH2L/ASH2, CXXC1/CFP1, HCFC1 and DPY30." [PMID:11687631, PMID:11742990, PMID:11805083, PMID:12488447, PMID:18508253, PMID:18838538]
GO:0048189
name: Lid2 complex
def: "A protein complex that is thought to be involved in regulation of chromatin remodeling. In Schizosaccharomyces the complex contains Lid1p, Ash2p, Ecm5p, Snt2p, and Sdc1p." [PMID:12488447]
GO:0048196
name: plant extracellular matrix
def: "The matrix external to the plant plasma membrane, composed of the cell wall and middle lamella." [GOC:jid, GOC:mtg_sensu, PMID:11351084, PMID:4327466]
GO:0048217
name: pectic matrix
def: "The gel-like pectin matrix consists of the interlinked acidic and neutral pectin networks that are further cross-linked by calcium bridges. Pectins consist largely of long chains of mostly galacturonic acid units (typically 1,4 linkages and sometimes methyl esters). Three major pectic polysaccharides (homogalacturonan, rhamnogalacturonan I and rhamnogalacturonan II) are thought to occur in all primary cell walls." [GOC:jid, PMID:11554482]
GO:0048222
name: glycoprotein network
def: "An extracellular matrix part that consists of cross-linked glycoproteins." [GOC:mah, PMID:18508691, PMID:7048321]
GO:0048223
name: hemicellulose network
def: "Network composed of hemicelluloses; members of a class of plant cell wall polysaccharide that cannot be extracted from the wall by hot water or chelating agents, but can be extracted by aqueous alkali. Includes xylan, glucuronoxylan, arabinoxylan, arabinogalactan II, glucomannan, xyloglucan and galactomannan." [DOI:10.1016/j.foodchem.2008.11.065, GOC:jid]
GO:0048224
name: lignin network
def: "An extracellular matrix part that consists of lignin in the form of a three-dimensional polymeric network. Lignins are complex racemic aromatic heteropolymers derived from a variety of phenylpropane monomers coupled together by an assortment of carbon-carbon and ether linkages. Lignin is crucial for structural integrity of the cell wall and stiffness and strength of the stem. In addition, lignin waterproofs the cell wall, enabling transport of water and solutes through the vascular system, and plays a role in protecting plants against pathogens." [GOC:jid, GOC:mah, PMID:14503002, PMID:16662709]
GO:0048225
name: suberin network
def: "An extracellular matrix part that consists of fatty acid-derived polymers, including both aromatic and aliphatic components. The suberin network is found in specialized plant cell walls, where it is laid down between the primary wall and plasma membrane, forms protective and wound-healing layers, and provides a water-impermeable diffusion barrier." [GOC:jid, GOC:mah, PMID:18440267, PMID:7706282]
GO:0048226
name: Casparian strip
def: "Region of plant cell wall specialised to act as a seal to prevent back leakage of secreted material (analogous to tight junction between epithelial cells). Found particularly where root parenchymal cells secrete solutes into xylem vessels. The barrier is composed of suberin; a fatty substance, containing long chain fatty acids and fatty esters, also found in the cell walls of cork cells (phellem) in higher plants." [GOC:jid]
GO:0048237
name: rough endoplasmic reticulum lumen
def: "The volume enclosed by the membranes of the rough endoplasmic reticulum." [GOC:jid]
GO:0048238
name: smooth endoplasmic reticulum lumen
def: "The volume enclosed by the membranes of the smooth endoplasmic reticulum." [GOC:jid]
GO:0048269
name: methionine adenosyltransferase complex
def: "A multimeric enzyme complex composed of variable numbers of catalytic alpha subunits, and noncatalytic beta subunits. The beta subunits are believed to have a regulatory function. The enzyme complex catalyzes the synthesis of S-adenosylmethionine (AdoMet), which is the major methyl group donor, participating in the methylation of proteins, DNA, RNA, phospholipids, and other small molecules." [EC:2.5.1.6, GOC:jid, PMID:10644686]
GO:0048353
name: primary endosperm nucleus
def: "Nucleus resulting from the fusion of the male gamete and two polar nuclei in the central cell of the embryo sac." [ISBN:0471245208]
GO:0048471
name: perinuclear region of cytoplasm
def: "Cytoplasm situated near, or occurring around, the nucleus." [GOC:jid]
GO:0048475
name: coated membrane
def: "A single or double lipid bilayer with any of several different proteinaceous coats that can associate with membranes. Membrane coats include those formed by clathrin plus an adaptor complex, the COPI and COPII complexes." [GOC:jid]
GO:0048476
name: Holliday junction resolvase complex
def: "A protein complex that mediates the conversion of a Holliday junction into two separate duplex DNA molecules; the complex includes a single- or multisubunit helicase that catalyzes the extension of heteroduplex DNA by branch migration and a nuclease that resolves the junction by nucleolytic cleavage." [PMID:11207366, PMID:12374758]
GO:0048492
name: ribulose bisphosphate carboxylase complex
def: "A complex containing either both large and small subunits or just small subunits which carries out the activity of producing 3-phosphoglycerate from carbon dioxide and ribulose-1,5-bisphosphate." [GOC:mlg]
GO:0048493
name: plasma membrane-derived thylakoid ribulose bisphosphate carboxylase complex
def: "A complex, located in the plasma membrane-derived thylakoid, containing either both large and small subunits or just small subunits. It carries out the activity of producing 3-phosphoglycerate from carbon dioxide and ribulose-1,5-bisphosphate." [GOC:mlg, GOC:mtg_sensu]
GO:0048494
name: chromatophore ribulose bisphosphate carboxylase complex
def: "A complex, located in the chromatophore, containing either both large and small subunits or just small subunits which carries out the activity of producing 3-phosphoglycerate from carbon dioxide and ribulose-1,5-bisphosphate." [GOC:mlg, GOC:mtg_sensu]
GO:0048500
name: signal recognition particle
def: "A complex of protein and RNA which facilitates translocation of proteins across membranes." [GOC:mlg]
GO:0048501
name: signal recognition particle, plasma membrane targeting
def: "A complex consisting of a protein and RNA component which binds the signal sequence of some proteins and facilitates their export to or across the plasma membrane." [GOC:mlg, GOC:mtg_sensu]
GO:0048555
name: generative cell nucleus
def: "The nucleus of the generative cell, a cell contained within the pollen grain that will divide to produce two haploid sperm cells." [GOC:tair_curators]
GO:0048556
name: microsporocyte nucleus
def: "The nucleus of the microsporocyte. The microsporocyte is a diploid cell in which meiosis will occur, resulting in four microspores. A microspore is a spore that, in vascular plants, gives rise to a male gametophyte." [GOC:tair_curators, ISBN:047186840X]
GO:0048770
name: pigment granule
def: "A small, subcellular membrane-bounded vesicle containing pigment and/or pigment precursor molecules. Pigment granule biogenesis is poorly understood, as pigment granules are derived from multiple sources including the endoplasmic reticulum, coated vesicles, lysosomes, and endosomes." [GOC:jid, GOC:mh]
GO:0048786
name: presynaptic active zone
def: "A specialized region of the plasma membrane and cell cortex of a presynaptic neuron; encompasses a region of the plasma membrane where synaptic vesicles dock and fuse, synaptic vesicles, and a specialized cortical cytoskeletal matrix." [GOC:dh, GOC:dl, GOC:ef, GOC:jid, PMID:3152289]
GO:0048787
name: presynaptic active zone membrane
def: "The membrane portion of the presynaptic active zone; it is the site where docking and fusion of synaptic vesicles occurs for the release of neurotransmitters." [PMID:12812759, PMID:12923177, PMID:3152289]
GO:0048788
name: presynaptic cytoskeletal matrix assembled at active zones
def: "The specialized cytoskeletal matrix assembled at the active zones of the presynaptic nerve terminal. The cytoskeletal matrix is a proteinaceous structure involved in organizing synaptic events such as immobilisation or translocation of synaptic vesicles, and assembling active zone components. The cytomatrix protein exclusively assembled at active zones, is thought to form a molecular scaffold that organizes neurotransmitter release sites." [GOC:dh, GOC:dl, GOC:ef, GOC:jid, NIF_Subcellular:sao1470121605, PMID:10944438]
GO:0051077
name: secondary cell septum
def: "Cell wall structures composed of linear polysaccharides which are deposited at both sides of the primary septum at 90 degrees to the primary septum." [GOC:mtg_sensu, PMID:15194814]
GO:0051233
name: spindle midzone
def: "The area in the center of the spindle where the spindle microtubules from opposite poles overlap." [GOC:ai, PMID:15296749]
GO:0051285
name: cell cortex of cell tip
def: "The region directly beneath the plasma membrane at the cell tip. The cell tip is the region at either end of the longest axis of a cylindrical or elongated cell." [GOC:ai]
GO:0051286
name: cell tip
def: "The region at either end of the longest axis of a cylindrical or elongated cell." [GOC:ai, GOC:mah]
GO:0052718
name: tRNA-specific adenosine-34 deaminase complex
def: "A protein complex that possesses tRNA-specific adenosine-34 deaminase activity. In eukaryotes the complex is a heterodimer; the subunits are known as Tad2p and Tad3p in yeasts and Adat2 and Adat3 in human." [PMID:17875641]
GO:0055028
name: cortical microtubule
def: "Arrays of microtubules underlying and connected to the plasma membrane in the cortical cytosol." [GOC:mtg_sensu]
GO:0055029
name: nuclear DNA-directed RNA polymerase complex
def: "A protein complex, located in the nucleus, that possesses DNA-directed RNA polymerase activity." [GOC:mtg_sensu]
GO:0055031
name: gamma-tubulin large complex, equatorial microtubule organizing center
def: "A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes located at equatorial microtubule organizing centers." [GOC:mtg_sensu]
GO:0055032
name: gamma-tubulin large complex, spindle pole body
def: "A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes located in the spindle pole body." [GOC:mtg_sensu]
GO:0055033
name: gamma-tubulin large complex, interphase microtubule organizing center
def: "A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes located at interphase microtubule organizing centers." [GOC:mtg_sensu]
GO:0055035
name: plastid thylakoid membrane
def: "The lipid bilayer membrane of any thylakoid within a plastid." [GOC:jid, GOC:rph]
GO:0055036
name: virion membrane
def: "The lipid bilayer surrounding a virion." [GOC:jid, GOC:rph, PMID:213106]
GO:0055037
name: recycling endosome
def: "Organelle consisting of networks of 60nm tubules organized around the microtubule organizing centre in some cell types. They transport molecules (e.g., receptors, transporters, lipids) derived from endosomes, the Golgi apparatus, or the cytoplasm to the plasma membrane. Transported molecules may be recycled for reuse, or may be newly synthesized." [GOC:dph, GOC:jid, GOC:kmv, GOC:rph, PMID:10930469, PMID:15601896, PMID:16246101, PMID:21556374, PMID:21562044]
GO:0055038
name: recycling endosome membrane
def: "The lipid bilayer surrounding a recycling endosome." [GOC:jid, GOC:rph, PMID:10930469, PMID:15601896, PMID:16246101]
GO:0055039
name: trichocyst
def: "A crystalline exocytotic organelle composed of small, acidic proteins existing primarily as disulphide-linked dimers. The trichocyst is an organelle that releases long filamentous proteins that capture predators in \"nets\" to slow them down when the cell is disturbed. The protein is nontoxic and shaped like a long, striated, fibrous shaft." [GOC:jid, GOC:rph, http://www.iscid.org/encyclopedia/, PMID:3667715]
GO:0055040
name: periplasmic flagellum
def: "Flagellar filaments located in the periplasmic space; characterized in spirochetes, in which they are essential for shape and motility. Composed of a core surrounded by two sheath layers, the flagella rotate to allow migration of the cell through viscous media, which would not be possible using external flagella." [GOC:jid, GOC:rph, PMID:15175283, PMID:1624463]
GO:0055044
name: symplast
def: "The interconnected cell membranes and intracellular regions of a plant. The interconnections occur via the plasmodesmata." [GOC:mtg_sensu]
GO:0055051
name: ATP-binding cassette (ABC) transporter complex, integrated substrate binding
def: "A complex for the transport of metabolites out of the cell, consisting of 4 domains: two ATP-binding domains and two membrane spanning domains. In some cases, all 4 domains are contained on 1 polypeptide, while in others one ATP-binding domain and one membrane spanning domain are together on one polypeptide in what is called a \"half transporter\". Two \"half-transporters\" come together to form a functional transporter. Transport of the substrate across the membrane is driven by the hydrolysis of ATP." [GOC:mlg, GOC:mtg_sensu]
GO:0055052
name: ATP-binding cassette (ABC) transporter complex, substrate-binding subunit-containing
def: "A complex for the transport of metabolites into the cell, consisting of 5 subunits: two ATP-binding subunits, two membrane spanning subunits, and one substrate-binding subunit. In organisms with two membranes, the substrate-binding protein moves freely in the periplasmic space and joins the other subunits only when bound with substrate. In organisms with only one membrane the substrate-binding protein is tethered to the cytoplasmic membrane and associated with the other subunits. Transport of the substrate across the membrane is driven by the hydrolysis of ATP." [GOC:mlg, GOC:mtg_sensu]
GO:0055087
name: Ski complex
def: "A protein complex that regulates RNA degradation by the exosome complex. In Saccharomyces the complex has a heterotetrameric stoichiometry consisting of one copy each of Ski2p and Ski3 and two copies of Ski8p." [GOC:mcc, PMID:10744028, PMID:15703439, PMID:16043509, PMID:18042677]
GO:0055120
name: striated muscle dense body
def: "A vinculin-containing myofibril attachment structure of striated muscle that connects sarcomeres to the extracellular matrix. In nematode body wall muscle, the dense body performs the dual role of Z-disk and costamere." [GOC:kmv, PMID:17492481]
GO:0055125
name: Nic96 complex
def: "OBSOLETE. A protein complex that forms part of the nuclear pore complex, and is required for its correct assembly. In Saccharomyces cerevisiae Nic96 contains Nsp1p, Nup57p, Nup49p, and Nic96p." [GOC:jh, PMID:12791264, PMID:15741174]
GO:0055126
name: Nup82 complex
def: "OBSOLETE. A protein complex that forms part of the nuclear pore complex. It forms a subcomplex with Nup159p and Nsp1p, interacts with Nup116p, and is required for proper localization of Nup116p. In Saccharomyces cerevisiae this complex contains Nup82p, Nsp1p, Nup159p, Nup116p, and Gle2p." [GOC:jh, PMID:12791264, PMID:15741174]
GO:0060053
name: neurofilament cytoskeleton
def: "Intermediate filament cytoskeletal structure that is made up of neurofilaments. Neurofilaments are specialized intermediate filaments found in neurons." [GOC:dph]
GO:0060076
name: excitatory synapse
def: "A synapse in which an action potential in the presynaptic cell increases the probability of an action potential occurring in the postsynaptic cell." [GOC:dph, GOC:ef]
GO:0060077
name: inhibitory synapse
def: "A synapse in which an action potential in the presynaptic cell reduces the probability of an action potential occurring in the postsynaptic cell." [GOC:dph, GOC:ef]
GO:0060091
name: kinocilium
def: "A nonmotile primary cilium that is found at the apical surface of auditory receptor cells. The kinocilium is surrounded by actin-based stereocilia." [GOC:dph, PMID:15882574]
GO:0060102
name: collagen and cuticulin-based cuticle extracellular matrix
def: "A collagen and cuticulin-based noncellular, multilayered structure that is synthesized by an underlying ectodermal (hypodermal) cell layer. The cuticle serves essential functions in body morphology, locomotion, and environmental protection. An example of this component is found in Caenorhabditis elegans." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060103
name: collagen and cuticulin-based cuticle extracellular matrix part
def: "Any constituent part of the collagen and cuticulin-based cuticle extracellular matrix, a collagen and cuticulin-based noncellular, multilayered structure that is synthesized by an underlying ectodermal (hypodermal) cell layer." [GOC:dph]
GO:0060104
name: surface coat of collagen and cuticulin-based cuticle extracellular matrix
def: "An electron dense, amorphous envelope that comprises the outermost layer of the cuticle. The surface coat is loosely apposed to the epicuticle, has distinct biochemical properties, is synthesized by cells other than the underlying hypodermis, and is labile. In addition to serving as a lubricant to protect against abrasion and dehydration, the surface coat may also play important roles in infection and immune evasion. An example of this component is found in Caenorhabditis elegans." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060105
name: epicuticle of collagen and cuticulin-based cuticle extracellular matrix
def: "A lipid-containing layer of cuticle that lies between the cortical layer and the surface coat. An example of this component is found in Caenorhabditis elegans." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060106
name: cortical layer of collagen and cuticulin-based cuticle extracellular matrix
def: "The cuticle layer that lies directly beneath the lipid-containing epicuticle. The cortical layer contains collagens and insoluble, non-collagenous cuticulins and is characterized by a distinct annular pattern consisting of regularly spaced annular ridges delineated by annular furrows. An example of this component is found in Caenorhabditis elegans." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060107
name: annuli extracellular matrix
def: "The extracellular matrix that is a regularly spaced circumferential ridge present in the cortical region of the cuticle. Annuli are delineated by annular furrows and are present throughout the cuticle with the exception of lateral regions where longitudinal alae are present." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060108
name: annular furrow extracellular matrix
def: "The extracellular matrix part that is a regularly spaced indentation in the outer cortical layer of the cuticle. The pattern of annular furrows corresponds to sites of invaginations in hypodermal cell membranes that, in turn, correspond to submembranous regions where actin microfilament bundles assemble early in lethargus, the first phase of the molting cycle in which activity and feeding decline." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060109
name: medial layer of collagen and cuticulin-based cuticle extracellular matrix
def: "The fluid-filled cuticle layer that lies between the cortical and basal layers and is characterized by the presence of regularly spaced columnar struts that lie on either side of the annular furrows and link the two surrounding layers. In C. elegans, a defined medial layer is found only in adult animals." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060110
name: basal layer of collagen and cuticulin-based cuticle extracellular matrix
def: "The layer of cuticle most closely apposed to the hypodermal cells. The morphology of the basal layer varies with life stage. In adult C. elegans animals, the basal layers is comprised of three sublayers: two fibrous layers whose fibers run in clockwise and counter-clockwise directions meeting one another at a 60 degree angle, and an amorphous basal layer that lies underneath the fibrous layers and directly contacts the hypodermis. In C. elegans dauer and L1 larval stage animals, the basal layer is characterized by a striated pattern that appears to derive from interwoven laminae. An example of this component is found in Caenorhabditis elegans." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060111
name: alae of collagen and cuticulin-based cuticle extracellular matrix
def: "Raised, thickened cuticular ridges that run longitudinally, and in parallel, along the left and right sides of the animal. The alae lie above the hypodermal cells known as the lateral seam cells. In C. elegans, alae are produced in L1 larvae, dauer larvae and adult stage animals, where they consist of three, five, and three ridges of distinct morphology, respectively." [GOC:dph, GOC:kmv, ISSN:15518507]
GO:0060170
name: ciliary membrane
def: "The portion of the plasma membrane surrounding a cilium." [GOC:cilia, GOC:dph, GOC:rph]
GO:0060171
name: stereocilium membrane
def: "The portion of the plasma membrane surrounding a stereocilium." [GOC:dph, GOC:rph]
GO:0060187
name: cell pole
def: "Either of two different areas at opposite ends of an axis of a cell." [GOC:dph]
GO:0060198
name: clathrin-sculpted vesicle
def: "A clathrin-sculpted lipid bilayer membrane-enclosed vesicle after clathrin release." [GOC:dph]
GO:0060199
name: clathrin-sculpted glutamate transport vesicle
def: "A clathrin-sculpted lipid bilayer membrane-enclosed vesicle after clathrin release and containing glutamate." [GOC:dph]
GO:0060200
name: clathrin-sculpted acetylcholine transport vesicle
def: "A clathrin-sculpted lipid bilayer membrane-enclosed vesicle after clathrin release and containing acetylcholine." [GOC:dph]
GO:0060201
name: clathrin-sculpted acetylcholine transport vesicle membrane
def: "The lipid bilayer surrounding a clathrin-sculpted acetylcholine transport vesicle." [GOC:dph]
GO:0060202
name: clathrin-sculpted acetylcholine transport vesicle lumen
def: "The volume enclosed by the membrane of the clathrin-sculpted acetylcholine transport vesicle." [GOC:dph]
GO:0060203
name: clathrin-sculpted glutamate transport vesicle membrane
def: "The lipid bilayer surrounding a clathrin-sculpted glutamate transport vesicle." [GOC:dph]
GO:0060204
name: clathrin-sculpted glutamate transport vesicle lumen
def: "The volume enclosed by the membrane of the clathrin-sculpted glutamate transport vesicle." [GOC:dph]
GO:0060205
name: cytoplasmic membrane-bounded vesicle lumen
def: "The volume enclosed by the membrane of a cytoplasmic membrane-bounded vesicle." [GOC:dph]
GO:0060293
name: germ plasm
def: "Differentiated cytoplasm associated with a pole of an oocyte, egg or early embryo that will be inherited by the cells that will give rise to the germ line." [GOC:dph]
GO:0060342
name: photoreceptor inner segment membrane
def: "The membrane surrounding the outer segment of a vertebrate photoreceptor. The photoreceptor inner segment contains mitochondria, ribosomes and membranes where opsin molecules are assembled and passed to be part of the outer segment discs." [GOC:dph]
GO:0060387
name: fertilization envelope
def: "A structure that lies outside the plasma membrane and surrounds the egg. The fertilization envelope forms from the vitelline membrane after fertilization as a result of cortical granule release." [GOC:dph, ISBN:0878932437]
GO:0060388
name: vitelline envelope
def: "A glycoprotein-based structure that lies outside the plasma membrane and surrounds the egg before fertilization." [GOC:dph, ISBN:0878932437]
GO:0060417
name: yolk
def: "The cytoplasmic part that serves as a nutrient reserve or energy source for the developing embryo." [GOC:dph, GOC:tb, PMID:18046696]
GO:0060418
name: yolk plasma
def: "Discrete structures that partition the water-soluble portion of the yolk of oocytes and ova, which may or may not be membrane enclosed." [GOC:dph, GOC:tb, PMID:18046696]
GO:0060473
name: cortical granule
def: "A secretory vesicle that is stored under the cell membrane of an egg. These vesicles fuse with the egg plasma membrane as part of egg activation and are part of the block to polyspermy." [GOC:dph]
GO:0060987
name: lipid tube
def: "A macromolecular complex that contains a tube of lipid surrounded by a protein coat." [GOC:ascb_2009, GOC:dph, GOC:tb]
GO:0061174
name: type I terminal bouton
def: "Terminal inflated portion of the axon of a glutamatergic neuron, containing the specialized apparatus necessary to release neurotransmitters that will induce the contraction of muscle. The axon terminus is considered to be the whole region of thickening and the terminal bouton is a specialized region of it." [GOC:dph, GOC:mc]
GO:0061175
name: type II terminal bouton
def: "Terminal inflated portion of the axon of a non-glutamatergic neuron, containing the specialized apparatus necessary to release neurotransmitters at a regulatory synapse. The axon terminus is considered to be the whole region of thickening and the terminal bouton is a specialized region of it." [GOC:dph, GOC:mc]
GO:0061176
name: type Ib terminal bouton
def: "Terminal inflated portion of the axon of a glutamatergic neuron, containing the specialized apparatus necessary for the tonic release neurotransmitters that will induce the contraction of muscle. Type Ib terminal boutons are larger than type Is terminal boutons." [GOC:dph, GOC:mc]
GO:0061177
name: type Is terminal bouton
def: "Terminal inflated portion of the axon of a glutamatergic neuron, containing the specialized apparatus necessary for the phasic release neurotransmitters that will induce the contraction of muscle. Type Is terminal boutons are smaller than type Ib terminal boutons." [GOC:dph, GOC:mc]
GO:0061200
name: clathrin-sculpted gamma-aminobutyric acid transport vesicle
def: "A clathrin-sculpted lipid bilayer membrane-enclosed vesicle after clathrin release and containing gamma-aminobutyric acid transport vesicle." [GOC:dph]
GO:0061201
name: clathrin-sculpted gamma-aminobutyric acid transport vesicle lumen
def: "The volume enclosed by the membrane of the clathrin-sculpted gamma-aminobutyric acid transport vesicle." [GOC:dph]
GO:0061202
name: clathrin-sculpted gamma-aminobutyric acid transport vesicle membrane
def: "The lipid bilayer surrounding a clathrin-sculpted gamma-aminobutyric acid transport vesicle." [GOC:dph]
GO:0061455
name: integral component of muscle cell projection membrane
def: "The component of the muscle cell projection membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:dph, GOC:tb]
GO:0061464
name: plasma membrane part of cell-substrate junction.
def: "The part of the plasma membrane that contributes to the structure of a cell-substrate junction." [GOC:dph]
GO:0061465
name: plasma membrane part of hemidesmosome
def: "The part of the plasma membrane that contributes to the structure of a hemidesmosome." [GOC:dph]
GO:0061466
name: plasma membrane part of cell junction
def: "The part of the plasma membrane that contributes to the structure of a cell junction." [GOC:dph]
GO:0061468
name: karyomere
def: "A membrane-bound intermediate cleavage-stage structure of individual or groups of chromosomes that coalesces and fuses with other karyomeres to form a nucleus during interphase. Karyomere formation occurs in blastomeres undergoing rapid cell division." [GOC:dph, PMID:12734396, PMID:22863006]
GO:0061474
name: phagolysosome membrane
def: "The lipid bilayer surrounding a phagolysosome." [GOC:dph, PMID:22073313]
GO:0061493
name: central plaque of mitotic spindle pole body
def: "One of three laminate structures that form the mitotic spindle pole body; the central plaque is embedded in the nuclear envelope." [GOC:dph]
GO:0061494
name: gamma-tubulin large complex, mitotic spindle pole body
def: "A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes located in the mitotic spindle pole body." [GOC:dph]
GO:0061495
name: gamma-tubulin small complex, mitotic spindle pole body
def: "A complex composed of two gamma-tubulin molecules and conserved non-tubulin proteins located in the mitotic spindle pole body and isolated by fractionation from cells. The complex, approximately 6S-9S, is analogous to the small complex in animal cells but contains fewer subunits, and is not thought to multimerize into larger functional units, like complexes in those organisms. An example of this structure is found in Saccharomyces cerevisiae." [GOC:dph]
GO:0061496
name: half bridge of mitotic spindle pole body
def: "Structure adjacent to the plaques of the mitotic spindle pole body." [GOC:dph]
GO:0061497
name: inner plaque of mitotic spindle pole body
def: "One of three laminate structures that form the mitotic spindle pole body; the inner plaque is in the nucleus." [GOC:dph, GOC:vw]
GO:0061498
name: intermediate layer of mitotic spindle pole body
def: "Structure between the central and outer plaques of the mitotic spindle pole body." [GOC:dph]
GO:0061499
name: outer plaque of mitotic spindle pole body
def: "One of three laminate structures that form the mitotic spindle pole body; the outer plaque is in the cytoplasm." [GOC:dph]
GO:0061574
name: ASAP complex
def: "A protein complex involved in regulation of mRNA processing and apoptosis. It binds to RNA in a sequence-independent manner and is recruited to the EJC prior to or during the splicing process. In humans the core proteins are RNPS1, SAP18 and ACIN1." [GOC:dph, PMID:12665594, PMID:16314458, PMID:22388736]
GO:0061576
name: acyl-CoA ceramide synthase complex
def: "A protein complex that catalyzes the reaction acyl-CoA + sphingosine = CoA + N-acylsphingosine. In S. cerevisiae it contains three subunits: lag1, lac1 and lip1." [GOC:dph, PMID:15692566]
GO:0061617
name: MICOS complex
def: "Mitochondrial inner membrane complex involved in maintenance of crista junctions, inner membrane architecture, and formation of contact sites to the outer membrane. In Saccharomyces cerevisiae the complex has six subunits: MIC10, MIC12, MIC19, MIC26, MIC27, and MIC60." [GOC:dph, PMID:21944719, PMID:21987634, PMID:22009199, PMID:24687277]
GO:0061618
name: sublamina densa
def: "The part of the basement membrane that lies beneath the lamina densa, containing anchoring fibrils, anchoring plaques, collagen fibers, and elastic fibers." [GOC:BHF, PMID:15623520]
GO:0061638
name: CENP-A containing chromatin
def: "The specialized chromatin located the centromeric core region or the entire centromeric region in organisms with point centromeres, which is enriched for CENP-A-containing nucleosomes. This chromatin forms a 3-dimensional structure which provides a platform for kinetochore assembly and microtubule attachment." [GOC:vw, PMID:20206496, PMID:22729156]
GO:0061645
name: endocytic patch
def: "The part of the cell cortex consisting of  an aggregation of proteins that will give rise to an endocytic vesicle." [GOC:dph, PMID:22949647]
GO:0061671
name: Cbp3p-Cbp6 complex
def: "A protein complex located at the mitochondrial ribosome tunnel exit that is involved in efficient translation and protein complex assembly." [GOC:dph, GOC:rb, PMID:21670217]
GO:0061672
name: glutathione hydrolase complex
def: "Enzyme complex that in S. cerevisiae has compnents Dug2/Dug3 and is able to catalyze the cleavage of glutathione into glutamate and Cys-Gly." [GOC:dph]
GO:0061673
name: mitotic spindle astral microtubule
def: "Any of the mitotic spindle microtubules that radiate in all directions from the spindle poles and are thought to contribute to the forces that separate the poles and position them in relation to the rest of the cell." [GOC:dph]
GO:0065010
name: extracellular membrane-bounded organelle
def: "Organized structure of distinctive morphology and function, bounded by a lipid bilayer membrane and occurring outside the cell." [GOC:isa_complete]
GO:0070013
name: intracellular organelle lumen
def: "An organelle lumen that is part of an intracellular organelle." [GOC:mah]
GO:0070014
name: sucrase-isomaltase complex
def: "A protein complex that possesses oligo-1,6-glucosidase activity; the complex is a heterodimer located in the cell membrane, and is formed by proteolytic cleavage of a single precursor polypeptide. The two subunits have different substrate specificities." [PMID:3366777]
GO:0070017
name: alphav-beta3 integrin-thrombospondin complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to thrombospondin." [PMID:2478219]
GO:0070018
name: transforming growth factor beta type I receptor homodimeric complex
def: "A receptor complex that consists of two transforming growth factor beta (TGF-beta) type I receptor monomers. TGF-beta type I receptor dimers form in the presence or absence of ligand, and can associate with ligand-bound TGF-beta type II receptor dimers." [Reactome:REACT_7737]
GO:0070019
name: transforming growth factor beta type II receptor homodimeric complex
def: "A receptor complex that consists of two transforming growth factor beta (TGF-beta) type II receptor monomers. TGF-beta type II receptor dimers form in the presence or absence of ligand, and upon ligand binding can associate with TGF-beta type I receptor dimers." [Reactome:REACT_7415]
GO:0070020
name: transforming growth factor beta1-type II receptor complex
def: "A protein complex that consists of a dimeric transforming growth factor beta (TGF-beta) type II receptor bound to a TGF-beta1 dimer." [Reactome:REACT_7218]
GO:0070021
name: transforming growth factor beta1-type II receptor-type I receptor complex
def: "A protein complex that is formed by the association of a ligand-bound TGF-beta type II receptor dimer with a TGF-beta type I receptor dimer." [Reactome:REACT_7425]
GO:0070022
name: transforming growth factor beta receptor homodimeric complex
def: "A homodimeric receptor complex that consists of two TGF-beta receptor monomers." [GOC:mah, Reactome:REACT_7415, Reactome:REACT_7737]
GO:0070023
name: interleukin-12-interleukin-12 receptor complex
def: "A protein complex that is formed by the association of a heterodimeric interleukin-12 receptor complex with an interleukin-12 heterodimer." [PMID:11900991]
GO:0070024
name: CD19-Vav-PIK3R1 complex
def: "A protein complex that contains the cell surface signaling molecule CD19, the Ras guanine nucleotide exchange factor Vav, and the regulatory subunit alpha of phosphatidylinositol 3-kinase (PI3K)." [PMID:7528218]
GO:0070029
name: alphav-beta3 integrin-osteopontin complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to osteopontin." [PMID:7532190]
GO:0070030
name: alphav-beta1 integrin-osteopontin complex
def: "A protein complex that consists of an alphav-beta1 integrin complex bound to osteopontin." [PMID:7592829]
GO:0070031
name: alphav-beta5 integrin-osteopontin complex
def: "A protein complex that consists of an alphav-beta5 integrin complex bound to osteopontin." [PMID:7592829]
GO:0070032
name: synaptobrevin 2-SNAP-25-syntaxin-1a-complexin I complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 1a, and complexin I (or orthologs thereof)." [PMID:7553862]
GO:0070033
name: synaptobrevin 2-SNAP-25-syntaxin-1a-complexin II complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 1a, and complexin II (or orthologs thereof)." [PMID:7553862]
GO:0070044
name: synaptobrevin 2-SNAP-25-syntaxin-1a complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, and syntaxin 1a (or orthologs thereof)." [PMID:10336434]
GO:0070045
name: synaptobrevin 2-SNAP-25-syntaxin-2 complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, and syntaxin 2 (or orthologs thereof)." [PMID:10336434]
GO:0070046
name: synaptobrevin 2-SNAP-25-syntaxin-3 complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, and syntaxin 3 (or orthologs thereof)." [PMID:10336434]
GO:0070047
name: synaptobrevin 2-SNAP-25-syntaxin-4 complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, and syntaxin 4 (or orthologs thereof)." [PMID:10336434]
GO:0070048
name: endobrevin-SNAP-25-syntaxin-1a complex
def: "A SNARE complex that contains endobrevin (VAMP8), SNAP-25, and syntaxin 1a (or orthologs thereof)." [PMID:10336434]
GO:0070049
name: endobrevin-SNAP-25-syntaxin-2 complex
def: "A SNARE complex that contains endobrevin (VAMP8), SNAP-25, and syntaxin 2 (or orthologs thereof)." [PMID:10336434]
GO:0070056
name: prospore membrane leading edge
def: "The region of the prospore membrane that extends to surround the spore nucleus; coated with specific proteins that are thought to play a role in prospore membrane organization." [GOC:mah, PMID:14702385]
GO:0070057
name: prospore membrane spindle pole body attachment site
def: "The region of the prospore membrane to which the spindle pole body (SPB) is anchored; the prospore membrane extends from the SPB attachment site to surround the spore nucleus." [GOC:mah, PMID:14702385]
GO:0070062
name: extracellular vesicular exosome
def: "A membrane-bounded vesicle that is released into the extracellular region by fusion of the limiting endosomal membrane of a multivesicular body with the plasma membrane." [GOC:BHF, GOC:mah, PMID:15908444, PMID:17641064]
GO:0070065
name: cellubrevin-VAMP4-syntaxin-16 complex
def: "A SNARE complex that contains cellubrevin (VAMP3), VAMP4, and syntaxin 16 (or orthologs thereof)." [PMID:11839770]
GO:0070066
name: cellubrevin-VAMP4-endobrevin-syntaxin-6 complex
def: "A SNARE complex that contains cellubrevin (VAMP3), VAMP4, endobrevin (VAMP8), and syntaxin 6 (or orthologs thereof)." [PMID:11839770]
GO:0070067
name: syntaxin-6-syntaxin-16-Vti1a complex
def: "A SNARE complex that contains syntaxin 6, syntaxin 16, and Vti1a (or orthologs thereof)." [PMID:11839770]
GO:0070068
name: VAMP4-syntaxin-6-syntaxin-16-Vti1a complex
def: "A SNARE complex that contains VAMP4, syntaxin 6, syntaxin 16, and Vti1a (or orthologs thereof)." [PMID:11839770]
GO:0070069
name: cytochrome complex
def: "A protein complex in which at least one of the proteins is a cytochrome, i.e. a heme-containing protein involved in catalysis of redox reactions." [GOC:mah]
GO:0070074
name: mononeme
def: "A secretory organelle that forms part of the apical complex; a small, threadlike structure located is close proximity to the subpellicular microtubules. Its contents include a rhomboid protease (PfROM1 in Plasmodium falciparum) that moves from the lateral asymmetric localization to the merozoite apical pole and the posterior pole upon release of merozoites from schizonts." [GOC:BHF, PMID:18048320]
GO:0070081
name: clathrin-sculpted monoamine transport vesicle
def: "A clathrin-sculpted lipid bilayer membrane-enclosed vesicle after clathrin release and containing monoamines." [GOC:mg2]
GO:0070082
name: clathrin-sculpted monoamine transport vesicle lumen
def: "The volume enclosed by the membrane of the clathrin-sculpted monoamine transport vesicle." [GOC:mg2]
GO:0070083
name: clathrin-sculpted monoamine transport vesicle membrane
def: "The lipid bilayer surrounding a clathrin-sculpted monoamine transport vesicle." [GOC:mg2]
GO:0070088
name: PHA granule
def: "An inclusion body located in the cytoplasm that consists of polyhydroxyalkanoate (PHA) molecules and associated proteins, surrounded by a phospholipid monolayer; the proteins include PHA synthase, PHA depolymerase and 3HB-oligomer hydroxylase, phasins (PhaPs), which are thought to be the major structural proteins of the membrane surrounding the inclusion, and the regulator of phasin expression PhaR." [GOC:mah, PMID:15762612]
GO:0070090
name: metaphase plate
def: "The intracellular plane, located halfway between the poles of the spindle, where chromosomes align during metaphase of mitotic or meiotic nuclear division." [GOC:mah]
GO:0070110
name: ciliary neurotrophic factor receptor complex
def: "A protein complex that acts as a receptor for the cytokine ciliary neurotrophic factor (CNTF). In humans the receptor complex is a hexamer composed of two molecules each of CNTF and CNTFR and one molecule each of gp130 and LIFR." [GOC:BHF, GOC:mah, GOC:rl, PMID:12707266]
GO:0070111
name: organellar chromatophore
def: "A bacteroid-containing symbiosome in which the bacterial component is a genetically highly reduced cyanobacterium that is photosynthetically active and incapable of an independent existence outside its host. The chromatophore functions as a photosynthetic organelle, and has been found and characterized in the amoeba Paulinella chromatophora." [GOC:expert_mm, PMID:18356055]
GO:0070112
name: organellar chromatophore membrane
def: "Either of the lipid bilayers that surround an organellar chromatophore." [GOC:mah]
GO:0070113
name: organellar chromatophore inner membrane
def: "The inner, i.e. lumen-facing, of the two lipid bilayers surrounding an organellar chromatophore." [GOC:mah]
GO:0070114
name: organellar chromatophore outer membrane
def: "The outer, i.e. cytoplasm-facing, of the two lipid bilayers surrounding an organellar chromatophore." [GOC:mah]
GO:0070115
name: organellar chromatophore intermembrane space
def: "The region between the inner and outer lipid bilayers that surround an organellar chromatophore." [GOC:mah]
GO:0070116
name: organellar chromatophore thylakoid
def: "A thylakoid located in an organellar chromatophore." [GOC:mah]
GO:0070117
name: organellar chromatophore thylakoid lumen
def: "The volume enclosed by an organellar chromatophore thylakoid membrane." [GOC:mah]
GO:0070118
name: organellar chromatophore thylakoid membrane
def: "The lipid bilayer membrane of any thylakoid within an organellar chromatophore." [GOC:mah]
GO:0070160
name: occluding junction
def: "A cell-cell junction that seals cells together in an epithelium in a way that prevents even small molecules from leaking from one side of the sheet to the other." [ISBN:0815332181]
GO:0070161
name: anchoring junction
def: "A cell junction that mechanically attaches a cell (and its cytoskeleton) to neighboring cells or to the extracellular matrix." [ISBN:0815332181]
GO:0070176
name: DRM complex
def: "A transcriptional repressor complex that contains the lin-9, lin-35, lin-37, lin-52, lin-53, lin-5is involved in 4-, dpl-1 and efl-1 proteins, and is involved in cell fate specification." [PMID:17075059]
GO:0070187
name: telosome
def: "A nuclear telomere cap complex that is formed by the association of telomeric ssDNA- and dsDNA-binding proteins with telomeric DNA, and is involved in telomere protection and recruitment of telomerase. The complex contains TRF1, TRF2, POT1, RAP1, TIN2 and TPP1 in mammalian cells, and Pot1, Tpz1, Ccq1, Poz1, Rap1, Rif1, Rif2and Rap1 in Schizosaccharomyces. Taz1 and Rap1 (or their mammalian equivalents) form a dsDNA-binding subcomplex, Pot1 and Tpz1 form an ssDNA-binding subcomplex, and the two subcomplexes are bridged by Poz1." [GOC:expert_mf, GOC:mah, GOC:vw, PMID:18828880]
GO:0070188
name: Stn1-Ten1 complex
def: "A nuclear telomere cap complex that is formed by the association of the Stn1 and Ten1 proteins with telomeric DNA; in some species a third protein is present." [GOC:mah, GOC:vw, PMID:17715303, PMID:19064932]
GO:0070195
name: growth hormone receptor complex
def: "A receptor complex that consists of two identical subunits and binds growth hormone." [GOC:BHF, GOC:mah, GOC:vk, PMID:11445442]
GO:0070209
name: ASTRA complex
def: "A protein complex that is part of the chromatin remodeling machinery; the acronym stands for ASsembly of Tel, Rvb and Atm-like kinase. In Saccharomyces cerevisiae this complex includes Rvb1p, Rvb2p, Tra1p, Tel2p, Asa1p, Ttilp and Tti2p." [GOC:rb, PMID:19040720]
GO:0070210
name: Rpd3L-Expanded complex
def: "A protein complex that contains a histone deacetylase and is part of the chromatin remodeling machinery. In Saccharomyces cerevisiae this complex contains the Rpd3p, Sin3p, Ume1p, Pho23p, Sap30p, Sds3p, Cti6p, Rxt2p, Rxt3p, Dep1p, Ume6p, Ash1p, Dot6p, Snt1, Sif2p, Set3p, Hos2p, Tos4p and Tod6p proteins." [GOC:rb, PMID:19040720]
GO:0070211
name: Snt2C complex
def: "A histone deacetylase complex that is part of the chromatin remodeling machinery. In Saccharomyces cerevisiae this complex contains Snt2p, Ecm5p and Rpd3p." [GOC:rb, PMID:19040720]
GO:0070214
name: CSK-GAP-A.p62 complex
def: "A protein complex that contains the protein-tyrosine kinase CSK and the GTPase-activating protein (GAP)-associated p62 (GAP-A.p62); may mediate translocation of proteins, including GAP and CSK, to membrane or cytoskeletal regions upon c-Src activation." [PMID:7544435]
GO:0070250
name: mating projection membrane
def: "The portion of the plasma membrane surrounding a mating projection, the projection formed by unicellular fungi in response to mating pheromone." [GOC:jp]
GO:0070258
name: inner membrane complex
def: "A membrane structure formed of two closely aligned lipid bilayers that lie beneath the plasma membrane and form part of the pellicle surrounding an apicomplexan parasite cell." [GOC:mah, PMID:12456714]
GO:0070263
name: external side of fungal-type cell wall
def: "The side of the fungal-type cell wall that is opposite to the side that faces the cell and its contents." [GOC:mah]
GO:0070264
name: transcription factor TFIIIE complex
def: "A transcription factor complex that is involved in regulating transcription from RNA polymerase III (Pol III) promoters. TFIIIE contains a specific subset of ribosomal proteins." [GOC:jp, PMID:19116144]
GO:0070274
name: RES complex
def: "A protein complex that is required for efficient splicing, and prevents leakage of unspliced pre-mRNAs from the nucleus (named for pre-mRNA REtention and Splicing). In Saccharomyces, the complex consists of Ist3p, Bud13p, and Pml1p." [PMID:15565172, PMID:18809678, PMID:19010333, PMID:19033360]
GO:0070288
name: ferritin complex
def: "A protein complex that binds iron and acts as a major iron storage system. Intracellular and extracellular ferritin complexes have different ratios of two types of ferritin monomer, the L (light) chain and H (heavy) chain." [GOC:mah, PMID:19154717]
GO:0070289
name: extracellular ferritin complex
def: "A ferritin complex located in the extracellular region. Extracellular ferritin complexes contain L (light) chains but few or no H (heavy) chains." [GOC:mah, PMID:19154717]
GO:0070310
name: ATR-ATRIP complex
def: "A protein complex that contains the protein kinase ATR and ATR-interacting protein (ATRIP) and binds single-stranded DNA; ssDNA binding affinity is increased in the presence of replication protein A." [GOC:mah, PMID:14724280]
GO:0070311
name: nucleosomal methylation activator complex
def: "A protein complex that contains eight subunits in common with the SWI/SNF complex, plus the ATPase BRG1 (SMARCA4) and the histone methyltransferase CARM1; the complex is involved in regulating nuclear receptor-dependent transcription." [GOC:mah, PMID:14729568]
GO:0070312
name: RAD52-ERCC4-ERCC1 complex
def: "A nucleotide-excision repair complex formed by the association of the heterodimeric endonuclease XPF/ERCC4-ERCC1 (Rad1p and Rad10p in S. cerevisiae) with the RAD52 protein." [PMID:14734547]
GO:0070313
name: RGS6-DNMT1-DMAP1 complex
def: "A protein complex formed by the association of RGS6, a negative regulator of heterotrimeric G protein signaling, with the DMAP1-Dnmt1 transcriptional repressor complex; in the complex, RGS6 inhibits the transcriptional repressor activity of DMAP1." [GOC:mah, PMID:14734556]
GO:0070319
name: Golgi to plasma membrane transport vesicle
def: "A transport vesicle that mediates transport from the Golgi to the plasma membrane, and fuses with the plasma membrane to release various cargo molecules, such as proteins or hormones, by exocytosis." [GOC:kad, GOC:mah]
GO:0070331
name: CD20-Lck-Fyn complex
def: "A protein complex that contains the cell-surface protein CD20 and the Src family tyrosine kinases Lck and Fyn." [GOC:mah, PMID:7545683]
GO:0070332
name: CD20-Lck-Lyn-Fyn complex
def: "A protein complex that contains the cell-surface protein CD20 and the Src family tyrosine kinases Lck, Lyn and Fyn." [GOC:mah, PMID:7545683]
GO:0070333
name: alpha6-beta4 integrin-Shc-Grb2 complex
def: "A protein complex that consists of an alpha6-beta4 integrin complex bound to the adaptor proteins Shc and Grb2." [PMID:7556090]
GO:0070334
name: alpha6-beta4 integrin-laminin 5 complex
def: "A protein complex that consists of an alpha6-beta4 integrin complex bound to laminin 5." [GOC:mah, PMID:7556090]
GO:0070353
name: GATA1-TAL1-TCF3-Lmo2 complex
def: "A protein complex that contains the zinc finger transcription factor GATA1, the LIM domain protein Lmo2 (RBTN2), the basic helix-loop-helix protein TAL1 and its binding partner TCF3. The complex is involved transcriptional regulation in hematopoiesis." [PMID:7568177]
GO:0070354
name: GATA2-TAL1-TCF3-Lmo2 complex
def: "A protein complex that contains the zinc finger transcription factor GATA2, the LIM domain protein Lmo2 (RBTN2), the basic helix-loop-helix protein TAL1 and its binding partner TCF3. The complex is involved transcriptional regulation in hematopoiesis." [PMID:7568177]
GO:0070355
name: synaptotagmin-synaptobrevin 2-SNAP-25-syntaxin-1a-syntaxin-1b-Rab3a-complexin II complex
def: "A SNARE complex that contains synaptotagmin, synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 1a, syntaxin1b, Rab3a, and complexin II (or orthologs thereof)." [PMID:7654227]
GO:0070356
name: synaptotagmin-synaptobrevin 2-SNAP-25-syntaxin-1a-syntaxin-1b-Rab3a complex
def: "A SNARE complex that contains synaptotagmin, synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 1a, syntaxin1b, and Rab3a (or orthologs thereof)." [PMID:7654227]
GO:0070357
name: alphav-beta3 integrin-CD47 complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to CD47 (also known as IAP)." [PMID:2277087, PMID:7691831]
GO:0070369
name: beta-catenin-TCF7L2 complex
def: "A protein complex that contains beta-catenin and TCF7L2 (TCF4), binds to the TCF DNA motif within a promoter element, and is involved in the regulation of WNT target gene transcription." [GOC:BHF, GOC:rl, PMID:9065401, PMID:9065402]
GO:0070381
name: endosome to plasma membrane transport vesicle
def: "A transport vesicle that mediates transport from the endosome to the plasma membrane, and fuses with the plasma membrane to release various cargo molecules, such as proteins or hormones, by exocytosis." [GOC:kad, GOC:mah, PMID:10679016, PMID:12110576]
GO:0070382
name: exocytic vesicle
def: "A transport vesicle that mediates transport from an intracellular compartment to the plasma membrane, and fuses with the plasma membrane to release various cargo molecules, such as proteins or hormones, by exocytosis." [GOC:kad, GOC:mah]
GO:0070385
name: egasyn-beta-glucuronidase complex
def: "A protein complex that contains beta-glucuronidase and the carboxyl esterase egasyn; formation of the complex causes beta-glucuronidase to be retained in the endoplasmic reticulum." [PMID:7744842]
GO:0070386
name: procollagen-proline 4-dioxygenase complex, alpha(I) type
def: "A procollagen-proline 4-dioxygenase complex that contains alpha subunits of the type I isoform; its activity is readily inhibited by poly(L-proline)." [PMID:14500733, PMID:7753822]
GO:0070387
name: procollagen-proline 4-dioxygenase complex, alpha(II) type
def: "A procollagen-proline 4-dioxygenase complex that contains alpha subunits of the type II isoform; its activity is inhibited by poly(L-proline) only at high concentrations." [PMID:14500733, PMID:7753822]
GO:0070388
name: procollagen-proline 4-dioxygenase complex, alpha(III) type
def: "A procollagen-proline 4-dioxygenase complex that contains alpha subunits of the type III isoform." [PMID:14500733]
GO:0070390
name: transcription export complex 2
def: "A protein complex that couples SAGA-dependent gene expression to mRNA export at the inner side of the nuclear pore complex (NPC). The TREX-2 complex is tethered to the inner side of the NPC via the nucleoporins Nup1 and Nup60; in S. cerevisiae it contains Sac3p, Thp1p, Sus1p and Cdc31p." [GOC:dgf, GOC:mah, PMID:17786152]
GO:0070418
name: DNA-dependent protein kinase complex
def: "A protein complex that is involved in the repair of DNA double-strand breaks and, in mammals, V(D)J recombination events. It consists of the DNA-dependent protein kinase catalytic subunit (DNA-PKcs) and the DNA end-binding heterodimer Ku." [GOC:mah, PMID:10854421, PMID:12235392]
GO:0070419
name: nonhomologous end joining complex
def: "A protein complex that plays a role in DNA double-strand break repair via nonhomologous end joining. Such complexes typically contain a specialized DNA ligase (e.g. Lig4 in eukaryotes) and one or more proteins that bind to DNA ends." [GOC:mah, PMID:17072889, PMID:17938628]
GO:0070420
name: Ku-DNA ligase complex
def: "A nonhomologous end joining complex that contains one or more Ku monomers and one or more DNA ligase molecules from the LigC or LigD family, and mediates nonhomologous end joining in bacteria." [GOC:mah, PMID:17938628]
GO:0070421
name: DNA ligase III-XRCC1 complex
def: "A protein complex that contains DNA ligase III and XRCC1, and is involved in base excision repair." [PMID:15141024, PMID:7760816]
GO:0070422
name: G-protein beta/gamma-Raf-1 complex
def: "A protein complex formed by the association of the serine-threonine protein kinase Raf-1 with the beta and gamma subunits of a heterotrimeric G protein." [GOC:mah, PMID:7782277]
GO:0070435
name: Shc-EGFR complex
def: "A protein complex that contains the epidermal growth factor receptor (EGFR) and the adaptor protein Shc, and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7798267]
GO:0070436
name: Grb2-EGFR complex
def: "A protein complex that contains the epidermal growth factor receptor (EGFR) and Grb2, and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7798267]
GO:0070437
name: Grb2-Shc complex
def: "A protein complex that contains Grb2 and the adaptor protein Shc, and is involved in linking epidermal growth factor receptor (EGFR) activation to the p21-Ras pathway." [GOC:mah, PMID:7798267]
GO:0070438
name: mTOR-FKBP12-rapamycin complex
def: "OBSOLETE. A protein complex that contains the mTOR (mammalian target of rapamycin) serine/threonine kinase, the peptidyl-prolyl cis-trans isomerase FKBP12 (FKBP1A) and rapamycin (sirolimus)." [GOC:sl, PMID:20005306, PMID:7822316]
GO:0070439
name: Mad-Max-mSin3A complex
def: "A transcriptional repressor complex that contains a heterodimer of the bHLH-ZIP proteins Mad and Max, plus mSin3A, a homolog of the yeast Sin3p." [PMID:7889570]
GO:0070440
name: Mad-Max-mSin3B complex
def: "A transcriptional repressor complex that contains a heterodimer of the bHLH-ZIP proteins Mad and Max, plus mSin3B, a homolog of the yeast Sin3p." [PMID:7889570]
GO:0070441
name: G-protein beta/gamma-Btk complex
def: "A protein complex formed by the association of the Bruton tyrosine protein kinase Btk, which is implicated in mammalian X-linked immunodeficiencies, with the beta and gamma subunits of a heterotrimeric G protein." [GOC:mah, PMID:7972043]
GO:0070442
name: integrin alphaIIb-beta3 complex
def: "An integrin complex that comprises one alphaIIb subunit and one beta3 subunit." [PMID:12297042]
GO:0070443
name: Mad-Max complex
def: "A transcriptional repressor complex that consists of a heterodimer of the bHLH-ZIP proteins Mad and Max." [PMID:8224841]
GO:0070449
name: elongin complex
def: "A transcription elongation factor complex that suppresses RNA polymerase II pausing, and may act by promoting proper alignment of the 3'-end of nascent transcripts with the polymerase catalytic site. Consists of a transcriptionally active Elongin A subunit (abut 100 kDa)and two smaller Elongin B (about 18 kDa) and Elongin C (about 15 kDa)subunits." [PMID:12676794]
GO:0070450
name: interleukin4-interleukin-4 receptor complex
def: "A protein complex that is formed by the association of a heterodimeric interleukin-4 receptor complex with an interleukin-4 molecule." [GOC:mah, PMID:10358772]
GO:0070451
name: cell hair
def: "A long, thin cell projection that contains F-actin and tubulin, with microtubules centrally located and F-actin peripherally located." [PMID:11526084]
GO:0070461
name: SAGA-type complex
def: "A histone acetyltransferase complex that acetylates nucleosomal H3 and H2B and is required for the expression of a subset of Pol II-transcribed genes. The budding yeast complex includes the acetyltransferase Gcn5p, several proteins of the Spt and Ada families, and several TBP-associate proteins (TAFs); analogous complexes in other species have analogous compositions, and usually contain homologs of the yeast proteins." [GOC:mah, PMID:10637607, PMID:17337012]
GO:0070464
name: alphav-beta3 integrin-collagen alpha3(VI) complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to the alpha3 chain of type VI collagen; the integrin binds most strongly to unfolded collagen." [PMID:8387021]
GO:0070465
name: alpha1-beta1 integrin-alpha3(VI) complex
def: "A protein complex that consists of an alpha1-beta1 integrin complex bound to a type VI collagen triple helix containing an alpha3(VI) chain." [PMID:8387021]
GO:0070466
name: alpha2-beta1 integrin-alpha3(VI) complex
def: "A protein complex that consists of an alpha2-beta1 integrin complex bound to a type VI collagen triple helix containing an alpha3(VI) chain." [PMID:8387021]
GO:0070467
name: RC-1 DNA recombination complex
def: "A protein complex that contains DNA ligase III, DNA polymerase epsilon, a 5'-3' exonuclease, and the SMC1 and SMC2 proteins, and is involved in recombinational repair of deletions and gaps in DNA." [PMID:8392064, PMID:8670910]
GO:0070469
name: respiratory chain
def: "The protein complexes that form the electron transport system (the respiratory chain), associated with a cell membrane, usually the plasma membrane (in prokaryotes) or the inner mitochondrial membrane (on eukaryotes). The respiratory chain complexes transfer electrons from an electron donor to an electron acceptor and are associated with a proton pump to create a transmembrane electrochemical gradient." [GOC:ecd, GOC:mah, ISBN:0198547684]
GO:0070470
name: plasma membrane respiratory chain
def: "A respiratory chain located in the plasma membrane of a cell; made up of the protein complexes that form the electron transport system (the respiratory chain), associated with the plasma membrane. The respiratory chain complexes transfer electrons from an electron donor to an electron acceptor and are associated with a proton pump to create a transmembrane electrochemical gradient." [GOC:curators, GOC:imk, GOC:mah, ISBN:0198547684]
GO:0070477
name: endospore core
def: "An intracellular part that represents the innermost portion of an endospore; the endospore core is dehydrated, enriched in dipicolinic acid and divalent cations, and metabolically inactive." [GOC:mah, PMID:15035041, PMID:18035610]
GO:0070505
name: pollen coat
def: "A layer of extracellular matrix deposited onto the surface of the pollen wall upon disintegration of the tapetal layer of the anther wall in the late stages of pollen development. The composition of this material is highly heterogeneous and includes waxes, lipid droplets, small aromatic molecules, and proteins. The pollen coat is proposed to have many functions, such as holding pollen in the anther until dispersal, facilitation of pollen dispersal, protection of pollen from water loss and UV radiation, and facilitation of adhesion of pollen to the stigma." [GOC:mah, GOC:rph, PMID:12930826, PMID:15012271]
GO:0070514
name: SRF-myogenin-E12 complex
def: "A transcription factor complex that contains the serum response factor (SRF) and the basic helix-loop-helix proteins myogenin and E12, and is involved in activating transcription of muscle-specific genes." [PMID:8617811]
GO:0070515
name: alphaIIb-beta3 integrin-talin complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to talin." [PMID:8663236]
GO:0070516
name: CAK-ERCC2 complex
def: "A protein complex formed by the association of the cyclin-dependent protein kinase activating kinase (CAK) holoenzyme complex with ERCC2." [PMID:8692841, PMID:8692842]
GO:0070517
name: DNA replication factor C core complex
def: "A protein complex containing three of the five subunits of eukaryotic replication factor C, those corresponding to human p40, p38, and p37." [PMID:8692848, PMID:9228079, PMID:9582326]
GO:0070518
name: alpha4-beta1 integrin-CD53 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to membrane protein CD53, a member of the tetraspan family." [PMID:8757325]
GO:0070519
name: alpha4-beta1 integrin-CD63 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to membrane protein CD63, a member of the tetraspan family." [PMID:8757325]
GO:0070520
name: alpha4-beta1 integrin-CD81 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to membrane protein CD81, a member of the tetraspan family." [PMID:10229664, PMID:8757325]
GO:0070521
name: alpha4-beta1 integrin-CD82 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to membrane protein CD82, a member of the tetraspan family." [PMID:8757325]
GO:0070522
name: ERCC4-ERCC1 complex
def: "A heterodimeric nucleotide-excision repair complex that has endonuclease activity specific for bubble structures characteristic of certain DNA lesions. The subunits are known as XPF/ERCC4 and ERCC1 in mammals, and Rad1p and Rad10p in S. cerevisiae." [PMID:14734547]
GO:0070531
name: BRCA1-A complex
def: "A protein complex that contains the BRCA1-BARD1 heterodimer, RAP80/UIMC1, BRCC3/BRCC36, BRE/BRCC45, FAM175A/CCDC98/Abraxas and MERIT40/NBA1, and specifically recognizes and binds K63-linked polyubiquitin chains present on histone H2A and H2AX at DNA damage sites." [GOC:mah, PMID:19261749]
GO:0070532
name: BRCA1-B complex
def: "A protein complex that contains the BRCA1-BARD1 heterodimer, BACH1 and TopBP1, and binds to DNA during S phase at DNA damage sites." [GOC:mah, PMID:16391231]
GO:0070533
name: BRCA1-C complex
def: "A protein complex that contains the BRCA1-BARD1 heterodimer, CtIP and Mre11/Rad50/NBS1 (M/R/N) complex, and binds to DNA at DNA damage sites. BRCA1-C binding ta damaged DNA is required for DNA damage-induced Chk1 phosphorylation and the G2/M transition checkpoint." [GOC:mah, PMID:15485915, PMID:16391231]
GO:0070545
name: PeBoW complex
def: "A protein complex that is involved in coordinating ribosome biogenesis with cell cycle progression. In human, it is composed of Pes1, Bop1, and WDR12; in Saccharomyces the proteins are known as Nop7p, Erb1 and Ytm1 respectively." [GOC:ab, GOC:mah, PMID:16043514, PMID:17353269]
GO:0070552
name: BRISC complex
def: "A protein complex that contains the FAM175B/ABRO1, BRCC3/BRCC36, BRE/BRCC45 and MERIT40/NBA1 proteins, and specifically cleaves K63-linked polyubiquitin chains." [GOC:mah, PMID:19214193]
GO:0070554
name: synaptobrevin 2-SNAP-25-syntaxin-3-complexin complex
def: "A SNARE complex that contains synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 3, and a complexin (or orthologs thereof)." [PMID:8824312]
GO:0070556
name: TAF4B-containing transcription factor TFIID complex
def: "A transcription factor TFIID complex that contains the TBP-associated factor TAF4B (also known as TAFII105 in human), a cell-type-specific variant of TAF4." [GOC:mah, PMID:8858156]
GO:0070557
name: PCNA-p21 complex
def: "A protein complex that contains the cyclin-dependent protein kinase inhibitor p21WAF1/CIP1 bound to PCNA; formation of the complex inhibits DNA replication." [GOC:mah, PMID:7911228, PMID:7915843]
GO:0070558
name: alphaM-beta2 integrin-CD63 complex
def: "A protein complex that consists of an alphaM-beta2 integrin complex bound to membrane protein CD63, a member of the tetraspan family." [PMID:8871662]
GO:0070559
name: lysosomal multienzyme complex
def: "A protein complex found in the lysosome that contains beta-galactosidase, cathepsin A, alpha-neuraminidase and N-acetylgalactosamine-6-sulfate sulfatase, and is involved in glycosaminoglycan catabolism." [GOC:mah, PMID:8910459]
GO:0070565
name: telomere-telomerase complex
def: "A complex of DNA and protein located at the end of a linear chromosome that enables replication of the telomeric repeat sequences at the end of a linear chromosome." [GOC:pde, PMID:19179534]
GO:0070578
name: RISC-loading complex
def: "A trimeric ribonucleoprotein complex that and is required for the formation of a mature RNA induced silencing complex (RISC). In humans the complex is composed of the endonuclease Dicer (DICER1) and TRBP (TARBP2) in association with the Argonaute protein Ago2 (EIF2C2/AGO2). Within the complex, Dicer and TRBP are required to process precursor miRNAs (pre-miRNAs) to mature miRNAs and then load them onto Ago2. Ago2 bound to the mature miRNA constitutes the minimal RISC and may subsequently dissociate from Dicer and TRBP." [GOC:ab, PMID:18178619]
GO:0070603
name: SWI/SNF superfamily-type complex
def: "A protein complex that contains an ortholog of the Saccharomyces ATPase Swi2/Snf2 as one of the core components and mediates assembly of nucleosomes, changes to the spacing or structure of nucleosomes, or some combination of those activities in a manner that requires ATP." [GOC:krc, GOC:mah, PMID:16155938]
GO:0070604
name: PBAF complex
def: "A SWI/SNF-type complex that contains the ATPase product of the mammalian BAF180 gene." [GOC:mah, PMID:16155938, PMID:8895581]
GO:0070618
name: Grb2-Sos complex
def: "A protein complex that contains Grb2 and the guanine nucleotide exchange factor Sos (or an ortholog thereof, such as mSos1), and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7798267, PMID:8940013]
GO:0070619
name: Shc-Grb2-Sos complex
def: "A protein complex that contains Grb2, the adaptor protein Shc and the guanine nucleotide exchange factor Sos (or an ortholog thereof, such as mSos1), and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7970708, PMID:8940013]
GO:0070620
name: EGFR-Grb2-Sos complex
def: "A protein complex that contains the epidermal growth factor receptor (EGFR), Grb2 and the guanine nucleotide exchange factor Sos (or an ortholog thereof, such as mSos1), and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7798267, PMID:8940013]
GO:0070621
name: EGFR-Shc-Grb2-Sos complex
def: "A protein complex that contains the epidermal growth factor receptor (EGFR), Grb2, the adaptor protein SHC and the guanine nucleotide exchange factor Sos (or an ortholog thereof, such as mSos1), and is involved in linking EGFR activation to the p21-Ras pathway." [GOC:mah, PMID:7798267, PMID:8940013]
GO:0070622
name: UDP-N-acetylglucosamine-lysosomal-enzyme N-acetylglucosaminephosphotransferase complex
def: "A protein complex that possesses UDP-N-acetylglucosamine-lysosomal-enzyme N-acetylglucosaminephosphotransferase activity; the bovine complex contains disulfide-linked homodimers of 166- and 51-kDa subunits and two identical, noncovalently associated 56-kDa subunits." [GOC:mah, PMID:8940155]
GO:0070645
name: Ubisch body
def: "A small, granular structure that is found in the extracellular matrix of cell of the secretory tapetal layer that surrounds developing pollen grains. Ubisch bodies have a sporopollenin coat, are attached to the peritapetal wall, and may play a role in pollen development." [GOC:ecd, GOC:mah, PMID:14612572, PMID:16524248]
GO:0070648
name: formin-nucleated actin cable
def: "An actin filament bundle that consists of short filaments organized into bundles of uniform polarity, and is nucleated by formins. In fungal cells, myosin motors transport cargo along actin cables toward sites of polarized cell growth; actin cables may play a similar role in pollen tube growth." [PMID:14671023, PMID:16959963]
GO:0070652
name: HAUS complex
def: "A protein complex that localizes to interphase centrosomes and to mitotic spindle tubules and regulates mitotic spindle assembly and centrosome integrity; in human, the complex consists of eight subunits, some of which are homologous to subunits of the Drosophila Augmin complex." [PMID:19427217]
GO:0070685
name: macropinocytic cup
def: "A cell projection that forms at the site of macropinocytosis, a form of endocytosis that results in the uptake of relatively large amounts of extracellular fluid. The macropinocytic cup membrane selectively excludes certain proteins, such as H36 or PM4C4 in Dictyostelium, and the underlying cytoskeleton is enriched in F-actin and coronin." [PMID:12538772, PMID:16968738, PMID:9044041]
GO:0070686
name: macropinocytic cup membrane
def: "The portion of the plasma membrane surrounding a macropinocytic cup." [GOC:mah]
GO:0070687
name: macropinocytic cup cytoskeleton
def: "The part of the cortical actin cytoskeleton that forms part of a macropinocytic cup." [GOC:mah]
GO:0070688
name: MLL5-L complex
def: "A protein complex that can methylate lysine-4 of histone H3 and plays an essential role in retinoic-acid-induced granulopoiesis. MLL5 is the catalytic methyltransferase subunit, and the complex also contains serine/threonine kinase 38 (STK38), protein phosphatase 1 catalytic subunits, the host cell factor-1 N-terminal subunit, beta-actin, and O-GlcNAc transferase; the human genes encoding the subunits are MLL5, STK38, PPP1CA, PPP1CB, PPP1CC, HCFC1, ACTB and OGT, respectively." [GOC:mah, PMID:19377461]
GO:0070691
name: dimeric positive transcription elongation factor complex b
def: "A positive transcription elongation factor complex b that comprises two subunits; an example is the budding yeast complex containing Svg1p (also called Bur1p) and Bur2p." [GOC:mah, PMID:16721054, PMID:19328067]
GO:0070692
name: trimeric positive transcription elongation factor complex b
def: "A positive transcription elongation factor complex b that comprises three subunits; an example is the budding yeast Ctk complex, which consists of Ctk1p, Ctk2p and Ctk3p." [GOC:mah, PMID:16721054, PMID:19328067]
GO:0070693
name: P-TEFb-cap methyltransferase complex
def: "A protein complex that is formed by the association of positive transcription elongation factor complex b (P-TEFb) with the mRNA capping methyltransferase." [PMID:17332744, PMID:19328067]
GO:0070695
name: FHF complex
def: "A protein complex that is composed of AKTIP/FTS, FAM160A2/p107FHIP, and one or more members of the Hook family of proteins, HOOK1, HOOK2, and HOOK3. The complex is thought to promote vesicle trafficking and/or fusion, and associates with the homotypic vesicular sorting complex (the HOPS complex)." [GOC:ab, GOC:mah, PMID:18799622]
GO:0070701
name: mucus layer
def: "An extracellular region part that consists of a protective layer of mucus secreted by epithelial cells lining tubular organs of the body such as the colon or secreted into fluids such as saliva. Mucus is a viscous slimy secretion consisting of mucins (i.e. highly glycosylated mucin proteins) and various inorganic salts dissolved in water, with suspended epithelial cells and leukocytes." [GOC:krc, GOC:mah, GOC:mm2, http://en.wikipedia.org/wiki/Mucin, PMID:18806221, PMID:19432394]
GO:0070702
name: inner mucus layer
def: "The inner of two mucus layers secreted by epithelial cells in the colon; the inner mucus layer is firmly attached to the epithelium, is densely packed with a compact stratified appearance and is devoid of bacteria." [GOC:mah, GOC:mm2, PMID:18806221, PMID:19432394]
GO:0070703
name: outer mucus layer
def: "The outer of two mucus layers secreted by epithelial cells in the colon; the outer mucus layer is loosely packed and can be colonized by bacteria." [GOC:mah, GOC:mm2, PMID:18806221, PMID:19432394]
GO:0070718
name: alphaPDGFR-SHP-2 complex
def: "A protein complex that contains the platelet-derived growth factor alpha receptor (alphaPDGFR; PDGFRA) and the adaptor protein SHP-2, and is involved signaling via the PDGFR signaling pathway." [GOC:mah, PMID:8943348]
GO:0070719
name: alphaPDGFR-PLC-gamma-1-PI3K-SHP-2 complex
def: "A protein complex that contains the platelet-derived growth factor alpha receptor (alphaPDGFR; PDGFRA), phospholipase C-gamma-1 (PLC-gamma-1), phosphatidylinositol 3-kinase (PI3K) and the adaptor protein SHP-2, and is involved signaling via the PDGFR signaling pathway." [GOC:mah, PMID:8943348]
GO:0070720
name: Grb2-SHP-2 complex
def: "A protein complex that contains the receptor adaptor proteins Grb2 and SHP-2, and is involved signaling via the PDGFR signaling pathway." [GOC:mah, PMID:8943348]
GO:0070721
name: ISGF3 complex
def: "A transcription factor complex that consists of a Stat1-Stat2 heterodimer and the IRF9 protein." [GOC:mah, PMID:8943351]
GO:0070722
name: Tle3-Aes complex
def: "A transcriptional repressor complex that consists of a heterodimer of the proteins Tle3 (also known as Grg3b) and Aes (Grg5), which are homologs of the Drosophila groucho gene product." [GOC:mah, PMID:8955148]
GO:0070724
name: BMP receptor complex
def: "A protein complex that acts as a receptor for bone morphogenetic proteins (BMPs); a homo- or heterodimer of type I and/or type II BMP receptor subunits." [GOC:mah, GOC:mh, PMID:19377468]
GO:0070725
name: Yb body
def: "A cytoplasmic part that appears as an electron-dense sphere of around 1.5 micron diameter containing Yb protein found in somatic cells of ovary and testis. There are one to two Yb bodies per cell." [GOC:sart, PMID:19433453]
GO:0070732
name: spindle envelope
def: "An organelle envelope that surrounds the chromosomes and the central part of the spindle apparatus during mitosis and meiosis; observed in many invertebrates. The spindle envelope consists of membrane layers, called parafusorial membranes, derived from endoplasmic reticulum membrane; in male meiosis it forms during prometaphase and persists until early in the ensuing interphase." [GOC:mah, GOC:sart, PMID:19417004, PMID:6428889]
GO:0070743
name: interleukin-23 complex
def: "A protein complex that is composed of an interleukin-23 alpha (p19, product of the IL23A gene) and an interleukin-12 beta (p40, product of the IL12B gene) subunit and is secreted into the extracellular space." [GOC:add, PMID:11114383, PMID:15999093]
GO:0070744
name: interleukin-27 complex
def: "A protein complex that is composed of an interleukin-27p28 subunit (product of the IL27 gene) and an EBI3 subunit and is secreted into the extracellular space." [GOC:add, PMID:15999093, PMID:19161428]
GO:0070745
name: interleukin-35 complex
def: "A protein complex that is composed of an interleukin-12 alpha subunit (p35, product of the IL12A gene) and an EBI3 subunit and is secreted into the extracellular space." [GOC:add, PMID:19161428, PMID:19161429]
GO:0070761
name: pre-snoRNP complex
def: "A ribonucleoprotein complex that contains a precursor small nucleolar RNA (pre-snoRNA) and associated proteins, and forms during small nucleolar ribonucleoprotein complex (snoRNP) assembly. Pre-snoRNP complexes may contain proteins not found in the corresponding mature snoRNP complexes." [GOC:BHF, GOC:mah, GOC:rl, PMID:17636026, PMID:17709390]
GO:0070762
name: nuclear pore transmembrane ring
def: "A subcomplex of the nuclear pore complex (NPC) that spans the nuclear membrane and anchors the NPC to the nuclear envelope. In S. cerevisiae, the transmembrane ring is composed of Pom152p, Pom34p, and Ndc1p. In vertebrates, it is composed of Gp210, Ndc1, and Pom121. Components are arranged in 8-fold symmetrical 'spokes' around the central transport channel. A single 'spoke', can be isolated and is sometime referred to as the Ndc1 complex." [GOC:dgf, PMID:18046406, PMID:19524430, PMID:20947011, PMID:22419078]
GO:0070763
name: Delta1 complex
def: "A protein complex that consists of homodimer of the Notch ligand Delta1." [PMID:12794186]
GO:0070764
name: gamma-secretase-Delta1 complex
def: "A protein complex that is formed by the association of the Notch ligand Delta1 with the gamma-secretase complex." [PMID:12794186]
GO:0070765
name: gamma-secretase complex
def: "A protein complex that has aspartic-type endopeptidase activity, and contains a catalytic subunit, presenilin (PS), that is a prototypical member of the GxGD-type aspartyl peptidases. The complex also contains additional subunits, including nicastrin, APH-1, PEN-2, and a regulatory subunit, CD147. Gamma-secretase cleaves several transmembrane proteins including the cell surface receptor Notch and the beta-amyloid precursor protein." [GOC:mah, PMID:15286082, PMID:15890777, PMID:17047368]
GO:0070766
name: endobrevin-synaptobrevin 2-alpha-SNAP-NSF-syntaxin-4 complex
def: "A SNARE complex that contains endobrevin (VAMP8), synaptobrevin 2 (VAMP2), alpha-SNAP, NSF, and syntaxin 4 (or orthologs thereof)." [PMID:8973549]
GO:0070767
name: BRCA1-Rad51 complex
def: "A protein complex that contains BRCA1 and Rad 51, and is involved in the control of recombination and of genome integrity." [GOC:mah, PMID:9008167]
GO:0070768
name: synaptotagmin-synaptobrevin 2-SNAP-25-syntaxin-1a-syntaxin-1b-Unc13 complex
def: "A SNARE complex that contains synaptotagmin, synaptobrevin 2 (VAMP2), SNAP-25, syntaxin 1a, syntaxin1b, and Unc13b (or orthologs thereof)." [PMID:8999968]
GO:0070769
name: alphaIIb-beta3 integrin-CIB complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to CIB, a protein that binds calcium as well as the alphaIIb-beta3 integrin." [PMID:9030514]
GO:0070770
name: alphaIIb-beta3 integrin-CD47-FAK complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to the cell surface antigen CD47 and the kinase FAK." [PMID:9169439]
GO:0070771
name: alphaIIb-beta3 integrin-CD47-Src complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to the cell surface antigen CD47 and the kinase c-Src." [PMID:9169439]
GO:0070772
name: PAS complex
def: "A class III phosphatidylinositol 3-kinase complex that contains a phosphatidylinositol-3-phosphate 5-kinase subunit (Fab1p in yeast; PIKfyve in mammals), a kinase activator, and a phosphatase, and may also contain additional proteins; it is involved in regulating the synthesis and turnover of phosphatidylinositol 3,5-bisphosphate. In mammals the complex is composed of PIKFYVE, FIG4 and VAC14. In yeast it is composed of Atg18p, Fig4p, Fab1p, Vac14p and Vac7p." [PMID:18950639, PMID:19037259, PMID:19158662]
GO:0070775
name: H3 histone acetyltransferase complex
def: "A multisubunit complex that catalyzes the acetylation of histone H3." [GOC:mah]
GO:0070776
name: MOZ/MORF histone acetyltransferase complex
def: "A histone acetyltransferase complex that has histone H3 acetyltransferase and coactivator activities. Subunits of the human complex include MYST3/MOZ, MYST4/MORF, ING5, EAF6 and one of BRPF1, BRD1/BRPF2 and BRPF3." [PMID:18794358]
GO:0070820
name: tertiary granule
def: "A secretory granule that contains cathepsin and gelatinase and is readily exocytosed upon cell activation; found primarily in mature neutrophil cells." [GOC:BHF, GOC:mah, GOC:rl, PMID:12070036]
GO:0070821
name: tertiary granule membrane
def: "The lipid bilayer surrounding a tertiary granule." [GOC:BHF, GOC:mah, GOC:rl, PMID:12070036]
GO:0070822
name: Sin3-type complex
def: "Any of a number of evolutionarily conserved histone deacetylase complexes (HDACs) containing a core consisting of a paired amphipathic helix motif protein (e.g. Sin3p in S. cerevisiae, Pst1 in S. pombe or Sin3A in mammals) at least one class I histone deacetylase (e.g. Rpd3p in S. cerevisiae, Clr6 in S. pombe, or HDAC1 and HDAC2 in mammals), and at least one WD40 repeat protein (e.g. Ume1p in S. cerevisiae, Prw1 in S. pombe, or RbAp46 and RbAp48 in mammals). These complexes also contain a variable number of other proteins that direct histone binding, DNA binding, or add other functionality to the complex." [PMID:15565322, PMID:18292778]
GO:0070823
name: HDA1 complex
def: "A tetrameric histone deacetylase complex that contains a Class II deacetylase catalytic subunit. In S. cerevisiae it is composed of two Hda1p subunits along with Hda2p and Hda3p." [GOC:dgf, GOC:mah, PMID:11287668, PMID:8663039]
GO:0070824
name: SHREC complex
def: "A histone deacetylase complex that contains a core of four proteins -- Clr1, Clr2, Clr3, and Mit1 in fission yeast -- and localizes to all heterochromatic regions in the genome as well as some euchromatic sites. The complex is involved in regulating nucleosome positioning to assemble higher-order chromatin structures." [GOC:mah, PMID:17289569]
GO:0070825
name: micropyle
def: "An external encapsulating structure part of the chorion. A single cone-shaped specialization that forms an opening in the chorion that allows sperm entry into the egg prior to fertilization." [GOC:cvs, GOC:mah, PMID:18649270]
GO:0070826
name: paraferritin complex
def: "A cytoplasmic protein complex that contains integrin, mobilferrin and a flavin monooxygenase, is capable of reducing Fe(III) to Fe(II) utilizing NADPH, and is involved in iron transport. Fe(II) is required in the cell as the substrate for ferrochelatase in the synthesis of heme." [GOC:mah, GOC:rph, PMID:11842004, PMID:8639593]
GO:0070847
name: core mediator complex
def: "A protein complex that interacts with the carboxy-terminal domain of the largest subunit of RNA polymerase II and plays an active role in transducing the signal from a transcription factor to the transcriptional machinery. The core mediator complex has a stimulatory effect on basal transcription, and contains most of the same subdomains as the larger form of mediator complex -- a head domain comprising proteins known in Saccharomyces as Srb2, -4, and -5, Med6, -8, and -11, and Rox3 proteins; a middle domain comprising Med1, -4, and -7, Nut1 and -2, Cse2, Rgr1, Soh1, and Srb7 proteins; and a tail consisting of Gal11p, Med2p, Pgd1p, and Sin4p -- but lacks the regulatory subcomplex comprising Ssn2, -3, and -8, and Srb8 proteins. Metazoan core mediator complexes have similar modular structures and include homologs of yeast Srb and Med proteins." [PMID:11454195, PMID:16168358, PMID:17870225]
GO:0070850
name: TACC/TOG complex
def: "A protein complex that contains the transforming acidic coiled coil (TACC) protein and the TOG protein (Mia1p/Alp7p and Alp14, respectively, in fission yeast), and is involved in microtubule array remodeling as cells progress through the cell cycle. The TACC/TOG complex is conserved in eukaryotes, associates with microtubules, and shuttles between the nucleus and the cytoplasm during interphase." [GOC:mah, GOC:vw, PMID:19606211]
GO:0070852
name: cell body fiber
def: "A neuron projection that is found in unipolar neurons and corresponds to the region between the cell body and the point at which the single projection branches." [GOC:dos, GOC:mah]
GO:0070860
name: RNA polymerase I core factor complex
def: "A RNA polymerase I-specific transcription factor complex that is required for the transcription of rDNA by RNA polymerase I. In yeast the complex consists of Rrn6p, Rrn7p, and Rrn11p." [PMID:8702872]
GO:0070864
name: sperm individualization complex
def: "A macromolecular complex that includes cytoskeletal components and part of the cell membrane. Forms at the nuclear end of a male germline syncytium, or cyst, and translocates the over the length of the syncytium in the course of sperm individualization. Each complex contains an array of 64 investment cones, one per nucleus, that move synchronously along the spermatogenic cyst." [GOC:sart, PMID:10588662, PMID:9550716]
GO:0070865
name: investment cone
def: "A cytoskeletal part that consists of a microfilament-rich cone that forms round each nucleus in a spermatogenic cyst and translocates the length of the cyst during sperm individualization." [GOC:sart, PMID:15829565, PMID:9550716]
GO:0070867
name: mating projection tip membrane
def: "The portion of the plasma membrane surrounding a mating projection tip." [GOC:mah]
GO:0070876
name: SOSS complex
def: "A protein complex that functions downstream of the MRN complex to promote DNA repair and the G2/M checkpoint. The SOSS complex associates with single-stranded DNA at DNA lesions and is composed of SOSS-B (SOSS-B1/OBFC2B or SOSS-B2/OBFC2A), SOSS-A/INTS3 and SOSS-C/C9orf80." [PMID:19683501]
GO:0070877
name: microprocessor complex
def: "A protein complex that binds to heme and to pri-miRNAs, and is required for the formation of a pre-microRNA (pre-miRNA), the initial step of microRNA (miRNA) biogenesis. The complex is composed of the double-stranded-RNA-specific RNase Drosha (also called RNASEN) and the RNA-binding protein DGCR8 (heme-free or heme-bound forms). Within the complex, DGCR8 function as a molecular anchor necessary for the recognition of pri-miRNA at dsRNA-ssRNA junction and directs RNASEN/Drosha to cleave the 3' and 5' strands of a stem-loop to release hairpin-shaped pre-miRNAs." [PMID:16963499, PMID:17159994]
GO:0070912
name: Ddb1-Ckn1 complex
def: "A heterodimeric nucleotide-excision repair complex that is involved in transcription-coupled repair. The subunits are known as Ddb1 and Ckn1 in S. pombe; Ddb1 contains a motif called the DDB-box that interacts with adaptor proteins for DDB1/cullin 4 ubiquitin ligases." [PMID:18794354]
GO:0070913
name: Ddb1-Wdr21 complex
def: "A heterodimeric nucleotide-excision repair complex that is involved in transcription-coupled repair. The subunits are known as Ddb1 and Wdr21 in S. pombe; Ddb1 contains a motif called the DDB-box that interacts with adaptor proteins for DDB1/cullin 4 ubiquitin ligases." [PMID:18794354]
GO:0070916
name: inositol phosphoceramide synthase complex
def: "A protein complex that possesses inositol phosphoceramide synthase activity and contains a catalytic subunit and a regulatory subunit (Aur1p and Kei1p, respectively, in Saccharomyces)." [GOC:mah, PMID:19726565]
GO:0070931
name: Golgi-associated vesicle lumen
def: "The volume enclosed by the membrane of a Golgi-associated vesicle." [GOC:mah]
GO:0070937
name: CRD-mediated mRNA stability complex
def: "A protein complex that binds to, and promotes stabilization of, mRNA molecules containing the coding region instability determinant (CRD). In human, IGF2BP1 and at least four additional proteins: HNRNPU, SYNCRIP, YBX1, and DHX9." [GOC:mah, PMID:19029303]
GO:0070938
name: contractile ring
def: "A cytoskeletal structure composed of filamentous protein that forms beneath the membrane of many cells or organelles, in the plane of cell or organelle division. Ring contraction is associated with centripetal growth of the membrane that divides the cytoplasm of the two daughter cells or organelles." [GOC:mah, ISBN:0123645859, ISBN:0792354923, PMID:10791428, PMID:17913889]
GO:0070939
name: Dsl1p complex
def: "A multisubunit tethering complex, i.e. a protein complex involved in mediating the initial interaction between vesicles and the membranes with which they fuse, that is involved in trafficking from the Golgi apparatus to the ER. In Saccharomyces cerevisiae the Dsl1p complex contains Dsl1p, Tip20p, and Sec39p." [GOC:jh, GOC:mah, PMID:19151722]
GO:0070969
name: ULK1-ATG13-FIP200 complex
def: "A protein complex that is essential for mammalian autophagy, and consists of at least 3 proteins: ULK1, ATG13 and RB1CC1(FIP200). This complex is the mammalian counterpart of Atg1-Atg13-Atg17 in yeast." [PMID:19211835, PMID:19258318, PMID:19597335]
GO:0070971
name: endoplasmic reticulum exit site
def: "An endoplasmic reticulum part at which COPII-coated vesicles are produced." [NIF_Subcellular:sao124393998, PMID:15623529, PMID:16957052]
GO:0070985
name: TFIIK complex
def: "A transcription factor complex that forms part of the holo TFIIH complex. In Saccharomyces/human, TFIIK contains Ccl1p/Cyclin H, Tfb3p/MAT1 and Kin2p/CDK7." [GOC:mah, PMID:19818408, PMID:22572993]
GO:0070992
name: translation initiation complex
def: "A ribonucleoprotein complex that contains a ribosome, mRNA, and initiator tRNA; the functional ribosome is at the AUG, with the methionyl/formyl-methionyl-tRNA positioned at the P site." [GOC:hjd, GOC:mah]
GO:0070993
name: translation preinitiation complex
def: "A ribonucleoprotein complex that contains the small ribosomal subunit, a translation initiation ternary complex (i.e. an initiator tRNA, GTP, and an IF2 or eIF2 complex), and an mRNA." [GOC:hjd, GOC:mah]
GO:0071001
name: U4/U6 snRNP
def: "A ribonucleoprotein complex that contains base-paired U4 and U6 small nuclear RNAs." [GOC:mah, PMID:14685174]
GO:0071002
name: U4atac/U6atac snRNP
def: "A ribonucleoprotein complex that contains base-paired U4atac and U6atac small nuclear RNAs." [GOC:mah, PMID:14685174]
GO:0071003
name: penta-snRNP complex
def: "A ribonucleoprotein complex formed by the association of the U1, U2, U4/U6 and U5 small nuclear ribonucleoproteins." [GOC:mah, PMID:11804584, PMID:12724403]
GO:0071004
name: U2-type prespliceosome
def: "A spliceosomal complex that is formed by association of the 5' splice site with the U1 snRNP, while the branch point sequence is recognized by the U2 snRNP. The prespliceosome includes many proteins in addition to those found in the U1 and U2 snRNPs. Commitment to a given pair of 5' and 3' splice sites occurs at the time of prespliceosome formation." [GOC:ab, GOC:krc, GOC:mah, PMID:17332742, PMID:19239890]
GO:0071005
name: U2-type precatalytic spliceosome
def: "A spliceosomal complex that is formed by the recruitment of the preassembled U4/U6.U5 tri-snRNP to the prespliceosome. Although all 5 snRNPs are present, the precatalytic spliceosome is catalytically inactive. The precatalytic spliceosome includes many proteins in addition to those found in the U1, U2 and U4/U6.U5 snRNPs." [GOC:ab, GOC:krc, GOC:mah, PMID:18322460, PMID:19239890]
GO:0071006
name: U2-type catalytic step 1 spliceosome
def: "A spliceosomal complex that is formed by the displacement of the U1 and U4 snRNPs from the precatalytic spliceosome; the U2, U5 and U6 snRNPs remain associated with the mRNA. This complex, sometimes called the activated spliceosome, is the catalytically active form of the spliceosome, and includes many proteins in addition to those found in the U2, and U5 and U6 snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:18322460, PMID:19239890]
GO:0071007
name: U2-type catalytic step 2 spliceosome
def: "A spliceosomal complex that contains the U2, U5 and U6 snRNPs bound to a splicing intermediate in which the first catalytic cleavage of the 5' splice site has occurred. The precise subunit composition differs significantly from that of the catalytic step 1, or activated, spliceosome, and includes many proteins in addition to those found in the U2, U5 and U6 snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:18322460, PMID:19239890]
GO:0071008
name: U2-type post-mRNA release spliceosomal complex
def: "A spliceosomal complex that is formed following the release of the spliced product from the post-spliceosomal complex and contains the excised intron and the U2, U5 and U6 snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:19239890]
GO:0071009
name: U4atac/U6atac x U5 tri-snRNP complex
def: "A ribonucleoprotein complex formed by the association of the U4atac/U6atac and U5 small nuclear ribonucleoproteins." [GOC:krc, GOC:mah, GOC:pr, PMID:16201866]
GO:0071010
name: prespliceosome
def: "A spliceosomal complex that is formed by association of the 5' splice site and the branch point sequence with specific snRNPs. The prespliceosome includes many proteins in addition to those found in the bound snRNPs. Commitment to a given pair of 5' and 3' splice sites occurs at the time of prespliceosome formation. Prespliceosome complexes are not active for splicing, but are instead an early step in the assembly of a spliceosomal complex." [GOC:ab, GOC:krc, GOC:mah, PMID:17332742, PMID:19239890]
GO:0071011
name: precatalytic spliceosome
def: "A spliceosomal complex that is formed by the recruitment of a preassembled U5-containing tri-snRNP to the prespliceosome. Although all 5 snRNPs are present, the precatalytic spliceosome is catalytically inactive. The precatalytic spliceosome includes many proteins in addition to those found in the associated snRNPs." [GOC:ab, GOC:krc, GOC:mah, PMID:18322460, PMID:19239890]
GO:0071012
name: catalytic step 1 spliceosome
def: "A spliceosomal complex that is formed by the displacement of the two snRNPs from the precatalytic spliceosome; three snRNPs including U5 remain associated with the mRNA. This complex, sometimes called the activated spliceosome, is the catalytically active form of the spliceosome, and includes many proteins in addition to those found in the associated snRNPs." [GOC:ab, GOC:krc, GOC:mah, PMID:18322460, PMID:19239890]
GO:0071013
name: catalytic step 2 spliceosome
def: "A spliceosomal complex that contains three snRNPs, including U5, bound to a splicing intermediate in which the first catalytic cleavage of the 5' splice site has occurred. The precise subunit composition differs significantly from that of the catalytic step 1, or activated, spliceosome, and includes many proteins in addition to those found in the associated snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:18322460, PMID:19239890]
GO:0071014
name: post-mRNA release spliceosomal complex
def: "A spliceosomal complex that is formed following the release of the spliced product from the post-spliceosomal complex and contains the excised intron and three snRNPs, including U5." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:19239890]
GO:0071015
name: U12-type prespliceosome
def: "A spliceosomal complex that is formed by the cooperative binding of the heterodimeric U11/U12 snRNP to the 5' splice site and the branch point sequence. The U12-type prespliceosome includes many proteins in addition to those found in the U11/U12 heterodimeric snRNPs. Commitment to a given pair of 5' and 3' splice sites occurs at the time of prespliceosome formation." [GOC:ab, GOC:krc, GOC:mah, PMID:10197985, PMID:16201866]
GO:0071016
name: U12-type precatalytic spliceosome
def: "A spliceosomal complex that is formed by the recruitment of the preassembled U4atac/U6atac.U5 tri-snRNP to the U12-type prespliceosome. Although all 5 snRNPs are present, the precatalytic spliceosome is catalytically inactive. The precatalytic spliceosome includes many proteins in addition to those found in the U11, U12 and U4atac/U6atac.U5 snRNPs." [GOC:ab, GOC:krc, GOC:mah, PMID:16201866]
GO:0071017
name: U12-type catalytic step 1 spliceosome
def: "A spliceosomal complex that is formed by the displacement of the U11 and U4atac snRNPs from the precatalytic spliceosome; the U12, U5 and U6atac snRNPs remain associated with the mRNA. This complex, sometimes called the activated spliceosome, is the catalytically active form of the spliceosome, and includes many proteins in addition to those found in the U12, and U5 and U6atac snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:16201866]
GO:0071018
name: U12-type catalytic step 2 spliceosome
def: "A spliceosomal complex that contains the U12, U5 and U6atac snRNPs bound to a splicing intermediate in which the first catalytic cleavage of the 5' splice site has occurred. The precise subunit composition differs significantly from that of the catalytic step 1, or activated, spliceosome, and includes many proteins in addition to those found in the U12, U5 and U6atac snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:16201866]
GO:0071019
name: U12-type post-mRNA release spliceosomal complex
def: "A spliceosomal complex that is formed following the release of the spliced product from the post-spliceosomal complex and contains the excised intron and the U12, U5 and U6atac snRNPs." [GOC:ab, GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393, PMID:16201866]
GO:0071020
name: post-spliceosomal complex
def: "A spliceosomal complex that is formed following the second splicing event and contains the spliced product, the excised intron, and three snRNPs, including U5." [GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393]
GO:0071021
name: U2-type post-spliceosomal complex
def: "A spliceosomal complex that is formed following the second splicing event and contains the spliced product, the excised intron, and three snRNPs, including U5." [GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393]
GO:0071022
name: U12-type post-spliceosomal complex
def: "A spliceosomal complex that is formed following the second splicing event and contains the spliced product, the excised intron, and three snRNPs, including U5." [GOC:krc, GOC:mah, ISBN:0879695897, ISBN:0879697393]
GO:0071023
name: trans spliceosomal complex
def: "A spliceosomal complex that forms during the addition of a specific spliced leader (SL) sequence to the 5'-end of a messenger RNA primary transcript, a process which occurs in a number of eukaryotic organisms, including trypanosomatid protozoans, euglenoids, nematodes, trematodes, and chordates." [GOC:krc, ISBN:0879697393]
GO:0071024
name: SL snRNP
def: "A ribonucleoprotein complex that contains spliced leader (SL) RNA." [GOC:krc, ISBN:0879697393]
GO:0071052
name: alpha9-beta1 integrin-ADAM1 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM1." [PMID:11882657]
GO:0071053
name: alpha9-beta1 integrin-ADAM2 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM2." [PMID:11882657]
GO:0071054
name: alpha9-beta1 integrin-ADAM3 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM3." [PMID:11882657]
GO:0071055
name: alpha9-beta1 integrin-ADAM9 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM9." [PMID:11882657]
GO:0071056
name: alpha9-beta1 integrin-ADAM15 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM15." [PMID:11882657]
GO:0071057
name: alphav-beta3 integrin-ADAM15 complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to the transmembrane metallopeptidase ADAM15." [PMID:10944520, PMID:11882657]
GO:0071058
name: alpha3-beta1 integrin-CD151 complex
def: "A protein complex that consists of an alpha3-beta1 integrin complex bound to the tetraspanin CD151." [PMID:10811835, PMID:11884516]
GO:0071059
name: alpha6-beta1 integrin-CD151 complex
def: "A protein complex that consists of an alpha6-beta1 integrin complex bound to the tetraspanin CD151." [PMID:11884516]
GO:0071060
name: alpha7-beta1 integrin-CD151 complex
def: "A protein complex that consists of an alpha7-beta1 integrin complex bound to the tetraspanin CD151." [PMID:11884516]
GO:0071061
name: alpha6-beta4 integrin-CD151 complex
def: "A protein complex that consists of an alpha6-beta4 integrin complex bound to the tetraspanin CD151." [PMID:10811835]
GO:0071062
name: alphav-beta3 integrin-vitronectin complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to vitronectin." [PMID:10835423]
GO:0071064
name: alphaE-beta7 integrin-E-cadherin complex
def: "A protein complex that consists of an alphaE-beta7 integrin complex bound to E-cadherin." [PMID:10837471]
GO:0071065
name: alpha9-beta1 integrin-vascular cell adhesion molecule-1 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to vascular cell adhesion molecule-1." [PMID:10209034]
GO:0071067
name: alphav-beta3 integrin-ADAM23 complex
def: "A protein complex that consists of an alphav-beta3 integrin complex bound to the transmembrane metallopeptidase ADAM23." [PMID:10749942]
GO:0071068
name: alpha9-beta1 integrin-ADAM12 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM12." [PMID:10944520]
GO:0071069
name: alpha4-beta1 integrin-thrombospondin-1 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to thrombospondin-1." [PMID:11980922]
GO:0071070
name: alpha4-beta1 integrin-thrombospondin-2 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to thrombospondin-2." [PMID:11980922]
GO:0071075
name: CUGBP1-eIF2 complex
def: "A protein complex that contains the eukaryotic translation initiation factor 2 complex (EIF2), CUG binding protein 1, and several endoplasmic reticulum proteins; the complex is involved in the regulation of translation." [PMID:16931514]
GO:0071078
name: fibronectin-tissue transglutaminase complex
def: "A protein complex that consists of fibronectin bound to tissue transglutaminase, and is involved in cell adhesion." [PMID:10684262]
GO:0071079
name: alpha2-beta1 integrin-chondroadherin complex
def: "A protein complex that consists of an alpha2-beta1 integrin complex bound to the cartilage matrix protein chondroadherin." [PMID:9281592]
GO:0071080
name: alpha3-beta1 integrin-basigin complex
def: "A protein complex that consists of an alpha3-beta1 integrin complex bound to the cell surface protein basigin." [PMID:9360995]
GO:0071081
name: alpha3-beta1 integrin-CD63 complex
def: "A protein complex that consists of an alpha3-beta1 integrin complex bound to the tetraspanin CD63." [PMID:7629079]
GO:0071082
name: alpha9-beta1 integrin-tenascin complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the extracellular matrix protein tenascin." [PMID:9565552]
GO:0071083
name: alphaV-beta3 integrin-CD47-FCER2 complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to the cell surface protein CD47 and the low-affinity immunoglobulin epsilon Fc receptor (FCER2)." [PMID:10037797]
GO:0071084
name: alpha2-beta1 integrin-CD47 complex
def: "A protein complex that consists of an alpha2-beta1 integrin complex bound to the cell surface protein CD47." [PMID:10397731]
GO:0071085
name: alphaIIb-beta3 integrin-CD9 complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to the cell surface protein CD9." [PMID:10429193]
GO:0071086
name: alphaIIb-beta3 integrin-CD9-CD47-platelet glycoprotein Ib complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to the cell surface proteins CD9 and CD47, and the heterodimeric platelet glycoprotein Ib." [PMID:10429193]
GO:0071087
name: alpha11-beta1 integrin-collagen type I complex
def: "A protein complex that consists of an alpha11-beta1 integrin complex bound to a type I collagen." [PMID:10464311]
GO:0071088
name: alpha5-beta1 integrin-tissue transglutaminase complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to tissue transglutaminase." [PMID:10684262]
GO:0071089
name: alphaV-beta3 integrin-tissue transglutaminase complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to tissue transglutaminase." [PMID:10684262]
GO:0071090
name: alphaIIb-beta3 integrin-fibronectin-tissue transglutaminase complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to fibronectin and tissue transglutaminase." [PMID:10684262]
GO:0071091
name: alpha1-beta1 integrin-tissue transglutaminase complex
def: "A protein complex that consists of an alpha1-beta1 integrin complex bound to tissue transglutaminase." [PMID:10684262]
GO:0071092
name: alpha3-beta1 integrin-tissue transglutaminase complex
def: "A protein complex that consists of an alpha3-beta1 integrin complex bound to tissue transglutaminase." [PMID:10684262]
GO:0071093
name: alpha5-beta1 integrin-fibronectin-tissue transglutaminase complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to fibronectin and tissue transglutaminase." [PMID:10684262]
GO:0071094
name: alpha6-beta4 integrin-CD9 complex
def: "A protein complex that consists of an alpha6-beta4 integrin complex bound to the cell surface protein CD9." [PMID:10711425]
GO:0071095
name: alpha3-beta1 integrin-thrombospondin complex
def: "A protein complex that consists of an alpha3-beta1 integrin complex bound to thrombospondin." [PMID:11358957]
GO:0071096
name: alphaV-beta3 integrin-gelsolin complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to gelsolin." [PMID:11577104]
GO:0071097
name: alphaV-beta3 integrin-paxillin-Pyk2 complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to paxillin and the FAK-related kinase Pyk2." [PMID:11683411]
GO:0071098
name: alpha6-beta4 integrin-Fyn complex
def: "A protein complex that consists of an alpha6-beta4 integrin complex bound to the Src family tyrosine kinase Fyn." [PMID:11684709]
GO:0071099
name: alphaV-beta6 integrin-TGFbeta-3 complex
def: "A protein complex that consists of an alphaV-beta6 integrin complex bound to transforming growth factor beta-3 (TGFbeta-3)." [PMID:11821050]
GO:0071100
name: alphaV-beta8 integrin-MMP14-TGFbeta-1 complex
def: "A protein complex that consists of an alphaV-beta8 integrin complex bound to matrix metalloproteinase 14 and transforming growth factor beta-1 (TGFbeta-1)." [PMID:11970960]
GO:0071101
name: alpha4-beta1 integrin-JAM2 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to the cell adhesion molecule JAM2." [PMID:12070135]
GO:0071102
name: alpha4-beta1 integrin-paxillin complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to paxillin." [PMID:12221126]
GO:0071112
name: alpha4-beta4 integrin-EMILIN-1 complex
def: "A protein complex that consists of an alpha4-beta4 integrin complex bound to EMILIN-1 (ElastinMicrofibril Interface Located ProteIN)." [PMID:12456677]
GO:0071113
name: alphaIIb-beta3 integrin-ICAM-4 complex
def: "A protein complex that consists of an alphaIIb-beta3 integrin complex bound to the cell adhesion molecule ICAM-4." [PMID:12477717]
GO:0071114
name: alphaV-beta3 integrin-tumstatin complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to tumstatin, the NC1 domain of the alpha3 chain of type IV collagen." [PMID:12682293]
GO:0071115
name: alpha5-beta1 integrin-endostatin complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to endostatin, the NC1 domain of the alpha1 chain of type XVIII collagen." [PMID:12682293]
GO:0071116
name: alpha6-beta1 integrin-CYR61 complex
def: "A protein complex that consists of an alpha6-beta1 integrin complex bound to CYR61, a cysteine-rich protein involved in angiogenesis." [PMID:12826661]
GO:0071117
name: alpha5-beta1 integrin-fibronectin-NOV complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to fibronectin and the extracellular matrix protein NOV." [PMID:12902636]
GO:0071118
name: alphaV-beta3 integrin-NOV complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to the extracellular matrix protein NOV." [PMID:12902636]
GO:0071119
name: alpha7-beta1 integrin-nicotinamide riboside kinase complex
def: "A protein complex that consists of an alpha7-beta1 integrin complex bound to nicotinamide riboside kinase 2 (also known as muscle integrin binding protein, MIBP)." [PMID:12941630]
GO:0071120
name: alpha4-beta1 integrin-CD47 complex
def: "A protein complex that consists of an alpha4-beta1 integrin complex bound to the cell surface antigen CD47." [PMID:15292185]
GO:0071121
name: alpha9-beta1 integrin-VEGF-D complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to vascular endothelial growth factor D." [PMID:15590642]
GO:0071122
name: alpha9-beta1 integrin-VEGF-A complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to vascular endothelial growth factor A." [PMID:17363377]
GO:0071123
name: alpha9-beta1 integrin-VEGF-C complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to vascular endothelial growth factor C." [PMID:15590642]
GO:0071124
name: alpha1-beta1 integrin-tyrosine-protein phosphatase non-receptor type 2 complex
def: "A protein complex that consists of an alpha1-beta1 integrin complex bound to tyrosine-protein phosphatase non-receptor type 2." [PMID:15592458]
GO:0071125
name: alphaV-beta3 integrin-EGFR complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to epidermal growth factor receptor." [PMID:15834425]
GO:0071126
name: alphaV-beta6 integrin-osteopontin complex
def: "A protein complex that consists of an alphaV-beta6 integrin complex bound to osteopontin." [PMID:16005200]
GO:0071127
name: alpha9-beta1 integrin-osteopontin complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to osteopontin." [PMID:16005200]
GO:0071128
name: alpha5-beta1 integrin-osteopontin complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to osteopontin." [PMID:16005200]
GO:0071129
name: alphaV-beta3 integrin-LPP3 complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to lipid phosphate phosphohydrolase-3." [PMID:16099422]
GO:0071130
name: alpha5-beta1 integrin-LPP3 complex
def: "A protein complex that consists of an alpha5-beta1 integrin complex bound to lipid phosphate phosphohydrolase-3." [PMID:16099422]
GO:0071131
name: alphaV-beta3 integrin-laminin alpha-4 complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to laminin alpha-4." [PMID:16824487]
GO:0071132
name: alphaX-beta2 integrin-ICAM-4 complex
def: "A protein complex that consists of an alphaX-beta2 integrin complex bound to intercellular adhesion molecule 4." [PMID:16985175]
GO:0071133
name: alpha9-beta1 integrin-ADAM8 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to the transmembrane metallopeptidase ADAM8." [PMID:16995821]
GO:0071134
name: alpha9-beta1 integrin-thrombospondin-1 complex
def: "A protein complex that consists of an alpha9-beta1 integrin complex bound to thrombospondin-1." [PMID:17413041]
GO:0071135
name: alpha7-beta1 integrin-focal adhesion kinase complex
def: "A protein complex that consists of an alpha7-beta1 integrin complex bound to focal adhesion kinase." [PMID:17598176]
GO:0071136
name: alpha7-beta1 integrin-laminin alpha-2 complex
def: "A protein complex that consists of an alpha7-beta1 integrin complex bound to laminin alpha-2." [PMID:17598176]
GO:0071137
name: alphaV-beta3 integrin-CD98 complex
def: "A protein complex that consists of an alphaV-beta3 integrin complex bound to the cell surface antigen CD98." [PMID:18032696]
GO:0071138
name: alpha5-beta5-fibronectin-SFRP2 complex
def: "A protein complex that consists of an alpha5-beta5 integrin complex bound to fibronectin and secreted frizzled-related protein 2." [PMID:14709558]
GO:0071141
name: SMAD protein complex
def: "A protein complex that consists of SMAD proteins; may be homomeric or heteromeric." [GOC:mah, PMID:9670020]
GO:0071142
name: SMAD2 protein complex
def: "A protein complex that consists of a SMAD2 homotrimer." [GOC:mah, PMID:9670020]
GO:0071143
name: SMAD3 protein complex
def: "A protein complex that consists of a SMAD3 homotrimer." [GOC:mah, PMID:9670020]
GO:0071144
name: SMAD2-SMAD3 protein complex
def: "A heteromeric SMAD protein complex that contains SMAD2 and SMAD3." [GOC:mah, PMID:9670020]
GO:0071145
name: SMAD2-SMAD4 protein complex
def: "A heteromeric SMAD protein complex that contains SMAD2 and SMAD4." [GOC:mah, PMID:9670020]
GO:0071146
name: SMAD3-SMAD4 protein complex
def: "A heteromeric SMAD protein complex that contains SMAD3 and SMAD4." [GOC:mah, PMID:9670020]
GO:0071147
name: TEAD-2 multiprotein complex
def: "A protein complex that consists of the DNA binding protein TEAD-2 bound to 12 other polypeptides including the transcriptional coactivator YAP, the multi-PDZ domain protein MPDZ (also called MUPP1), a 14-3-3 domain protein, and others." [CORUM:2870, GOC:mah, PMID:11358867]
GO:0071148
name: TEAD-1-YAP complex
def: "A transcription factor complex that is composed of the DNA binding protein TEAD-1 and the transcriptional coactivator YAP." [GOC:mah, PMID:11358867]
GO:0071149
name: TEAD-2-YAP complex
def: "A transcription factor complex that is composed of the DNA binding protein TEAD-2 and the transcriptional coactivator YAP." [GOC:mah, PMID:11358867]
GO:0071150
name: TEAD-3-YAP complex
def: "A transcription factor complex that is composed of the DNA binding protein TEAD-3 and the transcriptional coactivator YAP." [GOC:mah, PMID:11358867]
GO:0071151
name: TEAD-4-YAP complex
def: "A transcription factor complex that is composed of the DNA binding protein TEAD-4 and the transcriptional coactivator YAP." [GOC:mah, PMID:11358867]
GO:0071152
name: G-protein alpha(q)-synembrin complex
def: "A protein complex formed by the association of the guanine nucleotide exchange factor synembrin with the alpha(q) subunit of a heterotrimeric G protein." [GOC:mah, PMID:12509430]
GO:0071153
name: G-protein alpha(o)-synembrin complex
def: "A protein complex formed by the association of the guanine nucleotide exchange factor synembrin with the alpha(o) subunit of a heterotrimeric G protein." [GOC:mah, PMID:12509430]
GO:0071154
name: G-protein alpha(i)1-synembrin complex
def: "A protein complex formed by the association of the guanine nucleotide exchange factor synembrin with the alpha(i)1 subunit of a heterotrimeric G protein." [GOC:mah, PMID:12509430]
GO:0071155
name: G-protein alpha(13)-synembrin complex
def: "A protein complex formed by the association of the guanine nucleotide exchange factor synembrin with the alpha(13) subunit of a heterotrimeric G protein." [GOC:mah, PMID:12509430]
GO:0071159
name: NF-kappaB complex
def: "A protein complex that consists of a homo- or heterodimer of members of a family of structurally related proteins that contain a conserved N-terminal region called the Rel homology domain (RHD). In the nucleus, NF-kappaB complexes act as transcription factors. In unstimulated cells, NF-kappaB dimers are sequestered in the cytoplasm by IkappaB monomers; signals that induce NF-kappaB activity cause degradation of IkappaB, allowing NF-kappaB dimers to translocate to the nucleus and induce gene expression." [ISBN:0849327946]
GO:0071162
name: CMG complex
def: "A protein complex that contains the GINS complex, Cdc45p, and the heterohexameric MCM complex, and that is involved in unwinding DNA during replication." [GOC:rb, PMID:19228417]
GO:0071175
name: MAML2-RBP-Jkappa-ICN1 complex
def: "A protein complex that consists of the intracellular domain of Notch1 (ICN1), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-2 (MAML2); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071176
name: MAML2-RBP-Jkappa-ICN2 complex
def: "A protein complex that consists of the intracellular domain of Notch2 (ICN2), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-2 (MAML2); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071177
name: MAML2-RBP-Jkappa-ICN3 complex
def: "A protein complex that consists of the intracellular domain of Notch3 (ICN3), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-2 (MAML2); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071178
name: MAML2-RBP-Jkappa-ICN4 complex
def: "A protein complex that consists of the intracellular domain of Notch4 (ICN4), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-2 (MAML2); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071179
name: MAML3-RBP-Jkappa-ICN1 complex
def: "A protein complex that consists of the intracellular domain of Notch1 (ICN1), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-3 (MAML3); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071180
name: MAML3-RBP-Jkappa-ICN2 complex
def: "A protein complex that consists of the intracellular domain of Notch2 (ICN2), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-3 (MAML3); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071181
name: MAML3-RBP-Jkappa-ICN3 complex
def: "A protein complex that consists of the intracellular domain of Notch3 (ICN3), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-3 (MAML3); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071182
name: MAML3-RBP-Jkappa-ICN4 complex
def: "A protein complex that consists of the intracellular domain of Notch4 (ICN4), the DNA-binding transcription factor RBP-Jkappa, and the transcriptional coactivator Mastermind-like-3 (MAML3); the complex is involved in transcriptional activation in response to Notch-mediated signaling." [PMID:12370315]
GO:0071183
name: protocadherin-alpha-protocadherin-gamma complex
def: "A protein complex that contains two cell adhesion molecules, a protocadherin-alpha and a protocadherin-gamma, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071184
name: protocadherin-alpha-v4-protocadherin-gamma-a1 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v4 and protocadherin-gamma-a1, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071185
name: protocadherin-alpha-v4-protocadherin-gamma-a3 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v4 and protocadherin-gamma-a3, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071186
name: protocadherin-alpha-v4-protocadherin-gamma-b2 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v4 and protocadherin-gamma-b2, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071187
name: protocadherin-alpha-v4-protocadherin-gamma-b4 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v4 and protocadherin-gamma-b4, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071188
name: protocadherin-alpha-v7-protocadherin-gamma-a1 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v7 and protocadherin-gamma-a1, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071189
name: protocadherin-alpha-v7-protocadherin-gamma-a3 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v7 and protocadherin-gamma-a3, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071190
name: protocadherin-alpha-v7-protocadherin-gamma-b2 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v7 and protocadherin-gamma-b2, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071191
name: protocadherin-alpha-v7-protocadherin-gamma-b4 complex
def: "A protein complex that contains the cell adhesion molecules protocadherin-alpha-v7 and protocadherin-gamma-b4, and is involved in the regulation of protein localization to the plasma membrane." [PMID:15347688]
GO:0071192
name: Kv4.2-KChIP1 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv channel interacting protein KChIP1 associated with the channel via interaction with the Kv alpha subunit 4.2." [PMID:15356203]
GO:0071193
name: Kv4.2-KChIP2 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv channel interacting protein KChIP2 associated with the channel via interaction with the Kv alpha subunit 4.2." [PMID:15356203]
GO:0071194
name: Kv4.2-KChIP3 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv channel interacting protein KChIP3 associated with the channel via interaction with the Kv alpha subunit 4.2." [PMID:15356203]
GO:0071195
name: Kv4.2-KChIP4 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv channel interacting protein KChIP4 associated with the channel via interaction with the Kv alpha subunit 4.2." [PMID:15356203]
GO:0071196
name: Kv4.3-KChIP1 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv channel interacting protein KChIP1 associated with the channel via interaction with the Kv alpha subunit 4.3." [PMID:15356203]
GO:0071197
name: Kv4.2-Kv4.3 channel complex
def: "A voltage-gated potassium channel complex that contains the Kv alpha subunits 4.2 and 4.3." [PMID:15356203]
GO:0071198
name: Kv4.1-DPP6 channel complex
def: "A voltage-gated potassium channel complex that contains the peptidase-related protein DPP6 associated with the channel via interaction with the Kv alpha subunit 4.1." [PMID:15911355]
GO:0071199
name: Kv4.1-DPP10 channel complex
def: "A voltage-gated potassium channel complex that contains the peptidase-related protein DPP10 associated with the channel via interaction with the Kv alpha subunit 4.1." [PMID:15911355]
GO:0071200
name: Kv4.2-DPP6 channel complex
def: "A voltage-gated potassium channel complex that contains the peptidase-related protein DPP6 associated with the channel via interaction with the Kv alpha subunit 4.2." [PMID:12575952, PMID:15911355]
GO:0071201
name: Kv4.3-DPP6 channel complex
def: "A voltage-gated potassium channel complex that contains the peptidase-related protein DPP6 associated with the channel via interaction with the Kv alpha subunit 4.3." [PMID:12575952, PMID:15911355]
GO:0071202
name: Kv4.3-DPP10 channel complex
def: "A voltage-gated potassium channel complex that contains the peptidase-related protein DPP10 associated with the channel via interaction with the Kv alpha subunit 4.3." [PMID:15911355]
GO:0071203
name: WASH complex
def: "A protein complex that localizes at the surface of endosomes, where it recruits and activates the Arp2/3 complex to induce actin polymerization. In human, the WASH complex is composed of F-actin-capping protein subunits alpha and beta, WASH1, FAM21, KIAA1033, KIAA0196 and CCDC53." [GOC:sp, PMID:19922875]
GO:0071204
name: histone pre-mRNA 3'end processing complex
def: "A ribonucleoprotein that binds to specific sites in, and is required for cleavage of, the 3'-end of histone pre-mRNAs. The complex contains the U7 snRNP and additional proteins, including the stem-loop binding protein (SLBP) and the exonuclease 3'hExo/Eri-1." [GOC:mah, PMID:19470752]
GO:0071212
name: subsynaptic reticulum
def: "An elaborate tubulolamellar membrane system that underlies the postsynaptic cell membrane." [PMID:1460464, PMID:18171947, PMID:19244343, PMID:7946331]
GO:0071254
name: cytoplasmic U snRNP body
def: "A ribonucleoprotein complex that can be visualized as a focus in the cytoplasm, and contains uridine-rich small nuclear ribonucleoproteins (U snRNPs) and essential snRNP assembly factors. These U bodies are invariably found in association with P bodies." [GOC:sart, PMID:17595295]
GO:0071256
name: translocon complex
def: "A protein complex that constitutes a specific site of protein translocation across the endoplasmic reticulum, which involves the signal recognition particle receptor. The complex contains a core heterotrimer of alpha, beta and gamma subunits, and may contain additional proteins." [GOC:mah, PMID:10611978, PMID:18166647, PMID:8612571]
GO:0071261
name: Ssh1 translocon complex
def: "A translocon complex that contains a core heterotrimer of alpha, beta and gamma subunits, and may contain additional proteins (translocon-associated proteins or TRAPs); in budding yeast the core proteins are Ssh1p, Sbh2p, and Sss1p. The Ssh1 translocon complex is involved in the cotranslational pathway of protein transport across the ER membrane, and recognizes proteins bearing strongly hydrophobic signal sequences." [GOC:mah, PMID:12134063, PMID:8612571]
GO:0071339
name: MLL1 complex
def: "A protein complex that can methylate lysine-4 of histone H3. MLL1/MLL is the catalytic methyltransferase subunit, and the complex also contains the core components ASH2L, HCFC1/HCF1 WDR5 and RBBP5." [GOC:sp, PMID:15960975]
GO:0071341
name: medial cortical node
def: "A protein complex that contains the mid1, cdr2, wee1, klp8, and blt1 proteins, and is involved in contractile ring localization. Medial cortical node complexes appear as cortical dots in the middle of the cell during interphase, and function to recruit other ring components in early mitosis." [GOC:mah, GOC:vw, PMID:19474789, PMID:19959363]
GO:0071437
name: invadopodium
def: "A cell projection that emerges from the ECM-facing surface of a cell, is enriched in actin and associated cytoskeletal proteins, and displays localized proteolytic activity toward the substrate." [GOC:mah, PMID:16651416, PMID:19491051, PMID:19931459]
GO:0071438
name: invadopodium membrane
def: "The portion of the plasma membrane surrounding an invadopodium." [GOC:mah]
GO:0071439
name: clathrin complex
def: "A protein complex that consists of three clathrin heavy chains and three clathrin light chains, organized into a symmetrical three-legged structure called a triskelion. In clathrin-coated vesicles clathrin is the main component of the coat and forms a polymeric mechanical scaffold on the vesicle surface." [GOC:mah, PMID:16493411]
GO:0071458
name: integral component of cytoplasmic side of endoplasmic reticulum membrane
def: "The component of the endoplasmic reticulum membrane consisting of the gene products that penetrate only the cytoplasmic side of the membrane." [GOC:dos, GOC:mah]
GO:0071513
name: phosphopantothenoylcysteine decarboxylase complex
def: "A protein complex that catalyzes decarboxylation of 4'-phosphopantothenoylcysteine to yield 4'-phosphopantetheine; this is the third step in the biosynthesis of Coenzyme A. The complex is homotrimeric in many eukaryotes, but is a heterotrimer in Saccharomyces." [GOC:jh, PMID:19915539]
GO:0071521
name: Cdc42 GTPase complex
def: "A protein complex formed by the association of the small GTPase Cdc42 with additional proteins. In Schizosaccharomyces the complex contains the Cdc42, Ras1, Scd1, Scd2, andShk1 proteins, and functions in the Ras1-Scd GTPase signalling pathway." [GOC:mah, GOC:vw, PMID:10567532, PMID:7923372, PMID:8943016]
GO:0071540
name: eukaryotic translation initiation factor 3 complex, eIF3e
def: "An eukaryotic translation initiation factor 3 complex that contains the PCI-domain protein eIF3e." [PMID:15904532, PMID:19061185]
GO:0071541
name: eukaryotic translation initiation factor 3 complex, eIF3m
def: "An eukaryotic translation initiation factor 3 complex that contains the PCI-domain protein eIF3m." [PMID:15904532, PMID:19061185]
GO:0071546
name: pi-body
def: "A P granule that contains the PIWIL2-TDRD1 module, a set of proteins that act in the primary piRNA pathway. The pi-body corresponds to the cementing material between mitochondria found in gonocytes." [GOC:sp, PMID:20011505]
GO:0071547
name: piP-body
def: "A P granule that contains the PIWIL4-TDRD9 module, a set of proteins that act in the secondary piRNA pathway." [GOC:sp, PMID:20011505]
GO:0071556
name: integral component of lumenal side of endoplasmic reticulum membrane
def: "The component of the endoplasmic reticulum membrane consisting of the gene products that penetrate only the lumenal side of the membrane." [GOC:dos, GOC:mah]
GO:0071561
name: nucleus-vacuole junction
def: "An organelle membrane contact site formed between the vacuole membrane and the outer nuclear membrane. In S. cerevisiae these contacts are mediated through direct physical interaction between Vac8p and Nvj1p." [GOC:jp, PMID:16709156, PMID:16806880]
GO:0071563
name: Myo2p-Vac17p-Vac8p transport complex
def: "A protein complex that is involved in transport of vacuoles to a newly formed daughter cell. In yeast, this complex is composed of Myo2p, Vac17p, and Vac8p." [GOC:jp, PMID:12594460]
GO:0071564
name: npBAF complex
def: "A SWI/SNF-type complex that is found in neural stem or progenitor cells, and in human contains actin and proteins encoded by the ARID1A/BAF250A or ARID1B/BAF250B, SMARCD1/BAF60A, SMARCD3/BAF60C, SMARCA2/BRM/BAF190B, SMARCA4/BRG1/BAF190A, SMARCB1/BAF47, SMARCC1/BAF155, SMARCE1/BAF57, SMARCC2/BAF170, PHF10/BAF45A, ACTL6A/BAF53A genes. The npBAF complex is essential for the self-renewal/proliferative capacity of the multipotent neural stem cells." [GOC:mah, GOC:ss, PMID:17640523]
GO:0071565
name: nBAF complex
def: "A SWI/SNF-type complex that is found in post-mitotic neurons, and in human contains actin and proteins encoded by the ARID1A/BAF250A or ARID1B/BAF250B, SMARCD1/BAF60A, SMARCD3/BAF60C, SMARCA2/BRM/BAF190B, SMARCA4/BRG1/BAF190A, SMARCB1/BAF47, SMARCC1/BAF155, SMARCE1/BAF57, SMARCC2/BAF170, DPF1/BAF45B, DPF3/BAF45C, ACTL6B/BAF53B genes. The nBAF complex along with CREST plays a role regulating the activity of genes essential for dendrite growth." [GOC:mah, GOC:ss, PMID:17640523]
GO:0071575
name: integral component of external side of plasma membrane
def: "The component of the plasma membrane consisting of the gene products that penetrate only the external side of the membrane." [GOC:dos, GOC:mah]
GO:0071595
name: Nem1-Spo7 phosphatase complex
def: "A protein serine/threonine phosphatase complex that is involved in nuclear envelope organization, and contains proteins known in budding yeast as Nem1p and Spo7p." [GOC:mah, PMID:9822591]
GO:0071597
name: cellular birth scar
def: "Crater-like ring of chitinous scar tissue located on the surface of the daughter cell, in budding fungi, at the site of separation from the mother cell. It is formed after the newly emerged daughter cell separates, thereby marking the site of cytokinesis and septation." [GOC:mcc, PMID:16672383, PMID:7730409]
GO:0071598
name: neuronal ribonucleoprotein granule
def: "A ribonucleoprotein complex that is found in the cytoplasm of axons and dendrites, and transports translationally silenced mRNAs to dendritic synapses, where they are released and translated in response to specific exogenous stimuli." [GOC:BHF, GOC:go_curators, GOC:mah, PMID:19015237, PMID:20368989]
GO:0071601
name: sphere organelle
def: "A nuclear body that is found in the germinal vesicles of amphibian oocytes, and consist of three major parts: a remarkably spherical body about 5-10 pm in diameter, smaller spherical or nearly spherical granules on the surface, and inclusions of various sizes that strongly resemble the surface granules. The parts of the sphere organelle have distinct compositions, including splicing snRNAs and proteins." [PMID:7758244, PMID:8349728]
GO:0071627
name: integral component of fungal-type vacuolar membrane
def: "The component of the fungal-type vacuole membrane consisting of the gene products that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:dos, GOC:mah]
GO:0071628
name: intrinsic component of fungal-type vacuolar membrane
def: "The component of a fungal-type vacuole membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos, GOC:mah]
GO:0071664
name: catenin-TCF7L2 complex
def: "A protein complex that contains a catenin and TCF7L2 (TCF4), binds to the TCF DNA motif within a promoter element, and is involved in the regulation of WNT target gene transcription." [GOC:BHF, GOC:rl, GOC:vk, PMID:14661054]
GO:0071665
name: gamma-catenin-TCF7L2 complex
def: "A protein complex that contains gamma-catenin and TCF7L2 (TCF4), binds to the TCF DNA motif within a promoter element, and is involved in the regulation of WNT target gene transcription." [GOC:BHF, GOC:vk, PMID:14661054]
GO:0071666
name: Slit-Robo signaling complex
def: "A protein-carbohydrate complex that consists of a transmembrane roundabout (Robo) receptor, an extracellular Slit ligand and heparin/heparan sulfate." [GOC:sart, PMID:17062560, PMID:18359766]
GO:0071682
name: endocytic vesicle lumen
def: "The volume enclosed by the membrane of an endocytic vesicle." [GOC:pde]
GO:0071683
name: sensory dendrite
def: "A dendrite that is found on a sensory neuron, and directly transduces a sensory signal from the sensory neuron to another neuron." [GOC:dos, GOC:kmv, GOC:mah]
GO:0071685
name: NADH dehydrogenase complex (plastoquinone)
def: "An NADH dehydrogenase complex that catalyzes the transfer of electrons to plastoquinone. The complex is involved in the non-photochemical reduction of plastoquinones and the cyclic electron transport around photosystem I, and is found in plastid thylakoids." [DOI:10.1078/0176-1617-00593, GOC:mah]
GO:0071686
name: horsetail nucleus
def: "The elongated nucleus which forms during the rapid oscillatory movement at meiotic prophase; characterized in Schizosaccharomyces pombe." [GOC:vw, PMID:15030757]
GO:0071687
name: horsetail nucleus leading edge
def: "The part of the horsetail nucleus where telomeres cluster under the SPB and that leads horsetail movement. The horsetail nucleus is the elongated nucleus which forms during the rapid oscillatory movement at meiotic prophase; characterized in Schizosaccharomyces pombe." [GOC:mah, GOC:vw, PMID:15030757]
GO:0071735
name: IgG immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of an IgG isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgG immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071736
name: IgG immunoglobulin complex, circulating
def: "A protein complex composed of two identical immunoglobulin heavy chains of an IgG isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071737
name: IgG B cell receptor complex
def: "An IgG immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of an IgG isotype and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196]
GO:0071738
name: IgD immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgD isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgD immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:11282392]
GO:0071739
name: IgD immunoglobulin complex, circulating
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgD isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:11282392]
GO:0071740
name: IgD B cell receptor complex
def: "An IgD immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of the IgD isotype and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:11282392]
GO:0071741
name: IgD immunoglobulin complex, GPI-anchored
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgD isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and bound via a GPI-anchor to the plasma membrane of B cells." [GOC:add, ISBN:0781765196, PMID:11282392]
GO:0071742
name: IgE immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgE isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgE immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071743
name: IgE immunoglobulin complex, circulating
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgE isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071744
name: IgE B cell receptor complex
def: "An IgE immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of the IgE isotype and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196]
GO:0071745
name: IgA immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgA isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and sometimes complexed with J chain or J chain and secretory component. An IgA immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071746
name: IgA immunoglobulin complex, circulating
def: "A protein complex composed of two identical immunoglobulin heavy chains of an IgA isotype and two identical immunoglobulin light chains, held together by disulfide bonds, sometimes complexed with J chain or J chain and secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071747
name: IgA B cell receptor complex
def: "An IgA immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of an IgA isotype and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071748
name: monomeric IgA immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of an IgA isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071749
name: polymeric IgA immunoglobulin complex
def: "A protein complex composed of two, three, or four monomeric IgA immunoglobulin complexes linked through both direct disulfide bonds and through disulfide binded monomers of J chain acting as a bridge. Each IgA monomer consists of two identical immunoglobulin heavy chains of an IgA isotype and two identical immunoglobulin light chains, held together by disulfide bonds. Dimeric IgA is sometimes complexed additionally with secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071750
name: dimeric IgA immunoglobulin complex
def: "A protein complex composed of two monomeric IgA immunoglobulin complexes linked through both direct disulfide bonds and through a disulfide binded monomer of J chain acting as a bridge. Each IgA monomer consists of two identical immunoglobulin heavy chains of an IgA isotype and two identical immunoglobulin light chains, held together by disulfide bonds. Dimeric IgA is sometimes complexed additionally with secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071751
name: secretory IgA immunoglobulin complex
def: "A polymeric IgA immunoglobulin complex that is complexed with one chain of secretory component (SC). Polymeric IgA is present in mucosal areas, having been transported via a transcytosis mechanism in mucosal epithelial cells relying on the polymeric Ig receptor, a portion of which then remains bound to the polymeric IgA as secretory component." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071752
name: secretory dimeric IgA immunoglobulin complex
def: "A dimeric form of secretory IgA immunoglobulin complex." [GOC:add, ISBN:0781765196, PMID:16362985]
GO:0071753
name: IgM immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgM isotype and two identical immunoglobulin light chains, held together by disulfide bonds, and in its circulating form complexed with J chain in polymeric forms. An IgM immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:20176268]
GO:0071754
name: IgM immunoglobulin complex, circulating
def: "A polymer of five or six IgM core units each composed of two identical immunoglobulin heavy chains of the IgM isotype and two identical immunoglobulin light chains, held together by disulfide bonds; the individual IgM core units are held together via disulfide bonds with a single J chain polypeptide acting as a bridge between two of the polymeric units. Circulating IgM is present in the extracellular space, in mucosal areas or other tissues, or in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:20176268]
GO:0071755
name: IgM B cell receptor complex
def: "An IgM immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of the IgM isotype and two identical immunoglobulin light chains and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:20176268]
GO:0071756
name: pentameric IgM immunoglobulin complex
def: "A circulating form of IgM consisting of a pentamer of IgM core units with a single J chain polypeptide." [GOC:add, ISBN:0781765196, PMID:20176268]
GO:0071757
name: hexameric IgM immunoglobulin complex
def: "A circulating form of IgM consisting of a hexamer of IgM core units with a single J chain polypeptide." [GOC:add, ISBN:0781765196, PMID:20176268]
GO:0071758
name: IgW immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgW isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgW immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071759
name: IgX immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgX isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgX immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071760
name: IgY immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgY isotype and two identical immunoglobulin light chains, held together by disulfide bonds. An IgY immunoglobulin complex may be embedded in the plasma membrane or present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196]
GO:0071761
name: IgZ immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgZ isotype and two identical immunoglobulin light chains, held together by disulfide bonds. The IgZ isotype is also known as the IgT isotype in certain species of fish." [GOC:add, ISBN:0781765196]
GO:0071762
name: heavy chain immunoglobulin complex
def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgNAR isotype held together by disulfide bonds and lacking immunoglobulin light chains." [GOC:add, ISBN:0781765196, PMID:12543123, PMID:16051357]
GO:0071778
name: WINAC complex
def: "OBSOLETE. A SWI/SNF-type complex that directly interacts with the vitamin D receptor (VDR) through the Williams syndrome transcription factor (WSTF), and mediates the recruitment of unliganded VDR to VDR target sites in promoters. The WINAC complex contains at least 13 subunits, including WSTF, several SWI/SNF components, and DNA replication-related factors." [GOC:BHF, PMID:12837248]
GO:0071781
name: endoplasmic reticulum cisternal network
def: "An endoplasmic reticulum part that comprises the membranes with low curvature in cross-section." [GOC:vw, PMID:16469703, PMID:20434336]
GO:0071782
name: endoplasmic reticulum tubular network
def: "An endoplasmic reticulum part that comprises the membranes with high curvature in cross-section." [GOC:vw, PMID:16469703, PMID:20434336]
GO:0071797
name: LUBAC complex
def: "A ubiquitin ligase complex that catalyzes linear head-to-tail polyubiquitin conjugation on its targets. In human the complex consists of RBCK1, RNF31 and SHARPIN, and has an MW of approximately 600 kDa, suggesting a heteromultimeric assembly of its subunits. LUBAC stands for Linear Ubiquitin Chain Assembly Complex." [GOC:sp, PMID:17006537, PMID:19136968, PMID:21455180]
GO:0071808
name: satellite fibril
def: "An axoneme part that is found in the flagella of mammalian sperm and is located in the middle piece between the outer dense fibers (on the concave side of outer dense fibers as seen in cross-section)." [GOC:mah, GOC:sl, PMID:20108326]
GO:0071817
name: MMXD complex
def: "A protein complex that contains the proteins MMS19, MIP18 and XPD, localizes to mitotic spindle during mitosis, and is required for proper chromosome segregation." [GOC:sp, PMID:20797633]
GO:0071818
name: BAT3 complex
def: "An ER membrane insertion complex that acts by facilitating tail-anchored protein capture by ASNA1/TRC40. In mammals the complex contains Bat3, TRC35 and Ubl4A." [GOC:mah, GOC:sp, PMID:20676083]
GO:0071819
name: DUBm complex
def: "A protein complex that forms part of SAGA-type complexes SAGA and SLIK, and mediates deubiquitination of histone H2B. In S. cerevisiae, the DUBm consists of the proteins Ubp8p, Sgf11p, Sus1p, and Sgf73p." [PMID:19226466, PMID:20395473]
GO:0071821
name: FANCM-MHF complex
def: "A protein complex contains the proteins FANCM and MHF, or their orthologs, plays an essential role in DNA remodeling, protects replication forks, and is conserved in eukaryotes." [GOC:mah, GOC:vw, PMID:20347428]
GO:0071914
name: prominosome
def: "An extracellular membrane-bounded vesicle that contains prominin proteins (in mouse Prom1/CD33 or Prom2) and are found in body fluids including ventricular fluid, saliva, urine and seminal fluid. In the ventricular fluid of the developing mouse brain two major classes of these particles have been observed (P2 particles of 500-1000 nm and P4 particles of 50-80 nm) which likely originate from microvilli, primary cilia and/or the midbody of neuroepithelial cells. The physiological role is not known." [PMID:15976444, PMID:17109118, PMID:17283184]
GO:0071920
name: cleavage body
def: "A nuclear body that contains proteins involved in pre-mRNA 3'-end cleavage and polyadenylation, such as DDX1, CSTF2 and CPSFs, as well as the transcription factors TFIIE and TFIIF. Cleavage bodies are localized adjacent to Cajal bodies and are involved in mRNA3'-end processing." [PMID:10564273, PMID:11598190, PMID:8654386]
GO:0071942
name: XPC complex
def: "A nucleotide-excision repair complex that is involved in damage sensing during global genome nucleotide excision repair (GG-NER). It is part of the pre-incision (or initial recognition) complex bound to sites of DNA damage. In human, it is composed of XPC, RAD23B and CETN2." [PMID:11279143, PMID:15964821, PMID:19941824]
GO:0071943
name: Myc-Max complex
def: "A transcription factor complex that consists of a heterodimer of the bHLH-ZIP proteins Myc and Max." [GOC:cna, PMID:16620027, PMID:16620031, PMID:20170194]
GO:0071944
name: cell periphery
def: "The part of a cell encompassing the cell cortex, the plasma membrane, and any external encapsulating structures." [GOC:mah]
GO:0071953
name: elastic fiber
def: "An extracellular matrix part that consists of an insoluble core of polymerized tropoelastin monomers and a surrounding mantle of microfibrils. Elastic fibers provide elasticity and recoiling to tissues and organs, and maintain structural integrity against mechanical strain." [GOC:BHF, GOC:mah, PMID:20236620]
GO:0071957
name: old mitotic spindle pole body
def: "The spindle pole body that exists in a cell prior to spindle pole body duplication. An old spindle pole body segregates to the daughter cell upon mitosis, and lacks active proteins involved in signaling exit from mitosis." [GOC:mah, GOC:vw, PMID:15132994]
GO:0071958
name: new mitotic spindle pole body
def: "The spindle pole body that is formed by spindle pole body duplication, and to which proteins involved in mitotic exit signaling (for example, the septation initiation network in fission yeast) localize." [GOC:mah, GOC:vw, PMID:15132994]
GO:0071986
name: Ragulator complex
def: "A protein complex that contains MAPKSP1 (MP1, Map2k1ip1), ROBLD3 (p14, Mapbpip), and C11orf59 (p18). The complex is anchored to lipid rafts in late endosome membranes via C11orf59, recruits mTORC1 to lysosomal membranes in amino acid signaling to mTORC1, and is also involved in ERK/MAPK signaling." [PMID:19177150, PMID:20381137]
GO:0072324
name: ascus epiplasm
def: "Ascus cytoplasm that is not packaged into ascospores." [DOI:10.1016/S0953-7562(96)80057-8, GOC:mcc]
GO:0072357
name: PTW/PP1 phosphatase complex
def: "A protein serine/threonine phosphatase complex that contains a catalytic subunit (PPP1CA, PPP1CB or PPP1CC) and the regulatory subunits PPP1R10 (PNUTS), TOX4 and WDR82, and plays a role in the control of chromatin structure and cell cycle progression during the transition from mitosis into interphase." [GOC:mah, PMID:20516061]
GO:0072372
name: primary cilium
def: "A cilium found on many different cell types that is typically present in a single copy per cell. A primary cilium may have a variable array of axonemal microtubules and may or may not contain molecular motors." [GOC:curators, PMID:17009929, PMID:20144998]
GO:0072379
name: ER membrane insertion complex
def: "A protein complex that is involved in the post-translational delivery of tail-anchored (TA) membrane proteins to the endoplasmic reticulum. TA membrane proteins, also called type II transmembrane proteins, contain a single C-terminal transmembrane region. Some ER membrane insertion complex subunits are conserved between different species such as mammals and budding yeast." [GOC:mah, PMID:20676083, PMID:20850366]
GO:0072380
name: TRC complex
def: "An ER membrane insertion complex that contains subunits that recognize two types of transmembrane domain signals. In budding yeast the complex contains Get4p, Get5p, Sgt2p, and at least two heat shock proteins (HSPs)." [GOC:mah, PMID:20850366]
GO:0072487
name: MSL complex
def: "A histone acetyltransferase complex that catalyzes the acetylation of a histone H4 lysine residue at position 16. In human, it contains the catalytic subunit MOF, and MSL1, MSL2 and MSL3." [PMID:16227571, PMID:20018852]
GO:0072492
name: host cell mitochondrial intermembrane space
def: "The region between the inner and outer lipid bilayers of the host cell mitochondrial envelope." [GOC:ecd]
GO:0072493
name: host cell endosome lumen
def: "The volume enclosed by the membranes of the host cell endosome." [GOC:ecd]
GO:0072494
name: host multivesicular body
def: "A late endosome in which regions of the limiting host cell endosomal membrane invaginate to form internal vesicles; host membrane proteins that enter the internal vesicles are sequestered from the host cytoplasm." [GOC:rph]
GO:0072495
name: host cell Cajal body
def: "A class of nuclear body in the eukaryotic host cell, first seen after silver staining by Ramon y Cajal in 1903, enriched in small nuclear ribonucleoproteins, and certain general RNA polymerase II transcription factors; ultrastructurally, they appear as a tangle of coiled, electron-dense threads roughly 0.5 micrometers in diameter; involved in aspects of snRNP biogenesis; the protein coilin serves as a marker for Cajal bodies. Some argue that Cajal bodies are the sites for preassembly of transcriptosomes, unitary particles involved in transcription and processing of RNA. The host is the larger of the organisms involved in a symbiotic interaction." [GOC:rph]
GO:0072516
name: viral assembly compartment
def: "A membrane-bounded compartment that forms in the cytoplasm of virus-infected cells, in which virus assembly takes place." [GOC:BHF, PMID:20374631]
GO:0072517
name: host cell viral assembly compartment
def: "A membrane-bounded compartment that forms in the cytoplasm of the host cell, in which virus assembly takes place." [GOC:BHF, PMID:20374631]
GO:0072534
name: perineuronal net
def: "A dense extracellular matrix (ECM) structure that forms around many neuronal cell bodies and dendrites late in development and is responsible for synaptic stabilization in the adult brain." [GOC:sl, PMID:18364019]
GO:0072536
name: interleukin-23 receptor complex
def: "A protein complex that binds interleukin-23. The complex comprises two subunits, including the same beta subunit found in the interleukin-12 receptor." [GOC:BHF, GOC:mah, PMID:12023369]
GO:0072546
name: ER membrane protein complex
def: "A transmembrane protein complex that is involved in protein folding in the endoplasmic reticulum. In S. cerevisiae, it has six members: EMC1, EMC2, AIM27, EMC4, KRE27, and EMC6." [GOC:dgf, PMID:19325107]
GO:0072556
name: other organism presynaptic membrane
def: "A presynaptic membrane that is part of another organism, i.e. a secondary organism with which the first organism is interacting. A presynaptic membrane is specialized area of membrane of the axon terminal that faces the plasma membrane of the neuron or muscle fiber with which the axon terminal establishes a synaptic junction; many synaptic junctions exhibit structural presynaptic characteristics, such as conical, electron-dense internal protrusions, that distinguish it from the remainder of the axon plasma membrane." [GOC:mah]
GO:0072557
name: IPAF inflammasome complex
def: "A protein complex that consists of three components, IPAF, NAIP and caspase-1, and includes among its functions the sensing of flagellin derived from Legionella pneumophila, Salmonella typhimurium, Pseudomonas aeruginosa and Shigella flexneri." [GOC:add, GOC:BHF, GOC:vp, PMID:20303873]
GO:0072558
name: NLRP1 inflammasome complex
def: "A protein complex that consists of two components, NLRP1 (NALP1) and caspase-1 or caspase-5. The exact mechanisms of NLRP1 activation remain obscure, but potassium ion efflux appears to be essential." [GOC:add, GOC:BHF, GOC:vp, PMID:20303873]
GO:0072559
name: NLRP3 inflammasome complex
def: "A protein complex that consists of three components, NLRP3 (NALP3), PYCARD and caspase-1. It is activated upon exposure to whole pathogens, as well as a number of structurally diverse pathogen- and danger-associated molecular patterns (PAMPs and DAMPs) and environmental irritants. Whole pathogens demonstrated to activate the NLRP3 inflammasome complex include the fungi Candida albicans and Saccharomyces cerevisiae, bacteria that produce pore-forming toxins, including Listeria monocytogenes and Staphylococcus aureus, and viruses such as Sendai virus, adenovirus, and influenza virus." [GOC:add, GOC:BHF, GOC:vp, PMID:20303873]
GO:0072562
name: blood microparticle
def: "A phospholipid microvesicle that is derived from any of several cell types, such as platelets, blood cells, endothelial cells, or others, and contains membrane receptors as well as other proteins characteristic of the parental cell. Microparticles are heterogeneous in size, and are characterized as microvesicles free of nucleic acids." [GOC:BHF, GOC:mah, PMID:16373184]
GO:0072563
name: endothelial microparticle
def: "A blood microparticle that is derived from, and contains membrane receptors as well as other proteins characteristic of, an endothelial cell." [GOC:BHF, GOC:mah, PMID:16373184]
GO:0072588
name: box H/ACA RNP complex
def: "A ribonucleoprotein complex that contains an RNA of the box H/ACA type, a subtype of the small nucleolar RNA (snoRNA) family. RNA pseudouridylation (isomerization of uridine to pseudouridine) is the major, and most likely the ancestral, function of H/ACA RNPs, although some have evolved other functions. Pseudouridylation targets include both large and small ribosomal RNAs (rRNAs), and on U2 small nuclear RNA (U2 snRNA)." [GOC:mah, GOC:vw, PMID:17284456, PMID:20227365]
GO:0072589
name: box H/ACA scaRNP complex
def: "A box H/ACA RNP complex that is located in the Cajal body of the nucleoplasm. In higher eukaryotes, box H/ACA RNP located in Cajal bodies mediate pseudouridylation of spliceosomal snRNAs." [GOC:mah, PMID:17284456, PMID:20227365]
GO:0072669
name: tRNA-splicing ligase complex
def: "A protein complex that catalyzes the ligation of cleaved pre-tRNAs by directly joining spliced tRNA halves to mature-sized tRNAs by incorporating the precursor-derived splice junction phosphate into the mature tRNA as a canonical 3',5'-phosphodiester." [GOC:sp, PMID:21311021]
GO:0072686
name: mitotic spindle
def: "A spindle that forms as part of mitosis. Mitotic and meiotic spindles contain distinctive complements of proteins associated with microtubules." [GOC:mah, GOC:vw, PMID:11408572, PMID:18367542, PMID:8027178]
GO:0072687
name: meiotic spindle
def: "A spindle that forms as part of meiosis. Several proteins, such as budding yeast Spo21p, fission yeast Spo2 and Spo13, and C. elegans mei-1, localize specifically to the meiotic spindle and are absent from the mitotic spindle." [GOC:mah, GOC:vw, PMID:11408572, PMID:18367542, PMID:8027178]
GO:0075341
name: host cell PML body
def: "A nuclear body that reacts against SP100 auto-antibodies (PML = promyelocytic leukemia) located within a cell of a host organism." [GOC:BHF, GOC:jl]
GO:0080008
name: Cul4-RING E3 ubiquitin ligase complex
def: "A ubiquitin ligase complex in which a cullin from the Cul4 family and a RING domain protein form the catalytic core; substrate specificity is conferred by an adaptor protein." [PMID:16792691, PMID:18223036, PMID:18552200]
GO:0080085
name: signal recognition particle, chloroplast targeting
def: "A complex consisting of a protein and RNA component which binds the signal sequence of some proteins and facilitates their export to the chloroplast." [PMID:17513500]
GO:0085026
name: tubovesicular membrane network
def: "A complex, symbiont-induced host-derived organelle that is comprised of multiple protein and lipid domains." [GOC:pamgo_curators]
GO:0085035
name: haustorium
def: "A projection from a cell or tissue that penetrates the host's cell wall and invaginates the host cell membrane." [GOC:pamgo_curators]
GO:0085036
name: extrahaustorial matrix
def: "The space between the symbiont plasma membrane and the extrahaustorial membrane of the host." [GOC:pamgo_curators]
GO:0085037
name: extrahaustorial membrane
def: "The membrane surrounding the symbiont haustorium during symbiosis, derived from the host plasma membrane." [GOC:pamgo_curators]
GO:0085039
name: extra-invasive hyphal membrane
def: "A host-derived membrane surrounding the symbiont invasive hypha during symbiosis." [GOC:pamgo_curators]
GO:0085040
name: extra-invasive hyphal space
def: "The space between the symbiont plasma membrane and the extra-invasive hyphal membrane." [GOC:pamgo_curators]
GO:0085041
name: arbuscule
def: "Highly branched symbiont haustoria within host root cortex cells, responsible for nutrient exchange." [GOC:pamgo_curators]
GO:0085042
name: periarbuscular membrane
def: "A host-derived membrane surrounding the symbiont arbuscule during symbiosis." [GOC:pamgo_curators]
GO:0089701
name: U2AF
def: "A heterodimeric protein complex consisting of conserved large and small U2AF subunits that contributes to spliceosomal RNA splicing by binding to consensus sequences at the 3' splice site. U2AF is required to stabilize the association of the U2 snRNP with the branch point." [GOC:dos, GOC:mah, PMID:15231733, PMID:1538748, PMID:2963698, PMID:8657565]
GO:0089713
name: Cbf1-Met4-Met28 complex
def: "A heteromeric complex consisting of Cbf1 and basic leucine zipper (bZIP) containing transcriptional activators, Met4 and Met28, that forms over the sequence TCACGTG in the upstream activating sequence (UAS) of genes involved in sulfur amino acid metabolism, resulting in their transcriptional activation." [PMID:8665859, PMID:9171357]
GO:0089716
name: Pip2-Oaf1 complex
def: "A heterodimeric complex consisting of Zn(2)Cys(6) containing transcription factors Pip2 and Oaf1. It binds to the oleate response element (ORE), found in the promoters of fatty acid-inducible genes in Saccharomyces where, in the presence of oleate this bound complex activates the transcription of genes encoding peroxisomal proteins." [PMID:8972187, PMID:9288897]
GO:0089717
name: spanning component of membrane
def: "The component of a membrane consisting of gene products and protein complexes that have some part that spans both leaflets of the membrane." [GOC:dos]
GO:0090123
name: lysosomal glycocalyx
def: "The polysaccharide-based coating on the inner side of a lysosomal membrane that protects it from digestion by lysosomal enzymes." [GOC:ascb_2009, GOC:dph, GOC:tb]
GO:0090395
name: plant cell papilla
def: "A cell projection that is a short, rounded projection from a plant epidermal cell." [GOC:tb]
GO:0090396
name: leaf papilla
def: "A plant cell papilla that is part of a leaf papilla cell." [GOC:tb]
GO:0090397
name: stigma papilla
def: "A plant cell papilla that is part of a stigma papilla cell." [GOC:tb]
GO:0090404
name: pollen tube tip
def: "The region at growing end of the pollen tube cell, where polarized growth occurs." [GOC:tb, PO:0025195, PO:0025281]
GO:0090405
name: unicellular trichome branch
def: "A cell projection part that is a branch of a unicellular trichome." [GOC:tb, PO:0025537]
GO:0090406
name: pollen tube
def: "A tubular cell projection that is part of a pollen tube cell and extends from a pollen grain." [GOC:tb, PO:0025195, PO:0025281]
GO:0090443
name: FAR/SIN/STRIPAK complex
def: "A conserved protein phosphatase type 2A complex which contains a protein phosphatase type 2A, a protein phosphatase regulatory subunit, a striatin, an FHA domain protein and other subunits (at least six proteins). In fission yeast this complex negatively regulate the septation initiation network at the spindle pole body." [GOC:vw, PMID:21561862, PMID:22119525]
GO:0090498
name: extrinsic component of Golgi membrane
def: "The component of a Golgi membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:dos, PMID:21337012]
GO:0090512
name: eisosome membrane domain/MCC
def: "A plasma membrane part that is composed of a furrow-like plasma membrane domain and associated integral transmembrane proteins." [GOC:al, GOC:vw, PMID:22368779]
GO:0090533
name: cation-transporting ATPase complex
def: "Protein complex that carries out the reaction: ATP + H2O + cation(out) = ADP + phosphate + cation(in)." [GOC:BHF]
GO:0090534
name: calcium ion-transporting ATPase complex
def: "Protein complex that carries out the reaction: ATP + H2O + Ca2+(out) = ADP + phosphate + Ca2+(in)." [GOC:BHF]
GO:0090535
name: WICH complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (specifically SNF2H in mammals, which contain two ISWI homologs) and WSTF (Williams Syndrome Transcription Factor). WICH plays roles in regulation of RNAP I and III transcription and in DNA replication and repair." [GOC:krc, PMID:15284901, PMID:16568949, PMID:21810179]
GO:0090536
name: NoRC complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (specifically SNF2H in mammals, which contain two ISWI homologs) and a Tip5 homolog. In mammals, NoRC is involved in regulation of transcription from RNAP I and RNA polymerase III promoters." [GOC:krc]
GO:0090537
name: CERF complex
def: "An ISWI complex that contains an ATPase subunit of the ISWI family (specifically SNF2L in mammals, which contain two ISWI homologs) and a CECR2 homolog. In mammals, CERF is involved in regulation of transcription from RNA polymerase II promoters." [GOC:krc]
GO:0090543
name: Flemming body
def: "A cell part that is the central region of the midbody characterized by a gap in alpha-tubulin staining. It is a dense structure of antiparallel microtubules from the central spindle in the middle of the intercellular bridge." [GOC:pm, PMID:18641129, PMID:22522702]
GO:0090544
name: BAF-type complex
def: "A SWI/SNF-type complex that contains a subunit from the BAF (Brahma-Associated Factor) family." [GOC:krc, GOC:tb]
GO:0090545
name: CHD-type complex
def: "A SWI/SNF-type complex that contains a subunit from the CHD(Chromodomain helicase DNA-binding) family. The CHD family is characterized by two signature sequence motifs: tandem chromodomains located in the N-terminal region, and the SNF2-like ATPase domain located in the central region of the protein structure." [GOC:krc, GOC:tb, PMID:17350655]
GO:0090552
name: unicellular trichome apex
def: "A cell projection part that is the apical most portion of a unicellular trichome." [GOC:PO_curators, PO:0025537]
GO:0090553
name: unicellular trichome tip
def: "A cell projection part that is the apical most portion of a unicellular trichome apex." [GOC:PO_curators]
GO:0090568
name: nuclear transcriptional repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription." [GOC:tb]
GO:0090569
name: cytoplasmic transcriptional repressor complex
def: "A protein complex, located in the cytoplasm, that possesses activity that prevents or downregulates transcription." [GOC:tb]
GO:0090570
name: RNA polymerase I transcription repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription from a RNA polymerase I promoter." [GOC:tb]
GO:0090571
name: RNA polymerase II transcription repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription from a RNA polymerase II promoter." [GOC:tb]
GO:0090572
name: RNA polymerase III transcription repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription from a RNA polymerase III promoter." [GOC:tb]
GO:0090573
name: RNA polymerase IV transcription repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription from a RNA polymerase IV promoter." [GOC:tb]
GO:0090574
name: RNA polymerase V transcription repressor complex
def: "A protein complex, located in the nucleus, that possesses activity that prevents or downregulates transcription from a RNA polymerase V promoter." [GOC:tb]
GO:0090575
name: RNA polymerase II transcription factor complex
def: "A transcription factor complex that acts at promoters of genes transcribed by RNA polymerase II." [GOC:tb]
GO:0090576
name: RNA polymerase III transcription factor complex
def: "A transcription factor complex that acts at promoters of genes transcribed by RNA polymerase III." [GOC:tb]
GO:0090577
name: RNA polymerase IV transcription factor complex
def: "A transcription factor complex that acts at promoters of genes transcribed by RNA polymerase IV." [GOC:tb]
GO:0090578
name: RNA polymerase V transcription factor complex
def: "A transcription factor complex that acts at promoters of genes transcribed by RNA polymerase V." [GOC:tb]
GO:0090619
name: meiotic spindle pole
def: "Either of the ends of a meiotic spindle, a spindle that forms as part of meiosis, where spindle microtubules are organized; usually contains a microtubule organizing center and accessory molecules, spindle microtubules and astral microtubules." [GOC:ha, PMID:18250200]
GO:0090620
name: APC-Cdc20 complex
def: "An anaphase promoting complex bound to the \"fizzy family\" APC activator Cdc20/Slp1 which\nregulates the metaphase anaphase transition by activating the APC/C to target the anaphase inhibitor securin and promotes sister chromatid separation." [GOC:vw, PMID:10921876]
GO:0090621
name: APC-fizzy-related complex
def: "An anaphase promoting complex bound to the \"fizzy-related family\" APC activator FZR1/Cdh1/Srw1 that regulates mitotic exit by activating the APC/C to target mitotic cyclins for destruction during anaphase and telophase. Is also active during G1." [GOC:vw, PMID:10921876]
GO:0097002
name: mitochondrial inner boundary membrane
def: "The portion of the mitochondrial inner membrane that is not invaginated to form cristae. The inner boundary membrane lies parallel to the outer membrane." [GOC:mcc, PMID:16054341, PMID:19019989]
GO:0097013
name: phagocytic vesicle lumen
def: "The volume enclosed by the membrane of a phagocytic vesicle." [GOC:rs]
GO:0097014
name: ciliary cytoplasm
def: "All of the contents of a cilium, excluding the plasma membrane surrounding the cilium." [GOC:BHF, GOC:cilia, PMID:17895364]
GO:0097015
name: bacterial-type flagellar cytoplasm
def: "All of the contents of a bacterial-type flagellum, excluding the plasma membrane surrounding the flagellum." [GOC:BHF]
GO:0097025
name: MPP7-DLG1-LIN7 complex
def: "A heterotrimeric protein complex formed by the association of MMP7, DLG1 and either LIN7A or LIN7C; regulates the stability and localization of DLG1 to cell junctions." [GOC:BHF, PMID:17237226]
GO:0097038
name: perinuclear endoplasmic reticulum
def: "The portion of endoplasmic reticulum, the intracellular network of tubules and cisternae, that occurs near the nucleus. The lumen of the perinuclear endoplasmic reticulum is contiguous with the nuclear envelope lumen (also called perinuclear space), the region between the inner and outer nuclear membranes." [GOC:bf, GOC:mah, GOC:mcc, GOC:pr, GOC:vw]
GO:0097042
name: extrinsic component of fungal-type vacuolar membrane
def: "The component of a fungal-type vacuolar membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:jh, PMID:21454883]
GO:0097047
name: DNA replication termination region
def: "A chromosomal region that contains fork pausing elements influencing the progression and merging of DNA replication forks." [GOC:mcc, GOC:pr, PMID:20797631]
GO:0097057
name: TRAF2-GSTP1 complex
def: "A protein complex comprising tumor necrosis factor (TNF) receptor-associated factor 2 (TRAF2) and glutathione S-transferase pi 1 (GSTP1). This complex is thought to disrupt the TNF signaling cascade, thus down-regulating inflammatory responses." [GOC:BHF, PMID:16636664]
GO:0097058
name: CRLF-CLCF1 complex
def: "A heterodimeric protein complex that is composed of cardiotrophin-like cytokine factor 1 (product of the CLCF1 gene) and cytokine receptor-like factor 1 (product of the CRLF gene) and is secreted into the extracellular space. The CRLF-CLCF1 complex is a ligand for the ciliary neurotrophic factor (CNTF) receptor complex." [GOC:BHF, PMID:10966616]
GO:0097059
name: CNTFR-CLCF1 complex
def: "A protein complex that is composed of two soluble ciliary neurotrophic factor receptor alpha subunits (product of the CNTFR gene) and two molecules of cardiotrophin-like cytokine factor 1 (product of the CLCF1 gene). The complex is secreted into the extracellular space." [GOC:BHF, PMID:11285233]
GO:0097060
name: synaptic membrane
def: "A specialized area of membrane on either the presynaptic or the postsynaptic side of a synapse, the junction between a nerve fiber of one neuron and another neuron or muscle fiber or glial cell." [GOC:BHF, PMID:20410104]
GO:0097071
name: interferon regulatory factor complex
def: "A protein complex that consists of two interferon regulatory proteins (IRFs); may be homodimeric or heterodimeric. The activation of a latent closed conformation of IRF in the cytoplasm is triggered by phosphorylation of Ser/Thr residues in a C-terminal region. Phosphorylation stimulates the C-terminal autoinhibitory domain to attain a highly extended conformation triggering dimerization through extensive contacts to a second subunit." [GOC:cna, PMID:20043992]
GO:0097072
name: interferon regulatory factor 3 complex
def: "An interferon regulatory factor complex that consists of a homodimer of interferon regulatory factor 3." [GOC:cna, PMID:12855817, PMID:14556004, Reactome:REACT_7146]
GO:0097073
name: interferon regulatory factor 5 complex
def: "An interferon regulatory factor complex that consists of a homodimer of interferon regulatory factor 5." [GOC:cna, PMID:12138184, PMID:16751392]
GO:0097074
name: interferon regulatory factor 7 complex
def: "An interferon regulatory factor complex that consists of a homodimer of interferon regulatory factor 7." [GOC:cna, PMID:18068231, Reactome:REACT_21965]
GO:0097075
name: interferon regulatory factor 3-interferon regulatory factor 7 complex
def: "An interferon regulatory factor complex that consists of a heterodimer of interferon regulatory factor 3 and interferon regulatory factor 7." [GOC:cna, PMID:18068231, Reactome:REACT_26957]
GO:0097076
name: transforming growth factor beta activated kinase 1 complex
def: "A protein complex that possesses protein kinase activity and activates the I-kappa B kinase complex (IKK) and mitogen-activated protein (MAP) kinases in response to TRAF6 signaling. It comprises the catalytic subunit TAK1 complexed to the regulatory subunits, termed TABs (TAK1-binding subunits)." [GOC:cna, PMID:16410796, PMID:17496917, PMID:18021073]
GO:0097078
name: FAl1-SGD1 complex
def: "A protein complex involved in the 18S rRNA biogenesis. In S. cerevisiae this complex consists of Fal1p and Sgd1p and in humans this complex consists of NOM1 and eIF4AIII subunits." [GOC:rb, PMID:21576267]
GO:0097085
name: interferon regulatory factor 3-interferon regulatory factor 5 complex
def: "An interferon regulatory factor complex that consists of a heterodimer of interferon regulatory factor 3 and interferon regulatory factor 5." [GOC:cna, PMID:12138184]
GO:0097121
name: cyclin A1-CDK1 complex
def: "A protein complex consisting of cyclin A1 and cyclin-dependent kinase 1 (CDK1). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097122
name: cyclin A2-CDK1 complex
def: "A protein complex consisting of cyclin A2 and cyclin-dependent kinase 1 (CDK1). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097123
name: cyclin A1-CDK2 complex
def: "A protein complex consisting of cyclin A1 and cyclin-dependent kinase 2 (CDK2). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097124
name: cyclin A2-CDK2 complex
def: "A protein complex consisting of cyclin A2 and cyclin-dependent kinase 2 (CDK2). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097125
name: cyclin B1-CDK1 complex
def: "A protein complex consisting of cyclin B1 and cyclin-dependent kinase 1 (CDK1). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097126
name: cyclin B2-CDK1 complex
def: "A protein complex consisting of cyclin B2 and cyclin-dependent kinase 1 (CDK1). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097127
name: cyclin B3-CDK2 complex
def: "A protein complex consisting of cyclin B3 and cyclin-dependent kinase 2 (CDK2). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097128
name: cyclin D1-CDK4 complex
def: "A protein complex consisting of cyclin D1 and cyclin-dependent kinase 4 (CDK4). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619, PR:000026740]
GO:0097129
name: cyclin D2-CDK4 complex
def: "A protein complex consisting of cyclin D2 and cyclin-dependent kinase 4 (CDK4). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097130
name: cyclin D3-CDK4 complex
def: "A protein complex consisting of cyclin D3 and cyclin-dependent kinase 4 (CDK4). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097131
name: cyclin D1-CDK6 complex
def: "A protein complex consisting of cyclin D1 and cyclin-dependent kinase 6 (CDK6). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097132
name: cyclin D2-CDK6 complex
def: "A protein complex consisting of cyclin D2 and cyclin-dependent kinase 6 (CDK6). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097133
name: cyclin D3-CDK6 complex
def: "A protein complex consisting of cyclin D3 and cyclin-dependent kinase 6 (CDK6). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619]
GO:0097134
name: cyclin E1-CDK2 complex
def: "A protein complex consisting of cyclin E1 and cyclin-dependent kinase 2 (CDK2). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619, PR:000026745]
GO:0097135
name: cyclin E2-CDK2 complex
def: "A protein complex consisting of cyclin E2 and cyclin-dependent kinase 2 (CDK2). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [GOC:so, PMID:15935619, PR:000026746]
GO:0097136
name: Bcl-2 family protein complex
def: "A protein complex that consists of members of the Bcl-2 family of anti- and proapoptotic regulators. Bcl-2 proteins respond to cues from various forms of intracellular stress, such as DNA damage or cytokine deprivation, and interact with opposing family members to determine whether or not the caspase proteolytic cascade should be unleashed." [GOC:so, PMID:14634621]
GO:0097137
name: BAD-BCL-xl complex
def: "A heterodimeric protein complex consisting of BAD and BCL-xl, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097138
name: BAD-BCL-2 complex
def: "A heterodimeric protein complex consisting of BAD and BCL-2, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097139
name: BID-BCL-2 complex
def: "A heterodimeric protein complex consisting of BID and BCL-2, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097140
name: BIM-BCL-xl complex
def: "A heterodimeric protein complex consisting of BIM and BCL-xl, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097141
name: BIM-BCL-2 complex
def: "A heterodimeric protein complex consisting of BIM and BCL-2, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097142
name: PUMA-BCL-2 complex
def: "A heterodimeric protein complex consisting of PUMA and BCL-2, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097143
name: PUMA-BCL-xl complex
def: "A heterodimeric protein complex consisting of PUMA and BCL-xl, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097144
name: BAX complex
def: "An oligomeric protein complex consisting of BAX, a member of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097145
name: BAK complex
def: "An oligomeric protein complex consisting of BAK, a member of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097146
name: NOXA-BCL-xl complex
def: "A heterodimeric protein complex consisting of NOXA and BCL-xl, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097147
name: NOXA-BCL-2 complex
def: "A heterodimeric protein complex consisting of NOXA and BCL-2, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:so, PMID:14634621]
GO:0097148
name: BCL-2 complex
def: "A homodimeric protein complex consisting of BCL-2, a member of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:bhm, GOC:so, PMID:14634621]
GO:0097149
name: centralspindlin complex
def: "A heterotetrameric protein complex playing a key role in the formation of the central spindle in mitosis. Made up of two molecules each of a mitotic kinesin (ZEN-4 in Caenorhabditis elegans or MKLP1 in mammals) and of two molecules each of a GTPase activating protein (GAP) factor (CYK-4 in Caenorhabditis elegans or MgcRacGAP in mammals)." [GOC:ans, PMID:11782313, PMID:16236794]
GO:0097165
name: nuclear stress granule
def: "A dense aggregation in the nucleus composed of proteins and RNAs that appear when the cell is under stress." [GOC:ans, PMID:10359787, PMID:12865437]
GO:0097169
name: AIM2 inflammasome complex
def: "A protein complex that consists of AIM2, ASC, and caspase-1. AIM2 is a member of the HN-200 protein family that appears to be the sensor of cytosolic double-stranded DNA." [GOC:vp, PMID:20303873]
GO:0097179
name: protease inhibitor complex
def: "A heterodimeric protein complex that contains a protease inhibitor and a protease; formation of the complex inhibits protease activity." [GOC:ans, PMID:6323392]
GO:0097180
name: serine protease inhibitor complex
def: "A heterodimeric protein complex that contains a serine protease inhibitor and a protease; formation of the complex inhibits serine protease activity." [GOC:ans, PMID:6323392]
GO:0097181
name: protein C inhibitor-coagulation factor V complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and coagulation factor V (F5); formation of the complex inhibits the serine protease activity of coagulation factor V." [GOC:ans, PMID:6323392]
GO:0097182
name: protein C inhibitor-coagulation factor Xa complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and coagulation factor Xa (F10); formation of the complex inhibits the serine protease activity of coagulation factor Xa." [GOC:ans, PMID:6323392]
GO:0097183
name: protein C inhibitor-coagulation factor XI complex
def: "A heterodimeric protein complex that contains protein C inhibitor (SERPINA5) and coagulation factor XI (F11); formation of the complex inhibits the serine protease activity of coagulation factor XI." [GOC:ans, PMID:2844223]
GO:0097189
name: apoptotic body
def: "A fragment containing parts of a dying cell. Apoptotic bodies can be formed during the execution phase of the apoptotic process, when the cell's cytoskeleton breaks up and causes the membrane to bulge outward. These bulges may separate from the cell, taking a portion of cytoplasm with them, to become apoptotic bodies. These are then engulfed by phagocytic cells, and their components recycled." [GOC:mtg_apoptosis, http://en.wikipedia.org/wiki/Apoptosis, http://en.wikipedia.org/wiki/Bleb_(cell_biology)]
GO:0097196
name: Shu complex
def: "A protein complex involved in error-free DNA post-replication repair (PRR). In Saccharomyces cerevisiae the complex contains Csm2p, Psy3p, Shu1p, and Shu2p." [GOC:jh, PMID:15654096, PMID:19496932]
GO:0097197
name: tetraspanin-enriched microdomain
def: "A pre-organized unit composed either of adhesion molecules (mainly integrins and members of the Ig superfamily), signaling receptors and/or enzyme-enriched plasma membrane domains that compartmentalizes cellular processes. Tetraspanin-enriched microdomains might be specially suited for the regulation of avidity of adhesion receptors and the compartmentalization of enzymatic activities." [GOC:ans, PMID:19709882, PMID:21930792]
GO:0097203
name: phagocytic cup lip
def: "The tip or margin of the progressing circular lamella that engulfs a particle during phagocytosis. When the two lips of the cup fuse it is converted into a phagosome." [GOC:pf, PMID:20200225]
GO:0097204
name: phagocytic cup base
def: "The older part of the phagocytic cup where the actin cytoskeleton disassembles, allowing early incoming and outgoing vesicular trafficking." [GOC:pf, PMID:20200225]
GO:0097208
name: alveolar lamellar body
def: "A specialized secretory organelle found in type II pneumocytes and involved in the synthesis, secretion, and reutilization of pulmonary surfactant." [GOC:cjm, http://en.wikipedia.org/wiki/Lamellar_granule]
GO:0097209
name: epidermal lamellar body
def: "A specialized secretory organelle found in keratinocytes and involved in the formation of an impermeable, lipid-containing membrane that serves as a water barrier and is required for correct skin barrier function." [GOC:cjm, http://en.wikipedia.org/wiki/Lamellar_granule]
GO:0097217
name: sieve area
def: "A pit-like area in the cell wall of a sieve element; contains pores lined with callose and occupied by strands of protoplasmic material that interconnect the protoplasts of contiguous sieve elements." [ISBN:0471738433, POC:curators]
GO:0097218
name: sieve plate
def: "A part of the cell wall of a sieve tube member that bears one or more highly specialized sieve areas." [ISBN:0471738433, POC:curators]
GO:0097219
name: compound sieve plate
def: "A sieve plate that contains several specialized sieve areas in either a scalariform or reticulate arrangement." [ISBN:0471738433, POC:curators]
GO:0097220
name: simple sieve plate
def: "A sieve plate that contains a single specialized sieve area." [ISBN:0471738433, POC:curators]
GO:0097221
name: MADS box-forkhead transcription factor complex
def: "A protein complex that contains a MADS-box protein and two forkhead domain proteins, and binds to and regulates transcription from promoters of genes transcribed during the M/G1 transition of the cell cycle. In Schizosaccharomyces pombe, the complex contains the MADS-box protein Mbx1 and two forkhead proteins, Sep1 and Fkh2." [GOC:mah, PMID:18057023]
GO:0097223
name: sperm part
def: "Any constituent part of a sperm, a mature male germ cell that develops from a spermatid." [GOC:cjm]
GO:0097224
name: sperm connecting piece
def: "The segment of the sperm flagellum that attaches to the implantation fossa of the nucleus in the sperm head; from the remnant of the centriole at this point, the axoneme extends throughout the length of the flagellum." [GOC:cjm, MP:0009830]
GO:0097225
name: sperm midpiece
def: "The highly organized segment of the sperm flagellum which begins at the connecting piece and is characterized by the presence of 9 outer dense fibers (ODFs) that lie outside each of the 9 outer axonemal microtubule doublets and by a sheath of mitochondria that encloses the ODFs and the axoneme; the midpiece terminates about one-fourth of the way down the sperm flagellum at the annulus, which marks the beginning of the principal piece." [GOC:cjm, MP:0009831]
GO:0097226
name: sperm mitochondrial sheath
def: "The tightly packed helical sheath of ATP-producing mitochondria restricted to the midpiece of the sperm flagellum." [GOC:cjm, MP:0009832]
GO:0097227
name: sperm annulus
def: "The ring-like, filamentous structure located at the distal end of the midpiece of the sperm flagellum; the annulus is thought to form a diffusion barrier between the midpiece and the principal piece and serve as a stabilizing structure for tail rigidity." [GOC:cjm, MP:0009834]
GO:0097228
name: sperm principal piece
def: "The segment of the sperm flagellum where the mitochondrial sheath ends, and the outer dense fibers (ODFs) associated with outer axonemal doublets 3 and 8 are replaced by the 2 longitudinal columns of the fibrous sheath (FS) which run the length of the principal piece and are stabilized by circumferential ribs. The principal piece makes up ~2/3 of the length of the sperm flagellum and is defined by the presence of the FS and of only 7 (rather than 9) ODFs which taper and then terminate near the distal end of the principal piece." [GOC:cjm, MP:0009836]
GO:0097229
name: sperm end piece
def: "The short tip of the sperm flagellum, adjacent to the sperm principal piece and furthest from the sperm head, which contains only the axoneme surrounded by the plasma membrane." [GOC:cjm, GOC:sart, MP:0009837]
GO:0097232
name: lamellar body membrane
def: "The lipid bilayer surrounding a lamellar body. A lamellar body is a membrane-bounded organelle, specialized for the storage and secretion of various substances (surfactant phospholipids, glycoproteins and acid phosphates) which are arranged in the form of tightly packed, concentric, membrane sheets or lamellae. Has some similar properties to, but is distinct from, a lysosome." [GOC:sl, PMID:11940594]
GO:0097233
name: alveolar lamellar body membrane
def: "The lipid bilayer surrounding an alveolar lamellar body, a specialized secretory organelle found in type II pneumocytes and involved in the synthesis, secretion, and reutilization of pulmonary surfactant." [GOC:sl, PMID:11940594]
GO:0097234
name: epidermal lamellar body membrane
def: "The lipid bilayer surrounding an epidermal lamellar body, a specialized secretory organelle found in keratinocytes and involved in the formation of an impermeable, lipid-containing membrane that serves as a water barrier and is required for correct skin barrier function." [GOC:sl, PMID:11940594]
GO:0097249
name: mitochondrial respiratory chain supercomplex
def: "A set of respiratory enzyme complexes of the mitochondrial inner membrane (including, for example, Complex II, Complex III, Complex IV, or F1-F0 ATPase) arranged to form a large supercomplex." [GOC:mcc, PMID:21909073, PMID:22342701]
GO:0097255
name: R2TP complex
def: "A highly conserved protein complex comprised of two ATP-dependent DNA helicases (Rvb1p and Rvb2p in yeast, Pontin52 and Reptin52 in humans), Pih1p in yeast or PIH1D1 in humans, and Tah1 in yeast or RPAP3 in humans. The complex associates with Hsp90 and is thought to have a role in assembly of large protein or protein/nucleic acid complexes. In this role it is involved in multiple processes such as box C/D snoRNP biogenesis, phosphatidylinositol-3 kinase-related protein kinase (PIKK) signaling, RNA polymerase II assembly, and others." [GOC:mcc, PMID:15766533, PMID:21925213]
GO:0097268
name: cytoophidium
def: "A subcellular filamentary structure where CTP synthase is compartmentalized in a range of organisms including bacteria, yeast, fruit fly, rat and human." [GOC:mag, http://en.wikipedia.org/wiki/CTP_synthase#Cytoophidium, PMID:20513629, PMID:21930098]
GO:0097311
name: biofilm matrix
def: "A structure lying external to bacterial cells. A biofilm is an aggregate of surface-associated bacteria, and the biofilm matrix is the envelope of polymeric substances that surrounds the bacteria." [GOC:imk, http://en.wikipedia.org/wiki/Biofilm, PMID:22571672]
GO:0097312
name: biofilm matrix part
def: "Any constituent part of the biofilm matrix, a structure lying external to bacterial cells. A biofilm is an aggregate of surface-associated bacteria, and the biofilm matrix is the envelope of polymeric substances that surrounds the bacteria." [GOC:imk, http://en.wikipedia.org/wiki/Biofilm, PMID:22571672]
GO:0097313
name: biofilm matrix surface
def: "The external part of the biofilm matrix, a structure lying external to bacterial cells. A biofilm is an aggregate of surface-associated bacteria, and the biofilm matrix is the envelope of polymeric substances that surrounds the bacteria." [GOC:imk, http://en.wikipedia.org/wiki/Biofilm, PMID:22571672]
GO:0097342
name: ripoptosome
def: "A protein complex whose core components are the receptor-interacting serine/threonine-protein kinases RIPK1 and RIPK3 (also called RIP1 and RIP3). Formation of the ripoptosome can induce an extrinsic apoptotic signaling pathway or a necroptotic signaling pathway. The composition of this protein complex may depend on several factors including nature of the signal, cell type and more." [GOC:bhm, GOC:mtg_apoptosis, PMID:22265414, PMID:22274400]
GO:0097344
name: Rix1 complex
def: "A protein complex that comprises Rix1p, Ipi1p and Ipi3p, and is required for processing of ITS2 sequences from 35S pre-rRNA. The Rix1 complex has been identified in budding yeast and fission yeast, and members of this complex are conserved in higher eukaryotes." [GOC:vw, PMID:14759368, PMID:15260980, PMID:21385875]
GO:0097346
name: INO80-type complex
def: "A chromatin remodeling protein complex initially purified from S. cerevisiae and containing more than 10 subunits, including the SWR1-related complexes. INO80 (inositol requiring 80)-type complexes have diverse functions, including promoting transcriptional activation and DNA repair." [GOC:rb, PMID:19355820]
GO:0097347
name: TAM protein secretion complex
def: "A heterooligomeric protein complex that spans the bacterial periplasm and enables the secretion of adhesin proteins in Gram-negative bacteria. In Citrobacter rodentium, Salmonella enterica and Escherichia coli, the TAM complex consists of an Omp85-family protein, TamA, in the outer membrane and TamB in the inner membrane." [GOC:am, PMID:22466966]
GO:0097348
name: host cell endocytic vesicle membrane
def: "The lipid bilayer surrounding a host cell endocytic vesicle." [GOC:ecd]
GO:0097356
name: perinucleolar compartment
def: "The perinucleolar compartment (PNC) is a subnuclear structure associated with, but structurally distinct from, the nucleolus. The PNC contains large amounts of the heterogeneous nuclear ribonucleoprotein complex (hnRNP) called hnRNP 1 (PTB). Many RNA binding proteins as well as RNA polymerase III transcripts are highly enriched in this compartment. PTB and pol III transcripts are required for the integrity of the PNC." [GOC:vw, http://en.wikipedia.org/wiki/Perinucleolar_compartment, PMID:21385875]
GO:0097361
name: CIA complex
def: "The cytosolic iron-sulfur protein assembly (CIA) complex mediates the incorporation of iron-sulfur clusters into apoproteins involved in DNA metabolism and genomic integrity." [GOC:sp, PMID:22678362]
GO:0097362
name: MCM8-MCM9 complex
def: "A hexameric protein complex composed of MCM8 and MCM9 and involved in homologous recombination repair following DNA interstrand cross-links." [GOC:sp, PMID:22771115, PMID:22771120]
GO:0097373
name: MCM core complex
def: "A protein complex that contains Mcm4, Mcm6, and Mcm7 proteins, and possesses DNA helicase activity. In the heterohexameric MCM complex, the Mcm4/6/7 proteins form a stable core, and Mcm2, Mcm3, and Mcm5 are more peripherally associated." [GOC:mah, PMID:10770926, PMID:15007098, PMID:9305914]
GO:0097381
name: photoreceptor disc membrane
def: "Ovally-shaped membranous stack located inside the photoreceptor outer segment, and containing densely packed molecules of the photoreceptor protein rhodopsin that traverse the lipid bilayer. Disc membranes are apparently derived from the plasma membrane in the region of the cilium that connects the photoreceptor outer segment to the inner segment." [GOC:bj, PMID:11826267, PMID:2537204, PMID:7507907]
GO:0097386
name: glial cell projection
def: "A prolongation or process extending from a glial cell." [GOC:mc]
GO:0097387
name: capitate projection
def: "Simple or compound process of epithelial glial cells with a spherical head that inserts into photoreceptor axons. Capitate projections have only been observed in Brachycera (flies)." [GOC:mc, PMID:3098431]
GO:0097407
name: Bunina body
def: "Small granular inclusions (about 1-3 microns in diameter) found in the anterior horn cells, and appearing either singly or in a group. Sometimes they are arranged in small beaded chains. Bunina bodies express cystatin C and consist of electron-dense amorphous material that contains tubules or vesicular structures. The amorphous material frequently includes a cytoplasmic island containing neurofilaments and other micro-organelles." [NIF_Subcellular:nlx_subcell_20090101, PMID:18026741]
GO:0097408
name: fibrillary inclusion
def: "Cellular inclusion consisting of circular areas filled with fine slender filaments about 10 nanometers in diameter, delimited by a wall of varying complexity (either a single continuous membrane or a tubular network consisting of a fine filamentous material giving the wall a honeycomb appearance). Fibrillary inclusions are found in the cytoplasm of giant cells of Dieters in the lateral vestibular nucleus of the rat; similar structures have been described in the ventral cochlear nucleus, spinal cord, and substantia nigra." [NIF_Subcellular:sao967812059]
GO:0097409
name: glial cytoplasmic inclusion
def: "Non-membrane-bound cytoplasmic inclusions composed of 10-40 nm granule-coated fibrils. These inclusions have an abnormal accumulation of alpha-synuclein protein and are found in association with multiple system atrophy." [NIF_Subcellular:nlx_subcell_20090703, PMID:21562886, PMID:2559165]
GO:0097412
name: hyaline inclusion
def: "A glass-like, pale intracellular inclusion." [NIF_Subcellular:nlx_subcell_20090104]
GO:0097413
name: Lewy body
def: "Cytoplasmic, spherical inclusion commonly found in damaged neurons, and composed of abnormally phosphorylated, neurofilament proteins aggregated with ubiquitin and alpha-synuclein." [NIF_Subcellular:sao4933778419]
GO:0097414
name: classical Lewy body
def: "Cytoplasmic inclusion, 5 to 15 micrometers in diameter, with a dense core surrounded by a halo of 10 to 20 nm wide radially oriented alpha-synuclein fibrils." [NIF_Subcellular:sao4749542545]
GO:0097415
name: cortical Lewy body
def: "Cytoplasmic inclusion similar to a classical Lewy body but lacking a halo of protein fibrils." [NIF_Subcellular:sao4040591221]
GO:0097416
name: Lewy body-like hyaline inclusion
def: "Cytoplasmic inclusion found in neurons. It consists of filaments and granular materials, exhibits a dense core with a rough peripheral halo and lacks a limiting membrane. The filaments of these inclusions are composed of approximately 15-25 nm granule-coated fibrils in association with normal 10-nm neurofilaments." [NIF_Subcellular:nlx_subcell_20090105, PMID:18026741]
GO:0097417
name: nematosome
def: "Cytoplasmic, ball-like inclusion resembling a nucleolus and consisting of a convoluted network of electron-opaque strands embedded in a less dense matrix. It measures approximately 0.9 microns and lacks a limiting membrane. Its strands (diameter = 400-600 A) appear to be made of an entanglement of tightly packed filaments and particles approximately 25-50 A thick. Cytochemical studies suggest the presence of nonhistone proteins and some RNA. Usually only one such structure is present in a cell, and it appears to occur in most ganglion cells. Although they can be seen anywhere in the cell body, nematosomes are typically located in the perinuclear cytoplasm, where they are often associated with smooth-surfaced and coated vesicles." [NIF_Subcellular:sao138430598, PMID:5458990]
GO:0097418
name: neurofibrillary tangle
def: "Intracellular mass of paired, helically wound protein filaments (also called PHF) lying in the cytoplasm of neuronal cell bodies and neuritic cell processes. Neurofibrillary tangles contain an abnormally phosphorylated form of a microtubule-associated protein, tau. The shape of these inclusions may resemble a flame or a star." [NIF_Subcellular:nlx_subcell_20090201, NIF_Subcellular:nlx_subcell_20090202, NIF_Subcellular:sao2409833926]
GO:0097419
name: Pick body
def: "Cellular inclusion composed of numerous tau fibrils arranged in a disorderly array. Tau protein is a major component, though Pick bodies also contain ubiquitin, alpha-synuclein, and apolipoprotein E." [NIF_Subcellular:nlx_subcell_20090102]
GO:0097420
name: skein-like inclusion
def: "Intracytoplasmic filamentous structure frequently encountered in preparations immunostained for ubiquitin." [NIF_Subcellular:nlx_subcell_20090103, PMID:18026741]
GO:0097422
name: tubular endosome
def: "A network of fine tubules in the vicinity of the Golgi complex and around the centriole." [NIF_Subcellular:sao1570660411, NIF_Subcellular:sao694815499, PMID:11896161]
GO:0097423
name: mitochondrion-associated adherens complex
def: "An organelle arrangement comprised of the following elements: a mitochondrion positioned near the presynaptic membrane; an electron-dense mitochondrial plaque adjacent to the outer mitochondrial membrane that faces the presynaptic membrane; filament-like elements appearing to link the mitochondrial plaque to a cell-cell junction region (sometimes termed punctum adherens); tubular or vesicular-appearing membrane (also called vesicular chain) interposed among the filaments. Mitochondrion-associated adherens complexes were initially described in the dorsal horn of the spinal cord. They are found in calyces and other large terminals of the auditory brainstem, and in a variety of mammalian species including humans." [NIF_Subcellular:sao1933817066, PMID:20089910]
GO:0097424
name: nucleolus-associated heterochromatin
def: "Dense particles of heterochromatin, consisting of a loosely twisted strand about 600 Angstrom thick, found associated with the nucleolus." [NIF_Subcellular:sao1210952635]
GO:0097425
name: smooth endoplasmic reticulum part
def: "Any constituent part of the smooth endoplasmic reticulum (also called smooth ER, or SER)." [NIF_Subcellular:sao184202831]
GO:0097426
name: glial filament
def: "An intermediate filament composed of glial fibrillary acidic protein (GFAP) and found in astrocytes." [NIF_Subcellular:sao-1863852493]
GO:0097427
name: microtubule bundle
def: "An arrangement of closely apposed microtubules running parallel to each other." [NIF_Subcellular:sao1872343973]
GO:0097431
name: mitotic spindle pole
def: "Either of the ends of a mitotic spindle, a spindle that forms as part of mitosis, where spindle microtubules are organized; usually contains a microtubule organizing center and accessory molecules, spindle microtubules and astral microtubules." [GOC:vw]
GO:0097433
name: dense body
def: "An electron dense body which may contain granules." [ISBN:0195065719, NIF_Subcellular:sao730872736]
GO:0097440
name: apical dendrite
def: "A dendrite that emerges near the apical pole of a neuron. In bipolar neurons, apical dendrites are located on the opposite side of the soma from the axon." [NIF_Subcellular:sao273773228]
GO:0097441
name: basilar dendrite
def: "A dendrite that emerges near the basal pole of a neuron. In bipolar neurons, basal dendrites are either on the same side of the soma as the axon, or project toward the axon." [NIF_Subcellular:sao1079900774]
GO:0097442
name: CA3 pyramidal cell dendrite
def: "A dendrite of a hippocampal CA3 pyramidal cell." [NIF_Subcellular:nlx_subcell_1005001]
GO:0097443
name: sorting endosome
def: "A multivesicular body surrounded by and connected with multiple tubular compartments with associated vesicles." [NIF_Subcellular:sao1028571114]
GO:0097444
name: spine apparatus
def: "A specialization of the endomembrane system found in some classes of neuron spines consisting of two or more closely apposed lamellae with interspersed dense material." [NIF_Subcellular:sao725931194]
GO:0097445
name: presynaptic active zone dense projection
def: "Electron dense projection extending from the cytomatrix into the cytoplasm on which synaptic vesicles are tethered." [NIF_Subcellular:sao494258938, PMID:15381754]
GO:0097447
name: dendritic tree
def: "The entire complement of dendrites for a neuron, consisting of each primary dendrite and all its branches." [NIF_Subcellular:sao172297168]
GO:0097448
name: spine mat
def: "A configuration of neuron spines found on ciliary ganglion neurons in the embryonic and adult brain consisting of patches of closely spaced spines lying flat against the soma." [NIF_Subcellular:sao2128156969, PMID:10818137]
GO:0097449
name: astrocyte projection
def: "A prolongation or process extending from the soma of an astrocyte and wrapping around neurons." [NIF_Subcellular:sao1630537580]
GO:0097450
name: astrocyte end-foot
def: "Terminal process of astrocyte abutting non-neuronal surfaces in the brain." [NIF_Subcellular:sao388182739]
GO:0097451
name: glial limiting end-foot
def: "Terminal process of astrocyte that extends to the surface of the central nervous system. Together, glial limiting end-feet form the glial limiting membrane or glia limitans." [NIF_Subcellular:sao181458425]
GO:0097452
name: GAIT complex
def: "A protein complex which mediates interferon-gamma-induced transcript-selective translation inhibition in inflammation processes. The complex binds to stem loop-containing GAIT elements in the 3'-UTR of diverse inflammatory mRNAs and suppresses their translation by blocking the recruitment of the 43S ribosomal complex to m7G cap-bound eIF4G. In humans it includes RPL13A, EPRS, SYNCRIP and GAPDH; mouse complexes lack SYNCRIP." [GOC:br, PMID:15479637, PMID:23071094]
GO:0097453
name: mesaxon
def: "Portion of the ensheathing process (either myelin or non-myelin) where the enveloping lips of the ensheathing cell come together so that their apposed plasma membranes run parallel to each other, separated by a cleft 12 nm wide." [ISBN:01950657191, NIF_Subcellular:sao2127666702]
GO:0097454
name: Schwann cell microvillus
def: "Small finger-like extension of a Schwann cell that contacts the nodal membrane." [NIF_Subcellular:sao1890444066, PMID:15988042]
GO:0097455
name: spiny bracelet of Nageotte
def: "Paranodal terminations of Schwann cells that do not directly contact the paranodal axon membrane. Usually found in thicker myelin." [NIF_Subcellular:sao937871668, PMID:15988042]
GO:0097456
name: terminal loop
def: "Portion of myelin-forming Schwann cell consisting of terminal cytoplasmic extensions adhered to the axon at the beginning and end of the myelin sheath." [NIF_Subcellular:sao924713546]
GO:0097457
name: hippocampal mossy fiber
def: "Axon of dentate gyrus granule cell projecting to hippocampal area CA3, characterized by expansions (mossy fiber expansions) giving the fibers a mossy appearance. These unmyelinated axons were first described by Ramon y Cajal." [NIF_Subcellular:nlx_subcell_100312, PMID:17765709]
GO:0097458
name: neuron part
def: "Any constituent part of a neuron, the basic cellular unit of nervous tissue. A typical neuron consists of a cell body (often called the soma), an axon, and dendrites. Their purpose is to receive, conduct, and transmit impulses in the nervous system." [GOC:pr, http://en.wikipedia.org/wiki/Neuron]
GO:0097462
name: Lewy neurite
def: "Elongated neuronal process, often with side branches and more than one branching point, described in brains of patients with Parkinson's disease. Lewy neurites stain positively for ubiquitin in brainstem and forebrain regions affected in Parkinson's disease." [NIF_Subcellular:sao601362597]
GO:0097463
name: gemmule
def: "Spine-like process found on some neurons, e.g., periglomerular cells of olfactory cortex." [NIF_Subcellular:nlx_subcell_1005003]
GO:0097464
name: thorny excrescence
def: "Large complex spine protruding from a dendrite. Each excrescence is formed by a cluster of spine heads." [NIF_Subcellular:nlx_467, PMID:730852]
GO:0097465
name: somatic spine
def: "Spine emanating from the cell soma of a neuron." [NIF_Subcellular:sao2048514053]
GO:0097467
name: type III terminal bouton
def: "Terminal inflated portion of the axon of a non-glutamatergic neuron, containing the specialized apparatus necessary to release neurotransmitters at a regulatory synapse. The axon terminus is considered to be the whole region of thickening and the terminal bouton is a specialized region of it. Type III terminal boutons are larger than type II ones." [GOC:mc, PMID:10218156]
GO:0097470
name: ribbon synapse
def: "Type of synapse characterized by an electron-dense ribbon, lamella (bar) or spherical body in the presynaptic process cytoplasm." [NIF_Subcellular:sao1884931180]
GO:0097471
name: mossy fiber rosette
def: "A synapse of a mossy fiber onto the dendrite of a granule cell; each mossy fiber can have up to 50 rosettes." [http://en.wikipedia.org/wiki/Mossy_fiber_(cerebellum), NIF_Subcellular:nlx_subcell_091021]
GO:0097478
name: leaflet of membrane bilayer
def: "Any of the two layers of lipid molecules that constitute a membrane." [GOC:cjm]
GO:0097481
name: neuronal postsynaptic density
def: "A postsynaptic density that is part of a neuron." [GOC:BHF, GOC:pr, GOC:rl]
GO:0097482
name: muscle cell postsynaptic density
def: "A postsynaptic density that is part of a muscle cell." [GOC:pr]
GO:0097483
name: glial cell postsynaptic density
def: "A postsynaptic density that is part of a glial cell." [GOC:pr]
GO:0097486
name: multivesicular body lumen
def: "The volume enclosed by the outermost membrane of a multivesicular body." [GOC:pde, PMID:21183070]
GO:0097487
name: multivesicular body, internal vesicle
def: "A membrane-bounded vesicle wholly contained within a multivesicular body." [GOC:pde, PMID:21183070]
GO:0097488
name: multivesicular body, internal vesicle membrane
def: "The lipid bilayer surrounding a multivesicular body internal vesicle." [GOC:pde, PMID:21183070]
GO:0097489
name: multivesicular body, internal vesicle lumen
def: "The volume enclosed by the membrane of the multivesicular body internal vesicle." [GOC:pde, PMID:21183070]
GO:0097495
name: H-NS-Hha complex
def: "A trimeric protein complex made up of an H-NS homodimer and an Hha monomer. In Enterobacteriaceae, this complex negatively regulates transcription of a range of genes." [GOC:bhm, PMID:21600204]
GO:0097504
name: Gemini of coiled bodies
def: "Nuclear bodies frequently found near or associated with Cajal bodies (also called coiled bodies or CBs). Gemini of coiled bodies, or 'gems', are similar in size and shape to CBs, and often indistinguishable under the microscope. Unlike CBs, gems do not contain small nuclear ribonucleoproteins (snRNPs); they contain a protein called survivor of motor neurons (SMN) whose function relates to snRNP biogenesis. Gems are believed to assist CBs in snRNP biogenesis, and to play a role in the etiology of spinal muscular atrophy (SMA)." [GOC:pr, http://en.wikipedia.org/wiki/Cell_nucleus#Cajal_bodies_and_gems, PMID:11031238, PMID:9683623]
GO:0097505
name: Rad6-Rad18 complex
def: "A ubiquitin ligase complex found to be involved in post-replicative bypass of UV-damaged DNA and UV mutagenesis. In S. cerevisiae, the complex contains the ubiquitin conjugating enzyme Rad6 and Rad18, a protein containing a RING finger motif and a nucleotide binding motif. The yeast Rad6-Rad18 heterodimer has ubiquitin conjugating activity, binds single-stranded DNA, and possesses single-stranded DNA-dependent ATPase activity." [GOC:jd, PMID:9287349]
GO:0097511
name: dendritic cell dendrite
def: "A branched cellular projection (or cytoplasmic extension) that is extended from the surface of a dendritic immune cell, and which enables the cell to sample luminal pathogens and increase the surface area for antigen presentation to T cells." [CL:0000451, GOC:BHF, GOC:cjm, PMID:12200351]
GO:0097512
name: cardiac myofibril
def: "A cardiac myofibril is a myofibril specific to cardiac muscle cells." [GOC:cjm, GOC:devbiol]
GO:0097513
name: myosin II filament
def: "A bipolar filament composed of myosin II molecules." [GOC:cjm, GOC:mah]
GO:0097514
name: sexual spore wall
def: "A specialized envelope lying outside the cell membrane of a spore derived from a product of meiosis." [GOC:cjm, GOC:mah]
GO:0097515
name: asexual spore wall
def: "A specialized envelope lying outside the cell membrane of a spore derived from an asexual process. Examples of this process are found in bacterial and fungal species." [GOC:cjm, GOC:mah]
GO:0097516
name: microvillar actin bundle
def: "A parallel bundle of actin filaments at the core of a microvillus." [GOC:cjm, GOC:mah]
GO:0097517
name: contractile actin filament bundle
def: "An actin filament bundle in which the filaments are loosely packed (approximately 30-60 nm apart) and arranged with opposing polarities; the loose packing allows myosin (usually myosin-II) to enter the bundle." [GOC:cjm, GOC:mah, ISBN:0815316194]
GO:0097518
name: parallel actin filament bundle
def: "An actin filament bundle in which the filaments are tightly packed (approximately 10-20 nm apart) and oriented with the same polarity." [GOC:cjm, GOC:mah, ISBN:0815316194]
GO:0097519
name: DNA recombinase complex
def: "A protein-DNA complex consisting of a higher-order oligomer of strand exchange proteins (recombinases) on single-stranded DNA." [GOC:cjm, PMID:10357855]
GO:0097520
name: nucleotide-excision repair, preincision complex
def: "A multiprotein complex involved in damage recognition, DNA helix unwinding, and endonucleolytic cleavage at the site of DNA damage." [GOC:cjm, GOC:elh, PMID:10197977]
GO:0097522
name: protein-DNA ISRE complex
def: "A protein-DNA complex formed through interaction of the protein(s) with an interferon-stimulated response element (ISRE) in the DNA." [GOC:amm, GOC:cjm, PMID:11747630]
GO:0097523
name: transcription ternary complex
def: "A protein-DNA-RNA complex composed of RNA polymerase, template DNA, and an RNA transcript." [GOC:cjm, GOC:txnOH]
GO:0097524
name: sperm plasma membrane
def: "A plasma membrane that is part of a sperm cell." [GOC:cjm]
GO:0097525
name: spliceosomal snRNP complex
def: "A ribonucleoprotein complex involved in formation of the spliceosome and composed of one or more snRNA and multiple protein components." [GOC:pr, ISBN:0879695897]
GO:0097526
name: spliceosomal tri-snRNP complex
def: "A spliceosomal snRNP complex containing U4 and U6 (or U4atac and U6atac) snRNAs and U5 snRNAs and associated proteins." [GOC:pr, ISBN:0879695897, PMID:9452384]
GO:0097537
name: Y-shaped link
def: "A Y-shaped protein complex in the ciliary transition zone that connects the cilium axoneme to the ciliary necklace. Both protein sorting and protein gating occur at this point in the cilium allowing some, but not all proteins to enter the cilium." [GOC:cilia, PMID:22653444, PMID:4554367]
GO:0097538
name: ciliary necklace
def: "A protein complex located on the cilium membrane in the ciliary transition zone; it is connected to the cilium axoneme via Y-shaped links." [GOC:cilia, PMID:22653444, PMID:4554367]
GO:0097539
name: ciliary transition fiber
def: "A nine-bladed, propeller-like protein complex that links the distal end of the basal body and the cilium to the plasma membrane. Functions in protein sorting and gating (i.e. active and passive transport of proteins in and out of the cilium)." [GOC:cilia, GOC:kmv, PMID:22653444, PMID:24231678, PMID:5064817, PMID:5335827]
GO:0097540
name: axonemal central pair
def: "Part of the axoneme consisting of the inner two microtubule doublets of the 9+2 axoneme occurring in most motile cilia." [GOC:cilia, PMID:24283352]
GO:0097541
name: axonemal basal plate
def: "Part of the axoneme consisting of a highly electron-dense region at the distal end of the ciliary transition zone within the axonemal lumen at which the axonemal central pair of microtubules is connected to the rest of the axonemal structure." [GOC:cilia, PMID:23352055, PMID:4554367]
GO:0097542
name: ciliary tip
def: "Part of the cilium where the axoneme ends. The ciliary tip has been implicated in ciliary assembly and disassembly, as well as signal transduction." [GOC:cilia, PMID:23970417]
GO:0097543
name: ciliary inversin compartment
def: "Proximal part of the ciliary shaft to which the inversin protein (also called Inv) specifically localizes. The inversin compartment appears to have a different protein composition than the rest of the cilium, although there is no structure that separates it form the distal part of the cilium." [GOC:cilia, PMID:19050042]
GO:0097544
name: ciliary shaft
def: "The mid part of a cilium between the ciliary base and ciliary tip that extends into the extracellular space." [GOC:cilia, PMID:19866682]
GO:0097545
name: axonemal outer doublet
def: "Part of an axoneme consisting in a doublet microtubule. Nine of these outer doublets form the 9+0 axoneme, while the 9+2 axoneme also contains a central pair. Dynein arms attached to the doublets provide the mechanism of movement of the cilium." [GOC:cilia, GOC:pr, http://en.wikipedia.org/wiki/Axoneme, PMID:5044758, PMID:5664206]
GO:0097546
name: ciliary base
def: "Area of the cilium (also called flagellum) where the basal body and the axoneme are anchored to the plasma membrane. The ciliary base encompasses the distal part of the basal body, transition fibers and transition zone and is structurally and functionally very distinct from the rest of the cilium. In this area proteins are sorted and filtered before entering the cilium, and many ciliary proteins localize specifically to this area." [GOC:cilia, PMID:22653444]
GO:0097547
name: synaptic vesicle protein transport vesicle
def: "A cytoplasmic vesicle composed of both tubulovesicular and clear core vesicles that transport synaptic vesicle-associated proteins. Proteins carried by synaptic vesicle protein transport vesicles (STVs) include synaptophysin, synapsin Ia, synaptotagmin and synaptobrevin/vesicle-associated membrane protein 2 (VAMP2). STVs are packaged via the trans-Golgi network before being transported through the axon." [GOC:dr, PMID:21569270]
GO:0097550
name: transcriptional preinitiation complex
def: "A protein-DNA complex composed of proteins binding promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription." [GOC:di, PMID:22751016]
GO:0097554
name: left anterior flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It originates at the left anterior basal body, extends laterally through the cytoplasm, crosses the right anterior axoneme, and exits as a membrane-bound flagellum on the anterior left side of the cell." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097555
name: right anterior flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It originates at the right anterior basal body, extends laterally through the cytoplasm, crosses the left anterior axoneme, and exits as a membrane-bound flagellum on the anterior right side of the cell." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097556
name: left posteriolateral flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the left posteriolateral basal body and extends cytoplasmically toward the cell posterior, marking the left anterior boundary of the lateral shield and the left lateral region of the funis before exiting at the left lateral region of the cell body." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097557
name: right posteriolateral flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the right posteriolateral basal body and extends cytoplasmically toward the cell posterior, marking the right anterior boundary of the lateral shield and the right lateral region of the funis before exiting at the right lateral region of the cell body." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097558
name: left ventral flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the left ventral basal body and exits the cell body proximally and dorsal to the ventral disc." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097559
name: right ventral flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the right ventral basal body and exits the cell body proximally and dorsal to the ventral disc." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097560
name: left caudal flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the left caudal basal body, extending cytoplasmically and exiting at the posterior end of the cell body." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097561
name: right caudal flagellum
def: "A cilium (also called flagellum) found in Giardia species (trophozoite stage). It is nucleated by the right caudal basal body, extending cytoplasmically and exiting at the posterior end of the cell body." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097562
name: left lateral basal body pair
def: "Set of two basal bodies found in Giardia species (trophozoite stage). It comprises the anterior and ventral basal bodies located to the right of the left nucleus of the trophozoite when viewed dorsally." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097563
name: left middle basal body pair
def: "Set of two basal bodies found in Giardia species (trophozoite stage). It comprises the caudal and posteriolateral basal bodies located to the right of the left nucleus of the trophozoite when viewed dorsally." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097564
name: right lateral basal body pair
def: "Set of two basal bodies found in Giardia species (trophozoite stage). It comprises the anterior and ventral basal bodies located to the left of the right nucleus of the trophozoite when viewed dorsally." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097565
name: right middle basal body pair
def: "Set of two basal bodies found in Giardia species (trophozoite stage). It comprises the caudal and posteriolateral basal bodies located to the left of the right nucleus of the trophozoite when viewed dorsally." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097566
name: left tetrad
def: "Set of four basal bodies found in Giardia species (trophozoite stage). It comprises the left lateral basal body pair and the left middle basal body pair (i.e. the anterior, ventral, caudal and posteriolateral basal bodies located to the right of the left nucleus of the trophozoite when viewed dorsally)." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097567
name: right tetrad
def: "Set of four basal bodies found in Giardia species (trophozoite stage). It comprises the right lateral basal body pair and the right middle basal body pair (i.e. the anterior, ventral, caudal and posteriolateral basal bodies located to the left of the right nucleus of the trophozoite when viewed dorsally)." [GOC:giardia, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:0097568
name: median body
def: "A non-membrane bound, semi-organized microtubule array of unknown function found in Giardia species (trophozoite stage). It is located on the dorsal side of the trophozoite, slightly posterior to the ventral disc." [GOC:giardia, PMID:5961344]
GO:0097569
name: lateral shield
def: "Region of the ventral side of the cell body found in Giardia species (trophozoite stage). It is located posterior on either side of the ventral groove; the upper boundary is the ventral disc, and the lower boundary is marked by the posteriolateral flagella." [GOC:giardia, ISBN:9780124260207]
GO:0097570
name: cyst wall
def: "The specialized envelope lying outside the cell membrane of a cyst. A cyst is a resting or dormant stage of a microorganism, usually a bacterium or a protist or rarely an invertebrate animal, that helps the organism to survive in unfavorable environmental conditions. In protists such as protozoan parasites alternating cystic- and non-cystic stages, the cyst wall is usually composed of carbohydrates and proteins." [GOC:giardia, http://en.wikipedia.org/wiki/Microbial_cyst, PMID:15134259, PMID:2026212]
GO:0097571
name: left nucleus
def: "One of the two nuclei found in Giardia species (trophozoite stage). It is located on the left side of the cell when viewed from the dorsal side." [GOC:giardia, ISBN:0-444-81258-X]
GO:0097572
name: right nucleus
def: "One of the two nuclei found in Giardia species (trophozoite stage). It is located on the right side of the cell when viewed from the dorsal side." [GOC:giardia, ISBN:0-444-81258-X]
GO:0097574
name: lateral part of cell
def: "The region of a polarized cell other than its tips or ends (in some cell types, one end may be called the apex and the other the base). For example, in a polarized epithelial cell, the lateral part includes the cell sides which interface adjacent cells." [GOC:pr]
GO:0097575
name: lateral cell cortex
def: "The region directly beneath the plasma membrane of the lateral portion of the cell." [GOC:mah, PMID:24146635]
GO:0097582
name: dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt2p dimer complex
def: "A protein dimer complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity and, in S. cerevisiae, is composed of Pmt1p-Pmt2p." [GOC:jd, PMID:12551906]
GO:0097583
name: dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt3p dimer complex
def: "A protein dimer complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity and, in S. cerevisiae, is composed of Pmt1p-Pmt3p." [GOC:jd, PMID:12551906]
GO:0097584
name: dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt2p dimer complex
def: "A protein dimer complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity and, in S. cerevisiae, is composed of Pmt5p-Pmt2p." [GOC:jd, PMID:12551906]
GO:0097585
name: dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt3p dimer complex
def: "A protein dimer complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity and, in S. cerevisiae, is composed of Pmt5p-Pmt3p." [GOC:jd, PMID:12551906]
GO:0097586
name: dolichyl-phosphate-mannose-protein mannosyltransferase Pmt4p homodimer complex
def: "A protein dimer complex that possesses dolichyl-phosphate-mannose-protein mannosyltransferase activity and, in S. cerevisiae, is composed of Pmt4p." [GOC:bhm, GOC:jd, PMID:12551906]
GO:0097587
name: MutLgamma complex
def: "A heterodimer involved in the recognition of base-base and small insertion/deletion mismatches. In S. cerevisiae the complex consists of two subunits, Mlh1 and Mlh3." [GOC:jd, PMID:10570173]
GO:0097589
name: archaeal-type flagellum
def: "A non-membrane-bounded organelle superficially similar to a bacterial-type flagellum; they both consist of filaments extending outside the cell, and rotate to propel the cell, but the archaeal flagella (also called archaella) have a unique structure which lacks a central channel. Similar to bacterial type IV pilins, the archaeal flagellins (archaellins) are made with class 3 signal peptides and they are processed by a type IV prepilin peptidase-like enzyme. The archaellins are typically modified by the addition of N-linked glycans which are necessary for proper assembly and/or function." [GOC:cilia, GOC:krc, http://en.wikipedia.org/wiki/Flagellum#Archaeal, PMID:21265748, PMID:23146836, PMID:23204365, PMID:24330313]
GO:0097591
name: ventral disc lateral crest
def: "Fibrillar repetitive structure surrounding the ventral disc edge in Giardia species (trophozoite stage). The composition of the lateral crest is not fully known yet." [GOC:giardia]
GO:0097592
name: ventral disc overlap zone
def: "A region of the ventral disc of Giardia species (trophozoite stage) where two portions of the same array of microtubules overlap (the microtubule array makes a complete circle and overlaps on itself)." [GOC:giardia]
GO:0097593
name: ventral disc microtubule array
def: "A part of the ventral disc of Giardia species (trophozoite stage) consisting of a spiral array of microtubules linked to the ventral membrane. These microtubules form the base of the ventral disc dorsal microribbons that extend nearly perpendicular from the membrane." [GOC:giardia]
GO:0097594
name: ventral disc dorsal microribbon
def: "Trilaminar structure extending perpendicularly into the cytoplasm along the length of ventral disc microtubules in Giardia species (trophozoite stage). Constituents of dorsal microribbons (also called dorsal ribbons or microribbons) include alpha-coiled-helix proteins approximately 29 to 38 kDa in size. These proteins line the edges of the microribbons but are not found in microtubules. Tubulins are not found in microribbons." [GOC:giardia, PMID:11432808]
GO:0097595
name: ventral disc crossbridge
def: "Structure horizontally linking adjacent microribbons of the ventral disc in Giardia species (trophozoite stage). The composition of crossbridges is not fully known yet." [GOC:giardia]
GO:0097596
name: ventral disc supernumerary microtubule array
def: "A partial left-handed spiral array of microtubules that lies generally dorsal to the main ventral disc microtubule array in Giardia species (trophozoite stage)." [GOC:giardia, ISBN:9780124260207]
GO:0097597
name: ventral disc
def: "Specialized organelle found in Giardia species (trophozoite stage) and characterized by a spiral array of microtubules and microtubule-associated structures including dorsal microribbons and crossbridges. The edge of the ventral disc narrows into a lateral crest. The ventral disk mediates mechanical attachment of the trophozoite to the host's intestinal wall, and contains the contractile proteins actinin, alpha-actinin, myosin, and tropomyosin working towards contraction of the disk involved in adherence." [GOC:giardia, ISBN:9780124260207, PMID:11432808, PMID:4777416, PMID:5961344]
GO:0097598
name: sperm cytoplasmic droplet
def: "A small amount of cytoplasm surrounded by a cell membrane that is generally retained in spermatozoa after spermiogenesis, when the majority of the cytoplasm is phagocytosed by Sertoli cells to produce \"residual bodies\". Initially, the droplet is located at the neck just behind the head of an elongated spermatid. During epididymal transit, the cytoplasmic droplet migrates caudally to the annulus at the end of the midpiece; the exact position and time varies by species. The cytoplasmic droplet consists of lipids, lipoproteins, RNAs, a variety of hydrolytic enzymes, receptors, ion channels, and Golgi-derived vesicles. The droplet may be involved in regulatory volume loss (RVD) at ejaculation, and in most species, though not in humans, the cytoplasmic droplet is lost at ejaculation. Note that the cytoplasmic droplet is distinct from \"excessive residual cytoplasm\" that sometimes remains in epididymal spermatozoa, particularly when spermiogenesis has been disrupted." [GOC:krc, PMID:12672117, PMID:21076437, PMID:23159014]
GO:0097608
name: transverse flagellum
def: "A motile cilium found in dinoflagellates. It coils around the cell and provides the forward thrust for motility. It is often contained in a furrow called the cingulum, and emerges from a flagellar pore located in the cingulum." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097609
name: longitudinal flagellum
def: "A motile cilium found in dinoflagellates. It trails the cell and acts as a steering rudder. It is often partially contained in a furrow called the sulcus, and emerges from a flagellar pore located in the sulcus." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097610
name: cell surface furrow
def: "A furrow that may be found on the cell surface. Examples are the cleavage furrow observed during cytokinesis in animal cells, and the cingulum and sulcus found in some dinoflagellates." [GOC:pr]
GO:0097611
name: dinoflagellate cingulum
def: "A cell surface furrow that wraps around a dinoflagellate cell; the transverse flagellum lies in it." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097612
name: dinoflagellate sulcus
def: "A cell surface furrow that occurs on the ventral side of a dinoflagellate cell. It partially houses the longitudinal flagellum. The sulcus intersects with the cingulum on the ventral side of a dinoflagellate cell." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097613
name: dinoflagellate epicone
def: "The part of a dinoflagellate cell above the cingulum; also referred to as the anterior portion of a dinoflagellate cell. It is separated from the hypocone by the cingulum." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097614
name: dinoflagellate hypocone
def: "The part of a dinoflagellate cell below the cingulum; also referred to as the posterior portion of a dinoflagellate cell. It is separated from the epicone by the cingulum." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097618
name: dinoflagellate sulcal notch
def: "A dinoflagellate sulcus that extends all the way to the posterior end of the cell (also known as antapex). The presence of a sulcal notch makes the dinoflagellate hypocone appear bilobed." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097619
name: PTEX complex
def: "A protein complex that acts as a protein trafficking machinery and is responsible for the export of proteins across the parasitophorous (symbiont-containing) vacuolar membrane and into the human host cell. The PTEX complex is located in the vacuole membrane. It is ATP-powered, and comprises heat shock protein 101 (HSP101; a ClpA/B-like ATPase from the AAA+ superfamily, of a type commonly associated with protein translocons), a parasite protein termed PTEX150, and exported protein 2 (EXP2). EXP2 is the potential channel, as it is the membrane-associated component of the core PTEX complex. Two other proteins, PTEX88 and thioredoxin 2 (TRX2), were also identified as PTEX components." [GOC:pr, PMID:19536257, PMID:25043010, PMID:25043043]
GO:0097629
name: extrinsic component of omegasome membrane
def: "The component of the omegasome membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:mf, PMID:18725538, PMID:24591649]
GO:0097630
name: intrinsic component of omegasome membrane
def: "The component of the omegasome membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:mf, PMID:18725538, PMID:24591649]
GO:0097631
name: integral component of omegasome membrane
def: "The component of the omegasome membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:mf, PMID:18725538, PMID:24591649]
GO:0097632
name: extrinsic component of pre-autophagosomal structure membrane
def: "The component of the pre-autophagosomal structure membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:mf]
GO:0097633
name: intrinsic component of pre-autophagosomal structure membrane
def: "The component of the pre-autophagosomal structure membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:mf]
GO:0097634
name: integral component of pre-autophagosomal structure membrane
def: "The component of the pre-autophagosomal structure membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:mf]
GO:0097635
name: extrinsic component of autophagic vacuole membrane
def: "The component of the autophagic vacuole membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region." [GOC:mf]
GO:0097636
name: intrinsic component of autophagic vacuole membrane
def: "The component of the autophagic vacuole membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:mf]
GO:0097637
name: integral component of autophagic vacuole membrane
def: "The component of the autophagic vacuole membrane consisting of gene products and protein complexes that have some part that penetrates at least one leaflet of the membrane bilayer. This component includes gene products that are buried in the bilayer with no exposure outside the bilayer." [GOC:mf]
GO:0097648
name: G-protein coupled receptor complex
def: "A protein complex that contains G-protein coupled receptors." [GOC:bhm]
GO:0097649
name: A tubule
def: "A complete microtubule with 13 protofilaments that fuses with an incomplete microtubule called B tubule (containing 10 protofilaments only) to form an axonemal outer doublet. Inner and outer dynein arms, as well as the radial spoke, are attached to the A tubule." [GOC:cilia, ISBN:0716731363]
GO:0097650
name: B tubule
def: "An incomplete microtubule containing 10 protofilaments that fuses with a complete microtubule called A tubule (containing 13 protofilaments) to form an axonemal outer doublet." [GOC:cilia, ISBN:0716731363]
GO:0097651
name: phosphatidylinositol 3-kinase complex, class I
def: "A phosphatidylinositol 3-kinase complex that contains a catalytic and a regulatory subunit of a phosphatidylinositol 3-kinase (PI3K) enzyme, plus one or more adaptor proteins. Class I PI3Ks phosphorylate phosphatidylinositol [PI], phosphatidylinositol-4-phosphate [PI(4)P] and phosphatidylinositol-4,5-bisphosphate [PI(4,5)P2], and are divided into subclasses A and B according to the type of adaptor subunit with which they associate. The class I PI3K subfamily of genes comprises members in vertebrates, worm and fly, but none in yeast." [GOC:ha, PMID:24587488]
GO:0097652
name: phosphatidylinositol 3-kinase complex, class II
def: "A phosphatidylinositol 3-kinase complex that contains a catalytic subunit of a phosphatidylinositol 3-kinase (PI3K) enzyme and one or more adaptor proteins. There is no known obligatory regulatory subunit. The class II PI3K (PI3KC2) subfamily of genes has members in vertebrates, worm and fly, but none in yeast." [GOC:ha, PMID:24587488]
GO:0097653
name: unencapsulated part of cell
def: "The part of a cell encompassing the intracellular environment and the plasma membrane; it excludes any external encapsulating structures." [GOC:curators]
GO:0097654
name: platelet SNARE complex
def: "A SNARE complex that is capable of fusing intracellular vesicles to the plasma membrane of platelets for exocytosis of alpha-granules or dense granules. Contains isoforms of VAMP, SNAP and syntaxin proteins. Ternary SNARE complexes interact in a circular array to form ring complexes or channels around the membrane fusion. A common composition in human is VAMP-8, SNAP-23 and syntaxin-2 or -4." [GOC:bhm, PMID:12130530, PMID:19450911]
GO:0097658
name: Asi complex
def: "A nuclear ubiquitin ligase multiprotein complex located in the inner nuclear membrane (INM) that recognizes and ubiquitinates misfolded INM proteins and also some proteins involved in sterol biosynthesis, during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligases Asi1p and Asi3p." [GOC:mcc, PMID:25236469]
GO:0097660
name: SCF-Cdc4 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Cdc4 in S. cerevisiae." [GOC:jd, GOC:vw, PMID:9346238]
GO:0097661
name: SCF-Ctf13 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Ctf13 in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097662
name: SCF-Das1 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Das1 in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097663
name: SCF-Dia2/Pof3 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Dia2 in S. cerevisiae (Pof3 in S. pombe)." [GOC:jd, GOC:vw, PMID:14747994, PMID:15147268]
GO:0097664
name: SCF-Grr1/Pof2 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Grr1 in S. cerevisiae (Pof2 in S. pombe)." [GOC:jd, GOC:vw, PMID:10213692, PMID:15147268]
GO:0097665
name: SCF-Mdm30 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Mdm30 in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097666
name: SCF-Met30/Pof1 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Met30 in S. cerevisiae (Pof1 in S pombe)." [GOC:jd, GOC:vw, PMID:15147268, PMID:9499404]
GO:0097667
name: SCF-Rcy1/Pof6 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Rcy1 in S. cerevisiae (Pof6 in S. pombe)." [GOC:jd, GOC:vw, PMID:14747994, PMID:15147268]
GO:0097668
name: SCF-Saf1/Pof9 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Saf1 in S. cerevisiae (Pof9 in S. pombe)." [GOC:jd, GOC:vw, PMID:11283612, PMID:15147268]
GO:0097669
name: SCF-Skp2 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Skp2 in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097670
name: SCF-Ufo1/Pof10 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Ufo1 in S. cerevisiae (Pof10 in S. pombe)." [GOC:jd, GOC:vw, PMID:14747994, PMID:15147268]
GO:0097671
name: SCF-YDR131C ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is YDR131C in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097672
name: SCF-Pof5 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Pof5 in S. pombe (YDR306C in S. cerevisiae)." [GOC:jd, GOC:vw, PMID:14747994, PMID:15147268]
GO:0097673
name: SCF-YLR224W ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is YLR224W in S. cerevisiae." [GOC:jd, GOC:vw, PMID:14747994]
GO:0097674
name: SCF-YLR352W ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is YLR352W in S. cerevisiae." [GOC:jd, GOC:vw, PMID:19882662]
GO:0097675
name: SCF-Hrt3/Pof7 ubiquitin ligase complex
def: "An SCF ubiquitin ligase complex in which the F-box protein is Hrt3 in S. cerevisiae (Pof7 in S. pombe)." [GOC:jd, GOC:vw, PMID:14747994, PMID:15147268]
GO:0097679
name: other organism cytoplasm
def: "The cytoplasm of a secondary organism with which the first organism is interacting." [GOC:aa, PMID:23469034]
GO:0097683
name: dinoflagellate apex
def: "The anterior most point of a dinoflagellate epicone." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097684
name: dinoflagellate antapex
def: "The anterior most point of a dinoflagellate hypocone." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097685
name: dinoflagellate apical groove
def: "A cell surface furrow (or groove) found on a dinoflagellate apex. It typically loops around the apex." [GOC:at, http://en.wikipedia.org/wiki/Dinoflagellate#Morphology, http://tolweb.org/Dinoflagellates/2445, http://www.sms.si.edu/irlspec/Phyl_Dinofl_Glossary.htm, ISBN:0632009152]
GO:0097686
name: dinoflagellate apical horn
def: "A horn-shaped dinoflagellate apex found in thecate species." [GOC:at, http://species-identification.org]
GO:0097687
name: dinoflagellate antapical horn
def: "A horn-shaped dinoflagellate antapex found in thecate species." [GOC:at, http://species-identification.org]
GO:0098009
name: virus terminase, large subunit
def: "The part of the viral terminase complex that contains the translocase activity. The large subunit typically comprises a pentameric protein complex." [GOC:bm, PHI:0000073]
GO:0098015
name: virus tail
def: "Part of the virion that may be used to recognize, attach and inject the viral genome and accessory proteins into the host cell." [GOC:bm, PHI:0000174]
GO:0098017
name: viral capsid, major subunit
def: "The part of the viral capsid that comprises the most common capsomere type. For example, in a T=3 icosahedral capsid, which is composed of 12 pentameric and 20 hexameric capsomeres, the hexameric capsomeres are major subunits." [GOC:bm, PHI:0000066]
GO:0098018
name: viral capsid, minor subunit
def: "The part of the viral capsid that comprises the less common capsomere type. For example, in a T=3 icosahedral capsid, which is composed of 12 pentameric and 20 hexameric capsomeres, the pentameric capsomeres are minor subunits." [GOC:bm, PHI:00000185]
GO:0098019
name: virus tail, major subunit
def: "The part of the viral tail that comprises the most common subunit type." [GOC:bm, PHI:0000082]
GO:0098020
name: virus tail, minor subunit
def: "The part of the viral tail that comprises the least common subunit type." [GOC:bm, PHI:0000178]
GO:0098021
name: viral capsid, decoration
def: "Component of the virus capsid (head), located on the outer head surface. Involved in the stabilization of the head structure and usually non-essential." [GOC:bm, PHI:0000069]
GO:0098022
name: viral capsid, fiber
def: "A type of capsid decoration composed of fiber structures." [GOC:bm, PHI:0000070, PHI:0000176]
GO:0098023
name: virus tail, tip
def: "The basal end of the virus tail, which is used by the virus to attach to the host cell." [GOC:bm, PHI:0000087]
GO:0098024
name: virus tail, fiber
def: "The fibrous region of the virus tail used to scan, recognize and attach to the host cell." [GOC:bm, PHI:0000024, PHI:0000175]
GO:0098025
name: virus tail, baseplate
def: "Multiprotein component at the distal (head) end of the virus tail to which fibers of tailed viruses may be attached." [GOC:bm, PHI:0000088]
GO:0098026
name: virus tail, tube
def: "The internal tube of the contractile tails of some viruses. The virus tail tube is the channel for DNA ejection into the host cytoplasm." [GOC:bm, PHI:0000026]
GO:0098027
name: virus tail, sheath
def: "The external contractile envelope of the tail of some viruses. Its contraction ensures ejection of the virus DNA into the host cytoplasm." [GOC:bm, PHI:0000084]
GO:0098028
name: virus tail, shaft
def: "The tube of the non-contractile tails of some viruses." [GOC:bm, PHI:0000085]
GO:0098029
name: icosahedral viral capsid, spike
def: "A short structure attached to an icosahedral virion capsid, and used for attachment to the host cell." [GOC:bm, PHI:0000208]
GO:0098030
name: icosahedral viral capsid, neck
def: "A region of constriction located below the head and above the tail sheath of viruses with contractile tails (Myoviridae)." [GOC:bm, PHI:0000205, PHI:0000309]
GO:0098031
name: icosahedral viral capsid, collar
def: "A small disk located at the base of some icosahedral virus capsids." [GOC:bm, PHI:0000206, PHI:0000308]
GO:0098032
name: icosahedral viral capsid, collar fiber
def: "A fiber attached to the collar structure of some icosahedral viral capsids." [GOC:bm, PHI:0000207, PHI:0000307]
GO:0098033
name: icosahedral viral capsid, neck fiber
def: "A fiber attached to the neck at the base of some icosahedral viral capsids." [GOC:bm, PHI:0000075, PHI:0000310]
GO:0098046
name: type V protein secretion system complex
def: "A complex of proteins that permits the translocation of proteins across the outer membrane via a transmembrane pore, formed by a beta-barrel, into the extracellular milieu or directly into host cells; the secreted proteins contain all the information required for translocation of an effector molecule through the cell envelope. The type V secretion systems includes the autotransporters (type Va), the two-partner secretion system (type Vb) and the Oca family (type Vc)." [GOC:bf, GOC:bm, PMID:15119822, PMID:15590781]
GO:0098061
name: viral capsid, internal space
def: "The region of a virus contained within the capsid shell, and usually containing the viral genome and accessory proteins." [GOC:bm]
GO:0098520
name: excitatory neuromuscular junction
def: "The junction between the axon of a motor neuron and a muscle fiber. In response to the arrival of action potentials, the presynaptic button releases molecules of neurotransmitters into the synaptic cleft. These diffuse across the cleft and transmit the signal to the postsynaptic membrane of the muscle fiber, leading to a post-synaptic potential responsible for muscle contraction." [GOC:dos]
GO:0098521
name: inhibitory neuromuscular junction
def: "The junction between the axon of a motor neuron and a muscle fiber. In response to the arrival of action potentials, the presynaptic button releases molecules of neurotransmitters into the synaptic cleft. These diffuse across the cleft and transmit the signal to the postsynaptic membrane of the muscle fiber, leading to a change in post-synaptic potential that inhibits muscle contraction." [GOC:dos]
GO:0098522
name: neuromuscular junction of skeletal muscle fiber
def: "A neuromuscular junction in which the target muscle cell is a skeletal muscle fiber." [GOC:dos]
GO:0098523
name: neuromuscular junction of myotube
def: "A neuromuscular junction in which the target muscle cell is a myotube." [GOC:dos]
GO:0098524
name: neuromuscular junction of somatic muscle myotube
def: "A neuromuscular junction in which the target muscle cell is a somatic muscle myotube, such as an arthropod somatic muscle cell." [GOC:dos]
GO:0098525
name: excitatory neuromuscular junction of somatic myotube
def: "A neuromuscular junction that functions in the excitation of somatic muscle myotubes, such as an arthropod somatic muscle cells." [GOC:dos]
GO:0098526
name: inhibitory neuromuscular junction of somatic myotube
def: "A neuromuscular junction that functions in the inhibition of somatic muscle myotube contraction. Examples of somatic muscle myotubes include the somatic muscle cells of arthropods." [GOC:dos]
GO:0098527
name: neuromuscular junction of somatic muscle
def: "A neuromuscular junction in which the target muscle cell is a somatic muscle cell, such as those found in nematodes and arthropods." [GOC:dos]
GO:0098533
name: ATPase dependent transmembrane transport complex
def: "A transmembrane protein complex that functions in ATPase dependent active transport across a membrane." [GOC:dos]
GO:0098536
name: deuterosome
def: "A spherical, electron dense, cytoplasmic structure that is involved in de novo assembly of centrioles." [GOC:dos, PMID:24075808, PMID:5661997]
GO:0098537
name: lobed nucleus
def: "Nucleus with two or more lobes connected by a thin filament that contains no internal chromatin.  Examples include the nuclei of mature basophils, eosinophils and neutrophils in mice and humans." [GOC:dos, GOC:tfm]
GO:0098538
name: lumenal side of transport vesicle membrane
def: "The side (leaflet) of the transport vesicle membrane that faces the lumen." [GOC:ab]
GO:0098539
name: cytoplasmic side of transport vesicle membrane
def: "The side (leaflet) of the transport vesicle membrane that faces the cytoplasm." [GOC:ab]
GO:0098540
name: lumenal side of trans-Golgi network transport vesicle membrane
def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the lumen." [GOC:ab]
GO:0098541
name: cytoplasmic side of trans-Golgi network transport vesicle membrane
def: "The side (leaflet) of the trans-Golgi network transport vesicle membrane that faces the cytoplasm." [GOC:ab]
GO:0098547
name: lumenal side of Golgi membrane
def: "The side of the Golgi membrane that faces the lumen." [GOC:ab, GOC:dos]
GO:0098548
name: cytoplasmic side of Golgi membrane
def: "The side (leaflet) of the Golgi membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098549
name: somatic ring canal
def: "A stable intercellular bridge between somatic cells.  Examples include the intercellular bridges between ovarian follicle cells in insects and between imaginal disc cells in insects." [GOC:dos, PMID:22135360, PMID:670316]
GO:0098550
name: lumenal side of early endosome membrane
def: "The side (leaflet) of the early endosome membrane that faces the lumen." [GOC:lr]
GO:0098551
name: lumenal side of late endosome membrane
def: "The side (leaflet) of the late endosome membrane that faces the cytoplasm." [GOC:lr]
GO:0098552
name: side of membrane
def: "A cellular component consisting of one leaflet of a membrane bilayer and any proteins embedded or anchored in it or attached to its surface." [GOC:dos]
GO:0098553
name: lumenal side of endoplasmic reticulum membrane
def: "The side (leaflet) of the plasma membrane that faces the lumen." [GOC:ab, GOC:dos]
GO:0098554
name: cytoplasmic side of endoplasmic reticulum membrane
def: "The side (leaflet) of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098555
name: lumenal side of rough endoplasmic reticulum membrane
def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
GO:0098556
name: cytoplasmic side of rough endoplasmic reticulum membrane
def: "The side (leaflet) of the rough endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098557
name: cytoplasmic side of smooth endoplasmic reticulum membrane
def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098558
name: lumenal side of smooth endoplasmic reticulum membrane
def: "The side (leaflet) of the smooth endoplasmic reticulum membrane that faces the lumen." [GOC:ab, GOC:dos]
GO:0098559
name: cytoplasmic side of early endosome membrane
def: "The side (leaflet) of the early endosome membrane that faces the cytoplasm." [GOC:lr]
GO:0098560
name: cytoplasmic side of late endosome membrane
def: "The side (leaflet) of the late endosome membrane that faces the cytoplasm." [GOC:lr]
GO:0098561
name: methyl accepting chemotaxis protein complex
def: "A transmembrane protein complex that consists of multiple methyl-accepting chemoreceptor protein subunits, a histidine kinase and a connector protein and which functions in the regulation of flagellar rotary motor activity in response to an external chemical stimulus." [GOC:dos, PMID:1326408, PMID:15802240]
GO:0098562
name: cytoplasmic side of membrane
def: "The side of a membrane that faces the cytoplasm." [GOC:dos]
GO:0098563
name: intrinsic component of synaptic vesicle membrane
def: "The component of the synaptic vesicle membrane consisting of gene products and protein complexes that have some covalently attached part (e.g. peptide sequence or GPI anchor), which spans or is embedded in one or both leaflets of the membrane." [GOC:dos]
GO:0098564
name: trans-Golgi network transport vesicle lumen
def: "The volume enclosed within the membrane of a trans-Golgi network transport vesicle." [GOC:dos]
GO:0098565
name: lumenal side of endosome membrane
def: "The side (leaflet) of the endosome membrane that faces the lumen." [GOC:dos]
GO:0098566
name: transport vesicle lumen
def: "The volume enclosed within the membrane of a transport vesicle." [GOC:dos]
GO:0098567
name: periplasmic side of plasma membrane
def: "The side (leaflet) of a plasma membrane that faces the cytoplasm, and all proteins embedded in it or attached to its surface." [GOC:dos]
GO:0098568
name: external side of mycolate outer membrane
def: "The side (leaflet) of the mycolate outer membrane that faces the environment and any proteins embedded in it or loosely bound to its surface." [GOC:dos, PMID:18316738, PMID:18567661]
GO:0098569
name: internal side of mycolate outer membrane
def: "The side of the mycolate outer membrane that faces the cell wall peptidoglycan. It is rich in long-chain mycolic acids (hydroxylated branched-chain fatty acids) that are covalently linked to the cell wall peptidoglycan via an arabinogalactan network." [GOC:dos, PMID:18316738, PMID:18567661]
GO:0098570
name: stromal side of plastid inner membrane
def: "The side (leaflet) of the plastid inner membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
GO:0098571
name: lumenal side of plastid thylakoid membrane
def: "The side (leaflet) of the plastid thylakoid membrane that faces the lumen, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
GO:0098572
name: stromal side of plastid thylakoid membrane
def: "The side (leaflet) of the plastid thylakoid membrane that faces the stroma, and any proteins embedded in it or loosely bound to its surface." [GOC:dos]
GO:0098573
name: intrinsic component of mitochondrial membrane
def: "The component of the mitochondrial membrane consisting of the gene products that have some covalently attached part (e.g. peptide sequence or GPI anchor) which spans or is embedded in one or both leaflets of the membrane." [GOC:dos]
GO:0098574
name: cytoplasmic side of lysosomal membrane
def: "The side (leaflet) of the lysosomal membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098575
name: lumenal side of lysosomal membrane
def: "The side (leaflet) of the lysosomal membrane that faces the lumen." [GOC:dos]
GO:0098576
name: lumenal side of membrane
def: "Any side (leaflet) of a membrane that faces the lumen of an organelle." [GOC:dos]
GO:0098577
name: inactive sex chromosome
def: "A sex chromosome that has been inactivated." [GOC:dos]
GO:0098578
name: condensed chromatin of inactivated sex chromosome
def: "A condensed form of chromatin that is associated with an inactivated sex chromosome and which is responsible for its inactivation." [GOC:dos]
GO:0098579
name: active sex chromosome
def: "A sex chromosome that has not been inactivated." [GOC:dos]
GO:0098580
name: chromatin of active sex chromosome
def: "Chromatin that is part of an active sex chromosome." [GOC:dos, PMID:23816838]
GO:0098584
name: host cell synaptic vesicle
def: "A secretory organelle of a host cell, some 50 nm in diameter, of presynaptic nerve terminals; accumulates in high concentrations of neurotransmitters and secretes these into the synaptic cleft by fusion with the 'active zone' of the presynaptic plasma membrane." [GOC:dos]
GO:0098585
name: host cell synaptic vesicle membrane
def: "The lipid bilayer surrounding a host synaptic vesicle." [GOC:dos]
GO:0098588
name: bounding membrane of organelle
def: "The lipid bilayer that forms the outer-most layer of an organelle." [GOC:dos]
GO:0098589
name: membrane region
def: "A membrane that is a part of a larger membrane.  Examples include the apical region of the plasma membrane of an epithelial cell and the various regions of the endoplasmic reticulum membrane." [GOC:dos]
GO:0098590
name: plasma membrane region
def: "A membrane that is a (regional) part of the plasma membrane." [GOC:dos]
GO:0098591
name: external side of apical plasma membrane
def: "The leaflet the apical region of the plasma membrane that faces away from the cytoplasm and any proteins embedded or anchored in it or attached to its surface." [GOC:ab, GOC:dos]
GO:0098592
name: cytoplasmic side of apical plasma membrane
def: "The side (leaflet) of the apical region of the plasma membrane that faces the cytoplasm." [GOC:ab, GOC:dos]
GO:0098593
name: goblet cell theca
def: "A cup shaped specialization of the cytoskeleton that forms a thin layer located just below the apical mass of mature mucin secretory granules in the cytoplasm of goblet cells of the intestinal epithelium.  It consists of an orderly network of intermediate filaments and microtubules. Microtubules are arranged vertically, like barrel staves, along the inner aspect of the theta. Intermediate filaments form two networks: an inner, basketlike network and an outer series of circumferential bundles resembling the hoops of a barrel." [PMID:6541604]
GO:0098594
name: mucin granule
def: "A secretory granule that contains mucin." [PMID:16377632]
GO:0098595
name: perivitelline space
def: "The space between the membrane of an oocyte and a surrounding membranous structure (zona pellucida or perivitelline membrane)." [GOC:dos]
GO:0098635
name: protein complex involved in cell-cell adhesion
def: "Any protein complex that is capable of carrying out some part of the process of cell-cell adhesion." [GOC:dos]
GO:0098636
name: protein complex involved in cell adhesion
def: "Any protein complex that is capable of carrying out some part of the process of cell-cell adhesion." [GOC:dos]
GO:0098637
name: protein complex involved in cell-matrix adhesion
def: "Any protein complex that is capable of carrying out some part of the process of cell-matrix adhesion." [GOC:dos]
GO:0098642
name: network-forming collagen trimer
def: "A collagen trimer that forms networks." [PMID:21421911]
GO:0098643
name: banded collagen fibril
def: "A supramolecular assembly of fibrillar collagen complexes in the form of a long fiber (fibril) with transverse striations (bands)." [GOC:dos, PMID:21421911]
GO:0098644
name: complex of collagen trimers
def: "A complex of collagen trimers such as a fibril or collagen network." [GOC:dos]
GO:0098645
name: collagen network
def: "A protein complex that consists of collagen triple helices associated to form a network." [GOC:dos, PMID:21421911]
GO:0098646
name: collagen sheet
def: "A protein complex that consists of collagen triple helices associated to form a sheet-like network." [GOC:dos, PMID:21421911]
GO:0098647
name: collagen beaded filament
def: "A supramolecular assembly of collagen trimers with a 'beads on a string'-like structure." [GOC:dos, PMID:19693541]
GO:0098648
name: collagen anchoring fibril
def: "A specialised collagen fibril that functions as an anchor, binding to other collagen structures." [GOC:dos]
GO:0098651
name: basement membrane collagen trimer
def: "Any collagen timer that is part of a basement membrane." [GOC:dos, PMID:21421911]
GO:0098652
name: collagen type VII anchoring fibril
def: "An antiparallel dimer of two collagen VII trimers, one end of which is embedded in the lamina densa while the other end attaches to banded collagen fibrils in the dermis." [PMID:19693541]
GO:0098654
name: CENP-A recruiting complex
def: "A protein complex that includes Mis16(Yippee family) and/or Mis18 (WD repeat) subunits that is involved in the deposition of centromere specific (CENP-A containing) nucleosomes at the centromere." [PMID:24774534]
GO:0098687
name: chromosomal region
def: "Any subdivision of a chromosome along its length." [GOC:dos]
GO:0098723
name: skeletal muscle myofibril
def: "A myofibril of a skeletal muscle fiber." [GOC:dos]
GO:0098733
name: hemidesmosome associated protein complex
def: "Any protein complex that is part of or has some part in a hemidesmosome." [GOC:dos]
GO:0098745
name: Dcp1-Dcp2 complex
def: "A protein complex consisting of a Dcp1 regulatory subunit and a Dcp2 catalytic subunit that has mRNA cap binding activity and is involved in decapping of nuclear-transcribed mRNA." [GOC:dos, GOC:vw, PMID:22323607]
GO:0098752
name: integral component of the cytoplasmic side of the plasma membrane
def: "The component of the plasma membrane consisting of the gene products that penetrate only the cytoplasmic side of the membrane." [GOC:dos]
GO:0098753
name: anchored component of the cytoplasmic side of the plasma membrane
def: "The component of the plasma membrane consisting of gene products and protein complexes with covalently attached hyrdophobic anchors products that penetrate only the cytoplasmic side of the membrane." [GOC:dos]
GO:1901588
name: dendritic microtubule
def: "Any microtubule in a dendrite, a neuron projection." [GOC:TermGenie, NIF_Subcellular:sao110773650]
GO:1901589
name: axon microtubule bundle
def: "An arrangement of closely apposed microtubules running parallel to each other in the axon hillock and initial segment." [GOC:TermGenie, NIF_Subcellular:sao707332678]
GO:1902377
name: nuclear rDNA heterochromatin
def: "Any rDNA heterochromatin that is part of a nucleus." [GOC:TermGenie, PMID:20661445]
GO:1902493
name: acetyltransferase complex
def: "A protein complex which is capable of acetyltransferase activity." [GOC:bhm, GOC:TermGenie, PMID:8077207]
GO:1902494
name: catalytic complex
def: "A protein complex which is capable of catalytic activity." [GOC:bhm, GOC:TermGenie, PMID:8077207]
GO:1902495
name: transmembrane transporter complex
def: "A transmembrane protein complex which enables the transfer of a substance from one side of a membrane to the other." [GOC:bhm, GOC:TermGenie, PMID:18024586]
GO:1902500
name: vacuolar HOPS complex
def: "Any HOPS complex that is part of a vacuolar membrane." [GOC:TermGenie, PMID:23645161]
GO:1902501
name: lysosomal HOPS complex
def: "Any HOPS complex that is part of a lysosomal membrane." [GOC:TermGenie, PMID:23645161]
GO:1902502
name: multivesicular body HOPS complex
def: "Any HOPS complex that is part of a multivesicular body membrane." [GOC:TermGenie, PMID:23645161]
GO:1902503
name: adenylyltransferase complex
def: "A protein complex which is capable of adenylyltransferase activity." [GOC:bhm, GOC:TermGenie, PMID:11713534]
GO:1902507
name: thiazole synthase complex
def: "A protein complex which is capable of thiazole synthase activity." [GOC:bhm, GOC:TermGenie, PMID:12650933]
GO:1902508
name: 2-iminoacetate synthase complex
def: "A protein complex which is capable of 2-iminoacetate synthase activity." [GOC:bhm, GOC:TermGenie, PMID:12650933]
GO:1902509
name: methionine-importing complex
def: "A protein complex which is capable of methionine-importing activity." [GOC:pr, GOC:TermGenie, PMID:23748165]
GO:1902515
name: thioredoxin-disulfide reductase complex
def: "A protein complex which is capable of thioredoxin-disulfide reductase activity." [GOC:bhm, GOC:TermGenie, PMID:10947986]
GO:1902517
name: glycerol-3-phosphate-transporting ATPase complex
def: "A protein complex which is capable of glycerol-3-phosphate-transporting ATPase activity." [GOC:bhm, GOC:TermGenie, PMID:23013274]
GO:1902554
name: serine/threonine protein kinase complex
def: "A protein complex which is capable of protein serine/threonine kinase activity." [GOC:bhm, GOC:TermGenie, PMID:18191223]
GO:1902555
name: endoribonuclease complex
def: "A protein complex which is capable of endoribonuclease activity." [GOC:bhm, GOC:TermGenie, PMID:18191223]
GO:1902556
name: phosphatidylinositol transporter complex
def: "A protein complex which is capable of phosphatidylinositol transporter activity." [GOC:bhm, GOC:TermGenie, PMID:9890948]
GO:1902560
name: GMP reductase complex
def: "A protein complex which is capable of GMP reductase activity. It catalysis of the irreversible reaction: IMP + NADP(+) + NH(4)(+) <= GMP + 2 H(+) + NADPH." [GOC:bhm, GOC:TermGenie, PMID:12009299]
GO:1902562
name: H4 histone acetyltransferase complex
def: "A protein complex which is capable of H4 histone acetyltransferase activity." [GOC:bhm, GOC:TermGenie, PMID:23775086]
GO:1902636
name: kinociliary basal body
def: "A ciliary basal body that is part of a kinocilium." [GO_REF:0000064, GOC:cilia, GOC:krc, GOC:TermGenie, PMID:15855039, PMID:15882574]
GO:1902671
name: left anterior basal body
def: "Any ciliary basal body that is part of a left anterior flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902672
name: right anterior basal body
def: "Any ciliary basal body that is part of a right anterior flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902673
name: left posteriolateral basal body
def: "Any ciliary basal body that is part of a left posteriolateral flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902674
name: right posteriolateral basal body
def: "Any ciliary basal body that is part of a right posteriolateral flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902675
name: left ventral basal body
def: "Any ciliary basal body that is part of a left ventral flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902676
name: right ventral basal body
def: "Any ciliary basal body that is part of a right ventral flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902677
name: left caudal basal body
def: "Any ciliary basal body that is part of a left caudal flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902678
name: right caudal basal body
def: "Any ciliary basal body that is part of a right caudal flagellum found in Giardia species (trophozoite stage)." [GO_REF:0000064, GOC:giardia, GOC:TermGenie, ISBN:9780124260207, PMID:16607022, PMID:5961344]
GO:1902687
name: glucosidase complex
def: "A protein complex which is capable of glucosidase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:23826932]
GO:1902693
name: superoxide dismutase complex
def: "A protein complex which is capable of superoxide dismutase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:10026301]
GO:1902694
name: superoxide dismutase copper chaperone complex
def: "A protein complex which is capable of superoxide dismutase copper chaperone activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:10426947]
GO:1902695
name: metallochaperone complex
def: "A protein complex which is capable of metallochaperone activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:10426947]
GO:1902710
name: GABA receptor complex
def: "A protein complex which is capable of GABA receptor activity. Upon binding of gamma-aminobutyric acid (GABA) it transmits the signal from one side of the membrane to the other to initiate a change in cell activity. Major inhibitory receptor in vertebrate brain. Also found in other vertebrate tissues, invertebrates and possibly in plants. Effective benzodiazepine receptor." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:18790874]
GO:1902711
name: GABA-A receptor complex
def: "A protein complex which is capable of GABA-A receptor activity. In human, it is usually composed of either two alpha, two beta and one gamma chain of the GABA-A receptor subunits or 5 chains of the GABA-A receptor subunits rho1-3 (formally known as GABA-C receptor)." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:18790874]
GO:1902712
name: G-protein coupled GABA receptor complex
def: "A protein complex which is capable of G-protein coupled GABA receptor activity. In human, it is usually a heterodimer composed of GABA-B receptor subunits 1 and 2." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:18790874]
GO:1902716
name: cell cortex of growing cell tip
def: "Any cell cortex that is part of a growing cell tip." [GO_REF:0000064, GOC:TermGenie, PMID:24146635]
GO:1902737
name: dendritic filopodium
def: "A small, membranous protrusion found primarily on dendritic stretches of developing neurons. May receive synaptic input, and can develop into dendritic spines." [GO_REF:0000064, GOC:pad, GOC:PARL, GOC:TermGenie, http://en.wikipedia.org/wiki/Dendritic_filopodia, PMID:24464040]
GO:1902773
name: GTPase activator complex
def: "A protein complex which is capable of GTPase activator activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:16449187]
GO:1902792
name: pyrroline-5-carboxylate reductase complex
def: "A protein complex which is capable of pyrroline-5-carboxylate reductase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:2722838]
GO:1902793
name: glutamate decarboxylase complex
def: "A protein complex which is capable of glutamate decarboxylase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:17384644]
GO:1902911
name: protein kinase complex
def: "A protein complex which is capable of protein kinase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:24606918]
GO:1902912
name: pyruvate kinase complex
def: "A protein complex which is capable of pyruvate kinase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:24606918]
GO:1902929
name: plasma membrane of growing cell tip
def: "Any plasma membrane part that is part of a growing cell tip." [GO_REF:0000064, GOC:TermGenie, PMID:17085965]
GO:1902937
name: inward rectifier potassium channel complex
def: "A protein complex which is capable of inward rectifier potassium channel activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:16834334]
GO:1903095
name: ribonuclease III complex
def: "A protein complex which is capable of ribonuclease III activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:22393237]
GO:1903103
name: potassium:proton antiporter complex
def: "A protein complex which is capable of potassium:proton antiporter activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:21041667]
GO:1903113
name: copper ion transmembrane transporter complex
def: "A protein complex which is capable of copper ion transmembrane transporter activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:23122209]
GO:1903114
name: silver ion transmembrane transporter complex
def: "A protein complex which is capable of silver ion transmembrane transporter activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:11283292]
GO:1903143
name: adrenomedullin receptor complex
def: "A transmembrane, G-protein-coupled signalling receptor complex which is capable of adrenomedullin receptor activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:22102369]
GO:1903144
name: actomyosin contractile ring actin filament
def: "Any actin filament that is part of a actomyosin contractile ring." [GO_REF:0000064, GOC:TermGenie, PMID:20807799, PMID:24954052]
GO:1903145
name: actin filament of cell cortex of cell tip
def: "Any actin filament that is part of a cell cortex of cell tip." [GO_REF:0000064, GOC:TermGenie, PMID:20807799, PMID:24954052]
GO:1903269
name: ornithine carbamoyltransferase inhibitor complex
def: "A protein complex which is capable of ornithine carbamoyltransferase inhibitor activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:12679340]
GO:1903293
name: phosphatase complex
def: "A protein complex which is capable of phosphatase activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:24766807]
GO:1903349
name: omegasome membrane
def: "Any membrane that is part of an omegasome." [GO_REF:0000064, GOC:mf, GOC:TermGenie, PMID:18725538, PMID:24591649]
GO:1903439
name: calcitonin family receptor complex
def: "A protein complex which is capable of calcitonin family receptor activity. Calcitonin family receptors may form dimers, trimers or tetramers; adrenomedullin and amylin receptors have only been observed as dimers so far." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:10871296, PMID:12037140, PMID:18687416]
GO:1903440
name: amylin receptor complex
def: "A protein complex which is capable of amylin receptor activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:10871296, PMID:12037140, PMID:18687416]
GO:1903502
name: translation repressor complex
def: "A protein complex which is capable of translation repressor activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:14723848]
GO:1903503
name: ATPase inhibitor complex
def: "A protein complex which is capable of ATPase inhibitor activity." [GO_REF:0000088, GOC:bhm, GOC:TermGenie, PMID:16170325]
GO:1903561
name: extracellular vesicle
def: "Any vesicle that is part of the extracellular region." [GO_REF:0000064, GOC:pm, GOC:TermGenie, PMID:24769233]
GO:1903600
name: glutaminase complex
def: "A protein complex which is capable of glutaminase activity." [GO_REF:0000088, GOC:TermGenie, PMID:14764090]
GO:1903754
name: cortical microtubule plus-end
def: "The plus-end of a cortical microtubule." [GO_REF:0000064, GOC:TermGenie, GOC:vw]
GO:1903767
name: sweet taste receptor complex
def: "A protein complex which is capable of sweet taste receptor activity." [GO_REF:0000088, GOC:BHF, GOC:rl, GOC:TermGenie, PMID:16720576]
GO:1903768
name: taste receptor complex
def: "A protein complex which is capable of taste receptor activity." [GO_REF:0000088, GOC:BHF, GOC:rl, GOC:TermGenie, PMID:16720576]
GO:1990005
name: granular vesicle
def: "A cytoplasmic membrane-bounded vesicle of varying size, but usually larger than 45 nm, with an electron dense granular core, found in noradrenergic and peptidergic cells." [NIF_Subcellular:sao478230652]
GO:1990006
name: amorphous vesicle
def: "A cytoplasmic membrane-bounded vesicle first described in dendrites, categorized by smooth membranes, electron-lucent interiors and irregular shapes. Sometimes occurs in clumps. Amorphous vesicles have been found to contain material taken up from the extracellular space, therefore suggesting that they may be part of the endosomal pathway." [NIF_Subcellular:sao1531915298, PMID:11896161]
GO:1990007
name: membrane stack
def: "A configuration of endoplasmic reticulum (ER) found in Purkinje cells in the cerebellum and in axons in the lateral vestibular nucleus, consisting of parallel and interconnecting tubules whose outer surfaces are covered by particles or ringlike structures." [ISBN:01950657191, NIF_Subcellular:sao2114874506]
GO:1990008
name: neurosecretory vesicle
def: "A large cytoplasmic membrane-bounded vesicle with an electron dense granular core, up to 150-200 nm in diameter, found in neurosecretory cells in the hypothalamus." [ISBN:01950657191, NIF_Subcellular:sao2031592629]
GO:1990011
name: laminated body
def: "Inclusion body characterized by regularly spaced sheets of tubules arranged in a whorl pattern resembling a fingerprint. Laminated bodies have been observed in neurons of the lateral geniculate nucleus." [ISBN:01950657191, NIF_Subcellular:sao506721981]
GO:1990012
name: complex laminated body
def: "A cytoplasmic inclusion body found in some lateral geniculate neurons and composed of sheets of tubules (25 nm in diameter) separated by dense material (about 75 nm wide), which together with the tubules whorl give a structure resembling a fingerprint." [NIF_Subcellular:nlx_151681]
GO:1990013
name: presynaptic grid
def: "A hexagonal array of electron dense particles attached to the cytoplasmic face of the presynaptic membrane." [ISBN:0716723808, NIF_Subcellular:sao1730664005]
GO:1990014
name: orthogonal array
def: "Square array of closely spaced intramembrane particles, 4-6 nm in size, that form supramolecular aggregates found in the plasma membrane of astrocytes, skeletal muscle and epithelial cells. They have been shown to contain aquaporins (water channels)." [NIF_Subcellular:sao1747012216, PMID:22718347]
GO:1990015
name: ensheathing process
def: "A cell projection (often from glial cells such as Schwann cells) that surrounds an unmyelinated axon or cell soma." [NIF_Subcellular:sao1376748732]
GO:1990016
name: neck portion of tanycyte
def: "Elongated portion of a tanycyte that sticks into the periventricular layer of neuropil where it appears to contact a blood vessel; characterized by numerous cytoplasmic extensions. A tanycyte is a specialized elongated ventricular ependymal cell that has processes that extend to the outer, or pial, surface of the CNS." [ISBN:01950657191, NIF_Subcellular:sao901230115]
GO:1990017
name: somatic portion of tanycyte
def: "Portion of a tanycyte that lies within the ependyma and contains the nucleus. A tanycyte is a specialized elongated ventricular ependymal cell that has processes that extend to the outer, or pial, surface of the CNS." [ISBN:01950657191, NIF_Subcellular:sao401910342]
GO:1990018
name: tail portion of tanycyte
def: "Elongated process of a tanycyte, devoid of cytoplasmic extensions, that courses through the hypothalamic nuclei to form small endfoot processes that terminate either on blood vessels or at the pial surface of the brain. A tanycyte is a specialized elongated ventricular ependymal cell." [ISBN:01950657191, NIF_Subcellular:sao1749953771]
GO:1990020
name: recurrent axon collateral
def: "Axon collateral that ramifies in the area of the soma of the cell of origin." [NIF_Subcellular:sao1642494436]
GO:1990021
name: Schaffer axon collateral
def: "Part of axon of a CA3 pyramidal neuron that projects to hippocampal area CA1." [NIF_Subcellular:nlx_subcell_20090511]
GO:1990023
name: mitotic spindle midzone
def: "The area in the center of the anaphase spindle consisting of microtubules, microtubule bundling factors and kinesin motors where the spindle microtubules from opposite poles overlap in an antiparallel manner." [GOC:mtg_cell_cycle, GOC:vw]
GO:1990024
name: C bouton
def: "Synaptic bouton found in spinal cord on the soma and proximal dendrites of motor neurons." [NIF_Subcellular:nlx_subcell_100208]
GO:1990025
name: F bouton
def: "Synaptic bouton found in the ventral horn of the spinal cord. F boutons range in diameter from 0.5 to 7 um and contain flattened or pleomorphic synaptic vesicles." [NIF_Subcellular:nlx_subcell_100206]
GO:1990026
name: hippocampal mossy fiber expansion
def: "Synaptic expansion of hippocampal mossy fiber axon that makes contact with the thorny excrescences of hippocampal CA3 pyramidal cell dendrites." [NIF_Subcellular:nlx_subcell_1005002]
GO:1990027
name: S bouton
def: "Synaptic bouton found in the ventral horn of the spinal cord. S boutons range in diameter from 0.5 to 8 um and contain spherical synaptic vesicles." [NIF_Subcellular:nlx_subcell_100207]
GO:1990030
name: pericellular basket
def: "Ramification of basket cell axon surrounding cell bodies, forming the characteristic pericellular baskets from which the cell class derives its name." [NIF_Subcellular:sao413722576]
GO:1990031
name: pinceau fiber
def: "Dense plexus formed by the descending collaterals of cerebellar basket cells that wrap around a Purkinje cell axonal initial segment." [NIF_Subcellular:sao109906988]
GO:1990032
name: parallel fiber
def: "A parallel fiber results from the bifurcation of a cerebellar granule cell axon in the molecular layer into two diametrically opposed branches, that are oriented parallel to the long axis of the folium." [ISBN:0195159551, NIF_Subcellular:nlx_330]
GO:1990033
name: dendritic branch point
def: "The part of a dendrite where the cell projection branches, giving rise to a dendritic branch." [NIF_Subcellular:sao-1348591767]
GO:1990037
name: Lewy body core
def: "The center portion of a Lewy body. In Parkinson's disease, it contains a matted meshwork of filaments." [NIF_Subcellular:sao6587439252]
GO:1990038
name: Lewy body corona
def: "The periphery of a Lewy body. In Parkinson's disease, it contains spherical accumulations of filaments arranged in a loose, radiating array." [NIF_Subcellular:sao5764355747]
GO:1990039
name: hypolemmal cisterna
def: "Specialized part of the smooth endoplasmic reticulum that closely underlies the plasma membrane, usually within 60 nm or closer." [ISBN:0195065719, NIF_Subcellular:sao1634374950]
GO:1990040
name: sub-surface cisterna
def: "Specialization of the hypolemmal cisterna consisting of either single profiles or closely apposed stacks of endoplasmic reticulum in which the lumen is obliterated, lying 10-20 nm beneath the plasma membrane." [ISBN:0195065719, NIF_Subcellular:sao128470897]
GO:1990047
name: spindle matrix
def: "A proteinaceous, nuclear-derived structure that embeds the microtubule spindle apparatus from pole to pole in a microtubule-independent manner during mitosis." [GOC:ans, PMID:19273613, PMID:22855526]
GO:1990060
name: maltose transport complex
def: "Protein complex facilitating ATP-dependent maltose transport through inner cell membrane (periplasm to cytoplasm) in Gram-negative bacteria. In E. coli the system is composed of a periplasmic maltose-binding protein (MBP), two integral membrane proteins, MalF and MalG, and two copies of the cytoplasmic ATP-binding cassette MalK." [PMID:18033289]
GO:1990061
name: bacterial degradosome
def: "The degradosome is a protein complex playing a key role in mRNA degradation and RNA processing. It includes a RNA helicase, a 3'-5' phosphate-dependent PNPase and a RNase E bound-enolase." [GOC:bhm, PMID:21805185]
GO:1990062
name: RPAP3/R2TP/prefoldin-like complex
def: "A protein complex first characterized in human and comprised of a R2TP module (R2TP complex), a prefoldin-like module (containing both prefoldin-like proteins and canonical prefoldins), WD40 repeat protein Monad/WDR92 and DNA-dependent RNA polymerase subunit RPB5. This complex might have chaperone activity." [GOC:pr, PMID:20453924, PMID:21925213, PMID:22418846]
GO:1990063
name: Bam protein complex
def: "Protein complex which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. In E. coli it is composed of BamABCDE, of the outer membrane protein BamA, and four lipoproteins BamB, BamC, BamD and BamE. BamA interacts directly with BamB and the BamCDE subcomplex." [GOC:bhm, PMID:20378773]
GO:1990065
name: Dxr protein complex
def: "A protein complex that is involved in the MEP pathway of IPP biosynthesis. It catalyzes the NADP-dependent rearrangement and reduction of 1-deoxy-D-xylulose-5-phosphate (DXP) to 2-C-methyl-D-erythritol 4-phosphate (MEP)." [GOC:bhm, PMID:15339150]
GO:1990070
name: TRAPPI protein complex
def: "A complex that tethers COPII vesicles at ER-Golgi intermediate compartment. Its role in this part of the vesicular transport may start at the ER exit sites. Binds to a component of the COPII coat. In yeast it includes the following subunits: Bet3 (as homodimer), Bet5, Trs20, Trs23, Trs31, Trs33 which are regarded as the \"core subunits\" of all TRAPP complexes in yeast." [GOC:bhm, PMID:20375281, PMID:22669257]
GO:1990071
name: TRAPPII protein complex
def: "A complex that mediates intra-Golgi traffic, Golgi exit, endosome-to-Golgi traffic, and the trafficking of autophagy proteins from Golgi to the pre-autophagosomal structure. Binds to a component of the COPI coat. In yeast it includes the following subunits: Bet3 (as homodimer), Bet5, Tca17, Trs20, Trs23, Trs31, Trs33, Trs65, Tr120, Tr130. The whole complex is thought to dimerize with itself." [GOC:bhm, PMID:20375281, PMID:22669257]
GO:1990072
name: TRAPPIII protein complex
def: "A complex that functions in anterograde transport at the Golgi and also regulates autophagy. In yeast it includes at least the following subunits: Bet3 (as homodimer), Bet5, Trs20, Trs23, Trs31, Trs33, Trs85. TRAPPIII may include further, as yet undescribed, proteins." [GOC:bhm, PMID:20375281, PMID:22669257]
GO:1990073
name: perforation plate
def: "A cell wall part that is the part of a wall of a vessel member and bears one or more openings (perforations)." [GOC:PO_curators, ISBN:0471245194]
GO:1990075
name: periciliary membrane compartment
def: "A membrane domain at the base of cilia and flagella that is enriched in endocytosis-associated proteins and vesicles and that appears to function in regulating ciliary membrane homeostasis." [GOC:dr, PMID:22342749]
GO:1990077
name: primosome complex
def: "Any of a family of protein complexes that form at the origin of replication and function in replication restart in all organisms. Early complexes initiate double-stranded DNA unwinding. The core unit consists of a replicative helicase and a primase. The helicase further unwinds the DNA and recruits the polymerase machinery. The primase synthesizes RNA primers that act as templates for complementary stand replication by the polymerase machinery. The primosome contains a number of associated proteins and protein complexes and is part of the replication initiation process." [GOC:bhm, PMID:21856207]
GO:1990078
name: replication inhibiting complex
def: "A protein complex that inhibits multiple events of replication initiation during one replication cycle." [GOC:bhm, PMID:21708944]
GO:1990082
name: DnaA-L2 complex
def: "A protein complex that inhibits unwinding of DNA at the origin of replication and assembly of the pre-primosome. In E. coli, this complex is composed of DnaA and of the ribosomal protein L2." [GOC:bhm, PMID:21288885]
GO:1990083
name: DnaA-Hda complex
def: "A protein complex that inactivates the function of DnaA by inhibiting the phosphorylation of DnaA-ADP to DnaA-ATP and thereby preventing multiple events of replication initiation. In E. coli, this complex is composed of DnaA and Hda." [GOC:bhm, PMID:21708944]
GO:1990084
name: DnaA-Dps complex
def: "A protein complex that negatively regulates strand-opening at the origin of replication, thereby interfering with replication initiation. This complex is thought to be involved in the regulation of replication under oxidative stress conditions. In E. coli, this complex is composed of DnaA and Dps." [GOC:bhm, PMID:18284581]
GO:1990085
name: Hda-beta clamp complex
def: "A protein complex involved in inactivating the function of DnaA and thereby preventing multiple events of replication initiation. In E. coli, this complex is composed of the beta clamp (DnaN) and Hda." [GOC:bhm, PMID:15150238]
GO:1990097
name: SeqA-DNA complex
def: "A protein-DNA complex that contains an oligomer of SeqA bound to GATC sites in methylated or newly-synthesized, hemi-methylated double-stranded DNA, with preference for the latter. Binding of SeqA to hemimethylated DNA sequesters oriC, prevents re-methylation of DNA by Dam and in turn stops premature re-initiation of replication during one replication cycle." [GOC:bhm, PMID:12379844, PMID:15933720, PMID:23149570]
GO:1990098
name: core primosome complex
def: "A protein-DNA complex containing at least one DNA helicase and one primase. Can also contain associated proteins. The helicase component continues to unwind the double-stranded DNA (dsDNA) and the primase component synthesizes a RNA primer during initiation or restart of replication." [GOC:bhm, PMID:21856207]
GO:1990099
name: pre-primosome complex
def: "Any of the protein-DNA complexes that contain a DNA helicase and associated protein(s) at the origin of replication, and build up to assembling the core primosome. The associated protein(s) chaperone the helicase to the DNA, and assembly of the pre-primosome is essential for the initiation or restart of replication. Pre-primosome complexes lack a primase component." [GOC:bhm, PMID:18179598, PMID:20129058, PMID:8663105]
GO:1990100
name: DnaB-DnaC complex
def: "A protein complex containing homohexameric DNA helicase DnaB, and the DNA helicase loader DnaC. The helicase loader DnaC delivers DnaB to the chromosomal origin (oriC)." [GOC:bhm, PMID:20129058]
GO:1990101
name: DnaA-oriC complex
def: "A protein-DNA complex containing the initiator protein DnaA bound to high-affinity recognition sites in the unique origin of replication, oriC. DnaA-oriC binding is the first step in assembly of a bacterial pre-replicative complex (pre-RC) and is responsible for the timely initiation of replication once per cell cycle." [GOC:bhm, PMID:19833870]
GO:1990102
name: DnaA-DiaA complex
def: "A protein-DNA complex containing a tetramer of DiaA attached to multiple  DnaA molecule bound to oriC DNA. Regulates timely initiation of chromosomal replication during the cell cycle by stimulating assembly of DnaA-oriC complexes, conformational changes in ATP-DnaA initiation complexes, and unwinding of oriC duplex DNA." [GOC:bhm, PMID:15326179, PMID:17699754]
GO:1990103
name: DnaA-HU complex
def: "A protein-DNA complex containing DNA-bound DnaA attached to HU. HU is a dimer encoded by two closely related genes. Essential for the initiation of replication in bacteria; stimulates the DnaA-dependent unwinding of oriC." [GOC:bhm, PMID:18179598]
GO:1990104
name: DNA bending complex
def: "A protein-DNA complex that contains DNA in combination with a protein which binds to and bends DNA. Often plays a role in DNA compaction." [GOC:bhm, PMID:17097674]
GO:1990111
name: spermatoproteasome complex
def: "A proteasome specifically found in mammalian testis. Contains the proteasome activator PA200 in the regulatory particle, and beta1i, beta2i, beta5i and/or alpha4s in the core (20S) subunit. Beta1i, beta2i and beta5i are inducible catalytic subunits, closely related to beta1, beta2 and beta5. Alpha4s is a sperm-specific 20S subunit, but unlike other alternative 20S subunits alpha4s lies in the outer alpha-ring and lacks catalytic activity." [GOC:sp, PMID:23706739]
GO:1990112
name: RQC complex
def: "A multiprotein complex that forms a stable complex with 60S ribosomal subunits containing stalled polypeptides and triggers their degradation (ribosomal quality control). In budding yeast, this complex includes Cdc48p, Rkr1p, Tae2p, Rqc1p, Npl4p and Ufd1p proteins." [GOC:rb, PMID:23178123, PMID:23232563]
GO:1990121
name: H-NS complex
def: "A multimer of H-NS proteins that is involved in bacterial nucleoid condensation and negative regulation of global gene expression by directly binding to promoter regions. Recognizes both structural and sequence-specific motifs in double-stranded DNA and has binding preference for bent DNA." [GOC:bhm, PMID:12592399]
GO:1990124
name: messenger ribonucleoprotein complex
def: "A ribonucleoprotein complex containing both protein and messenger RNA (mRNA) molecules." [GOC:bf, PMID:15574591, PMID:21915786]
GO:1990125
name: DiaA complex
def: "A homotetrameric protein complex consisting of a symmetrical pair of DiaA homodimers. Facilitates DnaA binding to the origin of replication during replication initiation." [GOC:bhm, PMID:17699754]
GO:1990128
name: pre-primosome complex involved in replication initiation
def: "OBSOLETE. A protein-DNA complex involved in replication initiation at the origin of replication." [GOC:bhm, PMID:21856207]
GO:1990129
name: pre-primosome complex involved in replication restart
def: "OBSOLETE. A protein-DNA complex involved in replication restart after a stalled replication fork has been repaired." [GOC:bhm, PMID:17139333]
GO:1990130
name: Iml1 complex
def: "A protein complex involved in regulation of non-nitrogen-starvation (NNS) autophagic process. In S. cerevisiae this complex contains Iml1p, Npr2p and Npr3p proteins." [GOC:rb, PMID:21900499, PMID:23974112]
GO:1990131
name: Gtr1-Gtr2 GTPase complex
def: "A heterodimer GTPase complex. In S. cerevisiae, this complex contains Gtr1p and Gtr2p proteins." [GOC:rb, PMID:10388807, PMID:16143306]
GO:1990133
name: molybdopterin cofactor (Moco) biosynthesis adenylyltransferase complex
def: "A heterodimeric protein complex which adenylates two molecules of the sulfur carrier subunit of the molybdopterin (MPT) cofactor synthase using ATP as part of molybdopterin cofactor (Moco) biosynthesis. In E. coli the subunits are MoeB and MoaD; Moco biosynthesis and its constituent molecules are evolutionarily conserved." [GOC:bhm, pmid:11713534, pmid:16669776]
GO:1990140
name: MPT synthase complex
def: "A heterodimeric protein complex which catalyses sulfur transfer from the sulfur carrier subunit of MPT synthase to precursor Z to synthesize MPT as part of molybdopterin cofactor (Moco) biosynthesis. In E. coli the subunits are MoaE and MoaD; in human, MOCS2B and MOCS2A. Moco biosynthesis and its constituent molecules are evolutionarily conserved." [GOC:bhm, PMID:11135669, PMID:16669776, Reactome:REACT_25073.1]
GO:1990143
name: CoA-synthesizing protein complex
def: "A multisubunit complex likely involved in the synthesis of coenzyme A (CoA). In S. cerevisiae, the complex is formed by Cab2, Cab3, Cab4, Cab5, Sis2 and Vhs3." [GOC:rb, PMID:23789928]
GO:1990148
name: glutamate dehydrogenase complex
def: "A homomeric protein complex that possesses glutamate dehydrogenase activity. This complex is evolutionarily conserved except that the number of homoprotomers per complex varies." [GOC:bhm, PMID:22393408, PMID:23412807]
GO:1990150
name: VEGF-A complex
def: "A homodimeric, extracellular protein complex containing two VEGF-A monomers. Binds to and activates a receptor tyrosine kinase." [GOC:bf, GOC:bhm, PMID:12207021, PMID:19658168]
GO:1990154
name: enzyme IIA-maltose transporter complex
def: "A protein complex consisting of the pentameric maltose transporter complex bound to two enzyme IIA (EIIA) molecules. EIIA is a component of the glucose-specific phosphotransferase system that inhibits maltose transport from the periplasm to the cytoplasm. When EIIA-bound, the maltose transporter remains in the open, inward-facing conformation, which prevents binding of maltose-loaded maltose binding protein (MBP) to the transporter." [GOC:bf, GOC:bhm, PMID:23770568]
GO:1990156
name: DnaB-DnaG complex
def: "A protein complex containing homohexameric DnaB helicase, and DnaG (a primase). Facilitates the unwinding of double-stranded DNA and the synthesis of RNA primer sequences during DNA replication and repair in Prokaryotes." [GOC:bhm, PMID:14557266]
GO:1990157
name: DnaA-DnaB-DnaC complex
def: "A protein-DNA complex consisting of the helicase loading complex DnaB-DnaC bound to the DNA-bound DNA replication initiation protein DnaA. Essential for DNA replication initiation." [GOC:bhm, PMID:20129058]
GO:1990158
name: DnaB-DnaC-DnaT-PriA-PriB complex
def: "A protein-DNA complex consisting of the helicase loading complex DnaB-DnaC, replication restart proteins DnaT, PriA and PriB, and associated DNA. Involved in the restart of DNA replication after a stalled replication fork has been repaired." [GOC:bhm, PMID:8663105]
GO:1990159
name: DnaB-DnaC-DnaT-PriA-PriC complex
def: "A protein-DNA complex consisting of the helicase loading complex DnaB-DnaC, replication restart proteins DnaT, PriA and PriC, and associated DNA. Involved in the restart of DNA replication after a stalled replication fork has been repaired." [GOC:bhm, PMID:8663105]
GO:1990160
name: DnaB-DnaC-Rep-PriC complex
def: "A protein-DNA complex consisting of the helicase loading complex DnaB-DnaC, replication restart proteins Rep and PriC, and associated DNA. Involved in the restart of DNA replication after a stalled replication fork has been repaired." [GOC:bhm, PMID:19941825, PMID:8663105]
GO:1990161
name: DnaB helicase complex
def: "A homohexameric protein complex that possesses DNA helicase activity; functions during DNA replication and repair." [GOC:bhm, PMID:17947583]
GO:1990176
name: MalFGK2 complex
def: "Protein complex involved in maltose transport through the plasma membrane. In E. coli, the complex is a tetramer and consists of a cytoplasmic ATPase MalK homodimer together with a heterodimeric transmembrane subunit MalF-MalG." [GOC:bhm, PMID:19250913]
GO:1990177
name: IHF-DNA complex
def: "A protein-DNA complex containing IHF heterodimers (an alpha and a beta chain) bound to DNA. IHF binds to double-stranded DNA in a structure- and sequence-specific manner and bends the DNA into a nucleosome-like structure, the bacterial nucleoid." [GOC:bhm, PMID:17097674]
GO:1990178
name: HU-DNA complex
def: "A protein-DNA complex that consists of HU heterodimers (an alpha and a beta chain) assembled into octamers along DNA. HU binds to double-stranded DNA in a structure- and sequence-specific manner and bends the DNA into a nucleosome-like structure." [GOC:bhm, PMID:17360520]
GO:1990184
name: amino acid transport complex
def: "A heteromeric protein complex consisting of a multi-transmembrane spanning subunit (the light chain) and a type II glycoprotein subunit (the heavy chain) that functions to transport amino acids across a plasma membrane." [GOC:kmv, PMID:14668347]
GO:1990191
name: cobalamin transport complex
def: "Protein complex facilitating ATP-dependent cobalamin (vitamin B12) transport through inner cell membrane (periplasm to cytoplasm) in Gram-negative bacteria. In E. coli the system is composed of a periplasmic cobalamin-binding protein (BtuF), an integral membrane homodimer, BtuC, and a cytoplasmic ATP-binding homodimer BtuD." [GOC:bhm, PMID:22569249]
GO:1990193
name: BtuCD complex
def: "Protein complex involved in cobalamin (vitamin B12) transport through the plasma membrane. In E. coli, the complex is a tetramer and consists of the cytoplasmic ATPase BtuD homodimer together with the transmembrane BtuC homodimer." [GOC:bhm, PMID:22569249]
GO:1990195
name: macrolide transmembrane transporter complex
def: "A bacterial transmembrane transporter complex that spans the entire cell membrane system and possesses ATP-dependent xenobiotic transport activity pumping drugs (typically antibiotics) and other toxins directly from the cytosol out of the bacterial cell. Typically, it is trimeric consisting of a inner membrane ATPase (IMP), a periplasmic membrane fusion protein (MFP) and an outer membrane factor (OMF). In E. coli, macrolide transporter complexes may consists of MacB (IMP), MacA (MFP) and TolC (OMF) or AcrB (IMP), AcrA (MFP) and TolC (OMF). Trimeric TolC is a common OMF found in many macrolide transporter complexes." [GOC:bhm, PMID:10879525, PMID:18955484, PMID:19254725]
GO:1990196
name: MacAB-TolC complex
def: "The MacAB-TolC complex is a macrolide transporter complex found in E.coli and related gram-negative bacteria. Its transport activity is specific to macrolide compounds containing 14- and 15-membered lactones. It consists of the dimeric inner membrane ATPase MacB, the hexameric, periplasmic membrane fusion protein MacA and the trimeric outer membrane factor TolC." [GOC:bhm, PMID:10879525, PMID:18955484, PMID:19254725]
GO:1990197
name: ATP-dependent methionine-importing complex
def: "An ATP-binding cassette (ABC) transporter complex that is capable of methionine-importing activity. An example is the bacterial MetNIQ methionine transporter, that consists of the dimeric ATPase subunit MetN located at the cytoplasmic side of the plasma membrane and the dimeric transmembrane subunit MetI. MetQ is regarded as the periplasmic methionine-binding chaperon subunit, and is capable of transporting methionine from the periplasm into the cytoplasm in an ATP-dependent manner." [GOC:bhm, PMID:22095702]
GO:1990198
name: ModE complex
def: "A dimeric protein complex containing two ModE subunits. Binds directly to DNA to regulate transcription, and is involved in (positively and negatively) regulating various aspects of molybdenum metabolism." [GOC:bhm, PMID:12581638]
GO:1990199
name: MsbA transporter complex
def: "An ATP-binding cassette (ABC) transporter complex made up of a dimer of MsbA. Facilitates the export across the plasma membrane of, amongst others, lipid A and lipopolysaccharide. In contrast to most ABC transporter complexes, each chain of the homodimer contains both the transmembrane domain (TMD) and the cytoplasmic ATP-binding domain (NBD)." [GOC:bhm, PMID:18024585]
GO:1990200
name: SsuD-SsuE complex
def: "A protein complex containing an alkanesulfonate monooxygenase subunit (SsuD tetramer in E.coli) and a flavin oxidoreductase subunit (SsuE dimer in E.coli). Involved in the utilization of alkanesulfonates as sulfur sources under conditions of sulfate or cysteine starvation." [GOC:bhm, PMID:16997955]
GO:1990201
name: alkanesulfonate monooxygenase complex
def: "A protein complex capable of alkanesulfonate monooxygenase activity. Involved in the utilization of alkanesulfonates as sulfur sources under conditions of sulfate or cysteine starvation, catalyzing the conversion of alkanesulfonates into aldehydes and sulfite. In E.coli the complex consists of a SsuD tetramer." [GOC:bhm, PMID:10480865, PMID:16997955]
GO:1990202
name: FMN reductase complex
def: "A protein complex capable of FMN reductase activity. Reduces FMN to FMNH2 in a NAD(P)H-dependent manner. In E.coli, consists of a SsuE dimer." [GOC:bhm, PMID:10480865, PMID:16997955]
GO:1990203
name: MdtBC Complex
def: "A protein complex containing two transmembrane subunits; a MdtB dimer and one unit of MdtC. Capable of exporting substrates across the cell membrane. Involved in conferring antibiotic resistance of Gram-negative bacteria by transporting drugs across the membrane." [GOC:bhm, PMID:20038594]
GO:1990204
name: oxidoreductase complex
def: "Any protein complex that possesses oxidoreductase activity." [GOC:bhm, PMID:18982432]
GO:1990205
name: taurine dioxygenase complex
def: "A protein complex capable of catalyzing the conversion of taurine and alpha-ketoglutarate to sulfite, aminoacetaldehyde and succinate under sulfur or cysteine starvation conditions. Its expression is repressed by the presence of sulfate or cysteine. In E. coli it is a homodimer or homotetramer of the protein TauD." [GOC:bhm, PMID:12741810]
GO:1990207
name: EmrE multidrug transporter complex
def: "A transmembrane protein complex capable of transporting positively charged hydrophobic drugs across the plasma membrane thereby involved in conferring resistance to a wide range of toxic compounds (e.g. methyl viologen, ethidium bromide and acriflavine). It is commonly found in bacteria. In E. coli it forms a homodimer." [GOC:bhm, PMID:18024586]
GO:1990220
name: GroEL-GroES complex
def: "Bacterial chaperonin complex consisting of a heptameric 10kDa chaperonin subunit GroES and a tetradecameric (2x7) 60kDa chaperonin subunit GroEL. The 60kDa subunit possesses ATPase activity while the holo-enzyme is responsible for the correct folding of proteins." [GOC:bhm, PMID:15313620]
GO:1990221
name: L-cysteine desulfurase complex
def: "A protein complex capable of cysteine desulfurase activity decomposing L-cysteine to L-alanine and sulfur. It belongs to a ubiquitous family of pyridoxal 5-phosphate (PLP)-dependent enzymes. In E. coli it consists of a SufS dimer." [GOC:bhm, PMID:11827487]
GO:1990222
name: ProVWX complex
def: "The ProVWX complex belongs to the family of ATP-binding cassette (ABC) transporter proteins complexes. It consists of a cytoplasmic ATPase subunit ProV, a transmembrane subunit ProW and a periplasmic binding protein ProX. It is capable of translocating a wide variety of solute (e.g. glycine betaine) across the plasma membrane and is activated under osmotic stress conditions." [GOC:bhm, PMID:23249124]
GO:1990225
name: rhoptry neck
def: "Narrow, electron-dense part of the rhoptry that extends through the conoid at the apical tip of an apicomplexan parasite. The rhoptry neck serves as a duct through which the contents of the rhoptry are secreted after attachment to the host has been completed and at the commencement of invasion." [GOC:giardia, GOC:pr, PMID:23499754, PMID:23937520, PMID:24002067, PMID:24070999]
GO:1990228
name: sulfurtransferase complex
def: "A protein complex capable of catalyzing the transfer of sulfur atoms from one compound (donor) to another (acceptor)." [GOC:bhm, PMID:17350958]
GO:1990229
name: iron-sulfur cluster assembly complex
def: "A protein complex capable of assembling an iron-sulfur (Fe-S) cluster." [GOC:bhm, PMID:17350958]
GO:1990230
name: iron-sulfur cluster transfer complex
def: "A protein complex capable of catalyzing the transfer of an iron-sulfur (Fe-S) cluster from one compound (donor) to another (acceptor)." [GOC:bhm, PMID:19810706]
GO:1990231
name: STING complex
def: "A protein dimer containing two STING monomers. It binds cyclic purine di-nucleotides.  Activation of the sting complex by 2',5'-3'-5'-cyclic GMP-AMP activates nuclear transcription factor kB (NF-kB) and interferon regulatory factor 3 (IRF3) which then induce transcription of the genes encoding type I IFN and cytokines active in the  innate immune response." [GOC:bhm, PMID:22705373, PMID:23706668, PMID:23910378]
GO:1990232
name: phosphomannomutase complex
def: "A protein complex capable of phosphomannomutase activity." [GOC:bhm, PMID:16540464]
GO:1990233
name: intramolecular phosphotransferase complex
def: "A protein complex capable of catalyzing the transfer of a phosphate group from one position to another within a single molecule." [GOC:bhm, PMID:16540464]
GO:1990234
name: transferase complex
def: "A protein complex capable of catalyzing the transfer of a group, e.g. a methyl group, glycosyl group, acyl group, phosphorus-containing, or other groups, from one compound (generally regarded as the donor) to another compound (generally regarded as the acceptor)." [GOC:bhm, PMID:16540464]
GO:1990235
name: diamine N-acetyltransferase complex
def: "A protein complex which is capable of diamine N-acetyltransferase activity." [GOC:bhm, PMID:8077207]
GO:1990241
name: nucleotide binding complex
def: "OBSOLETE. A protein complex that interacts selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose. An example of this is STING in E. coli (Q86WV6)." [GOC:bhm, PMID:23910378]
GO:1990242
name: innate immune response complex
def: "OBSOLETE. A protein complex involved in the innate immune response." [GOC:bhm, PMID:23910378]
GO:1990243
name: atf1-pcr1 complex
def: "A heterodimeric transcription factor complex composed of the bZIP proteins atf1 and pcr1. The heterodimer binds m26 sites (homologous to CRE)." [PMID:24224056]
GO:1990246
name: uniplex complex
def: "A calcium channel complex in the mitochondrial inner membrane capable of highly-selective calcium channel activity. Its components include the EF-hand-containing proteins mitochondrial calcium uptake 1 (MICU1) and MICU2, the pore-forming subunit mitochondrial calcium uniporter (MCU) and its paralog MCUb, and the MCU regulator EMRE." [PMID:24231807]
GO:1990249
name: nucleotide-excision repair, DNA damage recognition complex
def: "A protein complex that is capable of identifying lesions in DNA, such as pyrimidine-dimers, intrastrand cross-links, and bulky adducts. The wide range of substrate specificity suggests that the repair complex recognizes distortions in the DNA helix. It subsequently recruits a nucleotide-excision repair, preincision complex." [GOC:bhm, PMID:22331906]
GO:1990250
name: transcription-coupled nucleotide-excision repair, DNA damage recognition complex
def: "A protein complex that is capable of identifying lesions in DNA on the actively transcribed strand of the DNA duplex as well as a small subset of lesions not recognized by the general nucleotide-excision repair pathway. The wide range of substrate specificity suggests that the repair complex recognizes distortions in the DNA helix. It subsequently recruits a nucleotide-excision repair, preincision complex." [GOC:bhm, PMID:22331906]
GO:1990251
name: Mmi1 nuclear focus complex
def: "A protein complex that forms during vegetative growth and is involved in the selective degradation of meiosis-specific transcripts. Contains at least Mmi1, or an ortholog of it." [GOC:al, GOC:vw, PMID:16823445, PMID:23980030]
GO:1990252
name: Syp1 complex
def: "A protein complex that contributes to the endocytic process and bud growth in yeast. It is involved in the precise timing of actin assembly during endocytosis." [GOC:bhm, PMID:19713939]
GO:1990257
name: piccolo-bassoon transport vesicle
def: "A cytoplasmic dense-core vesicle, or aggregate of vesicles and proteins, that range in size from approximately 80 nm in diameter for dense core vesicles to 130 nm by 220 nm in area for aggregates. Proteins transported by piccolo-bassoon transport vesicle (PTVs) include piccolo, bassoon, N-cadherin and syntaxin. PTVs are packaged via the trans-Golgi network before being transported through the axon." [GOC:dr, PMID:21569270]
GO:1990265
name: platelet-derived growth factor complex
def: "A protein complex consisting of two chains of platelet-derived growth factor (PDGF) subunits. PDGF dimers bind to PDGF receptors in the plasma membrane and induce receptor dimerisation and activation. PDGFs are involved in a wide variety of signalling processes. PDGFs are found in all vertebrates where at least 2 different chains (A and B) exist. In human (and other mammals), four types of PDGF chains (A, B, C, and D) are known which form five different dimers (AA, AB, BB, CC and DD)." [GOC:bhm, IntAct:EBI-2881436, IntAct:EBI-2881443, IntAct:EBI-2881451, PMID:11331882]
GO:1990270
name: platelet-derived growth factor receptor-ligand complex
def: "A tetrameric protein complex consisting of two platelet-derived growth factor (PDGF) receptor subunits and two PDGF ligand subunits. Binding of the PDGF ligand dimer to the PDGF receptor in the plasma membrane induces receptor dimerisation and activation. PDGFs are involved in a wide variety of signalling processes and are found in all vertebrates. At least two different receptor chains (A and B) and four types of ligand chains (A, B, C, and D) are known forming a wide variety of combinations of receptor-ligand complexes." [GOC:bhm, IntACT:EBI-9080360, PMID:11331882]
GO:1990281
name: efflux pump complex
def: "A protein complex that is capable of efflux transmembrane transporter activity." [GOC:dos, PMID:21556065, PMID:9417051]
GO:1990295
name: post-anaphase microtubule array
def: "A cytoskeletal part that consists of an array of microtubules and associated molecules that forms at the end of anaphase, and in which microtubules are nucleated from an equatorial microtubule organizing center." [PMID:11792817, PMID:17072892, PMID:9601091]
GO:1990296
name: scaffoldin complex
def: "A protein complex that contains one or more scaffoldin proteins and several cellulosomal enzymes." [PMID:15197390, PMID:20373916]
GO:1990298
name: bub1-bub3 complex
def: "Protein complex that associates with the kinetochores." [PMID:22521786]
GO:1990302
name: Bre1-Rad6 ubiquitin ligase complex
def: "A ubiquitin ligase complex consisting of Bre1 and Rad6 that mediates monoubiquitination of histone H2B to form H2BK123ub1. H2BK123ub1 gives a specific tag for epigenetic transcriptional activation, elongation by RNA polymerase II, telomeric silencing, and is also a prerequisite for H3K4me and H3K79me formation. It thereby plays a central role in histone code and gene regulation. It also modulates the formation of double-strand breaks during meiosis." [GOC:bhm, IntAct:EBI-9102228, PMID:19531475]
GO:1990303
name: UBR1-RAD6 ubiquitin ligase complex
def: "A ubiquitin ligase complex consisting of UBR1 and RAD6 components. It polyubiquitinates proteins containing non-acetylated N-terminal residues causing their subsequent degradation by the proteasome as part of the Ac/N-End Rule pathway. It recognizes non-acetylated N-terminal methionine if it is followed by a hydrophobic residue. Additionally, it acts in an N-end rule independent manner as a component of a novel quality control pathway for proteins synthesized on cytosolic ribosomes." [GOC:bhm, IntAct:EBI-9103673, PMID:19531475]
GO:1990304
name: MUB1-RAD6-UBR2 ubiquitin ligase complex
def: "A ubiquitin ligase complex consisting of MUB1, RAD6 and UBR2 components. It ubiquitinates, and targets for destruction, the RPN4 transcription factor, which upregulates the proteasome genes. The binding of MUB1 may position the RPN4 ubiquitylation site proximal to the Ubiquitin-RAD6 thioester and allow the transfer of Ubiquitin from RAD6 to RPN4. One of its components, MUB1, is a short-lived protein ubiquitinated by the UBR2-RAD6 ubiquitin conjugating enzyme." [GOC:bhm, IntAct:EBI-9117531, PMID:18070918]
GO:1990305
name: RAD6-UBR2 ubiquitin ligase complex
def: "A ubiquitin ligase complex consisting of RAD6 and UBR2 components. It may act in a quality control pathway for proteins synthesized on cytosolic ribosomes. The UBR2 component lacks sequence motifs required for N-end rule degradation." [GOC:bhm, IntAct:EBI-9116509, PMID:15504724]
GO:1990306
name: RSP5-BUL ubiquitin ligase complex
def: "A ubiquitin ligase complex consisting of RSP5 and BUL components. It polyubiquinates plasma membrane transporters and permeases, required for their endocytosis and subsequent degradation in the vacuole. BUL1 or BUL2, respectively, bind to the target protein, enabling ubiquitylation by Rsp5. Phosphorylation of BUL proteins results in binding to 14-3-3 proteins, protecting the permeases from down-regulation." [GOC:bhm, IntAct:EBI-9105217, IntAct:EBI-9105295, PMID:9931424]
GO:1990315
name: Mcs4 RR-MAPKKK complex
def: "A protein complex that consists of a phospho relay component and a MAPK cascade component.  The complex is involved in signaling oxidative stress and osmostress." [PMID:24255738]
GO:1990316
name: ATG1/UKL1 kinase complex
def: "A protein complex consisting of two subunits: ATG1/ULK1 and ATG13. This complex has serine/threonine protein kinase activity and is involved in autophagosome formation." [GOC:bhm, GOC:DOS, IntAct:EBI-2352849, PMID:22885598]
GO:1990317
name: Gin4 complex
def: "A protein complex involved in septin ring formation during mitosis. In Saccharomyces cerevisiae it consists of BNI5, CDC3, CDC10, CDC11, CDC12, GIN4, NAP1 and SHS1. At least 2 GIN4 molecules are involved." [GOC:bhm, IntAct:EBI-2431392, PMID:12058072]
GO:1990318
name: collagen type XIX trimer
def: "A collagen homotrimer of alpha1(XIX) chains; type XIX collagen triple helices localize to basement membrane zones in differentiating muscle cells." [GOC:bhm, IntAct:EBI-2529092, PMID:17876790]
GO:1990319
name: collagen type XX trimer
def: "A collagen homotrimer of alpha1(XX) chains." [GOC:bhm, IntAct:EBI-2529132, PMID:17876790]
GO:1990320
name: collagen type XXI trimer
def: "A collagen homotrimer of alpha1(XXI) chains; type XXI collagen triple helices found in the extracellular matrix component of blood vessel walls and in the cytoplasm of cultured human aortic smooth muscle." [GOC:bhm, IntAct:EBI-2529143, PMID:17876790]
GO:1990321
name: collagen type XXII trimer
def: "A collagen homotrimer of alpha1(XXII) chains; type XXII collagen triple helices acts as a cell adhesion ligand for skin epithelial cells and fibroblasts." [GOC:bhm, IntAct:EBI-2529189, PMID:17876790]
GO:1990322
name: collagen type XXIII trimer
def: "A collagen homotrimer of alpha1(XXIII) chains; type XXIII collagen triple helices span the plasma membrane." [GOC:bhm, IntAct:EBI-2529234, PMID:17876790]
GO:1990323
name: collagen type XXIV trimer
def: "A collagen homotrimer of alpha1(XXIV) chains; type XXIV collagen triple helices may participate in regulating type I collagen fibrillogenesis at specific anatomical locations during fetal development." [GOC:bhm, IntAct:EBI-2529258, PMID:17876790]
GO:1990324
name: collagen type XXVI trimer
def: "A collagen homotrimer of alpha1(XXVI) chains." [GOC:bhm, IntAct:EBI-2529362, PMID:17876790]
GO:1990325
name: collagen type XXVII trimer
def: "A collagen homotrimer of alpha1(XXVII) chains. These trimers form thin, non-striated fibrils. Type XXVII collagen triple helices play a role during the calcification of cartilage and the transition of cartilage to bone." [GOC:bhm, IntAct:EBI-2529377, PMID:17876790, PMID:21421911]
GO:1990326
name: collagen type XXVIII trimer
def: "A collagen homotrimer of alpha1(XXVIII) chains." [GOC:bhm, IntAct:EBI-2529426, PMID:17876790]
GO:1990327
name: collagen type XXV trimer
def: "A collagen homotrimer of alpha1(XXV) chains; type XXV collagen triple helices span the plasma membrane." [GOC:bhm, IntAct:EBI-2529312, PMID:17876790]
GO:1990328
name: RNA polymerase II, RPB4-RPB7 subcomplex
def: "A protein complexes that mediates transcription and the two major cytoplasmic mRNA decay pathways and is required for efficient translation initiation in association with RNA polymerase II (Pol II)." [GOC:bhm, IntAct:EBI-2945755, PMID:15591044]
GO:1990329
name: IscS-TusA complex
def: "A heterotetrameric protein complex involved in the sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm5s2U) at tRNA wobble positions. In E. coli it consists of a central IscS dimer with the two TusA protomers bound to one of the IscS units each via persulfide (-SSH) groups." [GOC:bhm, IntAct:EBI-8869931, PMID:20404999]
GO:1990330
name: IscS-IscU complex
def: "A heterotetrameric protein complex involved in the sulfur transfer during iron-sulfur cluster assembly and in the modification of tRNA wobble positions. In E. coli it consisting of a central IscS dimer with the IscU protomers attached to one of the IscS units each via a disulfide (-SSH) group." [GOC:bhm, IntAct:EBI-8870238, PMID:20404999]
GO:1990331
name: Hpa2 acetyltransferase complex
def: "A tetrameric protein complex capable of acetyltransferase activity. It can catalyze the transfer of an acetyl group from acetyl-CoA to an acceptor residue on histone H-3, histone H-4, or on polyamines. The complex is also capable of acetylating certain small basic proteins. The two Hpa2 dimers that make up the tetramer are held together by interactions between the bound acetyl-CoA molecules." [GOC:bhm, IntAct:EBI-8871758, PMID:10600387]
GO:1990332
name: Ire1 complex
def: "A type-I transmembrane protein complex located in the endoplasmic reticulum (ER) consisting of an IRE1-IRE1 dimer, which forms in response to the accumulation of unfolded protein in the ER. The dimeric complex has endoribonuclease (RNase) activity and evokes the unfolded protein response (UPR) by cleaving an intron of a mRNA coding for the transcription factor HAC1 in yeast or XBP1 in mammals; the complex cleaves a single phosphodiester bond in each of two RNA hairpins (with non-specific base paired stems and loops of consensus sequence CNCNNGN, where N is any base) to remove an intervening intron from the target transcript." [GOC:bf, GOC:bhm, IntAct:EBI-8872439, PMID:18191223, PMID:25437541]
GO:1990333
name: mitotic checkpoint complex, CDC20-MAD2 subcomplex
def: "A protein complex involved in the spindle checkpoint, preventing the activation of the anaphase-promoting complex until all chromosomes are correctly attached in a bipolar fashion to the mitotic spindle. In budding yeast this complex consists of Mad2p and Cdc20p, and in mammalian cells it consists of MAD2 and CDC20." [GOC:bhm, IntAct:EBI-1270646, PMID:15879521]
GO:1990334
name: Bfa1-Bub2 complex
def: "A protein complex that acts as a two-component GTPase-activating protein for Tem1 GTPase, thus regulating a signal transduction cascade, called the mitotic exit network (MEN), which is required for mitotic exit and cytokinesis. Bub2/Bfa1 keeps Tem1 inactive until the spindle is properly oriented, thus inhibiting MEN activation." [GOC:bhm, IntAct:EBI-1271344, PMID:16449187]
GO:1990338
name: laminin-13 complex
def: "A laminin complex composed of alpha4, beta2 and gamma3 polypeptide chains." [GOC:bhm, IntAct:EBI-2530020, PMID:17453709]
GO:1990339
name: laminin-14 complex
def: "A laminin complex composed of alpha5, beta2 and gamma2 polypeptide chains." [GOC:bhm, IntAct:EBI-2530034, PMID:17453709]
GO:1990340
name: laminin-15 complex
def: "A laminin complex composed of alpha5, beta2 and gamma3 polypeptide chains." [GOC:bhm, IntAct:EBI-2530049, PMID:17453709]
GO:1990341
name: thrombospondin complex
def: "A homotrimeric or homopentameric glycoprotein that functions at the interface of the cell membrane and the extracellular matrix through its interactions with proteins and proteoglycans, such as collagens, integrins and fibronectin, to regulate matrix structure and cellular behaviour." [GOC:bhm, IntAct:EBI-2530370, IntAct:EBI-2530917, IntAct:EBI-2530961, IntAct:EBI-2530975, IntAct:EBI-2531058, IntAct:EBI-9251379, IntAct:EBI-9251408, IntAct:EBI-9251455, IntAct:EBI-9251502, IntAct:EBI-9251524, PMID:18193164]
GO:1990342
name: heterochromatin island
def: "A region of heterochromatin that is formed dynamically in response to environmental signals by a process that does not require RNAi, and is enriched in histone H3 methylated on lysine 9 (H3K9me)." [PMID:22144463, PMID:24210919]
GO:1990343
name: heterochromatin domain
def: "A region of heterochromatin that is formed dynamically under specific growth conditions by a process that requires RNAi, and is enriched in histone H3 methylated on lysine 9 (H3K9me)." [PMID:23151475, PMID:24210919]
GO:1990345
name: MTREC complex
def: "A protein complex that consists of a heterodimer formed by Red1 and Mtl1 or homologs thereof, and that promotes degradation of mRNAs and noncoding RNAs and associates with different proteins to assemble heterochromatin via distinct mechanisms." [PMID:24210919]
GO:1990346
name: BID-BCL-xl complex
def: "A heterodimeric protein complex consisting of BID and BCL-xl, members of the Bcl-2 family of anti- and proapoptotic regulators." [GOC:bhm, IntAct:EBI-709568, IntAct:EBI-822507, IntAct:EBI-822526, PMID:14634621]
GO:1990350
name: glucose transporter complex
def: "A protein complex facilitating glucose transport into, out of or within a cell, or between cells." [GOC:bhm, IntAct:EBI-959455, IntAct:EBI-960644, IntAct:EBI-960674, PMID:15449578]
GO:1990351
name: transporter complex
def: "A protein complex facilitating transport of molecules (proteins, small molecules, nucleic acids) into, out of or within a cell, or between cells." [GOC:bhm, IntAct:EBI-959455, PMID:15449578]
GO:1990352
name: BRE1 E3 ubiquitin ligase complex
def: "A homodimeric protein complex composed of the E3 ubiquitin-protein ligase BRE1. Plays a role in regulating association of RNA polymerase II with active genes." [GOC:bhm, IntAct:EBI-9102292, PMID:19531475]
GO:1990353
name: Fused-Smurf ubiquitin ligase complex
def: "A ubiquitin ligase complex. In D. melanogaster, it regulates ubiquitination and proteolysis of the BMP receptor Thickveins in cystoblasts, potentially by controlling Tkv ubiquitination and degradation." [GOC:bhm, IntAct:EBI-3405462, PMID:21145463]
GO:1990354
name: activated SUMO-E1 ligase complex
def: "A protein complex consisting of a SUMO protein bound to a SUMO activating enzyme complex. Activation by the E1 complex and linkage to the E2 enzyme UBE2I is required for the formation of covalent bonds between SUMO and its ultimate target proteins." [GOC:bhm, Intact:EBI-9345927, PMID:15660128]
GO:1990356
name: sumoylated E2 ligase complex
def: "A protein complex consisting of a SUMO (small ubiquitin-related modifier) protein bound to a SUMO-conjugating E2 ligase. Sumoylation of the E2 ligase is an intermediate step required for the formation of covalent bonds between a SUMO protein and its ultimate protein target. SUMO is transferred to the E2 ligase by a SUMO-activating E1 enzyme. Sumoylation of the target protein is either facilitated directly by the sumoylated E2 ligase or aided by an optional E3 ligase." [GOC:bhm, IntAct:EBI-9347978, PMID:18691969]
GO:1990357
name: terminal web
def: "An actin-rich cytoskeletal network located beneath the microvilli of the apical plasma membrane of polarized epithelial cells. In addition to actin filaments, the terminal web may contain actin-binding proteins, myosin motor proteins, and intermediate filaments. The terminal web can function as a contractile structure that influences the spatial distribution of microvilli as well as the development and morphogenesis of tissues containing polarized epithelial cells." [GOC:kmv, http://en.wikipedia.org/wiki/Terminal_web, PMID:19437512, PMID:24677443, PMID:7511618]
GO:1990358
name: xylanosome
def: "A multifunctional supermolecular complex, containing several proteins with hemicellulase activity. Functions to hydrolyze hemicellulose." [PMID:16769147]
GO:1990360
name: PKM2-SAICAR protein kinase complex
def: "A protein complex capable of phosphorylating a large number of protein targets. Contributes to cell proliferation under glycose starvation conditions." [GOC:bhm, PMID:24606918]
GO:1990361
name: PKM2 pyruvate kinase complex
def: "A protein complex capable of pyruvate kinase activity. PKM2 only exists as homotetramer when bound to beta-d-fructofuranose 1,6-bisphosphate (CHEBI:28013)." [GOC:bhm, PMID:24606918]
GO:1990374
name: Kir2 inward rectifier potassium channel complex
def: "A inward rectifier potassium channel complex. Homo- or heterotetramer composed of subunits of the eukaryotic Kir2 protein family. Plays a key role in maintaining the correct resting potential in eukaryotic cells." [GOC:bhm, IntAct:EBI-9510554, IntAct:EBI-9511072, PMID:16834334]
GO:1990377
name: organomineral extracellular matrix
def: "An extracellular matrix consisting of a densely packed organomineral assembly in which the mineral phase represents the majority of the material by weight." [GOC:jh2, PMID:15994301]
GO:1990378
name: upstream stimulatory factor complex
def: "A protein complex capable of sequence-specific DNA binding RNA polymerase II transcription factor activity through binding to a symmetrical DNA sequence (E-boxes) (5'-CACGTG-3'). Found in a variety of viral and cellular promoters." [GOC:bhm, IntAct:EBI-9518693, PMID:8576131]
GO:1990385
name: meiotic spindle midzone
def: "The area in the center of the meiotic spindle where the spindle microtubules from opposite poles overlap." [GOC:kmv\,PMID\:12707312]
GO:1990389
name: CUE1-UBC7 ubiquitin-conjugating enzyme complex
def: "A protein complex capable of ubiquitin-conjugating enzyme activity during ER-associated protein degradation (ERAD). In S. cerevisiae, UBC7 is the ubiquitin-conjugating enzyme (E2) and requires binding to the ER surface by CUE1." [GOC:bhm, IntAct:EBI-9207004, PMID:23028185]
GO:1990391
name: DNA repair complex
def: "A protein complex involved in DNA repair processes including direct reversal, base excision repair, nucleotide excision repair, photoreactivation, bypass, double-strand break repair pathway, and mismatch repair pathway." [GOC:bhm, PMID:22749910]
GO:1990392
name: EFF-1 complex
def: "A trimeric cell-cell fusion complex that serves as a scaffold for zippering up the extracellular domains, bringing the transmembrane segments into close proximity such that they can continue zippering within the two membranes into one. Two prefusion monomers cluster at the surface of adjacent cells. Parallel EFF-1 interactions occur across cells and a third monomer, which can come from either cell, adds on to make an intermediate, extended trimer." [GOC:bhm, IntAct:EBI-9526622, PMID:24725407]
GO:1990393
name: 3M complex
def: "A protein complex, at least composed of CUL7, CCDC8 and OBSL1, that is required for maintaining microtubule and genome integrity." [PMID:24793695, PMID:24793696]
GO:1990398
name: Cus cation efflux complex
def: "Transmembrane complex that mediates resistance to copper and silver by cation efflux directly from the cell using the proton-motive force. Spans the inner membrane, periplasm, and outer membrane. Primarily activated under anaerobic conditions by CusR and CusS but also expressed under extreme copper stress, in aerobic growth." [GOC:bhm, IntAct:EBI-9010676, PMID:23122209]
GO:1990406
name: CGRP receptor complex
def: "A transmembrane, G-protein-coupled signalling receptor complex recognized by calcitonin gene-related peptides (CGRP)." [GOC:bhm, IntAct:EBI-9009008, PMID:20826335]
GO:1990413
name: eyespot apparatus
def: "A small pigmented organelle used in single-celled organisms to detect light." [Wikipedia:Eyespot_apparatus]
GO:1990415
name: Pex17p-Pex14p docking complex
def: "A protein complex involved in the peroxisomal import machinery. In S. cerevisiae, this complex contains the proteins Pex17p, Pex14p, Pex19, and Pex13p." [GOC:rb, PMID:12667447]
GO:1990421
name: subtelomeric heterochromatin
def: "Heterochromatic regions of the chromosome found at the subtelomeric regions." [PMID:18761674, SO:0001997]
GO:1990423
name: RZZ complex
def: "A kinetochore component required for both meiotic and mitotic spindle assembly checkpoints." [PMID:12686595, PMID:15922598, PMID:20462495]
GO:1990425
name: ryanodine receptor complex
def: "A voltage-gated calcium-release channel complex of the sarcoplasmic or endoplasmic reticulum. It plays an important role in the excitation-contraction (E-C) coupling of muscle cells. RyR comprises a family of ryanodine receptors, widely expressed throughout the animal kingdom." [GOC:ame, IntAct:EBI-9632656, PMID:22822064]
GO:1990427
name: stereocilia tip-link density
def: "An electron-dense plaque at either end of a stereocilia tip link that provides the anchor in the stereocilia membrane." [PMID:19447093, PMID:21709241]
GO:1990429
name: peroxisomal importomer complex
def: "A protein complex responsible for transporting proteins into the peroxisomal matrix. An example of this complex is Pex14 found in S. cerevisae which has 9 core components and 12 transient interaction partners." [PMID:20154681, PMID:22375831]
GO:1990433
name: CSL-Notch-Mastermind transcription factor complex
def: "A DNA binding transcription factor complex consisting of CSL and mastermind proteins in complex with the cleaved, intracellular domain of Notch. It is required for both repression and activation of Notch target genes." [GOC:bhm, GOC:dos, intAct:EBI-9636287, PMID:16530045]
GO:1990434
name: lower tip-link density
def: "An electron-dense plaque at the lower end of a stereocilia tip link that provides the anchor in the stereocilia membrane at the tip of the stereocilium from which the tip link rises." [PMID:19447093]
GO:1990435
name: upper tip-link density
def: "An electron-dense plaque at the upper end of a stereocilia tip link that provides the anchor in the stereocilia membrane on the side of the stereocilium where the tip link ends." [PMID:19447093]
GO:1990449
name: amylin receptor
def: "OBSOLETE. A G-protein coupled signalling receptor complex consisting of the calcitonin receptor and a receptor activity-modifying protein (RAMP). Amylin is produced in beta-islet cells of the pancreas. It is implicated in selective inhibition of insulin-stimulated glucose utilization and glycogen deposition in muscle, gastric emptying, gastric acid secretion, postprandial glucagon secretion and food intake and aids weight loss." [GOC:bhm, IntAct:EBI-9008682, IntAct:EBI-9685417, IntAct:EBI-9685439, PMID:10871269]
GO:1990452
name: Parkin-FBXW7-Cul1 ubiquitin ligase complex
def: "A ubiquitin ligase complex containing Parkin (PARK2), the F-box protein FBXW7 (also called SEL-10) and a cullin from the Cul1 subfamily; substrate specificity is conferred by the F-box protein." [GOC:bf, GOC:PARL, PMID:12628165]
GO:1990453
name: nucleosome disassembly/reassembly complex
def: "A protein complex involved in the disassembly and subsequent reassembly of nucleosomes. It associates with the coding region of transcriptionally active genes where it interacts with the RNA polymerase II and affects its processivity during co-transcriptional RNA processing and maturation. It exists as a functionally independent part of the NuA4 complex." [GOC:bhm, IntAct:EBI-9682603, PMID:24843044]
GO:1990454
name: L-type voltage-gated calcium channel complex
def: "A type of voltage-dependent calcium channel responsible for excitation-contraction coupling of skeletal, smooth, and cardiac muscle. 'L' stands for 'long-lasting' referring to the length of activation." [GOC:ame, PMID:12946355]
GO:1990455
name: PTEN phosphatase complex
def: "A phospholipid phosphatase complex that catalyses the hydrolysis of the second messenger PtdIns (3,4,5)P3. Will also dephosphorylate PtdIns(3,4)P2, PtdIns3P, and Ins(1,3,4,5)P4. Dimerization is critical for its lipid phosphatase function." [GOC:bhm, IntAct:EBI-9637198, IntAct:EBI-9637224, PMID:24766807]
GO:1990457
name: pexophagosome
def: "A membrane-bounded intracellular vesicle involved in the degradation of peroxisome by macropexophagy." [PMID:22536249]
GO:1990462
name: omegasome
def: "Omega-shaped (as in the Greek capital letter) intracellular membrane-bounded organelle enriched in phosphatidylinositol 3-phosphate and dynamically connected to the endoplasmic reticulum. Omegasomes are the first step of the formation of autophagic vacuoles via the pre-autophagosomal structures." [GOC:mf, PMID:18725538, PMID:24591649]
GO:1990463
name: lateral cortical node
def: "A protein complex that is anchored at the cortical face of the plasma membrane, and contains proteins involved in regulating cell cycle progression. In Schizosaccharomyces pombe, lateral cortical nodes are several megadaltons in size, and contain Slf1, which anchors the complex at the membrane, and the methyltransferase Skb1 in stoichiometric quantities, and may contain other proteins." [GOC:mah, PMID:25009287]
GO:1990467
name: NuA3a histone acetyltransferase complex
def: "A NuA3 complex that catalyzes the acetylation of Histone H3. In S. cerevisiae, this complex consists of Eaf6p, Nto1p, Sas3p, Taf14p, Yng1p and associates with H3K4me3 using Yng1p." [GOC:rb, PMID:25104842]
GO:1990468
name: NuA3b histone acetyltransferase complex
def: "A NuA3 complex that catalyzes the acetylation of Histone H3. In S. cerevisiae, this complex consists of Eaf6p, Nto1p, Sas3p, Taf14p, Pdp3 and associates with H3K4me3 via Pdp3p." [GOC:rb, PMID:25104842]
GO:1990469
name: Rhino-Deadlock-Cutoff Complex
def: "Protein complex found in Drosophila consisting of the gene products of cuff, del and rhi. It regulates the licensing of transcription of dual-strand PIWI interacting RNA (piRNA) source loci by binding to dual-strand-cluster chromatin, probably via the H3K9me3-binding activity of Rhi. Rhi binding brings the putative termination cofactor Cuff in close proximity to the nascent piRNA precursor transcript which it appears to protect from degradation." [GOC:bhm, IntAct:EBI-9694217, PMID:24906153]
GO:1990474
name: synaptic vesicle, readily releasable pool
def: "A pool of synaptic vesicles that are the first to be released as a result of chemical or electrical stimulation e.g. by an action potential, have the highest presynaptic membrane fusion probability and correspond to about 1% of the total number of synaptic vesicles at a resting terminal bouton." [GOC:pad, PMID:22745285]
GO:1990475
name: synaptic vesicle, recycling pool
def: "A pool of synaptic vesicles that repopulate vacancies within the readily releasable pool (RRP) of synaptic vesicles, and require more significant stimuli than the RRP in order to release neurotransmitter; contains about 10-15% of the total number of synaptic vesicles at a resting terminal bouton." [GOC:pad, PMID:22745285]
GO:1990476
name: synaptic vesicle, resting pool
def: "A pool of synaptic vesicles that remain unreleased even after prolonged stimulation that causes a saturating degree of vesicular turnover; contains about 50-80% of the total number of synaptic vesicles at a resting terminal bouton." [GOC:pad, PMID:22745285]
GO:1990477
name: NURS complex
def: "The nuclear RNA silencing (NURS) complex is a protein complex formed by Red1, Mtl1, Red5, Rmn1, Iss10/Pir1, and Ars2/Pir2 that regulates RNA degradation and histone H3 lysine 9 methylation. It is likely related to the human CBCN complex." [PMID:24713849]
GO:1990483
name: Clr6 histone deacetylase complex I''
def: "A histone deacetylase complex involved in chromatin organization. In Schizosaccharomyces pombe this complex consists of Clr6, Nts1, Mug165, and Png3." [PMID:25002536]
GO:1990490
name: archaeal proton-transporting A-type ATPase complex
def: "A large proton-transporting two-sector ATPase protein complex that catalyzes the synthesis or hydrolysis of ATP by a rotational mechanism, coupled to the transport of protons across a membrane and is found in Archaea." [GOC:mengo_curators, PMID:15473999, PMID:24650628]
GO:1990493
name: cyclin H-CDK7 complex
def: "OBSOLETE. A protein complex consisting of cyclin H and cyclin-dependent kinase 7 (CDK7). Cyclins are characterized by periodicity in protein abundance throughout the cell cycle. Cyclin-dependent kinases represent a family of serine/threonine protein kinases that become active upon binding to a cyclin regulatory partner." [PMID:9857180]
GO:1990498
name: mitotic spindle microtubule
def: "Any microtubule that is part of a mitotic spindle; anchored at one spindle pole." [GOC:vw]
GO:1990499
name: raps-insc complex
def: "Protein complex required for the asymmetric division of neuroblasts in Drosophila. Coordinates asymmetric localization of cell fate determinants with orientation of the mitotic spindle resulting in different daughter cells upon division. Localizes at the apical cortex of the neuroblast: Raps maintains, but does not initiate, Insc apically, while Insc segregates Raps asymmetrically. Complex appears to be conserved in mammals (composed of INSC and GPSM1 or GPSM2)." [GOC:bhm, IntAct:EBI-9674411, PMID:22171003]
GO:1990500
name: eif4e-cup complex
def: "A protein complex that causes translational repression in Drosophila. Prevents assembly of ribosomes at the mRNA by interfacing with a sequence-specific RNA-binding protein leading to recruitment of the CCR4 complex and consequently, reduction of the mRNA's poly(A) tail length. The complex is also required for dorso-ventral pattern formation in the embryo." [GOC:bhm, IntAct:EBI-9674930, PMID:14723848]
GO:1990501
name: exon-exon junction subcomplex mago-y14
def: "Component of the core exon-exon-junction complex (EJC). Fairly conserved in eukaryotes; in Drosophila, consists of the Mago and Y14 (tsunagi) gene products. Important for coupling nuclear and cytoplasmic events in gene expression. Inhibits the ATPase activity of eIF4AIII (Q9VHS8) to ensure a stable association of the EJC core with the mRNA." [GOC:bhm, IntAct:EBI-9549962, PMID:12730685]
GO:1990503
name: dendritic lamellar body
def: "A specialized secretory organelle found in neurons and associated with the formation of dendrodendritic gap junctions." [PMID:7869120]
GO:1990508
name: CKM complex
def: "Cyclin-dependent kinase complex which reversibly associates with the Mediator complex. In Saccharomyces cerevisiae it consists of SSN2, SSN3, SSN8 and SRB8." [GOC:bhm, IntAct:EBI-2640924, PMID:12200444]
GO:1990509
name: PYM-mago-Y14 complex
def: "Protein complex involved in the disassembly of Mago-Y14 from the spliced mRNA during first round of translation, independently of the translational machinery. Conserved from fission yeast to humans." [GOC:bhm, IntAct:EBI-9634190, PMID:14968132]
GO:1990512
name: Cry-Per complex
def: "Nuclear transcriptional repressor complex that is capable of negatively regulating CLOCK-BMAL-dependent transactivation of genes in a delayed negative feedback manner which generates circadian rhythms." [GOC:bhm, IntAct:EBI-9693126, IntAct:EBI-9693260, IntAct:EBI-9694282, IntAct:EBI-9695878, IntAct:EBI-9695921, IntAct:EBI-9695942, IntAct:EBI-9696219, IntAct:EBI-9696270, IntAct:EBI-9696293, IntAct:EBI-9696310, IntAct:EBI-9696328, IntAct:EBI-9696345, PMID:24855952]
GO:1990513
name: CLOCK-BMAL transcription complex
def: "Transcription factor complex which interacts with E-box regulatory elements in target genes, including Period (Per1, Per2, Per3) and Cryptochrome (Cry1, Cry2), to activate their transcription during the daytime. The CRY-PER complexes inhibit CLOCK-BMAL1-driven transcription in a negative feedback loop to generate circadian rhythms." [GOC:bhm, IntAct:EBI-9696362, IntAct:EBI-9696901, IntAct:EBI-9696921, IntAct:EBI-9696951, PMID:23229515]
GO:1990520
name: separase-securin complex
def: "A protein complex that includes separase and securin as components and that inhibits chromosome separation at mitosis." [GOC:dos, GOC:vw, PMID:8978688]
GO:1990524
name: INA complex
def: "A protein complex located in the inner membrane of mitochondria that is involved in the assembly of the peripheral (or stator) stalk of the mitochondrial proton-transporting ATP synthase (also known as the F1F0 ATP synthase). In budding yeast, this complex includes Ina22p and Ina17p." [GOC:rn, PMID:24942160]
GO:1990526
name: Ste12p-Dig1p-Dig2p complex
def: "A multiprotein complex that is involved in the transcription regulation of mating genes in the yeast S. cerevisiae." [GOC:rb, PMID:16782869]
GO:1990527
name: Tec1p-Ste12p-Dig1p complex
def: "A multiprotein complex that is involved in the transcription regulation of mating genes in the yeast S. cerevisiae." [GOC:rb, PMID:16782869]
GO:1990528
name: Rvs161p-Rvs167p complex
def: "A protein complex that is involved in endocytosis in the yeast S. cerevisiae." [GOC:rb, PMID:20610658]
GO:1990529
name: glycosylphosphatidylinositol-mannosyltransferase I complex
def: "A protein complex that is involved in the transfer of the four mannoses in the GPI-anchor precursor. In yeast S. cerevisiae this complex consists of Pbn1p and Gpi14p and in rat this complex consists of PIG-X and PIG-M." [GOC:dph, GOC:rb, PMID:15635094]
GO:1990530
name: Cdc50p-Drs2p complex
def: "A protein complex that assembles on the ER membrane and is essential for the ER exit of the Cdc50p-Drs2p complex." [GOC:rb, PMID:15090616]
GO:1990531
name: Lem3p-Dnf1p complex
def: "A protein complex that functions as a phospholipid-translocating P-Type ATPase." [GOC:dph, GOC:rb, PMID:15090616]
GO:1990533
name: Dom34-Hbs1 complex
def: "A protein complex consisting of one subunit known as Dom34 or Pelota that has similarity to translation termination factor eRF1, and another subunit, Hbs1, that is a GTPase with similarity to translation termination factor eRF3. The Dom34-Hbs1 complex has a role in cotranslational mRNA quality control by promoting ribosomal subunit dissociation and peptidyl-tRNA release when translation is stalled, facilitating no-go decay and nonstop decay." [GOC:mcc, PMID:20890290, PMID:21102444, PMID:21448132, PMID:22503425]
GO:1990537
name: mitotic spindle polar microtubule
def: "Any of the mitotic spindle microtubules that come from each pole and overlap at the spindle midzone." [PMID:16079915]
GO:1990562
name: syndecan-syntenin-ALIX complex
def: "An exosome complex that is assembled in the multivesicular body (MVB) membrane and chaperoned to the exosome by the ESCRT-III machinery." [GOC:bhm, IntAct:EBI-9985210, IntAct:EBI-9985797, PMID:22660413]
GO:1990563
name: extracellular vesicular exosome complex
def: "A protein complex that is wholly or partially contained within the lumen or membrane of the extracellular vesicular exosome." [GOC:bhm, IntAct:EBI-9985210, IntAct:EBI-9985797, PMID:22660413]
GO:1990565
name: HSP90-CDC37 chaperone complex
def: "A protein complex involved in stress-induced mitophagy (mitochondrial degradation). In mammals it consists of heat shock protein HSP90 and its co-chaperone CDC37. Experimental evidence suggests that the complex enables the autophosphorylation of ULK1. Phosphorylated ULK1 in turn phosphorylates ATG13 which is an essential step in mitophagy. HSP90-CDC37 binding prevents rapid ubiquitin-dependent proteosomal degradation of its targets." [GOC:bhm, IntAct:EBI-9990131, IntAct:EBI-9990540, PMID:21855797, PMID:22939624]
GO:1990566
name: I(KACh) inward rectifier potassium channel complex
def: "An inward rectifier potassium channel complex expressed in cardiac muscle, specifically the sinoatrial node and atria, where it controls the heart rate, via regulation by G-protein-coupled receptor signalling. In mammals it is composed of GIRK1 (or Kir3.1) and GIRK4 (or Kir3.4) subunits." [GOC:ame, IntAct:EBI-9975539, PMID:9765280]
GO:1990567
name: DPS complex
def: "A protein serine/threonine phosphatase complex that in S. pombe consists of the proteins Dis2, Ppn1, and Swd22." [PMID:24945319]
GO:1990568
name: MIS18 complex
def: "OBSOLETE. A centromere complex assembly protein that is required for the deposition of CENP-A on the centromere. The Mis18 complex localizes to centromeres just prior to the pre-nucleosomal HJURP/CENP-A/H4 complex and is absolutely required for the CENP-A-specific chaperone, Holliday junction recognition protein (HJURP) to reach the centromeres. Plk1 phosphorylation activates Mis18 complex recruitment to the centromeres during G1. CDK phosphorylation of MISBP1 during G2 and mitosis, prior to the metaphase-to-anaphase transition, negatively regulates complex assembly." [GOC:bhm, IntAct:EBI-9872194, IntAct:EBI-9873644, PMID:25036634]
GO:1990572
name: TERT-RMRP complex
def: "A ribonucleoprotein complex that has RNA-directed RNA polymerase (RdRP) activity, and is composed of telomerase reverse transcriptase (TERT) and the non-coding RNA component of mitochondrial RNA processing endoribonuclease (RMRP)." [GOC:bf, GOC:BHF, GOC:nc, PMID:19701182]
GO:1990574
name: meiotic spindle astral microtubule
def: "Any of the meiotic spindle microtubules that radiate in all directions from the spindle poles and are thought to contribute to the forces that separate the poles and position them in relation to the rest of the cell." [PMID:10366596]
GO:1990578
name: perinuclear endoplasmic reticulum membrane
def: "The membrane of the perinuclear endoplasmic reticulum, which is the portion of endoplasmic reticulum, the intracellular network of tubules and cisternae, that occurs near the nucleus." [PMID:25454947]
GO:1990584
name: cardiac Troponin complex
def: "A complex of accessory proteins (cardiac troponin T, cardiac troponin I and cardiac troponin C) found associated with actin in cardiac muscle thin filaments; involved in calcium regulation important for muscle contraction." [GOC:ame, IntAct:EBI-9980814, PMID:12840750]
GO:1990586
name: divisome complex
def: "A protein complex required for prokaryotic cell division (FtsZ-dependent cytokinesis). These complexes are assembled and recruited to the cell septum in a strictly controlled sequence and co-ordinate invagination of the cell membrane, inward growth of the peptidoglycan layer, constriction of the outer membrane and separation of daughter cells." [GOC:bhm, IntAct:EBI-9547109, IntAct:EBI-9997581, PMID:15165235, PMID:21784946]
GO:1990587
name: FtsQBL complex
def: "A protein complex required for prokaryotic cell division (FtsZ-dependent cytokinesis). Part of the divisome. Assembled independently of the other divisome components in the cytoplasm prior to transport to the cell septum. In E. coli consists of FtsB, FtsL and FtsQ." [GOC:bhm, IntAct:EBI-9997581, PMID:15165235, PMID:21784946]
GO:1990588
name: FtsBL complex
def: "A protein complex required for prokaryotic cell division (FtsZ-dependent cytokinesis). Part of the divisome. Assembled independently of the other divisome components in the cytoplasm prior to transport to the cell septum. In E. coli consists of FtsB and FtsL." [GOC:bhm, IntAct:EBI-9997581, PMID:15165235, PMID:21784946]
GO:1990589
name: ATF4-CREB1 transcription factor complex
def: "Transcription factor complex consisting of ATF4 and CREB1 subunits that is capable of binding to cAMP response element (CRE) (consensus: 5'-GTGACGT[AC][AG]-3') as part of the positive regulation of transcription. Regulatory targets include the GRP78 (HSPA5) promoter in humans, whose activation by this complex is part of the ER stress response pathway." [GOC:bhm, IntAct:EBI-10043081, IntAct:EBI-10043332, PMID:12871976]
GO:1990590
name: ATF1-ATF4 transcription factor complex
def: "Transcription factor complex consisting of ATF1 and ATF4 subunits that is capable of binding to cAMP response element (CRE) (consensus: 5'-GTGACGT[AC][AG]-3') of the GRP78 (HSPA5) promoter. Involved in the ER stress response pathway." [GOC:bhm, IntAct:EBI-10043123, IntAct:EBI-10043370, PMID:12871976]
GO:1990597
name: AIP1-IRE1 complex
def: "A protein complex consisting of IRE1 (inositol-requiring enzyme-1) bound to AIP1 (ASK1-interacting protein 1/DAB2-interacting protein)." [GOC:bf, GOC:PARL, PMID:18281285]
GO:1990602
name: importin alpha-subunit nuclear import complex
def: "A trimeric protein complex which functions to shuttle the importin alpha-subunit into the nucleus through the nuclear pore to facilitate another round of mRNP incorporation and regulation. In Drosophila it consists of Cdm (Imp13), Mago and Tsu (Y14)." [GOC:bhm, IntAct:EBI-9673795, PMID:20122403]
GO:1990604
name: IRE1-TRAF2-ASK1 complex
def: "A protein complex of the endoplasmic reticulum membrane that consists of IRE1 (Inositol-requiring enzyme-1), TRAF2 (TNF receptor-associated factor 2) and ASK1 (Apoptosis signal-regulating kinase 1, a MAP3K)." [GOC:bf, GOC:PARL, PMID:12050113, PMID:23000344]
GO:1990612
name: Sad1-Kms2 LINC complex
def: "A LINC complex implicated in the connection of DNA double strand breaks to the cytoskeleton during DNA double-strand break repair." [GOC:vw, PMID:24947240]
GO:1990615
name: Kelch-containing formin regulatory complex
def: "A protein complex that regulates actin cable formation, polarized cell growth, and cytokinesis in a formin-dependent manner. In S. cerevisiae the complex is composed of Bud14p and two Kelch family proteins, Kel1p and Kel2p." [PMID:24828508]
