GO:0000005
name: ribosomal chaperone activity
def: "OBSOLETE. Assists in the correct assembly of ribosomes or ribosomal subunits in vivo, but is not a component of the assembled ribosome when performing its normal biological function." [GOC:jl, PMID:12150913]
GO:0000006
name: high-affinity zinc uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Zn2+(out) = Zn2+(in), probably powered by proton motive force. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [TC:2.A.5.1.1]
GO:0000007
name: low-affinity zinc ion transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Zn2+ = Zn2+, probably powered by proton motive force. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:mtg_transport, ISBN:0815340729]
GO:0000008
name: thioredoxin
def: "OBSOLETE. A small disulfide-containing redox protein that serves as a general protein disulfide oxidoreductase. Interacts with a broad range of proteins by a redox mechanism, based on the reversible oxidation of 2 cysteine thiol groups to a disulfide, accompanied by the transfer of 2 electrons and 2 protons. The net result is the covalent interconversion of a disulfide and a dithiol." [GOC:kd]
GO:0000009
name: alpha-1,6-mannosyltransferase activity
def: "Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->6) linkage." [GOC:mcc, PMID:2644248]
GO:0000010
name: trans-hexaprenyltranstransferase activity
def: "Catalysis of the reaction: all-trans-hexaprenyl diphosphate + isopentenyl diphosphate = all-trans-heptaprenyl diphosphate + diphosphate." [KEGG:R05612, RHEA:20839]
GO:0000014
name: single-stranded DNA endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within a single-stranded deoxyribonucleic acid molecule by creating internal breaks." [GOC:mah]
GO:0000016
name: lactase activity
def: "Catalysis of the reaction: lactose + H2O = D-glucose + D-galactose." [EC:3.2.1.108]
GO:0000026
name: alpha-1,2-mannosyltransferase activity
def: "Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->2) linkage." [GOC:mcc, PMID:10521541]
GO:0000030
name: mannosyltransferase activity
def: "Catalysis of the transfer of a mannosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai, GOC:cjm]
GO:0000031
name: mannosylphosphate transferase activity
def: "Catalysis of the transfer of a mannosylphosphate group from one compound to another." [GOC:jl]
GO:0000033
name: alpha-1,3-mannosyltransferase activity
def: "Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->3) linkage." [GOC:mcc, PMID:10521541]
GO:0000034
name: adenine deaminase activity
def: "Catalysis of the reaction: adenine + H2O = hypoxanthine + NH3." [EC:3.5.4.2]
GO:0000035
name: acyl binding
def: "Interacting selectively and non-covalently with an acyl group, any group formally derived by removal of the hydroxyl group from the acid function of a carboxylic acid." [GOC:curators, ISBN:0198506732]
GO:0000036
name: ACP phosphopantetheine attachment site binding involved in fatty acid biosynthetic process
def: "Interacting selectively and non-covalently with the attachment site of the phosphopantetheine prosthetic group of an acyl carrier protein (ACP) as part of the process of fatty acid biosynthesis." [CHEBI:22221, GOC:jl, GOC:vw]
GO:0000039
name: plasma membrane long-chain fatty acid transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0000047
name: Rieske iron-sulfur protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0000048
name: peptidyltransferase activity
def: "Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2)." [EC:2.3.2.12, PMID:11433365, PMID:9242921]
GO:0000049
name: tRNA binding
def: "Interacting selectively and non-covalently with transfer RNA." [GOC:ai]
GO:0000062
name: fatty-acyl-CoA binding
def: "Interacting selectively and non-covalently with acyl-CoA, any derivative of coenzyme A in which the sulfhydryl group is in thiolester linkage with a fatty acyl group." [GOC:jl, ISBN:0198506732]
GO:0000064
name: L-ornithine transmembrane transporter activity
def: "Catalysis of the transfer of L-ornithine from one side of a membrane to the other. L-ornithine is 2,5-diaminopentanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0000095
name: S-adenosyl-L-methionine transmembrane transporter activity
def: "Catalysis of the transfer of S-adenosylmethionine from one side of a membrane to the other. S-adenosylmethionine is S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism." [GOC:ai]
GO:0000099
name: sulfur amino acid transmembrane transporter activity
def: "Catalysis of the transfer of sulfur amino acids from one side of a membrane to the other. Sulphur amino acids contain sulfur in the form of cystine, methionine or their derivatives." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0000100
name: S-methylmethionine transmembrane transporter activity
def: "Catalysis of the transfer of S-methylmethionine from one side of a membrane to the other." [GOC:ai]
GO:0000102
name: L-methionine secondary active transmembrane transporter activity
def: "Catalysis of the transfer of L-methionine from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0000104
name: succinate dehydrogenase activity
def: "Catalysis of the reaction: succinate + acceptor = fumarate + reduced acceptor." [EC:1.3.99.1, GOC:kd]
GO:0000107
name: imidazoleglycerol-phosphate synthase activity
def: "Catalysis of the reaction: phosphoribulosylformimino-AICAR-P + L-glutamine = D-erythro-imidazole-glycerol-phosphate + aminoimidazole carboxamide ribonucleotide + L-glutamate + 2 H(+)." [MetaCyc:GLUTAMIDOTRANS-RXN]
GO:0000121
name: glycerol-1-phosphatase activity
def: "Catalysis of the reaction: glycerol-1-phosphate + H2O = glycerol + phosphate." [EC:3.1.3.21]
GO:0000140
name: acylglycerone-phosphate reductase activity
def: "Catalysis of the reaction: 1-palmitoylglycerol-3-phosphate + NADP+ = palmitoylglycerone phosphate + NADPH + H(+)." [EC:1.1.1.101]
GO:0000146
name: microfilament motor activity
def: "Catalysis of movement along a microfilament, coupled to the hydrolysis of a nucleoside triphosphate (usually ATP)." [GOC:mah, ISBN:0815316194]
GO:0000149
name: SNARE binding
def: "Interacting selectively and non-covalently with a SNARE (soluble N-ethylmaleimide-sensitive factor attached protein receptor) protein." [PMID:12642621]
GO:0000150
name: recombinase activity
def: "Catalysis of the identification and base-pairing of homologous sequences between single-stranded DNA and double-stranded DNA." [GOC:elh]
GO:0000155
name: phosphorelay sensor kinase activity
def: "Catalysis of the phosphorylation of a histidine residue in response to detection of an extracellular signal such as a chemical ligand or change in environment, to initiate a change in cell state or activity. The two-component sensor is a histidine kinase that autophosphorylates a histidine residue in its active site. The phosphate is then transferred to an aspartate residue in a downstream response regulator, to trigger a response." [GOC:bf, GOC:mcc, PMID:10966457, PMID:20223701, PMID:9191038]
GO:0000156
name: phosphorelay response regulator activity
def: "Responds to a phosphorelay sensor to initiate a change in cell state or activity. The activity of the response regulator is regulated by transfer of a phosphate from a histidine residue in the sensor, to an aspartate residue in the response regulator. Many but not all response regulators act as transcriptional regulators to elicit a response." [GOC:bf, PMID:10966457, PMID:11842140]
GO:0000166
name: nucleotide binding
def: "Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose." [GOC:mah, ISBN:0198547684]
GO:0000170
name: sphingosine hydroxylase activity
def: "Catalysis of the hydroxylation of sphingolipid long chain bases." [PMID:9556590]
GO:0000171
name: ribonuclease MRP activity
def: "Catalysis of the site-specific cleavage of RNA by a catalytic RNA-mediated mechanism; substrates include the A3 site in the ITS1 of pre-rRNA." [PMID:17881380]
GO:0000175
name: 3'-5'-exoribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of an RNA molecule." [GOC:mah, ISBN:0198547684]
GO:0000179
name: rRNA (adenine-N6,N6-)-dimethyltransferase activity
def: "Catalysis of the dimethylation two adjacent A residues in the loop closing the 3'-terminal stem of the 18S rRNA, using S-adenosyl-L-methionine as a methyl donor." [ISBN:1555811337, PMID:10690410]
GO:0000182
name: rDNA binding
def: "Interacting selectively and non-covalently with DNA sequences encoding ribosomal RNA." [GOC:mah]
GO:0000210
name: NAD+ diphosphatase activity
def: "Catalysis of the reaction: NAD+ + H2O = AMP + NMN." [EC:3.6.1.22]
GO:0000211
name: protein degradation tagging activity
def: "OBSOLETE. Covalent addition of polyubiquitin to another protein, targeting the tagged protein for destruction." [GOC:cl, ISBN:0815316194]
GO:0000213
name: tRNA-intron endonuclease activity
def: "Catalysis of the endonucleolytic cleavage of pre-tRNA, producing 5'-hydroxyl and 2',3'-cyclic phosphate termini, and specifically removing the intron." [EC:3.1.27.9]
GO:0000215
name: tRNA 2'-phosphotransferase activity
def: "Catalysis of the reaction: 2'-phospho-[ligated tRNA] + NAD+ = mature tRNA + ADP ribose 1'',2''-phosphate + nicotinamide + H2O. This reaction is the transfer of the splice junction 2-phosphate from ligated tRNA to NAD+ to produce ADP-ribose 1'-2' cyclic phosphate." [EC:2.7.1.160, PMID:9148937]
GO:0000217
name: DNA secondary structure binding
def: "Interacting selectively and non-covalently with DNA containing secondary structure elements such as four-way junctions, bubbles, loops, Y-form DNA, or double-strand/single-strand junctions." [GOC:krc]
GO:0000224
name: peptide-N4-(N-acetyl-beta-glucosaminyl)asparagine amidase activity
def: "Catalysis of the reaction: 4-N-(N-acetyl-D-glucosaminyl)-protein + H2O = N-acetyl-beta-D-glucosaminylamine + peptide L-aspartate. This reaction is the hydrolysis of an N4-(acetyl-beta-D-glucosaminyl)asparagine residue in which the N-acetyl-D-glucosamine residue may be further glycosylated, to yield a (substituted) N-acetyl-beta-D-glucosaminylamine and the peptide containing an aspartic residue." [EC:3.5.1.52]
GO:0000225
name: N-acetylglucosaminylphosphatidylinositol deacetylase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosaminylphosphatidylinositol + H2O = D-glucosaminylphosphatidylinositol + acetate. This reaction is the second step of the biosynthesis of glycosylphosphatidylinositol (GPI), used to anchor various eukaryotic proteins to the cell-surface membrane." [EC:3.5.1.89]
GO:0000227
name: oxaloacetate secondary active transmembrane transporter activity
def: "Catalysis of the transfer of oxaloacetate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0000234
name: phosphoethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + ethanolamine phosphate = S-adenosyl-L-homocysteine + N-methylethanolamine phosphate." [EC:2.1.1.103]
GO:0000246
name: delta24(24-1) sterol reductase activity
def: "Catalysis of the reaction: ergosterol + NADP(+) = ergosta-5,7,22,24(24(1))-tetraen-3beta-ol + H(+) + NADPH." [EC:1.3.1.71, RHEA:18504]
GO:0000247
name: C-8 sterol isomerase activity
def: "Catalysis of the reaction which results in unsaturation at C-7 in the B ring of sterols." [MetaCyc:RXN3O-203, PMID:8988026]
GO:0000248
name: C-5 sterol desaturase activity
def: "Catalysis of the reaction: 5,7,24(28)-ergostatrienol + O2 + NADPH = 5,7,22,24(28)-ergostatetraenol + 2 H2O + NADP+." [MetaCyc:RXN3O-227]
GO:0000249
name: C-22 sterol desaturase activity
def: "Catalysis of the formation of the C-22(23) double bond in the sterol side chain. An example reaction: 5,7,24(28)-ergostatrienol + O2 + NADPH = 5,7,22,24(28)-ergostatetraenol + 2 H2O + NADP+." [MetaCyc:RXN3O-227]
GO:0000250
name: lanosterol synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = lanosterol. This is a cyclization reaction that forms the sterol nucleus." [EC:5.4.99.7, RHEA:14624]
GO:0000252
name: C-3 sterol dehydrogenase (C-4 sterol decarboxylase) activity
def: "Catalysis of the reaction: 3-beta-hydroxy-4-beta-methyl-5-alpha-cholesta-8,24-dien-4-alpha-carboxylate + NAD(P)+ = 4-alpha-methyl-5-alpha-cholesta-8,24-dien-3-one + CO2 + NAD(P)H." [EC:1.1.1.170, PMID:9811880]
GO:0000253
name: 3-keto sterol reductase activity
def: "Catalysis of the reaction: a 3-beta-hydroxyl sterol + NADP+ = a 3-keto sterol + NADPH + H(+)." [EC:1.1.1.270, GOC:mah, MetaCyc:1.1.1.270-RXN, MetaCyc:RXN3O-4110, MetaCyc:RXN66-19, MetaCyc:RXN66-24, MetaCyc:RXN66-314, MetaCyc:RXN66-319, PMID:9811880]
GO:0000254
name: C-4 methylsterol oxidase activity
def: "Catalysis of the reaction: 4,4-dimethyl-5-alpha-cholesta-8,24-dien-3-beta-ol + NAD(P)H + H(+) + O2 = 4-beta-hydroxymethyl-4-alpha-methyl-5-alpha-cholesta-8,24-dien-3-beta-ol + NAD(P)+ + H2O." [EC:1.14.13.72, PMID:9811880]
GO:0000257
name: nitrilase activity
def: "Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH3. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides." [EC:3.5.5.1, GOC:kd]
GO:0000258
name: isoleucine/valine:sodium symporter activity
def: "OBSOLETE. Catalysis of the reaction: (isoleucine or valine)(out) + Na+(out) = (isoleucine or valine)(in) + Na+(in)." [TC:2.A.26.1.1]
GO:0000259
name: intracellular nucleoside transmembrane transporter activity
def: "OBSOLETE. Enables the directed movement of a nucleoside, a nucleobase linked to either beta-D-ribofuranose (ribonucleoside) or 2-deoxy-beta-D-ribofuranose (a deoxyribonucleotide) within a cell." [GOC:ai]
GO:0000260
name: hydrogen-translocating V-type ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + H(+)(in) = ADP + phosphate + H+(out). Found in vacuoles of eukaryotes and in bacteria." [TC:3.A.2.2.1, TC:3.A.2.2.3]
GO:0000261
name: sodium-translocating V-type ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + Na+(in) = ADP + phosphate + Na+(out). Found in vacuoles of eukaryotes and in bacteria." [TC:3.A.2.2.2]
GO:0000263
name: heterotrimeric G-protein GTPase, alpha-subunit
def: "OBSOLETE. Subunit of a heterotrimeric G-protein GTPase that contains the guanine nucleotide binding site and possesses GTPase activity." [GOC:mah, ISBN:0198547684]
GO:0000264
name: heterotrimeric G-protein GTPase, beta-subunit
def: "OBSOLETE. Subunit of a heterotrimeric G-protein GTPase; associates tightly with the gamma subunit." [GOC:mah, ISBN:0198547684]
GO:0000265
name: heterotrimeric G-protein GTPase, gamma-subunit
def: "OBSOLETE. Smallest subunit of a heterotrimeric G-protein GTPase; associates tightly with the beta subunit." [GOC:mah, ISBN:0198547684]
GO:0000268
name: peroxisome targeting sequence binding
def: "Interacting selectively and non-covalently with a peroxisomal targeting sequence, any of several sequences of amino acids within a protein that can act as a signal for the localization of the protein into the peroxisome." [GOC:mah, ISBN:0879693568]
GO:0000269
name: toxin export channel activity
def: "Enables the energy independent passage of toxins, sized less than 1000 Da, across a membrane towards the outside of the cell. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport]
GO:0000277
name: [cytochrome c]-lysine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + cytochrome c L-lysine = S-adenosyl-L-homocysteine + cytochrome c N6-methyl-L-lysine. This is the addition of a methyl group to the N6 atom of a lysine residue in cytochrome c." [EC:2.1.1.59]
GO:0000285
name: 1-phosphatidylinositol-3-phosphate 5-kinase activity
def: "Catalysis of the reaction: a 1-phosphatidyl-1D-myo-inositol 3-phosphate + ATP = a 1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.150, RHEA:13612]
GO:0000286
name: alanine dehydrogenase activity
def: "Catalysis of the reaction: L-alanine + H2O + NAD+ = pyruvate + NH3 + NADH + H(+)." [EC:1.4.1.1]
GO:0000287
name: magnesium ion binding
def: "Interacting selectively and non-covalently with magnesium (Mg) ions." [GOC:ai]
GO:0000293
name: ferric-chelate reductase activity
def: "Catalysis of the reaction: 2 Fe2+ + NAD+ = 2 Fe3+ + NADH + H(+)." [EC:1.16.1.7]
GO:0000295
name: adenine nucleotide transmembrane transporter activity
def: "Catalysis of the transfer of adenine nucleotides (AMP, ADP, and ATP) from one side of the membrane to the other." [PMID:11566870]
GO:0000297
name: spermine transmembrane transporter activity
def: "Catalysis of the transfer of spermine from one side of the membrane to the other. Spermine is a polybasic amine found in human sperm, in ribosomes and in some viruses, which is involved in nucleic acid packaging. Synthesis is regulated by ornithine decarboxylase which plays a key role in control of DNA replication." [GOC:ai]
GO:0000298
name: endopolyphosphatase activity
def: "Catalysis of the reaction: polyphosphate + n H2O = (n+1) oligophosphate. The product contains 4 or 5 phosphate residues." [EC:3.6.1.10]
GO:0000309
name: nicotinamide-nucleotide adenylyltransferase activity
def: "Catalysis of the reaction: ATP + nicotinamide nucleotide = diphosphate + NAD+." [EC:2.7.7.1]
GO:0000310
name: xanthine phosphoribosyltransferase activity
def: "Catalysis of the reaction: 5-phospho-alpha-D-ribose 1-diphosphate + xanthine = (9-D-ribosylxanthine)-5'-phosphate + diphosphate." [EC:2.4.2.22, GOC:clt]
GO:0000319
name: sulfite transmembrane transporter activity
def: "Catalysis of the transfer of sulfite ions from one side of a membrane to the other." [GOC:as]
GO:0000332
name: template for synthesis of G-rich strand of telomere DNA activity
def: "Provision of the template used by reverse transcriptase to synthesize the G-rich strand of telomeric DNA." [PMID:11812242]
GO:0000334
name: 3-hydroxyanthranilate 3,4-dioxygenase activity
def: "Catalysis of the reaction: 3-hydroxyanthranilate + O(2) = cis,cis-2-amino-3-(3-oxoprop-1-enyl)but-2-enedioate + H(+)." [EC:1.13.11.6, RHEA:17956]
GO:0000339
name: RNA cap binding
def: "Interacting selectively and non-covalently with a 7-methylguanosine (m7G) group or derivative located at the 5' end of an RNA molecule." [GOC:krc]
GO:0000340
name: RNA 7-methylguanosine cap binding
def: "Interacting selectively and non-covalently with the 7-methylguanosine group added cotranscriptionally to the 5' end of RNA molecules transcribed by polymerase II." [GOC:krc]
GO:0000341
name: RNA trimethylguanosine cap binding
def: "Interacting selectively and non-covalently with the trimethylguanosine (m(3)(2,2,7)-GTP) group located at the 5' end of some RNA molecules. Such trimethylated cap structures, generally produced by posttranscriptional modification of a 7-methylguanosine cap, are often found on snRNAs and snoRNAs transcribed by RNA polymerase II, but have also be found on snRNAs transcribed by RNA polymerase III. They have also been found on a subset of the mRNA population in some species, e.g. C. elegans." [GOC:krc]
GO:0000342
name: RNA cap 4 binding
def: "Interacting selectively and non-covalently with a hypermethylated cap structure consisting of 7-methylguanosine (m(7)G) followed by four methylated nucleotides (cap 4): 7-methylguanosine-ppp-N6, N6, 2'-O-trimethyladenosine-p-2'-O-methyladenosine-p-2'-O-methylcytosine-p-N3, 2'-O-dimethyluridine Such caps are known to be found at the 5' ends of SL RNAs of trypanosomatid protozoa." [GOC:krc, PMID:10880518, PMID:12121975]
GO:0000362
name: first U2-type spliceosomal transesterification activity
def: "OBSOLETE. Catalysis of the first transesterification reaction of U2-type spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, which is still joined to the 3' exon. The catalytic site is thought to be formed by U6 and/or U2 snRNA, and/or associated proteins." [GOC:krc, ISBN:0879695897]
GO:0000363
name: first U12-type spliceosomal transesterification activity
def: "OBSOLETE. Catalysis of the first transesterification reaction of U12-type spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, which is still joined to the 3' exon. The catalytic site is thought to be formed by U6atac snRNA and/or U2atac snRNA, and/or associated proteins." [GOC:krc, ISBN:0879695897]
GO:0000364
name: second U2-type spliceosomal transesterification activity
def: "OBSOLETE. Catalysis of the second transesterification reaction of U2-type spliceosomal mRNA splicing. Ligation of the two exons occurs via a transesterification reaction where the free 3'-hydroxyl group of the 5' exon is the nucleophile attacking the 3' splice site. Non-expressed sequences are now detached from the exons. In cis splicing, the intron is in a lariat structure. The catalytic site is thought to be formed by U6 and/or U2 snRNAs and/or associated proteins." [GOC:krc, ISBN:0879695897]
GO:0000367
name: second U12-type spliceosomal transesterification activity
def: "OBSOLETE. Catalysis of the second transesterification reaction of U12-type spliceosomal mRNA splicing. Ligation of the two exons occurs via a transesterification reaction where the free 3'-hydroxyl group of the 5' exon is the nucleophile attacking the 3' splice site. Non-expressed sequences are now detached from the exons. In cis splicing, the intron is in a lariat structure. The catalytic site is thought to be formed by U6 and/or U2 snRNAs and/or associated proteins." [GOC:krc, ISBN:0879695897]
GO:0000384
name: first spliceosomal transesterification activity
def: "Catalysis of the first transesterification reaction of spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, while it is still joined to the 3' exon." [GOC:krc, ISBN:0879695897]
GO:0000386
name: second spliceosomal transesterification activity
def: "Catalysis of the second transesterification reaction of spliceosomal mRNA splicing. Ligation of the two exons occurs via a transesterification reaction where the free 3'-hydroxyl group of the 5' exon is the nucleophile attacking the 3' splice site. Non-expressed sequences are now detached from the exons. In cis splicing, the intron is in a lariat structure." [GOC:krc, ISBN:0879695897]
GO:0000400
name: four-way junction DNA binding
def: "Interacting selectively and non-covalently with DNA containing four-way junctions, also known as Holliday junctions, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices." [GOC:krc, ISBN:0815332181, PMID:15563464]
GO:0000401
name: open form four-way junction DNA binding
def: "Interacting selectively and non-covalently with DNA containing the open form of a four-way junction, also known as a Holliday junction, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices. The open form of a four-way junction can be diagrammed without any of the strands crossing over." [GOC:krc, ISBN:0815332181, PMID:15563464]
GO:0000402
name: crossed form four-way junction DNA binding
def: "Interacting selectively and non-covalently with DNA containing the crossed form of a four-way junction, also known as a Holliday junction, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices. The crossed form of a four-way junction cannot be diagrammed without any of the strands crossing over, and instead contains a single crossover between two of the strands." [GOC:krc, ISBN:0815332181, PMID:15563464]
GO:0000403
name: Y-form DNA binding
def: "Interacting selectively and non-covalently with segment of DNA shaped like a Y. This shape occurs when DNA contains a region of paired double-stranded DNA on one end and a region of unpaired DNA strands on the opposite end." [GOC:elh, PMID:16781730]
GO:0000404
name: heteroduplex DNA loop binding
def: "Interacting selectively and non-covalently with DNA containing a loop. A loop occurs when DNA contains a large insertion or deletion that causes a region of unpaired single-stranded DNA to loop out, while the rest of the DNA is in a paired double-stranded configuration." [GOC:elh, PMID:16781730]
GO:0000405
name: bubble DNA binding
def: "Interacting selectively and non-covalently with DNA that contains a bubble. A bubble occurs when DNA contains a region of unpaired, single-stranded DNA flanked on both sides by regions of paired, double-stranded DNA." [GOC:elh, GOC:vw, PMID:16781730]
GO:0000406
name: double-strand/single-strand DNA junction binding
def: "Interacting selectively and non-covalently with a region of DNA that contains double-stranded DNA flanked by a region of single-stranded DNA." [GOC:elh, PMID:16781730]
GO:0000496
name: base pairing
def: "Interacting selectively and non-covalently with nucleic acid via hydrogen bonds between the bases of a gene product molecule and the bases of a target nucleic acid molecule." [GOC:krc]
GO:0000497
name: base pairing with DNA
def: "Interacting selectively and non-covalently with nucleic acid via hydrogen bonds between the bases of a gene product molecule and the bases of a target DNA molecule." [GOC:krc]
GO:0000498
name: base pairing with RNA
def: "Interacting selectively and non-covalently with ribonucleic acid (RNA) via hydrogen bonds between the bases of a gene product molecule and the bases of a target RNA molecule." [GOC:krc]
GO:0000499
name: base pairing with mRNA
def: "Interacting selectively and non-covalently with messenger ribonucleic acid (mRNA) via hydrogen bonds between the bases of a gene product molecule and the bases of a target mRNA molecule." [GOC:krc]
GO:0000700
name: mismatch base pair DNA N-glycosylase activity
def: "Catalysis of the removal of single bases present in mismatches by the cleavage the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic/apyrimidinic (AP) site." [GOC:elh, PMID:9224623]
GO:0000701
name: purine-specific mismatch base pair DNA N-glycosylase activity
def: "Catalysis of the removal of purines present in mismatches, especially opposite oxidized purines, by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic (AP) site." [GOC:elh, PMID:9224623]
GO:0000702
name: oxidized base lesion DNA N-glycosylase activity
def: "Catalysis of the removal of oxidized bases by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic/apyrimidinic (AP) site." [GOC:elh, PMID:11554296]
GO:0000703
name: oxidized pyrimidine nucleobase lesion DNA N-glycosylase activity
def: "Catalysis of the removal oxidized pyrimidine bases by cleaving the N-C1' glycosidic bond between the oxidized pyrimidine and the deoxyribose sugar. The reaction involves formation of a covalent enzyme-pyrimidine base intermediate. Release of the enzyme and free base by a beta-elimination or a beta, gamma-elimination mechanism results in the cleavage of the DNA backbone 3' of the apyrimidinic (AP) site." [GOC:elh, PMID:11554296]
GO:0000704
name: pyrimidine dimer DNA N-glycosylase activity
def: "Catalysis of the removal of pyrimidine dimers by removing the 5' pyrimidine of the dimer by cleaving the N-C1' glycosidic bond between the 5' pyrimidine of the dimer and the deoxyribose sugar. The reaction releases the 5' pyrimidine of the dimer and leaves an apurinic (AP) site. The reaction involves the formation of a covalent enzyme substrate intermediate. Release of the enzyme and free base by a beta-elimination or a beta, gamma-elimination mechanism results in the cleavage of the DNA backbone 3' of the apyrimidinic (AP) site." [GOC:elh, PMID:9224623]
GO:0000721
name: (R,R)-butanediol dehydrogenase activity
def: "Catalysis of the reversible reaction: (R,R)-butane-2,3-diol + NAD+ = (R)-acetoin + NADH + H(+)." [EC:1.1.1.4]
GO:0000739
name: DNA strand annealing activity
def: "OBSOLETE. Facilitates the base-pairing of complementary single-stranded DNA." [GOC:elh]
GO:0000772
name: mating pheromone activity
def: "The activity of binding to and activating specific cell surface receptors, thereby inducing a behavioral or physiological response(s) from a responding organism or cell that leads to the transfer or union of genetic material between organisms or cells. The mating pheromone can either be retained on the cell surface or secreted." [GOC:clt, GOC:elh]
GO:0000773
name: phosphatidyl-N-methylethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phosphatidyl-N-methylethanolamine = S-adenosyl-L-homocysteine + phosphatidyl-N-dimethylethanolamine." [EC:2.1.1.71]
GO:0000774
name: adenyl-nucleotide exchange factor activity
def: "Stimulates the hydrolysis and exchange of adenyl nucleotides by other proteins." [GOC:kd]
GO:0000810
name: diacylglycerol diphosphate phosphatase activity
def: "Catalysis of the reaction: a 1,2-diacyl-sn-glycerol 3-diphosphate + H2O = a 1,2-diacyl-sn-glycerol 3-phosphate + phosphate." [EC:3.1.3.81, GOC:kad, MetaCyc:RXN-11277, PMID:8567632, PMID:9452443]
GO:0000822
name: inositol hexakisphosphate binding
def: "Interacting selectively and non-covalently with inositol hexakisphosphate." [GOC:go_curators]
GO:0000823
name: inositol-1,4,5-trisphosphate 6-kinase activity
def: "Catalysis of the reaction: D-myo-inositol 1,4,5-trisphosphate + ATP = D-myo-inositol 1,4,5,6-tetrakisphosphate + ADP + 2 H(+)." [MetaCyc:2.7.1.151-RXN]
GO:0000824
name: inositol tetrakisphosphate 3-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol 1,4,5,6-tetrakisphosphate = ADP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate." [GOC:elh]
GO:0000825
name: inositol tetrakisphosphate 6-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4,5-tetrakisphosphate + ATP = 1D-myo-inositol 1,3,4,5,6-pentakisphosphate + ADP." [GOC:elh]
GO:0000826
name: inositol pyrophosphate synthase activity
def: "OBSOLETE. Catalysis of the phosphorylation of inositol phosphates which possess diphosphate bonds." [GOC:elh, PMID:16429326]
GO:0000827
name: inositol-1,3,4,5,6-pentakisphosphate kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate = ADP + diphospho-1D-myo-inositol tetrakisphosphate. The isomeric configuration of diphospho-1D-myo-inositol tetrakisphosphate is unknown." [GOC:elh, PMID:11311242]
GO:0000828
name: inositol hexakisphosphate kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol 1,2,3,4,5,6-hexakisphosphate = ADP + diphospho-1D-myo-inositol-pentakisphosphate. The isomeric configuration of diphospho-1D-myo-inositol-pentakisphosphate (PP-IP5) is unknown." [GOC:elh, GOC:vw, PMID:16429326]
GO:0000829
name: inositol heptakisphosphate kinase activity
def: "Catalysis of the reaction: ATP + diphospho-1D-myo-inositol-pentakisphosphate = ADP + bis(diphospho)-1D-myo-inositol-tetrakisphosphate. The isomeric configurations of the diphospho-1D-myo-inositol-pentakisphosphate (PP-IP5) and bis(diphospho)-1D-myo-inositol-tetrakisphosphate (bis-PP-IP4) are unknown." [GOC:elh, PMID:16429326]
GO:0000830
name: inositol hexakisphosphate 4-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 4-diphospho-1D-myo-inositol (1,2,3,5,6)pentakisphosphate." [GOC:elh, PMID:16429326]
GO:0000831
name: inositol hexakisphosphate 6-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 6-diphospho-1D-myo-inositol (1,2,3,4,5)pentakisphosphate." [GOC:elh, PMID:16429326]
GO:0000832
name: inositol hexakisphosphate 5-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol 1,2,3,4,5,6-hexakisphosphate = ADP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate." [MetaCyc:2.7.1.152-RXN]
GO:0000833
name: inositol heptakisphosphate 4-kinase activity
def: "Catalysis of the reaction: ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 4,5-bisdiphosphoinositol-1D-myoinositol (1,2,3,6)tetrakisphosphate." [GOC:elh, PMID:16429326]
GO:0000834
name: inositol heptakisphosphate 6-kinase activity
def: "Catalysis of the reaction: ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 5,6-bisdiphosphoinositol-1D-myoinositol (1,2,3,4)tetrakisphosphate." [GOC:elh, PMID:16429326]
GO:0000900
name: translation repressor activity, nucleic acid binding
def: "Antagonizes the ribosome-mediated translation of mRNA into a polypeptide via direct binding (through a selective and non-covalent interaction) to nucleic acid." [GOC:clt, GOC:vw]
GO:0000901
name: translation repressor activity, non-nucleic acid binding
def: "Antagonizes the ribosome-mediated translation of mRNA into a polypeptide but does not bind directly to nucleic acid." [GOC:clt]
GO:0000906
name: 6,7-dimethyl-8-ribityllumazine synthase activity
def: "Catalysis of the reaction: 3,4-dihydroxy-2-butanone-4-phosphate + 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione = 6,7-dimethyl-8-ribityllumazine + phosphate." [EC:5.4.-.-, PMID:7559556]
GO:0000907
name: sulfonate dioxygenase activity
def: "Catalysis of the reaction: sulfonate + 2-oxoglutarate + O2 = sulfite + aminoacetaldehyde + succinate + CO2." [EC:1.14.11.-, GOC:clt, PMID:10482536]
GO:0000908
name: taurine dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate + O(2) + taurine = aminoacetaldehyde + CO(2) + succinate + sulfite." [EC:1.14.11.17, RHEA:15912]
GO:0000944
name: base pairing with rRNA
def: "Interacting selectively and non-covalently with ribosomal ribonucleic acid (rRNA) via hydrogen bonds between the bases of a gene product molecule and the bases of a target rRNA molecule." [GOC:krc]
GO:0000945
name: base pairing with snRNA
def: "Interacting selectively and non-covalently with small nuclear ribonucleic acid (snRNA) via hydrogen bonds between the bases of a gene product molecule and the bases of a target snRNA molecule." [GOC:krc]
GO:0000946
name: base pairing with tRNA
def: "Interacting selectively and non-covalently with transfer ribonucleic acid (mRNA) via hydrogen bonds between the bases of a gene product molecule and the bases of a target tRNA molecule." [GOC:krc]
GO:0000975
name: regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that regulates a DNA-based process. Such processes include transcription, DNA replication, and DNA repair." [GOC:txnOH]
GO:0000976
name: transcription regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls transcription of that section of the DNA. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH]
GO:0000977
name: RNA polymerase II regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls the transcription of a gene or cistron by RNA polymerase II." [GOC:txnOH]
GO:0000978
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II." [GOC:txnOH]
GO:0000979
name: RNA polymerase II core promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for transcription factors of the RNA polymerase II basal transcription machinery." [GOC:txnOH]
GO:0000980
name: RNA polymerase II distal enhancer sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a RNA polymerase II (Pol II) distal enhancer. In mammalian cells, enhancers are distal sequences that increase the utilization of some promoters, and can function in either orientation and in any location (upstream or downstream) relative to the core promoter." [GOC:txnOH]
GO:0000981
name: sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0000982
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) in order to modulate transcription by RNAP II." [GOC:txnOH]
GO:0000983
name: RNA polymerase II core promoter sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in an RNA polymerase II (Pol II) core promoter, the region composed of the transcription start site and binding sites for transcription factors of the Pol II basal transcription machinery, in order to modulate transcription by Pol II. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0000984
name: bacterial-type RNA polymerase regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is part of a regulatory region that controls the transcription of a gene or operon by a bacterial-type RNA polymerase." [GOC:txnOH]
GO:0000985
name: bacterial-type RNA polymerase core promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific DNA sequence in a bacterial-type RNA polymerase core promoter, the region composed of the transcription start site and binding sites for transcription factors of the basal transcription machinery, in order to modulate transcription. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0000986
name: bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a bacterial-type RNA polymerase." [GOC:txnOH]
GO:0000987
name: core promoter proximal region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to the core promoter. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH]
GO:0000988
name: protein binding transcription factor activity
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules), in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0000989
name: transcription factor binding transcription factor activity
def: "Interacting selectively and non-covalently with a specific transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0000990
name: core RNA polymerase binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0000991
name: core RNA polymerase II binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0000992
name: polymerase III regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is part of a regulatory region that controls transcription by RNA polymerase III. The transcribed region might be contain a single gene or a cistron containing multiple genes." [GOC:txnOH, PMID:12381659]
GO:0000993
name: RNA polymerase II core binding
def: "Interacting selectively and non-covalently with RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits." [GOC:txnOH]
GO:0000994
name: RNA polymerase III core binding
def: "Interacting selectively and non-covalently with RNA polymerase III core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of seventeen subunits." [GOC:txnOH]
GO:0000995
name: core RNA polymerase III binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH, PMID:12381659]
GO:0000996
name: core DNA-dependent RNA polymerase binding promoter specificity activity
def: "Interacting selectively and non-covalently with a DNA-dependent core RNA polymerase to form a holoenzyme complex and also, while present in the holoenzyme, interacting with promoter sequences in order to confer sequence specific recognition of promoter DNA sequence motifs." [GOC:txnOH]
GO:0000997
name: mitochondrial RNA polymerase core promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is part of the core promoter for the mitochondrial RNA polymerase." [GOC:txnOH]
GO:0001000
name: bacterial-type RNA polymerase core enzyme binding
def: "Interacting selectively and non-covalently with the bacterial-type RNA polymerase core enzyme, typically consisting of two alpha, one beta, one beta prime, and one omega subunit." [GOC:txnOH]
GO:0001001
name: mitochondrial single-subunit type RNA polymerase binding
def: "Interacting selectively and non-covalently with a single subunit mitochondrial RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6." [GOC:txnOH, PMID:20701995, PMID:2088182]
GO:0001002
name: RNA polymerase III type 1 promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 1 promoter that controls transcription by RNA polymerase III. Type 1 promoters are found in 5S rRNA genes, downstream of the transcription start site within the sequence of the mature RNA, and require TFIIIA for recognition." [GOC:txnOH, PMID:12381659]
GO:0001003
name: RNA polymerase III type 2 promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 2 promoter that controls transcription by RNA polymerase III. Type 2 promoters consist of an A box and a B box downstream of the transcription start site within the sequence within the sequence of the mature RNA. Type 2 promoters are found in many tRNA genes as well as in other small RNAs." [GOC:txnOH, PMID:12381659]
GO:0001004
name: RNA polymerase III promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence motif in an RNA polymerase III (Pol III) promoter in order to recruit the Pol III recruiting factor TFIIIB to the promoter." [GOC:txnOH, PMID:12381659]
GO:0001005
name: RNA polymerase III type 1 promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence motif in a type 1 RNA polymerase III (Pol III) promoter in order to recruit the Pol III recruiting factor TFIIIB to the promoter. For type 1 Pol III promoters, this requires both TFIIIA and TFIIIC." [GOC:txnOH, PMID:12381659]
GO:0001006
name: RNA polymerase III type 3 promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 3 promoter that controls transcription by RNA polymerase III (Pol III). A type 3 Pol III promoter is composed of elements upstream of the transcription start site, including a TATA box. The human U6 snRNA gene has a type 3 promoter. Type 3 Pol III promoters have not been observed in S. cerevisiae." [GOC:txnOH, PMID:12381659]
GO:0001007
name: RNA polymerase III transcription factor binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001008
name: RNA polymerase III type 2 promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence motif in a type 2 RNA polymerase III (Pol III) promoter in order to recruit TFIIIB to the promoter. For type 2 Pol III promoters, this requires TFIIIC activity." [GOC:txnOH, PMID:12381659]
GO:0001010
name: sequence-specific DNA binding transcription factor recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions." [GOC:txnOH]
GO:0001011
name: sequence-specific DNA binding RNA polymerase recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence and recruiting RNA polymerase to the DNA in order to form the preinitiation complex (PIC)." [GOC:txnOH]
GO:0001012
name: RNA polymerase II regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by RNA polymerase II. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH, GOC:vw]
GO:0001013
name: RNA polymerase I regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by RNA polymerase I. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH, GOC:vw]
GO:0001016
name: RNA polymerase III regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a gene by RNA polymerase III. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH, GOC:vw, PMID:12381659]
GO:0001017
name: bacterial-type RNA polymerase regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a gene or operon by a bacterial-type RNA polymerase. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH]
GO:0001018
name: mitochondrial RNA polymerase regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by mitochondrial RNA polymerase. Binding may occur as a high affinity and sequence specific interaction or as a lower affinity interaction observed once a factor has been recruited to the DNA by other factors." [GOC:txnOH, GOC:vw, PMID:20056105]
GO:0001019
name: plastid RNA polymerase regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by a plastid RNA polymerase. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH, GOC:vw]
GO:0001025
name: RNA polymerase III transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase III transcription factor, any protein required to initiate or regulate transcription by RNA polymerase III." [GOC:txnOH]
GO:0001026
name: TFIIIB-type transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way, Once recruited to an RNA polymerase III promoter by one or more other transcription factors, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH]
GO:0001027
name: RNA polymerase III type 1 promoter TFIIIB-type transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way. Once recruited to an RNA polymerase III type 1 promoter by TFIIIA-type and TFIIIC-type factors, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH, PMID:12381659]
GO:0001028
name: RNA polymerase III type 2 promoter TFIIIB-type transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way. Once recruited to an RNA polymerase III type 2 promoter by a TFIIIC-type factor, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH, PMID:12381659]
GO:0001029
name: RNA polymerase III type 3 promoter TFIIIB-type transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way. Once recruited to an RNA polymerase III type 3 promoter by SNAP-type and TFIIIC-type factors, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH, PMID:12381659]
GO:0001030
name: RNA polymerase III type 1 promoter DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that is a part of a type 1 promoter that controls transcription by RNA polymerase III. Type 1 promoters are found in 5S rRNA genes, downstream of the transcription start site within the sequence of the mature RNA, and require TFIIIA for recognition." [GOC:txnOH, PMID:12381659]
GO:0001031
name: RNA polymerase III type 2 promoter DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that is a part of a type 2 promoter that controls transcription by RNA polymerase III. Type 2 promoters consist of an A box and a B box downstream of the transcription start site within the sequence within the sequence of the mature RNA. Type 2 promoters are found in many tRNA genes as well as in other small RNAs." [GOC:txnOH, PMID:12381659]
GO:0001032
name: RNA polymerase III type 3 promoter DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that is a part of a type 3 promoter that controls transcription by RNA polymerase III. A type 3 Pol III promoter is composed of elements upstream of the transcription start site, including a TATA box. The human U6 snRNA gene has a type 3 promoter. Type 3 Pol III promoters have not been observed in S. cerevisiae." [GOC:txnOH, PMID:12381659]
GO:0001033
name: RNA polymerase III type 3 promoter TFIIIB recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence motif in a type 3 RNA polymerase III (Pol III) promoter in order to recruit TFIIIB to the promoter. For type 3 Pol III promoters, this requires SNAPc activity." [GOC:txnOH, PMID:12381659]
GO:0001034
name: sequence-specific DNA binding RNA polymerase III transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase III. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:12381659]
GO:0001037
name: RNA polymerase III hybrid type promoter DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that is a part of a hybrid promoter that controls transcription by RNA polymerase III (Pol III). A hybrid Pol III promoter contains both regulatory elements both upstream and downstream of the transcription initiation site. An example gene with such a promoter is the S. cerevisiae U6 gene." [GOC:txnOH, PMID:12381659]
GO:0001038
name: RNA polymerase III hybrid type promoter TFIIIB recruiting transcription factor activity
def: "Once recruited to an RNA polymerase III hybrid type promoter by sequence-specific DNA binding transcription factors, direct DNA binding, or a combination of both, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH, PMID:12381659]
GO:0001039
name: RNA polymerase III hybrid type promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a part of a hybrid type promoter that controls transcription by RNA polymerase III (Pol III). A hybrid Pol III promoter contains both regulatory elements both upstream and downstream of the transcription initiation site. An example gene with such a promoter is the S. cerevisiae U6 gene." [GOC:txnOH, PMID:12381659]
GO:0001040
name: RNA polymerase III hybrid type promoter TFIIIB-type transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase III (Pol III) complex, typically composed of seventeen subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way. Once recruited to an RNA polymerase III hybrid type promoter, binds to DNA, recruits RNA polymerase III and facilitates the transition from the closed to the open complex." [GOC:txnOH, PMID:12381659]
GO:0001042
name: RNA polymerase I core binding
def: "Interacting selectively and non-covalently with RNA polymerase I core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of seventeen subunits." [GOC:txnOH]
GO:0001044
name: mitochondrial RNA polymerase regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific sequence of DNA that controls the transcription of a region of DNA by mitochondrial RNA polymerase." [GOC:txnOH]
GO:0001045
name: mitochondrial RNA polymerase core promoter proximal region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a mitochondrial RNA polymerase." [GOC:txnOH, PMID:20670382]
GO:0001046
name: core promoter sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is part of a core promoter region composed of the transcription start site and binding sites for the basal transcription machinery. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH]
GO:0001047
name: core promoter binding
def: "Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for the basal transcription machinery. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:txnOH]
GO:0001048
name: RNA polymerase IV core binding
def: "Interacting selectively and non-covalently with RNA polymerase IV core enzyme, a multisubunit eukaryotic nuclear RNA polymerase found in plants and involved in siRNA production." [GOC:txnOH, PMID:19110459]
GO:0001049
name: RNA polymerase V core binding
def: "Interacting selectively and non-covalently with RNA polymerase V core enzyme, a multisubunit eukaryotic nuclear RNA polymerase found in plants and involved in production of noncoding transcripts at target loci for silencing." [GOC:txnOH, PMID:19110459]
GO:0001050
name: single-subunit type RNA polymerase binding
def: "Interacting selectively and non-covalently with a single subunit RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6." [GOC:txnOH, PMID:20701995]
GO:0001051
name: plastid single-subunit type RNA polymerase binding
def: "Interacting selectively and non-covalently with a single subunit plastid RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6." [GOC:txnOH, PMID:20701995]
GO:0001052
name: plastid PEP RNA polymerase core enzyme binding
def: "Interacting selectively and non-covalently with the bacterial-type plastid PEP RNA polymerase core enzyme, typically consisting of two alpha, one beta, one beta prime, and one double prime subunit." [GOC:txnOH, PMID:20701995]
GO:0001053
name: plastid sigma factor activity
def: "Interacting selectively and non-covalently with a plastid PEP core RNA polymerase to form a holoenzyme complex and also, while present in the holoenzyme, interacting with promoter sequences in order to confer sequence specific recognition of plastid PEP core promoter DNA sequence motifs." [GOC:txnOH, PMID:20701995]
GO:0001054
name: RNA polymerase I activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase I specific promoter to direct initiation and catalyzes DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001055
name: RNA polymerase II activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001056
name: RNA polymerase III activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase III specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001057
name: RNA polymerase IV activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase IV specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001058
name: RNA polymerase V activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase V specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001061
name: bacterial-type RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a bacterial-type specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001062
name: plastid PEP-A RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a plastid PEP-A RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH, PMID:20701995]
GO:0001063
name: plastid PEP-B RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a plastid PEP-B RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH, PMID:20701995]
GO:0001064
name: single subunit type RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single-subunit-type RNA polymerase-specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001065
name: mitochondrial single subunit type RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single-subunit-type mitochondrial RNA polymerase-specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH]
GO:0001066
name: plastid single subunit type RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single subunit type plastid RNA polymerase specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [GOC:txnOH, PMID:20701995]
GO:0001067
name: regulatory region nucleic acid binding
def: "Interacting selectively and non-covalently with a nucleic acid region that regulates a nucleic acid-based process. Such processes include transcription, DNA replication, and DNA repair." [GOC:txnOH]
GO:0001068
name: transcription regulatory region RNA binding
def: "Interacting selectively and non-covalently with a RNA region within the transcript that regulates the transcription of a region of DNA, which may be a gene, cistron, or operon." [GOC:txnOH]
GO:0001069
name: regulatory region RNA binding
def: "Interacting selectively and non-covalently with a RNA region that regulates a nucleic acid-based process. Such processes include transcription, DNA replication, and DNA repair." [GOC:txnOH]
GO:0001070
name: RNA binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA sequence in order to modulate transcription. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:8332211]
GO:0001071
name: nucleic acid binding transcription factor activity
def: "Interacting selectively and non-covalently with a DNA or RNA sequence in order to modulate transcription. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0001072
name: RNA binding transcription antitermination factor activity
def: "Binds to RNA, typically within the nascent RNA transcript, to promote readthrough of a transcription termination site and thus extending the length of the RNA transcript produced. Examples of antitermination factors which bind the nascent RNA include the lambda N protein and the HIV-1 tat protein." [GOC:txnOH, PMID:8332211]
GO:0001073
name: DNA binding transcription antitermination factor activity
def: "Binds to DNA, typically within region of the promoter and transcribed region, to promote readthrough of a transcription termination site and thus extending the length of the RNA transcript produced. Examples of antitermination factors which bind DNA include the lambda Q protein." [GOC:txnOH, PMID:8332211]
GO:0001074
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in preinitiation complex assembly
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) in order to promote assembly of the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription from an RNA polymerase II promoter." [GOC:txnOH]
GO:0001075
name: RNA polymerase II core promoter sequence-specific DNA binding transcription factor activity involved in preinitiation complex assembly
def: "Interacting selectively and non-covalently with a specific DNA sequence in an RNA polymerase II (Pol II) core promoter, the region composed of the transcription start site and binding sites for transcription factors of the Pol II basal transcription machinery, in order to promote assembly of the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription from an RNA polymerase II promoter." [GOC:txnOH]
GO:0001076
name: RNA polymerase II transcription factor binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001077
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:txnOH]
GO:0001078
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) in order to stop, prevent, or reduce the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:txnOH]
GO:0001082
name: RNA polymerase I transcription factor binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase I transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001083
name: RNA polymerase II basal transcription factor binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001084
name: TFIID-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIID class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001085
name: RNA polymerase II transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription factor, any protein required to initiate or regulate transcription by RNA polymerase II." [GOC:txnOH]
GO:0001086
name: TFIIA-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIA class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001087
name: TFIIB-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIB class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001088
name: TFIIE-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIE class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001089
name: TFIIF-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIF class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001090
name: TFIIH-class binding transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIH class in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH]
GO:0001091
name: RNA polymerase II basal transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor, any of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:txnOH, PMID:16858867]
GO:0001092
name: TFIIA-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIA class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001093
name: TFIIB-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIB class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001094
name: TFIID-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIID class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001095
name: TFIIE-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIE class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001096
name: TFIIF-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIF class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001097
name: TFIIH-class transcription factor binding
def: "Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor of the TFIIH class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor." [GOC:krc, PMID:16858867]
GO:0001098
name: basal transcription machinery binding
def: "Interacting selectively and non-covalently with the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
GO:0001099
name: basal RNA polymerase II transcription machinery binding
def: "Interacting selectively and non-covalently with the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase." [GOC:txnOH]
GO:0001102
name: RNA polymerase II activating transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription activating factor, a protein involved in positive regulation of transcription." [GOC:txnOH]
GO:0001103
name: RNA polymerase II repressing transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription repressing factor, a protein involved in negative regulation of transcription." [GOC:txnOH]
GO:0001104
name: RNA polymerase II transcription cofactor activity
def: "Interacting selectively and non-covalently with an RNA polymerase II (RNAP II) regulatory transcription factor and also with the RNAP II basal transcription machinery in order to modulate transcription. Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between regulatory transcription factors and the basal RNAP II transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0001105
name: RNA polymerase II transcription coactivator activity
def: "Interacting selectively and non-covalently with an RNA polymerase II (RNAP II) regulatory transcription factor and also with the RNAP II basal transcription machinery in order to increase the frequency, rate or extent of transcription. Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between activating transcription factors and the basal RNAP II transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0001106
name: RNA polymerase II transcription corepressor activity
def: "Interacting selectively and non-covalently with an RNA polymerase II repressing transcription factor and also with the RNA polymerase II basal transcription machinery in order to stop, prevent, or reduce the frequency, rate or extent of transcription. Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between repressive transcription factors and the basal transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0001108
name: bacterial-type RNA polymerase holo enzyme binding
def: "Interacting selectively and non-covalently with the basal transcription machinery which is composed of a bacterial-type RNA polymerase core enzyme and a sigma factor, the minimal set of factors required for formation of the preinitiation complex (PIC) by a bacterial-type RNA polymerase." [GOC:txnOH]
GO:0001128
name: RNA polymerase II transcription coactivator activity involved in preinitiation complex assembly
def: "Interacting selectively and non-covalently with an RNA polymerase II (RNAP II) regulatory transcription factor and also with the RNAP II basal transcription machinery in order to increase the frequency, rate or stability of the aggregation, arrangement and bonding together of proteins on RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC). Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between activating transcription factors and the basal RNAP II transcription machinery." [GOC:txnOH, PMID:16858867]
GO:0001129
name: TBP-class protein binding RNA polymerase II transcription factor activity involved in preinitiation complex assembly
def: "Interacting selectively and non-covalently with a member of the class of TATA-binding proteins (TBP), including any of the TBP-related factors (TRFs), to facilitate the aggregation, arrangement and bonding together of proteins on RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
GO:0001130
name: sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0001131
name: bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a bacterial-type RNA polymerase in order to modulate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0001132
name: TBP-class protein binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a member of the class of TATA-binding proteins (TBP), including any of the TBP-related factors (TRFs), in order to modulate transcription. The transcription factor may or may not also interact selectively with DNA as well." [GOC:txnOH, PMID:16858867]
GO:0001133
name: sequence-specific transcription regulatory region DNA binding RNA polymerase II transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls transcription of that section of the DNA by RNA polymerase II and recruiting another transcription factor to the DNA in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:16858867]
GO:0001134
name: transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a specific transcription factor, which may be a single protein or a complex, and with another protein, macromolecule, or complex, recruiting that specific transcription factor to the transcription machinery complex and thus permitting those molecules to function in a coordinated way, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH, PMID:16858867]
GO:0001135
name: RNA polymerase II transcription factor recruiting transcription factor activity
def: "The function of binding to an RNA polymerase II (RNAP II) transcription factor and recruiting it to the transcription machinery complex in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:16858867]
GO:0001136
name: TFIIE-class transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II (RNAP II) transcription factor of the TFIIE-class in order to recruit it to the RNAP II preinitiation complex (PIC) in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:16858867]
GO:0001137
name: TFIIF-class transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II (RNAP II) transcription factor of the TFIIF-class in order to recruit it to the RNAP II preinitiation complex (PIC) in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:16858867]
GO:0001138
name: TFIIH-class transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with a basal RNA polymerase II (RNAP II) transcription factor of the TFIIH-class in order to recruit it to the RNAP II preinitiation complex (PIC) in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:16858867]
GO:0001139
name: core RNA polymerase II recruiting transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase." [GOC:txnOH, PMID:16858867]
GO:0001140
name: bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a bacterial-type RNA polymerase in order to modulate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:20491932, PMID:20629756]
GO:0001141
name: bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a bacterial-type RNA polymerase in order to stop, prevent, or reduce the frequency, rate or extent of transcription from a bacterial-type RNA polymerase promoter." [GOC:txnOH, PMID:20491932, PMID:20629756]
GO:0001142
name: sequence-specific DNA binding mitochondrial RNA polymerase transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by mitochondrial RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:18391175]
GO:0001143
name: mitochondrial RNA polymerase core promoter sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence that is part of the core promoter for mitochondrial RNA polymerase in order to modulate transcription by mitochondrial RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:18391175]
GO:0001144
name: mitochondrial RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for a mitochondrial RNA polymerase in order to modulate transcription by mitochondrial RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH, PMID:18391175]
GO:0001145
name: mitochondrial RNA polymerase termination site sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a terminator for mitochondrial RNA polymerase." [GOC:txnOH, PMID:18391175]
GO:0001146
name: mitochondrial RNA polymerase terminator site sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is a terminator for mitochondrial RNA polymerase in order to promote transcription termination by mitochondrial RNA polymerase." [GOC:txnOH, PMID:18391175]
GO:0001147
name: transcription termination site sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that promotes termination by RNA polymerase. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH, PMID:18280161, PMID:18391175]
GO:0001148
name: bacterial-type RNA polymerase termination site sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is a terminator for bacterial-type RNA polymerase." [GOC:txnOH, PMID:18280161]
GO:0001149
name: bacterial-type RNA polymerase termination site sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is a termination site for bacterial-type RNA polymerase in order to promote transcription termination by bacterial-type RNA polymerase." [GOC:txnOH, PMID:18280161]
GO:0001150
name: bacterial-type RNA polymerase enhancer sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a sequence of DNA that is an enhancer region that helps activate transcription of a gene or operon by a bacterial-type RNA polymerase. Enhancers for sigma-54 holoenzymes are typically located 80 to 150 base pairs upstream from the transcription start site, although they can be further away or may be downstream of the promoter. Some transcription units dependent on sigma-70 holoenzymes may also include enhancer sequences." [GOC:txnOH, PMID:11282468, PMID:20629756]
GO:0001151
name: bacterial-type RNA polymerase transcription enhancer sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is a bacterial-type enhancer region in order to activate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex. Enhancers for sigma-54 holoenzymes are typically located 80 to 150 base pairs upstream from the transcription start site. Some transcription units dependent on sigma-70 holoenzymes may also include enhancer sequences." [GOC:txnOH, PMID:20629756]
GO:0001152
name: RNA polymerase III type 1 promoter sequence-specific DNA binding TFIIIC recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence motif in a type 1 RNA polymerase III (Pol III) promoter in order to recruit the transcription factor TFIIIC to the promoter." [GOC:txnOH, PMID:12381659]
GO:0001153
name: RNA polymerase III transcription factor recruiting transcription factor activity
def: "The function of binding to an RNA polymerase III (RNAP III) transcription factor and recruiting it to the transcription machinery complex in order to modulate transcription by RNAP III." [GOC:txnOH]
GO:0001154
name: TFIIIB-class transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase III transcription factor of the TFIIIB class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase III." [GOC:txnOH, PMID:12381659]
GO:0001155
name: TFIIIA-class transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase III transcription factor of the TFIIIA class, one of the factors involved in formation of the preinitiation complex (PIC) at type 1 promoters (5S genes) by RNA polymerase III." [GOC:txnOH, PMID:12381659]
GO:0001156
name: TFIIIC-class transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase III transcription factor of the TFIIIC class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase III." [GOC:txnOH, PMID:12381659]
GO:0001157
name: RNA polymerase III type 1 promoter TFIIIB recruiting transcription factor activity
def: "The function of binding to a specific DNA sequence motif in a type 1 RNA polymerase III (Pol III) promoter in order to recruit the Pol III recruiting factor TFIIIB to the promoter. For type 1 Pol III promoters, this requires both TFIIIA and TFIIIC." [GOC:txnOH, PMID:12381659]
GO:0001158
name: enhancer sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific sequence of DNA that is part of an enhancer, a transcription regulatory region that is somewhat distal from the core promoter and which enhances transcription from that promoter." [GOC:txnOH]
GO:0001159
name: core promoter proximal region DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that is in cis with and relatively close to the core promoter. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH]
GO:0001160
name: transcription termination site DNA binding
def: "Interacting selectively and non-covalently with a region of DNA that promotes termination by RNA polymerase. The transcribed region might be described as a gene, cistron, or operon." [GOC:txnOH]
GO:0001161
name: intronic transcription regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with an intronic DNA sequence that regulates the transcription of the transcript it is contained within." [GOC:txnOH]
GO:0001162
name: RNA polymerase II intronic transcription regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with an RNA polymerase II intronic DNA sequence that regulates the transcription of the transcript it is contained within." [GOC:txnOH]
GO:0001163
name: RNA polymerase I regulatory region sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls the transcription of a gene or cistron by RNA polymerase I." [GOC:txnOH]
GO:0001164
name: RNA polymerase I CORE element sequence-specific DNA binding
def: "Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for transcription factors of the RNA polymerase I transcription machinery. This site is often referred to as the CORE element. In mammalian cells, the CORE element functions in conjunction with the Upstream Control Element (UCE), while in fungi, protozoa, and plants, the CORE element functions without a UCE." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001165
name: RNA polymerase I upstream control element sequence-specific DNA binding
def: "Interacting selectively and non-covalently with the upstream control element (UCE, or alternately referred to as the upstream element), a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase I." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001166
name: RNA polymerase I enhancer sequence-specific DNA binding
def: "Interacting selectively and non-covalently with a RNA polymerase I (Pol I) enhancer. In mammalian cells, enhancers are distal sequences that increase the utilization of promoters, and can function in either orientation and in any location (upstream or downstream) relative to the core promoter." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001167
name: sequence-specific DNA binding RNA polymerase I transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase I. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:txnOH]
GO:0001168
name: RNA polymerase I upstream control element sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with the upstream control element (UCE, or alternately referred to as the upstream element, UE), a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase I in order to modulate transcription by RNA polymerase I." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001169
name: RNA polymerase I CORE element sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with CORE element, a regulatory region composed of the transcription start site and binding sites for transcription factors of the RNA polymerase I transcription machinery in order to modulate transcription by RNA polymerase I." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001170
name: RNA polymerase I enhancer sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a RNA polymerase I (Pol I) enhancer in order to modulate transcription by RNA polymerase I." [GOC:txnOH, PMID:12865296, PMID:14969726, PMID:8057832]
GO:0001179
name: RNA polymerase I transcription factor binding
def: "Interacting selectively and non-covalently with an RNA polymerase I transcription factor, any protein required to initiate or regulate transcription by RNA polymerase I." [GOC:txnOH]
GO:0001181
name: core RNA polymerase I binding transcription factor activity
def: "Interacting selectively and non-covalently with an RNA polymerase I (RNAP I) complex, typically composed of fourteen subunits, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001186
name: RNA polymerase I transcription factor recruiting transcription factor activity
def: "The function of binding to an RNA polymerase I (RNAP I) transcription factor and recruiting it to the transcription machinery complex in order to modulate transcription by RNAP I." [GOC:txnOH, PMID:12381659, PMID:14969726, PMID:8057832]
GO:0001187
name: RNA polymerase I CORE element sequence-specific DNA binding transcription factor recruiting transcription factor activity
def: "Interacting selectively and non-covalently with the CORE element, a regulatory region composed of the transcription start site and binding sites for transcription factors of the RNA polymerase I transcription machinery and also binding to an RNA polymerase I (RNAP I) transcription factor to recruit it to the transcription machinery complex in order to modulate transcription by RNA polymerase I." [GOC:txnOH, PMID:12381659, PMID:14969726, PMID:8057832]
GO:0001190
name: RNA polymerase II transcription factor binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to increase the frequency, rate or extent of transcription from an RNA polymerase II promoter. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH]
GO:0001191
name: RNA polymerase II transcription factor binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to stop, prevent, or reduce the frequency, rate or extent of transcription from an RNA polymerase II promoter. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well." [GOC:txnOH, PMID:9811836]
GO:0001199
name: metal ion regulated sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to modulate transcription by an RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:22084422, PMID:2674688]
GO:0001200
name: metal ion regulated sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to modulate transcription by RNA polymerase II. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:2674688]
GO:0001201
name: metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor that binds the core promoter proximal region in order to modulate transcription by RNA polymerase II. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:22084422, PMID:2674688]
GO:0001202
name: copper ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with copper(I) (Cu+) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of copper(I) (Cu+), such a way that copper ion binding regulates the activity of the transcription factor, perhaps by regulating the ability to interact selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II), or by regulating some other function of the transcription factor, in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:22084422, PMID:2674688]
GO:0001203
name: zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with zinc (Zn) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of zinc (Zn), in such a way that zinc ion binding regulates the activity of the transcription factor, perhaps by regulating the ability to interact selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II), or by regulating some other function of the transcription factor, in order to modulate transcription by RNAP II." [GOC:txnOH, PMID:22084422]
GO:0001204
name: metal ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity
def: "Interacting selectively and non-covalently with a specific bacterial-type transcription regulatory DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to modulate transcription by a bacterial-type RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:22084422, PMID:2674688]
GO:0001205
name: RNA polymerase II distal enhancer sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in a distal enhancer region for RNA polymerase II (RNAP II) in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:txnOH]
GO:0001206
name: RNA polymerase II distal enhancer sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in a distal enhancer region for RNA polymerase II (RNAP II) in order to stop, prevent, or reduce the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:txnOH]
GO:0001209
name: metal ion regulated sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a specific DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to activate or increase the frequency, rate or extent of cellular DNA-dependent transcription by an RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:22084422, PMID:2674688]
GO:0001210
name: metal ion regulated sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a specific DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to stop, prevent, or reduce the frequency, rate or extent of cellular DNA-dependent transcription by an RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:22084422, PMID:2674688]
GO:0001211
name: copper ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with copper(I) (Cu+) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of copper(I) (Cu+), such a way that copper ion binding regulates the activity of the transcription factor in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:txnOH, PMID:2674688]
GO:0001212
name: zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with zinc (Zn) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of zinc (Zn), such a way that copper ion binding regulates the activity of the transcription factor in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:txnOH]
GO:0001213
name: metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of metal ion, such a way that metal ion binding regulates the activity of the transcription factor in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:txnOH, PMID:22084422, PMID:2674688]
GO:0001214
name: metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II (RNAP II) and interacting selectively and non-covalently with metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of metal ion, such a way that metal ion binding regulates the activity of the transcription factor in order to stop, prevent, or reduce the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:txnOH, PMID:22084422, PMID:2674688]
GO:0001215
name: metal ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a specific bacterial-type transcription regulatory DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to activate or increase the frequency, rate or extent of cellular DNA-dependent transcription by a bacterial-type RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:2674688]
GO:0001216
name: sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:krc]
GO:0001217
name: sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by bacterial-type RNA polymerase. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:krc]
GO:0001218
name: metal ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a specific bacterial-type transcription regulatory DNA sequence and interacting selectively and non-covalently with any metal ion(s) and responding, e.g. by conformational change, to changes in the cellular level of those metal ions, in such a way that metal ion binding regulates the activity of the transcription factor in order to stop, prevent, or reduce the frequency, rate or extent of cellular DNA-dependent transcription by a bacterial-type RNA polymerase. Specific mechanisms include increasing or decreasing the ability of the transcription factor to interact selectively and non-covalently with a specific DNA sequence (for example S. cerevisiae Cup2p or the prokaryotic SmtB/ArsR family, changing the conformation of transcription factor bound DNA between the apo- and metal-bound forms of the transcription factor. Additional mechanisms may exist." [GOC:txnOH, PMID:12829264, PMID:12829265, PMID:2674688]
GO:0001219
name: copper ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a specific bacterial-type transcription regulatory DNA sequence and interacting selectively and non-covalently with copper(I) (Cu+) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of copper(I) (Cu+), in such a way that copper ion binding regulates the activity of the transcription factor in order to stop, prevent, or reduce the frequency, rate or extent of cellular DNA-dependent transcription by a bacterial-type RNA polymerase." [GOC:krc, PMID:12829264, PMID:12829265, PMID:19928961]
GO:0001220
name: cadmium ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a specific bacterial-type transcription regulatory DNA sequence and interacting selectively and non-covalently with cadmium (Cd) ion(s) and responding, e.g. by conformational change, to changes in the cellular level of cadmium (Cd), in such a way that cadmium (Cd) ion binding regulates the activity of the transcription factor in order to stop, prevent, or reduce the frequency, rate or extent of cellular DNA-dependent transcription by a bacterial-type RNA polymerase." [GOC:krc, PMID:12829264, PMID:12829265, PMID:19456862]
GO:0001221
name: transcription cofactor binding
def: "Interacting selectively and non-covalently with a transcription cofactor, any protein involved in regulation of transcription via protein-protein interactions with transcription factors and other transcription regulatory proteins. Cofactors do not bind DNA directly, but rather mediate protein-protein interactions between regulatory transcription factors and the basal transcription machinery." [GOC:krc]
GO:0001222
name: transcription corepressor binding
def: "Interacting selectively and non-covalently with a transcription corepressor, any protein involved in negative regulation of transcription via protein-protein interactions with transcription factors and other proteins that negatively regulate transcription. Transcription corepressors do not bind DNA directly, but rather mediate protein-protein interactions between repressing transcription factors and the basal transcription machinery." [GOC:krc]
GO:0001223
name: transcription coactivator binding
def: "Interacting selectively and non-covalently with a transcription coactivator, any protein involved in positive regulation of transcription via protein-protein interactions with transcription factors and other proteins that positively regulate transcription. Transcription coactivators do not bind DNA directly, but rather mediate protein-protein interactions between activating transcription factors and the basal transcription machinery." [GOC:krc]
GO:0001224
name: RNA polymerase II transcription cofactor binding
def: "Interacting selectively and non-covalently with a transcription cofactor for RNA polymerase II, any protein involved in regulation of transcription via protein-protein interactions with RNA polymerase II transcription factors and other transcription regulatory proteins. Cofactors do not bind DNA directly, but rather mediate protein-protein interactions between regulatory transcription factors and the basal transcription machinery of RNA polymerase II." [GOC:krc]
GO:0001225
name: RNA polymerase II transcription coactivator binding
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription coactivator, any protein involved in positive regulation of transcription of RNA polymerase II via protein-protein interactions with transcription factors and other proteins that positively regulate transcription. Transcription coactivators do not bind DNA directly, but rather mediate protein-protein interactions between activating transcription factors and the basal transcription machinery of RNA polymerase II." [GOC:krc]
GO:0001226
name: RNA polymerase II transcription corepressor binding
def: "Interacting selectively and non-covalently with an RNA polymerase II transcription corepressor, any protein involved in negative regulation of transcription by RNA polymerase II via protein-protein interactions with transcription factors and other proteins that negatively regulate transcription. Transcription corepressors do not bind DNA directly, but rather mediate protein-protein interactions between repressing transcription factors and the basal transcription machinery of RNA polymerase II." [GOC:krc]
GO:0001227
name: RNA polymerase II transcription regulatory region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in the regulatory region for RNA polymerase II (RNAP II) in order to stop, prevent, or reduce the frequency, rate or extent of transcription from an RNA polymerase II promoter." [GOC:krc]
GO:0001228
name: RNA polymerase II transcription regulatory region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription
def: "Interacting selectively and non-covalently with a sequence of DNA that is in the transcription regulatory region for RNA polymerase II (RNAP II) in order to activate or increase the frequency, rate or extent of transcription from the RNAP II promoter." [GOC:krc]
GO:0001406
name: glycerophosphodiester transmembrane transporter activity
def: "Catalysis of the transfer of glycerophosphodiesters from one side of a membrane to the other. Glycerophosphodiesters are small molecules composed of glycerol-3-phosphate and an alcohol, for example, glycerophosphoinositol." [GOC:mcc, PMID:12912892]
GO:0001409
name: guanine nucleotide transmembrane transporter activity
def: "Catalysis of the transfer of guanine nucleotides (GMP, GDP, and GTP) from one side of the membrane to the other." [GOC:mcc]
GO:0001509
name: legumain activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins and small-molecule substrates at Asn-Xaa bonds." [EC:3.4.22.34]
GO:0001511
name: fibrillin
def: "OBSOLETE. Large glycoprotein that is a calcium binding component of connective tissue microfibrils containing 34 six-cysteine (EGF-like) repeats and five eight-cysteine (TGFbeta-1 binding protein-like) repeats. Defects associated with Marfan syndrome." [ISBN:0198506732]
GO:0001512
name: dihydronicotinamide riboside quinone reductase activity
def: "Catalysis of the reaction: 1-(beta-D-ribofuranosyl)-1,4-dihydronicotinamide + a quinone = 1-(beta-D-ribofuranosyl)nicotinamide + a hydroquinone." [EC:1.10.99.2]
GO:0001515
name: opioid peptide activity
def: "Naturally occurring peptide that is an opioid (any non-alkaloid having an opiate-like effect that can be reversed by naloxone or other recognized morphine antagonist). These include Leu- and Met-enkephalin, dynorphin and neoendorphin, alpha, beta, gamma and delta endorphins formed from beta-lipotropin, various pronase-resistant peptides such as beta casamorphin, and other peptides whose opiate-like action seems to be indirect." [ISBN:0198506732]
GO:0001517
name: N-acetylglucosamine 6-O-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-glucosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-glucosamine 6-sulfate." [GOC:ai, GOC:hjd]
GO:0001524
name: globin
def: "OBSOLETE. The colorless and basic protein moiety of hemoglobin and myoglobins." [GOC:ai, ISBN:0198506732]
GO:0001528
name: elastin
def: "OBSOLETE. A major structural protein of mammalian connective tissues; composed of one third glycine, and also rich in proline, alanine, and valine. Chains are cross-linked together via lysine residues." [ISBN:0198506732]
GO:0001530
name: lipopolysaccharide binding
def: "Interacting selectively and non-covalently with lipopolysaccharide." [PMID:11079463]
GO:0001531
name: interleukin-21 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-21 receptor." [GOC:ai]
GO:0001532
name: interleukin-21 receptor activity
def: "Combining with interleukin-21 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0001537
name: N-acetylgalactosamine 4-O-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-galactosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-galactosamine 4-sulfate." [EC:2.8.2.-, GOC:ai]
GO:0001540
name: beta-amyloid binding
def: "Interacting selectively and non-covalently with beta-amyloid peptide/protein and/or its precursor." [GOC:hjd]
GO:0001565
name: phorbol ester receptor activity
def: "Combining with a phorbol ester and transmitting the signal to initiate a change in cell activity." [GOC:ai, GOC:signaling, PMID:10506570]
GO:0001566
name: non-kinase phorbol ester receptor activity
def: "Combining with a phorbol ester and transmitting the signal by a mechanism independent of kinase activity." [PMID:10506570]
GO:0001567
name: cholesterol 25-hydroxylase activity
def: "Catalysis of the reaction: AH(2) + cholesterol + O(2) = 25-hydroxycholesterol + A + H(2)O." [EC:1.14.99.38, RHEA:21107]
GO:0001571
name: non-tyrosine kinase fibroblast growth factor receptor activity
def: "Combining with fibroblast growth factor (FGF) and transmitting the signal from one side of the membrane to the other by a mechanism independent of tyrosine kinase activity." [GOC:signaling, PMID:11418238]
GO:0001577
name: galectin
def: "OBSOLETE. A lectin that exhibits calcium independent binding of beta-galactoside sugars." [PMID:9786891]
GO:0001584
name: rhodopsin-like receptor activity
def: "OBSOLETE. A G-protein coupled receptor that is structurally/functionally related to the rhodopsin receptor." [GOC:dph, GOC:mah, GOC:tb, IUPHAR_GPCR:1505]
GO:0001586
name: Gi/o-coupled serotonin receptor activity
def: "Combining with serotonin and transmitting the signal across the membrane by activation of the Gi/o subunit of an associated cytoplasmic heterotrimeric G protein complex. The Gi/o subunit subsequently inhibits adenylate cyclase and results in a decrease in cyclic AMP (cAMP) levels." [GOC:mah, PMID:18571247]
GO:0001587
name: Gq/11-coupled serotonin receptor activity
def: "Combining with serotonin and transmitting the signal across the membrane by activation of the Gq/11 subunit of an associated cytoplasmic heterotrimeric G protein complex. The Gq/11 subunit subsequently activates phospholipase C and results in an increase in inositol triphosphate (IP3) levels." [GOC:bf, GOC:mah, PMID:18571247, PMID:18703043]
GO:0001588
name: dopamine neurotransmitter receptor activity, coupled via Gs
def: "Combining with the neurotransmitter dopamine and activating adenylate cyclase via coupling to Gs to initiate a change in cell activity." [GOC:mah, ISBN:0953351033, IUPHAR_RECEPTOR:2252, IUPHAR_RECEPTOR:2260]
GO:0001591
name: dopamine neurotransmitter receptor activity, coupled via Gi/Go
def: "Combining with the neurotransmitter dopamine and activating adenylate cyclase via coupling to Gi/Go to initiate a change in cell activity." [GOC:mah, ISBN:0953351033, IUPHAR_RECEPTOR:2254, IUPHAR_RECEPTOR:2256, IUPHAR_RECEPTOR:2258]
GO:0001594
name: trace-amine receptor activity
def: "Combining with a trace amine to initiate a change in cell activity. Trace amines are biogenic amines that are synthesized from aromatic amino acids and are substrates for monoamine oxidase, and are therefore detectable only at trace levels in mammals." [GOC:mah, PMID:19325074]
GO:0001595
name: angiotensin receptor activity
def: "Combining with angiotensin to initiate a change in cell activity." [GOC:ai]
GO:0001596
name: angiotensin type I receptor activity
def: "An angiotensin receptor activity that acts via Gq-mediated activation of phospholipase C followed by phosphoinositide hydrolysis and Ca2+ signaling, and may act via additional signaling mechanisms." [GOC:mah, PMID:10977869]
GO:0001597
name: apelin-like receptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0001598
name: chemokine receptor-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001601
name: peptide YY receptor activity
def: "Combining with gut peptide YY to initiate a change in cell activity." [PMID:9315606]
GO:0001602
name: pancreatic polypeptide receptor activity
def: "Combining with pancreatic polypeptide PP to initiate a change in cell activity." [PMID:9315606]
GO:0001603
name: vasopressin-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:curators]
GO:0001604
name: urotensin II receptor activity
def: "Combining with urotensin II to initiate a change in cell activity." [GOC:mah, PMID:15102493]
GO:0001605
name: adrenomedullin receptor activity
def: "Combining with adrenomedullin to initiate a change in cell activity." [GOC:ai]
GO:0001606
name: GPR37/endothelin B-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001607
name: neuromedin U receptor activity
def: "Combining with neuromedin U to initiate a change in cell activity." [GOC:ai]
GO:0001608
name: G-protein coupled nucleotide receptor activity
def: "Combining with a nucleotide and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:dph, IUPHAR_GPCR:1294]
GO:0001609
name: G-protein coupled adenosine receptor activity
def: "Combining with adenosine and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:mah, PMID:9755289]
GO:0001614
name: purinergic nucleotide receptor activity
def: "Combining with a purine nucleotide and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0001615
name: thyrotropin releasing hormone and secretagogue-like receptors activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001616
name: growth hormone secretagogue receptor activity
def: "Combining with ghrelin to initiate a change in cell activity." [GOC:mah, PMID:17983853]
GO:0001617
name: growth hormone secretagogue-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001618
name: virus receptor activity
def: "Combining with a virus component and mediating entry of the virus into the cell." [GOC:bf, GOC:dph, PMID:7621403, UniProtKB-KW:KW-1183]
GO:0001619
name: lysosphingolipid and lysophosphatidic acid receptor activity
def: "Combining with lysosphingolipid or lysophosphatidic acid to initiate a change in cell activity." [GOC:dph]
GO:0001621
name: ADP receptor activity
def: "Combining with ADP and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling, PMID:11196645]
GO:0001626
name: nociceptin receptor activity
def: "Combining with the peptide nociceptin, and transmitting the signal across the membrane by activating an associated G-protein." [GOC:bf, GOC:mah, PMID:18670432]
GO:0001627
name: leucine-rich G-protein receptor-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001628
name: gastropyloric receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001629
name: G-protein receptor 45-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001630
name: GP40-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001631
name: cysteinyl leukotriene receptor activity
def: "Combining with a cysteinyl leukotriene to initiate a change in cell activity. Cysteinyl leukotrienes are leukotrienes that contain a peptide group based on cysteine." [GOC:ai, ISBN:0198506732]
GO:0001632
name: leukotriene B4 receptor activity
def: "Combining with leukotriene B4, LTB4, to initiate a change in cell activity. Leukotriene B4 is also known as (6Z, 8E, 10E, 14Z)-(5S, 12R)-5,12-dihydroxyicosa-6,8,10,14-tetraen-1-oate." [GOC:ai, ISBN:0198506732]
GO:0001633
name: secretin-like receptor activity
def: "OBSOLETE. A G-protein coupled receptor that is structurally/functionally related to the secretin receptor." [GOC:mah, IUPHAR_GPCR:1506]
GO:0001634
name: pituitary adenylate cyclase-activating polypeptide receptor activity
def: "A G-protein coupled receptor that interacts with pituitary adenylate cyclase-activating polypeptide." [GOC:dph, GOC:tb]
GO:0001635
name: calcitonin gene-related peptide receptor activity
def: "Combining with a calcitonin gene-related polypeptide (CGRP) to initiate a change in cell activity." [GOC:mah, PMID:12037140]
GO:0001636
name: corticotrophin-releasing factor gastric inhibitory peptide-like receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0001637
name: G-protein coupled chemoattractant receptor activity
def: "Combining with a chemoattractant and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:mah]
GO:0001639
name: PLC activating G-protein coupled glutamate receptor activity
def: "A G-protein coupled receptor that binds glutamate and is linked to the inositol 1,4,5-trisphosphate/calcium signaling system." [PMID:9016303]
GO:0001640
name: adenylate cyclase inhibiting G-protein coupled glutamate receptor activity
def: "Combining with glutamate and transmitting the signal across the membrane by activating the alpha-subunit of an associated heterotrimeric G-protein complex to inhibit downstream adenylate cyclase activity." [GOC:bf, GOC:dph]
GO:0001641
name: group II metabotropic glutamate receptor activity
def: "A G-protein coupled receptor that is activated by trans-1-aminocyclopentane-1,3-dicarboxylic acid (t-ACPD) and inhibits adenylate cyclase activity." [GOC:dph]
GO:0001642
name: group III metabotropic glutamate receptor activity
def: "A G-protein coupled receptor that is activated by L-AP-4 and inhibits adenylate cyclase activity." [PMID:9016303]
GO:0001646
name: cAMP receptor activity
def: "Combining with cAMP (cyclic AMP, adenosine 3',5'-cyclophosphate) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:pg]
GO:0001647
name: G-protein coupled cytokinin receptor activity
def: "Combining with cytokinin and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:dph]
GO:0001648
name: proteinase activated receptor activity
def: "A G-protein coupled peptide receptor activity that is initiated by cleavage of the N terminus of the receptor by a serine protease, resulting in the generation of a new tethered ligand that interacts with the receptor." [GOC:mah, PMID:11356985]
GO:0001653
name: peptide receptor activity
def: "Combining with an extracellular or intracellular peptide to initiate a change in cell activity." [GOC:jl]
GO:0001664
name: G-protein coupled receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled receptor." [GOC:ceb, GOC:dph]
GO:0001665
name: alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + glycano-(1->3)-(N-acetyl-alpha-D-galactosaminyl)-glycoprotein = CMP + glycano-[(2->6)-alpha-N-acetylneuraminyl]-(N-acetyl-D-galactosaminyl)-glycoprotein." [EC:2.4.99.3]
GO:0001671
name: ATPase activator activity
def: "Functions to increase the rate of ATP hydrolysis." [GOC:ajp]
GO:0001681
name: sialate O-acetylesterase activity
def: "Catalysis of the reaction: N-acetyl-O-acetylneuraminate (free or glycosidically bound) + H2O = N-acetylneuraminate + acetate." [EC:3.1.1.53, PMID:1991039]
GO:0001691
name: pseudophosphatase activity
def: "Maintains the phosphorylation state of certain molecules by associating with them and preventing them from associating with active phosphatases, and thus inhibiting the enzyme activity without interacting with the enzyme. Often pertains to proteins belonging to dual-specificity phosphatase family but lacking critical active site residues." [GOC:ajp]
GO:0001716
name: L-amino-acid oxidase activity
def: "Catalysis of the reaction: a L-amino acid + H2O + O2 = a 2-oxo acid + NH3 + hydrogen peroxide." [EC:1.4.3.2]
GO:0001727
name: lipid kinase activity
def: "Catalysis of the phosphorylation of a simple or complex lipid." [GOC:hjd]
GO:0001729
name: ceramide kinase activity
def: "Catalysis of the reaction: ATP + ceramide = ADP + ceramide-1-phosphate." [EC:2.7.1.138]
GO:0001730
name: 2'-5'-oligoadenylate synthetase activity
def: "Catalysis of the reaction: ATP = pppA(2'p5'A)n oligomers. This reaction requires the binding of double-stranded RNA." [ISBN:0198506732]
GO:0001733
name: galactosylceramide sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + a galactosylceramide = adenosine 3',5'-bisphosphate + a galactosylceramidesulfate." [EC:2.8.2.11, PMID:10727929]
GO:0001734
name: mRNA (N6-adenosine)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + RRACH = S-adenosyl-L-homocysteine + RRm6ACH; R is a purine, and H is C, A, or U." [GOC:hjd]
GO:0001735
name: prenylcysteine oxidase activity
def: "Catalysis of the reaction: S-prenyl-L-cysteine + O2 + H2O = a prenal + L-cysteine + H2O2." [GOC:hjd]
GO:0001758
name: retinal dehydrogenase activity
def: "Catalysis of the reaction: retinal + NAD+ + H2O = retinoate + NADH. Acts on both 11-trans and 13-cis forms of retinal." [EC:1.2.1.36]
GO:0001760
name: aminocarboxymuconate-semialdehyde decarboxylase activity
def: "Catalysis of the reaction: 2-amino-3-carboxymuconate 6-semialdehyde + H(+) = 2-aminomuconate 6-semialdehyde + CO(2)." [EC:4.1.1.45, RHEA:16560]
GO:0001761
name: beta-alanine transmembrane transporter activity
def: "Catalysis of the transfer of beta-alanine from one side of a membrane to the other. Beta-alanine is 3-aminopropanoic acid." [GOC:hjd]
GO:0001784
name: phosphotyrosine binding
def: "Interacting selectively and non-covalently with a phosphorylated tyrosine residue within a protein." [PMID:14636584]
GO:0001785
name: prostaglandin J receptor activity
def: "Combining with prostaglandin J (PGJ(2)), a metabolite of prostaglandin D (PGD(2)) to initiate a change in cell activity." [PMID:12878180]
GO:0001786
name: phosphatidylserine binding
def: "Interacting selectively and non-covalently with phosphatidylserine, a class of glycophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of L-serine." [ISBN:0198506732, PMID:12000961]
GO:0001790
name: polymeric immunoglobulin binding
def: "Interacting selectively and non-covalently with a J-chain-containing polymeric immunoglobulin of the IgA or IgM isotypes." [GOC:add, ISBN:0781735149]
GO:0001791
name: IgM binding
def: "Interacting selectively and non-covalently with an immunoglobulin of the IgM isotype." [GOC:add, ISBN:0781735149]
GO:0001792
name: polymeric immunoglobulin receptor activity
def: "Combining with a J-chain-containing polymeric immunoglobulin of the IgA or IgM isotypes via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001793
name: IgM receptor activity
def: "Combining with an immunoglobulin of the IgM isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001846
name: opsonin binding
def: "Interacting selectively and non-covalently with an opsonin, such as a complement component or antibody, deposited on the surface of a bacteria, virus, immune complex, or other particulate material." [GOC:add, ISBN:0781735149]
GO:0001847
name: opsonin receptor activity
def: "Combining with an opsonin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001848
name: complement binding
def: "Interacting selectively and non-covalently with any component or product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001849
name: complement component C1q binding
def: "Interacting selectively and non-covalently with the C1q component of the classical complement cascade." [GOC:add, ISBN:0781735149]
GO:0001850
name: complement component C3a binding
def: "Interacting selectively and non-covalently with the C3a product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001851
name: complement component C3b binding
def: "Interacting selectively and non-covalently with the C3b product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001852
name: complement component iC3b binding
def: "Interacting selectively and non-covalently with the iC3b product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001853
name: complement component C3dg binding
def: "Interacting selectively and non-covalently with the C3dg product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001854
name: complement component C3d binding
def: "Interacting selectively and non-covalently with the C3d product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001855
name: complement component C4b binding
def: "Interacting selectively and non-covalently with the C4b product of the classical complement cascade." [GOC:add, ISBN:0781735149]
GO:0001856
name: complement component C5a binding
def: "Interacting selectively and non-covalently with the C5a product of the complement cascade." [GOC:add, ISBN:0781735149]
GO:0001857
name: complement component C1q receptor activity
def: "Combining with the C1q component of the classical complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001858
name: complement component iC3b receptor activity
def: "Combining with the iC3b product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001859
name: complement component C3dg receptor activity
def: "Combining with the C3dg product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001860
name: complement component C3d receptor activity
def: "Combining with the C3d product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001861
name: complement component C4b receptor activity
def: "Combining with the C4b product of the classical complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001862
name: collectin binding
def: "Interacting selectively and non-covalently with a collectin, a member of a group of structurally related pattern recognition molecules characterized by having a carbohydrate recognition domain of the C-type lectin family at the C-terminus and a collagenous domain at the N-terminus." [GOC:add, ISBN:0781735149]
GO:0001863
name: collectin receptor activity
def: "Combining with a collectin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0001864
name: pentraxin binding
def: "Interacting selectively and non-covalently with a pentraxin, a member of a family of inflammatory proteins with a radially symmetric arrangement of five identical, noncovalently linked chains in a pentagonal array." [GOC:add, ISBN:0781735149]
GO:0001871
name: pattern binding
def: "Interacting selectively and non-covalently with a repeating or polymeric structure, such as a polysaccharide or peptidoglycan." [PMID:12072369, PMID:12225919, PMID:12507420, PMID:12925128, PMID:14523544]
GO:0001872
name: (1->3)-beta-D-glucan binding
def: "Interacting selectively and non-covalently with (1->3)-beta-D-glucans." [PMID:14707091]
GO:0001873
name: polysaccharide receptor activity
def: "Combining with a polysaccharide and transmitting the signal to initiate a change in cell activity. A polysaccharide is a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [CHEBI:18154, PMID:14707091]
GO:0001874
name: (1->3)-beta-D-glucan receptor activity
def: "Combining with (1->3)-beta-D-glucans to initiate a change in cell activity." [PMID:14707091]
GO:0001875
name: lipopolysaccharide receptor activity
def: "Combining with a lipopolysaccharide and transmitting the signal across the cell membrane to initiate a change in cell activity. Lipopolysaccharides (LPS) are major components of the outer membrane of Gram-negative bacteria, making them prime targets for recognition by the immune system." [PMID:14609719, PMID:15379975]
GO:0001876
name: lipoarabinomannan binding
def: "Interacting selectively and non-covalently with lipoarabinomannan." [PMID:10586073]
GO:0001877
name: lipoarabinomannan receptor activity
def: "Combining with lipoarabinomannan and transmitting the signal to initiate a change in cell activity." [PMID:10586073]
GO:0001882
name: nucleoside binding
def: "Interacting selectively and non-covalently with a nucleoside, a compound consisting of a purine or pyrimidine nitrogenous base linked either to ribose or deoxyribose." [GOC:hjd]
GO:0001883
name: purine nucleoside binding
def: "Interacting selectively and non-covalently with a purine nucleoside, a compound consisting of a purine base linked either to ribose or deoxyribose." [GOC:hjd]
GO:0001884
name: pyrimidine nucleoside binding
def: "Interacting selectively and non-covalently with a pyrimidine nucleoside, a compound consisting of a pyrimidine base linked either to ribose or deoxyribose." [GOC:hjd]
GO:0001888
name: glucuronyl-galactosyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-glucuronosyl-(1->3)-beta-D-galactosyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-xylosyl-proteoglycan = UDP + alpha-N-acetyl-D-glucosaminyl-(1->4)-beta-D-glucuronosyl-(1->3)-beta-D-galactosyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-xylosyl-proteoglycan." [EC:2.4.1.223]
GO:0001918
name: farnesylated protein binding
def: "Interacting selectively and non-covalently with a farnesylated protein." [GOC:add, PMID:14555765]
GO:0001948
name: glycoprotein binding
def: "Interacting selectively and non-covalently with a glycoprotein, a protein that contains covalently bound glycose (monosaccharide) residues. These also include proteoglycans." [GOC:hjd, ISBN:0198506732]
GO:0001962
name: alpha-1,3-galactosyltransferase activity
def: "Catalysis of the transfer of a galactose residue from a donor molecule, such as GDP-galactose or UDP-galactose, to an oligosaccharide, forming an alpha-(1->3) linkage." [GOC:hjd, PMID:10854427]
GO:0001965
name: G-protein alpha-subunit binding
def: "Interacting selectively and non-covalently with a G-protein alpha subunit. The alpha subunit binds a guanine nucleotide." [GOC:hjd]
GO:0001968
name: fibronectin binding
def: "Interacting selectively and non-covalently with a fibronectin, a group of related adhesive glycoproteins of high molecular weight found on the surface of animal cells, connective tissue matrices, and in extracellular fluids." [GOC:hjd]
GO:0001972
name: retinoic acid binding
def: "Interacting selectively and non-covalently with retinoic acid, 3,7-dimethyl-9-(2,6,-trimethyl-1-cyclohexen-1-yl)-2,4,6,8-nonatetraenoic acid." [GOC:hjd]
GO:0002020
name: protease binding
def: "Interacting selectively and non-covalently with any protease or peptidase." [GOC:hjd]
GO:0002039
name: p53 binding
def: "Interacting selectively and non-covalently with one of the p53 family of proteins." [GOC:hjd]
GO:0002046
name: opsin binding
def: "Interacting selectively and non-covalently with an opsin, any of a group of hydrophobic, integral membrane glycoproteins located primarily in the disc membrane of rods or cones, involved in photoreception." [GOC:hjd]
GO:0002054
name: nucleobase binding
def: "Interacting selectively and non-covalently with a nucleobase, any of a class of pyrmidines or purines, organic nitrogenous bases." [GOC:hjd]
GO:0002055
name: adenine binding
def: "Interacting selectively and non-covalently with adenine, a purine base." [GOC:hjd]
GO:0002056
name: cytosine binding
def: "Interacting selectively and non-covalently with cytosine." [GOC:hjd, GOC:vw]
GO:0002057
name: guanine binding
def: "Interacting selectively and non-covalently with guanine." [GOC:hjd]
GO:0002058
name: uracil binding
def: "Interacting selectively and non-covalently with uracil." [GOC:hjd]
GO:0002059
name: thymine binding
def: "Interacting selectively and non-covalently with thymine." [GOC:hjd]
GO:0002060
name: purine nucleobase binding
def: "Interacting selectively and non-covalently with a purine nucleobase, an organic nitrogenous base with a purine skeleton." [CHEBI:26386, GOC:hjd]
GO:0002061
name: pyrimidine nucleobase binding
def: "Interacting selectively and non-covalently with a pyrimidine nucleobase, an organic nitrogenous base with a pyrimidine skeleton." [CHEBI:26432, GOC:hjd]
GO:0002083
name: 4-hydroxybenzoate decaprenyltransferase activity
def: "Catalysis of the reaction: all-trans-decaprenyl diphosphate + 4-hydroxybenzoate = 3-decaprenyl-4-hydroxybenzoate + diphosphate." [MetaCyc:RXN-9230]
GO:0002094
name: polyprenyltransferase activity
def: "Catalysis of the transfer of multiple prenyl groups from one compound (donor) to another (acceptor)." [GOC:hjd]
GO:0002112
name: interleukin-33 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-33 receptor." [GOC:hjd]
GO:0002113
name: interleukin-33 binding
def: "Interacting selectively and non-covalently with interleukin-33." [GOC:hjd]
GO:0002114
name: interleukin-33 receptor activity
def: "Combining with interleukin-33 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:hjd, GOC:signaling]
GO:0002134
name: UTP binding
def: "Interacting selectively and non-covalently with UTP, uridine 5'-triphosphate." [GOC:hjd, ISBN:0198506732]
GO:0002135
name: CTP binding
def: "Interacting selectively and non-covalently with CTP, cytidine 5'-triphosphate." [GOC:hjd, ISBN:0124020607]
GO:0002145
name: 4-amino-5-hydroxymethyl-2-methylpyrimidine diphosphatase activity
def: "Catalysis of the reaction: 4-amino-5-hydroxymethyl-2-methylpyrimidine pyrophosphate + H2O = hydroxymethylpyrimidine phosphate + phosphate + H(+)." [MetaCyc:RXN0-3543]
GO:0002151
name: G-quadruplex RNA binding
def: "Interacting selectively and non-covalently with G-quadruplex RNA structures, in which groups of four guanines adopt a flat, cyclic hydrogen-bonding arrangement known as a guanine tetrad." [PMID:18294969, PMID:18568163, PMID:19330720]
GO:0002153
name: steroid receptor RNA activator RNA binding
def: "Interacting selectively and non-covalently with the steroid receptor RNA activator RNA (SRA). SRA enhances steroid hormone receptor transcriptional activity as an RNA transcript by an indirect mechanism that does not involve SRA-steroid receptor binding." [GOC:vw, PMID:10199399, PMID:15180993]
GO:0002161
name: aminoacyl-tRNA editing activity
def: "The hydrolysis of an incorrectly aminoacylated tRNA." [GOC:hjd, PMID:14663147, PMID:16087889]
GO:0002162
name: dystroglycan binding
def: "Interacting selectively and non-covalently with dystroglycan. Dystroglycan is glycoprotein found in non-muscle tissues as well as in muscle tissues, often in association with dystrophin. The native dystroglycan cleaved into two non-covalently associated subunits, alpha (N-terminal) and beta (C-terminal)." [GOC:hjd]
GO:0002170
name: high-affinity IgA receptor activity
def: "Combining with high affinity with an immunoglobulin of an IgA isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:hjd, GOC:signaling]
GO:0002171
name: low-affinity IgA receptor activity
def: "Combining with low affinity with an immunoglobulin of an IgA isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:hjd, GOC:signaling]
GO:0002172
name: high-affinity IgM receptor activity
def: "Combining with high affinity with an immunoglobulin of an IgM isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:hjd, GOC:signaling]
GO:0002173
name: low-affinity IgM receptor activity
def: "Combining with low affinity with an immunoglobulin of an IgM isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:hjd, GOC:signaling]
GO:0002196
name: Ser-tRNA(Ala) hydrolase activity
def: "Catalysis of the hydrolysis of misacylated Ser-tRNA(Ala)." [GOC:hjd, PMID:21285375]
GO:0002935
name: tRNA (adenine-C2-)-methyltransferase activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + adenine(37) in tRNA = S-adenosyl-L-homocysteine + 5'-deoxyadenosine + L-methionine + C2-methyladenine(37) in a tRNA." [PMID:22891362]
GO:0002948
name: archaeosine synthase activity
def: "Catalysis of the reaction: L-glutamine + 7-cyano-7-carbaguanine15 in tRNA + H2O = L-glutamate + archaeine15 in tRNA." [PMID:20129918, Wikipedia:Archaeosine_synthase]
GO:0003674
name: molecular_function
def: "Elemental activities, such as catalysis or binding, describing the actions of a gene product at the molecular level. A given gene product may exhibit one or more molecular functions." [GOC:go_curators]
GO:0003676
name: nucleic acid binding
def: "Interacting selectively and non-covalently with any nucleic acid." [GOC:jl]
GO:0003677
name: DNA binding
def: "Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid)." [GOC:dph, GOC:jl, GOC:tb, GOC:vw]
GO:0003678
name: DNA helicase activity
def: "Catalysis of the reaction: NTP + H2O = NDP + phosphate, to drive the unwinding of a DNA helix." [GOC:jl, GOC:mah]
GO:0003680
name: AT DNA binding
def: "Interacting selectively and non-covalently with oligo(A) and oligo(T) tracts of DNA (AT DNA)." [GOC:jl, PMID:2670564]
GO:0003681
name: bent DNA binding
def: "Interacting selectively and non-covalently with DNA in a bent conformation." [GOC:jl, PMID:12627977]
GO:0003682
name: chromatin binding
def: "Interacting selectively and non-covalently with chromatin, the network of fibers of DNA, protein, and sometimes RNA, that make up the chromosomes of the eukaryotic nucleus during interphase." [GOC:jl, ISBN:0198506732, PMID:20404130]
GO:0003683
name: lamin/chromatin binding
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003684
name: damaged DNA binding
def: "Interacting selectively and non-covalently with damaged DNA." [GOC:jl]
GO:0003685
name: DNA repair protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:elh]
GO:0003686
name: DNA repair enzyme
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:elh]
GO:0003687
name: DNA replication factor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0003688
name: DNA replication origin binding
def: "Interacting selectively and non-covalently with the DNA replication origin, a unique DNA sequence of a replicon at which DNA replication is initiated and proceeds bidirectionally or unidirectionally." [GOC:curators]
GO:0003689
name: DNA clamp loader activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the opening of the ring structure of the PCNA complex, or any of the related sliding clamp complexes, and their closing around the DNA duplex." [GOC:mah, GOC:vw, PMID:16082778]
GO:0003690
name: double-stranded DNA binding
def: "Interacting selectively and non-covalently with double-stranded DNA." [GOC:elh]
GO:0003691
name: double-stranded telomeric DNA binding
def: "Interacting selectively and non-covalently with double-stranded telomere-associated DNA." [GOC:jl, ISBN:0321000382]
GO:0003692
name: left-handed Z-DNA binding
def: "Interacting selectively and non-covalently with DNA in the Z form, i.e. a left-handed helix in which the phosphate backbone zigzags." [ISBN:0716720094]
GO:0003693
name: P-element binding
def: "Interacting selectively and non-covalently with any P-element, a class of Drosophila transposon responsible for hybrid dysgenesis." [GOC:jl, PMID:9440262]
GO:0003694
name: plasmid binding
def: "OBSOLETE. Interacting selectively with a plasmid, an extrachromosomal genetic element usually characterized as a covalently continuous double stranded DNA molecule found in bacteria and some other microorganisms." [ISBN:0198506732]
GO:0003695
name: random coil DNA binding
def: "Interacting selectively and non-covalently with DNA in a random coil configuration." [GOC:mah]
GO:0003696
name: satellite DNA binding
def: "Interacting selectively and non-covalently with satellite DNA, the many tandem repeats (identical or related) of a short basic repeating unit; many have a base composition or other property different from the genome average that allows them to be separated from the bulk (main band) genomic DNA." [GOC:jl, SO:0000005]
GO:0003697
name: single-stranded DNA binding
def: "Interacting selectively and non-covalently with single-stranded DNA." [GOC:elh]
GO:0003700
name: sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription. The transcription factor may or may not also interact selectively with a protein or macromolecular complex." [GOC:curators, GOC:txnOH]
GO:0003701
name: RNA polymerase I transcription factor activity
def: "OBSOLETE. Functions to initiate or regulate RNA polymerase I transcription." [GOC:jl]
GO:0003702
name: RNA polymerase II transcription factor activity
def: "OBSOLETE. Functions to initiate or regulate RNA polymerase II transcription." [GOC:jl]
GO:0003704
name: specific RNA polymerase II transcription factor activity
def: "OBSOLETE. Functions to enable the transcription of specific, or specific sets, of genes by RNA polymerase II." [GOC:ma]
GO:0003705
name: RNA polymerase II distal enhancer sequence-specific DNA binding transcription factor activity
def: "Interacting selectively and non-covalently with a sequence of DNA that is in a distal enhancer region for RNA polymerase II (RNAP II) in order to modulate transcription by RNAP II." [GOC:jl, GOC:txnOH]
GO:0003706
name: ligand-regulated transcription factor activity
def: "OBSOLETE. Functions to enable the transcription of specific, or specific sets, of genes by RNA polymerase II in response to a ligand." [GOC:curators]
GO:0003707
name: steroid hormone receptor activity
def: "Combining with a steroid hormone and transmitting the signal within the cell to initiate a change in cell activity or function." [GOC:signaling, PMID:14708019]
GO:0003708
name: retinoic acid receptor activity
def: "Combining with retinoic acid and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II. Retinoic acid is one of the forms of vitamin A." [GOC:signaling, PMID:1328967]
GO:0003709
name: RNA polymerase III transcription factor activity
def: "OBSOLETE. Functions to initiate or regulate RNA polymerase III transcription." [GOC:jl]
GO:0003711
name: transcription elongation regulator activity
def: "OBSOLETE. Any activity that modulates the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule following transcription initiation." [GOC:mah]
GO:0003712
name: transcription cofactor activity
def: "Interacting selectively and non-covalently with a regulatory transcription factor and also with the basal transcription machinery in order to modulate transcription. Cofactors generally do not bind the template nucleic acid, but rather mediate protein-protein interactions between regulatory transcription factors and the basal transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0003713
name: transcription coactivator activity
def: "Interacting selectively and non-covalently with a activating transcription factor and also with the basal transcription machinery in order to increase the frequency, rate or extent of transcription. Cofactors generally do not bind the template nucleic acid, but rather mediate protein-protein interactions between activating transcription factors and the basal transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0003714
name: transcription corepressor activity
def: "Interacting selectively and non-covalently with a repressing transcription factor and also with the basal transcription machinery in order to stop, prevent, or reduce the frequency, rate or extent of transcription. Cofactors generally do not bind the template nucleic acid, but rather mediate protein-protein interactions between repressive transcription factors and the basal transcription machinery." [GOC:txnOH, PMID:10213677, PMID:16858867]
GO:0003715
name: transcription termination factor activity
def: "OBSOLETE. Any activity that brings about termination of transcription." [GOC:mah]
GO:0003716
name: RNA polymerase I transcription termination factor activity
def: "OBSOLETE. Any activity that brings about termination of transcription by RNA polymerase I." [GOC:mah]
GO:0003717
name: RNA polymerase II transcription termination factor activity
def: "OBSOLETE. Any activity that brings about termination of transcription by RNA polymerase II." [GOC:mah]
GO:0003718
name: RNA polymerase III transcription termination factor activity
def: "OBSOLETE. Any activity that brings about termination of transcription by RNA polymerase III." [GOC:mah]
GO:0003719
name: transcription factor binding, cytoplasmic sequestering
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0003720
name: telomerase activity
def: "Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes extension of the 3'- end of a DNA strand by one deoxynucleotide at a time using an internal RNA template that encodes the telomeric repeat sequence." [GOC:krc]
GO:0003721
name: telomeric template RNA reverse transcriptase activity
def: "Catalysis of the extension of the 3' end of a DNA strand by one deoxynucleotide at a time. Cannot initiate a chain de novo; uses the RNA subunit of the telomerase enzyme complex as its template." [EC:2.7.7.49, PMID:11812242]
GO:0003723
name: RNA binding
def: "Interacting selectively and non-covalently with an RNA molecule or a portion thereof." [GOC:mah]
GO:0003724
name: RNA helicase activity
def: "Catalysis of the reaction: NTP + H2O = NDP + phosphate, to drive the unwinding of a RNA helix." [GOC:jl, GOC:mah]
GO:0003725
name: double-stranded RNA binding
def: "Interacting selectively and non-covalently with double-stranded RNA." [GOC:jl]
GO:0003726
name: double-stranded RNA adenosine deaminase activity
def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a double-stranded RNA molecule." [GOC:mah]
GO:0003727
name: single-stranded RNA binding
def: "Interacting selectively and non-covalently with single-stranded RNA." [GOC:jl]
GO:0003729
name: mRNA binding
def: "Interacting selectively and non-covalently with messenger RNA (mRNA), an intermediate molecule between DNA and protein. mRNA includes UTR and coding sequences, but does not contain introns." [GOC:kmv, SO:0000234]
GO:0003730
name: mRNA 3'-UTR binding
def: "Interacting selectively and non-covalently with the 3' untranslated region of an mRNA molecule." [GOC:mah]
GO:0003731
name: mRNA cap binding
def: "OBSOLETE. Interacting selectively with the 7-methylguanosine cap at the 5' end of a nascent messenger RNA transcript." [GOC:mah]
GO:0003732
name: snRNA cap binding
def: "OBSOLETE. Interacting selectively with the cap structure at the 5' end of a small nuclear RNA molecule." [GOC:mah]
GO:0003733
name: ribonucleoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003734
name: small nuclear ribonucleoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003735
name: structural constituent of ribosome
def: "The action of a molecule that contributes to the structural integrity of the ribosome." [GOC:mah]
GO:0003743
name: translation initiation factor activity
def: "Functions in the initiation of ribosome-mediated translation of mRNA into a polypeptide." [ISBN:0198506732]
GO:0003746
name: translation elongation factor activity
def: "Functions in chain elongation during polypeptide synthesis at the ribosome." [ISBN:0198506732]
GO:0003747
name: translation release factor activity
def: "Involved in catalyzing the release of a nascent polypeptide chain from a ribosome." [ISBN:0198547684]
GO:0003750
name: cell cycle regulator
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:krc]
GO:0003754
name: chaperone activity
def: "OBSOLETE. Assists in the correct non-covalent assembly of polypeptide-containing structures in vivo, but is not a component of these assembled structures when they are performing their normal biological function." [ISBN:0198547684]
GO:0003755
name: peptidyl-prolyl cis-trans isomerase activity
def: "Catalysis of the reaction: peptidyl-proline (omega=180) = peptidyl-proline (omega=0)." [EC:5.2.1.8]
GO:0003756
name: protein disulfide isomerase activity
def: "Catalysis of the rearrangement of both intrachain and interchain disulfide bonds in proteins." [EC:5.3.4.1]
GO:0003759
name: glycoprotein-specific chaperone activity
def: "OBSOLETE. Assists in the correct, non-covalent assembly of glycoproteins in vivo, but is not a component of the assembled structures when performing its normal biological function. Utilizes a lectin site as a means to associate with the unfolded glycoproteins." [GOC:jl, PMID:11337494]
GO:0003762
name: histone-specific chaperone activity
def: "OBSOLETE. Assists in chromatin assembly by chaperoning histones on to replicating DNA, but is not a component of the assembled nucleosome when performing its normal biological function." [GOC:jl, PMID:7600578, PMID:9325046]
GO:0003763
name: chaperonin ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP involved in maintaining an unfolded polypeptide structure before folding or to entry into mitochondria and chloroplasts." [EC:3.6.4.9, ISBN:0198547684]
GO:0003767
name: co-chaperone activity
def: "OBSOLETE. Co-chaperones are proteins that bind to chaperones and this complex then folds misfolded proteins. Co-chaperones by themselves do not possess chaperone activity." [GOC:rb]
GO:0003772
name: co-chaperonin activity
def: "OBSOLETE. Co-chaperonins are proteins that bind to chaperones and this complex then folds misfolded proteins. Co-chaperonins by themselves do not possess chaperone activity." [GOC:rb]
GO:0003773
name: heat shock protein activity
def: "OBSOLETE. Any of a group of specific proteins that are synthesized by both prokaryotic and eukaryotic cells after they have been exposed to a temperature that is higher than normal. Other stresses, e.g. free radical damage, have a similar effect. Many members of the hsp family are not induced but are present in all cells. They are characterized by their role as molecular chaperones." [ISBN:0198547684]
GO:0003774
name: motor activity
def: "Catalysis of the generation of force resulting either in movement along a microfilament or microtubule, or in torque resulting in membrane scission, coupled to the hydrolysis of a nucleoside triphosphate." [GOC:mah, GOC:vw, ISBN:0815316194, PMID:11242086]
GO:0003775
name: axonemal motor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003776
name: muscle motor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003777
name: microtubule motor activity
def: "Catalysis of movement along a microtubule, coupled to the hydrolysis of a nucleoside triphosphate (usually ATP)." [GOC:mah, ISBN:0815316194]
GO:0003778
name: dynactin motor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:hjd]
GO:0003779
name: actin binding
def: "Interacting selectively and non-covalently with monomeric or multimeric forms of actin, including actin filaments." [GOC:clt]
GO:0003780
name: actin cross-linking activity
def: "OBSOLETE. Interacting selectively with two actin filaments to anchor them together." [GOC:jid]
GO:0003781
name: actin bundling activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003782
name: F-actin capping activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003783
name: barbed-end actin capping activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003784
name: barbed-end actin capping/severing activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003785
name: actin monomer binding
def: "Interacting selectively and non-covalently with monomeric actin, also known as G-actin." [GOC:ai]
GO:0003786
name: actin lateral binding
def: "Interacting selectively and non-covalently with an actin filament along its length." [GOC:mah]
GO:0003787
name: actin depolymerizing activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003788
name: actin monomer sequestering activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0003789
name: actin filament severing activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003790
name: actin modulating activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003791
name: membrane associated actin binding
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003792
name: regulation of actin thin filament length activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003793
name: defense/immunity protein activity
def: "OBSOLETE. Any activity that plays a role in the defense/immune response of an organism against infection and disease." [GOC:go_curators]
GO:0003794
name: acute-phase response protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003795
name: antimicrobial peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, microbial cells." [GOC:go_curators]
GO:0003796
name: lysozyme activity
def: "Catalysis of the hydrolysis of the beta-(1->4) linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan and between N-acetyl-D-glucosamine residues in chitodextrins." [EC:3.2.1.17]
GO:0003797
name: antibacterial peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells." [GOC:go_curators]
GO:0003798
name: male-specific antibacterial peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, bacterial cells, but which is only expressed in males." [GOC:go_curators]
GO:0003799
name: antifungal peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, fungal cells." [GOC:go_curators]
GO:0003800
name: antiviral response protein activity
def: "OBSOLETE. A protein involved in an antiviral response." [GOC:ai]
GO:0003801
name: blood coagulation factor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0003802
name: coagulation factor VIIa activity
def: "OBSOLETE. Catalysis of the selective cleavage of one Arg-Ile bond in factor X to form factor Xa, and on factor IX to form factor IXa beta." [EC:3.4.21.21, PMID:12496253]
GO:0003803
name: coagulation factor IXa activity
def: "OBSOLETE. Catalysis of the selective cleavage of the Arg-Ile bond in factor X to form factor Xa." [EC:3.4.21.22]
GO:0003804
name: coagulation factor Xa activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Thr and then Arg-Ile bonds in prothrombin to form thrombin, and on factor VII, which it converts to a two-chain form (factor VIIa)." [EC:3.4.21.6, GOC:jl, PMID:7354023]
GO:0003805
name: coagulation factor XIa activity
def: "OBSOLETE. Catalysis of the selective cleavage of Arg-Ala and Arg-Val bonds in factor IX to form factor IXa." [EC:3.4.21.27]
GO:0003806
name: coagulation factor XIIa activity
def: "OBSOLETE. Catalysis of the selective cleavage of Arg-Ile bonds in factor VII to form factor VIIa and factor XI to form factor XIa." [EC:3.4.21.38]
GO:0003807
name: plasma kallikrein activity
def: "OBSOLETE. Catalysis of the selective cleavage of Arg-Xaa and Lys-Xaa bonds, including Lys-Arg and Arg-Ser bonds in (human) kininogen to release bradykinin." [EC:3.4.21.34]
GO:0003808
name: protein C (activated) activity
def: "OBSOLETE. Catalysis of the degradation of blood coagulation factors Va and VIIIa." [EC:3.4.21.69]
GO:0003809
name: thrombin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Gly; activates fibrinogen to fibrin and releases fibrinopeptide A and B." [EC:3.4.21.5]
GO:0003810
name: protein-glutamine gamma-glutamyltransferase activity
def: "Catalysis of the reaction: protein glutamine + alkylamine = protein N5-alkylglutamine + NH3. This reaction is the formation of the N6-(L-isoglutamyl)-L-lysine isopeptide, resulting in cross-linking polypeptide chains; the gamma-carboxamide groups of peptidyl-glutamine residues act as acyl donors, and the 6-amino-groups of peptidyl-lysine residues act as acceptors, to give intra- and intermolecular N6-(5-glutamyl)lysine cross-links." [EC:2.3.2.13, RESID:AA0124]
GO:0003811
name: complement activity
def: "OBSOLETE. Any of a set of activities involved in the complement cascade." [GOC:jl]
GO:0003812
name: alternative-complement-pathway C3/C5 convertase activity
def: "OBSOLETE. Catalysis of the cleavage of Arg-Ser bond in complement component C3 alpha-chain to yield C3a and C3b, and Arg bond in complement component C5 alpha-chain to yield C5a and C5b." [EC:3.4.21.47]
GO:0003813
name: classical-complement-pathway C3/C5 convertase activity
def: "OBSOLETE. Catalysis of the selective cleavage of Arg-Ser bond in complement component C3 alpha-chain to form C3a and C3b, and Arg bond in complement component C5 alpha-chain to form C5a and C5b." [EC:3.4.21.43]
GO:0003815
name: complement component C1r activity
def: "OBSOLETE. Catalysis of the selective cleavage of Lys(or Arg)-Ile bond in complement subcomponent C1s to form the active form of C1s (EC:3.4.21.42)." [EC:3.4.21.41]
GO:0003816
name: complement component C1s activity
def: "OBSOLETE. Catalysis of the cleavage of component C4 to C4a and C4b (Arg-Ala bond), and component C2 to C2a and C2b (Lys-Lys or Arg-Lys bond)." [EC:3.4.21.42]
GO:0003817
name: complement factor D activity
def: "OBSOLETE. Catalysis of the cleavage of component factor B (Arg-Lys) when in complex with C3b or with cobra venom factor (CVF)." [EC:3.4.21.46]
GO:0003818
name: complement factor I activity
def: "OBSOLETE. Catalysis of the inactivation of complement subcomponents C3b, iC3b and C4b by proteolytic cleavage." [EC:3.4.21.45]
GO:0003819
name: major histocompatibility complex antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003820
name: class I major histocompatibility complex antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003821
name: class II major histocompatibility complex antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003822
name: MHC-interacting protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003823
name: antigen binding
def: "Interacting selectively and non-covalently with an antigen, any substance which is capable of inducing a specific immune response and of reacting with the products of that response, the specific antibody or specifically sensitized T-lymphocytes, or both. Binding may counteract the biological activity of the antigen." [GOC:jl, ISBN:0198506732, ISBN:0721662544]
GO:0003824
name: catalytic activity
def: "Catalysis of a biochemical reaction at physiological temperatures. In biologically catalyzed reactions, the reactants are known as substrates, and the catalysts are naturally occurring macromolecular substances known as enzymes. Enzymes possess specific binding sites for substrates, and are usually composed wholly or largely of protein, but RNA that has catalytic activity (ribozyme) is often also regarded as enzymatic." [ISBN:0198506732]
GO:0003825
name: alpha,alpha-trehalose-phosphate synthase (UDP-forming) activity
def: "Catalysis of the reaction: UDP-glucose + D-glucose-6-phosphate = UDP + alpha,alpha-trehalose-6-phosphate." [EC:2.4.1.15]
GO:0003826
name: alpha-ketoacid dehydrogenase activity
def: "Catalysis of an oxidation-reduction (redox) reaction involving an alpha-ketoacid." [GOC:mah]
GO:0003827
name: alpha-1,3-mannosylglycoprotein 2-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: 3-(alpha-D-mannosyl)-beta-D-mannosyl-R + UDP-N-acetyl-alpha-D-glucosamine = 3-(2-[N-acetyl-beta-D-glucosaminyl]-alpha-D-mannosyl)-beta-D-mannosyl-R + H(+) + UDP." [EC:2.4.1.101, RHEA:11459]
GO:0003828
name: alpha-N-acetylneuraminate alpha-2,8-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-R = CMP + alpha-N-acetylneuraminyl-(2->8)-alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-R." [EC:2.4.99.8]
GO:0003829
name: beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-(1->3)-N-acetyl-D-galactosaminyl-R = UDP + beta-D-galactosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->6)]-N-acetyl-D-galactosaminyl-R." [EC:2.4.1.102]
GO:0003830
name: beta-1,4-mannosylglycoprotein 4-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-mannosyl-R = UDP + 4-(N-acetyl-beta-D-glucosaminyl)-beta-D-mannosyl-R." [EC:2.4.1.144]
GO:0003831
name: beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + N-acetyl-beta-D-glucosaminylglycopeptide = UDP + beta-D-galactosyl-(1->4)-N-acetyl-beta-D-glucosaminylglycopeptide." [EC:2.4.1.38]
GO:0003832
name: beta-alanyl-dopamine hydrolase activity
def: "Catalysis of the reaction: N-beta-alanyl dopamine + H2O = dopamine + beta-alanine." [GOC:bf, ISBN:0198506732, PMID:12957543]
GO:0003833
name: beta-alanyl-dopamine synthase activity
def: "Catalysis of the synthesis of beta-alanyl-dopamine from the precursor dopamine (3,4-dihydroxyphenylethylamine)." [GOC:bf, ISBN:0198506732, PMID:12957543]
GO:0003834
name: beta-carotene 15,15'-monooxygenase activity
def: "Catalysis of the reaction: beta-carotene + O(2) = 2 retinal." [EC:1.14.99.36, RHEA:10407]
GO:0003835
name: beta-galactoside alpha-2,6-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + beta-D-galactosyl-(1->4)-acetyl-beta-D-glucosamine = CMP + alpha-N-acetylneuraminyl-(2->6)-beta-D-galactosyl-(1->4)-N-acetyl-beta-D-glucosamine." [EC:2.4.99.1]
GO:0003836
name: beta-galactoside (CMP) alpha-2,3-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + beta-D-galactosyl-(1->3)-N-acetyl-alpha-D-galactosaminyl-R = CMP + alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-(1->3)-N-acetyl-alpha-D-galactosaminyl-R." [EC:2.4.99.4]
GO:0003837
name: beta-ureidopropionase activity
def: "Catalysis of the reaction: N-carbamoyl-beta-alanine + H2O = beta-alanine + CO2 + NH3." [EC:3.5.1.6]
GO:0003838
name: sterol 24-C-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 5-alpha-cholest-8,24-dien-3-beta-ol = S-adenosyl-L-homocysteine + 24-methylene-5-alpha-cholest-8-en-3-beta-ol." [EC:2.1.1.41]
GO:0003839
name: gamma-glutamylcyclotransferase activity
def: "Catalysis of the reaction: (5-L-glutamyl)-L-amino acid = 5-oxoproline + L-amino acid." [EC:2.3.2.4]
GO:0003840
name: gamma-glutamyltransferase activity
def: "Catalysis of the reaction: (5-L-glutamyl)-peptide + an amino acid = peptide + 5-L-glutamyl-amino acid." [EC:2.3.2.2]
GO:0003841
name: 1-acylglycerol-3-phosphate O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + 1-acyl-sn-glycerol-3-phosphate = CoA + 1,2-diacyl-sn-glycerol-3-phosphate." [EC:2.3.1.51, GOC:ab]
GO:0003842
name: 1-pyrroline-5-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: 1-pyrroline-5-carboxylate + NAD+ + H2O = L-glutamate + NADH + H(+)." [EC:1.5.1.12]
GO:0003843
name: 1,3-beta-D-glucan synthase activity
def: "Catalysis of the reaction: UDP-glucose + [(1->3)-beta-D-glucosyl](n) = UDP + [(1->3)-beta-D-glucosyl](n+1)." [EC:2.4.1.34]
GO:0003844
name: 1,4-alpha-glucan branching enzyme activity
def: "Catalysis of the transfer of a segment of a (1->4)-alpha-D-glucan chain to a primary hydroxyl group in a similar glucan chain." [EC:2.4.1.18]
GO:0003845
name: 11-beta-hydroxysteroid dehydrogenase [NAD(P)] activity
def: "Catalysis of the reaction: an 11-beta-hydroxysteroid + NAD(P)+ = an 11-oxosteroid + NAD(P)H + H(+)." [EC:1.1.1.146, PMID:15761036]
GO:0003846
name: 2-acylglycerol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + 2-acylglycerol = CoA + diacylglycerol." [EC:2.3.1.22]
GO:0003847
name: 1-alkyl-2-acetylglycerophosphocholine esterase activity
def: "Catalysis of the reaction: 2-acetyl-1-alkyl-sn-glycero-3-phosphocholine + H2O = 1-alkyl-sn-glycero-3-phosphocholine + acetate." [EC:3.1.1.47]
GO:0003848
name: 2-amino-4-hydroxy-6-hydroxymethyldihydropteridine diphosphokinase activity
def: "Catalysis of the reaction: 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine + ATP = (2-amino-4-hydroxy-7,8-dihydropteridin-6-yl)methyl diphosphate + AMP + 2 H(+)." [EC:2.7.6.3, RHEA:11415]
GO:0003849
name: 3-deoxy-7-phosphoheptulonate synthase activity
def: "Catalysis of the reaction: D-erythrose 4-phosphate + H(2)O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate." [EC:2.5.1.54, RHEA:14720]
GO:0003850
name: 2-deoxyglucose-6-phosphatase activity
def: "Catalysis of the reaction: 2-deoxy-D-glucose-6-phosphate + H2O = 2-deoxy-D-glucose + phosphate." [EC:3.1.3.68]
GO:0003851
name: 2-hydroxyacylsphingosine 1-beta-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + 2-(2-hydroxyacyl)sphingosine = UDP + 1-(beta-D-galactosyl)-2-(2-hydroxyacyl)sphingosine." [EC:2.4.1.45]
GO:0003852
name: 2-isopropylmalate synthase activity
def: "Catalysis of the reaction: 3-methyl-2-oxobutanoate + acetyl-CoA + H(2)O = (2S)-2-isopropylmalate + CoA + H(+)." [EC:2.3.3.13, RHEA:21527]
GO:0003853
name: 2-methylacyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2-methylbutanoyl-CoA + acceptor = 2-methylbut-2-enoyl-CoA + reduced acceptor." [EC:1.3.99.12]
GO:0003854
name: 3-beta-hydroxy-delta5-steroid dehydrogenase activity
def: "Catalysis of the reaction: a 3-beta-hydroxy-delta(5)-steroid + NAD+ = a 3-oxo-delta(5)-steroid + NADH + H(+)." [EC:1.1.1.145]
GO:0003855
name: 3-dehydroquinate dehydratase activity
def: "Catalysis of the reaction: 3-dehydroquinate = 3-dehydroshikimate + H(2)O." [EC:4.2.1.10, RHEA:21099]
GO:0003856
name: 3-dehydroquinate synthase activity
def: "Catalysis of the reaction: 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate = 3-dehydroquinate + phosphate." [EC:4.2.3.4, RHEA:21971]
GO:0003857
name: 3-hydroxyacyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD+ = 3-oxoacyl-CoA + NADH + H(+)." [EC:1.1.1.35]
GO:0003858
name: 3-hydroxybutyrate dehydrogenase activity
def: "Catalysis of the reaction: (R)-3-hydroxybutanoate + NAD(+) = acetoacetate + H(+) + NADH." [EC:1.1.1.30, RHEA:20524]
GO:0003859
name: 3-hydroxybutyryl-CoA dehydratase activity
def: "Catalysis of the reaction: (R)-3-hydroxybutanoyl-CoA = crotonoyl-CoA + H(2)O." [EC:4.2.1.55, RHEA:17852]
GO:0003860
name: 3-hydroxyisobutyryl-CoA hydrolase activity
def: "Catalysis of the reaction: 3-hydroxy-2-methylpropanoyl-CoA + H2O = CoA + 3-hydroxy-2-methylpropanoate." [EC:3.1.2.4]
GO:0003861
name: 3-isopropylmalate dehydratase activity
def: "Catalysis of the reaction: 3-isopropylmalate = 2-isopropylmaleate + H2O." [EC:4.2.1.33]
GO:0003862
name: 3-isopropylmalate dehydrogenase activity
def: "Catalysis of the reaction: 3-carboxy-2-hydroxy-4-methylpentanoate + NAD+ = 3-carboxy-4-methyl-2-oxopentanoate + NADH + H+." [EC:1.1.1.85]
GO:0003863
name: 3-methyl-2-oxobutanoate dehydrogenase (2-methylpropanoyl-transferring) activity
def: "Catalysis of the reaction: 3-methyl-2-oxobutanoate + lipoamide = S-(2-methylpropanoyl)dihydrolipoamide + CO2." [EC:1.2.4.4]
GO:0003864
name: 3-methyl-2-oxobutanoate hydroxymethyltransferase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + 3-methyl-2-oxobutanoate = tetrahydrofolate + 2-dehydropantoate." [EC:2.1.2.11]
GO:0003865
name: 3-oxo-5-alpha-steroid 4-dehydrogenase activity
def: "Catalysis of the reaction: a 3-oxo-5-alpha-steroid + acceptor = a 3-oxo-delta(4)-steroid + reduced acceptor." [EC:1.3.99.5]
GO:0003866
name: 3-phosphoshikimate 1-carboxyvinyltransferase activity
def: "Catalysis of the reaction: 3-phosphoshikimate + phosphoenolpyruvate = 5-O-(1-carboxyvinyl)-3-phosphoshikimate + phosphate." [EC:2.5.1.19, RHEA:21259]
GO:0003867
name: 4-aminobutyrate transaminase activity
def: "Catalysis of the reaction: 4-aminobutanoate + amino group acceptor = succinate semialdehyde + amino acid." [EC:2.6.1.19, GOC:mah]
GO:0003868
name: 4-hydroxyphenylpyruvate dioxygenase activity
def: "Catalysis of the reaction: 4-hydroxyphenylpyruvate + O2 = homogentisate + CO2." [EC:1.13.11.27]
GO:0003870
name: 5-aminolevulinate synthase activity
def: "Catalysis of the reaction: glycine + H(+) + succinyl-CoA = 5-aminolevulinate + CO(2) + CoA." [EC:2.3.1.37, RHEA:12924]
GO:0003871
name: 5-methyltetrahydropteroyltriglutamate-homocysteine S-methyltransferase activity
def: "Catalysis of the reaction: 5-methyltetrahydropteroyltri-L-glutamate + L-homocysteine = L-methionine + tetrahydropteroyltri-L-glutamate." [EC:2.1.1.14, RHEA:21199]
GO:0003872
name: 6-phosphofructokinase activity
def: "Catalysis of the reaction: ATP + D-fructose-6-phosphate = ADP + D-fructose 1,6-bisphosphate." [EC:2.7.1.11]
GO:0003873
name: 6-phosphofructo-2-kinase activity
def: "Catalysis of the reaction: beta-D-fructose 6-phosphate + ATP = beta-D-fructose 2,6-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.105, RHEA:15656]
GO:0003874
name: 6-pyruvoyltetrahydropterin synthase activity
def: "Catalysis of the reaction: 7,8-dihydroneopterin 3'-triphosphate = 6-pyruvoyl-5,6,7,8-tetrahydropterin + H(+) + triphosphate." [EC:4.2.3.12, RHEA:22051]
GO:0003875
name: ADP-ribosylarginine hydrolase activity
def: "Catalysis of the reaction: N2-(ADP-D-ribosyl)-L-arginine + H2O = L-arginine + ADP-ribose." [EC:3.2.2.19]
GO:0003876
name: AMP deaminase activity
def: "Catalysis of the reaction: AMP + H2O = IMP + NH3." [EC:3.5.4.6]
GO:0003877
name: ATP adenylyltransferase activity
def: "Catalysis of the reaction: ADP + ATP = phosphate + P(1),P(4)-bis(5'-adenosyl)tetraphosphate." [EC:2.7.7.53]
GO:0003878
name: ATP citrate synthase activity
def: "Catalysis of the reaction: acetyl-CoA + ADP + H(+) + oxaloacetate + phosphate = ATP + citrate + CoA." [EC:2.3.3.8, RHEA:21163]
GO:0003879
name: ATP phosphoribosyltransferase activity
def: "Catalysis of the reaction: 1-(5-phospho-D-ribosyl)-ATP + diphosphate = ATP + 5-phospho-alpha-D-ribose 1-diphosphate." [EC:2.4.2.17]
GO:0003880
name: protein C-terminal carboxyl O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the oxygen atom of a carboxyl group at the C-terminal of a protein." [PMID:8428937]
GO:0003881
name: CDP-diacylglycerol-inositol 3-phosphatidyltransferase activity
def: "Catalysis of the reaction: myo-inositol + CDP-diacylglycerol = 1-phosphatidyl-1D-myo-inositol + CMP + H(+)." [EC:2.7.8.11, RHEA:11583]
GO:0003882
name: CDP-diacylglycerol-serine O-phosphatidyltransferase activity
def: "Catalysis of the reaction: CDP-diacylglycerol + L-serine = CMP + O-sn-phosphatidyl-L-serine." [EC:2.7.8.8]
GO:0003883
name: CTP synthase activity
def: "Catalysis of the reaction: ATP + UTP + NH3 = ADP + phosphate + CTP." [EC:6.3.4.2]
GO:0003884
name: D-amino-acid oxidase activity
def: "Catalysis of the reaction: a D-amino acid + H2O + O2 = a 2-oxo acid + NH3 + hydrogen peroxide." [EC:1.4.3.3]
GO:0003885
name: D-arabinono-1,4-lactone oxidase activity
def: "Catalysis of the reaction: D-arabinono-1,4-lactone + O(2) = dehydro-D-arabinono-1,4-lactone + H(2)O(2) + H(+)." [EC:1.1.3.37, RHEA:23759]
GO:0003886
name: DNA (cytosine-5-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + DNA containing cytosine = S-adenosyl-L-homocysteine + DNA containing 5-methylcytosine." [EC:2.1.1.37]
GO:0003887
name: DNA-directed DNA polymerase activity
def: "Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1); the synthesis of DNA from deoxyribonucleotide triphosphates in the presence of a DNA template and a 3'hydroxyl group." [EC:2.7.7.7, GOC:vw, ISBN:0198547684]
GO:0003892
name: proliferating cell nuclear antigen
def: "OBSOLETE. A nuclear protein that associates as a trimer and then interacts with delta DNA polymerase and epsilon DNA polymerase, acting as an auxiliary factor for DNA replication and DNA repair." [ISBN:0123668387]
GO:0003896
name: DNA primase activity
def: "Catalysis of the synthesis of a short RNA primer on a DNA template, providing a free 3'-OH that can be extended by DNA-directed DNA polymerases. Catalyzed by a DNA-directed RNA polymerase that forms a complex with alpha DNA polymerase." [GOC:mah, ISBN:0716720094]
GO:0003899
name: DNA-directed RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template, i.e. the catalysis of DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [EC:2.7.7.6]
GO:0003900
name: DNA-directed RNA polymerase I activity
def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1)." [EC:2.7.7.6]
GO:0003901
name: DNA-directed RNA polymerase II activity
def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1)." [EC:2.7.7.6]
GO:0003902
name: DNA-directed RNA polymerase III activity
def: "OBSOLETE. Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1)." [EC:2.7.7.6]
GO:0003904
name: deoxyribodipyrimidine photo-lyase activity
def: "Catalysis of the reaction: cyclobutadipyrimidine (in DNA) = 2 pyrimidine residues (in DNA). This reaction represents the reactivation of irradiated DNA by light." [EC:4.1.99.3]
GO:0003905
name: alkylbase DNA N-glycosylase activity
def: "Catalysis of the reaction: DNA with alkylated base + H2O = DNA with abasic site + alkylated base. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar to remove an alkylated base, leaving an apyrimidinic or apurinic site." [EC:3.2.2.21, GOC:elh, PMID:10872450, PMID:9224623]
GO:0003906
name: DNA-(apurinic or apyrimidinic site) lyase activity
def: "Catalysis of the cleavage of the C-O-P bond 3' to the apurinic or apyrimidinic site in DNA by a beta-elimination reaction, leaving a 3'-terminal unsaturated sugar and a product with a terminal 5'-phosphate." [EC:4.2.99.18]
GO:0003908
name: methylated-DNA-[protein]-cysteine S-methyltransferase activity
def: "Catalysis of the reaction: DNA (containing 6-O-methylguanine) + (protein)-L-cysteine = DNA (without 6-O-methylguanine) + protein S-methyl-L-cysteine." [EC:2.1.1.63]
GO:0003909
name: DNA ligase activity
def: "Catalysis of the formation of a phosphodiester bond between the 3'-hydroxyl group at the end of one DNA chain and the 5'-phosphate group at the end of another. This reaction requires an energy source such as ATP or NAD+." [ISBN:0716720094]
GO:0003910
name: DNA ligase (ATP) activity
def: "Catalysis of the reaction: ATP + deoxyribonucleotide(n) + deoxyribonucleotide(m) = AMP + diphosphate + deoxyribonucleotide(n+m)." [EC:6.5.1.1]
GO:0003911
name: DNA ligase (NAD+) activity
def: "Catalysis of the reaction: NAD+ + deoxyribonucleotide(n) + deoxyribonucleotide(m) = AMP + nicotinamide nucleotide + deoxyribonucleotide(n+m)." [EC:6.5.1.2]
GO:0003912
name: DNA nucleotidylexotransferase activity
def: "Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1)." [EC:2.7.7.31]
GO:0003913
name: DNA photolyase activity
def: "Catalysis of the repair of a photoproduct resulting from ultraviolet irradiation of two adjacent pyrimidine residues in DNA." [GOC:mah, PMID:11124949]
GO:0003914
name: DNA (6-4) photolyase activity
def: "Catalysis of the reaction: pyrimidine-pyrimidone (6-4) photoproduct (in DNA) = 2 pyrimidine residues (in DNA). Catalyzes the reactivation of ultraviolet-irradiated DNA." [GOC:mah, PMID:11124949]
GO:0003916
name: DNA topoisomerase activity
def: "Catalysis of the transient cleavage and passage of individual DNA strands or double helices through one another, resulting a topological transformation in double-stranded DNA." [GOC:mah, PMID:8811192]
GO:0003917
name: DNA topoisomerase type I activity
def: "Catalysis of a DNA topological transformation by transiently cleaving one DNA strand at a time to allow passage of another strand; changes the linking number by +1 per catalytic cycle." [PMID:8811192]
GO:0003918
name: DNA topoisomerase type II (ATP-hydrolyzing) activity
def: "Catalysis of a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [PMID:8811192]
GO:0003919
name: FMN adenylyltransferase activity
def: "Catalysis of the reaction: ATP + FMN = diphosphate + FAD." [EC:2.7.7.2, RHEA:17240]
GO:0003920
name: GMP reductase activity
def: "Catalysis of the reaction: IMP + NADP(+) + NH(4)(+) = GMP + 2 H(+) + NADPH." [EC:1.7.1.7, RHEA:17188]
GO:0003921
name: GMP synthase activity
def: "Catalysis of the reaction: ATP + xanthosine 5'-phosphate + NH3 = AMP + diphosphate + GMP." [EC:6.3.4.1]
GO:0003922
name: GMP synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: ATP + xanthosine 5'-phosphate + L-glutamine + H2O = AMP + diphosphate + GMP + L-glutamate." [EC:6.3.5.2]
GO:0003923
name: GPI-anchor transamidase activity
def: "Catalysis of the formation of the linkage between a protein and a glycosylphosphatidylinositol anchor. The reaction probably occurs by subjecting a peptide bond to nucleophilic attack by the amino group of ethanolamine-GPI, transferring the protein from a signal peptide to the GPI anchor." [ISBN:0471331309]
GO:0003924
name: GTPase activity
def: "Catalysis of the reaction: GTP + H2O = GDP + phosphate." [ISBN:0198547684]
GO:0003925
name: small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. Small monomeric enzymes with a molecular mass of 21 kDa that are distantly related to the alpha-subunit of heterotrimeric G-protein GTPase. They are involved in cell-growth regulation (RAS subfamily), membrane vesicle traffic and uncoating (RAB and ARF subfamilies), nuclear protein import (RAN subfamily) and organization of the cytoskeleton (Rho and Rac subfamilies)." [EC:3.6.5.2, GOC:mah, ISBN:0198547684, MetaCyc:3.6.1.47-RXN]
GO:0003926
name: ARF small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47]
GO:0003927
name: heterotrimeric G-protein GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. GTP-hydrolyzing enzymes, where GTP and GDP alternate in binding. Includes stimulatory and inhibitory G-proteins such as Gs, Gi, Go and Golf, targeting adenylate cyclase and/or potassium ion and calcium ion channels; Gq stimulating phospholipase C; transducin activating cGMP phosphodiesterase; gustducin activating cAMP phosphodiesterase." [EC:3.6.5.1, ISBN:0198547684, MetaCyc:3.6.1.46-RXN]
GO:0003928
name: RAB small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47]
GO:0003929
name: RAN small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47]
GO:0003930
name: RAS small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47]
GO:0003931
name: Rho small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. Any member of the Rho subfamily of the RAS superfamily of monomeric GTPases. Proteins in the Rho subfamily are involved in relaying signals from cell-surface receptors to the actin cytoskeleton." [EC:3.6.1.47, GOC:mah, ISBN:0198547684]
GO:0003932
name: SAR small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47]
GO:0003933
name: GTP cyclohydrolase activity
def: "Catalysis of the hydrolysis of the imidazole ring of GTP, releasing formate. Two C-N bonds are hydrolyzed and the pentase unit is isomerized." [EC:3.5.4.16, EC:3.5.4.25, EC:3.5.4.29, GOC:curators]
GO:0003934
name: GTP cyclohydrolase I activity
def: "Catalysis of the reaction: GTP + H2O = formate + 7,8-dihydroneopterin 3'-triphosphate." [EC:3.5.4.16]
GO:0003935
name: GTP cyclohydrolase II activity
def: "Catalysis of the reaction: GTP + 3 H(2)O = 2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine + diphosphate + formate + 3 H(+)." [EC:3.5.4.25, RHEA:23707]
GO:0003936
name: hydrogen-transporting two-sector ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + H+(in) = ADP + phosphate + H+(out)." [EC:3.6.3.14, TC:3.A.3.-.-]
GO:0003937
name: IMP cyclohydrolase activity
def: "Catalysis of the reaction: IMP + H2O = 5-formamido-1-(5-phosphoribosyl)imidazole-4-carboxamide." [EC:3.5.4.10]
GO:0003938
name: IMP dehydrogenase activity
def: "Catalysis of the reaction: inosine 5'-phosphate + NAD+ + H2O = xanthosine 5'-phosphate + NADH + H+." [EC:1.1.1.205]
GO:0003939
name: L-iditol 2-dehydrogenase activity
def: "Catalysis of the reaction: L-iditol + NAD+ = L-sorbose + NADH + H+." [EC:1.1.1.14]
GO:0003940
name: L-iduronidase activity
def: "Catalysis of the hydrolysis of alpha-L-iduronosidic linkages in dermatan sulfate." [EC:3.2.1.76]
GO:0003941
name: L-serine ammonia-lyase activity
def: "Catalysis of the reaction: L-serine = pyruvate + NH3." [EC:4.3.1.17]
GO:0003942
name: N-acetyl-gamma-glutamyl-phosphate reductase activity
def: "Catalysis of the reaction: N-acetyl-L-glutamate 5-semialdehyde + NADP+ + phosphate = N-acetyl-5-glutamyl phosphate + NADPH + H+." [EC:1.2.1.38]
GO:0003943
name: N-acetylgalactosamine-4-sulfatase activity
def: "Catalysis of the hydrolysis of the 4-sulfate groups of the N-acetyl-D-galactosamine 4-sulfate units of chondroitin sulfate and dermatan sulfate." [EC:3.1.6.12]
GO:0003944
name: N-acetylglucosamine-1-phosphodiester alpha-N-acetylglucosaminidase activity
def: "Catalysis of the reaction: glycoprotein N-acetyl-D-glucosaminyl-phospho-D-mannose + H2O = N-acetyl-D-glucosamine + glycoprotein phospho-D-mannose." [EC:3.1.4.45]
GO:0003945
name: N-acetyllactosamine synthase activity
def: "Catalysis of the reaction: UDP-galactose + N-acetyl-D-glucosamine = UDP + N-acetyllactosamine." [EC:2.4.1.90]
GO:0003947
name: (N-acetylneuraminyl)-galactosylglucosylceramide N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-galactosamine + (N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide = UDP + N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide." [EC:2.4.1.92]
GO:0003948
name: N4-(beta-N-acetylglucosaminyl)-L-asparaginase activity
def: "Catalysis of the reaction: N(4)-(beta-N-acetyl-D-glucosaminyl)-L-asparagine + H(2)O = N-acetyl-beta-D-glucosaminylamine + L-aspartate + H(+)." [EC:3.5.1.26, RHEA:11547]
GO:0003949
name: 1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino]imidazole-4-carboxamide isomerase activity
def: "Catalysis of the reaction: 1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino]imidazole-4-carboxamide = 5-[(5-phospho-1-deoxy-D-ribulos-1-ylimino)methylamino]-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide." [EC:5.3.1.16, RHEA:15472]
GO:0003950
name: NAD+ ADP-ribosyltransferase activity
def: "Catalysis of the reaction: NAD+ + (ADP-D-ribosyl)(n)-acceptor = nicotinamide + (ADP-D-ribosyl)(n+1)-acceptor." [EC:2.4.2.30]
GO:0003951
name: NAD+ kinase activity
def: "Catalysis of the reaction: ATP + NAD(+) = ADP + 2 H(+) + NADP(+)." [EC:2.7.1.23, RHEA:18632]
GO:0003952
name: NAD+ synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: ATP + deamido-NAD+ + L-glutamine + H2O = AMP + diphosphate + NAD+ + L-glutamate." [EC:6.3.5.1]
GO:0003953
name: NAD+ nucleosidase activity
def: "Catalysis of the reaction: NAD+ + H2O = nicotinamide + ADP-ribose." [EC:3.2.2.5]
GO:0003954
name: NADH dehydrogenase activity
def: "Catalysis of the reaction: NADH + H+ + acceptor = NAD+ + reduced acceptor." [EC:1.6.99.3]
GO:0003955
name: NAD(P)H dehydrogenase (quinone) activity
def: "Catalysis of the reaction: NAD(P)H + H+ + a quinone = NAD(P)+ + a hydroquinone." [EC:1.6.5.2]
GO:0003956
name: NAD(P)+-protein-arginine ADP-ribosyltransferase activity
def: "Catalysis of the reaction: NAD(P)+ + L-arginine = nicotinamide + N2-(ADP-D-ribosyl)-L-arginine." [EC:2.4.2.31]
GO:0003957
name: NAD(P)+ transhydrogenase (B-specific) activity
def: "Catalysis of the reaction: NADPH + H+ + NAD+ = NADP+ + NADH + H+, driving the transfer of a solute or solutes from one side of a membrane to the other. In the course of the reaction (left to right) one H atom is transferred from inside the cell to outside. The reaction is B-specific (i.e. the pro-S hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to both NAD+ and NADP+." [EC:1.6.1.1, http://pubs.acs.org/cgi-bin/abstract.cgi/jacsat/1991/113/i07/f-pdf/f_ja00007a002.pdf, MetaCyc:PYRNUTRANSHYDROGEN-RXN]
GO:0003958
name: NADPH-hemoprotein reductase activity
def: "Catalysis of the reaction: NADPH + H+ + n oxidized hemoprotein = NADP+ + n reduced hemoprotein." [EC:1.6.2.4]
GO:0003959
name: NADPH dehydrogenase activity
def: "Catalysis of the reaction: NADPH + H+ + quinone acceptor = NADP+ + hydroquinone." [EC:1.6.99.1]
GO:0003960
name: NADPH:quinone reductase activity
def: "Catalysis of the reaction: NADPH + H+ + quinone = NADP+ + semiquinone." [EC:1.6.5.5]
GO:0003961
name: O-acetylhomoserine aminocarboxypropyltransferase activity
def: "Catalysis of the reaction: O-acetyl-L-homoserine + methanethiol = L-methionine + acetate. Also reacts with other thiols and H2S, producing homocysteine or thioethers." [EC:2.5.1.49]
GO:0003962
name: cystathionine gamma-synthase activity
def: "Catalysis of the reaction: O-succinyl-L-homoserine + L-cysteine = cystathionine + succinate." [EC:2.5.1.48]
GO:0003963
name: RNA-3'-phosphate cyclase activity
def: "Catalysis of the reaction: ATP + RNA 3'-terminal-phosphate = AMP + diphosphate + RNA terminal-2',3'-cyclic-phosphate." [EC:6.5.1.4]
GO:0003964
name: RNA-directed DNA polymerase activity
def: "Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes RNA-template-directed extension of the 3'- end of a DNA strand by one deoxynucleotide at a time." [EC:2.7.7.49]
GO:0003966
name: RNA-directed DNA polymerase, transposon encoded
def: "OBSOLETE. Catalysis of RNA-template-directed extension of the 3'- end of a DNA strand by one deoxynucleotide at a time; cannot initiate a chain de novo." [EC:2.7.7.49]
GO:0003967
name: RNA-directed DNA polymerase, group II intron encoded
def: "OBSOLETE. Catalysis of RNA-template-directed extension of the 3'- end of a DNA strand by one deoxynucleotide at a time; cannot initiate a chain de novo." [EC:2.7.7.49]
GO:0003968
name: RNA-directed RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1); uses an RNA template." [EC:2.7.7.48, GOC:mah]
GO:0003969
name: RNA editase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0003972
name: RNA ligase (ATP) activity
def: "Catalysis of the reaction: ATP + ribonucleotide(n) + ribonucleotide(m) = AMP + diphosphate + ribonucleotide(n+m)." [EC:6.5.1.3]
GO:0003973
name: (S)-2-hydroxy-acid oxidase activity
def: "Catalysis of the reaction: (S)-2-hydroxy-acid + O2 = 2-oxo acid + hydrogen peroxide." [EC:1.1.3.15]
GO:0003974
name: UDP-N-acetylglucosamine 4-epimerase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine = UDP-N-acetyl-D-galactosamine." [EC:5.1.3.7]
GO:0003975
name: UDP-N-acetylglucosamine-dolichyl-phosphate N-acetylglucosaminephosphotransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + dolichyl phosphate = UMP + N-acetyl-D-glucosaminyl-diphosphodolichol." [EC:2.7.8.15]
GO:0003976
name: UDP-N-acetylglucosamine-lysosomal-enzyme N-acetylglucosaminephosphotransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + lysosomal-enzyme D-mannose = UMP + lysosomal-enzyme N-acetyl-D-glucosaminyl-phospho-D-mannose." [EC:2.7.8.17]
GO:0003977
name: UDP-N-acetylglucosamine diphosphorylase activity
def: "Catalysis of the reaction: N-acetyl-alpha-D-glucosamine 1-phosphate + UTP = diphosphate + UDP-N-acetyl-alpha-D-glucosamine." [EC:2.7.7.23, RHEA:13512]
GO:0003978
name: UDP-glucose 4-epimerase activity
def: "Catalysis of the reaction: UDP-glucose = UDP-galactose." [EC:5.1.3.2]
GO:0003979
name: UDP-glucose 6-dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + 2 NAD(+) + UDP-alpha-D-glucose = 3 H(+) + 2 NADH + UDP-alpha-D-glucuronate." [EC:1.1.1.22, RHEA:23599]
GO:0003980
name: UDP-glucose:glycoprotein glucosyltransferase activity
def: "Catalysis of the addition of UDP-glucose on to asparagine-linked (N-linked) oligosaccharides of the form Man7-9GlcNAc2 on incorrectly folded glycoproteins." [GOC:al, PMID:10764828]
GO:0003983
name: UTP:glucose-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + UTP = diphosphate + UDP-D-glucose." [EC:2.7.7.9, RHEA:19892]
GO:0003984
name: acetolactate synthase activity
def: "Catalysis of the reaction: 2 pyruvate = 2-acetolactate + CO2." [EC:2.2.1.6]
GO:0003985
name: acetyl-CoA C-acetyltransferase activity
def: "Catalysis of the reaction: 2 acetyl-CoA = CoA + acetoacetyl-CoA." [EC:2.3.1.9]
GO:0003986
name: acetyl-CoA hydrolase activity
def: "Catalysis of the reaction: acetyl-CoA + H(2)O = acetate + CoA + H(+)." [EC:3.1.2.1, RHEA:20292]
GO:0003987
name: acetate-CoA ligase activity
def: "Catalysis of the reaction: ATP + acetate + CoA = AMP + diphosphate + acetyl-CoA." [EC:6.2.1.1]
GO:0003988
name: acetyl-CoA C-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + acetyl-CoA = CoA + 3-oxoacyl-CoA." [EC:2.3.1.16]
GO:0003989
name: acetyl-CoA carboxylase activity
def: "Catalysis of the reaction: ATP + acetyl-CoA + HCO3- = ADP + phosphate + malonyl-CoA." [EC:6.4.1.2]
GO:0003990
name: acetylcholinesterase activity
def: "Catalysis of the reaction: acetylcholine + H2O = choline + acetate." [EC:3.1.1.7]
GO:0003991
name: acetylglutamate kinase activity
def: "Catalysis of the reaction: ATP + N-acetyl-L-glutamate = ADP + N-acetyl-L-glutamate-5-phosphate." [EC:2.7.2.8]
GO:0003992
name: N2-acetyl-L-ornithine:2-oxoglutarate 5-aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-acetyl-L-ornithine = N-acetyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.11, RHEA:18052]
GO:0003993
name: acid phosphatase activity
def: "Catalysis of the reaction: an orthophosphoric monoester + H2O = an alcohol + phosphate, with an acid pH optimum." [EC:3.1.3.2]
GO:0003994
name: aconitate hydratase activity
def: "Catalysis of the reaction: citrate = isocitrate. The reaction occurs in two steps: (1) citrate = cis-aconitate + H2O, (2) cis-aconitate + H2O = isocitrate. This reaction is the interconversion of citrate and isocitrate via the labile, enzyme-bound intermediate cis-aconitate. Water is removed from one part of the citrate molecule and added back to a different atom to form isocitrate." [EC:4.2.1.3, GOC:pde, GOC:vw]
GO:0003995
name: acyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor." [EC:1.3.99.3]
GO:0003996
name: acyl-CoA ligase activity
def: "Catalysis of the reaction: ATP + a long-chain carboxylic acid + CoA = AMP + diphosphate + an acyl-CoA; long-chain fatty acids have chain lengths of C13 to C22." [CHEBI:15904, EC:6.2.1.3, GOC:mah]
GO:0003997
name: acyl-CoA oxidase activity
def: "Catalysis of the reaction: acyl-CoA + O2 = trans-2,3-dehydroacyl-CoA + hydrogen peroxide." [EC:1.3.3.6]
GO:0003998
name: acylphosphatase activity
def: "Catalysis of the reaction: an acyl phosphate + H2O = a carboxylate + phosphate." [EC:3.6.1.7]
GO:0003999
name: adenine phosphoribosyltransferase activity
def: "Catalysis of the reaction: AMP + diphosphate = adenine + 5-phospho-alpha-D-ribose 1-diphosphate." [EC:2.4.2.7]
GO:0004000
name: adenosine deaminase activity
def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3." [EC:3.5.4.4]
GO:0004001
name: adenosine kinase activity
def: "Catalysis of the reaction: ATP + adenosine = ADP + AMP." [EC:2.7.1.20, PMID:11223943]
GO:0004003
name: ATP-dependent DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of the DNA helix." [EC:3.6.1.3, GOC:jl]
GO:0004004
name: ATP-dependent RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of an RNA helix." [EC:3.6.1.3, GOC:jl]
GO:0004005
name: plasma membrane cation-transporting ATPase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004007
name: heavy metal-exporting ATPase activity
def: "OBSOLETE. Catalysis of the reaction: heavy metal ion(in) + ATP + H2O = heavy metal ion(out) + ADP + phosphate." [GOC:ai]
GO:0004008
name: copper-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Cu2+(in) -> ADP + phosphate + Cu2+(out)." [EC:3.6.3.4]
GO:0004009
name: ATP-binding cassette (ABC) transporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0004012
name: phospholipid-translocating ATPase activity
def: "Catalysis of the movement of phospholipids from one membrane bilayer leaflet to the other, driven by the hydrolysis of ATP." [EC:3.6.3.1, PMID:15919184, PMID:9099684]
GO:0004013
name: adenosylhomocysteinase activity
def: "Catalysis of the reaction: S-adenosyl-L-homocysteine + H2O = adenosine + L-homocysteine." [EC:3.3.1.1]
GO:0004014
name: adenosylmethionine decarboxylase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + H(+) = S-adenosylmethioninamine + CO(2)." [EC:4.1.1.50, RHEA:15984]
GO:0004015
name: adenosylmethionine-8-amino-7-oxononanoate transaminase activity
def: "Catalysis of the reaction: 8-amino-7-oxononanoate + S-adenosyl-L-methionine(1+) = 7,8-diaminononanoate + S-adenosyl-4-methylthio-2-oxobutanoate." [EC:2.6.1.62, RHEA:16864]
GO:0004016
name: adenylate cyclase activity
def: "Catalysis of the reaction: ATP = 3',5'-cyclic AMP + diphosphate." [EC:4.6.1.1]
GO:0004017
name: adenylate kinase activity
def: "Catalysis of the reaction: ATP + AMP = 2 ADP." [EC:2.7.4.3]
GO:0004018
name: N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming) activity
def: "Catalysis of the reaction: N6-(1,2-dicarboxyethyl)AMP = fumarate + AMP." [EC:4.3.2.2]
GO:0004019
name: adenylosuccinate synthase activity
def: "Catalysis of the reaction: L-aspartate + GTP + IMP = N(6)-(1,2-dicarboxyethyl)-AMP + GDP + 3 H(+) + phosphate." [EC:6.3.4.4, RHEA:15756]
GO:0004020
name: adenylylsulfate kinase activity
def: "Catalysis of the reaction: ATP + adenylylsulfate = ADP + 3'-phosphoadenosine 5'-phosphosulfate." [EC:2.7.1.25]
GO:0004021
name: L-alanine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-alanine = L-glutamate + pyruvate." [EC:2.6.1.2, RHEA:19456]
GO:0004022
name: alcohol dehydrogenase (NAD) activity
def: "Catalysis of the reaction: an alcohol + NAD+ = an aldehyde or ketone + NADH + H+." [EC:1.1.1.1]
GO:0004023
name: alcohol dehydrogenase activity, metal ion-independent
def: "Catalysis of the reaction: an alcohol + NAD+ = an aldehyde or ketone + NADH + H+; can proceed in the absence of a metal ion." [EC:1.1.1.1, GOC:mah]
GO:0004024
name: alcohol dehydrogenase activity, zinc-dependent
def: "Catalysis of the reaction: an alcohol + NAD+ = an aldehyde or ketone + NADH + H+, requiring the presence of zinc." [EC:1.1.1.1, GOC:mah]
GO:0004025
name: alcohol dehydrogenase activity, iron-dependent
def: "Catalysis of the reaction: an alcohol + NAD+ = an aldehyde or ketone + NADH + H+, requiring the presence of iron." [EC:1.1.1.1, GOC:mah]
GO:0004026
name: alcohol O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an alcohol = CoA + an acetyl ester." [EC:2.3.1.84]
GO:0004027
name: alcohol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + an alcohol = adenosine 3',5'-bisphosphate + an alkyl sulfate." [EC:2.8.2.2]
GO:0004028
name: 3-chloroallyl aldehyde dehydrogenase activity
def: "Catalysis of the reaction: 3-chloroallyl aldehyde + H2O = 2 H+ + 2 e- + 3-chloroacrylic acid." [UM-BBD_enzymeID:e0432]
GO:0004029
name: aldehyde dehydrogenase (NAD) activity
def: "Catalysis of the reaction: an aldehyde + NAD+ + H2O = an acid + NADH + H+." [EC:1.2.1.3]
GO:0004030
name: aldehyde dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: an aldehyde + NAD(P)+ + H2O = an acid + NAD(P)H + H+." [EC:1.2.1.5]
GO:0004031
name: aldehyde oxidase activity
def: "Catalysis of the reaction: an aldehyde + H2O + O2 = a carboxylic acid + hydrogen peroxide." [EC:1.2.3.1]
GO:0004032
name: alditol:NADP+ 1-oxidoreductase activity
def: "Catalysis of the reaction: an alditol + NADP+ = an aldose + NADPH + H+." [EC:1.1.1.21]
GO:0004033
name: aldo-keto reductase (NADP) activity
def: "Catalysis of the reaction: an alcohol + NADP+ = an aldehyde or a ketone + NADPH + H+." [GOC:ai]
GO:0004034
name: aldose 1-epimerase activity
def: "Catalysis of the reaction: alpha-D-glucose = beta-D-glucose. Also acts on L-arabinose, D-xylose, D-galactose, maltose and lactose." [EC:5.1.3.3]
GO:0004035
name: alkaline phosphatase activity
def: "Catalysis of the reaction: an orthophosphoric monoester + H2O = an alcohol + phosphate, with an alkaline pH optimum." [EC:3.1.3.1]
GO:0004037
name: allantoicase activity
def: "Catalysis of the reaction: allantoate + H(2)O = (S)-ureidoglycolate + urea." [EC:3.5.3.4, RHEA:11019]
GO:0004038
name: allantoinase activity
def: "Catalysis of the reaction: allantoin + H2O = allantoate." [EC:3.5.2.5]
GO:0004039
name: allophanate hydrolase activity
def: "Catalysis of the reaction: H(2)O + 3 H(+) + urea-1-carboxylate = 2 CO(2) + 2 NH(4)(+)." [EC:3.5.1.54, RHEA:19032]
GO:0004040
name: amidase activity
def: "Catalysis of the reaction: a monocarboxylic acid amide + H2O = a monocarboxylate + NH3." [EC:3.5.1.4]
GO:0004042
name: acetyl-CoA:L-glutamate N-acetyltransferase activity
def: "Catalysis of the reaction: L-glutamate + acetyl-CoA = N-acetyl-L-glutamate + CoA + H(+)." [EC:2.3.1.1, RHEA:24295]
GO:0004043
name: L-aminoadipate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: L-2-aminoadipate 6-semialdehyde + NADP+ + H2O = L-2-aminoadipate + NADPH + H+." [EC:1.2.1.31]
GO:0004044
name: amidophosphoribosyltransferase activity
def: "Catalysis of the reaction: 5-phospho-beta-D-ribosylamine + L-glutamate + diphosphate = 5-phospho-alpha-D-ribose 1-diphosphate + L-glutamine + H(2)O." [EC:2.4.2.14, RHEA:14908]
GO:0004045
name: aminoacyl-tRNA hydrolase activity
def: "Catalysis of the reaction: N-substituted aminoacyl-tRNA + H2O = N-substituted amino acid + tRNA." [EC:3.1.1.29]
GO:0004046
name: aminoacylase activity
def: "Catalysis of the reaction: an N-acyl-L-amino acid + H2O = a carboxylate + an L-amino acid." [EC:3.5.1.14]
GO:0004047
name: aminomethyltransferase activity
def: "Catalysis of the reaction: (6S)-tetrahydrofolate + S-aminomethyldihydrolipoylprotein = (6R)-5,10-methylenetetrahydrofolate + NH3 + dihydrolipoylprotein." [EC:2.1.2.10]
GO:0004048
name: anthranilate phosphoribosyltransferase activity
def: "Catalysis of the reaction: N-(5-phospho-beta-D-ribosyl)anthranilate + diphosphate = 5-phospho-alpha-D-ribose 1-diphosphate + anthranilate." [EC:2.4.2.18, RHEA:11771]
GO:0004049
name: anthranilate synthase activity
def: "Catalysis of the reaction: chorismate + L-glutamine = anthranilate + pyruvate + L-glutamate." [EC:4.1.3.27]
GO:0004050
name: apyrase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 2 H2O = AMP + 2 phosphate." [EC:3.6.1.5, MetaCyc:APYRASE-RXN]
GO:0004051
name: arachidonate 5-lipoxygenase activity
def: "Catalysis of the reaction: arachidonate + O2 = (6E,8Z,11Z,14Z)-(5S)-5-hydroperoxycosa-6,8,11,14-tetraenoate." [EC:1.13.11.34]
GO:0004052
name: arachidonate 12-lipoxygenase activity
def: "Catalysis of the reaction: arachidonate + O(2) = (5Z,8Z,10E,12S,14Z)-12-hydroperoxyicosa-5,8,10,14-tetraenoate." [EC:1.13.11.31, RHEA:10431]
GO:0004053
name: arginase activity
def: "Catalysis of the reaction: L-arginine + H2O = L-ornithine + urea." [EC:3.5.3.1]
GO:0004054
name: arginine kinase activity
def: "Catalysis of the reaction: L-arginine + ATP = N(omega)-phospho-L-arginine + ADP + 2 H(+)." [EC:2.7.3.3, RHEA:22943]
GO:0004055
name: argininosuccinate synthase activity
def: "Catalysis of the reaction: ATP + L-citrulline + L-aspartate = AMP + diphosphate + (N(omega)-L-arginino)succinate." [EC:6.3.4.5]
GO:0004056
name: argininosuccinate lyase activity
def: "Catalysis of the reaction: N-(L-arginino)succinate = fumarate + L-arginine." [EC:4.3.2.1]
GO:0004057
name: arginyltransferase activity
def: "Catalysis of the reaction: L-arginyl-tRNA + protein = tRNA + L-arginyl-protein." [EC:2.3.2.8]
GO:0004058
name: aromatic-L-amino-acid decarboxylase activity
def: "Catalysis of the reaction: L-amino acid + H+ = R-H + CO2." [EC:4.1.1.28]
GO:0004059
name: aralkylamine N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an aralkylamine = CoA + an N-acetylaralkylamine." [EC:2.3.1.87]
GO:0004060
name: arylamine N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an arylamine = CoA + an N-acetylarylamine." [EC:2.3.1.5]
GO:0004061
name: arylformamidase activity
def: "Catalysis of the reaction: N-formyl-L-kynurenine + H2O = formate + L-kynurenine." [EC:3.5.1.9]
GO:0004062
name: aryl sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + a phenol = adenosine 3',5'-bisphosphate + an aryl sulfate." [EC:2.8.2.1]
GO:0004063
name: aryldialkylphosphatase activity
def: "Catalysis of the reaction: aryl dialkyl phosphate + H2O = dialkyl phosphate + an aryl alcohol." [EC:3.1.8.1]
GO:0004064
name: arylesterase activity
def: "Catalysis of the reaction: a phenyl acetate + H2O = a phenol + acetate." [EC:3.1.1.2]
GO:0004065
name: arylsulfatase activity
def: "Catalysis of the reaction: a phenol sulfate + H2O = a phenol + sulfate." [EC:3.1.6.1]
GO:0004066
name: asparagine synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: ATP + L-aspartate + L-glutamine = AMP + diphosphate + L-asparagine + L-glutamate." [EC:6.3.5.4]
GO:0004067
name: asparaginase activity
def: "Catalysis of the reaction: L-asparagine + H2O = L-aspartate + NH3." [EC:3.5.1.1]
GO:0004068
name: aspartate 1-decarboxylase activity
def: "Catalysis of the reaction: L-aspartate = beta-alanine + CO2." [EC:4.1.1.11]
GO:0004069
name: L-aspartate:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-aspartate + 2-oxoglutarate = oxaloacetate + L-glutamate." [EC:2.6.1.1]
GO:0004070
name: aspartate carbamoyltransferase activity
def: "Catalysis of the reaction: L-aspartate + carbamoyl phosphate = N-carbamoyl-L-aspartate + H(+) + phosphate." [EC:2.1.3.2, RHEA:20016]
GO:0004071
name: aspartate-ammonia ligase activity
def: "Catalysis of the reaction: ATP + L-aspartate + NH3 = AMP + diphosphate + L-asparagine." [EC:6.3.1.1]
GO:0004072
name: aspartate kinase activity
def: "Catalysis of the reaction: L-aspartate + ATP = 4-phospho-L-aspartate + ADP + H(+)." [EC:2.7.2.4, RHEA:23779]
GO:0004073
name: aspartate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: L-aspartate 4-semialdehyde + NADP(+) + phosphate = 4-phospho-L-aspartate + H(+) + NADPH." [EC:1.2.1.11, RHEA:24287]
GO:0004074
name: biliverdin reductase activity
def: "Catalysis of the reaction: bilirubin + NAD(P)+ = biliverdin + NAD(P)H + H+." [EC:1.3.1.24]
GO:0004075
name: biotin carboxylase activity
def: "Catalysis of the reaction: ATP + biotin-carboxyl-carrier protein + CO2 = ADP + phosphate + carboxybiotin-carboxyl-carrier protein." [EC:6.3.4.14]
GO:0004076
name: biotin synthase activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + dethiobiotin + S(2-) = 2 5'-deoxyadenosine + 2 L-methionine + biotin + H(+)." [EC:2.8.1.6, RHEA:22063]
GO:0004077
name: biotin-[acetyl-CoA-carboxylase] ligase activity
def: "Catalysis of the reaction: ATP + biotin + apo-(acetyl-CoA:carbon-dioxide ligase (ADP forming)) = AMP + diphosphate + (acetyl-CoA:carbon-dioxide ligase (ADP forming))." [EC:6.3.4.15]
GO:0004078
name: biotin-[methylcrotonoyl-CoA-carboxylase] ligase activity
def: "Catalysis of the reaction: ATP + biotin + apo-(3-methylcrotonoyl-CoA:carbon-dioxide ligase (ADP-forming)) = AMP + diphosphate + (3-methylcrotonoyl-CoA:carbon-dioxide ligase (ADP-forming))." [EC:6.3.4.11]
GO:0004079
name: biotin-[methylmalonyl-CoA-carboxytransferase] ligase activity
def: "Catalysis of the reaction: ATP + biotin + apo-(methylmalonyl-CoA:pyruvate carboxytransferase) = AMP + diphosphate + (methylmalonyl-CoA:pyruvate carboxytransferase)." [EC:6.3.4.9]
GO:0004080
name: biotin-[propionyl-CoA-carboxylase (ATP-hydrolyzing)] ligase activity
def: "Catalysis of the reaction: ATP + biotin + apo-(propanoyl-CoA:carbon-dioxide ligase (ADP-forming)) = AMP + diphosphate + (propanoyl-CoA:carbon-dioxide ligase (ADP-forming))." [EC:6.3.4.10]
GO:0004081
name: bis(5'-nucleosyl)-tetraphosphatase (asymmetrical) activity
def: "Catalysis of the reaction: P(1),P(4)-bis(5'-nucleosyl)tetraphosphate + H2O = NTP + NMP. Acts on bis(5'-guanosyl)-, bis(5'-xanthosyl)-, bis(5'-adenosyl)- and bis(5'-uridyl)-tetraphosphate." [EC:3.6.1.17, PMID:4955726]
GO:0004082
name: bisphosphoglycerate mutase activity
def: "Catalysis of the reaction: 3-phospho-D-glyceroyl phosphate = 2,3-bisphospho-D-glycerate." [EC:5.4.2.4]
GO:0004083
name: bisphosphoglycerate 2-phosphatase activity
def: "Catalysis of the reaction: 2,3-bisphospho-D-glycerate + H(2)O = 3-phospho-D-glycerate + phosphate." [EC:3.1.3.13, RHEA:21907]
GO:0004084
name: branched-chain-amino-acid transaminase activity
def: "Catalysis of the reaction: a branched-chain amino acid + 2-oxoglutarate = L-glutamate + a 2-oxocarboxylate derived from the branched-chain amino acid." [EC:2.6.1.42, GOC:mah]
GO:0004085
name: butyryl-CoA dehydrogenase activity
def: "Catalysis of the reaction: butanoyl-CoA + electron-transfer flavoprotein = 2-butenoyl-CoA + reduced electron-transfer flavoprotein." [EC:1.3.8.1]
GO:0004086
name: carbamoyl-phosphate synthase activity
def: "OBSOLETE. Catalysis of a reaction that results in the formation of carbamoyl phosphate." [EC:6.3.4.16, EC:6.3.5.5, GOC:mah]
GO:0004087
name: carbamoyl-phosphate synthase (ammonia) activity
def: "Catalysis of the reaction: 2 ATP + CO(2) + H(2)O + NH(4)(+) = 2 ADP + carbamoyl phosphate + 5 H(+) + phosphate." [EC:6.3.4.16, RHEA:10627]
GO:0004088
name: carbamoyl-phosphate synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: 2 ATP + L-glutamine + CO2 + H2O = 2 ADP + phosphate + glutamate + carbamoyl phosphate." [EC:6.3.5.5]
GO:0004089
name: carbonate dehydratase activity
def: "Catalysis of the reaction: H2CO3 = CO2 + H2O." [EC:4.2.1.1]
GO:0004090
name: carbonyl reductase (NADPH) activity
def: "Catalysis of the reaction: R-CHOH-R' + NADP+ = R-CO-R' + NADPH + H+." [EC:1.1.1.184]
GO:0004092
name: carnitine O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + carnitine = (R)-O-acetylcarnitine + CoA." [EC:2.3.1.7, RHEA:21139]
GO:0004095
name: carnitine O-palmitoyltransferase activity
def: "Catalysis of the reaction: palmitoyl-CoA + L-carnitine = CoA + L-palmitoylcarnitine." [EC:2.3.1.21]
GO:0004096
name: catalase activity
def: "Catalysis of the reaction: 2 hydrogen peroxide = O2 + 2 H2O." [EC:1.11.1.6]
GO:0004097
name: catechol oxidase activity
def: "Catalysis of the reaction: 2 catechol + O2 = 2 1,2-benzoquinone + 2 H2O." [EC:1.10.3.1]
GO:0004098
name: cerebroside-sulfatase activity
def: "Catalysis of the reaction: a cerebroside 3-sulfate + H2O = a cerebroside + sulfate." [EC:3.1.6.8]
GO:0004099
name: chitin deacetylase activity
def: "Catalysis of the reaction: chitin + H2O = chitosan + acetate." [EC:3.5.1.41]
GO:0004100
name: chitin synthase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + [->4)-N-acetyl-beta-D-glucosaminyl-(1-](n) = UDP + [->4)-N-acetyl-beta-D-glucosaminyl-(1-](n+1)." [EC:2.4.1.16]
GO:0004102
name: choline O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + choline = acetylcholine + CoA." [EC:2.3.1.6, RHEA:18824]
GO:0004103
name: choline kinase activity
def: "Catalysis of the reaction: ATP + choline = ADP + choline phosphate + 2 H(+)." [EC:2.7.1.32, RHEA:12840]
GO:0004104
name: cholinesterase activity
def: "Catalysis of the reaction: an acylcholine + H2O = choline + a carboxylic acid anion." [EC:3.1.1.8]
GO:0004105
name: choline-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + choline phosphate = diphosphate + CDP-choline." [EC:2.7.7.15]
GO:0004106
name: chorismate mutase activity
def: "Catalysis of the reaction: chorismate = prephenate." [EC:5.4.99.5, RHEA:13900]
GO:0004107
name: chorismate synthase activity
def: "Catalysis of the reaction: 5-O-(1-carboxyvinyl)-3-phosphoshikimate = chorismate + phosphate." [EC:4.2.3.5, RHEA:21023]
GO:0004108
name: citrate (Si)-synthase activity
def: "Catalysis of the reaction: acetyl-CoA + H2O + oxaloacetate = citrate + CoA, where the acetyl group is added to the si-face of oxaloacetate; acetyl-CoA thus provides the two carbon atoms of the pro-S carboxymethyl group." [EC:2.3.3.1, ISBN:0121227073]
GO:0004109
name: coproporphyrinogen oxidase activity
def: "Catalysis of the reaction: coproporphyrinogen III + 2 H(+) + O(2) = 2 CO(2) + 2 H(2)O + protoporphyrinogen IX." [EC:1.3.3.3, RHEA:18260]
GO:0004110
name: corticosteroid side-chain-isomerase activity
def: "Catalysis of the reaction: 11-deoxycorticosterone = 20-hydroxy-3-oxopregn-4-en-21-al." [EC:5.3.1.21, RHEA:17864]
GO:0004111
name: creatine kinase activity
def: "Catalysis of the reaction: ATP + creatine = N-phosphocreatine + ADP + 2 H(+)." [EC:2.7.3.2, RHEA:17160]
GO:0004112
name: cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: a nucleoside cyclic phosphate + H2O = a nucleoside phosphate." [GOC:mah]
GO:0004113
name: 2',3'-cyclic-nucleotide 3'-phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 2',3'-cyclic phosphate + H2O = nucleoside 2'-phosphate." [EC:3.1.4.37]
GO:0004114
name: 3',5'-cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 3',5'-cyclic phosphate + H2O = nucleoside 5'-phosphate." [EC:3.1.4.17]
GO:0004115
name: 3',5'-cyclic-AMP phosphodiesterase activity
def: "Catalysis of the reaction: adenosine 3',5'-cyclic phosphate + H2O = adenosine 5'-phosphate." [GOC:ai]
GO:0004117
name: calmodulin-dependent cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 3',5'-cyclic phosphate + H2O = nucleoside 5'-phosphate; catalytic activity is regulated by calmodulin." [GOC:mah]
GO:0004118
name: cGMP-stimulated cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 3',5'-cyclic phosphate + H2O = nucleoside 5'-phosphate; catalytic activity is increased in the presence of cGMP." [GOC:mah]
GO:0004119
name: cGMP-inhibited cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 3',5'-cyclic phosphate + H2O = nucleoside 5'-phosphate; catalytic activity is decreased in the presence of cGMP." [GOC:mah]
GO:0004120
name: photoreceptor cyclic-nucleotide phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside cyclic phosphate + H2O = nucleoside phosphate. This reaction is the hydrolysis of bonds in a cyclic nucleotide." [EC:3.1.4.-]
GO:0004121
name: cystathionine beta-lyase activity
def: "Catalysis of the reaction: cystathionine + H2O = L-homocysteine + NH3 + pyruvate." [EC:4.4.1.8]
GO:0004122
name: cystathionine beta-synthase activity
def: "Catalysis of the reaction: L-serine + L-homocysteine = cystathionine + H2O." [EC:4.2.1.22]
GO:0004123
name: cystathionine gamma-lyase activity
def: "Catalysis of the reaction: L-cystathionine + H2O = 2-oxobutanoate + L-cysteine + NH4+." [RHEA:24918]
GO:0004124
name: cysteine synthase activity
def: "Catalysis of the reaction: O3-acetyl-L-serine + hydrogen sulfide = L-cysteine + acetate." [EC:4.2.99.8]
GO:0004125
name: L-seryl-tRNASec selenium transferase activity
def: "Catalysis of the reaction: L-seryl-tRNA(Sec) + selenophosphate = L-selenocysteinyl-tRNA(Sec) + H2O + phosphate." [EC:2.9.1.1]
GO:0004126
name: cytidine deaminase activity
def: "Catalysis of the reaction: cytidine + H2O = uridine + NH3." [EC:3.5.4.5]
GO:0004127
name: cytidylate kinase activity
def: "Catalysis of the reaction: ATP + (d)CMP = ADP + (d)CDP." [EC:2.7.4.14]
GO:0004128
name: cytochrome-b5 reductase activity, acting on NAD(P)H
def: "Catalysis of the reaction: NAD(P)H + H+ + 2 ferricytochrome b(5) = NAD(P)+ + 2 ferrocytochrome b(5)." [EC:1.6.2.2, ISBN:0198547684]
GO:0004129
name: cytochrome-c oxidase activity
def: "Catalysis of the reaction: 4 ferrocytochrome c + O2 + 4 H+ = 4 ferricytochrome c + 2 H2O." [EC:1.9.3.1]
GO:0004130
name: cytochrome-c peroxidase activity
def: "Catalysis of the reaction: 2 ferrocytochrome c + hydrogen peroxide = 2 ferricytochrome c + 2 H2O." [EC:1.11.1.5]
GO:0004131
name: cytosine deaminase activity
def: "Catalysis of the reaction: cytosine + H2O = uracil + NH3." [EC:3.5.4.1]
GO:0004132
name: dCMP deaminase activity
def: "Catalysis of the reaction: dCMP + H2O = dUMP + NH3." [EC:3.5.4.12]
GO:0004133
name: glycogen debranching enzyme activity
def: "Catalysis of the cleavage of branch points in branched glycogen polymers." [ISBN:0198506732]
GO:0004134
name: 4-alpha-glucanotransferase activity
def: "Catalysis of the transfer of a segment of a (1->4)-alpha-D-glucan to a new 4-position in an acceptor, which may be glucose or (1->4)-alpha-D-glucan." [EC:2.4.1.25]
GO:0004135
name: amylo-alpha-1,6-glucosidase activity
def: "Catalysis of the hydrolysis of (1->6)-alpha-D-glucosidic branch linkages in glycogen phosphorylase limit dextrin. Limit dextrin is the highly branched core that remains after exhaustive treatment of glycogen with glycogen phosphorylase. It is formed because these enzymes cannot hydrolyze the (1->6) glycosidic linkages present." [EC:3.2.1.33, ISBN:0198506732]
GO:0004136
name: deoxyadenosine kinase activity
def: "Catalysis of the reaction: 2'-deoxyadenosine + ATP = ADP + dAMP + 2 H(+)." [EC:2.7.1.76, RHEA:23455]
GO:0004137
name: deoxycytidine kinase activity
def: "Catalysis of the reaction: NTP + deoxycytidine = NDP + CMP." [EC:2.7.1.74]
GO:0004138
name: deoxyguanosine kinase activity
def: "Catalysis of the reaction: 2'-deoxyguanosine + ATP = ADP + dGMP + 2 H(+)." [EC:2.7.1.113, RHEA:19204]
GO:0004139
name: deoxyribose-phosphate aldolase activity
def: "Catalysis of the reaction: 2-deoxy-D-ribose 5-phosphate = D-glyceraldehyde 3-phosphate + acetaldehyde." [EC:4.1.2.4, RHEA:12824]
GO:0004140
name: dephospho-CoA kinase activity
def: "Catalysis of the reaction: 3'-dephospho-CoA + ATP = ADP + CoA + 2 H(+)." [EC:2.7.1.24, RHEA:18248]
GO:0004141
name: dethiobiotin synthase activity
def: "Catalysis of the reaction: 7,8-diaminononanoate + ATP + CO(2) = ADP + dethiobiotin + 4 H(+) + phosphate." [EC:6.3.3.3, RHEA:15808]
GO:0004142
name: diacylglycerol cholinephosphotransferase activity
def: "Catalysis of the reaction: CDP-choline + 1,2-diacylglycerol = CMP + a phosphatidylcholine." [EC:2.7.8.2]
GO:0004143
name: diacylglycerol kinase activity
def: "Catalysis of the reaction: NTP + 1,2-diacylglycerol = NDP + 1,2-diacylglycerol-3-phosphate." [EC:2.7.1.107, GOC:elh]
GO:0004144
name: diacylglycerol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + 1,2-diacylglycerol = CoA + triacylglycerol." [EC:2.3.1.20]
GO:0004145
name: diamine N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an alkane-alpha,omega-diamine = CoA + an N-acetyldiamine." [EC:2.3.1.57]
GO:0004146
name: dihydrofolate reductase activity
def: "Catalysis of the reaction: 5,6,7,8-tetrahydrofolate + NADP+ = 7,8-dihydrofolate + NADPH + H+." [EC:1.5.1.3]
GO:0004147
name: dihydrolipoamide branched chain acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + dihydrolipoamide = CoA + S-acyldihydrolipoamide, where the acyl group is a branched chain." [GOC:mah]
GO:0004148
name: dihydrolipoyl dehydrogenase activity
def: "Catalysis of the reaction: protein N6-(dihydrolipoyl)lysine + NAD+ = protein N6-(lipoyl)lysine + NADH + H+." [EC:1.8.1.4]
GO:0004149
name: dihydrolipoyllysine-residue succinyltransferase activity
def: "Catalysis of the reaction: succinyl-CoA + dihydrolipoamide = CoA + S-succinyldihydrolipoamide." [EC:2.3.1.61]
GO:0004150
name: dihydroneopterin aldolase activity
def: "Catalysis of the reaction: 2-amino-4-hydroxy-6-(D-erythro-1,2,3-trihydroxypropyl)-7,8-dihydropteridine = 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine + glycolaldehyde." [EC:4.1.2.25]
GO:0004151
name: dihydroorotase activity
def: "Catalysis of the reaction: (S)-dihydroorotate + H(2)O = N-carbamoyl-L-aspartate + H(+)." [EC:3.5.2.3, RHEA:24299]
GO:0004152
name: dihydroorotate dehydrogenase activity
def: "Catalysis of the reaction: (S)-dihydroorotate + A = AH(2) + orotate." [EC:1.3.5.2, RHEA:18076]
GO:0004153
name: dihydropterin deaminase activity
def: "Catalysis of the reaction: 7,8-dihydropterin + H2O = 7,8-dihydrolumazine + NH3." [FB:FBrf0039640, GOC:jl, http://www.abbs.info/fulltxt/eng/35030306.htm, ISSN:05829879]
GO:0004154
name: dihydropterin oxidase activity
def: "Catalysis of the reaction: a 7,8-dihydropteridine + O2 = a pterin + hydrogen peroxide." [GOC:mah, PMID:1745247, PMID:6815189]
GO:0004155
name: 6,7-dihydropteridine reductase activity
def: "Catalysis of the reaction: NADP+ + 5,6,7,8-tetrahydropteridine = NADPH + H+ + 6,7-dihydropteridine." [EC:1.5.1.34]
GO:0004156
name: dihydropteroate synthase activity
def: "Catalysis of the reaction: 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine diphosphate + 4-aminobenzoate = diphosphate + dihydropteroate." [EC:2.5.1.15]
GO:0004157
name: dihydropyrimidinase activity
def: "Catalysis of the reaction: 5,6-dihydrouracil + H2O = 3-ureidopropionate." [EC:3.5.2.2]
GO:0004158
name: dihydroorotate oxidase activity
def: "Catalysis of the reaction: (S)-dihydroorotate + O(2) = H(2)O(2) + orotate." [EC:1.3.3.1, RHEA:15444]
GO:0004159
name: dihydrouracil dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: 5,6-dihydrouracil + NAD+ = uracil + NADH + H+." [EC:1.3.1.1]
GO:0004160
name: dihydroxy-acid dehydratase activity
def: "Catalysis of the reaction: 2,3-dihydroxy-3-methylbutanoate = 3-methyl-2-oxobutanoate + H2O." [EC:4.2.1.9]
GO:0004161
name: dimethylallyltranstransferase activity
def: "Catalysis of the reaction: dimethylallyl diphosphate + isopentenyl diphosphate = diphosphate + geranyl diphosphate." [EC:2.5.1.1]
GO:0004162
name: dimethylnitrosamine demethylase activity
def: "Catalysis of the removal of a methyl group from N-nitrosodimethylamine." [GOC:mah]
GO:0004163
name: diphosphomevalonate decarboxylase activity
def: "Catalysis of the reaction: (R)-5-diphosphomevalonate + ATP = ADP + CO(2) + H(+) + isopentenyl diphosphate + phosphate." [EC:4.1.1.33, RHEA:23735]
GO:0004164
name: diphthine synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 2-(3-carboxy-3-aminopropyl)-L-histidine = S-adenosyl-L-homocysteine + 2-(3-carboxy-3-(methylammonio)propyl)-L-histidine." [EC:2.1.1.98]
GO:0004165
name: dodecenoyl-CoA delta-isomerase activity
def: "Catalysis of the reaction: 3-cis-dodecenoyl-CoA = 2-trans-dodecenoyl-CoA." [EC:5.3.3.8]
GO:0004166
name: dolichyl-phosphate alpha-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + dolichyl phosphate = UDP + dolichyl N-acetyl-alpha-D-glucosaminyl phosphate." [EC:2.4.1.153]
GO:0004167
name: dopachrome isomerase activity
def: "Catalysis of the reaction: L-dopachrome = 5,6-dihydroxyindole-2-carboxylate." [EC:5.3.3.12, RHEA:13044]
GO:0004168
name: dolichol kinase activity
def: "Catalysis of the reaction: CTP + dolichol = CDP + dolichyl phosphate." [EC:2.7.1.108]
GO:0004169
name: dolichyl-phosphate-mannose-protein mannosyltransferase activity
def: "Catalysis of the reaction: dolichyl phosphate D-mannose + protein = dolichyl phosphate + O-D-mannosylprotein." [EC:2.4.1.109, GOC:pr]
GO:0004170
name: dUTP diphosphatase activity
def: "Catalysis of the reaction: dUTP + H2O = dUMP + diphosphate." [EC:3.6.1.23]
GO:0004171
name: deoxyhypusine synthase activity
def: "OBSOLETE. Catalysis of the reaction: [eIF5A-precursor]-lysine + spermidine = [eIF5A-precursor]-deoxyhypusine + propane-1,3-diamine." [EC:2.5.1.46]
GO:0004172
name: ecdysteroid UDP-glucosyl/UDP-glucuronosyl transferase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004173
name: ecdysone O-acyltransferase activity
def: "Catalysis of the reaction: Ecdysone + palmitoyl-CoA = CoA + ecdysone palmitate." [EC:2.3.1.139, RHEA:15220]
GO:0004174
name: electron-transferring-flavoprotein dehydrogenase activity
def: "Catalysis of the reaction: reduced ETF + ubiquinone = ETF + ubiquinol." [EC:1.5.5.1]
GO:0004175
name: endopeptidase activity
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain." [http://merops.sanger.ac.uk/about/glossary.htm#ENDOPEPTIDASE]
GO:0004176
name: ATP-dependent peptidase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the hydrolysis of peptide bonds." [GOC:mah]
GO:0004177
name: aminopeptidase activity
def: "Catalysis of the hydrolysis of N-terminal amino acid residues from in a polypeptide chain." [GOC:jl, ISBN:0198506732]
GO:0004178
name: leucyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal amino acid, Xaa-Xbb-, in which Xaa is preferably Leu, but may be other amino acids including Pro although not Arg or Lys, and Xbb may be Pro." [EC:3.4.11.1]
GO:0004179
name: membrane alanyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal amino acid, Xaa-Xbb- from a peptide, amide or arylamide. Xaa is preferably Ala, but may be most amino acids including Pro (slow action). When a terminal hydrophobic residue is followed by a prolyl residue, the two may be released as an intact Xaa-Pro dipeptide." [EC:3.4.11.2]
GO:0004180
name: carboxypeptidase activity
def: "Catalysis of the hydrolysis of the terminal or penultimate peptide bond at the C-terminal end of a peptide or polypeptide." [ISBN:0198506732]
GO:0004181
name: metallocarboxypeptidase activity
def: "Catalysis of the hydrolysis of C-terminal amino acid residues from a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [http://merops.sanger.ac.uk/about/glossary.htm#CARBOXYPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, ISBN:0198506732]
GO:0004182
name: carboxypeptidase A activity
def: "OBSOLETE. Catalysis of the reaction: peptidyl-L-amino acid + H2O = peptide + L-amino acid. Little or no action with -Asp, -Glu, -Arg, -Lys or -Pro." [EC:3.4.17.1]
GO:0004183
name: carboxypeptidase E activity
def: "OBSOLETE. Catalysis of the reaction: peptidyl-L-lysine (or peptidyl-L-arginine) + H2O = peptide + L-lysine (or L-arginine). Function is activated by Co2+ and inhibited by 1,10-phenanthroline and other chelating agents." [EC:3.4.17.10]
GO:0004184
name: lysine carboxypeptidase activity
def: "OBSOLETE. Catalysis of the reaction: peptidyl-L-lysine (or peptidyl-L-arginine) + H2O = peptide + L-lysine (or L-arginine). Release of a C-terminal basic amino acid, preferentially lysine; inactivates bradykinin and anaphylatoxins in blood plasma." [EC:3.4.17.3]
GO:0004185
name: serine-type carboxypeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond not more than three residues from the C-terminus of a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [http://merops.sanger.ac.uk/about/glossary.htm#CARBOXYPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, ISBN:0716720094]
GO:0004186
name: carboxypeptidase C activity
def: "OBSOLETE. Catalysis of the release of a C-terminal amino acid with a broad specificity." [EC:3.4.16.5]
GO:0004187
name: carboxypeptidase D activity
def: "OBSOLETE. Catalysis of the preferential release of a C-terminal arginine or lysine residue. Function is inhibited by diisopropyl fluorophosphate and sensitive to thiol-blocking reagents." [EC:3.4.16.6]
GO:0004188
name: serine-type Pro-X carboxypeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a Pro-Xaa bond by a serine-type peptidase mechanism to release a C-terminal amino acid." [EC:3.4.16.2]
GO:0004189
name: tubulinyl-Tyr carboxypeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of the Glu-Tyr bond to release the C-terminal tyrosine residue from the native tyrosinated tubulin. Inactive on Z-Glu-Tyr." [EC:3.4.17.17]
GO:0004190
name: aspartic-type endopeptidase activity
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which a water molecule bound by the side chains of aspartic residues at the active center acts as a nucleophile." [ISBN:0198506732]
GO:0004191
name: barrierpepsin activity
def: "OBSOLETE. Catalysis of the selected cleavage of the Leu6-Lys7 bond in the pheromone alpha-mating factor." [EC:3.4.23.35]
GO:0004192
name: cathepsin D activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Specificity similar to, but narrower than, that of pepsin A. Does not cleave the Gln4-His5 bond in the B chain of insulin." [EC:3.4.23.5]
GO:0004193
name: cathepsin E activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Similar to cathepsin D, but slightly broader specificity." [EC:3.4.23.34]
GO:0004194
name: pepsin A activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Preferential cleavage: hydrophobic, preferably aromatic, residues in P1 and P1' positions. Cleaves Phe1-Val2, Gln4-His5, Glu13-Ala14, Ala14-Leu15, Leu15-Tyr16, Tyr16-Leu17, Gly23-Phe24, Phe24-Phe25 and Phe25-Tyr26 bonds in the B chain of insulin." [EC:3.4.23.1]
GO:0004195
name: renin activity
def: "OBSOLETE. Catalysis of the cleavage of Leu-Xaa bond in angiotensinogen to generate angiotensin I." [EC:3.4.23.15]
GO:0004196
name: saccharopepsin activity
def: "OBSOLETE. Catalysis of the degradation of gelatin; little activity on hemoglobin. Specificity on B chain of insulin more restricted than pepsin A; does not cleave Phe1-Val2, Gln4-His5 or Gly23-Phe24." [EC:3.4.23.25]
GO:0004197
name: cysteine-type endopeptidase activity
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#ENDOPEPTIDASE]
GO:0004198
name: calcium-dependent cysteine-type endopeptidase activity
def: "Catalysis of the hydrolysis of nonterminal peptide bonds in a polypeptide chain by a mechanism using a cysteine residue at the enzyme active center, and requiring the presence of calcium." [GOC:mah]
GO:0004200
name: signaling (initiator) caspase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004201
name: caspase-1 activity
def: "OBSOLETE. Catalysis of the release of interleukin 1-beta by specific cleavage of Asp116-Ala117 and Asp27-Gly28 bonds in precursor. Enzymes with this function can also hydrolyze the terminal bond in the small-molecule substrate, Ac-Tyr-Val-Ala-Asp-NHMec." [EC:3.4.22.36]
GO:0004202
name: caspase-2 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004203
name: caspase-4 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004204
name: caspase-5 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004205
name: caspase-8 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004206
name: caspase-10 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004207
name: effector caspase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004208
name: caspase-3 activity
def: "OBSOLETE. Catalysis of the cleavage at the terminal bond of the motif: Asp-Xaa-Xaa-Asp-Xaa. Enzymes with this function are members of the peptidase family C14 and they appear to function in the inactivation of proteins involved in cellular repair and homeostasis during the effector stage of apoptosis." [ISBN:0120793709]
GO:0004209
name: caspase-6 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004210
name: caspase-7 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004211
name: caspase-9 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004212
name: lysosomal cysteine-type endopeptidase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004213
name: cathepsin B activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds with a broad specificity. Preferentially cleaves the terminal bond of -Arg-Arg-Xaa motifs in small molecule substrates (thus differing from cathepsin L). In addition to being an endopeptidase, shows peptidyl-dipeptidase activity, liberating C-terminal dipeptides." [EC:3.4.22.1]
GO:0004214
name: dipeptidyl-peptidase I activity
def: "OBSOLETE. Catalysis of the release of an N-terminal dipeptide, Xaa-Xbb from Xaa-Xbb-Xcc, except when Xaa is Arg or Lys, or Xbb or Xcc is Pro." [EC:3.4.14.1]
GO:0004215
name: cathepsin H activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds, acting as an aminopeptidase (notably, cleaving Arg-Xaa bonds) as well as an endopeptidase." [EC:3.4.22.16]
GO:0004216
name: cathepsin K activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Has a broad proteolytic activity. With small-molecule substrates and inhibitors, the major determinant of specificity is P2, which is preferably Leu, Met > Phe, and not Arg." [EC:3.4.22.38]
GO:0004217
name: cathepsin L activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Specificity close to that of papain. As compared to cathepsin B, cathepsin L exhibits higher activity towards protein substrates, but has little activity on Z-Arg-Arg-NHMec, and no peptidyl-dipeptidase activity." [EC:3.4.22.15]
GO:0004218
name: cathepsin S activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Similar to cathepsin L, but with much less activity on the terminal bond of Z-Phe-Arg-NHMec, and more activity on the terminal bond of Z-Val-Val-Arg-Xaa compounds." [EC:3.4.22.27]
GO:0004219
name: pyroglutamyl-peptidase I activity
def: "OBSOLETE. Catalysis of the reaction: pyroglutamyl-peptide + H2O = pyroglutamate + peptide." [EC:3.4.19.3]
GO:0004221
name: ubiquitin thiolesterase activity
def: "OBSOLETE. Catalysis of the reaction: ubiquitin C-terminal thioester + H2O = ubiquitin + a thiol." [EC:3.1.2.15, GOC:jh2]
GO:0004222
name: metalloendopeptidase activity
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#ENDOPEPTIDASE]
GO:0004226
name: Gly-X carboxypeptidase activity
def: "OBSOLETE. Catalysis of the reaction: peptidyl-Gly-Xaa + H2O = peptidyl-Gly + Xaa." [EC:3.4.17.4]
GO:0004228
name: gelatinase A activity
def: "OBSOLETE. Catalysis of the cleavage of gelatin type I and collagen types IV, V, VII, X. Cleaves the collagen-like sequence Pro-Gln-Gly-Ile-Ala-Gly-Gln at the Gly-Ile bond." [EC:3.4.24.24]
GO:0004229
name: gelatinase B activity
def: "OBSOLETE. Catalysis of the cleavage of gelatin types I and V and collagen types IV and V." [EC:3.4.24.35]
GO:0004230
name: glutamyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of a N-terminal glutamate (and to a lesser extent aspartate) from a peptide." [EC:3.4.11.7]
GO:0004231
name: insulysin activity
def: "OBSOLETE. Catalysis of the degradation of insulin, glucagon and other polypeptides. No action on proteins." [EC:3.4.24.56]
GO:0004232
name: interstitial collagenase activity
def: "OBSOLETE. Catalysis of the preferential cleavage of one bond in native collagen. Cleavage of the triple helix of collagen at about three-quarters of the length of the molecule from the N-terminus, at Gly775-Ile776 in the alpha-1(I) chain. Cleaves synthetic substrates and alpha-macroglobulins at bonds where P1' is a hydrophobic residue." [EC:3.4.24.7]
GO:0004234
name: macrophage elastase activity
def: "OBSOLETE. Catalysis of the hydrolysis of soluble and insoluble elastin. Specific cleavages are also produced at Ala14-Leu15 and Tyr16-Leu17 in the B chain of insulin." [EC:3.4.24.65]
GO:0004235
name: matrilysin activity
def: "OBSOLETE. Catalysis of the cleavage of Ala14-Leu15 and Tyr16-Leu17 in B chain of insulin. No action on collagen types I, II, IV and V. Cleaves gelatin chain alpha-2(I) > alpha-1(I)." [EC:3.4.24.23]
GO:0004237
name: membrane dipeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of dipeptides." [EC:3.4.13.19]
GO:0004238
name: meprin A activity
def: "OBSOLETE. Catalysis of the hydrolysis of protein and peptide substrates preferentially on carboxyl side of hydrophobic residues." [EC:3.4.24.18]
GO:0004239
name: methionyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of N-terminal amino acids, preferentially methionine, from peptides and arylamides." [EC:3.4.11.18]
GO:0004240
name: mitochondrial processing peptidase activity
def: "OBSOLETE. Catalysis of the release of N-terminal transit peptides from precursor proteins imported into the mitochondrion, typically with Arg in position P2." [EC:3.4.24.64]
GO:0004241
name: alpha-mitochondrial processing peptidase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004242
name: beta-mitochondrial processing peptidase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004243
name: mitochondrial intermediate peptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal octapeptide as second stage of processing of some proteins imported in the mitochondrion." [EC:3.4.24.59]
GO:0004244
name: mitochondrial inner membrane peptidase activity
def: "OBSOLETE. Catalysis of the maturation of mitochondrial precursor proteins delivered to the intermembrane space." [PMID:12191769]
GO:0004245
name: neprilysin activity
def: "OBSOLETE. Catalysis of the preferential cleavage at the amino group of hydrophobic residues in insulin, casein, hemoglobin, and a number of other proteins and polypeptides." [EC:3.4.24.11]
GO:0004246
name: peptidyl-dipeptidase A activity
def: "OBSOLETE. Catalysis of the release of a C-terminal dipeptide, Xaa-Xbb from oligopeptide-Xaa-Xbb, when Xaa is not Pro, and Xbb is neither Asp nor Glu. Converts angiotensin I to angiotensin II." [EC:3.4.15.1]
GO:0004247
name: saccharolysin activity
def: "OBSOLETE. Catalysis of the cleavage of Pro-Phe and Ala-Ala bonds." [EC:3.4.24.37]
GO:0004248
name: stromelysin 1 activity
def: "OBSOLETE. Catalysis of the preferential cleavage where P1', P2' and P3' are hydrophobic residues." [EC:3.4.24.17]
GO:0004249
name: stromelysin 3 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0004250
name: aminopeptidase I activity
def: "OBSOLETE. Catalysis of the release of an N-terminal amino acid, preferably a neutral or hydrophobic one, from a polypeptide. Aminoacyl-arylamides are poor substrates." [EC:3.4.11.22]
GO:0004251
name: X-Pro dipeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of Xaa-Pro dipeptides; also acts on aminoacyl-hydroxyproline analogs. No action on Pro-Pro." [EC:3.4.13.9]
GO:0004252
name: serine-type endopeptidase activity
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, ISBN:0716720094]
GO:0004253
name: gamma-renin activity
def: "OBSOLETE. Catalysis of the cleavage of the Leu-Leu bond in synthetic tetradecapeptide renin substrate, producing angiotensin I, but not active on natural angiotensinogen. Also hydrolyzes BZ-Arg-para-nitroanilide." [EC:3.4.21.54]
GO:0004254
name: acylaminoacyl-peptidase activity
def: "OBSOLETE. Catalysis of the reaction: acylaminoacyl-peptide + H2O = acylamino acid + peptide." [EC:3.4.19.1]
GO:0004258
name: vacuolar carboxypeptidase Y
def: "OBSOLETE. Catalysis of the release of a C-terminal amino acid with a broad specificity." [EC:3.4.16.5]
GO:0004261
name: cathepsin G activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Specificity similar to chymotrypsin C." [EC:3.4.21.20]
GO:0004262
name: cerevisin activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins with broad specificity, and of BZ-Arg-OET > Ac-Tyr-OET. Does not hydrolyze peptide amides." [EC:3.4.21.48]
GO:0004263
name: chymotrypsin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Tyr-Xaa > Trp-Xaa > Phe-Xaa > Leu-Xaa." [EC:3.4.21.1, ISBN:0198506732]
GO:0004274
name: dipeptidyl-peptidase IV activity
def: "OBSOLETE. Catalysis of the release of an N-terminal dipeptide, by the hydrolysis of the Xbb-Xcc bond in Xaa-Xbb-Xcc, preferentially when Xbb is Pro, provided Xcc is neither Pro nor hydroxyproline." [EC:3.4.14.5]
GO:0004275
name: enteropeptidase activity
def: "OBSOLETE. Catalysis of the selective cleavage of Lys6-Ile7 bond in trypsinogen." [EC:3.4.21.9]
GO:0004276
name: furin activity
def: "OBSOLETE. Catalysis of the release of mature proteins from their proproteins by cleavage of the terminal bond of Arg-Xaa-Yaa-Arg-Z motifs where Xaa can be any amino acid and Yaa is Arg or Lys. Releases albumin, complement component C3 and von Willebrand factor from their respective precursors." [EC:3.4.21.75]
GO:0004277
name: granzyme A activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins, including fibronectin, type IV collagen and nucleolin. Preferential cleavage: Arg-Xaa > Lys-Xaa > Phe-Xaa in small molecule substrates." [EC:3.4.21.78]
GO:0004278
name: granzyme B activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Asp-Xaa > Asn-Xaa > Met-Xaa, Ser-Xaa." [EC:3.4.21.79]
GO:0004281
name: pancreatic elastase II activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Leu-Xaa, Met-Xaa and Phe-Xaa. Hydrolyzes elastin." [EC:3.4.21.71]
GO:0004283
name: plasmin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Lys-Xaa > Arg-Xaa; higher selectivity than trypsin. Converts fibrin into soluble products." [EC:3.4.21.7]
GO:0004284
name: acrosin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Xaa > Lys-Xaa." [EC:3.4.21.10]
GO:0004285
name: proprotein convertase 1 activity
def: "OBSOLETE. Catalysis of the release of protein hormones, neuropeptides and renin from their precursors, generally by cleavage of -Lys-Arg-Xaa at the Arg-Xaa bond." [EC:3.4.21.93]
GO:0004286
name: proprotein convertase 2 activity
def: "OBSOLETE. Catalysis of the release of protein hormones and neuropeptides from their precursors, generally by cleavage of -Lys-Arg-Xaa at the Arg-Xaa bond." [EC:3.4.21.94]
GO:0004287
name: prolyl oligopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of Pro-Xaa > Ala-Xaa in oligopeptides." [EC:3.4.21.26]
GO:0004289
name: subtilase activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0004290
name: kexin activity
def: "OBSOLETE. Catalysis of the cleavage of the Arg-Xaa bond in Lys-Arg-Xaa and Arg-Arg-Xaa to process Yeast alpha-factor pheromone and killer toxin precursors." [EC:3.4.21.61]
GO:0004291
name: subtilisin activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins with broad specificity for peptide bonds, and a preference for a large uncharged residue in P1. Hydrolyzes peptide amides." [EC:3.4.21.62]
GO:0004293
name: tissue kallikrein activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Xaa bonds in small molecule substrates. Highly selective action to release kallidin (lysyl-bradykinin) from kininogen involves hydrolysis of Met-Xaa or Leu-Xaa. The rat enzyme is unusual in liberating bradykinin directly from autologous kininogens by cleavage at two Arg-Xaa bonds." [EC:3.4.21.35]
GO:0004294
name: tripeptidyl-peptidase II activity
def: "OBSOLETE. Catalysis of the release of an N-terminal tripeptide from a polypeptide at neutral pH." [EC:3.4.14.10]
GO:0004295
name: trypsin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Xaa, Lys-Xaa." [EC:3.4.21.4]
GO:0004298
name: threonine-type endopeptidase activity
def: "Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism in which the hydroxyl group of a threonine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#ENDOPEPTIDASE]
GO:0004299
name: proteasome endopeptidase activity
def: "OBSOLETE. Catalysis of the cleavage at peptide bonds with very broad specificity." [EC:3.4.25.1]
GO:0004300
name: enoyl-CoA hydratase activity
def: "Catalysis of the reaction: (3S)-3-hydroxyacyl-CoA = trans-2-enoyl-CoA + H2O." [EC:4.2.1.17]
GO:0004301
name: epoxide hydrolase activity
def: "Catalysis of the reaction: an epoxide + H2O = a glycol." [EC:3.3.2.10]
GO:0004303
name: estradiol 17-beta-dehydrogenase activity
def: "Catalysis of the reaction: estradiol-17-beta + NADP+ = estrone + NADPH + H+." [EC:1.1.1.62]
GO:0004304
name: estrone sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + estrone = adenosine 3',5'-bisphosphate + estrone 3-sulfate." [EC:2.8.2.4]
GO:0004305
name: ethanolamine kinase activity
def: "Catalysis of the reaction: ATP + ethanolamine = ADP + 2 H(+) + phosphoethanolamine." [EC:2.7.1.82, RHEA:13072]
GO:0004306
name: ethanolamine-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + ethanolamine phosphate = diphosphate + CDP-ethanolamine." [EC:2.7.7.14]
GO:0004307
name: ethanolaminephosphotransferase activity
def: "Catalysis of the reaction: CDP-ethanolamine + 1,2-diacylglycerol = CMP + a phosphatidylethanolamine." [EC:2.7.8.1]
GO:0004308
name: exo-alpha-sialidase activity
def: "Catalysis of the hydrolysis of alpha-(2->3)-, alpha-(2->6)-, alpha-(2->8)-glycosidic linkages of terminal sialic residues in oligosaccharides, glycoproteins, glycolipids, colominic acid and synthetic substrates." [EC:3.2.1.18]
GO:0004309
name: exopolyphosphatase activity
def: "Catalysis of the reaction: polyphosphate(n) + H2O = polyphosphate(n-1) + phosphate." [EC:3.6.1.11]
GO:0004310
name: farnesyl-diphosphate farnesyltransferase activity
def: "Catalysis of the reaction: 2 farnesyl diphosphate = diphosphate + presqualene diphosphate." [EC:2.5.1.21]
GO:0004311
name: farnesyltranstransferase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate + isopentenyl diphosphate = diphosphate + geranylgeranyl diphosphate." [EC:2.5.1.29, RHEA:17656]
GO:0004312
name: fatty acid synthase activity
def: "Catalysis of the reaction: acetyl-CoA + n malonyl-CoA + 2n NADPH + 2n H+ = long-chain fatty acid + n+1 CoA + n CO2 + 2n NADP+." [EC:2.3.1.85]
GO:0004313
name: [acyl-carrier-protein] S-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + [acyl-carrier protein] = CoA + acetyl-[acyl-carrier protein]." [EC:2.3.1.38]
GO:0004314
name: [acyl-carrier-protein] S-malonyltransferase activity
def: "Catalysis of the reaction: malonyl-CoA + [acyl-carrier protein] = CoA + malonyl-[acyl-carrier protein]." [EC:2.3.1.39]
GO:0004315
name: 3-oxoacyl-[acyl-carrier-protein] synthase activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + malonyl-[acyl-carrier protein] = 3-oxoacyl-[acyl-carrier protein] + CO2 + [acyl-carrier protein]." [EC:2.3.1.41]
GO:0004316
name: 3-oxoacyl-[acyl-carrier-protein] reductase (NADPH) activity
def: "Catalysis of the reaction: (3R)-3-hydroxyacyl-[acyl-carrier protein] + NADP+ = 3-oxoacyl-[acyl-carrier protein] + NADPH + H+." [EC:1.1.1.100]
GO:0004317
name: 3-hydroxypalmitoyl-[acyl-carrier-protein] dehydratase activity
def: "Catalysis of the reaction: (3R)-3-hydroxypalmitoyl-[acyl-carrier protein] = 2-hexadecenoyl-[acyl-carrier protein] + H2O." [EC:4.2.1.61]
GO:0004318
name: enoyl-[acyl-carrier-protein] reductase (NADH) activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + NAD+ = trans-2,3-dehydroacyl-[acyl-carrier protein] + NADH + H+." [EC:1.3.1.9]
GO:0004319
name: enoyl-[acyl-carrier-protein] reductase (NADPH, B-specific) activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + NADP+ = trans-2,3-dehydroacyl-[acyl-carrier protein] + NADPH + H+." [EC:1.3.1.10]
GO:0004320
name: oleoyl-[acyl-carrier-protein] hydrolase activity
def: "Catalysis of the reaction: oleoyl-[acyl-carrier protein] + H2O = [acyl-carrier protein] + oleate." [EC:3.1.2.14]
GO:0004321
name: fatty-acyl-CoA synthase activity
def: "Catalysis of the reaction: acetyl-CoA + n malonyl-CoA + 2n NADH + 2n NADPH + 4n H+ = a long-chain acyl-CoA + n CoA + n CO2 + 2n NAD+ + 2n NADP+." [EC:2.3.1.86]
GO:0004322
name: ferroxidase activity
def: "Catalysis of the reaction: 4 Fe2+ + 4 H+ + O2 = 4 Fe3+ + 2 H2O." [EC:1.16.3.1]
GO:0004323
name: multicopper ferroxidase iron transport mediator activity
def: "OBSOLETE. Catalysis of the reaction: 4 Fe2+ + 4 H+ + O2 = 4 Fe3+ + 2 H2O." [EC:1.16.3.1]
GO:0004324
name: ferredoxin-NADP+ reductase activity
def: "Catalysis of the reaction: reduced ferredoxin + NADP+ = oxidized ferredoxin + NADPH + H+." [EC:1.18.1.2, GOC:kd]
GO:0004325
name: ferrochelatase activity
def: "Catalysis of the reaction: protoheme = Fe(2+) + protoporphyrin IX." [EC:4.99.1.1, RHEA:22587]
GO:0004326
name: tetrahydrofolylpolyglutamate synthase activity
def: "Catalysis of the reaction: ATP + tetrahydrofolyl-(Glu)(n) + L-glutamate = ADP + phosphate + tetrahydrofolyl-(Glu)(n+1)." [EC:6.3.2.17]
GO:0004327
name: formaldehyde dehydrogenase (glutathione) activity
def: "OBSOLETE. Catalysis of the reaction: formaldehyde + glutathione + NAD+ = S-formylglutathione + NADH + H+." [EC:1.2.1.1]
GO:0004328
name: formamidase activity
def: "Catalysis of the reaction: formamide + H(2)O = formate + NH(4)(+)." [EC:3.5.1.49, RHEA:21951]
GO:0004329
name: formate-tetrahydrofolate ligase activity
def: "Catalysis of the reaction: ATP + formate + tetrahydrofolate = ADP + phosphate + 10-formyltetrahydrofolate." [EC:6.3.4.3]
GO:0004331
name: fructose-2,6-bisphosphate 2-phosphatase activity
def: "Catalysis of the reaction: D-fructose 2,6-bisphosphate + H2O = D-fructose-6-phosphate + phosphate." [EC:3.1.3.46]
GO:0004332
name: fructose-bisphosphate aldolase activity
def: "Catalysis of the reaction: D-fructose 1,6-bisphosphate = glycerone phosphate + D-glyceraldehyde-3-phosphate." [EC:4.1.2.13]
GO:0004333
name: fumarate hydratase activity
def: "Catalysis of the reaction: (S)-malate = fumarate + H(2)O." [EC:4.2.1.2, RHEA:12463]
GO:0004334
name: fumarylacetoacetase activity
def: "Catalysis of the reaction: 4-fumarylacetoacetate + H(2)O = acetoacetate + fumarate + H(+)." [EC:3.7.1.2, RHEA:10247]
GO:0004335
name: galactokinase activity
def: "Catalysis of the reaction: D-galactose + ATP = alpha-D-galactose 1-phosphate + ADP + 2 H(+)." [EC:2.7.1.6, RHEA:13556]
GO:0004336
name: galactosylceramidase activity
def: "Catalysis of the reaction: D-galactosyl-N-acylsphingosine + H2O = D-galactose + N-acylsphingosine." [EC:3.2.1.46]
GO:0004337
name: geranyltranstransferase activity
def: "Catalysis of the reaction: geranyl diphosphate + isopentenyl diphosphate = 2-trans,6-trans-farnesyl diphosphate + diphosphate." [EC:2.5.1.10, RHEA:19364]
GO:0004338
name: glucan exo-1,3-beta-glucosidase activity
def: "Catalysis of the successive hydrolysis of beta-D-glucose units from the non-reducing ends of (1->3)-beta-D-glucans, releasing alpha-glucose." [EC:3.2.1.58]
GO:0004339
name: glucan 1,4-alpha-glucosidase activity
def: "Catalysis of the hydrolysis of terminal (1->4)-linked alpha-D-glucose residues successively from non-reducing ends of the chains with release of beta-D-glucose." [EC:3.2.1.3]
GO:0004340
name: glucokinase activity
def: "Catalysis of the reaction: ATP + D-glucose = ADP + D-glucose-6-phosphate." [EC:2.7.1.2]
GO:0004341
name: gluconolactonase activity
def: "Catalysis of the reaction: D-glucono-1,5-lactone + H2O = D-gluconate." [EC:3.1.1.17]
GO:0004342
name: glucosamine-6-phosphate deaminase activity
def: "Catalysis of the reaction: D-glucosamine 6-phosphate + H(2)O = beta-D-fructose 6-phosphate + NH(4)(+)." [EC:3.5.99.6, RHEA:12175]
GO:0004343
name: glucosamine 6-phosphate N-acetyltransferase activity
def: "Catalysis of the reaction: D-glucosamine 6-phosphate + acetyl-CoA = N-acetyl-D-glucosamine 6-phosphate + CoA + H(+)." [EC:2.3.1.4, RHEA:10295]
GO:0004344
name: glucose dehydrogenase activity
def: "Catalysis of the reaction: D-glucose + acceptor = D-glucono-1,5-lactone + reduced acceptor." [EC:1.1.99.10]
GO:0004345
name: glucose-6-phosphate dehydrogenase activity
def: "Catalysis of the reaction: D-glucose 6-phosphate + NADP+ = D-glucono-1,5-lactone 6-phosphate + NADPH + H+." [EC:1.1.1.49]
GO:0004346
name: glucose-6-phosphatase activity
def: "Catalysis of the reaction: D-glucopyranose 6-phosphate + H2O = D-glucose + phosphate. D-glucopyranose is also known as D-glucose 6-phosphate." [EC:3.1.3.9, RHEA:16692]
GO:0004347
name: glucose-6-phosphate isomerase activity
def: "Catalysis of the reaction: D-glucose 6-phosphate = D-fructose 6-phosphate." [EC:5.3.1.9]
GO:0004348
name: glucosylceramidase activity
def: "Catalysis of the reaction: D-glucosyl-N-acylsphingosine + H2O = D-glucose + N-acylsphingosine." [EC:3.2.1.45]
GO:0004349
name: glutamate 5-kinase activity
def: "Catalysis of the reaction: L-glutamate + ATP = L-glutamyl 5-phosphate + ADP + H(+)." [EC:2.7.2.11, RHEA:14880]
GO:0004350
name: glutamate-5-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: L-glutamate 5-semialdehyde + NADP(+) + phosphate = L-glutamyl 5-phosphate + H(+) + NADPH." [EC:1.2.1.41, RHEA:19544]
GO:0004351
name: glutamate decarboxylase activity
def: "Catalysis of the reaction: L-glutamate = 4-aminobutanoate + CO2." [EC:4.1.1.15]
GO:0004352
name: glutamate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: L-glutamate + H2O + NAD+ = 2-oxoglutarate + NH3 + NADH + H+." [EC:1.4.1.2]
GO:0004353
name: glutamate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: L-glutamate + H2O + NAD(P)+ = 2-oxoglutarate + NH3 + NAD(P)H + H+." [EC:1.4.1.3]
GO:0004354
name: glutamate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: L-glutamate + H2O + NADP+ = 2-oxoglutarate + NH3 + NADPH + H+." [EC:1.4.1.4]
GO:0004355
name: glutamate synthase (NADPH) activity
def: "Catalysis of the reaction: 2 L-glutamate + NADP(+) = 2-oxoglutarate + L-glutamine + H(+) + NADPH. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + NADP+ + H2O = NH3 + 2-oxoglutarate + NADPH + H+." [EC:1.4.1.13, RHEA:15504]
GO:0004356
name: glutamate-ammonia ligase activity
def: "Catalysis of the reaction: L-glutamate + ATP + NH(4)(+) = L-glutamine + ADP + 2 H(+) + phosphate." [EC:6.3.1.2, RHEA:16172]
GO:0004357
name: glutamate-cysteine ligase activity
def: "Catalysis of the reaction: L-cysteine + L-glutamate + ATP = L-gamma-glutamyl-L-cysteine + ADP + 2 H(+) + phosphate." [EC:6.3.2.2, RHEA:13288]
GO:0004358
name: glutamate N-acetyltransferase activity
def: "Catalysis of the reaction: N(2)-acetyl-L-ornithine + L-glutamate = N-acetyl-L-glutamate + L-ornithine." [EC:2.3.1.35, RHEA:15352]
GO:0004359
name: glutaminase activity
def: "Catalysis of the reaction: L-glutamine + H2O = L-glutamate + NH3." [EC:3.5.1.2]
GO:0004360
name: glutamine-fructose-6-phosphate transaminase (isomerizing) activity
def: "Catalysis of the reaction: beta-D-fructose 6-phosphate + L-glutamine = D-glucosamine 6-phosphate + L-glutamate." [EC:2.6.1.16, RHEA:13240]
GO:0004361
name: glutaryl-CoA dehydrogenase activity
def: "Catalysis of the reaction: glutaryl-CoA + acceptor = crotonoyl-CoA + CO2 + reduced acceptor." [EC:1.3.99.7]
GO:0004362
name: glutathione-disulfide reductase activity
def: "Catalysis of the reaction: 2 glutathione + NADP+ = glutathione disulfide + NADPH + H+." [EC:1.8.1.7, ISBN:0198506732]
GO:0004363
name: glutathione synthase activity
def: "Catalysis of the reaction: L-gamma-glutamyl-L-cysteine + ATP + glycine = ADP + glutathione + 2 H(+) + phosphate." [EC:6.3.2.3, RHEA:13560]
GO:0004364
name: glutathione transferase activity
def: "Catalysis of the reaction: R-X + glutathione = H-X + R-S-glutathione. R may be an aliphatic, aromatic or heterocyclic group; X may be a sulfate, nitrile or halide group." [EC:2.5.1.18]
GO:0004365
name: glyceraldehyde-3-phosphate dehydrogenase (NAD+) (phosphorylating) activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + phosphate + NAD+ = 3-phospho-D-glyceroyl phosphate + NADH + H+." [EC:1.2.1.12]
GO:0004366
name: glycerol-3-phosphate O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + sn-glycerol 3-phosphate = CoA + 1-acyl-sn-glycerol 3-phosphate." [EC:2.3.1.15]
GO:0004367
name: glycerol-3-phosphate dehydrogenase [NAD+] activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + NAD+ = glycerone phosphate + NADH + H+." [EC:1.1.1.8, EC:1.1.1.94]
GO:0004368
name: glycerol-3-phosphate dehydrogenase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + a quinone = glycerone phosphate + a quinol." [EC:1.1.5.3]
GO:0004369
name: glycerol-3-phosphate oxidase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + O(2) = glycerone phosphate + H(2)O(2)." [EC:1.1.3.21, RHEA:18372]
GO:0004370
name: glycerol kinase activity
def: "Catalysis of the reaction: ATP + glycerol = sn-glycerol 3-phosphate + ADP + 2 H(+)." [EC:2.7.1.30, RHEA:21647]
GO:0004371
name: glycerone kinase activity
def: "Catalysis of the reaction: ATP + glycerone = ADP + glycerone phosphate + 2 H(+)." [EC:2.7.1.29, RHEA:15776]
GO:0004372
name: glycine hydroxymethyltransferase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + glycine + H2O = tetrahydrofolate + L-serine." [EC:2.1.2.1]
GO:0004373
name: glycogen (starch) synthase activity
def: "Catalysis of the reaction: UDP-glucose + (1,4)-alpha-D-glucosyl(n) = UDP + (1,4)-alpha-D-glucosyl(n+1)." [EC:2.4.1.11]
GO:0004374
name: glycine cleavage system
def: "OBSOLETE. Catalysis of the reactions: glycine + lipoylprotein = S-aminomethyldihydrolipoylprotein + carbon dioxide (CO2), followed by S-aminomethyldihydrolipoylprotein + (6S)-tetrahydrofolate = dihydrolipoylprotein + (6R)-5,10-methylenetetrahydrofolate + ammonia. Made up of two components, aminomethyltransferase and glycine dehydrogenase (decarboxylating)." [EC:1.4.4.2, EC:2.1.2.10]
GO:0004375
name: glycine dehydrogenase (decarboxylating) activity
def: "Catalysis of the reaction: glycine + lipoylprotein = S-aminomethyldihydrolipoylprotein + CO2." [EC:1.4.4.2]
GO:0004376
name: glycolipid mannosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-mannosyl residue from GDP-mannose into lipid-linked oligosaccharide, forming an alpha-D-mannosyl-D-mannose linkage." [GOC:ai]
GO:0004377
name: GDP-Man:Man3GlcNAc2-PP-Dol alpha-1,2-mannosyltransferase activity
def: "Catalysis of the reaction: an alpha-D-Man-(1->3)-[alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + 2 GDP-alpha-D-mannose = an alpha-D-Man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-[alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + 2 GDP + 2 H+. This reaction is the transfer of an alpha-D-mannosyl residue from GDP-mannose into lipid-linked oligosaccharide, forming an alpha-(1->2)-D-mannosyl-D-mannose linkage." [EC:2.4.1.131]
GO:0004378
name: GDP-Man:Man1GlcNAc2-PP-Dol alpha-1,3-mannosyltransferase activity
def: "Catalysis of the reaction: beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + GDP-alpha-D-mannose = alpha-D-Man-(1->3)-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + GDP + H+. This reaction is the transfer of an alpha-D-mannosyl residue from GDP-mannose into lipid-linked oligosaccharide, forming an alpha-(1->3)-D-mannosyl-D-mannose linkage." [EC:2.4.1.132]
GO:0004379
name: glycylpeptide N-tetradecanoyltransferase activity
def: "Catalysis of the reaction: tetradecanoyl-CoA + glycyl-peptide = CoA + N-tetradecanoylglycyl-peptide." [EC:2.3.1.97]
GO:0004380
name: glycoprotein-fucosylgalactoside alpha-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-galactosamine + glycoprotein-alpha-L-fucosyl-(1,2)-D-galactose = UDP + glycoprotein-N-acetyl-alpha-D-galactosaminyl-(1,3)-(alpha-L-fucosyl-(1,2))-D-galactose." [EC:2.4.1.40]
GO:0004381
name: fucosylgalactoside 3-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + glycoprotein-alpha-L-fucosyl-(1,2)-D-galactose = UDP + glycoprotein-alpha-D-galactosyl-(1,3)-(alpha-L-fucosyl-(1,2))-D-galactose." [EC:2.4.1.37]
GO:0004382
name: guanosine-diphosphatase activity
def: "Catalysis of the reaction: GDP + H2O = GMP + phosphate." [EC:3.6.1.42, PMID:2989286]
GO:0004383
name: guanylate cyclase activity
def: "Catalysis of the reaction: GTP = 3',5'-cyclic GMP + diphosphate." [EC:4.6.1.2]
GO:0004384
name: membrane-associated guanylate kinase
def: "OBSOLETE. Catalysis of the reaction: ATP + GMP = ADP + GDP, associated with the cell membrane." [EC:2.7.4.8]
GO:0004385
name: guanylate kinase activity
def: "Catalysis of the reaction: ATP + GMP = ADP + GDP." [EC:2.7.4.8]
GO:0004386
name: helicase activity
def: "Catalysis of the reaction: NTP + H2O = NDP + phosphate, to drive the unwinding of a DNA or RNA helix." [GOC:mah, ISBN:0198506732]
GO:0004392
name: heme oxygenase (decyclizing) activity
def: "Catalysis of the reaction: heme + 3 donor-H2 + 3 O2 = biliverdin + Fe2+ + CO + 3 acceptor + 3 H2O." [EC:1.14.99.3]
GO:0004394
name: heparan sulfate 2-O-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + heparan sulfate = adenosine 3',5'-bisphosphate + heparan sulfate 2-O-sulfate; results in 2-O-sulfation of iduronic acid residues in heparan sulfate." [PMID:9153262]
GO:0004395
name: hexaprenyldihydroxybenzoate methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-hexaprenyl-4,5-dihydroxybenzoate = S-adenosyl-L-homocysteine + 3-hexaprenyl-4-hydroxy-5-methoxybenzoate." [EC:2.1.1.114]
GO:0004396
name: hexokinase activity
def: "Catalysis of the reaction: ATP + D-hexose = ADP + D-hexose 6-phosphate." [EC:2.7.1.1]
GO:0004397
name: histidine ammonia-lyase activity
def: "Catalysis of the reaction: L-histidine = urocanate + NH3." [EC:4.3.1.3]
GO:0004398
name: histidine decarboxylase activity
def: "Catalysis of the reaction: L-histidine = histamine + CO2." [EC:4.1.1.22]
GO:0004399
name: histidinol dehydrogenase activity
def: "Catalysis of the reaction: L-histidinol + NAD+ = L-histidine + NADH + H+." [EC:1.1.1.23]
GO:0004400
name: histidinol-phosphate transaminase activity
def: "Catalysis of the reaction: L-histidinol-phosphate + 2-oxoglutarate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate." [EC:2.6.1.9]
GO:0004401
name: histidinol-phosphatase activity
def: "Catalysis of the reaction: L-histidinol phosphate + H(2)O = L-histidinol + phosphate." [EC:3.1.3.15, RHEA:14468]
GO:0004402
name: histone acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + histone = CoA + acetyl-histone." [EC:2.3.1.48]
GO:0004407
name: histone deacetylase activity
def: "Catalysis of the reaction: histone N6-acetyl-L-lysine + H2O = histone L-lysine + acetate. This reaction represents the removal of an acetyl group from a histone, a class of proteins complexed to DNA in chromatin and chromosomes." [EC:3.5.1.-, PMID:9893272]
GO:0004408
name: holocytochrome-c synthase activity
def: "Catalysis of the reaction: holocytochrome c = apocytochrome c + heme." [EC:4.4.1.17]
GO:0004409
name: homoaconitate hydratase activity
def: "Catalysis of the reaction: (-)-homoisocitrate = cis-homoaconitate + H(2)O." [EC:4.2.1.36, RHEA:15488]
GO:0004410
name: homocitrate synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + acetyl-CoA + H(2)O = CoA + H(+) + homocitrate." [EC:2.3.3.14, RHEA:12932]
GO:0004411
name: homogentisate 1,2-dioxygenase activity
def: "Catalysis of the reaction: homogentisate + O(2) = 4-maleylacetoacetate + H(+)." [EC:1.13.11.5, RHEA:15452]
GO:0004412
name: homoserine dehydrogenase activity
def: "Catalysis of the reaction: L-homoserine + NADP+ = L-aspartate-4-semialdehyde + NADPH + H+." [EC:1.1.1.3]
GO:0004413
name: homoserine kinase activity
def: "Catalysis of the reaction: L-homoserine + ATP = O-phospho-L-homoserine + ADP + 2 H(+)." [EC:2.7.1.39, RHEA:13988]
GO:0004414
name: homoserine O-acetyltransferase activity
def: "Catalysis of the reaction: L-homoserine + acetyl-CoA = O-acetyl-L-homoserine + CoA." [EC:2.3.1.31, RHEA:13704]
GO:0004415
name: hyalurononglucosaminidase activity
def: "Catalysis of the random hydrolysis of (1->4) linkages between N-acetyl-beta-D-glucosamine and D-glucuronate residues in hyaluronate." [EC:3.2.1.35]
GO:0004416
name: hydroxyacylglutathione hydrolase activity
def: "Catalysis of the reaction: (S)-(2-hydroxyacyl)glutathione + H2O = glutathione + a 2-hydroxy carboxylate." [EC:3.1.2.6]
GO:0004417
name: hydroxyethylthiazole kinase activity
def: "Catalysis of the reaction: 5-(2-hydroxyethyl)-4-methylthiazole + ATP = 4-methyl-5-(2-phosphoethyl)-thiazole + ADP + 2 H(+)." [EC:2.7.1.50, RHEA:24215]
GO:0004418
name: hydroxymethylbilane synthase activity
def: "Catalysis of the reaction: H(2)O + 4 porphobilinogen = hydroxymethylbilane + 4 NH(4)(+)." [EC:2.5.1.61, RHEA:13188]
GO:0004419
name: hydroxymethylglutaryl-CoA lyase activity
def: "Catalysis of the reaction: (S)-3-hydroxy-3-methylglutaryl-CoA = acetoacetate + acetyl-CoA." [EC:4.1.3.4, RHEA:24407]
GO:0004420
name: hydroxymethylglutaryl-CoA reductase (NADPH) activity
def: "Catalysis of the reaction: (R)-mevalonate + CoA + 2 NADP(+) = (S)-3-hydroxy-3-methylglutaryl-CoA + 2 H(+) + 2 NADPH." [EC:1.1.1.34, RHEA:15992]
GO:0004421
name: hydroxymethylglutaryl-CoA synthase activity
def: "Catalysis of the reaction: acetoacetyl-CoA + acetyl-CoA + H(2)O = (S)-3-hydroxy-3-methylglutaryl-CoA + CoA + H(+)." [EC:2.3.3.10, RHEA:10191]
GO:0004422
name: hypoxanthine phosphoribosyltransferase activity
def: "Catalysis of the reaction: IMP + diphosphate = hypoxanthine + 5-phospho-alpha-D-ribose 1-diphosphate." [EC:2.4.2.8, GOC:curators]
GO:0004423
name: iduronate-2-sulfatase activity
def: "Catalysis of the hydrolysis of the 2-sulfate groups of the L-iduronate 2-sulfate units of dermatan sulfate, heparan sulfate and heparin." [EC:3.1.6.13]
GO:0004424
name: imidazoleglycerol-phosphate dehydratase activity
def: "Catalysis of the reaction: D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + H(2)O." [EC:4.2.1.19, RHEA:11043]
GO:0004425
name: indole-3-glycerol-phosphate synthase activity
def: "Catalysis of the reaction: 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate = 1-(indol-3-yl)glycerol 3-phosphate + CO2 + H2O." [EC:4.1.1.48]
GO:0004427
name: inorganic diphosphatase activity
def: "Catalysis of the reaction: diphosphate + H(2)O = H(+) + 2 phosphate." [EC:3.6.1.1, RHEA:24579]
GO:0004428
name: inositol or phosphatidylinositol kinase activity
def: "OBSOLETE. Catalysis of the phosphorylation of myo-inositol (1,2,3,5/4,6-cyclohexanehexol) or a phosphatidylinositol." [GOC:hb]
GO:0004430
name: 1-phosphatidylinositol 4-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol + ATP = 1-phosphatidyl-1D-myo-inositol 4-phosphate + ADP + 2 H(+)." [EC:2.7.1.67, RHEA:19880]
GO:0004432
name: 1-phosphatidylinositol-4-phosphate kinase, class IA
def: "OBSOLETE. A class I PI3K activated by tyrosine phosphorylation events." [PMID:11050418]
GO:0004433
name: 1-phosphatidylinositol-4-phosphate kinase, class IB
def: "OBSOLETE. A class I PI3K activated via heterotrimeric G-proteins." [PMID:11050418]
GO:0004435
name: phosphatidylinositol phospholipase C activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1,2-diacylglycerol + 1D-myo-inositol 1,4,5-trisphosphate + H(+)." [EC:3.1.4.11, RHEA:23915]
GO:0004436
name: phosphatidylinositol diacylglycerol-lyase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol = D-myo-inositol 1,2-cyclic phosphate + diacylglycerol." [EC:4.6.1.13]
GO:0004437
name: inositol or phosphatidylinositol phosphatase activity
def: "OBSOLETE. Catalysis of the removal of a phosphate group from phosphorylated myo-inositol (1,2,3,5/4,6-cyclohexanehexol) or a phosphatidylinositol." [GOC:hb]
GO:0004438
name: phosphatidylinositol-3-phosphatase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 3-phosphate + H2O = 1-phosphatidyl-1D-myo-inositol + phosphate." [EC:3.1.3.64]
GO:0004439
name: phosphatidylinositol-4,5-bisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 4-phosphate + phosphate." [EC:3.1.3.36, RHEA:22767]
GO:0004441
name: inositol-1,4-bisphosphate 1-phosphatase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,4-bisphosphate + H2O = 1D-myo-inositol 4-phosphate + phosphate." [EC:3.1.3.57, GOC:hb]
GO:0004442
name: inositol-1,4,-bisphosphate 3-phosphatase
def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 1,3-bisphosphate + H2O = D-myo-inositol 1-monophosphate + phosphate." [EC:3.1.3.65, GOC:go_curators]
GO:0004443
name: inositol-1,4,-bisphosphate 4-phosphatase
def: "OBSOLETE. Catalysis of the reaction: D-myo-inositol 3,4-bisphosphate + H2O = D-myo-inositol 3-monophosphate + phosphate." [EC:3.1.3.66, GOC:go_curators]
GO:0004444
name: inositol-1,4,5-trisphosphate 1-phosphatase
def: "OBSOLETE. The removal of a phosphate group from the carbon-1 position of D-myo-inositol 1,4,5-trisphosphate." [EC:3.1.3.61, GOC:hb]
GO:0004445
name: inositol-polyphosphate 5-phosphatase activity
def: "Catalysis of the reactions: D-myo-inositol 1,4,5-trisphosphate + H2O = myo-inositol 1,4-bisphosphate + phosphate, and 1D-myo-inositol 1,3,4,5-tetrakisphosphate + H2O = 1D-myo-inositol 1,3,4-trisphosphate + phosphate." [EC:3.1.3.56]
GO:0004446
name: inositol-hexakisphosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = myo-inositol pentakisphosphate (mixed isomers) + phosphate." [EC:3.1.3.62]
GO:0004447
name: iodide peroxidase activity
def: "Catalysis of the reaction: iodide + hydrogen peroxide = iodine + 2 H2O." [EC:1.11.1.8]
GO:0004448
name: isocitrate dehydrogenase activity
def: "Catalysis of the reaction: isocitrate + acceptor = 2-oxoglutarate + CO2 + reduced acceptor." [EC:1.1.1.41, EC:1.1.1.42]
GO:0004449
name: isocitrate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: isocitrate + NAD+ = 2-oxoglutarate + CO2 + NADH + H+." [EC:1.1.1.41]
GO:0004450
name: isocitrate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: isocitrate + NADP+ = 2-oxoglutarate + CO2 + NADPH + H+." [EC:1.1.1.42]
GO:0004451
name: isocitrate lyase activity
def: "Catalysis of the reaction: isocitrate = glyoxylate + succinate." [EC:4.1.3.1, RHEA:13248]
GO:0004452
name: isopentenyl-diphosphate delta-isomerase activity
def: "Catalysis of the reaction: isopentenyl diphosphate = dimethylallyl diphosphate." [EC:5.3.3.2, RHEA:23287]
GO:0004453
name: juvenile-hormone esterase activity
def: "Catalysis of the reaction: methyl (2E,6E)-(10R,11S)-10,11-epoxy-3,7,11-trimethyltrideca-2,6-dienoate + H2O = (2E,6E)-(10R,11S)-10,11-epoxy-3,7,11-trimethyltrideca-2,6-dienoate + methanol. A carboxylesterase that hydrolyzes the ester linkage of juvenile hormone." [EC:3.1.1.59, EMBL:AF304352]
GO:0004454
name: ketohexokinase activity
def: "Catalysis of the reaction: ATP + D-fructose = ADP + D-fructose 1-phosphate." [EC:2.7.1.3]
GO:0004455
name: ketol-acid reductoisomerase activity
def: "Catalysis of the reaction: (R)-2,3-dihydroxy-3-methylbutanoate + NADP+ = (S)-2-hydroxy-2-methyl-3-oxobutanoate + NADPH + H+." [EC:1.1.1.86]
GO:0004456
name: phosphogluconate dehydratase activity
def: "Catalysis of the reaction: 6-phospho-D-gluconate = 2-dehydro-3-deoxy-6-phospho-D-gluconate + H(2)O." [EC:4.2.1.12, RHEA:17280]
GO:0004457
name: lactate dehydrogenase activity
def: "Catalysis of the reaction: lactate + NAD+ = H+ + NADH + pyruvate." [GOC:ai, GOC:bf]
GO:0004458
name: D-lactate dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: (R)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c." [EC:1.1.2.4]
GO:0004459
name: L-lactate dehydrogenase activity
def: "Catalysis of the reaction: (S)-lactate + NAD+ = pyruvate + NADH + H+." [EC:1.1.1.27, RHEA:23447]
GO:0004460
name: L-lactate dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: (S)-lactate + 2 ferricytochrome c = pyruvate + 2 ferrocytochrome c." [EC:1.1.2.3]
GO:0004461
name: lactose synthase activity
def: "Catalysis of the reaction: UDP-galactose + D-glucose = UDP + lactose." [EC:2.4.1.22]
GO:0004462
name: lactoylglutathione lyase activity
def: "Catalysis of the reaction: (R)-S-lactoylglutathione = glutathione + methylglyoxal." [EC:4.4.1.5, RHEA:19072]
GO:0004463
name: leukotriene-A4 hydrolase activity
def: "Catalysis of the reaction: H(2)O + leukotriene A(4) = leukotriene B(4)." [EC:3.3.2.6, RHEA:22327]
GO:0004464
name: leukotriene-C4 synthase activity
def: "Catalysis of the reaction: leukotriene C(4) = glutathione + leukotriene A(4)." [EC:4.4.1.20, RHEA:17620]
GO:0004465
name: lipoprotein lipase activity
def: "Catalysis of the reaction: triacylglycerol + H2O = diacylglycerol + a carboxylate, where the triacylglycerol is part of a lipoprotein." [EC:3.1.1.34, GOC:bf]
GO:0004466
name: long-chain-acyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: acyl-CoA + ETF = 2,3-dehydroacyl-CoA + reduced ETF." [EC:1.3.99.13]
GO:0004467
name: long-chain fatty acid-CoA ligase activity
def: "Catalysis of the reaction: ATP + a long-chain carboxylic acid + CoA = AMP + diphosphate + an acyl-CoA; a long-chain fatty acid is a fatty acid with a chain length between C13 and C22." [CHEBI:15904, EC:6.2.1.3]
GO:0004468
name: lysine N-acetyltransferase activity, acting on acetyl phosphate as donor
def: "Catalysis of the reaction: acetyl phosphate + L-lysine = phosphate + N6-acetyl-L-lysine." [EC:2.3.1.32]
GO:0004470
name: malic enzyme activity
def: "Catalysis of the oxidative decarboxylation of malate with the concomitant production of pyruvate." [ISBN:0198506732]
GO:0004471
name: malate dehydrogenase (decarboxylating) (NAD+) activity
def: "Catalysis of the reaction: (S)-malate + NAD+ = pyruvate + CO2 + NADH + H+." [EC:1.1.1.38, EC:1.1.1.39]
GO:0004473
name: malate dehydrogenase (decarboxylating) (NADP+) activity
def: "Catalysis of the reaction: (S)-malate + NADP+ = pyruvate + CO2 + NADPH + H+." [EC:1.1.1.40]
GO:0004474
name: malate synthase activity
def: "Catalysis of the reaction: acetyl-CoA + glyoxylate + H(2)O = (S)-malate + CoA + H(+)." [EC:2.3.3.9, RHEA:18184]
GO:0004475
name: mannose-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: alpha-D-mannose 1-phosphate + GTP = diphosphate + GDP-alpha-D-mannose." [EC:2.7.7.13, RHEA:15232]
GO:0004476
name: mannose-6-phosphate isomerase activity
def: "Catalysis of the reaction: D-mannose 6-phosphate = D-fructose 6-phosphate." [EC:5.3.1.8]
GO:0004477
name: methenyltetrahydrofolate cyclohydrolase activity
def: "Catalysis of the reaction: 5,10-methenyltetrahydrofolate + H2O = 10-formyltetrahydrofolate." [EC:3.5.4.9]
GO:0004478
name: methionine adenosyltransferase activity
def: "Catalysis of the reaction: ATP + L-methionine + H2O = phosphate + diphosphate + S-adenosyl-L-methionine." [EC:2.5.1.6]
GO:0004479
name: methionyl-tRNA formyltransferase activity
def: "Catalysis of the reaction: 10-formyltetrahydrofolate + L-methionyl-tRNA + H2O = tetrahydrofolate + N-formylmethionyl-tRNA." [EC:2.1.2.9]
GO:0004481
name: methylene-fatty-acyl-phospholipid synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phospholipid olefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid methylene fatty acid." [EC:2.1.1.16]
GO:0004482
name: mRNA (guanine-N7-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + G(5')pppR-RNA = S-adenosyl-L-homocysteine + m7G(5')pppR-RNA. m7G(5')pppR-RNA is mRNA containing an N7-methylguanine cap; R may be guanosine or adenosine." [EC:2.1.1.56]
GO:0004483
name: mRNA (nucleoside-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + m7G(5')pppR-RNA = S-adenosyl-L-homocysteine + m7G(5')pppRm-RNA. R may be guanosine or adenosine." [EC:2.1.1.57]
GO:0004484
name: mRNA guanylyltransferase activity
def: "Catalysis of the reaction: GTP + (5')pp-Pur-mRNA = diphosphate + G(5')ppp-Pur-mRNA; G(5')ppp-Pur-mRNA is mRNA containing a guanosine residue linked 5' through three phosphates to the 5' position of the terminal residue." [EC:2.7.7.50]
GO:0004485
name: methylcrotonoyl-CoA carboxylase activity
def: "Catalysis of the reaction: 3-methylbut-2-enoyl-CoA + ATP + bicarbonate = trans-3-methylglutaconyl-CoA + ADP + 2 H(+) + phosphate." [EC:6.4.1.4, RHEA:13592]
GO:0004486
name: methylenetetrahydrofolate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + NAD(P)+ = 5,10-methenyltetrahydrofolate + NAD(P)H + H+." [EC:1.5.1.-, EC:1.5.1.15]
GO:0004487
name: methylenetetrahydrofolate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + NAD(+) = 5,10-methenyltetrahydrofolate + NADH." [EC:1.5.1.15, RHEA:22895]
GO:0004488
name: methylenetetrahydrofolate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + NADP(+) = 5,10-methenyltetrahydrofolate + NADPH." [EC:1.5.1.5, RHEA:22815]
GO:0004489
name: methylenetetrahydrofolate reductase (NAD(P)H) activity
def: "Catalysis of the reaction: 5-methyltetrahydrofolate + NAD(P)+ = 5,10-methylenetetrahydrofolate + NAD(P)H + H+." [EC:1.5.1.20]
GO:0004490
name: methylglutaconyl-CoA hydratase activity
def: "Catalysis of the reaction: (S)-3-hydroxy-3-methylglutaryl-CoA = trans-3-methylglutaconyl-CoA + H(2)O." [EC:4.2.1.18, RHEA:21539]
GO:0004491
name: methylmalonate-semialdehyde dehydrogenase (acylating) activity
def: "Catalysis of the reaction: 2-methyl-3-oxopropanoate + CoA + NAD+ = propanoyl-CoA + CO2 + NADH + H+." [EC:1.2.1.27]
GO:0004492
name: methylmalonyl-CoA decarboxylase activity
def: "Catalysis of the reaction: (S)-2-methyl-3-oxopropanoyl-CoA = propanoyl-CoA + CO2." [EC:4.1.1.41]
GO:0004493
name: methylmalonyl-CoA epimerase activity
def: "Catalysis of the reaction: (R)-methylmalonyl-CoA = (S)-methylmalonyl-CoA." [EC:5.1.99.1, RHEA:20556]
GO:0004494
name: methylmalonyl-CoA mutase activity
def: "Catalysis of the reaction: (R)-methylmalonyl-CoA = succinyl-CoA." [EC:5.4.99.2, RHEA:22891]
GO:0004495
name: mevaldate reductase activity
def: "Catalysis of the reaction: (R)-mevalonate + acceptor = mevaldate + reduced acceptor." [EC:1.1.1.32, EC:1.1.1.33]
GO:0004496
name: mevalonate kinase activity
def: "Catalysis of the reaction: (R)-mevalonate + ATP = (R)-5-phosphomevalonate + ADP + 2 H(+)." [EC:2.7.1.36, RHEA:17068]
GO:0004497
name: monooxygenase activity
def: "Catalysis of the incorporation of one atom from molecular oxygen into a compound and the reduction of the other atom of oxygen to water." [http://www.onelook.com/, ISBN:0198506732]
GO:0004498
name: calcidiol 1-monooxygenase activity
def: "Catalysis of the reaction: calcidiol + H(+) + NADPH + O(2) = calcitriol + H(2)O + NADP(+)." [EC:1.14.13.13, RHEA:20576]
GO:0004499
name: N,N-dimethylaniline monooxygenase activity
def: "Catalysis of the reaction: N,N-dimethylaniline + NADPH + H+ + O2 = N,N-dimethylaniline N-oxide + NADP+ + H2O." [EC:1.14.13.8]
GO:0004500
name: dopamine beta-monooxygenase activity
def: "Catalysis of the reaction: L-ascorbate + dopamine + O(2) = (R)-noradrenaline + dehydroascorbate + H(2)O." [EC:1.14.17.1, RHEA:19120]
GO:0004501
name: ecdysone 20-monooxygenase activity
def: "Catalysis of the reaction: AH(2) + Ecdysone + O(2) = 20-hydroxyecdysone + A + H(2)O." [EC:1.14.99.22, RHEA:14024]
GO:0004502
name: kynurenine 3-monooxygenase activity
def: "Catalysis of the reaction: L-kynurenine + H(+) + NADPH + O(2) = 3-hydroxy-L-kynurenine + H(2)O + NADP(+)." [EC:1.14.13.9, RHEA:20548]
GO:0004503
name: monophenol monooxygenase activity
def: "Catalysis of the reaction: L-tyrosine + L-DOPA + O2 = L-DOPA + DOPAquinone + H2O." [EC:1.14.18.1, GOC:bf, GOC:sart, PMID:4965136]
GO:0004504
name: peptidylglycine monooxygenase activity
def: "Catalysis of the reaction: peptidyl-glycine + ascorbate + O2 = peptidyl(2-hydroxyglycine) + dehydroascorbate + H2O." [EC:1.14.17.3]
GO:0004505
name: phenylalanine 4-monooxygenase activity
def: "Catalysis of the reaction: L-phenylalanine + tetrahydrobiopterin + O2 = L-tyrosine + 4-alpha-hydroxytetrahydrobiopterin." [EC:1.14.16.1]
GO:0004506
name: squalene monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + squalene = (S)-2,3-epoxysqualene + H(2)O + NADP(+)." [RHEA:25285]
GO:0004507
name: steroid 11-beta-monooxygenase activity
def: "Catalysis of the reaction: a steroid + reduced adrenal ferredoxin + O2 = an 11-beta-hydroxysteroid + oxidized adrenal ferredoxin + H2O." [EC:1.14.15.4]
GO:0004508
name: steroid 17-alpha-monooxygenase activity
def: "Catalysis of the reaction: a steroid + AH2 + O2 = a 17a-hydroxysteroid + A + H2O." [EC:1.14.99.9]
GO:0004509
name: steroid 21-monooxygenase activity
def: "Catalysis of the reaction: a steroid + donor-H2 + O2 = a 21-hydroxysteroid + acceptor + H2O." [EC:1.14.99.10]
GO:0004510
name: tryptophan 5-monooxygenase activity
def: "Catalysis of the reaction: L-tryptophan + tetrahydrobiopterin + O2 = 5-hydroxy-L-tryptophan + 4-alpha-hydroxytetrahydrobiopterin + H2O." [EC:1.14.16.4]
GO:0004511
name: tyrosine 3-monooxygenase activity
def: "Catalysis of the reaction: L-tyrosine + tetrahydrobiopterin + O2 = 3,4-dihydroxy-L-phenylalanine + 4-alpha-hydroxytetrahydrobiopterin + H2O." [EC:1.14.16.2]
GO:0004512
name: inositol-3-phosphate synthase activity
def: "Catalysis of the reaction: D-glucose 6-phosphate = 1D-myo-inositol 3-phosphate. This reaction requires NAD, which dehydrogenates the CHOH group to CO at C-5 of the glucose 6-phosphate, making C-6 into an active methylene, able to condense with the aldehyde at C-1. Finally, the enzyme-bound NADH reconverts C-5 into the CHOH form." [EC:5.5.1.4, RHEA:10719]
GO:0004513
name: neolactotetraosylceramide alpha-2,3-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide = CMP + alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-D-glucosylceramide." [EC:2.4.99.10]
GO:0004514
name: nicotinate-nucleotide diphosphorylase (carboxylating) activity
def: "Catalysis of the reaction: CO(2) + diphosphate + nicotinate D-ribonucleotide = 5-phospho-alpha-D-ribose 1-diphosphate + 2 H(+) + quinolinate." [EC:2.4.2.19, RHEA:12736]
GO:0004515
name: nicotinate-nucleotide adenylyltransferase activity
def: "Catalysis of the reaction: ATP + nicotinate ribonucleotide = diphosphate + deamido-NAD+." [EC:2.7.7.18]
GO:0004516
name: nicotinate phosphoribosyltransferase activity
def: "Catalysis of the reaction: diphosphate + nicotinate D-ribonucleotide = 5-phospho-alpha-D-ribose 1-diphosphate + H(+) + nicotinate." [EC:2.4.2.11, RHEA:13396]
GO:0004517
name: nitric-oxide synthase activity
def: "Catalysis of the reaction: L-arginine + n NADPH + n H+ + m O2 = citrulline + nitric oxide + n NADP+." [EC:1.14.13.39]
GO:0004518
name: nuclease activity
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids." [ISBN:0198547684]
GO:0004519
name: endonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by creating internal breaks." [GOC:mah, ISBN:0198547684]
GO:0004520
name: endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acid by creating internal breaks." [GOC:mah, ISBN:0198547684]
GO:0004521
name: endoribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within ribonucleic acid by creating internal breaks." [GOC:mah, ISBN:0198547684]
GO:0004522
name: pancreatic ribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of RNA to 3'-phosphomononucleotides and 3'-phosphooligonucleotides ending in C-P or U-P with 2',3'-cyclic phosphate intermediates." [EC:3.1.27.5]
GO:0004523
name: RNA-DNA hybrid ribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of RNA in RNA-DNA hybrids to 5'-phosphomonoesters." [EC:3.1.26.4]
GO:0004525
name: ribonuclease III activity
def: "Catalysis of the endonucleolytic cleavage of RNA with 5'-phosphomonoesters and 3'-OH termini; makes two staggered cuts in both strands of dsRNA, leaving a 3' overhang of 2 nt." [PMID:11157775, PMID:15242644]
GO:0004526
name: ribonuclease P activity
def: "Catalysis of the endonucleolytic cleavage of RNA, removing 5' extra nucleotides from tRNA precursor." [EC:3.1.26.5]
GO:0004527
name: exonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 3' or 5' end." [GOC:mah, ISBN:0198547684]
GO:0004528
name: phosphodiesterase I activity
def: "Catalysis of the sequential hydrolytic removal of 5'-nucleotides from the 3'-hydroxy termini of 3'-hydroxy-terminated oligonucleotides." [EC:3.1.4.1]
GO:0004529
name: exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' or 3' terminus of a DNA molecule." [GOC:mah, ISBN:0198547684]
GO:0004530
name: deoxyribonuclease I activity
def: "Catalysis of the endonucleolytic cleavage of DNA to 5'-phosphodinucleotide and 5'-phosphooligonucleotide end products." [EC:3.1.21.1]
GO:0004531
name: deoxyribonuclease II activity
def: "Catalysis of the endonucleolytic cleavage of DNA to 3'-phosphodinucleotide and 3'-phosphooligonucleotide end products." [EC:3.1.22.1]
GO:0004532
name: exoribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' or 3' terminus of an RNA molecule." [GOC:mah, ISBN:0198547684]
GO:0004533
name: exoribonuclease H activity
def: "Catalysis of the exonucleolytic cleavage of RNA to 5'-phosphomonoester oligonucleotides in both 5' to 3' and 3' to 5' directions." [EC:3.1.13.2, ISBN:0198547684]
GO:0004534
name: 5'-3' exoribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' terminus of an RNA molecule." [GOC:mah, ISBN:0198547684]
GO:0004535
name: poly(A)-specific ribonuclease activity
def: "Catalysis of the exonucleolytic cleavage of poly(A) to 5'-AMP." [EC:3.1.13.4, ISBN:0198547684]
GO:0004536
name: deoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acid." [GOC:mah, ISBN:0198547684]
GO:0004540
name: ribonuclease activity
def: "Catalysis of the hydrolysis of phosphodiester bonds in chains of RNA." [GOC:mah, ISBN:0198547684]
GO:0004549
name: tRNA-specific ribonuclease activity
def: "Catalysis of the hydrolysis of phosphodiester bonds in tRNA molecules." [GOC:mah]
GO:0004550
name: nucleoside diphosphate kinase activity
def: "Catalysis of the reaction: ATP + nucleoside diphosphate = ADP + nucleoside triphosphate." [EC:2.7.4.6]
GO:0004551
name: nucleotide diphosphatase activity
def: "Catalysis of the reaction: a dinucleotide + H2O = 2 mononucleotides." [EC:3.6.1.9]
GO:0004552
name: octanol dehydrogenase activity
def: "Catalysis of the reaction: 1-octanol + NAD(+) = 1-octanal + H(+) + NADH." [EC:1.1.1.73, RHEA:24623]
GO:0004553
name: hydrolase activity, hydrolyzing O-glycosyl compounds
def: "Catalysis of the hydrolysis of any O-glycosyl bond." [GOC:mah]
GO:0004555
name: alpha,alpha-trehalase activity
def: "Catalysis of the reaction: alpha,alpha-trehalose + H2O = 2 D-glucose." [EC:3.2.1.28]
GO:0004556
name: alpha-amylase activity
def: "Catalysis of the endohydrolysis of (1->4)-alpha-D-glucosidic linkages in polysaccharides containing three or more alpha-(1->4)-linked D-glucose units." [EC:3.2.1.1]
GO:0004557
name: alpha-galactosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-D-galactose residues in alpha-D-galactosides, including galactose oligosaccharides, galactomannans and galactohydrolase." [EC:3.2.1.22]
GO:0004558
name: alpha-1,4-glucosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-(1->4)-linked alpha-D-glucose residues with release of alpha-D-glucose." [EC:3.2.1.20]
GO:0004559
name: alpha-mannosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-D-mannose residues in alpha-D-mannosides." [EC:3.2.1.24]
GO:0004560
name: alpha-L-fucosidase activity
def: "Catalysis of the reaction: an alpha-L-fucoside + H2O = an alcohol + L-fucose." [EC:3.2.1.51]
GO:0004561
name: alpha-N-acetylglucosaminidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing N-acetyl-D-glucosamine residues in N-acetyl-alpha-D-glucosaminides." [EC:3.2.1.50]
GO:0004563
name: beta-N-acetylhexosaminidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing N-acetyl-D-hexosamine residues in N-acetyl-beta-D-hexosaminides." [EC:3.2.1.52]
GO:0004564
name: beta-fructofuranosidase activity
def: "Catalysis of the reaction: a fructofuranosylated fructofuranosyl acceptor + H2O = a non fructofuranosylated fructofuranosyl acceptor + a beta-D-fructofuranoside." [EC:3.2.1.26, MetaCyc:RXN-9985]
GO:0004565
name: beta-galactosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-galactose residues in beta-D-galactosides." [EC:3.2.1.23]
GO:0004566
name: beta-glucuronidase activity
def: "Catalysis of the reaction: a beta-D-glucuronoside + H2O = an alcohol + D-glucuronate." [EC:3.2.1.31]
GO:0004567
name: beta-mannosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-mannose residues in beta-D-mannosides." [EC:3.2.1.25]
GO:0004568
name: chitinase activity
def: "Catalysis of the hydrolysis of (1->4)-beta linkages of N-acetyl-D-glucosamine (GlcNAc) polymers of chitin and chitodextrins." [EC:3.2.1.14, GOC:bf, GOC:kah, GOC:pde, PMID:11468293]
GO:0004569
name: glycoprotein endo-alpha-1,2-mannosidase activity
def: "Catalysis of the hydrolysis of the terminal alpha-glucosyl-(1,3)-mannosyl unit from Glc-Man(9)-(GlcNAc)(2) oligosaccharide component of the glycoprotein produced in the Golgi membrane." [EC:3.2.1.130]
GO:0004571
name: mannosyl-oligosaccharide 1,2-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of the terminal (1->2)-linked alpha-D-mannose residues in the oligo-mannose oligosaccharide Man(9)(GlcNAc)(2)." [EC:3.2.1.113]
GO:0004572
name: mannosyl-oligosaccharide 1,3-1,6-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of the terminal (1->3)- and (1->6)-linked alpha-D-mannose residues in the mannosyl-oligosaccharide Man(5)(GlcNAc)(3)." [EC:3.2.1.114]
GO:0004573
name: mannosyl-oligosaccharide glucosidase activity
def: "Catalysis of the exohydrolysis of the non-reducing terminal glucose residue in the mannosyl-oligosaccharide Glc(3)Man(9)GlcNAc(2)." [EC:3.2.1.106]
GO:0004574
name: oligo-1,6-glucosidase activity
def: "Catalysis of the hydrolysis of (1->6)-alpha-D-glucosidic linkages in some oligosaccharides produced from starch and glycogen by alpha-amylase, and in isomaltose. Releases a free alpha-D-glucose." [EC:3.2.1.10]
GO:0004575
name: sucrose alpha-glucosidase activity
def: "Catalysis of the reaction: sucrose + H2O = alpha-D-glucose + beta-D-fructose." [EC:3.2.1.48, MetaCyc:RXN-1461]
GO:0004576
name: oligosaccharyl transferase activity
def: "Catalysis of the transfer of a oligosaccharyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
GO:0004577
name: N-acetylglucosaminyldiphosphodolichol N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + N-acetyl-D-glucosaminyl-diphosphodolichol = UDP + N,N''-diacetylchitobiosyldiphosphodolichol." [EC:2.4.1.141]
GO:0004578
name: chitobiosyldiphosphodolichol beta-mannosyltransferase activity
def: "Catalysis of the reaction: GDP-mannose + chitobiosyldiphosphodolichol = GDP + beta-D-mannosylchitobiosyldiphosphodolichol." [EC:2.4.1.142]
GO:0004579
name: dolichyl-diphosphooligosaccharide-protein glycotransferase activity
def: "Catalysis of the reaction: dolichyl diphosphooligosaccharide + protein L-asparagine = dolichyl diphosphate + a glycoprotein with the oligosaccharide chain attached by glycosylamine linkage to protein L-asparagine." [EC:2.4.1.119]
GO:0004581
name: dolichyl-phosphate beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + dolichyl phosphate = UDP + dolichyl beta-D-glucosyl phosphate." [EC:2.4.1.117]
GO:0004582
name: dolichyl-phosphate beta-D-mannosyltransferase activity
def: "Catalysis of the reaction: GDP-mannose + dolichyl phosphate = GDP + dolichyl D-mannosyl phosphate." [EC:2.4.1.83]
GO:0004583
name: dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-glucosyl residue from dolichyl-phosphate D-glucose into a membrane lipid-linked oligosaccharide." [GOC:mah]
GO:0004584
name: dolichyl-phosphate-mannose-glycolipid alpha-mannosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-mannosyl residue from dolichyl-phosphate D-mannose into membrane lipid-linked oligosaccharide." [EC:2.4.1.130]
GO:0004585
name: ornithine carbamoyltransferase activity
def: "Catalysis of the reaction: carbamoyl phosphate + L-ornithine = phosphate + L-citrulline." [EC:2.1.3.3]
GO:0004586
name: ornithine decarboxylase activity
def: "Catalysis of the reaction: L-ornithine + H(+) = CO(2) + putrescine." [EC:4.1.1.17, RHEA:22967]
GO:0004587
name: ornithine-oxo-acid transaminase activity
def: "Catalysis of the reaction: L-ornithine + a 2-oxo acid = L-glutamate 5-semialdehyde + an L-amino acid." [EC:2.6.1.13]
GO:0004588
name: orotate phosphoribosyltransferase activity
def: "Catalysis of the reaction: orotidine 5'-phosphate + diphosphate = orotate + 5-phospho-alpha-D-ribose 1-diphosphate." [EC:2.4.2.10]
GO:0004589
name: orotate reductase (NADH) activity
def: "Catalysis of the reaction: (S)-dihydroorotate + NAD(+) = H(+) + NADH + orotate." [EC:1.3.1.14, RHEA:13516]
GO:0004590
name: orotidine-5'-phosphate decarboxylase activity
def: "Catalysis of the reaction: H(+) + orotidine 5'-phosphate = CO(2) + UMP." [EC:4.1.1.23, RHEA:11599]
GO:0004591
name: oxoglutarate dehydrogenase (succinyl-transferring) activity
def: "Catalysis of the reaction: 2-oxoglutarate + lipoamide = S-succinyldihydrolipoamide + CO2." [EC:1.2.4.2]
GO:0004592
name: pantoate-beta-alanine ligase activity
def: "Catalysis of the reaction: ATP + (R)-pantoate + beta-alanine = AMP + diphosphate + (R)-pantothenate." [EC:6.3.2.1]
GO:0004593
name: pantothenase activity
def: "Catalysis of the reaction: (R)-pantothenate + H(2)O = (R)-pantoate + beta-alanine." [EC:3.5.1.22, RHEA:12451]
GO:0004594
name: pantothenate kinase activity
def: "Catalysis of the reaction: ATP + pantothenate = ADP + D-4'-phosphopantothenate." [EC:2.7.1.33]
GO:0004595
name: pantetheine-phosphate adenylyltransferase activity
def: "Catalysis of the reaction: ATP + pantetheine 4'-phosphate = 3'-dephospho-CoA + diphosphate." [EC:2.7.7.3, RHEA:19804]
GO:0004596
name: peptide alpha-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + peptide = CoA + N-alpha-acetylpeptide. This reaction is the acetylation of the N-terminal amino acid residue of a peptide or protein." [EC:2.3.1.88, GOC:mah]
GO:0004597
name: peptide-aspartate beta-dioxygenase activity
def: "Catalysis of the reaction: peptide L-aspartate + 2-oxoglutarate + O2 = peptide 3-hydroxy-L-aspartate + succinate + CO2." [EC:1.14.11.16]
GO:0004598
name: peptidylamidoglycolate lyase activity
def: "Catalysis of the reaction: peptidylamidoglycolate = peptidyl amide + glyoxylate." [EC:4.3.2.5]
GO:0004600
name: cyclophilin
def: "OBSOLETE. A protein to which cyclosporin A (an immunosuppressant) binds. Possesses peptidyl-prolyl isomerase activity." [EC:5.2.1.8, ISBN:0198506732]
GO:0004601
name: peroxidase activity
def: "Catalysis of the reaction: donor + hydrogen peroxide = oxidized donor + 2 H2O." [EC:1.11.1.7]
GO:0004602
name: glutathione peroxidase activity
def: "Catalysis of the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O." [EC:1.11.1.9]
GO:0004603
name: phenylethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phenylethanolamine = S-adenosyl-L-homocysteine + N-methylphenylethanolamine." [EC:2.1.1.28]
GO:0004604
name: phosphoadenylyl-sulfate reductase (thioredoxin) activity
def: "Catalysis of the reaction: adenosine 3',5'-diphosphate + H(+) + sulfite + thioredoxin disulfide = 3'-phospho-5'-adenylyl sulfate + thioredoxin. Thioredoxin disulfide is the oxidized form of thioredoxin; 3'-phosphoadenosine 5'-phosphosulfate is also known as PAPS." [EC:1.8.4.8, RHEA:11727]
GO:0004605
name: phosphatidate cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + phosphatidate = diphosphate + CDP-diacylglycerol." [EC:2.7.7.41]
GO:0004607
name: phosphatidylcholine-sterol O-acyltransferase activity
def: "Catalysis of the reaction: phosphatidylcholine + a sterol = a sterol ester + 1-acylglycerophosphocholine." [EC:2.3.1.43]
GO:0004608
name: phosphatidylethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phosphatidylethanolamine = S-adenosyl-L-homocysteine + H(+) + phosphatidyl-N-methylethanolamine." [EC:2.1.1.17, RHEA:11167]
GO:0004609
name: phosphatidylserine decarboxylase activity
def: "Catalysis of the reaction: H(+) + phosphatidyl-L-serine = CO(2) + phosphatidylethanolamine." [EC:4.1.1.65, RHEA:20831]
GO:0004610
name: phosphoacetylglucosamine mutase activity
def: "Catalysis of the reaction: N-acetyl-alpha-D-glucosamine 1-phosphate = N-acetyl-D-glucosamine 6-phosphate." [EC:5.4.2.3, RHEA:23807]
GO:0004611
name: phosphoenolpyruvate carboxykinase activity
def: "Catalysis of the reaction: source of phosphate + oxaloacetate = phosphoenolpyruvate + CO2 + other reaction products." [EC:4.1.1.-]
GO:0004612
name: phosphoenolpyruvate carboxykinase (ATP) activity
def: "Catalysis of the reaction: ATP + oxaloacetate = ADP + CO(2) + H(+) + phosphoenolpyruvate." [EC:4.1.1.49, RHEA:18620]
GO:0004613
name: phosphoenolpyruvate carboxykinase (GTP) activity
def: "Catalysis of the reaction: GTP + oxaloacetate = GDP + phosphoenolpyruvate + CO2." [EC:4.1.1.32]
GO:0004614
name: phosphoglucomutase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate = alpha-D-glucose 6-phosphate." [EC:5.4.2.2]
GO:0004615
name: phosphomannomutase activity
def: "Catalysis of the reaction: alpha-D-mannose 1-phosphate = D-mannose 6-phosphate." [EC:5.4.2.8, RHEA:11143]
GO:0004616
name: phosphogluconate dehydrogenase (decarboxylating) activity
def: "Catalysis of the reaction: 6-phospho-D-gluconate + NADP+ = D-ribulose 5-phosphate + CO2 + NADPH + H+." [EC:1.1.1.44]
GO:0004617
name: phosphoglycerate dehydrogenase activity
def: "Catalysis of the reaction: 3-phosphoglycerate + NAD+ = 3-phosphohydroxypyruvate + NADH + H+." [EC:1.1.1.95]
GO:0004618
name: phosphoglycerate kinase activity
def: "Catalysis of the reaction: 3-phospho-D-glycerate + ATP = 3-phospho-D-glyceroyl phosphate + ADP + H(+)." [EC:2.7.2.3, RHEA:14804]
GO:0004619
name: phosphoglycerate mutase activity
def: "Catalysis of the reaction: 2-phospho-D-glycerate = 3-phospho-D-glycerate." [EC:5.4.2.1]
GO:0004620
name: phospholipase activity
def: "Catalysis of the hydrolysis of a glycerophospholipid." [ISBN:0198506732]
GO:0004621
name: glycosylphosphatidylinositol phospholipase D activity
def: "Catalysis of the reaction: glycoprotein phosphatidylinositol + H2O = phosphatidate + glycoprotein inositol." [EC:3.1.4.50]
GO:0004622
name: lysophospholipase activity
def: "Catalysis of the reaction: 2-lysophosphatidylcholine + H2O = glycerophosphocholine + a carboxylate." [EC:3.1.1.5]
GO:0004623
name: phospholipase A2 activity
def: "Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a carboxylate." [EC:3.1.1.4]
GO:0004624
name: secreted phospholipase A2 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.4]
GO:0004625
name: calcium-dependent secreted phospholipase A2 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.4]
GO:0004626
name: cytosolic phospholipase A2 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.4]
GO:0004627
name: calcium-dependent cytosolic phospholipase A2 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.4]
GO:0004628
name: calcium-independent cytosolic phospholipase A2 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.4]
GO:0004629
name: phospholipase C activity
def: "Catalysis of the reaction: a phospholipid + H2O = 1,2-diacylglycerol + a phosphatidate." [EC:3.1.4.3, EC:3.1.4.4, GOC:mah]
GO:0004630
name: phospholipase D activity
def: "Catalysis of the reaction: a phosphatidylcholine + H2O = choline + a phosphatidate." [EC:3.1.4.4]
GO:0004631
name: phosphomevalonate kinase activity
def: "Catalysis of the reaction: (R)-5-phosphomevalonate + ATP = (R)-5-diphosphomevalonate + ADP + H(+)." [EC:2.7.4.2, RHEA:16344]
GO:0004632
name: phosphopantothenate--cysteine ligase activity
def: "Catalysis of the reaction: CTP + (R)-4'-phosphopantothenate + L-cysteine = CMP + diphosphate + (R)-4'-phosphopantothenoyl-L-cysteine. Cysteine can be replaced by some of its derivatives." [EC:6.3.2.5]
GO:0004633
name: phosphopantothenoylcysteine decarboxylase activity
def: "Catalysis of the reaction: N-[(R)-4-phosphonatopantothenoyl]-L-cysteinate + H(+) = CO(2) + pantetheine 4'-phosphate." [EC:4.1.1.36, RHEA:16796]
GO:0004634
name: phosphopyruvate hydratase activity
def: "Catalysis of the reaction: 2-phospho-D-glycerate = phosphoenolpyruvate + H2O." [EC:4.2.1.11, ISBN:0198506732]
GO:0004635
name: phosphoribosyl-AMP cyclohydrolase activity
def: "Catalysis of the reaction: 1-(5-phosphonatoribosyl)-5'-AMP + H(2)O = 1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino]imidazole-4-carboxamide." [EC:3.5.4.19, RHEA:20052]
GO:0004636
name: phosphoribosyl-ATP diphosphatase activity
def: "Catalysis of the reaction: 1-(5-phospho-D-ribosyl)-ATP + H(2)O = 1-(5-phosphonatoribosyl)-5'-AMP + diphosphate + H(+)." [EC:3.6.1.31, RHEA:22831]
GO:0004637
name: phosphoribosylamine-glycine ligase activity
def: "Catalysis of the reaction: 5-phospho-D-ribosylamine + ATP + glycine = N(1)-(5-phospho-D-ribosyl)glycinamide + ADP + 2 H(+) + phosphate." [EC:6.3.4.13, RHEA:17456]
GO:0004638
name: phosphoribosylaminoimidazole carboxylase activity
def: "Catalysis of the reaction: 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate + 2 H(+) = 5-amino-1-(5-phospho-D-ribosyl)imidazole + CO(2)." [EC:4.1.1.21, RHEA:10795]
GO:0004639
name: phosphoribosylaminoimidazolesuccinocarboxamide synthase activity
def: "Catalysis of the reaction: 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate + L-aspartate + ATP = (2S)-2-[5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamido]succinate + ADP + 2 H(+) + phosphate." [EC:6.3.2.6, RHEA:22631]
GO:0004640
name: phosphoribosylanthranilate isomerase activity
def: "Catalysis of the reaction: N-(5-phospho-beta-D-ribosyl)anthranilate = 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate." [EC:5.3.1.24, RHEA:21543]
GO:0004641
name: phosphoribosylformylglycinamidine cyclo-ligase activity
def: "Catalysis of the reaction: 2-(formamido)-N(1)-(5-phospho-D-ribosyl)acetamidine + ATP = 5-amino-1-(5-phospho-D-ribosyl)imidazole + ADP + 2 H(+) + phosphate." [EC:6.3.3.1, RHEA:23035]
GO:0004642
name: phosphoribosylformylglycinamidine synthase activity
def: "Catalysis of the reaction: N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide + L-glutamine + ATP + H(2)O = 2-(formamido)-N(1)-(5-phospho-D-ribosyl)acetamidine + L-glutamate + ADP + 2 H(+) + phosphate." [EC:6.3.5.3, RHEA:17132]
GO:0004643
name: phosphoribosylaminoimidazolecarboxamide formyltransferase activity
def: "Catalysis of the reaction: 10-formyltetrahydrofolate + 5'-phosphoribosyl-5-amino-4-imidazolecarboxamide = tetrahydrofolate + 5'-phosphoribosyl-5-formamido-4-imidazolecarboxamide." [EC:2.1.2.3]
GO:0004644
name: phosphoribosylglycinamide formyltransferase activity
def: "Catalysis of the reaction: 10-formyltetrahydrofolate + N1-(5-phospho-D-ribosyl)glycinamide = tetrahydrofolate + N2-formyl-N1-(5-phospho-D-ribosyl)glycinamide." [EC:2.1.2.2]
GO:0004645
name: phosphorylase activity
def: "Catalysis of the reaction: 1,4-alpha-D-glucosyl(n) + phosphate = 1,4-alpha-D-glucosyl(n-1) + alpha-D-glucose 1-phosphate. The name should be qualified in each instance by adding the name of the natural substrate, e.g. maltodextrin phosphorylase, starch phosphorylase, glycogen phosphorylase." [EC:2.4.1.1]
GO:0004647
name: phosphoserine phosphatase activity
def: "Catalysis of the reaction: L(or D)-O-phosphoserine + H2O = L(or D)-serine + phosphate." [EC:3.1.3.3]
GO:0004648
name: O-phospho-L-serine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: O-phospho-L-serine + 2-oxoglutarate = 3-phosphonooxypyruvate + L-glutamate." [EC:2.6.1.52]
GO:0004649
name: poly(ADP-ribose) glycohydrolase activity
def: "Catalysis of the hydrolysis of poly(ADP-ribose) at glycosidic (1''-2') linkage of ribose-ribose bond to produce free ADP-ribose." [EC:3.2.1.143]
GO:0004650
name: polygalacturonase activity
def: "Catalysis of the random hydrolysis of (1->4)-alpha-D-galactosiduronic linkages in pectate and other galacturonans." [EC:3.2.1.15]
GO:0004651
name: polynucleotide 5'-phosphatase activity
def: "Catalysis of the reaction: 5'-phosphopolynucleotide + H2O = polynucleotide + phosphate." [EC:3.1.3.33]
GO:0004652
name: polynucleotide adenylyltransferase activity
def: "Catalysis of the template-independent extension of the 3'- end of an RNA or DNA strand by addition of one adenosine molecule at a time. Cannot initiate a chain 'de novo'. The primer, depending on the source of the enzyme, may be an RNA or DNA fragment, or oligo(A) bearing a 3'-OH terminal group." [EC:2.7.7.19]
GO:0004653
name: polypeptide N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-galactosamine + polypeptide = UDP + N-acetyl-D-galactosaminyl-polypeptide. This reaction is the modification of serine or threonine residues in polypeptide chains by the transfer of a N-acetylgalactose from UDP-N-acetylgalactose to the hydroxyl group of the amino acid; it is the first step in O-glycan biosynthesis." [EC:2.4.1.41, ISBN:0879695595]
GO:0004654
name: polyribonucleotide nucleotidyltransferase activity
def: "Catalysis of the reaction: ATP + RNA(n) = diphosphate + RNA(n+1)." [EC:2.7.7.8]
GO:0004655
name: porphobilinogen synthase activity
def: "Catalysis of the reaction: 2 5-aminolevulinate = 2 H(2)O + H(+) + porphobilinogen." [EC:4.2.1.24, RHEA:24067]
GO:0004656
name: procollagen-proline 4-dioxygenase activity
def: "Catalysis of the reaction: procollagen L-proline + 2-oxoglutarate + O2 = procollagen trans-4-hydroxy-L-proline + succinate + CO2." [EC:1.14.11.2]
GO:0004657
name: proline dehydrogenase activity
def: "Catalysis of the reaction: L-proline + acceptor = (S)-1-pyrroline-5-carboxylate + reduced acceptor." [EC:1.5.99.8]
GO:0004658
name: propionyl-CoA carboxylase activity
def: "Catalysis of the reaction: ATP + propanoyl-CoA + HCO3- = ADP + phosphate + (S)-methylmalonyl-CoA." [EC:6.4.1.3]
GO:0004659
name: prenyltransferase activity
def: "Catalysis of the transfer of a prenyl group from one compound (donor) to another (acceptor)." [GOC:mah]
GO:0004660
name: protein farnesyltransferase activity
def: "Catalysis of the reaction: farnesyl diphosphate + protein-cysteine = S-farnesyl protein + diphosphate." [EC:2.5.1.58, PMID:8621375]
GO:0004661
name: protein geranylgeranyltransferase activity
def: "Catalysis of the covalent addition of a geranylgeranyl (20-carbon isoprenoid) group via thioether linkages to a cysteine residue at or near the C terminus of a protein." [PMID:8621375]
GO:0004662
name: CAAX-protein geranylgeranyltransferase activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate + protein-cysteine = S-geranylgeranyl-protein + diphosphate. This reaction is the formation of a thioether linkage between the C-1 atom of the geranylgeranyl group and a cysteine residue fourth from the C-terminus of the protein. The protein substrates have the C-terminal sequence CA1A2X, where the terminal residue, X, is preferably leucine and A2 should not be aromatic. Known substrates include most g-subunits of heterotrimeric G proteins and Ras-related GTPases such as members of the Ras and Rac/Rho families." [EC:2.5.1.59, PMID:8621375]
GO:0004663
name: Rab geranylgeranyltransferase activity
def: "Catalysis of the reaction: 2 geranylgeranyl diphosphate + protein-cysteine = 2 S-geranylgeranyl-protein + 2 diphosphate. This reaction is the formation of two thioether linkages between the C-1 atom of the geranylgeranyl groups and two cysteine residues within the terminal sequence motifs XXCC, XCXC or CCXX. Known substrates include Ras-related GTPases of a single family and the Rab family." [EC:2.5.1.60, GOC:mah, PMID:8621375]
GO:0004664
name: prephenate dehydratase activity
def: "Catalysis of the reaction: prephenate = phenylpyruvate + H2O + CO2." [EC:4.2.1.51]
GO:0004665
name: prephenate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + prephenate = (4-hydroxyphenyl)pyruvate + CO(2) + NADPH." [EC:1.3.1.13, RHEA:21643]
GO:0004666
name: prostaglandin-endoperoxide synthase activity
def: "Catalysis of the reaction: arachidonate + donor-H2 + 2 O2 = prostaglandin H2 + acceptor + H2O." [EC:1.14.99.1]
GO:0004667
name: prostaglandin-D synthase activity
def: "Catalysis of the reaction: prostaglandin H(2) = prostaglandin D(2)." [EC:5.3.99.2, RHEA:10603]
GO:0004668
name: protein-arginine deiminase activity
def: "Catalysis of the reaction: protein L-arginine + H2O = protein L-citrulline + NH3." [EC:3.5.3.15]
GO:0004671
name: protein C-terminal S-isoprenylcysteine carboxyl O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein C-terminal S-farnesyl-L-cysteine = S-adenosyl-L-homocysteine + protein C-terminal S-farnesyl-L-cysteine methyl ester." [EC:2.1.1.100]
GO:0004672
name: protein kinase activity
def: "Catalysis of the phosphorylation of an amino acid residue in a protein, usually according to the reaction: a protein + ATP = a phosphoprotein + ADP." [MetaCyc:PROTEIN-KINASE-RXN]
GO:0004673
name: protein histidine kinase activity
def: "Catalysis of the reaction: ATP + protein L-histidine = ADP + protein phospho-L-histidine." [EC:2.7.13.3, GOC:mah]
GO:0004674
name: protein serine/threonine kinase activity
def: "Catalysis of the reactions: ATP + protein serine = ADP + protein serine phosphate, and ATP + protein threonine = ADP + protein threonine phosphate." [GOC:bf]
GO:0004675
name: transmembrane receptor protein serine/threonine kinase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP protein serine = ADP + protein serine phosphate, and ATP + protein threonine = ADP + protein threonine phosphate." [EC:2.7.11.30]
GO:0004676
name: 3-phosphoinositide-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of a phosphatidylinositol-3-phosphate." [GOC:mah]
GO:0004677
name: DNA-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of DNA." [GOC:mah]
GO:0004679
name: AMP-activated protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of AMP." [GOC:mah]
GO:0004680
name: casein kinase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004681
name: casein kinase I activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0004682
name: protein kinase CK2 activity
def: "OBSOLETE. Catalysis of the reaction: casein + ATP = phosphocasein + ADP." [EC:2.7.11.1]
GO:0004683
name: calmodulin-dependent protein kinase activity
def: "Catalysis of the reactions: ATP + a protein serine = ADP + protein serine phosphate; and ATP + a protein threonine = ADP + protein threonine phosphate. These reactions require the presence of calcium-bound calmodulin." [GOC:mah, PMID:11264466]
GO:0004686
name: elongation factor-2 kinase activity
def: "Catalysis of the reaction: ATP + [elongation factor 2] = ADP + [elongation factor 2] phosphate." [EC:2.7.11.20, MetaCyc:2.7.11.20-RXN]
GO:0004687
name: myosin light chain kinase activity
def: "Catalysis of the reaction: ATP + myosin-light-chain = ADP + myosin-light-chain phosphate." [EC:2.7.11.18]
GO:0004689
name: phosphorylase kinase activity
def: "Catalysis of the reaction: 4 ATP + 2 phosphorylase b = 4 ADP + phosphorylase a." [EC:2.7.11.19]
GO:0004690
name: cyclic nucleotide-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of a cyclic nucleotide." [GOC:mah]
GO:0004691
name: cAMP-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of cAMP." [EC:2.7.11.11]
GO:0004692
name: cGMP-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires the presence of cGMP." [GOC:mah]
GO:0004693
name: cyclin-dependent protein serine/threonine kinase activity
def: "Catalysis of the reactions: ATP + protein serine = ADP + protein serine phosphate, and ATP + protein threonine = ADP + protein threonine phosphate. This reaction requires the binding of a regulatory cyclin subunit and full activity requires stimulatory phosphorylation by a CDK-activating kinase (CAK)." [GOC:pr, GOC:rn, PMID:7877684, PMID:9841670]
GO:0004694
name: eukaryotic translation initiation factor 2alpha kinase activity
def: "Catalysis of the reaction: ATP + [eukaryotic translation initiation factor 2 alpha subunit] = ADP + [eukaryotic translation initiation factor 2 alpha subunit] phosphate." [GOC:mah, InterPro:IPR015516]
GO:0004697
name: protein kinase C activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires diacylglycerol." [EC:2.7.11.13]
GO:0004698
name: calcium-dependent protein kinase C activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires diacylglycerol and calcium." [EC:2.7.11.13, GOC:mah]
GO:0004699
name: calcium-independent protein kinase C activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires diacylglycerol but not calcium." [EC:2.7.11.13, GOC:mah]
GO:0004702
name: receptor signaling protein serine/threonine kinase activity
def: "Conveys a signal from an upstream receptor or intracellular signal transducer by catalysis of the reaction: ATP protein serine = ADP + protein serine phosphate, and ATP + protein threonine = ADP + protein threonine phosphate." [EC:2.7.11.13, GOC:bf, GOC:mah]
GO:0004703
name: G-protein coupled receptor kinase activity
def: "Catalysis of the reaction: ATP + G-protein coupled receptor = ADP + G-protein coupled receptor phosphate." [GOC:dph]
GO:0004704
name: NF-kappaB-inducing kinase activity
def: "Catalysis of the phosphorylation of the alpha or beta subunit of the inhibitor of kappaB kinase complex (IKK)." [PMID:20685151]
GO:0004705
name: JUN kinase activity
def: "Catalysis of the reaction: JUN + ATP = JUN phosphate + ADP. This reaction is the phosphorylation and activation of members of the JUN family, a gene family that encodes nuclear transcription factors." [GOC:bf, ISBN:0198506732]
GO:0004706
name: JUN kinase kinase kinase activity
def: "Catalysis of the reaction: JNKK + ATP = JNKK phosphate + ADP. This reaction is the phosphorylation and activation of JUN kinase kinases (JNKKs)." [GOC:bf]
GO:0004707
name: MAP kinase activity
def: "Catalysis of the reaction: protein + ATP = protein phosphate + ADP. This reaction is the phosphorylation of proteins. Mitogen-activated protein kinase; a family of protein kinases that perform a crucial step in relaying signals from the plasma membrane to the nucleus. They are activated by a wide range of proliferation- or differentiation-inducing signals; activation is strong with agonists such as polypeptide growth factors and tumor-promoting phorbol esters, but weak (in most cell backgrounds) by stress stimuli." [GOC:ma, ISBN:0198547684]
GO:0004708
name: MAP kinase kinase activity
def: "Catalysis of the concomitant phosphorylation of threonine (T) and tyrosine (Y) residues in a Thr-Glu-Tyr (TEY) thiolester sequence in a MAP kinase (MAPK) substrate." [ISBN:0198547684]
GO:0004709
name: MAP kinase kinase kinase activity
def: "Catalysis of the phosphorylation and activation of a MAP kinase kinase; each MAP kinase kinase can be phosphorylated by any of several MAP kinase kinase kinases." [PMID:9561267]
GO:0004711
name: ribosomal protein S6 kinase activity
def: "Catalysis of the reaction: ribosomal protein S6 + ATP = ribosomal protein S6 phosphate + ATP." [GOC:mah, PMID:9822608]
GO:0004712
name: protein serine/threonine/tyrosine kinase activity
def: "Catalysis of the reactions: ATP + a protein serine = ADP + protein serine phosphate; ATP + a protein threonine = ADP + protein threonine phosphate; and ATP + a protein tyrosine = ADP + protein tyrosine phosphate." [GOC:mah]
GO:0004713
name: protein tyrosine kinase activity
def: "Catalysis of the reaction: ATP + a protein tyrosine = ADP + protein tyrosine phosphate." [EC:2.7.10]
GO:0004714
name: transmembrane receptor protein tyrosine kinase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate." [EC:2.7.10.1, GOC:mah]
GO:0004715
name: non-membrane spanning protein tyrosine kinase activity
def: "Catalysis of the reaction: ATP + a non-membrane spanning protein L-tyrosine = ADP + a non-membrane spanning protein L-tyrosine phosphate." [EC:2.7.10.2]
GO:0004716
name: receptor signaling protein tyrosine kinase activity
def: "Conveys a signal from an upstream receptor or intracellular signal transducer by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate." [EC:2.7.10.2]
GO:0004717
name: focal adhesion kinase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jid]
GO:0004719
name: protein-L-isoaspartate (D-aspartate) O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein L-beta-aspartate = S-adenosyl-L-homocysteine + protein L-beta-aspartate methyl ester." [EC:2.1.1.77]
GO:0004720
name: protein-lysine 6-oxidase activity
def: "Catalysis of the reaction: peptidyl-L-lysyl-peptide + H2O + O2 = peptidyl-allysyl-peptide + NH3 + hydrogen peroxide." [EC:1.4.3.13]
GO:0004721
name: phosphoprotein phosphatase activity
def: "Catalysis of the reaction: a phosphoprotein + H2O = a protein + phosphate. Together with protein kinases, these enzymes control the state of phosphorylation of cell proteins and thereby provide an important mechanism for regulating cellular activity." [EC:3.1.3.16, ISBN:0198547684]
GO:0004722
name: protein serine/threonine phosphatase activity
def: "Catalysis of the reaction: protein serine phosphate + H2O = protein serine + phosphate, and protein threonine phosphate + H2O = protein threonine + phosphate." [GOC:bf]
GO:0004723
name: calcium-dependent protein serine/threonine phosphatase activity
def: "Catalysis of the reactions: protein serine phosphate + H2O = protein serine + phosphate; and protein threonine phosphate + H2O = protein threonine + phosphate. These reactions require the presence of calcium ions." [EC:3.1.3.16, GOC:mah]
GO:0004724
name: magnesium-dependent protein serine/threonine phosphatase activity
def: "Catalysis of the reactions: protein serine phosphate + H2O = protein serine + phosphate; and protein threonine phosphate + H2O = protein threonine + phosphate. These reactions require the presence of magnesium." [EC:3.1.3.16]
GO:0004725
name: protein tyrosine phosphatase activity
def: "Catalysis of the reaction: protein tyrosine phosphate + H2O = protein tyrosine + phosphate." [EC:3.1.3.48]
GO:0004726
name: non-membrane spanning protein tyrosine phosphatase activity
def: "Catalysis of the reaction: non-membrane spanning protein tyrosine phosphate + H2O = non-membrane spanning protein tyrosine + phosphate." [EC:3.1.3.48]
GO:0004727
name: prenylated protein tyrosine phosphatase activity
def: "Catalysis of the reaction: prenylated-protein tyrosine phosphate + H2O = prenylated-protein tyrosine + phosphate." [EC:3.1.3.48]
GO:0004728
name: receptor signaling protein tyrosine phosphatase activity
def: "Conveys a signal from an upstream receptor or intracellular signal transducer by catalysis of the reaction: protein tyrosine phosphate + H2O = protein tyrosine + phosphate." [EC:3.1.3.48]
GO:0004729
name: oxygen-dependent protoporphyrinogen oxidase activity
def: "Catalysis of the reaction: 3 O(2) + protoporphyrinogen IX = 3 H(2)O(2) + protoporphyrin IX." [EC:1.3.3.4, RHEA:25579]
GO:0004730
name: pseudouridylate synthase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + uracil = H(2)O + pseudouridine 5'-phosphate." [EC:4.2.1.70, RHEA:18340]
GO:0004731
name: purine-nucleoside phosphorylase activity
def: "Catalysis of the reaction: purine nucleoside + phosphate = purine + alpha-D-ribose 1-phosphate." [EC:2.4.2.1]
GO:0004732
name: pyridoxal oxidase activity
def: "Catalysis of the reaction: pyridoxal + H2O + O2 = 4-pyridoxate + hydrogen peroxide." [EC:1.2.3.8]
GO:0004733
name: pyridoxamine-phosphate oxidase activity
def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH3 + hydrogen peroxide." [EC:1.4.3.5]
GO:0004734
name: pyrimidodiazepine synthase activity
def: "Catalysis of the reaction: a pyrimidodiazepine + oxidized glutathione = 6-pyruvoyltetrahydropterin + 2 glutathione." [EC:1.5.4.1]
GO:0004735
name: pyrroline-5-carboxylate reductase activity
def: "Catalysis of the reaction: L-proline + NADP+ = 1-pyrroline-5-carboxylate + NADPH + H+." [EC:1.5.1.2]
GO:0004736
name: pyruvate carboxylase activity
def: "Catalysis of the reaction: ATP + bicarbonate + pyruvate = ADP + 2 H(+) + oxaloacetate + phosphate." [EC:6.4.1.1, RHEA:20847]
GO:0004737
name: pyruvate decarboxylase activity
def: "Catalysis of the reaction: a 2-oxo acid = an aldehyde + CO2." [EC:4.1.1.1]
GO:0004738
name: pyruvate dehydrogenase activity
def: "Catalysis of the oxidative decarboxylation of pyruvate." [ISBN:0716020094]
GO:0004739
name: pyruvate dehydrogenase (acetyl-transferring) activity
def: "Catalysis of the reaction: pyruvate + lipoamide = S-acetyldihydrolipoamide + CO2." [EC:1.2.4.1]
GO:0004740
name: pyruvate dehydrogenase (acetyl-transferring) kinase activity
def: "Catalysis of the reaction: ATP + pyruvate dehydrogenase (acetyl-transferring) = ADP + pyruvate dehydrogenase (acetyl-transferring) phosphate." [EC:2.7.11.2]
GO:0004741
name: [pyruvate dehydrogenase (lipoamide)] phosphatase activity
def: "Catalysis of the reaction: [pyruvate dehydrogenase (lipoamide)] phosphate + H2O = [pyruvate dehydrogenase (lipoamide)] + phosphate." [EC:3.1.3.43]
GO:0004742
name: dihydrolipoyllysine-residue acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + dihydrolipoamide = CoA + S-acetyldihydrolipoamide." [EC:2.3.1.12]
GO:0004743
name: pyruvate kinase activity
def: "Catalysis of the reaction: ATP + pyruvate = ADP + phosphoenolpyruvate." [EC:2.7.1.40]
GO:0004744
name: retinal isomerase activity
def: "Catalysis of the reaction: all-trans-retinal = 11-cis-retinal." [RHEA:24127]
GO:0004745
name: retinol dehydrogenase activity
def: "Catalysis of the reaction: retinol + NAD+ = retinal + NADH + H+." [EC:1.1.1.105]
GO:0004746
name: riboflavin synthase activity
def: "Catalysis of the reaction: 2 6,7-dimethyl-8-(1-D-ribityl)lumazine + H(+) = 5-amino-6-(D-ribitylamino)uracil + riboflavin." [EC:2.5.1.9, RHEA:20775]
GO:0004747
name: ribokinase activity
def: "Catalysis of the reaction: ATP + D-ribose = ADP + D-ribose 5-phosphate." [EC:2.7.1.15]
GO:0004748
name: ribonucleoside-diphosphate reductase activity, thioredoxin disulfide as acceptor
def: "Catalysis of the reaction: 2'-deoxyribonucleoside diphosphate + thioredoxin disulfide + H2O = ribonucleoside diphosphate + thioredoxin. Thioredoxin disulfide is the oxidized form of thioredoxin." [EC:1.17.4.1]
GO:0004749
name: ribose phosphate diphosphokinase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + ATP = 5-phospho-alpha-D-ribose 1-diphosphate + AMP + 2 H(+)." [EC:2.7.6.1, RHEA:15612]
GO:0004750
name: ribulose-phosphate 3-epimerase activity
def: "Catalysis of the reaction: D-ribulose 5-phosphate = D-xylulose 5-phosphate." [EC:5.1.3.1, RHEA:13680]
GO:0004751
name: ribose-5-phosphate isomerase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate = D-ribulose 5-phosphate." [EC:5.3.1.6, RHEA:14660]
GO:0004753
name: saccharopine dehydrogenase activity
def: "Catalysis of the cleavage of N6-(L-1,3-dicarboxypropyl)-L-lysine to release an amino acid (lysine or glutamate), with the concomitant reduction of an electron acceptor." [GOC:mah]
GO:0004754
name: saccharopine dehydrogenase (NAD+, L-lysine-forming) activity
def: "Catalysis of the reaction: L-saccharopine + H(2)O + NAD(+) = 2-oxoglutarate + L-lysine + H(+) + NADH." [EC:1.5.1.7, RHEA:12443]
GO:0004755
name: saccharopine dehydrogenase (NADP+, L-glutamate-forming) activity
def: "Catalysis of the reaction: L-saccharopine + H(2)O + NADP(+) = L-allysine + L-glutamate + H(+) + NADPH." [EC:1.5.1.10, RHEA:10023]
GO:0004756
name: selenide, water dikinase activity
def: "Catalysis of the reaction: ATP + H(2)O + hydrogen selenide = AMP + 3 H(+) + phosphate + selenophosphorate." [EC:2.7.9.3, RHEA:18740]
GO:0004757
name: sepiapterin reductase activity
def: "Catalysis of the reaction: 7,8-dihydrobiopterin + NADP+ = sepiapterin + NADPH + H+." [EC:1.1.1.153]
GO:0004758
name: serine C-palmitoyltransferase activity
def: "Catalysis of the reaction: L-serine + H(+) + palmitoyl-CoA = 3-dehydrosphinganine + CO(2) + CoA." [EC:2.3.1.50, RHEA:14764]
GO:0004760
name: serine-pyruvate transaminase activity
def: "Catalysis of the reaction: L-serine + pyruvate = 3-hydroxypyruvate + L-alanine." [EC:2.6.1.51, RHEA:22855]
GO:0004764
name: shikimate 3-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: shikimate + NADP+ = 3-dehydroshikimate + NADPH + H+." [EC:1.1.1.25]
GO:0004765
name: shikimate kinase activity
def: "Catalysis of the reaction: ATP + shikimate = 3-phosphoshikimate + ADP + 2 H(+)." [EC:2.7.1.71, RHEA:13124]
GO:0004766
name: spermidine synthase activity
def: "Catalysis of the reaction: S-adenosylmethioninamine + putrescine = 5'-methylthioadenosine + spermidine." [EC:2.5.1.16]
GO:0004767
name: sphingomyelin phosphodiesterase activity
def: "Catalysis of the reaction: H(2)O + sphingomyelin = ceramide + choline phosphate + H(+)." [EC:3.1.4.12, RHEA:19256]
GO:0004768
name: stearoyl-CoA 9-desaturase activity
def: "Catalysis of the reaction: stearoyl-CoA + 2 ferrocytochrome b5 + O2 + 2 H+ = oleoyl-CoA + 2 ferricytochrome b5 + H2O." [EC:1.14.19.1]
GO:0004769
name: steroid delta-isomerase activity
def: "Catalysis of the reaction: a 3-oxo-delta(5)-steroid = a 3-oxo-delta(4)-steroid." [EC:5.3.3.1]
GO:0004771
name: sterol esterase activity
def: "Catalysis of the reaction: a steryl ester + H2O = a sterol + a fatty acid." [EC:3.1.1.13]
GO:0004772
name: sterol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + a sterol = CoA + a sterol ester." [EC:2.3.1.26, GOC:mah]
GO:0004773
name: steryl-sulfatase activity
def: "Catalysis of the reaction: 3-beta-hydroxyandrost-5-en-17-one 3-sulfate + H2O = 3-beta-hydroxyandrost-5-en-17-one + sulfate." [EC:3.1.6.2]
GO:0004774
name: succinate-CoA ligase activity
def: "Catalysis of the reaction: succinate + CoA + nucleotide triphosphate = nucleotide diphosphate + phosphate + succinyl-CoA." [EC:6.2.1.-, GOC:ai]
GO:0004775
name: succinate-CoA ligase (ADP-forming) activity
def: "Catalysis of the reaction: ATP + succinate + CoA = ADP + succinyl-CoA + phosphate." [EC:6.2.1.5]
GO:0004776
name: succinate-CoA ligase (GDP-forming) activity
def: "Catalysis of the reaction: GTP + succinate + CoA = GDP + succinyl-CoA + phosphate." [EC:6.2.1.4]
GO:0004777
name: succinate-semialdehyde dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: succinate semialdehyde + NAD+ + H2O = succinate + NADH + H+." [EC:1.2.1.24]
GO:0004778
name: succinyl-CoA hydrolase activity
def: "Catalysis of the reaction: H(2)O + succinyl-CoA = CoA + H(+) + succinate." [EC:3.1.2.3, RHEA:11519]
GO:0004779
name: sulfate adenylyltransferase activity
def: "Catalysis of the transfer of an adenylyl group from an adenosine nucleotide (ATP or ADP) to sulfate, forming adenylylsulfate." [GOC:mah, MetaCyc:SULFATE-ADENYLYLTRANS-RXN, MetaCyc:SULFATE-ADENYLYLTRANSFERASE-ADP-RXN]
GO:0004780
name: sulfate adenylyltransferase (ADP) activity
def: "Catalysis of the reaction: ADP + H(+) + sulfate = 5'-adenylyl sulfate + phosphate." [EC:2.7.7.5, RHEA:16532]
GO:0004781
name: sulfate adenylyltransferase (ATP) activity
def: "Catalysis of the reaction: ATP + sulfate = diphosphate + adenylylsulfate." [EC:2.7.7.4]
GO:0004782
name: sulfinoalanine decarboxylase activity
def: "Catalysis of the reaction: 3-sulfino-L-alanine = hypotaurine + CO2." [EC:4.1.1.29]
GO:0004783
name: sulfite reductase (NADPH) activity
def: "Catalysis of the reaction: hydrogen sulfide + 3 NADP+ + 3 H2O = sulfite + 3 NADPH + 3 H+." [EC:1.8.1.2]
GO:0004784
name: superoxide dismutase activity
def: "Catalysis of the reaction: 2 superoxide + 2 H+ = O2 + hydrogen peroxide." [EC:1.15.1.1, GOC:vw, PMID:15064408]
GO:0004786
name: Mn, Fe superoxide dismutase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0004787
name: thiamine-pyrophosphatase activity
def: "Catalysis of the reaction: TDP + H2O = TMP + phosphate." [EC:3.6.1.-, GOC:ai]
GO:0004788
name: thiamine diphosphokinase activity
def: "Catalysis of the reaction: ATP + thiamine = AMP + thiamine diphosphate." [EC:2.7.6.2]
GO:0004789
name: thiamine-phosphate diphosphorylase activity
def: "Catalysis of the reaction: 4-amino-2-methyl-5-diphosphomethylpyrimidine + 4-methyl-5-(2-phosphoethyl)-thiazole + H(+) = diphosphate + thiamine phosphate." [EC:2.5.1.3, RHEA:22331]
GO:0004790
name: thioether S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + dimethyl sulfide = S-adenosyl-L-homocysteine + trimethylsulfonium." [EC:2.1.1.96, RHEA:19616]
GO:0004791
name: thioredoxin-disulfide reductase activity
def: "Catalysis of the reaction: NADP(+) + thioredoxin = H(+) + NADPH + thioredoxin disulfide." [EC:1.8.1.9, RHEA:20348]
GO:0004792
name: thiosulfate sulfurtransferase activity
def: "Catalysis of the reaction: hydrogen cyanide + thiosulfate = H(+) + sulfite + thiocyanate." [EC:2.8.1.1, RHEA:16884]
GO:0004793
name: threonine aldolase activity
def: "Catalysis of the reaction: L-threonine = glycine + acetaldehyde." [EC:4.1.2.5]
GO:0004794
name: L-threonine ammonia-lyase activity
def: "Catalysis of the reaction: L-threonine = 2-oxobutanoate + NH3." [EC:4.3.1.19]
GO:0004795
name: threonine synthase activity
def: "Catalysis of the reaction: O-phospho-L-homoserine + H2O = L-threonine + phosphate." [EC:4.2.3.1]
GO:0004796
name: thromboxane-A synthase activity
def: "Catalysis of the reaction: prostaglandin H(2) = thromboxane A(2)." [EC:5.3.99.5, RHEA:17140]
GO:0004797
name: thymidine kinase activity
def: "Catalysis of the reaction: ATP + thymidine = ADP + thymidine 5'-phosphate." [EC:2.7.1.21]
GO:0004798
name: thymidylate kinase activity
def: "Catalysis of the reaction: ATP + thymidine 5'-phosphate = ADP + thymidine 5'-diphosphate." [EC:2.7.4.9]
GO:0004799
name: thymidylate synthase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + dUMP = 7,8-dihydrofolate + thymidylate." [EC:2.1.1.45, RHEA:12107]
GO:0004800
name: thyroxine 5'-deiodinase activity
def: "Catalysis of the reaction: 3,5,3'-L-triiodo-L-thyronine + iodide + acceptor + H+ = L-thyroxine + donor-H2." [EC:1.97.1.10]
GO:0004801
name: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase activity
def: "Catalysis of the reaction: sedoheptulose 7-phosphate + D-glyceraldehyde 3-phosphate = D-erythrose 4-phosphate + D-fructose 6-phosphate." [EC:2.2.1.2]
GO:0004802
name: transketolase activity
def: "Catalysis of the reversible transfer of a 2-carbon ketol group (CH2OH-CO-) from a ketose phosphate donor to an aldose phosphate acceptor." [EC:2.2.1.1, GOC:fmc]
GO:0004803
name: transposase activity
def: "Catalysis of the transposition of transposable elements or transposons. Transposases are involved in recombination required for transposition and are site-specific for the transposon/transposable element." [GOC:bm, ISBN:0198506732]
GO:0004805
name: trehalose-phosphatase activity
def: "Catalysis of the reaction: trehalose 6-phosphate + H2O = trehalose + phosphate." [EC:3.1.3.12]
GO:0004806
name: triglyceride lipase activity
def: "Catalysis of the reaction: triacylglycerol + H2O = diacylglycerol + a carboxylate." [EC:3.1.1.3]
GO:0004807
name: triose-phosphate isomerase activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate = glycerone phosphate." [EC:5.3.1.1, RHEA:18588]
GO:0004808
name: tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA containing 5-aminomethyl-2-thiouridine = S-adenosyl-L-homocysteine + tRNA containing 5-methylaminomethyl-2-thiouridylate." [EC:2.1.1.61, GOC:imk]
GO:0004809
name: tRNA (guanine-N2-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA containing guanine = S-adenosyl-L-homocysteine + tRNA containing N2-methylguanine." [EC:2.1.1.32]
GO:0004810
name: tRNA adenylyltransferase activity
def: "Catalysis of the reaction: ATP + tRNA(n) = diphosphate + tRNA(n+1)." [EC:2.7.7.25]
GO:0004812
name: aminoacyl-tRNA ligase activity
def: "Catalysis of the formation of aminoacyl-tRNA from ATP, amino acid, and tRNA with the release of diphosphate and AMP." [ISBN:0198506732]
GO:0004813
name: alanine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-alanine + tRNA(Ala) = AMP + diphosphate + L-alanyl-tRNA(Ala)." [EC:6.1.1.7]
GO:0004814
name: arginine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-arginine + tRNA(Arg) = AMP + diphosphate + L-arginyl-tRNA(Arg)." [EC:6.1.1.19]
GO:0004815
name: aspartate-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-aspartate + tRNA(Asp) = AMP + diphosphate + L-aspartyl-tRNA(Asp)." [EC:6.1.1.12]
GO:0004816
name: asparagine-tRNA ligase activity
def: "Catalysis of the reaction: L-asparagine + ATP + tRNA(Asn) = AMP + Asn-tRNA(Asn) + diphosphate + 2 H(+)." [EC:6.1.1.22, RHEA:11183]
GO:0004817
name: cysteine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-cysteine + tRNA(Cys) = AMP + diphosphate + L-cysteinyl-tRNA(Cys)." [EC:6.1.1.16]
GO:0004818
name: glutamate-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-glutamate + tRNA(Glu) = AMP + diphosphate + L-glutamyl-tRNA(Glu)." [EC:6.1.1.17]
GO:0004819
name: glutamine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-glutamine + tRNA(Gln) = AMP + diphosphate + L-glutaminyl-tRNA(Gln)." [EC:6.1.1.18]
GO:0004820
name: glycine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + glycine + tRNA(Gly) = AMP + diphosphate + glycyl-tRNA(Gly)." [EC:6.1.1.14]
GO:0004821
name: histidine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-histidine + tRNA(His) = AMP + diphosphate + L-histidyl-tRNA(His)." [EC:6.1.1.21]
GO:0004822
name: isoleucine-tRNA ligase activity
def: "Catalysis of the reaction: L-isoleucine + ATP + tRNA(Ile) = L-isoleucyl-tRNA(Ile) + AMP + diphosphate + 2 H(+)." [EC:6.1.1.5, RHEA:11063]
GO:0004823
name: leucine-tRNA ligase activity
def: "Catalysis of the reaction: L-leucine + ATP + tRNA(Leu) = AMP + diphosphate + 2 H(+) + Leu-tRNA(Leu)." [EC:6.1.1.4, RHEA:11691]
GO:0004824
name: lysine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-lysine + tRNA(Lys) = AMP + diphosphate + L-lysyl-tRNA(Lys)." [EC:6.1.1.6]
GO:0004825
name: methionine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-methionine + tRNA(Met) = AMP + diphosphate + L-methionyl-tRNA(Met)." [EC:6.1.1.10]
GO:0004826
name: phenylalanine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-phenylalanine + tRNA(Phe) = AMP + diphosphate + L-phenylalanyl-tRNA(Phe)." [EC:6.1.1.20]
GO:0004827
name: proline-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-proline + tRNA(Pro) = AMP + diphosphate + L-prolyl-tRNA(Pro)." [EC:6.1.1.15]
GO:0004828
name: serine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-serine + tRNA(Ser) = AMP + diphosphate + L-seryl-tRNA(Ser)." [EC:6.1.1.11]
GO:0004829
name: threonine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-threonine + tRNA(Thr) = AMP + diphosphate + L-threonyl-tRNA(Thr)." [EC:6.1.1.3]
GO:0004830
name: tryptophan-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-tryptophan + tRNA(Trp) = AMP + diphosphate + L-tryptophanyl-tRNA(Trp)." [EC:6.1.1.2]
GO:0004831
name: tyrosine-tRNA ligase activity
def: "Catalysis of the reaction: L-tyrosine + ATP + tRNA(Tyr) = L-tyrosyl-tRNA(Tyr) + AMP + diphosphate + 2 H(+)." [EC:6.1.1.1, RHEA:10223]
GO:0004832
name: valine-tRNA ligase activity
def: "Catalysis of the reaction: L-valine + ATP + tRNA(Val) = L-valyl-tRNA(Val) + AMP + diphosphate + 2 H(+)." [EC:6.1.1.9, RHEA:10707]
GO:0004833
name: tryptophan 2,3-dioxygenase activity
def: "Catalysis of the reaction: L-tryptophan + O2 = N-formyl-L-kynurenine." [EC:1.13.11.11]
GO:0004834
name: tryptophan synthase activity
def: "Catalysis of the reaction: L-serine + (1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate = L-tryptophan + glyceraldehyde 3-phosphate + H2O." [RHEA:10535]
GO:0004835
name: tubulin-tyrosine ligase activity
def: "Catalysis of the reaction: ATP + detyrosinated alpha-tubulin + L-tyrosine = alpha-tubulin + ADP + phosphate." [EC:6.3.2.25]
GO:0004836
name: tyramine-beta hydroxylase activity
def: "Catalysis of the hydroxylation of tyramine to form octopamine." [PMID:10745161]
GO:0004837
name: tyrosine decarboxylase activity
def: "Catalysis of the reaction: L-tyrosine = tyramine + CO2." [EC:4.1.1.25]
GO:0004838
name: L-tyrosine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-tyrosine + 2-oxoglutarate = 4-hydroxyphenylpyruvate + L-glutamate." [EC:2.6.1.5]
GO:0004839
name: ubiquitin activating enzyme activity
def: "Catalysis of the reaction: E1 + ubiquitin + ATP--> E1-ubiquitin + AMP + PPi, where the E1-ubiquitin linkage is a thioester bond between the C-terminal glycine of Ub and a sulfhydryl side group of an E1 cysteine residue. This is the first step in a cascade of reactions in which ubiquitin is ultimately added to a protein substrate." [GOC:BioGRID, http://www.bostonbiochem.com/E-301.html, PMID:10072378]
GO:0004842
name: ubiquitin-protein transferase activity
def: "Catalysis of the transfer of ubiquitin from one protein to another via the reaction X-Ub + Y --> Y-Ub + X, where both X-Ub and Y-Ub are covalent linkages." [GOC:BioGRID, GOC:jh2, PMID:9635407]
GO:0004843
name: ubiquitin-specific protease activity
def: "Catalysis of the thiol-dependent hydrolysis of a peptide bond formed by the C-terminal glycine of ubiquitin." [GOC:jh2, ISBN:0120793709]
GO:0004844
name: uracil DNA N-glycosylase activity
def: "Catalysis of the cleavage of the N-C1' glycosidic bond between the damaged DNA base and the deoxyribose sugar, releasing a free base and leaving an apyrimidinic (AP) site. Enzymes with this activity recognize and remove uracil bases in DNA that result from the deamination of cytosine or the misincorporation of dUTP opposite an adenine." [GOC:elh, GOC:pr, PMID:9224623]
GO:0004845
name: uracil phosphoribosyltransferase activity
def: "Catalysis of the reaction: diphosphate + UMP = 5-phospho-alpha-D-ribose 1-diphosphate + uracil." [EC:2.4.2.9, RHEA:13020]
GO:0004846
name: urate oxidase activity
def: "Catalysis of the reaction: urate + O2 + H2O = 5-hydroxyisourate + hydrogen peroxide." [EC:1.7.3.3]
GO:0004847
name: urea carboxylase activity
def: "Catalysis of the reaction: ATP + bicarbonate + urea = ADP + 2 H(+) + phosphate + urea-1-carboxylate." [EC:6.3.4.6, RHEA:20899]
GO:0004848
name: ureidoglycolate hydrolase activity
def: "Catalysis of the reaction: (S)-ureidoglycolate + H(2)O + 2 H(+) = CO(2) + glyoxylate + 2 NH(4)(+)." [EC:3.5.3.19, RHEA:19812]
GO:0004849
name: uridine kinase activity
def: "Catalysis of the reaction: ATP + uridine = ADP + UMP." [EC:2.7.1.48]
GO:0004850
name: uridine phosphorylase activity
def: "Catalysis of the reaction: uridine + phosphate = uracil + alpha-D-ribose 1-phosphate." [EC:2.4.2.3]
GO:0004851
name: uroporphyrin-III C-methyltransferase activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + uroporphyrin III = 2 S-adenosyl-L-homocysteine + precorrin-2." [EC:2.1.1.107]
GO:0004852
name: uroporphyrinogen-III synthase activity
def: "Catalysis of the reaction: hydroxymethylbilane = H(2)O + uroporphyrinogen III." [EC:4.2.1.75, RHEA:18968]
GO:0004853
name: uroporphyrinogen decarboxylase activity
def: "Catalysis of the reaction: uroporphyrinogen-III = coproporphyrinogen + 4 CO2." [EC:4.1.1.37]
GO:0004854
name: xanthine dehydrogenase activity
def: "Catalysis of the reaction: xanthine + NAD+ + H2O = urate + NADH + H+." [EC:1.17.1.4]
GO:0004855
name: xanthine oxidase activity
def: "Catalysis of the reaction: xanthine + H2O + O2 = urate + hydrogen peroxide." [EC:1.17.3.2]
GO:0004856
name: xylulokinase activity
def: "Catalysis of the reaction: D-xylulose + ATP = D-xylulose 5-phosphate + ADP + 2 H(+)." [EC:2.7.1.17, RHEA:10967]
GO:0004857
name: enzyme inhibitor activity
def: "Stops, prevents or reduces the activity of an enzyme." [GOC:ai]
GO:0004858
name: dUTP pyrophosphatase inhibitor activity
def: "Stops, prevents or reduces the activity of dUTP pyrophosphatase." [GOC:mah]
GO:0004859
name: phospholipase inhibitor activity
def: "Stops, prevents or reduces the activity of a phospholipase, an enzyme that catalyzes of the hydrolysis of a phospholipid." [GOC:ai, GOC:rl]
GO:0004860
name: protein kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a protein kinase, an enzyme which phosphorylates a protein." [GOC:ai]
GO:0004861
name: cyclin-dependent protein serine/threonine kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a cyclin-dependent protein serine/threonine kinase." [GOC:mah, GOC:pr]
GO:0004862
name: cAMP-dependent protein kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a cAMP-dependent protein kinase." [GOC:mah]
GO:0004864
name: protein phosphatase inhibitor activity
def: "Stops, prevents or reduces the activity of a protein phosphatase, an enzyme that hydrolyzes phosphate groups from phosphorylated proteins." [GOC:ai]
GO:0004865
name: protein serine/threonine phosphatase inhibitor activity
def: "Stops, prevents or reduces the activity of a serine/threonine protein phosphatase, an enzyme that catalyzes the reaction: protein serine/threonine phosphate + H2O = protein serine/threonine + phosphate." [GOC:dph, GOC:tb]
GO:0004866
name: endopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of an endopeptidase, any enzyme that hydrolyzes nonterminal peptide bonds in polypeptides." [GOC:jl]
GO:0004867
name: serine-type endopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of serine-type endopeptidases, enzymes that catalyze the hydrolysis of nonterminal peptide bonds in a polypeptide chain; a serine residue (and a histidine residue) are at the active center of the enzyme." [GOC:ai]
GO:0004868
name: serpin
def: "OBSOLETE. A superfamily of proteins, many of which inhibit serine proteinases and exhibit a high degree of homology with classical serine proteinase inhibitors such as alpha1-antitrypsin or antithrombin." [GOC:ai, ISBN:0198506732]
GO:0004869
name: cysteine-type endopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of a cysteine-type endopeptidase, any enzyme that hydrolyzes peptide bonds in polypeptides by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:dph, GOC:tb]
GO:0004871
name: signal transducer activity
def: "Conveys a signal across a cell to trigger a change in cell function or state. A signal is a physical entity or change in state that is used to transfer information in order to trigger a response." [GOC:go_curators]
GO:0004872
name: receptor activity
def: "Combining with an extracellular or intracellular messenger to initiate a change in cell activity." [GOC:ceb, ISBN:0198506732]
GO:0004873
name: asialoglycoprotein receptor activity
def: "Receiving an asialoglycoprotein, and delivering the asialoglycoprotein into the cell via endocytosis. An asialoglycoprotein is a plasma glycoproteins from which the terminal sialic acid residue on their complex carbohydrate groups has been removed. The asialoglycoprotein receptor recognizes the terminal galactose and N-acetylgalactosamine units of the asialoglycoprotein, the receptor-ligand complex is internalized and transported to a sorting organelle where disassociation occurs before the receptor is recycled to the cell membrane." [GOC:bf, PMID:11278827, PMID:7624395, Wikipedia:Asialoglycoprotein]
GO:0004874
name: aryl hydrocarbon receptor activity
def: "Combining with an aryl hydrocarbon and transmitting the signal to initiate a change in cell activity. The aryl hydrocarbon receptor is a ligand-activated transcription factor which translocates to the nucleus to activate transcription upon ligand-binding." [GOC:ai, GOC:signaling, PMID:1325649, PMID:7493958]
GO:0004875
name: complement receptor activity
def: "Combining with any component or product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149, PMID:11884446]
GO:0004876
name: complement component C3a receptor activity
def: "Combining with the C3a product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0004877
name: complement component C3b receptor activity
def: "Combining with the C3b product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0004878
name: complement component C5a receptor activity
def: "Combining with the C5a product of the complement cascade and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0004879
name: ligand-activated sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Combining with a signal and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, GOC:txnOH]
GO:0004880
name: juvenile hormone receptor activity
def: "Combining with juvenile hormone and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:15475158]
GO:0004882
name: androgen receptor activity
def: "Combining with an androgen and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with an androgen response element in DNA in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:19372015]
GO:0004883
name: glucocorticoid receptor activity
def: "Combining with a glucocorticoid and transmitting the signal within the cell trigger a change in cell activity or function." [GOC:signaling, PMID:17689856, PMID:20920967]
GO:0004884
name: ecdysteroid hormone receptor activity
def: "Combining with an ecdysteroid hormone and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:1653701]
GO:0004886
name: 9-cis retinoic acid receptor activity
def: "Combining with 9-cis retinoic acid and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:17132853]
GO:0004887
name: thyroid hormone receptor activity
def: "Combining with thyroid hormone and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling]
GO:0004888
name: transmembrane signaling receptor activity
def: "Combining with an extracellular or intracellular signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:go_curators, Wikipedia:Transmembrane_receptor]
GO:0004889
name: acetylcholine-activated cation-selective channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens upon binding acetylcholine." [GOC:mah, PMID:2466967]
GO:0004890
name: GABA-A receptor activity
def: "Combining with the amino acid gamma-aminobutyric acid (GABA, 4-aminobutyrate) to initiate a change in cell activity. GABA-A receptors function as chloride channels." [PMID:8974333]
GO:0004892
name: B cell receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0004894
name: T cell receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0004895
name: cell adhesion receptor activity
def: "OBSOLETE. Combining with cell adhesion molecules to initiate a change in cell activity." [GOC:ai]
GO:0004896
name: cytokine receptor activity
def: "Combining with a cytokine and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:mah]
GO:0004897
name: ciliary neurotrophic factor receptor activity
def: "Combining with ciliary neurotrophic factor (CNTF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0004898
name: gp130
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0004900
name: erythropoietin receptor activity
def: "Combining with erythropoietin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0004901
name: granulocyte macrophage colony-stimulating factor receptor activity
def: "Combining with granulocyte macrophage colony-stimulating factor (GM-CSF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0004902
name: granulocyte colony-stimulating factor receptor activity
def: "Combining with granulocyte colony-stimulating factor (G-CSF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0004903
name: growth hormone receptor activity
def: "Combining with a growth hormone and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0004904
name: interferon receptor activity
def: "Combining with an interferon and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:ai, GOC:signaling, PMID:9607096]
GO:0004905
name: type I interferon receptor activity
def: "Combining with a type I interferon and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. Type I interferons include the interferon-alpha, beta, delta, epsilon, zeta, kappa, tau, and omega gene families." [GOC:add, GOC:signaling, ISBN:0126896631, PMID:15546383, PMID:16681834, PR:000025848]
GO:0004906
name: interferon-gamma receptor activity
def: "Combining with interferon-gamma (a type II interferon) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:ai, GOC:signaling, ISBN:0126896631, PMID:15546383]
GO:0004908
name: interleukin-1 receptor activity
def: "Combining with interleukin-1 to initiate a change in cell activity. Interleukin-1 is produced mainly by activated macrophages and is involved in the inflammatory response." [GOC:jl]
GO:0004909
name: interleukin-1, Type I, activating receptor activity
def: "Combining with interleukin-1 to initiate a change in cell activity via signaling pathways and mediated by adaptor proteins." [PMID:15062641, PMID:18613828]
GO:0004910
name: interleukin-1, Type II, blocking receptor activity
def: "Combining with interleukin-1 to initiate a change in cell activity by inhibiting the activity of type I interleukin receptors." [PMID:15062641, PMID:18613828]
GO:0004911
name: interleukin-2 receptor activity
def: "Combining with interleukin-2 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004912
name: interleukin-3 receptor activity
def: "Combining with interleukin-3 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004913
name: interleukin-4 receptor activity
def: "Combining with interleukin-4 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004914
name: interleukin-5 receptor activity
def: "Combining with interleukin-5 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004915
name: interleukin-6 receptor activity
def: "Combining with interleukin-6 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004917
name: interleukin-7 receptor activity
def: "Combining with interleukin-7 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004918
name: interleukin-8 receptor activity
def: "Combining with interleukin-8 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004919
name: interleukin-9 receptor activity
def: "Combining with interleukin-9 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004920
name: interleukin-10 receptor activity
def: "Combining with interleukin-10 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004921
name: interleukin-11 receptor activity
def: "Combining with interleukin-11 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0004923
name: leukemia inhibitory factor receptor activity
def: "Combining with leukemia inhibitory factor (LIF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:mah, GOC:signaling]
GO:0004924
name: oncostatin-M receptor activity
def: "Combining with oncostatin-M and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0004925
name: prolactin receptor activity
def: "Combining with prolactin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0004927
name: sevenless receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0004928
name: frizzled receptor activity
def: "OBSOLETE. Combining with a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:go_curators]
GO:0004929
name: frizzled-2 receptor activity
def: "OBSOLETE. Combining with to a member of the Wnt-family of signaling molecules to initiate a change in cell activity." [GOC:go_curators]
GO:0004930
name: G-protein coupled receptor activity
def: "Combining with an extracellular signal and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, http://www.iuphar-db.org, Wikipedia:GPCR]
GO:0004931
name: extracellular ATP-gated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when extracellular ATP has been bound by the channel complex or one of its constituent parts." [GOC:bf, GOC:mah, PMID:9755289]
GO:0004932
name: mating-type factor pheromone receptor activity
def: "Combining with a mating-type factor pheromone to initiate a change in cell activity." [GOC:dph, GOC:vw]
GO:0004933
name: mating-type a-factor pheromone receptor activity
def: "Combining with the mating-type a-factor pheromone to initiate a change in cell activity." [GOC:mah]
GO:0004934
name: mating-type alpha-factor pheromone receptor activity
def: "Combining with the mating-type alpha-factor pheromone to initiate a change in cell activity." [GOC:mah]
GO:0004935
name: adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine and transmitting the signal across the membrane by activating the alpha-subunit of an associated heterotrimeric G-protein complex." [GOC:bf, GOC:mah, IUPHAR_GPCR:1274]
GO:0004936
name: alpha-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of alpha-adrenergic receptors." [GOC:mah, IUPHAR_GPCR:1274]
GO:0004937
name: alpha1-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of alpha1-adrenergic receptors; the activity involves transmitting the signal to the Gq alpha subunit of a heterotrimeric G protein." [GOC:cb, GOC:mah, IUPHAR_GPCR:1274]
GO:0004938
name: alpha2-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of alpha2-adrenergic receptors; the activity involves transmitting the signal to the Gi alpha subunit of a heterotrimeric G protein." [GOC:cb, GOC:mah, IUPHAR_GPCR:1274]
GO:0004939
name: beta-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of beta-adrenergic receptors; the activity involves transmitting the signal to the Gs alpha subunit of a heterotrimeric G protein." [GOC:cb, GOC:mah, IUPHAR_GPCR:1274]
GO:0004940
name: beta1-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of beta1-adrenergic receptors." [GOC:mah, IUPHAR_GPCR:1274]
GO:0004941
name: beta2-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of beta2-adrenergic receptors." [GOC:mah, IUPHAR_GPCR:1274]
GO:0004942
name: anaphylatoxin receptor activity
def: "Combining with anaphylatoxin to initiate a change in cell activity." [GOC:ai]
GO:0004943
name: C3a anaphylatoxin receptor activity
def: "Combining with anaphylatoxin C3a to initiate a change in cell activity." [GOC:mah]
GO:0004944
name: C5a anaphylatoxin receptor activity
def: "Combining with anaphylatoxin C5a to initiate a change in cell activity." [GOC:mah]
GO:0004945
name: angiotensin type II receptor activity
def: "An angiotensin receptor activity that acts via Gi protein coupling and cGMP (NO) generation, and may also act via additional signaling mechanisms." [GOC:mah, PMID:10977869]
GO:0004946
name: bombesin receptor activity
def: "Combining with bombesin to initiate a change in cell activity." [GOC:ai]
GO:0004947
name: bradykinin receptor activity
def: "Combining with bradykinin to initiate a change in cell activity." [GOC:ai]
GO:0004948
name: calcitonin receptor activity
def: "Combining with calcitonin and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:mah, GOC:signaling, PMID:21649645]
GO:0004949
name: cannabinoid receptor activity
def: "Combining with a cannabinoid to initiate a change in cell activity. Cannabinoids are a class of diverse chemical compounds that include the endocannabinoids and the phytocannabinoids." [GOC:dph, IUPHAR_GPCR:1279, Wikipedia:Cannabinoid]
GO:0004950
name: chemokine receptor activity
def: "Combining with a chemokine, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. Chemokines are a family of small chemotactic cytokines; their name is derived from their ability to induce directed chemotaxis in nearby responsive cells. All chemokines possess a number of conserved cysteine residues involved in intramolecular disulfide bond formation. Some chemokines are considered pro-inflammatory and can be induced during an immune response to recruit cells of the immune system to a site of infection, while others are considered homeostatic and are involved in controlling the migration of cells during normal processes of tissue maintenance or development. Chemokines are found in all vertebrates, some viruses and some bacteria." [GOC:BHF, GOC:rl, GOC:signaling, http://en.wikipedia.org/wiki/Chemokine, IUPHAR_GPCR:1280, PMID:12183377, PMID:8662823]
GO:0004951
name: cholecystokinin receptor activity
def: "Combining with cholecystokinin and transmitting the signal across the membrane by activating an associated G-protein to initiate a change in cell activity. Cholecystokinin can act as a neuropeptide or as a gastrointestinal hormone." [GOC:signaling, PMID:9835394]
GO:0004952
name: dopamine neurotransmitter receptor activity
def: "Combining with the neurotransmitter dopamine to initiate a change in cell activity." [IUPHAR_GPCR:1282]
GO:0004953
name: icosanoid receptor activity
def: "Combining with an icosanoid to initiate a change in cell activity." [GOC:dph]
GO:0004954
name: prostanoid receptor activity
def: "Combining with a prostanoid, any compound based on or derived from the prostanoate structure, to initiate a change in cell activity." [ISBN:0198506732]
GO:0004955
name: prostaglandin receptor activity
def: "Combining with a prostaglandin (PG) to initiate a change in cell activity." [ISBN:0198506732]
GO:0004956
name: prostaglandin D receptor activity
def: "Combining with prostaglandin D (PGD(2)) to initiate a change in cell activity." [ISBN:0198506732]
GO:0004957
name: prostaglandin E receptor activity
def: "Combining with prostaglandin E (PGE(2)) to initiate a change in cell activity." [ISBN:0198506732]
GO:0004958
name: prostaglandin F receptor activity
def: "Combining with prostaglandin F (PGF (2-alpha)) to initiate a change in cell activity." [ISBN:0198506732]
GO:0004960
name: thromboxane receptor activity
def: "Combining with a thromboxane (TXA) to initiate a change in cell activity." [ISBN:0198506732]
GO:0004961
name: thromboxane A2 receptor activity
def: "Combining with thromboxane A2 (TXA(2)) and transmitting the signal across the membrane to activate an associated G-protein." [GOC:signaling, ISBN:0198506732]
GO:0004962
name: endothelin receptor activity
def: "Combining with endothelin and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:dph, GOC:signaling, IUPHAR_GPCR:1283, IUPHAR_RECEPTOR:2263, IUPHAR_RECEPTOR:2265]
GO:0004963
name: follicle-stimulating hormone receptor activity
def: "Combining with follicle-stimulating hormone to initiate a change in cell activity." [GOC:mah]
GO:0004964
name: luteinizing hormone receptor activity
def: "Combining with luteinizing hormone (also called lutropin) to initiate a change in cell activity." [ISBN:0198506732, PMID:18848524, PMID:1922095]
GO:0004965
name: G-protein coupled GABA receptor activity
def: "Combining with the amino acid gamma-aminobutyric acid (GABA, 4-aminobutyrate) and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:ai, GOC:bf, IUPHAR_RECEPTOR:1276, Wikipedia:GABAB_receptor]
GO:0004966
name: galanin receptor activity
def: "Combining with galanin to initiate a change in cell activity." [GOC:ai]
GO:0004967
name: glucagon receptor activity
def: "Combining with glucagon and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:mah, GOC:signaling, PMID:22438981]
GO:0004968
name: gonadotropin-releasing hormone receptor activity
def: "Combining with gonadotropin-releasing hormone to initiate a change in cell activity. Gonadotropin-releasing hormone (GnRH) is a peptide hormone responsible for the release of follicle-stimulating hormone (FSH) and luteinizing hormone (LH) from the anterior pituitary. GnRH is synthesized and released by the hypothalamus." [GOC:mah]
GO:0004969
name: histamine receptor activity
def: "Combining with histamine to initiate a change in cell activity. Histamine is a physiologically active amine, found in plant and animal tissue and released from mast cells as part of an allergic reaction in humans." [GOC:ai]
GO:0004970
name: ionotropic glutamate receptor activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when glutamate has been bound by the channel complex or one of its constituent parts." [ISBN:0198506732]
GO:0004971
name: alpha-amino-3-hydroxy-5-methyl-4-isoxazole propionate selective glutamate receptor activity
def: "An ionotropic glutamate receptor activity that exhibits fast gating by glutamate and acts by opening a cation channel permeable to sodium, potassium, and, in the absence of a GluR2 subunit, calcium." [GOC:mah, PMID:10049997, PMID:8804111]
GO:0004972
name: N-methyl-D-aspartate selective glutamate receptor activity
def: "An ionotropic glutamate receptor activity that exhibits slow gating by glutamate, acts by opening a nonselective cation channel, and can be activated by L-aspartate." [GOC:mah, PMID:10049997]
GO:0004973
name: N-methyl-D-aspartate receptor-associated protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0004974
name: leukotriene receptor activity
def: "Combining with a leukotriene to initiate a change in cell activity. Leukotrienes are pharmacologically active substances with a set of three conjugated double bonds; some contain a peptide group based on cysteine." [GOC:ai, ISBN:0198506732]
GO:0004977
name: melanocortin receptor activity
def: "Combining with melanocortin to initiate a change in cell activity." [GOC:ai]
GO:0004978
name: corticotropin receptor activity
def: "Combining with corticotropin to initiate a change in cell activity." [GOC:ai]
GO:0004979
name: beta-endorphin receptor activity
def: "Combining with beta-endorphin, and transmitting the signal across the membrane by activating an associated G-protein. Beta-endorphin is a peptide, 31 amino acids long, resulting from processing of the precursor proopiomelanocortin (POMC)." [GOC:ai, GOC:bf, Wikipedia:Beta-endorphin]
GO:0004980
name: melanocyte-stimulating hormone receptor activity
def: "Combining with melanocyte-stimulating hormone to initiate a change in cell activity." [GOC:jl, PMID:7581459]
GO:0004982
name: N-formyl peptide receptor activity
def: "Combining with an N-formyl peptide to initiate a change in cell activity." [GOC:ai]
GO:0004983
name: neuropeptide Y receptor activity
def: "Combining with neuropeptide Y to initiate a change in cell activity." [PMID:9315606]
GO:0004984
name: olfactory receptor activity
def: "Combining with an odorant and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity in response to detection of smell." [GOC:bf, GOC:dph, GOC:sart, PMID:19135896, PMID:21041441]
GO:0004985
name: opioid receptor activity
def: "Combining with an opioid (any narcotic derived from or resembling opium), and transmitting the signal across the membrane by activating an associated G-protein." [CHEBI:60598, GOC:ai, GOC:bf, PMID:20494127]
GO:0004986
name: delta-opioid receptor activity
def: "OBSOLETE. Combining with an opioid to initiate a change in cell activity, with the pharmacological characteristics of delta-opioid receptors, including the activity of enkephalins as ligands." [IUPHAR_RECEPTOR:317, PMID:10471416]
GO:0004987
name: kappa-opioid receptor activity
def: "OBSOLETE. Combining with an opioid to initiate a change in cell activity, with the pharmacological characteristics of kappa-opioid receptors, including high affinity for dynorphins." [IUPHAR_RECEPTOR:318]
GO:0004988
name: mu-opioid receptor activity
def: "OBSOLETE. Combining with an opioid to initiate a change in cell activity, with the pharmacological characteristics of mu-opioid receptors, including high affinity for enkephalins and beta-endorphin but low affinity for dynorphins." [IUPHAR_RECEPTOR:319]
GO:0004989
name: octopamine receptor activity
def: "Combining with the biogenic amine octopamine to initiate a change in cell activity. Octopamine is found in both vertebrates and invertebrates and can have properties both of a hormone and a neurotransmitter and acts as an adrenergic agonist." [CHEBI:17134, GOC:ai]
GO:0004990
name: oxytocin receptor activity
def: "Combining with oxytocin to initiate a change in cell activity." [GOC:ai]
GO:0004991
name: parathyroid hormone receptor activity
def: "Combining with parathyroid hormone to initiate a change in cell activity." [GOC:mah]
GO:0004992
name: platelet activating factor receptor activity
def: "Combining with platelet activating factor to initiate a change in cell activity." [GOC:mah]
GO:0004993
name: serotonin receptor activity
def: "Combining with the biogenic amine serotonin and transmitting the signal across the membrane by activating an associated G-protein. Serotonin (5-hydroxytryptamine) is a neurotransmitter and hormone found in vertebrates and invertebrates." [CHEBI:28790, GOC:ai]
GO:0004994
name: somatostatin receptor activity
def: "Combining with somatostatin to initiate a change in cell activity. Somatostatin is a peptide hormone that regulates the endocrine system by signaling via G-protein coupled somatostatin receptors. Somatostatin has two active forms produced by proteolytic cleavage: a 14 amino acid peptide (SST-14) and a 28 amino acid peptide (SST-28)." [GOC:ai, GOC:bf, Wikipedia:Somatostatin]
GO:0004995
name: tachykinin receptor activity
def: "Combining with a tachykinin neuropeptide and transmitting the signal across the membrane by activating an associated G-protein." [GOC:ai, GOC:bf, PMID:7639617, Wikipedia:Tachykinin]
GO:0004996
name: thyroid-stimulating hormone receptor activity
def: "Combining with thyroid-stimulating hormone to initiate a change in cell activity." [GOC:mah]
GO:0004997
name: thyrotropin-releasing hormone receptor activity
def: "Combining with thyrotropin-releasing hormone to initiate a change in cell activity." [GOC:mah]
GO:0004998
name: transferrin receptor activity
def: "Combining selectively with transferrin, and delivering transferrin into the cell via endocytosis. Transferrin is a major iron carrier protein in vertebrates." [GOC:bf, PMID:2678449, PMID:3011819]
GO:0004999
name: vasoactive intestinal polypeptide receptor activity
def: "Combining with vasoactive intestinal polypeptide to initiate a change in cell activity." [GOC:mah]
GO:0005000
name: vasopressin receptor activity
def: "Combining with vasopressin to initiate a change in cell activity." [GOC:ai]
GO:0005001
name: transmembrane receptor protein tyrosine phosphatase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: protein tyrosine phosphate + H2O = protein tyrosine + phosphate." [EC:3.1.3.48]
GO:0005003
name: ephrin receptor activity
def: "Combining with an ephrin to initiate a change in cell activity." [GOC:mah, PMID:9530499]
GO:0005004
name: GPI-linked ephrin receptor activity
def: "Combining with a GPI-anchored ephrin to initiate a change in cell activity." [GOC:mah, PMID:9530499]
GO:0005005
name: transmembrane-ephrin receptor activity
def: "Combining with a transmembrane ephrin to initiate a change in cell activity." [GOC:mah, PMID:9530499]
GO:0005006
name: epidermal growth factor-activated receptor activity
def: "Combining with an epidermal growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf]
GO:0005007
name: fibroblast growth factor-activated receptor activity
def: "Combining with a fibroblast growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:mah]
GO:0005008
name: hepatocyte growth factor-activated receptor activity
def: "Combining with hepatocyte growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:mah]
GO:0005009
name: insulin-activated receptor activity
def: "Combining with insulin and transmitting the signal across the plasma membrane to initiate a change in cell activity." [ISBN:0198506732]
GO:0005010
name: insulin-like growth factor-activated receptor activity
def: "Combining with insulin-like growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:mah]
GO:0005011
name: macrophage colony-stimulating factor receptor activity
def: "Combining with macrophage colony-stimulating factor (M-CSF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate." [GOC:mah, GOC:signaling]
GO:0005012
name: Neu/ErbB-2 receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005013
name: neurotrophin TRK receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jid]
GO:0005014
name: neurotrophin TRKA receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jid]
GO:0005015
name: neurotrophin TRKB receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jid]
GO:0005016
name: neurotrophin TRKC receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jid]
GO:0005017
name: platelet-derived growth factor-activated receptor activity
def: "Combining with platelet-derived growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:mah]
GO:0005018
name: platelet-derived growth factor alpha-receptor activity
def: "Combining with platelet-derived growth factor isoform PDGF-AA, PDGF-BB or PDGF-AB to initiate a change in cell activity." [PMID:1657917]
GO:0005019
name: platelet-derived growth factor beta-receptor activity
def: "Combining with platelet-derived growth factor isoform PDGF-BB or PDGF-AB to initiate a change in cell activity." [PMID:1657917]
GO:0005020
name: stem cell factor receptor activity
def: "Combining with stem cell factor (SCF) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate. Stem cell factor is a cytokine that stimulates mast cell growth and differentiation." [GOC:jl, GOC:signaling, PMID:10698217]
GO:0005021
name: vascular endothelial growth factor-activated receptor activity
def: "Combining with a vascular endothelial growth factor (VEGF) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0005024
name: transforming growth factor beta-activated receptor activity
def: "Combining with a transforming growth factor beta (TGFbeta) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP protein serine = ADP + protein serine phosphate, and ATP + protein threonine = ADP + protein threonine phosphate." [GOC:mah, GOC:signaling]
GO:0005025
name: transforming growth factor beta receptor activity, type I
def: "Combining with a complex of transforming growth factor beta and a type II TGF-beta receptor to initiate a change in cell activity; upon binding, acts as a downstream transducer of TGF-beta signals." [GOC:mah, Reactome:REACT_6945.1]
GO:0005026
name: transforming growth factor beta receptor activity, type II
def: "Combining with transforming growth factor beta to initiate a change in cell activity; upon ligand binding, binds to and catalyzes the phosphorylation of a type I TGF-beta receptor." [GOC:mah, Reactome:REACT_6872.1]
GO:0005027
name: NGF/TNF (6 C-domain) receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005028
name: CD40 receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005029
name: CD27 receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005030
name: neurotrophin receptor activity
def: "Combining with a neurotrophin, any of a family of growth factors that prevent apoptosis in neurons and promote nerve growth, and transmitting the signal to initiate a change in cell activity." [GOC:jl, GOC:signaling, http://www.mercksource.com/]
GO:0005031
name: tumor necrosis factor-activated receptor activity
def: "Combining with tumor necrosis factor, a proinflammatory cytokine produced by monocytes and macrophages, to initiate a change in cell function." [GOC:jl, http://lookwayup.com/]
GO:0005034
name: osmosensor activity
def: "Sensing extracellular osmolarity to initiate a change in cell activity, and spanning the membrane of the cell." [GOC:dph, GOC:tb]
GO:0005035
name: death receptor activity
def: "Combining with an extracellular messenger (called a death ligand), and transmitting the signal from one side of the plasma membrane to the other to initiate apoptotic or necrotic cell death." [GOC:bf, GOC:BHF, GOC:ecd, GOC:mtg_apoptosis, GOC:rl, PMID:10209153]
GO:0005037
name: death receptor adaptor protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005038
name: death receptor interacting protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005039
name: death receptor-associated factor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005040
name: decoy death receptor activity
def: "Combining with an extracellular messenger (death ligand) without transmission of the signal. Decoy death receptors compete with death receptors for ligand binding, and do not initiate apoptosis." [GOC:bf, GOC:ecd, PMID:10209153]
GO:0005041
name: low-density lipoprotein receptor activity
def: "Combining with a low-density lipoprotein particle and delivering the low-density lipoprotein into the cell via endocytosis." [GOC:bf, ISBN:0198506732]
GO:0005042
name: netrin receptor activity
def: "Combining with a netrin signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:dph, GOC:signaling, PMID:15960985]
GO:0005043
name: netrin receptor activity involved in chemorepulsion
def: "Combining with a netrin signal and transmitting the signal from one side of the membrane to the other to contribute to the directed movement of a motile cell away from a higher concentration of netrin." [GOC:dph, GOC:signaling]
GO:0005044
name: scavenger receptor activity
def: "Combining with any modified low-density lipoprotein (LDL) or other polyanionic ligand and delivering the ligand into the cell via endocytosis. Ligands include acetylated and oxidized LDL, Gram-positive and Gram-negative bacteria, apoptotic cells, beta-amyloid fibrils, and advanced glycation end products (AGEs)." [GOC:bf, PMID:11790542, PMID:12379907, PMID:12621157, PMID:20981357]
GO:0005045
name: endoplasmic reticulum receptor activity
def: "OBSOLETE. A receptor in the endoplasmic reticulum." [GOC:ai]
GO:0005046
name: KDEL sequence binding
def: "Interacting selectively and non-covalently with a KDEL sequence, the C terminus tetrapeptide sequence Lys-Asp-Glu-Leu found in proteins that are to be retained in the endoplasmic reticulum." [GOC:ai]
GO:0005047
name: signal recognition particle binding
def: "Interacting selectively and non-covalently with the signal recognition particle." [ISBN:0198506732]
GO:0005048
name: signal sequence binding
def: "Interacting selectively and non-covalently with a signal sequence, a specific peptide sequence found on protein precursors or mature proteins that dictates where the mature protein is localized." [GOC:ai]
GO:0005049
name: nuclear export signal receptor activity
def: "Combining with a nuclear export signal (NES) to mediate transport of the NES-containing protein through the nuclear pore to the cytoplasm." [GOC:bf, PMID:11743003]
GO:0005050
name: peroxisome receptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005052
name: peroxisome matrix targeting signal-1 binding
def: "Interacting selectively and non-covalently with a type 1 peroxisome targeting signal, a tripeptide with the consensus sequence (S/A/C)-(K/R/H)-L." [GOC:mah, PMID:11687502]
GO:0005053
name: peroxisome matrix targeting signal-2 binding
def: "Interacting selectively and non-covalently with a type 2 peroxisome targeting signal, a nonapeptide with a broad consensus sequence of (R/K)-(L/V/I)-(XXXXX)-(H/Q)-(L/A/F)." [GOC:mah, PMID:11687502]
GO:0005054
name: peroxisome integral membrane receptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005055
name: laminin receptor activity
def: "Combining with a laminin, a glycoprotein that constitutes the majority of proteins in the basement membrane, to initiate a change in cell activity." [GOC:ai, PMID:2970671]
GO:0005056
name: tiggrin receptor activity
def: "Combining with the extracellular matrix ligand tiggrin, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling, PMID:9521906]
GO:0005057
name: receptor signaling protein activity
def: "Conveys a signal from an upstream receptor or intracellular signal transducer, converting the signal into a form where it can ultimately trigger a change in the state or activity of a cell." [GOC:bf]
GO:0005061
name: aryl hydrocarbon receptor nuclear translocator activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005065
name: heterotrimeric G-protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005066
name: transmembrane receptor protein tyrosine kinase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005068
name: transmembrane receptor protein tyrosine kinase adaptor activity
def: "The binding activity of a molecule that brings together a transmembrane receptor protein tyrosine kinase and one or more other molecules, permitting them to function in a coordinated way." [GOC:mtg_MIT_16mar07, PMID:10502414, PMID:20565848]
GO:0005070
name: SH3/SH2 adaptor activity
def: "Interacting selectively and non-covalently and simultaneously with one or more signal transduction molecules, usually acting as a scaffold to bring these molecules into close proximity either using their own SH2/SH3 domains (e.g. Grb2) or those of their target molecules (e.g. SAM68)." [GOC:mah, GOC:so]
GO:0005071
name: transmembrane receptor protein serine/threonine kinase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005072
name: transforming growth factor beta receptor, cytoplasmic mediator activity
def: "Activity of any of the molecules that transmit the signal from a TGF-beta receptor through the cytoplasm to the nucleus." [GOC:hjd]
GO:0005073
name: common-partner SMAD protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005074
name: inhibitory SMAD protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005075
name: pathway-specific SMAD protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005076
name: receptor signaling protein serine/threonine kinase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005077
name: MAP-kinase anchoring activity
def: "OBSOLETE. Binds to MAP kinase and anchors it to a particular subcellular location." [GOC:ai]
GO:0005078
name: MAP-kinase scaffold activity
def: "The binding activity of a molecule that functions as a physical support for the assembly of a multiprotein mitogen-activated protein kinase (MAPK) complex. Binds multiple kinases of the MAPKKK cascade, and also upstream signaling proteins, permitting those molecules to function in a coordinated way. Bringing together multiple enzymes and their substrates enables the signal to be transduced quickly and efficiently." [PMID:12511654, PMID:15213240, PMID:9405336]
GO:0005079
name: protein kinase A anchoring activity
def: "OBSOLETE. Binds to protein kinase A and anchors it to a particular subcellular location." [PMID:10354567]
GO:0005080
name: protein kinase C binding
def: "Interacting selectively and non-covalently with protein kinase C." [GOC:jl]
GO:0005081
name: receptor signaling protein serine/threonine phosphatase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005082
name: receptor signaling protein tyrosine phosphatase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005083
name: small GTPase regulator activity
def: "Modulates the rate of GTP hydrolysis by a small monomeric GTPase." [GOC:mah]
GO:0005085
name: guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:kd, GOC:mah]
GO:0005086
name: ARF guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with the GTPase ARF. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0005087
name: Ran guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Ran family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0005088
name: Ras guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Ras superfamily. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0005089
name: Rho guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Rho family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0005090
name: Sar guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Sar family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0005091
name: guanyl-nucleotide exchange factor adaptor activity
def: "The binding activity of a molecule that brings together a guanyl-nucleotide exchange factor and one or more other proteins, permitting them to function in a coordinated way." [GOC:mtg_MIT_16mar07, GOC:vw]
GO:0005092
name: GDP-dissociation inhibitor activity
def: "Prevents the dissociation of GDP from a GTPase, thereby preventing GTP from binding." [GOC:mah]
GO:0005093
name: Rab GDP-dissociation inhibitor activity
def: "Prevents the dissociation of GDP from the small GTPase Rab, thereby preventing GTP from binding." [GOC:mah]
GO:0005094
name: Rho GDP-dissociation inhibitor activity
def: "Prevents the dissociation of GDP from the small GTPase Rho, thereby preventing GTP from binding." [GOC:mah]
GO:0005095
name: GTPase inhibitor activity
def: "Stops, prevents or reduces the activity of any enzyme that catalyzes the hydrolysis of GTP to GDP and orthophosphate." [GOC:ai]
GO:0005096
name: GTPase activator activity
def: "Increases the activity of a GTPase, an enzyme that catalyzes the hydrolysis of GTP." [GOC:mah]
GO:0005097
name: Rab GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Rab family." [GOC:mah]
GO:0005098
name: Ran GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Ran family." [GOC:mah]
GO:0005099
name: Ras GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Ras superfamily." [GOC:mah]
GO:0005100
name: Rho GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Rho family." [GOC:mah]
GO:0005101
name: Sar GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Sar family." [GOC:mah]
GO:0005102
name: receptor binding
def: "Interacting selectively and non-covalently with one or more specific sites on a receptor molecule, a macromolecule that undergoes combination with a hormone, neurotransmitter, drug or intracellular messenger to initiate a change in cell function." [GOC:bf, GOC:ceb, ISBN:0198506732]
GO:0005104
name: fibroblast growth factor receptor binding
def: "Interacting selectively and non-covalently with the fibroblast growth factor receptor (FGFR)." [GOC:ceb]
GO:0005105
name: type 1 fibroblast growth factor receptor binding
def: "Interacting selectively and non-covalently with the type 1 fibroblast growth factor receptor (FGFR1)." [GOC:ceb, GOC:fb_curators]
GO:0005106
name: ephrin
def: "OBSOLETE. A class of proteins that interact with the ephrin receptors." [GOC:ai, ISBN:0198506732]
GO:0005107
name: GPI-linked ephrin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005108
name: transmembrane ephrin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005109
name: frizzled binding
def: "Interacting selectively and non-covalently with the frizzled (fz) receptor." [GOC:ceb, PR:000001315]
GO:0005110
name: frizzled-2 binding
def: "Interacting selectively and non-covalently with frizzled-2 (fz2)." [GOC:ceb, PR:000007719]
GO:0005111
name: type 2 fibroblast growth factor receptor binding
def: "Interacting selectively and non-covalently with the type 2 fibroblast growth factor receptor (FGFR2)." [GOC:fb_curators]
GO:0005112
name: Notch binding
def: "Interacting selectively and non-covalently with the Notch (N) protein, a surface receptor." [GOC:ceb]
GO:0005113
name: patched binding
def: "Interacting selectively and non-covalently with the patched (ptc) protein, a receptor for hedgehog proteins." [GOC:ceb, PMID:11731473]
GO:0005114
name: type II transforming growth factor beta receptor binding
def: "Interacting selectively and non-covalently with a type II transforming growth factor beta receptor." [GOC:ceb, GOC:mah, PMID:11252892]
GO:0005115
name: receptor tyrosine kinase-like orphan receptor binding
def: "Interacting selectively and non-covalently with the receptor tyrosine kinase-like orphan receptor (Ror)." [GOC:ceb, GOC:vw]
GO:0005117
name: wishful thinking binding
def: "Interacting selectively and non-covalently with wishful thinking (Wit), a type II bone morphogenic protein receptor." [GOC:ceb, PMID:11856529]
GO:0005118
name: sevenless binding
def: "Interacting selectively and non-covalently with the sevenless (sev) protein, a receptor tyrosine kinase." [GOC:ceb, PMID:3151175]
GO:0005119
name: smoothened binding
def: "Interacting selectively and non-covalently with the smoothened (smo) protein, which interacts with patched to transmit hedgehog signals." [GOC:ceb, PMID:11731473]
GO:0005121
name: Toll binding
def: "Interacting selectively and non-covalently with the Toll protein, a transmembrane receptor." [GOC:ceb]
GO:0005122
name: torso binding
def: "Interacting selectively and non-covalently with the torso (tor) protein, a receptor tyrosine kinase." [GOC:ceb, PMID:2927509]
GO:0005123
name: death receptor binding
def: "Interacting selectively and non-covalently with any member of the death receptor (DR) family. The DR family falls within the tumor necrosis factor receptor superfamily and is characterized by a cytoplasmic region of ~80 residues termed the death domain (DD)." [GOC:ceb, GOC:rl, PMID:15654015]
GO:0005124
name: scavenger receptor binding
def: "Interacting selectively and non-covalently with scavenger receptors, a family of proteins that are expressed on myeloid cells and are involved in the uptake of effete cellular components and foreign particles." [GOC:ceb]
GO:0005125
name: cytokine activity
def: "Functions to control the survival, growth, differentiation and effector function of tissues and cells." [ISBN:0198599471]
GO:0005126
name: cytokine receptor binding
def: "Interacting selectively and non-covalently with a cytokine receptor." [GOC:mah, GOC:vw]
GO:0005127
name: ciliary neurotrophic factor receptor binding
def: "Interacting selectively and non-covalently with the ciliary neurotrophic factor receptor." [GOC:ai]
GO:0005128
name: erythropoietin receptor binding
def: "Interacting selectively and non-covalently with the erythropoietin receptor." [GOC:ai]
GO:0005129
name: granulocyte macrophage colony-stimulating factor receptor binding
def: "Interacting selectively and non-covalently with the granulocyte macrophage colony-stimulating factor receptor." [GOC:ai]
GO:0005130
name: granulocyte colony-stimulating factor receptor binding
def: "Interacting selectively and non-covalently with the granulocyte colony-stimulating factor receptor." [GOC:ai]
GO:0005131
name: growth hormone receptor binding
def: "Interacting selectively and non-covalently with the growth hormone receptor." [GOC:ai]
GO:0005132
name: type I interferon receptor binding
def: "Interacting selectively and non-covalently with an interferon-type I receptor, a heterodimeric complex composed of an alpha subunit (IFNAR1) and a beta subunit (IFNAR2)." [GOC:ai, GOC:signaling, PMID:17502368]
GO:0005133
name: interferon-gamma receptor binding
def: "Interacting selectively and non-covalently with the interferon-gamma receptor." [GOC:ai]
GO:0005134
name: interleukin-2 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-2 receptor." [GOC:ai]
GO:0005135
name: interleukin-3 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-3 receptor." [GOC:ai]
GO:0005136
name: interleukin-4 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-4 receptor." [GOC:ai]
GO:0005137
name: interleukin-5 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-5 receptor." [GOC:ai]
GO:0005138
name: interleukin-6 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-6 receptor." [GOC:ai]
GO:0005139
name: interleukin-7 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-7 receptor." [GOC:ai]
GO:0005140
name: interleukin-9 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-9 receptor." [GOC:ai]
GO:0005141
name: interleukin-10 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-10 receptor." [GOC:ai]
GO:0005142
name: interleukin-11 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-11 receptor." [GOC:ai]
GO:0005143
name: interleukin-12 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-12 receptor." [GOC:ai]
GO:0005144
name: interleukin-13 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-13 receptor." [GOC:ai]
GO:0005145
name: interleukin-14 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-14 receptor." [GOC:ai]
GO:0005146
name: leukemia inhibitory factor receptor binding
def: "Interacting selectively and non-covalently with the leukemia inhibitory factor receptor." [GOC:ai]
GO:0005147
name: oncostatin-M receptor binding
def: "Interacting selectively and non-covalently with the oncostatin-M receptor." [GOC:ai]
GO:0005148
name: prolactin receptor binding
def: "Interacting selectively and non-covalently with the prolactin receptor." [GOC:ai]
GO:0005149
name: interleukin-1 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-1 receptor." [GOC:go_curators]
GO:0005150
name: interleukin-1, Type I receptor binding
def: "Interacting selectively and non-covalently with a Type I interleukin-1 receptor." [GOC:ai]
GO:0005151
name: interleukin-1, Type II receptor binding
def: "Interacting selectively and non-covalently with a Type II interleukin-1 receptor." [GOC:ai]
GO:0005152
name: interleukin-1 receptor antagonist activity
def: "Blocks the binding of interleukin-1 to the interleukin-1 receptor complex." [GOC:ebc]
GO:0005153
name: interleukin-8 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-8 receptor." [GOC:go_curators]
GO:0005154
name: epidermal growth factor receptor binding
def: "Interacting selectively and non-covalently with the epidermal growth factor receptor." [GOC:ai]
GO:0005155
name: epidermal growth factor receptor activating ligand activity
def: "OBSOLETE. This term was not defined before being made obsolete." [GOC:dph, GOC:mah, GOC:tb]
GO:0005156
name: epidermal growth factor receptor inhibiting ligand activity
def: "OBSOLETE. This term was not defined before being made obsolete." [GOC:dph, GOC:mah, GOC:tb]
GO:0005157
name: macrophage colony-stimulating factor receptor binding
def: "Interacting selectively and non-covalently with the macrophage colony-stimulating factor receptor." [GOC:ai]
GO:0005158
name: insulin receptor binding
def: "Interacting selectively and non-covalently with the insulin receptor." [GOC:ai]
GO:0005159
name: insulin-like growth factor receptor binding
def: "Interacting selectively and non-covalently with the insulin-like growth factor receptor." [GOC:jl]
GO:0005160
name: transforming growth factor beta receptor binding
def: "Interacting selectively and non-covalently with the transforming growth factor beta receptor." [GOC:ai]
GO:0005161
name: platelet-derived growth factor receptor binding
def: "Interacting selectively and non-covalently with the platelet-derived growth factor receptor." [GOC:ai]
GO:0005163
name: nerve growth factor receptor binding
def: "Interacting selectively and non-covalently with the nerve growth factor receptor." [GOC:ai, PMID:15654015]
GO:0005164
name: tumor necrosis factor receptor binding
def: "Interacting selectively and non-covalently with the tumor necrosis factor receptor." [GOC:ai]
GO:0005165
name: neurotrophin receptor binding
def: "Interacting selectively and non-covalently with a neurotrophin receptor." [GOC:ai]
GO:0005166
name: neurotrophin p75 receptor binding
def: "Interacting selectively and non-covalently with the neurotrophin p75 receptor." [GOC:ai]
GO:0005167
name: neurotrophin TRK receptor binding
def: "Interacting selectively and non-covalently with a neurotrophin TRK receptor." [GOC:ai]
GO:0005168
name: neurotrophin TRKA receptor binding
def: "Interacting selectively and non-covalently with the neurotrophin TRKA receptor." [GOC:ai]
GO:0005169
name: neurotrophin TRKB receptor binding
def: "Interacting selectively and non-covalently with the neurotrophin TRKB receptor." [GOC:ai]
GO:0005170
name: neurotrophin TRKC receptor binding
def: "Interacting selectively and non-covalently with the neurotrophin TRKC receptor." [GOC:ai]
GO:0005171
name: hepatocyte growth factor receptor binding
def: "Interacting selectively and non-covalently with the hepatocyte growth factor receptor." [GOC:ai]
GO:0005172
name: vascular endothelial growth factor receptor binding
def: "Interacting selectively and non-covalently with any vascular endothelial growth factor receptor." [GOC:ai]
GO:0005173
name: stem cell factor receptor binding
def: "Interacting selectively and non-covalently with the stem cell factor receptor (SCFR), a type III transmembrane kinase receptor." [GOC:jl, PMID:10698217]
GO:0005174
name: CD40 receptor binding
def: "Interacting selectively and non-covalently with CD40, a receptor found on the surface of all B-lymphocytes." [GOC:jl, ISBN:0120781859]
GO:0005175
name: CD27 receptor binding
def: "Interacting selectively and non-covalently with a CD27, a receptor found on the surface of T cells and some B cells and NK cells." [GOC:jl, ISBN:0120781859]
GO:0005176
name: ErbB-2 class receptor binding
def: "Interacting selectively and non-covalently with the protein-tyrosine kinase receptor Neu/ErbB-2/HER2." [GOC:jl]
GO:0005177
name: neuroligin
def: "OBSOLETE. A class of ligands for neurexins." [GOC:ai]
GO:0005178
name: integrin binding
def: "Interacting selectively and non-covalently with an integrin." [GOC:ceb]
GO:0005179
name: hormone activity
def: "The action characteristic of a hormone, any substance formed in very small amounts in one specialized organ or group of cells and carried (sometimes in the bloodstream) to another organ or group of cells in the same organism, upon which it has a specific regulatory action. The term was originally applied to agents with a stimulatory physiological action in vertebrate animals (as opposed to a chalone, which has a depressant action). Usage is now extended to regulatory compounds in lower animals and plants, and to synthetic substances having comparable effects; all bind receptors and trigger some biological process." [GOC:dph, GOC:mah, ISBN:0198506732]
GO:0005180
name: peptide hormone
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005181
name: glycopeptide hormone
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005182
name: lipopeptide hormone
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005183
name: gonadotropin hormone-releasing hormone activity
def: "The action characteristic of gonadotropin hormone-releasing hormone (GnRH), any of a family of decapeptide amide hormones that are released by the hypothalamus in response to neural and/or chemical stimuli. In at least mammals, upon receptor binding, GnRH causes the release of follicle-stimulating hormone (FSH) and luteinizing hormone (LH) by the anterior pituitary." [ISBN:0198506732, PMID:11026571, Wikipedia:Gonadotropin-releasing_hormone]
GO:0005184
name: neuropeptide hormone activity
def: "The action characteristic of a neuropeptide hormone, any peptide hormone that acts in the central nervous system. A neuropeptide is any of several types of molecules found in brain tissue, composed of short chains of amino acids; they include endorphins, enkephalins, vasopressin, and others. They are often localized in axon terminals at synapses and are classified as putative neurotransmitters, although some are also hormones." [GOC:mah]
GO:0005185
name: neurohypophyseal hormone activity
def: "The action characteristic of a neurohypophyseal hormone, any of a family of structurally and functionally related nonapeptides that are synthesized as part of a larger precursor molecule comprising a signal peptide, the nonapeptide hormone, and a neurophysin." [GOC:mah, PMID:19243634]
GO:0005186
name: pheromone activity
def: "The activity of binding to and activating specific cell surface receptors, thereby inducing behavioral, developmental, or physiological response(s) from a responding organism or cell. The substance may be released or retained on the cell surface. Pheromones may serve as a specific attractant, social communicator, or sexual stimulant." [GOC:sgd_curators, ISBN:0198506732]
GO:0005187
name: storage protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005188
name: larval serum protein (sensu Insecta)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005189
name: milk protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005190
name: seminal fluid protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005191
name: acidic epididymal glycoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005192
name: urinary protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005193
name: major urinary protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005194
name: cell adhesion molecule activity
def: "OBSOLETE. Mediates the adhesion of the cell to other cells or to the extracellular matrix." [ISBN:0198506732]
GO:0005198
name: structural molecule activity
def: "The action of a molecule that contributes to the structural integrity of a complex or assembly within or outside a cell." [GOC:mah]
GO:0005199
name: structural constituent of cell wall
def: "The action of a molecule that contributes to the structural integrity of a cell wall." [GOC:mah]
GO:0005200
name: structural constituent of cytoskeleton
def: "The action of a molecule that contributes to the structural integrity of a cytoskeletal structure." [GOC:mah]
GO:0005201
name: extracellular matrix structural constituent
def: "The action of a molecule that contributes to the structural integrity of the extracellular matrix." [GOC:mah]
GO:0005202
name: collagen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005203
name: proteoglycan
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005204
name: chondroitin sulfate proteoglycan
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005205
name: chondroitin sulfate/dermatan sulfate proteoglycan
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005206
name: heparin sulfate proteoglycan
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005207
name: extracellular matrix glycoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005208
name: amyloid protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005209
name: plasma protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005211
name: plasma glycoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005212
name: structural constituent of eye lens
def: "The action of a molecule that contributes to the structural integrity of the lens of an eye." [GOC:mah]
GO:0005213
name: structural constituent of chorion
def: "The action of a molecule that contributes to the structural integrity of a chorion. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:sensu]
GO:0005214
name: structural constituent of chitin-based cuticle
def: "The action of a molecule that contributes to the structural integrity of a chitin-based cuticle. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:mtg_sensu]
GO:0005215
name: transporter activity
def: "Enables the directed movement of substances (such as macromolecules, small molecules, ions) into, out of or within a cell, or between cells." [GOC:ai, GOC:dgf]
GO:0005216
name: ion channel activity
def: "Catalysis of facilitated diffusion of an ion (by an energy-independent process) by passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:cy, GOC:mtg_transport, ISBN:0815340729]
GO:0005217
name: intracellular ligand-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific intracellular ligand has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0005218
name: intracellular ligand-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when a specific intracellular ligand has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0005219
name: ryanodine-sensitive calcium-release channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when a ryanodine class ligand has been bound by the channel complex or one of its constituent parts." [GOC:dph, GOC:tb]
GO:0005220
name: inositol 1,4,5-trisphosphate-sensitive calcium-release channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when inositol 1,4,5-trisphosphate (IP3) has been bound by the channel complex or one of its constituent parts." [GOC:mah, GOC:signaling, PMID:8660280, Wikipedia:Inositol_trisphosphate_receptor]
GO:0005221
name: intracellular cyclic nucleotide activated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when intracellular cyclic nucleotide has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport]
GO:0005222
name: intracellular cAMP activated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when intracellular cAMP has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport]
GO:0005223
name: intracellular cGMP activated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when intracellular cGMP has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport]
GO:0005224
name: ATP-binding and phosphorylation-dependent chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when ATP has been bound by the channel complex or one of its constituent parts." [GOC:mah]
GO:0005225
name: volume-sensitive anion channel activity
def: "Catalysis of the transmembrane transfer of an anion by a volume-sensitive channel. An anion is a negatively charged ion. A volume-sensitive channel is a channel that responds to changes in the volume of a cell." [GOC:dph, GOC:tb]
GO:0005227
name: calcium activated cation channel activity
def: "Catalysis of the calcium concentration-regulatable energy-independent passage of cations across a lipid bilayer down a concentration gradient." [GOC:dph, GOC:mtg_transport]
GO:0005228
name: intracellular sodium activated potassium channel activity
def: "Catalysis of the transmembrane transfer of potassium by a channel that opens in response to stimulus by a sodium ion or ions. Transport by a channel involves catalysis of facilitated diffusion of a solute (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel, without evidence for a carrier-mediated mechanism. Sodium activated potassium channels have distinctive properties, including a large single channel conductance, subconductance states, and a block of single channel currents at positive potentials, similar to inward rectification." [GOC:mtg_transport, PMID:12628167]
GO:0005229
name: intracellular calcium activated chloride channel activity
def: "Catalysis of the transmembrane transfer of chloride by a channel that opens in response to stimulus by a calcium ion or ions. Transport by a channel involves catalysis of facilitated diffusion of a solute (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel, without evidence for a carrier-mediated mechanism." [GOC:mtg_transport]
GO:0005230
name: extracellular ligand-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific extracellular ligand has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0005231
name: excitatory extracellular ligand-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific extracellular excitatory ligand has been bound by the channel complex or one of its constituent parts. Excitatory ligands, such as acetylcholine or glutamate, open cation-selective channels." [GOC:mah, ISBN:0323037070]
GO:0005232
name: serotonin-activated cation-selective channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when a specific extracellular ligand has been bound by the channel complex or one of its constituent parts, and that is activated by serotonin." [GOC:mah]
GO:0005234
name: extracellular-glutamate-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when extracellular glutamate has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0005237
name: inhibitory extracellular ligand-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific extracellular inhibitory ligand has been bound by the channel complex or one of its constituent parts. Inhibitory ligands, such as GABA or glycine, open chloride-selective channels." [GOC:mah, ISBN:0323037070]
GO:0005240
name: glycine receptor-associated protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005241
name: inward rectifier channel
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005242
name: inward rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an inwardly-rectifying voltage-gated channel. An inwardly rectifying current-voltage relation is one where at any given driving force the inward flow of K+ ions exceeds the outward flow for the opposite driving force. The inward-rectification is due to a voltage-dependent block of the channel pore by a specific ligand or ligands, and as a result the macroscopic conductance depends on the difference between membrane voltage and the K+ equilibrium potential rather than on membrane voltage itself." [GOC:cb, GOC:mah, PMID:14977398]
GO:0005243
name: gap junction channel activity
def: "A wide pore channel activity that enables a direct cytoplasmic connection from one cell to an adjacent cell. The gap junction can pass large solutes as well as electrical signals between cells. Gap junctions consist of two gap junction hemi-channels, or connexons, one contributed by each membrane through which the gap junction passes." [GOC:dgh, GOC:mtg_transport, ISBN:0815340729]
GO:0005244
name: voltage-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a voltage-gated channel. An ion is an atom or group of atoms carrying an electric charge by virtue of having gained or lost one or more electrons. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0005245
name: voltage-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, GOC:tb, ISBN:0815340729]
GO:0005246
name: calcium channel regulator activity
def: "Modulates the activity of a calcium channel." [GOC:mah]
GO:0005247
name: voltage-gated chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a voltage-gated channel. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0005248
name: voltage-gated sodium channel activity
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0005249
name: voltage-gated potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0005250
name: A-type (transient outward) potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an outwardly-rectifying voltage-gated channel that produces a transient outward current upon a step change in membrane potential." [GOC:mah, PMID:5575340]
GO:0005251
name: delayed rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by a delayed rectifying voltage-gated channel. A delayed rectifying current-voltage relation is one where channel activation kinetics are time-dependent, and inactivation is slow." [GOC:mah, PMID:11343411, PMID:2462513]
GO:0005252
name: open rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an open rectifier voltage-gated channel. An open rectifier current-voltage relationship is one in which the direction of rectification depends on the external potassium ion concentration." [GOC:mah, PMID:8917578]
GO:0005253
name: anion channel activity
def: "Catalysis of the energy-independent passage of anions across a lipid bilayer down a concentration gradient." [GOC:dph, GOC:mtg_transport, ISBN:0815340729]
GO:0005254
name: chloride channel activity
def: "Catalysis of facilitated diffusion of a chloride (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, GOC:pr, ISBN:0815340729]
GO:0005260
name: channel-conductance-controlling ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate. An animal enzyme that is active in forming a chloride channel, the absence of which brings about cystic fibrosis. It is also involved in the functioning of other transmembrane channels." [EC:3.6.3.49]
GO:0005261
name: cation channel activity
def: "Catalysis of the energy-independent passage of cations across a lipid bilayer down a concentration gradient." [GOC:def, GOC:dph, GOC:mtg_transport, ISBN:0815340729]
GO:0005262
name: calcium channel activity
def: "Catalysis of facilitated diffusion of a calcium ion (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, GOC:pr, ISBN:0815340729]
GO:0005267
name: potassium channel activity
def: "Catalysis of facilitated diffusion of a potassium ion (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:BHF, GOC:mtg_transport, ISBN:0815340729]
GO:0005272
name: sodium channel activity
def: "Catalysis of facilitated diffusion of a sodium ion (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:BHF, GOC:mtg_transport, ISBN:0815340729]
GO:0005274
name: allantoin uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: allantoin(out) + H+(out) = allantoin(in) + H+(in) by secondary active transport." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.39.3.1]
GO:0005275
name: amine transmembrane transporter activity
def: "Catalysis of the transfer of amines, including polyamines, from one side of the membrane to the other. Amines are organic compounds that are weakly basic in character and contain an amino (-NH2) or substituted amino group." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0005276
name: vesicular hydrogen:amino acid antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a vesicle membrane to the other according to the reaction: H+(in) + amino acid(out) = H+(out) + amino acid(in)." [GOC:mah, Reactome:428625]
GO:0005277
name: acetylcholine transmembrane transporter activity
def: "Catalysis of the transfer of acetylcholine from one side of the membrane to the other. Acetylcholine is an acetic acid ester of the organic base choline and functions as a neurotransmitter, released at the synapses of parasympathetic nerves and at neuromuscular junctions." [GOC:ai]
GO:0005278
name: acetylcholine:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + acetylcholine(in) = H+(in) + acetylcholine(out)." [TC:2.A.1.2.13]
GO:0005280
name: hydrogen:amino acid symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: amino acid(out) + H+(out) = amino acid(in) + H+(in)." [GOC:ai]
GO:0005281
name: general amino acid permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005283
name: sodium:amino acid symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: amino acid(out) + Na+(out) = amino acid(in) + Na+(in)." [GOC:ai]
GO:0005287
name: high-affinity basic amino acid transmembrane transporter activity
def: "Catalysis of the transfer of basic amino acids from one side of a membrane to the other. Acidic amino acids have a pH above 7. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0005289
name: high-affinity arginine transmembrane transporter activity
def: "Catalysis of the transfer of arginine from one side of a membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0005290
name: L-histidine transmembrane transporter activity
def: "Catalysis of the transfer of L-histidine from one side of a membrane to the other. L-histidine is 2-amino-3-(1H-imidazol-4-yl)propanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005291
name: high-affinity L-histidine transmembrane transporter activity
def: "Catalysis of the transfer of L-histidine from one side of a membrane to the other. L-histidine is 2-amino-3-(1H-imidazol-4-yl)propanoic acid. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0005292
name: high-affinity lysine transmembrane transporter activity
def: "Catalysis of the transfer of lysine from one side of a membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0005294
name: neutral L-amino acid secondary active transmembrane transporter activity
def: "Catalysis of the transfer of a neutral L-amino acid from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0005295
name: neutral amino acid:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: neutral amino acid(out) + Na+(out) = neutral amino acid(in) + Na+(in)." [TC:2.A.23.3.1]
GO:0005297
name: hydrogen:proline symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: proline(out) + H+(out) = proline(in) + H+(in)." [GOC:ai]
GO:0005298
name: proline:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: proline(out) + Na+(out) = proline(in) + Na+(in)." [TC:2.A.22.2.1]
GO:0005300
name: high-affinity tryptophan transmembrane transporter activity
def: "Catalysis of the high-affinity transfer of L-tryptophan from one side of a membrane to the other. Tryptophan is 2-amino-3-(1H-indol-3-yl)propanoic acid. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport, ISBN:0815340729]
GO:0005301
name: valine/tyrosine/tryptophan permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005302
name: L-tyrosine transmembrane transporter activity
def: "Catalysis of the transfer of L-tyrosine from one side of a membrane to the other. L-tyrosine is 2-amino-3-(4-hydroxyphenyl)propanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005304
name: L-valine transmembrane transporter activity
def: "Catalysis of the transfer of L-valine from one side of a membrane to the other. L-valine is 2-amino-3-methylbutanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005307
name: choline:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: choline(out) + Na+(out) = choline(in) + Na+(in)." [TC:2.A.22.3.5]
GO:0005308
name: creatine transmembrane transporter activity
def: "Enables the directed movement of creatine across a membrane. Creatine is a compound synthesized from the amino acids arginine, glycine, and methionine that occurs in muscle." [GOC:ai]
GO:0005309
name: creatine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: creatine(out) + Na+(out) = creatine(in) + Na+(in)." [TC:2.A.22.3.4]
GO:0005310
name: dicarboxylic acid transmembrane transporter activity
def: "Catalysis of the transfer of dicarboxylic acids from one side of the membrane to the other. A dicarboxylic acid is an organic acid with two COOH groups." [GOC:ai]
GO:0005311
name: sodium:dicarboxylate/tricarboxylate symporter activity
def: "OBSOLETE. Catalysis of the reaction: (dicarboxylate or tricarboxylate)(out) + Na+(out) = (dicarboxylate or tricarboxylate)(in) + Na+(in)." [TC:2.A.47.1.5]
GO:0005313
name: L-glutamate transmembrane transporter activity
def: "Catalysis of the transfer of L-glutamate from one side of a membrane to the other. L-glutamate is the anion of 2-aminopentanedioic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005314
name: high-affinity glutamate transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glutamate(out) + H+(out) = glutamate(in) + H+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [TC:2.A.3.10.5]
GO:0005315
name: inorganic phosphate transmembrane transporter activity
def: "Catalysis of the transfer of a inorganic phosphate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0005316
name: high-affinity inorganic phosphate:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: HPO42-(out) + Na+(out) = HPO42-(in) + Na+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [TC:2.A.20.2.2]
GO:0005318
name: phosphate:hydrogen symporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005319
name: lipid transporter activity
def: "Enables the directed movement of lipids into, out of or within a cell, or between cells." [GOC:ai]
GO:0005320
name: apolipoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005321
name: high-density lipoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005322
name: low-density lipoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005323
name: very-low-density lipoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005324
name: long-chain fatty acid transporter activity
def: "Enables the directed movement of long-chain fatty acids into, out of or within a cell, or between cells. A long-chain fatty acid is a fatty acid with a chain length between C13 and C22." [CHEBI:15904, ISBN:0198506732]
GO:0005325
name: peroxisomal fatty-acyl-CoA transporter activity
def: "Catalysis of the reaction: ATP + H2O + fatty acyl CoA(cis) = ADP + phosphate + fatty acyl CoA(trans). The transport of fatty acyl CoA into and out of peroxisomes." [EC:3.6.3.47]
GO:0005326
name: neurotransmitter transporter activity
def: "Enables the directed movement of a neurotransmitter into, out of or within a cell, or between cells. Neurotransmitters are any chemical substance that is capable of transmitting (or inhibiting the transmission of) a nerve impulse from a neuron to another cell." [GOC:ai, ISBN:0198506732]
GO:0005328
name: neurotransmitter:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: neurotransmitter(out) + Na+(out) = neurotransmitter(in) + Na+(in)." [TC:2.A.22.-.-]
GO:0005329
name: dopamine transmembrane transporter activity
def: "Catalysis of the transfer of dopamine from one side of the membrane to the other. Dopamine is a catecholamine neurotransmitter and a metabolic precursor of noradrenaline and adrenaline." [GOC:ai]
GO:0005330
name: dopamine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: dopamine(out) + Na+(out) = dopamine(in) + Na+(in)." [TC:2.A.22.1.3]
GO:0005332
name: gamma-aminobutyric acid:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: gamma-aminobutyric acid(out) + Na+(out) = gamma-aminobutyric acid(in) + Na+(in)." [TC:2.A.22.3.2]
GO:0005333
name: norepinephrine transmembrane transporter activity
def: "Catalysis of the transfer of norepinephrine from one side of the membrane to the other. Norepinephrine (3,4-dihydroxyphenyl-2-aminoethanol) is a hormone secreted by the adrenal medulla and a neurotransmitter in the sympathetic peripheral nervous system and in some tracts of the CNS. It is also the biosynthetic precursor of epinephrine." [ISBN:0198506732]
GO:0005334
name: norepinephrine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: norepinephrine(out) + Na+(out) = norepinephrine(in) + Na+(in)." [TC:2.A.22.1.2]
GO:0005335
name: serotonin:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: serotonin(out) + Na+(out) = serotonin(in) + Na+(in)." [TC:2.A.22.1.1]
GO:0005337
name: nucleoside transmembrane transporter activity
def: "Catalysis of the transfer of a nucleoside, a nucleobase linked to either beta-D-ribofuranose (ribonucleoside) or 2-deoxy-beta-D-ribofuranose, (a deoxyribonucleotide) from one side of a membrane to the other." [GOC:ai]
GO:0005338
name: nucleotide-sugar transmembrane transporter activity
def: "Catalysis of the transfer of a nucleotide-sugar from one side of the membrane to the other. A nucleotide-sugar is any nucleotide in which the distal phosphoric residue of a nucleoside 5'-diphosphate is in glycosidic linkage with a monosaccharide or monosaccharide derivative." [GOC:ai, GOC:mtg_transport, ISBN:0815340729, PMID:15034926]
GO:0005340
name: nucleotide-sulfate transmembrane transporter activity
def: "Catalysis of the transfer of nucleotide-sulfate from one side of a membrane to the other." [GOC:mtg_transport]
GO:0005342
name: organic acid transmembrane transporter activity
def: "Catalysis of the transfer of organic acids, any acidic compound containing carbon in covalent linkage, from one side of the membrane to the other." [ISBN:0198506732]
GO:0005343
name: organic acid:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: organic acid(out) + Na+(out) = organic acid(in) + Na+(in)." [TC:2.A.28.1.1]
GO:0005344
name: oxygen transporter activity
def: "Enables the directed movement of oxygen into, out of or within a cell, or between cells." [GOC:ai]
GO:0005345
name: purine nucleobase transmembrane transporter activity
def: "Catalysis of the transfer of purine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, from one side of a membrane to the other." [CHEBI:26386, ISBN:0198506732]
GO:0005346
name: purine ribonucleotide transmembrane transporter activity
def: "Catalysis of the transfer of a purine ribonucleotide, any compound consisting of a purine ribonucleoside (a purine organic base attached to a ribose sugar) esterified with (ortho)phosphate, from one side of a membrane to the other." [GOC:ai]
GO:0005347
name: ATP transmembrane transporter activity
def: "Catalysis of the transfer of ATP, adenosine triphosphate, from one side of a membrane to the other." [GOC:ai]
GO:0005350
name: pyrimidine nucleobase transmembrane transporter activity
def: "Catalysis of the transfer of pyrimidine nucleobases, one of the two classes of nitrogen-containing ring compounds found in DNA and RNA, from one side of a membrane to the other." [CHEBI:26432, GOC:ai]
GO:0005351
name: sugar:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sugar(out) + H+(out) = sugar(in) + H+(in)." [TC:2.A.1.1.-]
GO:0005352
name: alpha-glucoside:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: alpha-glucoside(out) + H+(out) = alpha-glucoside(in) + H+(in). Alpha-glucosides include trehalose, maltose, turanose, isomaltose, alpha-methylglucoside, maltotriose, palatinose, trehalose and melezitose." [TC:2.A.1.1.11]
GO:0005353
name: fructose transmembrane transporter activity
def: "Catalysis of the transfer of fructose from one side of the membrane to the other. Fructose exists in a open chain form or as a ring compound. D-fructose is the sweetest of the sugars and is found free in a large number of fruits and honey." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005354
name: galactose transmembrane transporter activity
def: "Catalysis of the transfer of galactose from one side of the membrane to the other. D-galactose is widely distributed in combined form in plants, animals and microorganisms as a constituent of oligo- and polysaccharides; it also occurs in galactolipids and as its glucoside in lactose and melibiose." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005355
name: glucose transmembrane transporter activity
def: "Catalysis of the transfer of the hexose monosaccharide glucose from one side of the membrane to the other." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005356
name: hydrogen:glucose symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose + H+ = glucose + H+. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport]
GO:0005357
name: constitutive hydrogen:glucose symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose + H+ = glucose + H+. This activity is constitutive and therefore always present, regardless of demand. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport]
GO:0005358
name: high-affinity hydrogen:glucose symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose + H+ = glucose + H+. This activity is constitutive and therefore always present, regardless of demand. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0005359
name: low-affinity hydrogen:glucose symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose + H+ = glucose + H+. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport]
GO:0005360
name: insulin-responsive hydrogen:glucose symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose + H+ = glucose + H+, in response to a stimulus by insulin. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport]
GO:0005362
name: low-affinity glucose:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose(out) + Na+(out) = glucose(in) + Na+(in). In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [TC:2.A.21.3.-]
GO:0005363
name: maltose transmembrane transporter activity
def: "Catalysis of the transfer of maltose from one side of the membrane to the other. Maltose is the disaccharide 4-O-alpha-D-glucopyranosyl-D-glucopyranose, an intermediate in the enzymatic breakdown of glycogen and starch." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0005364
name: maltose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: maltose(out) + H+(out) = maltose(in) + H+(in)." [TC:2.A.1.1.10]
GO:0005365
name: myo-inositol transmembrane transporter activity
def: "Catalysis of the transfer of myo-inositol from one side of the membrane to the other. Myo-inositol is 1,2,3,4,5/4,6-cyclohexanehexol, a growth factor for animals and microorganisms." [GOC:ai]
GO:0005366
name: myo-inositol:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: myo-inositol(out) + H+(out) = myo-inositol(in) + H+(in)." [TC:2.A.1.1.8]
GO:0005367
name: myo-inositol:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: myo-inositol(out) + Na+(out) = myo-inositol(in) + Na+(in)." [TC:2.A.21.4.-]
GO:0005368
name: taurine transmembrane transporter activity
def: "Catalysis of the transfer of taurine from one side of the membrane to the other. Taurine (2-aminoethanesulfonic acid) is a sulphur-containing amino acid derivative which is important in the metabolism of fats." [GOC:ai]
GO:0005369
name: taurine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: taurine(out) + Na+(out) = taurine(in) + Na+(in)." [TC:2.A.22.3.3]
GO:0005371
name: tricarboxylate secondary active transmembrane transporter activity
def: "Catalysis of the transfer of tricarboxylate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport, ISBN:0815340729]
GO:0005372
name: water transmembrane transporter activity
def: "Enables the directed movement of water (H2O) from one side of a membrane to the other." [GOC:ai]
GO:0005373
name: heavy metal ion porter activity
def: "OBSOLETE. A transporter of heavy metal ions that utilizes a carrier-mediated process to catalyze uniport, symport or antiport between aqueous phases on either side of a lipid membrane." [GOC:ai]
GO:0005375
name: copper ion transmembrane transporter activity
def: "Catalysis of the transfer of copper (Cu) ions from one side of a membrane to the other." [GOC:ai]
GO:0005376
name: plasma membrane copper transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005377
name: intracellular copper ion transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005381
name: iron ion transmembrane transporter activity
def: "Catalysis of the transfer of iron (Fe) ions from one side of a membrane to the other." [GOC:ai]
GO:0005384
name: manganese ion transmembrane transporter activity
def: "Catalysis of the transfer of manganese (Mn) ions from one side of a membrane to the other." [GOC:dgf]
GO:0005385
name: zinc ion transmembrane transporter activity
def: "Catalysis of the transfer of zinc (Zn) ions from one side of a membrane to the other." [GOC:dgf]
GO:0005388
name: calcium-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Ca2+(cis) = ADP + phosphate + Ca2+(trans)." [EC:3.6.3.8]
GO:0005391
name: sodium:potassium-exchanging ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Na+(in) + K+(out) = ADP + phosphate + Na+(out) + K+(in)." [EC:3.6.3.9]
GO:0005395
name: eye pigment precursor transporter activity
def: "Catalysis of the reaction: ATP + H2O + eye pigment precursor(in) = ADP + phosphate + eye pigment precursor(out)." [TC:3.A.1.204.1]
GO:0005396
name: transmembrane conductance regulator activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0005400
name: peroxisomal membrane transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005402
name: cation:sugar symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sugar(out) + cation(out) = sugar(in) + cation(in)." [GOC:ai]
GO:0005412
name: glucose:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose(out) + Na+(out) = glucose(in) + Na+(in)." [TC:2.A.21.3.-]
GO:0005415
name: nucleoside:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: nucleoside(out) + Na+(out) = nucleoside(in) + Na+(in)." [GOC:ai]
GO:0005416
name: cation:amino acid symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: amino acid(out) + cation(out) = amino acid(in) + cation(in)." [GOC:ai]
GO:0005427
name: proton-dependent oligopeptide secondary active transmembrane transporter activity
def: "Catalysis of the transfer of a oligopeptide from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by proton movement." [GOC:mtg_transport, OMIM:600544]
GO:0005429
name: chromaffin granule amine transmembrane transporter activity
def: "Catalysis of the transfer of amines across chromaffin granule membranes." [GOC:mah]
GO:0005430
name: synaptic vesicle amine transmembrane transporter activity
def: "Catalysis of the transfer of amines across synaptic vesicle membranes." [GOC:ai]
GO:0005432
name: calcium:sodium antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Ca2+(in) + Na+(out) = Ca2+(out) + Na+(in)." [GOC:curators, PMID:16371597]
GO:0005436
name: sodium:phosphate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + phosphate(out) = Na+(in) + phosphate(in)." [GOC:ai]
GO:0005451
name: monovalent cation:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: monovalent cation(out) + H+(in) = monovalent cation(in) + H+(out)." [GOC:ai]
GO:0005452
name: inorganic anion exchanger activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: inorganic anion A(out) + inorganic anion B(in) = inorganic anion A(in) + inorganic anion B(out)." [GOC:mah]
GO:0005456
name: CMP-N-acetylneuraminate transmembrane transporter activity
def: "Catalysis of the transfer of a CMP-N-acetylneuraminate from one side of the membrane to the other." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005457
name: GDP-fucose transmembrane transporter activity
def: "Catalysis of the transfer of a GDP-fucose from one side of the membrane to the other. GDP-fucose is a substance composed of fucose in glycosidic linkage with guanosine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005458
name: GDP-mannose transmembrane transporter activity
def: "Catalysis of the transfer of a GDP-mannose from one side of the membrane to the other. GDP-mannose is a substance composed of mannose in glycosidic linkage with guanosine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005459
name: UDP-galactose transmembrane transporter activity
def: "Catalysis of the transfer of a UDP-galactose from one side of the membrane to the other. UDP-galactose is a substance composed of galactose in glycosidic linkage with uridine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005460
name: UDP-glucose transmembrane transporter activity
def: "Catalysis of the transfer of a UDP-glucose from one side of the membrane to the other. UDP-glucose is a substance composed of glucose in glycosidic linkage with uridine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005461
name: UDP-glucuronic acid transmembrane transporter activity
def: "Catalysis of the transfer of a UDP-glucuronic acid from one side of the membrane to the other. UDP-glucuronic acid is a substance composed of glucuronic acid in glycosidic linkage with uridine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005462
name: UDP-N-acetylglucosamine transmembrane transporter activity
def: "Catalysis of the transfer of a UDP-N-acetylglucosamine from one side of the membrane to the other. N-acetylglucosamine is a substance composed of N-acetylglucosamine, a common structural unit of oligosaccharides, in glycosidic linkage with uridine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005463
name: UDP-N-acetylgalactosamine transmembrane transporter activity
def: "Catalysis of the transfer of a N-acetylgalactosamine from one side of the membrane to the other. N-acetylgalactosamine is a substance composed of N-acetylgalactosamine, a common structural unit of oligosaccharides, in glycosidic linkage with uridine diphosphate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0005464
name: UDP-xylose transmembrane transporter activity
def: "Catalysis of the transfer of UDP-xylose from one side of the membrane to the other. UDP-xylose is a substance composed of xylose in glycosidic linkage with uridine diphosphate." [GOC:ai]
GO:0005468
name: small-molecule carrier or transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005469
name: succinate:fumarate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: succinate(out) + fumarate(in) = succinate(in) + fumarate(out)." [TC:2.A.29.13.1]
GO:0005471
name: ATP:ADP antiporter activity
def: "Catalysis of the reaction: ATP(out) + ADP(in) = ATP(in) + ADP(out)." [TC:2.A.29.1.1]
GO:0005472
name: FAD carrier activity
def: "Catalysis of the transfer of flavin adenine dinucleotide from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mah]
GO:0005476
name: carnitine:acyl carnitine antiporter activity
def: "Catalysis of the reaction: carnitine (mitochondrial) + acyl carnitine (cytoplasm) = carnitine (cytoplasm) + acyl carnitine (mitochondrial)." [PMID:9032458]
GO:0005477
name: pyruvate secondary active transmembrane transporter activity
def: "Catalysis of the transfer of pyruvate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0005479
name: vacuolar assembly
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005480
name: vesicle transport
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005481
name: vesicle fusion
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005482
name: vesicle targeting
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005483
name: soluble NSF attachment protein activity
def: "Interacting selectively and non-covalently with both N-ethylmaleimide-sensitive fusion protein (NSF) and a cis-SNARE complex (i.e. a SNARE complex in which all proteins are associated with the same membrane) and increasing the ATPase activity of NSF, thereby allowing ATP hydrolysis by NSF to disassemble the cis-SNARE complex." [GOC:mah, PMID:14570579, PMID:15556857]
GO:0005484
name: SNAP receptor activity
def: "Acting as a marker to identify a membrane and interacting selectively with one or more SNAREs on another membrane to mediate membrane fusion." [GOC:mah, PMID:14570579]
GO:0005487
name: nucleocytoplasmic transporter activity
def: "Enables the directed movement of substances between the nucleus and the cytoplasm of a cell." [GOC:ai]
GO:0005488
name: binding
def: "The selective, non-covalent, often stoichiometric, interaction of a molecule with one or more specific sites on another molecule." [GOC:ceb, GOC:mah, ISBN:0198506732]
GO:0005489
name: electron transporter activity
def: "OBSOLETE. Enables the directed movement of electrons into, out of, within or between cells." [GOC:ai]
GO:0005490
name: cytochrome P450
def: "OBSOLETE. A cytochrome b that has a sulfur atom ligated to the iron of the prosthetic group (heme-thiolate); enzymes:typically monooxygenases acting on lipophilic substrates." [ISBN:0198547684]
GO:0005496
name: steroid binding
def: "Interacting selectively and non-covalently with a steroid, any of a large group of substances that have in common a ring system based on 1,2-cyclopentanoperhydrophenanthrene." [GOC:jl, ISBN:0198506732]
GO:0005497
name: androgen binding
def: "Interacting selectively and non-covalently with any androgen, male sex hormones." [CHEBI:50113, GOC:jl]
GO:0005499
name: vitamin D binding
def: "Interacting selectively and non-covalently with vitamin D, any of a group of related, fat-soluble compounds that are derived from delta-5,7 steroids and play a central role in calcium metabolism. Specific forms of vitamin D include calciferol (ergocalciferol; vitamin D2) and cholecalciferol (calciol; vitamin D3)." [CHEBI:27300, GOC:mah, ISBN:0471331309]
GO:0005500
name: juvenile hormone binding
def: "Interacting selectively and non-covalently with juvenile hormone, the three sesquiterpenoid derivatives that function to maintain the larval state of insects at molting and that may be required for other processes, e.g. oogenesis." [GOC:jl, ISBN:0198506732]
GO:0005501
name: retinoid binding
def: "Interacting selectively and non-covalently with retinoids, any member of a class of isoprenoids that contain or are derived from four prenyl groups linked head-to-tail. Retinoids include retinol and retinal and structurally similar natural derivatives or synthetic compounds, but need not have vitamin A activity." [GOC:jl, ISBN:0198506732]
GO:0005502
name: 11-cis retinal binding
def: "Interacting selectively and non-covalently with 11-cis retinal, an isomer of retinal that plays an important role in the visual process in most vertebrates. 11-cis retinal combines with opsin in the rods (scotopsin) to form rhodopsin or visual purple. Retinal is one of the three compounds that makes up vitamin A." [CHEBI:16066]
GO:0005503
name: all-trans retinal binding
def: "Interacting selectively and non-covalently with all-trans retinal, a compound that plays an important role in the visual process in most vertebrates. All-trans retinal (trans r., visual yellow) results from the bleaching of rhodopsin by light, in which the 11-cis form is converted to the all-trans form. Retinal is one of the forms of vitamin A." [CHEBI:17898, GOC:curators]
GO:0005504
name: fatty acid binding
def: "Interacting selectively and non-covalently with fatty acids, aliphatic monocarboxylic acids liberated from naturally occurring fats and oils by hydrolysis." [ISBN:0198506732]
GO:0005505
name: heavy metal binding
def: "OBSOLETE. Interacting selectively with a heavy metal, a metal that can form a coordination bond with a protein, as opposed to an alkali or alkaline-earth metal that can only form an ionic bond; this definition includes the following biologically relevant heavy metals: Cd, Co, Cu, Fe, Hg, Mn, Mo, Ni, V, W, Zn." [GOC:kd, GOC:mah]
GO:0005506
name: iron ion binding
def: "Interacting selectively and non-covalently with iron (Fe) ions." [GOC:ai]
GO:0005507
name: copper ion binding
def: "Interacting selectively and non-covalently with copper (Cu) ions." [GOC:ai]
GO:0005508
name: copper/cadmium binding
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005509
name: calcium ion binding
def: "Interacting selectively and non-covalently with calcium ions (Ca2+)." [GOC:ai]
GO:0005514
name: calcium ion storage activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005515
name: protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules)." [GOC:go_curators]
GO:0005516
name: calmodulin binding
def: "Interacting selectively and non-covalently with calmodulin, a calcium-binding protein with many roles, both in the calcium-bound and calcium-free states." [GOC:krc]
GO:0005517
name: calmodulin inhibitor activity
def: "OBSOLETE. This term was not defined before being made obsolete." [GOC:tb]
GO:0005518
name: collagen binding
def: "Interacting selectively and non-covalently with collagen, a group of fibrous proteins of very high tensile strength that form the main component of connective tissue in animals. Collagen is highly enriched in glycine (some regions are 33% glycine) and proline, occurring predominantly as 3-hydroxyproline (about 20%)." [GOC:ai, ISBN:0198506732]
GO:0005519
name: cytoskeletal regulatory protein binding
def: "Interacting selectively and non-covalently with any protein involved in modulating the reorganization of the cytoskeleton." [GOC:go_curators, PMID:15163540]
GO:0005520
name: insulin-like growth factor binding
def: "Interacting selectively and non-covalently with an insulin-like growth factor, any member of a group of polypeptides that are structurally homologous to insulin and share many of its biological activities, but are immunologically distinct from it." [ISBN:0198506732]
GO:0005521
name: lamin binding
def: "Interacting selectively and non-covalently with lamin; any of a group of intermediate-filament proteins that form the fibrous matrix on the inner surface of the nuclear envelope." [GOC:jl, ISBN:0198506732]
GO:0005522
name: profilin binding
def: "Interacting selectively and non-covalently with profilin, an actin-binding protein that forms a complex with G-actin and prevents it from polymerizing to form F-actin." [ISBN:0721662544]
GO:0005523
name: tropomyosin binding
def: "Interacting selectively and non-covalently with tropomyosin, a protein associated with actin filaments both in cytoplasm and, in association with troponin, in the thin filament of striated muscle." [GOC:curators, ISBN:0815316194]
GO:0005524
name: ATP binding
def: "Interacting selectively and non-covalently with ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator." [ISBN:0198506732]
GO:0005525
name: GTP binding
def: "Interacting selectively and non-covalently with GTP, guanosine triphosphate." [GOC:ai]
GO:0005527
name: macrolide binding
def: "Interacting selectively and non-covalently with a macrolide, any of a large group of structurally related antibiotics produced by Streptomyces species." [GOC:jl, ISBN:0198506732]
GO:0005528
name: FK506 binding
def: "Interacting selectively and non-covalently with the 23-membered macrolide lactone FK506." [GOC:jl]
GO:0005530
name: lectin
def: "OBSOLETE. Lectins are proteins obtained particularly from the seeds of leguminous plants, but also from many other plant and animal sources, that have binding sites for specific mono or oligosaccharides in cell walls or membranes. They thereby change the physiology of the membrane to cause agglutination, mitosis, or other biochemical changes in the cell." [GOC:curators]
GO:0005531
name: galactose binding lectin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005532
name: mannose binding lectin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005533
name: N-acetylgalactosamine lectin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0005534
name: galactose binding
def: "Interacting selectively and non-covalently with the aldohexose galactose (galacto-hexose), a common constituent of many oligo- and polysaccharides." [CHEBI:28260, GOC:jl, ISBN:0198506732]
GO:0005536
name: glucose binding
def: "Interacting selectively and non-covalently with the D- or L-enantiomer of glucose." [CHEBI:17234, GOC:jl]
GO:0005537
name: mannose binding
def: "Interacting selectively and non-covalently with mannose, a monosaccharide hexose, stereoisomeric with glucose, that occurs naturally only in polymerized forms called mannans." [CHEBI:37684, GOC:jl, ISBN:0192800981]
GO:0005539
name: glycosaminoglycan binding
def: "Interacting selectively and non-covalently with any glycan (polysaccharide) containing a substantial proportion of aminomonosaccharide residues." [GOC:jl, ISBN:0198506732]
GO:0005540
name: hyaluronic acid binding
def: "Interacting selectively and non-covalently with hyaluronic acid, a polymer composed of repeating dimeric units of glucuronic acid and N-acetyl glucosamine." [CHEBI:16336, GOC:jl]
GO:0005541
name: acyl-CoA or acyl binding
def: "OBSOLETE. Interacting selectively with acyl-CoA or acyl, any derivative of coenzyme A in which the sulfhydryl group is in thiolester linkage with a fatty acyl group, or any group formally derived by removal of a hydroxyl group from the acid function of an organic acid." [GOC:jl, ISBN:0198506732]
GO:0005542
name: folic acid binding
def: "Interacting selectively and non-covalently with folic acid, pteroylglutamic acid. Folic acid is widely distributed as a member of the vitamin B complex and is essential for the synthesis of purine and pyrimidines." [GOC:jl, ISBN:0198506732]
GO:0005543
name: phospholipid binding
def: "Interacting selectively and non-covalently with phospholipids, a class of lipids containing phosphoric acid as a mono- or diester." [ISBN:0198506732]
GO:0005544
name: calcium-dependent phospholipid binding
def: "Interacting selectively and non-covalently with phospholipids, a class of lipids containing phosphoric acid as a mono- or diester, in the presence of calcium." [GOC:jl]
GO:0005545
name: 1-phosphatidylinositol binding
def: "Interacting selectively and non-covalently with phosphatidylinositol, any glycophospholipid with its sn-glycerol 3-phosphate residue is esterified to the 1-hydroxyl group of 1D-myo-inositol." [ISBN:0198506732]
GO:0005546
name: phosphatidylinositol-4,5-bisphosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-4,5-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 4' and 5' positions." [GOC:bf, GOC:jl]
GO:0005547
name: phosphatidylinositol-3,4,5-trisphosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-3,4,5-trisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3', 4' and 5' positions." [GOC:bf, GOC:jl]
GO:0005548
name: phospholipid transporter activity
def: "Enables the directed movement of phospholipids into, out of or within a cell, or between cells. Phospholipids are a class of lipids containing phosphoric acid as a mono- or diester." [GOC:ai, ISBN:0198506732]
GO:0005549
name: odorant binding
def: "Interacting selectively and non-covalently with an odorant, any substance capable of stimulating the sense of smell." [GOC:jl, ISBN:0721662544]
GO:0005550
name: pheromone binding
def: "Interacting selectively and non-covalently with a pheromone, a substance, or characteristic mixture of substances, that is secreted and released by an organism and detected by a second organism of the same or a closely related species, in which it causes a specific reaction, such as a definite behavioral reaction or a developmental process." [GOC:ai]
GO:0005551
name: obsolete ubiquitin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005552
name: polyubiquitin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0005553
name: ubiquitin-ribosomal protein fusion protein
def: "OBSOLETE. A protein encoded by some ubiquitin genes which consists of a single copy of ubiquitin fused to a ribosomal protein." [ISBN:0198506732]
GO:0005555
name: blood group antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005557
name: lymphocyte antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005558
name: minor histocompatibility antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005563
name: transfer RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005566
name: ribosomal RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005569
name: small nucleolar RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0005570
name: small nuclear RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008001
name: fibrinogen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008009
name: chemokine activity
def: "The function of a family of small chemotactic cytokines; their name is derived from their ability to induce directed chemotaxis in nearby responsive cells. All chemokines possess a number of conserved cysteine residues involved in intramolecular disulfide bond formation. Some chemokines are considered pro-inflammatory and can be induced during an immune response to recruit cells of the immune system to a site of infection, while others are considered homeostatic and are involved in controlling the migration of cells during normal processes of tissue maintenance or development. Chemokines are found in all vertebrates, some viruses and some bacteria." [GOC:BHF, GOC:rl, http://en.wikipedia.org/wiki/Chemokine, http://www.copewithcytokines.de/cope.cgi?key=Cytokines, PMID:12183377]
GO:0008010
name: structural constituent of chitin-based larval cuticle
def: "The action of a molecule that contributes to the structural integrity of the chitin-based cuticle of a larva. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:mtg_sensu]
GO:0008011
name: structural constituent of pupal chitin-based cuticle
def: "The action of a molecule that contributes to the structural integrity of the chitin-based cuticle of a pupa. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:mtg_sensu]
GO:0008012
name: structural constituent of adult chitin-based cuticle
def: "The action of a molecule that contributes to the structural integrity of the chitin-based cuticle of an adult organism. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:mtg_sensu]
GO:0008013
name: beta-catenin binding
def: "Interacting selectively and non-covalently with the beta subunit of the catenin complex." [GOC:bf]
GO:0008014
name: calcium-dependent cell adhesion molecule activity
def: "OBSOLETE. A calcium-dependent cell adhesion protein (type I membrane protein) that interacts in a homophilic manner in cell-cell interactions." [ISBN:0198506732]
GO:0008017
name: microtubule binding
def: "Interacting selectively and non-covalently with microtubules, filaments composed of tubulin monomers." [GOC:krc]
GO:0008018
name: structural protein of chorion (sensu Drosophila)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008019
name: macrophage receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008020
name: G-protein coupled photoreceptor activity
def: "Combining with incidental electromagnetic radiation, particularly visible light, and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:dph, ISBN:0198506732]
GO:0008022
name: protein C-terminus binding
def: "Interacting selectively and non-covalently with a protein C-terminus, the end of any peptide chain at which the 1-carboxy function of a constituent amino acid is not attached in peptide linkage to another amino-acid residue." [ISBN:0198506732]
GO:0008025
name: diazepam binding inhibitor activity
def: "OBSOLETE. The diazepam binding inhibitor is a 10kDa 86-residue polypeptide that acts as an endogenous ligand for a mitochondrial receptor (formerly regarded as a peripheral benzodiazepine binding site) in steroidogenic cells and regulates stimulation of steroidogenesis by tropic hormones. It also binds to the GABA-A receptor and modulates glucose-dependent insulin secretion and synthesis of acyl-CoA esters." [ISBN:0198506732, PMID:11883709]
GO:0008026
name: ATP-dependent helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the unwinding of a DNA or RNA helix." [EC:3.6.1.3, GOC:jl]
GO:0008028
name: monocarboxylic acid transmembrane transporter activity
def: "Catalysis of the transfer of monocarboxylic acids from one side of the membrane to the other. A monocarboxylic acid is an organic acid with one COOH group." [GOC:ai]
GO:0008029
name: pentraxin receptor activity
def: "Combining with a pentraxin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0008030
name: neuronal pentraxin receptor activity
def: "Combining with a neuronal pentraxin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling, PMID:18840757]
GO:0008031
name: eclosion hormone activity
def: "The action characteristic of eclosion hormone, a peptide hormone that, upon receptor binding, triggers the death of certain muscles and neurons during insect metamorphosis." [GOC:mah, ISBN:0198506732]
GO:0008034
name: lipoprotein binding
def: "OBSOLETE. Interacting selectively and non-covalently with any conjugated, water-soluble protein in which the nonprotein group consists of a lipid or lipids." [GOC:jl, ISBN:0198506732]
GO:0008035
name: high-density lipoprotein particle binding
def: "Interacting selectively and non-covalently with high-density lipoprotein particle, a lipoprotein particle with a high density (typically 1.063-1.21 g/ml) and a diameter of 5-10 nm that contains APOAs and may contain APOCs and APOE." [GOC:mah]
GO:0008036
name: diuretic hormone receptor activity
def: "Combining with a diuretic hormone and transmitting the signal to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0008041
name: storage protein of fat body (sensu Insecta)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008042
name: iron-sulfur electron transfer carrier
def: "OBSOLETE. An iron-sulfur protein that serves as an electron acceptor and electron donor in an electron transport system." [GOC:kd]
GO:0008046
name: axon guidance receptor activity
def: "Combining with an extracellular messenger and transmitting the signal from one side of the membrane to the other to results in a change in cellular activity involved in axon guidance." [GOC:dph, GOC:signaling, GOC:tb, PMID:15107857, PMID:15339666]
GO:0008047
name: enzyme activator activity
def: "Increases the activity of an enzyme." [GOC:mah]
GO:0008048
name: calcium sensitive guanylate cyclase activator activity
def: "Increases the activity of guanylate cyclase in response to a change in calcium ion concentration." [GOC:mah]
GO:0008060
name: ARF GTPase activator activity
def: "Increases the rate of GTP hydrolysis by the GTPase ARF." [GOC:mah]
GO:0008061
name: chitin binding
def: "Interacting selectively and non-covalently with chitin, a linear polysaccharide consisting of beta-(1->4)-linked N-acetyl-D-glucosamine residues." [GOC:jl, ISBN:0198506732]
GO:0008066
name: glutamate receptor activity
def: "Combining with glutamate and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0008067
name: metabotropic glutamate, GABA-B-like receptor activity
def: "OBSOLETE. A G-protein coupled receptor that is structurally/functionally related to the metabotropic glutamate receptor." [GOC:dph, GOC:mah, GOC:tb, IUPHAR_GPCR:1285]
GO:0008068
name: extracellular-glutamate-gated chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when extracellular glutamate has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0008073
name: ornithine decarboxylase inhibitor activity
def: "The stopping, prevention or reduction of the activity of the enzyme ornithine decarboxylase." [GOC:jl]
GO:0008075
name: receptor guanylate cyclase activity
def: "OBSOLETE. Catalysis of the reaction: GTP = 3',5'-cyclic GMP + diphosphate." [EC:4.6.1.2]
GO:0008077
name: Hsp70/Hsp90 organizing protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:rb]
GO:0008079
name: translation termination factor activity
def: "Functions in the termination of translation." [GOC:ma]
GO:0008080
name: N-acetyltransferase activity
def: "Catalysis of the transfer of an acetyl group to a nitrogen atom on the acceptor molecule." [GOC:ai]
GO:0008081
name: phosphoric diester hydrolase activity
def: "Catalysis of the hydrolysis of a phosphodiester to give a phosphomonoester and a free hydroxyl group." [EC:3.1.4, GOC:curators]
GO:0008083
name: growth factor activity
def: "The function that stimulates a cell to grow or proliferate. Most growth factors have other actions besides the induction of cell growth or proliferation." [ISBN:0815316194]
GO:0008084
name: imaginal disc growth factor receptor binding
def: "Interacting selectively and non-covalently with an imaginal disc growth factor receptor." [GOC:mah]
GO:0008086
name: light-activated voltage-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel that is activated in response to light. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport]
GO:0008092
name: cytoskeletal protein binding
def: "Interacting selectively and non-covalently with any protein component of any cytoskeleton (actin, microtubule, or intermediate filament cytoskeleton)." [GOC:mah]
GO:0008093
name: cytoskeletal adaptor activity
def: "The binding activity of a molecule that brings together a cytoskeletal protein and one or more other molecules, permitting them to function in a coordinated way." [GOC:mtg_MIT_16mar07]
GO:0008094
name: DNA-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction requires the presence of single- or double-stranded DNA, and it drives another reaction." [EC:3.6.1.3, GOC:jl]
GO:0008096
name: juvenile hormone epoxide hydrolase activity
def: "Catalysis of the hydrolysis of the epoxide in a juvenile hormone to the corresponding diol." [GOC:mah, PMID:8396141]
GO:0008097
name: 5S rRNA binding
def: "Interacting selectively and non-covalently with 5S ribosomal RNA, the smallest RNA constituent of a ribosome." [GOC:jl, ISBN:0321000382]
GO:0008098
name: 5S rRNA primary transcript binding
def: "Interacting selectively and non-covalently with the unprocessed 5S ribosomal RNA transcript." [GOC:jl]
GO:0008100
name: lipophorin
def: "OBSOLETE. Any member of the major class of lipid-transporting proteins found in the hemolymph of insects." [ISBN:0198506732]
GO:0008106
name: alcohol dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: an alcohol + NADP+ = an aldehyde + NADPH + H+." [EC:1.1.1.2]
GO:0008107
name: galactoside 2-alpha-L-fucosyltransferase activity
def: "Catalysis of the reaction: GDP-L-fucose + beta-D-galactosyl-R = GDP + alpha-L-fucosyl-(1,2)-beta-D-galactosyl-R." [EC:2.4.1.69]
GO:0008108
name: UDP-glucose:hexose-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: alpha-D-galactose 1-phosphate + UDP-D-glucose = alpha-D-glucose 1-phosphate + UDP-D-galactose." [EC:2.7.7.12, RHEA:13992]
GO:0008109
name: N-acetyllactosaminide beta-1,6-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,6-beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R." [EC:2.4.1.150]
GO:0008110
name: L-histidine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-histidine = 3-(imidazol-5-yl)pyruvate + L-glutamate." [EC:2.6.1.38, RHEA:16568]
GO:0008111
name: alpha-methylacyl-CoA racemase activity
def: "Catalysis of the reaction: (2S)-2-methylacyl-CoA = (2R)-2-methylacyl-CoA." [EC:5.1.99.4]
GO:0008112
name: nicotinamide N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + nicotinamide = 1-methylnicotinamide + S-adenosyl-L-homocysteine." [EC:2.1.1.1, RHEA:23887]
GO:0008113
name: peptide-methionine (S)-S-oxide reductase activity
def: "Catalysis of the reactions: peptide-L-methionine + thioredoxin disulfide + H2O = peptide-L-methionine (S)-S-oxide + thioredoxin, and L-methionine + thioredoxin disulfide + H2O = L-methionine (S)-S-oxide + thioredoxin. Can act on oxidized methionine in peptide linkage with specificity for the S enantiomer. Thioredoxin disulfide is the oxidized form of thioredoxin." [EC:1.8.4.11, GOC:mah, GOC:vw, PMID:11169920]
GO:0008114
name: phosphogluconate 2-dehydrogenase activity
def: "Catalysis of the reaction: 6-phospho-D-gluconate + NADP+ = 6-phospho-2-dehydro-D-gluconate + NADPH." [EC:1.1.1.43]
GO:0008115
name: sarcosine oxidase activity
def: "Catalysis of the reaction: H(2)O + O(2) + sarcosine = formaldehyde + glycine + H(2)O(2)." [EC:1.5.3.1, RHEA:13316]
GO:0008116
name: prostaglandin-I synthase activity
def: "Catalysis of the reaction: prostaglandin H(2) = prostaglandin I(2)." [EC:5.3.99.4, RHEA:23583]
GO:0008117
name: sphinganine-1-phosphate aldolase activity
def: "Catalysis of the reaction: sphinganine 1-phosphate = phosphoethanolamine + palmitaldehyde." [EC:4.1.2.27]
GO:0008118
name: N-acetyllactosaminide alpha-2,3-sialyltransferase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-glycoprotein = CMP + alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-glycoprotein." [EC:2.4.99.6]
GO:0008119
name: thiopurine S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a thiopurine = S-adenosyl-L-homocysteine + a thiopurine S-methylether." [EC:2.1.1.67]
GO:0008120
name: ceramide glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + N-acylsphingosine = UDP + D-glucosyl-N-acylsphingosine." [EC:2.4.1.80]
GO:0008121
name: ubiquinol-cytochrome-c reductase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: CoQH2 + 2 ferricytochrome c = CoQ + 2 ferrocytochrome c + 2 H+." [EC:1.10.2.2, ISBN:0198547684]
GO:0008123
name: cholesterol 7-alpha-monooxygenase activity
def: "Catalysis of the reaction: cholesterol + NADPH + H+ + O2 = 7-alpha-hydroxycholesterol + NADP+ + H2O." [EC:1.14.13.17]
GO:0008124
name: 4-alpha-hydroxytetrahydrobiopterin dehydratase activity
def: "Catalysis of the reaction: (6R)-6-(L-erythro-1,2-dihydroxypropyl)-5,6,7,8-tetrahydro-4a-hydroxypterin = (6R)-6-(L-erythro-1,2-dihydroxypropyl)-7,8-dihydro-6H-pterin + H(2)O." [EC:4.2.1.96, RHEA:11923]
GO:0008125
name: pancreatic elastase I activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins, including elastin. Preferential cleavage: Ala-Xaa." [EC:3.4.21.36]
GO:0008126
name: acetylesterase activity
def: "Catalysis of the reaction: an acetic ester + H2O = an alcohol + acetate." [EC:3.1.1.6]
GO:0008127
name: quercetin 2,3-dioxygenase activity
def: "Catalysis of the reaction: H(+) + O(2) + quercetin = 2-(3,4-dihydroxybenzoyloxy)-4,6-dihydroxybenzoate + CO." [EC:1.13.11.24, RHEA:15384]
GO:0008129
name: actinidain activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins with broad specificity for peptide bonds, with preference for a residue bearing a large hydrophobic side chain at the P2 position. Does not accept Val at P1'." [EC:3.4.22.14]
GO:0008130
name: neutrophil collagenase activity
def: "OBSOLETE. Catalysis of the cleavage of interstitial collagens in the triple helical domain. Unlike EC:3.4.24.7, this enzyme cleaves type III collagen more slowly than type I." [EC:3.4.24.34]
GO:0008131
name: primary amine oxidase activity
def: "Catalysis of the reaction: a primary amine + H2O + O2 = an aldehyde + NH3 + hydrogen peroxide." [EC:1.4.3.21, EC:1.4.3.22, EC:1.4.3.4, MetaCyc:RXN-9597]
GO:0008132
name: pancreatic elastase activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins, including elastin." [EC:3.4.21.36, EC:3.4.21.71]
GO:0008133
name: collagenase activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0008134
name: transcription factor binding
def: "Interacting selectively and non-covalently with a transcription factor, any protein required to initiate or regulate transcription." [ISBN:0198506732]
GO:0008135
name: translation factor activity, nucleic acid binding
def: "Functions during translation by interacting selectively and non-covalently with nucleic acids during polypeptide synthesis at the ribosome." [GOC:ai, GOC:vw]
GO:0008137
name: NADH dehydrogenase (ubiquinone) activity
def: "Catalysis of the reaction: NADH + H+ + ubiquinone = NAD+ + ubiquinol." [EC:1.6.5.3]
GO:0008138
name: protein tyrosine/serine/threonine phosphatase activity
def: "Catalysis of the reactions: protein serine + H2O = protein serine + phosphate; protein threonine phosphate + H2O = protein threonine + phosphate; and protein tyrosine phosphate + H2O = protein tyrosine + phosphate." [GOC:mah]
GO:0008139
name: nuclear localization sequence binding
def: "Interacting selectively and non-covalently with a nuclear localization sequence, a specific peptide sequence that acts as a signal to localize the protein within the nucleus." [GOC:ai]
GO:0008140
name: cAMP response element binding protein binding
def: "Interacting selectively and non-covalently with the cAMP response element binding protein." [GOC:mah]
GO:0008141
name: puparial glue (sensu Diptera)
def: "OBSOLETE. A glue which attaches the pupae to the substrate during metamorphosis, as in, but not restricted to, the true flies (Diptera, ncbi_taxonomy_id:7147)." [PMID:825230]
GO:0008142
name: oxysterol binding
def: "Interacting selectively and non-covalently with oxysterol, an oxidized form of cholesterol." [http://www.onelook.com/]
GO:0008143
name: poly(A) binding
def: "Interacting selectively and non-covalently with a sequence of adenylyl residues in an RNA molecule, such as the poly(A) tail, a sequence of adenylyl residues at the 3' end of eukaryotic mRNA." [GOC:jl]
GO:0008144
name: drug binding
def: "Interacting selectively and non-covalently with a drug, any naturally occurring or synthetic substance, other than a nutrient, that, when administered or applied to an organism, affects the structure or functioning of the organism; in particular, any such substance used in the diagnosis, prevention, or treatment of disease." [GOC:jl, ISBN:0198506732]
GO:0008145
name: phenylalkylamine binding
def: "Interacting selectively and non-covalently with phenylalkylamine or any of its derivatives." [GOC:jl]
GO:0008146
name: sulfotransferase activity
def: "Catalysis of the transfer of a sulfate group from 3'-phosphoadenosine 5'-phosphosulfate to the hydroxyl group of an acceptor, producing the sulfated derivative and 3'-phosphoadenosine 5'-phosphate." [EC:2.8.2, GOC:curators]
GO:0008147
name: structural constituent of bone
def: "The action of a molecule that contributes to the structural integrity of bone." [GOC:mah]
GO:0008148
name: negative transcription elongation factor activity
def: "OBSOLETE. Any activity that decreases the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule following transcription initiation." [GOC:mah]
GO:0008149
name: para-aminobenzoic acid (PABA) synthase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai, MetaCyc:PABASYN-CPLX]
GO:0008157
name: protein phosphatase 1 binding
def: "Interacting selectively and non-covalently with the enzyme protein phosphatase 1." [GOC:jl]
GO:0008158
name: hedgehog receptor activity
def: "Combining with a member of the hedgehog protein family and transmitting the signal across the membrane to initiate a change in cell activity." [GOC:bf, GOC:go_curators, PMID:9278137]
GO:0008159
name: positive transcription elongation factor activity
def: "OBSOLETE. Any activity that increases the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule following transcription initiation." [GOC:mah]
GO:0008160
name: protein tyrosine phosphatase activator activity
def: "Increases the activity of a phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a tyrosyl phenolic group of a protein." [GOC:ai, ISBN:0198506732]
GO:0008168
name: methyltransferase activity
def: "Catalysis of the transfer of a methyl group to an acceptor molecule." [ISBN:0198506732]
GO:0008169
name: C-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the carbon atom of an acceptor molecule." [GOC:ai]
GO:0008170
name: N-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the nitrogen atom of an acceptor molecule." [GOC:ai]
GO:0008171
name: O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the oxygen atom of an acceptor molecule." [GOC:ai]
GO:0008172
name: S-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the sulfur atom of an acceptor molecule." [GOC:ai]
GO:0008173
name: RNA methyltransferase activity
def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in an RNA molecule." [GOC:mah]
GO:0008174
name: mRNA methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an mRNA molecule." [GOC:mah]
GO:0008175
name: tRNA methyltransferase activity
def: "Catalysis of the transfer of a methyl group from a donor to a nucleoside residue in a tRNA molecule." [GOC:mah]
GO:0008176
name: tRNA (guanine-N7-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N7-methylguanine." [EC:2.1.1.33]
GO:0008177
name: succinate dehydrogenase (ubiquinone) activity
def: "Catalysis of the reaction: succinate + ubiquinone = fumarate + ubiquinol." [EC:1.3.5.1, RHEA:13716]
GO:0008179
name: adenylate cyclase binding
def: "Interacting selectively and non-covalently with the enzyme adenylate cyclase." [GOC:jl]
GO:0008181
name: tumor suppressor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008184
name: glycogen phosphorylase activity
def: "Catalysis of the reaction: glycogen + phosphate = maltodextrin + alpha-D-glucose 1-phosphate." [EC:2.4.1.1, MetaCyc:GLYCOPHOSPHORYL-RXN]
GO:0008186
name: RNA-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction requires the presence of RNA, and it drives another reaction." [EC:3.6.1.3, GOC:jl]
GO:0008187
name: poly-pyrimidine tract binding
def: "Interacting selectively and non-covalently with any stretch of pyrimidines (cytosine or uracil) in an RNA molecule." [GOC:jl]
GO:0008188
name: neuropeptide receptor activity
def: "Combining with a neuropeptide to initiate a change in cell activity." [GOC:ai]
GO:0008189
name: apoptosis inhibitor activity
def: "OBSOLETE. The function held by products which directly block any step in the process of apoptosis." [GOC:hb]
GO:0008190
name: eukaryotic initiation factor 4E binding
def: "Interacting selectively and non-covalently with eukaryotic initiation factor 4E, a polypeptide factor involved in the initiation of ribosome-mediated translation." [ISBN:0198506732]
GO:0008191
name: metalloendopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of metalloendopeptidases, enzymes that catalyze the hydrolysis of nonterminal peptide bonds in a polypeptide chain and contain a chelated metal ion at their active sites which is essential to their catalytic activity." [GOC:ai]
GO:0008192
name: RNA guanylyltransferase activity
def: "Catalysis of the posttranscriptional addition of a guanyl residue to the 5' end of an RNA molecule." [GOC:mah]
GO:0008193
name: tRNA guanylyltransferase activity
def: "Catalysis of the posttranscriptional addition of a guanyl residue to the 5' end of a tRNA molecule; observed for His tRNAs." [PMID:1660461]
GO:0008194
name: UDP-glycosyltransferase activity
def: "Catalysis of the transfer of a glycosyl group from a UDP-sugar to a small hydrophobic molecule." [InterPro:IPR004224, PMID:11846783]
GO:0008195
name: phosphatidate phosphatase activity
def: "Catalysis of the reaction: a 1,2-diacylglycerol 3-phosphate + H2O = a 1,2-diacyl-sn-glycerol + phosphate." [EC:3.1.3.4, GOC:pr]
GO:0008196
name: vitellogenin receptor activity
def: "Receiving vitellogenin, and delivering vitellogenin into the cell via endocytosis." [GOC:bf, PMID:12429745]
GO:0008197
name: yolk protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008198
name: ferrous iron binding
def: "Interacting selectively and non-covalently with ferrous iron, Fe(II)." [GOC:ai]
GO:0008199
name: ferric iron binding
def: "Interacting selectively and non-covalently with ferric iron, Fe(III)." [GOC:ai]
GO:0008200
name: ion channel inhibitor activity
def: "Stops, prevents, or reduces the activity of an ion channel." [GOC:mah]
GO:0008201
name: heparin binding
def: "Interacting selectively and non-covalently with heparin, any member of a group of glycosaminoglycans found mainly as an intracellular component of mast cells and which consist predominantly of alternating alpha-(1->4)-linked D-galactose and N-acetyl-D-glucosamine-6-sulfate residues." [GOC:jl, ISBN:0198506732]
GO:0008222
name: tumor antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008224
name: Gram-positive antibacterial peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-positive bacterial cells." [GOC:go_curators]
GO:0008225
name: Gram-negative antibacterial peptide activity
def: "OBSOLETE. Inhibits the growth of, or directly kills, Gram-negative bacterial cells." [GOC:go_curators]
GO:0008226
name: tyramine receptor activity
def: "Combining with the biogenic amine tyramine to initiate a change in cell activity. Tyramine is a sympathomimetic amine derived from tyrosine with an action resembling that of epinephrine." [http://www.onelook.com/]
GO:0008227
name: G-protein coupled amine receptor activity
def: "Combining with an extracellular amine and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:dph]
GO:0008229
name: opsonin activity
def: "OBSOLETE. Binds to microorganisms or other particulate material (for example, foreign erythrocytes) to increase the susceptibility of the latter to phagocytosis." [ISBN:0198506732]
GO:0008233
name: peptidase activity
def: "Catalysis of the hydrolysis of a peptide bond. A peptide bond is a covalent bond formed when the carbon atom from the carboxyl group of one amino acid shares electrons with the nitrogen atom from the amino group of a second amino acid." [GOC:jl, ISBN:0815332181]
GO:0008234
name: cysteine-type peptidase activity
def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0008235
name: metalloexopeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond not more than three residues from the N- or C-terminus of a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#EXOPEPTIDASE]
GO:0008236
name: serine-type peptidase activity
def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, ISBN:0716720094]
GO:0008237
name: metallopeptidase activity
def: "Catalysis of the hydrolysis of peptide bonds by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0008238
name: exopeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond not more than three residues from the N- or C-terminus of a polypeptide chain, in a reaction that requires a free N-terminal amino group, C-terminal carboxyl group or both." [http://merops.sanger.ac.uk/about/glossary.htm#EXOPEPTIDASE]
GO:0008239
name: dipeptidyl-peptidase activity
def: "Catalysis of the hydrolysis of N-terminal dipeptides from a polypeptide chain." [GOC:mb]
GO:0008240
name: tripeptidyl-peptidase activity
def: "Catalysis of the release of an N-terminal tripeptide from a polypeptide." [GOC:mah]
GO:0008241
name: peptidyl-dipeptidase activity
def: "Catalysis of the release of C-terminal dipeptides from a polypeptide chain." [GOC:mb]
GO:0008242
name: omega peptidase activity
def: "Catalysis of the removal of terminal peptide residues that are substituted, cyclized or linked by isopeptide bonds (peptide linkages other than those of alpha-carboxyl to alpha-amino groups)." [EC:3.4.19.-]
GO:0008243
name: plasminogen activator activity
def: "OBSOLETE. Catalysis of the specific cleavage of an Arg-Val bond in plasminogen to form plasmin." [EC:3.4.21.68, EC:3.4.21.73]
GO:0008246
name: electron transfer flavoprotein
def: "OBSOLETE. Works in conjunction with acyl-CoA dehydrogenase to catalyze the oxidation of CoA and reduce ubiquinone. Part of the mitochondrial electron transport system." [ISBN:0198506732]
GO:0008248
name: pre-mRNA splicing factor activity
def: "OBSOLETE. An activity involved in the removal of an intron from a pre-mRNA." [GOC:jl]
GO:0008251
name: tRNA-specific adenosine deaminase activity
def: "Catalysis of the reaction: adenosine + H2O = inosine + NH3, in a tRNA molecule." [GOC:mah, IMG:00700]
GO:0008252
name: nucleotidase activity
def: "Catalysis of the reaction: a nucleotide + H2O = a nucleoside + phosphate." [EC:3.1.3.31]
GO:0008253
name: 5'-nucleotidase activity
def: "Catalysis of the reaction: a 5'-ribonucleotide + H2O = a ribonucleoside + phosphate." [EC:3.1.3.5]
GO:0008254
name: 3'-nucleotidase activity
def: "Catalysis of the reaction: a 3'-ribonucleotide + H2O = a ribonucleoside + phosphate." [EC:3.1.3.6]
GO:0008255
name: ecdysis-triggering hormone activity
def: "The action characteristic of ecdysis-triggering hormone, a peptide hormone that, upon receptor binding, initiates pre-ecdysis and ecdysis (i.e. cuticle shedding) through direct action on the central nervous system." [GOC:mah, PMID:9020043]
GO:0008256
name: protein histidine pros-kinase activity
def: "Catalysis of the reaction: ATP + protein L-histidine = ADP + protein N(pi)-phospho-L-histidine." [EC:2.7.13.1]
GO:0008257
name: protein histidine tele-kinase activity
def: "Catalysis of the reaction: ATP + protein L-histidine = ADP + protein N(tau)-phospho-L-histidine." [EC:2.7.13.2]
GO:0008260
name: 3-oxoacid CoA-transferase activity
def: "Catalysis of the reaction: succinyl-CoA + a 3-oxo acid = succinate + a 3-oxo-acyl-CoA." [EC:2.8.3.5]
GO:0008261
name: allatostatin receptor activity
def: "Combining with allatostatin to initiate a change in cell activity." [GOC:ai]
GO:0008262
name: importin-alpha export receptor activity
def: "Interacting selectively and non-covalently with importin-alpha to mediate its transfer through the nuclear pore to the cytoplasm." [GOC:mah, GOC:vw, PMID:12486120]
GO:0008263
name: pyrimidine-specific mismatch base pair DNA N-glycosylase activity
def: "Catalysis of the removal of mismatched pyrimidine bases in DNA. Enzymes with this activity recognize and remove pyrimidines present in mismatches by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apyrimidinic (AP) site." [GOC:elh, PMID:9224623]
GO:0008265
name: Mo-molybdopterin cofactor sulfurase activity
def: "Catalysis of the sulfurylation of the desulfo form of molybdenum cofactor (MoCo), a cofactor required for the activity of some enzymes, such as aldehyde oxidase." [GOC:mah, PMID:11549764]
GO:0008266
name: poly(U) RNA binding
def: "Interacting selectively and non-covalently with a sequence of uracil residues in an RNA molecule." [GOC:mah]
GO:0008267
name: poly-glutamine tract binding
def: "Interacting selectively and non-covalently with a polyglutamine tract, i.e. a series of consecutive glutamine residues, in a protein." [GOC:mah]
GO:0008268
name: receptor signaling protein tyrosine kinase signaling protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0008269
name: JAK pathway signal transduction adaptor activity
def: "The binding activity of a molecule that brings together two molecules of the JAK signal transduction pathway, permitting them to function in a coordinated way." [GOC:mtg_MIT_16mar07]
GO:0008270
name: zinc ion binding
def: "Interacting selectively and non-covalently with zinc (Zn) ions." [GOC:ai]
GO:0008271
name: secondary active sulfate transmembrane transporter activity
def: "Catalysis of the secondary active transfer of sulfate from one side of the membrane to the other. Secondary active transport is catalysis of the transfer of a solute from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0008273
name: calcium, potassium:sodium antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Ca2+(in) + K+(in) + Na+(out) = Ca2+(out) + K+(out) + Na+(in)." [TC:2.A.19.4.1]
GO:0008276
name: protein methyltransferase activity
def: "Catalysis of the transfer of a methyl group (CH3-) to a protein." [GOC:jl, ISBN:0198506732]
GO:0008281
name: sulfonylurea receptor activity
def: "Combining with sulfonylurea, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0008288
name: boss receptor activity
def: "Combining with a protein bride of sevenless (boss) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate." [GOC:jl, GOC:signaling]
GO:0008289
name: lipid binding
def: "Interacting selectively and non-covalently with a lipid." [GOC:ai]
GO:0008294
name: calcium- and calmodulin-responsive adenylate cyclase activity
def: "Catalysis of the reaction: ATP = 3',5'-cyclic AMP + diphosphate, stimulated by calcium-bound calmodulin." [EC:4.6.1.1, GOC:mah]
GO:0008296
name: 3'-5'-exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of a DNA molecule." [GOC:mah]
GO:0008297
name: single-stranded DNA exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' or 3' terminus of a single-stranded DNA molecule." [GOC:mah]
GO:0008301
name: DNA binding, bending
def: "The activity of binding selectively and non-covalently to and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:krc, GOC:vw, PMID:10710711, PMID:19037758]
GO:0008307
name: structural constituent of muscle
def: "The action of a molecule that contributes to the structural integrity of a muscle fiber." [GOC:mah]
GO:0008308
name: voltage-gated anion channel activity
def: "Catalysis of the transmembrane transfer of an anion by a voltage-gated channel. An anion is a negatively charged ion. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0008309
name: double-stranded DNA exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' or 3' terminus of a double-stranded DNA molecule." [GOC:mah]
GO:0008310
name: single-stranded DNA 3'-5' exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of a single-stranded DNA molecule." [GOC:mah, PMID:22562358]
GO:0008311
name: double-stranded DNA 3'-5' exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of a double-stranded DNA molecule." [GOC:mah, PMID:22562358]
GO:0008312
name: 7S RNA binding
def: "Interacting selectively and non-covalently with 7S RNA, the RNA component of the signal recognition particle (SRP)." [GOC:jl, PMID:6181418]
GO:0008313
name: gurken-activated receptor activity
def: "Combining with the ligand Gurken to initiate a change in cell activity." [GOC:bf]
GO:0008316
name: structural constituent of vitelline membrane
def: "The action of a molecule that contributes to the structural integrity of the vitelline membrane of an egg. An example of this is found in Drosophila melanogaster." [GOC:mah, GOC:sensu]
GO:0008317
name: gurken receptor binding
def: "Interacting selectively and non-covalently with the gurken growth factor receptor." [GOC:ai]
GO:0008318
name: protein prenyltransferase activity
def: "Catalysis of the covalent addition of an isoprenoid group such as a farnesyl or geranylgeranyl group via thioether linkages to a cysteine residue in a protein." [GOC:mah]
GO:0008319
name: prenyl protein specific endopeptidase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008320
name: protein transmembrane transporter activity
def: "Catalysis of the transfer of a protein from one side of a membrane to the other." [GOC:jl]
GO:0008321
name: Ral guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Ral family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0008322
name: Pro-X carboxypeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a Pro-Xaa bond to release a C-terminal amino acid." [GOC:mah]
GO:0008324
name: cation transmembrane transporter activity
def: "Catalysis of the transfer of cation from one side of the membrane to the other." [GOC:dgf, GOC:mtg_transport, ISBN:0815340729]
GO:0008327
name: methyl-CpG binding
def: "Interacting selectively and non-covalently with a methylated cytosine/guanine dinucleotide." [GOC:jl, PMID:11746232]
GO:0008329
name: signaling pattern recognition receptor activity
def: "Combining with a pathogen-associated molecular pattern (PAMP), a structure conserved among microbial species, or damage-associated molecular pattern (DAMP), an endogenous molecule released from damaged cells), and transmitting a signal to initiate a change in cell activity." [GOC:ar, PMID:12507420, PMID:12925128, PMID:14523544, Wikipedia:Pattern_recognition_receptor]
GO:0008330
name: protein tyrosine/threonine phosphatase activity
def: "Catalysis of the reactions: protein threonine phosphate + H2O = protein threonine + phosphate; and protein tyrosine phosphate + H2O = protein tyrosine + phosphate." [GOC:mah]
GO:0008331
name: high voltage-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a high voltage-gated channel. A high voltage-gated channel is a channel whose open state is dependent on high voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729, PMID:16382099]
GO:0008332
name: low voltage-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a low voltage-gated channel. A low voltage-gated channel is a channel whose open state is dependent on low voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729, PMID:16382099]
GO:0008336
name: gamma-butyrobetaine dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate + 4-(trimethylammonio)butanoate + O(2) = carnitine + CO(2) + succinate." [EC:1.14.11.1, RHEA:24031]
GO:0008337
name: selectin
def: "OBSOLETE. A class of cell adhesion molecules that bind to carbohydrate via a lectin-like domain; integral membrane glycoproteins." [ISBN:0124325653]
GO:0008349
name: MAP kinase kinase kinase kinase activity
def: "Catalysis of the phosphorylation of serine and threonine residues in a mitogen-activated protein kinase kinase kinase (MAPKKK), resulting in activation of MAPKKK. MAPKKK signaling pathways relay, amplify and integrate signals from the plasma membrane to the nucleus in response to a diverse range of extracellular stimuli." [GOC:bf, PMID:11790549]
GO:0008350
name: kinetochore motor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008351
name: microtubule severing activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008353
name: RNA polymerase II carboxy-terminal domain kinase activity
def: "Catalysis of the reaction: ATP + (DNA-directed RNA polymerase II) = ADP + phospho-(DNA-directed RNA polymerase II); phosphorylation occurs on residues in the carboxy-terminal domain (CTD) repeats." [EC:2.7.11.23, GOC:mah]
GO:0008369
name: obsolete molecular function
def: "OBSOLETE. These are terms that have been removed from the active function ontology." [GOC:go_curators]
GO:0008373
name: sialyltransferase activity
def: "Catalysis of the transfer of sialic acid to an acceptor molecule, typically the terminal portions of the sialylated glycolipids (gangliosides) or to the N- or O-linked sugar chains of glycoproteins." [EC:2.4.99, GOC:cjm, Wikipedia:Sialyltransferase]
GO:0008374
name: O-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0008375
name: acetylglucosaminyltransferase activity
def: "Catalysis of the transfer of an N-acetylglucosaminyl residue from UDP-N-acetyl-glucosamine to a sugar." [ISBN:0198506732]
GO:0008376
name: acetylgalactosaminyltransferase activity
def: "Catalysis of the transfer of an N-acetylgalactosaminyl residue from UDP-N-acetyl-galactosamine to an oligosaccharide." [ISBN:0198506732]
GO:0008378
name: galactosyltransferase activity
def: "Catalysis of the transfer of a galactosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [ISBN:0198506732]
GO:0008379
name: thioredoxin peroxidase activity
def: "Catalysis of the reaction: thioredoxin + hydrogen peroxide = thioredoxin disulfide + H2O." [MetaCyc:RXN0-267]
GO:0008381
name: mechanically-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens in response to a mechanical stress." [GOC:mtg_transport, ISBN:0815340729]
GO:0008384
name: IkappaB kinase activity
def: "Catalysis of the reaction: ATP + IkappaB protein = ADP + IkappaB phosphoprotein." [EC:2.7.11.10]
GO:0008386
name: cholesterol monooxygenase (side-chain-cleaving) activity
def: "Catalysis of the reaction: cholesterol + reduced adrenal ferredoxin + O2 = pregnenolone + 4-methylpentanal + oxidized adrenal ferredoxin + H2O." [EC:1.14.15.6]
GO:0008387
name: steroid 7-alpha-hydroxylase activity
def: "Catalysis of the reaction: a steroid + donor-H2 + O2 = 7-alpha-hydroxysteroid + H2O." [GOC:mah]
GO:0008388
name: testosterone 15-alpha-hydroxylase activity
def: "Catalysis of the reaction: testosterone + donor-H2 + O2 = 15-alpha-hydroxytestosterone + H2O." [GOC:ai]
GO:0008389
name: coumarin 7-hydroxylase activity
def: "Catalysis of the reaction: coumarin + O2 + NADPH + H+ = hydroxycoumarin + H2O + NADP+." [Reactome:163103]
GO:0008390
name: testosterone 16-alpha-hydroxylase activity
def: "Catalysis of the reaction: testosterone + donor-H2 + O2 = 6-alpha-hydroxytestosterone + H2O." [GOC:ai]
GO:0008391
name: arachidonic acid monooxygenase activity
def: "Catalysis of the incorporation of one atom from molecular oxygen into arachidonic acid and the reduction of the other atom of oxygen to water." [GOC:mah]
GO:0008392
name: arachidonic acid epoxygenase activity
def: "Catalysis of an NADPH- and oxygen-dependent reaction that converts arachidonic acid to a cis-epoxyeicosatrienoic acid." [http://lipidlibrary.aocs.org/Lipids/eic_hete/index.htm, PMID:10681399]
GO:0008395
name: steroid hydroxylase activity
def: "Catalysis of the formation of a hydroxyl group on a steroid by incorporation of oxygen from O2." [ISBN:0721662544]
GO:0008396
name: oxysterol 7-alpha-hydroxylase activity
def: "Catalysis of the reaction: an oxysterol + NADPH + O2 = 7-alpha-hydroxylated oxysterol + NADP+ + H2O." [PMID:10882791]
GO:0008397
name: sterol 12-alpha-hydroxylase activity
def: "Catalysis of the reaction: a steroid + donor-H2 + O2 = 12-alpha-hydroxysteroid + H2O." [GOC:mah]
GO:0008398
name: sterol 14-demethylase activity
def: "Catalysis of the reaction: obtusifoliol + 3 O2 + 3 NADPH + 3 H+ = 4-alpha-methyl-5-alpha-ergosta-8,14,24(28)-trien-3-beta-ol + formate + 3 NADP+ + 3 H2O." [EC:1.14.13.70]
GO:0008399
name: naphthalene hydroxylase activity
def: "Catalysis of the conversion of naphthalene to naphthalene 1,2-oxide." [PMID:1742282, PMID:1981702]
GO:0008401
name: retinoic acid 4-hydroxylase activity
def: "Catalysis of the conversion of retinoic acid to 4-hydroxy-retinoic acid." [PMID:19519282]
GO:0008403
name: 25-hydroxycholecalciferol-24-hydroxylase activity
def: "Catalysis of the hydroxylation of C-24 of 25-hydroxycholecalciferol (25-hydroxyvitamin D3) to form 24(R),25-dihydroxycholecalciferol." [ISBN:0471331309]
GO:0008404
name: arachidonic acid 14,15-epoxygenase activity
def: "Catalysis of an NADPH- and oxygen-dependent reaction that converts arachidonic acid to cis-14,15-epoxyeicosatrienoic acid." [http://lipidlibrary.aocs.org/Lipids/eic_hete/index.htm, PMID:10681399]
GO:0008405
name: arachidonic acid 11,12-epoxygenase activity
def: "Catalysis of an NADPH- and oxygen-dependent reaction that converts arachidonic acid to cis-11,12-epoxyeicosatrienoic acid." [http://lipidlibrary.aocs.org/Lipids/eic_hete/index.htm, PMID:10681399]
GO:0008408
name: 3'-5' exonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 3' end." [GOC:ai]
GO:0008409
name: 5'-3' exonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 5' end." [GOC:ai]
GO:0008410
name: CoA-transferase activity
def: "Catalysis of the transfer of a coenzyme A (CoA) group from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0008411
name: 4-hydroxybutyrate CoA-transferase activity
def: "Catalysis of the transfer of a coenzyme A (CoA) group to 4-hydroxybutyrate." [GOC:jl]
GO:0008412
name: 4-hydroxybenzoate octaprenyltransferase activity
def: "Catalysis of the reaction: farnesylfarnesylgeranyl diphosphate + p-hydroxybenzoate = 3-octaprenyl-4-hydroxybenzoate + diphosphate." [MetaCyc:4OHBENZOATE-OCTAPRENYLTRANSFER-RXN]
GO:0008413
name: 8-oxo-7,8-dihydroguanosine triphosphate pyrophosphatase activity
def: "Catalysis of the reaction: 8-oxo-7,8-dihydroguanosine triphosphate = 8-oxo-7,8-dihydroguanosine phosphate + diphosphate. 8-oxo-7,8-dihydroguanosine triphosphate (8-oxo-GTP) is the oxidised form of the free guanine nucleotide and can act as a potent mutagenic substrate for transcription." [PMID:15878881]
GO:0008414
name: CDP-alcohol phosphotransferase activity
def: "Catalysis of the transfer of a CDP-alcohol group from one compound to another." [GOC:jl]
GO:0008416
name: delta5-delta2,4-dienoyl-CoA isomerase activity
def: "Catalysis of the isomerization of 3,5-dienoyl-CoA to 2,4-dienoyl-CoA." [PMID:11278886]
GO:0008417
name: fucosyltransferase activity
def: "Catalysis of the transfer of a fucosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
GO:0008418
name: protein-N-terminal asparagine amidohydrolase activity
def: "Catalysis of the reaction: protein-L-asparagine + H2O = protein-L-aspartate + NH3. This reaction is the deamidation of an N-terminal asparagine residue in a peptide or protein." [PMID:8910481]
GO:0008419
name: RNA lariat debranching enzyme activity
def: "Catalysis of the hydrolysis of branched RNA structures that contain vicinal 2'-5'- and 3'-5'-phosphodiester bonds at a branch point nucleotide." [PMID:7519612]
GO:0008420
name: CTD phosphatase activity
def: "Catalysis of the reaction: phospho-(DNA-directed RNA polymerase) + H2O = (DNA-directed RNA polymerase) + phosphate." [EC:3.1.3.16]
GO:0008421
name: long-chain fatty-acyl-glutamate deacylase activity
def: "Catalysis of the reaction: N-long-chain-fatty-acyl-L-glutamate + H2O = a fatty acid anion + L-glutamate." [EC:3.5.1.55]
GO:0008422
name: beta-glucosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-glucose residues with release of beta-D-glucose." [EC:3.2.1.21]
GO:0008423
name: bleomycin hydrolase activity
def: "OBSOLETE. Catalysis of the inactivation of bleomycin B2 (a cytotoxic glycometallopeptide) by hydrolysis of a peptide bond of beta-aminoalanine, but also shows general aminopeptidase activity. The specificity varies somewhat with source, but amino acid arylamides of Met, Leu and Ala are preferred." [EC:3.4.22.40]
GO:0008424
name: glycoprotein 6-alpha-L-fucosyltransferase activity
def: "Catalysis of the reaction: N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-N-acetyl-beta-D-glucosaminyl}-L-asparagine + GDP-L-fucose = N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-[alpha-L-fucosyl-(1->6)]-N-acetyl-beta-D-glucosaminyl}asparagine + GDP + H(+)." [EC:2.4.1.68, RHEA:12988]
GO:0008425
name: 2-polyprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-polyprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:kd, PMID:9083048]
GO:0008426
name: protein kinase C inhibitor activity
def: "Stops, prevents or reduces the activity of protein kinase C, an enzyme which phosphorylates a protein." [GOC:ai]
GO:0008427
name: calcium-dependent protein kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a calcium-dependent protein kinase." [GOC:mah]
GO:0008428
name: ribonuclease inhibitor activity
def: "Stops, prevents or reduces the activity of a ribonuclease, any enzyme that catalyzes the hydrolysis of phosphodiester bonds in chains of RNA." [GOC:ai]
GO:0008429
name: phosphatidylethanolamine binding
def: "Interacting selectively and non-covalently with phosphatidylethanolamine, any of a class of glycerophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of ethanolamine." [ISBN:0198506732]
GO:0008430
name: selenium binding
def: "Interacting selectively and non-covalently with selenium (Se)." [GOC:ai]
GO:0008431
name: vitamin E binding
def: "Interacting selectively and non-covalently with vitamin E, tocopherol, which includes a series of eight structurally similar compounds. Alpha-tocopherol is the most active form in humans and is a powerful biological antioxidant." [CHEBI:33234, GOC:curators, ISBN:0721662544]
GO:0008432
name: JUN kinase binding
def: "Interacting selectively and non-covalently with JUN kinase, an enzyme that catalyzes the phosphorylation and activation of members of the JUN family." [GOC:jl]
GO:0008434
name: calcitriol receptor activity
def: "Combining with calcitriol, the hormonally active form of vitamin D3, and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [CHEBI:17823, GOC:bf, GOC:signaling, PMID:16549446, PMID:17082781, Wikipedia:Calcitriol_receptor]
GO:0008435
name: anticoagulant activity
def: "OBSOLETE. Functions to retard or prevent coagulation. Often used in the context of blood or milk coagulation." [ISBN:0198506732]
GO:0008436
name: heterogeneous nuclear ribonucleoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008437
name: thyrotropin-releasing hormone activity
def: "The action characteristic of thyrotropin-releasing hormone (TRH), a hormone released by the mammalian hypothalamus into the hypophyseal-portal circulation in response to neural and/or chemical stimuli. Upon receptor binding, TRH increases the secretion of thyroid-stimulating hormone by the anterior pituitary." [ISBN:0198506732]
GO:0008438
name: 1-phosphatidylinositol-5-phosphate kinase
def: "OBSOLETE. Catalysis of the reaction: diphosphate + a purine nucleoside = phosphate + a purine mononucleotide." [EC:2.7.1.143]
GO:0008439
name: monophenol monooxygenase activator activity
def: "OBSOLETE. Increases the activity of the enzyme monophenol monooxygenase." [GOC:ai]
GO:0008440
name: inositol-1,4,5-trisphosphate 3-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,4,5-trisphosphate + ATP = 1D-myo-inositol 1,3,4,5-tetrakisphosphate + ADP + 2 H(+)." [EC:2.7.1.127, RHEA:11023]
GO:0008441
name: 3'(2'),5'-bisphosphate nucleotidase activity
def: "Catalysis of the reaction: adenosine 3',5'-bisphosphate + H2O = adenosine 5'-phosphate + phosphate." [EC:3.1.3.7]
GO:0008442
name: 3-hydroxyisobutyrate dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxy-2-methylpropanoate + NAD+ = 2-methyl-3-oxopropanoate + NADH + H+." [EC:1.1.1.31]
GO:0008443
name: phosphofructokinase activity
def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a fructose substrate molecule." [GOC:jl]
GO:0008444
name: CDP-diacylglycerol-glycerol-3-phosphate 3-phosphatidyltransferase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + CDP-diacylglycerol = 3-(3-sn-phosphatidyl)-sn-glycerol 1-phosphate + CMP + H(+)." [EC:2.7.8.5, RHEA:12596]
GO:0008445
name: D-aspartate oxidase activity
def: "Catalysis of the reaction: D-aspartate + H2O + O2 = oxaloacetate + NH3 + hydrogen peroxide." [EC:1.4.3.1]
GO:0008446
name: GDP-mannose 4,6-dehydratase activity
def: "Catalysis of the reaction: GDP-alpha-D-mannose = GDP-4-dehydro-6-deoxy-alpha-D-mannose + H(2)O." [EC:4.2.1.47, RHEA:23823]
GO:0008447
name: L-ascorbate oxidase activity
def: "Catalysis of the reaction: 2 L-ascorbate + O2 = 2 dehydroascorbate + 2 H2O." [EC:1.10.3.3]
GO:0008448
name: N-acetylglucosamine-6-phosphate deacetylase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosamine 6-phosphate + H2O = D-glucosamine 6-phosphate + acetate." [EC:3.5.1.25]
GO:0008449
name: N-acetylglucosamine-6-sulfatase activity
def: "Catalysis of the hydrolysis of the 6-sulfate group of the N-acetyl-D-glucosamine 6-sulfate units of heparan sulfate and keratan sulfate." [EC:3.1.6.14]
GO:0008450
name: O-sialoglycoprotein endopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of O-sialoglycoproteins; cleaves the -Arg31-Asp32- bond in glycophorin A. Does not cleave unglycosylated proteins, desialylated glycoproteins or glycoproteins that are only N-glycosylated." [EC:3.4.24.57]
GO:0008451
name: X-Pro aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of any N-terminal amino acid, including proline, that is linked with proline, even from a dipeptide or tripeptide." [EC:3.4.11.9]
GO:0008452
name: RNA ligase activity
def: "Catalysis of the formation of a phosphodiester bond between a hydroxyl group at the end of one RNA chain and the 5'-phosphate group at the end of another." [GOC:mah]
GO:0008453
name: alanine-glyoxylate transaminase activity
def: "Catalysis of the reaction: L-alanine + glyoxylate = pyruvate + glycine." [EC:2.6.1.44]
GO:0008454
name: alpha-1,3-mannosylglycoprotein 4-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + (N-acetyl-beta-D-glucosaminyl-1,2)-alpha-D-mannosyl-1,3-(beta-N-acetyl-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6)-beta-D-mannosyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,4-(N-acetyl-D-glucosaminyl-1,2)-alpha-D-mannosyl-1,3-(beta-N-acetyl-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6)-beta-D-mannosyl-R." [EC:2.4.1.145]
GO:0008455
name: alpha-1,6-mannosylglycoprotein 2-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + alpha-D-mannosyl-1,6-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,3)-beta-D-mannosyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,3)-beta-D-mannosyl-R." [EC:2.4.1.143]
GO:0008456
name: alpha-N-acetylgalactosaminidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing N-acetyl-D-galactosamine residues in N-acetyl-alpha-D-galactosaminides." [EC:3.2.1.49]
GO:0008457
name: beta-galactosyl-N-acetylglucosaminylgalactosylglucosyl-ceramide beta-1,3-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide = UDP + N-acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide." [EC:2.4.1.163]
GO:0008458
name: carnitine O-octanoyltransferase activity
def: "Catalysis of the reaction: (R)-carnitine + octanoyl-CoA = (S)-octanoylcarnitine + CoA." [EC:2.3.1.137, RHEA:17180]
GO:0008459
name: chondroitin 6-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + chondroitin = adenosine 3',5'-bisphosphate + chondroitin 6'-sulfate." [EC:2.8.2.17]
GO:0008460
name: dTDP-glucose 4,6-dehydratase activity
def: "Catalysis of the reaction: dTDP-glucose = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + H(2)O." [EC:4.2.1.46, RHEA:17224]
GO:0008462
name: endopeptidase Clp activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins to small peptides in the presence of ATP and magnesium. Alpha-casein is the usual test substrate. In the absence of ATP, only oligopeptides shorter than five residues are cleaved, for example, succinyl-Leu-Tyr-NHMec which is cleaved at the Tyr-NHMec bond, and Leu-Tyr-Leu-Tyr-Trp which is cleaved at the second Leu-Typ bond (cleavage of the Tyr-Leu and Tyr-Trp bonds also occurs)." [EC:3.4.21.92]
GO:0008463
name: formylmethionine deformylase activity
def: "Catalysis of the reaction: N-formyl-L-methionine + H(2)O = L-methionine + formate." [EC:3.5.1.31, RHEA:17784]
GO:0008464
name: gamma-glutamyl hydrolase activity
def: "OBSOLETE. Catalysis of the cleavage of a gamma-glutamyl bond to release an unsubstituted C-terminal amino acid." [EC:3.4.19.9]
GO:0008465
name: glycerate dehydrogenase activity
def: "Catalysis of the reaction: (R)-glycerate + NAD+ = hydroxypyruvate + NADH + H+." [EC:1.1.1.29]
GO:0008466
name: glycogenin glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + glycogenin = UDP + glucosylglycogenin." [EC:2.4.1.186]
GO:0008467
name: [heparan sulfate]-glucosamine 3-sulfotransferase 1 activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine 3',5'-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate. The [heparan sulfate]-glucosamine 3-sulfate has a substrate consensus sequence of Glc(N2S>NAc)+/-6S GlcA GlcN2S*+/-6S GlcA>IdoA+/-2S Glc(N2S/NAc)+/-6S." [EC:2.8.2.23]
GO:0008469
name: histone-arginine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone)-arginine = S-adenosyl-L-homocysteine + (histone)-N-methyl-arginine." [EC:2.1.1.125]
GO:0008470
name: isovaleryl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 3-methylbutanoyl-CoA + ETF = 3-methylbut-2-enoyl-CoA + reduced ETF." [EC:1.3.99.10]
GO:0008471
name: laccase activity
def: "OBSOLETE. Catalysis of the reaction: 4 benzenediol + O2 = 4 benzosemiquinone + 2 H2O." [EC:1.10.3.2]
GO:0008472
name: metallocarboxypeptidase D activity
def: "OBSOLETE. Catalysis of the reaction: peptidyl-L-lysine (or peptidyl-L-arginine) + H2O = peptide + L-lysine (or L-arginine). Function is activated by Co2+; inhibited by guanidinoethylmercaptosuccinic acid." [EC:3.4.17.22]
GO:0008473
name: ornithine cyclodeaminase activity
def: "Catalysis of the reaction: L-ornithine = L-proline + NH(4)(+)." [EC:4.3.1.12, RHEA:24371]
GO:0008474
name: palmitoyl-(protein) hydrolase activity
def: "Catalysis of the reaction: palmitoyl-protein + H2O = palmitate + protein." [EC:3.1.2.22]
GO:0008475
name: procollagen-lysine 5-dioxygenase activity
def: "Catalysis of the reaction: procollagen L-lysine + 2-oxoglutarate + O2 = procollagen 5-hydroxy-L-lysine + succinate + CO2." [EC:1.14.11.4]
GO:0008476
name: protein-tyrosine sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + protein tyrosine = adenosine 3',5'-bisphosphate + protein tyrosine-O-sulfate." [EC:2.8.2.20]
GO:0008477
name: purine nucleosidase activity
def: "Catalysis of the reaction: a N-D-ribosylpurine + H2O = a purine + D-ribose." [EC:3.2.2.1]
GO:0008478
name: pyridoxal kinase activity
def: "Catalysis of the reaction: ATP + pyridoxal = ADP + pyridoxal 5'-phosphate." [EC:2.7.1.35]
GO:0008479
name: queuine tRNA-ribosyltransferase activity
def: "Catalysis of the reaction: tRNA guanine + queuine = tRNA queuine + guanine." [EC:2.4.2.29]
GO:0008480
name: sarcosine dehydrogenase activity
def: "Catalysis of the reaction: sarcosine + H2O + electron-transfer flavoprotein = glycine + formaldehyde + reduced electron-transfer flavoprotein." [EC:1.5.8.3, RHEA:19796]
GO:0008481
name: sphinganine kinase activity
def: "Catalysis of the reaction: ATP + sphinganine = ADP + sphinganine 1-phosphate." [EC:2.7.1.91]
GO:0008482
name: sulfite oxidase activity
def: "Catalysis of the reaction: H(2)O + O(2) + sulfite = H(2)O(2) + H(+) + sulfate." [EC:1.8.3.1, RHEA:24603]
GO:0008483
name: transaminase activity
def: "Catalysis of the transfer of an amino group to an acceptor, usually a 2-oxo acid." [ISBN:0198506732]
GO:0008484
name: sulfuric ester hydrolase activity
def: "Catalysis of the reaction: RSO-R' + H2O = RSOOH + R'H. This reaction is the hydrolysis of any sulfuric ester bond, any ester formed from sulfuric acid, O=SO(OH)2." [GOC:ai]
GO:0008486
name: diphosphoinositol-polyphosphate diphosphatase activity
def: "Catalysis of the reaction: diphospho-myo-inositol polyphosphate + H2O = myo-inositol polyphosphate + phosphate." [EC:3.6.1.52]
GO:0008487
name: prenyl-dependent CAAX protease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008488
name: gamma-glutamyl carboxylase activity
def: "Catalysis of the reaction: peptidyl-glutamate + reduced vitamin K + CO2 + O2 = peptidyl-gamma-carboxyglutamate + vitamin K epoxide." [PMID:18374194]
GO:0008489
name: UDP-galactose:glucosylceramide beta-1,4-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-D-galactose + a glucosylceramide = a lactosylceramide + uridine-5'-diphosphate. The glucosylceramide has sphinganine as the long chain base." [MetaCyc:RXN-10764, PMID:9593693]
GO:0008490
name: arsenite secondary active transmembrane transporter activity
def: "Catalysis of the transfer of arsenite from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:jl]
GO:0008492
name: cAMP generating peptide activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008493
name: tetracycline transporter activity
def: "Enables the directed movement of tetracycline into, out of or within a cell, or between cells. Tetracycline is a broad spectrum antibiotic that blocks binding of aminoacyl tRNA to the ribosomes of both Gram-positive and Gram-negative organisms (and those of organelles)." [CHEBI:27902, GOC:curators]
GO:0008494
name: translation activator activity
def: "Any of a group of soluble proteins functioning in the activation of ribosome-mediated translation of mRNA into a polypeptide." [GOC:ai]
GO:0008495
name: protoheme IX farnesyltransferase activity
def: "Catalysis of the reaction: protoheme IX + (2E,6E)-farnesyl diphosphate + H2O = heme o + diphosphate." [MetaCyc:HEMEOSYN-RXN]
GO:0008496
name: mannan endo-1,6-alpha-mannosidase activity
def: "Catalysis of the random hydrolysis of (1->6)-alpha-D-mannosidic linkages in unbranched (1->6)-mannans." [EC:3.2.1.101]
GO:0008498
name: phospholipid scrambling
def: "OBSOLETE. The trans-bilayer migration of phospholipids accelerated by a phospholipid scramblase upon binding calcium ions." [OMIM:604170]
GO:0008499
name: UDP-galactose:beta-N-acetylglucosamine beta-1,3-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + N-acetylglucosamine = galactose-beta-1,3-N-acetylglucosamine + UDP." [PMID:10212226]
GO:0008500
name: glycine-, glutamate-, thienylcyclohexylpiperidine binding
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008502
name: melatonin receptor activity
def: "Combining with melatonin, N-acetyl-5-methoxytryptamine, to initiate a change in cell activity. Melatonin is a neuroendocrine substance that stimulates the aggregation of melanosomes in melanophores, thus lightening the skin." [GOC:ai, ISBN:0198506732]
GO:0008503
name: benzodiazepine receptor activity
def: "Combining with benzodiazepines, a class of drugs with hypnotic, anxiolytic, anticonvulsive, amnestic and myorelaxant properties, to initiate a change in cell activity." [GOC:jl]
GO:0008504
name: monoamine transmembrane transporter activity
def: "Catalysis of the transfer of monoamines, organic compounds that contain one amino group that is connected to an aromatic ring by an ethylene group (-CH2-CH2-), from one side of the membrane to the other." [CHEBI:25375, GOC:mah]
GO:0008506
name: sucrose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sucrose(out) + H+(out) = sucrose(in) + H+(in)." [TC:2.A.1.5.3]
GO:0008507
name: sodium:iodide symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: iodide(out) + Na+(out) = iodide(in) + Na+(in)." [TC:2.A.21.5.1]
GO:0008508
name: bile acid:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: bile acid(out) + Na+(out) = bile acid(in) + Na+(in)." [TC:2.A.28.-.-]
GO:0008509
name: anion transmembrane transporter activity
def: "Catalysis of the transfer of a negatively charged ion from one side of a membrane to the other." [GOC:dgf, GOC:mtg_transport, ISBN:0815340729]
GO:0008510
name: sodium:bicarbonate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + HCO3-(out) = Na+(in) + HCO3-(in)." [TC:2.A.31.2.1]
GO:0008511
name: sodium:potassium:chloride symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + K+(out) + Cl-(out) = Na+(in) + K+(in) + Cl-(in)." [TC:2.A.30.1.1]
GO:0008512
name: sulfate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + H+(out) = sulfate(in) + H+(in)." [TC:2.A.53.-.-]
GO:0008513
name: secondary active organic cation transmembrane transporter activity
def: "Catalysis of the transfer of organic cations from one side of the membrane to the other, up the solute's concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction." [GOC:curators]
GO:0008514
name: organic anion transmembrane transporter activity
def: "Catalysis of the transfer of organic anions from one side of a membrane to the other. Organic anions are atoms or small molecules with a negative charge which contain carbon in covalent linkage." [GOC:ai]
GO:0008515
name: sucrose transmembrane transporter activity
def: "Catalysis of the transfer of sucrose from one side of the membrane to the other. Sucrose is the disaccharide O-beta-D-fructofuranosyl-(2->1)-alpha-D-glucopyranoside, a sweet-tasting, non-reducing sugar isolated industrially from sugar beet or sugar cane." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0008516
name: hexose uniporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: hexose(out) = hexose(in)." [TC:2.A.1.1.5]
GO:0008517
name: folic acid transporter activity
def: "Enables the directed movement of folic acid (pteroylglutamic acid) into, out of or within a cell, or between cells. Folic acid is widely distributed as a member of the vitamin B complex and is essential for the synthesis of purine and pyrimidines." [GOC:ai]
GO:0008518
name: reduced folate carrier activity
def: "Catalysis of the transfer of reduced folate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0008519
name: ammonium transmembrane transporter activity
def: "Catalysis of the transfer of ammonium from one side of a membrane to the other. Ammonium is the cation NH4+ which is formed from N2 by root-nodule bacteria in leguminous plants and is an excretory product in ammonotelic animals." [GOC:go_curators, ISBN:0198506732]
GO:0008520
name: L-ascorbate:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ascorbate(out) + Na+(out) = ascorbate(in) + Na+(in)." [TC:2.A.40.6.1]
GO:0008521
name: acetyl-CoA transporter activity
def: "Enables the directed movement of acetyl-CoA into, out of or within a cell, or between cells. Acetyl-CoA is a derivative of coenzyme A in which the sulfhydryl group is acetylated; it is a metabolite derived from several pathways (e.g. glycolysis, fatty acid oxidation, amino-acid catabolism) and is further metabolized by the tricarboxylic acid cycle. It is a key intermediate in lipid and terpenoid biosynthesis." [GOC:ai]
GO:0008523
name: sodium-dependent multivitamin transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: multivitamin(out) + Na+(out) = multivitamin(in) + Na+(in). Multivitamins include pantothenate, biotin and lipoate." [TC:2.A.21.5.2]
GO:0008524
name: glucose 6-phosphate:phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose 6-phosphate(out) + phosphate(in) = glucose 6-phosphate(in) + phosphate(out)." [TC:2.A.1.-.-]
GO:0008525
name: phosphatidylcholine transporter activity
def: "Enables the directed movement of phosphatidylcholine into, out of or within a cell, or between cells. Phosphatidylcholine refers to a class of glycerophospholipids in which the phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:ai, ISBN:0198506732]
GO:0008526
name: phosphatidylinositol transporter activity
def: "Enables the directed movement of phosphatidylinositol into, out of or within a cell, or between cells. Phosphatidylinositol refers to any glycophospholipids with its sn-glycerol 3-phosphate residue is esterified to the 1-hydroxyl group of 1D-myo-inositol." [GOC:ai, ISBN:0198506732]
GO:0008527
name: taste receptor activity
def: "Combining with soluble compounds to initiate a change in cell activity. These receptors are responsible for the sense of taste." [GOC:dph]
GO:0008528
name: G-protein coupled peptide receptor activity
def: "Combining with a peptide and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:dph, GOC:tb]
GO:0008529
name: endogenous peptide receptor activity
def: "OBSOLETE. Combining with an intracellular peptide to initiate a change in cell activity." [GOC:ai]
GO:0008530
name: exogenous peptide receptor activity
def: "OBSOLETE. Combining with an extracellular peptide to initiate a change in cell activity." [GOC:ai]
GO:0008531
name: riboflavin kinase activity
def: "Catalysis of the reaction: ATP + riboflavin = ADP + FMN + 2 H(+)." [EC:2.7.1.26, RHEA:14360]
GO:0008532
name: N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-D-glucosaminyl-R." [EC:2.4.1.149]
GO:0008533
name: astacin activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds in substrates containing five or more amino acids, preferentially with Ala in P1', and Pro in P2'." [EC:3.4.24.21]
GO:0008534
name: oxidized purine nucleobase lesion DNA N-glycosylase activity
def: "Catalysis of the removal of oxidized purine bases by cleaving the N-C1' glycosidic bond between the oxidized purine and the deoxyribose sugar. The reaction involves the formation of a covalent enzyme-substrate intermediate. Release of the enzyme and free base by a beta-elimination or a beta, gamma-elimination mechanism results in the cleavage of the DNA backbone 3' of the apurinic (AP) site." [GOC:elh, PMID:11554296]
GO:0008536
name: Ran GTPase binding
def: "Interacting selectively and non-covalently with Ran, a conserved Ras-like GTP-binding protein, implicated in nucleocytoplasmic transport, cell cycle progression, spindle assembly, nuclear organization and nuclear envelope (NE) assembly." [GOC:rn, PMID:12787777, PMID:14726649]
GO:0008538
name: proteasome activator activity
def: "OBSOLETE. Catalysis of the activation of the proteasome, a large multisubunit complex which performs regulated ubiquitin-dependent cytosolic and nuclear proteolysis." [GOC:rn, PMID:10428771]
GO:0008539
name: proteasome inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the proteasome complex. The proteasome complex performs regulated ubiquitin-dependent cytosolic and nuclear proteolysis." [GOC:mah]
GO:0008545
name: JUN kinase kinase activity
def: "Catalysis of the phosphorylation of tyrosine and threonine residues in a c-Jun NH2-terminal kinase (JNK), a member of a subgroup of mitogen-activated protein kinases (MAPKs), which signal in response to cytokines and exposure to environmental stress. JUN kinase kinase (JNKK) is a dual-specificity protein kinase kinase and requires activation by a serine/threonine kinase JUN kinase kinase kinase." [GOC:bf, PMID:11057897, PMID:11790549]
GO:0008547
name: protein-synthesizing GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. A GTPase involved in protein synthesis. In the initiation factor complex, it is IF-2b (98 kDa) that binds GTP and subsequently hydrolyzes it in prokaryotes. In eukaryotes, it is eIF-2 (150 kDa) that binds GTP. In the elongation phase, the GTP-hydrolyzing proteins are the EF-Tu polypeptide of the prokaryotic transfer factor (43 kDa), the eukaryotic elongation factor EF-1a (53 kDa), the prokaryotic EF-G (77 kDa), the eukaryotic EF-2 (70-110 kDa) and the signal recognition particle that play a role in endoplasmic reticulum protein synthesis (325 kDa). EF-Tu and EF1a catalyze binding of aminoacyl-tRNA to the ribosomal A-site, while EF-G and EF-2 catalyze the translocation of peptidyl-tRNA from the A-site to the P-site. GTPase activity is also involved in polypeptide release from the ribosome with the aid of the pRFs and eRFs." [EC:3.6.5.3, MetaCyc:3.6.1.48-RXN]
GO:0008548
name: signal-recognition-particle GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. Activity is associated with the signal-recognition particle, a protein and RNA-containing structure involved in endoplasmic reticulum-associated protein synthesis." [EC:3.6.5.4, MetaCyc:3.6.1.49-RXN]
GO:0008549
name: dynamin GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. An enzyme that is involved in endocytosis and is instrumental in pinching off membrane vesicles." [EC:3.6.5.5, MetaCyc:3.6.1.50-RXN]
GO:0008550
name: tubulin GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate. An intrinsic activity of alpha-tubulin involved in tubulin folding, division plane formation in prokaryotic cells and others." [EC:3.6.5.6, MetaCyc:3.6.1.51-RXN]
GO:0008551
name: cadmium-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Cd2+(in) -> ADP + phosphate + Cd2+(out)." [EC:3.6.3.3]
GO:0008552
name: zinc, cadmium, cobalt, nickel, lead-efflux ATPase activity
def: "OBSOLETE. Catalysis of the reaction: Me2+(in) + ATP = Me2+(out) + ADP + phosphate, where Me is Zn2+, Cd2+, Co2+, Ni2+ or Pb2+." [TC:3.A.3.6.2]
GO:0008553
name: hydrogen-exporting ATPase activity, phosphorylative mechanism
def: "Catalysis of the transfer of protons from one side of a membrane to the other according to the reaction: ATP + H2O + H+(in) -> ADP + phosphate + H+(out). These transporters use a phosphorylative mechanism, which have a phosphorylated intermediate state during the ion transport cycle." [EC:3.6.3.6]
GO:0008554
name: sodium-exporting ATPase activity, phosphorylative mechanism
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Na+(in) -> ADP + phosphate + Na+(out); by a phosphorylative mechanism." [EC:3.6.3.7]
GO:0008555
name: chloride-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Cl-(out) = ADP + phosphate + Cl-(in)." [EC:3.6.3.11]
GO:0008556
name: potassium-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + K+(out) = ADP + phosphate + K+(in)." [EC:3.6.3.12]
GO:0008558
name: guanine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + guanine(out) = ADP + phosphate + guanine(in)." [EC:3.6.3.37]
GO:0008559
name: xenobiotic-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + xenobiotic(in) = ADP + phosphate + xenobiotic(out)." [EC:3.6.3.44]
GO:0008563
name: alpha-factor sex pheromone exporter
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + alpha-factor(in) = ADP + phosphate + alpha-factor(out). The export of the alpha-factor sex pheromone by an ABC-type (ATP-binding cassette-type) ATPase, characterized by the presence of two similar ATP-binding domains, that does not undergo phosphorylation during the transport process." [EC:3.6.3.48]
GO:0008564
name: protein-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + protein+(in) -> ADP + phosphate + protein+(out); drives the concomitant secretion of proteins." [EC:3.6.3.50]
GO:0008565
name: protein transporter activity
def: "Enables the directed movement of proteins into, out of or within a cell, or between cells." [ISBN:0198506732]
GO:0008566
name: mitochondrial protein-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate; drives the transport of proteins into the mitochondrion via the mitochondrial inner membrane translocase complex." [EC:3.6.3.51]
GO:0008567
name: dynein ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP by dynein that provides the energy for the movement of organelles (endosomes, lysosomes, mitochondria) along microtubules to the centrosome." [EC:3.6.4.2]
GO:0008568
name: microtubule-severing ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate. Catalysis of the severing of a microtubule at a specific spot along its length, coupled to the hydrolysis of ATP." [EC:3.6.4.3, PMID:10910766]
GO:0008569
name: ATP-dependent microtubule motor activity, minus-end-directed
def: "Catalysis of movement along a microtubule toward the minus end, coupled to the hydrolysis of ATP." [EC:3.6.4.5, GOC:mah, GOC:vw]
GO:0008570
name: myosin ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP by myosin that provides the energy for actomyosin contraction." [EC:3.6.4.1]
GO:0008571
name: non-chaperonin molecular chaperone ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. This is a highly diverse group of enzymes that perform many functions that are similar to those of chaperonins. They comprise a number of heat-shock-cognate proteins. They are also active in clathrin uncoating and in the oligomerization of actin." [EC:3.6.4.10]
GO:0008572
name: nucleoplasmin ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP required for the ATP-dependent assembly of nucleosome cores, in decondensation of sperm chromatin and in other histone-involving processes." [EC:3.6.4.11]
GO:0008573
name: peroxisome-assembly ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. ATP hydrolysis to import and assemble peroxisome components into the organelle." [EC:3.6.4.7]
GO:0008574
name: ATP-dependent microtubule motor activity, plus-end-directed
def: "Catalysis of movement along a microtubule toward the plus end, coupled to the hydrolysis of ATP." [EC:3.6.4.4, GOC:vw]
GO:0008575
name: proteasome ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP for channel gating and polypeptide unfolding before proteolysis in the proteasome. Six ATPase subunits are present in the regulatory particle (RP) of 26S proteasome." [EC:3.6.4.8]
GO:0008576
name: vesicle-fusing ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate. The hydrolysis of ATP by an AAA-ATPase, involved in the heterotypic fusion of membrane vesicles with target membranes and the homotypic fusion of various membrane compartments." [EC:3.6.4.6]
GO:0008579
name: JUN kinase phosphatase activity
def: "Catalysis of the reaction: JUN kinase serine/threonine/tyrosine phosphate + H2O = JUN kinase serine/threonine/tyrosine + phosphate." [GOC:mah]
GO:0008580
name: cytoskeletal regulator activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008581
name: ubiquitin-specific protease 5 activity
def: "OBSOLETE. Catalysis of the hydrolysis of the Gly76-Lys48 isopeptide bond of polyubiquitin." [ISBN:0120793709, MEROPS:c19p001]
GO:0008597
name: calcium-dependent protein serine/threonine phosphatase regulator activity
def: "Modulation of the activity of the enzyme calcium-dependent protein serine/threonine phosphatase." [EC:3.1.3.16, GOC:ai]
GO:0008599
name: protein phosphatase type 1 regulator activity
def: "Modulation of the activity of the enzyme protein phosphatase type 1." [GOC:ai]
GO:0008601
name: protein phosphatase type 2A regulator activity
def: "Modulation of the activity of the enzyme protein phosphatase type 2A." [GOC:ai]
GO:0008603
name: cAMP-dependent protein kinase regulator activity
def: "Modulation of the activity of the enzyme cAMP-dependent protein kinase." [GOC:ai]
GO:0008605
name: protein kinase CK2 regulator activity
def: "OBSOLETE. Modulation of the activity of the enzyme protein kinase CK2." [GOC:ai]
GO:0008607
name: phosphorylase kinase regulator activity
def: "Modulation of the activity of the enzyme phosphorylase kinase." [EC:2.7.11.19]
GO:0008609
name: alkylglycerone-phosphate synthase activity
def: "Catalysis of the reaction: 1-acyl-glycerone 3-phosphate + a long-chain alcohol = 1-alkyl-glycerone 3-phosphate + a long-chain acid anion." [EC:2.5.1.26]
GO:0008613
name: diuretic hormone activity
def: "The action characteristic of a diuretic hormone, any peptide hormone that, upon receptor binding, regulates water balance and fluid secretion." [GOC:mah, InterPro:IPR003621, PMID:8618894]
GO:0008619
name: RHEB small monomeric GTPase activity
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.47, PMID:12893813]
GO:0008638
name: protein tagging activity
def: "OBSOLETE. Covalent addition of a specific tagging molecule to a protein, targeting the tagged protein for some fate e.g. degradation." [GOC:jl]
GO:0008641
name: small protein activating enzyme activity
def: "Catalysis of the activation of small proteins, such as ubiquitin or ubiquitin-like proteins, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:jl, GOC:mah]
GO:0008648
name: tachykinin
def: "OBSOLETE. A family of hormones that stimulate secretion of saliva and cause smooth muscle contraction and vasodilation." [ISBN:0198506732, ISBN:0721662544]
GO:0008649
name: rRNA methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a nucleoside residue in an rRNA molecule." [GOC:mah]
GO:0008650
name: rRNA (uridine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methyluridine." [GOC:mah]
GO:0008651
name: actin polymerizing activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008656
name: cysteine-type endopeptidase activator activity involved in apoptotic process
def: "Increases the rate of proteolysis catalyzed by a cysteine-type endopeptidase involved in the apoptotic process." [GOC:mah, GOC:mtg_apoptosis]
GO:0008657
name: DNA topoisomerase (ATP-hydrolyzing) inhibitor activity
def: "Stops, prevents or reduces the activity of ATP-hydrolyzing DNA topoisomerase. ATP-hydrolyzing DNA topoisomerase catalyzes the DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:mah]
GO:0008658
name: penicillin binding
def: "Interacting selectively and non-covalently with penicillin, any antibiotic that contains the condensed beta-lactamthiazolidine ring system." [GOC:ai]
GO:0008659
name: (3R)-hydroxymyristoyl-[acyl-carrier-protein] dehydratase activity
def: "Catalysis of the reaction: (3R)-3-hydroxytetradecanoyl-[acyl-carrier protein] = tetradecenoyl-[acyl-carrier protein] + H2O." [EC:4.2.1.-, GOC:ai]
GO:0008660
name: 1-aminocyclopropane-1-carboxylate deaminase activity
def: "Catalysis of the reaction: 1-aminocyclopropane-1-carboxylate + H(2)O = 2-oxobutanate + NH(4)(+)." [EC:3.5.99.7, RHEA:16936]
GO:0008661
name: 1-deoxy-D-xylulose-5-phosphate synthase activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + H(+) + pyruvate = 1-deoxy-D-xylulose 5-phosphate + CO(2)." [EC:2.2.1.7, RHEA:12608]
GO:0008662
name: 1-phosphofructokinase activity
def: "Catalysis of the reaction: ATP + D-fructose 1-phosphate = ADP + D-fructose 1,6-bisphosphate." [EC:2.7.1.56]
GO:0008663
name: 2',3'-cyclic-nucleotide 2'-phosphodiesterase activity
def: "Catalysis of the reaction: nucleoside 2',3'-cyclic phosphate + H2O = nucleoside 3'-phosphate." [EC:3.1.4.16]
GO:0008664
name: 2'-5'-RNA ligase activity
def: "Catalysis of the formation of a phosphodiester bond between the 2'-hydroxyl group at the end of one DNA chain and the 5'-phosphate group at the end of another." [GOC:mah, PMID:8940112]
GO:0008665
name: 2'-phosphotransferase activity
def: "Catalysis of the transfer of a phosphate group from one compound to the 2' position of another." [GOC:jl, GOC:mah]
GO:0008666
name: 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase activity
def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + H(2)O + succinyl-CoA = L-2-succinylamino-6-oxopimelate + CoA." [EC:2.3.1.117, RHEA:17328]
GO:0008667
name: 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase activity
def: "Catalysis of the reaction: (2S,3S)-2,3-dihydroxy-2,3-dihydrobenzoate + NAD(+) = 2,3-dihydroxybenzoate + H(+) + NADH." [EC:1.3.1.28, RHEA:23827]
GO:0008668
name: (2,3-dihydroxybenzoyl)adenylate synthase activity
def: "Catalysis of the reaction: 2,3-dihydroxybenzoate + ATP = 2,3-dihydroxybenzoyl 5'-adenylate + diphosphate." [EC:2.7.7.58, RHEA:20232]
GO:0008670
name: 2,4-dienoyl-CoA reductase (NADPH) activity
def: "Catalysis of the reaction: trans-2,3-didehydroacyl-CoA + NADP+ = trans,trans-2,3,4,5-tetradehydroacyl-CoA + NADPH + H+." [EC:1.3.1.34]
GO:0008671
name: 2-dehydro-3-deoxygalactonokinase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-galactonate + ATP = 6-phospho-2-dehydro-3-deoxy-D-galactonate + ADP + 2 H(+)." [EC:2.7.1.58, RHEA:16528]
GO:0008672
name: 2-dehydro-3-deoxyglucarate aldolase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-glucarate = pyruvate + tartronate semialdehyde." [EC:4.1.2.20]
GO:0008673
name: 2-dehydro-3-deoxygluconokinase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-gluconate + ATP = 2-dehydro-3-deoxy-6-phospho-D-gluconate + ADP + 2 H(+)." [EC:2.7.1.45, RHEA:14800]
GO:0008674
name: 2-dehydro-3-deoxy-6-phosphogalactonate aldolase activity
def: "Catalysis of the reaction: 6-phospho-2-dehydro-3-deoxy-D-galactonate = D-glyceraldehyde 3-phosphate + pyruvate." [EC:4.1.2.21, RHEA:24467]
GO:0008675
name: 2-dehydro-3-deoxy-phosphogluconate aldolase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-gluconate 6-phosphate = pyruvate + D-glyceraldehyde 3-phosphate." [EC:4.1.2.14]
GO:0008676
name: 3-deoxy-8-phosphooctulonate synthase activity
def: "Catalysis of the reaction: D-arabinose 5-phosphate + H(2)O + phosphoenolpyruvate = 8-phospho-3-deoxy-D-manno-oct-2-ulosonate + 2 H(+) + phosphate." [EC:2.5.1.55, RHEA:14056]
GO:0008677
name: 2-dehydropantoate 2-reductase activity
def: "Catalysis of the reaction: (R)-pantoate + NADP(+) = 2-dehydropantoate + H(+) + NADPH." [EC:1.1.1.169, RHEA:16236]
GO:0008678
name: 2-deoxy-D-gluconate 3-dehydrogenase activity
def: "Catalysis of the reaction: 2-deoxy-D-gluconate + NAD+ = 3-dehydro-2-deoxy-D-gluconate + NADH + H+." [EC:1.1.1.125]
GO:0008679
name: 2-hydroxy-3-oxopropionate reductase activity
def: "Catalysis of the reaction: (R)-glycerate + NADP+ = 2-hydroxy-3-oxopropanoate + NADPH + H+." [EC:1.1.1.60]
GO:0008681
name: 2-octaprenyl-6-methoxyphenol hydroxylase activity
def: "Catalysis of the reaction: 2-octaprenyl-6-methoxyphenol + O2 + 4 H+ = 2-octaprenyl-6-methoxy-1,4-benzoquinol + H2O." [MetaCyc:2-OCTAPRENYL-6-METHOXYPHENOL-HYDROX-RXN]
GO:0008682
name: 2-octoprenyl-3-methyl-6-methoxy-1,4-benzoquinone hydroxylase activity
def: "Catalysis of the reaction: 2-octoprenyl-3-methyl-6-methoxy-1,4-benzoquinone + O2 = 2-octoprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone." [MetaCyc:OCTAPRENYL-METHYL-METHOXY-BENZOQ-OH-RXN]
GO:0008683
name: 2-oxoglutarate decarboxylase activity
def: "Catalysis of the reaction: 2-oxoglutarate + H(+) = CO(2) + succinate semialdehyde." [EC:4.1.1.71, RHEA:10527]
GO:0008684
name: 2-oxopent-4-enoate hydratase activity
def: "Catalysis of the reaction: 4-hydroxy-2-oxopentanoate = 2-oxopent-4-enoate + H2O." [EC:4.2.1.80]
GO:0008685
name: 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase activity
def: "Catalysis of the reaction: 4-CDP-2-C-methyl-D-erythritol 2-phosphate = 2-C-methyl-D-erythritol 2,4-cyclic diphosphate + CMP." [EC:4.6.1.12, RHEA:23867]
GO:0008686
name: 3,4-dihydroxy-2-butanone-4-phosphate synthase activity
def: "Catalysis of the reaction: D-ribulose 5-phosphate = (2S)-2-hydroxy-3-oxobutyl phosphate + formate + H(+)." [EC:4.1.99.12, RHEA:18460]
GO:0008687
name: 3,4-dihydroxyphenylacetate 2,3-dioxygenase activity
def: "Catalysis of the reaction: 3,4-dihydroxyphenylacetate + O(2) = 5-formyl-2-hydroxyhepta-2,4-dienedioate + H(+)." [EC:1.13.11.15, RHEA:15636]
GO:0008688
name: 3-(3-hydroxyphenyl)propionate hydroxylase activity
def: "Catalysis of the reaction: 3-(3-hydroxyphenyl)propionate + NADH + oxygen + H+ = 3-(2,3-dihydroxyphenyl)propionate + NAD+ + H2O." [MetaCyc:MHPHYDROXY-RXN]
GO:0008689
name: 3-demethylubiquinone-9 3-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-demethylubiquinone-9 = S-adenosyl-L-homocysteine + ubiquinone-9." [GOC:dph]
GO:0008690
name: 3-deoxy-manno-octulosonate cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + 3-deoxy-D-manno-octulosonate = diphosphate + CMP-3-deoxy-D-manno-octulosonate." [EC:2.7.7.38]
GO:0008691
name: 3-hydroxybutyryl-CoA dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-hydroxybutanoyl-CoA + NADP+ = 3-acetoacetyl-CoA + NADPH + H+." [EC:1.1.1.157]
GO:0008692
name: 3-hydroxybutyryl-CoA epimerase activity
def: "Catalysis of the reaction: (S)-3-hydroxybutanoyl-CoA = (R)-3-hydroxybutanoyl-CoA." [EC:5.1.2.3]
GO:0008693
name: 3-hydroxydecanoyl-[acyl-carrier-protein] dehydratase activity
def: "Catalysis of the reaction: (3R)-3-hydroxydecanoyl-[acyl-carrier protein] = 2,3-decenoyl-[acyl-carrier protein] or 3,4-decenoyl-[acyl-carrier protein] + H2O." [EC:4.2.1.60]
GO:0008694
name: 3-octaprenyl-4-hydroxybenzoate carboxy-lyase activity
def: "Catalysis of the reaction: 3-octaprenyl-4-hydroxy benzoate = 2-octaprenylphenol + CO2." [MetaCyc:3-OCTAPRENYL-4-OHBENZOATE-DECARBOX-RXN]
GO:0008695
name: 3-phenylpropionate dioxygenase activity
def: "Catalysis of the reaction: 3-phenylpropionate + NADH + H+ + O2 = NAD+ + cis-3-(3-carboxyethyl)-3,5-cyclohexadiene-1,2-diol." [UM-BBD_enzymeID:e0307]
GO:0008696
name: 4-amino-4-deoxychorismate lyase activity
def: "Catalysis of the reaction: 4-amino-4-deoxychorismate = 4-aminobenzoate + H(+) + pyruvate." [EC:4.1.3.38, RHEA:16204]
GO:0008697
name: 4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase activity
def: "Catalysis of the reaction: 5-dehydro-4-deoxy-D-glucuronate = 3-deoxy-D-glycero-2,5-hexodiulosonate." [EC:5.3.1.17, RHEA:23899]
GO:0008700
name: 4-hydroxy-2-oxoglutarate aldolase activity
def: "Catalysis of the reaction: 4-hydroxy-2-oxoglutarate = pyruvate + glyoxylate." [EC:4.1.3.16]
GO:0008701
name: 4-hydroxy-2-oxovalerate aldolase activity
def: "Catalysis of the reaction: 4-hydroxy-2-oxopentanoate = acetaldehyde + pyruvate." [EC:4.1.3.39, RHEA:22627]
GO:0008703
name: 5-amino-6-(5-phosphoribosylamino)uracil reductase activity
def: "Catalysis of the reaction: 5-amino-6-(5-phosphoribitylamino)uracil + NADP(+) = 5-amino-6-(5-phosphoribosylamino)uracil + H(+) + NADPH." [EC:1.1.1.193, RHEA:17848]
GO:0008704
name: 5-carboxymethyl-2-hydroxymuconate delta-isomerase activity
def: "Catalysis of the reaction: 5-carboxymethyl-2-hydroxymuconate = 5-carboxy-2-oxohept-3-enedioate." [EC:5.3.3.10]
GO:0008705
name: methionine synthase activity
def: "Catalysis of the reaction: (6S)-5-methyl-5,6,7,8-tetrahydrofolate + L-homocysteine = (6S)-5,6,7,8-tetrahydrofolate + L-methionine." [EC:2.1.1.13, RHEA:11175]
GO:0008706
name: 6-phospho-beta-glucosidase activity
def: "Catalysis of the reaction: 6-phospho-beta-D-glucoside-(1,4)-D-glucose + H2O = D-glucose 6-phosphate + glucose." [EC:3.2.1.86]
GO:0008707
name: 4-phytase activity
def: "Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = 1-myo-inositol 1,2,3,4,5-pentakisphosphate + phosphate." [EC:3.1.3.26]
GO:0008709
name: cholate 7-alpha-dehydrogenase activity
def: "Catalysis of the reaction: cholate + NAD(+) = 3alpha,12alpha-dihydroxy-7-oxo-5beta-cholanate + H(+) + NADH." [EC:1.1.1.159, RHEA:19412]
GO:0008710
name: 8-amino-7-oxononanoate synthase activity
def: "Catalysis of the reaction: L-alanine + H(+) + pimelyl-CoA = 8-amino-7-oxononanoate + CO(2) + CoA." [EC:2.3.1.47, RHEA:20715]
GO:0008711
name: ADP-L-glycero-D-manno-heptose synthase activity
def: "OBSOLETE." [GOC:curators]
GO:0008712
name: ADP-glyceromanno-heptose 6-epimerase activity
def: "Catalysis of the reaction: ADP-D-glycero-D-manno-heptose = ADP-L-glycero-D-manno-heptose." [EC:5.1.3.20, RHEA:17580]
GO:0008713
name: ADP-heptose-lipopolysaccharide heptosyltransferase activity
def: "Catalysis of the reaction: heptosyl-KDO2-lipid A + ADP-L-glycero-beta-D-manno-heptose = heptosyl2-KDO2-lipid A + ADP + H+." [MetaCyc:RXN0-5061]
GO:0008714
name: AMP nucleosidase activity
def: "Catalysis of the reaction: AMP + H(2)O = D-ribose 5-phosphate + adenine." [EC:3.2.2.4, RHEA:20132]
GO:0008715
name: CDP-diacylglycerol diphosphatase activity
def: "Catalysis of the reaction: CDP-diacylglycerol + H(2)O = a phosphatidate + CMP + 2 H(+)." [EC:3.6.1.26, RHEA:15224]
GO:0008716
name: D-alanine-D-alanine ligase activity
def: "Catalysis of the reaction: 2 D-alanine + ATP = D-alanyl-D-alanine + ADP + 2 H(+) + phosphate." [EC:6.3.2.4, RHEA:11227]
GO:0008717
name: D-alanyl-D-alanine endopeptidase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008718
name: D-amino-acid dehydrogenase activity
def: "Catalysis of the reaction: a D-amino acid + H2O + acceptor = a 2-oxo acid + NH3 + reduced acceptor." [EC:1.4.99.1]
GO:0008719
name: dihydroneopterin triphosphate 2'-epimerase activity
def: "Catalysis of the reaction: dihydroneopterin triphosphate = dihydromonapterin-triphosphate." [MetaCyc:H2NTPEPIM-RXN]
GO:0008720
name: D-lactate dehydrogenase activity
def: "Catalysis of the reaction: (R)-lactate + NAD(+) = H(+) + NADH + pyruvate." [EC:1.1.1.28, RHEA:16372]
GO:0008721
name: D-serine ammonia-lyase activity
def: "Catalysis of the reaction: D-serine = pyruvate + NH3." [EC:4.3.1.18]
GO:0008724
name: DNA topoisomerase IV activity
def: "OBSOLETE. Catalysis of the ATP-independent breakage of DNA, followed by passage and rejoining. It also catalyzes the relaxation of supercoiled DNA, and the decatenation and unknotting of DNA in vivo." [EC:5.99.1.-, PMID:11274059]
GO:0008725
name: DNA-3-methyladenine glycosylase activity
def: "Catalysis of the reaction: DNA containing 3-methyladenine + H2O = DNA with abasic site + 3-methyladenine. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the damaged DNA 3-methyladenine and the deoxyribose sugar to remove the 3-methyladenine, leaving an abasic site." [EC:3.2.2.20, GOC:elh, PMID:10872450, PMID:9224623]
GO:0008726
name: alkanesulfonate monooxygenase activity
def: "Catalysis of the reaction: an alkanesulfonate + O2 + FMNH2 = an aldehyde + sulfite + H2O + FMN." [MetaCyc:RXN0-280]
GO:0008727
name: GDP-mannose mannosyl hydrolase activity
def: "Catalysis of the reaction: GDP-alpha-D-mannose + H2O = GDP + D-mannose + H+." [MetaCyc:GDPMANMANHYDRO-RXN]
GO:0008728
name: GTP diphosphokinase activity
def: "Catalysis of the reaction: ATP + GTP = AMP + guanosine 3'-diphosphate 5'-triphosphate." [EC:2.7.6.5]
GO:0008730
name: L(+)-tartrate dehydratase activity
def: "Catalysis of the reaction: L-tartrate = H(2)O + oxaloacetate." [EC:4.2.1.32, RHEA:15416]
GO:0008732
name: L-allo-threonine aldolase activity
def: "Catalysis of the reaction: L-allo-threonine = glycine + acetaldehyde." [MetaCyc:LTAA-RXN, PMID:9228760]
GO:0008733
name: L-arabinose isomerase activity
def: "Catalysis of the reaction: L-arabinose = L-ribulose." [EC:5.3.1.4, RHEA:14824]
GO:0008734
name: L-aspartate oxidase activity
def: "Catalysis of the reaction: L-aspartate + O2 = iminosuccinate + hydrogen peroxide." [EC:1.4.3.16]
GO:0008735
name: carnitine dehydratase activity
def: "Catalysis of the reaction: (R)-carnitine = crotono-betaine + H(2)O." [EC:4.2.1.89, RHEA:14580]
GO:0008736
name: L-fucose isomerase activity
def: "Catalysis of the reaction: L-fucose = L-fuculose." [EC:5.3.1.25]
GO:0008737
name: L-fuculokinase activity
def: "Catalysis of the reaction: L-fuculose + ATP = L-fuculose 1-phosphate + ADP + 2 H(+)." [EC:2.7.1.51, RHEA:12379]
GO:0008738
name: L-fuculose-phosphate aldolase activity
def: "Catalysis of the reaction: L-fuculose 1-phosphate = (S)-lactaldehyde + glycerone phosphate." [EC:4.1.2.17, RHEA:12936]
GO:0008740
name: L-rhamnose isomerase activity
def: "Catalysis of the reaction: L-rhamnose = L-rhamnulose." [EC:5.3.1.14, RHEA:23163]
GO:0008741
name: ribulokinase activity
def: "Catalysis of the reaction: ATP + L(or D)-ribulose = ADP + L(or D)-ribulose 5-phosphate." [EC:2.7.1.16]
GO:0008742
name: L-ribulose-phosphate 4-epimerase activity
def: "Catalysis of the reaction: L-ribulose 5-phosphate = D-xylulose 5-phosphate." [EC:5.1.3.4, RHEA:22371]
GO:0008743
name: L-threonine 3-dehydrogenase activity
def: "Catalysis of the reaction: L-threonine + NAD(+) = aminoacetone + CO(2) + NADH." [EC:1.1.1.103, RHEA:25657]
GO:0008744
name: L-xylulokinase activity
def: "Catalysis of the reaction: ATP + L-xylulose = ADP + L-xylulose 5-phosphate." [EC:2.7.1.53]
GO:0008745
name: N-acetylmuramoyl-L-alanine amidase activity
def: "Catalysis of the hydrolysis of the link between N-acetylmuramoyl residues and L-amino acid residues in certain bacterial cell-wall glycopeptides." [EC:3.5.1.28]
GO:0008746
name: NAD(P)+ transhydrogenase activity
def: "Catalysis of the reaction: NADPH + H+ + NAD+ = NADP+ + NADH + H+." [EC:1.6.1.1, EC:1.6.1.2]
GO:0008747
name: N-acetylneuraminate lyase activity
def: "Catalysis of the reaction: N-acetylneuraminate = N-acetyl-D-mannosamine + pyruvate." [EC:4.1.3.3, RHEA:23299]
GO:0008748
name: N-ethylmaleimide reductase activity
def: "Catalysis of the reaction: N-ethylmaleimide + 2 H+ = N-ethylsuccinimide." [MetaCyc:RXN0-5101]
GO:0008750
name: NAD(P)+ transhydrogenase (AB-specific) activity
def: "Catalysis of the reaction: NADPH + H+ + NAD+ = NADP+ + NADH + H+. The reaction is A-specific (i.e. the pro-R hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to NAD+ and B-specific (i.e. the pro-S hydrogen is transferred) with respect to NADP+." [EC:1.6.1.2, http://pubs.acs.org/cgi-bin/abstract.cgi/jacsat/1991/113/i07/f-pdf/f_ja00007a002.pdf]
GO:0008751
name: NAD(P)H dehydrogenase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0008752
name: FMN reductase activity
def: "Catalysis of the reaction: FMNH2 + NAD(P)+ = FMN + NAD(P)H + H+." [EC:1.5.1.29]
GO:0008753
name: NADPH dehydrogenase (quinone) activity
def: "Catalysis of the reaction: NADPH + H+ + acceptor = NADP+ + reduced acceptor." [EC:1.6.5.10]
GO:0008754
name: O antigen ligase activity
def: "Catalysis of the reaction: Lipid A-core + colanic acid = MLPS." [MetaCyc:RXN0-5294]
GO:0008755
name: O antigen polymerase activity
def: "Catalysis of the polymerization of o-antigen chains. O-antigens are tetra- and pentasaccharide repeat units of the cell walls of Gram-negative bacteria and are a component of lipopolysaccharide." [GOC:jl, PMID:12045108]
GO:0008756
name: o-succinylbenzoate-CoA ligase activity
def: "Catalysis of the reaction: 2-succinylbenzoate + ATP + CoA = 2-succinylbenzoyl-CoA + AMP + diphosphate." [EC:6.2.1.26]
GO:0008757
name: S-adenosylmethionine-dependent methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a substrate." [GOC:mah]
GO:0008758
name: UDP-2,3-diacylglucosamine hydrolase activity
def: "Catalysis of the reaction: H2O + UDP-2,3-bis(3-hydroxymyristoyl)glucosamine = 2,3-bis(3-hydroxymyristoyl)-beta-D-glucosaminyl 1-phosphate + UMP." [MetaCyc:LIPIDXSYNTHESIS-RXN, PMID:12000770]
GO:0008759
name: UDP-3-O-[3-hydroxymyristoyl] N-acetylglucosamine deacetylase activity
def: "Catalysis of the removal of an acetyl group from the 2-N position of glucosamine in the lipid A precursor UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine." [PMID:10026271]
GO:0008760
name: UDP-N-acetylglucosamine 1-carboxyvinyltransferase activity
def: "Catalysis of the reaction: phosphoenolpyruvate + UDP-N-acetyl-alpha-D-glucosamine = phosphate + UDP-N-acetyl-3-O-(1-carboxyvinyl)-D-glucosamine." [EC:2.5.1.7, RHEA:18684]
GO:0008761
name: UDP-N-acetylglucosamine 2-epimerase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine = UDP-N-acetyl-D-mannosamine." [EC:5.1.3.14]
GO:0008762
name: UDP-N-acetylmuramate dehydrogenase activity
def: "Catalysis of the reaction: UDP-N-acetylmuramate + NADP+ = UDP-N-acetyl-3-O-(1-carboxyvinyl)-D-glucosamine + NADPH + H+." [EC:1.1.1.158]
GO:0008763
name: UDP-N-acetylmuramate-L-alanine ligase activity
def: "Catalysis of the reaction: L-alanine + ATP + UDP-N-acetylmuramate = ADP + 2 H(+) + phosphate + UDP-N-acetylmuramoyl-L-alanine." [EC:6.3.2.8, RHEA:23375]
GO:0008764
name: UDP-N-acetylmuramoylalanine-D-glutamate ligase activity
def: "Catalysis of the reaction: D-glutamate + ATP + UDP-N-acetylmuramoyl-L-alanine = ADP + 2 H(+) + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate." [EC:6.3.2.9, RHEA:16432]
GO:0008765
name: UDP-N-acetylmuramoylalanyl-D-glutamate-2,6-diaminopimelate ligase activity
def: "Catalysis of the reaction: meso-2,6-diaminopimelate + ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate = ADP + 2 H(+) + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-meso-2,6-diaminoheptanedioate." [EC:6.3.2.13, RHEA:23679]
GO:0008766
name: UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-diaminopimelate-D-alanyl-D-alanine ligase activity
def: "Catalysis of the reaction: ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminoheptanedioate + D-alanyl-D-alanine = ADP + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-6-carboxy-L-lysyl-D-alanyl-D-alanine." [EC:6.3.2.15]
GO:0008767
name: UDP-galactopyranose mutase activity
def: "Catalysis of the reaction: UDP-D-galactopyranose = UDP-D-galacto-1,4-furanose." [EC:5.4.99.9]
GO:0008768
name: UDP-sugar diphosphatase activity
def: "Catalysis of the reaction: UDP-sugar + H2O = UMP + sugar 1-phosphate." [EC:3.6.1.45]
GO:0008769
name: X-His dipeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of Xaa-His dipeptides." [EC:3.4.13.3]
GO:0008770
name: [acyl-carrier-protein] phosphodiesterase activity
def: "Catalysis of the reaction: [acyl-carrier protein] + H2O = 4'-phosphopantetheine + apoprotein." [EC:3.1.4.14]
GO:0008771
name: [citrate (pro-3S)-lyase] ligase activity
def: "Catalysis of the reaction: ATP + acetate + (citrate (pro-3S)-lyase) (thiol form) = AMP + diphosphate + (citrate (pro-3S)-lyase) (acetyl form)." [EC:6.2.1.22]
GO:0008772
name: [isocitrate dehydrogenase (NADP+)] kinase activity
def: "Catalysis of the reaction: ATP + (isocitrate dehydrogenase (NADP)) = ADP + (isocitrate dehydrogenase (NADP)) phosphate." [EC:2.7.11.5]
GO:0008773
name: [protein-PII] uridylyltransferase activity
def: "Catalysis of the reaction: UTP + (protein-PII) = diphosphate + uridylyl-(protein-PII)." [EC:2.7.7.59]
GO:0008774
name: acetaldehyde dehydrogenase (acetylating) activity
def: "Catalysis of the reaction: acetaldehyde + CoA + NAD+ = acetyl-CoA + NADH + H+." [EC:1.2.1.10]
GO:0008775
name: acetate CoA-transferase activity
def: "Catalysis of the reaction: acyl-CoA + acetate = a fatty acid anion + acetyl-CoA." [EC:2.8.3.8]
GO:0008776
name: acetate kinase activity
def: "Catalysis of the reaction: ATP + acetate = ADP + acetyl phosphate." [EC:2.7.2.1]
GO:0008777
name: acetylornithine deacetylase activity
def: "Catalysis of the reaction: N2-acetyl-L-ornithine + H2O = acetate + L-ornithine." [EC:3.5.1.16]
GO:0008779
name: acyl-[acyl-carrier-protein]-phospholipid O-acyltransferase activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + O-(2-acyl-sn-glycero-3-phospho)ethanolamine = [acyl-carrier protein] + O-(1-beta-acyl-2-acyl-sn-glycero-3-phospho)ethanolamine." [EC:2.3.1.40]
GO:0008780
name: acyl-[acyl-carrier-protein]-UDP-N-acetylglucosamine O-acyltransferase activity
def: "Catalysis of the reaction: (R)-3-hydroxytetradecanoyl-[acyl-carrier protein] + UDP-N-acetylglucosamine = [acyl-carrier protein] + UDP-3-O-(3-hydroxytetradecanoyl)-N-acetylglucosamine." [EC:2.3.1.129]
GO:0008781
name: N-acylneuraminate cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + N-acylneuraminate = diphosphate + CMP-N-acylneuraminate." [EC:2.7.7.43]
GO:0008782
name: adenosylhomocysteine nucleosidase activity
def: "Catalysis of the reaction: S-adenosyl-L-homocysteine + H2O = adenine + S-D-ribosyl-L-homocysteine." [EC:3.2.2.9]
GO:0008783
name: agmatinase activity
def: "Catalysis of the reaction: agmatine + H(2)O = putrescine + urea." [EC:3.5.3.11, RHEA:13932]
GO:0008784
name: alanine racemase activity
def: "Catalysis of the reaction: L-alanine = D-alanine." [EC:5.1.1.1, RHEA:20252]
GO:0008785
name: alkyl hydroperoxide reductase activity
def: "Catalysis of the reaction: octane hydroperoxide + NADH + H+ = H2O + NAD+ + 1-octanol." [UM-BBD_reactionID:r0684]
GO:0008786
name: allose 6-phosphate isomerase activity
def: "Catalysis of the reaction: D-allose-6-phosphate = D-allulose-6-phosphate." [MetaCyc:RXN0-303]
GO:0008787
name: allose kinase activity
def: "Catalysis of the reaction: ATP + D-allose = ADP + D-allose 6-phosphate." [EC:2.7.1.55]
GO:0008788
name: alpha,alpha-phosphotrehalase activity
def: "Catalysis of the reaction: alpha,alpha-trehalose 6-phosphate + H2O = D-glucose + D-glucose 6-phosphate." [EC:3.2.1.93]
GO:0008789
name: altronate dehydratase activity
def: "Catalysis of the reaction: D-altronate = 2-dehydro-3-deoxy-D-gluconate + H(2)O." [EC:4.2.1.7, RHEA:15960]
GO:0008790
name: arabinose isomerase activity
def: "Catalysis of the reaction: D-arabinose = D-ribulose." [EC:5.3.1.3]
GO:0008791
name: arginine N-succinyltransferase activity
def: "Catalysis of the reaction: succinyl-CoA + L-arginine = CoA + N2-succinyl-L-arginine." [EC:2.3.1.109]
GO:0008792
name: arginine decarboxylase activity
def: "Catalysis of the reaction: L-arginine + H(+) = agmatine + CO(2)." [EC:4.1.1.19, RHEA:17644]
GO:0008793
name: aromatic-amino-acid:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: an aromatic amino acid + 2-oxoglutarate = an aromatic oxo acid + L-glutamate." [EC:2.6.1.57]
GO:0008794
name: arsenate reductase (glutaredoxin) activity
def: "Catalysis of the reaction: arsenate + reduced glutaredoxin = arsenite + oxidized glutaredoxin. Glutaredoxin functions as the electron donor for arsenate reduction. The electron flow therefore is ( NADPH -> glutathione reductase (EC:1.6.4.2) -> ) glutathione -> glutaredoxin -> arsenate reductase, i.e. glutathione is reduced by glutathione reductase and glutaredoxin is reduced by glutathione." [EC:1.20.4.1, GOC:kd, PMID:10593884]
GO:0008795
name: NAD+ synthase activity
def: "Catalysis of the reaction: ATP + deamido-NAD+ + NH3 = AMP + diphosphate + NAD+." [EC:6.3.1.5]
GO:0008796
name: bis(5'-nucleosyl)-tetraphosphatase activity
def: "Catalysis of the hydrolysis of P(1),P(4)-bis(5'-nucleosyl)tetraphosphate into two nucleotides." [GOC:ai]
GO:0008797
name: aspartate ammonia-lyase activity
def: "Catalysis of the reaction: L-aspartate = fumarate + NH3." [EC:4.3.1.1]
GO:0008798
name: beta-aspartyl-peptidase activity
def: "Catalysis of the cleavage of a beta-linked aspartic residue from the N-terminus of a polypeptide." [EC:3.4.19.5]
GO:0008800
name: beta-lactamase activity
def: "Catalysis of the reaction: a beta-lactam + H2O = a substituted beta-amino acid." [EC:3.5.2.6]
GO:0008801
name: beta-phosphoglucomutase activity
def: "Catalysis of the reaction: beta-D-glucose 1-phosphate = beta-D-glucose 6-phosphate." [EC:5.4.2.6, RHEA:20116]
GO:0008802
name: betaine-aldehyde dehydrogenase activity
def: "Catalysis of the reaction: betaine aldehyde + NAD+ + H2O = betaine + NADH + H+." [EC:1.2.1.8]
GO:0008803
name: bis(5'-nucleosyl)-tetraphosphatase (symmetrical) activity
def: "Catalysis of the reaction: P(1),P(4)-bis(5'-adenosyl) tetraphosphate + H(2)O = 2 ADP + 2 H(+)." [EC:3.6.1.41, RHEA:24255]
GO:0008804
name: carbamate kinase activity
def: "Catalysis of the reaction: ATP + NH3 + CO2 = ADP + carbamoyl phosphate." [EC:2.7.2.2]
GO:0008805
name: carbon-monoxide oxygenase activity
def: "Catalysis of the reaction: CO + H2O + ferrocytochrome b-561 = CO2 + 2 H+ + 2 ferricytochrome b-561." [EC:1.2.2.4]
GO:0008806
name: carboxymethylenebutenolidase activity
def: "Catalysis of the reaction: 4-carboxymethylenebut-2-en-4-olide + H2O = 4 oxohex-2-enedioate." [EC:3.1.1.45]
GO:0008807
name: carboxyvinyl-carboxyphosphonate phosphorylmutase activity
def: "Catalysis of the reaction: 1-carboxyvinyl carboxyphosphonate = 3-(hydrohydroxyphosphoryl)pyruvate + CO2." [EC:2.7.8.23]
GO:0008808
name: cardiolipin synthase activity
def: "Catalysis of the reaction: phosphatidylglycerol + phosphatidylglycerol = diphosphatidylglycerol (cardiolipin) + glycerol." [GOC:jl]
GO:0008809
name: carnitine racemase activity
def: "Catalysis of the reaction: D-carnitine = L-carnitine." [MetaCyc:CARNRACE-RXN]
GO:0008810
name: cellulase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-glucosidic linkages in cellulose, lichenin and cereal beta-D-glucans." [EC:3.2.1.4]
GO:0008811
name: chloramphenicol O-acetyltransferase activity
def: "Catalysis of the reaction: chloramphenicol + acetyl-CoA = chloramphenicol 3-acetate + CoA." [EC:2.3.1.28, RHEA:18424]
GO:0008812
name: choline dehydrogenase activity
def: "Catalysis of the reaction: A + choline = AH(2) + betaine aldehyde." [EC:1.1.99.1, RHEA:17436]
GO:0008813
name: chorismate lyase activity
def: "Catalysis of the reaction: chorismate = 4-hydroxybenzoate + pyruvate." [EC:4.1.3.40, RHEA:16508]
GO:0008814
name: citrate CoA-transferase activity
def: "Catalysis of the reaction: acetyl-CoA + citrate = acetate + (3S)-citryl-CoA." [EC:2.8.3.10]
GO:0008815
name: citrate (pro-3S)-lyase activity
def: "Catalysis of the reaction: citrate = acetate + oxaloacetate." [EC:4.1.3.6, RHEA:10763]
GO:0008816
name: citryl-CoA lyase activity
def: "Catalysis of the reaction: (3S)-citryl-CoA = acetyl-CoA + oxaloacetate." [EC:4.1.3.34]
GO:0008817
name: cob(I)yrinic acid a,c-diamide adenosyltransferase activity
def: "Catalysis of the reaction: ATP + cob(I)alamin + H2O = phosphate + diphosphate + adenosylcobalamin." [EC:2.5.1.17]
GO:0008818
name: cobalamin 5'-phosphate synthase activity
def: "Catalysis of the reaction: adenosylcobinamide-GDP + alpha-ribazole-5'-phosphate = adenosylcobalamin-5'-phosphate + GMP." [MetaCyc:COBALAMIN5PSYN-RXN]
GO:0008819
name: cobinamide kinase activity
def: "Catalysis of the reaction: cobinamide + ATP = cobinamide phosphate + ADP. This reaction is the phosphorylation of the hydroxyl group of the 1-amino-2-propanol residue of cobinamide, in the presence of ATP, to form cobinamide phosphate." [http://www.mblab.gla.ac.uk/, PMID:1655696]
GO:0008820
name: cobinamide phosphate guanylyltransferase activity
def: "Catalysis of the reaction: adenosylcobinamide phosphate + GTP + 2 H(+) = adenosylcobinamide-GDP + diphosphate." [EC:2.7.7.62, RHEA:22715]
GO:0008821
name: crossover junction endodeoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage at a junction such as a reciprocal single-stranded crossover between two homologous DNA duplexes (Holliday junction)." [EC:3.1.22.4]
GO:0008822
name: crotonobetaine/carnitine-CoA ligase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008823
name: cupric reductase activity
def: "Catalysis of the reaction: Cu+ + NAD+ + H+ = Cu2+ + NADH." [PMID:10510271]
GO:0008824
name: cyanate hydratase activity
def: "Catalysis of the reaction: cyanate + H2O = carbamate." [EC:4.2.1.104]
GO:0008825
name: cyclopropane-fatty-acyl-phospholipid synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phospholipid olefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid cyclopropane fatty acid." [EC:2.1.1.79]
GO:0008826
name: cysteine sulfinate desulfinase activity
def: "Catalysis of the reaction: 3-sulfinoalanine = L-alanine + sulfite." [MetaCyc:RXN0-279]
GO:0008827
name: cytochrome o ubiquinol oxidase activity
def: "Catalysis of the reaction: ubiquinol + O2 = ubiquinone + 2 H2O; cytochrome O is the electron acceptor." [GOC:mah, MetaCyc:CYT-UBIQUINOL-OXID-RXN]
GO:0008828
name: dATP pyrophosphohydrolase activity
def: "Catalysis of the reaction: deoxy-ATP + H2O = dAMP + diphosphate." [MetaCyc:RXN0-384]
GO:0008829
name: dCTP deaminase activity
def: "Catalysis of the reaction: dCTP + H2O = dUTP + NH3." [EC:3.5.4.13]
GO:0008830
name: dTDP-4-dehydrorhamnose 3,5-epimerase activity
def: "Catalysis of the reaction: dTDP-4-dehydro-6-deoxy-alpha-D-glucose = dTDP-4-dehydro-6-deoxy-L-mannose." [EC:5.1.3.13, RHEA:16972]
GO:0008831
name: dTDP-4-dehydrorhamnose reductase activity
def: "Catalysis of the reaction: dTDP-6-deoxy-L-mannose + NADP(+) = dTDP-4-dehydro-6-deoxy-L-mannose + H(+) + NADPH." [EC:1.1.1.133, RHEA:21799]
GO:0008832
name: dGTPase activity
def: "Catalysis of the reaction: dGTP + H(2)O = 2'-deoxyguanosine + 2 H(+) + triphosphate." [EC:3.1.5.1, RHEA:15196]
GO:0008833
name: deoxyribonuclease IV (phage-T4-induced) activity
def: "Catalysis of the endonucleolytic cleavage to 5'-phosphooligonucleotide end-products." [EC:3.1.21.2]
GO:0008834
name: di-trans,poly-cis-decaprenylcistransferase activity
def: "Catalysis of the reaction: di-trans-poly-cis-decaprenyl diphosphate + isopentenyl diphosphate = diphosphate + di-trans-poly-cis-undecaprenyl diphosphate." [EC:2.5.1.31]
GO:0008835
name: diaminohydroxyphosphoribosylaminopyrimidine deaminase activity
def: "Catalysis of the reaction: 2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine + H(2)O + H(+) = 5-amino-6-(5-phosphoribosylamino)uracil + NH(4)(+)." [EC:3.5.4.26, RHEA:21871]
GO:0008836
name: diaminopimelate decarboxylase activity
def: "Catalysis of the reaction: meso-2,6-diaminopimelate + H(+) = L-lysine + CO(2)." [EC:4.1.1.20, RHEA:15104]
GO:0008837
name: diaminopimelate epimerase activity
def: "Catalysis of the reaction: LL-2,6-diaminopimelate = meso-2,6-diaminopimelate." [EC:5.1.1.7, RHEA:15396]
GO:0008838
name: diaminopropionate ammonia-lyase activity
def: "Catalysis of the reaction: 2,3-diaminopropionate + H2O = pyruvate + 2 NH3." [EC:4.3.1.15]
GO:0008839
name: 4-hydroxy-tetrahydrodipicolinate reductase
def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydropyridine-2,6-dicarboxylate + NAD(P)+ + H2O = (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + NAD(P)H + H+." [EC:1.17.1.8]
GO:0008840
name: 4-hydroxy-tetrahydrodipicolinate synthase
def: "Catalysis of the reaction: pyruvate + L-aspartate-4-semialdehyde = (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + H2O." [EC:4.3.3.7, RHEA:14848]
GO:0008841
name: dihydrofolate synthase activity
def: "Catalysis of the reaction: ATP + dihydropterate + L-glutamate = ADP + phosphate + dihydrofolate." [EC:6.3.2.12]
GO:0008842
name: diphosphate-purine nucleoside kinase activity
def: "Catalysis of the reaction: diphosphate + a purine nucleoside = monophosphate + a purine mononucleotide." [EC:2.7.1.143]
GO:0008843
name: endochitinase activity
def: "Catalysis of the hydrolysis of nonterminal (1->4)-beta linkages of N-acetyl-D-glucosamine (GlcNAc) polymers of chitin and chitodextrins. Typically, endochitinases cleave randomly within the chitin chain." [EC:3.2.1.-, GOC:bf, GOC:kah, GOC:pde, PMID:11468293]
GO:0008845
name: endonuclease VIII activity
def: "OBSOLETE (was not defined before being made obsolete)." [PMID:12713806]
GO:0008846
name: endopeptidase La activity
def: "OBSOLETE. Catalysis of the hydrolysis of large proteins such as globin, casein and denatured serum albumin, in presence of ATP." [EC:3.4.21.53]
GO:0008847
name: Enterobacter ribonuclease activity
def: "Catalysis of the endonucleolytic cleavage to 3'-phosphomononucleotides and 3'-phosphooligonucleotides with 2',3'-cyclic phosphate intermediates." [EC:3.1.27.6]
GO:0008848
name: enterobactin synthetase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0008849
name: enterochelin esterase activity
def: "Catalysis of the reaction: enterobactin + 3 H2O = 3 N-23-dihydroxybenzoyl-L-serine + 3 H+." [MetaCyc:RXN0-1661, PMID:4565531]
GO:0008851
name: ethanolamine ammonia-lyase activity
def: "Catalysis of the reaction: ethanolamine = acetaldehyde + NH3." [EC:4.3.1.7]
GO:0008852
name: exodeoxyribonuclease I activity
def: "Catalysis of the degradation of single-stranded DNA. It acts progressively in a 3' to 5' direction, releasing 5'-phosphomononucleotides." [EC:3.1.11.1]
GO:0008853
name: exodeoxyribonuclease III activity
def: "Catalysis of the degradation of double-stranded DNA. It acts progressively in a 3' to 5' direction, releasing 5'-phosphomononucleotides." [EC:3.1.11.2]
GO:0008854
name: exodeoxyribonuclease V activity
def: "Catalysis of the exonucleolytic cleavage (in the presence of ATP) in either 5' to 3' or 3' to 5' direction to yield 5'-phosphooligonucleotides." [EC:3.1.11.5]
GO:0008855
name: exodeoxyribonuclease VII activity
def: "Catalysis of the exonucleolytic cleavage in either 5' to 3' or 3' to 5' direction to yield 5'-phosphomononucleotides." [EC:3.1.11.6]
GO:0008856
name: exodeoxyribonuclease X activity
def: "Catalysis of the endonucleolytic cleavage of supercoiled plasma DNA to linear DNA duplexes." [EC:3.1.22.5]
GO:0008859
name: exoribonuclease II activity
def: "Catalysis of the reaction: RNA + H2O = 5'-phosphomononucleotides. Cleaves RNA in the 3' to 5' direction." [EC:3.1.13.1, ISBN:0198547684]
GO:0008860
name: ferredoxin-NAD+ reductase activity
def: "Catalysis of the reaction: reduced ferredoxin + NAD+ = oxidized ferredoxin + NADH + H+." [EC:1.18.1.3]
GO:0008861
name: formate C-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + formate = CoA + pyruvate." [EC:2.3.1.54]
GO:0008863
name: formate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: formate + NAD(+) = CO(2) + NADH." [EC:1.2.1.2, RHEA:15988]
GO:0008864
name: formyltetrahydrofolate deformylase activity
def: "Catalysis of the reaction: 10-formyltetrahydrofolate + H(2)O = (6S)-5,6,7,8-tetrahydrofolate + formate + H(+)." [EC:3.5.1.10, RHEA:19836]
GO:0008865
name: fructokinase activity
def: "Catalysis of the reaction: ATP + D-fructose = ADP + D-fructose 6-phosphate." [EC:2.7.1.4]
GO:0008866
name: fructuronate reductase activity
def: "Catalysis of the reaction: D-mannonate + NAD(+) = D-fructuronate + H(+) + NADH." [EC:1.1.1.57, RHEA:15732]
GO:0008867
name: galactarate dehydratase activity
def: "Catalysis of the reaction: galactarate = 5-dehydro-4-deoxy-D-glucarate + H(2)O." [EC:4.2.1.42, RHEA:16008]
GO:0008868
name: galactitol-1-phosphate 5-dehydrogenase activity
def: "Catalysis of the reaction: galactitol-1-phosphate + NAD+ = L-tagatose 6-phosphate + NADH + H+." [EC:1.1.1.251]
GO:0008869
name: galactonate dehydratase activity
def: "Catalysis of the reaction: D-galactonate = 2-dehydro-3-deoxy-D-galactonate + H(2)O." [EC:4.2.1.6, RHEA:18652]
GO:0008870
name: galactoside O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + a beta-D-galactoside = CoA + a 6-acetyl-beta-D-galactoside." [EC:2.3.1.18]
GO:0008871
name: aminoglycoside 2''-nucleotidyltransferase activity
def: "Catalysis of the reaction: nucleoside triphosphate + aminoglycoside = diphosphate + 2''-nucleotidylaminoglycoside." [EC:2.7.7.46, GOC:cb]
GO:0008872
name: glucarate dehydratase activity
def: "Catalysis of the reaction: D-glucarate = 5-dehydro-4-deoxy-D-glucarate + H2O." [EC:4.2.1.40]
GO:0008873
name: gluconate 2-dehydrogenase activity
def: "Catalysis of the reaction: D-gluconate + NADP+ = 2-dehydro-D-gluconate + NADPH + H+." [EC:1.1.1.215]
GO:0008874
name: gluconate 5-dehydrogenase activity
def: "Catalysis of the reaction: D-gluconate + NADP+ = 5-dehydro-D-gluconate + NADPH + H+." [EC:1.1.1.69]
GO:0008875
name: gluconate dehydrogenase activity
def: "Catalysis of the reaction: D-gluconate + NADP+ = dehydro-D-gluconate + NADPH + H+." [EC:1.1.1.215, EC:1.1.1.69]
GO:0008876
name: quinoprotein glucose dehydrogenase activity
def: "Catalysis of the reaction: D-glucose + ubiquinone = D-glucono-1,5-lactone + ubiquinol." [EC:1.1.5.2]
GO:0008877
name: glucose-1-phosphatase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + H2O = D-glucose + phosphate." [EC:3.1.3.10]
GO:0008878
name: glucose-1-phosphate adenylyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + ATP = ADP-glucose + diphosphate." [EC:2.7.7.27, RHEA:12123]
GO:0008879
name: glucose-1-phosphate thymidylyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + dTTP = diphosphate + dTDP-glucose." [EC:2.7.7.24, RHEA:15228]
GO:0008880
name: glucuronate isomerase activity
def: "Catalysis of the reaction: D-glucuronate = D-fructuronate." [EC:5.3.1.12]
GO:0008881
name: glutamate racemase activity
def: "Catalysis of the reaction: L-glutamate = D-glutamate." [EC:5.1.1.3, RHEA:12816]
GO:0008882
name: [glutamate-ammonia-ligase] adenylyltransferase activity
def: "Catalysis of the reaction: ATP + [L-glutamate:ammonia ligase (ADP-forming)] = diphosphate + adenylyl-[L-glutamate:ammonia ligase (ADP-forming)]." [EC:2.7.7.42]
GO:0008883
name: glutamyl-tRNA reductase activity
def: "Catalysis of the reaction: (S)-4-amino-5-oxopentanoate + NADP(+) + tRNA(Glu) = L-glutamyl-tRNA(Glu) + H(+) + NADPH." [EC:1.2.1.70, RHEA:12347]
GO:0008884
name: glutathionylspermidine amidase activity
def: "Catalysis of the reaction: N1-(gamma-L-glutamyl-L-cysteinyl-glycyl)-spermidine + H2O = gamma-L-glutamyl-L-cysteinyl-glycine + spermidine." [EC:3.5.1.78]
GO:0008885
name: glutathionylspermidine synthase activity
def: "Catalysis of the reaction: gamma-L-glutamyl-L-cysteinyl-glycine + spermidine + ATP = N1-(gamma-L-glutamyl-L-cysteinyl-glycyl)-spermidine + ADP + phosphate." [EC:6.3.1.8]
GO:0008886
name: glyceraldehyde-3-phosphate dehydrogenase (NADP+) (non-phosphorylating) activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + H(2)O + NADP(+) = 3-phospho-D-glycerate + 2 H(+) + NADPH." [EC:1.2.1.9, RHEA:14672]
GO:0008887
name: glycerate kinase activity
def: "Catalysis of the reaction: D-glycerate + ATP = 3-phospho-D-glycerate + ADP + 2 H(+)." [EC:2.7.1.31, RHEA:23519]
GO:0008888
name: glycerol dehydrogenase [NAD+] activity
def: "Catalysis of the reaction: glycerol + NAD(+) = glycerone + H(+) + NADH." [EC:1.1.1.6, RHEA:13772]
GO:0008889
name: glycerophosphodiester phosphodiesterase activity
def: "Catalysis of the reaction: a glycerophosphodiester + H2O = an alcohol + sn-glycerol 3-phosphate." [EC:3.1.4.46]
GO:0008890
name: glycine C-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + glycine = L-2-amino-3-oxobutanoate + CoA + H(+)." [EC:2.3.1.29, RHEA:20739]
GO:0008891
name: glycolate oxidase activity
def: "Catalysis of the reaction: glycolate + O2 = glyoxylate + hydrogen peroxide." [EC:1.1.3.15]
GO:0008892
name: guanine deaminase activity
def: "Catalysis of the reaction: guanine + H2O = xanthine + NH3." [EC:3.5.4.3]
GO:0008893
name: guanosine-3',5'-bis(diphosphate) 3'-diphosphatase activity
def: "Catalysis of the reaction: guanosine 3',5'-bis(diphosphate) + H2O = guanosine 5'-diphosphate + diphosphate." [EC:3.1.7.2]
GO:0008894
name: guanosine-5'-triphosphate,3'-diphosphate diphosphatase activity
def: "Catalysis of the reaction: guanosine 5'-triphosphate,3'-diphosphate + H2O = guanosine 5'-diphosphate,3'-diphosphate + phosphate." [EC:3.6.1.40]
GO:0008897
name: holo-[acyl-carrier-protein] synthase activity
def: "Catalysis of the reaction: CoA + substrate-serine = adenosine 3',5'-bisphosphate + substrate-serine-4'-phosphopantetheine. The transfer of the 4'-phosphopantetheine (Ppant) co-factor from coenzyme A to the hydroxyl side chain of the serine residue of acyl- or peptidyl-carrier protein (ACP or PCP) to convert them from the apo to the holo form." [EC:2.7.8.7, PMID:10320345, PMID:11867633, PMID:8939709]
GO:0008898
name: S-adenosylmethionine-homocysteine S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + L-homocysteine = S-adenosyl-L-homocysteine + L-methionine." [EC:2.1.1.10, GOC:BHF, GOC:dph]
GO:0008899
name: homoserine O-succinyltransferase activity
def: "Catalysis of the reaction: L-homoserine + succinyl-CoA = O-succinyl-L-homoserine + CoA." [EC:2.3.1.46, RHEA:22011]
GO:0008900
name: hydrogen:potassium-exchanging ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + H+(in) + K+(out) = ADP + phosphate + H+(out) + K+(in)." [EC:3.6.3.10]
GO:0008901
name: ferredoxin hydrogenase activity
def: "Catalysis of the reaction: 2 reduced ferredoxin + 2 H+ = 2 oxidized ferredoxin + H2." [EC:1.12.7.2]
GO:0008902
name: hydroxymethylpyrimidine kinase activity
def: "Catalysis of the reaction: 4-amino-5-hydroxymethyl-2-methylpyrimidine + ATP = 4-amino-2-methyl-5-phosphomethylpyrimidine + ADP + 2 H(+)." [EC:2.7.1.49, RHEA:23099]
GO:0008903
name: hydroxypyruvate isomerase activity
def: "Catalysis of the reaction: 3-hydroxypyruvate = 2-hydroxy-3-oxopropanoate." [EC:5.3.1.22, RHEA:11955]
GO:0008904
name: hygromycin-B 7''-O-phosphotransferase activity
def: "Catalysis of the reaction: ATP + hygromycin B = 7''-O-phosphohygromycin B + ADP + 2 H(+)." [EC:2.7.1.119, RHEA:23391]
GO:0008905
name: mannose-phosphate guanylyltransferase activity
def: "Catalysis of the transfer of a phosphate group from GTP or GDP to a mannose molecule." [GOC:mah]
GO:0008906
name: inosine kinase activity
def: "Catalysis of the reaction: ATP + inosine = ADP + IMP." [EC:2.7.1.73]
GO:0008907
name: integrase activity
def: "Catalysis of the integration of one segment of DNA into another." [GOC:mah]
GO:0008908
name: isochorismatase activity
def: "Catalysis of the reaction: H(2)O + isochorismate = 2,3-dihydroxy-2,3-dihydrobenzoate + pyruvate." [EC:3.3.2.1, RHEA:11115]
GO:0008909
name: isochorismate synthase activity
def: "Catalysis of the reaction: chorismate = isochorismate." [EC:5.4.4.2, RHEA:18988]
GO:0008910
name: kanamycin kinase activity
def: "Catalysis of the reaction: ATP + kanamycin = ADP + 2 H(+) + kanamycin 3'-phosphate." [EC:2.7.1.95, RHEA:24259]
GO:0008911
name: lactaldehyde dehydrogenase activity
def: "Catalysis of the reaction: (S)-lactaldehyde + NAD+ + H2O = (S)-lactate + NADH + H+." [EC:1.2.1.22]
GO:0008912
name: lactaldehyde reductase activity
def: "Catalysis of the reaction: propane-1,2-diol + NAD+ = lactaldehyde + NADH + H+." [EC:1.1.1.77]
GO:0008913
name: lauroyltransferase activity
def: "Catalysis of the transfer of a lauroyl (dodecanoyl) group from one compound to another." [GOC:jl, ISBN:0198506732]
GO:0008914
name: leucyltransferase activity
def: "Catalysis of the reaction: L-leucyl-tRNA + protein = tRNA + L-leucyl-protein." [EC:2.3.2.6]
GO:0008915
name: lipid-A-disaccharide synthase activity
def: "Catalysis of the reaction: 2,3-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl 1-phosphate + UDP-2,3-bis(3-hydroxytetradecanoyl)-D-glucosamine = 2,3-bis(3-hydroxytetradecanoyl)-D-glucosaminyl-(1->6)-beta-D-2,3-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl 1-phosphate + H(+) + UDP." [EC:2.4.1.182, RHEA:22671]
GO:0008917
name: lipopolysaccharide N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + lipopolysaccharide = UDP + N-acetyl-D-glucosaminyl-lipopolysaccharide." [EC:2.4.1.56, GOC:mr, KEGG:00540]
GO:0008918
name: lipopolysaccharide 3-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + lipopolysaccharide = UDP + 1,3 alpha-D-galactosyl-lipopolysaccharide." [EC:2.4.1.44, GOC:mr, KEGG:00540]
GO:0008919
name: lipopolysaccharide glucosyltransferase I activity
def: "Catalysis of the reaction: UDP-glucose + lipopolysaccharide = UDP + D-glucosyl-lipopolysaccharide." [EC:2.4.1.58, GOC:mr, KEGG:00540]
GO:0008920
name: lipopolysaccharide heptosyltransferase activity
def: "Catalysis of the reaction: a lipopolysaccharide + ADP-L-glycero-beta-D-manno-heptose = a heptosylated lipopolysaccharide + ADP + H+." [MetaCyc:RXN0-5061, MetaCyc:RXN0-5122, MetaCyc:RXN0-5127]
GO:0008921
name: lipopolysaccharide-1,6-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + lipopolysaccharide = UDP + 1,6 alpha-D-galactosyl-lipopolysaccharide." [GOC:ai]
GO:0008922
name: long-chain fatty acid [acyl-carrier-protein] ligase activity
def: "Catalysis of the reaction: ATP + an acid + [acyl-carrier protein] = AMP + diphosphate + acyl-[acyl-carrier protein]. A long-chain fatty acid is fatty acid with a chain length between C13 and C22." [CHEBI:15904, EC:6.2.1.20]
GO:0008923
name: lysine decarboxylase activity
def: "Catalysis of the reaction: L-lysine + H(+) = cadaverine + CO(2)." [EC:4.1.1.18, RHEA:22355]
GO:0008924
name: malate dehydrogenase (quinone) activity
def: "Catalysis of the reaction: (S)-malate + a quinone = oxaloacetate + a quinol." [MetaCyc:MALATE-DEHYDROGENASE-ACCEPTOR-RXN]
GO:0008925
name: maltose O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + maltose = CoA + acetyl-maltose." [EC:2.3.1.79]
GO:0008926
name: mannitol-1-phosphate 5-dehydrogenase activity
def: "Catalysis of the reaction: D-mannitol 1-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+." [EC:1.1.1.17]
GO:0008927
name: mannonate dehydratase activity
def: "Catalysis of the reaction: D-mannonate = 2-dehydro-3-deoxy-D-gluconate + H(2)O." [EC:4.2.1.8, RHEA:20100]
GO:0008928
name: mannose-1-phosphate guanylyltransferase (GDP) activity
def: "Catalysis of the reaction: alpha-D-mannose 1-phosphate + GDP + H(+) = GDP-alpha-D-mannose + phosphate." [EC:2.7.7.22, RHEA:12908]
GO:0008929
name: methylglyoxal synthase activity
def: "Catalysis of the reaction: glycerone phosphate = methylglyoxal + phosphate." [EC:4.2.3.3, RHEA:17940]
GO:0008930
name: methylthioadenosine nucleosidase activity
def: "Catalysis of the reaction: methylthioadenosine + H2O = adenine + 5-methylthio-D-ribose." [EC:3.2.2.16]
GO:0008931
name: murein DD-endopeptidase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008932
name: lytic endotransglycosylase activity
def: "Catalysis of the specific cleavage of the beta-(1->4) glycosidic linkage between N-acetylmuramyl and N-acetylglucosaminyl residues in peptidoglycan, with the concomitant formation of 1,6-anhydro-N-acetylmuramyl residues. Acts on linkages within peptidoglycan chains (i.e. not at the ends) to produce shorter strands with 1,6-anhydromuramic acid ends." [PMID:10964424, PMID:9642199]
GO:0008933
name: lytic transglycosylase activity
def: "Catalysis of the specific cleavage of the beta-(1->4) glycosidic linkage between N-acetylmuramyl and N-acetylglucosaminyl residues in peptidoglycan, with the concomitant formation of 1,6-anhydro-N-acetylmuramyl residues." [PMID:10964424]
GO:0008934
name: inositol monophosphate 1-phosphatase activity
def: "Catalysis of the reaction: myo-inositol 1-phosphate + H2O = myo-inositol + phosphate." [EC:3.1.3.25]
GO:0008935
name: 1,4-dihydroxy-2-naphthoyl-CoA synthase activity
def: "Catalysis of the reaction: O-succinylbenzoyl-CoA = 1,4-dihydroxy-2-naphthoate + CoA." [EC:4.1.3.36]
GO:0008936
name: nicotinamidase activity
def: "Catalysis of the reaction: nicotinamide + H2O = nicotinate + NH3." [EC:3.5.1.19]
GO:0008937
name: ferredoxin-NAD(P) reductase activity
def: "Catalysis of the reaction: reduced ferredoxin + NAD(P)+ = oxidized ferredoxin + NAD(P)H + H+." [EC:1.18.1.-]
GO:0008938
name: nicotinate N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + nicotinate = N-methylnicotinate + S-adenosyl-L-homocysteine." [EC:2.1.1.7, RHEA:20244]
GO:0008939
name: nicotinate-nucleotide-dimethylbenzimidazole phosphoribosyltransferase activity
def: "Catalysis of the reaction: 5,6-dimethylbenzimidazole + nicotinate D-ribonucleotide = alpha-ribazole 5'-phosphate + H(+) + nicotinate." [EC:2.4.2.21, RHEA:11199]
GO:0008940
name: nitrate reductase activity
def: "Catalysis of the reaction: nitrite + acceptor = nitrate + reduced acceptor." [EC:1.7.99.4]
GO:0008941
name: nitric oxide dioxygenase activity
def: "Catalysis of the reaction: 2 NO + 2 O2 + NADPH + H+ = 2 nitrate + NADP+." [EC:1.14.12.17]
GO:0008942
name: nitrite reductase [NAD(P)H] activity
def: "Catalysis of the reaction: ammonium hydroxide + 3 NAD(P)+ + H2O = nitrite + 3 NAD(P)H + 3 H+." [EC:1.7.1.4]
GO:0008943
name: glyceraldehyde-3-phosphate dehydrogenase activity
def: "OBSOLETE. Catalysis of the reactions: glyceraldehyde 3-phosphate + phosphate + NAD(P)+ = 3-phospho-D-glyceroyl-phosphate + NAD(P)H + H+, and glyceraldehyde 3-phosphate + H2O + NAD(P)+ = 3-phospho-D-glycerate + NAD(P)H + H+." [EC:1.2.1.12, EC:1.2.1.13, EC:1.2.1.9]
GO:0008944
name: oligopeptidase A activity
def: "OBSOLETE. Catalysis of the hydrolysis of oligopeptides, with broad specificity. Gly or Ala commonly occur as P1 or P1' residues, but more distant residues are also important, as is shown by the fact that Z-Gly-Pro-Gly-Gly-Pro-Ala is cleaved at the Gly-Gly bond, but not Z-(Gly)5." [EC:3.4.24.70]
GO:0008945
name: oligopeptidase B activity
def: "OBSOLETE. Catalysis of the hydrolysis of Arg-Xaa and Lys-Xaa bonds in oligopeptides, even when P1' residue is proline." [EC:3.4.21.83]
GO:0008946
name: oligonucleotidase activity
def: "Catalysis of the exonucleolytic cleavage of oligonucleotides to yield nucleoside 5'-phosphates." [EC:3.1.13.3]
GO:0008947
name: omptin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Xaa-Yaa in which both Xaa and Yaa are Arg or Lys." [EC:3.4.21.87]
GO:0008948
name: oxaloacetate decarboxylase activity
def: "Catalysis of the reaction: oxaloacetate = pyruvate + CO2." [EC:1.1.1.38, EC:1.1.1.40, EC:4.1.1.3]
GO:0008949
name: oxalyl-CoA decarboxylase activity
def: "Catalysis of the reaction: H(+) + oxalyl-CoA = CO(2) + formyl-CoA." [EC:4.1.1.8, RHEA:19336]
GO:0008950
name: p-aminobenzoate synthetase
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:krc]
GO:0008951
name: palmitoleoyl [acyl-carrier-protein]-dependent acyltransferase activity
def: "Catalysis of the reaction: a palmitoleoyl-[acyl-carrier protein] + alpha-KDO-(2->4)-alpha-KDO-(2->6)-lipid IVA = KDO2-(palmitoleoyl)-lipid IVA + a holo-[acyl-carrier protein]." [MetaCyc:PALMITOTRANS-RXN]
GO:0008953
name: penicillin amidase activity
def: "Catalysis of the reaction: penicillin + H2O = a carboxylate + 6-aminopenicillanate." [EC:3.5.1.11]
GO:0008954
name: peptidoglycan synthetase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008955
name: peptidoglycan glycosyltransferase activity
def: "Catalysis of the reaction: [GlcNAc-(1,4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n)-diphosphoundecaprenol + GlcNAc-(1,4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol = [GlcNAc-(1,4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n+1)-diphosphoundecaprenol + undecaprenyl diphosphate." [EC:2.4.1.129]
GO:0008956
name: peptidyl-dipeptidase Dcp activity
def: "OBSOLETE. Catalysis of the hydrolysis of unblocked, C-terminal dipeptides from oligopeptides, with broad specificity. Does not hydrolyze bonds in which P1' is Pro, or both P1 and P1' are Gly." [EC:3.4.15.5]
GO:0008957
name: phenylacetaldehyde dehydrogenase activity
def: "Catalysis of the reaction: phenylacetaldehyde + NAD+ + H2O = phenylacetate + NADH + H+." [EC:1.2.1.39]
GO:0008959
name: phosphate acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + phosphate = CoA + acetyl phosphate." [EC:2.3.1.8]
GO:0008960
name: phosphatidylglycerol-membrane-oligosaccharide glycerophosphotransferase activity
def: "Catalysis of the reaction: phosphatidylglycerol + membrane-derived-oligosaccharide D-glucose = 1,2-diacyl-sn-glycerol + membrane-derived-oligosaccharide 6-(glycerophospho)-D-glucose." [EC:2.7.8.20]
GO:0008961
name: phosphatidylglycerol-prolipoprotein diacylglyceryl transferase activity
def: "Catalysis of the transfer of the diacylglyceryl group from phosphatidylglycerol to the sulfhydryl group of the prospective N-terminal cysteine residue in an unmodified prolipoprotein." [PMID:8051048]
GO:0008962
name: phosphatidylglycerophosphatase activity
def: "Catalysis of the reaction: phosphatidylglycerophosphate + H2O = phosphatidylglycerol + phosphate." [EC:3.1.3.27]
GO:0008963
name: phospho-N-acetylmuramoyl-pentapeptide-transferase activity
def: "Catalysis of the reaction: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine + undecaprenyl phosphate = UMP + N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine-diphosphoundecaprenol." [EC:2.7.8.13]
GO:0008964
name: phosphoenolpyruvate carboxylase activity
def: "Catalysis of the reaction: phosphate + oxaloacetate = phosphoenolpyruvate + HCO3-." [EC:4.1.1.31]
GO:0008965
name: phosphoenolpyruvate-protein phosphotransferase activity
def: "Catalysis of the reaction: phosphoenolpyruvate + protein L-histidine = pyruvate + protein N(pi)-phospho-L-histidine." [EC:2.7.3.9]
GO:0008966
name: phosphoglucosamine mutase activity
def: "Catalysis of the reaction: alpha-D-glucosamine 1-phosphate = D-glucosamine 6-phosphate." [EC:5.4.2.10, RHEA:23427]
GO:0008967
name: phosphoglycolate phosphatase activity
def: "Catalysis of the reaction: 2-phosphoglycolate + H(2)O = glycolate + phosphate." [EC:3.1.3.18, RHEA:14372]
GO:0008968
name: D-sedoheptulose 7-phosphate isomerase activity
def: "Catalysis of the reaction: D-sedoheptulose-7-phosphate = D-alpha,beta-D-heptose 7-phosphate." [MetaCyc:RXN0-4301, PMID:11279237, PMID:8631969]
GO:0008969
name: phosphohistidine phosphatase activity
def: "Catalysis of the reaction: phosphohistidine + H2O = histidine + phosphate." [GOC:mah]
GO:0008970
name: phosphatidylcholine 1-acylhydrolase activity
def: "Catalysis of the reaction: phosphatidylcholine + H2O = 2-acylglycerophosphocholine + a carboxylate." [EC:3.1.1.32]
GO:0008972
name: phosphomethylpyrimidine kinase activity
def: "Catalysis of the reaction: ATP + 4-amino-2-methyl-5-phosphomethylpyrimidine = ADP + 4-amino-2-methyl-5-diphosphomethylpyrimidine." [EC:2.7.4.7]
GO:0008973
name: phosphopentomutase activity
def: "Catalysis of the reaction: D-ribose 1-phosphate = D-ribose 5-phosphate." [EC:5.4.2.7]
GO:0008974
name: phosphoribulokinase activity
def: "Catalysis of the reaction: D-ribulose 5-phosphate + ATP = D-ribulose 1,5-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.19, RHEA:19368]
GO:0008975
name: pitrilysin activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Tyr16-Leu17 and Phe25-Tyr26 bonds of oxidized insulin B chain. Also acts on other substrates of Molecular weight less than 7 kDa such as insulin and glucagon." [EC:3.4.24.55]
GO:0008976
name: polyphosphate kinase activity
def: "Catalysis of the reaction: ATP + phosphate(n) = ADP + phosphate(n+1)." [EC:2.7.4.1]
GO:0008977
name: prephenate dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + prephenate = (4-hydroxyphenyl)pyruvate + CO(2) + NADH." [EC:1.3.1.12, RHEA:13872]
GO:0008978
name: prepilin peptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a Gly-Phe bond to release an N-terminal, basic peptide of 5-8 residues from type IV prepilin, and then N-methylates the new N-terminal amino group, the methyl donor being S-adenosyl-L-methionine." [EC:3.4.23.43]
GO:0008979
name: prophage integrase activity
def: "Catalysis of the integration of prophage DNA into a target DNA molecule, usually a bacterial chromosome, via a sequence-specific recombination event which involves the formation of an intasome, a DNA-protein-complex designed for site-specific recombination of the phage and host DNA." [GOC:jl]
GO:0008980
name: propionate kinase activity
def: "Catalysis of the reaction: ATP + propanoate = ADP + propanoyl phosphate." [EC:2.7.2.15]
GO:0008981
name: protease IV activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0008982
name: protein-N(PI)-phosphohistidine-sugar phosphotransferase activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sugar(out) = protein histidine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [EC:2.7.1.69, GOC:mtg_transport, http://www.ucs.mun.ca/~n55lrb/general_pts.html, ISBN:0815340729, TC:4.A.-.-.-]
GO:0008983
name: protein-glutamate O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein L-glutamate = S-adenosyl-L-homocysteine + protein L-glutamate 5-methyl ester; this reaction is the methylation of peptidyl-L-glutamate to form peptidyl-L-glutamate 5-methyl ester." [EC:2.1.1.80, RESID:AA0072]
GO:0008984
name: protein-glutamate methylesterase activity
def: "Catalysis of the reaction: protein L-glutamate O(5)-methyl ester + H2O = protein L-glutamate + methanol." [EC:3.1.1.61, RESID:AA0072]
GO:0008985
name: pyruvate dehydrogenase (cytochrome) activity
def: "OBSOLETE. Catalysis of the reaction: pyruvate + ferricytochrome b1 + H2O = CO2 + acetate + ferrocytochrome b1." [EC:1.2.2.2]
GO:0008986
name: pyruvate, water dikinase activity
def: "Catalysis of the reaction: ATP + H(2)O + pyruvate = AMP + 3 H(+) + phosphate + phosphoenolpyruvate." [EC:2.7.9.2, RHEA:11367]
GO:0008987
name: quinolinate synthetase A activity
def: "Catalysis of the reaction: iminoaspartate + dihydroxy-acetone-phosphate = quinolinate + 2 H2O + phosphate." [GOC:jl, MetaCyc:QUINOLINATE-SYNTHA-RXN]
GO:0008988
name: rRNA (adenine-N6-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N6-methyladenine." [EC:2.1.1.48]
GO:0008989
name: rRNA (guanine-N1-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N1-methylguanine." [EC:2.1.1.51]
GO:0008990
name: rRNA (guanine-N2-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N2-methylguanine." [EC:2.1.1.52]
GO:0008991
name: serine-type signal peptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a signal peptide from a protein precursor by a serine endopeptidase mechanism." [GOC:mah]
GO:0008992
name: repressor LexA activity
def: "OBSOLETE. Catalysis of the hydrolysis of Ala-Gly bond in repressor lexA." [EC:3.4.21.88]
GO:0008993
name: rhamnulokinase activity
def: "Catalysis of the reaction: ATP + L-rhamnulose = ADP + L-rhamnulose 1-phosphate." [EC:2.7.1.5]
GO:0008994
name: rhamnulose-1-phosphate aldolase activity
def: "Catalysis of the reaction: L-rhamnulose 1-phosphate = glycerone phosphate + (S)-lactaldehyde." [EC:4.1.2.19]
GO:0008995
name: ribonuclease E activity
def: "Catalysis of the cleavage of single-stranded RNA that is monophosphorylated at its 5'-end; cleavage occurs predominantly at 5 nucleotides from the 5'-end and in A + U-rich regions, and is blocked by the presence of a 5'-triphosphate group." [PMID:10722715, PMID:16854990]
GO:0008996
name: ribonuclease G activity
def: "Catalysis of the cleavage of single-stranded RNA that is monophosphorylated at its 5'-end; cleavage occurs predominantly at positions 5 and 6 nucleotides from the 5'-end and in A + U-rich regions, and is blocked by the presence of a 5'-triphosphate group." [PMID:10722715, PMID:16854990]
GO:0008997
name: ribonuclease R activity
def: "Catalysis of the reaction: RNA + H2O = 5'-phosphomononucleotides. Cleaves RNA in the 3' to 5' direction, leaving an undigested core of 3-5 nucleotides." [PMID:11948193]
GO:0008998
name: ribonucleoside-triphosphate reductase activity
def: "Catalysis of the reaction: 2'-deoxyribonucleoside triphosphate + thioredoxin disulfide + H2O = ribonucleoside triphosphate + thioredoxin." [EC:1.17.4.2]
GO:0008999
name: ribosomal-protein-alanine N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + ribosomal-protein L-alanine = CoA + ribosomal-protein N-acetyl-L-alanine." [EC:2.3.1.128]
GO:0009000
name: selenocysteine lyase activity
def: "Catalysis of the reaction: L-selenocysteine + reduced acceptor = hydrogen selenide + L-alanine + acceptor." [EC:4.4.1.16]
GO:0009001
name: serine O-acetyltransferase activity
def: "Catalysis of the reaction: L-serine + acetyl-CoA = O-acetyl-L-serine + CoA." [EC:2.3.1.30, RHEA:24563]
GO:0009002
name: serine-type D-Ala-D-Ala carboxypeptidase activity
def: "Catalysis of the reaction: (Ac)2-L-Lys-D-alanyl-D-alanine + H2O = (Ac)2-L-Lys-D-alanine + D-alanine." [EC:3.4.16.4]
GO:0009003
name: signal peptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a signal peptide from a protein precursor." [GOC:mah]
GO:0009004
name: signal peptidase I activity
def: "OBSOLETE. Catalysis of the cleavage of N-terminal leader sequences from secreted and periplasmic proteins precursor." [EC:3.4.21.89]
GO:0009005
name: signal peptidase II activity
def: "OBSOLETE. Catalysis of the cleavage of N-terminal leader sequences from membrane prolipoproteins. Hydrolyzes the terminal bond of Xaa-Xbb-Xbb-Cys, in which Xaa is hydrophobic (preferably Leu), Xbb is often Ser or Ala, Xcc is often Gly or Ala, and the Cys is alkylated on sulfur with a diacylglyceryl group." [EC:3.4.23.36]
GO:0009006
name: siroheme synthase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0009007
name: site-specific DNA-methyltransferase (adenine-specific) activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + DNA adenine = S-adenosyl-L-homocysteine + DNA 6-methylaminopurine." [EC:2.1.1.72]
GO:0009008
name: DNA-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to a DNA molecule." [GOC:jl, ISBN:0198506732]
GO:0009009
name: site-specific recombinase activity
def: "Catalysis of the formation of new phosphodiester bonds between a pair of short, unique DNA target sequences." [GOC:elh]
GO:0009010
name: sorbitol-6-phosphate 2-dehydrogenase activity
def: "Catalysis of the reaction: D-sorbitol 6-phosphate + NAD+ = D-fructose 6-phosphate + NADH + H+." [EC:1.1.1.140]
GO:0009011
name: starch synthase activity
def: "Catalysis of the reaction: ADP-glucose + (1,4)-alpha-D-glucosyl(n) = ADP + (1,4)-alpha-D-glucosyl(n+1)." [EC:2.4.1.21]
GO:0009012
name: aminoglycoside 3''-adenylyltransferase activity
def: "Catalysis of the reaction: ATP + streptomycin = 3''-adenylylstreptomycin + diphosphate + H(+)." [EC:2.7.7.47, RHEA:20248]
GO:0009013
name: succinate-semialdehyde dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: succinate semialdehyde + NAD(P)+ + H2O = succinate + NAD(P)H + H+." [EC:1.2.1.16]
GO:0009014
name: succinyl-diaminopimelate desuccinylase activity
def: "Catalysis of the reaction: N-succinyl-LL-2,6-diaminopimelate + H(2)O = LL-2,6-diaminopimelate + succinate." [EC:3.5.1.18, RHEA:22611]
GO:0009015
name: N-succinylarginine dihydrolase activity
def: "Catalysis of the reaction: N(2)-succinyl-L-arginine + 2 H(2)O + 2 H(+) = N(2)-succinyl-L-ornithine + CO(2) + 2 NH(4)(+)." [EC:3.5.3.23, RHEA:19536]
GO:0009016
name: succinyldiaminopimelate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + N-succinyl-LL-2,6-diaminopimelate = L-2-succinylamino-6-oxopimelate + L-glutamate." [EC:2.6.1.17, RHEA:11963]
GO:0009017
name: succinylglutamate desuccinylase activity
def: "Catalysis of the reaction: N-succinyl-L-glutamate + H(2)O = L-glutamate + succinate." [EC:3.5.1.96, RHEA:15172]
GO:0009018
name: sucrose phosphorylase activity
def: "Catalysis of the reaction: sucrose + phosphate = D-fructose + alpha-D-glucose 1-phosphate." [EC:2.4.1.7]
GO:0009019
name: tRNA (guanine-N1-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N1-methylguanine." [EC:2.1.1.31]
GO:0009020
name: tRNA (guanosine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing 2'-O-methylguanosine." [EC:2.1.1.34]
GO:0009022
name: tRNA nucleotidyltransferase activity
def: "Catalysis of the reaction: tRNA(n+1) + phosphate = tRNA(n) + a nucleoside diphosphate." [EC:2.7.7.56]
GO:0009023
name: tRNA sulfurtransferase
def: "OBSOLETE. Catalysis of the reaction: L-cysteine + activated-tRNA = L-serine + tRNA containing a thionucleotide." [EC:2.8.1.4, GOC:go_curators]
GO:0009024
name: tagatose-6-phosphate kinase activity
def: "Catalysis of the reaction: ATP + D-tagatose 6-phosphate = ADP + D-tagatose 1,6-bisphosphate." [EC:2.7.1.144]
GO:0009025
name: tagatose-bisphosphate aldolase activity
def: "Catalysis of the reaction: D-tagatose 1,6-diphosphate = D-glyceraldehyde 3-phosphate + glycerone phosphate." [EC:4.1.2.40, RHEA:22951]
GO:0009026
name: tagaturonate reductase activity
def: "Catalysis of the reaction: D-altronate + NAD(+) = D-tagaturonate + H(+) + NADH." [EC:1.1.1.58, RHEA:17816]
GO:0009027
name: tartrate dehydrogenase activity
def: "Catalysis of the reaction: tartrate + NAD+ = oxaloglycolate + NADH + H+." [EC:1.1.1.93]
GO:0009028
name: tartronate-semialdehyde synthase activity
def: "Catalysis of the reaction: 2 glyoxylate + H(+) = 2-hydroxy-3-oxopropanoate + CO(2)." [EC:4.1.1.47, RHEA:10139]
GO:0009029
name: tetraacyldisaccharide 4'-kinase activity
def: "Catalysis of the reaction: 2,3-bis(3-hydroxytetradecanoyl)-D-glucosaminyl-(1->6)-beta-D-2,3-bis(3-hydroxytetradecanoyl)-beta-D-glucosaminyl 1-phosphate + ATP = ADP + 2 H(+) + lipid IV(a)." [EC:2.7.1.130, RHEA:20703]
GO:0009030
name: thiamine-phosphate kinase activity
def: "Catalysis of the reaction: ATP + thiamine phosphate = ADP + H(+) + thiamine diphosphate." [EC:2.7.4.16, RHEA:15916]
GO:0009032
name: thymidine phosphorylase activity
def: "Catalysis of the reaction: thymidine + phosphate = thymine + 2-deoxy-D-ribose 1-phosphate." [EC:2.4.2.4]
GO:0009033
name: trimethylamine-N-oxide reductase activity
def: "Catalysis of the reaction: NADH + H+ + trimethylamine-N-oxide = NAD+ + trimethylamine + H2O." [EC:1.6.6.9]
GO:0009034
name: tryptophanase activity
def: "Catalysis of the reaction: L-tryptophan + H(2)O = indole + NH(4)(+) + pyruvate." [EC:4.1.99.1, RHEA:19556]
GO:0009035
name: Type I site-specific deoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of DNA to give random double-stranded fragments with terminal 5' or 3' protrusions; ATP is simultaneously hydrolyzed. Cleavage is dependent on the presence in the DNA of a specific recognition site. Cleavage may occur hundreds or thousands of base pairs away from the recognition site due to translocation of DNA." [EC:3.1.21.3, PMID:15788748]
GO:0009036
name: Type II site-specific deoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of DNA to give specific double-stranded fragments with terminal 5'-phosphates and 3' hydroxyls. Cleavage is dependent on the presence in the DNA of a specific recognition site; cleavage occurs at or very near this recognition site." [EC:3.1.21.4, PMID:12654995]
GO:0009037
name: tyrosine-based site-specific recombinase activity
def: "Catalysis of the formation of new phosphodiester bonds between a pair of short, unique DNA target sequences; occurs through a phosphotyrosyl intermediate in which the target sequence is first cleaved by the nucleophilic attack by a tyrosine in the active site." [GOC:elh, PMID:11090626]
GO:0009038
name: undecaprenol kinase activity
def: "Catalysis of the reaction: ATP + undecaprenol = ADP + undecaprenyl phosphate." [EC:2.7.1.66, RHEA:23755]
GO:0009039
name: urease activity
def: "Catalysis of the reaction: urea + H2O = CO2 + 2 NH3." [EC:3.5.1.5]
GO:0009040
name: ureidoglycolate dehydrogenase activity
def: "Catalysis of the reaction: (S)-ureidoglycolate + NAD(P)+ = oxalureate + NAD(P)H + H+." [EC:1.1.1.154, PMID:23284870]
GO:0009041
name: uridylate kinase activity
def: "Catalysis of the reaction: ATP + (d)UMP = ADP + (d)UDP." [GOC:go_curators]
GO:0009042
name: valine-pyruvate transaminase activity
def: "Catalysis of the reaction: L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine." [EC:2.6.1.66, RHEA:22915]
GO:0009044
name: xylan 1,4-beta-xylosidase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-xylans so as to remove successive D-xylose residues from the non-reducing termini." [EC:3.2.1.37]
GO:0009045
name: xylose isomerase activity
def: "Catalysis of the reaction: D-xylose = D-xylulose." [EC:5.3.1.5]
GO:0009046
name: zinc D-Ala-D-Ala carboxypeptidase activity
def: "Catalysis of the cleavage of the D-alanyl-D-alanine bond in (Ac)2-L-lysyl-D-alanyl-D-alanine." [EC:3.4.17.14]
GO:0009049
name: aspartic-type signal peptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a signal peptide from a protein precursor by an aspartic endopeptidase mechanism." [GOC:mah]
GO:0009055
name: electron carrier activity
def: "Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient." [ISBN:0198506732]
GO:0009374
name: biotin binding
def: "Interacting selectively and non-covalently with biotin (cis-tetrahydro-2-oxothieno(3,4-d)imidazoline-4-valeric acid), the (+) enantiomer of which is very widely distributed in cells and serves as a carrier in a number of enzymatic beta-carboxylation reactions." [GOC:jl, ISBN:0198506732]
GO:0009377
name: HslUV protease activity
def: "OBSOLETE. Catalysis of the ATP-dependent hydrolysis of peptide bonds in substrates including E. coli SulA and misfolded proteins." [GOC:mah, PMID:10368141]
GO:0009378
name: four-way junction helicase activity
def: "Catalysis of the unwinding of the DNA helix of DNA containing four-way junctions, including Holliday junctions." [GOC:jl, PMID:9442895]
GO:0009381
name: excinuclease ABC activity
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acid at sites flanking regions of damaged DNA to which the Uvr ABC excinuclease complexes bind." [GOC:mah, PMID:15192705]
GO:0009383
name: rRNA (cytosine-C5-)-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to cytosine to form 5-methylcytosine in small subunit ribosomal RNA." [GOC:imk, PMID:10026269, PMID:18786544]
GO:0009384
name: N-acylmannosamine kinase activity
def: "Catalysis of the reaction: ATP + N-acyl-D-mannosamine = ADP + N-acyl-D-mannosamine 6-phosphate." [EC:2.7.1.60]
GO:0009385
name: N-acylmannosamine-6-phosphate 2-epimerase activity
def: "Catalysis of the reaction: N-acetyl-D-mannosamine-6-phosphate = N-acetyl-D-glucosamine-6-phosphate." [MetaCyc:NANE-RXN]
GO:0009388
name: antisense RNA
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0009389
name: dimethyl sulfoxide reductase activity
def: "Catalysis of the reaction: dimethyl sulfoxide + H+ = dimethyl sulfide + H2O." [UM-BBD_reactionID:r0207]
GO:0009392
name: N-acetyl-anhydromuramoyl-L-alanine amidase activity
def: "Catalysis of the reaction: GlcNAc-1,6-anhMurNAc-L-Ala-gamma-D-Glu-DAP-D-Ala + H2O glcNAc-1,6-anhMurNAc + L-Ala-gamma-D-Glu-DAP-D-Ala." [MetaCyc:RXN0-5225]
GO:0009400
name: receptor signaling protein serine/threonine phosphatase activity
def: "Conveys a signal from an upstream receptor or intracellular signal transducer by catalysis of the reaction: protein serine phosphate + H2O = protein serine + phosphate, and protein threonine phosphate + H2O = protein threonine + phosphate." [GOC:bf, GOC:mah]
GO:0009457
name: flavodoxin
def: "OBSOLETE. A group of small electron carriers, characteristic of anaerobic bacteria, photosynthetic bacteria, cyanobacteria and eukaryotic algae. Contain flavin mononucleotide." [GOC:kd]
GO:0009458
name: cytochrome
def: "OBSOLETE. A hemeprotein whose characteristic mode of action involves transfer of reducing equivalents associated with a reversible change in oxidation state of the prosthetic group. This redox change involves a single electron, reversible equilibrium between the Fe(II) and Fe(III) states of the central iron atom." [PMID:1655423]
GO:0009459
name: cytochrome a
def: "OBSOLETE. A cytochrome containing heme a." [GOC:kd]
GO:0009460
name: cytochrome b
def: "OBSOLETE. A cytochrome containing noncovalently bound protoheme (iron protoporphyrin IX; heme b)." [GOC:kd]
GO:0009461
name: cytochrome c
def: "OBSOLETE. A cytochrome containing heme bound to the protein by one or, more (commonly two) thioether bonds involving sulfhydryl groups of cysteine residues." [GOC:kd]
GO:0009462
name: cytochrome d
def: "OBSOLETE. A cytochrome in which the prosthetic group is a tetrapyrrolic chelate of iron in which the degree of conjugation of double bonds is less than in porphyrin. This definition would appear to include siroheme proteins (e.g. nitrite and sulfite reductases), but these are not cytochromes." [GOC:kd, PMID:1655423]
GO:0009463
name: cytochrome b/b6
def: "OBSOLETE. Diheme cytochrome b; cytochrome b has the hemes b(562) and b(566) and is a component of the mitochondrial respiratory chain complex III. Cytochrome b6 is a component of bc complex acting between photosystems II and I of photosynthesis." [ISBN:0198547684]
GO:0009464
name: cytochrome b5
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009465
name: soluble cytochrome b562
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009466
name: class I cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009467
name: monoheme class I cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009468
name: diheme class I cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009469
name: class II cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009470
name: class IIa cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009471
name: class III cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009472
name: cytochrome c3 (tetraheme)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009473
name: cytochrome c7 (triheme)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009474
name: nonaheme cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009475
name: high-molecular-weight cytochrome c (hexadecaheme)
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009476
name: class IV cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009477
name: cytochrome c1
def: "OBSOLETE. A cytochrome c that is an integral component of the mitochondrial respiratory complex III. Functions as an electron donor to cytochrome c." [PMID:1655423]
GO:0009478
name: cytochrome c554
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009479
name: cytochrome f
def: "OBSOLETE. A cytochrome c that is characteristic of green plants and transfers electrons to plastocyanin." [PMID:1655423]
GO:0009480
name: class IIb cytochrome c
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009481
name: aa3-type cytochrome c oxidase
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 4 ferrocytochrome c + O2 = 4 ferricytochrome c + 2 H2O." [EC:1.9.3.1]
GO:0009482
name: ba3-type cytochrome c oxidase
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 4 ferrocytochrome c + O2 = 4 ferricytochrome c + 2 H2O." [EC:1.9.3.1]
GO:0009483
name: caa3-type cytochrome c oxidase
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 4 ferrocytochrome c + O2 = 4 ferricytochrome c + 2 H2O." [EC:1.9.3.1]
GO:0009485
name: cbb3-type cytochrome c oxidase
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 4 ferrocytochrome c + O2 = 4 ferricytochrome c + 2 H2O." [EC:1.9.3.1]
GO:0009486
name: cytochrome bo3 ubiquinol oxidase activity
def: "Catalysis of the reaction: 2 ubiquinol + O2 + 4 H+ = 2 ubiquinone + 2 H2O + 4 H+ [periplasmic space]." [MetaCyc:RXN0-5268]
GO:0009487
name: glutaredoxin
def: "OBSOLETE. A small disulfide-containing redox protein that serves as a glutathione-disulfide oxidoreductase." [GOC:kd]
GO:0009488
name: amicyanin
def: "OBSOLETE. A copper-containing protein that acts as an electron carrier between methylamine dehydrogenase and cytochrome c." [PMID:1655423]
GO:0009489
name: rubredoxin
def: "OBSOLETE. A low molecular weight mononuclear iron protein involved in electron transfer, with an iron tetrahedrally coordinated by the sulfurs of four conserved cysteine residues." [GOC:kd]
GO:0009490
name: mononuclear iron electron carrier
def: "OBSOLETE. A mononuclear iron entity that serves as an electron acceptor and electron donor in an electron transport system." [GOC:ai]
GO:0009491
name: redox-active disulfide bond electron carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009492
name: 2Fe-2S electron transfer carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009493
name: adrenodoxin-type ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009494
name: chloroplast-type ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009495
name: thioredoxin-like 2Fe-2S ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009496
name: plastoquinol--plastocyanin reductase activity
def: "Catalysis of the reaction: 2 H(+)[side 1] + 2 oxidized plastocyanin + plastoquinol-1 = 2 H(+)[side 2] + 2 reduced plastocyanin + plastoquinone. This reaction involves the concomitant transfer of 2 H+ ions across a membrane." [EC:1.10.9.1]
GO:0009497
name: 3Fe-4S/4Fe-4S electron transfer carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009498
name: bacterial-type ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009499
name: monocluster bacterial-type ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009500
name: dicluster bacterial-type ferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0009540
name: zeaxanthin epoxidase [overall] activity
def: "Catalysis of the reactions: zeaxanthin + NAD(P)H + H+ + O2 = antheraxanthin + NAD(P)+ + H2O; and antheraxanthin + NAD(P)H + H+ + O2 = violaxanthin + NAD(P)+ + H2O." [EC:1.14.13.90]
GO:0009669
name: sucrose:monovalent cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sucrose(out) + monovalent cation(out) = sucrose(in) + monovalent cation(in)." [GOC:jy, TC:2.A.2.-.-, TC:2.A.2.4.1]
GO:0009670
name: triose-phosphate:phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: triose-phosphate(out) + phosphate(in) = triose-phosphate(in) + phosphate(out)." [GOC:bf, GOC:jl, GOC:mtg_transport, ISBN:0815340729, TC:2.A.7.-.-]
GO:0009671
name: nitrate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: nitrate(out) + H+(out) = nitrate(in) + H+(in)." [GOC:mah, PMID:10066586, PMID:1990981]
GO:0009672
name: auxin:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: auxin(out) + H+(out) = auxin(in) + H+(in)." [PMID:8688077]
GO:0009673
name: low-affinity phosphate transmembrane transporter activity
def: "Catalysis of the transfer of phosphate from one side of the membrane to the other. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [TC:2.A.20.-.-]
GO:0009674
name: potassium:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: K+(out) + Na+(out) = K+(in) + Na+(in)." [TC:2.A.38.3.1]
GO:0009675
name: high-affinity sulfate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + H+(out) = sulfate(in) + H+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mah]
GO:0009676
name: low-affinity sulfate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + H+(out) = sulfate(in) + H+(in). In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:mah, PMID:7568135]
GO:0009678
name: hydrogen-translocating pyrophosphatase activity
def: "Catalysis of the hydrolysis of pyrophosphate, which generates a proton motive force." [GOC:mtg_transport, ISBN:0815340729, TC:3.A.10.-.-]
GO:0009679
name: hexose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: hexose(out) + H+(out) = hexose(in) + H+(in)." [TC:2.A.1.-.-]
GO:0009702
name: L-arabinokinase activity
def: "Catalysis of the reaction: L-arabinose + ATP = beta-L-arabinose 1-phosphate + ADP + 2 H(+)." [EC:2.7.1.46, RHEA:20156]
GO:0009703
name: nitrate reductase (NADH) activity
def: "Catalysis of the reaction: nitrite + NAD+ + H2O = nitrate + NADH + H+." [EC:1.7.1.1]
GO:0009784
name: transmembrane receptor histidine kinase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-histidine = ADP + a protein-L-histidine phosphate." [GOC:lr, GOC:mah]
GO:0009815
name: 1-aminocyclopropane-1-carboxylate oxidase activity
def: "Catalysis of the reaction: 1-aminocyclopropane-1-carboxylate + L-ascorbate + O(2) = CO(2) + dehydroascorbate + ethylene + 2 H(2)O + hydrogen cyanide. Ethene is also known as ethylene." [EC:1.14.17.4, RHEA:23643]
GO:0009824
name: AMP dimethylallyltransferase activity
def: "Catalysis of the reaction: AMP + dimethylallyl diphosphate = N(6)-(dimethylallyl)adenosine 5'-phosphate + diphosphate." [EC:2.5.1.27, RHEA:15288]
GO:0009881
name: photoreceptor activity
def: "The function of absorbing and responding to incidental electromagnetic radiation, particularly visible light. The response may involve a change in conformation." [GOC:ai, GOC:go_curators]
GO:0009882
name: blue light photoreceptor activity
def: "The function of absorbing and responding to electromagnetic radiation with a wavelength of approximately 400-470nm. The response may involve a change in conformation." [GOC:tb]
GO:0009883
name: red or far-red light photoreceptor activity
def: "The function of absorbing and responding to electromagnetic radiation with a wavelength of approximately 660-730nm. The response may involve a change in conformation." [GOC:lr]
GO:0009884
name: cytokinin receptor activity
def: "Combining with a cytokinin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:lr, GOC:signaling]
GO:0009885
name: transmembrane histidine kinase cytokinin receptor activity
def: "Combining with a cytokinin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-histidine = ADP + a protein-L-histidine phosphate." [GOC:lr, GOC:mah]
GO:0009899
name: ent-kaurene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-kaur-16-ene + diphosphate." [EC:4.2.3.19, RHEA:22223]
GO:0009905
name: ent-copalyl diphosphate synthase activity
def: "Catalysis of the reaction: all-trans-geranylgeranyl diphosphate = ent-copalyl diphosphate." [EC:5.5.1.13, RHEA:14844]
GO:0009916
name: alternative oxidase activity
def: "Catalysis of the oxidation of ubiquinol by diverting electrons from the standard electron transfer chain, transferring them from ubiquinol to oxygen and generating water as the product." [ISBN:0943088399]
GO:0009917
name: sterol 5-alpha reductase activity
def: "Catalysis of the removal of a C-5 double bond in the B ring of a sterol." [ISBN:0943088399]
GO:0009918
name: sterol delta7 reductase activity
def: "Catalysis of the reaction: 5-dehydroepisterol = 24-methylenecholesterol." [ISBN:0943088399]
GO:0009922
name: fatty acid elongase activity
def: "Catalysis of the reaction: fatty acid (C-16 or longer) + 2-C = fatty acid (C-16 or longer + 2-C)." [GOC:tb]
GO:0009924
name: octadecanal decarbonylase activity
def: "Catalysis of the reaction: octadecanal = heptadecane + CO." [EC:4.1.99.5, GOC:tb]
GO:0009927
name: histidine phosphotransfer kinase activity
def: "Serves as a phospho-His intermediate enabling the transfer of phospho group between a hybrid kinase and a response regulator." [PMID:11842140]
GO:0009931
name: calcium-dependent protein serine/threonine kinase activity
def: "Catalysis of the reactions: ATP + a protein serine = ADP + protein serine phosphate; and ATP + a protein threonine = ADP + protein threonine phosphate. These reactions are dependent on the presence of calcium ions." [GOC:mah]
GO:0009936
name: expansin
def: "OBSOLETE. A group of proteins located within the cell walls of plants, both dicots and grasses, that play an essential role in loosening cell walls during cell growth. They are hydrophobic, non glycosylated proteins of about 30kDa." [ISBN:0198547684]
GO:0009940
name: amino-terminal vacuolar sorting propeptide binding
def: "Interacting selectively and non-covalently with an amino terminal propeptide, which functions as a sorting signal to sort away the soluble vacuolar protein from Golgi to lytic vacuole via clathrin-coated vesicles." [GOC:sm, PMID:10871276]
GO:0009973
name: adenylyl-sulfate reductase activity
def: "Catalysis of the reaction: AMP + sulfite + acceptor = adenylyl sulfate + reduced acceptor." [EC:1.8.99.2, PMID:5421934]
GO:0009974
name: zeinoxanthin epsilon hydroxylase activity
def: "Catalysis of the reaction: zeinoxanthin + NADPH + O2 + H+ = lutein + NADP+ + H2O. Adds a hydroxyl group to the epsilon ring of the alpha-carotene." [MetaCyc:RXN-5962, PMID:8837513]
GO:0009975
name: cyclase activity
def: "Catalysis of a ring closure reaction." [ISBN:0198547684]
GO:0009976
name: tocopherol cyclase activity
def: "Catalysis of the reaction: alkene group + alcohol group on same molecule = cyclic ether. Substrates are 2-methyl-6-phytyl-1,4- hydroquinone (forms delta-tocopherol) and 2,3-dimethyl-5-phytyl-1,4-hydroquinone (forms gamma-tocopherol)." [PMID:12213958]
GO:0009977
name: proton motive force dependent protein transmembrane transporter activity
def: "Catalysis of the transfer of proteins from one side of the membrane to the other. Transportation is dependent on pH gradient across the membrane." [PMID:11526245]
GO:0009978
name: allene oxide synthase activity
def: "Catalysis of the reaction: 13(S)-hydroperoxylinolenate = 12,13(S)-epoxylinolenate + H2O." [EC:4.2.1.92, MetaCyc:RXN1F-19, PMID:9778849]
GO:0009979
name: 16:0 monogalactosyldiacylglycerol desaturase activity
def: "Catalysis of the introduction of an omega-3 double bond into an unsaturated 16-carbon fatty acid in a monogalactosyldiacylglycerol molecule." [GOC:mah, MetaCyc:RXN-1728, MetaCyc:RXN-8304, MetaCyc:RXN-8307]
GO:0009980
name: glutamate carboxypeptidase activity
def: "OBSOLETE. Catalysis of the release of C-terminal glutamate residues from a wide range of N-acylating groups, including peptidyl, aminoacyl, benzoyl, benzyloxycarbonyl, folyl and pteroyl groups." [EC:3.4.17.11, MetaCyc:3.4.17.11-RXN]
GO:0009982
name: pseudouridine synthase activity
def: "Catalysis of the reaction: RNA uridine = RNA pseudouridine. Conversion of uridine in an RNA molecule to pseudouridine by rotation of the C1'-N-1 glycosidic bond of uridine in RNA to a C1'-C5." [EC:5.4.99.12, GOC:mah]
GO:0009983
name: tyrosine aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of N-terminal tyrosine from a peptide." [GOC:sm, ISBN:0198506732]
GO:0009984
name: adenylate forming enzyme activity
def: "OBSOLETE. Catalysis of the reaction: substrate + ATP = substrate-AMP + diphosphate." [PMID:12084835]
GO:0009985
name: dihydroflavonol(thiole) lyase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:tb]
GO:0010011
name: auxin binding
def: "Interacting selectively and non-covalently with auxin, plant hormones that regulate aspects of plant growth." [GOC:sm]
GO:0010012
name: steroid 22-alpha hydroxylase activity
def: "Catalysis of the reaction: 5-alpha-campestanaol + O2 = 6-deoxocathasterone + H2O." [GOC:tb]
GO:0010013
name: N-1-naphthylphthalamic acid binding
def: "Interacting selectively and non-covalently with N-1-naphthylphthalamic acid, an auxin transport inhibitor." [GOC:sm]
GO:0010174
name: nucleoside transmembrane transporter activity, against a concentration gradient
def: "Catalysis of the transfer of a nucleoside, from one side of a membrane to the other, up a concentration gradient." [GOC:tb]
GO:0010175
name: sphingosine transmembrane transporter activity
def: "Catalysis of the transfer of amino alcohol sphingosine from one side of the membrane to the other." [GOC:tb]
GO:0010176
name: homogentisate phytyltransferase activity
def: "Catalysis of the reaction: homogentisate + phytyl diphosphate + H+ = 2-methyl-6-phytyl-1,4-benzoquinone + CO2 + diphosphate. 2-methyl-6-phytyl-1,4-benzoquinone is also known as 2-methyl-6-phytylplastoquinol." [MetaCyc:RXN-2541, PMID:14512521]
GO:0010177
name: 2-(2'-methylthio)ethylmalate synthase activity
def: "Catalysis of the reaction: 2-oxo-4-methylthiobutanoate + acetyl-CoA + H2O = 2-(2'-methylthio)ethylmalic-acid + coenzyme A + H+." [MetaCyc:RXN-2202]
GO:0010178
name: IAA-amino acid conjugate hydrolase activity
def: "Catalysis of the cleavage of the amide bond between IAA (auxin) and the conjugated amino acid." [GOC:tb]
GO:0010179
name: IAA-Ala conjugate hydrolase activity
def: "Catalysis of the reaction: indole-3-acetyl-alanine + H2O = indole-3-acetate + L-alanine." [MetaCyc:RXN-2981]
GO:0010180
name: thioglucosidase binding
def: "Interacting selectively and non-covalently with the enzyme thioglucosidase." [GOC:tb]
GO:0010181
name: FMN binding
def: "Interacting selectively and non-covalently with FMN, flavin mononucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [GOC:tb]
GO:0010209
name: vacuolar sorting signal binding
def: "Interacting selectively and non-covalently with a vacuolar sorting signal, a specific peptide sequence that acts as a signal to localize the protein within the vacuole." [GOC:mah]
GO:0010210
name: IAA-Phe conjugate hydrolase activity
def: "Catalysis of the reaction: indole-3-acetyl-phenylalanine + H2O = indole-3-acetate + phenylalanine." [GOC:syr]
GO:0010211
name: IAA-Leu conjugate hydrolase activity
def: "Catalysis of the reaction: indole-3-acetyl-leucine + H2O = indole-3-acetate + L-leucine." [MetaCyc:RXN-2982]
GO:0010242
name: oxygen evolving activity
def: "Catalysis of the reaction: 2 H2O = O2 + 4 H+ + 4 e-. The evolution of oxygen from oxidizing water is carried out by the oxygen evolving complex in photosystem II of plants. P680+, the photochemically oxidized reaction-center chlorophyll of PSII, is a strong biological oxidant. The reduction potential of P680+ is more positive than that of water, and thus it can oxidize water to give O2 and H+ ions. The oxygen escapes as a gas while the H+ ions remain in solution inside the thylakoid vesicle." [GOC:kd, GOC:syr, PMID:17091926, PMID:7948862]
GO:0010276
name: phytol kinase activity
def: "Catalysis of the reaction: phytol + CTP = phytyl monophosphate + CDP + H+." [MetaCyc:RXN-7683]
GO:0010277
name: chlorophyllide a oxygenase [overall] activity
def: "Catalysis of the reactions: chlorophyllide a + O2 + NADPH + H+ = 7-hydroxychlorophyllide a + H2O + NADP+; and 7-hydroxychlorophyllide a + O2 + NADPH + H+ = chlorophyllide b + 2 H2O + NADP+." [EC:1.13.12.14, MetaCyc:RXN-7677]
GO:0010279
name: indole-3-acetic acid amido synthetase activity
def: "Catalysis of the reaction: indole-3-acetic acid + an amino acid = an indole-3-acetic acid amide conjugate." [PMID:15659623]
GO:0010280
name: UDP-L-rhamnose synthase activity
def: "Catalysis of the reaction: UDP-D-glucose + NADPH + H+ = UDP-L-rhamnose + NADP+ + H2O." [MetaCyc:RXN-5482, PMID:14701918]
GO:0010283
name: pinoresinol reductase activity
def: "Catalysis of the reaction: pinoresinol + NADPH + H+ = lariciresinol + NADP+." [PMID:10066819, PMID:7592828]
GO:0010284
name: lariciresinol reductase activity
def: "Catalysis of the reaction: lariciresinol + NADPH + H+ = secoisolariciresinol + NADP+." [PMID:10066819, PMID:7592828]
GO:0010285
name: L,L-diaminopimelate aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + LL-2,6-diaminopimelate = (S)-2,3,4,5-tetrahydrodipicolinate + L-glutamate + H(2)O + H(+)." [EC:2.6.1.83, RHEA:23991]
GO:0010290
name: chlorophyll catabolite transmembrane transporter activity
def: "Enables the directed movement of chlorophyll catabolites such as non-fluorescent chlorophyll catabolites (NCCs), from one side of the membrane to the other." [PMID:9681016]
GO:0010291
name: carotene beta-ring hydroxylase activity
def: "Catalysis of the reaction: a carotene + a reduced electron acceptor + O2 = C3-hydroxylated carotene + an oxidized electron acceptor + H2O. This is a general reaction to represent the C3 hydroxylation of the beta ring of a carotene." [MetaCyc:MONOMER-12386, PMID:16492736]
GO:0010292
name: GTP:GDP antiporter activity
def: "Catalysis of the reaction: GTP(out) + GDP(in) = GTP(in) + GDP(out)." [PMID:10514379, PMID:12553910, PMID:16553903]
GO:0010293
name: abscisic aldehyde oxidase activity
def: "Catalysis of the reaction: (+)-abscisic aldehyde + H(2)O + O(2) = abscisate + H(2)O(2) + H(+)." [EC:1.2.3.14, RHEA:20532]
GO:0010294
name: abscisic acid glucosyltransferase activity
def: "Catalysis of the reaction: (+)-abscisate + UDP-D-glucose = abscisic acid glucose ester + UDP." [DOI:10.1016/j.tetasy.2004.11.062]
GO:0010295
name: (+)-abscisic acid 8'-hydroxylase activity
def: "Catalysis of the reaction: (+)-abscisate + H(+) + NADPH + O(2) = (+)-8'-hydroxyabscisate + H(2)O + NADP(+)." [EC:1.14.13.93, RHEA:12900]
GO:0010296
name: prenylcysteine methylesterase activity
def: "Catalysis of the reaction: protein C-terminal S-farnesyl-L-cysteine methyl ester + H2O = protein C-terminal S-farnesyl-L-cysteine + methanol + H+." [MetaCyc:RXN-8409, PMID:16870359]
GO:0010297
name: heteropolysaccharide binding
def: "Interacting selectively and non-covalently with heteropolysaccharides. A heteropolysaccharide is a glycan composed of more than one type of monosaccharide residue." [PMID:16640603]
GO:0010298
name: dihydrocamalexic acid decarboxylase activity
def: "Catalysis of the reaction: dihydrocamalexic acid = camalexin + CO2 + H+." [MetaCyc:RXN-8275, PMID:16766671]
GO:0010301
name: xanthoxin dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + xanthoxin = (+)-abscisic aldehyde + H(+) + NADH." [EC:1.1.1.288, RHEA:12551]
GO:0010303
name: limit dextrinase activity
def: "Catalysis of the hydrolysis of (1,6)-alpha-D-glucosidic linkages in alpha- and beta-limit dextrins of amylopectin and glycogen, and in amylopectin and pullulan." [EC:3.2.1.142]
GO:0010307
name: acetylglutamate kinase regulator activity
def: "Modulates the enzyme activity of acetylglutamate kinase." [PMID:16377628]
GO:0010308
name: acireductone dioxygenase (Ni2+-requiring) activity
def: "Catalysis of the reaction: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + O(2) = 3-(methylthio)propanoate + CO + formate." [EC:1.13.11.53, RHEA:14164]
GO:0010309
name: acireductone dioxygenase [iron(II)-requiring] activity
def: "Catalysis of the reaction: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + O(2) = 4-methylthio-2-oxobutanoate + formate + H(+)." [EC:1.13.11.54, RHEA:24507]
GO:0010313
name: phytochrome binding
def: "Interacting selectively and non-covalently with phytochrome." [PMID:15486102]
GO:0010314
name: phosphatidylinositol-5-phosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-5-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 5' position." [GOC:bf, GOC:tair_curators]
GO:0010320
name: arginine/lysine endopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide linkages in oligopeptides or polypeptides by a reaction mechanism in which arginine or lysine residues act as nucleophiles." [GOC:tair_curators]
GO:0010326
name: methionine-oxo-acid transaminase activity
def: "Catalysis of the reaction: methionine + a 2-oxo acid = 2-oxo-4-methylthiobutanoate + an amino acid." [MetaCyc:RXN-2201, PMID:17056707]
GO:0010327
name: acetyl CoA:(Z)-3-hexen-1-ol acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + (Z)-3-hexen-1-ol = CoA + (Z)-3-hexen-1-yl acetate." [PMID:17163883]
GO:0010328
name: auxin influx transmembrane transporter activity
def: "Catalysis of the transfer of auxin, from one side of a membrane to the other, into a cell." [PMID:16839804]
GO:0010329
name: auxin efflux transmembrane transporter activity
def: "Catalysis of the transfer of auxin, from one side of a membrane to the other, out of a cell." [PMID:16839804]
GO:0010331
name: gibberellin binding
def: "Interacting selectively and non-covalently with gibberellins, plant hormones that regulate aspects of plant growth." [GOC:tair_curators]
GO:0010333
name: terpene synthase activity
def: "Catalysis of the formation of cyclic terpenes through the cyclization of linear terpenes (e.g. isopentenyl-PP, geranyl-PP, farnesyl-PP and geranylgeranyl-PP) containing varying numbers of isoprene units." [EC:4.2.3.-, GOC:tair_curators]
GO:0010334
name: sesquiterpene synthase activity
def: "Catalysis of the reaction: trans,trans-farnesyl diphosphate = a sesquiterpene + diphosphate. Sesquiterpenes are terpenes containing three isoprene units, i.e. 15 carbons." [EC:4.2.3.-, GOC:tair_curators]
GO:0010340
name: carboxyl-O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the carboxyl group of an acceptor molecule to form a methyl ester." [PMID:17220201]
GO:0010341
name: gibberellin carboxyl-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a gibberellin = S-adenosyl-L-homocysteine + a gibberellin methyl ester." [PMID:17220201]
GO:0010347
name: L-galactose-1-phosphate phosphatase activity
def: "Catalysis of the reaction: L-galactose-1-phosphate + H2O = L-galactose + phosphate." [PMID:15550539, PMID:16595667]
GO:0010348
name: lithium:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Li+(in) + H+(out) = Li+(out) + H+(in)." [PMID:17270011]
GO:0010349
name: L-galactose dehydrogenase activity
def: "Catalysis of the reaction: L-galactose + NAD+ = L-galactono-1,4-lactone + NADH + H+." [PMID:12047629]
GO:0010354
name: homogentisate prenyltransferase activity
def: "Catalysis of the transfer of a prenyl group from one compound (donor) to homogentisic acid." [PMID:16989822]
GO:0010355
name: homogentisate farnesyltransferase activity
def: "Catalysis of the reaction: homogentisic acid + farnesyl diphosphate = 2-methyl-6-farnesylplastoquinol." [PMID:16989822]
GO:0010356
name: homogentisate geranylgeranyltransferase activity
def: "Catalysis of the reaction: homogentisic acid + geranylgeranyl diphosphate = 2-methyl-6-geranylgeranylplastoquinol." [PMID:16989822]
GO:0010357
name: homogentisate solanesyltransferase activity
def: "Catalysis of the reaction: homogentisic acid + all-trans-nonaprenyl diphosphate + 3 H+ = 2-methyl-6-solanyl-1,4-benzoquinonone + CO2 + diphosphate. 2-methyl-6-solanyl-1,4-benzoquinonone is also known as 2-methyl-6-solanesylplastoquinol and all-trans-nonaprenyl diphosphate as solanesyl diphosphate." [PMID:16989822]
GO:0010385
name: double-stranded methylated DNA binding
def: "Interacting selectively and non-covalently with double-stranded methylated DNA. Methylation of cytosine or adenine in DNA is an important mechanism for establishing stable heritable epigenetic marks." [GOC:imk, PMID:17242155]
GO:0010420
name: polyprenyldihydroxybenzoate methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-polyprenyl-4,5-dihydroxybenzoate = S-adenosyl-L-homocysteine + 3-polyprenyl-4-hydroxy-5-methoxybenzoate + H+." [PMID:9628017]
GO:0010427
name: abscisic acid binding
def: "Interacting selectively and non-covalently with abscisic acid, plant hormones that regulate aspects of plant growth." [PMID:17347412]
GO:0010428
name: methyl-CpNpG binding
def: "Interacting selectively and non-covalently with a methylated cytosine/unspecified/guanine trinucleotide." [PMID:17239600]
GO:0010429
name: methyl-CpNpN binding
def: "Interacting selectively and non-covalently with a methylated cytosine/unspecified/unspecified trinucleotide." [PMID:17239600]
GO:0010435
name: 3-oxo-2-(2'-pentenyl)cyclopentane-1-octanoic acid CoA ligase activity
def: "Catalysis of the reaction: ATP + 3-oxo-2-(2'-pentenyl)-cyclopentane-1-octanoic acid + coenzyme A = AMP + diphosphate + 3-oxo-2-(2'-pentenyl)-cyclopentane-1-octanoyl-CoA + H+. 3-oxo-2-(2'-pentenyl)-cyclopentane-1-octanoic acid is also known as OPC-8:0." [PMID:16963437]
GO:0010436
name: carotenoid dioxygenase activity
def: "Catalysis of the oxidative cleavage of carotenoids." [PMID:16459333]
GO:0010437
name: 9,10 (9', 10')-carotenoid-cleaving dioxygenase activity
def: "Catalysis of the oxidative cleavage of carotenoids at the (9, 10) and/or (9', 10') double bond." [PMID:16459333]
GO:0010461
name: light-activated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens in response to a light stimulus." [GOC:dph, GOC:tb]
GO:0010465
name: nerve growth factor receptor activity
def: "Combining with nerve growth factor (NGF), to prevent apoptosis in neurons and promote nerve growth, or to initiate a change in cell activity." [GOC:dph, GOC:tb]
GO:0010471
name: GDP-galactose:mannose-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: GDP-L-galactose + alpha-D-mannose 1-phosphate = GDP-alpha-D-mannose + alpha-L-galactose-1-phosphate." [MetaCyc:RXN4FS-12, PMID:17485667]
GO:0010472
name: GDP-galactose:glucose-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: GDP-L-galactose + alpha-D-glucose 1-phosphate = alpha-L-galactose-1-phosphate + GDP-alpha-D-glucose." [MetaCyc:RXN4FS-13, PMID:17485667]
GO:0010473
name: GDP-galactose:myoinositol-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: GDP-L-galactose + myo-inositol 1-phosphate = alpha-L-galactose-1-phosphate + GDP-myoinositol." [PMID:17485667]
GO:0010474
name: glucose-1-phosphate guanylyltransferase (GDP) activity
def: "Catalysis of the reaction: GDP + D-glucose 1-phosphate = phosphate + GDP-glucose." [PMID:17462988]
GO:0010475
name: galactose-1-phosphate guanylyltransferase (GDP) activity
def: "Catalysis of the reaction: GDP + L-galactose 1-phosphate = phosphate + GDP-galactose." [PMID:17462988]
GO:0010484
name: H3 histone acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + histone H3 = CoA + acetyl-histone H3." [EC:2.3.1.48]
GO:0010485
name: H4 histone acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + histone H4 = CoA + acetyl-histone H4." [EC:2.3.1.48]
GO:0010486
name: manganese:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Mn2+(in) + H+(out) = Mn2+(out) + H+(in)." [PMID:17559518]
GO:0010487
name: thermospermine synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methioninamine + spermidine = S-methyl-5'-thioadenosine + thermospermine + H+." [EC:2.5.1.79, MetaCyc:RXN-11190, PMID:17560575]
GO:0010488
name: UDP-galactose:N-glycan beta-1,3-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + N-glycan = galactose-beta-1,3-N-glycan + UDP." [PMID:17630273]
GO:0010489
name: UDP-4-keto-6-deoxy-glucose-3,5-epimerase activity
def: "Catalysis of the reaction: UDP-4-keto-6-deoxyglucose = UDP-4-keto-rhamnose." [GOC:tair_curators, PMID:17190829]
GO:0010490
name: UDP-4-keto-rhamnose-4-keto-reductase activity
def: "Catalysis of the reaction: UDP-4-keto-rhamnose + NADPH = UDP-rhamnose + NADP+." [GOC:tair_curators, PMID:17190829]
GO:0010491
name: UTP:arabinose-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: alpha-L-arabinose 1-phosphate + UTP = UDP-L-arabinose + diphosphate." [PMID:17341835]
GO:0010521
name: telomerase inhibitor activity
def: "Stops, prevents or reduces the activity of a telomerase. Telomerase catalyzes the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes extension of the 3'- end of a DNA strand by one deoxynucleotide at a time using an internal RNA template that encodes the telomeric repeat sequence." [GOC:dph, GOC:krc, GOC:tb]
GO:0010538
name: Hsp27 protein regulator activity
def: "OBSOLETE. Modulates the activity of the Hsp27 molecular chaperone." [GOC:dph, GOC:tb, PMID:11546764]
GO:0010539
name: Hsp27 protein inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the Hsp27 molecular chaperone." [GOC:dph, GOC:tb]
GO:0010542
name: nitrate efflux transmembrane transporter activity
def: "Catalysis of the transfer of nitrate from the inside of the cell to the outside of the cell across a membrane." [GOC:mah]
GO:0010545
name: Hsp90 protein regulator activity
def: "OBSOLETE. Binds to and modulates the activity of the molecular chaperone Hsp90." [GOC:dph, GOC:tb, PMID:11146632]
GO:0010546
name: Hsp90 protein inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the Hsp90 molecular chaperone." [GOC:dph, GOC:tb, PMID:11146632]
GO:0010576
name: metalloenzyme regulator activity
def: "Modulates the activity of a metalloenzyme. A metalloenzyme is any enzyme that contains metal." [GOC:dph, GOC:tb]
GO:0010577
name: metalloenzyme activator activity
def: "Increases the activity of a metalloenzyme. A metalloenzyme is any enzyme that contains metal." [GOC:dph, GOC:tb]
GO:0010698
name: acetyltransferase activator activity
def: "Increases the activity of an aceyltransferase, an enzyme which catalyzes the transfer of an acetyl group to an acceptor molecule." [GOC:dph, GOC:jp, GOC:tb, PMID:18690240]
GO:0010736
name: serum response element binding
def: "Interacting selectively and non-covalently with the serum response element (SRE), a short sequence with dyad symmetry found in the promoters of some of the cellular immediate-early genes, regulated by serum." [GOC:BHF, GOC:dph, GOC:rl, GOC:tb]
GO:0010843
name: promoter binding
def: "OBSOLETE. Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for transcription factor complexes of the basal transcription machinery." [GOC:dph, GOC:tb, SO:0000167]
GO:0010844
name: recombination hotspot binding
def: "Interacting selectively and non-covalently with a region in a genome which promotes recombination." [GOC:dph, GOC:tb]
GO:0010851
name: cyclase regulator activity
def: "Modulates the activity of an enzyme that catalyzes a ring closure reaction." [GOC:dph, GOC:tb]
GO:0010852
name: cyclase inhibitor activity
def: "Decreases the activity of an enzyme that catalyzes a ring closure reaction." [GOC:dph, GOC:tb]
GO:0010853
name: cyclase activator activity
def: "Increases the activity of an enzyme that catalyzes a ring closure reaction." [GOC:dph, GOC:tb]
GO:0010854
name: adenylate cyclase regulator activity
def: "Modulates the activity of the enzyme that catalyzes the reaction: ATP = 3',5'-cyclic AMP + diphosphate." [GOC:dph, GOC:tb]
GO:0010855
name: adenylate cyclase inhibitor activity
def: "Decreases the activity of the enzyme that catalyzes the reaction: ATP = 3',5'-cyclic AMP + diphosphate." [GOC:dph, GOC:tb]
GO:0010856
name: adenylate cyclase activator activity
def: "Increases the activity of the enzyme that catalyzes the reaction: ATP = 3',5'-cyclic AMP + diphosphate." [GOC:dph, GOC:tb]
GO:0010857
name: calcium-dependent protein kinase activity
def: "Catalysis of the reaction: a protein + ATP = a phosphoprotein + ADP. This reaction requires the presence of calcium." [GOC:dph, GOC:tb]
GO:0010858
name: calcium-dependent protein kinase regulator activity
def: "Modulates the activity of a calcium-dependent protein kinase, an enzyme which phosphorylates a protein in a calcium-dependent manner." [GOC:dph, GOC:tb]
GO:0010859
name: calcium-dependent cysteine-type endopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of a calcium-dependent cysteine-type endopeptidase, any enzyme that hydrolyzes peptide bonds in polypeptides by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile in a calcium-dependent manner." [GOC:dph, GOC:tb]
GO:0010860
name: proteasome regulator activity
def: "OBSOLETE. Modulates the activity of the proteasome complex. The proteasome complex performs regulated ubiquitin-dependent cytosolic and nuclear proteolysis." [GOC:dph, GOC:tb]
GO:0010861
name: thyroid hormone receptor activator activity
def: "A receptor activator activity that binds to, thyroid hormone receptor in a ligand-dependent manner and enhances ligand-induced transcriptional activity of the receptor." [GOC:dph, GOC:mah, GOC:tb, PMID:9346901]
GO:0010943
name: NADPH pyrophosphatase activity
def: "Catalysis of the reaction: NADPH + H2O = NMNH + ADP." [GOC:tb]
GO:0010945
name: CoA pyrophosphatase activity
def: "Catalysis of the reaction: coenzyme A + H2O = 3',5'-ADP + 4'-phosphopantetheine." [GOC:tb]
GO:0010997
name: anaphase-promoting complex binding
def: "Interacting selectively and non-covalently with an anaphase-promoting complex. A ubiquitin ligase complex that degrades mitotic cyclins and anaphase inhibitory protein, thereby triggering sister chromatid separation and exit from mitosis." [GOC:BHF, GOC:dph, GOC:tb]
GO:0015000
name: polyferredoxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015001
name: high-potential iron-sulfur carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015002
name: heme-copper terminal oxidase activity
def: "Catalysis of the four-electron reduction of dioxygen (O2) to water, coupled to generation of a proton electrochemical gradient across a membrane." [GOC:kd]
GO:0015003
name: copper electron carrier
def: "OBSOLETE. A copper-containing entity that serves as an electron acceptor and electron donor in an electron transport system." [GOC:ai]
GO:0015004
name: small blue copper electron carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015005
name: azurin
def: "OBSOLETE. Brilliant blue copper-containing protein of low molecular weight found in some bacteria; thought to transfer electrons to cytochrome oxidase. This definition includes pseudoazurin." [ISBN:0198547684]
GO:0015006
name: plastocyanin
def: "OBSOLETE. A copper-containing electron carrier acting between cytochrome b(6)-f and P700 of photosystem I." [ISBN:0198547684]
GO:0015007
name: electron carrier, chlorophyll electron transport system
def: "OBSOLETE. A molecular entity that serves as an electron acceptor and electron donor in a chlorophyll electron transport system." [ISBN:0198506732]
GO:0015016
name: [heparan sulfate]-glucosamine N-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine 3',5'-bisphosphate + [heparan sulfate]-N-sulfoglucosamine." [EC:2.8.2.8]
GO:0015017
name: glypican
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0015018
name: galactosylgalactosylxylosylprotein 3-beta-glucuronosyltransferase activity
def: "Catalysis of the reaction: UDP-glucuronate + 3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein = UDP + 3-beta-D-glucuronosyl-3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein." [EC:2.4.1.135]
GO:0015019
name: heparan-alpha-glucosaminide N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + heparan alpha-D-glucosaminide = CoA + heparan N-acetyl-alpha-D-glucosaminide." [EC:2.3.1.78]
GO:0015020
name: glucuronosyltransferase activity
def: "Catalysis of the reaction: UDP-glucuronate + acceptor = UDP + acceptor beta-D-glucuronoside." [EC:2.4.1.17]
GO:0015021
name: heparin-sulfate lyase activity
def: "Catalysis of the elimination of sulfate; appears to act on linkages between N-acetyl-D-glucosamine and uronate. Product is an unsaturated sugar." [EC:4.2.2.8]
GO:0015023
name: syndecan
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0015024
name: glucuronate-2-sulfatase activity
def: "Catalysis of the hydrolysis of the 2-sulfate groups of the 2-O-sulfo-D-glucuronate residues of chondroitin sulfate, heparin and heparitin sulfate." [EC:3.1.6.18]
GO:0015025
name: GPI-anchored membrane-bound receptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015026
name: coreceptor activity
def: "Combining with an extracellular or intracellular messenger, and in cooperation with a nearby primary receptor, initiating a change in cell activity." [GOC:go_curators]
GO:0015029
name: internalization receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0015034
name: cytochrome P450 activity
def: "OBSOLETE. A cytochrome b-like protein that has a sulfur atom ligated to the iron of the prosthetic group (heme-thiolate); enzymes: typically monooxygenases acting on, typically, lipophilic substrates. The characteristic mode of action of these enzymes is not electron transfer (some P450 enzymes probably do not even involve the reversible Fe(II)/Fe(III) equilibrium), but rather oxygen atom transfer." [ISBN:0198547684, PMID:1655423]
GO:0015035
name: protein disulfide oxidoreductase activity
def: "Catalysis of the reaction: a protein with reduced sulfide groups = a protein with oxidized disulfide bonds." [MetaCyc:DISULFOXRED-RXN]
GO:0015036
name: disulfide oxidoreductase activity
def: "Catalysis of the reaction: substrate with reduced sulfide groups = substrate with oxidized disulfide bonds." [MetaCyc:DISULFOXRED-RXN]
GO:0015037
name: peptide disulfide oxidoreductase activity
def: "Catalysis of the reaction: a peptide with reduced sulfide groups = a peptide with oxidized disulfide bonds." [GOC:mah, MetaCyc:DISULFOXRED-RXN]
GO:0015038
name: glutathione disulfide oxidoreductase activity
def: "Catalysis of the reaction: 2 glutathione + electron acceptor = glutathione disulfide + electron donor." [GOC:mah]
GO:0015039
name: NADPH-adrenodoxin reductase activity
def: "Catalysis of the reaction: oxidized adrenodoxin + NADPH + H+ = reduced adrenodoxin + NADP+." [EC:1.18.1.6, GOC:kd]
GO:0015040
name: electron transfer flavoprotein, group I
def: "OBSOLETE. An electron transfer flavoprotein that functions as a housekeeping protein that links acyl-CoA dehydrogenase reactions with the respiratory chain, such as in the fatty acid degradation pathway." [PMID:8599534]
GO:0015041
name: electron transfer flavoprotein, group II
def: "OBSOLETE. An electron transfer flavoprotein that functions as a housekeeping protein that is synthesized only under certain specific growth conditions and receives electrons from the oxidation of specific substrates, e.g. trimethylamine, carnitine and in nitrogen fixation." [PMID:8599534]
GO:0015042
name: trypanothione-disulfide reductase activity
def: "Catalysis of the reaction: NADP+ + trypanothione = NADPH + H+ + trypanothione disulfide." [EC:1.8.1.12]
GO:0015043
name: leghemoglobin reductase activity
def: "Catalysis of the reaction: NADPH + H+ + 2 ferrileghemoglobin = NADP+ + 2 ferroleghemoglobin." [EC:1.6.2.6]
GO:0015044
name: rubredoxin-NAD+ reductase activity
def: "Catalysis of the reaction: reduced rubredoxin + NAD+ = oxidized rubredoxin + NADH + H+." [EC:1.18.1.1]
GO:0015045
name: rubredoxin-NAD(P)+ reductase activity
def: "Catalysis of the reaction: reduced rubredoxin + NAD(P)+ = oxidized rubredoxin + NAD(P)H + H+." [EC:1.18.1.4]
GO:0015046
name: rubredoxin-NADP reductase activity
def: "Catalysis of the reaction: reduced rubredoxin + NADP+ = oxidized rubredoxin + NADPH + H+." [EC:1.18.1.-]
GO:0015047
name: NADPH-cytochrome-c2 reductase activity
def: "Catalysis of the reaction: NADPH + H+ + 2 ferricytochrome c2 = NADP+ + 2 ferrocytochrome c2." [EC:1.6.2.5]
GO:0015048
name: phthalate dioxygenase reductase activity
def: "Catalysis of the transfer of electrons between pyridine nucleotides (obligatory two-electron carriers) and hemes or (2Fe-2S) centers (obligatory one-electron carriers) in respiration, photosynthesis, and many oxygenase systems." [PMID:7589982]
GO:0015049
name: methane monooxygenase activity
def: "Catalysis of the reaction: methane + NAD(P)H + H+ + O2 = methanol + NAD(P)+ + H2O." [EC:1.14.13.25]
GO:0015051
name: X-opioid receptor activity
def: "OBSOLETE. Combining with an opioid to initiate a change in cell activity, with the pharmacological characteristics of X-opioid receptors." [InterPro:IPR001420]
GO:0015052
name: beta3-adrenergic receptor activity
def: "Combining with epinephrine or norepinephrine to initiate a change in cell activity via activation of a G protein, with pharmacological characteristics of beta3-adrenergic receptors." [GOC:mah, IUPHAR_GPCR:1274]
GO:0015053
name: opsin
def: "OBSOLETE. Hydrophobic glycoprotein to which 11-cis-retinal binds as a Schiff base (in rhodopsin) or 3,4-didehydro-11-cis-retinal binds as a Schiff base in cyanopsin and porphyropsin." [ISBN:0198547684]
GO:0015054
name: gastrin receptor activity
def: "Combining with gastrin and transmitting the signal across the membrane by activating an associated G-protein to initiate a change in cell activity." [GOC:ai, GOC:signaling]
GO:0015055
name: secretin receptor activity
def: "Combining with secretin to initiate a change in cell activity." [GOC:mah]
GO:0015056
name: corticotrophin-releasing factor receptor activity
def: "Combining with the corticotrophin-releasing factor family of ligands, including the urocortins, to initiate a change in cell activity." [PMID:12032352]
GO:0015057
name: thrombin receptor activity
def: "Combining with thrombin to initiate a G-protein mediated change in cell activity. A G-protein is a signal transduction molecule that alternates between an inactive GDP-bound and an active GTP-bound state." [GOC:ai]
GO:0015058
name: epidermal growth factor-like module containing hormone receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0015059
name: blue-sensitive opsin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015060
name: green-sensitive opsin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015061
name: red-sensitive opsin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015062
name: violet-sensitive opsin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015063
name: long-wave-sensitive opsin
def: "OBSOLETE. An opsin with maximal absorption above 500 nm." [PMID:10594055]
GO:0015064
name: UV-sensitive opsin
def: "OBSOLETE. An opsin with maximal absorption below 400 nm." [PMID:10594055]
GO:0015065
name: uridine nucleotide receptor activity
def: "Combining with UTP or UDP to initiate a change in cell activity." [GOC:curators, PMID:8537335]
GO:0015066
name: alpha-amylase inhibitor activity
def: "Stops, prevents or reduces the activity of alpha-amylase." [GOC:mah]
GO:0015067
name: amidinotransferase activity
def: "Catalysis of the reversible transfer of an amidino group to an acceptor." [GOC:ai]
GO:0015068
name: glycine amidinotransferase activity
def: "Catalysis of the reaction: L-arginine + glycine = L-ornithine + guanidinoacetate." [EC:2.1.4.1]
GO:0015069
name: scyllo-inosamine-4-phosphate amidinotransferase activity
def: "Catalysis of the reaction: 1-amino-1-deoxy-scyllo-inositol 4-phosphate + L-arginine = 1-guanidino-1-deoxy-scyllo-inositol 4-phosphate + L-ornithine." [EC:2.1.4.2, RHEA:13268]
GO:0015070
name: toxin activity
def: "OBSOLETE. Acts as to cause injury to other living organisms." [GOC:jl]
GO:0015072
name: phosphatidylinositol 3-kinase, class I, catalyst activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 1-phosphatidyl-1D-myo-inositol = ADP + 1-phosphatidyl-1D-myo-inositol 3-phosphate." [EC:2.7.1.137]
GO:0015073
name: phosphatidylinositol 3-kinase, class I, regulator activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 1-phosphatidyl-1D-myo-inositol = ADP + 1-phosphatidyl-1D-myo-inositol 3-phosphate." [EC:2.7.1.137]
GO:0015075
name: ion transmembrane transporter activity
def: "Catalysis of the transfer of an ion from one side of a membrane to the other." [GOC:dgf, GOC:mtg_transport, ISBN:0815340729]
GO:0015076
name: heavy metal ion transporter activity
def: "OBSOLETE. Enables the directed movement of heavy metal ions into, out of or within a cell, or between cells. Heavy metals are those that can form a coordination bond with a protein, as opposed to an alkali or alkaline-earth metal that can only form an ionic bond; this definition includes the following biologically relevant heavy metals: Cd, Co, Cu, Fe, Hg, Mn, Mo, Ni, V, W, Zn." [GOC:ai]
GO:0015077
name: monovalent inorganic cation transmembrane transporter activity
def: "Catalysis of the transfer of a inorganic cations with a valency of one from one side of a membrane to the other. Inorganic cations are atoms or small molecules with a positive charge that do not contain carbon in covalent linkage." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015078
name: hydrogen ion transmembrane transporter activity
def: "Catalysis of the transfer of hydrogen ions from one side of a membrane to the other." [GOC:ai]
GO:0015079
name: potassium ion transmembrane transporter activity
def: "Catalysis of the transfer of potassium ions (K+) from one side of a membrane to the other." [GOC:ai]
GO:0015080
name: silver ion transmembrane transporter activity
def: "Catalysis of the transfer of silver (Ag) ions from one side of a membrane to the other." [GOC:ai]
GO:0015081
name: sodium ion transmembrane transporter activity
def: "Catalysis of the transfer of sodium ions (Na+) from one side of a membrane to the other." [GOC:ai, GOC:BHF]
GO:0015083
name: aluminum ion transmembrane transporter activity
def: "Catalysis of the transfer of aluminum (Al) ions from one side of a membrane to the other." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015085
name: calcium ion transmembrane transporter activity
def: "Catalysis of the transfer of calcium (Ca) ions from one side of a membrane to the other." [GOC:dgf]
GO:0015086
name: cadmium ion transmembrane transporter activity
def: "Catalysis of the transfer of cadmium (Cd) ions from one side of a membrane to the other." [GOC:dgf]
GO:0015087
name: cobalt ion transmembrane transporter activity
def: "Catalysis of the transfer of cobalt (Co) ions from one side of a membrane to the other." [GOC:dgf]
GO:0015088
name: copper uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Cu+(out) = Cu+(in)." [TC:9.A.12.-.-]
GO:0015089
name: high-affinity copper ion transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Cu2+(out) = Cu2+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [TC:9.A.11.1.1]
GO:0015090
name: low-affinity iron ion transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Fe2+(out) = Fe2+(in). In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [TC:9.A.9.1.1]
GO:0015091
name: ferric iron transmembrane transporter activity
def: "Catalysis of the transfer of ferric iron (Fe(III) or Fe3+) ions from one side of a membrane to the other." [ISBN:0198506732]
GO:0015092
name: high-affinity ferric uptake transmembrane transporter activity
def: "Catalysis of the transfer of Fe(III) from the outside of a cell to the inside of a cell across a membrane. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:ai, TC:9.A.10.1.1]
GO:0015093
name: ferrous iron transmembrane transporter activity
def: "Catalysis of the transfer of ferrous iron (Fe(II) or Fe2+) ions from one side of a membrane to the other." [ISBN:0198506732]
GO:0015094
name: lead ion transmembrane transporter activity
def: "Catalysis of the transfer of lead (Pb) ions from one side of a membrane to the other." [GOC:ai]
GO:0015095
name: magnesium ion transmembrane transporter activity
def: "Catalysis of the transfer of magnesium (Mg) ions from one side of a membrane to the other." [GOC:dgf]
GO:0015096
name: manganese resistance permease activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015097
name: mercury ion transmembrane transporter activity
def: "Catalysis of the transfer of mercury (Hg) ions from one side of a membrane to the other." [GOC:ai]
GO:0015098
name: molybdate ion transmembrane transporter activity
def: "Catalysis of the transfer of molybdate (MoO4 2-) ions from one side of a membrane to the other. Molybdate is the bivalent anion derived from molybdic acid." [ISBN:0198506732]
GO:0015099
name: nickel cation transmembrane transporter activity
def: "Catalysis of the transfer of nickel (Ni) cations from one side of a membrane to the other." [GOC:ai]
GO:0015100
name: vanadium ion transmembrane transporter activity
def: "Catalysis of the transfer of vanadium (V) ions from one side of a membrane to the other." [GOC:ai]
GO:0015101
name: organic cation transmembrane transporter activity
def: "Catalysis of the transfer of organic cations from one side of a membrane to the other. Organic cations are atoms or small molecules with a positive charge that contain carbon in covalent linkage." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015103
name: inorganic anion transmembrane transporter activity
def: "Catalysis of the transfer of inorganic anions from one side of a membrane to the other. Inorganic anions are atoms or small molecules with a negative charge which do not contain carbon in covalent linkage." [GOC:ai]
GO:0015104
name: antimonite transmembrane transporter activity
def: "Catalysis of the transfer of antimonite from one side of a membrane to the other." [GOC:ai]
GO:0015105
name: arsenite transmembrane transporter activity
def: "Catalysis of the transfer of arsenite from one side of a membrane to the other." [GOC:ai]
GO:0015106
name: bicarbonate transmembrane transporter activity
def: "Catalysis of the transfer of bicarbonate from one side of a membrane to the other. Bicarbonate is the hydrogencarbonate ion, HCO3-." [GOC:ai]
GO:0015107
name: chlorate transmembrane transporter activity
def: "Catalysis of the transfer of chlorate, ClO3-, from one side of a membrane to the other." [CHEBI:49709, GOC:curators]
GO:0015108
name: chloride transmembrane transporter activity
def: "Catalysis of the transfer of chloride ions from one side of a membrane to the other." [GOC:ai]
GO:0015109
name: chromate transmembrane transporter activity
def: "Catalysis of the transfer of chromate from one side of a membrane to the other. Chromate is the anion of chromic acid, H2CrO4 (aq) or CrO3." [GOC:ai]
GO:0015110
name: cyanate transmembrane transporter activity
def: "Catalysis of the transfer of cyanate, NCO-, the anion of cyanic acid, from one side of a membrane to the other." [GOC:ai]
GO:0015111
name: iodide transmembrane transporter activity
def: "Catalysis of the transfer of iodide ions from one side of a membrane to the other." [GOC:ai]
GO:0015112
name: nitrate transmembrane transporter activity
def: "Catalysis of the transfer of nitrate ions (NO3-) from one side of a membrane to the other." [GOC:ai]
GO:0015113
name: nitrite transmembrane transporter activity
def: "Catalysis of the transfer of nitrite (NO2-) ions from one side of a membrane to the other." [GOC:ai]
GO:0015114
name: phosphate ion transmembrane transporter activity
def: "Catalysis of the transfer of phosphate (PO4 3-) ions from one side of a membrane to the other." [GOC:ai]
GO:0015115
name: silicate transmembrane transporter activity
def: "Catalysis of the transfer of silicates from one side of a membrane to the other. Silicates are the salts of silicic acids, and are usually composed of silicon and oxygen (Si[x]O[y]), one or more metals, and possibly hydrogen. Types of silicate include unisilicates, metasilicates and hydrous silicates." [CHEBI:48123, GOC:ai]
GO:0015116
name: sulfate transmembrane transporter activity
def: "Catalysis of the transfer of sulfate ions, SO4(2-), from one side of a membrane to the other." [GOC:ai]
GO:0015117
name: thiosulfate transmembrane transporter activity
def: "Catalysis of the transfer of thiosulfate ions, S2O3(2-), from one side of a membrane to the other." [GOC:ai]
GO:0015118
name: tellurite transmembrane transporter activity
def: "Catalysis of the transfer of tellurite from one side of a membrane to the other. Tellurite is a salt of tellurous acid or an oxide of tellurium which occurs sparingly in tufts of white or yellowish crystals." [CHEBI:30477, GOC:ai]
GO:0015119
name: hexose phosphate transmembrane transporter activity
def: "Catalysis of the transfer of hexose phosphate from one side of the membrane to the other. Hexose phosphates is any of a group of monophosphorylated aldoses with a chain of six carbon atoms in the molecule." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015120
name: phosphoglycerate transmembrane transporter activity
def: "Catalysis of the transfer of phosphoglycerates from one side of the membrane to the other. Phosphoglycerates are important intermediates in glycolysis and 3-phosphoglycerate is a precursor in serine biosynthesis." [GOC:ai]
GO:0015121
name: phosphoenolpyruvate:phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: phosphoenolpyruvate(out) + phosphate(in) = phosphoenolpyruvate(in) + phosphate(out)." [GOC:bf, GOC:jl]
GO:0015123
name: acetate transmembrane transporter activity
def: "Catalysis of the transfer of acetate from one side of the membrane to the other. Acetate is the 2-carbon carboxylic acid ethanoic acid." [GOC:ai]
GO:0015124
name: allantoate transmembrane transporter activity
def: "Catalysis of the transfer of allantoate from one side of the membrane to the other. Allantoate is the end product of purine metabolism in mammals and some fish, formed form allantoin. It is widely distributed in plants as an important source of stored nitrogen." [GOC:ai, ISBN:0198547684]
GO:0015125
name: bile acid transmembrane transporter activity
def: "Catalysis of the transfer of bile acid from one side of the membrane to the other. Bile acids are any of a group of steroid carboxylic acids occurring in bile, where they are present as the sodium salts of their amides with glycine or taurine." [GOC:ai]
GO:0015126
name: canalicular bile acid transmembrane transporter activity
def: "The directed movement of bile acid and bile salts out of a hepatocyte and into the bile canaliculus by means of an agent such as a transporter or pore. Bile canaliculi are the thin tubes formed by hepatocyte membranes. Bile acids are any of a group of steroid carboxylic acids occurring in bile, where they are present as the sodium salts of their amides with glycine or taurine." [GOC:dph]
GO:0015127
name: bilirubin transmembrane transporter activity
def: "Catalysis of the transfer of bilirubin from one side of the membrane to the other. Bilirubin is a linear tetrapyrrole produced in the reticuloendothelial system from biliverdin and transported to the liver as a complex with serum albumin. In the liver, bilirubin is converted to bilirubin bisglucuronide, which is excreted in the bile." [GOC:ai, ISBN:0198547684]
GO:0015128
name: gluconate transmembrane transporter activity
def: "Catalysis of the transfer of gluconate from one side of the membrane to the other. Gluconate is the aldonic acid derived from glucose." [GOC:ai, ISBN:0198506732]
GO:0015129
name: lactate transmembrane transporter activity
def: "Catalysis of the transfer of lactate from one side of the membrane to the other. Lactate is 2-hydroxypropanoate, CH3-CHOH-COOH; L(+)-lactate is formed by anaerobic glycolysis in animal tissues, and DL-lactate is found in sour milk, molasses and certain fruit juices." [GOC:ai, ISBN:0198506732]
GO:0015130
name: mevalonate transmembrane transporter activity
def: "Catalysis of the transfer of mevalonate from one side of the membrane to the other. Mevalonate is the anion of mevalonic acid; its (R)-enantiomer is a strategic intermediate derived from hydroxymethylglutaryl-CoA in the biosynthesis of polyprenyl compounds." [GOC:ai, ISBN:0198506732]
GO:0015131
name: oxaloacetate transmembrane transporter activity
def: "Catalysis of the transfer of oxaloacetate, the anion of oxobutanedioic acid, from one side of the membrane to the other." [GOC:ai]
GO:0015132
name: prostaglandin transmembrane transporter activity
def: "Catalysis of the transfer of prostaglandins from one side of the membrane to the other. A prostaglandin is any of a group of biologically active metabolites which contain a cyclopentane ring due to the formation of a bond between two carbons of a fatty acid. They have a wide range of biological activities." [GOC:ai]
GO:0015133
name: uronic acid transmembrane transporter activity
def: "Catalysis of the transfer of uronic acid from one side of the membrane to the other. Uronic acids are any monocarboxylic acid formally derived by oxidizing to a carboxyl group the terminal hydroxymethylene group of either an aldose with four or more carbon atoms in the molecule, or of any glycoside derived from such an aldose." [GOC:ai]
GO:0015134
name: hexuronate transmembrane transporter activity
def: "Catalysis of the transfer of hexuronates from one side of the membrane to the other. A hexuronate is any monocarboxylic acid derived from a hexose by oxidation of C-6." [GOC:ai, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015135
name: glucuronate transmembrane transporter activity
def: "Catalysis of the transfer of glucuronate from one side of the membrane to the other. Glucuronate is the uronic acid formally derived from glucose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:ai]
GO:0015136
name: sialic acid transmembrane transporter activity
def: "Catalysis of the transfer of sialic acid from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0015137
name: citrate transmembrane transporter activity
def: "Catalysis of the transfer of citrate, 2-hydroxy-1,2,3-propanetricarboyxlate, from one side of the membrane to the other." [GOC:ai]
GO:0015138
name: fumarate transmembrane transporter activity
def: "Catalysis of the transfer of fumarate from one side of the membrane to the other. Fumarate is a key intermediate in metabolism and is formed in the TCA cycle from succinate and converted into malate." [GOC:ai]
GO:0015139
name: alpha-ketoglutarate transmembrane transporter activity
def: "Catalysis of the transfer of alpha-ketoglutarate from one side of the membrane to the other. Alpha-ketoglutarate (or oxoglutarate) is a compound with important roles in carbohydrate and amino acid metabolism, especially in transamination reactions and as a component of the TCA cycle." [GOC:ai, ISBN:0198547684]
GO:0015140
name: malate transmembrane transporter activity
def: "Catalysis of the transfer of malate from one side of the membrane to the other. Malate is a chiral hydroxydicarboxylic acid, hydroxybutanedioic acid. The (+) enantiomer is an important intermediate in metabolism as a component of both the TCA cycle and the glyoxylate cycle." [GOC:ai]
GO:0015141
name: succinate transmembrane transporter activity
def: "Catalysis of the transfer of succinate, the dianion of ethane dicarboxylic acid, from one side of the membrane to the other." [ISBN:0198506732]
GO:0015142
name: tricarboxylic acid transmembrane transporter activity
def: "Catalysis of the transfer of tricarboxylic acids from one side of the membrane to the other. Tricarboxylic acid are organic acids with three COOH groups." [GOC:ai]
GO:0015143
name: urate transmembrane transporter activity
def: "Catalysis of the transfer of urate from one side of the membrane to the other. Urate is the anion of uric acid, 2,6,8-trioxypurine, the end product of purine metabolism in certain mammals and the main excretory product in uricotelic animals." [GOC:ai]
GO:0015144
name: carbohydrate transmembrane transporter activity
def: "Catalysis of the transfer of carbohydrate from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0015145
name: monosaccharide transmembrane transporter activity
def: "Catalysis of the transfer of a monosaccharide from one side of a membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0015146
name: pentose transmembrane transporter activity
def: "Catalysis of the transfer of a pentose sugar, a monosaccharide with 5 carbon atoms, from one side of the membrane to the other." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015147
name: L-arabinose transmembrane transporter activity
def: "Catalysis of the transfer of L-arabinose from one side of the membrane to the other. Arabinose occurs free, for example in the heartwood of many conifers and in the combined states, in both furanose and pyranose forms, as a constituent of various plant hemicelluloses, bacterial polysaccharides, etc." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015148
name: D-xylose transmembrane transporter activity
def: "Catalysis of the transfer of D-xylose from one side of the membrane to the other. D-xylose (the naturally occurring enantiomer is always D-) is a constituent of plant polysaccharides." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015149
name: hexose transmembrane transporter activity
def: "Catalysis of the transfer of a hexose sugar, a monosaccharide with 6 carbon atoms, from one side of a membrane to the other." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015150
name: fucose transmembrane transporter activity
def: "Catalysis of the transfer of fucose from one side of the membrane to the other. Fucose is 6-deoxygalactose and has two enantiomers, D-fucose and L-fucose." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015151
name: alpha-glucoside transmembrane transporter activity
def: "Catalysis of the transfer of alpha-glucosides from one side of the membrane to the other. Alpha-glucosides are glycosides in which the sugar group is a glucose residue, and the anomeric carbon of the bond is in an alpha configuration." [GOC:jl, GOC:mtg_transport, http://www.biochem.purdue.edu/, ISBN:0198506732, ISBN:0815340729]
GO:0015152
name: glucose-6-phosphate transmembrane transporter activity
def: "Catalysis of the transfer of glucose-6-phosphate from one side of the membrane to the other. Glucose-6-phosphate is a monophosphorylated derivative of glucose with the phosphate group attached to C-6." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015153
name: rhamnose transmembrane transporter activity
def: "Catalysis of the transfer of rhamnose from one side of the membrane to the other. Rhamnose occurs commonly as a compound of plant glycosides, in polysaccharides of gums and mucilages, and in bacterial polysaccharides. It is also a component of some plant cell wall polysaccharides and frequently acts as the sugar components of flavonoids." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015154
name: disaccharide transmembrane transporter activity
def: "Catalysis of the transfer of disaccharide from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0015155
name: lactose transmembrane transporter activity
def: "Catalysis of the transfer of lactose from one side of the membrane to the other. Lactose is a disaccharide 4-O-beta-D-galactopyranosyl-D-glucose, and constitutes roughly 5% of the milk in almost all mammals." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015156
name: melibiose transmembrane transporter activity
def: "Catalysis of the transfer of melibiose from one side of the membrane to the other. Melibiose is the disaccharide 6-O-alpha-D-galactopyranosyl-D-glucose and occurs as a constituent of the trisaccharide raffinose or in the exudates and nectaries of a number of plants." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015157
name: oligosaccharide transmembrane transporter activity
def: "Catalysis of the transfer of oligosaccharide from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0015158
name: raffinose transmembrane transporter activity
def: "Catalysis of the transfer of raffinose from one side of the membrane to the other. Raffinose occurs in plants almost as commonly as sucrose and is present in cereal grains, cotton seeds, and many legumes. It is synthesized from sucrose by transfer of a galactopyranoside from myo-inositol." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015159
name: polysaccharide transmembrane transporter activity
def: "Catalysis of the transfer of polysaccharides from one side of the membrane to the other. A polysaccharide is a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [CHEBI:18154, GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015160
name: beta-glucan transmembrane transporter activity
def: "Catalysis of the transfer of beta-glucans from one side of the membrane to the other. Beta-glucans are compounds composed of glucose residues linked by beta-glucosidic bonds." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015161
name: capsular polysaccharide transmembrane transporter activity
def: "Catalysis of the transfer of capsular-polysaccharides from one side of the membrane to the other. Capsular polysaccharides make up the capsule, a protective structure surrounding some species of bacteria and fungi." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015162
name: teichoic acid transmembrane transporter activity
def: "Catalysis of the transfer of teichoic acid from one side of the membrane to the other. Teichoic acid is any polymer occurring in the cell wall, membrane or capsule of Gram-positive bacteria and containing chains of glycerol phosphate or ribitol phosphate residues." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015164
name: glucuronoside transmembrane transporter activity
def: "Catalysis of the transfer of a glucuronosides from one side of the membrane to the other. Glucuronosides are any compound formed by combination of glycosidic linkage of a hydroxy compound (e.g. an alcohol or a saccharide) with the anomeric carbon atom of glucuronate." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015165
name: pyrimidine nucleotide-sugar transmembrane transporter activity
def: "Catalysis of the transfer of a pyrimidine nucleotide-sugar from one side of the membrane to the other. Pyrimidine nucleotide-sugars are pyrimidine nucleotides in glycosidic linkage with a monosaccharide or monosaccharide derivative." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015166
name: polyol transmembrane transporter activity
def: "Catalysis of the transfer of a polyol from one side of the membrane to the other. A polyol is any polyhydric alcohol." [ISBN:0198506732]
GO:0015167
name: arabitol transmembrane transporter activity
def: "Catalysis of the transfer of an arabitol from one side of the membrane to the other. Arabitol is the pentitol derived from arabinose or lyxose by reduction of the aldehyde group. The D enantiomer is present in lichens and mushrooms." [ISBN:0198506732]
GO:0015168
name: glycerol transmembrane transporter activity
def: "Catalysis of the transfer of glycerol from one side of the membrane to the other. Glycerol is 1,2,3-propanetriol, a sweet, hygroscopic, viscous liquid, widely distributed in nature as a constituent of many lipids." [GOC:ai]
GO:0015169
name: glycerol-3-phosphate transmembrane transporter activity
def: "Catalysis of the transfer of glycerol-3-phosphate from one side of the membrane to the other. Glycerol-3-phosphate is a phosphoric monoester of glycerol." [GOC:ai]
GO:0015170
name: propanediol transmembrane transporter activity
def: "Catalysis of the transfer of propanediol from one side of the membrane to the other. Propanediol is a sweet colorless, viscous, hygroscopic liquid used as an antifreeze and in brake fluid; it is also as a humectant in cosmetics and personal care items, although it can be absorbed through the skin with harmful effects." [GOC:ai]
GO:0015171
name: amino acid transmembrane transporter activity
def: "Catalysis of the transfer of amino acids from one side of a membrane to the other. Amino acids are organic molecules that contain an amino group and a carboxyl group." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015172
name: acidic amino acid transmembrane transporter activity
def: "Catalysis of the transfer of acidic amino acids from one side of a membrane to the other. Acidic amino acids have a pH below 7." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015173
name: aromatic amino acid transmembrane transporter activity
def: "Catalysis of the transfer of aromatic amino acids from one side of a membrane to the other. Aromatic amino acids have an aromatic ring." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015174
name: basic amino acid transmembrane transporter activity
def: "Catalysis of the transfer of basic amino acids from one side of a membrane to the other. Basic amino acids have a pH above 7." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015175
name: neutral amino acid transmembrane transporter activity
def: "Catalysis of the transfer of neutral amino acids from one side of a membrane to the other. Neutral amino acids have a pH of 7." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015176
name: holin
def: "OBSOLETE. Primary function of holins appears to be transport of murein hydrolases across the cytoplasmic membrane to the cell wall of bacteria, where these enzymes hydrolyze the cell wall polymer as a prelude to cell lysis. When chromosomally encoded, these enzymes are therefore autolysins. Holins may also facilitate leakage of electrolytes and nutrients from the cell cytoplasm, thereby promoting cell death. Some catalyze export of nucleases." [TC:1.A.38.-.-]
GO:0015179
name: L-amino acid transmembrane transporter activity
def: "Catalysis of the transfer of an L-amino acid from one side of a membrane to the other. L-amino acids are the L-enantiomers of amino acids." [GOC:ai, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
GO:0015180
name: L-alanine transmembrane transporter activity
def: "Catalysis of the transfer of L-alanine from one side of a membrane to the other. L-alanine is the L-enantiomer of 2-aminopropanoic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015181
name: arginine transmembrane transporter activity
def: "Catalysis of the stereospecific transfer of arginine, 2-amino-5-guanidinopentanoic acid, across a biological membrane." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015182
name: L-asparagine transmembrane transporter activity
def: "Catalysis of the transfer of L-asparagine from one side of a membrane to the other. L-asparagine is the L-enantiomer of alpha-aminosuccinamic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015183
name: L-aspartate transmembrane transporter activity
def: "Catalysis of the transfer of L-aspartate from one side of a membrane to the other. L-aspartate is the anion derived from aspartic acid." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015184
name: L-cystine transmembrane transporter activity
def: "Catalysis of the transfer of L-cystine from one side of a membrane to the other." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015185
name: gamma-aminobutyric acid transmembrane transporter activity
def: "Catalysis of the transfer of gamma-aminobutyric acid from one side of a membrane to the other. Gamma-aminobutyric acid is 4-aminobutyrate (GABA)." [GOC:go_curators, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015186
name: L-glutamine transmembrane transporter activity
def: "Catalysis of the transfer of L-glutamine from one side of a membrane to the other. L-glutamine is 2-amino-4-carbamoylbutanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015187
name: glycine transmembrane transporter activity
def: "Catalysis of the transfer of glycine from one side of a membrane to the other. Glycine is aminoethanoic acid." [GOC:ai]
GO:0015188
name: L-isoleucine transmembrane transporter activity
def: "Catalysis of the transfer of L-isoleucine from one side of a membrane to the other. L-isoleucine is (2R*,3R*)-2-amino-3-methylpentanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015189
name: L-lysine transmembrane transporter activity
def: "Catalysis of the transfer of L-lysine from one side of a membrane to the other. L-lysine is 2,6-diaminohexanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015190
name: L-leucine transmembrane transporter activity
def: "Catalysis of the transfer of L-leucine from one side of a membrane to the other. L-leucine is 2-amino-4-methylpentanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015191
name: L-methionine transmembrane transporter activity
def: "Catalysis of the transfer of L-methionine from one side of a membrane to the other. L-methionine is 2-amino-4-(methylthio)butanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015192
name: L-phenylalanine transmembrane transporter activity
def: "Catalysis of the transfer of L-phenylalanine from one side of a membrane to the other. L-phenylalanine is 2-amino-3-phenylpropanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015193
name: L-proline transmembrane transporter activity
def: "Catalysis of the transfer of L-proline from one side of a membrane to the other. L-proline is pyrrolidine-2-carboxylic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015194
name: L-serine transmembrane transporter activity
def: "Catalysis of the transfer of L-serine from one side of a membrane to the other. L-serine is the L-enantiomer of 2-amino-3-hydroxypropanoic acid." [GOC:ai, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
GO:0015195
name: L-threonine transmembrane transporter activity
def: "Catalysis of the transfer of L-threonine from one side of a membrane to the other. L-threonine is (2R*,3S*)-2-amino-3-hydroxybutanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015196
name: L-tryptophan transmembrane transporter activity
def: "Catalysis of the transfer of L-tryptophan from one side of a membrane to the other. Tryptophan is 2-amino-3-(1H-indol-3-yl)propanoic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015197
name: peptide transporter activity
def: "Enables the directed movement of peptides, compounds of two or more amino acids where the alpha carboxyl group of one is bound to the alpha amino group of another, into, out of or within a cell, or between cells." [GOC:ai]
GO:0015198
name: oligopeptide transporter activity
def: "Enables the directed movement of oligopeptides into, out of or within a cell, or between cells. Oligopeptides are molecules that contain a small number (2 to 20) of amino-acid residues connected by peptide linkages." [ISBN:0198506732]
GO:0015199
name: amino-acid betaine transmembrane transporter activity
def: "Catalysis of the transfer of betaine, the N-trimethyl derivative of an amino acid, from one side of the membrane to the other." [GOC:ai]
GO:0015200
name: methylammonium transmembrane transporter activity
def: "Enables the directed movement of methylammonium, CH3NH2, into, out of or within a cell, or between cells." [GOC:ai]
GO:0015203
name: polyamine transmembrane transporter activity
def: "Catalysis of the transfer of polyamines, organic compounds containing two or more amino groups, from one side of the membrane to the other." [GOC:ai]
GO:0015204
name: urea transmembrane transporter activity
def: "Enables the directed movement of urea cross a membrane into, out of or within a cell, or between cells. Urea is the water soluble compound H2N-CO-NH2." [ISBN:0198506732]
GO:0015205
name: nucleobase transmembrane transporter activity
def: "Catalysis of the transfer of a nucleobase, any nitrogenous base that is a constituent of a nucleoside, nucleotide, or nucleic acidfrom one side of a membrane to the other." [ISBN:0198506732]
GO:0015207
name: adenine transmembrane transporter activity
def: "Catalysis of the transfer of adenine, 6-aminopurine, from one side of a membrane to the other." [GOC:go_curators]
GO:0015208
name: guanine transmembrane transporter activity
def: "Catalysis of the transfer of guanine, 2-amino-6-hydroxypurine, from one side of a membrane to the other." [GOC:go_curators]
GO:0015209
name: cytosine transmembrane transporter activity
def: "Catalysis of the transfer of cytosine, 4-amino-2-hydroxypyrimidine from one side of a membrane to the other." [GOC:go_curators]
GO:0015210
name: uracil transmembrane transporter activity
def: "Catalysis of the transfer of uracil, 2,4-dioxopyrimidine, from one side of a membrane to the other." [GOC:go_curators]
GO:0015211
name: purine nucleoside transmembrane transporter activity
def: "Catalysis of the transfer of a purine nucleoside, a purine base covalently bonded to a ribose or deoxyribose sugar, from one side of a membrane to the other." [GOC:ai]
GO:0015212
name: cytidine transmembrane transporter activity
def: "Catalysis of the transfer of cytidine, cytosine riboside, from one side of a membrane to the other." [GOC:go_curators]
GO:0015213
name: uridine transmembrane transporter activity
def: "Catalysis of the transfer of uridine, uracil riboside, from one side of a membrane to the other." [GOC:go_curators]
GO:0015214
name: pyrimidine nucleoside transmembrane transporter activity
def: "Catalysis of the transfer of a pyrimidine nucleoside, a pyrimidine base covalently bonded to a ribose or deoxyribose sugar from one side of a membrane to the other." [GOC:ai]
GO:0015215
name: nucleotide transmembrane transporter activity
def: "Catalysis of the transfer of a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate, from one side of a membrane to the other." [ISBN:0198506732]
GO:0015216
name: purine nucleotide transmembrane transporter activity
def: "Catalysis of the transfer of a purine nucleotide, any compound consisting of a purine nucleoside esterified with (ortho)phosphate, from one side of a membrane to the other." [GOC:ai]
GO:0015217
name: ADP transmembrane transporter activity
def: "Catalysis of the transfer of ADP, adenosine diphosphate, from one side of a membrane to the other." [GOC:ai]
GO:0015218
name: pyrimidine nucleotide transmembrane transporter activity
def: "Catalysis of the transfer of a pyrimidine nucleotide, any compound consisting of a pyrimidine nucleoside esterified with (ortho)phosphate, from one side of a membrane to the other." [GOC:ai]
GO:0015219
name: protein-DNA complex transmembrane transporter activity
def: "Catalysis of the transfer of protein-DNA complexes from one side of a membrane to the other." [GOC:ai]
GO:0015220
name: choline transmembrane transporter activity
def: "Catalysis of the transfer of choline from one side of the membrane to the other. Choline (2-hydroxyethyltrimethylammonium) is an amino alcohol that occurs widely in living organisms as a constituent of certain types of phospholipids and in the neurotransmitter acetylcholine." [GOC:ai]
GO:0015221
name: lipopolysaccharide transmembrane transporter activity
def: "Catalysis of the transfer of lipopolysaccharides from one side of the membrane to the other. A lipopolysaccharide is any of a group of related, structurally complex components of the outer membrane of Gram-negative bacteria. Lipopolysaccharides consist three covalently linked regions, lipid A, core oligosaccharide, and an O side chain. Lipid A is responsible for the toxicity of the lipopolysaccharide." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015222
name: serotonin transmembrane transporter activity
def: "Catalysis of the transfer of serotonin from one side of the membrane to the other. Serotonin (5-hydroxytryptamine) is a monoamine neurotransmitter occurring in the peripheral and central nervous systems." [GOC:ai]
GO:0015223
name: vitamin or cofactor transporter activity
def: "OBSOLETE. Enables the directed transport of vitamins or cofactors into, out of or within a cell, or between cells." [GOC:ai]
GO:0015224
name: biopterin transporter activity
def: "Enables the directed movement of biopterin into, out of or within a cell, or between cells. Biopterin is a growth factor for certain protozoans and some insects; it is widely distributed in tissues and functions in a reduced form, tetrahydrobiopterin, as a hydroxylation coenzyme." [ISBN:0198506732]
GO:0015225
name: biotin transporter activity
def: "Enables the directed movement of biotin into, out of or within a cell, or between cells. Biotin is cis-tetrahydro-2-oxothieno(3,4-d)imidazoline-4-valeric acid; the (+) enantiomer is very widely distributed in cells and serves as a carrier in a number of enzymatic beta-carboxylation reactions." [GOC:ai]
GO:0015226
name: carnitine transmembrane transporter activity
def: "Enables the directed movement of carnitine across a membrane. Carnitine is a compound that participates in the transfer of acyl groups across the inner mitochondrial membrane." [GOC:ai]
GO:0015227
name: acyl carnitine transmembrane transporter activity
def: "Enables the directed movement of acyl carnitine across a membrane. Acyl carnitine is the condensation product of a carboxylic acid and carnitine and is the transport form for a fatty acid crossing the mitochondrial membrane." [CHEBI:17387, GOC:ai]
GO:0015228
name: coenzyme A transmembrane transporter activity
def: "Enables the directed movement of coenzyme A across a membrane into, out of or within a cell, or between cells. Coenzyme A, 3'-phosphoadenosine-(5')diphospho(4')pantatheine, is an acyl carrier in many acylation and acyl-transfer reactions in which the intermediate is a thiol ester." [GOC:ai]
GO:0015229
name: L-ascorbic acid transporter activity
def: "Enables the directed movement of L-ascorbate into, out of or within a cell, or between cells. L-ascorbate, (2R)-2-[(1S)-1,2-dihydroxyethyl]-4-hydroxy-5-oxo-2,5-dihydrofuran-3-olate, is vitamin C and has co-factor and anti-oxidant activities in many species." [CHEBI:38290, ISBN:0198506732]
GO:0015230
name: FAD transmembrane transporter activity
def: "Enables the directed movement of flavin-adenine dinucleotide (FAD) across a membrane into, out of or within a cell, or between cells. FAD forms the coenzyme of the prosthetic group of various flavoprotein oxidoreductase enzymes, in which it functions as an electron acceptor by being reversibly converted to its reduced form." [ISBN:0198506732]
GO:0015231
name: 5-formyltetrahydrofolate transporter activity
def: "Enables the directed movement of 5-formyltetrahydrofolate, the formylated derivative of tetrahydrofolate, into, out of, within, or between cells." [GOC:ai]
GO:0015232
name: heme transporter activity
def: "Enables the directed movement of heme, any compound of iron complexed in a porphyrin (tetrapyrrole) ring, into, out of or within a cell, or between cells." [GOC:ai]
GO:0015233
name: pantothenate transmembrane transporter activity
def: "Enables the directed movement of pantothenate across a membrane. Pantothenate is the anion of pantothenic acid, the amide of beta-alanine and pantoic acid; it is a B complex vitamin that is a constituent of coenzyme A and is distributed ubiquitously in foods." [GOC:ai, ISBN:0721662544]
GO:0015234
name: thiamine transmembrane transporter activity
def: "Catalysis of the transfer of thiamine from one side of the membrane to the other. Thiamine is vitamin B1, a water soluble vitamin present in fresh vegetables and meats, especially liver." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015235
name: cobalamin transporter activity
def: "Enables the directed movement of cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom, into, out of or within a cell." [GOC:ai]
GO:0015238
name: drug transmembrane transporter activity
def: "Enables the directed movement of a drug from one side of a membrane to the other. A drug is any naturally occurring or synthetic substance, other than a nutrient, that, when administered or applied to an organism, affects the structure or functioning of the organism; in particular, any such substance used in the diagnosis, prevention, or treatment of disease." [ISBN:0198506732]
GO:0015240
name: amiloride transporter activity
def: "Enables the directed movement of amiloride into, out of or within a cell, or between cells. Amiloride is a potent and specific inhibitor of sodium ion entry into cells. It is used as a potassium-sparing diuretic." [ISBN:0198506732]
GO:0015241
name: aminotriazole transporter activity
def: "Enables the directed movement of aminotriazole into, out of or within a cell, or between cells. Aminotriazole is an effective weed killer that also possesses some antithyroid activity." [CHEBI:40036, GOC:curators]
GO:0015242
name: benomyl transporter activity
def: "Enables the directed movement of benomyl into, out of or within a cell, or between cells. Benomyl, methyl 1-(butylcarbamoyl)-2-benzimidazolecarbamate, is a systemic agricultural fungicide used for control of certain fungal diseases of stone fruit." [CHEBI:3015, GOC:curators]
GO:0015243
name: cycloheximide transporter activity
def: "Enables the directed movement of cycloheximide into, out of or within a cell, or between cells. Cycloheximide is an antibiotic produced by Streptomyces which interferes with protein synthesis in eukaryotes." [ISBN:0198506732]
GO:0015244
name: fluconazole transporter activity
def: "Enables the directed movement of fluconazole into, out of or within a cell, or between cells. Fluconazole is an antifungal drug used for oral candidiasis and cryptococcal meningitis; it is still under study for treatment of vaginal candidiasis and other fungal infections." [CHEBI:46081, GOC:curators]
GO:0015245
name: fatty acid transporter activity
def: "Enables the directed movement of fatty acids into, out of or within a cell, or between cells. Fatty acids are aliphatic monocarboxylic acids liberated from naturally occurring fats and oils by hydrolysis." [ISBN:0198506732]
GO:0015246
name: fatty-acyl group transporter activity
def: "Enables the directed movement of a fatty acyl group into, out of or within a cell, or between cells. A fatty acyl group is any acyl group derived from a fatty acid." [ISBN:0198506732]
GO:0015247
name: aminophospholipid transporter activity
def: "Enables the directed movement of aminophospholipids into, out of or within a cell, or between cells. Aminophospholipids contain phosphoric acid as a mono- or diester and an amino (NH2) group." [GOC:ai]
GO:0015248
name: sterol transporter activity
def: "Enables the directed movement of sterols into, out of or within a cell, or between cells. Sterol are steroids with one or more hydroxyl groups and a hydrocarbon side-chain in the molecule." [GOC:ai]
GO:0015250
name: water channel activity
def: "Transport systems of this type catalyze facilitated diffusion of water (by an energy-independent process) by passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, ISBN:0815340729]
GO:0015251
name: ammonium channel activity
def: "Catalysis of facilitated diffusion of ammonium (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, ISBN:0815340729]
GO:0015252
name: hydrogen ion channel activity
def: "Catalysis of facilitated diffusion of a hydrogen ion (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, ISBN:0815340729]
GO:0015253
name: sugar/polyol channel activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015254
name: glycerol channel activity
def: "Catalysis of facilitated diffusion of glycerol (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, ISBN:0815340729]
GO:0015255
name: propanediol channel activity
def: "Catalysis of facilitated diffusion of propanediol (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport]
GO:0015256
name: monocarboxylate channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015257
name: organic anion channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015258
name: gluconate channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015259
name: glutamate channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015260
name: isethionate channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015261
name: lactate channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015262
name: taurine channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015263
name: amine/amide/polyamine channel activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015264
name: methylammonium channel activity
def: "Catalysis of facilitated diffusion of methylammonium (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism. Methylammonium is CH3NH2." [GOC:mtg_transport]
GO:0015265
name: urea channel activity
def: "Catalysis of facilitated diffusion of urea (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel without evidence for a carrier-mediated mechanism." [GOC:mtg_transport]
GO:0015266
name: protein channel activity
def: "Catalysis of energy-independent facilitated diffusion, mediated by passage of proteins through a transmembrane aqueous pore or channel." [GOC:rb]
GO:0015267
name: channel activity
def: "Catalysis of energy-independent facilitated diffusion, mediated by passage of a solute through a transmembrane aqueous pore or channel. Stereospecificity is not exhibited but this transport may be specific for a particular molecular species or class of molecules." [GOC:mtg_transport, ISBN:0815340729, TC:1.-.-.-.-]
GO:0015269
name: calcium-activated potassium channel activity
def: "Catalysis of the calcium concentration-regulatable energy-independent passage of potassium ions across a lipid bilayer down a concentration gradient." [GOC:dph, GOC:mtg_transport]
GO:0015271
name: outward rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an outwardly-rectifying voltage-gated channel. An outwardly rectifying current-voltage relation is one where at any given driving force the outward flow of K+ ions exceeds the inward flow for the opposite driving force." [GOC:mah]
GO:0015272
name: ATP-activated inward rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an inwardly-rectifying voltage-gated channel, where the inward rectification is due to a voltage-dependent block of the channel pore by ATP. An inwardly rectifying current-voltage relation is one where at any given driving force the inward flow of K+ ions exceeds the outward flow for the opposite driving force." [GOC:cb, GOC:mah]
GO:0015274
name: organellar voltage-gated chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a voltage-gated channel. The membrane is an organellar membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015275
name: stretch-activated, cation-selective, calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens in response to a mechanical stress in the form of stretching." [GOC:mtg_transport]
GO:0015276
name: ligand-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0015277
name: kainate selective glutamate receptor activity
def: "An ionotropic glutamate receptor activity that exhibits fast gating by glutamate, acts by opening a cation channel permeable to sodium and potassium, and for which kainate is an agonist." [GOC:mah, PMID:10049997, PMID:8804111]
GO:0015278
name: calcium-release channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:mah]
GO:0015279
name: store-operated calcium channel activity
def: "A ligand-gated ion channel activity which transports calcium in response to emptying of intracellular calcium stores." [GOC:dph, GOC:tb, PMID:15788710]
GO:0015280
name: ligand-gated sodium channel activity
def: "Catalysis of the transmembrane transfer of a sodium ion by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:mah]
GO:0015282
name: NADPH oxidase-associated cytochrome b558 hydrogen channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015283
name: apoptogenic cytochrome c release channel activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015284
name: fructose uniporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: fructose(out) = fructose(in)." [TC:2.A.1.1.13]
GO:0015288
name: porin activity
def: "Catalysis of the transfer of substances, sized less than 1000 Da, from one side of the membrane to the other. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820, TC:1.B.1.-.-]
GO:0015289
name: pore-forming toxin activity
def: "OBSOLETE. Catalysis of the transport of electrolytes and other small molecules across a cell membrane. They are synthesized by one cell and secreted for insertion into the membrane of another cell where they form transmembrane pores. They may exert their toxic effects by allowing the free flow of electrolytes and other small molecules across the membrane, or they may allow entry into the target cell cytoplasm of a toxin protein that ultimately kills the cell." [PMID:10839820]
GO:0015291
name: secondary active transmembrane transporter activity
def: "Catalysis of the transfer of a solute from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy, not direct ATP coupling. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729, PMID:10839820]
GO:0015292
name: uniporter activity
def: "Catalysis of the transport of a single molecular species across a membrane; transport is independent of the movement of any other molecular species." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820]
GO:0015293
name: symporter activity
def: "Enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820]
GO:0015294
name: solute:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + cation(out) = solute(in) + cation(in)." [GOC:ai]
GO:0015295
name: solute:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + H+(out) = solute(in) + H+(in)." [GOC:ai]
GO:0015296
name: anion:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: anion(out) + cation(out) = anion(in) + cation(in)." [TC:2.A.1.14.-]
GO:0015297
name: antiporter activity
def: "Enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported in opposite directions in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy. The reaction is: solute A(out) + solute B(in) = solute A(in) + solute B(out)." [GOC:mtg_transport, ISBN:0815340729, PMID:10839820]
GO:0015298
name: solute:cation antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + cation(in) = solute(in) + cation(out)." [GOC:ai]
GO:0015299
name: solute:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + H+(in) = solute(in) + H+(out)." [GOC:ai]
GO:0015301
name: anion:anion antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: anion A(out) + anion B(in) = anion A(in) + anion B(out)." [GOC:ai, GOC:mtg_transport]
GO:0015303
name: galactose, glucose uniporter activity
def: "OBSOLETE. Catalysis of the reaction: galactose or glucose(out) = galactose or glucose(in)." [TC:2.A.1.1.6]
GO:0015304
name: glucose uniporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose(out) = glucose(in)." [TC:2.A.1.1.12, TC:2.A.1.1.4, TC:2.A.1.1.6]
GO:0015305
name: lactose, galactose:proton symporter activity
def: "OBSOLETE. Catalysis of the reaction: (lactose or galactose)(out) + H+(out) = (lactose or galactose)(in) + H+(in)." [TC:2.A.1.1.9]
GO:0015306
name: sialate:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sialate(out) + cation(out) = sialate(in) + cation(in)." [TC:2.A.1.14.10]
GO:0015307
name: drug:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + drug(in) = H+(in) + drug(out)." [TC:2.A.1.2.-, TC:2.A.1.3.-]
GO:0015308
name: amiloride:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + amiloride(in) = H+(in) + amiloride(out)." [TC:2.A.1.2.1]
GO:0015309
name: cycloheximide:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + cycloheximide(in) = H+(in) + cycloheximide(out)." [TC:2.A.1.2.2]
GO:0015310
name: benomyl:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + benomyl(in) = H+(in) + benomyl(out)." [TC:2.A.1.2.6]
GO:0015311
name: monoamine:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + monoamine(in) = H+(in) + monoamine(out)." [TC:2.A.1.2.11, TC:2.A.1.2.12]
GO:0015312
name: polyamine:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + polyamine(in) = H+(in) + polyamine(out)." [TC:2.A.1.2.16]
GO:0015313
name: fluconazole:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + fluconazole(in) = H+(in) + fluconazole(out)." [TC:2.A.1.2.17]
GO:0015314
name: aminotriazole:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + aminotriazole(in) = H+(in) + aminotriazole(out)." [TC:2.A.1.3.1]
GO:0015315
name: organophosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: organophosphate(out) + inorganic phosphate(in) = organophosphate(in) + inorganic phosphate(out)." [TC:2.A.1.4.-]
GO:0015316
name: nitrite/nitrate porter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015317
name: phosphate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: phosphate(out) + H+(out) = phosphate(in) + H+(in)." [TC:2.A.1.9.-]
GO:0015318
name: inorganic solute uptake transmembrane transporter activity
def: "Catalysis of the transfer of an inorganic solute from the outside of a cell to the inside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015319
name: sodium:inorganic phosphate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + inorganic phosphate(out) = Na+(in) + inorganic phosphate(in)." [TC:2.A.1.14.6]
GO:0015320
name: phosphate ion carrier activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: phosphate (cyt) + H+ (cyt) = phosphate (mitochondrion) + H+ (mitochondrion)." [GOC:curators, TC:2.A.29.4.1]
GO:0015321
name: sodium-dependent phosphate transmembrane transporter activity
def: "Catalysis of the transfer of phosphate (PO4 3-) ions from one side of a membrane to the other, requiring sodium ions." [GOC:jl]
GO:0015322
name: secondary active oligopeptide transmembrane transporter activity
def: "Catalysis of the transfer of an oligopeptide or oligopeptides from one side of the membrane to the other, up the solute's concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction." [GOC:mtg_transport]
GO:0015323
name: type V protein secretor activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015324
name: peptide-acetyl-CoA secondary active transmembrane transporter activity
def: "Catalysis of the transfer of peptide-acetyl-CoA from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0015325
name: acetyl-CoA:CoA antiporter activity
def: "Catalysis of the reaction: acetyl-CoA(out) + CoA(in) = acetyl-CoA(in) + CoA(out)." [TC:2.A.1.25.1]
GO:0015327
name: cystine:glutamate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: cystine(out) + glutamate(in) = cystine(in) + glutamate(out)." [TC:2.A.3.8.5]
GO:0015328
name: cystine secondary active transmembrane transporter activity
def: "Catalysis of the transfer of cystine from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0015330
name: high-affinity glutamine transmembrane transporter activity
def: "Catalysis of the high affinity transfer of glutamine from one side of the membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0015331
name: asparagine/glutamine permease activity
def: "OBSOLETE. Catalysis of the stereospecific transfer of asparagine or glutamine across a biological membrane." [GOC:ai]
GO:0015332
name: leucine/valine/isoleucine permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015333
name: peptide:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: peptide(out) + H+(out) = peptide(in) + H+(in). Catalysis of the transfer of a peptide from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by hydrogen ion movement." [GOC:mtg_transport, TC:2.A.17.-.-]
GO:0015334
name: high-affinity oligopeptide transporter activity
def: "Catalysis of the transfer of oligopeptide from one side of the membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations. Oligopeptides are molecules that contain a small number (2 to 20) of amino-acid residues connected by peptide linkages." [GOC:mtg_transport]
GO:0015335
name: heavy metal ion:hydrogen symporter activity
def: "OBSOLETE. Catalysis of the reaction: Me2+(out) + H+(out) = Me2+(in) + H+(in), where Me2+ is Fe2+, Zn2+, Mn2+, Cu2+, Cd2+, Co2+, Ni2+ or Pb2+." [TC:2.A.55.2.1, TC:2.A.55.2.2]
GO:0015336
name: high affinity metal ion uptake transporter activity
def: "OBSOLETE. Catalysis of the reaction: Me2+(out) + H+(out) = Me2+(in) + H+(in). Me can be Fe2+, Mn2+, Zn2+, Cu2+, Cd2+, Ni2+ or Co2+." [TC:2.A.55.1.1]
GO:0015337
name: low affinity metal ion uptake transporter activity
def: "OBSOLETE. Catalysis of the reaction: Me2+(out) + H+(out) = Me2+(in) + H+(in). Me can be Mn2+ or Cu2+." [TC:2.A.55.1.2]
GO:0015339
name: cobalt, zinc uptake permease activity
def: "OBSOLETE. Catalysis of the reaction: (Zn2+ or Co2+)(out) = (Zn2+ or Co2+)(in). The activity is driven by proton motive force, possibly by proton symport." [TC:2.A.4.2.1]
GO:0015340
name: zinc, cadmium uptake permease activity
def: "OBSOLETE. Catalysis of the reaction: (Zn2+ or Cd2+)(out) = (Zn2+ or Cd2+)(in). The activity is driven by proton motive force, possibly by proton symport." [TC:2.A.4.2.2]
GO:0015341
name: zinc efflux active transmembrane transporter activity
def: "Catalysis of the transfer of a zinc ion or zinc ions from the inside of the cell to the outside of the cell across a membrane: Zn2+(out) = Zn2+(in). The activity is driven by proton motive force." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.4.1.4, TC:2.A.4.2.3]
GO:0015342
name: zinc, iron permease activity
def: "OBSOLETE. Catalysis of the reaction: (Zn2+ or Fe2+)(out) = (Zn2+ or Fe2+)(in), probably powered by proton motive force." [TC:2.A.5.-.-]
GO:0015343
name: siderophore transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: siderophore-iron(out) + H+(out) = siderophore-iron(in) + H+(in)." [TC:2.A.1.16.-]
GO:0015344
name: siderophore uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: siderophore-iron(ferrioxamine)(out) + H+(out) = siderophore-iron(ferrioxamine)(in) + H+(in)." [TC:2.A.1.16.1]
GO:0015345
name: ferric enterobactin:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ferric enterobactin(out) + H+(out) = ferric enterobactin(in) + H+(in)." [TC:2.A.1.16.2]
GO:0015346
name: ferric triacetylfusarinine C:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ferric triacetylfusarinine C(out) + H+(out) = ferric triacetylfusarinine C(in) + H+(in)." [TC:2.A.1.16.3]
GO:0015347
name: sodium-independent organic anion transmembrane transporter activity
def: "Catalysis of the transfer of organic anions from one side of a membrane to the other, in a sodium independent manner." [GOC:go_curators]
GO:0015348
name: prostaglandin/thromboxane transporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015349
name: thyroid hormone transmembrane transporter activity
def: "Catalysis of the transfer of thyroid hormones from one side of the membrane to the other. Thyroid hormone are any of the compounds secreted by the thyroid gland, largely thyroxine and triiodothyronine." [GOC:ai]
GO:0015350
name: methotrexate transporter activity
def: "Enables the directed movement of methotrexate, 4-amino-10-methylformic acid, into, out of or within a cell, or between cells. Methotrexate is a folic acid analogue and a potent competitive inhibitor of dihydrofolate reductase." [GOC:ai]
GO:0015351
name: bilirubin secondary active transmembrane transporter activity
def: "Catalysis of the transfer of bilirubin from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mtg_transport]
GO:0015352
name: secondary active sterol transmembrane transporter activity
def: "Catalysis of the transfer of sterol from one side of the membrane to the other, up the solute's concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction." [GOC:mtg_transport]
GO:0015355
name: secondary active monocarboxylate transmembrane transporter activity
def: "Catalysis of the movement of a monocarboxylate, any compound containing a single carboxyl group (COOH or COO-), by uniport, symport or antiport across a membrane by a carrier-mediated mechanism." [GOC:bf, GOC:jl]
GO:0015356
name: monocarboxylate (lactate, pyruvate, mevalonate) uptake/efflux porter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015358
name: amino acid/choline transmembrane transporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015360
name: acetate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: acetate(out) + H+(out) = acetate(in) + H+(in)." [TC:2.A.44.4.1]
GO:0015361
name: low-affinity sodium:dicarboxylate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: dicarboxylate(out) + Na+(out) = dicarboxylate(in) + Na+(in). In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [TC:2.A.47.1.1]
GO:0015362
name: high-affinity sodium:dicarboxylate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: dicarboxylate(out) + Na+(out) = dicarboxylate(in) + Na+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [TC:2.A.47.1.4]
GO:0015363
name: dicarboxylate (succinate/fumarate/malate) antiporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015364
name: dicarboxylate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: dicarboxylate(out) + inorganic phosphate(in) = dicarboxylate(in) + inorganic phosphate(out)." [TC:2.A.29.2.3]
GO:0015366
name: malate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: malate(out) + H+(out) = malate(in) + H+(in)." [TC:2.A.16.2.1]
GO:0015367
name: oxoglutarate:malate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: oxoglutarate(out) + malate(in) = oxoglutarate(in) + malate(out)." [TC:2.A.29.2.1]
GO:0015368
name: calcium:cation antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Ca2+(in) + cation(out) = Ca2+(out) + cation(in)." [TC:2.A.19.-.-]
GO:0015369
name: calcium:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Ca2+(in) + H+(out) = Ca2+(out) + H+(in)." [TC:2.A.19.2.-]
GO:0015370
name: solute:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + Na+(out) = solute(in) + Na+(in)." [GOC:ai]
GO:0015371
name: galactose:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: galactose(out) + Na+(out) = glucose(in) + Na+(in)." [TC:2.A.21.3.-]
GO:0015372
name: glutamate/aspartate:sodium symporter activity
def: "OBSOLETE. Catalysis of the reaction: (glutamate or aspartate)(out) + Na+(out) = (glutamate or aspartate)(in) + Na+(in)." [TC:2.A.23.1.1]
GO:0015373
name: monovalent anion:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: monovalent anion(out) + Na+(out) = monovalent anion(in) + Na+(in)." [TC:2.A.21.5.-]
GO:0015374
name: neutral, basic amino acid:sodium:chloride symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: neutral/basic amino acid(out) + Na+(out) + Cl-(out) = neutral/basic amino acid(in) + Na+(in) + Cl-(in)." [TC:2.A.22.2.3]
GO:0015375
name: glycine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glycine(out) + Na+(out) = glycine(in) + Na+(in)." [GOC:ai]
GO:0015376
name: betaine/GABA:sodium symporter activity
def: "OBSOLETE. Catalysis of the reaction: (betaine or gamma-aminobutyric acid)(out) + Na+(out) = (betaine or gamma-aminobutyric acid)(in) + Na+(in)." [TC:2.A.22.3.1]
GO:0015377
name: cation:chloride symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: cation(out) + Cl-(out) = cation(in) + Cl-(in)." [TC:2.A.30.-.-]
GO:0015378
name: sodium:chloride symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + Cl-(out) = Na+(in) + Cl-(in)." [TC:2.A.30.4.-]
GO:0015379
name: potassium:chloride symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: K+(out) + Cl-(out) = K+(in) + Cl-(in)." [TC:2.A.30.5.-]
GO:0015381
name: high-affinity sulfate transmembrane transporter activity
def: "Catalysis of the secondary active high affinity transfer of sulfate from one side of the membrane to the other. Secondary active transport is catalysis of the transfer of a solute from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport. In high-affinity transport the transporter is able to bind thesolute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0015382
name: sodium:sulfate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + Na+(out) = sulfate(in) + Na+(in)." [TC:2.A.47.1.2]
GO:0015383
name: sulfate:bicarbonate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + bicarbonate(in) = sulfate(in) + bicarbonate(out)." [TC:2.A.53.2.2]
GO:0015385
name: sodium:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Na+(out) + H+(in) = Na+(in) + H+(out)." [TC:2.A.35.1.1, TC:2.A.36.-.-]
GO:0015386
name: potassium:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: K+(in) + H+(out) = K+(out) + H+(in)." [TC:2.A.37.-.-]
GO:0015387
name: potassium:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: K+(out) + H+(out) = K+(in) + H+(in)." [TC:2.A.38.-.-]
GO:0015388
name: potassium uptake transmembrane transporter activity
def: "Catalysis of the transfer of potassium from the outside of a cell to the inside of the cell across a membrane: K+(out) + energy = K+(in)." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.72.2.-]
GO:0015389
name: pyrimidine- and adenine-specific:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: (pyrimidine nucleoside or adenine)(out) + Na+(out) = (pyrimidine nucleoside or adenine)(in) + Na+(in)." [TC:2.A.41.2.3]
GO:0015390
name: purine-specific nucleoside:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: purine(out) + Na+(out) = nucleoside(in) + Na+(in)." [TC:2.A.41.2.1]
GO:0015391
name: nucleobase:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: nucleobase(out) + cation(out) = nucleobase(in) + cation(in)." [GOC:ai]
GO:0015393
name: uracil/uridine permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015394
name: uridine:hydrogen ion symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: uridine(out) + H+(out) = uridine(in) + H+(in)." [GOC:mtg_transport]
GO:0015395
name: nucleoside transmembrane transporter activity, down a concentration gradient
def: "Catalysis of the transfer of a nucleoside, from one side of a membrane to the other, down the concentration gradient." [PMID:10353709, PMID:11749958, PMID:12446811]
GO:0015398
name: high-affinity secondary active ammonium transmembrane transporter activity
def: "Catalysis of the transfer of ammonium from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mtg_transport]
GO:0015399
name: primary active transmembrane transporter activity
def: "Catalysis of the transfer of a solute from one side of the membrane to the other, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is powered by a primary energy source, directly using ATP. Primary energy sources known to be coupled to transport are chemical, electrical and solar sources." [GOC:mtg_transport, ISBN:0815340729, TC:3.-.-.-.-]
GO:0015400
name: low-affinity secondary active ammonium transmembrane transporter activity
def: "Catalysis of the transfer of ammonium from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:mtg_transport, ISBN:0815340729]
GO:0015401
name: urea:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: urea(out) + Na+(out) = urea(in) + Na+(in)." [TC:2.A.21.6.1]
GO:0015403
name: thiamine uptake transmembrane transporter activity
def: "Catalysis of the transfer of thiamine from the outside of a cell to the inside across a membrane." [GOC:mtg_transport]
GO:0015405
name: P-P-bond-hydrolysis-driven transmembrane transporter activity
def: "Primary active transport of a solute across a membrane, driven by the hydrolysis of the diphosphate bond of inorganic pyrophosphate, ATP, or another nucleoside triphosphate. The transport protein may or may not be transiently phosphorylated, but the substrate is not phosphorylated. Primary active transport is catalysis of the transport of a solute across a membrane, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is driven by a primary energy source." [GOC:mtg_transport, ISBN:0815340729, TC:3.A.-.-.-]
GO:0015406
name: ABC-type uptake permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0015407
name: monosaccharide-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + monosaccharide(out) = ADP + phosphate + monosaccharide(in). Ribose, xylose, arabinose, galactose and methylgalactoside are imported." [EC:3.6.3.17]
GO:0015408
name: ferric-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Fe3+(out) = ADP + phosphate + Fe3+(in)." [EC:3.6.3.30]
GO:0015410
name: manganese-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Mn2+(out) = ADP + phosphate + Mn2+(in)." [EC:3.6.3.35]
GO:0015411
name: taurine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + taurine(out) = ADP + phosphate + taurine(in)." [EC:3.6.3.36]
GO:0015412
name: molybdate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + molybdate(out) = ADP + phosphate + molybdate(in)." [EC:3.6.3.29]
GO:0015413
name: nickel-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Ni2+(out) = ADP + phosphate + Ni2+(in)." [EC:3.6.3.24]
GO:0015414
name: nitrate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + nitrate(out) = ADP + phosphate + nitrate(in)." [EC:3.6.3.26]
GO:0015415
name: phosphate ion transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + phosphate(out) = ADP + phosphate + phosphate(in)." [EC:3.6.3.27]
GO:0015416
name: organic phosphonate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + phosphonate(out) = ADP + phosphate + phosphonate(in). A phosphonate is any salt, anion, or ester of phosphonic acid (HPO(OH)2)." [EC:3.6.3.28]
GO:0015417
name: polyamine-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + polyamine(out) = ADP + phosphate + polyamine(in)." [EC:3.6.3.31]
GO:0015418
name: quaternary-ammonium-compound-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + quaternary amine(out) = ADP + phosphate + quaternary amine(in)." [EC:3.6.3.32]
GO:0015419
name: sulfate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + sulfate(out) = ADP + phosphate + sulfate(in)." [EC:3.6.3.25]
GO:0015420
name: cobalamin-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + cob(III)alamin(out) = ADP + phosphate + cob(III)alamin(in). Cobalamin is also known as vitamin B12." [EC:3.6.3.33]
GO:0015421
name: oligopeptide-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + oligopeptide(out) = ADP + phosphate + oligopeptide(in)." [EC:3.6.3.23]
GO:0015422
name: oligosaccharide-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + oligosaccharide(out) = ADP + phosphate + oligosaccharide(in)." [EC:3.6.3.18]
GO:0015423
name: maltose-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + maltose(out) = ADP + phosphate + maltose(in)." [EC:3.6.3.19]
GO:0015424
name: amino acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + amino acid(out/in) = ADP + phosphate + amino acid(in/out)." [GOC:ai, GOC:mah]
GO:0015425
name: nonpolar-amino acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + nonpolar amino acid(out) = ADP + phosphate + nonpolar amino acid(in)." [EC:3.6.3.22]
GO:0015426
name: polar-amino acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + polar amino acid(out) = ADP + phosphate + polar amino acid(in)." [EC:3.6.3.21]
GO:0015427
name: ABC-type efflux porter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0015428
name: type I protein secretor activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015429
name: peroxisomal fatty acyl transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015430
name: glycerol-3-phosphate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + glycerol-3-phosphate(out) = ADP + phosphate + glycerol-3-phosphate(in)." [EC:3.6.3.20]
GO:0015431
name: glutathione S-conjugate-exporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + glutathione S-conjugate(in) -> ADP + phosphate + glutathione S-conjugate(out)." [GOC:jl, PMID:1455517]
GO:0015432
name: bile acid-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: bile acid(in) + ATP + H2O -> bile acid(out) + ADP + phosphate." [TC:3.A.1.207.2]
GO:0015433
name: peptide antigen-transporting ATPase activity
def: "Catalysis of the reaction: peptide antigen(in) + ATP = peptide antigen(out) + ADP + phosphate." [TC:3.A.1.209.1]
GO:0015434
name: cadmium-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Cd (cytosol) = ADP + phosphate + Cd (vacuole)." [EC:3.6.3.46]
GO:0015435
name: ABC-type efflux permease activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0015436
name: capsular-polysaccharide-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + capsular polysaccharide(in) = ADP + phosphate + capsular polysaccharide(out)." [EC:3.6.3.38]
GO:0015437
name: lipopolysaccharide-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + lipopolysaccharide(in) = ADP + phosphate + lipopolysaccharide(out)." [EC:3.6.3.39]
GO:0015438
name: teichoic-acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + teichoic acid(in) = ADP + phosphate + teichoic acid(out)." [EC:3.6.3.40]
GO:0015439
name: heme-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + heme(in) = ADP + phosphate + heme(out)." [EC:3.6.3.41]
GO:0015440
name: peptide-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + peptide(in) = ADP + phosphate + peptide(out). Peptides exported include alpha-hemolysin, cyclolysin, colicin V and siderophores from Gram-negative bacteria, and bacteriocin, subtilin, competence factor and pediocin from Gram-positive bacteria." [EC:3.6.3.43]
GO:0015441
name: beta-glucan-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + beta-glucan(in) = ADP + phosphate + beta-glucan(out)." [EC:3.6.3.42]
GO:0015442
name: hydrogen-/sodium-translocating ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + (Na+ or H+)(in) = ADP + phosphate + (Na+ or H+)(out)." [TC:3.A.2.-.-]
GO:0015443
name: sodium-transporting two-sector ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + Na+(in) = ADP + phosphate + Na+(out)." [EC:3.6.3.15]
GO:0015444
name: magnesium-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Mg2+(out) -> ADP + phosphate + Mg2+(in)." [EC:3.6.3.2]
GO:0015445
name: silver-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Ag+(in) -> ADP + phosphate + Ag+(out)." [EC:3.6.3.53]
GO:0015446
name: arsenite-transmembrane transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + arsenite(in) = ADP + phosphate + arsenite(out)." [EC:3.6.3.16]
GO:0015447
name: type II protein secretor activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015448
name: type III protein (virulence-related) secretor activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015449
name: type IV protein (DNA-protein) secretor activity
def: "OBSOLETE." [GOC:mtg_transport]
GO:0015450
name: P-P-bond-hydrolysis-driven protein transmembrane transporter activity
def: "Primary active carrier-mediated transport of a protein across a membrane, driven by the hydrolysis of the diphosphate bond of inorganic pyrophosphate, ATP, or another nucleoside triphosphate. The transport protein may or may not be transiently phosphorylated, but the substrate is not phosphorylated." [GOC:mtg_transport, ISBN:0815340729]
GO:0015451
name: decarboxylation-driven active transmembrane transporter activity
def: "Primary active transport of a solute across a membrane driven by decarboxylation of a cytoplasmic substrate. Primary active transport is catalysis of the transport of a solute across a membrane, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is driven by a primary energy source." [GOC:mtg_transport, ISBN:0815340729, TC:3.B.-.-.-]
GO:0015452
name: methyl transfer-driven active transmembrane transporter activity
def: "Primary active transport of a solute across a membrane driven by a methyl transfer reaction. Primary active transport is catalysis of the transport of a solute across a membrane, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is driven by a primary energy source." [GOC:mtg_transport, ISBN:0815340729, TC:3.C.-.-.-]
GO:0015453
name: oxidoreduction-driven active transmembrane transporter activity
def: "Primary active transport of a solute across a membrane, driven by exothermic flow of electrons from a reduced substrate to an oxidized substrate. Primary active transport is catalysis of the transport of a solute across a membrane, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is driven by a primary energy source." [GOC:mtg_transport, ISBN:0815340729, TC:3.D.-.-.-]
GO:0015454
name: light-driven active transmembrane transporter activity
def: "Primary active transport of a solute across a membrane, driven by light. Primary active transport is catalysis of the transport of a solute across a membrane, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction and is driven by a primary energy source." [GOC:mtg_transport, ISBN:0815340729, TC:3.E.-.-.-]
GO:0015459
name: potassium channel regulator activity
def: "Modulates the activity of a potassium channel." [GOC:mah]
GO:0015461
name: endosomal oligosaccharide transporter
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015462
name: protein-transmembrane transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + protein(out) = ADP + phosphate + protein(in)." [EC:3.6.1.3, GOC:jl]
GO:0015464
name: acetylcholine receptor activity
def: "Combining with acetylcholine and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0015465
name: lysin activity
def: "OBSOLETE. An agent that can lyse cells." [ISBN:0198547684]
GO:0015466
name: autolysin activity
def: "OBSOLETE. An agent that can lyse the cell in which it is synthesized." [GOC:ma]
GO:0015467
name: G-protein activated inward rectifier potassium channel activity
def: "Catalysis of the transmembrane transfer of a potassium ion by an inwardly-rectifying voltage-gated channel, where the inward rectification is due to a voltage-dependent block of the channel pore by a G protein. An inwardly rectifying current-voltage relation is one where at any given driving force the inward flow of K+ ions exceeds the outward flow for the opposite driving force." [GOC:cb, GOC:mah]
GO:0015468
name: colicin
def: "OBSOLETE. Plasmid-encoded bacteriocins which are produced by enteric bacteria. Exert a lethal effect on other bacteria including E. coli strains that lack the Col plasmid. Bind to a cell surface receptor and are transported into the periplasm via an energy-dependent process involving a TonB- or TolA-dependent hetero-oligomeric protein complex. Some colicins kill their target cell by inserting into the cytoplasmic membrane where they form voltage-sensitive (trans-negative) channels that depolarize and deenergize the cell, and thereby kill it." [TC:1.C.1.-.-]
GO:0015469
name: channel-forming toxin activity
def: "OBSOLETE. A toxin that exerts its effects by forming a channel in a membrane that allows the unregulated passage of substances into and out of the cell." [GOC:ai]
GO:0015470
name: bacteriocin activity
def: "OBSOLETE. Polypeptide antibiotic secreted by bacteria and able to kill bacteria of susceptible strains after absorption by specific cell surface receptor." [ISBN:0198547684]
GO:0015471
name: nucleoside-specific channel forming porin activity
def: "Enables the energy independent passage of nucleoside, sized less than 1000 Da, across a membrane. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport]
GO:0015472
name: fimbrium-specific chaperone activity
def: "OBSOLETE. Assists in the correct assembly of fimbria, extracellular organelles that are used to attach a bacterial cell to a surface, but is not a component of the fimbrium when performing its normal biological function." [GOC:jl, GOC:rb, PMID:7906046]
GO:0015473
name: fimbrial usher porin activity
def: "A porin that acts in the assembly of fimbria together with fimbrial chaperone." [TC:1.B.11.-.-]
GO:0015474
name: autotransporter activity
def: "Transports a passenger protein from the periplasm to the external milieu; the passenger protein and the porin are the N- and C-terminal regions of the same protein, respectively." [GOC:mtg_transport, ISBN:0815340729, TC:1.B.12.-.-]
GO:0015475
name: adhesin autotransporter activity
def: "Catalysis of the transfer of adhesin from the periplasm to the external milieu; the adhesin and the porin are the N- and C-terminal regions of the same protein, respectively." [GOC:mtg_transport, ISBN:0815340729]
GO:0015476
name: hemaglutinin autotransporter activity
def: "Catalysis of the transfer of hemaglutinin from the periplasm to the external milieu; the hemaglutinin and the porin are the N- and C-terminal regions of the same protein, respectively." [GOC:mtg_transport, ISBN:0815340729]
GO:0015477
name: receptor porin activity
def: "OBSOLETE. A porin of the bacterial outer membrane that forms transmembrane pores and transports relatively large molecules from the external milieu to the periplasm in an energized process. Energizing of transport across the outer membrane requires a heterotrimeric complex of proteins, the TonB-ExbB-ExbD complex, or in some cases, the TolA-TolQ-TolR complex. Energizing requires proton motive force across the cytoplasmic membrane." [GOC:mtg_transport, ISBN:0815340729, TC:1.B.13.-.-]
GO:0015478
name: oligosaccharide transporting porin activity
def: "Catalysis of the transfer of oligosaccharide, sized less than 1000 Da, from one side of the membrane to the other. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport]
GO:0015479
name: outer membrane exporter porin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015480
name: secretin (sensu Bacteria)
def: "OBSOLETE. Secretins are Gram-negative bacterial outer membrane proteins that form multimeric pores through which macromolecules, usually proteins, can pass. Form homomultimeric ring structures, 10-20 subunits per complex, with large central pores (inner diameters of 5-10 nm)." [TC:1.B.22.-.-]
GO:0015481
name: maltose transporting porin activity
def: "Catalysis of the transfer of maltose from one side of the membrane to the other. Maltose is the disaccharide 4-O-alpha-D-glucopyranosyl-D-glucopyranose, an intermediate in the enzymatic breakdown of glycogen and starch. This transporter is a porin so enables the energy independent passage of substances, sized less than 1000 Da, across a membrane. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [GOC:mtg_transport]
GO:0015482
name: voltage-gated anion channel porin activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0015483
name: long-chain fatty acid transporting porin activity
def: "Catalysis of the transfer of long-chain fatty acids from one side of the membrane to the other. A long-chain fatty acid is a fatty acid with a chain length between C13 and C22. This transporter is a porin and so enables the energy independent passage of substances, sized less than 1000 Da, across a membrane. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria." [CHEBI:15904, GOC:mtg_transport]
GO:0015484
name: hemolysin activity
def: "OBSOLETE. Any substance that causes the lysis of red blood cells." [ISBN:0198506732]
GO:0015485
name: cholesterol binding
def: "Interacting selectively and non-covalently with cholesterol (cholest-5-en-3-beta-ol); the principal sterol of vertebrates and the precursor of many steroids, including bile acids and steroid hormones." [GOC:jl, ISBN:0198506732]
GO:0015486
name: glycoside-pentoside-hexuronide:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: (glycoside, pentoside or hexuronide)(out) + monovalent cation(out) = (glycoside, pentoside or hexuronide)(in) + monovalent cation(in). The cation is Na+, Li+ or H+." [TC:2.A.2.-.-]
GO:0015487
name: melibiose:monovalent cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: melibiose(out) + monovalent cation(out) = melibiose(in) + monovalent cation(in)." [TC:2.A.2.1.1]
GO:0015488
name: glucuronide:monovalent cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucuronide(out) + monovalent cation(out) = glucuronide(in) + monovalent cation(in)." [TC:2.A.2.3.1]
GO:0015489
name: putrescine transmembrane transporter activity
def: "Catalysis of the transfer of putrescine from one side of the membrane to the other. Putrescine is 1,4-diaminobutane, the polyamine formed by decarboxylation of ornithine and the metabolic precursor of spermidine and spermine." [GOC:ai]
GO:0015490
name: cadaverine transmembrane transporter activity
def: "Catalysis of the transfer of cadaverine, 1,5-pentanediamine, from one side of the membrane to the other." [GOC:ai]
GO:0015491
name: cation:cation antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: cation A(out) + cation B(in) = cation A(in) + cation B(out)." [GOC:ai]
GO:0015492
name: phenylalanine:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: phenylalanine(out) + H+(out) = phenylalanine(in) + H+(in)." [TC:2.A.3.1.1]
GO:0015493
name: lysine:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: lysine(out) + H+(out) = lysine(in) + H+(in)." [TC:2.A.3.1.2]
GO:0015494
name: aromatic amino acid:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: aromatic amino acid(out) + H+(out) = aromatic amino acid(in) + H+(in)." [TC:2.A.3.1.3]
GO:0015495
name: gamma-aminobutyric acid:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: gamma-aminobutyric acid(out) + H+(out) = gamma-aminobutyric acid(in) + H+(in)." [TC:2.A.18.5.1, TC:2.A.3.1.4, TC:2.A.3.4.2]
GO:0015496
name: putrescine:ornithine antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: putrescine(out) + ornithine(in) = putrescine(in) + ornithine(out)." [TC:2.A.3.2.1]
GO:0015498
name: pantothenate:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: pantothenate(out) + Na+(out) = pantothenate(in) + Na+(in)." [TC:2.A.21.1.1]
GO:0015499
name: formate transmembrane transporter activity
def: "Catalysis of the transfer of formate from one side of the membrane to the other. Formate is also known as methanoate, the anion HCOO- derived from methanoic (formic) acid." [GOC:ai]
GO:0015500
name: threonine/serine:sodium symporter activity
def: "OBSOLETE. Catalysis of the reaction: (threonine or serine)(out) + Na+(out) = (threonine or serine)(in) + Na+(in)." [TC:2.A.23.4.1]
GO:0015501
name: glutamate:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glutamate(out) + Na+(out) = glutamate(in) + Na+(in)." [TC:2.A.27.1.1]
GO:0015503
name: glutathione-regulated potassium exporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: K+(in) + H+(out) = K+(out) + H+(in), where glutathione maintains the closed state." [PMID:11053405, TC:2.A.37.1.1, TC:2.A.37.1.2]
GO:0015504
name: cytosine:hydrogen ion symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: cytosine(out) + H+(out) = cytosine(in) + H+(in)." [TC:2.A.39.1.1]
GO:0015505
name: uracil:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: uracil(out) + cation(out) = uracil(in) + cation(in)." [GOC:mtg_transport]
GO:0015506
name: nucleoside:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: nucleoside(out) + H+(out) = nucleoside(in) + H+(in)." [TC:2.A.1.10.1, TC:2.A.41.1.1]
GO:0015507
name: hydroxy/aromatic amino acid permease activity
def: "OBSOLETE. Permease for hydroxy and aromatic amino acids." [GOC:ai]
GO:0015513
name: nitrite uptake transmembrane transporter activity
def: "Catalysis of the transfer of nitrite from the outside of a cell to the inside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015514
name: nitrite efflux transmembrane transporter activity
def: "Catalysis of the transfer of nitrite from the inside of the cell to the outside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015515
name: citrate:succinate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: citrate(out) + succinate(in) = citrate(in) + succinate(out)." [TC:2.A.47.3.2]
GO:0015516
name: tartrate:succinate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: tartrate(out) + succinate(in) = tartrate(in) + succinate(out)." [TC:2.A.47.3.3]
GO:0015517
name: galactose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: galactose(out) + H+(out) = galactose(in) + H+(in)." [TC:2.A.1.1.1, TC:2.A.1.1.9]
GO:0015518
name: arabinose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: arabinose(out) + H+(out) = arabinose(in) + H+(in)." [TC:2.A.1.1.2]
GO:0015519
name: D-xylose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: D-xylose(out) + H+(out) = D-xylose(in) + H+(in)." [TC:2.A.1.1.3]
GO:0015520
name: tetracycline:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + tetracycline(in) = H+(in) + tetracycline(out)." [TC:2.A.1.2.4]
GO:0015521
name: bicyclomycin/sulfathiazole:hydrogen antiporter activity
def: "OBSOLETE. Catalysis of the reaction: (bicyclomycin or sulfathiazole)(in) + H+(out) = (bicyclomycin or sulfathiazole)(out) + H+(in)." [TC:2.A.1.2.7]
GO:0015522
name: hydrophobic uncoupler:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: hydrophobic uncoupler(in) + H+(out) = hydrophobic uncoupler(out) + H+(in). Hydrophobic uncouplers include CCCP, benzalkonium and SDS." [TC:2.A.1.2.9]
GO:0015523
name: arabinose efflux transmembrane transporter activity
def: "Catalysis of the transfer of arabinose from the inside of the cell to the outside of the cell across a membrane: H+(out) + arabinose(in) = H+(in) + arabinose(out)." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.1.2.14]
GO:0015524
name: L-arabinose/beta-D-thiogalactopyranoside:hydrogen antiporter activity
def: "OBSOLETE. Catalysis of the reaction: H+(out) + (L-arabinose or beta-D-thiogalactopyranoside)(in) = H+(in) + (L-arabinose or beta-D-thiogalactopyranoside)(out)." [TC:2.A.1.2.15]
GO:0015525
name: carbonyl cyanide m-chlorophenylhydrazone/nalidixic acid/organomercurials:hydrogen antiporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015526
name: hexose-phosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: hexose phosphate(out) + inorganic phosphate(in) = hexose phosphate(in) + inorganic phosphate(out)." [TC:2.A.1.4.1]
GO:0015527
name: glycerol-phosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glycerol phosphate(out) + inorganic phosphate(in) = glycerol phosphate(in) + inorganic phosphate(out)." [TC:2.A.1.4.3]
GO:0015528
name: lactose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: lactose(out) + H+(out) = lactose(in) + H+(in)." [TC:2.A.1.1.9, TC:2.A.1.5.1]
GO:0015529
name: raffinose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: raffinose(out) + H+(out) = raffinose(in) + H+(in)." [TC:2.A.1.5.2]
GO:0015530
name: shikimate transmembrane transporter activity
def: "Catalysis of the transfer of shikimate from one side of the membrane to the other. Shikimate is an important intermediate in the biosynthesis of aromatic amino acids." [GOC:ai]
GO:0015531
name: citrate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: citrate(out) + H+(out) = citrate(in) + H+(in)." [TC:2.A.1.6.1]
GO:0015532
name: alpha-ketoglutarate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: alpha-ketoglutarate(out) + H+(out) = alpha-ketoglutarate(in) + H+(in)." [TC:2.A.1.6.2]
GO:0015533
name: shikimate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: shikimate(out) + H+(out) = shikimate(in) + H+(in)." [TC:2.A.1.6.6]
GO:0015534
name: proline/glycine/betaine:hydrogen/sodium symporter activity
def: "OBSOLETE. Catalysis of the reaction: (proline, glycine or betaine)(out) + (H+ or Na+)(out) = (proline, glycine or betaine)(in) + (H+ or Na+)(in)." [TC:2.A.1.6.4]
GO:0015535
name: fucose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: fucose(out) + H+(out) = fucose(in) + H+(in)." [TC:2.A.1.7.1]
GO:0015537
name: xanthosine:hydrogen ion symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: xanthosine(out) + H+(out) = xanthosine(in) + H+(in)." [TC:2.A.1.10.2]
GO:0015538
name: sialic acid:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sialate(out) + H+(out) = sialate(in) + H+(in)." [TC:2.A.1.12.1]
GO:0015539
name: hexuronate:cation symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: hexuronate(out) + cation(out) = hexuronate(in) + cation(in). The hexuronate may be glucuronate or galacturonate." [TC:2.A.1.14.2]
GO:0015540
name: 3-hydroxyphenyl propionate:hydrogen ion symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 3-hydroxyphenyl propionate(out) + H+(out) = 3-hydroxyphenyl propionate(in) + H+(in)." [TC:2.A.1.15.2]
GO:0015541
name: secondary active cyanate uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: cyanate(out) = cyanate(in)." [TC:2.A.1.17.1]
GO:0015542
name: sugar efflux transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sugar(in) = sugar(out)." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.1.20.1]
GO:0015543
name: lactose/glucose efflux transporter activity
def: "OBSOLETE. Catalysis of the reaction: glucose or lactose(in) = glucose or lactose(out)." [TC:2.A.1.20.2]
GO:0015544
name: phenyl propionate uptake uniporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: phenyl propionate(out) = phenyl propionate(in)." [GOC:mtg_transport, TC:2.A.1.27.1]
GO:0015545
name: bicyclomycin transporter activity
def: "Enables the directed movement of bicyclomycin into, out of or within a cell, or between cells. Bicyclomycin (or bicozamycin) is an antibacterial drug often used as a livestock feed additive." [ISBN:91191028X]
GO:0015546
name: sulfathiazole transmembrane transporter activity
def: "Enables the directed movement of sulfathiazole across a membrane. Sulfathiazole is an antibacterial agent of the sulfonamide group." [CHEBI:9337, GOC:curators]
GO:0015547
name: nalidixic acid transporter activity
def: "Enables the directed movement of nalidixic acid into, out of or within a cell, or between cells. Nalidixic acid is a synthetic antibiotic that interferes with DNA gyrase and inhibits prokaryotic replication." [GOC:curators, PMID:12702699, PubChem_Compound:4221]
GO:0015548
name: organomercurial transporter activity
def: "Enables the directed movement of organomercurial compounds into, out of or within a cell, or between cells. Organomercurial substances are any organic compound containing a mercury atom." [GOC:ai]
GO:0015549
name: carbonyl cyanide m-chlorophenylhydrazone transporter activity
def: "Enables the directed movement of carbonyl cyanide m-chlorophenylhydrazone into, out of or within a cell, or between cells. Carbonyl cyanide m-chlorophenylhydrazone is a proton ionophore, commonly used as an uncoupling agent and inhibitor of photosynthesis because of its effects on mitochondrial and chloroplast membranes." [CHEBI:3259, GOC:curators]
GO:0015550
name: galacturonate transmembrane transporter activity
def: "Catalysis of the transfer of galacturonate from one side of the membrane to the other. Galacturonate is the uronic acid formally derived from galactose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [GOC:ai]
GO:0015551
name: 3-hydroxyphenyl propanoate transmembrane transporter activity
def: "Catalysis of the transfer of 3-hydroxyphenyl propanoate from one side of the membrane to the other." [GOC:ai]
GO:0015552
name: propionate transmembrane transporter activity
def: "Catalysis of the transfer of propionate from one side of the membrane to the other. Propionate (or propanoate) is the organic acid CH3-CH2-COOH." [GOC:ai]
GO:0015553
name: xanthosine transmembrane transporter activity
def: "Catalysis of the transfer of xanthosine, xanthine riboside, from one side of a membrane to the other." [ISBN:0198506732]
GO:0015554
name: tartrate transmembrane transporter activity
def: "Catalysis of the transfer of tartrate from one side of the membrane to the other. Tartrate is the anion of 2,3-dihydroxybutanedioic acid, one of the aldaric acids. The L(+) enantiomer occurs widely in plants, especially in grape juice, and in fungi and bacteria." [GOC:ai]
GO:0015556
name: C4-dicarboxylate transmembrane transporter activity
def: "Catalysis of the transfer of C4-dicarboxylate from one side of the membrane to the other." [GOC:krc]
GO:0015557
name: arginine targeting transmembrane transporter activity
def: "Enables the directed movement of fully folded proteins into, out of or within a cell, or between cells, by targeting the proteins to the transporter via a specialized N-terminal twin arginine signal peptide." [GOC:dph, GOC:tb, PMID:17956229]
GO:0015558
name: p-aminobenzoyl-glutamate uptake transmembrane transporter activity
def: "Catalysis of the transfer of p-aminobenzoyl-glutamate from the outside of a cell to the inside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015560
name: L-idonate/D-gluconate:hydrogen symporter activity
def: "OBSOLETE. Catalysis of the reaction: (L-idonate or D-gluconate)(out) + H+(out) = (L-iodonate or D-gluconate)(in) + H+(in)." [TC:2.A.8.1.2]
GO:0015561
name: rhamnose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: rhamnose(out) + H+(out) = rhamnose(in) + H+(in)." [TC:2.A.7.6.1]
GO:0015562
name: efflux transmembrane transporter activity
def: "Catalysis of the transfer of a specific substance or related group of substances from the inside of the cell to the outside of the cell across a membrane." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015563
name: uptake transmembrane transporter activity
def: "Catalysis of the transfer of a specific substance or related group of substances from the outside of a cell to the inside across a membrane." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015565
name: threonine efflux transmembrane transporter activity
def: "Catalysis of the transfer of threonine from the inside of the cell to the outside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015566
name: acriflavine transporter activity
def: "Enables the directed movement of acriflavin into, out of or within a cell, or between cells. Acriflavin is a fluorescent dye used as a local antiseptic and also as a biological stain. It intercalates into nucleic acids thereby inhibiting bacterial and viral replication." [PubChem_Compound:6842]
GO:0015567
name: alkane transporter activity
def: "Enables the directed movement of alkanes into, out of or within a cell, or between cells. Alkanes are saturated aliphatic hydrocarbon compounds." [GOC:ai]
GO:0015568
name: L-idonate transmembrane transporter activity
def: "Catalysis of the transfer of L-idonate from one side of the membrane to the other. L-idonate is an aldonic acid derived from L-idose, an aldohexose which is epimeric with D-glucose." [GOC:ai]
GO:0015569
name: p-aminobenzoyl-glutamate transmembrane transporter activity
def: "Catalysis of the transfer of p-aminobenzoyl-glutamate from one side of a membrane to the other. p-aminobenzoyl-glutamate is the anion of p-aminobenzoyl-glutamic acid." [GOC:ai]
GO:0015571
name: N-acetylgalactosamine transmembrane transporter activity
def: "Catalysis of the transfer of N-acetylgalactosamine from one side of the membrane to the other. N-acetylgalactosamine, 2-acetamido-2-deoxygalactopyranose, is the n-acetyl derivative of galactosamine." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015572
name: N-acetylglucosamine transmembrane transporter activity
def: "Catalysis of the transfer of N-acetylglucosamine from one side of the membrane to the other. The D isomer of N-acetylglucosamine is a common structural unit of glycoproteins in plants, bacteria and animals; it is often the terminal sugar of an oligosaccharide group of a glycoprotein." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015573
name: beta-glucoside transmembrane transporter activity
def: "Catalysis of the transfer of beta-glucosides from one side of the membrane to the other. Beta-glucosides are glycosides in which the sugar group is a glucose residue, and the anomeric carbon of the bond is in a beta configuration." [GOC:jl, GOC:mtg_transport, http://www.biochem.purdue.edu/, ISBN:0198506732, ISBN:0815340729]
GO:0015574
name: trehalose transmembrane transporter activity
def: "Catalysis of the transfer of trehalose from one side of the membrane to the other. Trehalose is the disaccharide alpha-D-glucopyranosyl-alpha-D-glucopyranoside that acts of a reserve carbohydrate in certain fungi, algae and lichens." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015575
name: mannitol transmembrane transporter activity
def: "Catalysis of the transfer of mannitol from one side of the membrane to the other. Mannitol is the alditol derived from D-mannose by reduction of the aldehyde group." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015576
name: sorbitol transmembrane transporter activity
def: "Catalysis of the transfer of sorbitol from one side of the membrane to the other. Sorbitol, also known as glucitol, is the hexitol derived by the reduction of the aldehyde group of glucose." [GOC:ai, ISBN:0198506732]
GO:0015577
name: galactitol transmembrane transporter activity
def: "Catalysis of the transfer of a galactitol from one side of the membrane to the other. Galactitol is the hexitol derived by the reduction of the aldehyde group of either D- or L-galactose." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015578
name: mannose transmembrane transporter activity
def: "Catalysis of the transfer of mannose from one side of the membrane to the other. Mannose is the aldohexose manno-hexose, the C-2 epimer of glucose. The D-(+)-form is widely distributed in mannans and hemicelluloses and is of major importance in the core oligosaccharide of N-linked oligosaccharides of glycoproteins." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015583
name: beta-glucoside [arbutin-salicin-cellobiose] permease activity
def: "OBSOLETE. Catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). The beta-glucoside may be arbutin, salicin or cellobiose." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015591
name: D-ribose transmembrane transporter activity
def: "Catalysis of the transfer of D-ribose from one side of the membrane to the other. As beta-D-ribofuranose, D-ribose forms the glycose group of all ribonucleosides, ribonucleotides and ribonucleic acids, and also of ribose phosphates, various glycosides, some coenzymes and some forms of vitamin B12." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015592
name: methylgalactoside transmembrane transporter activity
def: "Catalysis of the transfer of methylgalactoside from one side of the membrane to the other. Methylgalactoside is a compound in which the H of the OH group on carbon-1 of galactose is replaced by a methyl group." [CHEBI:55507, GOC:mtg_transport, ISBN:0815340729]
GO:0015593
name: allose transmembrane transporter activity
def: "Catalysis of the transfer of allose from one side of the membrane to the other. Allose is an aldohexose similar to glucose, differing only in the configuration of the hydroxyl group of C-3." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015594
name: putrescine-importing ATPase activity
def: "Catalysis of the reaction: ATP + H2O + putrescine(out) -> ADP + phosphate + putrescine(in)." [EC:3.6.3.31]
GO:0015595
name: spermidine-importing ATPase activity
def: "Catalysis of the reaction: ATP + H2O + spermidine(out) -> ADP + phosphate + spermidine(in)." [EC:3.6.3.31]
GO:0015596
name: glycine betaine/proline porter activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + quaternary amine(out) = ADP + phosphate + quaternary amine(in)." [EC:3.6.3.32, TC:3.A.1.12.1]
GO:0015597
name: histidine/arginine/lysine/ornithine porter activity
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + polar amino acid(out) = ADP + phosphate + polar amino acid(in)." [EC:3.6.3.21, TC:3.A.1.3.1]
GO:0015598
name: arginine-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + arginine(out) -> ADP + phosphate + arginine(in)." [EC:3.6.3.21]
GO:0015599
name: glutamine-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + glutamine(out) -> ADP + phosphate + glutamine(in)." [EC:3.6.3.21]
GO:0015600
name: glutamate/aspartate porter activity
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + polar amino acid(out) = ADP + phosphate + polar amino acid(in)." [EC:3.6.3.21, TC:3.A.1.3.4]
GO:0015601
name: cystine/diaminopimelate porter activity
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + polar amino acid(out) = ADP + phosphate + polar amino acid(in)." [EC:3.6.3.21, TC:3.A.1.3.10]
GO:0015602
name: leucine/isoleucine/valine porter activity
def: "OBSOLETE. Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + nonpolar amino acid(out) = ADP + phosphate + nonpolar amino acid(in)." [EC:3.6.3.22, TC:3.A.1.4.1]
GO:0015603
name: iron chelate transmembrane transporter activity
def: "Catalysis of the transfer of an iron chelate from one side of a membrane to the other. An iron chelate is a heterocyclic compound having a metal ion attached by coordinate bonds to at least two nonmetal ions." [http://www.onelook.com]
GO:0015604
name: organic phosphonate transmembrane transporter activity
def: "Catalysis of the transfer of phosphonate from one side of a membrane to the other. A phosphonate is any salt, anion, or ester of phosphonic acid (HPO(OH)2)." [GOC:ai]
GO:0015605
name: organophosphate ester transmembrane transporter activity
def: "Catalysis of the transfer of organophosphate esters from one side of a membrane to the other. Organophosphate esters are small organic molecules containing phosphate ester bonds." [GOC:mcc]
GO:0015606
name: spermidine transmembrane transporter activity
def: "Catalysis of the transfer of spermidine, N-(3-aminopropyl)-1,4-diaminobutane, from one side of the membrane to the other." [GOC:ai]
GO:0015607
name: fatty-acyl-CoA transporter activity
def: "Enables the directed movement of a fatty acyl CoA group into, out of or within a cell, or between cells. A fatty acyl CoA group is any acyl group derived from a fatty acid with a coenzyme A group attached to it." [GOC:ai]
GO:0015608
name: carbohydrate-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + carbohydrate(out) -> ADP + phosphate + carbohydrate(in)." [GOC:ai, TC:3.A.1.1.-, TC:3.A.1.2.-]
GO:0015609
name: maltooligosaccharide-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + maltooligosaccharide(out) -> ADP + phosphate + maltooligosaccharide(in)." [TC:3.A.1.-.-]
GO:0015610
name: glycerol phosphate-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + glycerol-phosphate(out) -> ADP + phosphate + glycerol-phosphate(in)." [TC:3.A.1.-.-]
GO:0015611
name: D-ribose-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-ribose(out) -> ADP + phosphate + D-ribose(in)." [TC:3.A.1.-.-]
GO:0015612
name: L-arabinose-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + L-arabinose(out) -> ADP + phosphate + L-arabinose(in)." [TC:3.A.1.-.-]
GO:0015613
name: galactose/glucose (methylgalactoside) porter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015614
name: D-xylose-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-xylose(out) -> ADP + phosphate + D-xylose(in)." [TC:3.A.1.-.-]
GO:0015615
name: D-allose-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-allose(out) -> ADP + phosphate + D-allose(in)." [TC:3.A.1.-.-]
GO:0015616
name: DNA translocase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive movement along a single- or double-stranded DNA molecule." [GOC:mah, PMID:16428451, PMID:17631491]
GO:0015617
name: pilin/fimbrilin exporter activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0015619
name: thiamine pyrophosphate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + thiamine pyrophosphate(out) = ADP + phosphate + thiamine pyrophosphate(in)." [TC:3.A.1.19.1]
GO:0015620
name: ferric-enterobactin transmembrane transporter activity
def: "Catalysis of the transfer of ferric-enterobactin from one side of a membrane to the other." [GOC:ai]
GO:0015621
name: ferric triacetylfusarinine C transmembrane transporter activity
def: "Catalysis of the transfer of ferric triacetylfusarinine C from one side of a membrane to the other." [GOC:ai]
GO:0015622
name: ferric-hydroxamate transmembrane transporter activity
def: "Catalysis of the transfer of ferric-hydroxamate from one side of a membrane to the other." [GOC:ai]
GO:0015623
name: iron-chelate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + iron chelate(out) = ADP + phosphate + iron chelate(in). Fe-enterobactin, Fe-dicitrate, Fe-hydroxamate and other siderophores are imported." [EC:3.6.3.34]
GO:0015624
name: ferric-enterobactin-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + ferric-enterobactin(out) = ADP + phosphate + ferric-enterobactin(in)." [TC:3.A.1.-.-]
GO:0015625
name: ferric-hydroxamate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + ferric-hydroxamate(out) = ADP + phosphate + ferric-hydroxamate(in)." [TC:3.A.1.-.-]
GO:0015626
name: L-diaminopimelate transmembrane transporter activity
def: "Catalysis of the transfer of L-diaminopimelate from one side of a membrane to the other. L-diaminopimelate is the L-enantiomer anion of 2,6-diaminoheptanedioic acid." [GOC:go_curators, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0015631
name: tubulin binding
def: "Interacting selectively and non-covalently with monomeric or multimeric forms of tubulin, including microtubules." [GOC:clt]
GO:0015633
name: zinc transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Zn2+(out) = ADP + phosphate + Zn2+(in)." [TC:3.A.1.-.-]
GO:0015634
name: lipopolysaccharide exporter activity
def: "Catalysis of the transfer of lipopolysaccharide from the inside of the cell to the outside of the cell across a membrane. A lipopolysaccharide is any of a group of related, structurally complex components of the outer membrane of Gram-negative bacteria. Lipopolysaccharides consist three covalently linked regions, lipid A, core oligosaccharide, and an O side chain. Lipid A is responsible for the toxicity of the lipopolysaccharide." [GOC:mtg_transport, ISBN:0815340729]
GO:0015635
name: short-chain fatty acid transporter activity
def: "Enables the directed movement of short-chain fatty acids into, out of or within a cell, or between cells. Short-chain fatty acids are fatty acids with a chain length of less than C6." [CHEBI:26666, ISBN:0198506732]
GO:0015636
name: short-chain fatty acid uptake transporter activity
def: "Enables the directed movement of short-chain fatty acids into a cell or organelle. Short-chain fatty acids are fatty acids with a chain length of less than C6." [CHEBI:26666, GOC:mah]
GO:0015638
name: microcin uptake transporter activity
def: "Catalysis of the transfer of a microcin from the outside of a cell to the inside of the cell across a membrane." [GOC:mah]
GO:0015639
name: ferrous iron uptake transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Fe2+(out) + energy = Fe2+(in)." [TC:9.A.8.1.1]
GO:0015640
name: peptidoglycan peptide transporter activity
def: "Enables the directed movement of peptidoglycan peptides into, out of or within a cell, or between cells. Peptidoglycan peptides are the oligopeptides found in peptidoglycan networks which cross-link the polysaccharide chains." [ISBN:0198506732]
GO:0015641
name: lipoprotein toxin
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0015642
name: bacteriolytic toxin activity
def: "OBSOLETE. Acts as to cause lysis of bacterial cells." [GOC:jl]
GO:0015643
name: toxic substance binding
def: "Interacting selectively and non-covalently with a toxic substance, a poisonous substance that causes damage to biological systems." [GOC:bf, GOC:curators, GOC:jl, GOC:pr]
GO:0015644
name: lipoprotein antitoxin
def: "OBSOLETE. Binds to a lipoprotein toxin, which is usually derived from a microorganism, thereby neutralizing it." [GOC:jl]
GO:0015645
name: fatty acid ligase activity
def: "Catalysis of the ligation of a fatty acid to an acceptor, coupled to the hydrolysis of ATP." [EC:6.2.1.20, EC:6.2.1.3, GOC:cjk, GOC:mah]
GO:0015647
name: peptidoglycan transporter activity
def: "Enables the directed movement of peptidoglycans, a class of glycoconjugates found in bacterial cell walls, into, out of or within a cell, or between cells." [GOC:ai]
GO:0015648
name: lipid-linked peptidoglycan transporter activity
def: "Enables the directed movement of lipid-linked peptidoglycans into, out of or within a cell, or between cells." [GOC:mah]
GO:0015649
name: 2-keto-3-deoxygluconate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: 2-keto-3-deoxygluconate(out) + H+(out) = 2-keto-3-deoxygluconate(in) + H+(in)." [TC:2.A.10.1.1]
GO:0015650
name: lactate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: lactate (out) + H+ (out) = lactate (in) + H+ (in)." [TC:2.A.14.1.1]
GO:0015651
name: quaternary ammonium group transmembrane transporter activity
def: "Catalysis of the transfer of quaternary ammonium groups from one side of a membrane to the other. Quaternary ammonium groups are any compound that can be regarded as derived from ammonium hydroxide or an ammonium salt by replacement of all four hydrogen atoms of the NH4+ ion by organic groups." [ISBN:0198506732]
GO:0015652
name: quaternary ammonium group:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: quaternary ammonium group(out) + H+(out) = quaternary ammonium group(in) + H+(in)." [GOC:ai]
GO:0015653
name: glycine betaine:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glycine betaine(out) + H+(out) = glycine betaine(in) + H+(in)." [TC:2.A.15.1.1]
GO:0015654
name: tellurite uptake transmembrane transporter activity
def: "Catalysis of the transfer of tellurite from the outside of a cell to the inside across a membrane: tellurite(out) + H+(out) = tellurite(in) + H+(in)." [GOC:mtg_transport, ISBN:0815340729, TC:2.A.16.1.1]
GO:0015655
name: alanine:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: alanine(out) + Na+(out) = alanine(in) + Na+(in)." [GOC:ai]
GO:0015657
name: branched-chain amino acid:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: branched-chain amino acid(out) + cation(out) = branched-chain amino acid(in) + cation(in)." [TC:2.A.26.1.1]
GO:0015658
name: branched-chain amino acid transmembrane transporter activity
def: "Catalysis of the transfer of branched-chain amino acids from one side of a membrane to the other. Branched-chain amino acids are amino acids with a branched carbon skeleton without rings." [CHEBI:22918, GOC:ai, GOC:bf, GOC:mtg_transport, ISBN:0815340729]
GO:0015659
name: formate uptake transmembrane transporter activity
def: "Catalysis of the transfer of formate from the outside of a cell to the inside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015660
name: formate efflux transmembrane transporter activity
def: "Catalysis of the transfer of formate from the inside of the cell to the outside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0015661
name: L-lysine efflux transmembrane transporter activity
def: "Catalysis of the transfer of L-lysine from the inside of the cell to the outside of the cell across a membrane." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0015662
name: ATPase activity, coupled to transmembrane movement of ions, phosphorylative mechanism
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate, to directly drive the transport of ions across a membrane. The reaction is characterized by the transient formation of a high-energy aspartyl-phosphoryl-enzyme intermediate." [PMID:10322420, PMID:10600683]
GO:0015663
name: nicotinamide mononucleotide transmembrane transporter activity
def: "Enables the directed movement of nicotinamide mononucleotide into, out of or within a cell, or between cells. Nicotinamide mononucleotide is a ribonucleotide in which the nitrogenous base, nicotinamide, is in beta-n-glycosidic linkage with the c-1 position of d-ribose. It is a constituent of NAD and NADP." [CHEBI:50383, GOC:curators, ISBN:0721662544]
GO:0015665
name: alcohol transmembrane transporter activity
def: "Catalysis of the transfer of an alcohol from one side of the membrane to the other. An alcohol is any carbon compound that contains a hydroxyl group." [ISBN:0198506732]
GO:0015666
name: restriction endodeoxyribonuclease activity
def: "Catalysis of endonucleolytic cleavage of DNA in a site-specific manner, resulting in double-strand breaks." [GOC:mlg]
GO:0015667
name: site-specific DNA-methyltransferase (cytosine-N4-specific) activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + DNA cytosine = S-adenosyl-L-homocysteine + DNA N4-methylcytosine." [EC:2.1.1.113]
GO:0015668
name: Type III site-specific deoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of DNA to give double-stranded fragments with terminal 5'-phosphates. ATP hydrolysis is required. Cleavage is dependent on the presence of two copies of a specific recognition sequence in an inverse orientation in the DNA. Cleavage occurs at a specific distance from one of the recognition sites." [EC:3.1.21.5, PMID:12654995]
GO:0015922
name: aspartate oxidase activity
def: "Catalysis of the reaction: aspartate + O2 = iminosuccinate + hydrogen peroxide." [EC:1.4.3.1, EC:1.4.3.16]
GO:0015923
name: mannosidase activity
def: "Catalysis of the hydrolysis of mannosyl compounds, substances containing a group derived from a cyclic form of mannose or a mannose derivative." [GOC:ai]
GO:0015924
name: mannosyl-oligosaccharide mannosidase activity
def: "Catalysis of the hydrolysis of the terminal alpha-D-mannose residues in oligo-mannose oligosaccharides." [EC:3.2.1.-, GOC:ai]
GO:0015925
name: galactosidase activity
def: "Catalysis of the hydrolysis of galactosyl compounds, substances containing a group derived from a cyclic form of galactose or a galactose derivative." [GOC:ai]
GO:0015926
name: glucosidase activity
def: "Catalysis of the hydrolysis of glucosyl compounds, substances containing a group derived from a cyclic form of glucose or a glucose derivative." [ISBN:0198506732]
GO:0015927
name: trehalase activity
def: "Catalysis of the hydrolysis of trehalose or a trehalose derivative." [GOC:ai]
GO:0015928
name: fucosidase activity
def: "Catalysis of the hydrolysis of fucosyl compounds, substances containing a group derived from a cyclic form of fucose or a fucose derivative." [GOC:ai]
GO:0015929
name: hexosaminidase activity
def: "Catalysis of the cleavage of hexosamine or N-acetylhexosamine residues (e.g. N-acetylglucosamine) residues from gangliosides or other glycoside oligosaccharides." [http://www.onelook.com/, ISBN:0721662544]
GO:0015930
name: glutamate synthase activity
def: "Catalysis of the formation of L-glutamine and 2-oxoglutarate from L-glutamate, using NADH, NADPH or ferredoxin as hydrogen acceptors." [EC:1.4.-.-]
GO:0015932
name: nucleobase-containing compound transmembrane transporter activity
def: "Catalysis of the transfer of nucleobases, nucleosides, nucleotides and nucleic acids from one side of a membrane to the other." [GOC:ai]
GO:0015933
name: flavin-containing electron transporter
def: "OBSOLETE. An oxidoreductase which contains either flavin-adenine dinucleotide or flavin mononucleotide as a prosthetic group, utilizes either NADH or NADPH and transfers electrons to other electron transfer proteins." [GOC:kd]
GO:0015997
name: ubiquinone biosynthetic process monooxygenase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016002
name: sulfite reductase activity
def: "Catalysis of the reaction: hydrogen sulfide + acceptor + 3 H2O = sulfite + reduced acceptor." [EC:1.8.99.1]
GO:0016004
name: phospholipase activator activity
def: "Increases the activity of a phospholipase, an enzyme that catalyzes of the hydrolysis of a glycerophospholipid." [GOC:ai]
GO:0016005
name: phospholipase A2 activator activity
def: "Increases the activity of the enzyme phospholipase A2." [GOC:ai]
GO:0016015
name: morphogen activity
def: "Acts as a trigger for a pattern specification process when present at a specific concentration within a gradient." [GOC:go_curators]
GO:0016016
name: short-wave-sensitive opsin
def: "OBSOLETE. An opsin with maximal absorption between 400 and 500 nm." [PMID:10594055]
GO:0016018
name: cyclosporin A binding
def: "Interacting selectively and non-covalently with cyclosporin A, a cyclic undecapeptide that contains several N-methylated and unusual amino acids." [GOC:mb]
GO:0016019
name: peptidoglycan receptor activity
def: "Combining with a peptidoglycan and transmitting the signal to initiate a change in cell activity." [PMID:14698226]
GO:0016025
name: proteasome endopeptidase regulator
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0016026
name: proteasome endopeptidase core
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0016034
name: maleylacetoacetate isomerase activity
def: "Catalysis of the reaction: 4-maleylacetoacetate = 4-fumarylacetoacetate." [EC:5.2.1.2]
GO:0016040
name: glutamate synthase (NADH) activity
def: "Catalysis of the reaction: 2 L-glutamate + NAD(+) = 2-oxoglutarate + L-glutamine + H(+) + NADH." [EC:1.4.1.14, RHEA:13756]
GO:0016041
name: glutamate synthase (ferredoxin) activity
def: "Catalysis of the reaction: 2 L-glutamate + 2 oxidized ferredoxin = L-glutamine + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+. This is a two-step reaction: (a) L-glutamate + NH3 = L-glutamine + H2O, (b) L-glutamate + 2 oxidized ferredoxin + H2O = NH3 + 2-oxoglutarate + 2 reduced ferredoxin + 2 H+." [EC:1.4.7.1]
GO:0016084
name: myostimulatory hormone activity
def: "The action characteristic of myostimulatory hormone, a peptide hormone that stimulates muscle contraction." [GOC:mah, PMID:12204246]
GO:0016085
name: myoinhibitory hormone activity
def: "The action characteristic of myostimulatory hormone, a peptide hormone that inhibits muscle contraction." [GOC:mah, PMID:8902848]
GO:0016086
name: allatostatin
def: "OBSOLETE. Peptide hormones produced by the corpora allata of insects that reversibly inhibit the production of juvenile hormone." [GOC:ai, PMID:10891383]
GO:0016087
name: ecdysiostatic hormone activity
def: "The action characteristic of ecdysiostatic hormone, a peptide hormone that inhibits ecdysone secretion." [DOI:10.1002/(SICI)1520-6327(1997)35\:1, GOC:mah]
GO:0016088
name: insulin
def: "OBSOLETE. A polypeptide hormone that stimulates glucose uptake by muscle and adipose tissue, and promotes glycogenesis, lipogenesis and the synthesis of proteins and nucleic acids." [ISBN:0198506732]
GO:0016146
name: protein-synthesizing GTPase activity, initiation
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.48]
GO:0016147
name: protein-synthesizing GTPase activity, elongation
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.48]
GO:0016148
name: protein-synthesizing GTPase activity, termination
def: "OBSOLETE. Catalysis of the reaction: GTP + H2O = GDP + phosphate." [EC:3.6.1.48]
GO:0016149
name: translation release factor activity, codon specific
def: "A translation release factor that is specific for one or more particular termination codons; acts at the ribosomal A-site and require polypeptidyl-tRNA at the P-site." [ISBN:0198547684]
GO:0016150
name: translation release factor activity, codon nonspecific
def: "A translation release factor that is not specific to particular codons; binds to guanine nucleotides." [ISBN:0198547684]
GO:0016151
name: nickel cation binding
def: "Interacting selectively and non-covalently with nickel (Ni) cations." [GOC:ai]
GO:0016152
name: mercury (II) reductase activity
def: "Catalysis of the reaction: H(+) + Hg + NADP(+) = Hg(2+) + NADPH." [EC:1.16.1.1, RHEA:23859]
GO:0016153
name: urocanate hydratase activity
def: "Catalysis of the reaction: 4-imidazolone-5-propanoate + H(+) = trans-urocanate + H(2)O." [EC:4.2.1.49, RHEA:13104]
GO:0016154
name: pyrimidine-nucleoside phosphorylase activity
def: "Catalysis of the reaction: pyrimidine nucleoside + phosphate = pyrimidine + alpha-D-ribose 1-phosphate." [EC:2.4.2.2]
GO:0016155
name: formyltetrahydrofolate dehydrogenase activity
def: "Catalysis of the reaction: 10-formyltetrahydrofolate + H(2)O + NADP(+) = (6S)-5,6,7,8-tetrahydrofolate + CO(2) + H(+) + NADPH." [EC:1.5.1.6, RHEA:10183]
GO:0016156
name: fumarate reductase (NADH) activity
def: "Catalysis of the reaction: NAD(+) + succinate = fumarate + H(+) + NADH." [EC:1.3.1.6, RHEA:18284]
GO:0016157
name: sucrose synthase activity
def: "Catalysis of the reaction: UDP-glucose + D-fructose = UDP + sucrose." [EC:2.4.1.13]
GO:0016158
name: 3-phytase activity
def: "Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = D-myo-inositol 1,2,4,5,6-pentakisphosphate + phosphate." [EC:3.1.3.8]
GO:0016159
name: muconolactone delta-isomerase activity
def: "Catalysis of the reaction: 2,5-dihydro-5-oxofuran-2-acetate = 3,4-dihydro-5-oxofuran-2-acetate." [EC:5.3.3.4]
GO:0016160
name: amylase activity
def: "Catalysis of the hydrolysis of amylose or an amylose derivative." [GOC:ai]
GO:0016161
name: beta-amylase activity
def: "Catalysis of the reaction: (1,4-alpha-D-glucosyl)(n+1) + H2O = (1,4-alpha-D-glucosyl)(n-1) + alpha-maltose. This reaction is the hydrolysis of 1,4-alpha-glucosidic linkages in polysaccharides so as to remove successive maltose units from the non-reducing ends of the chains." [EC:3.2.1.2]
GO:0016162
name: cellulose 1,4-beta-cellobiosidase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-glucosidic linkages in cellulose and cellotetraose, releasing cellobiose from the non-reducing ends of the chains." [EC:3.2.1.91]
GO:0016163
name: nitrogenase activity
def: "Catalysis of the reaction: 8 reduced ferredoxin + 8 H+ + nitrogen + 16 ATP = 8 oxidized ferredoxin + 2 NH3 + 16 ADP + 16 phosphate." [EC:1.18.6.1]
GO:0016164
name: Mo-molybdopterin oxidoreductase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016165
name: linoleate 13S-lipoxygenase activity
def: "Catalysis of the reaction: linoleate + O2 = (9Z,11E)-(13S)-13-hydroperoxyoctadeca-9,11-dienoate." [EC:1.13.11.12, GOC:lb]
GO:0016166
name: phytoene dehydrogenase activity
def: "Catalysis of the dehydrogenation of phytoene to produce a carotenoid intermediate such as phytofluene." [http://www.chem.qmul.ac.uk/iubmb/enzyme/reaction/terp/carot.html]
GO:0016167
name: glial cell-derived neurotrophic factor receptor activity
def: "Combining with glial cell line-derived neurotrophic factor and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0016168
name: chlorophyll binding
def: "Interacting selectively and non-covalently with chlorophyll; any compound of magnesium complexed in a porphyrin (tetrapyrrole) ring and which functions as a photosynthetic pigment." [GOC:jl]
GO:0016169
name: bacteriochlorophyll c binding
def: "Interacting selectively and non-covalently with bacteriochlorophyll c, a chlorophyll of photosynthetic bacteria, for example green sulfur bacteria." [ISBN:0192800981]
GO:0016170
name: interleukin-15 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-15 receptor." [GOC:ai]
GO:0016171
name: cell surface antigen
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016172
name: antifreeze activity
def: "OBSOLETE. Inhibits the formation of ice crystals in organismal fluid (e.g. blood) at below freezing exogenous temperatures." [GOC:jl]
GO:0016173
name: ice nucleation inhibitor activity
def: "OBSOLETE. Inhibits the formation of ice crystals." [GOC:ai]
GO:0016174
name: NAD(P)H oxidase activity
def: "Catalysis of the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + hydrogen peroxide." [EC:1.6.3.1]
GO:0016175
name: superoxide-generating NADPH oxidase activity
def: "Catalysis of the reaction: NAD(P)H + O2 = NAD(P)H + O2-." [GOC:ai, PMID:10806195]
GO:0016176
name: superoxide-generating NADPH oxidase activator activity
def: "Increases the activity of the enzyme superoxide-generating NADPH oxidase." [GOC:ai]
GO:0016205
name: selenocysteine methyltransferase activity
def: "Catalysis of the reaction: selenocysteine + S-adenosyl-L-methionine = Se-methylselenocysteine + S-adenosyl-homocysteine." [EC:2.1.1.-, PMID:10026151]
GO:0016206
name: catechol O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a catechol = S-adenosyl-L-homocysteine + a guaiacol." [EC:2.1.1.6]
GO:0016207
name: 4-coumarate-CoA ligase activity
def: "Catalysis of the reaction: ATP + 4-coumarate + CoA = AMP + diphosphate + 4-coumaroyl-CoA." [EC:6.2.1.12]
GO:0016208
name: AMP binding
def: "Interacting selectively and non-covalently with AMP, adenosine monophosphate." [GOC:go_curators]
GO:0016209
name: antioxidant activity
def: "Inhibition of the reactions brought about by dioxygen (O2) or peroxides. Usually the antioxidant is effective because it can itself be more easily oxidized than the substance protected. The term is often applied to components that can trap free radicals, thereby breaking the chain reaction that normally leads to extensive biological damage." [ISBN:0198506732]
GO:0016210
name: naringenin-chalcone synthase activity
def: "Catalysis of the reaction: 3 malonyl-CoA + 4-coumaroyl-CoA = 4 CoA + naringenin chalcone + 3 CO2." [EC:2.3.1.74]
GO:0016211
name: ammonia ligase activity
def: "Catalysis of the ligation of ammonia (NH3) to another substance via a carbon-nitrogen bond with concomitant breakage of a diphosphate linkage, usually in a nucleoside triphosphate." [GOC:jl]
GO:0016212
name: kynurenine-oxoglutarate transaminase activity
def: "Catalysis of the reaction: L-kynurenine + 2-oxoglutarate = 4-(2-aminophenyl)-2,4-dioxobutanoate + L-glutamate." [EC:2.6.1.7]
GO:0016213
name: linoleoyl-CoA desaturase activity
def: "Catalysis of the reaction: linoleoyl-CoA + reduced acceptor + O2 = gamma-linolenoyl-CoA + acceptor + 2 H2O." [EC:1.14.19.3]
GO:0016215
name: acyl-CoA desaturase activity
def: "Catalysis of the reaction: acyl-CoA + reduced acceptor + O2 = desaturated-acyl-CoA + acceptor + 2 H2O." [EC:1.14.19.1, EC:1.14.19.5, GOC:mah]
GO:0016216
name: isopenicillin-N synthase activity
def: "Catalysis of the reaction: N-[(5S)-5-amino-5-carboxypentanoyl]-L-cysteinyl-D-valine + O(2) = 2 H(2)O + isopenicillin N." [EC:1.21.3.1, RHEA:22431]
GO:0016217
name: N-ethylammeline chlorohydrolase activity
def: "Catalysis of the reaction: deethylsimazine + H2O = N-ethylammeline + chloride + H+." [MetaCyc:R465-RXN]
GO:0016218
name: polyketide synthase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:curators]
GO:0016223
name: beta-alanine-pyruvate transaminase activity
def: "Catalysis of the reaction: L-alanine + 2-oxopropanoate = pyruvate + beta-alanine." [EC:2.6.1.18]
GO:0016227
name: tRNA sulfurtransferase activity
def: "OBSOLETE. Catalysis of the reaction: L-cysteine + 'activated' tRNA = L-serine + tRNA containing a thionucleotide." [EC:2.8.1.4]
GO:0016229
name: steroid dehydrogenase activity
def: "Catalysis of an oxidation-reduction (redox) reaction in which one substrate is a sterol derivative." [GOC:mah]
GO:0016230
name: sphingomyelin phosphodiesterase activator activity
def: "Increases the activity of the enzyme sphingomyelin phosphodiesterase." [GOC:ai]
GO:0016231
name: beta-N-acetylglucosaminidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing N-acetyl-D-glucosamine residues in N-acetyl-beta-D-glucosaminides." [EC:3.2.1.52, MetaCyc:3.2.1.52-RXN]
GO:0016232
name: HNK-1 sulfotransferase activity
def: "Catalysis of the synthesis of the HKK-1 carbohydrate epitope; adds a sulfate group to a precursor, GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R), forming sulfo-3GlcA-beta-(1->3)-Gal-beta-(1->4)-GlcNAc-beta-(1->R)." [PMID:9478973]
GO:0016247
name: channel regulator activity
def: "Modulates the activity of a channel. A channel catalyzes energy-independent facilitated diffusion, mediated by passage of a solute through a transmembrane aqueous pore or channel." [GOC:mah]
GO:0016248
name: channel inhibitor activity
def: "Stops, prevents, or reduces the activity of a channel. A channel catalyzes energy-independent facilitated diffusion, mediated by passage of a solute through a transmembrane aqueous pore or channel." [GOC:mah]
GO:0016250
name: N-sulfoglucosamine sulfohydrolase activity
def: "Catalysis of the reaction: N-sulfo-D-glucosamine + H2O = D-glucosamine + sulfate." [EC:3.10.1.1]
GO:0016251
name: general RNA polymerase II transcription factor activity
def: "OBSOLETE. Any function that supports basal (unregulated) transcription of genes by core RNA polymerase II. Five general transcription factors are necessary and sufficient for such basal transcription in yeast: TFIIB, TFIID, TFIIE, TFIIF, TFIIH and TATA-binding protein (TBF)." [PMID:10384286, PMID:9774831]
GO:0016252
name: nonspecific RNA polymerase II transcription factor activity
def: "OBSOLETE. Any function that supports transcription of genes by RNA polymerase II, and is not specific to a particular gene or gene set." [GOC:jl]
GO:0016262
name: protein N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + protein = UDP + 4-N-(N-acetyl-D-glucosaminyl)-protein." [EC:2.4.1.94]
GO:0016263
name: glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase activity
def: "Catalysis of the addition of a galactosyl residue to a non-reducing O-linked N-acetylgalactosamine residue in an O-glycan." [EC:2.4.1.122, GOC:ma]
GO:0016273
name: arginine N-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to an amino group of an arginine residue." [GOC:mah]
GO:0016274
name: protein-arginine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (protein)-arginine = S-adenosyl-L-homocysteine + (protein)-N-methyl-arginine." [EC:2.1.1.125, EC:2.1.1.126, GOC:mah, PMID:12351636]
GO:0016275
name: [cytochrome c]-arginine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (cytochrome c)-arginine = S-adenosyl-L-homocysteine + (cytochrome c)-N(omega)-methyl-arginine." [EC:2.1.1.124, GOC:ma]
GO:0016277
name: [myelin basic protein]-arginine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (myelin basic protein)-arginine = S-adenosyl-L-homocysteine + (myelin basic protein)-N(omega)-methyl-arginine." [EC:2.1.1.126]
GO:0016278
name: lysine N-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the epsilon-amino group of a lysine residue." [GOC:mah]
GO:0016279
name: protein-lysine N-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the epsilon-amino group of a lysine residue in a protein substrate." [PMID:12054878]
GO:0016284
name: alanine aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal amino acid, preferentially alanine, from an oligopeptide or polypeptide." [GOC:jl]
GO:0016285
name: cytosol alanyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal amino acid, preferentially alanine, from a wide range of peptides, amides and arylamides." [EC:3.4.11.14]
GO:0016286
name: small conductance calcium-activated potassium channel activity
def: "Catalysis of the transmembrane transfer of potassium by a channel with a unit conductance of 2 to 20 picoSiemens that opens in response to stimulus by internal calcium ions. Small conductance calcium-activated potassium channels are more sensitive to calcium than are large conductance calcium-activated potassium channels. Transport by a channel involves catalysis of facilitated diffusion of a solute (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel, without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, OMIM:602754]
GO:0016287
name: glycerone-phosphate O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + glycerone phosphate = 1-acylglycerone 3-phosphate + CoA." [EC:2.3.1.42, RHEA:17660]
GO:0016289
name: CoA hydrolase activity
def: "Catalysis of the reaction: X-CoA + H2O = X + CoA; X may be any group." [GOC:ai]
GO:0016290
name: palmitoyl-CoA hydrolase activity
def: "Catalysis of the reaction: palmitoyl-CoA + H2O = CoA + palmitate." [EC:3.1.2.2]
GO:0016295
name: myristoyl-[acyl-carrier-protein] hydrolase activity
def: "Catalysis of the reaction: myristoyl-[acyl-carrier protein] + H2O = [acyl-carrier protein] + myristate." [EC:3.1.2.14, MetaCyc:RXN-10727]
GO:0016296
name: palmitoyl-[acyl-carrier-protein] hydrolase activity
def: "Catalysis of the reaction: palmitoyl-[acyl-carrier protein] + H2O = [acyl-carrier protein] + palmitate." [EC:3.1.2.14, MetaCyc:RXN-9549]
GO:0016297
name: acyl-[acyl-carrier-protein] hydrolase activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + H2O = [acyl-carrier protein] + a fatty acid." [EC:3.1.2.14, MetaCyc:RXN-7902]
GO:0016298
name: lipase activity
def: "Catalysis of the hydrolysis of a lipid or phospholipid." [GOC:mah]
GO:0016299
name: regulator of G-protein signaling activity
def: "OBSOLETE. Inhibits signal transduction the GTPase activity of G-protein alpha subunits, thereby driving them into their inactive GDP-bound form." [GOC:curators]
GO:0016300
name: tRNA (uracil) methyltransferase activity
def: "Catalysis of the transfer of a methyl group from a donor to a uracil residue in a tRNA molecule." [GOC:mah]
GO:0016301
name: kinase activity
def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a substrate molecule." [ISBN:0198506732]
GO:0016303
name: 1-phosphatidylinositol-3-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol + ATP = a 1-phosphatidyl-1D-myo-inositol 3-phosphate + ADP + 2 H(+)." [EC:2.7.1.137, RHEA:12712]
GO:0016304
name: phosphatidylinositol 3-kinase activity, class I
def: "OBSOLETE. A heterodimeric phosphoinositide 3-kinase which can phosphorylate phosphatidylinositol, phosphatidylinositol-4-phosphate or phosphatidylinositol-4,5-bisphosphate. Also possesses intrinsic protein kinase activity." [PMID:9759495]
GO:0016305
name: phosphatidylinositol 3-kinase activity, class II
def: "OBSOLETE. A phosphoinositide 3-kinase which can phosphorylate phosphatidylinositol and phosphatidylinositol-4-phosphate; the human form can phosphorylate phosphatidylinositol-4,5-bisphosphate in the presence of phosphatidylserine." [PMID:9759495]
GO:0016306
name: phosphatidylinositol 3-kinase activity, class III
def: "OBSOLETE. A phosphoinositide 3-kinase which can only phosphorylate phosphatidylinositol." [PMID:9759495]
GO:0016307
name: phosphatidylinositol phosphate kinase activity
def: "Catalysis of the reaction: ATP + a phosphatidylinositol phosphate = ADP + a phosphatidylinositol bisphosphate." [EC:2.7.1.-, PMID:9759495]
GO:0016308
name: 1-phosphatidylinositol-4-phosphate 5-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4-phosphate + ATP = 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.68, RHEA:14428]
GO:0016309
name: 1-phosphatidylinositol-5-phosphate 4-kinase activity
def: "Catalysis of the reaction: ATP + 1-phosphatidyl-1D-myo-inositol 5-phosphate = ADP + 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate." [EC:2.7.1.149]
GO:0016312
name: inositol bisphosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol bisphosphate + H2O = myo-inositol phosphate + phosphate." [GOC:hb]
GO:0016313
name: inositol-1,4,5-trisphosphate phosphatase
def: "OBSOLETE. The removal of one of the phosphate groups from an inositol triphosphate to produce an inositol bisphosphate." [GOC:hb]
GO:0016314
name: phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol-3,4,5-trisphosphate + H2O = phosphatidylinositol-4,5-bisphosphate + phosphate." [EC:3.1.3.67]
GO:0016316
name: phosphatidylinositol-3,4-bisphosphate 4-phosphatase activity
def: "Catalysis of the reaction: 1-phosphatidyl-myo-inositol 3,4-bisphosphate + H2O = 1-phosphatidyl-1D-myo-inositol 3-phosphate + phosphate." [EC:3.1.3.66, GOC:hb]
GO:0016326
name: kinesin motor activity
def: "OBSOLETE. The hydrolysis of ATP (and GTP) that drives the microtubular motor along microtubules." [GOC:hb]
GO:0016329
name: apoptosis regulator activity
def: "OBSOLETE. The function held by products which directly regulate any step in the process of apoptosis." [GOC:jl]
GO:0016343
name: cytoskeletal anchoring activity
def: "OBSOLETE. The direct or indirect linkage of cytoskeletal filaments to the plasma membrane." [ISBN:0198599323]
GO:0016347
name: calcium-independent cell adhesion molecule activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0016361
name: activin receptor activity, type I
def: "Combining with activin-bound type II activin receptor to initiate a change in cell activity; upon binding, acts as a downstream transducer of activin signals." [GOC:mah, PMID:8622651]
GO:0016362
name: activin receptor activity, type II
def: "Combining with activin to initiate a change in cell activity; upon ligand binding, binds to and catalyses the phosphorylation of a type I activin receptor." [GOC:mah, PMID:8622651]
GO:0016401
name: palmitoyl-CoA oxidase activity
def: "Catalysis of the reaction: palmitoyl-CoA + O2 = trans-2,3-dehydropalmitoyl-CoA + hydrogen peroxide." [GOC:jsg]
GO:0016402
name: pristanoyl-CoA oxidase activity
def: "Catalysis of the reaction: pristanoyl-CoA + O2 = trans-2,3-dehydropristanoyl-CoA + hydrogen peroxide." [GOC:jsg]
GO:0016403
name: dimethylargininase activity
def: "Catalysis of the reaction: N(G),N(G)-dimethyl-L-arginine + H2O = dimethylamine + L-citrulline." [EC:3.5.3.18]
GO:0016404
name: 15-hydroxyprostaglandin dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: (5Z,13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprost-13-enoate + NAD+ = (5Z,13E)-11-alpha-hydroxy-9,15-dioxoprost-13-enoate + NADH + H+." [EC:1.1.1.141]
GO:0016405
name: CoA-ligase activity
def: "Catalysis of the reaction: substrate + ATP + CoASH = AMP + diphosphate + substrate-CoA." [GOC:ai]
GO:0016406
name: carnitine O-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to an oxygen atom on the carnitine molecule." [GOC:ai]
GO:0016407
name: acetyltransferase activity
def: "Catalysis of the transfer of an acetyl group to an acceptor molecule." [GOC:ai]
GO:0016408
name: C-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to a carbon atom on the acceptor molecule." [GOC:ai]
GO:0016409
name: palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl (CH3-[CH2]14-CO-) group to an acceptor molecule." [GOC:ai]
GO:0016410
name: N-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to a nitrogen atom on the acceptor molecule." [GOC:ai]
GO:0016411
name: acylglycerol O-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to an oxygen atom on the acylglycerol molecule." [GOC:ai]
GO:0016412
name: serine O-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to an oxygen atom on the serine molecule." [GOC:ai]
GO:0016413
name: O-acetyltransferase activity
def: "Catalysis of the transfer of an acetyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0016414
name: O-octanoyltransferase activity
def: "Catalysis of the transfer of an octanoyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0016415
name: octanoyltransferase activity
def: "Catalysis of the transfer of an octanoyl (CH3-[CH2]6-CO-) group to an acceptor molecule." [GOC:ai]
GO:0016416
name: O-palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0016417
name: S-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to a sulfur atom on the acceptor molecule." [GOC:ai]
GO:0016418
name: S-acetyltransferase activity
def: "Catalysis of the transfer of an acetyl group to a sulfur atom on the acceptor molecule." [GOC:ai]
GO:0016419
name: S-malonyltransferase activity
def: "Catalysis of the transfer of a malonyl group to a sulfur atom on the acceptor molecule." [GOC:ai]
GO:0016420
name: malonyltransferase activity
def: "Catalysis of the transfer of a malonyl (HOOC-CH2-CO-) group to an acceptor molecule." [GOC:ai]
GO:0016421
name: CoA carboxylase activity
def: "Catalysis of the joining of a carboxyl group to a molecule that is attached to CoA, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:mah]
GO:0016422
name: mRNA (2'-O-methyladenosine-N6-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + m(7)G(5')pppAm = S-adenosyl-L-homocysteine + m(7)G(5')pppm(6)Am." [EC:2.1.1.62]
GO:0016423
name: tRNA (guanine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methylguanine." [EC:2.1.1.-]
GO:0016426
name: tRNA (adenine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methyladenine." [EC:2.1.1.-]
GO:0016427
name: tRNA (cytosine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing methylcytosine." [EC:2.1.1.-]
GO:0016428
name: tRNA (cytosine-5-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing 5-methylcytosine." [EC:2.1.1.29]
GO:0016429
name: tRNA (adenine-N1-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N1-methyladenine." [EC:2.1.1.36]
GO:0016430
name: tRNA (adenine-N6-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing N6-methyladenine." [EC:2.1.1.55]
GO:0016432
name: tRNA-uridine aminocarboxypropyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA uridine = 5'-methylthioadenosine + tRNA 3-(3-amino-3-carboxypropyl)-uridine." [EC:2.5.1.25]
GO:0016433
name: rRNA (adenine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyladenine." [EC:2.1.1.-]
GO:0016434
name: rRNA (cytosine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methylcytosine." [EC:2.1.1.-]
GO:0016435
name: rRNA (guanine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methylguanine." [EC:2.1.1.-]
GO:0016436
name: rRNA (uridine) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing methyluridine." [EC:2.1.1.-]
GO:0016437
name: tRNA cytidylyltransferase activity
def: "Catalysis of the reaction: CTP + tRNA(n) = diphosphate + tRNA(n+1)." [EC:2.7.7.21]
GO:0016438
name: tRNA-queuosine beta-mannosyltransferase activity
def: "Catalysis of the reaction: GDP-mannose + tRNA(Asp)-queuosine = GDP + tRNA(Asp)-O-5''-beta-D-mannosylqueuosine." [EC:2.4.1.110]
GO:0016443
name: bidentate ribonuclease III activity
def: "Catalysis of the digestion of double-stranded RNAs into 20 to 30-nucleotide products. These products typically associate to the RNA-induced silencing complex and serve as guide RNAs for posttranslational RNA interference." [PMID:15242644]
GO:0016453
name: C-acetyltransferase activity
def: "Catalysis of the transfer of an acetyl group to a carbon atom on the acceptor molecule." [GOC:ai]
GO:0016454
name: C-palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl group to a carbon atom on the acceptor molecule." [GOC:ai]
GO:0016462
name: pyrophosphatase activity
def: "Catalysis of the hydrolysis of a pyrophosphate bond between two phosphate groups, leaving one phosphate on each of the two fragments." [EC:3.6.1.-, GOC:curators]
GO:0016463
name: zinc-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Zn2+(in) -> ADP + phosphate + Zn2+(out)." [EC:3.6.3.5]
GO:0016464
name: chloroplast protein-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate; drives the transport of proteins into the chloroplast stroma." [EC:3.6.3.52]
GO:0016466
name: hydrogen-translocating A-type ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + H+(in) = ADP + phosphate + H+(out). Found in archae." [TC:3.A.2.3.1]
GO:0016467
name: hydrogen-translocating F-type ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + H+(in) = ADP + phosphate + H+(out). Found in eukaryotic mitochondria and chloroplasts and in bacteria." [TC:3.A.2.1.1, TC:3.A.2.1.3]
GO:0016468
name: sodium-translocating F-type ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O + Na+(in) = ADP + phosphate + Na+(out). Found in eukaryotic mitochondria and chloroplasts and in bacteria." [TC:3.A.2.1.2]
GO:0016483
name: tryptophan hydroxylase activator activity
def: "Increases the activity of the enzyme tryptophase hydroxylase." [GOC:ai]
GO:0016484
name: proprotein convertase 2 activator activity
def: "OBSOLETE. Required for the maturation and activation of proprotein convertase 2." [GOC:ma, PMID:10749852]
GO:0016490
name: structural constituent of peritrophic membrane
def: "The action of a molecule that contributes to the structural integrity of the peritrophic membrane, a tubular sheath of cuticle that shields the epithelial cells of the midgut from the gut contents. An example of this is found in Drosophila melanogaster." [GOC:mtg_sensu, ISBN:0879694238]
GO:0016491
name: oxidoreductase activity
def: "Catalysis of an oxidation-reduction (redox) reaction, a reversible chemical reaction in which the oxidation state of an atom or atoms within a molecule is altered. One substrate acts as a hydrogen or electron donor and becomes oxidized, while the other acts as hydrogen or electron acceptor and becomes reduced." [GOC:go_curators]
GO:0016492
name: G-protein coupled neurotensin receptor activity
def: "Combining with the tridecapeptide neurotensin to initiate a G-protein mediated change in cell activity. A G-protein is a signal transduction molecule that alternates between an inactive GDP-bound and an active GTP-bound state." [PMID:10390649]
GO:0016493
name: C-C chemokine receptor activity
def: "Combining with a C-C chemokine and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. C-C chemokines do not have an amino acid between the first two cysteines of the characteristic four-cysteine motif." [GOC:signaling, PMID:8662823]
GO:0016494
name: C-X-C chemokine receptor activity
def: "Combining with a C-X-C chemokine and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. A C-X-C chemokine has a single amino acid between the first two cysteines of the characteristic four cysteine motif." [GOC:signaling, PMID:8662823]
GO:0016495
name: C-X3-C chemokine receptor activity
def: "Combining with a C-X3-C chemokine and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. A C-X3-C chemokine has three amino acids between the first two cysteines of the characteristic four-cysteine motif." [GOC:dph, GOC:signaling]
GO:0016496
name: substance P receptor activity
def: "Combining with substance P, the peptide Arg-Pro-Lys-Pro-Gln-Gln-Phe-Phe-Gly-Leu-Met, to initiate a change in cell activity." [GOC:mah, ISBN:0198506732]
GO:0016497
name: substance K receptor activity
def: "Combining with substance K, the peptide His-Lys-Thr-Asp-Ser-Phe-Val-Gly-Leu-Met, to initiate a change in cell activity." [GOC:mah, ISBN:0198506732]
GO:0016498
name: neuromedin K receptor activity
def: "Combining with neuromedin K, the peptide Asp-Met-His-Asp-Phe-Phe-Val-Gly-Leu-Met to initiate a change in cell activity." [GOC:mah, ISBN:0198506732]
GO:0016499
name: orexin receptor activity
def: "Combining with orexin to initiate a change in cell activity." [GOC:ai]
GO:0016500
name: protein-hormone receptor activity
def: "Combining with a protein hormone to initiate a change in cell activity." [GOC:mah]
GO:0016501
name: prostacyclin receptor activity
def: "Combining with prostacyclin (PGI(2)) to initiate a change in cell activity." [ISBN:0198506732]
GO:0016502
name: nucleotide receptor activity
def: "Combining with a nucleotide and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. A nucleotide is a compound that consists of a nucleoside esterified with a phosphate molecule." [GOC:signaling, ISBN:0198506732]
GO:0016503
name: pheromone receptor activity
def: "Combining with a pheromone to initiate a change in cell activity. A pheromone is a substance used in olfactory communication between organisms of the same species eliciting a change in sexual or social behavior." [CHEBI:26013, GOC:hjd, ISBN:0198506732]
GO:0016504
name: peptidase activator activity
def: "Increases the activity of a peptidase, any enzyme that catalyzes the hydrolysis peptide bonds." [GOC:ai]
GO:0016505
name: peptidase activator activity involved in apoptotic process
def: "Increases the activity of a peptidase that is involved in the apoptotic process." [GOC:BHF, GOC:mah, GOC:mtg_apoptosis, GOC:rl]
GO:0016506
name: apoptosis activator activity
def: "OBSOLETE. The function held by products which directly activate any step in the process of apoptosis." [GOC:hb]
GO:0016508
name: long-chain-enoyl-CoA hydratase activity
def: "Catalysis of the reaction: a long-chain (3S)-3-hydroxyacyl-CoA = a long-chain trans-2-enoyl-CoA + H2O. A long-chain acyl-CoA is an acyl-CoA thioester where the acyl chain contains 13 to 22 carbon atoms." [EC:4.2.1.74]
GO:0016509
name: long-chain-3-hydroxyacyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a long-chain fatty acid residue. A long-chain fatty acid is a fatty acid with a chain length between C13 and C22." [CHEBI:15904, EC:1.1.1.211, GOC:pde]
GO:0016511
name: endothelin-converting enzyme activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016512
name: endothelin-converting enzyme 1 activity
def: "OBSOLETE. Catalysis of the formation of endothelin 1 by cleavage of the Trp21-Val22 bond in the precursor." [EC:3.4.24.71]
GO:0016515
name: interleukin-13 receptor activity
def: "Combining with interleukin-13 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0016517
name: interleukin-12 receptor activity
def: "Combining with interleukin-12 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0016518
name: interleukin-14 receptor activity
def: "Combining with interleukin-14 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0016519
name: gastric inhibitory peptide receptor activity
def: "Combining with gastric inhibitory peptide (GIP) and transmitting the signal across the membrane to activate an associated G-protein." [GOC:mah, PMID:8243312]
GO:0016520
name: growth hormone-releasing hormone receptor activity
def: "Combining with growth hormone-releasing hormone to initiate a change in cell activity." [PMID:12529933]
GO:0016521
name: pituitary adenylate cyclase activating polypeptide activity
def: "The action characteristic of pituitary adenylate cyclase activating polypeptide, a peptide produced in the hypothalamus that binds to receptors to exert pleiotropic effects including control of neurotransmitter release, vasodilation, bronchodilation, activation of intestinal motility, increase in insulin and histamine secretion, immune modulation, and stimulation of cell proliferation and differentiation." [GOC:mah, PMID:19805477]
GO:0016524
name: latrotoxin receptor activity
def: "Combining with alpha-latrotoxin, a potent presynaptic neurotoxin, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling, PMID:10025961]
GO:0016527
name: brain-specific angiogenesis inhibitor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016530
name: metallochaperone activity
def: "Assists in the delivery of metal ions to target proteins or compartments." [GOC:jl, PMID:11739376]
GO:0016531
name: copper chaperone activity
def: "Assists in the delivery of copper ions to target proteins or compartments." [PMID:10790544, PMID:11739376]
GO:0016532
name: superoxide dismutase copper chaperone activity
def: "A copper chaperone activity that specifically delivers copper to the Cu-Zn superoxide dismutase, to activate superoxide dismutase activity." [GOC:vw, http://link.springer-ny.com/link/service/journals/00335/papers/0011005/00110409.html, PMID:15064408, PMID:9295278]
GO:0016534
name: cyclin-dependent protein kinase 5 activator activity
def: "Increases the activity of cyclin-dependent protein kinase 5." [GOC:mah]
GO:0016536
name: cyclin-dependent protein kinase 5 activator regulator activity
def: "OBSOLETE. Modulation of the activity of cyclin-dependent protein kinase 5 activator." [GOC:ai]
GO:0016538
name: cyclin-dependent protein serine/threonine kinase regulator activity
def: "Modulates the activity of a cyclin-dependent protein serine/threonine kinase, enzymes of the protein kinase family that are regulated through association with cyclins and other proteins." [GOC:pr, GOC:rn, PMID:7877684, PMID:9442875]
GO:0016541
name: intein
def: "OBSOLETE. Intervening protein sequence excised from a protein precursor in protein splicing; inteins catalyze their own excision and many also possess endonuclease activity." [GOC:mah, http://www.neb.com/neb/inteins.html, PMID:8165123]
GO:0016563
name: transcription activator activity
def: "OBSOLETE. Any transcription regulator activity required for initiation or upregulation of transcription." [GOC:jl, ISBN:0124325653]
GO:0016564
name: transcription repressor activity
def: "OBSOLETE. Any transcription regulator activity that prevents or downregulates transcription." [GOC:mah]
GO:0016565
name: general transcriptional repressor activity
def: "OBSOLETE. Any activity that stops or downregulates transcription of genes globally, and is not specific to a particular gene or gene set." [GOC:mah]
GO:0016566
name: specific transcriptional repressor activity
def: "OBSOLETE. Any activity that stops or downregulates transcription of specific genes or sets of genes." [GOC:mah]
GO:0016594
name: glycine binding
def: "Interacting selectively and non-covalently with glycine, aminoethanoic acid." [GOC:ai]
GO:0016595
name: glutamate binding
def: "Interacting selectively and non-covalently with glutamate, the anion of 2-aminopentanedioic acid." [GOC:ai]
GO:0016596
name: thienylcyclohexylpiperidine binding
def: "Interacting selectively and non-covalently with thienylcyclohexylpiperidine." [GOC:jl]
GO:0016597
name: amino acid binding
def: "Interacting selectively and non-covalently with an amino acid, organic acids containing one or more amino substituents." [GOC:ai]
GO:0016603
name: glutaminyl-peptide cyclotransferase activity
def: "Catalysis of the reaction: L-glutaminyl-peptide = 5-oxoprolyl-peptide + NH3." [EC:2.3.2.5]
GO:0016608
name: growth hormone-releasing hormone activity
def: "The action characteristic of growth hormone-releasing hormone, any of a family of peptide hormones that act on the anterior pituitary to stimulate the secretion of growth hormone and exert a trophic effect on the gland." [ISBN:0198506732]
GO:0016614
name: oxidoreductase activity, acting on CH-OH group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group act as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016615
name: malate dehydrogenase activity
def: "Catalysis of the reversible conversion of pyruvate or oxaloacetate to malate." [GOC:mah, ISBN:0582227089]
GO:0016616
name: oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces NAD+ or NADP." [EC:1.1.1.-, GOC:ai]
GO:0016617
name: 4-oxoproline reductase activity
def: "Catalysis of the reaction: 4-hydroxy-L-proline + NAD+ = 4-oxoproline + NADH + H+." [EC:1.1.1.104]
GO:0016618
name: hydroxypyruvate reductase activity
def: "Catalysis of the reaction: D-glycerate + NADP+ = hydroxypyruvate + NADPH + H+." [EC:1.1.1.81]
GO:0016620
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016621
name: cinnamoyl-CoA reductase activity
def: "Catalysis of the reaction: cinnamaldehyde + CoA + NADP+ = cinnamoyl-CoA + NADPH + H+." [EC:1.2.1.44]
GO:0016622
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces a cytochrome." [GOC:jl]
GO:0016623
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016624
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces a disulfide." [GOC:jl]
GO:0016625
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0016626
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces an acceptor other than NAD, NADP, oxygen, an iron-sulfur protein, disulphide or a cytochrome." [GOC:jl]
GO:0016627
name: oxidoreductase activity, acting on the CH-CH group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016628
name: oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016629
name: 12-oxophytodienoate reductase activity
def: "Catalysis of the reaction: 8-[(1R,2R)-3-oxo-2-{(Z)-pent-2-en-1-yl}cyclopentyl]octanoate + NADP(+) = (15Z)-12-oxophyto-10,15-dienoate + H(+) + NADPH." [EC:1.3.1.42, RHEA:21891]
GO:0016630
name: protochlorophyllide reductase activity
def: "Catalysis of the reaction: chlorophyllide a + NADP+ = protochlorophyllide + NADPH + H+." [EC:1.3.1.33]
GO:0016631
name: enoyl-[acyl-carrier-protein] reductase activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + NAD(P)+ = trans-2,3-dehydroacyl-[acyl-carrier protein] + NAD(P)H + H+." [EC:1.3.1.9, GOC:rb]
GO:0016632
name: oxidoreductase activity, acting on the CH-CH group of donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a cytochrome." [GOC:jl]
GO:0016633
name: galactonolactone dehydrogenase activity
def: "Catalysis of the reaction: L-galactono-1,4-lactone + 2 ferricytochrome c = L-ascorbate + 2 ferrocytochrome c." [EC:1.3.2.3]
GO:0016634
name: oxidoreductase activity, acting on the CH-CH group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016635
name: oxidoreductase activity, acting on the CH-CH group of donors, quinone or related compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a quinone or related compound." [GOC:jl]
GO:0016636
name: oxidoreductase activity, acting on the CH-CH group of donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0016637
name: oxidoreductase activity, acting on the CH-CH group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces an acceptor other than quinone or related compound, a cytochrome, an iron-sulfur protein, NAD, NADP or oxygen." [GOC:jl]
GO:0016638
name: oxidoreductase activity, acting on the CH-NH2 group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016639
name: oxidoreductase activity, acting on the CH-NH2 group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces NAD+ or NADP." [GOC:ai]
GO:0016640
name: oxidoreductase activity, acting on the CH-NH2 group of donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces a cytochrome molecule." [GOC:ai]
GO:0016641
name: oxidoreductase activity, acting on the CH-NH2 group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces an oxygen molecule." [GOC:ai]
GO:0016642
name: oxidoreductase activity, acting on the CH-NH2 group of donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces a disulfide group." [GOC:ai]
GO:0016643
name: oxidoreductase activity, acting on the CH-NH2 group of donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:ai]
GO:0016644
name: oxidoreductase activity, acting on the CH-NH2 group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces an acceptor other than a cytochrome, disulfide, an iron-sulfur protein, NAD, NADP or oxygen." [GOC:ai]
GO:0016645
name: oxidoreductase activity, acting on the CH-NH group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016646
name: oxidoreductase activity, acting on the CH-NH group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016647
name: oxidoreductase activity, acting on the CH-NH group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016648
name: oxidoreductase activity, acting on the CH-NH group of donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces disulfide." [GOC:jl]
GO:0016649
name: oxidoreductase activity, acting on the CH-NH group of donors, quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces quinone or similar compound." [GOC:jl]
GO:0016650
name: oxidoreductase activity, acting on the CH-NH group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces an acceptor other than quinone or similar compound, disulfide, NAD, NADP, oxygen or a flavin." [GOC:jl]
GO:0016651
name: oxidoreductase activity, acting on NAD(P)H
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016652
name: oxidoreductase activity, acting on NAD(P)H, NAD(P) as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces NAD+ or NADP." [GOC:ai]
GO:0016653
name: oxidoreductase activity, acting on NAD(P)H, heme protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces a heme protein." [GOC:ai]
GO:0016655
name: oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces a quinone or a similar acceptor molecule." [GOC:ai]
GO:0016656
name: monodehydroascorbate reductase (NADH) activity
def: "Catalysis of the reaction: NADH + H+ + 2 monodehydroascorbate = NAD+ + 2 ascorbate." [EC:1.6.5.4]
GO:0016657
name: oxidoreductase activity, acting on NAD(P)H, nitrogenous group as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces a nitrogenous group." [GOC:ai]
GO:0016658
name: oxidoreductase activity, acting on NADH or NADPH, flavin as acceptor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0016659
name: oxidoreductase activity, acting on NADH or NADPH, other acceptor
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces an acceptor other than disulfide, a heme protein, NAD, NADP, a nitrogenous group, a quinone or similar compound or oxygen." [GOC:jl]
GO:0016661
name: oxidoreductase activity, acting on other nitrogenous compounds as donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016662
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces a cytochrome." [GOC:jl]
GO:0016663
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016664
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0016665
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces an acceptor other than a cytochrome, an iron-sulfur protein, oxygen, NAD or NADP." [GOC:jl]
GO:0016667
name: oxidoreductase activity, acting on a sulfur group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016668
name: oxidoreductase activity, acting on a sulfur group of donors, NAD(P) as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016669
name: oxidoreductase activity, acting on a sulfur group of donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces a cytochrome." [GOC:jl]
GO:0016670
name: oxidoreductase activity, acting on a sulfur group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016671
name: oxidoreductase activity, acting on a sulfur group of donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces disulfide." [GOC:jl]
GO:0016672
name: oxidoreductase activity, acting on a sulfur group of donors, quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces quinone or a related compound." [GOC:jl]
GO:0016673
name: oxidoreductase activity, acting on a sulfur group of donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0016674
name: oxidoreductase activity, acting on sulfur group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a sulfur-containing group acts as a hydrogen or electron donor and reduces an acceptor other than quinone or a related compound, oxygen, NAD, NADP, an iron-sulfur protein, disulfide or a cytochrome." [GOC:jl]
GO:0016675
name: oxidoreductase activity, acting on a heme group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a heme group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016676
name: oxidoreductase activity, acting on a heme group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a heme group acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016677
name: oxidoreductase activity, acting on a heme group of donors, nitrogenous group as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a heme group acts as a hydrogen or electron donor and reduces a nitrogenous group." [GOC:jl]
GO:0016678
name: oxidoreductase activity, acting on heme group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a heme group acts as a hydrogen or electron donor and reduces an acceptor other than a nitrogenous group or oxygen." [GOC:jl]
GO:0016679
name: oxidoreductase activity, acting on diphenols and related substances as donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a diphenol or related substance acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016680
name: oxidoreductase activity, acting on diphenols and related substances as donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a diphenol, or related compound, acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016681
name: oxidoreductase activity, acting on diphenols and related substances as donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a diphenol, or related compound, acts as a hydrogen or electron donor and reduces a cytochrome." [GOC:jl]
GO:0016682
name: oxidoreductase activity, acting on diphenols and related substances as donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a diphenol, or related compound, acts as a hydrogen or electron donor and reduces oxygen." [GOC:jl]
GO:0016683
name: oxidoreductase activity, acting on diphenols and related substances as donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a diphenol, or related compound, acts as a hydrogen or electron donor and reduces an acceptor other than a cytochrome, NAD, NADP or oxygen." [GOC:jl]
GO:0016684
name: oxidoreductase activity, acting on peroxide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which the peroxide group acts as a hydrogen or electron acceptor." [GOC:ai]
GO:0016688
name: L-ascorbate peroxidase activity
def: "Catalysis of the reaction: L-ascorbate + hydrogen peroxide = dehydroascorbate + 2 H2O." [EC:1.11.1.11]
GO:0016689
name: manganese peroxidase activity
def: "Catalysis of the reaction: 2 Mn2+ + 2 H+ + hydrogen peroxide = 2 Mn3+ + 2 H2O." [EC:1.11.1.13]
GO:0016690
name: diarylpropane peroxidase activity
def: "Catalysis of the reaction: (3,4-dimethoxyphenyl)methanol + H2O2 = 3,4-dimethoxybenzaldehyde + 2 H2O." [EC:1.11.1.14]
GO:0016691
name: chloride peroxidase activity
def: "Catalysis of the reaction: 2 R-H + 2 chloride + hydrogen peroxide = 2 R-Cl + 2 H2O." [EC:1.11.1.10]
GO:0016692
name: NADH peroxidase activity
def: "Catalysis of the reaction: H(2)O(2) + H(+) + NADH = 2 H(2)O + NAD(+)." [EC:1.11.1.1, RHEA:18512]
GO:0016694
name: bacterial catalase-peroxidase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016695
name: oxidoreductase activity, acting on hydrogen as donor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen acts as an electron donor." [GOC:jl]
GO:0016696
name: oxidoreductase activity, acting on hydrogen as donor, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen acts as an electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016697
name: oxidoreductase activity, acting on hydrogen as donor, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen acts as an electron donor and reduces a cytochrome." [GOC:jl]
GO:0016699
name: oxidoreductase activity, acting on hydrogen as donor, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen acts as an electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0016700
name: oxidoreductase activity, acting on hydrogen as donor, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which hydrogen is the electron donor and reduces an acceptor other than a cytochrome, an iron-sulfur protein, NAD, NADP or a quinone or similar compound." [GOC:ai]
GO:0016701
name: oxidoreductase activity, acting on single donors with incorporation of molecular oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from one donor, and molecular oxygen is incorporated into a donor." [GOC:mah]
GO:0016702
name: oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of two atoms of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from one donor, and two oxygen atoms is incorporated into a donor." [GOC:mah]
GO:0016703
name: oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of one atom of oxygen (internal monooxygenases or internal mixed function oxidases)
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from one donor, and one oxygen atom is incorporated into a donor." [GOC:mah]
GO:0016704
name: oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, miscellaneous
def: "OBSOLETE. A grouping term for oxidoreductases acting on single donors with incorporation of molecular oxygen that cannot be more accurated categorized." [GOC:ai]
GO:0016705
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from each of two donors, and molecular oxygen is reduced or incorporated into a donor." [GOC:mah]
GO:0016706
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, 2-oxoglutarate as one donor, and incorporation of one atom each of oxygen into both donors
def: "Catalysis of the reaction: A + 2-oxoglutarate + O2 = B + succinate + CO2. This is an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from 2-oxoglutarate and one other donor, and one atom of oxygen is incorporated into each donor." [GOC:mah]
GO:0016707
name: gibberellin 3-beta-dioxygenase activity
def: "Catalysis of the reaction: a gibberellin + 2-oxoglutarate + O2 = a 3-beta-hydroxy-gibberellin + succinate + CO2." [EC:1.14.11.15, GOC:kad]
GO:0016708
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of two atoms of oxygen into one donor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from NADH or NADPH and one other donor, and two atoms of oxygen are incorporated into one donor." [GOC:mah]
GO:0016709
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, NAD(P)H as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from NADH or NADPH and one other donor, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016710
name: trans-cinnamate 4-monooxygenase activity
def: "Catalysis of the reaction: trans-cinnamate + NADPH + H+ + O2 = 4-hydroxycinnamate + NADP+ + H2O." [EC:1.14.13.11]
GO:0016711
name: flavonoid 3'-monooxygenase activity
def: "Catalysis of the reaction: a flavonoid + NADPH + H+ + O2 = 3'-hydroxyflavonoid + NADP+ + H2O." [EC:1.14.13.21]
GO:0016712
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced flavin or flavoprotein as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from reduced flavin or flavoprotein and one other donor, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016713
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from reduced iron-sulfur protein and one other donor, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016714
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced pteridine as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from reduced pteridine and one other donor, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016715
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced ascorbate as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from reduced ascorbate and one other donor, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016716
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, another compound as one donor, and incorporation of one atom of oxygen
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from each of two donors, and one atom of oxygen is incorporated into one donor." [GOC:mah]
GO:0016717
name: oxidoreductase activity, acting on paired donors, with oxidation of a pair of donors resulting in the reduction of molecular oxygen to two molecules of water
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from each of two donors, and molecular oxygen is reduced to two molecules of water." [GOC:mah]
GO:0016718
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, miscellaneous
def: "OBSOLETE. A grouping term for oxidoreductases, acting on paired donors with incorporation or reduction of molecular oxygen, that cannot be more accurately categorized." [GOC:ai]
GO:0016719
name: carotene 7,8-desaturase activity
def: "Catalysis of the reaction: neurosporene + donor-H2 + O2 = lycopene + acceptor + 2 H2O." [EC:1.14.99.30]
GO:0016720
name: delta12-fatty acid dehydrogenase activity
def: "Catalysis of the reaction: AH(2) + linoleate + O(2) = A + crepenynate + 2 H(2)O." [EC:1.14.99.33, RHEA:23459]
GO:0016721
name: oxidoreductase activity, acting on superoxide radicals as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a superoxide radical (O2- or O2.-) acts as a hydrogen or electron acceptor." [GOC:ai]
GO:0016722
name: oxidoreductase activity, oxidizing metal ions
def: "Catalysis of an oxidation-reduction in which the oxidation state of metal ion is altered." [GOC:mah]
GO:0016723
name: oxidoreductase activity, oxidizing metal ions, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction in which the oxidation state of metal ion is altered and NAD+ or NADP+ acts as an electron acceptor." [GOC:mah]
GO:0016724
name: oxidoreductase activity, oxidizing metal ions, oxygen as acceptor
def: "Catalysis of an oxidation-reduction in which the oxidation state of metal ion is altered and oxygen acts as an electron acceptor." [GOC:mah]
GO:0016725
name: oxidoreductase activity, acting on CH or CH2 groups
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016726
name: oxidoreductase activity, acting on CH or CH2 groups, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces NAD+ or NADP." [GOC:ai]
GO:0016727
name: oxidoreductase activity, acting on CH or CH2 groups, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces an oxygen molecule." [GOC:ai]
GO:0016728
name: oxidoreductase activity, acting on CH or CH2 groups, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces a disulfide group." [GOC:ai]
GO:0016729
name: oxidoreductase activity, acting on CH2 groups, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces an acceptor other than disulfide, NAD, NADP or oxygen." [GOC:jl]
GO:0016730
name: oxidoreductase activity, acting on iron-sulfur proteins as donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which an iron-sulfur protein acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016731
name: oxidoreductase activity, acting on iron-sulfur proteins as donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an iron-sulfur protein acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0016732
name: oxidoreductase activity, acting on iron-sulfur proteins as donors, dinitrogen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an iron-sulfur protein acts as a hydrogen or electron donor and reduces dinitrogen." [GOC:jl]
GO:0016733
name: iron-iron nitrogenase activity
def: "OBSOLETE. Catalysis of the reaction: 8 reduced ferredoxin + 8 H+ + N2 + 16 ATP = 8 oxidized ferredoxin + 2 NH3 + 16 ADP + 16 phosphate." [EC:1.18.6.1]
GO:0016734
name: molybdenum-iron nitrogenase activity
def: "OBSOLETE. Catalysis of the reaction: 8 reduced ferredoxin + 8 H+ + N2 + 16 ATP = 8 oxidized ferredoxin + 2 NH3 + 16 ADP + 16 phosphate." [EC:1.18.6.1]
GO:0016735
name: vanadium-iron nitrogenase activity
def: "OBSOLETE. Catalysis of the reaction: 8 reduced ferredoxin + 8 H+ + nitrogen + 16 ATP = 8 oxidized ferredoxin + 2 NH3 + 16 ADP + 16 phosphate." [EC:1.18.6.1]
GO:0016737
name: oxidoreductase activity, acting on reduced flavodoxin as donor
def: "Catalysis of an oxidation-reduction (redox) reaction in which reduced flavodoxin acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016738
name: oxidoreductase activity, acting on reduced flavodoxin as donor, dinitrogen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which reduced flavodoxin acts as a hydrogen or electron donor and reduces dinitrogen." [GOC:jl]
GO:0016739
name: oxidoreductase activity, acting on other substrates
def: "OBSOLETE. A grouping term for oxidoreductase that cannot be more accurately categorized." [GOC:ai]
GO:0016740
name: transferase activity
def: "Catalysis of the transfer of a group, e.g. a methyl group, glycosyl group, acyl group, phosphorus-containing, or other groups, from one compound (generally regarded as the donor) to another compound (generally regarded as the acceptor). Transferase is the systematic name for any enzyme of EC class 2." [ISBN:0198506732]
GO:0016741
name: transferase activity, transferring one-carbon groups
def: "Catalysis of the transfer of a one-carbon group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016742
name: hydroxymethyl-, formyl- and related transferase activity
def: "Catalysis of the transfer of a hydroxymethyl- or formyl group from one compound (donor) to another (acceptor)." [EC:2.1.2, GOC:mah]
GO:0016743
name: carboxyl- or carbamoyltransferase activity
def: "Catalysis of the transfer of a carboxyl- or carbamoyl group from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016744
name: transferase activity, transferring aldehyde or ketonic groups
def: "Catalysis of the transfer of an aldehyde or ketonic group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016746
name: transferase activity, transferring acyl groups
def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016747
name: transferase activity, transferring acyl groups other than amino-acyl groups
def: "Catalysis of the transfer of an acyl group, other than amino-acyl, from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016748
name: succinyltransferase activity
def: "Catalysis of the transfer of a succinyl (3-carboxypropanoyl) group to an acceptor molecule." [GOC:ai]
GO:0016749
name: N-succinyltransferase activity
def: "Catalysis of the transfer of a succinyl group to a nitrogen atom on the acceptor molecule." [GOC:ai]
GO:0016750
name: O-succinyltransferase activity
def: "Catalysis of the transfer of a succinyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0016751
name: S-succinyltransferase activity
def: "Catalysis of the transfer of a succinyl group to a sulfur atom on the acceptor molecule." [GOC:ai]
GO:0016752
name: sinapoyltransferase activity
def: "Catalysis of the transfer of a sinapoyl group to an acceptor molecule." [GOC:ai]
GO:0016753
name: O-sinapoyltransferase activity
def: "Catalysis of the transfer of a sinapoyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0016754
name: sinapoylglucose-malate O-sinapoyltransferase activity
def: "Catalysis of the reaction: (S)-malate + 1-O-sinapoyl-beta-D-glucose = D-glucose + sinapoyl (S)-malate." [EC:2.3.1.92, RHEA:12628]
GO:0016755
name: transferase activity, transferring amino-acyl groups
def: "Catalysis of the transfer of an amino-acyl group from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016756
name: glutathione gamma-glutamylcysteinyltransferase activity
def: "Catalysis of the reaction: glutathione + Glu(-Cys)(n)-Gly = Gly + Glu(-Cys)(n+1)-Gly." [EC:2.3.2.15]
GO:0016757
name: transferase activity, transferring glycosyl groups
def: "Catalysis of the transfer of a glycosyl group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016758
name: transferase activity, transferring hexosyl groups
def: "Catalysis of the transfer of a hexosyl group from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016759
name: cellulose synthase activity
def: "Catalysis of the reaction: nucleoside-disphosphate-glucose + ((1,4)-beta-D-glucosyl)(n) = nucleoside-disphosphate + ((1,4)-beta-D-glucosyl)(n+1)." [EC:2.4.1.12, EC:2.4.1.29]
GO:0016760
name: cellulose synthase (UDP-forming) activity
def: "Catalysis of the reaction: UDP-glucose + ((1,4)-beta-D-glucosyl)(n) = UDP + ((1,4)-beta-D-glucosyl)(n+1)." [EC:2.4.1.12]
GO:0016761
name: cellulose synthase (GDP-forming) activity
def: "Catalysis of the reaction: GDP-glucose + ((1,4)-beta-D-glucosyl)(n) = GDP + ((1,4)-beta-D-glucosyl)(n+1)." [EC:2.4.1.29]
GO:0016762
name: xyloglucan:xyloglucosyl transferase activity
def: "Catalysis of the cleavage of a beta-(1->4) bond in the backbone of a xyloglucan and transfers the xyloglucanyl segment on to O-4 of the non-reducing terminal glucose residue of an acceptor, which can be a xyloglucan or an oligosaccharide of xyloglucan." [EC:2.4.1.207]
GO:0016763
name: transferase activity, transferring pentosyl groups
def: "Catalysis of the transfer of a pentosyl group from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016764
name: transferase activity, transferring other glycosyl groups
def: "OBSOLETE. Catalysis of the transfer of a glycosyl group, other than hexosyl or pentosyl, from one compound (donor) to another (acceptor)." [GOC:jl]
GO:0016765
name: transferase activity, transferring alkyl or aryl (other than methyl) groups
def: "Catalysis of the transfer of an alkyl or aryl (but not methyl) group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016767
name: geranylgeranyl-diphosphate geranylgeranyltransferase activity
def: "Catalysis of the reaction: 2 geranylgeranyl diphosphate = diphosphate + prephytoene diphosphate." [EC:2.5.1.32]
GO:0016768
name: spermine synthase activity
def: "Catalysis of the reaction: S-adenosylmethioninamine + spermidine = 5'-methylthioadenosine + spermine." [EC:2.5.1.22]
GO:0016769
name: transferase activity, transferring nitrogenous groups
def: "Catalysis of the transfer of a nitrogenous group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016770
name: oximinotransaminase activity
def: "Catalysis of the reversible transfer of an oxime group to an acceptor." [GOC:ai]
GO:0016771
name: transferase activity, transferring other nitrogenous groups
def: "OBSOLETE. Catalysis of the transfer of a nitrogenous group, other than amino, amidino or oxime, from one compound (donor) to another (acceptor)." [GOC:ai]
GO:0016772
name: transferase activity, transferring phosphorus-containing groups
def: "Catalysis of the transfer of a phosphorus-containing group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016773
name: phosphotransferase activity, alcohol group as acceptor
def: "Catalysis of the transfer of a phosphorus-containing group from one compound (donor) to an alcohol group (acceptor)." [GOC:jl]
GO:0016774
name: phosphotransferase activity, carboxyl group as acceptor
def: "Catalysis of the transfer of a phosphorus-containing group from one compound (donor) to a carboxyl group (acceptor)." [GOC:jl]
GO:0016775
name: phosphotransferase activity, nitrogenous group as acceptor
def: "Catalysis of the transfer of a phosphorus-containing group from one compound (donor) to a nitrogenous group (acceptor)." [GOC:jl]
GO:0016776
name: phosphotransferase activity, phosphate group as acceptor
def: "Catalysis of the transfer of a phosphorus-containing group from one compound (donor) to a phosphate group (acceptor)." [GOC:jl]
GO:0016778
name: diphosphotransferase activity
def: "Catalysis of the transfer of a diphosphate group from one compound (donor) to a another (acceptor)." [GOC:jl, http://www.chem.qmul.ac.uk/iubmb/enzyme/EC2/intro.html]
GO:0016779
name: nucleotidyltransferase activity
def: "Catalysis of the transfer of a nucleotidyl group to a reactant." [ISBN:0198506732]
GO:0016780
name: phosphotransferase activity, for other substituted phosphate groups
def: "Catalysis of the transfer of a substituted phosphate group, other than diphosphate or nucleotidyl residues, from one compound (donor) to a another (acceptor)." [GOC:jl, http://www.chem.qmul.ac.uk/iubmb/enzyme/EC2/intro.html]
GO:0016781
name: phosphotransferase activity, paired acceptors
def: "Catalysis of the transfer of two phosphate groups from a donor, such as ATP, to two different acceptors." [GOC:jl, http://www.chem.qmul.ac.uk/iubmb/enzyme/EC2/intro.html]
GO:0016782
name: transferase activity, transferring sulfur-containing groups
def: "Catalysis of the transfer of a sulfur-containing group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016783
name: sulfurtransferase activity
def: "Catalysis of the transfer of sulfur atoms from one compound (donor) to another (acceptor)." [GOC:ai, ISBN:0721662544]
GO:0016784
name: 3-mercaptopyruvate sulfurtransferase activity
def: "Catalysis of the reaction: 3-mercaptopyruvate + cyanide = pyruvate + thiocyanate." [EC:2.8.1.2]
GO:0016785
name: transferase activity, transferring selenium-containing groups
def: "Catalysis of the transfer of a selenium-containing group from one compound (donor) to another (acceptor)." [GOC:jl, ISBN:0198506732]
GO:0016787
name: hydrolase activity
def: "Catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3." [ISBN:0198506732]
GO:0016788
name: hydrolase activity, acting on ester bonds
def: "Catalysis of the hydrolysis of any ester bond." [GOC:jl]
GO:0016790
name: thiolester hydrolase activity
def: "Catalysis of the reaction: RCO-SR' + H2O = RCOOH + HSR'. This reaction is the hydrolysis of a thiolester bond, an ester formed from a carboxylic acid and a thiol (i.e., RCO-SR'), such as that found in acetyl-coenzyme A." [http://www.onelook.com]
GO:0016791
name: phosphatase activity
def: "Catalysis of the hydrolysis of phosphoric monoesters, releasing inorganic phosphate." [EC:3.1.3, EC:3.1.3.41, GOC:curators]
GO:0016793
name: triphosphoric monoester hydrolase activity
def: "Catalysis of the hydrolysis of a triphosphoester to give a triphosphate group and a free hydroxyl group." [GOC:ai]
GO:0016794
name: diphosphoric monoester hydrolase activity
def: "Catalysis of the hydrolysis of a diphosphoester to give a diphosphate group and a free hydroxyl group." [GOC:ai]
GO:0016795
name: phosphoric triester hydrolase activity
def: "Catalysis of the hydrolysis of a phosphoric triester." [EC:3.1.8, GOC:curators]
GO:0016796
name: exonuclease activity, active with either ribo- or deoxyribonucleic acids and producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 3' or 5' end to yield 5' phosphomonoesters." [GOC:mah]
GO:0016797
name: exonuclease activity, active with either ribo- or deoxyribonucleic acids and producing 3'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 3' or 5' end to yield 3' phosphomonoesters." [GOC:mah]
GO:0016798
name: hydrolase activity, acting on glycosyl bonds
def: "Catalysis of the hydrolysis of any glycosyl bond." [GOC:jl]
GO:0016799
name: hydrolase activity, hydrolyzing N-glycosyl compounds
def: "Catalysis of the hydrolysis of any N-glycosyl bond." [GOC:jl]
GO:0016801
name: hydrolase activity, acting on ether bonds
def: "Catalysis of the hydrolysis of any ether or thioether bond, -O- or -S- respectively." [GOC:ai, GOC:jl]
GO:0016802
name: trialkylsulfonium hydrolase activity
def: "Catalysis of the hydrolysis of a thioether bond, -S-." [EC:3.3.1.-, GOC:ai]
GO:0016803
name: ether hydrolase activity
def: "Catalysis of the hydrolysis of an ether bond, -O-." [EC:3.3.2.-, GOC:ai]
GO:0016804
name: prolyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of a N-terminal proline from a peptide." [EC:3.4.11.5]
GO:0016805
name: dipeptidase activity
def: "Catalysis of the hydrolysis of a dipeptide." [http://www.onelook.com]
GO:0016806
name: dipeptidyl-peptidase and tripeptidyl-peptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of N-terminal di- or tripeptides from a polypeptide chain." [GOC:ai]
GO:0016807
name: cysteine-type carboxypeptidase activity
def: "Catalysis of the hydrolysis of C-terminal peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CARBOXYPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0016808
name: proprotein convertase activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0016810
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
def: "Catalysis of the hydrolysis of any carbon-nitrogen bond, C-N, with the exception of peptide bonds." [GOC:jl]
GO:0016811
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a linear amide." [GOC:ai]
GO:0016812
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a cyclic amide." [GOC:ai]
GO:0016813
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amidines
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a linear amidine, a compound of the form R-C(=NH)-NH2." [ISBN:0198506732]
GO:0016814
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a cyclic amidine, a compound of the form R-C(=NH)-NH2." [ISBN:0198506732]
GO:0016815
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in nitriles
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a nitrile, a compound containing the cyano radical, -CN." [http://www.onelook.com/]
GO:0016816
name: hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in other compounds
def: "OBSOLETE. Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in substances other than amides, amidines and nitriles." [GOC:ai]
GO:0016817
name: hydrolase activity, acting on acid anhydrides
def: "Catalysis of the hydrolysis of any acid anhydride." [GOC:jl]
GO:0016818
name: hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides
def: "Catalysis of the hydrolysis of any acid anhydride which contains phosphorus." [GOC:jl]
GO:0016819
name: hydrolase activity, acting on acid anhydrides, in sulfonyl-containing anhydrides
def: "Catalysis of the hydrolysis of any acid anhydride which contains a sulfonyl group, -SO2-." [GOC:ai]
GO:0016820
name: hydrolase activity, acting on acid anhydrides, catalyzing transmembrane movement of substances
def: "Catalysis of the hydrolysis of an acid anhydride to directly drive the transport of a substance across a membrane." [GOC:mah]
GO:0016821
name: hydrolase activity, acting on acid anhydrides, involved in cellular and subcellular movement
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016822
name: hydrolase activity, acting on acid carbon-carbon bonds
def: "Catalysis of the hydrolysis of any acid carbon-carbon bond." [GOC:jl]
GO:0016823
name: hydrolase activity, acting on acid carbon-carbon bonds, in ketonic substances
def: "Catalysis of the hydrolysis of any acid carbon-carbon bond in a ketonic substance, a substance containing a keto (C=O) group." [GOC:ai]
GO:0016824
name: hydrolase activity, acting on acid halide bonds
def: "Catalysis of the hydrolysis of any acid halide bond." [GOC:jl]
GO:0016825
name: hydrolase activity, acting on acid phosphorus-nitrogen bonds
def: "Catalysis of the hydrolysis of any acid phosphorus-nitrogen bond." [GOC:jl]
GO:0016826
name: hydrolase activity, acting on acid sulfur-nitrogen bonds
def: "Catalysis of the hydrolysis of any acid sulfur-nitrogen bond." [GOC:jl]
GO:0016827
name: hydrolase activity, acting on acid carbon-phosphorus bonds
def: "Catalysis of the hydrolysis of any acid carbon-phosphorus bond." [GOC:jl]
GO:0016828
name: hydrolase activity, acting on acid sulfur-sulfur bonds
def: "Catalysis of the hydrolysis of any acid sulfur-sulfur bond." [GOC:jl]
GO:0016829
name: lyase activity
def: "Catalysis of the cleavage of C-C, C-O, C-N and other bonds by other means than by hydrolysis or oxidation, or conversely adding a group to a double bond. They differ from other enzymes in that two substrates are involved in one reaction direction, but only one in the other direction. When acting on the single substrate, a molecule is eliminated and this generates either a new double bond or a new ring." [EC:4.-.-.-, ISBN:0198547684]
GO:0016830
name: carbon-carbon lyase activity
def: "Catalysis of the cleavage of C-C bonds by other means than by hydrolysis or oxidation, or conversely adding a group to a double bond." [GOC:jl]
GO:0016831
name: carboxy-lyase activity
def: "Catalysis of the nonhydrolytic addition or removal of a carboxyl group to or from a compound." [http://www.mercksource.com/]
GO:0016832
name: aldehyde-lyase activity
def: "Catalysis of the cleavage of a C-C bond in a molecule containing a hydroxyl group and a carbonyl group to form two smaller molecules, each being an aldehyde or a ketone." [http://www.mercksource.com/]
GO:0016833
name: oxo-acid-lyase activity
def: "Catalysis of the cleavage of a C-C bond by other means than by hydrolysis or oxidation, of a 3-hydroxy acid." [EC:4.1.3, GOC:jl]
GO:0016834
name: other carbon-carbon lyase activity
def: "OBSOLETE. A grouping term for carbon-carbon lyases that cannot be more accurately categorized." [GOC:ai]
GO:0016835
name: carbon-oxygen lyase activity
def: "Catalysis of the breakage of a carbon-oxygen bond." [EC:4.2.-.-]
GO:0016836
name: hydro-lyase activity
def: "Catalysis of the cleavage of a carbon-oxygen bond by elimination of water." [EC:4.2.1]
GO:0016837
name: carbon-oxygen lyase activity, acting on polysaccharides
def: "Catalysis of the cleavage of a carbon-oxygen bond by the elimination of an alcohol from a polysaccharide." [EC:4.2.-.-]
GO:0016838
name: carbon-oxygen lyase activity, acting on phosphates
def: "Catalysis of the cleavage of a carbon-oxygen bond by elimination of a phosphate." [EC:4.2.-.-]
GO:0016839
name: other carbon-oxygen lyase activity
def: "OBSOLETE. Catalysis of the cleavage of a carbon-oxygen bond. Enzymes with this activity are 'miscellaneous' carbon-oxygen lyases that cannot be grouped into one of the specific subclasses of the carbon-oxygen lyases." [GOC:krc]
GO:0016840
name: carbon-nitrogen lyase activity
def: "Catalysis of the release of ammonia or one of its derivatives, with the formation of a double bond or ring. Enzymes with this activity may catalyze the actual elimination of the ammonia, amine or amide, e.g. CH-CH(-NH-R) = C=CH- + NH2-R. Others, however, catalyze elimination of another component, e.g. water, which is followed by spontaneous reactions that lead to breakage of the C-N bond, e.g. L-serine ammonia-lyase (EC:4.3.1.17), so that the overall reaction is C(-OH)-CH(-NH2) = CH2-CO- + NH3, i.e. an elimination with rearrangement. The sub-subclasses of EC:4.3 are the ammonia-lyases (EC:4.3.1), lyases acting on amides, amidines, etc. (EC:4.3.2), the amine-lyases (EC:4.3.3), and other carbon-nitrogen lyases (EC:4.3.99)." [EC:4.3.-.-]
GO:0016841
name: ammonia-lyase activity
def: "Catalysis of the release of ammonia by the cleavage of a carbon-nitrogen bond or the reverse reaction with ammonia as a substrate." [EC:4.3.-.-, GOC:krc]
GO:0016842
name: amidine-lyase activity
def: "Catalysis of the release of amides or amidines by the cleavage of a carbon-nitrogen bond or the reverse reaction with an amide or amidine as a substrate." [EC:4.3.-.-, GOC:krc]
GO:0016843
name: amine-lyase activity
def: "Catalysis of the release of amides by the cleavage of a carbon-nitrogen bond or the reverse reaction with an amine as a substrate." [EC:4.3.-.-, GOC:krc]
GO:0016844
name: strictosidine synthase activity
def: "Catalysis of the reaction: 3alpha(S)-strictosidine + H(2)O = secologanin + tryptamine." [EC:4.3.3.2, RHEA:15016]
GO:0016845
name: other carbon-nitrogen lyase activity
def: "OBSOLETE. Catalysis of the cleavage of a carbon-nitrogen bond. Enzymes with this activity are 'miscellaneous' carbon-nitrogen lyases that cannot be grouped into one of the specific subclasses of the carbon-nitrogen lyases." [GOC:krc]
GO:0016846
name: carbon-sulfur lyase activity
def: "Catalysis of the elimination of hydrogen sulfide or substituted H2S." [EC:4.4.-.-]
GO:0016847
name: 1-aminocyclopropane-1-carboxylate synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) = 1-aminocyclopropane-1-carboxylate + S-methyl-5'-thioadenosine + H(+)." [EC:4.4.1.14, RHEA:21747]
GO:0016848
name: carbon-halide lyase activity
def: "Catalysis of the breakage of a bond between carbon and any halogen atom." [GOC:mah]
GO:0016849
name: phosphorus-oxygen lyase activity
def: "Catalysis of the cleavage of a phosphorus-oxygen bond by other means than by hydrolysis or oxidation, or conversely adding a group to a double bond." [GOC:jl]
GO:0016850
name: other lyase activity
def: "OBSOLETE. A grouping term for lyases that cannot be more accurately categorized." [GOC:ai]
GO:0016851
name: magnesium chelatase activity
def: "Catalysis of the reaction: ATP + H(2)O + Mg(2+) + protoporphyrin IX = ADP + 2 H(+) + magnesium protoporphyrin IX + phosphate." [EC:6.6.1.1, RHEA:13964]
GO:0016852
name: sirohydrochlorin cobaltochelatase activity
def: "Catalysis of the reaction: sirohydrochlorin + Co2+ = cobalt-sirohydrochlorin + 2 H+." [EC:4.99.1.3]
GO:0016853
name: isomerase activity
def: "Catalysis of the geometric or structural changes within one molecule. Isomerase is the systematic name for any enzyme of EC class 5." [ISBN:0198506732]
GO:0016854
name: racemase and epimerase activity
def: "Catalysis of a reaction that alters the configuration of one or more chiral centers in a molecule." [GOC:mah, ISBN:0198506732]
GO:0016855
name: racemase and epimerase activity, acting on amino acids and derivatives
def: "Catalysis of a reaction that alters the configuration of one or more chiral centers in an amino acid." [GOC:mah]
GO:0016856
name: racemase and epimerase activity, acting on hydroxy acids and derivatives
def: "Catalysis of a reaction that alters the configuration of one or more chiral centers in a hydroxy acid molecule." [GOC:mah]
GO:0016857
name: racemase and epimerase activity, acting on carbohydrates and derivatives
def: "Catalysis of a reaction that alters the configuration of one or more chiral centers in a carbohydrate molecule." [GOC:mah]
GO:0016858
name: racemase and epimerase activity, acting on other compounds
def: "OBSOLETE. Racemase and epimerase activity on compounds other than amino acids, hydroxy acids, carbohydrates or their derivatives." [GOC:ai]
GO:0016859
name: cis-trans isomerase activity
def: "Catalysis of a reaction that interconverts cis and trans isomers. Atoms or groups are termed cis or trans to one another when they lie respectively on the same or on opposite sides of a reference plane identifiable as common among stereoisomers." [GOC:mah, ISBN:0198506732]
GO:0016860
name: intramolecular oxidoreductase activity
def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor are the same molecule, and no oxidized product appears." [EC:5.3.-.-]
GO:0016861
name: intramolecular oxidoreductase activity, interconverting aldoses and ketoses
def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor, which is an aldose or a ketose, are the same molecule, and no oxidized product appears." [GOC:jl]
GO:0016862
name: intramolecular oxidoreductase activity, interconverting keto- and enol-groups
def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor, which is a keto- or an enol-group, are the same molecule, and no oxidized product appears." [GOC:jl]
GO:0016863
name: intramolecular oxidoreductase activity, transposing C=C bonds
def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor are the same molecule, one or more carbon-carbon double bonds in the molecule are rearranged, and no oxidized product appears." [EC:5.3.3, GOC:mah]
GO:0016864
name: intramolecular oxidoreductase activity, transposing S-S bonds
def: "Catalysis of an oxidation-reduction (redox) reaction in which the hydrogen donor and acceptor are the same molecule, one or more sulfur-sulfur bonds in the molecule are rearranged, and no oxidized product appears." [EC:5.3.4, GOC:mah]
GO:0016865
name: intramolecular oxidoreductase activity, other intramolecular oxidoreductases
def: "OBSOLETE. A grouping term for intramolecular oxidoreductases that cannot be more accurately categorized." [GOC:ai]
GO:0016866
name: intramolecular transferase activity
def: "Catalysis of the transfer of a functional group from one position to another within a single molecule." [GOC:mah]
GO:0016867
name: intramolecular transferase activity, transferring acyl groups
def: "Catalysis of the transfer of an acyl group from one position to another within a single molecule." [GOC:mah]
GO:0016868
name: intramolecular transferase activity, phosphotransferases
def: "Catalysis of the transfer of a phosphate group from one position to another within a single molecule." [GOC:mah]
GO:0016869
name: intramolecular transferase activity, transferring amino groups
def: "Catalysis of the transfer of an amino group from one position to another within a single molecule." [GOC:mah]
GO:0016870
name: intramolecular transferase activity, transferring other groups
def: "OBSOLETE. A grouping term for intramolecular transferases that cannot be more accurately categorized." [GOC:ai]
GO:0016871
name: cycloartenol synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = cycloartenol." [EC:5.4.99.8, RHEA:21311]
GO:0016872
name: intramolecular lyase activity
def: "The catalysis of certain rearrangements of a molecule to break or form a ring." [GOC:jl, http://www.mercksource.com/]
GO:0016873
name: other isomerase activity
def: "OBSOLETE. A grouping term for isomerases that cannot be more accurately categorized." [GOC:ai]
GO:0016874
name: ligase activity
def: "Catalysis of the joining of two substances, or two groups within a single molecule, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6, GOC:mah]
GO:0016875
name: ligase activity, forming carbon-oxygen bonds
def: "Catalysis of the joining of two molecules via a carbon-oxygen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.1, GOC:mah]
GO:0016876
name: ligase activity, forming aminoacyl-tRNA and related compounds
def: "Catalysis of the joining of an amino acid and a nucleic acid (usually tRNA) or poly(ribitol phosphate), with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate. The reaction forms an aminoacyl-tRNA or a related compound." [EC:6.1.1, GOC:mah]
GO:0016877
name: ligase activity, forming carbon-sulfur bonds
def: "Catalysis of the joining of two molecules via a carbon-sulfur bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.2, GOC:mah]
GO:0016878
name: acid-thiol ligase activity
def: "Catalysis of the joining of an acid and a thiol via a carbon-sulfur bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.2.1, GOC:mah]
GO:0016879
name: ligase activity, forming carbon-nitrogen bonds
def: "Catalysis of the joining of two molecules, or two groups within a single molecule, via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.3, GOC:mah]
GO:0016880
name: acid-ammonia (or amide) ligase activity
def: "Catalysis of the ligation of an acid to ammonia (NH3) or an amide via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
GO:0016881
name: acid-amino acid ligase activity
def: "Catalysis of the ligation of an acid to an amino acid via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [GOC:jl, GOC:mah]
GO:0016882
name: cyclo-ligase activity
def: "Catalysis of the joining of two groups within a single molecule via a carbon-nitrogen bond, forming heterocyclic ring, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.3.3.1, EC:6.3.3.2, EC:6.3.3.4, GOC:jl, GOC:mah]
GO:0016883
name: other carbon-nitrogen ligase activity
def: "OBSOLETE. A grouping term for carbon-nitrogen ligases that cannot be more accurately categorized." [GOC:ai]
GO:0016884
name: carbon-nitrogen ligase activity, with glutamine as amido-N-donor
def: "Catalysis of the transfer of the amide nitrogen of glutamine to a variety of substrates. GATases catalyze two separate reactions at two active sites, which are located either on a single polypeptide chain or on different subunits. In the glutaminase reaction, glutamine is hydrolyzed to glutamate and ammonia, which is added to an acceptor substrate in the synthase reaction." [PMID:12360532]
GO:0016885
name: ligase activity, forming carbon-carbon bonds
def: "Catalysis of the joining of two molecules via a carbon-carbon bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.4, GOC:jl, GOC:mah]
GO:0016886
name: ligase activity, forming phosphoric ester bonds
def: "Catalysis of the joining of two molecules, or two groups within a single molecule, via a phosphoric ester bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.5, GOC:mah]
GO:0016887
name: ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate + 2 H+. May or may not be coupled to another reaction." [EC:3.6.1.3, GOC:jl]
GO:0016888
name: endodeoxyribonuclease activity, producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acids by creating internal breaks to yield 5'-phosphomonoesters." [GOC:ai]
GO:0016889
name: endodeoxyribonuclease activity, producing 3'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acids by creating internal breaks to yield 3'-phosphomonoesters." [GOC:ai]
GO:0016890
name: site-specific endodeoxyribonuclease activity, specific for altered base
def: "Catalysis of the hydrolysis of ester linkages at specific sites within a deoxyribonucleic acid molecule by creating internal breaks." [GOC:jl]
GO:0016891
name: endoribonuclease activity, producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within ribonucleic acids by creating internal breaks to yield 5'-phosphomonoesters." [GOC:ai]
GO:0016892
name: endoribonuclease activity, producing 3'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within ribonucleic acids by creating internal breaks to yield 3'-phosphomonoesters." [GOC:ai]
GO:0016893
name: endonuclease activity, active with either ribo- or deoxyribonucleic acids and producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by creating internal breaks to yield 5'-phosphomonoesters." [GOC:mah]
GO:0016894
name: endonuclease activity, active with either ribo- or deoxyribonucleic acids and producing 3'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within nucleic acids by creating internal breaks to yield 3'-phosphomonoesters." [GOC:mah]
GO:0016895
name: exodeoxyribonuclease activity, producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within deoxyribonucleic acids by removing nucleotide residues from the 3' or 5' end to yield 5' phosphomonoesters." [GOC:ai]
GO:0016896
name: exoribonuclease activity, producing 5'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within ribonucleic acids by removing nucleotide residues from the 3' or 5' end to yield 5' phosphomonoesters." [GOC:ai]
GO:0016897
name: exoribonuclease activity, producing 3'-phosphomonoesters
def: "Catalysis of the hydrolysis of ester linkages within ribonucleic acids by removing nucleotide residues from the 3' or 5' end to yield 3' phosphomonoesters." [GOC:ai]
GO:0016898
name: oxidoreductase activity, acting on the CH-OH group of donors, cytochrome as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces a cytochrome molecule." [GOC:ai]
GO:0016899
name: oxidoreductase activity, acting on the CH-OH group of donors, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces an oxygen molecule." [GOC:ai]
GO:0016900
name: oxidoreductase activity, acting on the CH-OH group of donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces a disulfide molecule." [GOC:ai]
GO:0016901
name: oxidoreductase activity, acting on the CH-OH group of donors, quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces a quinone or a similar acceptor molecule." [GOC:ai]
GO:0016902
name: oxidoreductase activity, acting on the CH-OH group of donors, other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces an acceptor other than a cytochrome, disulfide, NAD, NADP, oxygen or a quinone or similar compound." [GOC:ai]
GO:0016903
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:ai]
GO:0016905
name: myosin heavy chain kinase activity
def: "Catalysis of the reaction: ATP + myosin-heavy-chain = ADP + myosin-heavy-chain phosphate." [EC:2.7.11.7]
GO:0016906
name: sterol 3-beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + an O-glucosylsterol." [EC:2.4.1.173]
GO:0016907
name: G-protein coupled acetylcholine receptor activity
def: "Combining with acetylcholine and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:fj, GOC:mah]
GO:0016909
name: SAP kinase activity
def: "Catalysis of the phosphorylation of proteins. A family of protein kinases that perform a crucial step in relaying signals from the plasma membrane to the nucleus. Strongly activated by stress signals such as heat or osmotic shock, DNA-damaging agents, inhibitors of protein synthesis and pro-inflammatory cytokines." [GOC:ma]
GO:0016910
name: SAP kinase 3 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016911
name: SAP kinase 4 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016912
name: SAP kinase 5 activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0016913
name: follicle-stimulating hormone activity
def: "The action characteristic of follicle-stimulating hormone (FSH), a gonadotrophic glycoprotein hormone secreted, in mammals, by the anterior pituitary gland. Upon receptor binding, FSH stimulates growth of Graafian follicles in the ovaries in females, and stimulates the epithelium of the seminiferous tubules to increase spermatogenesis." [ISBN:0198547684]
GO:0016915
name: activin
def: "OBSOLETE. A nonsteroidal regulator, composed of two covalently linked beta subunits, that is synthesized in the pituitary gland and gonads and stimulates the secretion of follicle-stimulating hormone." [ISBN:0198506732, ISBN:0721662544]
GO:0016916
name: inhibin
def: "OBSOLETE. Either of two glycoproteins (designated A and B), secreted by the gonads and present in seminal plasma and follicular fluid, that inhibit pituitary production of follicle-stimulating hormone." [ISBN:0198506732, ISBN:0721662544]
GO:0016917
name: GABA receptor activity
def: "Combining with gamma-aminobutyric acid (GABA), and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. (GABA, 4-aminobutyrate) is an amino acid which acts as a neurotransmitter in some organisms." [GOC:jl, GOC:signaling, PMID:10637650]
GO:0016918
name: retinal binding
def: "Interacting selectively and non-covalently with retinal, one of the forms of vitamin A. Retinal plays an important role in the visual process in most vertebrates, combining with opsins to form visual pigments in the retina." [CHEBI:15035, GOC:curators, ISBN:0198506732]
GO:0016919
name: nardilysin activity
def: "OBSOLETE. Catalysis of the hydrolysis of polypeptides, preferably at the first bond of Xaa-Arg-Lys, and less commonly at the first bond of Arg-Arg-Xaa, in which Xaa is not Arg or Lys." [EC:3.4.24.61]
GO:0016920
name: pyroglutamyl-peptidase activity
def: "Catalysis of the release of the N-terminal pyroglutamyl group from a peptide or protein." [EC:3.4.19.3, EC:3.4.19.6, GOC:mah]
GO:0016921
name: pyroglutamyl-peptidase II activity
def: "OBSOLETE. Catalysis of the release of the N-terminal pyroglutamyl group from pGlu-His-Xaa tripeptides and pGlu-His-Xaa-Gly tetrapeptides." [EC:3.4.19.6]
GO:0016922
name: ligand-dependent nuclear receptor binding
def: "Interacting selectively and non-covalently, in a ligand dependent manner, with a nuclear receptor protein." [GOC:mah, PMID:7776974]
GO:0016923
name: ligand-dependent thyroid hormone receptor interactor activity
def: "OBSOLETE. Ligand dependent interaction with the thyroid hormone receptor." [PMID:7776974]
GO:0016929
name: SUMO-specific protease activity
def: "Catalysis of the hydrolysis of SUMO, a small ubiquitin-related modifier, from previously sumoylated substrates." [GOC:rn, PMID:10094048, PMID:11031248, PMID:11265250]
GO:0016933
name: extracellular-glycine-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when extracellular glycine has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0016934
name: extracellular-glycine-gated chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when extracellular glycine has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0016936
name: galactoside binding
def: "Interacting selectively and non-covalently with any glycoside in which the sugar group is galactose." [CHEBI:24163, GOC:jl, ISBN:0198506732]
GO:0016937
name: short-branched-chain-acyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor, where the acyl group is a short branched chain fatty acid residue." [GOC:mah]
GO:0016941
name: natriuretic peptide receptor activity
def: "Combining with a natriuretic peptide and transmitting the signal to initiate a change in cell activity." [GOC:mah, GOC:signaling]
GO:0016943
name: RNA polymerase I transcription elongation factor activity
def: "OBSOLETE. Any activity that modulates the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule catalyzed by RNA polymerase I following transcription initiation." [GOC:mah]
GO:0016944
name: RNA polymerase II transcription elongation factor activity
def: "OBSOLETE. Any activity that modulates the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule catalyzed by RNA polymerase II following transcription initiation." [GOC:mah]
GO:0016945
name: RNA polymerase III transcription elongation factor activity
def: "OBSOLETE. Any activity that modulates the rate of transcription elongation, the addition of ribonucleotides to an RNA molecule catalyzed by RNA polymerase III following transcription initiation." [GOC:mah]
GO:0016946
name: cathepsin F activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds. Cleavage of substrates with Phe and Leu in P2." [EC:3.4.22.41]
GO:0016962
name: receptor-associated protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0016963
name: alpha-2 macroglobulin receptor-associated protein activity
def: "OBSOLETE. Interaction with the alpha-2 macroglobulin receptor and glycoprotein gp330 forming a complex with the alpha-2 macroglobulin receptor light and heavy chains." [GOC:curators]
GO:0016964
name: alpha-2 macroglobulin receptor activity
def: "Combining with alpha-2 macroglobulin and delivering alpha-2 macroglobulin into the cell via receptor-mediated endocytosis." [GOC:bf, GOC:ma, PMID:6188403]
GO:0016966
name: nitric oxide reductase activity
def: "Catalysis of the reaction: H(2)O + 2 ferricytochrome c + nitrous oxide = 2 H(+) + 2 ferrocytochrome c + 2 nitric oxide." [EC:1.7.2.5]
GO:0016969
name: hemerythrin
def: "OBSOLETE. An oxygen carrier found in a few groups of invertebrates, e.g. sipunculid worms, certain molluscs, and crustaceans." [GOC:ai, ISBN:0198506732]
GO:0016970
name: hemocyanin
def: "OBSOLETE. A blue, copper-containing oxygen carrier present in many molluscs and arthropods." [GOC:ai, ISBN:0198506732]
GO:0016971
name: flavin-linked sulfhydryl oxidase activity
def: "Catalysis of the formation of disulfide bridges." [PMID:10899311]
GO:0016972
name: thiol oxidase activity
def: "Catalysis of the reaction: 4 R'C(R)SH + O2 = 2 R'C(R)S-S(R)CR' + 2 H2O." [EC:1.8.3.2]
GO:0016975
name: alpha-2 macroglobulin
def: "OBSOLETE. Inhibition of proteinase by a mechanism involving a bait region which contains specific sites, cleavage of which induces a conformational change that results in trapping of the proteinase; following cleavage in the bait region a thiolester bond is hydrolyzed and mediates the covalent binding of the protein to the proteinase; subsequently epsilon-amino groups of the proteinase react with thiolester linkages in the inhibitor to form stable amide links; the entrapped proteinase can now only act on low molecular mass substrates." [ISBN:0198547684]
GO:0016977
name: chitosanase activity
def: "Catalysis of the endohydrolysis of beta-1,4-linkages between N-acetyl-D-glucosamine and D-glucosamine residues in a partly acetylated chitosan." [EC:3.2.1.132]
GO:0016979
name: lipoate-protein ligase activity
def: "Catalysis of the formation of carbon-nitrogen bonds between lipoate (5-(1,2-dithiolan-3-yl)pentanoate) and a protein." [EC:6.3.4.-]
GO:0016980
name: creatinase activity
def: "Catalysis of the reaction: creatine + H(2)O = sarcosine + urea." [EC:3.5.3.3, RHEA:22459]
GO:0016984
name: ribulose-bisphosphate carboxylase activity
def: "Catalysis of the reaction: D-ribulose 1,5-bisphosphate + CO2 + H2O = 2 3-phospho-D-glycerate." [EC:4.1.1.39]
GO:0016985
name: mannan endo-1,4-beta-mannosidase activity
def: "Catalysis of the random hydrolysis of (1->4)-beta-D-mannosidic linkages in mannans, galactomannans, glucomannans, and galactoglucomannans." [EC:3.2.1.78]
GO:0016986
name: transcription initiation factor activity
def: "OBSOLETE. Plays a role in regulating transcription initiation." [GOC:curators]
GO:0016987
name: sigma factor activity
def: "A sigma factor is the promoter specificity subunit of eubacterial-type multisubunit RNA polymerases, those whose core subunit composition is often described as alpha(2)-beta-beta-prime. (This type of multisubunit RNA polymerase complex is known to be found in eubacteria and plant plastids). Although sigma does not bind DNA on its own, when combined with the core to form the holoenzyme, this binds specifically to promoter sequences, with the sigma factor making sequence specific contacts with the promoter elements. The sigma subunit is released from the elongating form of the polymerase and is thus free to act catalytically for multiple RNA polymerase core enzymes." [GOC:krc, GOC:txnOH]
GO:0016988
name: transcription initiation factor antagonist activity
def: "OBSOLETE. The function of binding to a transcription factor and stopping, preventing or reducing the rate of its transcriptional activity." [GOC:jl]
GO:0016989
name: sigma factor antagonist activity
def: "The function of binding to a sigma factor and stopping, preventing or reducing the rate of its transcriptional activity." [GOC:jl, GOC:txnOH, Wikipedia:Anti-sigma_factors]
GO:0016990
name: arginine deiminase activity
def: "Catalysis of the reaction: L-arginine + H2O = L-citrulline + NH3." [EC:3.5.3.6]
GO:0016992
name: lipoate synthase activity
def: "Catalysis of the reaction: protein N6-(octanoyl)lysine + 2 sulfur + 2 S-adenosyl-L-methionine = protein N6-(lipoyl)lysine + 2 L-methionine + 2 5'-deoxyadenosyl." [EC:2.8.1.8, PMID:18307109]
GO:0016993
name: precorrin-8X methylmutase activity
def: "Catalysis of the reaction: precorrin-8X = hydrogenobyrinate." [EC:5.4.1.2]
GO:0016994
name: precorrin-6A reductase activity
def: "Catalysis of the reaction: precorrin-6B + NADP+ = precorrin-6A + NADPH + H+." [EC:1.3.1.54]
GO:0016995
name: cholesterol oxidase activity
def: "Catalysis of the reaction: cholesterol + O2 = cholest-5-en-3-one + H2O2." [EC:1.1.3.6, RHEA:21331]
GO:0016996
name: endo-alpha-(2,8)-sialidase activity
def: "Catalysis of the endohydrolysis of (2->8)-alpha-sialosyl linkages in oligo- or poly(sialic) acids." [EC:3.2.1.129]
GO:0016997
name: alpha-sialidase activity
def: "Catalysis of the hydrolysis of alpha-glycosidic linkages in oligo- or poly(sialic) acids." [GOC:mah]
GO:0017002
name: activin-activated receptor activity
def: "Combining with activin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. Activin is one of two gonadal glycoproteins related to transforming growth factor beta." [GOC:mah, GOC:signaling, ISBN:0198506732]
GO:0017005
name: 3'-tyrosyl-DNA phosphodiesterase activity
def: "Catalysis of the hydrolysis of 3'-phosphotyrosyl groups formed as covalent intermediates (in DNA backbone breakage) between DNA topoisomerase I and DNA." [PMID:10521354, PMID:16751265]
GO:0017016
name: Ras GTPase binding
def: "Interacting selectively and non-covalently with any member of the Ras superfamily of monomeric GTPases." [GOC:mah]
GO:0017017
name: MAP kinase tyrosine/serine/threonine phosphatase activity
def: "Catalysis of the reaction: MAP kinase serine/threonine/tyrosine phosphate + H2O = MAP kinase serine/threonine/tyrosine + phosphate." [GOC:mah, PMID:12184814]
GO:0017018
name: myosin phosphatase activity
def: "Catalysis of the reaction: phosphomyosin + H2O = myosin + phosphate." [EC:3.1.3.16]
GO:0017020
name: myosin phosphatase regulator activity
def: "Modulation of the activity of the enzyme myosin phosphatase." [EC:3.1.3.16, GOC:ai]
GO:0017021
name: myosin phosphatase myosin binding
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0017022
name: myosin binding
def: "Interacting selectively and non-covalently with any part of a myosin complex; myosins are any of a superfamily of molecular motor proteins that bind to actin and use the energy of ATP hydrolysis to generate force and movement along actin filaments." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0017024
name: myosin I binding
def: "Interacting selectively and non-covalently with a class I myosin; myosin I heavy chains are single-headed, possess tails of various lengths, and do not self-associate into bipolar filaments." [GOC:bf, GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0017025
name: TBP-class protein binding
def: "Interacting selectively and non-covalently with a member of the class of TATA-binding proteins (TBP), including any of the TBP-related factors (TRFs)." [GOC:jl, GOC:txnOH, http://www.mblab.gla.ac.uk/, PMID:16858867]
GO:0017026
name: procollagen C-endopeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of the C-terminal propeptide at Ala-Asp in type I and II procollagens and at Arg-Asp in type III." [EC:3.4.24.19]
GO:0017027
name: transmembrane receptor protein serine/threonine kinase receptor-associated protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0017028
name: protein stabilization activity
def: "OBSOLETE. Strengthening of a bond between proteins. Proteins are large molecules composed of one or more chains of amino acids. The amino acids are joined in a specific order by peptide bonds." [GOC:jid, http://biotech.icmb.utexas.edu]
GO:0017029
name: lysosomal protein stabilization
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0017030
name: beta-galactosidase stabilization activity
def: "OBSOLETE. Stabilization of the structure of beta-galactosidase." [GOC:ai]
GO:0017032
name: potassium:amino acid symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: amino acid(out) + K+(out) = amino acid(in) + K+(in)." [GOC:ai]
GO:0017034
name: Rap guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Rap family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0017039
name: dipeptidyl-peptidase III activity
def: "OBSOLETE. Catalysis of the release of an N-terminal dipeptide from a peptide comprising four or more residues, with broad specificity; also acts on dipeptidyl 2-naphthylamides." [EC:3.4.14.4]
GO:0017040
name: ceramidase activity
def: "Catalysis of the reaction: N-acylsphingosine + H2O = a fatty acid + sphingosine." [EC:3.5.1.23]
GO:0017041
name: galactosylgalactosylglucosylceramidase activity
def: "Catalysis of the reaction: D-galactosyl-D-galactosyl-D-glucosyl-N-acylsphingosine + H2O = lactosyl-N-acylsphingosine + D-galactose." [EC:3.2.1.47]
GO:0017042
name: glycosylceramidase activity
def: "Catalysis of the reaction: glycosyl-N-acylsphingosine + H2O = a sugar + N-acylsphingosine." [EC:3.2.1.62]
GO:0017043
name: adrenocorticotropin
def: "OBSOLETE. A polypeptide hormone which stimulates the adrenal cortex to synthesize and secrete glucocorticoid hormones." [ISBN:0198506732]
GO:0017044
name: melanocyte-stimulating hormone activity
def: "The action characteristic of melanocyte-stimulating hormone, any of three peptide hormones that are produced by the intermediate lobe of the pituitary gland and, upon receptor binding, cause dispersal of melanosomes in melanophores of poikilothermic vertebrates." [ISBN:0198506732]
GO:0017045
name: corticotropin-releasing hormone activity
def: "The action characteristic of corticotropin-releasing hormone (CRH), any of a number of peptides released by the mammalian hypothalamus into the hypophyseal-portal circulation in response to neural and/or chemical stimuli. Upon receptor binding, CRH increases the rate of corticotropin secretion by the anterior pituitary." [ISBN:0198506732]
GO:0017046
name: peptide hormone binding
def: "Interacting selectively and non-covalently with any peptide with hormonal activity in animals." [GOC:jl, ISBN:0198506732]
GO:0017048
name: Rho GTPase binding
def: "Interacting selectively and non-covalently with Rho protein, any member of the Rho subfamily of the Ras superfamily of monomeric GTPases. Proteins in the Rho subfamily are involved in relaying signals from cell-surface receptors to the actin cytoskeleton." [ISBN:0198547684, PMID:12581856]
GO:0017049
name: GTP-Rho binding
def: "Interacting selectively and non-covalently with the GTP-bound form of the Rho protein." [GOC:mah]
GO:0017050
name: D-erythro-sphingosine kinase activity
def: "Catalysis of the reaction: sphingosine + ATP = sphingosine 1-phosphate + ADP." [MetaCyc:RXN3DJ-11417]
GO:0017051
name: retinol dehydratase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + retinol = adenosine 3',5'-bisphosphate + anhydroretinol." [PMID:9857081]
GO:0017056
name: structural constituent of nuclear pore
def: "The action of a molecule that contributes to the structural integrity of the nuclear pore complex." [GOC:mah]
GO:0017057
name: 6-phosphogluconolactonase activity
def: "Catalysis of the reaction: 6-O-phosphono-D-glucono-1,5-lactone + H(2)O = 6-phospho-D-gluconate + H(+)." [EC:3.1.1.31, RHEA:12559]
GO:0017058
name: FH1 domain binding
def: "Interacting selectively and non-covalently with a FH1 domain of a protein, a proline-rich domain, usually located in front of a FH2 domain." [GOC:go_curators]
GO:0017060
name: 3-galactosyl-N-acetylglucosaminide 4-alpha-L-fucosyltransferase activity
def: "Catalysis of the reaction: GDP-L-fucose + beta-D-galactosyl-(1,3)-N-acetyl-D-glucosaminyl-R = GDP + beta-D-galactosyl-(1,3)-[alpha-L-fucosyl-(1,4)]-N-acetyl-D-glucosaminyl-R." [EC:2.4.1.65]
GO:0017061
name: S-methyl-5-thioadenosine phosphorylase activity
def: "Catalysis of the reaction: 5'-methylthioadenosine + phosphate = adenine + 5-methylthio-D-ribose 1-phosphate." [EC:2.4.2.28]
GO:0017063
name: phosphatidylserine-specific phospholipase A1 activity
def: "OBSOLETE. Catalysis of the reaction: phosphatidylcholine + H2O = 2-acylglycerophosphocholine + a fatty acid anion." [EC:3.1.1.32]
GO:0017064
name: fatty acid amide hydrolase activity
def: "Catalysis of the hydrolysis of a fatty acid amide to yield a fatty acid." [PMID:15952893]
GO:0017065
name: single-strand selective uracil DNA N-glycosylase activity
def: "Catalysis of the cleavage of the N-C1' glycosidic bond between the damaged DNA base and the deoxyribose sugar, releasing a free base and leaving an apyrimidinic (AP) site. Enzymes with this activity recognize and remove uracil bases present in single-stranded DNA." [GOC:elh, PMID:9224623]
GO:0017067
name: tyrosine-ester sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + L-tyrosine methyl ester = L-tyrosine methyl ester 4-sulfate + adenosine 3',5'-diphosphate + H(+)." [EC:2.8.2.9, RHEA:19980]
GO:0017069
name: snRNA binding
def: "Interacting selectively and non-covalently with a small nuclear RNA (snRNA)." [GOC:mah]
GO:0017070
name: U6 snRNA binding
def: "Interacting selectively and non-covalently with the U6 small nuclear RNA (U6 snRNA)." [GOC:mah]
GO:0017072
name: tubulin-specific chaperone activity
def: "OBSOLETE. Assists in the correct, non-covalent assembly of tubulin-containing structures in vivo, but is not a component of the assembled structures when performing its normal biological function." [GOC:jl, PMID:11847227]
GO:0017074
name: procollagen N-endopeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of the N-propeptide of collagen chain alpha-1(I) at Pro-Gln and of alpha-1(II) and alpha-2(I) chains at Ala-Gln." [EC:3.4.24.14]
GO:0017075
name: syntaxin-1 binding
def: "Interacting selectively and non-covalently with the SNAP receptor syntaxin-1." [GOC:ai]
GO:0017076
name: purine nucleotide binding
def: "Interacting selectively and non-covalently with purine nucleotides, any compound consisting of a purine nucleoside esterified with (ortho)phosphate." [GOC:ai]
GO:0017077
name: oxidative phosphorylation uncoupler activity
def: "Catalysis of the transfer of protons from mitochondrial intermembrane space into mitochondrial matrix, dissipating the proton gradient across the mitochondrial inner membrane established by the electron transport chain during the oxidative phosphorylation (proton leak). Proton leak uncouples the processes of electron transport/proton generation and ATP synthesis." [PMID:15738989, PMID:16179945]
GO:0017078
name: Hsc70 protein regulator activity
def: "OBSOLETE. Binds to and modulates the activity of the molecular chaperone Hsc70." [GOC:jl, PMID:11121403]
GO:0017080
name: sodium channel regulator activity
def: "Modulates the activity of a sodium channel." [GOC:mah]
GO:0017081
name: chloride channel regulator activity
def: "Modulates the activity of a chloride channel." [GOC:mah]
GO:0017082
name: mineralocorticoid receptor activity
def: "Combining with a mineralocorticoid and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:20932876]
GO:0017083
name: 4-galactosyl-N-acetylglucosaminide 3-alpha-L-fucosyltransferase activity
def: "Catalysis of the reaction: GDP-beta-L-fucose + beta-D-galactosyl-(1,4)-N-acetyl-D-glucosaminyl-R = GDP + 1,4-beta-D-galactosyl-(1,4)-[alpha-L-fucosyl-(1,4)]-N-acetyl-D-glucosaminyl-R." [EC:2.4.1.152]
GO:0017084
name: delta1-pyrroline-5-carboxylate synthetase activity
def: "Catalysis of the reaction: L-glutamate + ATP + NADPH = ADP + L-glutamate gamma-semialdehyde + NADP+ + phosphate." [MetaCyc:PROLINE-MULTI]
GO:0017088
name: X-Pro dipeptidyl-peptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of the terminal bond of Xaa-Pro-Xaa motifs to release unblocked, N-terminal dipeptides from substrates including Ala-Pro-para-nitroanilide and (sequentially, at the second, fourth and sixth bonds) of the motif Tyr-Pro-Phe-Pro-Gly-Pro-Ile." [EC:3.4.14.11]
GO:0017089
name: glycolipid transporter activity
def: "Enables the directed movement of glycolipids, compounds containing (usually) 1-4 linked monosaccharide residues joined by a glycosyl linkage to a lipid, into, out of or within a cell, or between cells." [GOC:ai]
GO:0017091
name: AU-rich element binding
def: "Interacting selectively and non-covalently with a region of RNA containing frequent adenine and uridine bases." [GOC:bf, GOC:mah, PMID:7892223, PMID:8578590]
GO:0017092
name: sterol regulatory element-binding protein site 2 protease activity
def: "OBSOLETE. Catalysis of the cleavage, within the membrane-spanning helix, of the amino-terminal half (the intermediate form) of sterol regulatory element binding protein (SREBP). This activity releases the transcription factor domain of SREBP from the membrane, freeing it to enter the nucleus." [GOC:bf, PMID:12923525]
GO:0017093
name: sterol regulatory element-binding protein protease activity
def: "OBSOLETE. Catalysis of the hydrolysis of peptide bonds within a sterol regulatory element binding protein (SREBP). SREBPs are transcription factors that bind sterol regulatory elements (SREs), DNA motifs found in the promoters of target genes." [GOC:bf, PMID:12923525]
GO:0017094
name: sterol regulatory element-binding protein site 1 protease activity
def: "OBSOLETE. Catalysis of the cleavage of a Leu-Ser bond within the luminal loop of a sterol regulatory element binding protein (SREBP). This activity is the first of two sequential cleavage reactions and cleaves SREBP into two membrane-bound halves." [GOC:bf, PMID:12923525]
GO:0017095
name: heparan sulfate 6-O-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + heparan sulfate = adenosine 3',5'-bisphosphate + heparan sulfate 6-O-sulfate; results in 6-O-sulfation of glucosamine residues in heparan sulfate." [PMID:8631808]
GO:0017096
name: acetylserotonin O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + N-acetylserotonin = S-adenosyl-L-homocysteine + melatonin. Melatonin is also known as N-acetyl-5-methoxytryptamine." [EC:2.1.1.4]
GO:0017098
name: sulfonylurea receptor binding
def: "Interacting selectively and non-covalently with the sulfonylurea receptor, a regulatory subunit of the ATP-sensitive potassium ion channel." [GOC:ceb, PMID:11938023]
GO:0017099
name: very-long-chain-acyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor, where the acyl group is a very long chain fatty acid residue. A very long-chain fatty acid is a fatty acid which has a chain length greater than C22." [CHEBI:27283, GOC:mah]
GO:0017103
name: UTP:galactose-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: alpha-D-galactose 1-phosphate + UTP = diphosphate + UDP-D-galactose." [EC:2.7.7.10, RHEA:14212]
GO:0017105
name: acyl-CoA delta11-desaturase activity
def: "Catalysis of the reaction: acyl-CoA + reduced acceptor + O2 = delta11-acyl-CoA + acceptor + 2 H2O." [EC:1.14.19.5]
GO:0017106
name: activin inhibitor activity
def: "OBSOLETE. Acts to negatively regulate the activity of activin, a nonsteroidal regulator synthesized in the pituitary gland and gonads that stimulates the secretion of follicle-stimulating hormone." [ISBN:0198506732, ISBN:0721662544]
GO:0017107
name: anion exchanger adaptor activity
def: "The binding activity of a molecule that brings together an anion exchanger and one or more other molecules, permitting them to function in a coordinated way." [GOC:mtg_MIT_16mar07]
GO:0017108
name: 5'-flap endonuclease activity
def: "Catalysis of the cleavage of a 5' flap structure in DNA, but not other DNA structures; processes the 5' ends of Okazaki fragments in lagging strand DNA synthesis." [PMID:9778254]
GO:0017110
name: nucleoside-diphosphatase activity
def: "Catalysis of the reaction: a nucleoside diphosphate + H2O = a nucleotide + phosphate." [EC:3.6.1.6]
GO:0017111
name: nucleoside-triphosphatase activity
def: "Catalysis of the reaction: a nucleoside triphosphate + H2O = nucleoside diphosphate + phosphate." [EC:3.6.1.15]
GO:0017112
name: Rab guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Rab family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0017113
name: dihydropyrimidine dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: 5,6-dihydrouracil + NADP+ = uracil + NADPH + H+." [EC:1.3.1.2]
GO:0017114
name: wide-spectrum protease inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of a protease by a wide-spectrum protease inhibitor. A wide-spectrum protease inhibitor is a protein having a peptide stretch which contains specific cleavage sites for different proteinases enabling inhibition of all four classes of proteinases by formation of a covalent bond between the inhibitor and the protease." [GOC:dph, GOC:tb]
GO:0017116
name: single-stranded DNA-dependent ATP-dependent DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, in the presence of single-stranded DNA; drives the unwinding of a DNA helix." [EC:3.6.1.3, GOC:jl]
GO:0017118
name: lipoyltransferase activity
def: "Catalysis of the transfer of the lipoyl group from lipoyl-AMP to the lysine residue of lipoate-dependent enzyme." [PMID:10103005]
GO:0017120
name: polyphosphatidylinositol phosphatase activity
def: "OBSOLETE. Catalysis of the conversion of phosphatidylinositol-3-phosphate, phosphatidylinositol-4-phosphate, and phosphatidylinositol-3,5-bisphosphate, but not PI-4,5-bisphosphate, to phosphatidylinositol; PPIPase activity is a key regulator of membrane trafficking and actin cytoskeleton organization." [PMID:10224048]
GO:0017123
name: Ral GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Ral family." [GOC:mah]
GO:0017124
name: SH3 domain binding
def: "Interacting selectively and non-covalently with a SH3 domain (Src homology 3) of a protein, small protein modules containing approximately 50 amino acid residues found in a great variety of intracellular or membrane-associated proteins." [GOC:go_curators, Pfam:PF00018]
GO:0017125
name: deoxycytidyl transferase activity
def: "Catalysis of the insertion of a dCMP residue opposite a template abasic site in DNA." [PMID:10535901]
GO:0017127
name: cholesterol transporter activity
def: "Enables the directed movement of cholesterol into, out of or within a cell, or between cells. Cholesterol is the principal sterol of vertebrates and the precursor of many steroids, including bile acids and steroid hormones." [GOC:ai]
GO:0017128
name: phospholipid scramblase activity
def: "Catalysis of the movement of phospholipids from one membrane bilayer leaflet to the other, by an ATP-independent mechanism." [GOC:cjm, PMID:20043909, PMID:20302864]
GO:0017129
name: triglyceride binding
def: "Interacting selectively and non-covalently with any triester of glycerol." [GOC:jl, ISBN:0198506732]
GO:0017130
name: poly(C) RNA binding
def: "Interacting selectively and non-covalently with a sequence of cytosine residues in an RNA molecule." [GOC:mah]
GO:0017131
name: uridine-rich cytoplasmic polyadenylylation element binding
def: "Interacting selectively and non-covalently with U-rich sequence in the 3'-end of nuclear-transcribed mRNAs; required for cytoplasmic polyadenylylation." [GOC:krc, PMID:7954828]
GO:0017132
name: cyclic nucleotide-dependent guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase; dependent on the presence of a cyclic nucleotide." [GOC:mah, PMID:9856955]
GO:0017134
name: fibroblast growth factor binding
def: "Interacting selectively and non-covalently with a fibroblast growth factor." [PMID:9806903]
GO:0017135
name: membrane-associated protein with guanylate kinase activity interacting
def: "OBSOLETE. The interaction with synapse-associated membrane-associated protein with guanylate kinase activity." [PMID:10207005]
GO:0017136
name: NAD-dependent histone deacetylase activity
def: "Catalysis of the reaction: histone N6-acetyl-L-lysine + H2O = histone L-lysine + acetate. This reaction requires the presence of NAD, and represents the removal of an acetyl group from a histone." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0017137
name: Rab GTPase binding
def: "Interacting selectively and non-covalently with Rab protein, any member of the Rab subfamily of the Ras superfamily of monomeric GTPases." [GOC:mah]
GO:0017147
name: Wnt-protein binding
def: "Interacting selectively and non-covalently with Wnt-protein, a secreted growth factor involved in signaling." [GOC:jl]
GO:0017149
name: protein biosynthetic process inhibitor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0017150
name: tRNA dihydrouridine synthase activity
def: "Catalysis of the reaction: tRNA-uracil + acceptor = tRNA-dihydrouridine + reduced acceptor." [PMID:11983710]
GO:0017151
name: DEAD/H-box RNA helicase binding
def: "Interacting selectively and non-covalently with the enzyme DEAD/H-box RNA helicase." [GOC:jl]
GO:0017153
name: sodium:dicarboxylate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: dicarboxylate(out) + Na+(out) = dicarboxylate(in) + Na+(in)." [GOC:ai]
GO:0017154
name: semaphorin receptor activity
def: "Combining with a semaphorin, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:mah, GOC:signaling, PMID:15239958]
GO:0017155
name: sodium:hydrogen antiporter regulator activity
def: "OBSOLETE." [GOC:mtg_transport, ISBN:0815340729]
GO:0017159
name: pantetheine hydrolase activity
def: "Catalysis of the reaction: (R)-pantetheine + H(2)O = (R)-pantothenate + cysteamine." [EC:3.5.1.92, RHEA:13448]
GO:0017160
name: Ral GTPase binding
def: "Interacting selectively and non-covalently with Ral protein, any member of the Ral subfamily of the Ras superfamily of monomeric GTPases." [GOC:mah]
GO:0017161
name: inositol-1,3,4-trisphosphate 4-phosphatase activity
def: "Catalysis of the reaction: D-myo-inositol 1,3,4-trisphosphate + H2O = myo-inositol 1,3-bisphosphate + phosphate." [GOC:ai]
GO:0017162
name: aryl hydrocarbon receptor binding
def: "Interacting selectively and non-covalently with an aryl hydrocarbon receptor." [GOC:ai]
GO:0017163
name: basal transcription repressor activity
def: "OBSOLETE. Any transcription regulator activity that prevents or downregulates basal transcription. Basal transcription results from transcription that is controlled by the minimal complement of proteins necessary to reconstitute transcription from a minimal promoter." [GOC:dph, GOC:tb, http://tfib.med.harvard.edu/transcription/basaltx.html]
GO:0017164
name: nicotinic acetylcholine receptor-associated protein activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0017165
name: dipeptidase E activity
def: "OBSOLETE. Catalysis of the hydrolysis of dipeptides Asp-Xaa; does not act on peptides with N-terminal Glu, Asn or Gln, nor does it cleave isoaspartyl peptides." [EC:3.4.13.21]
GO:0017166
name: vinculin binding
def: "Interacting selectively and non-covalently with vinculin, a protein found in muscle, fibroblasts, and epithelial cells that binds actin and appears to mediate attachment of actin filaments to integral proteins of the plasma membrane." [ISBN:0721662544]
GO:0017168
name: 5-oxoprolinase (ATP-hydrolyzing) activity
def: "Catalysis of the reaction: 5-oxo-L-proline + ATP + 2 H(2)O = L-glutamate + ADP + 2 H(+) + phosphate." [EC:3.5.2.9, RHEA:10351]
GO:0017169
name: CDP-alcohol phosphatidyltransferase activity
def: "Catalysis of the reaction: CDP + alcohol = CMP + phosphatidyl alcohol." [GOC:ai]
GO:0017170
name: KU70 binding
def: "OBSOLETE. Interacting selectively and non-covalently with Ku70, a protein involved in non-homologous DNA end joining." [GOC:mah, PMID:14739985]
GO:0017171
name: serine hydrolase activity
def: "Catalysis of the hydrolysis of a substrate by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [Wikipedia:Serine_hydrolase]
GO:0017172
name: cysteine dioxygenase activity
def: "Catalysis of the reaction: L-cysteine + O(2) = 3-sulfino-L-alanine + H(+)." [EC:1.13.11.20, RHEA:20444]
GO:0017174
name: glycine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + glycine = S-adenosyl-L-homocysteine + sarcosine." [EC:2.1.1.20]
GO:0017175
name: IMP-GMP specific 5'-nucleotidase activity
def: "OBSOLETE. Catalysis of the conversion of 5'-ribonucleotides to ribonucleosides and phosphate, with specificity for IMP or GMP 5'-ribonucleotides and H2O as a nucleophile." [EC:3.1.3.5, GOC:krc]
GO:0017176
name: phosphatidylinositol N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + phosphatidylinositol = UDP + N-acetyl-D-glucosaminylphosphatidylinositol." [EC:2.4.1.198]
GO:0017178
name: diphthine-ammonia ligase activity
def: "Catalysis of the reaction: ATP + diphthine + NH(4)(+) = ADP + diphthamide + H(+) + phosphate." [EC:6.3.1.14, RHEA:19756]
GO:0017188
name: aspartate N-acetyltransferase activity
def: "Catalysis of the reaction: L-aspartate + acetyl-CoA = N-acetyl-L-aspartate + CoA + H(+)." [EC:2.3.1.17, RHEA:14168]
GO:0018010
name: glycoprotein N-palmitoyltransferase activity
def: "Catalysis of the reaction: palmitoyl-CoA + glycoprotein = CoA + N-palmitoylglycoprotein." [EC:2.3.1.96]
GO:0018024
name: histone-lysine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone L-lysine = S-adenosyl-L-homocysteine + histone N6-methyl-L-lysine. The methylation of peptidyl-lysine in histones forms N6-methyl-L-lysine, N6,N6-dimethyl-L-lysine and N6,N6,N6-trimethyl-L-lysine derivatives." [EC:2.1.1.43, RESID:AA0074, RESID:AA0075, RESID:AA0076]
GO:0018025
name: calmodulin-lysine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + calmodulin L-lysine = S-adenosyl-L-homocysteine + calmodulin N6-methyl-L-lysine." [EC:2.1.1.60]
GO:0018030
name: peptidyl-lysine N6-myristoyltransferase activity
def: "Catalysis of the transfer of a myristoyl group to the N6 nitrogen atom on a lysine residue of a peptide or protein molecule." [GOC:mah]
GO:0018031
name: peptidyl-lysine N6-palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl group to the N6 nitrogen atom on a lysine residue of a peptide or protein molecule." [GOC:mah]
GO:0018064
name: protein-histidine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein L-histidine = S-adenosyl-L-homocysteine + protein N-methyl-L-histidine." [EC:2.1.1.85]
GO:0018071
name: NAD(P)-cysteine ADP-ribosyltransferase activity
def: "Catalysis of the reaction: NAD(P)+ + L-cysteine = nicotinamide + N2-(ADP-D-ribosyl)-L-cysteine." [EC:2.4.2.-]
GO:0018110
name: histone arginine kinase activity
def: "Catalysis of the reaction: histone L-arginine + ATP = histone N(omega)-phospho-L-arginine + ADP + 2 H(+)." [GOC:mah]
GO:0018111
name: methionine racemase activity
def: "Catalysis of the reaction: L-methionine = D-methionine." [EC:5.1.1.2, RHEA:12495]
GO:0018112
name: proline racemase activity
def: "Catalysis of the reaction: L-proline = D-proline." [EC:5.1.1.4, RHEA:10683]
GO:0018113
name: lysine racemase activity
def: "Catalysis of the reaction: L-lysine = D-lysine." [EC:5.1.1.5]
GO:0018114
name: threonine racemase activity
def: "Catalysis of the reaction: L-threonine = D-threonine." [EC:5.1.1.6, RHEA:13916]
GO:0018121
name: NAD(P)-asparagine ADP-ribosyltransferase activity
def: "Catalysis of the reaction: NAD(P)+ + L-asparagine = nicotinamide + N2-(ADP-D-ribosyl)-L-asparagine." [EC:2.4.2.-]
GO:0018127
name: NAD(P)-serine ADP-ribosyltransferase activity
def: "Catalysis of the reaction: NAD(P)+ + L-serine = nicotinamide + N2-(ADP-D-ribosyl)-L-serine." [EC:2.4.2.-]
GO:0018128
name: peptidyl-serine cyclase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0018129
name: peptidyl-oxazoline dehydrogenase activity
def: "Catalysis of the reduction of a peptide-linked oxazoline to oxazole." [GOC:mah, PMID:19058272]
GO:0018135
name: peptidyl-cysteine cyclase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0018136
name: peptidyl-thiazoline dehydrogenase activity
def: "Catalysis of the reduction of a peptide-linked thiazoline to thiazole." [GOC:mah, PMID:19058272]
GO:0018169
name: ribosomal S6-glutamic acid ligase activity
def: "Catalysis of the posttranslational transfer of one or more glutamic acid residues to the C-terminus of ribosomal protein S6." [GOC:mah, PMID:2570347]
GO:0018237
name: urease activator activity
def: "Increases the activity of urease by promoting the incorporation of nickel into the active site." [GOC:mah, PMID:16244137]
GO:0018271
name: biotin-protein ligase activity
def: "Catalysis of the reaction: ATP + biotin + protein = AMP + diphosphate + biotin-protein." [GOC:mah]
GO:0018365
name: protein-serine epimerase activity
def: "Catalysis of the reaction: (protein)-L-serine = (protein)-D-serine." [EC:5.1.1.16]
GO:0018392
name: glycoprotein 3-alpha-L-fucosyltransferase activity
def: "Catalysis of the reaction: N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-N-acetyl-beta-D-glucosaminyl}-L-asparagine + GDP-L-fucose = N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-[alpha-L-fucosyl-(1->3)]-N-acetyl-beta-D-glucosaminyl}-L-asparagine + GDP + H(+)." [EC:2.4.1.214, RHEA:24447]
GO:0018421
name: UDP-N-acetylglucosamine:serine-protein N-acetylglucosamine-1-phosphotransferase activity
def: "Catalysis of the transfer of N-acetylglucosamine-1-phosphate to a serine residue in a protein." [GOC:mah, PMID:9353330]
GO:0018422
name: GDP-mannose:serine-protein mannose-1-phosphotransferase activity
def: "Catalysis of the transfer of mannose-1-phosphate to a serine residue in a protein." [GOC:mah, PMID:10037765]
GO:0018423
name: protein C-terminal leucine carboxyl O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein L-leucine = S-adenosyl-L-homocysteine + protein L-leucine methyl ester. This modification occurs only at the oxygen atoms of the free alpha carboxyl group of a leucine residue at the C-terminus of the protein." [PMID:8514774]
GO:0018445
name: prothoracicotrophic hormone activity
def: "The action characteristic of prothoracicotrophic hormone, a peptide hormone that is secreted by the brain and, upon receptor binding, acts on the prothoracic gland to stimulate the release of ecdysone in insects." [GOC:mah, PMID:3301403]
GO:0018446
name: pinocarveol dehydrogenase activity
def: "Catalysis of the reaction: pinocarveol = pinocarvone + 2 H+ + 2 e-." [UM-BBD_reactionID:r0717]
GO:0018447
name: chloral hydrate dehydrogenase activity
def: "Catalysis of the reactions: chloral hydrate = 3 H+ + 2 e- + trichloroacetate, and chloral hydrate + H2 = H2O + trichloroethanol." [UM-BBD_enzymeID:e0229]
GO:0018448
name: hydroxymethylmethylsilanediol oxidase activity
def: "Catalysis of the reaction: hydroxymethylmethylsilanediol + O2 + 2 H+ + 2 e- = formylmethylsilanediol + 2 H2O." [UM-BBD_reactionID:r0638]
GO:0018449
name: 1-phenylethanol dehydrogenase activity
def: "Catalysis of the reaction: (S)-1-phenylethanol = acetophenone + H2." [UM-BBD_reactionID:r0032]
GO:0018450
name: myrtenol dehydrogenase activity
def: "Catalysis of the reaction: myrtenol + O2 + 2 H+ + 2 e- = 2 H2O + myrtenal." [UM-BBD_reactionID:r0710]
GO:0018451
name: epoxide dehydrogenase activity
def: "Catalysis of the reaction: ethene oxide + NAD+ + CoA-SH = NADH + H+ + acetyl-CoA." [UM-BBD_reactionID:r0595]
GO:0018452
name: 5-exo-hydroxycamphor dehydrogenase activity
def: "Catalysis of the reaction: 5-exo-hydroxycamphor + NAD+ = NADH + H+ + 2,5-diketocamphane." [UM-BBD_reactionID:r0427]
GO:0018453
name: 2-hydroxytetrahydrofuran dehydrogenase activity
def: "Catalysis of the reaction: 2-hydroxytetrahydrofuran = 2 H+ + 2 e- + butyrolactone." [UM-BBD_reactionID:r0018]
GO:0018454
name: acetoacetyl-CoA reductase activity
def: "Catalysis of the reaction: (R)-3-hydroxyacyl-CoA + NADP+ = 3-oxoacyl-CoA + NADPH + H+." [EC:1.1.1.36]
GO:0018455
name: alcohol dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: an alcohol + NAD(P)+ = an aldehyde + NAD(P)H + H+." [EC:1.1.1.71]
GO:0018456
name: aryl-alcohol dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: an aromatic alcohol + NAD+ = an aromatic aldehyde + NADH + H+." [EC:1.1.1.90]
GO:0018457
name: perillyl-alcohol dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + perillyl alcohol = H(+) + NADH + perillyl aldehyde." [EC:1.1.1.144, RHEA:10667]
GO:0018458
name: isopiperitenol dehydrogenase activity
def: "Catalysis of the reaction: (1S,6R)-isopiperitenol + NAD(+) = (6R)-isoperitenone + H(+) + NADH." [EC:1.1.1.223, RHEA:20863]
GO:0018459
name: carveol dehydrogenase activity
def: "Catalysis of the reaction: (1S,5R)-carveol + NADP(+) = (R)-carvone + H(+) + NADPH." [EC:1.1.1.243, RHEA:13632]
GO:0018460
name: cyclohexanol dehydrogenase activity
def: "Catalysis of the reaction: cyclohexanol + NAD+ = cyclohexanone + NADH + H+." [EC:1.1.1.245]
GO:0018461
name: fluoren-9-ol dehydrogenase activity
def: "Catalysis of the reaction: fluoren-9-ol + 2 NADP+ = fluoren-9-one + 2 NADPH + 2 H+." [EC:1.1.1.256]
GO:0018462
name: 4-(hydroxymethyl)benzenesulfonate dehydrogenase activity
def: "Catalysis of the reaction: 4-(hydroxymethyl)benzenesulfonate + NAD(+) = 4-formylbenzenesulfonate + H(+) + NADH." [EC:1.1.1.257, RHEA:24415]
GO:0018463
name: 6-hydroxyhexanoate dehydrogenase activity
def: "Catalysis of the reaction: 6-hydroxyhexanoate + NAD(+) = 6-oxohexanoate + H(+) + NADH." [EC:1.1.1.258, RHEA:14228]
GO:0018464
name: 3-hydroxypimeloyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxypimelyl-CoA + NAD(+) = 3-oxopimelyl-CoA + H(+) + NADH." [EC:1.1.1.259, RHEA:11171]
GO:0018465
name: vanillyl-alcohol oxidase activity
def: "Catalysis of the reaction: O(2) + vanillyl alcohol = H(2)O(2) + vanillin." [EC:1.1.3.38, RHEA:10039]
GO:0018466
name: limonene-1,2-diol dehydrogenase activity
def: "Catalysis of the reaction: (1S,2S,4R)-limonene-1,2-diol + DCPIP+ = DCPIPH + H+ + (1S,4R)-1-hydroxy-2-oxolimonene." [UM-BBD_reactionID:r0735]
GO:0018467
name: formaldehyde dehydrogenase activity
def: "Catalysis of the reaction: formaldehyde + H(2)O + NAD(+) = formate + 2 H(+) + NADH." [EC:1.2.1.46, RHEA:16428]
GO:0018468
name: alcohol dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: a primary alcohol + acceptor = an aldehyde + reduced acceptor." [EC:1.1.99.8]
GO:0018469
name: myrtenal dehydrogenase activity
def: "Catalysis of the reaction: myrtenal + H2O = 2 H+ + 2 e- + myrtenic acid." [UM-BBD_reactionID:r0711]
GO:0018470
name: 4-hydroxybutaraldehyde dehydrogenase activity
def: "Catalysis of the reaction: 4-hydroxybutyraldehyde + H2O = 2 H+ + 2 e- + 4-hydroxybutanoate." [UM-BBD_reactionID:r0014]
GO:0018471
name: 4-chlorobenzaldehyde oxidase activity
def: "Catalysis of the reaction: 4-chlorobenzaldehyde + 2 H2O = 4-chlorobenzoate + 2 H+ + 2 e-." [UM-BBD_reactionID:r0447]
GO:0018472
name: 1-hydroxy-2-naphthaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 1-hydroxy-2-naphthaldehyde + NAD+ + H2O = NADH + H+ + 1-hydroxy-2-naphthoate." [UM-BBD_reactionID:r0485]
GO:0018473
name: cis-2-methyl-5-isopropylhexa-2,5-dienal dehydrogenase activity
def: "Catalysis of the reaction: cis-2-methyl-5-isopropylhexa-2,5-dienal + NAD+ + H2O = NADH + H+ + cis-2-methyl-5-isopropylhexa-2,5-dienoic acid." [UM-BBD_reactionID:r0744]
GO:0018474
name: 2-carboxybenzaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 2-carboxybenzaldehyde + NAD+ + H2O = NADH + 2 H+ + phthalate." [UM-BBD_reactionID:r0490]
GO:0018475
name: trans-2-methyl-5-isopropylhexa-2,5-dienal dehydrogenase activity
def: "Catalysis of the reaction: trans-2-methyl-5-isopropylhexa-2,5-dienal + NAD+ + H2O = NADH + H+ + trans-2-methyl-5-isopropylhexa-2,5-dienoic acid." [UM-BBD_reactionID:r0745]
GO:0018477
name: benzaldehyde dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: benzaldehyde + NADP+ + H2O = benzoate + NADPH + H+." [EC:1.2.1.7]
GO:0018478
name: malonate-semialdehyde dehydrogenase (acetylating) activity
def: "Catalysis of the reaction: 3-oxopropanoate + CoA + NADP+ = acetyl-CoA + CO2 + NADPH + H+." [EC:1.2.1.18]
GO:0018479
name: benzaldehyde dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: benzaldehyde + NAD+ + H2O = benzoate + NADH + H+." [EC:1.2.1.28]
GO:0018480
name: 5-carboxymethyl-2-hydroxymuconic-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: 5-carboxymethyl-2-hydroxymuconate semialdehyde + H2O + NAD+ = 5-carboxymethyl-2-hydroxymuconate + NADH + H+." [EC:1.2.1.60]
GO:0018481
name: 4-hydroxymuconic-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: <stereo>cis,trans</stereo>-4-hydroxymuconate semialdehyde + H(2)O + NAD(+) = 2 H(+) + maleylacetate + NADH." [EC:1.2.1.61, RHEA:22423]
GO:0018482
name: 4-formylbenzenesulfonate dehydrogenase activity
def: "Catalysis of the reaction: 4-formylbenzenesulfonate + H(2)O + NAD(+) = 4-sulfobenzoate + 2 H(+) + NADH." [EC:1.2.1.62, RHEA:18836]
GO:0018483
name: 6-oxohexanoate dehydrogenase activity
def: "Catalysis of the reaction: 6-oxohexanoate + NADP+ + H2O = adipate + NADPH + H+." [EC:1.2.1.63]
GO:0018484
name: 4-hydroxybenzaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 4-hydroxybenzaldehyde + NAD+ + H2O = 4-hydroxybenzoate + NADH + H+." [EC:1.2.1.64]
GO:0018485
name: salicylaldehyde dehydrogenase activity
def: "Catalysis of the reaction: salicylaldehyde + NAD+ + H2O = salicylate + NADH + H+." [EC:1.2.1.65]
GO:0018486
name: 2-butanone oxidase activity
def: "Catalysis of the reaction: methyl ethyl ketone + O2 = H2O + ethyl acetate." [UM-BBD_reactionID:r0169]
GO:0018487
name: vanillate O-demethylase (anaerobic) activity
def: "Catalysis of the reaction: vanillate + Co+ = Co3+-CH3 + 3,4-dihydroxybenzoate." [UM-BBD_reactionID:r0758]
GO:0018488
name: aryl-aldehyde oxidase activity
def: "Catalysis of the reaction: an aromatic aldehyde + O2 + H2O = an aromatic acid + hydrogen peroxide." [EC:1.2.3.9]
GO:0018489
name: vanillate monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + vanillate = 3,4-dihydroxybenzoate + formaldehyde + H(2)O + NAD(+)." [EC:1.14.13.82, RHEA:13024]
GO:0018490
name: 4-hydroxyphenylpyruvate oxidase activity
def: "Catalysis of the reaction: 2 (4-hydroxyphenyl)pyruvate + O(2) = 2 (4-hydroxyphenyl)acetate + 2 CO(2)." [EC:1.2.3.13, RHEA:17200]
GO:0018491
name: 2-oxobutyrate synthase activity
def: "Catalysis of the reaction: 2-oxobutanoate + CoA + oxidized ferredoxin = propanoyl-CoA + CO2 + reduced ferredoxin." [EC:1.2.7.2]
GO:0018492
name: carbon-monoxide dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: CO + H2O + acceptor = CO2 + reduced acceptor." [EC:1.2.99.2]
GO:0018493
name: formylmethanofuran dehydrogenase activity
def: "Catalysis of the reaction: N-formylmethanofuran + A + H(2)O + H(+) = AH(2) + CO(2) + methanofuran." [EC:1.2.99.5, RHEA:19844]
GO:0018494
name: carvone reductase activity
def: "Catalysis of the reaction: carvone + 2 H+ + 2 e- = dihydrocarvone." [UM-BBD_reactionID:r0732]
GO:0018495
name: 2-hydroxycyclohexane-1-carboxyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2-hydroxycyclohexane-1-carboxyl-CoA = 2 H+ + 2 e- + 2-ketocyclohexane-1-carboxyl-CoA." [UM-BBD_reactionID:r0192]
GO:0018496
name: 2,6-dihydroxycyclohexane-1-carboxyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2,6-dihydroxycyclohexane-1-carboxyl-CoA = 2 H+ + 2 e- + 6-oxo-2-hydroxycyclohexane-1-carboxyl-CoA." [UM-BBD_reactionID:r0205]
GO:0018497
name: 1-chloro-2,2-bis(4-chlorophenyl)ethane dehydrogenase activity
def: "Catalysis of the reaction: 1-chloro-2,2-bis(4-chlorophenyl)ethene + 2 H+ + 2 e- = 1-chloro-2,2-bis(4-chlorophenyl)ethane. 1-chloro-2,2-bis(4-chlorophenyl)ethene is also known as DDMU; 1-chloro-2,2-bis(4-chlorophenyl)ethane is also known as DDMS." [UM-BBD_reactionID:r0514]
GO:0018498
name: 2,3-dihydroxy-2,3-dihydro-phenylpropionate dehydrogenase activity
def: "Catalysis of the reaction: cis-3-(3-carboxyethyl)-3,5-cyclohexadiene-1,2-diol + NAD+ = NADH + H+ + 3-(2,3-dihydroxyphenyl)propionate." [UM-BBD_enzymeID:e0308]
GO:0018499
name: cis-2,3-dihydrodiol DDT dehydrogenase activity
def: "Catalysis of the reaction: cis-2,3-dihydrodiol DDT + NADP+ = NADPH + 2,3-dihydroxy DDT." [UM-BBD_reactionID:r0451]
GO:0018500
name: trans-9R,10R-dihydrodiolphenanthrene dehydrogenase activity
def: "Catalysis of the reaction: trans-9R,10R-dihydrodiolphenanthrene + NAD+ = NADH + H+ + 9,10-dihydroxyphenanthrene." [UM-BBD_reactionID:r0575]
GO:0018501
name: cis-chlorobenzene dihydrodiol dehydrogenase activity
def: "Catalysis of the conversion of a di- or tetrachlorinated dienol to the corresponding catechol." [UM-BBD_enzymeID:e0411]
GO:0018502
name: 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase activity
def: "Catalysis of the reaction: 2,5-dichloro-2,5-cyclohexadiene-1,4-diol + NAD+ = NADH + H+ + 2,5-dichlorohydroquinone." [UM-BBD_reactionID:r0553]
GO:0018503
name: trans-1,2-dihydrodiolphenanthrene dehydrogenase activity
def: "Catalysis of the reaction: trans-1,2-dihydrodiolphenanthrene + NAD+ = H+ + NADH + 1,2-dihydroxyphenanthrene." [UM-BBD_reactionID:r0574]
GO:0018504
name: cis-1,2-dihydrobenzene-1,2-diol dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydrobenzene-1,2-diol + NAD+ = catechol + NADH + H+." [EC:1.3.1.19]
GO:0018505
name: cis-1,2-dihydro-1,2-dihydroxynaphthalene dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydronaphthalene-1,2-diol + NAD+ = naphthalene-1,2-diol + NADH + H+." [EC:1.3.1.29]
GO:0018506
name: maleylacetate reductase activity
def: "Catalysis of the reaction: 3-oxoadipate + NAD(P)+ = 2-maleylacetate + NAD(P)H + H+." [EC:1.3.1.32]
GO:0018507
name: cis-3,4-dihydrophenanthrene-3,4-diol dehydrogenase activity
def: "Catalysis of the reaction: (3S,4R)-3,4-dihydrophenanthrene-3,4-diol + NAD(+) = H(+) + NADH + phenanthrene-3,4-diol." [EC:1.3.1.49, RHEA:16256]
GO:0018508
name: cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate + NAD+ = catechol + CO2 + NADH + H+." [MetaCyc:BENZOATE-CIS-DIOL-DEHYDROGENASE-RXN]
GO:0018509
name: cis-2,3-dihydrobiphenyl-2,3-diol dehydrogenase activity
def: "Catalysis of the reaction: cis-3-phenylcyclohexa-3,5-diene-1,2-diol + NAD+ = biphenyl-2,3-diol + NADH + H+." [EC:1.3.1.56]
GO:0018510
name: phloroglucinol reductase activity
def: "Catalysis of the reaction: dihydrophloroglucinol + NADP(+) = H(+) + NADPH + phloroglucinol." [EC:1.3.1.57, RHEA:10083]
GO:0018511
name: 2,3-dihydroxy-2,3-dihydro-p-cumate dehydrogenase activity
def: "Catalysis of the reaction: cis-2,3-dihydroxy-2,3-dihydro-p-cumate + NAD(+) = 2,3-dihydroxy-p-cumate + H(+) + NADH." [EC:1.3.1.58, RHEA:23775]
GO:0018512
name: 1,6-dihydroxy-5-methylcyclohexa-2,4-dienecarboxylate dehydrogenase activity
def: "OBSOLETE. Catalysis of the reaction: 1,6-dihydroxy-5-methylcyclohexa-2,4-dienecarboxylate + NAD+ = 3-methylcatechol + CO2 + NADH + H+." [EC:1.3.1.59]
GO:0018513
name: dibenzothiophene dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene + NAD(+) = 1,2-dihydroxydibenzothiophene + H(+) + NADH." [EC:1.3.1.60, RHEA:24191]
GO:0018515
name: pimeloyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + pimelyl-CoA = 2,3-didehydropimeloyl-CoA + H(+) + NADH." [EC:1.3.1.62, RHEA:19668]
GO:0018516
name: 2,4-dichlorobenzoyl-CoA reductase activity
def: "Catalysis of the reaction: 4-chlorobenzoyl-CoA + chloride + NADP(+) = 2,4-dichlorobenzoyl-CoA + NADPH." [EC:1.3.1.63, RHEA:23079]
GO:0018517
name: phthalate 4,5-cis-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: cis-4,5-dihydroxycyclohexa-2,6-diene-1,2-dicarboxylate + NAD(+) = 4,5-dihydroxyphthalate + H(+) + NADH." [EC:1.3.1.64, RHEA:13840]
GO:0018518
name: 5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-tetrahydroquinoline dehydrogenase activity
def: "Catalysis of the reaction: 5,6-dihydroxy-3-methyl-5,6-dihydroquinolin-2(1H)-one + NAD(+) = 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline + H(+) + NADH." [EC:1.3.1.65, RHEA:24559]
GO:0018519
name: cis-dihydroethylcatechol dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydro-3-ethylcatechol + NAD(+) = 3-ethylcatechol + H(+) + NADH." [EC:1.3.1.66, RHEA:18104]
GO:0018520
name: cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydroxy-4-methylcyclohexa-3,5-diene-1-carboxylate + NADP+ = 4-methylcatechol + NADPH + H+ + CO2." [EC:1.3.1.67]
GO:0018521
name: 1,2-dihydroxy-6-methylcyclohexa-3,5-dienecarboxylate dehydrogenase activity
def: "Catalysis of the reaction: 1,6-dihydroxy-2-methylcyclohexa-2,4-dienecarboxylate + NAD(+) = 3-methylcatechol + CO(2) + NADH." [EC:1.3.1.68, RHEA:15660]
GO:0018522
name: benzoyl-CoA reductase activity
def: "Catalysis of the reaction: 2 ADP + cyclohexa-1,5-diene-1-carbonyl-CoA + oxidized ferredoxin + 2 phosphate = 2 ATP + 2 H(2)O + benzoyl-CoA + reduced ferredoxin." [EC:1.3.7.8]
GO:0018523
name: quinoline 2-oxidoreductase activity
def: "Catalysis of the reaction: quinoline + acceptor + H2O = isoquinolin-1(2H)-one + reduced acceptor." [EC:1.3.99.17]
GO:0018524
name: acetophenone carboxylase activity
def: "Catalysis of the reaction: acetophenone + CO2 = H+ + benzoyl acetate." [UM-BBD_reactionID:r0033]
GO:0018525
name: 4-hydroxybenzoyl-CoA reductase activity
def: "Catalysis of the reaction: benzoyl-CoA + oxidized ferredoxin + H2O = 4-hydroxybenzoyl-CoA + reduced ferredoxin." [KEGG:R05316]
GO:0018526
name: 2-aminobenzoyl-CoA reductase activity
def: "Catalysis of the reaction: 2-aminobenzoyl-CoA + 2 H+ + 2 e- = NH3 + benzoyl-CoA." [UM-BBD_reactionID:r0342]
GO:0018527
name: cyclohexylamine oxidase activity
def: "Catalysis of the reaction: cyclohexylamine + O2 + H2O = cyclohexanone + NH3 + hydrogen peroxide." [EC:1.4.3.12]
GO:0018528
name: iminodiacetate dehydrogenase activity
def: "Catalysis of the reaction: iminodiacetate + OH- = H+ + 2 e- + glyoxylate + glycine." [UM-BBD_reactionID:r0589]
GO:0018529
name: nitrilotriacetate monooxygenase activity
def: "Catalysis of the reaction: nitrilotriacetate + NADH + H+ + O2 = NAD+ + H2O + glyoxylate + iminodiacetate." [UM-BBD_reactionID:r0587]
GO:0018530
name: (R)-6-hydroxynicotine oxidase activity
def: "Catalysis of the reaction: (R)-6-hydroxynicotine + H(2)O + O(2) = 6-hydroxypseudooxynicotine + H(2)O(2)." [EC:1.5.3.6, RHEA:10015]
GO:0018531
name: (S)-6-hydroxynicotine oxidase activity
def: "Catalysis of the reaction: (S)-6-hydroxynicotine + H(2)O + O(2) = 6-hydroxypseudooxynicotine + H(2)O(2)." [EC:1.5.3.5, RHEA:11883]
GO:0018532
name: 5,10-methenyl-5,6,7,8-tetrahydromethanopterin dehydrogenase activity
def: "Catalysis of the reaction: N5,N10-methenyltetrahydromethanopterin + H2 = N5,N10-methylenetetrahydromethanopterin." [UM-BBD_reactionID:r0353]
GO:0018534
name: nitrilotriacetate dehydrogenase activity
def: "Catalysis of the reaction: nitrilotriacetate + OH- = H+ + 2 e- + glyoxylate + iminodiacetate." [UM-BBD_reactionID:r0588]
GO:0018535
name: nicotine dehydrogenase activity
def: "Catalysis of the reaction: nicotine + acceptor + H2O = (S)-6-hydroxynicotine + reduced acceptor." [EC:1.5.99.4]
GO:0018537
name: coenzyme F420-dependent N5,N10-methenyltetrahydromethanopterin reductase activity
def: "Catalysis of the reaction: 5-methyltetrahydromethanopterin + coenzyme F420 + H(+) = 5,10-methylenetetrahydromethanopterin + reduced coenzyme F420." [EC:1.5.99.11, RHEA:21147]
GO:0018538
name: epoxide carboxylase activity
def: "Catalysis of the generic reaction: epoxide + CO2 + NAD+ + electron donor = beta-keto acid + NADH + reduced electron donor; the electron donor may be NADPH or a dithiol. This reaction is the ring opening and carboxylation of an epoxide; for example: epoxypropane + CO2 + NAD+ + NADPH = acetoacetate + NADH + NADP+." [PMID:9555888]
GO:0018541
name: p-benzoquinone reductase (NADPH) activity
def: "Catalysis of the reaction: 1,4-benzoquinone + H(+) + NADPH = hydroquinone + NADP(+)." [EC:1.6.5.6, RHEA:23491]
GO:0018542
name: 2,3-dihydroxy DDT 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxy DDT + O2 = 6-oxo-2-hydroxy-7-(4-chlorophenyl)-3,8,8,8-tetrachloroocta-2E,4E-dienoate." [UM-BBD_reactionID:r0452]
GO:0018543
name: 4-amino-2-nitroso-6-nitrotoluene reductase activity
def: "Catalysis of the reaction: 4-amino-2-hydroxylamino-6-nitrotoluene + NADP+ = NADPH + H+ + 4-amino-2-nitroso-6-nitrotoluene." [UM-BBD_reactionID:r0464]
GO:0018544
name: 4-carboxy-4'-sulfoazobenzene reductase activity
def: "Catalysis of the reaction: 4-carboxy-4'-sulfoazobenzene + 4 H+ + 4 e- = 4-aminobenzoate + 4-aminobenzenesulfonate." [UM-BBD_reactionID:r0543]
GO:0018545
name: NAD(P)H nitroreductase activity
def: "Catalysis of the conversion of a nitrate group to an amino or hydroxylamino group on toluene or a toluene derivative." [UM-BBD_enzymeID:e0346]
GO:0018546
name: nitrobenzene nitroreductase activity
def: "Catalysis of the reaction: nitrobenzene + 2 NADPH + 2 H+ = hydroxyaminobenzene + 2 NADP+ + H2O." [MetaCyc:RXN-8815]
GO:0018547
name: nitroglycerin reductase activity
def: "Catalysis of the removal of one or more nitrite (NO2-) groups from nitroglycerin or a derivative." [UM-BBD_enzymeID:e0038]
GO:0018548
name: pentaerythritol trinitrate reductase activity
def: "Catalysis of the reaction: pentaerythritol trinitrate + NADPH = NADP+ + nitrate + pentaerythritol dinitrate." [UM-BBD_reactionID:r0025]
GO:0018549
name: methanethiol oxidase activity
def: "Catalysis of the reaction: methanethiol + O2 + H2O = formaldehyde + hydrogen sulfide + hydrogen peroxide." [EC:1.8.3.4]
GO:0018550
name: tetrachloro-p-hydroquinone reductive dehalogenase activity
def: "Catalysis of the reaction: 2,3,5,6-tetrachlorohydroquinone + 2 glutathione = 2,3,6-trichlorohydroquinone + glutathione disulfide + HCl." [UM-BBD_reactionID:r0314]
GO:0018551
name: hydrogensulfite reductase activity
def: "Catalysis of the reaction: trithionate + acceptor + 2 H2O + OH- = 3 HSO3- + reduced acceptor." [EC:1.8.99.3]
GO:0018553
name: 3-(2,3-dihydroxyphenyl)propionate 1,2-dioxygenase activity
def: "Catalysis of the reaction: 3-(2,3-dihydroxyphenyl)propionate + O2 = 2-hydroxy-6-keto-nona-2,4-dienedioate." [UM-BBD_enzymeID:e0309]
GO:0018554
name: 1,2-dihydroxynaphthalene dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxynaphthalene + O2 = 2-hydroxychromene-2-carboxylate. C6 of the substrate molecular may have an NH2 group attached." [UM-BBD_enzymeID:e0255]
GO:0018555
name: phenanthrene dioxygenase activity
def: "Catalysis of the reaction: phenanthrene + NADH + H+ + O2 = NAD+ + cis-3,4-dihydroxy-3,4-dihydrophenanthrene." [UM-BBD_reactionID:r0455]
GO:0018556
name: 2,2',3-trihydroxybiphenyl dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxybenzenoid + O2 = H+ + distal extradiol ring cleavage. Substrates are 2,2',3-trihydroxybiphenyl (forms 2-hydroxy-6-oxo-6-(2-hydroxyphenyl)-hexa-2,4-dienoate) and 2,2',3-trihydroxydiphenylether (forms 2,3-hydroxy-6-oxo-6-(2-hydroxyphenyl)-hexa-2,4-dienoate)." [UM-BBD_enzymeID:e0032]
GO:0018557
name: 1,2-dihydroxyfluorene 1,1-alpha-dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxyfluorene + O2 = 2-hydroxy-4-(2-oxo-1,3-dihydro-2H-inden-1-ylidene) but-2-enoic acid." [UM-BBD_reactionID:r0422]
GO:0018558
name: 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline dioxygenase activity
def: "Catalysis of the reaction: 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline + O2 = 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon." [UM-BBD_reactionID:r0049]
GO:0018559
name: 1,1-dichloro-2-(dihydroxy-4-chlorophenyl)-(4-chlorophenyl)ethene 1,2-dioxygenase activity
def: "Catalysis of the reaction: 1,1-dichloro-2-(dihydroxy-4'-chorophenyl)-2-(4-chlorophenyl)ethene + O2 = 6-oxo-2-hydroxy-7-(4-chlorophenyl)-3,8,8-trichloroocta-2E,4E,7-trienoate." [UM-BBD_reactionID:r0442]
GO:0018560
name: protocatechuate 3,4-dioxygenase type II activity
def: "Catalysis of the reaction: 4-sulfocatechol + O2 = 3-sulfomuconate." [UM-BBD_reactionID:r0581]
GO:0018561
name: 2'-aminobiphenyl-2,3-diol 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2'-aminobiphenyl-2,3-diol + O2 = H+ + 2-hydroxy-6-oxo-(2'-aminophenyl)-hexa-2,4-dienoate." [UM-BBD_reactionID:r0457]
GO:0018562
name: 3,4-dihydroxyfluorene 4,4-alpha-dioxygenase activity
def: "Catalysis of the reaction: 3,4-dihydroxyfluorene + O2 = 2-hydroxy-4-(1-oxo-1,3-dihydro-2H-inden-2-ylidene) but-2-enoic acid." [UM-BBD_reactionID:r0415]
GO:0018563
name: 2,3-dihydroxy-ethylbenzene 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxyethylbenzene + O2 = H+ + 2-hydroxy-6-oxoocta-2,4-dienoate." [UM-BBD_reactionID:r0310]
GO:0018564
name: carbazole 1,9a-dioxygenase activity
def: "Catalysis of the reaction: carbazole + NADH + O2 + H+ = NAD+ + 2'-aminobiphenyl-2,3-diol." [UM-BBD_reactionID:r0456]
GO:0018565
name: dihydroxydibenzothiophene dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxydibenzothiophene + O2 = cis-4-(2-(3-hydroxy)-thionaphthenyl)-2-oxo-3-butenoate." [UM-BBD_reactionID:r0162]
GO:0018566
name: 1,2-dihydroxynaphthalene-6-sulfonate 1,8a-dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxynaphthalene-6-sulfonate + O2 = H+ + (Z)-4-(2-hydroxy-5-sulfonatophenyl)-2-oxo-3-butenoate." [UM-BBD_reactionID:r0324]
GO:0018567
name: styrene dioxygenase activity
def: "Catalysis of the reaction: styrene + O2 + NADH + H+ = NAD+ + styrene cis-glycol." [UM-BBD_reactionID:r0256]
GO:0018568
name: 3,4-dihydroxyphenanthrene dioxygenase activity
def: "Catalysis of the reaction: 3,4-dihydroxyphenanthrene + O2 = H+ + 2-hydroxy-2 H-benzo[h]chromene-2-carboxylate." [UM-BBD_reactionID:r0501]
GO:0018569
name: hydroquinone 1,2-dioxygenase activity
def: "Catalysis of the reaction: hydroquinone + O2 = cis,trans-4-hydroxymuconic semialdehyde." [UM-BBD_reactionID:r0228]
GO:0018570
name: p-cumate 2,3-dioxygenase activity
def: "Catalysis of the reaction: p-cumate + NADH + H+ + O2 = NAD+ + cis-2,3-dihydroxy-2,3-dihydro-p-cumate." [UM-BBD_reactionID:r0395]
GO:0018571
name: 2,3-dihydroxy-p-cumate dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxy-p-cumate + O2 = 2-hydroxy-3-carboxy-6-oxo-7-methylocta-2,4-dienoate." [MetaCyc:RXN-666, UM-BBD_reactionID:r0397]
GO:0018572
name: 3,5-dichlorocatechol 1,2-dioxygenase activity
def: "Catalysis of the reaction: 3,5-dichlorocatechol + O2 = 2 H+ + 2,4-dichloro-cis,cis-muconate." [UM-BBD_reactionID:r0276]
GO:0018573
name: 2-aminophenol 1,6-dioxygenase activity
def: "Catalysis of the reaction: 2-aminophenol + O2 = 2-aminomuconic semialdehyde." [UM-BBD_reactionID:r0305]
GO:0018574
name: 2,6-dichloro-p-hydroquinone 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,6-dichlorohydroquinone + O2 + H2O = HCl + 2 H+ + 2-chloromaleylacetate." [UM-BBD_enzymeID:e0422]
GO:0018575
name: chlorocatechol 1,2-dioxygenase activity
def: "Catalysis of the reaction: 3,6-dichlorocatechol + O2 = 2 H+ + 2,5-dichloro-cis,cis-muconate." [UM-BBD_reactionID:r0655]
GO:0018576
name: catechol 1,2-dioxygenase activity
def: "Catalysis of the reaction: catechol + O2 = cis,cis-muconate." [EC:1.13.11.1]
GO:0018577
name: catechol 2,3-dioxygenase activity
def: "Catalysis of the reaction: catechol + O2 = 2-hydroxymuconate semialdehyde." [EC:1.13.11.2]
GO:0018578
name: protocatechuate 3,4-dioxygenase activity
def: "Catalysis of the reaction: 3,4-dihydroxybenzoate + O2 = 3-carboxy-cis,cis-muconate." [EC:1.13.11.3]
GO:0018579
name: protocatechuate 4,5-dioxygenase activity
def: "Catalysis of the reaction: protocatechuate + O2 = 4-carboxy-2-hydroxymuconate semialdehyde." [EC:1.13.11.8]
GO:0018580
name: nitronate monooxygenase activity
def: "Catalysis of the reaction: ethylnitronate + O(2) = acetaldehyde + nitrite." [EC:1.13.12.16, RHEA:28770]
GO:0018581
name: hydroxyquinol 1,2-dioxygenase activity
def: "Catalysis of the reaction: benzene-1,2,4-triol + O2 = 3-hydroxy-cis,cis-muconate." [EC:1.13.11.37]
GO:0018582
name: 1-hydroxy-2-naphthoate 1,2-dioxygenase activity
def: "Catalysis of the reaction: 1-hydroxy-2-naphthoate + O2 = (3E)-4-(2-carboxyphenyl)-2-oxobut-3-enoate." [EC:1.13.11.38]
GO:0018583
name: biphenyl-2,3-diol 1,2-dioxygenase activity
def: "Catalysis of the reaction: biphenyl-2,3-diol + O2 = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate + H2O." [EC:1.13.11.39]
GO:0018584
name: 2,4,5-trichlorophenoxyacetic acid oxygenase activity
def: "Catalysis of the reaction: 2,4,5-trichlorophenoxyacetic acid + 1/2 O2 = glyoxylate + 2,4,5-trichlorophenol." [UM-BBD_reactionID:r0359]
GO:0018585
name: fluorene oxygenase activity
def: "Catalysis of the reaction: fluorene + 2 H+ + 2 e- + O2 = H2O + 9-fluorenol." [UM-BBD_reactionID:r0407]
GO:0018586
name: mono-butyltin dioxygenase activity
def: "Catalysis of the reaction: butyltin + O2 + 2 H+ + 2 e- = H2O + beta-hydroxybutyltin." [UM-BBD_reactionID:r0647]
GO:0018587
name: limonene 8-monooxygenase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0018588
name: tri-n-butyltin dioxygenase activity
def: "Catalysis of the reaction: tri-n-butyltin + O2 + 2 H+ + 2 e- = H2O + beta-hydroxybutyldibutyltin." [UM-BBD_reactionID:r0643]
GO:0018589
name: di-n-butyltin dioxygenase activity
def: "Catalysis of the reaction: dibutyltin + O2 + 2 H+ + 2 e- = H2O + beta-hydroxybutylbutyltin." [UM-BBD_reactionID:r0645]
GO:0018590
name: methylsilanetriol hydroxylase activity
def: "Catalysis of the reaction: methylsilanetriol + O2 + 2 H+ + 2 e- = H2O + hydroxymethylsilanetriol." [UM-BBD_reactionID:r0640]
GO:0018591
name: methyl tertiary butyl ether 3-monooxygenase activity
def: "Catalysis of the reaction: methyl tert-butyl ether + 1/2 O2 = tert-butyl alcohol + formaldehyde." [UM-BBD_reactionID:r1023]
GO:0018592
name: 4-nitrocatechol 4-monooxygenase activity
def: "Catalysis of the reaction: 4-nitrocatechol + O2 + 4 e- + 3 H+ = H2O + nitrite + 1,2,4-benzenetriol." [UM-BBD_reactionID:r0231]
GO:0018593
name: 4-chlorophenoxyacetate monooxygenase activity
def: "Catalysis of the reaction: 4-chlorophenoxyacetate + 1/2 O2 = glyoxylate + 4-chlorophenol." [UM-BBD_reactionID:r0281]
GO:0018594
name: tert-butanol 2-monooxygenase activity
def: "Catalysis of the reaction: tert-butanol + 1/2 O2 = 2-methyl-2-hydroxy-1-propanol." [UM-BBD_reactionID:r0615]
GO:0018595
name: alpha-pinene monooxygenase activity
def: "Catalysis of the reaction: alpha-pinene + O2 + 2 H+ + 2 e- = H2O + pinocarveol." [UM-BBD_reactionID:r0716]
GO:0018596
name: dimethylsilanediol hydroxylase activity
def: "Catalysis of the reaction: dimethylsilanediol + O2 + 2 H+ + 2 e- = H2O + hydroxymethylmethylsilanediol." [UM-BBD_reactionID:r0637]
GO:0018597
name: ammonia monooxygenase activity
def: "Catalysis of the oxidation of alkanes (up to C8) to alcohols and alkenes (up to C5) to epoxides and alcohols in the presence of ammonium ions." [PMID:16347810]
GO:0018598
name: hydroxymethylsilanetriol oxidase activity
def: "Catalysis of the reaction: hydroxymethylsilanetriol + O2 + 2 H+ + 2 e- = 2 H2O + formylsilanetriol." [UM-BBD_reactionID:r0641]
GO:0018599
name: 2-hydroxyisobutyrate 3-monooxygenase activity
def: "Catalysis of the reaction: 2-hydroxyisobutyrate + 1/2 O2 = 2,3-dihydroxy-2-methyl propionate." [UM-BBD_reactionID:r0619]
GO:0018600
name: alpha-pinene dehydrogenase activity
def: "Catalysis of the reaction: alpha-pinene + O2 + 2 H+ + 2 e- = H2O + myrtenol." [UM-BBD_reactionID:r0709]
GO:0018601
name: 4-nitrophenol 2-monooxygenase activity
def: "Catalysis of the reaction: 4-nitrophenol + H(+) + NADH + O(2) = 4-nitrocatechol + H(2)O + NAD(+)." [EC:1.14.13.29, RHEA:12571]
GO:0018602
name: 2,4-dichlorophenoxyacetate alpha-ketoglutarate dioxygenase activity
def: "Catalysis of the reaction: 2,4-dichlorophenoxyacetate + 2-oxoglutarate + oxygen = 2,4-dichlorophenol + glyoxylate + succinate + CO2." [UM-BBD_reactionID:r0274]
GO:0018603
name: nitrobenzene 1,2-dioxygenase activity
def: "Catalysis of the reaction: nitrobenzene + NADH + O2 = NAD+ + nitrite + catechol." [UM-BBD_reactionID:r0306]
GO:0018604
name: 4-aminobenzoate 3,4-dioxygenase (deaminating) activity
def: "Catalysis of the reaction: 4-aminobenzoate + 2 H+ + O2 + 2 e- = NH3 + 3,4-dihydroxybenzoate." [UM-BBD_reactionID:r0566]
GO:0018606
name: benzenesulfonate dioxygenase activity
def: "Catalysis of the reaction: toluene-4-sulfonate + NADH + O2 + H+ = NAD+ + HSO3(-) + 4-methylcatechol." [UM-BBD_reactionID:r0295]
GO:0018607
name: 1-indanone monooxygenase activity
def: "Catalysis of the reaction: 1-indanone + NADPH + 1/2 O2 = NADP+ + 3,4-dihydrocoumarin." [UM-BBD_reactionID:r0417]
GO:0018608
name: 1-indanone dioxygenase activity
def: "Catalysis of the reaction: 1-indanone + NADPH + 1/2 O2 = NADP+ + 3-hydroxy-1-indanone." [UM-BBD_reactionID:r0416]
GO:0018609
name: chlorobenzene dioxygenase activity
def: "Catalysis of the reaction: 2,3-unsubstituted benzenoid + O2 + NAD(P)H + H+ = 2,3-cis-dihydroxydihydrobenzenoid + NAD(P)+. Substrates include 1,4-dichlorobenzene (forms 3,6-dichloro-cis-1,2-dihydroxycyclohexa-3,5-diene), 1,2,3,4-tetrachlorobenzene (forms cis-chlorobenzene dihydrodiol) and 1,2,4-trichlorobenzene (forms 3,4,6-trichloro-cis-1,2-dihydroxycyclohexa-3,5-diene)." [UM-BBD_enzymeID:e0062]
GO:0018610
name: dibenzofuran 4,4a-dioxygenase activity
def: "Catalysis of the reaction: dibenzofuran + NADH + H+ + O2 = 2,2',3-trihydroxybiphenyl + NAD+." [MetaCyc:R606-RXN]
GO:0018611
name: toluate dioxygenase activity
def: "Catalysis of the reaction: methylbenzoate + NADH + O2 + H+ = NAD+ + 1,2-dihydroxymethylcyclohexa-3,5-dienecarboxylate." [UM-BBD_enzymeID:e0190]
GO:0018612
name: dibenzothiophene dioxygenase activity
def: "Catalysis of the reaction: dibenzothiophene + NADH + O2 + H+ = NAD+ + cis-1,2-dihydroxy-1,2-dihydrodibenzothiophene." [UM-BBD_reactionID:r0160]
GO:0018613
name: 9-fluorenone dioxygenase activity
def: "Catalysis of the reaction: 9-fluorenone + 2 NADPH + O2 = 2 NADP+ + 3,4-dihydroxy-3,4-dihydro-9-fluorenone." [UM-BBD_reactionID:r0409]
GO:0018614
name: ethylbenzene dioxygenase activity
def: "Catalysis of the reaction: ethylbenzene + O2 + NADH + H+ = NAD+ + cis-2,3-dihydroxy-2,3-dihydroethylbenzene." [UM-BBD_reactionID:r0247]
GO:0018615
name: 2-indanone monooxygenase activity
def: "Catalysis of the reaction: 2-indanone + NADPH + 1/2 O2 = NADP+ + 3-isochromanone." [UM-BBD_reactionID:r0424]
GO:0018616
name: trihydroxytoluene dioxygenase activity
def: "Catalysis of the reaction: 2,3,5-trihydroxytoluene + O2 = 2,4,6-trioxoheptanoate." [MetaCyc:R305-RXN, UM-BBD_reactionID:r0093]
GO:0018617
name: 4-aminobenzenesulfonate 3,4-dioxygenase (deaminating) activity
def: "Catalysis of the reaction: 4-aminobenzenesulfonate + 2 H+ + O2 + 2 e- = NH3 + 4-sulfocatechol." [UM-BBD_reactionID:r0580]
GO:0018618
name: anthranilate 1,2-dioxygenase (deaminating, decarboxylating) activity
def: "Catalysis of the reaction: anthranilate + NADPH + H+ + O2 = catechol + CO2 + NADP+ + NH3." [EC:1.14.12.1]
GO:0018619
name: benzene 1,2-dioxygenase activity
def: "Catalysis of the reaction: benzene + H(+) + NADH + O(2) = cis-cyclohexa-3,5-diene-1,2-diol + NAD(+)." [EC:1.14.12.3, RHEA:13816]
GO:0018620
name: phthalate 4,5-dioxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + phthalate = cis-4,5-dihydroxycyclohexa-2,6-diene-1,2-dicarboxylate + NAD(+)." [EC:1.14.12.7, RHEA:17492]
GO:0018621
name: 4-sulfobenzoate 3,4-dioxygenase activity
def: "Catalysis of the reaction: 4-sulfobenzoate + H(+) + NADH + O(2) = 3,4-dihydroxybenzoate + NAD(+) + sulfite." [EC:1.14.12.8, RHEA:13940]
GO:0018622
name: 4-chlorophenylacetate 3,4-dioxygenase activity
def: "Catalysis of the reaction: 4-chlorophenylacetate + NADH + O(2) = 3,4-dihydroxyphenylacetate + chloride + NAD(+)." [EC:1.14.12.9, RHEA:14692]
GO:0018623
name: benzoate 1,2-dioxygenase activity
def: "Catalysis of the reaction: benzoate + NADH + H+ + O2 = catechol + CO2 + NAD+." [EC:1.14.12.10]
GO:0018624
name: toluene dioxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + toluene = (1S,2R)-3-methylcyclohexa-3,5-diene-1,2-diol + NAD(+)." [EC:1.14.12.11, RHEA:16740]
GO:0018625
name: naphthalene 1,2-dioxygenase activity
def: "Catalysis of the reaction: naphthalene + NADH + H+ + O2 = (1R,2S)-1,2-dihydronaphthalene-1,2-diol + NAD+." [EC:1.14.12.12]
GO:0018626
name: 2-chlorobenzoate 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2-chlorobenzoate + NADH + H+ + O2 = catechol + chloride + NAD+ + CO2." [EC:1.14.12.13]
GO:0018627
name: 2-aminobenzenesulfonate 2,3-dioxygenase activity
def: "Catalysis of the reaction: 2-aminobenzenesulfonate + 2 H(+) + NADH + O(2) = 2,3-dihydroxybenzenesulfonate + NAD(+) + NH(4)(+). 2,3-dihydroxybenzenesulfonate is also known as 3-sulfocatechol." [EC:1.14.12.14, RHEA:23471]
GO:0018628
name: terephthalate 1,2-dioxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + terephthalate = (3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate + NAD(+)." [EC:1.14.12.15, RHEA:10315]
GO:0018629
name: 2-hydroxyquinoline 5,6-dioxygenase activity
def: "Catalysis of the reaction: quinolin-2-ol + NADH + H+ + O2 = 2,5,6-trihydroxy-5,6-dihydroquinoline + NAD+." [EC:1.14.12.16]
GO:0018630
name: 3,5-xylenol methylhydroxylase activity
def: "Catalysis of the reaction: 3-hydroxytoluene + NADH + O2 = NAD+ + OH- + 3-hydroxybenzyl alcohol." [UM-BBD_reactionID:r0081]
GO:0018631
name: phenylacetate hydroxylase activity
def: "Catalysis of the reaction: phenylacetate + NADH + O2 = NAD+ + OH- + 2-hydroxyphenylacetate." [UM-BBD_reactionID:r0036]
GO:0018632
name: 4-nitrophenol 4-monooxygenase activity
def: "Catalysis of the reaction: p-nitrophenol + O2 + NADPH = H2O + NADP+ + nitrite + p-benzoquinone." [UM-BBD_reactionID:r0226]
GO:0018633
name: dimethyl sulfide monooxygenase activity
def: "Catalysis of the reaction: dimethyl sulfide + NADH + O2 = NAD+ + OH- + methanethiol + formaldehyde." [UM-BBD_reactionID:r0208]
GO:0018634
name: alpha-pinene monooxygenase [NADH] activity
def: "Catalysis of the reaction: alpha-pinene + NADH + H+ + O2 = NAD+ + H2O + alpha-pinene oxide." [UM-BBD_reactionID:r0742]
GO:0018635
name: (R)-limonene 1,2-monooxygenase activity
def: "Catalysis of the reaction: (4R)-limonene + NAD(P)H + H+ + O2 = NAD(P)+ + H2O + (4R)-limonene-1,2-epoxide." [UM-BBD_enzymeID:e0462]
GO:0018636
name: phenanthrene 9,10-monooxygenase activity
def: "Catalysis of the reaction: phenanthrene + O2 + NADH + H+ = H2O + NAD+ + phenanthrene-9,10-oxide." [UM-BBD_reactionID:r0495]
GO:0018637
name: 1-hydroxy-2-naphthoate hydroxylase activity
def: "Catalysis of the reaction: 1-hydroxy-2-naphthoate + O2 + NADPH + 2 H+ = NADP+ + H2O + CO2 + 1,2-dihydroxynaphthalene." [UM-BBD_reactionID:r0491]
GO:0018638
name: toluene 4-monooxygenase activity
def: "Catalysis of the reaction: toluene + 1/2 O2 = 4-hydroxytoluene." [UM-BBD_enzymeID:e0225]
GO:0018639
name: xylene monooxygenase activity
def: "Catalysis of the reactions: toluene + 1/2 O2 = benzyl alcohol, and xylene + 1/2 O2 = methylbenzyl alcohol." [UM-BBD_enzymeID:e0172]
GO:0018640
name: dibenzothiophene monooxygenase activity
def: "Catalysis of the reaction: dibenzothiophene + NADH + H+ + O2 = dibenzothiophene-5-oxide + NAD+ + H2O." [MetaCyc:RXN-621]
GO:0018641
name: 6-hydroxy-3-methyl-2-oxo-1,2-dihydroquinoline 6-monooxygenase activity
def: "Catalysis of the reaction: 6-hydroxy-3-methyl-2-oxo-1,2-dihydroquinoline + O2 + 2 H+ + 2 e- = H2O + 5,6-dihydroxy-3-methyl-2-oxo-1,2-dihydroquinoline." [UM-BBD_reactionID:r0048]
GO:0018642
name: chlorophenol 4-monooxygenase activity
def: "Catalysis of the addition or substitution of an OH group on C4 of a halogenated phenol." [UM-BBD_enzymeID:e0252]
GO:0018643
name: carbon disulfide oxygenase activity
def: "Catalysis of the reaction: carbon disulfide + NADH + H+ + O2 = [S] + H2O + NAD+ + carbonyl sulfide." [UM-BBD_reactionID:r0599]
GO:0018644
name: toluene 2-monooxygenase activity
def: "Catalysis of the reaction: toluene + 1/2 O2 = 2-hydroxytoluene." [UM-BBD_enzymeID:e0222]
GO:0018645
name: alkene monooxygenase activity
def: "Catalysis of the reaction: propene + NADH + H+ + O2 = 1,2-epoxypropane + NAD+ + H2O." [EC:1.14.13.69]
GO:0018646
name: 1-hydroxy-2-oxolimonene 1,2-monooxygenase activity
def: "Catalysis of the reaction: (1S,4R)-1-hydroxy-2-oxolimonene + NADPH + O2 = NADP+ + OH- + (3R)-3-isopropenyl-6-oxoheptanoate." [UM-BBD_reactionID:r0736]
GO:0018647
name: phenanthrene 1,2-monooxygenase activity
def: "Catalysis of the reaction: phenanthrene + O2 + NADH + H+ = H2O + NAD+ + phenanthrene-1,2-oxide." [UM-BBD_enzymeID:e0333]
GO:0018648
name: methanesulfonate monooxygenase activity
def: "Catalysis of the reaction: methanesulfonate + NADH + H+ + O2 = formaldehyde + NAD+ + sulfite + H2O." [EC:1.14.13.111]
GO:0018649
name: tetrahydrofuran hydroxylase activity
def: "Catalysis of the reaction: tetrahydrofuran + O2 + 2 H+ + 2 e- = H2O + 2-hydroxytetrahydrofuran." [UM-BBD_reactionID:r0017]
GO:0018650
name: styrene monooxygenase activity
def: "Catalysis of the reaction: styrene + NADPH + FADH + O2 = NADP+ + FAD+ + H2O + styrene oxide." [UM-BBD_reactionID:r0225]
GO:0018651
name: toluene-4-sulfonate monooxygenase activity
def: "Catalysis of the reaction: toluene-4-sulfonate + 1/2 O2 + H+ = HSO3(-) + 4-hydroxytoluene." [UM-BBD_reactionID:r0296]
GO:0018652
name: toluene-sulfonate methyl-monooxygenase activity
def: "Catalysis of the reaction: toluene-4-sulfonate + NADH + O2 = NAD+ + OH- + 4-sulfobenzyl alcohol." [UM-BBD_reactionID:r0290]
GO:0018653
name: 3-methyl-2-oxo-1,2-dihydroquinoline 6-monooxygenase activity
def: "Catalysis of the reaction: 3-methyl-2-oxo-1,2-dihydroquinoline + O2 + 2 H+ + 2 e- = H2O + 6-hydroxy-3-methyl-2-oxo-1,2-dihydroquinoline." [EC:1.14.13.-]
GO:0018654
name: 2-hydroxy-phenylacetate hydroxylase activity
def: "Catalysis of the reaction: 2-hydroxyphenylacetate + NADH + O2 = NAD+ + OH- + homogentisate." [UM-BBD_reactionID:r0252]
GO:0018655
name: 2-oxo-delta3-4,5,5-trimethylcyclopentenylacetyl-CoA 1,2-monooxygenase activity
def: "Catalysis of the reaction: 2-oxo-delta3-4,5,5-trimethylcyclopentenylacetyl-CoA + NADH + H+ + O2 = NAD+ + H2O + delta2,5-3,4,4-trimethylpimelyl-CoA." [UM-BBD_reactionID:r0430]
GO:0018656
name: phenanthrene 3,4-monooxygenase activity
def: "Catalysis of the reaction: phenanthrene + O2 + NADH + H+ = H2O + NAD+ + phenanthrene-3,4-oxide." [UM-BBD_reactionID:r0508]
GO:0018657
name: toluene 3-monooxygenase activity
def: "Catalysis of the reaction: toluene + 1/2 O2 = 3-hydroxytoluene." [UM-BBD_enzymeID:e0224]
GO:0018658
name: salicylate 1-monooxygenase activity
def: "Catalysis of the reaction: salicylate + NADH + H+ + O2 = catechol + NAD+ + H2O + CO2." [EC:1.14.13.1]
GO:0018659
name: 4-hydroxybenzoate 3-monooxygenase activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + NADPH + H+ + O2 = protocatechuate + NADP+ + H2O." [EC:1.14.13.2]
GO:0018660
name: 4-hydroxyphenylacetate,NADH:oxygen oxidoreductase (3-hydroxylating) activity
def: "Catalysis of the reaction: 4-hydroxyphenylacetate + NADH + H+ + O2 = 3,4-dihydroxyphenylacetate + NAD+ + H2O." [KEGG:R02698]
GO:0018661
name: orcinol 2-monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + orcinol = 2,3,5-trihydroxytoluene + H(2)O + NAD(+)." [EC:1.14.13.6, RHEA:19604]
GO:0018662
name: phenol 2-monooxygenase activity
def: "Catalysis of the reaction: phenol + NADPH + H+ + O2 = catechol + NADP+ + H2O." [EC:1.14.13.7]
GO:0018663
name: 2,6-dihydroxypyridine 3-monooxygenase activity
def: "Catalysis of the reaction: 2,6-dihydroxypyridine + H(+) + NADH + O(2) = 2,3,6-trihydroxypyridine + H(2)O + NAD(+)." [EC:1.14.13.10, RHEA:16920]
GO:0018664
name: benzoate 4-monooxygenase activity
def: "Catalysis of the reaction: benzoate + H(+) + NADPH + O(2) = 4-hydroxybenzoate + H(2)O + NADP(+)." [EC:1.14.13.12, RHEA:18036]
GO:0018665
name: 4-hydroxyphenylacetate 1-monooxygenase activity
def: "Catalysis of the reaction: 4-hydroxyphenylacetate + NADPH + H+ + O2 = homogentisate + NADP+ + H2O." [EC:1.14.13.18]
GO:0018666
name: 2,4-dichlorophenol 6-monooxygenase activity
def: "Catalysis of the reaction: 2,4-dichlorophenol + NADPH + H+ + O2 = 3,5-dichlorocatechol + NADP+ + H2O." [EC:1.14.13.20]
GO:0018667
name: cyclohexanone monooxygenase activity
def: "Catalysis of the reaction: cyclohexanone + NADPH + H+ + O2 = 6-hexanolide + NADP+ + H2O." [EC:1.14.13.22]
GO:0018668
name: 3-hydroxybenzoate 4-monooxygenase activity
def: "Catalysis of the reaction: 3-hydroxybenzoate + H(+) + NADPH + O(2) = 3,4-dihydroxybenzoate + H(2)O + NADP(+)." [EC:1.14.13.23, RHEA:11483]
GO:0018669
name: 3-hydroxybenzoate 6-monooxygenase activity
def: "Catalysis of the reaction: 3-hydroxybenzoate + H(+) + NADH + O(2) = 2,5-dihydroxybenzoate + H(2)O + NAD(+)." [EC:1.14.13.24, RHEA:22695]
GO:0018670
name: 4-aminobenzoate 1-monooxygenase activity
def: "Catalysis of the reaction: 4-aminobenzoate + NADPH + H+ + O2 = 4-hydroxyaniline + NADP+ + H2O + CO2." [EC:1.14.13.27]
GO:0018671
name: 4-hydroxybenzoate 3-monooxygenase [NAD(P)H] activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + NAD(P)H + H+ + O2 = 3,4-dihydroxybenzoate + NAD(P)+ + H2O." [EC:1.14.13.33]
GO:0018672
name: anthranilate 3-monooxygenase (deaminating) activity
def: "Catalysis of the reaction: anthranilate + 2 H(+) + NADPH + O(2) = 2,3-dihydroxybenzoate + NADP(+) + NH(4)(+)." [EC:1.14.13.35, RHEA:21239]
GO:0018673
name: anthraniloyl-CoA monooxygenase activity
def: "Catalysis of the reaction: 2-aminobenzoyl-CoA + 2 NADPH + 2 H+ + O2 = 2-amino-5-oxocyclohex-1-enecarboxyl-CoA + H2O + 2 NADP+." [EC:1.14.13.40]
GO:0018674
name: (S)-limonene 3-monooxygenase activity
def: "Catalysis of the reaction: (4S)-limonene + H(+) + NADPH + O(2) = (1S,6R)-isopiperitenol + H(2)O + NADP(+)." [EC:1.14.13.47, RHEA:15132]
GO:0018675
name: (S)-limonene 6-monooxygenase activity
def: "Catalysis of the reaction: (-)-limonene + NADPH + H+ + O2 = (-)-trans-carveol + NADP+ + H2O." [EC:1.14.13.48]
GO:0018676
name: (S)-limonene 7-monooxygenase activity
def: "Catalysis of the reaction: (4S)-limonene + H(+) + NADPH + O(2) = (4S)-perillyl alcohol + H(2)O + NADP(+)." [EC:1.14.13.49, RHEA:23435]
GO:0018677
name: pentachlorophenol monooxygenase activity
def: "Catalysis of the reaction: pentachlorophenol + NADPH + H+ + O2 = tetrachlorohydroquinone + NADP+ + chloride." [EC:1.14.13.50]
GO:0018678
name: 4-hydroxybenzoate 1-hydroxylase activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + NADPH + H+ + O2 = hydroquinone + NADP+ + H2O + CO2." [EC:1.14.13.64]
GO:0018679
name: dibenzothiophene-5,5-dioxide monooxygenase activity
def: "Catalysis of the reaction: dibenzothiophene-5,5-dioxide + O2 + 2 NADH + H+ = 2 NAD+ + H2O + 2'-hydroxybiphenyl-2-sulfinate." [UM-BBD_reactionID:r0235]
GO:0018680
name: deethylatrazine monooxygenase activity
def: "Catalysis of the reaction: 2 deethylatrazine + O2 = 2 CH3COCH3 + 2 deisopropyldeethylatrazine." [UM-BBD_reactionID:r0128]
GO:0018681
name: deisopropylatrazine monooxygenase activity
def: "Catalysis of the reaction: 2 deisopropylatrazine + O2 = 2 acetaldehyde + 2 deisopropyldeethylatrazine." [KEGG:R05567]
GO:0018682
name: atrazine N-dealkylase activity
def: "Catalysis of the reaction: atrazine + O2 + 2 H+ = deethylatrazine + acetaldehyde + H2O." [MetaCyc:R461-RXN, UM-BBD_reactionID:r0127]
GO:0018683
name: camphor 5-monooxygenase activity
def: "Catalysis of the reaction: (+)-camphor + putidaredoxin + O2 = (+)-exo-5-hydroxycamphor + oxidized putidaredoxin + H2O." [EC:1.14.15.1]
GO:0018684
name: camphor 1,2-monooxygenase activity
def: "Catalysis of the reaction: (1R)-bornane-2,5-dione + reduced rubredoxin + O2 = 5-oxo-1,2-campholide + oxidized rubredoxin + H2O." [EC:1.14.15.2]
GO:0018685
name: alkane 1-monooxygenase activity
def: "Catalysis of the reaction: octane + reduced rubredoxin + O2 = 1-octanol + oxidized rubredoxin + H2O." [EC:1.14.15.3]
GO:0018686
name: 6-hydroxy pseudo-oxynicotine monooxygenase activity
def: "Catalysis of the reaction: 2 6-hydroxypseudooxynicotine + O2 = 2 2,6-dihydroxypseudooxynicotine." [UM-BBD_reactionID:r0480]
GO:0018687
name: biphenyl 2,3-dioxygenase activity
def: "Catalysis of the reaction: biphenyl + NADH + H+ + O2 = (2R,3S)-3-phenylcyclohexa-3,5-diene-1,2-diol + NAD+. This reaction requires Fe2+." [EC:1.14.12.18]
GO:0018688
name: DDT 2,3-dioxygenase activity
def: "Catalysis of the reaction: 1,1,1-trichloro-2,2-bis-(4-chlorophenyl)ethane + O2 + 2 H+ + 2 e- = cis-2,3-dihydrodiol DDT. 1,1,1-trichloro-2,2-bis-(4-chlorophenyl)ethane is also known as DDT." [UM-BBD_reactionID:r0450]
GO:0018689
name: naphthalene disulfonate 1,2-dioxygenase activity
def: "Catalysis of the reaction: (1/2)-unsubstituted naphthalenoid-(2/1)-sulfonate + 2 H+ + 2 e- + O2 = 1,2-dihydroxynaphthalene derivative + HSO3(-). Substrates include naphthalene-1,6-disulfonate (forms 1,2-dihydroxynaphthalene-6-sulfonate), naphthalene-1-sulfonate (forms 1,2-dihydroxynaphthalene), naphthalene-2,6-disulfonate (forms 1,2-dihydroxynaphthalene-6-sulfonate) and naphthalene-2-sulfonate (forms 1,2-dihydroxynaphthalene)." [UM-BBD_enzymeID:e0249]
GO:0018690
name: 4-methoxybenzoate monooxygenase (O-demethylating) activity
def: "Catalysis of the reaction: 4-methoxybenzoate + AH(2) + O(2) = 4-hydroxybenzoate + A + formaldehyde + H(2)O." [EC:1.14.99.15, RHEA:18616]
GO:0018693
name: ethylbenzene hydroxylase activity
def: "Catalysis of the reaction: A + ethylbenzene + H(2)O = (S)-1-phenylethanol + AH(2)." [EC:1.17.99.2, RHEA:17900]
GO:0018694
name: p-cymene methyl hydroxylase activity
def: "Catalysis of the reaction: p-cymene + NADH + O2 = NAD+ + OH- + p-cumic alcohol." [UM-BBD_reactionID:r0392]
GO:0018695
name: 4-cresol dehydrogenase (hydroxylating) activity
def: "Catalysis of the reaction: 4-cresol + acceptor + H2O = 4-hydroxybenzaldehyde + reduced acceptor." [EC:1.17.99.1]
GO:0018697
name: carbonyl sulfide nitrogenase activity
def: "Catalysis of the reaction: carbonyl sulfide + 2 H+ + 2 e- = hydrogen sulfide + carbon monoxide." [UM-BBD_reactionID:r0600]
GO:0018698
name: vinyl chloride reductive dehalogenase activity
def: "Catalysis of the reaction: vinyl chloride + 2 H+ + 2 e- = HCl + ethene." [UM-BBD_reactionID:r0352]
GO:0018699
name: 1,1,1-trichloroethane reductive dehalogenase activity
def: "Catalysis of the reaction: 1,1,1-trichloroethane + 2 H+ + 2 e- = 1,1-dichloroethane + HCl." [UM-BBD_reactionID:r1007]
GO:0018700
name: 2-chloro-N-isopropylacetanilide reductive dehalogenase activity
def: "Catalysis of the reaction: 2-chloro-N-isopropylacetanilide + H+ + 2 e- = Cl- + N-isopropylacetanilide." [UM-BBD_reactionID:r0719]
GO:0018701
name: 2,5-dichlorohydroquinone reductive dehalogenase activity
def: "Catalysis of the reaction: organohalide + 2 H+ + 2 e- = R-H + HCl. Reactants include chlorohydroquinone (forms hydroquinone) and 2,5-dichlorohydroquinone (forms chlorohydroquinone)." [UM-BBD_enzymeID:e0366]
GO:0018702
name: 1,1-dichloro-2,2-bis(4-chlorophenyl)ethene dehalogenase activity
def: "Catalysis of the reaction: 1,1-dichloro-2,2-bis(4-chlorophenyl)ethene + H+ + 2 e- = Cl- + 1-chloro-2,2-bis(4-chlorophenyl)ethene. 1,1-dichloro-2,2-bis(4-chlorophenyl)ethene is also known as DDE; 1-chloro-2,2-bis(4-chlorophenyl)ethene is also known as DDMU." [UM-BBD_reactionID:r0440]
GO:0018703
name: 2,4-dichlorophenoxyacetate dehalogenase activity
def: "Catalysis of the reaction: 2,4-dichlorophenoxyacetic acid + H+ + 2 e- = Cl- + 4-chlorophenoxyacetate." [UM-BBD_reactionID:r0280]
GO:0018704
name: 5-chloro-2-hydroxymuconic semialdehyde dehalogenase activity
def: "Catalysis of the reaction: 5-chloro-2-hydroxymuconic semialdehyde + H+ + 2 e- = Cl- + 2-hydroxymuconic semialdehyde." [UM-BBD_enzymeID:e0237]
GO:0018705
name: 1,2-dichloroethene reductive dehalogenase activity
def: "Catalysis of the reaction: 1,2-dichloroethene + 2 H+ + 2 e- = HCl + vinyl chloride." [UM-BBD_enzymeID:e0272]
GO:0018706
name: pyrogallol hydroxytransferase activity
def: "Catalysis of the reaction: 1,2,3,5-tetrahydroxybenzene + 1,2,3-trihydroxybenzene = 1,3,5-trihydroxybenzene + 1,2,3,5-tetrahydroxybenzene." [EC:1.97.1.2]
GO:0018707
name: 1-phenanthrol methyltransferase activity
def: "Catalysis of the reaction: 1-phenanthrol + X-CH3 = X + 1-methoxyphenanthrene." [UM-BBD_reactionID:r0493]
GO:0018708
name: thiol S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a thiol = S-adenosyl-L-homocysteine + a thioether." [EC:2.1.1.9]
GO:0018710
name: acetone carboxylase activity
def: "Catalysis of the reaction: acetone + ATP + CO(2) + 2 H(2)O = acetoacetate + AMP + 4 H(+) + 2 phosphate." [EC:6.4.1.6, RHEA:18388]
GO:0018711
name: benzoyl acetate-CoA thiolase activity
def: "Catalysis of the reaction: benzoyl acetyl-CoA + CoA = acetyl-CoA + benzoyl-CoA." [UM-BBD_reactionID:r0243]
GO:0018712
name: 3-hydroxybutyryl-CoA thiolase activity
def: "Catalysis of the reaction: 3-hydroxy-5-oxohexanoyl-CoA + CoASH = 3-hydroxybutyryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0010]
GO:0018713
name: 3-ketopimelyl-CoA thiolase activity
def: "Catalysis of the reaction: 3-ketopimeloyl-CoA + CoA = glutaryl-CoA + acetyl-CoA." [UM-BBD_reactionID:r0197]
GO:0018715
name: 9-phenanthrol UDP-glucuronosyltransferase activity
def: "Catalysis of the reaction: 9-phenanthrol + UDP-glucuronate = 9-phenanthryl-beta-D-glucuronide + UDP." [UM-BBD_reactionID:r0567]
GO:0018716
name: 1-phenanthrol glycosyltransferase activity
def: "Catalysis of the reaction: 1-phenanthrol + glucose = 1-phenanthryl-beta-D-glucopyranoside + H2O." [UM-BBD_reactionID:r0525]
GO:0018717
name: 9-phenanthrol glycosyltransferase activity
def: "Catalysis of the reaction: 9-phenanthrol + glucose = 9-phenanthryl-beta-D-glucopyranoside + H2O." [UM-BBD_reactionID:r0511]
GO:0018718
name: 1,2-dihydroxy-phenanthrene glycosyltransferase activity
def: "Catalysis of the reaction: 1,2-dihydroxyphenanthrene + UDP-glucose = 2-hydroxy-1-phenanthryl-beta-D-glucopyranoside + UDP." [UM-BBD_reactionID:r0569]
GO:0018719
name: 6-aminohexanoate transaminase activity
def: "Catalysis of the reaction: 6-aminohexanoate + alpha-ketoglutarate = glutamate + 6-oxohexanoate." [UM-BBD_reactionID:r0449]
GO:0018720
name: phenol kinase activity
def: "Catalysis of the reaction: phenol + X-HPO3- = XH + phenylphosphate." [UM-BBD_reactionID:r0155]
GO:0018721
name: trans-9R,10R-dihydrodiolphenanthrene sulfotransferase activity
def: "Catalysis of the reaction: trans-9R,10R-dihydrodiolphenanthrene + 2 X-SO3(-) = 2 HX + phenanthrene-9,10-dihydrodiolsulfate conjugate." [UM-BBD_reactionID:r0559]
GO:0018722
name: 1-phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: 1-phenanthrol + X-SO3(-) = HX + 1-phenanthrylsulfate." [UM-BBD_reactionID:r0565]
GO:0018723
name: 3-phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: 3-phenanthrol + X-SO3(-) = HX + 3-phenanthrylsulfate." [UM-BBD_reactionID:r0561]
GO:0018724
name: 4-phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: 4-phenanthrol + X-SO3(-) = HX + 4-phenanthrylsulfate." [UM-BBD_reactionID:r0562]
GO:0018725
name: trans-3,4-dihydrodiolphenanthrene sulfotransferase activity
def: "Catalysis of the reaction: trans-3,4-dihydrodiolphenanthrene + 2 X-SO3(-) = 2 HX + phenanthrene-3,4-dihydrodiolsulfate conjugate." [UM-BBD_reactionID:r0558]
GO:0018726
name: 9-phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: 9-phenanthrol + X-SO3(-) = HX + 9-phenanthrylsulfate." [UM-BBD_reactionID:r0564]
GO:0018727
name: 2-phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: 2-phenanthrol + X-SO3(-) = HX + 2-phenanthrylsulfate." [UM-BBD_reactionID:r0563]
GO:0018729
name: propionate CoA-transferase activity
def: "Catalysis of the reaction: acetyl-CoA + propanoate = acetate + propanoyl-CoA." [EC:2.8.3.1]
GO:0018730
name: glutaconate CoA-transferase activity
def: "Catalysis of the reaction: acetyl-CoA + (E)-glutaconate = acetate + glutaconyl-1-CoA." [EC:2.8.3.12]
GO:0018731
name: 1-oxa-2-oxocycloheptane lactonase activity
def: "Catalysis of the reaction: 1-oxa-2-oxocycloheptane + H2O = 6-hydroxyhexanoate." [UM-BBD_reactionID:r0167]
GO:0018732
name: sulfolactone hydrolase activity
def: "Catalysis of the reaction: 4-sulfolactone + OH- = HSO3(-) + maleylacetate." [UM-BBD_reactionID:r0583]
GO:0018733
name: 3,4-dihydrocoumarin hydrolase activity
def: "Catalysis of the reaction: 3,4-dihydrocoumarin + H2O = 3-(2-hydroxyphenyl) propionate." [UM-BBD_reactionID:r0419]
GO:0018734
name: butyrolactone hydrolase activity
def: "Catalysis of the reaction: butyrolactone + H2O = 4-hydroxybutanoate." [UM-BBD_reactionID:r0016]
GO:0018736
name: 6-oxo-2-hydroxycyclohexane-1-carboxyl-CoA hydrolase activity
def: "Catalysis of the reaction: 6-oxo-2-hydroxycyclohexane-1-carboxyl-CoA + H2O = 3-hydroxypimeloyl-CoA." [UM-BBD_reactionID:r0206]
GO:0018737
name: 2-ketocyclohexane-1-carboxyl-CoA hydrolase activity
def: "Catalysis of the reaction: 2-ketocyclohexane-1-carboxyl-CoA + H2O = pimeloyl-CoA." [UM-BBD_reactionID:r0193]
GO:0018738
name: S-formylglutathione hydrolase activity
def: "Catalysis of the reaction: S-formylglutathione + H(2)O = formate + glutathione + H(+)." [EC:3.1.2.12, RHEA:14964]
GO:0018739
name: 4-hydroxybenzoyl-CoA thioesterase activity
def: "Catalysis of the reaction: 4-hydroxybenzoyl-CoA + H(2)O = 4-hydroxybenzoate + CoA + H(+)." [EC:3.1.2.23, RHEA:11951]
GO:0018740
name: 2'-hydroxybiphenyl-2-sulfinate desulfinase activity
def: "Catalysis of the reaction: 2'-hydroxybiphenyl-2-sulfinate + H(2)O = biphenyl-2-ol + sulfite." [EC:3.13.1.3, RHEA:12948]
GO:0018741
name: alkyl sulfatase activity
def: "Catalysis of the reaction: dodecyl sulfate + H2O = sulfate + H+ + 1-dodecanol." [UM-BBD_reactionID:r0602]
GO:0018742
name: epoxide hydrolase B activity
def: "Catalysis of the hydrolysis of the ether in chloro- or hydroxyepoxypropane to produce chloropropane diol or glycerol. Acts on R enantiomers." [UM-BBD_enzymeID:e0051]
GO:0018743
name: phenanthrene-9,10-epoxide hydrolase (9R,10R-forming) activity
def: "Catalysis of the reaction: phenanthrene-9,10-oxide + H2O = trans-9R,10R-dihydrodiolphenanthrene." [UM-BBD_reactionID:r0560]
GO:0018744
name: limonene-1,2-epoxide hydrolase activity
def: "Catalysis of the reaction: limonene-1,2-epoxide + H2O = limonene-1,2-diol. Other substrates include alicyclic and 1-methyl-substituted epoxides, such as 1-methylcyclohexene oxide, indene oxide and cyclohexene oxide." [EC:3.3.2.8]
GO:0018745
name: epoxide hydrolase A activity
def: "Catalysis of the hydrolysis of the ether in chloro-, bromo- or hydroxyepoxypropane to produce a chloro- or bromopropane diol or glycerol." [UM-BBD_enzymeID:e0049]
GO:0018746
name: phenanthrene-3,4-epoxide hydrolase activity
def: "Catalysis of the reaction: phenanthrene-3,4-oxide + H2O = trans-3,4-dihydrodiolphenanthrene." [UM-BBD_reactionID:r0535]
GO:0018747
name: phenanthrene-1,2-epoxide hydrolase activity
def: "Catalysis of the reaction: phenanthrene-1,2-oxide + H2O = trans-1,2-dihydrodiolphenanthrene." [UM-BBD_reactionID:r0536]
GO:0018748
name: iprodione amidohydrolase activity
def: "Catalysis of the reaction: iprodione + OH- = 3,5-dichlorophenylcarboximide + N-isopropylcarbamate." [UM-BBD_reactionID:r0706]
GO:0018749
name: (3,5-dichlorophenylurea)acetate amidohydrolase activity
def: "Catalysis of the reaction: (3,5-dichlorophenylurea)acetate + OH- = 3,5-dichloroaniline + N-carboxyglycine." [UM-BBD_reactionID:r0708]
GO:0018750
name: biuret amidohydrolase activity
def: "Catalysis of the reaction: biuret + H2O = urea + CO2 + NH3." [EC:3.5.1.84]
GO:0018751
name: 3,5-dichlorophenylcarboximide hydrolase activity
def: "Catalysis of the reaction: 3,5-dichlorophenylcarboximide + OH- = (3,5-dichlorophenylurea)acetate." [UM-BBD_reactionID:r0707]
GO:0018752
name: epsilon-caprolactam lactamase activity
def: "Catalysis of the reaction: epsilon-caprolactam + H2O = H+ + 6-aminohexanoate." [UM-BBD_reactionID:r0448]
GO:0018753
name: cyanuric acid amidohydrolase activity
def: "Catalysis of the reaction: cyanuric acid + H2O = biuret + CO2." [EC:3.5.2.15]
GO:0018754
name: ammelide aminohydrolase activity
def: "Catalysis of the reaction: ammelide + H2O = cyanuric acid + NH3." [PMID:1991731]
GO:0018755
name: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine aminohydrolase activity
def: "Catalysis of the reaction: 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + OH- = 2,4-dihydroxy-6-amino-1,3,5-triazine + Cl-." [UM-BBD_reactionID:r1414]
GO:0018756
name: ammeline aminohydrolase activity
def: "Catalysis of the reaction: ammeline + H2O = ammelide + NH3." [PMID:1991731]
GO:0018757
name: deisopropylhydroxyatrazine aminohydrolase activity
def: "Catalysis of the reaction: deisopropylhydroxyatrazine + H2O = NH3 + 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine." [UM-BBD_reactionID:r0121]
GO:0018758
name: 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine aminohydrolase activity
def: "Catalysis of the reaction: 2,4-dihydroxy-6-(N'-ethyl)amino-1,3,5-triazine + H2O = CH3CH2NH2 + cyanuric acid." [UM-BBD_reactionID:r0122]
GO:0018759
name: methenyltetrahydromethanopterin cyclohydrolase activity
def: "Catalysis of the reaction: 5,10-methenyl-5,6,7,8-tetrahydromethanopterin + H(2)O = N(5)-formyl-5,6,7,8-tetrahydromethanopterin + H(+)." [EC:3.5.4.27, RHEA:19056]
GO:0018760
name: thiocyanate hydrolase activity
def: "Catalysis of the reaction: H(2)O + 2 H(+) + thiocyanate = carbonyl sulfide + NH(4)(+)." [EC:3.5.5.8, RHEA:21467]
GO:0018761
name: bromoxynil nitrilase activity
def: "Catalysis of the reaction: 3,5-dibromo-4-hydroxybenzonitrile + 2 H(2)O = 3,5-dibromo-4-hydroxybenzoate + NH(4)(+). Involved in the bacterial degradation of the herbicide bromoxynil." [EC:3.5.5.6, RHEA:22103]
GO:0018762
name: aliphatic nitrilase activity
def: "Catalysis of the reaction: R-CN + H2O = R-COOH + NH3." [EC:3.5.5.7]
GO:0018763
name: hydroxydechloroatrazine ethylaminohydrolase activity
def: "Catalysis of the reaction: 4-(ethylamino)-2-hydroxy-6-(isopropylamino)-1,3,5-triazine + H2O = N-isopropylammelide + ethylamine." [EC:3.5.99.3]
GO:0018764
name: N-isopropylammelide isopropylaminohydrolase activity
def: "Catalysis of the reaction: N-isopropylammelide + H2O = cyanuric acid + isopropylamine." [EC:3.5.99.4]
GO:0018765
name: 2-hydroxy-6-oxohepta-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: cis,cis-2-hydroxy-6-oxohept-2,4-dienoate + OH- = cis-2-hydroxypenta-2,4-dienoate + acetate." [UM-BBD_reactionID:r0263]
GO:0018766
name: dihydrophloroglucinol hydrolase activity
def: "Catalysis of the reaction: dihydrophloroglucinol + OH- = 3-hydroxy-5-oxohexanoate." [UM-BBD_reactionID:r0008]
GO:0018767
name: 2-hydroxy-6-oxo-7-methylocta-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: 2-hydroxy-6-oxo-7-methylocta-2,4-dienoate + H2O = cis-2-hydroxypenta-2,4-dienoate + isobutyrate." [UM-BBD_reactionID:r0399]
GO:0018768
name: 2-hydroxy-6-oxo-6-(2'-aminophenyl)hexa-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: 2-hydroxy-6-oxo-(2'-aminophenyl)-hexa-2,4-dienoate + H2O = 2-aminobenzoate + cis-2-hydroxypenta-2,4-dienoate." [UM-BBD_reactionID:r0458]
GO:0018769
name: 2-hydroxy-6-oxoocta-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: 2-hydroxy-6-oxoocta-2,4-dienoate + H2O = H+ + propanoate + cis-2-hydroxypenta-2,4-dienoate." [UM-BBD_reactionID:r0311]
GO:0018770
name: 6-oxo-2-hydroxy-7-(4'-chlorophenyl)-3,8,8-trichloroocta-2E,4E,7-trienoate hydrolase activity
def: "Catalysis of the reaction: 6-oxo-2-hydroxy-7-(4'-chlorophenyl)-3,8,8-trichloroocta-2Z,4Z,7-trienoate + H2O = 2-(4'-chlorophenyl)-3,3-dichloropropenoate + cis-2-Hydroxy-3-chloropenta-2,4-dienone + H+." [UM-BBD_reactionID:r0444]
GO:0018771
name: 2-hydroxy-6-oxonona-2,4-dienedioate hydrolase activity
def: "Catalysis of the reaction: (2E,4Z)-2-hydroxy-6-oxonona-2,4-dienedioate + H2O = (2E)-2-hydroxypenta-2,4-dienoate + H+ + succinate." [RHEA:24792]
GO:0018772
name: trioxoheptanoate hydrolase activity
def: "Catalysis of the reaction: 2,4,6-trioxoheptanoate + H2O = acetylpyruvate + acetate." [MetaCyc:R306-RXN, UM-BBD_reactionID:r0094]
GO:0018773
name: acetylpyruvate hydrolase activity
def: "Catalysis of the reaction: acetylpyruvate + H(2)O = acetate + H(+) + pyruvate." [EC:3.7.1.6, RHEA:16100]
GO:0018774
name: 2,6-dioxo-6-phenylhexa-3-enoate hydrolase activity
def: "Catalysis of the reaction: 2,6-dioxo-6-phenylhexa-3-enoate + H2O = benzoate + 2-oxopent-4-enoate." [EC:3.7.1.8]
GO:0018775
name: 2-hydroxymuconate-semialdehyde hydrolase activity
def: "Catalysis of the reaction: 2-hydroxymuconate semialdehyde + H2O = formate + 2-oxopent-4-enoate." [EC:3.7.1.9]
GO:0018776
name: trans-chloroacrylic acid dehalogenase activity
def: "Catalysis of the reaction: trans-3-chloroacrylic acid + H2O = H+ + malonate semialdehyde." [UM-BBD_reactionID:r0689]
GO:0018777
name: 1,3,4,6-tetrachloro-1,4-cyclohexadiene halidohydrolase activity
def: "Catalysis of the reaction: alkyl halide + H2O = alcohol + HCl. Substrates are 1,3(R),4,6(R)-tetrachloro-1,4-cyclohexadiene (forms 2,4,5-trichloro-2,5-cyclohexadiene-1-ol) and 2,4,5-trichloro-2,5-cyclohexadiene-1-ol (forms 2,5-dichloro-2,5-cyclohexadiene-1,4-diol)." [PMID:10464214]
GO:0018778
name: DL-2 haloacid dehalogenase activity
def: "Catalysis of the reaction: trichloroacetate + 2 H2O = 3 H+ + 3 Cl- + oxalate." [UM-BBD_reactionID:r0382]
GO:0018779
name: 2-chloro-4,6-dihydroxy-1,3,5-triazine hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0018780
name: dichloroacetate halidohydrolase activity
def: "Catalysis of the reaction: dichloroacetate + H2O = 2 HCl + glyoxylate." [UM-BBD_reactionID:r0383]
GO:0018781
name: S-triazine hydrolase activity
def: "Catalysis of the reaction: 1-halo or 1-pseudohalo-S-triazine = 1-hydroxy-S-triazine." [UM-BBD_ruleID:bt0330]
GO:0018782
name: cis-chloroacrylic acid dehalogenase activity
def: "Catalysis of the reaction: cis-3-chloroacrylic acid + H2O = H+ + HCl + malonate semialdehyde." [UM-BBD_reactionID:r0688]
GO:0018783
name: deisopropyldeethylatrazine hydrolase activity
def: "Catalysis of the reaction: deisopropyldeethylatrazine + H2O = 2-chloro-4-hydroxy-6-amino-1,3,5-triazine + NH3." [UM-BBD_reactionID:0129]
GO:0018784
name: (S)-2-haloacid dehalogenase activity
def: "Catalysis of the reaction: (S)-2-haloacid + H2O = (R)-2-hydroxyacid + halide." [EC:3.8.1.2]
GO:0018785
name: haloacetate dehalogenase activity
def: "Catalysis of the reaction: haloacetate + H2O = glycolate + halide." [EC:3.8.1.3]
GO:0018786
name: haloalkane dehalogenase activity
def: "Catalysis of the reaction: 1-haloalkane + H2O = a primary alcohol + halide." [EC:3.8.1.5]
GO:0018787
name: 4-chlorobenzoyl-CoA dehalogenase activity
def: "Catalysis of the reaction: 4-chlorobenzoyl-CoA + H2O = 4-hydroxybenzoyl CoA + chloride." [EC:3.8.1.7]
GO:0018788
name: atrazine chlorohydrolase activity
def: "Catalysis of the reaction: atrazine + H(2)O = 4-(ethylamino)-2-hydroxy-6-(isopropylamino)-1,3,5-triazine + chloride + H(+)." [EC:3.8.1.8, RHEA:11315]
GO:0018789
name: cyclamate sulfohydrolase activity
def: "Catalysis of the reaction: cyclohexylsulfamate + H(2)O = cyclohexylamine + sulfate." [EC:3.10.1.2, RHEA:18484]
GO:0018791
name: 2-hydroxy-3-carboxy-6-oxo-7-methylocta-2,4-dienoate decarboxylase activity
def: "Catalysis of the reaction: 2-hydroxy-3-carboxy-6-oxo-7-methylocta-2,4-dienoate = CO2 + 2-hydroxy-6-oxo-7-methylocta-2,4-dienoate." [UM-BBD_reactionID:r0398]
GO:0018792
name: bis(4-chlorophenyl)acetate decarboxylase activity
def: "Catalysis of the reaction: bis(4-chlorophenyl)acetate + H+ = CO2 + bis(4-chlorophenyl)methane. Bis(4-chlorophenyl)acetate is also known as DDA; bis(4-chlorophenyl)methane is also known as DDM." [UM-BBD_reactionID:r0520]
GO:0018793
name: 3,5-dibromo-4-hydroxybenzoate decarboxylase activity
def: "Catalysis of the reaction: 3,5-dibromo-4-hydroxybenzoate + H+ = CO2 + 2,6-dibromophenol." [UM-BBD_reactionID:r0546]
GO:0018794
name: 2-hydroxyisobutyrate decarboxylase activity
def: "Catalysis of the reaction: 2-hydroxyisobutyrate + H+ = CO2 + 2-propanol." [UM-BBD_reactionID:r0617]
GO:0018795
name: 2-hydroxy-2-methyl-1,3-dicarbonate decarboxylase activity
def: "Catalysis of the reaction: 2-hydroxy-2-methyl-1,3-dicarbonate + H+ = CO2 + L-lactate." [UM-BBD_reactionID:r0621]
GO:0018796
name: 4,5-dihydroxyphthalate decarboxylase activity
def: "Catalysis of the reaction: 4,5-dihydroxyphthalate = 3,4-dihydroxybenzoate + CO2." [EC:4.1.1.55]
GO:0018798
name: gallate decarboxylase activity
def: "Catalysis of the reaction: gallate + H(+) = CO(2) + pyrogallol." [EC:4.1.1.59, RHEA:12752]
GO:0018799
name: 4-hydroxybenzoate decarboxylase activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + H(+) = CO(2) + phenol." [EC:4.1.1.61, RHEA:10879]
GO:0018800
name: 5-oxopent-3-ene-1,2,5-tricarboxylate decarboxylase activity
def: "Catalysis of the reaction: 5-oxopent-3-ene-1,2,5-tricarboxylate = 2-oxohept-3-enedioate + CO2." [EC:4.1.1.68]
GO:0018801
name: glutaconyl-CoA decarboxylase activity
def: "Catalysis of the reaction: trans-glutaconyl-CoA + H(+) = but-2-enoyl-CoA + CO(2)." [EC:4.1.1.70, RHEA:23975]
GO:0018802
name: 2,4-dihydroxyhept-2-ene-1,7-dioate aldolase activity
def: "Catalysis of the reaction: 2,4-dihydroxy-hept-trans-2-ene-1,7-dioate = pyruvate + succinic semialdehyde." [UM-BBD_reactionID:r0370]
GO:0018803
name: 4-(2-carboxyphenyl)-2-oxobut-3-enoate aldolase activity
def: "Catalysis of the reaction: (3E)-4-(2-carboxyphenyl)-2-oxobut-3-enoate = 2-carboxybenzaldehyde + pyruvate." [EC:4.1.2.34]
GO:0018805
name: benzylsuccinate synthase activity
def: "Catalysis of the reaction: fumarate + toluene = 2-benzylsuccinate." [EC:4.1.99.11, RHEA:10419]
GO:0018807
name: 6-hydroxycyclohex-1-ene-1-carboxyl-CoA hydratase activity
def: "Catalysis of the reaction: 6-hydroxycyclohex-1-ene-1-carboxyl-CoA + H2O = 2,6-dihydroxycyclohexane-1-carboxyl-CoA." [UM-BBD_reactionID:r0204]
GO:0018808
name: trans-4-(1'-hydroxynaphth-2'-yl)-2-oxobut-3-enoate hydratase-aldolase activity
def: "Catalysis of the reaction: trans-4-(1'-hydroxynaphth-2'-yl)-2-oxobut-3-enoate + H2O = pyruvate + 1-hydroxy-2-naphthaldehyde." [UM-BBD_reactionID:r0484]
GO:0018809
name: E-phenylitaconyl-CoA hydratase activity
def: "Catalysis of the reaction: E-phenylitaconyl-CoA + H2O = (hydroxymethylphenyl)succinyl-CoA." [UM-BBD_reactionID:r0331]
GO:0018810
name: trans-4-[2-(3-hydroxy)-thionaphthenyl]-2-oxo-3-butenoate hydratase activity
def: "Catalysis of the reaction: trans-4-(2-(3-hydroxy)-thionaphthenyl)-2-oxo-3-butenoate + H2O = pyruvate + 3-hydroxy-2-formylbenzothiophene." [UM-BBD_reactionID:r0164]
GO:0018811
name: cyclohex-1-ene-1-carboxyl-CoA hydratase activity
def: "Catalysis of the reaction: cyclohex-1-ene-1-carboxyl-CoA + H2O = 2-hydroxycyclohexane-1-carboxyl-CoA." [MetaCyc:R266-RXN, UM-BBD_reactionID:r0191]
GO:0018812
name: 3-hydroxyacyl-CoA dehydratase activity
def: "Catalysis of the reaction: alkene-CoA + H2O = alcohol-CoA. Substrates are crotonoyl-CoA (producing 3-hydroxyacyl-CoA) and 2,3-didehydro-pimeloyl-CoA (producing 3-hydroxypimeloyl-CoA)." [UM-BBD_ruleID:bt0291]
GO:0018813
name: trans-o-hydroxybenzylidenepyruvate hydratase-aldolase activity
def: "Catalysis of the reaction: 2-oxo-3-enoate-4-benzenoid + H2O = pyruvate + benzaldehyde derivative. Substrates are (3E)-4-(5-amino-2-hydroxy-phenyl)-2-oxo-but-3-ene-1-oic-acid (forms 5-aminosalicylaldehyde) and trans-o-hydroxybenzylidenepyruvate (forms salicylaldehyde)." [UM-BBD_enzymeID:e0257]
GO:0018814
name: phenylacetaldoxime dehydratase activity
def: "Catalysis of the reaction: (trans)-phenylacetaldoxime = H(2)O + phenylacetonitrile." [EC:4.99.1.7, RHEA:20072]
GO:0018815
name: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon hydratase-aldolase activity
def: "Catalysis of the reaction: 3-methyl-5-hydroxy-6-(3-carboxy-3-oxopropenyl)-1H-2-pyridon + H2O = 2-oxobut-3-enanoate + 2,5,6-trihydroxy-3-methylpyridine." [MetaCyc:RXN-645, UM-BBD_reactionID:r0051]
GO:0018816
name: 2-hydroxyisobutyrate dehydratase activity
def: "Catalysis of the reaction: 2-hydroxyisobutyrate = H2O + methacrylate." [UM-BBD_reactionID:r0618]
GO:0018817
name: 2-oxo-hept-3-ene-1,7-dioate hydratase activity
def: "Catalysis of the reaction: cis-2-oxohept-3-ene-1,7-dioate + H2O = 2,4-dihydroxy-hept-trans-2-ene-1,7-dioate." [UM-BBD_reactionID:r0369]
GO:0018818
name: acetylene hydratase activity
def: "Catalysis of the reaction: acetaldehyde = acetylene + H(2)O." [EC:4.2.1.112, RHEA:17888]
GO:0018819
name: lactoyl-CoA dehydratase activity
def: "Catalysis of the reaction: lactoyl-CoA = acryloyl-CoA + H2O." [EC:4.2.1.54]
GO:0018820
name: cyanamide hydratase activity
def: "Catalysis of the reaction: urea = cyanamide + H(2)O." [EC:4.2.1.69, RHEA:23059]
GO:0018822
name: nitrile hydratase activity
def: "Catalysis of the reaction: an aliphatic amide = a nitrile + H2O." [EC:4.2.1.84]
GO:0018823
name: cyclohexa-1,5-dienecarbonyl-CoA hydratase activity
def: "Catalysis of the reaction: cyclohexa-1,5-diene-1-carbonyl-CoA + H(2)O = 6-hydroxycyclohex-1-enecarbonyl-CoA." [EC:4.2.1.100, RHEA:21859]
GO:0018824
name: (hydroxyamino)benzene mutase activity
def: "Catalysis of the reaction: (hydroxyamino)benzene = 2-aminophenol." [EC:5.4.4.1, UM-BBD_reactionID:r0304]
GO:0018825
name: triethanolamine lyase activity
def: "Catalysis of the reaction: triethanolamine = diethanolamine + acetaldehyde." [UM-BBD_enzymeID:e0421]
GO:0018826
name: methionine gamma-lyase activity
def: "Catalysis of the reaction: L-methionine = methanethiol + NH3 + 2-oxobutanoate." [EC:4.4.1.11]
GO:0018827
name: 1-chloro-2,2-bis(4-chlorophenyl)ethane dehydrochlorinase activity
def: "Catalysis of the reaction: 1-chloro-2,2-bis(4-chlorophenyl)ethane = HCl + unsym-bis(4-chlorophenyl)ethene. 1-chloro-2,2-bis(4-chlorophenyl)ethane is also known as DDMS; unsym-bis(4-chlorophenyl)ethene is also known as DDNU." [UM-BBD_reactionID:r0515]
GO:0018828
name: halohydrin hydrogen-halide-lyase A activity
def: "Catalysis of the elimination of HCl from a chloro- or bromopropanol, yielding an epoxypropane." [UM-BBD_enzymeID:e0048]
GO:0018829
name: 1,1-dichloro-2,2-bis(4-chlorophenyl)ethane dehydrochlorinase activity
def: "Catalysis of the reaction: 1,1-dichloro-2,2-bis(4-chlorophenyl)ethane = HCl + 1-chloro-2,2-bis(4-chlorophenyl)ethene. 1,1-dichloro-2,2-bis(4-chlorophenyl)ethane is also known as DDD; 1-chloro-2,2-bis(4-chlorophenyl)ethene is also known as DDMU." [UM-BBD_reactionID:r0513]
GO:0018830
name: gamma-hexachlorocyclohexane dehydrochlorinase activity
def: "Catalysis of the reaction: gamma-hexachlorocyclohexane + HCl = 1,3(R),4(S),5(S),6(R)-pentachlorocyclohexene." [UM-BBD_enzymeID:e0359]
GO:0018831
name: 5-chloro-1,2,4-trihydroxybenzene dechlorinase activity
def: "Catalysis of the reaction: 5-chloro-1,2,4-trihydroxybenzene = Cl- + H+ + 2-hydroxy-1,4-benzoquinone." [UM-BBD_reactionID:r0666]
GO:0018832
name: halohydrin hydrogen-halide-lyase B activity
def: "Catalysis of the elimination of HCl from a chloropropanol, yielding an epoxypropane." [UM-BBD_enzymeID:e0050]
GO:0018833
name: DDT-dehydrochlorinase activity
def: "Catalysis of the reaction: 1,1,1-trichloro-2,2-bis(4-chlorophenyl)ethane = 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene + chloride + H(+)." [EC:4.5.1.1, RHEA:19220]
GO:0018834
name: dichloromethane dehalogenase activity
def: "Catalysis of the reaction: dichloromethane + H(2)O = 2 chloride + formaldehyde + 2 H(+)." [EC:4.5.1.3, RHEA:15400]
GO:0018835
name: carbon phosphorus lyase activity
def: "Catalysis of the reaction: alkylphosphonic acid = R-CH3 + phosphate. Substrates include aminomethylphosphonic acid (AMPA) (forms methylamine), dimethylphosphinic acid (forms methylphosphonic acid), glyphosate (forms sarcosine) and methylphosphonic acid (forms phosphate)." [UM-BBD_enzymeID:e0057]
GO:0018836
name: alkylmercury lyase activity
def: "Catalysis of the reaction: R-Hg+ + H+ = R-H + Hg2+." [EC:4.99.1.2]
GO:0018837
name: 2-hydroxy-2H-benzo[h]chromene-2-carboxylate isomerase activity
def: "Catalysis of the reaction: 2-hydroxy-2 H-benzo[h]chromene-2-carboxylate = cis -4-(1'-hydroxynaphth-2'-yl)-2-oxobut-3-enoate." [UM-BBD_reactionID:r0502]
GO:0018838
name: mandelate racemase activity
def: "Catalysis of the reaction: (S)-mandelate = (R)-mandelate." [EC:5.1.2.2]
GO:0018839
name: cis-4-[2-(3-hydroxy)-thionaphthenyl]-2-oxo-3-butenoate isomerase activity
def: "Catalysis of the reaction: cis-4-(2-(3-hydroxy)-thionaphthenyl)-2-oxo-3-butenoate = trans-4-(2-(3-hydroxy)-thionaphthenyl)-2-oxo-3-butenoate." [UM-BBD_reactionID:r0163]
GO:0018842
name: 3-carboxymuconate cycloisomerase type II activity
def: "Catalysis of the reaction: 3-sulfomuconate = 4-sulfolactone." [UM-BBD_reactionID:r0582]
GO:0018844
name: 2-hydroxytetrahydrofuran isomerase activity
def: "Catalysis of the reaction: 2-hydroxytetrahydrofuran = 4-hydroxybutyraldehyde." [UM-BBD_reactionID:r0019]
GO:0018845
name: 2-hydroxychromene-2-carboxylate isomerase activity
def: "Catalysis of the reaction: 2-hydroxychromene-2-carboxylate = (3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate. (3E)-4-(2-hydroxyphenyl)-2-oxobut-3-enoate is also known as trans-o-hydroxybenzylidenepyruvate." [RHEA:27404]
GO:0018846
name: styrene-oxide isomerase activity
def: "Catalysis of the reaction: styrene oxide = phenylacetaldehyde." [EC:5.3.99.7, RHEA:21607]
GO:0018847
name: alpha-pinene lyase activity
def: "Catalysis of the reaction: alpha-pinene = limonene." [UM-BBD_reactionID:r0712]
GO:0018848
name: pinocarveol isomerase activity
def: "Catalysis of the reaction: pinocarveol = carveol." [UM-BBD_reactionID:r0715]
GO:0018849
name: muconate cycloisomerase activity
def: "Catalysis of the reaction: 2,5-dihydro-5-oxofuran-2-acetate = cis,cis-hexadienedioate." [EC:5.5.1.1]
GO:0018850
name: chloromuconate cycloisomerase activity
def: "Catalysis of the reaction: 2-chloro-2,5-dihydro-5-oxofuran-2-acetate = 3-chloro-cis,cis-muconate." [EC:5.5.1.7]
GO:0018851
name: alpha-pinene-oxide decyclase activity
def: "Catalysis of the reaction: alpha-pinene oxide = (Z)-2-methyl-5-isopropylhexa-2,5-dienal." [EC:5.5.1.10, RHEA:16696]
GO:0018852
name: dichloromuconate cycloisomerase activity
def: "Catalysis of the reaction: 2,4-dichloro-2,5-dihydro-5-oxofuran-2-acetate = 2,4-dichloro-cis,cis-muconate." [EC:5.5.1.11]
GO:0018853
name: perillyl-CoA synthetase activity
def: "OBSOLETE. Catalysis of the reaction: perillic acid + CoA-SH + ATP = H2O + ADP/AMP + mono/diphosphate + perillyl-CoA." [UM-BBD_reactionID:r0731]
GO:0018854
name: 3-isopropenyl-6-oxoheptanoyl-CoA synthetase activity
def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP/AMP + mono/diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [UM-BBD_reactionID:r0737]
GO:0018855
name: 2-oxo-delta3-4,5,5-trimethylcyclopentenylacetyl-CoA synthetase activity
def: "Catalysis of the reaction: 2-oxo-delta3-4,5,5-trimethylcyclopentenylacetate + ATP + CoA = AMP + diphosphate + 2-oxo-delta3-4,5,5-trimethylcyclopentenylacetyl-CoA." [UM-BBD_reactionID:r0429]
GO:0018856
name: benzoyl acetate-CoA ligase activity
def: "Catalysis of the reaction: 3-oxo-3-phenylpropionate + CoA + ATP = AMP + diphosphate + benzoyl acetyl-CoA." [UM-BBD_reactionID:r0242]
GO:0018857
name: 2,4-dichlorobenzoate-CoA ligase activity
def: "Catalysis of the reaction: 2,4-dichlorobenzoate + ATP + CoASH = AMP + diphosphate + 2,4-dichlorobenzoyl-CoA." [UM-BBD_reactionID:r0137]
GO:0018858
name: benzoate-CoA ligase activity
def: "Catalysis of the reaction: ATP + benzoate + CoA = AMP + benzoyl-CoA + diphosphate + H(+)." [EC:6.2.1.25, RHEA:10135]
GO:0018859
name: 4-hydroxybenzoate-CoA ligase activity
def: "Catalysis of the reaction: ATP + 4-hydroxybenzoate + CoA = AMP + diphosphate + 4-hydroxybenzoyl-CoA." [EC:6.2.1.27]
GO:0018860
name: anthranilate-CoA ligase activity
def: "Catalysis of the reaction: ATP + anthranilate + CoA = AMP + diphosphate + anthranilyl-CoA." [EC:6.2.1.32]
GO:0018861
name: 4-chlorobenzoate-CoA ligase activity
def: "Catalysis of the reaction: 4-chlorobenzoate + CoA + ATP = 4-chlorobenzoyl-CoA + AMP + diphosphate. This reaction requires magnesium and is part of the bacterial 2,4-dichlorobenzoate degradation pathway." [EC:6.2.1.33]
GO:0018862
name: phenylphosphate carboxylase activity
def: "Catalysis of the reaction: phenylphosphate + CO2 + H2O = H+ + phosphate + 4-hydroxybenzoate." [UM-BBD_reactionID:r0157]
GO:0018863
name: phenanthrene-9,10-epoxide hydrolase (9S,10S-forming) activity
def: "Catalysis of the reaction: phenanthrene-9,10-oxide + H2O = trans-9(S),10(S)-dihydrodiolphenanthrene." [UM-BBD_reactionID:r0496]
GO:0018997
name: electron transfer carrier
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0019000
name: endonuclease G activity
def: "OBSOLETE (was not defined before being made obsolete)." [PMID:12444964, PMID:12928502, PMID:12941691]
GO:0019001
name: guanyl nucleotide binding
def: "Interacting selectively and non-covalently with guanyl nucleotides, any compound consisting of guanosine esterified with (ortho)phosphate." [ISBN:0198506732]
GO:0019002
name: GMP binding
def: "Interacting selectively and non-covalently with GMP, guanosine monophosphate." [GOC:ai]
GO:0019003
name: GDP binding
def: "Interacting selectively and non-covalently with GDP, guanosine 5'-diphosphate." [GOC:ai]
GO:0019010
name: farnesoic acid O-methyltransferase activity
def: "Catalysis of the reaction: farnesoic acid + S-adenosyl-methionine = methyl farnesoate + S-adenosyl-L-homocysteine." [PMID:12135499]
GO:0019011
name: DNA replication accessory factor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0019039
name: viral-cell fusion molecule activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0019040
name: viral host shutoff protein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0019103
name: pyrimidine nucleotide binding
def: "Interacting selectively and non-covalently with pyrimidine nucleotide, any compound consisting of a pyrimidine nucleoside esterified with (ortho)phosphate." [GOC:ai]
GO:0019104
name: DNA N-glycosylase activity
def: "Catalysis of the removal of damaged bases by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic/apyrimidinic (AP) site." [GOC:elh, PMID:11554296]
GO:0019105
name: N-palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl group to a nitrogen atom on the acceptor molecule." [GOC:ai]
GO:0019107
name: myristoyltransferase activity
def: "Catalysis of the transfer of a myristoyl (CH3-[CH2]12-CO-) group to an acceptor molecule." [GOC:ai]
GO:0019108
name: aryl-aldehyde dehydrogenase (NAD) activity
def: "Catalysis of the reaction: an aromatic aldehyde + NAD+ + H2O = an aromatic acid + NADH + H+." [EC:1.2.1.29]
GO:0019111
name: phenanthrol sulfotransferase activity
def: "Catalysis of the reaction: phenanthrol + X-SO3(-) = HX + phenanthrylsulfate." [EC:2.8.2.-]
GO:0019112
name: phenanthrol glycosyltransferase activity
def: "Catalysis of the reaction: phenanthrol + glucose = phenanthryl-beta-D-glucopyranoside + H2O." [GOC:ai]
GO:0019113
name: limonene monooxygenase activity
def: "Catalysis of a monooxygenase reaction in which oxygen is incorporated into limonene." [GOC:mah]
GO:0019114
name: catechol dioxygenase activity
def: "Catalysis of the reaction: catechol + O2 = a muconate." [GOC:mah, MetaCyc:CATECHOL-12-DIOXYGENASE-RXN, MetaCyc:CATECHOL-23-DIOXYGENASE-RXN]
GO:0019115
name: benzaldehyde dehydrogenase activity
def: "Catalysis of the reaction: benzaldehyde + NAD(P)+ + H2O = benzoate + NAD(P)H + H+." [EC:1.2.1.28, EC:1.2.1.7]
GO:0019116
name: hydroxy-nicotine oxidase activity
def: "Catalysis of the reaction: 6-hydroxynicotine + H2O + O2 = 1-(6-hydroxypyrid-3-yl)-4-(methylamino)butan-1-one + hydrogen peroxide." [GOC:jl, http://umbbd.ahc.umn.edu/]
GO:0019117
name: dihydroxyfluorene dioxygenase activity
def: "Catalysis of the reaction: a dihydroxyfluorene + O2 = the corresponding 2-hydroxy-4-(oxo-1,3-dihydro-2H-inden-ylidene) but-2-enoic acid." [GOC:mah, UM-BBD_reactionID:r0415, UM-BBD_reactionID:r0422]
GO:0019118
name: phenanthrene-epoxide hydrolase activity
def: "Catalysis of the reaction: a phenanthrene dioxide + H2O = a dihydrodiolphenanthrene." [GOC:mah, UM-BBD_reactionID:r0535, UM-BBD_reactionID:r0536]
GO:0019119
name: phenanthrene-9,10-epoxide hydrolase activity
def: "Catalysis of the reaction: phenanthrene-9,10-oxide + H2O = trans-9,10-dihydrodiolphenanthrene." [GOC:mah, UM-BBD_reactionID:r0496, UM-BBD_reactionID:r0560]
GO:0019120
name: hydrolase activity, acting on acid halide bonds, in C-halide compounds
def: "Catalysis of the hydrolysis of any acid halide bond in substances containing halogen atoms in organic linkage." [ISBN:0198506732]
GO:0019131
name: tripeptidyl-peptidase I activity
def: "OBSOLETE. Catalysis of the release of an N-terminal tripeptide from a polypeptide; requires acid pH." [EC:3.4.14.9]
GO:0019132
name: C-terminal processing peptidase activity
def: "OBSOLETE. Catalysis of the cleavage of peptides after specific recognition of a C-terminal tripeptide, Xaa-Yaa-Zaa, in which Xaa is preferably Ala or Leu, Yaa is preferably Ala or Tyr, and Zaa is preferably Ala. Cleavage is at a variable distance from the C-terminus; a typical cleavage is -Ala-Ala-Arg-Ala-Ala-Lys-Glu-Asn-Tyr-Ala-Leu-Ala-Ala." [EC:3.4.21.102]
GO:0019133
name: choline monooxygenase activity
def: "Catalysis of the reaction: choline + 2 reduced ferredoxin + O2 + 2 H+ = betaine aldehyde hydrate + 2 oxidized ferredoxin + H2O." [EC:1.14.15.7]
GO:0019134
name: glucosamine-1-phosphate N-acetyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucosamine 1-phosphate + acetyl-CoA = N-acetyl-alpha-D-glucosamine 1-phosphate + CoA + H(+)." [EC:2.3.1.157, RHEA:13728]
GO:0019135
name: deoxyhypusine monooxygenase activity
def: "Catalysis of the reaction: protein N6-(4-aminobutyl)-L-lysine + donor-H2 + O2 = protein N6-((R)-4-amino-2-hydroxybutyl)-L-lysine + acceptor + H2O." [EC:1.14.99.29]
GO:0019136
name: deoxynucleoside kinase activity
def: "Catalysis of the reaction: ATP + 2'-deoxynucleoside = ADP + 2'-deoxynucleoside 5'-phosphate." [EC:2.7.1.145]
GO:0019137
name: thioglucosidase activity
def: "Catalysis of the reaction: a thioglucoside + H2O = a thiol + a sugar." [EC:3.2.1.147]
GO:0019139
name: cytokinin dehydrogenase activity
def: "Catalysis of the reaction: N6-dimethylallyladenine + acceptor + H2O = adenine + 3-methylbut-2-enal + reduced electron acceptor." [EC:1.5.99.12]
GO:0019140
name: inositol 3-kinase activity
def: "Catalysis of the reaction: ATP + myo-inositol = ADP + 1D-myo-inositol 3-phosphate." [EC:2.7.1.64]
GO:0019141
name: 2-dehydropantolactone reductase (B-specific) activity
def: "Catalysis of the reaction: (R)-pantolactone + NADP+ = 2-dehydropantolactone + NADPH + H+. The reaction is B-specific (i.e. the pro-S hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to NADP+." [EC:1.1.1.214]
GO:0019142
name: 2-hydroxyglutarate synthase activity
def: "Catalysis of the reaction: glyoxylate + H(2)O + propanoyl-CoA = 2-hydroxyglutarate + CoA + H(+)." [EC:2.3.3.11, RHEA:19188]
GO:0019143
name: 3-deoxy-manno-octulosonate-8-phosphatase activity
def: "Catalysis of the reaction: 8-phospho-3-deoxy-D-manno-oct-2-ulosonate + H(2)O = 3-deoxy-D-manno-octulosonate + phosphate." [EC:3.1.3.45, RHEA:11503]
GO:0019144
name: ADP-sugar diphosphatase activity
def: "Catalysis of the reaction: ADP-sugar + H2O = AMP + sugar 1-phosphate." [EC:3.6.1.21]
GO:0019145
name: aminobutyraldehyde dehydrogenase activity
def: "Catalysis of the reaction: 4-aminobutanal + NAD+ + H2O = 4-aminobutanoate + NADH + 2 H+." [EC:1.2.1.19]
GO:0019146
name: arabinose-5-phosphate isomerase activity
def: "Catalysis of the reaction: D-arabinose 5-phosphate = D-ribulose 5-phosphate + 2 H(+)." [EC:5.3.1.13, RHEA:23107]
GO:0019147
name: (R)-aminopropanol dehydrogenase activity
def: "Catalysis of the reaction: (R)-1-aminopropan-2-ol + NAD(+) = aminoacetone + H(+) + NADH." [EC:1.1.1.75, RHEA:16520]
GO:0019148
name: D-cysteine desulfhydrase activity
def: "Catalysis of the reaction: D-cysteine + H2O = sulfide + NH3 + pyruvate." [EC:4.4.1.15]
GO:0019149
name: 3-chloro-D-alanine dehydrochlorinase activity
def: "Catalysis of the reaction: 3-chloro-D-alanine + H2O = pyruvate + chloride + NH3." [EC:4.5.1.2]
GO:0019150
name: D-ribulokinase activity
def: "Catalysis of the reaction: ATP + D-ribulose = ADP + D-ribulose 5-phosphate." [EC:2.7.1.47]
GO:0019151
name: galactose 1-dehydrogenase activity
def: "Catalysis of the reaction: D-galactose + NAD+ = D-galactono-1,4-lactone + NADH + H+." [EC:1.1.1.48]
GO:0019152
name: acetoin dehydrogenase activity
def: "Catalysis of the reaction: acetoin + NAD+ = diacetyl + NADH + H+." [MetaCyc:ACETOINDEHYDROG-RXN]
GO:0019153
name: protein-disulfide reductase (glutathione) activity
def: "Catalysis of the reaction: 2 glutathione + protein-disulfide = oxidized glutathione + protein-dithiol." [EC:1.8.4.2]
GO:0019154
name: glycolate dehydrogenase activity
def: "Catalysis of the reaction: A + glycolate = AH(2) + glyoxylate." [EC:1.1.99.14, RHEA:21267]
GO:0019155
name: 3-(imidazol-5-yl)lactate dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-(imidazol-5-yl)lactate + NADP+ = 3-(imidazol-5-yl)pyruvate + NADPH + H+." [EC:1.1.1.111]
GO:0019156
name: isoamylase activity
def: "Catalysis of the hydrolysis of alpha-(1,6)-D-glucosidic branch linkages in glycogen, amylopectin and their beta-limits dextrins." [EC:3.2.1.68]
GO:0019157
name: malate oxidase activity
def: "Catalysis of the reaction: (S)-malate + O(2) = H(2)O(2) + oxaloacetate." [EC:1.1.3.3, RHEA:10515]
GO:0019158
name: mannokinase activity
def: "Catalysis of the reaction: ATP + D-mannose = ADP + D-mannose 6-phosphate." [EC:2.7.1.7]
GO:0019159
name: nicotinamide-nucleotide amidase activity
def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide + H2O = beta-nicotinate D-ribonucleotide + NH3." [EC:3.5.1.42]
GO:0019160
name: NMN nucleosidase activity
def: "Catalysis of the reaction: H(2)O + nicotinamide mononucleotide = D-ribose 5-phosphate + H(+) + nicotinamide." [EC:3.2.2.14, RHEA:23143]
GO:0019161
name: diamine transaminase activity
def: "Catalysis of the reaction: an alpha,omega-diamine + 2-oxoglutarate = an omega-aminoaldehyde + L-glutamate." [EC:2.6.1.29]
GO:0019162
name: pyridoxamine-oxaloacetate transaminase activity
def: "Catalysis of the reaction: oxaloacetate + pyridoxamine = L-aspartate + pyridoxal." [EC:2.6.1.31, RHEA:10847]
GO:0019163
name: pyridoxamine-phosphate transaminase activity
def: "Catalysis of the reaction: pyridoxamine 5'-phosphate + 2-oxoglutarate = pyridoxal 5'-phosphate + D-glutamate." [EC:2.6.1.54]
GO:0019164
name: pyruvate synthase activity
def: "Catalysis of the reaction: pyruvate + CoA + 2 oxidized ferredoxin = acetyl-CoA + CO2 + 2 reduced ferredoxin + 2 H+." [KEGG:R01196]
GO:0019165
name: thiamine kinase activity
def: "Catalysis of the reaction: ATP + thiamine = ADP + 2 H(+) + thiamine phosphate." [EC:2.7.1.89, RHEA:12015]
GO:0019166
name: trans-2-enoyl-CoA reductase (NADPH) activity
def: "Catalysis of the reaction: acyl-CoA + NADP+ = trans-2,3-dehydroacyl-CoA + NADPH + H+." [EC:1.3.1.38]
GO:0019168
name: 2-octaprenylphenol hydroxylase activity
def: "Catalysis of the reaction: 2-octaprenylphenol + NADPH + O2 + H+ = 2-octaprenyl-6-hydroxyphenol + NADP+ + H2O." [MetaCyc:2-OCTAPRENYLPHENOL-HYDROX-RXN]
GO:0019170
name: methylglyoxal reductase (NADH-dependent) activity
def: "Catalysis of the reaction: (R)-lactaldehyde + NAD+ = methylglyoxal + NADH + H+." [EC:1.1.1.78]
GO:0019171
name: 3-hydroxyacyl-[acyl-carrier-protein] dehydratase activity
def: "Catalysis of the reaction: a (3R)-3-hydroxyacyl-[acyl-carrier protein] = H2O + a trans-delta2-enoyl-acyl-[acyl-carrier protein]." [MetaCyc:3-HYDROXYDECANOYL-ACP-DEHYDR-RXN]
GO:0019172
name: glyoxalase III activity
def: "Catalysis of the reaction: methylglyoxal + H2O = D-lactate." [EC:4.2.1.130, GOC:mcc, http://www.brenda-enzymes.org/php/result_flat.php4?ecno=4.2.1.130, MetaCyc:GLYOXIII-RXN, PMID:21696459, PMID:7848303, RHEA:27757]
GO:0019174
name: tetrahydrothiophene 1-oxide reductase activity
def: "Catalysis of the reaction: tetrahydrothiophene 1-oxide + reduced acceptor = tetrahydrothiophene + acceptor." [MetaCyc:THTOREDUCT-RXN]
GO:0019176
name: dihydroneopterin monophosphate phosphatase activity
def: "Catalysis of the reaction: dihydroneopterin monophosphate = dihydroneopterin + phosphate." [MetaCyc:DIHYDRONEOPTERIN-MONO-P-DEPHOS-RXN]
GO:0019177
name: dihydroneopterin triphosphate pyrophosphohydrolase activity
def: "Catalysis of the reaction: dihydroneopterin triphosphate = dihydroneopterin phosphate + diphosphate." [MetaCyc:H2NEOPTERINP3PYROPHOSPHOHYDRO-RXN]
GO:0019178
name: NADP phosphatase activity
def: "Catalysis of the reaction: H2O + NADP+ = NAD+ + phosphate." [MetaCyc:NADPPHOSPHAT-RXN]
GO:0019179
name: dTDP-4-amino-4,6-dideoxy-D-glucose transaminase activity
def: "Catalysis of the reaction: dTDP-4-amino-4,6-dideoxy-D-glucose + 2-oxoglutarate = dTDP-4-dehydro-6-deoxy-D-glucose + L-glutamate." [EC:2.6.1.33]
GO:0019180
name: dTDP-4-amino-4,6-dideoxygalactose transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + dTDP-4-amino-4,6-dideoxy-D-galactose = L-glutamate + dTDP-4-dehydro-6-deoxy-D-galactose." [EC:2.6.1.59, RHEA:10371]
GO:0019181
name: halohydrin hydrogen-halide-lyase activity
def: "Catalysis of the reaction: a halohydrin = an epoxide + a hydrogen halide." [PMID:8017917]
GO:0019182
name: histamine-gated chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when histamine has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0019186
name: acyl-CoA N-acyltransferase activity
def: "Catalysis of the transfer of an acyl group from acyl-CoA to a nitrogen atom on an acceptor molecule." [GOC:mah]
GO:0019187
name: beta-1,4-mannosyltransferase activity
def: "Catalysis of the transfer of a mannose residue to an oligosaccharide, forming a beta-(1->4) linkage." [GOC:mcc, PMID:8166646]
GO:0019191
name: cellobiose transmembrane transporter activity
def: "Catalysis of the transfer of cellobiose from one side of the membrane to the other. Cellobiose, or 4-O-beta-D-glucopyranosyl-D-glucose, is a disaccharide that represents the basic repeating unit of cellulose." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0019194
name: sorbose transmembrane transporter activity
def: "Catalysis of the transfer of sorbose from one side of the membrane to the other. Sorbose is the ketohexose xylo-2-hexulose; L-sorbose is formed by bacterial oxidation of sorbitol. Sorbose is produced commercially by fermentation and is used as an intermediate in the manufacture of ascorbic acid." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0019196
name: galactosamine transmembrane transporter activity
def: "Catalysis of the transfer of galactosamine from one side of the membrane to the other. Galactosamine is an aminodeoxysugar; D-galactosamine is a constituent of some glycolipids and glycosaminoglycans, commonly as its N-acetyl derivative." [GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0019198
name: transmembrane receptor protein phosphatase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: a phosphoprotein + H2O = a protein + phosphate." [GOC:hjd]
GO:0019199
name: transmembrane receptor protein kinase activity
def: "Combining with a signal and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: a protein + ATP = a phosphoprotein + ADP." [GOC:mah]
GO:0019200
name: carbohydrate kinase activity
def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a carbohydrate substrate molecule." [GOC:jl]
GO:0019201
name: nucleotide kinase activity
def: "Catalysis of the reaction: ATP + nucleoside monophosphate = ADP + nucleoside diphosphate." [GOC:ai, ISBN:0198506732]
GO:0019202
name: amino acid kinase activity
def: "Catalysis of the transfer of a phosphate group, usually from ATP, to an amino acid substrate." [GOC:jl]
GO:0019203
name: carbohydrate phosphatase activity
def: "Catalysis of the reaction: carbohydrate phosphate + H2O = carbohydrate + phosphate." [GOC:mah]
GO:0019204
name: nucleotide phosphatase activity
def: "OBSOLETE: Catalysis of the reaction: phosphopolynucleotide + H2O = polynucleotide + phosphate." [GOC:mah]
GO:0019205
name: nucleobase-containing compound kinase activity
def: "Catalysis of the transfer of a phosphate group, usually from ATP or GTP, to a nucleobase, nucleoside, nucleotide or polynucleotide substrate." [GOC:jl]
GO:0019206
name: nucleoside kinase activity
def: "Catalysis of the reaction: ATP + nucleoside = ADP + nucleoside monophosphate." [GOC:ai]
GO:0019207
name: kinase regulator activity
def: "Modulates the activity of a kinase, an enzyme which catalyzes of the transfer of a phosphate group, usually from ATP, to a substrate molecule." [GOC:ai]
GO:0019208
name: phosphatase regulator activity
def: "Modulates the activity of a phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a substrate molecule." [GOC:ai]
GO:0019209
name: kinase activator activity
def: "Increases the activity of a kinase, an enzyme which catalyzes of the transfer of a phosphate group, usually from ATP, to a substrate molecule." [GOC:ai]
GO:0019210
name: kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a kinase, an enzyme which catalyzes of the transfer of a phosphate group, usually from ATP, to a substrate molecule." [GOC:mah]
GO:0019211
name: phosphatase activator activity
def: "Increases the activity of a phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a substrate molecule." [GOC:ai]
GO:0019212
name: phosphatase inhibitor activity
def: "Stops, prevents or reduces the activity of a phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a substrate molecule." [GOC:ai]
GO:0019213
name: deacetylase activity
def: "Catalysis of the hydrolysis of an acetyl group or groups from a substrate molecule." [GOC:jl]
GO:0019214
name: surfactant activity
def: "OBSOLETE. The action of reducing the surface tension of a liquid." [GOC:jl, ISBN:0198506732]
GO:0019215
name: intermediate filament binding
def: "Interacting selectively and non-covalently with an intermediate filament, a distinct elongated structure, characteristically 10 nm in diameter, that occurs in the cytoplasm of higher eukaryotic cells. Intermediate filaments form a fibrous system, composed of chemically heterogeneous subunits and involved in mechanically integrating the various components of the cytoplasmic space." [ISBN:0198506732]
GO:0019237
name: centromeric DNA binding
def: "Interacting selectively and non-covalently with a centromere, a region of chromosome where the spindle fibers attach during mitosis and meiosis." [GOC:jl, SO:0000577]
GO:0019238
name: cyclohydrolase activity
def: "Catalysis of the hydrolysis of any non-peptide carbon-nitrogen bond in a cyclic amidine, a compound of the form R-C(=NH)-NH2, in a reaction that involves the opening of a ring." [GOC:mah]
GO:0019239
name: deaminase activity
def: "Catalysis of the removal of an amino group from a substrate, producing ammonia (NH3)." [GOC:jl]
GO:0019531
name: oxalate transmembrane transporter activity
def: "Catalysis of the transfer of oxalate from one side of the membrane to the other. Oxalate, or ethanedioic acid, occurs in many plants and is highly toxic to animals." [ISBN:0198506732]
GO:0019534
name: toxin transporter activity
def: "Enables the directed movement of a toxin into, out of or within a cell, or between cells. A toxin is a poisonous compound (typically a protein) that is produced by cells or organisms and that can cause disease when introduced into the body or tissues of an organism." [ISBN:0198506732]
GO:0019535
name: ferric-vibriobactin transmembrane transporter activity
def: "Catalysis of the transfer of ferric-vibriobactin ions from one side of a membrane to the other." [GOC:ai]
GO:0019702
name: protein-arginine N5-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the delta-nitrogen atom of peptidyl-arginine residues." [PMID:9873020]
GO:0019705
name: protein-cysteine S-myristoyltransferase activity
def: "Catalysis of the transfer of a myristoyl group to a sulfur atom on a cysteine residue of a protein molecule." [GOC:ai]
GO:0019706
name: protein-cysteine S-palmitoyltransferase activity
def: "Catalysis of the transfer of a palmitoyl group to a sulfur atom on the cysteine of a protein molecule." [EC:2.3.1.225, GOC:ai, GOC:pr]
GO:0019707
name: protein-cysteine S-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to a sulfur atom on the cysteine of a protein molecule." [GOC:ai]
GO:0019726
name: mevaldate reductase (NADPH) activity
def: "Catalysis of the reaction: (R)-mevalonate + NADP(+) = H(+) + mevaldate + NADPH." [EC:1.1.1.33, RHEA:20196]
GO:0019727
name: mevaldate reductase (NAD+) activity
def: "Catalysis of the reaction: (R)-mevalonate + NAD(+) = H(+) + mevaldate + NADH." [EC:1.1.1.32, RHEA:13224]
GO:0019763
name: immunoglobulin receptor activity
def: "Combining with the Fc region of an immunoglobulin protein and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:signaling, ISBN:0198547684]
GO:0019764
name: high affinity Fc receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:add]
GO:0019765
name: low affinity Fc receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:add]
GO:0019766
name: IgA receptor activity
def: "Combining with an immunoglobulin of an IgA isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019767
name: IgE receptor activity
def: "Combining with an immunoglobulin of the IgE isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019768
name: high-affinity IgE receptor activity
def: "Combining with high affinity with an immunoglobulin of the IgE isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019769
name: low-affinity IgE receptor activity
def: "Combining with low affinity with an immunoglobulin of the IgE isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019770
name: IgG receptor activity
def: "Combining with an immunoglobulin of an IgG isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019771
name: high-affinity IgG receptor activity
def: "Combining with high affinity with an immunoglobulin of an IgG isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019772
name: low-affinity IgG receptor activity
def: "Combining with low affinity with an immunoglobulin of an IgG isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0019775
name: FAT10 transferase activity
def: "Catalysis of the transfer of FAT10 from one protein to another via the reaction X-FAT10 + Y --> Y-FAT10 + X, where both X-FAT10 and Y-FAT10 are covalent linkages." [GOC:dph, GOC:mah, PMID:12826404]
GO:0019776
name: Atg8 ligase activity
def: "Catalysis of the covalent attachment of the ubiquitin-like protein Atg8 to substrate molecules; phosphatidylethanolamine is a known substrate." [GOC:mah, PMID:12826404]
GO:0019777
name: Atg12 transferase activity
def: "Catalysis of the transfer of ATG12 from one protein to another via the reaction X-ATG12 + Y --> Y-ATG12 + X, where both X-ATG12 and Y-ATG12 are covalent linkages." [GOC:mah, PMID:12826404]
GO:0019778
name: Atg12 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier APG12, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0019779
name: Atg8 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier APG8, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0019780
name: FAT10 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier FAT10, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0019781
name: NEDD8 activating enzyme activity
def: "Catalysis of the initiation of the NEDD8 (RUB1) conjugation cascade." [PMID:12646924]
GO:0019782
name: ISG15 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier ISG15, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0019783
name: small conjugating protein-specific protease activity
def: "Catalysis of the hydrolysis of peptide or isopeptide bonds within small proteins such as ubiquitin or ubiquitin-like proteins (e.g. APG8, ISG15, NEDD8, SUMO), or between the small protein and a larger protein to which it has been conjugated." [GOC:ma, GOC:mah]
GO:0019784
name: NEDD8-specific protease activity
def: "Catalysis of the hydrolysis of NEDD8, a small ubiquitin-related modifier, from previously neddylated substrates." [GOC:mah]
GO:0019785
name: ISG15-specific protease activity
def: "Catalysis of the hydrolysis of ISG15, a small ubiquitin-related modifier, from previously modified substrates." [GOC:mah]
GO:0019786
name: Atg8-specific protease activity
def: "Catalysis of the hydrolysis of APG8, a small ubiquitin-related modifier." [GOC:mah]
GO:0019787
name: small conjugating protein transferase activity
def: "Catalysis of the transfer of ubiquitin from one protein to another via the reaction X-SCP + Y --> Y-SCP + X, where both X-SCP and Y-SCP are covalent linkages. SCP represents a small conjugating protein." [GOC:mah, GOC:rn, PMID:10806345, PMID:10884686]
GO:0019788
name: NEDD8 transferase activity
def: "Catalysis of the transfer of NEDD8 from one protein to another via the reaction X-NEDD8 + Y --> Y-NEDD8 + X, where both X-NEDD8 and Y-NEDD8 are covalent linkages." [GOC:mah]
GO:0019789
name: SUMO transferase activity
def: "Catalysis of the transfer of SUMO from one protein to another via the reaction X-SUMO + Y --> Y-SUMO + X, where both X-SUMO and Y-SUMO are covalent linkages." [GOC:rn, PMID:11031248, PMID:11265250]
GO:0019790
name: ubiquitin-like hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0019791
name: FAT10 hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0019792
name: APG12 hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0019793
name: ISG15 carrier activity
def: "OBSOLETE. Catalysis of the transfer of ISG15 (interferon-stimulated gene-15) from one side of the membrane to the other." [GOC:jl, http://www.copewithcytokines.de/]
GO:0019797
name: procollagen-proline 3-dioxygenase activity
def: "Catalysis of the reaction: procollagen L-proline + 2-oxoglutarate + O2 = procollagen trans-3-hydroxy-L-proline + succinate + CO2." [EC:1.14.11.7]
GO:0019798
name: procollagen-proline dioxygenase activity
def: "Catalysis of the reaction: procollagen L-proline + 2-oxoglutarate + O2 = procollagen trans-hydroxy-L-proline + succinate + CO2." [EC:1.14.11.2, EC:1.14.11.7, GOC:mah]
GO:0019799
name: tubulin N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + (alpha-tubulin) L-lysine = CoA + (alpha-tubulin) N6-acetyl-L-lysine." [EC:2.3.1.108]
GO:0019806
name: bromide peroxidase activity
def: "Catalysis of the reaction: 2 R-H + 2 bromide + hydrogen peroxide = 2 R-Br + 2 H2O. Enzymes with this activity often accept other halide ions as substrates, including chloride and iodide." [EC:1.11.1.10]
GO:0019807
name: aspartoacylase activity
def: "Catalysis of the reaction: N-acyl-L-aspartate + H2O = a fatty acid anion + L-aspartate." [EC:3.5.1.15]
GO:0019808
name: polyamine binding
def: "Interacting selectively and non-covalently with polyamines, organic compounds containing two or more amino groups." [GOC:ai]
GO:0019809
name: spermidine binding
def: "Interacting selectively and non-covalently with spermidine, N-(3-aminopropyl)-1,4-diaminobutane." [GOC:ai]
GO:0019810
name: putrescine binding
def: "Interacting selectively and non-covalently with putrescine, 1,4-diaminobutane, the polyamine formed by decarboxylation of ornithine and the metabolic precursor of spermidine and spermine." [GOC:ai]
GO:0019811
name: cocaine binding
def: "Interacting selectively and non-covalently with cocaine (2-beta-carbomethoxy-3-beta-benzoxytropane), an alkaloid obtained from dried leaves of the South American shrub Erythroxylon coca or by chemical synthesis." [GOC:jl, ISBN:0198506732]
GO:0019825
name: oxygen binding
def: "Interacting selectively and non-covalently with oxygen (O2)." [GOC:jl]
GO:0019826
name: oxygen sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of oxygen (O2)." [GOC:mah]
GO:0019828
name: aspartic-type endopeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of aspartic-type endopeptidases, enzymes that catalyze the hydrolysis of nonterminal peptide bonds in a polypeptide chain; the optimum reaction pH is below 5 due to an aspartic residue involved in the catalytic process." [GOC:ai]
GO:0019829
name: cation-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + cation(out) = ADP + phosphate + cation(in)." [GOC:ai]
GO:0019833
name: ice nucleation activity
def: "OBSOLETE. Catalysis of the formation of ice crystals in extracellular fluid at relatively high temperatures (up to -2 degrees C) to protect the organism from damage by intracellular ice formation. Ice nucleation proteins function by binding an ice crystal and then encouraging it to form larger crystals. Ice nucleation is a chemical process but these proteins can positively regulate it. There are two different uses of ice nucleation proteins: bacteria secrete them extracellularly to cause a host organism's cells to freeze and die, and fish use them to protect themselves from intracellular ice formation." [GOC:ai, GOC:jl]
GO:0019834
name: phospholipase A2 inhibitor activity
def: "Stops, prevents or reduces the activity of the enzyme phospholipase A2." [GOC:ai]
GO:0019838
name: growth factor binding
def: "Interacting selectively and non-covalently with any growth factor, proteins or polypeptides that stimulate a cell or organism to grow or proliferate." [GOC:curators]
GO:0019840
name: isoprenoid binding
def: "Interacting selectively and non-covalently with any isoprenoid compound, isoprene (2-methylbuta-1,3-diene) or compounds containing or derived from linked isoprene (3-methyl-2-butenylene) residues." [GOC:jl]
GO:0019841
name: retinol binding
def: "Interacting selectively and non-covalently with retinol, vitamin A1, 2,6,6-trimethyl-1-(9'-hydroxy-3',7'-dimethylnona-1',3',5',7'-tetraenyl)cyclohex-1-ene, one of the three components that makes up vitamin A. Retinol is an intermediate in the vision cycle and it also plays a role in growth and differentiation." [CHEBI:50211, GOC:curators]
GO:0019842
name: vitamin binding
def: "Interacting selectively and non-covalently with a vitamin, one of a number of unrelated organic substances that occur in many foods in small amounts and that are necessary in trace amounts for the normal metabolic functioning of the body." [GOC:ai]
GO:0019843
name: rRNA binding
def: "Interacting selectively and non-covalently with ribosomal RNA." [GOC:jl]
GO:0019844
name: endotoxin activity
def: "OBSOLETE. The function of any microbial toxin that cannot be easily separated from the structure of the cell." [ISBN:0198506732]
GO:0019845
name: exotoxin activity
def: "OBSOLETE. The function of a toxin formed by a microorganism and secreted into the surrounding medium." [ISBN:0198506732]
GO:0019846
name: enterotoxin activity
def: "OBSOLETE. Acts as to cause injury to the intestinal mucosa of other living organisms." [ISBN:0198506732]
GO:0019847
name: neurotoxin activity
def: "OBSOLETE. Acts to inhibit neural function in another living organism." [GOC:jl]
GO:0019848
name: conotoxin activity
def: "OBSOLETE. Acts to inhibit neural function in another organism by inhibiting voltage-gated calcium ion channels and neurotransmitter release. This function is thought to be specific to the venom of two marine snail species." [GOC:jl, ISBN:0198506732]
GO:0019849
name: cytotoxin activity
def: "OBSOLETE. Acts as to cause injury to other living cells." [GOC:jl]
GO:0019855
name: calcium channel inhibitor activity
def: "Stops, prevents, or reduces the activity of a calcium channel." [GOC:mah]
GO:0019862
name: IgA binding
def: "Interacting selectively and non-covalently with an immunoglobulin of an IgA isotype." [GOC:add, ISBN:0781735149]
GO:0019863
name: IgE binding
def: "Interacting selectively and non-covalently with an immunoglobulin of the IgE isotype." [GOC:add, ISBN:0781735149]
GO:0019864
name: IgG binding
def: "Interacting selectively and non-covalently with an immunoglobulin of an IgG isotype." [GOC:add, ISBN:0781735149]
GO:0019865
name: immunoglobulin binding
def: "Interacting selectively and non-covalently with an immunoglobulin." [GOC:ma]
GO:0019869
name: chloride channel inhibitor activity
def: "Stops, prevents, or reduces the activity of a chloride channel." [GOC:mah]
GO:0019870
name: potassium channel inhibitor activity
def: "Stops, prevents, or reduces the activity of a potassium channel." [GOC:mah]
GO:0019871
name: sodium channel inhibitor activity
def: "Stops, prevents, or reduces the activity of a sodium channel." [GOC:mah]
GO:0019874
name: 6-aminohexanoate-cyclic-dimer hydrolase activity
def: "Catalysis of the reaction: 1,8-diazacyclotetradecane-2,9-dione + H(2)O = N-(6-aminohexanoyl)-6-aminohexanoate." [EC:3.5.2.12, RHEA:16228]
GO:0019875
name: 6-aminohexanoate-dimer hydrolase activity
def: "Catalysis of the reaction: N-(6-aminohexanoyl)-6-aminohexanoate + H2O = 2 6-aminohexanoate." [EC:3.5.1.46]
GO:0019887
name: protein kinase regulator activity
def: "Modulates the activity of a protein kinase, an enzyme which phosphorylates a protein." [GOC:ai]
GO:0019888
name: protein phosphatase regulator activity
def: "Modulates the activity of a protein phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a protein substrate molecule." [GOC:ai]
GO:0019893
name: DNA replication inhibitor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0019894
name: kinesin binding
def: "Interacting selectively and non-covalently and stoichiometrically with kinesin, a member of a superfamily of microtubule-based motor proteins that perform force-generating tasks such as organelle transport and chromosome segregation." [GOC:curators, PMID:8606779]
GO:0019895
name: kinesin-associated mitochondrial adaptor activity
def: "The activity of linking kinesins, cytoplasmic proteins responsible for moving vesicles and organelles towards the distal end of microtubules, to mitochondria." [PMID:12495622]
GO:0019899
name: enzyme binding
def: "Interacting selectively and non-covalently with any enzyme." [GOC:jl]
GO:0019900
name: kinase binding
def: "Interacting selectively and non-covalently with a kinase, any enzyme that catalyzes the transfer of a phosphate group." [GOC:jl]
GO:0019901
name: protein kinase binding
def: "Interacting selectively and non-covalently with a protein kinase, any enzyme that catalyzes the transfer of a phosphate group, usually from ATP, to a protein substrate." [GOC:jl]
GO:0019902
name: phosphatase binding
def: "Interacting selectively and non-covalently with any phosphatase." [GOC:jl]
GO:0019903
name: protein phosphatase binding
def: "Interacting selectively and non-covalently with any protein phosphatase." [GOC:jl]
GO:0019904
name: protein domain specific binding
def: "Interacting selectively and non-covalently with a specific domain of a protein." [GOC:go_curators]
GO:0019905
name: syntaxin binding
def: "Interacting selectively and non-covalently with a syntaxin, a SNAP receptor involved in the docking of synaptic vesicles at the presynaptic zone of a synapse." [ISBN:0198506732]
GO:0019909
name: [pyruvate dehydrogenase (lipoamide)] phosphatase regulator activity
def: "Modification of the activity of the enzyme [pyruvate dehydrogenase (lipoamide)] phosphatase." [EC:3.1.3.43, GOC:ai]
GO:0019911
name: structural constituent of myelin sheath
def: "The action of a molecule that contributes to the structural integrity of the myelin sheath of a nerve." [GOC:mah]
GO:0019912
name: cyclin-dependent protein kinase activating kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein; increases the activity of a cyclin-dependent protein kinase (CDK)." [GOC:go_curators]
GO:0019914
name: cyclin-dependent protein kinase activating kinase regulator activity
def: "Modulation of the activity of the enzyme cyclin-dependent protein kinase activating kinase." [GOC:ai]
GO:0019948
name: SUMO activating enzyme activity
def: "Catalysis of the activation of the proteolytically processed small ubiquitin-related modifier SUMO, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:rn, PMID:10187858, PMID:11265250]
GO:0019955
name: cytokine binding
def: "Interacting selectively and non-covalently with a cytokine, any of a group of proteins that function to control the survival, growth and differentiation of tissues and cells, and which have autocrine and paracrine activity." [GOC:ai, GOC:bf, ISBN:0198599471]
GO:0019956
name: chemokine binding
def: "Interacting selectively and non-covalently with a chemokine. Chemokines are a family of small chemotactic cytokines; their name is derived from their ability to induce directed chemotaxis in nearby responsive cells. All chemokines possess a number of conserved cysteine residues involved in intramolecular disulfide bond formation. Some chemokines are considered pro-inflammatory and can be induced during an immune response to recruit cells of the immune system to a site of infection, while others are considered homeostatic and are involved in controlling the migration of cells during normal processes of tissue maintenance or development. Chemokines are found in all vertebrates, some viruses and some bacteria." [GOC:ai, GOC:BHF, GOC:rl, http://en.wikipedia.org/wiki/Chemokine, PMID:12183377]
GO:0019957
name: C-C chemokine binding
def: "Interacting selectively and non-covalently with a C-C chemokine; C-C chemokines do not have an amino acid between the first two cysteines of the characteristic four-cysteine motif." [GOC:ai]
GO:0019958
name: C-X-C chemokine binding
def: "Interacting selectively and non-covalently with a C-X-C chemokine; C-X-C chemokines have a single amino acid between the first two cysteines of the characteristic four cysteine motif." [GOC:ai]
GO:0019959
name: interleukin-8 binding
def: "Interacting selectively and non-covalently with interleukin-8." [GOC:jl]
GO:0019960
name: C-X3-C chemokine binding
def: "Interacting selectively and non-covalently with a C-X3-C chemokine; C-X3-C chemokines have three amino acids between the first two cysteines of the characteristic four-cysteine motif." [GOC:ai]
GO:0019961
name: interferon binding
def: "Interacting selectively and non-covalently with an interferon, a protein produced by the immune systems of many animals in response to a challenge by a foreign agent." [PMID:9607096, Wikipedia:Interferon]
GO:0019962
name: type I interferon binding
def: "Interacting selectively and non-covalently with a type I interferon. Type I interferons include the interferon-alpha, beta, delta, epsilon, zeta, kappa, tau, and omega gene families." [GOC:add, ISBN:0126896631, PMID:15546383, PMID:16681834]
GO:0019964
name: interferon-gamma binding
def: "Interacting selectively and non-covalently with interferon-gamma. Interferon gamma is the only member of the type II interferon found so far." [GOC:add, GOC:ai, ISBN:0126896631, PMID:15546383, PR:000000017]
GO:0019966
name: interleukin-1 binding
def: "Interacting selectively and non-covalently with interleukin-1." [GOC:jl, ISBN:0198506732]
GO:0019969
name: interleukin-10 binding
def: "Interacting selectively and non-covalently with interleukin-10." [GOC:jl]
GO:0019970
name: interleukin-11 binding
def: "Interacting selectively and non-covalently with interleukin-11." [GOC:jl]
GO:0019972
name: interleukin-12 binding
def: "Interacting selectively and non-covalently with interleukin-12." [GOC:jl]
GO:0019973
name: interleukin-13 binding
def: "Interacting selectively and non-covalently with interleukin-13." [GOC:jl]
GO:0019974
name: interleukin-14 binding
def: "Interacting selectively and non-covalently with interleukin-14." [GOC:jl]
GO:0019975
name: interleukin-17 binding
def: "Interacting selectively and non-covalently with any member of the interleukin-17 family of cytokines." [GOC:add, GOC:jl]
GO:0019976
name: interleukin-2 binding
def: "Interacting selectively and non-covalently with interleukin-2." [GOC:jl]
GO:0019977
name: interleukin-21 binding
def: "Interacting selectively and non-covalently with interleukin-21." [GOC:jl]
GO:0019978
name: interleukin-3 binding
def: "Interacting selectively and non-covalently with interleukin-3." [GOC:jl]
GO:0019979
name: interleukin-4 binding
def: "Interacting selectively and non-covalently with interleukin-4." [GOC:jl]
GO:0019980
name: interleukin-5 binding
def: "Interacting selectively and non-covalently with interleukin-5." [GOC:jl]
GO:0019981
name: interleukin-6 binding
def: "Interacting selectively and non-covalently with interleukin-6." [GOC:jl]
GO:0019982
name: interleukin-7 binding
def: "Interacting selectively and non-covalently with interleukin-7." [GOC:jl]
GO:0019983
name: interleukin-9 binding
def: "Interacting selectively and non-covalently with interleukin-9." [GOC:jl]
GO:0019992
name: diacylglycerol binding
def: "Interacting selectively and non-covalently with diacylglycerol, a diester of glycerol and two fatty acids." [GOC:ma]
GO:0020037
name: heme binding
def: "Interacting selectively and non-covalently with heme, any compound of iron complexed in a porphyrin (tetrapyrrole) ring." [CHEBI:30413, GOC:ai]
GO:0022803
name: passive transmembrane transporter activity
def: "Catalysis of the transfer of a solute from one side of the membrane to the other, down the solute's concentration gradient." [GOC:mtg_transport, ISBN:0815340729]
GO:0022804
name: active transmembrane transporter activity
def: "Catalysis of the transfer of a specific substance or related group of substances from one side of a membrane to the other, up the solute's concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction." [GOC:mtg_transport, ISBN:0815340729]
GO:0022809
name: mobile ion carrier activity
def: "This is a type of carrier produced by bacteria. It enables passive transport by shielding the ion that is being transported from the lipid membrane. It carries an ion across the membrane by enclosing the ion and travelling across the membrane. It does not form a fully open pore across the membrane." [GOC:mtg_transport, GOC:pr, ISBN:0815340729]
GO:0022810
name: membrane potential driven uniporter activity
def: "Enables the active transport of a solute across a membrane by a mechanism involving conformational change, where energy for active transport is derived from membrane potential if the solute is charged." [GOC:mtg_transport, ISBN:0815340729]
GO:0022814
name: facilitated diffusion
def: "Catalysis of the transfer of a single solute from one side of the membrane to the other by a mechanism involving conformational change, either by facilitated diffusion or in a membrane potential dependent process if the solute is charged." [GOC:mtg_transport, ISBN:0815340729]
GO:0022815
name: large uncharged polar molecule transmembrane transporter activity
def: "OBSOLETE. Catalysis of the transfer of a large uncharged polar molecule from one side of a membrane to the other." [GOC:mtg_transport, ISBN:0815340729]
GO:0022818
name: sodium ion uniporter activity
def: "Catalysis of the active transport of a sodium ion across a membrane by a mechanism involving conformational change, where energy for active transport is derived from membrane potential if the solute is charged." [GOC:mtg_transport, ISBN:0815340729]
GO:0022819
name: potassium ion uniporter activity
def: "Catalysis of the active transport of a potassium ion across a membrane by a mechanism involving conformational change, where energy for active transport is derived from membrane potential if the solute is charged." [GOC:mtg_transport, ISBN:0815340729]
GO:0022820
name: potassium ion symporter activity
def: "Catalysis of the active transport of a potassium ion across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport, ISBN:0815340729]
GO:0022821
name: potassium ion antiporter activity
def: "Catalysis of the active transport of a potassium ion across a membrane by a mechanism whereby two or more species are transported in opposite directions in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy." [GOC:mtg_transport, ISBN:0815340729]
GO:0022824
name: transmitter-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a specific neurotransmitter has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022825
name: copper-exporting ATPase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:go_curators]
GO:0022828
name: phosphorylation-gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens in response to phosphorylation of one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022829
name: wide pore channel activity
def: "Enables the transport of a solute across a membrane via a large pore, un-gated channel. Examples include gap junctions, which transport substances from one cell to another; and porins which transport substances in and out of bacteria, mitochondria and chloroplasts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022831
name: narrow pore, gated channel activity
def: "Enables the transport of a solute across a membrane via a narrow pore channel that opens in response to a particular stimulus." [GOC:mtg_transport, ISBN:0815340729]
GO:0022832
name: voltage-gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0022833
name: mechanically gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens in response to a mechanical stress." [GOC:mtg_transport, ISBN:0815340729]
GO:0022834
name: ligand-gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens when a specific ligand has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022835
name: transmitter-gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens when a specific neurotransmitter has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022836
name: gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens in response to a specific stimulus." [GOC:mtg_transport]
GO:0022838
name: substrate-specific channel activity
def: "Catalysis of energy-independent facilitated diffusion, mediated by passage of a specific solute through a transmembrane aqueous pore or channel. Stereospecificity is not exhibited but this transport may be specific for a particular molecular species or class of molecules." [GOC:mtg_transport]
GO:0022839
name: ion gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens in response to a specific ion stimulus." [GOC:mtg_transport]
GO:0022840
name: leak channel activity
def: "Enables the transport of a solute across a membrane via a narrow pore channel that is open even in an unstimulated or 'resting' state." [GOC:mtg_transport, ISBN:0815340729]
GO:0022841
name: potassium ion leak channel activity
def: "Catalysis of the transport of a potassium ion across a membrane via a narrow pore channel that is open even in an unstimulated or 'resting' state." [GOC:mtg_transport, ISBN:0815340729]
GO:0022842
name: narrow pore channel activity
def: "Enables the transport of a solute across a membrane via a narrow pore channel that may be gated or ungated." [GOC:mtg_transport, ISBN:0815340729]
GO:0022843
name: voltage-gated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a voltage-gated channel. A cation is a positively charged ion. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:mtg_transport, ISBN:0815340729]
GO:0022848
name: acetylcholine-gated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when acetylcholine has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022849
name: glutamate-gated calcium ion channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when glutamate has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022850
name: serotonin-gated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens when serotonin has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022851
name: GABA-gated chloride ion channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when GABA has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022852
name: glycine-gated chloride ion channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a channel that opens when glycine has been bound by the channel complex or one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022853
name: active ion transmembrane transporter activity
def: "Catalysis of the transfer of an ion from one side of a membrane to the other up the solute's concentration gradient. This is carried out by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction." [GOC:mtg_transport, ISBN:0815340729]
GO:0022854
name: active large uncharged polar molecule transmembrane transporter activity
def: "OBSOLETE. Catalysis of the transfer of a large uncharged polar molecule from one side of the membrane to the other, up the solute's concentration gradient, by binding the solute and undergoing a series of conformational changes. Transport works equally well in either direction." [GOC:mtg_transport, ISBN:0815340729]
GO:0022855
name: protein-N(PI)-phosphohistidine-glucose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + glucose(out) = protein histidine + glucose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022856
name: protein-N(PI)-phosphohistidine-sorbitol phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sorbitol(out) = protein histidine + sorbitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022857
name: transmembrane transporter activity
def: "Enables the transfer of a substance from one side of a membrane to the other." [GOC:mtg_transport, ISBN:0815340729]
GO:0022858
name: alanine transmembrane transporter activity
def: "Catalysis of the transfer of alanine from one side of a membrane to the other. Alanine is 2-aminopropanoic acid." [GOC:mtg_transport, ISBN:0815340729]
GO:0022859
name: dephosphorylation-gated channel activity
def: "Catalysis of the transmembrane transfer of a solute by a channel that opens in response to dephosphorylation of one of its constituent parts." [GOC:mtg_transport, ISBN:0815340729]
GO:0022865
name: transmembrane electron transfer carrier
def: "Enables electron flow across a biological membrane, from donors localized on one side of the membrane to acceptors localized on the other side. These systems contribute to or subtract from the membrane potential, depending on the direction of electron flow. They are therefore important to cellular energetics." [GOC:mtg_transport, ISBN:0815340729, TC:5]
GO:0022866
name: transmembrane 1-electron transfer carrier
def: "Enables transfer of one electron across a membrane." [GOC:mtg_transport, ISBN:0815340729, TC:5.B]
GO:0022867
name: transmembrane 2-electron transfer carrier
def: "Enables transfer of two electrons across a membrane." [GOC:mtg_transport, ISBN:0815340729, TC:5.A]
GO:0022869
name: protein-N(PI)-phosphohistidine-lactose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + lactose(out) = protein histidine + lactose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022870
name: protein-N(PI)-phosphohistidine-mannose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannose(out) = protein histidine + mannose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022871
name: protein-N(PI)-phosphohistidine-sorbose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sorbose(out) = protein histidine + sorbose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022872
name: protein-N(PI)-phosphohistidine-mannitol phosphotransferase system transmembrane transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + mannitol(out) = protein histidine + mannitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022873
name: protein-N(PI)-phosphohistidine-maltose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + maltose(out) = protein histidine + maltose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022874
name: protein-N(PI)-phosphohistidine-cellobiose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + cellobiose(out) = protein histidine + cellobiose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022875
name: protein-N(PI)-phosphohistidine-galactitol phosphotransferase system transmembrane transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactitol(out) = protein histidine + galactitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022876
name: protein-N(PI)-phosphohistidine-galactosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + galactosamine(out) = protein histidine + galactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022877
name: protein-N(PI)-phosphohistidine-fructose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + fructose(out) = protein histidine + fructose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022878
name: protein-N(PI)-phosphohistidine-sucrose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + sucrose(out) = protein histidine + sucrose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022879
name: protein-N(PI)-phosphohistidine-trehalose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + trehalose(out) = protein histidine + trehalose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022880
name: protein-N(PI)-phosphohistidine-N-acetylglucosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylglucosamine(out) = protein histidine + N-acetylglucosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022881
name: protein-N(PI)-phosphohistidine-N-acetylgalactosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetylgalactosamine(out) = protein histidine + N-acetylgalactosamine phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022882
name: protein-N(PI)-phosphohistidine-beta-glucoside phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + beta-glucoside(out) = protein histidine + beta-glucoside phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:mtg_transport, ISBN:0815340729]
GO:0022883
name: zinc efflux transmembrane transporter activity
def: "Catalysis of the transfer of a zinc ion or zinc ions from the inside of the cell to the outside of the cell across a membrane." [GOC:mtg_transport, ISBN:0815340729]
GO:0022884
name: macromolecule transmembrane transporter activity
def: "Catalysis of the transfer of a macromolecule from one side of the membrane to the other." [GOC:mtg_transport, ISBN:0815340729]
GO:0022885
name: bacteriocin transmembrane transporter activity
def: "Catalysis of the transfer of a bacteriocin from one side of the membrane to the other." [GOC:mtg_transport, ISBN:0815340729]
GO:0022886
name: channel-forming ionophore activity
def: "Enables transport of a solute across a membrane. This kind of transporter interacts much more weakly with the solute than the carrier does. It is an aqueous pore that extends across the membrane. It may change from closed to open and back. It transports faster than a carrier. It is always passive." [GOC:mtg_transport, ISBN:0815340729]
GO:0022889
name: serine transmembrane transporter activity
def: "Catalysis of the transfer of serine from one side of a membrane to the other. Serine is 2-amino-3-hydroxypropanoic acid." [GOC:mtg_transport, ISBN:0815340729]
GO:0022890
name: inorganic cation transmembrane transporter activity
def: "Catalysis of the transfer of inorganic cations from one side of a membrane to the other. Inorganic cations are atoms or small molecules with a positive charge that do not contain carbon in covalent linkage." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0022891
name: substrate-specific transmembrane transporter activity
def: "Enables the transfer of a specific substance or group of related substances from one side of a membrane to the other." [GOC:jid, GOC:mtg_transport, ISBN:0815340729]
GO:0022892
name: substrate-specific transporter activity
def: "Enables the directed movement of a specific substance or group of related substances (such as macromolecules, small molecules, ions) into, out of or within a cell, or between cells." [GOC:mtg_transport]
GO:0022893
name: low-affinity tryptophan transmembrane transporter activity
def: "Catalysis of the low-affinity transfer of L-tryptophan from one side of a membrane to the other. Tryptophan is 2-amino-3-(1H-indol-3-yl)propanoic acid. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:mtg_transport, ISBN:0815340729]
GO:0022894
name: Intermediate conductance calcium-activated potassium channel activity
def: "Catalysis of the transmembrane transfer of potassium by a channel with a unit conductance of 20 to 85 picoSiemens that opens in response to stimulus by internal calcium ions. Intermediate conductance calcium-activated potassium channels are more sensitive to calcium than are large conductance calcium-activated potassium channels. Transport by a channel involves catalysis of facilitated diffusion of a solute (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel, without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, OMIM:602754]
GO:0022897
name: proton-dependent peptide secondary active transmembrane transporter activity
def: "Catalysis of the transfer of a peptide from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by proton movement." [GOC:mtg_transport]
GO:0023023
name: MHC protein complex binding
def: "Interacting selectively and non-covalently with the major histocompatibility complex." [GOC:mtg_signal, GOC:vw]
GO:0023024
name: MHC class I protein complex binding
def: "Interacting selectively and non-covalently with the class I major histocompatibility complex." [GOC:mtg_signal, GOC:vw]
GO:0023025
name: MHC class Ib protein complex binding
def: "Interacting selectively and non-covalently with the class Ib major histocompatibility complex." [GOC:mtg_signal, GOC:vw]
GO:0023026
name: MHC class II protein complex binding
def: "Interacting selectively and non-covalently with the class II major histocompatibility complex." [GOC:mtg_signal, GOC:vw]
GO:0023027
name: MHC class I protein binding, via antigen binding groove
def: "Interacting selectively and non-covalently with major histocompatibility complex class I molecules via the antigen binding groove." [GOC:mtg_signal, GOC:vw]
GO:0023028
name: MHC class I protein binding, via lateral surface
def: "Interacting selectively and non-covalently with major histocompatibility complex class I molecules via the lateral surface." [GOC:mtg_signal, GOC:vw]
GO:0023029
name: MHC class Ib protein binding
def: "Interacting selectively and non-covalently with major histocompatibility complex class Ib molecules." [GOC:mtg_signal, GOC:vw]
GO:0023030
name: MHC class Ib protein binding, via antigen binding groove
def: "Interacting selectively and non-covalently with major histocompatibility complex class Ib molecules via the antigen binding groove." [GOC:mtg_signal, GOC:vw]
GO:0023031
name: MHC class Ib protein binding, via lateral surface
def: "Interacting selectively and non-covalently with major histocompatibility complex class Ib molecules via the lateral surface." [GOC:mtg_signal, GOC:vw]
GO:0030009
name: complement factor H activity
def: "OBSOLETE. A cofactor for the serine protease complement factor I." [ISBN:0198547684]
GO:0030019
name: tryptase activity
def: "OBSOLETE. Catalysis of the preferential cleavage of Arg-Xaa, Lys-Xaa, but with more restricted specificity than trypsin." [EC:3.4.21.59]
GO:0030020
name: extracellular matrix structural constituent conferring tensile strength
def: "A constituent of the extracellular matrix that enables the matrix to resist longitudinal stress." [GOC:mah, ISBN:0815316194]
GO:0030021
name: extracellular matrix structural constituent conferring compression resistance
def: "A constituent of the extracellular matrix that enables the matrix to resist compressive forces; often a proteoglycan." [GOC:mah, ISBN:0815316194]
GO:0030022
name: adhesive extracellular matrix constituent
def: "OBSOLETE. A constituent of the extracellular matrix that facilitates attachment of cells to the matrix." [GOC:mah, ISBN:0815316194]
GO:0030023
name: extracellular matrix constituent conferring elasticity
def: "A component of the extracellular matrix that enables the matrix to recoil after transient stretching." [GOC:mah, ISBN:0815316194]
GO:0030039
name: DNA unwinding factor
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0030051
name: FK506-sensitive peptidyl-prolyl cis-trans isomerase
def: "OBSOLETE. Interacts selectively with the immunosuppressant FK506, and possesses peptidyl-prolyl isomerase activity (catalysis of the reaction: peptidoproline (omega=180) = peptidylproline (omega=0))." [EC:5.2.1.8, ISBN:0198596732]
GO:0030052
name: parvulin
def: "OBSOLETE. A peptidyl-prolyl isomerase isolated from Escherichia coli. It does not have any function as an immunophilin." [ISBN:0198506732]
GO:0030053
name: immunophilin
def: "OBSOLETE. Any member of a family of receptors that includes the major FK506 binding protein FKBP and cyclophilin. These two proteins are unrelated in amino-acid sequence, but both possess peptidyl-prolyl isomerase activity which is blocked by immunosuppressants that block signal-transduction pathways leading to T cell activation such as FK506 and rapamycin, which block FKBP, or cyclosporin A, which blocks cyclophilin." [ISBN:0198506732]
GO:0030058
name: amine dehydrogenase activity
def: "Catalysis of the reaction: R-CH2-NH2 + H2O + acceptor = R-CHO + NH3 + reduced acceptor." [EC:1.4.99.3]
GO:0030059
name: aralkylamine dehydrogenase (azurin) activity
def: "Catalysis of the reaction: H(2)O + arCH(2)NH(2) + 2 azurin = 2 reduced azurin + ArCHO + NH(3). The symbol arCH(2)NH(2) represents an aralkylamine and ArCHO is an aryl aldehyde." [EC:1.4.9.2]
GO:0030060
name: L-malate dehydrogenase activity
def: "Catalysis of the reaction: (S)-malate + NAD+ = oxaloacetate + NADH + H+." [EC:1.1.1.37]
GO:0030066
name: cytochrome b6
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0030067
name: respiratory chain cytochrome b6
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0030106
name: MHC class I receptor activity
def: "OBSOLETE. A major histocompatibility complex class I receptor. These display processed antigens from the endocytosed bacteria. MHC class I proteins can form complexes with processed antigen peptides only if the antigen is synthesized in the same cell. As a consequence, T lymphocytes can only bind to class-I-positive cells that are synthesizing the antigen (e.g. virus-infected cells)." [ISBN:081533642X, ISBN:0879694971]
GO:0030107
name: HLA-A specific inhibitory MHC class I receptor activity
def: "Combining with a MHC class I molecule of the HLA-A subclass to mediate signaling that inhibits activation of a lymphocyte." [GOC:add, GOC:mah, PMID:11929129, PMID:9368779]
GO:0030108
name: HLA-A specific activating MHC class I receptor activity
def: "Combining with a MHC class I molecule of the HLA-A subclass to mediate signaling that activates a lymphocyte." [GOC:add, GOC:mah, PMID:11929129, PMID:9368779]
GO:0030109
name: HLA-B specific inhibitory MHC class I receptor activity
def: "Combining with a MHC class I molecule of the HLA-B subclass to mediate signaling that inhibits activation of a lymphocyte." [GOC:add, GOC:mah, PMID:11929129, PMID:9368779]
GO:0030110
name: HLA-C specific inhibitory MHC class I receptor activity
def: "Combining with a MHC class I molecule of the HLA-C subclass to mediate signaling that inhibits activation of a lymphocyte." [GOC:add, GOC:mah, PMID:11929129, PMID:9368779]
GO:0030116
name: glial cell line-derived neurotrophic factor receptor binding
def: "A growth factor that binds selectively and non-covalently to glial cell line-derived neurotrophic factor receptors." [GOC:vw, PMID:11476867]
GO:0030144
name: alpha-1,6-mannosylglycoprotein 6-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,3(6)-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl,1,6(3))-beta-D-mannosyl-1,4-N-acetyl-beta-D-glucosaminyl-R = UDP + N-acetyl-beta-D-glucosaminyl-1,2-(N-acetyl-beta-D-glucosaminyl-1,6)-1,2-alpha-D-mannosyl-1,3(6)-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6(3))-beta-D-mannosyl-1,4-N-acetyl-beta-D-glucosaminyl-R. Only branched mannose glycopeptides with non-reducing N-acetylglucosamine terminal residues act as acceptors." [EC:2.4.1.155]
GO:0030145
name: manganese ion binding
def: "Interacting selectively and non-covalently with manganese (Mn) ions." [GOC:ai]
GO:0030151
name: molybdenum ion binding
def: "Interacting selectively and non-covalently with molybdenum (Mo) ions." [GOC:ai]
GO:0030156
name: benzodiazepine receptor binding
def: "Interacting selectively and non-covalently with the peripheral benzodiazepine receptor (PBR)." [GOC:ceb, GOC:mah, PMID:9915832]
GO:0030158
name: protein xylosyltransferase activity
def: "Catalysis of the transfer of a beta-D-xylosyl residue from UDP-D-xylose to the serine hydroxyl group of an acceptor protein substrate." [EC:2.4.2.26]
GO:0030159
name: receptor signaling complex scaffold activity
def: "Functions to provide a physical support for the assembly of a multiprotein receptor signaling complex." [GOC:mah]
GO:0030160
name: GKAP/Homer scaffold activity
def: "Functions as a physical support bridging the N-methyl-D-aspartate receptor-PSD-95-GKAP complex and the mGluR-Homer complex, which are involved in receptor signaling in synapses." [PMID:10506216]
GO:0030161
name: calpain inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the enzyme calpain, which catalyzes of the preferential cleavage of Tyr-Xaa, Met-Xaa or Arg-Xaa with Leu or Val as the P2 residue." [GOC:ai]
GO:0030165
name: PDZ domain binding
def: "Interacting selectively and non-covalently with a PDZ domain of a protein, a domain found in diverse signaling proteins." [GOC:go_curators, Pfam:PF00595]
GO:0030169
name: low-density lipoprotein particle binding
def: "Interacting selectively and non-covalently with a low-density lipoprotein particle, a lipoprotein particle that is rich in cholesterol esters and low in triglycerides, is typically composed of APOB100 and APOE, and has a density of 1.02-1.06 g/ml and a diameter of between 20-25 nm." [GOC:mah]
GO:0030170
name: pyridoxal phosphate binding
def: "Interacting selectively and non-covalently with pyridoxal 5' phosphate, 3-hydroxy-5-(hydroxymethyl)-2-methyl4-pyridine carboxaldehyde 5' phosphate, the biologically active form of vitamin B6." [GOC:mah, ISBN:0198506732]
GO:0030171
name: voltage-gated proton channel activity
def: "Catalysis of the transmembrane transfer of a proton by a voltage-gated channel. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [ISBN:10615049]
GO:0030172
name: troponin C binding
def: "Interacting selectively and non-covalently with troponin C, the calcium-binding subunit of the troponin complex." [GOC:mah, ISBN:0815316194]
GO:0030184
name: nitric oxide transmembrane transporter activity
def: "Enables the directed movement of nitric oxide, nitrogen monoxide, from one side of a membrane to the other." [GOC:mah]
GO:0030188
name: chaperone regulator activity
def: "OBSOLETE. Modulates the activity of a molecular chaperone." [GOC:mah]
GO:0030189
name: chaperone activator activity
def: "OBSOLETE. Increases the activity of a molecular chaperone." [GOC:mah]
GO:0030190
name: chaperone inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of a molecular chaperone." [GOC:mah]
GO:0030191
name: Hsp70/Hsc70 protein inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of chaperones of the Hsp70/Hsc70 class." [GOC:mah]
GO:0030192
name: Hsp70/Hsc70 protein regulator activity
def: "OBSOLETE. Modulates the activity of chaperones of the Hsp70/Hsc70 class." [GOC:mah]
GO:0030196
name: cyanide hydratase activity
def: "Catalysis of the reaction: formamide = H(2)O + hydrogen cyanide." [EC:4.2.1.66, RHEA:21723]
GO:0030197
name: extracellular matrix constituent, lubricant activity
def: "Functions as a lubricant for an extracellular matrix, such as a mucous membrane." [GOC:mah]
GO:0030215
name: semaphorin receptor binding
def: "Interacting selectively and non-covalently with semaphorin receptors." [GOC:ceb, PMID:12001990]
GO:0030226
name: apolipoprotein receptor activity
def: "Combining with an apolipoprotein to initiate a change in cell activity." [GOC:mah, ISBN:0198506732]
GO:0030227
name: apolipoprotein E receptor activity
def: "OBSOLETE. Combining with apolipoprotein E to initiate a change in cell activity." [GOC:mah]
GO:0030228
name: lipoprotein particle receptor activity
def: "Combining with a lipoprotein particle and delivering the lipoprotein particle into the cell via endocytosis. A lipoprotein particle, also known as a lipoprotein, is a clathrate complex consisting of a lipid enwrapped in a protein host without covalent binding in such a way that the complex has a hydrophilic outer surface consisting of all the protein and the polar ends of any phospholipids." [CHEBI:6495, GOC:bf, GOC:mah, PMID:12827279]
GO:0030229
name: very-low-density lipoprotein particle receptor activity
def: "Combining with a very-low-density lipoprotein particle and delivering the very-low-density lipoprotein into the cell via endocytosis." [GOC:bf, ISBN:0198506732]
GO:0030233
name: deoxynucleotide transmembrane transporter activity
def: "Catalyzes transport of all four deoxy (d) NDPs, and, less efficiently, the corresponding dNTPs, in exchange for dNDPs, ADP, or ATP." [PMID:11226231]
GO:0030234
name: enzyme regulator activity
def: "Binds to and modulates the activity of an enzyme." [GOC:mah]
GO:0030235
name: nitric-oxide synthase regulator activity
def: "Modulates the activity of nitric oxide synthase." [GOC:mah]
GO:0030246
name: carbohydrate binding
def: "Interacting selectively and non-covalently with any carbohydrate, which includes monosaccharides, oligosaccharides and polysaccharides as well as substances derived from monosaccharides by reduction of the carbonyl group (alditols), by oxidation of one or more hydroxy groups to afford the corresponding aldehydes, ketones, or carboxylic acids, or by replacement of one or more hydroxy group(s) by a hydrogen atom. Cyclitols are generally not regarded as carbohydrates." [CHEBI:16646, GOC:mah]
GO:0030247
name: polysaccharide binding
def: "Interacting selectively and non-covalently with any polysaccharide, a polymer of many (typically more than 10) monosaccharide residues linked glycosidically." [CHEBI:18154, GOC:mah]
GO:0030248
name: cellulose binding
def: "Interacting selectively and non-covalently with cellulose." [GOC:mah]
GO:0030249
name: guanylate cyclase regulator activity
def: "Modulates the activity of guanylate cyclase." [GOC:mah]
GO:0030250
name: guanylate cyclase activator activity
def: "Increases the activity of guanylate cyclase." [GOC:mah]
GO:0030251
name: guanylate cyclase inhibitor activity
def: "Stops, prevents or reduces the activity of guanylate cyclase." [GOC:mah]
GO:0030266
name: quinate 3-dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: (-)-quinate + NAD+ = (-)-3-dehydroquinate + NADH + H+." [EC:1.1.1.24]
GO:0030267
name: glyoxylate reductase (NADP) activity
def: "Catalysis of the reaction: glycolate + NADP+ = glyoxylate + NADPH + H+." [EC:1.1.1.79]
GO:0030268
name: methylenetetrahydromethanopterin dehydrogenase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydromethanopterin + coenzyme F420 + 2 H(+) = 5,10-methenyl-5,6,7,8-tetrahydromethanopterin + reduced coenzyme F420." [EC:1.5.99.9, RHEA:16724]
GO:0030269
name: tetrahydromethanopterin S-methyltransferase activity
def: "Catalysis of the reaction: 5-methyltetrahydromethanopterin + coenzyme M = 5,6,7,8-tetrahydromethanopterin + methyl-coenzyme M. 2-(methylthio)ethanesulfonate is also known as methyl-CoM." [EC:2.1.1.86, RHEA:17588]
GO:0030270
name: formylmethanofuran-tetrahydromethanopterin N-formyltransferase activity
def: "Catalysis of the reaction: 5,6,7,8-tetrahydromethanopterin + N-formylmethanofuran + H(+) = N(5)-formyl-5,6,7,8-tetrahydromethanopterin + methanofuran." [EC:2.3.1.101, RHEA:18064]
GO:0030271
name: chymase activity
def: "OBSOLETE. Catalysis of the preferential cleavage: Phe-Xaa > Tyr-Xaa > Trp-Xaa > Leu-Xaa." [EC:3.4.21.39]
GO:0030272
name: 5-formyltetrahydrofolate cyclo-ligase activity
def: "Catalysis of the reaction: 5-formyltetrahydrofolate + ATP = 5,10-methenyltetrahydrofolate + ADP + H(+) + phosphate." [EC:6.3.3.2, RHEA:10491]
GO:0030273
name: melanin-concentrating hormone receptor activity
def: "Combining with the cyclic peptide hormone melanin-concentrating hormone to initiate a change in cell activity." [GOC:mah]
GO:0030274
name: LIM domain binding
def: "Interacting selectively and non-covalently with a LIM domain (for Lin-11 Isl-1 Mec-3) of a protein, a domain with seven conserved cysteine residues and a histidine, that binds two zinc ions and acts as an interface for protein-protein interactions." [GOC:go_curators, Pfam:PF00412]
GO:0030275
name: LRR domain binding
def: "Interacting selectively and non-covalently with a LRR domain (leucine rich repeats) of a protein." [GOC:go_curators, Pfam:PF00560]
GO:0030276
name: clathrin binding
def: "Interacting selectively and non-covalently with a clathrin heavy or light chain, the main components of the coat of coated vesicles and coated pits, and which also occurs in synaptic vesicles." [GOC:jl, GOC:mah, ISBN:0198506732]
GO:0030280
name: structural constituent of epidermis
def: "The action of a molecule that contributes to the structural integrity of an epidermal cutaneous structure." [GOC:mah]
GO:0030281
name: structural constituent of cutaneous appendage
def: "The action of a molecule that contributes to the structural integrity of cutaneous epidermal structures such as hairs, scales, or feathers." [GOC:mah, ISBN:0878932437]
GO:0030283
name: testosterone dehydrogenase [NAD(P)] activity
def: "Catalysis of the reaction: testosterone + NAD(P)+ = androst-4-ene-3,17-dione + NAD(P)H + H+." [EC:1.1.1.51]
GO:0030284
name: estrogen receptor activity
def: "Combining with estrogen and transmitting the signal within the cell to trigger a change in cell activity or function." [GOC:signaling, PMID:17615392]
GO:0030290
name: sphingolipid activator protein activity
def: "Any of a group of peptide cofactors of enzymes for the lysosomal degradation of sphingolipids. They stimulate various enzymes, including glucosylceramidase, galactosylceramidase, cerebroside-sulfatase, alpha-galactosidase, beta-galactosidase, and sphingomyelin phosphodiesterase." [ISBN:0198506732]
GO:0030291
name: protein serine/threonine kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a protein serine/threonine kinase." [GOC:mah]
GO:0030292
name: protein tyrosine kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a protein tyrosine kinase." [GOC:mah]
GO:0030293
name: transmembrane receptor protein tyrosine kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a transmembrane receptor protein tyrosine kinase." [GOC:mah]
GO:0030294
name: receptor signaling protein tyrosine kinase inhibitor activity
def: "Stops, prevents or reduces the activity of a receptor signaling protein tyrosine kinase." [GOC:mah]
GO:0030295
name: protein kinase activator activity
def: "Increases the activity of a protein kinase, an enzyme which phosphorylates a protein." [GOC:ai]
GO:0030296
name: protein tyrosine kinase activator activity
def: "Increases the activity of a protein tyrosine kinase, an enzyme which phosphorylates a tyrosyl phenolic group on a protein." [GOC:ai, ISBN:0198506732]
GO:0030297
name: transmembrane receptor protein tyrosine kinase activator activity
def: "Increases the activity of a transmembrane receptor protein tyrosine kinase." [GOC:mah]
GO:0030298
name: receptor signaling protein tyrosine kinase activator activity
def: "Increases the activity of a receptor signaling protein tyrosine kinase." [GOC:mah]
GO:0030303
name: stromelysin 2 activity
def: "OBSOLETE. Catalysis of the cleavage of peptide bonds in collagen with preferential cleavage where P1', P2' and P3' are hydrophobic residues; action on collagen types III, IV and V is weak." [EC:3.4.24.17, EC:3.4.24.22]
GO:0030304
name: trypsin inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the serine endopeptidase trypsin." [GOC:ai]
GO:0030305
name: heparanase activity
def: "Catalysis of the cleavage of heparan sulfate; can degrade both heparan sulfate and heparin glycosaminoglycan chains." [PMID:10916150]
GO:0030306
name: ADP-ribosylation factor binding
def: "Interacting selectively and non-covalently with ARF, ADP-ribosylation factor, a small monomeric cytosolic GTPase that, when bound to GTP, binds to the membranes of cells." [ISBN:0198506732]
GO:0030331
name: estrogen receptor binding
def: "Interacting selectively and non-covalently with an estrogen receptor." [GOC:ai]
GO:0030332
name: cyclin binding
def: "Interacting selectively and non-covalently with cyclins, proteins whose levels in a cell varies markedly during the cell cycle, rising steadily until mitosis, then falling abruptly to zero. As cyclins reach a threshold level, they are thought to drive cells into G2 phase and thus to mitosis." [GOC:ai]
GO:0030337
name: DNA polymerase processivity factor activity
def: "An enzyme regulator activity that increases the processivity of polymerization by DNA polymerase, by allowing the polymerase to move rapidly along DNA while remaining topologically bound to it." [GOC:mah, PMID:7903401, PMID:8087839]
GO:0030338
name: CMP-N-acetylneuraminate monooxygenase activity
def: "Catalysis of the reaction: CMP-N-acetylneuraminate + NADPH + H+ + O2 = CMP-N-glycoloylneuraminate + NADP+ + H2O." [EC:1.14.18.2]
GO:0030339
name: fatty-acyl-ethyl-ester synthase activity
def: "Catalysis of the reaction: a long-chain-acyl ethyl ester + H2O = a long-chain carboxylic acid + ethanol." [EC:3.1.1.67]
GO:0030340
name: hyaluronate lyase activity
def: "Catalysis of the reaction: hyaluronate = n 3-(4-deoxy-beta-D-gluc-4-enuronosyl)-N-acetyl-D-glucosamine." [EC:4.2.2.1]
GO:0030341
name: chondroitin AC lyase activity
def: "Catalysis of the eliminative degradation of polysaccharides containing 1,4-beta-D-hexosaminyl and 1,3-beta-D-glucuronosyl linkages to disaccharides containing 4-deoxy-beta-D-gluc-4-enuronosyl groups." [EC:4.2.2.5]
GO:0030342
name: 1-alpha,25-dihydroxyvitamin D3 24-hydroxylase activity
def: "Catalysis of the hydroxylation of C-24 of 1-alpha,25-hydroxycholecalciferol (25-hydroxyvitamin D3; calcitriol)." [PMID:8506296]
GO:0030343
name: vitamin D3 25-hydroxylase activity
def: "Catalysis of the reaction: vitamin D3 + NADPH + H+ + O2 = calcidiol + NADP+ + H2O." [ISBN:0471331309, MetaCyc:RXN-9829]
GO:0030345
name: structural constituent of tooth enamel
def: "The action of a molecule that contributes to the structural integrity of tooth enamel." [GOC:mah]
GO:0030346
name: protein phosphatase 2B binding
def: "Interacting selectively and non-covalently with the enzyme protein phosphatase 2B." [GOC:jl]
GO:0030348
name: syntaxin-3 binding
def: "Interacting selectively and non-covalently with the SNAP receptor syntaxin-3." [GOC:ai]
GO:0030350
name: iron-responsive element binding
def: "Interacting selectively and non-covalently with the iron-responsive element, a regulatory sequence found in the 5'- and 3'-untranslated regions of mRNAs encoding many iron-binding proteins." [PMID:3198610, PMID:8710843]
GO:0030351
name: inositol-1,3,4,5,6-pentakisphosphate 3-phosphatase activity
def: "Catalysis of the reaction: inositol-1,3,4,5,6-pentakisphosphate + H2O = inositol-1,4,5,6-tetrakisphosphate + phosphate." [GOC:ai]
GO:0030352
name: inositol-1,4,5,6-tetrakisphosphate 6-phosphatase activity
def: "Catalysis of the reaction: inositol-1,4,5,6-tetrakisphosphate + H2O = inositol-1,4,5-trisphosphate + phosphate." [GOC:ai]
GO:0030353
name: fibroblast growth factor receptor antagonist activity
def: "Interacts with the fibroblast growth factor receptor to reduce the action of another ligand, the agonist." [GOC:mah]
GO:0030354
name: melanin-concentrating hormone activity
def: "The action characteristic of melanin-concentrating hormone, a cyclic peptide hormone that, upon receptor binding, induces melanin aggregation in melanocytes, and is also involved in regulating food intake and energy balance in mammals." [GOC:mah, PMID:11416225, PMID:9792536]
GO:0030355
name: small nucleolar ribonucleoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0030356
name: small cytoplasmic ribonucleoprotein
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0030359
name: protein phosphatase type 2B regulator activity
def: "Modulation of the activity of the enzyme protein phosphatase type 2B." [GOC:ai]
GO:0030362
name: protein phosphatase type 4 regulator activity
def: "Modulation of the activity of the enzyme protein phosphatase type 4." [GOC:ai]
GO:0030363
name: pre-mRNA cleavage factor activity
def: "OBSOLETE. Any activity required for the process of mRNA cleavage." [GOC:mah, PMID:10357856]
GO:0030364
name: cleavage and polyadenylylation specificity factor activity
def: "OBSOLETE. A factor required in mRNA 3' end processing for both the cleavage and poly(A) addition reactions and, consistent with this function, recognizes AAUAAA, a signal also essential for both reactions." [PMID:10357856]
GO:0030365
name: cleavage stimulation factor activity
def: "OBSOLETE. A factor is necessary for cleavage but not for poly(A) addition in mRNA 3' end processing; can stimulate poly(A) addition on substrates with a CstF binding site upstream of the AAUAAA hexanucleotide." [PMID:10357856]
GO:0030366
name: molybdopterin synthase activity
def: "Catalysis of the conversion of precursor Z to molybdopterin, the final step in molybdopterin biosynthesis." [PMID:18154309, PMID:8514783]
GO:0030367
name: interleukin-17 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-17 receptor." [GOC:ai]
GO:0030368
name: interleukin-17 receptor activity
def: "Combining with any member of the interleukin-17 family of cytokines and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:jl, GOC:signaling]
GO:0030369
name: ICAM-3 receptor activity
def: "Combining with ICAM-3, intercellular adhesion molecule 3, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. ICAM-3, or CD50, are constitutively expressed on monocytes, granulocytes and lymphocytes; on physiological stimulation, they become transiently phosphorylated on serine residues." [GOC:ai, GOC:signaling, ISBN:0198506732, PMID:7515813]
GO:0030370
name: intercellular adhesion molecule-3 receptor binding
def: "Interacting selectively and non-covalently with receptors for intercellular adhesion molecule-3 (ICAM-3), such as DC-SIGN and LFA-1." [GOC:ceb, PMID:11473836]
GO:0030371
name: translation repressor activity
def: "Antagonizes ribosome-mediated translation of mRNA into a polypeptide." [GOC:ai, GOC:clt]
GO:0030372
name: high molecular weight B cell growth factor receptor binding
def: "Interacting selectively and non-covalently with the high molecular weight B cell growth factor receptor." [GOC:ai]
GO:0030373
name: high molecular weight B cell growth factor receptor activity
def: "Combining with a high molecular weight B cell growth factor and transmitting the signal to initiate a change in cell activity." [GOC:ai, GOC:signaling, PMID:2681271]
GO:0030374
name: ligand-dependent nuclear receptor transcription coactivator activity
def: "The function of a transcription cofactor that activates transcription in conjuction with a ligand-dependent nuclear receptor from a RNA polymerase II promoter; does not bind DNA itself." [GOC:dph, GOC:tb]
GO:0030375
name: thyroid hormone receptor coactivator activity
def: "The function of a transcription cofactor that activates transcription in conjunction with a thyroid hormone-dependent nuclear receptor from a RNA polymerase II promoter; does not bind DNA itself." [GOC:dph, GOC:tb]
GO:0030377
name: urokinase plasminogen activator receptor activity
def: "Combining with the urokinase plasminogen activator to initiate a change in cell activity." [GOC:mah, PMID:16456079]
GO:0030378
name: serine racemase activity
def: "Catalysis of the synthesis of free D-serine from L-serine." [GOC:kd]
GO:0030379
name: neurotensin receptor activity, non-G-protein coupled
def: "Combining with neurotensin, a neuropeptide active in the central and peripheral nervous system in mammals, and transmitting the signal from one side of the membrane to the other by a mechanism independent of coupling to G proteins." [GOC:mah, GOC:signaling, PMID:9756851]
GO:0030380
name: interleukin-17E receptor binding
def: "Interacting selectively and non-covalently with the interleukin-17E receptor." [GOC:ai]
GO:0030385
name: ferredoxin:thioredoxin reductase activity
def: "Catalysis of the two-electron reduction of the disulfide of thioredoxins with electrons from ferredoxin involving a 4Fe-4S cluster and an adjacent active-site disulfide." [PMID:14769790]
GO:0030387
name: fructosamine-3-kinase activity
def: "Catalysis of the phosphorylation of fructosamine to form fructosamine-3-kinase." [PMID:11016445]
GO:0030395
name: lactose binding
def: "Interacting selectively and non-covalently with lactose, a disaccharide of glucose and galactose, the carbohydrate of milk." [GOC:jl, ISBN:01928006X]
GO:0030400
name: protease substrate recruitment factor activity
def: "OBSOLETE. A recruiting factor or adaptor molecule associated with the proteasome that activates degradation of specific proteasomal substrates and links them to the degradation machinery; it is not involved in ubiquitination; does not possess proteolytic activity." [PMID:11500370]
GO:0030401
name: transcription antiterminator activity
def: "OBSOLETE. Functions to prevent the termination of RNA synthesis. Acts as a regulatory device, e.g. in phage lambda, enabling a terminator to be masked from RNA polymerase so that distal genes can be expressed." [ISBN:0198506732]
GO:0030402
name: matrilysin-2 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0030403
name: collagenase 4 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0030404
name: collagenase 3 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0030405
name: matrix metalloproteinase 19 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0030406
name: matrix metalloproteinase 25 activity
def: "OBSOLETE. Was not defined before being made obsolete." [GOC:mah]
GO:0030407
name: formimidoyltransferase activity
def: "Catalysis of the transfer of a formimino group from 5-formimidoyltetrahydrofolate to an acceptor molecule such as an amino acid." [EC:2.1.2.4, EC:2.1.2.5, GOC:mah]
GO:0030408
name: glycine formimidoyltransferase activity
def: "Catalysis of the reaction: 5-formimidoyltetrahydrofolate + glycine = (6S)-5,6,7,8-tetrahydrofolate + N-formimidoylglycine." [EC:2.1.2.4, RHEA:24291]
GO:0030409
name: glutamate formimidoyltransferase activity
def: "Catalysis of the reaction: 5-formimidoyltetrahydrofolate + L-glutamate = tetrahydrofolate + N-formimidoyl-L-glutamate." [EC:2.1.2.5]
GO:0030410
name: nicotianamine synthase activity
def: "Catalysis of the reaction: 3 S-adenosyl-L-methionine(1+) = 3 S-methyl-5'-thioadenosine + 3 H(+) + nicotianamine." [EC:2.5.1.43, RHEA:16484]
GO:0030411
name: scytalone dehydratase activity
def: "Catalysis of the reaction: scytalone = 1,3,8-trihydroxynaphthalene + H(2)O." [EC:4.2.1.94, RHEA:24399]
GO:0030412
name: formimidoyltetrahydrofolate cyclodeaminase activity
def: "Catalysis of the reaction: 5-formimidoyltetrahydrofolate + 2 H(+) = 5,10-methenyltetrahydrofolate + NH(4)(+)." [EC:4.3.1.4, RHEA:22739]
GO:0030413
name: competence pheromone activity
def: "A small peptide excreted by a naturally transformable bacterium (e.g. Bacillus subtilis) that transmits a signal required for the establishment of competence." [GOC:mah, PMID:7698645]
GO:0030414
name: peptidase inhibitor activity
def: "Stops, prevents or reduces the activity of a peptidase, any enzyme that catalyzes the hydrolysis peptide bonds." [GOC:jl]
GO:0030415
name: carboxypeptidase A inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the enzyme carboxypeptidase A." [GOC:ai]
GO:0030429
name: kynureninase activity
def: "Catalysis of the reaction: L-kynurenine + H2O = anthranilate + L-alanine." [EC:3.7.1.3]
GO:0030487
name: inositol-4,5-bisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: 1D-myo-inositol 4,5-bisphosphate + H2O = 1D-myo-inositol 4-phosphate + phosphate." [GOC:mah]
GO:0030492
name: hemoglobin binding
def: "Interacting selectively and non-covalently with hemoglobin, an oxygen carrying, conjugated protein containing four heme groups and globin." [CHEBI:35143, GOC:jl]
GO:0030504
name: inorganic diphosphate transmembrane transporter activity
def: "Catalysis of the transfer of inorganic diphosphate across a membrane." [PMID:11326272]
GO:0030506
name: ankyrin binding
def: "Interacting selectively and non-covalently with ankyrin, a 200 kDa cytoskeletal protein that attaches other cytoskeletal proteins to integral membrane proteins." [GOC:mah, ISBN:0198506732]
GO:0030507
name: spectrin binding
def: "Interacting selectively and non-covalently with spectrin, a protein that is the major constituent of the erythrocyte cytoskeletal network. It associates with band 4.1 (see band protein) and actin to form the cytoskeletal superstructure of the erythrocyte plasma membrane. It is composed of nonhomologous chains, alpha and beta, which aggregate side-to-side in an antiparallel fashion to form dimers, tetramers, and higher polymers." [GOC:mah, ISBN:0198506732]
GO:0030508
name: thiol-disulfide exchange intermediate activity
def: "OBSOLETE. Functions as an exchange intermediate in thiol-disulfide exchange reactions." [GOC:kd, GOC:mah]
GO:0030515
name: snoRNA binding
def: "Interacting selectively and non-covalently with small nucleolar RNA." [GOC:mah]
GO:0030519
name: snoRNP binding
def: "Interacting selectively and non-covalently with any part of a small nucleolar ribonucleoprotein particle." [GOC:mah]
GO:0030523
name: dihydrolipoamide S-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + dihydrolipoamide = CoA + S-acyldihydrolipoamide." [EC:2.3.1.12, GOC:mah]
GO:0030527
name: structural constituent of chromatin
def: "The action of a molecule that contributes to the structural integrity of chromatin." [GOC:ai]
GO:0030528
name: transcription regulator activity
def: "OBSOLETE. Plays a role in regulating transcription; may bind a promoter or enhancer DNA sequence or interact with a DNA-binding transcription factor." [GOC:mah]
GO:0030533
name: triplet codon-amino acid adaptor activity
def: "The codon binding activity of a tRNA that positions an activated amino acid, mediating its insertion at the correct point in the sequence of a nascent polypeptide chain during protein synthesis." [GOC:hjd, GOC:mtg_MIT_16mar07, ISBN:0198506732]
GO:0030544
name: Hsp70 protein binding
def: "Interacting selectively and non-covalently with Hsp70 proteins, any of a group of heat shock proteins around 70kDa in size." [ISBN:0198506732]
GO:0030545
name: receptor regulator activity
def: "The function of interacting (directly or indirectly) with receptors such that the proportion of receptors in the active form is changed." [GOC:ceb]
GO:0030546
name: receptor activator activity
def: "The function of interacting (directly or indirectly) with receptors such that the proportion of receptors in the active form is increased." [GOC:ceb]
GO:0030547
name: receptor inhibitor activity
def: "The function of interacting (directly or indirectly) with receptors such that the proportion of receptors in the active form is decreased." [GOC:ceb]
GO:0030548
name: acetylcholine receptor regulator activity
def: "Interacting (directly or indirectly) with acetylcholine receptors such that the proportion of receptors in the active form is changed." [GOC:mah]
GO:0030549
name: acetylcholine receptor activator activity
def: "Interacting (directly or indirectly) with acetylcholine receptors such that the proportion of receptors in the active form is increased." [GOC:mah]
GO:0030550
name: acetylcholine receptor inhibitor activity
def: "Interacting (directly or indirectly) with acetylcholine receptors such that the proportion of receptors in the active form is decreased." [GOC:mah]
GO:0030551
name: cyclic nucleotide binding
def: "Interacting selectively and non-covalently with a cyclic nucleotide, a nucleotide in which the phosphate group is in diester linkage to two positions on the sugar residue." [GOC:ai]
GO:0030552
name: cAMP binding
def: "Interacting selectively and non-covalently with cAMP, the nucleotide cyclic AMP (adenosine 3',5'-cyclophosphate)." [GOC:ai]
GO:0030553
name: cGMP binding
def: "Interacting selectively and non-covalently with cGMP, the nucleotide cyclic GMP (guanosine 3',5'-cyclophosphate)." [GOC:ai]
GO:0030554
name: adenyl nucleotide binding
def: "Interacting selectively and non-covalently with adenyl nucleotides, any compound consisting of adenosine esterified with (ortho)phosphate." [ISBN:0198506732]
GO:0030555
name: RNA modification guide activity
def: "Specifies the site of a posttranscriptional modification in an RNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030556
name: rRNA modification guide activity
def: "Specifies the site of a posttranscriptional modification in an rRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030557
name: tRNA modification guide activity
def: "Specifies the site of a posttranscriptional modification in a tRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030558
name: RNA pseudouridylation guide activity
def: "Specifies the site of pseudouridylation in an RNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030559
name: rRNA pseudouridylation guide activity
def: "Specifies the site of pseudouridylation in an rRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030560
name: tRNA pseudouridylation guide activity
def: "Specifies the site of pseudouridylation in a tRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030561
name: RNA 2'-O-ribose methylation guide activity
def: "Specifies the site of 2'-O-ribose methylation in an RNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030562
name: rRNA 2'-O-ribose methylation guide activity
def: "Specifies the site of 2'-O-ribose methylation in an rRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030563
name: snRNA 2'-O-ribose methylation guide activity
def: "Activity that provides specificity to a methylase by using base complementarity to guide site-specific 2'-O-ribose methylations to a small nuclear RNA molecule." [PMID:11733745]
GO:0030564
name: tRNA 2'-O-ribose methylation guide activity
def: "Specifies the site of 2'-O-ribose methylation in a tRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030565
name: snRNA pseudouridylation guide activity
def: "Activity that provides specificity to a pseudouridine synthetase by using base complementarity to guide site-specific pseudouridylations to a small nuclear RNA molecule." [PMID:11733745]
GO:0030566
name: snRNA modification guide activity
def: "Specifies the site of a posttranscriptional modification in an snRNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]
GO:0030567
name: thrombin activator activity
def: "OBSOLETE. Increases the rate of proteolysis catalyzed by thrombin." [GOC:mah]
GO:0030568
name: plasmin inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the serine endopeptidase plasmin." [GOC:mah]
GO:0030569
name: chymotrypsin inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of the serine endopeptidase chymotrypsin." [GOC:mah]
GO:0030570
name: pectate lyase activity
def: "Catalysis of the reaction: a pectate = a pectate + a pectate oligosaccharide with 4-(4-deoxy-alpha-D-galact-4-enuronosyl)-D-galacturonate end. This reaction is the eliminative cleavage of pectate to give oligosaccharides with 4-deoxy-alpha-D-gluc-4-enuronosyl groups at their non-reducing ends." [EC:4.2.2.2]
GO:0030572
name: phosphatidyltransferase activity
def: "Catalysis of the reaction involving the transfer of a phosphatidate (otherwise known as diacylglycerol 3-phosphosphate) group." [GOC:mb]
GO:0030580
name: quinone cofactor methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosylmethionine during the synthesis of quinone cofactors such as ubiquinone (coenzyme Q), menaquinone (vitamin K2), plastoquinone and phylloquinone (vitamin K1)." [GOC:mb]
GO:0030585
name: phosphoenolpyruvate carboxykinase (diphosphate) activity
def: "Catalysis of the reaction: diphosphate + oxaloacetate = CO(2) + phosphate + phosphoenolpyruvate." [EC:4.1.1.38, RHEA:22359]
GO:0030586
name: [methionine synthase] reductase activity
def: "Catalysis of the reaction: [methionine synthase]-cob(II)alamin + NADPH + H+ + S-adenosyl methionine = [methionine synthase]-methylcob(I)alamin + S-adenosylhomocysteine + NADP+." [EC:1.16.1.8]
GO:0030591
name: NAD DNA ADP-ribosyltransferase activity
def: "Catalysis of the transfer of the ADP-ribose group of NAD+ to the amino group at N2 of 2'-deoxyguanosine to yield N2-(alpha-ADP-ribos-1-yl)-2'-deoxyguanosine and its beta form." [PMID:11592983]
GO:0030594
name: neurotransmitter receptor activity
def: "Combining with a neurotransmitter and transmitting the signal to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0030596
name: alpha-L-rhamnosidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing alpha-L-rhamnose residues in alpha-L-rhamnosides." [EC:3.2.1.40]
GO:0030597
name: RNA glycosylase activity
def: "Catalysis of the hydrolysis of N-glycosidic bonds in an RNA molecule." [GOC:mah]
GO:0030598
name: rRNA N-glycosylase activity
def: "Catalysis of the hydrolysis of the N-glycosylic bond at A-4324 in 28S rRNA from rat ribosomes or corresponding sites in 28S RNA from other species." [EC:3.2.2.22, GOC:mah]
GO:0030599
name: pectinesterase activity
def: "Catalysis of the reaction: pectin + n H2O = n methanol + pectate." [EC:3.1.1.11]
GO:0030600
name: feruloyl esterase activity
def: "Catalysis of the reaction: feruloyl-polysaccharide + H2O = ferulate + polysaccharide." [EC:3.1.1.73]
GO:0030601
name: aminopeptidase B activity
def: "OBSOLETE. Catalysis of the release of N-terminal Arg and Lys from oligopeptides when P1' is not Pro. Also acts on arylamides of Arg and Lys." [EC:3.4.11.6]
GO:0030602
name: chymosin activity
def: "OBSOLETE. Catalysis of the lysis of peptide bonds with broad specificity similar to that of pepsin A. Clots milk by cleavage of a single Ser-Phe-l-Met-Ala bond in kappa-casein." [EC:3.4.23.4]
GO:0030603
name: oxaloacetase activity
def: "Catalysis of the reaction: H(2)O + oxaloacetate = acetate + H(+) + oxalate." [EC:3.7.1.1, RHEA:24435]
GO:0030604
name: 1-deoxy-D-xylulose-5-phosphate reductoisomerase activity
def: "Catalysis of the reaction: 2-C-methyl-D-erythritol 4-phosphate + NADP(+) = 1-deoxy-D-xylulose 5-phosphate + H(+) + NADPH." [EC:1.1.1.267, RHEA:13720]
GO:0030611
name: arsenate reductase activity
def: "Catalysis of the interconversion of arsenate and arsenite." [GOC:mah]
GO:0030612
name: arsenate reductase (thioredoxin) activity
def: "Catalysis of the reaction: arsenate + thioredoxin = arsenite + thioredoxin disulfide. Thioredoxin disulfide is also known as oxidized thioredoxin." [MetaCyc:RXN-10737]
GO:0030613
name: oxidoreductase activity, acting on phosphorus or arsenic in donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a phosphorus- or arsenic-containing group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:mah]
GO:0030614
name: oxidoreductase activity, acting on phosphorus or arsenic in donors, disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a phosphorus- or arsenic-containing group acts as a hydrogen or electron donor and reduces a disulfide." [GOC:mah]
GO:0030616
name: transforming growth factor beta receptor, common-partner cytoplasmic mediator activity
def: "A TGF-beta cytoplasmic mediator that forms a complex with a phosphorylated pathway-specific mediator. The heterocomplex translocates to the nucleus to regulate transcription." [GOC:hjd]
GO:0030617
name: transforming growth factor beta receptor, inhibitory cytoplasmic mediator activity
def: "A TGF-beta cytoplasmic mediator that inhibits the signaling function of common-partner and pathway-specific mediators." [PMID:9759503]
GO:0030618
name: transforming growth factor beta receptor, pathway-specific cytoplasmic mediator activity
def: "A TGF-beta cytoplasmic mediator that is phosphorylated by a TGFbeta receptor and complexes with a common-partner mediator. The- heterocomplex translocates to the nucleus to regulate transcription." [GOC:hjd]
GO:0030619
name: U1 snRNA binding
def: "Interacting selectively and non-covalently with the U1 small nuclear RNA (U1 snRNA)." [GOC:mah]
GO:0030620
name: U2 snRNA binding
def: "Interacting selectively and non-covalently with the U2 small nuclear RNA (U2 snRNA)." [GOC:jl]
GO:0030621
name: U4 snRNA binding
def: "Interacting selectively and non-covalently with the U4 small nuclear RNA (U4 snRNA)." [GOC:jl]
GO:0030622
name: U4atac snRNA binding
def: "Interacting selectively and non-covalently with the U4atac small nuclear RNA (U4atac snRNA)." [GOC:jl]
GO:0030623
name: U5 snRNA binding
def: "Interacting selectively and non-covalently with the U5 small nuclear RNA (U5 snRNA)." [GOC:jl]
GO:0030624
name: U6atac snRNA binding
def: "Interacting selectively and non-covalently with the U6atac small nuclear RNA (U6atac snRNA)." [GOC:jl]
GO:0030625
name: U11 snRNA binding
def: "Interacting selectively and non-covalently with the U11 small nuclear RNA (U11 snRNA)." [GOC:jl]
GO:0030626
name: U12 snRNA binding
def: "Interacting selectively and non-covalently with the U12 small nuclear RNA (U12 snRNA)." [GOC:jl]
GO:0030627
name: pre-mRNA 5'-splice site binding
def: "Interacting selectively and non-covalently with the pre-mRNA 5' splice site sequence." [GOC:jl]
GO:0030628
name: pre-mRNA 3'-splice site binding
def: "Interacting selectively and non-covalently with the pre-mRNA 3' splice site sequence." [GOC:jl]
GO:0030629
name: U6 snRNA 3'-end binding
def: "Interacting selectively and non-covalently with the 3' end of the U6 small nuclear RNA (U6 snRNA)." [GOC:mah]
GO:0030674
name: protein binding, bridging
def: "The binding activity of a molecule that brings together two or more protein molecules, or a protein and another macromolecule or complex, through a selective, non-covalent, often stoichiometric interaction, permitting those molecules to function in a coordinated way." [GOC:bf, GOC:mah, GOC:vw]
GO:0030675
name: Rac GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Rac family." [GOC:mah]
GO:0030676
name: Rac guanyl-nucleotide exchange factor activity
def: "Stimulates the exchange of guanyl nucleotides associated with a GTPase of the Rac family. Under normal cellular physiological conditions, the concentration of GTP is higher than that of GDP, favoring the replacement of GDP by GTP in association with the GTPase." [GOC:mah]
GO:0030693
name: caspase activity
def: "OBSOLETE. Catalysis of the hydrolysis of a peptide bond on the carboxyl side of an aspartate residue." [PMID:10872455]
GO:0030695
name: GTPase regulator activity
def: "Modulates the rate of GTP hydrolysis by a GTPase." [GOC:mah]
GO:0030696
name: tRNA (m5U54) methyltransferase activity
def: "Catalysis of the transfer of a methyl group from a donor to the C5 atom of the uridine residue at position 54 in a tRNA molecule." [ISBN:1555811337]
GO:0030697
name: S-adenosylmethionine-dependent tRNA (m5U54) methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing thymine at position U54 of a transfer RNA. This occurs in most Gram-negative bacteria, some archae, and eukaryotes." [GOC:hjd, ISBN:1555811337]
GO:0030698
name: 5,10-methylenetetrahydrofolate-dependent tRNA (m5U54) methyltransferase activity
def: "Catalysis of the transfer of a methyl group from 5,10-methylenetetrahydrofolate to the C5 atom of the uridine residue at position 54 in a tRNA molecule. This occurs in most Gram-positive bacteria and some Gram-negative bacteria." [GOC:hjd, ISBN:1555811337]
GO:0030699
name: glycine reductase activity
def: "Catalysis of the reaction: acetyl phosphate + H(2)O + NH(4)(+) + thioredoxin disulfide = glycine + H(+) + phosphate + thioredoxin." [EC:1.21.4.2, RHEA:12235]
GO:0030701
name: NAD+-dinitrogen-reductase ADP-D-ribosyltransferase activity
def: "Catalysis of the reaction: NAD+ + [dinitrogen reductase] = nicotinamide + ADP-D-ribosyl-[dinitrogen reductase]." [EC:2.4.2.37]
GO:0030729
name: acetoacetate-CoA ligase activity
def: "Catalysis of the reaction: acetoacetate + ATP + CoA = acetoacetyl-CoA + AMP + diphosphate + H(+)." [EC:6.2.1.16, RHEA:16120]
GO:0030731
name: guanidinoacetate N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanidinoacetate = S-adenosyl-L-homocysteine + creatine + H(+)." [EC:2.1.1.2, RHEA:10659]
GO:0030732
name: methionine S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + L-methionine = S-adenosyl-L-homocysteine + S-methyl-L-methionine." [EC:2.1.1.12]
GO:0030733
name: fatty acid O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a fatty acid = S-adenosyl-L-homocysteine + a fatty acid methyl ester." [EC:2.1.1.15]
GO:0030734
name: polysaccharide O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 1,4-N1-D-glucooligosaccharide = S-adenosyl-L-homocysteine + oligosaccharide containing 6-methyl-D-glucose units." [EC:2.1.1.18]
GO:0030735
name: carnosine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + carnosine = S-adenosyl-L-homocysteine + anserine + H(+)." [EC:2.1.1.22, RHEA:14208]
GO:0030736
name: phenol O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phenol = S-adenosyl-L-homocysteine + anisole + H(+)." [EC:2.1.1.25, RHEA:14812]
GO:0030737
name: iodophenol O-methyltransferase activity
def: "Catalysis of the reaction: 2-iodophenol + S-adenosyl-L-methionine = 1-iodo-2-methoxybenzene + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.26, RHEA:14316]
GO:0030738
name: tyramine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tyramine = N-methyltyramine + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.27, RHEA:14868]
GO:0030739
name: O-demethylpuromycin O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + O-demethylpuromycin = S-adenosyl-L-homocysteine + puromycin." [EC:2.1.1.38]
GO:0030740
name: inositol 3-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + myo-inositol = 1D-3-O-methyl-myo-inositol + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.39, RHEA:18880]
GO:0030741
name: inositol 1-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + myo-inositol = 1D-1-O-methyl-myo-inositol + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.40, RHEA:17568]
GO:0030742
name: GTP-dependent protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules) using energy from the hydrolysis of GTP." [GOC:go_curators]
GO:0030743
name: rRNA (adenosine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing a single residue of 2'-O-methyladenosine." [EC:2.1.1.66]
GO:0030744
name: luteolin O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + luteolin = 4',5,7-trihydroxy-3'-methoxyflavone + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.42, RHEA:14592]
GO:0030745
name: dimethylhistidine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + N-alpha,N-alpha-dimethyl-L-histidine = S-adenosyl-L-homocysteine + N-alpha,N-alpha,N-alpha-trimethyl-L-histidine." [EC:2.1.1.44]
GO:0030746
name: isoflavone 4'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + isoflavone = S-adenosyl-L-homocysteine + 4'-O-methylisoflavone." [EC:2.1.1.46]
GO:0030747
name: indolepyruvate C-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (indol-3-yl)pyruvate = S-adenosyl-L-homocysteine + (S)-3-(indol-3-yl)-2-oxobutanoate." [EC:2.1.1.47]
GO:0030748
name: amine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + an amine = S-adenosyl-L-homocysteine + a methylated amine." [EC:2.1.1.49]
GO:0030749
name: loganate O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + loganate = S-adenosyl-L-homocysteine + loganin." [EC:2.1.1.50]
GO:0030750
name: putrescine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + putrescine = N-methylputrescine + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.53, RHEA:15040]
GO:0030751
name: licodione 2'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + licodione = 2'-O-methyllicodione + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.65, RHEA:18524]
GO:0030752
name: 5-hydroxyfuranocoumarin 5-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 5-hydroxyfuranocoumarin = S-adenosyl-L-homocysteine + 5-methoxyfuranocoumarin." [EC:2.1.1.69]
GO:0030753
name: 8-hydroxyfuranocoumarin 8-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + xanthotoxol = S-adenosyl-L-homocysteine + xanthotoxin. Xanthotoxol is also known as 8-hydroxyfuranocoumarin and xanthotoxin as 8-methoxyfuranocoumarin." [EC:2.1.1.70]
GO:0030754
name: apigenin 4'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 5,7,4'-trihydroxyflavone = S-adenosyl-L-homocysteine + 4'-methoxy-5,7-dihydroxyflavone." [EC:2.1.1.75]
GO:0030755
name: quercetin 3-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3,5,7,3',4'-pentahydroxyflavone = S-adenosyl-L-homocysteine + 3-methoxy-5,7,3',4'-tetrahydroxy-flavone." [EC:2.1.1.76]
GO:0030756
name: isoorientin 3'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + isoorientin = S-adenosyl-L-homocysteine + H(+) + isoscoparin." [EC:2.1.1.78, RHEA:24099]
GO:0030757
name: 3-methylquercitin 7-O-methyltransferase activity
def: "Catalysis of the reaction: 3',4',5,7-tetrahydroxy-3-methoxyflavone + S-adenosyl-L-methionine(1+) = 3',4',5-trihydroxy-3,7-dimethoxyflavone + S-adenosyl-L-homocysteine." [EC:2.1.1.82, RHEA:16184]
GO:0030758
name: 3,7-dimethylquercitin 4'-O-methyltransferase activity
def: "Catalysis of the reaction: 3',4',5-trihydroxy-3,7-dimethoxyflavone + S-adenosyl-L-methionine(1+) = 3',5-dihydroxy-3,4',7-trimethoxyflavone + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.83, RHEA:21835]
GO:0030759
name: methylquercetagetin 6-O-methyltransferase activity
def: "Catalysis of the reaction: 3',4',5,6-tetrahydroxy-3,7-dimethoxyflavone + S-adenosyl-L-methionine(1+) = 3',4',5-trihydroxy-3,6,7-trimethoxyflavone + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.84, RHEA:18720]
GO:0030760
name: pyridine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + pyridine = N-methylpyridinium + S-adenosyl-L-homocysteine." [EC:2.1.1.87, RHEA:16896]
GO:0030761
name: 8-hydroxyquercitin 8-O-methyltransferase activity
def: "Catalysis of the reaction: 3,3',4',5,7,8-hexahydroxyflavone + S-adenosyl-L-methionine(1+) = 3,3',4',5,7-pentahydroxy-8-methoxyflavone + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.88, RHEA:16596]
GO:0030762
name: tetrahydrocolumbamine 2-O-methyltransferase activity
def: "Catalysis of the reaction: (S)-tetrahydrocolumbamine + S-adenosyl-L-methionine(1+) = S-adenosyl-L-homocysteine + H(+) + tetrahydropalmatine." [EC:2.1.1.89, RHEA:22539]
GO:0030763
name: isobutyraldoxime O-methyltransferase activity
def: "Catalysis of the reaction: 2-methylpropanal oxime + S-adenosyl-L-methionine = 2-methylpropanal O-methyloxime + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.91, RHEA:10999]
GO:0030766
name: 11-O-demethyl-17-O-deacetylvindoline O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 11-O-demethyl-17-O-deacetylvindoline = S-adenosyl-L-homocysteine + 17-O-deacetylvindoline." [EC:2.1.1.94]
GO:0030767
name: 3-hydroxyanthranilate 4-C-methyltransferase activity
def: "Catalysis of the reaction: 3-hydroxyanthranilate + S-adenosyl-L-methionine = 3-hydroxy-4-methylanthranilate + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.97, RHEA:17836]
GO:0030768
name: 16-methoxy-2,3-dihydro-3-hydroxytabersonine N-methyltransferase activity
def: "Catalysis of the reaction: (3R)-3-hydroxy-16-methoxy-2,3-dihydrotabersonine + S-adenosyl-L-methionine = S-adenosyl-L-homocysteine + deacetoxyvindoline + H(+)." [EC:2.1.1.99, RHEA:11339]
GO:0030769
name: macrocin O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + macrocin = S-adenosyl-L-homocysteine + H(+) + tylosin." [EC:2.1.1.101, RHEA:17272]
GO:0030770
name: demethylmacrocin O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + demethylmacrocin = S-adenosyl-L-homocysteine + H(+) + macrocin." [EC:2.1.1.102, RHEA:17576]
GO:0030771
name: N-benzoyl-4-hydroxyanthranilate 4-O-methyltransferase activity
def: "Catalysis of the reaction: N-benzoyl-4-hydroxyanthranilate + S-adenosyl-L-methionine(1+) = N-benzoyl-4-methoxyanthranilate + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.105, RHEA:17408]
GO:0030772
name: tryptophan 2-C-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + L-tryptophan = S-adenosyl-L-homocysteine + L-2-methyltryptophan + H(+)." [EC:2.1.1.106, RHEA:17324]
GO:0030773
name: 6-hydroxymellein O-methyltransferase activity
def: "Catalysis of the reaction: 6-hydroxymellein + S-adenosyl-L-methionine = 6-methoxymellein + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.108, RHEA:15204]
GO:0030774
name: anthranilate N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + anthranilate = N-methylanthranilate + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.111, RHEA:12183]
GO:0030775
name: glucuronoxylan 4-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + glucuronoxylan D-glucuronate = S-adenosyl-L-homocysteine + glucuronoxylan 4-O-methyl-D-glucuronate." [EC:2.1.1.112]
GO:0030776
name: (RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline = S-adenosyl-L-homocysteine + N-methyl-(RS)-1-benzyl-1,2,3,4-tetrahydroisoquinoline." [EC:2.1.1.115]
GO:0030777
name: (S)-scoulerine 9-O-methyltransferase activity
def: "Catalysis of the reaction: (S)-scoulerine + S-adenosyl-L-methionine(1+) = (S)-tetrahydrocolumbamine + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.117, RHEA:23811]
GO:0030778
name: columbamine O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + columbamine = S-adenosyl-L-homocysteine + H(+) + palmatine." [EC:2.1.1.118, RHEA:15376]
GO:0030779
name: 10-hydroxydihydrosanguinarine 10-O-methyltransferase activity
def: "Catalysis of the reaction: 10-hydroxydihydrosanguinarine + S-adenosyl-L-methionine(1+) = S-adenosyl-L-homocysteine + dihydrochelirubine + H(+)." [EC:2.1.1.119, RHEA:18544]
GO:0030780
name: 12-hydroxydihydrochelirubine 12-O-methyltransferase activity
def: "Catalysis of the reaction: 12-hydroxydihydrochelirubine + S-adenosyl-L-methionine(1+) = S-adenosyl-L-homocysteine + dihydromacarpine + H(+)." [EC:2.1.1.120, RHEA:21095]
GO:0030781
name: 6-O-methylnorlaudanosoline 5'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 6-O-methylnorlaudanosoline = S-adenosyl-L-homocysteine + nororientaline." [EC:2.1.1.121]
GO:0030782
name: (S)-tetrahydroprotoberberine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (S)-7,8,13,14-tetrahydroprotoberberine = S-adenosyl-L-homocysteine + cis-N-methyl-(S)-7,8,13,14-tetrahydroprotoberberine." [EC:2.1.1.122]
GO:0030783
name: [cytochrome c]-methionine S-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + [cytochrome c]-methionine = S-adenosyl-L-homocysteine + [cytochrome c]-S-methyl-methionine." [EC:2.1.1.123]
GO:0030784
name: 3'-hydroxy-N-methyl-(S)-coclaurine 4'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3'-hydroxy-N-methyl-(S)-coclaurine = S-adenosyl-L-homocysteine + (S)-reticuline." [EC:2.1.1.116]
GO:0030785
name: [ribulose-bisphosphate carboxylase]-lysine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + [ribulose-1,5-bisphosphate carboxylase]-lysine = S-adenosyl-L-homocysteine + [ribulose-1,5-bisphosphate carboxylase]-N6-methyl-L-lysine." [EC:2.1.1.127]
GO:0030786
name: (RS)-norcoclaurine 6-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (RS)-norcoclaurine = S-adenosyl-L-homocysteine + (RS)-coclaurine." [EC:2.1.1.128]
GO:0030787
name: inositol 4-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + myo-inositol = 1D-4-O-methyl-myo-inositol + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.129, RHEA:23251]
GO:0030788
name: precorrin-2 C20-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + precorrin-2 = S-adenosyl-L-homocysteine + H(+) + precorrin-3A." [EC:2.1.1.130, RHEA:16844]
GO:0030789
name: precorrin-3B C17-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + precorrin-3B = S-adenosyl-L-homocysteine + precorrin 4." [EC:2.1.1.131]
GO:0030790
name: chlorophenol O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + trichlorophenol = S-adenosyl-L-homocysteine + trichloroanisole." [EC:2.1.1.136]
GO:0030791
name: arsenite methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + arsenite = S-adenosyl-L-homocysteine + methylarsonate." [EC:2.1.1.137]
GO:0030792
name: methylarsonite methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + methylarsonite = S-adenosyl-L-homocysteine + dimethylarsinate." [EC:2.1.1.138]
GO:0030793
name: 3'-demethylstaurosporine O-methyltransferase activity
def: "Catalysis of the reaction: 3'-demethylstaurosporine + S-adenosyl-L-methionine = S-adenosyl-L-homocysteine + H(+) + staurosporine." [EC:2.1.1.139, RHEA:11699]
GO:0030794
name: (S)-coclaurine-N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (S)-coclaurine = S-adenosyl-L-homocysteine + (S)-N-methylcoclaurine." [EC:2.1.1.140]
GO:0030795
name: jasmonate O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + jasmonate = S-adenosyl-L-homocysteine + methyljasmonate." [EC:2.1.1.141]
GO:0030796
name: cycloartenol 24-C-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + cycloartenol = (24R)-24-methylcycloart-25-en-3beta-ol + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.142, RHEA:13140]
GO:0030797
name: 24-methylenesterol C-methyltransferase activity
def: "Catalysis of the reaction: 24-methylidenelophenol + S-adenosyl-L-methionine(1+) = (Z)-24-ethylidenelophenol + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.143, RHEA:21047]
GO:0030798
name: trans-aconitate 2-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + trans-aconitate = (E)-3-(methoxycarbonyl)pent-2-enedioate + S-adenosyl-L-homocysteine." [EC:2.1.1.144, RHEA:14972]
GO:0030848
name: threo-3-hydroxyaspartate ammonia-lyase activity
def: "Catalysis of the reaction: (3S)-3-hydroxy-L-aspartate = NH(4)(+) + oxaloacetate." [EC:4.3.1.16, RHEA:12427]
GO:0030881
name: beta-2-microglobulin binding
def: "Interacting selectively and non-covalently with beta-2-microglobulin." [GOC:mah]
GO:0030882
name: lipid antigen binding
def: "Interacting selectively and non-covalently with a lipid antigen." [PMID:14500461]
GO:0030883
name: endogenous lipid antigen binding
def: "Interacting selectively and non-covalently with an endogenous cellular lipid antigen." [PMID:14500461]
GO:0030884
name: exogenous lipid antigen binding
def: "Interacting selectively and non-covalently with an exogenous lipid antigen (examples include microbial lipids and glycolipids)." [PMID:14500461]
GO:0030898
name: actin-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate. This reaction requires the presence of an actin filament to accelerate release of ADP and phosphate." [GOC:mah]
GO:0030899
name: calcium-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate. This reaction requires the presence of calcium ion (Ca2+)." [GOC:mah]
GO:0030911
name: TPR domain binding
def: "Interacting selectively and non-covalently with a tetratricopeptide repeat (TPR) domain of a protein, the consensus sequence of which is defined by a pattern of small and large hydrophobic amino acids and a structure composed of helices." [GOC:mah]
GO:0030941
name: chloroplast targeting sequence binding
def: "Interacting selectively and non-covalently with a chloroplast targeting sequence, a specific peptide sequence that acts as a signal to localize the protein within the chloroplast." [GOC:mah]
GO:0030942
name: endoplasmic reticulum signal peptide binding
def: "Interacting selectively and non-covalently with an endoplasmic reticulum signal peptide, a specific peptide sequence that acts as a signal to localize the protein within the endoplasmic reticulum." [GOC:mah]
GO:0030943
name: mitochondrion targeting sequence binding
def: "Interacting selectively and non-covalently with a mitochondrion targeting sequence, a specific peptide sequence that acts as a signal to localize the protein within the mitochondrion." [GOC:mah]
GO:0030944
name: DDEL sequence binding
def: "Interacting selectively and non-covalently with a KDEL sequence, the C terminus tetrapeptide sequence Asp-Asp-Glu-Leu found in proteins that are to be retained in the endoplasmic reticulum." [GOC:mah]
GO:0030945
name: protein tyrosine phosphatase activity, via thiol-phosphate intermediate
def: "The catalysis of phosphate removal from a phosphotyrosine using cysteine as a nucleophile and proceed by means of a thiol-phosphate intermediate." [GOC:hjd]
GO:0030946
name: protein tyrosine phosphatase activity, metal-dependent
def: "Catalysis of the reaction: protein tyrosine phosphate + H2O = protein tyrosine + phosphate. This reaction requires metal ions." [GOC:mah]
GO:0030955
name: potassium ion binding
def: "Interacting selectively and non-covalently with potassium (K+) ions." [GOC:mah]
GO:0030957
name: Tat protein binding
def: "Interacting selectively and non-covalently with Tat, a viral transactivating regulatory protein from the human immunodeficiency virus, or the equivalent protein from another virus." [GOC:mah, PMID:9094689]
GO:0030971
name: receptor tyrosine kinase binding
def: "Interacting selectively and non-covalently with a receptor that possesses protein tyrosine kinase activity." [GOC:mah]
GO:0030973
name: molybdate ion binding
def: "Interacting selectively and non-covalently with molybdate (MoO4 2-) ions." [GOC:mlg]
GO:0030975
name: thiamine binding
def: "Interacting selectively and non-covalently with thiamine (vitamin B1), a water soluble vitamin present in fresh vegetables and meats, especially liver." [GOC:mlg]
GO:0030976
name: thiamine pyrophosphate binding
def: "Interacting selectively and non-covalently with thiamine pyrophosphate, the diphosphoric ester of thiamine. Acts as a coenzyme of several (de)carboxylases, transketolases, and alpha-oxoacid dehydrogenases." [CHEBI:45931, GOC:mlg]
GO:0030977
name: taurine binding
def: "Interacting selectively and non-covalently with taurine." [GOC:mlg]
GO:0030983
name: mismatched DNA binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing one or more mismatches." [GOC:mah]
GO:0030984
name: kininogen binding
def: "Interacting selectively and non-covalently with a kininogen, any of a group of plasma proteins that are kinin precursors." [GOC:mah, PMID:9520414]
GO:0030985
name: high molecular weight kininogen binding
def: "Interacting selectively and non-covalently with a kininogen of high molecular mass." [GOC:mah, PMID:9520414]
GO:0030986
name: low molecular weight kininogen binding
def: "Interacting selectively and non-covalently with a kininogen of low molecular mass." [GOC:mah, PMID:9520414]
GO:0030987
name: high molecular weight kininogen receptor binding
def: "Interacting selectively and non-covalently with a high molecular weight kininogen receptor." [GOC:mah]
GO:0031005
name: filamin binding
def: "Interacting selectively and non-covalently with a filamin, any member of a family of high molecular mass cytoskeletal proteins that crosslink actin filaments to form networks and stress fibers. Filamins contain an amino-terminal alpha-actinin-like actin binding domain, which is followed by a rod-domain composed of 4 to 24 100-residue repetitive segments including a carboxy-terminal dimerization domain." [GOC:mah, PMID:11336782]
GO:0031013
name: troponin I binding
def: "Interacting selectively and non-covalently with troponin I, the inhibitory subunit of the troponin complex." [GOC:mah, ISBN:0815316194]
GO:0031014
name: troponin T binding
def: "Interacting selectively and non-covalently with troponin T, the tropomyosin-binding subunit of the troponin complex." [GOC:mah, ISBN:0815316194]
GO:0031071
name: cysteine desulfurase activity
def: "Catalysis of the reaction: L-cysteine + [enzyme]-cysteine = L-alanine + [enzyme]-S-sulfanylcysteine." [EC:2.8.1.7]
GO:0031072
name: heat shock protein binding
def: "Interacting selectively and non-covalently with a heat shock protein, any protein synthesized or activated in response to heat shock." [GOC:mah, GOC:vw]
GO:0031073
name: cholesterol 26-hydroxylase activity
def: "Catalysis of the hydroxylation of cholesterol at position 26 of the side chain, to produce 26-hydroxycholesterol." [GOC:mah, PMID:950499]
GO:0031078
name: histone deacetylase activity (H3-K14 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 14) + H2O = histone H3 L-lysine (position 14) + acetate. This reaction represents the removal of an acetyl group from lysine at position 14 of the histone H3 protein." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0031079
name: picornain 3C activity
def: "OBSOLETE. Catalysis of the selective cleavage of GlnGly bond in the poliovirus polyprotein. In other picornavirus reactions Glu may be substituted for Gln, and Ser or Thr for Gly." [EC:3.4.22.28]
GO:0031127
name: alpha-(1,2)-fucosyltransferase activity
def: "Catalysis of the transfer of an L-fucosyl group from GDP-beta-L-fucose to an acceptor molecule to form an alpha-(1->2) linkage." [GOC:mah]
GO:0031132
name: serine 3-dehydrogenase activity
def: "Catalysis of the reaction: L-serine + NADP(+) = L-alpha-formylglycine + 2 H(+) + NADPH." [EC:1.1.1.276, RHEA:21599]
GO:0031151
name: histone methyltransferase activity (H3-K79 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 79) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 79). This reaction is the addition of a methyl group onto lysine at position 79 of the histone H3 protein." [GOC:mah, PMID:15371351]
GO:0031172
name: ornithine N5-monooxygenase activity
def: "Catalysis of the reaction: L-ornithine + O2 + H+ = N5-hydroxy-L-ornithine + H2O." [MetaCyc:RXN-11128, PMID:12828635]
GO:0031176
name: endo-1,4-beta-xylanase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-xylosidic linkages in xylans." [EC:3.2.1.8]
GO:0031177
name: phosphopantetheine binding
def: "Interacting selectively and non-covalently with phosphopantetheine, the vitamin pantetheine 4'-(dihydrogen phosphate)." [CHEBI:16858, GOC:mah, GOC:vw]
GO:0031208
name: POZ domain binding
def: "Interacting selectively and non-covalently with a POZ (poxvirus and zinc finger) domain of a protein, a protein-protein interaction domain found in many transcription factors." [PMID:7958847]
GO:0031210
name: phosphatidylcholine binding
def: "Interacting selectively and non-covalently with phosphatidylcholine, a class of glycophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of choline." [GOC:mah, ISBN:0198506732]
GO:0031216
name: neopullulanase activity
def: "Catalysis of the hydrolysis of pullulan to panose (6-alpha-D-glucosylmaltose)." [EC:3.2.1.135, GOC:mlg]
GO:0031217
name: glucan 1,4-beta-glucosidase activity
def: "Catalysis of the hydrolysis of (1->4) linkages in (1->4)-beta-D-glucans, to remove successive glucose units." [EC:3.2.1.74, GOC:mlg]
GO:0031218
name: arabinogalactan endo-1,4-beta-galactosidase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-galactosidic linkages in arabinogalactans." [EC:3.2.1.89, GOC:mlg]
GO:0031219
name: levanase activity
def: "Catalysis of the random hydrolysis of 2,6-beta-D-fructofuranosidic linkages in 2,6-beta-D-fructans (levans) containing more than 3 fructose units." [EC:3.2.1.65, GOC:mlg]
GO:0031220
name: maltodextrin phosphorylase activity
def: "Catalysis of the reaction: maltodextrin = glucose-1-phosphate." [GOC:mlg, PMID:10348846]
GO:0031249
name: denatured protein binding
def: "Interacting selectively and non-covalently with denatured proteins." [GOC:mlg]
GO:0031263
name: amine-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + amine(out) = ADP + phosphate + amine(in)." [GOC:mlg]
GO:0031267
name: small GTPase binding
def: "Interacting selectively and non-covalently with a small monomeric GTPase." [GOC:mah]
GO:0031278
name: alpha-1,2-galactosyltransferase activity
def: "Catalysis of the transfer of a galactose residue from a donor molecule, such as GDP-galactose or UDP-galactose, to an oligosaccharide, forming an alpha-1,2-linkage." [PMID:7522655]
GO:0031299
name: taurine-pyruvate aminotransferase activity
def: "Catalysis of the reaction: pyruvate + taurine = L-alanine + sulfoacetaldehyde." [EC:2.6.1.77, RHEA:10423]
GO:0031320
name: hexitol dehydrogenase activity
def: "Catalysis of the reaction: hexitol + acceptor = hexose + reduced acceptor." [GOC:mah]
GO:0031368
name: Pro-X metallocarboxypeptidase activity
def: "OBSOLETE. Catalysis of the cleavage of a Pro-Xaa bond by a metallopeptidase mechanism to release a C-terminal amino acid." [EC:3.4.17.16, GOC:mah]
GO:0031369
name: translation initiation factor binding
def: "Interacting selectively and non-covalently with a translation initiation factor, any polypeptide factor involved in the initiation of ribosome-mediated translation." [GOC:mah]
GO:0031370
name: eukaryotic initiation factor 4G binding
def: "Interacting selectively and non-covalently with eukaryotic initiation factor 4G, a polypeptide factor involved in the initiation of ribosome-mediated translation." [GOC:mah]
GO:0031386
name: protein tag
def: "Acting as an indicator or marker to facilitate recognition by other molecules in the cell. Recognition of the tag, which can be covalently attached to the target molecule, may result in modification, sequestration, transport or degradation of the molecule in question." [GOC:go_curators]
GO:0031402
name: sodium ion binding
def: "Interacting selectively and non-covalently with sodium ions (Na+)." [GOC:mah]
GO:0031403
name: lithium ion binding
def: "Interacting selectively and non-covalently with lithium ions (Li+)." [GOC:mah]
GO:0031404
name: chloride ion binding
def: "Interacting selectively and non-covalently with chloride ions (Cl-)." [GOC:mah]
GO:0031405
name: lipoic acid binding
def: "Interacting selectively and non-covalently with lipoic acid, 1,2-dithiolane-3-pentanoic acid." [GOC:mah, ISBN:0198506732]
GO:0031406
name: carboxylic acid binding
def: "Interacting selectively and non-covalently with a carboxylic acid, any organic acid containing one or more carboxyl (COOH) groups or anions (COO-)." [GOC:mah, ISBN:0198506732]
GO:0031409
name: pigment binding
def: "Interacting selectively and non-covalently with a pigment, any general or particular coloring matter in living organisms, e.g. melanin." [GOC:mah]
GO:0031418
name: L-ascorbic acid binding
def: "Interacting selectively and non-covalently with L-ascorbic acid, (2R)-2-[(1S)-1,2-dihydroxyethyl]-4-hydroxy-5-oxo-2,5-dihydrofuran-3-olate; L-ascorbic acid is vitamin C and has co-factor and anti-oxidant activities in many species." [CHEBI:38290, GOC:mah]
GO:0031419
name: cobalamin binding
def: "Interacting selectively and non-covalently with cobalamin (vitamin B12), a water-soluble vitamin characterized by possession of a corrin nucleus containing a cobalt atom." [GOC:mah]
GO:0031420
name: alkali metal ion binding
def: "Interacting selectively and non-covalently with any alkali metal ion; alkali metals are those elements in group Ia of the periodic table, with the exception of hydrogen." [GOC:mah]
GO:0031423
name: hexon binding
def: "Interacting selectively and non-covalently with a hexon, the major protein component of the icosahedral capsid of an adenovirus." [GOC:mah, PMID:12915569]
GO:0031432
name: titin binding
def: "Interacting selectively and non-covalently with titin, any of a family of giant proteins found in striated and smooth muscle. In striated muscle, single titin molecules span half the sarcomere, with their N- and C-termini in the Z-disc and M-line, respectively." [GOC:mah, PMID:10481174]
GO:0031433
name: telethonin binding
def: "Interacting selectively and non-covalently with telethonin, a protein found in the Z disc of striated muscle and which is a substrate of the titin kinase." [GOC:mah, PMID:10481174]
GO:0031434
name: mitogen-activated protein kinase kinase binding
def: "Interacting selectively and non-covalently with a mitogen-activated protein kinase kinase, any protein that can phosphorylate a MAP kinase." [GOC:mah]
GO:0031435
name: mitogen-activated protein kinase kinase kinase binding
def: "Interacting selectively and non-covalently with a mitogen-activated protein kinase kinase kinase, any protein that can phosphorylate a MAP kinase kinase." [GOC:bf]
GO:0031458
name: betaine-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + a betaine(out) = ADP + phosphate + a betaine(in)." [GOC:mlg]
GO:0031459
name: glycine betaine-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + glycine betaine(out) = ADP + phosphate + glycine betaine(in)." [GOC:mlg]
GO:0031473
name: myosin III binding
def: "Interacting selectively and non-covalently with a class III myosin; myosin III is monomeric and has an N terminal kinase domain." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031489
name: myosin V binding
def: "Interacting selectively and non-covalently with a class V myosin; myosin V is a dimeric molecule involved in intracellular transport." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0031490
name: chromatin DNA binding
def: "Interacting selectively and non-covalently with DNA that is assembled into chromatin." [GOC:mah]
GO:0031491
name: nucleosome binding
def: "Interacting selectively and non-covalently with a nucleosome, a complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA into higher order structures." [GOC:mah]
GO:0031492
name: nucleosomal DNA binding
def: "Interacting selectively and non-covalently with the DNA portion of a nucleosome." [GOC:mah]
GO:0031493
name: nucleosomal histone binding
def: "Interacting selectively and non-covalently with a histone that is assembled into a nucleosome." [GOC:mah]
GO:0031516
name: far-red light photoreceptor activity
def: "The function of absorbing and responding to electromagnetic radiation with a wavelength of approximately 730nm. The response may involve a change in conformation." [GOC:nln]
GO:0031517
name: red light photoreceptor activity
def: "The function of absorbing and responding to electromagnetic radiation with a wavelength of approximately 660nm. The response may involve a change in conformation." [GOC:nln]
GO:0031530
name: gonadotropin-releasing hormone receptor binding
def: "Interacting selectively and non-covalently with a receptor for gonadotropin-releasing hormone (GnRH), a peptide hormone that is synthesized and released by the hypothalamus and is responsible for the release of follicle-stimulating hormone (FSH) and luteinizing hormone (LH) from the anterior pituitary." [GOC:pr, PMID:15196882]
GO:0031531
name: thyrotropin-releasing hormone receptor binding
def: "Interacting selectively and non-covalently with a receptor for thyrotropin-releasing hormone, a tripeptide hormone that is produced by the hypothalamus and stimulates the release of thyroid-stimulating hormone (TSH) and prolactin by the anterior pituitary." [PMID:8592728]
GO:0031543
name: peptidyl-proline dioxygenase activity
def: "Catalysis of the reaction: peptidyl L-proline + 2-oxoglutarate + O2 = peptidyl hydroxy-L-proline + succinate + CO2." [GOC:mah, GOC:vw, PMID:24550447, PMID:24550462]
GO:0031544
name: peptidyl-proline 3-dioxygenase activity
def: "Catalysis of the reaction: peptidyl L-proline + 2-oxoglutarate + O2 = peptidyl trans-3-hydroxy-L-proline + succinate + CO2." [GOC:mah]
GO:0031545
name: peptidyl-proline 4-dioxygenase activity
def: "Catalysis of the reaction: peptidyl L-proline + 2-oxoglutarate + O2 = peptidyl trans-4-hydroxy-L-proline + succinate + CO2." [GOC:mah]
GO:0031546
name: brain-derived neurotrophic factor receptor binding
def: "Interacting selectively and non-covalently with the brain-derived neurotrophic factor receptor." [GOC:mah]
GO:0031559
name: oxidosqualene cyclase activity
def: "Catalysis of the cyclization of (S)-2,3-epoxysqualene to form a triterpene." [GOC:ct]
GO:0031593
name: polyubiquitin binding
def: "Interacting selectively and non-covalently with a polymer of ubiqutin." [GOC:mah]
GO:0031624
name: ubiquitin conjugating enzyme binding
def: "Interacting selectively and non-covalently with a ubiquitin conjugating enzyme, any of the E2 proteins." [GOC:vp]
GO:0031625
name: ubiquitin protein ligase binding
def: "Interacting selectively and non-covalently with a ubiquitin protein ligase enzyme, any of the E3 proteins." [GOC:vp]
GO:0031626
name: beta-endorphin binding
def: "Interacting selectively and non-covalently with beta-endorphin, a peptide generated by the cleavage of pro-opiomelanocortin." [GOC:nln, PMID:6267560]
GO:0031628
name: opioid receptor binding
def: "Interacting selectively and non-covalently with an opioid receptor." [GOC:nln]
GO:0031634
name: replication fork barrier binding
def: "Interacting selectively and non-covalently with replication fork barriers, sites that inhibit the progress of replication forks." [GOC:mah]
GO:0031679
name: NADH dehydrogenase (plastoquinone) activity
def: "Catalysis of the reaction: NADH + H+ + plastoquinone = NAD+ + plastoquinol." [EC:1.6.99.6, GOC:mah]
GO:0031681
name: G-protein beta-subunit binding
def: "Interacting selectively and non-covalently with a G-protein beta subunit." [GOC:mah]
GO:0031682
name: G-protein gamma-subunit binding
def: "Interacting selectively and non-covalently with a G-protein gamma subunit." [GOC:mah]
GO:0031683
name: G-protein beta/gamma-subunit complex binding
def: "Interacting selectively and non-covalently with a complex of G-protein beta/gamma subunits." [GOC:nln, GOC:vw]
GO:0031685
name: adenosine receptor binding
def: "Interacting selectively and non-covalently with an adenosine receptor." [GOC:mah, GOC:nln]
GO:0031686
name: A1 adenosine receptor binding
def: "Interacting selectively and non-covalently with an A1 adenosine receptor." [GOC:mah, GOC:nln]
GO:0031687
name: A2A adenosine receptor binding
def: "Interacting selectively and non-covalently with an A2A adenosine receptor." [GOC:mah, GOC:nln]
GO:0031688
name: A2B adenosine receptor binding
def: "Interacting selectively and non-covalently with an A2B adenosine receptor." [GOC:mah, GOC:nln]
GO:0031689
name: A3 adenosine receptor binding
def: "Interacting selectively and non-covalently with an A3 adenosine receptor." [GOC:mah, GOC:nln]
GO:0031690
name: adrenergic receptor binding
def: "Interacting selectively and non-covalently with an adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031691
name: alpha-1A adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-1A adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031692
name: alpha-1B adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-1B adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031693
name: alpha-1D adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-1D adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031694
name: alpha-2A adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-2A adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031695
name: alpha-2B adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-2B adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031696
name: alpha-2C adrenergic receptor binding
def: "Interacting selectively and non-covalently with an alpha-2C adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031697
name: beta-1 adrenergic receptor binding
def: "Interacting selectively and non-covalently with a beta-1 adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031698
name: beta-2 adrenergic receptor binding
def: "Interacting selectively and non-covalently with a beta-2 adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031699
name: beta-3 adrenergic receptor binding
def: "Interacting selectively and non-covalently with a beta-3 adrenergic receptor." [GOC:mah, GOC:nln]
GO:0031700
name: adrenomedullin receptor binding
def: "Interacting selectively and non-covalently with an adrenomedullin receptor." [GOC:mah, GOC:nln]
GO:0031701
name: angiotensin receptor binding
def: "Interacting selectively and non-covalently with an angiotensin receptor." [GOC:mah, GOC:nln]
GO:0031702
name: type 1 angiotensin receptor binding
def: "Interacting selectively and non-covalently with a type 1 angiotensin receptor." [GOC:mah, GOC:nln]
GO:0031703
name: type 2 angiotensin receptor binding
def: "Interacting selectively and non-covalently with a type 2 angiotensin receptor." [GOC:mah, GOC:nln]
GO:0031704
name: apelin receptor binding
def: "Interacting selectively and non-covalently with an apelin receptor." [GOC:mah, GOC:nln, GOC:vp, PMID:12787050]
GO:0031705
name: bombesin receptor binding
def: "Interacting selectively and non-covalently with a bombesin receptor." [GOC:mah, GOC:nln]
GO:0031706
name: subtype 3 bombesin receptor binding
def: "Interacting selectively and non-covalently with a subtype 3 bombesin receptor." [GOC:mah, GOC:nln]
GO:0031707
name: endothelin A receptor binding
def: "Interacting selectively and non-covalently with an endothelin A receptor." [GOC:mah, GOC:nln]
GO:0031708
name: endothelin B receptor binding
def: "Interacting selectively and non-covalently with an endothelin B receptor." [GOC:mah, GOC:nln]
GO:0031709
name: gastrin-releasing peptide receptor binding
def: "Interacting selectively and non-covalently with a gastrin-releasing peptide receptor." [GOC:mah, GOC:nln]
GO:0031710
name: neuromedin B receptor binding
def: "Interacting selectively and non-covalently with a neuromedin B receptor." [GOC:mah, GOC:nln]
GO:0031711
name: bradykinin receptor binding
def: "Interacting selectively and non-covalently with a bradykinin receptor." [GOC:mah, GOC:nln]
GO:0031712
name: B1 bradykinin receptor binding
def: "Interacting selectively and non-covalently with a B1 bradykinin receptor." [GOC:mah, GOC:nln]
GO:0031713
name: B2 bradykinin receptor binding
def: "Interacting selectively and non-covalently with a B2 bradykinin receptor." [GOC:mah, GOC:nln]
GO:0031714
name: C5a anaphylatoxin chemotactic receptor binding
def: "Interacting selectively and non-covalently with a C5a anaphylatoxin chemotactic receptor." [GOC:mah, GOC:nln]
GO:0031715
name: C5L2 anaphylatoxin chemotactic receptor binding
def: "Interacting selectively and non-covalently with a C5L2 anaphylatoxin chemotactic receptor." [GOC:mah, GOC:nln]
GO:0031716
name: calcitonin receptor binding
def: "Interacting selectively and non-covalently with a calcitonin receptor." [GOC:mah, GOC:nln]
GO:0031717
name: cannabinoid receptor binding
def: "Interacting selectively and non-covalently with a cannabinoid receptor." [GOC:mah, GOC:nln]
GO:0031718
name: type 1 cannabinoid receptor binding
def: "Interacting selectively and non-covalently with a type 1 cannabinoid receptor." [GOC:mah, GOC:nln]
GO:0031719
name: type 2 cannabinoid receptor binding
def: "Interacting selectively and non-covalently with a type 2 cannabinoid receptor." [GOC:mah, GOC:nln]
GO:0031720
name: haptoglobin binding
def: "Interacting selectively and non-covalently with a haptoglobin, any alpha2 globulin of blood plasma that can combine with free oxyhemoglobin to form a stable complex." [GOC:mah, ISBN:0198506732]
GO:0031721
name: hemoglobin alpha binding
def: "Interacting selectively and non-covalently with a hemoglobin alpha chain." [GOC:mah]
GO:0031722
name: hemoglobin beta binding
def: "Interacting selectively and non-covalently with a hemoglobin beta chain." [GOC:mah]
GO:0031723
name: CXCR4 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CXCR4 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031724
name: CXCR5 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CXCR5 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031725
name: CXCR6 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CXCR6 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031726
name: CCR1 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR1 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031727
name: CCR2 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR2 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031728
name: CCR3 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR3 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031729
name: CCR4 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR4 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031730
name: CCR5 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR5 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031731
name: CCR6 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR6 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031732
name: CCR7 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR7 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031733
name: CCR8 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR8 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031734
name: CCR9 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR9 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031735
name: CCR10 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR10 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031736
name: CCR11 chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR11 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031737
name: CX3C chemokine receptor binding
def: "Interacting selectively and non-covalently with a CX3C chemokine receptor." [GOC:mah, GOC:nln]
GO:0031738
name: XCR1 chemokine receptor binding
def: "Interacting selectively and non-covalently with a XCR1 chemokine receptor." [GOC:mah, GOC:nln]
GO:0031739
name: cholecystokinin receptor binding
def: "Interacting selectively and non-covalently with a cholecystokinin receptor." [GOC:mah, GOC:nln]
GO:0031740
name: type A cholecystokinin receptor binding
def: "Interacting selectively and non-covalently with a type A cholecystokinin receptor." [GOC:mah, GOC:nln]
GO:0031741
name: type B gastrin/cholecystokinin receptor binding
def: "Interacting selectively and non-covalently with a type B gastrin/cholecystokinin receptor." [GOC:mah, GOC:nln]
GO:0031745
name: cysteinyl leukotriene receptor binding
def: "Interacting selectively and non-covalently with a cysteinyl leukotriene receptor." [GOC:mah, GOC:nln]
GO:0031746
name: type 1 cysteinyl leukotriene receptor binding
def: "Interacting selectively and non-covalently with a type 1 cysteinyl leukotriene receptor." [GOC:mah, GOC:nln]
GO:0031747
name: type 2 cysteinyl leukotriene receptor binding
def: "Interacting selectively and non-covalently with a type 2 cysteinyl leukotriene receptor." [GOC:mah, GOC:nln]
GO:0031748
name: D1 dopamine receptor binding
def: "Interacting selectively and non-covalently with a D1 dopamine receptor." [GOC:mah, GOC:nln]
GO:0031749
name: D2 dopamine receptor binding
def: "Interacting selectively and non-covalently with a D2 dopamine receptor." [GOC:mah, GOC:nln]
GO:0031750
name: D3 dopamine receptor binding
def: "Interacting selectively and non-covalently with a D3 dopamine receptor." [GOC:mah, GOC:nln]
GO:0031751
name: D4 dopamine receptor binding
def: "Interacting selectively and non-covalently with a D4 dopamine receptor." [GOC:mah, GOC:nln]
GO:0031752
name: D5 dopamine receptor binding
def: "Interacting selectively and non-covalently with a D5 dopamine receptor." [GOC:mah, GOC:nln]
GO:0031753
name: endothelial differentiation G-protein coupled receptor binding
def: "Interacting selectively and non-covalently with an endothelial differentiation G-protein coupled receptor." [GOC:mah, GOC:nln]
GO:0031754
name: Edg-1 sphingosine 1-phosphate receptor binding
def: "Interacting selectively and non-covalently with an Edg-1 sphingosine 1-phosphate receptor." [GOC:mah, GOC:nln]
GO:0031755
name: Edg-2 lysophosphatidic acid receptor binding
def: "Interacting selectively and non-covalently with an Edg-2 lysophosphatidic acid receptor." [GOC:mah, GOC:nln]
GO:0031756
name: Edg-3 sphingosine 1-phosphate receptor binding
def: "Interacting selectively and non-covalently with an Edg-3 sphingosine 1-phosphate receptor." [GOC:mah, GOC:nln]
GO:0031757
name: Edg-4 lysophosphatidic acid receptor binding
def: "Interacting selectively and non-covalently with an Edg-4 lysophosphatidic acid receptor." [GOC:mah, GOC:nln]
GO:0031758
name: Edg-5 sphingosine 1-phosphate receptor binding
def: "Interacting selectively and non-covalently with an Edg-5 sphingosine 1-phosphate receptor." [GOC:mah, GOC:nln]
GO:0031759
name: Edg-6 sphingosine 1-phosphate receptor binding
def: "Interacting selectively and non-covalently with an Edg-6 sphingosine 1-phosphate receptor." [GOC:mah, GOC:nln]
GO:0031760
name: Edg-7 lysophosphatidic acid receptor binding
def: "Interacting selectively and non-covalently with an Edg-7 lysophosphatidic acid receptor." [GOC:mah, GOC:nln]
GO:0031761
name: fMet-Leu-Phe receptor binding
def: "Interacting selectively and non-covalently with a fMet-Leu-Phe receptor." [GOC:mah, GOC:nln]
GO:0031762
name: follicle-stimulating hormone receptor binding
def: "Interacting selectively and non-covalently with a follicle-stimulating hormone receptor." [GOC:mah, GOC:nln]
GO:0031763
name: galanin receptor binding
def: "Interacting selectively and non-covalently with a galanin receptor." [GOC:mah, GOC:nln]
GO:0031764
name: type 1 galanin receptor binding
def: "Interacting selectively and non-covalently with a type 1 galanin receptor." [GOC:mah, GOC:nln]
GO:0031765
name: type 2 galanin receptor binding
def: "Interacting selectively and non-covalently with a type 2 galanin receptor." [GOC:mah, GOC:nln]
GO:0031766
name: type 3 galanin receptor binding
def: "Interacting selectively and non-covalently with a type 3 galanin receptor." [GOC:mah, GOC:nln]
GO:0031767
name: gastric inhibitory polypeptide receptor binding
def: "Interacting selectively and non-covalently with a gastric inhibitory polypeptide receptor." [GOC:mah, GOC:nln]
GO:0031768
name: ghrelin receptor binding
def: "Interacting selectively and non-covalently with a ghrelin receptor." [GOC:mah, GOC:nln]
GO:0031769
name: glucagon receptor binding
def: "Interacting selectively and non-covalently with a glucagon receptor." [GOC:mah, GOC:nln]
GO:0031770
name: growth hormone-releasing hormone receptor binding
def: "Interacting selectively and non-covalently with a growth hormone-releasing hormone receptor." [GOC:mah, GOC:nln]
GO:0031771
name: type 1 hypocretin receptor binding
def: "Interacting selectively and non-covalently with a type 1 hypocretin receptor." [GOC:mah, GOC:nln]
GO:0031772
name: type 2 hypocretin receptor binding
def: "Interacting selectively and non-covalently with a type 2 hypocretin receptor." [GOC:mah, GOC:nln]
GO:0031773
name: kisspeptin receptor binding
def: "Interacting selectively and non-covalently with a kisspeptin receptor." [GOC:mah, GOC:nln]
GO:0031774
name: leukotriene receptor binding
def: "Interacting selectively and non-covalently with a leukotriene receptor." [GOC:mah, GOC:nln]
GO:0031775
name: lutropin-choriogonadotropic hormone receptor binding
def: "Interacting selectively and non-covalently with a lutropin-choriogonadotropic hormone receptor." [GOC:mah, GOC:nln]
GO:0031776
name: melanin-concentrating hormone receptor binding
def: "Interacting selectively and non-covalently with a melanin-concentrating hormone receptor." [GOC:mah, GOC:nln]
GO:0031777
name: type 1 melanin-concentrating hormone receptor binding
def: "Interacting selectively and non-covalently with a type 1 melanin-concentrating hormone receptor." [GOC:mah, GOC:nln]
GO:0031778
name: type 2 melanin-concentrating hormone receptor binding
def: "Interacting selectively and non-covalently with a type 2 melanin-concentrating hormone receptor." [GOC:mah, GOC:nln]
GO:0031779
name: melanocortin receptor binding
def: "Interacting selectively and non-covalently with a melanocortin receptor." [GOC:mah, GOC:nln]
GO:0031780
name: corticotropin hormone receptor binding
def: "Interacting selectively and non-covalently with a corticotropin hormone receptor." [GOC:dph, GOC:mah, GOC:nln]
GO:0031781
name: type 3 melanocortin receptor binding
def: "Interacting selectively and non-covalently with a type 3 melanocortin receptor." [GOC:mah, GOC:nln]
GO:0031782
name: type 4 melanocortin receptor binding
def: "Interacting selectively and non-covalently with a type 4 melanocortin receptor." [GOC:mah, GOC:nln]
GO:0031783
name: type 5 melanocortin receptor binding
def: "Interacting selectively and non-covalently with a type 5 melanocortin receptor." [GOC:mah, GOC:nln]
GO:0031784
name: melatonin receptor binding
def: "Interacting selectively and non-covalently with a melatonin receptor." [GOC:mah, GOC:nln]
GO:0031785
name: type 1A melatonin receptor binding
def: "Interacting selectively and non-covalently with a type 1A melatonin receptor." [GOC:mah, GOC:nln]
GO:0031786
name: type 1B melatonin receptor binding
def: "Interacting selectively and non-covalently with a type 1B melatonin receptor." [GOC:mah, GOC:nln]
GO:0031787
name: H9 melatonin receptor binding
def: "Interacting selectively and non-covalently with a H9 melatonin receptor." [GOC:mah, GOC:nln]
GO:0031788
name: motilin receptor binding
def: "Interacting selectively and non-covalently with a motilin receptor." [GOC:mah, GOC:nln]
GO:0031789
name: G-protein coupled acetylcholine receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled acetylcholine receptor." [GOC:bf, GOC:mah, GOC:nln]
GO:0031795
name: G-protein coupled GABA receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled (metabotropic) GABA receptor." [GOC:mah, GOC:nln]
GO:0031796
name: type 1 metabotropic GABA receptor binding
def: "Interacting selectively and non-covalently with a type 1 metabotropic GABA receptor." [GOC:mah, GOC:nln]
GO:0031797
name: type 2 metabotropic GABA receptor binding
def: "Interacting selectively and non-covalently with a type 2 metabotropic GABA receptor." [GOC:mah, GOC:nln]
GO:0031798
name: type 1 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 1 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031799
name: type 2 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 2 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031800
name: type 3 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 3 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031801
name: type 4 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 4 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031802
name: type 5 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 5 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031803
name: type 6 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 6 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031804
name: type 7 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 7 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031805
name: type 8 metabotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with a type 8 metabotropic glutamate receptor." [GOC:mah, GOC:nln]
GO:0031806
name: G-protein coupled histamine receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled (metabotropic) histamine receptor." [GOC:mah, GOC:nln, PMID:12679144]
GO:0031807
name: H1 histamine receptor binding
def: "Interacting selectively and non-covalently with a H1 histamine receptor." [GOC:mah, GOC:nln]
GO:0031808
name: H2 histamine receptor binding
def: "Interacting selectively and non-covalently with a H2 histamine receptor." [GOC:mah, GOC:nln]
GO:0031809
name: H3 histamine receptor binding
def: "Interacting selectively and non-covalently with a H3 histamine receptor." [GOC:mah, GOC:nln]
GO:0031810
name: H4 histamine receptor binding
def: "Interacting selectively and non-covalently with a H4 histamine receptor." [GOC:mah, GOC:nln]
GO:0031811
name: G-protein coupled nucleotide receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled (metabotropic) nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031812
name: P2Y1 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y1 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031813
name: P2Y2 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y2 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031814
name: P2Y4 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y4 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031815
name: P2Y5 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y5 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031816
name: P2Y6 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y6 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031817
name: P2Y8 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y8 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031818
name: P2Y9 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y9 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031819
name: P2Y10 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y10 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031820
name: P2Y11 nucleotide receptor binding
def: "Interacting selectively and non-covalently with a P2Y11 nucleotide receptor." [GOC:mah, GOC:nln]
GO:0031821
name: G-protein coupled serotonin receptor binding
def: "Interacting selectively and non-covalently with a metabotropic serotonin receptor." [GOC:mah, GOC:nln]
GO:0031822
name: type 1B serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 1B serotonin receptor." [GOC:mah, GOC:nln]
GO:0031823
name: type 1D serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 1D serotonin receptor." [GOC:mah, GOC:nln]
GO:0031824
name: type 1E serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 1E serotonin receptor." [GOC:mah, GOC:nln]
GO:0031825
name: type 1F serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 1F serotonin receptor." [GOC:mah, GOC:nln]
GO:0031826
name: type 2A serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 2A serotonin receptor." [GOC:mah, GOC:nln]
GO:0031827
name: type 2B serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 2B serotonin receptor." [GOC:mah, GOC:nln]
GO:0031828
name: type 2C serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 2C serotonin receptor." [GOC:mah, GOC:nln]
GO:0031829
name: type 4 serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 4 serotonin receptor." [GOC:mah, GOC:nln]
GO:0031830
name: type 5A serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 5A serotonin receptor." [GOC:mah, GOC:nln]
GO:0031831
name: type 5B serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 5B serotonin receptor." [GOC:mah, GOC:nln]
GO:0031832
name: type 6 serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 6 serotonin receptor." [GOC:mah, GOC:nln]
GO:0031833
name: type 7 serotonin receptor binding
def: "Interacting selectively and non-covalently with a type 7 serotonin receptor." [GOC:mah, GOC:nln]
GO:0031834
name: neurokinin receptor binding
def: "Interacting selectively and non-covalently with a neurokinin receptor." [GOC:mah, GOC:nln]
GO:0031835
name: substance P receptor binding
def: "Interacting selectively and non-covalently with a substance P receptor." [GOC:mah, GOC:nln]
GO:0031836
name: neuromedin K receptor binding
def: "Interacting selectively and non-covalently with a neuromedin K receptor." [GOC:mah, GOC:nln]
GO:0031837
name: substance K receptor binding
def: "Interacting selectively and non-covalently with a substance K receptor." [GOC:mah, GOC:nln]
GO:0031839
name: type 1 neuromedin U receptor binding
def: "Interacting selectively and non-covalently with a type 1 neuromedin U receptor." [GOC:mah, GOC:nln]
GO:0031840
name: type 2 neuromedin U receptor binding
def: "Interacting selectively and non-covalently with a type 2 neuromedin U receptor." [GOC:mah, GOC:nln]
GO:0031841
name: neuropeptide Y receptor binding
def: "Interacting selectively and non-covalently with a neuropeptide Y receptor." [GOC:mah, GOC:nln]
GO:0031842
name: type 1 neuropeptide Y receptor binding
def: "Interacting selectively and non-covalently with a type 1 neuropeptide Y receptor." [GOC:mah, GOC:nln]
GO:0031843
name: type 2 neuropeptide Y receptor binding
def: "Interacting selectively and non-covalently with a type 2 neuropeptide Y receptor." [GOC:mah, GOC:nln]
GO:0031844
name: type 4 neuropeptide Y receptor binding
def: "Interacting selectively and non-covalently with a type 4 neuropeptide Y receptor." [GOC:mah, GOC:nln]
GO:0031845
name: type 5 neuropeptide Y receptor binding
def: "Interacting selectively and non-covalently with a type 5 neuropeptide Y receptor." [GOC:mah, GOC:nln]
GO:0031846
name: neurotensin receptor binding
def: "Interacting selectively and non-covalently with a neurotensin receptor." [GOC:mah, GOC:nln]
GO:0031847
name: type 1 neurotensin receptor binding
def: "Interacting selectively and non-covalently with a type 1 neurotensin receptor." [GOC:mah, GOC:nln]
GO:0031849
name: olfactory receptor binding
def: "Interacting selectively and non-covalently with an olfactory receptor." [GOC:mah, GOC:nln]
GO:0031850
name: delta-type opioid receptor binding
def: "Interacting selectively and non-covalently with a delta-type opioid receptor." [GOC:mah, GOC:nln]
GO:0031851
name: kappa-type opioid receptor binding
def: "Interacting selectively and non-covalently with a kappa-type opioid receptor." [GOC:mah, GOC:nln]
GO:0031852
name: mu-type opioid receptor binding
def: "Interacting selectively and non-covalently with a mu-type opioid receptor." [GOC:mah, GOC:nln, GOC:sl]
GO:0031853
name: nociceptin receptor binding
def: "Interacting selectively and non-covalently with a nociceptin receptor." [GOC:mah, GOC:nln]
GO:0031854
name: orexigenic neuropeptide QRFP receptor binding
def: "Interacting selectively and non-covalently with an orexigenic neuropeptide QRFP receptor." [GOC:mah, GOC:nln]
GO:0031855
name: oxytocin receptor binding
def: "Interacting selectively and non-covalently with an oxytocin receptor." [GOC:mah, GOC:nln]
GO:0031856
name: parathyroid hormone receptor binding
def: "Interacting selectively and non-covalently with a parathyroid hormone receptor." [GOC:mah, GOC:nln]
GO:0031857
name: type 1 parathyroid hormone receptor binding
def: "Interacting selectively and non-covalently with a type 1 parathyroid hormone receptor." [GOC:mah, GOC:nln]
GO:0031858
name: pituitary adenylate cyclase-activating polypeptide receptor binding
def: "Interacting selectively and non-covalently with a pituitary adenylate cyclase-activating polypeptide receptor." [GOC:mah, GOC:nln]
GO:0031859
name: platelet activating factor receptor binding
def: "Interacting selectively and non-covalently with a platelet activating factor receptor." [GOC:mah, GOC:nln]
GO:0031861
name: prolactin-releasing peptide receptor binding
def: "Interacting selectively and non-covalently with a prolactin-releasing peptide receptor." [GOC:mah, GOC:nln]
GO:0031862
name: prostanoid receptor binding
def: "Interacting selectively and non-covalently with a prostanoid receptor." [GOC:mah, GOC:nln]
GO:0031863
name: prostaglandin D2 receptor binding
def: "Interacting selectively and non-covalently with a prostaglandin D2 receptor." [GOC:mah, GOC:nln]
GO:0031864
name: EP1 subtype prostaglandin E2 receptor binding
def: "Interacting selectively and non-covalently with an EP1 subtype prostaglandin E2 receptor." [GOC:mah, GOC:nln]
GO:0031865
name: EP2 subtype prostaglandin E2 receptor binding
def: "Interacting selectively and non-covalently with an EP2 subtype prostaglandin E2 receptor." [GOC:mah, GOC:nln]
GO:0031866
name: EP3 subtype prostaglandin E2 receptor binding
def: "Interacting selectively and non-covalently with an EP3 subtype prostaglandin E2 receptor." [GOC:mah, GOC:nln]
GO:0031867
name: EP4 subtype prostaglandin E2 receptor binding
def: "Interacting selectively and non-covalently with an EP4 subtype prostaglandin E2 receptor." [GOC:mah, GOC:nln]
GO:0031868
name: prostaglandin F2-alpha receptor binding
def: "Interacting selectively and non-covalently with a prostaglandin F2-alpha receptor." [GOC:mah, GOC:nln]
GO:0031869
name: prostacyclin receptor binding
def: "Interacting selectively and non-covalently with a prostacyclin receptor." [GOC:mah, GOC:nln]
GO:0031870
name: thromboxane A2 receptor binding
def: "Interacting selectively and non-covalently with a thromboxane A2 receptor." [GOC:mah, GOC:nln]
GO:0031871
name: proteinase activated receptor binding
def: "Interacting selectively and non-covalently with a proteinase activated receptor." [GOC:mah, GOC:nln]
GO:0031872
name: type 1 proteinase activated receptor binding
def: "Interacting selectively and non-covalently with a type 1 proteinase activated receptor." [GOC:mah, GOC:nln]
GO:0031873
name: type 2 proteinase activated receptor binding
def: "Interacting selectively and non-covalently with a type 2 proteinase activated receptor." [GOC:mah, GOC:nln]
GO:0031874
name: type 3 proteinase activated receptor binding
def: "Interacting selectively and non-covalently with a type 3 proteinase activated receptor." [GOC:mah, GOC:nln]
GO:0031875
name: type 4 proteinase activated receptor binding
def: "Interacting selectively and non-covalently with a type 4 proteinase activated receptor." [GOC:mah, GOC:nln]
GO:0031876
name: secretin receptor binding
def: "Interacting selectively and non-covalently with a secretin receptor." [GOC:mah, GOC:nln]
GO:0031877
name: somatostatin receptor binding
def: "Interacting selectively and non-covalently with a somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031878
name: type 1 somatostatin receptor binding
def: "Interacting selectively and non-covalently with a type 1 somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031879
name: type 2 somatostatin receptor binding
def: "Interacting selectively and non-covalently with a type 2 somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031880
name: type 3 somatostatin receptor binding
def: "Interacting selectively and non-covalently with a type 3 somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031881
name: type 4 somatostatin receptor binding
def: "Interacting selectively and non-covalently with a type 4 somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031882
name: type 5 somatostatin receptor binding
def: "Interacting selectively and non-covalently with a type 5 somatostatin receptor." [GOC:mah, GOC:nln]
GO:0031883
name: taste receptor binding
def: "Interacting selectively and non-covalently with a taste receptor." [GOC:mah, GOC:nln]
GO:0031884
name: type 1 member 1 taste receptor binding
def: "Interacting selectively and non-covalently with a type 1 member 1 taste receptor." [GOC:mah, GOC:nln]
GO:0031885
name: type 1 member 2 taste receptor binding
def: "Interacting selectively and non-covalently with a type 1 member 2 taste receptor." [GOC:mah, GOC:nln]
GO:0031886
name: type 1 member 3 taste receptor binding
def: "Interacting selectively and non-covalently with a type 1 member 3 taste receptor." [GOC:mah, GOC:nln]
GO:0031889
name: urotensin receptor binding
def: "Interacting selectively and non-covalently with a urotensin receptor." [GOC:mah, GOC:nln]
GO:0031890
name: vasoactive intestinal polypeptide receptor binding
def: "Interacting selectively and non-covalently with a vasoactive intestinal polypeptide receptor." [GOC:mah, GOC:nln]
GO:0031891
name: type 1 vasoactive intestinal polypeptide receptor binding
def: "Interacting selectively and non-covalently with a type 1 vasoactive intestinal polypeptide receptor." [GOC:mah, GOC:nln]
GO:0031892
name: type 2 vasoactive intestinal polypeptide receptor binding
def: "Interacting selectively and non-covalently with a type 2 vasoactive intestinal polypeptide receptor." [GOC:mah, GOC:nln]
GO:0031893
name: vasopressin receptor binding
def: "Interacting selectively and non-covalently with a vasopressin receptor." [GOC:mah, GOC:nln]
GO:0031894
name: V1A vasopressin receptor binding
def: "Interacting selectively and non-covalently with a V1A vasopressin receptor." [GOC:mah, GOC:nln]
GO:0031895
name: V1B vasopressin receptor binding
def: "Interacting selectively and non-covalently with a V1B vasopressin receptor." [GOC:mah, GOC:nln]
GO:0031896
name: V2 vasopressin receptor binding
def: "Interacting selectively and non-covalently with a V2 vasopressin receptor." [GOC:mah, GOC:nln]
GO:0031924
name: vitamin B6 transporter activity
def: "Enables the directed movement of any of the vitamin B6 compounds -- pyridoxal, pyridoxamine and pyridoxine and the active form, pyridoxal phosphate -- into, out of or within a cell, or between cells." [GOC:mah]
GO:0031925
name: pyridoxal transmembrane transporter activity
def: "Enables the directed movement of pyridoxal across a membrane into, out of or within a cell, or between cells. Pyridoxal, 3-hydroxy-5-(hydroxymethyl)-2-methyl-4-pyridinecarboxaldehyde, is one of the vitamin B6 compounds. Pyridoxal, pyridoxamine and pyridoxine are collectively known as vitamin B6, and are efficiently converted to the biologically active form of vitamin B6, pyridoxal phosphate." [GOC:mah]
GO:0031926
name: pyridoxal phosphate transmembrane transporter activity
def: "Enables the directed movement of pyridoxal phosphate into, out of or within a cell, or between cells; pyridoxal phosphate is pyridoxal phosphorylated at the hydroxymethyl group of C-5, and is the active form of vitamin B6." [GOC:mah]
GO:0031927
name: pyridoxamine transmembrane transporter activity
def: "Enables the directed movement of pyridoxamine into, out of or within a cell, or between cells. Pyridoxamine, 4-(aminomethyl)-5-(hydroxymethyl)-2-methylpyridin-3-ol, is one of the vitamin B6 compounds. Pyridoxal, pyridoxamine and pyridoxine are collectively known as vitamin B6, and are efficiently converted to the biologically active form of vitamin B6, pyridoxal phosphate." [GOC:mah]
GO:0031928
name: pyridoxine transmembrane transporter activity
def: "Enables the directed movement of pyridoxine across a membrane into, out of or within a cell, or between cells. Pyridoxine, 2-methyl-3-hydroxy-4,5-bis(hydroxymethyl)pyridine, is one of the vitamin B6 compounds. Pyridoxal, pyridoxamine and pyridoxine are collectively known as vitamin B6, and are efficiently converted to the biologically active form of vitamin B6, pyridoxal phosphate." [GOC:mah]
GO:0031955
name: short-chain fatty acid-CoA ligase activity
def: "Catalysis of the reaction: ATP + a short-chain carboxylic acid + CoA = AMP + diphosphate + an acyl-CoA; short-chain fatty acids are fatty acids with a chain length of less than C6." [CHEBI:26666, EC:6.2.1.3, GOC:mah]
GO:0031956
name: medium-chain fatty acid-CoA ligase activity
def: "Catalysis of the reaction: ATP + a medium-chain carboxylic acid + CoA = AMP + diphosphate + an acyl-CoA; a medium-chain fatty acid is any fatty acid with a chain length of between C6 and C12." [CHEBI:59554, EC:6.2.1.3, GOC:mah]
GO:0031957
name: very long-chain fatty acid-CoA ligase activity
def: "Catalysis of the reaction: ATP + a very-long-chain carboxylic acid + CoA = AMP + diphosphate + an acyl-CoA; a very long-chain fatty acid is a fatty acid which has a chain length greater than C22." [CHEBI:27283, EC:6.2.1.3, GOC:mah]
GO:0031961
name: cortisol receptor binding
def: "Interacting selectively and non-covalently with a cortisol receptor." [GOC:mah, PMID:12511169]
GO:0031962
name: mineralocorticoid receptor binding
def: "Interacting selectively and non-covalently with a mineralocorticoid receptor." [GOC:mah, PMID:12511169]
GO:0031963
name: cortisol receptor activity
def: "Combining with cortisol and transmitting the signal within the cell to trigger a change in cell activity or function." [GOC:mah, PMID:12511169]
GO:0031964
name: beta-alanyl-histamine hydrolase activity
def: "Catalysis of the reaction: N-beta-alanyl histamine + H2O = histamine + beta-alanine." [GOC:rc, PMID:16299587]
GO:0031992
name: energy transducer activity
def: "The biological transducer activity that accepts energy and converts it to another form, often by transfer to another molecule within the cell." [GOC:go_curators]
GO:0031993
name: light transducer activity
def: "Absorbing energy from one or more photons and transferring their energy to another molecule, usually a protein, within the cell." [GOC:mah, GOC:mlg]
GO:0031994
name: insulin-like growth factor I binding
def: "Interacting selectively and non-covalently with insulin-like growth factor I." [GOC:mah]
GO:0031995
name: insulin-like growth factor II binding
def: "Interacting selectively and non-covalently with insulin-like growth factor II." [GOC:mah]
GO:0031996
name: thioesterase binding
def: "Interacting selectively and non-covalently with any thioesterase enzyme." [GOC:dl]
GO:0031997
name: N-terminal myristoylation domain binding
def: "Interacting selectively and non-covalently with the N-terminus of a protein that has the potential to be, or has been, modified by N-terminal myristoylation. Binding affinity is typically altered by myristoylation; for example, N-terminal myristoylation of HIV Nef increases its affinity for calmodulin." [GOC:dl, GOC:jsg, PMID:15632291]
GO:0032001
name: 1,4-alpha-glucan 6-alpha-glucosyltransferase activity
def: "Catalysis of the transfer an alpha-D-glucosyl residue in a (1->4)-alpha-D-glucan to the primary hydroxy group of glucose, free or combined in a (1->4)-alpha-D-glucan." [EC:2.4.1.24]
GO:0032003
name: interleukin-28 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-28 receptor." [GOC:rph]
GO:0032018
name: 2-methylbutanol:NADP oxidoreductase activity
def: "Catalysis of the reaction: 2-methylbutanol + NADP+ = 2-methylbutanal + NADPH + H+." [GOC:mah, PMID:12210903]
GO:0032027
name: myosin light chain binding
def: "Interacting selectively and non-covalently with a light chain of a myosin complex." [GOC:mah]
GO:0032028
name: myosin head/neck binding
def: "Interacting selectively and non-covalently with the head/neck region of a myosin heavy chain." [GOC:mah]
GO:0032029
name: myosin tail binding
def: "Interacting selectively and non-covalently with the tail region of a myosin heavy chain." [GOC:mah]
GO:0032030
name: myosin I light chain binding
def: "Interacting selectively and non-covalently with a light chain of a myosin I complex." [GOC:mah]
GO:0032031
name: myosin I head/neck binding
def: "Interacting selectively and non-covalently with the head/neck region of a myosin I heavy chain." [GOC:mah]
GO:0032032
name: myosin I tail binding
def: "Interacting selectively and non-covalently with the tail region of a myosin I heavy chain." [GOC:mah]
GO:0032033
name: myosin II light chain binding
def: "Interacting selectively and non-covalently with a light chain of a myosin II complex." [GOC:mah]
GO:0032034
name: myosin II head/neck binding
def: "Interacting selectively and non-covalently with the head/neck region of a myosin II heavy chain." [GOC:mah]
GO:0032035
name: myosin II tail binding
def: "Interacting selectively and non-covalently with the tail region of a myosin II heavy chain." [GOC:mah]
GO:0032036
name: myosin heavy chain binding
def: "Interacting selectively and non-covalently with a heavy chain of a myosin complex." [GOC:mah]
GO:0032037
name: myosin I heavy chain binding
def: "Interacting selectively and non-covalently with a heavy chain of a myosin I complex." [GOC:mah]
GO:0032038
name: myosin II heavy chain binding
def: "Interacting selectively and non-covalently with a heavy chain of a myosin II complex." [GOC:mah]
GO:0032041
name: NAD-dependent histone deacetylase activity (H3-K14 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 14) + H2O = histone H3 L-lysine (position 14) + acetate. This reaction requires the presence of NAD, and represents the removal of an acetyl group from lysine at position 14 of the histone H3 protein." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0032050
name: clathrin heavy chain binding
def: "Interacting selectively and non-covalently with a clathrin heavy chain." [GOC:mah]
GO:0032051
name: clathrin light chain binding
def: "Interacting selectively and non-covalently with a clathrin light chain." [GOC:mah]
GO:0032052
name: bile acid binding
def: "Interacting selectively and non-covalently with bile acids, any of a group of steroid carboxylic acids occurring in bile." [GOC:rph]
GO:0032067
name: Type IV site-specific deoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of DNA in a site specific manner. Cleavage is dependent on the presence of a specific recognition site in the DNA which must be modified (e.g. methylated, hydroxymethylated, glucosyl-hydroxymethylated)." [PMID:12654995]
GO:0032089
name: NACHT domain binding
def: "Interacting selectively and non-covalently with a NACHT (NAIP, CIITA, HET-E and TP1) domain. The NACHT domain consists of seven distinct conserved motifs, including an ATP/GTPase specific P-loop, a Mg(2+)-binding site and five more specific motifs." [GOC:rl]
GO:0032090
name: Pyrin domain binding
def: "Interacting selectively and non-covalently with a Pyrin (PAAD/DAPIN) domain, a protein-protein interaction domain that has the same fold as the Death domain." [GOC:rl]
GO:0032093
name: SAM domain binding
def: "Interacting selectively and non-covalently with a SAM (Sterile Alpha Motif) domain, which is a 70-amino acid protein sequence that participates in protein-protein, protein-lipid, and protein-RNA interactions and is conserved from lower to higher eukaryotes." [GOC:mcc, PMID:16337230]
GO:0032115
name: sorbose reductase activity
def: "Catalysis of the reaction: D-glucitol + NADP(+) = L-sorbose + H(+) + NADPH. The reaction may occur, to a minor extent, in the reverse direction." [EC:1.1.1.289, RHEA:14612]
GO:0032129
name: histone deacetylase activity (H3-K9 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 9) + H2O = histone H3 L-lysine (position 9) + acetate. This reaction represents the removal of an acetyl group from lysine at position 9 of the histone H3 protein." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0032131
name: alkylated DNA binding
def: "Interacting selectively and non-covalently with alkylated residues in DNA." [GOC:mah]
GO:0032132
name: O6-alkylguanine-DNA binding
def: "Interacting selectively and non-covalently with O6-alkylguanine adducts in DNA." [GOC:mah, PMID:16679453]
GO:0032135
name: DNA insertion or deletion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing insertions or deletions." [GOC:vk]
GO:0032136
name: adenine/cytosine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing an A/C mispair." [GOC:vk]
GO:0032137
name: guanine/thymine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a G/T mispair." [GOC:vk]
GO:0032138
name: single base insertion or deletion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a single base insertion or deletion." [GOC:vk]
GO:0032139
name: dinucleotide insertion or deletion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a dinucleotide insertion or deletion." [GOC:vk]
GO:0032140
name: single adenine insertion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a single adenine insertion or a deletion that results in an unpaired adenine." [GOC:mah, GOC:vk]
GO:0032141
name: single cytosine insertion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a single cytosine insertion or a deletion that results in an unpaired cytosine." [GOC:mah, GOC:vk]
GO:0032142
name: single guanine insertion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a single guanine insertion or a deletion that results in an unpaired guanine." [GOC:mah, GOC:vk]
GO:0032143
name: single thymine insertion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a single thymine insertion or a deletion that results in an unpaired thymine." [GOC:mah, GOC:vk]
GO:0032145
name: succinate-semialdehyde dehydrogenase binding
def: "Interacting selectively and non-covalently with succinate-semialdehyde dehydrogenase." [GOC:mah]
GO:0032146
name: thiosulfate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + thiosulfate(out) = ADP + phosphate + thiosulfate(in)." [GOC:mlg]
GO:0032181
name: dinucleotide repeat insertion binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a dinucleotide repeat insertion or a deletion resulting in unpaired dinucleotide repeats." [GOC:mah, GOC:vk]
GO:0032182
name: small conjugating protein binding
def: "Interacting selectively and non-covalently with a small conjugating protein such as ubiquitin or a ubiquitin-like protein." [GOC:mah]
GO:0032183
name: SUMO binding
def: "Interacting selectively and non-covalently with the small ubiquitin-like protein SUMO." [GOC:mah]
GO:0032184
name: SUMO polymer binding
def: "Interacting selectively and non-covalently with a polymer of the small ubiquitin-like protein SUMO." [GOC:mah]
GO:0032190
name: acrosin binding
def: "Interacting selectively and non-covalently with acrosin, a protein that is found in the acrosomes of sperm and possesses protease and carbohydrate binding activities." [GOC:mah, PMID:12398221]
GO:0032216
name: glucosaminyl-phosphotidylinositol O-acyltransferase activity
def: "Catalysis of the reaction: glucosaminyl-phosphotidylinositol + fatty acyl-CoA = glucosaminyl-acyl-phosphotidylinositol + CoA." [Reactome:REACT_1049.1]
GO:0032217
name: riboflavin transporter activity
def: "Facilitates the directed movement of riboflavin into, out of or within a cell, or between cells. Riboflavin (vitamin B2) is a water-soluble B-complex vitamin, converted in the cell to FMN and FAD, cofactors required for the function of flavoproteins." [GOC:rn, PMID:16204239]
GO:0032266
name: phosphatidylinositol-3-phosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-3-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' position." [GOC:bf, PMID:10209156, PMID:11395417, PMID:11557775]
GO:0032267
name: tRNA(Ile)-lysidine synthase activity
def: "Catalysis of the ligation of lysine onto the cytidine residue present at the wobble position (usually position 34) of an AUA-specific isoleucine tRNA, to form the derivative lysidine. This modification converts both the codon specificity of tRNA(Ile) from AUG to AUA and its amino acid specificity from methionine to isoleucine." [PMID:14527414]
GO:0032296
name: double-stranded RNA-specific ribonuclease activity
def: "Catalysis of the hydrolysis of phosphodiester bonds in double-stranded RNA molecules." [GOC:mah]
GO:0032356
name: oxidized DNA binding
def: "Interacting selectively and non-covalently with oxidized residues in DNA." [GOC:vk]
GO:0032357
name: oxidized purine DNA binding
def: "Interacting selectively and non-covalently with oxidized purine residues in DNA." [GOC:vk]
GO:0032358
name: oxidized pyrimidine DNA binding
def: "Interacting selectively and non-covalently with oxidized pyrimidine residues in DNA." [GOC:vk]
GO:0032393
name: MHC class I receptor activity
def: "Combining with an MHC class I protein complex to initiate a change in cellular activity. Class I here refers to classical class I molecules." [GOC:add, ISBN:0781735149]
GO:0032394
name: MHC class Ib receptor activity
def: "Combining with an MHC class Ib protein complex and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. Class Ib here refers to non-classical class I molecules, such as those of the CD1 or HLA-E gene families." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0032395
name: MHC class II receptor activity
def: "Combining with an MHC class II protein complex and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, ISBN:0781735149]
GO:0032396
name: inhibitory MHC class I receptor activity
def: "Combining with a MHC class I protein complex to mediate signaling that inhibits activation of a lymphocyte." [GOC:add, PMID:11858820, PMID:9368779, PMID:9597134]
GO:0032397
name: activating MHC class I receptor activity
def: "Combining with a MHC class I protein complex to mediate signaling that activates a lymphocyte." [GOC:add, PMID:11858820, PMID:9597134]
GO:0032399
name: HECT domain binding
def: "Interacting selectively and non-covalently with a HECT, 'Homologous to the E6-AP Carboxyl Terminus', domain of a protein." [GOC:mah, Pfam:PF00632]
GO:0032403
name: protein complex binding
def: "Interacting selectively and non-covalently with any protein complex (a complex of two or more proteins that may include other nonprotein molecules)." [GOC:mah]
GO:0032404
name: mismatch repair complex binding
def: "Interacting selectively and non-covalently with a mismatch repair complex." [GOC:vk]
GO:0032405
name: MutLalpha complex binding
def: "Interacting selectively and non-covalently with the mismatch repair complex MutLalpha." [GOC:vk]
GO:0032406
name: MutLbeta complex binding
def: "Interacting selectively and non-covalently with the mismatch repair complex MutLbeta." [GOC:vk]
GO:0032407
name: MutSalpha complex binding
def: "Interacting selectively and non-covalently with the mismatch repair complex MutSalpha." [GOC:vk]
GO:0032408
name: MutSbeta complex binding
def: "Interacting selectively and non-covalently with the mismatch repair complex MutSbeta." [GOC:vk]
GO:0032422
name: purine-rich negative regulatory element binding
def: "Interacting selectively and non-covalently with a 30-bp purine-rich negative regulatory element; the best characterized such element is found in the first intronic region of the rat cardiac alpha-myosin heavy chain gene, and contains two palindromic high-affinity Ets-binding sites (CTTCCCTGGAAG). The presence of this element restricts expression of the gene containing it to cardiac myocytes." [GOC:mah, PMID:9819411]
GO:0032427
name: GBD domain binding
def: "Interacting selectively and non-covalently with the GTPase protein binding domain (GDB) domain of a protein. The GBD is a short motif, including a minimum region of 16 amino acids, identified in proteins that bind to small GTPases such as Cdc42 and Rac." [GOC:mah, GOC:pg, PMID:9119069]
GO:0032428
name: beta-N-acetylgalactosaminidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing N-acetyl-D-galactosamine residues in N-acetyl-beta-D-galactosaminides." [EC:3.2.1.53]
GO:0032440
name: 2-alkenal reductase [NAD(P)] activity
def: "Catalysis of the reaction: n-alkanal + NAD(P)+ = alk-2-enal + NAD(P)H + H+." [EC:1.3.1.74, PMID:16299173]
GO:0032441
name: pheophorbide a oxygenase activity
def: "Catalysis of the reaction: pheophorbide a + reduced ferredoxin + 2 O2 = red chlorophyll catabolite + oxidized ferredoxin + H2O." [PMID:14657372, PMID:16448336]
GO:0032442
name: phenylcoumaran benzylic ether reductase activity
def: "Catalysis of the NADPH-dependent 7-O-4' reduction of phenylcoumaran lignans to the corresponding diphenols; for example, catalysis of the reaction: dehydrodiconiferyl alcohol + NADPH + H+ = isodihydrodehydrodiconiferyl alcohol + NADP+." [PMID:11030549, PMID:13129921]
GO:0032448
name: DNA hairpin binding
def: "Interacting selectively and non-covalently with DNA containing a hairpin. A hairpin structure forms when a DNA strand folds back on itself and intrachain base pairing occurs between inverted repeat sequences." [GOC:mah, ISBN:0198506732]
GO:0032450
name: maltose alpha-glucosidase activity
def: "Catalysis of the reaction: alpha-maltose + H2O = 2 alpha-D-glucose." [MetaCyc:RXN-2141]
GO:0032451
name: demethylase activity
def: "Catalysis of the removal of a methyl group from a substrate." [GOC:mah]
GO:0032452
name: histone demethylase activity
def: "Catalysis of the removal of a methyl group from a histone." [GOC:mah]
GO:0032453
name: histone demethylase activity (H3-K4 specific)
def: "Catalysis of the removal of a methyl group from lysine at position 4 of the histone H3 protein." [GOC:mah]
GO:0032454
name: histone demethylase activity (H3-K9 specific)
def: "Catalysis of the reaction: histone H3 N6-methyl-L-lysine (position 9) + alpha-ketoglutarate + O2 = succinate + CO2 + formaldehyde + lysine. This reaction is the removal of a methyl group from lysine at position 9 of the histone H3 protein." [PMID:16362057]
GO:0032500
name: muramyl dipeptide binding
def: "Interacting selectively and non-covalently, in a non-covalent manner, with muramyl dipeptide; muramyl dipeptide is derived from peptidoglycan." [GOC:rl]
GO:0032518
name: amino acid-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + amino acid(in) -> ADP + phosphate + amino acid(out)." [GOC:mah]
GO:0032519
name: cysteine-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + cysteine(in) -> ADP + phosphate + cysteine(out)." [GOC:mah]
GO:0032520
name: amino acid-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + amino acid(out) -> ADP + phosphate + amino acid(in)." [GOC:mah]
GO:0032521
name: D-methionine-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-methionine(in) -> ADP + phosphate + D-methionine(out)." [GOC:mah]
GO:0032522
name: D-methionine-importing ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-methionine(out) -> ADP + phosphate + D-methionine(in)." [GOC:mah]
GO:0032523
name: silicon efflux transmembrane transporter activity
def: "Catalysis of the transfer of silicon from the inside of the cell to the outside of the cell across a membrane." [GOC:mah, PMID:17625566]
GO:0032542
name: sulfiredoxin activity
def: "Catalysis of the reaction: peroxiredoxin-(S-hydroxy-S-oxocysteine) + ATP + 2 R-SH = peroxiredoxin-(S-hydroxycysteine) + ADP + phosphate + R-S-S-R." [EC:1.8.98.2, PMID:16102934]
GO:0032546
name: deoxyribonucleoside binding
def: "Interacting selectively and non-covalently with a deoxyribonucleoside, a compound consisting of a purine or pyrimidine nitrogenous base linked to deoxyribose." [GOC:mah]
GO:0032547
name: purine deoxyribonucleoside binding
def: "Interacting selectively and non-covalently with a purine deoxyribonucleoside, a compound consisting of a purine base linked to deoxyribose." [GOC:mah]
GO:0032548
name: pyrimidine deoxyribonucleoside binding
def: "Interacting selectively and non-covalently with a pyrimidine deoxyribonucleoside, a compound consisting of a pyrimidine base linked to deoxyribose." [GOC:mah]
GO:0032549
name: ribonucleoside binding
def: "Interacting selectively and non-covalently with a ribonucleoside, a compound consisting of a purine or pyrimidine nitrogenous base linked to ribose." [GOC:mah]
GO:0032550
name: purine ribonucleoside binding
def: "Interacting selectively and non-covalently with a purine ribonucleoside, a compound consisting of a purine base linked to ribose." [GOC:mah]
GO:0032551
name: pyrimidine ribonucleoside binding
def: "Interacting selectively and non-covalently with a pyrimidine ribonucleoside, a compound consisting of a pyrimidine base linked to ribose." [GOC:mah]
GO:0032552
name: deoxyribonucleotide binding
def: "Interacting selectively and non-covalently with a deoxyribonucleotide, any compound consisting of a deoxyribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the deoxyribose moiety." [GOC:mah]
GO:0032553
name: ribonucleotide binding
def: "Interacting selectively and non-covalently with a ribonucleotide, any compound consisting of a ribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose moiety." [GOC:mah]
GO:0032554
name: purine deoxyribonucleotide binding
def: "Interacting selectively and non-covalently with a purine deoxyribonucleotide, any compound consisting of a purine deoxyribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the deoxyribose moiety." [GOC:mah]
GO:0032555
name: purine ribonucleotide binding
def: "Interacting selectively and non-covalently with a purine ribonucleotide, any compound consisting of a purine ribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose moiety." [GOC:mah]
GO:0032556
name: pyrimidine deoxyribonucleotide binding
def: "Interacting selectively and non-covalently with a pyrimidine deoxyribonucleotide, any compound consisting of a pyrimidine deoxyribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the deoxyribose moiety." [GOC:mah]
GO:0032557
name: pyrimidine ribonucleotide binding
def: "Interacting selectively and non-covalently with a pyrimidine ribonucleotide, any compound consisting of a pyrimidine ribonucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose moiety." [GOC:mah]
GO:0032558
name: adenyl deoxyribonucleotide binding
def: "Interacting selectively and non-covalently with an adenyl deoxyribonucleotide, any compound consisting of adenosine esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the deoxyribose moiety." [GOC:mah]
GO:0032559
name: adenyl ribonucleotide binding
def: "Interacting selectively and non-covalently with an adenyl ribonucleotide, any compound consisting of adenosine esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose moiety." [GOC:mah]
GO:0032560
name: guanyl deoxyribonucleotide binding
def: "Interacting selectively and non-covalently with a guanyl deoxyribonucleotide, any compound consisting of guanosine esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the deoxyribose moiety." [GOC:mah]
GO:0032561
name: guanyl ribonucleotide binding
def: "Interacting selectively and non-covalently with a guanyl ribonucleotide, any compound consisting of guanosine esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose moiety." [GOC:mah]
GO:0032562
name: dAMP binding
def: "Interacting selectively and non-covalently with dAMP, deoxyadenosine monophosphate." [GOC:mah]
GO:0032563
name: dADP binding
def: "Interacting selectively and non-covalently with dADP, deoxyadenosine diphosphate." [GOC:mah]
GO:0032564
name: dATP binding
def: "Interacting selectively and non-covalently with dATP, deoxyadenosine triphosphate." [GOC:mah]
GO:0032565
name: dGMP binding
def: "Interacting selectively and non-covalently with dGMP, deoxyguanosine monophosphate." [GOC:mah]
GO:0032566
name: dGDP binding
def: "Interacting selectively and non-covalently with dGDP, deoxyguanosine diphosphate." [GOC:mah]
GO:0032567
name: dGTP binding
def: "Interacting selectively and non-covalently with dGTP, deoxyguanosine triphosphate." [GOC:mah]
GO:0032574
name: 5'-3' RNA helicase activity
def: "Catalysis of the unwinding of an RNA helix in the direction 5' to 3'." [GOC:jp]
GO:0032575
name: ATP-dependent 5'-3' RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of an RNA helix in the direction 5' to 3'." [GOC:jp]
GO:0032576
name: O-linoleoyltransferase activity
def: "Catalysis of the transfer of a linoleoyl ((9Z,12Z)-octadeca-9,12-dienoyl) group to an oxygen atom on the acceptor molecule." [GOC:cb]
GO:0032577
name: phosphatidylcholine:cardiolipin O-linoleoyltransferase activity
def: "Catalysis of the transfer of a linoleoyl ((9Z,12Z)-octadeca-9,12-dienoyl) group from phosphatidylcholine to an oxygen atom on a cardiolipin molecule." [GOC:cb, GOC:mah]
GO:0032767
name: copper-dependent protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules), in the presence of copper." [GOC:ecd, PMID:16884690]
GO:0032778
name: cobalt-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + cobalt(out) = ADP + phosphate + cobalt(in)." [GOC:mlg, GOC:mtg_transport, ISBN:0815340729]
GO:0032791
name: lead ion binding
def: "Interacting selectively and non-covalently with lead (Pb) ions." [GOC:mah]
GO:0032794
name: GTPase activating protein binding
def: "Interacting selectively and non-covalently with a GTPase activating protein." [GOC:nln]
GO:0032795
name: heterotrimeric G-protein binding
def: "Interacting selectively and non-covalently with a heterotrimeric G-protein." [GOC:nln]
GO:0032810
name: sterol response element binding
def: "Interacting selectively and non-covalently with the sterol response element (SRE), a nonpalindromic sequence found in the promoters of genes involved in lipid metabolism." [GOC:vk, PMID:11994399]
GO:0032813
name: tumor necrosis factor receptor superfamily binding
def: "Interacting selectively and non-covalently with any member of the tumor necrosis factor receptor superfamily." [GOC:add]
GO:0032840
name: intramolecular proline-rich ligand binding
def: "Interacting selectively and non-covalently with a proline-rich region within the same polypeptide." [GOC:pf]
GO:0032841
name: calcitonin binding
def: "Interacting selectively and non-covalently with calcitonin, a peptide hormone responsible for reducing serum calcium levels by inhibiting osteoclastic bone reabsorption and promoting renal calcium excretion. It is synthesized and released by the C cells of the thyroid." [GOC:ecd]
GO:0032843
name: hydroperoxide reductase activity
def: "Catalysis of the reaction: 2 RSH + ROOH = RSSR + ROH + H2O. This reaction is the thiol-dependent conversion of an organic hydroperoxide to the corresponding alcohol." [GOC:mlg, PMID:12540833]
GO:0032866
name: D-xylose:NADP reductase activity
def: "Catalysis of the reaction: D-xylitol + NADP+ = D-xylose + NADPH + H+." [PMID:12724380, PMID:15184173]
GO:0032867
name: L-arabinose:NADP reductase activity
def: "Catalysis of the reaction: L-arabitol + NADP+ = L-arabinose + NADPH + H+." [PMID:12724380, PMID:15184173]
GO:0032896
name: palmitoyl-CoA 9-desaturase activity
def: "Catalysis of the reaction: palmitoyl-CoA + AH2 + O2 = palmitoleic acid (16:1delta9) + A + 2 H2O." [GOC:kmv]
GO:0032931
name: histone acetyltransferase activity (H3-K56 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 56) = CoA + histone H3 N6-acetyl-L-lysine (position 56)." [EC:2.3.1.48]
GO:0032934
name: sterol binding
def: "Interacting selectively and non-covalently with a sterol, any steroid containing a hydroxy group in the 3 position, closely related to cholestan-3-ol." [CHEBI:15889, GOC:mah]
GO:0032935
name: sterol sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of a sterol." [GOC:mah]
GO:0032942
name: inositol tetrakisphosphate 2-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol tetrakisphosphate + ATP = 1D-myo-inositol pentakisphosphate (containing 2-phosphate) + ADP." [GOC:hf]
GO:0032947
name: protein complex scaffold
def: "Functions to provide a physical support for the assembly of a multiprotein complex." [GOC:mah]
GO:0032977
name: membrane insertase activity
def: "Mediates the integration of proteins into a membrane from the inner side of the membrane. Membrane insertases are highly conserved and include the bacterial YidC family, the plant chloroplast Alb3 family, and the fungal and animal mitochondrial Oxa1/Cox18 family." [PMID:14739936]
GO:0033019
name: 5-hydroxyvalerate dehydrogenase activity
def: "Catalysis of the reaction: 5-hydroxyvalerate + NAD+ = 5-oxovalerate + NADH." [GOC:mlg, PMID:12406764]
GO:0033038
name: bitter taste receptor activity
def: "Combining with soluble bitter compounds to initiate a change in cell activity. These receptors are responsible for the sense of bitter taste." [GOC:mah]
GO:0033039
name: salty taste receptor activity
def: "Combining with soluble salty compounds to initiate a change in cell activity. These receptors are responsible for the sense of salty taste." [GOC:mah]
GO:0033040
name: sour taste receptor activity
def: "Combining with soluble sour compounds to initiate a change in cell activity. These receptors are responsible for the sense of sour taste." [GOC:mah]
GO:0033041
name: sweet taste receptor activity
def: "Combining with soluble sweet compounds to initiate a change in cell activity. These receptors are responsible for the sense of sweet taste." [GOC:mah]
GO:0033042
name: umami taste receptor activity
def: "Combining with soluble umami compounds to initiate a change in cell activity. These receptors are responsible for the sense of umami taste, the savory taste of meats and other foods that are rich in glutamates." [GOC:mah]
GO:0033094
name: butane-1,4-diamine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: putrescine + 2-oxoglutarate = L-glutamate + 1-pyrroline + H2O." [EC:2.6.1.82, GOC:mlg]
GO:0033130
name: acetylcholine receptor binding
def: "Interacting selectively and non-covalently with an acetylcholine receptor." [GOC:mah]
GO:0033134
name: ubiquitin activating enzyme binding
def: "Interacting selectively and non-covalently with a ubiquitin activating enzyme, any of the E1 proteins." [GOC:mah]
GO:0033142
name: progesterone receptor binding
def: "Interacting selectively and non-covalently with a progesterone receptor." [GOC:mah]
GO:0033149
name: FFAT motif binding
def: "Interacting selectively and non-covalently with the FFAT motif, a short motif containing diphenylalanine in an acidic tract that targets proteins to the cytosolic surface of the ER and to the nuclear membrane by binding directly to members of the VAP (VAMP-associated protein) protein family." [PMID:12727870, PMID:15455074, PMID:16004875]
GO:0033154
name: oligogalacturonide transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + oligogalacturonide(out) = ADP + phosphate + oligogalacturonide(in)." [GOC:mlg]
GO:0033155
name: oligogalacturonide transmembrane transporter activity
def: "Catalysis of the transfer of oligogalacturonide from one side of the membrane to the other." [GOC:mlg]
GO:0033161
name: mitogen-activated protein kinase kinase kinase kinase binding
def: "Interacting selectively and non-covalently with a mitogen-activated protein kinase kinase kinase kinase, any protein that can phosphorylate a MAP kinase kinase kinase." [GOC:mah]
GO:0033164
name: glycolipid 6-alpha-mannosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-mannosyl residue from GDP-mannose into lipid-linked oligosaccharide, forming an alpha-(1->6)-D-mannosyl-D-mannose linkage." [EC:2.4.1.232, GOC:mcc, PMID:16878994]
GO:0033170
name: protein-DNA loading ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the assembly of a protein or protein complex onto a DNA molecule." [GOC:mah, GOC:vw]
GO:0033171
name: nucleoprotein filament-forming ATPase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + H2O = ADP + phosphate to drive the assembly of proteins such as Rad51p onto single-stranded (ss) DNA to form a helical nucleoprotein filament." [GOC:mah, GOC:vw]
GO:0033187
name: inositol hexakisphosphate 4-kinase or 6-kinase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 4-diphospho-1D-myo-inositol (1,2,3,5,6)pentakisphosphate, and ATP + 1D-myo-inositol hexakisphosphate = ADP + 6-diphospho-1D-myo-inositol (1,2,3,4,5)pentakisphosphate." [PMID:17412958, PMID:17412959]
GO:0033188
name: sphingomyelin synthase activity
def: "Catalysis of the reaction: 1,2-diacyl-sn-glycero-3-phosphocholine + ceramide = 1,2-diacyl-sn-glycerol + sphingomyelin." [EC:2.7.8.27, RHEA:18768]
GO:0033190
name: solanapyrone synthase activity
def: "Catalysis of the cyclization of double bonds in prosolanapyrone II to form (-)-solanapyrone A." [GOC:cb, PMID:9659400]
GO:0033191
name: macrophomate synthase activity
def: "Catalysis of the reaction: a 2-pyrone + oxalacetate = macrophomate." [CHEBI:38228, GOC:cb, PMID:10984474]
GO:0033192
name: calmodulin-dependent protein phosphatase activity
def: "Catalysis of the reaction: protein serine/threonine phosphate + H2O = protein serine/threonine + phosphate, dependent on the presence of calcium-bound calmodulin." [GOC:mah, PMID:15359118]
GO:0033196
name: tryparedoxin peroxidase activity
def: "Catalysis of the reaction: tryparedoxin + hydrogen peroxide = tryparedoxin disulfide + H2O." [GOC:mah]
GO:0033199
name: inositol heptakisphosphate 4-kinase or 6-kinase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 4,5-bisdiphosphoinositol-1D-myoinositol (1,2,3,6)tetrakisphosphate, and ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 5,6-bisdiphosphoinositol-1D-myoinositol (1,2,3,4)tetrakisphosphate." [PMID:17412958]
GO:0033200
name: inositol heptakisphosphate 5-kinase activity
def: "Catalysis of the reaction: ATP + 4-diphospho-1D-myo-inositol (1,2,3,5,6)pentakisphosphate = 4,5-bisdiphosphoinositol-1D-myoinositol (1,2,3,6)tetrakisphosphate, and ATP + 6-diphospho-1D-myo-inositol (1,2,3,4,5)pentakisphosphate = 5,6-bisdiphosphoinositol-1D-myoinositol (1,2,3,4)tetrakisphosphate." [PMID:17412958]
GO:0033201
name: alpha-1,4-glucan synthase activity
def: "Catalysis of the reaction: RDP-glucose + [alpha-D-glucosyl-(1,4)]n = RDP + [alpha-D-glucosyl-(1,4)]n+1, where RDP is ADP or UDP." [PMID:17472966]
GO:0033204
name: ribonuclease P RNA binding
def: "Interacting selectively and non-covalently with the RNA subunit of ribonuclease P." [GOC:pg, PMID:11455963]
GO:0033207
name: beta-1,4-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the transfer of an N-acetylgalactosaminyl residue from UDP-N-acetyl-galactosamine to an acceptor molecule, forming a beta-1,4 linkage." [GOC:mah]
GO:0033208
name: UDP-N-acetylgalactosamine:N-acetylneuraminyl-alpha-2,3-galactosyl-beta-R 1,4-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetylgalactosamine + N-acetylneuraminyl-alpha-2,3-galactosyl-beta-oligosaccharide = UDP + N-acetylgalactosaminyl-N-acetylneuraminyl-alpha-2,3-galactosyl-beta-oligosaccharide." [GOC:mah, PMID:12678917, PMID:16024623]
GO:0033218
name: amide binding
def: "Interacting selectively and non-covalently with an amide, any derivative of an oxoacid in which an acidic hydroxy group has been replaced by an amino or substituted amino group." [CHEBI:32988, GOC:mah]
GO:0033219
name: urea binding
def: "Interacting selectively and non-covalently with urea, a water-soluble carboxamide with the structure H2N-CO-NH2." [CHEBI:16199, GOC:mah, ISBN:0198506732]
GO:0033220
name: amide-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + amide(out) = ADP + phosphate + amide(in)." [GOC:mah]
GO:0033221
name: urea-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + urea(out) = ADP + phosphate + urea(in)." [GOC:mlg]
GO:0033222
name: xylose binding
def: "Interacting selectively and non-covalently with the D- or L-enantiomer of xylose." [CHEBI:18222, GOC:mah]
GO:0033224
name: 2-aminoethylphosphonate transmembrane transporter activity
def: "Enables the directed movement of 2-aminoethylphosphonate into, out of or within a cell, or between cells." [GOC:mlg]
GO:0033225
name: 2-aminoethylphosphonate transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + 2-aminoethylphosphonate(out) = ADP + phosphate + 2-aminoethylphosphonate(in)." [GOC:mlg]
GO:0033226
name: 2-aminoethylphosphonate binding
def: "Interacting selectively and non-covalently with 2-aminoethylphosphonate." [GOC:mlg]
GO:0033229
name: cysteine transmembrane transporter activity
def: "Catalysis of the transfer of cysteine from one side of the membrane to the other." [GOC:mah]
GO:0033230
name: cysteine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + cysteine(out/in) = ADP + phosphate + cysteine(in/out)." [GOC:mlg]
GO:0033232
name: D-methionine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + D-methionine(out/in) = ADP + phosphate + D-methionine(in/out)." [GOC:mlg]
GO:0033236
name: 11-beta-hydroxysteroid dehydrogenase (NAD+) activity
def: "OBSOLETE. Catalysis of the reaction: an 11-beta-hydroxysteroid + NAD+ = an 11-oxosteroid + NADH + H+." [PMID:15761036]
GO:0033237
name: 11-beta-hydroxysteroid dehydrogenase (NADP+) activity
def: "OBSOLETE. Catalysis of the reaction: an 11-beta-hydroxysteroid + NADP+ = an 11-oxosteroid + NADPH + H+." [EC:1.1.1.146]
GO:0033250
name: penicillinase activity
def: "Catalysis of the reaction: a penicillin + H2O = a substituted beta-amino acid derivative of the penicillin." [GOC:mlg]
GO:0033251
name: cephalosporinase activity
def: "Catalysis of the reaction: a cephalosporin + H2O = a substituted beta-amino acid derivative of the cephalosporin." [GOC:mlg]
GO:0033264
name: bontoxilysin activity
def: "OBSOLETE. Catalysis of limited hydrolysis of proteins of the neuroexocytosis apparatus, synaptobrevins, SNAP25 or syntaxin. No detected action on small molecule substrates." [EC:3.4.24.69]
GO:0033265
name: choline binding
def: "Interacting selectively and non-covalently with choline, the amine 2-hydroxy-N,N,N-trimethylethanaminium." [CHEBI:15354, GOC:mlg]
GO:0033266
name: choline-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to directly drive the transport of choline across a membrane." [GOC:mlg]
GO:0033283
name: organic acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + organic acid(out/in) = ADP + phosphate + organic acid(in/out)." [GOC:mlg]
GO:0033284
name: carboxylic acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + carboxylic acid(out/in) = ADP + phosphate + carboxylic acid(in/out)." [GOC:mlg]
GO:0033285
name: monocarboxylic acid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + monocarboxylic acid(out/in) = ADP + phosphate + monocarboxylic acid(in/out)." [GOC:mlg]
GO:0033286
name: ectoine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + ectoine(out/in) = ADP + phosphate + ectoine(in/out)." [GOC:mlg]
GO:0033287
name: hydroxyectoine transmembrane transporter activity
def: "Catalysis of the transfer of hydroxyectoine from one side of the membrane to the other." [GOC:mlg]
GO:0033288
name: hydroxyectoine-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + hydroxyectoine(out/in) = ADP + phosphate + hydroxyectoine(in/out)." [GOC:mlg]
GO:0033293
name: monocarboxylic acid binding
def: "Interacting selectively and non-covalently with a monocarboxylic acid, any organic acid containing one carboxyl (COOH) group or anion (COO-)." [GOC:mah]
GO:0033294
name: ectoine binding
def: "Interacting selectively and non-covalently with ectoine, 1,4,5,6-tetrahydro-2-methyl-4-pyrimidinecarboxylic acid." [GOC:mah]
GO:0033295
name: hydroxyectoine binding
def: "Interacting selectively and non-covalently with hydroxyectoine." [GOC:mlg]
GO:0033296
name: rhamnose binding
def: "Interacting selectively and non-covalently with the D- or L-enantiomer of rhamnose." [CHEBI:26546, GOC:mah]
GO:0033297
name: rhamnose-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + rhamnose(out/in) = ADP + phosphate + rhamnose(in/out)." [GOC:mlg]
GO:0033300
name: dehydroascorbic acid transporter activity
def: "Enables the directed movement of dehydroascorbate, 5-(1,2-dihydroxyethyl)furan-2,3,4(5H)-trione, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore." [CHEBI:17242]
GO:0033328
name: peroxisome membrane targeting sequence binding
def: "Interacting selectively and non-covalently with a peroxisomal membrane targeting sequence, any of several sequences of amino acids within a protein that can act as a signal for the localization of the protein into the peroxisome membrane." [GOC:rb, PMID:15133130, PMID:17020786]
GO:0033392
name: actin homodimerization activity
def: "OBSOLETE. Interacting selectively and non-covalently with an identical actin monomer to form a homodimer." [GOC:mah]
GO:0033401
name: UUU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UUU codon." [GOC:mah]
GO:0033402
name: UUC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UUC codon." [GOC:mah]
GO:0033403
name: UUA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UUA codon." [GOC:mah]
GO:0033404
name: UUG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UUG codon." [GOC:mah]
GO:0033405
name: UCU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UCU codon." [GOC:mah]
GO:0033406
name: UCC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UCC codon." [GOC:mah]
GO:0033407
name: UCA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UCA codon." [GOC:mah]
GO:0033408
name: UCG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UCG codon." [GOC:mah]
GO:0033409
name: UAU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UAU codon." [GOC:mah]
GO:0033410
name: UAC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UAC codon." [GOC:mah]
GO:0033411
name: UAA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UAA codon." [GOC:mah]
GO:0033412
name: UAG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UAG codon." [GOC:mah]
GO:0033413
name: UGU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UGU codon." [GOC:mah]
GO:0033414
name: UGC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UGC codon." [GOC:mah]
GO:0033415
name: UGA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UGA codon." [GOC:mah]
GO:0033416
name: UGG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a UGG codon." [GOC:mah]
GO:0033417
name: CUU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CUU codon." [GOC:mah]
GO:0033418
name: CUC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CUC codon." [GOC:mah]
GO:0033419
name: CUA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CUA codon." [GOC:mah]
GO:0033420
name: CUG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CUG codon." [GOC:mah]
GO:0033421
name: CCU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CCU codon." [GOC:mah]
GO:0033422
name: CCC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CCC codon." [GOC:mah]
GO:0033423
name: CCA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CCA codon." [GOC:mah]
GO:0033424
name: CCG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CCG codon." [GOC:mah]
GO:0033425
name: CAU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CAU codon." [GOC:mah]
GO:0033426
name: CAC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CAC codon." [GOC:mah]
GO:0033427
name: CAA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CAA codon." [GOC:mah]
GO:0033428
name: CAG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CAG codon." [GOC:mah]
GO:0033429
name: CGU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CGU codon." [GOC:mah]
GO:0033430
name: CGC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CGC codon." [GOC:mah]
GO:0033431
name: CGA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CGA codon." [GOC:mah]
GO:0033432
name: CGG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a CGG codon." [GOC:mah]
GO:0033433
name: AUU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AUU codon." [GOC:mah]
GO:0033434
name: AUC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AUC codon." [GOC:mah]
GO:0033435
name: AUA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AUA codon." [GOC:mah]
GO:0033436
name: AUG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AUG codon." [GOC:mah]
GO:0033437
name: ACU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an ACU codon." [GOC:mah]
GO:0033438
name: ACC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an ACC codon." [GOC:mah]
GO:0033439
name: ACA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an ACA codon." [GOC:mah]
GO:0033440
name: ACG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an ACG codon." [GOC:mah]
GO:0033441
name: AAU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AAU codon." [GOC:mah]
GO:0033442
name: AAC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AAC codon." [GOC:mah]
GO:0033443
name: AAA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AAA codon." [GOC:mah]
GO:0033444
name: AAG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AAG codon." [GOC:mah]
GO:0033445
name: AGU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AGU codon." [GOC:mah]
GO:0033446
name: AGC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AGC codon." [GOC:mah]
GO:0033447
name: AGA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AGA codon." [GOC:mah]
GO:0033448
name: AGG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes an AGG codon." [GOC:mah]
GO:0033449
name: GUU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GUU codon." [GOC:mah]
GO:0033450
name: GUC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GUC codon." [GOC:mah]
GO:0033451
name: GUA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GUA codon." [GOC:mah]
GO:0033452
name: GUG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GUG codon." [GOC:mah]
GO:0033453
name: GCU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GCU codon." [GOC:mah]
GO:0033454
name: GCC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GCC codon." [GOC:mah]
GO:0033455
name: GCA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GCA codon." [GOC:mah]
GO:0033456
name: GCG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GCG codon." [GOC:mah]
GO:0033457
name: GAU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GAU codon." [GOC:mah]
GO:0033458
name: GAC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GAC codon." [GOC:mah]
GO:0033459
name: GAA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GAA codon." [GOC:mah]
GO:0033460
name: GAG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GAG codon." [GOC:mah]
GO:0033461
name: GGU codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GGU codon." [GOC:mah]
GO:0033462
name: GGC codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GGC codon." [GOC:mah]
GO:0033463
name: GGA codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GGA codon." [GOC:mah]
GO:0033464
name: GGG codon-amino acid adaptor activity
def: "A triplet codon-amino acid adaptor activity that recognizes a GGG codon." [GOC:mah]
GO:0033549
name: MAP kinase phosphatase activity
def: "Catalysis of the reaction: a phosphorylated MAP kinase + H2O = a MAP kinase + phosphate." [GOC:mah, PMID:12184814, PMID:17208316]
GO:0033550
name: MAP kinase tyrosine phosphatase activity
def: "Catalysis of the reaction: MAP kinase tyrosine phosphate + H2O = MAP kinase tyrosine + phosphate." [GOC:mah]
GO:0033556
name: dolichyl pyrophosphate Man7GlcNAc2 alpha-1,3-glucosyltransferase activity
def: "Catalysis of the addition of a glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Man(7)GlcNAc(2)-PP-Dol." [GOC:mah, PMID:10336995]
GO:0033558
name: protein deacetylase activity
def: "Catalysis of the hydrolysis of an acetyl group or groups from a protein substrate." [GOC:mah]
GO:0033560
name: folate reductase activity
def: "Catalysis of the reaction: 7,8-dihydrofolate + NADP+ = folate + NADPH + H+." [GOC:pde]
GO:0033568
name: lactoferrin receptor activity
def: "Combining with lactoferrin and delivering lactoferrin into the cell via endocytosis. Lactoferrin is an iron-binding glycoprotein which binds ferric iron most efficiently at low pH." [GOC:bf, GOC:mlg, PMID:16261254]
GO:0033569
name: lactoferrin transmembrane transporter activity
def: "Catalysis of the transfer of lactoferrin from one side of a membrane to the other." [GOC:mlg]
GO:0033570
name: transferrin transmembrane transporter activity
def: "Catalysis of the transfer of transferrin from one side of a membrane to the other." [GOC:mlg]
GO:0033592
name: RNA strand annealing activity
def: "Facilitates the base-pairing of complementary single-stranded RNA." [GOC:mah]
GO:0033608
name: formyl-CoA transferase activity
def: "Catalysis of the reaction: formyl-CoA + oxalate = formate + oxalyl-CoA." [EC:2.8.3.16, RHEA:16548]
GO:0033612
name: receptor serine/threonine kinase binding
def: "Interacting selectively and non-covalently with a receptor that possesses protein serine/threonine kinase activity." [GOC:mah]
GO:0033613
name: activating transcription factor binding
def: "Interacting selectively and non-covalently with an activating transcription factor, any protein whose activity is required to initiate or upregulate transcription." [GOC:mah, GOC:txnOH]
GO:0033676
name: double-stranded DNA-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction requires the presence of double-stranded DNA, and it drives another reaction." [GOC:mah]
GO:0033677
name: DNA/RNA helicase activity
def: "Catalysis of the reaction: NTP + H2O = NDP + phosphate; this reaction drives the unwinding of a DNA/RNA duplex, i.e. a double helix in which a strand of DNA pairs with a complementary strand of RNA." [GOC:mah]
GO:0033678
name: 5'-3' DNA/RNA helicase activity
def: "Catalysis of the unwinding of a DNA/RNA duplex in the direction 5' to 3'." [GOC:mah]
GO:0033679
name: 3'-5' DNA/RNA helicase activity
def: "Catalysis of the unwinding of a DNA/RNA duplex in the direction 3' to 5'." [GOC:mah]
GO:0033680
name: ATP-dependent DNA/RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of a DNA/RNA duplex." [GOC:mah]
GO:0033681
name: ATP-dependent 3'-5' DNA/RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of a DNA/RNA duplex in the direction 3' to 5'." [GOC:mah]
GO:0033682
name: ATP-dependent 5'-3' DNA/RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of a DNA/RNA duplex in the direction 5' to 3'." [GOC:mah]
GO:0033691
name: sialic acid binding
def: "Interacting selectively and non-covalently with sialic acid, any of a variety of N- or O- substituted derivatives of neuraminic acid, a nine carbon monosaccharide. Sialic acids often occur in polysaccharides, glycoproteins, and glycolipids in animals and bacteria." [CHEBI:26667, GOC:add, http://www.biology-online.org, ISBN:0721601465]
GO:0033694
name: oxidoreductase activity, acting on the CH-NH group of donors, iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:jl]
GO:0033695
name: oxidoreductase activity, acting on CH or CH2 groups, quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces a quinone or similar acceptor molecule." [EC:1.17.5, GOC:mah]
GO:0033699
name: DNA 5'-adenosine monophosphate hydrolase activity
def: "Catalysis of the reaction: 5'-AMP-DNA + H2O = AMP + DNA; nucleophilic release of a covalently linked adenylate residue from a DNA strand, leaving a 5' phosphate terminus." [GOC:mah, PMID:16547001, PMID:17276982]
GO:0033701
name: dTDP-galactose 6-dehydrogenase activity
def: "Catalysis of the reaction: dTDP-D-galactose + 2 NADP+ + H2O = dTDP-D-galacturonate + 2 NADPH + 2 H+." [EC:1.1.1.186]
GO:0033702
name: (+)-trans-carveol dehydrogenase activity
def: "Catalysis of the reaction: (1R,5S)-carveol + NAD(+) = (S)-carvone + H(+) + NADH." [EC:1.1.1.275, RHEA:14828]
GO:0033703
name: 3beta-hydroxy-5beta-steroid dehydrogenase activity
def: "Catalysis of the reaction: 3beta-hydroxy-5beta-pregnane-20-one + NADP(+) = 5beta-pregnan-3,20-dione + H(+) + NADPH." [EC:1.1.1.277, RHEA:22947]
GO:0033704
name: 3beta-hydroxy-5alpha-steroid dehydrogenase activity
def: "Catalysis of the reaction: 3beta-hydroxy-5alpha-pregnane-20-one + NADP(+) = 5alpha-pregnane-3,20-dione + H(+) + NADPH." [EC:1.1.1.278, RHEA:18140]
GO:0033705
name: GDP-4-dehydro-6-deoxy-D-mannose reductase activity
def: "Catalysis of the reaction: GDP-6-deoxy-D-mannose + NAD(P)+ = GDP-4-dehydro-6-deoxy-D-mannose + NAD(P)H + H+." [EC:1.1.1.281]
GO:0033706
name: quinate/shikimate dehydrogenase activity
def: "OBSOLETE. Catalysis of the reactions: L-quinate + NAD(P)+ = 3-dehydroquinate + NAD(P)H + H+, and shikimate + NAD(P)+ = 3-dehydroshikimate + NAD(P)H + H+." [EC:1.1.1.282]
GO:0033707
name: 3''-deamino-3''-oxonicotianamine reductase activity
def: "Catalysis of the reaction: 2'-deoxymugineic acid + NAD(P)+ = 3''-deamino-3''-oxonicotianamine + NAD(P)H + H+." [EC:1.1.1.285]
GO:0033708
name: isocitrate-homoisocitrate dehydrogenase activity
def: "Catalysis of the reactions: isocitrate + NAD+ = 2-oxoglutarate + CO2 + NADH, and (1R,2S)-1-hydroxybutane-1,2,4-tricarboxylate + NAD+ = 2-oxoadipate + CO2 + NADH + H+." [EC:1.1.1.286]
GO:0033709
name: D-arabinitol dehydrogenase, D-ribulose forming (NADP+) activity
def: "Catalysis of the reaction: D-arabinitol + NADP+ = D-ribulose + NADPH + H+." [EC:1.1.1.287]
GO:0033711
name: 4-phosphoerythronate dehydrogenase activity
def: "Catalysis of the reaction: 4-phospho-D-erythronate + NAD(+) = (R)-3-hydroxy-2-oxo-4-phosphonooxybutanoate + H(+) + NADH." [EC:1.1.1.290, RHEA:18832]
GO:0033712
name: 1,5-anhydro-D-fructose reductase (1,5-anhydro-D-mannitol-forming) activity
def: "Catalysis of the reaction: 1,5-anhydro-D-mannitol + NADP(+) = 1,5-anhydro-D-fructose + H(+) + NADPH." [EC:1.1.1.292, RHEA:24211]
GO:0033713
name: choline:oxygen 1-oxidoreductase activity
def: "Catalysis of the reaction: choline + O2 = betaine aldehyde + hydrogen peroxide." [EC:1.1.3.17]
GO:0033714
name: secondary-alcohol oxidase activity
def: "Catalysis of the reaction: O(2) + secondary alcohol = H(2)O(2) + ketone." [EC:1.1.3.18, RHEA:23183]
GO:0033715
name: nucleoside oxidase activity
def: "Catalysis of the reactions: inosine + O2 = 9-riburonosylhypoxanthine + 2 H2O; (1a) 2 inosine + O2 = 2 5'-dehydroinosine + 2 H2O, and (1b) 2 5'-dehydroinosine + O2 = 2 9-riburonosylhypoxanthine + 2 H2O." [EC:1.1.3.28]
GO:0033716
name: nucleoside oxidase (hydrogen peroxide-forming) activity
def: "Catalysis of the reactions: adenosine + 2 O2 = 9-riburonosyladenine + 2 hydrogen peroxide; (1a) adenosine + O2 = 5'-dehydroadenosine + hydrogen peroxide, and (1b) 5'-dehydroadenosine + O2 = 9-riburonosyladenine + hydrogen peroxide." [EC:1.1.3.39]
GO:0033717
name: gluconate 2-dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: D-gluconate + acceptor = 2-dehydro-D-gluconate + reduced acceptor." [EC:1.1.99.3]
GO:0033718
name: pyranose dehydrogenase (acceptor) activity
def: "Catalysis of the reactions: pyranose + acceptor = 2-dehydropyranose (or 3-dehydropyranose or 2,3-didehydropyranose) + reduced acceptor, and a pyranoside + acceptor = a 3-dehydropyranoside (or 3,4-didehydropyranoside) + reduced acceptor." [EC:1.1.99.29]
GO:0033719
name: 2-oxo-acid reductase activity
def: "Catalysis of the reaction: a (2R)-hydroxy-carboxylate + acceptor = a 2-oxo-carboxylate + reduced acceptor." [EC:1.1.99.30]
GO:0033720
name: (S)-mandelate dehydrogenase activity
def: "Catalysis of the reaction: (S)-2-hydroxy-2-phenylacetate + acceptor = 2-oxo-2-phenylacetate + reduced acceptor." [EC:1.1.99.31]
GO:0033721
name: aldehyde dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: an aldehyde + NADP+ + H2O = an acid + NADPH + H+." [EC:1.2.1.4]
GO:0033722
name: malonate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: 3-oxopropanoate + NAD(P)+ + H2O = malonate + NAD(P)H + H+." [EC:1.2.1.15]
GO:0033723
name: fluoroacetaldehyde dehydrogenase activity
def: "Catalysis of the reaction: fluoroacetaldehyde + NAD+ + H2O = fluoroacetate + NADH + 2 H+." [EC:1.2.1.69]
GO:0033726
name: aldehyde ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: an aldehyde + H2O + 2 oxidized ferredoxin = an acid + 2 H+ + 2 reduced ferredoxin." [EC:1.2.7.5]
GO:0033727
name: aldehyde dehydrogenase (FAD-independent) activity
def: "Catalysis of the reaction: an aldehyde + H2O + acceptor = a carboxylate + reduced acceptor." [EC:1.2.99.7]
GO:0033728
name: divinyl chlorophyllide a 8-vinyl-reductase activity
def: "Catalysis of the reaction: chlorophyllide a + NADP+ = divinyl chlorophyllide a + NADPH + H+." [EC:1.3.1.75]
GO:0033729
name: anthocyanidin reductase activity
def: "Catalysis of the reaction: a flavan-3-ol + 2 NAD(P)+ = an anthocyanidin + 2 NAD(P)H + H+." [EC:1.3.1.77]
GO:0033730
name: arogenate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: L-arogenate + NADP+ = L-tyrosine + NADPH + CO2." [EC:1.3.1.78]
GO:0033731
name: arogenate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: L-arogenate + NAD(P)+ = L-tyrosine + NAD(P)H + CO2." [EC:1.3.1.79]
GO:0033732
name: pyrroloquinoline-quinone synthase activity
def: "Catalysis of the reaction: 6-(2-amino-2-carboxyethyl)-7,8-dioxo-1,2,3,4,7,8-hexahydroquinoline-2,4-dicarboxylate + 3 O(2) = 2 H(2)O + 2 H(2)O(2) + H(+) + pyrroloquinoline quinone." [EC:1.3.3.11, RHEA:10695]
GO:0033734
name: (R)-benzylsuccinyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: (R)-2-benzylsuccinyl-CoA + 2 electron-transferring flavoprotein = (E)-2-benzylidenesuccinyl-CoA + 2 reduced electron-transferring flavoprotein." [EC:1.3.99.21]
GO:0033735
name: aspartate dehydrogenase activity
def: "Catalysis of the reaction: L-aspartate + H2O + NAD(P)+ = oxaloacetate + NH3 + NAD(P)H + H+." [EC:1.4.1.21]
GO:0033736
name: L-lysine 6-oxidase activity
def: "Catalysis of the reaction: L-lysine + H(2)O + O(2) = allysine + H(2)O(2) + NH(4)(+)." [EC:1.4.3.20, RHEA:22551]
GO:0033737
name: 1-pyrroline dehydrogenase activity
def: "Catalysis of the reaction: 1-pyrroline + NAD+ + 2 H2O = 4-aminobutanoate + NADH + 2 H+." [MetaCyc:1.5.1.35-RXN]
GO:0033738
name: methylenetetrahydrofolate reductase (ferredoxin) activity
def: "Catalysis of the reaction: 5-methyltetrahydrofolate + oxidized ferredoxin = 5,10-methylenetetrahydrofolate + reduced ferredoxin." [EC:1.5.7.1]
GO:0033739
name: preQ1 synthase activity
def: "Catalysis of the reaction: 7-aminomethyl-7-deazaguanine + 2 NADP(+) = 7-cyano-7-deazaguanine + 3 H(+) + 2 NADPH." [EC:1.7.1.13, RHEA:13412]
GO:0033740
name: hydroxylamine oxidoreductase activity
def: "Catalysis of the reactions: hydroxylamine + NH3 = hydrazine + H2O, and hydrazine + acceptor = N2 + reduced acceptor." [EC:1.7.99.8]
GO:0033741
name: adenylyl-sulfate reductase (glutathione) activity
def: "Catalysis of the reaction: AMP + glutathione disulfide + H(+) + sulfite = 5'-adenylyl sulfate + 2 glutathione." [EC:1.8.4.9, RHEA:14144]
GO:0033743
name: peptide-methionine (R)-S-oxide reductase activity
def: "Catalysis of the reaction: peptide-L-methionine + H(2)O + thioredoxin disulfide = peptide-L-methionine (R)-S-oxide + thioredoxin. Can act on oxidized methionine in peptide linkage with specificity for the R enantiomer. Thioredoxin disulfide is the oxidized form of thioredoxin." [EC:1.8.4.12, GOC:mah, GOC:vw, RHEA:24167]
GO:0033744
name: L-methionine:thioredoxin-disulfide S-oxidoreductase activity
def: "Catalysis of the reaction: L-methionine + thioredoxin disulfide + H2O = L-methionine (S)-S-oxide + thioredoxin." [EC:1.8.4.13]
GO:0033745
name: L-methionine-(R)-S-oxide reductase activity
def: "Catalysis of the reaction: L-methionine + thioredoxin disulfide + H2O = L-methionine (R)-S-oxide + thioredoxin." [EC:1.8.4.14]
GO:0033746
name: histone demethylase activity (H3-R2 specific)
def: "Catalysis of the removal of a methyl group from arginine at position 2 of the histone H3 protein." [GOC:mah]
GO:0033747
name: versatile peroxidase activity
def: "OBSOLETE. Catalysis of the reactions: Reactive Black 5 + hydrogen peroxide = oxidized Reactive Black 5 + 2 H2O, and donor + hydrogen peroxide = oxidized donor + 2 H2O." [EC:1.11.1.16]
GO:0033748
name: hydrogenase (acceptor) activity
def: "Catalysis of the reaction: H2 + A = AH2." [EC:1.12.99.6]
GO:0033749
name: histone demethylase activity (H4-R3 specific)
def: "Catalysis of the removal of a methyl group from arginine at position 3 of the histone H4 protein." [GOC:mah]
GO:0033751
name: linoleate diol synthase activity
def: "Catalysis of the reaction: linoleate + O2 = (9Z,12Z)-(7S,8S)-dihydroxyoctadeca-9,12-dienoate." [EC:1.13.11.44]
GO:0033752
name: acetylacetone-cleaving enzyme activity
def: "Catalysis of the reaction: pentane-2,4-dione + O2 = acetate + 2-oxopropanal." [EC:1.13.11.50]
GO:0033754
name: indoleamine 2,3-dioxygenase activity
def: "Catalysis of the reaction: tryptophan + O2 = N-formylkynurenine. The product of the reaction depends on the substrate; D-tryptophan produces N-formyl-D-kynurenine, and L-tryptophan produces N-formyl-L-kynurenine." [EC:1.13.11.52]
GO:0033755
name: sulfur oxygenase/reductase activity
def: "Catalysis of the reaction: 4 sulfur + 4 H2O + O2 = 2 hydrogen sulfide + 2 bisulfite + 2 H+." [EC:1.13.11.55]
GO:0033756
name: Oplophorus-luciferin 2-monooxygenase activity
def: "Catalysis of the reaction: Oplophorus luciferin + O2 = oxidized Oplophorus luciferin + CO2 + hnu." [EC:1.13.12.13]
GO:0033757
name: glucoside 3-dehydrogenase activity
def: "Catalysis of the reaction: sucrose + acceptor = 3-dehydro-alpha-D-glucosyl-beta-D-fructofuranoside + reduced acceptor." [EC:1.1.99.13]
GO:0033758
name: clavaminate synthase activity
def: "Catalysis of the reactions: deoxyamidinoproclavaminate + 2-oxoglutarate + O2 = amidinoproclavaminate + succinate + CO2 + H2O; proclavaminate + 2-oxoglutarate + O2 = dihydroclavaminate + succinate + CO2 + 2 H2O; and dihydroclavaminate + 2-oxoglutarate + O2 = clavaminate + succinate + CO2 + 2 H2O." [EC:1.14.11.21]
GO:0033759
name: flavone synthase activity
def: "Catalysis of the reaction: a flavanone + 2-oxoglutarate + O2 = a flavone + succinate + CO2 + H2O." [EC:1.14.11.22]
GO:0033760
name: 2'-deoxymugineic-acid 2'-dioxygenase activity
def: "Catalysis of the reaction: 2'-deoxymugineate + 2-oxoglutarate + O(2) = CO(2) + H(+) + mugineate + succinate." [EC:1.14.11.24, RHEA:12203]
GO:0033761
name: mugineic-acid 3-dioxygenase activity
def: "Catalysis of the reactions: mugineic acid + 2-oxoglutarate + O2 = 3-epihydroxymugineic acid + succinate + CO2." [EC:1.14.11.25]
GO:0033763
name: proline 3-hydroxylase activity
def: "Catalysis of the reaction: L-proline + 2-oxoglutarate + O2 = cis-3-hydroxy-L-proline + succinate + CO2." [EC:1.14.11.28]
GO:0033764
name: steroid dehydrogenase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-OH group acts as a hydrogen or electron donor and reduces NAD+ or NADP, and in which one substrate is a sterol derivative." [GOC:mah]
GO:0033765
name: steroid dehydrogenase activity, acting on the CH-CH group of donors
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor, and in which one substrate is a sterol derivative." [GOC:mah]
GO:0033766
name: 2-hydroxyquinoline 8-monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + quinolin-2-ol = H(2)O + NAD(+) + quinoline-2,8-diol." [EC:1.14.13.61, RHEA:22083]
GO:0033767
name: 4-hydroxyacetophenone monooxygenase activity
def: "Catalysis of the reaction: 4'-hydroxyacetophenone + H(+) + NADPH + O(2) = 4-hydroxyphenyl acetate + H(2)O + NADP(+)." [EC:1.14.13.84, RHEA:22919]
GO:0033769
name: glyceollin synthase activity
def: "Catalysis of the reactions: 2-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADPH + H+ + O2 = glyceollin + NADP+ + 2 H2O, and 4-dimethylallyl-(6aS,11aS)-3,6a,9-trihydroxypterocarpan + NADPH + H+ + O2 = glyceollin + NADP+ + 2 H2O." [EC:1.14.13.85]
GO:0033770
name: 2-hydroxyisoflavanone synthase activity
def: "Catalysis of the reaction: apigenin + 2 NADPH + 2 H+ + O2 = 2-hydroxy-2,3-dihydrogenistein + 2 NADP+ + H2O." [EC:1.14.13.86]
GO:0033771
name: licodione synthase activity
def: "Catalysis of the reaction: H(+) + liquiritigenin + NADPH + O(2) = H(2)O + licodione + NADP(+)." [EC:1.14.13.87, RHEA:15700]
GO:0033772
name: flavonoid 3',5'-hydroxylase activity
def: "Catalysis of the reactions: a flavanone + NADPH + H+ + O2 = a 3'-hydroxyflavanone + NADP+ + H2O, and a 3'-hydroxyflavanone + NADPH + H+ + O2 = a 3',5'-dihydroxyflavanone + NADP+ + H2O." [EC:1.14.13.88]
GO:0033773
name: isoflavone 2'-hydroxylase activity
def: "Catalysis of the reaction: an isoflavone + NADPH + H+ + O2 = a 2'-hydroxyisoflavone + NADP+ + H2O." [EC:1.14.13.89]
GO:0033775
name: deoxysarpagine hydroxylase activity
def: "Catalysis of the reaction: 10-deoxysarpagine + H(+) + NADPH + O(2) = H(2)O + NADP(+) + sarpagine." [EC:1.14.13.91, RHEA:14240]
GO:0033776
name: phenylacetone monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + phenylacetone = benzyl acetate + H(2)O + NADP(+)." [EC:1.14.13.92, RHEA:10127]
GO:0033777
name: lithocholate 6beta-hydroxylase activity
def: "Catalysis of the reaction: H(+) + lithocholate + NADPH + O(2) = 6-beta-hydroxylithocholate + H(2)O + NADP(+)." [EC:1.14.13.94, RHEA:18860]
GO:0033778
name: 7alpha-hydroxycholest-4-en-3-one 12alpha-hydroxylase activity
def: "Catalysis of the reaction: 7alpha-hydroxycholest-4-en-3-one + H(+) + NADPH + O(2) = 7alpha,12alpha-dihydroxycholest-4-en-3-one + H(2)O + NADP(+)." [EC:1.14.13.95, RHEA:10507]
GO:0033779
name: 5beta-cholestane-3alpha,7alpha-diol 12alpha-hydroxylase activity
def: "Catalysis of the reaction: 5beta-cholestane-3alpha,7alpha-diol + H(+) + NADPH + O(2) = 5beta-cholestane-3alpha,7alpha,12alpha-triol + H(2)O + NADP(+)." [EC:1.14.13.96, RHEA:15264]
GO:0033780
name: taurochenodeoxycholate 6alpha-hydroxylase activity
def: "Catalysis of the reactions: taurochenodeoxycholate + NADPH + H+ + O2 = taurohyocholate + NADP+ + H2O, and lithocholate + NADPH + H+ + O2 = hyodeoxycholate + NADP+ + H2O." [EC:1.14.13.97]
GO:0033781
name: cholesterol 24-hydroxylase activity
def: "Catalysis of the reaction: cholesterol + H(+) + NADPH + O(2) = (24S)-24-hydroxycholesterol + H(2)O + NADP(+)." [EC:1.14.13.98, RHEA:22719]
GO:0033782
name: 24-hydroxycholesterol 7alpha-hydroxylase activity
def: "Catalysis of the reaction: (24R)-cholest-5-ene-3beta,24-diol + H(+) + NADPH + O(2) = (24R)-7alpha,24-dihydroxycholesterol + H(2)O + NADP(+)." [EC:1.14.13.99, RHEA:16096]
GO:0033783
name: 25-hydroxycholesterol 7alpha-hydroxylase activity
def: "Catalysis of the reactions: cholest-5-ene-3beta,25-diol + NADPH + H+ + O2 = cholest-5-ene-3beta,7alpha,25-triol + NADP+ + H2O, and cholest-5-ene-3beta,27-diol + NADPH + H+ + O2 = cholest-5-ene-3beta,7alpha,27-triol + NADP+ + H2O." [EC:1.14.13.100]
GO:0033784
name: senecionine N-oxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + senecionine = H(2)O + NADP(+) + senecionine N-oxide." [EC:1.14.13.101, RHEA:11423]
GO:0033785
name: heptose 7-phosphate kinase activity
def: "Catalysis of the reaction: D-alpha,beta-D-heptose-7-phosphate + ATP = D-beta-D-heptose-1,7-bisphosphate + ADP." [MetaCyc:RXN0-4341]
GO:0033786
name: heptose-1-phosphate adenylyltransferase activity
def: "Catalysis of the reaction: D-beta-D-heptose-1-phosphate + ATP = ADP-D-glycero-D-manno-heptose." [MetaCyc:RXN0-4342]
GO:0033787
name: cyanocobalamin reductase (cyanide-eliminating) activity
def: "Catalysis of the reaction: cob(I)alamin + hydrogen cyanide + NADP(+) = cyanocob(III)alamin + H(+) + NADPH." [EC:1.16.1.6, RHEA:16116]
GO:0033788
name: leucoanthocyanidin reductase activity
def: "Catalysis of the reaction: (2R,3S)-catechin + NADP+ + H2O = 2,3-trans-3,4-cis-leucocyanidin + NADPH + H+." [EC:1.17.1.3]
GO:0033789
name: phenylacetyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2 1,4-benzoquinone + H(2)O + phenylacetyl-CoA = 2 hydroquinone + phenylglyoxylyl-CoA." [EC:1.17.5.1, RHEA:15708]
GO:0033790
name: hydroxymethylfurfural reductase activity
def: "Catalysis of the reaction: 5-hydroxymethylfurfural + NAD(P)H + H+ = 2,5-bis-hydroxymethylfuran + NAD(P)+." [GOC:jp, GOC:mah, PMID:15338422, PMID:16652391]
GO:0033791
name: 3alpha,7alpha,12alpha-trihydroxy-5beta-cholestanoyl-CoA 24-hydroxylase activity
def: "Catalysis of the reaction: (25R)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-26-oyl-CoA + H2O + acceptor = (24R,25R)-3alpha,7alpha,12alpha,24-tetrahydroxy-5beta-cholestan-26-oyl-CoA + reduced acceptor." [EC:1.17.99.3]
GO:0033792
name: bile-acid 7alpha-dehydroxylase activity
def: "Catalysis of the reactions: deoxycholate + FAD + H2O = cholate + FADH2, and lithocholate + FAD + H2O = chenodeoxycholate + FADH2." [EC:1.17.99.5]
GO:0033793
name: aureusidin synthase activity
def: "Catalysis of the reactions: 2',4,4',6'-tetrahydroxychalcone + O2 = aureusidin + H2O, and 2',3,4,4',6'-pentahydroxychalcone + 1/2 O2 = aureusidin + H2O." [EC:1.21.3.6]
GO:0033794
name: sarcosine reductase activity
def: "Catalysis of the reaction: acetyl phosphate + methylamine + thioredoxin disulfide = N-methylglycine + phosphate + thioredoxin." [EC:1.21.4.3]
GO:0033795
name: betaine reductase activity
def: "Catalysis of the reaction: acetyl phosphate + trimethylamine + thioredoxin disulfide = N,N,N-trimethylglycine + phosphate + thioredoxin." [EC:1.21.4.4]
GO:0033796
name: sulfur reductase activity
def: "Catalysis of the reduction of elemental sulfur or polysulfide to hydrogen sulfide." [EC:1.97.1.3]
GO:0033797
name: selenate reductase activity
def: "Catalysis of the reaction: 2 e(-) + 2 H(+) + selenate = H(2)O + selenite." [EC:1.97.1.9, RHEA:14031]
GO:0033798
name: thyroxine 5-deiodinase activity
def: "Catalysis of the reaction: 3,3',5'-triiodo-L-thyronine + iodide + A + H+ = L-thyroxine + AH2." [EC:1.97.1.11]
GO:0033799
name: myricetin 3'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + myricetin = S-adenosyl-L-homocysteine + laricitrin." [MetaCyc:RXN-8451]
GO:0033800
name: isoflavone 7-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + a 7-hydroxyisoflavone = S-adenosyl-L-homocysteine + a 7-methoxyisoflavone." [EC:2.1.1.150]
GO:0033801
name: vitexin 2''-O-rhamnoside 7-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + vitexin 2''-O-beta-L-rhamnoside = S-adenosyl-L-homocysteine + 7-O-methylvitexin 2''-O-beta-L-rhamnoside." [EC:2.1.1.153]
GO:0033802
name: isoliquiritigenin 2'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + isoliquiritigenin = 2'-O-methylisoliquiritigenin + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.154, RHEA:21611]
GO:0033803
name: kaempferol 4'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + kaempferol = S-adenosyl-L-homocysteine + H(+) + kaempferide." [EC:2.1.1.155, RHEA:15108]
GO:0033804
name: glycine/sarcosine N-methyltransferase activity
def: "OBSOLETE. Catalysis of the reactions: S-adenosyl-L-methionine + glycine = S-adenosyl-L-homocysteine + sarcosine, and S-adenosyl-L-methionine + sarcosine = S-adenosyl-L-homocysteine + N,N-dimethylglycine." [EC:2.1.1.156]
GO:0033805
name: sarcosine/dimethylglycine N-methyltransferase activity
def: "OBSOLETE. Catalysis of the reactions: S-adenosyl-L-methionine + sarcosine = S-adenosyl-L-homocysteine + N,N-dimethylglycine, and S-adenosyl-L-methionine + N,N-dimethylglycine = S-adenosyl-L-homocysteine + betaine." [EC:2.1.1.157]
GO:0033806
name: fluorothreonine transaldolase activity
def: "Catalysis of the reaction: L-threonine + fluoroacetaldehyde = acetaldehyde + 4-fluoro-L-threonine." [EC:2.2.1.8]
GO:0033807
name: icosanoyl-CoA synthase activity
def: "Catalysis of the reaction: stearoyl-CoA + malonyl-CoA + 2 NAD(P)H + 2 H+ = icosanoyl-CoA + CO2 + 2 NAD(P)+." [EC:2.3.1.119]
GO:0033808
name: 6'-deoxychalcone synthase activity
def: "Catalysis of the reaction: 3 malonyl-CoA + 4-coumaroyl-CoA + NADPH + H+ = 4 CoA + isoliquiritigenin + 3 CO2 + NADP+ + H2O." [EC:2.3.1.170]
GO:0033809
name: anthocyanin 6''-O-malonyltransferase activity
def: "Catalysis of the reaction: malonyl-CoA + an anthocyanidin 3-O-beta-D-glucoside = CoA + an anthocyanidin 3-O-(6-O-malonyl-beta-D-glucoside)." [EC:2.3.1.171]
GO:0033810
name: anthocyanin 5-O-glucoside 6'''-O-malonyltransferase activity
def: "Catalysis of the reaction: malonyl-CoA + pelargonidin 3-O-(6-caffeoyl-beta-D-glucoside) 5-O-beta-D-glucoside = CoA + 4'''-demalonylsalvianin." [EC:2.3.1.172]
GO:0033811
name: flavonol-3-O-triglucoside O-coumaroyltransferase activity
def: "Catalysis of the reaction: 4-coumaroyl-CoA + a flavonol 3-O-[beta-D-glucosyl-(1->2)-beta-D-glucosyl-(1->2)-beta-D-glucoside] = CoA + a flavonol 3-O-[6-(4-coumaroyl)-beta-D-glucosyl-(1->2)-beta-D-glucosyl-(1->2)-beta-D-glucoside]." [EC:2.3.1.173]
GO:0033812
name: 3-oxoadipyl-CoA thiolase activity
def: "Catalysis of the reaction: succinyl-CoA + acetyl-CoA = CoA + 3-oxoadipyl-CoA." [EC:2.3.1.174]
GO:0033813
name: deacetylcephalosporin-C acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + deacetylcephalosporin C = cephalosporin C + CoA." [EC:2.3.1.175, RHEA:23863]
GO:0033814
name: propanoyl-CoA C-acyltransferase activity
def: "Catalysis of the reaction: 3alpha,7alpha,12alpha-trihydroxy-5beta-cholanoyl-CoA + propanoyl-CoA = CoA + 3alpha,7alpha,12alpha-trihydroxy-24-oxo-5beta-cholestanoyl-CoA." [EC:2.3.1.176]
GO:0033815
name: biphenyl synthase activity
def: "Catalysis of the reaction: 3 malonyl-CoA + benzoyl-CoA = 4 CoA + 3,5-dihydroxybiphenyl + 4 CO2." [EC:2.3.1.177]
GO:0033816
name: diaminobutyrate acetyltransferase activity
def: "Catalysis of the reaction: L-2,4-diaminobutyrate + acetyl-CoA = N(4)-acetyl-L-2,4-diaminobutyrate + CoA + H(+)." [EC:2.3.1.178, RHEA:16904]
GO:0033817
name: beta-ketoacyl-acyl-carrier-protein synthase II activity
def: "Catalysis of the reaction: (Z)-hexadec-11-enoyl-[acyl-carrier protein] + malonyl-[acyl-carrier protein] = (Z)-3-oxooctadec-13-enoyl-[acyl-carrier protein] + CO2 + [acyl-carrier protein]." [EC:2.3.1.179]
GO:0033818
name: beta-ketoacyl-acyl-carrier-protein synthase III activity
def: "Catalysis of the reaction: acetyl-CoA + malonyl-[acyl-carrier protein] = acetoacyl-[acyl-carrier protein] + CoA + CO2." [EC:2.3.1.180]
GO:0033819
name: lipoyl(octanoyl) transferase activity
def: "Catalysis of the reaction: octanoyl-[acyl-carrier protein] + protein = protein N6-(octanoyl)lysine + acyl-carrier protein." [EC:2.3.1.181]
GO:0033820
name: DNA alpha-glucosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-glucosyl residue from UDP-glucose to a hydroxymethylcytosine residue in DNA." [EC:2.4.1.26]
GO:0033821
name: DNA beta-glucosyltransferase activity
def: "Catalysis of the transfer of a beta-D-glucosyl residue from UDP-glucose to a hydroxymethylcytosine residue in DNA." [EC:2.4.1.27]
GO:0033822
name: glucosyl-DNA beta-glucosyltransferase activity
def: "Catalysis of the transfer of a beta-D-glucosyl residue from UDP-glucose to a glucosylhydroxymethylcytosine residue in DNA." [EC:2.4.1.28]
GO:0033823
name: procollagen glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + 5-(D-galactosyloxy)-L-lysine-procollagen = UDP + 1,2-D-glucosyl-5-D-(galactosyloxy)-L-lysine-procollagen." [EC:2.4.1.66]
GO:0033824
name: alternansucrase activity
def: "Catalysis of the transfer of an alpha-D-glucosyl residue from sucrose to alternately the 6-position and the 3-position of the non-reducing terminal residue of an alpha-D-glucan, thus producing a glucan having alternating alpha-1,6- and alpha-1,3-linkages." [EC:2.4.1.140]
GO:0033825
name: oligosaccharide 4-alpha-D-glucosyltransferase activity
def: "Catalysis of the transfer of the non-reducing terminal alpha-D-glucose residue from a 1,4-alpha-D-glucan to the 4-position of an alpha-D-glucan, thus bringing about the hydrolysis of oligosaccharides." [EC:2.4.1.161]
GO:0033826
name: xyloglucan 4-glucosyltransferase activity
def: "Catalysis of the transfer of a beta-D-glucosyl residue from UDP-glucose on to a glucose residue in xyloglucan, forming a beta-1,4-D-glucosyl-D-glucose linkage." [EC:2.4.1.168]
GO:0033827
name: high-mannose-oligosaccharide beta-1,4-N-acetylglucosaminyltransferase activity
def: "Catalysis of the transfer of an N-acetyl-D-glucosamine residue from UDP-N-acetyl-D-glucosamine to the 4-position of a mannose linked alpha-1,6 to the core mannose of high-mannose oligosaccharides produced by Dictyostelium discoideum." [EC:2.4.1.197]
GO:0033828
name: glucosylglycerol-phosphate synthase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + ADP-glucose = 2-O-(beta-D-glucosyl)-sn-glycerol 3-phosphate + ADP + H(+)." [EC:2.4.1.213, RHEA:12884]
GO:0033829
name: O-fucosylpeptide 3-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the transfer of a beta-D-GlcNAc residue from UDP-D-GlcNAc to the fucose residue of a fucosylated protein acceptor." [EC:2.4.1.222]
GO:0033830
name: Skp1-protein-hydroxyproline N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetylglucosamine + Skp1-protein-hydroxyproline = UDP + Skp1-protein-O-(N-acetyl-D-glucosaminyl)hydroxyproline." [EC:2.4.1.229]
GO:0033831
name: kojibiose phosphorylase activity
def: "Catalysis of the reaction: kojibiose + phosphate = beta-D-glucose 1-phosphate + D-glucose." [EC:2.4.1.230, RHEA:11179]
GO:0033832
name: alpha,alpha-trehalose phosphorylase (configuration-retaining) activity
def: "Catalysis of the reaction: alpha,alpha-trehalose + phosphate = alpha-D-glucose + alpha-D-glucose 1-phosphate." [EC:2.4.1.231]
GO:0033833
name: hydroxymethylfurfural reductase (NADH) activity
def: "Catalysis of the reaction: 5-hydroxymethylfurfural + NADH + H+ = 2,5-bis-hydroxymethylfuran + NAD+." [GOC:jp, GOC:mah, PMID:15338422, PMID:16652391]
GO:0033834
name: kaempferol 3-O-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + kaempferol = UDP + kaempferol 3-O-beta-D-galactoside." [EC:2.4.1.234]
GO:0033835
name: flavanone 7-O-glucoside 2''-O-beta-L-rhamnosyltransferase activity
def: "Catalysis of the reaction: UDP-L-rhamnose + a flavanone 7-O-glucoside = UDP + a flavanone 7-O-[beta-L-rhamnosyl-(1->2)-beta-D-glucoside]." [EC:2.4.1.236]
GO:0033836
name: flavonol 7-O-beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a flavonol = UDP + a flavonol 7-O-beta-D-glucoside." [EC:2.4.1.237]
GO:0033837
name: anthocyanin 3'-O-beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + an anthocyanin = UDP + an anthocyanin 3'-O-beta-D-glucoside." [EC:2.4.1.238]
GO:0033838
name: flavonol-3-O-glucoside glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a flavonol 3-O-beta-D-glucoside = UDP + a flavonol 3-O-beta-D-glucosyl-(1->2)-beta-D-glucoside." [EC:2.4.1.239]
GO:0033839
name: flavonol-3-O-glycoside glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a flavonol 3-O-beta-D-glucosyl-(1->2)-beta-D-glucoside = UDP + a flavonol 3-O-beta-D-glucosyl-(1->2)-beta-D-glucosyl-(1->2)-beta-D-glucoside." [EC:2.4.1.240]
GO:0033840
name: NDP-glucose-starch glucosyltransferase activity
def: "Catalysis of the reaction: NDP-glucose + (1,4-alpha-D-glucosyl)n = NDP + (1,4-alpha-D-glucosyl)n+1." [EC:2.4.1.242]
GO:0033841
name: 6G-fructosyltransferase activity
def: "Catalysis of the reaction: [1-beta-D-fructofuranosyl-(2->1)-]m+1 alpha-D-glucopyranoside + [1-beta-D-fructofuranosyl-(2->1)-]n+1 alpha-D-glucopyranoside = [1-beta-D-fructofuranosyl-(2->1)-]m alpha-D-glucopyranoside + [1-beta-D-fructofuranosyl-(2->1)-]n+1 beta-D-fructofuranosyl-(2->6)-alpha-D-glucopyranoside (m > 0; n >= 0)." [EC:2.4.1.243]
GO:0033842
name: N-acetyl-beta-glucosaminyl-glycoprotein 4-beta-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-galactosamine + N-acetyl-beta-D-glucosaminyl group = UDP + N-acetyl-beta-D-galactosaminyl-(1->4)-N-acetyl-beta-D-glucosaminyl group." [EC:2.4.1.244]
GO:0033843
name: xyloglucan 6-xylosyltransferase activity
def: "Catalysis of the transfer of an alpha-D-xylosyl residue from UDP-D-xylose to a glucose residue in xyloglucan, forming an alpha-1,6-D-xylosyl-D-glucose linkage." [EC:2.4.2.39]
GO:0033844
name: galactose-6-sulfurylase activity
def: "Catalysis of the elimination of sulfate from the D-galactose 6-sulfate residues of porphyran, producing 3,6-anhydrogalactose residues." [EC:2.5.1.5]
GO:0033845
name: hydroxymethylfurfural reductase (NADPH) activity
def: "Catalysis of the reaction: 5-hydroxymethylfurfural + NADPH + H+ = 2,5-bis-hydroxymethylfuran + NADP+." [GOC:jp, GOC:mah, PMID:15338422, PMID:16652391]
GO:0033846
name: adenosyl-fluoride synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + fluoride = 5'-deoxy-5'-fluoroadenosine + L-methionine." [EC:2.5.1.63]
GO:0033847
name: O-phosphoserine sulfhydrylase activity
def: "Catalysis of the reaction: O-phospho-L-serine + hydrogen sulfide = L-cysteine + phosphate." [EC:2.5.1.65]
GO:0033848
name: N2-(2-carboxyethyl)arginine synthase activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + L-arginine = N(2)-(2-carboxyethyl)-L-arginine + H(+) + phosphate." [EC:2.5.1.66, RHEA:10559]
GO:0033849
name: chrysanthemyl diphosphate synthase activity
def: "Catalysis of the reaction: 2 dimethylallyl diphosphate = (R,R)-chrysanthemyl diphosphate + diphosphate." [EC:2.5.1.67, RHEA:14012]
GO:0033850
name: Z-farnesyl diphosphate synthase activity
def: "Catalysis of the reaction: geranyl diphosphate + isopentenyl diphosphate = 2-cis,6-trans-farnesyl diphosphate + diphosphate." [EC:2.5.1.68, RHEA:23303]
GO:0033851
name: lavandulyl diphosphate synthase activity
def: "Catalysis of the reaction: 2 dimethylallyl diphosphate = diphosphate + lavandulyl diphosphate." [EC:2.5.1.69, RHEA:21679]
GO:0033852
name: thyroid-hormone transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + 3,5,3'-triiodo-L-thyronine = 3,5,3'-triiodothyropyruvate + L-glutamate." [EC:2.6.1.26, RHEA:19136]
GO:0033853
name: aspartate-prephenate aminotransferase activity
def: "Catalysis of the reaction: L-arogenate + oxaloacetate = prephenate + L-aspartate." [EC:2.6.1.78]
GO:0033854
name: glutamate-prephenate aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-arogenate = L-glutamate + prephenate." [EC:2.6.1.79, RHEA:22883]
GO:0033855
name: nicotianamine aminotransferase activity
def: "Catalysis of the reaction: 2-oxoglutarate + nicotianamine = 3''-deamino-3''-oxonicotianamine + L-glutamate." [EC:2.6.1.80, RHEA:22107]
GO:0033856
name: pyridoxine 5'-phosphate synthase activity
def: "Catalysis of the reaction: 1-deoxy-D-xylulose 5-phosphate + 3-amino-2-oxopropyl phosphate = 2 H(2)O + H(+) + phosphate + pyridoxine 5'-phosphate." [EC:2.6.99.2, RHEA:15268]
GO:0033857
name: diphosphoinositol-pentakisphosphate kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol 5-diphosphate pentakisphosphate = ADP + 1D-myo-inositol bisdiphosphate tetrakisphosphate." [EC:2.7.4.24]
GO:0033858
name: N-acetylgalactosamine kinase activity
def: "Catalysis of the reaction: ATP + N-acetyl-D-galactosamine = ADP + N-acetyl-alpha-D-galactosamine 1-phosphate." [EC:2.7.1.157]
GO:0033862
name: UMP kinase activity
def: "Catalysis of the reaction: ATP + UMP = ADP + UDP." [EC:2.7.4.22]
GO:0033863
name: ribose 1,5-bisphosphate phosphokinase activity
def: "Catalysis of the reaction: D-ribose 1,5-diphosphate + ATP = 5-phospho-alpha-D-ribose 1-diphosphate + ADP + H(+)." [EC:2.7.4.23, RHEA:20112]
GO:0033867
name: Fas-activated serine/threonine kinase activity
def: "Catalysis of the reaction: ATP + Fas-activated serine/threonine protein = ADP + Fas-activated serine/threonine phosphoprotein." [EC:2.7.11.8]
GO:0033868
name: Goodpasture-antigen-binding protein kinase activity
def: "Catalysis of the reaction: ATP + Goodpasture antigen-binding protein = ADP + Goodpasture antigen-binding phosphoprotein." [EC:2.7.11.9]
GO:0033870
name: thiol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + a thiol = adenosine 3',5'-bisphosphate + an S-alkyl thiosulfate." [EC:2.8.2.16]
GO:0033871
name: [heparan sulfate]-glucosamine 3-sulfotransferase 2 activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine 3',5'-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate; modifies selected glucosamine residues preceded by GlcA2S." [EC:2.8.2.29]
GO:0033872
name: [heparan sulfate]-glucosamine 3-sulfotransferase 3 activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + [heparan sulfate]-glucosamine = adenosine 3',5'-bisphosphate + [heparan sulfate]-glucosamine 3-sulfate." [EC:2.8.2.30]
GO:0033873
name: petromyzonol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + 5alpha-cholane-3alpha,7alpha,12alpha,24-tetrol = 3alpha,7alpha,12alpha-trihydroxy-5alpha-cholan-24-yl sulfate + adenosine 3',5'-diphosphate + H(+)." [EC:2.8.2.31, RHEA:17000]
GO:0033874
name: scymnol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + 5beta-scymnol = 5beta-scymnol sulfate + adenosine 3',5'-diphosphate + H(+)." [EC:2.8.2.32, RHEA:15480]
GO:0033876
name: glycochenodeoxycholate sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + glycochenodeoxycholate = adenosine 3',5'-diphosphate + glycochenodeoxycholate 7-sulfate + H(+)." [EC:2.8.2.34, RHEA:17692]
GO:0033877
name: succinyl-CoA:(R)-benzylsuccinate CoA-transferase activity
def: "Catalysis of the reaction: (R)-2-benzylsuccinate + succinyl-CoA = (R)-2-benzylsuccinyl-CoA + succinate." [EC:2.8.3.15, RHEA:16472]
GO:0033878
name: hormone-sensitive lipase activity
def: "Catalysis of the reactions: diacylglycerol + H2O = monoacylglycerol + a carboxylate; triacylglycerol + H2O = diacylglycerol + a carboxylate; and monoacylglycerol + H2O = glycerol + a carboxylate." [EC:3.1.1.79]
GO:0033879
name: acetylajmaline esterase activity
def: "Catalysis of the reactions: 17-O-acetylajmaline + H2O = ajmaline + acetate, and 17-O-acetylnorajmaline + H2O = norajmaline + acetate." [EC:3.1.1.80]
GO:0033880
name: phenylacetyl-CoA hydrolase activity
def: "Catalysis of the reaction: H(2)O + phenylglyoxylyl-CoA = CoA + H(+) + phenylglyoxylate." [EC:3.1.2.25, RHEA:15340]
GO:0033881
name: bile-acid-CoA hydrolase activity
def: "Catalysis of the reaction: deoxycholoyl-CoA + H(2)O = CoA + deoxycholate + H(+)." [EC:3.1.2.26, RHEA:17696]
GO:0033882
name: choloyl-CoA hydrolase activity
def: "Catalysis of the reaction: choloyl-CoA + H2O = cholate + CoA." [EC:3.1.2.27]
GO:0033883
name: pyridoxal phosphatase activity
def: "Catalysis of the reaction: pyridoxal 5'-phosphate + H2O = pyridoxal + phosphate." [EC:3.1.3.74]
GO:0033884
name: phosphoethanolamine/phosphocholine phosphatase activity
def: "OBSOLETE. Catalysis of the reactions: O-phosphoethanolamine + H2O = ethanolamine + phosphate, and phosphocholine + H2O = choline + phosphate." [EC:3.1.3.75]
GO:0033885
name: 10-hydroxy-9-(phosphonooxy)octadecanoate phosphatase activity
def: "Catalysis of the reaction: (9S,10S)-10-hydroxy-9-(phosphonooxy)octadecanoate + H(2)O = (9S,10S)-9,10-dihydroxyoctadecanoate + phosphate." [EC:3.1.3.76, RHEA:16540]
GO:0033886
name: cellulose-polysulfatase activity
def: "Catalysis of the hydrolysis of the 2- and 3-sulfate groups of the polysulfates of cellulose and charonin." [EC:3.1.6.7]
GO:0033887
name: chondro-4-sulfatase activity
def: "Catalysis of the reaction: 4-deoxy-beta-D-gluc-4-enuronosyl-(1->3)-N-acetyl-D-galactosamine 4-sulfate + H(2)O = 4-deoxy-beta-D-gluc-4-enuronosyl-(1->3)-N-acetyl-D-galactosamine + H(+) + sulfate." [EC:3.1.6.9, RHEA:11447]
GO:0033888
name: chondro-6-sulfatase activity
def: "Catalysis of the reaction: 4-deoxy-beta-D-gluc-4-enuronosyl-(1->3)-N-acetyl-D-galactosamine 6-sulfate + H(2)O = 4-deoxy-beta-D-gluc-4-enuronosyl-(1->3)-N-acetyl-D-galactosamine + H(+) + sulfate." [EC:3.1.6.10, RHEA:10539]
GO:0033889
name: N-sulfoglucosamine-3-sulfatase activity
def: "Catalysis of the hydrolysis of the 3-sulfate groups of the N-sulfo-D-glucosamine 3-O-sulfate units of heparin." [EC:3.1.6.15]
GO:0033890
name: ribonuclease D activity
def: "Catalysis of the exonucleolytic cleavage that removes extra residues from the 3'-terminus of tRNA to produce 5'-mononucleotides." [EC:3.1.13.5]
GO:0033891
name: CC-preferring endodeoxyribonuclease activity
def: "Catalysis of the endonucleolytic cleavage to give 5'-phosphooligonucleotide end-products, with a preference for cleavage within the sequence CC." [EC:3.1.21.6]
GO:0033892
name: deoxyribonuclease (pyrimidine dimer) activity
def: "Catalysis of the endonucleolytic cleavage near pyrimidine dimers to products with 5'-phosphate." [EC:3.1.25.1]
GO:0033893
name: ribonuclease IV activity
def: "Catalysis of the endonucleolytic cleavage of poly(A) to fragments terminated by 3'-hydroxy and 5'-phosphate groups." [EC:3.1.26.6]
GO:0033894
name: ribonuclease P4 activity
def: "Catalysis of the endonucleolytic cleavage of RNA, removing 3'-extranucleotides from tRNA precursor." [EC:3.1.26.7]
GO:0033895
name: ribonuclease [poly-(U)-specific] activity
def: "Catalysis of the endonucleolytic cleavage of poly(U) to fragments terminated by 3'-hydroxy and 5'-phosphate groups." [EC:3.1.26.9]
GO:0033896
name: ribonuclease IX activity
def: "Catalysis of the endonucleolytic cleavage of poly(U) or poly(C) to fragments terminated by 3'-hydroxy and 5'-phosphate groups." [EC:3.1.26.10]
GO:0033897
name: ribonuclease T2 activity
def: "Catalysis of the two-stage endonucleolytic cleavage to nucleoside 3'-phosphates and 3'-phosphooligonucleotides with 2',3'-cyclic phosphate intermediates." [EC:3.1.27.1]
GO:0033898
name: Bacillus subtilis ribonuclease activity
def: "Catalysis of the endonucleolytic cleavage to 2',3'-cyclic nucleotides." [EC:3.1.27.2]
GO:0033899
name: ribonuclease U2 activity
def: "Catalysis of the two-stage endonucleolytic cleavage to nucleoside 3'-phosphates and 3'-phosphooligonucleotides ending in Ap or Gp with 2',3'-cyclic phosphate intermediates." [EC:3.1.27.4]
GO:0033900
name: ribonuclease F activity
def: "Catalysis of the endonucleolytic cleavage of RNA precursor into two, leaving 5'-hydroxy and 3'-phosphate groups." [EC:3.1.27.7]
GO:0033901
name: ribonuclease V activity
def: "Catalysis of the hydrolysis of poly(A), forming oligoribonucleotides and ultimately 3'-AMP." [EC:3.1.27.8]
GO:0033902
name: rRNA endonuclease activity
def: "Catalysis of the hydrolysis of the phosphodiester linkage between guanosine and adenosine residues at one specific position in 28S rRNA from rat ribosomes." [EC:3.1.27.10]
GO:0033903
name: endo-1,3(4)-beta-glucanase activity
def: "OBSOLETE. Catalysis of the endohydrolysis of 1,3- or 1,4-linkages in beta-D-glucans when the glucose residue whose reducing group is involved in the linkage to be hydrolysed is itself substituted at C-3." [EC:3.2.1.6]
GO:0033904
name: dextranase activity
def: "Catalysis of the endohydrolysis of 1,6-alpha-D-glucosidic linkages in dextran." [EC:3.2.1.11]
GO:0033905
name: xylan endo-1,3-beta-xylosidase activity
def: "Catalysis of the random hydrolysis of (1->3)-beta-D-glycosidic linkages in (1->3)-beta-D-xylans." [EC:3.2.1.32]
GO:0033906
name: hyaluronoglucuronidase activity
def: "Catalysis of the random hydrolysis of 1,3-linkages between beta-D-glucuronate and N-acetyl-D-glucosamine residues in hyaluronate." [EC:3.2.1.36]
GO:0033907
name: beta-D-fucosidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing beta-D-fucose residues in beta-D-fucosides." [EC:3.2.1.38]
GO:0033908
name: beta-L-rhamnosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing beta-L-rhamnose residues in beta-L-rhamnosides." [EC:3.2.1.43]
GO:0033909
name: fucoidanase activity
def: "Catalysis of the endohydrolysis of 1,2-alpha-L-fucoside linkages in fucoidan without release of sulfate." [EC:3.2.1.44]
GO:0033910
name: glucan 1,4-alpha-maltotetraohydrolase activity
def: "Catalysis of the hydrolysis of (1->4)-alpha-D-glucosidic linkages in amylaceous polysaccharides, to remove successive maltotetraose residues from the non-reducing chain ends." [EC:3.2.1.60]
GO:0033911
name: mycodextranase activity
def: "Catalysis of the endohydrolysis of 1,4-alpha-D-glucosidic linkages in alpha-D-glucans containing both 1,3- and 1,4-bonds." [EC:3.2.1.61]
GO:0033912
name: 2,6-beta-fructan 6-levanbiohydrolase activity
def: "Catalysis of the hydrolysis of (2->6)-beta-D-fructofuranan, to remove successive disaccharide residues as levanbiose, i.e. 6-(beta-D-fructofuranosyl)-D-fructose, from the end of the chain." [EC:3.2.1.64]
GO:0033913
name: glucan endo-1,2-beta-glucosidase activity
def: "Catalysis of the random hydrolysis of (1->2)-glucosidic linkages in (1->2)-beta-D-glucans." [EC:3.2.1.71]
GO:0033914
name: xylan 1,3-beta-xylosidase activity
def: "Catalysis of the hydrolysis of successive xylose residues from the non-reducing termini of (1->3)-beta-D-xylans." [EC:3.2.1.72]
GO:0033915
name: mannan 1,2-(1,3)-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of (1->2) and (1->3) linkages in mannan, releasing mannose." [EC:3.2.1.77]
GO:0033916
name: beta-agarase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-galactosidic linkages in agarose, giving the tetramer as the predominant product." [EC:3.2.1.81]
GO:0033917
name: exo-poly-alpha-galacturonosidase activity
def: "Catalysis of the hydrolysis of pectic acid from the non-reducing end, releasing digalacturonate." [EC:3.2.1.82]
GO:0033918
name: kappa-carrageenase activity
def: "Catalysis of the endohydrolysis of 1,4-beta-D-linkages between D-galactose 4-sulfate and 3,6-anhydro-D-galactose in kappa-carrageenans." [EC:3.2.1.83]
GO:0033919
name: glucan 1,3-alpha-glucosidase activity
def: "Catalysis of the hydrolysis of terminal (1->3)-alpha-D-glucosidic links in 1,3-alpha-D-glucans." [EC:3.2.1.84]
GO:0033920
name: 6-phospho-beta-galactosidase activity
def: "Catalysis of the reaction: a 6-phospho-beta-D-galactoside + H2O = 6-phospho-D-galactose + an alcohol." [EC:3.2.1.85]
GO:0033921
name: capsular-polysaccharide endo-1,3-alpha-galactosidase activity
def: "Catalysis of the random hydrolysis of (1->3)-alpha-D-galactosidic linkages in Aerobacter aerogenes capsular polysaccharide." [EC:3.2.1.87]
GO:0033922
name: peptidoglycan beta-N-acetylmuramidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing N-acetylmuramic residues." [EC:3.2.1.92]
GO:0033923
name: glucan 1,6-alpha-isomaltosidase activity
def: "Catalysis of the hydrolysis of (1->6)-alpha-D-glucosidic linkages in polysaccharides, to remove successive isomaltose units from the non-reducing ends of the chains." [EC:3.2.1.94]
GO:0033924
name: dextran 1,6-alpha-isomaltotriosidase activity
def: "Catalysis of the hydrolysis of (1->6)-alpha-D-glucosidic linkages in dextrans, to remove successive isomaltotriose units from the non-reducing ends of the chains." [EC:3.2.1.95]
GO:0033925
name: mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase activity
def: "Catalysis of the endohydrolysis of the N,N'-diacetylchitobiosyl unit in high-mannose glycopeptides and glycoproteins containing the -[Man(GlcNAc)2]Asn-structure. One N-acetyl-D-glucosamine residue remains attached to the protein; the rest of the oligosaccharide is released intact." [EC:3.2.1.96]
GO:0033926
name: glycopeptide alpha-N-acetylgalactosaminidase activity
def: "Catalysis of the reaction: D-galactosyl-3-(N-acetyl-alpha-D-galactosaminyl)-L-serine + H2O = D-galactosyl-3-N-acetyl-alpha-D-galactosamine + L-serine." [EC:3.2.1.97, MetaCyc:3.2.1.97-RXN]
GO:0033927
name: glucan 1,4-alpha-maltohexaosidase activity
def: "Catalysis of the hydrolysis of (1->4)-alpha-D-glucosidic linkages in amylaceous polysaccharides, to remove successive maltohexaose residues from the non-reducing chain ends." [EC:3.2.1.98]
GO:0033928
name: mannan 1,4-mannobiosidase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-mannosidic linkages in (1->4)-beta-D-mannans, to remove successive mannobiose residues from the non-reducing chain ends." [EC:3.2.1.100]
GO:0033929
name: blood-group-substance endo-1,4-beta-galactosidase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-galactosidic linkages in blood group A and B substances." [EC:3.2.1.102]
GO:0033930
name: keratan-sulfate endo-1,4-beta-galactosidase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-galactosidic linkages in keratan sulfate." [EC:3.2.1.103]
GO:0033931
name: endogalactosaminidase activity
def: "Catalysis of the endohydrolysis of (1->4)-alpha-D-galactosaminidic linkages in poly(D-galactosamine)." [EC:3.2.1.109]
GO:0033932
name: 1,3-alpha-L-fucosidase activity
def: "Catalysis of the hydrolysis of (1->3) linkages between alpha-L-fucose and N-acetylglucosamine residues in glycoproteins." [EC:3.2.1.111]
GO:0033933
name: branched-dextran exo-1,2-alpha-glucosidase activity
def: "Catalysis of the hydrolysis of (1->2)-alpha-D-glucosidic linkages at the branch points of dextrans and related polysaccharides, producing free D-glucose." [EC:3.2.1.115]
GO:0033934
name: glucan 1,4-alpha-maltotriohydrolase activity
def: "Catalysis of the hydrolysis of (1->4)-alpha-D-glucosidic linkages in amylaceous polysaccharides, to remove successive maltotriose residues from the non-reducing chain ends." [EC:3.2.1.116]
GO:0033935
name: oligoxyloglucan beta-glycosidase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-glucosidic links in oligoxyloglucans so as to remove successive isoprimeverose (i.e. alpha-xylo-1,6-beta-D-glucosyl-) residues from the non-reducing chain ends." [EC:3.2.1.120]
GO:0033936
name: polymannuronate hydrolase activity
def: "Catalysis of the endohydrolysis of the D-mannuronide linkages of polymannuronate." [EC:3.2.1.121]
GO:0033937
name: 3-deoxy-2-octulosonidase activity
def: "Catalysis of the endohydrolysis of the beta-ketopyranosidic linkages of 3-deoxy-D-manno-2-octulosonate in capsular polysaccharides." [EC:3.2.1.124]
GO:0033938
name: 1,6-alpha-L-fucosidase activity
def: "Catalysis of the hydrolysis of (1->6) linkages between alpha-L-fucose and N-acetyl-D-glucosamine in glycopeptides such as immunoglobulin G glycopeptide and fucosyl-asialo-agalacto-fetuin." [EC:3.2.1.127]
GO:0033939
name: xylan alpha-1,2-glucuronosidase activity
def: "Catalysis of the hydrolysis of alpha-D-(1->2)-(4-O-methyl)glucuronosyl links in the main chain of hardwood xylans." [EC:3.2.1.131]
GO:0033940
name: glucuronoarabinoxylan endo-1,4-beta-xylanase activity
def: "Catalysis of the endohydrolysis of (1->4)-beta-D-xylosyl links in some glucuronoarabinoxylans." [EC:3.2.1.136]
GO:0033941
name: mannan exo-1,2-1,6-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of (1->2)-alpha-D- and (1->6)-alpha-D- linkages in mannan, releasing D-mannose." [EC:3.2.1.137]
GO:0033942
name: 4-alpha-D-\{(1->4)-alpha-D-glucano}trehalose trehalohydrolase activity
def: "Catalysis of the hydrolysis of alpha-(1->4)-D-glucosidic linkage in 4-alpha-D-{(1->4)-alpha-D-glucanosyl}n trehalose to yield trehalose and alpha-(1->4)-D-glucan." [EC:3.2.1.141]
GO:0033943
name: galactan 1,3-beta-galactosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing beta-D-galactose residues in (1->3)-beta-D-galactopyranans." [EC:3.2.1.145]
GO:0033944
name: beta-galactofuranosidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing beta-D-galactofuranosides, releasing galactose." [EC:3.2.1.146]
GO:0033945
name: oligoxyloglucan reducing-end-specific cellobiohydrolase activity
def: "Catalysis of the hydrolysis of cellobiose from the reducing end of xyloglucans consisting of a beta-(1->4) linked glucan carrying alpha-D-xylosyl groups on O-6 of the glucose residues. To be a substrate, the first residue must be unsubstituted, the second residue may bear a xylosyl group, whether further glycosylated or not, and the third residue, which becomes the new terminus by the action of the enzyme, is preferably xylosylated, but this xylose residue must not be further substituted." [EC:3.2.1.150]
GO:0033946
name: xyloglucan-specific endo-beta-1,4-glucanase activity
def: "Catalysis of the reaction: xyloglucan + H2O = xyloglucan oligosaccharides. This reaction is the endohydrolysis of (1->4)-beta-D-glucosidic linkages in xyloglucan." [EC:3.2.1.151]
GO:0033947
name: mannosylglycoprotein endo-beta-mannosidase activity
def: "Catalysis of the hydrolysis of the alpha-D-mannosyl-(1->6)-beta-D-mannosyl-(1->4)-beta-D-N-acetylglucosaminyl-(1->4)-beta-D-N-acetylglucosaminyl sequence of glycoprotein to alpha-D-mannosyl-(1->6)-D-mannose and beta-D-N-acetylglucosaminyl-(1->4)-beta-D-N-acetylglucosaminyl sequences." [EC:3.2.1.152]
GO:0033948
name: fructan beta-(2,1)-fructosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing (2->1) linked beta-D-fructofuranose residues in fructans." [EC:3.2.1.153]
GO:0033949
name: fructan beta-(2,6)-fructosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing (2->6) linked beta-D-fructofuranose residues in fructans." [EC:3.2.1.154]
GO:0033950
name: xyloglucan-specific exo-beta-1,4-glucanase activity
def: "Catalysis of the reaction: xyloglucan + H2O = xyloglucan oligosaccharides. This reaction is the exohydrolysis of 1,4-beta-D-glucosidic linkages in xyloglucan." [EC:3.2.1.155]
GO:0033951
name: oligosaccharide reducing-end xylanase activity
def: "Catalysis of the hydrolysis of 1,4-beta-D-xylose residues from the reducing end of oligosaccharides." [EC:3.2.1.156]
GO:0033952
name: iota-carrageenase activity
def: "Catalysis of the endohydrolysis of 1,4-beta-D-linkages between D-galactose 4-sulfate and 3,6-anhydro-D-galactose-2-sulfate in iota-carrageenans." [EC:3.2.1.157]
GO:0033953
name: alpha-agarase activity
def: "Catalysis of the endohydrolysis of 1,3-alpha-L-galactosidic linkages in agarose, yielding agarotetraose as the major product." [EC:3.2.1.158]
GO:0033954
name: alpha-neoagaro-oligosaccharide hydrolase activity
def: "Catalysis of the hydrolysis of the 1,3-alpha-L-galactosidic linkages of neoagaro-oligosaccharides that are smaller than a hexamer, yielding 3,6-anhydro-L-galactose and D-galactose." [EC:3.2.1.159]
GO:0033956
name: beta-apiosyl-beta-glucosidase activity
def: "Catalysis of the reaction: 7-[beta-D-apiofuranosyl-(1->6)-beta-D-glucopyranosyloxy]isoflavonoid + H2O = a 7-hydroxyisoflavonoid + beta-D-apiofuranosyl-(1->6)-D-glucose." [EC:3.2.1.161]
GO:0033957
name: lambda-carrageenase activity
def: "Catalysis of the endohydrolysis of beta-1,4-linkages in the backbone of lambda-carrageenan, resulting in the tetrasaccharide alpha-D-Galp2,6S2-(1->3)-beta-D-Galp2S-(1->4)-alpha-D-Galp2,6S2-(1->3)-D-Galp2S." [EC:3.2.1.162]
GO:0033958
name: DNA-deoxyinosine glycosylase activity
def: "Catalysis of the hydrolysis of DNA and polynucleotides, releasing free hypoxanthine." [EC:3.2.2.15]
GO:0033959
name: deoxyribodipyrimidine endonucleosidase activity
def: "Catalysis of the cleavage of the N-glycosidic bond between the 5'-pyrimidine residue in cyclobutadipyrimidine (in DNA) and the corresponding deoxy-D-ribose residue." [EC:3.2.2.17]
GO:0033960
name: N-methyl nucleosidase activity
def: "Catalysis of the reaction: 7-methylxanthosine + H(2)O = 7-methylxanthine + H(+) + ribofuranose." [EC:3.2.2.25, RHEA:10883]
GO:0033961
name: cis-stilbene-oxide hydrolase activity
def: "Catalysis of the reaction: cis-stilbene oxide + H2O = (+)-(1R,2R)-1,2-diphenylethane-1,2-diol." [EC:3.3.2.9]
GO:0033963
name: cholesterol-5,6-oxide hydrolase activity
def: "Catalysis of the reactions: 5,6alpha-epoxy-5alpha-cholestan-3beta-ol + H2O = cholestane-3beta-5alpha,6beta-triol, and 5,6beta-epoxy-5beta-cholestan-3beta-ol + H2O = cholestane-3beta-5alpha,6beta-triol." [EC:3.3.2.11]
GO:0033964
name: glycosphingolipid deacylase activity
def: "Catalysis of the hydrolysis of gangliosides and neutral glycosphingolipids, releasing fatty acids to form the lyso-derivatives." [EC:3.5.1.69]
GO:0033965
name: aculeacin-A deacylase activity
def: "Catalysis of the hydrolysis of the amide bond in aculeacin A and related neutral lipopeptide antibiotics, releasing the long-chain fatty acid side-chain." [EC:3.5.1.70]
GO:0033966
name: N-substituted formamide deformylase activity
def: "Catalysis of the reaction: N-benzylformamide + H2O = formate + benzylamine." [EC:3.5.1.91]
GO:0033968
name: glutaryl-7-aminocephalosporanic-acid acylase activity
def: "Catalysis of the reaction: (7R)-7-(4-carboxybutanamido)cephalosporanate + H2O = (7R)-7-aminocephalosporanate + glutarate." [EC:3.5.1.93]
GO:0033969
name: gamma-glutamyl-gamma-aminobutyrate hydrolase activity
def: "Catalysis of the reaction: 4-(L-gamma-glutamylamino)butanoate + H(2)O = 4-aminobutanoate + L-glutamate." [EC:3.5.1.94, RHEA:19740]
GO:0033970
name: N-malonylurea hydrolase activity
def: "Catalysis of the reaction: 3-oxo-3-ureidopropanoate + H(2)O = H(+) + malonate + urea." [EC:3.5.1.95, RHEA:17364]
GO:0033971
name: hydroxyisourate hydrolase activity
def: "Catalysis of the reaction: 5-hydroxyisourate + H(2)O = 5-hydroxy-2-oxo-4-ureido-2,5-dihydro-1H-imidazole-5-carboxylate + H(+)." [EC:3.5.2.17, RHEA:23739]
GO:0033972
name: proclavaminate amidinohydrolase activity
def: "Catalysis of the reaction: amidinoproclavaminate + H(2)O = proclavaminate + urea." [EC:3.5.3.22, RHEA:17004]
GO:0033973
name: dCTP deaminase (dUMP-forming) activity
def: "Catalysis of the reaction: dCTP + 2 H(2)O = diphosphate + dUMP + H(+) + NH(4)(+)." [EC:3.5.4.30, RHEA:19208]
GO:0033974
name: nucleoside phosphoacylhydrolase activity
def: "Catalysis of the hydrolysis of mixed phospho-anhydride bonds." [EC:3.6.1.24]
GO:0033975
name: (R)-2-haloacid dehalogenase activity
def: "Catalysis of the reaction: (R)-2-haloacid + H2O = (S)-2-hydroxyacid + halide." [EC:3.8.1.9]
GO:0033976
name: 2-haloacid dehalogenase (configuration-inverting) activity
def: "Catalysis of the reactions: (S)-2-haloacid + H2O = (R)-2-hydroxyacid + halide, and (R)-2-haloacid + H2O = (S)-2-hydroxyacid + halide." [EC:3.8.1.10]
GO:0033977
name: 2-haloacid dehalogenase (configuration-retaining) activity
def: "Catalysis of the reactions: (S)-2-haloacid + H2O = (S)-2-hydroxyacid + halide, and (R)-2-haloacid + H2O = (R)-2-hydroxyacid + halide." [EC:3.8.1.11]
GO:0033978
name: phosphonopyruvate hydrolase activity
def: "Catalysis of the reaction: 3-phosphonopyruvate + H(2)O = phosphate + pyruvate." [EC:3.11.1.3, RHEA:16676]
GO:0033980
name: phosphonopyruvate decarboxylase activity
def: "Catalysis of the reaction: 3-phosphonopyruvate + 2 H(+) = CO(2) + phosphonoacetaldehyde." [EC:4.1.1.82, RHEA:20771]
GO:0033981
name: D-dopachrome decarboxylase activity
def: "Catalysis of the reaction: D-dopachrome + H(+) = 5,6-dihydroxyindole + CO(2)." [EC:4.1.1.84, RHEA:18444]
GO:0033982
name: 3-dehydro-L-gulonate-6-phosphate decarboxylase activity
def: "Catalysis of the reaction: 3-dehydro-L-gulonate 6-phosphate + H(+) = L-xylulose 5-phosphate + CO(2)." [EC:4.1.1.85, RHEA:14356]
GO:0033983
name: diaminobutyrate decarboxylase activity
def: "Catalysis of the reaction: L-2,4-diaminobutyrate + H(+) = 1,3-diaminopropane + CO(2)." [EC:4.1.1.86, RHEA:15692]
GO:0033984
name: indole-3-glycerol-phosphate lyase activity
def: "Catalysis of the reaction: (1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate = indole + D-glyceraldehyde 3-phosphate." [EC:4.1.2.8]
GO:0033987
name: 2-hydroxyisoflavanone dehydratase activity
def: "Catalysis of the reaction: 2,7,4'-trihydroxyisoflavanone = daidzein + H2O." [EC:4.2.1.105]
GO:0033988
name: bile-acid 7alpha-dehydratase activity
def: "Catalysis of the reaction: 7alpha,12alpha-dihydroxy-3-oxochol-4-en-24-oate = 12alpha-hydroxy-3-oxochola-4,6-dien-24-oate + H(2)O." [EC:4.2.1.106, RHEA:10439]
GO:0033989
name: 3alpha,7alpha,12alpha-trihydroxy-5beta-cholest-24-enoyl-CoA hydratase activity
def: "Catalysis of the reaction: (24R,25R)-3alpha,7alpha,12alpha,24-tetrahydroxy-5beta-cholestanoyl-CoA = (24E)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholest-24-enoyl-CoA + H2O." [EC:4.2.1.107]
GO:0033990
name: ectoine synthase activity
def: "Catalysis of the reaction: N(4)-acetyl-L-2,4-diaminobutyrate = ectoine + H(2)O." [EC:4.2.1.108, RHEA:17284]
GO:0033991
name: aldos-2-ulose dehydratase activity
def: "Catalysis of the reactions: 1,5-anhydro-D-fructose = 2-hydroxy-2-(hydroxymethyl)-2H-pyran-3(6H)-one + H2O; (1a) 1,5-anhydro-D-fructose = 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose + H2O and (1b) 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose = 2-hydroxy-2-(hydroxymethyl)-2H-pyran-3(6H)-one." [EC:4.2.1.110]
GO:0033992
name: 1,5-anhydro-D-fructose dehydratase activity
def: "Catalysis of the reaction: 1,5-anhydro-D-fructose = 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose + H2O." [EC:4.2.1.111]
GO:0033994
name: glucuronan lyase activity
def: "Catalysis of the reaction: (1->4)-beta-D-glucuronan = an oligosaccharide with 4-deoxy-beta-D-gluc-4-enuronosyl end + (1->4)-beta-D-glucuronan. This reaction is the eliminative cleavage of (1->4)-beta-D-glucuronans to give oligosaccharides with 4-deoxy-beta-D-gluc-4-enuronosyl groups at their non-reducing ends. Complete degradation of glucuronans results in the formation of tetrasaccharides." [EC:4.2.2.14]
GO:0033995
name: anhydrosialidase activity
def: "Catalysis of the reaction: an N-acetylneuraminate glycoside = 2,7-anhydro-alpha-N-acetylneuraminate + an alpha-sialyl group. This reaction is the elimination of alpha-sialyl groups in N-acetylneuraminic acid glycosides, releasing 2,7-anhydro-alpha-N-acetylneuraminate." [EC:4.2.2.15]
GO:0033996
name: levan fructotransferase (DFA-IV-forming) activity
def: "Catalysis of the reaction: beta-D-fructopyranosyl-(2->6)-[D-fructofuranosyl-(2->6)]n-D-fructofuranoside = beta-D-fructopyranosyl-(2->6)-[D-fructofuranosyl-(2->6)](n-1)-D-fructofuranoside + di-beta-D-fructofuranose 2,6':2',6-dianhydride. This reaction is the production of di-beta-D-fructofuranose 2,6':2',6-dianhydride (DFA IV) by successively eliminating the diminishing (2->6)-beta-D-fructan (levan) chain from the terminal D-fructosyl-D-fructosyl disaccharide." [EC:4.2.2.16]
GO:0033997
name: inulin fructotransferase (DFA-I-forming) activity
def: "Catalysis of the reaction: [(2->1)-beta-D-fructosyl](n) = [(2->1)-beta-D-fructosyl](n-1) + alpha-D-fructofuranose-beta-D-fructofuranose 1,2':1,2'-dianhydride. This reaction is the production of alpha-D-fructofuranose beta-D-fructofuranose 1,2':2,1'-dianhydride (DFA I) by successively eliminating the diminishing (2->1)-beta-D-fructan (inulin) chain from the terminal D-fructosyl-D-fructosyl disaccharide." [EC:4.2.2.17]
GO:0033998
name: inulin fructotransferase (DFA-III-forming) activity
def: "Catalysis of the reaction: [(2->1)-beta-D-fructosyl](n) = [(2->1)-beta-D-fructosyl](n-1) + alpha-D-fructofuranose beta-D-fructofuranose 1,2':2,3'-dianhydride. This reaction is the production of alpha-D-fructofuranose beta-D-fructofuranose 1,2':2,3'-dianhydride (DFA III) by successively eliminating the diminishing (2->1)-beta-D-fructan (inulin) chain from the terminal D-fructosyl-D-fructosyl disaccharide." [EC:4.2.2.18]
GO:0033999
name: chondroitin B lyase activity
def: "Catalysis of the reaction: dermatan sulfate = n 4-deoxy-beta-D-gluc-4-enuronosyl-(1,3)-N-acetyl-D-galactosamine 4-sulfate. This reaction is the eliminative cleavage of dermatan sulfate containing 1,4-beta-D-hexosaminyl and 1,3-beta-D-glucurosonyl or 1,3-alpha-L-iduronosyl linkages to disaccharides containing 4-deoxy-beta-D-gluc-4-enuronosyl groups to yield a 4,5-unsaturated dermatan-sulfate disaccharide (DeltaUA-GalNAC-4S). Chondroitin sulfate B is also known as dermatan sulfate." [EC:4.2.2.19]
GO:0034000
name: chondroitin-sulfate-ABC endolyase activity
def: "Catalysis of the endolytic cleavage of beta-1,4-galactosaminic bonds between N-acetylgalactosamine and either D-glucuronic acid or L-iduronic acid to produce a mixture of Delta4-unsaturated oligosaccharides of different sizes that are ultimately degraded to Delta4-unsaturated tetra- and disaccharides." [EC:4.2.2.20]
GO:0034001
name: chondroitin-sulfate-ABC exolyase activity
def: "Catalysis of the exolytic cleavage of disaccharide residues from the non-reducing ends of both polymeric chondroitin sulfates and their oligosaccharide fragments." [EC:4.2.2.21]
GO:0034002
name: (R)-limonene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = (4R)-limonene + diphosphate." [EC:4.2.3.20, RHEA:10943]
GO:0034003
name: vetispiradiene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = diphosphate + vetispiradiene." [EC:4.2.3.21, RHEA:10343]
GO:0034004
name: germacradienol synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate + H2O = (1E,4S,5E,7R)-germacra-1(10),5-dien-11-ol + diphosphate." [EC:4.2.3.22]
GO:0034005
name: germacrene-A synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (+)-(R)-germacrene A + diphosphate." [EC:4.2.3.23, RHEA:12519]
GO:0034006
name: amorpha-4,11-diene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = amorpha-4,11-diene + diphosphate." [EC:4.2.3.24, RHEA:18328]
GO:0034007
name: S-linalool synthase activity
def: "Catalysis of the reaction: geranyl diphosphate + H(2)O = (S)-linalool + diphosphate." [EC:4.2.3.25, RHEA:24119]
GO:0034008
name: R-linalool synthase activity
def: "Catalysis of the reaction: geranyl diphosphate + H(2)O = (R)-linalool + diphosphate." [EC:4.2.3.26, RHEA:15812]
GO:0034009
name: isoprene synthase activity
def: "Catalysis of the reaction: dimethylallyl diphosphate = diphosphate + isoprene." [EC:4.2.3.27, RHEA:13372]
GO:0034010
name: sulfolactate sulfo-lyase activity
def: "Catalysis of the reaction: 3-sulfolactate = pyruvate + sulfite." [EC:4.4.1.24, RHEA:21431]
GO:0034011
name: L-cysteate sulfo-lyase activity
def: "Catalysis of the reaction: L-cysteate + H(2)O = NH(4)(+) + pyruvate + sulfite." [EC:4.4.1.25, RHEA:13444]
GO:0034012
name: FAD-AMP lyase (cyclizing) activity
def: "Catalysis of the reaction: FAD = AMP + riboflavin cyclic-4',5'-phosphate." [EC:4.6.1.15]
GO:0034013
name: aliphatic aldoxime dehydratase activity
def: "Catalysis of the reaction: an aliphatic aldoxime = an aliphatic nitrile + H2O." [EC:4.99.1.5]
GO:0034015
name: L-ribulose-5-phosphate 3-epimerase activity
def: "Catalysis of the reaction: L-ribulose 5-phosphate = L-xylulose 5-phosphate." [EC:5.1.3.22]
GO:0034016
name: polyenoic fatty acid isomerase activity
def: "Catalysis of the reaction: all-cis-icosa-5,8,11,14,17-pentaenoate = (5Z,7E,9E,14Z,17Z)-icosapentaenoate." [EC:5.3.3.13, RHEA:14892]
GO:0034017
name: trans-2-decenoyl-acyl-carrier-protein isomerase activity
def: "Catalysis of the reaction: trans-dec-2-enoyl-[acyl-carrier protein] = cis-dec-3-enoyl-[acyl-carrier protein]." [EC:5.3.3.14, IMG:02508]
GO:0034018
name: ascopyrone tautomerase activity
def: "Catalysis of the reaction: 1,5-anhydro-4-deoxy-D-glycero-hex-3-en-2-ulose = 1,5-anhydro-4-deoxy-D-glycero-hex-1-en-3-ulose." [EC:5.3.3.15]
GO:0034019
name: capsanthin/capsorubin synthase activity
def: "OBSOLETE. Catalysis of the reactions: violaxanthin = capsorubin, and antheraxanthin = capsanthin." [EC:5.3.99.8]
GO:0034020
name: neoxanthin synthase activity
def: "Catalysis of the reaction: all-trans-violaxanthin = all-trans-neoxanthin." [EC:5.3.99.9, RHEA:10131]
GO:0034022
name: 3-(hydroxyamino)phenol mutase activity
def: "Catalysis of the reaction: 3-hydroxyaminophenol = aminohydroquinone." [EC:5.4.4.3, RHEA:20580]
GO:0034023
name: 5-(carboxyamino)imidazole ribonucleotide mutase activity
def: "Catalysis of the reaction: 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole = 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate." [EC:5.4.99.18]
GO:0034024
name: glutamate-putrescine ligase activity
def: "Catalysis of the reaction: L-glutamate + ATP + putrescine = gamma-L-glutamylputrescine + ADP + 2 H(+) + phosphate." [EC:6.3.1.11, RHEA:13636]
GO:0034025
name: D-aspartate ligase activity
def: "Catalysis of the reaction: ATP + D-aspartate + [beta-GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)]n = [beta-GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-6-N-(beta-D-Asp)-L-Lys-D-Ala-D-Ala)]n + ADP + phosphate." [EC:6.3.1.12]
GO:0034026
name: L-amino-acid alpha-ligase activity
def: "Catalysis of the reaction: ATP + an L-amino acid + an L-amino acid = ADP + phosphate + L-aminoacyl-L-amino acid." [EC:6.3.2.28]
GO:0034027
name: (carboxyethyl)arginine beta-lactam-synthase activity
def: "Catalysis of the reaction: N(2)-(2-carboxyethyl)-L-arginine + ATP = AMP + deoxyamidinoproclavaminate + diphosphate + 2 H(+)." [EC:6.3.3.4, RHEA:23623]
GO:0034028
name: 5-(carboxyamino)imidazole ribonucleotide synthase activity
def: "Catalysis of the reaction: 5-amino-1-(5-phospho-D-ribosyl)imidazole + ATP + bicarbonate = 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole + ADP + 3 H(+) + phosphate." [EC:6.3.4.18, RHEA:19320]
GO:0034029
name: 2-oxoglutarate carboxylase activity
def: "Catalysis of the reaction: 2-oxoglutarate + ATP + bicarbonate = ADP + 2 H(+) + oxalosuccinate + phosphate." [EC:6.4.1.7, RHEA:20428]
GO:0034038
name: deoxyhypusine synthase activity
def: "Catalysis of the reaction: [eIF5A-precursor]-lysine + spermidine = [eIF5A-precursor]-deoxyhypusine + propane-1,3-diamine. Four sub-reactions have been identified,in which the intermediates remain tightly associated with the enzyme: spermidine + NAD+ = dehydrospermidine + NADH; dehydrospermidine + [enzyme]-lysine = N-(4-aminobutylidene)-[enzyme]-lysine + propane-1,3-diamine; N-(4-aminobutylidene)-[enzyme]-lysine + [eIF5A-precursor]-lysine = N-(4-aminobutylidene)-[eIF5A-precursor]-lysine + [enzyme]-lysine; N-(4-aminobutylidene)-[eIF5A-precursor]-lysine + NADH + H+ = [eIF5A-precursor]-deoxyhypusine + NAD+." [EC:2.5.1.46, GOC:pde, MetaCyc:2.5.1.46-RXN]
GO:0034039
name: 8-oxo-7,8-dihydroguanine DNA N-glycosylase activity
def: "Catalysis of the removal of 8-oxo-7,8-dihydroguanine bases by cleaving the N-C1' glycosidic bond between the oxidized purine and the deoxyribose sugar." [GOC:mah, PMID:17641464]
GO:0034040
name: lipid-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + lipid(in) = ADP + phosphate + lipid(out)." [GOC:BHF, GOC:rl]
GO:0034041
name: sterol-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + sterol(in) = ADP + phosphate + sterol(out)." [GOC:BHF, GOC:rl]
GO:0034042
name: 5-formyluracil DNA N-glycosylase activity
def: "Catalysis of the removal of 5-formyluracil bases by cleaving the N-C1' glycosidic bond between the oxidized pyrimidine and the deoxyribose sugar." [GOC:mah, PMID:17641464]
GO:0034043
name: 5-hydroxymethyluracil DNA N-glycosylase activity
def: "Catalysis of the removal of 5-hydroxymethyluracil bases by cleaving the N-C1' glycosidic bond between the oxidized pyrimidine and the deoxyribose sugar." [GOC:mah, PMID:17641464]
GO:0034046
name: poly(G) binding
def: "Interacting selectively and non-covalently with a sequence of guanine residues in an RNA molecule." [GOC:mah]
GO:0034056
name: estrogen response element binding
def: "Interacting selectively and non-covalently with the estrogen response element (ERE), a conserved sequence found in the promoters of genes whose expression is regulated in response to estrogen." [GOC:ecd, PMID:15036253, PMID:17975005]
GO:0034057
name: RNA strand-exchange activity
def: "Facilitates the displacement of one strand of an RNA-RNA duplex and its replacement with a different strand of higher complementarity." [GOC:mcc, PMID:9769100]
GO:0034061
name: DNA polymerase activity
def: "Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1); the synthesis of DNA from deoxyribonucleotide triphosphates in the presence of a nucleic acid template and a 3'hydroxyl group." [EC:2.7.7.7, GOC:mah]
GO:0034062
name: RNA polymerase activity
def: "Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1); the synthesis of RNA from ribonucleotide triphosphates in the presence of a nucleic acid template." [EC:2.7.7.6, GOC:mah]
GO:0034068
name: aminoglycoside nucleotidyltransferase activity
def: "Catalysis of the reaction: nucleoside triphosphate + aminoglycoside = diphosphate + nucleotidylaminoglycoside." [GOC:cb]
GO:0034069
name: aminoglycoside N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + aminoglycoside = CoA + N-acetylaminoglycoside." [GOC:cb]
GO:0034070
name: aminoglycoside 1-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + aminoglycoside = CoA + 1-N-acetylaminoglycoside. This is acetylation of the 1-amino group of the central deoxystreptamine ring." [GOC:cb]
GO:0034071
name: aminoglycoside phosphotransferase activity
def: "Catalysis of the reaction: ATP + aminoglycoside = ADP + phosphoaminoglycoside." [GOC:cb, GOC:mah]
GO:0034072
name: squalene cyclase activity
def: "Catalysis of the reaction: squalene = triterpene." [GOC:cb, PMID:18033581]
GO:0034073
name: tetrahymanol cyclase activity
def: "Catalysis of the reaction: squalene = tetrahymanol." [CHEBI:9493, GOC:cb, PMID:18033581]
GO:0034074
name: marneral synthase activity
def: "Catalysis of the reaction: oxidosqualene = marneral." [GOC:cb, http://www.wiley-vch.de/contents/jc_2002/2006/z503420_s.pdf, PMID:16425307, PMID:18033581]
GO:0034075
name: arabidiol synthase activity
def: "Catalysis of the reaction: oxidosqualene + H2O = arabidiol ((13R,14R,17E)-malabarica-17,21-diene-3beta,14-diol)." [GOC:cb, PMID:16774269, PMID:17474751]
GO:0034076
name: cucurbitadienol synthase activity
def: "Catalysis of the reaction: oxidosqualene = cucurbitadienol." [GOC:cb, PMID:18033581]
GO:0034084
name: steryl deacetylase activity
def: "Catalysis of the hydrolysis of an acetyl group or groups from an acetylated sterol." [GOC:rb, PMID:18034159]
GO:0034185
name: apolipoprotein binding
def: "Interacting selectively and non-covalently with an apolipoprotein, the protein component of a lipoprotein complex." [GOC:BHF, GOC:rl]
GO:0034186
name: apolipoprotein A-I binding
def: "Interacting selectively and non-covalently with apolipoprotein A-I." [GOC:BHF, GOC:rl]
GO:0034187
name: apolipoprotein E binding
def: "OBSOLETE. Interacting selectively and non-covalently with apolipoprotein E." [GOC:BHF, GOC:rl]
GO:0034188
name: apolipoprotein A-I receptor activity
def: "Combining with apolipoprotein A-I and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:BHF, GOC:rl, GOC:signaling, PMID:16443932]
GO:0034189
name: very-low-density lipoprotein particle binding
def: "Interacting selectively and non-covalently with a very-low-density lipoprotein particle, a triglyceride-rich lipoprotein particle that is typically composed of APOB100, APOE and APOCs and has a density of about 1.006 g/ml and a diameter of between 20-80 nm." [GOC:BHF, GOC:mah]
GO:0034190
name: apolipoprotein receptor binding
def: "Interacting selectively and non-covalently with an apolipoprotein receptor." [GOC:BHF, GOC:rl]
GO:0034191
name: apolipoprotein A-I receptor binding
def: "Interacting selectively and non-covalently with an apolipoprotein A-I receptor." [GOC:BHF, GOC:rl]
GO:0034200
name: D,D-heptose 1,7-bisphosphate phosphatase activity
def: "Catalysis of the reaction: D-beta-D-heptose 1,7-bisphosphate + H2O = D-beta-D-heptose 1-phosphate + phosphate." [MetaCyc:RXN0-4361, PMID:11279237]
GO:0034202
name: glycolipid-translocating activity
def: "Catalysis of the movement of glycolipids from one membrane face to the other (glycolipid 'flippase' activity)." [GOC:krc, PMID:11807558]
GO:0034211
name: GTP-dependent protein kinase activity
def: "Catalysis of the reaction: ATP + a protein serine/threonine = ADP + protein serine/threonine phosphate, dependent on the presence of GTP." [GOC:ecd, PMID:17200152]
GO:0034212
name: peptide N-acetyltransferase activity
def: "Catalysis of the acetylation of an amino acid residue of a peptide or protein, according to the reaction: acetyl-CoA + peptide = CoA + N-acetylpeptide." [GOC:mah]
GO:0034215
name: thiamine:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: thiamine(out) + H+(out) = thiamine(in) + H+(in)." [GOC:mah]
GO:0034216
name: high-affinity thiamine:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: thiamine(out) + H+(out) = thiamine(in) + H+(in). In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:mah]
GO:0034228
name: ethanolamine transmembrane transporter activity
def: "Catalysis of the transfer of ethanolamine from one side of the membrane to the other. Ethanolamine (2-aminoethanol, monoethanolamine) is an amino alcohol that occurs widely in living organisms as a constituent of certain types of phospholipids, such as phosphatidylethanolamine." [GOC:rn, PMID:3514579]
GO:0034235
name: GPI anchor binding
def: "Interacting selectively and non-covalently with any glycosylphosphatidylinositol anchor. GPI anchors serve to attach membrane proteins to the lipid bilayer of cell membranes." [GOC:vw]
GO:0034236
name: protein kinase A catalytic subunit binding
def: "Interacting selectively and non-covalently with one or both of the catalytic subunits of protein kinase A." [GOC:mah]
GO:0034237
name: protein kinase A regulatory subunit binding
def: "Interacting selectively and non-covalently with one or both of the regulatory subunits of protein kinase A." [GOC:mah]
GO:0034246
name: mitochondrial RNA polymerase binding promoter specificity activity
def: "Interacting selectively and non-covalently with a mitochondrial RNA polymerase to form a holoenzyme complex and also, while present in the holoenzyme, interacting with promoter sequences in order to confer sequence specific recognition of mitochondrial promoter DNA sequence motifs." [GOC:txnOH]
GO:0034256
name: chlorophyll(ide) b reductase activity
def: "Catalysis of the reaction: 71-hydroxychlorophyll(ide) a + NAD(P)+ = chlorophyll(ide) b + NAD(P)H + H+." [EC:1.1.1.294, MetaCyc:RXN-7678]
GO:0034257
name: nicotinamide riboside transmembrane transporter activity
def: "Catalysis of the transfer of nicotinamide riboside, which is a pyridine-3-carboxamide covalently bonded to a ribose sugar, from one side of a membrane to the other." [GOC:se]
GO:0034277
name: ent-cassa-12,15-diene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-cassa-12,15-diene + diphosphate." [EC:4.2.3.28, RHEA:25535]
GO:0034278
name: stemar-13-ene synthase activity
def: "Catalysis of the reaction: 9-alpha-copalyl diphosphate = stemar-13-ene + diphosphate." [MetaCyc:RXN-4882, RHEA:25555]
GO:0034279
name: syn-pimara-7,15-diene synthase activity
def: "Catalysis of the reaction: 9-alpha-copalyl diphosphate = 9-beta-pimara-7,15-diene + diphosphate." [MetaCyc:RXN-4883, RHEA:25563]
GO:0034280
name: ent-sandaracopimaradiene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-sandaracopimara-8(14),15-diene + diphosphate." [MetaCyc:RXN-4884, RHEA:25539]
GO:0034281
name: ent-isokaurene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-isokaurene + diphosphate." [PMID:17141283]
GO:0034282
name: ent-pimara-8(14),15-diene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-pimara-8(14),15-diene + diphosphate." [RHEA:25543]
GO:0034283
name: syn-stemod-13(17)-ene synthase activity
def: "Catalysis of the reaction: 9-alpha-copalyl diphosphate = stemod-13(17)-ene + diphosphate." [RHEA:25559]
GO:0034290
name: holin activity
def: "Catalysis of the temporally-regulated formation of a transmembrane protein hole that directly or indirectly allows murein hydrolyases to access their cell wall substrate." [GOC:jh2, PMID:1406491]
GO:0034291
name: canonical holin activity
def: "Catalysis of the temporally-regulated formation of a non-specific transmembrane protein hole that allows release of a fully-folded phage-encoded endolysin (murein-degradase) from the cell." [GOC:jh2, GOC:mah, PMID:1406491]
GO:0034292
name: pinholin activity
def: "Catalysis of the temporally-regulated formation of a transmembrane protein hole that allows the passage of ions, causing a lethal, irreversible depolarization of the cell membrane." [GOC:jh2, GOC:mah, PMID:1406491]
GO:0034317
name: nicotinic acid riboside kinase activity
def: "Catalysis of the reaction: ATP + nicotinic acid riboside = ADP + nicotinic acid mononucleotide." [PMID:17914902]
GO:0034318
name: alcohol O-acyltransferase activity
def: "Catalysis of the transfer of an acyl group to an oxygen atom on an alcohol acceptor molecule." [GOC:mah]
GO:0034319
name: alcohol O-butanoyltransferase activity
def: "Catalysis of the reaction: butanoyl-CoA + an alcohol = CoA + a butyl ester." [GOC:mah, PMID:16361250]
GO:0034320
name: alcohol O-hexanoyltransferase activity
def: "Catalysis of the reaction: hexanoyl-CoA + an alcohol = CoA + a hexyl ester." [GOC:mah, PMID:16361250]
GO:0034321
name: alcohol O-octanoyltransferase activity
def: "Catalysis of the reaction: octanoyl-CoA + an alcohol = CoA + an octyl ester." [GOC:mah, PMID:16361250]
GO:0034322
name: alcohol O-decanoyltransferase activity
def: "Catalysis of the reaction: decanoyl-CoA + an alcohol = CoA + a decyl ester." [GOC:mah, PMID:16361250]
GO:0034323
name: O-butanoyltransferase activity
def: "Catalysis of the transfer of a butyl group to an oxygen atom on the acceptor molecule." [GOC:mah]
GO:0034324
name: O-hexanoyltransferase activity
def: "Catalysis of the transfer of a hexyl group to an oxygen atom on the acceptor molecule." [GOC:mah]
GO:0034325
name: O-decanoyltransferase activity
def: "Catalysis of the transfer of a decyl group to an oxygen atom on the acceptor molecule." [GOC:mah]
GO:0034326
name: butanoyltransferase activity
def: "Catalysis of the transfer of a butanoyl (CH3-[CH2]2-CO-) group to an acceptor molecule." [GOC:mah]
GO:0034327
name: hexanoyltransferase activity
def: "Catalysis of the transfer of a hexanoyl (CH3-[CH2]4-CO-) group to an acceptor molecule." [GOC:mah]
GO:0034328
name: decanoyltransferase activity
def: "Catalysis of the transfer of a decanoyl (CH3-[CH2]8-CO-) group to an acceptor molecule." [GOC:mah]
GO:0034335
name: DNA supercoiling activity
def: "OBSOLETE. Catalysis of the formation of positive or negative supercoils in a DNA molecule or region thereof; may be associated with a geometric or topological change in the DNA molecule." [GOC:mah]
GO:0034336
name: misfolded RNA binding
def: "Interacting selectively and non-covalently with an RNA molecule that has assumed an incorrect conformation." [GOC:mah, PMID:10393192]
GO:0034338
name: short-chain carboxylesterase activity
def: "Catalysis of the reaction: a carboxylic ester + H2O = an alcohol + a carboxylic anion, where the carboxylic chain has 8 or fewer carbon atoms." [GOC:jp]
GO:0034347
name: type III interferon binding
def: "Interacting selectively and non-covalently with a type III interferon. Interferon lambda is the only member of the type III interferon found so far." [GOC:add, ISBN:0126896631, PMID:15546383, PMID:16734557]
GO:0034348
name: type III interferon receptor activity
def: "Combining with a type III interferon and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. Interferon lambda is the only member of the type III interferon found so far." [GOC:add, GOC:signaling, ISBN:0126896631, PMID:15546383, PMID:16734557]
GO:0034353
name: RNA pyrophosphohydrolase activity
def: "Catalysis of the removal of a 5' terminal pyrophosphate from the 5'-triphosphate end of an RNA, leaving a 5'-monophosphate end." [GOC:jh2, PMID:17612492, PMID:18202662]
GO:0034386
name: 4-aminobutyrate:2-oxoglutarate transaminase activity
def: "Catalysis of the reaction: 4-aminobutanoate + 2-oxoglutarate = succinate semialdehyde + L-glutamate." [EC:2.6.1.19, GOC:mah]
GO:0034387
name: 4-aminobutyrate:pyruvate transaminase activity
def: "Catalysis of the reaction: 4-aminobutanoate + pyruvate = succinate semialdehyde + alanine." [EC:2.6.1.19, GOC:mah]
GO:0034416
name: bisphosphoglycerate phosphatase activity
def: "Catalysis of the reaction: 2,3-diphosphoglycerate + H2O = phosphoglycerate + phosphate." [EC:3.1.3.13, EC:3.1.3.80, GOC:mah, PMID:18413611]
GO:0034417
name: bisphosphoglycerate 3-phosphatase activity
def: "Catalysis of the reaction: 2,3-diphosphoglycerate + H2O = 2-phospho-D-glycerate + phosphate." [EC:3.1.3.80, GOC:mah, PMID:18413611]
GO:0034419
name: L-2-hydroxyglutarate oxidase activity
def: "Catalysis of the reaction: L-2-hydroxyglutarate + O2 = 2-oxoglutarate + hydrogen peroxide." [PMID:18390652]
GO:0034431
name: bis(5'-adenosyl)-hexaphosphatase activity
def: "Catalysis of the reaction: P1-P6-bis(5'-adenosyl) hexaphosphate + H2O = AMP + adenosine 5'-pentaphosphate." [PMID:10085096, PMID:9450008]
GO:0034432
name: bis(5'-adenosyl)-pentaphosphatase activity
def: "Catalysis of the reaction: P1-P6-bis(5'-adenosyl) pentaphosphate + H2O = AMP + adenosine 5'-tetraphosphate." [PMID:10085096, PMID:9450008]
GO:0034437
name: glycoprotein transporter activity
def: "Enables the directed movement of a glycoprotein, any protein that contains covalently bound glycose (i.e. monosaccharide) residues, into, out of or within a cell, or between cells." [GOC:BHF, GOC:mah, GOC:rl]
GO:0034450
name: ubiquitin-ubiquitin ligase activity
def: "Isoenergetic transfer of ubiquitin from one protein to an existing ubiquitin chain via the reaction X-ubiquitin + Y-ubiquitin -> Y-ubiquitin-ubiquitin + X, where both the X-ubiquitin and Y-ubiquitin-ubiquitin linkages are thioester bonds between the C-terminal glycine of ubiquitin and a sulfhydryl side group of a cysteine residue." [GOC:mah, GOC:mcc, PMID:10089879, PMID:17190603]
GO:0034452
name: dynactin binding
def: "Interacting selectively and non-covalently with any part of a dynactin complex; dynactin is a large protein complex that activates dynein-based motor activity." [GOC:BHF, GOC:mah]
GO:0034458
name: 3'-5' RNA helicase activity
def: "Catalysis of the unwinding of an RNA helix in the direction 3' to 5'." [GOC:mah]
GO:0034459
name: ATP-dependent 3'-5' RNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of an RNA helix in the direction 3' to 5'." [GOC:jp]
GO:0034479
name: phosphatidylglycerol phospholipase C activity
def: "Catalysis of the reaction: a phosphatidylglycerol + H2O = 1,2-diacylglycerol + glycerol 3-phosphate." [GOC:mah, PMID:18434318]
GO:0034480
name: phosphatidylcholine phospholipase C activity
def: "Catalysis of the reaction: phosphatidylcholine + H2O = 1,2-diacylglycerol + a choline phosphate." [EC:3.1.4.3, GOC:mah]
GO:0034481
name: chondroitin sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + chondroitin = adenosine 3',5'-bisphosphate + chondroitin sulfate." [EC:2.8.2.17, EC:2.8.2.5, GOC:mah]
GO:0034482
name: chondroitin 2-O-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + chondroitin = adenosine 3',5'-bisphosphate + chondroitin 2'-O-sulfate. Results in sulfation of glucuronic acid and iduronic acid residues." [PMID:17227754]
GO:0034483
name: heparan sulfate sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + heparan sulfate = adenosine 3',5'-bisphosphate + sulfated heparan sulfate." [GOC:mah]
GO:0034485
name: phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol-3,4,5-trisphosphate + H2O = phosphatidylinositol-3,4-bisphosphate + phosphate." [GOC:pf]
GO:0034511
name: U3 snoRNA binding
def: "Interacting selectively and non-covalently with U3 small nucleolar RNA." [GOC:mah]
GO:0034512
name: box C/D snoRNA binding
def: "Interacting selectively and non-covalently with box C/D small nucleolar RNA." [GOC:mah]
GO:0034513
name: box H/ACA snoRNA binding
def: "Interacting selectively and non-covalently with box H/ACA small nucleolar RNA." [GOC:mah]
GO:0034520
name: 2-naphthaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 2-naphthaldehyde + NAD+ + H2O = 2-naphthoate + NADH + H+." [UM-BBD_reactionID:r0772]
GO:0034521
name: 1-naphthoic acid dioxygenase activity
def: "Catalysis of the reaction: 1-naphthoic acid + NADH + O2 + H+ = cis-1,2-dihydroxy-1,2-dihydro-8-carboxynaphthalene + NAD+." [UM-BBD_reactionID:r0773]
GO:0034522
name: cis-1,2-dihydroxy-1,2-dihydro-8-carboxynaphthalene dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydroxy-1,2-dihydro-8-carboxynaphthalene = 1,2-dihydroxy-8-carboxynaphthalene + 2 H+ + 2 e-." [UM-BBD_reactionID:r0774]
GO:0034523
name: 3-formylsalicylate oxidase activity
def: "Catalysis of the reaction: 3-formylsalicylic acid + O2 + H2O = 2-hydroxyisophthalic acid + hydrogen peroxide." [UM-BBD_reactionID:r0777]
GO:0034524
name: 2-hydroxyisophthalate decarboxylase activity
def: "Catalysis of the reaction: 2-hydroxyisophthalic acid = salicylate + CO2." [UM-BBD_reactionID:r0776]
GO:0034525
name: 1-naphthaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 1-naphthaldehyde + NAD+ + H2O = 1-naphthoic acid + NADH + H+." [UM-BBD_reactionID:r0787]
GO:0034526
name: 2-methylnaphthalene hydroxylase activity
def: "Catalysis of the reaction: 2-methylnaphthalene + NADH + O2 + H+ = 2-hydroxymethylnaphthalene + NAD+ + H2O." [UM-BBD_reactionID:r0788]
GO:0034527
name: 1,2-dihydroxy-8-carboxynaphthalene dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxy-8-carboxynaphthalene + O2 = 2-carboxy-2-hydroxy-8-carboxychromene." [UM-BBD_reactionID:r0790]
GO:0034528
name: 2-carboxy-2-hydroxy-8-carboxychromene isomerase activity
def: "Catalysis of the reaction: 2-carboxy-2-hydroxy-8-carboxychromene = 2-hydroxy-3-carboxybenzalpyruvate." [UM-BBD_reactionID:r0791]
GO:0034529
name: 2-hydroxy-3-carboxy-benzalpyruvate hydratase-aldolase activity
def: "Catalysis of the reaction: 2-hydroxy-3-carboxybenzalpyruvate + H2O = 3-formylsalicylic acid + pyruvate." [UM-BBD_reactionID:r0792]
GO:0034530
name: 4-hydroxymethylsalicyaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 4-hydroxymethylsalicylaldehyde + NAD+ + H2O = 4-hydroxymethylsalicylate + NADH + 2 H+." [UM-BBD_reactionID:r0767]
GO:0034531
name: 2-hydroxy-4-hydroxymethylbenzalpyruvate hydratase-aldolase activity
def: "Catalysis of the reaction: 2-hydroxy-4-hydroxymethylbenzalpyruvate + H2O = pyruvate + 4-hydroxymethylsalicylaldehyde." [UM-BBD_reactionID:r0766]
GO:0034532
name: 2-hydroxy-7-hydroxymethylchromene-2-carboxylate isomerase activity
def: "Catalysis of the reaction: 2-hydroxy-7-hydroxymethylchromene-2-carboxylate = 2-hydroxy-4-hydroxymethylbenzalpyruvate." [UM-BBD_reactionID:r0765]
GO:0034533
name: 1,2-dihydroxy-7-hydroxymethylnaphthalene dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxy-7-hydroxymethylnaphthalene + O2 = 2-hydroxy-7-hydroxymethylchromene-2-carboxylate." [UM-BBD_reactionID:r0764]
GO:0034534
name: 1-methylnaphthalene hydroxylase activity
def: "Catalysis of the reaction: 1-methylnaphthalene + NADH + H+ + O2 = 1-hydroxymethylnaphthalene + NAD+ + H2O." [UM-BBD_reactionID:r0795]
GO:0034535
name: 1,2-dihydroxy-8-methylnaphthalene dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxy-8-methylnaphthalene + O2 = 2-hydroxy-8-methylchromene-2-carboxylate + H+." [UM-BBD_reactionID:r0781]
GO:0034536
name: 2-hydroxy-8-methylchromene-2-carboxylate isomerase activity
def: "Catalysis of the reaction: 2-hydroxy-8-methylchromene-2-carboxylate = 2-hydroxy-3-methylbenzalpyruvate." [UM-BBD_reactionID:r0782]
GO:0034537
name: 2-hydroxy-3-methylbenzalpyruvate hydratase-aldolase activity
def: "Catalysis of the reaction: 2-hydroxy-3-methylbenzalpyruvate + H2O = pyruvate + 3-methylsalicylaldehyde." [UM-BBD_reactionID:r0783]
GO:0034538
name: 3-methylsalicylaldehyde dehydrogenase activity
def: "Catalysis of the reaction: 3-methylsalicylaldehyde + NAD+ = 3-methylsalicylate + NADH + H+." [UM-BBD_reactionID:r0784]
GO:0034539
name: 3,3',5,5'-tetrabromobisphenol A reductive dehalogenase activity
def: "Catalysis of the reaction: 3,3',5,5'-tetrabromobisphenol A + 2 H+ + 2 e- = 3,3',5-tribromobisphenol A + HBr." [UM-BBD_reactionID:r0821]
GO:0034540
name: 3-monobromobisphenol A reductive dehalogenase activity
def: "Catalysis of the reaction: 3-monobromobisphenol A + 2 H+ + 2 e- = bisphenol A + HBr." [UM-BBD_reactionID:r0824]
GO:0034541
name: dimethylarsinite methyltransferase activity
def: "Catalysis of the reaction: dimethylarsenous acid + R2S-CH3 = trimethylarsine oxide + R2SH." [UM-BBD_reactionID:r0806]
GO:0034542
name: trimethylarsine oxidase activity
def: "Catalysis of the reaction: trimethylarsine oxide + 2 H+ + 2 e- = trimethylarsine + H2O." [UM-BBD_reactionID:r0807]
GO:0034543
name: 5-aminosalicylate dioxygenase activity
def: "Catalysis of the reaction: 5-aminosalicylate + O2 = cis-4-amino-6-carboxy-2-oxo-hexa-3,5-dienoate." [UM-BBD_reactionID:r0809]
GO:0034544
name: trans-ACOHDA hydrolase activity
def: "Catalysis of the reaction: trans-4-amino-6-carboxy-2-oxo-hexa-3,5-dienoate + H2O = fumarylpyruvate + NH3." [UM-BBD_reactionID:r0810]
GO:0034545
name: fumarylpyruvate hydrolase activity
def: "Catalysis of the reaction: fumarylpyruvate + H2O = fumarate + pyruvate + H+." [UM-BBD_reactionID:r0811]
GO:0034546
name: 2,4-dichloroaniline reductive dehalogenase activity
def: "Catalysis of the reaction: 2,4-dichloroaniline + 2 H+ + 2 e- = 4-chloroaniline + HCl." [UM-BBD_reactionID:r0819]
GO:0034547
name: N-cyclopropylmelamine deaminase activity
def: "Catalysis of the reaction: cyromazine + H2O = N-cyclopropylammeline + NH3." [UM-BBD_reactionID:r0825]
GO:0034548
name: N-cyclopropylammeline deaminase activity
def: "Catalysis of the reaction: N-cyclopropylammeline + H2O = N-cyclopropylammelide + NH3." [UM-BBD_reactionID:r0826]
GO:0034549
name: N-cyclopropylammelide alkylamino hydrolase activity
def: "Catalysis of the reaction: N-cyclopropylammelide + H2O = cyclopropylamine + cyanuric acid." [UM-BBD_reactionID:r0827]
GO:0034550
name: dimethylarsinate reductase activity
def: "Catalysis of the reaction: dimethylarsinate + 3 H+ + 2 e- = dimethylarsinous acid + H2O." [UM-BBD_reactionID:r0838]
GO:0034554
name: 3,3',5-tribromobisphenol A reductive dehalogenase activity
def: "Catalysis of the reaction: 3,3',5-tribromobisphenol A + 2 H+ + 2 e- = 3,3'-dibromobisphenol A + HBr." [UM-BBD_reactionID:r0842]
GO:0034555
name: 3,3'-dibromobisphenol A reductive dehalogenase activity
def: "Catalysis of the reaction: 3,3'-dibromobisphenol A + 2 H+ + 2 e- = 3-monobromobisphenol A + HBr." [UM-BBD_reactionID:r0844]
GO:0034556
name: nitrobenzoate nitroreductase activity
def: "Catalysis of the reaction: o-nitrobenzoate + NADPH + H+ = o-hydroxylaminobenzoate + NADP+." [UM-BBD_reactionID:r0849]
GO:0034557
name: 2-hydroxylaminobenzoate reductase activity
def: "Catalysis of the reaction: 2-hydroxylaminobenzoate + NAD(P)H = anthranilate + NAD(P)+ + H2O." [MetaCyc:RXN-8848]
GO:0034558
name: technetium (VII) reductase activity
def: "Catalysis of the reaction: pertechnetate ion + 3/2 H2 = technetium (IV) oxide + H2O + OH-." [UM-BBD_reactionID:r0859]
GO:0034559
name: bisphenol A hydroxylase B activity
def: "Catalysis of the reaction: bisphenol A + NADH + H+ + O2 = 1,2-bis(4-hydroxyphenyl)-2-propanol + NAD+ + H2O." [UM-BBD_reactionID:r0860]
GO:0034560
name: bisphenol A hydroxylase A activity
def: "Catalysis of the reaction: bisphenol A + NADH + H+ + O2 = 2,2-bis(4-hydroxyphenyl)-1-propanol + NAD+ + H2O." [UM-BBD_reactionID:r0861]
GO:0034561
name: 1,2-bis(4-hydroxyphenyl)-2-proponol dehydratase activity
def: "Catalysis of the reaction: 1,2-bis(4-hydroxyphenyl)-2-propanol = 4,4'-dihydroxy-alpha-methylstilbene + H2O." [UM-BBD_reactionID:r0862]
GO:0034562
name: 2,2-bis(4-hydroxyphenyl)-1-propanol hydroxylase activity
def: "Catalysis of the reaction: 2,2-bis(4-hydroxyphenyl)-1-propanol + NADH + H+ + O2 = 2,3-bis(4-hydroxyphenyl)-1,2-propanediol + NAD+ + H2O." [UM-BBD_reactionID:r0864]
GO:0034563
name: 2,3-bis(4-hydroxyphenyl)-1,2-propanediol dioxygenase activity
def: "Catalysis of the reaction: 2,3-bis(4-hydroxyphenyl)-1,2-propanediol + O2 = 4-hydroxyphenacyl alcohol + 4-hydroxybenzoate + 2 H+ + 2 e-." [UM-BBD_reactionID:r0867]
GO:0034564
name: 4,4'-dihydroxy-alpha-methylstilbene dioxygenase activity
def: "Catalysis of the reaction: 4,4'-dihydroxy-alpha-methylstilbene + O2 = 4-hydroxybenzaldehyde + 4-hydroxyacetophenone + 2 H+ + 2 e-." [UM-BBD_reactionID:r0866]
GO:0034565
name: 1-nitro-1,2-dihydro-1,3,5-triazine hydrolase activity
def: "Catalysis of the reaction: 1-nitro-1,2-dihydro-1,3,5-triazine + 2 H2O = 1-nitro-1,3,5-triazinane-2,4-diol." [UM-BBD_reactionID:r0872]
GO:0034567
name: chromate reductase activity
def: "Catalysis of the reaction: chromate = chromium (III)." [UM-BBD_reactionID:r0884]
GO:0034568
name: isoproturon dimethylaminedehydrogenase activity
def: "Catalysis of the reaction: isoproturon + H2O = formaldehyde + monodemethylisoproturon + 2 H+ + 2 e-." [UM-BBD_reactionID:r0892]
GO:0034569
name: monodemethylisoproturon dehydrogenase activity
def: "Catalysis of the reaction: monodemethylisoproturon + H2O = hydroxymonodemethylisoproturon + 2 H+ + 2 e-." [UM-BBD_reactionID:r0893]
GO:0034570
name: hydroxymonomethylisoproturon dimethylaminedehydrogenase activity
def: "Catalysis of the reaction: hydroxymonodemethylisoproturon + H2O = formaldehyde + 4'-(2-hydroxyisopropyl)phenylurea + 2 H+ + 2 e-." [UM-BBD_reactionID:r0894]
GO:0034571
name: 4'-(2-hydroxyisopropyl)phenylurea amidohydrolase activity
def: "Catalysis of the reaction: 4'-(2-hydroxyisopropyl)phenylurea + H2O = 4'-(2-hydroxyisopropyl)phenylaniline + carbamic acid." [UM-BBD_reactionID:r0895]
GO:0034572
name: monodemethylisoproturon dimethylaminedehydrogenase activity
def: "Catalysis of the reaction: monodemethylisoproturon + H2O = didemethylisoproturon + formaldehyde + 2 H+ + 2 e-." [UM-BBD_reactionID:r0897]
GO:0034573
name: didemethylisoproturon amidohydrolase activity
def: "Catalysis of the reaction: didemethylisoproturon + H2O = carbamic acid + 4-isopropylaniline." [UM-BBD_reactionID:r0898]
GO:0034574
name: didemethylisoproturon dehydrogenase activity
def: "Catalysis of the reaction: didemethylisoproturon + H2O = 4'-(2-hydroxyisopropyl)phenylurea + 2 H+ + 2 e-." [UM-BBD_reactionID:r0899]
GO:0034575
name: 4-isopropylaniline dehydrogenase activity
def: "Catalysis of the reaction: 4-isopropylaniline + H2O = 4'-(2-hydroxyisopropyl)phenylaniline + 2 H+ + 2 e-." [UM-BBD_reactionID:r0901]
GO:0034576
name: N-isopropylacetanilide amidohydrolase activity
def: "Catalysis of the reaction: N-isopropylacetanilide + OH- = N-isopropylaniline + acetate." [UM-BBD_reactionID:r0913]
GO:0034577
name: N-isopropylacetaniline monooxygenase activity
def: "Catalysis of the reaction: N-isopropylacetanilide + 1/2 O2 = acetanilide + acetone." [UM-BBD_reactionID:r0914]
GO:0034578
name: limonene 8-hydratase activity
def: "Catalysis of the reaction: limonene + H2O = alpha-terpineol." [UM-BBD_reactionID:r0916]
GO:0034579
name: (1-methylpentyl)succinate synthase activity
def: "Catalysis of the reaction: fumarate + n-hexane = (1-methylpentyl)succinate." [UM-BBD_reactionID:r0920]
GO:0034580
name: 4-methyloctanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 4-methyloctanoyl-CoA = 4-methyloct-2-enoyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r0924]
GO:0034581
name: 4-methyloct-2-enoyl-CoA hydratase activity
def: "Catalysis of the reaction: 4-methyloct-2-enoyl-CoA + H2O = 3-hydroxy-4-methyloctanoyl-CoA." [UM-BBD_reactionID:r0925]
GO:0034582
name: 3-hydroxy-4-methyloctanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxy-4-methyloctanoyl-CoA = 4-methyl-3-oxooctanoyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r0926]
GO:0034583
name: 21U-RNA binding
def: "Interacting selectively and non-covalently with a 21U-RNA, a 21-nucleotide RNA characterized by a uridine 5'-monophosphate and a modified 3' end resistant to periodate degradation. 21U-RNAs are derived from distinct, autonomously expressed loci within the genome." [GOC:kmv]
GO:0034584
name: piRNA binding
def: "Interacting selectively and non-covalently with a piRNA, a Piwi-associated RNA, a 24- to 30-nucleotide RNA derived from repeat or complex DNA sequence elements and processed by a Dicer-independent mechanism." [GOC:kmv]
GO:0034590
name: L-hydroxyproline transmembrane transporter activity
def: "Catalysis of the transfer of L-hydroxyproline from one side of a membrane to the other." [GOC:mah, PMID:14502423]
GO:0034593
name: phosphatidylinositol bisphosphate phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol bisphosphate + H2O = phosphatidylinositol phosphate + phosphate." [GOC:mah]
GO:0034594
name: phosphatidylinositol trisphosphate phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol trisphosphate + H2O = phosphatidylinositol bisphosphate + phosphate." [GOC:mah]
GO:0034595
name: phosphatidylinositol phosphate 5-phosphatase activity
def: "Catalysis of the removal of the 5-phosphate group of a phosphatidylinositol phosphate." [GOC:elh]
GO:0034596
name: phosphatidylinositol phosphate 4-phosphatase activity
def: "Catalysis of the removal of the 4-phosphate group of a phosphatidylinositol phosphate." [GOC:mah]
GO:0034597
name: phosphatidylinositol-4,5-bisphosphate 4-phosphatase activity
def: "Catalysis of the reaction: 1-phosphatidyl-myo-inositol 4,5-bisphosphate + H2O = 1-phosphatidyl-1D-myo-inositol 3-phosphate + phosphate." [GOC:mah]
GO:0034598
name: phosphothreonine lyase activity
def: "Catalysis of the removal of the phosphate group from phosphothreonine by cleavage of the C-OP bond with the concomitant abstraction of the alpha proton, generating a double bond-containing product." [PMID:17303758, PMID:18084305]
GO:0034601
name: oxoglutarate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: 2-oxoglutarate + CoA + NAD(P)+ = succinyl-CoA + CO2 + NAD(P)H." [GOC:mah]
GO:0034602
name: oxoglutarate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: 2-oxoglutarate + CoA + NAD+ = succinyl-CoA + CO2 + NADH." [GOC:mah]
GO:0034603
name: pyruvate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: pyruvate + CoA + NAD(P)+ = acetyl-CoA + CO2 + NAD(P)H." [GOC:mah]
GO:0034604
name: pyruvate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: pyruvate + CoA + NAD+ = acetyl-CoA + CO2 + NADH." [GOC:mah]
GO:0034610
name: oligodeoxyribonucleotidase activity
def: "Catalysis of the exonucleolytic cleavage of oligodeoxyribonucleotides to yield deoxyribonucleoside 5'-phosphates." [GOC:mah]
GO:0034611
name: oligoribonucleotidase activity
def: "Catalysis of the exonucleolytic cleavage of oligoribonucleotides to yield ribonucleoside 5'-phosphates." [GOC:mah]
GO:0034617
name: tetrahydrobiopterin binding
def: "Interacting selectively and non-covalently with a tetrahydrobiopterin, 5,6,7,8-tetrahydrobiopterin or a derivative thereof; tetrahydrobiopterins are enzyme cofactors that carry electrons in redox reactions." [CHEBI:15372, GOC:BHF, GOC:mah, GOC:rl]
GO:0034618
name: arginine binding
def: "Interacting selectively and non-covalently with 2-amino-5-(carbamimidamido)pentanoic acid." [CHEBI:29016, GOC:BHF, GOC:rl]
GO:0034632
name: retinol transporter activity
def: "Enables the directed movement of retinol into, out of or within a cell, or between cells. Retinol is vitamin A1, 2,6,6-trimethyl-1-(9'-hydroxy-3',7'-dimethylnona-1',3',5',7'-tetraenyl)cyclohex-1-ene, one of the three components that makes up vitamin A." [GOC:BHF, GOC:mah, GOC:vk]
GO:0034634
name: glutathione transmembrane transporter activity
def: "Enables the directed movement of glutathione, the tripeptide glutamylcysteinylglycine, across a membrane into, out of or within a cell, or between cells." [GOC:mah]
GO:0034639
name: L-amino acid efflux transmembrane transporter activity
def: "Catalysis of the transfer of an L-amino acid from the inside of the cell to the outside of the cell across a membrane." [GOC:mah]
GO:0034647
name: histone demethylase activity (H3-trimethyl-K4 specific)
def: "Catalysis of the removal of a methyl group from trimethylated lysine at position 4 of the histone H3 protein." [GOC:mah, PMID:17550896]
GO:0034648
name: histone demethylase activity (H3-dimethyl-K4 specific)
def: "Catalysis of the removal of a methyl group from dimethylated lysine at position 4 of the histone H3 protein." [GOC:mah, PMID:17550896]
GO:0034649
name: histone demethylase activity (H3-monomethyl-K4 specific)
def: "Catalysis of the reaction: histone H3 N6-methyl-L-lysine (position 4) + O2 + FAD + H2O = histone H3 L-lysine (position 4) + H2O2 + formaldehyde + FADH2. This reaction is the removal of a methyl group from monomethylated lysine at position 4 of the histone H3 protein." [GOC:mah, PMID:16223729]
GO:0034658
name: isopropylmalate transmembrane transporter activity
def: "Catalysis of the transfer of isopropylmalate from one side of the membrane to the other." [GOC:mah]
GO:0034700
name: allulose 6-phosphate 3-epimerase activity
def: "Catalysis of the reaction: D-allulose 6-phosphate = D-fructose 6-phosphate." [GOC:imk]
GO:0034701
name: tripeptidase activity
def: "Catalysis of the hydrolysis of a tripeptide." [GOC:mah]
GO:0034710
name: inhibin complex binding
def: "Interacting selectively and non-covalently with an inhibin complex, a dimer of one inhibin-alpha subunit and one inhibin-beta subunit." [GOC:BHF, GOC:mah]
GO:0034711
name: inhibin binding
def: "Interacting selectively and non-covalently with an inhibin monomer, any of the polypeptides that combine to form activin and inhibin dimers." [GOC:BHF, GOC:mah]
GO:0034713
name: type I transforming growth factor beta receptor binding
def: "Interacting selectively and non-covalently with a type I transforming growth factor beta receptor." [GOC:BHF, GOC:mah]
GO:0034714
name: type III transforming growth factor beta receptor binding
def: "Interacting selectively and non-covalently with a type III transforming growth factor beta receptor." [GOC:BHF, GOC:mah]
GO:0034722
name: gamma-glutamyl-peptidase activity
def: "Catalysis of the cleavage of a gamma-linked glutamate bond." [EC:3.4.19.9, MEROPS_fam:C26]
GO:0034736
name: cholesterol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + cholesterol = a cholesterol ester + CoA." [EC:2.3.1.26, RHEA:17732]
GO:0034737
name: ergosterol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + ergosterol = CoA + ergosterol ester." [GOC:mah]
GO:0034738
name: lanosterol O-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + lanosterol = CoA + lanosterol ester." [GOC:mah]
GO:0034739
name: histone deacetylase activity (H4-K16 specific)
def: "Catalysis of the reaction: histone H4 N6-acetyl-L-lysine (position 16) + H2O = histone H4 L-lysine (position 16) + acetate. This reaction represents the removal of an acetyl group from lysine at position 16 of the histone H4 protein." [EC:3.5.1.17, EC:3.5.1.98, GOC:vw, RHEA:24551]
GO:0034768
name: (E)-beta-ocimene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = (E)-beta-ocimene + diphosphate." [PMID:12624761]
GO:0034778
name: 2-hydroxy-4-isopropenylcyclohexane-1-carboxyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2-hydroxy-4-isopropenylcyclohexane-1-carboxyl-CoA = 4-isopropenyl-2-ketocyclohexane-1-carboxyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r1003]
GO:0034779
name: 4-isopropenyl-2-ketocyclohexane-1-carboxyl-CoA hydrolase activity
def: "Catalysis of the reaction: 4-isopropenyl-2-ketocyclohexane-1-carboxyl-CoA + H2O = 3-isopropenylpimelyl-CoA." [UM-BBD_reactionID:r1004]
GO:0034780
name: glyphosate dehydrogenase activity
def: "Catalysis of the reaction: glyphosate + OH- = glyoxylate + aminomethylphosphonic acid + H+ + 2 e-." [UM-BBD_reactionID:r0073]
GO:0034781
name: N-cyclohexylformamide amidohydrolase activity
def: "Catalysis of the reaction: N-cyclohexylformamide + OH- = cyclohexylamine + formate." [UM-BBD_reactionID:r1030]
GO:0034782
name: dimethylmalonate decarboxylase activity
def: "Catalysis of the reaction: dimethylmalonate + H+ = isobutyrate + CO2." [UM-BBD_reactionID:r1031]
GO:0034783
name: pivalate-CoA ligase activity
def: "Catalysis of the reaction: pivalate + H+ + HSCoA + ATP = pivalyl-CoA + PPi + AMP." [UM-BBD_reactionID:r1032]
GO:0034784
name: pivalyl-CoA mutase activity
def: "Catalysis of the reaction: pivalyl-CoA = 3-methylbutyryl-CoA." [UM-BBD_reactionID:r1033]
GO:0034785
name: salicylate 5-hydroxylase activity
def: "Catalysis of the reaction: salicylate + O2 + NAD(P)H + H+ = gentisate + H2O + NAD(P)+." [UM-BBD_reactionID:r1034]
GO:0034786
name: 9-fluorenone-3,4-dioxygenase activity
def: "Catalysis of the reaction: 9-fluorenone + O2 + 2 H+ + 2 e- = 1-hydro-1,1a-dihydroxy-9-fluorenone." [UM-BBD_reactionID:r1039]
GO:0034787
name: 1-hydro-1,1a-dihydroxy-9-fluorenone dehydrogenase activity
def: "Catalysis of the reaction: 1-hydro-1,1a-dihydroxy-9-fluorenone + H2O = 2,3-dihydroxy-2'-carboxybiphenyl + 3 H+ + 2 e-." [UM-BBD_reactionID:r1040]
GO:0034788
name: 2,3-dihydroxy-2'-carboxybiphenyl 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxy-2'-carboxybiphenyl + O2 = 2-hydroxy-6-oxo-6-(2-carboxyphenyl)-hexa-2,4-dienoate + H+." [UM-BBD_reactionID:r1041]
GO:0034789
name: 2-hydroxy-6-oxo-6-(2-carboxyphenyl)-hexa-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: 2-hydroxy-6-oxo-6-(2-carboxyphenyl)-hexa-2,4-dienoate + H2O = cis-2-hydroxypenta-2,4-dienoate + phthalate + H+." [UM-BBD_reactionID:r1042]
GO:0034790
name: 3,4-dihydroxy-3,4-dihydrofluorene dehydrogenase activity
def: "Catalysis of the reaction: (+)-(3S,4R)-cis-3,4-dihydroxy-3,4-dihydrofluorene = 3,4-dihydroxyfluorene + 2 H+ + 2 e-." [UM-BBD_reactionID:r1043]
GO:0034791
name: isobutylamine N-hydroxylase activity
def: "Catalysis of the reaction: isobutylamine + NADPH + O2 = isobutylhydroxylamine + NADP+ + H2O." [UM-BBD_reactionID:r1053]
GO:0034792
name: hypophosphite dioxygenase activity
def: "Catalysis of the reaction: hypophosphite + 2-oxoglutarate + O2 = succinate + phosphite + CO2." [UM-BBD_reactionID:r1058]
GO:0034793
name: cyclopropanecarboxylate-CoA ligase activity
def: "Catalysis of the reaction: cyclopropanecarboxylate + CoASH = cyclopropanecarboxyl-CoA + OH-." [UM-BBD_reactionID:r1056]
GO:0034794
name: cyclopropanecarboxyl-CoA decyclase activity
def: "Catalysis of the reaction: cyclopropanecarboxyl-CoA = crotonoyl-CoA." [UM-BBD_reactionID:r1057]
GO:0034795
name: cyclohexane monooxygenase activity
def: "Catalysis of the reaction: cyclohexane + O2 + NAD(P)H + H+ = cyclohexanol + NAD(P)+ + H2O." [UM-BBD_reactionID:r1059]
GO:0034796
name: adipate-CoA ligase activity
def: "Catalysis of the reaction: adipate + CoASH = adipyl-CoA + OH-." [UM-BBD_reactionID:r1060]
GO:0034797
name: fosfomycin 2-glutathione ligase activity
def: "Catalysis of the reaction: hydrogen (2R,3S)-3-methyloxiran-2-ylphosphonic acid + glutathione = hydrogen (1R,2R)-1-glutathio-2-hydroxypropylphosphonic acid." [UM-BBD_reactionID:r1073]
GO:0034798
name: fosfomycin 2-L-cysteine ligase activity
def: "Catalysis of the reaction: hydrogen (2R,3S)-3-methyloxiran-2-ylphosphonic acid + L-cysteine = hydrogen (1R,2R)-1-L-cysteine-2-hydroxypropylphosphonic acid." [UM-BBD_reactionID:r1074]
GO:0034799
name: dihydride TNP tautomerase activity
def: "Catalysis of the reaction: TNP dihydride Meisenheimer complex (aci form) = TNP dihydride Meisenheimer complex (nitro form)." [UM-BBD_reactionID:r1070]
GO:0034800
name: trinitrophenol dihydride denitratase activity
def: "Catalysis of the reaction: trinitrophenol dihydride Meisenheimer complex (aci form) = 2,4-dinitrophenol hydride Meisenheimer complex + NO2. Trinitrophenol is also known as TNP and dinitrophenol is also known as DNP." [UM-BBD_reactionID:r1067]
GO:0034801
name: 2,4-dinitrocyclohexanone hydrolase activity
def: "Catalysis of the reaction: 2,4-dinitrocyclohexanone + OH- = 4,6-dinitrohexanoate." [UM-BBD_reactionID:r1069]
GO:0034802
name: branched-chain dodecylbenzene sulfonate monooxygenase activity
def: "Catalysis of the reaction: branched-chain dodecylbenzene sulfonate + 1/2 O2 + H+ = sulfurous acid + branched-chain dodecyl-4-hydroxy-benzene + sulfite." [UM-BBD_reactionID:r1079]
GO:0034803
name: 3-hydroxy-2-naphthoate 2,3-dioxygenase activity
def: "Catalysis of the reaction: 3-hydroxy-2-naphthoate + O2 = (3E)-3-[(6Z)-6-(carboxymethylene)cyclohexa-2,4-dien-1-ylidene]-2-oxopropanate." [UM-BBD_reactionID:r1104]
GO:0034804
name: benzo(a)pyrene 11,12-epoxidase activity
def: "Catalysis of the reaction: benzo(a)pyrene + O2 = benzo(a)pyrene-11,12-epoxide." [UM-BBD_reactionID:r1119]
GO:0034805
name: benzo(a)pyrene-trans-11,12-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: benzo(a)pyrene-trans-11,12-dihydrodiol = 11,12-dihydroxybenzo(a)pyrene + 2 H+ + 2 e-." [UM-BBD_reactionID:r1121]
GO:0034806
name: benzo(a)pyrene 11,12-dioxygenase activity
def: "Catalysis of the reaction: benzo(a)pyrene + O2 = benzo(a)pyrene-cis-11,12-dihydrodiol." [UM-BBD_reactionID:r1124]
GO:0034807
name: 4,5-dihydroxybenzo(a)pyrene methyltransferase activity
def: "Catalysis of the reaction: 4,5-dihydroxybenzo(a)pyrene + C1 unit = hydroxymethoxybenzo(a)pyrene." [UM-BBD_reactionID:r1131]
GO:0034808
name: benzo(a)pyrene 4,5-dioxygenase activity
def: "Catalysis of the reaction: benzo(a)pyrene + O2 = benzo(a)pyrene-cis-4,5-dihydrodiol." [UM-BBD_reactionID:r1126]
GO:0034809
name: benzo(a)pyrene-cis-4,5-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: benzo(a)pyrene-cis-4,5-dihydrodiol = 4,5-dihydroxybenzo(a)pyrene + H2." [UM-BBD_reactionID:r1127]
GO:0034810
name: 4,5-dihydroxybenzo(a)pyrene dioxygenase activity
def: "Catalysis of the reaction: 4,5-dihydroxybenzo(a)pyrene + O2 = 4,5-chrysenedicarboxylate." [UM-BBD_reactionID:r1128]
GO:0034811
name: benzo(a)pyrene 9,10-dioxygenase activity
def: "Catalysis of the reaction: benzo(a)pyrene + O2 = benzo(a)pyrene-cis-9,10-dihydrodiol." [UM-BBD_reactionID:r1132]
GO:0034812
name: 9,10-dihydroxybenzo(a)pyrene dioxygenase activity
def: "Catalysis of the reaction: 9,10-dihydroxybenzo(a)pyrene + O2 = cis-4-(8-hydroxypyren-7-yl)-2-oxobut-3-enoate." [UM-BBD_reactionID:r1134]
GO:0034813
name: benzo(a)pyrene 7,8-dioxygenase activity
def: "Catalysis of the reaction: benzo(a)pyrene + O2 = benzo(a)pyrene-cis-7,8-dihydrodiol." [UM-BBD_reactionID:1137]
GO:0034814
name: 7,8-dihydroxy benzo(a)pyrene dioxygenase activity
def: "Catalysis of the reaction: benzo(a)pyrene-cis-7,8-dihydrodiol + O2 = cis-4-(7-hydroxypyren-8-yl)-2-oxobut-3-enoate." [UM-BBD_reactionID:r1138]
GO:0034815
name: cis-4-(8-hydroxypyren-7-yl)-2-oxobut-3-enoate lyase activity
def: "Catalysis of the reaction: cis-4-(8-hydroxypyren-7-yl)-2-oxobut-3-enoate = 10-oxabenzo(def)chrysen-9-one + formate + H+." [UM-BBD_reactionID:r1135]
GO:0034816
name: anthracene 9,10-dioxygenase activity
def: "Catalysis of the reaction: anthracene + 2 H2O = cis-9,10-dihydroanthracene-9,10-diol." [UM-BBD_reactionID:r1141]
GO:0034817
name: cis-9,10-dihydroanthracene-9,10-diol dehydrogenase activity
def: "Catalysis of the reaction: cis-9,10-dihydroanthracene-9,10-diol = 9,10-anthraquinone + 4 H+ + 4 e-." [UM-BBD_reactionID:r1144]
GO:0034818
name: ADD 9alpha-hydroxylase activity
def: "Catalysis of the reaction: androsta-1,4-diene-3,17-dione + reduced ferredoxin + O2 = 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + H2O + oxidized ferredoxin." [UM-BBD_reactionID:r1149]
GO:0034819
name: 3-HSA hydroxylase activity
def: "Catalysis of the reaction: 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + NADPH + H+ + O2 = 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + NADP+ + H2O." [UM-BBD_reactionID:r1150]
GO:0034820
name: 4,9-DSHA hydrolase activity
def: "Catalysis of the reaction: (3E,1Z)-4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrosta-1(10),2-diene-4-oate + H2O = (2E,4E)-2-hydroxyhexa-2,4-dienoate + 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oate + H+." [UM-BBD_reactionID:r1152]
GO:0034821
name: citronellol dehydrogenase activity
def: "Catalysis of the reaction: citronellol + NAD+ = citronellal + NADH + H+." [UM-BBD_reactionID:r1155]
GO:0034822
name: citronellal dehydrogenase activity
def: "Catalysis of the reaction: citronellal + NAD+ + OH- = citronellate + NADH + H+." [UM-BBD_reactionID:r1156]
GO:0034823
name: citronellyl-CoA ligase activity
def: "Catalysis of the reaction: citronellate + CoASH + ATP = citronellyl-CoA + AMP + PPi." [UM-BBD_reactionID:r1157]
GO:0034824
name: citronellyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: citronellyl-CoA + NAH+ = cis-geranyl-CoA + NADH + H+." [UM-BBD_reactionID:r1159]
GO:0034825
name: tetralin ring-hydroxylating dioxygenase activity
def: "Catalysis of the reaction: tetralin + O2 + NADH + H+ = cis-1,2-dihydroxy-1,2,5,6,7,8-hexahydronaphthalene + NAD+." [UM-BBD_reactionID:r1169]
GO:0034826
name: 1,2-dihydroxy-1,2,5,6,7,8-hexyhadronaphthalene dehydrogenase activity
def: "Catalysis of the reaction: cis-1,2-dihydroxy-1,2,5,6,7,8-hexahydronaphthalene + NAD+ = 1,2-dihydroxy-5,6,7,8-tetrahydronaphthalene + NADH + H+." [UM-BBD_reactionID:r1170]
GO:0034827
name: 1,2-dihydroxy-5,6,7,8-tetrahydronaphthalene extradiol dioxygenase activity
def: "Catalysis of the reaction: 1,2-dihydroxy-5,6,7,8-tetrahydronaphthalene + O2 = 4-(2-oxocyclohexyl)-2-hydroxy-buta-2,4-dienoate + H+." [UM-BBD_reactionID:r1171]
GO:0034828
name: 4-(2-oxocyclohexyl)-2-hydroxy-buta-2,4-dienoate hydrolase activity
def: "Catalysis of the reaction: 4-(2-oxocyclohexyl)-2-hydroxy-buta-2,4-dienoate + H2O = 2-hydroxydec-2,4-diene-1,10-dioate + H+." [UM-BBD_reactionID:r1172]
GO:0034829
name: 2-hydroxydec-2,4-diene-1,10-dioate hydratase activity
def: "Catalysis of the reaction: 2-hydroxydec-2,4-diene-1,10-dioate + H2O = (2Z)-2,4-dihydroxydec-2-enedioate." [UM-BBD_reactionID:r1172]
GO:0034830
name: (2Z)-2,4-dihydroxydec-2-enedioate aldolase activity
def: "Catalysis of the reaction: (2Z)-2,4-dihydroxydec-2-enedioate = pyruvate + 7-oxoheptanoate." [UM-BBD_reactionID:r1174]
GO:0034831
name: (R)-(-)-1,2,3,4-tetrahydronaphthol dehydrogenase activity
def: "Catalysis of the reaction: (R)-1,2,3,4-tetrahydronaphthol = 1,2,3,4-tetrahydronaphthalone + 2 H+ + 2 e-." [UM-BBD_reactionID:r1176]
GO:0034832
name: geranial dehydrogenase activity
def: "Catalysis of the reaction: geranial + NAD+ + OH- = geranylate + NADH + H+." [UM-BBD_reactionID:r1164]
GO:0034833
name: geranylate CoA-transferase activity
def: "Catalysis of the reaction: geranylate + CoASH = trans-geranyl-CoA + OH-." [UM-BBD_reactionID:r1165]
GO:0034834
name: 2-mercaptobenzothiazole dioxygenase activity
def: "Catalysis of the reaction: 2-mercaptobenzothiazole + O2 + 2 H+ + 2 e- = 2-mercaptobenzothiazole-cis-6,7-dihydrodiol." [UM-BBD_reactionID:r1177]
GO:0034835
name: 2-mercaptobenzothiazole monooxygenase activity
def: "Catalysis of the reaction: 2-mercaptobenzothiazole + 1/2 O2 = 6-hydroxy-2-mercaptobenzothiazole." [UM-BBD_reactionID:r1178]
GO:0034836
name: 6-hydroxy-2-mercaptobenzothiazole monooxygenase activity
def: "Catalysis of the reaction: 6-hydroxy-2-mercaptobenzothiazole + 1/2 O2 = 6,7-dihydroxy-2-mercaptobenzothiazole." [UM-BBD_reactionID:r1181]
GO:0034837
name: 2-mercaptobenzothiazole-cis-6,7-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: 2-mercaptobenzothiazole-cis-6,7-dihydrodiol = 6,7-dihydroxy-2-mercaptobenzothiazole + 2 H+ + 2 e-." [UM-BBD_reactionID:r1179]
GO:0034838
name: menthone dehydrogenase activity
def: "Catalysis of the reaction: (-)-(2S,5R)-menthone + NAD+ = (5R)-menth-2-enone + NADH + H+." [UM-BBD_reactionID:r1183]
GO:0034839
name: menth-2-enone hydratase activity
def: "Catalysis of the reaction: (5R)-menth-2-enone + H2O = (5R)-3-hydroxymenthone." [UM-BBD_reactionID:r1184]
GO:0034840
name: 3-hydroxymenthone dehydrogenase activity
def: "Catalysis of the reaction: (5R)-3-hydroxymenthone + NAD+ = mentha-1,3-dione + NADH + H+." [UM-BBD_reactionID:r1185]
GO:0034841
name: mentha-1,3-dione-CoA ligase activity
def: "Catalysis of the reaction: mentha-1,3-dione + CoASH = 3,7-dimethyl-5-oxo-octyl-CoA." [UM-BBD_reactionID:r1186]
GO:0034842
name: thiophene-2-carboxylate-CoA ligase activity
def: "Catalysis of the reaction: thiophene-2-carboxylate + ATP + CoASH = thiophene-2-carboxyl-CoA + AMP + PPi." [UM-BBD_reactionID:r1234]
GO:0034843
name: 2-oxoglutaryl-CoA thioesterase activity
def: "Catalysis of the reaction: 2-oxoglutaryl-CoA + OH- = 2-oxoglutarate + CoASH." [UM-BBD_reactionID:r1238]
GO:0034844
name: naphthyl-2-methyl-succinate CoA-transferase activity
def: "Catalysis of the reaction: naphthyl-2-methyl-succinate + succinyl-CoA = naphthyl-2-methyl-succinyl-CoA + succinate." [UM-BBD_reactionID:r1256]
GO:0034845
name: naphthyl-2-methyl-succinyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: naphthyl-2-methyl-succinyl-CoA = naphthyl-2-methylene-succinyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r1258]
GO:0034846
name: naphthyl-2-methylene-succinyl-CoA lyase activity
def: "Catalysis of the reaction: naphthyl-2-methylene-succinyl-CoA + H2O = naphthyl-2-hydroxymethyl-succinyl-CoA." [UM-BBD_reactionID:r1259]
GO:0034847
name: naphthyl-2-hydroxymethyl-succinyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: naphthyl-2-hydroxymethyl-succinyl-CoA = naphthyl-2-oxomethyl-succinyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r1260]
GO:0034848
name: naphthyl-2-oxomethyl-succinyl-CoA succinyl transferase activity
def: "Catalysis of the reaction: naphthyl-2-oxomethyl-succinyl-CoA + CoASH = 2-naphthoyl-CoA + succinyl-CoA." [UM-BBD_reactionID:r1261]
GO:0034849
name: 2-naphthoate CoA-transferase activity
def: "Catalysis of the reaction: 2-naphthoyl-CoA + OH- = 2-naphthoate + CoASH." [UM-BBD_reactionID:r1262]
GO:0034850
name: isooctane monooxygenase activity
def: "Catalysis of the reaction: isooctane + 1/2 O2 = 2,4,4-trimethyl-1-pentanol." [UM-BBD_reactionID:r1269]
GO:0034851
name: 2,4,4-trimethyl-3-oxopentanoyl-CoA 2-C-propanoyl transferase activity
def: "Catalysis of the reaction: 2,4,4-trimethyl-3-oxopentanoyl-CoA + CoASH = pivalyl-CoA + propanoyl-CoA." [UM-BBD_reactionID:r1274]
GO:0034852
name: 4,4-dimethyl-3-oxopentanal dehydrogenase activity
def: "Catalysis of the reaction: 4,4-dimethyl-3-oxopentanal + H2O = 4,4-dimethyl-3-oxopentanoate + 3 H+ + 2 e-." [UM-BBD_reactionID:r1309]
GO:0034853
name: 2,4,4-trimethyl-3-oxopentanoate decarboxylase activity
def: "Catalysis of the reaction: 2,4,4-trimethyl-3-oxopentanoate + H+ = 2,2-dimethyl-3-pentanone + CO2." [UM-BBD_reactionID:r1278]
GO:0034854
name: 4,4-dimethyl-3-oxopentanoate decarboxylase activity
def: "Catalysis of the reaction: 4,4-dimethyl-3-oxopentanoate + H+ = 3,3-dimethyl-2-butanone + CO2." [UM-BBD_reactionID:r1280]
GO:0034855
name: 4-AD 9alpha-hydroxylase activity
def: "Catalysis of the reaction: androst-4-ene-3,17-dione + O2 + 2 H+ + 2 e- = 9alpha-hydroxy-4-androstene-3,17-dione + H2O." [UM-BBD_reactionID:r1153]
GO:0034856
name: 2-hydroxyhexa-2,4-dienoate hydratase activity
def: "Catalysis of the reaction: (2E,4E)-2-hydroxyhexa-2,4-dienoate + H2O = 4-hydroxy-2-oxohexanoate." [UM-BBD_reactionID:r1281]
GO:0034857
name: 2-(methylthio)benzothiazole monooxygenase activity
def: "Catalysis of the reaction: 2-(methylthio)benzothiazole + 1/2 O2 = 2-(methylsulfinyl)benzothiazole." [UM-BBD_reactionID:r1287]
GO:0034858
name: 2-hydroxybenzothiazole monooxygenase activity
def: "Catalysis of the reaction: 2-hydroxybenzothiazole + 1/2 O2 = 2,6-dihydroxybenzothiazole." [UM-BBD_reactionID:r1291]
GO:0034859
name: benzothiazole monooxygenase activity
def: "Catalysis of the reaction: benzothiazole + 1/2 O2 = 2-hydroxybenzothiazole." [UM-BBD_reactionID:r1292]
GO:0034860
name: 2-mercaptobenzothiazole desulfurase activity
def: "Catalysis of the reaction: 2-mercaptobenzothiazole + reduced acceptor = benzothiazole + hydrogen sulfide + oxidized acceptor." [UM-BBD_reactionID:r1288]
GO:0034861
name: benzothiazole-2-sulfonate hydrolase activity
def: "Catalysis of the reaction: benzothiazole-2-sulfonate + H2O = 2-hydroxybenzothiazole + HSO3-." [UM-BBD_reactionID:r1290]
GO:0034862
name: 2,6-dihydroxybenzothiazole monooxygenase activity
def: "Catalysis of the reaction: 2,6-dihydroxybenzothiazole + 1/2 O2 = 2,6,7-trihydroxybenzothiazole." [UM-BBD_reactionID:r1294]
GO:0034863
name: 2,4,4-trimethyl-1-pentanol dehydrogenase activity
def: "Catalysis of the reaction: 2,4,4-trimethyl-1-pentanol = 2,4,4-trimethylpentanal + 2 H+ + 2 e-." [UM-BBD_reactionID:r1270]
GO:0034864
name: 2,4,4-trimethylpentanal dehydrogenase activity
def: "Catalysis of the reaction: 2,4,4-trimethylpentanal + H2O = 2,4,4-trimethylpentanoate + 3 H+ + 2 e-." [UM-BBD_reactionID:r1275]
GO:0034865
name: 2,4,4-trimethylpentanoate-CoA ligase activity
def: "Catalysis of the reaction: 2,4,4-trimethylpentanoate + CoASH = 2,4,4-trimethylpentanoyl-CoA + OH-." [UM-BBD_reactionID:r1271]
GO:0034866
name: 2,4,4-trimethylpentanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2,4,4-trimethylpentanoyl-CoA = 2,4,4-trimethylpent-2-enoyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r1276]
GO:0034867
name: 2,4,4-trimethylpent-2-enoyl-CoA hydratase activity
def: "Catalysis of the reaction: 2,4,4-trimethylpent-2-enoyl-CoA + H2O = 2,4,4-trimethyl-3-hydroxypentanoyl-CoA." [UM-BBD_reactionID:r1277]
GO:0034868
name: 2,4,4-trimethyl-3-hydroxypentanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2,4,4-trimethyl-3-hydroxypentanoyl-CoA = 2,4,4-trimethyl-3-oxopentanoyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r1273]
GO:0034869
name: 2,4,4-trimethyl-3-oxopentanoyl-CoA thioesterase activity
def: "Catalysis of the reaction: 2,4,4-trimethyl-3-oxopentanoyl-CoA + OH- = 2,4,4-trimethyl-3-oxopentanoate + CoASH." [UM-BBD_reactionID:r1307]
GO:0034870
name: pinacolone 5-monooxygenase activity
def: "Catalysis of the reaction: 2,2-dimethyl-3-pentanone + 1/2 O2 = 1-hydroxy-4,4-dimethylpentan-3-one." [UM-BBD_reactionID:r12979]
GO:0034871
name: 1-hydroxy-4,4-dimethylpentan-3-one dehydrogenase activity
def: "Catalysis of the reaction: 1-hydroxy-4,4-dimethylpentan-3-one = 4,4-dimethyl-3-oxopentanal + 2 H+ + 2 e-." [UM-BBD_reactionID:r1308]
GO:0034872
name: trans-geranyl-CoA isomerase activity
def: "Catalysis of the reaction: trans-geranyl-CoA = cis-geranyl-CoA." [UM-BBD_reactionID:r1310]
GO:0034873
name: thioacetamide S-oxygenase activity
def: "Catalysis of the reaction: thioacetamide + O2 + 2 H+ + 2 e- = thioacetamide S-oxide + H2O." [UM-BBD_reactionID:r1312]
GO:0034874
name: thioacetamide S-oxide S-oxygenase activity
def: "Catalysis of the reaction: thioacetamide S-oxide + O2 + 2 H+ + 2 e- = thioacetamide S,S-dioxide + H2O." [UM-BBD_reactionID:r1313]
GO:0034875
name: caffeine oxidase activity
def: "Catalysis of the reaction: caffeine + O2 + 2 H+ + 2 e- = 1,3,7-trimethyluric acid + H2O." [UM-BBD_reactionID:r1321]
GO:0034876
name: isonicotinic acid hydrazide hydrolase activity
def: "Catalysis of the reaction: isoniazid + H2O = isonicotinate + hydrazine." [UM-BBD_reactionID:r1336]
GO:0034877
name: isonicotinate dehydrogenase activity
def: "Catalysis of the reaction: isonicotinate + acceptor + H2O = 2-hydroxyisonicotinate + reduced acceptor." [UM-BBD_reactionID:r1337]
GO:0034878
name: 2-hydroxyisonicotinate dehydrogenase activity
def: "Catalysis of the reaction: 2-hydroxyisonicotinate + acceptor + H2O = citrazinate + reduced acceptor." [UM-BBD_reactionID:r1338]
GO:0034879
name: 2,3,6-trihydroxyisonicotinate decarboxylase activity
def: "Catalysis of the reaction: 2,3,6-trihydroxyisonicotinate = 2,3,6-trihydroxypyridine + CO2." [UM-BBD_reactionID:r1340]
GO:0034880
name: citrazinate dehydrogenase activity
def: "Catalysis of the reaction: citrazinate + H2O = 2,3,6-trihydroxyisonicotinate + 2 H+ + 2 e-." [UM-BBD_reactionID:r1339]
GO:0034881
name: citrazinate hydrolase activity
def: "Catalysis of the reaction: citrazinate + H2O = cis-aconitamide." [UM-BBD_reactionID:r1343]
GO:0034882
name: cis-aconitamide amidase activity
def: "Catalysis of the reaction: cis-aconitamide + H2O = cis-aconitate + HN3." [UM-BBD_reactionID:r1344]
GO:0034883
name: isonicotinate reductase activity
def: "OBSOLETE. Catalysis of the reaction: isonicotinate + 2 H+ + 2 e- = 1,4-dihydroisonicotinate." [UM-BBD_reactionID:r1347]
GO:0034884
name: gamma-N-formylaminovinylacetaldehyde dehydrogenase activity
def: "OBSOLETE. Catalysis of the reaction: gamma-N-formylaminovinylacetaldehyde + H2O = gamma-N-formylaminovinylacetate + 2 H+ + 2 e-." [UM-BBD_reactionID:r1349]
GO:0034885
name: gamma-N-formylaminovinylacetate hydrolase activity
def: "Catalysis of the reaction: gamma-N-formylaminovinylacetate + H2O = gamma-aminovinylacetate + HCOOH." [UM-BBD_reactionID:r1350]
GO:0034886
name: gamma-aminovinylacetate deaminase activity
def: "Catalysis of the reaction: gamma-aminovinylacetate + H2O = succinic semialdehyde + NH3." [UM-BBD_reactionID:r1351]
GO:0034887
name: 1,4-dihydroisonicotinate 2,3-dioxygenase activity
def: "OBSOLETE. Catalysis of the reaction: 1,4-dihydroisonicotinate + O2 = gamma-N-formylaminovinylacetaldehyde + CO2." [UM-BBD_reactionID:r1348]
GO:0034888
name: endosulfan monooxygenase I activity
def: "Catalysis of the reaction: endosulfan + O2 + 2 H+ + 2 e- = endosulfan sulfate + H2O." [UM-BBD_reactionID:r1382]
GO:0034889
name: endosulfan hemisulfate sulfatase activity
def: "Catalysis of the reaction: endosulfan hemisulfate + H2O = endosulfan monoalcohol + 2 H+ + sulfate." [UM-BBD_reactionID:r1384]
GO:0034890
name: endosulfan diol hydrolyase (cyclizing) activity
def: "Catalysis of the reaction: endosulfan diol = endosulfan ether + H2O." [UM-BBD_reactionID:r1386]
GO:0034891
name: endosulfan diol dehydrogenase activity
def: "Catalysis of the reaction: endosulfan diol = endosulfan hydroxyether + 2 H+ + 2 e-." [UM-BBD_reactionID:r1388]
GO:0034892
name: endosulfan lactone lactonase activity
def: "Catalysis of the reaction: endosulfan lactone + H2O = endosulfan hydroxycarboxylate + H+." [UM-BBD_reactionID:r1389]
GO:0034893
name: N-nitrodimethylamine hydroxylase activity
def: "Catalysis of the reaction: N-nitrodimethylamine + O2 + 2 H+ + 2 e- = N-nitromethylamine + formaldehyde + H2O." [UM-BBD_reactionID:r1395]
GO:0034894
name: 4-hydroxypyridine-3-hydroxylase activity
def: "Catalysis of the reaction: 4-hydroxypyridine + O2 + H+ + NADPH = pyridine-3,4-diol + H2O + NADP+." [UM-BBD_reactionID:r1397]
GO:0034895
name: pyridine-3,4-diol dioxygenase activity
def: "Catalysis of the reaction: pryidine-3,4-diol + O2 = 3-(N-formyl)-formiminopyruvate." [UM-BBD_reactionID:r1398]
GO:0034896
name: 3-formiminopyruvate hydrolase activity
def: "Catalysis of the reaction: 3-formiminopyruvate + H2O = 3-formylpyruvate + HN3." [UM-BBD_reactionID:r1400]
GO:0034897
name: 4-(1-ethyl-1,4-dimethyl-pentyl)phenol monoxygenase activity
def: "Catalysis of the reaction: 4-(1-ethyl-1,4-dimethyl-pentyl)phenol + O2 + 2 H+ + 2 e- = hydroquinone + 3,6-dimethylheptan-3-ol." [UM-BBD_reactionID:r1358]
GO:0034898
name: hexadecyltrimethylammonium chloride monooxygenase activity
def: "Catalysis of the reaction: hexadecyltrimethylammonium chloride + NAD(P)H + H+ + O2 = trimethylamine + hexadecanal + NAD(P)+ + H2O." [UM-BBD_reactionID:r1373]
GO:0034899
name: trimethylamine monooxygenase activity
def: "Catalysis of the reaction: N,N,N-trimethylamine + NADPH + H+ + O2 = N,N,N-trimethylamine N-oxide + NADP+ + H2O." [EC:1.14.13.148, UM-BBD_reactionID:r1407]
GO:0034900
name: 3-(N-formyl)-formiminopyruvate hydrolase activity
def: "Catalysis of the reaction: 3-(N-formyl)-formiminopyruvate + H2O = 3-formiminopyruvate + formate." [UM-BBD_reactionID:r1399]
GO:0034901
name: endosulfan hydroxyether dehydrogenase activity
def: "Catalysis of the reaction: endosulfan hydroxyether = endosulfan lactone + 2 H+ + 2 e-." [UM-BBD_reactionID:r1411]
GO:0034902
name: endosulfan sulfate hydrolase activity
def: "Catalysis of the reaction: endosulfan sulfate + H2O = endosulfan diol + sulfite." [UM-BBD_reactionID:r1387]
GO:0034903
name: endosulfan ether monooxygenase activity
def: "Catalysis of the reaction: endosulfan ether + O2 + 2 H+ + 2 e- = endosulfan hydroxyether + H2O." [UM-BBD_reactionID:r1413]
GO:0034904
name: 5-chloro-2-oxopent-4-enoate hydratase activity
def: "Catalysis of the reaction: 5-chloro-2-oxopent-4-enoate + H2O = 5-chloro-4-hydroxy-2-oxopentanate." [UM-BBD_reactionID:r1436]
GO:0034905
name: 5-chloro-4-hydroxy-2-oxopentanate aldolase activity
def: "Catalysis of the reaction: 5-chloro-4-hydroxy-2-oxopentanate = pyruvate + chloroacetaldehyde." [UM-BBD_reactionID:r1437]
GO:0034906
name: N-isopropylaniline 1,2-dixoxygenase activity
def: "Catalysis of the reaction: N-isopropylaniline + O2 + 2 H+ + NADH = catechol + NAD+ + isopropylamine." [UM-BBD_reactionID:r0721]
GO:0034907
name: acetanilide 1,2-dioxygenase activity
def: "Catalysis of the reaction: acetanilide + O2 + 2 H+ + NADH = catechol + NAD+ + acetamide." [UM-BBD_reactionID:r0723]
GO:0034908
name: 2-chloro-N-isopropylacetanilide 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2-chloro-N-isopropylacetanilide + O2 + 2 H+ + NADH = 2-chloro-N-isopropylacetamide + catechol + NAD+." [UM-BBD_reactionID:r0724]
GO:0034909
name: 6-hydroxypseudooxynicotine dehydrogenase activity
def: "Catalysis of the reaction: 6-hydroxypseudooxynicotine + H2O + OH- = 6-hydroxy-3-succinoylpyridine + 4 H+ + 4 e- + methylamine." [UM-BBD_reactionID:r1441]
GO:0034910
name: 6-hydroxy-3-succinoylpyridine hydrolase activity
def: "Catalysis of the reaction: 6-hydroxy-3-succinoylpyridine + H2O = succinic semialdehyde + 2,5-dihydroxypyridine." [UM-BBD_reactionID:r1442]
GO:0034911
name: phthalate 3,4-dioxygenase activity
def: "Catalysis of the reaction: phthalate + O2 + NADH + H+ = phthalate 3,4-cis-dihydrodiol + NAD+." [UM-BBD_reactionID:r1444]
GO:0034912
name: phthalate 3,4-cis-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: phthalate 3,4-cis-dihydrodiol + NAD+ = 3,4-dihydroxyphthalate + NADH + H+." [UM-BBD_reactionID:r1445]
GO:0034914
name: trinitrophenol hydride denitratase activity
def: "Catalysis of the reaction: trinitrophenol hydride Meisenheimer complex = 2,4-dinitrophenol + nitrite. Trinitrophenol is also known as TNP." [UM-BBD_reactionID:r1448]
GO:0034915
name: 2-methylhexanoyl-CoA C-acetyltransferase activity
def: "Catalysis of the reaction: 4-methyl-3-oxooctanoyl-CoA + CoA = acetyl-CoA + 2-methylhexanoyl-CoA." [UM-BBD_reactionID:r0927]
GO:0034916
name: 2-methylhexanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2-methylhexanoyl-CoA = 2-methylhex-2-enoyl-CoA + 2 H+ + e-." [UM-BBD_reactionID:r0928]
GO:0034917
name: 2-methylhex-2-enoyl-CoA hydratase activity
def: "Catalysis of the reaction: 2-methylhex-2-enoyl-CoA + H2O = 3-hydroxy-2-methylhexanoyl-CoA." [UM-BBD_reactionID:r0929]
GO:0034918
name: 3-hydroxy-2-methylhexanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxy-2-methylhexanoyl-CoA = 2-methyl-3-oxohexanoyl-CoA + 2 H+ + 2 e-." [UM-BBD_reactionID:r0930]
GO:0034919
name: butyryl-CoA 2-C-propionyltransferase activity
def: "Catalysis of the reaction: 2-methyl-3-oxohexanoyl-CoA + CoA = propanoyl-CoA + butyryl-CoA." [UM-BBD_reactionID:r0931]
GO:0034920
name: pyrene dioxygenase activity
def: "Catalysis of the reaction: pyrene + 2 H+ + 2 e- + O2 = cis-4,5-dihydroxy-4,5-dihydropyrene." [UM-BBD_reactionID:r0934]
GO:0034921
name: cis-4,5-dihydroxy-4,5-dihydropyrene dehydrogenase activity
def: "Catalysis of the reaction: cis-4,5-dihydroxy-4,5-dihydropyrene = 4,5-dihydroxypyrene + 2 H+ + 2 e-." [UM-BBD_reactionID:r0935]
GO:0034922
name: 4,5-dihydroxypyrene dioxygenase activity
def: "Catalysis of the reaction: 4,5-dihydroxypyrene + O2 = phenanthrene-4,5-dicarboxylate + 2 H+." [UM-BBD_reactionID:r0936]
GO:0034923
name: phenanthrene-4,5-dicarboxylate decarboxylase activity
def: "Catalysis of the reaction: phenanthrene-4,5-dicarboxylate + H+ = phenanthrene-4-carboxylate + CO2." [UM-BBD_reactionID:r0937]
GO:0034924
name: cis-3,4-phenanthrenedihydrodiol-4-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: cis-3,4-phenanthrenedihydrodiol-4-carboxylate = 3,4-dihydroxyphenanthrene + H+ + 2 e- + CO2." [UM-BBD_reactionID:r0940]
GO:0034925
name: pyrene 4,5-monooxygenase activity
def: "Catalysis of the reaction: pyrene + O2 + 2 H+ + 2 e- = pyrene-4,5-oxide + H2O." [UM-BBD_reactionID:r0941]
GO:0034926
name: pyrene-4,5-epoxide hydrolase activity
def: "Catalysis of the reaction: pyrene-4,5-oxide + H2O = trans-4,5-dihydroxy-4,5-dihydropyrene." [UM-BBD_reactionID:r0942]
GO:0034927
name: pyrene 1,2-monooxygenase activity
def: "Catalysis of the reaction: pyrene + O2 + 2 H+ + 2 e- = pyrene-1,2-oxide + H2O." [UM-BBD_reactionID:r0943]
GO:0034928
name: 1-hydroxypyrene 6,7-monooxygenase activity
def: "Catalysis of the reaction: 1-hydroxypyrene + O2 + 2 H+ + 2 e- = 1-hydroxypyrene-6,7-oxide + H2O." [UM-BBD_reactionID:r0946]
GO:0034929
name: 1-hydroxypyrene 7,8-monooxygenase activity
def: "Catalysis of the reaction: 1-hydroxypyrene + O2 + 2 H+ + 2 e- = 1-hydroxypyrene-7,8-oxide + H2O." [UM-BBD_reactionID:r0949]
GO:0034930
name: 1-hydroxypyrene sulfotransferase activity
def: "Catalysis of the reaction: 1-hydroxypyrene + XSO3- = 1-pyrenylsulfate + HX." [UM-BBD_reactionID:r0952]
GO:0034931
name: 1-hydroxypyrene methyltransferase activity
def: "Catalysis of the reaction: 1-hydroxypyrene + XCH3 = 1-methoxypyrene + HX." [UM-BBD_reactionID:r0953]
GO:0034932
name: 1-methoxypyrene 6,7-monooxygenase activity
def: "Catalysis of the reaction: 1-methoxypyrene + O2 + 2 H+ + 2 e- = 1-methoxypyrene-6,7-oxide + H2O." [UM-BBD_reactionID:r0954]
GO:0034933
name: 1-hydroxy-6-methoxypyrene methyltransferase activity
def: "Catalysis of the reaction: 1-hydroxy-6-methoxypyrene + XCH3 = 1,6-dimethoxypyrene + HX." [UM-BBD_reactionID:r0956]
GO:0034934
name: phenanthrene-4-carboxylate dioxygenase activity
def: "Catalysis of the reaction: phenanthrene-4-carboxylate + 2 H+ + 2 e- + O2 = cis-3,4-phenanthrenedihydrodiol-4-carboxylate." [UM-BBD_reactionID:r0939]
GO:0034935
name: tetrachlorobenzene dioxygenase activity
def: "Catalysis of the reaction: 2,4-dichlorotoluene + NADH + H+ + O2 = 4,6-dichloro-3-methyl-cis-1,2-dihydroxycyclohexa-3,5-diene + NAD+." [UM-BBD_reactionID:r0957]
GO:0034936
name: 4,6-dichloro-3-methylcatechol 1,2-dioxygenase activity
def: "Catalysis of the reaction: 4,6-dichloro-3-methylcatechol + O2 = 3,5-dichloro-3-methyl-cis,cis-muconate + 2 H+." [UM-BBD_reactionID:r0959]
GO:0034937
name: perchlorate reductase activity
def: "Catalysis of the reaction: perchlorate + 2 H+ + 2 e- = chlorate + H2O." [UM-BBD_reactionID:r0980]
GO:0034938
name: pyrrole-2-carboxylate monooxygenase activity
def: "Catalysis of the reaction: pryrole-2-carboxylate + NADH + O2 + H+ = 5-hydroxypyrrole-2-carboxylate + NAD+ + H2O." [UM-BBD_reactionID:r0968]
GO:0034939
name: 5-hydroxypyrrole-2-carboxylate tautomerase activity
def: "Catalysis of the reaction: 5-hydroxypyrrole-2-carboxylate = 5-oxo-4,5-dihydropyrrole-2-carboxylate." [UM-BBD_reactionID:r0969]
GO:0034940
name: 5-oxo-4,5-dihydropyrrole-2-carboxylate amidase activity
def: "Catalysis of the reaction: 5-oxo-4,5-dihydropyrrole-2-carboxylate + 2 H2O = 2-oxoglutarate + NH3." [UM-BBD_reactionID:r0984]
GO:0034941
name: pyrrole-2-carboxylate decarboxylase activity
def: "Catalysis of the reaction: pryrole-2-carboxylate + H+ = pyrrole + CO2." [UM-BBD_reactionID:r0970]
GO:0034942
name: cis-2-methyl-5-isopropylhexa-2,5-dienoate-CoA ligase activity
def: "Catalysis of the reaction: cis-2-methyl-5-isopropylhexa-2,5-dienoic acid + ATP + CoASH = cis-2-methyl-5-isopropylhexa-2,5-dienoyl-CoA + AMP + PPi." [UM-BBD_reactionID:r0988]
GO:0034943
name: trans-2-methyl-5-isopropylhexa-2,5-dienoate-CoA ligase activity
def: "Catalysis of the reaction: trans-2-methyl-5-isopropylhexa-2,5-dienoic acid + ATP + CoASH = trans-2-methyl-5-isopropylhexa-2,5-dienoyl-CoA + AMP + PPi." [UM-BBD_reactionID:r0989]
GO:0034944
name: 3-hydroxy-2,6-dimethyl-5-methylene-heptanoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxy-2,6-dimethyl-5-methylene-heptanoyl-CoA + NAD+ = 2,6-dimethyl-5-methylene-3-oxo-heptanoyl-CoA + NADH+ + H+." [UM-BBD_reactionID:r0986]
GO:0034945
name: 2,6-dimethyl-5-methylene-3-oxo-heptanoyl-CoA C-acetyltransferase activity
def: "Catalysis of the reaction: 2,6-dimethyl-5-methylene-3-oxo-heptanoyl-CoA + CoASH = 3-isopropylbut-3-enoyl-CoA + propanoyl-CoA." [UM-BBD_reactionID:r0987]
GO:0034946
name: 3-isopropylbut-3-enoyl-CoA thioesterase activity
def: "Catalysis of the reaction: 3-isopropylbut-3-enoyl-CoA + H2O = 3-isopropylbut-3-enoic acid + CoASH." [UM-BBD_reactionID:r0994]
GO:0034947
name: terephthalate decarboxylase activity
def: "Catalysis of the reaction: terephthalate + H2O = benzoate + HCO3-." [UM-BBD_reactionID:r0321]
GO:0034948
name: 2,6-dihydroxypseudooxynicotine hydrolase activity
def: "Catalysis of the reaction: 2,6-dihydroxypseudooxynicotine + H2O = 2,6-dihydroxypyridine + 4-methylaminobutyrate." [UM-BBD_reactionID:r0482]
GO:0034949
name: 1,1-dichloroethane reductive dehalogenase activity
def: "Catalysis of the reaction: 1,1-dichloroethane + 2 H+ + 2 e- = chloroethane + HCl." [UM-BBD_reactionID:r1008]
GO:0034950
name: phenylboronic acid monooxygenase activity
def: "Catalysis of the reaction: phenylboronic acid + O2 + 2 H+ + 2 e- = phenol + B(OH)3." [UM-BBD_reactionID:r1020]
GO:0034951
name: o-hydroxylaminobenzoate mutase activity
def: "Catalysis of the reaction: o-hydroxylaminobenzoate = 3-hydroxyanthranilate." [UM-BBD_reactionID:r1026]
GO:0034952
name: malonate semialdehyde decarboxylase activity
def: "Catalysis of the reaction: malonate semialdehyde + H+ = acetaldehyde + CO2." [UM-BBD_reactionID:r0266]
GO:0034953
name: perillyl-CoA hydratase activity
def: "Catalysis of the reaction: perillyl-CoA + H2O = 2-hydroxy-4-isopropenylcyclohexane-1-carboxyl-CoA." [UM-BBD_reactionID:r1002]
GO:0034954
name: diphenyl ether 2,3-dioxygenase activity
def: "Catalysis of the reaction: diphenyl ether + O2 = 2,3-dihydroxydiphenyl ether." [UM-BBD_reactionID:r1450]
GO:0034955
name: 2,3-dihydroxydiphenyl ether dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxydiphenyl ether + O2 + H2O = 2-hydroxymuconate + phenol." [UM-BBD_reactionID:r1451]
GO:0034956
name: diphenyl ether 1,2-dioxygenase activity
def: "Catalysis of the reaction: diphenyl ether + NADH + O2 + H+ = phenol + catechol + NAD+." [UM-BBD_reactionID:r1453]
GO:0034957
name: 3-nitrophenol nitroreductase activity
def: "Catalysis of the reaction: 3-nitrophenol + 2 NADH + 2 H+ = 3-hydroxyaminophenol + 2 NAD+ + H2O." [UM-BBD_reactionID:r1495]
GO:0034958
name: aminohydroquinone monooxygenase activity
def: "Catalysis of the reaction: aminohydroquinone + 2 e- + 2 H+ + 1/2 O2 = 1,2,4-benzenetriol + NH3." [UM-BBD_reactionID:r1497]
GO:0034979
name: NAD-dependent protein deacetylase activity
def: "Catalysis of the removal of one or more acetyl groups from a protein, requiring NAD." [GOC:BHF, GOC:mah]
GO:0034986
name: iron chaperone activity
def: "Assists in the delivery of iron ions to target proteins or compartments." [GOC:vk]
GO:0034987
name: immunoglobulin receptor binding
def: "Interacting selectively and non-covalently with one or more specific sites on an immunoglobulin receptor molecule." [GOC:BHF, GOC:vk]
GO:0034988
name: Fc-gamma receptor I complex binding
def: "Interacting selectively and non-covalently with one or more specific sites on the Fc-gamma receptor I complex. The complex functions primarily as an activating receptor for IgG." [GOC:BHF, GOC:vk]
GO:0035004
name: phosphatidylinositol 3-kinase activity
def: "Catalysis of the reaction: ATP + a phosphatidylinositol = ADP + a phosphatidylinositol 3-phosphate. This reaction is the addition of a phosphate group to phosphatidylinositol or one of its phosphorylated derivatives at the 3' position of the inositol ring." [GOC:bf, PMID:10209156, PMID:9255069]
GO:0035005
name: 1-phosphatidylinositol-4-phosphate 3-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4-phosphate + ATP = 1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.154, RHEA:18376]
GO:0035013
name: myosuppressin receptor activity
def: "Combining with the peptide myosuppressin to initiate a change in cell activity." [GOC:bf]
GO:0035014
name: phosphatidylinositol 3-kinase regulator activity
def: "Modulates the activity of any of the phosphatidylinositol 3-kinases (PI3Ks). Regulatory subunits can link a PI3K catalytic subunit to upstream signaling events and help position the catalytic subunits close to their lipid substrates." [GOC:bf, PMID:9255069]
GO:0035033
name: histone deacetylase regulator activity
def: "Modulates the activity of histone deacetylase." [GOC:bf]
GO:0035034
name: histone acetyltransferase regulator activity
def: "Modulates the activity of histone acetyltransferase." [GOC:bf]
GO:0035035
name: histone acetyltransferase binding
def: "Interacting selectively and non-covalently with the enzyme histone acetyltransferase." [GOC:bf]
GO:0035049
name: juvenile hormone acid methyltransferase activity
def: "Catalysis of the transfer of a methyl group to juvenile hormone acid." [GOC:bf]
GO:0035064
name: methylated histone binding
def: "Interacting selectively and non-covalently with a histone protein in which a residue has been modified by methylation. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:bf, PMID:14585615]
GO:0035091
name: phosphatidylinositol binding
def: "Interacting selectively and non-covalently with any inositol-containing glycerophospholipid, i.e. phosphatidylinositol (PtdIns) and its phosphorylated derivatives." [GOC:bf, ISBN:0198506732, PMID:11395417]
GO:0035100
name: ecdysone binding
def: "Interacting selectively and non-covalently with 20-hydroxyecdysone (ecdysone). Ecdysone is an ecdysteroid produced by the prothoracic glands of immature insects and the ovaries of adult females, which stimulates growth and molting." [GOC:bf, ISBN:0198506732, ISBN:0582227089]
GO:0035173
name: histone kinase activity
def: "Catalysis of the transfer of a phosphate group to a histone. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:bf]
GO:0035174
name: histone serine kinase activity
def: "Catalysis of the transfer of a phosphate group to a serine residue of a histone. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:bf]
GO:0035175
name: histone kinase activity (H3-S10 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-10 residue of the N-terminal tail of histone H3." [GOC:bf, PMID:15041176]
GO:0035184
name: histone threonine kinase activity
def: "Catalysis of the transfer of a phosphate group to a threonine residue of a histone. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:bf]
GO:0035197
name: siRNA binding
def: "Interacting selectively and non-covalently with a small interfering RNA, a 21-23 nucleotide RNA that is processed from double stranded RNA (dsRNA) by an RNAse enzyme." [PMID:15066275, PMID:15066283]
GO:0035198
name: miRNA binding
def: "Interacting selectively and non-covalently with a microRNA, a 21-23 nucleotide RNA that is processed from a stem-loop RNA precursor (pre-miRNA) that is encoded within plant and animal genomes." [PMID:15066283]
GO:0035226
name: glutamate-cysteine ligase catalytic subunit binding
def: "Interacting selectively and non-covalently with the catalytic subunit of glutamate-cysteine ligase." [PMID:12954617]
GO:0035236
name: proctolin receptor activity
def: "Combining with the neuropeptide proctolin, to initiate a change in cell activity." [GOC:bf]
GO:0035237
name: corazonin receptor activity
def: "Combining with the neuropeptide corazonin to initiate a change in cell activity." [GOC:bf]
GO:0035240
name: dopamine binding
def: "Interacting selectively and non-covalently with dopamine, a catecholamine neurotransmitter formed by aromatic-L-amino-acid decarboxylase from 3,4-dihydroxy-L-phenylalanine." [ISBN:0198506732]
GO:0035241
name: protein-arginine omega-N monomethyltransferase activity
def: "Catalysis of the addition of a methyl group to either of the unmethylated terminal nitrogen atoms (also called omega nitrogen) in peptidyl-arginine to form an omega-N-G-monomethylated arginine residue." [PMID:14705965, RESID:AA0069]
GO:0035242
name: protein-arginine omega-N asymmetric methyltransferase activity
def: "Catalysis of the addition of a second methyl group to methylated peptidyl-arginine. Methylation is on the same terminal nitrogen (omega nitrogen) residue that was previously methylated, resulting in asymmetrical peptidyl-N(omega),N(omega)-dimethylated arginine residues." [PMID:14705965, RESID:AA0068, RESID:AA0069]
GO:0035243
name: protein-arginine omega-N symmetric methyltransferase activity
def: "Catalysis of the addition of a second methyl group to methylated peptidyl-arginine. Methylation is on the terminal nitrogen (omega nitrogen) residue that is not already methylated, resulting in symmetrical peptidyl-N(omega),N'(omega)-dimethyled arginine residues." [PMID:14705965, RESID:AA0067, RESID:AA0069]
GO:0035244
name: peptidyl-arginine C-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the carbon atom of an arginine residue in a protein." [GOC:bf]
GO:0035248
name: alpha-1,4-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the transfer of an N-acetylgalactosaminyl residue from UDP-N-acetyl-galactosamine to an acceptor molecule, forming an alpha-1,4 linkage." [PMID:15130086]
GO:0035250
name: UDP-galactosyltransferase activity
def: "Catalysis of the transfer of a galactose group from UDP-galactose to an acceptor molecule." [http://www.chem.qmul.ac.uk/iubmb/enzyme/reaction/polysacc/UDPsugar.html]
GO:0035251
name: UDP-glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group from UDP-glucose to an acceptor molecule." [http://www.chem.qmul.ac.uk/iubmb/enzyme/reaction/polysacc/UDPsugar.html]
GO:0035252
name: UDP-xylosyltransferase activity
def: "Catalysis of the transfer of a xylosyl group from UDP-xylose to an acceptor molecule." [http://www.chem.qmul.ac.uk/iubmb/enzyme/reaction/polysacc/UDPsugar.html]
GO:0035254
name: glutamate receptor binding
def: "Interacting selectively and non-covalently with a glutamate receptor." [GOC:bf]
GO:0035255
name: ionotropic glutamate receptor binding
def: "Interacting selectively and non-covalently with an ionotropic glutamate receptor. Ionotropic glutamate receptors bind glutamate and exert an effect through the regulation of ion channels." [GOC:bf, ISBN:0198506732]
GO:0035256
name: G-protein coupled glutamate receptor binding
def: "Interacting selectively and non-covalently with a G-protein coupled glutamate receptor (a metabotropic glutamate receptor)." [GOC:bf, ISBN:0198506732, PMID:9069287]
GO:0035257
name: nuclear hormone receptor binding
def: "Interacting selectively and non-covalently with a nuclear hormone receptor, a ligand-dependent receptor found in the nucleus of the cell." [GOC:bf]
GO:0035258
name: steroid hormone receptor binding
def: "Interacting selectively and non-covalently with a steroid hormone receptor." [GOC:bf]
GO:0035259
name: glucocorticoid receptor binding
def: "Interacting selectively and non-covalently with a glucocorticoid receptor." [GOC:bf]
GO:0035273
name: phthalate binding
def: "Interacting selectively and non-covalently with a phthalate, any ester or salt of phthalic acid." [http://umbbd.ahc.umn.edu/pth/pth_map.html]
GO:0035274
name: diphenyl phthalate binding
def: "Interacting selectively and non-covalently with diphenyl phthalate, C(20)H(14)O(4)." [http://www.sigmaaldrich.com]
GO:0035275
name: dibutyl phthalate binding
def: "Interacting selectively and non-covalently with dibutyl phthalate, C(16)H(22)O(4)." [http://www.sigmaaldrich.com]
GO:0035276
name: ethanol binding
def: "Interacting selectively and non-covalently with ethanol, CH(3)-CH(2)-OH." [ISBN:0198506732]
GO:0035299
name: inositol pentakisphosphate 2-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4,5,6-pentakisphosphate + ATP = 1D-myo-inositol hexakisphosphate + ADP + 2 H(+)." [EC:2.7.1.158, RHEA:20316]
GO:0035300
name: inositol-1,3,4-trisphosphate 5/6-kinase activity
def: "OBSOLETE. Catalysis of the reactions: ATP + 1D-myo-inositol 1,3,4-trisphosphate = ADP + 1D-myo-inositol 1,3,4,5-tetrakisphosphate, and ATP + 1D-myo-inositol 1,3,4-trisphosphate = ADP + 1D-myo-inositol 1,3,4,6-tetrakisphosphate." [EC:2.7.1.159, PMID:9126335]
GO:0035302
name: ecdysteroid 25-hydroxylase activity
def: "Catalysis of the hydroxylation of an ecdysteroid at carbon position 25. Ecdysteroids are a group of polyhydroxylated ketosteroids which initiate post-embryonic development, particularly metamorphosis, in insects and other arthropods." [ISBN:0198506732, PMID:15350618]
GO:0035312
name: 5'-3' exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' terminus of a DNA molecule." [ISBN:0198547684]
GO:0035325
name: Toll-like receptor binding
def: "Interacting selectively and non-covalently with a Toll-like protein, a pattern recognition receptor that binds pattern motifs from a variety of microbial sources to initiate an innate immune response." [PMID:19076341]
GO:0035326
name: enhancer binding
def: "Interacting selectively and non-covalently with an enhancer, a transcription regulatory region that is somewhat distal from the core promoter and which enhances transcription from that promoter." [GOC:sart, GOC:txnOH, SO:0000165]
GO:0035368
name: selenocysteine insertion sequence binding
def: "Interacting selectively and non-covalently with the selenocysteine insertion sequence (SECIS), a regulatory sequence within mRNA which directs incorporation of a selenocysteine at a stop codon (UGA) during translation." [GOC:imk, PMID:10760958]
GO:0035373
name: chondroitin sulfate proteoglycan binding
def: "Interacting selectively and non-covalently with a chondroitin sulfate proteoglycan, any proteoglycan containing chondroitin sulfate as the glycosaminoglycan carbohydrate unit." [GOC:kmv, ISBN:0198506732]
GO:0035374
name: chondroitin sulfate binding
def: "Interacting selectively and non-covalently with chondroitin sulfate, a glycosaminoglycan made up of two alternating monosaccharides: D-glucuronic acid (GlcA) and N-acetyl-D-galactosamine (GalNAc)." [GOC:kmv, ISBN:0198506732]
GO:0035375
name: zymogen binding
def: "Interacting selectively and non-covalently with a zymogen, an enzymatically inactive precursor of an enzyme that is often convertible to an active enzyme by proteolysis." [ISBN:0198506732]
GO:0035379
name: carbon dioxide transmembrane transporter activity
def: "Enables the transfer of carbon dioxide (CO2) from one side of a membrane to the other." [GOC:yaf]
GO:0035380
name: very long-chain-3-hydroxyacyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-hydroxyacyl-CoA + NAD(P)+ = 3-oxoacyl-CoA + NAD(P)H + H+, where the acyl group is a very long-chain fatty acid residue. A very long-chain fatty acid is a fatty acid which has a chain length greater than C22." [CHEBI:27283, GOC:pde]
GO:0035381
name: ATP-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when ATP has been bound by the channel complex or one of its constituent parts." [GOC:bf]
GO:0035400
name: histone tyrosine kinase activity
def: "Catalysis of the transfer of a phosphate group to a tyrosine residue of a histone. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:bf]
GO:0035401
name: histone kinase activity (H3-Y41 specific)
def: "The modification of histone H3 by the addition of a phosphate group to a tyrosine residue at position 41 of the histone." [GOC:bf]
GO:0035402
name: histone kinase activity (H3-T11 specific)
def: "Catalysis of the transfer of a phosphate group to the threonine-11 residue of the N-terminal tail of histone H3." [GOC:bf]
GO:0035403
name: histone kinase activity (H3-T6 specific)
def: "Catalysis of the transfer of a phosphate group to the threonine-6 residue of the N-terminal tail of histone H3." [GOC:bf]
GO:0035410
name: dihydrotestosterone 17-beta-dehydrogenase activity
def: "Catalysis of the reaction: 5alpha-dihydrotestosterone + NAD+ = 5alpha-androstane-3,17-dione + NADH." [GOC:ecd, http://www.brenda-enzymes.org/php/result_flat.php4?ecno=1.1.1.63, PMID:4152755]
GO:0035438
name: cyclic-di-GMP binding
def: "Interacting selectively and non-covalently with cyclic-di-GMP, cyclic dimeric guanosine monophosphate." [CHEBI:49537, GOC:bf]
GO:0035439
name: halimadienyl-diphosphate synthase activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate = halima-5(6),13-dien-15-yl diphosphate." [EC:5.5.1.16]
GO:0035446
name: cysteine-glucosaminylinositol ligase activity
def: "Catalysis of the reaction: 1-(2-amino-2-deoxy-alpha-D-glucopyranoside)-1D-myo-inositol + L-cysteine + ATP = 1-D-myo-inosityl-2-L-cysteinylamido-2-deoxy-alpha-D-glucopyranoside + AMP + diphosphate + 2 H+. 1-(2-amino-2-deoxy-alpha-D-glucopyranoside)-1D-myo-inositol is also known as glucosaminyl-inositol or GlcN-Ins, and 1-D-myo-inosityl-2-L-cysteinylamido-2-deoxy-alpha-D-glucopyranoside as desacetylmycothiol or Cys-GlcN-Ins." [EC:6.3.1.13, MetaCyc:RXN1G-4, PMID:12033919]
GO:0035447
name: mycothiol synthase activity
def: "Catalysis of the reaction: 1-D-myo-inosityl-2-L-cysteinylamido-2-deoxy-alpha-D-glucopyranoside + acetyl-CoA = mycothiol + coenzyme A + H+. Mycothiol is also known as AcCys-GlcN-Ins and 1-D-myo-inosityl-2-L-cysteinylamido-2-deoxy-alpha-D-glucopyranoside as Cys-GlcN-Ins or desacetylmycothiol." [MetaCyc:MONOMER-9684, PMID:12033919]
GO:0035460
name: L-ascorbate 6-phosphate lactonase activity
def: "Catalysis of the reaction: L-ascorbate 6-phosphate + H2O = 3-keto-L-gulonate 6-phosphate." [PMID:18097099, PMID:20359483]
GO:0035472
name: choriogonadotropin hormone receptor activity
def: "Combining with the choriogonadotropin hormone to initiate a change in cell activity." [GOC:bf, ISBN:0198506732, PMID:1922095]
GO:0035473
name: lipase binding
def: "Interacting selectively and non-covalently with any lipase." [GOC:BHF]
GO:0035478
name: chylomicron binding
def: "Interacting selectively and non-covalently with a chylomicron, a large lipoprotein particle (diameter 75-1200 nm) composed of a central core of triglycerides and cholesterol surrounded by a protein-phospholipid coating. The proteins include one molecule of apolipoprotein B-48 and may include a variety of apolipoproteins, including APOAs, APOCs and APOE." [GOC:BHF, http://www.britannica.com/EBchecked/topic/117461/chylomicron]
GO:0035484
name: adenine/adenine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing an A/A mispair." [GOC:bf, GOC:jh]
GO:0035485
name: adenine/guanine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing an A/G mispair." [GOC:bf, GOC:jh]
GO:0035486
name: cytosine/cytosine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a C/C mispair." [GOC:bf, GOC:jh]
GO:0035487
name: thymine/thymine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a T/T mispair." [GOC:bf, GOC:jh]
GO:0035488
name: cytosine/thymine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a C/T mispair." [GOC:bf, GOC:jh]
GO:0035489
name: guanine/guanine mispair binding
def: "Interacting selectively and non-covalently with double-stranded DNA containing a G/G mispair." [GOC:bf, GOC:jh]
GO:0035496
name: lipopolysaccharide-1,5-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + lipopolysaccharide = UDP + 1,5 alpha-D-galactosyl-lipopolysaccharide." [EC:2.4.1.-, PMID:11304545]
GO:0035497
name: cAMP response element binding
def: "Interacting selectively and non-covalently with the cyclic AMP response element (CRE), a short palindrome-containing sequence found in the promoters of genes whose expression is regulated in response to cyclic AMP." [PMID:2875459, PMID:2900470]
GO:0035500
name: MH2 domain binding
def: "Interacting selectively and non-covalently with the MH2 (MAD homology 2) domain of a protein. The MH2 domain is found at the carboxy terminus of MAD related proteins such as Smads. The MH2 domain mediates interaction with a wide variety of proteins and provides specificity and selectivity to Smad function and also is critical for mediating interactions in Smad oligomers." [Pfam:PF03166]
GO:0035501
name: MH1 domain binding
def: "Interacting selectively and non-covalently with the MH1 (MAD homology 1) domain of a protein. The MH1 domain is found at the amino terminus of MAD related proteins such as Smads and can mediate DNA binding in some proteins. Smads also use the MH1 domain to interact with some transcription factors." [Pfam:PF03165]
GO:0035514
name: DNA demethylase activity
def: "Catalysis of the removal of a methyl group from one or more nucleosides within a DNA molecule." [GOC:bf]
GO:0035515
name: oxidative RNA demethylase activity
def: "Catalysis of the removal of a methyl group from one or more nucleosides within a RNA molecule involving the oxidation (i.e. electron loss) of one or more atoms." [PMID:12594517, PMID:16482161, PMID:18775698]
GO:0035516
name: oxidative DNA demethylase activity
def: "Catalysis of the removal of the methyl group from one or more nucleotides within a DNA molecule involving the oxidation (i.e. electron loss) of one or more atoms." [PMID:12594517, PMID:16482161, PMID:18775698]
GO:0035527
name: 3-hydroxypropionate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: 3-hydroxypropanoate + NADP+ = 3-oxopropanoate + H+ + NADPH." [RHEA:26441]
GO:0035529
name: NADH pyrophosphatase activity
def: "Catalysis of the reaction: NADH + H2O = AMP + NMNH + 2 H+." [MetaCyc:RXN0-4401, PMID:12399474, PMID:20181750]
GO:0035538
name: carbohydrate response element binding
def: "Interacting selectively and non-covalently with the carbohydrate response element (ChoRE) found in the promoters of genes whose expression is regulated in response to carbohydrates, such as the triglyceride synthesis genes." [GOC:BHF, PMID:20001964]
GO:0035539
name: 8-oxo-7,8-dihydrodeoxyguanosine triphosphate pyrophosphatase activity
def: "Catalysis of the reaction: 8-oxo-7,8-dihydrodeoxyguanosine-triphosphate + H2O = 8-oxo-7,8-dihydrodeoxyguanosine phosphate + diphosphate. 8-oxo-7,8-dihydrodeoxyguanosine-triphosphate, or 8-oxo-dGTP, is the oxidised form of the free guanine nucleotide and can act as a potent mutagenic substrate for DNA synthesis causing transversion mutations. 8-oxo-dGTPase hydrolyses 8-oxo-dGTP to its monophosphate form to prevent the misincorporation of 8-oxo-dGTP into cellular DNA." [PMID:17804481, PMID:7782328, PMID:7859359]
GO:0035560
name: pheophoridase activity
def: "Catalysis of the reaction: pheophorbide a + H2O = pyropheophorbide a + methanol + CO2. The reaction occurs in two steps; pheophoridase catalyzes the conversion of pheophorbide a to a precursor of pyropheophorbide a, C-13(2)-carboxylpyropheophorbide a, by demethylation, and then the precursor is decarboxylated non-enzymatically to yield pyropheophorbide a." [EC:3.1.1.82, PMID:16228561]
GO:0035575
name: histone demethylase activity (H4-K20 specific)
def: "Catalysis of the reaction: histone H4 N6-methyl-L-lysine (position 20) + 2-oxoglutarate + O2 = histone H4 L-lysine (position 20) + succinate + formaldehyde + CO2. This reaction is the removal of a methyl group from lysine at position 20 of the histone H4 protein." [EC:1.14.11.27, PMID:20622853]
GO:0035586
name: purinergic receptor activity
def: "Combining with a purine or purine derivative (purine nucleoside or purine nucleotide) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity. A nucleotide is a compound that consists of a nucleoside esterified with a phosphate molecule." [GOC:bf, GOC:BHF, GOC:signaling, PMID:9755289]
GO:0035591
name: signaling adaptor activity
def: "The binding activity of a molecule that brings together two or more molecules in a signaling pathway, permitting those molecules to function in a coordinated way. Adaptor molecules themselves do not have catalytic activity." [GOC:bf]
GO:0035594
name: ganglioside binding
def: "Interacting selectively and non-covalently with a ganglioside, a ceramide oligosaccharide carrying in addition to other sugar residues, one or more sialic acid residues." [GOC:yaf]
GO:0035595
name: N-acetylglucosaminylinositol deacetylase activity
def: "Catalysis of the reaction: 1D-myo-inositol 2-acetamido-2-deoxy-alpha-D-glucopyranoside + H2O = 1D-myo-inositol 2-amino-2-deoxy-alpha-D-glucopyranoside + acetate. This reaction is the hydrolysis of an acetyl group from N-acetylglucosaminylinositol." [EC:3.5.1.103, GOC:rs]
GO:0035596
name: methylthiotransferase activity
def: "Catalysis of the addition of a methylthioether group (-SCH3) to a nucleic acid or protein acceptor." [GOC:jh2, PMID:20472640]
GO:0035597
name: N6-isopentenyladenosine methylthiotransferase activity
def: "Catalysis of the methylthiolation (-SCH3 addition) at the C2 of the adenosine ring of N6-isopentenyladenosine (i6A) in tRNA, to form 2-methylthio-N6-isopentenyladenosine (ms2i6A)." [PMID:20472640]
GO:0035598
name: N6-threonylcarbomyladenosine methylthiotransferase activity
def: "Catalysis of the methylthiolation (-SCH3 addition) at the C2 of the adenosine ring of N6-threonylcarbomyladenosine (t6A) in tRNA, to form 2-methylthio-N6-threonylcarbamoyladenosine (ms2t6A)." [PMID:20472640, PMID:20584901]
GO:0035599
name: aspartic acid methylthiotransferase activity
def: "Catalysis of the methylthiolation (-SCH3 addition) of the beta-carbon of peptidyl-aspartic acid to form peptidyl-L-beta-methylthioaspartic acid." [PMID:18252828, PMID:8844851, RESID:AA0232]
GO:0035605
name: peptidyl-cysteine S-nitrosylase activity
def: "Catalysis of the transfer of a nitric oxide (NO) group to a sulphur atom within a cysteine residue of a protein." [EC:2.6.99.-, GOC:sp, PMID:20972425, PMID:20972426]
GO:0035612
name: AP-2 adaptor complex binding
def: "Interacting selectively and non-covalently with the AP-2 adaptor complex. The AP-2 adaptor complex is a heterotetrameric AP-type membrane coat adaptor complex that consists of alpha, beta2, mu2 and sigma2 subunits and links clathrin to the membrane surface of a vesicle. In at least humans, the AP-2 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different alpha genes (alphaA and alphaC)." [GOC:BHF, PMID:12221107, PMID:15728179, PMID:21097499]
GO:0035613
name: RNA stem-loop binding
def: "Interacting selectively and non-covalently with a stem-loop in an RNA molecule. An RNA stem-loop is a secondary RNA structure consisting of a double-stranded RNA (dsRNA) stem and a terminal loop." [GOC:sart, PMID:16568238, PMID:20455544]
GO:0035614
name: snRNA stem-loop binding
def: "Interacting selectively and non-covalently with a stem-loop in a small nuclear RNA (snRNA). An RNA stem-loop is a secondary RNA structure consisting of a double-stranded RNA (dsRNA) stem and a terminal loop." [GOC:sart, PMID:16568238, PMID:20455544]
GO:0035615
name: clathrin adaptor activity
def: "The binding activity of a molecule that brings together clathrin and one or more other molecules, permitting them to function in a coordinated way." [GOC:BHF, PMID:15728179]
GO:0035620
name: ceramide transporter activity
def: "Enables the directed movement of ceramides into, out of or within a cell, or between cells. Ceramides are a class of lipid composed of sphingosine linked to a fatty acid." [GOC:sart, PMID:14685229]
GO:0035639
name: purine ribonucleoside triphosphate binding
def: "Interacting selectively and non-covalently with a purine ribonucleoside triphosphate, a compound consisting of a purine base linked to a ribose sugar esterified with triphosphate on the sugar." [CHEBI:26398, GOC:BHF, GOC:ebc, ISBN:0198506732]
GO:0035642
name: histone methyltransferase activity (H3-R17 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone H3)-arginine (position 17) = S-adenosyl-L-homocysteine + (histone H3)-N-methyl-arginine (position 17). This reaction is the addition of a methyl group to arginine at position 17 of histone H3." [GOC:sp, PMID:11341840]
GO:0035643
name: L-DOPA receptor activity
def: "Combining with L-DOPA to initiate a change in cell activity. L-DOPA is the modified amino acid (2S)-2-amino-3-(3,4-dihydroxyphenyl) propanoic acid, and is the precursor to dopamine, norepinephrine (noradrenaline) and epinephrine." [CHEBI:15765, PMID:18828673, Wikipedia:L-DOPA]
GO:0035647
name: 3-oxo-delta(4,5)-steroid 5-beta-reductase activity
def: "Catalysis of the reaction: a 3-oxo-delta-4,5-steroid + NADPH + H(+) = a 5-beta-3-oxo-steroid + NADP(+)." [GOC:kad, MetaCyc:RXN-9726, PMID:19166903]
GO:0035650
name: AP-1 adaptor complex binding
def: "Interacting selectively and non-covalently with the AP-1 adaptor complex. The AP-1 adaptor complex is a heterotetrameric AP-type membrane coat adaptor complex that consists of beta1, gamma, mu1 and sigma1 subunits and links clathrin to the membrane surface of a vesicle. In at least humans, the AP-1 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different genes (gamma1 and gamma2, mu1A and mu1B, and sigma1A, sigma1B and sigma1C)." [PMID:21097499]
GO:0035651
name: AP-3 adaptor complex binding
def: "Interacting selectively and non-covalently with the AP-3 adaptor complex. The AP-3 adaptor complex is a heterotetrameric AP-type membrane coat adaptor complex that consists of beta3, delta, mu3 and sigma3 subunits and is found associated with endosomal membranes. In at least humans, the AP-3 complex can be heterogeneric due to the existence of multiple subunit isoforms encoded by different genes (beta3A and beta3B, mu3A and mu3B, and sigma3A and sigma3B)." [PMID:21097499]
GO:0035656
name: kinesin-associated melanosomal adaptor activity
def: "The activity of linking kinesins, cytoplasmic proteins responsible for moving vesicles and organelles towards the distal end of microtubules, to melanosomes." [PMID:19841138]
GO:0035662
name: Toll-like receptor 4 binding
def: "Interacting selectively and non-covalently with a Toll-like 4 protein, a pattern recognition receptor that binds bacterial lipopolysaccharide (LPS) to initiate an innate immune response." [GOC:BHF, PMID:18304834]
GO:0035663
name: Toll-like receptor 2 binding
def: "Interacting selectively and non-covalently with a Toll-like 2 protein, a pattern recognition receptor that binds microbial pattern motifs to initiate an innate immune response." [GOC:BHF]
GO:0035671
name: enone reductase activity
def: "Catalysis of the reaction: an enone + NADPH + H+ = a ketone + NADP+." [GOC:kad, PMID:17945329, PMID:19166903]
GO:0035673
name: oligopeptide transmembrane transporter activity
def: "Catalysis of the transfer of oligopeptides from one side of the membrane to the other." [GOC:vw]
GO:0035715
name: chemokine (C-C motif) ligand 2 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 2." [GOC:BHF]
GO:0035716
name: chemokine (C-C motif) ligand 12 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 12." [GOC:BHF]
GO:0035717
name: chemokine (C-C motif) ligand 7 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 7." [GOC:BHF]
GO:0035718
name: macrophage migration inhibitory factor binding
def: "Interacting selectively and non-covalently with the cytokine, macrophage migration inhibitory factor." [GOC:BHF, PMID:19601712]
GO:0035727
name: lysophosphatidic acid binding
def: "Interacting selectively and non-covalently with lysophosphatidic acid (LPA), a phospholipid derivative that acts as a potent mitogen due to its activation of high-affinity G-protein-coupled receptors." [CHEBI:52288]
GO:0035730
name: S-nitrosoglutathione binding
def: "Interacting selectively and non-covalently with S-nitrosoglutathione, a nitrosothiol considered to be a natural nitric oxide (NO) donor involved in S-nitrosylation, and in the storage and transport of nitric oxide in biological systems." [CHEBI:50091, GOC:BHF]
GO:0035731
name: dinitrosyl-iron complex binding
def: "Interacting selectively and non-covalently with a dinitrosyl-iron complex. Nitric oxide (NO) is stored as dinitrosyl-iron complexes, which form spontaneously from Glutathione (GSH), S-nitrosoglutathione, and trace amounts of ferrous ions, or by reaction of iron-sulfur centers with NO." [GOC:BHF, PMID:10534443]
GO:0035755
name: cardiolipin hydrolase activity
def: "Catalysis of the hydrolysis of cardiolipin (1,3-bis(3-phosphatidyl)glycerol) to form phosphatidic acid (PA)." [CHEBI:28494, GOC:sp, PMID:17028579, PMID:21397848]
GO:0035757
name: chemokine (C-C motif) ligand 19 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 19." [GOC:BHF]
GO:0035758
name: chemokine (C-C motif) ligand 21 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 21." [GOC:BHF]
GO:0035797
name: tellurite methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to tellurite [TeO3(2-)]. Methylated derivatives of tellurite include Te(CH3)2 (dimethyltelluride) and Te2(CH3)2 (dimethylditelluride)." [CHEBI:30477, GOC:bf, GOC:kad, PMID:11053398, PMID:21244361]
GO:0035798
name: 2-alkenal reductase (NADP+) activity
def: "Catalysis of the reaction: n-alkanal + NADP+ = alk-2-enal + NADPH + H+." [GOC:bf, GOC:kad, PMID:16299173]
GO:0035800
name: deubiquitinase activator activity
def: "Increases the activity of deubiquitinase, an enzyme that catalyzes the hydrolysis of various forms of polymeric ubiquitin sequences." [GOC:sart, ISBN:0120793709]
GO:0035804
name: structural constituent of egg coat
def: "The action of a molecule that contributes to the structural integrity of an egg coat. An egg coat is a specialized extracellular matrix that surrounds the ovum of animals. The egg coat provides structural support and can play an essential role in oogenesis, fertilization and early development." [PMID:16944418, PMID:17163408]
GO:0035827
name: rubidium ion transmembrane transporter activity
def: "Catalysis of the transfer of rubidium ions (Rb+) from one side of a membrane to the other." [CHEBI:49847, GOC:yaf]
GO:0035851
name: Krueppel-associated box domain binding
def: "Interacting selectively and non-covalently with a Krueppel-associated box (KRAB) domain of a protein. The approximately 75 amino acid KRAB domain is enriched in charged amino acids, and is found in the N-terminal regions of many zinc finger-containing transcription factors." [InterPro:IPR001909]
GO:0035870
name: dITP diphosphatase activity
def: "Catalysis of the reaction: dITP + H2O = dIMP + diphosphate." [EC:3.6.1.-, GOC:dgf, PMID:21548881, RHEA:28342]
GO:0035877
name: death effector domain binding
def: "Interacting selectively and non-covalently with a DED domain (death effector domain) of a protein, a homotypic protein interaction module composed of a bundle of six alpha-helices that is related in structure to the death domain (DD)." [GOC:ecd, InterPro:IPR001875]
GO:0035885
name: exochitinase activity
def: "Catalysis of the hydrolysis of terminal 1,4-beta-linkages of N-acetyl-D-glucosamine (GlcNAc) polymers of chitin and chitodextrins. Typically, exochitinases progressively cleave off two subunits from the reducing or non-reducing ends of the chitin chain." [EC:3.2.1.-, GOC:bf, GOC:kah, GOC:pde, PMID:11468293, PMID:16298970, PMID:21390509]
GO:0035888
name: isoguanine deaminase activity
def: "Catalysis of the reaction: isoguanine + H2O = xanthine + NH3." [EC:3.5.4.-, GOC:imk, PMID:21604715]
GO:0035923
name: flurbiprofen binding
def: "Interacting selectively and non-covalently with flurbiprofen." [CHEBI:5130, GOC:BHF, GOC:rl]
GO:0035925
name: mRNA 3'-UTR AU-rich region binding
def: "Interacting selectively and non-covalently with a region containing frequent adenine and uridine bases within the 3' untranslated region of a mRNA molecule." [GOC:vw]
GO:0035939
name: microsatellite binding
def: "Interacting selectively and non-covalently with a microsatellite, a repeat_region in DNA containing repeat units (2 to 4 base pairs) that is repeated multiple times in tandem." [GOC:yaf, PMID:21290414, SO:0000289]
GO:0035979
name: histone kinase activity (H2A-S139 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-139 residue of the C-terminal tail of histone H2A." [GOC:yaf, PMID:16061642]
GO:0035991
name: nitric oxide sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of nitric oxide (NO)." [GOC:kmv, PMID:21491957]
GO:0036002
name: pre-mRNA binding
def: "Interacting selectively and non-covalently with pre-messenger RNA (pre-mRNA), an intermediate molecule between DNA and protein that may contain introns and, at least in part, encodes one or more proteins. Introns are removed from pre-mRNA to form a mRNA molecule." [GOC:bf, GOC:kmv, PMID:21901112, SO:0000120]
GO:0036004
name: GAF domain binding
def: "Interacting selectively and non-covalently with the GAF domain of a protein." [GOC:yaf, InterPro:IPR003018]
GO:0036009
name: protein-glutamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + protein L-glutamine = S-adenosyl-L-homocysteine + protein N-methyl-L-glutamine." [GOC:imk, PMID:11847124]
GO:0036033
name: mediator complex binding
def: "Interacting selectively and non-covalently with a mediator complex. The mediator complex is a protein complex that interacts with the carboxy-terminal domain of the largest subunit of RNA polymerase II and plays an active role in transducing the signal from a transcription factor to the transcriptional machinery. The Saccharomyces complex contains several identifiable subcomplexes: a head domain comprising Srb2, -4, and -5, Med6, -8, and -11, and Rox3 proteins; a middle domain comprising Med1, -4, and -7, Nut1 and -2, Cse2, Rgr1, Soh1, and Srb7 proteins; a tail consisting of Gal11p, Med2p, Pgd1p, and Sin4p; and a regulatory subcomplex comprising Ssn2, -3, and -8, and Srb8 proteins. Metazoan mediator complexes have similar modular structures and include homologs of yeast Srb and Med proteins." [GOC:yaf, PMID:18391015]
GO:0036041
name: long-chain fatty acid binding
def: "Interacting selectively and non-covalently with a long-chain fatty acid. A long-chain fatty acid is a fatty acid with a chain length between C13 and C22." [CHEBI:15904, GOC:pm, PMID:12641450]
GO:0036042
name: long-chain fatty acyl-CoA binding
def: "Interacting selectively and non-covalently with a long-chain fatty acyl-CoA. A long-chain fatty acyl-CoA is any derivative of coenzyme A in which the sulfhydryl group is in a thioester linkage with a long-chain fatty-acyl group. Long-chain fatty-acyl-CoAs have chain lengths of C13 or more." [CHEBI:33184, GOC:pm]
GO:0036054
name: protein-malonyllysine demalonylase activity
def: "Catalysis of the reaction: protein-malonyllysine + H2O => protein-lysine + malonate. This reaction is the removal of a malonyl group (CO-CH2-CO) from a malonylated lysine residue of a protein or peptide." [GOC:sp, PMID:21908771, PMID:22076378]
GO:0036055
name: protein-succinyllysine desuccinylase activity
def: "Catalysis of the reaction: protein-succinyllysine + H2O => protein-lysine + succinate. This reaction is the removal of a succinyl group (CO-CH2-CH2-CO) from a succinylated lysine residue of a protein or peptide." [GOC:sp, PMID:22076378]
GO:0036080
name: purine nucleotide-sugar transmembrane transporter activity
def: "Catalysis of the transfer of a purine nucleotide-sugar from one side of the membrane to the other. Purine nucleotide-sugars are purine nucleotides in glycosidic linkage with a monosaccharide or monosaccharide derivative." [GOC:sart, PMID:19948734]
GO:0036081
name: extracellular ammonia-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when extracellular ammonia (NH3) has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
GO:0036082
name: extracellular phenylacetaldehyde-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when extracellular phenylacetaldehyde has been bound by the channel complex or one of its constituent parts." [GOC:sart, PMID:19135896]
GO:0036094
name: small molecule binding
def: "Interacting selectively and non-covalently with a small molecule, any low molecular weight, monomeric, non-encoded molecule." [GOC:curators, GOC:pde, GOC:pm]
GO:0036096
name: 3'-5'-exoribonuclease activity involved in pre-miRNA 3'-end processing
def: "OBSOLETE. Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of an RNA molecule that contributes to forming the mature 3' end of a miRNA from a pre-miRNA." [GOC:sart, PMID:22055292]
GO:0036105
name: peroxisome membrane class-1 targeting sequence binding
def: "Interacting selectively and non-covalently with a class I peroxisomal membrane targeting sequence, any of several sequences of amino acids within a protein that can act as a signal for the localization of the protein into the peroxisome membrane in a PEX19-dependent manner." [GOC:pm, PMID:14709540, PMID:17020786]
GO:0036106
name: peroxisome membrane class-2 targeting sequence binding
def: "Interacting selectively and non-covalently with a class II peroxisomal membrane targeting sequence, any of several sequences of amino acids within a protein that can act as a signal for the localization of the protein into the peroxisome membrane in a PEX19-independent manner." [GOC:pm, PMID:14709540, PMID:17020786]
GO:0036121
name: double-stranded DNA-dependent ATP-dependent DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, in the presence of double-stranded DNA; drives the unwinding of a DNA helix." [GOC:kmv]
GO:0036122
name: BMP binding
def: "Interacting selectively and non-covalently with a member of the bone morphogenetic protein (BMP) family." [GOC:BHF, PMID:9660951, PR:000000034]
GO:0036127
name: 3-sulfino-L-alanine binding
def: "Interacting selectively and non-covalently with 3-sulfino-L-alanine (cysteine sulfinate)." [CHEBI:61085, GOC:al, PMID:8346915]
GO:0036130
name: prostaglandin H2 endoperoxidase reductase activity
def: "Catalysis of the reaction: prostaglandin H2 + NADPH + H+ -> prostaglandin F2alpha + NADP+. This reaction is the reduction of prostaglandin H2 ((5Z,13E)-(15S)-9alpha,11alpha-Epidioxy-15-hydroxyprosta-5,13-dienoate) to prostaglandin F2alpha ((5Z,13E)-(15S)-9alpha,11alpha,15-Trihydroxyprosta-5,13-dienoate)." [GOC:mw, KEGG:R02264, PMID:10622721, PMID:14979715, PMID:16475787]
GO:0036131
name: prostaglandin D2 11-ketoreductase activity
def: "Catalysis of the reaction: prostaglandin D2 + H+ + NADPH -> 11-epi-prostaglandin F2alpha + NADP+." [EC:1.1.1.188, GOC:mw, KEGG:R02799, PMID:1504718, PMID:3862115]
GO:0036132
name: 13-prostaglandin reductase activity
def: "Catalysis of the reaction: 15-keto-prostaglandin + NAD(P)H + H+ -> 13,14-dihydro-15-keto-prostaglandin + NAD(P)+. This reaction is the reduction of 15-keto-prostaglandin." [EC:1.3.1.48, GOC:mw, KEGG:R04556, KEGG:R04557, PMID:17449869]
GO:0036133
name: 11-hydroxythromboxane B2 dehydrogenase activity
def: "Catalysis of the reaction: thromboxane B2 + NAD+ = 11-dehydro-thromboxane B2 + NADH + H+." [GOC:mw, KEGG:R05060, PMID:3461463, PMID:3823488, PMID:8200461]
GO:0036134
name: 12-hydroxyheptadecatrienoic acid synthase activity
def: "Catalysis of the reaction: prostaglandin H2 = 12-hydroxyheptadecatrienoic acid (HHT) + malonaldehyde (MDA)." [GOC:mw, PMID:11297515]
GO:0036136
name: kynurenine-oxaloacetate transaminase activity
def: "Catalysis of the reaction: L-kynurenine + 2-oxoglutarate = 4-(2-aminophenyl)-2,4-dioxobutanoate + L-aspartate." [EC:2.6.1.-, GOC:pde, PMID:15606768, PMID:4149765]
GO:0036137
name: kynurenine aminotransferase activity
def: "Catalysis of the transfer of an amino group from kynurenine to an acceptor, usually a 2-oxo acid." [EC:2.6.1.-, GOC:pde]
GO:0036139
name: peptidyl-histidine dioxygenase activity
def: "Catalysis of the reaction: peptidyl L-histidine + 2-oxoglutarate + O2 = peptidyl hydroxy-L-histidine + succinate + CO2." [GOC:reh, PMID:21251231]
GO:0036140
name: peptidyl-asparagine 3-dioxygenase activity
def: "Catalysis of the reaction: peptidyl L-asparagine + 2-oxoglutarate + O2 = peptidyl 3-hydroxy-L-asparagine + succinate + CO2." [GOC:reh, PMID:12215170]
GO:0036141
name: L-phenylalanine-oxaloacetate transaminase activity
def: "Catalysis of the reaction L-phenylalanine + oxaloacetate = phenylpyruvate + aspartate." [GOC:pde, PMID:15606768]
GO:0036143
name: kringle domain binding
def: "Interacting selectively and non-covalently with a kringle domain. Kringle domains are protein domains that fold into large loops stabilized by 3 disulfide linkages, and are important in protein-protein interactions with blood coagulation factors." [GOC:yaf, InterPro:IPR000001, Wikipedia:Kringle_domain]
GO:0036163
name: 3-hexaprenyl-4-hydroxy-5-methoxybenzoic acid decarboxylase activity
def: "Catalysis of the reaction: 3-hexaprenyl-4-hydroxy-5-methoxybenzoic acid -> 2-hexaprenyl-6-methoxyphenol + CO2." [GOC:mw, KEGG:R06866, PMID:620805, PMID:7028108]
GO:0036169
name: 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid decarboxylase activity
def: "Catalysis of the reaction: 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid -> 2-methoxy-6-decaprenylphenol + CO2." [GOC:mw, PMID:620805, PMID:7028108]
GO:0036173
name: thiosulfate binding
def: "Interacting selectively and non-covalently with the inorganic anion thiosulfate, a sulfur oxide that has formula O3S2." [CHEBI:16094, GOC:db, PMID:2188959]
GO:0036174
name: butane monooxygenase activity
def: "Catalysis of the reaction: butane + O2 + NAD(P)H + H+ = butanol + NAD(P)+ + H2O." [GOC:dh, PMID:17526838, PMID:19383682]
GO:0036175
name: ribonucleoside-diphosphate reductase, glutaredoxin disulfide as acceptor
def: "Catalysis of the reaction: 2'-deoxyribonucleoside diphosphate + glutaredoxin disulfide + H2O -> ribonucleoside diphosphate + glutaredoxin." [EC:1.17.4.1, GOC:bf, GOC:pde, PMID:7476363]
GO:0036185
name: 13-lipoxin reductase activity
def: "Definition: Catalysis of the reaction: 15-oxolipoxin A4 + NAD(P)H + H+ = 13,14-dihydro-15-oxolipoxin A4 + NAD(P)+." [GOC:mw, PMID:10837478]
GO:0036188
name: abieta-7,13-dien-18-al dehydrogenase activity
def: "Catalysis of the reaction: abieta-7,13-diene-18-al + H2O + NAD+ = abieta-7,13-diene-18-oate + NADH + H+." [EC:1.2.1.74]
GO:0036189
name: abieta-7,13-diene hydroxylase activity
def: "Catalysis of the reaction: abieta-7,13-diene + NADPH + H+ + O2 = abieta-7,13-dien-18-ol + NADP+ + H2O." [EC:1.14.13.108]
GO:0036190
name: indole-2-monooxygenase activity
def: "Catalysis of the reaction: indole + NAD(P)H + H+ + O2 = indolin-2-one + NAD(P)+ + H2O." [EC:1.14.13.137]
GO:0036191
name: indolin-2-one monooxygenase activity
def: "Catalysis of the reaction: indolin-2-one + NAD(P)H + H+ + O2 = 3-hydroxyindolin-2-one + NAD(P)+ + H2O." [EC:1.14.13.138]
GO:0036192
name: 3-hydroxyindolin-2-one monooxygenase activity
def: "Catalysis of the reaction: 3-hydroxyindolin-2-one + NAD(P)H + H+ + O2 = 2-hydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)+ + H2O." [EC:1.14.13.139]
GO:0036193
name: 2-hydroxy-1,4-benzoxazin-3-one monooxygenase activity
def: "Catalysis of the reaction: 2-hydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)H + H+ + O2 = 2,4-dihydroxy-2H-1,4-benzoxazin-3(4H)-one + NAD(P)+ + H2O." [EC:1.14.13.140]
GO:0036199
name: cholest-4-en-3-one 26-monooxygenase activity
def: "Catalysis of the reaction: cholest-4-en-3-one + NADH + H+ + O2 = 26-hydroxycholest-4-en-3-one + NAD+ + H2O. This reaction involves the hydroxylation of the C26 carbon, followed by oxidation of the alcohol to the carboxylic acid via the aldehyde intermediate." [EC:1.14.13.141]
GO:0036200
name: 3-ketosteroid 9-alpha-monooxygenase activity
def: "Catalysis of the reaction: androsta-1,4-diene-3,17-dione + NADH + H+ + O2 = 9alpha-hydroxyandrosta-1,4-diene-3,17-dione + NAD+ + H2O." [EC:1.14.13.142]
GO:0036201
name: ent-isokaurene C2-hydroxylase activity
def: "Catalysis of the reaction: ent-isokaurene + O2 + NADPH + H+ = ent-2alpha-hydroxyisokaurene + H2O + NADP+." [EC:1.14.13.143]
GO:0036202
name: ent-cassa-12,15-diene 11-hydroxylase activity
def: "Catalysis of the reaction: ent-cassa-12,15-diene + O2 + NADPH + H+ = ent-11beta-hydroxycassa-12,15-diene + NADP+ + H2O." [EC:1.14.13.145]
GO:0036203
name: taxoid 14-beta-hydroxylase activity
def: "Catalysis of the reaction: 10beta-hydroxytaxa-4(20),11-dien-5alpha-yl acetate + O2 + NADPH + H+ = 10beta,14beta-dihydroxytaxa-4(20),11-dien-5alpha-yl acetate + NADP+ + H2O." [EC:1.14.13.146]
GO:0036204
name: abieta-7,13-dien-18-ol hydroxylase activity
def: "Catalysis of the reaction: abieta-7,13-dien-18-ol + NADPH + H+ + O2 = abieta-7,13-dien-18-al + NADP+ + 2 H2O. This is a two step reaction. The first step is: abieta-7,13-dien-18-ol + NADPH + H+ + O2 = abieta-7,13-dien-18,18-diol + + NADP+ + H2O. The second step is a spontaneous reaction: abieta-7,13-dien-18,18-diol = abieta-7,13-dien-18-al + H2O." [EC:1.14.13.109]
GO:0036209
name: 9beta-pimara-7,15-diene oxidase activity
def: "Catalysis of the reaction: 9beta-pimara-7,15-diene + 3 O2 + 3 NADPH + 3 H+ = 9beta-pimara-7,15-dien-19-oate + 3 NADP+ + 4 H2O. This is a three-step reaction: (a) 9beta-pimara-7,15-diene + O2 + NADPH + H+ = 9beta-pimara-7,15-dien-19-ol + NADP+ + H2O, (b) 9beta-pimara-7,15-dien-19-ol + O2 + NADPH + H+ = 9neta-pimara-7,15-dien-19-al + NADP+ + 2 H2O, (c) 9beta-pimara-7,15-dien-19-al + O2 + NADPH + H+ = 9beta-pimara-7,15-dien-19-oate + NADP+ + H2O." [EC:1.14.13.144]
GO:0036217
name: dGTP diphosphatase activity
def: "Catalysis of the reaction: dGTP + H2O = dGMP + diphosphate." [GOC:dgf, KEGG:R01855, PMID:17090528, PMID:22531138]
GO:0036218
name: dTTP diphosphatase activity
def: "Catalysis of the reaction: dTTP + H2O = dTMP + diphosphate." [GOC:dgf, PMID:22531138]
GO:0036219
name: GTP diphosphatase activity
def: "Catalysis of the reaction: GTP + H2O = GMP + diphosphate." [GOC:dgf, KEGG:R00426, PMID:22531138]
GO:0036220
name: ITP diphosphatase activity
def: "Catalysis of the reaction: ITP + H2O = IMP + diphosphate." [GOC:dgf, KEGG:R00720, PMID:17899088, PMID:22531138]
GO:0036221
name: UTP diphosphatase activity
def: "Catalysis of the reaction: UTP + H2O = UMP + diphosphate." [GOC:dgf, KEGG:R00662, PMID:17899088]
GO:0036222
name: XTP diphosphatase activity
def: "Catalysis of the reaction: XTP + H2O = xanthosine 5'-phosphate + diphosphate." [GOC:dgf, PMID:22531138]
GO:0036237
name: acyl-glucuronidase activity
def: "Catalysis of the reaction: an acyl-glucuronoside + H2O = an alcohol + D-glucuronate." [GOC:BHF, GOC:vk, PMID:22294686]
GO:0036238
name: gallate dioxygenase activity
def: "Catalysis of the reaction: gallate + O2 = (1E)-4-oxobut-1-ene-1,2,4-tricarboxylate." [EC:1.13.11.57, PMID:16030014]
GO:0036239
name: taxoid 7beta-hydroxylase activity
def: "Catalysis of the reaction: taxusin + O2 + NADPH + H+ = 7beta-hydroxytaxusin + NADP+ + H2O." [EC:1.14.13.147]
GO:0036243
name: succinate-semialdehyde dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: succinate semialdehyde + NADP+ + H2O = succinate + NADPH + 2 H+." [EC:1.2.1.79, GOC:bf]
GO:0036263
name: L-DOPA monooxygenase activity
def: "Catalysis of the reaction: L-DOPA + O2 = ? + H2O. This reaction catalyzes the oxygenation of the dopamine precursor L-DOPA, to the corresponding o-quinone." [GOC:sart, PMID:22120533]
GO:0036264
name: dopamine monooxygenase activity
def: "Catalysis of the reaction: dopamine + O2 = ? + H2O. This reaction catalyzes the oxygenation of dopamine to the corresponding o-quinone." [GOC:sart, PMID:22120533]
GO:0036307
name: 23S rRNA (adenine(2030)-N(6))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + adenine(2030) in 23S rRNA = S-adenosyl-L-homocysteine + rRNA containing N(6)-methyladenine(2030) in 23S rRNA." [GOC:imk, PMID:22847818]
GO:0036308
name: 16S rRNA (guanine(1516)-N(2))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine(1516) in 16S rRNA = N(2)-methylguanosine(1516) in 16S rRNA + S-adenosyl-L-homocysteine." [GOC:imk, PMID:22079366]
GO:0036310
name: annealing helicase activity
def: "Catalysis of the ATP-dependent rewinding of single-stranded DNA (ssDNA) to reform base pairs between strands. Often acts on ssDNA bubbles bound by replication protein A (RPA)." [GOC:bf, GOC:sp, PMID:21078962, PMID:22704558, PMID:22705370, PMID:22759634]
GO:0036311
name: chitin disaccharide deacetylase activity
def: "Catalysis of the reaction: 2-(acetylamino)-4-O-[2-(acetylamino)-2-deoxy-beta-D-glucopyranosyl]-2-deoxy-beta-D-glucopyranose + H2O = 2-(acetylamino)-4-O-(2-amino-2-deoxy-beta-D-glucopyranosyl)-2-deoxy-beta-D-glucopyranose + acetate." [EC:3.5.1.105, GOC:imk]
GO:0036312
name: phosphatidylinositol 3-kinase regulatory subunit binding
def: "Interacting selectively and non-covalently with a regulatory subunit of phosphatidylinositol 3-kinase. The regulatory subunit associates with the catalytic subunit to regulate both its activity and subcellular location." [GOC:bf, PMID:20505341]
GO:0036313
name: phosphatidylinositol 3-kinase catalytic subunit binding
def: "Interacting selectively and non-covalently with the catalytic subunit of a phosphatidylinositol 3-kinase. The catalytic subunit catalyzes the addition of a phosphate group to an inositol lipid at the 3' position of the inositol ring." [GOC:bf, PMID:17475214]
GO:0036317
name: tyrosyl-RNA phosphodiesterase activity
def: "Catalysis of the hydrolysis of a 5' tyrosyl-RNA phosphodiester bond between a protein and RNA. In picornaviruses, this covalent bond connects VPg, a viral-encoded protein essential for RNA replication, to the 5' end of all nascent picornavirus genomes; it is cleaved from viral RNA prior to its engaging in protein synthesis." [GOC:bf, GOC:sp, PMID:21408223, PMID:22908287]
GO:0036318
name: peptide pheromone receptor activity
def: "Combining with a peptide pheromone, and transmitting the signal across the membrane to initiate a change in cell activity." [CHEBI:38579, GOC:al]
GO:0036319
name: mating-type M-factor pheromone receptor activity
def: "Combining with the mating-type peptide pheromone M-factor and transmitting the signal across the membrane to initiate a change in cell activity. M-factor is a nine-membered oligopeptide that consists of tyrosyl, threonyl, prolyl, lysyl, valyl, prolyl, tyrosyl, methionyl and methyl S-farnesylcysteinate residues joined in sequence, and is a peptide pheromone released by Schizosaccharomyces pombe cells of the cellular mating type Minus." [CHEBI:64120, GOC:al]
GO:0036320
name: mating-type P-factor pheromone receptor activity
def: "Combining with the mating-type peptide pheromone P-factor and transmitting the signal across the membrane to initiate a change in cell activity. P-factor is a polypeptide of 23 residues, with the sequence Thr-Tyr-Ala-Asp-Phe-Leu-Arg-Ala-Tyr-Gln-Ser-Trp-Asn-Thr-Phe-Val-Asn-Pro-Asp-Arg-Pro-Asn-Leu, and is a peptide pheromone released by Schizosaccharomyces pombe cells of the cellular mating type Plus." [CHEBI:64120, GOC:al]
GO:0036326
name: VEGF-A-activated receptor activity
def: "Combining with vascular endothelial growth factor A (VEGF-A) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:signaling, PR:000017284, Wikipedia:VEGF-A]
GO:0036327
name: VEGF-B-activated receptor activity
def: "Combining with vascular endothelial growth factor B (VEGF-B) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:signaling, PR:000003096, Wikipedia:VEGF-B]
GO:0036328
name: VEGF-C-activated receptor activity
def: "Combining with vascular endothelial growth factor C (VEGF-C) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:signaling, PR:000017285, Wikipedia:VEGF-C]
GO:0036329
name: VEGF-D-activated receptor activity
def: "Combining with vascular endothelial growth factor D (VEGF-D) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:signaling, PR:000007520, Wikipedia:VEGF-D]
GO:0036330
name: VEGF-E-activated receptor activity
def: "Combining with vascular endothelial growth factor E (VEGF-E) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:signaling, PMID:19909239]
GO:0036332
name: placental growth factor-activated receptor activity
def: "Combining with placental growth factor (PlGF) and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:uh, PMID:12871269, PMID:7929268, PR:000012605, Wikipedia:Placental_growth_factor]
GO:0036348
name: hydantoin racemase activity
def: "Catalysis of the reaction: D-5-monosubstituted hydantoin = L-5-monosubstituted hydantoin." [EC:5.1.99.5, InterPro:IPR015942]
GO:0036354
name: 2-desacetyl-2-hydroxyethyl bacteriochlorophyllide a dehydrogenase activity
def: "Catalysis of the reaction: 2-desacetyl-2-hydroxyethyl bacteriochlorophyllide a = bacteriochlorophyllide a + 2 H+." [GOC:crds, InterPro:IPR005903, MetaCyc:RXN-8787, PMID:8437569]
GO:0036355
name: 2-iminoacetate synthase activity
def: "Catalysis of the reaction: L-tyrosine + S-adenosyl-L-methionine + reduced acceptor = 2-iminoacetate + 4-methylphenol + 5'-deoxyadenosine + L-methionine + acceptor + 2 H+." [EC:4.1.99.19, GOC:crds, MetaCyc:RXN-11319, PMID:17403671]
GO:0036356
name: cyclic 2,3-diphosphoglycerate synthetase activity
def: "Catalysis of the reaction: 2,3-diphosphoglycerate (DPG) + ATP = cyclic 2,3-diphosphoglycerate (cDPG) + ADP + phosphate." [GOC:crds, PMID:2226838, PMID:8320225, PMID:9811660]
GO:0036357
name: 2-phosphoglycerate kinase activity
def: "Catalysis of the reaction: 2-phosphoglycerate + ATP = 2,3-diphosphoglycerate + ADP." [GOC:bf, InterPro:IPR020872, PMID:2226838, PMID:8159166]
GO:0036361
name: racemase activity, acting on amino acids and derivatives
def: "Catalysis of the interconversion of the two enantiomers of a chiral amino acid or amino acid derivative." [GOC:crds]
GO:0036370
name: D-alanyl carrier activity
def: "Accepts activated D-alanine and subsequently transfers D-alanine onto a teichoic acid acceptor molecule. The carrier protein provides an essential link between the D-alanine-D-alanyl carrier protein ligase and the incorporation of D-alanine into the substrate." [GOC:crds, PMID:11222605, PMID:22750871, PMID:8682792]
GO:0036373
name: L-fucose mutarotase activity
def: "Catalysis of the reaction: alpha-L-fucose = beta-L-fucose." [GOC:crds, PMID:15060078, RHEA:25583]
GO:0036374
name: glutathione hydrolase activity
def: "Catalysis of the reaction: glutathione + H2O = L-cysteinylglycine + L-glutamate." [EC:3.4.19.13, GOC:imk]
GO:0036380
name: UDP-N-acetylglucosamine-undecaprenyl-phosphate N-acetylglucosaminephosphotransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-alpha-D-glucosamine + ditrans,octacis-undecaprenyl phosphate = UMP + N-acetyl-alpha-D-glucosaminyldiphospho-ditrans,octacis-undecaprenol." [EC:2.7.8.33, GOC:rs]
GO:0036381
name: pyridoxal 5'-phosphate synthase (glutamine hydrolysing) activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + L-glutamine = pyridoxal 5'-phosphate + L-glutamate + 3 H2O + phosphate. The reaction occurs in two steps: L-glutamine + H2O = L-glutamate + NH3, and subsequently D-ribose 5-phosphate + D-glyceraldehyde 3-phosphate + NH3 = pyridoxal 5'-phosphate + 4 H2O + phosphate." [EC:4.3.3.6, GOC:rs]
GO:0036382
name: flavin reductase (NADH) activity
def: "Catalysis of the reaction: reduced flavin + NAD+ = flavin + NADH + H+." [EC:1.5.1.36, GOC:rs]
GO:0036383
name: 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione monooxygenase activity
def: "Catalysis of the reaction: 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + FMNH2 + O2 = 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + FMN + H2O." [EC:1.14.14.12, GOC:rs]
GO:0036384
name: cytidine diphosphatase activity
def: "Catalysis of the reaction: CDP + H2O = CMP + phosphate." [GOC:al]
GO:0036393
name: thiocyanate peroxidase activity
def: "Catalysis of the reaction: thiocyanate (SCN-) + hydrogen peroxide (H2O2) = hypothiocyanite (OSCN-) + 2 H2O. Catalyzes the hydrogen peroxide oxidation of thiocyanate." [GOC:pm, PMID:12626341]
GO:0036397
name: formate dehydrogenase (quinone) activity
def: "Catalysis of the reaction: formate + a quinone = CO2 + a quinol." [EC:1.1.5.6, GOC:bm]
GO:0036400
name: short neuropeptide F receptor activity
def: "Combining with a short neuropeptide F and transmitting the signal within the cell to initiate a change in cell activity. Short neuropeptide F is an arthropod peptide of less than 28 residues (as small as 8-10 residues in some species) with a C-terminal RFamide or LRFamide." [GOC:ha, PMID:16330127, PMID:21440021]
GO:0036401
name: pyrokinin receptor activity
def: "Combining with a pyrokinin and transmitting the signal within the cell to induce a change in cell activity. Pyrokinins are a group of insect neuropeptides that share the common C-terminal pentapeptide sequence Phe-X-Pro-Arg-Leu-NH2 (X = S, T, K, A, or G). They play a central role in diverse physiological processes including stimulation of gut motility, production and release of sex pheromones, diapause, and pupariation." [GOC:ha, PMID:12951076, PMID:19186060]
GO:0036402
name: proteasome-activating ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, which promotes unfolding of protein substrates, and channel opening of the core proteasome." [GOC:rb, PMID:11430818]
GO:0036403
name: arachidonate 8(S)-lipoxygenase activity
def: "Catalysis of the reaction: arachidonate + O(2) = (5Z,8S,9E,11Z,14Z)-8-hydroperoxyicosa-5,9,11,14-tetraenoate." [EC:1.13.11.-, GOC:lb, PMID:10625675]
GO:0036408
name: histone acetyltransferase activity (H3-K14 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 14) = CoA + histone H3 N6-acetyl-L-lysine (position 14)." [GOC:vw, PMID:21289066]
GO:0036412
name: acetyl-CoA:oxalate CoA-transferase
def: "Catalysis of the reaction: acetyl-CoA + oxalate = acetate + oxalyl-CoA." [GOC:imk, PMID:23935849]
GO:0036422
name: heptaprenyl diphosphate synthase activity
def: "Catalysis of the reaction: (2E,6E)-farnesyl diphosphate + 4 isopentenyl diphosphate = 4 diphosphate + all-trans-heptaprenyl diphosphate." [EC:2.5.1.30, RHEA:27797]
GO:0036423
name: hexaprenyl-diphosphate synthase ((2E,6E)-farnesyl-diphosphate specific) activity
def: "Catalysis of the reaction: (2E,6E)-farnesyl diphosphate + 3 isopentenyl diphosphate = 3 diphosphate + all-trans-hexaprenyl diphosphate." [EC:2.5.1.83, RHEA:27562]
GO:0036424
name: L-phosphoserine phosphatase activity
def: "Catalysis of the reaction: O-phospho-L-serine + H2O <=> L-serine + phosphate." [RHEA:21211]
GO:0036425
name: D-phosphoserine phosphatase activity
def: "Catalysis of the reaction: O-phospho-D-serine + H2O <=> L-serine + phosphate." [RHEA:24876]
GO:0036426
name: ditrans, polycis-undecaprenyl-phosphate mannosyltransferase activity
def: "Catalysis of the reaction: ditrans,octacis-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate+ GDP." [RHEA:28121]
GO:0036427
name: all-trans-undecaprenyl-phosphate mannosyltransferase activity
def: "Catalysis of the reaction: all-trans-undecaprenyl phosphate + GDP-alpha-D-mannose = D-mannosyl undecaprenyl phosphate + GDP." [RHEA:12784]
GO:0036428
name: adenosylcobinamide kinase (GTP-specific) activity
def: "Catalysis of the reaction: adenosylcobinamide + GTP = adenosylcobinamide phosphate + GDP + H+." [RHEA:15768]
GO:0036429
name: adenosylcobinamide kinase (ATP-specific) activity
def: "Catalysis of the reaction: adenosylcobinamide + ATP = adenosylcobinamide phosphate + ADP + H+." [RHEA:15772]
GO:0036430
name: CMP kinase activity
def: "Catalysis of the reaction: ATP + CMP = ADP + CDP." [RHEA:11603]
GO:0036431
name: dCMP kinase activity
def: "Catalysis of the reaction: ATP + dCMP = ADP + dCDP." [RHEA:25097]
GO:0036432
name: all-trans undecaprenol kinase activity
def: "Catalysis of the reaction: ATP + undecaprenol + all-trans-undecaprenyl phosphate + ADP + H+." [RHEA:23755]
GO:0036433
name: di-trans, poly-cis-undecaprenol kinase activity
def: "Catalysis of the reaction: di-trans, octa-cis-undecaprenol + ATP = di-trans,octa-cis-undecaprenyl phosphate + ADP + H+." [RHEA:28125]
GO:0036434
name: nitronate monooxygenase (FMN-linked) activity
def: "Catalysis of the reaction: ethylnitronate + FMNH(2) + O(2) = acetaldehyde + FMN + H(2)O + H(+) + nitrite." [RHEA:26461]
GO:0036435
name: K48-linked polyubiquitin binding
def: "Interacting selectively and non-covalently and non-covalently with a polymer of ubiquitin formed by linkages between lysine residues at position 48 of the ubiquitin monomers." [GOC:al, PMID:20739285]
GO:0036439
name: glycerol-3-phosphate dehydrogenase [NADP+] activity
def: "Catalysis of the reaction: glycerol 3-phosphate + NADP+ = glycerone phosphate + H+ + NADPH." [RHEA:11099]
GO:0036440
name: citrate synthase activity
def: "Catalysis of the reaction: acetyl-CoA + H2O + oxaloacetate = citrate + CoA." [RHEA:16848]
GO:0036441
name: 2-dehydropantolactone reductase activity
def: "Catalysis of the reaction: (R)-pantolactone + NADP+ = 2-dehydropantolactone + NADPH + H+." [RHEA:18984]
GO:0036442
name: hydrogen-exporting ATPase activity
def: "Catalysis of the transfer of protons from one side of a membrane to the other according to the reaction: ATP + H2O + H+(in) -> ADP + phosphate + H+(out)." [RHEA:20855]
GO:0036443
name: dermatan 6-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenylyl sulfate + dermatan = adenosine 3',5'-bisphosphate + dermatan 6'-sulfate." [EC:2.8.2.33, GOC:bf, KEGG:R07288]
GO:0036455
name: iron-sulfur transferase activity
def: "Catalysis of the transfer of a iron-sulfur cluster from one compound (donor) to another (acceptor)." [GOC:bhm]
GO:0036456
name: L-methionine-(S)-S-oxide reductase activity
def: "Catalysis of the reaction: L-methionine (S)-S-oxide + thioredoxin -> L-methionine + thioredoxin disulfide + H2O." [GOC:vw, RHEA:19995]
GO:0036458
name: hepatocyte growth factor binding
def: "Interacting selectively and non-covalently with a hepatocyte growth factor." [PR:000008534]
GO:0036459
name: ubiquitinyl hydrolase activity
def: "Catalysis of the thiol-dependent hydrolysis of an ester, thioester, amide, peptide or isopeptide bond formed by the C-terminal glycine of ubiquitin." [EC:3.4.19.12, GOC:bf, GOC:ka]
GO:0036461
name: BLOC-2 complex binding
def: "Interacting selectively and non-covalently with a BLOC-2 complex, a protein complex required for the biogenesis of specialized organelles of the endosomal-lysosomal system, such as melanosomes and platelet dense granules." [GOC:bf, GOC:PARL, PMID:22511774]
GO:0036463
name: TRAIL receptor activity
def: "Combining with the ligand TRAIL (tumor necrosis factor-related apoptosis-inducing ligand) and transmitting the signal from one side of the plasma membrane to the other to initiate apoptotic cell death." [GOC:bf, GOC:PARL]
GO:0036467
name: 5-hydroxy-L-tryptophan decarboxylase activity
def: "Catalysis of the reaction: 5-hydroxy-L-tryptophan + H+ = CO2 + serotonin." [GOC:bf, GOC:PARL, RHEA:18536]
GO:0036468
name: L-dopa decarboxylase activity
def: "Catalysis of the reaction: L-dopa + H+ = CO2 + dopamine." [GOC:bf, GOC:PARL, RHEA:12275]
GO:0036469
name: L-tryptophan decarboxylase activity
def: "Catalysis of the reaction: L-tryptophan + H+ = CO2 + tryptamine." [GOC:bf, GOC:PARL, RHEA:30342]
GO:0036470
name: tyrosine 3-monooxygenase activator activity
def: "Interacts with and increases tyrosine 3-monooxygenase (tyrosine hydroxylase) activity." [GOC:bf, GOC:PARL, PMID:19703902]
GO:0036478
name: L-dopa decarboxylase activator activity
def: "Interacts with and increases L-dopa decarboxylase activity." [GOC:bf, GOC:PARL]
GO:0036479
name: peroxidase inhibitor activity
def: "Interacts with, and stops, prevents or reduces the activity of a peroxidase." [GOC:bf, GOC:PARL]
GO:0036487
name: nitric-oxide synthase inhibitor activity
def: "Interacts with, and stops, prevents or reduces the activity of nitric oxide synthase." [GOC:BHF, GOC:rl, PMID:17242280]
GO:0038006
name: netrin receptor activity involved in chemoattraction
def: "Combining with a netrin signal and transmitting the signal from one side of the membrane to the other to contribute to the directed movement of a motile cell towards a higher concentration of netrin." [GOC:signaling]
GO:0038021
name: leptin receptor activity
def: "Combining with the fat-cell specific hormone leptin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:signaling, PMID:9102398, Wikipedia:Leptin_receptor]
GO:0038022
name: G-protein coupled olfactory receptor activity
def: "Combining with an odorant and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, GOC:sart, PMID:21041441]
GO:0038023
name: signaling receptor activity
def: "Receiving a signal and transmitting the signal in the cell to initiate a change in cell activity. A signal is a physical entity or change in state that is used to transfer information in order to trigger a response." [GOC:bf, GOC:signaling]
GO:0038024
name: cargo receptor activity
def: "Combining selectively with an extracellular macromolecule and delivering the macromolecule into the cell via endocytosis." [GOC:bf, GOC:signaling, PMID:15239958]
GO:0038025
name: reelin receptor activity
def: "Combining with the secreted glycoprotein reelin, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, PMID:12827279, PMID:20223215, PR:000013879]
GO:0038036
name: sphingosine-1-phosphate receptor activity
def: "Combining with the sphingolipid sphingosine-1-phosphate (S1P), and transmitting the signal across the membrane by activating an associated G-protein." [GOC:bf, PMID:12728273, Wikipedia:S1PR1]
GO:0038046
name: enkephalin receptor activity
def: "Combining with an enkephalin, and transmitting the signal across the membrane by activating an associated G-protein. A enkephalin is a pentapeptide (Tyr-Gly-Gly-Phe-Met or Tyr-Gly-Gly-Phe-Leu) involved in regulating nociception in the body." [GOC:bf, Wikipedia:Enkephalin]
GO:0038047
name: morphine receptor activity
def: "Combining with morphine (17-methyl-7,8-didehydro-4,5alpha-epoxymorphinan-3,6alpha-diol), and transmitting the signal across the membrane by activating an associated G-protein." [CHEBI:17303, GOC:bf]
GO:0038048
name: dynorphin receptor activity
def: "Combining with a dynorphin peptide, and transmitting the signal across the membrane by activating an associated G-protein. Dynorphin is any opioid peptide that is generated by cleavage of the precursor protein prodynorphin." [GOC:bf, Wikipedia:Dynorphin]
GO:0038049
name: ligand-activated RNA polymerase II transcription factor binding transcription factor activity
def: "Combining with a signal and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. For example, some steroid hormone receptors bind to transcription factor complexes to regulate transcription of genes whose promoters do not contain hormone response elements." [GOC:signaling, GOC:txnOH]
GO:0038050
name: glucocorticoid-activated sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Combining with a glucocorticoid and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:17689856]
GO:0038051
name: glucocorticoid-activated RNA polymerase II transcription factor binding transcription factor activity
def: "Combining with a glucocorticoid and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. For example, glucocorticoid-bound receptors can bind to transcription factor complexes to regulate transcription of genes whose promoters do not contain glucocorticoid response elements." [GOC:signaling, PMID:17689856]
GO:0038052
name: estrogen-activated sequence-specific DNA binding RNA polymerase II transcription factor activity
def: "Combining with estrogen and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II." [GOC:signaling, PMID:17615392]
GO:0038053
name: estrogen-activated RNA polymerase II transcription factor binding transcription factor activity
def: "Combining with estrogen and transmitting the signal to the transcriptional machinery by interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. For example, estrogen-bound receptors can bind to transcription factor complexes to regulate transcription of genes whose promoters do not contain estrogen response elements." [GOC:signaling, PMID:17615392]
GO:0038054
name: G-protein coupled estrogen receptor activity
def: "Combining with estrogen and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:signaling, PMID:17379646, PMID:20960099]
GO:0038057
name: TNFSF11 binding
def: "Interacting selectively and non-covalently with tumor necrosis factor ligand superfamily member 11 (TNFSF11), a member of the tumor necrosis factor (TNF) cytokine family." [GOC:cjm, PR:000002107]
GO:0038058
name: TNFSF11 receptor activity
def: "Combining with a tumor necrosis factor ligand superfamily member 11 (TNFSF11) and transmitting the signal across the cell membrane to initiate a change in cell activity or function." [GOC:bf, GOC:cjm, PR:000002107]
GO:0038062
name: protein tyrosine kinase collagen receptor activity
def: "Combining with collagen and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-tyrosine = ADP + a protein-L-tyrosine phosphate." [GOC:bf, GOC:uh, PMID:16626936, PMID:21568710]
GO:0038064
name: collagen receptor activity
def: "Combining with a collagen and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:uh, PMID:21568710]
GO:0038067
name: MAP kinase activity involved in cell wall organization or biogenesis
def: "Catalysis of the reaction: protein + ATP = protein phosphate + ADP by a mitogen-activated protein kinase, as part of a MAPK signaling cascade that contributes to cell wall organization or biogenesis." [GOC:signaling]
GO:0038068
name: MAP kinase kinase activity involved in cell wall organization or biogenesis
def: "Catalysis of the concomitant phosphorylation of threonine (T) and tyrosine (Y) residues in a Thr-Glu-Tyr (TEY) thiolester sequence in a MAP kinase (MAPK) substrate, as part of a MAPK signaling cascade that contributes to cell wall organization or biogenesis." [GOC:signaling]
GO:0038069
name: MAP kinase phosphatase activity involved in regulation of cell wall biogenesis
def: "Catalysis of the reaction: a phosphorylated MAP kinase + H2O = a MAP kinase + phosphate, where the MAP kinase (MAPK) is part of a MAPK signaling cascade that contributes to cell wall biogenesis." [GOC:signaling]
GO:0038070
name: MAP kinase kinase kinase activity involved in cell wall organization or biogenesis
def: "Catalysis of the phosphorylation and activation of a MAP kinase kinase (MAPKK), as part of a MAPK signaling cascade that contributes to cell wall organization or biogenesis." [GOC:signaling]
GO:0038071
name: MAP kinase activity involved in conjugation with cellular fusion
def: "Catalysis of the reaction: protein + ATP = protein phosphate + ADP by a mitogen-activated protein kinase, as part of a MAPK signaling cascade that contributes to conjugation with cellular fusion." [GOC:signaling]
GO:0038072
name: MAP kinase kinase activity involved in conjugation with cellular fusion
def: "Catalysis of the concomitant phosphorylation of threonine (T) and tyrosine (Y) residues in a Thr-Glu-Tyr (TEY) thiolester sequence in a MAP kinase (MAPK) substrate, as part of a MAPK signaling cascade that contributes to conjugation with cellular fusion." [GOC:signaling]
GO:0038073
name: MAP kinase kinase kinase activity involved in conjugation with cellular fusion
def: "Catalysis of the phosphorylation and activation of a MAP kinase kinase (MAPKK), as part of a MAPK signaling cascade that contributes to cell wall biogenesis." [GOC:signaling]
GO:0038074
name: MAP kinase phosphatase activity involved in regulation of conjugation with cellular fusion
def: "Catalysis of the reaction: a phosphorylated MAP kinase + H2O = a MAP kinase + phosphate, where the MAP kinase (MAPK) is part of a MAPK signaling cascade that contributes to conjugation with cellular fusion." [GOC:signaling]
GO:0038075
name: MAP kinase activity involved in innate immune response
def: "Catalysis of the reaction: protein + ATP = protein phosphate + ADP by a mitogen-activated protein kinase, as part of a MAPK signaling cascade that contributes to an innate immune response." [GOC:signaling]
GO:0038076
name: MAP kinase kinase activity involved in innate immune response
def: "Catalysis of the concomitant phosphorylation of threonine (T) and tyrosine (Y) residues in a Thr-Glu-Tyr (TEY) thiolester sequence in a MAP kinase (MAPK) substrate, as part of a MAPK signaling cascade that contributes to an innate immune response." [GOC:signaling]
GO:0038077
name: MAP kinase kinase kinase activity involved in innate immune response
def: "Catalysis of the phosphorylation and activation of a MAP kinase kinase (MAPKK), as part of a MAPK signaling cascade that contributes to an innate immune response." [GOC:signaling]
GO:0038078
name: MAP kinase phosphatase activity involved in regulation of innate immune response
def: "Catalysis of the reaction: a phosphorylated MAP kinase + H2O = a MAP kinase + phosphate, where the MAP kinase (MAPK) is part of a MAPK signaling cascade that contributes to an innate immune response." [GOC:signaling]
GO:0038079
name: MAP kinase activity involved in osmosensory signaling pathway
def: "Catalysis of the reaction: protein + ATP = protein phosphate + ADP by a mitogen-activated protein kinase, as part of a MAPK signaling cascade that passes on a signal within an osmosensory signaling pathway." [GOC:signaling]
GO:0038080
name: MAP kinase kinase activity involved in osmosensory signaling pathway
def: "Catalysis of the concomitant phosphorylation of threonine (T) and tyrosine (Y) residues in a Thr-Glu-Tyr (TEY) thiolester sequence in a MAP kinase (MAPK) substrate, as part of a MAPK signaling cascade that passes on a signal within an osmosensory signaling pathway." [GOC:signaling]
GO:0038081
name: MAP kinase kinase kinase activity involved in osmosensory signaling pathway
def: "Catalysis of the phosphorylation and activation of a MAP kinase kinase (MAPKK), as part of a MAPK signaling cascade that passes on a signal within an osmosensory signaling pathway." [GOC:signaling]
GO:0038082
name: MAP kinase phosphatase activity involved in regulation of osmosensory signaling pathway
def: "Catalysis of the reaction: a phosphorylated MAP kinase + H2O = a MAP kinase + phosphate, where the MAP kinase (MAPK) is part of a MAPK signaling cascade that passes on a signal within an osmosensory signaling pathway." [GOC:signaling]
GO:0038085
name: vascular endothelial growth factor binding
def: "Interacting selectively and non-covalently with a vascular endothelial growth factor." [PMID:17470632, PR:000003096, PR:000007520, PR:000017284, PR:000017285]
GO:0038100
name: nodal binding
def: "Interacting selectively and non-covalently with a nodal protein, a member of the transforming growth factor-beta superfamily." [GOC:bf, PMID:20629020, PR:000000105]
GO:0038102
name: activin receptor antagonist activity
def: "Interacting with an activin receptor complex to reduce the action of another ligand, the agonist. A receptor antagonist does not initiate signaling upon binding to a receptor, but instead blocks an agonist from binding to the receptor." [GOC:signaling, PMID:15062104]
GO:0038103
name: activin receptor antagonist activity involved in negative regulation of nodal signaling pathway
def: "Interacting with an activin receptor to reduce the action of the agonist nodal. A receptor antagonist does not initiate signaling upon binding to a receptor, but instead blocks an agonist from binding to the receptor." [GOC:signaling, PMID:15062104]
GO:0038106
name: choriogonadotropin hormone binding
def: "Interacting selectively and non-covalently with choriogonadotropin hormone, a heterodimer, with an alpha subunit identical to that of luteinizing hormone (LH), follicle-stimulating hormone (FSH) and thyroid-stimulating hormone (TSH), and a unique beta subunit." [GOC:BHF, GOC:rl, Wikipedia:Human_chorionic_gonadotropin]
GO:0038117
name: C-C motif chemokine 19 receptor activity
def: "Combining with the C-C motif chemokine 19 (CCL19) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:signaling, PMID:15059845, PR:000001991]
GO:0038121
name: C-C motif chemokine 21 receptor activity
def: "Combining with the C-C motif chemokine 21 (CCL21) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:signaling, PMID:15059845, PR:000001993]
GO:0038122
name: C-C motif chemokine 5 receptor activity
def: "Combining with the C-C motif chemokine 5 (CCL5) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:signaling, PR:000002094]
GO:0038131
name: neuregulin receptor activity
def: "Combining with a neuregulin, a member of the EGF family of growth factors, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:signaling, PMID:16460914, PMID:20672328]
GO:0038132
name: neuregulin binding
def: "Interacting selectively and non-covalently with a neuregulin, a member of the EGF family of growth factors." [GOC:bf, GOC:signaling]
GO:0038147
name: C-X-C motif chemokine 12 receptor activity
def: "Combining with the C-X-C motif chemokine 12 (CXCL12) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, PMID:22204316]
GO:0038149
name: C-C motif chemokine 2 receptor activity
def: "Combining with the C-C motif chemokine 2 (CCL2) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:signaling]
GO:0038164
name: thrombopoietin receptor activity
def: "Combining with the glycoprotein thrombopoietin and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:signaling, PMID:19630807]
GO:0038174
name: interleukin-17A receptor activity
def: "Combining with the cytokine interleukin-17A and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:bf, GOC:jc, GOC:signaling, PR:000001138]
GO:0038177
name: death receptor agonist activity
def: "Interacting with a death receptor such that the proportion of death receptors in an active form is increased. Ligand binding to a death receptor often induces a conformational change to activate the receptor." [GOC:mtg_apoptosis, GOC:pr]
GO:0038181
name: bile acid receptor activity
def: "Combining with a bile acid and transmitting the signal to initiate a change in cell activity. A bile acid is any member of a group of steroid carboxylic acids occurring in bile." [CHEBI:3098, GOC:bf, PMID:10334992, PMID:12718893]
GO:0038182
name: G-protein coupled bile acid receptor activity
def: "Combining with an extracellular bile acid and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:bf, PMID:12524422, Wikipedia:G_protein-coupled_bile_acid_receptor]
GO:0038186
name: lithocholic acid receptor activity
def: "Combining with lithocholic acid and transmitting the signal to initiate a change in cell activity." [CHEBI:16325, GOC:bf, PMID:12016314, PMID:12419312]
GO:0038187
name: pattern recognition receptor activity
def: "Combining with a pathogen-associated molecular pattern (PAMP), a structure conserved among microbial species, or damage-associated molecular pattern (DAMP), an endogenous molecule released from damaged cells), to initiate a change in cell activity." [GOC:ar, GOC:bf, Wikipedia:Pattern_recognition_receptor]
GO:0038191
name: neuropilin binding
def: "Interacting selectively and non-covalently with a member of the neuropilin family." [GOC:bf, PMID:23871893]
GO:0038198
name: auxin receptor activity
def: "Combining with auxin and transmitting the signal in the cell to initiate a change in cell activity. Auxin is a plant hormone (phytohormone)." [GOC:signaling, PMID:15917797]
GO:0038199
name: ethylene receptor activity
def: "Combining with ethylene and transmitting the signal in the cell to initiate a change in cell activity." [GOC:signaling, PMID:22467798, PMID:24012247]
GO:0038200
name: ethylene receptor histidine kinase activity
def: "Combining with ethylene and transmitting the signal within the cell to initiate a change in cell activity by catalysis of the reaction: ATP + a protein-L-histidine = ADP + a protein-L-histidine phosphate." [GOC:signaling, PMID:22467798]
GO:0039552
name: RIG-I binding
def: "Interacting selectively and non-covalently with RIG-I, a cytosolic pattern recognition receptor that initiates an antiviral signaling pathway upon binding to viral RNA." [GOC:bf, PMID:21233210]
GO:0039556
name: MDA-5 binding
def: "Interacting selectively and non-covalently with MDA-5, a cytosolic pattern recognition receptor that initiates an antiviral signaling pathway upon binding to viral dsRNA." [PMID:19019954]
GO:0039594
name: endoribonuclease activity involved in viral induction of host mRNA catabolic process
def: "Any endoribonuclease activity that contributes to the viral-induced catabolism of host mRNA." [GOC:bf, PMID:22046136]
GO:0039603
name: TBP-class protein binding involved in viral suppression of host transcription initiation from RNA polymerase II promoter
def: "Selective and non-covalent interaction of a viral protein with a member of the class of TATA-binding proteins (TBP) in the host, including any of the TBP-related factors (TRFs), which contributes to the viral-suppression of assembly of the RNA polymerase II preinitiation complex (PIC) at an RNA polymerase II promoter region of a host DNA template." [GOC:vw, PMID:11968006]
GO:0039605
name: TFIIB-class transcription factor binding involved in viral suppression of host transcription initiation from RNA polymerase II promoter
def: "Selective and non-covalent interaction of a viral protein with a member of the TFIIB-class of host transcription factors, which contributes to the viral-suppression of assembly of the RNA polymerase II preinitiation complex (PIC) at an RNA polymerase II promoter region of a host DNA template." [GOC:vw, PMID:18768974]
GO:0039630
name: RNA translocase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive movement along a single- or double-stranded RNA molecule." [GOC:bm, PMID:22713318]
GO:0039631
name: DNA translocase activity involved in viral DNA genome packaging
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive movement along a single- or double-stranded DNA molecule, that contributes to the packing of viral DNA into a capsid." [GOC:bm, PMID:17501915]
GO:0039632
name: RNA translocase activity involved in viral RNA genome packaging
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive movement along a single- or double-stranded RNA molecule, which contributes to the packaging of viral RNA into a nucleocapsid." [GOC:bm, PMID:22297533]
GO:0039660
name: structural constituent of virion
def: "The action of a molecule that contributes to the structural integrity of a virion." [GOC:bf, GOC:jl]
GO:0039706
name: co-receptor binding
def: "Interacting selectively and non-covalently with a coreceptor. A coreceptor acts in cooperation with a primary receptor to transmit a signal within the cell." [GOC:bf, GOC:jl]
GO:0042007
name: interleukin-18 binding
def: "Interacting selectively and non-covalently with interleukin-18." [GOC:jl]
GO:0042008
name: interleukin-18 receptor activity
def: "Combining with interleukin-18 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042009
name: interleukin-15 binding
def: "Interacting selectively and non-covalently with interleukin-15." [GOC:jl]
GO:0042010
name: interleukin-15 receptor activity
def: "Combining with interleukin-15 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042011
name: interleukin-16 binding
def: "Interacting selectively and non-covalently with interleukin-16." [GOC:jl]
GO:0042012
name: interleukin-16 receptor activity
def: "Combining with interleukin-16 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042013
name: interleukin-19 binding
def: "Interacting selectively and non-covalently with interleukin-19." [GOC:jl]
GO:0042014
name: interleukin-19 receptor activity
def: "Combining with interleukin-19 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042015
name: interleukin-20 binding
def: "Interacting selectively and non-covalently with interleukin-20." [GOC:jl]
GO:0042016
name: interleukin-20 receptor activity
def: "Combining with interleukin-20 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042017
name: interleukin-22 binding
def: "Interacting selectively and non-covalently with interleukin-22." [GOC:jl]
GO:0042018
name: interleukin-22 receptor activity
def: "Combining with interleukin-22 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042019
name: interleukin-23 binding
def: "Interacting selectively and non-covalently with interleukin-23." [GOC:jl]
GO:0042020
name: interleukin-23 receptor activity
def: "Combining with interleukin-23 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0042021
name: granulocyte macrophage colony-stimulating factor complex binding
def: "Interacting selectively and non-covalently with the granulocyte macrophage colony-stimulating factor complex." [GOC:ai]
GO:0042027
name: cyclophilin-type peptidyl-prolyl cis-trans isomerase activity
def: "OBSOLETE. Catalysis of the reaction: peptidylproline (omega=180) = peptidylproline (omega=0)." [EC:5.2.1.8]
GO:0042029
name: fibrolase activity
def: "OBSOLETE. Catalysis of the cleavage of 14-Ala-Leu-15 in insulin B chain and cleavage of 413-Lys-Leu-414 in alpha chain of fibrinogen." [EC:3.4.24.72, PMID:7725320]
GO:0042030
name: ATPase inhibitor activity
def: "Stops, prevents or reduces the activity of any enzyme that catalyzes the hydrolysis of ATP to ADP and orthophosphate." [GOC:jl]
GO:0042031
name: angiotensin-converting enzyme inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of angiotensin-converting enzyme, thereby preventing the synthesis of angiotensin II from its precursor, angiotensin I." [GOC:jl]
GO:0042043
name: neurexin family protein binding
def: "Interacting selectively and non-covalently with neurexins, synaptic cell surface proteins related to latrotoxin receptor, laminin and agrin. Neurexins act as cell recognition molecules at nerve terminals." [GOC:curators, GOC:pr, PMID:18923512]
GO:0042054
name: histone methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone = S-adenosyl-L-homocysteine + methyl-histone. Histone methylation generally occurs on either an arginine or lysine residue." [EC:2.1.1.43]
GO:0042056
name: chemoattractant activity
def: "Providing the environmental signal that initiates the directed movement of a motile cell or organism towards a higher concentration of that signal." [GOC:go_curators, ISBN:0198506732]
GO:0042057
name: transforming growth factor beta receptor anchoring activity
def: "OBSOLETE. Binds to transforming growth factor beta receptor and anchors it to a particular subcellular location." [GOC:ai]
GO:0042064
name: cell adhesion receptor regulator activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0042071
name: leucokinin receptor activity
def: "Combining with a leucokinin, any of several octapeptide hormones found in insects, and transmitting the signal to initiate a change in cell activity." [GOC:mah, GOC:signaling, PMID:2716741]
GO:0042072
name: cell adhesion receptor inhibitor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:ai]
GO:0042083
name: 5,10-methylenetetrahydrofolate-dependent methyltransferase activity
def: "Catalysis of the transfer of a methyl group to an acceptor molecule; dependent on the presence of 5,10-methylenetetrahydrofolate." [GOC:ai]
GO:0042084
name: 5-methyltetrahydrofolate-dependent methyltransferase activity
def: "Catalysis of the transfer of a methyl group to an acceptor molecule; dependent on the presence of 5-methyltetrahydrofolate." [GOC:ai]
GO:0042085
name: 5-methyltetrahydropteroyltri-L-glutamate-dependent methyltransferase activity
def: "Catalysis of the transfer of a methyl group to an acceptor molecule; dependent on the presence of 5-methyltetrahydropteroyltri-L-glutamate." [GOC:ai]
GO:0042086
name: 5-methyl-5,6,7,8-tetrahydromethanopterin-dependent methyltransferase activity
def: "Catalysis of the transfer of a methyl group to an acceptor molecule, dependent on the presence of 5-methyl-5,6,7,8-tetrahydromethanopterin." [GOC:ai]
GO:0042096
name: alpha-beta T cell receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0042099
name: gamma-delta T cell receptor activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:jl]
GO:0042123
name: glucanosyltransferase activity
def: "Catalysis of the splitting and linkage of glucan molecules, resulting in glucan chain elongation." [GOC:jl]
GO:0042124
name: 1,3-beta-glucanosyltransferase activity
def: "Catalysis of the splitting and linkage of (1->3)-beta-D-glucan molecules, resulting in (1->3)-beta-D-glucan chain elongation." [GOC:jl, PMID:10809732]
GO:0042131
name: thiamine phosphate phosphatase activity
def: "Catalysis of the hydrolysis of thiamine phosphate." [PMID:197075]
GO:0042132
name: fructose 1,6-bisphosphate 1-phosphatase activity
def: "Catalysis of the reaction: D-fructose 1,6-bisphosphate + H2O = D-fructose 6-phosphate + phosphate." [EC:3.1.3.11]
GO:0042134
name: rRNA primary transcript binding
def: "Interacting selectively and non-covalently with an unprocessed ribosomal RNA transcript." [GOC:jl]
GO:0042141
name: mating pheromone exporter
def: "OBSOLETE. Exports diffusible peptide signals that are responsible for binding to other cells and triggering a series of responses to facilitate mating." [GOC:jl]
GO:0042153
name: RPTP-like protein binding
def: "OBSOLETE. Interacting selectively and non-covalently with proteins with similar structure/function to receptor protein tyrosine phosphatases." [GOC:jl]
GO:0042156
name: zinc-mediated transcriptional activator activity
def: "OBSOLETE. Initiates or upregulates transcription in the presence of zinc." [GOC:jl]
GO:0042162
name: telomeric DNA binding
def: "Interacting selectively and non-covalently with a telomere, a specific structure at the end of a linear chromosome required for the integrity and maintenance of the end." [GOC:jl, SO:0000624]
GO:0042163
name: interleukin-12 beta subunit binding
def: "Interacting selectively and non-covalently with the beta subunit of interleukin-12." [GOC:mah]
GO:0042164
name: interleukin-12 alpha subunit binding
def: "Interacting selectively and non-covalently with the alpha subunit of interleukin-12." [GOC:mah]
GO:0042165
name: neurotransmitter binding
def: "Interacting selectively and non-covalently with a neurotransmitter, any chemical substance that is capable of transmitting (or inhibiting the transmission of) a nerve impulse from a neuron to another cell." [ISBN:0198506732]
GO:0042166
name: acetylcholine binding
def: "Interacting selectively and non-covalently with acetylcholine, an acetic acid ester of the organic base choline that functions as a neurotransmitter, released at the synapses of parasympathetic nerves and at neuromuscular junctions." [GOC:ai]
GO:0042169
name: SH2 domain binding
def: "Interacting selectively and non-covalently with a SH2 domain (Src homology 2) of a protein, a protein domain of about 100 amino-acid residues and belonging to the alpha + beta domain class." [GOC:go_curators, Pfam:PF00017]
GO:0042171
name: lysophosphatidic acid acyltransferase activity
def: "Catalysis of the transfer of acyl groups from an acyl-CoA to lysophosphatidic acid to form phosphatidic acid." [GOC:ab, PMID:16369050]
GO:0042242
name: cobyrinic acid a,c-diamide synthase activity
def: "Catalysis of the conversion of cobyrinic acid to cobyrinic acid a,c-diamide via the intermediate formation of cobyrinic acid c-monoamide." [PMID:2172209]
GO:0042263
name: neuropeptide F receptor activity
def: "Combining with neuropeptide F and transmitting the signal within the cell to initiate a change in cell activity. Neuropeptide F is an arthropod peptide of more than 28 residues (typically 28-45) with a consensus C-terminal RxRFamide (commonly RPRFa, but also RVRFa." [GOC:bf, GOC:ma, PMID:21440021]
GO:0042277
name: peptide binding
def: "Interacting selectively and non-covalently with peptides, any of a group of organic compounds comprising two or more amino acids linked by peptide bonds." [GOC:jl]
GO:0042279
name: nitrite reductase (cytochrome, ammonia-forming) activity
def: "Catalysis of the reaction: NH3 + 2 H2O + 6 ferricytochrome c = nitrite + 6 ferrocytochrome c + 7 H+." [EC:1.7.2.2]
GO:0042280
name: cell surface antigen activity, host-interacting
def: "OBSOLETE. Functions as an immunogenic target for the host immune system that masks other invariant surface molecules from immune recognition." [GOC:mb]
GO:0042281
name: dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase activity
def: "Catalysis of the addition of the first glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Man(9)GlcNAc(2)-PP-Dol." [GOC:al, MetaCyc:RXN-5470]
GO:0042282
name: hydroxymethylglutaryl-CoA reductase activity
def: "Catalysis of the reaction: (R)-mevalonate + CoA + 2 NAD(+) = (S)-3-hydroxy-3-methylglutaryl-CoA + 2 H(+) + 2 NADH." [EC:1.1.1.88, RHEA:14836]
GO:0042283
name: dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase activity
def: "Catalysis of the addition of the second glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Glc(1)Man(9)GlcNAc(2)-PP-Dol." [MetaCyc:RXN-5471, PMID:12480927]
GO:0042284
name: sphingolipid delta-4 desaturase activity
def: "Catalysis of the introduction of a trans double bond between C4 and C5 of the long chain base region of a sphingolipid. Sphingolipids are composed of a long chain base (LCB) amide-linked to a very long chain fatty acid." [PMID:12417141]
GO:0042285
name: xylosyltransferase activity
def: "Catalysis of the transfer of a xylosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
GO:0042286
name: glutamate-1-semialdehyde 2,1-aminomutase activity
def: "Catalysis of the reaction: (S)-4-amino-5-oxopentanoate = 5-aminolevulinate." [EC:5.4.3.8, RHEA:14268]
GO:0042287
name: MHC protein binding
def: "Interacting selectively and non-covalently with major histocompatibility complex molecules; a set of molecules displayed on cell surfaces that are responsible for lymphocyte recognition and antigen presentation." [GOC:jl]
GO:0042288
name: MHC class I protein binding
def: "Interacting selectively and non-covalently with major histocompatibility complex class I molecules; a set of molecules displayed on cell surfaces that are responsible for lymphocyte recognition and antigen presentation." [GOC:jl]
GO:0042289
name: MHC class II protein binding
def: "Interacting selectively and non-covalently with major histocompatibility complex class II molecules; a set of molecules displayed on cell surfaces that are responsible for lymphocyte recognition and antigen presentation." [GOC:jl]
GO:0042290
name: URM1 hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0042291
name: Hub1 hydrolase activity
def: "OBSOLETE (was not defined before being made obsolete)." [GOC:mah]
GO:0042292
name: URM1 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier URM1, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0042293
name: Hub1 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier Hub1, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:mah]
GO:0042294
name: URM1 transferase activity
def: "Catalysis of the transfer of URM1 from one protein to another via the reaction X-URM1 + Y --> Y-URM1 + X, where both X-URM1 and Y-URM1 are covalent linkages." [GOC:mah, PMID:12826404]
GO:0042296
name: ISG15 transferase activity
def: "Catalysis of the transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y --> Y-ISG15 + X, where both X-ISG15 and Y-ISG15 are covalent linkages." [GOC:mah, PMID:12826404]
GO:0042299
name: lupeol synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = lupeol. This reaction is the cyclization of (S)-2,3-epoxysqualene (2,3-oxidosqualene) to lupeol." [MetaCyc:RXN-111, PMID:9883589]
GO:0042300
name: beta-amyrin synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = beta-amyrin. This reaction is the cyclization and rearrangement of (S)-2,3-epoxysqualene (2,3-oxidosqualene) into beta-amyrin." [PMID:9746369]
GO:0042301
name: phosphate ion binding
def: "Interacting selectively and non-covalently with phosphate." [GOC:jl]
GO:0042302
name: structural constituent of cuticle
def: "The action of a molecule that contributes to the structural integrity of a cuticle." [GOC:jl]
GO:0042314
name: bacteriochlorophyll binding
def: "Interacting selectively and non-covalently with bacteriochlorophyll, a form of chlorophyll found in photosynthetic bacteria, such as the purple and green bacteria. There are several types, designated a to g. Bacteriochlorophyll a and bacteriochlorophyll b are structurally similar to the chlorophyll a and chlorophyll b found in plants." [ISBN:0192800981]
GO:0042315
name: cytosol nonspecific dipeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of dipeptides, preferentially hydrophobic dipeptides including prolyl amino acids." [EC:3.4.13.18]
GO:0042324
name: hypocretin receptor binding
def: "Interacting selectively and non-covalently with the hypocretin receptor." [GOC:ceb, PMID:11988773]
GO:0042328
name: heparan sulfate N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + heparan sulfate = UDP + (N-acetyl-D-glucosaminyl)-heparan sulfate." [GOC:ma]
GO:0042329
name: structural constituent of collagen and cuticulin-based cuticle
def: "The action of a molecule that contributes to the structural integrity of a collagen and cuticulin-based cuticle. An example of this process is found in Caenorhabditis elegans." [GOC:jl, GOC:mtg_sensu]
GO:0042349
name: guiding stereospecific synthesis activity
def: "The orientation of free radical substrates in such a way that only a particular stereoisomer is synthesized by an enzyme. Best characterized as a function during lignan biosynthesis." [GOC:ma]
GO:0042356
name: GDP-4-dehydro-D-rhamnose reductase activity
def: "Catalysis of the reaction: GDP-6-deoxy-D-mannose + NAD(P)+ = GDP-4-dehydro-6-deoxy-D-mannose + NAD(P)H + H+. In the reverse reaction, a mixture of GDP-D-rhamnose and its C-4 epimer is formed." [BRENDA:1.1.1.187, EC:1.1.1.187]
GO:0042379
name: chemokine receptor binding
def: "Interacting selectively and non-covalently with any chemokine receptor." [GOC:ai]
GO:0042380
name: hydroxymethylbutenyl pyrophosphate reductase activity
def: "Catalysis of the formation of both isopentenyl pyrophosphate and dimethylallyl pyrophosphate from (E)-4-hydroxy-3-methyl-but-2-enyl pyrophosphate." [GOC:js]
GO:0042389
name: omega-3 fatty acid desaturase activity
def: "Catalysis of the introduction of an omega-3 double bond into the fatty acid hydrocarbon chain." [GOC:jl, PMID:9037020]
GO:0042392
name: sphingosine-1-phosphate phosphatase activity
def: "Catalysis of the reaction: sphingosine 1-phosphate + H2O = sphingosine + phosphate." [GOC:jl, PMID:11331102]
GO:0042393
name: histone binding
def: "Interacting selectively and non-covalently with a histone, any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes. They are involved in the condensation and coiling of chromosomes during cell division and have also been implicated in nonspecific suppression of gene activity." [GOC:jl]
GO:0042408
name: myrcene/(E)-beta-ocimene synthase activity
def: "OBSOLETE. Catalysis of the conversion of geranyl diphosphate (GPP) into the acyclic monoterpenes beta-myrcene, (E)-beta-ocimene, and other minor cyclic monoterpenes." [GOC:cr, PMID:10700382]
GO:0042409
name: caffeoyl-CoA O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + caffeoyl-CoA = S-adenosyl-L-homocysteine + feruloyl-CoA." [EC:2.1.1.104]
GO:0042410
name: 6-carboxyhexanoate-CoA ligase activity
def: "Catalysis of the reaction: ATP + CoA + pimelate = AMP + diphosphate + H(+) + pimelyl-CoA." [EC:6.2.1.14, RHEA:14784]
GO:0042469
name: versicolorin reductase activity
def: "Catalysis of the reduction of versicolorin A to sterigmatocystin." [PMID:1339261]
GO:0042497
name: triacyl lipopeptide binding
def: "Interacting selectively and non-covalently with a lipopeptide containing a nonprotein moiety consisting of three acyl groups." [GOC:add, PMID:12077222, PMID:12524386, PMID:2757794]
GO:0042498
name: diacyl lipopeptide binding
def: "Interacting selectively and non-covalently with a lipopeptide containing a nonprotein moiety consisting of two acyl groups." [GOC:add, PMID:12077222, PMID:12524386, PMID:2757794]
GO:0042499
name: signal peptide peptidase activity
def: "OBSOLETE. Catalysis of the intramembrane proteolysis of a signal peptide, following its removal from a preprotein." [PMID:12077416]
GO:0042500
name: aspartic endopeptidase activity, intramembrane cleaving
def: "Catalysis of the hydrolysis of nonterminal peptide bonds in a polypeptide chain, occurring within a membrane." [GOC:jl, ISBN:0198506732]
GO:0042556
name: eukaryotic elongation factor-2 kinase regulator activity
def: "Modulates the activity of the enzyme eukaryotic elongation factor-2 kinase." [GOC:jl, PMID:11904175]
GO:0042557
name: eukaryotic elongation factor-2 kinase activator activity
def: "Increases the activity of the enzyme eukaryotic elongation factor-2 kinase." [GOC:jl, PMID:11904175]
GO:0042561
name: alpha-amyrin synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = alpha-amyrin. This reaction is a cyclization and rearrangement of (S)-2,3-epoxysqualene (2,3-oxidosqualene) into alpha-amyrin." [GOC:jl, MetaCyc:RXN-8434, PMID:10848960]
GO:0042562
name: hormone binding
def: "Interacting selectively and non-covalently with any hormone, naturally occurring substances secreted by specialized cells that affect the metabolism or behavior of other cells possessing functional receptors for the hormone." [CHEBI:24621, GOC:jl]
GO:0042576
name: aspartyl aminopeptidase activity
def: "OBSOLETE. Catalysis of the release of an N-terminal aspartate or glutamate from a peptide, with a preference for aspartate." [EC:3.4.11.21]
GO:0042577
name: lipid phosphatase activity
def: "Catalysis of the reaction: a phospholipid + H2O = a lipid + phosphate." [GOC:jl]
GO:0042578
name: phosphoric ester hydrolase activity
def: "Catalysis of the reaction: RPO-R' + H2O = RPOOH + R'H. This reaction is the hydrolysis of any phosphoric ester bond, any ester formed from orthophosphoric acid, O=P(OH)3." [GOC:jl]
GO:0042586
name: peptide deformylase activity
def: "Catalysis of the reaction: formyl-L-methionyl peptide + H2O = formate + methionyl peptide." [EC:3.5.1.88, GOC:jl]
GO:0042602
name: riboflavin reductase (NADPH) activity
def: "Catalysis of the reaction: reduced riboflavin + NADP+ = riboflavin + NADPH + 2 H+." [EC:1.5.1.30]
GO:0042605
name: peptide antigen binding
def: "Interacting selectively and non-covalently with an antigen peptide." [GOC:add, GOC:jl, GOC:rv]
GO:0042608
name: T cell receptor binding
def: "Interacting selectively and non-covalently with a T cell receptor, the antigen-recognizing receptor on the surface of T cells." [GOC:jl]
GO:0042609
name: CD4 receptor binding
def: "Interacting selectively and non-covalently with a CD4, a receptor found on the surface of T cells, monocytes and macrophages." [GOC:jl, MSH:D015704]
GO:0042610
name: CD8 receptor binding
def: "Interacting selectively and non-covalently with a CD8, a receptor found on the surface of thymocytes and cytotoxic and suppressor T-lymphocytes." [GOC:jl, MSH:D016827]
GO:0042614
name: CD70 receptor binding
def: "Interacting selectively and non-covalently with a CD70, a receptor found on the surface of most activated B cells and some activated T cells." [GOC:jl, ISBN:0120781859]
GO:0042615
name: CD154 receptor binding
def: "Interacting selectively and non-covalently with CD154, a receptor found on the surface of some activated lymphocytes." [GOC:jl, ISBN:0120781859]
GO:0042623
name: ATPase activity, coupled
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction directly drives some other reaction, for example ion transport across a membrane." [EC:3.6.1.3, GOC:jl]
GO:0042624
name: ATPase activity, uncoupled
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction is not directly coupled to any other reaction." [EC:3.6.1.3]
GO:0042625
name: ATPase activity, coupled to transmembrane movement of ions
def: "Catalysis of the transfer of an ion from one side of a membrane to the other, driven by the reaction: ATP + H2O = ADP + phosphate." [EC:3.6.1.3, GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042626
name: ATPase activity, coupled to transmembrane movement of substances
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to directly drive the active transport of a substance across a membrane." [EC:3.6.1.3, GOC:jl]
GO:0042654
name: ecdysis-triggering hormone receptor activity
def: "Combining with ecdysis-triggering hormone to initiate a change in cell activity." [GOC:ma]
GO:0042656
name: JUN kinase kinase kinase kinase activity
def: "Catalysis of the phosphorylation and activation of JUN kinase kinase kinases (JNKKKs)." [GOC:bf]
GO:0042657
name: MHC class II protein binding, via lateral surface
def: "Interacting selectively and non-covalently with the lateral surface of major histocompatibility complex class II molecules." [GOC:jl]
GO:0042658
name: MHC class II protein binding, via antigen binding groove
def: "Interacting selectively and non-covalently with the antigen binding groove of major histocompatibility complex class II molecules." [GOC:jl]
GO:0042708
name: elastase activity
def: "OBSOLETE. Catalysis of the hydrolysis of elastin." [ISBN:0198506732]
GO:0042731
name: PH domain binding
def: "Interacting selectively and non-covalently with a PH domain (pleckstrin homology) of a protein, a domain of about 100 residues that occurs in a wide range of proteins involved in intracellular signaling or as constituents of the cytoskeleton." [GOC:jl, Pfam:PF00169]
GO:0042736
name: NADH kinase activity
def: "Catalysis of the reaction: ATP + NADH = ADP + 2 H(+) + NADPH." [EC:2.7.1.86, RHEA:12263]
GO:0042767
name: ecdysteroid 22-hydroxylase activity
def: "Catalysis of the hydroxylation of an ecdysteroid at carbon position 22." [PMID:12177427]
GO:0042768
name: ecdysteroid 2-hydroxylase activity
def: "Catalysis of the hydroxylation of an ecdysteroid at carbon position 2." [PMID:12177427]
GO:0042781
name: 3'-tRNA processing endoribonuclease activity
def: "Catalysis of the endonucleolytic cleavage of RNA, removing extra 3' nucleotides from tRNA precursor, generating 3' termini of tRNAs. A 3'-hydroxy group is left at the tRNA terminus and a 5'-phosphoryl group is left at the trailer molecule." [EC:3.1.26.11]
GO:0042799
name: histone methyltransferase activity (H4-K20 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H4 L-lysine (position 20) = S-adenosyl-L-homocysteine + histone H4 N6-methyl-L-lysine (position 20). This reaction is the addition of a methyl group onto lysine at position 20 of the histone H4 protein." [PMID:12086618]
GO:0042800
name: histone methyltransferase activity (H3-K4 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 4) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 4). This reaction is the addition of a methyl group onto lysine at position 4 of the histone H3 protein." [PMID:12086618]
GO:0042801
name: polo kinase kinase activity
def: "Catalysis of the addition of a phosphate group onto a serine or threonine residue in any member of the polo kinase class of proteins." [GOC:ma]
GO:0042802
name: identical protein binding
def: "Interacting selectively and non-covalently with an identical protein or proteins." [GOC:jl]
GO:0042803
name: protein homodimerization activity
def: "Interacting selectively and non-covalently with an identical protein to form a homodimer." [GOC:jl]
GO:0042804
name: protein homooligomerization activity
def: "OBSOLETE. Interacting selectively with identical proteins to form a homooligomer." [GOC:jl]
GO:0042805
name: actinin binding
def: "Interacting selectively and non-covalently with actinin, any member of a family of proteins that crosslink F-actin." [GOC:jl, ISBN:0198506732]
GO:0042806
name: fucose binding
def: "Interacting selectively and non-covalently with fucose, the pentose 6-deoxygalactose." [CHEBI:33984, ISBN:0582227089]
GO:0042808
name: neuronal Cdc2-like kinase binding
def: "OBSOLETE. Interacting selectively and non-covalently with neuronal Cdc2-like kinase, an enzyme involved in the regulation of neuronal differentiation and neuro-cytoskeleton dynamics." [GOC:jl, PMID:10721722]
GO:0042809
name: vitamin D receptor binding
def: "Interacting selectively and non-covalently with the vitamin D receptor, a nuclear receptor that mediates the action of vitamin D by binding DNA and controlling the transcription of hormone-sensitive genes." [GOC:jl, PMID:12637589]
GO:0042813
name: Wnt-activated receptor activity
def: "Combining with a Wnt protein and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:go_curators]
GO:0042826
name: histone deacetylase binding
def: "Interacting selectively and non-covalently with the enzyme histone deacetylase." [GOC:jl]
GO:0042834
name: peptidoglycan binding
def: "Interacting selectively and non-covalently, in a non-covalent manner, with peptidoglycan, any of a class of glycoconjugates found in bacterial cell walls." [GOC:go_curators, PMID:14698226]
GO:0042835
name: BRE binding
def: "Interacting selectively and non-covalently with the RNA element BRE (Bruno response element)." [PMID:10893231]
GO:0042876
name: aldarate transmembrane transporter activity
def: "Catalysis of the transfer of aldarate from one side of the membrane to the other." [GOC:go_curators, PMID:15034926]
GO:0042878
name: D-glucarate transmembrane transporter activity
def: "Catalysis of the transfer of D-glucarate, the D-enantiomer of glucarate, from one side of the membrane to the other." [GOC:jl, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0042879
name: aldonate transmembrane transporter activity
def: "Catalysis of the transfer of aldonate from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042880
name: D-glucuronate transmembrane transporter activity
def: "Catalysis of the transfer of D-glucuronate, the D-enantiomer of glucuronate, from one side of the membrane to the other." [GOC:jl, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0042881
name: D-galactonate transmembrane transporter activity
def: "Catalysis of the transfer of D-galactonate, the D-enantiomer of galactonate, from one side of the membrane to the other." [GOC:jl, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0198506732, ISBN:0815340729]
GO:0042887
name: amide transmembrane transporter activity
def: "Enables the directed movement of an amide, any compound containing one, two, or three acyl groups attached to a nitrogen atom, from one side of the membrane to the other." [GOC:jl, ISBN:0198506732]
GO:0042888
name: molybdenum ion transmembrane transporter activity
def: "Catalysis of the transfer of molybdenum (Mo) ions from one side of a membrane to the other." [GOC:jl, ISBN:0198506732]
GO:0042890
name: 3-phenylpropionic acid transmembrane transporter activity
def: "Catalysis of the transfer of 3-phenylpropionic acid from one side of the membrane to the other." [GOC:jl]
GO:0042895
name: antibiotic transporter activity
def: "Enables the directed movement of an antibiotic, a substance produced by or derived from certain fungi, bacteria, and other organisms, that can destroy or inhibit the growth of other microorganisms, into, out of or within a cell, or between cells." [GOC:jl]
GO:0042896
name: chloramphenicol transporter activity
def: "Enables the directed movement of chloramphenicol, a broad-spectrum antibiotic that inhibits bacterial protein synthesis, into, out of or within a cell, or between cells." [CHEBI:17698, GOC:jl]
GO:0042897
name: polymyxin transporter activity
def: "Enables the directed movement of polymyxin, any of a group of related antibiotics produced by Bacillus polymyxa and active against most Gram-negative bacteria, into, out of or within a cell, or between cells." [GOC:jl, ISBN:0198506732]
GO:0042898
name: fosmidomycin transporter activity
def: "Enables the directed movement of fosmidomycin, a phosphonic acid derivative with potent activity against Gram-negative organisms, into, out of or within a cell, or between cells." [GOC:jl, PMID:12543685]
GO:0042900
name: arabinose transmembrane transporter activity
def: "Catalysis of the transfer of arabinose, a pentose monosaccharide that occurs in both D and L configurations, and as a polymer, from one side of the membrane to the other." [CHEBI:22599, GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042901
name: arabinan transmembrane transporter activity
def: "Catalysis of the transfer of an arabinan, a polysaccharide composed of arabinose residues, from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042903
name: tubulin deacetylase activity
def: "Catalysis of the reaction: N-acetyl(alpha-tubulin) + H2O = alpha-tubulin + acetate." [PMID:12024216, PMID:12486003]
GO:0042907
name: xanthine transmembrane transporter activity
def: "Catalysis of the transfer of xanthine from one side of a membrane to the other. Xanthine (2,6-dihydroxypurine) is a purine formed in the metabolic breakdown of guanine, but is not present in nucleic acids." [GOC:jl]
GO:0042910
name: xenobiotic transporter activity
def: "Enables the directed movement of a xenobiotic, a compound foreign to living organisms, into, out of or within a cell, or between cells." [GOC:go_curators]
GO:0042911
name: acridine transporter activity
def: "Enables the directed movement of acridine (10-azaanthracene), a heterocyclic ring compound found in crude coal-tar anthracene, into, out of or within a cell, or between cells." [CHEBI:36420, GOC:jl]
GO:0042912
name: colicin transmembrane transporter activity
def: "Catalysis of the transfer of a colicin from one side of a membrane to the other. Colicins are a group of antibiotics produced by E. coli and related species that are encoded by a group of naturally occurring plasmids, e.g. Col E1." [GOC:jl, GOC:mtg_transport, ISBN:0815340729, PMID:17347522]
GO:0042913
name: group A colicin transmembrane transporter activity
def: "Catalysis of the transfer of group A colicins (colicins E1, E2, E3, A, K, and N) from one side of a membrane to the other." [GOC:jl, GOC:mtg_transport, ISBN:0815340729, PMID:9171417]
GO:0042917
name: alkylphosphonate transmembrane transporter activity
def: "Catalysis of the transfer of an alkylphosphonate from one side of a membrane to the other." [GOC:go_curators]
GO:0042922
name: neuromedin U receptor binding
def: "Interacting selectively and non-covalently with one or more specific sites on a neuromedin U receptor." [GOC:jl, PMID:10899166]
GO:0042923
name: neuropeptide binding
def: "Interacting selectively and non-covalently and stoichiometrically with neuropeptides, peptides with direct synaptic effects (peptide neurotransmitters) or indirect modulatory effects on the nervous system (peptide neuromodulators)." [http://www.wormbook.org/chapters/www_neuropeptides/neuropeptides.html]
GO:0042924
name: neuromedin U binding
def: "Interacting selectively and non-covalently and stoichiometrically with neuromedin U, a hypothalamic peptide involved in energy homeostasis and stress responses." [GOC:jl, PMID:12584108]
GO:0042925
name: benzoate transporter activity
def: "Enables the directed movement of benzoate, the anion of benzoic acid (benzenecarboxylic acid) into, out of or within a cell, or between cells." [GOC:jl, ISBN:0721662544]
GO:0042926
name: 3-hydroxyphenylpropionic acid transporter activity
def: "Enables the directed movement of 3-hydroxyphenylpropionic acid into, out of or within a cell, or between cells." [GOC:jl]
GO:0042927
name: siderophore transporter activity
def: "Enables the directed movement of siderophores, low molecular weight Fe(III)-chelating substances, into, out of or within a cell, or between cells." [GOC:go_curators]
GO:0042929
name: ferrichrome transporter activity
def: "Enables the directed movement of a ferrichrome into, out of or within a cell, or between cells. Ferrichromes are any of a group of growth-promoting Fe(III) chelates formed by various genera of microfungi. They are homodetic cyclic hexapeptides made up of a tripeptide of glycine (or other small neutral amino acids) and a tripeptide of an N'acyl-N4-hydroxy-L-ornithine." [GOC:jl, ISBN:0198506732]
GO:0042931
name: enterobactin transporter activity
def: "Enables the directed movement of the siderochrome enterochelin, a cyclic trimer of 2, 3 dihydroxybenzoylserine into, out of or within a cell, or between cells." [CHEBI:28855, GOC:jl]
GO:0042933
name: chrysobactin transporter activity
def: "Enables the directed movement of the siderophore chrysobactin (alpha-N-(2,3-dihydroxybenzoyl)-D-lysyl-L-serine) into, out of or within a cell, or between cells." [GOC:jl, PMID:8837459]
GO:0042934
name: achromobactin transporter activity
def: "Enables the directed movement of achromobactin, a citrate siderophore, into, out of or within a cell, or between cells." [GOC:jl]
GO:0042936
name: dipeptide transporter activity
def: "Enables the directed movement of a dipeptide, a combination of two amino acids by means of a peptide (-CO-NH-) link, into, out of or within a cell, or between cells." [CHEBI:46761, GOC:jl]
GO:0042937
name: tripeptide transporter activity
def: "Enables the directed movement of a tripeptide, a compound containing three amino acids linked together by peptide bonds, into, out of or within a cell, or between cells." [CHEBI:47923, GOC:jl]
GO:0042943
name: D-amino acid transmembrane transporter activity
def: "Catalysis of the transfer of D-amino acids from one side of a membrane to the other. D-amino acids are the D-enantiomers of amino acids." [GOC:jl, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
GO:0042944
name: D-alanine transmembrane transporter activity
def: "Catalysis of the transfer of D-alanine from one side of a membrane to the other. D-alanine is the D-enantiomer of 2-aminopropanoic acid." [CHEBI:15570, GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042945
name: D-serine transmembrane transporter activity
def: "Catalysis of the transfer of D-serine from one side of a membrane to the other. D-serine is the D-enantiomer of 2-amino-3-hydroxypropanoic acid." [GOC:jl, GOC:jsg, GOC:mah, GOC:mtg_transport, ISBN:0815340729]
GO:0042947
name: glucoside transmembrane transporter activity
def: "Catalysis of the transfer of glucosides from one side of the membrane to the other. Glucosides are glycosides in which the sugar group is a glucose residue." [GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042950
name: salicin transmembrane transporter activity
def: "Catalysis of the transfer of salicin (saligenin-beta-D-glucopyranoside), a glucoside of o-hydroxybenzylalcohol, from one side of the membrane to the other." [CHEBI:17814, GOC:jl, GOC:mtg_transport, ISBN:0815340729]
GO:0042951
name: arbutin transmembrane transporter activity
def: "Catalysis of the transfer of arbutin, a glycoside found in the bearberry and related plants which has been used to treat urinary-tract diseases, from one side of the membrane to the other." [GOC:jl, GOC:mtg_transport, http://biotech.icmb.utexas.edu/, ISBN:0815340729]
GO:0042954
name: lipoprotein transporter activity
def: "Enables the directed movement of any conjugated, water-soluble protein in which the nonprotein group consists of a lipid or lipids, into, out of or within a cell, or between cells." [GOC:jl, ISBN:0198506732]
GO:0042957
name: dextrin transmembrane transporter activity
def: "Catalysis of the transfer of dextrin, any one, or the mixture, of the intermediate polysaccharides formed during the hydrolysis of starch, which are dextrorotatory, soluble in water, and precipitable in alcohol, from one side of the membrane to the other." [GOC:jl, GOC:vk, http://www.mercksource.com/]
GO:0042958
name: maltodextrin transmembrane transporter activity
def: "Catalysis of the transfer of maltodextrin, any polysaccharide of glucose residues in beta-(1,4) linkage, from one side of the membrane to the other." [GOC:jl, http://www.mercksource.com/, PMID:15034926]
GO:0042959
name: alkanesulfonate transporter activity
def: "Enables the directed movement of alkanesulfonate into, out of or within a cell, or between cells." [GOC:jl]
GO:0042960
name: antimonite secondary active transmembrane transporter activity
def: "Catalysis of the transfer of antimonite from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:jl]
GO:0042961
name: antimonite-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + antimonite(in) = ADP + phosphate + antimonite(out)." [EC:3.6.3.16]
GO:0042962
name: acridine:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + acridine(in) = H+(in) + acridine(out)." [PMID:10735876]
GO:0042970
name: homoserine transmembrane transporter activity
def: "Catalysis of the transfer of homoserine from one side of a membrane to the other. Homoserine is alpha-amino-gamma-hydroxybutyric acid, an intermediate in the biosynthesis of cystathionine, threonine and methionine." [GOC:go_curators, ISBN:0198506732]
GO:0042971
name: lactone transmembrane transporter activity
def: "Enables the directed movement of lactone from one side of a membrane to the other. A lactone is a cyclic ester of a hydroxy carboxylic acid, containing a 1-oxacycloalkan-2-one structure, or an analogue having unsaturation or heteroatoms replacing one or more carbon atoms of the ring." [CHEBI:25000, GOC:go_curators]
GO:0042972
name: licheninase activity
def: "Catalysis of the hydrolysis of (1->4)-beta-D-glucosidic linkages in beta-D-glucans containing (1->3) and (1->4) bonds." [EC:3.2.1.73]
GO:0042973
name: glucan endo-1,3-beta-D-glucosidase activity
def: "Catalysis of the hydrolysis of (1->3)-beta-D-glucosidic linkages in (1->3)-beta-D-glucans." [EC:3.2.1.39]
GO:0042974
name: retinoic acid receptor binding
def: "Interacting selectively and non-covalently with the retinoic acid receptor, a ligand-regulated transcription factor belonging to the nuclear receptor superfamily." [GOC:jl, PMID:12476796]
GO:0042975
name: peroxisome proliferator activated receptor binding
def: "Interacting selectively and non-covalently with any of the peroxisome proliferator activated receptors, alpha, beta or gamma." [GOC:jl, PMID:12769781]
GO:0042978
name: ornithine decarboxylase activator activity
def: "Upregulation of the activity of the enzyme ornithine decarboxylase." [GOC:jl]
GO:0042979
name: ornithine decarboxylase regulator activity
def: "Modulation of the activity of the enzyme ornithine decarboxylase." [GOC:jl]
GO:0042980
name: cystic fibrosis transmembrane conductance regulator binding
def: "OBSOLETE. Interacting selectively and non-covalently with the Cystic Fibrosis Transmembrane conductance Regulator (CFTR) protein." [GOC:jl]
GO:0042988
name: X11-like protein binding
def: "Interacting selectively and non-covalently with X11-like protein, a neuron-specific adaptor protein." [GOC:jl, PMID:12780348]
GO:0043008
name: ATP-dependent protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules) using energy from the hydrolysis of ATP." [GOC:jl]
GO:0043014
name: alpha-tubulin binding
def: "Interacting selectively and non-covalently with the microtubule constituent protein alpha-tubulin." [GOC:jl]
GO:0043015
name: gamma-tubulin binding
def: "Interacting selectively and non-covalently with the microtubule constituent protein gamma-tubulin." [GOC:jl]
GO:0043021
name: ribonucleoprotein complex binding
def: "Interacting selectively and non-covalently with any complex of RNA and protein." [GOC:bf, GOC:go_curators, GOC:vk]
GO:0043022
name: ribosome binding
def: "Interacting selectively and non-covalently with any part of a ribosome." [GOC:go_curators]
GO:0043023
name: ribosomal large subunit binding
def: "Interacting selectively and non-covalently with any part of the larger ribosomal subunit." [GOC:go_curators]
GO:0043024
name: ribosomal small subunit binding
def: "Interacting selectively and non-covalently with any part of the small ribosomal subunit." [GOC:go_curators]
GO:0043027
name: cysteine-type endopeptidase inhibitor activity involved in apoptotic process
def: "Stops, prevents or reduces the activity of a cysteine-type endopeptidase involved in the apoptotic process." [GOC:jl, GOC:mtg_apoptosis, PMID:14744432, Wikipedia:Caspase]
GO:0043028
name: cysteine-type endopeptidase regulator activity involved in apoptotic process
def: "Modulates the activity of a cysteine-type endopeptidase involved in the apoptotic process." [GOC:jl, GOC:mtg_apoptosis, PMID:14744432, Wikipedia:Caspase]
GO:0043035
name: chromatin insulator sequence binding
def: "Interacting selectively and non-covalently and stoichiometrically with a chromatin insulator sequence, a DNA sequence that prevents enhancer-mediated activation or repression of transcription." [GOC:jl, PMID:12783795]
GO:0043047
name: single-stranded telomeric DNA binding
def: "Interacting selectively and non-covalently with single-stranded telomere-associated DNA." [GOC:jl, ISBN:0321000382]
GO:0043110
name: rDNA spacer replication fork barrier binding
def: "Interacting selectively and non-covalently with replication fork barriers found in rDNA spacers, sites that inhibit replication forks in the direction opposite to rDNA transcription." [GOC:jl, GOC:mah, PMID:14645529]
GO:0043115
name: precorrin-2 dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + precorrin-2 = 2 H(+) + NADH + sirohydrochlorin." [EC:1.3.1.76, RHEA:15616]
GO:0043120
name: tumor necrosis factor binding
def: "Interacting selectively and non-covalently with tumor necrosis factor, a proinflammatory cytokine produced by monocytes and macrophages." [GOC:jl, http://lookwayup.com/]
GO:0043121
name: neurotrophin binding
def: "Interacting selectively and non-covalently with a neurotrophin, any of a family of growth factors that prevent apoptosis in neurons and promote nerve growth." [GOC:jl, http://www.mercksource.com/, PR:000021998]
GO:0043125
name: ErbB-3 class receptor binding
def: "Interacting selectively and non-covalently with the protein-tyrosine kinase receptor ErbB-3/HER3." [GOC:jl]
GO:0043130
name: ubiquitin binding
def: "Interacting selectively and non-covalently with ubiquitin, a protein that when covalently bound to other cellular proteins marks them for proteolytic degradation." [GOC:ecd]
GO:0043135
name: 5-phosphoribosyl 1-pyrophosphate pyrophosphatase activity
def: "Catalysis of the reaction: 5-phospho-alpha-D-ribose 1-diphosphate + H2O = ribose 1,5 bisphosphate + phosphate + H+." [MetaCyc:RXN-10969, PMID:12370170]
GO:0043136
name: glycerol-3-phosphatase activity
def: "Catalysis of the reaction: glycerol 3-phosphate + H2O = glycerol + phosphate." [GOC:jl]
GO:0043138
name: 3'-5' DNA helicase activity
def: "Catalysis of the unwinding of the DNA helix in the direction 3' to 5'." [GOC:jl]
GO:0043139
name: 5'-3' DNA helicase activity
def: "Catalysis of the unwinding of the DNA helix in the direction 5' to 3'." [GOC:jl]
GO:0043140
name: ATP-dependent 3'-5' DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; drives the unwinding of the DNA helix in the direction 3' to 5'." [GOC:jl]
GO:0043141
name: ATP-dependent 5'-3' DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; drives the unwinding of the DNA helix in the direction 5' to 3'." [GOC:jl]
GO:0043142
name: single-stranded DNA-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction requires the presence of single-stranded DNA, and it drives another reaction." [GOC:go_curators]
GO:0043167
name: ion binding
def: "Interacting selectively and non-covalently with ions, charged atoms or groups of atoms." [GOC:jl]
GO:0043168
name: anion binding
def: "Interacting selectively and non-covalently with anions, charged atoms or groups of atoms with a net negative charge." [GOC:jl]
GO:0043169
name: cation binding
def: "Interacting selectively and non-covalently with cations, charged atoms or groups of atoms with a net positive charge." [GOC:jl]
GO:0043175
name: RNA polymerase core enzyme binding
def: "Interacting selectively and non-covalently with an RNA polymerase core enzyme, containing a specific subunit composition defined as the core enzyme." [GOC:jl, GOC:txnOH]
GO:0043176
name: amine binding
def: "Interacting selectively and non-covalently with any organic compound that is weakly basic in character and contains an amino or a substituted amino group." [GOC:jl]
GO:0043177
name: organic acid binding
def: "Interacting selectively and non-covalently with an organic acid, any acidic compound containing carbon in covalent linkage." [GOC:jl, ISBN:0198506732]
GO:0043178
name: alcohol binding
def: "Interacting selectively and non-covalently with an alcohol, any of a class of alkyl compounds containing a hydroxyl group." [GOC:jl, ISBN:0198506732]
GO:0043183
name: vascular endothelial growth factor receptor 1 binding
def: "Interacting selectively and non-covalently with vascular endothelial growth factor receptor 1." [GOC:st]
GO:0043184
name: vascular endothelial growth factor receptor 2 binding
def: "Interacting selectively and non-covalently with vascular endothelial growth factor receptor 2." [GOC:st]
GO:0043185
name: vascular endothelial growth factor receptor 3 binding
def: "Interacting selectively and non-covalently with vascular endothelial growth factor receptor 3." [GOC:st]
GO:0043199
name: sulfate binding
def: "Interacting selectively and non-covalently with sulfate, SO4(2-), a negatively charged small molecule." [GOC:mlg]
GO:0043208
name: glycosphingolipid binding
def: "Interacting selectively and non-covalently with glycosphingolipid, a compound with residues of sphingoid and at least one monosaccharide." [GOC:jl]
GO:0043210
name: alkanesulfonate binding
def: "Interacting selectively and non-covalently with alkanesulfonates, the anion of alkanesulfonic acids, sulfonic acid derivatives containing an aliphatic hydrocarbon group." [GOC:mlg]
GO:0043211
name: carbohydrate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate, to directly drive the transport of carbohydrates across a membrane." [GOC:mlg]
GO:0043212
name: carbohydrate-exporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + carbohydrate(in) -> ADP + phosphate + carbohydrate(out)." [GOC:mlg]
GO:0043214
name: bacteriocin-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate; drives the transport of bacteriocins across a membrane." [GOC:mlg]
GO:0043216
name: daunorubicin-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + daunorubicin(in) = ADP + phosphate + daunorubicin(out)." [GOC:mlg]
GO:0043221
name: SMC family protein binding
def: "Interacting selectively and non-covalently with any protein from the structural maintenance of chromosomes (SMC) family, a group of chromosomal ATPases with a role in mitotic chromosome organization." [GOC:jl, GOC:vw, InterPro:IPR024704, PMID:9640531]
GO:0043225
name: anion transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + anion(out) = ADP + phosphate + anion(in)." [GOC:mlg]
GO:0043236
name: laminin binding
def: "Interacting selectively and non-covalently with laminins, glycoproteins that are major constituents of the basement membrane of cells." [GOC:ecd]
GO:0043237
name: laminin-1 binding
def: "Interacting selectively and non-covalently with laminin-1, a glycoprotein trimer with the subunit composition alpha1, beta1, gamma1." [GOC:go_curators]
GO:0043250
name: sodium-dependent organic anion transmembrane transporter activity
def: "Catalysis of the transfer of organic anions from one side of a membrane to the other, in a sodium dependent manner." [GOC:go_curators]
GO:0043262
name: adenosine-diphosphatase activity
def: "Catalysis of the reaction: ADP + H2O = AMP + phosphate." [EC:3.6.1.5, PMID:1470606]
GO:0043273
name: CTPase activity
def: "Catalysis of the reaction: CTP + H2O = CDP + phosphate. May or may not be coupled to another reaction." [GOC:go_curators]
GO:0043274
name: phospholipase binding
def: "Interacting selectively and non-covalently with any phospholipase, enzymes that catalyze of the hydrolysis of a glycerophospholipid." [GOC:jl]
GO:0043275
name: glutamate carboxypeptidase II activity
def: "OBSOLETE. Catalysis of the reaction: N-acetyl-L-Asp-L-Glu + H2O = N-acetyl-L-Asp + L-Glu." [BRENDA:3.4.17.21, GOC:jl]
GO:0043287
name: poly(3-hydroxyalkanoate) binding
def: "Interacting selectively and non-covalently with poly(3-hydroxyalkanoate)s, polyesters of 3-hydroxyacids produced as intracellular granules by a large variety of bacteria." [GOC:jl]
GO:0043295
name: glutathione binding
def: "Interacting selectively and non-covalently with glutathione; a tripeptide composed of the three amino acids cysteine, glutamic acid and glycine." [GOC:bf, ISBN:0198506732]
GO:0043325
name: phosphatidylinositol-3,4-bisphosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-3,4-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' and 4' positions." [GOC:bf, GOC:go_curators]
GO:0043333
name: 2-octaprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-octaprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:kd, PMID:9045837]
GO:0043334
name: 2-hexaprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-hexaprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-hexaprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:kd, PMID:9083048]
GO:0043336
name: site-specific telomere resolvase activity
def: "Catalysis of a site-specific breakage and reunion reaction that generates two hairpin telomeres from a replicated telomere substrate. Occurs via a two-step transesterification with a protein-DNA intermediate similar to that used by topoisomerases and site-specific recombinases." [GOC:jl, PMID:11804598]
GO:0043337
name: CDP-diacylglycerol-phosphatidylglycerol phosphatidyltransferase activity
def: "Catalysis of the reaction: CDP-diacylglycerol + phosphatidylglycerol = CMP + diphosphatidylglycerol." [GOC:jl]
GO:0043338
name: CTP:2,3-di-O-geranylgeranyl-sn-glycero-1-phosphate cytidyltransferase activity
def: "Catalysis of the reaction: CTP + 2,3-di-O-geranylgeranyl-sn-glycero-1-phosphate = CDP-2,3-di-O-geranylgeranyl-sn-glycerol." [GOC:jl, PMID:10960477]
GO:0043364
name: catalysis of free radical formation
def: "Catalysis of a reaction that generates a free radical, a highly reactive molecule with an unsatisfied electron valence pair." [GOC:jl]
GO:0043365
name: [formate-C-acetyltransferase]-activating enzyme activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + dihydroflavodoxin + [formate C-acetyltransferase]-glycine = 5'-deoxyadenosine + L-methionine + flavodoxin semiquinone + [formate C-acetyltransferase]-glycin-2-yl radical." [EC:1.97.1.4, GOC:jl, PMID:18307109]
GO:0043394
name: proteoglycan binding
def: "Interacting selectively and non-covalently with a proteoglycan, any glycoprotein in which the carbohydrate units are glycosaminoglycans." [ISBN:0198506732]
GO:0043395
name: heparan sulfate proteoglycan binding
def: "Interacting selectively and non-covalently with a heparan sulfate proteoglycan, any proteoglycan containing heparan sulfate as the glycosaminoglycan carbohydrate unit." [ISBN:0198506732]
GO:0043398
name: HLH domain binding
def: "Interacting selectively and non-covalently with Helix Loop Helix, a domain of 40-50 residues that occurs in specific DNA-binding proteins that act as transcription factors. The domain is formed of two amphipathic helices joined by a variable length linker region that can form a loop and it mediates protein dimerization." [GOC:go_curators, Prosite:PDOC0038]
GO:0043399
name: tRNA A64-2'-O-ribosylphosphate transferase activity
def: "Catalysis of the transfer of a phosphoribosyl group from 5'-phosphoribosyl-1'-pyrophosphate to position 64 of initiator tRNA." [GOC:jl, PMID:7954819]
GO:0043404
name: corticotropin-releasing hormone receptor activity
def: "Combining with corticotropin-releasing hormone and transmitting the signal to initiate a change in cell activity." [GOC:signaling, ISBN:0838577016, PMID:11027914, PMID:15134857]
GO:0043411
name: myopalladin binding
def: "OBSOLETE. Interacting selectively and non-covalently with myopalladin, a myofibrillar protein with titin-like Ig domains." [GOC:go_curators]
GO:0043422
name: protein kinase B binding
def: "Interacting selectively and non-covalently with protein kinase B, an intracellular kinase that is important in regulating glucose metabolism." [GOC:jl, http://www.heartandmetabolism.org/]
GO:0043423
name: 3-phosphoinositide-dependent protein kinase binding
def: "Interacting selectively and non-covalently with a 3-phosphoinositide-dependent protein kinase." [GOC:jl]
GO:0043424
name: protein histidine kinase binding
def: "Interacting selectively and non-covalently with protein histidine kinase." [GOC:jl]
GO:0043425
name: bHLH transcription factor binding
def: "Interacting selectively and non-covalently with any of the basic Helix-Loop-Helix (bHLH) superfamily of transcription factors, important regulatory components in transcriptional networks of many developmental pathways." [PMID:9144210]
GO:0043426
name: MRF binding
def: "Interacting selectively and non-covalently with Myogenic Regulatory Factor (MRF), a member of the basic Helix-Loop-Helix (bHLH) superfamily of transcription factors." [PMID:10966875]
GO:0043428
name: 2-heptaprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-heptaprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-heptaprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:jl, PMID:11583838]
GO:0043429
name: 2-nonaprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-nonaprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-nonaprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:jl, PMID:11583838]
GO:0043430
name: 2-decaprenyl-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-decaprenyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-methionine = 2-decaprenyl-3-methyl-6-methoxy-1,4-benzoquinone + S-adenosyl-L-homocysteine." [GOC:jl, PMID:11583838]
GO:0043431
name: 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol + S-adenosyl-L-methionine = ubiquinol + S-adenosyl-L-homocysteine." [GOC:jl, PMID:11583838, PMID:1479344]
GO:0043472
name: IgD binding
def: "Interacting selectively and non-covalently with an immunoglobulin of a D isotype." [PMID:12886015]
GO:0043492
name: ATPase activity, coupled to movement of substances
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, to directly drive the transport of a substance." [GOC:jl]
GO:0043495
name: protein anchor
def: "Interacting selectively and non-covalently with both a protein or protein complex and a membrane, in order to maintain the localization of the protein at a specific location on the membrane." [GOC:go_curators]
GO:0043498
name: cell surface binding
def: "OBSOLETE. Interacting selectively and non-covalently with any component on the surface of a cell." [GOC:jl]
GO:0043499
name: eukaryotic cell surface binding
def: "OBSOLETE. Interacting selectively and non-covalently with any component on the surface of a eukaryotic cell." [GOC:jl]
GO:0043515
name: kinetochore binding
def: "Interacting selectively and non-covalently with a kinetochore, a proteinaceous structure on a condensed chromosome, beside the centromere, to which the spindle fibers are attached." [GOC:jl, http://www.mercksource.com/]
GO:0043522
name: leucine zipper domain binding
def: "Interacting selectively and non-covalently with a leucine zipper domain, a protein secondary structure exhibiting a periodic repetition of leucine residues at every seventh position over a distance covering eight helical turns." [GOC:jl, InterPro:IPR002158]
GO:0043530
name: adenosine 5'-monophosphoramidase activity
def: "Catalysis of the reaction: adenosine 5'-monophosphoramidate = AMP + NH2. Other substrates include AMP-morpholidate, AMP-N-alanine methyl ester and AMP-alpha-acetyl lysine methyl ester." [PMID:11805111]
GO:0043531
name: ADP binding
def: "Interacting selectively and non-covalently with ADP, adenosine 5'-diphosphate." [GOC:jl]
GO:0043532
name: angiostatin binding
def: "Interacting selectively and non-covalently with angiostatin, a proteolytic product of plasminogen or plasmin containing at least one intact kringle domain, and which is an inhibitor of angiogenesis." [PMID:16043488]
GO:0043533
name: inositol 1,3,4,5 tetrakisphosphate binding
def: "Interacting selectively and non-covalently with inositol 1,3,4,5 tetrakisphosphate." [GOC:go_curators]
GO:0043539
name: protein serine/threonine kinase activator activity
def: "Increases the activity of a protein serine/threonine kinase." [GOC:go_curators]
GO:0043544
name: lipoamide binding
def: "Interacting selectively and non-covalently with lipoamide, the functional form of lipoic acid in which the carboxyl group is attached to protein by an amide linkage to a lysine amino group." [GOC:go_curators]
GO:0043546
name: molybdopterin cofactor binding
def: "Interacting selectively and non-covalently with the molybdopterin cofactor (Moco), essential for the catalytic activity of some enzymes, e.g. sulfite oxidase, xanthine dehydrogenase, and aldehyde oxidase. The cofactor consists of a mononuclear molybdenum (Mo-molybdopterin) or tungsten ion (W-molybdopterin) coordinated by one or two molybdopterin ligands." [ISSN:09498257]
GO:0043548
name: phosphatidylinositol 3-kinase binding
def: "Interacting selectively and non-covalently with a phosphatidylinositol 3-kinase, any enzyme that catalyzes the addition of a phosphate group to an inositol lipid at the 3' position of the inositol ring." [PMID:10209156, PMID:9255069]
GO:0043559
name: insulin binding
def: "Interacting selectively and non-covalently with insulin, a polypeptide hormone produced by the islets of Langerhans of the pancreas in mammals, and by the homologous organs of other organisms." [ISBN:0198506732]
GO:0043560
name: insulin receptor substrate binding
def: "Interacting selectively and non-covalently with any of the insulin receptor substrate (IRS) proteins, adaptor proteins that bind to the transphosphorylated insulin and insulin-like growth factor receptors, are themselves phosphorylated and in turn recruit SH2 domain-containing signaling molecules to form a productive signaling complex." [PMID:12829233]
GO:0043563
name: odorant transporter activity
def: "Enables the directed movement of odorants, any substance capable of stimulating the sense of smell, into, out of or within a cell, or between cells." [GOC:jl]
GO:0043565
name: sequence-specific DNA binding
def: "Interacting selectively and non-covalently with DNA of a specific nucleotide composition, e.g. GC-rich DNA binding, or with a specific sequence motif or type of DNA e.g. promotor binding or rDNA binding." [GOC:jl]
GO:0043566
name: structure-specific DNA binding
def: "Interacting selectively and non-covalently with DNA of a specific structure or configuration e.g. triplex DNA binding or bent DNA binding." [GOC:jl]
GO:0043621
name: protein self-association
def: "Interacting selectively and non-covalently with a domain within the same polypeptide." [GOC:jl]
GO:0043682
name: copper-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Cu2+ = ADP + phosphate + Cu2+, directly driving the transport of the copper ions across a membrane." [GOC:jl]
GO:0043712
name: 2-hydroxyisocaproate CoA-transferase activity
def: "Catalysis of the reaction: (R)-2-hydroxyisocaproate + isocaproyl-CoA = (R)-2-hydroxyisocaproyl-CoA + isocaproate." [PMID:16957230]
GO:0043713
name: (R)-2-hydroxyisocaproate dehydrogenase activity
def: "Catalysis of the reaction: 2-oxoisocaproate + NADH + H+ = (R)-2-hydroxyisocaproate + NAD+." [GOC:jl, PMID:16957230]
GO:0043714
name: (R)-citramalate synthase activity
def: "Catalysis of the reaction: pyruvate + acetyl-CoA + H2O = (R)-citramalate + CoA." [GOC:jl, PMID:9864346]
GO:0043715
name: 2,3-diketo-5-methylthiopentyl-1-phosphate enolase activity
def: "Catalysis of the reaction: 2,3-diketo-5-methylthiopentyl-1-phosphate = H+ + 2-hydroxy 3-keto-5-methylthiopentenyl-1-phosphate. 2,3-diketo-5-methylthiopentyl-1-phosphate is also known as DK-MTP-1-P, and 2-hydroxy 3-keto-5-methylthiopentenyl-1-phosphate as HK-MTPenyl-1-P." [MetaCyc:R82-RXN]
GO:0043716
name: 2-hydroxy-3-keto-5-methylthiopentenyl-1-phosphate phosphatase activity
def: "Catalysis of the reaction: 2-hydroxy-3-keto-5-methylthiopentenyl-1-phosphate + H2O = 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + phosphate. 2-hydroxy-3-keto-5-methylthiopentenyl-1-phosphate is also known as HK-MTPenyl-P, and 1,2-dihydroxy-3-keto-5-methylthiopentene as DHK-MTPene." [EC:3.1.3.77, IMG:01681, MetaCyc:R83-RXN]
GO:0043717
name: 2-hydroxyglutaryl-CoA dehydratase activity
def: "Catalysis of the reaction: (R)-2-hydroxyglutaryl-CoA = H2O + glutaconyl-CoA." [MetaCyc:RXN-1083]
GO:0043718
name: 2-hydroxymethylglutarate dehydrogenase activity
def: "Catalysis of the reaction: 2-(hydroxymethyl)glutarate + NAD(+) = 2-formylglutarate + H(+) + NADH." [EC:1.1.1.291, RHEA:15508]
GO:0043719
name: 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol hydroxylase activity
def: "Catalysis of the reaction: 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol + O2 + H+ = 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol + H2O." [GOC:jl, MetaCyc:OCTAPRENYL-METHYL-METHOXY-BENZOQ-OH-RXN]
GO:0043720
name: 3-keto-5-aminohexanoate cleavage activity
def: "Catalysis of the reaction: 3-keto-5-aminohexanoate + acetyl-CoA = L-3-aminobutyryl-CoA + acetoacetate." [MetaCyc:R125-RXN, PMID:13064]
GO:0043721
name: 4-hydroxybutanoyl-CoA dehydratase activity
def: "Catalysis of the reaction: 4-hydroxybutanoyl-CoA = vinylacetyl-CoA + H2O." [EC:4.2.1.120]
GO:0043722
name: 4-hydroxyphenylacetate decarboxylase activity
def: "Catalysis of the reaction: (4-hydroxyphenyl)acetate + H(+) = 4-cresol + CO(2)." [EC:4.1.1.83, RHEA:22735]
GO:0043723
name: 2,5-diamino-6-ribitylamino-4(3H)-pyrimidinone 5'-phosphate deaminase activity
def: "Catalysis of the reaction: 2,5-diamino-6-(5-phospho-D-ribitylamino)pyrimidin-4(3H)-one + H2O = 5-amino-6-(5-phospho-D-ribosylamino)uracil + ammonia." [MetaCyc:RXN-10058, PMID:11889103]
GO:0043724
name: 2-keto-3-deoxygalactonate aldolase activity
def: "Catalysis of the reaction: 2-keto-3-deoxygalactonate = D-glyceraldehyde + pyruvate." [PMID:12824170]
GO:0043725
name: 2-keto-3-deoxygluconate aldolase activity
def: "Catalysis of the reaction: 2-keto-3-deoxygluconate = D-glyceraldehyde + pyruvate." [PMID:12824170]
GO:0043726
name: 5-amino-6-(5-phosphoribitylamino)uracil phosphatase activity
def: "Catalysis of the reaction: 5-amino-6-(5-phosphoribitylamino)-2,4(1H,3H)-pyrimidinedione + H2O = 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione + orthophosphate." [GOC:jl, IMG:01282]
GO:0043727
name: 5-amino-4-imidazole carboxylate lyase activity
def: "Catalysis of the reaction: 5-aminoimidazole + CO2 = 5-amino-4-imidazole carboxylate." [GOC:jl, IMG:01835]
GO:0043728
name: 2-keto-4-methylthiobutyrate aminotransferase activity
def: "Catalysis of the reaction: 2-keto-4-methylthiobutyrate + L-glutamine = 2-oxoglutaramate + L-methionine." [GOC:jl, IMG:01683]
GO:0043729
name: 2-amino-5-formylamino-6-(5-phosphoribosylamino)pyrimidin-4(3H)-one formate-lyase activity
def: "Catalysis of the reaction: 2-amino-5-formylamino-6-(5-phosphoribosylamino)pyrimidin-4(3H)-one + H2O = 2,5-diamino-6-(5-phosphoribosylamino)pyrimidin-4(3H)-one + formate." [GOC:jl, IMG:01536]
GO:0043730
name: 5-ureido-4-imidazole carboxylate hydrolase activity
def: "Catalysis of the reaction: 5-ureido-4-imidazole carboxylate + H2O = 5-amino-4-imidazole carboxylate + NH3 + CO2." [GOC:jl, IMG:02121]
GO:0043731
name: 6-hydroxynicotinate 3-monooxygenase activity
def: "Catalysis of the oxidative decarboxylation of 6-hydroxynicotinate to 2,5-dihydroxypyridine, dependent on O2, NADH +H+ and FAD." [GOC:jl, PMID:10091591]
GO:0043732
name: 6-hydroxynicotinate dehydrogenase activity
def: "Catalysis of the reaction: 6-hydroxynicotinate + H(2)O + O(2) = 2,6-dihydroxynicotinate + H(2)O(2)." [EC:1.17.3.3, RHEA:22811]
GO:0043733
name: DNA-3-methylbase glycosylase activity
def: "Catalysis of the reaction: DNA containing 3-methylbase + H2O = DNA with abasic site + 3-methylbase. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the damaged DNA 3-methylpurine or 3-methylpyrimidine base and the deoxyribose sugar to remove the methylated base, leaving an apurinic or apyrimidinic site." [IMG:00576, PMID:10777493, PMID:14517230]
GO:0043734
name: DNA-N1-methyladenine dioxygenase activity
def: "Catalysis of the oxidative demethylation of N1-methyladenine and N3-methylcytosine in DNA and RNA, with concomitant decarboxylation of 2-oxoglutarate and releases oxidized methyl group on N1-methyladenine and N3-methylcytosine as formaldehyde." [IMG:00579, PMID:19786499]
GO:0043736
name: DNA-3-methyladenine glycosylase IV activity
def: "OBSOLETE. Catalysis of the hydrolysis of alkylated DNA; recognizes and removes both N-3- and N-7-methyl purines by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic (AP) site." [IMG:00578, PMID:10375529]
GO:0043737
name: deoxyribonuclease V activity
def: "Catalysis of the endonucleolytic cleavage at apurinic or apyrimidinic sites to products with a 5'-phosphate." [EC:3.1.21.7]
GO:0043738
name: reduced coenzyme F420 dehydrogenase activity
def: "Catalysis of the reaction: methanophenazine + reduced coenzyme F420 = dihydromethanophenazine + coenzyme F420." [MetaCyc:RXN-8106]
GO:0043739
name: G/U mismatch-specific uracil-DNA glycosylase activity
def: "Catalysis of the removal of uracil from a U*G mispair by the cleavage the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free uracil and leaves an apyrimidinic (AP) site." [IMG:00570, SP:O59825]
GO:0043740
name: GTP cyclohydrolase IIa activity
def: "Catalysis of the reaction: GTP + 3 H(2)O = 2-amino-5-formylamino-6-(1-D-ribosylamino)pyrimidin-4(3H)-one 5'-phosphate + 3 H(+) + 2 phosphate." [EC:3.5.4.29, RHEA:22471]
GO:0043741
name: L-2-aminoadipate N-acetyltransferase activity
def: "Catalysis of the reaction: L-2-aminoadipate + acetyl-CoA = N2-acetyl-L-aminoadipate + CoA." [IMG:01447]
GO:0043743
name: LPPG:FO 2-phospho-L-lactate transferase activity
def: "Catalysis of the reaction: 7,8-didemethyl-8-hydroxy-5-deazariboflavin + lactyl-2-diphospho-5'-guanosine = coenzyme F420-0 + GMP." [IMG:01551]
GO:0043744
name: N2-acetyl-L-aminoadipate kinase activity
def: "Catalysis of the reaction: ATP + N-acetyl-L-aminoadipate = ADP + N-acetyl-L-aminoadipate 5-phosphate." [IMG:01448]
GO:0043745
name: N2-acetyl-L-aminoadipate semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: N2-acetyl-L-aminoadipyl-delta-phosphate + NADPH + H+ = N2-acetyl-L-aminoadipate semialdehyde + NADP+ + orthophosphate." [IMG:01449]
GO:0043746
name: N2-acetyl-L-lysine aminotransferase activity
def: "Catalysis of the reaction: N2-acetyl-L-aminoadipate semialdehyde + L-glutamate = 2-oxoglutarate + N2-acetyl-L-lysine." [IMG:01450]
GO:0043747
name: N2-acetyl-L-lysine deacetylase activity
def: "Catalysis of the reaction: N2-acetyl-L-lysine + H2O = acetate + L-lysine." [IMG:01451]
GO:0043748
name: O-succinylbenzoate synthase activity
def: "Catalysis of the reaction: 2-succinylbenzoate + H2O = 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate." [IMG:01733]
GO:0043749
name: phenol, water dikinase activity
def: "Catalysis of the reaction: phenol + MgATP + H2O = phenylphosphate + MgAMP + orthophosphate." [PMID:15547277]
GO:0043750
name: phosphatidylinositol alpha-mannosyltransferase activity
def: "Catalysis of the transfer of one or more alpha-D-mannose residues from GDP-mannose to positions 2,6 and others in 1-phosphatidyl-myo-inositol." [EC:2.4.1.57]
GO:0043751
name: polyphosphate:AMP phosphotransferase activity
def: "Catalysis of the reaction: (polyphosphate)n + AMP = (polyphosphate)n-1 + ADP." [PMID:11237733]
GO:0043752
name: adenosylcobinamide kinase activity
def: "Catalysis of the reaction: RTP + adenosylcobinamide = adenosylcobinamide phosphate + RDP (where RTP is either ATP or GTP)." [EC:2.7.1.156]
GO:0043754
name: dihydrolipoyllysine-residue (2-methylpropanoyl)transferase activity
def: "Catalysis of the reaction: 2-methylpropanoyl-CoA + enzyme N6-(dihydrolipoyl)lysine = CoA + enzyme N6-(S-[2-methylpropanoyl]dihydrolipoyl)lysine." [EC:2.3.1.168]
GO:0043755
name: alpha-ribazole phosphatase activity
def: "Catalysis of the reaction: alpha-ribazole 5'-phosphate + H(2)O = alpha-ribazole + phosphate." [EC:3.1.3.73, RHEA:24459]
GO:0043756
name: adenosylcobinamide hydrolase activity
def: "Catalysis of the reaction: adenosylcobinamide + H(2)O = (R)-1-aminopropan-2-ol + adenosylcobyrate." [EC:3.5.1.90, RHEA:23507]
GO:0043757
name: adenosylcobinamide-phosphate synthase activity
def: "Catalysis of the reactions: ATP + adenosylcobyric acid + (R)-1-aminopropan-2-yl phosphate = ADP + phosphate + adenosylcobinamide phosphate, and ATP + adenosylcobyric acid + (R)-1-aminopropan-2-ol = ADP + phosphate + adenosylcobinamide." [EC:6.3.1.10]
GO:0043758
name: acetate-CoA ligase (ADP-forming) activity
def: "Catalysis of the reaction: ATP + acetate + CoA = ADP + phosphate + acetyl-CoA." [EC:6.2.1.13]
GO:0043759
name: branched-chain acyl-CoA synthetase (ADP-forming) activity
def: "Catalysis of the reaction: ATP + 2-methylbutanoate + CoA = ADP + orthophosphate + 2-methylbutanoyl-CoA." [IMG:01425]
GO:0043760
name: acetyldiaminopimelate aminotransferase activity
def: "Catalysis of the reaction: N-acetyl-L-2,6-diaminoheptanedioate + 2-oxoglutarate = N-acetyl-2-L-amino-6-oxoheptanedioate + L-glutamate." [IMG:00444, PMID:1906065]
GO:0043761
name: archaetidylserine synthase activity
def: "Catalysis of the reaction: CDP-digeranylgeranylglycerol + L-serine = archaetidylserine + CMP." [IMG:01567, PMID:12562787]
GO:0043762
name: aryl-CoA synthetase (ADP-forming) activity
def: "Catalysis of the reaction: ATP + phenylacetate + CoA = ADP + orthophosphate + phenylacetyl-CoA." [IMG:01427]
GO:0043763
name: UTP:glucose-1-phosphate uridylyltransferase regulator activity
def: "Modulates the activity of UTP:glucose-1-phosphate uridylyltransferase." [GOC:jl]
GO:0043764
name: UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase activity
def: "Catalysis of the reaction: UDP-3-O-(3-hydroxytetradecanoyl)glucosamine + (R)-3-hydroxytetradecanoyl-[acyl-carrier protein] = UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine + [acyl-carrier protein]." [IMG:00545, InterPro:IPR007691]
GO:0043765
name: T/G mismatch-specific endonuclease activity
def: "Catalysis of the repair of T/G mismatches arising from deamination of 5-methylcytosine in DNA by nicking double-stranded DNA within the sequence CT(AT)GN or NT(AT)GG next to the mismatched thymidine residue. The incision is mismatch-dependent and strand-specific, in favor of the G-containing strand. The incision serves as a starting point for subsequent excision repair by DNA polymerase I, which excises thymidine and reinserts cytidine." [IMG:00606, SP:P09184]
GO:0043766
name: Sep-tRNA:Cys-tRNA synthase activity
def: "Catalysis of the reaction: O-phospho-L-seryl-tRNACys + sulfide = L-cysteinyl-tRNACys + phosphate." [EC:2.5.1.73, IMG:00235, PMID:15790858, PMID:16380427, PMID:17110438]
GO:0043767
name: pyrrolysyl-tRNA synthetase activity
def: "Catalysis of the reaction: ATP + L-pyrrolysine + tRNA(Pyl) = AMP + diphosphate + L-pyrrolysyl-tRNA(Pyl)." [IMG:02553, PMID:15314242, SP:Q8TUB8]
GO:0043768
name: S-ribosylhomocysteine lyase activity
def: "Catalysis of the reaction: S-(5-deoxy-D-ribos-5-yl)-L-homocysteine = (S)-4,5-dihydroxypentane-2,3-dione + L-homocysteine." [EC:4.4.1.21, RHEA:17756]
GO:0043770
name: demethylmenaquinone methyltransferase activity
def: "Catalysis of the reaction: 2-demethylmenaquinone + S-adenosyl-L-methionine = menaquinone + S-adenosyl-L-homocysteine." [IMG:01848]
GO:0043771
name: cytidine kinase activity
def: "Catalysis of the reaction: ATP + cytidine = ADP + CMP." [IMG:01870]
GO:0043772
name: acyl-phosphate glycerol-3-phosphate acyltransferase activity
def: "Catalysis of the reaction: acyl phosphate + sn-glycerol 3-phosphate = 1-acyl-sn-glycerol 3-phosphate + orthophosphate." [IMG:01872, PMID:17308305]
GO:0043773
name: coenzyme F420-0 gamma-glutamyl ligase activity
def: "Catalysis of the reactions: (1) GTP + F420-0 + L-glutamate = GDP + phosphate + F420-1, and (2) GTP + F420-1 + L-glutamate = GDP + phosphate + gamma-F420-2. This is the GTP-dependent successive addition of two L-glutamates to the L-lactyl phosphodiester of 7,8-didemethyl-8-hydroxy-5-deazariboflavin (F420-0) to form F420-0-glutamyl-glutamate (F420-2), with a gamma-linkage between the two glutamates." [IMG:01552, PMID:17669425]
GO:0043774
name: coenzyme F420-2 alpha-glutamyl ligase activity
def: "Catalysis of the reaction: coenzyme F420-2 + L-glutamate + GTP = coenzyme F420-3 + GDP + orthophosphate." [IMG:01553]
GO:0043775
name: cobyrinate a,c-diamide synthase activity
def: "Catalysis of the reaction: cobyrinate + 2 L-glutamine + 2 ATP + 2 H2O = cob(II)yrinate a,c-diamide + 2 L-glutamate + 2 ADP + 2 orthophosphate." [IMG:01814]
GO:0043776
name: cobalt-precorrin-6B C5-methyltransferase activity
def: "Catalysis of the reaction: S-adenosylmethionine + cobalt-precorrin 6B = S-adenosylhomocysteine + cobalt-precorrin 7." [IMG:02115, MetaCyc:RXN-8766]
GO:0043777
name: cobalt-precorrin-7 C15-methyltransferase activity
def: "Catalysis of the reaction: cobalt-precorrin 7 + S-adenosyl-L-methionine = cobalt-precorrin 8 + S-adenosyl-L-homocysteine + CO2." [IMG:02116, MetaCyc:RXN-8767]
GO:0043778
name: cobalt-precorrin-8 methylmutase activity
def: "Catalysis of the reaction: cobalt-precorrin 8 = cobyrinate." [IMG:02117, MetaCyc:RXN-8768]
GO:0043779
name: cobalt-precorrin-5A acetaldehyde-lyase activity
def: "Catalysis of the reaction: cobalt-precorrin 5A + H2O = cobalt-precorrin 5B + acetaldehyde." [IMG:02112, MetaCyc:RXN-8763]
GO:0043780
name: cobalt-precorrin-5B C1-methyltransferase activity
def: "Catalysis of the reaction: cobalt-precorrin 5B + S-adenosylmethionine = S-adenosylhomocysteine + cobalt-precorrin 6A." [IMG:02112, MetaCyc:RXN-8764]
GO:0043781
name: cobalt-factor II C20-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + cobalt-factor II = S-adenosyl-L-homocysteine + cobalt-factor III." [EC:2.1.1.151, IMG:01812]
GO:0043782
name: cobalt-precorrin-3 C17-methyltransferase activity
def: "Catalysis of the reaction: cobalt-precorrin 3 + S-adenosyl-L-methionine = cobalt-precorrin 4 + S-adenosyl-L-homocysteine." [IMG:02110, MetaCyc:RXN-8761]
GO:0043783
name: oxidoreductase activity, oxidizing metal ions with flavin as acceptor
def: "Catalysis of an oxidation-reduction in which the oxidation state of metal ion is altered and flavin acts as an electron acceptor." [EC:1.16.8]
GO:0043784
name: cob(II)yrinic acid a,c-diamide reductase activity
def: "Catalysis of the reaction: 2 cob(I)yrinate a,c diamide + FMN + 3 H(+) = 2 cob(II)yrinate a,c diamide + FMNH(2)." [EC:1.16.8.1, RHEA:24303]
GO:0043785
name: cinnamoyl-CoA:phenyllactate CoA-transferase activity
def: "Catalysis of the reaction: (R)-3-phenyllactate + [(2R,3S,4R,5R)-5-(6-amino-9H-purin-9-yl)-4-hydroxy-3-(phosphonatooxy)oxolan-2-yl]methyl {[(3R)-3-hydroxy-2,2-dimethyl-3-({2-[(2-{[(2E)-3-phenylprop-2-enoyl]sulfanyl}ethyl)carbamoyl]ethyl}carbamoyl)propyl phosphonato]oxy}phosphonate = (R)-3-phenyllactoyl-CoA + trans-cinnamate." [EC:2.8.3.17, RHEA:15604]
GO:0043786
name: cinnamate reductase activity
def: "Catalysis of the reaction: 3-phenylpropanoate + NAD+ = (E)-cinnamate + NADH + H+." [EC:1.3.1.-, IMG:01615, PMID:10849007]
GO:0043791
name: dimethylamine methyltransferase activity
def: "Catalysis of the reaction: dimethylamine + a dimethylamine corrinoid protein = a methylated dimethylamine corrinoid protein + methylamine." [IMG:01321, MetaCyc:RXN-8100, PMID:9874228]
GO:0043792
name: enamidase activity
def: "Catalysis of the reaction: 1,4,5,6-tetrahydro-6-oxonicotinate + 2 H(2)O = 2-formylglutarate + NH(4)(+)." [EC:3.5.2.18, RHEA:17212]
GO:0043793
name: beta-ribofuranosylaminobenzene 5'-phosphate synthase activity
def: "Catalysis of the reaction: 4-aminobenzoate + 5-phospho-alpha-D-ribose 1-diphosphate = 4-(beta-D-ribofuranosyl)aminobenzene 5'-phosphate + CO2 + diphosphate." [GOC:jl, IMG:01544, PMID:12142414]
GO:0043794
name: formate dehydrogenase (coenzyme F420) activity
def: "Catalysis of the reaction: formate + coenzyme F420 = CO2 + reduced coenzyme F420." [IMG:01432, PMID:3801411]
GO:0043795
name: glyceraldehyde oxidoreductase activity
def: "Catalysis of the reaction: D-glyceraldehyde + H2O + acceptor = D-glycerate + reduced acceptor." [IMG:01506]
GO:0043796
name: glyceraldehyde dehydrogenase (NADP) activity
def: "Catalysis of the reaction: D-glyceraldehyde + H2O + NADP+ = D-glycerate + NADPH + H+." [IMG:01555]
GO:0043797
name: glyceraldehyde-3-phosphate dehydrogenase (ferredoxin) activity
def: "Catalysis of the reaction: D-glyceraldehyde-3-phosphate + H2O + 2 oxidized ferredoxin = 3-phospho-D-glycerate + 2 H+ + 2 reduced ferredoxin." [EC:1.2.7.6]
GO:0043798
name: glycerate 2-kinase activity
def: "Catalysis of the reaction: D-glycerate + ATP = 2-phospho-D-glycerate + ADP." [IMG:01510, MetaCyc:GKI-RXN, PMID:14413719]
GO:0043799
name: glycine oxidase activity
def: "Catalysis of the reactions: (1) glycine + H2O + O2 = glyoxylate + NH3 + hydrogen peroxide; (2) D-alanine + H2O + O2 = pyruvate + NH3 + hydrogen peroxide; (3) sarcosine + H2O + O2 = glyoxylate + methylamine + hydrogen peroxide; (4) N-ethylglycine + H2O + O2 = glyoxylate + ethylamine + hydrogen peroxide." [EC:1.4.3.19]
GO:0043800
name: hexulose-6-phosphate isomerase activity
def: "Catalysis of the reaction: D-arabino-3-hexulose 6-phosphate = D-fructose 6-phosphate." [IMG:01502, MetaCyc:R12-RXN, PMID:11839305]
GO:0043801
name: hexulose-6-phosphate synthase activity
def: "Catalysis of the reaction: D-ribulose 5-phosphate + formaldehyde = D-arabino-3-hexulose 6-phosphate." [IMG:01503, PMID:16075199]
GO:0043802
name: hydrogenobyrinic acid a,c-diamide synthase (glutamine-hydrolysing) activity
def: "Catalysis of the reaction: 2 L-glutamine + 2 ATP + 2 H(2)O + hydrogenobyrinate = 2 L-glutamate + 2 ADP + 4 H(+) + hydrogenobyrinate a,c-diamide + 2 phosphate." [EC:6.3.5.9, RHEA:12547]
GO:0043803
name: hydroxyneurosporene-O-methyltransferase activity
def: "Catalysis of the reaction: demethylspheroidene + S-adenosyl-L-methionine = spheroidene + S-adenosyl-L-homocysteine." [IMG:01759]
GO:0043804
name: imidazolone hydrolase activity
def: "Catalysis of the reaction: N-formiminoglycine = imidazolone + H2O." [IMG:02122]
GO:0043805
name: indolepyruvate ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: (indol-3-yl)pyruvate + CoA + oxidized ferredoxin = S-2-(indol-3-yl)acetyl-CoA + CO2 + reduced ferredoxin." [EC:1.2.7.8]
GO:0043806
name: keto acid formate lyase activity
def: "Catalysis of the reaction: 2-oxobutanoate + CoA = propionyl-CoA + formate." [IMG:00725, MetaCyc:KETOBUTFORMLY-RXN]
GO:0043807
name: 3-methyl-2-oxobutanoate dehydrogenase (ferredoxin) activity
def: "Catalysis of the reaction: 3-methyl-2-oxobutanoate + CoA + oxidized ferredoxin = S-(2-methylpropanoyl)-CoA + CO2 + reduced ferredoxin." [EC:1.2.7.7]
GO:0043808
name: lyso-ornithine lipid acyltransferase activity
def: "Catalysis of the reaction: lyso-ornithine lipid + acyl-[acyl-carrier protein] = ornithine lipid + [acyl-carrier protein]." [IMG:02097, PMID:15341653]
GO:0043810
name: ornithine-acyl [acyl carrier protein] N-acyltransferase activity
def: "Catalysis of the reaction: (3R)-3-hydroxyacyl-[acyl-carrier protein] + L-ornithine = lyso-ornithine lipid + [acyl-carrier protein]." [IMG:02096]
GO:0043811
name: phosphate:acyl-[acyl carrier protein] acyltransferase activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + orthophosphate = acyl phosphate + [acyl-carrier protein]." [IMG:01871]
GO:0043812
name: phosphatidylinositol-4-phosphate phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol-4-phosphate + H2O = phosphatidylinositol + orthophosphate." [IMG:01360]
GO:0043813
name: phosphatidylinositol-3,5-bisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol-3,5-bisphosphate + H2O = phosphatidylinositol-3-phosphate + orthophosphate." [IMG:01359, PMID:10806194, PMID:16607019]
GO:0043814
name: phospholactate guanylyltransferase activity
def: "Catalysis of the reaction: 2-phospho-(S)-lactate + GTP = lactyl-2-diphospho-5'-guanosine + diphosphate." [IMG:01547]
GO:0043815
name: phosphoribosylglycinamide formyltransferase 2 activity
def: "Catalysis of the reaction: formate + ATP + 5'-phospho-ribosylglycinamide = 5'-phosphoribosyl-N-formylglycinamide + ADP + diphosphate." [IMG:01297, PMID:8117714]
GO:0043816
name: phosphoserine-tRNA(Cys) ligase activity
def: "Catalysis of the reaction: tRNA(Cys) + O-phospho-L-serine + ATP = AMP + diphosphate + phosphoseryl-tRNA(Cys)." [IMG:00622, PMID:17110438]
GO:0043817
name: phosphosulfolactate synthase activity
def: "Catalysis of the reaction: (2R)-O-phospho-3-sulfolactate = phosphoenolpyruvate + sulfite." [EC:4.4.1.19, RHEA:22787]
GO:0043818
name: precorrin-3B synthase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + precorrin-3A = H(2)O + NAD(+) + precorrin-3B." [EC:1.14.13.83, RHEA:17296]
GO:0043819
name: precorrin-6A synthase (deacetylating) activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + H(2)O + precorrin-5 = S-adenosyl-L-homocysteine + acetate + 2 H(+) + precorrin-6X." [EC:2.1.1.152, RHEA:18264]
GO:0043820
name: propionyl-CoA dehydrogenase activity
def: "Catalysis of the irreversible NADH-dependent formation of propionyl-CoA from acryloyl-CoA." [PMID:12603323]
GO:0043821
name: propionyl-CoA:succinate CoA-transferase activity
def: "Catalysis of the reaction: succinate + propionyl-CoA = succinyl-CoA + propionate." [IMG:01589, MetaCyc:RXN0-268]
GO:0043822
name: ribonuclease M5 activity
def: "Catalysis of the endonucleolytic cleavage of RNA, removing 21 and 42 nucleotides, respectively, from the 5'- and 3'-termini of a 5S-rRNA precursor." [EC:3.1.26.8, IMG:02522]
GO:0043823
name: spheroidene monooxygenase activity
def: "Catalysis of the reaction: spheroidene + O2 = spheroidenone + H2O." [IMG:01761, MetaCyc:RXN-10670, PMID:16086104, PMID:16158287]
GO:0043824
name: succinylglutamate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: N-succinyl-L-glutamate 5-semialdehyde + H(2)O + NAD(+) = N-succinyl-L-glutamate + 2 H(+) + NADH." [EC:1.2.1.71, RHEA:10815]
GO:0043825
name: succinylornithine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + N(2)-succinyl-L-ornithine = N-succinyl-L-glutamate 5-semialdehyde + L-glutamate." [EC:2.6.1.81, RHEA:16956]
GO:0043826
name: sulfur oxygenase reductase activity
def: "Catalysis of the reaction: 5 sulfur + oxygen + 4 H2O = sulfite + thiosulfate + 2 hydrogen sulfide + 9 H+." [IMG:00713, MetaCyc:RXN-8226, PMID:1522063]
GO:0043827
name: tRNA (adenine-57, 58-N(1)-) methyltransferase activity
def: "Catalysis of the methylation of adenine-57 and adenine-58 in the T-loop of tRNA." [IMG:00706, PMID:14739239]
GO:0043828
name: tRNA 2-selenouridine synthase activity
def: "Catalysis of the reaction: 5-methylaminomethyl-2-thiouridine + selenophosphate = 5-methylaminomethyl-2-selenouridine + phosphate (at the wobble position in tRNA)." [IMG:02609, MetaCyc:RXN0-2281]
GO:0043829
name: tRNA-specific adenosine-37 deaminase activity
def: "Catalysis of the reaction: adenosine-37 + H2O = inosine-37 + NH3, in a tRNA-Ala molecule." [IMG:00703, PMID:8915538, PMID:9707437]
GO:0043830
name: thiol-driven fumarate reductase activity
def: "Catalysis of the reaction: fumarate + coenzyme M + coenzyme B = succinate + coenzyme M + coenzyme B + heterodisulfide." [IMG:01366, PMID:2509466]
GO:0043831
name: thiosulfate dehydrogenase (quinone) activity
def: "Catalysis of the reaction: 6-decylubiquinone + 2 thiosulfate = 6-decylubiquinol + tetrathionate." [EC:1.8.5.2, RHEA:10939]
GO:0043833
name: methylamine-specific methylcobalamin:coenzyme M methyltransferase activity
def: "Catalysis of the reaction: a methylated methylamine corrinoid protein + coenzyme M = a methylamine corrinoid protein + methyl-CoM. This reaction is the transfer of the methyl group from the methylated corrinoid cofactor of a methylamine corrinoid protein to coenzyme M." [IMG:01318, MetaCyc:RXN-8099]
GO:0043834
name: trimethylamine methyltransferase activity
def: "Catalysis of the reaction: trimethylamine + a trimethylamine corrinoid protein = a methylated trimethylamine corrinoid protein + dimethylamine." [IMG:01323, MetaCyc:RXN-8102]
GO:0043835
name: uracil/thymine dehydrogenase activity
def: "OBSOLETE. Catalysis of the reactions: uracil + H2O + acceptor = barbiturate + reduced acceptor; and thymine + H2O + acceptor = 5-methylbarbiturate + reduced acceptor." [EC:1.17.99.4, IMG:01821]
GO:0043836
name: xanthine hydrolase activity
def: "Catalysis of the reaction: xanthine + H2O = 4-ureido-5-imidazole carboxylate." [IMG:01834, MetaCyc:R127-RXN]
GO:0043837
name: valine dehydrogenase (NAD) activity
def: "Catalysis of the reaction: L-valine + H2O + NAD+ = 3-methyl-2-oxobutanoate + NH3 + NADH." [IMG:00766, PMID:10612726, PMID:2803248]
GO:0043838
name: phosphatidylethanolamine:Kdo2-lipid A phosphoethanolamine transferase activity
def: "Catalysis of the reaction: Kdo2-lipid A + phosphatidylethanolamine = phosphoethanolamine-Kdo2-lipid A + diacylglycerol." [IMG:00182, PMID:15795227]
GO:0043839
name: lipid A phosphate methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosylmethionine (SAM) to the 1-phosphate group of lipid A." [IMG:02094, PMID:15994324]
GO:0043840
name: branched-chain amino acid:2-keto-4-methylthiobutyrate aminotransferase activity
def: "Catalysis of the reaction: 2-keto-4-methylthiobutyrate + L-phenylalanine = phenylpyruvate + L-methionine." [IMG:01684, PMID:12670965]
GO:0043841
name: (S)-lactate 2-kinase activity
def: "Catalysis of the reaction: (S)-lactate + GTP = 2-phospho-(S)-lactate + GDP." [IMG:01546, PMID:11535063]
GO:0043842
name: Kdo transferase activity
def: "Catalysis of the reactions: (KDO)-lipid IVA + CMP-3-deoxy-D-manno-octulosonate = KDO2-lipid IVA + CMP, and lipid IVA + CMP-3-deoxy-D-manno-octulosonate = (KDO)-lipid IVA + CMP." [IMG:00549, PMID:1577828, PMID:2033061, PMID:9195966]
GO:0043843
name: ADP-specific glucokinase activity
def: "Catalysis of the reaction: ADP + D-glucose = AMP + D-glucose 6-phosphate." [EC:2.7.1.147, IMG:01468]
GO:0043844
name: ADP-specific phosphofructokinase activity
def: "Catalysis of the reaction: ADP + D-fructose 6-phosphate = AMP + D-fructose 1,6-bisphosphate." [EC:2.7.1.146, IMG:01469]
GO:0043848
name: excinuclease cho activity
def: "Catalysis of the incision of damaged DNA on the 3' side of a lesion, typically at the ninth phosphodiester bond 3' of the damage." [IMG:00499, PMID:11818552]
GO:0043849
name: Ras palmitoyltransferase activity
def: "Catalysis of the reaction: palmitoyl-CoA + protein-cysteine = S-palmitoyl protein + CoA, specific for Ras proteins." [IMG:00473, PMID:16000296]
GO:0043851
name: methanol-specific methylcobalamin:coenzyme M methyltransferase activity
def: "Catalysis of the reaction: a methylated methanol corrinoid protein + coenzyme M = a methanol corrinoid protein + methyl-CoM. This reaction is the transfer of the methyl group from the methylated corrinoid cofactor of a methylamine corrinoid protein to coenzyme M." [IMG:01315, MetaCyc:RXN-8096]
GO:0043852
name: monomethylamine methyltransferase activity
def: "Catalysis of the reaction: monomethylamine + a monomethylamine corrinoid protein = a methylated monomethylamine corrinoid protein + NH3." [IMG:01319, MetaCyc:RXN-8098]
GO:0043854
name: cyclic nucleotide-gated mechanosensitive ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens in response to a mechanical stress and when a cyclic nucleotide has been bound by the channel complex or one of its constituent parts." [GOC:jl, IMG:02416]
GO:0043855
name: cyclic nucleotide-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens when a cyclic nucleotide has been bound by the channel complex or one of its constituent parts." [GOC:jl]
GO:0043856
name: anti-sigma factor antagonist activity
def: "The function of binding to an anti-sigma factor and stopping, preventing or reducing the rate of its activity." [GOC:jl, GOC:txnOH, PMID:15576799]
GO:0043857
name: N-acetylornithine carbamoyltransferase activity
def: "Catalysis of the reaction: N(2)-acetyl-L-ornithine + carbamoyl phosphate = N(2)-acetyl-L-citrulline + H(+) + phosphate." [EC:2.1.3.9, RHEA:18612]
GO:0043858
name: arginine:ornithine antiporter activity
def: "Catalysis of the reaction: arginine(out) + ornithine(in) = arginine(in) + ornithine(out)." [GOC:jl, PMID:17110979]
GO:0043859
name: cyanophycinase activity
def: "OBSOLETE. Catalysis of the hydrolytic cleavage of multi-L-arginyl-poly-L-aspartic acid (cyanophycin; a water-insoluble reserve polymer) into aspartate-arginine dipeptides." [PMID:10429200]
GO:0043860
name: cyanophycin synthetase activity
def: "Catalysis of the ATP-dependent polymerization of arginine and aspartate to multi-L-arginyl-poly-L-aspartic acid (cyanophycin; a water-insoluble reserve polymer)." [EC:6.3.2.29, EC:6.3.2.30, GOC:jl]
GO:0043861
name: agmatine:putrescine antiporter activity
def: "Catalysis of the reaction: agmatine(out) + putrescine(in) = agmatine(in) + putrescine(out)." [GOC:jl, PMID:17028272]
GO:0043862
name: arginine:agmatine antiporter activity
def: "Catalysis of the reaction: arginine(out) + agmatine(in) = arginine(in) + agmatine(out)." [GOC:jl, PMID:17099215]
GO:0043863
name: 4-hydroxy-2-ketopimelate aldolase activity
def: "Catalysis of the reaction: 4-hydroxy-2-ketopimelate = succinate semialdehyde + pyruvate." [MetaCyc:4-HYDROXY-2-KETOPIMELATE-LYSIS-RXN]
GO:0043864
name: indoleacetamide hydrolase activity
def: "Catalysis of the reaction: indole-3-acetamide + H2O = indole-3-acetate + NH3. Indole-3-acetamide is known as IAM and indole-3-acetate as IAA." [GOC:jl, MetaCyc:RXNN-404]
GO:0043865
name: methionine transmembrane transporter activity
def: "Catalysis of the transfer of methionine from one side of a membrane to the other." [GOC:jl]
GO:0043866
name: adenylyl-sulfate reductase (thioredoxin) activity
def: "Catalysis of the reaction: AMP + sulfite + thioredoxin disulfide = 5'-adenylyl sulfate + thioredoxin." [EC:1.8.4.10]
GO:0043867
name: 7-cyano-7-deazaguanine tRNA-ribosyltransferase activity
def: "Catalysis of the reaction: tRNA guanine + 7-cyano-7-deazaguanine = tRNA 7-cyano-7-deazaguanine + guanine." [PMID:16407303, PMID:7748953]
GO:0043869
name: alpha-aminoadipate acetyltransferase activity
def: "Catalysis of the reaction: alpha-aminoadipate + acetyl-CoA = N2-acetyl-alpha-aminoadipate + coenzyme A." [MetaCyc:RXN-5181, PMID:10613839, PMID:12925802]
GO:0043870
name: N-acetyl-gamma-aminoadipyl-phosphate reductase activity
def: "Catalysis of the reaction: N(2)-acetyl-L-aminoadipate-semialdehyde + NADP+ + phosphate = N(2)-acetyl-L-gamma-aminoadipyl phosphate + NADPH." [MetaCyc:RXN-5183]
GO:0043871
name: delta1-piperideine-6-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: delta1-piperideine-6-carboxylate + NAD+ + 2 H2O = 2-aminoadipate + NADH + H+. Delta1-piperideine-6-carboxylate is also known as 2,3,4,5-tetrahydropyridine-2-carboxylate." [MetaCyc:RXN-8162, PMID:16237033]
GO:0043872
name: lysine:cadaverine antiporter activity
def: "Catalysis of the reaction: lysine(out) + cadaverine(in) = lysine(in) + cadaverine(out)." [GOC:jl, PMID:10986235, TC:2.A.3.2.2]
GO:0043873
name: pyruvate-flavodoxin oxidoreductase activity
def: "Catalysis of the reaction: pyruvate + CoA + oxidized flavodoxin = acetyl-CoA + CO2 + reduced flavodoxin." [PMID:6352705]
GO:0043874
name: acireductone synthase activity
def: "Catalysis of the reactions: 5-(methylthio)-2,3-dioxopentyl phosphate + H2O = 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + phosphate; (1a) 5-(methylthio)-2,3-dioxopentyl phosphate = 2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate; (1b) 2-hydroxy-5-(methylthio)-3-oxopent-1-enyl phosphate + H2O = 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one + phosphate." [EC:3.1.3.77]
GO:0043875
name: 2-ketobutyrate formate-lyase activity
def: "Catalysis of the reaction: 2-oxobutanoate + coenzyme A = propionyl-CoA + formate." [MetaCyc:KETOBUTFORMLY-RXN, PMID:9484901]
GO:0043876
name: D-threonine aldolase activity
def: "Catalysis of the reaction: D-threonine (or D-allo-threonine) = glycine + acetaldehyde." [MetaCyc:4.1.2.42-RXN, PMID:9642221]
GO:0043877
name: galactosamine-6-phosphate isomerase activity
def: "Catalysis of the reaction: D-galactosamine 6-phosphate + H2O = D-tagatose 6-phosphate + NH3." [PMID:10931310]
GO:0043878
name: glyceraldehyde-3-phosphate dehydrogenase (NAD+) (non-phosphorylating) activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + NAD+ + H2O = 3-phospho-D-glycerate + NADH + H+." [PMID:9497334]
GO:0043879
name: glycolate transmembrane transporter activity
def: "Catalysis of the transfer of glycolate from one side of a membrane to the other. Glycolate is the smallest alpha-hydroxy acid (AHA)." [CHEBI:28905, GOC:jl]
GO:0043880
name: crotonyl-CoA reductase activity
def: "Catalysis of the reduction of crotonyl-CoA to butyryl-CoA." [InterPro:IPR010085, PMID:11162231]
GO:0043881
name: mesaconyl-CoA hydratase activity
def: "Catalysis of the hydration of mesaconyl-CoA to beta-methylmalyl-CoA." [PMID:16856935, PMID:16856937]
GO:0043882
name: malate:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: malate(out) + Na+(out) = malate(in) + Na+(in)." [GOC:jl, PMID:10903309]
GO:0043883
name: malolactic enzyme activity
def: "Catalysis of the reaction: malate + H+ = L-lactate + CO2." [MetaCyc:RXN8E-5623, PMID:3139053]
GO:0043884
name: CO-methylating acetyl-CoA synthase activity
def: "Catalysis of the reaction: acetyl-CoA + corrinoid protein = CO + methylcorrinoid protein + CoA." [EC:2.3.1.169, PMID:1748656]
GO:0043885
name: carbon-monoxide dehydrogenase (ferredoxin) activity
def: "Catalysis of the reaction: CO + H2O + oxidized ferredoxin = CO2 + reduced ferredoxin." [EC:1.2.7.4]
GO:0043886
name: structural constituent of carboxysome
def: "The action of a molecule that contributes to the structural integrity of a carboxysome, an organelle found in the Cyanobacteria consisting of a proteinaceous coat and enzymes for the fixation of carbon dioxide." [GOC:jl]
GO:0043887
name: melibiose:sodium symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: melibiose(out) + Na+(out) = melibiose(in) + Na+(in)." [PMID:1970646]
GO:0043888
name: (S)-2,3-di-O-geranylgeranylglyceryl phosphate synthase activity
def: "Catalysis of the transfer of a geranylgeranyl group from geranylgeranyl diphosphate to (S)-3-O-geranylgeranylglyceryl phosphate to form (S)-2,3-di-O-geranylgeranylglyceryl phosphate." [PMID:15356000, PMID:16494480]
GO:0043889
name: (S)-3-O-geranylgeranylglyceryl phosphate synthase activity
def: "Catalysis of the alkylation of the primary hydroxyl group in (S)-glyceryl phosphate by geranylgeranyl diphosphate to form (S)-3-O-geranylgeranylglyceryl phosphate." [PMID:12801917, PMID:17253090, PMID:8408023]
GO:0043890
name: N-acetylgalactosamine-6-sulfatase activity
def: "Catalysis of the hydrolysis of the 6-sulfate groups of the N-acetyl-D-galactosamine 6-sulfate units of chondroitin sulfate and of the D-galactose 6-sulfate units of keratan sulfate." [EC:3.1.6.4]
GO:0043891
name: glyceraldehyde-3-phosphate dehydrogenase (NAD(P)+) (phosphorylating) activity
def: "Catalysis of the reaction: D-glyceraldehyde 3-phosphate + phosphate + NAD(P)+ = 3-phospho-D-glyceroyl phosphate + NAD(P)H + H+." [EC:1.2.1.59]
GO:0043892
name: methylglyoxal reductase (NADPH-dependent) activity
def: "Catalysis of the reaction: lactaldehyde + NADP+ = methylglyoxal + NADPH + H+." [EC:1.1.1.283]
GO:0043893
name: acetate:cation symporter activity
def: "Catalysis of the transfer of acetate from one side of a membrane to the other according to the reaction: acetate(out) + cation(out) = acetate(in) + cation(in)." [GOC:jl]
GO:0043894
name: acetyl-CoA synthetase acetyltransferase activity
def: "Catalysis of the acetylation of residue Lys609 of the enzyme acetyl-CoA synthetase, using acetyl-CoA as substrate." [PMID:15236963]
GO:0043895
name: cyclomaltodextrin glucanotransferase activity
def: "Catalysis of the cyclization of part of a 1,4-alpha-D-glucan chain by formation of a 1,4-alpha-D-glucosidic bond." [EC:2.4.1.19]
GO:0043896
name: glucan 1,6-alpha-glucosidase activity
def: "Catalysis of the hydrolysis of (1->6)-alpha-D-glucosidic linkages in (1->6)-alpha-D-glucans and derived oligosaccharides." [EC:3.2.1.70]
GO:0043897
name: glucan 1,4-alpha-maltohydrolase activity
def: "Catalysis of the hydrolysis of (1->4)-alpha-D-glucosidic linkages in polysaccharides so as to remove successive alpha-maltose residues from the non-reducing ends of the chains." [EC:3.2.1.133]
GO:0043898
name: 2,3-dihydroxybiphenyl 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxybiphenyl + O2 = 2-hydroxy-6-phenylhexa-2,4-dienoic acid." [GOC:jl, PMID:15715866]
GO:0043899
name: phosphoserine:homoserine phosphotransferase activity
def: "Catalysis of the transfer of a phosphoryl group from phosphoserine to homoserine to form phosphohomoserine." [GOC:jl, PMID:14699121]
GO:0043904
name: isochorismate pyruvate lyase activity
def: "Catalysis of the reaction: isochorismate = salicylate + pyruvate." [GOC:jl, PMID:16248620]
GO:0043905
name: Ser-tRNA(Thr) hydrolase activity
def: "Catalysis of the hydrolysis of misacylated Ser-tRNA(Thr)." [GOC:jl, PMID:15240874]
GO:0043906
name: Ala-tRNA(Pro) hydrolase activity
def: "Catalysis of the hydrolysis of misacylated Ala-tRNA(Pro)." [GOC:jl, PMID:14663147]
GO:0043907
name: Cys-tRNA(Pro) hydrolase activity
def: "Catalysis of the hydrolysis of misacylated Cys-tRNA(Pro)." [GOC:jl, PMID:15886196]
GO:0043908
name: Ser(Gly)-tRNA(Ala) hydrolase activity
def: "Catalysis of the hydrolysis of misacylated Ser-tRNA(Ala) and Gly-tRNA(Ala)." [GOC:jl, PMID:14663147]
GO:0043909
name: N-acetylcitrulline deacetylase activity
def: "Catalysis of the reaction: N-acetyl-L-citrulline + H2O = citrulline + acetate." [GOC:jl, IMG:02799, PMID:16750290]
GO:0043910
name: ATP:coenzyme F420 adenylyltransferase activity
def: "Catalysis of the reaction: ATP + factor gamma-F420-2 + H+ = coenzyme F390-A + diphosphate." [GOC:jl, IMG:02803, MetaCyc:RXN-9385, PMID:7957247, PMID:8550473]
GO:0043911
name: D-lysine transaminase activity
def: "Catalysis of the reaction: D-lysine + 2-oxoglutarate = L-glutamate + 6-amino-2-oxohexanoate." [GOC:jl, IMG:02956, PMID:17259313]
GO:0043912
name: D-lysine oxidase activity
def: "Catalysis of the reaction: D-lysine + O2 + H2O = 6-amino-2-oxohexanoate + NH3 + hydrogen peroxide." [GOC:jl, IMG:02957, PMID:17259313]
GO:0043914
name: NADPH:sulfur oxidoreductase activity
def: "Catalysis of the reaction: NADPH + H+ + sulfur = hydrogen sulfide + NADP+." [GOC:jl, PMID:17449625]
GO:0043915
name: L-seryl-tRNA(Sec) kinase activity
def: "Catalysis of the reaction: ATP + L-seryl-tRNA(Sec) = ADP + O-phospho-L-seryl-tRNA(Sec)." [GOC:jl, IMG:03019, PMID:16201757]
GO:0043916
name: DNA-7-methylguanine glycosylase activity
def: "Catalysis of the reaction: DNA containing 7-methylguanine + H2O = DNA with abasic site + 7-methylguanine. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the damaged DNA 7-methylguanine and the deoxyribose sugar to remove the 7-methylguanine, leaving an abasic site." [GOC:jl, PMID:16468998]
GO:0043917
name: ribose 1,5-bisphosphate isomerase activity
def: "Catalysis of the reaction: D-ribose 1,5-bisphosphate = D-ribulose 1,5-bisphosphate." [GOC:jl, IMG:03024]
GO:0043918
name: cadaverine aminopropyltransferase activity
def: "Catalysis of the reaction: S-adenosylmethioninamine + cadaverine = 5'-methylthioadenosine + N-(3-aminopropyl)cadaverine." [GOC:jl, IMG:00522, PMID:17545282]
GO:0043919
name: agmatine aminopropyltransferase activity
def: "Catalysis of the reaction: agmatine + S-adenosylmethioninamine = N1-aminopropylagmatine + 5'-methylthioadenosine." [GOC:jl, IMG:03077, PMID:15983049]
GO:0043920
name: aminopropylagmatine ureohydrolase activity
def: "Catalysis of the reaction: N1-aminopropylagmatine + H2O = spermidine + urea." [GOC:jl, IMG:03079, PMID:15983049]
GO:0043924
name: suramin binding
def: "Interacting selectively and non-covalently with suramin, a naphthalenesulfonic acid compound which is used in the treatment of diseases caused by trypanosomes and worms." [CHEBI:45906, GOC:jl, Wikipedia:Suramin]
GO:0043955
name: 3-hydroxypropionyl-CoA synthetase activity
def: "Catalysis of the reaction: 3-hydroxypropionate + ATP + CoA = 3-hydroxypropionyl-CoA + AMP + diphosphate." [GOC:jl, PMID:11821399]
GO:0043956
name: 3-hydroxypropionyl-CoA dehydratase activity
def: "Catalysis of the reaction: 3-hydroxypropionyl-CoA = acrylyl-CoA + H2O." [GOC:jl, PMID:11821399]
GO:0043957
name: acryloyl-CoA reductase (NADP+) activity
def: "Catalysis of the reaction: acryloyl-CoA + NADPH + H+ = propionyl-CoA + NADP+." [GOC:jl, PMID:11821399]
GO:0043958
name: acryloyl-CoA reductase activity
def: "Catalysis of the reaction: acryloyl-CoA + NADH + H+ + a reduced electron-transfer flavoprotein = propionyl-CoA + NAD+ + an oxidized electron-transfer flavoprotein." [GOC:jl, PMID:12603323]
GO:0043959
name: L-erythro-3-methylmalyl-CoA lyase activity
def: "Catalysis of the reaction: propionyl-CoA + glyoxylate = L-erythro-3-methylmalyl-CoA." [GOC:jl, IMG:03303]
GO:0043960
name: L-erythro-3-methylmalyl-CoA dehydratase activity
def: "Catalysis of the reaction: L-erythro-3-methylmalyl-CoA = mesaconyl-CoA + H2O." [GOC:jl, IMG:03304]
GO:0043961
name: succinyl-CoA:(R)-citramalate CoA-transferase activity
def: "Catalysis of the reaction: succinyl-CoA + (R)-citramalate = succinate + (R)-citramalyl-CoA." [GOC:jl, IMG:03306, PMID:17259315]
GO:0043992
name: histone acetyltransferase activity (H3-K9 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 9) = CoA + histone H3 N6-acetyl-L-lysine (position 9)." [EC:2.3.1.48]
GO:0043993
name: histone acetyltransferase activity (H3-K18 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 18) = CoA + histone H3 N6-acetyl-L-lysine (position 18)." [EC:2.3.1.48]
GO:0043994
name: histone acetyltransferase activity (H3-K23 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 23) = CoA + histone H3 N6-acetyl-L-lysine (position 23)." [EC:2.3.1.48]
GO:0043995
name: histone acetyltransferase activity (H4-K5 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H4 L-lysine (position 5) = CoA + histone H4 N6-acetyl-L-lysine (position 5)." [EC:2.3.1.48]
GO:0043996
name: histone acetyltransferase activity (H4-K8 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H4 L-lysine (position 8) = CoA + histone H4 N6-acetyl-L-lysine (position 8)." [EC:2.3.1.48]
GO:0043997
name: histone acetyltransferase activity (H4-K12 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H4 L-lysine (position 12) = CoA + histone H4 N6-acetyl-L-lysine (position 12)." [EC:2.3.1.48]
GO:0043998
name: H2A histone acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + histone H2A L-lysine = CoA + histone H2A N6-acetyl-L-lysine." [EC:2.3.1.48]
GO:0043999
name: histone acetyltransferase activity (H2A-K5 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H2A L-lysine (position 5) = CoA + histone H2A N6-acetyl-L-lysine (position 5)." [EC:2.3.1.48]
GO:0044012
name: histone acetyltransferase activity (H2A-K9 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H2A L-lysine (position 9) = CoA + histone H2A N6-acetyl-L-lysine (position 9)." [EC:2.3.1.48]
GO:0044013
name: H2B histone acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + histone H2B L-lysine = CoA + histone H2B N6-acetyl-L-lysine." [EC:2.3.1.48]
GO:0044014
name: histone acetyltransferase activity (H2B-K5 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H2B L-lysine (position 5) = CoA + histone H2B N6-acetyl-L-lysine (position 5)." [EC:2.3.1.48]
GO:0044015
name: histone acetyltransferase activity (H2B-K12 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H2B L-lysine (position 12) = CoA + histone H2B N6-acetyl-L-lysine (position 12)." [EC:2.3.1.48]
GO:0044016
name: histone acetyltransferase activity (H3-K4 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 4) = CoA + histone H3 N6-acetyl-L-lysine (position 4)." [EC:2.3.1.48]
GO:0044017
name: histone acetyltransferase activity (H3-K27 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 27) = CoA + histone H3 N6-acetyl-L-lysine (position 27)." [EC:2.3.1.48]
GO:0044018
name: histone acetyltransferase activity (H3-K36 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 36) = CoA + histone H3 N6-acetyl-L-lysine (position 36)." [EC:2.3.1.48]
GO:0044019
name: histone acetyltransferase activity (H3-K72 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H3 L-lysine (position 72) = CoA + histone H3 N6-acetyl-L-lysine (position 72)." [EC:2.3.1.48]
GO:0044020
name: histone methyltransferase activity (H4-R3 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone H4)-arginine (position 3) = S-adenosyl-L-homocysteine + (histone H4)-N-methyl-arginine (position 3). This reaction is the addition of a methyl group to arginine at position 3 of histone H4." [EC:2.1.1.125, GOC:mah, PMID:17898714]
GO:0044022
name: histone kinase activity (H3-S28 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-28 residue of the N-terminal tail of histone H3." [GOC:jl]
GO:0044023
name: histone kinase activity (H4-S1 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-1 residue of the N-terminal tail of histone H4." [GOC:jl]
GO:0044024
name: histone kinase activity (H2A-S1 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-1 residue of the N-terminal tail of histone H2A." [GOC:jl]
GO:0044025
name: histone kinase activity (H2B-S14 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-14 or an equivalent residue of the N-terminal tail of histone H2B." [GOC:jl]
GO:0044101
name: (R)-citramalyl-CoA lyase activity
def: "Catalysis of the reaction: (R)-citramalyl-CoA = pyruvate + acetyl-CoA." [GOC:jl, IMG:03307]
GO:0044102
name: purine deoxyribosyltransferase activity
def: "Catalysis of deoxyribose exchange between purine deoxyribonucleoside as a donor and purine base as an acceptor." [GOC:jl, IMG:03313, PMID:11836245]
GO:0044103
name: L-arabinose 1-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: L-arabinose + NADP+ = L-arabinono-1,4-lactone + NADPH + H+." [GOC:jl, IMG:03333]
GO:0044104
name: 2,5-dioxovalerate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: 2,5-dioxopentanoate + NAD+ + H2O = 2-oxoglutarate + NADH + H+." [IMG:03338, PMID:16835232, PMID:17202142]
GO:0044105
name: L-xylulose reductase (NAD+) activity
def: "Catalysis of the reaction: xylitol + NAD+ = L-xylulose + NADH + H+." [IMG:03341, PMID:14736891]
GO:0044183
name: protein binding involved in protein folding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules) that contributes to the process of protein folding." [GOC:mtg_cambridge_2009]
GO:0044197
name: Rel homology domain binding
def: "Interacting selectively and non-covalently with a Rel Homology Domain (RHD) of a protein. The RHD is found in a family of eukaryotic transcription factors, which includes NF-kappaB, Dorsal, Relish and NFAT." [InterPro:IPR011539, Wikipedia:Rel_homology_domain]
GO:0044198
name: zf-TRAF domain binding
def: "Interacting selectively and non-covalently with a TRAF-type zinc finger domain of a protein." [InterPro:IPR001293]
GO:0044212
name: transcription regulatory region DNA binding
def: "Interacting selectively and non-covalently with a DNA region that regulates the transcription of a region of DNA, which may be a gene, cistron, or operon. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors." [GOC:jl, GOC:txnOH, SO:0005836]
GO:0044213
name: intronic transcription regulatory region DNA binding
def: "Interacting selectively and non-covalently with an intronic DNA region that regulates the transcription of the transcript it is contained within." [GOC:ecd, GOC:txnOH]
GO:0044318
name: L-aspartate:fumarate oxidoreductase activity
def: "Catalysis of the reaction: L-aspartate + fumarate = alpha-iminosuccinate + succinate." [PMID:20149100]
GO:0044323
name: retinoic acid-responsive element binding
def: "Interacting selectively and non-covalently with a retinoic acid-responsive element, a variable direct repeat of the sequence PuGGTCA spaced by five nucleotides (DR5) found in the promoters of retinoic acid-responsive genes, to which retinoic acid receptors bind." [GOC:jl, GOC:vw, GOC:yaf, PMID:11327309, PMID:19917671]
GO:0044325
name: ion channel binding
def: "Interacting selectively and non-covalently with one or more specific sites on an ion channel, a protein complex that spans a membrane and forms a water-filled channel across the phospholipid bilayer allowing selective ion transport down its electrochemical gradient." [GOC:BHF, GOC:jl]
GO:0044373
name: cytokinin binding
def: "Interacting selectively and non-covalently with a cytokinin, any of a class of adenine-derived compounds that can function in plants as growth regulators." [GOC:jl]
GO:0044374
name: sequence-specific DNA binding, bending
def: "The activity of binding selectively and non-covalently to DNA in a sequence-specific manner and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:jl, GOC:vw]
GO:0044377
name: RNA polymerase II core promoter proximal region sequence-specific DNA binding, bending
def: "Interacting selectively and non-covalently with a sequence of DNA that is in cis with and relatively close to a core promoter for RNA polymerase II and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:jl]
GO:0044378
name: non-sequence-specific DNA binding, bending
def: "The activity of binding selectively and non-covalently to DNA in a sequence-independent manner and distorting the original structure of DNA, typically a straight helix, into a bend, or increasing the bend if the original structure was intrinsically bent due to its sequence." [GOC:jl, GOC:vw, PMID:20123079]
GO:0044388
name: small protein activating enzyme binding
def: "Interacting selectively and non-covalently with a small protein activating enzyme, such as ubiquitin-activating enzyme." [GOC:jl]
GO:0044389
name: small conjugating protein ligase binding
def: "Interacting selectively and non-covalently with a small protein ligase, such as ubiquitin-ligase." [GOC:jl]
GO:0044390
name: small protein conjugating enzyme binding
def: "Interacting selectively and non-covalently with a small protein conjugating enzyme such as ubiquitin conjugating enzyme." [GOC:jl]
GO:0044466
name: glutaryl-CoA hydrolase activity
def: "Catalysis of the reaction: glutaryl-CoA + H2O = CoA + glutarate." [GOC:pm, PMID:16141203]
GO:0044508
name: glucagon-like peptide 1 receptor activity
def: "Combining with glucagon-like peptide 1 and transmitting the signal across the membrane by activating an associated G-protein." [GOC:jl, PMID:12529935]
GO:0044535
name: very-long-chain fatty acyl-CoA oxidase activity
def: "Catalysis of the reaction: very-long-chain fatty acyl-CoA (C22 - C24) + O2 = trans-2,3-dehydroacyl-CoA + hydrogen peroxide." [PMID:17458872]
GO:0044540
name: L-cystine L-cysteine-lyase (deaminating)
def: "Catalysis of the reaction: L-cystine + H2O <=> pyruvate + NH3 + thiocysteine. Thiocysteine is also known as cysteine persulfide." [GOC:jl, RHEA:24930]
GO:0044547
name: DNA topoisomerase binding
def: "Interacting selectively and non-covalently with a DNA topoisomerase." [GOC:jl]
GO:0044548
name: S100 protein binding
def: "Interacting selectively and non-covalently with a S100 protein. S100 is a small calcium and zinc binding protein produced in astrocytes that is implicated in Alzheimer's disease, Down Syndrome and ALS." [GOC:jid]
GO:0044549
name: GTP cyclohydrolase binding
def: "Interacting selectively and non-covalently with a GTP cyclohydrolase." [GOC:jl]
GO:0044583
name: cellotriose binding
def: "Interacting selectively and non-covalently with cellotriose." [GOC:mengo_curators, GOC:tt]
GO:0044584
name: cellodextrin binding
def: "Interacting selectively and non-covalently with cellodextrin, a glucose polymer of 2 or more glucose monomers." [GOC:mengo_curators, GOC:tt, PMID:18952792]
GO:0044585
name: cellobiose binding
def: "Interacting selectively and non-covalently with cellobiose, a disaccharide that represents the basic repeating unit of cellulose." [GOC:mengo_curators, GOC:tt]
GO:0044586
name: cellotetraose binding
def: "Interacting selectively and non-covalently with cellotetraose, an oligosaccharide consisting of four glucose residues resulting from hydrolysis of cellulose." [GOC:mengo_curators, GOC:tt]
GO:0044587
name: cellopentaose binding
def: "Interacting selectively and non-covalently with cellopentaose, an oligosaccharide consisting of four glucose residues resulting from hydrolysis of cellulose." [GOC:mengo_curators, GOC:tt]
GO:0044588
name: laminaribiose binding
def: "Interacting selectively and non-covalently with laminaribiose, a disaccharide." [GOC:mengo_curators, GOC:tt]
GO:0044589
name: pectin binding
def: "Interacting selectively and non-covalently with pectin." [GOC:mengo_curators, GOC:tt]
GO:0044590
name: iron-sulfur-molybdenum cofactor binding
def: "Interacting selectively and non-covalently with iron molybdenum cofactor, the cofactor located at the active site of the molybdenum nitrogenase." [GOC:mengo_curators, GOC:tt, PMID:18429691]
GO:0044594
name: 17-beta-hydroxysteroid dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: a 17-beta-hydroxysteroid + NAD+ = a 17-oxosteroid + NADH + H+." [PMID:17074428]
GO:0044595
name: decaprenyldihydroxybenzoate methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-decaprenyl-4,5-dihydroxybenzoate = S-adenosyl-L-homocysteine + 3-decaprenyl-4-hydroxy-5-methoxybenzoate." [PMID:10777520]
GO:0044596
name: 3-demethylubiquinone-10 3-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-demethylubiquinone-10 = S-adenosyl-L-homocysteine + ubiquinone-10." [PMID:10777520]
GO:0044600
name: protein guanylyltransferase activity
def: "Catalysis of the reaction: GTP + protein = diphosphate + guanylyl-protein; mediates the addition of an guanylyl (guanosine 5'-monophosphate; GMP group) to specific residues of target proteins." [GOC:sp, PMID:20651120]
GO:0044603
name: protein adenylylhydrolase activity
def: "Catalysis of the reaction: adenylyl-protein+ H2O = adenylate + protein; mediates the removal of an adenylyl (adenosine 5'-monophosphate; AMP group) from specific residues of target proteins." [PMID:21734656]
GO:0044604
name: phytochelatin transmembrane transporter ATPase activity
def: "Catalysis of the reaction: ATP + H2O + phytochelatin (in) = ADP + phosphate + phytochelatin (out)." [GOC:al, PMID:7876244]
GO:0044605
name: phosphocholine transferase activity
def: "Catalysis of the reaction: CDP-choline + protein-serine = CMP + protein-serine-choline phosphate." [GOC:sp, PMID:21822290]
GO:0044606
name: phosphocholine hydrolase activity
def: "Catalysis of the reaction: protein-serine-choline phosphate + H2O = protein-serine + choline phosphate." [GOC:sp, PMID:22158903]
GO:0044610
name: FMN transmembrane transporter activity
def: "Enables the directed movement of flavine mononucleotide (FMN) into, out of or within a cell, or between cells." [GOC:ans, PMID:22185573]
GO:0044620
name: ACP phosphopantetheine attachment site binding
def: "Interacting selectively and non-covalently with the attachment site of the phosphopantetheine prosthetic group of an acyl carrier protein (ACP)." [GOC:jl, GOC:vw]
GO:0044653
name: dextrin alpha-glucosidase activity
def: "Catalysis of the reaction: dextrin + H2O = alpha-D-glucose." [PMID:18556189]
GO:0044654
name: starch alpha-glucosidase activity
def: "Catalysis of the reaction: starch + H2O = alpha-D-glucose." [PMID:18556189]
GO:0044667
name: (R)-carnitine:4-(trimethylammonio)butanoate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: (R)-carnitine(out) + 4-(trimethylammonio)butanoate(in) = (R)-carnitine(in) + 4-(trimethylammonio)butanoate(out)." [GOC:crds]
GO:0044668
name: sodium:malonate symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sodium(out)+ malonate(out) = sodium(in) + malonate(in)." [GOC:crds]
GO:0044669
name: sodium:galactoside symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sodium(out)+ galactoside(out) = sodium(in) + galactoside(in)." [GOC:crds]
GO:0044670
name: sodium:alanine symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sodium(out)+ alanine(out) = sodium(in) + alanine(in)." [GOC:crds]
GO:0044682
name: archaeal-specific GTP cyclohydrolase activity
def: "Catalysis of the reaction: GTP + H2O <=> 7,8-dihydro-D-neopterin 2',3'-cyclic phosphate + diphosphate +formate + H+. This activity is part of the biosynthesis of methanopterin in Archaea, and requires Fe2+." [GOC:mengo_curators, PMID:17497938]
GO:0044683
name: methylthiol:coenzyme M methyltransferase activity
def: "Catalysis of the overall reaction: methyl-Co(III) methylated-thiol-specific corrinoid protein + coenzyme M = Co(I) methylated--thiol-specific corrinoid protein + methyl-CoM." [MetaCyc:RXN-8125, PMID:9371433]
GO:0044684
name: dihydromethanopterin reductase activity
def: "Catalysis of the reaction: 7,8-dihydromethanopterin + NADPH = 5,6,7,8-tetrahydromethanopterin + NADP." [GOC:mengo_curators, PMID:15028691]
GO:0044685
name: tetrahydromethanopterin-dependent serine hydroxymethyltransferase activity
def: "Catalysis of the reaction: tetrahydromethanopterin + L-serine = 5,10-methylenetetrahydromethanopterin + glycine + H2O." [GOC:mengo_curators, PMID:12902326]
GO:0044686
name: cysteate synthase activity
def: "Catalysis of the reaction: L-phosphoserine + SO32- = L-cysteate + HPO4-." [GOC:mengo_curators, PMID:19761441]
GO:0044687
name: geranylfarnesyl diphosphate synthase activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate + isopentenyl diphosphate = (2E,6E,10E,14E)-geranylfarnesyl diphosphate + diphosphate." [GOC:mengo_curators, PMID:20097171]
GO:0044688
name: 7,8-dihydro-D-neopterin 2',3'-cyclic phosphate phosphodiesterase activity
def: "Catalysis of the reaction: 7,8-dihydro-D-neopterin 2',3'-cyclic phosphate + H2O = 7,8-dihydroneopterin 3'-phosphate + H+." [GOC:mengo_curators, PMID:19746965]
GO:0044689
name: 7,8-didemethyl-8-hydroxy-5-deazariboflavin synthase activity
def: "Catalysis of the reaction: 5-amino-6-(D-ribitylamino)uracil + 4-hydroxyphenylpyruvate + 2 S-adenosyl-L-methionine + H2O = 7,8-didemethyl-8-hydroxy-5-deazariboflavin + 2 5'-deoxyadenosine + 2 L-methionine + oxalate + ammonia + 4 H+." [GOC:mengo_curators, PMID:11948155, PMID:14593448]
GO:0044692
name: exoribonuclease activator activity
def: "Increases the activity of an exoribonuclease." [GOC:rb, PMID:22570495]
GO:0044693
name: trehalose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: trehalose(out) + H+(out) = trehalose(in) + H+(in)." [PMID:11136464]
GO:0044713
name: 2-hydroxy-adenosine triphosphate pyrophosphatase activity
def: "Catalysis of the reaction: 2-hydroxy-adenosine-triphosphate + H2O = 2-hydroxy-adenosine phosphate + diphosphate." [GOC:pde, PMID:11139615]
GO:0044714
name: 2-hydroxy-(deoxy)adenosine-triphosphate pyrophosphatase activity
def: "Catalysis of the reaction: 2-hydroxy-(deoxy)adenosine-triphosphate + H2O = 2-hydroxy-(deoxy)adenosine phosphate + diphosphate." [GOC:pde, PMID:11139615]
GO:0044715
name: 8-oxo-dGDP phosphatase activity
def: "Catalysis of the reaction 8-oxo-dGDP + H2O = 8-oxo-dGMP + phosphate." [GOC:pde, PMID:22556419]
GO:0044716
name: 8-oxo-GDP phosphatase activity
def: "Catalysis of the reaction 8-oxo-GDP + H2O = 8-oxo-GMP + phosphate." [GOC:pde, PMID:22556419]
GO:0044717
name: 8-hydroxy-dADP phosphatase activity
def: "Catalysis of the reaction8-hydroxy-dADP + H2O = 8-hydroxy-dAMP + phosphate." [GOC:pde, PMID:22556419]
GO:0044729
name: hemi-methylated DNA-binding
def: "Interacting selectively and non-covalently with double-stranded hemi-methylated DNA at replication foci (one strand methylated, while the other strand is unmethylated). Methylation of cytosine or adenine in DNA is an important mechanism for establishing stable heritable epigenetic marks." [GOC:imk, GOC:sp, PMID:18772889]
GO:0044730
name: bone sialoprotein binding
def: "Interacting selectively and non-covalently with a bone sialoprotein, an extracellular matrix glycoprotein found on the surface of bones and dentin." [PMID:10642520]
GO:0044736
name: acid-sensing ion channel activity
def: "Catalysis of the transmembrane transfer of a sodium ion by a neuronal, voltage-insensitive channel that opens when a an extracellular proton has been bound by the channel complex." [GOC:jl]
GO:0044748
name: 3'-5'-exoribonuclease activity involved in mature miRNA 3'-end processing
def: "Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of an RNA molecule that contributes to forming distinct miRNA isoforms from a mature miRNA." [GOC:sart]
GO:0044750
name: high-affinity nickel cation transmembrane transporter activity
def: "Catalysis of the high-affinity transfer of nickel (Ni) cations from one side of a membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:jl]
GO:0044769
name: ATPase activity, coupled to transmembrane movement of ions, rotational mechanism
def: "Catalysis of the transfer of ions from one side of a membrane to the other according to the reaction: ATP + H2O + ion(in) = ADP + phosphate + ion(out), by a rotational mechanism." [GOC:jl]
GO:0044822
name: poly(A) RNA binding
def: "Interacting non-covalently with a poly(A) RNA, a RNA molecule which has a tail of adenine bases." [GOC:jl]
GO:0044823
name: retroviral integrase activity
def: "Catalysis of the covalent insertion of double-stranded retroviral DNA into host DNA. Proceeds by an endonucleolytic cleavage at each 3'-OH extremity of the viral genome, named 3'-processing, followed by a strand transfer reaction leading to the insertion of the processed viral DNA into the target DNA by a trans-esterification mechanism." [PMID:19091057]
GO:0044824
name: retroviral 3' processing activity
def: "The catalysis of the removal of two di- or tri-nucleotides from each 3' end of double-stranded viral DNA, exposing recessed 3' hydroxyls." [PMID:22580823, Reactome:REACT_9069.1]
GO:0044825
name: retroviral strand transfer activity
def: "Catalysis of the covalent insertion of processed 3'-viral DNA ends into host chromosomal DNA by a trans-esterification reaction." [PMID:22580823]
GO:0044875
name: gamma-glutamyl hercynylcysteine S-oxide synthase
def: "Catalysis of the reaction: L-cysteine + L-histidine + O2 <=> L-alanin-3-yl L-histidin-5-yl sulfoxide + H2O." [GOC:jl, PMID:24828577]
GO:0044876
name: hercynylselenocysteine S-oxide synthase
def: "Catalysis of the reaction: 2 L-selenocysteine + 2 hercynine + O2 <=> 2 H2O + 2 hercynylselenocysteine." [GOC:jl, PMID:24828577]
GO:0044877
name: macromolecular complex binding
def: "Interacting selectively and non-covalently with any macromolecular complex." [GOC:jl]
GO:0045012
name: MHC class II receptor activity
def: "OBSOLETE. A major histocompatibility complex class II receptor. These display processed antigens from virally-infected or transformed cells. Class-II-positive cells ('antigen-presenting cells') can take up antigens from outside by endocytosis, degrade them into small peptides, and re-export the peptides (now bound to MHC class II protein) to the cell surface. These peptide-MHC class II complexes can then be recognized by specific CD4+ lymphocytes." [ISBN:081533642X, ISBN:0879694971]
GO:0045015
name: HDEL sequence binding
def: "Interacting selectively and non-covalently with a HDEL sequence, the C terminus tetrapeptide sequence His-Asp-Glu-Leu found in proteins that are to be retained in the endoplasmic reticulum." [PMID:1327759]
GO:0045024
name: peptidyl-glutamyl peptide hydrolyzing enzyme activity
def: "OBSOLETE. Catalysis of the cleavage of bonds after acidic amino acids and after branched chain amino acids." [PMID:11735414]
GO:0045027
name: DNA end binding
def: "Interacting selectively and non-covalently with the ends of DNA that are exposed by the creation of double-strand breaks (DSBs)." [GOC:jl]
GO:0045028
name: G-protein coupled purinergic nucleotide receptor activity
def: "Combining with a purine nucleotide and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:mah, PMID:9755289]
GO:0045029
name: UDP-activated nucleotide receptor activity
def: "Combining with a nucleotide and transmitting the signal to a heterotrimeric G-protein complex to initiate a change in cell activity, activated by UDP." [GOC:mah]
GO:0045030
name: UTP-activated nucleotide receptor activity
def: "Combining with a nucleotide and transmitting the signal to a heterotrimeric G-protein complex to initiate a change in cell activity, activated by UTP." [GOC:mah]
GO:0045031
name: ATP-activated nucleotide receptor activity
def: "Combining with adenosine and transmitting the signal to a heterotrimeric G-protein complex to initiate a change in cell activity, activated by ATP." [GOC:mah]
GO:0045032
name: ADP-activated nucleotide receptor activity
def: "Combining with adenosine and transmitting the signal to a heterotrimeric G-protein complex to initiate a change in cell activity, activated by ADP." [GOC:mah]
GO:0045093
name: interleukin-18 alpha subunit binding
def: "Interacting selectively and non-covalently with the alpha subunit of interleukin-18. IL-18a is a component of IL-18 that is essential for IL-18 binding on the surface of T-helper 1 cells." [PMID:10653850]
GO:0045094
name: interleukin-18 beta subunit binding
def: "Interacting selectively and non-covalently with the beta subunit of interleukin-18. IL-18b is a ligand non-binding chain and is required for signaling of IL-18 that binds with IL-18a." [PMID:10653850]
GO:0045118
name: azole transporter activity
def: "Enables the directed movement of azoles, heterocyclic compound found in many biologically important substances, into, out of or within a cell, or between cells." [GOC:go_curators, ISBN:3527307206, Wikipedia:Azole]
GO:0045119
name: azole:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: H+(out) + azole(in) = H+(in) + azole(out). Azoles are heterocyclic compounds found in many biologically important substances." [GOC:ai, ISBN:3527307206, Wikipedia:Azole]
GO:0045125
name: bioactive lipid receptor activity
def: "Combining with a bioactive lipid and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex. A bioactive lipid is a lipid for which changes in lipid levels result in functional consequences in a variety of cellular processes." [GOC:bf, GOC:mah, PMID:12215548, PMID:18216770]
GO:0045127
name: N-acetylglucosamine kinase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosamine + ATP = N-acetyl-D-glucosamine 6-phosphate + ADP + 2 H(+)." [EC:2.7.1.59, RHEA:17420]
GO:0045129
name: NAD-independent histone deacetylase activity
def: "Catalysis of the reaction: histone N6-acetyl-L-lysine + H2O = histone L-lysine + acetate. This reaction does not require the presence of NAD, and represents the removal of an acetyl group from a histone." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0045130
name: keratan sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + keratan = adenosine 3',5'-bisphosphate + keratan 6'-sulfate." [EC:2.8.2.21]
GO:0045131
name: pre-mRNA branch point binding
def: "Interacting selectively and non-covalently with a pre-mRNA branch point sequence, located upstream of the 3' splice site." [PMID:11691992, PMID:9722632]
GO:0045133
name: 2,3-dihydroxybenzoate 3,4-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxybenzoate + O(2) = 2-hydroxy-3-(3-oxoprop-1-enyl)but-2-enedioate + H(+)." [EC:1.13.11.14, RHEA:18480]
GO:0045134
name: uridine-diphosphatase activity
def: "Catalysis of the reaction: UDP + H2O = UMP + phosphate." [EC:3.6.1.6]
GO:0045135
name: poly(beta-D-mannuronate) lyase activity
def: "Catalysis of the reaction: polysaccharides containing beta-D-mannuronate residues = oligosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enopyranuronosyl end. This reaction is the eliminative cleavage of polysaccharides containing beta-D-mannuronate residues to give oligosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enopyranuronosyl groups at their ends." [EC:4.2.2.3]
GO:0045140
name: inositol phosphoceramide synthase activity
def: "Catalysis of the reaction: phytoceramide + inositol phosphate = inositol phosphoceramide." [MetaCyc:RXN3O-581, PMID:9405490, PMID:9614099]
GO:0045142
name: triplex DNA binding
def: "Interacting selectively and non-covalently with a DNA triple helix. The formation of triple helical DNA has been evoked in several cellular processes including transcription, replication, and recombination." [PMID:10681538]
GO:0045145
name: single-stranded DNA 5'-3' exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of nucleotides (such as mononucleotides or dinucleotides) from a free 5' terminus of a single-stranded DNA molecule." [GOC:ai, GOC:elh, PMID:20086101]
GO:0045148
name: tripeptide aminopeptidase activity
def: "Catalysis of the release of the N-terminal residue from a tripeptide." [EC:3.4.11.4]
GO:0045152
name: antisigma factor binding
def: "Interacting selectively and non-covalently with an antisigma factor, a factor which inhibits the ability of the sigma factor to function as a transcriptional initiator." [GOC:mlg]
GO:0045153
name: electron transporter, transferring electrons within CoQH2-cytochrome c reductase complex activity
def: "Enables the directed movement of electrons within the CoQH2-cytochrome c reductase complex." [GOC:ai, ISBN:0716731363]
GO:0045154
name: electron transporter, transferring electrons within cytochrome c oxidase complex activity
def: "Enables the directed movement of electrons within the cytochrome c oxidase complex." [GOC:ai, ISBN:0716731363]
GO:0045155
name: electron transporter, transferring electrons from CoQH2-cytochrome c reductase complex and cytochrome c oxidase complex activity
def: "Enables the directed movement of electrons from the CoQH2-cytochrome c reductase complex and the cytochrome c oxidase complex." [GOC:ai, ISBN:0716731363]
GO:0045156
name: electron transporter, transferring electrons within the cyclic electron transport pathway of photosynthesis activity
def: "Enables the directed movement of electrons within the cyclic electron transport pathway of photosynthesis." [GOC:ai, ISBN:0716731363]
GO:0045157
name: electron transporter, transferring electrons within the noncyclic electron transport pathway of photosynthesis activity
def: "Enables the directed movement of electrons within the noncyclic electron transport pathway of photosynthesis." [GOC:ai, ISBN:0716731363]
GO:0045158
name: electron transporter, transferring electrons within cytochrome b6/f complex of photosystem II activity
def: "Enables the directed movement of electrons within the cytochrome b6/f complex of photosystem II." [GOC:ai, ISBN:0716731363]
GO:0045159
name: myosin II binding
def: "Interacting selectively and non-covalently with a class II myosin, any member of the class of 'conventional' double-headed myosins that includes muscle myosin." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html]
GO:0045164
name: secretin (sensu Mammalia)
def: "OBSOLETE. Secretin is a hormone that takes part in the digestion process. It also has effects on organs other than gastrointestinal tract." [PMID:11320551]
GO:0045174
name: glutathione dehydrogenase (ascorbate) activity
def: "Catalysis of the reaction: dehydroascorbate + 2 glutathione = L-ascorbate + glutathione disulfide." [EC:1.8.5.1, RHEA:24427]
GO:0045181
name: glutamate synthase activity, NAD(P)H as acceptor
def: "Catalysis of the reaction: 2 L-glutamate + NAD(P)+ = L-glutamine + 2-oxoglutarate + NAD(P)H + H+." [EC:1.4.1.13, EC:1.4.1.14]
GO:0045182
name: translation regulator activity
def: "Any molecular function involved in the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome." [GOC:ai]
GO:0045183
name: translation factor activity, non-nucleic acid binding
def: "A translation regulator activity that does not involve binding to nucleic acids." [GOC:ai, GOC:dph, GOC:tb]
GO:0045205
name: MAPK transporter activity
def: "OBSOLETE. Enables the directed movement of MAP kinase into, out of or within a cell, or between cells." [GOC:ebc, GOC:mtg_transport, ISBN:0815340729]
GO:0045206
name: MAPK phosphatase transporter activity
def: "OBSOLETE. Enables the directed movement of MAPK phosphatase into, out of or within a cell, or between cells." [GOC:ebc, GOC:mtg_transport, ISBN:0815340729]
GO:0045233
name: natural killer cell receptor activity
def: "OBSOLETE. A receptor found on the surface of natural killer cells which binds Class I MHC antigens and is required for activation of NK activity. It belongs to the Ly49i family." [GOC:ebc]
GO:0045236
name: CXCR chemokine receptor binding
def: "Interacting selectively and non-covalently with a chemokine receptor in the CXCR family." [GOC:ceb, PMID:11910892]
GO:0045237
name: CXCR1 chemokine receptor binding
def: "Interacting selectively and non-covalently with the CXCR1 chemokine receptor." [GOC:ceb, PMID:11910892]
GO:0045238
name: CXCR2 chemokine receptor binding
def: "Interacting selectively and non-covalently with the CXCR2 chemokine receptor." [GOC:ceb, PMID:11910892]
GO:0045289
name: luciferin monooxygenase activity
def: "Catalysis of the generalized reaction: luciferin + O2 = oxidized luciferin + CO2 + light. There may be additional substrates and reactants involved in the reaction. The reaction results in light emission as luciferin returns to the ground state after enzymatic oxidation." [GOC:bf]
GO:0045290
name: D-arabinose 1-dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: D-arabinose + NAD(P)+ = D-arabinono-1,4-lactone + NAD(P)H + H+." [EC:1.1.1.117]
GO:0045294
name: alpha-catenin binding
def: "Interacting selectively and non-covalently with the alpha subunit of the catenin complex." [GOC:bf]
GO:0045295
name: gamma-catenin binding
def: "Interacting selectively and non-covalently with the gamma subunit of the catenin complex." [GOC:bf]
GO:0045296
name: cadherin binding
def: "Interacting selectively and non-covalently with cadherin, a type I membrane protein involved in cell adhesion." [GOC:bf]
GO:0045300
name: acyl-[acyl-carrier-protein] desaturase activity
def: "Catalysis of the reaction: stearoyl-[acyl-carrier protein] + reduced acceptor + O2 = oleoyl-[acyl-carrier protein] + acceptor + H2O. The enzyme requires ferredoxin." [EC:1.14.19.2]
GO:0045301
name: tRNA-(2-methylthio-N-6-(cis-hydroxy)isopentenyl adenosine)-hydroxylase activity
def: "Catalysis of the reaction: tRNA-(2-methylthio-N-6-isopentenyl adenosine) = tRNA-(2-methylthio-N-6-(cis-hydroxy)isopentenyl adenosine) + O2. 2-methylthio-N-6-isopentenyl adenosine is also known as ms2i6A; 2-methylthio-N-6-(cis-hydroxy)isopentenyl adenosine is also known as ms2io6A and 2-methylthio-cis-ribozeatin." [GOC:mlg, PMID:8253666]
GO:0045302
name: choloylglycine hydrolase activity
def: "Catalysis of the reaction: 3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholan-24-oylglycine + H2O = 3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholanate + glycine." [EC:3.5.1.24]
GO:0045303
name: diaminobutyrate-2-oxoglutarate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-2,4-diaminobutyrate = L-aspartate 4-semialdehyde + L-glutamate." [EC:2.6.1.76, RHEA:11163]
GO:0045305
name: regulator of establishment of competence for transformation activity
def: "OBSOLETE. Functions to either promote or inhibit the establishment of competence for transformation." [GOC:mlg]
GO:0045306
name: inhibitor of the establishment of competence for transformation activity
def: "OBSOLETE. Inhibits the establishment of competence for transformation." [GOC:mlg]
GO:0045307
name: activator of the establishment of competence for transformation activity
def: "OBSOLETE. Activates the establishment of competence for transformation." [GOC:mlg]
GO:0045309
name: protein phosphorylated amino acid binding
def: "Interacting selectively and non-covalently with a phosphorylated amino acid residue within a protein." [GOC:go_curators]
GO:0045310
name: phosphoserine/phosphothreonine binding
def: "OBSOLETE. Interacting selectively with a phosphorylated serine or threonine residue within a protein." [GOC:go_curators]
GO:0045322
name: unmethylated CpG binding
def: "Interacting selectively and non-covalently with unmethylated CpG motifs. Unmethylated CpG dinucleotides are often associated with gene promoters." [GOC:ai, PMID:10688657]
GO:0045330
name: aspartyl esterase activity
def: "Catalysis of the hydrolysis of an ester bond by a mechanism involving a catalytically active aspartic acid residue." [GOC:mah, UniProtKB-KW:KW-0063]
GO:0045331
name: coenzyme-M-7-mercaptoheptanoylthreonine-phosphate-heterodisulfide hydrogenase activity
def: "OBSOLETE. Catalysis of the reaction: coenzyme-M 7-mercaptoheptanoylthreonine-phosphate heterodisulfide + H2 = coenzyme-M + N-(7-mercaptoheptanoyl)threonine O3-phosphate." [EC:1.12.99.2]
GO:0045340
name: mercury ion binding
def: "Interacting selectively and non-covalently with mercury (Hg) ions." [GOC:go_curators]
GO:0045352
name: interleukin-1 Type I receptor antagonist activity
def: "Blocks the binding of interleukin-1 to interleukin-1 Type I receptors." [GOC:ebc]
GO:0045353
name: interleukin-1 Type II receptor antagonist activity
def: "Blocks the binding of interleukin-1 to interleukin-1 Type II receptors." [GOC:ebc]
GO:0045430
name: chalcone isomerase activity
def: "Catalysis of the reaction: a chalcone = a flavanone." [EC:5.5.1.6]
GO:0045431
name: flavonol synthase activity
def: "Catalysis of the reaction: a dihydroflavonol + 2-oxoglurate + O2 = a flavonol + succinate + CO2 + H2O." [EC:1.14.11.23, ISBN:0943088372, PMID:7904213]
GO:0045435
name: lycopene epsilon cyclase activity
def: "Catalysis of the cyclization of an epsilon ring at one end of the lycopene molecule (psi, psi-carotene) to form delta-carotene (epsilon, psi-carotene)." [PMID:8837512]
GO:0045436
name: lycopene beta cyclase activity
def: "Catalysis of the cyclization of beta rings at one or both ends of the lycopene molecule (psi, psi-carotene) to form gamma-carotene or the bicyclic beta-carotene (beta, beta-carotene), respectively." [PMID:8837512]
GO:0045437
name: uridine nucleosidase activity
def: "Catalysis of the reaction: H(2)O + uridine = ribofuranose + uracil." [EC:3.2.2.3, RHEA:15580]
GO:0045438
name: delta-(L-alpha-aminoadipyl)-L-cysteinyl-D-valine synthetase activity
def: "Catalysis of the formation of delta-(L-alpha-aminoadipyl)-L-cysteinyl-D-valine from constituent amino acids and ATP in the presence of magnesium ions and dithioerythritol." [PMID:1572368, PMID:2061333]
GO:0045439
name: isopenicillin-N epimerase activity
def: "Catalysis of the reaction: isopenicillin N = penicillin N." [EC:5.1.1.17, RHEA:20036]
GO:0045442
name: deacetoxycephalosporin-C hydroxylase activity
def: "Catalysis of the reaction: 2-oxoglutarate + deacetoxycephalosporin C + O(2) = CO(2) + deacetylcephalosporin C + succinate." [EC:1.14.11.26, RHEA:16808]
GO:0045462
name: trichothecene 3-O-acetyltransferase activity
def: "Catalysis of the 3-O-acetylation of a trichothecene. Trichothecenes are sesquiterpene epoxide mycotoxins that act as potent inhibitors of eukaryotic protein synthesis." [PMID:10583973]
GO:0045480
name: galactose oxidase activity
def: "Catalysis of the reaction: D-galactose + O2 = D-galacto-hexodialdose + hydrogen peroxide." [EC:1.1.3.9]
GO:0045481
name: 6-endo-hydroxycineole dehydrogenase activity
def: "Catalysis of the reaction: 6-endo-hydroxycineole + NAD(+) = 6-oxocineole + H(+) + NADH." [EC:1.1.1.241, RHEA:11739]
GO:0045482
name: trichodiene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = diphosphate + trichodiene." [EC:4.2.3.6, RHEA:12055]
GO:0045483
name: aristolochene synthase activity
def: "Catalysis of the reaction: trans,trans-farnesyl diphosphate = aristolochene + diphosphate." [EC:4.2.3.9]
GO:0045484
name: L-lysine 6-transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-lysine = L-glutamate + allysine." [EC:2.6.1.36, RHEA:21203]
GO:0045485
name: omega-6 fatty acid desaturase activity
def: "Catalysis of the introduction of an omega-6 double bond into the fatty acid hydrocarbon chain." [PMID:7846158]
GO:0045486
name: naringenin 3-dioxygenase activity
def: "Catalysis of the reaction: naringenin + 2-oxoglutarate + O2 = dihydrokaempferol + succinate + CO2." [EC:1.14.11.9]
GO:0045499
name: chemorepellent activity
def: "Providing the environmental signal that initiates the directed movement of a motile cell or organism towards a lower concentration of that signal." [GOC:ai]
GO:0045502
name: dynein binding
def: "Interacting selectively and non-covalently with dynein, the multisubunit protein complex that is associated with microtubules." [GOC:bf]
GO:0045503
name: dynein light chain binding
def: "Interacting selectively and non-covalently with a light chain of the dynein complex." [GOC:bf]
GO:0045504
name: dynein heavy chain binding
def: "Interacting selectively and non-covalently with a heavy chain of the dynein complex." [GOC:bf]
GO:0045505
name: dynein intermediate chain binding
def: "Interacting selectively and non-covalently with an intermediate chain of the dynein complex." [GOC:bf]
GO:0045506
name: interleukin-24 receptor activity
def: "Combining with interleukin-24 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0045507
name: interleukin-25 receptor activity
def: "Combining with interleukin-25 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0045508
name: interleukin-26 receptor activity
def: "Combining with interleukin-26 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0045509
name: interleukin-27 receptor activity
def: "Combining with interleukin-27 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:jl, GOC:signaling]
GO:0045510
name: interleukin-24 binding
def: "Interacting selectively and non-covalently with interleukin-24." [GOC:go_curators]
GO:0045511
name: interleukin-25 binding
def: "Interacting selectively and non-covalently with interleukin-25." [GOC:go_curators]
GO:0045512
name: interleukin-26 binding
def: "Interacting selectively and non-covalently with interleukin-26." [GOC:go_curators]
GO:0045513
name: interleukin-27 binding
def: "Interacting selectively and non-covalently with interleukin-27." [GOC:go_curators]
GO:0045514
name: interleukin-16 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-16 receptor." [GOC:go_curators]
GO:0045515
name: interleukin-18 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-18 receptor." [GOC:go_curators]
GO:0045516
name: interleukin-19 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-19 receptor." [GOC:go_curators]
GO:0045517
name: interleukin-20 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-20 receptor." [GOC:go_curators]
GO:0045518
name: interleukin-22 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-22 receptor." [GOC:go_curators]
GO:0045519
name: interleukin-23 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-23 receptor." [GOC:go_curators]
GO:0045520
name: interleukin-24 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-24 receptor." [GOC:go_curators]
GO:0045521
name: interleukin-25 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-25 receptor." [GOC:go_curators]
GO:0045522
name: interleukin-26 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-26 receptor." [GOC:go_curators]
GO:0045523
name: interleukin-27 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-27 receptor." [GOC:go_curators]
GO:0045543
name: gibberellin 2-beta-dioxygenase activity
def: "Catalysis of the reaction: a gibberellin + 2-oxoglutarate + O2 = a 2-beta-hydroxygibberellin + succinate + CO2." [EC:1.14.11.13, GOC:kad]
GO:0045544
name: gibberellin 20-oxidase activity
def: "Catalysis of the oxidation of C-20 gibberellins to form the corresponding C-19 lactones." [PMID:7604047]
GO:0045545
name: syndecan binding
def: "Interacting selectively and non-covalently with syndecan, an integral membrane proteoglycan (250-300 kDa) associated largely with epithelial cells." [GOC:go_curators, PMID:9355727]
GO:0045547
name: dehydrodolichyl diphosphate synthase activity
def: "Catalysis of the condensation of isopentenyl diphosphate and farnesyl diphosphate in the cis-configuration to form dehydrodolichyl diphosphate." [PMID:9858571]
GO:0045548
name: phenylalanine ammonia-lyase activity
def: "Catalysis of the reaction: L-phenylalanine = NH(4)(+) + trans-cinnamate." [RHEA:21387]
GO:0045549
name: 9-cis-epoxycarotenoid dioxygenase activity
def: "Catalysis of the reactions: a 9-cis-epoxycarotenoid + O2 = 2-cis,4-trans-xanthoxin + a 12'-apo-carotenal; 9-cis-violaxanthin + O2 = 2-cis,4-trans-xanthoxin + (3S,5R,6S)-5,6-epoxy-3-hydroxy-5,6-dihydro-12'-apo-beta-caroten-12'-al; and 9'-cis-neoxanthin + O2 = 2-cis,4-trans-xanthoxin + (3S,5R,6R)-5,6-dihydroxy-6,7-didehydro-5,6-dihydro-12'-apo-beta-caroten-12'-al." [EC:1.13.11.51]
GO:0045550
name: geranylgeranyl reductase activity
def: "Catalysis of the formation of phytyl group from the stepwise reduction of a geranylgeranyl group." [PMID:9492312]
GO:0045551
name: cinnamyl-alcohol dehydrogenase activity
def: "Catalysis of the reaction: cinnamyl alcohol + NADP+ = cinnamaldehyde + NADPH + H+." [EC:1.1.1.195]
GO:0045552
name: dihydrokaempferol 4-reductase activity
def: "Catalysis of the reaction: cis-3,4-leucopelargonidin + NADP+ = (+)-dihydrokaempferol + NADPH + H+." [EC:1.1.1.219]
GO:0045569
name: TRAIL binding
def: "Interacting selectively and non-covalently with TRAIL (TNF-related apoptosis inducing ligand), a member of the tumor necrosis factor ligand family that rapidly induces apoptosis in a variety of transformed cell lines." [GOC:go_curators, PMID:9082980]
GO:0045703
name: ketoreductase activity
def: "Catalysis of the reduction of a ketone group to form the corresponding alcohol." [EC:1.1.-.-]
GO:0045735
name: nutrient reservoir activity
def: "Functions in the storage of nutritious substrates." [GOC:ai]
GO:0046025
name: precorrin-6Y C5,15-methyltransferase (decarboxylating) activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + precorrin-6Y = 2 S-adenosyl-L-homocysteine + precorrin-8X + CO2." [EC:2.1.1.132]
GO:0046026
name: precorrin-4 C11-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + precorrin-4 = S-adenosyl-L-homocysteine + precorrin 5." [EC:2.1.1.133]
GO:0046027
name: phospholipid:diacylglycerol acyltransferase activity
def: "Catalysis of the reaction: phospholipid + 1,2-diacylglycerol = lysophospholipid + triacylglycerol." [EC:2.3.1.158]
GO:0046028
name: electron transporter, transferring electrons from cytochrome b6/f complex of photosystem II activity
def: "Enables the directed movement of electrons from the cytochrome b6/f complex of photosystem II." [GOC:ai, ISBN:0716731363]
GO:0046029
name: mannitol dehydrogenase activity
def: "Catalysis of the reaction: D-mannitol + NAD(+) = D-mannose + H(+) + NADH." [EC:1.1.1.255, RHEA:15032]
GO:0046030
name: inositol trisphosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol trisphosphate + H2O = myo-inositol bisphosphate + phosphate." [GOC:bf]
GO:0046316
name: gluconokinase activity
def: "Catalysis of the reaction: D-gluconate + ATP = 6-phospho-D-gluconate + ADP + 2 H(+)." [EC:2.7.1.12, RHEA:19436]
GO:0046332
name: SMAD binding
def: "Interacting selectively and non-covalently with a SMAD signaling protein." [GOC:ai]
GO:0046353
name: aminoglycoside 3-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + aminoglycoside = CoA + 3-N-acetylaminoglycoside. This is acetylation of the 3-amino group of the central deoxystreptamine ring." [GOC:cb]
GO:0046403
name: polynucleotide 3'-phosphatase activity
def: "Catalysis of the reaction: 3'-phosphopolynucleotide + H2O = a polynucleotide + phosphate. Hydrolyzes the free 3'-phosphate resulting from single strand breaks in DNA due to oxidative damage." [EC:3.1.3.32]
GO:0046404
name: ATP-dependent polydeoxyribonucleotide 5'-hydroxyl-kinase activity
def: "Catalysis of the reaction: ATP + 5'-dephospho-DNA = ADP + 5'-phospho-DNA." [EC:2.7.1.78]
GO:0046405
name: glycerol dehydratase activity
def: "Catalysis of the reaction: glycerol = 3-hydroxypropanal + H2O." [EC:4.2.1.30, PMID:18307109]
GO:0046406
name: magnesium protoporphyrin IX methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + magnesium protoporphyrin IX = S-adenosyl-L-homocysteine + H(+) + magnesium protoporphyrin IX 13-monomethyl ester." [EC:2.1.1.11, RHEA:17812]
GO:0046408
name: chlorophyll synthetase activity
def: "Catalysis of the reaction: chlorophyllide a + 2 H(+) + phytyl diphosphate = chlorophyll a + diphosphate." [EC:2.5.1.62, RHEA:17320]
GO:0046409
name: p-coumarate 3-hydroxylase activity
def: "Catalysis of the reaction: shikimate or quinate ester of p-coumaric acid + NADPH + H+ + O2 = caffeic acid conjugate (caffeoyl shikimic acid or chlorogenic acid) + H2O + NADP+." [PMID:11429408, PMID:11891223]
GO:0046410
name: 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase activity
def: "OBSOLETE. Catalysis of the reaction: 2-oxoglutarate + isochorismate = 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate + pyruvate + CO2." [EC:2.5.1.64, PMID:1459959]
GO:0046421
name: methylisocitrate lyase activity
def: "Catalysis of the reaction: (2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = pyruvate + succinate." [EC:4.1.3.30, RHEA:16812]
GO:0046422
name: violaxanthin de-epoxidase activity
def: "Catalysis of the reaction: violaxanthin + 2 ascorbate = zeaxanthin + 2 dehydroascorbate + 2 H2O; and antheraxanthin + ascorbate = zeaxanthin + dehydroascorbate + H2O." [EC:1.10.99.3, GOC:ai, ISBN:0471331309]
GO:0046423
name: allene-oxide cyclase activity
def: "Catalysis of the reaction: (9Z,13S,15Z)-12,13-epoxyoctadeca-9,11,15-trienoate = (15Z)-12-oxophyto-10,15-dienoate." [EC:5.3.99.6, RHEA:22595]
GO:0046424
name: ferulate 5-hydroxylase activity
def: "Catalysis of the reaction: ferulic acid + NADPH + H+ + O2 = 5-hydroxyferulic acid + H2O + NADP+." [PMID:8692910, PMID:9880351]
GO:0046428
name: 1,4-dihydroxy-2-naphthoate octaprenyltransferase activity
def: "Catalysis of the reaction: 1,4-dihydroxy-2-naphthoate + polyprenylpyrophosphate = dimethylmenaquinone + diphosphate + CO2." [MetaCyc:DMK-RXN]
GO:0046429
name: 4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase activity
def: "Catalysis of the reaction: (E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate + H2O + 2 oxidized ferredoxin = 2-C-methyl-D-erythritol 2,4-cyclodiphosphate + 2 reduced ferredoxin." [EC:1.17.7.1, PMID:11752431]
GO:0046480
name: galactolipid galactosyltransferase activity
def: "Catalysis of the reaction: 2 mono-beta-D-galactosyldiacylglycerol = alpha-D-galactosyl-beta-D-galactosyldiacylglycerol + 1,2-diacylglycerol." [EC:2.4.1.184]
GO:0046481
name: digalactosyldiacylglycerol synthase activity
def: "Catalysis of the reaction: 1,2-diacyl-3-beta-D-galactosyl-sn-glycerol + UDP-D-galactose = 3-[alpha-D-galactosyl-(1->6)-beta-D-galactosyl]-1,2-diacyl-sn-glycerol + H(+) + UDP." [EC:2.4.1.241, RHEA:10523]
GO:0046507
name: UDPsulfoquinovose synthase activity
def: "Catalysis of the reaction: sulfite + UDP-D-glucose = H(2)O + UDP-6-sulfoquinovose." [EC:3.13.1.1, RHEA:13200]
GO:0046508
name: hydrolase activity, acting on carbon-sulfur bonds
def: "Catalysis of the hydrolysis of any carbon-sulfur bond, C-S." [GOC:jl]
GO:0046509
name: 1,2-diacylglycerol 3-beta-galactosyltransferase activity
def: "Catalysis of the reaction: 1,2-diacyl-sn-glycerol + UDP-D-galactose = 1,2-diacyl-3-beta-D-galactosyl-sn-glycerol + H(+) + UDP." [EC:2.4.1.46, RHEA:14948]
GO:0046510
name: UDP-sulfoquinovose:DAG sulfoquinovosyltransferase activity
def: "Catalysis of the reaction: UDP-sulfoquinovose + 1,2-diacylglycerol = sulfoquinovosyldiacylglycerol + UDP." [MetaCyc:RXN-1224]
GO:0046522
name: S-methyl-5-thioribose kinase activity
def: "Catalysis of the reaction: S-methyl-5-thio-D-ribose + ATP = S-methyl-5-thio-alpha-D-ribose 1-phosphate + ADP + 2 H(+)." [EC:2.7.1.100, RHEA:22315]
GO:0046523
name: S-methyl-5-thioribose-1-phosphate isomerase activity
def: "Catalysis of the reaction: S-methyl-5-thio-alpha-D-ribose 1-phosphate = S-methyl-5-thio-D-ribulose 1-phosphate." [EC:5.3.1.23, RHEA:19992]
GO:0046524
name: sucrose-phosphate synthase activity
def: "Catalysis of the reaction: UDP-glucose + D-fructose 6-phosphate = UDP + sucrose 6-phosphate." [EC:2.4.1.14]
GO:0046525
name: xylosylprotein 4-beta-galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + O-beta-D-xylosylprotein = UDP + 4-beta-D-galactosyl-O-beta-D-xylosylprotein." [EC:2.4.1.133]
GO:0046526
name: D-xylulose reductase activity
def: "Catalysis of the reaction: NAD(+) + xylitol = D-xylulose + H(+) + NADH." [EC:1.1.1.9, RHEA:20436]
GO:0046527
name: glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [ISBN:0198506732]
GO:0046537
name: 2,3-bisphosphoglycerate-independent phosphoglycerate mutase activity
def: "Catalysis of the reaction: 2-phospho-D-glycerate = 3-phospho-D glycerate; this reaction does not require the cofactor 2,3-bisphosphoglycerate." [EC:5.4.2.1]
GO:0046538
name: 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase activity
def: "Catalysis of the reaction: 2-phospho-D-glycerate = 3-phospho-D-glycerate; this reaction requires the cofactor 2,3-bisphosphoglycerate." [EC:5.4.2.1]
GO:0046539
name: histamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + histamine = N(tau)-methylhistamine + S-adenosyl-L-homocysteine + H(+)." [EC:2.1.1.8, RHEA:19304]
GO:0046547
name: trans-aconitate 3-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + trans-aconitate = (E)-2-(methoxycarbonylmethyl)but-2-enedioate + S-adenosyl-L-homocysteine." [EC:2.1.1.145, RHEA:22203]
GO:0046553
name: D-malate dehydrogenase (decarboxylating) activity
def: "Catalysis of the reaction: (R)-malate + NAD(+) = CO(2) + NADH + pyruvate." [EC:1.1.1.83, RHEA:18368]
GO:0046554
name: malate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: (S)-malate + NADP+ = oxaloacetate + NADPH + H+." [EC:1.1.1.82]
GO:0046555
name: acetylxylan esterase activity
def: "Catalysis of the deacetylation of xylans and xylo-oligosaccharides." [EC:3.1.1.72]
GO:0046556
name: alpha-L-arabinofuranosidase activity
def: "Catalysis of the hydrolysis of terminal non-reducing alpha-L-arabinofuranoside residues in alpha-L-arabinosides." [EC:3.2.1.55, GOC:mf]
GO:0046557
name: glucan endo-1,6-beta-glucosidase activity
def: "Catalysis of the random hydrolysis of (1->6) linkages in (1->6)-beta-D-glucans." [EC:3.2.1.75]
GO:0046558
name: arabinan endo-1,5-alpha-L-arabinosidase activity
def: "Catalysis of the endohydrolysis of (1->5)-alpha-arabinofuranosidic linkages in (1->5) arabinans." [EC:3.2.1.99]
GO:0046559
name: alpha-glucuronidase activity
def: "Catalysis of the reaction: an alpha-D-glucuronoside + H2O = an alcohol + D-glucuronate." [EC:3.2.1.139]
GO:0046560
name: scytalidopepsin B activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins with broad specificity, cleaving Phe24-Phe, but not Leu15-Tyr and Phe25-Tyr in the B chain of insulin." [EC:3.4.23.32]
GO:0046561
name: penicillopepsin activity
def: "OBSOLETE. Catalysis of the hydrolysis of proteins with broad specificity similar to that of pepsin A, preferring hydrophobic residues at P1 and P1', but also cleaving Gly20-Glu in the B chain of insulin. Clots milk, and activates trypsinogen." [EC:3.4.23.20]
GO:0046562
name: glucose oxidase activity
def: "Catalysis of the reaction: beta-D-glucose + O2 = D-glucono-1,5-lactone + H2O2." [EC:1.1.3.4, GOC:mah]
GO:0046563
name: methanol oxidase activity
def: "Catalysis of the reaction: O2 + methanol = H2O2 + formaldehyde." [MetaCyc:METHANOL-OXIDASE-RXN]
GO:0046564
name: oxalate decarboxylase activity
def: "Catalysis of the reaction: H(+) + oxalate = CO(2) + formate." [EC:4.1.1.2, RHEA:16512]
GO:0046565
name: 3-dehydroshikimate dehydratase activity
def: "Catalysis of the reaction: 3-dehydroshikimate = 3,4-dihydroxybenzoate + H2O. 3,4-dihydroxybenzoate is also known as protocatechuate." [EC:4.2.1.-, MetaCyc:DHSHIKIMATE-DEHYDRO-RXN]
GO:0046566
name: DOPA dioxygenase activity
def: "Catalysis of the 4,5-ring opening reaction: 3,4-dihydroxyphenylalanine + O2 = 4,5-seco-DOPA. 4,5-seco-DOPA spontaneously recyclizes to form betalamic acid." [PMID:11711071]
GO:0046567
name: aphidicolan-16 beta-ol synthase activity
def: "Catalysis of the reaction: 9-alpha-copalyl diphosphate + H2O = aphidicolan-16-beta-ol + diphosphate." [EC:4.2.3.42, PMID:12149019]
GO:0046568
name: 3-methylbutanol:NAD(P) oxidoreductase activity
def: "Catalysis of the reaction: 3-methylbutanol + NAD(P)+ = 3-methylbutanal + NAD(P)H + H+. 3-methylbutanal is also known as isovaleraldehyde." [EC:1.1.1.265]
GO:0046569
name: glyoxal oxidase activity
def: "Catalysis of the reaction: glyoxal + O2 + H2O = glyoxalate + H2O2." [EC:1.2.3.-, PMID:11733005]
GO:0046570
name: methylthioribulose 1-phosphate dehydratase activity
def: "Catalysis of the reaction: S-methyl-5-thio-D-ribulose 1-phosphate = 5-(methylthio)-2,3-dioxopentyl phosphate + H(2)O." [EC:4.2.1.109, RHEA:15552]
GO:0046571
name: aspartate-2-keto-4-methylthiobutyrate transaminase activity
def: "Catalysis of the reaction: 2-keto-4-methylthiobutyrate + aspartate = methionine + oxaloacetate." [MetaCyc:R15-RXN]
GO:0046572
name: versicolorin B synthase activity
def: "Catalysis of the reaction: versiconal = versicolorin B + H2O." [MetaCyc:RXN-9494, PMID:8784203]
GO:0046573
name: lactonohydrolase activity
def: "Catalysis of the hydrolysis of lactone rings (intramolecular cyclic esters) to produce a hydroxyl group and a carboxyl group." [PMID:11640988]
GO:0046574
name: glycuronidase activity
def: "Catalysis of the hydrolysis of glucuronosides, yielding free glucuronic acid." [PMID:10441389, PMID:12044176]
GO:0046575
name: rhamnogalacturonan acetylesterase activity
def: "Catalysis of the removal of acetylesters (as acetate) from galacturonic acid residues in the backbone of rhamnogalacturonan." [PMID:10801485]
GO:0046576
name: rhamnogalacturonan alpha-L-rhamnopyranosyl-(1->4)-alpha-D-galactopyranosyluronide lyase activity
def: "Catalysis of the cleavage of rhamnogalacturonan, generating oligosaccharides of the form alpha-D-us-galacturonic acid-(1,2)-alpha-L-rhamnose-(1,4)-alpha-D-galacturonate-(1,2)-L-rhamnose-(1,2)-alpha-L-rhamnose-p-(1,4)-alpha-D-galacturonic acid, terminating at the non-reducing end with a hex-4-enopyranosyluronic acid residue." [PMID:8587995, PMID:8720076]
GO:0046577
name: long-chain-alcohol oxidase activity
def: "Catalysis of the reaction: 2 long-chain alcohol + O2 = 2 long-chain aldehyde + 2 H2O." [EC:1.1.3.20]
GO:0046582
name: Rap GTPase activator activity
def: "Increases the rate of GTP hydrolysis by a GTPase of the Rap family." [GOC:mah]
GO:0046583
name: cation efflux transmembrane transporter activity
def: "Catalysis of the transfer of a cation or cations from the inside of the cell to the outside of the cell across a membrane." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0046589
name: ribonuclease T1 activity
def: "Catalysis of the endonucleolytic cleavage to nucleoside 3'-phosphates and 3'-phosphooligonucleotides ending in Gp with 2',3'-cyclic phosphate intermediates." [EC:3.1.27.3]
GO:0046592
name: polyamine oxidase activity
def: "Catalysis of the oxidative degradation or interconversion of polyamines." [EC:1.5.3.11, PMID:1567380]
GO:0046593
name: mandelonitrile lyase activity
def: "Catalysis of the reaction: mandelonitrile = cyanide + benzaldehyde." [EC:4.1.2.10]
GO:0046608
name: carotenoid isomerase activity
def: "Catalysis of the isomerization of poly-cis-carotenoids to all-trans-carotenoids." [PMID:11884677]
GO:0046609
name: voltage-gated sulfate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: sulfate(out) + solute(in) = solute(in) + sulfate(out), by a channel in a cell membrane whose opening is governed by the membrane potential." [GOC:mah]
GO:0046623
name: sphingolipid-translocating ATPase activity
def: "Catalysis of the movement of sphingolipids from one membrane face to the other ('flippase' activity), driven by the hydrolysis of ATP." [GOC:ai, PMID:12034738]
GO:0046624
name: sphingolipid transporter activity
def: "Enables the directed movement of sphingolipids into, out of or within a cell, or between cells. Sphingolipids are a class of lipids containing the long-chain amine diol sphingosine or a closely related base (a sphingoid)." [GOC:ai, ISBN:0198506732]
GO:0046625
name: sphingolipid binding
def: "Interacting selectively and non-covalently with sphingolipids, a class of lipids containing the long-chain amine diol sphingosine or a closely related base (a sphingoid)." [ISBN:0198506732]
GO:0046659
name: digestive hormone activity
def: "The action characteristic of a hormone that takes part in the digestion process." [GOC:ai]
GO:0046702
name: galactoside 6-L-fucosyltransferase activity
def: "Catalysis of the transfer of an L-fucosyl group from GDP-beta-L-fucose to a galactoside acceptor molecule, usually an N-glycan, to form an alpha(1,6)-fucosylated galactoside." [PMID:12413479]
GO:0046703
name: natural killer cell lectin-like receptor binding
def: "Interacting selectively and non-covalently with a lectin-like natural killer cell receptor." [GOC:ai]
GO:0046714
name: borate binding
def: "Interacting selectively and non-covalently with borate, the anion (BO3)3-." [CHEBI:22908, GOC:curators]
GO:0046715
name: borate transmembrane transporter activity
def: "Catalysis of the transport of borate across a membrane against the concentration gradient." [PMID:12447444]
GO:0046789
name: host cell surface receptor binding
def: "Interacting selectively and non-covalently with a receptor on the host cell surface." [GOC:ai, PMID:11511370]
GO:0046790
name: virion binding
def: "Interacting selectively and non-covalently with a virion, either by binding to components of the capsid or the viral envelope." [GOC:ai]
GO:0046810
name: host cell extracellular matrix binding
def: "Interacting selectively and non-covalently with the extracellular matrix of a host cell." [PMID:7996163]
GO:0046811
name: histone deacetylase inhibitor activity
def: "Stops, prevents or reduces the activity of histone deacetylase, which catalyzes of the removal of acetyl groups from histones, proteins complexed to DNA in chromatin and chromosomes." [GOC:ai, PMID:10482575]
GO:0046812
name: host cell surface binding
def: "Interacting selectively and non-covalently with the surface of a host cell." [GOC:ai]
GO:0046817
name: chemokine receptor antagonist activity
def: "Interacts with chemokine receptors to reduce the action of a chemokine." [GOC:ai, ISBN:0781718325]
GO:0046820
name: 4-amino-4-deoxychorismate synthase activity
def: "Catalysis of the reaction: L-glutamine + chorismate = 4-amino-4-deoxychorismate + L-glutamate. It is composed of two enzymatic activities (which may be present on one or two polypeptides); the first is a glutaminase which yields ammonia from glutamine, releasing glutamate. The ammonia is used by the second activity which catalyzes the amination of chorismate to form 4-amino-4-deoxychorismate." [EC:2.6.1.85, RHEA:11675]
GO:0046848
name: hydroxyapatite binding
def: "Interacting selectively and non-covalently with hydroxyapatite, the calcium phosphate mineral of formula Ca10(PO4)6(OH)2 found both in rocks of nonorganic origin and as a component of bone and dentin." [CHEBI:52255, GOC:curators]
GO:0046857
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, with NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:jl]
GO:0046863
name: ribulose-1,5-bisphosphate carboxylase/oxygenase activator activity
def: "Increases the activity of rubisco by the removal of otherwise inhibitory sugar phosphates: RuBP, and in some plants, 2-carboxyarabinitol 1-phosphate." [PMID:10430961, PMID:10965036, PMID:2404515]
GO:0046870
name: cadmium ion binding
def: "Interacting selectively and non-covalently with cadmium (Cd) ions." [GOC:ai]
GO:0046871
name: N-acetylgalactosamine binding
def: "Interacting selectively and non-covalently with N-acetylgalactosamine, 2-acetamido-2-deoxygalactopyranose, the n-acetyl derivative of galactosamine." [CHEBI:28800, GOC:ai]
GO:0046872
name: metal ion binding
def: "Interacting selectively and non-covalently with any metal ion." [GOC:ai]
GO:0046873
name: metal ion transmembrane transporter activity
def: "Catalysis of the transfer of metal ions from one side of a membrane to the other." [GOC:ai]
GO:0046875
name: ephrin receptor binding
def: "Interacting selectively and non-covalently with an ephrin receptor." [GOC:ai]
GO:0046876
name: 3,4-didehydroretinal binding
def: "Interacting selectively and non-covalently with 3,4-didehydroretinal, a form of retinal that plays a role in the visual process in freshwater fish and some amphibians analogous to that of all-trans retinal in other vertebrates. 3,4-didehydro-11-cis-retinal combines with an opsin to form cyanopsin (cone) or porphyropsin (rod)." [GOC:ai, ISBN:0198506732]
GO:0046899
name: nucleoside triphosphate adenylate kinase activity
def: "Catalysis of the reaction: nucleoside triphosphate + AMP = nucleoside diphosphate + ADP." [EC:2.7.4.10]
GO:0046904
name: calcium oxalate binding
def: "Interacting selectively and non-covalently with calcium oxalate, CaC2O4, a salt of oxalic acid. In animals, it may be excreted in urine or retained in the form of urinary calculi." [ISBN:0721662544]
GO:0046905
name: phytoene synthase activity
def: "Catalysis of the reaction: prephytoene pyrophosphate = phytoene + diphosphate." [GOC:ai, PMID:12641468]
GO:0046906
name: tetrapyrrole binding
def: "Interacting selectively and non-covalently with a tetrapyrrole, a compound containing four pyrrole nuclei variously substituted and linked to each other through carbons at the alpha position." [CHEBI:26932, GOC:curators, ISBN:0198506732]
GO:0046910
name: pectinesterase inhibitor activity
def: "Stops, prevents or reduces the activity of any pectinesterase enzyme." [GOC:ai, PMID:10880981]
GO:0046911
name: metal chelating activity
def: "The formation of bonds from two or more atoms within the same ligand to a metal atom in complexes in which the metal is part of a ring." [ISBN:0198506732, ISBN:0716731363]
GO:0046912
name: transferase activity, transferring acyl groups, acyl groups converted into alkyl on transfer
def: "Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor), with the acyl group being converted into alkyl on transfer." [GOC:jl]
GO:0046914
name: transition metal ion binding
def: "Interacting selectively and non-covalently with a transition metal ions; a transition metal is an element whose atom has an incomplete d-subshell of extranuclear electrons, or which gives rise to a cation or cations with an incomplete d-subshell. Transition metals often have more than one valency state. Biologically relevant transition metals include vanadium, manganese, iron, copper, cobalt, nickel, molybdenum and silver." [ISBN:0198506732]
GO:0046915
name: transition metal ion transmembrane transporter activity
def: "Catalysis of the transfer of transition metal ions from one side of a membrane to the other. A transition metal is an element whose atom has an incomplete d-subshell of extranuclear electrons, or which gives rise to a cation or cations with an incomplete d-subshell. Transition metals often have more than one valency state. Biologically relevant transition metals include vanadium, manganese, iron, copper, cobalt, nickel, molybdenum and silver." [ISBN:0198506732]
GO:0046917
name: triphosphoribosyl-dephospho-CoA synthase activity
def: "Catalysis of the reaction: ATP + 3-dephospho-CoA = 2'-(5''-triphosphoribosyl)-3'-dephospho-CoA + adenine." [EC:2.7.8.25]
GO:0046919
name: pyruvyltransferase activity
def: "Catalysis of the transfer of a pyruvyl (oxopropanoyl) group from one compound to another." [GOC:ai]
GO:0046920
name: alpha-(1->3)-fucosyltransferase activity
def: "Catalysis of the transfer of an L-fucosyl group from GDP-beta-L-fucose to an acceptor molecule to form an alpha-(1->6) linkage." [GOC:ai]
GO:0046921
name: alpha-(1->6)-fucosyltransferase activity
def: "Catalysis of the transfer of an L-fucosyl group from GDP-beta-L-fucose to an acceptor molecule to form an alpha-(1->6) linkage." [GOC:ai]
GO:0046922
name: peptide-O-fucosyltransferase activity
def: "Catalysis of the transfer of an alpha-L-fucosyl residue from GDP- beta-L-fucose to the serine hydroxy group of a protein acceptor." [EC:2.4.1.221]
GO:0046923
name: ER retention sequence binding
def: "Interacting selectively and non-covalently with an endoplasmic reticulum (ER) retention sequence, a specific peptide sequence that ensures a protein is retained within the ER." [GOC:ai]
GO:0046932
name: sodium-transporting ATP synthase activity, rotational mechanism
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ADP + phosphate + Na+(out) = ATP + H2O + Na+(in), by a rotational mechanism." [EC:3.6.3.15, TC:3.A.2.1.2]
GO:0046933
name: proton-transporting ATP synthase activity, rotational mechanism
def: "Catalysis of the transfer of protons from one side of a membrane to the other according to the reaction: ADP + H2O + phosphate + H+(in) = ATP + H+(out), by a rotational mechanism." [EC:3.6.3.14, TC:3.A.2.1.1]
GO:0046934
name: phosphatidylinositol-4,5-bisphosphate 3-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + ATP = a 1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate + ADP + 2 H(+)." [EC:2.7.1.153, RHEA:21295]
GO:0046935
name: 1-phosphatidylinositol-3-kinase regulator activity
def: "Modulates the activity of the enzyme 1-phosphatidylinositol-3-kinase activity." [GOC:ai]
GO:0046936
name: deoxyadenosine deaminase activity
def: "Catalysis of the reaction: deoxyadenosine + H2O = deoxyinosine + NH3." [EC:3.5.4.4, GOC:ai]
GO:0046941
name: azetidine-2-carboxylic acid acetyltransferase activity
def: "Catalysis of the reaction: L-azetidine-2-carboxylic acid + acetyl-CoA = CoA-SH + N-acetyl azetidine-2-carboxylic acid." [PMID:12761200]
GO:0046943
name: carboxylic acid transmembrane transporter activity
def: "Catalysis of the transfer of carboxylic acids from one side of the membrane to the other. Carboxylic acids are organic acids containing one or more carboxyl (COOH) groups or anions (COO-)." [GOC:ai]
GO:0046961
name: proton-transporting ATPase activity, rotational mechanism
def: "Catalysis of the transfer of protons from one side of a membrane to the other according to the reaction: ATP + H2O + H+(in) = ADP + phosphate + H+(out), by a rotational mechanism." [EC:3.6.3.14]
GO:0046962
name: sodium-transporting ATPase activity, rotational mechanism
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Na+(in) = ADP + phosphate + Na+(out), by a rotational mechanism." [EC:3.6.3.15]
GO:0046964
name: 3'-phosphoadenosine 5'-phosphosulfate transmembrane transporter activity
def: "Catalysis of the transfer of 3'-phosphoadenosine 5'-phosphosulfate, a naturally occurring mixed anhydride synthesized from adenosine 5'-phosphosulfate, from one side of a membrane to the other." [ISBN:0198506732]
GO:0046965
name: retinoid X receptor binding
def: "Interacting selectively and non-covalently with a retinoid X receptor." [GOC:ai]
GO:0046966
name: thyroid hormone receptor binding
def: "Interacting selectively and non-covalently with a thyroid hormone receptor." [GOC:ai]
GO:0046969
name: NAD-dependent histone deacetylase activity (H3-K9 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 9) + H2O = histone H3 L-lysine (position 9) + acetate. This reaction requires the presence of NAD, and represents the removal of an acetyl group from lysine at position 9 of the histone H3 protein." [EC:3.5.1.17, EC:3.5.1.98, RHEA:24551]
GO:0046970
name: NAD-dependent histone deacetylase activity (H4-K16 specific)
def: "Catalysis of the reaction: histone H4 N6-acetyl-L-lysine (position 16) + H2O = histone H4 L-lysine (position 16) + acetate. This reaction requires the presence of NAD, and represents the removal of an acetyl group from lysine at position 16 of the histone H4 protein." [EC:3.5.1.17, EC:3.5.1.98, GOC:vw, RHEA:24551]
GO:0046972
name: histone acetyltransferase activity (H4-K16 specific)
def: "Catalysis of the reaction: acetyl-CoA + histone H4 L-lysine (position 16) = CoA + histone H4 N6-acetyl-L-lysine (position 16). This reaction represents the addition of an acetyl group to the lysine at position 16 of histone H4." [EC:2.3.1.48]
GO:0046973
name: histone lysine N-methyltransferase activity (H3-K24 specific)
def: "OBSOLETE. Catalysis of the addition of a methyl group onto lysine at position 24 of the histone H3 protein." [GOC:ai]
GO:0046974
name: histone methyltransferase activity (H3-K9 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 9) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 9). This reaction is the addition of a methyl group onto lysine at position 9 of the histone H3 protein." [GOC:ai]
GO:0046975
name: histone methyltransferase activity (H3-K36 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 36) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 36). This reaction is the addition of a methyl group onto lysine at position 36 of the histone H3 protein." [GOC:ai]
GO:0046976
name: histone methyltransferase activity (H3-K27 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + histone H3 L-lysine (position 27) = S-adenosyl-L-homocysteine + histone H3 N6-methyl-L-lysine (position 27). This reaction is the addition of a methyl group onto lysine at position 27 of the histone H3 protein." [GOC:ai]
GO:0046977
name: TAP binding
def: "Interacting selectively and non-covalently with TAP protein, transporter associated with antigen processing protein. TAP protein is a heterodimeric peptide transporter consisting of the subunits TAP1 and TAP2." [PMID:11133832]
GO:0046978
name: TAP1 binding
def: "Interacting selectively and non-covalently with the TAP1 subunit of TAP (transporter associated with antigen processing) protein." [PMID:11133832]
GO:0046979
name: TAP2 binding
def: "Interacting selectively and non-covalently with the TAP2 subunit of TAP (transporter associated with antigen processing) protein." [PMID:11133832]
GO:0046980
name: tapasin binding
def: "Interacting selectively and non-covalently with tapasin, a member of the MHC class I loading complex which bridges the TAP peptide transporter to class I molecules." [PMID:12594855]
GO:0046981
name: beta-1,4-mannosylglycolipid beta-1,3-N-acetylglucosaminyltransferase activity
def: "Catalysis of the transfer of N-acetylglucosamine (GlcNAc) in a beta-1,3 linkage to the mannose(beta-1,4)Glc disaccharide core of glycolipids." [GOC:bf, PMID:12130631, PMID:12130651]
GO:0046982
name: protein heterodimerization activity
def: "Interacting selectively and non-covalently with a nonidentical protein to form a heterodimer." [GOC:ai]
GO:0046983
name: protein dimerization activity
def: "The formation of a protein dimer, a macromolecular structure consists of two noncovalently associated identical or nonidentical subunits." [ISBN:0198506732]
GO:0046987
name: N-acetyllactosamine beta-1,3-glucuronosyltransferase activity
def: "Catalysis of the transfer, in a beta 1,3 linkage, of D-glucuronic acid (GlcUA) from UDP-D-glucuronic acid to N-acetyllactosamine (galactosyl beta-1,4-N-acetylglucosamine)." [GOC:bf, PMID:12511570]
GO:0046988
name: asioloorosomucoid beta-1,3-glucuronosyltransferase activity
def: "Catalysis of the transfer, in a beta 1,3 linkage, of D-glucuronic acid (GlcUA) from UDP-GlcUA to asioloorosomucoid." [GOC:bf, PMID:12511570]
GO:0046989
name: galactosyl beta-1,3 N-acetylgalactosamine beta-1,3-glucuronosyltransferase activity
def: "Catalysis of the transfer, in a beta 1,3 linkage, of D-glucuronic acid (GlcUA) from UDP-GlcUA to the disaccharide galactosyl beta-1,3 N-acetylgalactosamine, a common component of glycoproteins and glycolipids." [GOC:bf, PMID:12511570]
GO:0046990
name: N-hydroxyarylamine O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an N-hydroxyarylamine = CoA + an N-acetoxyarylamine." [EC:2.3.1.118, MetaCyc:2.3.1.118-RXN]
GO:0046992
name: oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond
def: "Catalysis of an oxidation-reduction (redox) reaction in which X-H and Y-H form X-Y." [GOC:ai]
GO:0046993
name: oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond, with oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which X-H and Y-H form X-Y and the acceptor is oxygen." [GOC:ai]
GO:0046994
name: oxidoreductase activity, acting on hydrogen as donor, with a quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen acts as an electron donor and reduces quinone or similar compound." [GOC:jl]
GO:0046995
name: oxidoreductase activity, acting on hydrogen as donor, with other known acceptors
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen reduces a known acceptor other than a cytochrome, an iron-sulfur protein, NAD, NADP, or a quinone or similar compound." [GOC:ai]
GO:0046996
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, with NAD(P)H as one donor, and the other dehydrogenated
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from NADH or NADPH and one other donor, and the latter donor is dehydrogenated." [GOC:mah]
GO:0046997
name: oxidoreductase activity, acting on the CH-NH group of donors, flavin as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH group acts as a hydrogen or electron donor and reduces a flavin." [GOC:jl]
GO:0046998
name: (S)-usnate reductase activity
def: "Catalysis of the reaction: (6R)-2-acetyl-6-(3-acetyl-2,4,6-trihydroxy-5-methylphenyl)-3-hydroxy-6-methylcyclohexa-2,4-dien-1-one + NAD(+) = (S)-usnate + 2 H(+) + NADH." [EC:1.1.1.199, RHEA:21879]
GO:0047000
name: 2-dehydro-3-deoxy-D-gluconate 6-dehydrogenase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-gluconate + NADP(+) = (4S,5S)-4,5-dihydroxy-2,6-dioxohexanoate + H(+) + NADPH." [EC:1.1.1.126, RHEA:15112]
GO:0047001
name: 2-dehydro-3-deoxy-D-gluconate 5-dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 2-dehydro-3-deoxy-D-gluconate = NADH + (4S)-4,6-dihydroxy-2,5-dioxohexanoate." [EC:1.1.1.127, MetaCyc:1.1.1.127-RXN]
GO:0047002
name: L-arabinitol 2-dehydrogenase activity
def: "Catalysis of the reaction: L-arabinitol + NAD(+) = L-ribulose + H(+) + NADH." [EC:1.1.1.13, RHEA:21359]
GO:0047003
name: dTDP-6-deoxy-L-talose 4-dehydrogenase activity
def: "Catalysis of the reaction: dTDP-6-deoxy-L-talose + NADP(+) = dTDP-4-dehydro-6-deoxy-L-mannose + H(+) + NADPH." [EC:1.1.1.134, RHEA:23651]
GO:0047004
name: UDP-N-acetylglucosamine 6-dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + 2 NAD(+) + UDP-N-acetyl-alpha-D-glucosamine = 3 H(+) + 2 NADH + UDP-N-acetyl-2-amino-2-deoxy-D-glucuronate." [EC:1.1.1.136, RHEA:13328]
GO:0047005
name: 16-alpha-hydroxysteroid dehydrogenase activity
def: "Catalysis of the reaction: NADP+ + 16-alpha-hydroxysteroid = NADPH + H+ + 16-oxosteroid." [EC:1.1.1.147, MetaCyc:1.1.1.147-RXN]
GO:0047006
name: 17-alpha,20-alpha-dihydroxypregn-4-en-3-one dehydrogenase activity
def: "Catalysis of the reaction: NAD(P)+ + 17-alpha,20-alpha-dihydroxypregn-4-en-3-one = NAD(P)H + H+ + 17-alpha-hydroxyprogesterone." [EC:1.1.1.149, MetaCyc:1.1.1.149-RXN]
GO:0047007
name: pregnan-21-ol dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: NAD(+) + pregnan-21-ol = H(+) + NADH + pregnan-21-al." [EC:1.1.1.150, RHEA:11451]
GO:0047008
name: pregnan-21-ol dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + pregnan-21-ol = H(+) + NADPH + pregnan-21-al." [EC:1.1.1.151, RHEA:23715]
GO:0047009
name: 3-alpha-hydroxy-5-beta-androstane-17-one 3-alpha-dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 3-alpha-hydroxy-5-beta-androstane-17-one = NADH + H+ + 5-beta-androstane-3,17-dione." [EC:1.1.1.152, MetaCyc:1.1.1.152-RXN]
GO:0047010
name: hydroxycyclohexanecarboxylate dehydrogenase activity
def: "Catalysis of the reaction: (1S,3R,4S)-3,4-dihydroxycyclohexane-1-carboxylate + NAD(+) = (1S,4S)-4-hydroxy-3-oxocyclohexane-1-carboxylate + H(+) + NADH." [EC:1.1.1.166, RHEA:10519]
GO:0047011
name: 2-dehydropantolactone reductase (A-specific) activity
def: "Catalysis of the reaction: (R)-pantolactone + NADP+ = 2-dehydropantolactone + NADPH + H+. The reaction is A-specific (i.e. the pro-R hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to NADP+." [EC:1.1.1.168, MetaCyc:1.1.1.168-RXN]
GO:0047012
name: sterol-4-alpha-carboxylate 3-dehydrogenase (decarboxylating) activity
def: "Catalysis of the reaction: NAD(P)+ + 3-beta-hydroxy-4-alpha-methyl-5-alpha-cholest-7-ene-4-beta-carboxylate = NAD(P)H + H+ + CO2 + 4-alpha-methyl-5-alpha-cholest-7-en-3-one." [EC:1.1.1.170, MetaCyc:1.1.1.170-RXN]
GO:0047013
name: cholate 12-alpha dehydrogenase activity
def: "Catalysis of the reaction: cholate + NADP(+) = 3alpha,7alpha-dihydroxy-12-oxo-5beta-cholanate + H(+) + NADPH." [EC:1.1.1.176, RHEA:14132]
GO:0047014
name: glycerol-3-phosphate 1-dehydrogenase [NADP+] activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + NADP(+) = D-glyceraldehyde 3-phosphate + H(+) + NADPH." [EC:1.1.1.177, RHEA:19776]
GO:0047015
name: 3-hydroxy-2-methylbutyryl-CoA dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 2-methyl-3-hydroxybutyryl-CoA = NADH + H+ + 2-methylaceto-acetyl-CoA." [EC:1.1.1.178, MetaCyc:1.1.1.178-RXN]
GO:0047016
name: cholest-5-ene-3-beta,7-alpha-diol 3-beta-dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 7-alpha-hydroxycholesterol = NADH + H+ + 7-alpha-hydroxycholest-4-en-3-one." [EC:1.1.1.181, MetaCyc:1.1.1.181-RXN]
GO:0047017
name: prostaglandin-F synthase activity
def: "Catalysis of the reaction: NADP+ + (5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate = NADPH + H+ + (5Z,13E)-(15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate." [EC:1.1.1.188, MetaCyc:1.1.1.188-RXN]
GO:0047018
name: indole-3-acetaldehyde reductase (NADH) activity
def: "Catalysis of the reaction: indole-3-ethanol + NAD(+) = (indol-3-yl)acetaldehyde + H(+) + NADH." [EC:1.1.1.190, RHEA:14876]
GO:0047019
name: indole-3-acetaldehyde reductase (NADPH) activity
def: "Catalysis of the reaction: indole-3-ethanol + NADP(+) = (indol-3-yl)acetaldehyde + H(+) + NADPH." [EC:1.1.1.191, RHEA:17040]
GO:0047020
name: 15-hydroxyprostaglandin-D dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP+ + (5Z,13E)-(15S)-9-alpha,15-dihydroxy-11-oxoprosta-5,13-dienoate = NADPH + H+ + (5Z,13E)-9-alpha-hydroxy-11,15-dioxoprosta-5,13-dienoate." [EC:1.1.1.196, MetaCyc:1.1.1.196-RXN]
GO:0047021
name: 15-hydroxyprostaglandin dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + prostaglandin E(1) = 15-dehydro-prostaglandin E1 + H(+) + NADPH." [EC:1.1.1.197, RHEA:11639]
GO:0047022
name: 7-beta-hydroxysteroid dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP+ + a 7-beta-hydroxysteroid = NADPH + H+ + a 7-oxosteroid." [EC:1.1.1.201, MetaCyc:1.1.1.201-RXN]
GO:0047023
name: androsterone dehydrogenase activity
def: "Catalysis of the reaction: NAD(P)+ + androsterone = NAD(P)H + H+ + 5-alpha-androstane-3,17-dione." [EC:1.1.1.209, MetaCyc:1.1.1.209-RXN]
GO:0047024
name: 5alpha-androstane-3beta,17beta-diol dehydrogenase activity
def: "Catalysis of the reaction: 5alpha-androstane-3beta,17beta-diol + NADP(+) = 17beta-hydroxy-5alpha-androstan-3-one + H(+) + NADPH." [EC:1.1.1.210, RHEA:16300]
GO:0047025
name: 3-oxoacyl-[acyl-carrier-protein] reductase (NADH) activity
def: "Catalysis of the reaction: NAD+ + OH-acyl-[acyl-carrier protein] = NADH + H+ + B-ketoacyl-[acyl-carrier protein]." [EC:1.1.1.212, MetaCyc:1.1.1.212-RXN]
GO:0047026
name: androsterone dehydrogenase (A-specific) activity
def: "Catalysis of the reaction: NAD(P)+ + androsterone = NAD(P)H + H+ + 5-alpha-androstane-3,17-dione. The reaction is A-specific (i.e. the pro-R hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to NAD(P)+." [EC:1.1.1.213, MetaCyc:1.1.1.213-RXN]
GO:0047027
name: benzyl-2-methyl-hydroxybutyrate dehydrogenase activity
def: "Catalysis of the reaction: benzyl (2R,3S)-2-methyl-3-hydroxybutanoate + NADP(+) = benzyl 2-methyl-3-oxobutanoate + H(+) + NADPH." [EC:1.1.1.217, RHEA:16408]
GO:0047028
name: 6-pyruvoyltetrahydropterin 2'-reductase activity
def: "Catalysis of the reaction: NADP+ + 6-lactoyl-5,6,7,8-tetrahydropterin = NADPH + H+ + 6-pyruvoyltetrahydropterin." [EC:1.1.1.220, MetaCyc:1.1.1.220-RXN]
GO:0047029
name: (R)-4-hydroxyphenyllactate dehydrogenase activity
def: "Catalysis of the reaction: NAD(P)+ + (R)-3-(4-hydroxyphenyl)lactate = NAD(P)H + H+ + 3-(4-hydroxyphenyl)pyruvate." [EC:1.1.1.222, MetaCyc:1.1.1.222-RXN]
GO:0047030
name: 4-hydroxycyclohexanecarboxylate dehydrogenase activity
def: "Catalysis of the reaction: trans-4-hydroxycyclohexanecarboxylate + NAD(+) = 4-oxocyclohexanecarboxylate + H(+) + NADH." [EC:1.1.1.226, RHEA:17432]
GO:0047031
name: diethyl 2-methyl-3-oxosuccinate reductase activity
def: "Catalysis of the reaction: diethyl (2R,3R)-2-methyl-3-hydroxysuccinate + NADP(+) = diethyl 2-methyl-3-oxosuccinate + H(+) + NADPH." [EC:1.1.1.229, RHEA:21011]
GO:0047032
name: 3-alpha-hydroxyglycyrrhetinate dehydrogenase activity
def: "Catalysis of the reaction: 3alpha-hydroxyglycyrrhetinate + NADP(+) = 3-oxoglycyrrhetinate + H(+) + NADPH." [EC:1.1.1.230, RHEA:20819]
GO:0047033
name: 15-hydroxyprostaglandin-I dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + prostaglandin I(2) = 15-dehydro-prostaglandin I(2) + H(+) + NADPH." [EC:1.1.1.231, RHEA:21423]
GO:0047034
name: 15-hydroxyicosatetraenoate dehydrogenase activity
def: "Catalysis of the reaction: NAD(P)+ + (15S)-15-hydroxy-5,8,11-cis-13-trans-icosatetraenoate = NAD(P)H + H+ + 15-oxo-5,8,11-cis-13-trans-icosatetraenoate." [EC:1.1.1.232, MetaCyc:1.1.1.232-RXN]
GO:0047035
name: testosterone dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: testosterone + NAD+ = androst-4-ene-3,17-dione + NADH." [EC:1.1.1.239, MetaCyc:1.1.1.239-RXN]
GO:0047036
name: codeinone reductase (NADPH) activity
def: "Catalysis of the reaction: codeine + NADP(+) = codeinone + H(+) + NADPH." [EC:1.1.1.247, RHEA:19212]
GO:0047037
name: salutaridine reductase (NADPH) activity
def: "Catalysis of the reaction: (7S)-salutaridinol + NADP(+) = H(+) + NADPH + salutaridine." [EC:1.1.1.248, RHEA:10111]
GO:0047038
name: D-arabinitol 2-dehydrogenase activity
def: "Catalysis of the reaction: D-arabinitol + NAD(+) = D-ribulose + H(+) + NADH." [EC:1.1.1.250, RHEA:17392]
GO:0047039
name: tetrahydroxynaphthalene reductase activity
def: "Catalysis of the reaction: NADP+ + scytalone = NADPH + H+ + 1,3,6,8-naphthalenetetrol." [EC:1.1.1.252, MetaCyc:1.1.1.252-RXN]
GO:0047040
name: pteridine reductase activity
def: "Catalysis of the reaction: 5,6,7,8-tetrahydrobiopterin + 2 NADP(+) = biopterin + 2 H(+) + 2 NADPH." [EC:1.5.1.33, RHEA:19512]
GO:0047041
name: (S)-carnitine 3-dehydrogenase activity
def: "Catalysis of the reaction: (S)-carnitine + NAD(+) = 3-dehydrocarnitine + H(+) + NADH." [EC:1.1.1.254, RHEA:11559]
GO:0047042
name: androsterone dehydrogenase (B-specific) activity
def: "Catalysis of the reaction: NAD(P)+ + androsterone = NAD(P)H + H+ + 5-alpha-androstane-3,17-dione. The reaction is B-specific (i.e. the pro-S hydrogen is transferred from the 4-position of reduced nicotinamide cofactor) with respect to NAD(P)+." [EC:1.1.1.50, MetaCyc:1.1.1.50-RXN]
GO:0047043
name: 3-alpha-hydroxycholanate dehydrogenase activity
def: "Catalysis of the reaction: lithocholate + NAD(+) = 3-oxo-5beta-cholanate + H(+) + NADH." [EC:1.1.1.52, RHEA:19588]
GO:0047044
name: androstan-3-alpha,17-beta-diol dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + androstan-3-alpha,17-beta-diol = 17-beta-hydroxyandrostan-3-one + NADH + H+." [EC:1.1.1.53, MetaCyc:1.1.1.53-RXN]
GO:0047045
name: testosterone 17-beta-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP+ + testosterone = NADPH + H+ + androst-4-ene-3,17-dione." [EC:1.1.1.64, MetaCyc:1.1.1.64-RXN]
GO:0047046
name: homoisocitrate dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 3-carboxy-2-hydroxyadipate = NADH + H+ + CO2 + 2-keto-adipate." [EC:1.1.1.155, EC:1.1.1.87, MetaCyc:1.1.1.87-RXN]
GO:0047047
name: oxaloglycolate reductase (decarboxylating) activity
def: "Catalysis of the reaction: glycerate + CO2 + NAD(P)+ = NAD(P)H + H+ + 2-hydroxy-3-oxosuccinate." [EC:1.1.1.92, MetaCyc:1.1.1.92-RXN]
GO:0047048
name: 3-hydroxybenzyl-alcohol dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxybenzyl alcohol + NADP(+) = 3-hydroxybenzaldehyde + H(+) + NADPH." [EC:1.1.1.97, RHEA:22343]
GO:0047049
name: (R)-2-hydroxy-fatty acid dehydrogenase activity
def: "Catalysis of the reaction: (R)-2-hydroxystearate + NAD(+) = 2-oxostearate + H(+) + NADH." [EC:1.1.1.98, RHEA:15952]
GO:0047050
name: (S)-2-hydroxy-fatty acid dehydrogenase activity
def: "Catalysis of the reaction: (S)-2-hydroxystearate + NAD(+) = 2-oxostearate + H(+) + NADH." [EC:1.1.1.99, RHEA:11387]
GO:0047051
name: D-lactate dehydrogenase (cytochrome c-553) activity
def: "Catalysis of the reaction: 2 ferricytochrome C-553 + D-lactate = 2 ferrocytochrome C-553 + pyruvate." [EC:1.1.2.5, MetaCyc:1.1.2.5-RXN]
GO:0047052
name: (S)-stylopine synthase activity
def: "Catalysis of the reaction: (S)-cheilanthifoline + H(+) + NADPH + O(2) = (S)-stylopine + 2 H(2)O + NADP(+)." [EC:1.14.21.1, RHEA:13776]
GO:0047053
name: (S)-cheilanthifoline synthase activity
def: "Catalysis of the reaction: (S)-scoulerine + H(+) + NADPH + O(2) = (S)-cheilanthifoline + 2 H(2)O + NADP(+)." [EC:1.14.21.2, RHEA:20488]
GO:0047054
name: berbamunine synthase activity
def: "Catalysis of the reaction: O2 + NADPH + H+ + R-N-methylcoclaurine + S-N-methylcoclaurine = 2 H2O + NADP+ + berbamunine." [EC:1.14.21.3, MetaCyc:1.1.3.34-RXN]
GO:0047055
name: salutaridine synthase activity
def: "Catalysis of the reaction: (R)-reticuline + H(+) + NADPH + O(2) = 2 H(2)O + NADP(+) + salutaridine." [EC:1.14.21.4, RHEA:17716]
GO:0047056
name: (S)-canadine synthase activity
def: "Catalysis of the reaction: (S)-tetrahydrocolumbamine + H(+) + NADPH + O(2) = (S)-canadine + 2 H(2)O + NADP(+)." [EC:1.14.21.5, RHEA:21459]
GO:0047057
name: vitamin-K-epoxide reductase (warfarin-sensitive) activity
def: "Catalysis of the reaction: 2-methyl-3-phytyl-1,4-naphthoquinone + oxidized dithiothreitol + H2O = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol." [EC:1.1.4.1, MetaCyc:1.1.4.1-RXN]
GO:0047058
name: vitamin-K-epoxide reductase (warfarin-insensitive) activity
def: "Catalysis of the reaction: 3-hydroxy-2-methyl-3-phytyl-2,3-dihydronaphthoquinone + oxidized dithiothreitol + H2O = 2,3-epoxy-2,3-dihydro-2-methyl-3-phytyl-1,4-naphthoquinone + 1,4-dithiothreitol." [EC:1.1.4.2, MetaCyc:1.1.4.2-RXN]
GO:0047059
name: polyvinyl alcohol dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: polyvinyl alcohol + ferricytochrome c = oxidized polyvinyl alcohol + ferrocytochrome c + H+." [EC:1.1.2.6]
GO:0047060
name: (R)-pantolactone dehydrogenase (flavin) activity
def: "Catalysis of the reaction: (R)-pantolactone + A = 2-dehydropantolactone + AH(2)." [EC:1.1.99.27, RHEA:21007]
GO:0047061
name: glucose-fructose oxidoreductase activity
def: "Catalysis of the reaction: D-fructose + D-glucose = D-glucitol + D-glucono-1,5-lactone." [EC:1.1.99.28, RHEA:20640]
GO:0047062
name: trans-acenaphthene-1,2-diol dehydrogenase activity
def: "Catalysis of the reaction: (+-)-trans-acenaphthene-1,2-diol + 2 NADP(+) = acenaphthene-1,2-dione + 2 H(+) + 2 NADPH." [EC:1.10.1.1, RHEA:22187]
GO:0047063
name: L-ascorbate-cytochrome-b5 reductase activity
def: "Catalysis of the reaction: ferricytochrome B5 + ascorbate = ferrocytochrome B5 + monodehydroascorbate." [EC:1.10.2.1, MetaCyc:1.10.2.1-RXN]
GO:0047064
name: sulochrin oxidase [(+)-bisdechlorogeodin-forming] activity
def: "Catalysis of the reaction: O(2) + 2 sulochrin = 2 (2S)-bisdechlorogeodin + 2 H(2)O." [EC:1.21.3.4, RHEA:24095]
GO:0047065
name: sulochrin oxidase [(-)-bisdechlorogeodin-forming] activity
def: "Catalysis of the reaction: O(2) + 2 sulochrin = 2 (2R)-bisdechlorogeodin + 2 H(2)O." [EC:1.21.3.5, RHEA:22619]
GO:0047066
name: phospholipid-hydroperoxide glutathione peroxidase activity
def: "Catalysis of the reaction: a lipid hydroperoxide + 2 reduced glutathione = 2 H2O + lipid + 2 oxidized glutathione." [EC:1.11.1.12, MetaCyc:1.11.1.12-RXN]
GO:0047067
name: hydrogen:quinone oxidoreductase activity
def: "Catalysis of the reaction: H(2) + menaquinone = reduced menaquinone." [EC:1.12.5.1, RHEA:18644]
GO:0047068
name: N5,N10-methenyltetrahydromethanopterin hydrogenase activity
def: "Catalysis of the reaction: 5,10-methenyl-5,6,7,8-tetrahydromethanopterin + H(2) = 5,10-methylenetetrahydromethanopterin + H(+)." [EC:1.12.98.2, RHEA:20020]
GO:0047069
name: 7,8-dihydroxykynurenate 8,8a-dioxygenase activity
def: "Catalysis of the reaction: 7,8-dihydroxykynurenate + O(2) = 5-(3-carboxylato-3-oxoprop-1-en-1-yl)-4,6-dihydroxypyridine-2-carboxylate + H(+)." [EC:1.13.11.10, RHEA:23403]
GO:0047070
name: 3-carboxyethylcatechol 2,3-dioxygenase activity
def: "Catalysis of the reaction: O2 + 3-(2,3-dihydroxyphenyl)propanoate = 2-hydroxy-6-oxonona-2,4-diene-1,9-dioate." [EC:1.13.11.16, MetaCyc:1.13.11.16-RXN]
GO:0047071
name: 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione 4,5-dioxygenase activity
def: "Catalysis of the reaction: 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione + O(2) = 3-hydroxy-5,9,17-trioxo-4,5:9,10-disecoandrosta-1(10),2-dien-4-oate + H(+)." [EC:1.13.11.25, RHEA:21355]
GO:0047072
name: 2,3-dihydroxybenzoate 2,3-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxybenzoate + O(2) = 2-carboxy-cis,cis-muconate + 2 H(+)." [EC:1.13.11.28, RHEA:15372]
GO:0047073
name: 2,4'-dihydroxyacetophenone dioxygenase activity
def: "Catalysis of the reaction: 2,4'-dihydroxyacetophenone + O(2) = 4-hydroxybenzoate + formate + 2 H(+)." [EC:1.13.11.41, RHEA:24419]
GO:0047074
name: 4-hydroxycatechol 1,2-dioxygenase activity
def: "Catalysis of the reaction: O2 + benzene-1,2,4-triol = maleylacetate." [EC:1.13.11.-, MetaCyc:R308-RXN]
GO:0047075
name: 2,5-dihydroxypyridine 5,6-dioxygenase activity
def: "Catalysis of the reaction: H2O + O2 + 2,5-dihydroxypyridine = formate + maleamate." [EC:1.13.11.9, MetaCyc:1.13.11.9-RXN]
GO:0047077
name: Photinus-luciferin 4-monooxygenase (ATP-hydrolyzing) activity
def: "Catalysis of the reaction: O2 + ATP + Photinus luciferin = light + diphosphate + AMP + CO2 + oxidized Photinus luciferin." [EC:1.13.12.7, MetaCyc:1.13.12.7-RXN]
GO:0047078
name: 3-hydroxy-4-oxoquinoline 2,4-dioxygenase activity
def: "Catalysis of the reaction: O2 + 3-hydroxy-1H-quinolin-4-one = carbon monoxide + N-formylanthranilate." [EC:1.13.11.47, MetaCyc:1.13.11.47-RXN]
GO:0047079
name: deoxyuridine 1'-dioxygenase activity
def: "Catalysis of the reaction: 2'-deoxyuridine + 2-oxoglutarate + O(2) = 2-deoxy-D-ribono-1,4-lactone + CO(2) + succinate + uracil." [EC:1.14.11.10, RHEA:23319]
GO:0047080
name: deoxyuridine 2'-dioxygenase activity
def: "Catalysis of the reaction: 2'-deoxyuridine + 2-oxoglutarate + O(2) = CO(2) + succinate + uridine." [EC:1.14.11.3, RHEA:21079]
GO:0047081
name: 3-hydroxy-2-methylpyridinecarboxylate dioxygenase activity
def: "Catalysis of the reaction: O2 + NAD(P)H + H+ + 3-hydroxy-2-methylpyridine-5-carboxylate = NAD(P)+ + 2-(acetamidomethylene)succinate." [EC:1.14.12.4, MetaCyc:1.14.12.4-RXN]
GO:0047082
name: 3,9-dihydroxypterocarpan 6a-monooxygenase activity
def: "Catalysis of the reaction: (6aR,11aR)-3,9-dihydroxypterocarpan + H(+) + NADPH + O(2) = (6aS,11aS)-3,6a,9-trihydroxypterocarpan + H(2)O + NADP(+). (6aS,11aS)-3,6a,9-trihydroxypterocarpan is also known as (-)-glycinol." [EC:1.14.13.28, RHEA:15324]
GO:0047083
name: 5-O-(4-coumaroyl)-D-quinate 3'-monooxygenase activity
def: "Catalysis of the reaction: O2 + NADPH + H+ + trans-5-O-(4-coumaroyl)-D-quinate = H2O + NADP+ + trans-5-O-caffeoyl-D-quinate." [EC:1.14.13.36, MetaCyc:1.14.13.36-RXN]
GO:0047084
name: methyltetrahydroprotoberberine 14-monooxygenase activity
def: "Catalysis of the reaction: O2 + NADPH + H+ + (S)-N-methylcanadine = H2O + NADP+ + allocryptopine." [EC:1.14.13.37, MetaCyc:1.14.13.37-RXN]
GO:0047085
name: hydroxyphenylacetonitrile 2-monooxygenase activity
def: "Catalysis of the reaction: 4-hydroxyphenylacetonitrile + H(+) + NADPH + O(2) = 4-hydroxymandelonitrile + H(2)O + NADP(+)." [RHEA:23743]
GO:0047086
name: ketosteroid monooxygenase activity
def: "Catalysis of the reaction: O2 + NADPH + progesterone = H2O + NADP+ + testosterone acetate." [EC:1.14.13.54, MetaCyc:1.14.13.54-RXN]
GO:0047087
name: protopine 6-monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + protopine = 6-hydroxyprotopine + H(2)O + NADP(+)." [EC:1.14.13.55, RHEA:22647]
GO:0047088
name: dihydrosanguinarine 10-monooxygenase activity
def: "Catalysis of the reaction: dihydrosanguinarine + H(+) + NADPH + O(2) = 10-hydroxydihydrosanguinarine + H(2)O + NADP(+)." [EC:1.14.13.56, RHEA:10531]
GO:0047089
name: dihydrochelirubine 12-monooxygenase activity
def: "Catalysis of the reaction: dihydrochelirubine + H(+) + NADPH + O(2) = 12-hydroxydihydrochelirubine + H(2)O + NADP(+)." [EC:1.14.13.57, RHEA:10159]
GO:0047090
name: benzoyl-CoA 3-monooxygenase activity
def: "Catalysis of the reaction: benzoyl-CoA + H(+) + NADPH + O(2) = 3-hydroxybenzoyl-CoA + H(2)O + NADP(+)." [EC:1.14.13.58, RHEA:23219]
GO:0047091
name: L-lysine 6-monooxygenase (NADPH) activity
def: "Catalysis of the reaction: L-lysine + NADPH + O(2) = N(6)-hydroxy-L-lysine + H(2)O + NADP(+)." [EC:1.14.13.59, RHEA:23231]
GO:0047092
name: 27-hydroxycholesterol 7-alpha-monooxygenase activity
def: "Catalysis of the reaction: O2 + NADPH + 27-hydroxycholesterol = H2O + NADP+ + 7-alpha,27-dihydroxycholesterol." [EC:1.14.13.60, MetaCyc:1.14.13.60-RXN]
GO:0047093
name: 4-hydroxyquinoline 3-monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADH + O(2) + quinolin-4-ol = H(2)O + NAD(+) + quinoline-3,4-diol." [EC:1.14.13.62, RHEA:19328]
GO:0047094
name: 3-hydroxyphenylacetate 6-hydroxylase activity
def: "Catalysis of the reaction: O2 + NAD(P)H + 3-hydroxyphenylacetate = H2O + NAD(P)+ + homogentisate." [EC:1.14.13.63, MetaCyc:1.14.13.63-RXN]
GO:0047095
name: 2-hydroxycyclohexanone 2-monooxygenase activity
def: "Catalysis of the reaction: 2-hydroxycyclohexan-1-one + NADPH + O(2) = 6-oxohexanoate + H(2)O + NADP(+)." [EC:1.14.13.66, RHEA:25172]
GO:0047096
name: androst-4-ene-3,17-dione monooxygenase activity
def: "Catalysis of the reaction: AH(2) + androst-4-ene-3,17-dione + O(2) = A + H(2)O + testololactone." [EC:1.14.99.12, RHEA:22699]
GO:0047097
name: phylloquinone monooxygenase (2,3-epoxidizing) activity
def: "Catalysis of the reaction: AH(2) + O(2) + phylloquinone = 2,3-epoxyphylloquinone + A + H(2)O." [EC:1.14.99.20, RHEA:16748]
GO:0047098
name: Latia-luciferin monooxygenase (demethylating) activity
def: "Catalysis of the reaction: 2 O2 + donor-H2 + Latia luciferin = light + H2O + acceptor + formate + CO2 + oxidized Latia luciferin." [EC:1.14.99.21, MetaCyc:1.14.99.21-RXN]
GO:0047099
name: CDP-4-dehydro-6-deoxyglucose reductase activity
def: "Catalysis of the reaction: H2O + NAD(P)+ + CDP-4-dehydro-3,6-dideoxy-D-glucose = NAD(P)H + CDP-4-dehydro-6-deoxy-D-glucose." [EC:1.17.1.1, MetaCyc:1.17.1.1-RXN]
GO:0047100
name: glyceraldehyde-3-phosphate dehydrogenase (NADP+) (phosphorylating) activity
def: "Catalysis of the reaction: phosphate + NADP+ + glyceraldehyde-3-phosphate = NADPH + 3-phospho-D-glyceroyl-phosphate." [EC:1.2.1.13, MetaCyc:1.2.1.13-RXN]
GO:0047101
name: 2-oxoisovalerate dehydrogenase (acylating) activity
def: "Catalysis of the reaction: NAD+ + CoA + 2-keto-isovalerate = NADH + CO2 + isobutyryl-CoA." [EC:1.2.1.25, MetaCyc:1.2.1.25-RXN]
GO:0047102
name: aminomuconate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: H2O + NAD+ + 2-aminomuconate semialdehyde = NADH + 2-amino-muconate." [EC:1.2.1.32, MetaCyc:1.2.1.32-RXN]
GO:0047103
name: 3-alpha,7-alpha,12-alpha-trihydroxycholestan-26-al 26-oxidoreductase activity
def: "Catalysis of the reaction: H2O + NAD+ + 3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-al = NADH + 3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestanate." [EC:1.2.1.40, MetaCyc:1.2.1.40-RXN]
GO:0047104
name: hexadecanal dehydrogenase (acylating) activity
def: "Catalysis of the reaction: CoA + NAD(+) + palmitaldehyde = H(+) + NADH + palmitoyl-CoA." [EC:1.2.1.42, RHEA:19708]
GO:0047105
name: 4-trimethylammoniobutyraldehyde dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 4-trimethylammoniobutanal = NADH + 4-trimethylammoniobutanoate." [EC:1.2.1.47, MetaCyc:1.2.1.47-RXN]
GO:0047106
name: 4-hydroxyphenylacetaldehyde dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + (4-hydroxyphenyl)acetaldehyde + H2O = NADH + 4-hydroxyphenylacetate." [EC:1.2.1.53, MetaCyc:1.2.1.53-RXN]
GO:0047107
name: gamma-guanidinobutyraldehyde dehydrogenase activity
def: "Catalysis of the reaction: 4-guanidinobutanal + H(2)O + NAD(+) = 4-guanidinobutanoate + 2 H(+) + NADH." [EC:1.2.1.54, RHEA:14384]
GO:0047108
name: (R)-3-hydroxyacid-ester dehydrogenase activity
def: "Catalysis of the reaction: ethyl (R)-3-hydroxyhexanoate + NADP(+) = ethyl 3-oxohexanoate + H(+) + NADPH." [EC:1.1.1.279, RHEA:24355]
GO:0047109
name: (S)-3-hydroxyacid-ester dehydrogenase activity
def: "Catalysis of the reaction: ethyl (S)-3-hydroxyhexanoate + NADP(+) = ethyl 3-oxohexanoate + H(+) + NADPH." [EC:1.1.1.280, RHEA:18272]
GO:0047110
name: phenylglyoxylate dehydrogenase (acylating) activity
def: "Catalysis of the reaction: CoA + NAD(+) + phenylglyoxylate = benzoyl-CoA + CO(2) + NADH." [EC:1.2.1.58, RHEA:10375]
GO:0047111
name: formate dehydrogenase (cytochrome-c-553) activity
def: "Catalysis of the reaction: ferricytochrome C-553 + formate = ferrocytochrome C-553 + CO2." [EC:1.2.2.3, MetaCyc:1.2.2.3-RXN]
GO:0047112
name: pyruvate oxidase activity
def: "Catalysis of the reaction: H(+) + O(2) + phosphate + pyruvate = acetyl phosphate + CO(2) + H(2)O(2)." [EC:1.2.3.3, RHEA:20851]
GO:0047113
name: aldehyde dehydrogenase (pyrroloquinoline-quinone) activity
def: "Catalysis of the reaction: an aldehyde + acceptor + H2O = a carboxylate + reduced acceptor." [EC:1.2.99.3, MetaCyc:1.2.99.3-RXN]
GO:0047114
name: kynurenate-7,8-dihydrodiol dehydrogenase activity
def: "Catalysis of the reaction: 7,8-dihydro-7,8-dihydroxykynurenate + NAD(+) = 7,8-dihydroxykynurenate + H(+) + NADH." [EC:1.3.1.18, RHEA:22251]
GO:0047115
name: trans-1,2-dihydrobenzene-1,2-diol dehydrogenase activity
def: "Catalysis of the reaction: NADP+ + trans-1,2-dihydrobenzene-1,2-diol = NADPH + catechol." [EC:1.3.1.20, MetaCyc:1.3.1.20-RXN]
GO:0047116
name: 1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: NAD+ + 1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate = NADH + CO2 + catechol." [EC:1.3.1.25, MetaCyc:1.3.1.25-RXN]
GO:0047117
name: enoyl-[acyl-carrier-protein] reductase (NADPH, A-specific) activity
def: "Catalysis of the reaction: acyl-[acyl-carrier protein] + NADP+ = trans-D2-enoyl-[acyl-carrier protein] + NADPH + H+." [EC:1.3.1.39, MetaCyc:1.3.1.39-RXN]
GO:0047118
name: 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate reductase activity
def: "Catalysis of the reaction: 2,6-dioxo-6-phenylhexanoate + NADP(+) = 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate + H(+) + NADPH." [EC:1.3.1.40, RHEA:24271]
GO:0047119
name: 2-methyl-branched-chain-enoyl-CoA reductase activity
def: "Catalysis of the reaction: 2-methylbutanoyl-CoA + NAD(+) = 2-methylbut-2-enoyl-CoA + H(+) + NADH." [EC:1.3.1.52, RHEA:24535]
GO:0047120
name: (3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate dehydrogenase activity
def: "Catalysis of the reaction: (3S,4R)-3,4-dihydroxycyclohexa-1,5-diene-1,4-dicarboxylate + NAD(+) = 3,4-dihydroxybenzoate + CO(2) + NADH." [EC:1.3.1.53, RHEA:10747]
GO:0047121
name: isoquinoline 1-oxidoreductase activity
def: "Catalysis of the reaction: A + H(2)O + isoquinoline = AH(2) + isoquinolin-1(2H)-one." [EC:1.3.99.16, RHEA:11591]
GO:0047122
name: quinaldate 4-oxidoreductase activity
def: "Catalysis of the reaction: A + H(2)O + quinaldate = AH(2) + kynurenate." [EC:1.3.99.18, RHEA:16700]
GO:0047123
name: quinoline-4-carboxylate 2-oxidoreductase activity
def: "Catalysis of the reaction: A + H(2)O + quinoline-4-carboxylate = 2-oxo-1,2-dihydroquinoline-4-carboxylate + AH(2)." [EC:1.3.99.19, RHEA:14952]
GO:0047124
name: L-erythro-3,5-diaminohexanoate dehydrogenase activity
def: "Catalysis of the reaction: (3S,5S)-3,5-diaminohexanoate + H(2)O + NAD(+) = (S)-5-amino-3-oxo-hexanoate + H(+) + NADH + NH(4)(+)." [EC:1.4.1.11, RHEA:19636]
GO:0047125
name: delta1-piperideine-2-carboxylate reductase activity
def: "Catalysis of the reaction: NADP+ + L-pipecolate = NADPH + delta1-piperideine-2-carboxylate." [EC:1.5.1.21]
GO:0047126
name: N5-(carboxyethyl)ornithine synthase activity
def: "Catalysis of the reaction: N(5)-[1(S)-1-carboxyethyl]-L-ornithine + H(2)O + NADP(+) = L-ornithine + H(+) + NADPH + pyruvate." [EC:1.5.1.24, RHEA:18664]
GO:0047127
name: thiomorpholine-carboxylate dehydrogenase activity
def: "Catalysis of the reaction: NAD(P)+ + thiomorpholine-3-carboxylate = NAD(P)H + 3,4-dehydro-1,4-thiomorpholine-3-carboxylate." [EC:1.5.1.25, MetaCyc:1.5.1.25-RXN]
GO:0047128
name: 1,2-dehydroreticulinium reductase (NADPH) activity
def: "Catalysis of the reaction: (R)-reticuline + NADP(+) = 1,2-dehydroreticuline + H(+) + NADPH." [EC:1.5.1.27, RHEA:17572]
GO:0047129
name: opine dehydrogenase activity
def: "Catalysis of the reaction: (2S)-2-[(R)-1-carboxyethylamino]pentanoate + H(2)O + NAD(+) = L-2-aminopentanoate + H(+) + NADH + pyruvate." [EC:1.5.1.28, RHEA:21595]
GO:0047130
name: saccharopine dehydrogenase (NADP+, L-lysine-forming) activity
def: "Catalysis of the reaction: L-saccharopine + H(2)O + NADP(+) = 2-oxoglutarate + L-lysine + H(+) + NADPH." [EC:1.5.1.8, RHEA:19376]
GO:0047131
name: saccharopine dehydrogenase (NAD+, L-glutamate-forming) activity
def: "Catalysis of the reaction: L-saccharopine + H(2)O + NAD(+) = L-glutamate + allysine + H(+) + NADH." [EC:1.5.1.9, RHEA:24523]
GO:0047132
name: dihydrobenzophenanthridine oxidase activity
def: "Catalysis of the reaction: O2 + dihydrosanguinarine = H2O2 + sanguinarine." [EC:1.5.3.12, MetaCyc:1.5.3.12-RXN]
GO:0047133
name: dimethylamine dehydrogenase activity
def: "Catalysis of the reaction: electron-transferring flavoprotein + H2O + dimethylamine = reduced electron-transferring flavoprotein + formaldehyde + methylamine." [EC:1.5.8.1, MetaCyc:1.5.8.1-RXN]
GO:0047134
name: protein-disulfide reductase activity
def: "Catalysis of the reaction: protein-dithiol + NAD(P)+ = protein-disulfide + NAD(P)H + H+." [EC:1.8.1.8, MetaCyc:1.6.4.4-RXN]
GO:0047135
name: bis-gamma-glutamylcystine reductase activity
def: "Catalysis of the reaction: 2 L-gamma-glutamyl-L-cysteine + NADP(+) = bis-gamma-glutamylcystine + H(+) + NADPH." [EC:1.8.1.13, RHEA:11983]
GO:0047136
name: 4-(dimethylamino)phenylazoxybenzene reductase activity
def: "Catalysis of the reaction: 4-(dimethylamino)azobenzene + H(2)O + NADP(+) = 4-(dimethylamino)phenylazoxybenzene + H(+) + NADPH." [EC:1.7.1.11, RHEA:19792]
GO:0047137
name: N-hydroxy-2-acetamidofluorene reductase activity
def: "Catalysis of the reaction: 2-acetamidofluorene + NAD(P)+ + H2O = N-hydroxy-2-acetamidofluorene + NAD(P)H + H+." [EC:1.7.1.12, MetaCyc:1.7.1.12-RXN]
GO:0047138
name: aquacobalamin reductase activity
def: "Catalysis of the reaction: 2 cob(II)alamin + NAD+ = 2 aquacob(III)alamin + NADH + H+." [EC:1.16.1.3, MetaCyc:1.6.99.12-RXN]
GO:0047139
name: glutathione-homocystine transhydrogenase activity
def: "Catalysis of the reaction: homocystine + 2 reduced glutathione = oxidized glutathione + 2 homocysteine." [EC:1.8.4.1, MetaCyc:1.8.4.1-RXN]
GO:0047140
name: glutathione-CoA-glutathione transhydrogenase activity
def: "Catalysis of the reaction: oxidized glutathione + CoA = reduced glutathione + CoA-glutathione." [EC:1.8.4.3, MetaCyc:1.8.4.3-RXN]
GO:0047141
name: glutathione-cystine transhydrogenase activity
def: "Catalysis of the reaction: cystine + 2 reduced glutathione = oxidized glutathione + 2 L-cysteine." [EC:1.8.4.4, MetaCyc:1.8.4.4-RXN]
GO:0047142
name: enzyme-thiol transhydrogenase (glutathione-disulfide) activity
def: "Catalysis of the reaction: oxidized glutathione + [xanthine dehydrogenase] = reduced glutathione + xanthine-oxidase." [EC:1.8.4.7, MetaCyc:1.8.4.7-RXN]
GO:0047143
name: chlorate reductase activity
def: "Catalysis of the reaction: AH(2) + chlorate = A + chlorite + H(2)O + H(+)." [EC:1.97.1.1, RHEA:16352]
GO:0047144
name: 2-acylglycerol-3-phosphate O-acyltransferase activity
def: "Catalysis of the reaction: 2-acyl-sn-glycerol 3-phosphate + acyl-CoA = L-phosphatidate + CoA." [EC:2.3.1.52, GOC:ab, MetaCyc:2-ACYL2.3.1.15-RXN, RHEA:14236]
GO:0047145
name: demethylsterigmatocystin 6-O-methyltransferase activity
def: "Catalysis of the reaction: 6-demethylsterigmatocystin + S-adenosyl-L-methionine = sterigmatocystin + S-adenosyl-homocysteine." [EC:2.1.1.109, MetaCyc:2.1.1.109-RXN]
GO:0047146
name: sterigmatocystin 7-O-methyltransferase activity
def: "Catalysis of the reaction: sterigmatocystin + S-adenosyl-L-methionine = 7-O-methylsterigmatocystin + S-adenosyl-homocysteine." [EC:2.1.1.110, MetaCyc:2.1.1.110-RXN]
GO:0047147
name: trimethylsulfonium-tetrahydrofolate N-methyltransferase activity
def: "Catalysis of the reaction: (6S)-5,6,7,8-tetrahydrofolate + trimethylsulfonium = (6S)-5-methyl-5,6,7,8-tetrahydrofolate + dimethyl sulfide + H(+)." [EC:2.1.1.19, RHEA:13696]
GO:0047148
name: methylamine-glutamate N-methyltransferase activity
def: "Catalysis of the reaction: L-glutamate + methylammonium = N-methyl-L-glutamate + NH(4)(+)." [EC:2.1.1.21, RHEA:15840]
GO:0047149
name: thetin-homocysteine S-methyltransferase activity
def: "Catalysis of the reaction: L-homocysteine + dimethylsulfonioacetate = (methylthio)acetate + L-methionine + H(+)." [EC:2.1.1.3, RHEA:22791]
GO:0047150
name: betaine-homocysteine S-methyltransferase activity
def: "Catalysis of the reaction: L-homocysteine + betaine = N,N-dimethylglycine + L-methionine." [EC:2.1.1.5, RHEA:22339]
GO:0047151
name: methylenetetrahydrofolate-tRNA-(uracil-5-)-methyltransferase (FADH2-oxidizing) activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + tRNA containing uridine at position 54 + FADH + H+ = tetrahydrofolate + tRNA containing ribothymidine at position 54 + FAD+." [EC:2.1.1.74, MetaCyc:2.1.1.74-RXN]
GO:0047152
name: methanol-5-hydroxybenzimidazolylcobamide Co-methyltransferase activity
def: "Catalysis of the reaction: 5-hydroxybenzimidazolylcobamide + methanol = H2O + Co-methyl-Co-5-hydroxybenzimidazolylcob(I)amide." [EC:2.1.1.90, MetaCyc:2.1.1.90-RXN]
GO:0047153
name: deoxycytidylate 5-hydroxymethyltransferase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + dCMP + H(2)O = (6S)-5,6,7,8-tetrahydrofolate + 5-hydroxymethyldeoxycytidylate." [EC:2.1.2.8, RHEA:11283]
GO:0047154
name: methylmalonyl-CoA carboxytransferase activity
def: "Catalysis of the reaction: pyruvate + D-methylmalonyl-CoA = oxaloacetic acid + propionyl-CoA." [EC:2.1.3.1, MetaCyc:2.1.3.1-RXN]
GO:0047155
name: 3-hydroxymethylcephem carbamoyltransferase activity
def: "Catalysis of the reaction: a 3-hydroxymethylceph-3-em-4-carboxylate + carbamoyl-phosphate = phosphate + a 3-carbamoyloxymethylcephem." [EC:2.1.3.7, MetaCyc:2.1.3.7-RXN]
GO:0047156
name: acetoin-ribose-5-phosphate transaldolase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + acetoin = 1-deoxy-D-altro-heptulose 7-phosphate + acetaldehyde." [EC:2.2.1.4, RHEA:21507]
GO:0047157
name: myelin-proteolipid O-palmitoyltransferase activity
def: "Catalysis of the reaction: [myelin proteolipid] + palmityl-CoA = [myelin proteolipid] O-palmitoylprotein + CoA." [EC:2.3.1.100, MetaCyc:2.3.1.100-RXN]
GO:0047158
name: sinapoylglucose-sinapoylglucose O-sinapoyltransferase activity
def: "Catalysis of the reaction: 2 1-O-sinapoyl-beta-D-glucose = 1,2-di-O-sinapoyl-beta-D-glucose + D-glucose." [EC:2.3.1.103, RHEA:22667]
GO:0047159
name: 1-alkenylglycerophosphocholine O-acyltransferase activity
def: "Catalysis of the reaction: 1-alkenylglycerophosphocholine + acyl-CoA = 1-alkenyl-2-acylglycerophosphocholine + CoA." [EC:2.3.1.104, MetaCyc:2.3.1.104-RXN]
GO:0047160
name: alkylglycerophosphate 2-O-acetyltransferase activity
def: "Catalysis of the reaction: 1-alkyl-sn-glycerol 3-phosphate + acetyl-CoA = 1-alkyl-2-acetyl-sn-glycerol 3-phosphate + CoA." [EC:2.3.1.105, RHEA:18560]
GO:0047161
name: tartronate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: hydroxymalonate + sinapoyl-CoA = CoA + sinapoyltartronate." [EC:2.3.1.106, RHEA:10955]
GO:0047162
name: 17-O-deacetylvindoline O-acetyltransferase activity
def: "Catalysis of the reaction: (1R,9R,10S,11R,12R,19R)-12-ethyl-10,11-dihydroxy-5-methoxy-10-(methoxycarbonyl)-8-methyl-8,16-diazapentacyclo[10.6.1.0^{1,9}.0^{2,7}.0^{16,19}]nonadeca-2(7),3,5,13-tetraen-16-ium + acetyl-CoA = (1R,9R,10S,11R,12R,19R)-11-(acetyloxy)-12-ethyl-10-hydroxy-5-methoxy-10-(methoxycarbonyl)-8-methyl-8,16-diazapentacyclo[10.6.1.0^{1,9}.0^{2,7}.0^{16,19}]nonadeca-2(7),3,5,13-tetraen-16-ium + CoA." [EC:2.3.1.107, RHEA:24499]
GO:0047163
name: 3,4-dichloroaniline N-malonyltransferase activity
def: "Catalysis of the reaction: 3,4-dichloroaniline + malonyl-CoA = N-(3,4-dichlorophenyl)malonamate + CoA." [EC:2.3.1.114, RHEA:21063]
GO:0047164
name: isoflavone-7-O-beta-glucoside 6''-O-malonyltransferase activity
def: "Catalysis of the reaction: biochanin-A + malonyl-CoA = 6'-malonyl-biochanin A + CoA." [EC:2.3.1.115, MetaCyc:2.3.1.115-RXN]
GO:0047165
name: flavonol-3-O-beta-glucoside O-malonyltransferase activity
def: "Catalysis of the reaction: flavonol 3-O-beta-D-glucoside + malonyl-CoA = malonyl-flavonol 3-O-beta-D-glucoside + CoA." [EC:2.3.1.116, MetaCyc:2.3.1.116-RXN]
GO:0047166
name: 1-alkenylglycerophosphoethanolamine O-acyltransferase activity
def: "Catalysis of the reaction: 1-alkenylglycerophosphoethanolamine + acyl-CoA = 1-alkenyl-2-acyl-glycerophosphoethanolamine + CoA." [EC:2.3.1.121, MetaCyc:2.3.1.121-RXN]
GO:0047167
name: 1-alkyl-2-acetylglycerol O-acyltransferase activity
def: "Catalysis of the reaction: 1-O-alkyl-2-acetyl-sn-glycerol + acyl-CoA = 1-O-alkyl-2-acetyl-3-acyl-sn-glycerol + CoA." [EC:2.3.1.125, MetaCyc:2.3.1.125-RXN]
GO:0047168
name: isocitrate O-dihydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: caffeoyl-CoA + isocitrate = 2-caffeoylisocitrate + CoA." [EC:2.3.1.126, RHEA:20759]
GO:0047169
name: galactarate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: feruloyl-CoA + galactarate = 2-(E)-O-feruloyl-D-galactarate + CoA." [EC:2.3.1.130, RHEA:13000]
GO:0047170
name: glucarate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: D-glucarate + sinapoyl-CoA = 2-O-sinapoyl-D-glucarate + CoA." [EC:2.3.1.131, RHEA:23311]
GO:0047171
name: glucarolactone O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: glucarolactone + sinapoyl-CoA = O-sinapoylglucarolactone + CoA." [EC:2.3.1.132, MetaCyc:2.3.1.132-RXN]
GO:0047172
name: shikimate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: shikimate + coumaroyl-CoA = 4-coumaroylshikimate + CoA." [EC:2.3.1.133, MetaCyc:2.3.1.133-RXN]
GO:0047173
name: phosphatidylcholine-retinol O-acyltransferase activity
def: "Catalysis of the reaction: retinol-[cellular-retinol-binding-protein] + phosphatidylcholine = retinyl-ester-[cellular-retinol-binding-protein] + 2-acylglycerophosphocholine." [EC:2.3.1.135, MetaCyc:2.3.1.135-RXN]
GO:0047174
name: putrescine N-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: caffeoyl-CoA + putrescine = N-caffeoylputrescine + CoA + H(+)." [EC:2.3.1.138, RHEA:12439]
GO:0047175
name: galactosylacylglycerol O-acyltransferase activity
def: "Catalysis of the reaction: sn-3-D-galactosyl-sn-2-acylglycerol + acyl-[acyl-carrier protein] = D-galactosyldiacylglycerol + [acyl-carrier protein]." [EC:2.3.1.141, MetaCyc:2.3.1.141-RXN]
GO:0047176
name: beta-glucogallin-tetrakisgalloylglucose O-galloyltransferase activity
def: "Catalysis of the reaction: 1,2,3,6-tetrakis-O-galloyl-beta-D-glucose + 1-O-galloyl-beta-D-glucose = 1,2,3,4,6-pentakis-O-galloyl-beta-D-glucose + D-glucose." [EC:2.3.1.143, RHEA:19112]
GO:0047177
name: glycerophospholipid arachidonoyl-transferase (CoA-independent) activity
def: "Catalysis of the reaction: 1-alkyl-2-lyso-sn-glycero-3-phosphoethanolamine + 1-alkyl-2-arachidonyl-sn-glycero-3-phosphocholine = 1-alkyl-2-lyso-sn-glycero-3-phosphocholine + 1-alkyl-2-arachidonyl-sn-glycero-3-phosphoethanolamine." [EC:2.3.1.147, MetaCyc:2.3.1.147-RXN]
GO:0047178
name: glycerophospholipid acyltransferase (CoA-dependent) activity
def: "Catalysis of the reaction: 1-radyl-2-lyso-sn-glycero-3-phosphoethanolamine + 1-radyl-2-acyl-sn-glycero-3-phosphocholine = 1-radyl-2-lyso-sn-glycero-3-phosphocholine + 1-radyl-2-acyl-sn-glycero-3-phosphoethanolamine." [EC:2.3.1.148, MetaCyc:2.3.1.148-RXN]
GO:0047179
name: platelet-activating factor acetyltransferase activity
def: "Catalysis of the reaction: 1-radyl-2-acyl-sn-glycero-3-phospholipid + 1-alkyl-2-acetyl-sn-glycero-3-phosphocholine = 1-alkyl-2-lyso-sn-glycero-3-phosphocholine + 1-radyl-2-acetyl-sn-glycero-3-phospholipid." [EC:2.3.1.149, MetaCyc:2.3.1.149-RXN]
GO:0047180
name: salutaridinol 7-O-acetyltransferase activity
def: "Catalysis of the reaction: (7S)-salutaridinol + acetyl-CoA = (7S)-O-acetylsalutaridinol + CoA." [EC:2.3.1.150, RHEA:22859]
GO:0047181
name: benzophenone synthase activity
def: "Catalysis of the reaction: 3-hydroxybenzoyl-CoA + 3 malonyl-CoA = 3 CO2 + 2,3',4,6-tetrahydroxybenzophenone + 4 coenzyme A." [EC:2.3.1.151, MetaCyc:2.3.1.151-RXN]
GO:0047182
name: alcohol O-cinnamoyltransferase activity
def: "Catalysis of the reaction: an alcohol + 1-O-trans-cinnamoyl-beta-D-glucopyranose = beta-D-glucose + alkyl cinnamate." [EC:2.3.1.152, MetaCyc:2.3.1.152-RXN]
GO:0047183
name: anthocyanin 5-aromatic acyltransferase activity
def: "Catalysis of the reaction: anthocyanidin-3,5-diglucoside + hydroxycinnamoyl-CoA = anthocyanidin 3-glucoside-5-hydroxycinnamoylglucoside + CoA." [EC:2.3.1.153, MetaCyc:2.3.1.153-RXN]
GO:0047184
name: 1-acylglycerophosphocholine O-acyltransferase activity
def: "Catalysis of the reaction: 1-acyl-sn-glycero-3-phosphocholine + acyl-CoA = phosphatidylcholine + CoA." [EC:2.3.1.23, MetaCyc:2.3.1.23-RXN]
GO:0047185
name: N-acetylneuraminate 4-O-acetyltransferase activity
def: "Catalysis of the reaction: N-acetylneuraminate + acetyl-CoA = N-acetyl-4-O-acetylneuraminate + CoA." [EC:2.3.1.44, RHEA:18308]
GO:0047186
name: N-acetylneuraminate 7-O(or 9-O)-acetyltransferase activity
def: "Catalysis of the reaction: N-acetylneuraminate + acetyl-CoA = N-acetyl-7-O(or 9-O)-acetylneuraminate + CoA." [EC:2.3.1.45, MetaCyc:2.3.1.45-RXN]
GO:0047187
name: deacetyl-[citrate-(pro-3S)-lyase] S-acetyltransferase activity
def: "Catalysis of the reaction: deacetyl-[citrate-oxaloacetate-lyase ((pro-3S)-CH(2)COO(-)-acetate)] + S-acetylphosphopantetheine = [citrate oxaloacetate-lyase ((pro-3S)-CH(2)COO(-)-acetate)] + pantetheine 4'-phosphate." [EC:2.3.1.49, MetaCyc:2.3.1.49-RXN]
GO:0047188
name: aromatic-hydroxylamine O-acetyltransferase activity
def: "Catalysis of the reaction: N-hydroxy-4-aminobiphenyl + N-hydroxy-4-acetylaminonbiphenyl = N-acetoxy-4-aminobiphenyl + N-hydroxy-4-aminobiphenyl." [EC:2.3.1.56, MetaCyc:2.3.1.56-RXN]
GO:0047189
name: 2,3-diaminopropionate N-oxalyltransferase activity
def: "Catalysis of the reaction: 3-amino-L-alanine + oxalyl-CoA = N(3)-oxalyl-L-2,3-diaminopropanoate + CoA." [EC:2.3.1.58, RHEA:13468]
GO:0047190
name: 2-acylglycerophosphocholine O-acyltransferase activity
def: "Catalysis of the reaction: 2-acyl-sn-glycero-3-phosphocholine + acyl-CoA = 1,2-diacyl-sn-glycero-3-phosphocholine + CoA." [EC:2.3.1.62, RHEA:10335]
GO:0047191
name: 1-alkylglycerophosphocholine O-acyltransferase activity
def: "Catalysis of the reaction: 1-alkyl-sn-glycero-3-phosphocholine + acyl-CoA = 1-alkyl-2-acyl-sn-glycero-3-phosphocholine + CoA." [EC:2.3.1.63, MetaCyc:2.3.1.63-RXN]
GO:0047192
name: 1-alkylglycerophosphocholine O-acetyltransferase activity
def: "Catalysis of the reaction: 1-alkyl-sn-glycero-3-phosphocholine + acetyl-CoA = 1-alkyl-2-acetyl-sn-glycero-3-phosphocholine + CoA." [EC:2.3.1.67, MetaCyc:2.3.1.67-RXN]
GO:0047193
name: CDP-acylglycerol O-arachidonoyltransferase activity
def: "OBSOLETE. Catalysis of the reaction: CDP-acylglycerol + arachidonyl-CoA = CDP-diacylglycerol + CoA." [EC:2.3.1.70, MetaCyc:2.3.1.70-RXN]
GO:0047194
name: indoleacetylglucose-inositol O-acyltransferase activity
def: "Catalysis of the reaction: 1-O-(indol-3-ylacetyl)-beta-D-glucose + myo-inositol = 1L-1-O-(indol-3-yl)acetyl-myo-inositol + D-glucose." [EC:2.3.1.72, RHEA:21183]
GO:0047195
name: diacylglycerol-sterol O-acyltransferase activity
def: "Catalysis of the reaction: sterol + 1,2-diacylglycerol = sterol ester + acylglycerol." [EC:2.3.1.73, MetaCyc:2.3.1.73-RXN]
GO:0047196
name: long-chain-alcohol O-fatty-acyltransferase activity
def: "Catalysis of the reaction: a long-chain-alcohol + acyl-CoA = a long-chain ester + CoA." [EC:2.3.1.75, MetaCyc:2.3.1.75-RXN]
GO:0047197
name: triglyceride-sterol O-acyltransferase activity
def: "Catalysis of the reaction: a 3-beta-hydroxysterol + triacylglycerol = a 3-beta-hydroxysterol ester + 1,2-diacylglycerol." [EC:2.3.1.77, MetaCyc:2.3.1.77-RXN]
GO:0047198
name: cysteine-S-conjugate N-acetyltransferase activity
def: "Catalysis of the reaction: S-substituted L-cysteine + acetyl-CoA = S-substituted N-acetyl-L-cysteine + CoA + H(+)." [EC:2.3.1.80, RHEA:19216]
GO:0047199
name: phosphatidylcholine-dolichol O-acyltransferase activity
def: "Catalysis of the reaction: 1,2-diacyl-sn-glycero-3-phosphocholine + dolichol = 1-acyl-sn-glycero-3-phosphocholine + acyldolichol." [EC:2.3.1.83, RHEA:19288]
GO:0047200
name: tetrahydrodipicolinate N-acetyltransferase activity
def: "Catalysis of the reaction: (S)-2,3,4,5-tetrahydrodipicolinate + acetyl-CoA + H(2)O = L-2-acetamido-6-oxopimelate + CoA." [EC:2.3.1.89, RHEA:13088]
GO:0047201
name: beta-glucogallin O-galloyltransferase activity
def: "Catalysis of the reaction: 2 1-O-galloyl-beta-D-glucose = 1,6-bis-O-galloyl-beta-D-glucose + D-glucose." [EC:2.3.1.90, RHEA:11419]
GO:0047202
name: sinapoylglucose-choline O-sinapoyltransferase activity
def: "Catalysis of the reaction: 1-O-sinapoyl-beta-D-glucose + choline = O-sinapoylcholine + D-glucose." [EC:2.3.1.91, RHEA:12027]
GO:0047203
name: 13-hydroxylupinine O-tigloyltransferase activity
def: "Catalysis of the reaction: 13-hydroxylupanine + 2-methylcrotonoyl-CoA = 13-(2-methylcrotonoyloxy)lupanine + CoA." [EC:2.3.1.93, RHEA:12363]
GO:0047204
name: chlorogenate-glucarate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: D-glucarate + chlorogenate = (-)-quinate + 2-O-caffeoylglucarate." [EC:2.3.1.98, RHEA:23207]
GO:0047205
name: quinate O-hydroxycinnamoyltransferase activity
def: "Catalysis of the reaction: feruloyl-CoA + quinate = O-feruloylquinate + CoA." [EC:2.3.1.99, MetaCyc:2.3.1.99-RXN]
GO:0047206
name: UDP-N-acetylmuramoylpentapeptide-lysine N6-alanyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine + L-alanyl-tRNA = UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-N6-(L-alanyl)-L-lysyl-D-alanyl-D-alanine + tRNA." [EC:2.3.2.10, MetaCyc:2.3.2.10-RXN]
GO:0047207
name: 1,2-beta-fructan 1F-fructosyltransferase activity
def: "Catalysis of the reaction: [(1->2)-beta-D-fructosyl](n) + [(1->2)-beta-D-fructosyl](m) = [(1->2)-beta-D-fructosyl](n+1) + [(1->2)-beta-D-fructosyl](m-1)." [EC:2.4.1.100, MetaCyc:2.4.1.100-RXN]
GO:0047208
name: o-dihydroxycoumarin 7-O-glucosyltransferase activity
def: "Catalysis of the reaction: 7,8-dihydroxycoumarin + UDP-D-glucose = daphnin + H(+) + UDP." [EC:2.4.1.104, RHEA:14328]
GO:0047209
name: coniferyl-alcohol glucosyltransferase activity
def: "Catalysis of the reaction: coniferyl alcohol + UDP-D-glucose = coniferin + UDP." [EC:2.4.1.111, MetaCyc:2.4.1.111-RXN]
GO:0047211
name: alpha-1,4-glucan-protein synthase (ADP-forming) activity
def: "Catalysis of the reaction: ADP-D-glucose + protein = alpha-D-glucosyl-protein + ADP." [EC:2.4.1.113, MetaCyc:2.4.1.113-RXN]
GO:0047212
name: 2-coumarate O-beta-glucosyltransferase activity
def: "Catalysis of the reaction: trans-2-coumarate + UDP-D-glucose = trans-beta-D-glucosyl-2-hydroxycinnamate + H(+) + UDP." [EC:2.4.1.114, RHEA:10239]
GO:0047213
name: anthocyanidin 3-O-glucosyltransferase activity
def: "Catalysis of the reaction: anthocyanidin + UDP-D-glucose = anthocyanidin-3-O-D-glucoside + UDP." [EC:2.4.1.115, MetaCyc:2.4.1.115-RXN]
GO:0047214
name: cyanidin-3-rhamnosylglucoside 5-O-glucosyltransferase activity
def: "Catalysis of the reaction: cyanidin-3-O-D-rhamnosyl-(1,6)-D-glucoside + UDP-D-glucose = cyanidin-3-O-[D-rhamnosyl-(1,6)-D-glucoside]-5-O-D-glucoside + UDP." [EC:2.4.1.116, MetaCyc:2.4.1.116-RXN]
GO:0047215
name: indole-3-acetate beta-glucosyltransferase activity
def: "Catalysis of the reaction: (indol-3-yl)acetate + UDP-D-glucose = 1-O-(indol-3-ylacetyl)-beta-D-glucose + UDP." [EC:2.4.1.121, RHEA:14924]
GO:0047216
name: inositol 3-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: myo-inositol + UDP-galactose = O-alpha-D-galactosyl-(1,3)-1D-myo-inositol + UDP." [EC:2.4.1.123, MetaCyc:2.4.1.123-RXN]
GO:0047217
name: sucrose-1,6-alpha-glucan 3(6)-alpha-glucosyltransferase activity
def: "Catalysis of the reaction: sucrose + 1,6-alpha-D-glucosyl(n) = 1,6-alpha-D-glucosyl(n+1) + fructose." [EC:2.4.1.125, MetaCyc:2.4.1.125-RXN]
GO:0047218
name: hydroxycinnamate 4-beta-glucosyltransferase activity
def: "Catalysis of the reaction: 4-coumarate + UDP-D-glucose = 4-O-beta-D-glucosyl-4-hydroxycinnamate + UDP." [EC:2.4.1.126, MetaCyc:2.4.1.126-RXN]
GO:0047219
name: monoterpenol beta-glucosyltransferase activity
def: "Catalysis of the reaction: (-)-menthol + UDP-D-glucose = (-)-menthyl beta-D-glucoside + H(+) + UDP." [EC:2.4.1.127, RHEA:11523]
GO:0047220
name: galactosylxylosylprotein 3-beta-galactosyltransferase activity
def: "Catalysis of the reaction: 4-beta-D-galactosyl-O-beta-D-xylosylprotein + UDP-galactose = 3-beta-D-galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein + UDP." [EC:2.4.1.134, MetaCyc:2.4.1.134-RXN]
GO:0047221
name: sn-glycerol-3-phosphate 2-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + UDP-D-galactose = 2-(alpha-D-galactosyl)-sn-glycerol 3-phosphate + H(+) + UDP." [EC:2.4.1.137, RHEA:14288]
GO:0047222
name: mannotetraose 2-alpha-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: 1,3-alpha-D-mannosyl-1,2-alpha-D-mannosyl-1,2-alpha-D-mannosyl-D-mannose + UDP-N-acetyl-D-glucosamine = 1,3-alpha-D-mannosyl-1,2-(N-acetyl-alpha-D-glucosaminyl-alpha-D-mannosyl)-1,2-alpha-D-mannosyl-D-mannose + UDP." [EC:2.4.1.138, MetaCyc:2.4.1.138-RXN]
GO:0047223
name: beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: beta-D-galactosyl-1,3-(N-acetyl-D-glucosaminyl-1,6)-N-acetyl-D-galactosaminyl-R + UDP-N-acetyl-D-glucosamine = N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,3-(N-acetyl-beta-D-glucosaminyl-1,6)-N-acetyl-D-galactosaminyl-R + UDP." [EC:2.4.1.146, MetaCyc:2.4.1.146-RXN]
GO:0047224
name: acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: N-acetyl-D-galactosalaminyl-R + UDP-N-acetyl-D-glucosamine = N-acetyl-beta-D-glucosaminyl-1,3-N-acetyl-D-galactosaminyl-R + UDP." [EC:2.4.1.147, MetaCyc:2.4.1.147-RXN]
GO:0047225
name: acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: N-acetyl-beta-D-glucosaminyl-1,3-N-acetyl-D-galactosaminyl-R + UDP-N-acetyl-D-glucosamine = N-acetyl-beta-D-glucosaminyl-1,6-(N-acetyl-beta-D-glucosaminyl-1,3)-N-acetyl-D-galactosaminyl-R + UDP." [EC:2.4.1.148, MetaCyc:2.4.1.148-RXN]
GO:0047227
name: indolylacetyl-myo-inositol galactosyltransferase activity
def: "Catalysis of the reaction: 1L-1-O-(indol-3-yl)acetyl-myo-inositol + UDP-D-galactose = 5-O-(indol-3-ylacetyl)-myo-inositol D-galactoside + H(+) + UDP." [EC:2.4.1.156, RHEA:21151]
GO:0047228
name: 1,2-diacylglycerol 3-glucosyltransferase activity
def: "Catalysis of the reaction: 1,2-diacylglycerol + UDP-D-glucose = 3-D-glucosyl-1,2-diacylglycerol + UDP." [EC:2.4.1.157, MetaCyc:2.4.1.157-RXN]
GO:0047229
name: 13-hydroxydocosanoate 13-beta-glucosyltransferase activity
def: "Catalysis of the reaction: 13-hydroxydocosanoate + UDP-D-glucose = 13-beta-D-glucosyloxydocosanoate + UDP." [EC:2.4.1.158, MetaCyc:2.4.1.158-RXN]
GO:0047230
name: flavonol-3-O-glucoside L-rhamnosyltransferase activity
def: "Catalysis of the reaction: flavonol 3-O-D-glucoside + UDP-L-rhamnose = flavonol 3-O-L-rhamnosylglucoside + UDP." [EC:2.4.1.159, MetaCyc:2.4.1.159-RXN]
GO:0047231
name: pyridoxine 5'-O-beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: pyridoxine + UDP-D-glucose = 5'-O-beta-D-glucosylpyridoxine + H(+) + UDP." [EC:2.4.1.160, RHEA:20180]
GO:0047232
name: galactosyl-N-acetylglucosaminylgalactosylglucosyl-ceramide beta-1,6-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide + UDP-N-acetyl-D-glucosamine = N-acetyl-D-glucosaminyl-1,6-beta-D-galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosyceramide + UDP." [EC:2.4.1.164, MetaCyc:2.4.1.164-RXN]
GO:0047233
name: N-acetylneuraminylgalactosylglucosylceramide beta-1,4-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: N-acetylneuraminyl-2,3-alpha-D-galactosyl-1,4-beta-D-glucosylceramide + UDP-N-acetylgalactosamine = N-acetyl-beta-D-galactosaminyl-1,4-(N-acetyl-alpha-neuraminyl-2,3)-beta-D-galactosyl-1,4-beta-D-glucosylceramide + UDP." [EC:2.4.1.165, MetaCyc:2.4.1.165-RXN]
GO:0047234
name: raffinose-raffinose alpha-galactotransferase activity
def: "Catalysis of the reaction: 2 raffinose = sucrose + 1F-alpha-D-galactosylraffinose." [EC:2.4.1.166, MetaCyc:2.4.1.166-RXN]
GO:0047235
name: sucrose 6F-alpha-galactotransferase activity
def: "Catalysis of the reaction: sucrose + UDP-galactose = 6F-alpha-D-galactosylsucrose + UDP." [EC:2.4.1.167, MetaCyc:2.4.1.167-RXN]
GO:0047236
name: methyl-ONN-azoxymethanol beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: methylazoxymethanol + UDP-D-glucose = H+ + cycasin + UDP." [EC:2.4.1.171, MetaCyc:2.4.1.171-RXN]
GO:0047237
name: glucuronylgalactosylproteoglycan 4-beta-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: D-glucuronyl-1,3-beta-D-galactosylproteoglycan + UDP-N-acetylgalactosamine = N-acetyl-D-galactosaminyl-1,4-beta-D-glucuronyl-1,3-beta-D-galactosylproteoglycan + UDP." [EC:2.4.1.174, MetaCyc:2.4.1.174-RXN]
GO:0047238
name: glucuronosyl-N-acetylgalactosaminyl-proteoglycan 4-beta-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: D-glucuronyl-N-acetyl-1,3-beta-D-galactosaminylproteoglycan + UDP-N-acetylgalactosamine = N-acetyl-D-galactosaminyl-1,4-beta-D-glucuronyl-N-acetyl-1,3-beta-D-galactosaminylproteoglycan + UDP." [EC:2.4.1.175, MetaCyc:2.4.1.175-RXN]
GO:0047239
name: hydroxymandelonitrile glucosyltransferase activity
def: "Catalysis of the reaction: 4-hydroxymandelonitrile + UDP-D-glucose = H(+) + taxiphyllin + UDP." [EC:2.4.1.178, RHEA:15964]
GO:0047240
name: lactosylceramide beta-1,3-galactosyltransferase activity
def: "Catalysis of the reaction: D-galactosyl-(1->4)-beta-D-glucosyl-R + UDP-D-galactose = D-galactosyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-glucosyl-R + H(+) + UDP." [EC:2.4.1.179, RHEA:18416]
GO:0047241
name: lipopolysaccharide N-acetylmannosaminouronosyltransferase activity
def: "Catalysis of the reaction: lipopolysaccharide + UDP-N-acetylmannosaminouronate = N-acetyl-beta-D-mannosaminouronosyl-1,4-lipopolysaccharide + UDP." [EC:2.4.1.180, MetaCyc:2.4.1.180-RXN]
GO:0047242
name: hydroxyanthraquinone glucosyltransferase activity
def: "Catalysis of the reaction: a hydroxyanthraquinone + UDP-D-glucose = a glucosyloxyanthraquinone + UDP." [EC:2.4.1.181, MetaCyc:2.4.1.181-RXN]
GO:0047243
name: flavanone 7-O-beta-glucosyltransferase activity
def: "Catalysis of the reaction: a flavanone + UDP-D-glucose = a flavanone 7-O-beta-D-glucoside + UDP." [EC:2.4.1.185, MetaCyc:2.4.1.185-RXN]
GO:0047244
name: N-acetylglucosaminyldiphosphoundecaprenol N-acetyl-beta-D-mannosaminyltransferase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosaminyldiphosphoundecaprenol + UDP-N-acetyl-D-mannosamine = N-acetyl-beta-D-mannosaminyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenol + UDP." [EC:2.4.1.187, MetaCyc:2.4.1.187-RXN]
GO:0047245
name: N-acetylglucosaminyldiphosphoundecaprenol glucosyltransferase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosaminyldiphosphoundecaprenol + UDP-D-glucose = beta-D-glucosyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenol + UDP." [EC:2.4.1.188, MetaCyc:2.4.1.188-RXN]
GO:0047246
name: luteolin-7-O-glucuronide 7-O-glucuronosyltransferase activity
def: "Catalysis of the reaction: luteolin 7-O-beta-D-glucosiduronate + UDP-alpha-D-glucuronate = H(+) + luteolin 7-O-[(beta-D-glucosiduronate)-(1->2)-(beta-D-glucosiduronate)] + UDP." [EC:2.4.1.190, RHEA:14152]
GO:0047247
name: luteolin-7-O-diglucuronide 4'-O-glucuronosyltransferase activity
def: "Catalysis of the reaction: luteolin 7-O-[(beta-D-glucosiduronate)-(1->2)-(beta-D-glucosiduronate)] + UDP-alpha-D-glucuronate = H(+) + luteolin 7-O-[(beta-D-glucosiduronate)-(1->2)-(beta-D-glucosiduronate)] 4'-O-beta-D-glucosiduronate + UDP." [EC:2.4.1.191, RHEA:22119]
GO:0047248
name: nuatigenin 3-beta-glucosyltransferase activity
def: "Catalysis of the reaction: nuatigenin + UDP-D-glucose = H(+) + nuatigenin 3-beta-D-glucopyranoside + UDP." [EC:2.4.1.192, RHEA:19332]
GO:0047249
name: sarsapogenin 3-beta-glucosyltransferase activity
def: "Catalysis of the reaction: (25S)-5beta-spirostan-3beta-ol + UDP-D-glucose = (25S)-5beta-spirostan-3beta-yl beta-D-glucoside + H(+) + UDP." [EC:2.4.1.193, RHEA:14464]
GO:0047250
name: 4-hydroxybenzoate 4-O-beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + UDP-D-glucose = 4-(beta-D-glucosyloxy)benzoate + H(+) + UDP." [EC:2.4.1.194, RHEA:15156]
GO:0047251
name: thiohydroximate beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: phenylthioacetohydroximate + UDP-D-glucose = desulfoglucotropeolin + UDP." [EC:2.4.1.195, MetaCyc:2.4.1.195-RXN]
GO:0047252
name: beta-mannosylphosphodecaprenol-mannooligosaccharide 6-mannosyltransferase activity
def: "Catalysis of the reaction: (1->6)-alpha-D-mannosyloligosaccharide + beta-D-mannosylphosphodecaprenol = (1->6)-alpha-D-mannosyl-(1->6)-alpha-D-mannosyl-oligosaccharide + decaprenol phosphate." [EC:2.4.1.199, MetaCyc:2.4.1.199-RXN]
GO:0047253
name: alpha-1,6-mannosylglycoprotein 4-beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: N-acetyl-beta-D-glucosaminyl-1,6-beta-D-(N-acetyl-B-glucosaminyl-1,2)-beta-D-mannosyl-R + UDP-N-acetyl-D-glucosamine = N-acetyl-beta-D-glucosaminyl-1,6-beta-D-(N-acetyl-D-glucosaminyl-1,2-beta)-(N-acetyl-D-glucosaminyl-1,4-beta)-D-mannosyl-R + UDP." [EC:2.4.1.201, MetaCyc:2.4.1.201-RXN]
GO:0047254
name: 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one 2-D-glucosyltransferase activity
def: "Catalysis of the reaction: 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one + UDP-D-glucose = 2,4-dihydroxy-7-methoxy-2H-1,4-benzoxazin-3(4H)-one 2-D-glucoside + UDP." [EC:2.4.1.202, MetaCyc:2.4.1.202-RXN]
GO:0047255
name: galactogen 6-beta-galactosyltransferase activity
def: "Catalysis of the reaction: galactogen + UDP-galactose = 1,6-beta-D-galctosylgalactogen + UDP." [EC:2.4.1.205, MetaCyc:2.4.1.205-RXN]
GO:0047256
name: lactosylceramide 1,3-N-acetyl-beta-D-glucosaminyltransferase activity
def: "Catalysis of the reaction: cytolipin-H + UDP-N-acetyl-D-glucosamine = N-acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide + UDP." [EC:2.4.1.206, MetaCyc:2.4.1.206-RXN]
GO:0047257
name: diglucosyl diacylglycerol synthase activity
def: "Catalysis of the reaction: 1,2-diacyl-3-O-(alpha-D-glucopyranosyl)-sn-glycerol + UDP-D-glucose = UDP + 1,2-diacyl-3-O-(alpha-D-glucopyranosyl(1,2)-O-alpha-D-glucopyranosyl)-sn-glycerol." [EC:2.4.1.208, MetaCyc:2.4.1.208-RXN]
GO:0047258
name: sphingosine beta-galactosyltransferase activity
def: "Catalysis of the reaction: sphingosine + UDP-D-galactose = H(+) + psychosine + UDP." [EC:2.4.1.23, RHEA:19488]
GO:0047259
name: glucomannan 4-beta-mannosyltransferase activity
def: "Catalysis of the reaction: glucomannan(n) + GDP-mannose = glucomannan(n+1) + GDP." [EC:2.4.1.32, MetaCyc:2.4.1.32-RXN]
GO:0047260
name: alpha,alpha-trehalose-phosphate synthase (GDP-forming) activity
def: "Catalysis of the reaction: GDP-D-glucose + glucose-6-phosphate = alpha,alpha-trehalose 6-phosphate + GDP." [EC:2.4.1.36, MetaCyc:2.4.1.36-RXN]
GO:0047261
name: steroid N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: estradiol-17alpha 3-D-glucuronoside + UDP-N-acetyl-alpha-D-glucosamine = 17alpha-(N-acetyl-D-glucosaminyl)-estradiol 3-D-glucuronoside + H(+) + UDP." [EC:2.4.1.39, RHEA:14156]
GO:0047262
name: polygalacturonate 4-alpha-galacturonosyltransferase activity
def: "Catalysis of the reaction: UDP-D-galacturonate + 1,4-alpha-D-galacturonosyl(n) = 1,4-alpha-D-galacturonosyl(n+1) + UDP." [EC:2.4.1.43, MetaCyc:2.4.1.43-RXN]
GO:0047263
name: N-acylsphingosine galactosyltransferase activity
def: "Catalysis of the reaction: ceramide + UDP-galactose = D-galactosylceramide + UDP." [EC:2.4.1.47, MetaCyc:2.4.1.47-RXN]
GO:0047264
name: heteroglycan alpha-mannosyltransferase activity
def: "Catalysis of the reaction: heteroglycan + GDP-mannose = alpha-D-mannosylheteroglycan + GDP." [EC:2.4.1.48]
GO:0047265
name: poly(glycerol-phosphate) alpha-glucosyltransferase activity
def: "Catalysis of the reaction: poly(glycerol phosphate) + UDP-D-glucose = alpha-D-glucosylpoly(glycerol phosphate) + UDP." [EC:2.4.1.52, MetaCyc:2.4.1.52-RXN]
GO:0047266
name: poly(ribitol-phosphate) beta-glucosyltransferase activity
def: "Catalysis of the reaction: poly(ribitol phosphate) + UDP-D-glucose = beta-D-glucosylpoly(ribitol phosphate) + UDP." [EC:2.4.1.53, MetaCyc:2.4.1.53-RXN]
GO:0047267
name: undecaprenyl-phosphate mannosyltransferase activity
def: "Catalysis of the reaction: GDP-mannose + undecaprenyl phosphate = GDP + D-mannosyl-1-phosphoundecaprenol." [EC:2.4.1.54]
GO:0047268
name: galactinol-raffinose galactosyltransferase activity
def: "Catalysis of the reaction: raffinose + 1-alpha-D-galactosyl-myo-inositol = stachyose + myo-inositol." [EC:2.4.1.67, MetaCyc:2.4.1.67-RXN]
GO:0047269
name: poly(ribitol-phosphate) N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: poly(ribitol phosphate) + UDP-N-acetyl-D-glucosamine = N-acetyl-D-glucosaminyl-poly(ribitol phosphate) + UDP." [EC:2.4.1.70, MetaCyc:2.4.1.70-RXN]
GO:0047270
name: lipopolysaccharide glucosyltransferase II activity
def: "Catalysis of the reaction: lipopolysaccharide + UDP-D-glucose = D-glucosyl-lipopolysaccharide + UDP." [EC:2.4.1.73, GOC:mr, GOC:pr, MetaCyc:2.4.1.73-RXN]
GO:0047271
name: glycosaminoglycan galactosyltransferase activity
def: "Catalysis of the reaction: glycosaminoglycan + UDP-galactose = D-galactosylglycosaminoglycan + UDP." [EC:2.4.1.74, MetaCyc:2.4.1.74-RXN]
GO:0047272
name: phosphopolyprenol glucosyltransferase activity
def: "Catalysis of the reaction: polyprenyl phosphate + UDP-D-glucose = polyprenylphosphate-glucose + UDP." [EC:2.4.1.78, MetaCyc:2.4.1.78-RXN]
GO:0047273
name: galactosylgalactosylglucosylceramide beta-D-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-galactosamine + alpha-D-galactosyl-(1->4)-beta-D-galactosyl-(1->4)-beta-D-glucosylceramide = UDP + beta-N-acetyl-D-galactosaminyl-(1->3)-alpha-D-galactosyl-(1->4)-beta-D-galactosyl-(1->4)-beta-D-glucosylceramide." [EC:2.4.1.79, MetaCyc:2.4.1.79-RXN]
GO:0047274
name: galactinol-sucrose galactosyltransferase activity
def: "Catalysis of the reaction: sucrose + 1-alpha-D-galactosyl-myo-inositol = raffinose + myo-inositol." [EC:2.4.1.82, MetaCyc:2.4.1.82-RXN]
GO:0047275
name: glucosaminylgalactosylglucosylceramide beta-galactosyltransferase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosaminyl-(1,3)-D-galactosyl-(1,4)-D-glucosylceramide + UDP-galactose = D-galactosyl-N-acetyl-D-glucosaminyl-(1,3)-D-galactosyl-(1,4)-D-glucosylceramide + UDP." [EC:2.4.1.86, MetaCyc:2.4.1.86-RXN]
GO:0047276
name: N-acetyllactosaminide 3-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: beta-D-galactosyl-(1,4)-beta-N-acetyl-D-glucosaminyl-R + UDP-galactose = alpha-D-galactosyl-(1,3)-beta-D-galactosyl-(1,4)-beta-N-acetyl-D-glucosaminyl-R + UDP." [EC:2.4.1.87, MetaCyc:2.4.1.87-RXN]
GO:0047277
name: globoside alpha-N-acetylgalactosaminyltransferase activity
def: "Catalysis of the reaction: N-acetyl-D-galactosaminyl-(1,3)-D-galactosyl-(1,4)-D-galactosyl-(1,4)-D-glucosylceramide + UDP-N-acetylgalactosamine = N-acetyl-D-galactosaminyl-N-acetyl-D-galactosaminyl-(1,3)-D-galactosyl-(1,4)-D-galactosyl-(1,4)-D-glucosylceramide + UDP." [EC:2.4.1.88, MetaCyc:2.4.1.88-RXN]
GO:0047278
name: bilirubin-glucuronoside glucuronosyltransferase activity
def: "Catalysis of the reaction: 2 bilirubin-glucuronoside = bilirubin + bilirubin-bisglucuronoside." [EC:2.4.1.95, RHEA:16888]
GO:0047279
name: sn-glycerol-3-phosphate 1-galactosyltransferase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + UDP-D-galactose = 1-O-alpha-D-galactosyl-sn-glycerol 3-phosphate + H(+) + UDP." [EC:2.4.1.96, RHEA:20344]
GO:0047280
name: nicotinamide phosphoribosyltransferase activity
def: "Catalysis of the reaction: diphosphate + nicotinamide mononucleotide = 5-phospho-alpha-D-ribose 1-diphosphate + H(+) + nicotinamide." [EC:2.4.2.12, RHEA:16152]
GO:0047281
name: dioxotetrahydropyrimidine phosphoribosyltransferase activity
def: "Catalysis of the reaction: pyrophosphate + a 2,4-dioxotetrahydropyrimidine D-ribonucleotide = PRPP + a 2,4-dioxotetrahydropyrimidine." [EC:2.4.2.20, MetaCyc:2.4.2.20-RXN]
GO:0047282
name: dTDP-dihydrostreptose-streptidine-6-phosphate dihydrostreptosyltransferase activity
def: "Catalysis of the reaction: dTDP-L-dihydrostreptose + streptidine 6-phosphate = O-(1->4)-alpha-L-dihydrostreptosyl-streptidine 6-phosphate + dTDP + H(+)." [EC:2.4.2.27, RHEA:24395]
GO:0047283
name: dolichyl-phosphate D-xylosyltransferase activity
def: "Catalysis of the reaction: dolichol-phosphate + UDP-D-xylose = dolichyl D-xylosyl phosphate + UDP." [EC:2.4.2.32, MetaCyc:2.4.2.32-RXN]
GO:0047284
name: dolichyl-xylosyl-phosphate-protein xylosyltransferase activity
def: "Catalysis of the reaction: dolichyl D-xylosyl phosphate + protein = dolichol-phosphate + D-xylosylprotein." [EC:2.4.2.33, MetaCyc:2.4.2.33-RXN]
GO:0047285
name: flavonol-3-O-glycoside xylosyltransferase activity
def: "Catalysis of the reaction: flavonol 3-O-glycoside + UDP-D-xylose = flavonol 3-O-D-xylosylglycoside + UDP." [EC:2.4.2.35, MetaCyc:2.4.2.35-RXN]
GO:0047286
name: NAD+-diphthamide ADP-ribosyltransferase activity
def: "Catalysis of the reaction: peptide diphthamide + NAD+ = peptide N-(ADP-D-ribosyl)diphthamide + niacinamide." [EC:2.4.2.36]
GO:0047287
name: lactosylceramide alpha-2,6-N-sialyltransferase activity
def: "Catalysis of the reaction: cytolipin-H + CMP-N-acetylneuraminate = alpha-N-acetylneuraminyl-2,6-beta-galactosyl-1,4-beta-D-glucosylceramide + CMP." [EC:2.4.99.11, MetaCyc:2.4.99.11-RXN]
GO:0047288
name: monosialoganglioside sialyltransferase activity
def: "Catalysis of the reaction: D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide + CMP-N-acetylneuraminate = N-acetylneuraminyl-D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide + CMP." [EC:2.4.99.2, MetaCyc:2.4.99.2-RXN]
GO:0047289
name: galactosyldiacylglycerol alpha-2,3-sialyltransferase activity
def: "Catalysis of the reaction: 1,2-diacyl-3-beta-D-galactosyl-sn-glycerol + CMP-N-acetyl-beta-neuraminate = 1,2-diacyl-3-[3-(alpha-D-N-acetylneuraminyl)-beta-D-galactosyl]-sn-glycerol + CMP + H(+)." [EC:2.4.99.5, RHEA:11667]
GO:0047290
name: (alpha-N-acetylneuraminyl-2,3-beta-galactosyl-1,3)-N-acetyl-galactosaminide 6-alpha-sialyltransferase activity
def: "Catalysis of the reaction: alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-(1->3)-N-acetyl-D-galactosaminyl-R + CMP-N-acetyl-beta-neuraminate = alpha-N-acetylneuraminyl-(2->3)-beta-D-galactosyl-(1->3)-[N-acetyl-alpha-neuraminyl-(2->6)]-N-acetyl-D-galactosaminyl-R + CMP." [EC:2.4.99.7, RHEA:10479]
GO:0047291
name: lactosylceramide alpha-2,3-sialyltransferase activity
def: "Catalysis of the reaction: cytolipin-H + CMP-N-acetylneuraminate = alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide + CMP. Alpha-N-acetylneuraminyl-2,3-beta-D-galactosyl-1,4-beta-D-glucosylceramide is also known as GM3." [EC:2.4.99.9, MetaCyc:2.4.99.9-RXN]
GO:0047292
name: trihydroxypterocarpan dimethylallyltransferase activity
def: "Catalysis of the reaction: (6AS,11AS)-3,6A,9-trihydroxypterocarpan + dimethylallyl-pyrophosphate = glyceollin + diphosphate." [EC:2.5.1.36, MetaCyc:2.5.1.36-RXN]
GO:0047293
name: 4-hydroxybenzoate nonaprenyltransferase activity
def: "Catalysis of the reaction: p-hydroxybenzoate + solanesyl pyrophosphate = nonaprenyl-4-hydroxybenzoate + diphosphate." [EC:2.5.1.39, MetaCyc:2.5.1.39-RXN]
GO:0047294
name: phosphoglycerol geranylgeranyltransferase activity
def: "Catalysis of the reaction: sn-glycerol 1-phosphate + all-trans-geranylgeranyl diphosphate = sn-3-O-(geranylgeranyl)glycerol 1-phosphate + diphosphate." [EC:2.5.1.41, RHEA:23407]
GO:0047295
name: geranylgeranylglycerol-phosphate geranylgeranyltransferase activity
def: "Catalysis of the reaction: sn-3-O-(geranylgeranyl)glycerol 1-phosphate + all-trans-geranylgeranyl diphosphate = 2,3-di-O-(geranylgeranyl)glycerol 1-phosphate + diphosphate." [EC:2.5.1.42, RHEA:18112]
GO:0047296
name: homospermidine synthase activity
def: "Catalysis of the reaction: 2 putrescine = NH3 + sym-homospermidine." [EC:2.5.1.44, MetaCyc:2.5.1.44-RXN]
GO:0047297
name: asparagine-oxo-acid transaminase activity
def: "Catalysis of the reaction: a 2-oxo acid + L-asparagine = an amino acid + 2-oxosuccinamate." [EC:2.6.1.14, MetaCyc:2.6.1.14-RXN]
GO:0047298
name: (S)-3-amino-2-methylpropionate transaminase activity
def: "Catalysis of the reaction: (S)-3-amino-2-methylpropanoate + 2-oxoglutarate = 2-methyl-3-oxopropanoate + L-glutamate." [EC:2.6.1.22, RHEA:13996]
GO:0047299
name: tryptophan-phenylpyruvate transaminase activity
def: "Catalysis of the reaction: keto-phenylpyruvate + L-tryptophan = 3-(indol-3-yl)pyruvate + L-phenylalanine." [EC:2.6.1.28, RHEA:13744]
GO:0047300
name: pyridoxamine-pyruvate transaminase activity
def: "Catalysis of the reaction: pyridoxamine + pyruvate = L-alanine + pyridoxal." [EC:2.6.1.30, RHEA:12844]
GO:0047301
name: valine-3-methyl-2-oxovalerate transaminase activity
def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + L-valine = 3-methyl-2-oxobutanoate + L-isoleucine." [EC:2.6.1.32, RHEA:11471]
GO:0047302
name: UDP-2-acetamido-4-amino-2,4,6-trideoxyglucose transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + UDP-2-acetamido-4-amino-2,4,6-trideoxy-D-glucose = L-glutamate + UDP-2-acetamido-4-dehydro-2,6-dideoxy-beta-D-glucose." [EC:2.6.1.34, RHEA:18716]
GO:0047303
name: glycine-oxaloacetate transaminase activity
def: "Catalysis of the reaction: glycine + oxaloacetate = L-aspartate + glyoxylate." [EC:2.6.1.35, RHEA:17144]
GO:0047304
name: 2-aminoethylphosphonate-pyruvate transaminase activity
def: "Catalysis of the reaction: (2-aminoethyl)phosphonate + pyruvate = L-alanine + phosphonoacetaldehyde." [EC:2.6.1.37, RHEA:17024]
GO:0047305
name: (R)-3-amino-2-methylpropionate-pyruvate transaminase activity
def: "Catalysis of the reaction: (2R)-3-amino-2-methylpropanoate + pyruvate = 2-methyl-3-oxopropanoate + L-alanine." [EC:2.6.1.40, RHEA:18396]
GO:0047306
name: D-methionine-pyruvate transaminase activity
def: "Catalysis of the reaction: D-methionine + pyruvate = 4-methylthio-2-oxobutanoate + L-alanine." [EC:2.6.1.41, RHEA:23839]
GO:0047307
name: diaminobutyrate-pyruvate transaminase activity
def: "Catalysis of the reaction: L-2,4-diaminobutyrate + pyruvate = L-alanine + L-aspartate 4-semialdehyde." [EC:2.6.1.46, RHEA:12383]
GO:0047308
name: alanine-oxomalonate transaminase activity
def: "Catalysis of the reaction: L-alanine + oxomalonate = aminomalonate + pyruvate." [EC:2.6.1.47, RHEA:18812]
GO:0047309
name: dihydroxyphenylalanine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-dopa = 3,4-dihydroxyphenylpyruvate + L-glutamate." [EC:2.6.1.49, RHEA:15276]
GO:0047310
name: glutamine-scyllo-inositol transaminase activity
def: "Catalysis of the reaction: 2,4,6/3,5-pentahydroxycyclohexanone + L-glutamine = 1-amino-1-deoxy-scyllo-inositol + 2-oxoglutaramate." [EC:2.6.1.50, RHEA:22923]
GO:0047311
name: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol transaminase activity
def: "Catalysis of the reaction: 1D-1-guanidino-3-amino-1,3-dideoxy-scyllo-inositol + pyruvate = 1D-1-guanidino-1-deoxy-3-dehydro-scyllo-inositol + L-alanine." [EC:2.6.1.56, RHEA:15500]
GO:0047312
name: L-phenylalanine:pyruvate aminotransferase activity
def: "Catalysis of the reaction: pyruvate + L-phenylalanine = phenylpyruvate + L-alanine." [EC:2.6.1.58, MetaCyc:2.6.1.58-RXN]
GO:0047313
name: aromatic-amino-acid-glyoxylate transaminase activity
def: "Catalysis of the reaction: glyoxylate + an aromatic amino acid = L-glycine + an aromatic oxo acid." [EC:2.6.1.60, MetaCyc:2.6.1.60-RXN]
GO:0047315
name: kynurenine-glyoxylate transaminase activity
def: "Catalysis of the reaction: L-kynurenine + glyoxylate = 4-(2-aminophenyl)-2,4-dioxobutanoate + glycine." [EC:2.6.1.63, RHEA:19252]
GO:0047316
name: glutamine-phenylpyruvate transaminase activity
def: "Catalysis of the reaction: keto-phenylpyruvate + L-glutamine = 2-oxoglutaramate + L-phenylalanine." [EC:2.6.1.64, RHEA:17596]
GO:0047317
name: N6-acetyl-beta-lysine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + 3-amino-6-acetamidohexanoate = L-glutamate + 3-oxo-6-acetamidohexanoate." [EC:2.6.1.65, MetaCyc:2.6.1.65-RXN]
GO:0047319
name: aspartate-phenylpyruvate transaminase activity
def: "Catalysis of the reaction: keto-phenylpyruvate + L-aspartate = L-phenylalanine + oxaloacetate." [EC:2.6.1.70, RHEA:14100]
GO:0047320
name: D-4-hydroxyphenylglycine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + D-4-hydroxyphenylglycine = 4-hydroxyphenylglyoxylate + L-glutamate." [EC:2.6.1.72, RHEA:15592]
GO:0047321
name: diphosphate-protein phosphotransferase activity
def: "Catalysis of the reaction: microsomal-membrane protein + diphosphate = diphosphate + O-phospho-microsomal-membrane protein." [EC:2.7.99.1, MetaCyc:2.7.99.1-RXN]
GO:0047322
name: [hydroxymethylglutaryl-CoA reductase (NADPH)] kinase activity
def: "Catalysis of the reaction: [3-hydroxy-3-methylglutaryl-CoA reductase (NADPH)] + ATP = [3-hydroxy-3-methylglutaryl-CoA reductase (NADPH)] phosphate + ADP." [EC:2.7.11.31, MetaCyc:2.7.1.109-RXN]
GO:0047323
name: [3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring)] kinase activity
def: "Catalysis of the reaction: ATP + 3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring) = ADP + 3-methyl-2-oxobutanoate dehydrogenase (acetyl-transferring) phosphate." [EC:2.7.11.4]
GO:0047324
name: phosphoenolpyruvate-glycerone phosphotransferase activity
def: "Catalysis of the reaction: glycerone + phosphoenolpyruvate = glycerone phosphate + pyruvate." [EC:2.7.1.121, RHEA:18384]
GO:0047325
name: inositol tetrakisphosphate 1-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 3,4,5,6-tetrakisphosphate + ATP = 1D-myo-inositol 1,3,4,5,6-pentakisphosphate + ADP." [EC:2.7.1.134, MetaCyc:2.7.1.134-RXN]
GO:0047326
name: inositol tetrakisphosphate 5-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4,6-tetrakisphosphate + ATP = 1D-myo-inositol 1,3,4,5,6-pentakisphosphate + ADP." [EC:2.7.1.140, MetaCyc:2.7.1.140-RXN]
GO:0047327
name: glycerol-3-phosphate-glucose phosphotransferase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + D-glucose = D-glucose 6-phosphate + glycerol." [EC:2.7.1.142, RHEA:21291]
GO:0047328
name: acyl-phosphate-hexose phosphotransferase activity
def: "Catalysis of the reaction: D-hexose + acyl phosphate = D-hexose phosphate + an acid." [EC:2.7.1.61, MetaCyc:2.7.1.61-RXN]
GO:0047329
name: phosphoramidate-hexose phosphotransferase activity
def: "Catalysis of the reaction: hexose + phosphoramidate = hexose 1-phosphate + NH3." [EC:2.7.1.62, MetaCyc:2.7.1.62-RXN]
GO:0047330
name: polyphosphate-glucose phosphotransferase activity
def: "Catalysis of the reaction: beta-D-glucose + long chain polyphosphate = glucose-6-phosphate + long chain polyphosphate." [EC:2.7.1.63, MetaCyc:2.7.1.63-RXN]
GO:0047331
name: diphosphate-glycerol phosphotransferase activity
def: "Catalysis of the reaction: glycerol + diphosphate = glycerol 1-phosphate + H(+) + phosphate." [EC:2.7.1.79, RHEA:13692]
GO:0047332
name: diphosphate-serine phosphotransferase activity
def: "Catalysis of the reaction: L-serine + diphosphate = O-phospho-L-serine + H(+) + phosphate." [EC:2.7.1.80, RHEA:23767]
GO:0047333
name: dihydrostreptomycin-6-phosphate 3'-alpha-kinase activity
def: "Catalysis of the reaction: ATP + dihydrostreptomycin 6-phosphate = ADP + dihydrostreptomycin 3'alpha,6-bisphosphate + 2 H(+)." [EC:2.7.1.88, RHEA:16284]
GO:0047334
name: diphosphate-fructose-6-phosphate 1-phosphotransferase activity
def: "Catalysis of the reaction: fructose-6-phosphate + diphosphate = phosphate + fructose-1,6-bisphosphate." [EC:2.7.1.90, MetaCyc:2.7.1.90-RXN]
GO:0047335
name: 3-phosphoglyceroyl-phosphate-polyphosphate phosphotransferase activity
def: "Catalysis of the reaction: long-chain-polyphosphate + 3-phospho-D-glyceroyl-phosphate = long-chain-polyphosphate + 3-phosphoglycerate." [EC:2.7.4.17, MetaCyc:2.7.4.17-RXN]
GO:0047336
name: 5-methyldeoxycytidine-5'-phosphate kinase activity
def: "Catalysis of the reaction: 2'-deoxy-5-methyl-5'-cytidylate + ATP = 5-methyldeoxycytidine diphosphate + ADP + H(+)." [EC:2.7.4.19, RHEA:11399]
GO:0047337
name: dolichyl-diphosphate-polyphosphate phosphotransferase activity
def: "Catalysis of the reaction: dolichyl diphosphate + long-chain-polyphosphate = dolichol-phosphate + long-chain-polyphosphate." [EC:2.7.4.20, MetaCyc:2.7.4.20-RXN]
GO:0047338
name: UTP:xylose-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: alpha-D-xylose 1-phosphate + UTP = UDP-D-xylose + diphosphate." [EC:2.7.7.11, MetaCyc:2.7.7.11-RXN]
GO:0047339
name: nucleoside-triphosphate-hexose-1-phosphate nucleotidyltransferase activity
def: "Catalysis of the reaction: hexose 1-phosphate + nucleoside triphosphate = NDP-hexose + diphosphate." [EC:2.7.7.28, MetaCyc:2.7.7.28-RXN]
GO:0047341
name: fucose-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: beta-L-fucose 1-phosphate + GTP = diphosphate + GDP-L-fucose." [EC:2.7.7.30, RHEA:13552]
GO:0047342
name: galactose-1-phosphate thymidylyltransferase activity
def: "Catalysis of the reaction: alpha-D-galactose 1-phosphate + dTTP = diphosphate + dTDP-D-galactose." [EC:2.7.7.32, RHEA:17168]
GO:0047343
name: glucose-1-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + CTP = CDP-D-glucose + diphosphate." [EC:2.7.7.33, RHEA:18216]
GO:0047344
name: glucose-1-phosphate guanylyltransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + GTP = diphosphate + GDP-D-glucose." [EC:2.7.7.34, RHEA:10711]
GO:0047345
name: ribose-5-phosphate adenylyltransferase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + ADP + H(+) = ADP-ribose + phosphate." [EC:2.7.7.35, RHEA:14532]
GO:0047346
name: aldose-1-phosphate adenylyltransferase activity
def: "Catalysis of the reaction: aldose 1-phosphate + ADP = phosphate + ADP-aldose." [EC:2.7.7.36, MetaCyc:2.7.7.36-RXN]
GO:0047347
name: aldose-1-phosphate nucleotidyltransferase activity
def: "Catalysis of the reaction: aldose 1-phosphate + NDP = phosphate + NDP-aldose." [EC:2.7.7.37, MetaCyc:2.7.7.37-RXN]
GO:0047348
name: glycerol-3-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + CTP = CDP-glycerol + diphosphate." [EC:2.7.7.39, RHEA:13364]
GO:0047349
name: D-ribitol-5-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: D-ribitol 5-phosphate + CTP = CDP-ribitol + diphosphate." [EC:2.7.7.40, RHEA:12459]
GO:0047350
name: glucuronate-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: 1-phospho-alpha-D-glucuronate + UTP = diphosphate + UDP-alpha-D-glucuronate." [EC:2.7.7.44, RHEA:16328]
GO:0047351
name: guanosine-triphosphate guanylyltransferase activity
def: "Catalysis of the reaction: 2 GTP = P(1),P(4)-bis(5'-guanosyl) tetraphosphate + diphosphate + H(+)." [EC:2.7.7.45, RHEA:18156]
GO:0047352
name: adenylylsulfate-ammonia adenylyltransferase activity
def: "Catalysis of the reaction: 5'-adenylyl sulfate + NH(4)(+) = adenosine 5'-phosphoramidate + 2 H(+) + sulfate." [EC:2.7.7.51, RHEA:19200]
GO:0047353
name: N-methylphosphoethanolamine cytidylyltransferase activity
def: "Catalysis of the reaction: N-methylethanolamine phosphate + CTP = CDP-N-methylethanolamine + diphosphate." [EC:2.7.7.57, RHEA:10579]
GO:0047354
name: sphingosine cholinephosphotransferase activity
def: "Catalysis of the reaction: CDP-choline + sphingosine = CMP + H(+) + sphingosyl-phosphocholine." [EC:2.7.8.10, RHEA:21227]
GO:0047355
name: CDP-glycerol glycerophosphotransferase activity
def: "Catalysis of the reaction: glycerophosphate(n) + CDP-glycerol = glycerophosphate(n+1) + CMP." [EC:2.7.8.12, MetaCyc:2.7.8.12-RXN]
GO:0047356
name: CDP-ribitol ribitolphosphotransferase activity
def: "Catalysis of the reaction: ribitol phosphate(n) + CDP-ribitol = ribitol phosphate(n+1) + CMP." [EC:2.7.8.14, MetaCyc:2.7.8.14-RXN]
GO:0047357
name: UDP-galactose-UDP-N-acetylglucosamine galactose phosphotransferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-alpha-D-glucosamine + UDP-D-galactose = H(+) + UDP-N-acetyl-6-(D-galactose-1-phospho)-D-glucosamine + UMP." [EC:2.7.8.18, RHEA:22443]
GO:0047358
name: UDP-glucose-glycoprotein glucose phosphotransferase activity
def: "Catalysis of the reaction: glycoprotein D-mannose + UDP-D-glucose = glycoprotein 6-(D-glucose-1-phospho)-D-mannose + UMP." [EC:2.7.8.19, MetaCyc:2.7.8.19-RXN]
GO:0047359
name: 1-alkenyl-2-acylglycerol choline phosphotransferase activity
def: "Catalysis of the reaction: 1-alkenyl-2-acylglycerol + CDP-choline = plasmenylcholine + CMP." [EC:2.7.8.22, MetaCyc:2.7.8.22-RXN]
GO:0047360
name: undecaprenyl-phosphate galactose phosphotransferase activity
def: "Catalysis of the reaction: all-trans-undecaprenyl phosphate + UDP-D-galactose = alpha-D-galactosyl-diphosphoundecaprenol + UMP." [EC:2.7.8.6, RHEA:11655]
GO:0047361
name: phosphomannan mannosephosphotransferase activity
def: "Catalysis of the reaction: phosphomannan(n) + GDP-mannose = phosphomannan(n+1) + GMP." [EC:2.7.8.9, MetaCyc:2.7.8.9-RXN]
GO:0047362
name: thiosulfate-dithiol sulfurtransferase activity
def: "Catalysis of the reaction: dithioerythritol + thiosulfate = hydrogen sulfide + dithioerythritol disulfide + sulfite." [EC:2.8.1.5, MetaCyc:2.8.1.5-RXN]
GO:0047363
name: triglucosylalkylacylglycerol sulfotransferase activity
def: "Catalysis of the reaction: alpha-D-glucosyl-1,6-alpha-D-glucosyl-1,6-alpha-D-glucosyl-1,3-1-O-alkyl-2-O-acylglycerol + 3'-phosphoadenosine 5'-phosphosulfate = 6-sulfo-alpha-D-glucosyl-1,6-alpha-D-glucosyl-1,6-alpha-D-glucosyl-1,3-1-O-alkyl-2-O-acylglycerol + adenosine 3',5'-bisphosphate." [EC:2.8.2.19, MetaCyc:2.8.2.19-RXN]
GO:0047364
name: desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + desulfoglucotropeolin = adenosine 3',5'-diphosphate + glucotropeolin + H(+)." [EC:2.8.2.24, RHEA:20284]
GO:0047365
name: quercetin-3-sulfate 3'-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + quercetin 3-sulfate = adenosine 3',5'-diphosphate + H(+) + quercetin 3,3'-disulfate." [EC:2.8.2.26, RHEA:22507]
GO:0047366
name: quercetin-3-sulfate 4'-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + quercetin 3-sulfate = adenosine 3',5'-diphosphate + H(+) + quercetin 3,4'-disulfate." [EC:2.8.2.27, RHEA:17208]
GO:0047367
name: quercetin-3,3'-bissulfate 7-sulfotransferase activity
def: "Catalysis of the reaction: quercetin 3,3'-bissulfate + 3'-phosphoadenosine 5'-phosphosulfate = quercetin 3,3',7-trissulfate + adenosine 3',5'-bisphosphate." [EC:2.8.2.28, MetaCyc:2.8.2.28-RXN]
GO:0047368
name: UDP-N-acetylgalactosamine-4-sulfate sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + UDP-N-acetyl-D-galactosamine 4-sulfate = adenosine 3',5'-diphosphate + H(+) + UDP-N-acetyl-D-galactosamine 4,6-disulfate." [EC:2.8.2.7, RHEA:14340]
GO:0047369
name: succinate-hydroxymethylglutarate CoA-transferase activity
def: "Catalysis of the reaction: (S)-3-hydroxy-3-methylglutarate + succinyl-CoA = 3-hydroxy-3-methyl-glutaryl-CoA + succinate." [EC:2.8.3.13, MetaCyc:2.8.3.13-RXN]
GO:0047370
name: succinate-citramalate CoA-transferase activity
def: "Catalysis of the reaction: S-citramalate + succinyl-CoA = citramalyl-CoA + succinate." [EC:2.8.3.7, MetaCyc:2.8.3.7-RXN]
GO:0047371
name: butyrate-acetoacetate CoA-transferase activity
def: "Catalysis of the reaction: acetoacetate + butanoyl-CoA = acetoacetyl-CoA + butanoate." [EC:2.8.3.9, RHEA:12964]
GO:0047372
name: acylglycerol lipase activity
def: "Catalysis of the reaction: H2O + acylglycerol = a fatty acid + glycerol." [EC:3.1.1.23, MetaCyc:3.1.1.23-RXN]
GO:0047373
name: acetoxybutynylbithiophene deacetylase activity
def: "Catalysis of the reaction: 5-(4-acetoxybut-1-ynyl)-2,2'-bithiophene + H(2)O = 5-(4-hydroxy-but-1-ynyl)-2,2'-bithiophene + acetate + H(+)." [EC:3.1.1.54, RHEA:11551]
GO:0047374
name: methylumbelliferyl-acetate deacetylase activity
def: "Catalysis of the reaction: 4-methylumbelliferyl acetate + H(2)O = 4-methylumbelliferone + acetate + H(+)." [EC:3.1.1.56, RHEA:12211]
GO:0047375
name: N-acetylgalactosaminoglycan deacetylase activity
def: "Catalysis of the reaction: H2O + N-acetyl-D-galactosaminoglycan = acetate + D-galactosaminoglycan." [EC:3.1.1.58, MetaCyc:3.1.1.58-RXN]
GO:0047376
name: all-trans-retinyl-palmitate hydrolase, all-trans-retinol forming activity
def: "Catalysis of the reaction: all-trans-retinyl palmitate + H2O = all-trans-retinol + H+ + palmitate." [MetaCyc:3.1.1.64-RXN, RHEA:13936]
GO:0047377
name: 5-(3,4-diacetoxybut-1-ynyl)-2,2'-bithiophene deacetylase activity
def: "Catalysis of the reaction: 5-(3,4-diacetoxybut-1-ynyl)-2,2'-bithiophene + H(2)O = 5-(3-hydroxy-4-acetoxybut-1-ynyl)-2,2'-bithiophene + acetate + H(+)." [EC:3.1.1.66, RHEA:16316]
GO:0047378
name: acetylalkylglycerol acetylhydrolase activity
def: "Catalysis of the reaction: 2-acetyl-1-alkyl-sn-glycerol + H(2)O = 1-alkyl-sn-glycerol + acetate + H(+)." [EC:3.1.1.71, RHEA:11555]
GO:0047379
name: ADP-dependent short-chain-acyl-CoA hydrolase activity
def: "Catalysis of the reaction: H2O + a short-chain acyl-CoA = a short-chain carboxylate + CoA." [EC:3.1.2.18, MetaCyc:3.1.2.18-RXN]
GO:0047380
name: ADP-dependent medium-chain-acyl-CoA hydrolase activity
def: "Catalysis of the reaction: H2O + a medium-chain acyl-CoA = a medium-chain carboxylate + CoA. Requires ADP." [EC:3.1.2.19, MetaCyc:3.1.2.19-RXN]
GO:0047381
name: dodecanoyl-[acyl-carrier-protein] hydrolase activity
def: "Catalysis of the reaction: H2O + dodecanoyl-[acyl-carrier protein] = dodecanoate + [acyl-carrier protein]." [EC:3.1.2.21, MetaCyc:3.1.2.21-RXN]
GO:0047382
name: methylphosphothioglycerate phosphatase activity
def: "Catalysis of the reaction: S-methyl-3-phospho-1-thio-D-glycerate + H(2)O = S-methyl-1-thio-D-glycerate + phosphate." [EC:3.1.3.14, RHEA:16084]
GO:0047383
name: guanidinodeoxy-scyllo-inositol-4-phosphatase activity
def: "Catalysis of the reaction: 1-guanidino-1-deoxy-scyllo-inositol 4-phosphate + H(2)O = 1-guanidino-1-deoxy-scyllo-inositol + phosphate." [EC:3.1.3.40, RHEA:15780]
GO:0047384
name: [hydroxymethylglutaryl-CoA reductase (NADPH)]-phosphatase activity
def: "Catalysis of the reaction: H2O + [hydroxymethylglutaryl-CoA reductase (NADPH)] phosphate = phosphate + [hydroxymethylglutaryl-CoA reductase (NADPH)]." [EC:3.1.3.47, MetaCyc:3.1.3.47-RXN]
GO:0047385
name: [3-methyl-2-oxobutanoate dehydrogenase (lipoamide)]-phosphatase activity
def: "Catalysis of the reaction: H2O + [3-methyl-2-oxobutanoate dehydrogenase (lipoamide)] phosphate = phosphate + [3-methyl-2-oxobutanoate dehydrogenase (lipoamide)]." [EC:3.1.3.52, MetaCyc:3.1.3.52-RXN]
GO:0047386
name: fructose-2,6-bisphosphate 6-phosphatase activity
def: "Catalysis of the reaction: beta-D-fructose 2,6-bisphosphate + H(2)O = beta-D-fructofuranose 2-phosphate + phosphate." [EC:3.1.3.54, RHEA:13336]
GO:0047387
name: serine-ethanolaminephosphate phosphodiesterase activity
def: "Catalysis of the reaction: H(2)O + serine phosphoethanolamine = H(+) + phosphoethanolamine + serine." [EC:3.1.4.13, RHEA:17116]
GO:0047388
name: adenylyl-[glutamate-ammonia ligase] hydrolase activity
def: "Catalysis of the reaction: adenylyl-[L-glutamate:ammonia ligase (ADP-forming)] + H2O = AMP + [L-glutamate:ammonia ligase (ADP-forming)]." [EC:3.1.4.15, MetaCyc:3.1.4.15-RXN]
GO:0047389
name: glycerophosphocholine phosphodiesterase activity
def: "Catalysis of the reaction: H2O + L-1-glycero-3-phosphocholine = glycerol-3-phosphate + choline." [EC:3.1.4.2, MetaCyc:3.1.4.2-RXN]
GO:0047390
name: glycerophosphocholine cholinephosphodiesterase activity
def: "Catalysis of the reaction: sn-glycero-3-phosphocholine + H(2)O = choline phosphate + glycerol + H(+)." [EC:3.1.4.38, RHEA:19548]
GO:0047391
name: alkylglycerophosphoethanolamine phosphodiesterase activity
def: "Catalysis of the reaction: H2O + 1-alkyl-sn-glycero-3-phosphoethanolamine = ethanolamine + 1-alkyl-sn-glycerol 3-phosphate." [EC:3.1.4.39, MetaCyc:3.1.4.39-RXN]
GO:0047392
name: CMP-N-acylneuraminate phosphodiesterase activity
def: "Catalysis of the reaction: H2O + CMP-N-acylneuraminate = N-acylneuraminate + CMP." [EC:3.1.4.40, MetaCyc:3.1.4.40-RXN]
GO:0047393
name: glycerol-1,2-cyclic-phosphate 2-phosphodiesterase activity
def: "Catalysis of the reaction: glycerol 1,2-cyclic phosphate + H(2)O = glycerol 1-phosphate + H(+)." [EC:3.1.4.42, RHEA:16496]
GO:0047394
name: glycerophosphoinositol inositolphosphodiesterase activity
def: "Catalysis of the reaction: H2O + 1-(sn-glycero-3-phospho)-1D-myoinositol = 1D-myo-inositol 1-phosphate + glycerol." [EC:3.1.4.43, MetaCyc:3.1.4.43-RXN]
GO:0047395
name: glycerophosphoinositol glycerophosphodiesterase activity
def: "Catalysis of the reaction: 1-(sn-glycero-3-phospho)-1D-myo-inositol + H(2)O = sn-glycerol 3-phosphate + myo-inositol + H(+)." [EC:3.1.4.44, RHEA:16504]
GO:0047396
name: glycosylphosphatidylinositol diacylglycerol-lyase activity
def: "Catalysis of the reaction: 6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol = 1,2-diacyl-sn-glycerol + 6-(alpha-D-glucosaminyl)-1D-myo-inositol 1,2-cyclic phosphate." [EC:4.6.1.14, MetaCyc:3.1.4.47-RXN]
GO:0047397
name: dolichylphosphate-glucose phosphodiesterase activity
def: "Catalysis of the reaction: dolichyl beta-D-glucosyl phosphate + H2O = dolichol-phosphate + beta-D-glucose." [EC:3.1.4.48, MetaCyc:3.1.4.48-RXN]
GO:0047398
name: dolichylphosphate-mannose phosphodiesterase activity
def: "Catalysis of the reaction: dolichyl beta-D-mannosyl phosphate + H2O = dolichol-phosphate + mannose." [EC:3.1.4.49, MetaCyc:3.1.4.49-RXN]
GO:0047399
name: glucose-1-phospho-D-mannosylglycoprotein phosphodiesterase activity
def: "Catalysis of the reaction: H2O + 6-(D-glucose-1-phospho)-D-mannosylglycoprotein = D-mannosylglycoprotein + D-glucose-alpha-1-phosphate." [EC:3.1.4.51, MetaCyc:3.1.4.51-RXN]
GO:0047400
name: phosphonoacetate hydrolase activity
def: "Catalysis of the reaction: H(2)O + phosphonoacetate = acetate + H(+) + phosphate." [EC:3.11.1.2, RHEA:16752]
GO:0047401
name: trithionate hydrolase activity
def: "Catalysis of the reaction: H(2)O + trithionate = H(+) + sulfate + thiosulfate." [EC:3.12.1.1, RHEA:21887]
GO:0047402
name: protein-glucosylgalactosylhydroxylysine glucosidase activity
def: "Catalysis of the reaction: H2O + protein alpha-D-glucosyl-1,2-beta-D-galactosyl-L-hydroxylysine = protein beta-D-galactosyl-L-hydroxylysine + beta-D-glucose." [EC:3.2.1.107, MetaCyc:3.2.1.107-RXN]
GO:0047403
name: lacto-N-biosidase activity
def: "Catalysis of the reaction: H2O + beta-D-Gal-(1,3)-beta-D-GlcNAc-(1,3)-beta-D-Gal-(1,4)-D-Glc = beta-D-Gal-(1,4)-D-Glc + beta-D-Gal-(1,3)-D-GlcNAc." [EC:3.2.1.140, MetaCyc:3.2.1.140-RXN]
GO:0047404
name: glucuronosyl-disulfoglucosamine glucuronidase activity
def: "Catalysis of the reaction: H2O + 3-D-glucuronosyl-N2-,6-disulfo-beta-D-glucosamine = glucuronate + N2,6-disulfo-D-glucosamine." [EC:3.2.1.56, MetaCyc:3.2.1.56-RXN]
GO:0047405
name: pyrimidine-5'-nucleotide nucleosidase activity
def: "Catalysis of the reaction: H2O + a pyrimidine 5'-nucleotide = ribose-5-phosphate + a pyrimidine." [EC:3.2.2.10, MetaCyc:3.2.2.10-RXN]
GO:0047406
name: beta-aspartyl-N-acetylglucosaminidase activity
def: "Catalysis of the reaction: N(4)-(beta-N-acetyl-D-glucosaminyl)-L-asparagine + H(2)O = N-acetyl-D-glucosamine + L-asparagine." [EC:3.2.2.11, RHEA:12327]
GO:0047407
name: ADP-ribosyl-[dinitrogen reductase] hydrolase activity
def: "Catalysis of the reaction: ADP-ribosyl-[dinitrogen reductase] = adenosine diphosphate ribose + [dinitrogen reductase]." [EC:3.2.2.24, MetaCyc:3.2.2.24-RXN]
GO:0047408
name: alkenylglycerophosphocholine hydrolase activity
def: "Catalysis of the reaction: H2O + 1-(1-alkenyl)-sn-glycero-3-phosphocholine = L-1-glycero-3-phosphocholine + an aldehyde." [EC:3.3.2.2, MetaCyc:3.3.2.2-RXN]
GO:0047409
name: alkenylglycerophosphoethanolamine hydrolase activity
def: "Catalysis of the reaction: H2O + 1-(1-alkenyl)-sn-glycero-3-phosphoethanolamine = sn-glycero-3-phosphoethanolamine + an aldehyde." [EC:3.3.2.5, MetaCyc:3.3.2.5-RXN]
GO:0047410
name: N-formylmethionylaminoacyl-tRNA deformylase activity
def: "Catalysis of the reaction: H2O + charged-fMet-tRNAs = L-methionylaminoacyl-tRNA + formate." [EC:3.5.1.27, MetaCyc:3.5.1.27-RXN]
GO:0047411
name: 2-(acetamidomethylene)succinate hydrolase activity
def: "Catalysis of the reaction: 2 H2O + 2-(acetamidomethylene)succinate = CO2 + NH3 + succinate semialdehyde + acetate." [EC:3.5.1.29, MetaCyc:3.5.1.29-RXN]
GO:0047412
name: N-(long-chain-acyl)ethanolamine deacylase activity
def: "Catalysis of the reaction: H2O + N-(long-chain-acyl)ethanolamine = ethanolamine + a fatty acid." [EC:3.5.1.60, MetaCyc:3.5.1.60-RXN]
GO:0047413
name: N(alpha)-benzyloxycarbonylleucine hydrolase activity
def: "Catalysis of the reaction: N-benzyloxycarbonyl-L-leucine + H(2)O + H(+) = L-leucine + benzyl alcohol + CO(2)." [EC:3.5.1.64, RHEA:18904]
GO:0047414
name: 2-(hydroxymethyl)-3-(acetamidomethylene)succinate hydrolase activity
def: "Catalysis of the reaction: (2Z)-2-(acetamidomethylene)-3-(hydroxymethyl)succinate + 2 H(2)O + H(+) = 2-(hydroxymethyl)-4-oxobutanoate + acetate + CO(2) + NH(4)(+)." [EC:3.5.1.66, RHEA:17680]
GO:0047415
name: D-benzoylarginine-4-nitroanilide amidase activity
def: "Catalysis of the reaction: N(2)-benzoyl-D-arginine-4-nitroanilide + H(2)O = 4-nitroaniline + N(2)-benzoyl-D-arginine + H(+)." [EC:3.5.1.72, RHEA:14424]
GO:0047416
name: arylalkyl acylamidase activity
def: "Catalysis of the reaction: H2O + N-acetylarylalkylamine = acetate + arylalkylamine." [EC:3.5.1.76, MetaCyc:3.5.1.76-RXN]
GO:0047417
name: N-carbamoyl-D-amino acid hydrolase activity
def: "Catalysis of the reaction: H2O + N-carbamoyl-D-amino acid = CO2 + NH3 + D-amino acid." [EC:3.5.1.77, MetaCyc:3.5.1.77-RXN]
GO:0047418
name: phthalyl amidase activity
def: "Catalysis of the reaction: H2O + a phthalylamide = phthalate + substituted amine." [EC:3.5.1.79, MetaCyc:3.5.1.79-RXN]
GO:0047419
name: N-acetylgalactosamine-6-phosphate deacetylase activity
def: "Catalysis of the reaction: H2O + N-acetyl-D-galactosamine 6-phosphate = acetate + D-galactosamine 6-phosphate." [EC:3.5.1.-, MetaCyc:3.5.1.80-RXN]
GO:0047420
name: N-acyl-D-amino-acid deacylase activity
def: "Catalysis of the reaction: H2O + N-acyl-D-amino acid = D-amino acid + an acid." [EC:3.5.1.81, MetaCyc:3.5.1.81-RXN]
GO:0047421
name: N-acyl-D-glutamate deacylase activity
def: "Catalysis of the reaction: N-acyl-D-glutamate + H(2)O = D-glutamate + a carboxylate." [EC:3.5.1.82, RHEA:12836]
GO:0047422
name: N-acyl-D-aspartate deacylase activity
def: "Catalysis of the reaction: N-acyl-D-aspartate + H(2)O = D-aspartate + a carboxylate." [EC:3.5.1.83, RHEA:18288]
GO:0047423
name: N-methylhydantoinase (ATP-hydrolyzing) activity
def: "Catalysis of the reaction: N-methylhydantoin + ATP + 2 H(2)O = N-carbamoylsarcosine + ADP + 3 H(+) + phosphate." [EC:3.5.2.14, RHEA:11723]
GO:0047424
name: methylenediurea deaminase activity
def: "Catalysis of the reaction: 2 H2O + methylenediurea = CO2 + 2 NH3 + N-hydroxymethylurea." [EC:3.5.3.21, MetaCyc:3.5.3.21-RXN]
GO:0047425
name: 1-pyrroline-4-hydroxy-2-carboxylate deaminase activity
def: "Catalysis of the reaction: 4-hydroxy-1-pyrroline-2-carboxylate + H(2)O + H(+) = 2,5-dioxopentanoate + NH(4)(+)." [EC:3.5.4.22, RHEA:10563]
GO:0047426
name: ricinine nitrilase activity
def: "Catalysis of the reaction: H2O + ricinine = NH3 + 3-carboxy-4-methoxy-N-methyl-2-pyridone." [EC:3.5.5.2, MetaCyc:3.5.5.2-RXN]
GO:0047427
name: cyanoalanine nitrilase activity
def: "Catalysis of the reaction: 3-cyano-L-alanine + 2 H(2)O + H(+) = L-aspartate + NH(4)(+)." [EC:3.5.5.4, RHEA:25310]
GO:0047428
name: arylacetonitrilase activity
def: "Catalysis of the reaction: 2 H2O + 4-chlorophenylacetonitrile = 4-chlorophenylacetate + NH3." [EC:3.5.5.5, MetaCyc:3.5.5.5-RXN]
GO:0047429
name: nucleoside-triphosphate diphosphatase activity
def: "Catalysis of the reaction: H2O + a nucleoside triphosphate = diphosphate + a nucleotide." [EC:3.6.1.19, MetaCyc:3.6.1.19-RXN]
GO:0047430
name: oligosaccharide-diphosphodolichol diphosphatase activity
def: "Catalysis of the reaction: H2O + oligosaccharide-diphosphodolichol = dolichol-phosphate + oligosaccharide phosphate." [EC:3.6.1.44, MetaCyc:3.6.1.44-RXN]
GO:0047431
name: 3-hydroxy-2-methylpyridine-4,5-dicarboxylate 4-decarboxylase activity
def: "Catalysis of the reaction: 5-hydroxy-6-methylpyridine-3,4-dicarboxylate + H(+) = 5-hydroxy-6-methylpyridine-3-carboxylate + CO(2)." [EC:4.1.1.51, RHEA:13672]
GO:0047432
name: 2,2-dialkylglycine decarboxylase (pyruvate) activity
def: "Catalysis of the reaction: 2,2-dialkylglycine + H(+) + pyruvate = L-alanine + CO(2) + dialkyl ketone." [EC:4.1.1.64, RHEA:16076]
GO:0047433
name: branched-chain-2-oxoacid decarboxylase activity
def: "Catalysis of the reaction: (S)-3-methyl-2-oxopentanoate + H(+) = 2-methylbutanal + CO(2)." [EC:4.1.1.72, RHEA:21111]
GO:0047434
name: indolepyruvate decarboxylase activity
def: "Catalysis of the reaction: indolepyruvate = CO2 + indole acetaldehyde." [EC:4.1.1.74, MetaCyc:4.1.1.74-RXN]
GO:0047435
name: 5-guanidino-2-oxopentanoate decarboxylase activity
def: "Catalysis of the reaction: 5-guanidino-2-oxopentanoate + H(+) = 4-guanidinobutanal + CO(2)." [EC:4.1.1.75, RHEA:11343]
GO:0047436
name: arylmalonate decarboxylase activity
def: "Catalysis of the reaction: 2-aryl-2-methylmalonate + H(+) = 2-arylpropionate + CO(2)." [EC:4.1.1.76, RHEA:20516]
GO:0047437
name: 4-oxalocrotonate decarboxylase activity
def: "Catalysis of the reaction: 4-oxalocrotonate = CO2 + 2-oxopent-4-enoate." [EC:4.1.1.77, MetaCyc:4.1.1.77-RXN]
GO:0047438
name: 2-dehydro-3-deoxy-L-pentonate aldolase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-L-pentonate = glycolaldehyde + pyruvate." [EC:4.1.2.18, MetaCyc:4.1.2.18-RXN]
GO:0047439
name: 3-deoxy-D-manno-octulosonate aldolase activity
def: "Catalysis of the reaction: 3-deoxy-D-manno-octulosonate = D-arabinose + pyruvate." [EC:4.1.2.23, RHEA:23343]
GO:0047440
name: 2-dehydro-3-deoxy-D-pentonate aldolase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-arabinonate = glycolaldehyde + pyruvate." [EC:4.1.2.28, RHEA:20612]
GO:0047441
name: 5-dehydro-2-deoxyphosphogluconate aldolase activity
def: "Catalysis of the reaction: 6-phospho-5-dehydro-2-deoxy-D-gluconate = 3-oxopropanoate + glycerone phosphate." [EC:4.1.2.29, RHEA:13180]
GO:0047442
name: 17-alpha-hydroxyprogesterone aldolase activity
def: "Catalysis of the reaction: 17-alpha-hydroxyprogesterone = acetaldehyde + 4-androstene-3,17-dione." [EC:4.1.2.30, MetaCyc:4.1.2.30-RXN]
GO:0047443
name: 4-hydroxy-4-methyl-2-oxoglutarate aldolase activity
def: "Catalysis of the reaction: 4-hydroxy-4-methyl-2-oxoglutarate = 2 pyruvate." [EC:4.1.3.17, MetaCyc:4.1.3.17-RXN]
GO:0047444
name: N-acylneuraminate-9-phosphate synthase activity
def: "Catalysis of the reaction: H2O + phosphoenolpyruvate + N-acyl-D-mannosamine 6-phosphate = phosphate + N-acylneuraminate 9-phosphate." [EC:2.5.1.57, MetaCyc:4.1.3.20-RXN]
GO:0047445
name: 3-hydroxy-3-isohexenylglutaryl-CoA lyase activity
def: "Catalysis of the reaction: 3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA + 4 H(+) = 7-methyl-3-oxooct-6-enoyl-CoA + acetate." [EC:4.1.3.26, RHEA:23087]
GO:0047446
name: (1-hydroxycyclohexan-1-yl)acetyl-CoA lyase activity
def: "Catalysis of the reaction: (1-hydroxycyclohexan-1-yl)acetyl-CoA = acetyl-CoA + cyclohexanone." [EC:4.1.3.35, RHEA:23871]
GO:0047447
name: erythro-3-hydroxyaspartate ammonia-lyase activity
def: "Catalysis of the reaction: erythro-3-hydroxy-L-aspartate = NH3 + oxaloacetic acid." [EC:4.3.1.20, MetaCyc:4.3.1.20-RXN]
GO:0047448
name: 5-dehydro-4-deoxyglucarate dehydratase activity
def: "Catalysis of the reaction: 5-dehydro-4-deoxy-D-glucarate + H(+) = 2,5-dioxopentanoate + CO(2) + H(2)O." [EC:4.2.1.41, RHEA:24611]
GO:0047449
name: 2-dehydro-3-deoxy-L-arabinonate dehydratase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-L-arabinonate = 2,5-dioxopentanoate + H(2)O." [EC:4.2.1.43, RHEA:17204]
GO:0047450
name: crotonoyl-[acyl-carrier-protein] hydratase activity
def: "Catalysis of the reaction: (3R)-3-hydroxybutanoyl-[acyl-carrier protein] = H2O + but-2-enoyl-[acyl-carrier protein]." [EC:4.2.1.58, MetaCyc:4.2.1.58-RXN]
GO:0047451
name: 3-hydroxyoctanoyl-[acyl-carrier-protein] dehydratase activity
def: "Catalysis of the reaction: (3R)-3-hydroxyoctanoyl-[acyl-carrier protein] = H2O + 2-octenoyl-[acyl-carrier protein]." [EC:4.2.1.59, MetaCyc:4.2.1.59-RXN]
GO:0047452
name: protoaphin-aglucone dehydratase (cyclizing) activity
def: "Catalysis of the reaction: protoaphin aglucone = H(2)O + xanthoaphin." [EC:4.2.1.73, RHEA:23879]
GO:0047453
name: ATP-dependent NAD(P)H-hydrate dehydratase activity
def: "Catalysis of the reaction: (6S)-6beta-hydroxy-1,4,5,6-tetrahydronicotinamide adenine dinucleotide + ATP = ADP + 3 H(+) + NADH + phosphate." [EC:4.2.1.93, RHEA:19020]
GO:0047454
name: phaseollidin hydratase activity
def: "Catalysis of the reaction: phaseollidin hydrate = H(2)O + phaseollidin." [EC:4.2.1.97, RHEA:19772]
GO:0047455
name: 16-alpha-hydroxyprogesterone dehydratase activity
def: "Catalysis of the reaction: 16-alpha-hydroxyprogesterone = H2O + 16-dehydroprogesterone." [EC:4.2.1.98, MetaCyc:4.2.1.98-RXN]
GO:0047456
name: 2-methylisocitrate dehydratase activity
def: "Catalysis of the reaction: (2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = cis-2-methylaconitate + H(2)O." [EC:4.2.1.99, RHEA:17944]
GO:0047457
name: exo-(1,4)-alpha-D-glucan lyase activity
def: "Catalysis of the reaction: linear alpha-D-glucan = 1,5-anhydro-D-fructose + beta-D-glucose." [EC:4.2.2.13, MetaCyc:4.2.2.13-RXN]
GO:0047458
name: beta-pyrazolylalanine synthase activity
def: "Catalysis of the reaction: O-acetyl-L-serine + pyrazole = 3-(pyrazol-1-yl)-L-alanine + acetate + H(+)." [EC:2.5.1.51, RHEA:13120]
GO:0047459
name: 3-aminobutyryl-CoA ammonia-lyase activity
def: "Catalysis of the reaction: (S)-3-aminobutanoyl-CoA = crotonoyl-CoA + NH(4)(+)." [EC:4.3.1.14, RHEA:10059]
GO:0047460
name: L-2-amino-4-chloropent-4-enoate dehydrochlorinase activity
def: "Catalysis of the reaction: L-2-amino-4-chloropent-4-enoate + H(2)O = 2-oxopent-4-enoate + chloride + H(+) + NH(4)(+)." [EC:4.5.1.4, RHEA:11623]
GO:0047461
name: (+)-delta-cadinene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = diphosphate + (+)-delta-cadinene." [EC:4.2.3.13, MetaCyc:4.6.1.11-RXN]
GO:0047462
name: phenylalanine racemase (ATP-hydrolyzing) activity
def: "Catalysis of the reaction: L-phenylalanine + ATP + H(2)O = D-phenylalanine + AMP + diphosphate + 2 H(+)." [EC:5.1.1.11, RHEA:20204]
GO:0047463
name: 2-aminohexano-6-lactam racemase activity
def: "Catalysis of the reaction: L-2-aminohexano-6-lactam = D-2-aminohexano-6-lactam." [EC:5.1.1.15, RHEA:14816]
GO:0047464
name: heparosan-N-sulfate-glucuronate 5-epimerase activity
def: "Catalysis of the reaction: heparosan-N-sulfate D-glucuronate = heparosan-N-sulfate L-iduronate." [EC:5.1.3.17, MetaCyc:5.1.3.17-RXN]
GO:0047465
name: N-acylglucosamine-6-phosphate 2-epimerase activity
def: "Catalysis of the reaction: N-acyl-D-glucosamine 6-phosphate = N-acyl-D-mannosamine 6-phosphate." [EC:5.1.3.9, MetaCyc:5.1.3.9-RXN]
GO:0047466
name: 2-chloro-4-carboxymethylenebut-2-en-1,4-olide isomerase activity
def: "Catalysis of the reaction: cis-2-chloro-4-carboxymethylenebut-2-en-1,4-olide = trans-2-chloro-4-carboxymethylenebut-2-en-1,4-olide." [EC:5.2.1.10, RHEA:10927]
GO:0047467
name: 4-hydroxyphenylacetaldehyde-oxime isomerase activity
def: "Catalysis of the reaction: (E)-4-hydroxyphenylacetaldehyde oxime = (Z)-4-hydroxyphenylacetaldehyde oxime." [EC:5.2.1.11, MetaCyc:5.2.1.11-RXN]
GO:0047468
name: phosphoglucomutase (glucose-cofactor) activity
def: "Catalysis of the reaction: glucose-1-phosphate = glucose-6-phosphate; using D-glucose as a cofactor." [EC:5.4.2.5, MetaCyc:5.4.2.5-RXN]
GO:0047469
name: 4-carboxymethyl-4-methylbutenolide mutase activity
def: "Catalysis of the reaction: 4-carboxymethyl-4-methylbut-2-en-1,4-olide = 4-carboxymethyl-3-methylbut-2-en-1,4-olide." [EC:5.4.99.14, RHEA:19240]
GO:0047470
name: (1,4)-alpha-D-glucan 1-alpha-D-glucosylmutase activity
def: "Catalysis of the reaction: 4-[(1->4)-alpha-D-glucosyl](n-1)-D-glucose = 1-alpha-D-[(1->4)-alpha-D-glucosyl](n-1)-alpha-D-glucopyranoside." [EC:5.4.99.15, MetaCyc:5.4.99.15-RXN]
GO:0047471
name: maltose alpha-D-glucosyltransferase activity
def: "Catalysis of the reaction: maltose = trehalose." [EC:5.4.99.16, MetaCyc:5.4.99.16-RXN]
GO:0047472
name: 3-carboxy-cis,cis-muconate cycloisomerase activity
def: "Catalysis of the reaction: 2-(carboxymethyl)-5-oxo-2,5-dihydro-2-furoate = 3-carboxy-cis,cis-muconate + H(+)." [EC:5.5.1.2, RHEA:23659]
GO:0047473
name: D-alanine-poly(phosphoribitol) ligase activity
def: "Catalysis of the reaction: poly(ribitol phosphate) + D-alanine + ATP = O-D-alanyl-poly(ribitol phosphate) + diphosphate + AMP." [EC:6.1.1.13, MetaCyc:6.1.1.13-RXN]
GO:0047474
name: long-chain fatty acid luciferin component ligase activity
def: "Catalysis of the reaction: protein + an acid + ATP = an acyl-protein thiolester + diphosphate + AMP. A long-chain fatty acid is a fatty acid with a chain length between C13 and C22." [CHEBI:15904, EC:6.2.1.19, MetaCyc:6.2.1.19-RXN]
GO:0047475
name: phenylacetate-CoA ligase activity
def: "Catalysis of the reaction: ATP + CoA + phenylacetate = AMP + diphosphate + H(+) + phenylacetyl-CoA." [EC:6.2.1.30, RHEA:20959]
GO:0047476
name: 3-alpha,7-alpha-dihydroxy-5-beta-cholestanate-CoA ligase activity
def: "Catalysis of the reaction: CoA + 3-alpha,7-alpha-dihydroxy-5-beta-cholestanate + ATP = 3-alpha,7-alpha-dihydroxy-5-beta-cholestanoyl-CoA + diphosphate + AMP." [EC:6.2.1.28, MetaCyc:6.2.1.28-RXN]
GO:0047478
name: aspartate-ammonia ligase (ADP-forming) activity
def: "Catalysis of the reaction: L-aspartate + ATP + NH(4)(+) = L-asparagine + ADP + 2 H(+) + phosphate." [EC:6.3.1.4, RHEA:14200]
GO:0047479
name: trypanothione synthase activity
def: "Catalysis of the reaction: reduced glutathione + glutathionylspermidine + ATP = trypanothione + ADP + phosphate." [EC:6.3.1.9, MetaCyc:6.3.1.9-RXN]
GO:0047480
name: UDP-N-acetylmuramoyl-tripeptide-D-alanyl-D-alanine ligase activity
def: "Catalysis of the reaction: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine + ATP + D-alanyl-D-alanine = phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine + ADP." [EC:6.3.2.10, MetaCyc:6.3.2.10-RXN]
GO:0047481
name: D-alanine-alanyl-poly(glycerolphosphate) ligase activity
def: "Catalysis of the reaction: alanyl-poly(glycerolphosphate) + D-alanine + ATP = D-alanyl-alanyl-poly(glycerolphosphate) + phosphate + ADP." [EC:6.3.2.16, MetaCyc:6.3.2.16-RXN]
GO:0047482
name: UDP-N-acetylmuramoyl-L-alanyl-D-glutamate-L-lysine ligase activity
def: "Catalysis of the reaction: L-lysine + ATP + UDP-N-acetylmuramoyl-L-alanyl-D-glutamate = ADP + 2 H(+) + phosphate + UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine." [EC:6.3.2.7, RHEA:17972]
GO:0047483
name: imidazoleacetate-phosphoribosyldiphosphate ligase activity
def: "Catalysis of the reaction: 5-phospho-alpha-D-ribose 1-diphosphate + ATP + H(2)O + imidazol-4-ylacetate = 1-(5-phosphoribosyl)imidazol-4-ylacetate + ADP + diphosphate + 2 H(+) + phosphate." [EC:6.3.4.8, RHEA:16488]
GO:0047485
name: protein N-terminus binding
def: "Interacting selectively and non-covalently with a protein N-terminus, the end of any peptide chain at which the 2-amino (or 2-imino) function of a constituent amino acid is not attached in peptide linkage to another amino-acid residue." [ISBN:0198506732]
GO:0047486
name: chondroitin ABC lyase activity
def: "Catalysis of the eliminative degradation of polysaccharides containing 1,4-beta-D-hexosaminyl and 1,3-beta-D-glucuronosyl or 1,3-alpha-L-iduronosyl linkages to disaccharides containing 4-deoxy-beta-D-gluc-4-enuronosyl groups." [EC:4.2.2.4]
GO:0047487
name: oligogalacturonide lyase activity
def: "Catalysis of the reaction: 4-(4-deoxy-alpha-D-gluc-4-enuronosyl)-D-galacturonate = 2 5-dehydro-4-deoxy-D-glucuronate." [EC:4.2.2.6, MetaCyc:OLIGOGALACTURONIDE-LYASE-RXN]
GO:0047488
name: heparin lyase activity
def: "Catalysis of the eliminative cleavage of polysaccharides containing 1,4-linked D-glucuronate or L-iduronate residues and 1,4-alpha-linked 2-sulfoamino-2-deoxy-6-sulfo-D-glucose residues to give oligosaccharides with terminal 4-deoxy-alpha-D-gluc-4-enuronosyl groups at their nonreducing ends." [EC:4.2.2.7, MetaCyc:4.2.2.7-RXN]
GO:0047489
name: pectate disaccharide-lyase activity
def: "Catalysis of the reaction: a pectate = a pectate + 4-(4-deoxy-alpha-D-galact-4-enuronosyl)-D-galacturonate. This reaction is the eliminative cleavage of 4-(4-deoxy-alpha-D-galact-4-enuronosyl)-D-galacturonate from the reducing end of pectate, i.e. de-esterified pectin." [EC:4.2.2.9]
GO:0047490
name: pectin lyase activity
def: "Catalysis of the reaction: a pectin = an oligosaccharide with 4-deoxy-6-O-methyl-alpha-D-galact-4-enuronate end + a pectin. This reaction is the eliminative cleavage of (1->4)-alpha-D-galacturonan methyl ester to give oligosaccharides with 4-deoxy-6-O-methyl-alpha-D-galact-4-enuronosyl groups at their nonreducing ends." [EC:4.2.2.10]
GO:0047491
name: poly(alpha-L-guluronate) lyase activity
def: "Catalysis of the reaction: polysaccharides containing a terminal alpha-L-guluronate group = oligosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enuronosyl end. This reaction is the eliminative cleavage of polysaccharides containing a terminal a-L-guluronate group, to give oligopolysaccharides with 4-deoxy-a-L-erythro-hex-4-enuronosyl groups at their nonreducing ends." [EC:4.2.2.11]
GO:0047492
name: xanthan lyase activity
def: "Catalysis of the reaction: xanthan = oligosaccharide with 4-deoxy-alpha-L-threo-hex-4-enuronosyl end + pyruvylate mannose. This reaction is the eliminative cleavage of the terminal beta-D-mannosyl-beta-D-1,4-glucuronosyl linkage of the side-chain of the polysaccharide xanthan, leaving a 4-deoxy-alpha-L-threo-hex-4-enuronosyl group at the terminus of the side-chain." [EC:4.2.2.12]
GO:0047493
name: ceramide cholinephosphotransferase activity
def: "Catalysis of the reaction: CDP-choline + ceramide = CMP + H(+) + sphingomyelin." [EC:2.7.8.3, RHEA:16276]
GO:0047494
name: serine-phosphoethanolamine synthase activity
def: "Catalysis of the reaction: L-serine + CDP-ethanolamine = L-serine-phosphoethanolamine + CMP + H(+)." [EC:2.7.8.4, RHEA:22659]
GO:0047495
name: membrane-oligosaccharide glycerophosphotransferase activity
def: "Catalysis of the transfer of a glycerophospho group from one membrane-derived oligosaccharide to another." [EC:2.7.8.21]
GO:0047498
name: calcium-dependent phospholipase A2 activity
def: "Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a carboxylate. This reaction requires Ca2+." [EC:3.1.1.4]
GO:0047499
name: calcium-independent phospholipase A2 activity
def: "Catalysis of the reaction: phosphatidylcholine + H2O = 1-acylglycerophosphocholine + a carboxylate. This reaction does not require Ca2+." [EC:3.1.1.4]
GO:0047500
name: (+)-borneol dehydrogenase activity
def: "Catalysis of the reaction: (+)-borneol + NAD(+) = (1R, 4R)-camphor + H(+) + NADH." [EC:1.1.1.198, RHEA:17332]
GO:0047501
name: (+)-neomenthol dehydrogenase activity
def: "Catalysis of the reaction: (+)-neomenthol + NADP(+) = (2S,5R)-menthone + H(+) + NADPH." [EC:1.1.1.208, RHEA:23815]
GO:0047502
name: (+)-sabinol dehydrogenase activity
def: "Catalysis of the reaction: (+)-cis-sabinol + NAD(+) = (1S,5S)-sabinone + H(+) + NADH." [EC:1.1.1.228, RHEA:18332]
GO:0047503
name: (-)-borneol dehydrogenase activity
def: "Catalysis of the reaction: (-)-borneol + NAD(+) = (1S,4S)-camphor + H(+) + NADH." [EC:1.1.1.227, RHEA:22131]
GO:0047504
name: (-)-menthol dehydrogenase activity
def: "Catalysis of the reaction: (-)-menthol + NADP(+) = (2S,5R)-menthone + H(+) + NADPH." [EC:1.1.1.207, RHEA:13920]
GO:0047505
name: (-)-menthol monooxygenase activity
def: "Catalysis of the reaction: (-)-menthol + H(+) + NADPH + O(2) = 1,4-menthane-3,8-diol + H(2)O + NADP(+)." [EC:1.14.13.46, RHEA:11651]
GO:0047506
name: (deoxy)adenylate kinase activity
def: "Catalysis of the reaction: ATP + dAMP = ADP + dADP." [EC:2.7.4.11, MetaCyc:DEOXYADENYLATE-KINASE-RXN]
GO:0047507
name: (deoxy)nucleoside-phosphate kinase activity
def: "Catalysis of the reaction: ATP + deoxynucleoside phosphate = ADP + deoxynucleoside diphosphate." [EC:2.7.4.13, MetaCyc:DEOXYNUCLEOSIDE-PHOSPHATE-KINASE-RXN]
GO:0047508
name: (R)-2-methylmalate dehydratase activity
def: "Catalysis of the reaction: (R)-citramalate = 2-methylmaleate + H(2)O." [EC:4.2.1.35, RHEA:22335]
GO:0047509
name: (R)-dehydropantoate dehydrogenase activity
def: "Catalysis of the reaction: (R)-4-dehydropantoate + H(2)O + NAD(+) = (R)-3,3-dimethylmalate + 2 H(+) + NADH." [EC:1.2.1.33, RHEA:19352]
GO:0047510
name: (S)-2-methylmalate dehydratase activity
def: "Catalysis of the reaction: S-citramalate = H(2)O + mesaconate." [EC:4.2.1.34, RHEA:13532]
GO:0047511
name: (S)-methylmalonyl-CoA hydrolase activity
def: "Catalysis of the reaction: (S)-methylmalonyl-CoA + H(2)O = CoA + H(+) + methylmalonate." [EC:3.1.2.17, RHEA:17348]
GO:0047512
name: (S,S)-butanediol dehydrogenase activity
def: "Catalysis of the reaction: (S,S)-butane-2,3-diol + NAD(+) = acetoin + H(+) + NADH." [EC:1.1.1.76, RHEA:12187]
GO:0047513
name: 1,2-alpha-L-fucosidase activity
def: "Catalysis of the reaction: H2O + methyl-2-alpha-L-fucopyranosyl-beta-D-galactoside = L-fucose + methyl beta-D-galactoside." [EC:3.2.1.63, MetaCyc:12-ALPHA-L-FUCOSIDASE-RXN]
GO:0047514
name: 1,3-beta-D-glucan phosphorylase activity
def: "Catalysis of the reaction: [(1->3)-beta-D-glucosyl](n) + phosphate = [(1->3)-beta-D-glucosyl](n-1) + alpha-D-glucose 1-phosphate; substrates include laminarin." [EC:2.4.1.97, MetaCyc:13-BETA-GLUCAN-PHOSPHORYLASE-RXN]
GO:0047515
name: 1,3-beta-oligoglucan phosphorylase activity
def: "Catalysis of the reaction: [oligomeric (1->3)-beta-D-glucosyl](n) + phosphate = [(1->3)-beta-D-glucosyl](n-1) + alpha-D-glucose 1-phosphate." [EC:2.4.1.30, MetaCyc:13-BETA-OLIGOGLUCAN-PHOSPHORYLASE-RXN]
GO:0047516
name: 1,3-propanediol dehydrogenase activity
def: "Catalysis of the reaction: propane-1,3-diol + NAD+ = 3-hydroxypropanal + NADH + H+." [EC:1.1.1.202, MetaCyc:13-PROPANEDIOL-DEHYDROGENASE-RXN]
GO:0047517
name: 1,4-beta-D-xylan synthase activity
def: "Catalysis of the reaction: UDP-D-xylose + [(1->4)-beta-D-xylan](n) = UDP + [(1->4)-beta-D-xylan](n+1)." [EC:2.4.2.24, MetaCyc:1\,4-BETA-D-XYLAN-SYNTHASE-RXN]
GO:0047518
name: 1-methyladenosine nucleosidase activity
def: "Catalysis of the reaction: 1-methyladenosine + H(2)O = 1-methyladenine + ribofuranose." [EC:3.2.2.13, RHEA:12868]
GO:0047519
name: quinate dehydrogenase (quinone) activity
def: "Catalysis of the reaction: (-)-quinate + pyrroloquinoline-quinone = (-)-3-dehydroquinate + pyrroloquinoline-quinol." [RHEA:23675]
GO:0047520
name: 11-cis-retinyl-palmitate hydrolase activity
def: "Catalysis of the reaction: 11-cis-retinyl palmitate + H(2)O = 11-cis-retinol + H(+) + palmitate." [EC:3.1.1.63, RHEA:19700]
GO:0047521
name: 3alpha,7alpha,12beta-trihydroxy-5beta-cholanate dehydrogenase activity
def: "Catalysis of the reaction: 3alpha,7alpha,12beta-trihydroxy-5beta-cholanate + NADP(+) = 3alpha,7alpha-dihydroxy-12-oxo-5beta-cholanate + H(+) + NADPH." [EC:1.1.1.238, RHEA:21427]
GO:0047522
name: 15-oxoprostaglandin 13-oxidase activity
def: "Catalysis of the reaction: (5Z)-(15S)-11-alpha-hydroxy-9,15-dioxoprostanoate + NAD(P)+ -> (5Z)-(15S)-11-alpha-hydroxy-9,15-dioxoprosta-13-enoate + NAD(P)H + H+." [EC:1.3.1.48, KEGG:R04556, KEGG:R04557, MetaCyc:15-OXOPROSTAGLANDIN-13-REDUCTASE-RXN]
GO:0047524
name: 16-hydroxysteroid epimerase activity
def: "Catalysis of the reaction: 16-alpha-hydroxysteroid = 16-beta-hydroxysteroid." [EC:5.1.99.2, MetaCyc:16-HYDROXYSTEROID-EPIMERASE-RXN]
GO:0047525
name: 2'-hydroxydaidzein reductase activity
def: "Catalysis of the reaction: 2'-hydroxydihydrodaidzein + NADP+ = 2'-hydroxydaidzein + NADPH + H+." [EC:1.3.1.51, MetaCyc:2'-HYDROXYDAIDZEIN-REDUCTASE-RXN]
GO:0047526
name: 2'-hydroxyisoflavone reductase activity
def: "Catalysis of the reaction: vestitone + NADP+ = 2'-hydroxyformononetin + NADPH + H+." [EC:1.3.1.45, MetaCyc:2-HYDROXYISOFLAVONE-REDUCTASE-RXN]
GO:0047527
name: 2,3-dihydroxybenzoate-serine ligase activity
def: "Catalysis of the reaction: ATP + 2,3-dihydroxybenzoate + L-serine = products of ATP breakdown + N-(2,3-dihydroxybenzoyl)-L-serine." [EC:6.3.2.14, MetaCyc:1\,3-DIHYDROXYBENZOATE--SERINE-LIGASE-RXN]
GO:0047528
name: 2,3-dihydroxyindole 2,3-dioxygenase activity
def: "Catalysis of the reaction: 2,3-dihydroxyindole + O(2) = anthranilate + CO(2) + H(+)." [EC:1.13.11.23, RHEA:19448]
GO:0047529
name: 2,3-dimethylmalate lyase activity
def: "Catalysis of the reaction: (2R,3S)-2,3-dimethylmalate = propanoate + pyruvate." [EC:4.1.3.32, RHEA:10475]
GO:0047530
name: 2,4-diaminopentanoate dehydrogenase activity
def: "Catalysis of the reaction: 2,4-diaminopentanoate + H2O + NAD(P)+ = 2-amino-4-oxopentanoate + NH3 + NAD(P)H + H+." [EC:1.4.1.12, MetaCyc:24-DIAMINOPENTANOATE-DEHYDROGENASE-RXN]
GO:0047531
name: 2,5-diaminovalerate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + ornithine = 5-amino-2-oxopentanoate + L-glutamate." [EC:2.6.1.8, RHEA:16020]
GO:0047532
name: 2,5-dioxopiperazine hydrolase activity
def: "Catalysis of the reaction: 2,5-dioxopiperazine + H(2)O = glycylglycine." [EC:3.5.2.13, RHEA:21811]
GO:0047533
name: 2,5-dioxovalerate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: 2,5-dioxopentanoate + NADP+ + H2O = 2-oxoglutarate + NADPH + H+." [EC:1.2.1.26, IMG:03338, MetaCyc:25-DIOXOVALERATE-DEHYDROGENASE-RXN]
GO:0047534
name: 2-acetolactate mutase activity
def: "Catalysis of the reaction: 2-acetolactate = 3-hydroxy-3-methyl-2-oxobutanoate." [EC:5.4.99.3, MetaCyc:2-ACETOLACTATE-MUTASE-RXN]
GO:0047535
name: 2-alkyn-1-ol dehydrogenase activity
def: "Catalysis of the reaction: 2-butyne-1,4-diol + NAD(+) = 4-hydroxy-2-butynal + H(+) + NADH." [EC:1.1.1.165, RHEA:19104]
GO:0047536
name: 2-aminoadipate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-2-aminoadipate = 2-oxoadipate + L-glutamate." [EC:2.6.1.39, RHEA:12604]
GO:0047537
name: 2-aminohexanoate transaminase activity
def: "Catalysis of the reaction: L-2-aminohexanoate + 2-oxoglutarate = 2-oxohexanoate + L-glutamate." [EC:2.6.1.67, MetaCyc:2-AMINOHEXANOATE-AMINOTRANSFERASE-RXN]
GO:0047538
name: 2-carboxy-D-arabinitol-1-phosphatase activity
def: "Catalysis of the reaction: 2-carboxy-D-arabinitol 1-phosphate + H(2)O = 2-carboxy-D-arabinitol + phosphate." [EC:3.1.3.63, RHEA:17840]
GO:0047539
name: 2-deoxyglucosidase activity
def: "Catalysis of the reaction: H2O + a 2-deoxy-alpha-D-glucoside = 2-deoxy-D-glucose + an alcohol." [EC:3.2.1.112, MetaCyc:2-DEOXYGLUCOSIDASE-RXN]
GO:0047540
name: 2-enoate reductase activity
def: "Catalysis of the reaction: butanoate + NAD+ = 2-butenoate + NADH + H+." [EC:1.3.1.31, MetaCyc:2-ENOATE-REDUCTASE-RXN]
GO:0047541
name: 2-furoate-CoA ligase activity
def: "Catalysis of the reaction: 2-furoate + ATP + CoA = 2-furoyl-CoA + AMP + diphosphate + H(+)." [EC:6.2.1.31, RHEA:19272]
GO:0047542
name: 2-furoyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: 2-furoyl-CoA + A + H(2)O = 5-hydroxy-2-furoyl-CoA + AH(2) + H(+)." [EC:1.3.99.8, RHEA:21483]
GO:0047543
name: 2-hexadecenal reductase activity
def: "Catalysis of the reaction: NADP(+) + palmitaldehyde = trans-hexadec-2-enal + H(+) + NADPH." [EC:1.3.1.27, RHEA:12447]
GO:0047544
name: 2-hydroxybiphenyl 3-monooxygenase activity
def: "Catalysis of the reaction: biphenyl-2-ol + H(+) + NADH + O(2) = biphenyl-2,3-diol + H(2)O + NAD(+)." [EC:1.14.13.44, RHEA:11999]
GO:0047545
name: 2-hydroxyglutarate dehydrogenase activity
def: "Catalysis of the reaction: (S)-2-hydroxyglutarate + acceptor = 2-oxoglutarate + reduced acceptor." [EC:1.1.99.2, MetaCyc:2-HYDROXYGLUTARATE-DEHYDROGENASE-RXN]
GO:0047546
name: 2-hydroxypyridine 5-monooxygenase activity
def: "Catalysis of the reaction: 2-hydroxypyridine + AH(2) + O(2) = 2,5-dihydroxypyridine + A + H(2)O." [EC:1.14.99.26, RHEA:16976]
GO:0047547
name: 2-methylcitrate dehydratase activity
def: "Catalysis of the reaction: (2S,3S)-2-methylcitrate = cis-2-methylaconitate + H(2)O." [EC:4.2.1.79, RHEA:17728]
GO:0047548
name: 2-methyleneglutarate mutase activity
def: "Catalysis of the reaction: 2-methyleneglutarate = 2-methylene-3-methylsuccinate." [EC:5.4.99.4, RHEA:13796]
GO:0047549
name: 2-nitrophenol 2-monooxygenase activity
def: "Catalysis of the reaction: 2-nitrophenol + 2 H(+) + 2 NADPH + O(2) = catechol + H(2)O + 2 NADP(+) + nitrite." [EC:1.14.13.31, RHEA:19460]
GO:0047550
name: 2-oxoadipate reductase activity
def: "Catalysis of the reaction: 2-hydroxyadipate + NAD(+) = 2-oxoadipate + H(+) + NADH." [EC:1.1.1.172, RHEA:14796]
GO:0047551
name: 2-oxoaldehyde dehydrogenase (NAD) activity
def: "Catalysis of the reaction: a 2-oxoaldehyde + NAD+ + H2O = a 2-oxo acid + NADH + H+." [EC:1.2.1.23, MetaCyc:2-OXOALDEHYDE-DEHYDROGENASE-NAD+-RXN]
GO:0047552
name: 2-oxoaldehyde dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: a 2-oxoaldehyde + NADP+ + H2O = a 2-oxo acid + NADPH + H+." [EC:1.2.1.49, MetaCyc:2-OXOALDEHYDE-DEHYDROGENASE-NADP+-RXN]
GO:0047553
name: 2-oxoglutarate synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + CoA + oxidized ferredoxin = succinyl-CoA + CO2 + reduced ferredoxin." [EC:1.2.7.3, MetaCyc:2-OXOGLUTARATE-SYNTHASE-RXN]
GO:0047554
name: 2-pyrone-4,6-dicarboxylate lactonase activity
def: "Catalysis of the reaction: 2-oxo-2H-pyran-4,6-dicarboxylate + H(2)O = 4-carboxy-2-hydroxyhexa-2,4-dienedioate + H(+)." [EC:3.1.1.57, RHEA:10647]
GO:0047555
name: 3',5'-cyclic-GMP phosphodiesterase activity
def: "Catalysis of the reaction: guanosine 3',5'-cyclic phosphate + H2O = guanosine 5'-phosphate." [EC:3.1.4.35, MetaCyc:35-CYCLIC-GMP-PHOSPHODIESTERASE-RXN]
GO:0047556
name: 3,4-dihydroxyphthalate decarboxylase activity
def: "Catalysis of the reaction: 3,4-dihydroxyphthalate + H(+) = 3,4-dihydroxybenzoate + CO(2)." [EC:4.1.1.69, RHEA:18604]
GO:0047557
name: 3-aci-nitropropanoate oxidase activity
def: "Catalysis of the reaction: 3-aci-nitropropanoate + H(2)O + O(2) = 3-oxopropanoate + H(2)O(2) + nitrite." [EC:1.7.3.5, RHEA:22375]
GO:0047558
name: 3-cyanoalanine hydratase activity
def: "Catalysis of the reaction: L-asparagine = 3-cyano-L-alanine + H(2)O + H(+)." [EC:4.2.1.65, RHEA:15388]
GO:0047559
name: 3-dehydro-L-gulonate 2-dehydrogenase activity
def: "Catalysis of the reaction: 3-dehydro-L-gulonate + NAD(P)+ = (4R,5S)-4,5,6-trihydroxy-2,3-dioxohexanoate + NAD(P)H + H+." [EC:1.1.1.130, MetaCyc:3-DEHYDRO-L-GULONATE-2-DEHYDROGENASE-RXN]
GO:0047560
name: 3-dehydrosphinganine reductase activity
def: "Catalysis of the reaction: NADP(+) + sphinganine = 3-dehydrosphinganine + H(+) + NADPH." [EC:1.1.1.102, RHEA:22643]
GO:0047561
name: 3-hydroxyanthranilate oxidase activity
def: "Catalysis of the reaction: 3-hydroxyanthranilate + O(2) = 6-imino-5-oxocyclohexa-1,3-dienecarboxylate + H(2)O(2)." [EC:1.10.3.5, RHEA:17248]
GO:0047562
name: 3-hydroxyaspartate aldolase activity
def: "Catalysis of the reaction: (3R)-3-hydroxy-L-aspartate = glycine + glyoxylate." [EC:4.1.3.14, RHEA:14380]
GO:0047563
name: 3-hydroxybenzoate 2-monooxygenase activity
def: "Catalysis of the reaction: 3-hydroxybenzoate + AH(2) + O(2) = 2,3-dihydroxybenzoate + A + H(2)O." [EC:1.14.99.23, RHEA:14196]
GO:0047564
name: 3-hydroxycyclohexanone dehydrogenase activity
def: "Catalysis of the reaction: 3-hydroxycyclohexanone + A = AH(2) + cyclohexane-1,3-dione." [EC:1.1.99.26, RHEA:15908]
GO:0047565
name: 3-hydroxypropionate dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: 3-hydroxypropanoate + NAD(+) = 3-oxopropanoate + H(+) + NADH." [EC:1.1.1.59, RHEA:13360]
GO:0047566
name: 3-ketovalidoxylamine C-N-lyase activity
def: "Catalysis of the reaction: 4-nitrophenyl-3-ketovalidamine = 4-nitroaniline + 5-D-(5/6)-5-C-(hydroxymethyl)-2,6-dihydroxycyclohex-2-en-1-one + H(+)." [EC:4.3.3.1, RHEA:22771]
GO:0047567
name: 3-methyleneoxindole reductase activity
def: "Catalysis of the reaction: 3-methyloxindole + NADP(+) = 3-methyleneoxindole + H(+) + NADPH." [EC:1.3.1.17, RHEA:20260]
GO:0047568
name: 3-oxo-5-beta-steroid 4-dehydrogenase activity
def: "Catalysis of the reaction: a 3-oxo-5-beta-steroid + acceptor = a 3-oxo-D4-steroid + reduced acceptor." [EC:1.3.99.6, MetaCyc:3-OXO-5-BETA-STEROID-4-DEHYDROGENASE-RXN]
GO:0047569
name: 3-oxoadipate CoA-transferase activity
def: "Catalysis of the reaction: succinyl-CoA + 3-oxoadipate = succinate + 3-oxoadipyl-CoA." [EC:2.8.3.6, MetaCyc:3-OXOADIPATE-COA-TRANSFERASE-RXN]
GO:0047570
name: 3-oxoadipate enol-lactonase activity
def: "Catalysis of the reaction: 3-oxoadipate enol-lactone + H2O = 3-oxoadipate." [EC:3.1.1.24, MetaCyc:3-OXOADIPATE-ENOL-LACTONASE-RXN]
GO:0047571
name: 3-oxosteroid 1-dehydrogenase activity
def: "Catalysis of the reaction: a 3-oxosteroid + acceptor = a 3-oxo-D1-steroid + reduced acceptor." [EC:1.3.99.4, MetaCyc:3-OXOSTEROID-1-DEHYDROGENASE-RXN]
GO:0047572
name: 3-phosphoglycerate phosphatase activity
def: "Catalysis of the reaction: 3-phospho-D-glycerate + H(2)O = D-glycerate + phosphate." [EC:3.1.3.38, RHEA:12415]
GO:0047573
name: 4-acetamidobutyrate deacetylase activity
def: "Catalysis of the reaction: 4-acetamidobutanoate + H2O = acetate + 4-aminobutanoate." [EC:3.5.1.63, MetaCyc:4-ACETAMIDOBUTYRATE-DEACETYLASE-RXN]
GO:0047574
name: 4-acetamidobutyryl-CoA deacetylase activity
def: "Catalysis of the reaction: 4-acetamidobutanoyl-CoA + H(2)O = 4-aminobutanoyl-CoA + acetate." [EC:3.5.1.51, RHEA:22931]
GO:0047575
name: 4-carboxymuconolactone decarboxylase activity
def: "Catalysis of the reaction: (R)-2-(carboxymethyl)-5-oxo-2,5-dihydro-2-furoate + H(+) = 5-oxo-4,5-dihydro-2-furylacetate + CO(2)." [EC:4.1.1.44, RHEA:23351]
GO:0047576
name: 4-chlorobenzoate dehalogenase activity
def: "Catalysis of the reaction: 4-chlorobenzoate + H(2)O = 4-hydroxybenzoate + chloride + H(+)." [EC:3.8.1.6, RHEA:23443]
GO:0047577
name: 4-hydroxybutyrate dehydrogenase activity
def: "Catalysis of the reaction: 4-hydroxybutanoate + NAD(+) = H(+) + NADH + succinate semialdehyde." [EC:1.1.1.61, RHEA:23951]
GO:0047578
name: 4-hydroxyglutamate transaminase activity
def: "Catalysis of the reaction: 4-hydroxy-L-glutamate + 2-oxoglutarate = 4-hydroxy-2-oxoglutarate + L-glutamate." [EC:2.6.1.23, MetaCyc:4-HYDROXYGLUTAMATE-AMINOTRANSFERASE-RXN]
GO:0047579
name: 4-hydroxymandelate oxidase activity
def: "Catalysis of the reaction: (S)-4-hydroxymandelate + H(+) + O(2) = 4-hydroxybenzaldehyde + CO(2) + H(2)O(2)." [EC:1.1.3.19, RHEA:15836]
GO:0047580
name: 4-hydroxyproline epimerase activity
def: "Catalysis of the reaction: trans-4-hydroxy-L-proline = cis-4-hydroxy-D-proline." [EC:5.1.1.8, RHEA:21155]
GO:0047581
name: 4-methyleneglutamate-ammonia ligase activity
def: "Catalysis of the reaction: 4-methylene-L-glutamate + ATP + NH(4)(+) = 4-methylene-L-glutamine + AMP + diphosphate + 2 H(+)." [EC:6.3.1.7, RHEA:13856]
GO:0047582
name: 4-methyleneglutaminase activity
def: "Catalysis of the reaction: 4-methylene-L-glutamine + H(2)O = 4-methylene-L-glutamate + NH(4)(+)." [EC:3.5.1.67, RHEA:14744]
GO:0047583
name: 4-methyloxaloacetate esterase activity
def: "Catalysis of the reaction: 4-methoxy-2,4-dioxobutanoate + H(2)O = H(+) + methanol + oxaloacetate." [EC:3.1.1.44, RHEA:10567]
GO:0047584
name: 4-oxalmesaconate hydratase activity
def: "Catalysis of the reaction: 2-hydroxy-4-oxobutane-1,2,4-tricarboxylate = (1E)-4-oxobut-1-ene-1,2,4-tricarboxylate + H(2)O." [EC:4.2.1.83, RHEA:17404]
GO:0047585
name: 4-pyridoxolactonase activity
def: "Catalysis of the reaction: 4-pyridoxolactone + H(2)O = 4-pyridoxate + H(+)." [EC:3.1.1.27, RHEA:14304]
GO:0047586
name: 5'-acylphosphoadenosine hydrolase activity
def: "Catalysis of the reaction: 5'-acylphosphoadenosine + H2O = AMP + a carboxylate." [EC:3.6.1.20, MetaCyc:5-ACYLPHOSPHOADENOSINE-HYDROLASE-RXN]
GO:0047587
name: 5-alpha-hydroxysteroid dehydratase activity
def: "Catalysis of the reaction: 5alpha-ergosta-7,22-diene-3beta,5-diol = ergosterol + H(2)O." [EC:4.2.1.62, RHEA:22067]
GO:0047588
name: 5-aminopentanamidase activity
def: "Catalysis of the reaction: 5-aminopentanamide + H2O = 5-aminopentanoate + NH3." [EC:3.5.1.30, MetaCyc:5-AMINOPENTANAMIDASE-RXN]
GO:0047589
name: 5-aminovalerate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + 5-aminopentanoate = 5-oxopentanoate + L-glutamate." [EC:2.6.1.48, RHEA:10215]
GO:0047590
name: 5-dehydro-2-deoxygluconokinase activity
def: "Catalysis of the reaction: ATP + 5-dehydro-2-deoxy-D-gluconate = ADP + 6-phospho-5-dehydro-2-deoxy-D-gluconate." [EC:2.7.1.92, MetaCyc:5-DEHYDRO-2-DEOXYGLUCONOKINASE-RXN]
GO:0047591
name: 5-hydroxypentanoate CoA-transferase activity
def: "Catalysis of the reaction: 5-hydroxypentanoate + acetyl-CoA = 5-hydroxy-pentanoyl-CoA + acetate." [EC:2.8.3.14, RHEA:23499]
GO:0047592
name: 5-pyridoxate dioxygenase activity
def: "Catalysis of the reaction: 5-pyridoxate + NADPH + O(2) = (2Z)-2-(acetamidomethylene)-3-(hydroxymethyl)succinate + NADP(+)." [EC:1.14.12.5, RHEA:11155]
GO:0047593
name: 6-acetylglucose deacetylase activity
def: "Catalysis of the reaction: 6-acetyl-D-glucose + H(2)O = D-glucose + acetate + H(+)." [EC:3.1.1.33, RHEA:18488]
GO:0047594
name: 6-beta-hydroxyhyoscyamine epoxidase activity
def: "Catalysis of the reaction: (6S)-6-hydroxyhyoscyamine + 2-oxoglutarate + O(2) = CO(2) + H(2)O + H(+) + scopolamine + succinate." [EC:1.14.11.14, RHEA:12800]
GO:0047595
name: 6-hydroxynicotinate reductase activity
def: "Catalysis of the reaction: 1,4,5,6-tetrahydro-6-oxonicotinate + oxidized ferredoxin = 6-hydroxynicotinate + reduced ferredoxin." [EC:1.3.7.1, MetaCyc:6-HYDROXYNICOTINATE-REDUCTASE-RXN]
GO:0047596
name: 6-methylsalicylate decarboxylase activity
def: "Catalysis of the reaction: 6-methylsalicylate + H(+) = 3-cresol + CO(2)." [EC:4.1.1.52, RHEA:23115]
GO:0047597
name: 6-oxocineole dehydrogenase activity
def: "Catalysis of the reaction: 6-oxocineole + H(+) + NADPH + O(2) = 1,6,6-trimethyl-2,7-dioxabicyclo[3.2.2]nonan-3-one + H(2)O + NADP(+)." [EC:1.14.13.51, RHEA:24327]
GO:0047598
name: 7-dehydrocholesterol reductase activity
def: "Catalysis of the reaction: cholesterol + NADP+ = cholesta-5,7-dien-3-beta-ol + NADPH + H+." [EC:1.3.1.21]
GO:0047599
name: 8-oxocoformycin reductase activity
def: "Catalysis of the reaction: coformycin + NADP(+) = 8-oxocoformycin + 2 H(+) + NADPH." [EC:1.1.1.235, RHEA:23171]
GO:0047600
name: abequosyltransferase activity
def: "Catalysis of the reaction: CDP-abequose + D-mannosyl-L-rhamnosyl-D-galactose-1-diphospholipid = CDP + D-abequosyl-D-mannosyl-rhamnosyl-D-galactose-1-diphospholipid." [EC:2.4.1.60, MetaCyc:ABEQUOSYLTRANSFERASE-RXN]
GO:0047601
name: acetate kinase (diphosphate) activity
def: "Catalysis of the reaction: acetate + diphosphate = acetyl phosphate + phosphate." [EC:2.7.2.12, RHEA:24279]
GO:0047602
name: acetoacetate decarboxylase activity
def: "Catalysis of the reaction: acetoacetate + H(+) = acetone + CO(2)." [EC:4.1.1.4, RHEA:19732]
GO:0047603
name: acetoacetyl-CoA hydrolase activity
def: "Catalysis of the reaction: acetoacetyl-CoA + H(2)O = acetoacetate + CoA + H(+)." [EC:3.1.2.11, RHEA:15676]
GO:0047604
name: acetoin racemase activity
def: "Catalysis of the reaction: (S)-acetoin = (R)-acetoin." [EC:5.1.2.4, RHEA:12095]
GO:0047605
name: acetolactate decarboxylase activity
def: "Catalysis of the reaction: (S)-2-hydroxy-2-methyl-3-oxobutanoate = (R)-2-acetoin + CO2." [EC:4.1.1.5, MetaCyc:ACETOLACTATE-DECARBOXYLASE-RXN]
GO:0047606
name: hydroxynitrilase activity
def: "Catalysis of the reaction: a hydroxynitrile = cyanide + an aldehyde or ketone." [EC:4.1.2, GOC:mah, MetaCyc:ACETONE-CYANHYDRIN-LYASE-RXN]
GO:0047608
name: acetylindoxyl oxidase activity
def: "Catalysis of the reaction: N-acetylindoxyl + O2 = N-acetylisatin + unknown." [EC:1.7.3.2, MetaCyc:ACETYLINDOXYL-OXIDASE-RXN]
GO:0047609
name: acetylputrescine deacetylase activity
def: "Catalysis of the reaction: N-acetylputrescine + H(2)O = acetate + putrescine." [EC:3.5.1.62, RHEA:23415]
GO:0047610
name: acetylsalicylate deacetylase activity
def: "Catalysis of the reaction: acetylsalicylate + H(2)O = acetate + H(+) + salicylate." [EC:3.1.1.55, RHEA:11755]
GO:0047611
name: acetylspermidine deacetylase activity
def: "Catalysis of the reaction: N(8)-acetylspermidine + H(2)O = acetate + spermidine." [EC:3.5.1.48, RHEA:23931]
GO:0047612
name: acid-CoA ligase (GDP-forming) activity
def: "Catalysis of the reaction: a carboxylate + CoA + GTP = acyl-CoA + GDP + H(+) + phosphate." [EC:6.2.1.10, RHEA:10971]
GO:0047613
name: aconitate decarboxylase activity
def: "Catalysis of the reaction: cis-aconitate + H(+) = CO(2) + itaconate." [EC:4.1.1.6, RHEA:15256]
GO:0047614
name: aconitate delta-isomerase activity
def: "Catalysis of the reaction: trans-aconitate = cis-aconitate." [EC:5.3.3.7, RHEA:17268]
GO:0047615
name: actinomycin lactonase activity
def: "Catalysis of the reaction: actinomycin + H2O = actinomycinic monolactone." [EC:3.1.1.39, MetaCyc:ACTINOMYCIN-LACTONASE-RXN]
GO:0047616
name: acyl-CoA dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: acyl-CoA + NADP+ = 2,3-dehydroacyl-CoA + NADPH + H+." [EC:1.3.1.8, MetaCyc:ACYL-COA-DEHYDROGENASE-NADP+-RXN]
GO:0047617
name: acyl-CoA hydrolase activity
def: "Catalysis of the reaction: acyl-CoA + H2O = CoA + a carboxylate." [EC:3.1.2.20, MetaCyc:ACYL-COA-HYDROLASE-RXN]
GO:0047618
name: acylagmatine amidase activity
def: "Catalysis of the reaction: N(4)-benzoylagmatine + H(2)O = agmatine + benzoate." [EC:3.5.1.40, RHEA:15068]
GO:0047619
name: acylcarnitine hydrolase activity
def: "Catalysis of the reaction: O-acylcarnitine + H2O = a fatty acid + L-carnitine." [EC:3.1.1.28, MetaCyc:ACYLCARNITINE-HYDROLASE-RXN]
GO:0047620
name: acylglycerol kinase activity
def: "Catalysis of the reaction: ATP + acylglycerol = ADP + acyl-sn-glycerol 3-phosphate." [EC:2.7.1.94, MetaCyc:ACYLGLYCEROL-KINASE-RXN]
GO:0047621
name: acylpyruvate hydrolase activity
def: "Catalysis of the reaction: a 3-acylpyruvate + H2O = a carboxylate + pyruvate." [EC:3.7.1.5, MetaCyc:ACYLPYRUVATE-HYDROLASE-RXN]
GO:0047622
name: adenosine nucleosidase activity
def: "Catalysis of the reaction: adenosine + H2O = D-ribose + adenine." [EC:3.2.2.7, MetaCyc:ADENOSINE-NUCLEOSIDASE-RXN]
GO:0047623
name: adenosine-phosphate deaminase activity
def: "Catalysis of the reaction: an adenosine-phosphate + H20 = an inosine phosphate + NH3. Catalyzes the deamination of AMP, ADP or ATP." [EC:3.5.4.17, GOC:bf, MetaCyc:ADENOSINE-PHOSPHATE-DEAMINASE-RXN]
GO:0047624
name: adenosine-tetraphosphatase activity
def: "Catalysis of the reaction: adenosine 5'-tetraphosphate + H2O = ATP + phosphate." [EC:3.6.1.14, MetaCyc:ADENOSINE-TETRAPHOSPHATASE-RXN]
GO:0047625
name: adenosylmethionine cyclotransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) = S-methyl-5'-thioadenosine + homoserine lactone." [EC:2.5.1.4, RHEA:21935]
GO:0047626
name: adenosylmethionine hydrolase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + H(2)O = S-methyl-5'-thioadenosine + L-homoserine + H(+)." [EC:3.3.1.2, RHEA:14648]
GO:0047627
name: adenylylsulfatase activity
def: "Catalysis of the reaction: 5'-adenylyl sulfate + H(2)O = AMP + 2 H(+) + sulfate." [EC:3.6.2.1, RHEA:17044]
GO:0047628
name: ADP-thymidine kinase activity
def: "Catalysis of the reaction: ADP + thymidine = AMP + thymidine 5'-phosphate." [EC:2.7.1.118, MetaCyc:ADP--THYMIDINE-KINASE-RXN]
GO:0047629
name: ADP deaminase activity
def: "Catalysis of the reaction: ADP + H2O = IDP + NH3." [EC:3.5.4.7, MetaCyc:ADP-DEAMINASE-RXN]
GO:0047630
name: ADP-phosphoglycerate phosphatase activity
def: "Catalysis of the reaction: 3-ADP-2-phosphoglycerate + H(2)O = 3-ADP-glycerate + phosphate." [EC:3.1.3.28, RHEA:15864]
GO:0047631
name: ADP-ribose diphosphatase activity
def: "Catalysis of the reaction: ADP-ribose + H2O = AMP + D-ribose 5-phosphate." [EC:3.6.1.13, MetaCyc:ADP-RIBOSE-PYROPHOSPHATASE-RXN]
GO:0047632
name: agmatine deiminase activity
def: "Catalysis of the reaction: agmatine + H2O = N-carbamoylputrescine + NH3." [EC:3.5.3.12, MetaCyc:AGMATINE-DEIMINASE-RXN]
GO:0047633
name: agmatine kinase activity
def: "Catalysis of the reaction: agmatine + ATP = N(4)-phosphoagmatine + ADP + 3 H(+)." [EC:2.7.3.10, RHEA:15956]
GO:0047634
name: agmatine N4-coumaroyltransferase activity
def: "Catalysis of the reaction: 4-coumaroyl-CoA + agmatine = N-(4-guanidiniumylbutyl)-4-hydroxycinnamamide + CoA + H(+)." [EC:2.3.1.64, RHEA:13408]
GO:0047635
name: alanine-oxo-acid transaminase activity
def: "Catalysis of the reaction: L-alanine + a 2-oxo acid = pyruvate + an L-amino acid." [EC:2.6.1.12, MetaCyc:ALANINE--OXO-ACID-AMINOTRANSFERASE-RXN]
GO:0047636
name: alanopine dehydrogenase activity
def: "Catalysis of the reaction: 2,2'-iminodipropanoate + H(2)O + NAD(+) = L-alanine + H(+) + NADH + pyruvate." [EC:1.5.1.17, RHEA:17592]
GO:0047637
name: alanylphosphatidylglycerol synthase activity
def: "Catalysis of the reaction: L-alanyl-tRNA + phosphatidylglycerol = tRNA + 3-O-L-alanyl-1-O-phosphatidylglycerol." [EC:2.3.2.11, MetaCyc:ALANYLPHOSPHATIDYLGLYCEROL-SYNTHASE-RXN]
GO:0047638
name: albendazole monooxygenase activity
def: "Catalysis of the reaction: albendazole + H(+) + NADPH + O(2) = albendazole S-oxide + H(2)O + NADP(+)." [EC:1.14.13.32, RHEA:10799]
GO:0047639
name: alcohol oxidase activity
def: "Catalysis of the reaction: a primary alcohol + O2 = an aldehyde + H2O2." [EC:1.1.3.13, MetaCyc:ALCOHOL-OXIDASE-RXN]
GO:0047640
name: aldose 1-dehydrogenase activity
def: "Catalysis of the reaction: D-aldose + NAD+ = D-aldonolactone + NADH." [EC:1.1.1.121, MetaCyc:ALDOSE-1-DEHYDROGENASE-RXN]
GO:0047641
name: aldose-6-phosphate reductase (NADPH) activity
def: "Catalysis of the reaction: D-glucitol 6-phosphate + NADP(+) = D-glucose 6-phosphate + H(+) + NADPH." [EC:1.1.1.200, RHEA:20040]
GO:0047642
name: aldose beta-D-fructosyltransferase activity
def: "Catalysis of the reaction: alpha-D-aldosyl1 beta-D-fructoside + D-aldose2 = D-aldose1 + alpha-D-aldosyl2 beta-D-fructoside." [EC:2.4.1.162, MetaCyc:ALDOSE-BETA-FRUCTOSYLTRANSFERASE-RXN]
GO:0047643
name: alginate synthase activity
def: "Catalysis of the reaction: GDP-D-mannuronate + alginate(n) = GDP + alginate(n+1)." [EC:2.4.1.33, MetaCyc:ALGINATE-SYNTHASE-RXN]
GO:0047644
name: alizarin 2-beta-glucosyltransferase activity
def: "Catalysis of the reaction: alizarin + UDP-D-glucose = 1-hydroxy-2-(beta-D-glucosyloxy)-9,10-anthraquinone + H(+) + UDP." [EC:2.4.1.103, RHEA:20680]
GO:0047645
name: alkan-1-ol dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: primary alcohol + acceptor = aldehyde + reduced acceptor." [EC:1.1.99.20, MetaCyc:ALKAN-1-OL-DEHYDROGENASE-ACCEPTOR-RXN]
GO:0047646
name: alkanal monooxygenase (FMN-linked) activity
def: "Catalysis of the reaction: R-CHO + reduced FMN + O2 = R-COOH + FMN + H2O + light." [EC:1.14.14.3, MetaCyc:ALKANAL-MONOOXYGENASE-FMN-LINKED-RXN]
GO:0047647
name: alkylacetylglycerophosphatase activity
def: "Catalysis of the reaction: 1-alkyl-2-acetyl-sn-glycerol 3-phosphate + H(2)O = 2-acetyl-1-alkyl-sn-glycerol + phosphate." [EC:3.1.3.59, RHEA:18224]
GO:0047648
name: alkylamidase activity
def: "Catalysis of the reaction: N-methylhexanamide + H(2)O = hexanoate + methylammonium." [EC:3.5.1.39, RHEA:20084]
GO:0047649
name: alkylglycerol kinase activity
def: "Catalysis of the reaction: 1-alkyl-sn-glycerol + ATP = 1-alkyl-sn-glycerol 3-phosphate + ADP + 2 H(+)." [EC:2.7.1.93, RHEA:16940]
GO:0047650
name: alkylglycerone kinase activity
def: "Catalysis of the reaction: O-alkylglycerone + ATP = O-alkylglycerone phosphate + ADP + 2 H(+)." [EC:2.7.1.84, RHEA:23091]
GO:0047651
name: alkylhalidase activity
def: "Catalysis of the reaction: bromochloromethane + H(2)O = bromide + chloride + formaldehyde + 2 H(+)." [EC:3.8.1.1, RHEA:13768]
GO:0047652
name: allantoate deiminase activity
def: "Catalysis of the reaction: allantoate + H2O + H+ = CO2 + NH3 + ureidoglycine." [EC:3.5.3.9, MetaCyc:ALLANTOATE-DEIMINASE-RXN]
GO:0047653
name: allantoin racemase activity
def: "Catalysis of the reaction: (S)-(+)-allantoin = (R)-(-)-allantoin." [EC:5.1.99.3, RHEA:10807]
GO:0047654
name: alliin lyase activity
def: "Catalysis of the reaction: an S-alkyl-L-cysteine S-oxide = an alkyl sulfenate + 2-aminoacrylate." [EC:4.4.1.4, MetaCyc:ALLIIN-LYASE-RXN]
GO:0047655
name: allyl-alcohol dehydrogenase activity
def: "Catalysis of the reaction: allyl alcohol + NADP(+) = acrolein + H(+) + NADPH." [EC:1.1.1.54, RHEA:12171]
GO:0047656
name: alpha,alpha-trehalose phosphorylase activity
def: "Catalysis of the reaction: alpha,alpha-trehalose + phosphate = D-glucose + beta-D-glucose 1-phosphate." [EC:2.4.1.64, MetaCyc:ALPHAALPHA-TREHALOSE-PHOSPHORYLASE-RXN]
GO:0047657
name: alpha-1,3-glucan synthase activity
def: "Catalysis of the reaction: UDP-glucose + [alpha-D-glucosyl-(1,3)]n = UDP + [alpha-D-glucosyl-(1,3)]n+1." [EC:2.4.1.183, MetaCyc:ALPHA-13-GLUCAN-SYNTHASE-RXN]
GO:0047658
name: alpha-amino-acid esterase activity
def: "Catalysis of the reaction: an alpha-amino acid ester + H2O = an alpha-amino acid + an alcohol." [EC:3.1.1.43, MetaCyc:ALPHA-AMINO-ACID-ESTERASE-RXN]
GO:0047659
name: alpha-santonin 1,2-reductase activity
def: "Catalysis of the reaction: 1,2-dihydrosantonin + NAD(P)+ = alpha-santonin + NAD(P)H + H+." [EC:1.3.1.47, MetaCyc:ALPHA-SANTONIN-12-REDUCTASE-RXN]
GO:0047660
name: amidinoaspartase activity
def: "Catalysis of the reaction: N-amidino-L-aspartate + H(2)O = L-aspartate + urea." [EC:3.5.3.14, RHEA:14852]
GO:0047661
name: amino-acid racemase activity
def: "Catalysis of the reaction: an L-amino acid = a D-amino acid." [EC:5.1.1.10, MetaCyc:AMINO-ACID-RACEMASE-RXN]
GO:0047662
name: aminobenzoate decarboxylase activity
def: "Catalysis of the reaction: 4(or 2)-aminobenzoate = aniline + CO2." [EC:4.1.1.24, MetaCyc:AMINOBENZOATE-DECARBOXYLASE-RXN]
GO:0047663
name: aminoglycoside 6'-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + kanamycin B = N(6')-acetylkanamycin B + CoA + H(+). This is acetylation of the 6'-amino group of the 6-deoxy-6-aminoglucose ring." [EC:2.3.1.82, RHEA:16452]
GO:0047664
name: aminoimidazolase activity
def: "Catalysis of the reaction: 4-aminoimidazole + H2O = unidentified product + NH3." [EC:3.5.4.8, MetaCyc:AMINOIMIDAZOLASE-RXN]
GO:0047665
name: aminolevulinate transaminase activity
def: "Catalysis of the reaction: 5-aminolevulinate + pyruvate = 4,5-dioxopentanoate + L-alanine." [EC:2.6.1.43, RHEA:12483]
GO:0047666
name: ammonia kinase activity
def: "Catalysis of the reaction: ATP + NH(4)(+) = ADP + 3 H(+) + phosphoramidate." [EC:2.7.3.8, RHEA:11027]
GO:0047667
name: AMP-thymidine kinase activity
def: "Catalysis of the reaction: AMP + thymidine = adenosine + thymidine 5'-phosphate." [EC:2.7.1.114, MetaCyc:AMP--THYMIDINE-KINASE-RXN]
GO:0047668
name: amygdalin beta-glucosidase activity
def: "Catalysis of the reaction: (R)-amygdalin + H(2)O = (R)-prunasin + D-glucose." [EC:3.2.1.117, RHEA:14180]
GO:0047669
name: amylosucrase activity
def: "Catalysis of the reaction: sucrose + 1,4-alpha-D-glucosyl(n) = D-fructose + 1,4-alpha-D-glucosyl(n+1)." [EC:2.4.1.4, MetaCyc:AMYLOSUCRASE-RXN]
GO:0047670
name: anhydrotetracycline monooxygenase activity
def: "Catalysis of the reaction: anhydrotetracycline + H(+) + NADPH + O(2) = 12-dehydrotetracycline + H(2)O + NADP(+)." [EC:1.14.13.38, RHEA:11979]
GO:0047671
name: anthranilate adenylyltransferase activity
def: "Catalysis of the reaction: anthranilate + ATP = N-adenylylanthranilate + diphosphate + H(+)." [EC:2.7.7.55, RHEA:22415]
GO:0047672
name: anthranilate N-benzoyltransferase activity
def: "Catalysis of the reaction: anthranilate + benzoyl-CoA = N-benzoylanthranilate + CoA." [EC:2.3.1.144, RHEA:21603]
GO:0047673
name: anthranilate N-malonyltransferase activity
def: "Catalysis of the reaction: anthranilate + malonyl-CoA = N-malonylanthranilate + CoA." [EC:2.3.1.113, RHEA:17560]
GO:0047674
name: apiose 1-reductase activity
def: "Catalysis of the reaction: D-apiitol + NAD(+) = D-apiose + H(+) + NADH." [EC:1.1.1.114, RHEA:15304]
GO:0047675
name: arabinonate dehydratase activity
def: "Catalysis of the reaction: D-arabinonate = 2-dehydro-3-deoxy-D-arabinonate + H(2)O." [EC:4.2.1.5, RHEA:21839]
GO:0047676
name: arachidonate-CoA ligase activity
def: "Catalysis of the reaction: arachidonate + ATP + CoA = AMP + arachidonoyl-CoA + diphosphate + H(+)." [EC:6.2.1.15, RHEA:19716]
GO:0047677
name: arachidonate 8(R)-lipoxygenase activity
def: "Catalysis of the reaction: arachidonate + O(2) = (5Z,8R,9E,11Z,14Z)-8-hydroperoxyicosa-5,9,11,14-tetraenoate." [EC:1.13.11.40, RHEA:14988]
GO:0047678
name: arginine 2-monooxygenase activity
def: "Catalysis of the reaction: L-arginine + O(2) = 4-guanidinobutanamide + CO(2) + H(2)O." [EC:1.13.12.1, RHEA:10551]
GO:0047679
name: arginine racemase activity
def: "Catalysis of the reaction: L-arginine = D-arginine." [EC:5.1.1.9, MetaCyc:ARGININE-RACEMASE-RXN]
GO:0047680
name: aryl-acylamidase activity
def: "Catalysis of the reaction: anilide + H(2)O = a carboxylate + aniline + H(+)." [EC:3.5.1.13, RHEA:20300]
GO:0047681
name: aryl-alcohol dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: an aromatic alcohol + NADP+ = an aromatic aldehyde + NADPH." [EC:1.1.1.91, MetaCyc:ARYL-ALCOHOL-DEHYDROGENASE-NADP+-RXN]
GO:0047682
name: aryl-alcohol oxidase activity
def: "Catalysis of the reaction: an aromatic primary alcohol + O2 = an aromatic aldehyde + H2O2." [EC:1.1.3.7, MetaCyc:ARYL-ALCOHOL-OXIDASE-RXN]
GO:0047683
name: aryl-aldehyde dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: an aromatic aldehyde + NADP+ + AMP + diphosphate + H2O = an aromatic acid + NADPH + ATP." [EC:1.2.1.30, MetaCyc:ARYL-ALDEHYDE-DEHYDROGENASE-NADP+-RXN]
GO:0047684
name: arylamine glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + an arylamine = UDP + an N-D-glucosylarylamine." [EC:2.4.1.71, MetaCyc:ARYLAMINE-GLUCOSYLTRANSFERASE-RXN]
GO:0047685
name: amine sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + an amine = adenosine 3',5'-bisphosphate + a sulfamate." [EC:2.8.2.3, MetaCyc:ARYLAMINE-SULFOTRANSFERASE-RXN]
GO:0047686
name: arylsulfate sulfotransferase activity
def: "Catalysis of the reaction: an aryl sulfate + a phenol = a phenol + an aryl sulfate." [EC:2.8.2.22, MetaCyc:ARYLSULFATE-SULFOTRANSFERASE-RXN]
GO:0047687
name: ascorbate 2,3-dioxygenase activity
def: "OBSOLETE. Catalysis of the reaction: L-ascorbate + H(2)O + O(2) = L-threonate + 2 H(+) + oxalate." [EC:1.13.11.13, RHEA:21787]
GO:0047688
name: aspartate 4-decarboxylase activity
def: "Catalysis of the reaction: L-aspartate = L-alanine + CO2." [EC:4.1.1.12, MetaCyc:ASPARTATE-4-DECARBOXYLASE-RXN]
GO:0047689
name: aspartate racemase activity
def: "Catalysis of the reaction: L-aspartate = D-aspartate." [EC:5.1.1.13, RHEA:14976]
GO:0047690
name: aspartyltransferase activity
def: "Catalysis of the reaction: L-asparagine + H(+) + hydroxylamine = beta-L-aspartylhydroxamate + NH(4)(+)." [EC:2.3.2.7, RHEA:11255]
GO:0047691
name: aspulvinone dimethylallyltransferase activity
def: "Catalysis of the reaction: aspulvinone E + 2 dimethylallyl diphosphate = aspulvinone H + 2 diphosphate." [EC:2.5.1.35, RHEA:13812]
GO:0047692
name: ATP deaminase activity
def: "Catalysis of the reaction: ATP + H2O = ITP + NH3." [EC:3.5.4.18, MetaCyc:ATP-DEAMINASE-RXN]
GO:0047693
name: ATP diphosphatase activity
def: "Catalysis of the reaction: ATP + H2O = AMP + diphosphate." [EC:3.6.1.8, MetaCyc:ATP-PYROPHOSPHATASE-RXN]
GO:0047694
name: barbiturase activity
def: "Catalysis of the reaction: barbiturate + H2O = malonate + urea." [EC:3.5.2.1, MetaCyc:BARBITURASE-RXN]
GO:0047695
name: benzoin aldolase activity
def: "Catalysis of the reaction: benzoin = 2 benzaldehyde." [EC:4.1.2.38, RHEA:21463]
GO:0047696
name: beta-adrenergic receptor kinase activity
def: "Catalysis of the reaction: ATP + beta-adrenergic receptor = ADP + phospho-beta-adrenergic receptor." [EC:2.7.11.15, MetaCyc:BETA-ADRENERGIC-RECEPTOR-KINASE-RXN]
GO:0047697
name: beta-alanopine dehydrogenase activity
def: "Catalysis of the reaction: (R)-beta-alanopine + H(2)O + NAD(+) = beta-alanine + H(+) + NADH + pyruvate." [EC:1.5.1.26, RHEA:21687]
GO:0047698
name: beta-alanyl-CoA ammonia-lyase activity
def: "Catalysis of the reaction: beta-alanyl-CoA = acryloyl-CoA + NH3." [EC:4.3.1.6, MetaCyc:BETA-ALANYL-COA-AMMONIA-LYASE-RXN]
GO:0047699
name: beta-diketone hydrolase activity
def: "Catalysis of the reaction: H(2)O + nonane-4,6-dione = butanoate + H(+) + pentan-2-one." [EC:3.7.1.7, RHEA:11911]
GO:0047700
name: beta-glucoside kinase activity
def: "Catalysis of the reaction: ATP + cellobiose = ADP + 6-phospho-beta-D-glucosyl-(1,4)-D-glucose." [EC:2.7.1.85, MetaCyc:BETA-GLUCOSIDE-KINASE-RXN]
GO:0047701
name: beta-L-arabinosidase activity
def: "Catalysis of the reaction: H2O + a beta-L-arabinoside = L-arabinose + an alcohol." [EC:3.2.1.88, MetaCyc:BETA-L-ARABINOSIDASE-RXN]
GO:0047702
name: beta-lysine 5,6-aminomutase activity
def: "Catalysis of the reaction: (3S)-3,6-diaminohexanoate = (3S,5S)-3,5-diaminohexanoate." [EC:5.4.3.3, RHEA:21739]
GO:0047703
name: beta-nitroacrylate reductase activity
def: "Catalysis of the reaction: 3-nitropropanoate + NADP(+) = 3-nitroacrylate + H(+) + NADPH." [EC:1.3.1.16, RHEA:23895]
GO:0047704
name: bile-salt sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + taurolithocholate = adenosine 3',5'-bisphosphate + taurolithocholate sulfate." [EC:2.8.2.14, MetaCyc:BILE-SALT-SULFOTRANSFERASE-RXN]
GO:0047705
name: bilirubin oxidase activity
def: "Catalysis of the reaction: 2 bilirubin + O(2) = 2 biliverdin + 2 H(2)O." [EC:1.3.3.5, RHEA:20983]
GO:0047706
name: biochanin-A reductase activity
def: "Catalysis of the reaction: dihydrobiochanin A + NADP(+) = biochanin A + H(+) + NADPH." [EC:1.3.1.46, RHEA:12820]
GO:0047707
name: biotin-CoA ligase activity
def: "Catalysis of the reaction: ATP + biotin + CoA = AMP + diphosphate + biotinyl-CoA." [EC:6.2.1.11, MetaCyc:BIOTIN--COA-LIGASE-RXN]
GO:0047708
name: biotinidase activity
def: "Catalysis of the reaction: biotin amide + H2O = biotin + NH3." [EC:3.5.1.12, MetaCyc:BIOTINIDASE-RXN]
GO:0047709
name: bis(2-ethylhexyl)phthalate esterase activity
def: "Catalysis of the reaction: bis(2-ethylhexyl)phthalate + H(2)O = 2-ethylhexan-1-ol + 2-ethylhexyl phthalate + H(+)." [EC:3.1.1.60, RHEA:15532]
GO:0047710
name: bis(5'-adenosyl)-triphosphatase activity
def: "Catalysis of the reaction: P(1),P(3)-bis(5'-adenosyl) triphosphate + H(2)O = ADP + AMP + 2 H(+)." [EC:3.6.1.29, RHEA:13896]
GO:0047711
name: blasticidin-S deaminase activity
def: "Catalysis of the reaction: blasticidin S + H2O = deaminohydroxyblasticidin S + NH3." [EC:3.5.4.23, MetaCyc:BLASTICIDIN-S-DEAMINASE-RXN]
GO:0047712
name: Cypridina-luciferin 2-monooxygenase activity
def: "Catalysis of the reaction: Cypridina luciferin + O2 = oxidized Cypridina luciferin + CO2 + light." [EC:1.13.12.6, MetaCyc:CYPRIDINA-LUCIFERIN-2-MONOOXYGENASE-RXN]
GO:0047713
name: galactitol 2-dehydrogenase activity
def: "Catalysis of the reaction: galactitol + NAD(+) = D-tagatose + H(+) + NADH." [EC:1.1.1.16, RHEA:20688]
GO:0047714
name: galactolipase activity
def: "Catalysis of the reaction: 1,2-diacyl-3-beta-D-galactosyl-sn-glycerol + 2 H2O = 3-beta-D-galactosyl-sn-glycerol + 2 carboxylates." [EC:3.1.1.26, MetaCyc:GALACTOLIPASE-RXN]
GO:0047715
name: hypotaurocyamine kinase activity
def: "Catalysis of the reaction: ATP + hypotaurocyamine = N(omega)-phosphohypotaurocyamine + ADP + 2 H(+)." [EC:2.7.3.6, RHEA:24011]
GO:0047716
name: imidazole N-acetyltransferase activity
def: "Catalysis of the reaction: 1H-imidazole + acetyl-CoA = N-acetylimidazole + CoA." [EC:2.3.1.2, RHEA:15816]
GO:0047717
name: imidazoleacetate 4-monooxygenase activity
def: "Catalysis of the reaction: H(+) + imidazol-4-ylacetate + NADH + O(2) = 5-hydroxyimidazole-4-acetate + H(2)O + NAD(+)." [EC:1.14.13.5, RHEA:19428]
GO:0047718
name: indanol dehydrogenase activity
def: "Catalysis of the reaction: indan-1-ol + NAD(P)+ = indanone + NAD(P)H + H+." [EC:1.1.1.112, MetaCyc:INDANOL-DEHYDROGENASE-RXN]
GO:0047719
name: indole 2,3-dioxygenase activity
def: "Catalysis of the reaction: indole + O(2) = 2-formamidobenzaldehyde." [EC:1.13.11.17, RHEA:11215]
GO:0047720
name: indoleacetaldoxime dehydratase activity
def: "Catalysis of the reaction: (indol-3-yl)acetaldehyde oxime = (indol-3-yl)acetonitrile + H(2)O." [EC:4.99.1.6, RHEA:23159]
GO:0047721
name: indoleacetate-lysine synthetase activity
def: "Catalysis of the reaction: (indol-3-yl)acetate + L-lysine + ATP = N(6)-[(indole-3-yl)acetyl]-L-lysine + ADP + 2 H(+) + phosphate." [EC:6.3.2.20, RHEA:14860]
GO:0047722
name: indolelactate dehydrogenase activity
def: "Catalysis of the reaction: 3-(indol-3-yl)lactate + NAD(+) = 3-(indol-3-yl)pyruvate + H(+) + NADH." [EC:1.1.1.110, RHEA:20136]
GO:0047723
name: inosinate nucleosidase activity
def: "Catalysis of the reaction: H(2)O + IMP = D-ribose 5-phosphate + hypoxanthine." [EC:3.2.2.12, RHEA:20472]
GO:0047724
name: inosine nucleosidase activity
def: "Catalysis of the reaction: inosine + H2O = D-ribose + hypoxanthine." [EC:3.2.2.2, MetaCyc:INOSINE-NUCLEOSIDASE-RXN]
GO:0047725
name: inulosucrase activity
def: "Catalysis of the reaction: sucrose + 2,1-beta-D-fructosyl(n) = glucose + 2,1-beta-D-fructosyl(n+1)." [EC:2.4.1.9, MetaCyc:INULOSUCRASE-RXN]
GO:0047726
name: iron-cytochrome-c reductase activity
def: "Catalysis of the reaction: ferrocytochrome c + Fe3+ = ferricytochrome c + Fe2+." [EC:1.9.99.1, MetaCyc:IRON--CYTOCHROME-C-REDUCTASE-RXN]
GO:0047727
name: isobutyryl-CoA mutase activity
def: "Catalysis of the reaction: isobutyryl-CoA = butanoyl-CoA." [EC:5.4.99.13, RHEA:13144]
GO:0047728
name: carnitine 3-dehydrogenase activity
def: "Catalysis of the reaction: carnitine + NAD(+) = 3-dehydrocarnitine + H(+) + NADH." [EC:1.1.1.108, RHEA:19268]
GO:0047729
name: carnitine decarboxylase activity
def: "Catalysis of the reaction: carnitine + H(+) = 2-methylcholine + CO(2)." [EC:4.1.1.42, RHEA:21579]
GO:0047730
name: carnosine synthase activity
def: "Catalysis of the reaction: ATP + L-histidine + beta-alanine = AMP + diphosphate + carnosine." [EC:6.3.2.11, MetaCyc:CARNOSINE-SYNTHASE-RXN]
GO:0047731
name: catechol oxidase (dimerizing) activity
def: "Catalysis of the reaction: 4 catechol + 3 O2 = 2 dibenzo[1,4]dioxin-2,3-dione + 6 H2O." [EC:1.1.3.14, MetaCyc:CATECHOL-OXIDASE-DIMERIZING-RXN]
GO:0047732
name: CDP-abequose epimerase activity
def: "Catalysis of the reaction: CDP-3,6-dideoxy-D-glucose = CDP-3,6-dideoxy-D-mannose." [EC:5.1.3.10, RHEA:21659]
GO:0047733
name: CDP-glucose 4,6-dehydratase activity
def: "Catalysis of the reaction: CDP-D-glucose = CDP-4-dehydro-6-deoxy-D-glucose + H(2)O." [EC:4.2.1.45, RHEA:17156]
GO:0047734
name: CDP-glycerol diphosphatase activity
def: "Catalysis of the reaction: CDP-glycerol + H(2)O = sn-glycerol 3-phosphate + CMP + 2 H(+)." [EC:3.6.1.16, RHEA:21695]
GO:0047735
name: cellobiose dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: cellobiose + acceptor = cellobiono-1,5-lactone + reduced acceptor." [EC:1.1.99.18, MetaCyc:CELLOBIOSE-DEHYDROGENASE-ACCEPTOR-RXN]
GO:0047736
name: cellobiose epimerase activity
def: "Catalysis of the reaction: cellobiose = D-glucosyl-D-mannose." [EC:5.1.3.11, MetaCyc:CELLOBIOSE-EPIMERASE-RXN]
GO:0047738
name: cellobiose phosphorylase activity
def: "Catalysis of the reaction: cellobiose + phosphate = alpha-D-glucose 1-phosphate + D-glucose." [EC:2.4.1.20, MetaCyc:CELLOBIOSE-PHOSPHORYLASE-RXN]
GO:0047739
name: cephalosporin-C deacetylase activity
def: "Catalysis of the reaction: cephalosporin C + H(2)O = acetate + deacetylcephalosporin C + H(+)." [EC:3.1.1.41, RHEA:22599]
GO:0047740
name: cephalosporin-C transaminase activity
def: "Catalysis of the reaction: (7R)-7-(5-carboxylato-5-oxopentanamido)deacetylcephalosporanate + D-glutamate = 2-oxoglutarate + cephalosporin C." [EC:2.6.1.74, RHEA:14556]
GO:0047741
name: cetraxate benzylesterase activity
def: "Catalysis of the reaction: benzyl cetraxate + H(2)O = benzyl alcohol + cetraxate + H(+)." [EC:3.1.1.70, RHEA:23463]
GO:0047742
name: chenodeoxycholoyltaurine hydrolase activity
def: "Catalysis of the reaction: chenodeoxycholoyltaurine + H2O = chenodeoxycholate + taurine." [EC:3.5.1.74, MetaCyc:CHENODEOXYCHOLOYLTAURINE-HYDROLASE-RXN]
GO:0047743
name: chlordecone reductase activity
def: "Catalysis of the reaction: chlordecone alcohol + NADP(+) = chlordecone + H(+) + NADPH." [EC:1.1.1.225, RHEA:14404]
GO:0047744
name: chloridazon-catechol dioxygenase activity
def: "Catalysis of the reaction: 5-amino-4-chloro-2-(2,3-dihydroxyphenyl)pyridazin-3(2H)-one + O(2) = 5-amino-4-chloro-2-(2-hydroxymuconoyl)pyridazin-3(2H)-one + 2 H(+)." [EC:1.13.11.36, RHEA:20452]
GO:0047745
name: chlorogenate hydrolase activity
def: "Catalysis of the reaction: chlorogenate + H(2)O = (-)-quinate + cis-caffeate + H(+)." [EC:3.1.1.42, RHEA:20692]
GO:0047746
name: chlorophyllase activity
def: "Catalysis of the reaction: chlorophyll + H2O = phytol + chlorophyllide." [EC:3.1.1.14, MetaCyc:CHLOROPHYLLASE-RXN]
GO:0047747
name: cholate-CoA ligase activity
def: "Catalysis of the reactions: (1) ATP + cholate + CoA = AMP + diphosphate + choloyl-CoA and (2) ATP + (25R)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-26-oate + CoA = AMP + diphosphate + (25R)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholestanoyl-CoA." [EC:6.2.1.7, MetaCyc:CHOLATE--COA-LIGASE-RXN]
GO:0047748
name: cholestanetetraol 26-dehydrogenase activity
def: "Catalysis of the reaction: 5-beta-cholestane-3-alpha,7-alpha,12-alpha,26-tetraol + NAD+ = 3-alpha,7-alpha,12-alpha-trihydroxy-5-beta-cholestan-26-al + NADH." [EC:1.1.1.161, MetaCyc:CHOLESTANETETRAOL-26-DEHYDROGENASE-RXN]
GO:0047749
name: cholestanetriol 26-monooxygenase activity
def: "Catalysis of the reaction: 5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol + NADPH + O2 = 5-beta-cholestane-3-alpha,7-alpha,12-alpha,26-tetraol + NADP+ + H2O." [EC:1.14.13.15, MetaCyc:CHOLESTANETRIOL-26-MONOOXYGENASE-RXN]
GO:0047750
name: cholestenol delta-isomerase activity
def: "Catalysis of the reaction: 5-alpha-cholest-7-en-3-beta-ol = 5-alpha-cholest-8-en-3-beta-ol." [EC:5.3.3.5, MetaCyc:CHOLESTENOL-DELTA-ISOMERASE-RXN]
GO:0047751
name: cholestenone 5-alpha-reductase activity
def: "Catalysis of the reaction: 5alpha-cholestan-3-one + NADP(+) = cholest-4-en-3-one + H(+) + NADPH." [EC:1.3.1.22, RHEA:24555]
GO:0047753
name: choline-sulfatase activity
def: "Catalysis of the reaction: choline sulfate + H(2)O = choline + H(+) + sulfate." [EC:3.1.6.6, RHEA:20823]
GO:0047754
name: choline sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + choline = adenosine 3',5'-diphosphate + choline sulfate + H(+)." [EC:2.8.2.6, RHEA:21987]
GO:0047755
name: isocitrate epimerase activity
def: "Catalysis of the reaction: D-threo-isocitrate = D-erythro-isocitrate." [EC:5.1.2.6, RHEA:10823]
GO:0047756
name: chondroitin 4-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + chondroitin = adenosine 3',5'-bisphosphate + chondroitin 4'-sulfate." [EC:2.8.2.5, MetaCyc:CHONDROITIN-4-SULFOTRANSFERASE-RXN]
GO:0047757
name: chondroitin-glucuronate 5-epimerase activity
def: "Catalysis of the reaction: chondroitin D-glucuronate = dermatan L-iduronate." [EC:5.1.3.19, MetaCyc:CHONDROITIN-GLUCURONATE-5-EPIMERASE-RXN]
GO:0047758
name: ATP:2-methylpropanoate phosphotransferase activity
def: "Catalysis of the reaction: 2-methylpropanoate + ATP = 2-methylpropanoyl phosphate + ADP + H(+)." [EC:2.7.2.14, RHEA:24159]
GO:0047759
name: butanal dehydrogenase activity
def: "Catalysis of the reaction: butanal + CoA + NAD(P)+ = butanoyl-CoA + NAD(P)H + H+." [EC:1.2.1.57, MetaCyc:BUTANAL-DEHYDROGENASE-RXN]
GO:0047760
name: butyrate-CoA ligase activity
def: "Catalysis of the reaction: ATP + an acid + CoA = AMP + diphosphate + an acyl-CoA." [EC:6.2.1.2, MetaCyc:BUTYRATE--COA-LIGASE-RXN]
GO:0047761
name: butyrate kinase activity
def: "Catalysis of the reaction: ATP + butanoate = ADP + butanoyl phosphate + H(+)." [EC:2.7.2.7, RHEA:13588]
GO:0047762
name: caffeate 3,4-dioxygenase activity
def: "Catalysis of the reaction: trans-caffeate + O(2) = 3-(2-carboxyethenyl)-cis,cis-muconate + 2 H(+)." [EC:1.13.11.22, RHEA:22219]
GO:0047763
name: caffeate O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3,4-dihydroxy-trans-cinnamate = S-adenosyl-L-homocysteine + 3-methoxy-4-hydroxy-trans-cinnamate." [EC:2.1.1.68, MetaCyc:CAFFEATE-O-METHYLTRANSFERASE-RXN]
GO:0047764
name: caldesmon kinase activity
def: "Catalysis of the reaction: ATP + caldesmon = ADP + caldesmon phosphate." [GOC:curators]
GO:0047765
name: caldesmon-phosphatase activity
def: "Catalysis of the reaction: caldesmon phosphate + H2O = caldesmon + phosphate." [EC:3.1.3.55, MetaCyc:CALDESMON-PHOSPHATASE-RXN]
GO:0047766
name: carbamoyl-serine ammonia-lyase activity
def: "Catalysis of the reaction: O-carbamoyl-L-serine + H(2)O + H(+) = CO(2) + 2 NH(4)(+) + pyruvate." [EC:4.3.1.13, RHEA:15448]
GO:0047768
name: carboxy-cis,cis-muconate cyclase activity
def: "Catalysis of the reaction: 3-carboxy-2,5-dihydro-5-oxofuran-2-acetate = 3-carboxy-cis,cis-muconate." [EC:5.5.1.5, RHEA:14980]
GO:0047769
name: arogenate dehydratase activity
def: "Catalysis of the reaction: L-arogenate = L-phenylalanine + H2O + CO2." [EC:4.2.1.91, MetaCyc:CARBOXYCYCLOHEXADIENYL-DEHYDRATASE-RXN]
GO:0047770
name: carboxylate reductase activity
def: "Catalysis of the reaction: an aldehyde + acceptor + H2O = a carboxylate + reduced acceptor." [EC:1.2.99.6, MetaCyc:CARBOXYLATE-REDUCTASE-RXN]
GO:0047771
name: carboxymethylhydantoinase activity
def: "Catalysis of the reaction: L-5-carboxymethylhydantoin + H(2)O = N-carbamoyl-L-aspartate + H(+)." [EC:3.5.2.4, RHEA:12031]
GO:0047772
name: carboxymethyloxysuccinate lyase activity
def: "Catalysis of the reaction: carboxymethoxysuccinate = fumarate + glycolate." [EC:4.2.99.12, RHEA:12339]
GO:0047773
name: carnitinamidase activity
def: "Catalysis of the reaction: (R)-carnitinamide + H(2)O = (R)-carnitine + NH(4)(+)." [EC:3.5.1.73, RHEA:17540]
GO:0047774
name: cis-2-enoyl-CoA reductase (NADPH) activity
def: "Catalysis of the reaction: acyl-CoA + NADP+ = cis-2,3-dehydroacyl-CoA + NADPH." [EC:1.3.1.37, MetaCyc:CIS-2-ENOYL-COA-REDUCTASE-NADPH-RXN]
GO:0047775
name: citramalate CoA-transferase activity
def: "Catalysis of the reaction: acetyl-CoA + citramalate = acetate + (3S)-citramalyl-CoA." [EC:2.8.3.11, MetaCyc:CITRAMALATE-COA-TRANSFERASE-RXN]
GO:0047776
name: citramalate lyase activity
def: "Catalysis of the reaction: S-citramalate = acetate + pyruvate." [EC:4.1.3.22, RHEA:15548]
GO:0047777
name: (3S)-citramalyl-CoA lyase activity
def: "Catalysis of the reaction: (3S)-citramalyl-CoA = acetyl-CoA + pyruvate." [EC:4.1.3.25, RHEA:22615]
GO:0047778
name: [citrate-(pro-3S)-lyase] thiolesterase activity
def: "Catalysis of the reaction: [citrate (pro-3S)-lyase](acetyl form) + H2O = [citrate (pro-3S)-lyase](thiol form) + acetate." [EC:3.1.2.16, MetaCyc:CITRATE-PRO-3S-LYASE-THIOLESTERASE-RXN]
GO:0047779
name: citrate-CoA ligase activity
def: "Catalysis of the reaction: ATP + citrate + CoA = (3S)-citryl-CoA + ADP + H(+) + phosphate." [EC:6.2.1.18, RHEA:21475]
GO:0047780
name: citrate dehydratase activity
def: "Catalysis of the reaction: citrate = cis-aconitate + H2O." [EC:4.2.1.4]
GO:0047781
name: citrullinase activity
def: "Catalysis of the reaction: H2O + citrulline = NH3 + CO2 + L-ornithine." [EC:3.5.1.20, MetaCyc:CITRULLINASE-RXN]
GO:0047782
name: coniferin beta-glucosidase activity
def: "Catalysis of the reaction: H2O + coniferin = D-glucose + coniferol." [EC:3.2.1.126, MetaCyc:CONIFERIN-BETA-GLUCOSIDASE-RXN]
GO:0047783
name: corticosterone 18-monooxygenase activity
def: "Catalysis of the reaction: corticosterone + reduced adrenal ferredoxin + O2 = 18-hydroxycorticosterone + oxidized adrenal ferredoxin + H2O." [EC:1.14.15.5, MetaCyc:CORTICOSTERONE-18-MONOOXYGENASE-RXN]
GO:0047784
name: cortisol O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + cortisol = CoA + cortisol 21-acetate." [EC:2.3.1.27, RHEA:17076]
GO:0047785
name: cortisol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + cortisol = adenosine 3',5'-diphosphate + cortisol 21-sulfate + H(+)." [EC:2.8.2.18, RHEA:11887]
GO:0047786
name: cortisone alpha-reductase activity
def: "Catalysis of the reaction: 4,5alpha-dihydrocortisone + NADP(+) = cortisone + H(+) + NADPH." [EC:1.3.1.4, RHEA:17984]
GO:0047787
name: delta4-3-oxosteroid 5beta-reductase activity
def: "Catalysis of the reactions: (1) 5beta-cholestan-3-one + NADP+ = cholest-4-en-3-one + NADPH + H+ and (2) 17,21-dihydroxy-5beta-pregnane-3,11,20-trione + NADP+ = cortisone + NADPH + H+." [EC:1.3.1.3, MetaCyc:CORTISONE-BETA-REDUCTASE-RXN]
GO:0047788
name: 2-coumarate reductase activity
def: "Catalysis of the reaction: 3-(2-hydroxyphenyl)propanoate + NAD(+) = trans-2-coumarate + H(+) + NADH." [EC:1.3.1.11, RHEA:21447]
GO:0047789
name: creatininase activity
def: "Catalysis of the reaction: creatinine + H(2)O = creatine." [EC:3.5.2.10, RHEA:14536]
GO:0047790
name: creatinine deaminase activity
def: "Catalysis of the reaction: creatinine + H2O = N-methylhydantoin + NH3." [EC:3.5.4.21, MetaCyc:CREATININE-DEAMINASE-RXN]
GO:0047791
name: cucurbitacin delta23-reductase activity
def: "Catalysis of the reaction: 23,24-dihydrocucurbitacin + NAD(P)+ = cucurbitacin + NAD(P)H + H+." [EC:1.3.1.5, MetaCyc:CUCURBITACIN-DELTA-23-REDUCTASE-RXN]
GO:0047792
name: cyanohydrin beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + (S)-4-hydroxymandelonitrile = UDP + (S)-4-hydroxy-mandelonitrile beta-D-glucoside." [EC:2.4.1.85, MetaCyc:CYANOHYDRIN-BETA-GLUCOSYLTRANSFERASE-RXN]
GO:0047793
name: cycloeucalenol cycloisomerase activity
def: "Catalysis of the reaction: cycloeucalenol = obtusifoliol." [EC:5.5.1.9, RHEA:22803]
GO:0047794
name: cyclohexadienyl dehydrogenase activity
def: "Catalysis of the reaction: L-arogenate + NAD+ = L-tyrosine + NADH + CO2." [EC:1.3.1.43, MetaCyc:CYCLOHEXADIENYL-DEHYDROGENASE-RXN]
GO:0047795
name: cyclohexane-1,2-diol dehydrogenase activity
def: "Catalysis of the reaction: trans-cyclohexane-1,2-diol + NAD+ = 2-hydroxycyclohexan-1-one + NADH." [EC:1.1.1.174, MetaCyc:CYCLOHEXANE-12-DIOL-DEHYDROGENASE-RXN]
GO:0047796
name: cyclohexane-1,3-dione hydrolase activity
def: "Catalysis of the reaction: cyclohexane-1,3-dione + H(2)O = 5-oxohexanoate + H(+)." [EC:3.7.1.10, RHEA:16476]
GO:0047797
name: cyclohexanone dehydrogenase activity
def: "Catalysis of the reaction: A + cyclohexanone = AH(2) + cyclohex-2-enone." [EC:1.3.99.14, RHEA:21783]
GO:0047798
name: cyclomaltodextrinase activity
def: "Catalysis of the reaction: H2O + cyclomaltodextrin = linear maltodextrin." [EC:3.2.1.54, MetaCyc:CYCLOMALTODEXTRINASE-RXN]
GO:0047799
name: cyclopentanone monooxygenase activity
def: "Catalysis of the reaction: cyclopentanone + H(+) + NADPH + O(2) = 5-valerolactone + H(2)O + NADP(+)." [EC:1.14.13.16, RHEA:15740]
GO:0047800
name: cysteamine dioxygenase activity
def: "Catalysis of the reaction: cysteamine + O(2) = H(+) + hypotaurine." [EC:1.13.11.19, RHEA:14412]
GO:0047801
name: L-cysteine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-cysteine + 2-oxoglutarate = mercaptopyruvate + L-glutamate." [EC:2.6.1.3, MetaCyc:CYSTEINE-AMINOTRANSFERASE-RXN]
GO:0047802
name: cysteine-conjugate transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + S-(4-bromophenyl)-L-cysteine = (4-bromophenylsulfanyl)pyruvate + L-glutamate." [EC:2.6.1.75, RHEA:13488]
GO:0047803
name: cysteine lyase activity
def: "Catalysis of the reaction: L-cysteine + sulfite = L-cysteate + sulfide." [EC:4.4.1.10, MetaCyc:CYSTEINE-LYASE-RXN]
GO:0047804
name: cysteine-S-conjugate beta-lyase activity
def: "Catalysis of the reaction: RS-CH2-CH(NH3+)COO- = RSH + NH3 + pyruvate." [EC:4.4.1.13, MetaCyc:CYSTEINE-S-CONJUGATE-BETA-LYASE-RXN]
GO:0047805
name: cytidylate cyclase activity
def: "Catalysis of the reaction: CTP = 3',5'-cyclic CMP + diphosphate + H(+)." [EC:4.6.1.6, RHEA:14740]
GO:0047806
name: cytochrome-c3 hydrogenase activity
def: "Catalysis of the reaction: 2 H2 + ferricytochrome c3 = 4 H+ + ferrocytochrome c3." [EC:1.12.2.1, MetaCyc:CYTOCHROME-C3-HYDROGENASE-RXN]
GO:0047807
name: cytokinin 7-beta-glucosyltransferase activity
def: "Catalysis of the reaction: 6-alkylaminopurine + UDP-D-glucose = 6-alkylamino-7-beta-D-glucosylpurine + H+ + UDP. This reaction is an N-glucosylation event." [EC:2.4.1.118, MetaCyc:CYTOKININ-7-BETA-GLUCOSYLTRANSFERASE-RXN]
GO:0047808
name: D(-)-tartrate dehydratase activity
def: "Catalysis of the reaction: D-tartrate = H(2)O + oxaloacetate." [EC:4.2.1.81, RHEA:18292]
GO:0047809
name: D-2-hydroxy-acid dehydrogenase activity
def: "Catalysis of the reaction: (R)-lactate + A = AH(2) + pyruvate." [EC:1.1.99.6, RHEA:15092]
GO:0047810
name: D-alanine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: D-alanine + 2-oxoglutarate = pyruvate + D-glutamate." [EC:2.6.1.21, MetaCyc:D-ALANINE-AMINOTRANSFERASE-RXN]
GO:0047811
name: D-alanine gamma-glutamyltransferase activity
def: "Catalysis of the reaction: D-alanine + L-glutamine = gamma-L-glutamyl-D-alanine + NH(4)(+)." [EC:2.3.2.14, RHEA:23559]
GO:0047812
name: D-amino-acid N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + a D-amino acid = CoA + an N-acetyl-D-amino-acid." [EC:2.3.1.36, MetaCyc:D-AMINO-ACID-N-ACETYLTRANSFERASE-RXN]
GO:0047813
name: D-arabinitol 4-dehydrogenase activity
def: "Catalysis of the reaction: D-arabinitol + NAD+ = D-xylulose + NADH." [EC:1.1.1.11, MetaCyc:D-ARABINITOL-4-DEHYDROGENASE-RXN]
GO:0047814
name: D-arabinokinase activity
def: "Catalysis of the reaction: D-arabinose + ATP = D-arabinose 5-phosphate + ADP." [EC:2.7.1.54, RHEA:24591]
GO:0047815
name: D-arabinonolactonase activity
def: "Catalysis of the reaction: D-arabinono-1,4-lactone + H(2)O = D-arabinonate + H(+)." [EC:3.1.1.30, RHEA:23111]
GO:0047816
name: D-arabinose 1-dehydrogenase (NAD) activity
def: "Catalysis of the reaction: D-arabinose + NAD+ = D-arabinono-1,4-lactone + NADH." [EC:1.1.1.116, MetaCyc:D-ARABINOSE-1-DEHYDROGENASE-RXN]
GO:0047817
name: D-arginase activity
def: "Catalysis of the reaction: D-arginine + H(2)O = D-ornithine + urea." [EC:3.5.3.10, RHEA:12904]
GO:0047818
name: D-fuconate dehydratase activity
def: "Catalysis of the reaction: D-fuconate = 2-dehydro-3-deoxy-D-fuconate + H(2)O." [EC:4.2.1.67, RHEA:12952]
GO:0047819
name: D-glutamate(D-aspartate) oxidase activity
def: "Catalysis of the reaction: D-glutamate + H2O + O2 = 2-oxoglutarate + NH3 + H2O2, and D-aspartate + H2O + O2 = oxaloacetate + NH3 + H2O2." [EC:1.4.3.15, MetaCyc:D-GLUTAMATED-ASPARTATE-OXIDASE-RXN]
GO:0047820
name: D-glutamate cyclase activity
def: "Catalysis of the reaction: D-glutamate = 5-oxo-D-proline + H(2)O." [EC:4.2.1.48, RHEA:22363]
GO:0047821
name: D-glutamate oxidase activity
def: "Catalysis of the reaction: D-glutamate + H2O + O2 = 2-oxoglutarate + NH3 + H2O2." [EC:1.4.3.7, MetaCyc:D-GLUTAMATE-OXIDASE-RXN]
GO:0047822
name: hypotaurine dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + hypotaurine + NAD(+) = H(+) + NADH + taurine." [EC:1.8.1.3, RHEA:17388]
GO:0047823
name: D-glutamyltransferase activity
def: "Catalysis of the reaction: L(or D)-glutamine + D-glutamyl-peptide = NH3 + 5-glutamyl-D-glutamyl-peptide." [EC:2.3.2.1, MetaCyc:D-GLUTAMYLTRANSFERASE-RXN]
GO:0047824
name: D-iditol 2-dehydrogenase activity
def: "Catalysis of the reaction: D-iditol + NAD+ = D-sorbose + NADH." [EC:1.1.1.15, MetaCyc:D-IDITOL-2-DEHYDROGENASE-RXN]
GO:0047825
name: D-lactate-2-sulfatase activity
def: "Catalysis of the reaction: (R)-2-O-sulfolactate + H(2)O = (R)-lactate + H(+) + sulfate." [EC:3.1.6.17, RHEA:20340]
GO:0047826
name: D-lysine 5,6-aminomutase activity
def: "Catalysis of the reaction: D-lysine = 2,5-diaminohexanoate." [EC:5.4.3.4, RHEA:18244]
GO:0047827
name: D-lysopine dehydrogenase activity
def: "Catalysis of the reaction: D-lysopine + H(2)O + NADP(+) = L-lysine + H(+) + NADPH + pyruvate." [EC:1.5.1.16, RHEA:17628]
GO:0047828
name: D-lyxose ketol-isomerase activity
def: "Catalysis of the reaction: D-lyxose = D-xylulose." [EC:5.3.1.15, RHEA:14204]
GO:0047829
name: D-nopaline dehydrogenase activity
def: "Catalysis of the reaction: N2-(D-1,3-dicarboxypropyl)-L-arginine + NADP+ + H2O = L-arginine + 2-oxoglutarate + NADPH." [EC:1.5.1.19, MetaCyc:D-NOPALINE-DEHYDROGENASE-RXN]
GO:0047830
name: D-octopine dehydrogenase activity
def: "Catalysis of the reaction: N2-(D-1-carboxyethyl)-L-arginine + NAD+ + H2O = L-arginine + pyruvate + NADH." [EC:1.5.1.11, MetaCyc:D-OCTOPINE-DEHYDROGENASE-RXN]
GO:0047831
name: D-ornithine 4,5-aminomutase activity
def: "Catalysis of the reaction: D-ornithine = (2R,4S)-2,4-diaminopentanoate." [EC:5.4.3.5, RHEA:14896]
GO:0047832
name: D-pinitol dehydrogenase activity
def: "Catalysis of the reaction: 5D-5-O-methyl-chiro-inositol + NADP(+) = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + H(+) + NADPH." [EC:1.1.1.142, RHEA:20440]
GO:0047833
name: D-sorbitol dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: D-sorbitol + acceptor = L-sorbose + reduced acceptor." [EC:1.1.99.21, MetaCyc:D-SORBITOL-DEHYDROGENASE-RXN]
GO:0047834
name: D-threo-aldose 1-dehydrogenase activity
def: "Catalysis of the reaction: a D-threo-aldose + NAD+ = a D-threo-aldono-1,5-lactone + NADH." [EC:1.1.1.122, MetaCyc:D-THREO-ALDOSE-1-DEHYDROGENASE-RXN]
GO:0047835
name: D-tryptophan N-acetyltransferase activity
def: "Catalysis of the reaction: D-tryptophan + acetyl-CoA = N-acetyl-D-tryptophan + CoA + H(+)." [EC:2.3.1.34, RHEA:10063]
GO:0047836
name: D-tryptophan N-malonyltransferase activity
def: "Catalysis of the reaction: D-tryptophan + malonyl-CoA = N(2)-malonyl-D-tryptophan + CoA + H(+)." [EC:2.3.1.112, RHEA:23323]
GO:0047837
name: D-xylose 1-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: D-xylose + NADP(+) = D-xylono-1,5-lactone + H(+) + NADPH." [EC:1.1.1.179, RHEA:22003]
GO:0047838
name: D-xylose 1-dehydrogenase (NAD) activity
def: "Catalysis of the reaction: D-xylose + NAD+ = D-xylonolactone + NADH." [EC:1.1.1.175, MetaCyc:D-XYLOSE-1-DEHYDROGENASE-RXN]
GO:0047839
name: dATP(dGTP)-DNA purinetransferase activity
def: "Catalysis of the reaction: dATP + depurinated DNA = ribose triphosphate + DNA." [EC:2.6.99.1, MetaCyc:DATPDGTP--DNA-PURINE-TRANSFERASE-RXN]
GO:0047840
name: dCTP diphosphatase activity
def: "Catalysis of the reaction: dCTP + H2O = dCMP + diphosphate." [EC:3.6.1.12, MetaCyc:DCTP-PYROPHOSPHATASE-RXN]
GO:0047841
name: dehydrogluconokinase activity
def: "Catalysis of the reaction: 2-dehydro-D-gluconate + ATP = 6-phospho-2-dehydro-D-gluconate + ADP + 2 H(+)." [EC:2.7.1.13, RHEA:10791]
GO:0047842
name: dehydro-L-gulonate decarboxylase activity
def: "Catalysis of the reaction: 3-dehydro-L-gulonate + H(+) = L-xylulose + CO(2)." [EC:4.1.1.34, RHEA:11087]
GO:0047843
name: dehydrogluconate dehydrogenase activity
def: "Catalysis of the reaction: 2-dehydro-D-gluconate + A = 2,5-didehydro-D-gluconate + AH(2)." [EC:1.1.99.4, RHEA:12371]
GO:0047844
name: deoxycytidine deaminase activity
def: "Catalysis of the reaction: deoxycytidine + H2O = deoxyuridine + NH3." [EC:3.5.4.14, MetaCyc:DEOXYCYTIDINE-DEAMINASE-RXN]
GO:0047845
name: deoxylimonate A-ring-lactonase activity
def: "Catalysis of the reaction: deoxylimonoate + H(2)O = deoxylimononate D-ring-lactone + H(+)." [EC:3.1.1.46, RHEA:15000]
GO:0047846
name: deoxynucleotide 3'-phosphatase activity
def: "Catalysis of the reaction: a deoxynucleoside 3'-phosphate + H2O = a deoxynucleoside + phosphate." [EC:3.1.3.34, MetaCyc:DEOXYNUCLEOTIDE-3-PHOSPHATASE-RXN]
GO:0047847
name: deoxyuridine phosphorylase activity
def: "Catalysis of the reaction: deoxyuridine + phosphate = uracil + deoxy-D-ribose 1-phosphate." [EC:2.4.2.23, MetaCyc:DEOXYURIDINE-PHOSPHORYLASE-RXN]
GO:0047848
name: dephospho-[reductase kinase] kinase activity
def: "Catalysis of the reaction: ATP + dephospho-[[hydroxymethylglutaryl-CoA reductase (NADPH)] kinase] = ADP + [[hydroxymethylglutaryl-CoA reductase (NADPH)] kinase]." [EC:2.7.11.3, MetaCyc:DEPHOSPHO-REDUCTASE-KINASE-KINASE-RXN]
GO:0047849
name: dextransucrase activity
def: "Catalysis of the reaction: sucrose + 1,6-alpha-D-glucosyl(n) = D-fructose + 1,6-alpha-D-glucosyl(n+1)." [EC:2.4.1.5, MetaCyc:DEXTRANSUCRASE-RXN]
GO:0047850
name: diaminopimelate dehydrogenase activity
def: "Catalysis of the reaction: meso-2,6-diaminopimelate + H(2)O + NADP(+) = L-2-amino-6-oxopimelate + H(+) + NADPH + NH(4)(+)." [EC:1.4.1.16, RHEA:13564]
GO:0047851
name: dicarboxylate-CoA ligase activity
def: "Catalysis of the reaction: ATP + an omega-dicarboxylic acid = AMP + diphosphate + an omega-carboxyacyl-CoA." [EC:6.2.1.23, MetaCyc:DICARBOXYLATE--COA-LIGASE-RXN]
GO:0047852
name: diferric-transferrin reductase activity
def: "Catalysis of the reaction: transferrin[Fe2+]2 + NAD+ = transferrin[Fe3+]2 + NADH." [EC:1.16.1.2, MetaCyc:DIFFERIC-TRANSFERRIN-REDUCTASE-RXN]
GO:0047853
name: difructose-anhydride synthase activity
def: "Catalysis of the reaction: H2O + bis-D-fructose 2',1:2,1'-dianhydride = inulobiose." [EC:3.2.1.134, MetaCyc:DIFRUCTOSE-ANHYDRIDE-SYNTHASE-RXN]
GO:0047854
name: diguanidinobutanase activity
def: "Catalysis of the reaction: 1,4-diguanidinobutane + H(2)O = agmatine + urea." [EC:3.5.3.20, RHEA:13600]
GO:0047855
name: dihydrobunolol dehydrogenase activity
def: "Catalysis of the reaction: dihydrobunolol + NADP(+) = bunolol + H(+) + NADPH." [EC:1.1.1.160, RHEA:15928]
GO:0047856
name: dihydrocoumarin hydrolase activity
def: "Catalysis of the reaction: 3,4-dihydrocoumarin + H(2)O = 3-(2-hydroxyphenyl)propanoate + H(+)." [EC:3.1.1.35, RHEA:10363]
GO:0047857
name: dihydrouracil oxidase activity
def: "Catalysis of the reaction: 5,6-dihydrouracil + O(2) = H(2)O(2) + uracil." [EC:1.3.3.7, RHEA:12387]
GO:0047858
name: dihydroxyfumarate decarboxylase activity
def: "Catalysis of the reaction: dihydroxyfumarate + H(+) = 2-hydroxy-3-oxopropanoate + CO(2)." [EC:4.1.1.54, RHEA:13848]
GO:0047859
name: dihydroxyphenylalanine ammonia-lyase activity
def: "OBSOLETE. Catalysis of the reaction: 3,4-dihydroxy-L-phenylalanine = trans-caffeate + NH3." [EC:4.3.1.11, MetaCyc:DIHYDROXYPHENYLALANINE-AMMONIA-LYASE-RXN]
GO:0047860
name: diiodophenylpyruvate reductase activity
def: "Catalysis of the reaction: 3-(3,5-diiodo-4-hydroxyphenyl)lactate + NAD(+) = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + H(+) + NADH." [EC:1.1.1.96, RHEA:20296]
GO:0047861
name: diiodotyrosine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + 3,5-diiodo-L-tyrosine = 3-(3,5-diiodo-4-hydroxyphenyl)pyruvate + L-glutamate." [EC:2.6.1.24, RHEA:19784]
GO:0047862
name: diisopropyl-fluorophosphatase activity
def: "Catalysis of the reaction: diisopropyl fluorophosphate + H(2)O = diisopropyl phosphate + 2 H(+) + hydrogen fluoride." [EC:3.1.8.2, RHEA:24103]
GO:0047863
name: dimethylallylcistransferase activity
def: "Catalysis of the reaction: dimethylallyl diphosphate + isopentenyl diphosphate = diphosphate + neryl diphosphate." [EC:2.5.1.28, RHEA:11331]
GO:0047864
name: dimethylaniline-N-oxide aldolase activity
def: "Catalysis of the reaction: N,N-dimethylaniline N-oxide = N-methylaniline + formaldehyde." [EC:4.1.2.24, RHEA:19324]
GO:0047865
name: dimethylglycine dehydrogenase activity
def: "Catalysis of the reaction: N,N-dimethylglycine + electron-transfer flavoprotein + H2O = sarcosine + formaldehyde + reduced electron-transfer flavoprotein." [EC:1.5.8.4, RHEA:22499]
GO:0047866
name: dimethylglycine oxidase activity
def: "Catalysis of the reaction: N,N-dimethylglycine + H(2)O + O(2) = formaldehyde + H(2)O(2) + sarcosine." [EC:1.5.3.10, RHEA:17080]
GO:0047867
name: dimethylmalate dehydrogenase activity
def: "Catalysis of the reaction: (R)-3,3-dimethylmalate + NAD(+) = 3-methyl-2-oxobutanoate + CO(2) + NADH." [EC:1.1.1.84, RHEA:13324]
GO:0047868
name: dimethylmaleate hydratase activity
def: "Catalysis of the reaction: (2R,3S)-2,3-dimethylmalate = dimethylmaleate + H(2)O." [EC:4.2.1.85, RHEA:20256]
GO:0047869
name: dimethylpropiothetin dethiomethylase activity
def: "Catalysis of the reaction: S,S-dimethyl-beta-propiothetin = acrylate + dimethyl sulfide + H(+)." [EC:4.4.1.3, RHEA:19968]
GO:0047870
name: discadenine synthase activity
def: "Catalysis of the reaction: N(6)-dimethylallyladenine + S-adenosyl-L-methionine(1+) = S-methyl-5'-thioadenosine + discadenine + H(+)." [EC:2.5.1.24, RHEA:19584]
GO:0047871
name: disulfoglucosamine-6-sulfatase activity
def: "Catalysis of the reaction: N(2),6-disulfo-D-glucosamine + H(2)O = N-sulfo-D-glucosamine + H(+) + sulfate." [EC:3.1.6.11, RHEA:15520]
GO:0047872
name: dolichol O-acyltransferase activity
def: "Catalysis of the reaction: palmitoyl-CoA + dolichol = CoA + dolichyl palmitate." [EC:2.3.1.123, MetaCyc:DOLICHOL-O-ACYLTRANSFERASE-RXN]
GO:0047873
name: dolichyl-phosphatase activity
def: "Catalysis of the reaction: dolichyl phosphate + H2O = dolichol + phosphate." [EC:3.1.3.51, MetaCyc:DOLICHYL-PHOSPHATASE-RXN]
GO:0047874
name: dolichyldiphosphatase activity
def: "Catalysis of the reaction: dolichyl diphosphate + H2O = dolichyl phosphate + phosphate." [EC:3.6.1.43, MetaCyc:DOLICHYLDIPHOSPHATASE-RXN]
GO:0047875
name: ecdysone oxidase activity
def: "Catalysis of the reaction: Ecdysone + O(2) = 3-dehydroecdysone + H(2)O(2)." [EC:1.1.3.16, RHEA:11799]
GO:0047876
name: endoglycosylceramidase activity
def: "Catalysis of the reaction: H2O + oligoglycosylglucosylceramide = ceramide + oligoglycosylglucose." [EC:3.2.1.123, MetaCyc:ENDOGLYCOSYLCERAMIDASE-RXN]
GO:0047877
name: ephedrine dehydrogenase activity
def: "Catalysis of the reaction: (1R,2S)-ephedrine + NAD(+) = (R)-2-methylimino-1-phenylpropan-1-ol + 2 H(+) + NADH." [EC:1.5.1.18, RHEA:16292]
GO:0047878
name: erythritol kinase activity
def: "Catalysis of the reaction: ATP + erythritol = D-erythritol 4-phosphate + ADP + 2 H(+)." [EC:2.7.1.27, RHEA:20711]
GO:0047879
name: erythronolide synthase activity
def: "Catalysis of the reaction: 6 malonyl-CoA + propionyl-CoA = 7 CoA + 6-deoxyerythronolide B." [EC:2.3.1.94, MetaCyc:ERYTHRONOLIDE-SYNTHASE-RXN]
GO:0047880
name: erythrulose reductase activity
def: "Catalysis of the reaction: D-threitol + NADP(+) = D-erythrulose + H(+) + NADPH." [EC:1.1.1.162, RHEA:18008]
GO:0047881
name: estradiol 17-alpha-dehydrogenase activity
def: "Catalysis of the reaction: estradiol-17-alpha + NAD(P)+ = estrone + NAD(P)H + H+." [EC:1.1.1.148, MetaCyc:ESTRADIOL-17-ALPHA-DEHYDROGENASE-RXN]
GO:0047882
name: estradiol 6-beta-monooxygenase activity
def: "Catalysis of the reaction: AH(2) + estradiol-17beta + O(2) = 6beta-hydroxyestradiol-17beta + A + H(2)O." [EC:1.14.99.11, RHEA:19140]
GO:0047883
name: ethanolamine oxidase activity
def: "Catalysis of the reaction: ethanolamine + H2O + O2 = glycolaldehyde + NH3 + H2O2." [EC:1.4.3.8, MetaCyc:ETHANOLAMINE-OXIDASE-RXN]
GO:0047884
name: FAD diphosphatase activity
def: "Catalysis of the reaction: FAD + H2O = AMP + FMN." [EC:3.6.1.18, MetaCyc:FAD-PYROPHOSPHATASE-RXN]
GO:0047885
name: farnesol 2-isomerase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesol = 2-cis,6-trans-farnesol." [EC:5.2.1.9, RHEA:13404]
GO:0047886
name: farnesol dehydrogenase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesol + NADP(+) = 2-trans,6-trans-farnesal + H(+) + NADPH." [EC:1.1.1.216, RHEA:14700]
GO:0047887
name: farnesyl diphosphate kinase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate + ATP = 2-trans,6-trans-farnesyl triphosphate + ADP." [EC:2.7.4.18, RHEA:21547]
GO:0047888
name: fatty acid peroxidase activity
def: "Catalysis of the reaction: 2 H(2)O(2) + H(+) + palmitate = CO(2) + 3 H(2)O + pentadecanal." [EC:1.11.1.3, RHEA:23963]
GO:0047889
name: ferredoxin-nitrate reductase activity
def: "Catalysis of the reaction: nitrite + H2O + 2 oxidized ferredoxin = nitrate + 2 reduced ferredoxin." [EC:1.7.7.2, MetaCyc:FERREDOXIN--NITRATE-REDUCTASE-RXN]
GO:0047890
name: flavanone 4-reductase activity
def: "Catalysis of the reaction: (2S)-flavan-4-ol + NADP+ = (2S)-flavanone + NADPH." [EC:1.1.1.234, MetaCyc:FLAVANONE-4-REDUCTASE-RXN]
GO:0047891
name: flavone 7-O-beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + 5,7,3',4'-tetrahydroxyflavone = UDP + 7-O-beta-D-glucosyl-5,7,3',4'-tetrahydroxyflavone." [EC:2.4.1.81, MetaCyc:FLAVONE-7-O-BETA-GLUCOSYLTRANSFERASE-RXN]
GO:0047892
name: flavone apiosyltransferase activity
def: "Catalysis of the reaction: UDP-apiose + 7-O-beta-D-glucosyl-5,7,4'-trihydroxyflavone = UDP + 7-O-(beta-D-apiofuranosyl-1,2-beta-D-glucosyl)-5,7,4'-trihydroxyflavone." [EC:2.4.2.25, MetaCyc:FLAVONE-APIOSYLTRANSFERASE-RXN]
GO:0047893
name: flavonol 3-O-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a flavonol = UDP + a flavonol 3-O-D-glucoside." [EC:2.4.1.91, MetaCyc:FLAVONOL-3-O-GLUCOSYLTRANSFERASE-RXN]
GO:0047894
name: flavonol 3-sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + quercetin = adenosine 3',5'-diphosphate + H(+) + quercetin 3-sulfate." [EC:2.8.2.25, RHEA:13456]
GO:0047895
name: formaldehyde dismutase activity
def: "Catalysis of the reaction: 2 formaldehyde + H2O = methanol + formate." [EC:1.2.99.4, MetaCyc:FORMALDEHYDE-DISMUTASE-RXN]
GO:0047896
name: formaldehyde transketolase activity
def: "Catalysis of the reaction: D-xylulose 5-phosphate + formaldehyde = glyceraldehyde 3-phosphate + glycerone." [EC:2.2.1.3, MetaCyc:FORMALDEHYDE-TRANSKETOLASE-RXN]
GO:0047897
name: formate-dihydrofolate ligase activity
def: "Catalysis of the reaction: 7,8-dihydrofolate + ATP + formate = 10-formyldihydrofolate + ADP + H(+) + phosphate." [EC:6.3.4.17, RHEA:24331]
GO:0047898
name: formate dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: formate + ferricytochrome b1 = CO2 + ferrocytochrome b1." [EC:1.2.2.1, MetaCyc:FORMATE-DEHYDROGENASE-CYTOCHROME-RXN]
GO:0047899
name: formate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: formate + NADP(+) = CO(2) + NADPH." [EC:1.2.1.43, RHEA:12003]
GO:0047900
name: formate kinase activity
def: "Catalysis of the reaction: ATP + formate = ADP + formyl phosphate + H(+)." [EC:2.7.2.6, RHEA:16012]
GO:0047901
name: formyl-CoA hydrolase activity
def: "Catalysis of the reaction: formyl-CoA + H(2)O = CoA + formate + H(+)." [EC:3.1.2.10, RHEA:19744]
GO:0047902
name: formylaspartate deformylase activity
def: "Catalysis of the reaction: N-formyl-L-aspartate + H2O = formate + L-aspartate." [EC:3.5.1.8, MetaCyc:FORMYLASPARTATE-DEFORMYLASE-RXN]
GO:0047903
name: fructose 5-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: D-fructose + NADP+ = 5-dehydro-D-fructose + NADPH." [EC:1.1.1.124, MetaCyc:FRUCTOSE-5-DEHYDROGENASE-NADP+-RXN]
GO:0047904
name: fructose 5-dehydrogenase activity
def: "Catalysis of the reaction: D-fructose + A = 5-dehydro-D-fructose + AH(2)." [EC:1.1.99.11, RHEA:22307]
GO:0047905
name: fructose-6-phosphate phosphoketolase activity
def: "Catalysis of the reaction: D-fructose 6-phosphate + phosphate = acetyl phosphate + D-erythrose 4-phosphate + H2O." [EC:4.1.2.22, MetaCyc:FRUCTOSE-6-PHOSPHATE-PHOSPHOKETOLASE-RXN]
GO:0047906
name: fucosterol-epoxide lyase activity
def: "Catalysis of the reaction: (24R,24'R)-fucosterol epoxide = acetaldehyde + desmosterol." [EC:4.1.2.33, RHEA:10887]
GO:0047907
name: furylfuramide isomerase activity
def: "Catalysis of the reaction: (E)-2-(2-furyl)-3-(5-nitro-2-furyl)acrylamide = (Z)-2-(2-furyl)-3-(5-nitro-2-furyl)acrylamide." [EC:5.2.1.6, RHEA:21851]
GO:0047908
name: fusarinine-C ornithinesterase activity
def: "Catalysis of the reaction: N5-acyl-L-ornithine ester + H2O = N5-acyl-L-ornithine + an alcohol." [EC:3.1.1.48, MetaCyc:FUSARININE-C-ORNITHINESTERASE-RXN]
GO:0047909
name: galactolipid O-acyltransferase activity
def: "Catalysis of the reaction: 2 mono-beta-D-galactosyldiacylglycerol = acylmono-beta-D-galactosyl-diacylglycerol + mono-beta-D-galactosylacylglycerol." [EC:2.3.1.134, MetaCyc:GALACTOLIPID-O-ACYLTRANSFERASE-RXN]
GO:0047910
name: galactose 1-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: D-galactose + NADP+ = D-galactonolactone + NADPH." [EC:1.1.1.120, MetaCyc:GALACTOSE-1-DEHYDROGENASE-NADP+-RXN]
GO:0047911
name: galacturan 1,4-alpha-galacturonidase activity
def: "Catalysis of the reaction: [(1->4)-alpha-D-galacturonide](n) + H2O = [(1->4)-alpha-D-galacturonide](n-1) + D-galacturonate." [EC:3.2.1.67, MetaCyc:GALACTURAN-14-ALPHA-GALACTURONIDASE-RXN]
GO:0047912
name: galacturonokinase activity
def: "Catalysis of the reaction: alpha-D-galacturonate + ATP = 1-phospho-alpha-D-galacturonate + ADP + 2 H(+)." [EC:2.7.1.44, RHEA:12968]
GO:0047913
name: gallate 1-beta-glucosyltransferase activity
def: "Catalysis of the reaction: gallate + UDP-D-glucose = 1-O-galloyl-beta-D-glucose + UDP." [EC:2.4.1.136, RHEA:15252]
GO:0047914
name: gamma-glutamylhistamine synthase activity
def: "Catalysis of the reaction: histamine + L-glutamate + ATP = N(alpha)-gamma-L-glutamylhistamine + products of ATP breakdown." [EC:6.3.2.18, MetaCyc:GAMMA-GLUTAMYLHISTAMINE-SYNTHASE-RXN]
GO:0047915
name: ganglioside galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-1,4-beta-D-glucosyl-N-acylsphingosine = UDP + D-galactosyl-1,3-beta-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosyl-N-acylsphingosine." [EC:2.4.1.62, MetaCyc:GANGLIOSIDE-GALACTOSYLTRANSFERASE-RXN]
GO:0047916
name: GDP-6-deoxy-D-talose 4-dehydrogenase activity
def: "Catalysis of the reaction: GDP-6-deoxy-D-talose + NAD(P)+ = GDP-4-dehydro-6-deoxy-D-talose + NAD(P)H + H+." [EC:1.1.1.135, MetaCyc:GDP-6-DEOXY-D-TALOSE-4-DEHYDROGENASE-RXN]
GO:0047917
name: GDP-glucosidase activity
def: "Catalysis of the reaction: GDP-D-glucose + H(2)O = D-glucose + GDP + H(+)." [EC:3.2.1.42, RHEA:15052]
GO:0047918
name: GDP-mannose 3,5-epimerase activity
def: "Catalysis of the reaction: GDP-mannose = GDP-L-galactose." [EC:5.1.3.18]
GO:0047919
name: GDP-mannose 6-dehydrogenase activity
def: "Catalysis of the reaction: GDP-alpha-D-mannose + H(2)O + 2 NAD(+) = GDP-D-mannuronate + 3 H(+) + 2 NADH." [EC:1.1.1.132, RHEA:21731]
GO:0047920
name: geissoschizine dehydrogenase activity
def: "Catalysis of the reaction: geissoschizine + NADP(+) = 4,21-dehydrogeissoschizine + H(+) + NADPH." [EC:1.3.1.36, RHEA:11379]
GO:0047921
name: aminoglycoside 2'-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + gentamicin C(1a) = N(2')-acetylgentamicin C(1a) + CoA + H(+). This is acetylation of the 2'-amino group of the 6-deoxy-6-aminoglucose ring." [EC:2.3.1.59, RHEA:24519]
GO:0047922
name: gentisate 1,2-dioxygenase activity
def: "Catalysis of the reaction: 2,5-dihydroxybenzoate + O(2) = 3-maleylpyruvate + H(+)." [EC:1.13.11.4, RHEA:18240]
GO:0047923
name: gentisate decarboxylase activity
def: "Catalysis of the reaction: 2,5-dihydroxybenzoate + H(+) = CO(2) + hydroquinone." [EC:4.1.1.62, RHEA:21315]
GO:0047924
name: geraniol dehydrogenase activity
def: "Catalysis of the reaction: geraniol + NADP+ = geranial + NADPH." [EC:1.1.1.183, MetaCyc:GERANIOL-DEHYDROGENASE-RXN]
GO:0047925
name: geranoyl-CoA carboxylase activity
def: "Catalysis of the reaction: ATP + bicarbonate + geranoyl-CoA = 3-(4-methylpent-3-en-1-yl)pent-2-enedioyl-CoA + ADP + 2 H(+) + phosphate." [EC:6.4.1.5, RHEA:17704]
GO:0047926
name: geranyl-diphosphate cyclase activity
def: "Catalysis of the reaction: geranyl diphosphate = (2S)-bornyl diphosphate." [EC:5.5.1.8, RHEA:18212]
GO:0047927
name: gibberellin-44 dioxygenase activity
def: "Catalysis of the reaction: gibberellin 44 + 2-oxoglutarate + O2 = gibberellin 19 + succinate + CO2." [EC:1.14.11.12, MetaCyc:GIBBERELLIN-44-DIOXYGENASE-RXN]
GO:0047928
name: gibberellin beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + gibberellin = UDP + gibberellin 2-O-beta-D-glucoside." [EC:2.4.1.176, MetaCyc:GIBBERELLIN-BETA-GLUCOSYLTRANSFERASE-RXN]
GO:0047929
name: gluconate dehydratase activity
def: "Catalysis of the reaction: D-gluconate = 2-dehydro-3-deoxy-D-gluconate + H(2)O." [EC:4.2.1.39, RHEA:21615]
GO:0047930
name: glucosaminate ammonia-lyase activity
def: "Catalysis of the reaction: D-glucosaminate = 2-dehydro-3-deoxy-D-gluconate + NH3." [EC:4.3.1.9, MetaCyc:GLUCOSAMINATE-AMMONIA-LYASE-RXN]
GO:0047931
name: glucosamine kinase activity
def: "Catalysis of the reaction: ATP + D-glucosamine = ADP + D-glucosamine phosphate." [EC:2.7.1.8, MetaCyc:GLUCOSAMINE-KINASE-RXN]
GO:0047932
name: glucosamine N-acetyltransferase activity
def: "Catalysis of the reaction: D-glucosamine + acetyl-CoA = N-acetyl-D-glucosamine + CoA + H(+)." [EC:2.3.1.3, RHEA:21335]
GO:0047933
name: glucose-1,6-bisphosphate synthase activity
def: "Catalysis of the reaction: 3-phospho-D-glyceroyl phosphate + alpha-D-glucose 1-phosphate = 3-phospho-D-glycerate + alpha-D-glucose 1,6-bisphosphate + H(+)." [EC:2.7.1.106, RHEA:16772]
GO:0047934
name: glucose 1-dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: D-glucose + NAD+ = D-glucono-1,5-lactone + NADH." [EC:1.1.1.118, MetaCyc:GLUCOSE-1-DEHYDROGENASE-NAD+-RXN]
GO:0047935
name: glucose 1-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: D-glucose + NADP+ = D-glucono-1,4-lactone + NADPH." [EC:1.1.1.119, MetaCyc:GLUCOSE-1-DEHYDROGENASE-NADP+-RXN]
GO:0047936
name: glucose 1-dehydrogenase [NAD(P)] activity
def: "Catalysis of the reaction: beta-D-glucose + NAD(P)+ = D-glucono-1,5-lactone + NAD(P)H." [EC:1.1.1.47, MetaCyc:GLUCOSE-1-DEHYDROGENASE-RXN]
GO:0047937
name: glucose-1-phosphate phosphodismutase activity
def: "Catalysis of the reaction: 2 D-glucose 1-phosphate = D-glucose + D-glucose 1,6-bisphosphate." [EC:2.7.1.41, MetaCyc:GLUCOSE-1-PHOSPHATE-PHOSPHODISMUTASE-RXN]
GO:0047938
name: glucose-6-phosphate 1-epimerase activity
def: "Catalysis of the reaction: alpha-D-glucose 6-phosphate = beta-D-glucose 6-phosphate." [EC:5.1.3.15, MetaCyc:GLUCOSE-6-PHOSPHATE-1-EPIMERASE-RXN]
GO:0047939
name: L-glucuronate reductase activity
def: "Catalysis of the reaction: L-gulonate + NADP(+) = D-glucuronate + H(+) + NADPH." [EC:1.1.1.19, RHEA:14912]
GO:0047940
name: glucuronokinase activity
def: "Catalysis of the reaction: D-glucuronate + ATP = 1-phospho-alpha-D-glucuronate + ADP + 2 H(+)." [EC:2.7.1.43, RHEA:17008]
GO:0047941
name: glucuronolactone reductase activity
def: "Catalysis of the reaction: L-gulono-1,4-lactone + NADP(+) = D-glucurono-3,6-lactone + H(+) + NADPH." [EC:1.1.1.20, RHEA:18928]
GO:0047942
name: glutamate-ethylamine ligase activity
def: "Catalysis of the reaction: L-glutamate + ATP + ethylamine = N(5)-ethyl-L-glutamine + ADP + 2 H(+) + phosphate." [EC:6.3.1.6, RHEA:20528]
GO:0047943
name: glutamate-methylamine ligase activity
def: "Catalysis of the reaction: L-glutamate + ATP + methylammonium = N(5)-methyl-L-glutamine + ADP + 2 H(+) + phosphate." [EC:6.3.4.12, RHEA:17120]
GO:0047944
name: glutamate 1-kinase activity
def: "Catalysis of the reaction: L-glutamate + ATP = alpha-L-glutamyl phosphate + ADP + H(+)." [EC:2.7.2.13, RHEA:17220]
GO:0047945
name: L-glutamine:pyruvate aminotransferase activity
def: "Catalysis of the reaction: L-glutamine + pyruvate = 2-oxoglutaramate + L-alanine." [EC:2.6.1.15, RHEA:10403]
GO:0047946
name: glutamine N-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + L-glutamine = CoA + N-acyl-L-glutamine." [EC:2.3.1.68, MetaCyc:GLUTAMINE-N-ACYLTRANSFERASE-RXN]
GO:0047947
name: glutamine N-phenylacetyltransferase activity
def: "Catalysis of the reaction: phenylacetyl-CoA + L-glutamine = CoA + alpha-N-phenylacetyl-L-glutamine." [EC:2.3.1.14, MetaCyc:GLUTAMINE-N-PHENYLACETYLTRANSFERASE-RXN]
GO:0047948
name: glutarate-CoA ligase activity
def: "Catalysis of the reaction: ATP + CoA + glutarate = ADP + glutaryl-CoA + H(+) + phosphate." [EC:6.2.1.6, RHEA:14172]
GO:0047949
name: glutarate-semialdehyde dehydrogenase activity
def: "Catalysis of the reaction: glutarate semialdehyde + NAD+ + H2O = glutarate + NADH." [EC:1.2.1.20, MetaCyc:GLUTARATE-SEMIALDEHYDE-DEHYDROGENASE-RXN]
GO:0047950
name: glutathione oxidase activity
def: "Catalysis of the reaction: 2 glutathione + O(2) = glutathione disulfide + H(2)O(2)." [EC:1.8.3.3, RHEA:24115]
GO:0047951
name: glutathione thiolesterase activity
def: "Catalysis of the reaction: S-acylglutathione + H(2)O = a carboxylate + glutathione + H(+)." [EC:3.1.2.7, RHEA:22711]
GO:0047952
name: glycerol-3-phosphate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + NAD(P)+ = glycerone phosphate + NAD(P)H + H+." [EC:1.1.1.94, MetaCyc:GLYC3PDEHYDROGBIOSYN-RXN]
GO:0047953
name: glycerol 2-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: glycerol + NADP(+) = glycerone + H(+) + NADPH." [EC:1.1.1.156, RHEA:12756]
GO:0047954
name: glycerol-2-phosphatase activity
def: "Catalysis of the reaction: glycerol 2-phosphate + H(2)O = glycerol + phosphate." [EC:3.1.3.19, RHEA:13108]
GO:0047955
name: glycerol dehydrogenase (acceptor) activity
def: "Catalysis of the reaction: A + glycerol = AH(2) + glycerone." [EC:1.1.99.22, RHEA:17496]
GO:0047956
name: glycerol dehydrogenase [NADP+] activity
def: "Catalysis of the reaction: glycerol + NADP+ = D-glyceraldehyde + NADPH." [EC:1.1.1.72, MetaCyc:GLYCEROL-DEHYDROGENASE-NADP+-RXN]
GO:0047957
name: 4'-methoxyisoflavone 2'-hydroxylase activity
def: "Catalysis of the reaction: formononetin + NADPH + O2 = 2'-hydroxyformononetin + NADP+ + H2O." [EC:1.14.13.53, MetaCyc:ISOFLAVONE-2-HYDROXYLASE-RXN]
GO:0047958
name: glycine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: glycine + 2-oxoglutarate = glyoxylate + L-glutamate." [EC:2.6.1.4, MetaCyc:GLYCINE-AMINOTRANSFERASE-RXN]
GO:0047959
name: glycine dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: glycine + H2O + 2 ferricytochrome c = glyoxylate + NH3 + 2 ferrocytochrome c." [EC:1.4.2.1, MetaCyc:GLYCINE-DEHYDROGENASE-CYTOCHROME-RXN]
GO:0047960
name: glycine dehydrogenase activity
def: "Catalysis of the reaction: glycine + H2O + NAD+ = glyoxylate + NH3 + NADH." [EC:1.4.1.10, MetaCyc:GLYCINE-DEHYDROGENASE-RXN]
GO:0047961
name: glycine N-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + glycine = CoA + N-acylglycine." [EC:2.3.1.13, MetaCyc:GLYCINE-N-ACYLTRANSFERASE-RXN]
GO:0047962
name: glycine N-benzoyltransferase activity
def: "Catalysis of the reaction: benzoyl-CoA + glycine = N-benzoylglycine + CoA + H(+)." [EC:2.3.1.71, RHEA:18496]
GO:0047963
name: glycine N-choloyltransferase activity
def: "Catalysis of the reaction: choloyl-CoA + glycine = CoA + glycocholate." [EC:2.3.1.65, MetaCyc:GLYCINE-N-CHOLOYLTRANSFERASE-RXN]
GO:0047964
name: glyoxylate reductase activity
def: "Catalysis of the reaction: glycolate + NAD+ = glyoxylate + NADH." [EC:1.1.1.26, MetaCyc:GLYCOLATE-REDUCTASE-RXN]
GO:0047965
name: glycoprotein O-fatty-acyltransferase activity
def: "Catalysis of the reaction: palmitoyl-CoA + mucus glycoprotein = CoA + O-palmitoylglycoprotein." [EC:2.3.1.142, MetaCyc:GLYCOPROTEIN-O-FATTY-ACYLTRANSFERASE-RXN]
GO:0047966
name: glycosulfatase activity
def: "Catalysis of the reaction: D-glucose 6-sulfate + H(2)O = D-glucose + H(+) + sulfate." [EC:3.1.6.3, RHEA:19148]
GO:0047967
name: glycyrrhizinate beta-glucuronidase activity
def: "Catalysis of the reaction: glycyrrhizate + H(2)O = 2-(beta-D-glucuronosyl)-D-glucuronate + glycyrrhetinate." [EC:3.2.1.128, RHEA:17372]
GO:0047968
name: glyoxylate dehydrogenase (acylating) activity
def: "Catalysis of the reaction: CoA + glyoxylate + NADP(+) = H(+) + NADPH + oxalyl-CoA." [EC:1.2.1.17, RHEA:21027]
GO:0047969
name: glyoxylate oxidase activity
def: "Catalysis of the reaction: glyoxylate + H(2)O + O(2) = H(2)O(2) + H(+) + oxalate." [EC:1.2.3.5, RHEA:14840]
GO:0047970
name: guanidinoacetase activity
def: "Catalysis of the reaction: guanidinoacetate + H(2)O = glycine + urea." [EC:3.5.3.2, RHEA:23271]
GO:0047971
name: guanidinobutyrase activity
def: "Catalysis of the reaction: 4-guanidinobutanoate + H(2)O = 4-aminobutanoate + urea." [EC:3.5.3.7, RHEA:19504]
GO:0047972
name: guanidinopropionase activity
def: "Catalysis of the reaction: 3-guanidinopropanoate + H(2)O = beta-alanine + urea." [EC:3.5.3.17, RHEA:16032]
GO:0047973
name: guanidinoacetate kinase activity
def: "Catalysis of the reaction: ATP + guanidinoacetate = ADP + 2 H(+) + phosphoguanidinoacetate." [EC:2.7.3.1, RHEA:14148]
GO:0047974
name: guanosine deaminase activity
def: "Catalysis of the reaction: guanosine + H2O = xanthosine + NH3." [EC:3.5.4.15, MetaCyc:GUANOSINE-DEAMINASE-RXN]
GO:0047975
name: guanosine phosphorylase activity
def: "Catalysis of the reaction: guanosine + phosphate = guanine + D-ribose 1-phosphate." [EC:2.4.2.15, MetaCyc:GUANPHOSPHOR-RXN]
GO:0047976
name: hamamelose kinase activity
def: "Catalysis of the reaction: D-hamamelose + ATP = D-hamamelose 2'-phosphate + ADP + 2 H(+)." [EC:2.7.1.102, RHEA:22799]
GO:0047977
name: hepoxilin-epoxide hydrolase activity
def: "Catalysis of the reaction: (5Z,9E,14Z)-(8x,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate + H2O = (5Z,9E,14Z)-(8x,11x,12S)-8,11,12-trihydroxyicosa-5,9,14-trienoate." [EC:3.3.2.7, MetaCyc:HEPOXILIN-EPOXIDE-HYDROLASE-RXN]
GO:0047978
name: hexadecanol dehydrogenase activity
def: "Catalysis of the reaction: hexadecanol + NAD+ = hexadecanal + NADH." [EC:1.1.1.164, MetaCyc:HEXADECANOL-DEHYDROGENASE-RXN]
GO:0047979
name: hexose oxidase activity
def: "Catalysis of the reaction: hexose + O2 = aldono-1,5-lactone + H202." [EC:1.1.3.5, MetaCyc:HEXOSE-OXIDASE-RXN]
GO:0047980
name: hippurate hydrolase activity
def: "Catalysis of the reaction: N-benzoylglycine + H(2)O = benzoate + glycine." [EC:3.5.1.32, RHEA:10427]
GO:0047981
name: histidine N-acetyltransferase activity
def: "Catalysis of the reaction: L-histidine + acetyl-CoA = N(alpha)-acetyl-L-histidine + CoA + H(+)." [EC:2.3.1.33, RHEA:24599]
GO:0047982
name: homocysteine desulfhydrase activity
def: "Catalysis of the reaction: L-homocysteine + H2O = sulfide + NH3 + 2-oxobutanoate." [EC:4.4.1.2, MetaCyc:HOMOCYSTEINE-DESULFHYDRASE-RXN]
GO:0047983
name: homoglutathione synthase activity
def: "Catalysis of the reaction: beta-alanine + L-gamma-glutamyl-L-cysteine + ATP = gamma-L-glutamyl-L-cysteinyl-beta-alanine + ADP + 2 H(+) + phosphate." [EC:6.3.2.23, RHEA:17996]
GO:0047985
name: hydrogen dehydrogenase activity
def: "Catalysis of the reaction: H2 + NAD+ = H+ + NADH." [EC:1.12.1.2, MetaCyc:HYDROGEN-DEHYDROGENASE-RXN]
GO:0047986
name: hydrogen-sulfide S-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + S(2-) = CoA + thioacetate." [EC:2.3.1.10, RHEA:16628]
GO:0047987
name: hydroperoxide dehydratase activity
def: "Catalysis of the reaction: (9Z,11E,14Z)-(13S)-hydroperoxyoctadeca-9,11,14-trienoate = (9Z)-(13S)-12,13-epoxyoctadeca-9,11-dienoate + H2O." [EC:4.2.1.92, MetaCyc:HYDROPEROXIDE-DEHYDRATASE-RXN]
GO:0047988
name: hydroxyacid-oxoacid transhydrogenase activity
def: "Catalysis of the reaction: (S)-3-hydroxybutanoate + 2-oxoglutarate = acetoacetate + (R)-2-hydroxyglutarate." [EC:1.1.99.24, MetaCyc:HYDROXYACID-OXOACID-TRANSHYDROGENASE-RXN]
GO:0047989
name: hydroxybutyrate-dimer hydrolase activity
def: "Catalysis of the reaction: (R)-3-[(R)-3-hydroxybutanoyloxy]butanoate + H(2)O = 2 (R)-3-hydroxybutanoate + H(+)." [EC:3.1.1.22, RHEA:10175]
GO:0047990
name: hydroxyglutamate decarboxylase activity
def: "Catalysis of the reaction: 3-hydroxy-L-glutamate + H(+) = 4-amino-3-hydroxybutanoate + CO(2)." [EC:4.1.1.16, RHEA:14076]
GO:0047991
name: hydroxylamine oxidase activity
def: "Catalysis of the reaction: hydroxylamine + O(2) = H(2)O + H(+) + nitrite." [EC:1.7.3.4, RHEA:19972]
GO:0047992
name: hydroxylysine kinase activity
def: "Catalysis of the reaction: erythro-5-hydroxy-L-lysine + GTP = 5-phosphonooxy-L-lysine + GDP + 2 H(+)." [EC:2.7.1.81, RHEA:19052]
GO:0047993
name: hydroxymalonate dehydrogenase activity
def: "Catalysis of the reaction: hydroxymalonate + NAD(+) = H(+) + NADH + oxomalonate." [EC:1.1.1.167, RHEA:11287]
GO:0047994
name: hydroxymethylglutaryl-CoA hydrolase activity
def: "Catalysis of the reaction: (S)-3-hydroxy-3-methylglutaryl-CoA + H(2)O = 3-hydroxy-3-methylglutarate + CoA + H(+)." [EC:3.1.2.5, RHEA:16308]
GO:0047995
name: hydroxyphenylpyruvate reductase activity
def: "Catalysis of the reaction: 3-(4-hydroxyphenyl)lactate + NAD+ = 3-(4-hydroxyphenyl)pyruvate + NADH." [EC:1.1.1.237, MetaCyc:HYDROXYPHENYLPYRUVATE-REDUCTASE-RXN]
GO:0047996
name: hydroxyphytanate oxidase activity
def: "Catalysis of the reaction: (2S)-2-hydroxyphytanate + O(2) = 2-oxophytanate + H(2)O(2)." [EC:1.1.3.27, RHEA:21683]
GO:0047997
name: hydroxypyruvate decarboxylase activity
def: "Catalysis of the reaction: 3-hydroxypyruvate + H(+) = CO(2) + glycolaldehyde." [EC:4.1.1.40, RHEA:20564]
GO:0047998
name: hyoscyamine (6S)-dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-hyoscyamine + O(2) = (6S)-6-hydroxyhyoscyamine + CO(2) + succinate." [EC:1.14.11.11, RHEA:12632]
GO:0047999
name: hyponitrite reductase activity
def: "Catalysis of the reaction: 2 hydroxylamine + 2 NAD(+) = 2 H(+) + hyponitrous acid + 2 NADH." [EC:1.7.1.5, RHEA:19340]
GO:0048000
name: isoflavone 3'-hydroxylase activity
def: "Catalysis of the reaction: formononetin + NADPH + O2 = calycosin + NADP+ + H2O." [EC:1.14.13.52, MetaCyc:ISOFLAVONE-3'-HYDROXYLASE-RXN]
GO:0048001
name: erythrose-4-phosphate dehydrogenase activity
def: "Catalysis of the reaction: D-erythrose 4-phosphate + H(2)O + NAD(+) = 4-phospho-D-erythronate + 2 H(+) + NADH." [EC:1.2.1.72, RHEA:12059]
GO:0048018
name: receptor agonist activity
def: "Interacts with receptors such that the proportion of receptors in the active form is increased." [GOC:ceb, ISBN:0198506732]
GO:0048019
name: receptor antagonist activity
def: "Interacts with receptors to reduce the action of another ligand, the agonist." [GOC:ceb, ISBN:0198506732]
GO:0048020
name: CCR chemokine receptor binding
def: "Interacting selectively and non-covalently with a CCR chemokine receptor." [GOC:ai]
GO:0048027
name: mRNA 5'-UTR binding
def: "Interacting selectively and non-covalently with the 5' untranslated region of an mRNA molecule." [GOC:jid]
GO:0048028
name: galacturonan binding
def: "Interacting selectively and non-covalently with any simple or complex galacturonan. Galacturonan is any glycan composed solely of galacturonic acid residues, a specific type of glycuronan, and a constituent of some pectins." [GOC:jid]
GO:0048029
name: monosaccharide binding
def: "Interacting selectively and non-covalently with any monosaccharide. Monosaccharides are the simplest carbohydrates; they are polyhydroxy aldehydes H[CH(OH)]nC(=O)H or polyhydroxy ketones H[CHOH]nC(=O)[CHOH]mH with three or more carbon atoms. They form the constitutional repeating units of oligo- and polysaccharides." [CHEBI:35381, GOC:jid]
GO:0048030
name: disaccharide binding
def: "Interacting selectively and non-covalently with any disaccharide. Disaccharides are sugars composed of two monosaccharide units." [GOC:jid]
GO:0048031
name: trisaccharide binding
def: "Interacting selectively and non-covalently with any trisaccharide. Trisaccharides are sugars composed of three monosaccharide units." [GOC:jid]
GO:0048032
name: galacturonate binding
def: "Interacting selectively and non-covalently with any galacturonate. Galacturonate is the anion of galacturonic acid, the uronic acid formally derived from galactose by oxidation of the hydroxymethylene group at C-6 to a carboxyl group." [CHEBI:33812, GOC:jid]
GO:0048037
name: cofactor binding
def: "Interacting selectively and non-covalently with a cofactor, a substance that is required for the activity of an enzyme or other protein. Cofactors may be inorganic, such as the metal atoms zinc, iron, and copper in certain forms, or organic, in which case they are referred to as coenzymes. Cofactors may either be bound tightly to active sites or bind loosely with the substrate." [ISBN:0198506732]
GO:0048038
name: quinone binding
def: "Interacting selectively and non-covalently with a quinone, any member of a class of diketones derivable from aromatic compounds by conversion of two CH groups into CO groups with any necessary rearrangement of double bonds." [ISBN:0198506732]
GO:0048039
name: ubiquinone binding
def: "Interacting selectively and non-covalently with ubiquinone, a quinone derivative with a tail of isoprene units." [GOC:jid, ISBN:0582227089]
GO:0048040
name: UDP-glucuronate decarboxylase activity
def: "Catalysis of the reaction: H(+) + UDP-alpha-D-glucuronate = CO(2) + UDP-alpha-D-xylose." [EC:4.1.1.35, RHEA:23919]
GO:0048045
name: trans-pentaprenyltranstransferase activity
def: "Catalysis of the reaction: all-trans-pentaprenyl diphosphate + isopentenyl diphosphate = all-trans-hexaprenyl diphosphate + diphosphate." [RHEA:22635]
GO:0048101
name: calcium- and calmodulin-regulated 3',5'-cyclic-GMP phosphodiesterase activity
def: "Catalysis of the reaction: guanosine 3',5'-cyclic phosphate + H2O = guanosine 5'-phosphate. The reaction is calmodulin and calcium-sensitive." [GOC:jid]
GO:0048156
name: tau protein binding
def: "Interacting selectively and non-covalently with tau protein. tau is a microtubule-associated protein, implicated in Alzheimer's disease, Down Syndrome and ALS." [GOC:jid]
GO:0048184
name: follistatin binding
def: "OBSOLETE. Interacting selectively and non-covalently with the peptide hormone follistatin." [GOC:jid, GOC:mah]
GO:0048185
name: activin binding
def: "Interacting selectively and non-covalently with activin, a dimer of inhibin-beta subunits." [GOC:jid, GOC:mah]
GO:0048191
name: peptide stabilization activity
def: "OBSOLETE. Strengthening of a bond between peptides. Peptides are compounds of two or more amino acids where the alpha carboxyl group of one is bound to the alpha amino group of another." [GOC:jid]
GO:0048192
name: peptide antigen stabilization activity
def: "OBSOLETE. Strengthening of a bond with a peptide antigen; a fragment of a foreign protein derived by proteolysis within the cell." [GOC:jid]
GO:0048244
name: phytanoyl-CoA dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate + O(2) + phytanoyl-CoA = 2-hydroxyphytanoyl-CoA + CO(2) + succinate." [EC:1.14.11.18, RHEA:16068]
GO:0048248
name: CXCR3 chemokine receptor binding
def: "Interacting selectively and non-covalently with a the CXCR3 chemokine receptor." [GOC:jid, PMID:10556837]
GO:0048249
name: high-affinity phosphate transmembrane transporter activity
def: "Catalysis of the transfer of phosphate from one side of the membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:jid, PMID:8709965]
GO:0048256
name: flap endonuclease activity
def: "Catalysis of the cleavage of a flap structure in DNA, but not other DNA structures; processes the ends of Okazaki fragments in lagging strand DNA synthesis." [GOC:jid]
GO:0048257
name: 3'-flap endonuclease activity
def: "Catalysis of the cleavage of a 3' flap structure in DNA, but not other DNA structures; processes the 3' ends of Okazaki fragments in lagging strand DNA synthesis." [GOC:jid, PMID:10635319]
GO:0048258
name: 3-ketoglucose-reductase activity
def: "Catalysis of the reaction: NADP+ + 3-dehydro-alpha-D-glucose = NADPH + alpha-D-glucose." [MetaCyc:KETOGLUCOSE-REDUCTASE-RXN]
GO:0048270
name: methionine adenosyltransferase regulator activity
def: "Modulates the activity of methionine adenosyltransferase." [EC:2.5.1.6, GOC:jid, PMID:10644686]
GO:0048273
name: mitogen-activated protein kinase p38 binding
def: "Interacting selectively and non-covalently with mitogen-activated protein kinase p38, an enzyme that catalyzes the transfer of phosphate from ATP to hydroxyl side chains on proteins in response to mitogen activation." [GOC:curators, PMID:17827184]
GO:0048306
name: calcium-dependent protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules), in the presence of calcium." [GOC:jid, PMID:10485905]
GO:0048307
name: ferredoxin-nitrite reductase activity
def: "Catalysis of the reaction: NH3 + 2 H2O + 6 oxidized ferredoxin = nitrite + 6 reduced ferredoxin + 7 H+." [EC:1.7.7.1, GOC:jid]
GO:0048365
name: Rac GTPase binding
def: "Interacting selectively and non-covalently with Rac protein, any member of the Rac subfamily of the Ras superfamily of monomeric GTPases." [GOC:ebc, PMID:11702775]
GO:0048403
name: brain-derived neurotrophic factor binding
def: "Interacting selectively and non-covalently with brain-derived neurotrophic factor." [GOC:dgh]
GO:0048406
name: nerve growth factor binding
def: "Interacting selectively and non-covalently with nerve growth factor (NGF)." [GOC:dgh]
GO:0048407
name: platelet-derived growth factor binding
def: "Interacting selectively and non-covalently with platelet-derived growth factor." [GOC:dgh]
GO:0048408
name: epidermal growth factor binding
def: "Interacting selectively and non-covalently with epidermal growth factor." [GOC:dgh]
GO:0048472
name: threonine-phosphate decarboxylase activity
def: "Catalysis of the reaction: O-phospho-L-threonine + H(+) = (R)-1-aminopropan-2-yl phosphate + CO(2)." [EC:4.1.1.81, RHEA:11495]
GO:0048474
name: D-methionine transmembrane transporter activity
def: "Catalysis of the transfer of D-methionine from one side of a membrane to the other." [GOC:mlg, GOC:mtg_transport, ISBN:0815340729]
GO:0048487
name: beta-tubulin binding
def: "Interacting selectively and non-covalently with the microtubule constituent protein beta-tubulin." [GOC:krc]
GO:0048495
name: Roundabout binding
def: "Interacting selectively and non-covalently with the Roundabout (ROBO) receptor, a transmembrane receptor." [GOC:ecd, PMID:10102268, PMID:10197527]
GO:0048502
name: thiamine-transporting ATPase activity
def: "Catalysis of the reaction: ATP + H2O + thiamine(out) = ADP + phosphate + thiamine(in)." [GOC:mlg, PMID:9535878]
GO:0048503
name: GPI anchor binding
def: "OBSOLETE. Interacting selectively with any glycosylphosphatidylinositol anchor. GPI anchors serve to attach membrane proteins to the lipid bilayer of cell membranes." [GOC:vw]
GO:0048529
name: magnesium-protoporphyrin IX monomethyl ester (oxidative) cyclase activity
def: "Catalysis of the reaction: magnesium protoporphyrin IX 13-monomethyl ester + 3 NADPH + 3 H+ + 3 O2 = divinylprotochlorophyllide + 3 NADP+ + 5 H2O." [EC:1.14.13.81]
GO:0048531
name: beta-1,3-galactosyltransferase activity
def: "Catalysis of the transfer of a galactose residue from a donor molecule to an oligosaccharide, forming a beta-1,3-linkage." [PMID:11551958]
GO:0048551
name: metalloenzyme inhibitor activity
def: "Stops, prevents or reduces the activity of a metalloenzyme. A metalloenzyme is any enzyme that contains metal." [GOC:ebc]
GO:0048763
name: calcium-induced calcium release activity
def: "Release of calcium ions from an intracellular vesicle on induction by increased calcium concentration." [GOC:jid, GOC:nln, PMID:2990997, PMID:8381210, PMID:8653752]
GO:0050001
name: D-glutaminase activity
def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + D-glutamate." [EC:3.5.1.35, MetaCyc:D-GLUTAMINASE-RXN]
GO:0050002
name: D-proline reductase (dithiol) activity
def: "Catalysis of the reaction: lipoate + 5-aminopentanoate = dihydrolipoate + D-proline." [EC:1.21.4.1, MetaCyc:D-PROLINE-REDUCTASE-(DITHIOL)-RXN]
GO:0050003
name: deoxycytidylate C-methyltransferase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + dCMP = 2'-deoxy-5-methyl-5'-cytidylate + 7,8-dihydrofolate." [EC:2.1.1.54, RHEA:11571]
GO:0050004
name: isoflavone 7-O-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + isoflavone = UDP + isoflavone 7-O-beta-D-glucoside." [EC:2.4.1.170, MetaCyc:ISOFLAVONE-7-O-GLUCOSYLTRANSFERASE-RXN]
GO:0050005
name: isohexenylglutaconyl-CoA hydratase activity
def: "Catalysis of the reaction: 3-hydroxy-3-(4-methylpent-3-en-1-yl)glutaryl-CoA = 3-(4-methylpent-3-en-1-yl)pent-2-enedioyl-CoA + H(2)O." [EC:4.2.1.57, RHEA:24147]
GO:0050006
name: isomaltulose synthase activity
def: "Catalysis of the reaction: sucrose = 6-O-alpha-D-glucopyranosyl-D-fructofuranose." [EC:5.4.99.11, MetaCyc:ISOMALTULOSE-SYNTHASE-RXN]
GO:0050007
name: isonocardicin synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + nocardicin E = S-methyl-5'-thioadenosine + H(+) + isonocardicin A." [EC:2.5.1.38, RHEA:19848]
GO:0050008
name: isopiperitenone delta-isomerase activity
def: "Catalysis of the reaction: isopiperitenone = piperitenone." [EC:5.3.3.11, RHEA:21519]
GO:0050009
name: isopropanol dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + propan-2-ol = acetone + H(+) + NADPH." [EC:1.1.1.80, RHEA:21795]
GO:0050010
name: isovitexin beta-glucosyltransferase activity
def: "Catalysis of the reaction: isovitexin + UDP-D-glucose = H(+) + isovitexin 2''-O-beta-D-glucoside + UDP." [EC:2.4.1.106, RHEA:19532]
GO:0050011
name: itaconyl-CoA hydratase activity
def: "Catalysis of the reaction: citramalyl-CoA = itaconyl-CoA + H2O." [EC:4.2.1.56, MetaCyc:ITACONYL-COA-HYDRATASE-RXN]
GO:0050012
name: juglone 3-monooxygenase activity
def: "Catalysis of the reaction: 5-hydroxy-1,4-naphthoquinone + AH(2) + O(2) = 3,5-dihydroxy-1,4-naphthoquinone + A + H(2)O + H(+)." [EC:1.14.99.27, RHEA:18748]
GO:0050013
name: 2-dehydropantoate aldolase activity
def: "Catalysis of the reaction: 2-dehydropantoate = 3-methyl-2-oxobutanoate + formaldehyde." [EC:4.1.2.12, MetaCyc:KETOPANTOALDOLASE-RXN]
GO:0050014
name: ketotetrose-phosphate aldolase activity
def: "Catalysis of the reaction: L-erythrulose 1-phosphate = formaldehyde + glycerone phosphate." [EC:4.1.2.2, RHEA:20935]
GO:0050015
name: kievitone hydratase activity
def: "Catalysis of the reaction: kievitone hydrate = H(2)O + H(+) + kievitone." [EC:4.2.1.95, RHEA:23607]
GO:0050016
name: kynurenine 7,8-hydroxylase activity
def: "Catalysis of the reaction: kynurenate + donor-H2 + O2 = 7,8-dihydro-7,8-dihydroxykynurenate + acceptor." [EC:1.14.99.2, MetaCyc:KYNURENINE-78-HYDROXYLASE-RXN]
GO:0050017
name: L-3-cyanoalanine synthase activity
def: "Catalysis of the reaction: L-cysteine + HCN = sulfide + L-3-cyanoalanine." [EC:4.4.1.9, MetaCyc:L-3-CYANOALANINE-SYNTHASE-RXN]
GO:0050018
name: L-amino-acid dehydrogenase activity
def: "Catalysis of the reaction: an L-amino acid + H2O + NAD+ = a 2-oxo acid + NH3 + NADH." [EC:1.4.1.5, MetaCyc:L-AMINO-ACID-DEHYDROGENASE-RXN]
GO:0050019
name: L-arabinitol 4-dehydrogenase activity
def: "Catalysis of the reaction: L-arabinitol + NAD(+) = L-xylulose + H(+) + NADH." [EC:1.1.1.12, RHEA:16384]
GO:0050020
name: L-arabinonate dehydratase activity
def: "Catalysis of the reaction: L-arabinonate = 2-dehydro-3-deoxy-L-arabinonate + H(2)O." [EC:4.2.1.25, RHEA:20971]
GO:0050021
name: L-arabinonolactonase activity
def: "Catalysis of the reaction: L-arabinono-1,4-lactone + H(2)O = L-arabinonate + H(+)." [EC:3.1.1.15, RHEA:16220]
GO:0050022
name: L-arabinose 1-dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: L-arabinose + NAD+ = L-arabinono-1,4-lactone + NADH." [EC:1.1.1.46, MetaCyc:L-ARABINOSE-1-DEHYDROGENASE-RXN]
GO:0050023
name: L-fuconate dehydratase activity
def: "Catalysis of the reaction: L-fuconate = 2-dehydro-3-deoxy-L-fuconate + H(2)O." [EC:4.2.1.68, RHEA:22775]
GO:0050024
name: L-galactonolactone oxidase activity
def: "Catalysis of the reaction: L-galactono-1,4-lactone + O(2) = L-ascorbate + H(2)O(2) + H(+)." [EC:1.3.3.12, RHEA:20620]
GO:0050025
name: L-glutamate oxidase activity
def: "Catalysis of the reaction: L-glutamate + O2 + H2O = 2-oxoglutarate + NH3 + H2O2." [EC:1.4.3.11, MetaCyc:L-GLUTAMATE-OXIDASE-RXN]
GO:0050026
name: L-glycol dehydrogenase activity
def: "Catalysis of the reaction: an L-glycol + NAD(P)+ = a 2-hydroxycarbonyl compound + NAD(P)H + H+." [EC:1.1.1.185, MetaCyc:L-GLYCOL-DEHYDROGENASE-RXN]
GO:0050027
name: L-idonate 2-dehydrogenase activity
def: "OBSOLETE. Catalysis of the reaction: L-idonate + NADP+ = 5-dehydro-D-gluconate + NADPH." [EC:1.1.1.128, MetaCyc:L-IDONATE-2-DEHYDROGENASE-RXN]
GO:0050028
name: L-lysine-lactamase activity
def: "Catalysis of the reaction: L-2-aminohexano-6-lactam + H(2)O = L-lysine." [EC:3.5.2.11, RHEA:21391]
GO:0050029
name: L-lysine oxidase activity
def: "Catalysis of the reaction: L-lysine + H(2)O + O(2) = 6-amino-2-oxohexanoate + H(2)O(2) + NH(4)(+)." [EC:1.4.3.14, RHEA:14440]
GO:0050030
name: L-pipecolate dehydrogenase activity
def: "Catalysis of the reaction: L-pipecolate + acceptor = delta1-piperideine-6-carboxylate + reduced acceptor. Delta1-piperideine-6-carboxylate is also known as 2,3,4,5-tetrahydropyridine-2-carboxylate." [EC:1.5.99.3, MetaCyc:L-PIPECOLATE-DEHYDROGENASE-RXN]
GO:0050031
name: L-pipecolate oxidase activity
def: "Catalysis of the reaction: L-pipecolate + O(2) = 2,3,4,5-tetrahydropyridine-2-carboxylate + H(2)O(2) + H(+). Delta1-piperideine-6-carboxylate is also known as 2,3,4,5-tetrahydropyridine-2-carboxylate." [EC:1.5.3.7, RHEA:11995]
GO:0050032
name: L-rhamnonate dehydratase activity
def: "Catalysis of the reaction: L-rhamnonate = 2-dehydro-3-deoxy-L-rhamnonate + H(2)O." [EC:4.2.1.90, RHEA:23083]
GO:0050033
name: L-rhamnono-1,4-lactonase activity
def: "Catalysis of the reaction: L-rhamnono-1,4-lactone + H(2)O = L-rhamnonate + H(+)." [EC:3.1.1.65, RHEA:10291]
GO:0050034
name: L-rhamnose 1-dehydrogenase activity
def: "Catalysis of the reaction: L-rhamnofuranose + NAD(+) = L-rhamnono-1,4-lactone + H(+) + NADH." [EC:1.1.1.173, RHEA:12652]
GO:0050035
name: L-sorbose oxidase activity
def: "Catalysis of the reaction: L-sorbose + O(2) = 5-dehydro-D-fructose + H(2)O(2)." [EC:1.1.3.11, RHEA:17856]
GO:0050036
name: L-threonate 3-dehydrogenase activity
def: "Catalysis of the reaction: L-threonate + NAD(+) = 3-dehydro-L-threonate + H(+) + NADH." [EC:1.1.1.129, RHEA:23379]
GO:0050037
name: L-xylose 1-dehydrogenase activity
def: "Catalysis of the reaction: aldehydo-L-xylose + NADP(+) = L-xylono-1,4-lactone + H(+) + NADPH." [EC:1.1.1.113, RHEA:15792]
GO:0050038
name: L-xylulose reductase (NADP+) activity
def: "Catalysis of the reaction: NADP(+) + xylitol = L-xylulose + H(+) + NADPH." [EC:1.1.1.10, RHEA:17028]
GO:0050039
name: lactaldehyde reductase (NADPH) activity
def: "Catalysis of the reaction: NADP(+) + propane-1,2-diol = (S)-lactaldehyde + H(+) + NADPH." [EC:1.1.1.55, RHEA:15888]
GO:0050040
name: lactate 2-monooxygenase activity
def: "Catalysis of the reaction: (S)-lactate + O(2) = acetate + CO(2) + H(2)O." [EC:1.13.12.4, RHEA:16516]
GO:0050041
name: lactate aldolase activity
def: "Catalysis of the reaction: (S)-lactate = acetaldehyde + formate." [EC:4.1.2.36, RHEA:17912]
GO:0050042
name: lactate-malate transhydrogenase activity
def: "Catalysis of the reaction: (S)-lactate + oxaloacetate = malate + pyruvate." [EC:1.1.99.7, RHEA:10987]
GO:0050043
name: lactate racemase activity
def: "Catalysis of the reaction: (S)-lactate = (R)-lactate." [EC:5.1.2.1, RHEA:10963]
GO:0050044
name: galactose-6-phosphate isomerase activity
def: "Catalysis of the reaction: D-galactose 6-phosphate = D-tagatose 6-phosphate." [EC:5.3.1.26, RHEA:13036]
GO:0050045
name: laminaribiose phosphorylase activity
def: "Catalysis of the reaction: 3-beta-D-glucosyl-D-glucose + phosphate = D-glucose + alpha-D-glucose 1-phosphate." [EC:2.4.1.31, MetaCyc:LAMINARIBIOSE-PHOSPHORYLASE-RXN]
GO:0050046
name: lathosterol oxidase activity
def: "Catalysis of the reaction: 5-alpha-cholest-7-en-3-beta-ol + O2 = cholesta-5,7-dien-3-beta-ol + H2O2." [EC:1.14.21.6, MetaCyc:LATHOSTEROL-OXIDASE-RXN]
GO:0050047
name: leucine 2,3-aminomutase activity
def: "Catalysis of the reaction: L-leucine = (3R)-beta-leucine." [EC:5.4.3.7, RHEA:10287]
GO:0050048
name: L-leucine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-leucine + 2-oxoglutarate = 4-methyl-2-oxopentanoate + L-glutamate." [EC:2.6.1.6, MetaCyc:LEUCINE-AMINOTRANSFERASE-RXN]
GO:0050049
name: leucine dehydrogenase activity
def: "Catalysis of the reaction: L-leucine + H2O + NAD+ = 4-methyl-2-oxopentanoate + NH3 + NADH." [EC:1.4.1.9, MetaCyc:LEUCINE-DEHYDROGENASE-RXN]
GO:0050050
name: leucine N-acetyltransferase activity
def: "Catalysis of the reaction: L-leucine + acetyl-CoA = N-acetyl-L-leucine + CoA + H(+)." [EC:2.3.1.66, RHEA:20092]
GO:0050051
name: leukotriene-B4 20-monooxygenase activity
def: "Catalysis of the reaction: (6Z,8E,10E,14Z)-(5S,12R)-5,12-dihydroxyicosa-6,8,10,14-tetraenoate + NADPH + H+ + O2 = (6Z,8E,10E,14Z)-(5S,12R)-5,12,20-trihydroxyicosa-6,8,10,14-tetraenoate + NADP+ + H2O." [EC:1.14.13.30, MetaCyc:LEUKOTRIENE-B4-20-MONOOXYGENASE-RXN]
GO:0050052
name: leukotriene-E4 20-monooxygenase activity
def: "Catalysis of the reaction: H(+) + leukotriene E(4) + NADPH + O(2) = 20-hydroxy-leukotriene E(4) + H(2)O + NADP(+)." [EC:1.14.13.34, RHEA:24123]
GO:0050053
name: levansucrase activity
def: "Catalysis of the reaction: sucrose + 2,6-beta-D-fructosyl(n) = glucose + 2,6-beta-D-fructosyl(n+1)." [EC:2.4.1.10, MetaCyc:LEVANSUCRASE-RXN]
GO:0050054
name: lignostilbene alpha beta-dioxygenase activity
def: "Catalysis of the reaction: 1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene + O(2) = 2 vanillin." [EC:1.13.11.43, RHEA:21343]
GO:0050055
name: limonin-D-ring-lactonase activity
def: "Catalysis of the reaction: limonoate D-ring-lactone + H2O = limonoate." [EC:3.1.1.36, MetaCyc:LIMONIN-D-RING-LACTONASE-RXN]
GO:0050056
name: linalool 8-monooxygenase activity
def: "Catalysis of the reaction: AH(2) + linalool + O(2) = (2E)-2,6-dimethylocta-2,7-diene-1,6-diol + A + H(2)O." [EC:1.14.99.28, RHEA:20763]
GO:0050057
name: linamarin synthase activity
def: "Catalysis of the reaction: UDP-glucose + 2-hydroxy-2-methylpropanenitrile = UDP + linamarin." [EC:2.4.1.63, MetaCyc:LINAMARIN-SYNTHASE-RXN]
GO:0050058
name: linoleate isomerase activity
def: "Catalysis of the reaction: linoleate = 9-cis,11-trans-octadecadienoate." [EC:5.2.1.5, RHEA:17384]
GO:0050059
name: lombricine kinase activity
def: "Catalysis of the reaction: ATP + lombricine = ADP + N-phospholombricine." [EC:2.7.3.5, MetaCyc:LOMBRICINE-KINASE-RXN]
GO:0050060
name: long-chain-alcohol dehydrogenase activity
def: "Catalysis of the reaction: a long-chain alcohol + 2 NAD+ + H2O = a long-chain carboxylate + 2 NADH." [EC:1.1.1.192, MetaCyc:LONG-CHAIN-ALCOHOL-DEHYDROGENASE-RXN]
GO:0050061
name: long-chain-aldehyde dehydrogenase activity
def: "Catalysis of the reaction: a long-chain aldehyde + NAD+ = a long-chain carboxylate + NADH + H+. A long-chain aldehyde is one with more than 12 carbons." [EC:1.2.1.48, MetaCyc:LONG-CHAIN-ALDEHYDE-DEHYDROGENASE-RXN]
GO:0050062
name: long-chain-fatty-acyl-CoA reductase activity
def: "Catalysis of the reaction: a long-chain aldehyde + CoA + NADP+ = a long-chain acyl-CoA + NADPH." [EC:1.2.1.50, MetaCyc:LONG-CHAIN-FATTY-ACYL-COA-REDUCTASE-RXN]
GO:0050063
name: low-density-lipoprotein particle receptor kinase activity
def: "Catalysis of the reaction: ATP + low-density lipoprotein L-serine = ADP + low-density lipoprotein O-phospho-L-serine." [EC:2.7.11.29, MetaCyc:LOW-DENSITY-LIPOPROTEIN-KINASE-RXN]
GO:0050064
name: luteolin 7-O-glucuronosyltransferase activity
def: "Catalysis of the reaction: luteolin + UDP-alpha-D-glucuronate = luteolin 7-O-beta-D-glucosiduronate + UDP." [EC:2.4.1.189, RHEA:10571]
GO:0050065
name: lysine-pyruvate 6-transaminase activity
def: "Catalysis of the reaction: L-lysine + pyruvate = L-alanine + L-allysine." [EC:2.6.1.71, RHEA:19396]
GO:0050066
name: lysine 2,3-aminomutase activity
def: "Catalysis of the reaction: L-lysine = (3S)-3,6-diaminohexanoate." [EC:5.4.3.2, RHEA:19180]
GO:0050067
name: lysine 2-monooxygenase activity
def: "Catalysis of the reaction: L-lysine + O(2) = 5-aminopentanamide + CO(2) + H(2)O." [EC:1.13.12.2, RHEA:14604]
GO:0050068
name: lysine carbamoyltransferase activity
def: "Catalysis of the reaction: L-lysine + carbamoyl phosphate = L-homocitrulline + H(+) + phosphate." [EC:2.1.3.8, RHEA:17124]
GO:0050069
name: lysine dehydrogenase activity
def: "Catalysis of the reaction: L-lysine + NAD+ = 1,2-didehydropiperidine-2-carboxylate + NH3 + NADH." [EC:1.4.1.15, MetaCyc:LYSINE-DEHYDROGENASE-RXN]
GO:0050070
name: lysolecithin acylmutase activity
def: "Catalysis of the reaction: 1-acyl-sn-glycero-3-phosphocholine = 2-acyl-sn-glycero-3-phosphocholine." [EC:5.4.1.1, RHEA:24359]
GO:0050071
name: lysyltransferase activity
def: "Catalysis of the reaction: L-lysyl-tRNA + phosphatidylglycerol = tRNA + 3-phosphatidyl-1'-(3'-O-L-lysyl)glycerol." [EC:2.3.2.3, MetaCyc:LYSYLTRANSFERASE-RXN]
GO:0050072
name: m7G(5')pppN diphosphatase activity
def: "Catalysis of the reaction: 7-methylguanosine 5'-triphospho-5'-polynucleotide + H2O = 7-methylguanosine 5'-phosphate + polynucleotide." [EC:3.6.1.30, MetaCyc:M7G5PPPN-PYROPHOSPHATASE-RXN]
GO:0050073
name: macrolide 2'-kinase activity
def: "Catalysis of the reaction: ATP + oleandomycin = ADP + 2 H(+) + oleandomycin 2'-O-phosphate." [EC:2.7.1.136, RHEA:18336]
GO:0050074
name: malate-CoA ligase activity
def: "Catalysis of the reaction: ATP + malate + CoA = ADP + phosphate + malyl-CoA." [EC:6.2.1.9, MetaCyc:MALATE--COA-LIGASE-RXN]
GO:0050075
name: maleate hydratase activity
def: "Catalysis of the reaction: (R)-malate = H(2)O + maleate." [EC:4.2.1.31, RHEA:23695]
GO:0050076
name: maleate isomerase activity
def: "Catalysis of the reaction: maleate = fumarate." [EC:5.2.1.1, RHEA:13172]
GO:0050077
name: maleylpyruvate isomerase activity
def: "Catalysis of the reaction: 3-maleylpyruvate = 3-fumarylpyruvate." [EC:5.2.1.4, MetaCyc:MALEYLPYRUVATE-ISOMERASE-RXN]
GO:0050078
name: malonate CoA-transferase activity
def: "Catalysis of the reaction: acetyl-CoA + malonate = acetate + malonyl-CoA." [EC:2.8.3.3, RHEA:18820]
GO:0050079
name: acetylenecarboxylate hydratase activity, producing 3-oxopropanoate
def: "Catalysis of the reaction: 3-oxopropanoate = propynoate + H2O." [EC:4.2.1.27, MetaCyc:MALONATE-SEMIALDEHYDE-DEHYDRATASE-RXN]
GO:0050080
name: malonyl-CoA decarboxylase activity
def: "Catalysis of the reaction: malonyl-CoA = acetyl-CoA + CO2." [EC:4.1.1.9, MetaCyc:MALONYL-COA-DECARBOXYLASE-RXN]
GO:0050081
name: maltose-6'-phosphate glucosidase activity
def: "Catalysis of the reaction: H2O + maltose 6'-phosphate = D-glucose + D-glucose 6-phosphate." [EC:3.2.1.122, MetaCyc:MALTOSE-6-PHOSPHATE-GLUCOSIDASE-RXN]
GO:0050082
name: maltose phosphorylase activity
def: "Catalysis of the reaction: maltose + phosphate = D-glucose + beta-D-glucose 1-phosphate." [EC:2.4.1.8, MetaCyc:MALTOSE-PHOSPHORYLASE-RXN]
GO:0050083
name: malyl-CoA lyase activity
def: "Catalysis of the reaction: (3S)-3-carboxy-3-hydroxypropanoyl-CoA = acetyl-CoA + glyoxylate." [EC:4.1.3.24, MetaCyc:MALYL-COA-LYASE-RXN]
GO:0050084
name: mannitol-1-phosphatase activity
def: "Catalysis of the reaction: D-mannitol 1-phosphate + H(2)O = D-mannitol + 2 H(+) + phosphate." [EC:3.1.3.22, RHEA:19540]
GO:0050085
name: mannitol 2-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: D-mannitol + NADP(+) = D-fructose + H(+) + NADPH." [EC:1.1.1.138, RHEA:16768]
GO:0050086
name: mannitol 2-dehydrogenase activity
def: "Catalysis of the reaction: D-mannitol + NAD+ = D-fructose + NADH." [EC:1.1.1.67, MetaCyc:MANNITOL-2-DEHYDROGENASE-RXN]
GO:0050087
name: mannitol dehydrogenase (cytochrome) activity
def: "Catalysis of the reaction: D-mannitol + ferricytochrome c = D-fructose + ferrocytochrome c." [EC:1.1.2.2, MetaCyc:MANNITOL-DEHYDROGENASE-CYTOCHROME-RXN]
GO:0050088
name: mannose-6-phosphate 6-reductase activity
def: "Catalysis of the reaction: D-mannitol 1-phosphate + NADP(+) = D-mannose 6-phosphate + 3 H(+) + NADPH." [EC:1.1.1.224, RHEA:14928]
GO:0050089
name: mannose isomerase activity
def: "Catalysis of the reaction: D-mannose = D-fructose." [EC:5.3.1.7, RHEA:22607]
GO:0050090
name: mannuronate reductase activity
def: "Catalysis of the reaction: D-mannonate + NAD(P)+ = D-mannuronate + NAD(P)H + H+." [EC:1.1.1.131, MetaCyc:MANNURONATE-REDUCTASE-RXN]
GO:0050091
name: melilotate 3-monooxygenase activity
def: "Catalysis of the reaction: 3-(2-hydroxyphenyl)propanoate + H(+) + NADH + O(2) = 3-(2,3-dihydroxyphenyl)propanoate + H(2)O + NAD(+)." [EC:1.14.13.4, RHEA:17672]
GO:0050092
name: meso-tartrate dehydrogenase activity
def: "Catalysis of the reaction: (2R,3S)-tartrate + NAD(+) = dihydroxyfumarate + H(+) + NADH." [EC:1.3.1.7, RHEA:18556]
GO:0050093
name: methanol dehydrogenase activity
def: "Catalysis of the reaction: methanol + NAD(+) = formaldehyde + H(+) + NADH." [EC:1.1.1.244, RHEA:19404]
GO:0050094
name: methionine-glyoxylate transaminase activity
def: "Catalysis of the reaction: L-methionine + glyoxylate = 4-methylthio-2-oxobutanoate + glycine." [EC:2.6.1.73, RHEA:22887]
GO:0050095
name: methionine decarboxylase activity
def: "Catalysis of the reaction: L-methionine + H(+) = 3-methylthiopropanamine + CO(2)." [EC:4.1.1.57, RHEA:17760]
GO:0050096
name: methylaspartate ammonia-lyase activity
def: "Catalysis of the reaction: threo-3-methyl-L-aspartate = mesaconate + NH(4)(+)." [EC:4.3.1.2, RHEA:12832]
GO:0050097
name: methylaspartate mutase activity
def: "Catalysis of the reaction: threo-3-methyl-L-aspartate = L-glutamate." [EC:5.4.99.1, RHEA:12860]
GO:0050098
name: methylguanidinase activity
def: "Catalysis of the reaction: H(2)O + methylguanidine = methylammonium + urea." [EC:3.5.3.16, RHEA:11767]
GO:0050099
name: methylglutamate dehydrogenase activity
def: "Catalysis of the reaction: N-methyl-L-glutamate + A + H(2)O = L-glutamate + AH(2) + formaldehyde." [EC:1.5.99.5, RHEA:22575]
GO:0050100
name: methylitaconate delta-isomerase activity
def: "Catalysis of the reaction: 2-methylene-3-methylsuccinate = dimethylmaleate." [EC:5.3.3.6, RHEA:23483]
GO:0050101
name: mimosinase activity
def: "Catalysis of the reaction: L-mimosine + H(2)O = 3-hydroxy-4H-pyrid-4-one + L-serine." [EC:3.5.1.61, RHEA:13376]
GO:0050102
name: cellodextrin phosphorylase activity
def: "Catalysis of the reaction: 1,4-beta-D-glucosyl(n) + phosphate = 1,4-beta-D-glucosyl(n-1) + alpha-D-glucose 1-phosphate." [EC:2.4.1.49, MetaCyc:CELLODEXTRIN-PHOSPHORYLASE-RXN]
GO:0050103
name: dextrin dextranase activity
def: "Catalysis of the reaction: 1,4-alpha-D-glucosyl(n) + 1,6-alpha-D-glucosyl(m) = 1,4-alpha-D-glucosyl(n-1) + 1,6-alpha-D-glucosyl(m+1)." [EC:2.4.1.2, MetaCyc:DEXTRIN-DEXTRANASE-RXN]
GO:0050104
name: L-gulonate 3-dehydrogenase activity
def: "Catalysis of the reaction: L-gulonate + NAD(+) = 3-dehydro-L-gulonate + H(+) + NADH." [EC:1.1.1.45, RHEA:12892]
GO:0050105
name: L-gulonolactone oxidase activity
def: "Catalysis of the reaction: L-gulono-1,4-lactone + O2 = L-xylo-hex-3-ulonolactone + H2O2." [EC:1.1.3.8, MetaCyc:L-GULONOLACTONE-OXIDASE-RXN]
GO:0050106
name: monomethyl-sulfatase activity
def: "Catalysis of the reaction: H(2)O + monomethyl sulfate = H(+) + methanol + sulfate." [EC:3.1.6.16, RHEA:14224]
GO:0050107
name: monoterpenol O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + a monoterpenol = CoA + a monoterpenol acetate ester." [EC:2.3.1.69, MetaCyc:MONOTERPENOL-O-ACETYLTRANSFERASE-RXN]
GO:0050108
name: monoterpenyl-diphosphatase activity
def: "Catalysis of the reaction: monoterpenyl diphosphate + H2O = monoterpenol + diphosphate." [EC:3.1.7.3, MetaCyc:MONOTERPENYL-PYROPHOSPHATASE-RXN]
GO:0050109
name: morphine 6-dehydrogenase activity
def: "Catalysis of the reaction: morphine + NAD(P)+ = morphinone + NAD(P)H + H+." [EC:1.1.1.218, MetaCyc:MORPHINE-6-DEHYDROGENASE-RXN]
GO:0050110
name: mucinaminylserine mucinaminidase activity
def: "Catalysis of the reaction: D-galactosyl-3-(N-acetyl-beta-D-galactosaminyl)-L-serine + H2O = D-galactosyl-3-N-acetyl-beta-D-galactosamine + L-serine." [MetaCyc:3.2.1.110-RXN]
GO:0050111
name: mycocerosate synthase activity
def: "Catalysis of the reaction: acyl-CoA + 7n H(+) + n methylmalonyl-CoA + 2n NADPH = n CO(2) + n CoA + n H(2)O + multi-methyl-branched acyl-CoA + 2n NADP(+)." [EC:2.3.1.111, RHEA:10591]
GO:0050112
name: inositol 2-dehydrogenase activity
def: "Catalysis of the reaction: myo-inositol + NAD(+) = 2,4,6/3,5-pentahydroxycyclohexanone + H(+) + NADH." [EC:1.1.1.18, RHEA:16952]
GO:0050113
name: inositol oxygenase activity
def: "Catalysis of the reaction: myo-inositol + O(2) = D-glucuronate + H(2)O + H(+)." [EC:1.13.99.1, RHEA:23699]
GO:0050114
name: myo-inosose-2 dehydratase activity
def: "Catalysis of the reaction: 2,4,6/3,5-pentahydroxycyclohexanone = 3D-3,5/4-trihydroxycyclohexane-1,2-dione + H(2)O." [EC:4.2.1.44, RHEA:14068]
GO:0050115
name: myosin-light-chain-phosphatase activity
def: "Catalysis of the reaction: myosin light-chain phosphate + H2O = myosin light chain + phosphate." [EC:3.1.3.53, MetaCyc:MYOSIN-LIGHT-CHAIN-PHOSPHATASE-RXN]
GO:0050116
name: N,N-dimethylformamidase activity
def: "Catalysis of the reaction: N,N-dimethylformamide + H(2)O = dimethylamine + formate." [EC:3.5.1.56, RHEA:19520]
GO:0050117
name: N-acetyl-beta-alanine deacetylase activity
def: "Catalysis of the reaction: N-acetyl-beta-alanine + H(2)O = beta-alanine + acetate." [EC:3.5.1.21, RHEA:23215]
GO:0050118
name: N-acetyldiaminopimelate deacetylase activity
def: "Catalysis of the reaction: N-acetyl-LL-2,6-diaminopimelate + H(2)O = LL-2,6-diaminopimelate + acetate." [EC:3.5.1.47, RHEA:20408]
GO:0050119
name: N-acetylglucosamine deacetylase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosamine + H(2)O = D-glucosamine + acetate." [EC:3.5.1.33, RHEA:20596]
GO:0050120
name: N-acetylhexosamine 1-dehydrogenase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosamine + H(2)O + NAD(+) = N-acetyl-D-glucosaminate + 2 H(+) + NADH." [EC:1.1.1.240, RHEA:23147]
GO:0050121
name: N-acylglucosamine 2-epimerase activity
def: "Catalysis of the reaction: N-acyl-D-glucosamine = N-acyl-D-mannosamine." [EC:5.1.3.8, MetaCyc:N-ACYLGLUCOSAMINE-2-EPIMERASE-RXN]
GO:0050122
name: N-acylhexosamine oxidase activity
def: "Catalysis of the reaction: N-acetyl-D-glucosamine + H(2)O + O(2) = N-acetyl-D-glucosaminate + H(2)O(2) + H(+)." [EC:1.1.3.29, RHEA:13032]
GO:0050123
name: N-acylmannosamine 1-dehydrogenase activity
def: "Catalysis of the reaction: N-acyl-D-mannosamine + NAD(+) = N-acyl-D-mannosaminolactone + H(+) + NADH." [EC:1.1.1.233, RHEA:11543]
GO:0050124
name: N-acylneuraminate-9-phosphatase activity
def: "Catalysis of the reaction: N-acylneuraminate 9-phosphate + H2O = N-acylneuraminate + phosphate." [EC:3.1.3.29, MetaCyc:N-ACYLNEURAMINATE-9-PHOSPHATASE-RXN]
GO:0050125
name: N-benzyloxycarbonylglycine hydrolase activity
def: "Catalysis of the reaction: N-benzyloxycarbonylglycine + H(2)O + H(+) = benzyl alcohol + CO(2) + glycine." [EC:3.5.1.58, RHEA:20903]
GO:0050126
name: N-carbamoylputrescine amidase activity
def: "Catalysis of the reaction: N-carbamoylputrescine + H(2)O + 2 H(+) = CO(2) + NH(4)(+) + putrescine." [EC:3.5.1.53, RHEA:22287]
GO:0050127
name: N-carbamoylsarcosine amidase activity
def: "Catalysis of the reaction: N-carbamoylsarcosine + H(2)O + 2 H(+) = CO(2) + NH(4)(+) + sarcosine." [EC:3.5.1.59, RHEA:20060]
GO:0050128
name: N-feruloylglycine deacylase activity
def: "Catalysis of the reaction: N-feruloylglycine + H(2)O = ferulate + glycine." [EC:3.5.1.71, RHEA:10487]
GO:0050129
name: N-formylglutamate deformylase activity
def: "Catalysis of the reaction: N-formyl-L-glutamate + H(2)O = L-glutamate + formate." [EC:3.5.1.68, RHEA:12479]
GO:0050130
name: N-methyl-2-oxoglutaramate hydrolase activity
def: "Catalysis of the reaction: N-methyl-2-oxoglutaramate + H(2)O = 2-oxoglutarate + methylammonium." [EC:3.5.1.36, RHEA:24111]
GO:0050131
name: N-methyl-L-amino-acid oxidase activity
def: "Catalysis of the reaction: an N-methyl-L-amino acid + H2O + O2 = an L-amino acid + formaldehyde + H2O2." [EC:1.5.3.2, MetaCyc:N-METHYL-L-AMINO-ACID-OXIDASE-RXN]
GO:0050132
name: N-methylalanine dehydrogenase activity
def: "Catalysis of the reaction: N-methyl-L-alanine + H(2)O + NADP(+) = H(+) + methylammonium + NADPH + pyruvate." [EC:1.4.1.17, RHEA:21771]
GO:0050133
name: N6-hydroxylysine O-acetyltransferase activity
def: "Catalysis of the reaction: N(6)-hydroxy-L-lysine + acetyl-CoA = N(6)-acetyl-N(6)-hydroxy-L-lysine + CoA." [EC:2.3.1.102, RHEA:22391]
GO:0050134
name: N6-methyl-lysine oxidase activity
def: "Catalysis of the reaction: N(6)-methyl-L-lysine + H(2)O + O(2) = L-lysine + formaldehyde + H(2)O(2)." [EC:1.5.3.4, RHEA:23203]
GO:0050135
name: NAD(P)+ nucleosidase activity
def: "Catalysis of the reaction: NAD(P)+ + H2O = ADP-ribose(P) + nicotinamide." [EC:3.2.2.6, MetaCyc:NADP+-NUCLEOSIDASE-RXN]
GO:0050136
name: NADH dehydrogenase (quinone) activity
def: "Catalysis of the reaction: NADH + H+ + a quinone = NAD+ + a quinol." [EC:1.6.99.5, GOC:mah, MetaCyc:NADH-DEHYDROGENASE-QUINONE-RXN]
GO:0050137
name: NADPH peroxidase activity
def: "Catalysis of the reaction: H(2)O(2) + H(+) + NADPH = 2 H(2)O + NADP(+)." [EC:1.11.1.2, RHEA:15176]
GO:0050138
name: nicotinate dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + NADP(+) + nicotinate = 6-hydroxynicotinate + H(+) + NADPH." [EC:1.17.1.5, RHEA:12239]
GO:0050139
name: nicotinate glucosyltransferase activity
def: "Catalysis of the reaction: nicotinate + UDP-D-glucose = N-(beta-D-glucosyl)nicotinate + UDP." [EC:2.4.1.196, RHEA:19440]
GO:0050140
name: nitrate reductase (cytochrome) activity
def: "Catalysis of the reaction: ferrocytochrome + nitrate = ferricytochrome + nitrite." [EC:1.9.6.1, MetaCyc:NITRATE-REDUCTASE-CYTOCHROME-RXN]
GO:0050141
name: nitroethane oxidase activity
def: "Catalysis of the reaction: nitroethane + H2O + O2 = acetaldehyde + nitrite + H2O2." [EC:1.7.3.1, MetaCyc:NITROETHANE-OXIDASE-RXN]
GO:0050142
name: nitrogenase (flavodoxin) activity
def: "Catalysis of the reaction: 6 reduced flavodoxin + 6 H+ + N2 + n ATP = 6 oxidized flavodoxin + 2 NH3 + n ADP + n phosphate." [EC:1.19.6.1, MetaCyc:NITROGENASE-FLAVODOXIN-RXN]
GO:0050143
name: nocardicin-A epimerase activity
def: "Catalysis of the reaction: isonocardicin A = nocardicin A." [EC:5.1.1.14, RHEA:22795]
GO:0050144
name: nucleoside deoxyribosyltransferase activity
def: "Catalysis of the reaction: 2-deoxy-D-ribosyl-base1 + base2 = 2-deoxy-D-ribosyl-base2 + base1." [EC:2.4.2.6, MetaCyc:NUCLEOSIDE-DEOXYRIBOSYLTRANSFERASE-RXN]
GO:0050145
name: nucleoside phosphate kinase activity
def: "Catalysis of the reaction: ATP + nucleoside phosphate = ADP + nucleoside diphosphate." [EC:2.7.4.4, MetaCyc:NUCLEOSIDE-PHOSPHATE-KINASE-RXN]
GO:0050146
name: nucleoside phosphotransferase activity
def: "Catalysis of the reaction: a nucleotide + a 2'-deoxynucleoside = a nucleoside + a 2'-deoxynucleoside 5'-monophosphate." [EC:2.7.1.77, MetaCyc:NUCLEOSIDE-PHOSPHOTRANSFERASE-RXN]
GO:0050147
name: nucleoside ribosyltransferase activity
def: "Catalysis of the reaction: D-ribosyl-base1 + base2 = D-ribosyl-base2 + base1." [EC:2.4.2.5, MetaCyc:NUCLEOSIDE-RIBOSYLTRANSFERASE-RXN]
GO:0050148
name: nucleotide diphosphokinase activity
def: "Catalysis of the reaction: ATP + nucleoside 5'-phosphate = AMP + 5'-phosphonucleoside 3'-diphosphate." [EC:2.7.6.4, MetaCyc:NUCLEOTIDE-PYROPHOSPHOKINASE-RXN]
GO:0050149
name: o-aminophenol oxidase activity
def: "Catalysis of the reaction: 2 2-aminophenol + 3 O2 = 2 isophenoxazine + 6 H2O." [EC:1.10.3.4, MetaCyc:O-AMINOPHENOL-OXIDASE-RXN]
GO:0050150
name: o-pyrocatechuate decarboxylase activity
def: "Catalysis of the reaction: 2,3-dihydroxybenzoate + H(+) = catechol + CO(2)." [EC:4.1.1.46, RHEA:21495]
GO:0050151
name: oleate hydratase activity
def: "Catalysis of the reaction: (R)-10-hydroxystearate = H(2)O + oleate." [EC:4.2.1.53, RHEA:21855]
GO:0050152
name: omega-amidase activity
def: "Catalysis of the reaction: a monoamide of a dicarboxylic acid + H2O = a dicarboxylate + NH3." [EC:3.5.1.3, MetaCyc:OMEGA-AMIDASE-RXN]
GO:0050153
name: omega-hydroxydecanoate dehydrogenase activity
def: "Catalysis of the reaction: 10-hydroxydecanoate + NAD(+) = 10-oxodecanoate + H(+) + NADH." [EC:1.1.1.66, RHEA:20883]
GO:0050154
name: opheline kinase activity
def: "Catalysis of the reaction: ATP + guanidinoethyl methyl phosphate = N'-phosphoguanidinoethyl methylphosphate + ADP + 2 H(+)." [EC:2.7.3.7, RHEA:17556]
GO:0050155
name: ornithine(lysine) transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-ornithine = H2O + L-glutamate + 3,4-dihydro-2H-pyrrole-2-carboxylate." [EC:2.6.1.68, MetaCyc:ORNITHINELYSINE-AMINOTRANSFERASE-RXN]
GO:0050156
name: ornithine N-benzoyltransferase activity
def: "Catalysis of the reaction: L-ornithine + 2 benzoyl-CoA = N(2),N(5)-dibenzoyl-L-ornithine + 2 CoA + 2 H(+)." [EC:2.3.1.127, RHEA:16932]
GO:0050157
name: ornithine racemase activity
def: "Catalysis of the reaction: L-ornithine = D-ornithine." [EC:5.1.1.12, MetaCyc:ORNITHINE-RACEMASE-RXN]
GO:0050158
name: orotate reductase (NADPH) activity
def: "Catalysis of the reaction: (S)-dihydroorotate + NADP(+) = H(+) + NADPH + orotate." [EC:1.3.1.15, RHEA:14864]
GO:0050159
name: orsellinate decarboxylase activity
def: "Catalysis of the reaction: o-orsellinate + H(+) = CO(2) + orcinol." [EC:4.1.1.58, RHEA:16736]
GO:0050160
name: orsellinate-depside hydrolase activity
def: "Catalysis of the reaction: H(2)O + orsellinate depside = 2 o-orsellinate + H(+)." [EC:3.1.1.40, RHEA:19552]
GO:0050161
name: succinyl-CoA:oxalate CoA-transferase
def: "Catalysis of the reaction: oxalate + succinyl-CoA = oxalyl-CoA + succinate." [EC:2.8.3.2, RHEA:23591]
GO:0050162
name: oxalate oxidase activity
def: "Catalysis of the reaction: 2 H(+) + O(2) + oxalate = 2 CO(2) + H(2)O(2)." [EC:1.2.3.4, RHEA:21881]
GO:0050163
name: oxaloacetate tautomerase activity
def: "Catalysis of the reaction: oxaloacetate = <stereo>enol</stereo>-oxaloacetate." [EC:5.3.2.2, RHEA:16024]
GO:0050164
name: oxoglutarate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: 2-oxoglutarate + CoA + NADP(+) = CO(2) + NADPH + succinyl-CoA." [EC:1.2.1.52, RHEA:21403]
GO:0050165
name: pantetheine kinase activity
def: "Catalysis of the reaction: ATP + pantetheine = ADP + pantetheine 4'-phosphate." [EC:2.7.1.34, MetaCyc:PANTETHEINE-KINASE-RXN]
GO:0050166
name: pantoate 4-dehydrogenase activity
def: "Catalysis of the reaction: (R)-pantoate + NAD(+) = (R)-4-dehydropantoate + H(+) + NADH." [EC:1.1.1.106, RHEA:23003]
GO:0050167
name: pantothenoylcysteine decarboxylase activity
def: "Catalysis of the reaction: N-[(R)-pantothenoyl]-L-cysteine + H(+) = (R)-pantetheine + CO(2)." [EC:4.1.1.30, RHEA:15080]
GO:0050168
name: pentanamidase activity
def: "Catalysis of the reaction: H(2)O + pentanamide = NH(4)(+) + valerate." [EC:3.5.1.50, RHEA:10003]
GO:0050169
name: peptide-tryptophan 2,3-dioxygenase activity
def: "Catalysis of the reaction: peptide tryptophan + O2 = peptide formylkynurenine." [EC:1.13.11.26, MetaCyc:PEPTIDE-TRYPTOPHAN-23-DIOXYGENASE-RXN]
GO:0050170
name: peptidyl-glutaminase activity
def: "Catalysis of the reaction: alpha-N-peptidyl-L-glutamine + H2O = alpha-N-peptidyl-L-glutamate + NH3." [EC:3.5.1.43, MetaCyc:PEPTIDYL-GLUTAMINASE-RXN]
GO:0050171
name: phenol beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a phenol = UDP + an aryl beta-D-glucoside." [EC:2.4.1.35, MetaCyc:PHENOL-BETA-GLUCOSYLTRANSFERASE-RXN]
GO:0050172
name: phenylalanine 2-monooxygenase activity
def: "Catalysis of the reaction: L-phenylalanine + O(2) = 2-phenylacetamide + CO(2) + H(2)O." [EC:1.13.12.9, RHEA:10715]
GO:0050173
name: phenylalanine adenylyltransferase activity
def: "Catalysis of the reaction: L-phenylalanine + ATP = N-adenylyl-L-phenylalanine + diphosphate + 2 H(+)." [EC:2.7.7.54, RHEA:17192]
GO:0050174
name: phenylalanine decarboxylase activity
def: "Catalysis of the reaction: L-phenylalanine = phenylethylamine + CO2." [EC:4.1.1.53, MetaCyc:PHENYLALANINE-DECARBOXYLASE-RXN]
GO:0050175
name: phenylalanine dehydrogenase activity
def: "Catalysis of the reaction: L-phenylalanine + H2O + NAD+ = phenylpyruvate + NH3 + NADH." [EC:1.4.1.20, MetaCyc:PHENYLALANINE-DEHYDROGENASE-RXN]
GO:0050176
name: phenylalanine N-acetyltransferase activity
def: "Catalysis of the reaction: L-phenylalanine + acetyl-CoA = N-acetyl-L-phenylalanine + CoA + H(+)." [EC:2.3.1.53, RHEA:17804]
GO:0050177
name: phenylpyruvate decarboxylase activity
def: "Catalysis of the reaction: phenylpyruvate = phenylacetaldehyde + CO2." [EC:4.1.1.43, MetaCyc:PHENYLPYRUVATE-DECARBOXYLASE-RXN]
GO:0050178
name: phenylpyruvate tautomerase activity
def: "Catalysis of the reaction: keto-phenylpyruvate = enol-phenylpyruvate." [EC:5.3.2.1, MetaCyc:PHENYLPYRUVATE-TAUTOMERASE-RXN]
GO:0050179
name: phenylserine aldolase activity
def: "Catalysis of the reaction: L-threo-3-phenylserine = benzaldehyde + glycine." [EC:4.1.2.26, RHEA:21715]
GO:0050180
name: phloretin hydrolase activity
def: "Catalysis of the reaction: H(2)O + phloretin = H(+) + phloretate + phloroglucinol." [EC:3.7.1.4, RHEA:23399]
GO:0050181
name: phorbol-diester hydrolase activity
def: "Catalysis of the reaction: H(2)O + phorbol 12,13-dibutanoate = butanoate + H(+) + phorbol 13-butanoate." [EC:3.1.1.51, RHEA:21319]
GO:0050182
name: phosphate butyryltransferase activity
def: "Catalysis of the reaction: butanoyl-CoA + phosphate = butanoyl phosphate + CoA." [EC:2.3.1.19, RHEA:20895]
GO:0050183
name: phosphatidylcholine 12-monooxygenase activity
def: "Catalysis of the reaction: 1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + H(+) + NADH + O(2) = 1-acyl-2-[(S)-12-hydroxyoleoyl]-sn-glycero-3-phosphocholine + H(2)O + NAD(+)." [EC:1.14.13.26, RHEA:11371]
GO:0050184
name: phosphatidylcholine desaturase activity
def: "Catalysis of the reaction: 1-acyl-2-oleoyl-sn-glycero-3-phosphocholine + NAD(+) = 1-acyl-2-linoleoyl-sn-glycero-3-phosphocholine + H(+) + NADH." [EC:1.3.1.35, RHEA:12567]
GO:0050185
name: phosphatidylinositol deacylase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol + H(2)O = 1-acyl-sn-glycero-3-phospho-D-myo-inositol + a carboxylate + H(+)." [EC:3.1.1.52, RHEA:18004]
GO:0050186
name: phosphoadenylylsulfatase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + H2O = adenosine 3',5'-bisphosphate + sulfate." [EC:3.6.2.2, MetaCyc:PHOSPHOADENYLYLSULFATASE-RXN]
GO:0050187
name: phosphoamidase activity
def: "Catalysis of the reaction: N-phosphocreatine + H(2)O = creatine + phosphate." [EC:3.9.1.1, RHEA:12980]
GO:0050188
name: phosphoenolpyruvate mutase activity
def: "Catalysis of the reaction: phosphoenolpyruvate = 3-phosphonopyruvate." [EC:5.4.2.9, RHEA:17016]
GO:0050189
name: phosphoenolpyruvate phosphatase activity
def: "Catalysis of the reaction: H(2)O + phosphoenolpyruvate = phosphate + pyruvate." [EC:3.1.3.60, RHEA:20000]
GO:0050190
name: phosphoglucokinase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + ATP = alpha-D-glucose 1,6-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.10, RHEA:13380]
GO:0050191
name: phosphoglycerate kinase (GTP) activity
def: "Catalysis of the reaction: 3-phospho-D-glycerate + GTP = 3-phospho-D-glyceroyl phosphate + GDP + H(+)." [EC:2.7.2.10, RHEA:23335]
GO:0050192
name: phosphoglycerate phosphatase activity
def: "Catalysis of the reaction: 2-phospho-D-glycerate + H(2)O = D-glycerate + phosphate." [EC:3.1.3.20, RHEA:21159]
GO:0050193
name: phosphoketolase activity
def: "Catalysis of the reaction: D-xylulose 5-phosphate + phosphate = acetyl phosphate + D-glyceraldehyde 3-phosphate + H2O." [EC:4.1.2.9, MetaCyc:PHOSPHOKETOLASE-RXN]
GO:0050194
name: phosphonoacetaldehyde hydrolase activity
def: "Catalysis of the reaction: H(2)O + phosphonoacetaldehyde = acetaldehyde + H(+) + phosphate." [EC:3.11.1.1, RHEA:18908]
GO:0050195
name: phosphoribokinase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + ATP = D-ribose 1,5-diphosphate + ADP + 2 H(+)." [EC:2.7.1.18, RHEA:21219]
GO:0050196
name: [phosphorylase] phosphatase activity
def: "Catalysis of the reaction: [phosphorylase a] + 4 H2O = 2 [phosphorylase b] + 4 phosphate." [EC:3.1.3.17, MetaCyc:PHOSPHORYLASE-PHOSPHATASE-RXN]
GO:0050197
name: phytanate-CoA ligase activity
def: "Catalysis of the reaction: ATP + CoA + phytanate = AMP + diphosphate + H(+) + phytanoyl-CoA." [EC:6.2.1.24, RHEA:21383]
GO:0050198
name: pinosylvin synthase activity
def: "Catalysis of the reaction: trans-cinnamoyl-CoA + 3 H(+) + 3 malonyl-CoA = 4 CO(2) + 4 CoA + pinosylvin." [EC:2.3.1.146, RHEA:12555]
GO:0050199
name: piperidine N-piperoyltransferase activity
def: "Catalysis of the reaction: (E,E)-piperoyl-CoA + piperidine = N-[(E,E)-piperoyl]piperidine + CoA + H(+)." [EC:2.3.1.145, RHEA:14564]
GO:0050200
name: plasmalogen synthase activity
def: "Catalysis of the reaction: acyl-CoA + 1-O-alk-1-enyl-glycero-3-phosphocholine = CoA + plasmenylcholine." [EC:2.3.1.25, MetaCyc:PLASMALOGEN-SYNTHASE-RXN]
GO:0050201
name: fucokinase activity
def: "Catalysis of the reaction: L-fucose + ATP = beta-L-fucose 1-phosphate + ADP + 2 H(+)." [EC:2.7.1.52, RHEA:13244]
GO:0050202
name: octopamine dehydratase activity
def: "Catalysis of the reaction: 1-(4-hydroxyphenyl)-2-aminoethanol = (4-hydroxyphenyl)acetaldehyde + NH(4)(+)." [EC:4.2.1.87, RHEA:18176]
GO:0050203
name: oxalate-CoA ligase activity
def: "Catalysis of the reaction: ATP + CoA + oxalate = AMP + diphosphate + H(+) + oxalyl-CoA." [EC:6.2.1.8, RHEA:18296]
GO:0050204
name: oxalomalate lyase activity
def: "Catalysis of the reaction: 3-oxalomalate = glyoxylate + oxaloacetate." [EC:4.1.3.13, RHEA:22035]
GO:0050205
name: oxamate carbamoyltransferase activity
def: "Catalysis of the reaction: carbamoyl phosphate + oxamate = oxalurate + phosphate." [EC:2.1.3.5, RHEA:22987]
GO:0050206
name: oximinotransferase activity
def: "Catalysis of the reaction: 2-(hydroxyimino)propanoate + acetone = acetone oxime + pyruvate." [EC:2.6.3.1, RHEA:11627]
GO:0050207
name: plasmanylethanolamine desaturase activity
def: "Catalysis of the reaction: O-1-alkyl-2-acyl-sn-glycero-3-phosphoethanolamine + donor-H2 + O2 = O-1-alk-1-enyl-2-acyl-sn-glycero-3-phosphoethanolamine + acceptor + 2 H2O." [EC:1.14.99.19, MetaCyc:PLASMANYLETHANOLAMINE-DESATURASE-RXN]
GO:0050208
name: polysialic-acid O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + an alpha-2,8-linked polymer of sialic acid = CoA + polysialic acid acetylated at O-7 or O-9." [EC:2.3.1.136, MetaCyc:POLYSIALIC-ACID-O-ACETYLTRANSFERASE-RXN]
GO:0050209
name: polyvinyl-alcohol oxidase activity
def: "Catalysis of the reaction: polyvinyl alcohol + O2 = oxidized polyvinyl alcohol + H2O2." [EC:1.1.3.30, MetaCyc:POLYVINYL-ALCOHOL-OXIDASE-RXN]
GO:0050210
name: prenyl-diphosphatase activity
def: "Catalysis of the reaction: prenyl diphosphate + H2O = prenol + diphosphate." [EC:3.1.7.1, MetaCyc:PRENYL-PYROPHOSPHATASE-RXN]
GO:0050211
name: procollagen galactosyltransferase activity
def: "Catalysis of the reaction: UDP-galactose + procollagen 5-hydroxy-L-lysine = UDP + procollagen 5-(D-galactosyloxy)-L-lysine." [EC:2.4.1.50, MetaCyc:PROCOLLAGEN-GALACTOSYLTRANSFERASE-RXN]
GO:0050212
name: progesterone 11-alpha-monooxygenase activity
def: "Catalysis of the reaction: AH(2) + O(2) + progesterone = 11alpha-hydroxyprogesterone + A + H(2)O." [EC:1.14.99.14, RHEA:18208]
GO:0050213
name: progesterone 5-alpha-reductase activity
def: "Catalysis of the reaction: 5-alpha-pregnan-3,20-dione + NADP+ = progesterone + NADPH." [EC:1.3.1.30, MetaCyc:PROGESTERONE-5-ALPHA-REDUCTASE-RXN]
GO:0050214
name: progesterone monooxygenase activity
def: "Catalysis of the reaction: AH(2) + O(2) + progesterone = A + H(2)O + testosterone acetate." [EC:1.14.99.4, RHEA:11987]
GO:0050215
name: propanediol dehydratase activity
def: "Catalysis of the reaction: propane-1,2-diol = H(2)O + propanal." [EC:4.2.1.28, RHEA:14572]
GO:0050216
name: propanediol-phosphate dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + propane-1,2-diol 1-phosphate = H(+) + hydroxyacetone phosphate + NADH." [EC:1.1.1.7, RHEA:21587]
GO:0050217
name: propioin synthase activity
def: "Catalysis of the reaction: 4-hydroxyhexan-3-one = 2 propanal." [EC:4.1.2.35, RHEA:11103]
GO:0050218
name: propionate-CoA ligase activity
def: "Catalysis of the reaction: ATP + propanoate + CoA = AMP + diphosphate + propanoyl-CoA." [EC:6.2.1.17, MetaCyc:PROPIONATE--COA-LIGASE-RXN]
GO:0050219
name: prostaglandin-A1 delta-isomerase activity
def: "Catalysis of the reaction: prostaglandin A1 = prostaglandin C1." [EC:5.3.3.9, RHEA:10463]
GO:0050220
name: prostaglandin-E synthase activity
def: "Catalysis of the reaction: prostaglandin H(2) = prostaglandin E(2)." [EC:5.3.99.3, RHEA:12896]
GO:0050221
name: prostaglandin-E2 9-reductase activity
def: "Catalysis of the reaction: (5Z,13E)-(15S)-9-alpha,11-alpha,15-trihydroxyprosta-5,13-dienoate + NADP+ = (5Z,13E)-(15S)-11-alpha,15-dihydroxy-9-oxoprosta-5,13-dienoate + NADPH." [EC:1.1.1.189, MetaCyc:PROSTAGLANDIN-E2-9-REDUCTASE-RXN]
GO:0050223
name: protocatechuate decarboxylase activity
def: "Catalysis of the reaction: 3,4-dihydroxybenzoate + H(+) = catechol + CO(2)." [EC:4.1.1.63, RHEA:22419]
GO:0050224
name: prunasin beta-glucosidase activity
def: "Catalysis of the reaction: (R)-prunasin + H(2)O = D-glucose + mandelonitrile." [EC:3.2.1.118, RHEA:16492]
GO:0050225
name: pseudouridine kinase activity
def: "Catalysis of the reaction: ATP + pseudouridine = ADP + 2 H(+) + pseudouridine 5'-phosphate." [EC:2.7.1.83, RHEA:22451]
GO:0050226
name: psychosine sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + galactosylsphingosine = adenosine 3',5'-bisphosphate + psychosine sulfate." [EC:2.8.2.13, MetaCyc:PSYCHOSINE-SULFOTRANSFERASE-RXN]
GO:0050227
name: pteridine oxidase activity
def: "Catalysis of the reaction: 2-amino-4-hydroxypteridine + O2 = 2-amino-4,7-dihydroxypteridine + unknown." [EC:1.17.3.1, MetaCyc:PTERIDINE-OXIDASE-RXN]
GO:0050228
name: pterin deaminase activity
def: "Catalysis of the reaction: 2-amino-4-hydroxypteridine + H2O = 2,4-dihydroxypteridine + NH3." [EC:3.5.4.11, MetaCyc:PTERIN-DEAMINASE-RXN]
GO:0050229
name: pterocarpin synthase activity
def: "Catalysis of the reaction: medicarpin + NADP+ = vestitone + NADPH." [EC:1.1.1.246, MetaCyc:PTEROCARPIN-SYNTHASE-RXN]
GO:0050230
name: purine imidazole-ring cyclase activity
def: "Catalysis of the reaction: DNA 4,6-diamino-5-formamidopyrimidine = DNA adenine + H2O." [EC:4.3.2.4, MetaCyc:PURINE-IMIDAZOLE-RING-CYCLASE-RXN]
GO:0050231
name: putrescine carbamoyltransferase activity
def: "Catalysis of the reaction: carbamoyl phosphate + putrescine = N-carbamoylputrescine + H(+) + phosphate." [EC:2.1.3.6, RHEA:21939]
GO:0050232
name: putrescine oxidase activity
def: "Catalysis of the reaction: putrescine + O2 + H2O = 4-aminobutanal + NH3 + H2O2." [EC:1.4.3.10, MetaCyc:PUTRESCINE-OXIDASE-RXN]
GO:0050233
name: pyranose oxidase activity
def: "Catalysis of the reaction: D-glucose + O(2) = 2-dehydro-D-glucose + H(2)O(2)." [EC:1.1.3.10, RHEA:10555]
GO:0050234
name: pyrazolylalanine synthase activity
def: "Catalysis of the reaction: L-serine + pyrazole = 3-(pyrazol-1-yl)-L-alanine + H(2)O." [EC:4.2.1.50, RHEA:24515]
GO:0050235
name: pyridoxal 4-dehydrogenase activity
def: "Catalysis of the reaction: NAD(+) + pyridoxal = 4-pyridoxolactone + H(+) + NADH." [EC:1.1.1.107, RHEA:21339]
GO:0050236
name: pyridoxine:NADP 4-dehydrogenase activity
def: "Catalysis of the reaction: NADP(+) + pyridoxine = H(+) + NADPH + pyridoxal." [EC:1.1.1.65, RHEA:16132]
GO:0050237
name: pyridoxine 4-oxidase activity
def: "Catalysis of the reaction: pyridoxine + O2 = pyridoxal + H2O2." [EC:1.1.3.12, MetaCyc:PYRIDOXINE-4-OXIDASE-RXN]
GO:0050238
name: pyridoxine 5-dehydrogenase activity
def: "Catalysis of the reaction: pyridoxine + acceptor = isopyridoxal + reduced acceptor." [EC:1.1.99.9, MetaCyc:PYRIDOXINE-5-DEHYDROGENASE-RXN]
GO:0050239
name: pyrithiamine deaminase activity
def: "Catalysis of the reaction: 1-(4-amino-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + H2O = 1-(4-hydroxy-2-methylpyrimid-5-ylmethyl)-3-(beta-hydroxyethyl)-2-methylpyridinium bromide + NH3." [EC:3.5.4.20, MetaCyc:PYRITHIAMIN-DEAMINASE-RXN]
GO:0050240
name: pyrogallol 1,2-oxygenase activity
def: "Catalysis of the reaction: O(2) + pyrogallol = (Z)-5-oxohex-2-enedioate + 2 H(+)." [EC:1.13.11.35, RHEA:19676]
GO:0050241
name: pyrroline-2-carboxylate reductase activity
def: "Catalysis of the reaction: L-proline + NAD(P)+ = 1-pyrroline-2-carboxylate + NAD(P)H + H+." [EC:1.5.1.1, MetaCyc:PYRROLINE-2-CARBOXYLATE-REDUCTASE-RXN]
GO:0050242
name: pyruvate, phosphate dikinase activity
def: "Catalysis of the reaction: ATP + phosphate + pyruvate = AMP + diphosphate + 2 H(+) + phosphoenolpyruvate." [EC:2.7.9.1, RHEA:10759]
GO:0050243
name: pyruvate dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: CoA + NADP(+) + pyruvate = acetyl-CoA + CO(2) + NADPH." [EC:1.2.1.51, RHEA:17428]
GO:0050244
name: pyruvate oxidase (CoA-acetylating) activity
def: "Catalysis of the reaction: CoA + H(+) + O(2) + pyruvate = acetyl-CoA + CO(2) + H(2)O(2)." [EC:1.2.3.6, RHEA:21915]
GO:0050245
name: quercitrinase activity
def: "Catalysis of the reaction: H(2)O + quercitrin = L-rhamnose + quercetin." [EC:3.2.1.66, RHEA:17468]
GO:0050246
name: questin monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + questin = demethylsulochrin + NADP(+)." [EC:1.14.13.43, RHEA:10839]
GO:0050247
name: raucaffricine beta-glucosidase activity
def: "Catalysis of the reaction: H(2)O + raucaffricine = D-glucose + vomilenine." [EC:3.2.1.125, RHEA:14560]
GO:0050248
name: Renilla-luciferin 2-monooxygenase activity
def: "Catalysis of the reaction: Renilla luciferin + O2 = oxidized Renilla luciferin + CO2 + light." [EC:1.13.12.5, MetaCyc:RENILLA-LUCIFERIN-2-MONOOXYGENASE-RXN]
GO:0050249
name: Renilla-luciferin sulfotransferase activity
def: "Catalysis of the reaction: 3'-phospho-5'-adenylyl sulfate + Renilla luciferin = adenosine 3',5'-diphosphate + H(+) + luciferyl sulfate." [EC:2.8.2.10, RHEA:20484]
GO:0050250
name: retinal oxidase activity
def: "Catalysis of the reaction: retinal + O2 + H2O = retinoate + H2O2." [EC:1.2.3.11, MetaCyc:RETINAL-OXIDASE-RXN]
GO:0050251
name: retinol isomerase activity
def: "Catalysis of the reaction: all-trans-retinol = 11-cis-retinol." [EC:5.2.1.7, RHEA:19144]
GO:0050252
name: retinol O-fatty-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + retinol = CoA + retinyl ester." [EC:2.3.1.76, MetaCyc:RETINOL-O-FATTY-ACYLTRANSFERASE-RXN]
GO:0050253
name: retinyl-palmitate esterase activity
def: "Catalysis of the reaction: retinyl palmitate + H2O = retinol + palmitate + H+." [MetaCyc:RETINYL-PALMITATE-ESTERASE-RXN, RHEA:21511]
GO:0050254
name: rhodopsin kinase activity
def: "Catalysis of the reaction: ATP + rhodopsin = ADP + phosphorhodopsin." [EC:2.7.11.14, MetaCyc:RHODOPSIN-KINASE-RXN]
GO:0050255
name: ribitol 2-dehydrogenase activity
def: "Catalysis of the reaction: D-ribitol + NAD(+) = D-ribulose + H(+) + NADH." [EC:1.1.1.56, RHEA:20056]
GO:0050256
name: ribitol-5-phosphate 2-dehydrogenase activity
def: "Catalysis of the reaction: D-ribitol 5-phosphate + NAD(P)+ = D-ribulose 5-phosphate + NAD(P)H + H+." [EC:1.1.1.137, MetaCyc:RIBITOL-5-PHOSPHATE-2-DEHYDROGENASE-RXN]
GO:0050257
name: riboflavin phosphotransferase activity
def: "Catalysis of the reaction: alpha-D-glucose 1-phosphate + riboflavin = D-glucose + FMN." [EC:2.7.1.42, RHEA:20412]
GO:0050258
name: riboflavinase activity
def: "Catalysis of the reaction: H(2)O + H(+) + riboflavin = D-ribitol + lumichrome." [EC:3.5.99.1, RHEA:11411]
GO:0050259
name: ribose 1-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: H(2)O + NADP(+) + ribofuranose = D-ribonate + 2 H(+) + NADPH." [EC:1.1.1.115, RHEA:11679]
GO:0050260
name: ribose-5-phosphate-ammonia ligase activity
def: "Catalysis of the reaction: D-ribose 5-phosphate + ATP + NH(4)(+) = 5-phospho-D-ribosylamine + ADP + 2 H(+) + phosphate." [EC:6.3.4.7, RHEA:13780]
GO:0050261
name: ribose isomerase activity
def: "Catalysis of the reaction: ribofuranose = D-ribulose." [EC:5.3.1.20, RHEA:20799]
GO:0050262
name: ribosylnicotinamide kinase activity
def: "Catalysis of the reaction: N-ribosylnicotinamide + ATP = ADP + 2 H(+) + nicotinamide mononucleotide." [EC:2.7.1.22, RHEA:14020]
GO:0050263
name: ribosylpyrimidine nucleosidase activity
def: "Catalysis of the reaction: an N-D-ribosylpyrimidine + H2O = D-ribose + a pyrimidine." [EC:3.2.2.8, MetaCyc:RIBOSYLPYRIMIDINE-NUCLEOSIDASE-RXN]
GO:0050264
name: rifamycin-B oxidase activity
def: "Catalysis of the reaction: 2 H(+) + O(2) + rifamycin B = H(2)O(2) + rifamycin O." [EC:1.10.3.6, RHEA:11295]
GO:0050265
name: RNA uridylyltransferase activity
def: "Catalysis of the reaction: UTP + RNA(n) = diphosphate + RNA(n+1)." [EC:2.7.7.52, MetaCyc:RNA-URIDYLYLTRANSFERASE-RXN]
GO:0050266
name: rosmarinate synthase activity
def: "Catalysis of the reaction: caffeoyl-CoA + 3-(3,4-dihydroxyphenyl)lactate = CoA + rosmarinate." [EC:2.3.1.140, MetaCyc:ROSMARINATE-SYNTHASE-RXN]
GO:0050267
name: rubber cis-polyprenylcistransferase activity
def: "Catalysis of the reaction: poly-cis-polyprenyl diphosphate + isopentenyl diphosphate = diphosphate + a poly-cis-polyprenyl diphosphate longer by one C5 unit." [EC:2.5.1.20, MetaCyc:RUBBER-CIS-POLYPRENYLCISTRANSFERASE-RXN]
GO:0050268
name: coniferyl-alcohol dehydrogenase activity
def: "Catalysis of the reaction: coniferyl alcohol + NADP+ = coniferyl aldehyde + NADPH." [EC:1.1.1.194, MetaCyc:RXN-1107]
GO:0050269
name: coniferyl-aldehyde dehydrogenase activity
def: "Catalysis of the reaction: coniferyl aldehyde + H2O + NAD(P)+ = ferulate + NAD(P)H + H+." [EC:1.2.1.68, MetaCyc:RXN-1241]
GO:0050270
name: S-adenosylhomocysteine deaminase activity
def: "Catalysis of the reaction: S-adenosyl-L-homocysteine + H(2)O + H(+) = S-inosyl-L-homocysteine + NH(4)(+)." [EC:3.5.4.28, RHEA:20719]
GO:0050271
name: S-alkylcysteine lyase activity
def: "Catalysis of the reaction: an S-alkyl-L-cysteine + H2O = an alkyl thiol + NH3 + pyruvate." [EC:4.4.1.6, MetaCyc:S-ALKYLCYSTEINE-LYASE-RXN]
GO:0050272
name: S-carboxymethylcysteine synthase activity
def: "Catalysis of the reaction: 3-chloro-L-alanine + thioglycolate = S-carboxymethyl-L-cysteine + chloride + H(+)." [EC:4.5.1.5, RHEA:22871]
GO:0050273
name: S-succinylglutathione hydrolase activity
def: "Catalysis of the reaction: S-succinylglutathione + H(2)O = glutathione + H(+) + succinate." [EC:3.1.2.13, RHEA:16716]
GO:0050274
name: salicyl-alcohol beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: salicyl alcohol + UDP-D-glucose = H(+) + salicin + UDP." [EC:2.4.1.172, RHEA:11515]
GO:0050275
name: scopoletin glucosyltransferase activity
def: "Catalysis of the reaction: scopoletin + UDP-D-glucose = H(+) + scopolin + UDP." [EC:2.4.1.128, RHEA:20456]
GO:0050276
name: scyllo-inosamine 4-kinase activity
def: "Catalysis of the reaction: 1-amino-1-deoxy-scyllo-inositol + ATP = 1-amino-1-deoxy-scyllo-inositol 4-phosphate + ADP + 2 H(+)." [EC:2.7.1.65, RHEA:18608]
GO:0050277
name: sedoheptulokinase activity
def: "Catalysis of the reaction: ATP + sedoheptulose = ADP + 2 H(+) + sedoheptulose 7-phosphate." [EC:2.7.1.14, RHEA:23847]
GO:0050278
name: sedoheptulose-bisphosphatase activity
def: "Catalysis of the reaction: sedoheptulose 1,7-bisphosphate + H2O = sedoheptulose 7-phosphate + phosphate." [EC:3.1.3.37, MetaCyc:SEDOHEPTULOSE-BISPHOSPHATASE-RXN]
GO:0050279
name: sepiapterin deaminase activity
def: "Catalysis of the reaction: sepiapterin + H2O = xanthopterin-B2 + NH3." [EC:3.5.4.24, MetaCyc:SEPIAPTERIN-DEAMINASE-RXN]
GO:0050280
name: sequoyitol dehydrogenase activity
def: "Catalysis of the reaction: 1D-5-O-methyl-myo-inositol + NAD(+) = 2D-5-O-methyl-2,3,5/4,6-pentahydroxycyclohexanone + H(+) + NADH." [EC:1.1.1.143, RHEA:11303]
GO:0050281
name: serine-glyoxylate transaminase activity
def: "Catalysis of the reaction: L-serine + glyoxylate = 3-hydroxypyruvate + glycine." [EC:2.6.1.45, MetaCyc:SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN]
GO:0050282
name: serine 2-dehydrogenase activity
def: "Catalysis of the reaction: L-serine + H2O + NAD+ = 3-hydroxypyruvate + NH3 + NADH." [EC:1.4.1.7, MetaCyc:SERINE-DEHYDROGENASE-RXN]
GO:0050283
name: serine-sulfate ammonia-lyase activity
def: "Catalysis of the reaction: L-serine O-sulfate + H2O = pyruvate + NH3 + sulfate." [EC:4.3.1.10, MetaCyc:SERINE-SULFATE-AMMONIA-LYASE-RXN]
GO:0050284
name: sinapate 1-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + sinapate = UDP + 1-sinapoyl-D-glucose." [EC:2.4.1.120, MetaCyc:SINAPATE-1-GLUCOSYLTRANSFERASE-RXN]
GO:0050285
name: sinapine esterase activity
def: "Catalysis of the reaction: O-sinapoylcholine + H(2)O = choline + H(+) + sinapate." [EC:3.1.1.49, RHEA:10019]
GO:0050286
name: sorbitol-6-phosphatase activity
def: "Catalysis of the reaction: D-glucitol 6-phosphate + H(2)O = D-glucitol + phosphate." [EC:3.1.3.50, RHEA:24583]
GO:0050287
name: sorbose 5-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: L-sorbose + NADP(+) = 5-dehydro-D-fructose + H(+) + NADPH." [EC:1.1.1.123, RHEA:15004]
GO:0050288
name: sorbose dehydrogenase activity
def: "Catalysis of the reaction: L-sorbose + A = 5-dehydro-D-fructose + AH(2)." [EC:1.1.99.12, RHEA:14716]
GO:0050289
name: spermidine dehydrogenase activity
def: "Catalysis of the reaction: spermidine + acceptor + H2O = 1,3-diaminopropane + 4-aminobutanal + reduced acceptor." [EC:1.5.99.6, MetaCyc:SPERMIDINE-DEHYDROGENASE-RXN]
GO:0050290
name: sphingomyelin phosphodiesterase D activity
def: "Catalysis of the reaction: H(2)O + sphingomyelin = ceramide 1-phosphate + choline + H(+)." [EC:3.1.4.41, RHEA:20987]
GO:0050291
name: sphingosine N-acyltransferase activity
def: "Catalysis of the reaction: acyl-CoA + sphingosine = CoA + N-acylsphingosine." [EC:2.3.1.24, MetaCyc:SPHINGOSINE-N-ACYLTRANSFERASE-RXN, PMID:12069845]
GO:0050292
name: steroid 9-alpha-monooxygenase activity
def: "Catalysis of the reaction: AH(2) + O(2) + pregna-4,9(11)-diene-3,20-dione = 9,11alpha-epoxypregn-4-ene-3,20-dione + A + H(2)O." [EC:1.14.99.24, RHEA:19560]
GO:0050293
name: steroid-lactonase activity
def: "Catalysis of the reaction: H(2)O + testololactone = H(+) + testolate." [EC:3.1.1.37, RHEA:13724]
GO:0050294
name: steroid sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + a phenolic steroid = adenosine 3',5'-bisphosphate + steroid O-sulfate." [EC:2.8.2.15, MetaCyc:STEROID-SULFOTRANSFERASE-RXN]
GO:0050295
name: steryl-beta-glucosidase activity
def: "Catalysis of the reaction: cholesteryl-beta-D-glucoside + H(2)O = D-glucose + cholesterol." [EC:3.2.1.104, RHEA:11959]
GO:0050296
name: stipitatonate decarboxylase activity
def: "Catalysis of the reaction: H(2)O + stipitatonate = CO(2) + H(+) + stipitatate." [EC:4.1.1.60, RHEA:13888]
GO:0050297
name: stizolobate synthase activity
def: "Catalysis of the reaction: 3,4-dihydroxy-L-phenylalanine + O2 = 4-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde." [EC:1.13.11.29, MetaCyc:STIZOLOBATE-SYNTHASE-RXN]
GO:0050298
name: stizolobinate synthase activity
def: "Catalysis of the reaction: 3,4-dihydroxy-L-phenylalanine + O2 = 5-(L-alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde." [EC:1.13.11.30, MetaCyc:STIZOLOBINATE-SYNTHASE-RXN]
GO:0050299
name: streptomycin 3''-kinase activity
def: "Catalysis of the reaction: ATP + streptomycin = ADP + 2 H(+) + streptomycin 3''-phosphate." [EC:2.7.1.87, RHEA:18380]
GO:0050300
name: aminoglycoside 6-kinase activity
def: "Catalysis of the reaction: ATP + streptomycin = ADP + 2 H(+) + streptomycin 6-phosphate." [EC:2.7.1.72, RHEA:22271]
GO:0050301
name: streptomycin-6-phosphatase activity
def: "Catalysis of the reaction: H(2)O + streptomycin 6-phosphate = phosphate + streptomycin." [EC:3.1.3.39, RHEA:10691]
GO:0050302
name: indole-3-acetaldehyde oxidase activity
def: "Catalysis of the reaction: (indol-3-yl)acetaldehyde + H(2)O + O(2) = (indol-3-yl)acetate + H(2)O(2) + H(+)." [EC:1.2.3.7, RHEA:16280]
GO:0050303
name: lysine 6-dehydrogenase activity
def: "Catalysis of the reaction: H2O + NAD+ + L-lysine = NH3 + NADH + allysine." [EC:1.4.1.18, MetaCyc:LYSINE-6-DEHYDROGENASE-RXN]
GO:0050304
name: nitrous-oxide reductase activity
def: "Catalysis of the reaction: H(2)O + 2 cytochrome c + nitrogen = 2 reduced cytochrome c + nitrous oxide." [EC:1.7.2.4, MetaCyc:RXN-12130]
GO:0050305
name: strombine dehydrogenase activity
def: "Catalysis of the reaction: N-(carboxymethyl)-D-alanine + H(2)O + NAD(+) = glycine + H(+) + NADH + pyruvate." [EC:1.5.1.22, RHEA:14064]
GO:0050306
name: sucrose 1F-fructosyltransferase activity
def: "Catalysis of the reaction: 2 sucrose = D-glucose + 1F-beta-D-fructosylsucrose." [EC:2.4.1.99, MetaCyc:SUCROSE-1F-FRUCTOSYLTRANSFERASE-RXN]
GO:0050307
name: sucrose-phosphate phosphatase activity
def: "Catalysis of the reaction: sucrose 6F-phosphate + H2O = sucrose + phosphate." [EC:3.1.3.24, MetaCyc:SUCROSE-PHOSPHATASE-RXN]
GO:0050308
name: sugar-phosphatase activity
def: "Catalysis of the reaction: sugar phosphate + H2O = sugar + phosphate." [EC:3.1.3.23, MetaCyc:SUGAR-PHOSPHATASE-RXN]
GO:0050309
name: sugar-terminal-phosphatase activity
def: "Catalysis of the reaction: H2O + sugar phosphorylated on the terminal carbon = a sugar + phosphate." [EC:3.1.3.58, MetaCyc:SUGAR-TERMINAL-PHOSPHATASE-RXN]
GO:0050310
name: sulfite dehydrogenase activity
def: "Catalysis of the reaction: sulfite + 2 ferricytochrome c + H2O = sulfate + 2 ferrocytochrome c." [EC:1.8.2.1, MetaCyc:SULFITE-DEHYDROGENASE-RXN]
GO:0050311
name: sulfite reductase (ferredoxin) activity
def: "Catalysis of the reaction: hydrogen sulfide + 3 oxidized ferredoxin + 3 H2O = sulfite + 3 reduced ferredoxin." [EC:1.8.7.1, MetaCyc:SULFITE-REDUCTASE-FERREDOXIN-RXN]
GO:0050312
name: sulfoacetaldehyde lyase activity
def: "Catalysis of the reaction: 2-sulfoacetaldehyde + H2O = acetate + sulfite." [EC:4.4.1.12, MetaCyc:SULFOACETALDEHYDE-LYASE-RXN]
GO:0050313
name: sulfur dioxygenase activity
def: "Catalysis of the reaction: sulfur + O2 + H2O = sulfite." [EC:1.13.11.18, MetaCyc:SULFUR-DIOXYGENASE-RXN]
GO:0050314
name: sym-norspermidine synthase activity
def: "Catalysis of the reaction: 1,3-diaminopropane + S-adenosylmethioninamine = S-methyl-5'-thioadenosine + bis(3-aminopropyl)amine + H(+)." [EC:2.5.1.23, RHEA:23247]
GO:0050315
name: synephrine dehydratase activity
def: "Catalysis of the reaction: synephrine = (4-hydroxyphenyl)acetaldehyde + methylammonium." [EC:4.2.1.88, RHEA:17516]
GO:0050316
name: T2-induced deoxynucleotide kinase activity
def: "Catalysis of the reactions: ATP + dGMP = ADP + dGDP, and ATP + dTMP = ADP + dTDP." [EC:2.7.4.12, MetaCyc:T2-INDUCED-DEOXYNUCLEOTIDE-KINASE-RXN]
GO:0050317
name: tagatose kinase activity
def: "Catalysis of the reaction: D-tagatose + ATP = D-tagatose 6-phosphate + ADP + 2 H(+)." [EC:2.7.1.101, RHEA:15516]
GO:0050318
name: tannase activity
def: "Catalysis of the reaction: digallate + H(2)O = 2 gallate + H(+)." [EC:3.1.1.20, RHEA:16368]
GO:0050319
name: tartrate decarboxylase activity
def: "Catalysis of the reaction: L-tartrate + H(+) = D-glycerate + CO(2)." [EC:4.1.1.73, RHEA:13320]
GO:0050320
name: tartrate epimerase activity
def: "Catalysis of the reaction: L-tartrate = (2R,3S)-tartrate." [EC:5.1.2.5, RHEA:22215]
GO:0050321
name: tau-protein kinase activity
def: "Catalysis of the reaction: ATP + tau-protein = ADP + O-phospho-tau-protein." [EC:2.7.11.26, MetaCyc:TAU-PROTEIN-KINASE-RXN]
GO:0050322
name: taurine-2-oxoglutarate transaminase activity
def: "Catalysis of the reaction: taurine + 2-oxoglutarate = sulfoacetaldehyde + L-glutamate." [EC:2.6.1.55, MetaCyc:RXN-2301]
GO:0050323
name: taurine dehydrogenase activity
def: "Catalysis of the reaction: A + H(2)O + taurine = AH(2) + NH(4)(+) + sulfoacetaldehyde." [EC:1.4.99.2, RHEA:18712]
GO:0050324
name: taurocyamine kinase activity
def: "Catalysis of the reaction: ATP + taurocyamine = N-phosphotaurocyamine + ADP + 2 H(+)." [EC:2.7.3.4, RHEA:22519]
GO:0050325
name: tauropine dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + NAD(+) + tauropine = H(+) + NADH + pyruvate + taurine." [EC:1.5.1.23, RHEA:12583]
GO:0050326
name: taxifolin 8-monooxygenase activity
def: "Catalysis of the reaction: taxifolin + NAD(P)H + H+ + O2 = 2,3-dihydrogossypetin + NAD(P)+ + H2O." [EC:1.14.13.19, MetaCyc:TAXIFOLIN-8-MONOOXYGENASE-RXN]
GO:0050328
name: tetrahydroberberine oxidase activity
def: "Catalysis of the reaction: (S)-tetrahydroberberine + 2 O2 = berberine + 2 H2O2." [EC:1.3.3.8, MetaCyc:TETRAHYDROBERBERINE-OXIDASE-RXN]
GO:0050329
name: tetrahydroxypteridine cycloisomerase activity
def: "Catalysis of the reaction: tetrahydroxypteridine = H(+) + xanthine-8-carboxylate." [EC:5.5.1.3, RHEA:18100]
GO:0050330
name: theanine hydrolase activity
def: "Catalysis of the reaction: N(5)-ethyl-L-glutamine + H(2)O = L-glutamate + ethylamine." [EC:3.5.1.65, RHEA:18016]
GO:0050331
name: thiamine-diphosphate kinase activity
def: "Catalysis of the reaction: ATP + thiamin diphosphate = ADP + thiamin triphosphate." [EC:2.7.4.15, MetaCyc:THIAMIN-DIPHOSPHATE-KINASE-RXN]
GO:0050332
name: thiamine pyridinylase activity
def: "Catalysis of the reaction: pyridine + thiamine = 5-(2-hydroxyethyl)-4-methylthiazole + heteropyrithiamine." [EC:2.5.1.2, RHEA:17700]
GO:0050333
name: thiamin-triphosphatase activity
def: "Catalysis of the reaction: H(2)O + thiamine triphosphate = H(+) + phosphate + thiamine diphosphate." [EC:3.6.1.28, RHEA:11747]
GO:0050334
name: thiaminase activity
def: "Catalysis of the reaction: H(2)O + thiamine = 4-amino-5-hydroxymethyl-2-methylpyrimidine + 5-(2-hydroxyethyl)-4-methylthiazole + H(+)." [EC:3.5.99.2, RHEA:17512]
GO:0050335
name: thiocyanate isomerase activity
def: "Catalysis of the reaction: benzyl isothiocyanate = benzyl thiocyanate." [EC:5.99.1.1, RHEA:10007]
GO:0050336
name: thioethanolamine S-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + cysteamine = S-acetylcysteamine + CoA." [EC:2.3.1.11, RHEA:23283]
GO:0050337
name: thiosulfate-thiol sulfurtransferase activity
def: "Catalysis of the reaction: thiosulfate + 2 glutathione = sulfite + glutathione disulfide + sulfide." [EC:2.8.1.3, MetaCyc:THIOSULFATE--THIOL-SULFURTRANSFERASE-RXN]
GO:0050338
name: thiosulfate dehydrogenase activity
def: "Catalysis of the reaction: 2 thiosulfate + 2 ferricytochrome c = tetrathionate + 2 ferrocytochrome c." [EC:1.8.2.2, MetaCyc:THIOSULFATE-DEHYDROGENASE-RXN]
GO:0050339
name: thymidine-triphosphatase activity
def: "Catalysis of the reaction: dTTP + H2O = dTDP + phosphate." [EC:3.6.1.39, MetaCyc:THYMIDINE-TRIPHOSPHATASE-RXN]
GO:0050340
name: thymidylate 5'-phosphatase activity
def: "Catalysis of the reaction: thymidylate + H2O = thymidine + phosphate." [EC:3.1.3.35, MetaCyc:THYMIDYLATE-5-PHOSPHATASE-RXN]
GO:0050341
name: thymine dioxygenase activity
def: "Catalysis of the reaction: thymine + 2-oxoglutarate + O2 = 5-hydroxymethyluracil + succinate + CO2." [EC:1.14.11.6, MetaCyc:THYMINE-DIOXYGENASE-RXN]
GO:0050342
name: tocopherol O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + gamma-tocopherol = S-adenosyl-L-homocysteine + alpha-tocopherol." [EC:2.1.1.95, MetaCyc:TOCOPHEROL-O-METHYLTRANSFERASE-RXN]
GO:0050343
name: trans-2-enoyl-CoA reductase (NAD+) activity
def: "Catalysis of the reaction: acyl-CoA + NAD+ = trans-didehydroacyl-CoA + NADH." [EC:1.3.1.44, MetaCyc:TRANS-2-ENOYL-COA-REDUCTASE-NAD+-RXN]
GO:0050344
name: trans-cinnamate 2-monooxygenase activity
def: "Catalysis of the reaction: trans-cinnamate + H(+) + NADPH + O(2) = 2-coumarate + H(2)O + NADP(+)." [EC:1.14.13.14, RHEA:10959]
GO:0050345
name: trans-epoxysuccinate hydrolase activity
def: "Catalysis of the reaction: trans-2,3-epoxysuccinate + H(2)O = (2R,3S)-tartrate." [EC:3.3.2.4, RHEA:20743]
GO:0050346
name: trans-L-3-hydroxyproline dehydratase activity
def: "Catalysis of the reaction: trans-L-3-hydroxyproline = 1-pyrroline-2-carboxylate + H(2)O + H(+)." [EC:4.2.1.77, RHEA:10323]
GO:0050347
name: trans-octaprenyltranstransferase activity
def: "Catalysis of the reaction: all-trans-octaprenyl diphosphate + isopentenyl diphosphate = all-trans-nonaprenyl diphosphate + diphosphate." [RHEA:11327]
GO:0050348
name: trehalose O-mycolyltransferase activity
def: "Catalysis of the reaction: 2 alpha,alpha'-trehalose 6-mycolate = alpha,alpha'-trehalose 6,6'-bismycolate + alpha,alpha-trehalose." [EC:2.3.1.122, RHEA:23475]
GO:0050349
name: triacetate-lactonase activity
def: "Catalysis of the reaction: H(2)O + triacetate lactone = triacetate." [EC:3.1.1.38, RHEA:22263]
GO:0050350
name: trihydroxystilbene synthase activity
def: "Catalysis of the reaction: 3 malonyl-CoA + 4-coumaroyl-CoA = 4 CoA + 3,4',5-trihydroxy-stilbene + 4 CO2." [EC:2.3.1.95, MetaCyc:TRIHYDROXYSTILBENE-SYNTHASE-RXN]
GO:0050351
name: trimetaphosphatase activity
def: "Catalysis of the reaction: H(2)O + trimetaphosphate = 2 H(+) + triphosphate." [EC:3.6.1.2, RHEA:11091]
GO:0050352
name: trimethylamine-oxide aldolase activity
def: "Catalysis of the reaction: H(+) + trimethylamine N-oxide = dimethylamine + formaldehyde." [EC:4.1.2.32, RHEA:20220]
GO:0050353
name: trimethyllysine dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate + N(6),N(6),N(6)-trimethyl-L-lysine + O(2) = 3-hydroxy-N(6),N(6),N(6)-trimethyl-L-lysine + CO(2) + succinate." [EC:1.14.11.8, RHEA:14184]
GO:0050354
name: triokinase activity
def: "Catalysis of the reaction: D-glyceraldehyde + ATP = D-glyceraldehyde 3-phosphate + ADP + 2 H(+)." [EC:2.7.1.28, RHEA:13944]
GO:0050355
name: triphosphatase activity
def: "Catalysis of the reaction: H(2)O + triphosphate = diphosphate + phosphate." [EC:3.6.1.25, RHEA:14160]
GO:0050356
name: tropine dehydrogenase activity
def: "Catalysis of the reaction: NADP(+) + tropine = H(+) + NADPH + tropinone." [EC:1.1.1.206, RHEA:18360]
GO:0050357
name: tropinesterase activity
def: "Catalysis of the reaction: atropine + H(2)O = H(+) + tropate + tropine." [EC:3.1.1.10, RHEA:23307]
GO:0050358
name: tropinone reductase activity
def: "Catalysis of the reaction: NADP(+) + pseudotropine = H(+) + NADPH + tropinone." [EC:1.1.1.236, RHEA:24247]
GO:0050359
name: tropomyosin kinase activity
def: "Catalysis of the reaction: ATP + tropomyosin = ADP + O-phosphotropomyosin." [EC:2.7.11.28, MetaCyc:TROPOMYOSIN-KINASE-RXN]
GO:0050360
name: tryptophan 2'-dioxygenase activity
def: "Catalysis of the reaction: L-tryptophan + O2 = 3-indoleglycolaldehyde + CO2 + NH3." [EC:1.13.99.3, MetaCyc:TRYPTOPHAN-2-DIOXYGENASE-RXN]
GO:0050361
name: tryptophan 2-monooxygenase activity
def: "Catalysis of the reaction: L-tryptophan + O(2) = CO(2) + H(2)O + indole-3-acetamide." [EC:1.13.12.3, RHEA:16168]
GO:0050362
name: L-tryptophan:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-tryptophan + 2-oxoglutarate = indolepyruvate + L-glutamate." [EC:2.6.1.27, MetaCyc:TRYPTOPHAN-AMINOTRANSFERASE-RXN]
GO:0050363
name: tryptophan dehydrogenase activity
def: "Catalysis of the reaction: L-tryptophan + NAD(P)+ = (indol-3-yl)pyruvate + NH3 + NAD(P)H + H+." [EC:1.4.1.19, MetaCyc:TRYPTOPHAN-DEHYDROGENASE-RXN]
GO:0050364
name: tryptophan dimethylallyltransferase activity
def: "Catalysis of the reaction: dimethylallyl diphosphate + L-tryptophan = diphosphate + 4-(3-methylbut-2-enyl)-L-tryptophan." [EC:2.5.1.34, MetaCyc:TRYPTOPHAN-DIMETHYLALLYLTRANSFERASE-RXN]
GO:0050365
name: tryptophanamidase activity
def: "Catalysis of the reaction: L-tryptophanamide + H(2)O = L-tryptophan + NH(4)(+)." [EC:3.5.1.57, RHEA:11015]
GO:0050366
name: tyramine N-feruloyltransferase activity
def: "Catalysis of the reaction: feruloyl-CoA + tyramine = CoA + N-feruloyltyramine." [EC:2.3.1.110, MetaCyc:TYRAMINE-N-FERULOYLTRANSFERASE-RXN]
GO:0050367
name: tyrosine-arginine ligase activity
def: "Catalysis of the reaction: L-arginine + L-tyrosine + ATP = L-tyrosyl-L-arginine + AMP + diphosphate + 2 H(+)." [EC:6.3.2.24, RHEA:15348]
GO:0050368
name: tyrosine 2,3-aminomutase activity
def: "Catalysis of the reaction: L-tyrosine = 3-amino-3-(4-hydroxyphenyl)propanoate." [EC:5.4.3.6, RHEA:15784]
GO:0050369
name: [tyrosine 3-monooxygenase] kinase activity
def: "Catalysis of the reaction: ATP + [tyrosine-3-monooxygenase] = ADP + phospho-[tyrosine-3-monooxygenase]." [EC:2.7.11.6, MetaCyc:TYROSINE-3-MONOOXYGENASE-KINASE-RXN]
GO:0050370
name: tyrosine N-monooxygenase activity
def: "Catalysis of the reaction: tyrosine + O2 + NADPH + H+ = N-hydroxytyrosine + NADP+ + H2O." [EC:1.14.13.41, MetaCyc:TYROSINE-N-MONOOXYGENASE-RXN]
GO:0050371
name: tyrosine phenol-lyase activity
def: "Catalysis of the reaction: L-tyrosine + H(2)O = NH(4)(+) + phenol + pyruvate." [EC:4.1.99.2, RHEA:21707]
GO:0050372
name: ubiquitin-calmodulin ligase activity
def: "Catalysis of the reaction: n ATP + calmodulin + n ubiquitin = n AMP + n diphosphate + ubiquitin(n)-calmodulin." [MetaCyc:UBIQUITIN--CALMODULIN-LIGASE-RXN]
GO:0050373
name: UDP-arabinose 4-epimerase activity
def: "Catalysis of the reaction: UDP-L-arabinose = UDP-alpha-D-xylose." [EC:5.1.3.5, RHEA:11323]
GO:0050374
name: UDP-galacturonate decarboxylase activity
def: "Catalysis of the reaction: H(+) + UDP-alpha-D-galacturonate = CO(2) + UDP-L-arabinose." [EC:4.1.1.67, RHEA:19728]
GO:0050376
name: UDP-glucosamine 4-epimerase activity
def: "Catalysis of the reaction: UDP-glucosamine = UDP-galactosamine." [EC:5.1.3.16, MetaCyc:UDP-GLUCOSAMINE-EPIMERASE-RXN]
GO:0050377
name: UDP-glucose 4,6-dehydratase activity
def: "Catalysis of the reaction: UDP-D-glucose = H(2)O + UDP-4-dehydro-6-deoxy-D-glucose." [EC:4.2.1.76, RHEA:21503]
GO:0050378
name: UDP-glucuronate 4-epimerase activity
def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-alpha-D-galacturonate." [EC:5.1.3.6, RHEA:11407]
GO:0050379
name: UDP-glucuronate 5'-epimerase activity
def: "Catalysis of the reaction: UDP-alpha-D-glucuronate = UDP-L-iduronate." [EC:5.1.3.12, RHEA:14680]
GO:0050380
name: undecaprenyl-diphosphatase activity
def: "Catalysis of the reaction: all-trans-undecaprenyl diphosphate + H(2)O = all-trans-undecaprenyl phosphate + H(+) + phosphate." [EC:3.6.1.27, RHEA:17072]
GO:0050382
name: uracil-5-carboxylate decarboxylase activity
def: "Catalysis of the reaction: H(+) + uracil 5-carboxylate = CO(2) + uracil." [EC:4.1.1.66, RHEA:17688]
GO:0050383
name: uracil dehydrogenase activity
def: "Catalysis of the reaction: uracil + acceptor = barbiturate + reduced acceptor." [EC:1.17.99.4, MetaCyc:URACIL-DEHYDROGENASE-RXN]
GO:0050384
name: urate-ribonucleotide phosphorylase activity
def: "Catalysis of the reaction: 3-(beta-D-ribofuranosyl)uric acid + phosphate = alpha-D-ribose 1-phosphate + H(+) + urate." [EC:2.4.2.16, RHEA:13912]
GO:0050385
name: ureidoglycolate lyase activity
def: "Catalysis of the reaction: (S)-ureidoglycolate = glyoxylate + urea." [EC:4.3.2.3, RHEA:11307]
GO:0050386
name: ureidosuccinase activity
def: "Catalysis of the reaction: N-carbamoyl-L-aspartate + H(2)O + 2 H(+) = L-aspartate + CO(2) + NH(4)(+)." [EC:3.5.1.7, RHEA:14368]
GO:0050387
name: urethanase activity
def: "Catalysis of the reaction: H(2)O + H(+) + urethane = CO(2) + ethanol + NH(4)(+)." [EC:3.5.1.75, RHEA:21375]
GO:0050388
name: uronate dehydrogenase activity
def: "Catalysis of the reaction: D-galacturonate + H(2)O + NAD(+) = galactarate + 2 H(+) + NADH." [EC:1.1.1.203, RHEA:22407]
GO:0050389
name: uronolactonase activity
def: "Catalysis of the reaction: D-glucurono-6,2-lactone + H2O = D-glucuronate." [EC:3.1.1.19, MetaCyc:URONOLACTONASE-RXN]
GO:0050390
name: valine decarboxylase activity
def: "Catalysis of the reaction: L-valine + H(+) = 2-methylpropanamine + CO(2)." [EC:4.1.1.14, RHEA:18992]
GO:0050391
name: valine dehydrogenase (NADP) activity
def: "Catalysis of the reaction: L-valine + H2O + NADP+ = 3-methyl-2-oxobutanoate + NH3 + NADPH." [EC:1.4.1.8, MetaCyc:VALINE-DEHYDROGENASE-NADP+-RXN]
GO:0050392
name: vicianin beta-glucosidase activity
def: "Catalysis of the reaction: (R)-vicianin + H(2)O = mandelonitrile + vicianose." [EC:3.2.1.119, RHEA:14044]
GO:0050393
name: vinylacetyl-CoA delta-isomerase activity
def: "Catalysis of the reaction: vinylacetyl-CoA = crotonoyl-CoA." [EC:5.3.3.3, RHEA:10575]
GO:0050394
name: viomycin kinase activity
def: "Catalysis of the reaction: ATP + viomycin = ADP + O-phosphoviomycin." [EC:2.7.1.103, MetaCyc:VIOMYCIN-KINASE-RXN]
GO:0050395
name: vitexin beta-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-D-glucose + vitexin = H(+) + UDP + vitexin 2''-O-beta-D-glucoside." [EC:2.4.1.105, RHEA:21959]
GO:0050396
name: vomifoliol 4'-dehydrogenase activity
def: "Catalysis of the reaction: (6S,9R)-6-hydroxy-3-oxo-alpha-ionol + NAD(+) = (6S)-6-hydroxy-3-oxo-alpha-ionone + H(+) + NADH." [EC:1.1.1.221, RHEA:22807]
GO:0050397
name: Watasenia-luciferin 2-monooxygenase activity
def: "Catalysis of the reaction: Watasenia luciferin + O2 = oxidized Watasenia luciferin + CO2 + light." [EC:1.13.12.8, MetaCyc:WATASEMIA-LUCIFERIN-2-MONOOXYGENASE-RXN]
GO:0050398
name: wax-ester hydrolase activity
def: "Catalysis of the reaction: a wax ester + H2O = a long-chain alcohol + a long-chain carboxylate." [EC:3.1.1.50, MetaCyc:WAX-ESTER-HYDROLASE-RXN]
GO:0050399
name: xanthommatin reductase activity
def: "Catalysis of the reaction: 5,12-dihydroxanthommatin + NAD(+) = H(+) + NADH + xanthommatin." [EC:1.3.1.41, RHEA:13420]
GO:0050400
name: xylitol kinase activity
def: "Catalysis of the reaction: ATP + xylitol = ADP + 2 H(+) + xylitol 5-phosphate." [EC:2.7.1.122, RHEA:20212]
GO:0050401
name: xylonate dehydratase activity
def: "Catalysis of the reaction: D-xylonate = 2-dehydro-3-deoxy-D-arabinonate + H(2)O." [EC:4.2.1.82, RHEA:19160]
GO:0050402
name: xylono-1,4-lactonase activity
def: "Catalysis of the reaction: D-xylono-1,4-lactone + H2O = D-xylonate." [EC:3.1.1.68, MetaCyc:XYLONO-14-LACTONASE-RXN]
GO:0050403
name: trans-zeatin O-beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: trans-zeatin + UDP-D-glucose = O-beta-D-glucosyl-trans-zeatin + H(+) + UDP." [EC:2.4.1.203, RHEA:23227]
GO:0050404
name: zeatin O-beta-D-xylosyltransferase activity
def: "Catalysis of the reaction: UDP-alpha-D-xylose + zeatin = O-beta-D-xylosylzeatin + H(+) + UDP." [EC:2.4.2.40, RHEA:14724]
GO:0050405
name: [acetyl-CoA carboxylase] kinase activity
def: "Catalysis of the reaction: ATP + [acetyl-CoA carboxylase] = ADP + [acetyl-CoA carboxylase] phosphate." [EC:2.7.11.27, MetaCyc:[ACETYL-COA-CARBOXYLASE\]-KINASE-RXN]
GO:0050406
name: [acetyl-CoA carboxylase]-phosphatase activity
def: "Catalysis of the reaction: [acetyl-CoA carboxylase]-phosphate + H2O = [acetyl-CoA carboxylase] + phosphate." [EC:3.1.3.44, MetaCyc:ACETYL-COA-CARBOXYLASE-PHOSPHATASE-RXN]
GO:0050407
name: [glycogen-synthase-D] phosphatase activity
def: "Catalysis of the reaction: [glycogen-synthase D] + H2O = [glycogen-synthase I] + phosphate." [EC:3.1.3.42, MetaCyc:GLYCOGEN-SYNTHASE-D-PHOSPHATASE-RXN]
GO:0050408
name: [pyruvate kinase]-phosphatase activity
def: "Catalysis of the reaction: [pyruvate kinase] phosphate + H2O = [pyruvate kinase] + phosphate." [EC:3.1.3.49, MetaCyc:PYRUVATE-KINASE-PHOSPHATASE-RXN]
GO:0050409
name: indolylacetylinositol arabinosyltransferase activity
def: "Catalysis of the reaction: 1L-1-O-(indol-3-yl)acetyl-myo-inositol + UDP-L-arabinose = (indol-3-yl)acetyl-myo-inositol 3-L-arabinoside + H(+) + UDP." [EC:2.4.2.34, RHEA:19508]
GO:0050410
name: 3-oxolaurate decarboxylase activity
def: "Catalysis of the reaction: 3-oxolaurate + H(+) = 2-undecanone + CO(2)." [EC:4.1.1.56, RHEA:13388]
GO:0050411
name: agaritine gamma-glutamyltransferase activity
def: "Catalysis of the reaction: agaritine + acceptor-NH2 = 4-hydroxymethylphenylhydrazine + gamma-L-glutamyl-acceptor." [EC:2.3.2.9, MetaCyc:AGARITINE-GAMMA-GLUTAMYLTRANSFERASE-RXN]
GO:0050412
name: cinnamate beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: trans-cinnamate + UDP-D-glucose = 1-O-trans-cinnamoyl-beta-D-glucopyranose + UDP." [EC:2.4.1.177, RHEA:13440]
GO:0050413
name: D-alanine 2-hydroxymethyltransferase activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + D-alanine + H(2)O = (6S)-5,6,7,8-tetrahydrofolate + 2-methylserine." [EC:2.1.2.7, RHEA:10067]
GO:0050414
name: formimidoylaspartate deiminase activity
def: "Catalysis of the reaction: N-formimidoyl-L-aspartate + H2O = N-formyl-L-aspartate + NH3." [EC:3.5.3.5, MetaCyc:FORMIMINOASPARTATE-DEIMINASE-RXN]
GO:0050415
name: formimidoylglutamase activity
def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H(2)O = L-glutamate + formamide." [EC:3.5.3.8, RHEA:22495]
GO:0050416
name: formimidoylglutamate deiminase activity
def: "Catalysis of the reaction: N-formimidoyl-L-glutamate + H2O = N-formyl-L-glutamate + NH3." [EC:3.5.3.13, MetaCyc:FORMIMINOGLUTAMATE-DEIMINASE-RXN]
GO:0050417
name: glutamin-(asparagin-)ase activity
def: "Catalysis of the reaction: H2O + L-glutamine = NH3 + L-glutamate; and H2O + L-asparagine = NH3 + L-aspartate." [EC:3.5.1.38, MetaCyc:GLUTAMINASE-ASPARAGIN-ASE-RXN]
GO:0050418
name: hydroxylamine reductase activity
def: "Catalysis of the reaction: NH3 + H2O + acceptor = hydroxylamine + reduced acceptor." [EC:1.7.99.1, MetaCyc:HYDROXYLAMINE-REDUCTASE-RXN]
GO:0050419
name: hydroxymandelonitrile lyase activity
def: "Catalysis of the reaction: 4-hydroxymandelonitrile = 4-hydroxybenzaldehyde + hydrocyanate." [EC:4.1.2.11, MetaCyc:HYDROXYMANDELONITRILE-LYASE-RXN]
GO:0050420
name: maltose synthase activity
def: "Catalysis of the reaction: 2 alpha-D-glucose 1-phosphate = maltose + 2 phosphate." [EC:2.4.1.139, MetaCyc:MALTOSE-SYNTHASE-RXN]
GO:0050421
name: nitrite reductase (NO-forming) activity
def: "Catalysis of the reaction: nitric oxide + H2O + ferricytochrome c = nitrite + ferrocytochrome c + 2 H+." [EC:1.7.2.1, MetaCyc:NITRITE-REDUCTASE-CYTOCHROME-RXN]
GO:0050422
name: strictosidine beta-glucosidase activity
def: "Catalysis of the reaction: 3alpha(S)-strictosidine + H(2)O = D-glucose + strictosidine aglycone." [EC:3.2.1.105, RHEA:12920]
GO:0050423
name: thiamine oxidase activity
def: "Catalysis of the reaction: thiamine + 2 O2 = thiamine acetic acid + 2 H2O2." [EC:1.1.3.23, MetaCyc:THIAMIN-OXIDASE-RXN]
GO:0050424
name: alanine carboxypeptidase activity
def: "OBSOLETE. Catalysis of the reaction: H2O + peptidyl-L-alanine = L-alanine + peptide. The release of a C-terminal alanine from a peptide or a variety of pteroyl or acyl groups." [EC:3.4.17.6, MetaCyc:ALANINE-CARBOXYPEPTIDASE-RXN]
GO:0050425
name: carboxypeptidase B activity
def: "OBSOLETE. Catalysis of the reaction: H2O + peptidyl-L-lysine (or L-arginine) = L-lysine (or L-arginine) + peptide. Preferential release of a C-terminal lysine or arginine amino acid." [EC:3.4.17.2, MetaCyc:CARBOXYPEPTIDASE-B-RXN]
GO:0050426
name: peptidyl-glycinamidase activity
def: "OBSOLETE. Catalysis of the reaction: H2O + peptidyl-glycinamide = glycinamide + peptide. Cleavage of C-terminal glycinamide from a polypeptide." [EC:3.4.19.2, MetaCyc:PEPTIDYL-GLYCINAMIDASE-RXN]
GO:0050429
name: calcium-dependent phospholipase C activity
def: "Catalysis of the reaction: a phosphatidylcholine + H2O = 1,2-diacylglycerol + choline phosphate. This reaction requires Ca2+." [EC:3.1.4.3]
GO:0050431
name: transforming growth factor beta binding
def: "Interacting selectively and non-covalently with TGF-beta, transforming growth factor beta, a multifunctional peptide that controls proliferation, differentiation and other functions in many cell types." [ISBN:0198506732]
GO:0050436
name: microfibril binding
def: "Interacting selectively and non-covalently with a microfibril, any small fibril occurring in biological material." [GOC:ai]
GO:0050437
name: (-)-endo-fenchol synthase activity
def: "Catalysis of the reaction: geranyl diphosphate + H(2)O = (-)-endo-fenchol + diphosphate." [EC:4.2.3.10, RHEA:20568]
GO:0050438
name: 2-ethylmalate synthase activity
def: "Catalysis of the reaction: 2-oxobutanate + acetyl-CoA + H(2)O = (R)-2-ethylmalate + CoA + H(+)." [EC:2.3.3.6, RHEA:23043]
GO:0050439
name: 2-hydroxy-3-oxoadipate synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + glyoxylate + H(+) = 2-hydroxy-3-oxoadipate + CO(2)." [EC:2.2.1.5, RHEA:14344]
GO:0050440
name: 2-methylcitrate synthase activity
def: "Catalysis of the reaction: H(2)O + oxaloacetate + propanoyl-CoA = (2R,3S)-2-methylcitrate + CoA + H(+)." [EC:2.3.3.5, RHEA:23783]
GO:0050441
name: 3-ethylmalate synthase activity
def: "Catalysis of the reaction: butanoyl-CoA + glyoxylate + H(2)O = 3-ethylmalate + CoA + H(+)." [EC:2.3.3.7, RHEA:10503]
GO:0050442
name: 3-propylmalate synthase activity
def: "Catalysis of the reaction: glyoxylate + H(2)O + pentanoyl-CoA = 3-propylmalate + CoA + H(+)." [EC:2.3.3.12, RHEA:14460]
GO:0050444
name: aquacobalamin reductase (NADPH) activity
def: "Catalysis of the reaction: 2 cob(II)alamin + NADP+ = 2 aquacob(III)alamin + NADPH + H+." [EC:1.16.1.5, MetaCyc:AQUACOBALAMIN-REDUCTASE-NADPH-RXN]
GO:0050445
name: asparagusate reductase activity
def: "Catalysis of the reaction: 3-mercapto-2-mercaptomethylpropanoate + NAD(+) = asparagusate + H(+) + NADH." [EC:1.8.1.11, RHEA:14884]
GO:0050446
name: azobenzene reductase activity
def: "Catalysis of the reaction: N,N-dimethyl-1,4-phenylenediamine + aniline + NADP+ = 4-(dimethylamino)azobenzene + NADPH + H+." [EC:1.7.1.6, MetaCyc:AZOBENZENE-REDUCTASE-RXN]
GO:0050447
name: zeatin 9-aminocarboxyethyltransferase activity
def: "Catalysis of the reaction: O-acetyl-L-serine + zeatin = L-lupinate + acetate + H(+)." [EC:2.5.1.50, RHEA:17336]
GO:0050448
name: beta-cyclopiazonate dehydrogenase activity
def: "Catalysis of the reaction: beta-cyclopiazonate + A = alpha-cyclopiazonate + AH(2)." [EC:1.21.99.1, RHEA:14526]
GO:0050449
name: casbene synthase activity
def: "Catalysis of the reaction: all-trans-geranylgeranyl diphosphate = casbene + diphosphate." [EC:4.2.3.8, RHEA:14904]
GO:0050450
name: citrate (Re)-synthase activity
def: "Catalysis of the reaction: acetyl-CoA + H2O + oxaloacetate = citrate + CoA, where the acetyl group is added to the re-face of oxaloacetate; acetyl-CoA provides the two carbon atoms of the pro-R carboxymethyl group." [EC:2.3.3.3, ISBN:0121227073, MetaCyc:CITRATE-RE-SYNTHASE-RXN]
GO:0050451
name: CoA-disulfide reductase activity
def: "Catalysis of the reaction: 2 CoA + NAD+ = CoA-disulfide + NADH + H+." [EC:1.8.1.14, MetaCyc:COA-DISULFIDE-REDUCTASE-NADH-RXN]
GO:0050452
name: CoA-glutathione reductase activity
def: "Catalysis of the reaction: CoA + glutathione + NADP+ = CoA-glutathione + NADPH + H+." [EC:1.8.1.10, MetaCyc:COA-GLUTATHIONE-REDUCTASE-NADPH-RXN]
GO:0050453
name: cob(II)alamin reductase activity
def: "Catalysis of the reaction: 2 cob(I)alamin + H(+) + NAD(+) = 2 cob(II)alamin + NADH." [EC:1.16.1.4, RHEA:17484]
GO:0050454
name: coenzyme F420 hydrogenase activity
def: "Catalysis of the reaction: coenzyme F420 + H(2) + H(+) = reduced coenzyme F420." [EC:1.12.98.1, RHEA:23763]
GO:0050455
name: columbamine oxidase activity
def: "Catalysis of the reaction: 2 columbamine + O(2) = 2 berberine + 2 H(2)O." [EC:1.21.3.2, RHEA:23567]
GO:0050456
name: cystine reductase activity
def: "Catalysis of the reaction: 2 L-cysteine + NAD(+) = L-cystine + H(+) + NADH." [EC:1.8.1.6, RHEA:20600]
GO:0050457
name: decylcitrate synthase activity
def: "Catalysis of the reaction: H(2)O + lauroyl-CoA + oxaloacetate = (2S,3S)-2-hydroxytridecane-1,2,3-tricarboxylate + CoA + H(+)." [EC:2.3.3.2, RHEA:16608]
GO:0050458
name: decylhomocitrate synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + H(2)O + lauroyl-CoA = (3S,4S)-3-hydroxytetradecane-1,3,4-tricarboxylate + CoA + H(+)." [EC:2.3.3.4, RHEA:10367]
GO:0050459
name: ethanolamine-phosphate phospho-lyase activity
def: "Catalysis of the reaction: H(2)O + phosphoethanolamine = acetaldehyde + NH(4)(+) + phosphate." [EC:4.2.3.2, RHEA:17892]
GO:0050460
name: hydroxylamine reductase (NADH) activity
def: "Catalysis of the reaction: NH3 + NAD+ + H2O = hydroxylamine + NADH + H+." [EC:1.7.1.10, MetaCyc:HYDROXYLAMINE-REDUCTASE-NADH-RXN]
GO:0050461
name: L-mimosine synthase activity
def: "Catalysis of the reaction: 3,4-dihydroxypyridine + O-acetyl-L-serine = 3-(3,4-dihydroxypyridinium-1-yl)-L-alanine + acetate." [EC:2.5.1.52, RHEA:12696]
GO:0050462
name: N-acetylneuraminate synthase activity
def: "Catalysis of the reaction: phosphoenolpyruvate + N-acetyl-D-mannosamine + H2O = phosphate + N-acetylneuraminate." [EC:2.5.1.56, MetaCyc:N-ACETYLNEURAMINATE-SYNTHASE-RXN]
GO:0050463
name: nitrate reductase [NAD(P)H] activity
def: "Catalysis of the reaction: nitrite + NAD(P)+ + H2O = nitrate + NAD(P)H + H+." [EC:1.7.1.2, MetaCyc:NITRATE-REDUCTASE-NADPH-RXN]
GO:0050464
name: nitrate reductase (NADPH) activity
def: "Catalysis of the reaction: nitrite + NADP+ + H2O = nitrate + NADPH + H+." [EC:1.7.1.3, MetaCyc:NITRATE-REDUCTASE-NADPH-RXN]
GO:0050465
name: nitroquinoline-N-oxide reductase activity
def: "Catalysis of the reaction: 4-(hydroxyamino)quinoline N-oxide + 2 NAD(P)+ + H2O = 4-nitroquinoline N-oxide + 2 NAD(P)H + 2 H+." [EC:1.7.1.9, MetaCyc:1.7.1.9-RXN]
GO:0050466
name: oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond, with other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which X-H and Y-H form X-Y and the acceptor is not disulfide or oxygen." [GOC:ai]
GO:0050467
name: pentalenene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = diphosphate + pentalenene." [EC:4.2.3.7, RHEA:18084]
GO:0050468
name: reticuline oxidase activity
def: "Catalysis of the reaction: (S)-reticuline + O(2) = (S)-scoulerine + H(2)O(2) + H(+)." [EC:1.21.3.3, RHEA:19888]
GO:0050469
name: sabinene-hydrate synthase activity
def: "Catalysis of the reaction: geranyl diphosphate + H(2)O = diphosphate + sabinene hydrate." [EC:4.2.3.11, RHEA:19568]
GO:0050470
name: trimethylamine dehydrogenase activity
def: "Catalysis of the reaction: trimethylamine + H2O + electron-transferring flavoprotein = dimethylamine + formaldehyde + reduced electron-transferring flavoprotein." [EC:1.5.8.2, MetaCyc:1.5.8.2-RXN]
GO:0050471
name: uracilylalanine synthase activity
def: "Catalysis of the reaction: O3-acetyl-L-serine + uracil = 3-(uracil-1-yl)-L-alanine + acetate." [EC:2.5.1.53, MetaCyc:URACILYLALANINE-SYNTHASE-RXN]
GO:0050472
name: zeatin reductase activity
def: "Catalysis of the reaction: dihydrozeatin + NADP(+) = H(+) + NADPH + zeatin." [EC:1.3.1.69, RHEA:12760]
GO:0050473
name: arachidonate 15-lipoxygenase activity
def: "Catalysis of the reaction: arachidonate + O2 = (5Z,8Z,11Z,13E)-(15S)-15-hydroperoxyicosa-5,8,11,13-tetraenoate." [EC:1.13.11.33]
GO:0050474
name: (S)-norcoclaurine synthase activity
def: "Catalysis of the reaction: 4-(2-aminoethyl)benzene-1,2-diol + 4-hydroxyphenylacetaldehyde = (S)-norcoclaurine + H2O." [EC:4.2.1.78]
GO:0050476
name: acetylenedicarboxylate decarboxylase activity
def: "Catalysis of the reaction: H2O + acetylenedicarboxylate = CO2 + pyruvate." [EC:4.1.1.78]
GO:0050477
name: acyl-lysine deacylase activity
def: "Catalysis of the reaction: H2O + N6-acyl-L-lysine = L-lysine + a carboxylate." [EC:3.5.1.17]
GO:0050478
name: anthranilate 3-monooxygenase activity
def: "Catalysis of the reaction: 5,6,7,8-tetrahydrobiopterin + anthranilate + O(2) = 3-hydroxyanthranilate + 7,8-dihydrobiopterin + H(2)O." [EC:1.14.16.3, RHEA:21171]
GO:0050479
name: glyceryl-ether monooxygenase activity
def: "Catalysis of the reaction: 1-alkyl-sn-glycerol + O2 + (tetrahydrobiopterin/tetrahydropteridine) = 1-hydroxyalkyl-sn-glycerol + H2O + (dihydrobiopterin/dihydropteridine)." [EC:1.14.16.5, MetaCyc:GLYCERYL-ETHER-MONOOXYGENASE-RXN]
GO:0050480
name: imidazolonepropionase activity
def: "Catalysis of the reaction: (S)-3-(4-oxo-4,5-dihydro-1H-imidazol-5-yl)propanoic acid + H(2)O = N-formimidoyl-L-glutamate + H(+)." [EC:3.5.2.7, RHEA:23663]
GO:0050481
name: mandelate 4-monooxygenase activity
def: "Catalysis of the reaction: (S)-mandelate + 5,6,7,8-tetrahydrobiopterin + O(2) = (S)-4-hydroxymandelate + 7,8-dihydrobiopterin + H(2)O. (S)-2-hydroxy-2-phenylacetate is also known as S-mandelate." [EC:1.14.16.6, RHEA:21719]
GO:0050483
name: IMP 5'-nucleotidase activity
def: "Catalysis of the reaction: 5'-IMP + H2O = inosine + phosphate." [GOC:ai]
GO:0050484
name: GMP 5'-nucleotidase activity
def: "Catalysis of the reaction: 5'-GMP + H2O = guanosine + phosphate." [GOC:ai]
GO:0050485
name: oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond, with a disulfide as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which X-H and Y-H form X-Y and the acceptor is disulfide." [GOC:ai]
GO:0050486
name: intramolecular transferase activity, transferring hydroxy groups
def: "Catalysis of the transfer of a hydroxyl group from one position to another within a single molecule." [GOC:mah]
GO:0050487
name: sulfoacetaldehyde acetyltransferase activity
def: "Catalysis of the reaction: acetyl phosphate + sulfite = phosphate + sulfoacetaldehyde." [EC:2.3.3.15, RHEA:24207]
GO:0050488
name: ecdysteroid UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + ecdysteroid = UDP + glucosyl-ecdysteroid." [GOC:ai, PMID:10073711]
GO:0050490
name: 1,4-lactonase activity
def: "Catalysis of the reaction: H2O + a 1,4-lactone = a 4-hydroxyacid." [EC:3.1.1.25, MetaCyc:14-LACTONASE-RXN]
GO:0050491
name: sulcatone reductase activity
def: "Catalysis of the reaction: NAD(+) + sulcatol = H(+) + NADH + sulcatone." [EC:1.1.1.260, RHEA:24487]
GO:0050492
name: glycerol-1-phosphate dehydrogenase [NAD(P)+] activity
def: "Catalysis of the reaction: NAD(P)+ + sn-glycerol-1-phosphate = NAD(P)H + H+ + dihydroxy-acetone-phosphate." [EC:1.1.1.261, MetaCyc:1.1.1.261-RXN]
GO:0050497
name: transferase activity, transferring alkylthio groups
def: "Catalysis of the transfer of an alkylthio group from one compound (donor) to another (acceptor)." [GOC:ai]
GO:0050498
name: oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, with 2-oxoglutarate as one donor, and the other dehydrogenated
def: "Catalysis of an oxidation-reduction (redox) reaction in which hydrogen or electrons are transferred from 2-oxoglutarate and one other donor, and the latter donor is dehydrogenated." [GOC:mah]
GO:0050499
name: oxidoreductase activity, acting on phosphorus or arsenic in donors, with NAD(P)+ as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a phosphorus- or arsenic-containing group acts as a hydrogen or electron donor and reduces a NAD(P)+ to NAD(P)H." [GOC:mah]
GO:0050500
name: 1,3-beta-galactosyl-N-acetylhexosamine phosphorylase activity
def: "Catalysis of the reaction: phosphate + beta-D-galactopyranosyl-(1->3)-N-acetyl-D-glucosamine = N-acetyl-D-glucosamine + alpha-D-galactopyranose 1-phosphate." [EC:2.4.1.211, MetaCyc:2.4.1.211-RXN]
GO:0050501
name: hyaluronan synthase activity
def: "Catalysis of the reaction: UDP-D-glucuronate + UDP-N-acetyl-D-glucosamine = [beta-N-acetyl-D-glucosaminyl-(1->4)-beta-D-glucuronosyl-(1->3)](n) + 2n UDP." [EC:2.4.1.212, MetaCyc:2.4.1.212-RXN]
GO:0050502
name: cis-zeatin O-beta-D-glucosyltransferase activity
def: "Catalysis of the reaction: cis-zeatin + UDP-D-glucose = O-beta-D-glucosyl-cis-zeatin + H(+) + UDP." [EC:2.4.1.215, RHEA:20684]
GO:0050503
name: trehalose 6-phosphate phosphorylase activity
def: "Catalysis of the reaction: trehalose 6-phosphate + phosphate = glucose 6-phosphate + beta-D-glucose 1-phosphate." [EC:2.4.1.216, MetaCyc:2.4.1.216-RXN]
GO:0050504
name: mannosyl-3-phosphoglycerate synthase activity
def: "Catalysis of the reaction: 3-phospho-D-glycerate + GDP-alpha-D-mannose = 2-(alpha-D-mannosyl)-3-phosphoglycerate + GDP + H(+)." [EC:2.4.1.217, RHEA:13540]
GO:0050505
name: hydroquinone glucosyltransferase activity
def: "Catalysis of the reaction: hydroquinone + UDP-D-glucose = H(+) + hydroquinone O-beta-D-glucopyranoside + UDP." [EC:2.4.1.218, RHEA:12563]
GO:0050506
name: vomilenine glucosyltransferase activity
def: "Catalysis of the reaction: UDP-D-glucose + vomilenine = H(+) + raucaffricine + UDP." [EC:2.4.1.219, RHEA:19388]
GO:0050507
name: indoxyl-UDPG glucosyltransferase activity
def: "Catalysis of the reaction: indoxyl + UDP-D-glucose = H(+) + indican + UDP." [EC:2.4.1.220, RHEA:12007]
GO:0050508
name: glucuronosyl-N-acetylglucosaminyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: beta-D-glucuronosyl-(1,4)-N-acetyl-alpha-D-glucosaminyl-proteoglycan + UDP-N-acetyl-D-glucosamine = N-acetyl-alpha-D-glucosaminyl-(1,4)-beta-D-glucuronosyl-(1,4)-N-acetyl-alpha-D-glucosaminyl-proteoglycan + UDP." [EC:2.4.1.224, MetaCyc:2.4.1.224-RXN]
GO:0050509
name: N-acetylglucosaminyl-proteoglycan 4-beta-glucuronosyltransferase activity
def: "Catalysis of the reaction: N-acetyl-alpha-D-glucosaminyl-(1,4)-beta-D-glucuronosyl-proteoglycan + UDP-alpha-D-glucuronate = beta-D-glucuronosyl-(1,4)-N-acetyl-alpha-D-glucosaminyl-(1,4)-beta-D-glucuronosyl-proteoglycan + UDP." [EC:2.4.1.225, MetaCyc:2.4.1.225-RXN]
GO:0050510
name: N-acetylgalactosaminyl-proteoglycan 3-beta-glucuronosyltransferase activity
def: "Catalysis of the reaction: N-acetyl-beta-D-galactosaminyl-(1,4)-beta-D-glucuronosyl-proteoglycan + UDP-alpha-D-glucuronate = beta-D-glucuronosyl-(1,3)-N-acetyl-beta-D-galactosaminyl-(1,4)-beta-D-glucuronosyl-proteoglycan + UDP." [EC:2.4.1.226, MetaCyc:2.4.1.226-RXN]
GO:0050511
name: undecaprenyldiphospho-muramoylpentapeptide beta-N-acetylglucosaminyltransferase activity
def: "Catalysis of the reaction: Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol + UDP-N-acetyl-D-glucosamine = GlcNAc-(1,4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol + UDP." [EC:2.4.1.227, MetaCyc:2.4.1.227-RXN]
GO:0050512
name: lactosylceramide 4-alpha-galactosyltransferase activity
def: "Catalysis of the reaction: beta-D-galactosyl-(1,4)-D-glucosylceramide + UDP-galactose = alpha-D-galactosyl-(1,4)-beta-D-galactosyl-(1,4)-D-glucosylceramide + UDP." [EC:2.4.1.228, MetaCyc:2.4.1.228-RXN]
GO:0050513
name: glycoprotein 2-beta-D-xylosyltransferase activity
def: "Catalysis of the reaction: N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-N-acetyl-beta-D-glucosaminyl}-L-asparagine + UDP-alpha-D-xylose = N(4)-{N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->3)-[N-acetyl-beta-D-glucosaminyl-(1->2)-alpha-D-mannosyl-(1->6)]-[beta-D-xylosyl-(1->2)]-beta-D-mannosyl-(1->4)-N-acetyl-beta-D-glucosaminyl-(1->4)-N-acetyl-beta-D-glucosaminyl}-L-asparagine + H(+) + UDP." [EC:2.4.2.38, RHEA:10615]
GO:0050514
name: homospermidine synthase (spermidine-specific) activity
def: "Catalysis of the reaction: spermidine + putrescine = sym-homospermidine + propane-1,3-diamine." [EC:2.5.1.45, MetaCyc:2.5.1.45-RXN]
GO:0050515
name: 4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol kinase activity
def: "Catalysis of the reaction: 4-CDP-2-C-methyl-D-erythritol + ATP = 4-CDP-2-C-methyl-D-erythritol 2-phosphate + ADP + 2 H(+)." [EC:2.7.1.148, RHEA:18440]
GO:0050516
name: inositol polyphosphate multikinase activity
def: "OBSOLETE. Catalysis of the reactions: ATP + 1D-myo-inositol 1,4,5-trisphosphate = ADP + 1D-myo-inositol 1,4,5,6-tetrakisphosphate, and ATP + 1D-myo-inositol 1,4,5,6-tetrakisphosphate = ADP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate." [EC:2.7.1.151, MetaCyc:2.7.1.151-RXN]
GO:0050517
name: inositol hexakisphosphate kinase activity
def: "OBSOLETE. Catalysis of the reactions: ATP + 1D-myo-inositol hexakisphosphate = ADP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate, and ATP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate = ADP + diphospho-1D-myo-inositol tetrakisphosphate (isomeric configuration unknown)." [EC:2.7.4.21, MetaCyc:2.7.4.21-RXN]
GO:0050518
name: 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase activity
def: "Catalysis of the reaction: 2-C-methyl-D-erythritol 4-phosphate + CTP = 4-CDP-2-C-methyl-D-erythritol + diphosphate." [EC:2.7.7.60, RHEA:13432]
GO:0050519
name: holo-citrate lyase synthase activity
def: "Catalysis of the reaction: apo-citrate lyase + 2'-(5''-triphosphoribosyl)-3'-dephospho-CoA = diphosphate + holo-citrate lyase." [EC:2.7.7.61, MetaCyc:2.7.7.61-RXN]
GO:0050520
name: phosphatidylcholine synthase activity
def: "Catalysis of the reaction: CDP-diacylglycerol + choline = 1,2-diacyl-sn-glycero-3-phosphocholine + CMP + H(+)." [EC:2.7.8.24, RHEA:14600]
GO:0050521
name: alpha-glucan, water dikinase activity
def: "Catalysis of the reaction: ATP + alpha-glucan + H2O = AMP + phospho-alpha-glucan + phosphate." [EC:2.7.9.4, MetaCyc:2.7.9.4-RXN]
GO:0050522
name: oxidoreductase activity, acting on phosphorus or arsenic in donors, with other known acceptors
def: "Catalysis of an oxidation-reduction (redox) reaction in which phosphorus or arsenic acts as a hydrogen or electron donor and reduces a known acceptor other than disulfide, NAD or NADP." [GOC:ai]
GO:0050523
name: oxidoreductase activity, acting on phosphorus or arsenic in donors, with other acceptors
def: "OBSOLETE. Catalysis of an oxidation-reduction (redox) reaction in which phosphorus or arsenic acts as a hydrogen or electron donor and reduces an acceptor other than disulfide, NAD or NADP." [GOC:ai]
GO:0050524
name: coenzyme-B sulfoethylthiotransferase activity
def: "Catalysis of the reaction: coenzyme B + methyl-coenzyme M = coenzyme M-coenzyme B heterodisulfide + methane. Methyl-CoM is also known as 2-(methylthio)ethanesulfonate, coenzyme B as N-(7-mercaptoheptanoyl)threonine 3-O-phosphate, and coenzyme M-coenzyme B heterodisulfide as CoM-S-S-CoB." [EC:2.8.4.1, RHEA:12535]
GO:0050525
name: cutinase activity
def: "Catalysis of the reaction: cutin + H2O = cutin monomers." [EC:3.1.1.74, MetaCyc:3.1.1.74-RXN]
GO:0050526
name: poly(3-hydroxybutyrate) depolymerase activity
def: "Catalysis of the reaction: H2O + poly[(R)-3-hydroxybutanoate](n) = poly[(R)-3-hydroxybutanoate](x) + poly[(R)-3-hydroxybutanoate](n-x); x is 1-5." [EC:3.1.1.75, MetaCyc:3.1.1.75-RXN]
GO:0050527
name: poly(3-hydroxyoctanoate) depolymerase activity
def: "Catalysis of the reaction: H2O + poly[(R)-3-hydroxyoctanoate](n) = poly[(R)-3-hydroxyoctanoate](x) + poly[(R)-3-hydroxyoctanoate](n-x); x is 1-5." [EC:3.1.1.76, MetaCyc:3.1.1.76-RXN]
GO:0050528
name: acyloxyacyl hydrolase activity
def: "Catalysis of the reaction: 3-(acyloxy)acyl group of bacterial toxin = 3-hydroxyacyl group of bacterial toxin + a fatty acid." [EC:3.1.1.77, MetaCyc:3.1.1.77-RXN]
GO:0050529
name: polyneuridine-aldehyde esterase activity
def: "Catalysis of the reaction: H(2)O + polyneuridine aldehyde = 16-epivellosimine + CO(2) + methanol." [EC:3.1.1.78, RHEA:17504]
GO:0050530
name: glucosylglycerol 3-phosphatase activity
def: "Catalysis of the reaction: 2-O-(beta-D-glucosyl)-sn-glycerol 3-phosphate + H(2)O = 2-O-(beta-D-glucosyl)-sn-glycerol + phosphate." [EC:3.1.3.69, RHEA:22655]
GO:0050531
name: mannosyl-3-phosphoglycerate phosphatase activity
def: "Catalysis of the reaction: 2-(alpha-D-mannosyl)-3-phosphoglycerate + H(2)O = 2-(alpha-D-mannosyl)-D-glycerate + phosphate." [EC:3.1.3.70, RHEA:19312]
GO:0050532
name: 2-phosphosulfolactate phosphatase activity
def: "Catalysis of the reaction: (2R)-O-phospho-3-sulfolactate + H(2)O = (R)-3-sulfolactate + phosphate." [EC:3.1.3.71, RHEA:23419]
GO:0050533
name: 5-phytase activity
def: "Catalysis of the reaction: H2O + myo-inositol hexakisphosphate = phosphate + 1L-myo-inositol 1,2,3,4,6-pentakisphosphate." [EC:3.1.3.72, MetaCyc:3.1.3.72-RXN]
GO:0050534
name: 3-deoxyoctulosonase activity
def: "Catalysis of the reaction: 3-deoxyoctulosonyl-lipopolysaccharide + H2O = 3-deoxyoctulosonic acid + lipopolysaccharide." [EC:3.2.1.144, MetaCyc:3.2.1.144-RXN]
GO:0050535
name: beta-primeverosidase activity
def: "Catalysis of the reaction: a 6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranoside + H2O = 6-O-(beta-D-xylopyranosyl)-beta-D-glucopyranose + an alcohol." [EC:3.2.1.149, MetaCyc:3.2.1.149-RXN]
GO:0050536
name: (S)-N-acetyl-1-phenylethylamine hydrolase activity
def: "Catalysis of the reaction: N-acetylphenylethylamine + H(2)O = acetate + phenylethylamine." [EC:3.5.1.85, RHEA:23955]
GO:0050537
name: mandelamide amidase activity
def: "Catalysis of the reaction: (R)-mandelamide + H(2)O = (R)-mandelate + NH(4)(+)." [EC:3.5.1.86, RHEA:22879]
GO:0050538
name: N-carbamoyl-L-amino-acid hydrolase activity
def: "Catalysis of the reaction: N-carbamoyl-L-2-amino acid + H2O = L-2-amino acid + NH3 + CO2. The N-carbamoyl-L-2-amino acid is a 2-ureido carboxylate." [EC:3.5.1.87, MetaCyc:3.5.1.87-RXN]
GO:0050539
name: maleimide hydrolase activity
def: "Catalysis of the reaction: H(2)O + maleimide = H(+) + maleamate." [EC:3.5.2.16, RHEA:24479]
GO:0050540
name: 2-aminomuconate deaminase activity
def: "Catalysis of the reaction: 2-aminomuconate + H(2)O + H(+) = (Z)-5-oxohex-2-enedioate + NH(4)(+)." [EC:3.5.99.5, RHEA:20999]
GO:0050541
name: beta,beta-carotene-9',10'-dioxygenase activity
def: "Catalysis of the reaction: beta-carotene + O2 = beta-apo-10'-carotenal + beta-ionone." [PMID:11278918]
GO:0050542
name: icosanoid binding
def: "Interacting selectively and non-covalently with icosanoids, any C20 polyunsaturated fatty acids or their derivatives, including the leukotrienes and the prostanoids." [ISBN:0198506732]
GO:0050543
name: icosatetraenoic acid binding
def: "Interacting selectively and non-covalently with icosatetraenoic acid, any straight-chain fatty acid with twenty carbon atoms and four double bonds per molecule." [ISBN:0198506732]
GO:0050544
name: arachidonic acid binding
def: "Interacting selectively and non-covalently with arachidonic acid, a straight chain fatty acid with 20 carbon atoms and four double bonds per molecule. Arachidonic acid is the all-Z-(5,8,11,14)-isomer." [GOC:ai]
GO:0050545
name: sulfopyruvate decarboxylase activity
def: "Catalysis of the reaction: 3-sulfopyruvate + H(+) = CO(2) + sulfoacetaldehyde." [EC:4.1.1.79, RHEA:20951]
GO:0050546
name: 4-hydroxyphenylpyruvate decarboxylase activity
def: "Catalysis of the reaction: (4-hydroxyphenyl)pyruvate + H(+) = (4-hydroxyphenyl)acetaldehyde + CO(2)." [EC:4.1.1.80, RHEA:18700]
GO:0050547
name: vanillin synthase activity
def: "Catalysis of the reaction: 3-hydroxy-3-(4-hydroxy-3-methoxyphenyl)propanoyl-CoA = acetyl-CoA + vanillin." [EC:4.1.2.41, RHEA:18728]
GO:0050548
name: trans-feruloyl-CoA hydratase activity
def: "Catalysis of the reaction: trans-feruloyl-CoA + H2O = 4-hydroxy-3-methoxyphenyl-beta-hydroxypropionyl-CoA." [EC:4.2.1.101, MetaCyc:4.2.1.101-RXN]
GO:0050549
name: cyclohexyl-isocyanide hydratase activity
def: "Catalysis of the reaction: N-cyclohexylformamide + H(+) = cyclohexyl isocyanide + H(2)O." [EC:4.2.1.103, RHEA:18200]
GO:0050550
name: pinene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = pinene + diphosphate." [EC:4.2.3.14, MetaCyc:4.2.3.14-RXN]
GO:0050551
name: myrcene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = diphosphate + myrcene." [EC:4.2.3.15, RHEA:16968]
GO:0050552
name: (4S)-limonene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = (4S)-limonene + diphosphate." [EC:4.2.3.16, MetaCyc:4.2.3.16-RXN]
GO:0050553
name: taxadiene synthase activity
def: "Catalysis of the reaction: all-trans-geranylgeranyl diphosphate = diphosphate + taxa-4,11-diene." [EC:4.2.3.17, RHEA:20915]
GO:0050554
name: abietadiene synthase activity
def: "Catalysis of the reaction: (+)-copalyl diphosphate = (-)-abietadiene + diphosphate." [EC:4.2.3.18, RHEA:13876]
GO:0050555
name: 2-hydroxypropyl-CoM lyase activity
def: "Catalysis of the reactions: (R)-2-hydroxypropyl-CoM = H-S-CoM + (R)-1,2-epoxypropane, and (S)-2-hydroxypropyl-CoM = H-S-CoM + (S)-1,2-epoxypropane." [EC:4.2.99.19, MetaCyc:4.2.99.19-RXN]
GO:0050556
name: deacetylisoipecoside synthase activity
def: "Catalysis of the reaction: deacetylisoipecoside + H(2)O = dopamine + secologanin." [EC:4.3.3.3, RHEA:21759]
GO:0050557
name: deacetylipecoside synthase activity
def: "Catalysis of the reaction: deacetylipecoside + H(2)O = dopamine + secologanin." [EC:4.3.3.4, RHEA:12299]
GO:0050558
name: maltose epimerase activity
def: "Catalysis of the reaction: alpha-maltose = beta-maltose." [EC:5.1.3.21, RHEA:21231]
GO:0050559
name: copalyl diphosphate synthase activity
def: "Catalysis of the reaction: all-trans-geranylgeranyl diphosphate = (+)-copalyl diphosphate." [EC:5.5.1.12, RHEA:24319]
GO:0050560
name: aspartate-tRNA(Asn) ligase activity
def: "Catalysis of the reaction: tRNA(Asx) + L-aspartate + ATP = aspartyl-tRNA(Asx) + diphosphate + AMP." [EC:6.1.1.23, MetaCyc:6.1.1.23-RXN]
GO:0050561
name: glutamate-tRNA(Gln) ligase activity
def: "Catalysis of the reaction: tRNA(Glx) + L-glutamate + ATP = glutamyl-tRNA(Glx) + diphosphate + AMP." [EC:6.1.1.24, MetaCyc:6.1.1.24-RXN]
GO:0050562
name: lysine-tRNA(Pyl) ligase activity
def: "Catalysis of the reaction: tRNA(Pyl) + L-lysine + ATP = L-lysyl-tRNA(Pyl) + diphosphate + AMP." [EC:6.1.1.25, MetaCyc:6.1.1.25-RXN]
GO:0050563
name: trans-feruloyl-CoA synthase activity
def: "Catalysis of the reaction: ferulic acid + CoASH + ATP = trans-feruloyl-CoA + products of ATP breakdown." [EC:6.2.1.34, MetaCyc:6.2.1.34-RXN]
GO:0050564
name: N-(5-amino-5-carboxypentanoyl)-L-cysteinyl-D-valine synthase activity
def: "Catalysis of the reaction: L-2-aminoadipate + L-cysteine + L-valine + 3 ATP + H(2)O = N-[(5S)-5-amino-5-carboxypentanoyl]-L-cysteinyl-D-valine + 3 AMP + 3 diphosphate + 6 H(+)." [EC:6.3.2.26, RHEA:23199]
GO:0050565
name: aerobactin synthase activity
def: "Catalysis of the reaction: 2 N(6)-acetyl-N(6)-hydroxy-L-lysine + 4 ATP + citrate + 2 H(2)O = 4 ADP + aerobactin + 8 H(+) + 4 phosphate." [EC:6.3.2.27, RHEA:11763]
GO:0050566
name: asparaginyl-tRNA synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: L-glutamine + aspartyl-tRNA(Asn) + ATP = L-glutamate + asparaginyl-tRNA(Asn) + phosphate + ADP." [EC:6.3.5.6, MetaCyc:6.3.5.6-RXN]
GO:0050567
name: glutaminyl-tRNA synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: L-glutamine + glutamyl-tRNA(Gln) + ATP = L-glutamate + glutaminyl-tRNA(Gln) + phosphate + ADP." [EC:6.3.5.7, MetaCyc:6.3.5.7-RXN]
GO:0050568
name: protein-glutamine glutaminase activity
def: "Catalysis of the reaction: protein L-glutamine + H2O = protein L-glutamate + NH3." [EC:3.5.1.44, MetaCyc:CHEBDEAMID-RXN]
GO:0050569
name: glycolaldehyde dehydrogenase activity
def: "Catalysis of the reaction: glycolaldehyde + H(2)O + NAD(+) = glycolate + 2 H(+) + NADH." [EC:1.2.1.21, RHEA:20004]
GO:0050570
name: 4-hydroxythreonine-4-phosphate dehydrogenase activity
def: "Catalysis of the reaction: 4-(phosphonooxy)-threonine + NAD+ = 2-amino-3-oxo-4-phosphonooxybutyrate + NADH + H+." [EC:1.1.1.262, MetaCyc:1.1.1.262-RXN]
GO:0050571
name: 1,5-anhydro-D-fructose reductase activity
def: "Catalysis of the reaction: 1,5-anhydro-D-glucitol + NADP(+) = 1,5-anhydro-D-fructose + H(+) + NADPH." [EC:1.1.1.263, RHEA:20668]
GO:0050572
name: L-idonate 5-dehydrogenase activity
def: "Catalysis of the reaction: L-idonate + NAD(P)+ = 5-dehydrogluconate + NAD(P)H + H+." [EC:1.1.1.264, MetaCyc:1.1.1.264-RXN]
GO:0050573
name: dTDP-4-dehydro-6-deoxyglucose reductase activity
def: "Catalysis of the reaction: dTDP-D-fucose + NADP(+) = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + H(+) + NADPH." [EC:1.1.1.266, RHEA:21903]
GO:0050574
name: 2-(R)-hydroxypropyl-CoM dehydrogenase activity
def: "Catalysis of the reaction: 2-(R)-hydroxypropyl-coenzyme M + NAD(+) = 2-oxopropyl-coenzyme M + H(+) + NADH." [EC:1.1.1.268, RHEA:13252]
GO:0050575
name: 2-(S)-hydroxypropyl-CoM dehydrogenase activity
def: "Catalysis of the reaction: 2-(S)-hydroxypropyl-coenzyme M + NAD(+) = 2-oxopropyl-coenzyme M + H(+) + NADH." [EC:1.1.1.269, RHEA:21055]
GO:0050577
name: GDP-L-fucose synthase activity
def: "Catalysis of the reaction: GDP-L-fucose + NAD+ = GDP-4-dehydro-6-deoxy-D-mannose + NADH + H+." [EC:1.1.1.271, MetaCyc:1.1.1.271-RXN]
GO:0050578
name: (R)-2-hydroxyacid dehydrogenase activity
def: "Catalysis of the reaction: (S)-3-sulfolactate + NAD(P)+ = 3-sulfopyruvate + NAD(P)H + H+." [EC:1.1.1.272, MetaCyc:1.1.1.272-RXN]
GO:0050579
name: vellosimine dehydrogenase activity
def: "Catalysis of the reaction: 10-deoxysarpagine + NADP(+) = H(+) + NADPH + vellosimine." [EC:1.1.1.273, RHEA:20032]
GO:0050580
name: 2,5-didehydrogluconate reductase activity
def: "Catalysis of the reaction: 2-dehydro-D-gluconate + NADP+ = 2,5-didehydro-D-gluconate + NADPH + H+." [EC:1.1.1.274, MetaCyc:1.1.1.274-RXN]
GO:0050581
name: D-mannitol oxidase activity
def: "Catalysis of the reaction: mannitol + O2 = mannose + H2O2." [EC:1.1.3.40, MetaCyc:1.1.3.40-RXN]
GO:0050582
name: xylitol oxidase activity
def: "Catalysis of the reaction: xylitol + O2 = xylose + H2O2." [EC:1.1.3.41, MetaCyc:1.1.3.41-RXN]
GO:0050583
name: hydrogen dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: NADP+ + H2 = NADPH + H+." [EC:1.12.1.3, MetaCyc:1.12.1.3-RXN]
GO:0050584
name: linoleate 11-lipoxygenase activity
def: "Catalysis of the reaction: linoleate + O(2) = (9Z,11S,12Z)-11-hydroperoxyoctadeca-9,12-dienoate." [EC:1.13.11.45, RHEA:18996]
GO:0050585
name: 4-hydroxymandelate synthase activity
def: "Catalysis of the reaction: 4-hydroxyphenylpyruvate + O2 = 4-hydroxymandelate + CO2." [EC:1.13.11.46, MetaCyc:1.13.11.46-RXN]
GO:0050586
name: 3-hydroxy-2-methylquinolin-4-one 2,4-dioxygenase activity
def: "Catalysis of the reaction: 3-hydroxy-2-methylquinolin-4(1H)-one + H(+) + O(2) = N-acetylanthranilate + CO." [EC:1.13.11.48, RHEA:21575]
GO:0050587
name: chlorite O2-lyase activity
def: "Catalysis of the reaction: chloride + O(2) = chlorite." [EC:1.13.11.49, RHEA:21407]
GO:0050588
name: apo-beta-carotenoid-14',13'-dioxygenase activity
def: "Catalysis of the reaction: 8'-apo-beta-carotenol + O(2) = (E,E)-7-hydroxy-6-methylhepta-3,5-dienal + 14'-apo-beta-carotenal." [EC:1.13.12.12, RHEA:26026]
GO:0050589
name: leucocyanidin oxygenase activity
def: "Catalysis of the reaction: leucocyanidin + 2-oxoglutarate + O2 = cis- or trans-dihydroquercetin + succinate + CO2 + 2 H2O." [EC:1.14.11.19, MetaCyc:1.14.11.19-RXN]
GO:0050590
name: desacetoxyvindoline 4-hydroxylase activity
def: "Catalysis of the reaction: desacetoxyvindoline + 2-oxoglutarate + O2 = desacetylvindoline + succinate + CO2." [EC:1.14.11.20, MetaCyc:1.14.11.20-RXN]
GO:0050591
name: quinine 3-monooxygenase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + quinine = 3-hydroxyquinine + H(2)O + NADP(+)." [EC:1.14.13.67, RHEA:20152]
GO:0050592
name: 4-hydroxyphenylacetaldehyde oxime monooxygenase activity
def: "Catalysis of the reaction: (Z)-(4-hydroxyphenyl)acetaldehyde oxime + H(+) + NADPH + O(2) = (S)-4-hydroxymandelonitrile + 2 H(2)O + NADP(+)." [EC:1.14.13.68, RHEA:18404]
GO:0050593
name: N-methylcoclaurine 3'-monooxygenase activity
def: "Catalysis of the reaction: (S)-N-methylcoclaurine + H(+) + NADPH + O(2) = (S)-3'-hydroxy-N-methylcoclaurine + H(2)O + NADP(+)." [EC:1.14.13.71, RHEA:16652]
GO:0050594
name: tabersonine 16-hydroxylase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + tabersonine = 16-hydroxytabersonine + H(2)O + NADP(+)." [EC:1.14.13.73, RHEA:14136]
GO:0050595
name: 7-deoxyloganin 7-hydroxylase activity
def: "Catalysis of the reaction: 7-deoxyloganin + NADPH + H+ + O2 = loganin + NADP+ + H2O." [EC:1.14.13.74, MetaCyc:1.14.13.74-RXN]
GO:0050596
name: vinorine hydroxylase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + vinorine = H(2)O + NADP(+) + vomilenine." [EC:1.14.13.75, RHEA:17260]
GO:0050597
name: taxane 10-beta-hydroxylase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + taxa-4(20),11-dien-5alpha-yl acetate = 10beta-hydroxytaxa-4(20),11-dien-5alpha-yl acetate + H(2)O + NADP(+)." [EC:1.14.13.76, RHEA:15244]
GO:0050598
name: taxane 13-alpha-hydroxylase activity
def: "Catalysis of the reaction: H(+) + NADPH + O(2) + taxa-4(20),11-dien-5alpha-ol = H(2)O + NADP(+) + taxa-4(20),11-dien-5alpha,13alpha-diol." [EC:1.14.13.77, RHEA:18952]
GO:0050599
name: deacetoxycephalosporin-C synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + O(2) + penicillin N = CO(2) + deacetoxycephalosporin C + H(2)O + succinate." [EC:1.14.20.1, RHEA:20751]
GO:0050600
name: myristoyl-CoA 11-(E) desaturase activity
def: "Catalysis of the reaction: myristoyl-CoA + NAD(P)H + H+ + O2 = (E)-11-tetradecenoyl-CoA + NAD(P)+ + 2 H2O." [EC:1.14.99.31, MetaCyc:1.14.99.31-RXN]
GO:0050601
name: myristoyl-CoA 11-(Z) desaturase activity
def: "Catalysis of the reaction: myristoyl-CoA + NAD(P)H + H+ + O2 = (Z)-11-tetradecenoyl-CoA + NAD(P)+ + 2 H2O." [EC:1.14.99.32, MetaCyc:1.14.99.32-RXN]
GO:0050602
name: monoprenyl isoflavone epoxidase activity
def: "Catalysis of the reaction: O2 + NADPH + H+ + 7-O-methylluteone = H2O + NADP+ + dihydrofurano derivatives." [EC:1.14.99.34, MetaCyc:1.14.99.34-RXN]
GO:0050603
name: thiophene-2-carbonyl-CoA monooxygenase activity
def: "Catalysis of the reaction: AH(2) + O(2) + thiophene-2-carbonyl-CoA = 5-hydroxythiophene-2-carbonyl-CoA + A + H(2)O + H(+)." [EC:1.14.99.35, RHEA:18932]
GO:0050604
name: taxadiene 5-alpha-hydroxylase activity
def: "Catalysis of the reaction: AH(2) + O(2) + taxa-4,11-diene = A + H(2)O + taxa-4(20),11-dien-5alpha-ol." [EC:1.14.99.37, RHEA:14052]
GO:0050605
name: superoxide reductase activity
def: "Catalysis of the reaction: superoxide + reduced rubredoxin + 2 H+ = H2O2 + rubredoxin." [EC:1.15.1.2, MetaCyc:1.15.1.2-RXN]
GO:0050606
name: 4-carboxy-2-hydroxymuconate semialdehyde hemiacetal dehydrogenase activity
def: "Catalysis of the reaction: 4-carboxy-2-hydroxymuconate semialdehyde hemiacetal + NADP(+) = 2-oxo-2H-pyran-4,6-dicarboxylate + H(+) + NADPH." [RHEA:29590]
GO:0050607
name: mycothiol-dependent formaldehyde dehydrogenase activity
def: "Catalysis of the reaction: formaldehyde + mycothiol + NAD+ = S-formylmycothiol + NADH + H+." [EC:1.2.1.66, MetaCyc:1.2.1.66-RXN]
GO:0050608
name: vanillin dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + NAD(+) + vanillin = 2 H(+) + NADH + vanillate." [EC:1.2.1.67, RHEA:13312]
GO:0050609
name: phosphonate dehydrogenase activity
def: "Catalysis of the reaction: H(2)O + NAD(+) + phosphonate = 2 H(+) + NADH + phosphate." [EC:1.20.1.1, RHEA:13176]
GO:0050610
name: methylarsonate reductase activity
def: "Catalysis of the reaction: 2 glutathione + H(+) + methylarsonate = glutathione disulfide + H(2)O + methylarsonous acid." [EC:1.20.4.2, RHEA:15972]
GO:0050611
name: arsenate reductase (azurin) activity
def: "Catalysis of the reaction: H(2)O + arsenite + 2 oxidized azurin = 2 H(+) + 2 reduced azurin + arsenate." [EC:1.20.9.1, MetaCyc:1.20.98.1-RXN]
GO:0050612
name: arsenate reductase (donor) activity
def: "Catalysis of the reaction: A + arsenite + H(2)O = AH(2) + arsenate + 2 H(+)." [EC:1.20.99.1, RHEA:18452]
GO:0050613
name: delta14-sterol reductase activity
def: "Catalysis of the reaction: NADP+ + 4,4-dimethyl-5-alpha-cholesta-8,24-dien-3-beta-ol = NADPH + H+ + 4,4-dimethyl-5-alpha-cholesta-8,14,24-trien-3-beta-ol." [EC:1.3.1.70, MetaCyc:1.3.1.70-RXN]
GO:0050614
name: delta24-sterol reductase activity
def: "Catalysis of the reaction: NADP+ + 5-alpha-cholest-7-en-3-beta-ol = NADPH + H+ + 5-alpha-cholesta-7,24-dien-3-beta-ol." [EC:1.3.1.72, MetaCyc:1.3.1.72-RXN]
GO:0050615
name: 1,2-dihydrovomilenine reductase activity
def: "Catalysis of the reaction: 17-O-acetylnorajmaline + NADP(+) = 1,2-dihydrovomilenine + H(+) + NADPH." [EC:1.3.1.73, RHEA:12323]
GO:0050616
name: secologanin synthase activity
def: "Catalysis of the reaction: loganin + NADPH + H+ + O2 = secologanin + NADP+ + 2 H2O." [EC:1.3.3.9, MetaCyc:1.3.3.9-RXN]
GO:0050617
name: 15,16-dihydrobiliverdin:ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: 15,16-dihydrobiliverdin + oxidized ferredoxin = biliverdin IXa + reduced ferredoxin." [EC:1.3.7.2, MetaCyc:1.3.7.2-RXN]
GO:0050618
name: phycoerythrobilin:ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: (3Z)-phycoerythrobilin + oxidized ferredoxin = 15,16-dihydrobiliverdin + reduced ferredoxin." [EC:1.3.7.3, MetaCyc:1.3.7.3-RXN]
GO:0050619
name: phytochromobilin:ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: (3Z)-phytochromobilin + oxidized ferredoxin = biliverdin IXa + reduced ferredoxin." [EC:1.3.7.4, MetaCyc:1.3.7.4-RXN]
GO:0050620
name: phycocyanobilin:ferredoxin oxidoreductase activity
def: "Catalysis of the reaction: (3Z)-phycocyanobilin + oxidized ferredoxin = biliverdin IXa + reduced ferredoxin." [EC:1.3.7.5, MetaCyc:1.3.7.5-RXN]
GO:0050621
name: tryptophan alpha,beta-oxidase activity
def: "Catalysis of the reaction: L-tryptophan + O(2) = alpha,beta-didehydrotryptophan + H(2)O(2) + H(+)." [EC:1.3.3.10, RHEA:19904]
GO:0050622
name: glycine dehydrogenase (cyanide-forming) activity
def: "Catalysis of the reaction: glycine + 2 A = HCN + CO2 + 2 AH2." [EC:1.4.99.5, MetaCyc:1.4.99.5-RXN]
GO:0050623
name: berberine reductase activity
def: "Catalysis of the reaction: (R)-canadine + 2 NADP(+) = berberine + H(+) + 2 NADPH." [EC:1.5.1.31, RHEA:21271]
GO:0050624
name: vomilenine reductase activity
def: "Catalysis of the reaction: 1,2-dihydrovomilenine + NADP(+) = H(+) + NADPH + vomilenine." [EC:1.5.1.32, RHEA:16412]
GO:0050625
name: 2-hydroxy-1,4-benzoquinone reductase activity
def: "Catalysis of the reaction: 2-hydroxy-1,4-benzoquinone + 2 H(+) + NADH = benzene-1,2,4-triol + NAD(+)." [EC:1.6.5.7, RHEA:12431]
GO:0050626
name: trimethylamine-N-oxide reductase (cytochrome c) activity
def: "Catalysis of the reaction: trimethylamine + 2 (ferricytochrome c)-subunit + H2O = trimethylamine-N-oxide + 2 (ferrocytochrome c)-subunit + 2 H+." [EC:1.7.2.3, MetaCyc:1.7.2.3-RXN]
GO:0050627
name: mycothione reductase activity
def: "Catalysis of the reaction: NAD(P)+ + mycothiol = NAD(P)H + H+ + mycothione." [EC:1.8.1.15, MetaCyc:1.8.1.15-RXN]
GO:0050628
name: 2-oxopropyl-CoM reductase (carboxylating) activity
def: "Catalysis of the reaction: acetoacetate + coenzyme M + NADP(+) = 2-oxopropyl-coenzyme M + CO(2) + NADPH." [EC:1.8.1.5, RHEA:16980]
GO:0050629
name: tetrachloroethene reductive dehalogenase activity
def: "Catalysis of the reaction: trichloroethene + chloride + acceptor = tetrachloroethene + reduced acceptor." [EC:1.97.1.8, MetaCyc:1.97.1.8-RXN]
GO:0050630
name: (iso)eugenol O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + isoeugenol = S-adenosyl-L-homocysteine + isomethyleugenol." [EC:2.1.1.146, MetaCyc:2.1.1.146-RXN]
GO:0050631
name: corydaline synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 2 NADPH + palmatine = S-adenosyl-L-homocysteine + corydaline + 2 NADP(+)." [EC:2.1.1.147, RHEA:14776]
GO:0050632
name: propionyl-CoA C2-trimethyltridecanoyltransferase activity
def: "Catalysis of the reaction: 4,8,12-trimethyltridecanoyl-CoA + propanoyl-CoA = 3-oxopristanoyl-CoA + CoA." [EC:2.3.1.154, RHEA:10411]
GO:0050633
name: acetyl-CoA C-myristoyltransferase activity
def: "Catalysis of the reaction: myristoyl-CoA + acetyl-CoA = 3-oxopalmitoyl-CoA + CoA." [EC:2.3.1.155, MetaCyc:2.3.1.155-RXN]
GO:0050634
name: phloroisovalerophenone synthase activity
def: "Catalysis of the reaction: isovaleryl-CoA + 3 malonyl-CoA = 4 CoASH + 3 CO2 + 3-methyl-1-(2,4,6-trihydroxyphenyl)butan-1-one." [EC:2.3.1.156, MetaCyc:2.3.1.156-RXN]
GO:0050635
name: acridone synthase activity
def: "Catalysis of the reaction: N-methylanthranilyl-CoA + 3 H(+) + 3 malonyl-CoA = 1,3-dihydroxy-N-methylacridone + 3 CO(2) + 4 CoA + H(2)O." [EC:2.3.1.159, RHEA:22227]
GO:0050636
name: vinorine synthase activity
def: "Catalysis of the reaction: 16-epivellosimine + acetyl-CoA = CoA + vinorine." [EC:2.3.1.160, RHEA:24019]
GO:0050637
name: lovastatin nonaketide synthase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine(1+) + acetyl-CoA + 18 H(+) + 8 malonyl-CoA + 11 NADPH = S-adenosyl-L-homocysteine + 8 CO(2) + 9 CoA + dihydromonacolin L + 6 H(2)O + 11 NADP(+)." [EC:2.3.1.161, RHEA:18568]
GO:0050638
name: taxadien-5-alpha-ol O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + taxa-4(20),11-dien-5alpha-ol = CoA + taxa-4(20),11-dien-5alpha-yl acetate." [EC:2.3.1.162, RHEA:22031]
GO:0050639
name: 10-hydroxytaxane O-acetyltransferase activity
def: "Catalysis of the reaction: 10-desacetyltaxuyunnanin C + acetyl-CoA = CoA + taxuyunnanin C." [EC:2.3.1.163, RHEA:18840]
GO:0050640
name: isopenicillin-N N-acyltransferase activity
def: "Catalysis of the reaction: phenylacetyl-CoA + isopenicillin N + H2O = CoA + penicillin G + L-2-aminohexanedioate." [EC:2.3.1.164, MetaCyc:2.3.1.164-RXN]
GO:0050641
name: 6-methylsalicylic acid synthase activity
def: "Catalysis of the reaction: acetyl-CoA + 3 H(+) + 3 malonyl-CoA + NADPH = 6-methylsalicylate + 3 CO(2) + 4 CoA + H(2)O + NADP(+)." [EC:2.3.1.165, RHEA:12243]
GO:0050642
name: 2-alpha-hydroxytaxane 2-O-benzoyltransferase activity
def: "Catalysis of the reaction: 10-deacetyl-2-debenzoylbaccatin III + benzoyl-CoA = 10-deacetylbaccatin III + CoA." [EC:2.3.1.166, RHEA:18744]
GO:0050643
name: 10-deacetylbaccatin III 10-O-acetyltransferase activity
def: "Catalysis of the reaction: 10-deacetylbaccatin III + acetyl-CoA = baccatin III + CoA." [EC:2.3.1.167, RHEA:20140]
GO:0050644
name: cis-p-coumarate glucosyltransferase activity
def: "Catalysis of the reaction: cis-4-coumarate + UDP-D-glucose = 4'-O-beta-D-glucosyl-cis-4-coumarate + H(+) + UDP." [EC:2.4.1.209, RHEA:13132]
GO:0050645
name: limonoid glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + limonin = glucosyl-limonin + UDP." [EC:2.4.1.210]
GO:0050646
name: 5-oxo-6E,8Z,11Z,14Z-icosatetraenoic acid binding
def: "Interacting selectively and non-covalently with 5-oxo-6E,8Z,11Z,14Z-icosatetraenoic acid, a straight-chain fatty acid with twenty carbon atoms and four double bonds." [GOC:ai]
GO:0050647
name: 5-hydroxy-6E,8Z,11Z,14Z-icosatetraenoic acid binding
def: "Interacting selectively and non-covalently with 5-hydroxy-6E,8Z,11Z,14Z-icosatetraenoic acid, a straight-chain fatty acid with twenty carbon atoms and four double bonds." [GOC:ai]
GO:0050648
name: 5(S)-hydroxyperoxy-6E,8Z,11Z,14Z-icosatetraenoic acid binding
def: "Interacting selectively and non-covalently with 5(S)-hydroxyperoxy-6E,8Z,11Z,14Z-icosatetraenoic acid, a straight-chain fatty acid with twenty carbon atoms and four double bonds." [GOC:ai]
GO:0050649
name: testosterone 6-beta-hydroxylase activity
def: "Catalysis of the reaction: testosterone + donor-H2 + O2 = 6-beta-hydroxytestosterone + H2O." [GOC:ai, PMID:11726664]
GO:0050656
name: 3'-phosphoadenosine 5'-phosphosulfate binding
def: "Interacting selectively and non-covalently with 3'-phosphoadenosine 5'-phosphosulfate (PAPS), a naturally occurring mixed anhydride. It is an intermediate in the formation of a variety of sulfo compounds in biological systems." [GOC:ai]
GO:0050659
name: N-acetylgalactosamine 4-sulfate 6-O-sulfotransferase activity
def: "Catalysis of the reactions: 3'-phosphoadenylyl sulfate + dermatan = adenosine 3',5'-bisphosphate + dermatan 6'-sulfate and 3'-phosphoadenylyl sulfate + chondroitin = adenosine 3',5'-bisphosphate + chondroitin 6'-sulfate." [EC:2.8.2.33]
GO:0050660
name: flavin adenine dinucleotide binding
def: "Interacting selectively and non-covalently with FAD, flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes, in either the oxidized form, FAD, or the reduced form, FADH2." [CHEBI:24040, GOC:ai, GOC:imk, ISBN:0198506732]
GO:0050661
name: NADP binding
def: "Interacting selectively and non-covalently with nicotinamide-adenine dinucleotide phosphate, a coenzyme involved in many redox and biosynthetic reactions; binding may be to either the oxidized form, NADP+, or the reduced form, NADPH." [GOC:ai]
GO:0050662
name: coenzyme binding
def: "Interacting selectively and non-covalently with a coenzyme, any of various nonprotein organic cofactors that are required, in addition to an enzyme and a substrate, for an enzymatic reaction to proceed." [ISBN:0198506732]
GO:0050664
name: oxidoreductase activity, acting on NAD(P)H, oxygen as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which NADH or NADPH acts as a hydrogen or electron donor and reduces an oxygen molecule." [EC:1.6.3.-]
GO:0050681
name: androgen receptor binding
def: "Interacting selectively and non-covalently with an androgen receptor." [GOC:ai]
GO:0050682
name: AF-2 domain binding
def: "Interacting selectively and non-covalently with the AF-2 domain of a protein, a highly conserved ligand-dependent transactivation domain which is essential for receptor-mediated transcriptional activation." [PMID:9682036]
GO:0050683
name: AF-1 domain binding
def: "Interacting selectively and non-covalently with the AF-1 domain of a protein, a ligand-independent transactivation domain which is required for the full transcriptional activity of the receptor." [PMID:9682036]
GO:0050692
name: DBD domain binding
def: "Interacting selectively and non-covalently with the DBD, DNA binding domain, of a protein. The DNA binding domain of the vitamin D receptor, one of a family of receptors with the DBD, is split into three regions, the P, D and T boxes. Residues that are critical for target sequence selectivity form the P-box. The D-box contains residues that are important for homodimerization of class I nuclear receptors. The T-box is essential for both DNA-binding and transactivation of the VDR; this region may also be important for dimerization with RXR for class II nuclear receptors." [PMID:9682036]
GO:0050693
name: LBD domain binding
def: "Interacting selectively and non-covalently with the LBD, the ligand binding domain found in nuclear receptors. In general, the LBDs consist of three layers comprised of twelve alpha-helices and several beta-strands that are organized around a lipophilic ligand-binding pocket." [PMID:9682036]
GO:0050694
name: galactose 3-O-sulfotransferase activity
def: "Catalysis of the reaction: N-acetyllactosamine + 3'-phosphoadenosine 5'-phosphosulfate = 3-sulfo-N-acetyllactosamine + adenosine 3',5'-bisphosphate. N-acetyllactosamine residues are found in a number of different carbohydrate types. N-acetyllactosamine can also be written as Gal-beta-(1,4)-GlcNAc." [GOC:ai, PMID:11323440, PMID:11356829]
GO:0050695
name: benzoylformate decarboxylase activity
def: "Catalysis of the reaction: benzoylformate = benzaldehyde + CO2." [EC:4.1.1.7, MetaCyc:BENZOYLFORMATE-DECARBOXYLASE-RXN]
GO:0050697
name: 1,1,2-trichloroethene reductive dehalogenase activity
def: "Catalysis of the reaction: trichloroethene + 2 H+ + 2 e- = HCl + 1,2-dichloroethene." [MetaCyc:TCEREDCHLOR-RXN, UM-BBD_enzymeID:e0271]
GO:0050698
name: proteoglycan sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + proteoglycan = adenosine 3',5'-bisphosphate + proteoglycan sulfate. A proteoglycan is a glycoprotein whose carbohydrate units are glycosaminoglycans." [EC:2.8.2.-, GOC:ai]
GO:0050699
name: WW domain binding
def: "Interacting selectively and non-covalently with a WW domain of a protein, a small module composed of 40 amino acids and plays a role in mediating protein-protein interactions via proline-rich regions." [PMID:14531730]
GO:0050700
name: CARD domain binding
def: "Interacting selectively and non-covalently with a CARD (N-terminal caspase recruitment) domain, a protein-protein interaction domain that belongs to the death domain-fold superfamily. These protein molecule families are similar in structure with each consisting of six or seven anti-parallel alpha-helices that form highly specific homophilic interactions between signaling partners. CARD exists in the N-terminal prodomains of several caspases and in apoptosis-regulatory proteins and mediates the assembly of CARD-containing proteins that participate in activation or suppression of CARD carrying members of the caspase family." [PMID:12054670]
GO:0050733
name: RS domain binding
def: "Interacting selectively and non-covalently with an RS domain of a protein; RS domains are usually highly phosphorylated and characterized by the presence of arginine (R)/serine (S) dipeptides. The RS domain promotes protein-protein interactions and directs subcellular localization and, in certain situations, nucleocytoplasmic shuttling of individual SR proteins. They also play a role in splicing." [PMID:11684676, PMID:12215544]
GO:0050734
name: hydroxycinnamoyltransferase activity
def: "Catalysis of the transfer of a hydroxycinnamoyl group to an acceptor molecule." [GOC:ai]
GO:0050735
name: N-malonyltransferase activity
def: "Catalysis of the transfer of a malonyl group to a nitrogen atom on the acceptor molecule." [GOC:ai]
GO:0050736
name: O-malonyltransferase activity
def: "Catalysis of the transfer of a malonyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0050737
name: O-hydroxycinnamoyltransferase activity
def: "Catalysis of the transfer of a hydroxycinnamoyl group to an oxygen atom on the acceptor molecule." [GOC:ai]
GO:0050738
name: fructosyltransferase activity
def: "Catalysis of the transfer of a fructosyl group to an acceptor molecule, typically another carbohydrate or a lipid." [GOC:ai]
GO:0050749
name: apolipoprotein E receptor binding
def: "OBSOLETE. Interacting selectively with an apolipoprotein E receptor." [GOC:ai]
GO:0050750
name: low-density lipoprotein particle receptor binding
def: "Interacting selectively and non-covalently with a low-density lipoprotein receptor." [GOC:ai]
GO:0050780
name: dopamine receptor binding
def: "Interacting selectively and non-covalently with a dopamine receptor." [GOC:ai]
GO:0050781
name: ortho-trichlorophenol reductive dehalogenase activity
def: "Catalysis of the reaction: 2,4,6-trichlorophenol + 2 H+ + 2 e- = 2,4-dichlorophenol + HCl." [GOC:ai, PMID:12697029]
GO:0050782
name: galactose uniporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: galactose (out) = galactose(in)." [GOC:ai, TC:2.A.1.1.6]
GO:0050785
name: advanced glycation end-product receptor activity
def: "Combining with advanced glycation end-products and transmitting the signal to initiate a change in cell activity. Advanced glycation end-products (AGEs) form from a series of chemical reactions after an initial glycation event (a non-enzymatic reaction between reducing sugars and free amino groups of proteins)." [GOC:signaling, PMID:12453678, PMID:12707408, PMID:7592757, PMID:9224812, Wikipedia:RAGE_(receptor)]
GO:0050786
name: RAGE receptor binding
def: "Interacting selectively and non-covalently with the RAGE receptor, the receptor for advanced glycation end-products." [GOC:ai]
GO:0050797
name: thymidylate synthase (FAD) activity
def: "Catalysis of the reaction: 5,10-methylenetetrahydrofolate + dUMP + NADPH + H+ = dTMP + tetrahydrofolate + NADP+." [EC:2.1.1.148]
GO:0050800
name: hydrolase activity, acting on acid anhydrides, acting on GTP, involved in cellular and subcellular movement
def: "OBSOLETE. Catalysis of the hydrolysis of GTP to directly drive the cellular or subcellular transport of a substance." [EC:3.6.5.-, GOC:ai]
GO:0050809
name: diazepam binding
def: "Interacting selectively and non-covalently with diazepam, one of the most widely used benzodiazepine drugs. It is used as an anti-anxiety-hypnotic agent and has the proprietary name Valium." [ISBN:0198506732]
GO:0050811
name: GABA receptor binding
def: "Interacting selectively and non-covalently with the gamma-aminobutyric acid (GABA, 4-aminobutyrate) receptor." [GOC:ai]
GO:0050815
name: phosphoserine binding
def: "Interacting selectively and non-covalently with a phosphorylated serine residue within a protein." [GOC:ai]
GO:0050816
name: phosphothreonine binding
def: "Interacting selectively and non-covalently with a phosphorylated threonine residue within a protein." [GOC:ai]
GO:0050824
name: water binding
def: "Interacting selectively and non-covalently with water (H2O)." [GOC:ai]
GO:0050825
name: ice binding
def: "Interacting selectively and non-covalently with ice, water reduced to the solid state by cold temperature. It is a white or transparent colorless substance, crystalline, brittle, and viscoidal." [GOC:curators]
GO:0050827
name: toxin receptor binding
def: "Interacting selectively and non-covalently with a toxin receptor, a receptor for substances that cause injury to living organisms." [GOC:ai]
GO:0050833
name: pyruvate transmembrane transporter activity
def: "Catalysis of the transfer of pyruvate, 2-oxopropanoate, from one side of the membrane to the other." [GOC:ai]
GO:0050839
name: cell adhesion molecule binding
def: "Interacting selectively and non-covalently with a cell adhesion molecule." [GOC:ai]
GO:0050840
name: extracellular matrix binding
def: "Interacting selectively and non-covalently with a component of the extracellular matrix." [GOC:ai]
GO:0050897
name: cobalt ion binding
def: "Interacting selectively and non-covalently with a cobalt (Co) ion." [GOC:ai]
GO:0050997
name: quaternary ammonium group binding
def: "Interacting selectively and non-covalently with a quaternary ammonium group, including glycine betaine, choline, carnitine and proline. A quaternary ammonium group is any compound that can be regarded as derived from ammonium hydroxide or an ammonium salt by replacement of all four hydrogen atoms of the NH4+ ion by organic groups." [GOC:ai]
GO:0050998
name: nitric-oxide synthase binding
def: "Interacting selectively and non-covalently with the enzyme nitric-oxide synthase." [GOC:ai]
GO:0051002
name: ligase activity, forming nitrogen-metal bonds
def: "Catalysis of the joining of a metal ion to a molecule via a nitrogen-metal bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate." [EC:6.6, GOC:mah]
GO:0051003
name: ligase activity, forming nitrogen-metal bonds, forming coordination complexes
def: "Catalysis of the ligation of two substances via a nitrogen-metal bond, forming a coordination complex." [EC:6.6.1.-]
GO:0051007
name: squalene-hopene cyclase activity
def: "Catalysis of the reaction: squalene = hop-22(29)-ene." [EC:5.4.99.17]
GO:0051008
name: Hsp27 protein binding
def: "Interacting selectively and non-covalently with Hsp27 proteins, a lightweight heat shock protein." [GOC:ai]
GO:0051009
name: O-acetylhomoserine sulfhydrylase activity
def: "Catalysis of the reaction: O-acetyl-L-homoserine + hydrogen sulfide = homocysteine + acetate." [MetaCyc:ACETYLHOMOSER-CYS-RXN, RHEA:27825]
GO:0051010
name: microtubule plus-end binding
def: "Interacting selectively and non-covalently with the plus end of a microtubule." [GOC:ai, PMID:14557818, PMID:14614826]
GO:0051011
name: microtubule minus-end binding
def: "Interacting selectively and non-covalently with the minus end of a microtubule." [GOC:ai, PMID:14557818, PMID:14614826]
GO:0051015
name: actin filament binding
def: "Interacting selectively and non-covalently with an actin filament, also known as F-actin, a helical filamentous polymer of globular G-actin subunits." [ISBN:0198506732]
GO:0051018
name: protein kinase A binding
def: "Interacting selectively and non-covalently with any subunit of protein kinase A." [GOC:ai]
GO:0051019
name: mitogen-activated protein kinase binding
def: "Interacting selectively and non-covalently with a mitogen-activated protein kinase." [GOC:ai]
GO:0051020
name: GTPase binding
def: "Interacting selectively and non-covalently with a GTPase, any enzyme that catalyzes the hydrolysis of GTP." [GOC:ai]
GO:0051021
name: GDP-dissociation inhibitor binding
def: "Interacting selectively and non-covalently with a GDP-dissociation inhibitor protein." [GOC:ai]
GO:0051022
name: Rho GDP-dissociation inhibitor binding
def: "Interacting selectively and non-covalently with a Rho GDP-dissociation inhibitor protein." [GOC:ai]
GO:0051032
name: nucleic acid transmembrane transporter activity
def: "Catalysis of the transfer of nucleic acids from one side of a membrane to the other. Nucleic acids are single or double-stranded polynucleotides involved in the storage, transmission and transfer of genetic information." [GOC:ai]
GO:0051033
name: RNA transmembrane transporter activity
def: "Catalysis of the transfer of RNA, ribonucleic acid, from one side of a membrane to the other." [GOC:ai]
GO:0051034
name: tRNA transmembrane transporter activity
def: "Catalysis of the transfer of tRNA, transfer ribonucleic acid, from one side of a membrane to the other." [GOC:ai]
GO:0051035
name: DNA transmembrane transporter activity
def: "Catalysis of the transfer of DNA, deoxyribonucleic acid, from one side of a membrane to the other." [GOC:ai]
GO:0051059
name: NF-kappaB binding
def: "Interacting selectively and non-covalently with NF-kappaB, a transcription factor for eukaryotic RNA polymerase II promoters." [GOC:ai]
GO:0051060
name: pullulanase activity
def: "Catalysis of the hydrolysis of (1,6)-alpha-D-glucosidic linkages in pullulan (a linear polymer of alpha-(1,6)-linked maltotriose units) and in amylopectin and glycogen, and the a- and b-limit dextrins of amylopectin and glycogen." [EC:3.2.1.41]
GO:0051061
name: ADP reductase activity
def: "Catalysis of the reaction: dADP + thioredoxin disulfide + H2O = ADP + thioredoxin." [MetaCyc:ADPREDUCT-RXN]
GO:0051062
name: UDP reductase activity
def: "Catalysis of the reaction: dUDP + thioredoxin disulfide + H2O = UDP + thioredoxin." [MetaCyc:UDPREDUCT-RXN]
GO:0051063
name: CDP reductase activity
def: "Catalysis of the reaction: dCDP + thioredoxin disulfide + H2O = CDP + thioredoxin." [MetaCyc:CDPREDUCT-RXN]
GO:0051064
name: TTP reductase activity
def: "Catalysis of the reaction: dTTP + thioredoxin disulfide + H2O = TTP + thioredoxin. Thioredoxin disulfide is the oxidized form of thioredoxin." [MetaCyc:1.17.4.2-RXN]
GO:0051065
name: CTP reductase activity
def: "Catalysis of the reaction: dCTP + thioredoxin disulfide + H2O = CTP + thioredoxin. Thioredoxin disulfide is the oxidized form of thioredoxin." [MetaCyc:1.17.4.2-RXN]
GO:0051073
name: adenosylcobinamide-GDP ribazoletransferase activity
def: "Catalysis of the reaction: adenosylcobinamide-GDP + alpha-ribazole = GMP + adenosylcobalamin." [EC:2.7.8.26, MetaCyc:COBALAMINSYN-RXN]
GO:0051074
name: protein tetramerization activity
def: "OBSOLETE. The formation of a protein tetramer, a macromolecular structure consisting of four noncovalently associated identical or nonidentical subunits." [GOC:ecd]
GO:0051075
name: S-adenosylmethionine:tRNA ribosyltransferase-isomerase activity
def: "Catalysis of the reaction: S-adenosylmethionine + 7-(aminomethyl)-7-deazaguanosine-tRNA = adenine + methionine + epoxyqueuosine-tRNA. 7-(aminomethyl)-7-deazaguanosine-tRNA is also known as preQ1-tRNA, and epoxyqueuosine-tRNA as oQ-tRNA." [PMID:12731872]
GO:0051082
name: unfolded protein binding
def: "Interacting selectively and non-covalently with an unfolded protein." [GOC:ai]
GO:0051087
name: chaperone binding
def: "Interacting selectively and non-covalently with a chaperone protein, a class of proteins that bind to nascent or unfolded polypeptides and ensure correct folding or transport." [http://www.onelook.com]
GO:0051108
name: carnitine-CoA ligase activity
def: "Catalysis of the reaction: D-carnitine + CoA + ATP = AMP + diphosphate + D-carnitinyl-CoA." [MetaCyc:DCARNCOALIG-RXN]
GO:0051109
name: crotonobetaine-CoA ligase activity
def: "Catalysis of the reaction: CoA + crotono-betaine + ATP = AMP + diphosphate + crotonobetainyl-CoA." [MetaCyc:CROTCOALIG-RXN]
GO:0051116
name: cobaltochelatase activity
def: "Catalysis of the reaction: ATP + Co(2+) + H(2)O + hydrogenobyrinate a,c-diamide = ADP + cob(II)yrinate a,c diamide + 4 H(+) + phosphate." [EC:6.6.1.2, RHEA:15344]
GO:0051117
name: ATPase binding
def: "Interacting selectively and non-covalently with an ATPase, any enzyme that catalyzes the hydrolysis of ATP." [GOC:ai]
GO:0051118
name: glucan endo-1,3-alpha-glucosidase activity
def: "Catalysis of the endohydrolysis of (1->3)-alpha-D-glucosidic linkages in isolichenin, pseudonigeran and nigeran." [EC:3.2.1.59]
GO:0051119
name: sugar transmembrane transporter activity
def: "Catalysis of the transfer of a sugar from one side of the membrane to the other. A sugar is any member of a class of sweet, water-soluble, crystallizable carbohydrates, which are the monosaccharides and smaller oligosaccharides." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0051120
name: hepoxilin A3 synthase activity
def: "Catalysis of the reaction: 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid = (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate. 12S-5Z,8Z,10E,14Z-12-hydro(pero)xy-eicosa-5,8,10,14-tetraenoic acid is also known as 12S-HpETE, and (5Z,9E,14Z)-(8,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate as hepoxilin A3." [PMID:15123652]
GO:0051139
name: metal ion:proton antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: metal ion(in) + H+(out) = metal ion(out) + H+(in)." [GOC:mlg]
GO:0051183
name: vitamin transporter activity
def: "Enables the directed movement of vitamins into, out of or within a cell, or between cells. A vitamin is one of a number of unrelated organic substances that occur in many foods in small amounts and that are necessary in trace amounts for the normal metabolic functioning of the body." [GOC:ai]
GO:0051184
name: cofactor transporter activity
def: "Enables the directed movement of a cofactor into, out of or within a cell, or between cells. A cofactor is a substance that is required for the activity of an enzyme or other protein." [GOC:ai]
GO:0051185
name: coenzyme transporter activity
def: "Enables the directed movement of a coenzyme into, out of or within a cell, or between cells. A coenzyme is any of various nonprotein organic cofactors that are required, in addition to an enzyme and a substrate, for an enzymatic reaction to proceed." [GOC:ai]
GO:0051192
name: prosthetic group binding
def: "Interacting selectively and non-covalently with a prosthetic group, the non-amino acid portion of certain protein molecules. Prosthetic groups may be inorganic or organic and are usually required for the biological activity of the protein." [GOC:ai, GOC:vw]
GO:0051212
name: vanadium ion binding
def: "Interacting selectively and non-covalently with vanadium (V) ions." [GOC:ai]
GO:0051213
name: dioxygenase activity
def: "Catalysis of an oxidation-reduction (redox) reaction in which both atoms of oxygen from one molecule of O2 are incorporated into the (reduced) product(s) of the reaction. The two atoms of oxygen may be distributed between two different products." [DOI:10.1016/S0040-4020(03)00944-X, GOC:bf, http://www.onelook.com/]
GO:0051219
name: phosphoprotein binding
def: "Interacting selectively and non-covalently with a phosphorylated protein." [GOC:ai]
GO:0051264
name: mono-olein transacylation activity
def: "Catalysis of the reaction: mono-olein + mono-olein = diolein + glycerol. Mono-olein, also known as mono-oleoylglycerol, is the monoglyceride formed from oleic acid, 9-octodecenoic acid; diolein is also known as dioleoylglycerol." [GOC:ai, PMID:15364929]
GO:0051265
name: diolein transacylation activity
def: "Catalysis of the reaction: diolein + mono-olein = triolein + glycerol. Mono-olein, also known as mono-oleoylglycerol, is the monoglyceride formed from oleic acid, 9-octodecenoic acid; diolein is also known as dioleoylglycerol, and triolein as trioleoylglycerol and olein." [GOC:ai, PMID:15364929]
GO:0051266
name: sirohydrochlorin ferrochelatase activity
def: "Catalysis of the reaction: siroheme + 2 H+ = Fe(2+) + sirohydrochlorin." [EC:4.99.1.4, RHEA:24363]
GO:0051267
name: CP2 mannose-ethanolamine phosphotransferase activity
def: "Catalysis of the reaction: ethanolamine phosphate + Man-alpha-(1,2)-Man-alpha-(1,2)-Man-alpha-(1,6)-R = Man-alpha-(1,2)-Man-alpha-6-P-EtN-(1,2)-Man-alpha-(1,6)-R; R is Man-alpha(1,4)-GlcNH2-inositol-PO4-lipid. This reaction is the transfer of ethanolamine phosphate to C6 of second mannose in the GPI lipid precursor CP2." [PMID:14985347, PMID:15452134]
GO:0051268
name: alpha-keto amide reductase activity
def: "Catalysis of the reaction: alpha-keto amide + 2 H+ (from donor) = (R)-hydroxy amide. Alpha-keto amides are of the form R-CO-CONH2, where R may be aromatic or aliphatic." [GOC:ai, PMID:15564669]
GO:0051269
name: alpha-keto ester reductase activity
def: "Catalysis of the reaction: alpha-keto ester + 2 H+ (from donor) = (R)-hydroxy ester. Alpha-keto esters are of the form R(1)-CO-CO-O-R(2), where the R groups may be aromatic or aliphatic." [GOC:ai, PMID:15564669]
GO:0051287
name: NAD binding
def: "Interacting selectively and non-covalently with nicotinamide adenine dinucleotide, a coenzyme involved in many redox and biosynthetic reactions; binding may be to either the oxidized form, NAD+, or the reduced form, NADH." [GOC:ai]
GO:0051370
name: ZASP binding
def: "OBSOLETE. Interacting selectively and non-covalently with Z-band alternatively spliced PDZ motif protein (ZASP). ZASP is a Z-band protein specifically expressed in heart and skeletal muscle. This protein contains N-terminal PDZ domain and C-terminal LIM domain." [PMID:10427098, PMID:11699871]
GO:0051371
name: muscle alpha-actinin binding
def: "Interacting selectively and non-covalently with muscle isoforms of actinin. Muscle alpha-actinin isoforms are found in skeletal and cardiac muscle and are localized to the Z-disc." [PMID:10984498, PMID:11699871, PMID:15014165]
GO:0051373
name: FATZ binding
def: "Interacting selectively and non-covalently with a member of the FATZ family of proteins, filamin-, actinin-, and telethonin-binding proteins of the Z-disc of striated muscle. FATZ proteins are located in the Z-disc of the sarcomere and are involved in a complex network of interactions with other Z-band components." [PMID:10984498, PMID:11699871]
GO:0051377
name: mannose-ethanolamine phosphotransferase activity
def: "Catalysis of the transfer of ethanolamine phosphate to a mannose residue in the GPI lipid precursor." [PMID:15632136]
GO:0051378
name: serotonin binding
def: "Interacting selectively and non-covalently with serotonin (5-hydroxytryptamine), a monoamine neurotransmitter occurring in the peripheral and central nervous systems, also having hormonal properties." [GOC:ai]
GO:0051379
name: epinephrine binding
def: "Interacting selectively and non-covalently with epinephrine, a hormone produced by the medulla of the adrenal glands that increases heart activity, improves the power and prolongs the action of muscles, and increases the rate and depth of breathing. It is synthesized by the methylation of norepinephrine." [GOC:ai]
GO:0051380
name: norepinephrine binding
def: "Interacting selectively and non-covalently with norepinephrine, (3,4-dihydroxyphenyl-2-aminoethanol), a hormone secreted by the adrenal medulla and a neurotransmitter in the sympathetic peripheral nervous system and in some tracts of the CNS. It is also the biosynthetic precursor of epinephrine." [GOC:ai]
GO:0051381
name: histamine binding
def: "Interacting selectively and non-covalently with histamine, a physiologically active amine, found in plant and animal tissue and released from mast cells as part of an allergic reaction in humans." [GOC:ai]
GO:0051392
name: tRNA N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + cytidine = CoA + N4-acetylcytidine. The cytidine is within the polynucleotide chain of a tRNA." [PMID:15037780]
GO:0051393
name: alpha-actinin binding
def: "Interacting selectively and non-covalently with alpha-actinin, one of a family of proteins that cross-link F-actin as antiparallel homodimers. Alpha-actinin has a molecular mass of 93-103 KDa; at the N-terminus there are two calponin homology domains, at the C-terminus there are two EF-hands. These two domains are connected by the rod domain. This domain is formed by triple-helical spectrin repeats." [PMID:10984498, PMID:11699871, PMID:15014165]
GO:0051397
name: N-terminal basic amino acid aminopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of basic amino acid residues at the N-terminal of an oligopeptide or polypeptide chain." [GOC:ai]
GO:0051398
name: N-terminal lysine aminopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of a lysine residue from the N-terminal of an oligopeptide or polypeptide chain." [GOC:ai, PMID:12799365]
GO:0051399
name: N-terminal arginine aminopeptidase activity
def: "OBSOLETE. Catalysis of the hydrolysis of an arginine residue from the N-terminal of an oligopeptide or polypeptide chain." [GOC:ai, PMID:12799365]
GO:0051400
name: BH domain binding
def: "Interacting selectively and non-covalently with the Bcl-2 homology (BH) domain of a protein. Bcl-2-related proteins share homology in one to four conserved regions designated the Bcl-2 homology (BH) domains BH1, BH2, BH3 and BH4. These domains contribute at multiple levels to the function of these proteins in cell death and survival. Anti-apoptotic members of the Bcl-2 family have four BH domains (BH1-BH4). Pro-apoptotic members have fewer BH domains." [PMID:11048732, PMID:12133724, PMID:9020082, PMID:9704409]
GO:0051401
name: CH domain binding
def: "Interacting selectively and non-covalently with the calponin homology domain of a protein, a domain of 100 residues that occurs in signaling and cytoskeletal proteins." [PMID:11911887, Prosite:PDOC50021]
GO:0051404
name: clostripain activity
def: "OBSOLETE. Catalysis of the cleavage of Arg-Xaa bonds." [EC:3.4.22.8]
GO:0051405
name: microbial collagenase activity
def: "OBSOLETE. Catalysis of the digestion of native collagen in the triple helical region at Xaa-Gly bonds." [EC:3.4.24.3]
GO:0051407
name: glycerone phosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glycerone phosphate(out) + phosphate(in) = glycerone phosphate(in) + phosphate(out)." [GOC:ai]
GO:0051408
name: glyceraldehyde 3-phosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glyceraldehyde 3-phosphate(out) + phosphate(in) = glyceraldehyde 3-phosphate(in) + phosphate(out)." [GOC:ai]
GO:0051411
name: ALP binding
def: "OBSOLETE. Interacting selectively and non-covalently wih ALP, actinin-associated LIM protein of the Z band. ALP is a PDZ/LIM domain protein found in the Z band." [PMID:11699871]
GO:0051416
name: myotilin binding
def: "OBSOLETE. Interacting selectively and non-covalently with myotilin, a structural component of the Z-discs in human skeletal and cardiac muscle. Its C-terminus contains two immunoglobulin-like domains and the unique N-terminal half has a serine-rich region, with numerous potential phosphorylation sites, and a stretch of hydrophobic amino acids. Myotilin forms homodimers and binds to alpha-actinin, F-actin, and filamin C." [PMID:11699871, PMID:15752755]
GO:0051419
name: nebulin binding
def: "OBSOLETE. Interacting selectively and non-covalently with nebulin, a large protein (approximately 800kD) that is anchored at the Z-disc by its C-terminal region and spans the length of the thin filament, ending at the edge of the H-zone." [PMID:11699871]
GO:0051420
name: nebulette binding
def: "OBSOLETE. Interacting selectively and non-covalently with nebulette, a 107 kDa protein associated with the I-Z-I complex of cardiac myofibrils. It shows a high degree of homology with skeletal muscle nebulin." [PMID:11699871]
GO:0051424
name: corticotropin-releasing hormone binding
def: "Interacting selectively and non-covalently with corticotropin-releasing hormone, a polypeptide hormone involved in the stress response. It is released by the hypothalamus and stimulates the release of corticotropin by the anterior pituitary gland." [http://www.onelook.com]
GO:0051425
name: PTB domain binding
def: "Interacting selectively and non-covalently with a phosphotyrosine-binding (PTB) domain of a protein." [Pfam:PF02174.5, PMID:15924411]
GO:0051427
name: hormone receptor binding
def: "Interacting selectively and non-covalently with a receptor for hormones." [GOC:ai]
GO:0051428
name: peptide hormone receptor binding
def: "Interacting selectively and non-covalently with a receptor for peptide hormones." [GOC:ai]
GO:0051429
name: corticotropin-releasing hormone receptor binding
def: "Interacting selectively and non-covalently with a receptor for corticotropin-releasing hormone (CRH), a polypeptide hormone involved in the stress response. It is released by the hypothalamus and stimulates the release of corticotropin by the anterior pituitary gland." [GOC:ai]
GO:0051430
name: corticotropin-releasing hormone receptor 1 binding
def: "Interacting selectively and non-covalently with the corticotropin-releasing hormone receptor 1 (CRHR1). CRHR1 is the major subtype in the pituitary corticotroph, and mediates the stimulatory actions of corticotropin-releasing hormone on corticotropin hormone secretion. CRHR1 are also located in cortical areas of the brain, cerebellum and limbic system." [PMID:15134857]
GO:0051431
name: corticotropin-releasing hormone receptor 2 binding
def: "Interacting selectively and non-covalently with the corticotropin-releasing hormone receptor type 2 (CRHR2). The CRHR2 has several splice variants that are located in sub-cortical areas of the brain and in the periphery." [PMID:15134857]
GO:0051432
name: BH1 domain binding
def: "Interacting selectively and non-covalently with the BH1 domain of a protein of the Bcl-2 family. Proteins that act as inhibitors of apoptosis harbour at least three BH domains: BH1, BH2 and BH3; the BH1 and BH2 domains are found in all death antagonists of the Bcl-2 family but only in one class of death agonists." [PMID:11048732, PMID:12133724, PMID:9020082, PMID:9704409, Prosite:PS01080]
GO:0051433
name: BH2 domain binding
def: "Interacting selectively and non-covalently with the BH2 domain of a protein of the Bcl-2 family. Proteins that act as inhibitors of apoptosis harbour at least three BH domains: BH1, BH2 and BH3; the BH1 and BH2 domains are found in all death antagonists of the Bcl-2 family but only in one class of death agonists." [PMID:11048732, PMID:12133724, PMID:9020082, PMID:9704409, Prosite:PS01258]
GO:0051434
name: BH3 domain binding
def: "Interacting selectively and non-covalently with the BH3 domain of a protein of the Bcl-2 family. The BH3 domain is a potent death domain and has an important role in protein-protein interactions and in cell death." [PMID:11048732, PMID:12133724, PMID:9020082, PMID:9704409, Prosite:PS01259]
GO:0051435
name: BH4 domain binding
def: "Interacting selectively and non-covalently with the BH4 domain of a protein of the Bcl-2 family. All anti-apoptotic proteins contain BH1 and BH2 domains; some also contain an additional N-terminal BH4 domain, which is almost never seen in pro-apoptotic proteins. Loss of the BH4 domain can diminish or abrogate anti-apoptotic function or even impart outright death-promoting properties to the protein." [InterPro:IPR003093, PMID:11048732, PMID:12133724, PMID:9020082, PMID:9704409, Prosite:PS01260, Prosite:PS50063]
GO:0051448
name: gonadotropin-releasing hormone binding
def: "Interacting selectively and non-covalently with gonadotropin-releasing hormone (GnRH), a peptide hormone responsible for the release of follicle-stimulating hormone (FSH) and luteinizing hormone (LH) from the anterior pituitary. GnRH is synthesized and released by the hypothalamus." [GOC:pr, PMID:1984190]
GO:0051449
name: thyrotropin-releasing hormone binding
def: "Interacting selectively and non-covalently with thyrotropin-releasing hormone, a tripeptide hormone that stimulates the release of thyroid-stimulating hormone (TSH) and prolactin by the anterior pituitary and it is produced by the hypothalamus and travels across the median eminence to the pituitary via the pituitary portal system." [GOC:ai]
GO:0051471
name: ectoine transmembrane transporter activity
def: "Catalysis of the transfer of ectoine from one side of the membrane to the other. Ectoine (1,4,5,6-tetrahydro-2-methyl-4-pyrimidinecarboxylic acid) is a tetrahydropyrimidine commonly synthesized by halophilic bacteria." [GOC:ai]
GO:0051474
name: glucosylglycerol transmembrane transporter activity
def: "Catalysis of the transfer of a glucosylglycerol from one side of the membrane to the other. A glucosylglycerol is an alpha-D-glucopyranosyl-alpha-(1,2)-glycerol." [GOC:ai, GOC:mtg_transport, ISBN:0815340729]
GO:0051477
name: mannosylglycerate transmembrane transporter activity
def: "Catalysis of the transfer of a mannosylglycerate from one side of the membrane to the other." [GOC:ai, PMID:15034926]
GO:0051498
name: syn-copalyl diphosphate synthase activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate = 9alpha-copalyl diphosphate." [MetaCyc:RXN-8528]
GO:0051499
name: D-aminoacyl-tRNA deacylase activity
def: "Catalysis of the reaction: D-aminoacyl-tRNA = D-amino acid + tRNA. Hydrolysis of the removal of D-amino acids from residues in charged tRNA." [PMID:14527667]
GO:0051500
name: D-tyrosyl-tRNA(Tyr) deacylase activity
def: "Catalysis of the reaction: D-tyrosyl-tRNA(Tyr) = D-tyrosine + tRNA(Tyr). Hydrolysis of the removal of D-tyrosine from tyrosine residues in charged tRNA." [PMID:14527667]
GO:0051505
name: cholesterol UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + O-glucosyl-cholesterol." [GOC:ai]
GO:0051506
name: ergosterol UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + O-glucosyl-ergosterol." [GOC:ai]
GO:0051507
name: beta-sitosterol UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + O-glucosyl-beta-sitosterol." [GOC:ai]
GO:0051508
name: stigmasterol UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + O-glucosyl-stigmasterol." [GOC:ai]
GO:0051509
name: tomatidine UDP-glucosyltransferase activity
def: "Catalysis of the reaction: UDP-glucose + a sterol = UDP + O-glucosyl-tomatidine." [GOC:ai]
GO:0051525
name: NFAT protein binding
def: "Interacting selectively and non-covalently with NFAT (nuclear factor of activated T cells) proteins, a family of transcription factors. NFAT proteins have crucial roles in the development and function of the immune system." [PMID:15928679]
GO:0051536
name: iron-sulfur cluster binding
def: "Interacting selectively and non-covalently with an iron-sulfur cluster, a combination of iron and sulfur atoms." [GOC:ai]
GO:0051537
name: 2 iron, 2 sulfur cluster binding
def: "Interacting selectively and non-covalently with a 2 iron, 2 sulfur (2Fe-2S) cluster; this cluster consists of two iron atoms, with two inorganic sulfur atoms found between the irons and acting as bridging ligands." [GOC:ai, PMID:15952888, Wikipedia:Iron-sulfur_cluster]
GO:0051538
name: 3 iron, 4 sulfur cluster binding
def: "Interacting selectively and non-covalently with a 3 iron, 4 sulfur (3Fe-4S) cluster; this cluster consists of three iron atoms, with the inorganic sulfur atoms found between the irons and acting as bridging ligands. It is essentially a 4Fe-4S cluster with one iron missing." [GOC:ai, PMID:15952888, Wikipedia:Iron-sulfur_cluster]
GO:0051539
name: 4 iron, 4 sulfur cluster binding
def: "Interacting selectively and non-covalently with a 4 iron, 4 sulfur (4Fe-4S) cluster; this cluster consists of four iron atoms, with the inorganic sulfur atoms found between the irons and acting as bridging ligands." [GOC:ai, PMID:15952888, Wikipedia:Iron-sulfur_cluster]
GO:0051540
name: metal cluster binding
def: "Interacting selectively and non-covalently with a cluster of atoms including both metal ions and nonmetal atoms, usually sulfur and oxygen. Examples include iron-sulfur clusters and nickel-iron-sulfur clusters." [GOC:jsg]
GO:0051566
name: anthocyanidin-3-glucoside rhamnosyltransferase activity
def: "Catalysis of the reaction: anthocyanidin 3-glucoside + UDP-rhamnose = anthocyanidin 3-rutinoside + UDP." [PMID:8130800]
GO:0051575
name: 5'-deoxyribose-5-phosphate lyase activity
def: "Catalysis of the beta-elimination of the 5' deoxyribose-5-phosphate at an abasic site in DNA where a DNA-(apurinic or apyrimidinic site) lyase has already cleaved the C-O-P bond 3' to the apurinic or apyrimidinic site." [PMID:11251121, PMID:16120966]
GO:0051635
name: bacterial cell surface binding
def: "OBSOLETE. Interacting selectively and non-covalently with any component on the surface of a bacterial cell." [GOC:ai]
GO:0051636
name: Gram-negative bacterial cell surface binding
def: "OBSOLETE. Interacting selectively and non-covalently with any component on the surface of a Gram-negative bacterial cell." [GOC:ai]
GO:0051637
name: Gram-positive bacterial cell surface binding
def: "OBSOLETE. Interacting selectively and non-covalently with any component on the surface of a Gram-positive bacterium." [GOC:ai]
GO:0051669
name: fructan beta-fructosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing 2,1- and 2,6-linked beta-D-fructofuranose residues in fructans." [EC:3.2.1.80]
GO:0051670
name: inulinase activity
def: "Catalysis of the endohydrolysis of 2,1-beta-D-fructosidic linkages in inulin." [EC:3.2.1.7]
GO:0051675
name: isopullulanase activity
def: "Catalysis of the hydrolysis of pullulan to isopanose (6-alpha-maltosylglucose)." [EC:3.2.1.57]
GO:0051698
name: saccharopine oxidase activity
def: "Catalysis of the reaction: L-saccharopine + O2 = L-2-aminoadipic 6-semialdehyde + L-glutamate + H2O2." [PMID:16233628]
GO:0051699
name: proline oxidase activity
def: "Catalysis of the reaction: L-proline + O2 + H2O = L-delta1-pyrroline-5-carboxylate + H2O2." [MetaCyc:RXN-821]
GO:0051700
name: fructosyl-amino acid oxidase activity
def: "Catalysis of the reaction: fructosyl-amino acid + O2 = corresponding amino acid + glucosone + H2O2." [PMID:16233628]
GO:0051717
name: inositol-1,3,4,5-tetrakisphosphate 3-phosphatase activity
def: "Catalysis of the reaction: inositol-1,3,4,5-tetrakisphosphate + H2O = inositol-1,4,5-trisphosphate + phosphate." [GOC:bf, MetaCyc:3.1.3.62-RXN]
GO:0051718
name: DNA (cytosine-5-)-methyltransferase activity, acting on CpG substrates
def: "Catalysis of the reaction: S-adenosyl-L-methionine + CpG (in DNA) = S-adenosyl-L-homocysteine + 5-MeCpG (in DNA)." [EC:2.1.1.37, PMID:15689527]
GO:0051719
name: DNA (cytosine-5-)-methyltransferase activity, acting on CpN substrates
def: "Catalysis of the reaction: S-adenosyl-L-methionine + DNA containing CpN = S-adenosyl-L-homocysteine + DNA containing 5-MeCpN." [EC:2.1.1.37, PMID:15689527]
GO:0051720
name: DNA (cytosine-5-)-methyltransferase activity, acting on CpNpG substrates
def: "Catalysis of the reaction: S-adenosyl-L-methionine + DNA containing CpNpG = S-adenosyl-L-homocysteine + DNA containing 5-MeCpNpG." [EC:2.1.1.37, PMID:15689527]
GO:0051721
name: protein phosphatase 2A binding
def: "Interacting selectively and non-covalently with the enzyme protein phosphatase 2A." [GOC:ai]
GO:0051722
name: protein C-terminal methylesterase activity
def: "Catalysis of the reaction: C-terminal protein amino acid methyl ester + H2O = protein amino acid + methanol." [PMID:10318862, PMID:8650216]
GO:0051723
name: protein methylesterase activity
def: "Catalysis of the reaction: protein amino acid methyl ester + H2O = protein amino acid + methanol." [GOC:ai]
GO:0051724
name: NAD transporter activity
def: "Enables the directed movement of nicotinamide adenine dinucleotide into, out of or within a cell, or between cells; transport may be of either the oxidized form, NAD, or the reduced form, NADH." [GOC:ai]
GO:0051730
name: GTP-dependent polyribonucleotide 5'-hydroxyl-kinase activity
def: "Catalysis of the reaction: GTP + 5'-dephospho-RNA = GDP + 5'-phospho-RNA." [EC:2.7.1.78, PMID:8428918]
GO:0051731
name: polynucleotide 5'-hydroxyl-kinase activity
def: "Catalysis of the reaction: NTP + 5'-dephosphopolynucleotide = NDP + 5'-phosphopolynucleotide. The polynucleotide may be DNA or RNA." [EC:2.7.1.78]
GO:0051732
name: polyribonucleotide kinase activity
def: "Catalysis of the reaction: NTP + 5'-dephospho-RNA = NDP + 5'-phospho-RNA." [EC:2.7.1.78]
GO:0051733
name: polydeoxyribonucleotide kinase activity
def: "Catalysis of the reaction: NTP + 5'-dephospho-DNA = NDP + 5'-phospho-DNA." [EC:2.7.1.78]
GO:0051734
name: ATP-dependent polynucleotide kinase activity
def: "Catalysis of the reaction: ATP + 5'-dephosphopolynucleotide = ADP + 5'-phosphopolynucleotide. The polynucleotide may be DNA or RNA." [EC:2.7.1.78]
GO:0051735
name: GTP-dependent polynucleotide kinase activity
def: "Catalysis of the reaction: GTP + 5'-dephosphopolynucleotide = GDP + 5'-phosphopolynucleotide. The polynucleotide may be DNA or RNA." [EC:2.7.1.78]
GO:0051736
name: ATP-dependent polyribonucleotide 5'-hydroxyl-kinase activity
def: "Catalysis of the reaction: ATP + 5'-dephospho-RNA = ADP + 5'-phospho-RNA." [EC:2.7.1.78]
GO:0051737
name: GTP-dependent polydeoxyribonucleotide 5'-hydroxyl-kinase activity
def: "Catalysis of the reaction: GTP + 5'-dephospho-DNA = GDP + 5'-phospho-DNA." [EC:2.7.1.78, PMID:8428918]
GO:0051738
name: xanthophyll binding
def: "Interacting selectively and non-covalently with xanthophylls, any of several neutral yellow to orange carotenoid pigments containing oxygen." [ISBN:0122146743]
GO:0051740
name: ethylene binding
def: "Interacting selectively and non-covalently with ethylene (C2-H4, ethene), a simple hydrocarbon gas that can function in plants as a growth regulator." [GOC:ai]
GO:0051741
name: 2-methyl-6-phytyl-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-methyl-6-phytyl-1,4-benzoquinone + S-adenosyl-methionine = 2,3-dimethyl-6-phytyl-1,4-benzoquinone + S-adenosyl-homocysteine." [MetaCyc:RXN-2542]
GO:0051742
name: 2-methyl-6-solanyl-1,4-benzoquinone methyltransferase activity
def: "Catalysis of the reaction: 2-methyl-6-solanyl-1,4-benzoquinone + S-adenosyl-methionine = 2,3-dimethyl-6-solanyl-1,4-benzoquinone + S-adenosyl-homocysteine." [MetaCyc:RXN-2762]
GO:0051743
name: red chlorophyll catabolite reductase activity
def: "Catalysis of the reaction: red chlorophyll catabolite + reduced ferredoxin + 2 H+ = primary fluorescent catabolite + oxidized ferredoxin. This reaction is the reduction of the C20/C1 double bond in the pyrrole system of red chlorophyll catabolite (RCC) to a colorless tetrapyrrole (pFCC) with a strong blue fluorescence." [PMID:10743659]
GO:0051744
name: 3,8-divinyl protochlorophyllide a 8-vinyl reductase activity
def: "Catalysis of the reaction: divinyl protochlorophyllide a + NADPH + H+ = monovinyl protochlorophyllide a + NADP+." [MetaCyc:RXN1F-72]
GO:0051745
name: 4-hydroxy-3-methylbut-2-en-1-yl diphosphate reductase activity
def: "Catalysis of the reaction: (E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate + NAD(P)H + H+ = isopentenyl diphosphate + NAD(P)+ + H2O. Note that (E)-4-hydroxy-3-methylbut-2-en-1-yl diphosphate is an alternative name for 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate." [EC:1.17.1.2]
GO:0051746
name: thalianol synthase activity
def: "Catalysis of the cyclization of 3(S)-oxidosqualene to (3S,13S,14R)-malabarica-8,17,21-trien-3-ol (thalianol)." [PMID:15125655]
GO:0051747
name: cytosine C-5 DNA demethylase activity
def: "Catalysis of the reaction: methyl-dCpdG DNA + H2O = dCpdG DNA + methanol. This reaction is the hydrolytic removal of the methyl group on the 5 position of cytosine in DNA." [PMID:10050851]
GO:0051748
name: UTP-monosaccharide-1-phosphate uridylyltransferase activity
def: "Catalysis of the reaction: UTP + a monosaccharide 1-phosphate = diphosphate + UDP-monosaccharide." [EC:2.7.7.64, PMID:15326166]
GO:0051749
name: indole acetic acid carboxyl methyltransferase activity
def: "Catalysis of the reaction: indole acetic acid + S-adenosyl-methionine = methyl indole acetic acid ester + S-adenosyl-homocysteine." [PMID:16169896]
GO:0051750
name: delta3,5-delta2,4-dienoyl-CoA isomerase activity
def: "Catalysis of the isomerization of 3,5-dienoyl-CoA to 2,4-dienoyl-CoA." [PMID:16040662]
GO:0051751
name: alpha-1,4-mannosyltransferase activity
def: "Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->4) linkage." [PMID:15772281]
GO:0051752
name: phosphoglucan, water dikinase activity
def: "Catalysis of the reaction: ATP + [phospho-alpha-glucan] + H2O = AMP + O-phospho-[phospho-alpha-glucan] + phosphate." [EC:2.7.9.5, PMID:15618411]
GO:0051753
name: mannan synthase activity
def: "Catalysis of the reaction: mannan(n) + GDP-mannose = mannan(n+1) + GDP. This reaction is the formation of the beta-(1->4)-linked mannan backbone in substrates such as galactomannan." [PMID:14726589]
GO:0051765
name: inositol tetrakisphosphate kinase activity
def: "Catalysis of the reaction: inositol tetrakisphosphate + ATP = inositol pentakisphosphate + ADP." [GOC:ai]
GO:0051766
name: inositol trisphosphate kinase activity
def: "Catalysis of the reaction: inositol trisphosphate + ATP = inositol tetrakisphosphate + ADP." [GOC:ai]
GO:0051777
name: ent-kaurenoate oxidase activity
def: "Catalysis of the reaction: ent-kaurenoate + NADPH + O2 = ent-7-alpha-hydroxykaurenoate + NADP+ + H2O. This is the first of three successive reactions resulting in the oxidation of ent-kaurenoate (ent-kaurenoic acid) to gibberellin 12 (GA12)." [EC:1.14.13.79, MetaCyc:RXN1F-159]
GO:0051778
name: ent-7-alpha-hydroxykaurenoate oxidase activity
def: "Catalysis of the reaction: ent-7-alpha-hydroxykaurenoate + NADPH + H+ + O2 = gibberellin 12-aldehyde + NADP+ + 2 H2O. This is the second of three successive reactions resulting in the oxidation of ent-kaurenoate (ent-kaurenoic acid) to gibberellin 12 (GA12)." [EC:1.14.13.79, MetaCyc:RXN1F-160]
GO:0051779
name: gibberellin 12-aldehyde oxidase activity
def: "Catalysis of the reaction: gibberellin 12-aldehyde + NADPH + H+ + O2 = gibberellin 12 + NADP+ + H2O. This is the third of three successive reactions resulting in the oxidation of ent-kaurenoate (ent-kaurenoic acid) to gibberellin 12 (GA12)." [EC:1.14.13.79, MetaCyc:RXN1F-161]
GO:0051786
name: all-trans-retinol 13,14-reductase activity
def: "Catalysis of the reaction: all-trans-13,14-dihydroretinol + A = all-trans-retinol + AH(2). Note that this reaction has only been observed to occur in the opposite direction." [EC:1.3.99.23, RHEA:19196]
GO:0051787
name: misfolded protein binding
def: "Interacting selectively and non-covalently with a misfolded protein." [GOC:ai]
GO:0051800
name: phosphatidylinositol-3,4-bisphosphate 3-phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol-3,4-bisphosphate + H2O = phosphatidylinositol-4-phosphate + phosphate." [GOC:bf, PMID:9811831]
GO:0051861
name: glycolipid binding
def: "Interacting selectively and non-covalently with a glycolipid, any compound containing one or more monosaccharide residues bound by a glycosidic linkage to a hydrophobic group such as an acylglycerol, a sphingoid, a ceramide (N-acylsphingoid) or a prenyl phosphate." [http://www.chem.qmul.ac.uk/iupac/misc/glylp.html#2.1]
GO:0051864
name: histone demethylase activity (H3-K36 specific)
def: "Catalysis of the reaction: histone H3 N6-methyl-L-lysine (position 36) + alpha-ketoglutarate + O2 = succinate + CO2 + formaldehyde + lysine. This reaction is the removal of a methyl group from lysine at position 36 of the histone H3 protein." [PMID:16362057]
GO:0051870
name: methotrexate binding
def: "Interacting selectively and non-covalently with methotrexate, an antineoplastic antimetabolite with immunosuppressant properties. It is an inhibitor of tetrahydrofolate reductase and prevents the formation of tetrahydrofolate, necessary for synthesis of thymidylate, an essential component of DNA." [GOC:nln]
GO:0051871
name: dihydrofolic acid binding
def: "Interacting selectively and non-covalently with dihydrofolic acid, a folic acid in which the bicyclic pteridine structure is in the dihydro, partially reduced form; they are intermediates in folate metabolism and are reduced to their tetrahydro, reduced forms." [ISBN:0721662544]
GO:0051879
name: Hsp90 protein binding
def: "Interacting selectively and non-covalently with Hsp90 proteins, any of a group of heat shock proteins around 90kDa in size." [GOC:ai]
GO:0051880
name: G-quadruplex DNA binding
def: "Interacting selectively and non-covalently with G-quadruplex DNA structures, in which groups of four guanines adopt a flat, cyclic Hoogsteen hydrogen-bonding arrangement known as a guanine tetrad. The stacking of guanine tetrads results in G-quadruplex DNA structures. G-quadruplex DNA can form under physiological conditions from some G-rich sequences, such as those found in telomeres, immunoglobulin switch regions, gene promoters, fragile X repeats, and the dimerization domain in the human immunodeficiency virus (HIV) genome." [PMID:16142245, PMID:9512530]
GO:0051903
name: S-(hydroxymethyl)glutathione dehydrogenase activity
def: "Catalysis of the reaction: S-(hydroxymethyl)glutathione + NAD(P)+ = S-formylglutathione + NAD(P)H + H+." [EC:1.1.1.284]
GO:0051907
name: S-(hydroxymethyl)glutathione synthase activity
def: "Catalysis of the reaction: S-(hydroxymethyl)glutathione = formaldehyde + glutathione." [EC:4.4.1.22, RHEA:22491]
GO:0051908
name: double-stranded DNA 5'-3' exodeoxyribonuclease activity
def: "Catalysis of the sequential cleavage of mononucleotides from a free 5' terminus of a double-stranded DNA molecule." [GOC:ai]
GO:0051909
name: acetylenecarboxylate hydratase activity, producing 3-hydroxypropenoate
def: "Catalysis of the reaction: 3-hydroxypropenoate = propynoate + H2O." [MetaCyc:ACETYLENECARBOXYLATE-HYDRATASE-RXN]
GO:0051911
name: Methanosarcina-phenazine hydrogenase activity
def: "Catalysis of the reaction: H2 + 2-(2,3-dihydropentaprenyloxy)phenazine = 2-dihydropentaprenyloxyphenazine." [EC:1.12.98.3]
GO:0051912
name: CoB--CoM heterodisulfide reductase activity
def: "Catalysis of the reaction: coenzyme B + coenzyme M + methanophenazine = N-{7-[(2-sulfoethyl)dithio]heptanoyl}-3-O-phospho-L-threonine + dihydromethanophenazine." [EC:1.8.98.1]
GO:0051916
name: granulocyte colony-stimulating factor binding
def: "Interacting selectively and non-covalently with granulocyte colony-stimulating factor, G-CSF." [GOC:ai]
GO:0051920
name: peroxiredoxin activity
def: "Catalysis of the reaction: 2 R'-SH + ROOH = R'-S-S-R' + H2O + ROH." [EC:1.11.1.15]
GO:0051921
name: adenosylcobyric acid synthase (glutamine-hydrolyzing) activity
def: "Catalysis of the reaction: 4 L-glutamine + adenosylcob(III)yrinate a,c-diamide + 4 ATP + 4 H(2)O = 4 L-glutamate + adenosylcobyrate + 4 ADP + 8 H(+) + 4 phosphate." [EC:6.3.5.10, RHEA:23259]
GO:0051922
name: cholesterol sulfotransferase activity
def: "Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + cholesterol = adenosine 3',5'-bisphosphate + cholesterol sulfate." [PMID:12730293]
GO:0051959
name: dynein light intermediate chain binding
def: "Interacting selectively and non-covalently with a light intermediate chain of the dynein complex." [GOC:bf]
GO:0051978
name: lysophospholipid transporter activity
def: "Enables the directed movement of lysophospholipids into, out of or within a cell, or between cells. A lysophospholipid is a phospholipid that lacks one of its fatty acyl chains; it is an intermediate formed during digestion of dietary and biliary phospholipids." [GOC:ai]
GO:0051980
name: iron-nicotianamine transmembrane transporter activity
def: "Catalysis of the transfer of the iron chelate iron-nicotianamine (Fe-NA) from one side of a membrane to the other." [GOC:ai]
GO:0051981
name: copper chelate transmembrane transporter activity
def: "Catalysis of the transfer of a copper chelate from one side of a membrane to the other. A copper chelate is a heterocyclic compound having a metal ion attached by coordinate bonds to at least two nonmetal ions." [http://www.onelook.com]
GO:0051982
name: copper-nicotianamine transmembrane transporter activity
def: "Catalysis of the transfer of the copper chelate copper-nicotianamine (Cu-NA) from one side of a membrane to the other." [GOC:ai]
GO:0051989
name: coproporphyrinogen dehydrogenase activity
def: "Catalysis of the reaction: coproporphyrinogen III + 2 S-adenosyl-L-methionine = protoporphyrinogen IX + 2 CO2 + 2 L-methionine + 2 5'-deoxyadenosine." [EC:1.3.99.22]
GO:0051990
name: (R)-2-hydroxyglutarate dehydrogenase activity
def: "Catalysis of the reaction: (R)-2-hydroxyglutarate + acceptor = 2-oxoglutarate + reduced acceptor." [EC:1.1.99.2, MetaCyc:2-HYDROXYGLUTARATE-DEHYDROGENASE-RXN]
GO:0051991
name: UDP-N-acetyl-D-glucosamine:N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine-diphosphoundecaprenol 4-beta-N-acetylglucosaminlytransferase activity
def: "Catalysis of the reaction: N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine-diphosphoundecaprenol + UDP-N-acetyl-D-glucosamine = N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine-diphosphoundecaprenyl-N-acetylglucosamine + UDP." [MetaCyc:NACGLCTRANS-RXN]
GO:0051992
name: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine:undecaprenyl-phosphate transferase activity
def: "Catalysis of the reaction: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine + di-trans,poly-cis-undecaprenyl phosphate = UMP + N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimelyl-D-alanyl-D-alanine-diphosphoundecaprenol." [EC:2.7.8.13, MetaCyc:PHOSNACMURPENTATRANS-RXN]
GO:0051993
name: abscisic acid glucose ester beta-glucosidase activity
def: "Catalysis of the reaction: abscisic acid glucose ester + H2O = abscisic acid + beta-D-glucose." [PMID:16990135]
GO:0051994
name: P-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the phosphorus atom of an acceptor molecule." [GOC:ai]
GO:0051995
name: Se-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to the selenium atom of an acceptor molecule." [GOC:ai]
GO:0051996
name: squalene synthase activity
def: "Catalysis of the reaction: presqualene diphosphate + NADPH = squalene + NADP+ + diphosphate." [EC:2.5.1.21]
GO:0051997
name: 2-oxo-4-hydroxy-4-carboxy-5-ureidoimidazoline decarboxylase activity
def: "Catalysis of the reaction: 5-hydroxy-2-oxo-4-ureido-2,5-dihydro-1H imidazole-5-carboxylate + H+ = S-allantoin + CO2." [MetaCyc:RXN-6201]
GO:0051998
name: protein carboxyl O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group to a carboxyl group on a protein." [GOC:ai]
GO:0052381
name: tRNA dimethylallyltransferase activity
def: "Catalysis of the reaction: dimethylallyl diphosphate + tRNA = diphosphate + tRNA containing 6-dimethylallyladenosine." [EC:2.5.1.75]
GO:0052577
name: germacrene-D synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (-)-germacrene D + diphosphate." [EC:4.2.3.22]
GO:0052578
name: alpha-farnesene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (E,E)-alpha-farnesene + diphosphate." [MetaCyc:RXN-8574]
GO:0052579
name: (+)-pulegone reductase, (+)-isomenthone as substrate, activity
def: "Catalysis of the reaction: (+)-isomenthone + NADP+ = (+)-pulegone + NADPH + H+." [EC:1.3.1.81, MetaCyc:RXN-5164]
GO:0052580
name: (+)-pulegone reductase, (-)-menthone as substrate, activity
def: "Catalysis of the reaction: (-)-menthone + NADP+ = (+)-pulegone + NADPH + H+." [EC:1.3.1.81, MetaCyc:RXN-5163]
GO:0052581
name: (-)-isopiperitenone reductase activity
def: "Catalysis of the reaction: (6R)-isoperitenone + H(+) + NADPH = (2R,5R)-isopulegone + NADP(+)." [EC:1.3.1.82, RHEA:25651]
GO:0052582
name: (+)-menthofuran synthase activity
def: "Catalysis of the reaction: (R)-pulegone + H(+) + NADPH + O(2) = (R)-menthofuran + 2 H(2)O + NADP(+)." [EC:1.14.13.104, RHEA:25661]
GO:0052583
name: oxidoreductase activity, acting on halogen in donors
def: "Catalysis of an oxidation-reduction in which a halogen in the donor substance acts as a hydrogen or electron donor and reduces a hydrogen or electron acceptor." [GOC:mah]
GO:0052584
name: oxidoreductase activity, acting on halogen in donors, with NAD or NADP as acceptor
def: "Catalysis of an oxidation-reduction in which a halogen in the donor substance acts as a hydrogen or electron donor and reduces NAD or NADP." [GOC:mah]
GO:0052585
name: oxidoreductase activity, acting on the CH-NH2 group of donors, with a quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and reduces a quinone or similar compound." [GOC:ai]
GO:0052586
name: oxidoreductase activity, acting on other nitrogenous compounds as donors, with a quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a nitrogenous group, excluding NH and NH2 groups, acts as a hydrogen or electron donor and reduces a quinone or similar compound." [GOC:jl]
GO:0052587
name: diacetyl reductase ((R)-acetoin forming) activity
def: "Catalysis of the reaction: (R)-acetoin + NAD(+) = diacetyl + H(+) + NADH." [EC:1.1.1.303, RHEA:22903]
GO:0052588
name: diacetyl reductase ((S)-acetoin forming) activity
def: "Catalysis of the reaction: (S)-acetoin + NAD(+) = diacetyl + H(+) + NADH." [EC:1.1.1.304, RHEA:27289]
GO:0052589
name: malate dehydrogenase (menaquinone) activity
def: "Catalysis of the reaction: (S)-malate + a menaquinone = oxaloacetate + a menaquinol." [MetaCyc:RXNI-3]
GO:0052590
name: sn-glycerol-3-phosphate:ubiquinone oxidoreductase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + a ubiquinone = glycerone phosphate + a ubiquinol." [MetaCyc:RXN0-5258]
GO:0052591
name: sn-glycerol-3-phosphate:ubiquinone-8 oxidoreductase activity
def: "Catalysis of the reaction: sn-glycerol 3-phosphate + ubiquinone-8 = glycerone phosphate + ubiquinol-8." [MetaCyc:GLYC3PDEHYDROG-RXN]
GO:0052592
name: oxidoreductase activity, acting on CH or CH2 groups, with an iron-sulfur protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH2 group acts as a hydrogen or electron donor and reduces an iron-sulfur protein." [GOC:ai]
GO:0052593
name: tryptamine:oxygen oxidoreductase (deaminating) activity
def: "Catalysis of the reaction: tryptamine + H2O + O2 = NH3 + indole acetaldehyde + hydrogen peroxide + H+." [MetaCyc:RXN-1401]
GO:0052594
name: aminoacetone:oxygen oxidoreductase(deaminating) activity
def: "Catalysis of the reaction: aminoacetone + H2O + O2 = methylglyoxal + NH3 + hydrogen peroxide + H+." [MetaCyc:AMACETOXID-RXN]
GO:0052595
name: aliphatic-amine oxidase activity
def: "Catalysis of the reaction: an aliphatic amine + H2O + O2 = an aldehyde + NH3 + hydrogen peroxide + H+." [MetaCyc:AMINEOXID-RXN]
GO:0052596
name: phenethylamine:oxygen oxidoreductase (deaminating) activity
def: "Catalysis of the reaction: phenylethylamine + O2 + H2O = phenylacetaldehyde + NH3 + hydrogen peroxide + H+." [MetaCyc:AMINEPHEN-RXN]
GO:0052597
name: diamine oxidase activity
def: "Catalysis of the reaction: a diamine + H2O + O2 = a monoamine + NH3 + hydrogen peroxide." [MetaCyc:RXN-9599]
GO:0052598
name: histamine oxidase activity
def: "Catalysis of the reaction: histamine + H2O + O2 = imidazole-4-acetaldehyde + NH3 + hydrogen peroxide + H+." [MetaCyc:RXN-9600]
GO:0052599
name: methylputrescine oxidase activity
def: "Catalysis of the reaction: N-methylputrescine + H2O + O2 = N-methylaminobutanal + NH3 + hydrogen peroxide + H+." [MetaCyc:RXN-8244]
GO:0052600
name: propane-1,3-diamine oxidase activity
def: "Catalysis of the reaction: propane-1,3-diamine + H2O + O2 = 3-aminopropanal + NH3 + hydrogen peroxide + H+." [MetaCyc:RXN-6381]
GO:0052601
name: (S)-limonene 1,2-monooxygenase activity
def: "Catalysis of the reaction: (4S)-limonene + NAD(P)H + H+ + O2 = NAD(P)+ + H2O + (4S)-limonene-1,2-epoxide." [MetaCyc:RXN-9409]
GO:0052602
name: 4-chloronitrobenzene nitroreductase activity
def: "Catalysis of the reaction: 4-chloronitrobenzene + NADPH + H+ = 1-chloro-4-nitrosobenzene + NADP+ + H2O." [MetaCyc:RXN-8833]
GO:0052603
name: 1-chloro-4-nitrosobenzene nitroreductase activity
def: "Catalysis of the reaction: 1-chloro-4-nitrosobenzene + NADPH + H+ = 1-chloro-4-hydroxylaminobenzene + NADP+ + H2O." [MetaCyc:RXN-8834]
GO:0052604
name: delta-tocopherol cyclase activity
def: "Catalysis of the reaction: 2-methyl-6-phytyl-1,4-benzoquinone = delta-tocopherol." [MetaCyc:RXN-2561, PMID:12213958]
GO:0052605
name: gamma-tocopherol cyclase activity
def: "Catalysis of the reaction: 2,3-dimethyl-6-phytyl-1,4-benzoquinone = gamma-tocopherol." [MetaCyc:RXN-2543, PMID:12213958]
GO:0052606
name: chlorophyllide a oxygenase activity
def: "Catalysis of the reaction: chlorophyllide a + NADPH + O2 + 2 H+ = 7-hydroxychlorophyllide a + NADP+ + H2O." [EC:1.13.12.14, MetaCyc:RXN-7676]
GO:0052607
name: 7-hydroxy-chlorophyllide a oxygenase activity
def: "Catalysis of the reaction: 7-hydroxychlorophyllide a + NADPH + O2 + H+ = chlorophyllide b + NADP+ + 2 H2O." [EC:1.13.12.14, MetaCyc:RXN-7677]
GO:0052608
name: echinenone 3-hydroxylase activity
def: "Catalysis of the reaction: echinenone + a reduced electron acceptor + oxygen = 3-hydroxyechinenone + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8214]
GO:0052609
name: 4-ketotorulene 3-hydroxylase activity
def: "Catalysis of the reaction: 4-ketotorulene + a reduced electron acceptor + oxygen = 3-hydroxy-4-ketotorulene + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8218]
GO:0052610
name: beta-cryptoxanthin hydroxylase activity
def: "Catalysis of the reaction: beta-cryptoxanthin + a reduced electron acceptor + oxygen = zeaxanthin + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8026]
GO:0052611
name: beta-carotene 3-hydroxylase activity
def: "Catalysis of the reaction: beta-carotene + a reduced electron acceptor + oxygen = beta-cryptoxanthin + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8025]
GO:0052612
name: adonirubin 3-hydroxylase activity
def: "Catalysis of the reaction: adonirubin + a reduced electron acceptor + oxygen = 3S,3'S-astaxanthin + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8187]
GO:0052613
name: canthaxanthin 3-hydroxylase activity
def: "Catalysis of the reaction: canthaxanthin + a reduced electron acceptor + oxygen = adonirubin + an oxidized electron acceptor + H2O." [MetaCyc:RXN-8186]
GO:0052614
name: uracil oxygenase activity
def: "Catalysis of the reaction: uracil + NADH + O2 + H+ = ureidoacrylate peracid + NAD+. Ureidoacrylate peracid is spontaneously reduced by NADH to form ureidoacrylate." [MetaCyc:RXN0-6444, PMID:20369853, PMID:20400551]
GO:0052615
name: ent-kaurene oxidase activity
def: "Catalysis of the reaction: H+ + NADPH + ent-kaur-16-ene + oxygen = H2O + NADP+ + ent-kaur-16-en-19-ol." [MetaCyc:1.14.13.78-RXN]
GO:0052616
name: ent-kaur-16-en-19-ol oxidase activity
def: "Catalysis of the reaction: H+ + NADPH + ent-kaur-16-en-19-ol + oxygen = 2 H2O + NADP+ + ent-kaur-16-en-19-al." [MetaCyc:RXN-5242]
GO:0052617
name: ent-kaur-16-en-19-al oxidase activity
def: "Catalysis of the reaction: NADPH + ent-kaur-16-en-19-al + oxygen = H2O + NADP+ + ent-kaurenoate." [MetaCyc:RXN-7580]
GO:0052618
name: coenzyme F420-0:L-glutamate ligase activity
def: "Catalysis of the reaction: GTP + L-glutamate + factor F420-0 = GDP + H+ + factor gamma-F420-1 + phosphate." [MetaCyc:RXN-8080]
GO:0052619
name: coenzyme F420-1:gamma-L-glutamate ligase activity
def: "Catalysis of the reaction: GTP + L-glutamate + factor gamma-F420-1 = GDP + H+ + factor gamma-F420-2 + phosphate." [MetaCyc:RXN-8081]
GO:0052620
name: thymine dehydrogenase activity
def: "Catalysis of the reaction: H2O + thymine + acceptor = 5-methyl-barbiturate + donor-H2." [MetaCyc:RXN-8646]
GO:0052621
name: diguanylate cyclase activity
def: "Catalysis of the reaction: 2 GTP = cyclic di-3',5'-guanylate + 2 diphosphate + 2 H(+)." [EC:2.7.7.65, RHEA:24901]
GO:0052622
name: ATP dimethylallyltransferase activity
def: "Catalysis of the reaction: delta(2)-isopentenyl diphosphate + ATP = diphosphate + N6-(delta(2)-isopentenyl)adenosine 5'-triphosphate." [EC:2.5.1.27, MetaCyc:RXN-4303]
GO:0052623
name: ADP dimethylallyltransferase activity
def: "Catalysis of the reaction: delta2-isopentenyl diphosphate + ADP = diphosphate + N6-(delta(2)-isopentenyl)adenosine 5'-diphosphate." [EC:2.5.1.27, MetaCyc:RXN-4305]
GO:0052624
name: 2-phytyl-1,4-naphthoquinone methyltransferase activity
def: "Catalysis of the reaction: demethylphylloquinone + S-adenosyl-L-methionine = phylloquinone + S-adenosyl-L-homocysteine + H+." [MetaCyc:RXN-6723, MetaCyc:RXN-7569, PMID:14617060]
GO:0052625
name: 4-aminobenzoate amino acid synthetase activity
def: "Catalysis of the reaction: 4-aminobenzoate + ATP + amino acid = 4-aminobenzoyl amino acid conjugate + AMP + diphosphate." [MetaCyc:RXN-10884, PMID:19189963]
GO:0052626
name: benzoate amino acid synthetase activity
def: "Catalysis of the reaction: benzoate + ATP + amino acid = benzoyl amino acid conjugate + AMP + diphosphate." [MetaCyc:RXN-10886, PMID:19189963]
GO:0052627
name: vanillate amino acid synthetase activity
def: "Catalysis of the reaction: vanillate + ATP + amino acid = vanillate amino acid conjugate + AMP + diphosphate." [MetaCyc:RXN-10885, PMID:19189963]
GO:0052628
name: 4-hydroxybenzoate amino acid synthetase activity
def: "Catalysis of the reaction: 4-hydroxybenzoate + ATP + amino acid = 4-hydroxybenzoyl amino acid conjugate + AMP + diphosphate." [MetaCyc:RXN-10884, PMID:19189963]
GO:0052629
name: phosphatidylinositol-3,5-bisphosphate 3-phosphatase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate + H2O = a 1-phosphatidyl-1D-myo-inositol 5-phosphate + phosphate + 2 H+." [MetaCyc:RXN-10958, PMID:19901554]
GO:0052630
name: UDP-N-acetylgalactosamine diphosphorylase activity
def: "Catalysis of the reaction: UTP + N-acetyl-alpha-D-galactosamine 1-phosphate = diphosphate + UDP-N-acetyl-D-galactosamine." [EC:2.7.7.23]
GO:0052631
name: sphingolipid delta-8 desaturase activity
def: "Catalysis of the formation of a double bond between C8 and C9 of the long chain base of a sphingolipid. For example, sphinganine (d18:0) = 8-sphingenine (d18:1delta8); phytosphinganine (t18:0) = 8-phytosphingenine (t18:1delta8); and 4-sphingenine (18:1delta4) = 4,8-sphingadienine (d18:2delta4,8)." [PMID:17600137, PMID:9786850]
GO:0052633
name: isocitrate hydro-lyase (cis-aconitate-forming) activity
def: "Catalysis of the reaction: cis-aconitate + H2O = isocitrate." [EC:4.2.1.3, GOC:pde, GOC:vw, MetaCyc:ACONITATEHYDR-RXN]
GO:0052634
name: C-19 gibberellin 2-beta-dioxygenase activity
def: "Catalysis of the reaction: a C-19 gibberellin + 2-oxoglutarate + O2 = a C-19 2-beta-hydroxygibberellin + succinate + CO2. C-19 gibberellin refers to a gibberellin with nineteen carbons." [EC:1.14.11.13, GOC:kad]
GO:0052635
name: C-20 gibberellin 2-beta-dioxygenase activity
def: "Catalysis of the reaction: a C-20 gibberellin + 2-oxoglutarate + O2 = a C-20 2-beta-hydroxygibberellin + succinate + CO2. C-20 gibberellin refers to a gibberellin with twenty carbons." [EC:1.14.11.13, GOC:kad]
GO:0052636
name: arabinosyltransferase activity
def: "Catalysis of the transfer of a pentosyl group from one compound (donor) to another (acceptor)." [GOC:ai]
GO:0052637
name: delta 3-trans-hexadecenoic acid phosphatidylglycerol desaturase activity
def: "Catalysis of the reaction: 1-18:3-3-16:0-phosphatidylglycerol = 1-18:3-3-trans-16:1-phosphatidylglycerol + 2 H+. This reaction is the formation of a trans double bond between carbon 3 and carbon 4 (counting from the carboxyl end) of palmitic acid, which is specifically esterified to the sn-2 glyceryl carbon of phosphatidylglycerol." [GOC:ai, MetaCyc:RXN-8319, PMID:19682287]
GO:0052638
name: indole-3-butyrate beta-glucosyltransferase activity
def: "Catalysis of the reaction: indole-3-butyrate + UDP-D-glucose = indole-3-butyryl-beta-1-D-glucose + UDP." [MetaCyc:RXN-11655]
GO:0052639
name: salicylic acid glucosyltransferase (ester-forming) activity
def: "Catalysis of the reaction: salicylic acid + UDP-glucose = salicylic acid glucose ester + UDP." [MetaCyc:RXN-11659]
GO:0052640
name: salicylic acid glucosyltransferase (glucoside-forming) activity
def: "Catalysis of the reaction: salicylic acid + UDP-glucose = salicylic acid 2-O-glucoside + UDP." [MetaCyc:RXN-11658]
GO:0052641
name: benzoic acid glucosyltransferase activity
def: "Catalysis of the reaction: benzoic acid + UDP-glucose = benzoic acid glucose ester + UDP." [MetaCyc:RXN-11660]
GO:0052642
name: lysophosphatidic acid phosphatase activity
def: "Catalysis of the reaction: lysophosphatidic acid + H2O = phosphate + monoacylglycerol." [PMID:20045079, PMID:7966317]
GO:0052650
name: NADP-retinol dehydrogenase activity
def: "Catalysis of the reaction: all-trans-retinol + NADP+ = all-trans-retinal + NADPH + H+." [RHEA:25036]
GO:0052654
name: L-leucine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-leucine = 4-methyl-2-oxopentanoate + L-glutamatic acid." [EC:2.6.1.42, MetaCyc:BRANCHED-CHAINAMINOTRANSFERLEU-RXN]
GO:0052655
name: L-valine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-valine = 3-methyl-2-oxobutanoic acid + L-glutamatic acid." [EC:2.6.1.42, MetaCyc:BRANCHED-CHAINAMINOTRANSFERVAL-RXN]
GO:0052656
name: L-isoleucine transaminase activity
def: "Catalysis of the reaction: 2-oxoglutarate + L-isoleucine = (S)-3-methyl-2-oxopentanoic acid + L-glutamic acid." [EC:2.6.1.42, MetaCyc:BRANCHED-CHAINAMINOTRANSFERILEU-RXN]
GO:0052657
name: guanine phosphoribosyltransferase activity
def: "Catalysis of the reaction: GMP + diphosphate = guanine + 5-phospho-alpha-D-ribose 1-diphosphate." [EC:2.4.2.8, GOC:curators]
GO:0052658
name: inositol-1,4,5-trisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,4,5-trisphosphate + H2O = 1D-myo-inositol 1,4-bisphosphate + phosphate." [EC:3.1.3.56, RHEA:19800]
GO:0052659
name: inositol-1,3,4,5-tetrakisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4,5-tetrakisphosphate + H2O = 1D-myo-inositol 1,3,4-trisphosphate + phosphate." [EC:3.1.3.56, RHEA:11395]
GO:0052660
name: R-lactaldehyde reductase activity
def: "Catalysis of the reaction: (R)-propane-1,2-diol + NAD+ = (R)-lactaldehyde + NADH + H+." [RHEA:23875]
GO:0052661
name: S-lactaldehyde reductase activity
def: "Catalysis of the reaction: (S)-propane-1,2-diol + NAD+ = (S)-lactaldehyde + NADH + H+." [RHEA:15936]
GO:0052662
name: zeaxanthin epoxidase activity
def: "Catalysis of the reaction: zeaxanthin + NAD(P)H + H+ + O2 = antheraxanthin + NAD(P)+ + H2O." [EC:1.14.13.90]
GO:0052663
name: antheraxanthin epoxidase activity
def: "Catalysis of the reaction: antheraxanthin + NAD(P)H + H+ + O2 = all-trans-violaxanthin + NAD(P)+ + H2O." [MetaCyc:RXN-7979]
GO:0052664
name: nitroalkane oxidase activity
def: "Catalysis of the reaction: nitroalkane + H2O + O2 = an aldehyde or ketone + nitrite + H2O2." [EC:1.7.3.1]
GO:0052665
name: tRNA (uracil-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing 2'-O-methyluracil." [EC:2.1.1.34]
GO:0052666
name: tRNA (cytosine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing 2'-O-methylcytosine." [EC:2.1.1.34]
GO:0052667
name: phosphomethylethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: N-methylethanolamine phosphate + S-adenosyl-L-methionine = N,N-dimethylethanolamine phosphate + S-adenosyl-L-homocysteine + H(+)." [KEGG:R06868, MetaCyc:RXN-5642, RHEA:25322]
GO:0052668
name: farnesol kinase activity
def: "Catalysis of the reaction: farnesol + nucleoside triphosphate = farnesyl monophosphate + nucleoside diphosphate." [GOC:kd, MetaCyc:RXN-11625]
GO:0052669
name: CTP:2-trans,-6-trans-farnesol kinase activity
def: "Catalysis of the reaction: 2-trans,-6-trans-farnesol + CTP = 2-trans,-6-trans-farnesyl monophosphate + CDP + H+." [MetaCyc:RXN-11625]
GO:0052670
name: geraniol kinase activity
def: "Catalysis of the reaction: geraniol + nucleoside triphosphate = geranyl monophosphate + nucleoside diphosphate." [GOC:kd]
GO:0052671
name: geranylgeraniol kinase activity
def: "Catalysis of the reaction: geranylgeraniol + nucleoside triphosphate = all-trans-geranyl-geranyl monophosphate + nucleoside diphosphate." [GOC:kd, MetaCyc:RXN-11629]
GO:0052672
name: CTP:geranylgeraniol kinase activity
def: "Catalysis of the reaction: geranylgeraniol + CTP = all-trans-geranyl-geranyl monophosphate + CDP." [MetaCyc:RXN-11629]
GO:0052673
name: prenol kinase activity
def: "Catalysis of the reaction: prenol + nucleoside triphosphate = prenyl phosphate + nucleoside diphosphate activity." [GOC:ai, GOC:kd]
GO:0052674
name: ent-pimara-9(11),15-diene synthase activity
def: "Catalysis of the reaction: ent-copalyl diphosphate = ent-pimara-9(11),15-diene + diphosphate." [RHEA:25547]
GO:0052675
name: 3-methylbutanol:NADP oxidoreductase activity
def: "Catalysis of the reaction: 3-methylbutanol + NADP+ = 3-methylbutanal + NADPH + H+. 3-methylbutanal is also known as isovaleraldehyde." [EC:1.1.1.265, KEGG:R05686]
GO:0052676
name: 3-methylbutanol:NAD oxidoreductase activity
def: "Catalysis of the reaction: 3-methylbutanol + NAD+ = 3-methylbutanal + NADH + H+. 3-methylbutanal is also known as isovaleraldehyde." [EC:1.1.1.265, KEGG:R05685]
GO:0052677
name: D-arabinitol dehydrogenase, D-xylulose forming (NADP+) activity
def: "Catalysis of the reaction: D-arabinitol + NADP+ = D-xylulose + NADPH + H+." [EC:1.1.1.287]
GO:0052678
name: levopimaradiene synthase activity
def: "Catalysis of the reaction: (+)-copalyl diphosphate = abieta-8(14),12-diene + diphosphate." [RHEA:25551]
GO:0052679
name: terpentetriene synthase activity
def: "Catalysis of the reaction: terpentedienyl diphosphate = diphosphate + terpentetriene." [RHEA:25620]
GO:0052680
name: epi-isozizaene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (+)-epi-isozizaene + diphosphate." [RHEA:25995]
GO:0052681
name: alpha-bisabolene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (E,R)-alpha-bisabolene + diphosphate." [RHEA:25439]
GO:0052682
name: epi-cedrol synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate + H2O = epi-cedrol + diphosphate." [RHEA:26118]
GO:0052683
name: (Z)-gamma-bisabolene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (Z)-gamma-bisabolene + diphosphate." [RHEA:26084]
GO:0052684
name: L-serine hydro-lyase (adding indole, L-tryptophan-forming) activity
def: "Catalysis of the reaction: indole + L-serine = L-tryptophan + H2O." [MetaCyc:RXN0-2382]
GO:0052685
name: perillic acid:CoA ligase (ADP-forming) activity
def: "Catalysis of the reaction: perillic acid + CoA-SH + ATP = H2O + ADP + phosphate + perillyl-CoA." [KEGG:R06368]
GO:0052686
name: perillic acid:CoA ligase (AMP-forming) activity
def: "Catalysis of the reaction: perillic acid + CoA-SH + ATP = H2O + AMP + diphosphate + perillyl-CoA." [KEGG:R06368]
GO:0052687
name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (ADP-forming) activity
def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + ADP + phosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG:R06396]
GO:0052688
name: (3R)-3-isopropenyl-6-oxoheptanoate:CoA ligase (AMP-forming) activity
def: "Catalysis of the reaction: (3R)-3-isopropenyl-6-oxoheptanoate + CoA-SH + ATP = H2O + AMP + diphosphate + (3R)-3-isopropenyl-6-oxoheptanoyl-CoA." [KEGG:R06515]
GO:0052689
name: carboxylic ester hydrolase activity
def: "Catalysis of the hydrolysis of a carboxylic ester bond." [EC:3.1.1, EC:3.1.1.1, GOC:curators]
GO:0052690
name: trichloro-p-hydroquinone reductive dehalogenase activity
def: "Catalysis of the reaction: 2,3,6-trichlorohydroquinone + 2 glutathione = 2,6-dichlorohydroquinone + glutathione disulfide + HCl." [UM-BBD_reactionID:r0031, UM-BBD_reactionID:r0315]
GO:0052691
name: UDP-arabinopyranose mutase activity
def: "Catalysis of the reaction: UDP-beta-L-arabinofuranose = UDP-beta-L-arabinopyranose." [EC:5.4.99.30, RHEA:28353]
GO:0052692
name: raffinose alpha-galactosidase activity
def: "Catalysis of the reaction: raffinose + H2O = alpha-D-galactose + sucrose." [MetaCyc:RXN-11502]
GO:0052693
name: epoxyqueuosine reductase activity
def: "Catalysis of the reaction: epoxyqueuosine in tRNA + reductant = queuosine in tRNA + oxidised reductant." [PMID:21502530]
GO:0052694
name: jasmonoyl-isoleucine-12-hydroxylase activity
def: "Catalysis of the reaction: jasmonoyl-isoleucine + NADPH + H+ + O2 = 12-hydroxy-jasmonoyl-isoleucine + NADP+ + H2O." [MetaCyc:RXN-12421, PMID:21576464]
GO:0052705
name: methylhistidine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + N-alpha-methyl-L-histidine = S-adenosyl-L-homocysteine + N-alpha,N-alpha-dimethyl-L-histidine." [EC:2.1.1.44]
GO:0052706
name: histidine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + L-histidine = S-adenosyl-L-homocysteine + N-alpha-trimethyl-L-histidine." [EC:2.1.1.44]
GO:0052712
name: inositol phosphosphingolipid phospholipase activity
def: "Catalysis of the reaction: inositol phosphosphingolipid + H2O = sphingolipid + phosphorylinositol." [GOC:ai]
GO:0052713
name: inositol phosphorylceramide phospholipase activity
def: "Catalysis of the reaction: inositol phosphorylceramide + H2O = C26-phytoceramide + phosphorylinositol." [GOC:ai]
GO:0052714
name: mannosyl-inositol phosphorylceramide phospholipase activity
def: "Catalysis of the reaction: mannosyl-inositol phosphorylceramide + H2O = C26-phytoceramide + mannosylphosphorylinositol." [GOC:ai]
GO:0052715
name: mannosyl-diinositol phosphorylceramide phospholipase activity
def: "Catalysis of the reaction: mannosyl-diinositol phosphorylceramide + H2O = C26-phytoceramide + mannosyldiphosphorylinositol." [GOC:ai]
GO:0052716
name: hydroquinone:oxygen oxidoreductase activity
def: "Catalysis of the reaction: 4 hydroquinone + O2 = 4 benzosemiquinone + 4 H2O." [EC:1.10.3.2]
GO:0052717
name: tRNA-specific adenosine-34 deaminase activity
def: "Catalysis of the reaction: adenosine-34 + H2O = inosine-34 + NH3, in a tRNA-Ala molecule." [PMID:17875641]
GO:0052719
name: apurinic/apyrimidinic endoribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages immediately 5' to an apurinic/apyrimidinic (AP; also called abasic) site within a ribonucleic acid molecule by creating internal breaks, generating a single-strand break with 5'-ribose phosphate and 3'-hydroxyl ends." [PMID:19401441]
GO:0052720
name: apurinic/apyrimidinic endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages immediately 5' to an apurinic/apyrimidinic (AP; also called abasic) site within a deoxyribonucleic acid molecule by creating internal breaks, generating a single-strand break with 5'-deoxyribose phosphate and 3'-hydroxyl ends." [PMID:19401441]
GO:0052722
name: fatty acid in-chain hydroxylase activity
def: "Catalysis of the reaction: fatty acid + O2 + 2 NADPH + H+ = fatty acid with in-chain hydroxy group + 2 NADP+ + H2O." [MetaCyc:RXN-12186]
GO:0052723
name: inositol hexakisphosphate 1-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 1-diphospho-1D-myo-inositol (2,3,4,5,6)pentakisphosphate." [GOC:jp, PMID:18981179]
GO:0052724
name: inositol hexakisphosphate 3-kinase activity
def: "Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 3-diphospho-1D-myo-inositol (1,2,4,5,6)pentakisphosphate." [GOC:jp, PMID:18981179]
GO:0052725
name: inositol-1,3,4-trisphosphate 6-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4-trisphosphate + ATP = 1D-myo-inositol 1,3,4,6-tetrakisphosphate + ADP + 2 H(+)." [EC:2.7.1.134]
GO:0052726
name: inositol-1,3,4-trisphosphate 5-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 1,3,4-trisphosphate + ATP = 1D-myo-inositol 1,3,4,5-tetrakisphosphate + ADP + 2 H(+)." [EC:2.7.1.134]
GO:0052727
name: capsanthin synthase activity
def: "Catalysis of the reaction: antheraxanthin = capsanthin." [EC:5.3.99.8]
GO:0052728
name: capsorubin synthase activity
def: "Catalysis of the reaction: violaxanthin = capsorubin." [EC:5.3.99.8]
GO:0052729
name: dimethylglycine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + N,N-dimethylglycine = S-adenosyl-L-homocysteine + betaine." [EC:2.1.1.157]
GO:0052730
name: sarcosine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + sarcosine = S-adenosyl-L-homocysteine + N,N-dimethylglycine." [EC:2.1.1.157]
GO:0052731
name: phosphocholine phosphatase activity
def: "Catalysis of the reaction: choline phosphate + H2O = choline + phosphate." [EC:3.1.3.75]
GO:0052732
name: phosphoethanolamine phosphatase activity
def: "Catalysis of the reaction: O-phosphoethanolamine + H2O = ethanolamine + phosphate." [EC:3.1.3.75]
GO:0052733
name: quinate 3-dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: quinate + NADP+ = 3-dehydroquinate + NADPH + H+." [RHEA:18428]
GO:0052734
name: shikimate 3-dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: shikimate + NAD+ = 3-dehydroshikimate + NADH + H+." [RHEA:17744]
GO:0052735
name: tRNA (cytosine-3-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA = S-adenosyl-L-homocysteine + tRNA containing 3-methylcytosine." [PMID:21518804, PMID:21518805]
GO:0052736
name: beta-glucanase activity
def: "Catalysis of the hydrolysis of linkages in beta-D-glucans; beta-glucans are polysaccharides of D-glucose monomers linked by beta-glycosidic bonds." [Wikipedia:Beta-glucan]
GO:0052737
name: pyruvate dehydrogenase (quinone) activity
def: "Catalysis of the reaction: a ubiquinone + H2O + pyruvate = a ubiquinol + acetate + CO2." [EC:1.2.5.1]
GO:0052738
name: oxidoreductase activity, acting on the aldehyde or oxo group of donors, with a quinone or similar compound as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which an aldehyde or ketone (oxo) group acts as a hydrogen or electron donor and reduces a quinone or similar compound." [EC:1.2.5]
GO:0052739
name: phosphatidylserine 1-acylhydrolase activity
def: "Catalysis of the reaction: phosphatidylserine + H2O = 2-acyl-sn-glycero-3-phosphoserine + fatty acid." [KEGG:R04034]
GO:0052740
name: 1-acyl-2-lysophosphatidylserine acylhydrolase activity
def: "Catalysis of the reaction: 1-acyl-2-lysophosphatidylserine + H2O = sn-glycerol-phosphoserine + a carboxylate." [BRENDA:3.1.1.32]
GO:0052741
name: (R)-limonene 6-monooxygenase activity
def: "Catalysis of the reaction: (4R)-limonene + H+ + NADPH + O2 = (1R,5S)-carveol + H2O + NADP+." [EC:1.14.13.80, RHEA:18960]
GO:0052742
name: phosphatidylinositol kinase activity
def: "Catalysis of the reaction: ATP + a phosphatidylinositol = ADP + a phosphatidylinositol phosphate." [GOC:ai]
GO:0052743
name: inositol tetrakisphosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol tetrakisphosphate + H2O = myo-inositol trisphosphate + phosphate." [GOC:ai]
GO:0052744
name: phosphatidylinositol monophosphate phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol monophosphate + H2O = phosphatidylinositol + phosphate." [GOC:ai]
GO:0052745
name: inositol phosphate phosphatase activity
def: "Catalysis of the reaction: inositol phosphate(n) + H2O = inositol phosphate(n-1) + phosphate. This reaction is the removal of a phosphate group from an inositol phosphate." [GOC:ai]
GO:0052747
name: sinapyl alcohol dehydrogenase activity
def: "Catalysis of the reaction: sinapaldehyde + NADPH + H+ = sinapyl-alcohol + NADP+." [GOC:mengo_curators, MetaCyc:RXN-1125]
GO:0052748
name: baicalin beta-D-glucuronidase activity
def: "Catalysis of the reaction: baicalin + H2O = baicalein + D-glucuronate." [GOC:mengo_curators, RHEA:28133]
GO:0052749
name: glucose-6-phosphate dehydrogenase (coenzyme F420) activity
def: "Catalysis of the reaction: beta-D-glucose 6-phosphate + coenzyme F420 + H+ = 6-O-phosphono-D-glucono-1,5-lactone + reduced coenzyme F420." [EC:1.1.98.2, GOC:mengo_curators, RHEA:27297]
GO:0052750
name: reactive-black-5:hydrogen-peroxide oxidoreductase activity
def: "Catalysis of the reaction: reactive black 5 + hydrogen peroxide = oxidized reactive black 5 + 2 H2O." [KEGG:R07612, MetaCyc:RXN-8666]
GO:0052751
name: GDP-mannose hydrolase activity
def: "Catalysis of the reaction: GDP-mannose + H2O = GMP + mannose-1-phosphate." [PMID:16766526]
GO:0052752
name: reduced coenzyme F420:heterodisulfide oxidoreductase activity
def: "Catalysis of the reaction: reduced coenzyme F420 + CoB-S-S-CoM = coenzyme F420 + CoM-SH + CoB-SH." [GOC:mengo_curators, PMID:9914308]
GO:0052753
name: propan-2-ol:coenzyme F420 oxidoreductase activity
def: "Catalysis of the reaction: propan-2-ol + coenzyme F420 = acetone + reduced coenzyme F420." [GOC:mengo_curators, PMID:15016352, PMID:1879431, PMID:8706724]
GO:0052754
name: GTP:coenzyme F420 guanyltransferase activity
def: "Catalysis of the reaction: GTP + factor gamma-F420-2 + H+ = coenzyme F390-G + diphosphate." [GOC:mengo_curators, MetaCyc:MONOMER-13942, MetaCyc:RXN-9385]
GO:0052755
name: reduced coenzyme F420:quinone oxidoreductase activity
def: "Catalysis of the reaction: reduced coenzyme F420 + 2,3-dimethyl-1,4-naphthoquinone = coenzyme F420 + reduced 2,3-dimethyl-1,4-naphthoquinone. Reduced 2,3-dimethyl-1,4-naphthoquinone is also known as 2,3-dimethyl-1,4-hydronaphthoquinone." [GOC:mengo_curators, PMID:10971593, PMID:8055920]
GO:0052756
name: chitobiose phosphorylase activity
def: "Catalysis of the reaction: chitobiose + phosphate = N-acetyl-D-glucosamine + N-acetyl-alpha-D-glucosamine 1-phosphate. This reaction is the phosphorolysis of chitobiose, (GlcNAc)2, a dimer of beta-(1->4) linked glucosamine units." [GOC:mengo_curators, PMID:15274915, Wikipedia:chitobiose]
GO:0052757
name: chondroitin hydrolase activity
def: "Catalysis of the hydrolysis of hexosaminic linkages in chondroitin, a linear polymer structure composed of the repeating disaccharide unit [->4)-D-glucuronic acid-(1->3)-N-acetyl-D-galactosamine-(1-], also written as [->4GlcUA1->3GalNAc1-]." [GOC:mengo_curators, PMID:18390555]
GO:0052758
name: coenzyme F420-dependent 2,4,6-trinitrophenol reductase activity
def: "Catalysis of the reaction: 2,4,6-trinitrophenol + H- = 2,4,6-trinitrophenol hydride Meisenheimer complex. Coenzyme F420 supplies the hydride (H-) in the reaction." [GOC:mengo_curators, PMID:11995829, UM-BBD_reactionID:r1065]
GO:0052759
name: coenzyme F420-dependent 2,4,6-trinitrophenol hydride reductase activity
def: "Catalysis of the reaction: trinitrophenol hydride Meisenheimer complex + H- = trinitrophenol dihydride Meisenheimer complex (aci form). Coenzyme F420 supplies the hydride (H-) in the reaction." [GOC:mengo_curators, UM-BBD_reactionID:r1066]
GO:0052760
name: coenzyme F420-dependent 2,4-dinitrophenol reductase activity
def: "Catalysis of the reaction: 2,4-dinitrophenol + H- = 2,4-dinitrophenol hydride Meisenheimer complex. Coenzyme F420 supplies the hydride (H-) in the reaction." [GOC:mengo_curators, PMID:11995829, UM-BBD_reactionID:r1071]
GO:0052761
name: exo-1,4-beta-D-glucosaminidase activity
def: "Catalysis of the reaction: [beta-(1->4)-D-glucosamine]n-[N-acetyl-D-glucosamine]m = D-glucosamine + [beta-(1->4)-D-glucosamine](n-1)-[N-acetyl-D-glucosamine]m. This reaction is the hydrolysis of chitosan or chitosan oligosaccharides to remove a D-glucosamine residue from the non-reducing termini; chitosan is a linear polysaccharide composed of randomly distributed beta-(1->4)-linked D-glucosamine and N-acetyl-D-glucosamine units." [EC:3.2.1.165, GOC:mengo_curators, MetaCyc:3.2.1.165-RXN]
GO:0052762
name: gellan lyase activity
def: "Catalysis of the reaction: gellan = n beta-D-4-deoxy-delta4,5-GlcAp-(1->4)-beta-D-Glcp-(1->4)-alpha-L-Rhap-(1->3)-beta-D-Glcp. This reaction is the eliminative cleavage of beta-D-glucopyranosyl-(1->4)-beta-D-glucopyranosyluronate bonds of gellan backbone, releasing tetrasaccharides containing a 4-deoxy-4,5-unsaturated D-glucopyranosyluronic acid at the non-reducing end; in the product, the abbreviations are D-glucose (Glc), D-glucuronic acid (GlcA), and L-rhamnose (Rha)." [GOC:mengo_curators, MetaCyc:RXN-12269]
GO:0052763
name: ulvan lyase activity
def: "Catalysis of the cleavage of a carbon-oxygen bond in ulvan, a carbohydrate composed of a repeating structure of [->4)-beta-D-GlcA-(1,4)-alpha-L-Rha 3S-(1->4)-alpha-L-IdoA-(1->4)-alpha-L-Rha 3S-(1-]n. Continued digest of ulvan with an enzyme that can catalyze this reaction results in ulvanobiouronic acid A 3-sulfate [->4)-beta-D-GlcpA-(1->4)-alpha-L-Rhap 3-sulfate-(1-]n with 4-deoxy-L-threo-hex-4-enopyranosiduronic acid at the non-reducing end." [GOC:mengo_curators, PMID:9468631]
GO:0052764
name: exo-oligoalginate lyase activity
def: "Catalysis of the cleavage of glycosidic bonds through a beta-elimination reaction on alginate, a linear polysaccharide consisting of guluronate (G) and mannuronate (M) as the monomer constituents. An oligoalginate is a linear polymer of two, three or four units of (1->4)-alpha-L-guluronic acid and beta-D-mannuronic acid, releasing monosaccharides with 4-deoxy-alpha-L-erythro-hex-4-enopyranuronosyl groups at their ends." [GOC:mengo_curators, PMID:20925655]
GO:0052765
name: reduced coenzyme F420 oxidase activity
def: "Catalysis of the reaction: 2 reduced coenzyme F420 + O2 = 2 coenzyme F420 + 2 H2O." [GOC:mengo_curators, PMID:15340796]
GO:0052766
name: mannoside alpha-1,4-mannosidase activity
def: "Catalysis of the hydrolysis of the alpha-(1->4) linkage of the terminal, non-reducing alpha-D-mannose residues in alpha-D-mannosides." [GOC:mengo_curators, PMID:20081828]
GO:0052767
name: mannosyl-oligosaccharide 1,6-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of the alpha-(1->6) bonds of alpha-D-mannose residues in mannosyl-oligosaccharide." [GOC:mengo_curators, PMID:1849817, PMID:2338081]
GO:0052768
name: mannosyl-oligosaccharide 1,3-alpha-mannosidase activity
def: "Catalysis of the hydrolysis of the alpha-(1->3) bonds of alpha-D-mannose residues in mannosyl-oligosaccharide." [GOC:mengo_curators, PMID:1849817, PMID:2338081]
GO:0052769
name: beta-6-sulfate-N-acetylglucosaminidase activity
def: "Catalysis of the glycosidic cleavage of the terminal 2-acetamido-2-deoxy-beta-D-glucopyranoside 6-sulfate (6-SO3-GlcNAc) residue from sulfomucin, a sulfated mucin derivative." [GOC:mengo_curators, PMID:15716424]
GO:0052770
name: coenzyme F390-A hydrolase activity
def: "Catalysis of the reaction: coenzyme F390-A = AMP + coenzyme F420." [GOC:mengo_curators, PMID:8536708, PMID:9352911]
GO:0052771
name: coenzyme F390-G hydrolase activity
def: "Catalysis of the reaction: coenzyme F390-G = GMP + coenzyme F420." [GOC:mengo_curators, PMID:8536708, PMID:9352911]
GO:0052772
name: brefeldin A esterase activity
def: "Catalysis of the hydrolysis of brefeldin A to produce brefeldin A acid. Brefeldin A is also known as gamma,4-dihydroxy-2-(6-hydroxy-1-heptenyl)-4-cyclopentanecrotonic acid lambda-lactone." [GOC:mengo_curators, PMID:10201402, PMID:8106385]
GO:0052773
name: diacetylchitobiose deacetylase activity
def: "Catalysis of the reaction: N,N'-diacetylchitobiose (GlcNac2) + H2O = acetate + 2-acetamido-4-O-(2-amino-2-deoxy-beta-D-glucopyranosyl)-2-deoxy-D-glucose (GlcN-GlcNAc)." [GOC:bf, GOC:mengo_curators, MetaCyc:RXN-12543, PMID:15136574, PMID:16232910, PMID:16736587]
GO:0052774
name: glucosyl-N-acetylglucosamine glucosaminidase activity
def: "Catalysis of the reaction: glucosyl-N-acetylglucosamine + H2O = glucosamine + N-acetylglucosamine." [GOC:mengo_curators, PMID:15136574, PMID:16232910, PMID:16736587]
GO:0052775
name: endo-1,3-alpha-L-rhamnosidase activity
def: "Catalysis of the reaction: R1-L-rhamnose-(1->3)-alpha-L-rhamnose-R2 + H2O = R1-L-rhamnose + L-rhamnose-R2. This reaction is the hydrolysis of an alpha-(1->3) linkage between two rhamnose residues in a polysaccharide chain." [GOC:mengo_curators, PMID:10439404]
GO:0052788
name: d-4,5 unsaturated beta-glucuronyl hydrolase activity
def: "Catalysis of the hydrolysis of the glycosidic bond in an unsaturated saccharide between the unsaturated glucuronyl residue at the nonreducing terminus and the saccharide linked to the residue." [GOC:mengo_curators, PMID:12729728]
GO:0052789
name: mannan 1,3-beta-mannosidase activity
def: "Catalysis of the hydrolysis of (1->3)-beta-D-mannosidic linkages in mannans, releasing mannose." [EC:3.2.1.78, GOC:mengo_curators]
GO:0052790
name: chitooligosaccharide deacetylase activity
def: "Catalysis of the reaction: chitooligosaccharide with N-acetylglucosamine at nonreducing terminal + H2O = chitooligosaccharide with glucosamine at nonreducing terminal + acetate. This reaction is the deacetylation of a chitooligosaccharide at the nonreducing N-acetylglucosamine residue; chitooligosaccharide are composed of (1->4)-linked D-glucosamine (GlcN) and N-acetyl-D-glucosamine (GlcNAc) in varying proportions." [GOC:mengo_curators, PMID:8421697, PMID:8986807]
GO:0052791
name: 3-deoxy-D-glycero-D-galacto-2-nonulosonic acid hydrolase activity
def: "Catalysis of the reaction: (2-keto-3-deoxynononic acid)n + H2O = (2-keto-3-deoxynononic acid)n-1 + 2-keto-3-deoxynononic acid. This reaction is the hydrolysis of a 2-keto-3-deoxynononic acid residue from a poly-2-keto-3-deoxynononic acid chain." [GOC:mengo_curators, PMID:21247893]
GO:0052792
name: endo-xylogalacturonan hydrolase activity
def: "Catalysis of the endohydrolysis of xylogalacturonate by cleavage of the alpha-(1,4)-linkage. Xylogalacturonate (XGA) is composed of a chain of alpha-(1,4)-linked D-galacturonic acid residues with beta-D-xylose substituted at the O3 position." [GOC:mengo_curators, PMID:10618200]
GO:0052793
name: pectin acetylesterase activity
def: "Catalysis of the reaction: pectin + H2O = pectate + acetate. This reaction is the hydrolysis of acetyl esters of pectin, producing pectate, partially esterified pectin." [GOC:mengo_curators, PMID:9218776]
GO:0052794
name: exo-alpha-(2->3)-sialidase activity
def: "Catalysis of the hydrolysis of alpha-(2->3)-glycosidic linkages of terminal sialic residues in substrates." [EC:3.2.1.18, GOC:mengo_curators]
GO:0052795
name: exo-alpha-(2->6)-sialidase activity
def: "Catalysis of the hydrolysis of alpha-(2->6)-glycosidic linkages of terminal sialic residues in substrates." [EC:3.2.1.18, GOC:mengo_curators]
GO:0052796
name: exo-alpha-(2->8)-sialidase activity
def: "Catalysis of the hydrolysis of alpha-(2->8)-glycosidic linkages of terminal sialic residues in substrates." [EC:3.2.1.18, GOC:mengo_curators]
GO:0052797
name: 4-O-methyl-glucuronoyl methylesterase activity
def: "Catalysis of the reaction: [X]-4-O-methyl-D-glucuronic acid + H2O = [X]-OH + methyl-D-glucuronic acid. This reaction is the hydrolysis of the ester linkage between 4-O-methyl-D-glucuronic acid (MeGlcA) and an alcohol (-OH) group attached to a molecule, denoted here as [X]." [GOC:mengo_curators, PMID:16876163]
GO:0052798
name: beta-galactoside alpha-2,3-sialyltransferase activity
def: "Catalysis of the transfer of sialyl residues alpha-2,3-linked to a beta galactosyl residue on the donor to form an alpha-2,3 linkage to a terminal beta galactosyl residue on the acceptor." [GOC:mengo_curators, PMID:7826016, PMID:8405811]
GO:0052806
name: sulfite reductase (coenzyme F420) activity
def: "Catalysis of the reaction: sulfite + 3 1,5-dihydrocoenzyme F420 = hydrogen sulfide + 3 H2O + 3 coenzyme F420. 1,5-dihydrocoenzyme F420 is also known as reduced coenzyme F420." [GOC:mengo_curators, PMID:16048999]
GO:0052807
name: aflatoxin reductase (coenzyme F420) activity
def: "Catalysis of the reaction: aflatoxin + 1,5-dihydrocoenzyme F420 = aflatoxin with reduced furanocoumarin moiety + coenzyme F420. 1,5-dihydrocoenzyme F420 is also known as reduced coenzyme F420." [GOC:mengo_curators, PMID:20807200]
GO:0052808
name: reduced coenzyme F420:NADP+ oxidoreductase activity
def: "Catalysis of the reaction: NADP+ + 1,5-dihydrocoenzyme F420 = NADPH + H+ + coenzyme F420. 1,5-dihydrocoenzyme F420 is also known as reduced coenzyme F420." [GOC:mengo_curators, PMID:11726492]
GO:0052809
name: acharan sulfate lyase activity
def: "Catalysis of the cleavage of a carbon-oxygen bond in acharan sulfate, a glycosaminoglycan with a uniformly repeating disaccharide structure of alpha-D-N-acetylglucosaminyl-2-O-sulfo-alpha-L-iduronic acid." [GOC:mengo_curators, PMID:19566715]
GO:0052810
name: 1-phosphatidylinositol-5-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol + ATP = a 1-phosphatidyl-1D-myo-inositol 5-phosphate + ADP + 2 H(+)." [EC:2.7.1.137]
GO:0052811
name: 1-phosphatidylinositol-3-phosphate 4-kinase activity
def: "Catalysis of the reaction: a 1-phosphatidyl-1D-myo-inositol 4-phosphate + ATP = a 1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate + ADP + 2 H(+)." [EC:2.7.1.150]
GO:0052812
name: phosphatidylinositol-3,4-bisphosphate 5-kinase activity
def: "Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate + ATP = a 1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate + ADP + 2 H(+)." [EC:2.7.1.153]
GO:0052813
name: phosphatidylinositol bisphosphate kinase activity
def: "Catalysis of the reaction: ATP + a phosphatidylinositol bisphosphate = ADP + a phosphatidylinositol trisphosphate." [GOC:ai]
GO:0052814
name: medium-chain-aldehyde dehydrogenase activity
def: "Catalysis of the reaction: a medium-chain aldehyde + NAD+ = a medium-chain carboxylate + NADH + H+. Medium-chain aldehydes have a chain length of between 8 and 12 carbons." [EC:1.2.1.48]
GO:0052815
name: medium-chain acyl-CoA hydrolase activity
def: "Catalysis of the reaction: H2O + a medium-chain acyl-CoA = a medium-chain carboxylate + CoA. A medium chain is a chain of between eight and twelve carbons in length." [EC:3.1.2.19]
GO:0052816
name: long-chain acyl-CoA hydrolase activity
def: "Catalysis of the reaction: H2O + a long-chain acyl-CoA = a long-chain carboxylate + CoA. A long chain is a chain of greater than 12 carbons in length." [EC:3.1.2.19]
GO:0052817
name: very long chain acyl-CoA hydrolase activity
def: "Catalysis of the reaction: H2O + a very long chain acyl-CoA = a very long chain carboxylate + CoA. A very long chain has chain length greater than C18." [EC:3.1.2.19]
GO:0052818
name: heteroglycan 3-alpha-mannosyltransferase activity
def: "Catalysis of the reaction: heteroglycan + GDP-mannose = (1->3)-alpha-D-mannosylheteroglycan + GDP." [EC:2.4.1.48, MetaCyc:RXN-7992]
GO:0052819
name: heteroglycan 2-alpha-mannosyltransferase activity
def: "Catalysis of the reaction: heteroglycan + GDP-mannose = (1->2)-alpha-D-mannosylheteroglycan + GDP." [EC:2.4.1.48, MetaCyc:2.4.1.48-RXN]
GO:0052820
name: DNA-1,N6-ethenoadenine N-glycosylase activity
def: "Catalysis of the reaction: DNA with 1-N6-ethenoadenine + H2O = DNA with abasic site + 1-N6-ethenoadenine. This reaction is the removal of 1,N6-ethenoadenine by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar." [PMID:21960007]
GO:0052821
name: DNA-7-methyladenine glycosylase activity
def: "Catalysis of the reaction: DNA containing 7-methyladenine + H2O = DNA with abasic site + 7-methyladenine. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the damaged DNA 7-methyladenine and the deoxyribose sugar to remove the 7-methyladenine, leaving an abasic site." [GOC:jl, PMID:16468998]
GO:0052822
name: DNA-3-methylguanine glycosylase activity
def: "Catalysis of the reaction: DNA containing 3-methylguanine + H2O = DNA with abasic site + 3-methylguanine. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the damaged DNA 3-methylguanine and the deoxyribose sugar to remove the 3-methylguanine, leaving an abasic site." [EC:3.2.2.21, GOC:elh, PMID:10872450, PMID:9224623]
GO:0052823
name: 2-hydroxy-6-oxonona-2,4,7-trienedioate hydrolase activity
def: "Catalysis of the reaction: (2E,4Z,7E)-2-hydroxy-6-oxonona-2,4,7-trienedioate + H2O = (2E)-2-hydroxypenta-2,4-dienoate + fumarate + H+." [RHEA:25049]
GO:0052824
name: dolichyl-pyrophosphate Man7GlcNAc2 alpha-1,6-mannosyltransferase activity
def: "Catalysis of the addition of mannose to dolichyl-pyrophosphate Man7GlcNAc2 (also written as Man7GlcNAc2-PP-Dol) in alpha-(1->6) linkage, producing Man8GlcNAc2-PP-Dol." [PMID:10336995]
GO:0052825
name: inositol-1,3,4,5,6-pentakisphosphate 1-phosphatase activity
def: "Catalysis of the reaction: inositol-1,3,4,5,6-pentakisphosphate + H2O = inositol-3,4,5,6-tetrakisphosphate + phosphate." [GOC:ai]
GO:0052826
name: inositol hexakisphosphate 2-phosphatase activity
def: "Catalysis of the reaction: myo-inositol hexakisphosphate + H2O = myo-inositol 1,3,4,5,6-pentakisphosphate + phosphate." [EC:3.1.3.62]
GO:0052827
name: inositol pentakisphosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol pentakisphosphate + H2O = myo-inositol tetrakisphosphate + phosphate." [GOC:bf]
GO:0052828
name: inositol-3,4-bisphosphate 4-phosphatase activity
def: "Catalysis of the reaction: 1D-myo-inositol 3,4-bisphosphate + H2O = 1D-myo-inositol 3-phosphate + phosphate." [GOC:mah]
GO:0052829
name: inositol-1,3,4-trisphosphate 1-phosphatase activity
def: "Catalysis of the reaction: D-myo-inositol 1,3,4-trisphosphate + H2O = myo-inositol 3,4-bisphosphate + phosphate." [GOC:ai]
GO:0052830
name: inositol-1,3,4,6-tetrakisphosphate 6-phosphatase activity
def: "Catalysis of the reaction: inositol-1,3,4,6-tetrakisphosphate + H2O = inositol-1,3,4-trisphosphate + phosphate." [GOC:ai]
GO:0052831
name: inositol-1,3,4,6-tetrakisphosphate 1-phosphatase activity
def: "Catalysis of the reaction: inositol-1,3,4,6-tetrakisphosphate + H2O = inositol-3,4,6-trisphosphate + phosphate." [GOC:ai]
GO:0052832
name: inositol monophosphate 3-phosphatase activity
def: "Catalysis of the reaction: myo-inositol 3-phosphate + H2O = myo-inositol + phosphate." [EC:3.1.3.25]
GO:0052833
name: inositol monophosphate 4-phosphatase activity
def: "Catalysis of the reaction: myo-inositol 4-phosphate + H2O = myo-inositol + phosphate." [EC:3.1.3.25]
GO:0052834
name: inositol monophosphate phosphatase activity
def: "Catalysis of the reaction: myo-inositol phosphate + H2O = myo-inositol + phosphate." [EC:3.1.3.25]
GO:0052835
name: inositol-3,4,6-trisphosphate 1-kinase activity
def: "Catalysis of the reaction: 1D-myo-inositol 3,4,6-trisphosphate + ATP = 1D-myo-inositol 1,3,4,6-tetrakisphosphate + ADP + 2 H(+)." [EC:2.7.1.134]
GO:0052836
name: inositol 5-diphosphate pentakisphosphate 5-kinase activity
def: "Catalysis of the reaction: ATP + inositol 5-diphosphate pentakisphosphate = ADP + inositol 5-triphosphate pentakisphosphate." [PMID:11502751, PMID:18355727]
GO:0052839
name: inositol diphosphate tetrakisphosphate kinase activity
def: "Catalysis of the reaction: Catalysis of the reaction: ATP + inositol diphosphate tetrakisphosphate = ADP + inositol bisdiphosphate trisphosphate." [PMID:10827188, PMID:11502751, PMID:18355727]
GO:0052840
name: inositol diphosphate tetrakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol diphosphate tetrakisphosphate + H2O = inositol 1,3,4,5,6-pentakisphosphate + phosphate." [MetaCyc:RXN-10963, PMID:10827188, PMID:11502751]
GO:0052841
name: inositol bisdiphosphate tetrakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol bisdiphosphate tetrakisphosphate + H2O = inositol diphosphate pentakisphosphate + phosphate." [MetaCyc:RXN-10965, MetaCyc:RXN-10975, MetaCyc:RXN-10976, PMID:10827188, PMID:11502751]
GO:0052842
name: inositol diphosphate pentakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol diphosphate pentakisphosphate + H2O = inositol hexakisphosphate + phosphate." [MetaCyc:RXN-10964, MetaCyc:RXN-10977, MetaCyc:RXN-10978, PMID:10827188, PMID:11502751]
GO:0052843
name: inositol-1-diphosphate-2,3,4,5,6-pentakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol 1-diphosphate 2,3,4,5,6-pentakisphosphate + H2O = inositol 1,2,3,4,5,6-hexakisphosphate + phosphate + 2 H+." [MetaCyc:RXN-10977, PMID:10827188, PMID:11502751]
GO:0052844
name: inositol-3-diphosphate-1,2,4,5,6-pentakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol 3-diphosphate 1,2,4,5,6-pentakisphosphate + H2O = inositol 1,2,3,4,5,6-hexakisphosphate + phosphate + 2 H+." [MetaCyc:RXN-10978, PMID:10827188, PMID:11502751]
GO:0052845
name: inositol-5-diphosphate-1,2,3,4,6-pentakisphosphate diphosphatase activity
def: "Catalysis of the reaction: inositol 5-diphosphate 1,2,3,4,6-pentakisphosphate + H2O = inositol 1,2,3,4,5,6-hexakisphosphate + phosphate + 2 H+." [MetaCyc:RXN-10964, PMID:10827188, PMID:11502751]
GO:0052846
name: inositol-1,5-bisdiphosphate-2,3,4,6-tetrakisphosphate 1-diphosphatase activity
def: "Catalysis of the reaction: 1,5-bisdiphosphoinositol-1D-myo-inositol 2,3,4,6-tetrakisphosphate + H2O = 1-diphospho-1D-myo-inositol 1,2,3,4,6-pentakisphosphate + phosphate + H+." [MetaCyc:RXN-10965, PMID:10827188, PMID:11502751]
GO:0052847
name: inositol-1,5-bisdiphosphate-2,3,4,6-tetrakisphosphate 5-diphosphatase activity
def: "Catalysis of the reaction: 1,5-bisdiphosphoinositol-1D-myo-inositol 2,3,4,6-tetrakisphosphate + H2O = 1-diphospho-1D-myo-inositol 2,3,4,5,6-pentakisphosphate + phosphate + H+." [MetaCyc:RXN-10975, PMID:10827188, PMID:11502751]
GO:0052848
name: inositol-3,5-bisdiphosphate-2,3,4,6-tetrakisphosphate 5-diphosphatase activity
def: "Catalysis of the reaction: 3,5-bisdiphosphoinositol-1D-myo-inositol 2,3,4,6-tetrakisphosphate + H2O = 3-diphospho-1D-myo-inositol 1,2,4,5,6-pentakisphosphate + phosphate + H+." [MetaCyc:RXN-10976, PMID:10827188, PMID:11502751]
GO:0052849
name: NADPH-dependent curcumin reductase activity
def: "Catalysis of the reaction: curcumin + NADPH + H+ = dihydrocurcumin + NADP+." [MetaCyc:RXN0-6676]
GO:0052850
name: NADPH-dependent dihydrocurcumin reductase activity
def: "Catalysis of the reaction: dihydrocurcumin + NADPH+ + H+ = tetrahydrocurcumin + NADP+." [MetaCyc:RXN0-6677]
GO:0052851
name: ferric-chelate reductase (NADPH) activity
def: "Catalysis of the reaction: 2 Fe3+ + NADPH = 2 Fe2+ + H+ + NADP+ ." [MetaCyc:RXN0-6555]
GO:0052852
name: very-long-chain-(S)-2-hydroxy-acid oxidase activity
def: "Catalysis of the reaction: very-long-chain (S)-2-hydroxy-acid + O2 = very-long-chain 2-oxo acid + hydrogen peroxide. Very long chain refers to a chain length of greater than 18 carbons." [EC:1.1.3.15]
GO:0052853
name: long-chain-(S)-2-hydroxy-long-chain-acid oxidase activity
def: "Catalysis of the reaction: long-chain-(S)-2-hydroxy-acid + O2 = long-chain-2-oxo acid + hydrogen peroxide. Long chain refers to a chain length of 14 to 18 carbons." [EC:1.1.3.15]
GO:0052854
name: medium-chain-(S)-2-hydroxy-acid oxidase activity
def: "Catalysis of the reaction: medium-chain-(S)-2-hydroxy-acid + O2 = medium-chain-2-oxo acid + hydrogen peroxide. Medium chain refers to a chain length of between 8 and 12 carbons." [EC:1.1.3.15]
GO:0052855
name: ADP-dependent NAD(P)H-hydrate dehydratase activity
def: "Catalysis of the reaction: (6S)-6beta-hydroxy-1,4,5,6-tetrahydronicotinamide adenine dinucleotide + ADP = AMP + 3 H(+) + NADH + phosphate." [EC:4.2.1.93, PMID:21994945]
GO:0052856
name: NADHX epimerase activity
def: "Catalysis of the reaction: (R)-NADHX = (S)-NADHX." [PMID:21994945]
GO:0052857
name: NADPHX epimerase activity
def: "Catalysis of the reaction: (R)-NADPHX = (S)-NADPHX." [PMID:21994945]
GO:0052858
name: peptidyl-lysine N-acetyltransferase activity, acting on acetyl phosphate as donor
def: "Catalysis of the reaction: acetyl phosphate + peptidyl-L-lysine = phosphate + peptidyl-N6-acetyl-L-lysine." [GOC:tb]
GO:0052859
name: glucan endo-1,4-beta-glucosidase activity
def: "Catalysis of the random hydrolysis of (1->4) linkages in (1->4)-beta-D-glucans." [EC:3.2.1.75]
GO:0052860
name: 2'-deoxymugineic-acid 3-dioxygenase activity
def: "Catalysis of the reaction: 2'-deoxymugineic acid + 2-oxoglutarate + O2 = 3-epihydroxy-2'-deoxymugineic acid + succinate + CO2." [EC:1.14.11.25]
GO:0052861
name: glucan endo-1,3-beta-glucanase activity, C-3 substituted reducing group
def: "Catalysis of the endohydrolysis of (1->3)-linkages in beta-D-glucans when the glucose residue whose reducing group is involved in the linkage to be hydrolysed is itself substituted at C-3." [EC:3.2.1.6]
GO:0052862
name: glucan endo-1,4-beta-glucanase activity, C-3 substituted reducing group
def: "Catalysis of the endohydrolysis of (1->4)-linkages in beta-D-glucans when the glucose residue whose reducing group is involved in the linkage to be hydrolysed is itself substituted at C-3." [EC:3.2.1.6]
GO:0052866
name: phosphatidylinositol phosphate phosphatase activity
def: "Catalysis of the reaction: phosphatidylinositol phosphate(n) + H2O = phosphatidylinositol phosphate(n-1) + phosphate. This reaction is the removal of a phosphate group from a phosphatidylinositol phosphate." [GOC:ai]
GO:0052867
name: phosphatidylinositol-1,4,5-trisphosphate 5-phosphatase activity
def: "Catalysis of the reaction: phosphatidyl-1D-myo-inositol 1,4,5-trisphosphate + H(2)O = 1-phosphatidyl-1D-myo-inositol 1,4-bisphosphate + phosphate." [EC:3.1.3.-]
GO:0052868
name: protein-lysine lysyltransferase activity
def: "Catalysis of the reaction: protein-lysine + protein-lysine = protein N6-(lysyl)-L-lysine + protein. This reaction is the addition of lysine group from one protein to a lysine residue in a second protein, producing N6-(lysyl)-L-lysine." [PMID:20729861]
GO:0052869
name: arachidonic acid omega-hydroxylase activity
def: "Catalysis of the reaction: arachidonic acid + O2 + NADPH + H+ = 20-HETE + NADP+ + H2O. Arachidonic acid is also known as (5Z,8Z,11Z,14Z)-icosatetraenoic acid, and 20-HETE is also known as (5Z,8Z,11Z,14Z)-20-hydroxyicosa-5,8,11,14-tetraenoic acid." [KEGG:R07041]
GO:0052870
name: tocopherol omega-hydroxylase activity
def: "Catalysis of the reaction: tocopherol + O2 + NADPH + H+ = 13'-hydroxy-tocopherol + NADP+ + H2O ." [MetaCyc:RXN-11003]
GO:0052871
name: alpha-tocopherol omega-hydroxylase activity
def: "Catalysis of the reaction: alpha-tocopherol + O2 + NADPH + H+ = 13'-hydroxy-alpha-tocopherol + NADP+ + H2O ." [MetaCyc:RXN-11003]
GO:0052872
name: tocotrienol omega-hydroxylase activity
def: "Catalysis of the reaction: tocotrienol + O2 + NADPH + H+ = 13'-hydroxy-tocotrienol + NADP+ + H2O ." [MetaCyc:RXN-11003]
GO:0052873
name: FMN reductase (NADPH) activity
def: "Catalysis of the reaction: FMNH2 + NADP+ = FMN + NADPH + 2 H+." [RHEA:21627]
GO:0052874
name: FMN reductase (NADH) activity
def: "Catalysis of the reaction: FMNH2 + NAD+ = FMN + NADH + 2 H+." [RHEA:21623]
GO:0052875
name: riboflavin reductase (NADH) activity
def: "Catalysis of the reaction: reduced riboflavin + NAD+ = riboflavin + NADH + 2 H+." [RHEA:31458]
GO:0052876
name: methylamine dehydrogenase (amicyanin) activity
def: "Catalysis of the reaction: methylamine + H2O + amicyanin = formaldehyde + ammonia + reduced amicyanin." [KEGG:R00606]
GO:0052877
name: oxidoreductase activity, acting on the CH-NH2 group of donors, with a copper protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-NH2 group acts as a hydrogen or electron donor and a copper protein is the acceptor." [GOC:ai]
GO:0052878
name: linoleate 8R-lipoxygenase activity
def: "Catalysis of the reaction: O(2) + linoleate = (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate." [RHEA:25398]
GO:0052879
name: 9,12-octadecadienoate 8-hydroperoxide 8S-isomerase activity
def: "Catalysis of the reaction: (8R,9Z,12Z)-8-hydroperoxyoctadeca-9,12-dienoate = (7S,8S,9Z,12Z)-7,8-dihydroxyoctadeca-9,12-dienoate." [RHEA:25402]
GO:0052880
name: oxidoreductase activity, acting on diphenols and related substances as donors, with copper protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a diphenol, or related compound, acts as a hydrogen or electron donor and reduces a copper protein." [GOC:jl]
GO:0052881
name: 4-hydroxyphenylacetate 3-monooxygenase activity
def: "Catalysis of the reaction: (4-hydroxyphenyl)acetate + FADH(2) + O(2) = 3,4-dihydroxyphenylacetate + FAD + H(+) + H(2)O." [RHEA:30598]
GO:0052882
name: oxidoreductase activity, acting on phosphorus or arsenic in donors, with a copper protein as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a phosphorus- or arsenic-containing group acts as a hydrogen or electron donor and reduces a copper protein." [GOC:jl]
GO:0052883
name: tyrosine ammonia-lyase activity
def: "Catalysis of the reaction: L-tyrosine = NH(4)(+) + trans-4-coumarate." [RHEA:24909]
GO:0052884
name: all-trans-retinyl-palmitate hydrolase, 11-cis retinol forming activity
def: "Catalysis of the reaction: H(2)O + all-trans-retinyl palmitate = 11-cis-retinol + H(+) + palmitate." [RHEA:31778]
GO:0052885
name: all-trans-retinyl-ester hydrolase, 11-cis retinol forming activity
def: "Catalysis of the reaction: H(2)O + all-trans-retinyl ester = 11-cis-retinol + fatty acid." [RHEA:31774]
GO:0052886
name: 9,9'-dicis-carotene:quinone oxidoreductase activity
def: "Catalysis of the reaction: 9,9'-di-cis-zeta-carotene + a quinone = 7,9,9'-tri-cis-neurosporene + a quinol." [EC:1.3.5.6]
GO:0052887
name: 7,9,9'-tricis-neurosporene:quinone oxidoreductase activity
def: "Catalysis of the reaction: 7,9,9'-tri-cis-neurosporene + a quinone = 7,9,7',9'-tetra-cis-lycopene + a quinol." [EC:1.3.5.6]
GO:0052888
name: OBSOLETE:dihydroorotate oxidase (fumarate) activity
def: "Catalysis of the reaction: (S)-dihydroorotate + fumarate = orotate + succinate." [EC:1.3.98.1, RHEA:30062]
GO:0052890
name: oxidoreductase activity, acting on the CH-CH group of donors, with a flavin as acceptor
def: "Catalysis of an oxidation-reduction (redox) reaction in which a CH-CH group acts as a hydrogen or electron donor and reduces a flavin." [GOC:jl]
GO:0052891
name: aliphatic (S)-hydroxynitrile lyase activity
def: "Catalysis of the reaction: an aliphatic (S)-hydroxynitrile = an aliphatic aldehyde or ketone + cyanide." [EC:4.1.2.47]
GO:0052892
name: aromatic (S)-hydroxynitrile lyase activity
def: "Catalysis of the reaction: an aromatic (S)-hydroxynitrile = an aromatic aldehyde + cyanide." [EC:4.1.2.47, KEGG:R09359]
GO:0052893
name: N1-acetylspermine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity
def: "Catalysis of the reaction: N1-acetylspermine + oxygen + H2O = N-(3-acetamidopropyl)-4-aminobutanal + propane-1,3-diamine + hydrogen peroxide." [MetaCyc:RXN-10465]
GO:0052894
name: norspermine:oxygen oxidoreductase activity
def: "Catalysis of the reaction: norspermine + oxygen + H2O = norspermidine + 3-aminopropanal + hydrogen peroxide." [MetaCyc:RXN-10464]
GO:0052895
name: N1-acetylspermine:oxygen oxidoreductase (N1-acetylspermidine-forming) activity
def: "Catalysis of the reaction: N1-acetylspermine + oxygen + H2O = N1-acetylspermidine + 3-aminopropanal + hydrogen peroxide." [MetaCyc:POLYAMINE-OXIDASE-RXN]
GO:0052896
name: spermidine oxidase (propane-1,3-diamine-forming) activity
def: "Catalysis of the reaction: H(2)O + O(2) + spermidine = 1,3-diaminopropane + 4-aminobutanal + H(2)O(2)." [RHEA:25823]
GO:0052897
name: N8-acetylspermidine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity
def: "Catalysis of the reaction: H(2)O + N(8)-acetylspermidine + O(2) = 1,3-diaminopropane + 4-acetamidobutanal + H(2)O(2)." [RHEA:25975]
GO:0052898
name: N1-acetylspermidine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity
def: "Catalysis of the reaction: H(2)O + N(1)-acetylspermidine + O(2) = 1,3-diaminopropane + 4-acetamidobutanal + H(2)O(2)." [RHEA:25867]
GO:0052899
name: N(1),N(12)-diacetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H(2)O + N(1),N(12)-diacetylspermine + O(2) = 3-acetamidopropanal + H(2)O(2) + N(1)-acetylspermidine." [RHEA:25871]
GO:0052900
name: spermine oxidase (propane-1,3-diamine-forming) activity
def: "Catalysis of the reaction: H(2)O + O(2) + spermine = 1,3-diaminopropane + H(2)O(2) + N-(3-aminopropyl)-4-aminobutanal." [RHEA:25827]
GO:0052901
name: spermine:oxygen oxidoreductase (spermidine-forming) activity
def: "Catalysis of the reaction: H(2)O + O(2) + spermine = 3-aminopropanal + H(2)O(2) + spermidine." [RHEA:25807]
GO:0052902
name: spermidine:oxygen oxidoreductase (3-aminopropanal-forming) activity
def: "Catalysis of the reaction: H(2)O + O(2) + spermidine = 3-aminopropanal + H(2)O(2) + putrescine." [RHEA:25811]
GO:0052903
name: N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H(2)O + N(1)-acetylspermine + O(2) = 3-acetamidopropanal + H(2)O(2) + spermidine." [RHEA:25803]
GO:0052904
name: N1-acetylspermidine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H(2)O + N(1)-acetylspermidine + O(2) = 3-acetamidopropanal + H(2)O(2) + putrescine." [RHEA:25815]
GO:0052905
name: tRNA (guanine(9)-N(1))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanine(9) in tRNA = N(1)-methylguanine(9) in tRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.221]
GO:0052906
name: tRNA (guanine(37)-N(1))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanine(37) in tRNA = N(1)-methylguanine(37) in tRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.228]
GO:0052907
name: 23S rRNA (adenine(1618)-N(6))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + adenine(1618) in 23S rRNA = S-adenosyl-L-homocysteine + rRNA containing N(6)-methyladenine(1618) in 23S rRNA." [EC:2.1.1.181]
GO:0052908
name: 16S rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase activity
def: "Catalysis of the reaction: 4 S-adenosyl-L-methionine + adenine(1518)/adenine(1519) in 16S rRNA = 4 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(1518)/N(6)-dimethyladenine(1519) in 16S rRNA." [EC:2.1.1.182]
GO:0052909
name: 18S rRNA (adenine(1779)-N(6)/adenine(1780)-N(6))-dimethyltransferase activity
def: "Catalysis of the reaction: 4 S-adenosyl-L-methionine + adenine(1779)/adenine(1780) in 18S rRNA = 4 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(1779)/N(6)-dimethyladenine(1780) in 18S rRNA." [EC:2.1.1.183]
GO:0052910
name: 23S rRNA (adenine(2085)-N(6))-dimethyltransferase activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + adenine(2085) in 23S rRNA = 2 S-adenosyl-L-homocysteine + N(6)-dimethyladenine(2085) in 23S rRNA." [EC:2.1.1.184]
GO:0052911
name: 23S rRNA (guanine(745)-N(1))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanine(745) in 23S rRNA = N(1)-methylguanine(745) in 23S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.187]
GO:0052912
name: 23S rRNA (guanine(748)-N(1))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanine(748) in 23S rRNA = N(1)-methylguanine(748) in 23S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.188]
GO:0052913
name: 16S rRNA (guanine(966)-N(2))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine(966) in 16S rRNA = N(2)-methylguanosine(966) in 16S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.171]
GO:0052914
name: 16S rRNA (guanine(1207)-N(2))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine(1207) in 16S rRNA = N(2)-methylguanosine(1207) in 16S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.172]
GO:0052915
name: 23S rRNA (guanine(2445)-N(2))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine(2445) in 23S rRNA = N(2)-methylguanosine(2445) in 23S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.173]
GO:0052916
name: 23S rRNA (guanine(1835)-N(2))-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + guanosine(1835) in 23S rRNA = N(2)-methylguanosine(1835) in 23S rRNA + S-adenosyl-L-homocysteine." [EC:2.1.1.174]
GO:0052917
name: dol-P-Man:Man(7)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase activity
def: "Catalysis of the reaction: alpha-D-man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl D-mannosyl phosphate = H(+) + alpha-D-Man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->6))-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl phosphate." [EC:2.4.1.260, RHEA:29538]
GO:0052918
name: dol-P-Man:Man(8)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase activity
def: "Catalysis of the reaction: alpha-D-man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->6))-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl D-mannosyl phosphate = H(+) + alpha-D-Man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-[alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->2)-alpha-D-Man-(1->6))-alpha-D-Man-(1->6)]-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl phosphate." [EC:2.4.1.261, RHEA:29542]
GO:0052919
name: aliphatic (R)-hydroxynitrile lyase activity
def: "Catalysis of the reaction: an aliphatic (R)-hydroxynitrile = an aliphatic aldehyde or ketone + hydrogen cyanide." [EC:4.1.2.46]
GO:0052920
name: (2R)-2-hydroxy-2-methylbutanenitrile lyase activity
def: "Catalysis of the reaction: (2R)-2-hydroxy-2-methylbutanenitrile = butan-2-one + hydrogen cyanide." [EC:4.1.2.46]
GO:0052921
name: acetone-cyanohydrin acetone-lyase (cyanide-forming) activity
def: "Catalysis of the reaction: acetone cyanohydrin = hydrogen cyanide + acetone." [KEGG:R01553, MetaCyc:ACETONE-CYANHYDRIN-LYASE-RXN]
GO:0052922
name: hexaprenyl diphosphate synthase (geranylgeranyl-diphosphate specific) activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate + 2 isopentenyl diphosphate = 2 diphosphate + all-trans-hexaprenyl diphosphate." [RHEA:27558]
GO:0052923
name: all-trans-nonaprenyl-diphosphate synthase (geranyl-diphosphate specific) activity
def: "Catalysis of the reaction: geranyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate + all-trans-nonaprenyl diphosphate." [EC:2.5.1.84, RHEA:27566]
GO:0052924
name: all-trans-nonaprenyl-diphosphate synthase (geranylgeranyl-diphosphate specific) activity
def: "Catalysis of the reaction: geranylgeranyl diphosphate + 5 isopentenyl diphosphate = 5 diphosphate + all-trans-nonaprenyl diphosphate." [EC:2.5.1.85]
GO:0052925
name: dol-P-Man:Man(5)GlcNAc(2)-PP-Dol alpha-1,3-mannosyltransferase activity
def: "Catalysis of the reaction: an alpha-D-man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl D-mannosyl phosphate = H(+) + alpha-D-Man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->3)-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl phosphate." [RHEA:29530]
GO:0052926
name: dol-P-Man:Man(6)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase activity
def: "Catalysis of the reaction: alpha-D-man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->3)-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl D-mannosyl phosphate = H(+) + alpha-D-Man-(1->2)-alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-(alpha-D-Man-(1->2)-alpha-D-Man-(1->3)-alpha-D-Man-(1->6))-beta-D-Man-(1->4)-beta-D-GlcNAc-(1->4)-D-GlcNAc-diphosphodolichol + dolichyl phosphate." [EC:2.4.1.259, RHEA:29534]
GO:0052927
name: CTP:tRNA cytidylyltransferase activity
def: "Catalysis of the reaction: a tRNA precursor + CTP = a tRNA with a 3' cytidine end + diphosphate." [KEGG:R09383]
GO:0052928
name: CTP:3'-cytidine-tRNA cytidylyltransferase activity
def: "Catalysis of the reaction: a tRNA with a 3' cytidine + CTP = a tRNA with a 3' CC end + diphosphate." [KEGG:R09384]
GO:0052929
name: ATP:3'-cytidine-cytidine-tRNA adenylyltransferase activity
def: "Catalysis of the reaction: a tRNA with a 3' CC end + ATP = a tRNA with a 3' CCA end + diphosphate." [KEGG:R09386]
GO:0052930
name: methanol ferricytochrome-c oxidoreductase activity
def: "Catalysis of the reaction: methanol + 2 ferricytochrome c = formaldehyde + 2 ferrocytochrome c + 2 H+." [KEGG:R01146]
GO:0052931
name: ethanol cytochrome-c oxidoreductase activity
def: "Catalysis of the reaction: ethanol + 2 cytochrome c(L) = acetaldehyde + 2 reduced cytochrome c(L) + 2 H+." [KEGG:R09127]
GO:0052932
name: 2-chloroethanol cytochrome-c oxidoreductase activity
def: "Catalysis of the reaction: 2-chloroethanol + 2 cytochrome c(L) = chloroacetaldehyde + 2 reduced cytochrome c(L)." [KEGG:R09128]
GO:0052933
name: alcohol dehydrogenase (cytochrome c(L)) activity
def: "Catalysis of the reaction: primary alcohol + 2 cytochrome c(L) = 2 reduced cytochrome c(L) + an aldehyde + 2 H+." [EC:1.1.2.7]
GO:0052934
name: alcohol dehydrogenase (cytochrome c) activity
def: "Catalysis of the reaction: primary alcohol + 2 cytochrome c = 2 reduced cytochrome c + an aldehyde + 2 H+." [EC:1.1.2.8]
GO:0052935
name: ethanol:cytochrome c oxidoreductase activity
def: "Catalysis of the reaction: ethanol + 2 cytochrome c = 2 reduced cytochrome c + acetaldehyde." [KEGG:R05198]
GO:0052936
name: 2-chloroethanol:cytochrome c oxidoreductase activity
def: "Catalysis of the reaction: 2-chloroethanol + 2 cytochrome c = chloroacetaldehyde + 2 reduced cytochrome c." [KEGG:R05285]
GO:0055041
name: cyclopentanol dehydrogenase activity
def: "Catalysis of the reaction: cyclopentanol + NAD(+) = cyclopentanone + H(+) + NADH." [EC:1.1.1.163, RHEA:11731]
GO:0055042
name: 5-valerolactone hydrolase activity
def: "Catalysis of the reaction: 5-valerolactone + H2O = 5-hydroxyvalerate." [GOC:jid, GOC:mlg]
GO:0055043
name: 5-oxovalerate dehydrogenase activity
def: "Catalysis of the reaction: 5-oxovalerate + NADP+ + H2O = glutarate + NADPH + H+." [GOC:jid, GOC:mlg]
GO:0055053
name: mannose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: mannose + H+ = mannose + H+." [GOC:ct]
GO:0055054
name: fructose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: fructose + H+ = fructose + H+." [GOC:ct]
GO:0055055
name: D-glucose:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: D-glucose + H+ = D-glucose + H+. Symporter activity enables the active transport of a solute across a membrane by a mechanism whereby two or more species are transported together in the same direction in a tightly coupled process not directly linked to a form of energy other than chemiosmotic energy. D-glucose is the dextrorotatory D-enantiomer of glucose." [GOC:ct]
GO:0055056
name: D-glucose transmembrane transporter activity
def: "Catalysis of the transfer of the D-enantiomer of the hexose monosaccharide glucose from one side of the membrane to the other." [GOC:jid, GOC:jsg, GOC:mah]
GO:0055077
name: gap junction hemi-channel activity
def: "A wide pore channel activity that enables the transport of a solute across a membrane via a gap junction hemi-channel. Two gap junction hemi-channels coupled together form a complete gap junction." [GOC:dgh]
GO:0055100
name: adiponectin binding
def: "Interacting selectively and non-covalently with adiponectin, a protein hormone produced by adipose tissue that modulates a number of metabolic processes, including glucose regulation and fatty acid catabolism." [GOC:BHF, GOC:rl, PMID:15210937]
GO:0055101
name: glycerophospholipase inhibitor activity
def: "OBSOLETE. Stops, prevents or reduces the activity of a glycerophospholipase, an enzyme that catalyzes of the hydrolysis of a glycerophospholipid." [GOC:ai, GOC:BHF, GOC:rl]
GO:0055102
name: lipase inhibitor activity
def: "Stops, prevents or reduces the activity of a lipase, an enzyme that catalyzes of the hydrolysis of a lipid." [GOC:BHF, GOC:rl]
GO:0055103
name: ligase regulator activity
def: "Modulates the activity of a ligase." [GOC:BHF, GOC:rl]
GO:0055104
name: ligase inhibitor activity
def: "Stops, prevents or reduces the activity of a ligase." [GOC:BHF, GOC:rl]
GO:0055105
name: ubiquitin-protein transferase inhibitor activity
def: "Stops, prevents or reduces the activity of a ubiquitin-protein transferase." [GOC:BHF, GOC:rl]
GO:0055106
name: ubiquitin-protein transferase regulator activity
def: "Modulates the activity of a ubiquitin-protein transferase, an enzyme that catalyzes the covalent attachment of ubiquitin to lysine in a substrate protein." [GOC:BHF, GOC:rl]
GO:0055131
name: C3HC4-type RING finger domain binding
def: "Interacting selectively and non-covalently with a C3HC4-type zinc finger domain of a protein. The C3HC4-type zinc finger is a variant of RING finger, is a cysteine-rich domain of 40 to 60 residues that coordinates two zinc ions, and has the consensus sequence: C-X2-C-X(9-39)-C-X(1-3)-H-X(2-3)-C-X2-C-X(4-48)-C-X2-C, where X is any amino acid. Many proteins containing a C3HC4-type RING finger play a key role in the ubiquitination pathway." [GOC:amm, InterPro:IPR001841, InterPro:IPR018957]
GO:0060001
name: minus-end directed microfilament motor activity
def: "Catalysis of movement along a microfilament towards the minus end, coupled to the hydrolysis of a nucleoside triphosphate (usually ATP). The minus end of an actin filament is the end that does not preferentially add actin monomers." [GOC:dph, PMID:10519557]
GO:0060002
name: plus-end directed microfilament motor activity
def: "Catalysis of movement along a microfilament towards the plus end, coupled to the hydrolysis of a nucleoside triphosphate (usually ATP). The plus end of an actin filament is the end that preferentially adds actin monomers." [GOC:dph, PMID:10519557]
GO:0060072
name: large conductance calcium-activated potassium channel activity
def: "Catalysis of the transmembrane transfer of potassium by a channel with a unit conductance of 100 to 220 picoSiemens that opens in response to stimulus by concerted actions of internal calcium ions and membrane potential. Large conductance calcium-activated potassium channels are less sensitive to calcium than are small or intermediate conductance calcium-activated potassium channels. Transport by a channel involves catalysis of facilitated diffusion of a solute (by an energy-independent process) involving passage through a transmembrane aqueous pore or channel, without evidence for a carrier-mediated mechanism." [GOC:mtg_transport, ISBN:0815340729, PMID:17115074]
GO:0060089
name: molecular transducer activity
def: "The molecular function that accepts an input of one form and creates an output of a different form." [GOC:mtg_MIT_16mar07]
GO:0060090
name: binding, bridging
def: "The binding activity of a molecule that brings together two or more molecules through a selective, non-covalent, often stoichiometric interaction, permitting those molecules to function in a coordinated way." [GOC:mtg_MIT_16mar07, GOC:vw]
GO:0060175
name: brain-derived neurotrophic factor-activated receptor activity
def: "Combining with a brain-derived neurotrophic factor and transmitting the signal across the plasma membrane to initiate a change in cell activity." [GOC:bf, GOC:dph]
GO:0060182
name: apelin receptor activity
def: "Combining with the peptide apelin to initiate a change in cell activity." [GOC:dph]
GO:0060228
name: phosphatidylcholine-sterol O-acyltransferase activator activity
def: "Increases the activity of phosphatidylcholine-sterol O-acyltransferase, an enzyme that converts cholesterol and phosphatidylcholine (lecithins) to cholesteryl esters and lyso-phosphatidylcholines." [GOC:BHF, GOC:dph, GOC:tb, PMID:4335615]
GO:0060229
name: lipase activator activity
def: "Increases the activity of a lipase, an enzyme that catalyzes of the hydrolysis of a lipid." [GOC:BHF, GOC:dph, GOC:tb]
GO:0060230
name: lipoprotein lipase activator activity
def: "Increases the activity of a lipoprotein lipase, an enzyme that catalyzes of the hydrolysis of a lipid within a lipoprotein." [GOC:BHF, GOC:dph, GOC:tb, PMID:10727238]
GO:0060241
name: lysozyme inhibitor activity
def: "Stops, prevents or reduces the activity of a lysozyme, an enzyme that hydrolyses the beta- (1,4) glycosidic bond between N-acetylmuramic acid and N-acetylglucosamine in peptidoglycan." [GOC:dph]
GO:0060308
name: GTP cyclohydrolase I regulator activity
def: "Modulates the activity of GTP cyclohydrolase I. GTP cyclohydrolase I activity catalyzes the reaction: GTP + 2 H2O = formate + 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)-dihydropteridine triphosphate." [GOC:dph, GOC:tb]
GO:0060422
name: peptidyl-dipeptidase inhibitor activity
def: "Stops, prevents or reduces the activity of a peptidyl-dipeptidase. Peptidyl-dipeptidase activity catalyzes the release of C-terminal dipeptides from a polypeptide chain." [GOC:dph, GOC:tb]
GO:0060589
name: nucleoside-triphosphatase regulator activity
def: "Modulates the rate of NTP hydrolysis by a NTPase." [GOC:dph, GOC:tb]
GO:0060590
name: ATPase regulator activity
def: "Modulates the rate of ATP hydrolysis by an ATPase." [GOC:dph, GOC:tb]
GO:0060698
name: endoribonuclease inhibitor activity
def: "Stops, prevents or reduces the activity of an endoribonuclease, any enzyme that catalyzes the hydrolysis of ester linkages within ribonucleic acid by creating internal breaks." [GOC:dph, GOC:tb]
GO:0060703
name: deoxyribonuclease inhibitor activity
def: "Stops, prevents or reduces the activity of deoxyribonuclease." [GOC:dph, GOC:tb]
GO:0060724
name: coreceptor activity involved in epidermal growth factor receptor signaling pathway
def: "Combining with an extracellular messenger, and in cooperation with a primary EGF receptor, initiating a change in cell activity through the EGF receptor signaling pathway." [GOC:dph, GOC:tb]
GO:0060961
name: phospholipase D inhibitor activity
def: "Stops, prevents or reduces the activity of a phospholipase D, an enzyme that catalyzes the reaction: a phosphatidylcholine + H2O = choline + a phosphatidate." [GOC:dph, GOC:tb]
GO:0061133
name: endopeptidase activator activity
def: "Increases the activity of an endopeptidase, any enzyme that hydrolyzes nonterminal peptide bonds in polypeptides." [GOC:dph, GOC:tb]
GO:0061134
name: peptidase regulator activity
def: "Modulates the activity of a peptidase, any enzyme that catalyzes the hydrolysis peptide bonds." [GOC:dph, GOC:tb]
GO:0061135
name: endopeptidase regulator activity
def: "Modulates the activity of a peptidase, any enzyme that hydrolyzes nonterminal peptide bonds in polypeptides." [GOC:dph, GOC:tb]
GO:0061459
name: L-arginine transmembrane transporter activity
def: "Catalysis of the stereospecific transfer of L-arginine, 2-amino-5-guanidinopentanoic acid, across a biological membrane." [GOC:dph]
GO:0061463
name: O-acetyl-ADP-ribose deacetylase activity
def: "Catalysis of the reaction O-acetyl-ADP-ribose + H2O = ADP-ribose + acetate." [GOC:dph]
GO:0061473
name: murein tripeptide carboxypeptidase activity
def: "Catalysis of the reaction L-Ala-gamma;-D-Glu-meso-Dap (murein tripeptide) + H2O = L-Ala-&gamma;-D-Glu + meso-diaminopimelate." [GOC:dph, PMID:22970852]
GO:0061483
name: sulfinylpropanyl adenylate synthase
def: "Catalysis of the reaction: cysteine sulfanate + GTP + IMP = sulfinylpropanyl adenylate + GDP + 3 H(+) + phosphate." [GOC:dph, PMID:8346915]
GO:0061486
name: high-affinity fructose transmembrane transporter activity
def: "Catalysis of the transfer of fructose from one side of a membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:dph, PMID:10735857]
GO:0061501
name: cyclic-GMP-AMP synthase activity
def: "Catalysis of the reaction: ATP + GTP = 2 diphosphate + cyclic GMP-AMP." [GOC:dph, PMID:23258413]
GO:0061503
name: tRNA threonylcarbamoyladenosine dehydratase
def: "Catalysis of the ATP-dependent dehydration of t6A to form cyclic t6A." [GOC:dph, PMID:23242255]
GO:0061505
name: DNA topoisomerase II activity
def: "Catalysis of a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; changes the linking number in multiples of 2." [GOC:dph]
GO:0061506
name: DNA topoisomerase type II (ATP-independent) activity
def: "Catalysis of a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is not coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:dph]
GO:0061507
name: cyclic-GMP-AMP binding
def: "Interacting selectively and non-covalently with cyclic-GMP-AMP (cGAMP) cyclic nucleotide." [GOC:dph, PMID:23258412]
GO:0061513
name: glucose 6-phosphate:inorganic phosphate antiporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucose 6-phosphate(out) + inorganic phosphate(in) = glucose 6-phosphate(in) + inorganic phosphate(out)." [GOC:dph, PMID:18337460]
GO:0061522
name: 1,4-dihydroxy-2-naphthoyl-CoA thioesterase activity
def: "Catalysis of the reaction 1,4-dihydroxy-2-naphthoyl-CoA + H2O = 1,4-dihydroxy-2-naphthoate + CoA." [GOC:dph]
GO:0061542
name: 3-demethylubiquinone-n 3-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-demethylubiquinone-n = S-adenosyl-L-homocysteine + ubiquinone-n." [EC:2.1.1.64, GOC:dph]
GO:0061543
name: 3-demethylubiquinone-6 3-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + 3-demethylubiquinone-6 = S-adenosyl-L-homocysteine + ubiquinone-6." [GOC:dph]
GO:0061547
name: glycogen synthase activity, transferring glucose-1-phosphate
def: "Catalysis of the reaction: UDP-glucose + (1,4)-alpha-D-glucosyl(n) = UMP + (1,4)-alpha-D-glucosyl(n)-glucose-1-phosphate." [GOC:dph, PMID:21356517]
GO:0061575
name: cyclin-dependent protein serine/threonine kinase activator activity
def: "Increases the activity of a cyclin-dependent protein serine/threonine kinase." [GOC:dph, PMID:2569363, PMID:3322810]
GO:0061578
name: Lys63-specific deubiquitinase activity
def: "Hydrolysis of Lys63-Linked ubiquitin unit(s) from a ubiquitinated protein." [GOC:dph, GOC:pg, PMID:18313383]
GO:0061579
name: N-acyl homoserine lactone synthase activity
def: "Catalyzing the reaction: acyl-[acyl-carrier-protein] + S-adenosyl-L-methionine -> [acyl-carrier- protein] + S-methyl-5'-thioadenosine + N-acyl-L-homoserine lactone." [GOC:dph]
GO:0061593
name: sulfoquinovose isomerase activity
def: "Catalysis of the reaction sulfoquinovose = 6-deoxy-6-sulfofructose." [GOC:dph, PMID:24463506]
GO:0061594
name: 6-deoxy-6-sulfofructose kinase activity
def: "Catalysis of the reaction 6-deoxy-6-sulfofructose + ATP = 6-deoxy-6-sulfofructose-1-phosphate + ADP." [PMID:24463506]
GO:0061595
name: 6-deoxy-6-sulfofructose-1-phosphate aldolase activity
def: "Catalysis of the reaction 6-deoxy-6-sulfofructose-1-phosphate = 3-sulfolactaldehyde + dihydroxyacetone phosphate." [PMID:24463506]
GO:0061596
name: 3-sulfolactaldehyde reductase activity
def: "Catalysis of the reaction 2,3-dihydroxypropane-1-sulfonate + NAD+ = 3-sulfolactaldehyde + NADH + H+." [GOC:dph, PMID:24463506]
GO:0061597
name: cyclic pyranopterin monophosphate synthase activity
def: "Catalysis of the reaction GTP = cyclic pyranopterin phosphate + diphosphate." [EC:4.1.99.18, GOC:dph, PMID:18154309]
GO:0061598
name: molybdopterin adenylyltransferase activity
def: "Catalysis of the reaction ATP + molybdopterin = diphosphate + adenylyl-molybdopterin." [EC:2.7.7.75, GOC:dph]
GO:0061599
name: molybdopterin molybdotransferase activity
def: "Catalysis of the reaction adenylyl-molybdopterin + molybdate = molybdenum cofactor + AMP." [EC:2.10.1.1, GOC:dph]
GO:0061602
name: molybdenum cofactor cytidylyltransferase activity
def: "Catalysis of the reaction CTP + molybdenum cofactor = diphosphate + cytidylyl molybdenum cofactor." [EC:2.7.7.76, GOC:dph]
GO:0061603
name: molybdenum cofactor guanylyltransferase activity
def: "Catalysis of the reaction GTP + molybdenum cofactor = diphosphate + guanylyl molybdenum cofactor." [EC:2.7.7.77, GOC:dph]
GO:0061604
name: molybdopterin-synthase sulfurtransferase activity
def: "Catalysis of the reaction: [Molybdopterin-synthase sulfur-carrier protein]-Gly-Gly-AMP + [cysteine desulfurase]-S-sulfanyl-L-cysteine <=> AMP [molybdopterin-synthase sulfur-carrier protein]-Gly-NH-CH(2)-C(O)SH + cysteine desulfurase." [EC:2.8.1.11, GOC:dph, PMID:18154309, PMID:22370186]
GO:0061605
name: molybdopterin-synthase adenylyltransferase activity
def: "Catalysis of the reaction: ATP [molybdopterin-synthase sulfur-carrier protein]-Gly-Gly = diphosphate [molybdopterin-synthase sulfur-carrier protein]-Gly-Gly-AMP." [EC:2.7.7.80, GOC:dph, PMID:18154309, PMID:22370186]
GO:0061607
name: peptide alpha-N-propionyltransferase activity
def: "Catalysis of the reaction: propionyl-CoA + peptide = CoA + N-alpha-propionylpeptide. This reaction is the propionylation of the N-terminal amino acid residue of a peptide or protein." [GOC:dph, PMID:23043182]
GO:0061608
name: nuclear import signal receptor activity
def: "Combining with a nuclear import signal (NIS) to mediate transport of the NIS-containing protein through the nuclear pore to the nucleus." [GOC:dph, GOC:vw]
GO:0061609
name: fructose-1-phosphate aldolase activity
def: "Catalysis of the reaction: D-fructose-1-phosphate = dihydroxyacetone phosphate + D-glyceraldehyde." [GOC:dph, GOC:glycolysis, ISBN:0201090910, RHEA:30854]
GO:0061627
name: S-methylmethionine-homocysteine S-methyltransferase activity
def: "Catalysis of the reaction: S-methyl-L-methionine + L-homocysteine = 2 L-methionine + H+." [EC:2.1.1.10, GOC:BHF, GOC:dph]
GO:0061628
name: H3K27me3 modified histone binding
def: "Interacting selectively and non-covalently with a histone H3 in which the lysine residue at position 27 has been modified by trimethylation." [GOC:dph, PMID:23948251]
GO:0061629
name: RNA polymerase II sequence-specific DNA binding transcription factor binding
def: "Interacting selectively and non-covalently with a sequence-specific DNA binding RNA polymerase II transcription factor, any of the factors that interact selectively and non-covalently with a specific DNA sequence in order to modulate transcription." [GOC:dph, GOC:vw]
GO:0061630
name: ubiquitin protein ligase activity
def: "Catalysis of the transfer of ubiquitin to a substrate protein via the reaction X-ubiquitin + S --> X + S-ubiquitin, where X is either an E2 or E3 enzyme, the X-ubiquitin linkage is a thioester bond, and the S-ubiquitin linkage is an isopeptide bond between the C-terminal glycine of ubiquitin and the epsilon-amino group of lysine residues in the substrate. Note that this may include the extension of ubiquitin chains." [GOC:BioGRID, GOC:dph]
GO:0061631
name: ubiquitin conjugating enzyme activity
def: "Isoenergetic transfer of ubiquitin from one protein to another via the reaction X-ubiquitin + Y -> Y-ubiquitin + X, where both the X-ubiquitin and Y-ubiquitin linkages are thioester bonds between the C-terminal glycine of ubiquitin and a sulfhydryl side group of a cysteine residue." [GOC:BioGRID, GOC:dph]
GO:0061632
name: RNA lariat debranching enzyme activator activity
def: "Increases the activity of an enzyme that catalyzes the hydrolysis of branched RNA structures that contain vicinal 2'-5'- and 3'-5'-phosphodiester bonds at a branch point nucleotide." [GOC:dph, PMID:24919400]
GO:0061634
name: alpha-D-xyloside xylohydrolase
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-D-xylose residues with release of alpha-D-xylose." [EC:3.2.1.177]
GO:0061636
name: mitotic anaphase-promoting complex activator activity
def: "Increases the activity of a mitotic anaphase promoting complex, an ubiquitin-protein transferase activity that regulates the mitotic metaphase/anaphase transition." [GOC:dph]
GO:0061637
name: mitotic anaphase-promoting complex inhibitor activity
def: "Decreases the activity of a mitotic anaphase promoting complex, an ubiquitin-protein transferase activity that regulates the mitotic metaphase/anaphase transition." [GOC:dph]
GO:0061649
name: ubiquitinated histone binding
def: "Interacting selectively and non-covalently with a histone protein in which a residue has been modified by ubiquitination. Histones are any of a group of water-soluble proteins found in association with the DNA of plant and animal chromosomes." [GOC:dph, PMID:24526689]
GO:0061650
name: small conjugating protein conjugating enzyme activity
def: "Isoenergetic transfer of a small conjugating protein (SCP) from one protein to another via the reaction X-SCP + Y -> Y-SCP + X, where both the X-SCP and Y-SCP linkages are thioester bonds between the C-terminal amino acid of SCP and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061651
name: Atg12 conjugating enzyme activity
def: "Isoenergetic transfer of Atg12 from one protein to another via the reaction X-Atg12 + Y -> Y-Atg12 + X, where both the X-Atg12 and Y-Atg12 linkages are thioester bonds between the C-terminal amino acid of Atg12 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061652
name: FAT10 conjugating enzyme activity
def: "Isoenergetic transfer of FAT10 from one protein to another via the reaction X-FAT10 + Y -> Y-FAT10 + X, where both the X-FAT10 and Y-FAT10 linkages are thioester bonds between the C-terminal amino acid of FAT10 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061653
name: ISG15 conjugating enxyme activity
def: "Isoenergetic transfer of ISG15 from one protein to another via the reaction X-ISG15 + Y -> Y-ISG15 + X, where both the X-ISG15 and Y-ISG15 linkages are thioester bonds between the C-terminal amino acid of ISG15 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061654
name: NEDD8 conjugating enzyme activity
def: "Isoenergetic transfer of NEDD8 from one protein to another via the reaction X-NEDD8 + Y -> Y-NEDD8 + X, where both the X-NEDD8 and Y-NEDD8 linkages are thioester bonds between the C-terminal amino acid of NEDD8 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061655
name: Pup conjugating enzyme activity
def: "Isoenergetic transfer of Pup from one protein to another via the reaction X-Pup + Y -> Y-Pup + X, where both the X-Pup and Y-Pup linkages are thioester bonds between the C-terminal amino acid of Pup and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061656
name: SUMO conjugating enzyme activity
def: "Isoenergetic transfer of SUMO from one protein to another via the reaction X-SUMO + Y -> Y-SUMO + X, where both the X-SUMO and Y-SUMO linkages are thioester bonds between the C-terminal amino acid of SUMO and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061657
name: UFM1 conjugating enzyme activity
def: "Isoenergetic transfer of UFM1 from one protein to another via the reaction X-UFM1 + Y -> Y-UFM1 + X, where both the X-UFM1 and Y-UFM1 linkages are thioester bonds between the C-terminal amino acid of UFM1 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061658
name: URM1 conjugating enzyme activity
def: "Isoenergetic transfer of URM1 from one protein to another via the reaction X-URM1 + Y -> Y-URM1 + X, where both the X-URM1 and Y-URM1 linkages are thioester bonds between the C-terminal amino acid of URM1 and a sulfhydryl side group of a cysteine residue." [GOC:dph]
GO:0061659
name: small conjugating protein ligase activity
def: "Catalysis of the transfer of a small conjugating protein (SCP) to a substrate protein via the reaction X-SCP + S --> X + S-SCP, where X is either an E2 or E3 enzyme, the X-SCP linkage is a thioester bond, and the S-SCP linkage is an isopeptide bond between the C-terminal glycine of SCP and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061660
name: Atg12 ligase activity
def: "Catalysis of the transfer of Atg12 to a substrate protein via the reaction X-Atg12 + S --> X + S-Atg12, where X is either an E2 or E3 enzyme, the X-Atg12 linkage is a thioester bond, and the S-Atg12 linkage is an isopeptide bond between the C-terminal amino acid of Atg12 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061661
name: FAT10 ligase activity
def: "Catalysis of the transfer of FAT10 to a substrate protein via the reaction X-FAT10 + S --> X + S-FAT10, where X is either an E2 or E3 enzyme, the X-FAT10 linkage is a thioester bond, and the S-FAT10 linkage is an isopeptide bond between the C-terminal glycine of FAT10 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061662
name: ISG15 ligase activity
def: "Catalysis of the transfer of a ISG15 to a substrate protein via the reaction X-ISG15 + S --> X + S-ISG15, where X is either an E2 or E3 enzyme, the X-ISG15 linkage is a thioester bond, and the S-ISG15 linkage is an isopeptide bond between the C-terminal amino acid of ISG15 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061663
name: NEDD8 ligase activity
def: "Catalysis of the transfer of NEDD8 to a substrate protein via the reaction X-NEDD8 + S --> X + S-NEDD8, where X is either an E2 or E3 enzyme, the X-NEDD8 linkage is a thioester bond, and the S-NEDD8 linkage is an isopeptide bond between the C-terminal amino acid of NEDD8 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061664
name: Pup ligase activity
def: "Catalysis of the transfer of Pup to a substrate protein via the reaction X-Pup + S --> X + S-Pup, where X is either an E2 or E3 enzyme, the X-Pup linkage is a thioester bond, and the S-Pup linkage is an isopeptide bond between the C-terminal amino acid of Pup and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061665
name: SUMO ligase activity
def: "Catalysis of the transfer of SUMO to a substrate protein via the reaction X-SUMO + S --> X + S-SUMO, where X is either an E2 or E3 enzyme, the X-SUMO linkage is a thioester bond, and the S-SUMO linkage is an isopeptide bond between the C-terminal amino acid of SUMO and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061666
name: UFM1 ligase activity
def: "Catalysis of the transfer of UFM1 to a substrate protein via the reaction X-UFM1 + S --> X + S-UFM1, where X is either an E2 or E3 enzyme, the X-UFM1 linkage is a thioester bond, and the S-UFM1 linkage is an isopeptide bond between the C-terminal amino acid of UFM1 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061667
name: URM1 ligase activity
def: "Catalysis of the transfer of URM1 to a substrate protein via the reaction X-URM1 + S --> X + S-URM1, where X is either an E2 or E3 enzyme, the X-URM1 linkage is a thioester bond, and the S-URM1 linkage is an isopeptide bond between the C-terminal amino acid of URM1 and the epsilon-amino group of lysine residues in the substrate." [GOC:dph]
GO:0061675
name: RBL family protein binding
def: "Interacting selectively and non-covalently with any member of the rhamnose-binding lectin (RBL) family, a family of animal lectins that show specific binding activities to L-rhamnose or D-galactose." [PMID:22312473]
GO:0061676
name: importin-alpha family protein binding
def: "Interacting selectively and non-covalently with any member of the importin-alpha family." [PMID:15350979, PMID:17170104, PMID:23734157]
GO:0061677
name: 2-dehydro-3-deoxy-D-gluconate aldolase activity
def: "Catalysis of the reaction: 2-dehydro-3-deoxy-D-gluconate <=> pyruvate + D-glyceraldehyde." [EC:4.1.2.51, GOC:dph]
GO:0070001
name: aspartic-type peptidase activity
def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which a water molecule bound by the side chains of aspartic residues at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070002
name: glutamic-type peptidase activity
def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070003
name: threonine-type peptidase activity
def: "Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which the hydroxyl group of a threonine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070004
name: cysteine-type exopeptidase activity
def: "Catalysis of the hydrolysis of C- or N-terminal peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#EXOPEPTIDASE]
GO:0070005
name: cysteine-type aminopeptidase activity
def: "Catalysis of the hydrolysis of N-terminal peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#AMINOPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070006
name: metalloaminopeptidase activity
def: "Catalysis of the hydrolysis of N-terminal amino acid residues from a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#AMINOPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070007
name: glutamic-type endopeptidase activity
def: "Catalysis of the hydrolysis of internal peptide bonds in a polypeptide chain by a mechanism involving a glutamate/glutamine catalytic dyad." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#ENDOPEPTIDASE]
GO:0070008
name: serine-type exopeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond not more than three residues from the N- or C-terminus of a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, http://merops.sanger.ac.uk/about/glossary.htm#EXOPEPTIDASE, ISBN:0716720094]
GO:0070009
name: serine-type aminopeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond not more than three residues from the N-terminus of a polypeptide chain by a catalytic mechanism that involves a catalytic triad consisting of a serine nucleophile that is activated by a proton relay involving an acidic residue (e.g. aspartate or glutamate) and a basic residue (usually histidine)." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#AMINOPEPTIDASE, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE, ISBN:0716720094]
GO:0070010
name: peptidase activity, acting on D-amino acid peptides
def: "Catalysis of the hydrolysis of peptide bonds formed between D-amino acids." [GOC:mah]
GO:0070011
name: peptidase activity, acting on L-amino acid peptides
def: "Catalysis of the hydrolysis of peptide bonds formed between L-amino acids." [GOC:mah]
GO:0070012
name: oligopeptidase activity
def: "Catalysis of the hydrolysis of a peptide bond in an oligopeptide, i.e. a molecule containing a small number (2 to 20) of amino acid residues connected by peptide bonds." [GOC:mah, ISBN:0198506732]
GO:0070016
name: armadillo repeat domain binding
def: "Interacting selectively and non-covalently with the armadillo repeat domain of a protein, an approximately 40 amino acid long tandemly repeated sequence motif first identified in the Drosophila segment polarity protein armadillo. Arm-repeat proteins are involved in various processes, including intracellular signalling and cytoskeletal regulation." [GOC:BHF, GOC:mah, GOC:vk, InterPro:IPR000225]
GO:0070025
name: carbon monoxide binding
def: "Interacting selectively and non-covalently with carbon monoxide (CO)." [GOC:ecd]
GO:0070026
name: nitric oxide binding
def: "Interacting selectively and non-covalently with nitric oxide (NO)." [GOC:ecd]
GO:0070027
name: carbon monoxide sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of carbon monoxide (CO)." [GOC:ecd]
GO:0070034
name: telomeric RNA binding
def: "Interacting selectively and non-covalently with the telomerase RNA template or to RNA complementary to it." [GOC:krc, PMID:16884717]
GO:0070035
name: purine NTP-dependent helicase activity
def: "Catalysis of the reaction: purine NTP + H2O = purine NDP + phosphate, to drive the unwinding of a DNA or RNA helix." [GOC:mah]
GO:0070036
name: GTP-dependent helicase activity
def: "Catalysis of the reaction: GTP + H2O = GDP + phosphate, to drive the unwinding of a DNA or RNA helix." [GOC:mah]
GO:0070037
name: rRNA (pseudouridine) methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to a pseudouridine residue in an rRNA molecule." [GOC:imk, GOC:mah]
GO:0070038
name: rRNA (pseudouridine-N3-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N3-methylpseudouridine." [GOC:imk, GOC:mah]
GO:0070039
name: rRNA (guanosine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylguanosine." [GOC:imk, GOC:mah]
GO:0070040
name: rRNA (adenine-C2-)-methyltransferase activity
def: "Catalysis of the reaction: 2 S-adenosyl-L-methionine + adenine(2503) in 23S rRNA = S-adenosyl-L-homocysteine + 5'-deoxyadenosine + L-methionine + rRNA containing C2-methyladenine(2503) in 23S rRNA." [GOC:imk, PMID:20007606, PMID:20184321, PMID:21368151, PMID:21415317, PMID:21527678]
GO:0070041
name: rRNA (uridine-C5-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing C5-methyluridine." [GOC:imk, GOC:mah]
GO:0070042
name: rRNA (uridine-N3-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N3-methyluridine." [GOC:imk, GOC:mah]
GO:0070043
name: rRNA (guanine-N7-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N7-methylguanine." [GOC:imk, GOC:mah]
GO:0070051
name: fibrinogen binding
def: "Interacting selectively and non-covalently with fibrinogen, a highly soluble hexameric glycoprotein complex that is found in blood plasma and is converted to fibrin by thrombin in the coagulation cascade." [GOC:BHF, GOC:mah, GOC:vk]
GO:0070052
name: collagen V binding
def: "Interacting selectively and non-covalently with a type V collagen trimer." [GOC:BHF, GOC:mah]
GO:0070053
name: thrombospondin receptor activity
def: "Combining with thrombospondin and transmitting the signal to initiate a change in cell activity." [GOC:BHF, GOC:signaling, GOC:vk]
GO:0070061
name: fructose binding
def: "Interacting selectively and non-covalently with the D- or L-enantiomer of fructose, the ketohexose arabino-hex-2-ulose." [CHEBI:28757, GOC:BHF, GOC:mah]
GO:0070063
name: RNA polymerase binding
def: "Interacting selectively and non-covalently with an RNA polymerase molecule or complex." [GOC:BHF, GOC:mah, GOC:txnOH]
GO:0070064
name: proline-rich region binding
def: "Interacting selectively and non-covalently with a proline-rich region, i.e. a region that contains a high proportion of proline residues, in a protein." [GOC:mah]
GO:0070080
name: titin Z domain binding
def: "Interacting selectively and non-covalently with the titin Z domain, which recognizes and binds to the C-terminal calmodulin-like domain of alpha-actinin-2 (Act-EF34), adopts a helical structure, and binds in a groove formed by the two planes between the helix pairs of Act-EF34." [GOC:mah, InterPro:IPR015129]
GO:0070087
name: chromo shadow domain binding
def: "Interacting selectively and non-covalently with a chromo shadow domain, a protein domain that is distantly related, and found in association with, the chromo domain." [GOC:BHF, GOC:vk, InterPro:IPR008251, PMID:7667093]
GO:0070089
name: chloride-activated potassium channel activity
def: "Catalysis of the chloride concentration-regulatable energy-independent passage of potassium ions across a lipid bilayer down a concentration gradient." [GOC:kmv, GOC:mtg_transport]
GO:0070095
name: fructose-6-phosphate binding
def: "Interacting selectively and non-covalently with fructose 6-phosphate." [GOC:mah]
GO:0070097
name: delta-catenin binding
def: "Interacting selectively and non-covalently with the delta subunit of the catenin complex." [GOC:rph]
GO:0070119
name: ciliary neurotrophic factor binding
def: "Interacting selectively and non-covalently with the cytokine ciliary neurotrophic factor." [GOC:BHF, GOC:mah]
GO:0070122
name: isopeptidase activity
def: "Catalysis of the hydrolysis of an isopeptide bond. An isopeptide bond is an amide linkage between a carboxyl group of one amino acid and an amino group of another amino acid in which at least one of these groups is not on the a-carbon of one of the amino acids (for example, the link between an epsilon-amino group of a lysine molecule to a carboxyl group on a second amino acid is an isopeptide bond)." [GOC:mah, http://128.240.24.212/cgi-bin/omd?isopeptide+bond]
GO:0070123
name: transforming growth factor beta receptor activity, type III
def: "Combining with transforming growth factor beta to initiate a change in cell activity; facilitates ligand binding to type I and type II TGF-beta receptors." [GOC:BHF, GOC:mah, PMID:9759503]
GO:0070137
name: small conjugating protein-specific endopeptidase activity
def: "Catalysis of the hydrolysis of peptide bonds between an alpha-carboxyl group and an alpha-amino group within a small protein such as ubiquitin or a ubiquitin-like protein (e.g. APG8, ISG15, NEDD8, SUMO)." [GOC:mah]
GO:0070138
name: small conjugating protein-specific isopeptidase activity
def: "Catalysis of the hydrolysis of an isopeptide bond between a small protein such as ubiquitin or a ubiquitin-like protein (e.g. APG8, ISG15, NEDD8, SUMO) and a protein to which the small protein has been conjugated." [GOC:mah]
GO:0070139
name: SUMO-specific endopeptidase activity
def: "Catalysis of the hydrolysis of peptide bonds between an alpha-carboxyl group and an alpha-amino group within the small conjugating protein SUMO." [GOC:mah]
GO:0070140
name: SUMO-specific isopeptidase activity
def: "Catalysis of the hydrolysis of an isopeptide bond between the small conjugating protein SUMO and a protein to which SUMO has been conjugated." [GOC:mah]
GO:0070180
name: large ribosomal subunit rRNA binding
def: "Interacting selectively and non-covalently with the large ribosomal subunit RNA (LSU rRNA), a constituent of the large ribosomal subunit. In S. cerevisiae, this is the 25S rRNA." [GOC:elh]
GO:0070181
name: small ribosomal subunit rRNA binding
def: "Interacting selectively and non-covalently with the small ribosomal subunit RNA (SSU rRNA), a constituent of the small ribosomal subunit. In S. cerevisiae, this is the 18S rRNA." [GOC:elh]
GO:0070182
name: DNA polymerase binding
def: "Interacting selectively and non-covalently with a DNA polymerase." [GOC:BHF, GOC:mah]
GO:0070186
name: growth hormone activity
def: "The action characteristic of growth hormone, a peptide hormone that is secreted by the anterior pituitary or the placenta into the circulation, and binds to membrane receptors in target tissues to stimulate body growth." [GOC:BHF, GOC:mah, PMID:11445442]
GO:0070190
name: inositol hexakisphosphate 1-kinase or 3-kinase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 1-diphospho-1D-myo-inositol (2,3,4,5,6)pentakisphosphate, and ATP + 1D-myo-inositol hexakisphosphate = ADP + 3-diphospho-1D-myo-inositol (1,2,4,5,6)pentakisphosphate." [GOC:jp, PMID:18981179]
GO:0070191
name: methionine-R-sulfoxide reductase activity
def: "Catalysis of the reaction: L-methionine R-oxide + thioredoxin = L-methionine + thioredoxin disulfide; can act on free oxidized methionine with specificity for the R enantiomer; does not act on oxidized methionine in peptide linkage. Thioredoxin disulfide is the oxidized form of thioredoxin." [GOC:mcc, PMID:17535911, PMID:19049972]
GO:0070204
name: 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase activity
def: "Catalysis of the reaction: 2-oxoglutarate + H(+) + isochorismate = 5-enolpyruvoyl-6-hydroxy-2-succinyl-cyclohex-3-ene-1-carboxylate + CO(2)." [EC:2.2.1.9, RHEA:25596]
GO:0070205
name: 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase activity
def: "Catalysis of the reaction: 5-enolpyruvoyl-6-hydroxy-2-succinyl-cyclohex-3-ene-1-carboxylate = (1R,6R)-2-succinyl-6-hydroxycyclohexa-2,4-diene-1-carboxylate + pyruvate." [EC:4.2.99.20, RHEA:25600]
GO:0070215
name: MDM2 binding
def: "OBSOLETE. Interacting selectively and non-covalently with any isoform of the MDM2 protein, a negative regulator of p53." [GOC:mah, GOC:nln]
GO:0070216
name: MDM4 binding
def: "OBSOLETE. Interacting selectively and non-covalently with any isoform of the MDM4 protein, a negative regulator of p53." [GOC:mah, GOC:nln]
GO:0070224
name: sulfide:quinone oxidoreductase activity
def: "Catalysis of the reaction: hydrogen sulfide + a quinone = S0 + a hydroquinone." [MetaCyc:R17-RXN]
GO:0070225
name: sulfide dehydrogenase activity
def: "Catalysis of the reaction: hydrogen sulfide + oxidized cytochrome c = S0 + reduced cytochrome c." [MetaCyc:RXN-8156]
GO:0070226
name: sulfur:ferric ion oxidoreductase activity
def: "Catalysis of the reaction: a perthiol + 4 Fe3+ + 3 H2O = sulfite + a thiol + 4 Fe2+ + 8 H+." [MetaCyc:SULFFEOXIDO-RXN]
GO:0070251
name: pristanate-CoA ligase activity
def: "Catalysis of the reaction: ATP + pristanate + CoA = AMP + diphosphate + pristanoyl-CoA." [GOC:pde, PMID:10198260]
GO:0070259
name: tyrosyl-DNA phosphodiesterase activity
def: "Catalysis of the hydrolysis of phosphotyrosyl groups formed as covalent intermediates (in DNA backbone breakage) between a DNA topoisomerase and DNA." [GOC:elh, PMID:16751265]
GO:0070260
name: 5'-tyrosyl-DNA phosphodiesterase activity
def: "Catalysis of the hydrolysis of 5'-phosphotyrosyl groups formed as covalent intermediates (in DNA backbone breakage) between DNA topoisomerase II and DNA." [PMID:16751265]
GO:0070273
name: phosphatidylinositol-4-phosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-4-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 4' position." [GOC:bf, GOC:mah]
GO:0070279
name: vitamin B6 binding
def: "Interacting selectively and non-covalently with any of the vitamin B6 compounds: pyridoxal, pyridoxamine and pyridoxine and the active form, pyridoxal phosphate." [GOC:mah]
GO:0070280
name: pyridoxal binding
def: "Interacting selectively and non-covalently with pyridoxal, 3-hydroxy-5-(hydroxymethyl)-2-methylpyridine-4-carbaldehyde, a form of vitamin B6." [CHEBI:17310, GOC:mah]
GO:0070281
name: pyridoxamine binding
def: "Interacting selectively and non-covalently with pyridoxamine, 4-(aminomethyl)-5-(hydroxymethyl)-2-methylpyridin-3-ol, a form of vitamin B6." [CHEBI:16410, GOC:mah]
GO:0070282
name: pyridoxine binding
def: "Interacting selectively and non-covalently with pyridoxine, 4,5-bis(hydroxymethyl)-2-methylpyridin-3-ol, a form of vitamin B6." [CHEBI:16709, GOC:mah]
GO:0070283
name: radical SAM enzyme activity
def: "Catalysis of a reaction in which S-adenosyl-L-methionine (SAM) undergoes reductive cleavage to serve as a source of the 5'-deoxyadenosyl radical." [PMID:17291766, PMID:18307109]
GO:0070284
name: 4-amino-5-hydroxymethyl-2-methylpyrimidine phosphate synthase activity
def: "Catalysis of the reaction: 5-aminoimidazole ribonucleotide + S-adenosylmethionine = 4-amino-5-hydroxymethyl-2-methylpyrimidine phosphate + 5'deoxyadenosine." [PMID:18953358]
GO:0070287
name: ferritin receptor activity
def: "Combining with ferritin, and delivering ferritin into the cell via endocytosis." [GOC:bf, PMID:17459943, PMID:19154717]
GO:0070290
name: N-acylphosphatidylethanolamine-specific phospholipase D activity
def: "Catalysis of the release of N-acylethanolamine from N-acyl-phosphatidylethanolamine (NAPE) to generate N-acylethanolamine (NAE)." [GOC:elh, PMID:14634025, PMID:15878693]
GO:0070300
name: phosphatidic acid binding
def: "Interacting selectively and non-covalently with phosphatidic acid, any of a class of glycerol phosphate in which both the remaining hydroxyl groups of the glycerol moiety are esterified with fatty acids." [CHEBI:16337, GOC:jp, ISBN:0198506732]
GO:0070320
name: inward rectifier potassium channel inhibitor activity
def: "Stops, prevents, or reduces the activity of an inwardly rectifying potassium channel." [GOC:mah]
GO:0070324
name: thyroid hormone binding
def: "Interacting selectively and non-covalently with thyroxine (T4) or triiodothyronine (T3), tyrosine-based hormones produced by the thyroid gland." [GOC:rph]
GO:0070325
name: lipoprotein particle receptor binding
def: "Interacting selectively and non-covalently with a lipoprotein particle receptor." [GOC:BHF, GOC:rl]
GO:0070326
name: very-low-density lipoprotein particle receptor binding
def: "Interacting selectively and non-covalently with a very-low-density lipoprotein receptor." [GOC:BHF, GOC:mah]
GO:0070330
name: aromatase activity
def: "Catalysis of the reduction of an aliphatic ring to yield an aromatic ring." [GOC:cb]
GO:0070335
name: aspartate binding
def: "Interacting selectively and non-covalently with aspartate, the alpha-amino-acid anion of 2-aminobutanedioic acid that has formula C4H5NO4." [CHEBI:29995, GOC:mah]
GO:0070336
name: flap-structured DNA binding
def: "Interacting selectively and non-covalently with a flap structure in DNA. A DNA flap structure is one in which a single-stranded length of DNA or RNA protrudes from a double-stranded DNA molecule." [GOC:mah, PMID:15189154]
GO:0070337
name: 3'-flap-structured DNA binding
def: "Interacting selectively and non-covalently with a 3'-flap structure in DNA. A DNA flap structure is one in which a single-stranded 3'-end of DNA or RNA protrudes from a double-stranded DNA molecule." [GOC:mah, PMID:15189154]
GO:0070338
name: 5'-flap-structured DNA binding
def: "Interacting selectively and non-covalently with a 5'-flap structure in DNA. A DNA flap structure is one in which a single-stranded 5'-end of DNA or RNA protrudes from a double-stranded DNA molecule. 5'-flap structures can be formed during DNA repair or lagging strand synthesis; in the latter case RNA flaps form from lagging strand RNA primers." [GOC:mah, PMID:15189154]
GO:0070361
name: mitochondrial light strand promoter anti-sense binding
def: "Interacting selectively and non-covalently with the anti-sense strand of the light strand promoter, a promoter located on the light, or cytosine-rich, strand of mitochondrial DNA." [GOC:mah, PMID:9485316]
GO:0070362
name: mitochondrial heavy strand promoter anti-sense binding
def: "Interacting selectively and non-covalently with the anti-sense strand of the heavy strand promoter, a promoter located on the heavy, or guanine-rich, strand of mitochondrial DNA." [GOC:mah, PMID:9485316]
GO:0070363
name: mitochondrial light strand promoter sense binding
def: "Interacting selectively and non-covalently with the sense strand of the light strand promoter, a promoter located on the light, or cytosine-rich, strand of mitochondrial DNA." [GOC:mah, PMID:9485316]
GO:0070364
name: mitochondrial heavy strand promoter sense binding
def: "Interacting selectively and non-covalently with the sense strand of the heavy strand promoter, a promoter located on the heavy, or guanine-rich, strand of mitochondrial DNA." [GOC:mah, PMID:9485316]
GO:0070379
name: high mobility group box 1 binding
def: "Interacting selectively and non-covalently with high mobility group box 1 (HMBGB1)." [GOC:add, PMID:18431461]
GO:0070380
name: high mobility group box 1 receptor activity
def: "Combining with high mobility group box 1 (HMBGB1) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling, PMID:18431461]
GO:0070401
name: NADP+ binding
def: "Interacting selectively and non-covalently with the oxidized form, NADP+, of nicotinamide-adenine dinucleotide phosphate, a coenzyme involved in many redox and biosynthetic reactions." [GOC:mah]
GO:0070402
name: NADPH binding
def: "Interacting selectively and non-covalently with the reduced form, NADPH, of nicotinamide-adenine dinucleotide phosphate, a coenzyme involved in many redox and biosynthetic reactions." [GOC:mah]
GO:0070403
name: NAD+ binding
def: "Interacting selectively and non-covalently with the oxidized form, NAD, of nicotinamide adenine dinucleotide, a coenzyme involved in many redox and biosynthetic reactions." [GOC:mah]
GO:0070404
name: NADH binding
def: "Interacting selectively and non-covalently with the reduced form, NADH, of nicotinamide adenine dinucleotide, a coenzyme involved in many redox and biosynthetic reactions." [GOC:mah]
GO:0070405
name: ammonium ion binding
def: "Interacting selectively and non-covalently with ammonium ions (NH4+)." [CHEBI:28938, GOC:ecd]
GO:0070406
name: glutamine binding
def: "Interacting selectively and non-covalently with glutamine, 2,5-diamino-5-oxopentanoic acid." [CHEBI:28300, GOC:ecd]
GO:0070410
name: co-SMAD binding
def: "Interacting selectively and non-covalently with a common mediator SMAD signaling protein." [GOC:BHF, GOC:vk, PMID:19114992]
GO:0070411
name: I-SMAD binding
def: "Interacting selectively and non-covalently with an inhibitory SMAD signaling protein." [GOC:BHF, GOC:vk, PMID:19114992]
GO:0070412
name: R-SMAD binding
def: "Interacting selectively and non-covalently with a receptor-regulated SMAD signaling protein." [GOC:BHF, GOC:vk, PMID:19114992]
GO:0070448
name: laricitrin 5'-O-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + laricitrin = S-adenosyl-L-homocysteine + syringetin." [MetaCyc:RXN-8452]
GO:0070456
name: galactose-1-phosphate phosphatase activity
def: "Catalysis of the reaction: galactose-1-phosphate + H2O = galactose + phosphate." [GOC:mah]
GO:0070457
name: D-galactose-1-phosphate phosphatase activity
def: "Catalysis of the reaction: D-galactose-1-phosphate + H2O = D-galactose + phosphate." [GOC:mah, PMID:9462881]
GO:0070463
name: tubulin-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate. This reaction requires the presence of a tubulin dimer to accelerate release of ADP and phosphate." [GOC:mah, PMID:16906148]
GO:0070491
name: repressing transcription factor binding
def: "Interacting selectively and non-covalently with a transcription repressor, any protein whose activity is required to prevent or downregulate transcription." [GOC:mah, GOC:txnOH]
GO:0070492
name: oligosaccharide binding
def: "Interacting selectively and non-covalently with any oligosaccharide, a molecule with between two and (about) 20 monosaccharide residues connected by glycosidic linkages." [GOC:mah]
GO:0070497
name: 6-carboxy-5,6,7,8-tetrahydropterin synthase activity
def: "Catalysis of the reaction: 7,8-dihydroneopterin triphosphate + H2O = 6-carboxy-5,6,7,8-tetrahydropterin + triphosphate + acetaldehyde + 2 H+." [GOC:imk, MetaCyc:RXN0-5507, PMID:19231875]
GO:0070506
name: high-density lipoprotein particle receptor activity
def: "Combining with a high-density lipoprotein particle and delivering the high-density lipoprotein into the cell via endocytosis." [GOC:bf, GOC:BHF, GOC:rl, PMID:9211901]
GO:0070513
name: death domain binding
def: "Interacting selectively and non-covalently with a death domain of a protein. The death domain (DD) is a homotypic protein interaction module composed of a bundle of six alpha-helices. DD bind each other forming oligomers. Some DD-containing proteins are involved in the regulation of apoptosis and inflammation through their activation of caspases and NF-kappaB." [GOC:BHF, GOC:rl, InterPro:IPR000488, Pfam:PF00531]
GO:0070523
name: 11-beta-hydroxysteroid dehydrogenase (NAD+) activity
def: "Catalysis of the reaction: an 11-beta-hydroxysteroid + NAD+ = an 11-oxosteroid + NADH + H+." [PMID:15761036]
GO:0070524
name: 11-beta-hydroxysteroid dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: an 11-beta-hydroxysteroid + NADP+ = an 11-oxosteroid + NADPH + H+." [EC:1.1.1.146]
GO:0070529
name: L-tryptophan aminotransferase activity
def: "Catalysis of the transfer of an amino group from L-tryptophan to an acceptor, usually a 2-oxo acid." [GOC:mah]
GO:0070530
name: K63-linked polyubiquitin binding
def: "Interacting selectively and non-covalently and non-covalently with a polymer of ubiquitin formed by linkages between lysine residues at position 63 of the ubiquitin monomers." [GOC:mah, PMID:15556404, PMID:17525341]
GO:0070538
name: oleic acid binding
def: "Interacting selectively and non-covalently with oleic acid, the 18-carbon monounsaturated fatty acid (9Z)-octadec-9-enoic acid." [CHEBI:16196, GOC:lp, GOC:mah]
GO:0070539
name: linoleic acid binding
def: "Interacting selectively and non-covalently with linoleic acid, the 18-carbon unsaturated fatty acid (9Z,12Z)-octadeca-9,12-dienoic acid." [CHEBI:17351, GOC:lp, GOC:mah]
GO:0070540
name: stearic acid binding
def: "Interacting selectively and non-covalently with stearic acid, the 18-carbon saturated fatty acid octadecanoic acid." [CHEBI:28842, GOC:lp, GOC:mah]
GO:0070546
name: L-phenylalanine aminotransferase activity
def: "Catalysis of the transfer of an amino group from L-phenylalanine to an acceptor, usually a 2-oxo acid." [GOC:mah]
GO:0070547
name: L-tyrosine aminotransferase activity
def: "Catalysis of the transfer of an amino group from L-tyrosine to an acceptor, usually a 2-oxo acid." [GOC:mah]
GO:0070548
name: L-glutamine aminotransferase activity
def: "Catalysis of the transfer of an amino group from L-glutamine to an acceptor, usually a 2-oxo acid." [GOC:mah]
GO:0070551
name: endoribonuclease activity, cleaving siRNA-paired mRNA
def: "Catalysis of the endonucleolytic cleavage of the mRNA in a double-stranded RNA molecule formed by the base pairing of an mRNA with an siRNA, yielding 5'-phosphomonoesters." [GOC:mah, PMID:15105377]
GO:0070553
name: nicotinic acid receptor activity
def: "Combining with nicotinic acid to initiate a change in cell activity." [CHEBI:15940, GOC:mah, PMID:12522134]
GO:0070566
name: adenylyltransferase activity
def: "Catalysis of the transfer of an adenylyl group to an acceptor." [GOC:mah]
GO:0070567
name: cytidylyltransferase activity
def: "Catalysis of the transfer of a cytidylyl group to an acceptor." [GOC:mah]
GO:0070568
name: guanylyltransferase activity
def: "Catalysis of the transfer of a guanylyl group to an acceptor." [GOC:mah]
GO:0070569
name: uridylyltransferase activity
def: "Catalysis of the transfer of an uridylyl group to an acceptor." [GOC:mah]
GO:0070573
name: metallodipeptidase activity
def: "Catalysis of the hydrolysis of a dipeptide by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions." [GOC:mah, http://merops.sanger.ac.uk/about/glossary.htm#CATTYPE]
GO:0070576
name: vitamin D 24-hydroxylase activity
def: "Catalysis of the hydroxylation of C-24 of any form of vitamin D." [GOC:BHF, GOC:mah, PMID:15546903]
GO:0070577
name: lysine-acetylated histone binding
def: "Interacting selectively and non-covalently with a histone in which a lysine residue has been modified by acetylation." [GOC:BHF, GOC:mah, GOC:rl, PMID:17582821]
GO:0070579
name: methylcytosine dioxygenase activity
def: "Catalysis of the reaction: methylcytosine + 2-oxoglutarate + O2 = 5-hydroxymethylcytosine + succinate + CO2." [PMID:19372391]
GO:0070594
name: juvenile hormone response element binding
def: "Interacting selectively and non-covalently with the juvenile hormone response element (JHRE), a conserved sequence found in the promoters of genes whose expression is regulated in response to juvenile hormone." [GOC:sart, PMID:17956872]
GO:0070611
name: histone methyltransferase activity (H3-R2 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone H3)-arginine (position 2) = S-adenosyl-L-homocysteine + (histone H3)-N-methyl-arginine (position 2). This reaction is the addition of a methyl group to arginine at position 2 of histone H3." [EC:2.1.1.125, GOC:mah, PMID:17898714]
GO:0070612
name: histone methyltransferase activity (H2A-R3 specific)
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone H2A)-arginine (position 3) = S-adenosyl-L-homocysteine + (histone H2A)-N-methyl-arginine (position 3). This reaction is the addition of a methyl group to arginine at position 3 of histone H2A." [EC:2.1.1.125, GOC:mah, PMID:17898714]
GO:0070615
name: nucleosome-dependent ATPase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate. This reaction requires the presence of one or more nucleosomes." [GOC:mah, PMID:19165147]
GO:0070626
name: (S)-2-(5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamido)succinate AMP-lyase (fumarate-forming) activity
def: "Catalysis of the reaction: (S)-2-(5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamido)succinate = fumarate + 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide." [GOC:mah, GOC:pde]
GO:0070628
name: proteasome binding
def: "Interacting selectively and non-covalently with a proteasome, a large multisubunit protein complex that catalyzes protein degradation." [GOC:mah]
GO:0070635
name: nicotinamide riboside hydrolase activity
def: "Catalysis of the reaction: nicotinamide riboside + H2O = nicotinamide + D-ribose." [MetaCyc:RXN-8441, PMID:19001417]
GO:0070636
name: nicotinic acid riboside hydrolase activity
def: "Catalysis of the reaction: nicotinic acid riboside + H2O = nicotinic acid + D-ribose." [GOC:mah, PMID:19001417]
GO:0070643
name: vitamin D 25-hydroxylase activity
def: "Catalysis of the hydroxylation of C-25 of any form of vitamin D." [GOC:BHF, GOC:mah]
GO:0070644
name: vitamin D response element binding
def: "Interacting selectively and non-covalently with the vitamin D response element (VDRE), a short sequence with dyad symmetry found in the promoters of some of the cellular immediate-early genes, regulated by serum." [GOC:BHF, GOC:vk, PMID:17426122]
GO:0070653
name: high-density lipoprotein particle receptor binding
def: "Interacting selectively and non-covalently with a high-density lipoprotein receptor." [GOC:BHF, GOC:mah]
GO:0070674
name: hypoxanthine dehydrogenase activity
def: "Catalysis of the reaction: hypoxanthine + NAD+ + H2O = xanthine + NADH + H+." [GOC:mah, GOC:pde]
GO:0070675
name: hypoxanthine oxidase activity
def: "Catalysis of the reaction: hypoxanthine + H2O + O2 = xanthine + H2O2." [GOC:mah, GOC:pde]
GO:0070677
name: rRNA (cytosine-2'-O-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing 2'-O-methylcytosine." [GOC:mah, PMID:19400805]
GO:0070678
name: preprotein binding
def: "Interacting selectively and non-covalently with a preprotein, the unprocessed form of a protein destined to undergo co- or post-translational processing." [GOC:imk, GOC:mah, PMID:12914940]
GO:0070679
name: inositol 1,4,5 trisphosphate binding
def: "Interacting selectively and non-covalently with inositol 1,4,5 trisphosphate." [GOC:BHF, GOC:mah]
GO:0070694
name: deoxyribonucleoside 5'-monophosphate N-glycosidase activity
def: "Catalysis of the reaction: a deoxyribonucleoside 5'-monophosphate + H2O = deoxyribose 5-monophosphate + a purine or pyrimidine base." [GOC:ab, PMID:17234634]
GO:0070696
name: transmembrane receptor protein serine/threonine kinase binding
def: "Interacting selectively and non-covalently with a receptor that spans a cell membrane and possesses protein serine/threonine kinase activity." [GOC:BHF, GOC:mah]
GO:0070697
name: activin receptor binding
def: "Interacting selectively and non-covalently with an activin receptor." [GOC:BHF, GOC:vk]
GO:0070698
name: type I activin receptor binding
def: "Interacting selectively and non-covalently with a type I activin receptor." [GOC:BHF, GOC:vk]
GO:0070699
name: type II activin receptor binding
def: "Interacting selectively and non-covalently with a type II activin receptor." [GOC:BHF, GOC:vk]
GO:0070700
name: BMP receptor binding
def: "Interacting selectively and non-covalently with a BMP receptor." [GOC:BHF, GOC:vk]
GO:0070704
name: sterol desaturase activity
def: "Catalysis of the introduction of a double bond into a sterol molecule." [GOC:mah, GOC:vw]
GO:0070717
name: poly-purine tract binding
def: "Interacting selectively and non-covalently with any stretch of purines (adenine or guanine) in an RNA molecule." [GOC:mah]
GO:0070728
name: leucine binding
def: "Interacting selectively and non-covalently with 2-amino-4-methylpentanoic acid." [CHEBI:25017, GOC:BHF, GOC:mah]
GO:0070733
name: protein adenylyltransferase activity
def: "Catalysis of the reaction: ATP + protein = diphosphate + adenylyl-protein; mediates the addition of an adenylyl (adenosine 5'-monophosphate; AMP group) to specific residues of target proteins." [GOC:mah, PMID:19039103, PMID:19362538]
GO:0070735
name: protein-glycine ligase activity
def: "Catalysis of the posttranslational transfer of one or more glycine residues to a specific glutamate residue on a target protein." [GOC:mah, PMID:19524510]
GO:0070736
name: protein-glycine ligase activity, initiating
def: "Catalysis of the posttranslational transfer of a glycine residue to the gamma-carboxyl group(s) of one or more specific glutamate residues on a target protein." [GOC:mah, PMID:19524510]
GO:0070737
name: protein-glycine ligase activity, elongating
def: "Catalysis of the posttranslational transfer of one or more glycine residues to a glycine residue covalently attached to the gamma-carboxyl group of a glutamate residue on a target protein, resulting in the elongation of a polyglycine side chain." [GOC:mah, PMID:19524510]
GO:0070738
name: tubulin-glycine ligase activity
def: "Catalysis of the posttranslational transfer of one or more glycine residues to a specific glutamate residue on a target tubulin molecule; acts on alpha or beta tubulin." [GOC:mah, PMID:19524510]
GO:0070739
name: protein-glutamic acid ligase activity
def: "Catalysis of the posttranslational transfer of one or more glutamate residues to a specific residue on a target protein." [GOC:mah, PMID:19524510]
GO:0070740
name: tubulin-glutamic acid ligase activity
def: "Catalysis of the posttranslational transfer of one or more glutamate residues to the gamma-carboxyl group(s) of one or more specific glutamate residues on a tubulin molecule." [GOC:mah, PMID:19524510]
GO:0070742
name: C2H2 zinc finger domain binding
def: "Interacting selectively and non-covalently with a C2H2-type zinc finger domain of a protein. The C2H2 zinc finger is the classical zinc finger domain, in which two conserved cysteines and histidines co-ordinate a zinc ion." [GOC:BHF, GOC:mah, Pfam:PF00096]
GO:0070746
name: interleukin-35 binding
def: "Interacting selectively and non-covalently with interleukin-35." [GOC:add]
GO:0070747
name: interleukin-35 receptor activity
def: "Combining with interleukin-35 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity." [GOC:add, GOC:signaling]
GO:0070748
name: interleukin-35 receptor binding
def: "Interacting selectively and non-covalently with the interleukin-35 receptor." [GOC:add]
GO:0070773
name: protein-N-terminal glutamine amidohydrolase activity
def: "Catalysis of the reaction: protein-N-terminal-L-glutamine + H2O = protein-N-terminal-L-glutamate + NH3. This reaction is the deamidation of an N-terminal glutamine residue of a protein." [PMID:19560421]
GO:0070774
name: phytoceramidase activity
def: "Catalysis of the reaction: a phytoceramide + H2O = a fatty acid + phytosphingosine." [GOC:pde, PMID:11356846]
GO:0070780
name: dihydrosphingosine-1-phosphate phosphatase activity
def: "Catalysis of the reaction: dihydrosphingosine 1-phosphate + H2O = dihydrosphingosine + phosphate." [GOC:pde, PMID:12815058]
GO:0070812
name: glycerol-2-phosphate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + glycerol-2-phosphate(out) = ADP + phosphate + glycerol-2-phosphate(in)." [GOC:mah]
GO:0070815
name: peptidyl-lysine 5-dioxygenase activity
def: "Catalysis of the reaction: protein L-lysine + 2-oxoglutarate + O2 = protein 5-hydroxy-L-lysine + succinate + CO2." [PMID:19574390]
GO:0070818
name: protoporphyrinogen oxidase activity
def: "Catalysis of the reaction: protoporphyrinogen IX + acceptor = protoporphyrin IX + reduced acceptor." [EC:1.3.3.4, GOC:mah, PMID:19583219]
GO:0070819
name: menaquinone-dependent protoporphyrinogen oxidase activity
def: "Catalysis of the reaction: protoporphyrinogen IX + menaquinone = protoporphyrin IX + reduced menaquinone." [GOC:mah, PMID:19583219]
GO:0070835
name: chromium ion transmembrane transporter activity
def: "Catalysis of the transfer of chromium (Cr) ions from one side of a membrane to the other." [GOC:mah, GOC:yaf]
GO:0070840
name: dynein complex binding
def: "Interacting selectively and non-covalently with a dynein complex, a protein complex that contains two or three dynein heavy chains and several light chains, and has microtubule motor activity." [GOC:BHF, GOC:mah]
GO:0070851
name: growth factor receptor binding
def: "Interacting selectively and non-covalently with a growth factor receptor." [GOC:mah, GOC:vw]
GO:0070853
name: myosin VI binding
def: "Interacting selectively and non-covalently with a class VI myosin. The myosin VI heavy chain has a single IQ motif in the neck and a tail region with a coiled coil domain followed by a unique globular domain, a unique insertion that enables myosin VI to move towards the pointed or minus end of actin filaments." [GOC:mah, http://www.mrc-lmb.cam.ac.uk/myosin/Review/Reviewframeset.html, PMID:15473855]
GO:0070854
name: myosin VI heavy chain binding
def: "Interacting selectively and non-covalently with a heavy chain of a myosin VI complex." [GOC:sart]
GO:0070855
name: myosin VI head/neck binding
def: "Interacting selectively and non-covalently with the head/neck region of a myosin VI heavy chain." [GOC:sart]
GO:0070856
name: myosin VI light chain binding
def: "Interacting selectively and non-covalently with a light chain of a myosin VI complex." [GOC:sart]
GO:0070866
name: sterol-dependent protein binding
def: "Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules) in the presence of sterols." [GOC:ecd]
GO:0070878
name: primary miRNA binding
def: "Interacting selectively and non-covalently with a primary microRNA (pri-miRNA) transcript, an RNA molecule that is processed into a short hairpin-shaped structure called a pre-miRNA and finally into a functional miRNA. Both double-stranded and single-stranded regions of a pri-miRNA are required for binding." [GOC:sl, PMID:15531877, PMID:15574589]
GO:0070883
name: pre-miRNA binding
def: "Interacting selectively and non-covalently with a precursor microRNA (pre-miRNA) transcript, a stem-loop-containing precursor of microRNA." [PMID:18951094]
GO:0070888
name: E-box binding
def: "Interacting selectively and non-covalently with an E-box, a DNA motif with the consensus sequence CANNTG that is found in the promoters of a wide array of genes expressed in neurons, muscle and other tissues." [GOC:BHF, GOC:vk, PMID:11812799]
GO:0070890
name: sodium-dependent L-ascorbate transmembrane transporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: L-ascorbate(out) + Na+(out) = L-ascorbate(in) + Na+(in)." [GOC:mah, GOC:yaf, PMID:18094143]
GO:0070891
name: lipoteichoic acid binding
def: "Interacting selectively and non-covalently with lipoteichoic acid." [GOC:add, PMID:14665680]
GO:0070892
name: lipoteichoic acid receptor activity
def: "Combining with lipoteichoic acid and transmitting the signal to initiate a change in cell activity." [GOC:add, PMID:14665680]
GO:0070905
name: serine binding
def: "Interacting selectively and non-covalently with 2-amino-3-hydroxypropanoic acid." [CHEBI:17822, GOC:rph]
GO:0070906
name: aspartate:alanine antiporter activity
def: "Catalysis of the reaction: aspartate(out) + alanine(in) = aspartate(in) + alanine(out)." [GOC:dh]
GO:0070907
name: histidine:histamine antiporter activity
def: "Catalysis of the reaction: histidine(out) + histamine(in) = histidine(in) + histamine(out)." [GOC:dh]
GO:0070908
name: tyrosine:tyramine antiporter activity
def: "Catalysis of the reaction: tyrosine(out) + tyramine(in) = tyrosine(in) + tyramine(out)." [GOC:dh]
GO:0070909
name: glutamate:gamma-aminobutyric acid antiporter activity
def: "Catalysis of the reaction: tyrosine(out) + tyramine(in) = tyrosine(in) + tyramine(out)." [GOC:dh]
GO:0070915
name: lysophosphatidic acid receptor activity
def: "Combining with the phospholipid derivative lysophosphatidic acid, and transmitting the signal across the membrane by activating an associated G-protein." [CHEBI:52288, GOC:bf, GOC:mah, PMID:15755723]
GO:0070917
name: inositol phosphoceramide synthase regulator activity
def: "Modulates the activity of inositol phosphoceramide synthase." [GOC:mah]
GO:0070967
name: coenzyme F420 binding
def: "Interacting selectively and non-covalently with F420, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [CHEBI:16848, GOC:dh]
GO:0070968
name: pyrroloquinoline quinone binding
def: "Interacting selectively and non-covalently with pyrroloquinoline quinone, PQQ, the coenzyme or the prosthetic group of certain alcohol dehydrogenases and glucose dehydrogenases." [CHEBI:18315, GOC:dh]
GO:0070974
name: POU domain binding
def: "Interacting selectively and non-covalently with a POU domain of a protein. The POU domain is a bipartite DNA binding domain composed of two subunits separated by a non-conserved region of 15-55 amino acids; it is found in several eukaryotic transcription factors." [GOC:mah, GOC:yaf, InterPro:IPR000327]
GO:0070975
name: FHA domain binding
def: "Interacting selectively and non-covalently with a FHA domain of a protein. The FHA domain is a phosphopeptide recognition domain found in many regulatory proteins, and consists of approximately 80-100 amino acid residues folded into an 11-stranded beta sandwich." [GOC:mah, InterPro:IPR000253]
GO:0070976
name: TIR domain binding
def: "Interacting selectively and non-covalently with a Toll-Interleukin receptor (TIR) domain of a protein. The TIR domain is an intracellular 200 residue domain that is found in the Toll protein, the interleukin-1 receptor (IL-1R), and MyD88; it contains three highly-conserved regions, and mediates protein-protein interactions between the Toll-like receptors (TLRs) and signal-transduction components." [GOC:mah, InterPro:IPR000157]
GO:0070984
name: SET domain binding
def: "Interacting selectively and non-covalently with a SET domain of a protein. SET domains are named after three Drosophila proteins that contain this domain: Su(var), E(z) and trithorax. SET domains are associated with histone lysine methylation." [GOC:sart, Pfam:PF00856, PMID:12575990]
GO:0070990
name: snRNP binding
def: "Interacting selectively and non-covalently with any part of a small nuclear ribonucleoprotein particle." [GOC:BHF, GOC:mah, GOC:rl]
GO:0070991
name: medium-chain-acyl-CoA dehydrogenase activity
def: "Catalysis of the reaction: acyl-CoA + acceptor = 2,3-dehydroacyl-CoA + reduced acceptor, where the acyl group is a medium-chain fatty acid residue. A medium chain fatty acid is any fatty acid with a chain length of between C6 and C12." [CHEBI:59554, GOC:BHF, GOC:mah]
GO:0070996
name: type 1 melanocortin receptor binding
def: "Interacting selectively and non-covalently with a type 1 melanocortin receptor." [GOC:BHF, GOC:mah]
GO:0071074
name: eukaryotic initiation factor eIF2 binding
def: "Interacting selectively and non-covalently with eukaryotic initiation factor eIF2, a protein complex involved in the initiation of ribosome-mediated translation." [GOC:hjd]
GO:0071077
name: adenosine 3',5'-bisphosphate transmembrane transporter activity
def: "Catalysis of the transfer of adenosine 3',5'-bisphosphate from one side of a membrane to the other." [CHEBI:37240, GOC:mah, PubChem_Compound:159296]
GO:0071111
name: cyclic-guanylate-specific phosphodiesterase activity
def: "Catalysis of the reaction: cyclic di-3',5'-guanylate + H(2)O = 5'-phosphoguanylyl(3'->5')guanosine + H(+)." [EC:3.1.4.52, RHEA:24905]
GO:0071160
name: cyanophycin synthetase activity (L-aspartate-adding)
def: "Catalysis of the reaction: ATP + [L-Asp(4-L-Arg)]n + L-Asp = ADP + phosphate + [L-Asp(4-L-Arg)]n-L-Asp." [EC:6.3.2.29]
GO:0071161
name: cyanophycin synthetase activity (L-arginine-adding)
def: "Catalysis of the reaction: ATP + [L-Asp(4-L-Arg)]n-L-Asp + L-arginine = ADP + phosphate + [L-Asp(4-L-Arg)]n+1." [EC:6.3.2.30]
GO:0071164
name: RNA trimethylguanosine synthase activity
def: "Catalysis of two successive methyl transfer reactions from AdoMet to the N-2 atom of guanosine, thereby converting 7-methylguanosine in an RNA cap to 2,2,7 trimethylguanosine." [GOC:BHF, PMID:11983179, PMID:18775984]
GO:0071172
name: dihydromonapterin reductase activity
def: "Catalysis of the reaction: 7,8-dihydromonapterin + NADPH = tetrahydromonapterin + NADP+." [GOC:imk, PMID:19897652]
GO:0071207
name: histone pre-mRNA stem-loop binding
def: "Interacting selectively and non-covalently with a conserved stem-loop structure found in histone pre-mRNAs." [PMID:19470752]
GO:0071208
name: histone pre-mRNA DCP binding
def: "Interacting selectively and non-covalently with the downstream cleavage product (DCP) generated by histone pre-mRNA 3'-end processing." [PMID:19470752]
GO:0071209
name: U7 snRNA binding
def: "Interacting selectively and non-covalently with the U7 small nuclear RNA (U7 snRNA)." [GOC:mah, PMID:12975319]
GO:0071253
name: connexin binding
def: "Interacting selectively and non-covalently with a connexin, any of a group of related proteins that assemble to form gap junctions." [GOC:mah, PMID:19864490]
GO:0071424
name: rRNA (cytosine-N4-)-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + rRNA = S-adenosyl-L-homocysteine + rRNA containing N4-methylcytosine." [GOC:imk, PMID:19965768]
GO:0071443
name: tDNA binding
def: "Interacting selectively and non-covalently with DNA sequences encoding transfer RNA." [GOC:mah]
GO:0071518
name: autoinducer-2 kinase activity
def: "Catalysis of the reaction: 4,5-dihydroxy-pentane-2,3-dione + ATP = 5-phospho-4-hydroxy-pentane-2,3-dione (P-DPD) + ADP." [GOC:imk, PMID:17274596, PMID:20025244]
GO:0071522
name: ureidoglycine aminohydrolase activity
def: "Catalysis of the reaction: ureidoglycine + H2O = S-ureidoglycolate + NH3." [MetaCyc:URUR-RXN, PMID:19935661, PMID:20038185]
GO:0071532
name: ankyrin repeat binding
def: "Interacting selectively and non-covalently with an ankyrin repeat of a protein. Ankyrin repeats are tandemly repeated modules of about 33 amino acids; each repeat folds into a helix-loop-helix structure with a beta-hairpin/loop region projecting out from the helices at a 90-degree angle, and repeats stack to form an L-shaped structure." [GOC:mah, InterPro:IPR002110]
GO:0071535
name: RING-like zinc finger domain binding
def: "Interacting selectively and non-covalently with a RING-like zinc finger domain domain of a protein. The RING-like domain is a zinc finger domain that is related to the C3HC4 RING finger domain." [GOC:mah, InterPro:IPR014857]
GO:0071551
name: RIP homotypic interaction motif binding
def: "Interacting selectively and non-covalently with a RIP homotypic interaction motif (RHIM) of a protein. The RHIM is a 16-amino-acid motif found in some members, including RIP3, of a family of related kinases." [GOC:mah, PMID:11734559]
GO:0071553
name: G-protein coupled pyrimidinergic nucleotide receptor activity
def: "Combining with a pyrimidine nucleotide and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [GOC:sl, PMID:10736418, PMID:12369950, PMID:15796906]
GO:0071558
name: histone demethylase activity (H3-K27 specific)
def: "Catalysis of the removal of a methyl group from lysine at position 27 of the histone H3 protein." [GOC:sp, PMID:20622853]
GO:0071566
name: UFM1 activating enzyme activity
def: "Catalysis of the activation of the small ubiquitin-related modifier UFM1, through the formation of an ATP-dependent high-energy thiolester bond." [GOC:sp, PMID:20018847]
GO:0071567
name: UFM1 hydrolase activity
def: "Catalysis of the hydrolysis of UFM1, a small ubiquitin-related modifier, from previously modified substrates." [GOC:sp, PMID:20018847]
GO:0071568
name: UFM1 transferase activity
def: "Catalysis of the transfer of UFM1 from one protein to another via the reaction X-UFM1 + Y --> Y-UFM1 + X, where both X-UFM1 and Y-UFM1 are covalent linkages." [GOC:sp, PMID:20018847]
GO:0071576
name: tetrahydrodictyopterin binding
def: "Interacting selectively and non-covalently with tetrahydrodictyopterin, the pterin 2-amino-6-[(1R,2R)-1,2-dihydroxypropyl]-5,6,7,8-tetrahydropteridin-4(3H)-one." [CHEBI:52447, GOC:mah, GOC:vw]
GO:0071614
name: linoleic acid epoxygenase activity
def: "Catalysis of an NADPH- and oxygen-dependent reaction that converts linoleic acid to a cis-epoxyoctadecenoic acid." [GOC:BHF, PMID:11042099]
GO:0071617
name: lysophospholipid acyltransferase activity
def: "Catalysis of the transfer of acyl groups from an acyl-CoA to a lysophospholipid." [GOC:cjk]
GO:0071618
name: lysophosphatidylethanolamine acyltransferase activity
def: "Catalysis of the transfer of acyl groups from an acyl-CoA to lysophosphatidylethanolamine." [GOC:cjk]
GO:0071633
name: dihydroceramidase activity
def: "Catalysis of the reaction: a dihydroceramide + H2O = a fatty acid + dihydrosphingosine." [GOC:mah, PMID:10900202]
GO:0071667
name: DNA/RNA hybrid binding
def: "Interacting selectively and non-covalently with a RNA/DNA hybrid." [GOC:ecd]
GO:0071713
name: para-aminobenzoyl-glutamate hydrolase activity
def: "Catalysis of the reaction: para-aminobenzoyl-glutamate + H2O = para-aminobenzoate + L-glutamate." [GOC:imk, PMID:20190044]
GO:0071714
name: icosanoid transmembrane transporter activity
def: "Catalysis of the transfer of icosanoids from one side of the membrane to the other." [GOC:sl]
GO:0071723
name: lipopeptide binding
def: "Interacting selectively and non-covalently with a lipopeptide, any of a group of organic compounds comprising two or more amino acids linked by peptide bonds and containing a nonprotein group consisting of a lipid or lipids." [GOC:add, PMID:12077222, PMID:12524386, PMID:2757794]
GO:0071734
name: biotin-[pyruvate-carboxylase] ligase activity
def: "Catalysis of the reaction: ATP + biotin + apo-(pyruvate-carboxylase) = AMP + diphosphate + biotin-(pyruvate-carboxylase)." [GOC:mah, PMID:10551847]
GO:0071771
name: aldehyde decarbonylase activity
def: "Catalysis of the reaction: a C(n) aldehyde = C(n-1) alkane + CO." [GOC:kad, PMID:6593720, PMID:8718622]
GO:0071791
name: chemokine (C-C motif) ligand 5 binding
def: "Interacting selectively and non-covalently with chemokine (C-C motif) ligand 5." [GOC:add, GOC:amm]
GO:0071794
name: CAP-Gly domain binding
def: "Interacting selectively and non-covalently with a CAP-Gly domain of a protein. The CAP_Gly domain is a conserved, glycine-rich domain of about 42 residues found in some cytoskeleton-associated proteins, and features a novel protein fold containing three beta-sheets." [GOC:mah, InterPro:IPR000938]
GO:0071795
name: K11-linked polyubiquitin binding
def: "Interacting selectively and non-covalently and non-covalently with a polymer of ubiquitin formed by linkages between lysine residues at position 11 of the ubiquitin monomers." [GOC:sp, PMID:18775313]
GO:0071796
name: K6-linked polyubiquitin binding
def: "Interacting selectively and non-covalently and non-covalently with a polymer of ubiquitin formed by linkages between lysine residues at position 6 of the ubiquitin monomers." [GOC:sp, PMID:17525341, PMID:20351172]
GO:0071813
name: lipoprotein particle binding
def: "Interacting selectively and non-covalently with a lipoprotein particle. A lipoprotein particle, also known as a lipoprotein, is a clathrate complex consisting of a lipid enwrapped in a protein host without covalent binding in such a way that the complex has a hydrophilic outer surface consisting of all the protein and the polar ends of any phospholipids." [CHEBI:6495, GOC:BHF, GOC:mah]
GO:0071814
name: protein-lipid complex binding
def: "Interacting selectively and non-covalently with a protein-lipid complex, any macromolecular complex that contains both protein and lipid molecules." [GOC:BHF, GOC:mah]
GO:0071815
name: intermediate-density lipoprotein particle binding
def: "Interacting selectively and non-covalently with a intermediate-density lipoprotein particle, a triglyceride-rich lipoprotein particle that typically contains APOB100, APOE and APOCs and has a density of 1.006-1.019 g/ml and a diameter of between 25-30 nm." [GOC:BHF, GOC:mah]
GO:0071820
name: N-box binding
def: "Interacting selectively and non-covalently with an N-box, a DNA motif with the consensus sequence CACNAG that is found in the promoters of genes expressed preferentially at synapses." [GOC:yaf, PMID:11498047]
GO:0071837
name: HMG box domain binding
def: "Interacting selectively and non-covalently with an HMG box domain, a protein domain that consists of three helices in an irregular array. HMG-box domains are found in one or more copies in HMG-box proteins, which form a large, diverse family involved in the regulation of DNA-dependent processes such as transcription, replication, and strand repair, all of which require the bending and unwinding of chromatin." [GOC:yaf, InterPro:IPR009071, PMID:18445004]
GO:0071855
name: neuropeptide receptor binding
def: "Interacting selectively and non-covalently with a neuropeptide receptor." [GOC:kmv, GOC:mah]
GO:0071857
name: beta-endorphin receptor binding
def: "Interacting selectively and non-covalently with a beta-endorphin receptor." [GOC:kmv, GOC:mah]
GO:0071858
name: corazonin receptor binding
def: "Interacting selectively and non-covalently with a corazonin receptor." [GOC:kmv, GOC:mah]
GO:0071859
name: neuropeptide F receptor binding
def: "Interacting selectively and non-covalently with a neuropeptide F receptor." [GOC:kmv, GOC:mah]
GO:0071860
name: proctolin receptor binding
def: "Interacting selectively and non-covalently with a proctolin receptor." [GOC:kmv, GOC:mah]
GO:0071861
name: tachykinin receptor binding
def: "Interacting selectively and non-covalently with a tachykinin receptor." [GOC:kmv, GOC:mah]
GO:0071862
name: protein phosphatase type 1 activator activity
def: "Increases the activity of the enzyme protein phosphatase type 1." [GOC:vw]
GO:0071884
name: vitamin D receptor activator activity
def: "Interacting (directly or indirectly) with vitamin D receptors such that the proportion of receptors in the active form is increased." [GOC:BHF, GOC:vk]
GO:0071885
name: N-terminal protein N-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine (AdoMet) to the alpha-amino group of the N-terminal amino or imino acid residue of a protein substrate. For example, yeast Tae1p and mammalian family member METTL11A preferentially modify the N-terminal residue of substrates with the N-terminal sequence X-Pro-Lys, where X can be Pro, Ala, or Ser." [PMID:20481588]
GO:0071886
name: 1-(4-iodo-2,5-dimethoxyphenyl)propan-2-amine binding
def: "Interacting selectively and non-covalently with the amine 1-(4-iodo-2,5-dimethoxyphenyl)propan-2-amine, a serotonin receptor agonist that can act as a psychedelic drug." [CHEBI:100436, GOC:yaf, PMID:19057895]
GO:0071889
name: 14-3-3 protein binding
def: "Interacting selectively and non-covalently with a 14-3-3 protein. A 14-3-3 protein is any of a large family of approximately 30kDa acidic proteins which exist primarily as homo- and heterodimers within all eukaryotic cells, and have been implicated in the modulation of distinct biological processes by binding to specific phosphorylated sites on diverse target proteins, thereby forcing conformational changes or influencing interactions between their targets and other molecules. Each 14-3-3 protein sequence can be roughly divided into three sections: a divergent amino terminus, the conserved core region and a divergent carboxyl terminus. The conserved middle core region of the 14-3-3s encodes an amphipathic groove that forms the main functional domain, a cradle for interacting with client proteins." [GOC:cna, GOC:mah, InterPro:IPR000308, PMID:15167810, PMID:19575580, PR:000003237]
GO:0071890
name: bicarbonate binding
def: "Interacting selectively and non-covalently with bicarbonate (CHO3-) ions." [CHEBI:17544]
GO:0071906
name: CRD domain binding
def: "Interacting selectively and non-covalently with a CRD (context dependent regulatory) domain, a domain of about 130 residues that is the most divergent region among the LEF/TCF proteins." [GOC:yaf, PMID:19460168]
GO:0071913
name: citrate secondary active transmembrane transporter activity
def: "Catalysis of the transfer of citrate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:mah, GOC:mtg_transport, GOC:vw]
GO:0071916
name: dipeptide transmembrane transporter activity
def: "Enables the directed movement of a dipeptide across a membrane into, out of or within a cell, or between cells. A dipeptide is a combination of two amino acids linked together by a peptide (-CO-NH-) bond." [GOC:mah]
GO:0071917
name: triose-phosphate transmembrane transporter activity
def: "Catalysis of the transfer of a triose phosphate from one side of a membrane to the other." [GOC:mah, GOC:vw]
GO:0071933
name: Arp2/3 complex binding
def: "Interacting selectively and non-covalently with an Arp2/3 complex, a protein complex that contains two actin-related proteins, Arp2 and Arp3, and five novel proteins (ARPC1-5)." [GOC:mah]
GO:0071936
name: coreceptor activity involved in Wnt signaling pathway
def: "In cooperation with a primary Wnt receptor, initiating a change in cell activity through the Wnt signaling pathway." [GOC:BHF, GOC:mah]
GO:0071949
name: FAD binding
def: "Interacting selectively and non-covalently with the oxidized form, FAD, of flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [GOC:mah]
GO:0071950
name: FADH2 binding
def: "Interacting selectively and non-covalently with the reduced form, FADH2, of flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes." [GOC:mah]
GO:0071967
name: lipopolysaccharide core heptosyltransferase activity
def: "Catalysis of the reaction: glucosyl-heptosyl2-KDO2-lipid A-phosphate + ADP-L-glycero-beta-D-manno-heptose = glucosyl-heptosyl3-KDO2-lipid A-phosphate + ADP + H+." [MetaCyc:RXN0-5122]
GO:0071968
name: lipid A-core heptosyltransferase activity
def: "Catalysis of the reaction: galactosyl-glucosyl3-heptosyl3-KDO2-lipid A-bisphosphate + ADP-L-glycero-beta-D-manno-heptose = lipid A-core + ADP + H+." [MetaCyc:RXN0-5127]
GO:0071972
name: peptidoglycan L,D-transpeptidase activity
def: "Catalysis of the reaction: 2 a peptidoglycan dimer (tetrapeptide) + 3 H2O = a peptidoglycan tetramer with L,D cross-links (L-Lys-D-Asn-L-Lys) + di-trans,poly-cis-undecaprenyl diphosphate + 4 D-alanine." [MetaCyc:RXN-11349]
GO:0071987
name: WD40-repeat domain binding
def: "Interacting selectively and non-covalently with a WD40 repeat domain of a protein. The WD40 repeat is a short structural motif of approximately 40 amino acids, often terminating in a tryptophan-aspartic acid (W-D) dipeptide. Several of these repeats are combined to form a type of protein domain called the WD domain." [GOC:yaf, InterPro:IPR017986]
GO:0071991
name: phytochelatin transporter activity
def: "Enables the directed movement of a phytochelatin into, out of or within a cell, or between cells. Phytochelatins are a group of peptides that bind metals (Cd, Zn, Cu, Pb, Hg) in thiolate coordination complexes." [GOC:mah, ISBN:0198506732]
GO:0071992
name: phytochelatin transmembrane transporter activity
def: "Enables the directed movement of a phytochelatin across a membrane into, out of or within a cell, or between cells. Phytochelatins are a group of peptides that bind metals (Cd, Zn, Cu, Pb, Hg) in thiolate coordination complexes." [GOC:mah, ISBN:0198506732]
GO:0071997
name: glutathione S-conjugate-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O = ADP + phosphate, to directly drive the transport of glutathione S-conjugate across a membrane." [GOC:mah]
GO:0072317
name: glucan endo-1,3-beta-D-glucosidase activity involved in ascospore release from ascus
def: "Catalysis of the hydrolysis of 1,3-beta-D-glucosidic linkages in 1,3-beta-D-glucans that contributes to the release of ascospores from the ascus." [GOC:jl]
GO:0072320
name: volume-sensitive chloride channel activity
def: "Catalysis of the transmembrane transfer of a chloride ion by a volume-sensitive channel. A volume-sensitive channel is a channel that responds to changes in the volume of a cell." [GOC:mah]
GO:0072328
name: alkene binding
def: "Interacting selectively and non-covalently with an alkene, any acyclic branched or unbranched hydrocarbon having one carbon-carbon double bond and the general formula CnH2n." [CHEBI:32878, GOC:mah]
GO:0072341
name: modified amino acid binding
def: "Interacting selectively and non-covalently with a modified amino acid." [CHEBI:25359, GOC:mah]
GO:0072345
name: NAADP-sensitive calcium-release channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when nicotinic acid adenine dinucleotide phosphate (NAADP) has been bound by the channel complex or one of its constituent parts." [PMID:19387438, PMID:19557428]
GO:0072346
name: cADPR-sensitive calcium-release channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens when cyclic adenosine diphosphate ribose (cADPR) has been bound by the channel complex or one of its constituent parts." [PMID:11752598]
GO:0072349
name: modified amino acid transmembrane transporter activity
def: "Catalysis of the transfer of modified amino acids from one side of a membrane to the other." [CHEBI:25359, GOC:mah]
GO:0072354
name: histone kinase activity (H3-T3 specific)
def: "Catalysis of the transfer of a phosphate group to the threonine-3 residue of the N-terminal tail of histone H3." [GOC:mah]
GO:0072371
name: histone kinase activity (H2A-S121 specific)
def: "Catalysis of the transfer of a phosphate group to the serine-121 residue of the N-terminal tail of histone H2A." [GOC:mah, PMID:19965387]
GO:0072373
name: alpha-carotene epsilon hydroxylase activity
def: "Catalysis of the reaction: alpha-carotene + NADPH + O2 + H+ = alpha-cryptoxanthin + NADP+ + H2O." [GOC:mah, MetaCyc:CPD-7421, MetaCyc:CPD1F-118, MetaCyc:RXN-5962]
GO:0072374
name: carotene epsilon hydroxylase activity
def: "Catalysis of the reaction: a carotene + NADPH + O2 + H+ = a beta-ring hydroxylcarotene + NADP+ + H2O. Adds a hydroxyl group to the epsilon ring of the alpha-carotene." [GOC:mah, MetaCyc:CPD-7421, MetaCyc:CPD1F-118, MetaCyc:RXN-5962]
GO:0072496
name: Pup transferase activity
def: "Catalysis of the transfer of Pup from one protein to another via the reaction X-Pup + Y --> Y-Pup + X, where both X-Pup and Y-Pup are covalent linkages." [GOC:sp]
GO:0072509
name: divalent inorganic cation transmembrane transporter activity
def: "Catalysis of the transfer of inorganic cations with a valency of two from one side of the membrane to the other. Inorganic cations are atoms or small molecules with a positive charge that do not contain carbon in covalent linkage." [GOC:mah]
GO:0072510
name: trivalent inorganic cation transmembrane transporter activity
def: "Catalysis of the transfer of inorganic cations with a valency of three from one side of the membrane to the other. Inorganic cations are atoms or small molecules with a positive charge that do not contain carbon in covalent linkage." [GOC:mah]
GO:0072518
name: Rho-dependent protein serine/threonine kinase activity
def: "Catalysis of the reaction: ATP + a protein = ADP + a phosphoprotein. This reaction requires binding of the GTPase Rho." [GOC:ecd, PMID:12778124, PMID:20230755]
GO:0072532
name: tri-(feruloyl or hydroxyferuloyl) spermidine meta-hydroxylase activity
def: "Catalysis of the meta-hydroxylation of any of the three phenolic rings on triferuloyl spermidine or any of its mono- or di-(hydroxyferuloyl)-spermidine derivatives." [GOC:kad, PMID:19779199]
GO:0072533
name: tri-(coumaroyl or caffeoyl) spermidine meta-hydroxylase activity
def: "Catalysis of the meta-hydroxylation of any of the three phenolic rings on tricoumaroyl spermidine or any of its mono- or dicaffeoyl spermidine derivatives." [GOC:kad, PMID:19779199]
GO:0072541
name: peroxynitrite reductase activity
def: "Catalysis of the reaction: 2 R-SH + ONOO- = R-SS-R + NO2- + H2O." [GOC:rs, PMID:11001062]
GO:0072542
name: protein phosphatase activator activity
def: "Increases the activity of a protein phosphatase, an enzyme which catalyzes of the removal of a phosphate group from a protein substrate molecule." [GOC:mah]
GO:0072543
name: protein phosphatase type 2A activator activity
def: "Increases the activity of the enzyme protein phosphatase type 2A." [GOC:vw]
GO:0072544
name: L-DOPA binding
def: "Interacting selectively and non-covalently with L-DOPA, the modified amino acid (2S)-2-amino-3-(3,4-dihydroxyphenyl)propanoic acid." [CHEBI:15765, GOC:mah, GOC:vw]
GO:0072545
name: tyrosine binding
def: "Interacting selectively and non-covalently with2-amino-3-(4-hydroxyphenyl)propanoic acid." [CHEBI:18186, GOC:mah]
GO:0072547
name: tricoumaroylspermidine meta-hydroxylase activity
def: "Catalysis of the reaction: tricoumaroyl spermidine + NADPH + O2 = dicoumaroyl monocaffeoyl spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072548
name: dicoumaroyl monocaffeoyl spermidine meta-hydroxylase activity
def: "Catalysis of the reaction: dicoumaroyl monocaffeoyl spermidine + NADPH + O2 = monocoumaroyl dicaffeoyl spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072549
name: monocoumaroyl dicaffeoyl spermidine meta-hydroxylase activity
def: "Catalysis of the reaction: monocoumaroyl dicaffeoyl spermidine + NADPH + O2 = tricaffeoyl spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072550
name: triferuloylspermidine meta-hydroxylase activity
def: "Catalysis of the reaction: triferuloyl spermidine + NADPH + O2 = diferuloyl mono-(hydroxyferuloyl) spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072551
name: diferuloyl mono-(hydroxyferuloyl) spermidine meta-hydroxylase activity
def: "Catalysis of the reaction: diferuloyl mono-(hydroxyferuloyl) spermidine + NADPH + O2 = monoferuloyl di-(hydroxyferuloyl) spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072552
name: monoferuloyl di-(hydroxyferuloyl) spermidine meta-hydroxylase activity
def: "Catalysis of the reaction: monoferuloyl di-(hydroxyferuloyl) spermidine + NADPH + O2 = tri-(hydroxyferuloyl) spermidine + NADP+ + H2O." [GOC:kad, PMID:19779199]
GO:0072555
name: 17-beta-ketosteroid reductase activity
def: "Catalysis of the reaction: a 17-beta-ketosteroid + NADPH + H+ = a 17-beta-hydroxysteroid + NADP+." [GOC:kad, PMID:17074428]
GO:0072570
name: ADP-D-ribose binding
def: "Interacting selectively and non-covalently with ADP-D-ribose, an ADP-aldose having ribose as the aldose fragment." [CHEBI:16864, GOC:mah, GOC:sart, PMID:20088964]
GO:0072571
name: mono-ADP-D-ribose binding
def: "Interacting selectively and non-covalently with monomeric ADP-D-ribose, an ADP-aldose having ribose as the aldose fragment." [CHEBI:16864, GOC:mah, GOC:sart, PMID:20088964]
GO:0072572
name: poly-ADP-D-ribose binding
def: "Interacting selectively and non-covalently with polymeric ADP-D-ribose, a polymer that is composed of poly-ADP-D-ribose units linked through 1,2-glycosidic bonds at the ribose ring." [CHEBI:61151, GOC:mah, GOC:sart, PMID:20088964]
GO:0072582
name: 17-beta-hydroxysteroid dehydrogenase (NADP+) activity
def: "Catalysis of the reaction: a 17-beta-hydroxysteroid + NADP+ = a 17-oxosteroid + NADPH + H+." [GOC:kad, PMID:17074428]
GO:0072585
name: xanthosine nucleotidase activity
def: "Catalysis of the reaction: xanthosine + H2O = D-ribose + xanthine." [GOC:kad, MetaCyc:RXN0-363, PMID:21235647]
GO:0072586
name: DNA topoisomerase (ATP-hydrolyzing) regulator activity
def: "Modulates the activity of ATP-hydrolyzing DNA topoisomerase. DNA topoisomerase (ATP-hydrolyzing) regulator activity catalyzes a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:mah]
GO:0072587
name: DNA topoisomerase (ATP-hydrolyzing) activator activity
def: "Increases the activity of ATP-hydrolyzing DNA topoisomerase. DNA topoisomerase (ATP-hydrolyzing) regulator activity catalyzes a DNA topological transformation by transiently cleaving a pair of complementary DNA strands to form a gate through which a second double-stranded DNA segment is passed, after which the severed strands in the first DNA segment are rejoined; product release is coupled to ATP binding and hydrolysis; changes the linking number in multiples of 2." [GOC:mah]
GO:0072590
name: N-acetyl-L-aspartate-L-glutamate ligase activity
def: "Catalysis of the reaction: ATP + N-acetyl-L-aspartate + L-glutamate = ADP + phosphate + N-acetylaspartyl-glutamate." [PMID:20643647, PMID:20657015]
GO:0072591
name: citrate-L-glutamate ligase activity
def: "Catalysis of the reaction: ATP + citrate + L-glutamate = ADP + phosphate + beta-citryl-L-glutamate." [PMID:20657015]
GO:0080002
name: UDP-glucose:4-aminobenzoate acylglucosyltransferase activity
def: "Catalysis of the reaction: 4-aminobenzoate + UDP-glucose = p-aminobenzoate-beta-D-glucopyranosyl ester + UDP." [EC:2.4.1.-, PMID:18385129]
GO:0080004
name: thalian-diol desaturase activity
def: "Catalysis of the reaction: a thalian-diol = a desaturated thalian-diol. This reaction is the introduction of a double bond to a thalian-diol molecule at carbon 15." [PMID:18356490]
GO:0080007
name: S-nitrosoglutathione reductase activity
def: "Catalysis of the reaction: glutathione N-hydroxysulfenamide + NADH + H+ = S-nitrosoglutathione + NAD+." [MetaCyc:RXN-10742, PMID:11260719]
GO:0080011
name: baruol synthase activity
def: "Catalysis of the reaction: (S)-2,3-epoxysqualene = baruol. Baruol is also known as D:B-Friedo-Baccharan-5,21-dien-3-ol." [MetaCyc:RXN-9685, PMID:17705488]
GO:0080012
name: trihydroxyferuloyl spermidine O-methyltransferase activity
def: "Catalysis of the reaction: trihydroxyferuloyl spermidine + S-adenosyl-L-methionine = dihydroxyferuloyl-sinapoyl spermidine + S-adenosyl-L-homocysteine + H+." [PMID:18557837]
GO:0080013
name: (E,E)-geranyllinalool synthase activity
def: "Catalysis of the reaction: all-trans-geranyl-geranyl diphosphate + H2O = (E,E)-geranyllinalool + diphosphate." [MetaCyc:RXN-10441, PMID:18398052]
GO:0080014
name: thalianol hydroxylase activity
def: "Catalysis of the reaction: a thalianol = a thalian-diol. This reaction is the addition of a hydroxyl group to thalianol ((13R,14R,17E)-podioda-8,17,21-trien-3beta-ol) to create a thalian-diol ((13R,14R,17E)-podioda-8,17,21-trien-3beta,X-diol), where the hydroxyl group may be attached at one of several different available carbons in ring B or C of thalianol, indicated by the X." [MetaCyc:RXN-9631, PMID:17474751, PMID:18356490]
GO:0080015
name: sabinene synthase activity
def: "Catalysis of the reaction: geranyl diphosphate = sabinene + diphosphate." [MetaCyc:RXN-5103, PMID:12566586, PMID:9747540]
GO:0080016
name: (-)-E-beta-caryophyllene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = (-)-E-beta-caryophyllene + diphosphate." [MetaCyc:RXN-8414, PMID:12566586, PMID:9442047]
GO:0080017
name: alpha-humulene synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate = alpha-humulene + diphosphate." [PMID:12566586, PMID:9442047]
GO:0080018
name: anthocyanin 5-O-glucosyltransferase activity
def: "Catalysis of the reaction: an anthocyanin + UDP-D-glucose = an anthocyanin-5-O-glucoside + UDP." [PMID:15807784]
GO:0080019
name: fatty-acyl-CoA reductase (alcohol-forming) activity
def: "Catalysis of the reaction: a very long chain fatty acyl-CoA + NADPH + H+ = a very long chain primary alcohol + NADP+ + CoA." [PMID:16980563]
GO:0080023
name: 3R-hydroxyacyl-CoA dehydratase activity
def: "Catalysis of the reaction: 3R-hydroxyacyl-CoA = 2E-enoyl-CoA + H2O." [PMID:16982622]
GO:0080025
name: phosphatidylinositol-3,5-bisphosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol-3,5-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' and 5' positions." [GOC:bf, PMID:18397324]
GO:0080030
name: methyl indole-3-acetate esterase activity
def: "Catalysis of the reaction: methyl indole-3-acetate + H2O = indole-3-acetate + methanol + H+." [MetaCyc:RXN-10711, PMID:18467465]
GO:0080031
name: methyl salicylate esterase activity
def: "Catalysis of the reaction: methyl salicylate + H2O = salicylic acid + methanol + H+." [MetaCyc:RXNQT-4366, PMID:18467465, PMID:18643994]
GO:0080032
name: methyl jasmonate esterase activity
def: "Catalysis of the reaction: a methyl jasmonate + H2O = a jasmonic acid + methanol." [PMID:15233793, PMID:18467465]
GO:0080039
name: xyloglucan endotransglucosylase activity
def: "Catalysis of the hydrolysis of a beta-(1,4) bond in the backbone of a xyloglucan and transfers the xyloglucanyl segment on to O-4 of the non-reducing terminal glucose residue of a xyloglucan or an oligosaccharide of xyloglucan." [EC:2.4.1.207, GOC:ask, PMID:1400418, PMID:1554366]
GO:0080041
name: ADP-ribose pyrophosphohydrolase activity
def: "Catalysis of the reaction: ADP-ribose + H2O = AMP + ribose-1-phosphate." [GOC:tb]
GO:0080042
name: ADP-glucose pyrophosphohydrolase activity
def: "Catalysis of the reaction: ADP-glucose + H2O = AMP + glucose-1-phosphate." [GOC:tb]
GO:0080043
name: quercetin 3-O-glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group from UDP-glucose to the 3-hydroxy group of a quercetin molecule." [PMID:15352060]
GO:0080044
name: quercetin 7-O-glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group from UDP-glucose to the 7-hydroxy group of a quercetin molecule." [PMID:15352060]
GO:0080045
name: quercetin 3'-O-glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group from UDP-glucose to the 3'-hydroxy group of a quercetin molecule." [PMID:15352060]
GO:0080046
name: quercetin 4'-O-glucosyltransferase activity
def: "Catalysis of the transfer of a glucosyl group from UDP-glucose to the 4'-hydroxy group of a quercetin molecule." [PMID:15352060]
GO:0080047
name: GDP-L-galactose phosphorylase activity
def: "Catalysis of the reaction: GDP-L-galactose + phosphate = L-galactose-1-P + GDP." [PMID:18463094]
GO:0080048
name: GDP-D-glucose phosphorylase activity
def: "Catalysis of the reaction: GDP-D-glucose + phosphate = D-glucose-1-P + GDP." [PMID:18463094]
GO:0080049
name: L-gulono-1,4-lactone dehydrogenase activity
def: "Catalysis of the reaction: L-gulono-1,4-lactone + 2 ferricytochrome c = L-ascorbate + 2 ferrocytochrome c." [PMID:18190525]
GO:0080054
name: low-affinity nitrate transmembrane transporter activity
def: "Catalysis of the transfer of nitrate ions (NO3-) from one side of a membrane to the other. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [PMID:19050168]
GO:0080059
name: flavonol 3-O-arabinosyltransferase activity
def: "Catalysis of the reaction: UDP-arabinose + a flavonol = UDP + a flavonol 3-O-D-arabinoside." [PMID:18757557]
GO:0080061
name: indole-3-acetonitrile nitrilase activity
def: "Catalysis of the reaction: indole-3-acetonitrile + 2 H2O = indole-3-acetic acid + NH3." [EC:3.5.5.1, MetaCyc:RXN-1404]
GO:0080062
name: cytokinin 9-beta-glucosyltransferase activity
def: "Catalysis of the reaction: 6-alkylaminopurine + UDP-D-glucose = 6-alkylamino-9-beta-D-glucosylpurine + H+ + UDP. This reaction is an N-glucosylation event." [KEGG:R08369, PMID:15342621]
GO:0080066
name: 3-methylthiopropyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 3-methylthiopropyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = 3-methylthiopropyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080067
name: 4-methylthiobutyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 4-methylthiobutyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = 4-methylthiobutyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080068
name: 5-methylthiopentyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 5-methylthiopentyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = 5-methylthiopentyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080069
name: 7-methylthioheptyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 7-methylthioheptyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = 7-methylthioheptyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080070
name: 8-methylthiooctyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: 8-methylthiooctyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = 8-methylthiooctyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080071
name: indol-3-yl-methyl-desulfoglucosinolate sulfotransferase activity
def: "Catalysis of the reaction: indol-3-yl-methyl-desulfoglucosinolate + 3'-phosphoadenosine 5'-phosphosulfate = indol-3-yl-methyl-glucosinolate + adenosine 3',5'-bisphosphate." [PMID:19077143]
GO:0080072
name: spermidine:sinapoyl CoA N-acyltransferase activity
def: "Catalysis of the transfer of a sinapoyl group to a nitrogen atom on the spermidine molecule." [PMID:19077165]
GO:0080073
name: spermidine:coumaroyl CoA N-acyltransferase activity
def: "Catalysis of the transfer of a coumaroyl group to a nitrogen atom on the spermidine molecule." [PMID:19077165]
GO:0080074
name: spermidine:caffeoyl CoA N-acyltransferase activity
def: "Catalysis of the transfer of a caffeoyl group to a nitrogen atom on the spermidine molecule." [PMID:19077165]
GO:0080075
name: spermidine:feruloyl CoA N-acyltransferase activity
def: "Catalysis of the transfer of a feruloyl group to a nitrogen atom on the spermidine molecule." [PMID:19077165]
GO:0080076
name: caffeoyl CoA:S-adenosyl-L-methionine O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a caffeoyl CoA molecule." [PMID:19077165]
GO:0080077
name: trihydroxyferuloyl spermidine:S-adenosyl-L-methionine O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a trihydroxyferuloyl spermidine molecule." [PMID:19077165]
GO:0080078
name: tricaffeoyl spermidine:S-adenosyl-L-methionine O-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a tricaffeoyl spermidine molecule." [PMID:19077165]
GO:0080079
name: cellobiose glucosidase activity
def: "Catalysis of the reaction: cellobiose + H2O = 2 D-glucose." [PMID:15604686]
GO:0080081
name: 4-methylumbelliferyl-beta-D-glucopyranoside beta-glucosidase activity
def: "Catalysis of the hydrolysis of glucosidic link in 4-methylumbelliferyl-beta-D-glucopyranoside." [PMID:15604686]
GO:0080082
name: esculin beta-glucosidase activity
def: "Catalysis of the hydrolysis of glucosidic link in esculin." [PMID:15604686]
GO:0080083
name: beta-gentiobiose beta-glucosidase activity
def: "Catalysis of the hydrolysis of glucosidic link in beta-gentiobiose." [PMID:15604686]
GO:0080084
name: 5S rDNA binding
def: "Interacting selectively and non-covalently with the 5S rDNA sequence encoding ribosomal 5S rRNA, which is individually transcribed by RNA polymerase III, rather than by RNA polymerase I, in species where it exists." [PMID:12711688]
GO:0080089
name: sinapoyl spermidine:sinapoyl CoA N-acyltransferase activity
def: "Catalysis of the transfer of a sinapoyl group to a nitrogen atom on the spermidine molecule." [PMID:19168716]
GO:0080095
name: phosphatidylethanolamine-sterol O-acyltransferase activity
def: "Catalysis of the reaction: a phosphatidylethanolamine + a sterol = a sterol ester + a lysophosphatidylethanolamine." [PMID:16020547]
GO:0080096
name: phosphatidate-sterol O-acyltransferase activity
def: "Catalysis of the reaction: a phosphatidate + a sterol = a sterol ester + a lysophosphatidate." [PMID:16020547]
GO:0080097
name: L-tryptophan:pyruvate aminotransferase activity
def: "Catalysis of the reaction: L-tryptophan + pyruvate = 3-(indol-3-yl)pyruvate + L-alanine." [MetaCyc:RXN-10139]
GO:0080098
name: L-tyrosine:pyruvate aminotransferase activity
def: "Catalysis of the reaction: L-tyrosine + pyruvate = (4-hydroxyphenyl)pyruvate + L-alanine." [MetaCyc:RXN3O-4157]
GO:0080099
name: L-methionine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-methionine + 2-oxoglutarate = 4-methylthio-2-oxobutyrate + L-glutamate." [PMID:18394996]
GO:0080100
name: L-glutamine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-glutamine + 2-oxoglutarate = 2-oxoglutaramate + L-glutamate." [PMID:18394996]
GO:0080101
name: phosphatidyl-N-dimethylethanolamine N-methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + phosphatidyl-N-dimethylethanolamine = S-adenosyl-L-homocysteine + phosphatidylcholine." [EC:2.1.1.71, PMID:19366698]
GO:0080102
name: 3-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 3-methylthiopropyl-glucosinolate = 3-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080103
name: 4-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 4-methylthiopropyl-glucosinolate = 4-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080104
name: 5-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 5-methylthiopropyl-glucosinolate = 5-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080105
name: 6-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 6-methylthiopropyl-glucosinolate = 6-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080106
name: 7-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 7-methylthiopropyl-glucosinolate = 7-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080107
name: 8-methylthiopropyl glucosinolate S-oxygenase activity
def: "Catalysis of the reaction: 8-methylthiopropyl-glucosinolate = 8-methylsulfinylpropyl-glucosinolate." [PMID:18799661]
GO:0080108
name: S-alkylthiohydroximate lyase activity
def: "Catalysis of the conversion of a S-alkylthiohydroximate to a thiohydroximate." [PMID:14871316]
GO:0080109
name: indole-3-acetonitrile nitrile hydratase activity
def: "Catalysis of the reaction: indole-3-acetonitrile + H2O = indole-3-acetamide." [EC:4.2.1.84, MetaCyc:RXN-7567, PMID:11607511, PMID:12430025]
GO:0080115
name: myosin XI tail binding
def: "Interacting selectively and non-covalently with the tail region of a myosin XI heavy chain." [PMID:18703495]
GO:0080116
name: glucuronoxylan glucuronosyltransferase activity
def: "Catalysis of the transfer of glucuronate to the xylan backbone of glucuronoxylan molecule." [PMID:18980649]
GO:0080118
name: brassinosteroid sulfotransferase activity
def: "Catalysis of the reaction: a brassinosteroid + 3'-phosphoadenosine-5'-phosphosulfate = sulfated brassinosteroid + adenosine-3',5'-diphosphate. This reaction is the transfer of a sulfate group to the hydroxyl group of a brassinosteroid acceptor, producing the sulfated brassinosteroid derivative." [CHEBI:22921, PMID:10409637, PMID:17039368]
GO:0080122
name: AMP transmembrane transporter activity
def: "Catalysis of the transfer of AMP, adenosine monophosphate, from one side of a membrane to the other." [PMID:18923018]
GO:0080123
name: jasmonate-amino synthetase activity
def: "Catalysis of the reaction: jasmonate + an amino acid = an amide-linked jasmonyl-amino acid conjugate. The substrates of this reaction include non-standard amino acids, such as ACC (1-aminocyclopropane-1-carboxylate)." [PMID:15258265, PMID:17291501]
GO:0080124
name: pheophytinase activity
def: "Catalysis of the reaction: pheophytin + H2O = phytol + pheophorbide." [PMID:19304936]
GO:0080130
name: L-phenylalanine:2-oxoglutarate aminotransferase activity
def: "Catalysis of the reaction: L-phenylalanine + 2-oxoglutarate = phenylpyruvate + L-glutamate." [GOC:pmn_curators, PMID:18394996]
GO:0080131
name: hydroxyjasmonate sulfotransferase activity
def: "Catalysis of the reaction: a hydroxyjasmonate + 3'-phosphoadenosine-5'-phosphosulfate = a hydroxyjasmonate sulfate + adenosine-3',5'-diphosphate." [EC:2.8.2.-, GOC:pmn_curators, MetaCyc:RXN-10451, MetaCyc:RXN-10453, PMID:12637544]
GO:0080132
name: fatty acid alpha-hydroxylase activity
def: "Catalysis of the conversion of a fatty acid to an alpha-hydroxylated fatty acid. A hydroxyl group is added to the second carbon, counted from the carboxyl end, of a fatty acid chain." [PMID:19054355]
GO:0080133
name: midchain alkane hydroxylase activity
def: "Catalysis of the conversion of an alkane to a secondary alcohol." [PMID:17905869]
GO:0080138
name: borate uptake transmembrane transporter activity
def: "Catalysis of the transfer of borate from the outside of a cell to the inside of the cell across a membrane." [PMID:18603465]
GO:0080139
name: borate efflux transmembrane transporter activity
def: "Catalysis of the transfer of borate from the inside of the cell to the outside of the cell across a membrane." [PMID:18603465]
GO:0080146
name: L-cysteine desulfhydrase activity
def: "Catalysis of the reaction: L-cysteine + H2O = ammonia + pyruvate + hydrogen sulfide + H+." [MetaCyc:LCYSDESULF-RXN, PMID:19955263]
GO:0080150
name: S-adenosyl-L-methionine:benzoic acid carboxyl methyl transferase activity
def: "Catalysis of the reaction: benzoate + S-adenosyl-L-methionine = methylbenzoate + S-adenosyl-L-homocysteine." [MetaCyc:RXN-6722, PMID:10852939]
GO:0080161
name: auxin transmembrane transporter activity
def: "Catalysis of the transfer of auxins from one side of the membrane to the other. Auxins are plant hormones that regulate aspects of plant growth." [PMID:19506555]
GO:0080176
name: xyloglucan 1,6-alpha-xylosidase activity
def: "Catalysis of the hydrolysis of xyloglucan side chains so as to remove unsubstituted D-xylose residues attached to the glucose located at the non-reducing terminus." [PMID:20801759]
GO:0086006
name: voltage-gated sodium channel activity involved in cardiac muscle cell action potential
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of a cardiac muscle cell contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086007
name: voltage-gated calcium channel activity involved in cardiac muscle cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel across the plasma membrane of a cardiac muscle cell that contributes to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086008
name: voltage-gated potassium channel activity involved in cardiac muscle cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of a cardiac muscle cell contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086020
name: gap junction channel activity involved in SA node cell-atrial cardiac muscle cell electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from an SA node cell to an atrial cardiomyocyte. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086037
name: sodium:potassium-exchanging ATPase activity involved in regulation of cardiac muscle cell membrane potential
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + Na+(in) + K+(out) = ADP + phosphate + Na+(out) + K+(in), that contributes to regulating the membrane potential of a cardiac muscle cell." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086038
name: calcium:sodium antiporter activity involved in regulation of cardiac muscle cell membrane potential
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Ca2+(in) + Na+(out) = Ca2+(out) + Na+(in), which contributes to regulating the membrane potential of a cardiac muscle cell." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086039
name: calcium-transporting ATPase activity involved in regulation of cardiac muscle cell membrane potential
def: "Catalysis of the transfer of a solute or solutes from one side of a cardiac muscle cell plasma membrane to the other according to the reaction: ATP + H2O + Ca2+(cis) = ADP + phosphate + Ca2+(trans). The transfer contributes to the regulation of the plasma membrane potential." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086040
name: sodium:proton antiporter activity involved in regulation of cardiac muscle cell membrane potential
def: "Catalysis of the transfer of a solute or solutes from one side of a cardiac muscle cell membrane to the other according to the reaction: Na+(out) + H+(in) = Na+(in) + H+(out). This transfer contributes to the regulation of the cardiac muscle cell plasma membrane potential." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086041
name: voltage-gated potassium channel activity involved in SA node cell action potential depolarization
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of an SA node cardiac muscle cell contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086056
name: voltage-gated calcium channel activity involved in AV node cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel across the plasma membrane of an AV node cardiac muscle cell that contributes to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086057
name: voltage-gated calcium channel activity involved in bundle of His cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel across the plasma membrane of a bundle of His cardiac muscle cell that contributes to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086058
name: voltage-gated calcium channel activity involved in Purkinje myocyte cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel across the plasma membrane of an Purkinje myocyte cell that contributes to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086059
name: voltage-gated calcium channel activity involved SA node cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a voltage-gated channel across the plasma membrane of an SA node cardiac muscle cell that contributes to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086060
name: voltage-gated sodium channel activity involved in AV node cell action potential
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of an AV node cardiac muscle cell contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086061
name: voltage-gated sodium channel activity involved in bundle of His cell action potential
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of a bundle of His cardiac muscle cell contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086062
name: voltage-gated sodium channel activity involved in Purkinje myocyte action potential
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of a Purkinje myocyte contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086063
name: voltage-gated sodium channel activity involved in SA node cell action potential
def: "Catalysis of the transmembrane transfer of a sodium ion by a voltage-gated channel through the plasma membrane of an SA node cardiac muscle cell contributing to the depolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086075
name: gap junction channel activity involved in cardiac conduction electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from one cardiomyocyte to an adjacent cardiomyocyte. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086076
name: gap junction channel activity involved in atrial cardiac muscle cell-AV node cell electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from an atrial cardiomyocyte to an AV node cell. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086077
name: gap junction channel activity involved in AV node cell-bundle of His cell electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from an AV node cell to a bundle of His cell. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086078
name: gap junction channel activity involved in bundle of His cell-Purkinje myocyte electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from a bundle of His cell to a Purkinje myocyte. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086079
name: gap junction channel activity involved in Purkinje myocyte-ventricular cardiac muscle cell electrical coupling
def: "A wide pore channel activity that enables a direct cytoplasmic connection from a Purkinje myocyte to a ventricular cardiac muscle cell. The gap junction passes electrical signals between the cells contributing to cardiac conduction." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086080
name: protein binding involved in heterotypic cell-cell adhesion
def: "Interacting selectively and non-covalently with any protein or protein complex contributing to the adhesion of two different types of cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086081
name: cell adhesive protein binding involved in atrial cardiac muscle cell-AV node cell communication
def: "Interacting selectively and non-covalently with any protein or protein complex that results in the connection of an atrial cardiomyocyte with an AV node cell and contributes to the communication between the two cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086082
name: cell adhesive protein binding involved in AV node cell-bundle of His cell communication
def: "Interacting selectively and non-covalently with any protein or protein complex that results in the connection of an AV node cell with a bundle of His cell and contributes to the communication between the two cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086083
name: cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication
def: "Interacting selectively and non-covalently with any protein or protein complex that results in the connection of a bundle of His cell with a Purkinje myocyte and contributes to the communication between the two cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086084
name: cell adhesive protein binding involved in Purkinje myocyte-ventricular cardiac muscle cell communication
def: "Interacting selectively and non-covalently with any protein or protein complex that results in the connection of a Purkinje myocyte with an ventricular cardiac muscle cell and contributes to the communication between the two cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086085
name: cell adhesive protein binding involved in SA cardiac muscle cell-atrial cardiac muscle cell communication
def: "Interacting selectively and non-covalently with any protein or protein complex that results in the connection of an SA cardiomyocyte with an atrial cardiomyocyte and contributes to the communication between the two cells." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086086
name: voltage-gated potassium channel activity involved in AV node cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of an AV node cardiac muscle cell contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086087
name: voltage-gated potassium channel activity involved in bundle of His cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of a bundle of His cell contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086088
name: voltage-gated potassium channel activity involved in Purkinje myocyte action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of a Purkinje myocyte contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086089
name: voltage-gated potassium channel activity involved in atrial cardiac muscle cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of an atrial cardiomyocyte contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0086090
name: voltage-gated potassium channel activity involved in SA node cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of an SA node cell contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11]
GO:0089702
name: undecaprenyl-phosphate glucose phosphotransferase activity
def: "Catalysis of the reaction: UDP-glucose + ditrans,octacis-undecaprenyl phosphate = UMP + alpha-D-glucopyranosyl-diphospho-ditrans,octacis-undecaprenol." [EC:2.7.8.31, GOC:dos, GOC:imk]
GO:0089710
name: endocytic targeting sequence binding
def: "Interacting selectively and non-covalently with a endocytic signal sequence, a specific peptide sequence, of 4-6 amino acids with an essential tyrosine (Y), found on cytoplasmic tails of some cell surface membrane proteins, which directs internalization by clathrin-coated pits." [PMID:8918456]
GO:0089714
name: UDP-N-acetyl-D-mannosamine dehydrogenase activity
def: "Catalysis of the reaction: UDP-N-acetyl-alpha-D-mannosamine + 2 NAD+ + H2O = UDP-N-acetyl-alpha-D-mannosaminuronate + 2 NADH + 2 H+." [EC:1.1.1.336]
GO:0089715
name: tRNA m6t6A37 methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + tRNA containing N6-threonylcarbamoyladenosine = S-adenosyl-L-homocysteine + tRNA containing N6-methylthreonylcarbamoyladenosine." [PMID:25063302]
GO:0089719
name: RHG protein domain binding
def: "Interacting selectively and non-covalently with an RHG (reaper/hid/grimm) domain/motif (AKA iap binding motif)." [GOC:dos, GOC:ha]
GO:0089720
name: caspase binding
def: "Interacting selectively and non-covalently with a caspase family protein." [GOC:dos, GOC:ha]
GO:0089721
name: phosphoenolpyruvate transmembrane transporter activity
def: "Catalysis of the transfer of a phosphoenolpyruvate from one side of a membrane to the other." [GOC:dos]
GO:0090079
name: translation regulator activity, nucleic acid binding
def: "Any selective and non-covalent interaction with a nucleic acid involved in the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome." [GOC:dph, GOC:tb, GOC:vw]
GO:0090302
name: APC-Cdc20 complex activity
def: "A ubiquitin-protein ligase activity that increases the rate or extent of the metaphase to anaphase transition of the mitotic cell cycle." [GOC:dph, GOC:mah, GOC:tb]
GO:0090353
name: polygalacturonase inhibitor activity
def: "Stops, prevents or reduces the activity of a polygalacturonase. Polygalacturonases catalyze the random hydrolysis of 1,4-alpha-D-galactosiduronic linkages in pectate and other galacturonans." [GOC:tb]
GO:0090369
name: ornithine carbamoyltransferase inhibitor activity
def: "Stops, prevents, or reduces ornithine carbamoyltransferase activity, the catalysis of the reaction: carbamoyl phosphate + L-ornithine = phosphate + L-citrulline." [GOC:dph, GOC:jp, GOC:tb]
GO:0090409
name: malonyl-CoA synthetase activity
def: "Catalysis of the reaction: malonate + ATP + coenzyme A = malonyl-CoA + AMP + diphosphate." [MetaCyc:RXN-12359]
GO:0090411
name: brassinosteroid binding
def: "Interacting selectively and non-covalently with a brassinosteroid." [GOC:tb]
GO:0090415
name: 7-hydroxymethyl chlorophyll a reductase activity
def: "Catalysis of the reaction: 7-hydroxymethyl chlorophyll a + 2 reduced ferredoxin + 2 H+ chlorophyll a + 2 oxidized ferredoxin + H2O." [GOC:kad, PMID:21934147]
GO:0090416
name: nicotinate transporter activity
def: "Enables the directed movement of nicotinate into, out of or within a cell, or between cells." [GOC:tb]
GO:0090417
name: N-methylnicotinate transporter activity
def: "Enables the directed movement of N-methylnicotinate into, out of or within a cell, or between cells." [GOC:tb]
GO:0090422
name: thiamine pyrophosphate transporter activity
def: "Enables the directed movement of thiamine pyrophosphate into, out of or within a cell, or between cells." [GOC:tb]
GO:0090430
name: caffeoyl-CoA: alcohol caffeoyl transferase activity
def: "Catalysis of the reaction: caffeoyl-CoA + a saturated primary alcohol = an alkyl caffeate + CoA." [GOC:pz]
GO:0090432
name: myristoyl-CoA ligase activity
def: "Catalysis of the reaction: ATP + myristic acid + CoA = AMP + diphosphate + myristoyl-CoA." [GOC:al, PMID:18071249]
GO:0090433
name: palmitoyl-CoA ligase activity
def: "Catalysis of the reaction: ATP + palmitic acid + CoA = AMP + diphosphate + palmitoyl-CoA." [GOC:al, PMID:18071249]
GO:0090434
name: oleoyl-CoA ligase activity
def: "Catalysis of the reaction: ATP + oleic acid + CoA = AMP + diphosphate + oleoyl-CoA." [GOC:al, PMID:18071249]
GO:0090438
name: camelliol C synthase activity
def: "Catalyzes the reaction: (3S)-2,3-epoxy-2,3-dihydrosqualene = camelliol C." [GOC:tb, PMID:17985917]
GO:0090439
name: tetraketide alpha-pyrone synthase activity
def: "Catalyzes the reaction: a hydroxyacyl-CoA + 3 malonyl-CoA + 2 H+ = a hydroxylated tetraketide alpha-pyrone + 3 CO2 + 4 coenzyme A ." [MetaCyc:RXN-12183, PMID:21193570]
GO:0090440
name: abscisic acid transporter activity
def: "Enables the directed movement of abscisic acid into, out of or within a cell, or between cells." [GOC:tb]
GO:0090447
name: glycerol-3-phosphate 2-O-acyltransferase activity
def: "Catalysis of the reaction: an acyl-CoA + sn-glycerol 3-phosphate = CoA + a 2-acyl-sn-glycerol 3-phosphate." [EC:2.3.1.198]
GO:0090448
name: glucosinolate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: glucosinolate(out) + H+(out) = glucosinolate(in) + H+(in)." [PMID:22864417]
GO:0090450
name: inosine-diphosphatase activity
def: "Catalysis of the reaction: IDP + H2O = IMP + phosphate." [PMID:22849572]
GO:0090471
name: 9,15,9'-tri-cis-zeta-carotene isomerase activity
def: "Catalysis of the reaction: 9,15,9'-tricis-zeta-carotene = 9,9'-dicis-zeta-carotene." [EC:5.2.1.12]
GO:0090482
name: vitamin transmembrane transporter activity
def: "Enables the transfer of a vitamin from one side of a membrane to the other." [GOC:tb]
GO:0090483
name: phosphatidylglycerol-phosphatidylethanolamine phosphatidyltransferase activity
def: "Catalysis of the reaction: phosphatidylglycerol + phosphatidylethanolamine = cardiolipin + ethanolamine." [PMID:22988102]
GO:0090484
name: drug transporter activity
def: "Enables the directed movement of a drug into, out of or within a cell, or between cells. A drug is any naturally occurring or synthetic substance, other than a nutrient, that, when administered or applied to an organism, affects the structure or functioning of the organism; in particular, any such substance used in the diagnosis, prevention, or treatment of disease." [GOC:tb]
GO:0090486
name: small RNA 2'-O-methyltransferase
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the oxygen atom of a nucleoside residue in a small RNA molecule. Reaction: S-adenosyl-L-methionine + small RNA <=> S-adenosyl-L-homocysteine + small RNA containing a 3'-terminal 2'-O-methylnucleotide." [EC:2.1.1.n8, GOC:tb, GOC:vw]
GO:0090488
name: polo box domain specific binding
def: "Interacting selectively and non-covalently with a polo box domain of a protein. The polo box domain is involved in binding substrates of polo kinases." [GOC:al, GOC:tb, Pfam:PF00659, PMID:12352953]
GO:0090489
name: L-tryptophan,NADPH:oxygen oxidoreductase (N-hydroxylating, decarboxylating)
def: "Catalyzes the multi-step reaction: L-Tryptophan + 2 Oxygen + 2 NADPH + 2 H+ = Indole-3-acetaldehyde oxime + 3 H2O + 2 NADP+ + CO2. The individual reactions are: (1a) L-tryptophan + O2 + NADPH + H+ = N-hydroxy-L-tryptophan + NADP+ + H2O,(1b) N-hydroxy-L-tryptophan + O2 + NADPH + H+ = N,N-dihydroxy-L-tryptophan + NADP+ + H2O, and (1c) N,N-dihydroxy-L-tryptophan = (E)-indol-3-ylacetaldoxime + CO2 + H2O." [EC:1.14.13.125, KEGG:R08160]
GO:0090490
name: L-tryptophan,NADPH:oxygen oxidoreductase (N-hydroxylating)
def: "Catalyzes the reaction: L-Tryptophan + Oxygen + NADPH + H+ = N-Hydroxy-L-tryptophan + NADP+ + H2O." [KEGG:R09583]
GO:0090491
name: N-hydroxy-L-tryptophan,NADPH:oxygen oxidoreductase (N-hydroxylating)
def: "Catalyzes the reaction: N-Hydroxy-L-tryptophan + Oxygen + NADPH + H+ = N,N-Dihydroxy-L-tryptophan + NADP+ + H2O." [KEGG:R09584]
GO:0090492
name: N,N-Dihydroxy-L-tryptophan decarboxylase activity
def: "Catalyzes the reaction: N,N-Dihydroxy-L-tryptophan = Indole-3-acetaldehyde oxime + CO2 + H2O." [KEGG:R09585]
GO:0090499
name: pimelyl-[acyl-carrier protein] methyl ester esterase activity
def: "Catalysis of the reaction: pimelyl-[acyl-carrier protein] methyl ester + H2O = pimelyl-[acyl-carrier protein] + methanol." [EC:3.1.1.85, PMID:23045647]
GO:0090523
name: cytochrome-b5 reductase activity, acting on NADPH
def: "Catalysis of the reaction: NADPH + H+ + 2 ferricytochrome b(5) = NADP+ + 2 ferrocytochrome b(5)." [GOC:tb]
GO:0090524
name: cytochrome-b5 reductase activity, acting on NADH
def: "Catalysis of the reaction: NADH + H+ + 2 ferricytochrome b(5) = NAD+ + 2 ferrocytochrome b(5)." [GOC:tb]
GO:0090541
name: MIT domain binding
def: "Interacting selectively and non-covalently with the MIT domain of a protein. The MIT domain is found in vacuolar sorting proteins, spastin (probable ATPase involved in the assembly or function of nuclear protein complexes), and a sorting nexin, which may play a role in intracellular trafficking." [GOC:pm, InterPro:IPR007330]
GO:0090542
name: ELYC domain binding
def: "Interacting selectively and non-covalently with the ELYC domain of a protein. The ELYC domain is an approximately 150 amino acid sequence which contains a highly conserved tetrapeptide sequence, ELYC." [GOC:pm, PMID:18032582, PMID:19525971]
GO:0090554
name: phosphatidylcholine-translocating ATPase activity
def: "Catalysis of the movement of phosphatidylcholine from one membrane bilayer leaflet to the other, driven by the hydrolysis of ATP." [CHEBI:64612, GOC:ab, PMID:16452632, RHEA:38584]
GO:0090555
name: phosphatidylethanolamine-translocating ATPase activity
def: "Catalysis of the movement of phosphatidylethanolamine from one membrane bilayer leaflet to the other, driven by the hydrolysis of ATP." [CHEBI:64612, GOC:ab, PMID:16452632, RHEA:36440]
GO:0090556
name: phosphatidylserine-translocating ATPase activity
def: "Catalysis of the movement of phosphatidylserine from one membrane bilayer leaflet to the other, driven by the hydrolysis of ATP." [CHEBI:57262, GOC:ab, PMID:16452632, PMID:20224745, RHEA:38568]
GO:0090560
name: 2-(3-amino-3-carboxypropyl)histidine synthase activity
def: "Catalysis of the reaction S-adenosyl-L-methionine + L-histidine-[translation elongation factor 2] = S-methyl-5-thioadenosine + 2-[(3S)-3-amino-3-carboxypropyl]-L-histidine-[translation elongation factor 2]." [BRENDA:2.5.1.108, GOC:pde, PMID:15485916]
GO:0090562
name: protein-N(PI)-phosphohistidine-N,N'-diacetylchitobiose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N,N'-diacetylchitobiose(out) = protein histidine + N,N'-diacetylchitobiose phosphate(in)." [GOC:am, PMID:10913119]
GO:0090563
name: protein-phosphocysteine-sugar phosphotransferase activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + sugar(out) = protein cysteine + sugar phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:am]
GO:0090564
name: protein-phosphocysteine-glucose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + glucose(out) = protein cysteine + glucose phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:am]
GO:0090565
name: protein-phosphocysteine-mannitol phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + mannitol(out) = protein cysteine + mannitol phosphate(in). This differs from primary and secondary active transport in that the solute is modified during transport." [GOC:am]
GO:0090566
name: protein-phosphocysteine-N,N'-diacetylchitobiose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + N,N'-diacetylchitobiose(out) = protein cysteine + N,N'-diacetylchitobiose phosphate(in)." [GOC:am, PMID:10913119]
GO:0090580
name: phosphodiesterase activity, acting on 3'-phosphoglycolate-terminated DNA strands
def: "Catalysis of the hydrolytic removal of phosphoglycolate from the 3'-terminus of a 3'-phosphoglycolate-terminated oligonucleotides." [GOC:pde, GOC:rb, PMID:11238902]
GO:0090581
name: protein-phosphocysteine-mannosylglycerate-phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + mannosylglycerate(out) = protein cysteine + mannosylglycerate phosphate(in)." [PMID:14645248]
GO:0090582
name: protein-phosphocysteine-D-fructose-phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + D-fructose(out) = protein cysteine + D-fructose-1-phosphate(in)." [PMID:8626640]
GO:0090583
name: protein-phosphocysteine-D-sorbitol-phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + D-sorbitol(out) = protein cysteine + D-sorbitol-1-phosphate(in)." [PMID:8875915]
GO:0090584
name: protein-phosphocysteine-galactitol-phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + galactitol(out) = protein cysteine + galactitol-6-phosphate(in)." [PMID:8955298]
GO:0090585
name: protein-phosphocysteine-L-ascorbate-phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + L-ascorbate(out) = protein cysteine + L-ascorbate-6-phosphate(in)." [PMID:15153772]
GO:0090586
name: protein-phosphocysteine-N-acetylglucosamine phosphotransferase system transporter activity
def: ": Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + N-acetylglucosamine (out) = protein cysteine + N-acetylglucosamine-6-phosphate (in)." [PMID:8246840]
GO:0090587
name: protein-phosphocysteine-glucosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + glucosamine (out) = protein cysteine + glucosamine-6-phosphate (in)." [PMID:8246840]
GO:0090588
name: protein-phosphocysteine-N-acetylmuramate phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + N-acetylmuramate (out) = protein cysteine + N-acetylmuramate-6-phosphate (in)." [PMID:15060041]
GO:0090589
name: protein-phosphocysteine-trehalose phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein S-phosphocysteine + trehalose (out) = protein cysteine + trehalose-6-phosphate (in)." [PMID:7608078]
GO:0090590
name: protein-N(PI)-phosphohistidine-D-glucosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + D-glucosamine(out) = protein histidine + glucosamine-6-phosphate(in)." [PMID:8246840]
GO:0090591
name: protein-N(PI)-phosphohistidine-N-acetyl-mannosamine phosphotransferase system transporter activity
def: "Catalysis of the PEP-dependent, phosphoryl transfer-driven transport of substances across a membrane. The transport happens by catalysis of the reaction: protein N-phosphohistidine + N-acetyl-mannosamine(out) = protein histidine +N-acetyl- mannosamine-6-phosphate(in)." [PMID:9864311]
GO:0090595
name: acetyl-CoA:L-lysine N6-acetyltransferase
def: "Catalysis of the reaction: L-lysine + acetyl-CoA = N6-acetyl-L-lysine + CoA + H(+)." [MetaCyc:LYSACET-RXN]
GO:0090599
name: alpha-glucosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-linked alpha-D-glucose residue with release of alpha-D-glucose." [GOC:tb]
GO:0090600
name: alpha-1,3-glucosidase activity
def: "Catalysis of the hydrolysis of terminal, non-reducing alpha-(1->3)-linked alpha-D-glucose residues with release of alpha-D-glucose." [GOC:sd, GOC:tb]
GO:0090612
name: cAMP deaminase activity
def: "Catalysis of the reaction: cyclic adenosine monophosphate + H2O = cyclic inosine monophosphate + NH3." [PMID:24074367]
GO:0090613
name: 5'-deoxyadenosine deaminase activity
def: "Catalysis of the reaction: 5'deoxyadenosine + H2O = 5'deoxyinosine + NH3." [PMID:23968233]
GO:0090614
name: 5'-methylthioadenosine deaminase activity
def: "Catalysis of the reaction: 5'methyl thioadenosine + H2O = 5'methyl thioinosine + NH3." [PMID:23968233]
GO:0090622
name: APC-fizzy related complex activity
def: "A ubiquitin-protein ligase activity that increases the rate or extent of the mitotic exit." [GOC:vw, PMID:10921876]
GO:0090624
name: endoribonuclease activity, cleaving miRNA-paired mRNA
def: "Catalysis of the endonucleolytic cleavage of the mRNA in a double-stranded RNA molecule formed by the base pairing of an mRNA with an miRNA, yielding 5'-phosphomonoesters." [GOC:BHF_miRNA, GOC:rph, PMID:15260970]
GO:0097001
name: ceramide binding
def: "Interacting selectively and non-covalently with any ceramide, a class of lipids that is composed of sphingosine linked to a fatty acid. Ceramides are a major component of cell membranes." [CHEBI:17761, GOC:sart]
GO:0097003
name: adipokinetic hormone receptor activity
def: "Combining with an adipokinetic hormone to initiate a change in cell activity. Adipokinetic hormones (AKHs) are peptide hormones that are involved in the mobilization of sugar and lipids from the insect fat body during energy-requiring activities such as flight and locomotion. They also contribute to hemolymph sugar homeostasis." [GOC:sart, PMID:11904407]
GO:0097004
name: adipokinetic hormone binding
def: "Interacting selectively and non-covalently with an adipokinetic hormone. Adipokinetic hormones (AKHs) are peptide hormones that are involved in the mobilization of sugar and lipids from the insect fat body during energy-requiring activities such as flight and locomotion. They also contribute to hemolymph sugar homeostasis." [GOC:sart, PMID:11904407]
GO:0097005
name: adipokinetic hormone receptor binding
def: "Interacting selectively and non-covalently with an adipokinetic hormone receptor. Adipokinetic hormones (AKHs) are peptide hormones that are involved in the mobilization of sugar and lipids from the insect fat body during energy-requiring activities such as flight and locomotion. They also contribute to hemolymph sugar homeostasis." [GOC:sart, PMID:11904407]
GO:0097007
name: 4,8,12-trimethyltrideca-1,3,7,11-tetraene synthase activity
def: "Catalysis of the reaction: (EE)-geranyllinalool + NADPH + O2 = 4,8,12-trimethyl-1,3,7,11-tridecatetraene + NADP+ + 2 H2O. It is unknown whether this reaction proceeds by the direct release of the 4-carbon compound but-1-en-3-one, or whether the substrate is first degraded to C18-farnesylacetone and then cleaved to produce 4,8,12-trimethyl-1,3,7,11-tridecatetraene (TMTT) and acetone." [GOC:kad, MetaCyc:RXN-8620, PMID:21088219]
GO:0097008
name: (3E)-4,8-dimethyl-1,3,7-nonatriene synthase activity
def: "Catalysis of the reaction: (E)-nerolidol + NADPH + O2 = (3E)-4,8-dimethylnona-1,3,7-triene + NADP+ + 2 H2O. It is unknown whether this reaction proceeds by the direct release of the 4-carbon compound but-1-en-3-one, or whether the substrate is first degraded to C11-geranylacetone and then cleaved to produce (3E)-4,8-dimethylnona-1,3,7-triene (DMNT) and acetone." [GOC:kad, MetaCyc:RXN-8619, PMID:21088219]
GO:0097016
name: L27 domain binding
def: "Interacting selectively and non-covalently with a L27 domain of a protein. L27 is composed of conserved negatively charged amino acids and a conserved aromatic amino acid. L27 domains can assemble proteins involved in signaling and establishment and maintenance of cell polarity into complexes by interacting in a heterodimeric manner." [GOC:BHF, PMID:15241471, PMID:17237226, Prosite:PDOC51022]
GO:0097020
name: COPII adaptor activity
def: "The binding activity of a molecule that brings together the COPII vesicle proteins and one or more other molecules, permitting them to function in a coordinated way." [GOC:rb, PMID:16957051, PMID:20236934]
GO:0097023
name: fructose 6-phosphate aldolase activity
def: "Catalysis of the reaction: D-fructose-6-phosphate = dihydroxyacetone + D-glyceraldehyde-3-phosphate." [GOC:imk, PMID:11120740, PMID:21290439]
GO:0097027
name: ubiquitin-protein transferase activator activity
def: "Increases the activity of a ubiquitin-protein transferase, an enzyme that catalyzes the covalent attachment of ubiquitin to lysine in a substrate protein." [GOC:rb, PMID:18321851]
GO:0097030
name: CENP-A containing nucleosome binding
def: "Interacting selectively and non-covalently with a centromere-specific nucleosome, a form of nucleosome located only at the centromere, in which the histone H3 is replaced by the variant form CENP-A (sometimes known as CenH3)." [GOC:jp, PMID:21412236]
GO:0097063
name: cadmium ion sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of cadmium (Cd++)." [GOC:rs, PMID:19456862]
GO:0097077
name: copper ion sensor activity
def: "Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of copper(I) (Cu+)." [GOC:rs, PMID:19928961]
GO:0097079
name: selenite:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: selenite(out) + H+(out) = selenite(in) + H+(in)." [GOC:mcc, PMID:20861301]
GO:0097098
name: DNA/RNA hybrid annealing activity
def: "Facilitates the base-pairing of single-stranded RNA to double-stranded DNA resulting in the formation of R-loops." [GOC:imk, PMID:21699496]
GO:0097099
name: structural constituent of albumen
def: "The action of a molecule that contributes to the structural integrity of albumen (also called egg white). Albumen is the clear liquid contained within an egg and consists of water and proteins, among which are ovomucin and ovomucoid. It protects the egg yolk and provides additional nutrition for the growth of the embryo." [GOC:jj, Wikipedia:Albumen]
GO:0097100
name: supercoiled DNA binding
def: "Interacting selectively and non-covalently with supercoiled DNA. For example, during replication and transcription, template DNA is negatively supercoiled in the receding downstream DNA and positively supercoiled in the approaching downstream DNA." [GOC:pr, GOC:rph, PMID:20723754, PMID:21345933, Wikipedia:DNA_supercoil]
GO:0097108
name: hedgehog family protein binding
def: "Interacting selectively and non-covalently with a member of the hedgehog protein family, signaling proteins involved in development." [GOC:BHF, GOC:pr, PMID:10050855]
GO:0097109
name: neuroligin family protein binding
def: "Interacting selectively and non-covalently with a member of the neuroligin protein family, neuronal cell surface proteins that mediate synapse formation." [GOC:BHF, GOC:pr, GOC:sjp, PMID:21424692]
GO:0097110
name: scaffold protein binding
def: "Interacting selectively and non-covalently with a scaffold protein. Scaffold proteins are crucial regulators of many key signaling pathways. Although not strictly defined in function, they are known to interact and/or bind with multiple members of a signaling pathway, tethering them into complexes." [GOC:BHF, GOC:sjp, PMID:10433269, Wikipedia:Scaffold_protein]
GO:0097153
name: cysteine-type endopeptidase activity involved in apoptotic process
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile, and contributing to the apoptotic process." [GOC:mtg_apoptosis]
GO:0097157
name: pre-mRNA intronic binding
def: "Interacting selectively and non-covalently with an intronic sequence of a pre-messenger RNA (pre-mRNA)." [GOC:ans, PMID:16260624]
GO:0097158
name: pre-mRNA intronic pyrimidine-rich binding
def: "Interacting selectively and non-covalently with a pyrimidine-rich (CU-rich) intronic sequence of a pre-messenger RNA (pre-mRNA)." [GOC:ans, PMID:16260624, PMID:16777844]
GO:0097159
name: organic cyclic compound binding
def: "Interacting selectively and non-covalently with an organic cyclic compound, any molecular entity that contains carbon arranged in a cyclic molecular structure." [CHEBI:33832, GOC:sjw, PMID:7583672]
GO:0097160
name: polychlorinated biphenyl binding
def: "Interacting selectively and non-covalently with a polychlorinated biphenyl (PCB), a biphenyl compound containing between 2 and 10 chlorine atoms attached to the two benzene rings." [CHEBI:53156, GOC:sjw, PMID:7583672]
GO:0097161
name: DH domain binding
def: "Interacting selectively and non-covalently with a DH (Dbl homology) domain of a protein. The DH domain contains three structurally conserved regions separated by more variable regions. It is composed of 11 alpha helices that are folded into a flattened, elongated alpha-helix bundle in which two of the three conserved regions, conserved region 1 (CR1) and conserved region 3 (CR3), are exposed near the centre of one surface. CR1 and CR3, together with a part of alpha-6 and the DH/PH (pleckstrin homology) junction site, constitute the Rho GTPase interacting pocket." [GOC:yaf, InterPro:IPR000219, PMID:12775584]
GO:0097162
name: MADS box domain binding
def: "Interacting selectively and non-covalently with a MADS box domain, a protein domain that encodes the DNA-binding MADS domain. The MADS domain binds to DNA sequences of high similarity to the motif CC[A/T]6GG termed the CArG-box. MADS-domain proteins are generally transcription factors. The length of the MADS-box is in the range of 168 to 180 base pairs." [GOC:yaf, InterPro:IPR002100, PMID:18296735, Wikipedia:MADS-box]
GO:0097163
name: sulfur carrier activity
def: "Enables the directed movement of sulfur into, out of or within a cell, or between cells." [GOC:imk, PMID:16387657]
GO:0097177
name: mitochondrial ribosome binding
def: "Interacting selectively and non-covalently with any part of a mitochondrial ribosome, a ribosome found in the mitochondrion of a eukaryotic cell." [GOC:ans, PMID:20739282]
GO:0097199
name: cysteine-type endopeptidase activity involved in apoptotic signaling pathway
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile, and contributing to the apoptotic signaling pathway." [GOC:mtg_apoptosis, http://en.wikipedia.org/wiki/Caspase, PMID:11717445]
GO:0097200
name: cysteine-type endopeptidase activity involved in execution phase of apoptosis
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile, and contributing to the execution phase of apoptosis." [GOC:mtg_apoptosis, http://en.wikipedia.org/wiki/Caspase]
GO:0097216
name: guanosine tetraphosphate binding
def: "Interacting selectively and non-covalently with guanosine tetraphosphate (5'-ppGpp-3'), a guanosine bisphosphate having diphosphate groups at both the 3' and 5'-positions." [CHEBI:17633, GOC:imk, PMID:15109491, PMID:16968770, PMID:18359660]
GO:0097243
name: flavonoid binding
def: "Interacting selectively and non-covalently with a flavonoid, a compound containing two or more aromatic rings, each bearing at least one aromatic hydroxyl and connected with a carbon bridge." [CHEBI:47916, GOC:sl, PMID:20599706]
GO:0097244
name: flavonol binding
def: "Interacting selectively and non-covalently with a flavonol, a flavonoid that contains a 3-hydroxy-2-phenylchromen-4-one backbone." [CHEBI:5078, GOC:sl]
GO:0097245
name: flavanol binding
def: "Interacting selectively and non-covalently with a flavanol." [CHEBI:24036, GOC:sl]
GO:0097246
name: catechin binding
def: "Interacting selectively and non-covalently with a catechin, a polyphenolic antioxidant plant metabolite with a flavonoid or flavan-3-ol structure." [CHEBI:23053, GOC:sl]
GO:0097247
name: epigallocatechin 3-gallate binding
def: "Interacting selectively and non-covalently with epigallocatechin 3-gallate, a compound that is a gallic acid ester of a catechin." [CHEBI:4806, GOC:sl, PMID:21307292]
GO:0097253
name: beta-hydroxybutyrate transmembrane transporter activity
def: "Catalysis of the transfer of beta-hydroxybutyrate from one side of the membrane to the other. Beta-hydroxybutyrate is the conjugate base of (R)-3-hydroxybutyric acid." [CHEBI:10983, GOC:dsf, PMID:22302940]
GO:0097256
name: phenyllactate dehydrogenase activity
def: "Catalysis of the oxidation of phenyllactate to produce phenylpyruvate." [GOC:pde, PMID:15421980]
GO:0097257
name: leukotriene B4 12-hydroxy dehydrogenase activity
def: "Catalysis of the reaction: leukotriene B4 + NADP(+) = 12-oxo-leukotriene B4 + NADPH + H(+)." [GOC:mw, KEGG:R03864, PMID:8394361, PMID:9461497]
GO:0097258
name: 20-hydroxy-leukotriene B4 omega oxidase activity
def: "Catalysis of the reaction: 20-hydroxy-leukotriene B4 + O2 + NADPH + H(+) = 20-aldehyde-leukotriene B4 + 2H2O + NADP(+)." [GOC:mw, PMID:2836406]
GO:0097259
name: 20-aldehyde-leukotriene B4 20-monooxygenase activity
def: "Catalysis of the reaction: 20-aldehyde-leukotriene B4 + O2 + NADPH + H(+) = 20-carboxy-leukotriene B4 + H2O + NADP(+)." [GOC:mw, PMID:2549038, PMID:2836406]
GO:0097260
name: eoxin A4 synthase activity
def: "Catalysis of the reaction: leukotriene A4 = eoxin A4." [GOC:mw, PMID:18184802, PMID:18647347]
GO:0097261
name: eoxin C4 synthase activity
def: "Catalysis of the reaction: eoxin A4 + glutathione = eoxin C4." [GOC:mw, PMID:18184802, PMID:18647347]
GO:0097262
name: eoxin D4 synthase activity
def: "Catalysis of the reaction: eoxin C4 = eoxin D4 + 5-L-glutamyl amino acid." [GOC:mw, PMID:18184802, PMID:18647347]
GO:0097263
name: eoxin E4 synthase activity
def: "Catalysis of the reaction: eoxin D4 + H20 = eoxin E4 + glycine." [GOC:mw, PMID:18184802, PMID:18647347]
GO:0097265
name: 5(S)-hydroxyeicosatetraenoic acid dehydrogenase activity
def: "Catalysis of the reaction: 5-HETE + NADP(+) = 5-oxo-ETE + NADPH + H(+)." [GOC:mw, PMID:1326548]
GO:0097266
name: phenylacetyl-CoA 1,2-epoxidase activity
def: "Catalysis of the reaction: phenylacetyl-CoA + H(+) + NADPH + O2 = 2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA + H2O + NADP(+)." [EC:1.14.13.149, GOC:bf, GOC:gk, PMID:20660314, PMID:21247899]
GO:0097269
name: all-trans-decaprenyl-diphosphate synthase activity
def: "Catalysis of the reaction: 2-trans,6-trans-farnesyl diphosphate + 7 isopentenyl diphosphate = 7 diphosphate + all-trans-decaprenyl diphosphate." [EC:2.5.1.91, GOC:mw, PMID:16262699]
GO:0097299
name: cysteine-type endopeptidase activity involved in plant-type hypersensitive response
def: "Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile, and contributing to plant-type hypersensitive response, the rapid, localized death of plant cells in response to invasion by a pathogen." [GOC:mtg_apoptosis]
GO:0097322
name: 7SK snRNA binding
def: "Interacting selectively and non-covalently with a 7SK small nuclear RNA (7SK snRNA)." [GOC:nhn, PMID:21853533]
GO:0097351
name: toxin-antitoxin pair type II binding
def: "Interacting selectively and non-covalently with a toxic protein, disabling its function. There may be more than one antitoxin to a toxic protein. Instances of this activity are known only in prokaryotes, where the toxic protein may be a ribonuclease, a DNA gyrase, or other." [GOC:rs, http://en.wikipedia.org/wiki/Toxin-antitoxin_system#Type_II, PMID:19143615, PMID:19325885, PMID:21819231, PMID:22545240]
GO:0097358
name: D-leucyl-tRNA(Leu) deacylase activity
def: "Catalysis of the reaction: D-leucyl-tRNA(Leu) = D-leucine + tRNA(Leu). Hydrolysis of the removal of D-leucine from residues in charged tRNA(Leu)." [GOC:se, PMID:10918062]
GO:0097363
name: protein O-GlcNAc transferase activity
def: "Catalysis of the reaction: UDP-N-acetyl-D-glucosamine + [protein]-L-serine = UDP + [protein]-3-O-(N-acetyl-D-glucosaminyl)-L-serine, or UDP-N-acetyl-D-glucosamine + [protein]-L-threonine = UDP + [protein]-3-O-(N-acetyl-D-glucosaminyl)-L-threonine." [EC:2.4.1.255, GOC:jsg, GOC:sart, PMID:22158438]
GO:0097364
name: stretch-activated, cation-selective, calcium channel activity involved in regulation of action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens in response to a mechanical stress in the form of stretching, and contributing to the regulation of action potential." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11, PMID:21290758]
GO:0097365
name: stretch-activated, cation-selective, calcium channel activity involved in regulation of cardiac muscle cell action potential
def: "Catalysis of the transmembrane transfer of a calcium ion by a channel that opens in response to a mechanical stress in the form of stretching, and contributing to the regulation of action potential in a cardiac muscle cell." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11, PMID:21290758]
GO:0097367
name: carbohydrate derivative binding
def: "Interacting selectively and non-covalently with a carbohydrate derivative." [GOC:pr]
GO:0097371
name: MDM2/MDM4 family protein binding
def: "Interacting selectively and non-covalently with any isoform of the MDM2/MDM4 protein family, comprising negative regulators of p53." [InterPro:IPR016495]
GO:0097372
name: NAD-dependent histone deacetylase activity (H3-K18 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 18) + H2O = histone H3 L-lysine (position 18) + acetate. This reaction requires the presence of NAD, and represents the removal of an acetyl group from lysine at position 18 of the histone H3 protein." [EC:3.5.1.17, EC:3.5.1.98, GOC:sp, PMID:22722849, RHEA:24551]
GO:0097382
name: deoxynucleoside-diphosphatase activity
def: "Catalysis of the reaction: a deoxynucleoside diphosphate + H2O = a deoxynucleotide + phosphate." [GOC:pde]
GO:0097383
name: dIDP diphosphatase activity
def: "Catalysis of the reaction: dIDP + H2O = dIMP + phosphate." [GOC:pde, PMID:20385596]
GO:0097429
name: amino acid ligation activity by nonribosomal peptide synthase
def: "Catalysis of the ligation of an amino acid to another amino acid via a carbon-nitrogen bond, with the concomitant hydrolysis of the diphosphate bond in ATP or a similar triphosphate, carried out by a nonribosomal peptide synthase." [GOC:vw]
GO:0097434
name: succinate:proton symporter activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: succinate(out) + H+(out) = succinate(in) + H+(in)." [GOC:al, PMID:1293882]
GO:0097469
name: cyclin-dependent protein tyrosine kinase activity
def: "OBSOLETE. Catalysis of the reaction: ATP + a protein tyrosine = ADP + protein tyrosine phosphate. This reaction requires the binding of a regulatory cyclin subunit." [GOC:al, GOC:vw, PMID:1372994]
GO:0097472
name: cyclin-dependent protein kinase activity
def: "Catalysis of the phosphorylation of an amino acid residue in a protein, usually according to the reaction: a protein + ATP = a phosphoprotein + ADP. This reaction requires the binding of a regulatory cyclin subunit and full activity requires stimulatory phosphorylation by a CDK-activating kinase (CAK)." [GOC:pr]
GO:0097493
name: structural molecule activity conferring elasticity
def: "The action of a molecule that contributes to the structural integrity of a complex or assembly within or outside a cell, providing elasticity and recoiling." [GOC:BHF, GOC:rl, PMID:23283722]
GO:0097506
name: deaminated base DNA N-glycosylase activity
def: "DNA N-glycosylase activity acting on deaminated bases." [GOC:al, PMID:18789404]
GO:0097507
name: hypoxanthine DNA N-glycosylase activity
def: "DNA N-glycosylase activity acting on deaminated adenine (hypoxanthine)." [GOC:al, PMID:18789404]
GO:0097508
name: xanthine DNA N-glycosylase activity
def: "DNA N-glycosylase activity acting on deaminated guanine (xanthine)." [GOC:al, PMID:18789404]
GO:0097509
name: oxanine DNA N-glycosylase activity
def: "DNA N-glycosylase activity acting on deaminated guanine where the resulting base (oxanine) is generated by NO- or HNO2-induced nitrosative deamination." [GOC:al, PMID:18789404]
GO:0097573
name: glutathione oxidoreductase activity
def: "Catalysis of the reaction: protein-S-S-glutathione + glutathione-SH = protein-SH + glutathione-S-S-glutathione." [GOC:jd, PMID:18992757]
GO:0097599
name: xylanase activity
def: "Catalysis of the hydrolysis of xylans, homopolysaccharides composed of xylose residues." [GOC:jh2, ISBN:81-7736-269-0]
GO:0097600
name: exoxylanase activity
def: "A xylanase activity that acts on one of the ends of a xylan polymer which does not contain side chains." [GOC:jh2, ISBN:81-7736-269-0, PMID:16535010]
GO:0097602
name: cullin family protein binding
def: "Interacting selectively and non-covalently with any member of the cullin family, hydrophobic proteins that act as scaffolds for ubiquitin ligases (E3)." [GOC:ha, InterPro:IPR016158, PMID:18698375]
GO:0097603
name: temperature-gated ion channel activity
def: "Catalysis of the transmembrane transfer of an ion by a channel that opens in response to a temperature stimulus (e.g. exposure to a temperature range different than the optimal temperature for that organism)." [GOC:ha, GOC:pr, PMID:23027824]
GO:0097604
name: temperature-gated cation channel activity
def: "Catalysis of the transmembrane transfer of a cation by a channel that opens in response to a temperature stimulus (e.g. exposure to a temperature range different than the optimal temperature for that organism)." [GOC:ha, GOC:pr, PMID:23027824]
GO:0097617
name: annealing activity
def: "A nucleic acid binding activity that brings together complementary sequences of nucleic acids so that they pair by hydrogen bonds to form a double-stranded polynucleotide." [GOC:mba, http://en.wikipedia.org/wiki/Nucleic_acid_thermodynamics#Annealing]
GO:0097620
name: (R)-mandelate dehydrogenase activity
def: "Catalysis of the reaction: (R)-2-hydroxy-2-phenylacetate + acceptor = phenylglyoxylate + reduced acceptor." [GOC:pr, PMID:1731758, RHEA:43112]
GO:0097621
name: monoamine oxidase activity
def: "Catalysis of the reaction: RCH2NHR' + H2O + O2 = RCHO + R'NH2 + H2O2." [EC:1.4.3.4, GOC:pr, RHEA:26417]
GO:0097625
name: low-affinity basic amino acid transmembrane transporter activity
def: "Catalysis of the transfer of basic amino acids from one side of a membrane to the other. Basic amino acids have a pH above 7. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:pr]
GO:0097626
name: low-affinity L-arginine transmembrane transporter activity
def: "Catalysis of the transfer of L-arginine from one side of a membrane to the other. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations." [GOC:krc, PMID:8195186]
GO:0097627
name: high-affinity L-ornithine transmembrane transporter activity
def: "Catalysis of the transfer of L-ornithine from one side of a membrane to the other. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations." [GOC:krc, PMID:8195186]
GO:0097641
name: alpha-ketoglutarate-dependent xanthine dioxygenase activity
def: "Catalysis of the reaction: 2-oxoglutarate (alpha-ketoglutarate) + O2 + xanthine = CO2 + succinate + urate." [GOC:vw, PMID:15948966, PMID:17429948, RHEA:43120]
GO:0097642
name: calcitonin family receptor activity
def: "Combining with any member of the calcitonin family (e.g. adrenomedullin, adrenomedullin 2 (intermedin), amylin, calcitonin and calcitonin gene-related peptides (CGRPs)) to initiate a change in cell activity." [GOC:bhm, InterPro:IPR003287, PMID:10871296, PMID:12037140, PMID:18687416]
GO:0097643
name: amylin receptor activity
def: "Combining with amylin to initiate a change in cell activity." [GOC:bhm, PMID:10871296, PMID:12037140, PMID:18687416]
GO:0097644
name: calcitonin family binding
def: "Interacting selectively and non-covalently with any member of the calcitonin family (e.g. adrenomedullin, adrenomedullin 2 (intermedin), amylin, calcitonin and calcitonin gene-related peptides (CGRPs))." [GOC:bhm, InterPro:IPR021116, PMID:10871296, PMID:12037140, PMID:18687416]
GO:0097645
name: amylin binding
def: "Interacting selectively and non-covalently with amylin." [GOC:bhm, PMID:10871296, PMID:12037140, PMID:18687416]
GO:0097655
name: serpin family protein binding
def: "Interacting selectively and non-covalently with any member of the serpin protein family (serine protease inhibitors or classified inhibitor family I4). Serpins are a broadly distributed family of protease inhibitors that use a conformational change to inhibit target enzymes. They are central in controlling many important proteolytic cascades. The majority of serpins inhibit serine proteases, but serpins that inhibit caspases and papain-like cysteine proteases have also been identified. Rarely, serpins perform a noninhibitory function; for example, several human serpins function as hormone transporters and certain serpins function as molecular chaperones or tumor suppressors." [GOC:mr, InterPro:IPR000215, PMID:16737556]
GO:0097657
name: 3',5'-nucleotide bisphosphate phosphatase activity
def: "Catalysis of the reaction: 3',5'-nucleotide bisphosphate + H20 = 5'-nucleotide monophosphate + phosphate." [GOC:jh2, PMID:24401123, RHEA:43532]
GO:0097677
name: STAT family protein binding
def: "Interacting selectively and non-covalently with any member of the signal transducers and activators of transcription (STAT) protein family. STATs are, as the name indicates, both signal transducers and transcription factors. STATs are activated by cytokines and some growth factors and thus control important biological processes including cell growth, cell differentiation, apoptosis and immune responses." [GOC:mr, InterPro:IPR001217, PMID:21447371, PMID:24470978]
GO:0097678
name: SOCS family protein binding
def: "Interacting selectively and non-covalently with any member of the suppressor of cytokine signaling (SOCS) family of proteins. SOCS represent an important mechanism to extinguish cytokine and growth factor receptor signaling. Individual SOCS proteins are typically induced by specific cytokines and growth factors, thereby generating a negative feedback loop. SOCS proteins have important functions in development and homeostasis, and in disease, particularly tumor suppression and anti-inflammatory functions." [GOC:mr, InterPro:IPR028413, PMID:23885323, PMID:24705897]
GO:0097682
name: intracellular phosphatidylinositol-3,5-bisphosphate-sensitive cation channel activity
def: "Catalysis of the transmembrane transfer of cations by a channel that opens when phosphatidylinositol-3,5-bisphosphate has been bound by the channel complex or one of its constituent parts." [GOC:ha, PMID:24375408]
GO:0098505
name: G-rich strand telomeric DNA binding
def: "Interacting selectively and non-covalently with G-rich, single-stranded, telomere-associated DNA." [PMID:11349150]
GO:0098518
name: polynucleotide phosphatase activity
def: "Catalysis of the reaction: phosphopolynucleotide + H2O = polynucleotide + phosphate." [GOC:mah]
GO:0098519
name: nucleotide phosphatase activity, acting on free nucleotides
def: "Catalysis of the reaction: nucleotide + H2O = nucleotide + phosphate." [GOC:dos]
GO:0098531
name: direct ligand regulated sequence-specific DNA binding transcription factor activity
def: "A DNA binding transcription factor activity that is directly regulated by binding of a ligand to the protein with this activity.  Examples include the lac and trp repressors in E.coli and many steroid hormone receptors." [GOC:dos, http://www.ecocyc.org/ECOLI/NEW-IMAGE?object=BC-3.1.2.3]
GO:0098546
name: 2',5-3',5'-cyclic GMP-AMP binding
def: "Interacting selectively and non-covalently with c[G(2',5')pA(2',5')p], a cyclic purine dinucleotide that consists of AMP and GMP units cyclized via 2',5' and 3',5' linkages." [GOC:dos, PMID:23910378]
GO:0098599
name: palmitoyl hydrolase activity
def: "Catalysis of a hydrolase reaction that removes a palmitoyl moiety from some substrate." [GOC:dos, GOC:pg]
GO:0098600
name: selenomethionine gamma-lyase activity
def: "Catalysis of the reaction: L-Selenomethionine + H2O => Methaneselenol + Ammonia + 2-oxobutanoic acid." [PMID:11578145, PMID:16037612, PMID:16444005]
GO:0098601
name: selenomethionine adenosyltransferase activity
def: "Catalysis of the reaction: ATP + L-Selenomethionine + H2O => Orthophosphate + Diphosphate + Se-Adenosylselenomethionine." [PMID:2339986]
GO:0098603
name: selenol Se-methyltransferase activity
def: "Catalysis of the reaction: R + Se-Adenosylselenomethionine => CH3-R + Se-Adenosyl-L-selenohomocysteine." [PMID:1711890]
GO:0098604
name: adenosylselenohomocysteinase activity
def: "Catalysis of the reaction: Se-Adenosyl-L-selenohomocysteine + H2O => Adenosine + Selenohomocysteine." [GOC:dos, PMID:1711890, PMID:7305945]
GO:0098605
name: selenocystathionine beta-synthase activity
def: "Catalysis of the reaction: L-Serine + Selenohomocysteine => L-Selenocystathionine + H2O." [PMID:6456763]
GO:0098606
name: selenocystathionine gamma-lyase activity
def: "Catalysis of the reaction: L-Selenocystathionine + H2O => L-Selenocysteine + NH3 + 2-Oxobutanoic acid." [PMID:6456763]
GO:0098607
name: methylselenocysteine deselenhydrase activity
def: "Catalysis of the reaction: Se-Methyl-L-selenocysteine + H2O => pyruvic acid + NH3 + Methaneselenol." [PMID:17451884, PMID:20383543]
GO:0098608
name: methylselenol demethylase activity
def: "Catalysis of the reaction: methylselenol + H2O => H2Se + CH3OH." [PMID:17451884, PMID:17988700]
GO:0098613
name: methaneselenol methyltransferase activity
def: "Catalysis of the reaction: S-Adenosyl-L-methionine + Methaneselenol => S-Adenosyl-L-homocysteine + Dimethyl selenide." [PMID:14705, PMID:17988700, PMID:4380351]
GO:0098614
name: hydrogen selenide methyltransferase activity
def: "Catalysis of the reaction: S-Adenosyl-L-methionine + Hydrogen selenide => S-Adenosyl-L-homocysteine + Methaneselenol." [PMID:14705, PMID:17988700]
GO:0098615
name: dimethyl selenide methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + dimethyl selenide => S-adenosyl-L-homocysteine + trimethylselenonium." [PMID:17988700, PMID:3350800]
GO:0098616
name: selenate adenylyltransferase (ATP) activity
def: "Catalysis of the reaction: ATP + H2SeO4 => Diphosphate + Adenylylselenate." [PMID:2537056]
GO:0098617
name: adenylylselenate kinase activity
def: "Catalysis of the reaction: ATP + Adenylylselenate => ADP + 3'-Phosphoadenylylselenate." [PMID:2537056]
GO:0098618
name: selenomethionine-tRNA ligase activity
def: "Catalysis of the reaction: ATP + L-Selenomethionine + tRNA(Met) => AMP + Diphosphate + Selenomethionyl-tRNA(Met)." [PMID:16661668, PMID:16661782]
GO:0098619
name: selenocysteine-tRNA ligase activity
def: "Catalysis of the reaction: tRNASec + L-Ser + ATP ---> Ser-tRNASec + AMP + Diphosphate." [PMID:8890909, PMID:9431993, PMID:9637248]
GO:0098620
name: seryl-selenocysteinyl-tRNA kinase activity
def: "Catalysis of the reaction: Ser-tRNA(Sec) + ATP ---> Sep-tRNA(Sec) + ADP" [PMID:15317934]
GO:0098621
name: phosphoseryl-selenocysteinyl-tRNA selenium transferase activity
def: "Catalysis of the reaction: O-Phosphoseryl-tRNA(Sec) + Selenophosphoric acid + H2O => L-Selenocysteinyl-tRNA(Sec) + 2 phosphoric acid." [PMID:17142313, PMID:19608919]
GO:0098622
name: selenodiglutathione-disulfide reductase activity
def: "Catalysis of the reaction: H+ + selenodiglutathione + NADPH => gluthathioselenol + glutathione + NADP+." [PMID:1569062]
GO:0098623
name: selenite reductase activity
def: "Catalysis of the reaction: SeO3(2-) + 3NADPH + 5H+ ---> H2Se + 3NADP+ + 3H2O." [PMID:1321713]
GO:0098624
name: 3'-Phosphoadenylylselenate reductase activity
def: "Catalysis of the reaction: 3'-Phosphoadenylylselenate + NADPH => Adenosine 3',5'-bisphosphate + Selenite + NADP+ + H+." [PMID:14723223]
GO:0098625
name: methylselenol reductase activity
def: "Catalysis of the reaction: NADPH + H+ + CH3SeOH => NADP+ + CH3SeH + H2O" [PMID:11782468]
GO:0098626
name: methylseleninic acid reductase activity
def: "Catalysis of the reaction: NADPH + H+ + CH3SeO2H => NADP+ + CH3SeOH + H2O." [PMID:11782468]
GO:0098627
name: protein arginine phosphatase activity
def: "Catalysis of the reaction: protein arginine phosphate + H2O = protein arginine + phosphate." [PMID:23770242]
GO:0098631
name: protein binding involved in cell adhesion
def: "Any protein binding that is involved in cell adhesion." [GOC:dos]
GO:0098632
name: protein binding involved in cell-cell adhesion
def: "Any protein binding that is involved in cell-cell adhesion." [GOC:dos]
GO:0098633
name: collagen fibril binding
def: "Interacting selectively and non-covalently with a collagen fibril." [GOC:dos, PMID:21421911]
GO:0098634
name: protein binding involved in cell-matrix adhesion
def: "Any protein binding that is involved in cell-matrix adhesion." [GOC:dos]
GO:0098638
name: laminin binding involved in cell-matrix adhesion
def: "Any laminin protein binding that occurs as part of cell-matrix adhesion." [GOC:dos]
GO:0098639
name: collagen binding involved in cell-matrix adhesion
def: "Any collagen binding that occurs as part of cell-matrix adhesion." [GOC:dos]
GO:0098640
name: integrin binding involved in cell-matrix adhesion
def: "Any integrin binding that occurs as part of the process of cell-matrix adhesion." [GOC:dos]
GO:0098641
name: cadherin binding involved in cell-cell adhesion
def: "Any cadherin binding that occurs as part of the process of cell-cell adhesion." [GOC:dos]
GO:0098650
name: peptidyl-proline 4-dioxygenase binding
def: "Interacting selectively and non-covalently with a peptidyl-proline 4-dioxygenase." [GOC:dos, GOC:kvm]
GO:0098663
name: transmembrane transporter activity involved in import into cell
def: "Any transmembrane transporter activity that is involved in importing some substance into a cell." [GOC:dos, ISBN:0-8249-3695-6]
GO:0098744
name: 1-phosphatidylinositol 4-kinase activator activity
def: "Increases the activity of 1-phosphatidylinositol 4-kinase." [PMID:21288895]
GO:0098748
name: endocytic adaptor activity
def: "The binding activity of a molecule that brings together two or more protein molecules, or a protein and another macromolecule or complex as a key step in receptor mediated endocytosis." [GOC:dos, GOC:vw, PMID:21536832]
GO:0098750
name: FYXD domain binding
def: "Interacting selectively and non-covalently with a FXYD domain." [GOC:dos, GOC:mr, PMID:10950925, PMID:16403837, PMID:18000745]
GO:0098769
name: TIMP family protein binding
def: "Interacting selectively and non-covalently with any member of the Tissue inhibitors of metalloproteinases (TIMPs) family. TIMPs are endogenous protein regulators of the matrix metalloproteinase (MMPs) family" [PMID:22078297]
GO:0098770
name: FBXO family protein binding
def: "Interacting selectively and non-covalently with any member of the FBXO protein family. Members of this family have an F-box protein motif of approximately 50 amino acids that functions as a site of protein-protein interaction." [PMID:11178263]
GO:0098772
name: molecular function regulator
def: "A molecular function modulates the activity of a gene product or complex.  Examples include enzyme regulators and channel regulators." [GOC:dos, GOC:pt]
GO:1900750
name: oligopeptide binding
def: "Interacting selectively and non-covalently with an oligopeptide." [GOC:TermGenie, PMID:21854595]
GO:1900995
name: ubiquinone-6 binding
def: "Interacting selectively and non-covalently with ubiquinone-6. Ubiquinone-6 is a ubiquinone compound having a (2E,6E,10E,14E,18E)-3,7,11,15,19,23-hexamethyltetracosa-2,6,10,14,18,22-hexaen-1-yl substituent at position 2." [CHEBI:52971, GOC:al, GOC:TermGenie]
GO:1901149
name: salicylic acid binding
def: "Interacting selectively and non-covalently with salicylic acid." [GOC:TermGenie, PMID:22699612]
GO:1901235
name: (R)-carnitine transmembrane transporter activity
def: "Catalysis of the transfer of (R)-carnitine from one side of the membrane to the other." [GOC:TermGenie, PMID:16365042, PMID:20357772, PMID:20829798]
GO:1901236
name: 4-(trimethylammonio)butanoate transmembrane transporter activity
def: "Catalysis of the transfer of 4-(trimethylammonio)butanoate from one side of the membrane to the other." [GOC:TermGenie, PMID:16952940, PMID:21784948]
GO:1901237
name: tungstate transmembrane transporter activity
def: "Catalysis of the transfer of tungstate from one side of the membrane to the other." [GOC:TermGenie, PMID:16952940, PMID:21784948]
GO:1901238
name: tungstate transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + tungstate(in) = ADP + phosphate + tungstate(out)." [GOC:TermGenie, PMID:16952940, PMID:21784948]
GO:1901239
name: malonate(1-) transmembrane transporter activity
def: "Catalysis of the transfer of malonate(1-) from one side of the membrane to the other." [GOC:TermGenie, PMID:9128730, PMID:9573154]
GO:1901241
name: 4-hydroxyphenylacetate transmembrane transporter activity
def: "Catalysis of the transfer of 4-hydroxyphenylacetate from one side of the membrane to the other." [GOC:TermGenie, PMID:9315705]
GO:1901242
name: doxorubicin transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + doxorubicin(in) = ADP + phosphate + doxorubicin(out)." [GOC:TermGenie, PMID:12057006]
GO:1901243
name: methionine transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + methionine(in) = ADP + phosphate + methionine(out)." [GOC:TermGenie, PMID:12169620, PMID:12819857]
GO:1901265
name: nucleoside phosphate binding
def: "Interacting selectively and non-covalently with nucleoside phosphate." [GOC:TermGenie]
GO:1901338
name: catecholamine binding
def: "Interacting selectively and non-covalently with catecholamine." [GOC:TermGenie]
GO:1901359
name: tungstate binding
def: "Interacting selectively and non-covalently with tungstate." [GOC:TermGenie]
GO:1901363
name: heterocyclic compound binding
def: "Interacting selectively and non-covalently with heterocyclic compound." [GOC:TermGenie]
GO:1901368
name: NAD transmembrane transporter activity
def: "Catalysis of the transfer of NAD from one side of the membrane to the other." [GOC:TermGenie]
GO:1901375
name: acetate ester transmembrane transporter activity
def: "Catalysis of the transfer of an acetate ester from one side of the membrane to the other." [GOC:TermGenie]
GO:1901474
name: azole transmembrane transporter activity
def: "Catalysis of the transfer of azole from one side of the membrane to the other." [GOC:TermGenie]
GO:1901476
name: carbohydrate transporter activity
def: "Enables the directed movement of carbohydrate into, out of or within a cell, or between cells." [GOC:TermGenie]
GO:1901478
name: aminotriazole transmembrane transporter activity
def: "Catalysis of the transfer of amitrole from one side of the membrane to the other." [GOC:TermGenie]
GO:1901479
name: benomyl transmembrane transporter activity
def: "Catalysis of the transfer of benomyl from one side of the membrane to the other." [GOC:TermGenie]
GO:1901480
name: oleate transporter activity
def: "Enables the directed movement of oleate into, out of or within a cell, or between cells." [GOC:TermGenie, RHEA:33658]
GO:1901505
name: carbohydrate derivative transporter activity
def: "Enables the directed movement of carbohydrate derivative into, out of or within a cell, or between cells." [GOC:pr, GOC:TermGenie]
GO:1901513
name: lipo-chitin oligosaccharide transmembrane transporter activity
def: "Catalysis of the transfer of lipo-chitin oligosaccharide from one side of the membrane to the other." [GOC:TermGenie]
GO:1901514
name: lipo-chitin oligosaccharide transmembrane-transporting ATPase activity
def: "Catalysis of the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: ATP + H2O + lipo-chitin oligosaccharide(in) = ADP + phosphate + lipo-chitin oligosaccharide(out)." [GOC:TermGenie]
GO:1901515
name: poly-beta-1,6-N-acetyl-D-glucosamine transmembrane transporter activity
def: "Catalysis of the transfer of poly-beta-1,6-N-acetyl-D-glucosamine from one side of the membrane to the other." [GOC:TermGenie]
GO:1901531
name: hypochlorite binding
def: "Interacting selectively and non-covalently with hypochlorite." [GOC:pr, GOC:TermGenie, PMID:22223481]
GO:1901549
name: malonic acid uptake transmembrane transporter activity
def: "Catalysis of the transfer of malonic acid from the outside of a cell to the inside across a membrane." [GOC:TermGenie]
GO:1901567
name: fatty acid derivative binding
def: "Interacting selectively and non-covalently with fatty acid derivative." [GOC:pr, GOC:TermGenie]
GO:1901584
name: tetrapeptide transmembrane transporter activity
def: "Catalysis of the transfer of tetrapeptide from one side of the membrane to the other." [GOC:TermGenie]
GO:1901602
name: dethiobiotin binding
def: "Interacting selectively and non-covalently with dethiobiotin." [GOC:TermGenie]
GO:1901603
name: biotin transmembrane transporter activity
def: "Catalysis of the transfer of biotin from one side of the membrane to the other." [GOC:TermGenie]
GO:1901604
name: dethiobiotin transmembrane transporter activity
def: "Catalysis of the transfer of dethiobiotin from one side of the membrane to the other." [GOC:TermGenie]
GO:1901611
name: phosphatidylglycerol binding
def: "Interacting selectively and non-covalently with phosphatidylglycerol." [GOC:kmv, GOC:TermGenie]
GO:1901612
name: cardiolipin binding
def: "Interacting selectively and non-covalently with cardiolipin." [GOC:kmv, GOC:TermGenie]
GO:1901618
name: organic hydroxy compound transmembrane transporter activity
def: "Catalysis of the transfer of organic hydroxy compound from one side of the membrane to the other." [GOC:pr, GOC:TermGenie]
GO:1901640
name: XTP binding
def: "Interacting selectively and non-covalently with XTP." [GOC:TermGenie]
GO:1901641
name: ITP binding
def: "Interacting selectively and non-covalently with ITP." [GOC:TermGenie]
GO:1901677
name: phosphate transmembrane transporter activity
def: "Catalysis of the transfer of phosphate from one side of the membrane to the other." [GOC:pr, GOC:TermGenie]
GO:1901680
name: sulfur-containing amino acid secondary active transmembrane transporter activity
def: "Catalysis of the transfer of sulfur-containing amino acid from one side of the membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Chemiosmotic sources of energy include uniport, symport or antiport." [GOC:pr, GOC:TermGenie]
GO:1901681
name: sulfur compound binding
def: "Interacting selectively and non-covalently with a sulfur compound." [GOC:pr, GOC:TermGenie]
GO:1901682
name: sulfur compound transmembrane transporter activity
def: "Catalysis of the transfer of a sulfur compound from one side of the membrane to the other." [GOC:pr, GOC:TermGenie]
GO:1901683
name: arsenate ion transmembrane transporter activity
def: "Catalysis of the transfer of an arsenate ion from one side of the membrane to the other." [GOC:TermGenie]
GO:1901691
name: proton binding
def: "Interacting selectively and non-covalently with proton." [GOC:TermGenie]
GO:1901702
name: salt transmembrane transporter activity
def: "Catalysis of the transfer of salt from one side of the membrane to the other." [GOC:pr, GOC:TermGenie]
GO:1901707
name: leptomycin B binding
def: "Interacting selectively and non-covalently with leptomycin B." [GOC:TermGenie]
GO:1901916
name: protein kinase activity involved in regulation of protein localization to cell division site involved in cytokinesis
def: "Any protein kinase activity that is involved in regulation of protein localization to cell division site involved in cytokinesis." [GOC:dph, GOC:TermGenie, PMID:22573892]
GO:1901973
name: proline binding
def: "Interacting selectively and non-covalently with proline." [GOC:pm, GOC:TermGenie, PMID:7730362]
GO:1901974
name: glycerate transmembrane transporter activity
def: "Catalysis of the transfer of glycerate from one side of the membrane to the other." [GOC:TermGenie, pmid:23382251]
GO:1901981
name: phosphatidylinositol phosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol phosphate." [GOC:TermGenie, PMID:23445487]
GO:1901982
name: maltose binding
def: "Interacting selectively and non-covalently with maltose." [GOC:TermGenie, PMID:21566157]
GO:1902051
name: (25S)-Delta(4)-dafachronate binding
def: "Interacting selectively and non-covalently with (25S)-Delta(4)-dafachronate." [GOC:TermGenie, PMID:16529801]
GO:1902052
name: (25S)-Delta(7)-dafachronate binding
def: "Interacting selectively and non-covalently with (25S)-Delta(7)-dafachronate." [GOC:TermGenie, PMID:16529801]
GO:1902098
name: calcitriol binding
def: "Interacting selectively and non-covalently with calcitriol. Calcitriol (1,25-dihydroxycholecalciferol) is the hormonally active form of vitamin D3." [GOC:TermGenie, PMID:21872797, Wikipedia:Calcitriol_receptor]
GO:1902118
name: calcidiol binding
def: "Interacting selectively and non-covalently with calcidiol." [GOC:bf, GOC:TermGenie, PMID:11799400]
GO:1902121
name: lithocholic acid binding
def: "Interacting selectively and non-covalently with lithocholic acid." [GOC:bf, GOC:TermGenie, PMID:20371703]
GO:1902122
name: chenodeoxycholic acid binding
def: "Interacting selectively and non-covalently with chenodeoxycholic acid." [GOC:bf, GOC:TermGenie, PMID:10334992]
GO:1902248
name: 5-O-phosphono-alpha-D-ribofuranosyl diphosphate binding
def: "Interacting selectively and non-covalently with 5-O-phosphono-alpha-D-ribofuranosyl diphosphate." [GOC:mah, GOC:TermGenie, PMID:4314233]
GO:1902249
name: IMP binding
def: "Interacting selectively and non-covalently with IMP, inosine monophosphate." [GOC:mah, GOC:TermGenie, PMID:4314233]
GO:1902271
name: D3 vitamins binding
def: "Interacting selectively and non-covalently with D3 vitamins." [GOC:bf, GOC:TermGenie, PMID:9127467]
GO:1902282
name: voltage-gated potassium channel activity involved in ventricular cardiac muscle cell action potential repolarization
def: "Catalysis of the transmembrane transfer of a potassium ion by a voltage-gated channel through the plasma membrane of a ventricular cardiomyocyte contributing to the repolarization phase of an action potential. A voltage-gated channel is a channel whose open state is dependent on the voltage across the membrane in which it is embedded." [GOC:BHF, GOC:mtg_cardiac_conduct_nov11, GOC:rl, GOC:TermGenie, PMID:8528244]
GO:1902301
name: galactarate transmembrane transporter activity
def: "Catalysis of the transfer of galactaric acid anion (galactarate) from one side of the membrane to the other." [GOC:pr, GOC:TermGenie]
GO:1902314
name: hydroquinone binding
def: "Interacting selectively and non-covalently with hydroquinone." [GOC:bhm, GOC:TermGenie, pmid:15667223]
GO:1902379
name: chemoattractant activity involved in axon guidance
def: "Any chemoattractant activity that is involved in axon guidance." [GOC:BHF, GOC:rl, GOC:TermGenie, PMID:21658587]
GO:1902387
name: ceramide 1-phosphate binding
def: "Interacting selectively and non-covalently with ceramide 1-phosphate." [GOC:TermGenie, PMID:23863933]
GO:1902388
name: ceramide 1-phosphate transporter activity
def: "Enables the directed movement of ceramide 1-phosphate into, out of or within a cell, or between cells." [GOC:TermGenie, PMID:23863933]
GO:1902417
name: (+)-abscisic acid D-glucopyranosyl ester transmembrane transporter activity
def: "Catalysis of the transfer of (+)-abscisic acid D-glucopyranosyl ester from one side of the membrane to the other." [GOC:TermGenie, PMID:24028845]
GO:1902444
name: riboflavin binding
def: "Interacting selectively and non-covalently with riboflavin." [GOC:TermGenie, PMID:12083520]
GO:1902485
name: L-cysteine binding
def: "Interacting selectively and non-covalently with L-cysteine." [GOC:bhm, GOC:TermGenie, PMID:12941942]
GO:1902496
name: protein binding involved in negative regulation of telomere maintenance via telomerase
def: "Any protein binding that is involved in negative regulation of telomere maintenance via telomerase." [GOC:dph, GOC:TermGenie, GOC:vw, PMID:24013504]
GO:1902516
name: sn-glycerol 3-phosphate binding
def: "Interacting selectively and non-covalently with sn-glycerol 3-phosphate." [GOC:bhm, GOC:TermGenie, PMID:23013274]
GO:1902557
name: 5'-adenylyl sulfate transmembrane transporter activity
def: "Catalysis of the transfer of 5'-adenylyl sulfate from one side of the membrane to the other." [GOC:TermGenie, PMID:24296033]
GO:1902670
name: carbon dioxide binding
def: "Interacting selectively and non-covalently with carbon dioxide." [GO_REF:0000067, GOC:bhm, GOC:TermGenie, PMID:15491402]
GO:1902936
name: phosphatidylinositol bisphosphate binding
def: "Interacting selectively and non-covalently with phosphatidylinositol bisphosphate." [GO_REF:0000067, GOC:bhm, GOC:TermGenie, PMID:18690034]
GO:1902944
name: aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process
def: "Any aspartic-type endopeptidase activity that is involved in amyloid precursor protein catabolic process." [GO_REF:0000061, GOC:sjp, GOC:TermGenie, PMID:10206644, PMID:24577224]
GO:1902945
name: metalloendopeptidase activity involved in amyloid precursor protein catabolic process
def: "Any metalloendopeptidase activity that is involved in amyloid precursor protein catabolic process." [GO_REF:0000061, GOC:sjp, GOC:TermGenie, PMID:14598310, PMID:17855360]
GO:1903089
name: 5-amino-1-ribofuranosylimidazole-4-carboxamide transporter activity
def: "Enables the directed movement of 5-amino-1-ribofuranosylimidazole-4-carboxamide into, out of or within a cell, or between cells." [GO_REF:0000066, GOC:TermGenie, PMID:24778186]
GO:1903135
name: cupric ion binding
def: "Interacting selectively and non-covalently with cupric ion, copper(2+)." [GO_REF:0000067, GOC:bf, GOC:PARL, GOC:TermGenie, PMID:24567322]
GO:1903136
name: cuprous ion binding
def: "Interacting selectively and non-covalently with cuprous ion, copper(1+)." [GO_REF:0000067, GOC:bf, GOC:PARL, GOC:TermGenie, PMID:24567322]
GO:1903148
name: uracil transmembrane transporter activity involved in uracil import into cell
def: "OBSOLETE. Any uracil transmembrane transporter activity that is involved in uracil import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903149
name: adenine transmembrane transporter activity involved in adenine import into cell
def: "OBSOLETE. Any adenine transmembrane transporter activity that is involved in adenine import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903150
name: calcium ion transmembrane transporter activity involved in calcium ion import into cell
def: "OBSOLETE. Any calcium ion transmembrane transporter activity that is involved in calcium ion import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903151
name: carbohydrate transmembrane transporter activity involved in carbohydrate import into cell
def: "OBSOLETE. Any carbohydrate transmembrane transporter activity that is involved in carbohydrate import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903152
name: copper ion transmembrane transporter activity involved in copper ion import into cell
def: "OBSOLETE. Any copper ion transmembrane transporter activity that is involved in copper ion import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903153
name: ferrous iron transmembrane transporter activity involved in ferrous iron import into cell
def: "OBSOLETE. Any ferrous iron transmembrane transporter activity that is involved in ferrous iron import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903154
name: glucose transmembrane transporter activity involved in glucose import into cell
def: "OBSOLETE. Any glucose transmembrane transporter activity that is involved in glucose import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903155
name: glutathione transmembrane transporter activity involved in glutathione import into cell
def: "OBSOLETE. Any glutathione transmembrane transporter activity that is involved in glutathione import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903156
name: guanine transmembrane transporter activity involved in guanine import into cell
def: "OBSOLETE. Any guanine transmembrane transporter activity that is involved in guanine import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903157
name: iron ion transmembrane transporter activity involved in iron ion import into cell
def: "OBSOLETE. Any iron ion transmembrane transporter activity that is involved in iron ion import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903158
name: L-glutamate transmembrane transporter activity involved in L-glutamate import into cell
def: "OBSOLETE. Any L-glutamate transmembrane transporter activity that is involved in L-glutamate import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903159
name: malate transmembrane transporter activity involved in malate import into cell
def: "OBSOLETE. Any malate transmembrane transporter activity that is involved in malate import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903160
name: nickel cation transmembrane transporter activity involved in nickel cation import into cell
def: "OBSOLETE. Any nickel cation transmembrane transporter activity that is involved in nickel cation import into cell." [GO_REF:0000061, GOC:dos, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903161
name: pantothenate transmembrane transporter activity involved in pantothenate import into cell
def: "OBSOLETE. Any pantothenate transmembrane transporter activity that is involved in pantothenate import into cell." [GO_REF:0000061, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903162
name: serine transmembrane transporter activity involved in serine import into cell
def: "OBSOLETE. Any serine transmembrane transporter activity that is involved in serine import into cell." [GO_REF:0000061, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903163
name: sodium ion transmembrane transporter activity involved in sodium ion import into cell
def: "OBSOLETE. Any sodium ion transmembrane transporter activity that is involved in sodium ion import into cell." [GO_REF:0000061, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903164
name: succinate transmembrane transporter activity involved in succinate import into cell
def: "OBSOLETE. Any succinate transmembrane transporter activity that is involved in succinate import into cell." [GO_REF:0000061, GOC:TermGenie, ISBN:0-8249-3695-6]
GO:1903230
name: miRNA binding involved in posttranscriptional gene silencing
def: "OBSOLETE. Any miRNA binding that is involved in posttranscriptional gene silencing." [GO_REF:0000061, GOC:BHF, GOC:jl, GOC:TermGenie]
GO:1903231
name: mRNA binding involved in posttranscriptional gene silencing
def: "Any mRNA binding that is involved in posttranscriptional gene silencing." [GO_REF:0000061, GOC:BHF, GOC:BHF_miRNA, GOC:jl, GOC:TermGenie]
GO:1903264
name: nitrate reductase activity involved in anaerobic electron transport chain
def: "Any nitrate reductase activity that is involved in anaerobic electron transport chain." [GO_REF:0000061, GOC:dos, GOC:TermGenie, PMID:12910261]
GO:1903425
name: fluoride transmembrane transporter activity
def: "Catalysis of the transfer of fluoride from one side of the membrane to the other." [GO_REF:0000070, GOC:TermGenie, PMID:24173035]
GO:1903763
name: gap junction channel activity involved in cell communication by electrical coupling
def: "Any gap junction channel activity that is involved in cell communication by electrical coupling." [GO_REF:0000061, GOC:BHF, GOC:mtg_cardiac_conduct_nov11, GOC:rl, GOC:TermGenie, PMID:24587307]
GO:1903777
name: melibiose binding
def: "Interacting selectively and non-covalently with melibiose." [GO_REF:0000067, GOC:mr, GOC:TermGenie, PMID:11471732]
GO:1903794
name: cortisol binding
def: "Interacting selectively and non-covalently with cortisol." [GO_REF:0000067, GOC:mr, GOC:TermGenie, PMID:18483153]
GO:1990002
name: methylglyoxal reductase (NADPH-dependent, acetol producing)
def: "Catalysis of the reaction: H+ + methylglyoxal + NADPH <=> hydroxyacetone + NADP+." [EC:1.1.1.-, MetaCyc:RXN0-4281, PMID:16077126, RHEA:27989]
GO:1990003
name: IDP phosphatase activity
def: "Catalysis of the reaction: IDP + H2O = IMP + phosphate." [PMID:20385596]
GO:1990004
name: XDP phosphatase activity
def: "Catalysis of the reaction: XDP + H2O = XMP + phosphate." [PMID:20385596]
GO:1990028
name: intermediate voltage-gated calcium channel activity
def: "Catalysis of the transmembrane transfer of a calcium ion by an intermediate voltage-gated channel. An intermediate voltage-gated channel is a channel whose open state is dependent on intermediate voltage across the membrane in which it is embedded." [GOC:BHF, GOC:rl, http://en.wikipedia.org/wiki/Calcium_channel, PMID:16382099]
GO:1990042
name: glycerol dehydrogenase [NAD(P)+] activity
def: "Catalysis of an oxidation-reduction (redox) reaction in which glycerol is converted into glycerone and NAD+ or NADP is reduced." [PMID:22979944]
GO:1990043
name: 5' deoxyribonuclease (pyrimidine dimer) activity
def: "Catalysis of the endonucleolytic cleavage immediately 5' to pyrimidine dimers to products with 5'-phosphate." [EC:3.1.25.-, GOC:al, PMID:9708997]
GO:1990050
name: phosphatidic acid transporter activity
def: "Enables the directed movement of phosphatidic acid into, out of or within a cell, or between cells. Phosphatidic acid refers to a glycophospholipids with, in general, a saturated fatty acid bonded to carbon-1, an unsaturated fatty acid bonded to carbon-2, and a phosphate group bonded to carbon-3." [PMID:23042293]
GO:1990053
name: DNA-5-methylcytosine glycosylase activity
def: "Catalysis of the reaction: DNA containing 5-methylcytosine + H2O = DNA with abasic site + 5-methylcytosine. This reaction is the hydrolysis of DNA by cleavage of the N-C1' glycosidic bond between the DNA 5-methylcytosine and the deoxyribose sugar to remove the 5-methylcytosine, leaving an abasic site." [PMID:23316050]
GO:1990055
name: phenylacetaldehyde synthase activity
def: "Catalyzes the reaction: L-phenylalanine + O2 + H2O -> phenylacetaldehyde + ammonia + hydrogen peroxide + CO2." [MetaCyc:RXN-8990, PMID:16766535, PMID:23204519]
GO:1990056
name: protein kinase activity required for targeting substrate to proteasomal ubiquitin-dependent protein catabolic process
def: "OBSOLETE. Catalysis of the phosphorylation of an amino acid residue in a substrate protein, usually according to the reaction: a protein + ATP = a phosphoprotein + ADP, thereby targeting the substrate to the proteasomal ubiquitin mediated protein catabolic process" [PMID:21098119, PMID:21993622, PMID:23264631]
GO:1990080
name: 2-phenylethylamine receptor activity
def: "Combining with the biogenic amine 2-phenylethylamine to initiate a change in cell activity." [PMID:16878137]
GO:1990081
name: trimethylamine receptor activity
def: "Combining with the biogenic amine trimethylamine to initiate a change in cell activity." [PMID:16878137]
GO:1990088
name: [methyl-Co(III) methanol-specific corrinoid protein]:coenzyme M methyltransferase activity
def: "Catalysis of the reaction: a [methyl-Co(III) methanol-specific corrinoid protein] + coenzyme M = methyl-CoM + a [Co(I) methanol-specific corrinoid protein]." [GOC:hjd, PMID:10077852]
GO:1990107
name: thiazole synthase activity
def: "Catalysis of the reaction: 1-deoxy-D-xylulose 5-phosphate + 2-iminoacetate + thiocarboxy-adenylate-[sulfur-carrier protein ThiS] = 2-[(2R,5Z)-2-carboxy-4-methylthiazol-5(2H)-ylidene]ethyl phosphate + [sulfur-carrier protein ThiS] + 2 H2O." [EC:2.8.1.10, GOC:cjk, PMID:22031445, RHEA:26300]
GO:1990119
name: ATP-dependent RNA helicase inhibitor activity
def: "Stops, prevents or reduces the activity of ATP-dependent RNA helicase." [GOC:rb, PMID:23721653]
GO:1990135
name: flavonoid sulfotransferase activity
def: "Catalysis of the reaction: a flavonoid + 3'-phosphoadenosine-5'-phosphosulfate = sulfated flavonoid + adenosine-3',5'-diphosphate. This reaction is the transfer of a sulfate group to the hydroxyl group of a flavonoid acceptor, producing the sulfated flavonoid derivative." [PMID:23611783]
GO:1990136
name: linoleate 9S-lipoxygenase activity
def: "Catalysis of the reaction: linoleate + O2 = (9S,10E,12Z)-9-hydroperoxy-10,12-octadecadienoate." [EC:1.13.11.58, GOC:rph, RHEA:30294]
GO:1990137
name: plant seed peroxidase activity
def: "Catalysis of the reaction: R1H + R2OOH = R1OH + R2OH." [PMID:19467604]
GO:1990147
name: talin binding
def: "Interacting selectively and non-covalently with a talin, a family of related cytoskeletal proteins that play a role in assembly of actin filaments and migration of various cell types." [GOC:hjd, PMID:23372168]
GO:1990162
name: histone deacetylase activity (H3-K4 specific)
def: "Catalysis of the reaction: histone H3 N6-acetyl-L-lysine (position 4) + H2O = histone H3 L-lysine (position 4) + acetate. This reaction represents the removal of an acetyl group from lysine at position 4 of the histone H3 protein." [EC:3.5.1.98, GOC:al, PMID:23771057]
GO:1990163
name: ATP-dependent four-way junction helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, where this reaction drives the unwinding of the DNA helix of DNA containing four-way junctions, including Holliday junctions." [GOC:al, PMID:22723423]
GO:1990165
name: single-strand break-containing DNA binding
def: "Interacting selectively and non-covalently with damaged DNA containing single-strand breaks (SSBs)." [GOC:al, PMID:21984210]
GO:1990174
name: phosphodiesterase decapping endonuclease activity
def: "Catalysis of the removal of the cap from an unmethylated 5'-end capped RNA resulting in the release of the entire cap structure (GpppN) and a 5' monophosphorylated RNA." [GOC:dgf, PMID:20802481]
GO:1990175
name: EH domain binding
def: "Interacting selectively and non-covalently with an EH domain of a protein. The EH stand for Eps15 homology. This was originally identified as a motif present in three copies at the NH2-termini of Eps15 and of the related molecule Eps15R." [GOC:hjd, PMID:11911876, PMID:21115825]
GO:1990188
name: euchromatin binding
def: "Interacting selectively and non-covalently with euchromatin, a dispersed and relatively uncompacted form of chromatin." [GOC:vw, PMID:22431512]
GO:1990189
name: peptide-serine-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + N-terminal L-serine in peptide = CoA + N-acetyl-L-serine-peptide." [GOC:al, PMID:23912279]
GO:1990190
name: peptide-glutamate-N-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + N-terminal L-glutamate in peptide = CoA + N-acetyl-L-glutamate-peptide." [GOC:al, PMID:23912279]
GO:1990206
name: jasmonyl-Ile conjugate hydrolase activity
def: "Catalysis of the reaction: jasmonyl-Ile + H2O = jasmonic acid + L-isoleucine." [PMID:23943861]
GO:1990224
name: NMN phosphatase activity
def: "Catalysis of the reaction: beta-nicotinamide D-ribonucleotide (NMN-) + H2O = beta-nicotinamide D-riboside (nicotinamide ribose, NmR) + phosphate." [GOC:rb, PMID:21349851, RHEA:30818]
GO:1990226
name: histone methyltransferase binding
def: "Interacting selectively and non-covalently with a histone methyltransferase enzyme." [GOC:ame, GOC:BHF, PMID:19486527]
GO:1990238
name: double-stranded DNA endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within a double-stranded deoxyribonucleic acid molecule by creating internal breaks." [GOC:PG, PMID:22885404]
GO:1990239
name: steroid hormone binding
def: "Interacting selectively and non-covalently with a steroid hormone." [GOC:ln]
GO:1990240
name: methionine-importing activity
def: "Catalysis of the transfer of methionine into a cell or organelle, across a membrane." [GOC:pr]
GO:1990244
name: histone kinase activity (H2A-T120 specific)
def: "Catalysis of the transfer of a phosphate group to the threonine-120 residue of histone H2A." [PMID:24140421]
GO:1990247
name: N6-methyladenosine-containing RNA binding
def: "Interacting selectively and non-covalently with an RNA molecule modified by N6-methyladenosine (m6A), a modification present at internal sites of mRNAs and some non-coding RNAs." [PMID:22575960, PMID:24284625]
GO:1990254
name: keratin filament binding
def: "Interacting selectively and non-covalently with a keratin filament, an intermediate filament composed of acidic and basic keratins (types I and II), typically expressed in epithelial cells." [GOC:krc, PMID:6170061]
GO:1990259
name: histone-glutamine methyltransferase activity
def: "Catalysis of the reaction: S-adenosyl-L-methionine + (histone)-glutamine = S-adenosyl-L-homocysteine + (histone)-N5-methyl-glutamine." [PMID:24352239]
GO:1990269
name: RNA polymerase II C-terminal domain phosphoserine binding
def: "Interacting selectively and non-covalently with phosphorylated serine residues in the C-terminal domain of RNA polymerase II." [GOC:di, PMID:22796944]
GO:1990271
name: anti-Mullerian hormone
def: "OBSOLETE. Combining with anti-Mullerian hormone to initiate a change in cell activity." [GOC:hjd, PMID:23624077]
GO:1990272
name: anti-Mullerian hormone receptor activity
def: "Combining with anti-Mullerian hormone to initiate a change in cell activity." [GOC:hjd, PMID:23624077]
GO:1990275
name: preribosome binding
def: "Interacting selectively and non-covalently with any part of a preribosome." [GOC:di, PMID:22735702]
GO:1990276
name: RNA 5'-methyltransferase activity
def: "Catalysis of the transfer of a methyl group from S-adenosyl-L-methionine to the 5'-gamma-phosphate in an RNA molecule." [GOC:al, GOC:vw, PMID:22740346]
GO:1990300
name: cellulosome binding
def: "Interacting selectively and non-covalently with the cellulosome, an extracellular multi-enzyme complex containing several enzymes aligned on a non-catalytic scaffolding that functions to hydrolyze plant cell wall polysaccharides." [PMID:11893054, PMID:15197390]
GO:1990301
name: scaffoldin complex binding
def: "A protein complex that contains one or more scaffoldin proteins and several cellulosomal enzymes ant that interacts selectively and non-covalently with the scaffoldin complex." [PMID:11893054, PMID:15197390]
GO:1990308
name: Type-I dockerin domain binding
def: "Interacting selectively and non-covalently with a type-I dockerin domain of a protein. Type-I dockerin domain is the binding partner of type-1 cohesin domain." [PMID:23195689, PMID:24080387]
GO:1990309
name: Type-II dockerin domain binding
def: "Interacting selectively and non-covalently with a type-II dockerin domain of a protein. Type-II dockerin domain is the binding partner of type-II cohesin domain." [PMID:23195689, PMID:24080387]
GO:1990310
name: Type-III dockerin domain binding
def: "Interacting selectively and non-covalently with a type-III dockerin domain of a protein. Type-III dockerin domain is the binding partner of type-III cohesin domain." [PMID:23195689, PMID:24080387]
GO:1990311
name: Type-I cohesin domain binding
def: "Interacting selectively and non-covalently with a type-I cohesin domain of a protein. Type-I cohesin domain is the binding partner of type-I dockerin domain." [PMID:23195689, PMID:24080387]
GO:1990312
name: Type-II cohesin domain binding
def: "Interacting selectively and non-covalently with a type-II cohesin domain of a protein. Type-II cohesin domain is the binding partner of type-II dockerin domain." [PMID:23195689, PMID:24080387]
GO:1990313
name: Type-III cohesin domain binding
def: "Interacting selectively and non-covalently with a type-III cohesin domain of a protein. Type-III cohesin domain is the binding partner of type-III dockerin domain." [PMID:23195689, PMID:24080387]
GO:1990347
name: G*/A mismatch-specific adenine-DNA glycosylase activity
def: "OBSOLETE. Catalysis of the removal of adenine misinserted into nascent strand opposite 8-oxoG in the template by adenine DNA glycosylase activity. The reaction leaves an apyrimidinic (AP) site." [PMID:24559510]
GO:1990348
name: G/A mismatch specific adenine DNA glycosylase activity
def: "OBSOLETE. Catalysis of the removal of adenine misinserted into nascent strand opposite guanine in the template by adenine DNA glycosylase activity. The reaction leaves an apurinic AP site. I also requested same term but for Go/A mismatch. If you think it is better to make one term for mismatched adenine that is fine by me." [PMID:9737967]
GO:1990362
name: butanol dehydrogenase activity
def: "Catalysis of the reaction: butanal + NADH + H+ => n-butanol + NAD+." [GOC:mengo_curators, PMID:1999395]
GO:1990380
name: Lys48-specific deubiquitinase activity
def: "Hydrolysis of Lys48-linked ubiquitin unit(s) from a ubiquitinated protein." [GOC:bf, GOC:PARL, PMID:22970133]
GO:1990381
name: ubiquitin-specific protease binding
def: "Interacting selectively and non-covalently with a ubiquitin-specific protease." [GOC:bf, GOC:PARL, PMID:24063750]
GO:1990400
name: mitochondrial ribosomal large subunit rRNA binding
def: "Interacting selectively and non-covalently with the mitochondrial large ribosomal subunit RNA (LSU rRNA), a constituent of the mitochondrial large ribosomal subunit." [PMID:24206665]
GO:1990404
name: protein ADP-ribosylase activity
def: "The transfer, from NAD, of ADP-ribose to a protein amino acid residue." [PMID:1899243, RESID:AA0040, RESID:AA0168, RESID:AA0169, RESID:AA0231, RESID:AA0237, RESID:AA0295, wikipedia:ADP-ribosylation]
GO:1990405
name: protein antigen binding
def: "Interacting selectively and non-covalently with a protein antigen." [PMID:9360996]
GO:1990407
name: calcitonin gene-related peptide binding
def: "Interacting selectively and non-covalently with calcitonin gene-related peptide (CGRP)." [GOC:bhm, PMID:10882736]
GO:1990409
name: adrenomedullin binding
def: "Interacting selectively and non-covalently with adrenomedullin (AM)." [GOC:bhm, PMID:10882736]
GO:1990411
name: hercynylcysteine sulfoxide lyase activity (ergothioneine-forming)
def: "Catalysis of the reaction: hercynylcysteine sulfoxide + 2H+ = ergothioneine + pyruvate + ammonium." [PMID:24828577]
GO:1990412
name: hercynylselenocysteine lyase activity (selenoneine-forming)
def: "Catalysis of the reaction: hercynylselenocysteine + 2H+ = selenoneine + pyruvate + ammonium." [PMID:24828577]
GO:1990422
name: glyoxalase (glycolic acid-forming) activity
def: "Catalysis of the reaction: glyoxal + H2O = glycolic acid. Catalysis occurs in the absence of a cofactor." [GOC:bf, GOC:PARL, PMID:22523093]
GO:1990424
name: protein arginine kinase activity
def: "Catalysis of the reaction: ATP + a protein arginine = ADP + protein arginine phosphate." [GOC:imk, PMID:22517742]
GO:1990430
name: extracellular matrix protein binding
def: "Interacting selectively and non-covalently with a protein that is part of an extracellular matrix." [PMID:22355679]
GO:1990439
name: MAP kinase threonine phosphatase activity
def: "Catalysis of the reaction: MAP kinase threonine phosphate + H2O = MAP kinase tyrosine + phosphate." [PMID:10398679]
GO:1990444
name: F-box domain binding
def: "Interacting selectively and non-covalently with an F-box domain of a protein." [GOC:bf, GOC:PARL, InterPro:IPR001810, PMID:12628165]
GO:1990446
name: U1 snRNP binding
def: "Interacting selectively and non-covalently with any part of a U1 small nuclear ribonucleoprotein particle." [PMID:14713954]
GO:1990447
name: U2 snRNP binding
def: "Interacting selectively and non-covalently with any part of a U2 small nuclear ribonucleoprotein particle." [PMID:14713954]
GO:1990448
name: exon-exon junction complex binding
def: "Interacting selectively and non-covalently with an exon-exon junction complex, a protein complex deposited by the spliceosome upstream of messenger RNA exon-exon junctions. The exon-exon junction complex provides a binding platform for factors involved in mRNA export and nonsense-mediated mRNA decay." [GOC:sart, PMID:24967911]
GO:1990450
name: linear polyubiquitin binding
def: "Interacting selectively and non-covalently with a linear polymer of ubiquitin. Linear ubiquitin polymers are formed by linking the amino-terminal methionine (M1) of one ubiquitin molecule to the carboxy-terminal glycine (G76) of the next." [GOC:bf, GOC:PARL, PMID:23453807]
GO:1990458
name: lipooligosaccharide binding
def: "Interacting selectively and non-covalently with lipooligosaccharide. Lipooligosaccharides (LOSs) are the major glycolipids expressed on mucosal Gram-negative bacteria." [GOC:hjd, PMID:8894399]
GO:1990459
name: transferrin receptor binding
def: "Interacting selectively and non-covalently with the transferrin receptor." [PMID:9819414, PMID:pm]
GO:1990460
name: leptin receptor binding
def: "Interacting selectively and non-covalently with the leptin receptor." [GOC:pm, PMID:22405007]
GO:1990464
name: D-2-hydroxyacid dehydrogenase (quinone) activity
def: "Catalysis of the reaction: (R)-2-hydroxyacid + a quinone = 2-oxoacid + a quinol." [EC:1.1.5.10, GOC:am, PMID:3013300, PMID:4582730]
GO:1990465
name: aldehyde oxygenase (deformylating) activity
def: "Catalysis of the reaction a long-chain aldehyde + O(2) + 2 NADPH = an alkane + formate + H(2)O + 2 NADP(+)." [GOC:mengo_curators, PMID:22947199]
GO:1990470
name: piRNA cluster binding
def: "Interacting selectively and non-covalently with piRNA clusters, double-stranded DNA regions that give rise to PIWI-interacting RNAs (piRNAs)." [GOC:bhm, PMID:24906153]
GO:1990471
name: piRNA uni-strand cluster binding
def: "Interacting selectively and non-covalently with uni-strand piRNA clusters, double-stranded DNA regions that give rise to PIWI-interacting RNAs (piRNAs) that map predominantly to only one strand and exhibit hallmarks of canonical Pol II transcription. Uni-strand piRNA clusters are found in many taxa." [GOC:bhm, PMID:24906153]
GO:1990472
name: piRNA dual-strand cluster binding
def: "Interacting selectively and non-covalently with dual-strand piRNA clusters, double-stranded DNA regions that give rise to PIWI-interacting RNAs (piRNAs) where piRNAs originate from both DNA strands via noncanonical transcription." [GOC:bhm, PMID:24906153]
GO:1990473
name: ciliary targeting signal binding
def: "Interacting selectively and non-covalently with a ciliary targeting sequence, a specific peptide sequence that acts as a signal to localize a membrane protein to the ciliary membrane." [GOC:krc, PMID:18256283, PMID:19575670, PMID:20603001, PMID:20697559]
GO:1990480
name: obsolete geranyl diphosphate synthase
def: "OBSOLETE. Catalyzes the condensation of dimethylallyl diphosphate and isopentenyl diphosphate to geranyl diphosphate, the key precursor of monoterpene biosynthesis." [GOC:mengo_curators, PMID:10557273]
GO:1990482
name: sphingolipid alpha-glucuronosyltransferase activity
def: "Catalysis of the reaction: UDP-glucuronate + inositol phosphorylceramide (IPC) = UDP + GlcA-IPC." [GOC:tb, PMID:25122154]
GO:1990518
name: single-stranded DNA-dependent ATP-dependent 3'-5' DNA helicase activity
def: "Catalysis of the reaction: ATP + H2O = ADP + phosphate, in the presence of single-stranded DNA; drives the unwinding of the DNA helix in the direction 3' to 5'." [PMID:25165823]
GO:1990521
name: m7G(5')pppN diphosphatase activator activity
def: "Binds to and increases the activity of m7G(5')pppN diphosphatase." [PMID:22323607]
GO:1990525
name: BIR domain binding
def: "Interacting selectively and non-covalently with a Baculovirus Inhibitor of apoptosis protein Repeat (BIR) domain." [GOC:ha, InterPro:IPR001370]
GO:1990534
name: thermospermine oxidase activity
def: "Catalysis of the reaction: S-methyl-5'-thioadenosine + thermospermine + H+ = S-adenosyl 3-(methylthio)propylamine + spermidine." [PMID:24906355]
GO:1990538
name: xylan O-acetyltransferase activity
def: "Catalysis of the reaction: acetyl-CoA + a xylan= CoA + an acetylated xylan." [PMID:25141999]
GO:1990560
name: DNA methyltransferase binding
def: "OBSOLETE. Interacting selectively and non-covalently with a DNA methyltransferase." [PMID:22880885]
GO:1990576
name: G-protein coupled glucose receptor activity
def: "Combining with an extracellular glucose molecule and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex." [PMID:15667320]
GO:1990582
name: intracellular membrane-bounded organelle binding
def: "OBSOLETE. The temporary binding of a protein or protein complex to the membrane of an intracellular membrane-bounded organelle." [PMID:16100119, PMID:9538255]
GO:1990583
name: phospholipase D activator activity
def: "Increases the activity of the enzyme phospholipase D." [PMID:7972129]
GO:1990585
name: hydroxyproline O-arabinosyltransferase activity
def: "Catalysis of the reaction: UDP-beta-L-arabinofuranose + a [protein]-trans-4-hydroxy-L-proline <=> a protein-O-(beta-L-arabinofuranose)-trans-4-hydroxy-L-proline + UDP + H+." [EC:2.4.2, PMID:24036508]
GO:1990593
name: nascent polypeptide-associated complex binding
def: "Interacting selectively and non-covalently with the nascent polypeptide-associated complex, which is a heterodimeric protein complex that can reversibly bind to ribosomes and is located in direct proximity to newly synthesized polypeptide chains as they emerge from the ribosome." [PMID:25487825]
GO:1990594
name: L-altrarate dehydratase activity
def: "Catalysis of the reaction: L-altrarate = 5-dehydro-4-deoxy-D-glucarate + H(2)O." [PMID:17649980]
GO:1990595
name: mast cell secretagogue receptor activity
def: "Combining with basic secretagogues to initiate pseudo-allergic reactions in mast cells." [GOC:sp, PMID:25517090]
GO:1990599
name: 3' overhang single-stranded DNA endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within 3' overhang single-stranded deoxyribonucleic acid by creating internal breaks." [PMID:25203555]
GO:1990600
name: single-stranded DNA endodeoxyribonuclease activator activity
def: "Increases the activity of a single-stranded DNA endodeoxyribonuclease activator activity." [PMID:25203555]
GO:1990601
name: 5' overhang single-stranded DNA endodeoxyribonuclease activity
def: "Catalysis of the hydrolysis of ester linkages within 5' overhang single-stranded deoxyribonucleic acid by creating internal breaks." [PMID:25203555]
GO:1990605
name: GU repeat RNA binding
def: "Interacting selectively and non-covalently with an RNA molecule containing GU repeats." [PMID:20081200]
GO:1990606
name: membrane scission GTPase motor activity
def: "Catalysis of the generation of a 'twisting' activity resulting in the scission of a membrane, coupled to the hydrolysis of a nucleoside triphosphate." [PMID:11242086, PMID:23530241, PMID:24515348]
GO:2001062
name: xylan binding
def: "Interacting selectively and non-covalently with xylan." [GOC:mengo_curators]
GO:2001063
name: glucomannan binding
def: "Interacting selectively and non-covalently with glucomannan." [GOC:mengo_curators]
GO:2001064
name: cellooligosaccharide binding
def: "Interacting selectively and non-covalently with cellooligosaccharide." [GOC:mengo_curators]
GO:2001065
name: mannan binding
def: "Interacting selectively and non-covalently with mannan." [GOC:mengo_curators]
GO:2001066
name: amylopectin binding
def: "Interacting selectively and non-covalently with amylopectin." [GOC:mengo_curators]
GO:2001067
name: pullulan binding
def: "Interacting selectively and non-covalently with pullulan." [GOC:mengo_curators]
GO:2001068
name: arabinoxylan binding
def: "Interacting selectively and non-covalently with arabinoxylan." [GOC:mengo_curators]
GO:2001069
name: glycogen binding
def: "Interacting selectively and non-covalently with glycogen." [GOC:mengo_curators]
GO:2001070
name: starch binding
def: "Interacting selectively and non-covalently with starch." [GOC:mengo_curators]
GO:2001071
name: maltoheptaose binding
def: "Interacting selectively and non-covalently with maltoheptaose." [GOC:mengo_curators]
GO:2001072
name: galactomannan binding
def: "Interacting selectively and non-covalently with galactomannan." [GOC:mengo_curators]
GO:2001073
name: cyclodextrin binding
def: "Interacting selectively and non-covalently with cyclodextrin." [GOC:mengo_curators]
GO:2001077
name: (1->3),(1->4)-beta-glucan binding
def: "Interacting selectively and non-covalently with (1->3),(1->4)-beta-glucan." [GOC:mengo_curators]
GO:2001078
name: (1->6)-beta-D-glucan binding
def: "Interacting selectively and non-covalently with (1->6)-beta-D-glucan." [GOC:mengo_curators]
GO:2001079
name: beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->3)-beta-D-Gal-(1->4)-D-Glc binding
def: "Interacting selectively and non-covalently with beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->3)-beta-D-Gal-(1->4)-D-Glc." [GOC:mengo_curators]
GO:2001080
name: chitosan binding
def: "Interacting selectively and non-covalently with chitosan." [GOC:mengo_curators]
GO:2001081
name: (1->4)-beta-D-galactan binding
def: "Interacting selectively and non-covalently with (1->4)-beta-D-galactan." [GOC:mengo_curators]
GO:2001082
name: inulin binding
def: "Interacting selectively and non-covalently with inulin." [GOC:mengo_curators]
GO:2001083
name: alpha-D-glucan binding
def: "Interacting selectively and non-covalently with alpha-D-glucan." [GOC:mengo_curators]
GO:2001084
name: L-arabinofuranose binding
def: "Interacting selectively and non-covalently with L-arabinofuranose." [GOC:mengo_curators]
GO:2001085
name: arabinogalactan binding
def: "Interacting selectively and non-covalently with arabinogalactan." [GOC:mengo_curators]
GO:2001147
name: camalexin binding
def: "Interacting selectively and non-covalently with camalexin." [GOC:obol]
GO:2001227
name: quercitrin binding
def: "Interacting selectively and non-covalently with quercitrin." [GOC:obol]
